
<html lang="en"     class="pb-page"  data-request-id="fe0a4513-3068-4880-8460-b9b47b80754c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c00929;requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-14;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Novel PDEδ Degraders for the Treatment of KRAS Mutant Colorectal Cancer" /></meta><meta name="dc.Creator" content="Junfei  Cheng" /></meta><meta name="dc.Creator" content="Yu  Li" /></meta><meta name="dc.Creator" content="Xu  Wang" /></meta><meta name="dc.Creator" content="Guoqiang  Dong" /></meta><meta name="dc.Creator" content="Chunquan  Sheng" /></meta><meta name="dc.Description" content="KRAS-PDEδ protein–protein interaction represents an appealing target for cancer therapy. However, fast release of high-affinity inhibitors from PDEδ hampered drug binding affinity and antiprolifera..." /></meta><meta name="Description" content="KRAS-PDEδ protein–protein interaction represents an appealing target for cancer therapy. However, fast release of high-affinity inhibitors from PDEδ hampered drug binding affinity and antiprolifera..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 30, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00929" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00929" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00929" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00929" /></link>
        
    
    

<title>Discovery of Novel PDEδ Degraders for the Treatment of KRAS Mutant Colorectal Cancer | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00929" /></meta><meta property="og:title" content="Discovery of Novel PDEδ Degraders for the Treatment of KRAS Mutant Colorectal Cancer" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0012.jpeg" /></meta><meta property="og:description" content="KRAS-PDEδ protein–protein interaction represents an appealing target for cancer therapy. However, fast release of high-affinity inhibitors from PDEδ hampered drug binding affinity and antiproliferative activity. To overcome the limitations, the first proteolysis-targeting chimeric (PROTAC) small molecules targeting PDEδ were designed. By employment of PDEδ inhibitor deltazinone (2) and cereblon ligand pomalidomide (6), a series of potent PROTAC PDEδ degraders were obtained. The most promising compound 17f efficiently induced PDEδ degradation and demonstrated significantly improved antiproliferative potency in KRAS mutant SW480 cells. Compound 17f also achieved significant tumor growth inhibition in the SW480 colorectal cancer xenograft model. This proof-of-concept study provided a new strategy to validate the druggability of KRAS-PDEδ interaction and offered an effective lead compound for the treatment of KRAS mutant cancer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00929"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00929">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00929&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00929&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00929&amp;href=/doi/10.1021/acs.jmedchem.0c00929" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 7892-7905</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00807" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01003" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Novel PDEδ Degraders for the Treatment of KRAS Mutant Colorectal Cancer</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Junfei Cheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Junfei Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, School of Pharmacy, Second Military Medical University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Junfei++Cheng">Junfei Cheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, School of Pharmacy, Second Military Medical University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Li">Yu Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xu Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xu Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, School of Pharmacy, Second Military Medical University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xu++Wang">Xu Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guoqiang Dong</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guoqiang Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, School of Pharmacy, Second Military Medical University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</div></div><span class="conrtib-corresp"><strong>*</strong>G.D.: email, <a href="/cdn-cgi/l/email-protection#385c5f4915000978090e0b165b5755"><span class="__cf_email__" data-cfemail="016566702c3930413037322f626e6c">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guoqiang++Dong">Guoqiang Dong</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Chunquan Sheng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chunquan Sheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, School of Pharmacy, Second Military Medical University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</div></div><span class="conrtib-corresp"><strong>*</strong>C.S.: email, <a href="/cdn-cgi/l/email-protection#fc8f9499929b9f8dbc8f919189d2999889d29f92"><span class="__cf_email__" data-cfemail="addec5c8c3cacedceddec0c0d883c8c9d883cec3">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chunquan++Sheng">Chunquan Sheng</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9489-804X" title="Orcid link">http://orcid.org/0000-0001-9489-804X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00929&amp;href=/doi/10.1021%2Facs.jmedchem.0c00929" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 7892–7905</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 30, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 June 2020</li><li><span class="item_label"><b>Published</b> online</span>30 June 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 July 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00929" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00929</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7892%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJunfei%2BCheng%252C%2BYu%2BLi%252C%2BXu%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D14%26contentID%3Dacs.jmedchem.0c00929%26title%3DDiscovery%2Bof%2BNovel%2BPDE%25CE%25B4%2BDegraders%2Bfor%2Bthe%2BTreatment%2Bof%2BKRAS%2BMutant%2BColorectal%2BCancer%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7905%26publicationDate%3DJuly%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00929"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2882</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">6</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00929" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Novel PDEδ Degraders for the Treatment of KRAS Mutant Colorectal Cancer&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Junfei&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Xu&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Guoqiang&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Chunquan&quot;,&quot;last_name&quot;:&quot;Sheng&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;7892-7905&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00929&quot;},&quot;abstract&quot;:&quot;KRAS-PDEδ protein–protein interaction represents an appealing target for cancer therapy. However, fast release of high-affinity inhibitors from PDEδ hampered drug binding affinity and antiproliferative activity. To overcome the limitations, the first proteolysis-targeting chimeric (PROTAC) small molecules targeting PDEδ were designed. By employment of PDEδ inhibitor deltazinone (2) and cereblon ligand pomalidomide (6), a series of potent PROTAC PDEδ degraders were obtained. The most promising compound 17f efficiently induced PDEδ degradation and demonstrated significantly improved antiproliferative potency in KRAS mutant SW480 cells. Compound 17f also achieved significant tumor growth inhibition in the SW480 colorectal cancer xenograft model. This proof-of-concept study provided a new strategy to validate the druggability of KRAS-PDEδ interaction and offered an effective lead compound for the treatment of KRAS mutant cancer.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00929&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00929" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00929&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00929" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00929&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00929" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00929&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00929&amp;href=/doi/10.1021/acs.jmedchem.0c00929" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00929" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00929" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (12 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00929%26sid%3Dliteratum%253Aachs%26pmid%3D32603594%26genre%3Darticle%26aulast%3DCheng%26date%3D2020%26atitle%3DDiscovery%2Bof%2BNovel%2BPDE%25CE%25B4%2BDegraders%2Bfor%2Bthe%2BTreatment%2Bof%2BKRAS%2BMutant%2BColorectal%2BCancer%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D14%26spage%3D7892%26epage%3D7905%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292037" title="Carbene compounds">Carbene compounds</a>,</li><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/jmcmar.2020.63.issue-14/20200723/jmcmar.2020.63.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">KRAS-PDEδ protein–protein interaction represents an appealing target for cancer therapy. However, fast release of high-affinity inhibitors from PDEδ hampered drug binding affinity and antiproliferative activity. To overcome the limitations, the first proteolysis-targeting chimeric (PROTAC) small molecules targeting PDEδ were designed. By employment of PDEδ inhibitor deltazinone (<b>2</b>) and cereblon ligand pomalidomide (<b>6</b>), a series of potent PROTAC PDEδ degraders were obtained. The most promising compound <b>17f</b> efficiently induced PDEδ degradation and demonstrated significantly improved antiproliferative potency in KRAS mutant SW480 cells. Compound <b>17f</b> also achieved significant tumor growth inhibition in the SW480 colorectal cancer xenograft model. This proof-of-concept study provided a new strategy to validate the druggability of KRAS-PDEδ interaction and offered an effective lead compound for the treatment of KRAS mutant cancer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41381" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41381" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">RAS proteins regulate many cellular processes including cell proliferation and differentiation and play a causal role in human cancers.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Mutational activation of RAS is a common feature accounting for about 30% of human cancers.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> KRAS, the most frequently mutated isoform of RAS, is commonly found in approximately 30% of lung cancer, 45% of colorectal, and 90% of pancreatic ductal carcinomas.<a onclick="showRef(event, 'ref2 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref4 ref5">(2,4,5)</a> Therefore, KRAS is well recognized as an appealing target for cancer therapy.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> However, direct targeting KRAS by small molecules remains to be a highly challenging task without drugs approved in the clinics.<a onclick="showRef(event, 'ref6 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref8 ref9">(6,8,9)</a></div><div class="NLM_p">The function of KRAS signaling critically depends on its proper localization at the plasma membrane (PM).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> To get to the PM, KRAS binds to a shuttling factor called PDEδ through its farnesylated hypervariable region.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> By this process, PDEδ prevents KRAS from binding to endomembranes and thereby facilitates its diffusion throughout the cell.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Subsequently, KRAS is released from PDEδ by release factor Arl2 and shuttled to PM via vesicular transport.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> On the basis of the crucial role of PDEδ, Waldmann’s group reported a series of small molecule KRAS-PDEδ inhibitors binding to the prenyl binding pocket of PDEδ with low nanomolar affinity, such as deltarasin (<b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), deltazinone (<b>2</b>), and deltasonamide (<b>3</b>).<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> However, despite potent binding in vitro (<i>K</i><sub>D</sub> < 10 nM), compounds <b>1</b>–<b>3</b> only showed moderate antitumor activity at micromolar concentrations.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> The discrepancy between molecular and cellular potency was also observed for KRAS-PDEδ inhibitors <b>4</b> and <b>5</b> discovered by our group.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> The main reason was that Arl2 induced fast release of high-affinity inhibitors from PDEδ and finally reduced their antitumor efficacy (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> As a result, most PDEδ inhibitors failed to exert in vivo antitumor activity. To overcome these inherent limitations, it is urgent to develop new strategies to target PDEδ more effectively.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of representative KRAS-PDEδ inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Limitations of PDEδ inhibitors and PROTAC strategy for efficient PDEδ targeting: (A) release of high affinity inhibitors from PDEδ by Arl2; (B) illustration of the catalytic cycle of the ubiquitination and degradation of PDEδ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Proteolysis-targeting chimera (PROTAC) represents an emerging therapeutic strategy to eliminate disease-causing proteins.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> Generally, PROTAC molecules are bifunctional compounds with one end binding to the protein of interest (POI) and the other end targeting to an E3 ligase, which were connected by a chemical linker (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Upon ternary complex formation, the chimeric molecules hijack E3 ligase to induce POI polyubiquitination, leading to subsequent degradation by the proteasome.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> This strategy has been successfully applied to a number of well-validated drug targets, such as bromodomain and extra-terminal (BET), androgen receptor (AR), estrogen receptor (ER), Bruton’s tyrosine kinase (BTK), and focal adhesion kinase (Fak).<a onclick="showRef(event, 'ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25 ref26">(22−26)</a> However, translating cellular degradative effects into in vivo potency and ultimately clinical efficacy is a highly challenging task.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> There are still very few examples of heterobifunctional degraders with favorable physicochemical/pharmacokinetic properties and potent in vivo activity.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Up to now, only two PROTAC molecules, namely, ARV-110 and ARV-471 developed by Arvinas, have been progressed into clinical trials.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> Moreover, despite numerous PROTAC cases dealing with traditional drug targets,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> the investigations of degraders against proteins that are considered to be undruggable by small molecule inhibitors are still rare.<a onclick="showRef(event, 'ref28 ref32'); return false;" href="javascript:void(0);" class="ref ref28 ref32">(28,32)</a></div><div class="NLM_p last">To address the limitations of conventional KRAS-PDEδ inhibitors, we envisioned that PDEδ degraders might achieve robust and durable target suppression and exert better antitumor potency. Herein, we designed the first generation of PDEδ degraders by connecting PDEδ inhibitor <b>2</b> and cereblon (CRBN) ligand pomalidomide (<b>6</b>). Among them, PDEδ degrader <b>17f</b> effectively induced PDEδ degradation and demonstrated significantly improved antiproliferative potency against KRAS mutant colorectal cancer cells both in vitro and in vivo. Thus, PROTAC <b>17f</b> provided a new chemical tool or lead compound for navigating the druggablility of KRAS-PDEδ interaction.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58164" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58164" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Molecular Design</h3><div class="NLM_p">PDEδ-degrading PROTACs were designed based on PDEδ inhibitor <b>2</b>, which was proven to be highly selective without unspecific binding to other proteins.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Molecular modeling revealed that compound <b>2</b> adopted a conformation fitting well with the prenyl binding pocket of PDEδ and formed key hydrogen bonds with Arg61, Gln78, and Tyr149 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). The homobenzyl group was exposed to the solvent region, providing a feasible site for E3 ligase ligand tethering. Meanwhile, CRBN ligand pomalidomide (<b>6</b>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) was chosen because of its reasonably drug-like physiochemical properties and broad utility in PROTAC design.<a onclick="showRef(event, 'ref19 ref20 ref33'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref33">(19,20,33)</a> To facilitate the chemical synthesis, the homobenzyl moiety in compound <b>2</b> was removed and the remaining amide group was used as the tethering site for PROTAC design (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). This structural adjustment facilitated linker incorporation via amide bond formation without disrupting the critical hydrogen bonds. As a result, a series of PROTACs with different lengths of alkyl or ethylene glycol (PEG) linkers were designed to give compounds <b>17a</b>–<b>h</b>. In addition, compound <b>17i</b> was synthesized as a negative control by methylation of the amino group of the glutarimide in <b>6</b>, with the purpose of blocking the binding to CRBN completely.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Finally, in order to evaluate other attachment points for linker conjugation, an alternative tethering position in phenyl ring of the CRBN ligand <b>6</b> was explored (compound <b>17j</b>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of PDEδ PROTACs: (A) predicted binding model of compound <b>2</b> with PDEδ (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E80">5E80</a><a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a>). (B) Structure of CRBN ligand pomalidomide. (C) Design strategy of PDEδ degraders.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">The synthetic routes of the target compounds were outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Starting from commercially available 4-methylaniline (<b>7</b>), the key intermediate <b>12</b> was synthesized via five steps according to the reported protocols.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Then, condensation of 4-fluoroisobenzofuran-1,3-dione (<b>13</b>) with 2,6-dioxopiperidine-3-ammonium chloride (<b>14</b>) afforded compound <b>15</b>, which was reacted with various aliphatic amines to form intermediates <b>16a</b>–<b>h</b>. After removal of the Boc-protecting group under acidic conditions, compounds <b>16a</b>–<b>h</b> were coupled with <b>12</b> to yield compounds <b>17a</b>–<b>h</b>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Target Compounds <b>17a</b>–<b>h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaNO<sub>2</sub>, 6 M HCl, 0 °C; (b) ethyl 2-chloro-3-oxobutanoate, NaOAc, EtOH/H<sub>2</sub>O = 7:1, 0 °C, 4 h, 70%; (c) pentane-2,4-dione, NaOEt, EtOH, rt, 16 h, 93%; (d) hydrazine hydrate, EtOH, 110 °C, 6 h, 78%; (e) methyl 4-bromobutanoate, NaH, DMF, 0 °C, 4 h, 80%; (f) LiOH, THF/MeOH/H<sub>2</sub>O = 3:2:1, 1 h, 81%; (g) NaOAc, AcOH, reflux, 12 h, 90%; (h) DIPEA, DMF, 90 °C, 12 h, 40–50%; (i) DCM, TFA, rt, 0.5 h; (j) <b>12</b>, HATU, DIPEA, DMF, rt, 2 h, 40–50%.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>17i</b>–<b>j</b> was depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The key intermediate <b>18</b> was obtained by methylation from compound <b>15</b>, followed by removal of Boc-protecting group under acidic conditions. Compound <b>17i</b> was synthesized by amide condensation reaction with intermediate <b>12</b>. Condensation of 5-fluoroisobenzofuran-1,3-dione (<b>19</b>) with <b>14</b> gave compound <b>20</b>, which was reacted with 1-Boc-1,7-diaminoheptane to form intermediate <b>16i</b>. After removal of the Boc-protecting group under acidic conditions, compound <b>16i</b> was coupled with <b>12</b> to yield compound <b>17j</b>.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route of Target Compounds <b>17i</b>–<b>j</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, CH<sub>3</sub>I, DMF, 0 °C, 4 h, 80%; (b) DIPEA, DMF, 90 °C, 12 h, 40–50%; (c) DCM, TFA, rt, 0.5 h; (d) NaOAc, AcOH, reflux, 12 h, 90%; (e) compound <b>12</b>, HATU, DIPEA, DMF, rt, 12 h, 40–50%.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Biological Evaluations and Structure–Activity Relationships</h3><div class="NLM_p">Initially, the binding affinity of all the designed PDEδ degraders was determined by fluorescence polarization assay described previously.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, all the compounds showed nanomolar affinity with <i>K</i><sub>D</sub> values ranging from 9 to 216 nM. The alkyl derivatives were more potent than the PEG analogues. The linker length dependence played an important role in PDEδ binding, and the binding affinity was significantly increased with the extension of carbon chain. In particular, compound <b>17f</b> with linker length of eight carbon atoms showed the best potency with PDEδ (<i>K</i><sub>D</sub> = 9.0 nM), which was comparable to positive control <b>2</b> (<i>K</i><sub>D</sub> = 8.0 nM).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. PDEδ Degraders Derived from Pomalidomide with Various Linkers</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0010.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">linker</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>D</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SW480 DC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">SW480 IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">HCT116 IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17a</b></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>3</sub></td><td class="colsep0 rowsep0" align="left">216 ± 9.6</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">81 ± 8.2</td><td class="colsep0 rowsep0" align="left">68 ± 8.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17b</b></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>4</sub></td><td class="colsep0 rowsep0" align="left">30 ± 3.8</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">76 ± 3.9</td><td class="colsep0 rowsep0" align="left">51 ± 2.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17c</b></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>5</sub></td><td class="colsep0 rowsep0" align="left">29 ± 4.6</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">17 ± 1.8</td><td class="colsep0 rowsep0" align="left">32 ± 4.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17d</b></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>6</sub></td><td class="colsep0 rowsep0" align="left">17 ± 2.1</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">15 ± 3.4</td><td class="colsep0 rowsep0" align="left">19 ± 1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17e</b></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>7</sub></td><td class="colsep0 rowsep0" align="left">14 ± 3.4</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">12 ± 1.4</td><td class="colsep0 rowsep0" align="left">20 ± 4.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17f</b></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>8</sub></td><td class="colsep0 rowsep0" align="left">9.0 ± 1.3</td><td class="colsep0 rowsep0" align="left">3.6</td><td class="colsep0 rowsep0" align="left">8.0 ± 2.7</td><td class="colsep0 rowsep0" align="left">8.0 ± 3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17g</b></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>(CH<sub>2</sub>OCH<sub>2</sub>)<sub>2</sub>CH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">160 ± 8.8</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">58 ± 9.8</td><td class="colsep0 rowsep0" align="left">25 ± 1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17h</b></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>2</sub>(CH<sub>2</sub>OCH<sub>2</sub>)<sub>3</sub>(CH<sub>2</sub>)<sub>2</sub></td><td class="colsep0 rowsep0" align="left">70 ± 6.4</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">26 ± 3.9</td><td class="colsep0 rowsep0" align="left">23 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">8.0 ± 2.1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">56 ± 4.3</td><td class="colsep0 rowsep0" align="left">74 ± 5.4</td></tr></tbody></table></div></div><div class="NLM_p">Then, PDEδ degradation and cell growth inhibitory effect were tested in the cell culture. KRAS mutation occurs in about 45% colorectal cancers, which is often associated with poor prognosis and emergence of drug resistance.<a onclick="showRef(event, 'ref2 ref7 ref34'); return false;" href="javascript:void(0);" class="ref ref2 ref7 ref34">(2,7,34)</a> To assess the antitumor potency of PDEδ degraders, KRAS mutant colorectal cancer SW480 and HCT116 cells were selected for the assay. The SW480 cell line is homozygote for oncogenic KRAS (G12V). HCT116 cells have one mutant (G13D) and one wild type KRAS allele.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> The degradative potency for PDEδ was evaluated in SW480 cells according to Western blotting analysis. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, most PDEδ PROTACs could efficiently induce PDEδ degradation at the concentration of 10 μM. The degradation potency was increased with the extension of linker length. Compound <b>17f</b> with a diaminoheptane linker exerted the most potent degradation efficiency with the DC<sub>50</sub> value of 3.6 μM in SW480 cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). As assessed by the Caco2 assay, compound <b>17f</b> showed low cell permeability and significant efflux (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_001.pdf" class="ext-link">Table S1, Supporting Information</a>), which might be the reason for its moderate activity in SW480 cells. Interestingly, the degradation potency correlated well with the binding affinity and antitumor activity.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Screen and characterization of potent PDEδ degraders. (A) PDEδ levels in SW480 cells upon 24 h drug treatment at 10 μM. (B, D) PDEδ levels in SW480 cells treated with <b>2</b> or <b>17f</b> at various concentrations for 24 h. (C) PDEδ levels in SW480 cells treated with <b>17f</b> at 10 μM for indicated time points. (E) SW480 cells were pretreated with the negative control <b>17i</b> or CRBN ligand <b>6</b> or PDEδ inhibitor <b>2</b> or proteasome inhibitor MG-132 followed by treatment with DMSO or PDEδ degrader <b>17f</b>. (F) Cell growth inhibition activity of the PDEδ inhibitor <b>2</b>, negative control <b>17i</b>, and PDEδ degrader <b>17f</b> in the absence or presence of <b>6</b> in SW480 cells. (G) IKZF1/3 levels in SW480 cells treated with <b>6</b> or <b>17f</b> at various concentrations for 24 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As illustrated in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, all the compounds possessed moderate to good antitumor activity against SW480 and HCT116 cells. Similar to the degradation potency, the antitumor activity was also increased with the extension of linker length. Among them, the most potent PDEδ degrader <b>17f</b> also showed the best antitumor activity in suppression of cell proliferation in SW480 (IC<sub>50</sub> = 8 μM) and HCT116 cells (IC<sub>50</sub> = 8 μM). As compared with PDEδ inhibitor <b>2</b>, PDEδ degrader <b>17f</b> showed significantly better antitumor activity, indicating the distinct advantages of PDEδ degradation over inhibition. Furthermore, other possible tethering position in the phenyl ring of pomalidomide was investigated. Compound <b>17j</b> demonstrated comparable biological activity to compound <b>17f</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), suggesting that both tethering positions in pomalidomide were favorable for the design of PDEδ degraders. PDEδ degradation and antitumor activity were also evaluated in RAS-dependent (MiaPaca-2) and independent (Panc-1) pancreatic cancer cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_001.pdf" class="ext-link">Figure S1, Supporting Information</a>). Compound <b>17f</b> could efficiently induce PDEδ degradation at 10 μM and demonstrated moderate antitumor activity against RAS-dependent MiaPaca-2 cells. For RAS-independent Panc-1 cells, compound <b>17f</b> was less potent in PDEδ degradation.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. PDEδ Degraders Linked from Different Positions of Pomalidomide</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0011.gif" alt="" id="fx2fx3" /></img><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Mechanism and Validation of PDEδ Degrader <b>17f</b></h3><div class="NLM_p last">A time-course study was performed to assess the kinetics of PDEδ degradation induced by compound <b>17f</b> in SW480 cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C, compound <b>17f</b> reduced PDEδ protein level in a time-dependent manner. PDEδ degradation was observed at time points as early as 1 h after treatment with compound <b>17f</b>. Meanwhile, compound <b>2</b> alone could not elicit the degradation of PDEδ even at 30 μM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). To further explore the mechanism of action of PDEδ degradation induced by compound <b>17f</b>, compound <b>17i</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) by methylation of pomalidomide part was synthesized as a negative control. Although compound <b>17i</b> demonstrated potent binding affinity to PDEδ (<i>K</i><sub>D</sub> = 30 nM), it failed to induce PDEδ degradation in SW480 cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E). Compound <b>17h</b>, behaved as a PDEδ inhibitor, was more than 8 times less potent (IC<sub>50</sub> = 70 μM) than <b>17f</b> in cell growth inhibition (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>F), further confirming that PDEδ degradation was a better strategy than PDEδ inhibition in exerting antitumor activity. Similarly, CRBN ligand <b>6</b> alone could not induce PDEδ degradation. In contrast, the addition of PDEδ inhibitor <b>2</b> or CRBN ligand <b>6</b> efficiently suppressed the PDEδ degradation mediated by compound <b>17f</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E) in SW480 cells. Moreover, compound <b>6</b> reduced the antitumor activity of <b>17f</b> against SW480 cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>F) with the IC<sub>50</sub> value increased from 8 μM to 71 μM. In addition, pretreatment with proteasome inhibitor MG-132 could block the PDEδ degradation induced by compound <b>17f</b> in SW480 cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E). IKZF1/3 degradation is a typical off target effect of CRBN PROTACs. Therefore, we further evaluated the selectivity of compound <b>17f</b> by monitoring degradation of IKZF1 and IKZF3. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>G, compound <b>17f</b> had a limited effect on IKFZ degradation, indicating little or no off-target IKZF1 and IKZF3 activity. All these observations confirmed the fact that compound <b>17f</b> bound to PDEδ and CRBN simultaneously, hijacked CRBN, and subsequently induced PDEδ degradation by the ubiquitin–proteasome system.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Compound <b>17f</b> Inhibits RAS Signaling and Induces the Apoptosis of KRAS Mutant SW480 Cells</h3><div class="NLM_p">Given the enhanced antitumor potency induced by PDEδ degradation, the best PDEδ degrader <b>17f</b> was selected to assess the effect on RAS signaling downstream. RAS proteins regulate MAPK and mTOR pathways associated with cell growth, proliferation, and differentiation.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> After 5 min stimulation with EGF, SW480 cells were treated with compound <b>17f</b> for 12 h. Compound <b>2</b> was used as the control. The phosphorylation of extracellular signal-regulated kinase (Erk) and Akt was determined by Western blotting assay. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, compound <b>17f</b> effectively reduced Akt and Erk phosphorylation at 5 μM in SW480 cells and decreased p-Akt and p-Erk levels in a dose-dependent manner. In contrast, compound <b>2</b> was still ineffective in reducing Akt and Erk phosphorylation at 20 μM. The results confirmed that PDEδ degradation induced by <b>17f</b> significantly downregulated the levels of both p-Akt and p-Erk. Furthermore, we also investigated the ability of compound <b>17f</b> in inducing apoptosis in SW480 cells. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, PDEδ degrader <b>17f</b> caused significant induction of apoptosis in SW480 cells at a concentration of 20 μM. The percentage of apoptotic cells was 38.31%, which was much higher than that of PDEδ inhibitor <b>2</b> (11.1%). Moreover, the apoptotic rate reached 65.53% when SW480 cells were treated with <b>17f</b> at a concentration of 40 μM. Finally, immunofluorescence staining (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B) showed that compound <b>17f</b> caused significant PDEδ degradation and induced a distribution of endogenous RAS to endomembranes. These results validated the significant advantages of PDEδ degrader at the cellular level.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>17f</b> inhibits RAS signaling. Shown are hosphorylation levels of Erk and Akt in SW480 cells stimulated with EGF (125 ng/mL, 5 min). From top to bottom: phosphorylated Akt on S473 (p-Akt), total level of Akt1 (t-Akt1), phosphorylated Erk on Thr202 and Tyr204 (p-Erk), total level of Erk (t-Erk), and loading control (GAPDH). ***<i>P</i> value of <0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>17f</b> inhibits RAS signaling and initiates the apoptosis of KRAS mutant SW480 cells: (A) flow cytometry analysis of the apoptosis induction by <b>17f</b> in SW480 cells; (B) immunostaining of SW480 with anti-PDEδ (green) and antipan-RAS (red) after treatment with compound <b>17f</b> at 10 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> In Vivo Antitumor Efficacy and Preliminary PK Study of Compound <b>17f</b></h3><div class="NLM_p">To access the therapeutic potential of PDEδ degraders, compound <b>17f</b> was selected to investigate in vivo antitumor potency in SW480 xenograft nude mice models. The SW480 colorectal cancer xenograft model was prepared using the reported procedure.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> After the tumor volume reached 100 mm<sup>3</sup> in each group, compound <b>17f</b> and positive control were administered intraperitoneally (ip) once a day for 12 consecutive days. As depicted in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A, PDEδ degrader <b>17f</b> (50 mg/kg) achieved significant tumor growth inhibition with tumor growth inhibition (TGI) values of 43%. In contrast, ip dosing with PDEδ inhibitor <b>2</b> (50 mg/kg) failed to exert significant tumor growth inhibition (TGI = 24%) as compared with the vehicle group. Thus, degrader <b>17f</b> demonstrated significantly better antineoplastic effects in vivo than that of inhibitor <b>2</b>. Moreover, no significant body weight loss and no obvious adverse effects were observed during the study (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B), suggesting the low toxicity of compound <b>17f</b>. In order to further clarify the mechanism of <b>17f</b> in vivo, immunofluorescence assays were performed to detect the expression of PDEδ, RAS, and p-Akt in tumor tissues. Consistently, treatment with compound <b>17f</b> led to significant decrease in PDEδ and p-Akt proteins in vivo (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Notably, compound <b>2</b> showed no effect on PDEδ protein levels and slightly decreased p-Akt protein levels. The in vivo results highlighted the distinct advantages of PDEδ degradation over PDEδ inhibition and provided an efficient strategy for the treatment of KRAS mutant cancer.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. In vivo efficacy of compound <b>17f</b>. (A) Growth curve of implanted SW480 xenograft in nude mice. Data are expressed as the mean ± standard deviation: not significant (n.s.), <i>P</i> value of >0.05; *<i>P</i> value of <0.05. (B) Body weight loss in SW480 xenograft experiments. (C) Immunofluorescence analysis of PDEδ, RAS, and p-Akt protein expression in transplanted nude mice. (D) PK parameters of compound <b>17f</b> in rats. Data are expressed as the mean ± standard deviation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Furthermore, pharmacokinetic (PK) studies of compound <b>17f</b> were evaluated in Sprague-Dawley (SD) rats. After ip administration dosing at 50 mg/kg, the concentrations of compound <b>17f</b> in plasma were analyzed. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C, the half-life of <b>17f</b> was approximately 5.1 h and the peak concentration <i>C</i><sub>max</sub> was 564 ng/mL. Despite its relatively large size (MW = 723), compound <b>17f</b> could be effectively absorbed and achieved a sufficient plasma exposure in rats, with the area under the curve (AUC) value of 4710 h·ng/mL.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47372" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47372" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, the first-in-class PDEδ degraders were identified by the PROTAC approach derived from conventional PDEδ inhibitors. The most promising compound <b>17f</b> simultaneously bound to both PDEδ and CRBN and efficiently induced PDEδ degradation. It significantly induced apoptosis, downregulated EGF-induced Erk and Akt phosphorylation in KRAS mutant SW480 cells. As compared to PDEδ inhibitor <b>2</b>, degrader <b>17f</b> showed improved in vitro and in vivo antitumor activity. More importantly, degrader <b>17f</b> achieved significant tumor growth inhibition in SW480 colorectal cancer xenograft model, which provided important evidence to validate the druggability of KRAS-PDEδ interaction. Taken together, this proof-of-concept study highlighted the advantages of PDEδ degradation in target validation and novel antitumor drug discovery. Further structural optimization studies to improve the degradation potency and antitumor efficacy of lead compound <b>17f</b> are in progress.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71380" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71380" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry. General Methods</h3><div class="NLM_p last">All starting materials were commercially available and analytical pure. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded on Bruker AVANCE II 300 or AVANCE II 600 spectrometer (Bruker Company, Germany), using TMS as an internal standard and DMSO-<i>d</i><sub>6</sub> as solvents. Chemical shifts (δ values) and coupling constants (<i>J</i> values) are given in ppm and Hz, respectively. The mass spectra were recorded on an Esquire 3000 LC–MS mass spectrometer. Silica gel thin-layer chromatography was performed on precoated plates GF-254 (Qingdao Haiyang Chemical, China). Silica gel column chromatography was performed with silica gel 60G (Qingdao Haiyang Chemical, China). Purity of the compound was analyzed by HPLC (Agilent Eclipse Plus C<sub>18</sub> 5 μm, 4.6 mm × 250 mm), and final compounds exhibited a purity greater than 95%.</div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Methyl 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)butanoate (<b>11</b>)</h3><div class="NLM_p last">Compound <b>9</b> (3.5 g, 12 mmol) was dissolved in 20 mL of ethanol and placed in a microwave tube with hydrazine hydrate (1.8 g, 36 mmol). After 6 h at 110 °C, a large number of yellow solids were precipitated, filtered, and washed to give intermediate <b>10</b> (2.4 g, 78% yield) which was synthesized according to literature methods.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Intermediate <b>10</b> (0.5 g, 1.9 mmol) was dissolved in DMF (3 mL) with NaH (0.12 g, 2.85 mmol) at 0 °C and stirred for 0.5 h. Then, the methyl 3-bromopropanoate (0.66 g, 3.8 mmol) was added slowly, and the mixture was stirred for 6 h at room temperature. Then the mixture was poured into brine (50 mL). The white precipitate was then filtered and dried to give the crude product. The crude product was purified by column chromatography (DCM/MeOH = 100:1) to afford pure product <b>11</b> (0.22 g, 80% yield) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 600 MHz) δ: 7.49 (d, <i>J</i> = 8.49 Hz, 2H), 7.43 (d, <i>J</i> = 8.13 Hz, 2H), 4.08 (t, <i>J</i> = 6.72 Hz, 2H), 3.57 (s, 3H), 2.60 (s, 3H), 2.52 (s, 3H), 2.43 (s, 3H), 2.35 (t, <i>J</i> = 7.07 Hz, 2H), 1.98–1.93 (m, 2H).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)butanoic Acid (<b>12</b>)</h3><div class="NLM_p last">To a solution of intermediate <b>11</b> (0.22 g, 0.65 mmol) in THF–MeOH–H<sub>2</sub>O solvent system (18 mL, THF/MeOH/H<sub>2</sub>O = 3:2:1) was slowly added aqueous LiOH (1 M, 55 mg, 1.3 mmol). The mixture was stirred for 1 h at room temperature. Then, the solvent was removed under the reduced pressure and the pH of the solution was adjusted to 2 with 2 M HCl. The white precipitate was then filtered and dried to give intermediate <b>12</b> (0.17 g, 80% yield) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 600 MHz) δ: 12.06 (s, 1H), 7.49 (d, <i>J</i> = 8.38 Hz, 2H,), 7.43 (d, <i>J</i> = 8.38 Hz, 2H,), 4.07 (t, <i>J</i> = 6.82 Hz, 2H,), 2.60 (s, 3H), 2.52 (s, 3H), 2.43 (s, 3H), 2.26 (t, <i>J</i> = 7.41 Hz, 2H), 1.90–1.96 (m, 2H).</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 2-(2,6-Dioxopiperidin-3-yl)-4-fluoroisoindoline-1,3-dione (<b>15</b>)</h3><div class="NLM_p last">A solution of 4-fluoroisobenzofuran-1,3-dione (0.5 g, 3 mmol), 2,6-dioxopiperidine-3-ammonium chloride (0.5 g, 3 mmol), and NaOAc (0.3 g, 3.6 mmol) in AcOH (50 mL) was reflux for 12 h. The reaction mixture was poured into water (300 mL) and extracted with EtOAc (3 × 150 mL). The organic phases were combined, dried with anhydrous sodium sulfate, and concentrated under the reduced pressure. The crude product was purified by column chromatography (DCM/MeOH = 100:1) to give intermediate <b>15</b> as a white solid (0.74 g, 90% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.15 (s, 1H), 7.93–7.97 (m, 1H), 7.79 (d, <i>J</i> = 7.31 Hz, 1H), 7.73 (t, <i>J</i> = 8.86 Hz, 1H), 5.16 (dd, <i>J</i> = 13.16, 5.35 Hz, 1H), 2.93–2.86 (m, 1H), 2.63–2.58 (m, 1H), 2.56–2.47 (m, 1H), 2.09–2.04 (m, 1H). Compound <b>20</b> was synthesized according to the similar procedure described for <b>15</b>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>tert</i>-Butyl (3-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)propyl)carbamate (<b>16a</b>)</h3><div class="NLM_p last">A solution of intermediate <b>15</b> (0.2 g, 0.7 mmol) and <i>tert</i>-butyl (3-aminopropyl)carbamate (0.14 g, 0.8 mmol) in DMF (2 mL) was treated with DIPEA (0.18 g, 1.4 mmol) under 90 °C for 12 h. The reaction mixture was poured into water (20 mL) and extracted with EtOAc (3 × 15 mL). The combined organic phases were dried and concentrated under the reduced pressure. The crude product was purified by column chromatography (DCM/MeOH = 100:1) to give intermediate <b>16a</b> as a yellow solid (0.12 g, 40% yield). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.10 (s, 1H), 7.61–7.54 (m, 1H), 7.08 (d, <i>J</i> = 8.74 Hz, 1H), 7.02 (d, <i>J</i> = 7.12 Hz, 1H), 6.93 (t, <i>J</i> = 5.18 Hz, 1H), 6.67 (t, <i>J</i> = 5.18 Hz, 1H), 5.05 (dd, <i>J</i> = 12.62, 5.18 Hz, 1H), 3.36–3.26 (m, 2H), 3.05–2.95 (m, 2H), 2.91–2.81 (m, 1H), 2.65–2.58 (m, 1H), 2.57–2.52 (m, 1H), 2.08–2.00 (m, 1H), 1.73–1.59 (m, 2H), 1.38 (s, 9H). Compounds <b>16b</b>–<b>i</b> were synthesized according to a similar procedure described for <b>16a</b>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> <i>tert</i>-Butyl (4-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)butyl)carbamate (<b>16b</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.08 (s, 1H), 7.59–7.56 (m, 1H), 7.10 (d, <i>J</i> = 8.94 Hz, 1H), 7.02 (d, <i>J</i> = 7.45 Hz, 1H), 6.83 (t, <i>J</i> = 5.96 Hz, 1H), 6.55 (t, <i>J</i> = 5.47 Hz, 1H), 5.05 (dd, <i>J</i> = 12.92, 4.97 Hz, 1H), 3.32–3.27 (m, 2H), 2.97–2.92 (m, 2H), 2.90–2.87 (m, 1H), 2.62–2.56 (m, 1H), 2.55–2.51 (m, 1H), 2.05–2.00 (m, 1H), 1.59–1.51 (m, 2H), 1.48–1.41 (m, 2H), 1.37 (s, 9H).</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> <i>tert</i>-Butyl (5-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)pentyl)carbamate (<b>16c</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.09 (s, 1H), 7.60–7.56 (m, 1H), 7.10 (d, <i>J</i> = 8.26 Hz, 1H), 7.02 (d, <i>J</i> = 8.26 Hz, 1H), 6.78 (t, <i>J</i> = 5.08 Hz, 1H), 6.53 (t, <i>J</i> = 5.72 Hz, 1H), 5.05 (dd, <i>J</i> = 12.54, 5.53 Hz, 1H), 3.30–3.25 (m, 2H), 2.93–2.88 (m, 2H), 2.88–2.84 (m, 1H), 2.62–2.56 (m, 1H), 2.56–2.51 (m, 1H), 2.05–2.00 (m, 1H), 1.60–1.53 (m, 2H), 1.44–1.38 (m, 2H), 1.36 (s, 9H), 1.33–1.29 (m,2H).</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>tert</i>-Butyl (6-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)hexyl)carbamate (<b>16d</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.04 (s, 1H), 7.61–7.57 (m, 1H), 7.10 (d, <i>J</i> = 8.52 Hz, 1H), 7.03 (d, <i>J</i> = 7.01 Hz, 1H), 6.78 (t, <i>J</i> = 5.45 Hz, 1H), 6.54 (t, <i>J</i> = 5.96 Hz, 1H), 5.06 (dd, <i>J</i> = 13.12, 5.62 Hz, 1H), 3.32–3.27 (m, 2H), 2.92–2.89 (m, 2H), 2.88–2.85 (m, 1H), 2.63–2.57 (m, 1H), 2.57–2.52 (m, 1H), 2.06–2.00 (m, 1H), 1.60–1.54 (m, 2H), 1.37 (s, 9H), 1.37–1.35 (m, 2H), 1.35–1.32 (m, 2H), 1.31–1.27 (m, 2H).</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> <i>tert</i>-Butyl (8-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)octyl)carbamate (<b>16e</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.07 (s, 1H), 7.58–7.55 (m, 1H), 7.08 (d, <i>J</i> = 8.51 Hz, 1H), 7.00 (d, <i>J</i> = 7.25 Hz, 1H), 6.73 (t, <i>J</i> = 5.36 Hz, 1H), 6.51 (t, <i>J</i> = 5.68 Hz, 1H), 5.03 (dd, <i>J</i> = 13.24, 5.36 Hz, 1H), 3.30–3.25 (m, 2H), 2.87–2.83 (m, 2H), 2.61–2.58 (m, 1H), 2.57–2.55 (m, 1H), 2.55–2.51 (m, 1H), 2.04–1.99 (m, 1H), 1.58–1.52 (m, 2H), 1.35 (s, 9H), 1.33–1.21 (m, 10H).</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>tert</i>-Butyl (2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethoxy)ethyl)carbamate (<b>16f</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.10 (s, 1H), 7.58 (t, <i>J</i> = 8.29 Hz, 1H), 7.15 (d, <i>J</i> = 8.29 Hz, 1H), 7.04 (d, <i>J</i> = 6.74 Hz, 1H), 6.75 (t, <i>J</i> = 5.18 Hz, 1H), 6.61 (t, <i>J</i> = 5.18 Hz, 1H), 5.06 (dd, <i>J</i> = 12.96, 5.18 Hz, 1H), 3.66–3.39 (m, 10H), 3.13–2.99 (m, 2H), 2.98–2.79 (m, 1H), 2.67–2.52 (m, 2H), 2.09–2.00 (m, 1H), 1.36 (s, 9H).</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>tert</i>-Butyl (3-(2-(2-(3-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)propoxy)ethoxy)ethoxy)propyl)carbamate (<b>16g</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.10 (s, 1H), 7.58 (t, <i>J</i> = 7.94 Hz, 1H), 7.10 (d, <i>J</i> = 8.58 Hz, 1H), 7.02 (d, <i>J</i> = 6.67 Hz, 1H), 6.75 (t, <i>J</i> = 5.08 Hz, 1H), 6.67 (t, <i>J</i> = 5.72 Hz, 1H), 5.05 (dd, <i>J</i> = 13.03, 5.08 Hz, 1H), 3.59–3.42 (m, 10H), 3.42–3.40 (m, 4H), 3.00–2.90 (m, 2H), 2.90–2.80 (m, 1H), 2.66–2.54 (m, 2H), 2.08–1.99 (m, 1H), 1.88–1.75 (m, 2H), 1.66–1.51 (m, 2H), 1.36 (s, 9H).</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(3-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)propyl)butanamide (<b>17a</b>)</h3><div class="NLM_p">The intermediate <b>16a</b> (0.2 g, 0.47 mmol) was dissolved in DCM (2 mL), and then TFA (1 mL) was added to the reaction. The reaction was stirred for 1 h at room temperature. Then, the reaction mixture was evaporated under reduced pressure and the crude product was dissolved in DMF (2 mL) with HATU (0.27 g, 0.7 mmol), DIPEA (0.18 g, 1.4 mmol), and compound <b>12</b> (0.24 g, 0.7 mmol). After 2 h, the reaction mixture was poured into water (20 mL) and extracted with EtOAc (3 × 15 mL). The combined organic phases were dried and concentrated under the reduced pressure. The crude product was purified by column chromatography (DCM/MeOH = 100:1) to give intermediate <b>17a</b> as a yellow solid (0.12 g, 40% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.08 (s, 1H, imino), 7.90 (t, <i>J</i> = 5.69 Hz, 1H, imino), 7.57–7.53 (m, 1H, aromatic), 7.47 (d, <i>J</i> = 8.19 Hz, 2H, aromatic), 7.42 (d, <i>J</i> = 8.19 Hz, 2H, aromatic), 7.08 (d, <i>J</i> = 8.70 Hz, 1H, aromatic), 6.99 (d, <i>J</i> = 6.91 Hz, 1H, aromatic), 6.66 (t, <i>J</i> = 6.14 Hz, 1H, imino), 5.05–5.00 (m, 1H, hypomethyl), 4.03 (t, <i>J</i> = 7.17 Hz, 2H, methylene), 3.33–3.28 (m, 2H, methylene), 3.12–3.07 (m, 2H, methylene), 2.90–2.82 (m, 1H, methylene), 2.61–2.55 ((m, 1H, methylene; s, 3H, methyl), 2.54–2.50 (m, 1H, methylene), 2.50 (s, 3H, methyl), 2.41 (s, 3H, methyl), 2.11 (t, <i>J</i> = 7.68 Hz, 2H, methylene), 2.04–1.98 (m, 1H, methylene), 1.95–1.89 (m, 2H, methylene), 1.68–1.62 (m, 2H, methylene). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 173.3 (C═O), 172.0 (C═O), 170.6 (C═O), 169.3 (C═O), 167.8 (C═O), 155.6 (C═O), 146.8 (C-aromatic), 141.6, 141.5, 139.8 (C-aromatic), 137.9, 136.7 (C-aromatic), 136.3 (C-aromatic), 132.7 (C-aromatic), 130.4 (C-aromatic), 126.2 (C-aromatic), 117.6 (C-aromatic), 110.8 (C-aromatic), 109.6 (C-aromatic), 49.02, 49.0, 36.4, 33.1, 31.4, 29.5, 29.2, 25.0, 22.6, 21.2 (C-methyl), 19.9 (C-methyl), 12.3 (C-methyl). HR-MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>34</sub>H<sub>37</sub>N<sub>8</sub>O<sub>6</sub>, 653.2831; found, 653.2832. HPLC purity 98.9%.</div><div class="NLM_p last">Compounds <b>17b</b>–<b>i</b> were synthesized according to a similar procedure described for <b>17a</b>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(4-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)butyl)butanamide (<b>17b</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.11 (s, 1H, imino), 7.85 (t, <i>J</i> = 5.53 Hz, 1H, imino), 7.57–7.53 (m, 1H, aromatic), 7.49 (d, <i>J</i> = 9.22 Hz, 2H, aromatic), 7.44 (d, <i>J</i> = 8.61 Hz, 2H, aromatic), 7.09 (d, <i>J</i> = 8.53 Hz, 1H, aromatic), 6.97 (d, <i>J</i> = 7.13 Hz, 1H, aromatic), 6.55 (t, <i>J</i> = 5.81 Hz, 1H, imino), 5.06–5.02 (m, 1H, hypomethyl), 4.02 (t, <i>J</i> = 6.87 Hz, 2H, methylene), 3.32–3.27 (m, 2H, methylene), 3.10–3.05 (m, 2H, methylene), 2.91–2.83 (m, 1H, methylene), 2.63–2.56 (m, 1H, methylene; s, 3H, methyl), 2.55–2.53 (m, 1H methylene), 2.51 (s, 3H, methyl), 2.43 (s, 3H, methyl), 2.11 (t, <i>J</i> = 7.13 Hz, 2H, methylene), 2.05–1.99 (m, 1H, methylene), 1.95–1.88 (m, 2H, methylene), 1.60–1.53 (m, 2H, methylene), 1.49–1.43 (m, 2H, methylene). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 173.3 (C═O), 171.9 (C═O), 170.6 (C═O), 169.4 (C═O), 167.8 (C═O), 155.6 (C═O), 146.9 (C-aromatic), 141.6, 141.5, 139.9 (C-aromatic), 137.9, 136.7 (C-aromatic), 136.3 (C-aromatic), 132.6 (C-aromatic), 130.4 (C-aromatic), 126.2 (C-aromatic), 117.7 (C-aromatic), 117.6 (C-aromatic), 110.8 (C-aromatic), 109.4 (C-aromatic), 49.1, 49.0, 42.0, 38.5, 33.1, 31.4, 26.9, 26.6, 25.2, 22.6, 21.2 (C-methyl), 19.8 (C-methyl), 12.3 (C-methyl). HR-MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>35</sub>H<sub>39</sub>N<sub>8</sub>O<sub>6</sub>, 667.2987; found, 667.2987. HPLC purity 95.2%.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(5-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)pentyl)butanamide (<b>17c</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.10 (s, 1H), 7.82 (t, <i>J</i> = 5.59 Hz, 1H), 7.58–7.54 (m, 1H), 7.62 (d, <i>J</i> = 8.13 Hz, 2H), 7.44 (d, <i>J</i> = 8.13 Hz, 2H), 7.09 (d, <i>J</i> = 8.70 Hz, 1H), 6.99 (d, <i>J</i> = 6.91 Hz, 1H), 6.52 (t, <i>J</i> = 6.02 Hz, 1H), 5.07–5.02 (m, 1H), 4.03 (t, <i>J</i> = 7.14 Hz, 2H), 3.32–3.25 (m, 2H), 3.07–3.01 (m, 2H), 2.91–2.84 (m, 1H), 2.63–2.57 (m, 4H), 2.56–2.52 (m, 1H), 2.52 (s, 3H), 2.43 (s, 3H), 2.10 (t, <i>J</i> = 7.58 Hz, 2H), 2.07–2.00 (m, 1H), 1.96–1.88 (m, 2H), 1.61–1.54 (m, 2H), 1.46–1.40 (m, 2H), 1.38–1.31 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 173.3, 171.9, 170.6, 169.4, 167.8, 155.6, 146.9, 141.6, 141.5, 139.8, 137.9, 136.7, 136.3, 132.6, 130.4, 126.2, 117.6, 117.6, 110.8, 109.4, 49.1, 49.0, 42.3, 38.8, 33.1, 31.4, 29.3, 28.8, 25.1, 24.2, 22.6, 21.2, 19.8, 12.3. HR-MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>36</sub>H<sub>41</sub>N<sub>8</sub>O<sub>6</sub>, 681.3144; found, 681.3148. HPLC purity 95.2%.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(6-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)hexyl)butanamide (<b>17d</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.07 (s, 1H imino), 7.75 (t, <i>J</i> = 5.56 Hz, 1H imino), 7.57–7.51 (m, 1H, aromatic), 7.46 (d, <i>J</i> = 8.42 Hz, 2H, aromatic), 7.41 (d, <i>J</i> = 8.16 Hz, 2H, aromatic), 7.05 (d, <i>J</i> = 8.50 Hz, 1H, aromatic), 6.97 (d, <i>J</i> = 6.99 Hz, 1H, aromatic), 6.50 (t, <i>J</i> = 5.81 Hz, 1H, imino), 5.06–5.00 (m, 1H, hypomethyl), 4.02 (t, <i>J</i> = 6.73 Hz, 2H, methylene), 3.28–3.23 (m, 2H, methylene), 3.03–2.98 (m, 2H, methylene), 2.91–2.82 (m, 1H, methylene), 2.62–2.55 (m, 1H, methylene; s, 3H, methyl), 2.54–2.50 (m, 1H, methylene), 2.49 (s, 3H, methyl), 2.41 (s, 3H, methyl), 2.08 (t, <i>J</i> = 7.42 Hz, 2H, methylene), 2.05–1.98 (m, 1H, methylene), 1.94–1.86 (m, 2H, methylene), 1.59–1.51 (m, 2H, methylene), 1.41–1.35 (m, 2H, methylene), 1.34–1.29 (m, 4H, methylene). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 173.3 (C═O), 171.7 (C═O), 170.5 (C═O), 169.4 (C═O), 167.8 (C═O), 155.6 (C═O), 146.9 (C-aromatic), 141.6, 141.4, 139.8 (C-aromatic), 137.8, 136.7 (C-aromatic), 136.4 (C-aromatic), 132.6 (C-aromatic), 130.4 (C-aromatic), 126.2 (C-aromatic), 117.6 (C-aromatic), 110.8 (C-aromatic), 109.5 (C-aromatic), 49.1, 49.0, 42.2, 38.8, 33.1, 31.4, 29.5, 29.1, 26.6, 26.5, 25.2, 22.6, 21.2 (C-methyl), 19.9 (C-methyl), 12.3 (C-methyl). HR-MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>37</sub>H<sub>43</sub>N<sub>8</sub>O<sub>6</sub>, 695.3300; found, 695.3304.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(7-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)heptyl)butanamide (<b>17e</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.10 (s, 1H, imino), 7.78 (t, <i>J</i> = 5.42 Hz, 1H, imino), 7.56 (dd, <i>J</i> = 8.51, 7.3 Hz, 1H, aromatic), 7.49 (d, <i>J</i> = 8.24 Hz, 2H, aromatic), 7.44 (d, <i>J</i> = 8.13 Hz, 2H, aromatic), 7.07 (d, <i>J</i> = 8.52 Hz, 1H, aromatic), 6.99 (d, <i>J</i> = 6.91 Hz, 1H, aromatic), 6.51 (t, <i>J</i> = 5.72 Hz, 1H, imino), 5.06 (dd, <i>J</i> = 12.94, 5.52 Hz, 1H, hypomethyl), 4.04 (t, <i>J</i> = 7.01 Hz, 2H, methylene), 3.30–3.25 (m, 2H, methylene), 3.05–3.00 (m, 2H, methylene), 2.93–2.85 (m, 1H, methylene), 2.63–2.59 (m, 1H, methylene), 2.59 (s, 3H, methyl), 2.56–2.51 ((m, 1H, methylene; s, 3H, methyl), 2.43 (s, 3H, methyl), 2.10 (t, <i>J</i> = 7.32 Hz, 2H, methylene), 2.06–2.00 (m, 1H, methylene), 1.96–1.89 (m, 2H, methylene), 1.61–1.53 (m, 2H, methylene), 1.42–1.36 (m, 2H, methylene), 1.35–1.26 (m, 6H, methylene). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 173.3 (C═O), 171.7 (C═O), 170.5 (C═O), 169.4 (C═O), 167.8 (C═O), 155.6 (C═O), 146.9 (C-aromatic), 141.6, 141.4, 139.8 (C-aromatic), 137.8, 136.7 (C-aromatic), 136.3 (C-aromatic), 132.6 (C-aromatic), 130.4 (C-aromatic), 126.2 (C-aromatic), 117.6 (C-aromatic), 110.8 (C-aromatic), 109.5 (C-aromatic), 49.1, 49.0, 42.3, 38.9, 33.1, 31.5, 29.5, 29.1, 28.9, 26.8, 26.7, 25.2, 22.6, 21.2 (C-methyl), 19.9 (C-methyl), 12.3 (C-methyl). HR-MS (ESI+): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>38</sub>H<sub>45</sub>N<sub>8</sub>O<sub>6</sub>, 709.3457; found, 709.3455. HPLC purity 99.4%.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(8-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)octyl)butanamide (<b>17f</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.09 (s, 1H, imino), 7.76 (t, <i>J</i> = 5.62 Hz, 1H, imino), 7.56 (dd, <i>J</i> = 8.41, 7.01 Hz, 1H, aromatic), 7.48 (d, <i>J</i> = 8.44 Hz, 2H, aromatic), 7.43 (d, <i>J</i> = 8.44 Hz, 2H, aromatic), 7.06 (d, <i>J</i> = 8.51 Hz, 1H, aromatic), 7.00 (d, <i>J</i> = 7.02 Hz, 1H, aromatic), 6.51 (t, <i>J</i> = 5.84 Hz, 1H, imino), 5.05 (dd, <i>J</i> = 12.82, 5.41 Hz, 1H, hypomethyl), 4.04 (t, <i>J</i> = 7.01 Hz, 2H, methylene), 3.29–3.24 (m, 2H, methylene), 3.04–2.98 (m, 2H, methylene), 2.92–2.85 (m, 1H, methylene), 2.62–2.57 (m, 1H, methylene; s, 3H, methyl), 2.54–2.51 (m, 1H, methylene; s, 3H, methyl), 2.42 (s, 3H, methyl), 2.10 (t, <i>J</i> = 7.32 Hz, 2H, methylene), 2.05–2.00 (m, 1H, methylene), 1.95–1.89 (m, 2H, methylene), 1.59–1.53 (m, 2H, methylene), 1.41–1.35 (m, 2H, methylene), 1.35–1.26 (m, 8H, methylene). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 173.3 (C═O), 171.7 (C═O), 170.5 (C═O), 169.4 (C═O), 167.8 (C═O), 155.6 (C═O), 146.9 (C-aromatic), 141.6, 141.4, 139.8 (C-aromatic), 137.8, 136.7 (C-aromatic), 136.3 (C-aromatic), 132.6 (C-aromatic), 130.3 (C-aromatic), 126.2 (C-aromatic), 117.6 (C-aromatic), 110.8 (C-aromatic), 109.5 (C-aromatic), 49.1, 49.0, 42.3, 38.9, 33.2, 31.5, 30.3, 29.5, 29.2, 29.1, 26.8, 26.7, 25.2, 22.6, 21.2 (C-methyl), 19.9 (C-methyl), 12.3 (C-methyl). HR-MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>39</sub>H<sub>47</sub>N<sub>8</sub>O<sub>6</sub>, 723.3613; found, 723.3616. HPLC purity 97.7%.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethoxy)ethyl)butanamide (<b>17g</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.05 (s, 1H, imino), 7.82 (t, <i>J</i> = 5.72 Hz, 1H, imino), 7.52–7.49 (m, 1H, aromatic), 7.42 (d, <i>J</i> = 8.32 Hz, 2H, aromatic), 7.37 (d, <i>J</i> = 8.19 Hz, 2H, aromatic), 7.06 (d, <i>J</i> = 8.45 Hz, 1H, aromatic), 6.96 (d, <i>J</i> = 6.89 Hz, 1H, aromatic), 6.54 (t, <i>J</i> = 5.59 Hz, 1H, imino), 5.03–4.99 (m, 1H, hypomethyl), 3.98 (t, <i>J</i> = 7.02 Hz, 2H, methylene), 3.56 (t, <i>J</i> = 5.33 Hz, 2H, methylene), 3.53–3.50 (m, 2H, methylene), 3.49–3.46 (m, 2H, methylene), 3.42–3.38 (m, 2H, methylene), 3.37–3.34 (m, 2H, methylene), 3.17–3.12 (m, 2H, methylene), 2.89–2.80 (m, 1H, methylene), 2.58–2.51 (m, 1H, methylene; s, 3H, methyl), 2.50–2.46 (m, 1H, methylene), 2.45 (s, 3H, methyl), 2.37 (s, 3H, methyl), 2.08 (t, <i>J</i> = 7.93 Hz, 2H, methylene), 2.01–1.96 (m, 1H, methylene), 1.90–1.83 (m, 2H, methylene). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 173.3 (C═O), 172.0 (C═O), 170.5 (C═O), 169.4 (C═O), 167.7 (C═O), 155.6 (C═O), 146.8 (C-aromatic), 141.6, 141.4, 139.8 (C-aromatic), 137.8, 136.6 (C-aromatic), 136.3 (C-aromatic), 132.5 (C-aromatic), 130.3 (C-aromatic), 126.1 (C-aromatic), 117.8 (C-aromatic), 117.6 (C-aromatic), 111.1 (C-aromatic), 109.7 (C-aromatic), 70.1, 70.0, 69.6, 69.3, 49.0, 42.2, 39.0, 33.0, 31.5, 25.1, 22.6, 21.2 (C-methyl), 19.8 (C-methyl), 12.3 (C-methyl). HR-MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>37</sub>H<sub>43</sub>N<sub>8</sub>O<sub>8</sub>, 727.3198; found, 727.3206.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(3-(2-(2-(3-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)propoxy)ethoxy)ethoxy)propyl)butanamide (<b>17h</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.07 (s, 1H, imino), 7.75 (t, <i>J</i> = 5.00 Hz, 1H, imino), 7.54 (t, <i>J</i> = 7.01 Hz, 1H, aromatic), 7.46 (d, <i>J</i> = 8.27 Hz, 2H, aromatic), 7.41 (d, <i>J</i> = 7.93 Hz, 2H, aromatic), 7.07 (d, <i>J</i> = 8.96 Hz, 1H, aromatic), 6.99 (d, <i>J</i> = 6.89 Hz, 1H, aromatic), 6.64 (t, <i>J</i> = 5.17 Hz, 1H, imino), 5.07–4.98 (m, 1H, hypomethyl), 4.02 (t, <i>J</i> = 6.55 Hz, 2H, methylene), 3.54–3.44 (m, 10H, methylene), 3.34–3.38 (m, 4H, methylene), 3.08–3.01 (m, 2H, methylene), 2.92–2.83 (m, 1H, methylene), 2.61–2.55 (m, 1H, methylene; s, 3H, methyl), 2.54–2.49 (m, 1H, methylene; s, 3H, methyl), 2.41 (s, 3H, methyl), 2.10–2.05 (m, 2H, methylene), 2.04–1.97 (m, 1H, methylene), 1.93–1.86 (m, 2H, methylene), 1.82–1.74 (m, 2H, methylene), 1.63–1.54 (m, 2H, methylene). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 173.3 (C═O), 171.8 (C═O), 170.5 (C═O), 169.3 (C═O), 167.8 (C═O), 155.6 (C═O), 146.9 (C-aromatic), 141.6, 141.4, 139.8 (C-aromatic), 137.8, 136.7 (C-aromatic), 136.3 (C-aromatic), 132.6 (C-aromatic), 130.3 (C-aromatic), 126.2 (C-aromatic), 117.6 (C-aromatic), 117.5 (C-aromatic), 110.8 (C-aromatic), 109.5 (C-aromatic), 70.3, 70.2, 70.2, 70.0, 68.7, 68.6, 49.04, 49.0, 40.1, 36.3, 33.1, 31.4, 29.8, 29.3, 25.1, 22.6, 21.2 (C-methyl), 19.9 (C-methyl), 12.3 (C-methyl). HR-MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>41</sub>H<sub>51</sub>N<sub>8</sub>O<sub>9</sub>, 799.3774; found, 799.3777.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 4-Fluoro-2-(1-methyl-2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>18</b>)</h3><div class="NLM_p last">A solution of intermediate <b>15</b> (0.2 g, 0.7 mmol) and NaH (0.19 g, 0.8 mmol) in DMF (2 mL) was treated with CH<sub>3</sub>I (0.11 g, 0.8 mmol) under 0 °C for 4 h. The reaction mixture was poured into water (20 mL) and extracted with EtOAc (3 × 15 mL). The combined organic phases were dried and concentrated under the reduced pressure. The crude product was purified by column chromatography (DCM/MeOH = 100:1) to give intermediate <b>18</b> as a white solid (0.16 g, 80% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 7.98<i>–</i>7.94 (m, 1H), 7.80 (d, <i>J</i> = 7.76 Hz, 1H), 7.75 (t, <i>J</i> = 9.24 Hz, 1H), 5.23 (dd, <i>J</i> = 12.94, 5.18 Hz, 1H), 3.03 (s, 3H), 2.99–2.93 (m, 1H), 2.80–2.76 (m, 1H), 2.58–2.52 (m, 1H), 2.10–2.06 (m, 1H).</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>tert</i>-Butyl (8-((2-(1-Methyl-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)octyl)carbamate (<b>16h</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 7.49–7.54 (m, 1H), 7.02 (d, <i>J</i> = 8.59 Hz, 1H), 6.95 (d, <i>J</i> = 7.00 Hz, 1H), 6.67 (t, <i>J</i> = 5.42 Hz, 1H), 6.46 (t, <i>J</i> = 5.42 Hz, 1H), 5.05 (dd, <i>J</i> = 13.08, 5.42 Hz, 1H), 3.25–3.19 (m, 2H), 2.95 (s, 3H), 2.93–2.84 (m, 1H), 2.84–2.78 (m, 2H), 2.72–2.66 (m, 1H), 2.51–2.45 (m, 1H), 2.00–1.95 (m, 1H), 1.53–1.47 (m, 2H), 1.29 (s, 9H), 1.29–1.14 (m, 10H).</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(8-((2-(1-methyl-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)octyl)butanamide (<b>17i</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 7.69 (t, <i>J</i> = 5.42, 1H, imino), 7.52–7.48 (m, 1H, aromatic)), 7.41 (d, <i>J</i> = 8.29, 2H, aromatic), 7.36 (d, <i>J</i> = 8.29, 2H, aromatic), 7.00 (d, <i>J</i> = 8.29, 1H, aromatic), 6.94 (d, <i>J</i> = 6.82, 1H, aromatic), 6.45 (t, <i>J</i> = 6.09, 1H, imino), 5.07–5.03 (m, 1H, hypomethyl), 3.97 (t, <i>J</i> = 7.07, 2H, methylene), 3.22–3.17 (m, 2H, methylene), 2.97–2.92 (m, 2H, methylene; s, 3H, methyl), 2.91–2.83 (m, 1H methylene), 2.72–2.66 (m, 1H, methylene), 2.57–2.46 (m, 1H, methylene; s, 3H, methyl), 2.45 (s, 3H, methyl), 2.36 (s, 3H, methyl), 2.05–2.00 (m, 2H, methylene), 1.99–1.94 (m, 1H, methylene), 1.89–1.81 (m, 2H, methylene), 1.52–1.46 (m, 2H, methylene), 1.33–1.28 (m, 2H, methylene), 1.28–1.18 (m, 8H, methylene). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 172.3 (C═O), 171.7 (C═O), 170.3 (C═O), 169.4 (C═O), 167.7 (C═O), 155.6 (C═O), 146.9 (C-aromatic), 141.6, 141.4, 139.8 (C-aromatic), 137.8, 136.8 (C-aromatic), 136.4 (C-aromatic), 132.6 (C-aromatic), 130.4 (C-aromatic), 126.2 (C-aromatic), 117.7 (C-aromatic), 117.6 (C-aromatic), 110.8 (C-aromatic), 109.4 (C-aromatic), 49.6, 49.1, 42.3, 38.9, 33.1, 31.6, 29.5, 29.1, 27.1, 26.8, 26.7, 25.2, 21.8, 21.2 (C-methyl), 19.9 (C-methyl), 12.3 (C-methyl). HR-MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>40</sub>H<sub>49</sub>N<sub>8</sub>O<sub>6</sub>, 737.3770; found, 737.3778. HPLC purity 95.5%.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 2-(2,6-Dioxopiperidin-3-yl)-5-fluoroisoindoline-1,3-dione (<b>20</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.13 (s, 1H), 8.02–8.00 (m, 1H), 7.85 (dd, <i>J</i> = 7.44, 2.31 Hz, 1H), 7.74–7.71 (m, 1H), 5.16 (dd, <i>J</i> = 13.00, 5.57 Hz, 1H), 2.92–2.86 (m, 1H), 2.63–2.58 (m, 1H), 2.57–2.51 (m, 1H), 2.09–2.04 (m, 1H).</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>tert</i>-Butyl (8-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)octyl)carbamate (<b>16i</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.05 (s, 1H), 7.54 (d, <i>J</i> = 8.41 Hz, 1H), 7.08 (t, <i>J</i> = 10.22, 4.95 Hz, 1H), 6.93–6.91 (m, 1H), 6.84–6.81 (m, 1H), 6.73 (t, <i>J</i> = 5.37 Hz, 1H), 5.04–4.99 (m, 1H), 3.15–3.11 (m, 2H), 2.90–2.86 (m, 2H), 2.86–2.82 (m, 1H), 2.59–2.54 (m, 1H), 2.53–2.49 (m, 1H), 2.00–1.95 (m, 1H), 1.58–1.52 (m, 2H), 1.35 (s, 9H), 1.33–1.21 (m, 10H).</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 4-(3,4-Dimethyl-7-oxo-2-(<i>p</i>-tolyl)-2,7-dihydro-6<i>H</i>-pyrazolo[3,4-<i>d</i>]pyridazin-6-yl)-<i>N</i>-(8-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)octyl)butanamide (<b>17j</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 11.03 (s, 1H, imino), 7.74 (t, <i>J</i> = 5.85 Hz, 1H, imino), 7.53 (d, <i>J</i> = 8.32 Hz, 1H, aromatic), 7.46 (d, <i>J</i> = 8.32 Hz, 2H, aromatic), 7.42 (d, <i>J</i> = 8.32 Hz, 2H, aromatic), 7.09–7.04 (m, 1H, aromatic), 6.93–6.89 (m, 1H, aromatic), 6.83–6.78 (m, 1H, imino), 5.03–4.98 (m, 1H, hypomethyl), 4.02 (t, <i>J</i> = 7.20 Hz, 2H, methylene), 3.14–3.08 (m, 2H, methylene), 3.03–2.97 (m, 2H, methylene), 2.90–2.82 (m, 1H, methylene), 2.63–2.54 (m, 1H, methylene; s, 3H, methyl), 2.53–2.49 (m, 1H, methylene; s, 3H, methyl), 2.41 (s, 3H, methyl), 2.07 (t, <i>J</i> = 7.42 Hz, 2H, methylene), 2.01–1.94 (m, 1H, methylene), 1.93–1.87 (m, 2H, methylene), 1.58–1.50 (m, 2H, methylene), 1.37–1.33 (m, 2H, methylene), 1.31–1.24 (m, 8H, methylene). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 150 MHz) δ: 173.3 (C═O), 171.7 (C═O), 170.6 (C═O), 168.2 (C═O), 167.6 (C═O), 155.6 (C═O), 154.6 (C-aromatic), 141.6, 141.4, 139.8 (C-aromatic), 137.9, 136.4 (C-aromatic), 134.7 (C-aromatic), 130.4 (C-aromatic), 126.2 (C-aromatic), 125.5 (C-aromatic), 117.6 (C-aromatic), 116.2 (C-aromatic), 49.0, 43.0, 38.9, 33.1, 31.5, 30.3, 29.5, 29.2, 26.9, 26.8, 25.2, 22.7, 21.2, 19.9 (C-methyl), 14.4 (C-methyl), 12.3 (C-methyl). HR-MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> C<sub>39</sub>H<sub>47</sub>N<sub>8</sub>O<sub>6</sub>, 723.3613; found, 723.3617. HPLC purity 99.2%.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Fluorescence Anisotropy Assay</h3><div class="NLM_p last">The fluorescence anisotropy assay was performed using our established standard protocol.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Cell Culture</h3><div class="NLM_p last">SW480 and HCT116 cells were cultured in DMEM (Hyclone) containing 10% fetal calf serum (Gibco) and 100 U/mL penicillin and 100 μg/mL streptomycin (Gibco). Cells were maintained in a humidified incubator with 95% air/5% CO<sub>2</sub> at 37 °C.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> In Vitro Antiproliferative Assay</h3><div class="NLM_p last">The cellular growth inhibitory activity was determined using KRAS-dependent SW480 and HCT116. Cells were seeded to 96-well plates at a density of 7 × 10<sup>3</sup> per well. After 24 h, the cells were treated with test compounds at different concentrations and 0.5% DMSO for control. After 72 h of incubation, the cell culture medium was removed. Then, 100 μL of fresh culture medium containing 10 μL of cell counting kit-8 (CCK8) was added to each well, and the plate was incubated at 37 °C for 20 min. The absorbance (OD) was tested using Biotek Synergy H2 at 450 nm. The concentration of the compounds that inhibited cell growth by 50% (IC<sub>50</sub>) was calculated.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Cell Apoptosis Assay</h3><div class="NLM_p last">SW480 cells were seeded at a density of 3 × 10<sup>5</sup> cells/well in 6-well plates. After 24 h, the cells were treated with compounds for another 24 h. Cells were then washed twice with PBS and detached by trypsin without EDTA. After centrifugation and removal of the supernatants, cells were resuspended in 300 μL of 1× binding buffer to which was added 5 μL of annexin V-FITC and 10 μL of PI according to the manufacturer’s protocol. The cells were then incubated at room temperature for 15 min in the dark. The stained cells were analyzed by a flow cytometer (BD Accuri C6).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Western Blotting</h3><div class="NLM_p last">SW480 cells were seeded at a density of 6 × 10<sup>5</sup> cells/well in 6-well plates (Corning). After 24 h, SW480 cells were treated with compounds for 24 h and then washed with PBS. Cells were then lysed with RIPA buffer containing protease inhibitors (EpiZyme no. GRF101) and phosphatase inhibitor (EpiZyme no. GRF102) on ice for 15 min. Then the cells were scraped off and centrifuged at 12 000<i>g</i> for 15 min at 4 °C. The cell supernatant was collected, and the protein concentration was tested with the BCA assay. Protein supernatant with equal total protein content was loaded on SDS–PAGE gels. Then the proteins were transferred to polyvinylidene fluoride membranes and blocked for 2 h with 5% BSA blocking buffer (5% BSA in TBST) at room temperature. The membranes were incubated overnight at 4 °C with primary antibodies: anti-PDEδ (GeneTex no. GTX109240, 1:1000), p44/42 MAPK (Erk1/2) (Cell Signaling Technology no. 9107, 1:1000), phospho p44/42 MAPK (Erk1/2) (Thr202/Tyr204) (Cell Signaling Technology no. 4370, 1:1000), anti-Akt1 (Epitomics no. 2118, 1:10000), anti-phospho Akt (Cell Signaling Technology no. 4060, 1:1000), GAPDH (Affinity no. AF7021, 1:3000). After a three-time-wash with TBST, the membranes were incubated with secondary antibodies: goat anti-rabbit (Abcam no. ab216777, 1:5000) and goat anti-mouse (Abcam no. ab216776, 1:5000). After samples were washed with TBST three times, blots were scanned on a LI-COR Odyssey imaging system. The protein levels were quantified by the gray values of the bands in the resulting images using the control group as the standard.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> In Vivo Antitumor Experiment</h3><div class="NLM_p last">The experimental procedures and the animal use and care protocols were approved by the Committee on Ethics of Medicine, Navy Medical University. BALB/C nude female mice (Certificate SCXK-2007-0005, weighing 18–20 g) were obtained from Shanghai Experimental Animal Center, Chinese Academy of Sciences. Each mouse was injected with 2 mL of SW480 cell suspensions containing 5 × 10<sup>6</sup> cells. When the volumes of the tumors reached around 100–300 mm<sup>3</sup>, animals were treated with compounds by intraperitoneal injection. Animals were randomly divided into three groups (5 mice in each group) and given test compounds <b>2</b>, <b>17f</b>, and vehicle (0.5% carboxymethylcellulose) separately. Mice were administered by ip injection with compounds or vehicle at the dose of 50 mg/kg once a day for 12 consecutive days. During treatment, tumor volumes were monitored by caliper measurement of the length and width and calculated using the formula of TV = <sup>1</sup>/<sub>2</sub> × <i>a</i> × <i>b</i><sup>2</sup>, where <i>a</i> is the tumor length and <i>b</i> is the width. Tumor volumes and body weights were monitored every 2 days. Mice were sacrificed on days 23–25 after implantation of cells, and tumors were removed and recorded for analysis.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Pharmacokinetic Studies</h3><div class="NLM_p last">This assay was conducted by Shanghai Medicilon Inc., China. Male SD rats (6–8 weeks old, body weight 180–220 g) were obtained from Sino-British SIPPR/BK Lab Animal Ltd., Shanghai. Three animals were given intraperitoneally at a dose of 50 mg/kg of compound <b>17f</b>. The compound was dissolved in 0.5% carboxymethylcellulose sodium (CMCNa, 800–1200 cps, in saline), and the dose concentration was 5 mg/mL. After injection of compound <b>17f</b>, the blood was taken via jugular vein at 0.083, 0.25, 0.5, 1, 2, 4, 6, 8, and 24 h, 0.20 mL/time point. Samples were placed in tubes containing K2-EDTA and stored on ice until centrifuged. The blood samples were centrifuged at 6800<i>g</i> for 6 min at 2–8 °C within 1 h after collection and stored frozen at approximately −80 °C until thawed for LC–MS/MS. Data acquisition: Analyst version 1.5.1. Pharmacokinetic data analysis was performed on mean plasma concentration–time data, and pharmacokinetic parameters were calculated using WinNonlin version 5.2.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00929" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03338" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03338" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00929?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00929</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular docking, Caco-2 permeability assay, immunofluorescence staining, and <sup>1</sup>H and <sup>13</sup>C NMR spectra of representative compound (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular docking of compound 2 with PDEδ (2_5E80.pdb) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">SMILES molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_001.pdf">jm0c00929_si_001.pdf (959.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_002.pdb">jm0c00929_si_002.pdb (203.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_003.csv">jm0c00929_si_003.csv (1.42 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00929" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61683" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61683" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guoqiang Dong</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry,
School of Pharmacy, Second Military Medical
University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1d797a6c30252c5d2c2b2e337e7270"><span class="__cf_email__" data-cfemail="d2b6b5a3ffeae392e3e4e1fcb1bdbf">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunquan Sheng</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry,
School of Pharmacy, Second Military Medical
University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9489-804X" title="Orcid link">http://orcid.org/0000-0001-9489-804X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e19289848f868290a1928c8c94cf848594cf828f"><span class="__cf_email__" data-cfemail="6e1d060b00090d1f2e1d03031b400b0a1b400d00">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junfei Cheng</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry,
School of Pharmacy, Second Military Medical
University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Li</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry,
School of Pharmacy, Second Military Medical
University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xu Wang</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry,
School of Pharmacy, Second Military Medical
University, 325 Guohe Road, Shanghai 200433, People’s Republic of China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.C. and Y.L. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2885-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74730" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Natural Science Foundation of China (Grants 21738002 and 81725020 to C. S., Grant 81903436 to Y.L.) and the National Key R&D Program of China (Grant 2017YFA0506000 to C.S.). Shanghai Jrdun Biotechnology Co. Ltd. provided technical assistance for Figure 7C.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis-targeting chimera</p></td></tr><tr><td class="NLM_term">Erk</td><td class="NLM_def"><p class="first last">extracellular signal-regulated kinase</p></td></tr><tr><td class="NLM_term">EGF</td><td class="NLM_def"><p class="first last">epidermal growth factor</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">2-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high performance liquid chromatography</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00867" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00867" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 37 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simanshu, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissley, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, F.</span></span> <span> </span><span class="NLM_article-title">RAS Proteins and Their Regulators in Human Disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.cell.2017.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=28666118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=17-33&author=D.+K.+Simanshuauthor=D.+V.+Nissleyauthor=F.+McCormick&title=RAS+Proteins+and+Their+Regulators+in+Human+Disease&doi=10.1016%2Fj.cell.2017.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">RAS Proteins and their regulators in human disease</span></div><div class="casAuthors">Simanshu, Dhirendra K.; Nissley, Dwight V.; McCormick, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-33</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  RAS proteins are binary switches, cycling between ON and OFF states during signal transduction.  These switches are normally tightly controlled, but in RAS-related diseases, such as cancer, RASopathies, and many psychiatric disorders, mutations in the RAS genes or their regulators render RAS proteins persistently active.  The structural basis of the switch and many of the pathways that RAS controls are well known, but the precise mechanisms by which RAS proteins function are less clear.  All RAS biol. occurs in membranes: a precise understanding of RAS' interaction with membranes is essential to understand RAS action and to intervene in RAS-driven diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoH9cHufj4FLVg90H21EOLACvtfcHk0lh3Ea6yOcNCiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsLfP&md5=638a0c0efac88f2873c352715e332a92</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DSimanshu%26aufirst%3DD.%2BK.%26aulast%3DNissley%26aufirst%3DD.%2BV.%26aulast%3DMcCormick%26aufirst%3DF.%26atitle%3DRAS%2520Proteins%2520and%2520Their%2520Regulators%2520in%2520Human%2520Disease%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D17%26epage%3D33%26doi%3D10.1016%2Fj.cell.2017.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratcioglu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Ras Conformational Ensembles, Allostery, and Signaling</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6607</span>– <span class="NLM_lpage">6665</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00542</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00542" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6607-6665&author=S.+Luauthor=H.+Jangauthor=S.+Muratciogluauthor=A.+Gursoyauthor=O.+Keskinauthor=R.+Nussinovauthor=J.+Zhang&title=Ras+Conformational+Ensembles%2C+Allostery%2C+and+Signaling&doi=10.1021%2Facs.chemrev.5b00542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Ras Conformational Ensembles, Allostery, and Signaling</span></div><div class="casAuthors">Lu, Shaoyong; Jang, Hyunbum; Muratcioglu, Serena; Gursoy, Attila; Keskin, Ozlem; Nussinov, Ruth; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6607-6665</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ras proteins are classical members of small GTPases that function as mol. switches by alternating between inactive GDP-bound and active GTP-bound states.  Ras activation is regulated by guanine nucleotide exchange factors that catalyze the exchange of GDP by GTP, and inactivation is terminated by GTPase-activating proteins that accelerate the intrinsic GTP hydrolysis rate by orders of magnitude.  In this review, we focus on data that have accumulated over the past few years pertaining to the conformational ensembles and the allosteric regulation of Ras proteins and their interpretation from our conformational landscape standpoint.  The Ras ensemble embodies all states, including the ligand-bound conformations, the activated (or inactivated) allosteric modulated states, post-translationally modified states, mutational states, transition states, and nonfunctional states serving as a reservoir for emerging functions.  The ensemble is shifted by distinct mutational events, cofactors, post-translational modifications, and different membrane compns.  A better understanding of Ras biol. can contribute to therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY2oezVaRCvbVg90H21EOLACvtfcHk0lhbDGeS0al02w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhu74%253D&md5=7600b971015bd3cf635ca1b4299dc7d5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00542%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DMuratcioglu%26aufirst%3DS.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DRas%2520Conformational%2520Ensembles%252C%2520Allostery%252C%2520and%2520Signaling%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6607%26epage%3D6665%26doi%3D10.1021%2Facs.chemrev.5b00542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forbes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bindal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kok, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beare, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teague, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futreal, P. A.</span></span> <span> </span><span class="NLM_article-title">COSMIC: Mining Complete Cancer Genomes in the Catalogue of Somatic Mutations in Cancer</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">D945</span>– <span class="NLM_lpage">950</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1093%2Fnar%2Fgkq929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=20952405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivF2msbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D945-950&author=S.+A.+Forbesauthor=N.+Bindalauthor=S.+Bamfordauthor=C.+Coleauthor=C.+Y.+Kokauthor=D.+Beareauthor=M.+Jiaauthor=R.+Shepherdauthor=K.+Leungauthor=A.+Menziesauthor=J.+W.+Teagueauthor=P.+J.+Campbellauthor=M.+R.+Strattonauthor=P.+A.+Futreal&title=COSMIC%3A+Mining+Complete+Cancer+Genomes+in+the+Catalogue+of+Somatic+Mutations+in+Cancer&doi=10.1093%2Fnar%2Fgkq929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">COSMIC: mining complete cancer genomes in the catalogue of somatic mutations in cancer</span></div><div class="casAuthors">Forbes, Simon A.; Bindal, Nidhi; Bamford, Sally; Cole, Charlotte; Kok, Chai Yin; Beare, David; Jia, Mingming; Shepherd, Rebecca; Leung, Kenric; Menzies, Andrew; Teague, Jon W.; Campbell, Peter J.; Stratton, Michael R.; Futreal, P. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">D945-D950</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">COSMIC (http://www.sanger.ac.uk/cosmic) curates comprehensive information on somatic mutations in human cancer.  Release v48 (July 2010) describes over 136 000 coding mutations in almost 542 000 tumor samples; of the 18 490 genes documented, 4803 (26%) have one or more mutations.  Full scientific literature curations are available on 83 major cancer genes and 49 fusion gene pairs (19 new cancer genes and 30 new fusion pairs this year) and this no. is continually increasing.  Key amongst these is TP53, now available through a collaboration with the IARC p53 database.  In addn. to data from the Cancer Genome Project (CGP) at the Sanger Institute, UK, and The Cancer Genome Atlas project (TCGA), large systematic screens are also now curated.  Major website upgrades now make these data much more mineable, with many new selection filters and graphics.  A Biomart is now available allowing more automated data mining and integration with other biol. databases.  Annotation of genomic features has become a significant focus; COSMIC has begun curating full-genome resequencing expts., developing new web pages, export formats and graphics styles.  With all genomic information recently updated to GRCh37, COSMIC integrates many diverse types of mutation information and is making much closer links with Ensembl and other data resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpalgzfv6Y9s7Vg90H21EOLACvtfcHk0lhbDGeS0al02w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivF2msbo%253D&md5=3456c7ae37b9f81da68a624a09d0d524</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq929%26sid%3Dliteratum%253Aachs%26aulast%3DForbes%26aufirst%3DS.%2BA.%26aulast%3DBindal%26aufirst%3DN.%26aulast%3DBamford%26aufirst%3DS.%26aulast%3DCole%26aufirst%3DC.%26aulast%3DKok%26aufirst%3DC.%2BY.%26aulast%3DBeare%26aufirst%3DD.%26aulast%3DJia%26aufirst%3DM.%26aulast%3DShepherd%26aufirst%3DR.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DMenzies%26aufirst%3DA.%26aulast%3DTeague%26aufirst%3DJ.%2BW.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26atitle%3DCOSMIC%253A%2520Mining%2520Complete%2520Cancer%2520Genomes%2520in%2520the%2520Catalogue%2520of%2520Somatic%2520Mutations%2520in%2520Cancer%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD945%26epage%3D950%26doi%3D10.1093%2Fnar%2Fgkq929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, K. L.</span></span> <span> </span><span class="NLM_article-title">RAS Isoforms and Mutations in Cancer at a Glance</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">1287</span>– <span class="NLM_lpage">1292</span>, <span class="refDoi"> DOI: 10.1242/jcs.182873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1242%2Fjcs.182873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=26985062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKltr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2016&pages=1287-1292&author=G.+A.+Hobbsauthor=C.+J.+Derauthor=K.+L.+Rossman&title=RAS+Isoforms+and+Mutations+in+Cancer+at+a+Glance&doi=10.1242%2Fjcs.182873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">RAS isoforms and mutations in cancer at a glance</span></div><div class="casAuthors">Hobbs, G. Aaron; Der, Channing J.; Rossman, Kent L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1287-1292</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">RAS proteins (KRAS4A, KRAS4B, NRAS and HRAS) function as GDP-GTP-regulated binary on-off switches, which regulate cytoplasmic signaling networks that control diverse normal cellular processes.  Gain-of-function missense mutations in RAS genes are found in ∼ 25% of human cancers, prompting interest in identifying anti-RAS therapeutic strategies for cancer treatment.  However, despite more than three decades of intense effort, no anti-RAS therapies have reached clin. application.  Contributing to this failure has been an underestimation of the complexities of RAS.  First, there is now appreciation that the four human RAS proteins are not functionally identical.  Second, with >130 different missense mutations found in cancer, there is an emerging view that there are mutation-specific consequences on RAS structure, biochem. and biol., and mutation-selective therapeutic strategies are needed.  In this Cell Science at a Glance article and accompanying poster, we provide a snapshot of the differences between RAS isoforms and mutations, as well as the current status of anti-RAS drug-discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNufX89IRJk7Vg90H21EOLACvtfcHk0lhbDGeS0al02w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKltr7I&md5=bf50d212b900fb437cad2b650a9dc07a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1242%2Fjcs.182873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.182873%26sid%3Dliteratum%253Aachs%26aulast%3DHobbs%26aufirst%3DG.%2BA.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26aulast%3DRossman%26aufirst%3DK.%2BL.%26atitle%3DRAS%2520Isoforms%2520and%2520Mutations%2520in%2520Cancer%2520at%2520a%2520Glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2016%26volume%3D129%26spage%3D1287%26epage%3D1292%26doi%3D10.1242%2Fjcs.182873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karachaliou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magri, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimenez-Capitan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina-Vila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosell, R.</span></span> <span> </span><span class="NLM_article-title">KRAS Mutations in Lung Cancer</span>. <i>Clin. Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/j.cllc.2012.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.cllc.2012.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=23122493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlslWitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=205-214&author=N.+Karachaliouauthor=C.+Mayoauthor=C.+Costaauthor=I.+Magriauthor=A.+Gimenez-Capitanauthor=M.+A.+Molina-Vilaauthor=R.+Rosell&title=KRAS+Mutations+in+Lung+Cancer&doi=10.1016%2Fj.cllc.2012.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">KRAS Mutations in Lung Cancer</span></div><div class="casAuthors">Karachaliou, Niki; Mayo, Clara; Costa, Carlota; Magri, Ignacio; Gimenez-Capitan, Ana; Molina-Vila, Miguel Angel; Rosell, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Lung Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-214</span>CODEN:
                <span class="NLM_cas:coden">CLCLCA</span>;
        ISSN:<span class="NLM_cas:issn">1525-7304</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Epidermal growth factor receptor (EGFR) gene mutations and increased EGFR copy nos. have been assocd. with a favorable response to EGFR tyrosine kinase inhibitors (TKI) in patients with non-small-cell lung cancer (NSCLC), and several markers have been identified that predict response to treatment.  Lung adenocarcinomas also harbor activating mutations in the downstream GTPase, v-Ki-ras2 Kirsten rat sarcoma viral oncogene (KRAS), and mutations in EGFR and KRAS appear to be mutually exclusive.  Even though KRAS mutations were identified in NSCLC tumors more than 20 years ago, we have only just begun to appreciate the clin. value of detg. KRAS tumor status.  Recent studies indicate that patients with mutant KRAS tumors fail to benefit from adjuvant chemotherapy and do not respond to EGFR inhibitors.  There is a clear need for therapies specifically developed for patients with KRAS-mutant NSCLC.  In this review, we summarize the clin. and pathol. characteristics of patients with NSCLC and with KRAS mutations, describe work that explores the predictive and prognostic influence of KRAS mutations, and provide an overview of the "synthetic lethal" interactions and current approaches to targeting KRAS-mutant NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzzE2yYQyuAbVg90H21EOLACvtfcHk0lisC_H1oomKYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlslWitbY%253D&md5=99127a23a5dfc3d06eaab4a206dfde45</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cllc.2012.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cllc.2012.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DKarachaliou%26aufirst%3DN.%26aulast%3DMayo%26aufirst%3DC.%26aulast%3DCosta%26aufirst%3DC.%26aulast%3DMagri%26aufirst%3DI.%26aulast%3DGimenez-Capitan%26aufirst%3DA.%26aulast%3DMolina-Vila%26aufirst%3DM.%2BA.%26aulast%3DRosell%26aufirst%3DR.%26atitle%3DKRAS%2520Mutations%2520in%2520Lung%2520Cancer%26jtitle%3DClin.%2520Lung%2520Cancer%26date%3D2013%26volume%3D14%26spage%3D205%26epage%3D214%26doi%3D10.1016%2Fj.cllc.2012.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Drugging RAS: Know the Enemy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1158</span>– <span class="NLM_lpage">1163</span>, <span class="refDoi"> DOI: 10.1126/science.aam7622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1126%2Fscience.aam7622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=28302824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1158-1163&author=B.+Papkeauthor=C.+J.+Der&title=Drugging+RAS%3A+Know+the+Enemy&doi=10.1126%2Fscience.aam7622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging RAS: Know the enemy</span></div><div class="casAuthors">Papke, Bjoern; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1158-1163</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  The three RAS oncogenes make up the most frequently mutated gene family in human cancer.  The well-validated role of mutationally activated RAS genes in driving cancer development and growth has stimulated comprehensive efforts to develop therapeutic strategies to block mutant RAS function for cancer treatment.  Disappointingly, despite more than three decades of research effort, clin. effective anti-RAS therapies have remained elusive, prompting a perception that RAS may be undruggable.  However, with a greater appreciation of the complexities of RAS that thwarted past efforts, and armed with new technologies and directions, the field is experiencing renewed excitement that mutant RAS may finally be conquered.  Here the authors summarize where these efforts stand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhrMhg-D4EM7Vg90H21EOLACvtfcHk0lisC_H1oomKYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmtrY%253D&md5=ce9da51019033a624c765ef2507bac2e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7622%26sid%3Dliteratum%253Aachs%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DDrugging%2520RAS%253A%2520Know%2520the%2520Enemy%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1158%26epage%3D1163%26doi%3D10.1126%2Fscience.aam7622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Drugging the Undruggable RAS: Mission Possible?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">828</span>– <span class="NLM_lpage">851</span>, <span class="refDoi"> DOI: 10.1038/nrd4389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnrd4389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=25323927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslKmu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=828-851&author=A.+D.+Coxauthor=S.+W.+Fesikauthor=A.+C.+Kimmelmanauthor=J.+Luoauthor=C.+J.+Der&title=Drugging+the+Undruggable+RAS%3A+Mission+Possible%3F&doi=10.1038%2Fnrd4389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the undruggable RAS: Mission Possible?</span></div><div class="casAuthors">Cox, Adrienne D.; Fesik, Stephen W.; Kimmelman, Alec C.; Luo, Ji; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">828-851</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite more than three decades of intensive effort, no effective pharmacol. inhibitors of the RAS oncoproteins have reached the clinic, prompting the widely held perception that RAS proteins are 'undruggable'.  However, recent data from the lab. and the clinic have renewed our hope for the development of RAS-inhibitory mols.  In this Review, we summarize the progress and the promise of five key approaches.  Firstly, we focus on the prospects of using direct inhibitors of RAS.  Secondly, we address the issue of whether blocking RAS membrane assocn. is a viable approach.  Thirdly, we assess the status of targeting RAS downstream effector signalling, which is arguably the most favorable current approach.  Fourthly, we address whether the search for synthetic lethal interactors of mutant RAS still holds promise.  Finally, RAS-mediated changes in cell metab. have recently been described and we discuss whether these changes could be exploited for new therapeutic directions.  We conclude with perspectives on how addnl. complexities, which are not yet fully understood, may affect each of these approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvWzW1JqxN4LVg90H21EOLACvtfcHk0lisC_H1oomKYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslKmu7zE&md5=6c8645293b079023de7d31011ae1a192</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd4389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4389%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DDrugging%2520the%2520Undruggable%2520RAS%253A%2520Mission%2520Possible%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D828%26epage%3D851%26doi%3D10.1038%2Fnrd4389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Bryan, J. P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Targeting of RAS: Recent Success with Direct Inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">503</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2018.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.phrs.2018.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=30366101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVCitbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2019&pages=503-511&author=J.+P.+O%E2%80%99Bryan&title=Pharmacological+Targeting+of+RAS%3A+Recent+Success+with+Direct+Inhibitors&doi=10.1016%2Fj.phrs.2018.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of RAS: Recent success with direct inhibitors</span></div><div class="casAuthors">O'Bryan, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">503-511</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  RAS has long been viewed as undruggable due to its lack of deep pockets for binding of small mol. inhibitors.  However, recent successes in the development of direct RAS inhibitors suggest that the goal of pharmacol. inhibition of RAS in patients may soon be realized.  This review will discuss the role of RAS in cancer, the approaches used to develop direct RAS inhibitors, and highlight recent successes in the development of novel RAS inhibitory compds. that target different aspects of RAS biochem.  In particular, this review will discuss the different properties of RAS that have been targeted by various inhibitors including membrane localization, the different activation states of RAS, effector binding, and nucleotide exchange.  In addn., this review will highlight the recent success with mutation-specific inhibitors that exploit the unique biochem. of the RAS(G12C) mutant.  Although this mutation in KRAS accounts for 11% of all KRAS mutations in cancer, it is the most prominent KRAS mutant in lung cancer suggesting that G12C-specific inhibitors may provide a new approach for treating the subset of lung cancer patients harboring this mutant allele.  Finally, this review will discuss the involvement of dimerization in RAS function and highlight new approaches to inhibit RAS by specifically interfering with RAS:RAS interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94gO4ck3GCbVg90H21EOLACvtfcHk0lhIJa5ttfX3dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVCitbjK&md5=5a62a1fe189f3fc6f3b8d05b1d7c6e0a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2018.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2018.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Bryan%26aufirst%3DJ.%2BP.%26atitle%3DPharmacological%2520Targeting%2520of%2520RAS%253A%2520Recent%2520Success%2520with%2520Direct%2520Inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2019%26volume%3D139%26spage%3D503%26epage%3D511%26doi%3D10.1016%2Fj.phrs.2018.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span> <span> </span><span class="NLM_article-title">Drugging K-Ras(G12C) through Covalent Inhibitors: Mission Possible?</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>202</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2019.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.pharmthera.2019.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=31233765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Ojs77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=202&publication_year=2019&pages=1-17&author=D.+Niauthor=X.+Liauthor=X.+Heauthor=H.+Zhangauthor=J.+Zhangauthor=S.+Lu&title=Drugging+K-Ras%28G12C%29+through+Covalent+Inhibitors%3A+Mission+Possible%3F&doi=10.1016%2Fj.pharmthera.2019.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging K-RasG12C through covalent inhibitors: Mission possible?</span></div><div class="casAuthors">Ni, Duan; Li, Xinyi; He, Xinheng; Zhang, Hao; Zhang, Jian; Lu, Shaoyong</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">202</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-17</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Ras, whose mutants are present in approx. 30% of human tumors, is one of the most important oncogenes.  Drugging Ras is thus regarded as the quest for the Holy Grail in cancer therapeutics development.  Despite more than three decades of efforts, drug discovery targeting Ras constantly fails, rendering Ras undruggable, due to its smooth surface and picomolar affinity towards guanosine substrates.  The most frequently mutated isoform of Ras is K-Ras, accounting for >85% of Ras-driven cancers, and one majority of them is the G12C mutation.  Recent advances in structural biol. shed light on drugging Ras, and one of the cutting-edge breakthroughs is the design of covalent G12C-specific inhibitors targeting the mutated cysteine.  This type of inhibitor can be classified into substrate-competitive orthosteric inhibitors and non-competitive allosteric inhibitors.  They display improved selectivity and enhanced potency due to their G12-specific and irreversible covalent binding nature.  Thus, they represent a new hope for revolutionizing the conventional characterization of Ras as "undruggable" and pave a promising avenue for further drug discovery.  Here, we provide comprehensive structural and medicinal chem. insights into K-Ras covalent inhibitors specific for the G12C mutant.  We first present an in-depth anal. of the conformations of the inhibitor binding pockets.  Then, all the latest covalent ligands selectively inhibiting K-RasG12C are reviewed.  Finally, we examine the current challenges faced by this new class of anti-Ras inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLkKui-Vl_zbVg90H21EOLACvtfcHk0lhIJa5ttfX3dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Ojs77M&md5=57887102a4d4ca32e55f76580fde55a7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2019.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2019.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%26atitle%3DDrugging%2520K-Ras%2528G12C%2529%2520through%2520Covalent%2520Inhibitors%253A%2520Mission%2520Possible%253F%26jtitle%3DPharmacol.%2520Ther.%26date%3D2019%26volume%3D202%26spage%3D1%26epage%3D17%26doi%3D10.1016%2Fj.pharmthera.2019.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span> <span> </span><span class="NLM_article-title">Ras Moves to Stay in Place</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">190</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2015.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.tcb.2015.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=25759176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsFGnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=190-197&author=M.+Schmickauthor=A.+Kraemerauthor=P.+I.+Bastiaens&title=Ras+Moves+to+Stay+in+Place&doi=10.1016%2Fj.tcb.2015.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Ras moves to stay in place</span></div><div class="casAuthors">Schmick, Malte; Kraemer, Astrid; Bastiaens, Philippe I. H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">190-197</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Ras is a major intracellular signaling hub.  This elevated position comes at a precarious cost: a single point mutation can cause aberrant signaling.  The capacity of Ras for signaling is inextricably linked to its enrichment at the plasma membrane (PM).  This PM localization is dynamically maintained by three essential elements: alteration of membrane affinities via lipidation and membrane-interaction motifs; trapping on specific membranes coupled with unidirectional vesicular transport to the PM; and regulation of diffusion via interaction with a solubilization factor.  This system constitutes a cycle that primarily corrects for the entropic equilibration of Ras to all membranes that dils. its signaling capacity.  We illuminate how this reaction-diffusion system maintains an out-of-equil. localization of Ras GTPases and thereby confers signaling functionality to the PM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8MYFhn_IGMLVg90H21EOLACvtfcHk0lhIJa5ttfX3dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsFGnsLc%253D&md5=ebcac88ec427423d2173b53ee51c9043</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2015.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2015.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DSchmick%26aufirst%3DM.%26aulast%3DKraemer%26aufirst%3DA.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26atitle%3DRas%2520Moves%2520to%2520Stay%2520in%2520Place%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2015%26volume%3D25%26spage%3D190%26epage%3D197%26doi%3D10.1016%2Fj.tcb.2015.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bierbaum, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gremer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triola, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span> <span> </span><span class="NLM_article-title">Arl2-GTP and Arl3-GTP Regulate a GDI-like Transport System for Farnesylated Cargo</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">942</span>– <span class="NLM_lpage">949</span>, <span class="refDoi"> DOI: 10.1038/nchembio.686</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnchembio.686" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=22002721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWltb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=942-949&author=S.+A.+Ismailauthor=Y.+X.+Chenauthor=A.+Rusinovaauthor=A.+Chandraauthor=M.+Bierbaumauthor=L.+Gremerauthor=G.+Triolaauthor=H.+Waldmannauthor=P.+I.+Bastiaensauthor=A.+Wittinghofer&title=Arl2-GTP+and+Arl3-GTP+Regulate+a+GDI-like+Transport+System+for+Farnesylated+Cargo&doi=10.1038%2Fnchembio.686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Arl2-GTP and Arl3-GTP regulate a GDI-like transport system for farnesylated cargo</span></div><div class="casAuthors">Ismail, Shehab A.; Chen, Yong-Xiang; Rusinova, Alexandra; Chandra, Anchal; Bierbaum, Martin; Gremer, Lothar; Triola, Gemma; Waldmann, Herbert; Bastiaens, Philippe I. H.; Wittinghofer, Alfred</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">942-949</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lipidated Rho and Rab GTP-binding proteins are transported between membranes in complex with solubilizing factors called 'guanine nucleotide dissocn. inhibitors' (GDIs).  Unloading from GDIs using GDI displacement factors (GDFs) has been proposed but remains mechanistically elusive.  PDEδ is a putative solubilizing factor for several prenylated Ras-subfamily proteins.  Here we report the structure of fully modified farnesylated Rheb-GDP in complex with PDEδ.  The structure explains the nucleotide-independent binding of Rheb to PDEδ and the relaxed specificity of PDEδ.  We demonstrate that the G proteins Arl2 and Arl3 act in a GTP-dependent manner as allosteric release factors for farnesylated cargo.  We thus describe a new transport system for farnesylated G proteins involving a GDI-like mol. and an unequivocal GDF.  Considering the importance of PDEδ for proper Ras and Rheb signaling, this study is instrumental in developing a new target for anticancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrENzFj11Fgq7Vg90H21EOLACvtfcHk0lhIJa5ttfX3dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWltb3F&md5=94dff580269780042ea4e6e5bd03f6a0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.686%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DS.%2BA.%26aulast%3DChen%26aufirst%3DY.%2BX.%26aulast%3DRusinova%26aufirst%3DA.%26aulast%3DChandra%26aufirst%3DA.%26aulast%3DBierbaum%26aufirst%3DM.%26aulast%3DGremer%26aufirst%3DL.%26aulast%3DTriola%26aufirst%3DG.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26aulast%3DWittinghofer%26aufirst%3DA.%26atitle%3DArl2-GTP%2520and%2520Arl3-GTP%2520Regulate%2520a%2520GDI-like%2520Transport%2520System%2520for%2520Farnesylated%2520Cargo%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D942%26epage%3D949%26doi%3D10.1038%2Fnchembio.686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozdemir, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Arl2-Mediated Allosteric Release of Farnesylated KRas4B from Shuttling Factor PDEdelta</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">7503</span>– <span class="NLM_lpage">7513</span>, <span class="refDoi"> DOI: 10.1021/acs.jpcb.8b04347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpcb.8b04347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Cmu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2018&pages=7503-7513&author=E.+S.+Ozdemirauthor=H.+Jangauthor=A.+Gursoyauthor=O.+Keskinauthor=R.+Nussinov&title=Arl2-Mediated+Allosteric+Release+of+Farnesylated+KRas4B+from+Shuttling+Factor+PDEdelta&doi=10.1021%2Facs.jpcb.8b04347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Arl2-Mediated Allosteric Release of Farnesylated KRas4B from Shuttling Factor PDEδ</span></div><div class="casAuthors">Ozdemir, E. Sila; Jang, Hyunbum; Gursoy, Attila; Keskin, Ozlem; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">7503-7513</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proper localization of Ras proteins at the plasma membrane (PM) is crucial for their functions.  To get to the PM, KRas4B, as well as some other Ras family proteins, bind to the cGMP phosphodiesterase δ-subunit (PDEδ) shuttling protein through their farnesylated hypervariable regions (HVRs).  The docking of their farnesyl (and to a lesser extent geranylgeranyl) in PDEδ's hydrophobic pocket stabilizes the interaction.  At the PM, GTP-bound Arf-like protein 2 (Arl2) assists in the release of Ras from PDEδ.  However, exactly how is still unclear.  Here, using all-atom mol. dynamics simulations, we unraveled the detailed mechanism of Arl2-mediated release of KRas4B, the most abundant oncogenic Ras isoform, from PDEδ.  We simulated ternary Arl2-PDEδ-KRas4B HVR complexes and obsd. that Arl2 binding weakened the PDEδ-farnesylated HVR interaction.  Our detailed anal. showed that allosteric changes (involving β6 of PDEδ and addnl. PDEδ residues) compressed the hydrophobic PDEδ pocket and pushed the HVR out.  Mutating PDEδ residues that mediate allosteric changes in PDEδ terminated the release process.  Mutant Ras proteins are enriched in human cancers, with currently no drugs in the clinics.  This mechanistic account may inspire efforts to develop drugs suppressing oncogenic KRas4B release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5Blyv3puJz7Vg90H21EOLACvtfcHk0lhrd5Wf1SIVJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cmu7zO&md5=0275a619c9714189483cd816423700bf</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jpcb.8b04347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpcb.8b04347%26sid%3Dliteratum%253Aachs%26aulast%3DOzdemir%26aufirst%3DE.%2BS.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DArl2-Mediated%2520Allosteric%2520Release%2520of%2520Farnesylated%2520KRas4B%2520from%2520Shuttling%2520Factor%2520PDEdelta%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2018%26volume%3D122%26spage%3D7503%26epage%3D7513%26doi%3D10.1021%2Facs.jpcb.8b04347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartak, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovacevic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truxius, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmannek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I. H.</span></span> <span> </span><span class="NLM_article-title">KRAS Localizes to the Plasma Membrane by Spatial Cycles of Solubilization, Trapping and Vesicular Transport</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.02.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.cell.2014.02.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=24725411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmslals7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=459-471&author=M.+Schmickauthor=N.+Vartakauthor=B.+Papkeauthor=M.+Kovacevicauthor=D.+C.+Truxiusauthor=L.+Rossmannekauthor=P.+I.+H.+Bastiaens&title=KRAS+Localizes+to+the+Plasma+Membrane+by+Spatial+Cycles+of+Solubilization%2C+Trapping+and+Vesicular+Transport&doi=10.1016%2Fj.cell.2014.02.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">KRas Localizes to the Plasma Membrane by Spatial Cycles of Solubilization, Trapping and Vesicular Transport</span></div><div class="casAuthors">Schmick, Malte; Vartak, Nachiket; Papke, Bjoern; Kovacevic, Marija; Truxius, Dina C.; Rossmannek, Lisaweta; Bastiaens, Philippe I. H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">459-471</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRas is a major proto-oncogene product whose signaling activity depends on its level of enrichment on the plasma membrane (PM).  This PM localization relies on posttranslational prenylation for membrane affinity, while PM specificity has been attributed to electrostatic interactions between neg. charged phospholipids in the PM and basic amino-acids in the C terminus of KRas.  By measuring kinetic parameters of KRas dynamics in living cells with a cellular-automata-based data-fitting approach in realistic cell-geometries, we show that charge-based specificity is not sufficient to generate PM enrichment in light of the total surface area of endomembranes.  Instead, mislocalized KRas is continuously sequestered from endomembranes by cytosolic PDEδ to be unloaded in an Arl2-dependent manner to perinuclear membranes.  Electrostatic interactions then trap KRas at the recycling endosome (RE), from where vesicular transport restores enrichment on the PM.  This energy driven reaction-diffusion cycle explains how small mol. targeting of PDEδ affects the spatial organization of KRas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnu-qiKkSobVg90H21EOLACvtfcHk0lhrd5Wf1SIVJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmslals7w%253D&md5=c8e31ee5983495be339785b97a94bac8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.02.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.02.051%26sid%3Dliteratum%253Aachs%26aulast%3DSchmick%26aufirst%3DM.%26aulast%3DVartak%26aufirst%3DN.%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DKovacevic%26aufirst%3DM.%26aulast%3DTruxius%26aufirst%3DD.%2BC.%26aulast%3DRossmannek%26aufirst%3DL.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%2BH.%26atitle%3DKRAS%2520Localizes%2520to%2520the%2520Plasma%2520Membrane%2520by%2520Spatial%2520Cycles%2520of%2520Solubilization%252C%2520Trapping%2520and%2520Vesicular%2520Transport%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D459%26epage%3D471%26doi%3D10.1016%2Fj.cell.2014.02.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murarka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Gago, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovacevic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truxius, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fansa, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinelt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Saabi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span> <span> </span><span class="NLM_article-title">Identification of Pyrazolopyridazinones as PDEdelta Inhibitors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11360</span>, <span class="refDoi"> DOI: 10.1038/ncomms11360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fncomms11360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=27094677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1Kns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=11360&author=B.+Papkeauthor=S.+Murarkaauthor=H.+A.+Vogelauthor=P.+Martin-Gagoauthor=M.+Kovacevicauthor=D.+C.+Truxiusauthor=E.+K.+Fansaauthor=S.+Ismailauthor=G.+Zimmermannauthor=K.+Heineltauthor=C.+Schultz-Fademrechtauthor=A.+Al+Saabiauthor=M.+Baumannauthor=P.+Nussbaumerauthor=A.+Wittinghoferauthor=H.+Waldmannauthor=P.+I.+Bastiaens&title=Identification+of+Pyrazolopyridazinones+as+PDEdelta+Inhibitors&doi=10.1038%2Fncomms11360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of pyrazolopyridazinones as PDEδ inhibitors</span></div><div class="casAuthors">Papke, Bjoern; Murarka, Sandip; Vogel, Holger A.; Martin-Gago, Pablo; Kovacevic, Marija; Truxius, Dina C.; Fansa, Eyad K.; Ismail, Shehab; Zimmermann, Gunther; Heinelt, Kaatje; Schultz-Fademrecht, Carsten; Al Saabi, Alaa; Baumann, Matthias; Nussbaumer, Peter; Wittinghofer, Alfred; Waldmann, Herbert; Bastiaens, Philippe I. H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11360</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The prenyl-binding protein PDEδ is crucial for the plasma membrane localization of prenylated Ras.  Recently, we have reported that the small-mol. Deltarasin binds to the prenyl-binding pocket of PDEδ, and impairs Ras enrichment at the plasma membrane, thereby affecting the proliferation of KRas-dependent human pancreatic ductal adenocarcinoma cell lines.  Here, using structure-based compd. design, we have now identified pyrazolopyridazinones as a novel, unrelated chemotype that binds to the prenyl-binding pocket of PDEδ with high affinity, thereby displacing prenylated Ras proteins in cells.  Our results show that the new PDEδ inhibitor, named Deltazinone 1, is highly selective, exhibits less unspecific cytotoxicity than the previously reported Deltarasin and demonstrates a high correlation with the phenotypic effect of PDEδ knockdown in a set of human pancreatic cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwY6Yus7Koa7Vg90H21EOLACvtfcHk0lhrd5Wf1SIVJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1Kns7s%253D&md5=7d7d6e83bc2ff4e54fc1132330b9cff3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncomms11360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms11360%26sid%3Dliteratum%253Aachs%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DMurarka%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%2BA.%26aulast%3DMartin-Gago%26aufirst%3DP.%26aulast%3DKovacevic%26aufirst%3DM.%26aulast%3DTruxius%26aufirst%3DD.%2BC.%26aulast%3DFansa%26aufirst%3DE.%2BK.%26aulast%3DIsmail%26aufirst%3DS.%26aulast%3DZimmermann%26aufirst%3DG.%26aulast%3DHeinelt%26aufirst%3DK.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DAl%2BSaabi%26aufirst%3DA.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DNussbaumer%26aufirst%3DP.%26aulast%3DWittinghofer%26aufirst%3DA.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26atitle%3DIdentification%2520of%2520Pyrazolopyridazinones%2520as%2520PDEdelta%2520Inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D11360%26doi%3D10.1038%2Fncomms11360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartak, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triola, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibition of the KRAS-PDEdelta Interaction Impairs Oncogenic KRAS Signalling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>497</i></span>,  <span class="NLM_fpage">638</span>– <span class="NLM_lpage">642</span>, <span class="refDoi"> DOI: 10.1038/nature12205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnature12205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=23698361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotFSgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=497&publication_year=2013&pages=638-642&author=G.+Zimmermannauthor=B.+Papkeauthor=S.+Ismailauthor=N.+Vartakauthor=A.+Chandraauthor=M.+Hoffmannauthor=S.+A.+Hahnauthor=G.+Triolaauthor=A.+Wittinghoferauthor=P.+I.+Bastiaensauthor=H.+Waldmann&title=Small+Molecule+Inhibition+of+the+KRAS-PDEdelta+Interaction+Impairs+Oncogenic+KRAS+Signalling&doi=10.1038%2Fnature12205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of the KRAS-PDEδ interaction impairs oncogenic KRAS signalling</span></div><div class="casAuthors">Zimmermann, Gunther; Papke, Bjoern; Ismail, Shehab; Vartak, Nachiket; Chandra, Anchal; Hoffmann, Maike; Hahn, Stephan A.; Triola, Gemma; Wittinghofer, Alfred; Bastiaens, Philippe I. H.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">497</span>
        (<span class="NLM_cas:issue">7451</span>),
    <span class="NLM_cas:pages">638-642</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The KRAS oncogene product is considered a major target in anticancer drug discovery.  However, direct interference with KRAS signalling has not yet led to clin. useful drugs.  Correct localization and signalling by farnesylated KRAS is regulated by the prenyl-binding protein PDEδ, which sustains the spatial organization of KRAS by facilitating its diffusion in the cytoplasm.  Here we report that interfering with binding of mammalian PDEδ to KRAS by means of small mols. provides a novel opportunity to suppress oncogenic RAS signalling by altering its localization to endomembranes.  Biochem. screening and subsequent structure-based hit optimization yielded inhibitors of the KRAS-PDEδ interaction that selectively bind to the prenyl-binding pocket of PDEδ with nanomolar affinity, inhibit oncogenic RAS signalling and suppress in vitro and in vivo proliferation of human pancreatic ductal adenocarcinoma cells that are dependent on oncogenic KRAS.  Our findings may inspire novel drug discovery efforts aimed at the development of drugs targeting oncogenic RAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4LMNyMIV4LVg90H21EOLACvtfcHk0lhA3m1aOqtwrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotFSgtbs%253D&md5=e8dfd498bfd4dafbebf07fee9128e6b0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature12205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12205%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DG.%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DIsmail%26aufirst%3DS.%26aulast%3DVartak%26aufirst%3DN.%26aulast%3DChandra%26aufirst%3DA.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DHahn%26aufirst%3DS.%2BA.%26aulast%3DTriola%26aufirst%3DG.%26aulast%3DWittinghofer%26aufirst%3DA.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DSmall%2520Molecule%2520Inhibition%2520of%2520the%2520KRAS-PDEdelta%2520Interaction%2520Impairs%2520Oncogenic%2520KRAS%2520Signalling%26jtitle%3DNature%26date%3D2013%26volume%3D497%26spage%3D638%26epage%3D642%26doi%3D10.1038%2Fnature12205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Gago, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fansa, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murarka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schurmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">A PDE6delta-KRAS Inhibitor Chemotype with up to Seven H-Bonds and Picomolar Affinity that Prevents Efficient Inhibitor Release by Arl2</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2423</span>– <span class="NLM_lpage">2428</span>, <span class="refDoi"> DOI: 10.1002/anie.201610957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1002%2Fanie.201610957" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsValt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=2423-2428&author=P.+Martin-Gagoauthor=E.+K.+Fansaauthor=C.+H.+Kleinauthor=S.+Murarkaauthor=P.+Janningauthor=M.+Schurmannauthor=M.+Metzauthor=S.+Ismailauthor=C.+Schultz-Fademrechtauthor=M.+Baumannauthor=P.+I.+Bastiaensauthor=A.+Wittinghoferauthor=H.+Waldmann&title=A+PDE6delta-KRAS+Inhibitor+Chemotype+with+up+to+Seven+H-Bonds+and+Picomolar+Affinity+that+Prevents+Efficient+Inhibitor+Release+by+Arl2&doi=10.1002%2Fanie.201610957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A PDE6δ-KRas Inhibitor Chemotype with up to Seven H-Bonds and Picomolar Affinity that Prevents Efficient Inhibitor Release by Arl2</span></div><div class="casAuthors">Martin-Gago, Pablo; Fansa, Eyad K.; Klein, Christian H.; Murarka, Sandip; Janning, Petra; Schuermann, Marc; Metz, Malte; Ismail, Shehab; Schultz-Fademrecht, Carsten; Baumann, Matthias; Bastiaens, Philippe I. H.; Wittinghofer, Alfred; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2423-2428</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Small-mol. inhibition of the interaction between the KRas oncoprotein and the chaperone PDE6δ impairs KRas spatial organization and signaling in cells.  However, despite potent binding in vitro (KD<10 nM), interference with Ras signaling and growth inhibition require 5-20 μm compd. concns.  We demonstrate that these findings can be explained by fast release of high-affinity inhibitors from PDE6δ by the release factor Arl2.  This limitation is overcome by novel highly selective inhibitors that bind to PDE6δ with up to 7 hydrogen bonds, resulting in picomolar affinity.  Their release by Arl2 is greatly decreased, and representative compds. selectively inhibit growth of KRas mutated and -dependent cells with the highest activity recorded yet.  Our findings indicate that very potent inhibitors of the KRas-PDE6δ interaction may impair the growth of tumors driven by oncogenic KRas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGRXyzHR151LVg90H21EOLACvtfcHk0lhA3m1aOqtwrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsValt7k%253D&md5=2b66846ac0d9806f58b428d83819a7c7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fanie.201610957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201610957%26sid%3Dliteratum%253Aachs%26aulast%3DMartin-Gago%26aufirst%3DP.%26aulast%3DFansa%26aufirst%3DE.%2BK.%26aulast%3DKlein%26aufirst%3DC.%2BH.%26aulast%3DMurarka%26aufirst%3DS.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DSchurmann%26aufirst%3DM.%26aulast%3DMetz%26aufirst%3DM.%26aulast%3DIsmail%26aufirst%3DS.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26aulast%3DWittinghofer%26aufirst%3DA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DA%2520PDE6delta-KRAS%2520Inhibitor%2520Chemotype%2520with%2520up%2520to%2520Seven%2520H-Bonds%2520and%2520Picomolar%2520Affinity%2520that%2520Prevents%2520Efficient%2520Inhibitor%2520Release%2520by%2520Arl2%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D2423%26epage%3D2428%26doi%3D10.1002%2Fanie.201610957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Structural Biology-Inspired Discovery of Novel KRAS-PDEdelta Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9400</span>– <span class="NLM_lpage">9406</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01243</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01243" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeiurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9400-9406&author=Y.+Jiangauthor=C.+Zhuangauthor=L.+Chenauthor=J.+Luauthor=G.+Dongauthor=Z.+Miaoauthor=W.+Zhangauthor=J.+Liauthor=C.+Sheng&title=Structural+Biology-Inspired+Discovery+of+Novel+KRAS-PDEdelta+Inhibitors&doi=10.1021%2Facs.jmedchem.7b01243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Biology-Inspired Discovery of Novel KRAS-PDEδ Inhibitors</span></div><div class="casAuthors">Jiang, Yan; Zhuang, Chunlin; Chen, Long; Lu, Junjie; Dong, Guoqiang; Miao, Zhenyuan; Zhang, Wannian; Li, Jian; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9400-9406</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural biol. is a powerful tool for investigating the stereospecific interactions between a protein and its ligand.  Herein, an unprecedented chiral binding pattern was obsd. for inhibitors of KRAS-PDEδ interactions.  Virtual screening and X-ray crystallog. studies revealed that two enantiomers of a racemic inhibitor could bind at different sites.  Fragment-based drug design was used to identify highly potent PDEδ inhibitors that can be used as promising lead compds. for target validation and antitumor drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo93B5ydLZud7Vg90H21EOLACvtfcHk0lhA3m1aOqtwrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeiurbM&md5=4755e148413d9eeb0c8eb818b9a4cc53</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01243%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DZhuang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DStructural%2520Biology-Inspired%2520Discovery%2520of%2520Novel%2520KRAS-PDEdelta%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9400%26epage%3D9406%26doi%3D10.1021%2Facs.jmedchem.7b01243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel KRAS-PDEdelta Inhibitors by Fragment-Based Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2604</span>– <span class="NLM_lpage">2610</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00057</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00057" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvFWnsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2604-2610&author=L.+Chenauthor=C.+Zhuangauthor=J.+Luauthor=Y.+Jiangauthor=C.+Sheng&title=Discovery+of+Novel+KRAS-PDEdelta+Inhibitors+by+Fragment-Based+Drug+Design&doi=10.1021%2Facs.jmedchem.8b00057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel KRAS-PDEδ Inhibitors by Fragment-Based Drug Design</span></div><div class="casAuthors">Chen, Long; Zhuang, Chunlin; Lu, Junjie; Jiang, Yan; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2604-2610</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting KRAS-PDEδ protein-protein interactions with small mols. represents a promising opportunity for developing novel antitumor agents.  However, current KRAS-PDEδ inhibitors are limited by poor cellular antitumor potency and the druggability of the target remains to be validated by new inhibitors.  To tackle these challenges, herein, novel, highly potent KRAS-PDEδ inhibitors were identified by fragment-based drug design, providing promising lead compds. or chem. probes for investigating the biol. functions and druggability of KRAS-PDEδ interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqayXxD-smS5LVg90H21EOLACvtfcHk0ljCXXupx2cDSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvFWnsLw%253D&md5=0b1ad029636a14f2794d548235a30360</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00057%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZhuang%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520Novel%2520KRAS-PDEdelta%2520Inhibitors%2520by%2520Fragment-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2604%26epage%3D2610%26doi%3D10.1021%2Facs.jmedchem.8b00057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainolfi, N.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation Mechanisms</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.ddtec.2019.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=31200860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=53-60&author=Y.+Zhangauthor=C.+Lohauthor=J.+Chenauthor=N.+Mainolfi&title=Targeted+Protein+Degradation+Mechanisms&doi=10.1016%2Fj.ddtec.2019.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation mechanisms</span></div><div class="casAuthors">Zhang Yi; Loh Christine; Chen Jesse; Mainolfi Nello</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeted protein degradation mediated by small molecule degraders represents an exciting new therapeutic opportunity to eliminate disease-causing proteins.  These molecules recruit E3 ubiquitin ligases to the protein of interest and mediate its ubiquitination and subsequent proteolysis by the proteasome.  Significant advancements have been made in the discovery and development of clinically relevant degraders.  In this review we will focus on the recent progress in understanding ternary complex formation and structures, ubiquitination, and other critical factors that govern the efficiency of degraders both in vitro and in vivo.  With deeper knowledges of these areas, the field is building guiding principles to reduce the level of empiricism and to identify therapeutically relevant degraders more rationally and efficiently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRBrI9DTGjC0fW6udTcc2eZ4UTiT6PAU2bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtg%253D%253D&md5=bff167555f8bcab6b672cad3c2946a7b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLoh%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMainolfi%26aufirst%3DN.%26atitle%3DTargeted%2520Protein%2520Degradation%2520Mechanisms%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D53%26epage%3D60%26doi%3D10.1016%2Fj.ddtec.2019.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation by PROTACs</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">138</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.pharmthera.2017.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=28223226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=138-144&author=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=C.+M.+Crews&title=Targeted+Protein+Degradation+by+PROTACs&doi=10.1016%2Fj.pharmthera.2017.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation by PROTACs</span></div><div class="casAuthors">Neklesa, Taavi K.; Winkler, James D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">138-144</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using the PROTAC technol. is emerging as a novel therapeutic method to address diseases driven by the aberrant expression of a disease-causing protein.  PROTAC mols. are bifunctional small mols. that simultaneously bind a target protein and an E3-ubiquitin ligase, thus causing ubiquitination and degrdn. of the target protein by the proteasome.  Like small mols., PROTAC mols. possess good tissue distribution and the ability to target intracellular proteins.  Herein, we highlight the advantages of protein degrdn. using PROTACs, and provide specific examples where degrdn. offers therapeutic benefit over classical enzyme inhibition.  Foremost, PROTACs can degrade proteins regardless of their function.  This includes the currently "undruggable" proteome, which comprises approx. 85% of all human proteins.  Other beneficial aspects of protein degrdn. include the ability to target overexpressed and mutated proteins, as well as the potential to demonstrate prolonged pharmacodynamics effect beyond drug exposure.  Lastly, due to their catalytic nature and the pre-requisite ubiquitination step, an exquisitely potent mols. with a high degree of degrdn. selectivity can be designed.  Impressive preclin. in vitro and in vivo PROTAC data have been published, and these data have propelled the development of clin. viable PROTACs.  With the mol. wt. falling in the 700-1000 Da range, the delivery and bioavailability of PROTACs remain the largest hurdles on the way to the clinic.  Solving these issues and demonstrating proof of concept clin. data will be the focus of many labs over the next few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4RSLWYqqBMrVg90H21EOLACvtfcHk0ljCXXupx2cDSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D&md5=08a74402a274a13c8c484cd68167a5a1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%2520by%2520PROTACs%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D174%26spage%3D138%26epage%3D144%26doi%3D10.1016%2Fj.pharmthera.2017.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: New Approaches to Protein Degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+New+Approaches+to+Protein+Degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0liEhOgv8I5UDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520New%2520Approaches%2520to%2520Protein%2520Degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+Kinase-Independent+Functions+of+Fak+via+PROTAC-Mediated+Degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0liEhOgv8I5UDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520Kinase-Independent%2520Functions%2520of%2520Fak%2520via%2520PROTAC-Mediated%2520Degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-Induced BET Protein Degradation as a Therapy for Castration-resistant Prostate Cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-Induced+BET+Protein+Degradation+as+a+Therapy+for+Castration-resistant+Prostate+Cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0liEhOgv8I5UDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-Induced%2520BET%2520Protein%2520Degradation%2520as%2520a%2520Therapy%2520for%2520Castration-resistant%2520Prostate%2520Cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Estrogen Receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.+Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+Highly+Potent+Proteolysis+Targeting+Chimera+%28PROTAC%29+Degrader+of+Estrogen+Receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lheC8sj9kEehw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520Highly%2520Potent%2520Proteolysis%2520Targeting%2520Chimera%2520%2528PROTAC%2529%2520Degrader%2520of%2520Estrogen%2520Receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-Induced BTK Degradation as a Novel Therapy for Mutated BTK C481S Induced Ibrutinib-Resistant B-cell Malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-Induced+BTK+Degradation+as+a+Novel+Therapy+for+Mutated+BTK+C481S+Induced+Ibrutinib-Resistant+B-cell+Malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0lheC8sj9kEehw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-Induced%2520BTK%2520Degradation%2520as%2520a%2520Novel%2520Therapy%2520for%2520Mutated%2520BTK%2520C481S%2520Induced%2520Ibrutinib-Resistant%2520B-cell%2520Malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, S. J.</span></span> <span> </span><span class="NLM_article-title">Chimeric Molecules Facilitate the Degradation of Androgen Receptors and Repress the Growth of LNCaP Cells</span>. <i>Asian J. Androl.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1038/aja.2008.26</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Faja.2008.26" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=19050678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnvFCqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=119-126&author=Y.+Q.+Tangauthor=B.+M.+Hanauthor=X.+Q.+Yaoauthor=Y.+Hongauthor=Y.+Wangauthor=F.+J.+Zhaoauthor=S.+Q.+Yuauthor=X.+W.+Sunauthor=S.+J.+Xia&title=Chimeric+Molecules+Facilitate+the+Degradation+of+Androgen+Receptors+and+Repress+the+Growth+of+LNCaP+Cells&doi=10.1038%2Faja.2008.26"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Chimeric molecules facilitate the degradation of androgen receptors and repress the growth of LNCaP cells</span></div><div class="casAuthors">Tang, Yue-Qing; Han, Bang-Min; Yao, Xin-Quan; Hong, Yan; Wang, Yan; Zhao, Fu-Jun; Yu, Sheng-Qiang; Sun, Xiao-Wen; Xia, Shu-Jie</div><div class="citationInfo"><span class="NLM_cas:title">Asian Journal of Andrology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">119-126</span>CODEN:
                <span class="NLM_cas:coden">ASJAF8</span>;
        ISSN:<span class="NLM_cas:issn">1008-682X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Post-translational degrdn. of protein plays an important role in cell life.  The authors employed chimeric mols. (dihydrotestosterone-based proteolysis-targeting chimeric mol. [DHT-PROTAC]) to facilitate androgen receptor (AR) degrdn. via the ubiquitin-proteasome pathway (UPP) and to investigate the role of AR in cell proliferation and viability in androgen-sensitive prostate cancer cells.  Western blot anal. and immunohistochem. were applied to analyze AR levels in LNCaP cells after DHT-PROTAC treatment.  Cell counting and the 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl tetrazolium bromide (MTT) cell viability assay were used to evaluate cell proliferation and viability after AR elimination in both LNCaP and PC-3 cells.  AR was tagged for elimination via the UPP by DHT-PROTAC, and this could be blocked by proteasome inhibitors.  Degrdn. of AR depended on DHT-PROTAC concn., and either DHT or an ALAPYIP-(arg)8 peptide could compete with DHT-PROTAC.  Inhibition of cell proliferation and decreased viability were obsd. in LNCaP cells, but not in PC-3 or 786-O cells after DHT-PROTAC treatment.  These data indicate that AR elimination is facilitated via the UPP by DHT-PROTAC, and that the growth of LNCaP cells is repressed after AR degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxakeBlqmp1LVg90H21EOLACvtfcHk0lheC8sj9kEehw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnvFCqsr4%253D&md5=7bc5fedd68553b2898725bfa877e13c0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Faja.2008.26&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Faja.2008.26%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DY.%2BQ.%26aulast%3DHan%26aufirst%3DB.%2BM.%26aulast%3DYao%26aufirst%3DX.%2BQ.%26aulast%3DHong%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DF.%2BJ.%26aulast%3DYu%26aufirst%3DS.%2BQ.%26aulast%3DSun%26aufirst%3DX.%2BW.%26aulast%3DXia%26aufirst%3DS.%2BJ.%26atitle%3DChimeric%2520Molecules%2520Facilitate%2520the%2520Degradation%2520of%2520Androgen%2520Receptors%2520and%2520Repress%2520the%2520Growth%2520of%2520LNCaP%2520Cells%26jtitle%3DAsian%2520J.%2520Androl.%26date%3D2009%26volume%3D11%26spage%3D119%26epage%3D126%26doi%3D10.1038%2Faja.2008.26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-Induced+Protein+Degradation+in+Drug+Discovery%3A+Breaking+the+Rules+or+Just+Making+New+Ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0ljlw6-0uvSBmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-Induced%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%253A%2520Breaking%2520the%2520Rules%2520or%2520Just%2520Making%2520New%2520Ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span> <span> </span><span class="NLM_article-title">Development of Targeted Protein Degradation Therapeutics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">944</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0362-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fs41589-019-0362-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=31527835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=937-944&author=P.+P.+Chamberlainauthor=L.+G.+Hamann&title=Development+of+Targeted+Protein+Degradation+Therapeutics&doi=10.1038%2Fs41589-019-0362-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Development of targeted protein degradation therapeutics</span></div><div class="casAuthors">Chamberlain, Philip P.; Hamann, Lawrence G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">937-944</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted protein degrdn. as a therapeutic modality has seen dramatic progress and massive investment in recent years because of the convergence of two key scientific breakthroughs: optimization of first-generation peptidic proteolysis-targeted chimeras (PROTACs) into more drug-like mols. able to support in vivo proof of concept and the discovery that clin. mols. function as degraders by binding and repurposing the proteins cereblon and DCAF15.  This provided clin. validation for the general approach through the cereblon modulator class of drugs and provided highly drug-like and ligand-efficient E3 ligase binders upon which to tether target-binding moieties.  Increasingly rational and systematic approaches including biophys. and structural studies on ternary complexes are being leveraged as the field advances.  In this Perspective we summarize the discoveries that have laid the foundation for future degrdn. therapeutics, focusing on those classes of small mols. that redirect E3 ubiquitin ligases to non-native substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfcIEwVR-Xq7Vg90H21EOLACvtfcHk0ljlw6-0uvSBmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF&md5=468e3fd87ba2b21607dac8b495e5ecf6</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0362-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0362-y%26sid%3Dliteratum%253Aachs%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26atitle%3DDevelopment%2520of%2520Targeted%2520Protein%2520Degradation%2520Therapeutics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D937%26epage%3D944%26doi%3D10.1038%2Fs41589-019-0362-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">First Targeted Protein Degrader Hits the Clinic</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00043-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fd41573-019-00043-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=237-239&author=A.+Mullard&title=First+Targeted+Protein+Degrader+Hits+the+Clinic&doi=10.1038%2Fd41573-019-00043-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00043-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00043-6%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DFirst%2520Targeted%2520Protein%2520Degrader%2520Hits%2520the%2520Clinic%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D237%26epage%3D239%26doi%3D10.1038%2Fd41573-019-00043-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><a href="https://ir.arvinas.com/press-releases?page=1" class="extLink">https://ir.arvinas.com/press-releases?page=1</a> (accessed Oct 15, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fir.arvinas.com%2Fpress-releases%3Fpage%3D1+%28accessed+Oct+15%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">The PROTAC Technology in Drug Development</span>. <i>Cell Biochem. Funct.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1002/cbf.3369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1002%2Fcbf.3369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=30604499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=21-30&author=Y.+Zouauthor=D.+Maauthor=Y.+Wang&title=The+PROTAC+Technology+in+Drug+Development&doi=10.1002%2Fcbf.3369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The PROTAC technology in drug development</span></div><div class="casAuthors">Zou, Yutian; Ma, Danhui; Wang, Yinyin</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biochemistry and Function</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-30</span>CODEN:
                <span class="NLM_cas:coden">CBFUDH</span>;
        ISSN:<span class="NLM_cas:issn">1099-0844</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Currently, a new technol. termed PROTAC, proteolysis targeting chimera, has been developed for inducing the protein degrdn. by a targeting mol.  This technol. takes advantage of a moiety of targeted protein and a moiety of recognizing E3 ubiquitin ligase and produces a hybrid mol. to specifically knock down a targeted protein.  During the first decade, three pedigreed groups worked on the development of this technol.  To date, this technol. has been extended by different groups, aiming to develop new drugs against different diseases including cancers.  This review summarizes the contributions of the groups for the development of PROTAC.  Significance of the study : This review summarized the development of the PROTAC technol. for readers and also presented the author's opinions on the application of the technol. in tumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprT1CUHOqrfbVg90H21EOLACvtfcHk0liemPekKzkhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygsQ%253D%253D&md5=2a7417e2c385d01f7eaef52a72c7ea52</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcbf.3369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbf.3369%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DThe%2520PROTAC%2520Technology%2520in%2520Drug%2520Development%26jtitle%3DCell%2520Biochem.%2520Funct.%26date%3D2019%26volume%3D37%26spage%3D21%26epage%3D30%26doi%3D10.1002%2Fcbf.3369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced Protein Degradation: an Emerging Drug Discovery Paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+Protein+Degradation%3A+an+Emerging+Drug+Discovery+Paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0liemPekKzkhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520Protein%2520Degradation%253A%2520an%2520Emerging%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis Targeting Chimeras (PROTACs) in ’Beyond Rule-of-Five’ Chemical Space: Recent Progress and Future Challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+Targeting+Chimeras+%28PROTACs%29+in+%E2%80%99Beyond+Rule-of-Five%E2%80%99+Chemical+Space%3A+Recent+Progress+and+Future+Challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0liemPekKzkhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2599Beyond%2520Rule-of-Five%25E2%2580%2599%2520Chemical%2520Space%253A%2520Recent%2520Progress%2520and%2520Future%2520Challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Misale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scala, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janakiraman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liska, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valtorta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiavo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buscarino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siravegna, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bencardino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cercek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartore-Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gambacorta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallicchio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakiani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boscaro, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medico, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Nicolantonio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span> <span> </span><span class="NLM_article-title">Emergence of KRAS Mutations and Acquired Resistance to Anti-EGFR Therapy in Colorectal Cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>486</i></span>,  <span class="NLM_fpage">532</span>– <span class="NLM_lpage">536</span>, <span class="refDoi"> DOI: 10.1038/nature11156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnature11156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=22722830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVSrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=532-536&author=S.+Misaleauthor=R.+Yaegerauthor=S.+Hoborauthor=E.+Scalaauthor=M.+Janakiramanauthor=D.+Liskaauthor=E.+Valtortaauthor=R.+Schiavoauthor=M.+Buscarinoauthor=G.+Siravegnaauthor=K.+Bencardinoauthor=A.+Cercekauthor=C.+T.+Chenauthor=S.+Veroneseauthor=C.+Zanonauthor=A.+Sartore-Bianchiauthor=M.+Gambacortaauthor=M.+Gallicchioauthor=E.+Vakianiauthor=V.+Boscaroauthor=E.+Medicoauthor=M.+Weiserauthor=S.+Sienaauthor=F.+Di+Nicolantonioauthor=D.+Solitauthor=A.+Bardelli&title=Emergence+of+KRAS+Mutations+and+Acquired+Resistance+to+Anti-EGFR+Therapy+in+Colorectal+Cancer&doi=10.1038%2Fnature11156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of KRAS mutations and acquired resistance to anti-EGFR therapy in colorectal cancer</span></div><div class="casAuthors">Misale, Sandra; Yaeger, Rona; Hobor, Sebastijan; Scala, Elisa; Janakiraman, Manickam; Liska, David; Valtorta, Emanuele; Schiavo, Roberta; Buscarino, Michela; Siravegna, Giulia; Bencardino, Katia; Cercek, Andrea; Chen, Chin-Tung; Veronese, Silvio; Zanon, Carlo; Sartore-Bianchi, Andrea; Gambacorta, Marcello; Gallicchio, Margherita; Vakiani, Efsevia; Boscaro, Valentina; Medico, Enzo; Weiser, Martin; Siena, Salvatore; Di Nicolantonio, Federica; Solit, David; Bardelli, Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">486</span>
        (<span class="NLM_cas:issue">7404</span>),
    <span class="NLM_cas:pages">532-536</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A main limitation of therapies that selectively target kinase signalling pathways is the emergence of secondary drug resistance.  Cetuximab, a monoclonal antibody that binds the extracellular domain of epidermal growth factor receptor (EGFR), is effective in a subset of KRAS wild-type metastatic colorectal cancers.  After an initial response, secondary resistance invariably ensues, thereby limiting the clin. benefit of this drug.  The mol. bases of secondary resistance to cetuximab in colorectal cancer are poorly understood.  Here we show that mol. alterations (in most instances point mutations) of KRAS are causally assocd. with the onset of acquired resistance to anti-EGFR treatment in colorectal cancers.  Expression of mutant KRAS under the control of its endogenous gene promoter was sufficient to confer cetuximab resistance, but resistant cells remained sensitive to combinatorial inhibition of EGFR and mitogen-activated protein-kinase kinase (MEK).  Anal. of metastases from patients who developed resistance to cetuximab or panitumumab showed the emergence of KRAS amplification in one sample and acquisition of secondary KRAS mutations in 60% (6 out of 10) of the cases.  KRAS mutant alleles were detectable in the blood of cetuximab-treated patients as early as 10 mo before radiog. documentation of disease progression.  In summary, the results identify KRAS mutations as frequent drivers of acquired resistance to cetuximab in colorectal cancers, indicate that the emergence of KRAS mutant clones can be detected non-invasively months before radiog. progression and suggest early initiation of a MEK inhibitor as a rational strategy for delaying or reversing drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6h0UDIrrnbLVg90H21EOLACvtfcHk0lhd7gcL6jU2sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsVSrsb4%253D&md5=8f06d4203e06a40ae8e5157d54a432e7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature11156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11156%26sid%3Dliteratum%253Aachs%26aulast%3DMisale%26aufirst%3DS.%26aulast%3DYaeger%26aufirst%3DR.%26aulast%3DHobor%26aufirst%3DS.%26aulast%3DScala%26aufirst%3DE.%26aulast%3DJanakiraman%26aufirst%3DM.%26aulast%3DLiska%26aufirst%3DD.%26aulast%3DValtorta%26aufirst%3DE.%26aulast%3DSchiavo%26aufirst%3DR.%26aulast%3DBuscarino%26aufirst%3DM.%26aulast%3DSiravegna%26aufirst%3DG.%26aulast%3DBencardino%26aufirst%3DK.%26aulast%3DCercek%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DC.%2BT.%26aulast%3DVeronese%26aufirst%3DS.%26aulast%3DZanon%26aufirst%3DC.%26aulast%3DSartore-Bianchi%26aufirst%3DA.%26aulast%3DGambacorta%26aufirst%3DM.%26aulast%3DGallicchio%26aufirst%3DM.%26aulast%3DVakiani%26aufirst%3DE.%26aulast%3DBoscaro%26aufirst%3DV.%26aulast%3DMedico%26aufirst%3DE.%26aulast%3DWeiser%26aufirst%3DM.%26aulast%3DSiena%26aufirst%3DS.%26aulast%3DDi%2BNicolantonio%26aufirst%3DF.%26aulast%3DSolit%26aufirst%3DD.%26aulast%3DBardelli%26aufirst%3DA.%26atitle%3DEmergence%2520of%2520KRAS%2520Mutations%2520and%2520Acquired%2520Resistance%2520to%2520Anti-EGFR%2520Therapy%2520in%2520Colorectal%2520Cancer%26jtitle%3DNature%26date%3D2012%26volume%3D486%26spage%3D532%26epage%3D536%26doi%3D10.1038%2Fnature11156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eide, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilertsen, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielsen, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eknaes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hektoen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lothe, R. A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic and Genetic Features of 24 Colon Cancer Cell Lines</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">e71</span>, <span class="refDoi"> DOI: 10.1038/oncsis.2013.35</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Foncsis.2013.35" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=24042735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCmsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e71&author=D.+Ahmedauthor=P.+W.+Eideauthor=I.+A.+Eilertsenauthor=S.+A.+Danielsenauthor=M.+Eknaesauthor=M.+Hektoenauthor=G.+E.+Lindauthor=R.+A.+Lothe&title=Epigenetic+and+Genetic+Features+of+24+Colon+Cancer+Cell+Lines&doi=10.1038%2Foncsis.2013.35"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic and genetic features of 24 colon cancer cell lines</span></div><div class="casAuthors">Ahmed, D.; Eide, P. W.; Eilertsen, I. A.; Danielsen, S. A.; Eknaes, M.; Hektoen, M.; Lind, G. E.; Lothe, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">Sept.</span>),
    <span class="NLM_cas:pages">e71</span>CODEN:
                <span class="NLM_cas:coden">ONCOCT</span>;
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cell lines are invaluable biomedical research tools, and recent literature has emphasized the importance of genotype authentication and characterization.  In the present study, 24 out of 27 cell line identities were confirmed by short tandem repeat profiling.  The mol. phenotypes of the 24 colon cancer cell lines were examd., and microsatellite instability (MSI) and CpG island methylator phenotype (CIMP) were detd., using the Bethesda panel mononucleotide repeat loci and two epimarker panels, resp.  Furthermore, the BRAF, KRAS and PIK3CA oncogenes were analyzed for mutations in known hotspots, while the entire coding sequences of the PTEN and TP53 tumor suppressors were investigated.  Nine cell lines showed MSI.  Thirteen and nine cell lines were found to be CIMP pos., using the Issa panel and the Weisenberger et al. panel, resp.  The latter was found to be superior for CIMP classification of colon cancer cell lines.  Seventeen cell lines harbored disrupting TP53 mutations.  Altogether, 20/24 cell lines had the mitogen-activated protein kinase pathway activating mutually exclusive KRAS or BRAF mutations.  PIK3CA and PTEN mutations leading to hyperactivation of the phosphoinositide 3-kinase/AKT pathway were obsd. in 13/24 cell lines.  Interestingly, in four cell lines there were no mutations in neither BRAF, KRAS, PIK3CA nor in PTEN.  In conclusion, this study presents mol. features of a large no. of colon cancer cell lines to aid the selection of suitable in vitro models for descriptive and functional research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrowLQizkeM57Vg90H21EOLACvtfcHk0lhd7gcL6jU2sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCmsrzL&md5=397a60bad73ca436da303de020df186a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Foncsis.2013.35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Foncsis.2013.35%26sid%3Dliteratum%253Aachs%26aulast%3DAhmed%26aufirst%3DD.%26aulast%3DEide%26aufirst%3DP.%2BW.%26aulast%3DEilertsen%26aufirst%3DI.%2BA.%26aulast%3DDanielsen%26aufirst%3DS.%2BA.%26aulast%3DEknaes%26aufirst%3DM.%26aulast%3DHektoen%26aufirst%3DM.%26aulast%3DLind%26aufirst%3DG.%2BE.%26aulast%3DLothe%26aufirst%3DR.%2BA.%26atitle%3DEpigenetic%2520and%2520Genetic%2520Features%2520of%252024%2520Colon%2520Cancer%2520Cell%2520Lines%26jtitle%3DOncogenesis%26date%3D2013%26volume%3D2%26spage%3De71%26doi%3D10.1038%2Foncsis.2013.35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirasawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasazuki, T.</span></span> <span> </span><span class="NLM_article-title">Altered Growth of Human Colon Cancer Cell Lines Disrupted at Activated Ki-ras</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>260</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1126/science.8465203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1126%2Fscience.8465203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=8465203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADyaK3sXisFagurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=260&publication_year=1993&pages=85-88&author=S.+Shirasawaauthor=M.+Furuseauthor=N.+Yokoyamaauthor=T.+Sasazuki&title=Altered+Growth+of+Human+Colon+Cancer+Cell+Lines+Disrupted+at+Activated+Ki-ras&doi=10.1126%2Fscience.8465203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Altered growth of human colon cancer cell lines disrupted at activated Ki-ras</span></div><div class="casAuthors">Shirasawa, Senji; Furuse, Masanori; Yokoyama, Nobuhiko; Sasazuki, Takehiko</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">260</span>
        (<span class="NLM_cas:issue">5104</span>),
    <span class="NLM_cas:pages">85-8</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Point mutations that activate the Ki-ras proto-oncogene are present in about 50 percent of human colorectal tumors.  To study the functional significance of these mutations, the activated Ki-ras genes in two human colon carcinoma cell lines, DLD-1 and HCT116, were disrupted by homologous recombination.  Compared with parental cells, cells disrupted at the activated Ki-ras gene were morphol. altered, lost the capacity for anchorage-independent growth, grew more slowly both in vitro and in nude mice, and showed reduced expression of c-myc.  Thus, the activated ki-ras gene plays a key role in colorectal tumorigenesis through altered cell differentiation and cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM2FBzJqjRorVg90H21EOLACvtfcHk0lhd7gcL6jU2sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisFagurw%253D&md5=06dd636f868a5354676d483aa6d96bd2</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.8465203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8465203%26sid%3Dliteratum%253Aachs%26aulast%3DShirasawa%26aufirst%3DS.%26aulast%3DFuruse%26aufirst%3DM.%26aulast%3DYokoyama%26aufirst%3DN.%26aulast%3DSasazuki%26aufirst%3DT.%26atitle%3DAltered%2520Growth%2520of%2520Human%2520Colon%2520Cancer%2520Cell%2520Lines%2520Disrupted%2520at%2520Activated%2520Ki-ras%26jtitle%3DScience%26date%3D1993%26volume%3D260%26spage%3D85%26epage%3D88%26doi%3D10.1126%2Fscience.8465203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors Simultaneously Targeting Cancer Metabolism and Epigenetics: Discovery of Novel Nicotinamide Phosphoribosyltransferase (NAMPT) and Histone Deacetylase (HDAC) Dual Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7965</span>– <span class="NLM_lpage">7983</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00467</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00467" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKhurrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7965-7983&author=G.+Dongauthor=W.+Chenauthor=X.+Wangauthor=X.+Yangauthor=T.+Xuauthor=P.+Wangauthor=W.+Zhangauthor=Y.+Raoauthor=C.+Miaoauthor=C.+Sheng&title=Small+Molecule+Inhibitors+Simultaneously+Targeting+Cancer+Metabolism+and+Epigenetics%3A+Discovery+of+Novel+Nicotinamide+Phosphoribosyltransferase+%28NAMPT%29+and+Histone+Deacetylase+%28HDAC%29+Dual+Inhibitors&doi=10.1021%2Facs.jmedchem.7b00467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors Simultaneously Targeting Cancer Metabolism and Epigenetics: Discovery of Novel Nicotinamide Phosphoribosyltransferase (NAMPT) and Histone Deacetylase (HDAC) Dual Inhibitors</span></div><div class="casAuthors">Dong, Guoqiang; Chen, Wei; Wang, Xia; Yang, Xinglin; Xu, Tianying; Wang, Pei; Zhang, Wannian; Rao, Yu; Miao, Chaoyu; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7965-7983</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cancer metab. and epigenetics are among the most intensely pursued research areas in anticancer drug discovery.  Here we report the first small mols. that simultaneously inhibit nicotinamide phosphoribosyltransferase (NAMPT)and histone deacetylase (HDAC), two important targets of cancer metab. and epigenetics, resp.  Through iterative structure-based drug design, chem. synthesis, and biol. assays, a highly potent dual NAMPT and HDAC inhibitor was successfully identified.  Compd. 35 possessed excellent and balanced activities against both NAMPT (IC50 = 31 nM) and HDAC1 (IC50 = 55 nM).  It could effectively induce cell apoptosis and autophagy and ultimately led to cell death.  Importantly, compd. 35 showed excellent in vivo antitumor efficacy in the HCT116 xenograft model.  This proof-of-concept study demonstrates the feasibility of discovering an inhibitor targeting cancer metab. and epigenetics and provides an efficient strategy for multitarget antitumor drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9UvfGattu1rVg90H21EOLACvtfcHk0lhfAoIbwBTHMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKhurrO&md5=65df0b485cbbc169886d34ae4421f814</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00467%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DC.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520Simultaneously%2520Targeting%2520Cancer%2520Metabolism%2520and%2520Epigenetics%253A%2520Discovery%2520of%2520Novel%2520Nicotinamide%2520Phosphoribosyltransferase%2520%2528NAMPT%2529%2520and%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Dual%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7965%26epage%3D7983%26doi%3D10.1021%2Facs.jmedchem.7b00467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 6 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kun  Chen</span>, <span class="hlFld-ContribAuthor ">Yalei  Zhang</span>, <span class="hlFld-ContribAuthor ">Ling  Qian</span>, <span class="hlFld-ContribAuthor ">Peng  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging strategies to target RAS signaling in human cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-021-01127-w" title="DOI URL">https://doi.org/10.1186/s13045-021-01127-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-021-01127-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-021-01127-w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DEmerging%252Bstrategies%252Bto%252Btarget%252BRAS%252Bsignaling%252Bin%252Bhuman%252Bcancer%252Btherapy%26aulast%3DChen%26aufirst%3DKun%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Long  Chen</span>, <span class="hlFld-ContribAuthor ">Jing  Zhang</span>, <span class="hlFld-ContribAuthor ">Xinjing  Wang</span>, <span class="hlFld-ContribAuthor ">Yu  Li</span>, <span class="hlFld-ContribAuthor ">Lu  Zhou</span>, <span class="hlFld-ContribAuthor ">Xiongxiong  Lu</span>, <span class="hlFld-ContribAuthor ">Guoqiang  Dong</span>, <span class="hlFld-ContribAuthor ">Chunquan  Sheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel KRAS‒PDEδ inhibitors with potent activity in patient-derived human pancreatic tumor xenograft models. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>546 </em><a href="https://doi.org/10.1016/j.apsb.2021.07.009" title="DOI URL">https://doi.org/10.1016/j.apsb.2021.07.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2021.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2021.07.009%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Bnovel%252BKRAS%2525E2%252580%252592PDE%2525CE%2525B4%252Binhibitors%252Bwith%252Bpotent%252Bactivity%252Bin%252Bpatient-derived%252Bhuman%252Bpancreatic%252Btumor%252Bxenograft%252Bmodels%26aulast%3DChen%26aufirst%3DLong%26date%3D2021%26volume%3D546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vasilii V.  Pelipko</span>, <span class="hlFld-ContribAuthor ">Kirill A.  Gomonov</span>. </span><span class="cited-content_cbyCitation_article-title">Formation of five- and six-membered nitrogen-containing heterocycles on the basis of hydrazones derived from α-dicarbonyl compounds (microreview). </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2021,</strong> <em>57 </em>
                                    (6)
                                     , 624-626. <a href="https://doi.org/10.1007/s10593-021-02958-8" title="DOI URL">https://doi.org/10.1007/s10593-021-02958-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-021-02958-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-021-02958-8%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DFormation%252Bof%252Bfive-%252Band%252Bsix-membered%252Bnitrogen-containing%252Bheterocycles%252Bon%252Bthe%252Bbasis%252Bof%252Bhydrazones%252Bderived%252Bfrom%252B%2525CE%2525B1-dicarbonyl%252Bcompounds%252B%252528microreview%252529%26aulast%3DPelipko%26aufirst%3DVasilii%2BV.%26date%3D2021%26date%3D2021%26volume%3D57%26issue%3D6%26spage%3D624%26epage%3D626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoshun  Luo</span>, <span class="hlFld-ContribAuthor ">Zhenbang  Li</span>, <span class="hlFld-ContribAuthor ">Xin  Lin</span>, <span class="hlFld-ContribAuthor ">Xinyu  Li</span>, <span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Kun  Xi</span>, <span class="hlFld-ContribAuthor ">Maoxu  Xiao</span>, <span class="hlFld-ContribAuthor ">Hanlin  Wei</span>, <span class="hlFld-ContribAuthor ">Lizhe  Zhu</span>, <span class="hlFld-ContribAuthor ">Hua  Xiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of an orally active VHL-recruiting PROTAC that achieves robust HMGCR degradation and potent hypolipidemic activity in vivo. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (5)
                                     , 1300-1314. <a href="https://doi.org/10.1016/j.apsb.2020.11.001" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.11.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.11.001%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Ban%252Borally%252Bactive%252BVHL-recruiting%252BPROTAC%252Bthat%252Bachieves%252Brobust%252BHMGCR%252Bdegradation%252Band%252Bpotent%252Bhypolipidemic%252Bactivity%252Bin%2525C2%2525A0vivo%26aulast%3DLuo%26aufirst%3DGuoshun%26date%3D2021%26volume%3D11%26issue%3D5%26spage%3D1300%26epage%3D1314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruth  Nussinov</span>, <span class="hlFld-ContribAuthor ">Hyunbum  Jang</span>, <span class="hlFld-ContribAuthor ">Attila  Gursoy</span>, <span class="hlFld-ContribAuthor ">Ozlem  Keskin</span>, <span class="hlFld-ContribAuthor ">Vadim  Gaponenko</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Nonfunctional Ras. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (2)
                                     , 121-133. <a href="https://doi.org/10.1016/j.chembiol.2020.12.012" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.12.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.12.012%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DInhibition%252Bof%252BNonfunctional%252BRas%26aulast%3DNussinov%26aufirst%3DRuth%26date%3D2021%26volume%3D28%26issue%3D2%26spage%3D121%26epage%3D133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinrui  Yang</span>, <span class="hlFld-ContribAuthor ">He  Yin</span>, <span class="hlFld-ContribAuthor ">Richard D.  Kim</span>, <span class="hlFld-ContribAuthor ">Jason B.  Fleming</span>, <span class="hlFld-ContribAuthor ">Hao  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">Preclinical and Clinical Advances of Targeted Protein Degradation as a Novel Cancer Therapeutic Strategy: An Oncologist Perspective. </span><span class="cited-content_cbyCitation_journal-name">Targeted Oncology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (1)
                                     , 1-12. <a href="https://doi.org/10.1007/s11523-020-00782-2" title="DOI URL">https://doi.org/10.1007/s11523-020-00782-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11523-020-00782-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11523-020-00782-2%26sid%3Dliteratum%253Aachs%26jtitle%3DTargeted%2520Oncology%26atitle%3DPreclinical%252Band%252BClinical%252BAdvances%252Bof%252BTargeted%252BProtein%252BDegradation%252Bas%252Ba%252BNovel%252BCancer%252BTherapeutic%252BStrategy%25253A%252BAn%252BOncologist%252BPerspective%26aulast%3DYang%26aufirst%3DXinrui%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D1%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of representative KRAS-PDEδ inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Limitations of PDEδ inhibitors and PROTAC strategy for efficient PDEδ targeting: (A) release of high affinity inhibitors from PDEδ by Arl2; (B) illustration of the catalytic cycle of the ubiquitination and degradation of PDEδ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of PDEδ PROTACs: (A) predicted binding model of compound <b>2</b> with PDEδ (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E80">5E80</a><a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a>). (B) Structure of CRBN ligand pomalidomide. (C) Design strategy of PDEδ degraders.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Target Compounds <b>17a</b>–<b>h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaNO<sub>2</sub>, 6 M HCl, 0 °C; (b) ethyl 2-chloro-3-oxobutanoate, NaOAc, EtOH/H<sub>2</sub>O = 7:1, 0 °C, 4 h, 70%; (c) pentane-2,4-dione, NaOEt, EtOH, rt, 16 h, 93%; (d) hydrazine hydrate, EtOH, 110 °C, 6 h, 78%; (e) methyl 4-bromobutanoate, NaH, DMF, 0 °C, 4 h, 80%; (f) LiOH, THF/MeOH/H<sub>2</sub>O = 3:2:1, 1 h, 81%; (g) NaOAc, AcOH, reflux, 12 h, 90%; (h) DIPEA, DMF, 90 °C, 12 h, 40–50%; (i) DCM, TFA, rt, 0.5 h; (j) <b>12</b>, HATU, DIPEA, DMF, rt, 2 h, 40–50%.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route of Target Compounds <b>17i</b>–<b>j</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, CH<sub>3</sub>I, DMF, 0 °C, 4 h, 80%; (b) DIPEA, DMF, 90 °C, 12 h, 40–50%; (c) DCM, TFA, rt, 0.5 h; (d) NaOAc, AcOH, reflux, 12 h, 90%; (e) compound <b>12</b>, HATU, DIPEA, DMF, rt, 12 h, 40–50%.</p></p></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Screen and characterization of potent PDEδ degraders. (A) PDEδ levels in SW480 cells upon 24 h drug treatment at 10 μM. (B, D) PDEδ levels in SW480 cells treated with <b>2</b> or <b>17f</b> at various concentrations for 24 h. (C) PDEδ levels in SW480 cells treated with <b>17f</b> at 10 μM for indicated time points. (E) SW480 cells were pretreated with the negative control <b>17i</b> or CRBN ligand <b>6</b> or PDEδ inhibitor <b>2</b> or proteasome inhibitor MG-132 followed by treatment with DMSO or PDEδ degrader <b>17f</b>. (F) Cell growth inhibition activity of the PDEδ inhibitor <b>2</b>, negative control <b>17i</b>, and PDEδ degrader <b>17f</b> in the absence or presence of <b>6</b> in SW480 cells. (G) IKZF1/3 levels in SW480 cells treated with <b>6</b> or <b>17f</b> at various concentrations for 24 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>17f</b> inhibits RAS signaling. Shown are hosphorylation levels of Erk and Akt in SW480 cells stimulated with EGF (125 ng/mL, 5 min). From top to bottom: phosphorylated Akt on S473 (p-Akt), total level of Akt1 (t-Akt1), phosphorylated Erk on Thr202 and Tyr204 (p-Erk), total level of Erk (t-Erk), and loading control (GAPDH). ***<i>P</i> value of <0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>17f</b> inhibits RAS signaling and initiates the apoptosis of KRAS mutant SW480 cells: (A) flow cytometry analysis of the apoptosis induction by <b>17f</b> in SW480 cells; (B) immunostaining of SW480 with anti-PDEδ (green) and antipan-RAS (red) after treatment with compound <b>17f</b> at 10 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/medium/jm0c00929_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. In vivo efficacy of compound <b>17f</b>. (A) Growth curve of implanted SW480 xenograft in nude mice. Data are expressed as the mean ± standard deviation: not significant (n.s.), <i>P</i> value of >0.05; *<i>P</i> value of <0.05. (B) Body weight loss in SW480 xenograft experiments. (C) Immunofluorescence analysis of PDEδ, RAS, and p-Akt protein expression in transplanted nude mice. (D) PK parameters of compound <b>17f</b> in rats. Data are expressed as the mean ± standard deviation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00929/20200716/images/large/jm0c00929_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00929&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30842" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30842" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 37 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simanshu, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissley, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, F.</span></span> <span> </span><span class="NLM_article-title">RAS Proteins and Their Regulators in Human Disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.cell.2017.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=28666118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=17-33&author=D.+K.+Simanshuauthor=D.+V.+Nissleyauthor=F.+McCormick&title=RAS+Proteins+and+Their+Regulators+in+Human+Disease&doi=10.1016%2Fj.cell.2017.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">RAS Proteins and their regulators in human disease</span></div><div class="casAuthors">Simanshu, Dhirendra K.; Nissley, Dwight V.; McCormick, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-33</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  RAS proteins are binary switches, cycling between ON and OFF states during signal transduction.  These switches are normally tightly controlled, but in RAS-related diseases, such as cancer, RASopathies, and many psychiatric disorders, mutations in the RAS genes or their regulators render RAS proteins persistently active.  The structural basis of the switch and many of the pathways that RAS controls are well known, but the precise mechanisms by which RAS proteins function are less clear.  All RAS biol. occurs in membranes: a precise understanding of RAS' interaction with membranes is essential to understand RAS action and to intervene in RAS-driven diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoH9cHufj4FLVg90H21EOLACvtfcHk0lgO6EcYOOs2GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsLfP&md5=638a0c0efac88f2873c352715e332a92</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DSimanshu%26aufirst%3DD.%2BK.%26aulast%3DNissley%26aufirst%3DD.%2BV.%26aulast%3DMcCormick%26aufirst%3DF.%26atitle%3DRAS%2520Proteins%2520and%2520Their%2520Regulators%2520in%2520Human%2520Disease%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D17%26epage%3D33%26doi%3D10.1016%2Fj.cell.2017.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratcioglu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Ras Conformational Ensembles, Allostery, and Signaling</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6607</span>– <span class="NLM_lpage">6665</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00542</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00542" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6607-6665&author=S.+Luauthor=H.+Jangauthor=S.+Muratciogluauthor=A.+Gursoyauthor=O.+Keskinauthor=R.+Nussinovauthor=J.+Zhang&title=Ras+Conformational+Ensembles%2C+Allostery%2C+and+Signaling&doi=10.1021%2Facs.chemrev.5b00542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Ras Conformational Ensembles, Allostery, and Signaling</span></div><div class="casAuthors">Lu, Shaoyong; Jang, Hyunbum; Muratcioglu, Serena; Gursoy, Attila; Keskin, Ozlem; Nussinov, Ruth; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6607-6665</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ras proteins are classical members of small GTPases that function as mol. switches by alternating between inactive GDP-bound and active GTP-bound states.  Ras activation is regulated by guanine nucleotide exchange factors that catalyze the exchange of GDP by GTP, and inactivation is terminated by GTPase-activating proteins that accelerate the intrinsic GTP hydrolysis rate by orders of magnitude.  In this review, we focus on data that have accumulated over the past few years pertaining to the conformational ensembles and the allosteric regulation of Ras proteins and their interpretation from our conformational landscape standpoint.  The Ras ensemble embodies all states, including the ligand-bound conformations, the activated (or inactivated) allosteric modulated states, post-translationally modified states, mutational states, transition states, and nonfunctional states serving as a reservoir for emerging functions.  The ensemble is shifted by distinct mutational events, cofactors, post-translational modifications, and different membrane compns.  A better understanding of Ras biol. can contribute to therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY2oezVaRCvbVg90H21EOLACvtfcHk0lgO6EcYOOs2GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhu74%253D&md5=7600b971015bd3cf635ca1b4299dc7d5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00542%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DMuratcioglu%26aufirst%3DS.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DRas%2520Conformational%2520Ensembles%252C%2520Allostery%252C%2520and%2520Signaling%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6607%26epage%3D6665%26doi%3D10.1021%2Facs.chemrev.5b00542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forbes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bindal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kok, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beare, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teague, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futreal, P. A.</span></span> <span> </span><span class="NLM_article-title">COSMIC: Mining Complete Cancer Genomes in the Catalogue of Somatic Mutations in Cancer</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">D945</span>– <span class="NLM_lpage">950</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1093%2Fnar%2Fgkq929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=20952405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivF2msbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D945-950&author=S.+A.+Forbesauthor=N.+Bindalauthor=S.+Bamfordauthor=C.+Coleauthor=C.+Y.+Kokauthor=D.+Beareauthor=M.+Jiaauthor=R.+Shepherdauthor=K.+Leungauthor=A.+Menziesauthor=J.+W.+Teagueauthor=P.+J.+Campbellauthor=M.+R.+Strattonauthor=P.+A.+Futreal&title=COSMIC%3A+Mining+Complete+Cancer+Genomes+in+the+Catalogue+of+Somatic+Mutations+in+Cancer&doi=10.1093%2Fnar%2Fgkq929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">COSMIC: mining complete cancer genomes in the catalogue of somatic mutations in cancer</span></div><div class="casAuthors">Forbes, Simon A.; Bindal, Nidhi; Bamford, Sally; Cole, Charlotte; Kok, Chai Yin; Beare, David; Jia, Mingming; Shepherd, Rebecca; Leung, Kenric; Menzies, Andrew; Teague, Jon W.; Campbell, Peter J.; Stratton, Michael R.; Futreal, P. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">D945-D950</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">COSMIC (http://www.sanger.ac.uk/cosmic) curates comprehensive information on somatic mutations in human cancer.  Release v48 (July 2010) describes over 136 000 coding mutations in almost 542 000 tumor samples; of the 18 490 genes documented, 4803 (26%) have one or more mutations.  Full scientific literature curations are available on 83 major cancer genes and 49 fusion gene pairs (19 new cancer genes and 30 new fusion pairs this year) and this no. is continually increasing.  Key amongst these is TP53, now available through a collaboration with the IARC p53 database.  In addn. to data from the Cancer Genome Project (CGP) at the Sanger Institute, UK, and The Cancer Genome Atlas project (TCGA), large systematic screens are also now curated.  Major website upgrades now make these data much more mineable, with many new selection filters and graphics.  A Biomart is now available allowing more automated data mining and integration with other biol. databases.  Annotation of genomic features has become a significant focus; COSMIC has begun curating full-genome resequencing expts., developing new web pages, export formats and graphics styles.  With all genomic information recently updated to GRCh37, COSMIC integrates many diverse types of mutation information and is making much closer links with Ensembl and other data resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpalgzfv6Y9s7Vg90H21EOLACvtfcHk0li5TmhUbp_Pdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivF2msbo%253D&md5=3456c7ae37b9f81da68a624a09d0d524</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq929%26sid%3Dliteratum%253Aachs%26aulast%3DForbes%26aufirst%3DS.%2BA.%26aulast%3DBindal%26aufirst%3DN.%26aulast%3DBamford%26aufirst%3DS.%26aulast%3DCole%26aufirst%3DC.%26aulast%3DKok%26aufirst%3DC.%2BY.%26aulast%3DBeare%26aufirst%3DD.%26aulast%3DJia%26aufirst%3DM.%26aulast%3DShepherd%26aufirst%3DR.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DMenzies%26aufirst%3DA.%26aulast%3DTeague%26aufirst%3DJ.%2BW.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26atitle%3DCOSMIC%253A%2520Mining%2520Complete%2520Cancer%2520Genomes%2520in%2520the%2520Catalogue%2520of%2520Somatic%2520Mutations%2520in%2520Cancer%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD945%26epage%3D950%26doi%3D10.1093%2Fnar%2Fgkq929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, K. L.</span></span> <span> </span><span class="NLM_article-title">RAS Isoforms and Mutations in Cancer at a Glance</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">1287</span>– <span class="NLM_lpage">1292</span>, <span class="refDoi"> DOI: 10.1242/jcs.182873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1242%2Fjcs.182873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=26985062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKltr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2016&pages=1287-1292&author=G.+A.+Hobbsauthor=C.+J.+Derauthor=K.+L.+Rossman&title=RAS+Isoforms+and+Mutations+in+Cancer+at+a+Glance&doi=10.1242%2Fjcs.182873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">RAS isoforms and mutations in cancer at a glance</span></div><div class="casAuthors">Hobbs, G. Aaron; Der, Channing J.; Rossman, Kent L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1287-1292</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">RAS proteins (KRAS4A, KRAS4B, NRAS and HRAS) function as GDP-GTP-regulated binary on-off switches, which regulate cytoplasmic signaling networks that control diverse normal cellular processes.  Gain-of-function missense mutations in RAS genes are found in ∼ 25% of human cancers, prompting interest in identifying anti-RAS therapeutic strategies for cancer treatment.  However, despite more than three decades of intense effort, no anti-RAS therapies have reached clin. application.  Contributing to this failure has been an underestimation of the complexities of RAS.  First, there is now appreciation that the four human RAS proteins are not functionally identical.  Second, with >130 different missense mutations found in cancer, there is an emerging view that there are mutation-specific consequences on RAS structure, biochem. and biol., and mutation-selective therapeutic strategies are needed.  In this Cell Science at a Glance article and accompanying poster, we provide a snapshot of the differences between RAS isoforms and mutations, as well as the current status of anti-RAS drug-discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNufX89IRJk7Vg90H21EOLACvtfcHk0li5TmhUbp_Pdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKltr7I&md5=bf50d212b900fb437cad2b650a9dc07a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1242%2Fjcs.182873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.182873%26sid%3Dliteratum%253Aachs%26aulast%3DHobbs%26aufirst%3DG.%2BA.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26aulast%3DRossman%26aufirst%3DK.%2BL.%26atitle%3DRAS%2520Isoforms%2520and%2520Mutations%2520in%2520Cancer%2520at%2520a%2520Glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2016%26volume%3D129%26spage%3D1287%26epage%3D1292%26doi%3D10.1242%2Fjcs.182873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karachaliou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magri, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimenez-Capitan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina-Vila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosell, R.</span></span> <span> </span><span class="NLM_article-title">KRAS Mutations in Lung Cancer</span>. <i>Clin. Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/j.cllc.2012.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.cllc.2012.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=23122493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlslWitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=205-214&author=N.+Karachaliouauthor=C.+Mayoauthor=C.+Costaauthor=I.+Magriauthor=A.+Gimenez-Capitanauthor=M.+A.+Molina-Vilaauthor=R.+Rosell&title=KRAS+Mutations+in+Lung+Cancer&doi=10.1016%2Fj.cllc.2012.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">KRAS Mutations in Lung Cancer</span></div><div class="casAuthors">Karachaliou, Niki; Mayo, Clara; Costa, Carlota; Magri, Ignacio; Gimenez-Capitan, Ana; Molina-Vila, Miguel Angel; Rosell, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Lung Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-214</span>CODEN:
                <span class="NLM_cas:coden">CLCLCA</span>;
        ISSN:<span class="NLM_cas:issn">1525-7304</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Epidermal growth factor receptor (EGFR) gene mutations and increased EGFR copy nos. have been assocd. with a favorable response to EGFR tyrosine kinase inhibitors (TKI) in patients with non-small-cell lung cancer (NSCLC), and several markers have been identified that predict response to treatment.  Lung adenocarcinomas also harbor activating mutations in the downstream GTPase, v-Ki-ras2 Kirsten rat sarcoma viral oncogene (KRAS), and mutations in EGFR and KRAS appear to be mutually exclusive.  Even though KRAS mutations were identified in NSCLC tumors more than 20 years ago, we have only just begun to appreciate the clin. value of detg. KRAS tumor status.  Recent studies indicate that patients with mutant KRAS tumors fail to benefit from adjuvant chemotherapy and do not respond to EGFR inhibitors.  There is a clear need for therapies specifically developed for patients with KRAS-mutant NSCLC.  In this review, we summarize the clin. and pathol. characteristics of patients with NSCLC and with KRAS mutations, describe work that explores the predictive and prognostic influence of KRAS mutations, and provide an overview of the "synthetic lethal" interactions and current approaches to targeting KRAS-mutant NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzzE2yYQyuAbVg90H21EOLACvtfcHk0li5TmhUbp_Pdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlslWitbY%253D&md5=99127a23a5dfc3d06eaab4a206dfde45</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cllc.2012.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cllc.2012.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DKarachaliou%26aufirst%3DN.%26aulast%3DMayo%26aufirst%3DC.%26aulast%3DCosta%26aufirst%3DC.%26aulast%3DMagri%26aufirst%3DI.%26aulast%3DGimenez-Capitan%26aufirst%3DA.%26aulast%3DMolina-Vila%26aufirst%3DM.%2BA.%26aulast%3DRosell%26aufirst%3DR.%26atitle%3DKRAS%2520Mutations%2520in%2520Lung%2520Cancer%26jtitle%3DClin.%2520Lung%2520Cancer%26date%3D2013%26volume%3D14%26spage%3D205%26epage%3D214%26doi%3D10.1016%2Fj.cllc.2012.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Drugging RAS: Know the Enemy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1158</span>– <span class="NLM_lpage">1163</span>, <span class="refDoi"> DOI: 10.1126/science.aam7622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1126%2Fscience.aam7622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=28302824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1158-1163&author=B.+Papkeauthor=C.+J.+Der&title=Drugging+RAS%3A+Know+the+Enemy&doi=10.1126%2Fscience.aam7622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging RAS: Know the enemy</span></div><div class="casAuthors">Papke, Bjoern; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1158-1163</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  The three RAS oncogenes make up the most frequently mutated gene family in human cancer.  The well-validated role of mutationally activated RAS genes in driving cancer development and growth has stimulated comprehensive efforts to develop therapeutic strategies to block mutant RAS function for cancer treatment.  Disappointingly, despite more than three decades of research effort, clin. effective anti-RAS therapies have remained elusive, prompting a perception that RAS may be undruggable.  However, with a greater appreciation of the complexities of RAS that thwarted past efforts, and armed with new technologies and directions, the field is experiencing renewed excitement that mutant RAS may finally be conquered.  Here the authors summarize where these efforts stand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhrMhg-D4EM7Vg90H21EOLACvtfcHk0li5TmhUbp_Pdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmtrY%253D&md5=ce9da51019033a624c765ef2507bac2e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7622%26sid%3Dliteratum%253Aachs%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DDrugging%2520RAS%253A%2520Know%2520the%2520Enemy%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1158%26epage%3D1163%26doi%3D10.1126%2Fscience.aam7622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Drugging the Undruggable RAS: Mission Possible?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">828</span>– <span class="NLM_lpage">851</span>, <span class="refDoi"> DOI: 10.1038/nrd4389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnrd4389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=25323927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslKmu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=828-851&author=A.+D.+Coxauthor=S.+W.+Fesikauthor=A.+C.+Kimmelmanauthor=J.+Luoauthor=C.+J.+Der&title=Drugging+the+Undruggable+RAS%3A+Mission+Possible%3F&doi=10.1038%2Fnrd4389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the undruggable RAS: Mission Possible?</span></div><div class="casAuthors">Cox, Adrienne D.; Fesik, Stephen W.; Kimmelman, Alec C.; Luo, Ji; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">828-851</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite more than three decades of intensive effort, no effective pharmacol. inhibitors of the RAS oncoproteins have reached the clinic, prompting the widely held perception that RAS proteins are 'undruggable'.  However, recent data from the lab. and the clinic have renewed our hope for the development of RAS-inhibitory mols.  In this Review, we summarize the progress and the promise of five key approaches.  Firstly, we focus on the prospects of using direct inhibitors of RAS.  Secondly, we address the issue of whether blocking RAS membrane assocn. is a viable approach.  Thirdly, we assess the status of targeting RAS downstream effector signalling, which is arguably the most favorable current approach.  Fourthly, we address whether the search for synthetic lethal interactors of mutant RAS still holds promise.  Finally, RAS-mediated changes in cell metab. have recently been described and we discuss whether these changes could be exploited for new therapeutic directions.  We conclude with perspectives on how addnl. complexities, which are not yet fully understood, may affect each of these approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvWzW1JqxN4LVg90H21EOLACvtfcHk0lh3mXk6oKe1ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslKmu7zE&md5=6c8645293b079023de7d31011ae1a192</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd4389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4389%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DDrugging%2520the%2520Undruggable%2520RAS%253A%2520Mission%2520Possible%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D828%26epage%3D851%26doi%3D10.1038%2Fnrd4389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Bryan, J. P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Targeting of RAS: Recent Success with Direct Inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">503</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2018.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.phrs.2018.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=30366101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVCitbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2019&pages=503-511&author=J.+P.+O%E2%80%99Bryan&title=Pharmacological+Targeting+of+RAS%3A+Recent+Success+with+Direct+Inhibitors&doi=10.1016%2Fj.phrs.2018.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of RAS: Recent success with direct inhibitors</span></div><div class="casAuthors">O'Bryan, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">503-511</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  RAS has long been viewed as undruggable due to its lack of deep pockets for binding of small mol. inhibitors.  However, recent successes in the development of direct RAS inhibitors suggest that the goal of pharmacol. inhibition of RAS in patients may soon be realized.  This review will discuss the role of RAS in cancer, the approaches used to develop direct RAS inhibitors, and highlight recent successes in the development of novel RAS inhibitory compds. that target different aspects of RAS biochem.  In particular, this review will discuss the different properties of RAS that have been targeted by various inhibitors including membrane localization, the different activation states of RAS, effector binding, and nucleotide exchange.  In addn., this review will highlight the recent success with mutation-specific inhibitors that exploit the unique biochem. of the RAS(G12C) mutant.  Although this mutation in KRAS accounts for 11% of all KRAS mutations in cancer, it is the most prominent KRAS mutant in lung cancer suggesting that G12C-specific inhibitors may provide a new approach for treating the subset of lung cancer patients harboring this mutant allele.  Finally, this review will discuss the involvement of dimerization in RAS function and highlight new approaches to inhibit RAS by specifically interfering with RAS:RAS interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94gO4ck3GCbVg90H21EOLACvtfcHk0lh3mXk6oKe1ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVCitbjK&md5=5a62a1fe189f3fc6f3b8d05b1d7c6e0a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2018.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2018.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Bryan%26aufirst%3DJ.%2BP.%26atitle%3DPharmacological%2520Targeting%2520of%2520RAS%253A%2520Recent%2520Success%2520with%2520Direct%2520Inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2019%26volume%3D139%26spage%3D503%26epage%3D511%26doi%3D10.1016%2Fj.phrs.2018.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span> <span> </span><span class="NLM_article-title">Drugging K-Ras(G12C) through Covalent Inhibitors: Mission Possible?</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>202</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2019.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.pharmthera.2019.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=31233765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Ojs77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=202&publication_year=2019&pages=1-17&author=D.+Niauthor=X.+Liauthor=X.+Heauthor=H.+Zhangauthor=J.+Zhangauthor=S.+Lu&title=Drugging+K-Ras%28G12C%29+through+Covalent+Inhibitors%3A+Mission+Possible%3F&doi=10.1016%2Fj.pharmthera.2019.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging K-RasG12C through covalent inhibitors: Mission possible?</span></div><div class="casAuthors">Ni, Duan; Li, Xinyi; He, Xinheng; Zhang, Hao; Zhang, Jian; Lu, Shaoyong</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">202</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-17</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Ras, whose mutants are present in approx. 30% of human tumors, is one of the most important oncogenes.  Drugging Ras is thus regarded as the quest for the Holy Grail in cancer therapeutics development.  Despite more than three decades of efforts, drug discovery targeting Ras constantly fails, rendering Ras undruggable, due to its smooth surface and picomolar affinity towards guanosine substrates.  The most frequently mutated isoform of Ras is K-Ras, accounting for >85% of Ras-driven cancers, and one majority of them is the G12C mutation.  Recent advances in structural biol. shed light on drugging Ras, and one of the cutting-edge breakthroughs is the design of covalent G12C-specific inhibitors targeting the mutated cysteine.  This type of inhibitor can be classified into substrate-competitive orthosteric inhibitors and non-competitive allosteric inhibitors.  They display improved selectivity and enhanced potency due to their G12-specific and irreversible covalent binding nature.  Thus, they represent a new hope for revolutionizing the conventional characterization of Ras as "undruggable" and pave a promising avenue for further drug discovery.  Here, we provide comprehensive structural and medicinal chem. insights into K-Ras covalent inhibitors specific for the G12C mutant.  We first present an in-depth anal. of the conformations of the inhibitor binding pockets.  Then, all the latest covalent ligands selectively inhibiting K-RasG12C are reviewed.  Finally, we examine the current challenges faced by this new class of anti-Ras inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLkKui-Vl_zbVg90H21EOLACvtfcHk0lh3mXk6oKe1ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Ojs77M&md5=57887102a4d4ca32e55f76580fde55a7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2019.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2019.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%26atitle%3DDrugging%2520K-Ras%2528G12C%2529%2520through%2520Covalent%2520Inhibitors%253A%2520Mission%2520Possible%253F%26jtitle%3DPharmacol.%2520Ther.%26date%3D2019%26volume%3D202%26spage%3D1%26epage%3D17%26doi%3D10.1016%2Fj.pharmthera.2019.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span> <span> </span><span class="NLM_article-title">Ras Moves to Stay in Place</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">190</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2015.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.tcb.2015.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=25759176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsFGnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=190-197&author=M.+Schmickauthor=A.+Kraemerauthor=P.+I.+Bastiaens&title=Ras+Moves+to+Stay+in+Place&doi=10.1016%2Fj.tcb.2015.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Ras moves to stay in place</span></div><div class="casAuthors">Schmick, Malte; Kraemer, Astrid; Bastiaens, Philippe I. H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">190-197</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Ras is a major intracellular signaling hub.  This elevated position comes at a precarious cost: a single point mutation can cause aberrant signaling.  The capacity of Ras for signaling is inextricably linked to its enrichment at the plasma membrane (PM).  This PM localization is dynamically maintained by three essential elements: alteration of membrane affinities via lipidation and membrane-interaction motifs; trapping on specific membranes coupled with unidirectional vesicular transport to the PM; and regulation of diffusion via interaction with a solubilization factor.  This system constitutes a cycle that primarily corrects for the entropic equilibration of Ras to all membranes that dils. its signaling capacity.  We illuminate how this reaction-diffusion system maintains an out-of-equil. localization of Ras GTPases and thereby confers signaling functionality to the PM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8MYFhn_IGMLVg90H21EOLACvtfcHk0lgQ_lFDuvylVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsFGnsLc%253D&md5=ebcac88ec427423d2173b53ee51c9043</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2015.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2015.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DSchmick%26aufirst%3DM.%26aulast%3DKraemer%26aufirst%3DA.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26atitle%3DRas%2520Moves%2520to%2520Stay%2520in%2520Place%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2015%26volume%3D25%26spage%3D190%26epage%3D197%26doi%3D10.1016%2Fj.tcb.2015.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bierbaum, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gremer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triola, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span> <span> </span><span class="NLM_article-title">Arl2-GTP and Arl3-GTP Regulate a GDI-like Transport System for Farnesylated Cargo</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">942</span>– <span class="NLM_lpage">949</span>, <span class="refDoi"> DOI: 10.1038/nchembio.686</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnchembio.686" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=22002721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWltb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=942-949&author=S.+A.+Ismailauthor=Y.+X.+Chenauthor=A.+Rusinovaauthor=A.+Chandraauthor=M.+Bierbaumauthor=L.+Gremerauthor=G.+Triolaauthor=H.+Waldmannauthor=P.+I.+Bastiaensauthor=A.+Wittinghofer&title=Arl2-GTP+and+Arl3-GTP+Regulate+a+GDI-like+Transport+System+for+Farnesylated+Cargo&doi=10.1038%2Fnchembio.686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Arl2-GTP and Arl3-GTP regulate a GDI-like transport system for farnesylated cargo</span></div><div class="casAuthors">Ismail, Shehab A.; Chen, Yong-Xiang; Rusinova, Alexandra; Chandra, Anchal; Bierbaum, Martin; Gremer, Lothar; Triola, Gemma; Waldmann, Herbert; Bastiaens, Philippe I. H.; Wittinghofer, Alfred</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">942-949</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lipidated Rho and Rab GTP-binding proteins are transported between membranes in complex with solubilizing factors called 'guanine nucleotide dissocn. inhibitors' (GDIs).  Unloading from GDIs using GDI displacement factors (GDFs) has been proposed but remains mechanistically elusive.  PDEδ is a putative solubilizing factor for several prenylated Ras-subfamily proteins.  Here we report the structure of fully modified farnesylated Rheb-GDP in complex with PDEδ.  The structure explains the nucleotide-independent binding of Rheb to PDEδ and the relaxed specificity of PDEδ.  We demonstrate that the G proteins Arl2 and Arl3 act in a GTP-dependent manner as allosteric release factors for farnesylated cargo.  We thus describe a new transport system for farnesylated G proteins involving a GDI-like mol. and an unequivocal GDF.  Considering the importance of PDEδ for proper Ras and Rheb signaling, this study is instrumental in developing a new target for anticancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrENzFj11Fgq7Vg90H21EOLACvtfcHk0lgQ_lFDuvylVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWltb3F&md5=94dff580269780042ea4e6e5bd03f6a0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.686%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DS.%2BA.%26aulast%3DChen%26aufirst%3DY.%2BX.%26aulast%3DRusinova%26aufirst%3DA.%26aulast%3DChandra%26aufirst%3DA.%26aulast%3DBierbaum%26aufirst%3DM.%26aulast%3DGremer%26aufirst%3DL.%26aulast%3DTriola%26aufirst%3DG.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26aulast%3DWittinghofer%26aufirst%3DA.%26atitle%3DArl2-GTP%2520and%2520Arl3-GTP%2520Regulate%2520a%2520GDI-like%2520Transport%2520System%2520for%2520Farnesylated%2520Cargo%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D942%26epage%3D949%26doi%3D10.1038%2Fnchembio.686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozdemir, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Arl2-Mediated Allosteric Release of Farnesylated KRas4B from Shuttling Factor PDEdelta</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">7503</span>– <span class="NLM_lpage">7513</span>, <span class="refDoi"> DOI: 10.1021/acs.jpcb.8b04347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpcb.8b04347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Cmu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2018&pages=7503-7513&author=E.+S.+Ozdemirauthor=H.+Jangauthor=A.+Gursoyauthor=O.+Keskinauthor=R.+Nussinov&title=Arl2-Mediated+Allosteric+Release+of+Farnesylated+KRas4B+from+Shuttling+Factor+PDEdelta&doi=10.1021%2Facs.jpcb.8b04347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Arl2-Mediated Allosteric Release of Farnesylated KRas4B from Shuttling Factor PDEδ</span></div><div class="casAuthors">Ozdemir, E. Sila; Jang, Hyunbum; Gursoy, Attila; Keskin, Ozlem; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">7503-7513</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proper localization of Ras proteins at the plasma membrane (PM) is crucial for their functions.  To get to the PM, KRas4B, as well as some other Ras family proteins, bind to the cGMP phosphodiesterase δ-subunit (PDEδ) shuttling protein through their farnesylated hypervariable regions (HVRs).  The docking of their farnesyl (and to a lesser extent geranylgeranyl) in PDEδ's hydrophobic pocket stabilizes the interaction.  At the PM, GTP-bound Arf-like protein 2 (Arl2) assists in the release of Ras from PDEδ.  However, exactly how is still unclear.  Here, using all-atom mol. dynamics simulations, we unraveled the detailed mechanism of Arl2-mediated release of KRas4B, the most abundant oncogenic Ras isoform, from PDEδ.  We simulated ternary Arl2-PDEδ-KRas4B HVR complexes and obsd. that Arl2 binding weakened the PDEδ-farnesylated HVR interaction.  Our detailed anal. showed that allosteric changes (involving β6 of PDEδ and addnl. PDEδ residues) compressed the hydrophobic PDEδ pocket and pushed the HVR out.  Mutating PDEδ residues that mediate allosteric changes in PDEδ terminated the release process.  Mutant Ras proteins are enriched in human cancers, with currently no drugs in the clinics.  This mechanistic account may inspire efforts to develop drugs suppressing oncogenic KRas4B release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5Blyv3puJz7Vg90H21EOLACvtfcHk0lgQ_lFDuvylVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cmu7zO&md5=0275a619c9714189483cd816423700bf</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jpcb.8b04347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpcb.8b04347%26sid%3Dliteratum%253Aachs%26aulast%3DOzdemir%26aufirst%3DE.%2BS.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DArl2-Mediated%2520Allosteric%2520Release%2520of%2520Farnesylated%2520KRas4B%2520from%2520Shuttling%2520Factor%2520PDEdelta%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2018%26volume%3D122%26spage%3D7503%26epage%3D7513%26doi%3D10.1021%2Facs.jpcb.8b04347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartak, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovacevic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truxius, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmannek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I. H.</span></span> <span> </span><span class="NLM_article-title">KRAS Localizes to the Plasma Membrane by Spatial Cycles of Solubilization, Trapping and Vesicular Transport</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.02.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.cell.2014.02.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=24725411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmslals7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=459-471&author=M.+Schmickauthor=N.+Vartakauthor=B.+Papkeauthor=M.+Kovacevicauthor=D.+C.+Truxiusauthor=L.+Rossmannekauthor=P.+I.+H.+Bastiaens&title=KRAS+Localizes+to+the+Plasma+Membrane+by+Spatial+Cycles+of+Solubilization%2C+Trapping+and+Vesicular+Transport&doi=10.1016%2Fj.cell.2014.02.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">KRas Localizes to the Plasma Membrane by Spatial Cycles of Solubilization, Trapping and Vesicular Transport</span></div><div class="casAuthors">Schmick, Malte; Vartak, Nachiket; Papke, Bjoern; Kovacevic, Marija; Truxius, Dina C.; Rossmannek, Lisaweta; Bastiaens, Philippe I. H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">459-471</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRas is a major proto-oncogene product whose signaling activity depends on its level of enrichment on the plasma membrane (PM).  This PM localization relies on posttranslational prenylation for membrane affinity, while PM specificity has been attributed to electrostatic interactions between neg. charged phospholipids in the PM and basic amino-acids in the C terminus of KRas.  By measuring kinetic parameters of KRas dynamics in living cells with a cellular-automata-based data-fitting approach in realistic cell-geometries, we show that charge-based specificity is not sufficient to generate PM enrichment in light of the total surface area of endomembranes.  Instead, mislocalized KRas is continuously sequestered from endomembranes by cytosolic PDEδ to be unloaded in an Arl2-dependent manner to perinuclear membranes.  Electrostatic interactions then trap KRas at the recycling endosome (RE), from where vesicular transport restores enrichment on the PM.  This energy driven reaction-diffusion cycle explains how small mol. targeting of PDEδ affects the spatial organization of KRas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnu-qiKkSobVg90H21EOLACvtfcHk0ljPhe53fgHSHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmslals7w%253D&md5=c8e31ee5983495be339785b97a94bac8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.02.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.02.051%26sid%3Dliteratum%253Aachs%26aulast%3DSchmick%26aufirst%3DM.%26aulast%3DVartak%26aufirst%3DN.%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DKovacevic%26aufirst%3DM.%26aulast%3DTruxius%26aufirst%3DD.%2BC.%26aulast%3DRossmannek%26aufirst%3DL.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%2BH.%26atitle%3DKRAS%2520Localizes%2520to%2520the%2520Plasma%2520Membrane%2520by%2520Spatial%2520Cycles%2520of%2520Solubilization%252C%2520Trapping%2520and%2520Vesicular%2520Transport%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D459%26epage%3D471%26doi%3D10.1016%2Fj.cell.2014.02.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murarka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Gago, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovacevic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truxius, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fansa, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinelt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Saabi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span> <span> </span><span class="NLM_article-title">Identification of Pyrazolopyridazinones as PDEdelta Inhibitors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11360</span>, <span class="refDoi"> DOI: 10.1038/ncomms11360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fncomms11360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=27094677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1Kns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=11360&author=B.+Papkeauthor=S.+Murarkaauthor=H.+A.+Vogelauthor=P.+Martin-Gagoauthor=M.+Kovacevicauthor=D.+C.+Truxiusauthor=E.+K.+Fansaauthor=S.+Ismailauthor=G.+Zimmermannauthor=K.+Heineltauthor=C.+Schultz-Fademrechtauthor=A.+Al+Saabiauthor=M.+Baumannauthor=P.+Nussbaumerauthor=A.+Wittinghoferauthor=H.+Waldmannauthor=P.+I.+Bastiaens&title=Identification+of+Pyrazolopyridazinones+as+PDEdelta+Inhibitors&doi=10.1038%2Fncomms11360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of pyrazolopyridazinones as PDEδ inhibitors</span></div><div class="casAuthors">Papke, Bjoern; Murarka, Sandip; Vogel, Holger A.; Martin-Gago, Pablo; Kovacevic, Marija; Truxius, Dina C.; Fansa, Eyad K.; Ismail, Shehab; Zimmermann, Gunther; Heinelt, Kaatje; Schultz-Fademrecht, Carsten; Al Saabi, Alaa; Baumann, Matthias; Nussbaumer, Peter; Wittinghofer, Alfred; Waldmann, Herbert; Bastiaens, Philippe I. H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11360</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The prenyl-binding protein PDEδ is crucial for the plasma membrane localization of prenylated Ras.  Recently, we have reported that the small-mol. Deltarasin binds to the prenyl-binding pocket of PDEδ, and impairs Ras enrichment at the plasma membrane, thereby affecting the proliferation of KRas-dependent human pancreatic ductal adenocarcinoma cell lines.  Here, using structure-based compd. design, we have now identified pyrazolopyridazinones as a novel, unrelated chemotype that binds to the prenyl-binding pocket of PDEδ with high affinity, thereby displacing prenylated Ras proteins in cells.  Our results show that the new PDEδ inhibitor, named Deltazinone 1, is highly selective, exhibits less unspecific cytotoxicity than the previously reported Deltarasin and demonstrates a high correlation with the phenotypic effect of PDEδ knockdown in a set of human pancreatic cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwY6Yus7Koa7Vg90H21EOLACvtfcHk0ljPhe53fgHSHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1Kns7s%253D&md5=7d7d6e83bc2ff4e54fc1132330b9cff3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncomms11360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms11360%26sid%3Dliteratum%253Aachs%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DMurarka%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%2BA.%26aulast%3DMartin-Gago%26aufirst%3DP.%26aulast%3DKovacevic%26aufirst%3DM.%26aulast%3DTruxius%26aufirst%3DD.%2BC.%26aulast%3DFansa%26aufirst%3DE.%2BK.%26aulast%3DIsmail%26aufirst%3DS.%26aulast%3DZimmermann%26aufirst%3DG.%26aulast%3DHeinelt%26aufirst%3DK.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DAl%2BSaabi%26aufirst%3DA.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DNussbaumer%26aufirst%3DP.%26aulast%3DWittinghofer%26aufirst%3DA.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26atitle%3DIdentification%2520of%2520Pyrazolopyridazinones%2520as%2520PDEdelta%2520Inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D11360%26doi%3D10.1038%2Fncomms11360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartak, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triola, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibition of the KRAS-PDEdelta Interaction Impairs Oncogenic KRAS Signalling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>497</i></span>,  <span class="NLM_fpage">638</span>– <span class="NLM_lpage">642</span>, <span class="refDoi"> DOI: 10.1038/nature12205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnature12205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=23698361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotFSgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=497&publication_year=2013&pages=638-642&author=G.+Zimmermannauthor=B.+Papkeauthor=S.+Ismailauthor=N.+Vartakauthor=A.+Chandraauthor=M.+Hoffmannauthor=S.+A.+Hahnauthor=G.+Triolaauthor=A.+Wittinghoferauthor=P.+I.+Bastiaensauthor=H.+Waldmann&title=Small+Molecule+Inhibition+of+the+KRAS-PDEdelta+Interaction+Impairs+Oncogenic+KRAS+Signalling&doi=10.1038%2Fnature12205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of the KRAS-PDEδ interaction impairs oncogenic KRAS signalling</span></div><div class="casAuthors">Zimmermann, Gunther; Papke, Bjoern; Ismail, Shehab; Vartak, Nachiket; Chandra, Anchal; Hoffmann, Maike; Hahn, Stephan A.; Triola, Gemma; Wittinghofer, Alfred; Bastiaens, Philippe I. H.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">497</span>
        (<span class="NLM_cas:issue">7451</span>),
    <span class="NLM_cas:pages">638-642</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The KRAS oncogene product is considered a major target in anticancer drug discovery.  However, direct interference with KRAS signalling has not yet led to clin. useful drugs.  Correct localization and signalling by farnesylated KRAS is regulated by the prenyl-binding protein PDEδ, which sustains the spatial organization of KRAS by facilitating its diffusion in the cytoplasm.  Here we report that interfering with binding of mammalian PDEδ to KRAS by means of small mols. provides a novel opportunity to suppress oncogenic RAS signalling by altering its localization to endomembranes.  Biochem. screening and subsequent structure-based hit optimization yielded inhibitors of the KRAS-PDEδ interaction that selectively bind to the prenyl-binding pocket of PDEδ with nanomolar affinity, inhibit oncogenic RAS signalling and suppress in vitro and in vivo proliferation of human pancreatic ductal adenocarcinoma cells that are dependent on oncogenic KRAS.  Our findings may inspire novel drug discovery efforts aimed at the development of drugs targeting oncogenic RAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4LMNyMIV4LVg90H21EOLACvtfcHk0ljPhe53fgHSHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotFSgtbs%253D&md5=e8dfd498bfd4dafbebf07fee9128e6b0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature12205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12205%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DG.%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DIsmail%26aufirst%3DS.%26aulast%3DVartak%26aufirst%3DN.%26aulast%3DChandra%26aufirst%3DA.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DHahn%26aufirst%3DS.%2BA.%26aulast%3DTriola%26aufirst%3DG.%26aulast%3DWittinghofer%26aufirst%3DA.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DSmall%2520Molecule%2520Inhibition%2520of%2520the%2520KRAS-PDEdelta%2520Interaction%2520Impairs%2520Oncogenic%2520KRAS%2520Signalling%26jtitle%3DNature%26date%3D2013%26volume%3D497%26spage%3D638%26epage%3D642%26doi%3D10.1038%2Fnature12205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Gago, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fansa, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murarka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schurmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastiaens, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">A PDE6delta-KRAS Inhibitor Chemotype with up to Seven H-Bonds and Picomolar Affinity that Prevents Efficient Inhibitor Release by Arl2</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2423</span>– <span class="NLM_lpage">2428</span>, <span class="refDoi"> DOI: 10.1002/anie.201610957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1002%2Fanie.201610957" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsValt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=2423-2428&author=P.+Martin-Gagoauthor=E.+K.+Fansaauthor=C.+H.+Kleinauthor=S.+Murarkaauthor=P.+Janningauthor=M.+Schurmannauthor=M.+Metzauthor=S.+Ismailauthor=C.+Schultz-Fademrechtauthor=M.+Baumannauthor=P.+I.+Bastiaensauthor=A.+Wittinghoferauthor=H.+Waldmann&title=A+PDE6delta-KRAS+Inhibitor+Chemotype+with+up+to+Seven+H-Bonds+and+Picomolar+Affinity+that+Prevents+Efficient+Inhibitor+Release+by+Arl2&doi=10.1002%2Fanie.201610957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A PDE6δ-KRas Inhibitor Chemotype with up to Seven H-Bonds and Picomolar Affinity that Prevents Efficient Inhibitor Release by Arl2</span></div><div class="casAuthors">Martin-Gago, Pablo; Fansa, Eyad K.; Klein, Christian H.; Murarka, Sandip; Janning, Petra; Schuermann, Marc; Metz, Malte; Ismail, Shehab; Schultz-Fademrecht, Carsten; Baumann, Matthias; Bastiaens, Philippe I. H.; Wittinghofer, Alfred; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2423-2428</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Small-mol. inhibition of the interaction between the KRas oncoprotein and the chaperone PDE6δ impairs KRas spatial organization and signaling in cells.  However, despite potent binding in vitro (KD<10 nM), interference with Ras signaling and growth inhibition require 5-20 μm compd. concns.  We demonstrate that these findings can be explained by fast release of high-affinity inhibitors from PDE6δ by the release factor Arl2.  This limitation is overcome by novel highly selective inhibitors that bind to PDE6δ with up to 7 hydrogen bonds, resulting in picomolar affinity.  Their release by Arl2 is greatly decreased, and representative compds. selectively inhibit growth of KRas mutated and -dependent cells with the highest activity recorded yet.  Our findings indicate that very potent inhibitors of the KRas-PDE6δ interaction may impair the growth of tumors driven by oncogenic KRas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGRXyzHR151LVg90H21EOLACvtfcHk0lgnThFouLCseg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsValt7k%253D&md5=2b66846ac0d9806f58b428d83819a7c7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fanie.201610957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201610957%26sid%3Dliteratum%253Aachs%26aulast%3DMartin-Gago%26aufirst%3DP.%26aulast%3DFansa%26aufirst%3DE.%2BK.%26aulast%3DKlein%26aufirst%3DC.%2BH.%26aulast%3DMurarka%26aufirst%3DS.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DSchurmann%26aufirst%3DM.%26aulast%3DMetz%26aufirst%3DM.%26aulast%3DIsmail%26aufirst%3DS.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DBastiaens%26aufirst%3DP.%2BI.%26aulast%3DWittinghofer%26aufirst%3DA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DA%2520PDE6delta-KRAS%2520Inhibitor%2520Chemotype%2520with%2520up%2520to%2520Seven%2520H-Bonds%2520and%2520Picomolar%2520Affinity%2520that%2520Prevents%2520Efficient%2520Inhibitor%2520Release%2520by%2520Arl2%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D2423%26epage%3D2428%26doi%3D10.1002%2Fanie.201610957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Structural Biology-Inspired Discovery of Novel KRAS-PDEdelta Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9400</span>– <span class="NLM_lpage">9406</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01243</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01243" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeiurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9400-9406&author=Y.+Jiangauthor=C.+Zhuangauthor=L.+Chenauthor=J.+Luauthor=G.+Dongauthor=Z.+Miaoauthor=W.+Zhangauthor=J.+Liauthor=C.+Sheng&title=Structural+Biology-Inspired+Discovery+of+Novel+KRAS-PDEdelta+Inhibitors&doi=10.1021%2Facs.jmedchem.7b01243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Biology-Inspired Discovery of Novel KRAS-PDEδ Inhibitors</span></div><div class="casAuthors">Jiang, Yan; Zhuang, Chunlin; Chen, Long; Lu, Junjie; Dong, Guoqiang; Miao, Zhenyuan; Zhang, Wannian; Li, Jian; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9400-9406</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural biol. is a powerful tool for investigating the stereospecific interactions between a protein and its ligand.  Herein, an unprecedented chiral binding pattern was obsd. for inhibitors of KRAS-PDEδ interactions.  Virtual screening and X-ray crystallog. studies revealed that two enantiomers of a racemic inhibitor could bind at different sites.  Fragment-based drug design was used to identify highly potent PDEδ inhibitors that can be used as promising lead compds. for target validation and antitumor drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo93B5ydLZud7Vg90H21EOLACvtfcHk0lgnThFouLCseg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeiurbM&md5=4755e148413d9eeb0c8eb818b9a4cc53</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01243%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DZhuang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DStructural%2520Biology-Inspired%2520Discovery%2520of%2520Novel%2520KRAS-PDEdelta%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9400%26epage%3D9406%26doi%3D10.1021%2Facs.jmedchem.7b01243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel KRAS-PDEdelta Inhibitors by Fragment-Based Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2604</span>– <span class="NLM_lpage">2610</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00057</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00057" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvFWnsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2604-2610&author=L.+Chenauthor=C.+Zhuangauthor=J.+Luauthor=Y.+Jiangauthor=C.+Sheng&title=Discovery+of+Novel+KRAS-PDEdelta+Inhibitors+by+Fragment-Based+Drug+Design&doi=10.1021%2Facs.jmedchem.8b00057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel KRAS-PDEδ Inhibitors by Fragment-Based Drug Design</span></div><div class="casAuthors">Chen, Long; Zhuang, Chunlin; Lu, Junjie; Jiang, Yan; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2604-2610</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting KRAS-PDEδ protein-protein interactions with small mols. represents a promising opportunity for developing novel antitumor agents.  However, current KRAS-PDEδ inhibitors are limited by poor cellular antitumor potency and the druggability of the target remains to be validated by new inhibitors.  To tackle these challenges, herein, novel, highly potent KRAS-PDEδ inhibitors were identified by fragment-based drug design, providing promising lead compds. or chem. probes for investigating the biol. functions and druggability of KRAS-PDEδ interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqayXxD-smS5LVg90H21EOLACvtfcHk0lgnThFouLCseg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvFWnsLw%253D&md5=0b1ad029636a14f2794d548235a30360</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00057%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZhuang%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520Novel%2520KRAS-PDEdelta%2520Inhibitors%2520by%2520Fragment-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2604%26epage%3D2610%26doi%3D10.1021%2Facs.jmedchem.8b00057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainolfi, N.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation Mechanisms</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.ddtec.2019.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=31200860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=53-60&author=Y.+Zhangauthor=C.+Lohauthor=J.+Chenauthor=N.+Mainolfi&title=Targeted+Protein+Degradation+Mechanisms&doi=10.1016%2Fj.ddtec.2019.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation mechanisms</span></div><div class="casAuthors">Zhang Yi; Loh Christine; Chen Jesse; Mainolfi Nello</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeted protein degradation mediated by small molecule degraders represents an exciting new therapeutic opportunity to eliminate disease-causing proteins.  These molecules recruit E3 ubiquitin ligases to the protein of interest and mediate its ubiquitination and subsequent proteolysis by the proteasome.  Significant advancements have been made in the discovery and development of clinically relevant degraders.  In this review we will focus on the recent progress in understanding ternary complex formation and structures, ubiquitination, and other critical factors that govern the efficiency of degraders both in vitro and in vivo.  With deeper knowledges of these areas, the field is building guiding principles to reduce the level of empiricism and to identify therapeutically relevant degraders more rationally and efficiently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRBrI9DTGjC0fW6udTcc2eaDJdzY3bgkJLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtg%253D%253D&md5=bff167555f8bcab6b672cad3c2946a7b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLoh%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMainolfi%26aufirst%3DN.%26atitle%3DTargeted%2520Protein%2520Degradation%2520Mechanisms%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D53%26epage%3D60%26doi%3D10.1016%2Fj.ddtec.2019.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation by PROTACs</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">138</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.pharmthera.2017.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=28223226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=138-144&author=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=C.+M.+Crews&title=Targeted+Protein+Degradation+by+PROTACs&doi=10.1016%2Fj.pharmthera.2017.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation by PROTACs</span></div><div class="casAuthors">Neklesa, Taavi K.; Winkler, James D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">138-144</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using the PROTAC technol. is emerging as a novel therapeutic method to address diseases driven by the aberrant expression of a disease-causing protein.  PROTAC mols. are bifunctional small mols. that simultaneously bind a target protein and an E3-ubiquitin ligase, thus causing ubiquitination and degrdn. of the target protein by the proteasome.  Like small mols., PROTAC mols. possess good tissue distribution and the ability to target intracellular proteins.  Herein, we highlight the advantages of protein degrdn. using PROTACs, and provide specific examples where degrdn. offers therapeutic benefit over classical enzyme inhibition.  Foremost, PROTACs can degrade proteins regardless of their function.  This includes the currently "undruggable" proteome, which comprises approx. 85% of all human proteins.  Other beneficial aspects of protein degrdn. include the ability to target overexpressed and mutated proteins, as well as the potential to demonstrate prolonged pharmacodynamics effect beyond drug exposure.  Lastly, due to their catalytic nature and the pre-requisite ubiquitination step, an exquisitely potent mols. with a high degree of degrdn. selectivity can be designed.  Impressive preclin. in vitro and in vivo PROTAC data have been published, and these data have propelled the development of clin. viable PROTACs.  With the mol. wt. falling in the 700-1000 Da range, the delivery and bioavailability of PROTACs remain the largest hurdles on the way to the clinic.  Solving these issues and demonstrating proof of concept clin. data will be the focus of many labs over the next few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4RSLWYqqBMrVg90H21EOLACvtfcHk0lgAvV0k5LfsNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D&md5=08a74402a274a13c8c484cd68167a5a1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%2520by%2520PROTACs%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D174%26spage%3D138%26epage%3D144%26doi%3D10.1016%2Fj.pharmthera.2017.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: New Approaches to Protein Degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+New+Approaches+to+Protein+Degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lgAvV0k5LfsNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520New%2520Approaches%2520to%2520Protein%2520Degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+Kinase-Independent+Functions+of+Fak+via+PROTAC-Mediated+Degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0lj5lS07PZbm7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520Kinase-Independent%2520Functions%2520of%2520Fak%2520via%2520PROTAC-Mediated%2520Degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-Induced BET Protein Degradation as a Therapy for Castration-resistant Prostate Cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-Induced+BET+Protein+Degradation+as+a+Therapy+for+Castration-resistant+Prostate+Cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lj5lS07PZbm7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-Induced%2520BET%2520Protein%2520Degradation%2520as%2520a%2520Therapy%2520for%2520Castration-resistant%2520Prostate%2520Cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Estrogen Receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.+Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+Highly+Potent+Proteolysis+Targeting+Chimera+%28PROTAC%29+Degrader+of+Estrogen+Receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lj5lS07PZbm7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520Highly%2520Potent%2520Proteolysis%2520Targeting%2520Chimera%2520%2528PROTAC%2529%2520Degrader%2520of%2520Estrogen%2520Receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-Induced BTK Degradation as a Novel Therapy for Mutated BTK C481S Induced Ibrutinib-Resistant B-cell Malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-Induced+BTK+Degradation+as+a+Novel+Therapy+for+Mutated+BTK+C481S+Induced+Ibrutinib-Resistant+B-cell+Malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0ljPS-3cA8Y8LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-Induced%2520BTK%2520Degradation%2520as%2520a%2520Novel%2520Therapy%2520for%2520Mutated%2520BTK%2520C481S%2520Induced%2520Ibrutinib-Resistant%2520B-cell%2520Malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, S. J.</span></span> <span> </span><span class="NLM_article-title">Chimeric Molecules Facilitate the Degradation of Androgen Receptors and Repress the Growth of LNCaP Cells</span>. <i>Asian J. Androl.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1038/aja.2008.26</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Faja.2008.26" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=19050678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnvFCqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=119-126&author=Y.+Q.+Tangauthor=B.+M.+Hanauthor=X.+Q.+Yaoauthor=Y.+Hongauthor=Y.+Wangauthor=F.+J.+Zhaoauthor=S.+Q.+Yuauthor=X.+W.+Sunauthor=S.+J.+Xia&title=Chimeric+Molecules+Facilitate+the+Degradation+of+Androgen+Receptors+and+Repress+the+Growth+of+LNCaP+Cells&doi=10.1038%2Faja.2008.26"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Chimeric molecules facilitate the degradation of androgen receptors and repress the growth of LNCaP cells</span></div><div class="casAuthors">Tang, Yue-Qing; Han, Bang-Min; Yao, Xin-Quan; Hong, Yan; Wang, Yan; Zhao, Fu-Jun; Yu, Sheng-Qiang; Sun, Xiao-Wen; Xia, Shu-Jie</div><div class="citationInfo"><span class="NLM_cas:title">Asian Journal of Andrology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">119-126</span>CODEN:
                <span class="NLM_cas:coden">ASJAF8</span>;
        ISSN:<span class="NLM_cas:issn">1008-682X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Post-translational degrdn. of protein plays an important role in cell life.  The authors employed chimeric mols. (dihydrotestosterone-based proteolysis-targeting chimeric mol. [DHT-PROTAC]) to facilitate androgen receptor (AR) degrdn. via the ubiquitin-proteasome pathway (UPP) and to investigate the role of AR in cell proliferation and viability in androgen-sensitive prostate cancer cells.  Western blot anal. and immunohistochem. were applied to analyze AR levels in LNCaP cells after DHT-PROTAC treatment.  Cell counting and the 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl tetrazolium bromide (MTT) cell viability assay were used to evaluate cell proliferation and viability after AR elimination in both LNCaP and PC-3 cells.  AR was tagged for elimination via the UPP by DHT-PROTAC, and this could be blocked by proteasome inhibitors.  Degrdn. of AR depended on DHT-PROTAC concn., and either DHT or an ALAPYIP-(arg)8 peptide could compete with DHT-PROTAC.  Inhibition of cell proliferation and decreased viability were obsd. in LNCaP cells, but not in PC-3 or 786-O cells after DHT-PROTAC treatment.  These data indicate that AR elimination is facilitated via the UPP by DHT-PROTAC, and that the growth of LNCaP cells is repressed after AR degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxakeBlqmp1LVg90H21EOLACvtfcHk0ljPS-3cA8Y8LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnvFCqsr4%253D&md5=7bc5fedd68553b2898725bfa877e13c0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Faja.2008.26&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Faja.2008.26%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DY.%2BQ.%26aulast%3DHan%26aufirst%3DB.%2BM.%26aulast%3DYao%26aufirst%3DX.%2BQ.%26aulast%3DHong%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DF.%2BJ.%26aulast%3DYu%26aufirst%3DS.%2BQ.%26aulast%3DSun%26aufirst%3DX.%2BW.%26aulast%3DXia%26aufirst%3DS.%2BJ.%26atitle%3DChimeric%2520Molecules%2520Facilitate%2520the%2520Degradation%2520of%2520Androgen%2520Receptors%2520and%2520Repress%2520the%2520Growth%2520of%2520LNCaP%2520Cells%26jtitle%3DAsian%2520J.%2520Androl.%26date%3D2009%26volume%3D11%26spage%3D119%26epage%3D126%26doi%3D10.1038%2Faja.2008.26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-Induced+Protein+Degradation+in+Drug+Discovery%3A+Breaking+the+Rules+or+Just+Making+New+Ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0ljPS-3cA8Y8LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-Induced%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%253A%2520Breaking%2520the%2520Rules%2520or%2520Just%2520Making%2520New%2520Ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span> <span> </span><span class="NLM_article-title">Development of Targeted Protein Degradation Therapeutics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">944</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0362-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fs41589-019-0362-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=31527835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=937-944&author=P.+P.+Chamberlainauthor=L.+G.+Hamann&title=Development+of+Targeted+Protein+Degradation+Therapeutics&doi=10.1038%2Fs41589-019-0362-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Development of targeted protein degradation therapeutics</span></div><div class="casAuthors">Chamberlain, Philip P.; Hamann, Lawrence G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">937-944</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted protein degrdn. as a therapeutic modality has seen dramatic progress and massive investment in recent years because of the convergence of two key scientific breakthroughs: optimization of first-generation peptidic proteolysis-targeted chimeras (PROTACs) into more drug-like mols. able to support in vivo proof of concept and the discovery that clin. mols. function as degraders by binding and repurposing the proteins cereblon and DCAF15.  This provided clin. validation for the general approach through the cereblon modulator class of drugs and provided highly drug-like and ligand-efficient E3 ligase binders upon which to tether target-binding moieties.  Increasingly rational and systematic approaches including biophys. and structural studies on ternary complexes are being leveraged as the field advances.  In this Perspective we summarize the discoveries that have laid the foundation for future degrdn. therapeutics, focusing on those classes of small mols. that redirect E3 ubiquitin ligases to non-native substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfcIEwVR-Xq7Vg90H21EOLACvtfcHk0lg0QvtxH1H5sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF&md5=468e3fd87ba2b21607dac8b495e5ecf6</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0362-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0362-y%26sid%3Dliteratum%253Aachs%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26atitle%3DDevelopment%2520of%2520Targeted%2520Protein%2520Degradation%2520Therapeutics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D937%26epage%3D944%26doi%3D10.1038%2Fs41589-019-0362-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">First Targeted Protein Degrader Hits the Clinic</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00043-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fd41573-019-00043-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=237-239&author=A.+Mullard&title=First+Targeted+Protein+Degrader+Hits+the+Clinic&doi=10.1038%2Fd41573-019-00043-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00043-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00043-6%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DFirst%2520Targeted%2520Protein%2520Degrader%2520Hits%2520the%2520Clinic%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D237%26epage%3D239%26doi%3D10.1038%2Fd41573-019-00043-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><a href="https://ir.arvinas.com/press-releases?page=1" class="extLink">https://ir.arvinas.com/press-releases?page=1</a> (accessed Oct 15, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fir.arvinas.com%2Fpress-releases%3Fpage%3D1+%28accessed+Oct+15%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">The PROTAC Technology in Drug Development</span>. <i>Cell Biochem. Funct.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1002/cbf.3369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1002%2Fcbf.3369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=30604499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=21-30&author=Y.+Zouauthor=D.+Maauthor=Y.+Wang&title=The+PROTAC+Technology+in+Drug+Development&doi=10.1002%2Fcbf.3369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The PROTAC technology in drug development</span></div><div class="casAuthors">Zou, Yutian; Ma, Danhui; Wang, Yinyin</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biochemistry and Function</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-30</span>CODEN:
                <span class="NLM_cas:coden">CBFUDH</span>;
        ISSN:<span class="NLM_cas:issn">1099-0844</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Currently, a new technol. termed PROTAC, proteolysis targeting chimera, has been developed for inducing the protein degrdn. by a targeting mol.  This technol. takes advantage of a moiety of targeted protein and a moiety of recognizing E3 ubiquitin ligase and produces a hybrid mol. to specifically knock down a targeted protein.  During the first decade, three pedigreed groups worked on the development of this technol.  To date, this technol. has been extended by different groups, aiming to develop new drugs against different diseases including cancers.  This review summarizes the contributions of the groups for the development of PROTAC.  Significance of the study : This review summarized the development of the PROTAC technol. for readers and also presented the author's opinions on the application of the technol. in tumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprT1CUHOqrfbVg90H21EOLACvtfcHk0lg0QvtxH1H5sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygsQ%253D%253D&md5=2a7417e2c385d01f7eaef52a72c7ea52</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcbf.3369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbf.3369%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DThe%2520PROTAC%2520Technology%2520in%2520Drug%2520Development%26jtitle%3DCell%2520Biochem.%2520Funct.%26date%3D2019%26volume%3D37%26spage%3D21%26epage%3D30%26doi%3D10.1002%2Fcbf.3369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced Protein Degradation: an Emerging Drug Discovery Paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+Protein+Degradation%3A+an+Emerging+Drug+Discovery+Paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lg0QvtxH1H5sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520Protein%2520Degradation%253A%2520an%2520Emerging%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis Targeting Chimeras (PROTACs) in ’Beyond Rule-of-Five’ Chemical Space: Recent Progress and Future Challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+Targeting+Chimeras+%28PROTACs%29+in+%E2%80%99Beyond+Rule-of-Five%E2%80%99+Chemical+Space%3A+Recent+Progress+and+Future+Challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0liKSrIJQjrJqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2599Beyond%2520Rule-of-Five%25E2%2580%2599%2520Chemical%2520Space%253A%2520Recent%2520Progress%2520and%2520Future%2520Challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Misale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scala, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janakiraman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liska, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valtorta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiavo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buscarino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siravegna, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bencardino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cercek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartore-Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gambacorta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallicchio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakiani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boscaro, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medico, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Nicolantonio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span> <span> </span><span class="NLM_article-title">Emergence of KRAS Mutations and Acquired Resistance to Anti-EGFR Therapy in Colorectal Cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>486</i></span>,  <span class="NLM_fpage">532</span>– <span class="NLM_lpage">536</span>, <span class="refDoi"> DOI: 10.1038/nature11156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Fnature11156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=22722830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVSrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=532-536&author=S.+Misaleauthor=R.+Yaegerauthor=S.+Hoborauthor=E.+Scalaauthor=M.+Janakiramanauthor=D.+Liskaauthor=E.+Valtortaauthor=R.+Schiavoauthor=M.+Buscarinoauthor=G.+Siravegnaauthor=K.+Bencardinoauthor=A.+Cercekauthor=C.+T.+Chenauthor=S.+Veroneseauthor=C.+Zanonauthor=A.+Sartore-Bianchiauthor=M.+Gambacortaauthor=M.+Gallicchioauthor=E.+Vakianiauthor=V.+Boscaroauthor=E.+Medicoauthor=M.+Weiserauthor=S.+Sienaauthor=F.+Di+Nicolantonioauthor=D.+Solitauthor=A.+Bardelli&title=Emergence+of+KRAS+Mutations+and+Acquired+Resistance+to+Anti-EGFR+Therapy+in+Colorectal+Cancer&doi=10.1038%2Fnature11156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of KRAS mutations and acquired resistance to anti-EGFR therapy in colorectal cancer</span></div><div class="casAuthors">Misale, Sandra; Yaeger, Rona; Hobor, Sebastijan; Scala, Elisa; Janakiraman, Manickam; Liska, David; Valtorta, Emanuele; Schiavo, Roberta; Buscarino, Michela; Siravegna, Giulia; Bencardino, Katia; Cercek, Andrea; Chen, Chin-Tung; Veronese, Silvio; Zanon, Carlo; Sartore-Bianchi, Andrea; Gambacorta, Marcello; Gallicchio, Margherita; Vakiani, Efsevia; Boscaro, Valentina; Medico, Enzo; Weiser, Martin; Siena, Salvatore; Di Nicolantonio, Federica; Solit, David; Bardelli, Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">486</span>
        (<span class="NLM_cas:issue">7404</span>),
    <span class="NLM_cas:pages">532-536</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A main limitation of therapies that selectively target kinase signalling pathways is the emergence of secondary drug resistance.  Cetuximab, a monoclonal antibody that binds the extracellular domain of epidermal growth factor receptor (EGFR), is effective in a subset of KRAS wild-type metastatic colorectal cancers.  After an initial response, secondary resistance invariably ensues, thereby limiting the clin. benefit of this drug.  The mol. bases of secondary resistance to cetuximab in colorectal cancer are poorly understood.  Here we show that mol. alterations (in most instances point mutations) of KRAS are causally assocd. with the onset of acquired resistance to anti-EGFR treatment in colorectal cancers.  Expression of mutant KRAS under the control of its endogenous gene promoter was sufficient to confer cetuximab resistance, but resistant cells remained sensitive to combinatorial inhibition of EGFR and mitogen-activated protein-kinase kinase (MEK).  Anal. of metastases from patients who developed resistance to cetuximab or panitumumab showed the emergence of KRAS amplification in one sample and acquisition of secondary KRAS mutations in 60% (6 out of 10) of the cases.  KRAS mutant alleles were detectable in the blood of cetuximab-treated patients as early as 10 mo before radiog. documentation of disease progression.  In summary, the results identify KRAS mutations as frequent drivers of acquired resistance to cetuximab in colorectal cancers, indicate that the emergence of KRAS mutant clones can be detected non-invasively months before radiog. progression and suggest early initiation of a MEK inhibitor as a rational strategy for delaying or reversing drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6h0UDIrrnbLVg90H21EOLACvtfcHk0liKSrIJQjrJqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsVSrsb4%253D&md5=8f06d4203e06a40ae8e5157d54a432e7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature11156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11156%26sid%3Dliteratum%253Aachs%26aulast%3DMisale%26aufirst%3DS.%26aulast%3DYaeger%26aufirst%3DR.%26aulast%3DHobor%26aufirst%3DS.%26aulast%3DScala%26aufirst%3DE.%26aulast%3DJanakiraman%26aufirst%3DM.%26aulast%3DLiska%26aufirst%3DD.%26aulast%3DValtorta%26aufirst%3DE.%26aulast%3DSchiavo%26aufirst%3DR.%26aulast%3DBuscarino%26aufirst%3DM.%26aulast%3DSiravegna%26aufirst%3DG.%26aulast%3DBencardino%26aufirst%3DK.%26aulast%3DCercek%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DC.%2BT.%26aulast%3DVeronese%26aufirst%3DS.%26aulast%3DZanon%26aufirst%3DC.%26aulast%3DSartore-Bianchi%26aufirst%3DA.%26aulast%3DGambacorta%26aufirst%3DM.%26aulast%3DGallicchio%26aufirst%3DM.%26aulast%3DVakiani%26aufirst%3DE.%26aulast%3DBoscaro%26aufirst%3DV.%26aulast%3DMedico%26aufirst%3DE.%26aulast%3DWeiser%26aufirst%3DM.%26aulast%3DSiena%26aufirst%3DS.%26aulast%3DDi%2BNicolantonio%26aufirst%3DF.%26aulast%3DSolit%26aufirst%3DD.%26aulast%3DBardelli%26aufirst%3DA.%26atitle%3DEmergence%2520of%2520KRAS%2520Mutations%2520and%2520Acquired%2520Resistance%2520to%2520Anti-EGFR%2520Therapy%2520in%2520Colorectal%2520Cancer%26jtitle%3DNature%26date%3D2012%26volume%3D486%26spage%3D532%26epage%3D536%26doi%3D10.1038%2Fnature11156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eide, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilertsen, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielsen, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eknaes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hektoen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lothe, R. A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic and Genetic Features of 24 Colon Cancer Cell Lines</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">e71</span>, <span class="refDoi"> DOI: 10.1038/oncsis.2013.35</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1038%2Foncsis.2013.35" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=24042735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCmsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e71&author=D.+Ahmedauthor=P.+W.+Eideauthor=I.+A.+Eilertsenauthor=S.+A.+Danielsenauthor=M.+Eknaesauthor=M.+Hektoenauthor=G.+E.+Lindauthor=R.+A.+Lothe&title=Epigenetic+and+Genetic+Features+of+24+Colon+Cancer+Cell+Lines&doi=10.1038%2Foncsis.2013.35"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic and genetic features of 24 colon cancer cell lines</span></div><div class="casAuthors">Ahmed, D.; Eide, P. W.; Eilertsen, I. A.; Danielsen, S. A.; Eknaes, M.; Hektoen, M.; Lind, G. E.; Lothe, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">Sept.</span>),
    <span class="NLM_cas:pages">e71</span>CODEN:
                <span class="NLM_cas:coden">ONCOCT</span>;
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cell lines are invaluable biomedical research tools, and recent literature has emphasized the importance of genotype authentication and characterization.  In the present study, 24 out of 27 cell line identities were confirmed by short tandem repeat profiling.  The mol. phenotypes of the 24 colon cancer cell lines were examd., and microsatellite instability (MSI) and CpG island methylator phenotype (CIMP) were detd., using the Bethesda panel mononucleotide repeat loci and two epimarker panels, resp.  Furthermore, the BRAF, KRAS and PIK3CA oncogenes were analyzed for mutations in known hotspots, while the entire coding sequences of the PTEN and TP53 tumor suppressors were investigated.  Nine cell lines showed MSI.  Thirteen and nine cell lines were found to be CIMP pos., using the Issa panel and the Weisenberger et al. panel, resp.  The latter was found to be superior for CIMP classification of colon cancer cell lines.  Seventeen cell lines harbored disrupting TP53 mutations.  Altogether, 20/24 cell lines had the mitogen-activated protein kinase pathway activating mutually exclusive KRAS or BRAF mutations.  PIK3CA and PTEN mutations leading to hyperactivation of the phosphoinositide 3-kinase/AKT pathway were obsd. in 13/24 cell lines.  Interestingly, in four cell lines there were no mutations in neither BRAF, KRAS, PIK3CA nor in PTEN.  In conclusion, this study presents mol. features of a large no. of colon cancer cell lines to aid the selection of suitable in vitro models for descriptive and functional research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrowLQizkeM57Vg90H21EOLACvtfcHk0liKSrIJQjrJqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCmsrzL&md5=397a60bad73ca436da303de020df186a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Foncsis.2013.35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Foncsis.2013.35%26sid%3Dliteratum%253Aachs%26aulast%3DAhmed%26aufirst%3DD.%26aulast%3DEide%26aufirst%3DP.%2BW.%26aulast%3DEilertsen%26aufirst%3DI.%2BA.%26aulast%3DDanielsen%26aufirst%3DS.%2BA.%26aulast%3DEknaes%26aufirst%3DM.%26aulast%3DHektoen%26aufirst%3DM.%26aulast%3DLind%26aufirst%3DG.%2BE.%26aulast%3DLothe%26aufirst%3DR.%2BA.%26atitle%3DEpigenetic%2520and%2520Genetic%2520Features%2520of%252024%2520Colon%2520Cancer%2520Cell%2520Lines%26jtitle%3DOncogenesis%26date%3D2013%26volume%3D2%26spage%3De71%26doi%3D10.1038%2Foncsis.2013.35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirasawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasazuki, T.</span></span> <span> </span><span class="NLM_article-title">Altered Growth of Human Colon Cancer Cell Lines Disrupted at Activated Ki-ras</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>260</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1126/science.8465203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=10.1126%2Fscience.8465203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=8465203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADyaK3sXisFagurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=260&publication_year=1993&pages=85-88&author=S.+Shirasawaauthor=M.+Furuseauthor=N.+Yokoyamaauthor=T.+Sasazuki&title=Altered+Growth+of+Human+Colon+Cancer+Cell+Lines+Disrupted+at+Activated+Ki-ras&doi=10.1126%2Fscience.8465203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Altered growth of human colon cancer cell lines disrupted at activated Ki-ras</span></div><div class="casAuthors">Shirasawa, Senji; Furuse, Masanori; Yokoyama, Nobuhiko; Sasazuki, Takehiko</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">260</span>
        (<span class="NLM_cas:issue">5104</span>),
    <span class="NLM_cas:pages">85-8</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Point mutations that activate the Ki-ras proto-oncogene are present in about 50 percent of human colorectal tumors.  To study the functional significance of these mutations, the activated Ki-ras genes in two human colon carcinoma cell lines, DLD-1 and HCT116, were disrupted by homologous recombination.  Compared with parental cells, cells disrupted at the activated Ki-ras gene were morphol. altered, lost the capacity for anchorage-independent growth, grew more slowly both in vitro and in nude mice, and showed reduced expression of c-myc.  Thus, the activated ki-ras gene plays a key role in colorectal tumorigenesis through altered cell differentiation and cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM2FBzJqjRorVg90H21EOLACvtfcHk0lgLXM2qaUfdUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisFagurw%253D&md5=06dd636f868a5354676d483aa6d96bd2</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.8465203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8465203%26sid%3Dliteratum%253Aachs%26aulast%3DShirasawa%26aufirst%3DS.%26aulast%3DFuruse%26aufirst%3DM.%26aulast%3DYokoyama%26aufirst%3DN.%26aulast%3DSasazuki%26aufirst%3DT.%26atitle%3DAltered%2520Growth%2520of%2520Human%2520Colon%2520Cancer%2520Cell%2520Lines%2520Disrupted%2520at%2520Activated%2520Ki-ras%26jtitle%3DScience%26date%3D1993%26volume%3D260%26spage%3D85%26epage%3D88%26doi%3D10.1126%2Fscience.8465203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors Simultaneously Targeting Cancer Metabolism and Epigenetics: Discovery of Novel Nicotinamide Phosphoribosyltransferase (NAMPT) and Histone Deacetylase (HDAC) Dual Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7965</span>– <span class="NLM_lpage">7983</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00467</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00467" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKhurrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7965-7983&author=G.+Dongauthor=W.+Chenauthor=X.+Wangauthor=X.+Yangauthor=T.+Xuauthor=P.+Wangauthor=W.+Zhangauthor=Y.+Raoauthor=C.+Miaoauthor=C.+Sheng&title=Small+Molecule+Inhibitors+Simultaneously+Targeting+Cancer+Metabolism+and+Epigenetics%3A+Discovery+of+Novel+Nicotinamide+Phosphoribosyltransferase+%28NAMPT%29+and+Histone+Deacetylase+%28HDAC%29+Dual+Inhibitors&doi=10.1021%2Facs.jmedchem.7b00467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors Simultaneously Targeting Cancer Metabolism and Epigenetics: Discovery of Novel Nicotinamide Phosphoribosyltransferase (NAMPT) and Histone Deacetylase (HDAC) Dual Inhibitors</span></div><div class="casAuthors">Dong, Guoqiang; Chen, Wei; Wang, Xia; Yang, Xinglin; Xu, Tianying; Wang, Pei; Zhang, Wannian; Rao, Yu; Miao, Chaoyu; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7965-7983</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cancer metab. and epigenetics are among the most intensely pursued research areas in anticancer drug discovery.  Here we report the first small mols. that simultaneously inhibit nicotinamide phosphoribosyltransferase (NAMPT)and histone deacetylase (HDAC), two important targets of cancer metab. and epigenetics, resp.  Through iterative structure-based drug design, chem. synthesis, and biol. assays, a highly potent dual NAMPT and HDAC inhibitor was successfully identified.  Compd. 35 possessed excellent and balanced activities against both NAMPT (IC50 = 31 nM) and HDAC1 (IC50 = 55 nM).  It could effectively induce cell apoptosis and autophagy and ultimately led to cell death.  Importantly, compd. 35 showed excellent in vivo antitumor efficacy in the HCT116 xenograft model.  This proof-of-concept study demonstrates the feasibility of discovering an inhibitor targeting cancer metab. and epigenetics and provides an efficient strategy for multitarget antitumor drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9UvfGattu1rVg90H21EOLACvtfcHk0lgLXM2qaUfdUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKhurrO&md5=65df0b485cbbc169886d34ae4421f814</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00467%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DC.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520Simultaneously%2520Targeting%2520Cancer%2520Metabolism%2520and%2520Epigenetics%253A%2520Discovery%2520of%2520Novel%2520Nicotinamide%2520Phosphoribosyltransferase%2520%2528NAMPT%2529%2520and%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Dual%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7965%26epage%3D7983%26doi%3D10.1021%2Facs.jmedchem.7b00467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E80" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E80','PDB','5E80'); return false;">PDB: 5E80</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i48"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00929">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_52534"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00929?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00929</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular docking, Caco-2 permeability assay, immunofluorescence staining, and <sup>1</sup>H and <sup>13</sup>C NMR spectra of representative compound (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular docking of compound 2 with PDEδ (2_5E80.pdb) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_002.pdb">PDB</a>)</p></li><li><p class="inline">SMILES molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_001.pdf">jm0c00929_si_001.pdf (959.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_002.pdb">jm0c00929_si_002.pdb (203.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00929/suppl_file/jm0c00929_si_003.csv">jm0c00929_si_003.csv (1.42 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00929&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00929%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-14%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00929" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                12MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a7f645a00191d","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
