
<html lang="en"     class="pb-page"  data-request-id="a92acc28-a88e-4dc4-85ff-fe2cb727d036"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2018.61.issue-19;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.7b01921"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Emerging Approaches for the Identification of Protein Targets of Small Molecules - A Practitioners’ Perspective" /></meta><meta name="dc.Creator" content="Kenneth M.  Comess" /></meta><meta name="dc.Creator" content="Shaun M.  McLoughlin" /></meta><meta name="dc.Creator" content="Jon A.  Oyer" /></meta><meta name="dc.Creator" content="Paul L.  Richardson" /></meta><meta name="dc.Creator" content="Henning  Stöckmann" /></meta><meta name="dc.Creator" content="Anil  Vasudevan" /></meta><meta name="dc.Creator" content="Scott E.  Warder" /></meta><meta name="dc.Description" content="Small-molecule (SM) leads in the early drug discovery pipeline are progressed primarily based on potency against the intended target(s) and selectivity against a very narrow slice of the proteome. ..." /></meta><meta name="Description" content="Small-molecule (SM) leads in the early drug discovery pipeline are progressed primarily based on potency against the intended target(s) and selectivity against a very narrow slice of the proteome. ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 2, 2018" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01921" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01921" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01921" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01921" /></link>
        
    
    

<title>Emerging Approaches for the Identification of Protein Targets of Small Molecules - A Practitioners’ Perspective | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01921" /></meta><meta property="og:title" content="Emerging Approaches for the Identification of Protein Targets of Small Molecules - A Practitioners’ Perspective" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0017.jpeg" /></meta><meta property="og:description" content="Small-molecule (SM) leads in the early drug discovery pipeline are progressed primarily based on potency against the intended target(s) and selectivity against a very narrow slice of the proteome. So, why is there a tendency to wait until SMs are matured before probing for a deeper mechanistic understanding? For one, there is a concern about the interpretation of complex -omic data outputs and the resources needed to test these hypotheses. However, with recent advances in broad endpoint profiling assays that have companion reference databases and refined technology integration strategies, we argue that data complexity can translate into meaningful decision-making. This same strategy can also prioritize phenotypic screening hits to increase the likelihood of accessing unprecedented target space. In this Perspective. we will highlight a cohesive process that supports SM hit prosecution, providing a data-driven rationale and a suite of methods for direct identification of SM targets driving relevant biological end points." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01921"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01921">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01921&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01921&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01921&amp;href=/doi/10.1021/acs.jmedchem.7b01921" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 19</span><span class="cit-fg-pageRange">, 8504-8535</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/19" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01898" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.8b00249" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Emerging Approaches for the Identification of Protein Targets of Small Molecules - A Practitioners’ Perspective</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Kenneth M. Comess</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kenneth M. Comess</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">AbbVie Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kenneth+M.++Comess">Kenneth M. Comess</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shaun M. McLoughlin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaun M. McLoughlin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">AbbVie Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaun+M.++McLoughlin">Shaun M. McLoughlin</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jon A. Oyer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jon A. Oyer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">AbbVie Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jon+A.++Oyer">Jon A. Oyer</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul L. Richardson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul L. Richardson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">AbbVie Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul+L.++Richardson">Paul L. Richardson</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Henning Stöckmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Henning Stöckmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">AbbVie Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Henning++St%C3%B6ckmann">Henning Stöckmann</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath5">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2409-5476" title="Orcid link">http://orcid.org/0000-0002-2409-5476</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anil Vasudevan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anil Vasudevan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">AbbVie Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#e1a08f888dcfb7809294858497808fa1a08383978884cf828e8c"><span class="__cf_email__" data-cfemail="7637181f1a5820170503121300171836371414001f135815191b">[email protected]</span></a>. Phone number: (847) 938 6594.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anil++Vasudevan">Anil Vasudevan</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath6">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0004-0497" title="Orcid link">http://orcid.org/0000-0002-0004-0497</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Scott E. Warder</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott E. Warder</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">AbbVie Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott+E.++Warder">Scott E. Warder</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath7">View Biography</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01921&amp;href=/doi/10.1021%2Facs.jmedchem.7b01921" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 19</span><span class="cit-pageRange">, 8504–8535</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 2, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 December 2017</li><li><span class="item_label"><b>Published</b> online</span>2 May 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 October 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01921" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01921</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8504%26pageCount%3D32%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DKenneth%2BM.%2BComess%252C%2BShaun%2BM.%2BMcLoughlin%252C%2BJon%2BA.%2BOyer%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D19%26contentID%3Dacs.jmedchem.7b01921%26title%3DEmerging%2BApproaches%2Bfor%2Bthe%2BIdentification%2Bof%2BProtein%2BTargets%2Bof%2BSmall%2BMolecules%2B-%2BA%2BPractitioners%25E2%2580%2599%2BPerspective%26numPages%3D32%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8535%26publicationDate%3DOctober%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01921"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">6629</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">32</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01921" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Emerging Approaches for the Identification of Protein Targets of Small Molecules - A Practitioners’ Perspective&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Kenneth&quot;,&quot;last_name&quot;:&quot;M. Comess&quot;},{&quot;first_name&quot;:&quot;Shaun&quot;,&quot;last_name&quot;:&quot;M. McLoughlin&quot;},{&quot;first_name&quot;:&quot;Jon&quot;,&quot;last_name&quot;:&quot;A. Oyer&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;L. Richardson&quot;},{&quot;first_name&quot;:&quot;Henning&quot;,&quot;last_name&quot;:&quot;Stöckmann&quot;},{&quot;first_name&quot;:&quot;Anil&quot;,&quot;last_name&quot;:&quot;Vasudevan&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;E. Warder&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;19&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;8504-8535&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01921&quot;},&quot;abstract&quot;:&quot;Small-molecule (SM) leads in the early drug discovery pipeline are progressed primarily based on potency against the intended target(s) and selectivity against a very narrow slice of the proteome. So, why is there a tendency to wait until SMs are matured before probing for a deeper mechanistic understanding? For one, there is a concern about the interpretation of complex -omic data outputs and the resources needed to test these hypotheses. However, with recent advances in broad endpoint profiling assays that have companion reference databases and refined technology integration strategies, we argue that data complexity can translate into meaningful decision-making. This same strategy can also prioritize phenotypic screening hits to increase the likelihood of accessing unprecedented target space. In this Perspective. we will highlight a cohesive process that supports SM hit prosecution, providing a data-driven rationale and a suite of methods for direct identification of SM targets driving relevant biological e&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01921&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01921" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01921&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01921" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01921&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01921" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01921&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01921&amp;href=/doi/10.1021/acs.jmedchem.7b01921" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01921" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01921" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (11 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01921%26sid%3Dliteratum%253Aachs%26pmid%3D29718665%26genre%3Darticle%26aulast%3DComess%26date%3D2018%26atitle%3DEmerging%2BApproaches%2Bfor%2Bthe%2BIdentification%2Bof%2BProtein%2BTargets%2Bof%2BSmall%2BMolecules%2B-%2BA%2BPractitioners%25E2%2580%2599%2BPerspective%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D19%26spage%3D8504%26epage%3D8535%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292601" title="Probes">Probes</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/19" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/jmcmar.2018.61.issue-19/20181011/jmcmar.2018.61.issue-19.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Small-molecule (SM) leads in the early drug discovery pipeline are progressed primarily based on potency against the intended target(s) and selectivity against a very narrow slice of the proteome. So, why is there a tendency to wait until SMs are matured before probing for a deeper mechanistic understanding? For one, there is a concern about the interpretation of complex -omic data outputs and the resources needed to test these hypotheses. However, with recent advances in broad endpoint profiling assays that have companion reference databases and refined technology integration strategies, we argue that data complexity can translate into meaningful decision-making. This same strategy can also prioritize phenotypic screening hits to increase the likelihood of accessing unprecedented target space. In this Perspective. we will highlight a cohesive process that supports SM hit prosecution, providing a data-driven rationale and a suite of methods for direct identification of SM targets driving relevant biological end points.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The pharmaceutical industry strives to discover safe and efficacious molecules that modulate disease-dependent biology, with the ultimate goal of identifying and treating patients responsive to therapy. This mission presents many challenges, including sources of new therapeutic drug targets and selecting small molecules (SMs) with the highest likelihood of success. Although phenotypic drug discovery (PDD) has contributed to the development of best-in-class medicines by connecting SMs to unique mechanisms of action (MOA), this approach presents significant complexity.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> Ideally, PDD campaigns utilize physiologically relevant models that are linked to patient-derived biology, which in turn may self-select SM phenotypic hits affecting pathways and protein targets most relevant to the disease of interest. To enable this disease relevance, important considerations and assay design criteria have been encouraged, including the phenotypic rule of 3<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and the concept of a chain of translatability.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> However, hit progression from PDD is often impeded due to the resource-intensive and unpredictable time lines associated with SM-target deconvolution campaigns. As a result, the industry relies mainly on target-based drug discovery (TDD) with a large number of the prosecuted targets originating from the literature. In both cases, SM prioritization from TDD and PDD would benefit from a deeper appreciation of the MOA and target engagement profile.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">Recently, PDD has experienced a renewed interest from drug hunters and sparked a series of lively discussions and debates. One of the most common questions is: “PDD or TDD”? The pros and cons of this question have been thoroughly summarized in a recent review.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Fortunately, the field is recognizing the value of a balanced approach with potential to uncover unique biology. For example, a PDD campaign that leads to discovery of a new target often requires TDD follow-on because the initial hit active in the experimental model system is usually a cellular probe that lacks chemical properties required for an effective drug in humans. In cases where the PDD hit originates from an informer deck and tracks with an annotated target, this may initiate a repurposing exercise and not require additional HTS.</div><div class="NLM_p">Lately, the PDD debate has focused on progressing SMs before a molecular target has been identified. Several commentaries have outlined the main scenarios and considerations for SM advancement, including impressive examples of perseverance.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> This topic continues to be of interest, in part due to the complexity and high failure associated with target deconvolution. In our experience, outside of repurposing SMs previously optimized for in vivo target modulation, PDD probes worthy of nominating for a “blind medicinal chemistry campaign” are the exception and not the norm. In other words, we often lack the detailed cellular MOA and evidence for unique biology to confidently de-risk such an effort. In addition, prosecution in the absence of a target precludes the use of structure-based drug design and the opportunity to have back-up chemical series for consideration. Moving forward, we feel that our commitment to understanding MOA on a deeper cellular level will afford more opportunities to not only progress SMs that lack a defined interaction but increase the success rate of identifying the target(s) driving the desired phenotype.</div><div class="NLM_p">Approaches for target identification and elucidation of MOA have been described in detail.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12">(8−12)</a> Beyond capturing the technology options driving success in our laboratories, our aim is to lay out a rational strategy for technology integration that merges foundational assays with additional fit-for-purpose solutions. The operational framework and recommended themes that we expand upon in this Perspective are summarized in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. The process was initially implemented to progress PDD screening hits in a data-driven manner, but the added benefit of identifying higher quality TDD SMs for advancement was also quickly realized. The process is dynamic, with a constant influx of technology assessments. Ideally, for each new technology applied, we test multiple compounds that cover broad target-class diversity and generally avoid methods where value is restricted to a relatively small niche of SMs. In fact, new methods are gauged by how their data complements existing approaches in addition to their individual impact because data integration is the foundation of our target deconvolution strategy. We believe that with refinement of our stratagem, including a deeper appreciation of the limitations of technology offerings, a process will evolve that leads to better choices for progression of SMs from PDD and TDD.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Process roadmap for elucidation of small-molecule (SM) mechanism of action (MOA) and target identification.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The aim of this Perspective is to advocate for more-informed decision making throughout the progression of SMs in the early and late drug discovery pipeline. Because a single technology is unlikely to consistently deliver identification of targets and salient mechanistic biology, we present a core set of assays to set the SM knowledge baseline. In addition, reference SM profiles derived from data-rich biological assays are described as a powerful approach to rule out common MOAs, biologically bin SM cellular perturbations, and focus target identification strategies. SM enrichment of binding partners using chemically modified hits and label-free approaches are summarized, including examples of impact on PDD and TDD campaigns.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  SM Profiling for Differentiation, Prioritization, and MOA-Target ID</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62468" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62468" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  In Silico Profiling</h3><div class="NLM_p">There are varied approaches for the computational identification of drug targets of SMs. These methods can be broadly classified into chemical similarity searching, data mining/machine learning, panel docking, and the analysis of bioactivity spectra. Recently, other classes such as protein-structure-based methods have been proposed. A summary of some of the readily accessible tools for target prediction can be found in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, and a detailed description of the databases, web servers, and computational models can be found elsewhere.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> An elegant example by Kremer et al. that combines PDD with cheminformatic target identification demonstrates the potential utility of this approach to bypass the laborious target identification and validation process.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> With the advent of large data sets of well annotated biological activity such as PubChem, KEGG, and CheMBL that can be combined with proprietary data sets such as those available within pharmaceutical companies, it is anticipated that this will continue to be an area of considerable value to complement experimental approaches to target identifications. An equally important and useful component of target prediction engines is the information on the <i>mechanism</i> of drug actions, corresponding to the prediction results in terms of target proteins.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Adapted from Cambridge MedChem Consulting<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a> with Permission</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">online tool</th><th class="colsep0 rowsep0" align="center">similarity method/brief description</th><th class="colsep0 rowsep0" align="center">database</th><th class="colsep0 rowsep0" align="center">no. of targets</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://www.gdb.unibe.ch/" class="ext-link">Polypharmacology Browser (PPB)</a></td><td class="colsep0 rowsep0" align="left">This platform predicts possible targets for small molecules by searching for nearest neighbors using 10 different fingerprints describing composition, substructures, molecular shape, and pharmacophores. PPB searches through 4613 groups of at least 10 same target annotated bioactive molecules from ChEMBL and returns a list of predicted targets ranked by consensus voting scheme and p-value.</td><td class="colsep0 rowsep0" align="left">ChEMBL 21 2.7 million structures</td><td class="colsep0 rowsep0" align="center">4613</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref158'); return false;" href="javascript:void(0);" class="ref ref158">(158)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://www.pharmaexpert.ru/passonline/" class="ext-link">PASS Online</a></td><td class="colsep0 rowsep0" align="left">The approach is based on the suggestion that activity = <i>f</i>(structure). Thus, by “comparing” the structure of a new compound with structures of well-known biologically active substance, it is possible to estimate if a new compound may have a particular effect.</td><td class="colsep0 rowsep0" align="left">WDI and ACD</td><td class="colsep0 rowsep0" align="center">3678</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref159'); return false;" href="javascript:void(0);" class="ref ref159">(159)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://www.dddc.ac.cn/tarfisdock/" class="ext-link">TarFisDock</a></td><td class="colsep0 rowsep0" align="left">Searches for possible binding proteins for the given small molecule by use of a docking approach. The ligand–protein interaction energy terms of theprogram DOCK are adopted for ranking the proteins.</td><td class="colsep0 rowsep0" align="left">PDTD (potential drug target database)</td><td class="colsep0 rowsep0" align="center">841</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref160'); return false;" href="javascript:void(0);" class="ref ref160">(160)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://sea.bkslab.org/" class="ext-link">Similarity ensemble approach</a></td><td class="colsep0 rowsep0" align="left">Quantitatively groups and relates proteins based on the chemical similarity of their ligands. The similarity score between each set of ligands is calculated using ligand topology.</td><td class="colsep0 rowsep0" align="left">CHEMBL 16, WOMBAT, MDDR, and StarLite 65000 structures.</td><td class="colsep0 rowsep0" align="center">ref</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref161'); return false;" href="javascript:void(0);" class="ref ref161">(161)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://59.78.96.61/pharmmapper/" class="ext-link">PharmMapper</a></td><td class="colsep0 rowsep0" align="left">Aligns the target molecule onto each pharmacophore model of proteins in the target list, presents the aligned pose with the corresponding pharmacophore model and prioritizes candidate targets based on fit values to analyze the reverse mapping result.</td><td class="colsep0 rowsep0" align="left">TargetBank, DrugBank, BindingDB, PDTD</td><td class="colsep0 rowsep0" align="center">7000</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref162'); return false;" href="javascript:void(0);" class="ref ref162">(162)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://cpi.bio-x.cn/drar/" class="ext-link">DRAR-CPI</a></td><td class="colsep0 rowsep0" align="left">Docking scores of all the library drugs plus the uploaded drug towards all the targets are transformed into a matrix of Z′ scores.</td><td class="colsep0 rowsep0" align="left">PDB, DrugBank</td><td class="colsep0 rowsep0" align="center">385</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref163'); return false;" href="javascript:void(0);" class="ref ref163">(163)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://www.cbligand.org/TargetHunter" class="ext-link">TargetHunter</a></td><td class="colsep0 rowsep0" align="left">ECfp6, ECfp4, and Openbabel FP2. Predicts targets using Targets Associated with its MOst SImilar Counterparts(TAMOSIC) algorithm.</td><td class="colsep0 rowsep0" align="left">ChEMBL 22 and PubChem bioassay 117535 structures</td><td class="colsep0 rowsep0" align="center">794</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref164'); return false;" href="javascript:void(0);" class="ref ref164">(164)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://lilab.ecust.edu.cn/chemmapper/" class="ext-link">ChemMapper</a></td><td class="colsep0 rowsep0" align="left">Openbabel FP2, MACSS, SHAFT, and USR, 2D and 3D similarity. 3D similarity is driven by hybrid information of molecular shape and chemotype features.</td><td class="colsep0 rowsep0" align="left">ChEMBL 14, BindingDB, DrugBank, KEGG, and PDB 350000 structures.</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref165'); return false;" href="javascript:void(0);" class="ref ref165">(165)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://mips.helmholtz-muenchen.de/proj/hitpick" class="ext-link">HitPick</a></td><td class="colsep0 rowsep0" align="left">Combines two 2D molecular similarity based methods, namely simple 1-Nearest-Neighbor (1NN) similarity searching (2) and a machine learning method based on Laplacian-modified naive Bayesian models.</td><td class="colsep0 rowsep0" align="left">STITCH 99572 structures</td><td class="colsep0 rowsep0" align="center">1375</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref166'); return false;" href="javascript:void(0);" class="ref ref166">(166)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://modlab-cadd.ethz.ch/software/spider/" class="ext-link">SPiDER</a></td><td class="colsep0 rowsep0" align="left">Chemically advanced template search (CATS), topological pharmacophoredescriptor (SOM1), Molecular Operating Environment (MOE), physicochemical properties andindices (SOM2).</td><td class="colsep0 rowsep0" align="left">COBRA 12661 structures</td><td class="colsep0 rowsep0" align="center">251</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref167'); return false;" href="javascript:void(0);" class="ref ref167">(167)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://www.swisstargetprediction.ch/" class="ext-link">SwissTargetPrediction</a></td><td class="colsep0 rowsep0" align="left">Combines chemical and shape similarity.</td><td class="colsep0 rowsep0" align="left">ChEMBL 16</td><td class="colsep0 rowsep0" align="center">2686</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref168'); return false;" href="javascript:void(0);" class="ref ref168">(168)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://prediction.charite.de/index.php" class="ext-link">SuperPred</a></td><td class="colsep0 rowsep0" align="left">Compares 2D as well as 3D structures using a superposition algorithm. Extended-Connectivity Fingerprints (ECFP4) are applied for 2D and fragment similarity. A superposition algorithm is used for 3D similarity and represented by RMSD values.</td><td class="colsep0 rowsep0" align="left">ChEMBL, SuperTarget, and BindingDB, 341000 structures</td><td class="colsep0 rowsep0" align="center">1800</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref169'); return false;" href="javascript:void(0);" class="ref ref169">(169)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://202.127.19.75:5555/" class="ext-link">TarPred</a></td><td class="colsep0 rowsep0" align="left">K Nearest Neighbor (KNN)-based data fusion with molecular similarity searching.</td><td class="colsep0 rowsep0" align="left">BindingDB, 179807 structures</td><td class="colsep0 rowsep0" align="center">533</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref170'); return false;" href="javascript:void(0);" class="ref ref170">(170)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="http://potentia.cbs.dtu.dk/ChemProt" class="ext-link">ChemProt</a></td><td class="colsep0 rowsep0" align="left">Ligand-based similarity, target-based promiscuity, QSAR (quantitative structure–activity relationship) methodology and network biology-based enrichment analyses.</td><td class="colsep0 rowsep0" align="left">ChEMBL 19, BindingDB, DrugBank, PharmGKB, PubChem bioassay, WOMBAT, IUPHAR, CTD, and STITCH 1.7 million structures.</td><td class="colsep0 rowsep0" align="center">2140</td><td class="colsep0 rowsep0" align="center"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup><sup>a</sup></sup><p class="last"><a href="https://www.cambridgemedchemconsulting.com" class="extLink">https://www.cambridgemedchemconsulting.com</a>.</p></div></div></div><div class="NLM_p">Artificial intelligence (AI) and machine learning are two areas currently trending because of their potential to transform several facets of drug discovery including target identification. While there could be natural efficiency gains with these methods across a broad spectrum of drug discovery challenges, the promise of substantially improving target identification approaches is perhaps most compelling. Given the astronomical magnitude of scientific data that is being generated across the world both in terms of breadth and depth, it is naïve to assume scientists will be aware of new advances, let alone turn those diverse pieces of information into applicable knowledge without the computational-assisted ability to correlate, assimilate, and connect all this data.</div><div class="NLM_p last">A thorough discussion of the various AI algorithms and burgeoning companies developing this expertise is beyond the scope of this Perspective. Given the plurality of targets that most likely drive a phenotypic response, algorithms that connect this polypharmacology to a network of targets would significantly enhance the quest for highly validated biological pathways. Efforts on the automated design of ligands with predefined multitarget profiles represent an exciting step toward bridging the gap between pharmacology and the one-molecule one-target approach toward drug discovery.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In addition, high-speed and fidelity structure-based approaches to mapping the modulator landscape of nearest neighbors applied to phenotypic screening hits could provide a robust amplification of some of the target identification methods referred to in the following sections.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.2.  Target/Liability Assays</h3><div class="NLM_p">The most straightforward means for confirming the direct target of small molecules can be screening purported targets in a high-throughput manner to directly measure effect. Because a phenotypic screen generally provides a functional readout, a panel of candidate biochemical assays related to that function should theoretically provide direct information about the target. However, as can be seen in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, the practical number of targets that can be screened in this way is far less than the 100000+ potential targets in an organism (including splice variants and post-translational modifications)<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> or even the 500–1400 targets in the typical organelle.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Even the limited number of targets for which assays exist can quickly overwhelm the throughput of internal screening capacity or the financial resources necessary for contracting external companies that provide screening services (e.g., Eurofins DiscoverX, CEREP, or Promega). Also, although assays have been developed for a majority of enzymes comprising specific target classes, such as kinases or GPCRs, these best characterized targets are less likely to yield discovery of functionally novel small molecules. In contrast, panels of existing optimized HTS assays are unlikely to include truly functionally novel targets (e.g., orphan receptors, pseudoenzymes, regulatory elements) and would not be interrogated by this approach. Efforts toward efficient parallel screening of multiple targets using affinity-based techniques are discussed in <a class="ref internalNav" href="#sec4_6" aria-label="section 4.6">section 4.6</a>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Insufficiency of profiling panels to recapitulate the diversity of targets from a biological source. A well-designed phenotypic assay may identify small molecules that interact productively with any of a vast array of potential targets. Discrete profiling panels can only interrogate a small subset of the potential targets.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because of the technical limitations imposed by various screening formats, high-throughput screens for targets may not match the form of the target existing in the more biologically relevant tissue or organism. Unless the protein is directly obtained from a relevant biological source, it may have been obtained via heterologous overexpression in either truncated form to provide the relevant catalytic domain or with extra tags/reporters to allow subsequent high-throughput screening (e.g., affinity tags for target isolation/antibody detection or other reporters for visualization: fluorophores, enzymes, etc.). In addition, the actual format of the screen can play a significant role in the success of a profiling effort due to the choice of the assay strategy (e.g., activity-based versus binding). Chemical matter can impact a target by binding the inactive or pre/pro form of the target, which may be missed if an activity assay on the already active protein is used for screening. Conversely, a binding assay format would work with inactive targets, but only if such a binding assay exists for the inactive form of the protein. The more novel the target, the less likely a binding or activity-based assay will be available for screening at all.</div><div class="NLM_p">Ultimately, the success of profiling methods for target deconvolution depends on the target(s) of interest being present in the screening panel, using an assay that recapitulates the function that the phenotypic assay provides, and the target needs to be in a relevant form including appropriate cofactors at appropriate concentrations/ratios using an assay format that does not change the function of the target protein. However, if a target hypothesis can be derived by other means, confirming activity with a screening panel can provide valuable confirmation of direct interaction with that target and subsequently allow efficient screening of other chemical matter to provide the most desirable compound profile for further lead development.</div><div class="NLM_p">If the hypothetical targets fall within a target class well-covered by profiling screens, then direct assays can be quite useful for triaging screening hits for desired profiles. For example with kinase-based projects, we use a combination of internal screening via TR-FRET probe displacement assays and judicious screening via commercial panels for those kinases not in our routine internal panel. In this way, we are able to triangulate on-target activities of phenotypic screening hits and determine off-target liabilities for a subset of compounds produced by internal medicinal chemistry in order to correlate function with target profile and avoid targets with known liabilities.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Likewise, targets in other classes known to correlate with toxicity or other unwanted effects are screened via proprietary internal bioprofiling screening assays coupled with outside screens using CEREP panels for targets of known toxicities<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7">2.2.1.  Promiscuous Interactions: Tales from Microtubules and the Electron Transport Chain</h4><div class="NLM_p">While discrete protein interactions are, perhaps, the most logical entities to examine, effects of compounds on larger cellular structures are equally important. In a recent cell viability phenotypic screen, the identification of several colchicine analogues coupled with the observation of key gene expression (GEx) markers raised the specter of microtubule disruption-induced mitotic catastrophe as a principle mechanism for several hit compounds. To explore the prevalence of this phenomenon, 204 hits were examined in replicate at a single dose in an in vitro porcine model of tubulin polymerization. The results, shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, clearly demonstrate the pervasiveness of the antimitotic effect across the hit set. The majority of the compounds examined in the assay triggered a decrease in the rate of the microtubule growth phase. In many of these cases, the effect may be considered minor, producing a less than 30% reduction in growth rate and still achieving maximal polymerization over the assay duration. It should be noted, however, that orthogonal studies have observed that tubulin-driven antimitotic effects observed at nanomolar compound levels in cells require micromolar compound quantities to recapitulate the effect in vitro. Regardless, it is conspicuous that nearly one-third of the compounds assayed produced a significant reduction in the rate of microtubule assembly and this effect was not confined to privileged chemotypes. Nearly every structural cluster shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A possessed analogues that were antimitotic. It should be emphasized that many of the original hits from our viability screen originated from nononcology programs and supports monitoring this activity during SAR campaigns. Perhaps the most striking observation is that relatively minor structural changes, such as those shown in the top two panels of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, may result in dramatic differences in tubulin polymerization. In the case of hits <b>1</b> and <b>2</b> in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, conversion of an internal cyclopropane ring to an alkene and/or halogen addition alters the tubulin polymerization profile from a colchicine-like destabilization affect to one that looks more like a taxol-like tubulin stabilizing affect (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). This would suggest that during the process of optimizing a structure–activity relationship, either in conventional or in phenotypic drug discovery processes, alteration in tubulin binding characteristics may obfuscate assay results, potentially providing misleading target validation outcomes. Indeed, this appears be the case with purported inhibitors of MuT homologue <b>1</b> (MTH1) TH287 and TH588. In assessment of this target, we were unable to correlate enzyme knock-down/-out or enzyme inhibition with an antiproliferative phenotype. However, we were able to establish that cell growth inhibition was only observed for SMs that had measurable inhibition of microtubule polymerization. This observation was supported by a recent publication, concluding that these compounds are likely affecting cancer cells through their tubulin-driven antimitotic effects.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effects of structure on microtubule polymerization. (A) Effects of 204 compounds derived from a growth inhibition phenotypic screen on microtubule polymerization in vitro. Structural classes are denoted by color and linked together by lines. The shaded area denotes changes in microtubule assembly that are relatively weak, whereas compounds falling outside this range have a strong effect. The change in growth phase rate was calculated by taking the slope of the linear regression of the growth phase (5–15 min) and comparing it to the slope of the control. Data for the compounds reported in (B) and (C) are highlighted in red. (B) Structures of compound <b>1</b>, compound <b>2</b>, colchicine, and taxol. (C) Two representative compounds from the phenotypic screen were profiled for their ability to modulate microtubule assembly in vitro. At 10 μM, compound <b>1</b> (top left) reduced microtubule growth in a manner consistent with a known destabilizer, colchicine (bottom left). Conversely, a closely related analogue, Compound <b>2</b> (top right), may have had a growth enhancing affect similar to a known stabilizer, taxol (bottom right). DMSO (blue), 10 μM Compound (red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The mitochondria also represent another major source of proteins that bind and are functionally affected by numerous chemotypes. One of the more notable examples originates from electron transport chain (ETC) proteins (complex I–V). In fact, many compounds, including marketed drugs (e.g., troglitazone), inhibit more than one of the ETC complex proteins. In a study by Pfizer, it was noted that many pharmaceutical agents that resulted in mitochondrial dysfunction also had a propensity for unfavorable physiochemical properties.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In the case of oncology programs where the desired endpoint is cell death, ETC inhibition can be confounding, especially if this activity is a component of a SM’s polypharmacology. For example, inhibitors of complexes I and V result in a benign phenotype across a broad range of primary cell types. Although one can test these molecules for the “Crabtree effect” using a “glucose-galactose” assay to inform on ETC impingement, it is usually only predictive when this is the main activity of the SM.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> This assay involves culturing HepG2 cells in galactose as an energy source to make the cells sensitive to the ATP depletion that is induced by ETC inhibition. A more conclusive measurement involves direct assessment of mitochondrial respiration, but this assay usually has a much lower throughput. In addition to routine assay panels, hits and leads from TDD and PPD programs should be routinely assessed for activity against microtubule and mitochondrial ETC proteins. Beyond these mechanisms, broader activities leading to mitochondrial dysfunction also need to be measured during SAR campaigns.</div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.3.  Broad Profiling with a Companion Reference Database</h3><div class="NLM_p">In addition to the aforementioned panels of enzyme and direct mechanistic assays, methods that are more target agnostic yet, through a broad survey at the gene, protein, or signaling level, offer valuable insight to either phenotypic screening hits or chemical series from a TDD campaign. Comprehensive readouts such as genome-wide transcription often suffer from the complexity of data-driven reduction of “long gene lists to short gene lists”. One approach we regularly employ to utilize these complex data sets is to compare profiles generated by our experimental SMs to reference databases of profiles associated with well-characterized mechanisms. L1000 GEx profiling (Genometry, Inc., Cambridge MA) and the BioMAP platform (Eurofins DiscoverX, Fremont, CA) are two commercial offerings that we have utilized in our PDD and TDD programs, in part because of their strong reference database offerings. In the case of phenotypic screening hits, utility is provided by both positive and negative results. For example, SM effects that show strong correlation to database profiles of ciclopirox, thapsigargin, or bortezomib could indicate common MOAs such as metal chelation, endoplasmic reticulum (ER) stress, or proteasome inhibition, respectively. Conversely, the lack of a correlation to reference molecules may strengthen the case for a SM possessing a unique MOA, particularly when complementing results from orthogonal methods. The following discussion will detail our recent advances utilizing these approaches and internal case studies of successful implementation.</div><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9">2.3.1.  The L1000 Platform</h4><div class="NLM_p">In a landmark study by Lamb et al., the concept of connecting human disease, genes, and SMs/drugs that modulate biology was elegantly demonstrated in a project termed the “Connectivity Map” (CMap).<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> A database of μArray gene expression profiles for 164 SM perturbagens across three cell lines was generated and mined using publicly available pattern-matching software. The success of initial case studies provided the impetus to greatly expand CMap as a community resource project, but the genome-wide μArray profiling was cost prohibitive at the necessary scale. The L1000 platform effectively solved this challenge by establishing a high-throughput reduced-representation GEx analysis.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> This platform, matured by the Broad Institute and recently commercialized by Genometry (<a href="http://genometry.com" class="extLink">http://genometry.com</a>), requires crude human cell lysates which are subjected to a multiplex ligation-mediated amplification assay in 384-well plates. The L1000 assay directly measures 978 probes representing 1000 Landmark (LM) genes, and computational inference predicts correlated effects on the remaining portion of the full transcriptome.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Over a decade after the initial description of CMap and many published success stories, the L1000 technology platform has fueled the NIH Library of Integrated Network-Based Cellular Signatures (LINCS) program to include >1.3 million publically available L1000 profiles.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> These profiles were generated from 42553 perturbagens that include 19811 small-molecule compounds, 18493 shRNAs, 3627 cDNAs, and 622 biologics. The cell line coverage was also increased from the original three to a core set of nine cell lines, in addition to select data across a variable number of cell lines (3–77) for SMs with unknown MOAs.</div><div class="NLM_p">The enormity of this publicly available data has prompted a number of studies to address study design, data quality control, and analysis pipelines.<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29−31)</a> In a recent example, deep learning techniques converted 978-dimensional continuous L1000 data for 3699 SMs into 100-dimensional binary barcode representations.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The barcode captures SM structure and target information and predicts HTS promiscuity to a greater degree than the original data measurements. This model demonstrated high performance, captured underlying biology, and predicted the targets of structurally diverse SMs with unappreciated MOA. This and the other studies emphasize the need to enhance the quality of data outputs from noisy, large-scale data as a necessary step prior to integration with orthogonal technologies for hypothesis generation.</div><div class="NLM_p">It is intriguing to speculate how the available L1000 data may be uniquely exploited for probing biological complexity beyond the 2D cell cultures from which they were derived. In a recent report, Senkowski et al. investigated SM sensitivity for tumor cells in 3D culture.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Their study was based on a previous observation that quiescent cell spheroids have reduced sensitivity to cytotoxic SMs but acquired sensitivity to inhibitors of oxidative phosphorylation (OXPHOS). HCT-116 tumor cells in 2D culture, proliferating spheroids, and quiescent spheroids were treated with a dose escalation of OXPHOS inhibitors. Gene sets consisting of the top 30 regulated LM genes revealed dose-dependent and selective upregulation of mevalonate pathway genes for the quiescent spheroids. Moreover, cotreatment of the quiescent spheroids with OXPHOS and mevalonate pathway inhibitors (e.g., simvastatin) resulted in synergistic cell killing. In addition to revealing actionable mechanistic insights, the authors also emphasized the value of monitoring GEx changes over a dose–response, both to capture salient changes and to increase the confidence of observed modulations. This concept was originally reported using μArray data and stressed by the authors as a viable approach to capture polypharmacology.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The minimal use of this methodology has likely been due to the cost-prohibitive nature of μArray platforms. Dose–response (DR)-L1000 profiling provides an opportunity to rigorously assess this strategy, and we will describe both TDD and PDD examples of successful implementation.</div><div class="NLM_p">As described in <a class="ref internalNav" href="#sec2_2" aria-label="section 2.2">section 2.2</a>, even the most subtle modification can transform a chemical series into a potent MTI. For oncology TDD programs, a large boost in the cellular potency with an absence of a comparable enhancement of the in vitro binding or activity assay warrants direct assessment of the SM in an MTI assay. However, on numerous occasions, we have observed activity disconnects that are not straightforward. Our general approach to inform on SM MOA is to expose cells in a 384-well plate to a short-term compound treatment (6 h) and then monitor DR-L1000 profiles (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). As an example, two chemical series from a recent oncology medicinal chemistry TDD campaign were essentially indistinguishable when assessed by in vitro target-binding assays. While the compounds were equipotent against most cell lines using an ATP quantitation assay (CellTiter-Glo, Promega Corporation) and cellular confluence, other lines were identified that exhibited significant potency differences (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). If the enhanced activity could be attributed to an off-target that did not directly impact the biology of interest, then this information may be useful for series (de)prioritization. To start to address this disconnect, we wanted to understand if the cell growth inhibition was directly correlated with an on-target cellular response. However, interpretation was hampered by the lack of a robust cellular PD marker. Using DR-L1000, a cluster of target-related genes were identified that strongly modulated compound <b>2</b> compared to <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C, cell line A), concomitant with the observed tumor cell line growth inhibition (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, top). Several of these genes have well established mechanistic connections to target biology while others have not been previously described. Although these data do not specifically identify the target/MOA of the cellular activity disconnect, they reveal that both SM activities are driven by a mechanism that modulates the on-target biology. Follow-on studies include confirming cellular target engagement and correlating SM potency and GEx profiles to generate hypotheses for candidate targets driving the observed cellular activities.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Dose–response L1000 (DR-L1000) to address SM cellular potency differences. (A) Strategy for data collection and integration. (B) CellTiter-Glo (CTG) ATP/viability assessment indicating compounds <b>1</b> and <b>2</b> exhibit significantly different potencies when profiled in cell line A but are equipotent in cell line B. (C) Heat-map for a gene set that correlates with cell line potencies. Cell lines 1 and 2 were treated with increasing doses (5 nM to 10 μM) of each compound. Red and blue represent increases and decreases in GEx, respectively. The extent of regulation (fold-change) varies directly with the color intensity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">DR-L1000 has also provided significant value for our PDD campaigns. Ideally, a screen is designed with the “phenotypic rule of 3” as a guiding principle and focuses on the physiological relevance of the system, stimulus, and readout.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> For some biological questions of interest, it is not always possible to fulfill all of these criteria, so strategies are needed to address critical gaps. For example, we recently completed a screen using a primary cell system, patient relevant stimulation, and reporter assay readout. Although individual reporter assay end points have been considered to be of lower relevance, a recent demonstration of molecule phenotyping (MP) using GEx signatures has demonstrated great promise.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> We also rationalized that augmenting the reporter assay hits with DR-L1000 profiles would provide hit confirmation and possibly define a “GEx consensus signature” of target pathway inhibition. For a series of screening hits that did not modulate any targets when assessed in binding and activity assay panels, we compared the reporter assay IC<sub>50</sub> plots (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, left panel) to DR-L1000 data of genes affected by compound treatment (an example of one gene is shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, right panel). This comparison reveals an exceptionally strong correlation between inhibition of the pathway reporter and specific GEx effects and thus demonstrates how GEx effects can reflect inhibitory potency of individual compounds. The broad end points provided by L1000 also have the potential to differentiate compounds through induced effects outside the common target response. For example, a heat-map comparing a gene set for six SMs (three medium and three high potency inhibitors) revealed common perturbations suggestive of target inhibition (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, bottom) but also highlighted genes that were affected only by one inhibitor class, suggestive of two distinct MOAs (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, top). Many of the monitored genes shared established connections to the pathway biology of interest, but unappreciated mechanisms were also uncovered. DR-L1000 also provided value in compound characterization by generating dose–response profiles that segregated into discrete potency bins. In <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C, we plotted three genes related to the phenotypic end point. The most potent response (gene 1) was for a transcription factor coregulator, followed by an established genomic target of the transcription factor (gene 2). Gene 3 is consistent with a general stress response with established association to the pathway and either originates from a cellular feed-back mechanism or represents polypharmacology of the SM. Although this latter interpretation has been emphasized as the main value for DR-GEx,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> we have been accruing evidence that the engagement of a single target may manifest multiple response profiles that inform on complexity of SM MOAs. In the absence of the broad dose–response chosen, the actual gene regulation would not be appreciated. For example, profiles from a single high dose would lead to the conclusion that the three genes were minimally regulated (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Reporter assay EC<sub>50</sub> values (left) and corresponding DR-L1000 plots for a representative gene (right). (B) (Top) gene set in common with two distinct MOAs of pathway inhibition; (bottom) gene sets distinct for two different MOAs. (C) Dose response for multiple genes originating from an individual SM treatment</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10">2.3.2.  The BioMAP Platform</h4><div class="NLM_p">BioMAP systems are comprised of human primary cell-based models and represent a diversity of tissue and disease models. Primary cells alone or in coculture are treated with factors and stimulated to mimic the complexities of disease biology. Panels of protein markers, many with established clinical relevance, are measured to generate compound-specific signatures. The signatures are then compared to a reference database of SMs and biologics to reveal correlation and thereby inform on MOA. The process is summarized in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A. Establishment of the platform and application to SM MOA has been described in detail.<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) BioMAP profiling platform. (B) Process for early triage of large sets of compounds derived from TDD or PDD campaigns.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Our early use of this technology was mainly limited to differentiation of later-stage clinical candidates or comparison to reference molecules. The desire to profile hits from PDD at an earlier stage was predicated on the large reference database (>4500 test articles) and strength of this approach for identifying common MOAs. However, our initial access to this technology for phenotypic screening hits was restricted by the cost per data point and lack of strong evidence to support the quality of these early leads. Ironically, we argued that broad endpoint profiling with a companion reference database was part of the foundational data needed to understand and prioritize hits for target deconvolution.</div><div class="NLM_p">In 2016, we profiled 12 compounds originating from three PDD campaigns (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In the case of PDD-1, we generated BioMAP profiles for two molecules prioritized from a large Cancer Cell Line (CCL) profiling screen. Previous data for both molecules supported unprecedented mechanisms of tumor cell growth inhibition. The lack of a strong database correlation supported the unique MOA hypotheses and reinforced the decision to pursue additional target deconvolution approaches. BioMAP data for compounds derived from PDD-2 and PDD-3 identified direct targets for the lead SMs. Overall, data from this profiling platform significantly impacted the chemistry strategy for chemical probe generation.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. BioMAP Profiling of PDD Hits<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">SM source</th><th class="colsep0 rowsep0" align="center">strong database match?</th><th class="colsep0 rowsep0" align="center">direct target hypothesis?</th><th class="colsep0 rowsep0" align="center">confirmation of reference SM in screening assay?</th><th class="colsep0 rowsep0" align="center">confirmation of direct target?</th><th class="colsep0 rowsep0" align="center">impact</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDD-1a</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="left">Novel target identified by other approaches; possible connection from the BioMAP MOA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDD-1b</td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="left">SM determined to be a pro-drug using other approaches</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDD-2a</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="left">MOA not desirable; halted a significant chemical biology effort to synthesize target deconvolution probes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDD-2b</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="left">MOA not desirable; halted a significant chemical biology effort to synthesize target deconvolution probes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDD-2c</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y, multiple</td><td class="colsep0 rowsep0" align="center">DNP</td><td class="colsep0 rowsep0" align="center">DNP</td><td class="colsep0 rowsep0" align="left">Probe chemistry was deprioritized</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDD-3a</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="left">MOA undesirable; halted a significant chemical biology effort to synthesize target deconvolution probes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDD-3b</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="left">Uncovered unique aspects of precedented target space</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDD-3c,d,e,f</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="left">Distinct chemical matter with the same, undesirable MOA; halted a significant chemical biology effort to synthesize target deconvolution probes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDD-3g</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="center">Y</td><td class="colsep0 rowsep0" align="left">Confirmed known target</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">PDD, Phenotypic Drug Discovery Screen; NA, not applicable; DNP, did not pursue</p></div></div></div><div class="NLM_p last">It is clear that PDD and TDD campaigns can benefit from characterizing and prioritizing leads using the BioMAP platform. To increase access to the technology, a process has been developed that provides a more cost-effective mechanism for prioritizing larger numbers of SMs for downstream analyses. <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B outlines the introduction of an Early Triage Panel (ETP) and Diversity Screen (DS).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The ETP reduces the breadth of experimental outputs measured to expand throughput and focus on a central goal to deprioritize compounds with overt cytotoxicity to primary human cell types. Then, the DS stage enables a program-specific analysis across 12 BioMAP cellular systems (148 end points) that captures the total number of activities, cytotoxicities, and scaffold-specific biology and provides flags for key toxicity alerts. The ability to query the reference database or test for mechanism classification<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> may improve the usefulness of DS analysis, although because the DS uses only one concentration of test article, the result could be less accurate. Importantly, individual assays and endpoints that prove informative from the BioMAP systems may be implemented as counter-screens during lead optimization. Overall, this approach should improve the quality of chemical matter that is progressed and build institutional knowledge regarding chemical cores in screening libraries.</div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">2.4.  CRISPR: Small-Molecule Positive Selection Genome-Wide</h3><div class="NLM_p">The ability to enable protein recruitment to genomic sites of interest through application of CRISPR systems<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> has greatly expanded our capabilities in target identification. Forward genetic screens that segregate cell lines or samples by an observed phenotype in order to contrast differences in genetic sequence or expression have an established track record for application in target identification.<a onclick="showRef(event, 'ref6 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref6 ref41 ref42">(6,41,42)</a> An example of the general utility of CRIPSR targeted knockouts was shown in validation of a forward genetic screen that consistently identified <i>HPRT</i> mutations in cells that spontaneously acquired resistance to the cytotoxic effects of 6-thioguanine (6-TG).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> To confirm that these mutations were causative, CRISPR reagents targeting <i>HPRT</i> coding exons were utilized and shown to generate 6-TG resistance in the parental line through gene disruption. However it is important to note that targeted genetic perturbations will not always parallel a chemically induced phenotype. Discrepancies could arise from genetic perturbations causing complete loss of a protein versus small-molecule treatment inhibiting just one facet of that protein’s total activity, e.g., inhibiting enzymatic activity while preserving scaffolding functions. Also, targeted knockouts via CRISPR will have specificity based on nucleotide sequence, whereas small-molecule engagement will be driven by protein structure and conformation. Therefore a small-molecule-induced phenotype driven by polypharmacology or uncharacterized off-targets will not be fully recapitulated by genetic perturbation of a single gene. This aspect could prove a liability in instances where highly conserved genes express functionally redundant proteins and genetic targeting of a single gene will not elicit a phenotype associated with a small molecule that engages all conserved protein targets. However this single gene specificity of CRISPR targeting can also be exploited to facilitate target identification, as elegantly demonstrated in a recent publication that established chemical-induced cytotoxicity in basal breast cancer lines was not related to MELK activity or protein levels.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><div class="NLM_p">We have expanded from the functional test at a single gene level to essentially generate a shortcut to acquired resistance by pairing available genome-wide guide RNA (gRNA) libraries with phenotypic selection (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). A key benefit of this approach is the unbiased nature of the pooled libraries. Therefore, results are not restricted to pre-existing hypotheses. This use of pooled lentiviral CRISPR screening presents a shortcut through two key features. First, genes are systematically disrupted via CRISPR targeting, as opposed to relying on mutations that have spontaneously occurred in the population. Second, the integrated targeting gRNA construct can be sequenced and serve as a surrogate identifier of the gene disruption that occurred in a cell that survived selection. This direct readout of individual targeted perturbations avoids the significant challenge of a global RNaseq or DNaseq search for the key mediators of resistance among a pool of downstream or passenger effects.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Schematic that outlines pooled CRISPR screening paired with positive selection. A library of diverse sgRNAs is used to transform a clonal cell line into a pool of single gene targeted disruptions. Positive selection enriches mutant cells that acquired a novel trait distinct from the unmodified parental line and directly related to that particular selective challenge. Sequencing identifies targeting constructs as a surrogate of mutant genes in the enriched cell population.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The utility of this approach in target ID was shown in the initial demonstration of genome-wide pooled CRISPR screening in human cells, which revealed several components of the mismatch repair pathway as additional mediators of 6-TG sensitivity and genes required for etoposide cytotoxicity, <i>TOP2A</i> and <i>CDK6</i>.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div><div class="NLM_p last">We have integrated pooled CRISPR screening into our target identification technology stack as a tool capable of profiling SMs of unknown MOA or lacking well-characterized targets. A recent example of the value provided by this technology occurred in our characterization of phenotypic screening hits that inhibit a disease-specific signaling pathway. This pathway is fairly typical with signaling activation transmitted through successive phosphorylation events, but the lead compounds showed no inhibitory activity when profiled against a kinome panel. The initial phenotypic screen was also performed in a primary human cell line with pathway activation mediated by exogenous stimulus. However, we capitalized on the range of mutations and disrupted signaling networks that occur in human cancers. We identified clonal cancer cell lines with constitutive activation of this same signaling network but driven by intrinsic mutations as opposed to external stimuli. Transcriptional profiling established commonalities between the primary cell model and these cancer lines, supporting their use as a surrogate model. The proliferation capacity and effectiveness of viral transduction in this cancer cell model allowed us to apply genome-wide screening. Importantly, these cancer cell lines also exhibited addiction to this pathway for maintained viability and proliferation. A clonal cell culture model with constitutive activation of the pathway was chosen for the screen because compound treatment induced cell death, presumably as a direct consequence of pathway inhibition. This compound-associated selection was applied and enriched cells with CRISPR-targeted gene disruptions that conferred resistance to pathway inhibition. We hypothesized that particular gene disruptions that alter the cellular response to compound treatment would inform our target hypotheses. This approach may not directly identify the precise target of a molecular inhibitor because disruption of the target protein is expected to phenocopy the small-molecule effect as opposed to creating resistance. However, the diversity of pathway inputs and regulation that exist in biological systems create the potential to reveal nodes and regulators proximal to the direct target of a small-molecule inhibitor. This particular instance revealed gene disruptions that strongly focused on mitochondrial functions and specifically components of the oxidative phosphorylation pathway. This prompted follow-up with immunopurified enzyme assays that validated direct inhibitory activity against an enzyme complex that was not previously linked to the biology under investigation. Therefore, this case demonstrated how pooled CRISPR knockout screening can be used to reveal MOA of uncharacterized SMs and establish novel biological connections. Additional studies have similarly applied this approach beyond proof of concept and employed the technology to clarify the target mechanism of rigosertib<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> or detail mechanisms of cytotoxicity associated with the antiviral compound GSK983.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> In the examination of GSK983 cytotoxicity, a genome-wide shRNA screen identified the direct inhibitory target, DHODH, whereas the parallel CRISPR knockout screen revealed distinct but related gene targets. This difference highlights how the particular perturbation used can affect the experimental results. Targeted mutations generated by CRISPR are more likely to yield complete loss of function alleles, whereas shRNA transfection is more likely to result in a hypomorph phenotype or partial reduction in activity. More specifically, viral transduction of CRISPR reagents yields a pool of cells with heterogeneous indel mutations. Many of these indels completely disrupt protein production or activity, but a fraction of nucleotide mutations will retain functional expression and protein activity. In contrast, transduction of shRNA reagents may not completely eliminate protein production and subsequent activity, but a more uniform effect could be observed from cell to cell across the total population. In screens where selection is possible, the potential phenotypic variance associated with mixed CRISPR indels may be desirable. We initiate these screens with enough cells and virus so that each library gRNA is represented in at least 100 cells at the initial infection. At this level of representation, several distinct mutational outcomes and allelic combinations should be generated for the downstream phenotypic selection. Analysis will identify particular gRNAs and gene targets prioritized in the screen, and validation efforts could ascertain whether particular mutation profiles were required for the phenotypic effect, e.g., loss of a single allele for partial knockdown. We also seek to maximize potential impact of the data generated by performing parallel phenotypic selections with different levels of stringency. Situations where strong viability selections (maximal growth inhibition) are applied may identify a small number of genes required for cytotoxic effects, but lower concentrations of compound (IC<sub>20</sub> or IC<sub>80</sub>) may reveal additional genetic interactions such as gene disruptions that sensitize to compound-induced cytotoxicity or confer partial tolerance. The total number and diversity of genetic perturbations that significantly alter cellular response to compound are unpredictable, especially when MOA is unknown, thus it can be beneficial to plan multiple selection concentrations for an initial screen. The example given above relies on compound cytotoxicity as the experimental means for positive selection of resistant cells. However, alternative means of cell isolation should also be feasible with pooled screening approaches such as FACS enrichment of live cell populations based on levels of fluorescent reporters or cell surface markers. This approach, two serial rounds of FACS enrichment based on GFP reporter activity or staining of cell surface markers, was elegantly used in genome-wide screens that identified host factors required for HIV infection or regulators of PD-L1 expression in tumors, respectively.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> To further broaden the utility of the CRISPR system, a number of enzymatically inactive Cas9 variants, e.g., dCas9-KRAB or dCas9-VP64, have been developed, which enable modulation of transcription levels as an alternative to gene disruption.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> Which of these particular systems is best suited to uncovering unknown MOA is unlikely to be consistent across all scenarios, but each pooled screening approach has strong potential to guide toward the target biology due to the unbiased genome-wide interrogation.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">3.  Target Enrichment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06015" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06015" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">3.1.  Chemical Probe-Based Target Enrichment: Intracellular Targets</h3><div id="sec3_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15">3.1.1.  Considerations for Probe-Based Target Identification</h4><div class="NLM_p">The resurgence of phenotypic screening as a viable and orthogonal drug discovery platform has been accompanied by renewed interest in target identification strategies. Across the academic and industrial research sectors, considerable time and investment has been poured into target identification technologies, both novel and established. Because many of these techniques leverage the use of chemical probes, it is prudent to first consider the challenges involved in their design and implementation.</div><div class="NLM_p">When contemplating the development of a chemical probe from a lead molecule, the predominant challenge entails the effective tethering of a reporter component while avoiding the introduction of unwanted negative characteristics. Common concerns include the disruption of the primary drug–target interaction and potential changes in physical–chemical properties including changes in solubility, nonspecific protein binding, cell permeability, and probe localization. The judicious choice of linkers may mitigate these effects For example, a short polyethylene glycol (PEG) linker terminated with an azide (−N<sub>3</sub>) or Boc-protected amine (−NHBoc) can maintain the solubility of the probe, minimize nonspecific protein binding, but result in some loss of cell permeability (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) as evidenced in multiple cell based assays (data not shown) as long as the position of attachment is not critical for target binding. When desired, the azide/Boc-protected amine is converted to a primary amine and reacted with an application-specific reporter (e.g., solid supports, fluorophores, biotin, expression tags, anionic permeability modifiers, etc.).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. General schematic describing linked compounds for target ID/validation. (A) Linked versions of active compound via multiple vectors in order to produce phenotypically active intermediates. Site of linking determined by combination of compound–target structural information, SAR, and chemical tractability. (B) A few representative chemistries for introducing linked functionality. (C) Conversion of linked intermediates into target ID/validation probes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">While inclusion of a suitably long linker possessing desirable chemical character is important, it is also imperative that it is attached to a functionally benign vector on the compound of interest. Compounds whose targets are well elaborated may benefit from the application of structural or modeling data to facilitate probe construction. In the absence of this information, available structure–activity relationship data can be used to conduct a hypothesis-driven approach to probe design. Alternatively, the chemistry of certain hits may afford the facile attachment of linkers, which may be subsequently examined in activity assays for their effect. Such attachments may be opportunistic, based on the nature of the hits, or engineered during synthesis of the compounds. Advances in C–H activation chemistry may provide additional routes to attachment not yet realized.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><div class="NLM_p last">A more comprehensive approach to ligand attachment has been developed by Kanoh et al,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and extended by Nishiya et al.,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> whereby a heterobifunctional linker with a photoprobe on one termini and either affinity resin or an amine reactive functional group on the other termini is first photocrosslinked to the compound of interest. This creates a population of cross-linked compounds that is then immobilized on affinity resin and used for chemoprecipitation as described below. While this is very efficient from a workflow point of view relative to designing and making specific probes and would be very useful for difficult to synthesize natural products, it suffers from several challenges. Most notably, reproducing and characterizing the cross-linked pool would be difficult as a variety of products are obtained and testing to be sure the mixture maintains activity would prove difficult or impossible, so failure to identify a target would be difficult to interpret. In our experience, the ability to be sure the linked probe retains activity is crucial, and to date we have only rarely been unable to deliver a linked probe that maintains some level of the parent drugʼs activity.</div></div><div id="sec3_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16">3.1.2.  Affinity Capture “Chemoprecipitation”</h4><div class="NLM_p">The most common application of chemical probes for the purposes of target identification entails their use as “bait” to capture and enrich cellular entities to which they have a reasonably strong affinity. Usually, the probes are conjugated to a solid bead-based support comprised of sepharose or agarose, which is then incubated in a cell or tissue homogenate. However, several permutations to the technique exist including the use of arrays or conjugation of a soluble affinity tag such as biotin or FLAG. After incubation in a biological matrix, the targets are recovered through the collection and washing of the beads. This process mirrors that of immuno-affinity precipitation (immunoprecipitation) and therefore may be considered to be chemical-affinity precipitation or chemoprecipitation.</div><div class="NLM_p">Chemoprecipitation, or affinity capture-mass spectrometry (AC-MS), is a popular target identification technique owing to its ease, scalability, and direct interaction with the cellular targets. Despite these advantages, chemoprecipitation suffers from many limitations both technological and biological. Most obvious is the potential inability to create an affinity probe that retains its target binding capability, either due to the complexity of the binding mode or synthetic limitations. Lack of target binding may also be driven by the metabolism of the compounds into active yet unidentified forms, obviating the ability to create a competent chemoprecipitation reagent. Biologically speaking, chemoprecipitation has been limited to cell and tissue homogenates due to a lack of probe permeability. The decompartmentalization, dilution, and pH normalization experienced upon lysis may result in target denaturation, binding partner dissociation, and proteolysis among other undesired events. Recently, ligation of a chloroalkane tag to compounds has shown promise for target enrichment using Promega’s HaloTag system. Because the tag permits the probe to retain cellular permeability comparable to parent compound, in-cell chemoprecipitation is fast becoming a reality.<a onclick="showRef(event, 'ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref55 ref56 ref57">(55−57)</a></div><div class="NLM_p last">Given the considerations, it is expected that success rates will vary depending upon the nature of the screen, the types of targets being pursued, the design of the compound library, and the differential allocation of resources to target identification efforts between research institutions. Implementation of additional techniques such as the use of quantitative mass spectrometry have improved success rates.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> However, the enhanced value of chemoprecipitation-based data sets is realized when paired with orthogonal target identification technologies.</div></div><div id="sec3_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17">3.1.3.  Photo-Affinity Labeling Enrichments</h4><div class="NLM_p last">A key failing point of chemoprecipitation experiments is poor target affinity or denaturation of the target upon cell lysis. This limitation can be offset through the introduction of a covalent bond between the probe and its target while the cells remain in culture. A popular technique to achieve this objective while minimizing nonspecific covalent labeling of the biological matrix is photoaffinity labeling (PAL). Photoaffinity labeling uses an analogue of a biologically active small molecule that bears both photoreactive and reporter functional groups.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The photoaffinity probe is added to proliferating cells and irradiated with UV light. Irradiation of the photoreactive group generates a reactive chemical species (e.g., carbene, nitrene, or radical) that covalently cross-links the photoaffinity probe to its binding partner(s) based upon the proximity of the two. Photocross-linked protein targets are then visualized by the reporter group (e.g., fluorophore, biotin, or radioactive label). There are several photoreactive functional groups frequently used in PAL (e.g., benzophenones, diazirines, aryl azides, and acyltetrazoles).<a onclick="showRef(event, 'ref59 ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref59 ref60 ref61">(59−61)</a> Recent advances in small-molecule bioimaging techniques have made it possible to further couple such an “in situ profiling” approach with imaging of drug uptake and subcellular distribution.<a onclick="showRef(event, 'ref62 ref63 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref62 ref63 ref64 ref65">(62−65)</a> In a recent advance, coupling PAL with in situ screening and target discovery, Cravatt and colleagues describe an approach that marries fragment-based ligand discovery with quantitative chemical proteomics to map thousands of reversible small-molecule–protein interactions in human cells.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> This approach of fragment-based PAL was combined with phenotypic screening to identify small-molecule ligands for a poorly characterized membrane protein, PGRMC2. While PAL has the potential to mitigate deficiencies in conventional chemoprecipitation approaches, it also contains several challenges that may reduce its effectiveness. In particular, the rapid quenching of the activated photochemical intermediate often results in low cross-linking efficiency. This necessitates the design of a probe that positions the photochemical warhead in close proximity of the protein binding partner(s). When the target is elaborated, photoaffinity probe design may be facilitated thorough existing structural data and/or computational modeling. However in the absence of de novo target information, photoaffinity probe design may become particularly onerous. For target ID, therefore, probe design involves a significant level of empirical assessment and is advocated as a complement to other techniques rather than a single approach.</div></div><div id="sec3_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18">3.1.4.  Activity-Based Protein Profiling</h4><div class="NLM_p">For situations where molecular probes for chemoprecipitation or photoaffinity labeling cannot be designed, activity based protein profiling (ABPP) offers an alternative. Activity-based protein profiling is a target identification technique that entails screening compounds against probes that have been developed to interact in a functionally dependent manner with broad protein classes including proteases, kinases, and others.<a onclick="showRef(event, 'ref67 ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref67 ref68 ref69">(67−69)</a> Because ABPP requires a functional set of targets for enrichment, it can discriminate between inactive pro-enzymes and cleaved functional entities (e.g., hydrolases/proteases),<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> as well as proteins activated by post-translational modification. ABPP reagents are comprised of a reporter, a short spacer, and a warhead capable of covalently binding to a structural motif shared between active members of a protein class. Early forms of ABPP used biotin or fluorophore reporters (see <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>).<a onclick="showRef(event, 'ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref71 ref72">(71,72)</a> Because these probes suffered reduced cell permeability, they were applied to cell and tissue homogenates after dosing the drug candidate of interest. The targets of the dosed compounds were subsequently inferred by loss of ABPP reporter signals (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). In more recent applications, the ABPP platform has been adapted into a two-stage process. After dosing a compound of interest, an ABPP probe with a bioorthogonal handle (e.g., alkyne, azide) that maintains cell permeability is applied to cells in culture.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Following reaction with its targets in situ, the cells are lysed and treated with a reporter molecule attached to a bioorthogonal handle that pairs with the ABPP probe. This process allows both compound and ABPP probe access to targets in live cells and even in vivo, an important advantage among most target ID methods.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Schematic overview of activity based protein profiling (ABPP). Cells or tissue are dosed either with vehicle (DMSO) or compound at desired doses. Following abstraction of the target milieu, ABPP probes are applied to covalently bind to open active sites. The samples are then visualized by a reporter.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A common application of ABPP is profiling compounds for their off-targets and quantifying target engagement to the desired target. As an example, Ahn et al. were able to use a fluorophosphonate ABPP to demonstrate that their FAAH inhibitor, PF-3845, was highly selective versus other serine hydrolases<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> By quantitating the level of target engagement required to raise anandamide levels in the brains of mice, ABPP was able to predict the expected levels of target engagement that would be required for human applications of FAAH inhibitors. Likewise, we used the same ABPP method to identify relevant off-targets of our own preclinical FAAH inhibitors.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> More recently, a FAAH inhibitor, BIA 10-2474, led to severe adverse reactions in the clinic, including one fatality. Subsequent ABPP analysis indicated that BIA 10-2474 also interacted with a number of serine lipases. It was hypothesized that alteration of the lipid networks in cortical neurons was responsible for the toxicity.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></div><div class="NLM_p">Because ABPP probes are often carefully optimized, target profiling and identification may be streamlined without the concern of developing a new set of molecular probes for each project. The major limitation in using ABPP for target identification is a lack of probes for the majority of druggable protein space let alone allowing for the discovery of alternative binding modes and allosteric modulators within the established ABPP-covered target space.</div><div class="NLM_p last">To overcome the limited scope of true “activity”-based protein profiling, reactivity-based profiling (RBP) takes advantage of the characteristics of certain functional groups in biomolecules by expanding ABPP to nucleophilic side chains such as free sulfhydryls<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> and amines<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> or even specific post-translational modifications such as sulfenic acids.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> One unique application of RBP capitalizes on the propensity of metabolizing enzymes oxidizing the drug to create a cross-reactive moiety in situ. By choosing a biorthogonal handle that is resistant to metabolism, the in vivo targets of reactive drug metabolites may be assessed. While the majority of reactive metabolites react preferentially with free thiols, enriched proteins are not necessarily responsible for metabolism itself.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> The photoaffinity modified fragment screening method described in <a class="ref internalNav" href="#sec3_1_3" aria-label="section 3.1.3">section 3.1.3</a> also can be considered as RBP, and it demonstrates utility in creating selective chemical matter and identifying novel targets simultaneously and for targets at compound affinities too low for noncross-linking methods.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> The key to effective ABPP/RBP is the identification of interesting targets that can be labeled by a probe and then be used competitively to test compounds; the absence of a probe for the target family of interest renders ABPP/RPB ineffective.</div></div><div id="sec3_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19">3.1.5.  Target Display Techniques</h4><div class="NLM_p">While chemoprecipitation, photoaffinity labeling, and activity-based protein profiling have evolved to enrich targets directly from lysates, cells, and tissues, the immense heterogeneity in protein expression level may preclude a successful target identification campaign. In these cases, screening compounds against expression-enriched libraries of artificially generated proteins, such as those generated by display methods, may provide target insights that direct approaches are too insensitive to obtain.</div><div class="NLM_p">Several methods have been employed across academic and industrial research centers for pooled protein display-based target identification. These techniques are distinct from arrayed protein methods such as the Retrogenix (High Peak, UK) method discussed in <a class="ref internalNav" href="#sec3_3" aria-label="section 3.3">section 3.3</a>. The pool-based techniques rely on foundational strategies of linking gene to protein and include bacteriophage, bacterial, yeast, ribosome, and mRNA display methods. <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a> outlines the methodology for bacteriophage display and an example from our laboratory.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Bacteriophage display for small-molecule target ID (T7 bacteriophage are adapted from SIB (Swiss Institute of Bioinformatics), an open access resource<a onclick="showRef(event, 'ref171'); return false;" href="javascript:void(0);" class="ref ref171">(171)</a>). (A) Heterologous expression library creation process. (B) After cloning the cDNA library into T7 bacteriophage organisms, phages are applied to wells containing the immobilized small-molecule probe. After washing, phage particles most strongly associated with the compound are recovered, amplified, and the process repeated until no phage are shed during the binding and washing steps. (C) As an internal example to highlight the utility of the approach, a control biotinylated linker and probe based on JQ1<a onclick="showRef(event, 'ref172'); return false;" href="javascript:void(0);" class="ref ref172">(172)</a> were synthesized and used for preclearing and capture steps over six rounds of selection. (D) Bioanalysis image showing enrichment of specific DNA molecular weight bands from round 0 to round 6 after parent JQ1 compound elution each round. Fold enrichment refers to next generation DNA sequence analysis for the BRD2 gene. From 7000 significantly expressed sequences, BRD2 represented the largest increase from round 0 to round 6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Critically, when display technologies are applied to target identification, the goal is to identify the biological protein partners for ligand binding, even if the physiological protein abundance or affinity is low. In contrast, for other display applications (for example, antibody affinity maturation), when the goal is to identify or create a very high affinity partner, combinatorial optimization of amino acid sequences may be used to create the highest possible affinity target regardless of whether it exists in nature. Thus a differentiating requirement in the application of display technologies for target identification is capture of small-molecule/target-binding interactions over a wide range of affinities. Bacteriophage display using full length cDNA libraries as a source of genetic material is a proven method for unbiased small-molecule target identification;<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> multiple examples of success are collated in a review by Takakusagi et al.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> As a reverse chemical genomics technique, it allows low abundance and weakly binding targets to be detected through the amplification of phage DNA. It is aided by recent advances in modern DNA sequencing, which have afforded more sensitive quantitation of candidate targets (see <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>E for an example). Further, the cDNA library may be tailored to fit relevant biology from any DNA or RNA source. Massive oversampling of expressed proteins is afforded by phage amplification in <i>Escherichia coli</i>, so that thousands of copies of every gene product are presented for binding. Despite these advantages, phage display technologies face important limitations. Perhaps the most perturbing challenge involves the bacterial expression of eukaryotic genes, which limits the scope of post-translational modification. Drug candidates that bind targets in a modification-dependent manner therefore will be lost in these types of screens. Additionally, strategies must be employed to reduce the amplification of smaller proteins or protein fragments over larger proteins. Nevertheless, we find bacteriophage displays an important component to our suite of target identification methods due to its ability to amplify and detect weak but functionally relevant drug–target interactions.</div></div><div id="sec3_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20">3.1.6.  Yeast–Chem Hybrid</h4><div class="NLM_p last">Similar to display approaches, artificial cellular systems may be leveraged for target identification by positive selection screening. On the basis of work by Licitra and Liu,<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> a chemical yeast–three hybrid genetic system, or yeast–chem hybrid, can be employed, whereby cell growth on limited media is controlled by the presence of a molecular probe that brings two parts of a synthetic essential receptor into proximity. In this application, haploid yeast cells expressing an anchor protein fused to a DNA binding domain are mated with a library of haploid yeast cells, each containing a separate target “prey” fused to an activation domain. The resulting diploids are treated with a molecular probe constructed from the compound of interest linked to an anchor protein binding element. In only those diploids where the molecular probe binds both the anchor protein and a prey protein are biosynthetic genes encoding the production of histidine activated, allowing cellular growth on histidine depleted media. By creating libraries of genetically encoded “prey” targets from cDNA expression libraries, entire proteomes can be interrogated for their ability to bind the molecular probe.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> Several companies have commercialized this technology (e.g., Hybrigenics, Cambridge, MA) and specialize in the identification of direct targets, confirmation of targets identified by other methods, and finding off-targets for lead candidates. As with display, these systems have the advantage of normalizing protein levels relative to native biological systems so that expression level differences do not impact the results. However, depending on the probe used and the artificial nature of the protein targets profiled, the technique is prone to high false positive/negatives, once again highlighting the importance of orthogonal methods.</div></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21">3.2.  Chemical Probe-Based Plasma Membrane Target Identification</h3><div id="sec3_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22">3.2.1.  Considerations for Plasma Membrane Protein Target Identification</h4><div class="NLM_p last">While target identification from preparations of intracellular proteins poses formidable challenges, interrogation of proteins embedded in the plasma membrane requires separate and unique consideration. More than a third of the expressed proteome is embedded in the plasma membrane, and greater than 50% of marketed small-molecule drugs target these proteins. Despite this, methods to broadly and robustly measure compound engagement at the level of the membrane are lackluster. This is largely driven by the hydrophobic character of these proteins, which confounds their extraction and recovery. When taken together with their lower overall copy number and propensity to aggregate, generalized methods are needed to profile this critical group of candidate targets.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></div></div><div id="sec3_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23">3.2.2.  Ligand-Directed Chemical Biology</h4><div class="NLM_p">Direct large-scale interrogation of the cellular plasma membrane proteome is fraught with biological and technological complications, the greatest of which include poor solution-phase behavior and sample preparations contaminated with intracellular proteins. Chemical biology techniques have been forwarded to help mitigate these circumstances that capitalize on covalent associations and unique cell surface features in order to maximize sample recovery and purity.</div><div class="NLM_p">Every cell is coated in a layer of carbohydrates consisting of large branched sugar chains called glycans. Glycans are physically attached to plasma membrane proteins through asparagine (N-linked) and serine/threonine (O-linked) side chains. The majority of proteins embedded in the cell membrane possess glycans, which contrasts with intracellular proteins that contain few. Because the chemistry of cellular glycans frequently can be manipulated with little to no detriment to the cell, glycans provide a unique and convenient avenue toward the enrichment of plasma membrane proteins. Consequently, chemical biology applications have focused on the development of soluble trifunctional probes. One arm of the probe is the selectivity function and consists of the compound whose targets are under investigation. The second arm consists of a reactive moiety designed to specifically and covalently bind to the cell surface glycan chains. The third and final arm is an enrichment function, which permits purification of the covalently bound membrane associated glycoproteins from the cellular milieu.</div><div class="NLM_p">In one application of this paradigm, cells<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> and even animals<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> can be labeled with non-natural sugars under physiological conditions, introducing new functional groups for a wide range of chemical modifications. In a recent example, a glycophosphatidylinositol (GPI)-anchored receptor was labeled with sugars containing azide functionalities. A trifunctional probe consisting of a receptor-binding ligand, biotin, and an azide-reactive cyclooctyne was introduced. Because the covalent reaction between azides and cyclooctynes occurs only at high concentrations, enrichment of the receptor would only transpire if the probe were bound to it for an appropriate time. This was observed after avidin purification by Western blotting and mass spectrometry (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). Sensitivity and specificity of the process may be increased further by cleaving the sugar–peptide bond with a glycosidase enzyme (PNGase F), limiting matrix effects. The method can also be used to map cross-linking sites, image receptors, and their internalization.<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a></div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Schematic overview of the glycan-mediated ligand-controlled click chemistry (“GLiCo-Click”) strategy. After the cells have been metabolically labeled with azidosugars, the chemical probe binds to its cell surface receptor and brings a bioorthogonal alkyne into close proximity to the receptor glycans, and this triggers the formation of a covalent cross-link between the alkyne and the azidosugar, thereby irreversibly equipping the target receptor with biotin. Biotinylated proteins can be analyzed by microscopy to track endocytosis or by chemoproteomics after cell lysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The metabolic labeling of cell surface glycans is an attractive method owing to high label incorporation, lack of deleterious cellular effects, and overall cleanliness of the sample preparation. For primary and nonproliferating cells, metabolic labeling may be less efficient. In these cases, a different trifunctional probe (TriCEPS, Dualsystems Biotech) may be employed, which similarly uses cell surface glycans as a mechanism for plasma membrane protein enrichment. When treated in low concentrations of sodium <i>meta</i>-periodate, glycan alcohols are carbonylated, resulting in the decoration of the cell surface in aldehydes. A trifunctional probe containing a hydrazine as its reactive functionality will rapidly form covalent hydrazone bonds with the aldehydes. Applications of this technique are fairly recent and have involved the use of biotherapeutics in lieu of small molecules. <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a> shows a study in our laboratory involving TriCEPS modification of three antibodies genetically engineered to display a spectrum of affinities for the surface receptor EGFR (0.0014, 0.3, 1 μM). These were tethered to the trifunctional hydrazine probe via an <i>N</i>-hydroxysuccinimide functionality (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A) and applied to cell lines containing as few as 2000 copies of EGFR on their surface. In all three cases, EGFR was successfully enriched, demonstrating the applicability to the method to low affinity and low abundance surface receptors (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B).</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. EGFR was able to be enriched and observed by ligand receptor capture using a low receptor number and weak antibody. (A) Annotated structure of the TriCEPS ligand. (B) Ligand receptor capture and mass spectrometric detection from NCI-H358 cells, which contain a low 2000 receptor copy number per cell.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As an alternative to glycans, certain amino acids on the protein backbone can serve as anchoring points for ligand-directed chemistries. One of the early successful examples of so-called traceless (native function) affinity labeling<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> is ligand-directed tosyl (LDT) chemistry (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A). By unifying covalent bond formation and ligand cleavage, a phenyl sulfonate (tosyl) linker can serve as both a linker between the ligand and the reactive group as well as a leaving group. The S<sub>N</sub>2-type reaction between the tosyl group and a nucleophilic amino acid results in the release of the ligand during the labeling reaction. LDT chemistry was first demonstrated by modification of human carbonic anhydrase (hCA). LDT reagents comprised of a benzenesulfonamide, a fluorophore (Dc, 7-dimetylaminocoumarin), or a biotin capture tag were connected through a tosylate group. Human red blood cells were successfully labeled, with no apparent hemolysis, and SDS-PAGE analysis after cell lysis indicated a single fluorescent band that corresponded to endogenous hCA. Blocking studies with an inhibitor did not lead to hCA labeling, suggesting that selective labeling was achieved by an affinity-based reaction. hCA labeling was also conducted in live animals, where the biotin-type labeling reagent was intravenously injected into Slc:ICR mice. Analysis by Western blotting showed that hCA-selective labeling occurred, demonstrating the power of ligand-directed chemistry in the context of complex proteomes.<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> Proteins that have been successfully captured to date are FK506 binding protein 12 (FKBP12), Src homology 2 (SH2) domain, and congerin II, whereby LDT reagents contained a range of ligands, such as the synthetic analogue of FK506, a peptide, and carbohydrate ligands.<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> LDT has been used for selective protein labeling, such as heat shock cognate 70 (Hsc70),<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> a molecular chaperone, and 14–3–3 proteins.<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a></div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. General processes involved in ligand directed chemistries. (A) The molecular probe is recruited to the receptor by virtue of the ligand. Upon binding, the reactive group covalently binds to the protein in a proximity driven manner, ejecting the ligand in the process. (B) The molecular probe is again recruited to the receptor by its ligand, however, the reactive group forms a covalent bond, ejecting its own leaving group, while the ligand and pull down handle remain affixed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite its high target and site selectivity, the LDT method is limited by its slow rate and low labeling efficiency and labeling membrane-bound receptors in live cell systems has been problematic. Ligand-directed acyl imidazole (LDAI) chemistry was developed to address this problem (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A).<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> Because of the controlled reactivity of acyl imidazole and the selective binding driven by the ligand, acyl transfer from the LDAI reagent to a natural amino acid on the target protein surface was efficiently accelerated by a proximity effect to afford labeled receptors. Successful examples of receptor capture include GPI-anchored receptors,<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> the bradykinin B2 receptor,<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> and the NMDA receptor.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a></div><div class="NLM_p">An alternative traceless affinity labeling reaction called affinity-guided DMAP (AGD) chemistry involves the use of a catalyst to facilitate the chemical modification of target proteins. In particular, the organocatalyst 4-dimethylaminopyridine (DMAP), a commonly used catalyst for acyl transfer reactions, has been used for membrane protein labeling. The strategy employs an affinity ligand tethered to the DMAP catalyst, which, in the presence of appropriate acyl donors, facilitates the acyl transfer reaction to a nucleophilic amino acid residue near the active site of target proteins. A series of ligand-tethered DMAP catalysts specific to a range of proteins has been designed. For instance, the SH2 domain and FKBP12 probes had high target specificity in bacterial cell lysates and animal tissue extracts.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a></div><div class="NLM_p">Recently, Ueda et al. have reported a study evaluating LDT and epoxide-based ligand-directed chemistries to capture human FKBP12 (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>B). They compared probes for labeling the target protein surface and probes with reactive groups for labeling of the interior of the ligand binding pocket and showed that only the ligand binding pocket-oriented reagents labeled the target protein, whereas the protein surface-oriented reagent was ineffective.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a></div><div class="NLM_p">Although powerful, the amino acid-based proximity-driven chemistries reported to date share the limitation that the reactive groups on the molecular probes are inherently reactive with functional groups already present in cellular systems. Moreover, they rely on the presence of specific amino acids in close proximity to the ligand-binding site, which is problematic for on- and off-target protein identification when no a priori knowledge about the target proteome is available.</div><div class="NLM_p last">Similar to intracellular target identification, cell-surface receptors have been shown to be amenable to photoaffinity labeling (PAL) in a method termed capture compound mass spectrometry (CCMS).<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> In this case, trifunctional compounds are designed that entail a drug molecule attached as the selectivity function, a photoactivatible reactivity function to induce a covalent cross-link with the target protein subsequent to equilibrium binding by the selectivity function, and a function for the isolation of compound–protein conjugates. As a case study to demonstrate the targeting and capture capability of CCMS, a workflow to capture GPCRs directly on the surface of living cells, has been reported.<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> Here, capture compounds based on a GPCR ligand targeting the dopamine D2 receptor have been employed as a proof of concept.<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a></div></div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24">3.3.  Cellular Protein Expression Arrays</h3><div class="NLM_p">As described previously with display technologies, screening for targets against libraries of artificially produced proteins can be advantageous. Plasma membrane proteins notoriously have been incompatible with most protein display and array techniques, however, owing to their biophysical properties. Retrogenix (High Peak, UK) invented a method for cellular surface protein microarray that has been applied successfully to multiple target identification questions across various therapeutic areas, including antiviral,<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> antiparasitic,<a onclick="showRef(event, 'ref99 ref100'); return false;" href="javascript:void(0);" class="ref ref99 ref100">(99,100)</a> and oncologic.<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> By comparison with more precedented arrayed protein techniques,<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> the Retrogenix format confers several advantages, principally the enablement of membrane protein overexpression and presentation in a mammalian cellular context. This method comprises a reverse transfection array of cDNA spots encoding GFP and a single human membrane protein in a bicistronic message. These spots are prepared on a glass surface amenable to cell attachment. HEK293 cells are overlaid, and reverse transfection occurs via lipofection to express GFP and a unique membrane protein in the cell monolayer at the positions defined by the initial array. Similar to recombinant protein arrays,<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> a fluorescently labeled small molecule, peptide, or antibody is allowed to bind to overexpressed protein targets followed by washing and imaging. The array of GFP spots allows registration of the positions of all test proteins. A key advantage to this process over more conventional recombinant protein arrays is that it potentially affords well-folded membrane protein presentation in a physiologically relevant milieu, decreasing the false negative discovery rate. Similar to recombinant protein arrays, all proteins are expected to be presented at roughly similar concentrations in contrast to methods using physiologically correct cell or tissue lysate matrices where critical target proteins may be expressed at levels too low to measure.</div><div class="NLM_p">Using the same three EGFR antibodies modified to display a spectrum of affinity (0.0014, 0.3, 1 μM) described in the previous section, we assessed the sensitivity of the platform. In <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>, the antibodies show clear replicate binding to live cells overexpressing EGFR on their surface. Because antibody binding is detected by application of a fluorescently labeled secondary antibody, application to small molecules requires tagging the molecule directly with biotin or a fluorophore. Using alprenolol, mixed results were observed, where binding to the expected β-2 adrenergic receptor was confirmed, but other known targets were not seen (data not shown). An impressive feature we have observed so far with the Retrogenix approach is an absence of false positives. The fact that either a validated target is identified or no targets are identified at all contrasts with the broad swath of disproven targets found in every other target identification method. It remains to be seen whether false negative results occur to mitigate the zero false positive rate.</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Antibodies with varying affinity to EGFR are all accurately detected and identified by using cell surface microarray technology. FCGR2B, low affinity immunoglobulin gamma Fc region receptor II-b, used as positive control for detection antibody. ZsGreen1, antibody for detection of GFP expression pattern.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec3_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25">3.3.1.  PRESTO-Tango</h4><div class="NLM_p last">Genetically engineered cell lines overexpressing membrane target proteins and pathway-dependent reporter genes have been used to identify targets. For example, Bassilana et al. have identified GPR39 as the target for a Hedgehog pathway inhibitor.<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> Roth et al. have described a parallelized platform to interrogate a large number of druggable human GPCRs via a G protein-independent β-arrestin recruitment assay. GPCRs are known to induce arrestin translocation, which can be measured by transcriptional activation following arrestin translocation (“Tango”).<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> This assay was translated into a platform technology by parallel receptor expression and screening via transcriptional output through Tango (“PRESTO-Tango”),<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> whereby the required reagents and methods are available to the scientific community and hence can be considered open source. In a modular design, a Tango gene construct was furnished for each GPCR in the human genome. The 5′ end entails a cleavable signal sequence for membrane localization and a FLAG epitope tag to enable monitoring of cell-surface expression. The 3′ end includes the sequences for the tobacco etch virus (TEV) nuclear inclusion, an endopeptidase-cleavage site, and the tetracycline transactivator (tTA) protein. Ligand binding to the target GPCR stimulates recruitment of the arrestin–TEV protease fusion, triggering the release of tTA. Free tTA then enters the nucleus and triggers a measurable reporter gene activity. PRESTO-Tango has been validated on 120 nonorphan human GPCR targets, but not all GPCRs are accessible, possibly because some receptors do not interact with arrestins in an agonist-dependent fashion.</div></div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26">3.4.  Label-Free Target Enrichment</h3><div id="sec3_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27">3.4.1.  Considerations for Going Label-Free</h4><div class="NLM_p last">The approaches described in the previous two sections share a common goal: the generation of target candidates and mechanistic hypotheses that may be actioned for validation. However, none of the technologies are sufficiently robust when used in isolation and benefit from combination. A drawback from leveraging orthogonal techniques resides in the upfront material costs and labor required to properly execute a study with the understanding that generation of molecular probes and labels may ultimately fail. Consequently, application and integration of techniques that do not require probe chemistry but rather use the compound leads as they are is an important part of a target identification platform. A few of these types of applications are described herein.</div></div><div id="sec3_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28">3.4.2.  Nematic Protein Organization Technology (NPOT)</h4><div class="NLM_p last">NPOT technology utilizes a transitory pH gradient to mimic both the intra- and extracellular pH. This gradient allows ligand–protein interactions to occur at their physiological pH, followed by a precipitation event that is well-described by the Kirkwood–Buff theory.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> In practice, tissue or cell lysates are prepared at 4 °C, in the absence of detergent, reducing agent, or protease/phosphatase inhibitors. All dilutions and washes utilize buffer with equal osmolality, trace elements, vitamins, and salts, in concentrations that recapitulate the interstitial medium or cytoplasm. Compound (≤10<sup>–6</sup> M) is incubated with the lysate, and the macromolecular assemblies are separated using a differential microdialysis system. The macromolecules (protein groups) migrate in a transitory continuous pH gradient (5–10) to their mean zwitterionic point. Nematic crystals gradually form from the macromolecular clusters into heteroassemblies. This compound–protein mixture is subsequently trapped in mineral oil and isolated. The precipitate is solubilized and proteins identified by MS. To asses this approach, five compounds were studied, both to complement target identification efforts from PDD hits of interest and to investigate off-target effects. For three of the SMs, NPOT data supported and/or led to the prioritization of specific hypotheses. In the absence of complementary data sets, it may be challenging to prioritize the information-rich protein lists to directly identify the direct binding partner(s). However, when these data are subjected to orthogonal methods for protein enrichment and pathway mapping, a potentially unique snapshot of the SM’s interactome may be revealed.</div></div><div id="sec3_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29">3.4.3.  Unique Polymer Technology (UPT)</h4><div class="NLM_p last">When lacking the capability to generate a functional molecular probe for chemoprecipitation, a method by which the compounds may be immobilized to a surface in a random orientation with no modification is desirable. Unique polymer technology (UPT) offers this advantage. Unmodified molecules of interest are noncovalently immobilized on a unique polymer surface, presumably via many weak interactions.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The appropriate polymer and binding conditions are empirically determined by quantifying the amount of small molecule released during a series of washing steps.<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> A biological lysate of interest is incubated with the affinity matrix and washed, and retained proteins are identified by using gel electrophoresis-MS. In a recent example, histone deacetylase 2 and prohibitin were implicated as the targets of a small-molecule targeting BrCa cells.<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> The UPT method claims that 60% of enriched proteins are either directly interacting or part of an enriched complex.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div></div><div id="sec3_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30">3.4.4.  CETSA-MS</h4><div class="NLM_p">The relatively new thermal shift assay (TSA) method of cellular thermal shift analysis–mass spectrometry (CETSA-MS) affords simultaneous determination of compound binding and target identification across a wide array of soluble and membrane-bound targets.<a onclick="showRef(event, 'ref109 ref110 ref111 ref112'); return false;" href="javascript:void(0);" class="ref ref109 ref110 ref111 ref112">(109−112)</a> The principle is as follows. Using either living cells or cell lysates, with or without added test compound, multiple replicate samples are heated, each at a separate temperature, for a preset period of time. The heating causes proteins to unfold and aggregate, creating a loss of detectable signal after centrifugation. In a target engagement embodiment, the signal is measured by Western immunoblotting and gel quantitation.<a onclick="showRef(event, 'ref113 ref114'); return false;" href="javascript:void(0);" class="ref ref113 ref114">(113,114)</a> The midpoint temperature of the signal loss, designated the protein unfolding transition (<i>T</i><sub>m</sub>), is measured for each species. The measured <i>T</i><sub>m</sub> of individual proteins is affected by ligand binding; typically, binding leads to thermodynamic stabilization and a clearly measurable change in the <i>T</i><sub>m</sub> value by several degrees C (called the “Δ<i>T</i><sub>m</sub>”). In the CETSA-MS target identification embodiment, as many as 4000–6000 proteins are measured at 10 temperatures by mass spectrometric proteomics techniques, and the <i>T</i><sub>m</sub> values and Δ<i>T</i><sub>m</sub> values of liganded proteins are assigned.<a onclick="showRef(event, 'ref109 ref110 ref112'); return false;" href="javascript:void(0);" class="ref ref109 ref110 ref112">(109,110,112)</a> By comparing the <i>T</i><sub>m</sub> values for all of the observed proteins in a cell or tissue sample with and without test compound incubation, proteins whose structure is thermodynamically affected by ligand binding are identified as potential targets. The analysis may be taken further by comparing changes in <i>T</i><sub>m</sub> values between intact cells and prelysed cells. These observations permit inference of the value of protein–protein interactions and endogenous cofactor binding.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a></div><div class="NLM_p last">In an extension of this work, the dimensionality and efficiency of the process was increased by including a parallel set of experiments in which cells or lysate samples were preincubated with varying concentrations of the test compound.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> Because both test compound concentration and denaturation temperatures are simultaneously varied, the experiment comprises a two-dimensional profiling strategy. As above, proteins bound by a small-molecule ligand will demonstrate altered thermodynamic stability and thus a shift in the midpoint temperature of the unfolding transition. In the one-dimensional version of CETSA, a binary binding measurement is ascertained at a high compound concentration (optimally, >10× <i>K</i><sub>i</sub> for any target or off-target), and additional compound doses are used in follow-on experiments to deconvolute the relative affinities for selected proteins. The simultaneous temperature and compound concentration response profiling in the two-dimensional strategy affords an in situ confirmation of the binding, as all real binders should show a dose response. By simultaneously dosing temperature and compound concentration in a single two-dimensional experiment, identification, confirmation, and affinity rank ordering of ligand–target interactions are determined in a single experiment. While a reduced number of drug concentrations is typically used in two-dimensional CETSA compared to a typical isothermal dose–response experiment, and thus the quantitative resolution of affinities of the drug to different cellular targets is reduced, the efficiency of identifying and characterizing so many interactions in one experiment should more than compensate for the increased imprecision.</div></div><div id="sec3_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31">3.4.5.  Yeast Haploinsufficiency (HIP) and Homozygous (HOP) Profiling</h4><div class="NLM_p last">The use of yeast deletion strains to provide genome-wide measurements for uncovering SM MOA are well-established.<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> A HIP assay involves treating a set of heterozygous deletion diploid strains with SMs that have measurable cytotoxicity. Reducing the gene dosage in half increases the sensitivity of the drug’s cytotoxic effect and is referred to as drug-induced haploinsufficiency. This may lead to identification of the direct target(s) of the SM or reveal proteins that are mechanistically related to function of the target(s). HOP profiling utilizes yeast haploid or diploid strains with complete deletion of nonessential genes. Protein knockouts that confer resistance to SM treatment are identified and used in concert with HIP data to provide comprehensive analyses of SM MOAs.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> As part of a target deconvolution effort for SMs derived from a CCL profiling campaign, we integrated HIP data with the suite of approaches described herein. In all accounts, these data failed to provide hypotheses or add to existing biological themes. With the advent of additional data analyses options, and knowledge of the targets for some of these SMs, we are currently reevaluating these screens to assess the potential hidden mechanistic value of these data.</div></div><div id="sec3_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32">3.4.6.  Other Target Engagement Technologies</h4><div class="NLM_p last">As mentioned previously, CETSA is a method that relies on the thermal stabilization of the target protein to determine ligand engagement. In a similar manner, other methods such as drug affinity responsive target stability (DARTS), stability of proteins from rates of oxidation (SPROX), and protein–ligand interactions by mass spectrometry, titration, and H/D exchange (PLIMSTEX) rely on enhanced target stabilization to inform on ligand engagement.<a onclick="showRef(event, 'ref118 ref119 ref120'); return false;" href="javascript:void(0);" class="ref ref118 ref119 ref120">(118−120)</a> With DARTS, engagement is measured by increased resistance to proteolysis, typically by Pronase. When considering SPROX, engagement is measured by a ligand-induced reduction in nonspecific oxidation owing to the retention of a folded protein state. PLIMSTEX is similar to SPROX in this regard, only it measures the propensity of the target to undergo hydrogen–deuterium exchange instead of oxidation. Each of these techniques possess unique advantages and liabilities in their implementation but have all been used to demonstrate ligand binding and, in some cases, ligand interaction strength.</div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">4.  Target Verification and Engagement</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26551" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26551" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34">4.1.  Considerations for Confirmation of Target Hypotheses</h3><div class="NLM_p last">Once a candidate target or list of target candidates for small-molecule binding is generated through either direct binding or indirect mechanism of action studies, target engagement assays typically are required both for confirmation of the binding hypothesis and evaluation of additional small-molecule chemical matter. It is unlikely a PDD hit meets all criteria for advancement to in vivo studies without chemical optimization and confirmation. At a minimum, a dozen or more related compounds should be synthesized to understand the structure–activity relationships of the ligand–target interaction. More typically, the identification and confirmation of a disease-related target (or toxicity-related off-target) galvanizes development and implementation of a full high-throughput screening campaign to identify both structurally related and unrelated chemical matter that might act more favorably or selectively on the target and the disease phenotype. Because the direct target engagement assay (or enzymatic activity-based assay, if relevant) is linked to the phenotypic assay from which the effort originated, it is likely advantaged over a target-based assay in both potentially exploiting previously unidentified targets and in foreknowledge that the target is druggable.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35">4.2.  CETSA</h3><div class="NLM_p last">As described above, CETSA is employed to characterize the temperature at which individual proteins within cells or lysates unfold. In the original target engagement example of CETSA,<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> individual proteins were monitored by using immunoblot analysis. This comparatively straightforward technique allows one to confirm binding to and to affinity-rank order compounds both in intact cells and lysates, in cell culture, and in intact tissue and even whole animals.<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> One can gain mechanistic insight into compound effects on protein–protein interaction and protein–cofactor interaction through analyzing intact cell versus lysate affinities.<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> The breadth of possibilities is rich, as long as an adequate antibody can be obtained, and recent reviews and many original research publications document that the technique can be applied across a variety of target types and in many different laboratories.<a onclick="showRef(event, 'ref122 ref123'); return false;" href="javascript:void(0);" class="ref ref122 ref123">(122,123)</a> It is worth noting that in some cases lack of a CETSA binding stabilization or destabilization signal may be a biophysical consequence of the type of binding rather than actual inability of the small molecule to bind. Such false negatives are reported to occur in rare cases.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36">4.3.  NanoBRET</h3><div class="NLM_p last">The NanoBRET platform was introduced for the analysis of protein–protein interactions in 2015<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a> and was rapidly followed by publication of a technique for intracellular SM–target interaction targets<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> akin to in vitro time-resolved fluorescent resonance energy transfer (TR-FRET) probe displacement assays.<a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a> Both the in vitro TR-FRET and intracellular NanoBRET versions of the assay comprise homogeneous (no wash) formats and provide a highly specific measurement of target engagement. In the in vitro method, the protein of interest is the only target present; in the NanoBRET method, specific tagging of the target of interest affords discrimination. The target specificity inherent in the NanoBRET cellular version is critical so that the presence of so many other potential targets in a cell does not interfere. While the compound and probe molecule may bind many proteins, a NanoBRET ratiometric signal is only measured when probe engages the tagged protein. Further, tagged proteins can be introduced to cells by transient transfection under conditions where expression level is as low as endogenous protein<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> in order to minimize the cellular consequences of overexpression; the assay detectability remains high due to the high sensitivity engineered NanoLuc luciferase (Nluc) enzyme and novel cell permeable substrate system.<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> In several recent instances, the sensitivity afforded by the detection system was taken advantage of to remove the limitations of exogenous expression entirely. White et al. fused DNA encoding full length Nluc into the endogenous genomic loci of genes coding for β-arrestin2 and CSCR4 to measure β-arrestin recruitment and GPCR internalization and trafficking.<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> Schwinn et al. describe a general technique for using CRISPR/Cas9 technology and ribonucleoprotein electroporation to efficiently tag endogenous proteins in multiple cell types.<a onclick="showRef(event, 'ref129'); return false;" href="javascript:void(0);" class="ref ref129">(129)</a> Both approaches promise to enhance the quality and efficiency of the NanoBRET intracellular target engagement format and can afford low-throughput ligand–target binding confirmation and high-throughput screening for new ligands for confirmed targets. Finally, because the NanoBRET probe displacement assay is nonlytic, all measurements including kinetic displacement analyses can be carried out in intact cells. Thus compounds can be characterized not only for intracellular target engagement but also for their kinetic binding off-rate.<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> While the technology holds many significant advantages in efficiency over CETSA, the fact that CETSA requires no protein tagging allows CETSA the advantage of more cell types and affords translatability. Additionally, a small molecule that binds at a different site on the target of interest than the probe and does not modulate binding of the probe is effectively invisible in NanoBRET probe displacement assays but not CETSA. A probe can be made from a small molecule of interest in order to expand the ability to discover new ligands to a novel binding site, but initial target engagement studies by NanoBRET are most easily afforded when the target hypothesis falls into a well described enzyme class where probes exist for a range of targets within the class, for example, kinase enzymes. This is nicely demonstrated by the publication of a 178-kinase NanoBRET array for assessing intracellular kinase selectivity.<a onclick="showRef(event, 'ref130'); return false;" href="javascript:void(0);" class="ref ref130">(130)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37">4.4.  Bump-Hole</h3><div class="NLM_p last">One challenge associated with matching a target with a compound identified from a phenotypic screen is definitively confirming that the target identified drives the phenotype. Although many of the techniques described in this publication can show whether a compound interacts with a target, there is still a risk that off-targets are the real driver of said phenotype. One exceptionally powerful method for confirming a target hypothesis is the “bump-and-hole” strategy whereby the test compound is modified with a sterically demanding group (a “bump”) that matches a target with amino acid substitutions that accommodates the bump (the “hole”) such that only the hole-modified target can bind the bump-modified drug, but neither the native on-target nor potential off-targets are still able to bind the bumpy drug.<a onclick="showRef(event, 'ref131 ref132'); return false;" href="javascript:void(0);" class="ref ref131 ref132">(131,132)</a> Then in cells expressing the mutated target, the bump-modified drug can show the phenotype only via interaction with the modified target. This has been used for a number of targets including kinases, BET-family proteins, and others.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38">4.5.  Imaging and Target Localization</h3><div class="NLM_p">Target localization is a process to determine where the drug target is located in cells and even whole organisms. Knowledge of target localization can increase success in target identification in the drug discovery process. For example, cell-membrane proteins require a different target identification strategy than cytosolic or secreted proteins. Determining the subcellular localization may also be useful in prioritizing drug targets. Extracellular domains of plasma membrane proteins and secreted proteins are easily accessible by drug molecules and could be prioritized over cytosolic proteins in certain cases.</div><div class="NLM_p">Several techniques are available to determine target localization in cells and tissues. For instance, FRET leverages the proximity between a ligand and its target, resulting in a quantitative readout for target occupancy.<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> Ligand-directed protein labeling covalently tags a target protein with a reporter and can be used not only to visualize or isolate target proteins but also to quantify target engagement.<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a> CETSA can monitor binding of unlabeled drug compounds to target proteins. Recently, single cell, spatially resolved CETSA has been achieved with adherent cells.<a onclick="showRef(event, 'ref135'); return false;" href="javascript:void(0);" class="ref ref135">(135)</a></div><div class="NLM_p">The ability to visualize targets in live cells, or even in whole organisms, not only provides insights into target localization for the purpose of target identification but could also improve testing of clinical efficacy of new small-molecule inhibitors. A number of small-molecule target imaging agents have been developed that leverage bioorthogonal two-step labeling methods. An attractive strategy entails equipping the drug molecule with a reactive <i>trans</i>-cyclooctene (TCO), which leaves the drug cell-membrane permeable and, upon binding its target, can be visualized using a bioorthogonal tetrazine (Tz)-labeled fluorophore. Using this strategy, it was possible to image aurora kinase A (AKA) in live cells and to quantify cellular kinase expression.<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> Other successful examples leveraging Tz/TCO bio-orthogonal pretargeting include BTK,<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a> MET,<a onclick="showRef(event, 'ref138'); return false;" href="javascript:void(0);" class="ref ref138">(138)</a> EGFR,<a onclick="showRef(event, 'ref139'); return false;" href="javascript:void(0);" class="ref ref139">(139)</a> and PARP.<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(140)</a></div><div class="NLM_p">Molecular imaging, often via positron emission tomography (PET), is a noninvasive biomedical technique that enables the quantification and visualization of drug targets in human and preclinical subjects by monitoring the time-dependent distribution of minute quantities of drug labeled with positron-emitting isotopes.</div><div class="NLM_p last">Although one of the major applications of PET is the study of target occupancy, it is a prime technology to determine target localization in humans and preclinical species. By measuring the effect of different drug doses on radioligand binding, tissue penetration, target engagement, and the relationship between plasma exposure and target occupancy can be determined in vivo. Although PET is a powerful technology to study target localization both clinically and preclinically, it relies on radioligands that must be selective for the target of interest and possess a range of physicochemical and pharmacological characteristics that allow them to be radiolabeled with short-lived positron-emitting isotopes, safely administered to humans, and quantified on the target in vivo.<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39">4.6.  SM–Protein Target Engagement Methods (ASMS and TSA)</h3><div class="NLM_p last"><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a class="ref internalNav" href="#sec2_2" aria-label="section 2.2">section 2.2</a> above discuss the scale of challenge in agnostically screening PDD hits against many targets. However, it is noteworthy that modalities exist to screen multiple protein targets in parallel on a compound collection in affinity-based HTS screens. For example, a collection of purified proteins of relevance to a particular pathway can be screened rapidly by affinity selection–mass spectrometry (ASMS) and then confirmed in cellular pathway activity assays.<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> In this type of study, there is no need to tag the small molecules, and all proteins are controlled to same concentration. However, the library of proteins is restricted to those that are individually cloned, expressed, and purified. Several iterations of mass spectrometry-based direct ligand detection methods based on either size-exclusion chromatography or ultrafiltration are all robust in our estimation and can be applied to this parallel protein target and ligand matrixed ID approach.<a onclick="showRef(event, 'ref143 ref144 ref145'); return false;" href="javascript:void(0);" class="ref ref143 ref144 ref145">(143−145)</a> Thermal shift analysis (TSA) (also known as differential scanning fluorimetry) methods also can be applied to detect ligands for an array of purified protein targets.<a onclick="showRef(event, 'ref146 ref147'); return false;" href="javascript:void(0);" class="ref ref146 ref147">(146,147)</a> The detection method is less direct than ASMS techniques, as binding is inferred by thermal stabilization or destabilization of the protein being measured, but the protein detection is more direct, in a sense, because one measures the kinetic process of thermally induced protein unfolding. This protein detection-based method requires excess compound in order to saturate binding sites and provides a clear signal, akin to the requirements for lower throughput NMR-based methods.<a onclick="showRef(event, 'ref148'); return false;" href="javascript:void(0);" class="ref ref148">(148)</a></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40">5.  Technology Integration: Case Studies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24665" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24665" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41">5.1.  Strategic Considerations for Technology Integration</h3><div class="NLM_p">Because a single technology does not exist that will routinely identify the direct targets for SMs of interest, there is a need for a coherent integration strategy. Many of the enabling approaches are only available commercially, so routine utilization of multiple compound-technology combinations is generally cost-prohibitive. Our initial objective for SMs derived from both PDD and TDD is to identify or rule out common targets and mechanisms. The key to this prioritization step involves profiling technologies that broadly survey SM biological perturbations and couple this data with a companion reference database. In cases where a project team has a question about a small set of compounds, either from TDD or PDD campaigns, we generally suggest starting with BioMAP profiling (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). We then initiate DR-L1000 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) and include the initial compounds, key analogues, and BioMAP reference database match compounds in our profiling submission. Conversely, if we have a large number of phenotypic screening hits of interest (e.g., 10–50), we usually start with DR-L1000 profiling to inform on the number of MOA buckets that exist prior to BioMAP DiversityPlus profiling. Moving forward, we envision profiling a larger set of TDD or PDD hits using the BioMAP ETP and DS as additional level of prioritization prior to implementing both DR-L1000 and BioMAP DiversityPlus.</div><div class="NLM_p">It is important to emphasize that a reference database match may not be due to direct engagement of the SM with the protein target. Although a direct binding hypothesis should be followed up, the data could also be consistent with an indirect perturbation. In the absence of a strong database hit, target enrichment approaches should be considered. If the SMs of interest are “mature”, either derived from a PDD informer library or part of an active TDD medicinal chemistry effort, they are usually more amenable to target enrichment strategies involving a modified parent compound. For example, a linker-modified probe for AC-MS (<a class="ref internalNav" href="#sec3_1_2" aria-label="section 3.1.2">section 3.1.2</a>) may be synthesized from existing intermediates using established SAR. Although cellular potency is not a direct predictor of successful target identification, it definitely provides a theoretical advantage for noncovalent protein enrichment and identification. In addition, SMs optimized against a specific target may have a reduced off-target profile, thus minimizing the complexity of data deconvolution. Emerging technologies that facilitate label-free enrichment will likely play an important role in the target identification tool box and complement the linker-based techniques.</div><div class="NLM_p last">To demonstrate successful implementation of the concepts described, we have highlighted two case studies. The first demonstrates successful use of phenotypic screening to prioritize a hit based on perceived unique biology, followed by AC-MS for target identification. In the second example, we lacked a clear target enrichment hypothesis but identified a gain-of-function target using SM-mediated acquired resistance (the spontaneous and positive selection genome-wide CRISPR approach). Engagement of compound CGS-0059 with the putative target was verified using CETSA.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42">5.2.  High-Content PDD: AbbVie’s BETi Story</h3><div class="NLM_p">During 2006–2008, we initiated a campaign to elucidate the selectivity, MOA, and target engagement profiles for SMs derived from a variety of sources, including pipeline TDD projects, PDD screens, and SM cellular active libraries. One of the main technology platforms that we adopted was Protein-Fragment Complementation Assays (PCAs) developed by Odyssey Thera (ODT), Inc.<a onclick="showRef(event, 'ref149 ref150'); return false;" href="javascript:void(0);" class="ref ref149 ref150">(149,150)</a> PCAs facilitate the detection and localization of complexes formed by two proteins that interact as part of a normal cellular pathway. A PCA is created by expressing genes of a pair of interacting proteins, each linked in-frame to fragments of a rationally dissected reporter gene. The association of two proteins of interest brings together complementary reporter fragments and enables productive folding into an active structure. The resulting quantifiable signal can then be spatially localized in living cells by automated high-content microscopy. Using PCA, ODT developed a profiling platform to probe the activity of over 130 protein–protein interactions covering critical pathways in the context of living cells. Each assay was run at two or three time-points in HEK-293 cells, with a subset receiving a pretreatment stimulation with pathway activators or inhibitors. A complementary panel of immunofluorescence and biochemical assays provided additional value for interpreting the effects of compound treatment. One of the more attractive features of this platform was access to a SM reference database. This allowed for rapid identification of common MOAs and ultimately provided a “SM uniqueness index” that supported data-driven decision making. Using the ODT platform and an integrated technology approach, we were able to impact programs across multiple therapeutic areas. The most noteworthy example was origination of our BETi program.</div><div class="NLM_p">Compound CMPD-604 was initially investigated as a lead from an anti-inflammatory program (WO 2009084693 A1).<a onclick="showRef(event, 'ref151'); return false;" href="javascript:void(0);" class="ref ref151">(151)</a> However, observations of growth inhibition and cytotoxicity against a small panel of cell lines prompted a closer analysis of the PCA perturbation profiles. With multiple activities correlating to stress/inflammation, cell cycle control, and apoptosis, we focused our attention on the potential of this molecule as an anticancer compound. Potent antiproliferative effects were observed for the tumor cell lines profiled in the ODT panel (SKOV3; HCT116) and the PCA signature shared activities with preclinical and clinical oncology compounds included in the ODT reference library. The PCA signature for CMPD-604 did not correlate directly with a specific reference molecule, suggesting that the MOA was potentially unique or at least not a common MOA (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>A). Prior to initiating a target identification campaign, CMPD-604 was profiled against panels of enzymes and receptors. The compound was mostly inactive but did inhibit the peripheral benzodiazepine receptor (p-BZD), with an IC<sub>50</sub> of 400nM. These data prompted profiling CMPD-604 analogues against p-BZD and correlating this activity with growth inhibition of a DoHH2 tumor cell line. The lack of correlation suggested that p-BZD was not a driver of the antiproliferative activity (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>B). Additional cancer cell line profiling established CMPD-604 as a broadly active compound with enhanced potency and maximal growth inhibition against hematological nonadherent tumor cell lines. Beyond an attractive cellular profile, the decision to initiate chemistry to synthesize an AC-MS probe for a target deconvolution campaign is often influenced by the existence of chemical functionalities that can be directly modified. CMPD-604 had carboxylate and aryl halide functionalities, facilitating the synthesis of the resin-bound SM using multiple vectors. The resin was incubated with K562 tumor cell lysate, in the presence and absence of free CMPD-604 (10 μM). The bromodomain family of proteins (BRD2, BRD3, BRD4, and BRDT) were highly enriched and completed with free CMPD-604, nominating these proteins as putative mediators of the antiproliferative activities (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>C). Because the parent compound was amenable to direct modification, it took <2 weeks to generate these target hypotheses. However, a proposal that the thienodiazepine core interacted at the bromodomain lysine-binding site was received with skepticism based on the lack of obvious similarity of CMPD-604 to an acetyl-lysine mimic. This prompted an affinity capture experiment using tumor cell lysate, followed by an on-beads protease digestion and in-gel protein identification. We were able to establish that full-length BRD4, when bound to CMPD-604, degraded in the presence of protease to fragments corresponding to BDI and BDII. In the absence of CMPD-604, BRD4 was completely degraded. These data lead to expressing recombinant protein and protein fragments in addition to establishing TR-FRET binding assays for the individual BD domains. Multiple lines of data, including NMR chemical shift analysis of <sup>13</sup>C-labeled protein and protein cocrystal structures, strongly supported CMPD-604 binding to the BD acetyl-lysine binding sites.</div><figure id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Discovery of BET family of proteins as targets for an oncology program (A) Prioritization of CMPD-604 from a phenotypic screen (B) Profiling of CMPD-604 against select profiling panels and tumor cell lines (C) Probe synthesis, AC-MS target hypotheses, localization of binding site, and target engagement. (D) Translation of screening PCA into Alpha-LISA assay, confirmation of MOA, and in vivo detection of PD marker</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To efficiently drive a medicinal chemistry effort, a robust cellular PD assay was desired. Investigation of literature-based signaling nodes did not provide robust end points for monitoring CMPD-604 inhibition, so we nominated PCAs for translation to orthogonal cellular assays. Activation of Mnk1 signaling, concomitant with potent inhibition of NF-κB reporter and translocation assays, initiated efforts to define the nature of the Mnk1 interactions. Using an Alpha-LISA assay for p-eIF4E (Perkin-Elmer, Waltham, MA), we were able to establish that the increase in the Mnk1/eIF4E PCA was driven by a phosphorylation event (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>D). Multiple chemotypes over a broad potency range demonstrated an increased p-eIF4E signal with CMPD-604-mediated growth inhibition. Using dose escalation of a Mnk inhibitor (CGP53780), we were able to block the p-eIF4E signal induced by CMPD-604 (10 μM), further establishing the role of this signaling axis. Because of the maturity of CMPD-604 and related analogues, we were able to demonstrate oral exposure in mice. Moreover, tumor growth inhibition was established in mouse tumor xenograft models and CMPD-604-mediated modulation of p-eIF4E was quantified from the corresponding tumor derived lysates (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>D).</div><div class="NLM_p last">Mechanistic data implicating Brd4 and p-eIF4E signaling in the innate immune response was recently reported.<a onclick="showRef(event, 'ref152'); return false;" href="javascript:void(0);" class="ref ref152">(152)</a> This study specifically implicated the Mnk2-eIF4E axis in LPS treated bone marrow-derived macrophages (BMDMs) that harbored a conditional deletion of Brd4. In addition, the authors did not observe differential activation of p38 or Erk when they compared WT or Brd-4-deficient BMDMs and concluded that Brd4 activates Mnk2 by a unique mechanism. In contrast to this gene perturbation study, our data suggest that pharmacologic inhibition of the BET activates p38-Mnk1, leading to p-eIF4E and inhibition of NF-κB (reporter and translocation assays). It should be noted that our study was performed in HEK293 cells, and we did not discriminate the role of Mnk1 or Mnk2 because the phosphorylation site we monitored (Ser209) is modulated by both kinases. Regardless of these differences, we were able to translate a PCA into a robust high-throughput PD assay and implicate Mnk-eIF4E/translational control to BRD biology.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43">5.3.  Chemical Genomics Screening: AbbVie’s MGST1 Story</h3><div class="NLM_p">CCL profiling has been widely implemented in an attempt to identify cellular dependencies for SMs. Typically, cell lines with corresponding genomic annotation are treated with a library of SMs and growth inhibition is monitored using nuclei staining or ATP levels as a correlate to cell number. As part of a multiyear CCL campaign, we initially screened 15 NSCLC cell lines with either established KRAS mutant dependency or c-MYC amplification against a library of 120000 SMs.<a onclick="showRef(event, 'ref153'); return false;" href="javascript:void(0);" class="ref ref153">(153)</a> Although our cell line panel was statistically underpowered and less likely to identify SM activity–cellular genotype correlations, we rationalized that the high degree of chemical diversity may sample unique biological space. To test this assumption, we implemented a highly integrated and data-driven strategy for hit selection, hit prioritization, target identification, and target engagement (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>). Compounds meeting our hit criteria were subjected to dose–response confirmation against a nine cell line panel. A compound was identified (CGS-0059) that potently inhibited growth of cell lines with established KRAS mutant-dependency or c-MYC amplification and was highly inactive in assay panels. CGS-0059 exhibited a unique ER-stress induced L1000 dose–response profile in addition to detectable apoptosis, providing evidence for an unprecedented MOA. BioMAP DiversityPlus profiling did not return a high confidence database match, adding to weight-of-evidence that CGS-0059 was modulating unique biology. These data provided the impetus for an aggressive target deconvolution campaign, including target enrichment methods using linker-modified CGS-0059. Because these direct-binding efforts did not support a solid target hypothesis, we generated CGS-0059-resistant cells using spontaneous acquired resistance and a positive-selection genome-wide pooled CRISPR screen. Both of these approaches identified microsomal glutathione transferase 1 (MGST1) as the most significant gene conferring resistance to CGS-0059. Using both gene knockdown (siRNA) and gene deletion (CRISPR), we established clearly that MGST1 expression was required for CGS-0059-mediated cell killing. To test whether MGST1 was a direct target, we attempted several engagement approaches. Using CETSA, we were able to demonstrate that cell lysate-derived MGST1 in the presence of CMPD-0059 was stabilized as compared to DMSO-treated lysate. MGST1 is a membrane bound enzyme involved in the detoxification of reactive electrophiles and protection of membranes from oxidative stress. In addition, MGST1 expression in tumors has been implicated as a resistance mechanism following cisplatin treatment.<a onclick="showRef(event, 'ref154'); return false;" href="javascript:void(0);" class="ref ref154">(154)</a> MGST1 has been localized to both the mitochondria and ER and implicated in the functioning of the mitochondrial permeability transition pore (MPTP) complex.<a onclick="showRef(event, 'ref155'); return false;" href="javascript:void(0);" class="ref ref155">(155)</a> Because MPTP function has been linked to apoptotic mechanisms, we are testing the notion that this may be relevant to the CGS-0059 gain-of-function MOA. The discovery of CGS-0059 provides an intriguing cellular probe to study an unprecedented mechanism of tumor growth inhibition.</div><figure id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Discovery of MGST1 as an unprecedented oncology target. CGS-0059 was identified from a CCL profiling campaign, initially based on a growth inhibition–GEx correlation. The SM was further prioritized based on a perceived unique MOA and subjected to target identification using generation of resistant clones and direct target enrichment. Engagement of CGS-0059 with MGST1 was supported by CETSA experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">When CCL profiling is implemented on a large scale, correlating chemical sensitivity and basal gene expression is a powerful approach for revealing cellular MOA. This is best exemplified by a recent study that revealed direct target, pathway, metabolic, and transport connections for a series of SMs with varying degrees of previous mechanistic insight.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The study also included examples of uncovering novel targets, again demonstrating the strength of this approach to inform on SM MOA. Of particular interest was their analysis to determine the number of CCLs required for identification of the mechanisms highlighted in this publication. The minimum number of CCLs was 68 (MOA of austocystin), however, to establish 80% of the 43 SM-target connections, more than 400 CCLs were needed. To realize the full potential of this approach, CCL sensitivity data needs to be generated for a wide diversity of chemical matter. Until recently, it was a Herculean effort to screen 600–800 cell lines against a reasonable-sized SM library. The development of the PRISM platform has enabled bar-coded cell lines (e.g., 25) to be pooled in a single well with enumeration of the bar codes reflecting the number of cells at the end of the assay.<a onclick="showRef(event, 'ref156'); return false;" href="javascript:void(0);" class="ref ref156">(156)</a> The multiplexed nature of PRISM should provide unprecedented access to large SM-tumor cell line panel matrices.</div><div class="NLM_p last">Even with the recent technological advancements that enable more efficient large-scale CCL screening, it will be interesting to see if cell death screens become a highly preferred strategy. It is often argued that pathway-specific screens are more relevant and provide a biological anchor because SM hits are forced to impinge on the biology of interest. This strategy is logical, but it must be emphasized that this approach does not guarantee that one is necessarily “closer to the target”. In other words, if the SM does not modulate a known signaling node and a candidate protein approach fails to provide the target, the deconvolution challenge still exists. Arguably, the more general the output of the screen, e.g., cell growth inhibition, the more likely it is that resource-intense pathway triangulation will be necessary. However, general end point screens have the advantage of sampling a much broader range of MOAs and potentially accessing additional biological target space.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">6.  Future Directions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65151" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65151" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this Perspective we have outlined strategies for SM target identification utilizing technologies that have contributed to progression of drug discovery programs. We believe that the efficient implementation of these technologies and a sustained commitment to novel methodologies is essential to address the paucity of novel targets with enhanced validation. An emphasis has been placed on prioritizing screening sets prior to TDD and PDD hit prosecution. Successful demonstration of this paradigm was recently published and introduced the concept of MP as an approach to add a layer of pathway biology information to the standard complement of SM selection criteria, namely chemical tractability, potency, and efficacy.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> At this time, the field needs more examples of biological characterization of hits earlier in the discovery process to fully appreciate the value and challenges. Verbist et al., detailed the application of GEx during lead optimization for several drug discovery programs and highlighted critical factors for implementation.<a onclick="showRef(event, 'ref157'); return false;" href="javascript:void(0);" class="ref ref157">(157)</a> More recently, L1000 has been applied at a SM library scale, providing an unprecedented snapshot of SM-induced GEx signatures, which will offer increasing value as more cell models and perturbations (genetic and SM) are continually profiled. Beyond GEx, we have emphasized the value of protein profiling to augment gaps that exist with transcription profiling.</div><div class="NLM_p last">The high attrition rate for target identification technologies highlights the importance of making better decisions about the SMs that are progressed from TDD and PDD programs. Because it is unlikely that an individual protein-enrichment method will routinely reveal SM targets, creative tactics will have to be implemented. Internally, we have invested heavily in chemical biology to expand our AC-MS and in-cell target interrogation techniques. In addition, technologies that utilize label-free SMs for target enrichment need to be exploited because chemistry resources are not required to make linked probes. Another angle that we are pursuing for SMs derived from PDD is to identify druggable signaling nodes that connect target engagement (known or unknown) with the phenotypic end point. For example, deep-dive phosphoproteomic profiling may reveal druggable targets that connect SM-mediated target modulation with pathway inhibition. This data-driven approach may reveal unappreciated mechanisms that would not be visible using a candidate protein strategy. Layering SM-positive selection genome-wide pooled CRIPSR screening, where applicable, to functionalize proteins in the signaling axes may provide another level of orthogonality. Overall, more detailed information sharing will drive best practices and increase success rates for revealing SM targets and MOA.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b01921" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04520" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04520" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anil Vasudevan</span> - <span class="hlFld-Affiliation affiliation">AbbVie
Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0004-0497" title="Orcid link">http://orcid.org/0000-0002-0004-0497</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bbfad5d2d795eddac8cedfdecddad5fbfad9d9cdd2de95d8d4d6"><span class="__cf_email__" data-cfemail="90d1fef9fcbec6f1e3e5f4f5e6f1fed0d1f2f2e6f9f5bef3fffd">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kenneth M. Comess</span> - <span class="hlFld-Affiliation affiliation">AbbVie
Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaun M. McLoughlin</span> - <span class="hlFld-Affiliation affiliation">AbbVie
Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jon A. Oyer</span> - <span class="hlFld-Affiliation affiliation">AbbVie
Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul L. Richardson</span> - <span class="hlFld-Affiliation affiliation">AbbVie
Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Henning Stöckmann</span> - <span class="hlFld-Affiliation affiliation">AbbVie
Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2409-5476" title="Orcid link">http://orcid.org/0000-0002-2409-5476</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott E. Warder</span> - <span class="hlFld-Affiliation affiliation">AbbVie
Inc., 1 Waukegan Road, North Chicago, Illinois 60064-1802, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70304" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70304" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Kenneth M. Comess</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=BIO-d7e1880-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Kenneth M. Comess</b> is a Principal Research Scientist at AbbVie. He received his B.S. degree at Stanford University and then helped develop ferrioxamine, an early MRI contrast agent, at Salutar (Sunnyvale, CA). He moved to MIT to study cisplatin mechanism of action as a graduate student and then completed an American Cancer Society postdoctoral fellowship with Christopher Walsh at Harvard Medical School. In 1993, he joined Genetics Institute (Cambridge, MA) as a staff scientist, where he elucidated the PSGL-1/P-selectin molecular interaction determinants. Ken joined Abbott Laboratories (now AbbVie) in 1997 and has assumed positions of increasing responsibility focusing on high throughput screening and target ID technologies, including development of multiple ultrahigh throughput methods for ligand affinity-based HTS.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Shaun M. McLoughlin</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=BIO-d7e1885-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Shaun M. McLoughlin</b> received a BS in chemistry from Xavier University and a Ph.D. in chemistry from the University of Illinois Urbana—Champaign in the lab of Prof. Neil Kelleher studying natural product assembly by top-down proteomics. He has been with AbbVie for 12 years, focusing on target and mechanism identification using proteomics and orthogonal technologies.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Jon A. Oyer</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=BIO-d7e1890-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jon A. Oyer</b> is a Senior Scientist within the Genomic Research Center at AbbVie. Here he specializes in applying transcriptional profiling and gene targeting approaches to small-molecule characterization. Jon received his undergraduate degree from University of Washington and subsequently studied contributions of genomic instability on tumor progression in the laboratory of Peter Rabinovitch. Next, he completed his graduate studies by establishing a sequence of molecular events that initiate aberrant epigenetic silencing in the laboratory of Mitch Turker at Oregon Health & Science University. Prior to joining AbbVie, Jon conducted postdoctoral research examining epigenetic regulation during neuronal differentiation with Gail Mandel at Oregon Health & Science University and epigenetic disruptions that drive hematological malignancies with Jonathan Licht at Northwestern University.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Paul L. Richardson</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=BIO-d7e1895-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Paul L. Richardson</b> is a Principal Research Scientist and heads the Molecular Probes Group in Discovery Chemistry and Technologies, AbbVie. He has been in the Molecular Probes Group since joining Abbott Laboratories (now AbbVie) in 1994, pursuing a wide variety of synthetic/chemical biology interests including all manner of peptide technologies, fluorescence-based assays, affinity/photoaffinity-based chemoproteomics target/off-target identification/confirmation, and target engagement/validation methods. Prior to joining Abbott Laboratories, Paul was a graduate researcher at Yale University, where he received his Ph.D. in Organic Chemistry under Professor Alanna Schepartz. He received his BS honors degree in chemistry from the University of Wisconsin—Madison in 1988 and studied solid-phase photochemistry under Professor Howard Zimmerman and agronomy under Professor James Coors.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Henning Stöckmann</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=BIO-d7e1900-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Henning Stöckmann</b> received his MS degree in Chemical Biology at the University of Leeds (UK), followed by a Ph.D. degree in Chemical Biology at the University of Cambridge (UK), where he developed new agents for in vivo cancer imaging. In 2012, he joined an Irish biotech company, where he pioneered a leading high-throughput bioanalytical platform for systems biology. In 2014, he joined AbbVie as a Senior Scientist. A major focus of his work has been the development of new target identification methods for cell-surface proteins as well as technology to enable high-sensitivity target engagement and ligand binding studies for small-molecule and biologic drugs. He also leads projects in adjacent areas such as new technology for molecular imaging.</p></figure></div><div class="bio" rid="ath6"><figure id="ath6" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Anil Vasudevan</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=BIO-d7e1905-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Anil Vasudevan</b> is currently a Director at AbbVie in North Chicago, where he has responsibilities for platform chemistry technologies, flexible chemistry, chemical biology, molecular probes and new therapeutic modalities. In his career at Abbott and AbbVie, he has worked on several lead generation and optimization programs in metabolic diseases, oncology, immunology, immuno-oncology and neuroscience. In addition, his team has developed multiple chemistry and chemical biology techniques aimed at compressing cycle times and improving the probability of technical success. He is a member of the Global Medicinal Chemistry Leadership team that is responsible for setting the overall Discovery chemistry strategy across AbbVie.</p></figure></div><div class="bio" rid="ath7"><figure id="ath7" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scott E. Warder</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=BIO-d7e1910-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Scott E. Warder</b> is a Principal Research Scientist and leads the Target Identification and Mechanism of Action Group, part of the Genomics Research Center at AbbVie. One of the primary goals of this team is to prioritize early and later-stage leads (small molecules, biologics, and genetic perturbations) from target-based and phenotypic drug discovery campaigns. Scott joined Abbott (now AbbVie) in 2001 as part of a Proteomics initiative. He has been a member of the Exploratory Toxicology, Oncology, and High-Throughput Biology Groups. Scott’s efforts have resulted in the discovery and advancement of unprecedented targets, including both preclinical and clinical programs. Scott received his B.Sc. from the University of Waterloo and his Ph.D. from the University of Notre Dame.</p></figure></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">SM</td><td class="NLM_def"><p class="first last">small molecule</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">PDD</td><td class="NLM_def"><p class="first last">phenotypic drug discovery</p></td></tr><tr><td class="NLM_term">TDD</td><td class="NLM_def"><p class="first last">target-based drug discovery</p></td></tr><tr><td class="NLM_term">AI</td><td class="NLM_def"><p class="first last">artificial intelligence</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high throughput screening</p></td></tr><tr><td class="NLM_term">GEx</td><td class="NLM_def"><p class="first last">gene expression</p></td></tr><tr><td class="NLM_term">ETC</td><td class="NLM_def"><p class="first last">electron transport chain</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">endoplasmic reticulum</p></td></tr><tr><td class="NLM_term">CMap</td><td class="NLM_def"><p class="first last">connectivity map</p></td></tr><tr><td class="NLM_term">LINCS</td><td class="NLM_def"><p class="first last">Library of Integrated Network-Based Cellular Signatures</p></td></tr><tr><td class="NLM_term">OXPHOS</td><td class="NLM_def"><p class="first last">oxidative phosphorylation</p></td></tr><tr><td class="NLM_term">DR</td><td class="NLM_def"><p class="first last">dose response</p></td></tr><tr><td class="NLM_term">MTI</td><td class="NLM_def"><p class="first last">microtubule inhibitor</p></td></tr><tr><td class="NLM_term">MP</td><td class="NLM_def"><p class="first last">molecule phenotyping</p></td></tr><tr><td class="NLM_term">DR-L1000</td><td class="NLM_def"><p class="first last">dose–response L1000</p></td></tr><tr><td class="NLM_term">CCL</td><td class="NLM_def"><p class="first last">cancer cell line</p></td></tr><tr><td class="NLM_term">ETP</td><td class="NLM_def"><p class="first last">early triage panel</p></td></tr><tr><td class="NLM_term">DS</td><td class="NLM_def"><p class="first last">diversity screen</p></td></tr><tr><td class="NLM_term">gRNA</td><td class="NLM_def"><p class="first last">guide RNA</p></td></tr><tr><td class="NLM_term">DHODH</td><td class="NLM_def"><p class="first last">dihydroorotate dehydrogenase</p></td></tr><tr><td class="NLM_term">shRNA</td><td class="NLM_def"><p class="first last">short hairpin RNA</p></td></tr><tr><td class="NLM_term">CRISPR</td><td class="NLM_def"><p class="first last">clustered regularly interspaced short palindromic repeats</p></td></tr><tr><td class="NLM_term">FACS</td><td class="NLM_def"><p class="first last">fluorescence activated cell sorting</p></td></tr><tr><td class="NLM_term">GFP</td><td class="NLM_def"><p class="first last">green fluorescent protein</p></td></tr><tr><td class="NLM_term">PAL</td><td class="NLM_def"><p class="first last">photo affinity labeling</p></td></tr><tr><td class="NLM_term">ABPP</td><td class="NLM_def"><p class="first last">activity based protein profiling</p></td></tr><tr><td class="NLM_term">RBP</td><td class="NLM_def"><p class="first last">reactivity-based profiling</p></td></tr><tr><td class="NLM_term">GPI</td><td class="NLM_def"><p class="first last">glycophosphatidylinositol</p></td></tr><tr><td class="NLM_term">LDT</td><td class="NLM_def"><p class="first last">ligand-directed tosyl</p></td></tr><tr><td class="NLM_term">LDAI</td><td class="NLM_def"><p class="first last">ligand-directed acyl imidazole</p></td></tr><tr><td class="NLM_term">CCMS</td><td class="NLM_def"><p class="first last">capture compound mass spectrometry</p></td></tr><tr><td class="NLM_term">TEV</td><td class="NLM_def"><p class="first last">tobacco etch virus</p></td></tr><tr><td class="NLM_term">TSA</td><td class="NLM_def"><p class="first last">thermal shift analysis</p></td></tr><tr><td class="NLM_term">CETSA-MS</td><td class="NLM_def"><p class="first last">cellular thermal shift analysis–mass spectrometry</p></td></tr><tr><td class="NLM_term">HIP</td><td class="NLM_def"><p class="first last">haploinsufficiency</p></td></tr><tr><td class="NLM_term">HOP</td><td class="NLM_def"><p class="first last">homozygous profiling</p></td></tr><tr><td class="NLM_term">DARTS</td><td class="NLM_def"><p class="first last">drug affinity responsive target stability</p></td></tr><tr><td class="NLM_term">SPROX</td><td class="NLM_def"><p class="first last">stability of proteins from rates of oxidation</p></td></tr><tr><td class="NLM_term">PLIMSTEX</td><td class="NLM_def"><p class="first last">protein–ligand interactions by mass spectrometry, titration and H/D exchange</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescent resonance energy transfer</p></td></tr><tr><td class="NLM_term">Nluc</td><td class="NLM_def"><p class="first last">NanoLuc luciferase</p></td></tr><tr><td class="NLM_term">TCO</td><td class="NLM_def"><p class="first last"><i>trans</i>-cyclooctene</p></td></tr><tr><td class="NLM_term">ASMS</td><td class="NLM_def"><p class="first last">affinity selection mass spectrometry</p></td></tr><tr><td class="NLM_term">AC-MS</td><td class="NLM_def"><p class="first last">affinity capture-mass spectrometry</p></td></tr><tr><td class="NLM_term">PCA</td><td class="NLM_def"><p class="first last">Protein-fragment complementation assays</p></td></tr><tr><td class="NLM_term">BMDMs</td><td class="NLM_def"><p class="first last">bone marrow-derived macrophages</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i53">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53368" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53368" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 172 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedrani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span> <span> </span><span class="NLM_article-title">The discovery of first-in-class drugs: origins and evolution</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1038/nrd4336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd4336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25033734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=577-587&author=J.+Ederauthor=R.+Sedraniauthor=C.+Wiesmann&title=The+discovery+of+first-in-class+drugs%3A+origins+and+evolution&doi=10.1038%2Fnrd4336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of first-in-class drugs: origins and evolution</span></div><div class="casAuthors">Eder, Jorg; Sedrani, Richard; Wiesmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">577-587</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Anal. of the origins of new drugs approved by the US Food and Drug Administration (FDA) from 1999 to 2008 suggested that phenotypic screening strategies had been more productive than target-based approaches in the discovery of first-in-class small-mol. drugs.  However, given the relatively recent introduction of target-based approaches in the context of the long time frames of drug development, their full impact might not yet have become apparent.  Here, we present an anal. of the origins of all 113 first-in-class drugs approved by the FDA from 1999 to 2013, which shows that the majority (78) were discovered through target-based approaches (45 small-mol. drugs and 33 biologics).  In addn., of 33 drugs identified in the absence of a target hypothesis, 25 were found through a chemocentric approach in which compds. with known pharmacol. served as the starting point, with only eight coming from what we define here as phenotypic screening: testing a large no. of compds. in a target-agnostic assay that monitors phenotypic changes.  We also discuss the implications for drug discovery strategies, including viewing phenotypic screening as a novel discipline rather than as a neoclassical approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqthEelveRWN7Vg90H21EOLACvtfcHk0ljATM7MHNJknw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I&md5=b2061d96a71893a329b5b654acec5420</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd4336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4336%26sid%3Dliteratum%253Aachs%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DWiesmann%26aufirst%3DC.%26atitle%3DThe%2520discovery%2520of%2520first-in-class%2520drugs%253A%2520origins%2520and%2520evolution%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D577%26epage%3D587%26doi%3D10.1038%2Fnrd4336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span> <span> </span><span class="NLM_article-title">Phenotypic screening in cancer drug discovery - past, present and future</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1038/nrd4366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd4366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25033736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=588-602&author=J.+G.+Moffatauthor=J.+Rudolphauthor=D.+Bailey&title=Phenotypic+screening+in+cancer+drug+discovery+-+past%2C+present+and+future&doi=10.1038%2Fnrd4366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening in cancer drug discovery - past, present and future</span></div><div class="casAuthors">Moffat, John G.; Rudolph, Joachim; Bailey, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">588-602</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in the use of phenotypic screens in drug discovery as an alternative to target-focused approaches.  Given that oncol. is currently the most active therapeutic area, and also one in which target-focused approaches have been particularly prominent in the past two decades, we investigated the contribution of phenotypic assays to oncol. drug discovery by analyzing the origins of all new small-mol. cancer drugs approved by the US Food and Drug Administration (FDA) over the past 15 years and those currently in clin. development.  Although the majority of these drugs originated from target-based discovery, we identified a significant no. whose discovery depended on phenotypic screening approaches.  We postulate that the contribution of phenotypic screening to cancer drug discovery has been hampered by a reliance on 'classical' nonspecific drug effects such as cytotoxicity and mitotic arrest, exacerbated by a paucity of mechanistically defined cellular models for therapeutically translatable cancer phenotypes.  However, tech. and biol. advances that enable such mechanistically informed phenotypic models have the potential to empower phenotypic drug discovery in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTWtoTIgWqvbVg90H21EOLACvtfcHk0ljATM7MHNJknw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L&md5=0e8efed90e5bc4b47f959b83a6ab5b96</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd4366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4366%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DD.%26atitle%3DPhenotypic%2520screening%2520in%2520cancer%2520drug%2520discovery%2520-%2520past%252C%2520present%2520and%2520future%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D588%26epage%3D602%26doi%3D10.1038%2Fnrd4366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunotto, M.</span></span> <span> </span><span class="NLM_article-title">Opportunities and challenges in phenotypic drug discovery: an industry perspective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd.2017.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28685762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=531-543&author=J.+G.+Moffatauthor=F.+Vincentauthor=J.+A.+Leeauthor=J.+Ederauthor=M.+Prunotto&title=Opportunities+and+challenges+in+phenotypic+drug+discovery%3A+an+industry+perspective&doi=10.1038%2Fnrd.2017.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities and challenges in phenotypic drug discovery: an industry perspective</span></div><div class="casAuthors">Moffat, John G.; Vincent, Fabien; Lee, Jonathan A.; Eder, Jorg; Prunotto, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">531-543</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Phenotypic drug discovery (PDD) approaches do not rely on knowledge of the identity of a specific drug target or a hypothesis about its role in disease, in contrast to the target-based strategies that have been widely used in the pharmaceutical industry in the past three decades.  However, in recent years, there has been a resurgence in interest in PDD approaches based on their potential to address the incompletely understood complexity of diseases and their promise of delivering first-in-class drugs, as well as major advances in the tools for cell-based phenotypic screening.  Nevertheless, PDD approaches also have considerable challenges, such as hit validation and target deconvolution.  This article focuses on the lessons learned by researchers engaged in PDD in the pharmaceutical industry and considers the impact of 'omics' knowledge in defining a cellular disease phenotype in the era of precision medicine, introducing the concept of a chain of translatability.  We particularly aim to identify features and areas in which PDD can best deliver value to drug discovery portfolios and can contribute to the identification and the development of novel medicines, and to illustrate the challenges and uncertainties that are assocd. with PDD in order to help set realistic expectations with regard to its benefits and costs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrD5TsU8_kLVg90H21EOLACvtfcHk0lj9L-3INEyQBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ&md5=d83196715c879a81d9eabcf544e303db</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.111%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DJ.%2BA.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DPrunotto%26aufirst%3DM.%26atitle%3DOpportunities%2520and%2520challenges%2520in%2520phenotypic%2520drug%2520discovery%253A%2520an%2520industry%2520perspective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D531%26epage%3D543%26doi%3D10.1038%2Fnrd.2017.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, J.</span></span> <span> </span><span class="NLM_article-title">How were new medicines discovered?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1038/nrd3480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd3480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21701501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=507-519&author=D.+C.+Swinneyauthor=J.+Anthony&title=How+were+new+medicines+discovered%3F&doi=10.1038%2Fnrd3480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">How were new medicines discovered?</span></div><div class="casAuthors">Swinney, David C.; Anthony, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">507-519</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Preclin. strategies that are used to identify potential drug candidates include target-based screening, phenotypic screening, modification of natural substances and biol.-based approaches.  To investigate whether some strategies have been more successful than others in the discovery of new drugs, we analyzed the discovery strategies and the mol. mechanism of action (MMOA) for new mol. entities and new biologics that were approved by the US Food and Drug Administration between 1999 and 2008.  Out of the 259 agents that were approved, 75 were first-in-class drugs with new MMOAs, and out of these, 50 (67%) were small mols. and 25 (33%) were biologics.  The results also show that the contribution of phenotypic screening to the discovery of first-in-class small-mol. drugs exceeded that of target-based approaches - with 28 and 17 of these drugs coming from the two approaches, resp. - in an era in which the major focus was on target-based approaches.  We postulate that a target-centric approach for first-in-class drugs, without consideration of an optimal MMOA, may contribute to the current high attrition rates and low productivity in pharmaceutical research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE8Ah1iNz76bVg90H21EOLACvtfcHk0lj9L-3INEyQBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D&md5=7503620f8fbadb0a829f6ea2029936a2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd3480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3480%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26aulast%3DAnthony%26aufirst%3DJ.%26atitle%3DHow%2520were%2520new%2520medicines%2520discovered%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D507%26epage%3D519%26doi%3D10.1038%2Fnrd3480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pregel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitching, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nocka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyonnas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, M. C.</span></span> <span> </span><span class="NLM_article-title">Developing predictive assays: the phenotypic screening ″rule of 3″</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">293ps15</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aab1201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscitranslmed.aab1201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26109101" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=293ps15&author=F.+Vincentauthor=P.+Loriaauthor=M.+Pregelauthor=R.+Stantonauthor=L.+Kitchingauthor=K.+Nockaauthor=R.+Doyonnasauthor=C.+Steppanauthor=A.+Gilbertauthor=T.+Schroeterauthor=M.+C.+Peakman&title=Developing+predictive+assays%3A+the+phenotypic+screening+%E2%80%B3rule+of+3%E2%80%B3&doi=10.1126%2Fscitranslmed.aab1201"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aab1201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aab1201%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLoria%26aufirst%3DP.%26aulast%3DPregel%26aufirst%3DM.%26aulast%3DStanton%26aufirst%3DR.%26aulast%3DKitching%26aufirst%3DL.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DDoyonnas%26aufirst%3DR.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DGilbert%26aufirst%3DA.%26aulast%3DSchroeter%26aufirst%3DT.%26aulast%3DPeakman%26aufirst%3DM.%2BC.%26atitle%3DDeveloping%2520predictive%2520assays%253A%2520the%2520phenotypic%2520screening%2520%25E2%2580%25B3rule%2520of%25203%25E2%2580%25B3%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2015%26volume%3D7%26spage%3D293ps15%26doi%3D10.1126%2Fscitranslmed.aab1201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rees, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seashore-Ludlow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheah, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javaid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coletti, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodycombe, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soule, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hon, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefeld, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dancik, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Correlating chemical sensitivity and basal gene expression reveals mechanism of action</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1986</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.1986" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26656090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=109-116&author=M.+G.+Reesauthor=B.+Seashore-Ludlowauthor=J.+H.+Cheahauthor=D.+J.+Adamsauthor=E.+V.+Priceauthor=S.+Gillauthor=S.+Javaidauthor=M.+E.+Colettiauthor=V.+L.+Jonesauthor=N.+E.+Bodycombeauthor=C.+K.+Souleauthor=B.+Alexanderauthor=A.+Liauthor=P.+Montgomeryauthor=J.+D.+Kotzauthor=C.+S.+Honauthor=B.+Munozauthor=T.+Liefeldauthor=V.+Dancikauthor=D.+A.+Haberauthor=C.+B.+Clishauthor=J.+A.+Bittkerauthor=M.+Palmerauthor=B.+K.+Wagnerauthor=P.+A.+Clemonsauthor=A.+F.+Shamjiauthor=S.+L.+Schreiber&title=Correlating+chemical+sensitivity+and+basal+gene+expression+reveals+mechanism+of+action&doi=10.1038%2Fnchembio.1986"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Correlating chemical sensitivity and basal gene expression reveals mechanism of action</span></div><div class="casAuthors">Rees, Matthew G.; Seashore-Ludlow, Brinton; Cheah, Jaime H.; Adams, Drew J.; Price, Edmund V.; Gill, Shubhroz; Javaid, Sarah; Coletti, Matthew E.; Jones, Victor L.; Bodycombe, Nicole E.; Soule, Christian K.; Alexander, Benjamin; Li, Ava; Montgomery, Philip; Kotz, Joanne D.; Hon, C. Suk-Yee; Munoz, Benito; Liefeld, Ted; Dancik, Vlado; Haber, Daniel A.; Clish, Clary B.; Bittker, Joshua A.; Palmer, Michelle; Wagner, Bridget K.; Clemons, Paul A.; Shamji, Alykhan F.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">109-116</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Changes in cellular gene expression in response to small-mol. or genetic perturbations have yielded signatures that can connect unknown mechanisms of action (MoA) to ones previously established.  We hypothesized that differential basal gene expression could be correlated with patterns of small-mol. sensitivity across many cell lines to illuminate the actions of compds. whose MoA are unknown.  To test this idea, we correlated the sensitivity patterns of 481 compds. with ∼19,000 basal transcript levels across 823 different human cancer cell lines and identified selective outlier transcripts.  This process yielded many novel mechanistic insights, including the identification of activation mechanisms, cellular transporters and direct protein targets.  We found that ML239, originally identified in a phenotypic screen for selective cytotoxicity in breast cancer stem-like cells, most likely acts through activation of fatty acid desaturase 2 (FADS2).  These data and anal. tools are available to the research community through the Cancer Therapeutics Response Portal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTUYwiQeWDrVg90H21EOLACvtfcHk0lgOPcEyvDgV3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtb3K&md5=8e4b5c55bd8486b0b06f917ac937df7f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1986&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1986%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DM.%2BG.%26aulast%3DSeashore-Ludlow%26aufirst%3DB.%26aulast%3DCheah%26aufirst%3DJ.%2BH.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DPrice%26aufirst%3DE.%2BV.%26aulast%3DGill%26aufirst%3DS.%26aulast%3DJavaid%26aufirst%3DS.%26aulast%3DColetti%26aufirst%3DM.%2BE.%26aulast%3DJones%26aufirst%3DV.%2BL.%26aulast%3DBodycombe%26aufirst%3DN.%2BE.%26aulast%3DSoule%26aufirst%3DC.%2BK.%26aulast%3DAlexander%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DMontgomery%26aufirst%3DP.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DHon%26aufirst%3DC.%2BS.%26aulast%3DMunoz%26aufirst%3DB.%26aulast%3DLiefeld%26aufirst%3DT.%26aulast%3DDancik%26aufirst%3DV.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DCorrelating%2520chemical%2520sensitivity%2520and%2520basal%2520gene%2520expression%2520reveals%2520mechanism%2520of%2520action%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D109%26epage%3D116%26doi%3D10.1038%2Fnchembio.1986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swalley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoosear, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLellan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuierer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaustein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berenshteyn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacoste, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span> <span> </span><span class="NLM_article-title">SMN2 splice modulators enhance U1-pre-mRNA association and rescue SMA mice</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.1837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26030728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=511-517&author=J.+Palacinoauthor=S.+E.+Swalleyauthor=C.+Songauthor=A.+K.+Cheungauthor=L.+Shuauthor=X.+Zhangauthor=M.+Van+Hoosearauthor=Y.+Shinauthor=D.+N.+Chinauthor=C.+G.+Kellerauthor=M.+Beibelauthor=N.+A.+Renaudauthor=T.+M.+Smithauthor=M.+Salciusauthor=X.+Shiauthor=M.+Hildauthor=R.+Servaisauthor=M.+Jainauthor=L.+Dengauthor=C.+Bullockauthor=M.+McLellanauthor=S.+Schuiererauthor=L.+Murphyauthor=M.+J.+Blommersauthor=C.+Blausteinauthor=F.+Berenshteynauthor=A.+Lacosteauthor=J.+R.+Thomasauthor=G.+Romaauthor=G.+A.+Michaudauthor=B.+S.+Tsengauthor=J.+A.+Porterauthor=V.+E.+Myerauthor=J.+A.+Tallaricoauthor=L.+G.+Hamannauthor=D.+Curtisauthor=M.+C.+Fishmanauthor=W.+F.+Dietrichauthor=N.+A.+Dalesauthor=R.+Sivasankaran&title=SMN2+splice+modulators+enhance+U1-pre-mRNA+association+and+rescue+SMA+mice&doi=10.1038%2Fnchembio.1837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splice modulators enhance U1-pre-mRNA association and rescue SMA mice</span></div><div class="casAuthors">Palacino, James; Swalley, Susanne E.; Song, Cheng; Cheung, Atwood K.; Shu, Lei; Zhang, Xiaolu; Van Hoosear, Mailin; Shin, Youngah; Chin, Donovan N.; Keller, Caroline Gubser; Beibel, Martin; Renaud, Nicole A.; Smith, Thomas M.; Salcius, Michael; Shi, Xiaoying; Hild, Marc; Servais, Rebecca; Jain, Monish; Deng, Lin; Bullock, Caroline; McLellan, Michael; Schuierer, Sven; Murphy, Leo; Blommers, Marcel J. J.; Blaustein, Cecile; Berenshteyn, Frada; Lacoste, Arnaud; Thomas, Jason R.; Roma, Guglielmo; Michaud, Gregory A.; Tseng, Brian S.; Porter, Jeffery A.; Myer, Vic E.; Tallarico, John A.; Hamann, Lawrence G.; Curtis, Daniel; Fishman, Mark C.; Dietrich, William F.; Dales, Natalie A.; Sivasankaran, Rajeev</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">511-517</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), which results from the loss of expression of the survival of motor neuron-1 (SMN1) gene, represents the most common genetic cause of pediatric mortality.  A duplicate copy (SMN2) is inefficiently spliced, producing a truncated and unstable protein.  We describe herein a potent, orally active, small-mol. enhancer of SMN2 splicing that elevates full-length SMN protein and extends survival in a severe SMA mouse model.  We demonstrate that the mol. mechanism of action is via stabilization of the transient double-strand RNA structure formed by the SMN2 pre-mRNA and U1 small nuclear ribonucleic protein (snRNP) complex.  The binding affinity of U1 snRNP to the 5' splice site is increased in a sequence-selective manner, discrete from constitutive recognition.  This new mechanism demonstrates the feasibility of small mol.-mediated, sequence-selective splice modulation and the potential for leveraging this strategy in other splicing diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdMqLIC0lw17Vg90H21EOLACvtfcHk0lgOPcEyvDgV3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN&md5=15e56c8b3f48feffa150e975a81633a0</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1837%26sid%3Dliteratum%253Aachs%26aulast%3DPalacino%26aufirst%3DJ.%26aulast%3DSwalley%26aufirst%3DS.%2BE.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVan%2BHoosear%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DKeller%26aufirst%3DC.%2BG.%26aulast%3DBeibel%26aufirst%3DM.%26aulast%3DRenaud%26aufirst%3DN.%2BA.%26aulast%3DSmith%26aufirst%3DT.%2BM.%26aulast%3DSalcius%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DBullock%26aufirst%3DC.%26aulast%3DMcLellan%26aufirst%3DM.%26aulast%3DSchuierer%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DL.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%26aulast%3DBlaustein%26aufirst%3DC.%26aulast%3DBerenshteyn%26aufirst%3DF.%26aulast%3DLacoste%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DJ.%2BR.%26aulast%3DRoma%26aufirst%3DG.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DTseng%26aufirst%3DB.%2BS.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DFishman%26aufirst%3DM.%2BC.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DDales%26aufirst%3DN.%2BA.%26aulast%3DSivasankaran%26aufirst%3DR.%26atitle%3DSMN2%2520splice%2520modulators%2520enhance%2520U1-pre-mRNA%2520association%2520and%2520rescue%2520SMA%2520mice%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D511%26epage%3D517%26doi%3D10.1038%2Fnchembio.1837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span> <span> </span><span class="NLM_article-title">Target identification for biologically active small molecules using chemical biology approaches</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1193</span>– <span class="NLM_lpage">1201</span>, <span class="refDoi"> DOI: 10.1007/s12272-016-0791-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2Fs12272-016-0791-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27387321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFertLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2016&pages=1193-1201&author=H.+Leeauthor=J.+W.+Lee&title=Target+identification+for+biologically+active+small+molecules+using+chemical+biology+approaches&doi=10.1007%2Fs12272-016-0791-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Target identification for biologically active small molecules using chemical biology approaches</span></div><div class="casAuthors">Lee, Heesu; Lee, Jae Wook</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1193-1201</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">A review.  The identification and validation of the targets of biol. active mols. is an important step in the field of chem. biol.  While recent advances in proteomic and genomic technol. have accelerated this identification process, the discovery of small mol. targets remains the most challenging step.  A general method for the identification of these small mol. targets has not yet been established.  To overcome the difficulty in target identification, new technol. derived from the fields of genomics, proteomics, and bioinformatics has been developed.  To date, pull-down methods using small mols. immobilized on a solid support followed by mass spectrometry have been the most successful approach.  Here, current procedures for target identification are discussed.  The authors also review the most recent target identification approaches and present several examples that illustrate advanced target identification technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEIcJ1_suxrbVg90H21EOLACvtfcHk0lhecu9ee3r7bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFertLrF&md5=f97036e221fd6e1265d58020b1478a5a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs12272-016-0791-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12272-016-0791-z%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26atitle%3DTarget%2520identification%2520for%2520biologically%2520active%2520small%2520molecules%2520using%2520chemical%2520biology%2520approaches%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2016%26volume%3D39%26spage%3D1193%26epage%3D1201%26doi%3D10.1007%2Fs12272-016-0791-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saxena, C.</span></span> <span> </span><span class="NLM_article-title">Identification of protein binding partners of small molecules using label-free methods</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1017</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1080/17460441.2016.1227316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1080%2F17460441.2016.1227316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27554241" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1017-1025&author=C.+Saxena&title=Identification+of+protein+binding+partners+of+small+molecules+using+label-free+methods&doi=10.1080%2F17460441.2016.1227316"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1080%2F17460441.2016.1227316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2016.1227316%26sid%3Dliteratum%253Aachs%26aulast%3DSaxena%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520protein%2520binding%2520partners%2520of%2520small%2520molecules%2520using%2520label-free%2520methods%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2016%26volume%3D11%26spage%3D1017%26epage%3D1025%26doi%3D10.1080%2F17460441.2016.1227316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span> <span> </span><span class="NLM_article-title">The impact of CRISPR-Cas9 on target identification and validation</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.12.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.drudis.2014.12.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25572406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=450-457&author=J.+D.+Moore&title=The+impact+of+CRISPR-Cas9+on+target+identification+and+validation&doi=10.1016%2Fj.drudis.2014.12.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of CRISPR-Cas9 on target identification and validation</span></div><div class="casAuthors">Moore, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">450-457</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The addn. of an RNA-guided nuclease, Cas9, to the gene editing toolbox has increased the accessibility of gene editing technologies by greatly simplifying the design of editing reagents.  Only a single 75-100 nucleotide RNA is required to guide Cas9 to the target gene of interest, which has meant that the established infrastructure of short-hairpin RNA interference screen could be readily adapted to genome-wide knock out screens.  Cas9-based editing technol. should streamline the generation of animal and cell-line models, make the generation of activity-dead mutations in target validation routine, and enable the discovery of a new generation of targets across therapeutic areas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbf_b_V0znILVg90H21EOLACvtfcHk0lj52JudtrFhfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFWkuw%253D%253D&md5=5d1fcdcf51f9511cc4dad583c65f958d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.12.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.12.016%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520impact%2520of%2520CRISPR-Cas9%2520on%2520target%2520identification%2520and%2520validation%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D450%26epage%3D457%26doi%3D10.1016%2Fj.drudis.2014.12.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">The power of sophisticated phenotypic screening and modern mechanism-of-action methods</span>. <i>Cell Chem Biol</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2015.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26933731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlSqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=3-9&author=B.+K.+Wagnerauthor=S.+L.+Schreiber&title=The+power+of+sophisticated+phenotypic+screening+and+modern+mechanism-of-action+methods&doi=10.1016%2Fj.chembiol.2015.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The Power of Sophisticated Phenotypic Screening and Modern Mechanism-of-Action Methods</span></div><div class="casAuthors">Wagner, Bridget K.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The enthusiasm for phenotypic screening as an approach for small-mol. discovery has increased dramatically over the last several years.  The recent increase in phenotype-based discoveries is in part due to advancements in phenotypic readouts in improved disease models that recapitulate clin. relevant biol. in cell culture.  Of course, a major historical barrier to using phenotypic assays in chem. biol. has been the challenge in detg. the mechanism of action (MoA) for compds. of interest.  With the combination of medically inspired phenotypic screening and the development of modern MoA methods, we can now start implementing this approach in chem. probe and drug discovery.  In this Perspective, we highlight recent advances in phenotypic readouts and MoA detn. by discussing several case studies in which both activities were required for understanding the chem. biol. involved and, in some cases, advancing toward clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVxdoheLgSbbVg90H21EOLACvtfcHk0lj52JudtrFhfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlSqsb4%253D&md5=e3219e156466fa5a32b2cb4bf6869a90</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DThe%2520power%2520of%2520sophisticated%2520phenotypic%2520screening%2520and%2520modern%2520mechanism-of-action%2520methods%26jtitle%3DCell%2520Chem%2520Biol%26date%3D2016%26volume%3D23%26spage%3D3%26epage%3D9%26doi%3D10.1016%2Fj.chembiol.2015.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pries, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Target identification for small bioactive molecules: finding the needle in the haystack</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2744</span>– <span class="NLM_lpage">2792</span>, <span class="refDoi"> DOI: 10.1002/anie.201208749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201208749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23418026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislSksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=2744-2792&author=S.+Zieglerauthor=V.+Priesauthor=C.+Hedbergauthor=H.+Waldmann&title=Target+identification+for+small+bioactive+molecules%3A+finding+the+needle+in+the+haystack&doi=10.1002%2Fanie.201208749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Target Identification for Small Bioactive Molecules: Finding the Needle in the Haystack</span></div><div class="casAuthors">Ziegler, Slava; Pries, Verena; Hedberg, Christian; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2744-2792</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Identification and confirmation of bioactive small-mol. targets is a crucial, often decisive step both in academic and pharmaceutical research.  Through the development and availability of several new exptl. techniques, target identification is, in principle, feasible, and the no. of successful examples steadily grows.  However, a generic methodol. that can successfully be applied in the majority of the cases has not yet been established.  Herein we summarize current methods for target identification of small mols., primarily for a chem. audience but also the biol. community, for example, the chemist or biologist attempting to identify the target of a given bioactive compd.  We describe the most frequently employed exptl. approaches for target identification and provide several representative examples illustrating the state-of-the-art.  Among the techniques currently available, protein affinity isolation using suitable small-mol. probes (pulldown) and subsequent mass spectrometric anal. of the isolated proteins appears to be most powerful and most frequently applied.  To provide guidance for rapid entry into the field and based on our own experience we propose a typical workflow for target identification, which centers on the application of chem. proteomics as the key step to generate hypotheses for potential target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYC9-7L3abMrVg90H21EOLACvtfcHk0lh0ejIcF6rbzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislSksLk%253D&md5=3a124205e3a57a53f479b889073bda76</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201208749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201208749%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DPries%26aufirst%3DV.%26aulast%3DHedberg%26aufirst%3DC.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DTarget%2520identification%2520for%2520small%2520bioactive%2520molecules%253A%2520finding%2520the%2520needle%2520in%2520the%2520haystack%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D2744%26epage%3D2792%26doi%3D10.1002%2Fanie.201208749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Drug-target interaction prediction: databases, web servers and computational models</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">696</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1093/bib/bbv066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbib%2Fbbv066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26283676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlsVSru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=696-712&author=X.+Chenauthor=C.+C.+Yanauthor=X.+Zhangauthor=X.+Zhangauthor=F.+Daiauthor=J.+Yinauthor=Y.+Zhang&title=Drug-target+interaction+prediction%3A+databases%2C+web+servers+and+computational+models&doi=10.1093%2Fbib%2Fbbv066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target interaction prediction: databases, web servers and computational models</span></div><div class="casAuthors">Chen, Xing; Yan, Chenggang Clarence; Zhang, Xiaotian; Zhang, Xu; Dai, Feng; Yin, Jian; Zhang, Yongdong</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">696-712</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1467-5463</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Identification of drug-target interactions is an important process in drug discovery.  Although high-throughput screening and other biol. assays are becoming available, exptl. methods for drug-target interaction identification remain to be extremely costly, time-consuming and challenging even nowadays.  Therefore, various computational models have been developed to predict potential drug-target assocns. on a large scale.  In this review, databases and web servers involved in drug-target identification and drug discovery are summarized.  In addn., we mainly introduced some state-of-the-art computational models for drug-target interactions prediction, including network-based method, machine learning-based method and so on.  Specially, for the machine learning-based method, much attention was paid to super-vised and semi-supervised models, which have essential difference in the adoption of neg. samples.  Although significant improvements for drug-target interaction prediction have been obtained by many effective computational models, both network-based and machine learning-based methods have their disadvantages, resp.  Furthermore, we discuss the future directions of the network-based drug discovery and network approach for personalized drug discovery based on personalized medicine, genome sequencing, tumor clone-based network and cancer hallmark-based network.  Finally, we discussed the new evaluation validation framework and the formulation of drug-target interactions prediction problem by more realistic regression formulation based on quant. bioactivity data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMvmzj7AuWvrVg90H21EOLACvtfcHk0lh0ejIcF6rbzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlsVSru7g%253D&md5=fa9d08bd0fdf4b039acac3baad7d8d60</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbv066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbv066%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DC.%2BC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DF.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDrug-target%2520interaction%2520prediction%253A%2520databases%252C%2520web%2520servers%2520and%2520computational%2520models%26jtitle%3DBriefings%2520Bioinf.%26date%3D2016%26volume%3D17%26spage%3D696%26epage%3D712%26doi%3D10.1093%2Fbib%2Fbbv066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kremer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habenberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choidas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kordes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholer, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterneckert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel inhibitor of the Hedgehog signaling pathway through cell-based compound discovery and target prediction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">13021</span>– <span class="NLM_lpage">13025</span>, <span class="refDoi"> DOI: 10.1002/anie.201707394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201707394" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=13021-13025&author=L.+Kremerauthor=C.+Schultz-Fademrechtauthor=M.+Baumannauthor=P.+Habenbergerauthor=A.+Choidasauthor=B.+Kleblauthor=S.+Kordesauthor=H.+R.+Scholerauthor=J.+Sterneckertauthor=S.+Zieglerauthor=G.+Schneiderauthor=H.+Waldmann&title=Discovery+of+a+novel+inhibitor+of+the+Hedgehog+signaling+pathway+through+cell-based+compound+discovery+and+target+prediction&doi=10.1002%2Fanie.201707394"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707394%26sid%3Dliteratum%253Aachs%26aulast%3DKremer%26aufirst%3DL.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DHabenberger%26aufirst%3DP.%26aulast%3DChoidas%26aufirst%3DA.%26aulast%3DKlebl%26aufirst%3DB.%26aulast%3DKordes%26aufirst%3DS.%26aulast%3DScholer%26aufirst%3DH.%2BR.%26aulast%3DSterneckert%26aufirst%3DJ.%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520inhibitor%2520of%2520the%2520Hedgehog%2520signaling%2520pathway%2520through%2520cell-based%2520compound%2520discovery%2520and%2520target%2520prediction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D13021%26epage%3D13025%26doi%3D10.1002%2Fanie.201707394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecassis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguiz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeons, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidah, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constam, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetsel, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.+P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lgx2JUpb7MP6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Somody, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKinnon, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windemuth, A.</span></span> <span> </span><span class="NLM_article-title">Structural coverage of the proteome for pharmaceutical applications</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1792</span>– <span class="NLM_lpage">1799</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.drudis.2017.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28843631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtl2ltLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1792-1799&author=J.+C.+Somodyauthor=S.+S.+MacKinnonauthor=A.+Windemuth&title=Structural+coverage+of+the+proteome+for+pharmaceutical+applications&doi=10.1016%2Fj.drudis.2017.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structural coverage of the proteome for pharmaceutical applications</span></div><div class="casAuthors">Somody, Joseph C.; MacKinnon, Stephen S.; Windemuth, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1792-1799</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Structure-based computational drug discovery efforts have traditionally focused on the structure of a single, well-known drug target.  Important applications, such as target deconvolution and the anal. of polypharmacol., require proteome-scale mol. docking and have been inaccessible to structure-based in silico approaches.  One important reason for this inaccessibility was that the structure of most proteins was not known.  Lately, this 'structure gap' has been closing rapidly, and proteome-scale mol. docking seems within reach.  Here, we survey the current state of structural coverage of the human genome and find that coverage is truly proteome-wide, both overall and in most pharmaceutically relevant categories of proteins.  The time is right for structure-based approaches to target deconvolution and polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozH-nC3SqUs7Vg90H21EOLACvtfcHk0ljhj4rijLOwcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtl2ltLbL&md5=a3aae9e89ea541b6218b4f513d7218f3</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DSomody%26aufirst%3DJ.%2BC.%26aulast%3DMacKinnon%26aufirst%3DS.%2BS.%26aulast%3DWindemuth%26aufirst%3DA.%26atitle%3DStructural%2520coverage%2520of%2520the%2520proteome%2520for%2520pharmaceutical%2520applications%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D1792%26epage%3D1799%26doi%3D10.1016%2Fj.drudis.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savage, N.</span></span> <span> </span><span class="NLM_article-title">Proteomics: High-protein research</span>. <i>Nature (London, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>527</i></span>,  <span class="NLM_fpage">S6</span>– <span class="NLM_lpage">S7</span>, <span class="refDoi"> DOI: 10.1038/527S6a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2F527S6a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslOrtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=527&publication_year=2015&pages=S6-S7&author=N.+Savage&title=Proteomics%3A+High-protein+research&doi=10.1038%2F527S6a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics: High-protein research</span></div><div class="casAuthors">Savage, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">527</span>
        (<span class="NLM_cas:issue">7576</span>),
    <span class="NLM_cas:pages">S6-S7</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on how proteomics affects cell biol., the effort to catalog proteins going deeper in a push to make genetics research deliver practical benefits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqksnuhmBvFLVg90H21EOLACvtfcHk0ljhj4rijLOwcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslOrtb7E&md5=10b10630a4c2b2114de73f58facade89</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2F527S6a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F527S6a%26sid%3Dliteratum%253Aachs%26aulast%3DSavage%26aufirst%3DN.%26atitle%3DProteomics%253A%2520High-protein%2520research%26jtitle%3DNature%2520%2528London%252C%2520U.%2520K.%2529%26date%3D2015%26volume%3D527%26spage%3DS6%26epage%3DS7%26doi%3D10.1038%2F527S6a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drissi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boisvert, F.-M.</span></span> <span> </span><span class="NLM_article-title">Proteomics methods for subcellular proteome analysis</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">5626</span>– <span class="NLM_lpage">5634</span>, <span class="refDoi"> DOI: 10.1111/febs.12502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1111%2Ffebs.12502" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=5626-5634&author=R.+Drissiauthor=M.-L.+Duboisauthor=F.-M.+Boisvert&title=Proteomics+methods+for+subcellular+proteome+analysis&doi=10.1111%2Ffebs.12502"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12502%26sid%3Dliteratum%253Aachs%26aulast%3DDrissi%26aufirst%3DR.%26aulast%3DDubois%26aufirst%3DM.-L.%26aulast%3DBoisvert%26aufirst%3DF.-M.%26atitle%3DProteomics%2520methods%2520for%2520subcellular%2520proteome%2520analysis%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D5626%26epage%3D5634%26doi%3D10.1111%2Ffebs.12502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goedken, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argiriadi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiamengo, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ihle, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merta, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalak, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murdock, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, J. W.</span></span> <span> </span><span class="NLM_article-title">Tricyclic covalent inhibitors selectively target Jak3 through an active site thiol</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">4573</span>– <span class="NLM_lpage">4589</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.595181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1074%2Fjbc.M114.595181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25552479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsF2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=4573-4589&author=E.+R.+Goedkenauthor=M.+A.+Argiriadiauthor=D.+L.+Banachauthor=B.+A.+Fiamengoauthor=S.+E.+Foleyauthor=K.+E.+Frankauthor=J.+S.+Georgeauthor=C.+M.+Harrisauthor=A.+D.+Hobsonauthor=D.+C.+Ihleauthor=D.+Marcotteauthor=P.+J.+Mertaauthor=M.+E.+Michalakauthor=S.+E.+Murdockauthor=M.+J.+Tomlinsonauthor=J.+W.+Voss&title=Tricyclic+covalent+inhibitors+selectively+target+Jak3+through+an+active+site+thiol&doi=10.1074%2Fjbc.M114.595181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Tricyclic Covalent Inhibitors Selectively Target Jak3 through an Active Site Thiol</span></div><div class="casAuthors">Goedken, Eric R.; Argiriadi, Maria A.; Banach, David L.; Fiamengo, Bryan A.; Foley, Sage E.; Frank, Kristine E.; George, Jonathan S.; Harris, Christopher M.; Hobson, Adrian D.; Ihle, David C.; Marcotte, Douglas; Merta, Philip J.; Michalak, Mark E.; Murdock, Sara E.; Tomlinson, Medha J.; Voss, Jeffrey W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4573-4589</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The action of Janus kinases (JAKs) is required for multiple cytokine signaling pathways, and as such, JAK inhibitors hold promise for treatment of autoimmune disorders, including rheumatoid arthritis, inflammatory bowel disease, and psoriasis.  However, due to high similarity in the active sites of the four members (Jak1, Jak2, Jak3, and Tyk2), developing selective inhibitors within this family is challenging.  We have designed and characterized substituted, tricyclic Jak3 inhibitors that selectively avoid inhibition of the other JAKs.  This is accomplished through a covalent interaction between an inhibitor contg. a terminal electrophile and an active site cysteine (Cys-909).  We found that these ATP competitive compds. are irreversible inhibitors of Jak3 enzyme activity in vitro.  They possess high selectivity against other kinases and can potently (IC50 < 100 nm) inhibit Jak3 activity in cell-based assays.  These results suggest irreversible inhibitors of this class may be useful selective agents, both as tools to probe Jak3 biol. and potentially as therapies for autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjt9jFjpWGbVg90H21EOLACvtfcHk0lgE7KpgHoVCyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsF2lsLY%253D&md5=4d99cf5b13fd05df5c4abc197c1fe284</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.595181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.595181%26sid%3Dliteratum%253Aachs%26aulast%3DGoedken%26aufirst%3DE.%2BR.%26aulast%3DArgiriadi%26aufirst%3DM.%2BA.%26aulast%3DBanach%26aufirst%3DD.%2BL.%26aulast%3DFiamengo%26aufirst%3DB.%2BA.%26aulast%3DFoley%26aufirst%3DS.%2BE.%26aulast%3DFrank%26aufirst%3DK.%2BE.%26aulast%3DGeorge%26aufirst%3DJ.%2BS.%26aulast%3DHarris%26aufirst%3DC.%2BM.%26aulast%3DHobson%26aufirst%3DA.%2BD.%26aulast%3DIhle%26aufirst%3DD.%2BC.%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DMichalak%26aufirst%3DM.%2BE.%26aulast%3DMurdock%26aufirst%3DS.%2BE.%26aulast%3DTomlinson%26aufirst%3DM.%2BJ.%26aulast%3DVoss%26aufirst%3DJ.%2BW.%26atitle%3DTricyclic%2520covalent%2520inhibitors%2520selectively%2520target%2520Jak3%2520through%2520an%2520active%2520site%2520thiol%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D4573%26epage%3D4589%26doi%3D10.1074%2Fjbc.M114.595181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, J. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vleet, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittelstadt, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomme, E. A. G.</span></span> <span> </span><span class="NLM_article-title">Potential functional and pathological side effects related to off-target pharmacological activity</span>. <i>J. Pharmacol. Toxicol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">108</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.vascn.2017.02.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.vascn.2017.02.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28216264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVGltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2017&pages=108-126&author=J.+J.+Lynchauthor=T.+R.+Van+Vleetauthor=S.+W.+Mittelstadtauthor=E.+A.+G.+Blomme&title=Potential+functional+and+pathological+side+effects+related+to+off-target+pharmacological+activity&doi=10.1016%2Fj.vascn.2017.02.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Potential functional and pathological side effects related to off-target pharmacological activity</span></div><div class="casAuthors">Lynch, James J., III; Van Vleet, Terry R.; Mittelstadt, Scott W.; Blomme, Eric A. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological and Toxicological Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108-126</span>CODEN:
                <span class="NLM_cas:coden">JPTMEZ</span>;
        ISSN:<span class="NLM_cas:issn">1056-8719</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Most pharmaceutical companies test their discovery-stage proprietary mols. in a battery of in vitro pharmacol. assays to try to det. off-target interactions.  During all phases of drug discovery and development, various questions arise regarding potential side effects assocd. with such off-target pharmacol. activity.  Here we present a scientific literature curation effort undertaken to det. and summarize the most likely functional and pathol. outcomes assocd. with interactions at 70 receptors, enzymes, ion channels and transporters with established links to adverse effects.  To that end, the scientific literature was reviewed using an online database, and the most commonly reported effects were summarized in tabular format.  The resultant table should serve as a practical guide for research scientists and clin. investigators for the prediction and interpretation of adverse side effects assocd. with mols. interacting with components of this screening battery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr73k3f-8KK17Vg90H21EOLACvtfcHk0lgE7KpgHoVCyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsVGltrg%253D&md5=86b82acd8981a020f19e236d2d4ecb16</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.vascn.2017.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.vascn.2017.02.020%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BJ.%26aulast%3DVan%2BVleet%26aufirst%3DT.%2BR.%26aulast%3DMittelstadt%26aufirst%3DS.%2BW.%26aulast%3DBlomme%26aufirst%3DE.%2BA.%2BG.%26atitle%3DPotential%2520functional%2520and%2520pathological%2520side%2520effects%2520related%2520to%2520off-target%2520pharmacological%2520activity%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2017%26volume%3D87%26spage%3D108%26epage%3D126%26doi%3D10.1016%2Fj.vascn.2017.02.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viklund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsblom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresaugues, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quanz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak-Reppel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irlbacher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queisser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haegebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjanacz, M.</span></span> <span> </span><span class="NLM_article-title">Novel class of potent and cellularly active inhibitors devalidates MTH1 as broad-spectrum cancer target</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1986</span>– <span class="NLM_lpage">1992</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1986-1992&author=M.+Ellermannauthor=A.+Eheimauthor=F.+Rahmauthor=J.+Viklundauthor=J.+Guentherauthor=M.+Anderssonauthor=U.+Ericssonauthor=R.+Forsblomauthor=T.+Ginmanauthor=J.+Lindstromauthor=C.+Silvanderauthor=L.+Tresauguesauthor=A.+Gieseauthor=S.+Bunseauthor=R.+Neuhausauthor=J.+Weiskeauthor=M.+Quanzauthor=A.+Glasauerauthor=K.+Nowak-Reppelauthor=B.+Baderauthor=H.+Irlbacherauthor=H.+Meyerauthor=N.+Queisserauthor=M.+Bauserauthor=A.+Haegebarthauthor=M.+Gorjanacz&title=Novel+class+of+potent+and+cellularly+active+inhibitors+devalidates+MTH1+as+broad-spectrum+cancer+target&doi=10.1021%2Facschembio.7b00370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Class of Potent and Cellularly Active Inhibitors Devalidates MTH1 as Broad-Spectrum Cancer Target</span></div><div class="casAuthors">Ellermann, Manuel; Eheim, Ashley; Rahm, Fredrik; Viklund, Jenny; Guenther, Judith; Andersson, Martin; Ericsson, Ulrika; Forsblom, Rickard; Ginman, Tobias; Lindstroem, Johan; Silvander, Camilla; Tresaugues, Lionel; Giese, Anja; Bunse, Stefanie; Neuhaus, Roland; Weiske, Joerg; Quanz, Maria; Glasauer, Andrea; Nowak-Reppel, Katrin; Bader, Benjamin; Irlbacher, Horst; Meyer, Hanna; Queisser, Nina; Bauser, Marcus; Haegebarth, Andrea; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1986-1992</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MTH1 is a hydrolase responsible for sanitization of oxidized purine nucleoside triphosphates to prevent their incorporation into replicating DNA.  Early tool compds. published in the literature inhibited the enzymic activity of MTH1 and subsequently induced cancer cell death; however recent studies have questioned the reported link between these two events.  Therefore, it is important to validate MTH1 as a cancer dependency with high quality chem. probes.  Here, we present BAY-707, a substrate-competitive, highly potent and selective inhibitor of MTH1, chem. distinct compared to those previously published.  Despite superior cellular target engagement and pharmacokinetic properties, inhibition of MTH1 with BAY-707 resulted in a clear lack of in vitro or in vivo anticancer efficacy either in mono- or in combination therapies.  Therefore, we conclude that MTH1 is dispensable for cancer cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHT2Ivo0Gx87Vg90H21EOLACvtfcHk0lg2-uZ6iXY5eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsrbP&md5=69aa2247fae019cf695df8352e2ade18</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00370%26sid%3Dliteratum%253Aachs%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DEheim%26aufirst%3DA.%26aulast%3DRahm%26aufirst%3DF.%26aulast%3DViklund%26aufirst%3DJ.%26aulast%3DGuenther%26aufirst%3DJ.%26aulast%3DAndersson%26aufirst%3DM.%26aulast%3DEricsson%26aufirst%3DU.%26aulast%3DForsblom%26aufirst%3DR.%26aulast%3DGinman%26aufirst%3DT.%26aulast%3DLindstrom%26aufirst%3DJ.%26aulast%3DSilvander%26aufirst%3DC.%26aulast%3DTresaugues%26aufirst%3DL.%26aulast%3DGiese%26aufirst%3DA.%26aulast%3DBunse%26aufirst%3DS.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DQuanz%26aufirst%3DM.%26aulast%3DGlasauer%26aufirst%3DA.%26aulast%3DNowak-Reppel%26aufirst%3DK.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DIrlbacher%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DQueisser%26aufirst%3DN.%26aulast%3DBauser%26aufirst%3DM.%26aulast%3DHaegebarth%26aufirst%3DA.%26aulast%3DGorjanacz%26aufirst%3DM.%26atitle%3DNovel%2520class%2520of%2520potent%2520and%2520cellularly%2520active%2520inhibitors%2520devalidates%2520MTH1%2520as%2520broad-spectrum%2520cancer%2520target%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D1986%26epage%3D1992%26doi%3D10.1021%2Facschembio.7b00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blomme, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, Y.</span></span> <span> </span><span class="NLM_article-title">Toxicology strategies for drug discovery: present and future</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">504</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrestox.5b00407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.5b00407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOhu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=473-504&author=E.+A.+Blommeauthor=Y.+Will&title=Toxicology+strategies+for+drug+discovery%3A+present+and+future&doi=10.1021%2Facs.chemrestox.5b00407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicology Strategies for Drug Discovery: Present and Future</span></div><div class="casAuthors">Blomme, Eric A. G.; Will, Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-504</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Attrition due to nonclin. safety represents a major issue for the productivity of pharmaceutical research and development (R&D) organizations, esp. during the compd. optimization stages of drug discovery and the early stages of clin. development.  Focusing on decreasing nonclin. safety-related attrition is not a new concept, and various approaches have been experimented with over the last two decades.  Front-loading testing funnels in Discovery with in vitro toxicity assays designed to rapidly identify unfavorable mols. was the approach adopted by most pharmaceutical R&D organizations a few years ago.  However, this approach has also a non-negligible opportunity cost.  Hence, significant refinements to the "fail early, fail often" paradigm have been proposed recently to reflect the complexity of accurately categorizing compds. with early data points without taking into account other important contextual aspects, in particular efficacious systemic and tissue exposures.  This review provides an overview of toxicol. approaches and models that can be used in pharmaceutical Discovery at the series/lead identification and lead optimization stages to guide and inform chem. efforts, as well as a personal view on how to best use them to meet nonclin. safety-related attrition objectives consistent with a sustainable pharmaceutical R&D model.  The scope of this review is limited to small mols., as large mols. are assocd. with challenges that are quite different.  Finally, a perspective on how several emerging technologies may impact toxicity evaluation is also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnswsW0Ve2_rVg90H21EOLACvtfcHk0lgsBEibLTsTqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOhu7vI&md5=ec2ad8b3576b7ba00f750e3f93469a2f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.5b00407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.5b00407%26sid%3Dliteratum%253Aachs%26aulast%3DBlomme%26aufirst%3DE.%2BA.%26aulast%3DWill%26aufirst%3DY.%26atitle%3DToxicology%2520strategies%2520for%2520drug%2520discovery%253A%2520present%2520and%2520future%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2016%26volume%3D29%26spage%3D473%26epage%3D504%26doi%3D10.1021%2Facs.chemrestox.5b00407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marroquin, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamieson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, Y.</span></span> <span> </span><span class="NLM_article-title">Circumventing the Crabtree effect: replacing media glucose with galactose increases susceptibility of HepG2 cells to mitochondrial toxicants</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfm052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Ftoxsci%2Fkfm052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=17361016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsV2is7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2007&pages=539-547&author=L.+D.+Marroquinauthor=J.+Hynesauthor=J.+A.+Dykensauthor=J.+D.+Jamiesonauthor=Y.+Will&title=Circumventing+the+Crabtree+effect%3A+replacing+media+glucose+with+galactose+increases+susceptibility+of+HepG2+cells+to+mitochondrial+toxicants&doi=10.1093%2Ftoxsci%2Fkfm052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Circumventing the Crabtree effect: replacing media glucose with galactose increases susceptibility of HepG2 cells to mitochondrial toxicants</span></div><div class="casAuthors">Marroquin, Lisa D.; Hynes, James; Dykens, James A.; Jamieson, Joseph D.; Will, Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">539-547</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-6080</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Many highly proliferative cells generate almost all ATP via glycolysis despite abundant O2 and a normal complement of fully functional mitochondria, a circumstance known as the Crabtree effect.  Such anaerobically poised cells are resistant to xenobiotics that impair mitochondrial function, such as the inhibitors rotenone, antimycin, oligomycin, and compds. like carbonyl cyanide 4-(trifluoromethoxy)phenylhydrazone (FCCP), that uncouple the respiratory electron transfer system from phosphorylation.  These cells are also resistant to the toxicity of many drugs whose deleterious side effect profiles are either caused, or exacerbated, by impairment of mitochondrial function.  Drug-induced mitochondrial toxicity is shown by members of important drug classes, including the thiazolidinediones, statins, fibrates, antivirals, antibiotics, and anticancer agents.  To increase detection of drug-induced mitochondrial effects in a preclin. cell-based assay, HepG2 cells were forced to rely on mitochondrial oxidative phosphorylation rather than glycolysis by substituting galactose for glucose in the growth media.  Oxygen consumption doubles in galactose-grown HepG2 cells and their susceptibility to canonical mitochondrial toxicants correspondingly increases.  Similarly, toxicity of several drugs with known mitochondrial liabilities is more readily apparent in aerobically poised HepG2 cells compared to glucose-grown cells.  Some drugs were equally toxic to both glucose- and galactose-grown cells, suggesting that mitochondrial impairment is likely secondary to other cytotoxic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Zb0elenmNbVg90H21EOLACvtfcHk0lgsBEibLTsTqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsV2is7w%253D&md5=53d6a955ad0a942d1ced5043668bed19</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfm052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfm052%26sid%3Dliteratum%253Aachs%26aulast%3DMarroquin%26aufirst%3DL.%2BD.%26aulast%3DHynes%26aufirst%3DJ.%26aulast%3DDykens%26aufirst%3DJ.%2BA.%26aulast%3DJamieson%26aufirst%3DJ.%2BD.%26aulast%3DWill%26aufirst%3DY.%26atitle%3DCircumventing%2520the%2520Crabtree%2520effect%253A%2520replacing%2520media%2520glucose%2520with%2520galactose%2520increases%2520susceptibility%2520of%2520HepG2%2520cells%2520to%2520mitochondrial%2520toxicants%26jtitle%3DToxicol.%2520Sci.%26date%3D2007%26volume%3D97%26spage%3D539%26epage%3D547%26doi%3D10.1093%2Ftoxsci%2Fkfm052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span> <span> </span><span class="NLM_article-title">The Connectivity Map: a new tool for biomedical research</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1038/nrc2044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrc2044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=17186018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCht7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=54-60&author=J.+Lamb&title=The+Connectivity+Map%3A+a+new+tool+for+biomedical+research&doi=10.1038%2Fnrc2044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Connectivity Map: a new tool for biomedical research</span></div><div class="casAuthors">Lamb, Justin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">54-60</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The ultimate objective of biomedical research is to connect human diseases with the genes that underlie them and drugs that treat them.  But this remains a daunting task, and even the most inspired researchers still have to resort to laborious screens of genetic or chem. libraries.  What if at least some parts of this screening process could be systematized and centralized.  And hits found and hypotheses generated with something resembling an internet search engine.  These are the questions the Connectivity Map project set out to answer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3OVVSFxDDUbVg90H21EOLACvtfcHk0ljulM1ZA9ElBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCht7nF&md5=8f23f79ae51eb97eb067f5357baef280</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnrc2044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2044%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DJ.%26atitle%3DThe%2520Connectivity%2520Map%253A%2520a%2520new%2520tool%2520for%2520biomedical%2520research%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D54%26epage%3D60%26doi%3D10.1038%2Fnrc2044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blat, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hieronymus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">1929</span>– <span class="NLM_lpage">1935</span>, <span class="refDoi"> DOI: 10.1126/science.1132939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1132939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=17008526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&author=J.+Lambauthor=E.+D.+Crawfordauthor=D.+Peckauthor=J.+W.+Modellauthor=I.+C.+Blatauthor=M.+J.+Wrobelauthor=J.+Lernerauthor=J.+P.+Brunetauthor=A.+Subramanianauthor=K.+N.+Rossauthor=M.+Reichauthor=H.+Hieronymusauthor=G.+Weiauthor=S.+A.+Armstrongauthor=S.+J.+Haggartyauthor=P.+A.+Clemonsauthor=R.+Weiauthor=S.+A.+Carrauthor=E.+S.+Landerauthor=T.+R.+Golub&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease&doi=10.1126%2Fscience.1132939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The connectivity map: Using gene-expression signatures to connect small molecules, genes, and disease</span></div><div class="casAuthors">Lamb, Justin; Crawford, Emily D.; Peck, David; Modell, Joshua W.; Blat, Irene C.; Wrobel, Matthew J.; Lerner, Jim; Brunet, Jean-Philippe; Subramanian, Aravind; Ross, Kenneth N.; Reich, Michael; Hieronymus, Haley; Wei, Guo; Armstrong, Scott A.; Haggarty, Stephen J.; Clemons, Paul A.; Wei, Ru; Carr, Steven A.; Lander, Eric S.; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">5795</span>),
    <span class="NLM_cas:pages">1929-1935</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">To pursue a systematic approach to the discovery of functional connections among diseases, genetic perturbation, and drug action, we have created the first installment of a ref. collection of gene-expression profiles from cultured human cells treated with bioactive small mols., together with pattern-matching software to mine these data.  We demonstrate that this "Connectivity Map" resource can be used to find connections among small mols. sharing a mechanism of action, chems. and physiol. processes, and diseases and drugs.  These results indicate the feasibility of the approach and suggest the value of a large-scale community Connectivity Map project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtu0M2BUO2w7Vg90H21EOLACvtfcHk0ljulM1ZA9ElBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN&md5=1fad887ad515ffdd8064e526ff25a4db</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1126%2Fscience.1132939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1132939%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DE.%2BD.%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DModell%26aufirst%3DJ.%2BW.%26aulast%3DBlat%26aufirst%3DI.%2BC.%26aulast%3DWrobel%26aufirst%3DM.%2BJ.%26aulast%3DLerner%26aufirst%3DJ.%26aulast%3DBrunet%26aufirst%3DJ.%2BP.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DHieronymus%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DThe%2520Connectivity%2520Map%253A%2520using%2520gene-expression%2520signatures%2520to%2520connect%2520small%2520molecules%252C%2520genes%252C%2520and%2520disease%26jtitle%3DScience%26date%3D2006%26volume%3D313%26spage%3D1929%26epage%3D1935%26doi%3D10.1126%2Fscience.1132939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span> <span> </span><span class="NLM_article-title">A method for high-throughput gene expression signature analysis</span>. <i>Genome Biol</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">R61</span>, <span class="refDoi"> DOI: 10.1186/gb-2006-7-7-r61</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1186%2Fgb-2006-7-7-r61" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=R61&author=D.+Peckauthor=E.+D.+Crawfordauthor=K.+N.+Rossauthor=K.+Stegmaierauthor=T.+R.+Golubauthor=J.+Lamb&title=A+method+for+high-throughput+gene+expression+signature+analysis&doi=10.1186%2Fgb-2006-7-7-r61"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1186%2Fgb-2006-7-7-r61&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2006-7-7-r61%26sid%3Dliteratum%253Aachs%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DCrawford%26aufirst%3DE.%2BD.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLamb%26aufirst%3DJ.%26atitle%3DA%2520method%2520for%2520high-throughput%2520gene%2520expression%2520signature%2520analysis%26jtitle%3DGenome%2520Biol%26date%3D2006%26volume%3D7%26spage%3DR61%26doi%3D10.1186%2Fgb-2006-7-7-r61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niepel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhlich, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouillard, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma’ayan, A.</span></span> <span> </span><span class="NLM_article-title">LINCS Canvas Browser: interactive web app to query, browse and interrogate LINCS L1000 gene expression signatures</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">W449</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1093/nar/gku476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgku476" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W449-460&author=Q.+Duanauthor=C.+Flynnauthor=M.+Niepelauthor=M.+Hafnerauthor=J.+L.+Muhlichauthor=N.+F.+Fernandezauthor=A.+D.+Rouillardauthor=C.+M.+Tanauthor=E.+Y.+Chenauthor=T.+R.+Golubauthor=P.+K.+Sorgerauthor=A.+Subramanianauthor=A.+Ma%E2%80%99ayan&title=LINCS+Canvas+Browser%3A+interactive+web+app+to+query%2C+browse+and+interrogate+LINCS+L1000+gene+expression+signatures&doi=10.1093%2Fnar%2Fgku476"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku476%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DQ.%26aulast%3DFlynn%26aufirst%3DC.%26aulast%3DNiepel%26aufirst%3DM.%26aulast%3DHafner%26aufirst%3DM.%26aulast%3DMuhlich%26aufirst%3DJ.%2BL.%26aulast%3DFernandez%26aufirst%3DN.%2BF.%26aulast%3DRouillard%26aufirst%3DA.%2BD.%26aulast%3DTan%26aufirst%3DC.%2BM.%26aulast%3DChen%26aufirst%3DE.%2BY.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DMa%25E2%2580%2599ayan%26aufirst%3DA.%26atitle%3DLINCS%2520Canvas%2520Browser%253A%2520interactive%2520web%2520app%2520to%2520query%252C%2520browse%2520and%2520interrogate%2520LINCS%2520L1000%2520gene%2520expression%2520signatures%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DW449%26epage%3D460%26doi%3D10.1093%2Fnar%2Fgku476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsello, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natoli, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubelli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asiedu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahr, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadden, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orzechowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enache, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccioni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrcic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosains, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardlie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenside, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read-Button, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronco, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doench, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">A next generation Connectivity Map: L1000 platform and the first 1,000,000 profiles</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1437</span>– <span class="NLM_lpage">1452</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.10.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.cell.2017.10.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=29195078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452&author=A.+Subramanianauthor=R.+Narayanauthor=S.+M.+Corselloauthor=D.+D.+Peckauthor=T.+E.+Natoliauthor=X.+Luauthor=J.+Gouldauthor=J.+F.+Davisauthor=A.+A.+Tubelliauthor=J.+K.+Asieduauthor=D.+L.+Lahrauthor=J.+E.+Hirschmanauthor=Z.+Liuauthor=M.+Donahueauthor=B.+Julianauthor=M.+Khanauthor=D.+Waddenauthor=I.+C.+Smithauthor=D.+Lamauthor=A.+Liberzonauthor=C.+Toderauthor=M.+Bagulauthor=M.+Orzechowskiauthor=O.+M.+Enacheauthor=F.+Piccioniauthor=S.+A.+Johnsonauthor=N.+J.+Lyonsauthor=A.+H.+Bergerauthor=A.+F.+Shamjiauthor=A.+N.+Brooksauthor=A.+Vrcicauthor=C.+Flynnauthor=J.+Rosainsauthor=D.+Y.+Takedaauthor=R.+Huauthor=D.+Davisonauthor=J.+Lambauthor=K.+Ardlieauthor=L.+Hogstromauthor=P.+Greensideauthor=N.+S.+Grayauthor=P.+A.+Clemonsauthor=S.+Silverauthor=X.+Wuauthor=W.+N.+Zhaoauthor=W.+Read-Buttonauthor=X.+Wuauthor=S.+J.+Haggartyauthor=L.+V.+Roncoauthor=J.+S.+Boehmauthor=S.+L.+Schreiberauthor=J.+G.+Doenchauthor=J.+A.+Bittkerauthor=D.+E.+Rootauthor=B.+Wongauthor=T.+R.+Golub&title=A+next+generation+Connectivity+Map%3A+L1000+platform+and+the+first+1%2C000%2C000+profiles&doi=10.1016%2Fj.cell.2017.10.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles</span></div><div class="casAuthors">Subramanian, Aravind; Narayan, Rajiv; Corsello, Steven M.; Peck, David D.; Natoli, Ted E.; Lu, Xiaodong; Gould, Joshua; Davis, John F.; Tubelli, Andrew A.; Asiedu, Jacob K.; Lahr, David L.; Hirschman, Jodi E.; Liu, Zihan; Donahue, Melanie; Julian, Bina; Khan, Mariya; Wadden, David; Smith, Ian C.; Lam, Daniel; Liberzon, Arthur; Toder, Courtney; Bagul, Mukta; Orzechowski, Marek; Enache, Oana M.; Piccioni, Federica; Johnson, Sarah A.; Lyons, Nicholas J.; Berger, Alice H.; Shamji, Alykhan F.; Brooks, Angela N.; Vrcic, Anita; Flynn, Corey; Rosains, Jacqueline; Takeda, David Y.; Hu, Roger; Davison, Desiree; Lamb, Justin; Ardlie, Kristin; Hogstrom, Larson; Greenside, Peyton; Gray, Nathanael S.; Clemons, Paul A.; Silver, Serena; Wu, Xiaoyun; Zhao, Wen-Ning; Read-Button, Willis; Wu, Xiaohua; Haggarty, Stephen J.; Ronco, Lucienne V.; Boehm, Jesse S.; Schreiber, Stuart L.; Doench, John G.; Bittker, Joshua A.; Root, David E.; Wong, Bang; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1437-1452.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We previously piloted the concept of a Connectivity Map (CMap), whereby genes, drugs, and disease states are connected by virtue of common gene-expression signatures.  Here, we report more than a 1000-fold scale-up of the CMap as part of the NIH LINCS Consortium, made possible by a new, low-cost, high-throughput reduced representation expression profiling method that we term L1000.  We show that L1000 is highly reproducible, comparable to RNA sequencing, and suitable for computational inference of the expression levels of 81% of non-measured transcripts.  We further show that the expanded CMap can be used to discover mechanism of action of small mols., functionally annotate genetic variants of disease genes, and inform clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIWrWy5IxY-rVg90H21EOLACvtfcHk0ligKhB0TtAB_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE&md5=d2b13f2b80fe94021ac03f6a8c859a4a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.10.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.10.049%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DNarayan%26aufirst%3DR.%26aulast%3DCorsello%26aufirst%3DS.%2BM.%26aulast%3DPeck%26aufirst%3DD.%2BD.%26aulast%3DNatoli%26aufirst%3DT.%2BE.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DGould%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DJ.%2BF.%26aulast%3DTubelli%26aufirst%3DA.%2BA.%26aulast%3DAsiedu%26aufirst%3DJ.%2BK.%26aulast%3DLahr%26aufirst%3DD.%2BL.%26aulast%3DHirschman%26aufirst%3DJ.%2BE.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDonahue%26aufirst%3DM.%26aulast%3DJulian%26aufirst%3DB.%26aulast%3DKhan%26aufirst%3DM.%26aulast%3DWadden%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DI.%2BC.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DLiberzon%26aufirst%3DA.%26aulast%3DToder%26aufirst%3DC.%26aulast%3DBagul%26aufirst%3DM.%26aulast%3DOrzechowski%26aufirst%3DM.%26aulast%3DEnache%26aufirst%3DO.%2BM.%26aulast%3DPiccioni%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DS.%2BA.%26aulast%3DLyons%26aufirst%3DN.%2BJ.%26aulast%3DBerger%26aufirst%3DA.%2BH.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DVrcic%26aufirst%3DA.%26aulast%3DFlynn%26aufirst%3DC.%26aulast%3DRosains%26aufirst%3DJ.%26aulast%3DTakeda%26aufirst%3DD.%2BY.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DDavison%26aufirst%3DD.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DHogstrom%26aufirst%3DL.%26aulast%3DGreenside%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DSilver%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%2BN.%26aulast%3DRead-Button%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRonco%26aufirst%3DL.%2BV.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DDoench%26aufirst%3DJ.%2BG.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DA%2520next%2520generation%2520Connectivity%2520Map%253A%2520L1000%2520platform%2520and%2520the%2520first%25201%252C000%252C000%2520profiles%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D1437%26epage%3D1452%26doi%3D10.1016%2Fj.cell.2017.10.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span> <span> </span><span class="NLM_article-title">Systematic quality control analysis of LINCS data</span>. <i>CPT: Pharmacometrics Syst. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1002/psp4.12107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fpsp4.12107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27796074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGgur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=588-598&author=L.+Chengauthor=L.+Li&title=Systematic+quality+control+analysis+of+LINCS+data&doi=10.1002%2Fpsp4.12107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Quality Control Analysis of LINCS Data</span></div><div class="casAuthors">Cheng, L.; Li, L.</div><div class="citationInfo"><span class="NLM_cas:title">CPT: Pharmacometrics & Systems Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">588-598</span>CODEN:
                <span class="NLM_cas:coden">CPSPBR</span>;
        ISSN:<span class="NLM_cas:issn">2163-8306</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Library of Integrated Cellular Signatures (LINCS) project provides comprehensive transcriptome profiling of human cell lines before and after chem. and genetic perturbations.  Its L1000 platform utilizes 978 landmark genes to infer the transcript levels of 14,292 genes computationally.  Here we conducted the L1000 data quality control anal. by using MCF7, PC3, and A375 cell lines as representative examples.  Before perturbations, a promising 80% correlation in transcriptome was obsd. between L1000- and Affymetrix HU133A-platforms.  After library-based shRNA perturbations, a moderate 30% of differentially expressed genes overlapped between any two selected controls viral vectors using the L1000 platform.  The mitogen-activated protein kinase, vascular endothelial growth factor, and T-cell receptor pathways were identified as the most significantly shared pathways between chem. and genetic perturbations in cancer cells.  In conclusion, L1000 platform is reliable in assessing transcriptome before perturbation.  Its response to perturbagens needs to be interpreted with caution.  A quality control anal. pipeline of L1000 is recommended before addressing biol. questions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLk1tBMwFMlLVg90H21EOLACvtfcHk0lgukcUUrFOFTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGgur7O&md5=bfb45386df427049788bec6c33e6b5ac</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fpsp4.12107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsp4.12107%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DSystematic%2520quality%2520control%2520analysis%2520of%2520LINCS%2520data%26jtitle%3DCPT%253A%2520Pharmacometrics%2520Syst.%2520Pharmacol.%26date%3D2016%26volume%3D5%26spage%3D588%26epage%3D598%26doi%3D10.1002%2Fpsp4.12107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Wolf, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bondt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlmann, H. W.</span></span> <span> </span><span class="NLM_article-title">Transcriptional characterization of compounds: lessons learned from the public LINCS data</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1089/adt.2016.715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1089%2Fadt.2016.715" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=252-260&author=H.+De+Wolfauthor=A.+De+Bondtauthor=H.+Turnerauthor=H.+W.+Gohlmann&title=Transcriptional+characterization+of+compounds%3A+lessons+learned+from+the+public+LINCS+data&doi=10.1089%2Fadt.2016.715"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1089%2Fadt.2016.715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2016.715%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BWolf%26aufirst%3DH.%26aulast%3DDe%2BBondt%26aufirst%3DA.%26aulast%3DTurner%26aufirst%3DH.%26aulast%3DGohlmann%26aufirst%3DH.%2BW.%26atitle%3DTranscriptional%2520characterization%2520of%2520compounds%253A%2520lessons%2520learned%2520from%2520the%2520public%2520LINCS%2520data%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2016%26volume%3D14%26spage%3D252%26epage%3D260%26doi%3D10.1089%2Fadt.2016.715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span> <span> </span><span class="NLM_article-title">Compound signature detection on LINCS L1000 big data</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">714</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1039/C4MB00677A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2FC4MB00677A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25609570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtFGisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=714-722&author=C.+Liuauthor=J.+Suauthor=F.+Yangauthor=K.+Weiauthor=J.+Maauthor=X.+Zhou&title=Compound+signature+detection+on+LINCS+L1000+big+data&doi=10.1039%2FC4MB00677A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Compound signature detection on LINCS L1000 big data</span></div><div class="casAuthors">Liu, Chenglin; Su, Jing; Yang, Fei; Wei, Kun; Ma, Jinwen; Zhou, Xiaobo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">714-722</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The Library of Integrated Network-based Cellular Signatures (LINCS) L1000 big data provide gene expression profiles induced by over 10 000 compds., shRNAs, and kinase inhibitors using the L1000 platform.  We developed csNMF, a systematic compd. signature discovery pipeline covering from raw L1000 data processing to drug screening and mechanism generation.  The csNMF pipeline demonstrated better performance than the original L1000 pipeline.  The discovered compd. signatures of breast cancer were consistent with the LINCS KINOMEscan data and were clin. relevant.  The csNMF pipeline provided a novel and complete tool to expedite signature-based drug discovery leveraging the LINCS L1000 resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK2ETNNHh4ULVg90H21EOLACvtfcHk0lgukcUUrFOFTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtFGisg%253D%253D&md5=cb02442649e0bf33ca0c7cfd6d0603df</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1039%2FC4MB00677A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MB00677A%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DX.%26atitle%3DCompound%2520signature%2520detection%2520on%2520LINCS%2520L1000%2520big%2520data%26jtitle%3DMol.%2520BioSyst.%26date%3D2015%26volume%3D11%26spage%3D714%26epage%3D722%26doi%3D10.1039%2FC4MB00677A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filzen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudor, M.</span></span> <span> </span><span class="NLM_article-title">Representing high throughput expression profiles via perturbation barcodes reveals compound targets</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e1005335</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1371%2Fjournal.pcbi.1005335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28182661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCqsbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=e1005335&author=T.+M.+Filzenauthor=P.+S.+Kutchukianauthor=J.+D.+Hermesauthor=J.+Liauthor=M.+Tudor&title=Representing+high+throughput+expression+profiles+via+perturbation+barcodes+reveals+compound+targets&doi=10.1371%2Fjournal.pcbi.1005335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Representing high throughput expression profiles via perturbation barcodes reveals compound targets</span></div><div class="casAuthors">Filzen, Tracey M.; Kutchukian, Peter S.; Hermes, Jeffrey D.; Li, Jing; Tudor, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1005335/1-e1005335/19</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">High throughput mRNA expression profiling can be used to characterize the response of cell culture models to perturbations such as pharmacol. modulators and genetic perturbations.  As profiling campaigns expand in scope, it is important to homogenize, summarize, and analyze the resulting data in a manner that captures significant biol. signals in spite of various noise sources such as batch effects and stochastic variation.  We used the L1000 platform for large-scale profiling of 978 representative genes across thousands of compd. treatments.  Here, a method is described that uses deep learning techniques to convert the expression changes of the landmark genes into a perturbation barcode that reveals important features of the underlying data, performing better than the raw data in revealing important biol. insights.  The barcode captures compd. structure and target information, and predicts a compd.'s high throughput screening promiscuity, to a higher degree than the original data measurements, indicating that the approach uncovers underlying factors of the expression data that are otherwise entangled or masked by noise.  Furthermore, we demonstrate that visualizations derived from the perturbation barcode can be used to more sensitively assign functions to unknown compds. through a guilt-by-assocn. approach, which we use to predict and exptl. validate the activity of compds. on the MAPK pathway.  The demonstrated application of deep metric learning to large-scale chem. genetics projects highlights the utility of this and related approaches to the extn. of insights and testable hypotheses from big, sometimes noisy data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgD8qURxAqhbVg90H21EOLACvtfcHk0lhg_KqJm05ZJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCqsbvP&md5=5f2f46c468401ba7bced65ee0bbc3244</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005335%26sid%3Dliteratum%253Aachs%26aulast%3DFilzen%26aufirst%3DT.%2BM.%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DHermes%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DTudor%26aufirst%3DM.%26atitle%3DRepresenting%2520high%2520throughput%2520expression%2520profiles%2520via%2520perturbation%2520barcodes%2520reveals%2520compound%2520targets%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3De1005335%26doi%3D10.1371%2Fjournal.pcbi.1005335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Senkowski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarvius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengqvist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nygren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kultima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fryknas, M.</span></span> <span> </span><span class="NLM_article-title">Large-scale gene expression profiling platform for identification of context-dependent drug responses in multicellular tumor spheroids</span>. <i>Cell Chem Biol</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1428</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2016.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27984028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOqsL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1428-1438&author=W.+Senkowskiauthor=M.+Jarviusauthor=J.+Rubinauthor=J.+Lengqvistauthor=M.+G.+Gustafssonauthor=P.+Nygrenauthor=K.+Kultimaauthor=R.+Larssonauthor=M.+Fryknas&title=Large-scale+gene+expression+profiling+platform+for+identification+of+context-dependent+drug+responses+in+multicellular+tumor+spheroids&doi=10.1016%2Fj.chembiol.2016.09.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Large-Scale Gene Expression Profiling Platform for Identification of Context-Dependent Drug Responses in Multicellular Tumor Spheroids</span></div><div class="casAuthors">Senkowski, Wojciech; Jarvius, Malin; Rubin, Jenny; Lengqvist, Johan; Gustafsson, Mats G.; Nygren, Peter; Kultima, Kim; Larsson, Rolf; Fryknaes, Maarten</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1428-1438</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Cancer cell lines grown as two-dimensional (2D) cultures have been an essential model for studying cancer biol. and anticancer drug discovery.  However, 2D cancer cell cultures have major limitations, as they do not closely mimic the heterogeneity and tissue context of in vivo tumors.  Developing three-dimensional (3D) cell cultures, such as multicellular tumor spheroids, has the potential to address some of these limitations.  Here, we combined a high-throughput gene expression profiling method with a tumor spheroid-based drug-screening assay to identify context-dependent treatment responses.  As a proof of concept, we examd. drug responses of quiescent cancer cells to oxidative phosphorylation (OXPHOS) inhibitors.  Use of multicellular tumor spheroids led to discovery that the mevalonate pathway is upregulated in quiescent cells during OXPHOS inhibition, and that OXPHOS inhibitors and mevalonate pathway inhibitors were synergistically toxic to quiescent spheroids.  This work illustrates how 3D cellular models yield functional and mechanistic insights not accessible via 2D cultures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj9b2oo56yvrVg90H21EOLACvtfcHk0lhg_KqJm05ZJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOqsL7N&md5=95f3efebeac31078d368dfef55b9faf7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DSenkowski%26aufirst%3DW.%26aulast%3DJarvius%26aufirst%3DM.%26aulast%3DRubin%26aufirst%3DJ.%26aulast%3DLengqvist%26aufirst%3DJ.%26aulast%3DGustafsson%26aufirst%3DM.%2BG.%26aulast%3DNygren%26aufirst%3DP.%26aulast%3DKultima%26aufirst%3DK.%26aulast%3DLarsson%26aufirst%3DR.%26aulast%3DFryknas%26aufirst%3DM.%26atitle%3DLarge-scale%2520gene%2520expression%2520profiling%2520platform%2520for%2520identification%2520of%2520context-dependent%2520drug%2520responses%2520in%2520multicellular%2520tumor%2520spheroids%26jtitle%3DCell%2520Chem%2520Biol%26date%3D2016%26volume%3D23%26spage%3D1428%26epage%3D1438%26doi%3D10.1016%2Fj.chembiol.2016.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Silva, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruccoleri, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayne, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penhallow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockett, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemers, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross-Macdonald, P.</span></span> <span> </span><span class="NLM_article-title">Transcriptional profiling of the dose response: a more powerful approach for characterizing drug activities</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e1000512</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1000512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1371%2Fjournal.pcbi.1000512" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000512&author=R.+R.+Jiauthor=H.+de+Silvaauthor=Y.+Jinauthor=R.+E.+Bruccoleriauthor=J.+Caoauthor=A.+Heauthor=W.+Huangauthor=P.+S.+Kayneauthor=I.+M.+Neuhausauthor=K.+H.+Ottauthor=B.+Penhallowauthor=M.+I.+Cockettauthor=M.+G.+Neubauerauthor=N.+O.+Siemersauthor=P.+Ross-Macdonald&title=Transcriptional+profiling+of+the+dose+response%3A+a+more+powerful+approach+for+characterizing+drug+activities&doi=10.1371%2Fjournal.pcbi.1000512"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000512%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DR.%2BR.%26aulast%3Dde%2BSilva%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DBruccoleri%26aufirst%3DR.%2BE.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DKayne%26aufirst%3DP.%2BS.%26aulast%3DNeuhaus%26aufirst%3DI.%2BM.%26aulast%3DOtt%26aufirst%3DK.%2BH.%26aulast%3DPenhallow%26aufirst%3DB.%26aulast%3DCockett%26aufirst%3DM.%2BI.%26aulast%3DNeubauer%26aufirst%3DM.%2BG.%26aulast%3DSiemers%26aufirst%3DN.%2BO.%26aulast%3DRoss-Macdonald%26aufirst%3DP.%26atitle%3DTranscriptional%2520profiling%2520of%2520the%2520dose%2520response%253A%2520a%2520more%2520powerful%2520approach%2520for%2520characterizing%2520drug%2520activities%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000512%26doi%3D10.1371%2Fjournal.pcbi.1000512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drawnel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benmansour, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araujo Del Rosario, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen Zoffmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delobel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prummer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weibel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iacone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Certa, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunotto, M.</span></span> <span> </span><span class="NLM_article-title">Molecular phenotyping combines molecular information, biological relevance, and patient data to Improve productivity of early drug discovery</span>. <i>Cell Chem Biol</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">624</span>– <span class="NLM_lpage">634</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2017.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28434878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsVWitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=624-634&author=F.+M.+Drawnelauthor=J.+D.+Zhangauthor=E.+Kungauthor=N.+Aoyamaauthor=F.+Benmansourauthor=A.+Araujo+Del+Rosarioauthor=S.+Jensen+Zoffmannauthor=F.+Delobelauthor=M.+Prummerauthor=F.+Weibelauthor=C.+Carlsonauthor=B.+Ansonauthor=R.+Iaconeauthor=U.+Certaauthor=T.+Singerauthor=M.+Ebelingauthor=M.+Prunotto&title=Molecular+phenotyping+combines+molecular+information%2C+biological+relevance%2C+and+patient+data+to+Improve+productivity+of+early+drug+discovery&doi=10.1016%2Fj.chembiol.2017.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Phenotyping Combines Molecular Information, Biological Relevance, and Patient Data to Improve Productivity of Early Drug Discovery</span></div><div class="casAuthors">Drawnel, Faye Marie; Zhang, Jitao David; Kung, Erich; Aoyama, Natsuyo; Benmansour, Fethallah; Araujo Del Rosario, Andrea; Jensen Zoffmann, Sannah; Delobel, Frederic; Prummer, Michael; Weibel, Franziska; Carlson, Coby; Anson, Blake; Iacone, Roberto; Certa, Ulrich; Singer, Thomas; Ebeling, Martin; Prunotto, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">624-634.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Today, novel therapeutics are identified in an environment which is intrinsically different from the clin. context in which they are ultimately evaluated.  Using mol. phenotyping and an in vitro model of diabetic cardiomyopathy, we show that by quantifying pathway reporter gene expression, mol. phenotyping can cluster compds. based on pathway profiles and dissect assocns. between pathway activities and disease phenotypes simultaneously.  Mol. phenotyping was applicable to compds. with a range of binding specificities and triaged false positives derived from high-content screening assays.  The technique identified a class of calcium-signaling modulators that can reverse disease-regulated pathways and phenotypes, which was validated by structurally distinct compds. of relevant classes.  Our results advocate for application of mol. phenotyping in early drug discovery, promoting biol. relevance as a key selection criterion early in the drug development cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph9aTnyxvKj7Vg90H21EOLACvtfcHk0lhtrlC_-vkWFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsVWitbw%253D&md5=757558351b5a00dcc0f3f107c8f61d89</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DDrawnel%26aufirst%3DF.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%2BD.%26aulast%3DKung%26aufirst%3DE.%26aulast%3DAoyama%26aufirst%3DN.%26aulast%3DBenmansour%26aufirst%3DF.%26aulast%3DAraujo%2BDel%2BRosario%26aufirst%3DA.%26aulast%3DJensen%2BZoffmann%26aufirst%3DS.%26aulast%3DDelobel%26aufirst%3DF.%26aulast%3DPrummer%26aufirst%3DM.%26aulast%3DWeibel%26aufirst%3DF.%26aulast%3DCarlson%26aufirst%3DC.%26aulast%3DAnson%26aufirst%3DB.%26aulast%3DIacone%26aufirst%3DR.%26aulast%3DCerta%26aufirst%3DU.%26aulast%3DSinger%26aufirst%3DT.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DPrunotto%26aufirst%3DM.%26atitle%3DMolecular%2520phenotyping%2520combines%2520molecular%2520information%252C%2520biological%2520relevance%252C%2520and%2520patient%2520data%2520to%2520Improve%2520productivity%2520of%2520early%2520drug%2520discovery%26jtitle%3DCell%2520Chem%2520Biol%26date%3D2017%26volume%3D24%26spage%3D624%26epage%3D634%26doi%3D10.1016%2Fj.chembiol.2017.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunkel, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plavec, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melrose, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosler, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hytopoulos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bateman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butcher, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E. L.</span></span> <span> </span><span class="NLM_article-title">Rapid structure-activity and selectivity analysis of kinase inhibitors by BioMAP analysis in complex human primary cell-based models</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1089/adt.2004.2.431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1089%2Fadt.2004.2.431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=15357924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntlGksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=431-441&author=E.+J.+Kunkelauthor=I.+Plavecauthor=D.+Nguyenauthor=J.+Melroseauthor=E.+S.+Roslerauthor=L.+T.+Kaoauthor=Y.+Wangauthor=E.+Hytopoulosauthor=A.+C.+Bishopauthor=R.+Batemanauthor=K.+M.+Shokatauthor=E.+C.+Butcherauthor=E.+L.+Berg&title=Rapid+structure-activity+and+selectivity+analysis+of+kinase+inhibitors+by+BioMAP+analysis+in+complex+human+primary+cell-based+models&doi=10.1089%2Fadt.2004.2.431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Structure-Activity and Selectivity Analysis of Kinase Inhibitors by BioMAP Analysis in Complex Human Primary Cell-Based Models</span></div><div class="casAuthors">Kunkel, Eric J.; Plavec, Ivan; Nguyen, Dat; Melrose, Jennifer; Rosler, Elen S.; Kao, Leon T.; Wang, Yuker; Hytopoulos, Evangelos; Bishop, Anthony C.; Bateman, Raynard; Shokat, Kevan M.; Butcher, Eugene C.; Berg, Ellen L.</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">431-441</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Rapid, quant. methods for characterizing the biol. activities of kinase inhibitors in complex human cell systems could allow the biol. consequences of differential target selectivity to be monitored early in development, improving the selection of drug candidates.  We have previously shown that Biol. Multiplexed Activity Profiling (BioMAP) permits rapid characterization of drug function based on statistical anal. of protein expression data sets from complex primary human cell based models of disease biol.  Here, using four such model systems contg. primary human endothelial cells and peripheral blood mononuclear cells in which multiple signaling pathways relevant to inflammation and immune responses are simultaneously activated, we demonstrate that BioMAP anal. can detect and distinguish a wide range of inhibitors directed against different kinase targets.  Using a panel of p38 mitogen-activated protein kinase antagonists as a test set, we show further that related compds. can be distinguished by unique features of the biol. responses they induce in complex systems, and can be classified according to their induction of shared (on-target) and secondary activities.  Statistical comparisons of quant. BioMAP profiles and anal. of profile features allow correlation of induced biol. effects with chem. structure and mapping of biol. responses to chem. series or substituents on a common scaffold.  Integration of automated BioMAP anal. for prioritization of hits and for structure-activity relation studies may improve and accelerate the design and selection of optimal therapeutic candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtlA37tRw7P7Vg90H21EOLACvtfcHk0lh0hQ4N-RmovQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntlGksLY%253D&md5=eae8485e89ea69af97d5e60738bee209</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1089%2Fadt.2004.2.431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2004.2.431%26sid%3Dliteratum%253Aachs%26aulast%3DKunkel%26aufirst%3DE.%2BJ.%26aulast%3DPlavec%26aufirst%3DI.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DMelrose%26aufirst%3DJ.%26aulast%3DRosler%26aufirst%3DE.%2BS.%26aulast%3DKao%26aufirst%3DL.%2BT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHytopoulos%26aufirst%3DE.%26aulast%3DBishop%26aufirst%3DA.%2BC.%26aulast%3DBateman%26aufirst%3DR.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DButcher%26aufirst%3DE.%2BC.%26aulast%3DBerg%26aufirst%3DE.%2BL.%26atitle%3DRapid%2520structure-activity%2520and%2520selectivity%2520analysis%2520of%2520kinase%2520inhibitors%2520by%2520BioMAP%2520analysis%2520in%2520complex%2520human%2520primary%2520cell-based%2520models%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2004%26volume%3D2%26spage%3D431%26epage%3D441%26doi%3D10.1089%2Fadt.2004.2.431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polokoff, M. A.</span></span> <span> </span><span class="NLM_article-title">Building predictive models for mechanism-of-action classification from phenotypic assay data sets</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1260</span>– <span class="NLM_lpage">1269</span>, <span class="refDoi"> DOI: 10.1177/1087057113505324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F1087057113505324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24088371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWjsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1260-1269&author=E.+L.+Bergauthor=J.+Yangauthor=M.+A.+Polokoff&title=Building+predictive+models+for+mechanism-of-action+classification+from+phenotypic+assay+data+sets&doi=10.1177%2F1087057113505324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Building predictive models for mechanism-of-action classification from phenotypic assay data sets</span></div><div class="casAuthors">Berg, Ellen L.; Yang, Jian; Polokoff, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1260-1269, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Compd. mechanism-of-action information can be crit. for drug development decisions but is often challenging for phenotypic drug discovery programs.  One concern is that compds. selected by phenotypic screening will have a previously known but undesirable target mechanism.  Here we describe a useful method for assigning mechanism class to compds. and bioactive agents using an 84-feature signature from a panel of primary human cell systems (BioMAP systems).  For this approach, a ref. data set of well-characterized compds. was used to develop predictive models for 28 mechanism classes using support vector machines.  These mechanism classes encompass safety and efficacy-related mechanisms, include both target-specific and pathway-based classes, and cover the most common mechanisms identified in phenotypic screens, such as inhibitors of mitochondrial and microtubule function, histone deacetylase, and cAMP elevators.  Here we describe the performance and the application of these predictive models in a decision scheme for triaging phenotypic screening hits using a previously published data set of 309 environmental chems. tested as part of the Environmental Protection Agency's ToxCast program.  By providing quantified membership in specific mechanism classes, this approach is suitable for identification of off-target toxicity mechanisms as well as enabling target deconvolution of phenotypic drug discovery hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSbGfCpuChA7Vg90H21EOLACvtfcHk0lghEZJJRTAnHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWjsLvP&md5=1177a3a1dbae60621bbc2249f0e8e0ac</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1177%2F1087057113505324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113505324%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DE.%2BL.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DPolokoff%26aufirst%3DM.%2BA.%26atitle%3DBuilding%2520predictive%2520models%2520for%2520mechanism-of-action%2520classification%2520from%2520phenotypic%2520assay%2520data%2520sets%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D1260%26epage%3D1269%26doi%3D10.1177%2F1087057113505324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E. L.</span></span> <span> </span><span class="NLM_article-title">Phenotypic chemical biology for predicting safety and efficacy</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2017.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.ddtec.2017.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28647086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC1cjhsFaitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=53-60&author=E.+L.+Berg&title=Phenotypic+chemical+biology+for+predicting+safety+and+efficacy&doi=10.1016%2Fj.ddtec.2017.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic chemical biology for predicting safety and efficacy</span></div><div class="casAuthors">Berg Ellen L</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Phenotypic assays using in vitro cell cultures to forecast compound effects in people are transforming pharmaceutical research and contribute to alternative methods for chemical safety testing.  How these assays are validated for human disease relevance is a critical factor for developing more predictive assays.  Chemical biology, using drugs as well as target-selective chemical probes, is a direct and efficient approach for establishing disease relevance.  Chemical probes can connect information across assays and associate targets to clinical effects.  When applied at scale, phenotypic chemical biology advances our understanding of drug and toxicity mechanisms enabling construction of disease outcome pathways.  To improve the physiological relevance of phenotypic assays, standardized testing of a curated set of phenotypic pathway probes can provide a higher level of validation for phenotypic assay best practices.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRF0mGW8Hm3ecvLc4ARHEmzfW6udTcc2eaBBfA2ZgcRJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjhsFaitg%253D%253D&md5=eb67d89d2f5ba017fe9609141cee7959</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2017.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2017.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DE.%2BL.%26atitle%3DPhenotypic%2520chemical%2520biology%2520for%2520predicting%2520safety%2520and%2520efficacy%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2017%26volume%3D23%26spage%3D53%26epage%3D60%26doi%3D10.1016%2Fj.ddtec.2017.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E. L.</span></span>, personal communication.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Berg%2C+E.+L.%2C+personal+communication."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcommun%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DE.%2BL." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jinek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">East, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudna, J.</span></span> <span> </span><span class="NLM_article-title">RNA-programmed genome editing in human cells</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">e00471</span>, <span class="refDoi"> DOI: 10.7554/eLife.00471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.7554%2FeLife.00471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23386978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00471&author=M.+Jinekauthor=A.+Eastauthor=A.+Chengauthor=S.+Linauthor=E.+Maauthor=J.+Doudna&title=RNA-programmed+genome+editing+in+human+cells&doi=10.7554%2FeLife.00471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">RNA-programmed genome editing in human cells</span></div><div class="casAuthors">Jinek, Martin; East, Alexandra; Cheng, Aaron; Lin, Steven; Ma, Enbo; Doudna, Jennifer</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00471/1-e00471/9</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Type II CRISPR immune systems in bacteria use a dual RNA-guided DNA endonuclease, Cas9, to cleave foreign DNA at specific sites.  We show here that Cas9 assembles with hybrid guide RNAs in human cells and can induce the formation of double-strand DNA breaks (DSBs) at a site complementary to the guide RNA sequence in genomic DNA.  This cleavage activity requires both Cas9 and the complementary binding of the guide RNA.  Expts. using exts. from transfected cells show that RNA expression and/or assembly into Cas9 is the limiting factor for Cas9-mediated DNA cleavage.  In addn., we find that extension of the RNA sequence at the 3' end enhances DNA targeting activity in vivo.  These results show that RNA-programmed genome editing is a facile strategy for introducing site-specific genetic changes in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvfsdoVCMm_bVg90H21EOLACvtfcHk0lhcYaB5NHj0vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWjsw%253D%253D&md5=5d3a051b052b4301fd5bf935d12e5f6c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.7554%2FeLife.00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00471%26sid%3Dliteratum%253Aachs%26aulast%3DJinek%26aufirst%3DM.%26aulast%3DEast%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DE.%26aulast%3DDoudna%26aufirst%3DJ.%26atitle%3DRNA-programmed%2520genome%2520editing%2520in%2520human%2520cells%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00471%26doi%3D10.7554%2FeLife.00471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goralski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaskill, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capota, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ting, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nijhawan, D.</span></span> <span> </span><span class="NLM_article-title">Anticancer sulfonamides target splicing by inducing RBM39 degradation via recruitment to DCAF15</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>356</i></span>,  <span class="NLM_fpage">eaal3755</span>, <span class="refDoi"> DOI: 10.1126/science.aal3755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.aal3755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28302793" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2017&pages=eaal3755&author=T.+Hanauthor=M.+Goralskiauthor=N.+Gaskillauthor=E.+Capotaauthor=J.+Kimauthor=T.+C.+Tingauthor=Y.+Xieauthor=N.+S.+Williamsauthor=D.+Nijhawan&title=Anticancer+sulfonamides+target+splicing+by+inducing+RBM39+degradation+via+recruitment+to+DCAF15&doi=10.1126%2Fscience.aal3755"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1126%2Fscience.aal3755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aal3755%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DT.%26aulast%3DGoralski%26aufirst%3DM.%26aulast%3DGaskill%26aufirst%3DN.%26aulast%3DCapota%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DTing%26aufirst%3DT.%2BC.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DWilliams%26aufirst%3DN.%2BS.%26aulast%3DNijhawan%26aufirst%3DD.%26atitle%3DAnticancer%2520sulfonamides%2520target%2520splicing%2520by%2520inducing%2520RBM39%2520degradation%2520via%2520recruitment%2520to%2520DCAF15%26jtitle%3DScience%26date%3D2017%26volume%3D356%26spage%3Deaal3755%26doi%3D10.1126%2Fscience.aal3755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghtaling, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, T. M.</span></span> <span> </span><span class="NLM_article-title">Using transcriptome sequencing to identify mechanisms of drug action and resistance</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1038/nchembio.779</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.779" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=22327403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XitFalsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=235-237&author=S.+A.+Wackerauthor=B.+R.+Houghtalingauthor=O.+Elementoauthor=T.+M.+Kapoor&title=Using+transcriptome+sequencing+to+identify+mechanisms+of+drug+action+and+resistance&doi=10.1038%2Fnchembio.779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Using transcriptome sequencing to identify mechanisms of drug action and resistance</span></div><div class="casAuthors">Wacker, Sarah A.; Houghtaling, Benjamin R.; Elemento, Olivier; Kapoor, Tarun M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">235-237</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Detg. mechanisms of drug action in human cells remains a major challenge.  Here we describe an approach in which multiple-drug-resistant clones are isolated and transcriptome sequencing is used to find mutations in each clone.  Further anal. of mutations common to more than one clone can identify a drug's physiol. target and indirect resistance mechanisms, as indicated by our proof-of-concept studies of the cytotoxic anticancer drugs BI 2536 and bortezomib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTYM1eLSgI47Vg90H21EOLACvtfcHk0ljEyKNLbXVsiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitFalsr0%253D&md5=d4a28e70029356c94f65c4895158b96d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.779%26sid%3Dliteratum%253Aachs%26aulast%3DWacker%26aufirst%3DS.%2BA.%26aulast%3DHoughtaling%26aufirst%3DB.%2BR.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DKapoor%26aufirst%3DT.%2BM.%26atitle%3DUsing%2520transcriptome%2520sequencing%2520to%2520identify%2520mechanisms%2520of%2520drug%2520action%2520and%2520resistance%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D235%26epage%3D237%26doi%3D10.1038%2Fnchembio.779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smurnyy, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWhinnie, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span> <span> </span><span class="NLM_article-title">DNA sequencing and CRISPR-Cas9 gene editing for target validation in mammalian cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">625</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.1550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24929529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=623-625&author=Y.+Smurnyyauthor=M.+Caiauthor=H.+Wuauthor=E.+McWhinnieauthor=J.+A.+Tallaricoauthor=Y.+Yangauthor=Y.+Feng&title=DNA+sequencing+and+CRISPR-Cas9+gene+editing+for+target+validation+in+mammalian+cells&doi=10.1038%2Fnchembio.1550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">DNA sequencing and CRISPR-Cas9 gene editing for target validation in mammalian cells</span></div><div class="casAuthors">Smurnyy, Yegor; Cai, Mi; Wu, Hua; McWhinnie, Elizabeth; Tallarico, John A.; Yang, Yi; Feng, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">623-625</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Identification and validation of drug-resistant mutations can provide important insights into the mechanism of action of a compd.  Here we demonstrate the feasibility of such an approach in mammalian cells using next-generation sequencing of drug-resistant clones and CRISPR-Cas9-mediated gene editing on two drug-target pairs, 6-thioguanine-HPRT1 and triptolide-ERCC3.  We showed that disrupting functional HPRT1 allele or introducing ERCC3 point mutations by gene editing can confer drug resistance in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDft4QWmx6YbVg90H21EOLACvtfcHk0ljEyKNLbXVsiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslGisb4%253D&md5=b05f570ebd84f2edb49c759214272abc</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1550%26sid%3Dliteratum%253Aachs%26aulast%3DSmurnyy%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMcWhinnie%26aufirst%3DE.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DY.%26atitle%3DDNA%2520sequencing%2520and%2520CRISPR-Cas9%2520gene%2520editing%2520for%2520target%2520validation%2520in%2520mammalian%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D623%26epage%3D625%26doi%3D10.1038%2Fnchembio.1550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLauchlan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">MELK is not necessary for the proliferation of basal-like breast cancer cells</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e22693</span>, <span class="refDoi"> DOI: 10.7554/eLife.26693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.7554%2FeLife.26693" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=e22693&author=H.+T.+Huangauthor=H.+S.+Seoauthor=T.+Zhangauthor=Y.+Wangauthor=B.+Jiangauthor=Q.+Liauthor=D.+L.+Buckleyauthor=B.+Nabetauthor=J.+M.+Robertsauthor=J.+Paulkauthor=S.+Dastjerdiauthor=G.+E.+Winterauthor=H.+McLauchlanauthor=J.+Moranauthor=J.+E.+Bradnerauthor=M.+J.+Eckauthor=S.+Dhe-Paganonauthor=J.+J.+Zhaoauthor=N.+S.+Gray&title=MELK+is+not+necessary+for+the+proliferation+of+basal-like+breast+cancer+cells&doi=10.7554%2FeLife.26693"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.7554%2FeLife.26693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.26693%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BT.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DMcLauchlan%26aufirst%3DH.%26aulast%3DMoran%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DMELK%2520is%2520not%2520necessary%2520for%2520the%2520proliferation%2520of%2520basal-like%2520breast%2520cancer%2520cells%26jtitle%3DeLife%26date%3D2017%26volume%3D6%26spage%3De22693%26doi%3D10.7554%2FeLife.26693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span> <span> </span><span class="NLM_article-title">Genetic screens in human cells using the CRISPR-Cas9 system</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1126/science.1246981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1246981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24336569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkslOq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=80-84&author=T.+Wangauthor=J.+J.+Weiauthor=D.+M.+Sabatiniauthor=E.+S.+Lander&title=Genetic+screens+in+human+cells+using+the+CRISPR-Cas9+system&doi=10.1126%2Fscience.1246981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic screens in human cells using the CRISPR-Cas9 system</span></div><div class="casAuthors">Wang, Tim; Wei, Jenny J.; Sabatini, David M.; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6166</span>),
    <span class="NLM_cas:pages">80-84</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The bacterial clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9 system for genome editing has greatly expanded the toolbox for mammalian genetics, enabling the rapid generation of isogenic cell lines and mice with modified alleles.  Here, we describe a pooled, loss-of-function genetic screening approach suitable for both pos. and neg. selection that uses a genome-scale lentiviral single-guide RNA (sgRNA) library.  SgRNA expression cassettes were stably integrated into the genome, which enabled a complex mutant pool to be tracked by massively parallel sequencing.  We used a library contg. 73,000 sgRNAs to generate knockout collections and performed screens in two human cell lines.  A screen for resistance to the nucleotide analog 6-thioguanine identified all expected members of the DNA mismatch repair pathway, whereas another for the DNA topoisomerase II (TOP2A) poison etoposide identified TOP2A, as expected, and also cyclin-dependent kinase 6, CDK6.  A neg. selection screen for essential genes identified numerous gene sets corresponding to fundamental processes.  Last, we show that sgRNA efficiency is assocd. with specific sequence motifs, enabling the prediction of more effective sgRNAs.  Collectively, these results establish Cas9/sgRNA screens as a powerful tool for systematic genetic anal. in mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKrKxZ3oDKMrVg90H21EOLACvtfcHk0ljLBY7BpedfPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkslOq&md5=d69a508793812f82efd087d8ee1ed6f6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.1246981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1246981%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DJ.%2BJ.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DGenetic%2520screens%2520in%2520human%2520cells%2520using%2520the%2520CRISPR-Cas9%2520system%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D80%26epage%3D84%26doi%3D10.1126%2Fscience.1246981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jost, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlbeck, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krenning, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menchon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prota, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhmanova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanenbaum, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Combined CRISPRi/a-based chemical genetic screens reveal that Rigosertib Is a microtubule-destabilizing agent</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.09.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.molcel.2017.09.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28985505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Wqs77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2017&pages=210-223&author=M.+Jostauthor=Y.+Chenauthor=L.+A.+Gilbertauthor=M.+A.+Horlbeckauthor=L.+Krenningauthor=G.+Menchonauthor=A.+Raiauthor=M.+Y.+Choauthor=J.+J.+Sternauthor=A.+E.+Protaauthor=M.+Kampmannauthor=A.+Akhmanovaauthor=M.+O.+Steinmetzauthor=M.+E.+Tanenbaumauthor=J.+S.+Weissman&title=Combined+CRISPRi%2Fa-based+chemical+genetic+screens+reveal+that+Rigosertib+Is+a+microtubule-destabilizing+agent&doi=10.1016%2Fj.molcel.2017.09.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Combined CRISPRi/a-Based Chemical Genetic Screens Reveal that Rigosertib Is a Microtubule-Destabilizing Agent</span></div><div class="casAuthors">Jost, Marco; Chen, Yuwen; Gilbert, Luke A.; Horlbeck, Max A.; Krenning, Lenno; Menchon, Gregory; Rai, Ankit; Cho, Min Y.; Stern, Jacob J.; Prota, Andrea E.; Kampmann, Martin; Akhmanova, Anna; Steinmetz, Michel O.; Tanenbaum, Marvin E.; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-223.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Chem. libraries paired with phenotypic screens can now readily identify compds. with therapeutic potential.  A central limitation to exploiting these compds., however, has been in identifying their relevant cellular targets.  Here, we present a two-tiered CRISPR-mediated chem.-genetic strategy for target identification: combined genome-wide knockdown and overexpression screening as well as focused, comparative chem.-genetic profiling.  Application of these strategies to rigosertib, a drug in phase 3 clin. trials for high-risk myelodysplastic syndrome whose mol. target had remained controversial, pointed singularly to microtubules as rigosertib's target.  We showed that rigosertib indeed directly binds to and destabilizes microtubules using cell biol., in vitro, and structural approaches.  Finally, expression of tubulin with a structure-guided mutation in the rigosertib-binding pocket conferred resistance to rigosertib, establishing that rigosertib kills cancer cells by destabilizing microtubules.  These results demonstrate the power of our chem.-genetic screening strategies for pinpointing the physiol. relevant targets of chem. agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBsbZCZePrwLVg90H21EOLACvtfcHk0ljLBY7BpedfPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Wqs77L&md5=115f89a6cc9eef8c3709c06b74109c51</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.09.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.09.012%26sid%3Dliteratum%253Aachs%26aulast%3DJost%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26aulast%3DHorlbeck%26aufirst%3DM.%2BA.%26aulast%3DKrenning%26aufirst%3DL.%26aulast%3DMenchon%26aufirst%3DG.%26aulast%3DRai%26aufirst%3DA.%26aulast%3DCho%26aufirst%3DM.%2BY.%26aulast%3DStern%26aufirst%3DJ.%2BJ.%26aulast%3DProta%26aufirst%3DA.%2BE.%26aulast%3DKampmann%26aufirst%3DM.%26aulast%3DAkhmanova%26aufirst%3DA.%26aulast%3DSteinmetz%26aufirst%3DM.%2BO.%26aulast%3DTanenbaum%26aufirst%3DM.%2BE.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DCombined%2520CRISPRi%252Fa-based%2520chemical%2520genetic%2520screens%2520reveal%2520that%2520Rigosertib%2520Is%2520a%2520microtubule-destabilizing%2520agent%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D68%26spage%3D210%26epage%3D223%26doi%3D10.1016%2Fj.molcel.2017.09.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deans, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgens, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okesli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlbeck, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glenn, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carette, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khosla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span> <span> </span><span class="NLM_article-title">Parallel shRNA and CRISPR-Cas9 screens enable antiviral drug target identification</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">366</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.2050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27018887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XksFCru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=361-366&author=R.+M.+Deansauthor=D.+W.+Morgensauthor=A.+Okesliauthor=S.+Pillayauthor=M.+A.+Horlbeckauthor=M.+Kampmannauthor=L.+A.+Gilbertauthor=A.+Liauthor=R.+Mateoauthor=M.+Smithauthor=J.+S.+Glennauthor=J.+E.+Caretteauthor=C.+Khoslaauthor=M.+C.+Bassik&title=Parallel+shRNA+and+CRISPR-Cas9+screens+enable+antiviral+drug+target+identification&doi=10.1038%2Fnchembio.2050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel shRNA and CRISPR-Cas9 screens enable antiviral drug target identification</span></div><div class="casAuthors">Deans, Richard M.; Morgens, David W.; Okesli, Ayse; Pillay, Sirika; Horlbeck, Max A.; Kampmann, Martin; Gilbert, Luke A.; Li, Amy; Mateo, Roberto; Smith, Mark; Glenn, Jeffrey S.; Carette, Jan E.; Khosla, Chaitan; Bassik, Michael C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">361-366</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Broad-spectrum antiviral drugs targeting host processes could potentially treat a wide range of viruses while reducing the likelihood of emergent resistance.  Despite great promise as therapeutics, such drugs remain largely elusive.  Here the authors used parallel genome-wide high-coverage short hairpin RNA (shRNA) and clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9 screens to identify the cellular target and mechanism of action of GSK983, a potent broad-spectrum antiviral with unexplained cytotoxicity.  The authors found that GSK983 blocked cell proliferation and dengue virus replication by inhibiting the pyrimidine biosynthesis enzyme dihydroorotate dehydrogenase (DHODH).  Guided by mechanistic insights from both genomic screens, the authors found that exogenous deoxycytidine markedly reduced GSK983 cytotoxicity but not antiviral activity, providing an attractive new approach to improve the therapeutic window of DHODH inhibitors against RNA viruses.  The authors' results highlight the distinct advantages and limitations of each screening method for identifying drug targets, and demonstrate the utility of parallel knockdown and knockout screens for comprehensive probing of drug activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1-QBjLboncbVg90H21EOLACvtfcHk0lgHTL4lFcsMbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksFCru7w%253D&md5=f1c93508518375e686dd87fd38c8e994</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2050%26sid%3Dliteratum%253Aachs%26aulast%3DDeans%26aufirst%3DR.%2BM.%26aulast%3DMorgens%26aufirst%3DD.%2BW.%26aulast%3DOkesli%26aufirst%3DA.%26aulast%3DPillay%26aufirst%3DS.%26aulast%3DHorlbeck%26aufirst%3DM.%2BA.%26aulast%3DKampmann%26aufirst%3DM.%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DMateo%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DM.%26aulast%3DGlenn%26aufirst%3DJ.%2BS.%26aulast%3DCarette%26aufirst%3DJ.%2BE.%26aulast%3DKhosla%26aufirst%3DC.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26atitle%3DParallel%2520shRNA%2520and%2520CRISPR-Cas9%2520screens%2520enable%2520antiviral%2520drug%2520target%2520identification%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D361%26epage%3D366%26doi%3D10.1038%2Fnchembio.2050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparbier, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beavis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolzenburg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgens, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neeson, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darcy, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voskoboinik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cebon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehner, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">CMTM6 maintains the expression of PD-L1 and regulates anti-tumour immunity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>549</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1038/nature23643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnature23643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28813417" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGktrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=549&publication_year=2017&pages=101-105&author=M.+L.+Burrauthor=C.+E.+Sparbierauthor=Y.+C.+Chanauthor=J.+C.+Williamsonauthor=K.+Woodsauthor=P.+A.+Beavisauthor=E.+Y.+N.+Lamauthor=M.+A.+Hendersonauthor=C.+C.+Bellauthor=S.+Stolzenburgauthor=O.+Gilanauthor=S.+Bloorauthor=T.+Nooriauthor=D.+W.+Morgensauthor=M.+C.+Bassikauthor=P.+J.+Neesonauthor=A.+Behrenauthor=P.+K.+Darcyauthor=S.+J.+Dawsonauthor=I.+Voskoboinikauthor=J.+A.+Trapaniauthor=J.+Cebonauthor=P.+J.+Lehnerauthor=M.+A.+Dawson&title=CMTM6+maintains+the+expression+of+PD-L1+and+regulates+anti-tumour+immunity&doi=10.1038%2Fnature23643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">CMTM6 maintains the expression of PD-L1 and regulates anti-tumour immunity</span></div><div class="casAuthors">Burr, Marian L.; Sparbier, Christina E.; Chan, Yih-Chih; Williamson, James C.; Woods, Katherine; Beavis, Paul A.; Lam, Enid Y. N.; Henderson, Melissa A.; Bell, Charles C.; Stolzenburg, Sabine; Gilan, Omer; Bloor, Stuart; Noori, Tahereh; Morgens, David W.; Bassik, Michael C.; Neeson, Paul J.; Behren, Andreas; Darcy, Phillip K.; Dawson, Sarah-Jane; Voskoboinik, Ilia; Trapani, Joseph A.; Cebon, Jonathan; Lehner, Paul J.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">549</span>
        (<span class="NLM_cas:issue">7670</span>),
    <span class="NLM_cas:pages">101-105</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells exploit the expression of the programmed death-1 (PD-1) ligand 1 (PD-L1) to subvert T-cell-mediated immunosurveillance.  The success of therapies that disrupt PD-L1-mediated tumor tolerance has highlighted the need to understand the mol. regulation of PD-L1 expression.  Here we identify the uncharacterized protein CMTM6 as a crit. regulator of PD-L1 in a broad range of cancer cells, by using a genome-wide CRISPR-Cas9 screen.  CMTM6 is a ubiquitously expressed protein that binds PD-L1 and maintains its cell surface expression.  CMTM6 is not required for PD-L1 maturation but co-localizes with PD-L1 at the plasma membrane and in recycling endosomes, where it prevents PD-L1 from being targeted for lysosome-mediated degrdn.  Using a quant. approach to profile the entire plasma membrane proteome, we find that CMTM6 displays specificity for PD-L1.  Notably, CMTM6 depletion decreases PD-L1 without compromising cell surface expression of MHC class I.  CMTM6 depletion, via the redn. of PD-L1, significantly alleviates the suppression of tumor-specific T cell activity in vitro and in vivo.  These findings provide insights into the biol. of PD-L1 regulation, identify a previously unrecognized master regulator of this crit. immune checkpoint and highlight a potential therapeutic target to overcome immune evasion by tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOaE7qlrP04LVg90H21EOLACvtfcHk0lgHTL4lFcsMbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGktrvK&md5=c37c58dcfa75b3057fe4e9635789281d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature23643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature23643%26sid%3Dliteratum%253Aachs%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DSparbier%26aufirst%3DC.%2BE.%26aulast%3DChan%26aufirst%3DY.%2BC.%26aulast%3DWilliamson%26aufirst%3DJ.%2BC.%26aulast%3DWoods%26aufirst%3DK.%26aulast%3DBeavis%26aufirst%3DP.%2BA.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DHenderson%26aufirst%3DM.%2BA.%26aulast%3DBell%26aufirst%3DC.%2BC.%26aulast%3DStolzenburg%26aufirst%3DS.%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DBloor%26aufirst%3DS.%26aulast%3DNoori%26aufirst%3DT.%26aulast%3DMorgens%26aufirst%3DD.%2BW.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26aulast%3DNeeson%26aufirst%3DP.%2BJ.%26aulast%3DBehren%26aufirst%3DA.%26aulast%3DDarcy%26aufirst%3DP.%2BK.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DVoskoboinik%26aufirst%3DI.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26aulast%3DCebon%26aufirst%3DJ.%26aulast%3DLehner%26aufirst%3DP.%2BJ.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DCMTM6%2520maintains%2520the%2520expression%2520of%2520PD-L1%2520and%2520regulates%2520anti-tumour%2520immunity%26jtitle%3DNature%26date%3D2017%26volume%3D549%26spage%3D101%26epage%3D105%26doi%3D10.1038%2Fnature23643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koundakjian, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hultquist, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamothe-Molina, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krupzcak, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Beltran, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piechocka-Trocha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hacohen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, B. D.</span></span> <span> </span><span class="NLM_article-title">A genome-wide CRISPR screen identifies a restricted set of HIV host dependency factors</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">203</span>, <span class="refDoi"> DOI: 10.1038/ng.3741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fng.3741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27992415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWnsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=193-203&author=R.+J.+Parkauthor=T.+Wangauthor=D.+Koundakjianauthor=J.+F.+Hultquistauthor=P.+Lamothe-Molinaauthor=B.+Monelauthor=K.+Schumannauthor=H.+Yuauthor=K.+M.+Krupzcakauthor=W.+Garcia-Beltranauthor=A.+Piechocka-Trochaauthor=N.+J.+Kroganauthor=A.+Marsonauthor=D.+M.+Sabatiniauthor=E.+S.+Landerauthor=N.+Hacohenauthor=B.+D.+Walker&title=A+genome-wide+CRISPR+screen+identifies+a+restricted+set+of+HIV+host+dependency+factors&doi=10.1038%2Fng.3741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A genome-wide CRISPR screen identifies a restricted set of HIV host dependency factors</span></div><div class="casAuthors">Park, Ryan J.; Wang, Tim; Koundakjian, Dylan; Hultquist, Judd F.; Lamothe-Molina, Pedro; Monel, Blandine; Schumann, Kathrin; Yu, Haiyan; Krupzcak, Kevin M.; Garcia-Beltran, Wilfredo; Piechocka-Trocha, Alicja; Krogan, Nevan J.; Marson, Alexander; Sabatini, David M.; Lander, Eric S.; Hacohen, Nir; Walker, Bruce D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">193-203</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Host proteins are essential for HIV entry and replication and can be important nonviral therapeutic targets.  Large-scale RNA interference (RNAi)-based screens have identified nearly a thousand candidate host factors, but there is little agreement among studies and few factors have been validated.  Here we demonstrate that a genome-wide CRISPR-based screen identifies host factors in a physiol. relevant cell system.  We identify five factors, including the HIV co-receptors CD4 and CCR5, that are required for HIV infection yet are dispensable for cellular proliferation and viability.  Tyrosylprotein sulfotransferase 2 (TPST2) and solute carrier family 35 member B2 (SLC35B2) function in a common pathway to sulfate CCR5 on extracellular tyrosine residues, facilitating CCR5 recognition by the HIV envelope.  Activated leukocyte cell adhesion mol. (ALCAM) mediates cell aggregation, which is required for cell-to-cell HIV transmission.  We validated these pathways in primary human CD4+ T cells through Cas9-mediated knockout and antibody blockade.  Our findings indicate that HIV infection and replication rely on a limited set of host-dispensable genes and suggest that these pathways can be studied for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK_vf-hKNi6rVg90H21EOLACvtfcHk0lilcxhDC3DCfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWnsbbK&md5=42dac62c174b3f60e895d65d1f0273fd</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fng.3741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3741%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DKoundakjian%26aufirst%3DD.%26aulast%3DHultquist%26aufirst%3DJ.%2BF.%26aulast%3DLamothe-Molina%26aufirst%3DP.%26aulast%3DMonel%26aufirst%3DB.%26aulast%3DSchumann%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DKrupzcak%26aufirst%3DK.%2BM.%26aulast%3DGarcia-Beltran%26aufirst%3DW.%26aulast%3DPiechocka-Trocha%26aufirst%3DA.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26aulast%3DMarson%26aufirst%3DA.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DHacohen%26aufirst%3DN.%26aulast%3DWalker%26aufirst%3DB.%2BD.%26atitle%3DA%2520genome-wide%2520CRISPR%2520screen%2520identifies%2520a%2520restricted%2520set%2520of%2520HIV%2520host%2520dependency%2520factors%26jtitle%3DNat.%2520Genet.%26date%3D2017%26volume%3D49%26spage%3D193%26epage%3D203%26doi%3D10.1038%2Fng.3741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlbeck, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalta, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panning, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploegh, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Genome-scale CRISPR-mediated control of gene repression and activation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">661</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.cell.2014.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25307932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslelsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=647-661&author=L.%0AA.+Gilbertauthor=M.+A.+Horlbeckauthor=B.+Adamsonauthor=J.+E.+Villaltaauthor=Y.+Chenauthor=E.+H.+Whiteheadauthor=C.+Guimaraesauthor=B.+Panningauthor=H.+L.+Ploeghauthor=M.+C.+Bassikauthor=L.+S.+Qiauthor=M.+Kampmannauthor=J.+S.+Weissman&title=Genome-scale+CRISPR-mediated+control+of+gene+repression+and+activation&doi=10.1016%2Fj.cell.2014.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-scale CRISPR-mediated control of gene repression and activation</span></div><div class="casAuthors">Gilbert, Luke A.; Horlbeck, Max A.; Adamson, Britt; Villalta, Jacqueline E.; Chen, Yuwen; Whitehead, Evan H.; Guimaraes, Carla; Panning, Barbara; Ploegh, Hidde L.; Bassik, Michael C.; Qi, Lei S.; Kampmann, Martin; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">647-661</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">While the catalog of mammalian transcripts and their expression levels in different cell types and disease states is rapidly expanding, our understanding of transcript function lags behind.  We present a robust technol. enabling systematic investigation of the cellular consequences of repressing or inducing individual transcripts.  We identify rules for specific targeting of transcriptional repressors (CRISPRi), typically achieving 90%-99% knockdown with minimal off-target effects, and activators (CRISPRa) to endogenous genes via endonuclease-deficient Cas9.  Together they enable modulation of gene expression over a ∼1000-fold range.  Using these rules, we construct genome-scale CRISPRi and CRISPRa libraries, each of which we validate with two pooled screens.  Growth-based screens identify essential genes, tumor suppressors, and regulators of differentiation.  Screens for sensitivity to a cholera-diphtheria toxin provide broad insights into the mechanisms of pathogen entry, retrotranslocation and toxicity.  Our results establish CRISPRi and CRISPRa as powerful tools that provide rich and complementary information for mapping complex pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWZTeLYH9msrVg90H21EOLACvtfcHk0lilcxhDC3DCfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslelsrfJ&md5=3d7df984d13eb14b6f1b80f3b2e93b8d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26aulast%3DHorlbeck%26aufirst%3DM.%2BA.%26aulast%3DAdamson%26aufirst%3DB.%26aulast%3DVillalta%26aufirst%3DJ.%2BE.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWhitehead%26aufirst%3DE.%2BH.%26aulast%3DGuimaraes%26aufirst%3DC.%26aulast%3DPanning%26aufirst%3DB.%26aulast%3DPloegh%26aufirst%3DH.%2BL.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26aulast%3DQi%26aufirst%3DL.%2BS.%26aulast%3DKampmann%26aufirst%3DM.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DGenome-scale%2520CRISPR-mediated%2520control%2520of%2520gene%2520repression%2520and%2520activation%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D647%26epage%3D661%26doi%3D10.1016%2Fj.cell.2014.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thakore, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Ippolito, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabadi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gersbach, C. A.</span></span> <span> </span><span class="NLM_article-title">Highly specific epigenome editing by CRISPR-Cas9 repressors for silencing of distal regulatory elements</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1143</span>– <span class="NLM_lpage">1149</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnmeth.3630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26501517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslant7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1143-1149&author=P.+I.+Thakoreauthor=A.+M.+D%E2%80%99Ippolitoauthor=L.+Songauthor=A.+Safiauthor=N.+K.+Shivakumarauthor=A.+M.+Kabadiauthor=T.+E.+Reddyauthor=G.+E.+Crawfordauthor=C.+A.+Gersbach&title=Highly+specific+epigenome+editing+by+CRISPR-Cas9+repressors+for+silencing+of+distal+regulatory+elements&doi=10.1038%2Fnmeth.3630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Highly specific epigenome editing by CRISPR-Cas9 repressors for silencing of distal regulatory elements</span></div><div class="casAuthors">Thakore, Pratiksha I.; D'Ippolito, Anthony M.; Song, Lingyun; Safi, Alexias; Shivakumar, Nishkala K.; Kabadi, Ami M.; Reddy, Timothy E.; Crawford, Gregory E.; Gersbach, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1143-1149</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenome editing with the CRISPR (clustered, regularly interspaced, short palindromic repeats)-Cas9 platform is a promising technol. for modulating gene expression to direct cell phenotype and to dissect the causal epigenetic mechanisms of gene regulation.  Fusions of nuclease-inactive dCas9 to the Kruppel-assocd. box (KRAB) repressor (dCas9-KRAB) can silence target gene expression, but the genome-wide specificity and the extent of heterochromatin formation catalyzed by dCas9-KRAB are not known.  We targeted dCas9-KRAB to the HS2 enhancer, a distal regulatory element that orchestrates the expression of multiple globin genes, and obsd. highly specific induction of H3K9 trimethylation (H3K9me3) at the enhancer and decreased chromatin accessibility of both the enhancer and its promoter targets.  Targeted epigenetic modification of HS2 silenced the expression of multiple globin genes, with minimal off-target changes in global gene expression.  These results demonstrate that repression mediated by dCas9-KRAB is sufficiently specific to disrupt the activity of individual enhancers via local modification of the epigenome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1C8tZjvdyCbVg90H21EOLACvtfcHk0lhCHSOpMLMzOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslant7fO&md5=aca39b98762607cafc8f9568b2b8200e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3630%26sid%3Dliteratum%253Aachs%26aulast%3DThakore%26aufirst%3DP.%2BI.%26aulast%3DD%25E2%2580%2599Ippolito%26aufirst%3DA.%2BM.%26aulast%3DSong%26aufirst%3DL.%26aulast%3DSafi%26aufirst%3DA.%26aulast%3DShivakumar%26aufirst%3DN.%2BK.%26aulast%3DKabadi%26aufirst%3DA.%2BM.%26aulast%3DReddy%26aufirst%3DT.%2BE.%26aulast%3DCrawford%26aufirst%3DG.%2BE.%26aulast%3DGersbach%26aufirst%3DC.%2BA.%26atitle%3DHighly%2520specific%2520epigenome%2520editing%2520by%2520CRISPR-Cas9%2520repressors%2520for%2520silencing%2520of%2520distal%2520regulatory%2520elements%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1143%26epage%3D1149%26doi%3D10.1038%2Fnmeth.3630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grasso, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishihara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, P. S.</span></span> <span> </span><span class="NLM_article-title">Bioconjugation by native chemical tagging of C-H bonds</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">12994</span>– <span class="NLM_lpage">12997</span>, <span class="refDoi"> DOI: 10.1021/ja407739y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja407739y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjur3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=12994-12997&author=Q.+Zhouauthor=J.+Guiauthor=C.-M.+Panauthor=E.+Alboneauthor=X.+Chengauthor=E.+M.+Suhauthor=L.+Grassoauthor=Y.+Ishiharaauthor=P.+S.+Baran&title=Bioconjugation+by+native+chemical+tagging+of+C-H+bonds&doi=10.1021%2Fja407739y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Bioconjugation by Native Chemical Tagging of C-H Bonds</span></div><div class="casAuthors">Zhou, Qianghui; Gui, Jinghan; Pan, Chung-Mao; Albone, Earl; Cheng, Xin; Suh, Edward M.; Grasso, Luigi; Ishihara, Yoshihiro; Baran, Phil S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12994-12997</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A general C-H functionalization method for the tagging of natural products and pharmaceuticals is described.  An azide-contg. sulfinate reagent allows the appendage of azidoalkyl chains onto heteroaroms., the product of which can then be attached to a monoclonal antibody by a "click" reaction.  This strategy expands the breadth of bioactive small mols. that can be linked to macromols. in a manner that is beyond the scope of existing methods in bioconjugation to permit tagging of the "seemingly untaggable".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVwraFgGUte7Vg90H21EOLACvtfcHk0lhCHSOpMLMzOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjur3P&md5=dca0179112d7d412fa38073c986a0730</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja407739y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja407739y%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DGui%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DC.-M.%26aulast%3DAlbone%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DSuh%26aufirst%3DE.%2BM.%26aulast%3DGrasso%26aufirst%3DL.%26aulast%3DIshihara%26aufirst%3DY.%26aulast%3DBaran%26aufirst%3DP.%2BS.%26atitle%3DBioconjugation%2520by%2520native%2520chemical%2520tagging%2520of%2520C-H%2520bonds%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D12994%26epage%3D12997%26doi%3D10.1021%2Fja407739y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanoh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simizu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muroi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osada, H.</span></span> <span> </span><span class="NLM_article-title">Photo-cross-linked small-molecule affinity matrix for facilitating forward and reverse chemical genetics</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">3559</span>– <span class="NLM_lpage">3562</span>, <span class="refDoi"> DOI: 10.1002/anie.200462370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.200462370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsVyktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=3559-3562&author=N.+Kanohauthor=K.+Hondaauthor=S.+Simizuauthor=M.+Muroiauthor=H.+Osada&title=Photo-cross-linked+small-molecule+affinity+matrix+for+facilitating+forward+and+reverse+chemical+genetics&doi=10.1002%2Fanie.200462370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Photo-cross-linked small-molecule affinity matrix for facilitating forward and reverse chemical genetics</span></div><div class="casAuthors">Kanoh, Naoki; Honda, Kaori; Simizu, Siro; Muroi, Makoto; Osada, Hiroyuki</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3559-3562</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A photoaffinity reaction has been used to photo-cross-link small mols. on agarose beads to identify and purify the binding proteins.  The authors prepd. photoaffinity-linker-coated (PALC) agarose/Sepharose beads and demonstrated that the photo-cross-linked small-mol. agarose beads can be used to identify and purify the binding protein.  Complex small mols. such as FK506 and cyclosporin A could be introduced on the affinity matrix by using a uniform and straightforward procedure.  Although the present procedure can not be applied to small mols. that degrade upon UV irradn., technologies aimed at immobilizing ligands on an affinity matrix in a functional-group-independent manner are expected to facilitate the identification of mol. targets of small mols. in forward chem. genetics, and is also considered potentially useful for the confirmation of specificity between small mols. and proteins of interest in the field of reverse chem. genetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8ONPNgv4zlbVg90H21EOLACvtfcHk0lhCHSOpMLMzOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsVyktrY%253D&md5=82d1f6a2425c50dfd4cdd92bd9ed9fb5</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462370%26sid%3Dliteratum%253Aachs%26aulast%3DKanoh%26aufirst%3DN.%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DSimizu%26aufirst%3DS.%26aulast%3DMuroi%26aufirst%3DM.%26aulast%3DOsada%26aufirst%3DH.%26atitle%3DPhoto-cross-linked%2520small-molecule%2520affinity%2520matrix%2520for%2520facilitating%2520forward%2520and%2520reverse%2520chemical%2520genetics%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D3559%26epage%3D3562%26doi%3D10.1002%2Fanie.200462370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishiya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takashima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsutsumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okawa, K.</span></span> <span> </span><span class="NLM_article-title">A new efficient method of generating photoaffinity beads for drug target identification</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.bmcl.2017.01.021" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=834-840&author=Y.+Nishiyaauthor=T.+Hamadaauthor=M.+Abeauthor=M.+Takashimaauthor=K.+Tsutsumiauthor=K.+Okawa&title=A+new+efficient+method+of+generating+photoaffinity+beads+for+drug+target+identification&doi=10.1016%2Fj.bmcl.2017.01.021"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DNishiya%26aufirst%3DY.%26aulast%3DHamada%26aufirst%3DT.%26aulast%3DAbe%26aufirst%3DM.%26aulast%3DTakashima%26aufirst%3DM.%26aulast%3DTsutsumi%26aufirst%3DK.%26aulast%3DOkawa%26aufirst%3DK.%26atitle%3DA%2520new%2520efficient%2520method%2520of%2520generating%2520photoaffinity%2520beads%2520for%2520drug%2520target%2520identification%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D834%26epage%3D840%26doi%3D10.1016%2Fj.bmcl.2017.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedman Ohana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uyeda, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Encell, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Deciphering the cellular targets of bioactive compounds using a chloroalkane capture tag</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2316</span>– <span class="NLM_lpage">2324</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00351</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00351" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFGitLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2316-2324&author=R.+Friedman+Ohanaauthor=T.+A.+Kirklandauthor=C.+C.+Woodroofeauthor=S.+Levinauthor=H.+T.+Uyedaauthor=P.+Ottoauthor=R.+Hurstauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=L.+P.+Encellauthor=K.+V.+Wood&title=Deciphering+the+cellular+targets+of+bioactive+compounds+using+a+chloroalkane+capture+tag&doi=10.1021%2Facschembio.5b00351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Deciphering the Cellular Targets of Bioactive Compounds Using a Chloroalkane Capture Tag</span></div><div class="casAuthors">Friedman Ohana, Rachel; Kirkland, Thomas A.; Woodroofe, Carolyn C.; Levin, Sergiy; Uyeda, H. Tetsuo; Otto, Paul; Hurst, Robin; Robers, Matthew B.; Zimmerman, Kris; Encell, Lance P.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2316-2324</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phenotypic screening of compd. libraries is a significant trend in drug discovery, yet success can be hindered by difficulties in identifying the underlying cellular targets.  Current approaches rely on tethering bioactive compds. to a capture tag or surface to allow selective enrichment of interacting proteins for subsequent identification by mass spectrometry.  Such methods are often constrained by ineffective capture of low affinity and low abundance targets.  In addn., these methods are often not compatible with living cells and therefore cannot be used to verify the pharmacol. activity of the tethered compds.  The authors have developed a novel chloroalkane capture tag that minimally affects compd. potency in cultured cells, allowing binding interactions with the targets to occur under conditions relevant to the desired cellular phenotype.  Subsequent isolation of the interacting targets is achieved through rapid lysis and capture onto immobilized HaloTag protein.  Exchanging the chloroalkane tag for a fluorophore, the putative targets identified by mass spectrometry can be verified for direct binding to the compd. through resonance energy transfer.  Using the interaction between histone deacetylases (HDACs) and the inhibitor, Vorinostat (SAHA), as a model system, the authors were able to identify and verify all the known HDAC targets of SAHA as well as two previously undescribed targets, ADO and CPPED1.  The discovery of ADO as a target may provide mechanistic insight into a reported connection between SAHA and Huntington's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw7IYjjAcESrVg90H21EOLACvtfcHk0liJtlt-eryoHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFGitLfN&md5=85d569074a909033683ca519e4b44d00</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00351%26sid%3Dliteratum%253Aachs%26aulast%3DFriedman%2BOhana%26aufirst%3DR.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DUyeda%26aufirst%3DH.%2BT.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DHurst%26aufirst%3DR.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DEncell%26aufirst%3DL.%2BP.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DDeciphering%2520the%2520cellular%2520targets%2520of%2520bioactive%2520compounds%2520using%2520a%2520chloroalkane%2520capture%2520tag%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2316%26epage%3D2324%26doi%3D10.1021%2Facschembio.5b00351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedman Ohana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uyeda, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Encell, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Improved deconvolution of protein targets for bioactive compounds using a palladium cleavable chloroalkane capture tag</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2608</span>– <span class="NLM_lpage">2617</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00408</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00408" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKms7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2608-2617&author=R.+Friedman+Ohanaauthor=S.+Levinauthor=M.+G.+Woodauthor=K.+Zimmermanauthor=M.+L.+Dartauthor=M.+K.+Schwinnauthor=T.+A.+Kirklandauthor=R.+Hurstauthor=H.+T.+Uyedaauthor=L.+P.+Encellauthor=K.+V.+Wood&title=Improved+deconvolution+of+protein+targets+for+bioactive+compounds+using+a+palladium+cleavable+chloroalkane+capture+tag&doi=10.1021%2Facschembio.6b00408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Deconvolution of Protein Targets for Bioactive Compounds Using a Palladium Cleavable Chloroalkane Capture Tag</span></div><div class="casAuthors">Friedman Ohana, Rachel; Levin, Sergiy; Wood, Monika G.; Zimmerman, Kris; Dart, Melanie L.; Schwinn, Marie K.; Kirkland, Thomas A.; Hurst, Robin; Uyeda, H. Tetsuo; Encell, Lance P.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2608-2617</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The benefits provided by phenotypic screening of compd. libraries are often countered by difficulties in identifying the underlying cellular targets.  We recently described a new approach utilizing a chloroalkane capture tag, which can be chem. attached to bioactive compds. to facilitate the isolation of their resp. targets for subsequent identification by mass spectrometry.  The tag minimally affects compd. potency and membrane permeability, enabling target engagement inside cells.  Effective enrichment of these targets is achieved through selectivity in both their rapid capture onto immobilized HaloTag and their subsequent release by competitive elution.  Here, we describe a significant improvement to this method where selective elution was achieved through palladium-catalyzed cleavage of an allyl-carbamate linkage incorporated into the chloroalkane capture tag.  Selective tag cleavage provided robust release of captured targets exhibiting different modes of binding to the bioactive compd., including prolonged residence time and covalent interactions.  Using the kinase inhibitors ibrutinib and BIRB796 as model compds., we demonstrated the capability of this new method to identify both expected targets and "off-targets" exhibiting a range of binding affinities, cellular abundances, and binding characteristics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkjMO4VhhlNLVg90H21EOLACvtfcHk0liJtlt-eryoHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKms7rO&md5=dd52a1d47f74edb7b554ac985e3dbdc5</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00408%26sid%3Dliteratum%253Aachs%26aulast%3DFriedman%2BOhana%26aufirst%3DR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHurst%26aufirst%3DR.%26aulast%3DUyeda%26aufirst%3DH.%2BT.%26aulast%3DEncell%26aufirst%3DL.%2BP.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DImproved%2520deconvolution%2520of%2520protein%2520targets%2520for%2520bioactive%2520compounds%2520using%2520a%2520palladium%2520cleavable%2520chloroalkane%2520capture%2520tag%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2608%26epage%3D2617%26doi%3D10.1021%2Facschembio.6b00408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendez-Johnson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman Ohana, R.</span></span> <span> </span><span class="NLM_article-title">Target identification using cell permeable and cleavable chloroalkane derivatized small molecules</span>. <i>Methods Mol. Biol. (N. Y., NY, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1647</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-7201-2_6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2F978-1-4939-7201-2_6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28808997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC1cfns1Kguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1647&publication_year=2017&pages=91-108&author=J.+L.+Mendez-Johnsonauthor=D.+L.+Danielsauthor=M.+Urhauthor=R.+Friedman+Ohana&title=Target+identification+using+cell+permeable+and+cleavable+chloroalkane+derivatized+small+molecules&doi=10.1007%2F978-1-4939-7201-2_6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Target Identification Using Cell Permeable and Cleavable Chloroalkane Derivatized Small Molecules</span></div><div class="casAuthors">Mendez-Johnson Jacqui L; Daniels Danette L; Urh Marjeta; Friedman Ohana Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in molecular biology (Clifton, N.J.)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1647</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">91-108</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">An important aspect for gaining functional insight into the activity of small molecules revealed through phenotypic screening is the identification of their interacting proteins.  Yet, isolating and validating these interacting proteins remains difficult.  Here, we present a new approach utilizing a chloroalkane (CA) moiety capture handle, which can be chemically attached to small molecules to isolate their respective protein targets.  Derivatization of small molecules with the CA moiety has been shown to not significantly impact their cell permeability or potency, allowing for phenotypic validation of the derivatized small molecule prior to capture.  The retention of cell permeability also allows for treatment of live cells with the derivatized small molecule and the CA moiety enables rapid covalent capture onto HaloTag coated magnetic beads.  Additionally, several options are available for the elution of interacting proteins, including chemical cleavage of the CA moiety, competitive elution using excess unmodified small molecule, or sodium dodecyl sulfate (SDS) elution.  These features taken together yield a highly robust and efficient process for target identification, including capture of weak or low abundance interactors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQa3ZJ12fU6eXthwrmtH4OBfW6udTcc2eZ7Jt9q85HZZLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfns1Kguw%253D%253D&md5=b457db4ebabec8ddeb9bc0ef6213db98</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7201-2_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-7201-2_6%26sid%3Dliteratum%253Aachs%26aulast%3DMendez-Johnson%26aufirst%3DJ.%2BL.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DFriedman%2BOhana%26aufirst%3DR.%26atitle%3DTarget%2520identification%2520using%2520cell%2520permeable%2520and%2520cleavable%2520chloroalkane%2520derivatized%2520small%2520molecules%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2017%26volume%3D1647%26spage%3D91%26epage%3D108%26doi%3D10.1007%2F978-1-4939-7201-2_6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span> <span> </span><span class="NLM_article-title">Identifying cellular targets of small-molecule probes and drugs with biochemical enrichment and SILAC</span>. <i>Methods Mol. Biol. (N. Y., NY, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>803</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-364-6_9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2F978-1-61779-364-6_9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=22065222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlersLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=803&publication_year=2012&pages=129-140&author=S.+E.+Ongauthor=X.+Liauthor=M.+Schenoneauthor=S.+L.+Schreiberauthor=S.+A.+Carr&title=Identifying+cellular+targets+of+small-molecule+probes+and+drugs+with+biochemical+enrichment+and+SILAC&doi=10.1007%2F978-1-61779-364-6_9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying cellular targets of small-molecule probes and drugs with biochemical enrichment and SILAC</span></div><div class="casAuthors">Ong, Shao-En; Li, Xiaoyu; Schenone, Monica; Schreiber, Stuart L.; Carr, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">803</span>
        (<span class="NLM_cas:issue">Chemical Proteomics</span>),
    <span class="NLM_cas:pages">129-140</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Sequencing of the human genome in the last decade has not yet led to a concomitant increase in the nos. of novel drug targets.  While the pharmaceutical industry has invested heavily in improving drugs for existing protein targets, it has not tended toward a similar investment in exptl. approaches to identify cellular targets of drugs.  It is striking that the targets of numerous widely used FDA-approved drugs remain unknown.  The development of robust, unbiased methods for target identification would greatly enhance our understanding the mechanisms-of-action of small mols.  Cell-based phenotypic screens followed by unbiased target identification have the potential to identify novel combinations of small mols. and their protein targets, shed light on drug polypharmacol., and enable unbiased screening approaches to drug discovery.  Classical biochem. enrichment with immobilized small mols. has been used for over four decades but has been limited by issues concerning specificity and sensitivity.  The application of mass spectrometry-based quant. proteomics in combination with these affinity reagents has proven to be esp. useful in addressing these common issues in affinity purifn. expts.  We describe the use of SILAC in identifying proteins that bind small-mol. probes and drugs in a cellular context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-0ZARyhSotbVg90H21EOLACvtfcHk0lgVEWbMeTw4ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlersLjE&md5=4d95b8356fc6c4f9130465643e8c8e4b</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-364-6_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-364-6_9%26sid%3Dliteratum%253Aachs%26aulast%3DOng%26aufirst%3DS.%2BE.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26atitle%3DIdentifying%2520cellular%2520targets%2520of%2520small-molecule%2520probes%2520and%2520drugs%2520with%2520biochemical%2520enrichment%2520and%2520SILAC%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2012%26volume%3D803%26spage%3D129%26epage%3D140%26doi%3D10.1007%2F978-1-61779-364-6_9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murale, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haque, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span> <span> </span><span class="NLM_article-title">Photo-affinity labeling (PAL) in chemical proteomics: a handy tool to investigate protein-protein interactions (PPIs)</span>. <i>Proteome Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">14</span>, <span class="refDoi"> DOI: 10.1186/s12953-017-0123-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1186%2Fs12953-017-0123-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28652856" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=14&author=D.+P.+Muraleauthor=S.+C.+Hongauthor=M.+M.+Haqueauthor=J.+S.+Lee&title=Photo-affinity+labeling+%28PAL%29+in+chemical+proteomics%3A+a+handy+tool+to+investigate+protein-protein+interactions+%28PPIs%29&doi=10.1186%2Fs12953-017-0123-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1186%2Fs12953-017-0123-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12953-017-0123-3%26sid%3Dliteratum%253Aachs%26aulast%3DMurale%26aufirst%3DD.%2BP.%26aulast%3DHong%26aufirst%3DS.%2BC.%26aulast%3DHaque%26aufirst%3DM.%2BM.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26atitle%3DPhoto-affinity%2520labeling%2520%2528PAL%2529%2520in%2520chemical%2520proteomics%253A%2520a%2520handy%2520tool%2520to%2520investigate%2520protein-protein%2520interactions%2520%2528PPIs%2529%26jtitle%3DProteome%2520Sci.%26date%3D2017%26volume%3D15%26spage%3D14%26doi%3D10.1186%2Fs12953-017-0123-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marjanovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewandowski, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesbauer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ready, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span> <span> </span><span class="NLM_article-title">2-Aryl-5-carboxytetrazole as a new photoaffinity label for drug target identification</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">14609</span>– <span class="NLM_lpage">14615</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b06645</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b06645" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ylu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=14609-14615&author=A.+Hernerauthor=J.+Marjanovicauthor=T.+M.+Lewandowskiauthor=V.+Marinauthor=M.+Pattersonauthor=L.+Miesbauerauthor=D.+Readyauthor=J.+Williamsauthor=A.+Vasudevanauthor=Q.+Lin&title=2-Aryl-5-carboxytetrazole+as+a+new+photoaffinity+label+for+drug+target+identification&doi=10.1021%2Fjacs.6b06645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">2-Aryl-5-carboxytetrazole as a New Photoaffinity Label for Drug Target Identification</span></div><div class="casAuthors">Herner, Andras; Marjanovic, Jasmina; Lewandowski, Tracey M.; Marin, Violeta; Patterson, Melanie; Miesbauer, Laura; Ready, Damien; Williams, Jon; Vasudevan, Anil; Lin, Qing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">14609-14615</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Photoaffinity labels are powerful tools for dissecting ligand-protein interactions, and they have a broad utility in medicinal chem. and drug discovery.  Traditional photoaffinity labels work through nonspecific C-H/X-H bond insertion reactions with the protein of interest by the highly reactive photogenerated intermediate.  Herein, the authors report a new photoaffinity label, 2-aryl-5-carboxytetrazole (ACT), that interacts with the target protein via a unique mechanism in which the photogenerated carboxynitrile imine reacts with a proximal nucleophile near the target active site.  In two distinct case studies, the authors demonstrate that the attachment of ACT to a ligand does not significantly alter the binding affinity and specificity of the parent drug.  Compared with diazirine and benzophenone, two commonly used photoaffinity labels, in two case studies ACT showed higher photo-crosslinking yields toward their protein targets in vitro based on mass spectrometry anal.  In the in situ target identification studies, ACT successfully captured the desired targets with an efficiency comparable to the diazirine.  The authors expect that further development of this class of photoaffinity labels will lead to a broad range of applications across target identification, and validation and elucidation of the binding site in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4CPJmr7ODJLVg90H21EOLACvtfcHk0lgTxIMZ0nplWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ylu73O&md5=266b5013741b7ed1863cdc49094ba840</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b06645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b06645%26sid%3Dliteratum%253Aachs%26aulast%3DHerner%26aufirst%3DA.%26aulast%3DMarjanovic%26aufirst%3DJ.%26aulast%3DLewandowski%26aufirst%3DT.%2BM.%26aulast%3DMarin%26aufirst%3DV.%26aulast%3DPatterson%26aufirst%3DM.%26aulast%3DMiesbauer%26aufirst%3DL.%26aulast%3DReady%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DJ.%26aulast%3DVasudevan%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DQ.%26atitle%3D2-Aryl-5-carboxytetrazole%2520as%2520a%2520new%2520photoaffinity%2520label%2520for%2520drug%2520target%2520identification%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D14609%26epage%3D14615%26doi%3D10.1021%2Fjacs.6b06645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Targeting protein kinases with selective and semipromiscuous covalent inhibitors</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>548</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-397918-6.00004-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2FB978-0-12-397918-6.00004-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25399643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Onu7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=548&publication_year=2014&pages=93-116&author=R.+M.+Millerauthor=J.+Taunton&title=Targeting+protein+kinases+with+selective+and+semipromiscuous+covalent+inhibitors&doi=10.1016%2FB978-0-12-397918-6.00004-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein kinases with selective and semipromiscuous covalent inhibitors</span></div><div class="casAuthors">Miller, Rand M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">548</span>
        (<span class="NLM_cas:issue">Protein Kinase Inhibitors in Research and Medicine</span>),
    <span class="NLM_cas:pages">93-116</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Protein kinase inhibitors are an important class of therapeutics.  In addn., selective kinase inhibitors can often reveal unexpected biol. insights, augmenting genetic approaches and playing a decisive role in preclin. target validation studies.  Nevertheless, developing protein kinase inhibitors with sufficient selectivity and pharmacodynamic potency presents significant challenges.  Targeting noncatalytic cysteines with covalent inhibitors is a powerful approach to address both challenges simultaneously.  Here, we describe our efforts to design irreversible and reversible electrophilic inhibitors with varying degrees of kinase selectivity.  Highly selective covalent inhibitors have been used to elucidate the roles of p90 ribosomal protein S6 kinases in animal models of atherosclerosis and diabetes.  By contrast, semipromiscuous covalent inhibitors have revealed new therapeutic targets in disease-causing parasites and have shown utility as chemoproteomic probes for interrogating kinase occupancy in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvwGf3R34QG7Vg90H21EOLACvtfcHk0lgTxIMZ0nplWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Onu7nE&md5=7146d2377fbbc3d10cbf04ea0e304a77</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-397918-6.00004-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-397918-6.00004-5%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DTargeting%2520protein%2520kinases%2520with%2520selective%2520and%2520semipromiscuous%2520covalent%2520inhibitors%26jtitle%3DMethods%2520Enzymol.%26date%3D2014%26volume%3D548%26spage%3D93%26epage%3D116%26doi%3D10.1016%2FB978-0-12-397918-6.00004-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baruch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffery, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Enzyme activity–it’s all about image</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2003.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.tcb.2003.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=14729178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFegsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=29-35&author=A.+Baruchauthor=D.+A.+Jefferyauthor=M.+Bogyo&title=Enzyme+activity%E2%80%93it%E2%80%99s+all+about+image&doi=10.1016%2Fj.tcb.2003.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme activity - it's all about image</span></div><div class="casAuthors">Baruch, Amos; Jeffery, Douglas A.; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-35</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review and discussion.  Unraveling the functional roles of proteins is a major challenge facing the postgenome researcher.  Advances toward this goal have been made through the development of both chem. and biochem. tools for monitoring protein activity.  Recently, a myriad of fluorescence-based imaging tools have emerged for in vitro, in vivo, and whole animal applications.  These tools have provided methods to monitor the spatial and temporal distribution of proteins and bioorg. mols. dynamically.  Here, recent advances in chem. and biochem. techniques that allow the detection of enzymic activity within intact cells and in vivo are reviewed.  Such technologies have the potential to be integrated into drug-development programs to facilitate both the functional validation of pharmaceutical targets and the treatment of human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV67k9Zmph0bVg90H21EOLACvtfcHk0lh52w_7XYUZwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFegsA%253D%253D&md5=9e28761f959869c5184c9294f584cec7</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2003.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2003.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DBaruch%26aufirst%3DA.%26aulast%3DJeffery%26aufirst%3DD.%2BA.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DEnzyme%2520activity%25E2%2580%2593it%25E2%2580%2599s%2520all%2520about%2520image%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2004%26volume%3D14%26spage%3D29%26epage%3D35%26doi%3D10.1016%2Fj.tcb.2003.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodani, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. J.</span></span> <span> </span><span class="NLM_article-title">Reaction-based small-molecule fluorescent probes for chemoselective bioimaging</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">973</span>– <span class="NLM_lpage">984</span>, <span class="refDoi"> DOI: 10.1038/nchem.1500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchem.1500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23174976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslahurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=973-984&author=J.+Chanauthor=S.+C.+Dodaniauthor=C.+J.+Chang&title=Reaction-based+small-molecule+fluorescent+probes+for+chemoselective+bioimaging&doi=10.1038%2Fnchem.1500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Reaction-based small-molecule fluorescent probes for chemoselective bioimaging</span></div><div class="casAuthors">Chan, Jefferson; Dodani, Sheel C.; Chang, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">973-984</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The dynamic chem. diversity of elements, ions and mols. that form the basis of life offers both a challenge and an opportunity for study.  Small-mol. fluorescent probes can make use of selective, bioorthogonal chemistries to report on specific analytes in cells and in more complex biol. specimens.  These probes offer powerful reagents to interrogate the physiol. and pathol. of reactive chem. species in their native environments with minimal perturbation to living systems.  This Review presents a survey of tools and tactics for using such probes to detect biol. important chem. analytes.  The authors highlight design criteria for effective chem. tools for use in biol. applications as well as gaps for future exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp136q67e3PjLVg90H21EOLACvtfcHk0lh52w_7XYUZwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslahurnM&md5=2ab2ef20873f40a189b64fb5e2888998</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1500%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DDodani%26aufirst%3DS.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BJ.%26atitle%3DReaction-based%2520small-molecule%2520fluorescent%2520probes%2520for%2520chemoselective%2520bioimaging%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D973%26epage%3D984%26doi%3D10.1038%2Fnchem.1500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dubach, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinegoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumene Feruglio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">In vivo imaging of specific drug-target binding at subcellular resolution</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3946</span>, <span class="refDoi"> DOI: 10.1038/ncomms4946</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fncomms4946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24867710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVChs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3946&author=J.+M.+Dubachauthor=C.+Vinegoniauthor=R.+Mazitschekauthor=P.+Fumene+Feruglioauthor=L.+A.+Cameronauthor=R.+Weissleder&title=In+vivo+imaging+of+specific+drug-target+binding+at+subcellular+resolution&doi=10.1038%2Fncomms4946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo imaging of specific drug-target binding at subcellular resolution</span></div><div class="casAuthors">Dubach, J. M.; Vinegoni, C.; Mazitschek, R.; Fumene Feruglio, P.; Cameron, L. A.; Weissleder, R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3946</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The possibility of measuring binding of small-mol. drugs to desired targets in live cells could provide a better understanding of drug action.  However, current approaches mostly yield static data, require lysis or rely on indirect assays and thus often provide an incomplete understanding of drug action.  Here, we present a multiphoton fluorescence anisotropy microscopy live cell imaging technique to measure and map drug-target interaction in real time at subcellular resoln.  This approach is generally applicable using any fluorescently labeled drug and enables high-resoln. spatial and temporal mapping of bound and unbound drug distribution.  To illustrate our approach we measure intracellular target engagement of the chemotherapeutic Olaparib, a poly(ADP-ribose) polymerase inhibitor, in live cells and within a tumor in vivo.  These results are the first generalizable approach to directly measure drug-target binding in vivo and present a promising tool to enhance understanding of drug activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdsAoUwKS-W7Vg90H21EOLACvtfcHk0lh52w_7XYUZwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVChs70%253D&md5=dac66bc6dc9df7b13ab0863f4f1adf02</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fncomms4946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4946%26sid%3Dliteratum%253Aachs%26aulast%3DDubach%26aufirst%3DJ.%2BM.%26aulast%3DVinegoni%26aufirst%3DC.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DFumene%2BFeruglio%26aufirst%3DP.%26aulast%3DCameron%26aufirst%3DL.%2BA.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DIn%2520vivo%2520imaging%2520of%2520specific%2520drug-target%2520binding%2520at%2520subcellular%2520resolution%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3946%26doi%3D10.1038%2Fncomms4946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Protein organic chemistry and applications for labeling and engineering in live-cell systems</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4088</span>– <span class="NLM_lpage">4106</span>, <span class="refDoi"> DOI: 10.1002/anie.201207089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201207089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivVKhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=4088-4106&author=Y.+Takaokaauthor=A.+Ojidaauthor=I.+Hamachi&title=Protein+organic+chemistry+and+applications+for+labeling+and+engineering+in+live-cell+systems&doi=10.1002%2Fanie.201207089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Organic Chemistry and Applications for Labeling and Engineering in Live-Cell Systems</span></div><div class="casAuthors">Takaoka, Yousuke; Ojida, Akio; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4088-4106</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The modification of proteins with synthetic probes is a powerful means of elucidating and engineering the functions of proteins both in vitro and in live cells or in vivo.  Herein the authors review recent progress in chem.-based protein modification methods and their application in protein engineering, with particular emphasis on the following four strategies: (1) the bioconjugation reactions of amino acids on the surfaces of natural proteins, mainly applied in test-tube settings; (2) the bioorthogonal reactions of proteins with non-natural functional groups; (3) the coupling of recognition and reactive sites using an enzyme or short peptide tag-probe pair for labeling natural amino acids; and (4) ligand-directed labeling chemistries for the selective labeling of endogenous proteins in living systems.  Overall, these techniques represent a useful set of tools for application in chem. biol., with the methods 2-4 in particular being applicable to crude (living) habitats.  Although still in its infancy, the use of org. chem. for the manipulation of endogenous proteins, with subsequent applications in living systems, represents a worthy challenge for many chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocsb_JlBFIH7Vg90H21EOLACvtfcHk0lhME3HIsdqmZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivVKhsLY%253D&md5=9f7ef6846cc3e64f802254bd25d25941</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fanie.201207089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201207089%26sid%3Dliteratum%253Aachs%26aulast%3DTakaoka%26aufirst%3DY.%26aulast%3DOjida%26aufirst%3DA.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DProtein%2520organic%2520chemistry%2520and%2520applications%2520for%2520labeling%2520and%2520engineering%2520in%2520live-cell%2520systems%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D4088%26epage%3D4106%26doi%3D10.1002%2Fanie.201207089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galmozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joslyn, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallaro, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narvaiza, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Ligand and target discovery by fragment-based screening in human cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.cell.2016.12.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28111073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=527-541&author=C.+G.+Parkerauthor=A.+Galmozziauthor=Y.+Wangauthor=B.+E.+Correiaauthor=K.+Sasakiauthor=C.+M.+Joslynauthor=A.+S.+Kimauthor=C.+L.+Cavallaroauthor=R.+M.+Lawrenceauthor=S.+R.+Johnsonauthor=I.+Narvaizaauthor=E.+Saezauthor=B.+F.+Cravatt&title=Ligand+and+target+discovery+by+fragment-based+screening+in+human+cells&doi=10.1016%2Fj.cell.2016.12.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand and Target Discovery by Fragment-Based Screening in Human Cells</span></div><div class="casAuthors">Parker, Christopher G.; Galmozzi, Andrea; Wang, Yujia; Correia, Bruno E.; Sasaki, Kenji; Joslyn, Christopher M.; Kim, Arthur S.; Cavallaro, Cullen L.; Lawrence, R. Michael; Johnson, Stephen R.; Narvaiza, Inigo; Saez, Enrique; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">527-541.e29</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Advances in the synthesis and screening of small-mol. libraries have accelerated the discovery of chem. probes for studying biol. processes.  Still, only a small fraction of the human proteome has chem. ligands.  Here, we describe a platform that marries fragment-based ligand discovery with quant. chem. proteomics to map thousands of reversible small mol.-protein interactions directly in human cells, many of which can be site-specifically detd.  We show that fragment hits can be advanced to furnish selective ligands that affect the activity of proteins heretofore lacking chem. probes.  We further combine fragment-based chem. proteomics with phenotypic screening to identify small mols. that promote adipocyte differentiation by engaging the poorly characterized membrane protein PGRMC2.  Fragment-based screening in human cells thus provides an extensive proteome-wide map of protein ligandability and facilitates the coordinated discovery of bioactive small mols. and their mol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPk-YJwFlDwbVg90H21EOLACvtfcHk0lhME3HIsdqmZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeiu7Y%253D&md5=835264291015037ab9b7e446992caba3</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DC.%2BG.%26aulast%3DGalmozzi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DSasaki%26aufirst%3DK.%26aulast%3DJoslyn%26aufirst%3DC.%2BM.%26aulast%3DKim%26aufirst%3DA.%2BS.%26aulast%3DCavallaro%26aufirst%3DC.%2BL.%26aulast%3DLawrence%26aufirst%3DR.%2BM.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DNarvaiza%26aufirst%3DI.%26aulast%3DSaez%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DLigand%2520and%2520target%2520discovery%2520by%2520fragment-based%2520screening%2520in%2520human%2520cells%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D527%26epage%3D541%26doi%3D10.1016%2Fj.cell.2016.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitorino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: applications to biomarker discovery, in vivo imaging and drug discovery</span>. <i>Am. J. PharmacoGenomics</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.2165/00129785-200404060-00004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.2165%2F00129785-200404060-00004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=15651898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslOmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=371-381&author=A.+B.+Bergerauthor=P.+M.+Vitorinoauthor=M.+Bogyo&title=Activity-based+protein+profiling%3A+applications+to+biomarker+discovery%2C+in+vivo+imaging+and+drug+discovery&doi=10.2165%2F00129785-200404060-00004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: applications to biomarker discovery, in vivo imaging and drug discovery</span></div><div class="casAuthors">Berger, Alicia B.; Vitorino, Phillip M.; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of PharmacoGenomics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">371-381</span>CODEN:
                <span class="NLM_cas:coden">AJPMC8</span>;
        ISSN:<span class="NLM_cas:issn">1175-2203</span>.
    
            (<span class="NLM_cas:orgname">Adis International Ltd.</span>)
        </div><div class="casAbstract">A review.  The genomic revolution has created a wealth of information regarding the fundamental genetic code that defines the inner workings of a cell.  However, it has become clear that analyzing genome sequences alone will not lead to new therapies to fight human disease.  Rather, an understanding of protein function within the context of complex cellular networks will be required to facilitate the discovery of novel drug targets and, subsequently, new therapies directed against them.  The past ten years has seen a dramatic increase in technologies that allow large-scale, systems-based methods for anal. of global biol. processes and disease states.  In the field of proteomics, several well-established methods persist as a means to resolve and analyze complex mixts. of proteins derived from cells and tissues.  However, the resolving power of these methods is often challenged by the diverse and dynamic nature of the proteome.  The field of activity-based proteomics, or chem. proteomics, was established in an attempt to focus proteomic efforts on subsets of physiol. important protein targets.  This new approach to proteomics is centered around the use of small mols. termed activity-based probes (ABPs) as a means to tag, enrich, and isolate, distinct sets of proteins based on their enzymic activity.  Chem. probes can be 'tuned' to react with defined enzymic targets through the use of chem. reactive warhead groups, fused to selective binding elements that control their overall specificity.  As a result, ABPs function as highly specific; mechanism-based reagents that provide a direct readout of enzymic activity within complex proteomes.  Modification of protein targets by an ABP facilitates their purifn. and isolation, thereby eliminating many of the confounding issues of dynamic range in protein abundance.  In this review, the authors outline recent advances in the field of chem. proteomics.  Specifically, the authors highlight how this technol. can be applied to advance the fields of biomarker discovery, in vivo imaging, and small mol. screening and drug target discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaxZGkCOsvc7Vg90H21EOLACvtfcHk0lgQbdwwKgq1Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslOmsg%253D%253D&md5=42186e406fcac17c088252644e216e0f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.2165%2F00129785-200404060-00004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F00129785-200404060-00004%26sid%3Dliteratum%253Aachs%26aulast%3DBerger%26aufirst%3DA.%2BB.%26aulast%3DVitorino%26aufirst%3DP.%2BM.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520applications%2520to%2520biomarker%2520discovery%252C%2520in%2520vivo%2520imaging%2520and%2520drug%2520discovery%26jtitle%3DAm.%2520J.%2520PharmacoGenomics%26date%3D2004%26volume%3D4%26spage%3D371%26epage%3D381%26doi%3D10.2165%2F00129785-200404060-00004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Z.-C.</span></span> <span> </span><span class="NLM_article-title">Target identification with quantitative activity based protein profiling (ABPP)</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1600212</span>– <span class="NLM_lpage">4</span>, <span class="refDoi"> DOI: 10.1002/pmic.201600212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fpmic.201600212" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=1600212-4&author=X.+Chenauthor=Y.+K.+Wongauthor=J.+Wangauthor=J.+Zhangauthor=Y.-M.+Leeauthor=H.-M.+Shenauthor=Q.+Linauthor=Z.-C.+Hua&title=Target+identification+with+quantitative+activity+based+protein+profiling+%28ABPP%29&doi=10.1002%2Fpmic.201600212"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Fpmic.201600212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.201600212%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWong%26aufirst%3DY.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DY.-M.%26aulast%3DShen%26aufirst%3DH.-M.%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DHua%26aufirst%3DZ.-C.%26atitle%3DTarget%2520identification%2520with%2520quantitative%2520activity%2520based%2520protein%2520profiling%2520%2528ABPP%2529%26jtitle%3DProteomics%26date%3D2017%26volume%3D17%26spage%3D1600212%26epage%3D4%26doi%3D10.1002%2Fpmic.201600212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pichler, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysiak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breinbauer, R.</span></span> <span> </span><span class="NLM_article-title">Target identification of covalently binding drugs by activity-based protein profiling (ABPP)</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3291</span>– <span class="NLM_lpage">3303</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.03.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.bmc.2016.03.050" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=3291-3303&author=C.+M.+Pichlerauthor=J.+Krysiakauthor=R.+Breinbauer&title=Target+identification+of+covalently+binding+drugs+by+activity-based+protein+profiling+%28ABPP%29&doi=10.1016%2Fj.bmc.2016.03.050"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.03.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.03.050%26sid%3Dliteratum%253Aachs%26aulast%3DPichler%26aufirst%3DC.%2BM.%26aulast%3DKrysiak%26aufirst%3DJ.%26aulast%3DBreinbauer%26aufirst%3DR.%26atitle%3DTarget%2520identification%2520of%2520covalently%2520binding%2520drugs%2520by%2520activity-based%2520protein%2520profiling%2520%2528ABPP%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D3291%26epage%3D3303%26doi%3D10.1016%2Fj.bmc.2016.03.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based proteomics of enzyme superfamilies: serine hydrolases as a case study</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">11051</span>– <span class="NLM_lpage">11055</span>, <span class="refDoi"> DOI: 10.1074/jbc.R109.097600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1074%2Fjbc.R109.097600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=20147750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktFGlsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=11051-11055&author=G.+M.+Simonauthor=B.+F.+Cravatt&title=Activity-based+proteomics+of+enzyme+superfamilies%3A+serine+hydrolases+as+a+case+study&doi=10.1074%2Fjbc.R109.097600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based Proteomics of Enzyme Superfamilies: Serine Hydrolases as a Case Study</span></div><div class="casAuthors">Simon, Gabriel M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">11051-11055</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Genome sequencing projects have uncovered thousands of uncharacterized enzymes in eukaryotic and prokaryotic organisms.  Deciphering the physiol. functions of enzymes requires tools to profile and perturb their activities in native biol. systems.  Activity-based protein profiling has emerged as a powerful chemoproteomic strategy to achieve these objectives through the use of chem. probes that target large swaths of enzymes that share active-site features.  Here, we review activity-based protein profiling and its implementation to annotate the enzymic proteome, with particular attention given to probes that target serine hydrolases, a diverse superfamily of enzymes replete with many uncharacterized members.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp04j7PfUvT_LVg90H21EOLACvtfcHk0lgQbdwwKgq1Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktFGlsro%253D&md5=719e17677c5229574c9fc50006a16048</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R109.097600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R109.097600%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520proteomics%2520of%2520enzyme%2520superfamilies%253A%2520serine%2520hydrolases%2520as%2520a%2520case%2520study%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D11051%26epage%3D11055%26doi%3D10.1074%2Fjbc.R109.097600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: the serine hydrolases</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">14694</span>– <span class="NLM_lpage">14699</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.26.14694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.96.26.14694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10611275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=14694-14699&author=Y.+Liuauthor=M.+P.+Patricelliauthor=B.+F.+Cravatt&title=Activity-based+protein+profiling%3A+the+serine+hydrolases&doi=10.1073%2Fpnas.96.26.14694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: the serine hydrolases</span></div><div class="casAuthors">Liu, Yongsheng; Patricelli, Matthew P.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">14694-14699</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">With the postgenome era rapidly approaching, new strategies for the functional anal. of proteins are needed.  To date, proteomics efforts have primarily been confined to recording variations in protein level rather than activity.  The ability to profile classes of proteins on the basis of changes in their activity would greatly accelerate both the assignment of protein function and the identification of potential pharmaceutical targets.  Here, we describe the chem. synthesis and utility of an active-site directed probe for visualizing dynamics in the expression and function of an entire enzyme family, the serine hydrolases.  By reacting this probe, a biotinylated fluorophosphonate referred to as FP-biotin, with crude tissue exts., we quickly and with high sensitivity detect numerous serine hydrolases, many of which display tissue-restricted patterns of expression.  Addnl., we show that FP-biotin labels these proteins in an activity-dependent manner that can be followed kinetically, offering a powerful means to monitor dynamics simultaneously in both protein function and expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8i5m0pTTunrVg90H21EOLACvtfcHk0lgfdIFc350jYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D&md5=d5d1a09328fc7715e9d30ca392733426</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.26.14694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.26.14694%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520the%2520serine%2520hydrolases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D14694%26epage%3D14699%26doi%3D10.1073%2Fpnas.96.26.14694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giang, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamp, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbaum, J. J.</span></span> <span> </span><span class="NLM_article-title">Direct visualization of serine hydrolase activities in complex proteomes using fluorescent active site-directed probes</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1002/1615-9861(200109)1:9<1067::AID-PROT1067>3.0.CO;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=11990500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsFOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=1067-1071&author=M.+P.+Patricelliauthor=D.+K.+Giangauthor=L.+M.+Stampauthor=J.+J.+Burbaum&title=Direct+visualization+of+serine+hydrolase+activities+in+complex+proteomes+using+fluorescent+active+site-directed+probes&doi=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Direct visualization of serine hydrolase activities in complex proteomes using fluorescent active site-directed probes</span></div><div class="casAuthors">Patricelli, Matthew P.; Giang, Dan K.; Stamp, Lisa M.; Burbaum, Jonathan J.</div><div class="citationInfo"><span class="NLM_cas:title">Proteomics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1067-1071</span>CODEN:
                <span class="NLM_cas:coden">PROTC7</span>;
        ISSN:<span class="NLM_cas:issn">1615-9853</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The field of biochem. is currently faced with the enormous challenge of assigning functional significance to more than thirty thousand predicted protein products encoded by the human genome.  In order to accomplish this daunting task, methods will be required that facilitate the global anal. of proteins in complex biol. systems.  Recently, methods have been described for simultaneously monitoring the activity of multiple enzymes in crude proteomes based on their reactivity with tagged chem. probes.  These activity based probes (ABPs) have used either radiochem. or biotin/avidin-based detection methods to allow consolidated visualization of numerous enzyme activities.  Here we report the synthesis and evaluation of fluorescent activity based probes for the serine hydrolase super-family of enzymes.  The fluorescent methods detailed herein provide superior throughput, sensitivity, and quant. accuracy when compared to previously described ABPs, and provide a straight-forward platform for high-throughput proteome anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrunfp6WitoIbVg90H21EOLACvtfcHk0lgfdIFc350jYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsFOmtrs%253D&md5=5ed5063f140af14e24f1ab2444785eb4</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1615-9861%2528200109%25291%253A9%253C1067%253A%253AAID-PROT1067%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DGiang%26aufirst%3DD.%2BK.%26aulast%3DStamp%26aufirst%3DL.%2BM.%26aulast%3DBurbaum%26aufirst%3DJ.%2BJ.%26atitle%3DDirect%2520visualization%2520of%2520serine%2520hydrolase%2520activities%2520in%2520complex%2520proteomes%2520using%2520fluorescent%2520active%2520site-directed%2520probes%26jtitle%3DProteomics%26date%3D2001%26volume%3D1%26spage%3D1067%26epage%3D1071%26doi%3D10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Speers, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Profiling enzyme activities in vivo using click chemistry methods</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2004.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2004.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=15123248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1Cms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=535-546&author=A.+E.+Speersauthor=B.+F.+Cravatt&title=Profiling+enzyme+activities+in+vivo+using+click+chemistry+methods&doi=10.1016%2Fj.chembiol.2004.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling Enzyme Activities In Vivo Using Click Chemistry Methods</span></div><div class="casAuthors">Speers, Anna E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">535-546</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methods for profiling the activity of enzymes in vivo are needed to understand the role that these proteins and their endogenous regulators play in physiol. and pathol. processes.  Recently, we introduced a tag-free strategy for activity-based protein profiling (ABPP) that utilizes the copper(I)-catalyzed azide-alkyne cycloaddn. reaction ("click chem.") to analyze the functional state of enzymes in living cells and organisms.  Here, we report a detailed characterization of the reaction parameters that affect click chem.-based ABPP and identify conditions that maximize the speed, sensitivity, and bioorthogonality of this approach.  Using these optimized conditions, we compare the enzyme activity profiles of living and homogenized breast cancer cells, resulting in the identification of several enzymes that are labeled by activity-based probes in situ but not in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH0eSSbsaoDbVg90H21EOLACvtfcHk0lgfdIFc350jYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1Cms78%253D&md5=467d291cac7fa5cd105cc45248c26eb5</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DSpeers%26aufirst%3DA.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProfiling%2520enzyme%2520activities%2520in%2520vivo%2520using%2520click%2520chemistry%2520methods%26jtitle%3DChem.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D535%26epage%3D546%26doi%3D10.1016%2Fj.chembiol.2004.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mileni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beidler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinney, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadagopan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liimatta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazerwith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiff, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamtekar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Becelaere, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of a highly selective FAAH inhibitor that reduces inflammatory pain</span>. <i>Chem. Biol. (Oxford, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">411</span>– <span class="NLM_lpage">420</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2009.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2009.02.013" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=411-420&author=K.+Ahnauthor=D.+S.+Johnsonauthor=M.+Mileniauthor=D.+Beidlerauthor=J.+Z.+Longauthor=M.+K.+McKinneyauthor=E.+Weerapanaauthor=N.+Sadagopanauthor=M.+Liimattaauthor=S.+E.+Smithauthor=S.+Lazerwithauthor=C.+Stiffauthor=S.+Kamtekarauthor=K.+Bhattacharyaauthor=Y.+Zhangauthor=S.+Swaneyauthor=K.+Van+Becelaereauthor=R.+C.+Stevensauthor=B.+F.+Cravatt&title=Discovery+and+characterization+of+a+highly+selective+FAAH+inhibitor+that+reduces+inflammatory+pain&doi=10.1016%2Fj.chembiol.2009.02.013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DD.%2BS.%26aulast%3DMileni%26aufirst%3DM.%26aulast%3DBeidler%26aufirst%3DD.%26aulast%3DLong%26aufirst%3DJ.%2BZ.%26aulast%3DMcKinney%26aufirst%3DM.%2BK.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DSadagopan%26aufirst%3DN.%26aulast%3DLiimatta%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DS.%2BE.%26aulast%3DLazerwith%26aufirst%3DS.%26aulast%3DStiff%26aufirst%3DC.%26aulast%3DKamtekar%26aufirst%3DS.%26aulast%3DBhattacharya%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSwaney%26aufirst%3DS.%26aulast%3DVan%2BBecelaere%26aufirst%3DK.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520a%2520highly%2520selective%2520FAAH%2520inhibitor%2520that%2520reduces%2520inflammatory%2520pain%26jtitle%3DChem.%2520Biol.%2520%2528Oxford%252C%2520U.%2520K.%2529%26date%3D2009%26volume%3D16%26spage%3D411%26epage%3D420%26doi%3D10.1016%2Fj.chembiol.2009.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saraf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolasa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lannoye, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brioni, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surowy, C. S.</span></span> <span> </span><span class="NLM_article-title">Fatty acid amide hydrolase inhibitors display broad selectivity and inhibit multiple carboxylesterases as off-targets</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1095</span>– <span class="NLM_lpage">1105</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2006.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.neuropharm.2006.11.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2007&pages=1095-1105&author=D.+Zhangauthor=A.+Sarafauthor=T.+Kolasaauthor=P.+Bhatiaauthor=G.+Z.+Zhengauthor=M.+Patelauthor=G.+S.+Lannoyeauthor=P.+Richardsonauthor=A.+Stewartauthor=J.+C.+Rogersauthor=J.+D.+Brioniauthor=C.+S.+Surowy&title=Fatty+acid+amide+hydrolase+inhibitors+display+broad+selectivity+and+inhibit+multiple+carboxylesterases+as+off-targets&doi=10.1016%2Fj.neuropharm.2006.11.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2006.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2006.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DSaraf%26aufirst%3DA.%26aulast%3DKolasa%26aufirst%3DT.%26aulast%3DBhatia%26aufirst%3DP.%26aulast%3DZheng%26aufirst%3DG.%2BZ.%26aulast%3DPatel%26aufirst%3DM.%26aulast%3DLannoye%26aufirst%3DG.%2BS.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DStewart%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DJ.%2BC.%26aulast%3DBrioni%26aufirst%3DJ.%2BD.%26aulast%3DSurowy%26aufirst%3DC.%2BS.%26atitle%3DFatty%2520acid%2520amide%2520hydrolase%2520inhibitors%2520display%2520broad%2520selectivity%2520and%2520inhibit%2520multiple%2520carboxylesterases%2520as%2520off-targets%26jtitle%3DNeuropharmacology%26date%3D2007%26volume%3D52%26spage%3D1095%26epage%3D1105%26doi%3D10.1016%2Fj.neuropharm.2006.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cognetta, A. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shpak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Kroeg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantae, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baggelaar, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vrij, F. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fezza, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Wel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soethoudt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mock, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Dulk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baak, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florea, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendriks, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Petrocellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overkleeft, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hankemeier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Zeeuw, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Marzo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maccarrone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kushner, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling reveals off-target proteins of the FAAH inhibitor BIA 10–2474</span>. <i>Science (Washington, DC, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>356</i></span>,  <span class="NLM_fpage">1084</span>– <span class="NLM_lpage">1087</span>, <span class="refDoi"> DOI: 10.1126/science.aaf7497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.aaf7497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28596366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt12jsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2017&pages=1084-1087&author=A.+C.+M.+van+Esbroeckauthor=A.+P.+A.+Janssenauthor=A.+B.+Cognettaauthor=D.+Ogasawaraauthor=G.+Shpakauthor=M.+van+der%0AKroegauthor=V.+Kantaeauthor=M.+P.+Baggelaarauthor=F.+M.+S.+de+Vrijauthor=H.+Dengauthor=M.+Allaraauthor=F.+Fezzaauthor=Z.+Linauthor=T.+van+der+Welauthor=M.+Soethoudtauthor=E.+D.+Mockauthor=H.+den+Dulkauthor=I.+L.+Baakauthor=B.+I.+Floreaauthor=G.+Hendriksauthor=L.+De+Petrocellisauthor=H.+S.+Overkleeftauthor=T.+Hankemeierauthor=C.+I.+De+Zeeuwauthor=V.+Di+Marzoauthor=M.+Maccarroneauthor=B.+F.+Cravattauthor=S.+A.+Kushnerauthor=M.+van+der%0AStelt&title=Activity-based+protein+profiling+reveals+off-target+proteins+of+the+FAAH+inhibitor+BIA+10%E2%80%932474&doi=10.1126%2Fscience.aaf7497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling reveals off-target proteins of the FAAH inhibitor BIA 10-2474</span></div><div class="casAuthors">van Esbroeck, Annelot C. M.; Janssen, Antonius P. A.; Cognetta, Armand B., III; Ogasawara, Daisuke; Shpak, Guy; van der Kroeg, Mark; Kantae, Vasudev; Baggelaar, Marc P.; de Vrij, Femke M. S.; Deng, Hui; Allara, Marco; Fezza, Filomena; Lin, Zhanmin; van der Wel, Tom; Soethoudt, Marjolein; Mock, Elliot D.; den Dulk, Hans; Baak, Ilse L.; Florea, Bogdan I.; Hendriks, Giel; De Petrocellis, Luciano; Overkleeft, Herman S.; Hankemeier, Thomas; De Zeeuw, Chris I.; Di Marzo, Vincenzo; Maccarrone, Mauro; Cravatt, Benjamin F.; Kushner, Steven A.; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">356</span>
        (<span class="NLM_cas:issue">6342</span>),
    <span class="NLM_cas:pages">1084-1087</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A recent phase 1 trial of the fatty acid amide hydrolase (FAAH) inhibitor BIA 10-2474 led to the death of one volunteer and produced mild-to-severe neurol. symptoms in four others.  Although the cause of the clin. neurotoxicity is unknown, it has been postulated, given the clin. safety profile of other tested FAAH inhibitors, that off-target activities of BIA 10-2474 may have played a role.  Here the authors use activity-based proteomic methods to det. the protein interaction landscape of BIA 10-2474 in human cells and tissues.  This anal. revealed that the drug inhibits several lipases that are not targeted by PF04457845, a highly selective and clin. tested FAAH inhibitor.  BIA 10-2474, but not PF04457845, produced substantial alterations in lipid networks in human cortical neurons, suggesting that promiscuous lipase inhibitors have the potential to cause metabolic dysregulation in the nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_cUEzfW-Nd7Vg90H21EOLACvtfcHk0lje28q7k0WSmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt12jsb0%253D&md5=6133b21e104942f7c6818ba1de0d73f4</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf7497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf7497%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3DCognetta%26aufirst%3DA.%2BB.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DShpak%26aufirst%3DG.%26aulast%3Dvan%2Bder%2BKroeg%26aufirst%3DM.%26aulast%3DKantae%26aufirst%3DV.%26aulast%3DBaggelaar%26aufirst%3DM.%2BP.%26aulast%3Dde%2BVrij%26aufirst%3DF.%2BM.%2BS.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DAllara%26aufirst%3DM.%26aulast%3DFezza%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3Dvan%2Bder%2BWel%26aufirst%3DT.%26aulast%3DSoethoudt%26aufirst%3DM.%26aulast%3DMock%26aufirst%3DE.%2BD.%26aulast%3Dden%2BDulk%26aufirst%3DH.%26aulast%3DBaak%26aufirst%3DI.%2BL.%26aulast%3DFlorea%26aufirst%3DB.%2BI.%26aulast%3DHendriks%26aufirst%3DG.%26aulast%3DDe%2BPetrocellis%26aufirst%3DL.%26aulast%3DOverkleeft%26aufirst%3DH.%2BS.%26aulast%3DHankemeier%26aufirst%3DT.%26aulast%3DDe%2BZeeuw%26aufirst%3DC.%2BI.%26aulast%3DDi%2BMarzo%26aufirst%3DV.%26aulast%3DMaccarrone%26aufirst%3DM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DKushner%26aufirst%3DS.%2BA.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DActivity-based%2520protein%2520profiling%2520reveals%2520off-target%2520proteins%2520of%2520the%2520FAAH%2520inhibitor%2520BIA%252010%25E2%2580%25932474%26jtitle%3DScience%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2017%26volume%3D356%26spage%3D1084%26epage%3D1087%26doi%3D10.1126%2Fscience.aaf7497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Disparate proteome reactivity profiles of carbon electrophiles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1038/nchembio.91</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.91" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=18488014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=405-407&author=E.+Weerapanaauthor=G.+M.+Simonauthor=B.+F.+Cravatt&title=Disparate+proteome+reactivity+profiles+of+carbon+electrophiles&doi=10.1038%2Fnchembio.91"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Disparate proteome reactivity profiles of carbon electrophiles</span></div><div class="casAuthors">Weerapana, Eranthie; Simon, Gabriel M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">405-407</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Insights into the proteome reactivity of electrophiles are crucial for designing activity-based probes for enzymes lacking cognate affinity labels.  Here, the authors show that different classes of carbon electrophiles exhibit markedly distinct amino acid labeling profiles in proteomes, ranging from selective reactivity with cysteine to adducts with several amino acids.  These data specify electrophilic chemotypes with restricted and permissive reactivity profiles to guide the design of next-generation functional proteomics probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI4Jh6351SYLVg90H21EOLACvtfcHk0lgtOy3Xi4H11w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ejtL8%253D&md5=69eb0ae589c50f46a8695dc6731366e2</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.91&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.91%26sid%3Dliteratum%253Aachs%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DDisparate%2520proteome%2520reactivity%2520profiles%2520of%2520carbon%2520electrophiles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D405%26epage%3D407%26doi%3D10.1038%2Fnchembio.91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hacker, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazear, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Global profiling of lysine reactivity and ligandability in the human proteome</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1038/nchem.2826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchem.2826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=29168484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1181-1190&author=S.+M.+Hackerauthor=K.+M.+Backusauthor=M.+R.+Lazearauthor=S.+Forliauthor=B.+E.+Correiaauthor=B.+F.+Cravatt&title=Global+profiling+of+lysine+reactivity+and+ligandability+in+the+human+proteome&doi=10.1038%2Fnchem.2826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Global profiling of lysine reactivity and ligandability in the human proteome</span></div><div class="casAuthors">Hacker, Stephan M.; Backus, Keriann M.; Lazear, Michael R.; Forli, Stefano; Correia, Bruno E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nucleophilic amino acids make important contributions to protein function, including performing key roles in catalysis and serving as sites for post-translational modification.  Electrophilic groups that target amino-acid nucleophiles have been used to create covalent ligands and drugs, but have, so far, been mainly limited to cysteine and serine.  Here, the authors report a chem. proteomic platform for the global and quant. anal. of lysine residues in native biol. systems.  The authors have quantified, in total, >9000 lysines in human cell proteomes and have identified several hundred residues with heightened reactivity that are enriched at protein functional sites and can frequently be targeted by electrophilic small mols.  The authors also discovered lysine-reactive fragment electrophiles that inhibit enzymes by active site and allosteric mechanisms, as well as disrupt protein-protein interactions in transcriptional regulatory complexes, emphasizing the broad potential and diverse functional consequences of liganding lysine residues throughout the human proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRKvUVcT5YRrVg90H21EOLACvtfcHk0lgtOy3Xi4H11w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN&md5=1dac8f079c1caffc8ae584291480fc61</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2826%26sid%3Dliteratum%253Aachs%26aulast%3DHacker%26aufirst%3DS.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DLazear%26aufirst%3DM.%2BR.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DGlobal%2520profiling%2520of%2520lysine%2520reactivity%2520and%2520ligandability%2520in%2520the%2520human%2520proteome%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1181%26epage%3D1190%26doi%3D10.1038%2Fnchem.2826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seo, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, K. S.</span></span> <span> </span><span class="NLM_article-title">Profiling protein thiol oxidation in tumor cells using sulfenic acid-specific antibodies</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">16163</span>– <span class="NLM_lpage">16168</span>, <span class="refDoi"> DOI: 10.1073/pnas.0903015106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.0903015106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=19805274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OjsbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=16163-16168&author=Y.+H.+Seoauthor=K.+S.+Carroll&title=Profiling+protein+thiol+oxidation+in+tumor+cells+using+sulfenic+acid-specific+antibodies&doi=10.1073%2Fpnas.0903015106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling protein thiol oxidation in tumor cells using sulfenic acid-specific antibodies</span></div><div class="casAuthors">Seo, Young Ho; Carroll, Kate S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">16163-16168, S16163/1-S16163/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hydrogen peroxide (H2O2) functions as a second messenger that can activate cell proliferation through chemoselective oxidn. of cysteine residues in signaling proteins.  The connection between H2O2 signaling, thiol oxidn., and activation of growth pathways has emerged as fertile ground for the development of strategies for cancer treatment.  Central to achieving this goal is the development of tools and assays that facilitate characterization of the mol. events assocd. with tumorigenesis and evaluation of patient response to therapy.  Here we report on the development of an immunochem. method for detecting sulfenic acid, the initial oxidn. product that results when a thiolate reacts with H2O2.  For this approach, the sulfenic acid is derivatized with a chem. tag to generate a unique epitope for recognition.  The elicited antibody is exquisitely specific, context-independent, and capable of visualizing sulfenic acid formation in cells.  Applying this approach to several systems, including cancer cell lines, shows it can be used to monitor differences in thiol redox status and reveals a diverse pattern of sulfenic acid modifications across different subtypes of breast tumors.  These studies demonstrate a general strategy for producing antibodies against a specific oxidn. state of cysteine and show the utility of these reagents for profiling thiol oxidn. assocd. with pathol. conditions such as breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4-72mmCOXrrVg90H21EOLACvtfcHk0ljgt7J3NycLqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OjsbbE&md5=126a4c8a21227fb8ca7c74c86c9d28ba</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0903015106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0903015106%26sid%3Dliteratum%253Aachs%26aulast%3DSeo%26aufirst%3DY.%2BH.%26aulast%3DCarroll%26aufirst%3DK.%2BS.%26atitle%3DProfiling%2520protein%2520thiol%2520oxidation%2520in%2520tumor%2520cells%2520using%2520sulfenic%2520acid-specific%2520antibodies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D16163%26epage%3D16168%26doi%3D10.1073%2Fpnas.0903015106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piggott, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karuso, P.</span></span> <span> </span><span class="NLM_article-title">Identifying the cellular targets of natural products using T7 phage display</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">626</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1039/C5NP00128E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2FC5NP00128E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26964751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFegsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=626-636&author=A.+M.+Piggottauthor=P.+Karuso&title=Identifying+the+cellular+targets+of+natural+products+using+T7+phage+display&doi=10.1039%2FC5NP00128E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying the cellular targets of natural products using T7 phage display</span></div><div class="casAuthors">Piggott, Andrew M.; Karuso, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">626-636</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covering: up to the end of 2015While Nature continues to deliver a myriad of potent and structurally diverse biol. active small mols., the cellular targets and modes of action of these natural products are rarely identified, significantly hindering their development as new chemotherapeutic agents.  This article provides an introductory tutorial on the use of T7 phage display as a tool to rapidly identify the cellular targets of natural products and is aimed specifically at natural products chemists who may have only limited experience in mol. biol.  A brief overview of T7 phage display is provided, including its strengths, weaknesses, and the type of problems that can and cannot be tackled with this technol.  Affinity probe construction is reviewed, including linker design and natural product derivatisation strategies.  A detailed description of the T7 phage biopanning procedure is provided, with valuable tips for optimizing each step in the process, as well as advice for identifying and avoiding the most commonly encountered challenges and pitfalls along the way.  Finally, a brief discussion is provided on techniques for validating the cellular targets identified using T7 phage display.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW7KABYJ48i7Vg90H21EOLACvtfcHk0ljgt7J3NycLqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFegsLo%253D&md5=a070c3c88527b50b2591798731751b57</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1039%2FC5NP00128E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5NP00128E%26sid%3Dliteratum%253Aachs%26aulast%3DPiggott%26aufirst%3DA.%2BM.%26aulast%3DKaruso%26aufirst%3DP.%26atitle%3DIdentifying%2520the%2520cellular%2520targets%2520of%2520natural%2520products%2520using%2520T7%2520phage%2520display%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2016%26volume%3D33%26spage%3D626%26epage%3D636%26doi%3D10.1039%2FC5NP00128E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takakusagi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takakusagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, K.</span></span> <span> </span><span class="NLM_article-title">Use of phage display technology for the determination of the targets for small-molecule therapeutics</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1517/17460441003653155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1517%2F17460441003653155" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=361-389&author=Y.+Takakusagiauthor=K.+Takakusagiauthor=F.+Sugawaraauthor=K.+Sakaguchi&title=Use+of+phage+display+technology+for+the+determination+of+the+targets+for+small-molecule+therapeutics&doi=10.1517%2F17460441003653155"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1517%2F17460441003653155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441003653155%26sid%3Dliteratum%253Aachs%26aulast%3DTakakusagi%26aufirst%3DY.%26aulast%3DTakakusagi%26aufirst%3DK.%26aulast%3DSugawara%26aufirst%3DF.%26aulast%3DSakaguchi%26aufirst%3DK.%26atitle%3DUse%2520of%2520phage%2520display%2520technology%2520for%2520the%2520determination%2520of%2520the%2520targets%2520for%2520small-molecule%2520therapeutics%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D361%26epage%3D389%26doi%3D10.1517%2F17460441003653155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Licitra, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. O.</span></span> <span> </span><span class="NLM_article-title">A three-hybrid system for detecting small ligand-protein receptor interactions</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">12817</span>– <span class="NLM_lpage">12821</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.23.12817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.93.23.12817" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=12817-12821&author=E.+J.+Licitraauthor=J.+O.+Liu&title=A+three-hybrid+system+for+detecting+small+ligand-protein+receptor+interactions&doi=10.1073%2Fpnas.93.23.12817"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.23.12817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.23.12817%26sid%3Dliteratum%253Aachs%26aulast%3DLicitra%26aufirst%3DE.%2BJ.%26aulast%3DLiu%26aufirst%3DJ.%2BO.%26atitle%3DA%2520three-hybrid%2520system%2520for%2520detecting%2520small%2520ligand-protein%2520receptor%2520interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D12817%26epage%3D12821%26doi%3D10.1073%2Fpnas.93.23.12817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rezwan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auerbach, D.</span></span> <span> </span><span class="NLM_article-title">Yeast ″N″-hybrid systems for protein-protein and drug-protein interaction discovery</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2012.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.ymeth.2012.06.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=423-429&author=M.+Rezwanauthor=D.+Auerbach&title=Yeast+%E2%80%B3N%E2%80%B3-hybrid+systems+for+protein-protein+and+drug-protein+interaction+discovery&doi=10.1016%2Fj.ymeth.2012.06.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2012.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2012.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DRezwan%26aufirst%3DM.%26aulast%3DAuerbach%26aufirst%3DD.%26atitle%3DYeast%2520%25E2%2580%25B3N%25E2%2580%25B3-hybrid%2520systems%2520for%2520protein-protein%2520and%2520drug-protein%2520interaction%2520discovery%26jtitle%3DMethods%26date%3D2012%26volume%3D57%26spage%3D423%26epage%3D429%26doi%3D10.1016%2Fj.ymeth.2012.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santoni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabilloud, T.</span></span> <span> </span><span class="NLM_article-title">Membrane proteins and proteomics: un amour impossible?</span>. <i>Electrophoresis</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1054</span>– <span class="NLM_lpage">1070</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1522-2683(20000401)21:6<1054::AID-ELPS1054>3.0.CO;2-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1054%3A%3AAID-ELPS1054%3E3.0.CO%3B2-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10786880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3cXis12kurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=1054-1070&author=V.+Santoniauthor=M.+Molloyauthor=T.+Rabilloud&title=Membrane+proteins+and+proteomics%3A+un+amour+impossible%3F&doi=10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1054%3A%3AAID-ELPS1054%3E3.0.CO%3B2-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane proteins and proteomics: un amour impossible?</span></div><div class="casAuthors">Santoni, Veronique; Molloy, Mark; Rabilloud, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Electrophoresis</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1054-1070</span>CODEN:
                <span class="NLM_cas:coden">ELCTDN</span>;
        ISSN:<span class="NLM_cas:issn">0173-0835</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">A review with 136 refs.  Proteome anal. implies the ability to sep. proteins as a first step prior to characterization.  Thus, the overall performance of the anal. strongly depends on the performance of the sepn. tool, usually two-dimensional electrophoresis.  This review shows how two-dimensional electrophoresis performs with membrane proteins from bacteria or animal or vegetable cells and tissues, the recent progress in this field, and it examines future prospects in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbh4b0Ke4VKbVg90H21EOLACvtfcHk0lgyPd2qi4X5wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXis12kurk%253D&md5=43a7c9679485adfbb68975dd126c982b</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1054%3A%3AAID-ELPS1054%3E3.0.CO%3B2-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291522-2683%252820000401%252921%253A6%253C1054%253A%253AAID-ELPS1054%253E3.0.CO%253B2-8%26sid%3Dliteratum%253Aachs%26aulast%3DSantoni%26aufirst%3DV.%26aulast%3DMolloy%26aufirst%3DM.%26aulast%3DRabilloud%26aufirst%3DT.%26atitle%3DMembrane%2520proteins%2520and%2520proteomics%253A%2520un%2520amour%2520impossible%253F%26jtitle%3DElectrophoresis%26date%3D2000%26volume%3D21%26spage%3D1054%26epage%3D1070%26doi%3D10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1054%3A%3AAID-ELPS1054%3E3.0.CO%3B2-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stockmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stairs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ireland-Zecchini, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindle, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeper, F. J.</span></span> <span> </span><span class="NLM_article-title">Development and evaluation of new cyclooctynes for cell surface glycan imaging in cancer cells</span>. <i>Chem Sci</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">932</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1039/c0sc00631a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2Fc0sc00631a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=22724056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC2srmslaqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=932-936&author=H.+Stockmannauthor=A.+A.+Nevesauthor=S.+Stairsauthor=H.+Ireland-Zecchiniauthor=K.+M.+Brindleauthor=F.+J.+Leeper&title=Development+and+evaluation+of+new+cyclooctynes+for+cell+surface+glycan+imaging+in+cancer+cells&doi=10.1039%2Fc0sc00631a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of new cyclooctynes for cell surface glycan imaging in cancer cells</span></div><div class="casAuthors">Stockmann Henning; Neves Andre A; Stairs Shaun; Ireland-Zecchini Heather; Brindle Kevin M; Leeper Finian J</div><div class="citationInfo"><span class="NLM_cas:title">Chemical science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">932-936</span>
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    </div><div class="casAbstract">Two reagents have been synthesized for selective labeling of cell surface azidoglycans, an unusually stable version of a dibenzocyclooctyne (TMDIBO) and a third-generation difluorinated cyclooctyne (DIFO3).  Both syntheses are efficient with minimal purification, and the dibenzocyclooctyne is stable under basic and acidic conditions.  Flow cytometric measurements with azidosugar labeled cancer cells, in which these reagents were linked to the fluorophore Alexa Fluor 647, gave a signal-to-background ratio of up to 35 with TMDIBO as compared to ≈10 for DIFO3 and ≈5 for a phosphine reagent.  TMDIBO-based probes should have applications in molecular imaging of cell surface glycans in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzfo0elEmHyUEksK8CslK8fW6udTcc2eaofup-NzA5N7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmslaqug%253D%253D&md5=f533b9dd572ce4182541603316f13661</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1039%2Fc0sc00631a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0sc00631a%26sid%3Dliteratum%253Aachs%26aulast%3DStockmann%26aufirst%3DH.%26aulast%3DNeves%26aufirst%3DA.%2BA.%26aulast%3DStairs%26aufirst%3DS.%26aulast%3DIreland-Zecchini%26aufirst%3DH.%26aulast%3DBrindle%26aufirst%3DK.%2BM.%26aulast%3DLeeper%26aufirst%3DF.%2BJ.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%2520new%2520cyclooctynes%2520for%2520cell%2520surface%2520glycan%2520imaging%2520in%2520cancer%2520cells%26jtitle%3DChem%2520Sci%26date%3D2011%26volume%3D2%26spage%3D932%26epage%3D936%26doi%3D10.1039%2Fc0sc00631a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neves, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmston, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pryor, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ireland-Zecchini, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeper, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindle, K. M.</span></span> <span> </span><span class="NLM_article-title">Imaging sialylated tumor cell glycans in vivo</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2528</span>– <span class="NLM_lpage">2537</span>, <span class="refDoi"> DOI: 10.1096/fj.10-178590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1096%2Ffj.10-178590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21493886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFersr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=2528-2537&author=A.+A.+Nevesauthor=H.+Stockmannauthor=R.+R.+Harmstonauthor=H.+J.+Pryorauthor=I.+S.+Alamauthor=H.+Ireland-Zecchiniauthor=D.+Y.+Lewisauthor=S.+K.+Lyonsauthor=F.+J.+Leeperauthor=K.+M.+Brindle&title=Imaging+sialylated+tumor+cell+glycans+in+vivo&doi=10.1096%2Ffj.10-178590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Imaging sialylated tumor cell glycans in vivo</span></div><div class="casAuthors">Neves, Andre A.; Stockmann, Henning; Harmston, Rebecca R.; Pryor, Helen J.; Alam, Israt S.; Ireland-Zecchini, Heather; Lewis, David Y.; Lyons, Scott K.; Leeper, Finian J.; Brindle, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2528-2537, 10.1096/fj.10-178590</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Cell surface glycans are involved in numerous physiol. processes that involve cell-cell interactions and migration, including lymphocyte trafficking and cancer metastasis.  We have used a bioorthogonal metabolic labeling strategy to detect cell surface glycans and demonstrate, for the first time, fluorescence and radionuclide imaging of sialylated glycans in a murine tumor model in vivo.  Peracetylated azido-labeled N-acetyl-mannosamine, injected i.p., was used as the metabolic precursor for the biosynthesis of 5-azidoneuraminic, or azidosialic acid.  Azidosialic acid-labeled cell surface glycans were then reacted, by Staudinger ligation, with a biotinylated phosphine injected i.p., and the biotin was detected by subsequent i.v. injection of a fluorescent or radiolabeled avidin deriv.  At 24 h after administration of NeutrAvidin, labeled with either a far-red fluorophore or 111In, there was a significant azido-labeled N-acetyl-mannosamine-dependent increase in tumor-to-tissue contrast, which was detected using optical imaging or single-photon-emission computed tomog. (SPECT), resp.  The technique has the potential to translate to the clinic, where, given the prognostic relevance of altered sialic acid expression in cancer, it could be used to monitor disease progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBcmMsnI9T9rVg90H21EOLACvtfcHk0lgyPd2qi4X5wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFersr0%253D&md5=93a49206ecec034ed100e40840c69edc</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1096%2Ffj.10-178590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.10-178590%26sid%3Dliteratum%253Aachs%26aulast%3DNeves%26aufirst%3DA.%2BA.%26aulast%3DStockmann%26aufirst%3DH.%26aulast%3DHarmston%26aufirst%3DR.%2BR.%26aulast%3DPryor%26aufirst%3DH.%2BJ.%26aulast%3DAlam%26aufirst%3DI.%2BS.%26aulast%3DIreland-Zecchini%26aufirst%3DH.%26aulast%3DLewis%26aufirst%3DD.%2BY.%26aulast%3DLyons%26aufirst%3DS.%2BK.%26aulast%3DLeeper%26aufirst%3DF.%2BJ.%26aulast%3DBrindle%26aufirst%3DK.%2BM.%26atitle%3DImaging%2520sialylated%2520tumor%2520cell%2520glycans%2520in%2520vivo%26jtitle%3DFASEB%2520J.%26date%3D2011%26volume%3D25%26spage%3D2528%26epage%3D2537%26doi%3D10.1096%2Ffj.10-178590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stockmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marin, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balut, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, A.</span></span> <span> </span><span class="NLM_article-title">Glycan-mediated, ligand-controlled click chemistry for drug-target identification</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1002/cbic.201500590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fcbic.201500590" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=150-154&author=H.+Stockmannauthor=V.+L.+Marinauthor=P.+Nimmerauthor=C.+M.+Balutauthor=D.+J.+Davidsonauthor=P.+L.+Richardsonauthor=A.+Vasudevan&title=Glycan-mediated%2C+ligand-controlled+click+chemistry+for+drug-target+identification&doi=10.1002%2Fcbic.201500590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201500590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201500590%26sid%3Dliteratum%253Aachs%26aulast%3DStockmann%26aufirst%3DH.%26aulast%3DMarin%26aufirst%3DV.%2BL.%26aulast%3DNimmer%26aufirst%3DP.%26aulast%3DBalut%26aufirst%3DC.%2BM.%26aulast%3DDavidson%26aufirst%3DD.%2BJ.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DVasudevan%26aufirst%3DA.%26atitle%3DGlycan-mediated%252C%2520ligand-controlled%2520click%2520chemistry%2520for%2520drug-target%2520identification%26jtitle%3DChemBioChem%26date%3D2016%26volume%3D17%26spage%3D150%26epage%3D154%26doi%3D10.1002%2Fcbic.201500590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Traceless affinity labeling of endogenous proteins for functional analysis in living cells</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1469</span>, <span class="refDoi"> DOI: 10.1021/ar200334r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar200334r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1Oltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=1460-1469&author=T.+Hayashiauthor=I.+Hamachi&title=Traceless+affinity+labeling+of+endogenous+proteins+for+functional+analysis+in+living+cells&doi=10.1021%2Far200334r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Traceless Affinity Labeling of Endogenous Proteins for Functional Analysis in Living Cells</span></div><div class="casAuthors">Hayashi, Takahiro; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1460-1469</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein labeling and imaging techniques have provided tremendous opportunities to study the structure, function, dynamics, and localization of individual proteins in the complex environment of living cells.  Mol. biol.-based approaches, such as GFP-fusion tags and monoclonal antibodies, have served as important tools for the visualization of individual proteins in cells.  Although these techniques continue to be valuable for live cell imaging, they have a no. of limitations that have only been addressed by recent progress in chem.-based approaches.  These chem. approaches benefit greatly from the smaller probe sizes that should result in fewer perturbations to proteins and to biol. systems as a whole.  Despite the research in this area, so far none of these labeling techniques permit labeling and imaging of selected endogenous proteins in living cells.  Researchers have widely used affinity labeling, in which the protein of interest is labeled by a reactive group attached to a ligand, to identify and characterize proteins.  Since the first report of affinity labeling in the early 1960s, efforts to fine-tune the chem. structures of both the reactive group and ligand have led to protein labeling with excellent target selectivity in the whole proteome of living cells.  Although the chem. probes used for affinity labeling generally inactivate target proteins, this strategy holds promise as a valuable tool for the labeling and imaging of endogenous proteins in living cells and by extension in living animals.  In this Account, we summarize traceless affinity labeling, a technique explored mainly in our lab.  In our overview of the different labeling techniques, we emphasize the challenge of designing chem. probes that allow for dissocn. of the affinity module (often a ligand) after the labeling reaction so that the labeled protein retains its native function.  This feature distinguishes the traceless labeling approach from the traditional affinity labeling method and allows for real-time monitoring of protein activity.  With the high target specificity and biocompatibility of this technique, we have achieved individual labeling and imaging of endogenously expressed proteins in samples of high biol. complexity.  We also highlight applications in which our current approach enabled the monitoring of important biol. events, such as ligand binding, in living cells.  These novel chem. labeling techniques are expected to provide a mol. toolbox for studying a wide variety of proteins and beyond in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSs46czrmjCLVg90H21EOLACvtfcHk0ljJEAB2jIbLdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot1Oltb4%253D&md5=0a02bcf8fdb9df79abc93021cab7f99c</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Far200334r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far200334r%26sid%3Dliteratum%253Aachs%26aulast%3DHayashi%26aufirst%3DT.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DTraceless%2520affinity%2520labeling%2520of%2520endogenous%2520proteins%2520for%2520functional%2520analysis%2520in%2520living%2520cells%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2012%26volume%3D45%26spage%3D1460%26epage%3D1469%26doi%3D10.1021%2Far200334r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsukiji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Ligand-directed tosyl chemistry for protein labeling in vivo</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">341</span>– <span class="NLM_lpage">343</span>, <span class="refDoi"> DOI: 10.1038/nchembio.157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=19330012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsl2gurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=341-343&author=S.+Tsukijiauthor=M.+Miyagawaauthor=Y.+Takaokaauthor=T.+Tamuraauthor=I.+Hamachi&title=Ligand-directed+tosyl+chemistry+for+protein+labeling+in+vivo&doi=10.1038%2Fnchembio.157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-directed tosyl chemistry for protein labeling in vivo</span></div><div class="casAuthors">Tsukiji, Shinya; Miyagawa, Masayoshi; Takaoka, Yousuke; Tamura, Tomonori; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">341-343</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here the authors describe a method for the site-selective attachment of synthetic mols. into specific 'endogenous' proteins in vivo using ligand-directed tosyl (LDT) chem.  This approach was applied not only for chem. labeling proteins in living cells, tissues and mice but also for constructing a biosensor directly inside cells without genetic engineering.  These data establish LDT chem. as a new tool for the study and manipulation of biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0kFQSeuTHX7Vg90H21EOLACvtfcHk0ljJEAB2jIbLdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsl2gurY%253D&md5=604943938355591f1f4f5e861c6a8278</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.157%26sid%3Dliteratum%253Aachs%26aulast%3DTsukiji%26aufirst%3DS.%26aulast%3DMiyagawa%26aufirst%3DM.%26aulast%3DTakaoka%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DLigand-directed%2520tosyl%2520chemistry%2520for%2520protein%2520labeling%2520in%2520vivo%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D341%26epage%3D343%26doi%3D10.1038%2Fnchembio.157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baek, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, I.</span></span> <span> </span><span class="NLM_article-title">A small molecule that binds to an ATPase domain of Hsc70 promotes membrane trafficking of mutant cystic fibrosis transmembrane conductance regulator</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">20267</span>– <span class="NLM_lpage">20276</span>, <span class="refDoi"> DOI: 10.1021/ja206762p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja206762p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGgsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=20267-20276&author=H.+J.+Choauthor=H.+Y.+Geeauthor=K.+H.+Baekauthor=S.+K.+Koauthor=J.+M.+Parkauthor=H.+Leeauthor=N.+D.+Kimauthor=M.+G.+Leeauthor=I.+Shin&title=A+small+molecule+that+binds+to+an+ATPase+domain+of+Hsc70+promotes+membrane+trafficking+of+mutant+cystic+fibrosis+transmembrane+conductance+regulator&doi=10.1021%2Fja206762p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule That Binds to an ATPase Domain of Hsc70 Promotes Membrane Trafficking of Mutant Cystic Fibrosis Transmembrane Conductance Regulator</span></div><div class="casAuthors">Cho, Hyungseoph J.; Gee, Heon Yung; Baek, Kyung-Hwa; Ko, Sung-Kyun; Park, Jong-Moon; Lee, Hookeun; Kim, Nam-Doo; Lee, Min Goo; Shin, Injae</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">20267-20276</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cystic fibrosis transmembrane conductance regulator (CFTR) is a cell-surface anion channel that permeates chloride and bicarbonate ions.  The most frequent mutation of CFTR that causes cystic fibrosis is the deletion of phenylalanine at position 508 (ΔF508), which leads to defects in protein folding and cellular trafficking to the plasma membrane.  The lack of the cell-surface CFTR results in a redn. in the lifespan due to chronic lung infection with progressive deterioration of lung function.  Hsc70 plays a crucial role in degrdn. of mutant CFTR by the ubiquitin-proteasome system.  To date, various Hsc70 inhibitors and transcription regulators have been tested to det. whether they correct the defective activity of mutant CFTR.  However, they exhibited limited or questionable effects on restoring the chloride channel activity in cystic fibrosis cells.  Herein, the authors show that a small mol. apoptozole (Az) has high cellular potency to promote membrane trafficking of mutant CFTR and its chloride channel activity in cystic fibrosis cells.  Results from affinity chromatog. and ATPase activity assay indicate that Az inhibits the ATPase activity of Hsc70 by binding to its ATPase domain.  In addn., a ligand-directed protein labeling and mol. modeling studies also suggest the binding of Az to an ATPase domain, in particular, an ATP-binding pocket.  It is proposed that Az suppresses ubiquitination of ΔF508-CFTR maybe by blocking interaction of the mutant with Hsc70 and CHIP, and, as a consequence, it enhances membrane trafficking of the mutant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhNslz_-hHxbVg90H21EOLACvtfcHk0lhpi0aiOzGkbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGgsb3P&md5=574564da07a3f0c14e09cefac46408db</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fja206762p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja206762p%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%2BJ.%26aulast%3DGee%26aufirst%3DH.%2BY.%26aulast%3DBaek%26aufirst%3DK.%2BH.%26aulast%3DKo%26aufirst%3DS.%2BK.%26aulast%3DPark%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DShin%26aufirst%3DI.%26atitle%3DA%2520small%2520molecule%2520that%2520binds%2520to%2520an%2520ATPase%2520domain%2520of%2520Hsc70%2520promotes%2520membrane%2520trafficking%2520of%2520mutant%2520cystic%2520fibrosis%2520transmembrane%2520conductance%2520regulator%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D20267%26epage%3D20276%26doi%3D10.1021%2Fja206762p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkanda, J.</span></span> <span> </span><span class="NLM_article-title">Phosphopeptide-dependent labeling of 14–3-3 zeta proteins by fusicoccin-based fluorescent probes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">512</span>, <span class="refDoi"> DOI: 10.1002/anie.201106995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201106995" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=509-512&author=M.+Takahashiauthor=A.+Kawamuraauthor=N.+Katoauthor=T.+Nishiauthor=I.+Hamachiauthor=J.+Ohkanda&title=Phosphopeptide-dependent+labeling+of+14%E2%80%933-3+zeta+proteins+by+fusicoccin-based+fluorescent+probes&doi=10.1002%2Fanie.201106995"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1002%2Fanie.201106995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201106995%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DNishi%26aufirst%3DT.%26aulast%3DHamachi%26aufirst%3DI.%26aulast%3DOhkanda%26aufirst%3DJ.%26atitle%3DPhosphopeptide-dependent%2520labeling%2520of%252014%25E2%2580%25933-3%2520zeta%2520proteins%2520by%2520fusicoccin-based%2520fluorescent%2520probes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D509%26epage%3D512%26doi%3D10.1002%2Fanie.201106995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujishima, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Ligand-directed acyl imidazole chemistry for labeling of membrane-bound proteins on live cells</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">3961</span>– <span class="NLM_lpage">3964</span>, <span class="refDoi"> DOI: 10.1021/ja2108855</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja2108855" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XislSqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=3961-3964&author=S.+H.+Fujishimaauthor=R.+Yasuiauthor=T.+Mikiauthor=A.+Ojidaauthor=I.+Hamachi&title=Ligand-directed+acyl+imidazole+chemistry+for+labeling+of+membrane-bound+proteins+on+live+cells&doi=10.1021%2Fja2108855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Directed Acyl Imidazole Chemistry for Labeling of Membrane-Bound Proteins on Live Cells</span></div><div class="casAuthors">Fujishima, Sho-hei; Yasui, Ryosuke; Miki, Takayuki; Ojida, Akio; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3961-3964</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem.-based protein labeling in living cells is undoubtedly useful for understanding natural protein functions and for biol./pharmaceutical applications.  Here, the authors report a novel approach for endogenous membrane-bound protein labeling for both in vitro and live cell conditions.  A moderately reactive alkyloxyacyl imidazole (AI) assisted by ligand-binding affinity (ligand-directed AI (LDAI)) chem. allowed the authors to selectively modify natural proteins, such as dihydrofolate reductase (DHFR) and folate receptor (FR), neither of which could be efficiently labeled using the recently developed ligand-directed tosylate approach.  It was clear that LDAI selectively labeled a single Lys(K32) in DHFR, proximal to the ligand-binding pocket.  The authors also demonstrate that the fluorescein-labeled (endogenous, by LDAI) FR works as a fluorescent biosensor on the live KB cell surface, which allowed the authors to carry out unprecedented in situ kinetic anal. of ligand binding to FR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ZPOEzKbO1LVg90H21EOLACvtfcHk0lhpi0aiOzGkbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislSqu7w%253D&md5=6ccb65da77cad1b62892b738f42e1c55</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1021%2Fja2108855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja2108855%26sid%3Dliteratum%253Aachs%26aulast%3DFujishima%26aufirst%3DS.%2BH.%26aulast%3DYasui%26aufirst%3DR.%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DOjida%26aufirst%3DA.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DLigand-directed%2520acyl%2520imidazole%2520chemistry%2520for%2520labeling%2520of%2520membrane-bound%2520proteins%2520on%2520live%2520cells%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D3961%26epage%3D3964%26doi%3D10.1021%2Fja2108855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujishima, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyonaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">LDAI-based chemical labeling of intact membrane proteins and its pulse-chase analysis under live cell conditions</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1013</span>– <span class="NLM_lpage">1022</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2014.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25126991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlymtLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1013-1022&author=T.+Mikiauthor=S.+H.+Fujishimaauthor=K.+Komatsuauthor=K.+Kuwataauthor=S.+Kiyonakaauthor=I.+Hamachi&title=LDAI-based+chemical+labeling+of+intact+membrane+proteins+and+its+pulse-chase+analysis+under+live+cell+conditions&doi=10.1016%2Fj.chembiol.2014.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">LDAI-Based Chemical Labeling of Intact Membrane Proteins and Its Pulse-Chase Analysis under Live Cell Conditions</span></div><div class="casAuthors">Miki, Takayuki; Fujishima, Sho-hei; Komatsu, Kazuhiro; Kuwata, Keiko; Kiyonaka, Shigeki; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1013-1022</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The functions of membrane proteins are tightly controlled by the dynamics such as protein trafficking and degrdn.  We demonstrated that ligand-directed acyl imidazole (LDAI) chem. is broadly applicable to selective chem. labeling of various types of membrane-bound proteins under live cell conditions without a need for any tag fragments.  The LDAI chem. enabled pulse-chase anal. of these proteins to det. the half-life, as well as their degrdn. pathways by the imaging, under almost natural cellular conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFbXNTJLffMbVg90H21EOLACvtfcHk0lgaBpa8czqfPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlymtLjK&md5=1599f71635cc66d96ed8d9edf8f7f842</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DFujishima%26aufirst%3DS.%2BH.%26aulast%3DKomatsu%26aufirst%3DK.%26aulast%3DKuwata%26aufirst%3DK.%26aulast%3DKiyonaka%26aufirst%3DS.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DLDAI-based%2520chemical%2520labeling%2520of%2520intact%2520membrane%2520proteins%2520and%2520its%2520pulse-chase%2520analysis%2520under%2520live%2520cell%2520conditions%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1013%26epage%3D1022%26doi%3D10.1016%2Fj.chembiol.2014.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koshi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minato, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyonaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukiji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Chemical cell-surface receptor engineering using affinity-guided, multivalent organocatalysts</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">12220</span>– <span class="NLM_lpage">12228</span>, <span class="refDoi"> DOI: 10.1021/ja204422r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja204422r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=12220-12228&author=H.+Wangauthor=Y.+Koshiauthor=D.+Minatoauthor=H.+Nonakaauthor=S.+Kiyonakaauthor=Y.+Moriauthor=S.+Tsukijiauthor=I.+Hamachi&title=Chemical+cell-surface+receptor+engineering+using+affinity-guided%2C+multivalent+organocatalysts&doi=10.1021%2Fja204422r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Cell-Surface Receptor Engineering Using Affinity-Guided, Multivalent Organocatalysts</span></div><div class="casAuthors">Wang, Hangxiang; Koshi, Yoichiro; Minato, Daishiro; Nonaka, Hiroshi; Kiyonaka, Shigeki; Mori, Yasuo; Tsukiji, Shinya; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">12220-12228</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Catalysts hold promise as tools for chem. protein modification.  However, the application of catalysts or catalyst-mediated reactions to proteins has only recently begun to be addressed, mainly in in vitro systems.  By radically improving the affinity-guided DMAP (4-dimethylaminopyridine) (AGD) catalysts that the authors previously reported, here the authors have developed a new organocatalyst-based approach that allows specific chem. acylation of a receptor protein on the surface of live cells.  The catalysts consist of a set of multivalent DMAP groups (the acyl transfer catalyst) fused to a ligand specific to the target protein.  It was clearly demonstrated by in vitro expts. that the catalyst multivalency enables remarkable enhancement of protein acylation efficiency in the labeling of three different proteins: congerin II, a Src homol. 2 (SH2) domain, and FKBP12.  Using a multivalent AGD catalyst and optimized acyl donors contg. a chosen probe, the authors successfully achieved selective chem. labeling of bradykinin B2 receptor (B2R), a G-protein coupled receptor, on the live cell-surface.  Furthermore, the present tool allowed the authors to construct a membrane protein (B2R)-based fluorescent biosensor, the fluorescence of which is enhanced (tuned on) in response to the antagonist ligand binding.  The biosensor should be applicable to rapid and quant. screening and assay of potent drug candidates in the cellular context.  The design concept of the affinity-guided, multivalent catalysts should facilitate further development of diverse catalyst-based protein modification tools, providing new opportunities for org. chem. in biol. research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZGVvnwgJKrbVg90H21EOLACvtfcHk0lgaBpa8czqfPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSksLs%253D&md5=870e8766a9ab6216298c91a01586e022</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1021%2Fja204422r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja204422r%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DKoshi%26aufirst%3DY.%26aulast%3DMinato%26aufirst%3DD.%26aulast%3DNonaka%26aufirst%3DH.%26aulast%3DKiyonaka%26aufirst%3DS.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DTsukiji%26aufirst%3DS.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DChemical%2520cell-surface%2520receptor%2520engineering%2520using%2520affinity-guided%252C%2520multivalent%2520organocatalysts%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D12220%26epage%3D12228%26doi%3D10.1021%2Fja204422r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nukadzuka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, M.</span></span> <span> </span><span class="NLM_article-title">Reactive group-embedded affinity labeling reagent for efficient intracellular protein labeling</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2888</span>– <span class="NLM_lpage">2894</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.02.059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.bmc.2017.02.059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2888-2894&author=Y.+Takaokaauthor=Y.+Nukadzukaauthor=M.+Ueda&title=Reactive+group-embedded+affinity+labeling+reagent+for+efficient+intracellular+protein+labeling&doi=10.1016%2Fj.bmc.2017.02.059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.02.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.02.059%26sid%3Dliteratum%253Aachs%26aulast%3DTakaoka%26aufirst%3DY.%26aulast%3DNukadzuka%26aufirst%3DY.%26aulast%3DUeda%26aufirst%3DM.%26atitle%3DReactive%2520group-embedded%2520affinity%2520labeling%2520reagent%2520for%2520efficient%2520intracellular%2520protein%2520labeling%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2888%26epage%3D2894%26doi%3D10.1016%2Fj.bmc.2017.02.059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koster, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqi, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marappan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, P.</span></span> <span> </span><span class="NLM_article-title">Capture compound mass spectrometry: a technology for the investigation of small molecule protein interactions</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1089/adt.2006.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1089%2Fadt.2006.039" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=381-390&author=H.+Kosterauthor=D.+P.+Littleauthor=P.+Luanauthor=R.+Mullerauthor=S.+M.+Siddiqiauthor=S.+Marappanauthor=P.+Yip&title=Capture+compound+mass+spectrometry%3A+a+technology+for+the+investigation+of+small+molecule+protein+interactions&doi=10.1089%2Fadt.2006.039"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1089%2Fadt.2006.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2006.039%26sid%3Dliteratum%253Aachs%26aulast%3DKoster%26aufirst%3DH.%26aulast%3DLittle%26aufirst%3DD.%2BP.%26aulast%3DLuan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DR.%26aulast%3DSiddiqi%26aufirst%3DS.%2BM.%26aulast%3DMarappan%26aufirst%3DS.%26aulast%3DYip%26aufirst%3DP.%26atitle%3DCapture%2520compound%2520mass%2520spectrometry%253A%2520a%2520technology%2520for%2520the%2520investigation%2520of%2520small%2520molecule%2520protein%2520interactions%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2007%26volume%3D5%26spage%3D381%26epage%3D390%26doi%3D10.1089%2Fadt.2006.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blex, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrey, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furkert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichhorst, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartho, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gyapon Quast, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marais, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hakelberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niquet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popp, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroll, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sefkow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koster, H.</span></span> <span> </span><span class="NLM_article-title">Targeting G Protein-coupled receptors by capture compound mass spectrometry: A case study with Sertindole</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1639</span>– <span class="NLM_lpage">1649</span>, <span class="refDoi"> DOI: 10.1002/cbic.201700152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fcbic.201700152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28557180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFaqtb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1639-1649&author=C.+Blexauthor=S.+Michaelisauthor=A.+K.+Schreyauthor=J.+Furkertauthor=J.+Eichhorstauthor=K.+Barthoauthor=F.+Gyapon+Quastauthor=A.+Maraisauthor=M.+Hakelbergauthor=U.+Gruberauthor=S.+Niquetauthor=O.+Poppauthor=F.+Krollauthor=M.+Sefkowauthor=R.+Schuleinauthor=M.+Dregerauthor=H.+Koster&title=Targeting+G+Protein-coupled+receptors+by+capture+compound+mass+spectrometry%3A+A+case+study+with+Sertindole&doi=10.1002%2Fcbic.201700152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting G Protein-Coupled Receptors by Capture Compound Mass Spectrometry: A Case Study with Sertindole</span></div><div class="casAuthors">Blex, Christian; Michaelis, Simon; Schrey, Anna K.; Furkert, Jens; Eichhorst, Jenny; Bartho, Kathrin; Gyapon Quast, Frederick; Marais, Anett; Hakelberg, Matthias; Gruber, Uschi; Niquet, Sylvia; Popp, Oliver; Kroll, Friedrich; Sefkow, Michael; Schuelein, Ralf; Dreger, Mathias; Koester, Hubert</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1639-1649</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Unbiased chemoproteomic profiling of small-mol. interactions with endogenous proteins is important for drug discovery.  For meaningful results, all protein classes have to be tractable, including G protein-coupled receptors (GPCRs).  These receptors are hardly tractable by affinity pulldown from lysates.  We report a capture compd. (CC)-based strategy to target and identify GPCRs directly from living cells.  We synthesized CCs with sertindole attached to the CC scaffold in different orientations to target the dopamine D2 receptor (DRD2) heterologously expressed in HEK 293 cells.  The structure-activity relationship of sertindole for DRD2 binding was reflected in the activities of the sertindole CCs in radioligand displacement, cell-based assays, and capture compd. mass spectrometry (CCMS).  The activity pattern was rationalized by mol. modeling.  The most-active CC showed activities very similar to that of unmodified sertindole.  A concn. of DRD2 in living cells well below 100 fmol used as an exptl. input was sufficient for unambiguous identification of captured DRD2 by mass spectrometry.  Our new CCMS workflow broadens the arsenal of chemoproteomic technologies to close a crit. gap for the comprehensive characterization of drug-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi26C37KrfrrVg90H21EOLACvtfcHk0lhMzRSf22TLpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFaqtb7F&md5=364e3679dce522cd7294e762a79748c9</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201700152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201700152%26sid%3Dliteratum%253Aachs%26aulast%3DBlex%26aufirst%3DC.%26aulast%3DMichaelis%26aufirst%3DS.%26aulast%3DSchrey%26aufirst%3DA.%2BK.%26aulast%3DFurkert%26aufirst%3DJ.%26aulast%3DEichhorst%26aufirst%3DJ.%26aulast%3DBartho%26aufirst%3DK.%26aulast%3DGyapon%2BQuast%26aufirst%3DF.%26aulast%3DMarais%26aufirst%3DA.%26aulast%3DHakelberg%26aufirst%3DM.%26aulast%3DGruber%26aufirst%3DU.%26aulast%3DNiquet%26aufirst%3DS.%26aulast%3DPopp%26aufirst%3DO.%26aulast%3DKroll%26aufirst%3DF.%26aulast%3DSefkow%26aufirst%3DM.%26aulast%3DSchulein%26aufirst%3DR.%26aulast%3DDreger%26aufirst%3DM.%26aulast%3DKoster%26aufirst%3DH.%26atitle%3DTargeting%2520G%2520Protein-coupled%2520receptors%2520by%2520capture%2520compound%2520mass%2520spectrometry%253A%2520A%2520case%2520study%2520with%2520Sertindole%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D1639%26epage%3D1649%26doi%3D10.1002%2Fcbic.201700152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sosnovtsev, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval-Jaime, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parra, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tin, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, K. Y.</span></span> <span> </span><span class="NLM_article-title">Identification of human junctional adhesion molecule 1 as a functional receptor for the Hom-1 calicivirus on human cells</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e00031-17</span>, <span class="refDoi"> DOI: 10.1128/mBio.00031-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1128%2FmBio.00031-17" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=e00031-17&author=S.+V.+Sosnovtsevauthor=C.+Sandoval-Jaimeauthor=G.+I.+Parraauthor=C.+M.+Tinauthor=R.+W.+Jonesauthor=J.+Sodenauthor=D.+Barnesauthor=J.+Freethauthor=A.+W.+Smithauthor=K.+Y.+Green&title=Identification+of+human+junctional+adhesion+molecule+1+as+a+functional+receptor+for+the+Hom-1+calicivirus+on+human+cells&doi=10.1128%2FmBio.00031-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1128%2FmBio.00031-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.00031-17%26sid%3Dliteratum%253Aachs%26aulast%3DSosnovtsev%26aufirst%3DS.%2BV.%26aulast%3DSandoval-Jaime%26aufirst%3DC.%26aulast%3DParra%26aufirst%3DG.%2BI.%26aulast%3DTin%26aufirst%3DC.%2BM.%26aulast%3DJones%26aufirst%3DR.%2BW.%26aulast%3DSoden%26aufirst%3DJ.%26aulast%3DBarnes%26aufirst%3DD.%26aulast%3DFreeth%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DA.%2BW.%26aulast%3DGreen%26aufirst%3DK.%2BY.%26atitle%3DIdentification%2520of%2520human%2520junctional%2520adhesion%2520molecule%25201%2520as%2520a%2520functional%2520receptor%2520for%2520the%2520Hom-1%2520calicivirus%2520on%2520human%2520cells%26jtitle%3DmBio%26date%3D2017%26volume%3D8%26spage%3De00031-17%26doi%3D10.1128%2FmBio.00031-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavstsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avril, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brazier, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jespersen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magistrado, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lusingu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theander, T. G.</span></span> <span> </span><span class="NLM_article-title">Severe malaria is associated with parasite binding to endothelial protein C receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>498</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">505</span>, <span class="refDoi"> DOI: 10.1038/nature12216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnature12216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23739325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXovVSrt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=498&publication_year=2013&pages=502-505&author=L.+Turnerauthor=T.+Lavstsenauthor=S.+S.+Bergerauthor=C.+W.+Wangauthor=J.+E.+Petersenauthor=M.+Avrilauthor=A.+J.+Brazierauthor=J.+Freethauthor=J.+S.+Jespersenauthor=M.+A.+Nielsenauthor=P.+Magistradoauthor=J.+Lusinguauthor=J.+D.+Smithauthor=M.+K.+Higginsauthor=T.+G.+Theander&title=Severe+malaria+is+associated+with+parasite+binding+to+endothelial+protein+C+receptor&doi=10.1038%2Fnature12216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Severe malaria is associated with parasite binding to endothelial protein C receptor</span></div><div class="casAuthors">Turner, Louise; Lavstsen, Thomas; Berger, Sanne S.; Wang, Christian W.; Petersen, Jens E. V.; Avril, Marion; Brazier, Andrew J.; Freeth, Jim; Jespersen, Jakob S.; Nielsen, Morten A.; Magistrado, Pamela; Lusingu, John; Smith, Joseph D.; Higgins, Matthew K.; Theander, Thor G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">498</span>
        (<span class="NLM_cas:issue">7455</span>),
    <span class="NLM_cas:pages">502-505</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sequestration of Plasmodium falciparum-infected erythrocytes in host blood vessels is a key triggering event in the pathogenesis of severe childhood malaria, which is responsible for about one million deaths every year.  Sequestration is mediated by specific interactions between members of the P. falciparum erythrocyte membrane protein 1 (PfEMP1) family and receptors on the endothelial lining.  Severe childhood malaria is assocd. with expression of specific PfEMP1 subtypes contg. domain cassettes (DCs) 8 and 13 (ref. 3), but the endothelial receptor for parasites expressing these proteins was unknown.  Here we identify endothelial protein C receptor (EPCR), which mediates the cytoprotective effects of activated protein C, as the endothelial receptor for DC8 and DC13 PfEMP1.  We show that EPCR binding is mediated through the amino-terminal cysteine-rich interdomain region (CIDRα1) of DC8 and group A PfEMP1 subfamilies, and that CIDRα1 interferes with protein C binding to EPCR.  This PfEMP1 adhesive property links P. falciparum cytoadhesion to a host receptor involved in anticoagulation and endothelial cytoprotective pathways, and has implications for understanding malaria pathol. and the development of new malaria interventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9Vvb39EmdnbVg90H21EOLACvtfcHk0lhMzRSf22TLpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXovVSrt7w%253D&md5=733b581c14de51378d761b25e133d45d</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1038%2Fnature12216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12216%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DL.%26aulast%3DLavstsen%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BS.%26aulast%3DWang%26aufirst%3DC.%2BW.%26aulast%3DPetersen%26aufirst%3DJ.%2BE.%26aulast%3DAvril%26aufirst%3DM.%26aulast%3DBrazier%26aufirst%3DA.%2BJ.%26aulast%3DFreeth%26aufirst%3DJ.%26aulast%3DJespersen%26aufirst%3DJ.%2BS.%26aulast%3DNielsen%26aufirst%3DM.%2BA.%26aulast%3DMagistrado%26aufirst%3DP.%26aulast%3DLusingu%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DM.%2BK.%26aulast%3DTheander%26aufirst%3DT.%2BG.%26atitle%3DSevere%2520malaria%2520is%2520associated%2520with%2520parasite%2520binding%2520to%2520endothelial%2520protein%2520C%2520receptor%26jtitle%3DNature%26date%3D2013%26volume%3D498%26spage%3D502%26epage%3D505%26doi%3D10.1038%2Fnature12216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrichsen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandercock, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Munoz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmentier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovanovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyland, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lamki, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leishman, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rust, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jermutus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valge-Archer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, R. W.</span></span> <span> </span><span class="NLM_article-title">Phenotypic screening reveals TNFR2 as a promising target for cancer immunotherapy</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">68278</span>– <span class="NLM_lpage">68291</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.11943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.18632%2Foncotarget.11943" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=68278-68291&author=G.+S.+Williamsauthor=B.+Mistryauthor=S.+Guillardauthor=J.+C.+Ulrichsenauthor=A.+M.+Sandercockauthor=J.+Wangauthor=A.+Gonzalez-Munozauthor=J.+Parmentierauthor=C.+Blackauthor=J.+Sodenauthor=J.+Freethauthor=J.+Jovanovicauthor=R.+Leylandauthor=R.+S.+Al-Lamkiauthor=A.+J.+Leishmanauthor=S.+J.+Rustauthor=R.+Stewartauthor=L.+Jermutusauthor=J.+R.+Bradleyauthor=V.+Bedianauthor=V.+Valge-Archerauthor=R.+Minterauthor=R.+W.+Wilkinson&title=Phenotypic+screening+reveals+TNFR2+as+a+promising+target+for+cancer+immunotherapy&doi=10.18632%2Foncotarget.11943"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.11943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.11943%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DG.%2BS.%26aulast%3DMistry%26aufirst%3DB.%26aulast%3DGuillard%26aufirst%3DS.%26aulast%3DUlrichsen%26aufirst%3DJ.%2BC.%26aulast%3DSandercock%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGonzalez-Munoz%26aufirst%3DA.%26aulast%3DParmentier%26aufirst%3DJ.%26aulast%3DBlack%26aufirst%3DC.%26aulast%3DSoden%26aufirst%3DJ.%26aulast%3DFreeth%26aufirst%3DJ.%26aulast%3DJovanovic%26aufirst%3DJ.%26aulast%3DLeyland%26aufirst%3DR.%26aulast%3DAl-Lamki%26aufirst%3DR.%2BS.%26aulast%3DLeishman%26aufirst%3DA.%2BJ.%26aulast%3DRust%26aufirst%3DS.%2BJ.%26aulast%3DStewart%26aufirst%3DR.%26aulast%3DJermutus%26aufirst%3DL.%26aulast%3DBradley%26aufirst%3DJ.%2BR.%26aulast%3DBedian%26aufirst%3DV.%26aulast%3DValge-Archer%26aufirst%3DV.%26aulast%3DMinter%26aufirst%3DR.%26aulast%3DWilkinson%26aufirst%3DR.%2BW.%26atitle%3DPhenotypic%2520screening%2520reveals%2520TNFR2%2520as%2520a%2520promising%2520target%2520for%2520cancer%2520immunotherapy%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D68278%26epage%3D68291%26doi%3D10.18632%2Foncotarget.11943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sandercock, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rust, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachsenmeier, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holoweckyj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herpers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jermutus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollingsworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minter, R.</span></span> <span> </span><span class="NLM_article-title">Identification of anti-tumour biologics using primary tumour models, 3-D phenotypic screening and image-based multi-parametric profiling</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">147</span>, <span class="refDoi"> DOI: 10.1186/s12943-015-0415-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1186%2Fs12943-015-0415-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26227951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FpsFWjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=147&author=A.+M.+Sandercockauthor=S.+Rustauthor=S.+Guillardauthor=K.+F.+Sachsenmeierauthor=N.+Holoweckyjauthor=C.+Hayauthor=M.+Flynnauthor=Q.+Huangauthor=K.+Yanauthor=B.+Herpersauthor=L.+S.+Priceauthor=J.+Sodenauthor=J.+Freethauthor=L.+Jermutusauthor=R.+Hollingsworthauthor=R.+Minter&title=Identification+of+anti-tumour+biologics+using+primary+tumour+models%2C+3-D+phenotypic+screening+and+image-based+multi-parametric+profiling&doi=10.1186%2Fs12943-015-0415-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of anti-tumour biologics using primary tumour models, 3-D phenotypic screening and image-based multi-parametric profiling</span></div><div class="casAuthors">Sandercock Alan M; Rust Steven; Guillard Sandrine; Jermutus Lutz; Minter Ralph; Sachsenmeier Kris F; Holoweckyj Nick; Hay Carl; Flynn Matt; Huang Qihui; Hollingsworth Robert; Yan Kuan; Herpers Bram; Price Leo S; Soden Jo; Freeth Jim</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">147</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Monolayer cultures of immortalised cell lines are a popular screening tool for novel anti-cancer therapeutics, but these methods can be a poor surrogate for disease states, and there is a need for drug screening platforms which are more predictive of clinical outcome.  In this study, we describe a phenotypic antibody screen using three-dimensional cultures of primary cells, and image-based multi-parametric profiling in PC-3 cells, to identify anti-cancer biologics against new therapeutic targets.  METHODS:  ScFv Antibodies and designed ankyrin repeat proteins (DARPins) were isolated using phage display selections against primary non-small cell lung carcinoma cells.  The selected molecules were screened for anti-proliferative and pro-apoptotic activity against primary cells grown in three-dimensional culture, and in an ultra-high content screen on a 3-D cultured cell line using multi-parametric profiling to detect treatment-induced phenotypic changes.  The targets of molecules of interest were identified using a cell-surface membrane protein array.  An anti-CUB domain containing protein 1 (CDCP1) antibody was tested for tumour growth inhibition in a patient-derived xenograft model, generated from a stage-IV non-small cell lung carcinoma, with and without cisplatin.  RESULTS:  Two primary non-small cell lung carcinoma cell models were established for antibody isolation and primary screening in anti-proliferative and apoptosis assays.  These assays identified multiple antibodies demonstrating activity in specific culture formats.  A subset of the DARPins was profiled in an ultra-high content multi-parametric screen, where 300 morphological features were measured per sample.  Machine learning was used to select features to classify treatment responses, then antibodies were characterised based on the phenotypes that they induced.  This method co-classified several DARPins that targeted CDCP1 into two sets with different phenotypes.  Finally, an anti-CDCP1 antibody significantly enhanced the efficacy of cisplatin in a patient-derived NSCLC xenograft model.  CONCLUSIONS:  Phenotypic profiling using complex 3-D cell cultures steers hit selection towards more relevant in vivo phenotypes, and may shed light on subtle mechanistic variations in drug candidates, enabling data-driven decisions for oncology target validation.  CDCP1 was identified as a potential target for cisplatin combination therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9qh7axt418RpyrDyM_m9cfW6udTcc2ebcy60nNfbDSbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FpsFWjtg%253D%253D&md5=9f6fa8799c7febcb8a9900c59729a0ab</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1186%2Fs12943-015-0415-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-015-0415-0%26sid%3Dliteratum%253Aachs%26aulast%3DSandercock%26aufirst%3DA.%2BM.%26aulast%3DRust%26aufirst%3DS.%26aulast%3DGuillard%26aufirst%3DS.%26aulast%3DSachsenmeier%26aufirst%3DK.%2BF.%26aulast%3DHoloweckyj%26aufirst%3DN.%26aulast%3DHay%26aufirst%3DC.%26aulast%3DFlynn%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DYan%26aufirst%3DK.%26aulast%3DHerpers%26aufirst%3DB.%26aulast%3DPrice%26aufirst%3DL.%2BS.%26aulast%3DSoden%26aufirst%3DJ.%26aulast%3DFreeth%26aufirst%3DJ.%26aulast%3DJermutus%26aufirst%3DL.%26aulast%3DHollingsworth%26aufirst%3DR.%26aulast%3DMinter%26aufirst%3DR.%26atitle%3DIdentification%2520of%2520anti-tumour%2520biologics%2520using%2520primary%2520tumour%2520models%252C%25203-D%2520phenotypic%2520screening%2520and%2520image-based%2520multi-parametric%2520profiling%26jtitle%3DMol.%2520Cancer%26date%3D2015%26volume%3D14%26spage%3D147%26doi%3D10.1186%2Fs12943-015-0415-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Identification of human UMP/CMP Kinase 1 as Doxorubicin binding target using protein microarray</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1015</span>, <span class="refDoi"> DOI: 10.1177/2472555217707704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F2472555217707704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28459633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKkt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1007-1015&author=S.+Chenauthor=X.+Wangauthor=X.+Yeauthor=D.+Maauthor=C.+Chenauthor=J.+Caiauthor=Y.+Fuauthor=X.+Chengauthor=Y.+Chenauthor=X.+Gongauthor=J.+Jin&title=Identification+of+human+UMP%2FCMP+Kinase+1+as+Doxorubicin+binding+target+using+protein+microarray&doi=10.1177%2F2472555217707704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Human UMP/CMP Kinase 1 as Doxorubicin Binding Target Using Protein Microarray</span></div><div class="casAuthors">Chen, Shuxian; Wang, Xu; Ye, Xianghui; Ma, Donghui; Chen, Caiwei; Cai, Junlong; Fu, Yongfeng; Cheng, Xunjia; Chen, Yun; Gong, Xiaohai; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1007-1015</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Doxorubicin (DOX) is a leading anthracycline drug with exceptional efficacy; however, little is known about the mol. mechanisms of its side effects, which include heart muscle damage, noncancerous cell death, and drug resistance.  A total of 17,950 human proteins expressed in HEK293 cells were screened and yielded 14 hits.  Competitive and binding expts. further verified the binding of DOX to UMP/CMP kinase 1 (CMPK1), and microscale thermophoresis showed that DOX binds to CMPK1 with a Kd of 1216 nM.  In addn., we obsd. that the binding of DOX to CMPK1 activated the phosphorylation of CMP, dCMP, and UMP.  A significant activation was obsd. at the concn. of 30 μM DOX and reached plateau at the concn. of DOX 30 μM, 150 μM, and 100 μM, resp.  DOX would add up stimulation of CMPK1 by DTT and overcome inhibition of CMPK1 by NaF, EDTA.  In summary, we showed that DOX might bind to the nonactive site of CMPK1 and regulate its activity with magnesium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGJWa1yckkXbVg90H21EOLACvtfcHk0lgyCYDEMuUHdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKkt7o%253D&md5=5fcbc3d47d6ae6bff18c2f7f7adec665</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1177%2F2472555217707704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217707704%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520human%2520UMP%252FCMP%2520Kinase%25201%2520as%2520Doxorubicin%2520binding%2520target%2520using%2520protein%2520microarray%26jtitle%3DSLAS%2520Discov%26date%3D2017%26volume%3D22%26spage%3D1007%26epage%3D1015%26doi%3D10.1177%2F2472555217707704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bassilana, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DaSilva, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosshans, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilmeringwetter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llamas, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigollier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourret, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramamurthy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbinski, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plonsky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruffner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sailer, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouwmeester, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finan, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelleher, J. F.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luchansky, S. J.</span></span> <span> </span><span class="NLM_article-title">Target identification for a Hedgehog pathway inhibitor reveals the receptor GPR39</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">349</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.1481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24633354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlGrt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=343-349&author=F.+Bassilanaauthor=A.+Carlsonauthor=J.+A.+DaSilvaauthor=B.+Grosshansauthor=S.+Vidalauthor=V.+Beckauthor=B.+Wilmeringwetterauthor=L.+A.+Llamasauthor=T.+B.+Showalterauthor=P.+Rigollierauthor=A.+Bourretauthor=A.+Ramamurthyauthor=X.+Wuauthor=F.+Harbinskiauthor=S.+Plonskyauthor=L.+Leeauthor=H.+Ruffnerauthor=P.+Grandiauthor=M.+Schirleauthor=J.+Jenkinsauthor=A.+W.+Sailerauthor=T.+Bouwmeesterauthor=J.+A.+Porterauthor=V.+Myerauthor=P.+M.+Finanauthor=J.+A.+Tallaricoauthor=J.+F.+Kelleherauthor=K.+Seuwenauthor=R.+K.+Jainauthor=S.+J.+Luchansky&title=Target+identification+for+a+Hedgehog+pathway+inhibitor+reveals+the+receptor+GPR39&doi=10.1038%2Fnchembio.1481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Target identification for a Hedgehog pathway inhibitor reveals the receptor GPR39</span></div><div class="casAuthors">Bassilana, Frederic; Carlson, Adam; Da Silva, Jennifer A.; Grosshans, Bianka; Vidal, Solange; Beck, Valerie; Wilmeringwetter, Barbara; Llamas, Luis A.; Showalter, Todd B.; Rigollier, Pascal; Bourret, Aaron; Ramamurthy, Arun; Wu, Xu; Harbinski, Fred; Plonsky, Samantha; Lee, Lac; Ruffner, Heinz; Grandi, Paola; Schirle, Markus; Jenkins, Jeremy; Sailer, Andreas W.; Bouwmeester, Tewis; Porter, Jeffrey A.; Myer, Vic; Finan, Peter M.; Tallarico, John A.; Kelleher, Joseph F., III; Seuwen, Klaus; Jain, Rishi K.; Luchansky, Sarah J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hedgehog (Hh) signaling dets. cell fate during development and can drive tumorigenesis.  We performed a screen for new compds. that can impinge on Hh signaling downstream of Smoothened (Smo).  A series of cyclohexyl-Me aminopyrimidine chemotype compds. ('CMAPs') were identified that could block pathway signaling in a Smo-independent manner.  In addn. to inhibiting Hh signaling, the compds. generated inositol phosphates through an unknown GPCR.  Correlation of GPCR mRNA expression levels with compd. activity across cell lines suggested the target to be the orphan receptor GPR39.  RNA interference or cDNA overexpression of GPR39 demonstrated that the receptor is necessary for compd. activity.  We propose a model in which CMAPs activate GPR39, which signals to the Gli transcription factors and blocks signaling.  In addn. to the discovery of GPR39 as a new target that impinges on Hh signaling, we report on small-mol. modulators of the receptor that will enable in vitro interrogation of GPR39 signaling in different cellular contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpauWdqJMnkc7Vg90H21EOLACvtfcHk0lgyCYDEMuUHdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlGrt7c%253D&md5=1b5bc844ae5698e1e69578eb6ac053cd</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1481%26sid%3Dliteratum%253Aachs%26aulast%3DBassilana%26aufirst%3DF.%26aulast%3DCarlson%26aufirst%3DA.%26aulast%3DDaSilva%26aufirst%3DJ.%2BA.%26aulast%3DGrosshans%26aufirst%3DB.%26aulast%3DVidal%26aufirst%3DS.%26aulast%3DBeck%26aufirst%3DV.%26aulast%3DWilmeringwetter%26aufirst%3DB.%26aulast%3DLlamas%26aufirst%3DL.%2BA.%26aulast%3DShowalter%26aufirst%3DT.%2BB.%26aulast%3DRigollier%26aufirst%3DP.%26aulast%3DBourret%26aufirst%3DA.%26aulast%3DRamamurthy%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHarbinski%26aufirst%3DF.%26aulast%3DPlonsky%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DL.%26aulast%3DRuffner%26aufirst%3DH.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%26aulast%3DSailer%26aufirst%3DA.%2BW.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DMyer%26aufirst%3DV.%26aulast%3DFinan%26aufirst%3DP.%2BM.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DKelleher%26aufirst%3DJ.%2BF.%26aulast%3DSeuwen%26aufirst%3DK.%26aulast%3DJain%26aufirst%3DR.%2BK.%26aulast%3DLuchansky%26aufirst%3DS.%2BJ.%26atitle%3DTarget%2520identification%2520for%2520a%2520Hedgehog%2520pathway%2520inhibitor%2520reveals%2520the%2520receptor%2520GPR39%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D343%26epage%3D349%26doi%3D10.1038%2Fnchembio.1481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnea, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strapps, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrada, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloss, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. J.</span></span> <span> </span><span class="NLM_article-title">The genetic design of signaling cascades to record receptor activation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1073/pnas.0710487105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.0710487105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=18165312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpvV2ktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=64-69&author=G.+Barneaauthor=W.+Strappsauthor=G.+Herradaauthor=Y.+Bermanauthor=J.+Ongauthor=B.+Klossauthor=R.+Axelauthor=K.+J.+Lee&title=The+genetic+design+of+signaling+cascades+to+record+receptor+activation&doi=10.1073%2Fpnas.0710487105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">The genetic design of signaling cascades to record receptor activation</span></div><div class="casAuthors">Barnea, Gilad; Strapps, Walter; Herrada, Gilles; Berman, Yemiliya; Ong, Jane; Kloss, Brian; Axel, Richard; Lee, Kevin J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-69</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The authors have developed an exptl. strategy to monitor protein interactions in a cell with a high degree of selectivity and sensitivity.  A transcription factor is tethered to a membrane-bound receptor with a linker that contains a cleavage site for a specific protease.  Activation of the receptor recruits a signaling protein fused to the protease that then cleaves and releases the transcription factor to activate reporter genes in the nucleus.  This strategy converts a transient interaction into a stable and amplifiable reporter gene signal to record the activation of a receptor without interference from endogenous signaling pathways.  The authors have developed this assay for three classes of receptors: G protein-coupled receptors, receptor tyrosine kinases, and steroid hormone receptors.  Finally, the authors use the assay to identify a ligand for the orphan receptor GPR1, suggesting a role for this receptor in the regulation of inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2ugB7pgEqlrVg90H21EOLACvtfcHk0lgWxxWZQDk1jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpvV2ktA%253D%253D&md5=3b49dfdb2949d947e145084ac87bb343</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0710487105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0710487105%26sid%3Dliteratum%253Aachs%26aulast%3DBarnea%26aufirst%3DG.%26aulast%3DStrapps%26aufirst%3DW.%26aulast%3DHerrada%26aufirst%3DG.%26aulast%3DBerman%26aufirst%3DY.%26aulast%3DOng%26aufirst%3DJ.%26aulast%3DKloss%26aufirst%3DB.%26aulast%3DAxel%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DK.%2BJ.%26atitle%3DThe%2520genetic%2520design%2520of%2520signaling%2520cascades%2520to%2520record%2520receptor%2520activation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D64%26epage%3D69%26doi%3D10.1073%2Fpnas.0710487105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguere, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">369</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnsmb.3014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25895059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=362-369&author=W.+K.+Kroezeauthor=M.+F.+Sassanoauthor=X.+P.+Huangauthor=K.+Lansuauthor=J.+D.+McCorvyauthor=P.+M.+Giguereauthor=N.+Sciakyauthor=B.+L.+Roth&title=PRESTO-Tango+as+an+open-source+resource+for+interrogation+of+the+druggable+human+GPCRome&doi=10.1038%2Fnsmb.3014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span></div><div class="casAuthors">Kroeze, Wesley K.; Sassano, Maria F.; Huang, Xi-Ping; Lansu, Katherine; McCorvy, John D.; Giguere, Patrick M.; Sciaky, Noah; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">362-369</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are essential mediators of cellular signaling and are important targets of drug action.  Of the approx. 350 nonolfactory human GPCRs, more than 100 are still considered to be 'orphans' because their endogenous ligands remain unknown.  Here, we describe a unique open-source resource that allows interrogation of the druggable human GPCRome via a G protein-independent β-arrestin-recruitment assay.  We validate this unique platform at more than 120 nonorphan human GPCR targets, demonstrate its utility for discovering new ligands for orphan human GPCRs and describe a method (parallel receptorome expression and screening via transcriptional output, with transcriptional activation following arrestin translocation (PRESTO-Tango)) for the simultaneous and parallel interrogation of the entire human nonolfactory GPCRome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVbl7caV9gUrVg90H21EOLACvtfcHk0lgWxxWZQDk1jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D&md5=2a78b74a5f1425fa9e8b79b730c16be6</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3014%26sid%3Dliteratum%253Aachs%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DGiguere%26aufirst%3DP.%2BM.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPRESTO-Tango%2520as%2520an%2520open-source%2520resource%2520for%2520interrogation%2520of%2520the%2520druggable%2520human%2520GPCRome%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D362%26epage%3D369%26doi%3D10.1038%2Fnsmb.3014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gee, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. E.</span></span> <span> </span><span class="NLM_article-title">Kirkwood-Buff theory of molecular and protein association, aggregation, and cellular crowding</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">165101</span>, <span class="refDoi"> DOI: 10.1063/1.3253299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1063%2F1.3253299" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=165101&author=M.+B.+Geeauthor=P.+E.+Smith&title=Kirkwood-Buff+theory+of+molecular+and+protein+association%2C+aggregation%2C+and+cellular+crowding&doi=10.1063%2F1.3253299"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1063%2F1.3253299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3253299%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DM.%2BB.%26aulast%3DSmith%26aufirst%3DP.%2BE.%26atitle%3DKirkwood-Buff%2520theory%2520of%2520molecular%2520and%2520protein%2520association%252C%2520aggregation%252C%2520and%2520cellular%2520crowding%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2009%26volume%3D131%26spage%3D165101%26doi%3D10.1063%2F1.3253299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bathula, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of novel 5-arylidenethiazolidinones with apoptotic properties via a three component reaction using piperidine as a bifunctional reagent</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">8053</span>– <span class="NLM_lpage">8063</span>, <span class="refDoi"> DOI: 10.1039/C6OB01257D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2FC6OB01257D" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=8053-8063&author=C.+Bathulaauthor=S.+Tripathiauthor=R.+Srinivasanauthor=K.+K.+Jhaauthor=A.+Ganguliauthor=G.+Chakrabartiauthor=S.+Singhauthor=P.+Munshiauthor=S.+Sen&title=Synthesis+of+novel+5-arylidenethiazolidinones+with+apoptotic+properties+via+a+three+component+reaction+using+piperidine+as+a+bifunctional+reagent&doi=10.1039%2FC6OB01257D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1039%2FC6OB01257D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6OB01257D%26sid%3Dliteratum%253Aachs%26aulast%3DBathula%26aufirst%3DC.%26aulast%3DTripathi%26aufirst%3DS.%26aulast%3DSrinivasan%26aufirst%3DR.%26aulast%3DJha%26aufirst%3DK.%2BK.%26aulast%3DGanguli%26aufirst%3DA.%26aulast%3DChakrabarti%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DMunshi%26aufirst%3DP.%26aulast%3DSen%26aufirst%3DS.%26atitle%3DSynthesis%2520of%2520novel%25205-arylidenethiazolidinones%2520with%2520apoptotic%2520properties%2520via%2520a%2520three%2520component%2520reaction%2520using%2520piperidine%2520as%2520a%2520bifunctional%2520reagent%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D8053%26epage%3D8063%26doi%3D10.1039%2FC6OB01257D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, S.</span></span> <span> </span><span class="NLM_article-title">Spiro[pyrrolidine-3, 3 -oxindole] as potent anti-breast cancer compounds: Their design, synthesis, biological evaluation and cellular target identification</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">32213</span>, <span class="refDoi"> DOI: 10.1038/srep32213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fsrep32213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27573798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCqt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=32213&author=S.+Hatiauthor=S.+Tripathyauthor=P.+K.+Duttaauthor=R.+Agarwalauthor=R.+Srinivasanauthor=A.+Singhauthor=S.+Singhauthor=S.+Sen&title=Spiro%5Bpyrrolidine-3%2C+3+-oxindole%5D+as+potent+anti-breast+cancer+compounds%3A+Their+design%2C+synthesis%2C+biological+evaluation+and+cellular+target+identification&doi=10.1038%2Fsrep32213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Spiro[pyrrolidine-3,3'-oxindole] as potent anti-breast cancer compounds: Their design, synthesis, biological evaluation and cellular target identification</span></div><div class="casAuthors">Hati, Santanu; Tripathy, Sayantan; Dutta, Pratip Kumar; Agarwal, Rahul; Srinivasan, Ramprasad; Singh, Ashutosh; Singh, Shailja; Sen, Subhabrata</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32213pp.</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design and synthesis of a library of compds. bearing spiro[pyrrolidine-3, 3'-oxindole] motifs I (Ar = C6H5, 4-CH3C6H4, thiophen-2-yl, etc.) that demonstrated exceptional inhibitory activity against the proliferation of MCF-7 breast cancer cells has been reported.  The synthesis involved a one pot Pictet Spengler-Oxidative ring contraction of tryptamine to the desired scaffolds and occurred in 1:1 THF and water with catalytic trifluoroacetic acid and stoichiometric N-bromosuccinimide as an oxidant.  Phenotypic profiling indicated that these mols. induce apoptotic cell death in MCF-7 cells.  Target deconvolution with most potent compd. I (Ar = C6H5) from the library, using chem. proteomics indicated histone deacetylase 2 (HDAC2) and prohibitin 2 as the potential cellular binding partners.  Mol. docking of compd. I (Ar = C6H5) with HDAC2 provided insights pertinent to putative binding interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo95MMqBQ5m37Vg90H21EOLACvtfcHk0lgwp4FSJy2yxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCqt7vK&md5=a39920c7c7d8f6f3fe9eb7bda65ecc7f</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1038%2Fsrep32213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep32213%26sid%3Dliteratum%253Aachs%26aulast%3DHati%26aufirst%3DS.%26aulast%3DTripathy%26aufirst%3DS.%26aulast%3DDutta%26aufirst%3DP.%2BK.%26aulast%3DAgarwal%26aufirst%3DR.%26aulast%3DSrinivasan%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DSen%26aufirst%3DS.%26atitle%3DSpiro%255Bpyrrolidine-3%252C%25203%2520-oxindole%255D%2520as%2520potent%2520anti-breast%2520cancer%2520compounds%253A%2520Their%2520design%252C%2520synthesis%252C%2520biological%2520evaluation%2520and%2520cellular%2520target%2520identification%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D32213%26doi%3D10.1038%2Fsrep32213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franken, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweetman, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gade, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhard, F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span> <span> </span><span class="NLM_article-title">Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1567</span>– <span class="NLM_lpage">1593</span>, <span class="refDoi"> DOI: 10.1038/nprot.2015.101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnprot.2015.101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26379230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1567-1593&author=H.+Frankenauthor=T.+Mathiesonauthor=D.+Childsauthor=G.+M.+Sweetmanauthor=T.+Wernerauthor=I.+Togelauthor=C.+Doceauthor=S.+Gadeauthor=M.+Bantscheffauthor=G.+Drewesauthor=F.+B.+Reinhardauthor=W.+Huberauthor=M.+M.+Savitski&title=Thermal+proteome+profiling+for+unbiased+identification+of+direct+and+indirect+drug+targets+using+multiplexed+quantitative+mass+spectrometry&doi=10.1038%2Fnprot.2015.101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry</span></div><div class="casAuthors">Franken, Holger; Mathieson, Toby; Childs, Dorothee; Sweetman, Gavain M. A.; Werner, Thilo; Toegel, Ina; Doce, Carola; Gade, Stephan; Bantscheff, Marcus; Drewes, Gerard; Reinhard, Friedrich B. M.; Huber, Wolfgang; Savitski, Mikhail M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1567-1593</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The direct detection of drug-protein interactions in living cells is a major challenge in drug discovery research.  Recently, the authors introduced an approach termed thermal proteome profiling (TPP), which enables the monitoring of changes in protein thermal stability across the proteome using quant. mass spectrometry.  The authors detd. the intracellular thermal profiles for up to 7000 proteins, and by comparing profiles derived from cultured mammalian cells in the presence or absence of a drug it was possible to identify direct and indirect targets of drugs in living cells in an unbiased manner.  Here the authors demonstrate the complete workflow using the histone deacetylase inhibitor panobinostat.  The key to this approach is the use of isobaric tandem mass tag 10-plex (TMT10) reagents to label digested protein samples corresponding to each temp. point in the melting curve so that the samples can be analyzed by multiplexed quant. mass spectrometry.  Important steps in the bioinformatic anal. include data normalization, melting curve fitting and statistical significance detn. of compd. concn.-dependent changes in protein stability.  All anal. tools are made freely available as R and Python packages.  The workflow can be completed in 2 wk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS4LU9YprZqLVg90H21EOLACvtfcHk0lgwp4FSJy2yxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzN&md5=d9c225616a35fc5dcc97db234452d7e1</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.101%26sid%3Dliteratum%253Aachs%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DChilds%26aufirst%3DD.%26aulast%3DSweetman%26aufirst%3DG.%2BM.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DTogel%26aufirst%3DI.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DGade%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DReinhard%26aufirst%3DF.%2BB.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26atitle%3DThermal%2520proteome%2520profiling%2520for%2520unbiased%2520identification%2520of%2520direct%2520and%2520indirect%2520drug%2520targets%2520using%2520multiplexed%2520quantitative%2520mass%2520spectrometry%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D1567%26epage%3D1593%26doi%3D10.1038%2Fnprot.2015.101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olek, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1055</span>– <span class="NLM_lpage">1057</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnmeth.3590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26389571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1055-1057&author=K.+V.+Huberauthor=K.+M.+Olekauthor=A.+C.+Mullerauthor=C.+S.+Tanauthor=K.+L.+Bennettauthor=J.+Colingeauthor=G.+Superti-Furga&title=Proteome-wide+drug+and+metabolite+interaction+mapping+by+thermal-stability+profiling&doi=10.1038%2Fnmeth.3590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span></div><div class="casAuthors">Huber, Kilian V. M.; Olek, Karin M.; Mueller, Andre C.; Tan, Chris Soon Heng; Bennett, Keiryn L.; Colinge, Jacques; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1055-1057</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal stabilization of proteins after ligand binding provides an efficient means to assess the binding of small mols. to proteins.  We show here that in combination with quant. mass spectrometry, the approach allows for the systematic survey of protein engagement by cellular metabolites and drugs.  We profiled the targets of the drugs methotrexate and (S)-crizotinib and the metabolite 2'3'-cGAMP in intact cells and identified the 2'3'-cGAMP cognate transmembrane receptor STING, involved in immune signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrow66ugdg1oLVg90H21EOLACvtfcHk0liKdWQGnLTUTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI&md5=619a2c08c4f151f4e3ab8bd8fc45c2b6</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3590%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DOlek%26aufirst%3DK.%2BM.%26aulast%3DMuller%26aufirst%3DA.%2BC.%26aulast%3DTan%26aufirst%3DC.%2BS.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DProteome-wide%2520drug%2520and%2520metabolite%2520interaction%2520mapping%2520by%2520thermal-stability%2520profiling%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1055%26epage%3D1057%26doi%3D10.1038%2Fnmeth.3590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reinhard, F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franken, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span> <span> </span><span class="NLM_article-title">Thermal proteome profiling monitors ligand interactions with cellular membrane proteins</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1129</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3652</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnmeth.3652" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26524241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1129-1131&author=F.+B.+Reinhardauthor=D.+Eberhardauthor=T.+Wernerauthor=H.+Frankenauthor=D.+Childsauthor=C.+Doceauthor=M.+F.+Savitskiauthor=W.+Huberauthor=M.+Bantscheffauthor=M.+M.+Savitskiauthor=G.+Drewes&title=Thermal+proteome+profiling+monitors+ligand+interactions+with+cellular+membrane+proteins&doi=10.1038%2Fnmeth.3652"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal proteome profiling monitors ligand interactions with cellular membrane proteins</span></div><div class="casAuthors">Reinhard, Friedrich B. M.; Eberhard, Dirk; Werner, Thilo; Franken, Holger; Childs, Dorothee; Doce, Carola; Savitski, Maria Faelth; Huber, Wolfgang; Bantscheff, Marcus; Savitski, Mikhail M.; Drewes, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1129-1131</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We extended thermal proteome profiling to detect transmembrane protein-small mol. interactions in cultured human cells.  When we assessed the effects of detergents on ATP-binding profiles, we obsd. shifts in denaturation temp. for ATP-binding transmembrane proteins.  We also obsd. cellular thermal shifts in pervanadate-induced T cell-receptor signaling, delineating the membrane target CD45 and components of the downstream pathway, and with drugs affecting the transmembrane transporters ATP1A1 and MDR1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo45i1e8jbFF7Vg90H21EOLACvtfcHk0liKdWQGnLTUTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7fO&md5=ec4651a2753f7969b16811faa8affa23</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3652&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3652%26sid%3Dliteratum%253Aachs%26aulast%3DReinhard%26aufirst%3DF.%2BB.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DChilds%26aufirst%3DD.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BF.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DThermal%2520proteome%2520profiling%2520monitors%2520ligand%2520interactions%2520with%2520cellular%2520membrane%2520proteins%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1129%26epage%3D1131%26doi%3D10.1038%2Fnmeth.3652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhard, F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franken, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovega, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span> <span> </span><span class="NLM_article-title">Tracking cancer drugs in living cells by thermal profiling of the proteome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span>,  <span class="NLM_fpage">1255784</span>, <span class="refDoi"> DOI: 10.1126/science.1255784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1255784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25278616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC2M7ntVKgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=1255784&author=M.+M.+Savitskiauthor=F.+B.+Reinhardauthor=H.+Frankenauthor=T.+Wernerauthor=M.+F.+Savitskiauthor=D.+Eberhardauthor=D.+Martinez+Molinaauthor=R.+Jafariauthor=R.+B.+Dovegaauthor=S.+Klaegerauthor=B.+Kusterauthor=P.+Nordlundauthor=M.+Bantscheffauthor=G.+Drewes&title=Tracking+cancer+drugs+in+living+cells+by+thermal+profiling+of+the+proteome&doi=10.1126%2Fscience.1255784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Tracking cancer drugs in living cells by thermal profiling of the proteome</span></div><div class="casAuthors">Savitski Mikhail M; Reinhard Friedrich B M; Franken Holger; Werner Thilo; Savitski Maria Falth; Eberhard Dirk; Bantscheff Marcus; Drewes Gerard; Martinez Molina Daniel; Jafari Rozbeh; Dovega Rebecca Bakszt; Klaeger Susan; Kuster Bernhard; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6205</span>),
    <span class="NLM_cas:pages">1255784</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The thermal stability of proteins can be used to assess ligand binding in living cells.  We have generalized this concept by determining the thermal profiles of more than 7000 proteins in human cells by means of mass spectrometry.  Monitoring the effects of small-molecule ligands on the profiles delineated more than 50 targets for the kinase inhibitor staurosporine.  We identified the heme biosynthesis enzyme ferrochelatase as a target of kinase inhibitors and suggest that its inhibition causes the phototoxicity observed with vemurafenib and alectinib.  Thermal shifts were also observed for downstream effectors of drug treatment.  In live cells, dasatinib induced shifts in BCR-ABL pathway proteins, including CRK/CRKL.  Thermal proteome profiling provides an unbiased measure of drug-target engagement and facilitates identification of markers for drug efficacy and toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS0ur3d8ba-Bu2mnZh2OgS0fW6udTcc2eau2C7qHSQCILntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7ntVKgsg%253D%253D&md5=493a89ad107e656e0f0786ff5c8e5f63</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1126%2Fscience.1255784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1255784%26sid%3Dliteratum%253Aachs%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DReinhard%26aufirst%3DF.%2BB.%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DSavitski%26aufirst%3DM.%2BF.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DDovega%26aufirst%3DR.%2BB.%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DTracking%2520cancer%2520drugs%2520in%2520living%2520cells%2520by%2520thermal%2520profiling%2520of%2520the%2520proteome%26jtitle%3DScience%26date%3D2014%26volume%3D346%26spage%3D1255784%26doi%3D10.1126%2Fscience.1255784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundback, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundbackauthor=P.+Nordlundauthor=D.+Martinez+Molina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0libHqGo2HASAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundback%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+Martinez+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span></div><div class="casAuthors">Martinez Molina Daniel; Jafari Rozbeh; Ignatushchenko Marina; Seki Takahiro; Larsson E Andreas; Dan Chen; Sreekumar Lekshmy; Cao Yihai; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophysical principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clinical targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRRn5K_mF1rcrHHuJhbyicZfW6udTcc2eazjMsWobC0Arntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D&md5=a73bb49adb0d4f445223ea6f74fe3c10</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaal, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkers, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzpatrick, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span> <span> </span><span class="NLM_article-title">Thermal profiling reveals phenylalanine hydroxylase as an off-target of panobinostat</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">908</span>– <span class="NLM_lpage">910</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.2185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27669419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOmsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=908-910&author=I.+Becherauthor=T.+Wernerauthor=C.+Doceauthor=E.+A.+Zaalauthor=I.+Togelauthor=C.+A.+Khanauthor=A.+Ruegerauthor=M.+Muelbaierauthor=E.+Salzerauthor=C.+R.+Berkersauthor=P.+F.+Fitzpatrickauthor=M.+Bantscheffauthor=M.+M.+Savitski&title=Thermal+profiling+reveals+phenylalanine+hydroxylase+as+an+off-target+of+panobinostat&doi=10.1038%2Fnchembio.2185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal profiling reveals phenylalanine hydroxylase as an off-target of panobinostat</span></div><div class="casAuthors">Becher, Isabelle; Werner, Thilo; Doce, Carola; Zaal, Esther A.; Toegel, Ina; Khan, Crystal A.; Rueger, Anne; Muelbaier, Marcel; Salzer, Elsa; Berkers, Celia R.; Fitzpatrick, Paul F.; Bantscheff, Marcus; Savitski, Mikhail M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">908-910</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe a two-dimensional thermal proteome profiling strategy that can be combined with an orthogonal chemoproteomics approach to enable comprehensive target profiling of the marketed histone deacetylase inhibitor panobinostat.  The N-hydroxycinnamide moiety is identified as crit. for potent and tetrahydrobiopterin-competitive inhibition of phenylalanine hydroxylase leading to increases in phenylalanine and decreases in tyrosine levels.  These findings provide a rationale for adverse clin. observations and suggest repurposing of the drug for treatment of tyrosinemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ75ESSwULbbVg90H21EOLACvtfcHk0libHqGo2HASAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOmsLrJ&md5=f08addd4f25749aebd827dd99edf157b</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2185%26sid%3Dliteratum%253Aachs%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DZaal%26aufirst%3DE.%2BA.%26aulast%3DTogel%26aufirst%3DI.%26aulast%3DKhan%26aufirst%3DC.%2BA.%26aulast%3DRueger%26aufirst%3DA.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DSalzer%26aufirst%3DE.%26aulast%3DBerkers%26aufirst%3DC.%2BR.%26aulast%3DFitzpatrick%26aufirst%3DP.%2BF.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26atitle%3DThermal%2520profiling%2520reveals%2520phenylalanine%2520hydroxylase%2520as%2520an%2520off-target%2520of%2520panobinostat%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D908%26epage%3D910%26doi%3D10.1038%2Fnchembio.2185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St. Onge, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proctor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nile, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spagnuolo, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jitkova, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gronda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung-Ong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spear, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlecht, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duby, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heisler, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surendra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbanus, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebbia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lissina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeghouf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherfils, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trimble, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimmer, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankaitis, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nislow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaever, G.</span></span> <span> </span><span class="NLM_article-title">Mapping the cellular response to small molecules using chemogenomic fitness signatures</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>344</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1126/science.1250217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1250217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24723613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlslSmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2014&pages=208-211&author=A.+Y.+Leeauthor=R.+P.+St.+Ongeauthor=M.+J.+Proctorauthor=I.+M.+Wallaceauthor=A.+H.+Nileauthor=P.+A.+Spagnuoloauthor=Y.+Jitkovaauthor=M.+Grondaauthor=Y.+Wuauthor=M.+K.+Kimauthor=K.+Cheung-Ongauthor=N.+P.+Torresauthor=E.+D.+Spearauthor=M.+K.+Hanauthor=U.+Schlechtauthor=S.+Sureshauthor=G.+Dubyauthor=L.+E.+Heislerauthor=A.+Surendraauthor=E.+Fungauthor=M.+L.+Urbanusauthor=M.+Gebbiaauthor=E.+Lissinaauthor=M.+Mirandaauthor=J.+H.+Chiangauthor=A.+M.+Aparicioauthor=M.+Zeghoufauthor=R.+W.+Davisauthor=J.+Cherfilsauthor=M.+Boutryauthor=C.+A.+Kaiserauthor=C.+L.+Cumminsauthor=W.+S.+Trimbleauthor=G.+W.+Brownauthor=A.+D.+Schimmerauthor=V.+A.+Bankaitisauthor=C.+Nislowauthor=G.+D.+Baderauthor=G.+Giaever&title=Mapping+the+cellular+response+to+small+molecules+using+chemogenomic+fitness+signatures&doi=10.1126%2Fscience.1250217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping the Cellular Response to Small Molecules Using Chemogenomic Fitness Signatures</span></div><div class="casAuthors">Lee, Anna Y.; St. Onge, Robert P.; Proctor, Michael J.; Wallace, Iain M.; Nile, Aaron H.; Spagnuolo, Paul A.; Jitkova, Yulia; Gronda, Marcela; Wu, Yan; Kim, Moshe K.; Cheung-Ong, Kahlin; Torres, Nikko P.; Spear, Eric D.; Han, Mitchell K. L.; Schlecht, Ulrich; Suresh, Sundari; Duby, Geoffrey; Heisler, Lawrence E.; Surendra, Anuradha; Fung, Eula; Urbanus, Malene L.; Gebbia, Marinella; Lissina, Elena; Miranda, Molly; Chiang, Jennifer H.; Aparicio, Ana Maria; Zeghouf, Mahel; Davis, Ronald W.; Cherfils, Jacqueline; Boutry, Marc; Kaiser, Chris A.; Cummins, Carolyn L.; Trimble, William S.; Brown, Grant W.; Schimmer, Aaron D.; Bankaitis, Vytas A.; Nislow, Corey; Bader, Gary D.; Giaever, Guri</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">6180</span>),
    <span class="NLM_cas:pages">208-211</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Genome-wide characterization of the in vivo cellular response to perturbation is fundamental to understanding how cells survive stress.  Identifying the proteins and pathways perturbed by small mols. affects biol. and medicine by revealing the mechanisms of drug action.  We used a yeast chemogenomics platform that quantifies the requirement for each gene for resistance to a compd. in vivo to profile 3250 small mols. in a systematic and unbiased manner.  We identified 317 compds. that specifically perturb the function of 121 genes and characterized the mechanism of specific compds.  Global anal. revealed that the cellular response to small mols. is limited and described by a network of 45 major chemogenomic signatures.  Our results provide a resource for the discovery of functional interactions among genes, chems., and biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo3_kXSDgt07Vg90H21EOLACvtfcHk0ljyt1JSquQIvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlslSmtbw%253D&md5=543c8d40d54c94f789521ed29859d403</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1126%2Fscience.1250217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1250217%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DA.%2BY.%26aulast%3DSt.%2BOnge%26aufirst%3DR.%2BP.%26aulast%3DProctor%26aufirst%3DM.%2BJ.%26aulast%3DWallace%26aufirst%3DI.%2BM.%26aulast%3DNile%26aufirst%3DA.%2BH.%26aulast%3DSpagnuolo%26aufirst%3DP.%2BA.%26aulast%3DJitkova%26aufirst%3DY.%26aulast%3DGronda%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DM.%2BK.%26aulast%3DCheung-Ong%26aufirst%3DK.%26aulast%3DTorres%26aufirst%3DN.%2BP.%26aulast%3DSpear%26aufirst%3DE.%2BD.%26aulast%3DHan%26aufirst%3DM.%2BK.%26aulast%3DSchlecht%26aufirst%3DU.%26aulast%3DSuresh%26aufirst%3DS.%26aulast%3DDuby%26aufirst%3DG.%26aulast%3DHeisler%26aufirst%3DL.%2BE.%26aulast%3DSurendra%26aufirst%3DA.%26aulast%3DFung%26aufirst%3DE.%26aulast%3DUrbanus%26aufirst%3DM.%2BL.%26aulast%3DGebbia%26aufirst%3DM.%26aulast%3DLissina%26aufirst%3DE.%26aulast%3DMiranda%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DJ.%2BH.%26aulast%3DAparicio%26aufirst%3DA.%2BM.%26aulast%3DZeghouf%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DR.%2BW.%26aulast%3DCherfils%26aufirst%3DJ.%26aulast%3DBoutry%26aufirst%3DM.%26aulast%3DKaiser%26aufirst%3DC.%2BA.%26aulast%3DCummins%26aufirst%3DC.%2BL.%26aulast%3DTrimble%26aufirst%3DW.%2BS.%26aulast%3DBrown%26aufirst%3DG.%2BW.%26aulast%3DSchimmer%26aufirst%3DA.%2BD.%26aulast%3DBankaitis%26aufirst%3DV.%2BA.%26aulast%3DNislow%26aufirst%3DC.%26aulast%3DBader%26aufirst%3DG.%2BD.%26aulast%3DGiaever%26aufirst%3DG.%26atitle%3DMapping%2520the%2520cellular%2520response%2520to%2520small%2520molecules%2520using%2520chemogenomic%2520fitness%2520signatures%26jtitle%3DScience%26date%3D2014%26volume%3D344%26spage%3D208%26epage%3D211%26doi%3D10.1126%2Fscience.1250217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span> <span> </span><span class="NLM_article-title">Network-assisted target identification for haploinsufficiency and homozygous profiling screens</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e1005553</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1371%2Fjournal.pcbi.1005553" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=e1005553&author=S.+Wangauthor=J.+Peng&title=Network-assisted+target+identification+for+haploinsufficiency+and+homozygous+profiling+screens&doi=10.1371%2Fjournal.pcbi.1005553"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005553%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DJ.%26atitle%3DNetwork-assisted%2520target%2520identification%2520for%2520haploinsufficiency%2520and%2520homozygous%2520profiling%2520screens%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3De1005553%26doi%3D10.1371%2Fjournal.pcbi.1005553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, H. J.</span></span> <span> </span><span class="NLM_article-title">Advances in identification and validation of protein targets of natural products without chemical modification</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">719</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1039/C5NP00107B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2FC5NP00107B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26964663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFegsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=719-730&author=J.+Changauthor=Y.+Kimauthor=H.+J.+Kwon&title=Advances+in+identification+and+validation+of+protein+targets+of+natural+products+without+chemical+modification&doi=10.1039%2FC5NP00107B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in identification and validation of protein targets of natural products without chemical modification</span></div><div class="casAuthors">Chang, J.; Kim, Y.; Kwon, H. J.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">719-730</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covering: up to Feb. 2016Identification of the target proteins of natural products is pivotal to understanding the mechanisms of action to develop natural products for use as mol. probes and potential therapeutic drugs.  Affinity chromatog. of immobilized natural products has been conventionally used to identify target proteins, and has yielded good results.  However, this method has limitations, in that labeling or tagging for immobilization and affinity purifn. often result in reduced or altered activity of the natural product.  New strategies have recently been developed and applied to identify the target proteins of natural products and synthetic small mols. without chem. modification of the natural product.  These direct and indirect methods for target identification of label-free natural products include drug affinity responsive target stability (DARTS), stability of proteins from rates of oxidn. (SPROX), cellular thermal shift assay (CETSA), thermal proteome profiling (TPP), and bioinformatics-based anal. of connectivity.  This review focuses on and reports case studies of the latest advances in target protein identification methods for label-free natural products.  The integration of newly developed technologies will provide new insights and highlight the value of natural products for use as biol. probes and new drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF67ZpeaXLxLVg90H21EOLACvtfcHk0lguamcmWpNJ6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFegsL0%253D&md5=7cdabe40955c8c15971d68c29c92b99e</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1039%2FC5NP00107B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5NP00107B%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DKwon%26aufirst%3DH.%2BJ.%26atitle%3DAdvances%2520in%2520identification%2520and%2520validation%2520of%2520protein%2520targets%2520of%2520natural%2520products%2520without%2520chemical%2520modification%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2016%26volume%3D33%26spage%3D719%26epage%3D730%26doi%3D10.1039%2FC5NP00107B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoghegan, K. F.</span></span> <span> </span><span class="NLM_article-title">Emerging methods in chemoproteomics with relevance to drug discovery</span>. <i>Methods Mol. Biol. (N. Y., NY, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1513</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-6539-7_2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2F978-1-4939-6539-7_2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27807827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1513&publication_year=2017&pages=11-22&author=C.+Nguyenauthor=G.+M.+Westauthor=K.+F.+Geoghegan&title=Emerging+methods+in+chemoproteomics+with+relevance+to+drug+discovery&doi=10.1007%2F978-1-4939-6539-7_2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging methods in chemoproteomics with relevance to drug discovery</span></div><div class="casAuthors">Nguyen, Chuong; West, Graham M.; Geoghegan, Kieran F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1513</span>
        (<span class="NLM_cas:issue">Cancer Gene Networks</span>),
    <span class="NLM_cas:pages">11-22</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A powerful interplay exists between the recognition of gene families, sensitive techniques in proteomics, and the interrogation of protein function using chem. probes.  The most prominent methods, such as affinity capture, activity-based protein profiling and photoaffinity labeling, are extensively reviewed in the literature.  Here we briefly review addnl. methods developed in the past 15 years.  These include "stability proteomics" methods such as proteomically analyzed cellular thermal shift assays and the use of chem. oxidn. as a probe of structure, the use of multiple bead-linked kinase inhibitors to analyze inhibitor specificities, and advances in the use of proteolysis-targeting chimeras for selective protein elimination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjKkAKWHyj8bVg90H21EOLACvtfcHk0lguamcmWpNJ6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCksrw%253D&md5=e6db46559bc807e432ffa20101edd1ff</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-6539-7_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-6539-7_2%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DGeoghegan%26aufirst%3DK.%2BF.%26atitle%3DEmerging%2520methods%2520in%2520chemoproteomics%2520with%2520relevance%2520to%2520drug%2520discovery%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2017%26volume%3D1513%26spage%3D11%26epage%3D22%26doi%3D10.1007%2F978-1-4939-6539-7_2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rempel, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, M. L.</span></span> <span> </span><span class="NLM_article-title">Quantification of protein-ligand interactions by mass spectrometry, titration, and H/D exchange: PLIMSTEX</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">5252</span>– <span class="NLM_lpage">5253</span>, <span class="refDoi"> DOI: 10.1021/ja029460d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja029460d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFagtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=5252-5253&author=M.+M.+Zhuauthor=D.+L.+Rempelauthor=Z.+Duauthor=M.+L.+Gross&title=Quantification+of+protein-ligand+interactions+by+mass+spectrometry%2C+titration%2C+and+H%2FD+exchange%3A+PLIMSTEX&doi=10.1021%2Fja029460d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of protein-ligand interactions by mass spectrometry, titration, and H/D exchange: PLIMSTEX</span></div><div class="casAuthors">Zhu, Mei M.; Rempel, Don L.; Du, Zhaohui; Gross, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5252-5253</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand binding and the concomitant conformational change in the protein are of crucial importance in biophysics and drug design.  We report a novel method to quantify protein-ligand interactions in soln. by mass spectrometry, titrn., and H/D exchange (PLIMSTEX).  The approach can det. the conformational change, binding stoichiometry, and affinity in protein-ligand interactions including those that involve small mols., metal ions, and peptides.  Binding consts. obtained by PLIMSTEX for four model protein-ligand systems agree with K values measured by conventional methods.  At higher protein concn., the method can be used to det. quickly the binding stoichiometry and possibly the purity of proteins.  Taking advantage of concg. the protein on-column and desalting, we are able to use different concns. of proteins, buffer systems, salts, and pH in the exchange protocol.  High picomole quantities of proteins are sufficient, offering significantly better sensitivity than that of NMR and x-ray crystallog.  Automation could make PLIMSTEX a high throughput method for library screening, drug discovery, and proteomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppgAt2vdlBSLVg90H21EOLACvtfcHk0lguamcmWpNJ6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFagtro%253D&md5=6f30e9b7ae2c07ecd3be549b95ee4c68</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Fja029460d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja029460d%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DM.%2BM.%26aulast%3DRempel%26aufirst%3DD.%2BL.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DGross%26aufirst%3DM.%2BL.%26atitle%3DQuantification%2520of%2520protein-ligand%2520interactions%2520by%2520mass%2520spectrometry%252C%2520titration%252C%2520and%2520H%252FD%2520exchange%253A%2520PLIMSTEX%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D5252%26epage%3D5253%26doi%3D10.1021%2Fja029460d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani-Yoshihara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakariya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yogo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, T.</span></span> <span> </span><span class="NLM_article-title">CETSA quantitatively verifies in vivo target engagement of novel RIPK1 inhibitors in various biospecimens</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13000</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12513-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fs41598-017-12513-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=29026104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC1M7gtFSmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=13000&author=T.+Ishiiauthor=T.+Okaiauthor=M.+Iwatani-Yoshiharaauthor=M.+Mochizukiauthor=S.+Unnoauthor=M.+Kunoauthor=M.+Yoshikawaauthor=S.+Shibataauthor=M.+Nakakariyaauthor=T.+Yogoauthor=T.+Kawamoto&title=CETSA+quantitatively+verifies+in+vivo+target+engagement+of+novel+RIPK1+inhibitors+in+various+biospecimens&doi=10.1038%2Fs41598-017-12513-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">CETSA quantitatively verifies in vivo target engagement of novel RIPK1 inhibitors in various biospecimens</span></div><div class="casAuthors">Ishii Tsuyoshi; Iwatani-Yoshihara Misa; Mochizuki Manabu; Kawamoto Tomohiro; Okai Takuro; Unno Satoko; Kuno Masako; Yoshikawa Masato; Yogo Takatoshi; Shibata Sachio; Nakakariya Masanori</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13000</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The proof of target engagement (TE) is a key element for evaluating potential investment in drug development.  The cellular thermal shift assay (CETSA) is expected to facilitate direct measurement of intracellular TE at all stages of drug development.  However, there have been no reports of applying this technology to comprehensive animal and clinical studies.  This report demonstrates that CETSA can not only quantitatively evaluate the drug-TE in mouse peripheral blood, but also confirm TE in animal tissues exemplified by using the receptor interacting protein 1 kinase (RIPK1) lead compound we have developed.  Our established semi-automated system allows evaluation of the structure-activity relationship using native RIPK1 in culture cell lines, and also enables estimation of drug occupancy ratio in mouse peripheral blood mononuclear cells.  Moreover, optimized tissue homogenisation enables monitoring of the in vivo drug-TE in spleen and brain.  Our results indicate that CETSA methodology will provide an efficient tool for preclinical and clinical drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSVxZ2bIR2iP3O4gDb7JggCfW6udTcc2eZFW3JNpY_t27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7gtFSmsw%253D%253D&md5=ae576418ac1d2ab3788ea241f74f59de</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12513-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12513-1%26sid%3Dliteratum%253Aachs%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DOkai%26aufirst%3DT.%26aulast%3DIwatani-Yoshihara%26aufirst%3DM.%26aulast%3DMochizuki%26aufirst%3DM.%26aulast%3DUnno%26aufirst%3DS.%26aulast%3DKuno%26aufirst%3DM.%26aulast%3DYoshikawa%26aufirst%3DM.%26aulast%3DShibata%26aufirst%3DS.%26aulast%3DNakakariya%26aufirst%3DM.%26aulast%3DYogo%26aufirst%3DT.%26aulast%3DKawamoto%26aufirst%3DT.%26atitle%3DCETSA%2520quantitatively%2520verifies%2520in%2520vivo%2520target%2520engagement%2520of%2520novel%2520RIPK1%2520inhibitors%2520in%2520various%2520biospecimens%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D13000%26doi%3D10.1038%2Fs41598-017-12513-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay: a novel biophysical assay for in situ drug target engagement and mechanistic biomarker studies</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1146%2Fannurev-pharmtox-010715-103715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26566155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCqs7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=141-161&author=D.+Martinez+Molinaauthor=P.+Nordlund&title=The+cellular+thermal+shift+assay%3A+a+novel+biophysical+assay+for+in+situ+drug+target+engagement+and+mechanistic+biomarker+studies&doi=10.1146%2Fannurev-pharmtox-010715-103715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">The Cellular Thermal Shift Assay: A Novel Biophysical Assay for In Situ Drug Target Engagement and Mechanistic Biomarker Studies</span></div><div class="casAuthors">Martinez Molina, Daniel; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-161</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A drug must engage its intended target to achieve its therapeutic effect.  However, conclusively measuring target engagement (TE) in situ is challenging.  This complicates preclin. development and is considered a key factor in the high rate of attrition in clin. trials.  Here, we discuss a recently developed, label-free, biophys. assay, the cellular thermal shift assay (CETSA), which facilitates the direct assessment of TE in cells and tissues at various stages of drug development.  CETSA also reveals biochem. events downstream of drug binding and therefore provides a promising means of establishing mechanistic biomarkers.  The implementation of proteome-wide CETSA using quant. mass spectrometry represents a novel strategy for defining off-target toxicity and polypharmacol. and for identifying downstream mechanistic biomarkers.  The first year of CETSA applications in the literature has focused on TE studies in cell culture systems and has confirmed the broad applicability of CETSA to many different target families.  The next phase of CETSA applications will likely encompass comprehensive animal and patient studies, and CETSA will likely serve as a very valuable tool in many stages of preclin. and clin. drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCOdmURGPcTbVg90H21EOLACvtfcHk0lgN3ZDMlTURgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCqs7nI&md5=092ac50d54d92ef9746a90af8c634e94</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103715%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%253A%2520a%2520novel%2520biophysical%2520assay%2520for%2520in%2520situ%2520drug%2520target%2520engagement%2520and%2520mechanistic%2520biomarker%2520studies%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26spage%3D141%26epage%3D161%26doi%3D10.1146%2Fannurev-pharmtox-010715-103715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waszkiewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallal, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creasy, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbert, M. A.</span></span> <span> </span><span class="NLM_article-title">A high-throughput dose-response cellular thermal shift assay for rapid screening of drug target engagement in living cells, exemplified using SMYD3 and IDO1</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1177/2472555217732014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F2472555217732014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28957646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=34-46&author=D.+E.+McNultyauthor=W.+G.+Bonnetteauthor=H.+Qiauthor=L.+Wangauthor=T.+F.+Hoauthor=A.+Waszkiewiczauthor=L.+A.+Kallalauthor=R.+P.+Nagarajanauthor=M.+Sternauthor=A.+M.+Quinnauthor=C.+L.+Creasyauthor=D.+S.+Suauthor=A.+P.+Gravesauthor=R.+S.+Annanauthor=S.+M.+Sweitzerauthor=M.+A.+Holbert&title=A+high-throughput+dose-response+cellular+thermal+shift+assay+for+rapid+screening+of+drug+target+engagement+in+living+cells%2C+exemplified+using+SMYD3+and+IDO1&doi=10.1177%2F2472555217732014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">A High-Throughput Dose-Response Cellular Thermal Shift Assay for Rapid Screening of Drug Target Engagement in Living Cells, Exemplified Using SMYD3 and IDO1</span></div><div class="casAuthors">McNulty, Dean E.; Bonnette, William G.; Qi, Hongwei; Wang, Liping; Ho, Thau F.; Waszkiewicz, Anna; Kallal, Lorena A.; Nagarajan, Raman P.; Stern, Melissa; Quinn, Amy M.; Creasy, Caretha L.; Su, Dai-Shi; Graves, Alan P.; Annan, Roland S.; Sweitzer, Sharon M.; Holbert, Marc A.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-46</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A persistent problem in early small-mol. drug discovery is the frequent lack of rank-order correlation between biochem. potencies derived from initial screens using purified proteins and the diminished potency and efficacy obsd. in subsequent disease-relevant cellular phenotypic assays.  The introduction of the cellular thermal shift assay (CETSA) has bridged this gap by enabling assessment of drug target engagement directly in live cells based on ligand-induced changes in protein thermal stability.  Initial success in applying CETSA across multiple drug target classes motivated our investigation into replacing the low-throughput, manually intensive Western blot readout with a quant., automated higher-throughput assay that would provide sufficient capacity to use CETSA as a primary hit qualification strategy.  We introduce a high-throughput dose-response cellular thermal shift assay (HTDR-CETSA), a single-pot homogenous assay adapted for high-d. microtiter plate format.  The assay features titratable BacMam expression of full-length target proteins fused to the DiscoverX 42 amino acid ePL tag in HeLa suspension cells, facilitating enzyme fragment complementation-based chemiluminescent quantification of ligand-stabilized sol. protein.  This simplified format can accommodate detn. of full-dose CETSA curves for hundreds of individual compds./analyst/day in replicates.  HTDR-CETSA data generated for substrate site and alternate binding mode inhibitors of the histone-lysine N-methyltransferase SMYD3 in HeLa suspension cells demonstrate excellent correlation with rank-order potencies obsd. in cellular mechanistic assays and direct translation to target engagement of endogenous Smyd3 in cancer-relevant cell lines.  We envision this workflow to be generically applicable to HTDR-CETSA screening spanning a wide variety of sol. intracellular protein target classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOf7ifeefGGbVg90H21EOLACvtfcHk0lgN3ZDMlTURgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKksrY%253D&md5=24b28b2bec845c423f4a3c446974f002</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1177%2F2472555217732014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217732014%26sid%3Dliteratum%253Aachs%26aulast%3DMcNulty%26aufirst%3DD.%2BE.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DWaszkiewicz%26aufirst%3DA.%26aulast%3DKallal%26aufirst%3DL.%2BA.%26aulast%3DNagarajan%26aufirst%3DR.%2BP.%26aulast%3DStern%26aufirst%3DM.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DSu%26aufirst%3DD.%2BS.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DAnnan%26aufirst%3DR.%2BS.%26aulast%3DSweitzer%26aufirst%3DS.%2BM.%26aulast%3DHolbert%26aufirst%3DM.%2BA.%26atitle%3DA%2520high-throughput%2520dose-response%2520cellular%2520thermal%2520shift%2520assay%2520for%2520rapid%2520screening%2520of%2520drug%2520target%2520engagement%2520in%2520living%2520cells%252C%2520exemplified%2520using%2520SMYD3%2520and%2520IDO1%26jtitle%3DSLAS%2520Discov%26date%3D2018%26volume%3D23%26spage%3D34%26epage%3D46%26doi%3D10.1177%2F2472555217732014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET–a novel BRET platform for the analysis of protein-protein interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+Mendezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET%E2%80%93a+novel+BRET+platform+for+the+analysis+of+protein-protein+interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DMendez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET%25E2%2580%2593a%2520novel%2520BRET%2520platform%2520for%2520the%2520analysis%2520of%2520protein-protein%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niles, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Target engagement and drug residence time can be observed in living cells with BRET</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">10091</span>, <span class="refDoi"> DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26631872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=10091&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.+Q.+Chengauthor=K.+V.+Wood&title=Target+engagement+and+drug+residence+time+can+be+observed+in+living+cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement and drug residence time can be observed in living cells with BRET</span></div><div class="casAuthors">Robers, Matthew B.; Dart, Melanie L.; Woodroofe, Carolyn C.; Zimprich, Chad A.; Kirkland, Thomas A.; Machleidt, Thomas; Kupcho, Kevin R.; Levin, Sergiy; Hartnett, James R.; Zimmerman, Kristopher; Niles, Andrew L.; Ohana, Rachel Friedman; Daniels, Danette L.; Slater, Michael; Wood, Monika G.; Cong, Mei; Cheng, Yi-Qiang; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10091</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The therapeutic action of drugs is predicated on their phys. engagement with cellular targets.  Here we describe a broadly applicable method using bioluminescence resonance energy transfer (BRET) to reveal the binding characteristics of a drug with selected targets within intact cells.  Cell-permeable fluorescent tracers are used in a competitive binding format to quantify drug engagement with the target proteins fused to Nanoluc luciferase.  The approach enabled us to profile isoenzyme-specific engagement and binding kinetics for a panel of histone deacetylase (HDAC) inhibitors.  Our anal. was directed particularly to the clin. approved prodrug FK228 (Istodax/Romidepsin) because of its unique and largely unexplained mechanism of sustained intracellular action.  Anal. of the binding kinetics by BRET revealed remarkably long intracellular residence times for FK228 at HDAC1, explaining the protracted intracellular behavior of this prodrug.  Our results demonstrate a novel application of BRET for assessing target engagement within the complex milieu of the intracellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFFk0LNiL_LVg90H21EOLACvtfcHk0li8kRWBtKwBiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO&md5=723cc5dd7d229b1115da4c2c9d75c913</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.%2BQ.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520engagement%2520and%2520drug%2520residence%2520time%2520can%2520be%2520observed%2520in%2520living%2520cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D10091%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebakken, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddle, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frazee, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliason, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichling, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, K. W.</span></span> <span> </span><span class="NLM_article-title">Development and applications of a broad-coverage, TR-FRET-based kinase binding assay platform</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1177/1087057109339207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F1087057109339207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=19564447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=924-935&author=C.+S.+Lebakkenauthor=S.+M.+Riddleauthor=U.+Singhauthor=W.+J.+Frazeeauthor=H.+C.+Eliasonauthor=Y.+Gaoauthor=L.+J.+Reichlingauthor=B.+D.+Marksauthor=K.+W.+Vogel&title=Development+and+applications+of+a+broad-coverage%2C+TR-FRET-based+kinase+binding+assay+platform&doi=10.1177%2F1087057109339207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Development and applications of a broad-coverage, TR-FRET-based kinase binding assay platform</span></div><div class="casAuthors">Lebakken, Connie S.; Riddle, Steven M.; Singh, Upinder; Frazee, W. Jack; Eliason, Hildegard C.; Gao, Yi; Reichling, Laurie J.; Marks, Bryan D.; Vogel, Kurt W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">924-935</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The expansion of kinase assay technologies over the past decade has mirrored the growing interest in kinases as drug targets.  As a result, there is no shortage of convenient, fluorescence-based methods available to assay targets that span the kinome.  The authors recently reported on the development of a non-activity-based assay to characterize kinase inhibitors that depended on displacement of an Alexa Fluor 647 conjugate of staurosporine (a "tracer") from a particular kinase.  Kinase inhibitors were characterized by a change in fluorescence lifetime of the tracer when it was bound to a kinase relative to when it was displaced by an inhibitor.  Here, the authors report on improvements to this strategy by reconfiguring the assay in a time-resolved fluorescence resonance energy transfer (TR-FRET) format that simplifies instrumentation requirements and allows for the use of a substantially lower concn. of kinase than was required in the fluorescence-lifetime-based format.  The authors use this new assay to demonstrate several aspects of the binding assay format that are advantageous relative to traditional activity-based assays.  The TR-FRET binding format facilitates the assay of compds. against low-activity kinases, allows for the characterization of type II kinase inhibitors either using nonactivated kinases or by monitoring compd. potency over time, and ensures that the signal being detected is specific to the kinase of interest and not a contaminating kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqPbcMF0-PD7Vg90H21EOLACvtfcHk0li8kRWBtKwBiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLnM&md5=f8df51adf90d821e480cf706213cc096</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1177%2F1087057109339207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109339207%26sid%3Dliteratum%253Aachs%26aulast%3DLebakken%26aufirst%3DC.%2BS.%26aulast%3DRiddle%26aufirst%3DS.%2BM.%26aulast%3DSingh%26aufirst%3DU.%26aulast%3DFrazee%26aufirst%3DW.%2BJ.%26aulast%3DEliason%26aufirst%3DH.%2BC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DReichling%26aufirst%3DL.%2BJ.%26aulast%3DMarks%26aufirst%3DB.%2BD.%26aulast%3DVogel%26aufirst%3DK.%2BW.%26atitle%3DDevelopment%2520and%2520applications%2520of%2520a%2520broad-coverage%252C%2520TR-FRET-based%2520kinase%2520binding%2520assay%2520platform%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D924%26epage%3D935%26doi%3D10.1177%2F1087057109339207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binkowski, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valley, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidugiris, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benink, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggers, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaubert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Encell, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Engineered luciferase reporter from a deep sea shrimp utilizing a novel imidazopyrazinone substrate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1848</span>– <span class="NLM_lpage">1857</span>, <span class="refDoi"> DOI: 10.1021/cb3002478</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3002478" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Smu7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1848-1857&author=M.+P.+Hallauthor=J.+Unchauthor=B.+F.+Binkowskiauthor=M.+P.+Valleyauthor=B.+L.+Butlerauthor=M.+G.+Woodauthor=P.+Ottoauthor=K.+Zimmermanauthor=G.+Vidugirisauthor=T.+Machleidtauthor=M.+B.+Robersauthor=H.+A.+Beninkauthor=C.+T.+Eggersauthor=M.+R.+Slaterauthor=P.+L.+Meisenheimerauthor=D.+H.+Klaubertauthor=F.+Fanauthor=L.+P.+Encellauthor=K.+V.+Wood&title=Engineered+luciferase+reporter+from+a+deep+sea+shrimp+utilizing+a+novel+imidazopyrazinone+substrate&doi=10.1021%2Fcb3002478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Engineered Luciferase Reporter from a Deep Sea Shrimp Utilizing a Novel Imidazopyrazinone Substrate</span></div><div class="casAuthors">Hall, Mary P.; Unch, James; Binkowski, Brock F.; Valley, Michael P.; Butler, Braeden L.; Wood, Monika G.; Otto, Paul; Zimmerman, Kristopher; Vidugiris, Gediminas; Machleidt, Thomas; Robers, Matthew B.; Benink, Helene A.; Eggers, Christopher T.; Slater, Michael R.; Meisenheimer, Poncho L.; Klaubert, Dieter H.; Fan, Frank; Encell, Lance P.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1848-1857</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bioluminescence methodologies have been extraordinarily useful due to their high sensitivity, broad dynamic range, and operational simplicity.  These capabilities have been realized largely through incremental adaptations of native enzymes and substrates, originating from luminous organisms of diverse evolutionary lineages.  We engineered both an enzyme and substrate in combination to create a novel bioluminescence system capable of more efficient light emission with superior biochem. and phys. characteristics.  Using a small luciferase subunit (19 kDa) from the deep sea shrimp Oplophorus gracilirostris, we have improved luminescence expression in mammalian cells ∼2.5 million-fold by merging optimization of protein structure with development of a novel imidazopyrazinone substrate (furimazine).  The new luciferase, NanoLuc, produces glow-type luminescence (signal half-life >2 h) with a specific activity ∼150-fold greater than that of either firefly (Photinus pyralis) or Renilla luciferases similarly configured for glow-type assays.  In mammalian cells, NanoLuc shows no evidence of post-translational modifications or subcellular partitioning.  The enzyme exhibits high phys. stability, retaining activity with incubation up to 55 °C or in culture medium for >15 h at 37 °C.  As a genetic reporter, NanoLuc may be configured for high sensitivity or for response dynamics by appending a degrdn. sequence to reduce intracellular accumulation.  Appending a signal sequence allows NanoLuc to be exported to the culture medium, where reporter expression can be measured without cell lysis.  Fusion onto other proteins allows luminescent assays of their metab. or localization within cells.  Reporter quantitation is achievable even at very low expression levels to facilitate more reliable coupling with endogenous cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlTT1m2z_bZLVg90H21EOLACvtfcHk0ljZMBELixEp1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Smu7%252FN&md5=0581a7e44935671471577ade1ff538eb</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Fcb3002478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3002478%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DM.%2BP.%26aulast%3DUnch%26aufirst%3DJ.%26aulast%3DBinkowski%26aufirst%3DB.%2BF.%26aulast%3DValley%26aufirst%3DM.%2BP.%26aulast%3DButler%26aufirst%3DB.%2BL.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DVidugiris%26aufirst%3DG.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DBenink%26aufirst%3DH.%2BA.%26aulast%3DEggers%26aufirst%3DC.%2BT.%26aulast%3DSlater%26aufirst%3DM.%2BR.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DKlaubert%26aufirst%3DD.%2BH.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DEncell%26aufirst%3DL.%2BP.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DEngineered%2520luciferase%2520reporter%2520from%2520a%2520deep%2520sea%2520shrimp%2520utilizing%2520a%2520novel%2520imidazopyrazinone%2520substrate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1848%26epage%3D1857%26doi%3D10.1021%2Fcb3002478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanyai, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">See, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, E. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfleger, K. D. G.</span></span> <span> </span><span class="NLM_article-title">Using nanoBRET and CRISPR/Cas9 to monitor proximity to a genome-edited protein in real-time</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3187</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-03486-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fs41598-017-03486-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28600500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC1cnmvFSqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=3187&author=C.+W.+Whiteauthor=H.+K.+Vanyaiauthor=H.+B.+Seeauthor=E.+K.+M.+Johnstoneauthor=K.+D.+G.+Pfleger&title=Using+nanoBRET+and+CRISPR%2FCas9+to+monitor+proximity+to+a+genome-edited+protein+in+real-time&doi=10.1038%2Fs41598-017-03486-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Using nanoBRET and CRISPR/Cas9 to monitor proximity to a genome-edited protein in real-time</span></div><div class="casAuthors">White Carl W; See Heng B; Johnstone Elizabeth K M; Pfleger Kevin D G; White Carl W; Vanyai Hannah K; See Heng B; Johnstone Elizabeth K M; Pfleger Kevin D G; Vanyai Hannah K; Pfleger Kevin D G</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3187</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bioluminescence resonance energy transfer (BRET) has been a vital tool for understanding G protein-coupled receptor (GPCR) function.  It has been used to investigate GPCR-protein and/or -ligand interactions as well as GPCR oligomerisation.  However the utility of BRET is limited by the requirement that the fusion proteins, and in particular the donor, need to be exogenously expressed.  To address this, we have used CRISPR/Cas9-mediated homology-directed repair to generate protein-Nanoluciferase (Nluc) fusions under endogenous promotion, thus allowing investigation of proximity between the genome-edited protein and an exogenously expressed protein by BRET.  Here we report BRET monitoring of GPCR-mediated β-arrestin2 recruitment and internalisation where the donor luciferase was under endogenous promotion, in live cells and in real time.  We have investigated the utility of CRISPR/Cas9 genome editing to create genome-edited fusion proteins that can be used as BRET donors and propose that this strategy can be used to overcome the need for exogenous donor expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQGM460n2USOpQHPYJuLg8TfW6udTcc2eYyEUfaxJnNj7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnmvFSqug%253D%253D&md5=dfeeabcf281a05fb5ee429955502dc85</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-03486-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-03486-2%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DC.%2BW.%26aulast%3DVanyai%26aufirst%3DH.%2BK.%26aulast%3DSee%26aufirst%3DH.%2BB.%26aulast%3DJohnstone%26aufirst%3DE.%2BK.%2BM.%26aulast%3DPfleger%26aufirst%3DK.%2BD.%2BG.%26atitle%3DUsing%2520nanoBRET%2520and%2520CRISPR%252FCas9%2520to%2520monitor%2520proximity%2520to%2520a%2520genome-edited%2520protein%2520in%2520real-time%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D3187%26doi%3D10.1038%2Fs41598-017-03486-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggers, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Encell, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binkowski, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">CRISPR-mediated tagging of endogenous proteins with a luminescent peptide</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00549</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00549" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKmur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=467-474&author=M.+K.+Schwinnauthor=T.+Machleidtauthor=K.+Zimmermanauthor=C.+T.+Eggersauthor=A.+S.+Dixonauthor=R.+Hurstauthor=M.+P.+Hallauthor=L.+P.+Encellauthor=B.+F.+Binkowskiauthor=K.+V.+Wood&title=CRISPR-mediated+tagging+of+endogenous+proteins+with+a+luminescent+peptide&doi=10.1021%2Facschembio.7b00549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">CRISPR-Mediated Tagging of Endogenous Proteins with a Luminescent Peptide</span></div><div class="casAuthors">Schwinn, Marie K.; Machleidt, Thomas; Zimmerman, Kris; Eggers, Christopher T.; Dixon, Andrew S.; Hurst, Robin; Hall, Mary P.; Encell, Lance P.; Binkowski, Brock F.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intracellular signaling pathways are mediated by changes in protein abundance and post-translational modifications.  A common approach for investigating signaling mechanisms and the effects induced by synthetic compds. is through overexpression of recombinant reporter genes.  Genome editing with CRISPR/Cas9 offers a means to better preserve native biol. by appending reporters directly onto the endogenous genes.  An optimal reporter for this purpose would be small to negligibly influence intracellular processes, be readily linked to the endogenous genes with minimal exptl. effort, and be sensitive enough to detect low expressing proteins.  HiBiT is a 1.3 kDa peptide (11 amino acids) capable of producing bright and quant. luminescence through high affinity complementation (KD = 700 pM) with an 18 kDa subunit derived from NanoLuc (LgBiT).  Using CRISPR/Cas9, we demonstrate that HiBiT can be rapidly and efficiently integrated into the genome to serve as a reporter tag for endogenous proteins.  Without requiring clonal isolation of the edited cells, we were able to quantify changes in abundance of the hypoxia inducible factor 1A (HIF1α) and several of its downstream transcriptional targets in response to various stimuli.  In combination with fluorescent antibodies, we further used HiBiT to directly correlate HIF1α levels with the hydroxyproline modification that mediates its degrdn.  These results demonstrate the ability to efficiently tag endogenous proteins with a small luminescent peptide, allowing sensitive quantitation of the response dynamics in their regulated expression and covalent modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzCZoXuuYB37Vg90H21EOLACvtfcHk0ljZMBELixEp1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKmur7I&md5=0d02aa711b02f68a5a54b72b968f6292</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00549%26sid%3Dliteratum%253Aachs%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DEggers%26aufirst%3DC.%2BT.%26aulast%3DDixon%26aufirst%3DA.%2BS.%26aulast%3DHurst%26aufirst%3DR.%26aulast%3DHall%26aufirst%3DM.%2BP.%26aulast%3DEncell%26aufirst%3DL.%2BP.%26aulast%3DBinkowski%26aufirst%3DB.%2BF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DCRISPR-mediated%2520tagging%2520of%2520endogenous%2520proteins%2520with%2520a%2520luminescent%2520peptide%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D467%26epage%3D474%26doi%3D10.1021%2Facschembio.7b00549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell Chem Biol</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.+T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+Mullerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0lgJZlfLimNWxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.%2BT.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell%2520Chem%2520Biol%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evanko, D.</span></span> <span> </span><span class="NLM_article-title">Using ’bumps’ and ’holes’ to control protein activity</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1038/nmeth0306-154b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnmeth0306-154b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=16523567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFOrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=154-155&author=D.+Evanko&title=Using+%E2%80%99bumps%E2%80%99+and+%E2%80%99holes%E2%80%99+to+control+protein+activity&doi=10.1038%2Fnmeth0306-154b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Using 'bumps' and 'holes' to control protein activity</span></div><div class="casAuthors">Evanko, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">154-155</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The creation of orthogonal pairs of rapamycin analogs contg. engineered 'bumps' and protein binding partners with complementary 'holes' permits temporal control of multiple proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgaVLc10MT4bVg90H21EOLACvtfcHk0lgJZlfLimNWxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFOrur0%253D&md5=7ff17647d6554a6218f8f8858e9728b4</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1038%2Fnmeth0306-154b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth0306-154b%26sid%3Dliteratum%253Aachs%26aulast%3DEvanko%26aufirst%3DD.%26atitle%3DUsing%2520%25E2%2580%2599bumps%25E2%2580%2599%2520and%2520%25E2%2580%2599holes%25E2%2580%2599%2520to%2520control%2520protein%2520activity%26jtitle%3DNat.%2520Methods%26date%3D2006%26volume%3D3%26spage%3D154%26epage%3D155%26doi%3D10.1038%2Fnmeth0306-154b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deirmengian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Engineering unnatural nucleotide specificity for Rous sarcoma virus tyrosine kinase to uniquely label its direct substrates</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">3565</span>– <span class="NLM_lpage">3570</span>, <span class="refDoi"> DOI: 10.1073/pnas.94.8.3565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.94.8.3565" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=9108016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADyaK2sXis1amtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=3565-3570&author=K.+Shahauthor=Y.+Liuauthor=C.+Deirmengianauthor=K.+M.+Shokat&title=Engineering+unnatural+nucleotide+specificity+for+Rous+sarcoma+virus+tyrosine+kinase+to+uniquely+label+its+direct+substrates&doi=10.1073%2Fpnas.94.8.3565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering unnatural nucleotide specificity for Rous sarcoma virus tyrosine kinase to uniquely label its direct substrates</span></div><div class="casAuthors">Shah, Kavita; Liu, Yi; Deirmengian, Carl; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3565-3570</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein phosphorylation plays a central role in controlling many diverse signal transduction pathways in all cells.  Novel protein kinases are identified at a rapid rate using homol. cloning methods and genetic screens or selections; however identification of the direct substrates of kinases has proven elusive to genetic methods because of the tremendous redundancy and overlapping of substrate specificities among protein kinases.  We describe the development of a protein engineering-based method to identify the direct substrates of the prototypical protein tyrosine kinase v-Src, which controls fibroblast transformation by the Rous sarcoma virus.  To differentiate the substrates of v-Src from all other kinase substrates, we mutated the ATP binding site of v-Src such that the engineered v-Src uniquely accepted an ATP analog.  We show that the engineered v-Src kinase displayed catalytic efficiency with the ATP analog, N6-(cyclopentyl) ATP, which is similar to the wild-type kinase catalytic efficiency with ATP itself.  However, the N6-(cyclopentyl) ATP analog was not accepted by the wild-type kinase.  Furthermore, the engineered v-Src exhibited the same protein target specificity as wild-type v-Src despite the proximity of the re-engineered nucleotide binding site to the phosphoacceptor binding site.  The successful engineering of v-Src's active site to accept a unique nucleotide analog provides a unique handle by which the direct substrates of one kinase (v-Src) can be traced in the presence of any no. of cellular kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDWluj07-54LVg90H21EOLACvtfcHk0lhrdcBcOig7Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1amtb4%253D&md5=ff97000a4e9b1c440f5d40ef784a64e8</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.8.3565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.8.3565%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDeirmengian%26aufirst%3DC.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DEngineering%2520unnatural%2520nucleotide%2520specificity%2520for%2520Rous%2520sarcoma%2520virus%2520tyrosine%2520kinase%2520to%2520uniquely%2520label%2520its%2520direct%2520substrates%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1997%26volume%3D94%26spage%3D3565%26epage%3D3570%26doi%3D10.1073%2Fpnas.94.8.3565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emami-Nemini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourrier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamarque, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinquet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohse, M. J.</span></span> <span> </span><span class="NLM_article-title">Time-resolved fluorescence ligand binding for G protein-coupled receptors</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1307</span>– <span class="NLM_lpage">1320</span>, <span class="refDoi"> DOI: 10.1038/nprot.2013.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnprot.2013.073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1307-1320&author=A.+Emami-Neminiauthor=T.+Rouxauthor=M.+Leblayauthor=E.+Bourrierauthor=L.+Lamarqueauthor=E.+Trinquetauthor=M.+J.+Lohse&title=Time-resolved+fluorescence+ligand+binding+for+G+protein-coupled+receptors&doi=10.1038%2Fnprot.2013.073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2013.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2013.073%26sid%3Dliteratum%253Aachs%26aulast%3DEmami-Nemini%26aufirst%3DA.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DLeblay%26aufirst%3DM.%26aulast%3DBourrier%26aufirst%3DE.%26aulast%3DLamarque%26aufirst%3DL.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26atitle%3DTime-resolved%2520fluorescence%2520ligand%2520binding%2520for%2520G%2520protein-coupled%2520receptors%26jtitle%3DNat.%2520Protoc.%26date%3D2013%26volume%3D8%26spage%3D1307%26epage%3D1320%26doi%3D10.1038%2Fnprot.2013.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsukiji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Ligand-directed tosyl chemistry for in situ native protein labeling and engineering in living systems: from basic properties to applications</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2014.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.cbpa.2014.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25129055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVOhs7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=136-143&author=S.+Tsukijiauthor=I.+Hamachi&title=Ligand-directed+tosyl+chemistry+for+in+situ+native+protein+labeling+and+engineering+in+living+systems%3A+from+basic+properties+to+applications&doi=10.1016%2Fj.cbpa.2014.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-directed tosyl chemistry for in situ native protein labeling and engineering in living systems: from basic properties to applications</span></div><div class="casAuthors">Tsukiji, Shinya; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">136-143</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ability to introduce any chem. probe to any endogenous target protein in its native environment, that is in cells and in vivo, is anticipated to provide various new exciting tools for biol. and biomedical research.  Although still at the prototype stage, the ligand-directed tosyl (LDT) chem. is a novel type of affinity labeling technique that we developed for such a dream.  This chem. allows for modifying native proteins by various chem. probes with high specificity in various biol. settings ranging from in vitro (in test tubes) to in living cells and in vivo.  Since the first report, the list of proteins that are successfully labeled by the LDT chem. has been increasing.  A growing no. of studies have demonstrated its utility to create semisynthetic proteins directly in cellular contexts.  The in situ generated semisynthetic proteins are applicable for various types of anal. and imaging of intracellular biol. processes.  In this review, we summarize the basic properties of the LDT chem. and its applications toward in situ engineering and anal. of native proteins in living systems.  Current limitations and future challenges of this area are also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWpiBaEqh9_7Vg90H21EOLACvtfcHk0lhrdcBcOig7Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVOhs7nK&md5=0e1580b982702ef2074672524209f5e2</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2014.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2014.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DTsukiji%26aufirst%3DS.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DLigand-directed%2520tosyl%2520chemistry%2520for%2520in%2520situ%2520native%2520protein%2520labeling%2520and%2520engineering%2520in%2520living%2520systems%253A%2520from%2520basic%2520properties%2520to%2520applications%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D136%26epage%3D143%26doi%3D10.1016%2Fj.cbpa.2014.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otrocka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundback, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seashore-Ludlow, B.</span></span> <span> </span><span class="NLM_article-title">In situ target engagement studies in adherent cells</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">942</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b01079</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b01079" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFaksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=942&author=H.+Axelssonauthor=H.+Almqvistauthor=M.+Otrockaauthor=M.+Vallinauthor=S.+Lundqvistauthor=P.+Hanssonauthor=U.+Karlssonauthor=T.+Lundbackauthor=B.+Seashore-Ludlow&title=In+situ+target+engagement+studies+in+adherent+cells&doi=10.1021%2Facschembio.7b01079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">In Situ Target Engagement Studies in Adherent Cells</span></div><div class="casAuthors">Axelsson, Hanna; Almqvist, Helena; Otrocka, Magdalena; Vallin, Michaela; Lundqvist, Sara; Hansson, Pia; Karlsson, Ulla; Lundbaeck, Thomas; Seashore-Ludlow, Brinton</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">942-950</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A prerequisite for successful drugs is effective binding of the desired target protein in the complex environment of a living system.  Drug-target engagement has typically been difficult to monitor in physiol. relevant models, and with current methods, esp., while maintaining spatial information.  One recent technique for quantifying drug-target engagement is the cellular thermal shift assay (CETSA), in which ligand-induced protein stabilization is measured after a heat challenge.  Here, we describe a CETSA protocol in live A431 cells for p38α (MAPK14), where remaining sol. protein is detected in situ, using high-content imaging in 384-well, microtiter plates.  We validate this assay concept using a no. of known p38α inhibitors and further demonstrate the potential of this technol. for chem. probe and drug discovery purposes by performing a small pilot screen for novel p38α binders.  Importantly, this protocol creates a workflow that is amenable to adherent cells in their native state and yields spatially resolved target engagement information measurable at the single-cell level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2DNlNjNsMnbVg90H21EOLACvtfcHk0lhrdcBcOig7Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFaksr4%253D&md5=5d050e94828e9e5d6e32a92b70b837e4</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b01079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b01079%26sid%3Dliteratum%253Aachs%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DOtrocka%26aufirst%3DM.%26aulast%3DVallin%26aufirst%3DM.%26aulast%3DLundqvist%26aufirst%3DS.%26aulast%3DHansson%26aufirst%3DP.%26aulast%3DKarlsson%26aufirst%3DU.%26aulast%3DLundback%26aufirst%3DT.%26aulast%3DSeashore-Ludlow%26aufirst%3DB.%26atitle%3DIn%2520situ%2520target%2520engagement%2520studies%2520in%2520adherent%2520cells%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D942%26doi%3D10.1021%2Facschembio.7b01079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinegoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">Bioorthogonal imaging of aurora kinase A in live cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">6598</span>– <span class="NLM_lpage">6603</span>, <span class="refDoi"> DOI: 10.1002/anie.201200994</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201200994" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1Sqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=6598-6603&author=K.+S.+Yangauthor=G.+Budinauthor=T.+Reinerauthor=C.+Vinegoniauthor=R.+Weissleder&title=Bioorthogonal+imaging+of+aurora+kinase+A+in+live+cells&doi=10.1002%2Fanie.201200994"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Bioorthogonal Imaging of Aurora Kinase A in Live Cells</span></div><div class="casAuthors">Yang, Katherine S.; Budin, Ghyslain; Reiner, Thomas; Vinegoni, Claudio; Weissleder, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">6598-6603, S6598/1-S6598/32</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The first use of MLN8054-TCO (13) as an imaging agent for AKA visualization in live cells and as a tool for quantifying cellular AKA expression is reported here.  The authors show that endogenous AKA can be imaged in live cells by way of a two-step bioorthogonal reaction. As an imaging agent, MLN8054-TCO (13) in combination with CFDA-Tz demonstrated superior selectivity, fast reaction kinetics, and a sufficient target-to-back-ground ratio to enable AKA visualization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW_s6BEbgs27Vg90H21EOLACvtfcHk0lh7ch6GDY-kZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1Sqtbk%253D&md5=a620de4622bbf073a0acf092dd778e9b</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1002%2Fanie.201200994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201200994%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DK.%2BS.%26aulast%3DBudin%26aufirst%3DG.%26aulast%3DReiner%26aufirst%3DT.%26aulast%3DVinegoni%26aufirst%3DC.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DBioorthogonal%2520imaging%2520of%2520aurora%2520kinase%2520A%2520in%2520live%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D6598%26epage%3D6603%26doi%3D10.1002%2Fanie.201200994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohler, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubach, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikula, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">Optimized near-IR fluorescent agents for in vivo imaging of Btk expression</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1513</span>– <span class="NLM_lpage">1518</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.5b00152</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.5b00152" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=1513-1518&author=E.+Kimauthor=K.+S.+Yangauthor=R.+H.+Kohlerauthor=J.+M.+Dubachauthor=H.+Mikulaauthor=R.+Weissleder&title=Optimized+near-IR+fluorescent+agents+for+in+vivo+imaging+of+Btk+expression&doi=10.1021%2Facs.bioconjchem.5b00152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Optimized Near-IR Fluorescent Agents for in Vivo Imaging of Btk Expression</span></div><div class="casAuthors">Kim, Eunha; Yang, Katherine S.; Kohler, Rainer H.; Dubach, John M.; Mikula, Hannes; Weissleder, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1513-1518</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bruton's tyrosine kinase (Btk) is intricately involved in anti-apoptotic signaling pathways in cancer and in regulating innate immune response.  A no. of Btk inhibitors are in development for use in treating B-cell malignancies and certain immunol. diseases.  To develop robust companion imaging diagnostics for in vivo use, we set out to explore the effects of red wavelength fluorochrome modifications of two highly potent irreversible Btk inhibitors, Ibrutinib and AVL-292.  Surprisingly, we found that subtle chem. differences in the fluorochrome had considerable effects on target localization.  Based on iterative designs, we developed a single optimized version with superb in vivo imaging characteristics enabling single cell Btk imaging in vivo.  This agent (Ibrutinib-SiR-COOH) is expected to be a valuable chem. tool in deciphering Btk biol. in cancer and host cells in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5fRkToD0SFLVg90H21EOLACvtfcHk0lh7ch6GDY-kZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKrur0%253D&md5=78ad36d0e97044a4bcf239a60f957e35</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.5b00152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.5b00152%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DK.%2BS.%26aulast%3DKohler%26aufirst%3DR.%2BH.%26aulast%3DDubach%26aufirst%3DJ.%2BM.%26aulast%3DMikula%26aufirst%3DH.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DOptimized%2520near-IR%2520fluorescent%2520agents%2520for%2520in%2520vivo%2520imaging%2520of%2520Btk%2520expression%26jtitle%3DBioconjugate%2520Chem.%26date%3D2015%26volume%3D26%26spage%3D1513%26epage%3D1518%26doi%3D10.1021%2Facs.bioconjchem.5b00152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">Bioorthogonal small molecule imaging agents allow single-cell imaging of MET</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e81275</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0081275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1371%2Fjournal.pone.0081275" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e81275&author=E.+Kimauthor=K.+S.+Yangauthor=R.+Weissleder&title=Bioorthogonal+small+molecule+imaging+agents+allow+single-cell+imaging+of+MET&doi=10.1371%2Fjournal.pone.0081275"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0081275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0081275%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DK.%2BS.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DBioorthogonal%2520small%2520molecule%2520imaging%2520agents%2520allow%2520single-cell%2520imaging%2520of%2520MET%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De81275%26doi%3D10.1371%2Fjournal.pone.0081275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meimetis, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilderbrand, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">BODIPY-tetrazine derivatives as superbright bioorthogonal turn-on probes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6917</span>– <span class="NLM_lpage">6920</span>, <span class="refDoi"> DOI: 10.1002/anie.201301100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201301100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=6917-6920&author=J.+C.+Carlsonauthor=L.+G.+Meimetisauthor=S.+A.+Hilderbrandauthor=R.+Weissleder&title=BODIPY-tetrazine+derivatives+as+superbright+bioorthogonal+turn-on+probes&doi=10.1002%2Fanie.201301100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">BODIPY-Tetrazine Derivatives as Superbright Bioorthogonal Turn-on Probes</span></div><div class="casAuthors">Carlson, Jonathan C. T.; Meimetis, Labros G.; Hilderbrand, Scott A.; Weissleder, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">6917-6920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">As an alternative way of designing more efficient turn-on probes, one might consider adapting through-bond energy transfer (TBET) for fluorescence quenching.  With these goals in mind, the authors synthesized a series of new bioorthogonal boron dipyrromethene (BODIPY) tetrazine derivs.  These structures enhance spatial donor-acceptor proximity, provide predictable donor-acceptor transition-dipole orientation, and afford the possibility of accessing alternate modes of fluorescence quenching.  Having advanced the understanding of the quenching mechanism, the authors sought to validate the utility of fluorogenic BODIPY-tetrazines for biol. imaging.  Both extracellular and intracellular trans-cyclooctenol-labeled targets were readily visualized, with excellent signal intensity, very low background, and with no washing steps required after addn. of the dye soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3itwIffX8-LVg90H21EOLACvtfcHk0ljPMEpWm_pzOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCisrs%253D&md5=d12b9d6d6bec2aadf46e8a0138f27951</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1002%2Fanie.201301100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201301100%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DJ.%2BC.%26aulast%3DMeimetis%26aufirst%3DL.%2BG.%26aulast%3DHilderbrand%26aufirst%3DS.%2BA.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DBODIPY-tetrazine%2520derivatives%2520as%2520superbright%2520bioorthogonal%2520turn-on%2520probes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D6917%26epage%3D6920%26doi%3D10.1002%2Fanie.201301100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span> <span> </span><span class="NLM_article-title">A modular probe strategy for drug localization, target identification and target occupancy measurement on single cell level</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2541</span>– <span class="NLM_lpage">2550</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00346</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFais7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2541-2550&author=A.+Rutkowskaauthor=D.+W.+Thomsonauthor=J.+Vappianiauthor=T.+Wernerauthor=K.+M.+Muellerauthor=L.+Dittusauthor=J.+Krauseauthor=M.+Muelbaierauthor=G.+Bergaminiauthor=M.+Bantscheff&title=A+modular+probe+strategy+for+drug+localization%2C+target+identification+and+target+occupancy+measurement+on+single+cell+level&doi=10.1021%2Facschembio.6b00346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">A Modular Probe Strategy for Drug Localization, Target Identification and Target Occupancy Measurement on Single Cell Level</span></div><div class="casAuthors">Rutkowska, Anna; Thomson, Douglas W.; Vappiani, Johanna; Werner, Thilo; Mueller, Katrin M.; Dittus, Lars; Krause, Jana; Muelbaier, Marcel; Bergamini, Giovanna; Bantscheff, Marcus</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2541-2550</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Late stage failures of candidate drug mols. are frequently caused by off-target effects or inefficient target engagement in vivo.  In order to address these fundamental challenges in drug discovery, we developed a modular probe strategy based on bioorthogonal chem. that enables the attachment of multiple reporters to the same probe in cell exts. and live cells.  In a systematic evaluation, we identified the inverse electron demand Diels-Alder reaction between trans-cyclooctene labeled probe mols. and tetrazine-tagged reporters to be the most efficient bioorthogonal reaction for this strategy.  Bioorthogonal biotinylation of the probe allows the identification of drug targets in a chemoproteomics competition binding assay using quant. mass spectrometry.  Attachment of a fluorescent reporter enables monitoring of spatial localization of probes as well as drug-target colocalization studies.  Finally, direct target occupancy of unlabeled drugs can be detd. at single cell resoln. by competitive binding with fluorescently labeled probe mols.  The feasibility of the modular probe strategy is demonstrated with noncovalent PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodaDW6dNqXzrVg90H21EOLACvtfcHk0ljPMEpWm_pzOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFais7vF&md5=708fbb531d90863eca83989b9a9b85b4</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00346%26sid%3Dliteratum%253Aachs%26aulast%3DRutkowska%26aufirst%3DA.%26aulast%3DThomson%26aufirst%3DD.%2BW.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DMueller%26aufirst%3DK.%2BM.%26aulast%3DDittus%26aufirst%3DL.%26aulast%3DKrause%26aufirst%3DJ.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26atitle%3DA%2520modular%2520probe%2520strategy%2520for%2520drug%2520localization%252C%2520target%2520identification%2520and%2520target%2520occupancy%2520measurement%2520on%2520single%2520cell%2520level%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2541%26epage%3D2550%26doi%3D10.1021%2Facschembio.6b00346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Honer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martarello, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comley, R. A.</span></span> <span> </span><span class="NLM_article-title">Radioligand development for molecular imaging of the central nervous system with positron emission tomography</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1936</span>– <span class="NLM_lpage">1944</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.08.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.drudis.2014.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25173703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyhtrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1936-1944&author=M.+Honerauthor=L.+Gobbiauthor=L.+Martarelloauthor=R.+A.+Comley&title=Radioligand+development+for+molecular+imaging+of+the+central+nervous+system+with+positron+emission+tomography&doi=10.1016%2Fj.drudis.2014.08.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Radioligand development for molecular imaging of the central nervous system with positron emission tomography</span></div><div class="casAuthors">Honer, Michael; Gobbi, Luca; Martarello, Laurent; Comley, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1936-1944</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Positron emission tomog. (PET) is routinely used to support the development of drugs to treat neurol. and psychiatric disorders.  PET radioligands must not only be selective for the target of interest but must also possess a range of physicochem. and pharmacol. characteristics that allow them to be radiolabeled with short-lived positron-emitting isotopes, safely administered to humans, and for the degree of target binding to be quantified in vivo.  We review the ligand development process, including target selection, radioligand discovery (in vitro and preclin. evaluation), radiochem. and evaluation in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMOrjEdZ64wLVg90H21EOLACvtfcHk0ljPMEpWm_pzOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyhtrbE&md5=9ae3bb3030cdc3a701be1df568190371</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DHoner%26aufirst%3DM.%26aulast%3DGobbi%26aufirst%3DL.%26aulast%3DMartarello%26aufirst%3DL.%26aulast%3DComley%26aufirst%3DR.%2BA.%26atitle%3DRadioligand%2520development%2520for%2520molecular%2520imaging%2520of%2520the%2520central%2520nervous%2520system%2520with%2520positron%2520emission%2520tomography%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1936%26epage%3D1944%26doi%3D10.1016%2Fj.drudis.2014.08.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutilek, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smotrov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickbarg, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauvagnat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curran, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boinay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saradjian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsen, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandliker, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariv, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutel, B.</span></span> <span> </span><span class="NLM_article-title">Integration of affinity selection-mass spectrometry and functional cell-based assays to rapidly triage druggable target space within the NF-kappaB pathway</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1177/1087057116637353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F1087057116637353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26969322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XitV2qtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=608-619&author=V.+D.+Kutilekauthor=C.+L.+Andrewsauthor=M.+P.+Richardsauthor=Z.+Xuauthor=T.+Sunauthor=Y.+Chenauthor=A.+Hashkeauthor=N.+Smotrovauthor=R.+Fernandezauthor=E.+B.+Nickbargauthor=C.+Chamberlinauthor=B.+Sauvagnatauthor=P.+J.+Curranauthor=R.+Boinayauthor=P.+Saradjianauthor=S.+J.+Allenauthor=N.+Byrneauthor=N.+L.+Elsenauthor=R.+E.+Fordauthor=D.+L.+Hallauthor=M.+Kornienkoauthor=K.+W.+Rickertauthor=S.+Sharmaauthor=J.+M.+Shipmanauthor=K.+J.+Lumbauthor=K.+Colemanauthor=P.+J.+Dandlikerauthor=I.+Karivauthor=B.+Beutel&title=Integration+of+affinity+selection-mass+spectrometry+and+functional+cell-based+assays+to+rapidly+triage+druggable+target+space+within+the+NF-kappaB+pathway&doi=10.1177%2F1087057116637353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of affinity selection-mass spectrometry and functional cell-based assays to rapidly triage druggable target space within the NF-κB pathway</span></div><div class="casAuthors">Kutilek, Victoria D.; Andrews, Christine L.; Richards, Matthew P.; Xu, Zangwei; Sun, Tianxiao; Chen, Yiping; Hashke, Andrew; Smotrov, Nadya; Fernandez, Rafael; Nickbarg, Elliott B.; Chamberlin, Chad; Sauvagnat, Berengere; Curran, Patrick J.; Boinay, Ryan; Saradjian, Peter; Allen, Samantha J.; Byrne, Noel; Elsen, Nathaniel L.; Ford, Rachael E.; Hall, Dawn L.; Kornienko, Maria; Rickert, Keith W.; Sharma, Sujata; Shipman, Jennifer M.; Lumb, Kevin J.; Coleman, Kevin; Dandliker, Peter J.; Kariv, Ilona; Beutel, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">608-619</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The primary objective of early drug discovery is to assoc. druggable target space with a desired phenotype.  The inability to efficiently assoc. these often leads to failure early in the drug discovery process.  In this proof-of-concept study, the most tractable starting points for drug discovery within the NF-κB pathway model system were identified by integrating affinity selection-mass spectrometry (AS-MS) with functional cellular assays.  The AS-MS platform Automated Ligand Identification System (ALIS) was used to rapidly screen 15 NF-κB proteins in parallel against large-compd. libraries.  ALIS identified 382 target-selective compds. binding to 14 of the 15 proteins.  Without any chem. optimization, 22 of the 382 target-selective compds. exhibited a cellular phenotype consistent with the resp. target assocd. in ALIS.  Further studies on structurally related compds. distinguished two chem. series that exhibited a preliminary structure-activity relationship and confirmed target-driven cellular activity to NF-κB1/p105 and TRAF5, resp.  These two series represent new drug discovery opportunities for chem. optimization.  The results described herein demonstrate the power of combining ALIS with cell functional assays in a high-throughput, target-based approach to det. the most tractable drug discovery opportunities within a pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQAPIJwi2tULVg90H21EOLACvtfcHk0lgA6-om-XFhFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitV2qtLjN&md5=d1130842a2d22e65c6903164dbddb341</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1177%2F1087057116637353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116637353%26sid%3Dliteratum%253Aachs%26aulast%3DKutilek%26aufirst%3DV.%2BD.%26aulast%3DAndrews%26aufirst%3DC.%2BL.%26aulast%3DRichards%26aufirst%3DM.%2BP.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DHashke%26aufirst%3DA.%26aulast%3DSmotrov%26aufirst%3DN.%26aulast%3DFernandez%26aufirst%3DR.%26aulast%3DNickbarg%26aufirst%3DE.%2BB.%26aulast%3DChamberlin%26aufirst%3DC.%26aulast%3DSauvagnat%26aufirst%3DB.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DBoinay%26aufirst%3DR.%26aulast%3DSaradjian%26aufirst%3DP.%26aulast%3DAllen%26aufirst%3DS.%2BJ.%26aulast%3DByrne%26aufirst%3DN.%26aulast%3DElsen%26aufirst%3DN.%2BL.%26aulast%3DFord%26aufirst%3DR.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BL.%26aulast%3DKornienko%26aufirst%3DM.%26aulast%3DRickert%26aufirst%3DK.%2BW.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DShipman%26aufirst%3DJ.%2BM.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DDandliker%26aufirst%3DP.%2BJ.%26aulast%3DKariv%26aufirst%3DI.%26aulast%3DBeutel%26aufirst%3DB.%26atitle%3DIntegration%2520of%2520affinity%2520selection-mass%2520spectrometry%2520and%2520functional%2520cell-based%2520assays%2520to%2520rapidly%2520triage%2520druggable%2520target%2520space%2520within%2520the%2520NF-kappaB%2520pathway%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D608%26epage%3D619%26doi%3D10.1177%2F1087057116637353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Annis, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazef, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, H. M.</span></span> <span> </span><span class="NLM_article-title">A general technique to rank protein-ligand binding affinities and determine allosteric versus direct binding site competition in compound mixtures</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">15495</span>– <span class="NLM_lpage">15503</span>, <span class="refDoi"> DOI: 10.1021/ja048365x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja048365x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptlOisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=15495-15503&author=D.+A.+Annisauthor=N.+Nazefauthor=C.+C.+Chuangauthor=M.+P.+Scottauthor=H.+M.+Nash&title=A+general+technique+to+rank+protein-ligand+binding+affinities+and+determine+allosteric+versus+direct+binding+site+competition+in+compound+mixtures&doi=10.1021%2Fja048365x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">A general technique to rank protein-ligand binding affinities and determine allosteric versus direct binding site competition in compound mixtures</span></div><div class="casAuthors">Annis, D. Allen; Nazef, Naim; Chuang, Cheng-Chi; Scott, Margaret Porter; Nash, Huw M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">15495-15503</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To realize the full potential of combinatorial chem.-based drug discovery, generic and efficient tools must be developed that apply the strengths of diversity-oriented chem. synthesis to the identification and optimization of lead compds. for disease-assocd. protein targets.  We report an affinity selection-mass spectrometry (AS-MS) method for protein-ligand affinity ranking and the classification of ligands by binding site.  The method incorporates the following steps: (1) an affinity selection stage, where protein-binding compds. are selected from pools of ligands in the presence of varying concns. of a competitor ligand, (2) a first chromatog. stage to sep. unbound ligands from protein-ligand complexes, and (3) a second chromatog. stage to dissoc. the ligands from the complexes for identification and quantification by MS.  The ability of the competitor ligand to displace a target-bound library member, as measured by MS, reveals the binding site classification and affinity ranking of the mixt. components.  The technique requires no radiolabel incorporation or direct biochem. assay, no modification or immobilization of the compds. or target protein, and all reaction components, including any buffers or cofactors required for protein stability, are free in soln.  We demonstrate the method for several compds. of wide structural variety against representatives of the most important protein classes in contemporary drug discovery, including novel ATP-competitive and allosteric inhibitors of the Akt-1 (PKB) and Zap-70 kinases, and previously undisclosed antagonists of the M2 muscarinic acetylcholine receptor, a G-protein coupled receptor (GPCR).  The theor. basis of the technique is analyzed math., allowing quant. estn. of binding affinities and, in the case of allosteric interaction, abs. detn. of binding cooperativity.  The method is readily applicable to high-throughput screening hit triage, combinatorial library-based affinity optimization, and developing structure-activity relationships among multiple ligands to a given receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpREa-teJ7tC7Vg90H21EOLACvtfcHk0lgA6-om-XFhFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptlOisrk%253D&md5=bee110f0d9f40e75b5e22d120238a742</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Fja048365x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja048365x%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DChuang%26aufirst%3DC.%2BC.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DNash%26aufirst%3DH.%2BM.%26atitle%3DA%2520general%2520technique%2520to%2520rank%2520protein-ligand%2520binding%2520affinities%2520and%2520determine%2520allosteric%2520versus%2520direct%2520binding%2520site%2520competition%2520in%2520compound%2520mixtures%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D15495%26epage%3D15503%26doi%3D10.1021%2Fja048365x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Comess, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schurdak, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorbach, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trumbull, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCall, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutel, B. A.</span></span> <span> </span><span class="NLM_article-title">An ultraefficient affinity-based high-throughout screening process: application to bacterial cell wall biosynthesis enzyme MurF</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">743</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1177/1087057106289971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F1087057106289971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=16973923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1SgsL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=743-754&author=K.+M.+Comessauthor=M.+E.+Schurdakauthor=M.+J.+Voorbachauthor=M.+Coenauthor=J.+D.+Trumbullauthor=H.+Yangauthor=L.+Gaoauthor=H.+Tangauthor=X.+Chengauthor=C.+G.+Lernerauthor=J.+O.+McCallauthor=D.+J.+Burnsauthor=B.+A.+Beutel&title=An+ultraefficient+affinity-based+high-throughout+screening+process%3A+application+to+bacterial+cell+wall+biosynthesis+enzyme+MurF&doi=10.1177%2F1087057106289971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">An ultraefficient affinity-based high-throughout screening process: application to bacterial cell wall biosynthesis enzyme MurF</span></div><div class="casAuthors">Comess, Kenneth M.; Schurdak, Mark E.; Voorbach, Martin J.; Coen, Michael; Trumbull, Jonathan D.; Yang, Houjun; Gao, Lan; Tang, Hua; Cheng, Xueheng; Lerner, Claude G.; McCall, J. Owen; Burns, David J.; Beutel, Bruce A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">743-754</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The authors describe the discovery of a new class of inhibitors to an essential Streptococcus pneumoniae cell wall biosynthesis enzyme, MurF, by a novel affinity screening method.  The strategy involved screening very large mixts. of diverse small org. mols. against the protein target on the basis of equil. binding, followed by iterative ultrafiltration steps and ligand identification by mass spectrometry.  Hits from any affinity-based screening method often can be relatively nonselective ligands, sometimes referred to as "nuisance" or "promiscuous" compds.  Ligands selective in their binding affinity for the MurF target were readily identified through electronic subtraction of an empirically detd. subset of promiscuous compds. in the library without subsequent selectivity panels.  The complete strategy for discovery and identification of novel specific ligands can be applied to all sol. protein targets and a wide variety of ligand libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgdKbNMU3brVg90H21EOLACvtfcHk0liZi8A0lGIBGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1SgsL7F&md5=949a2a17ddba4d6c001bad259a3c75e1</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1177%2F1087057106289971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106289971%26sid%3Dliteratum%253Aachs%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DSchurdak%26aufirst%3DM.%2BE.%26aulast%3DVoorbach%26aufirst%3DM.%2BJ.%26aulast%3DCoen%26aufirst%3DM.%26aulast%3DTrumbull%26aufirst%3DJ.%2BD.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DLerner%26aufirst%3DC.%2BG.%26aulast%3DMcCall%26aufirst%3DJ.%2BO.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DBeutel%26aufirst%3DB.%2BA.%26atitle%3DAn%2520ultraefficient%2520affinity-based%2520high-throughout%2520screening%2520process%253A%2520application%2520to%2520bacterial%2520cell%2520wall%2520biosynthesis%2520enzyme%2520MurF%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D743%26epage%3D754%26doi%3D10.1177%2F1087057106289971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieth, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroh, J. G.</span></span> <span> </span><span class="NLM_article-title">Solution-based indirect affinity selection mass spectrometry–a general tool for high-throughput screening of pharmaceutical compound libraries</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">7413</span>– <span class="NLM_lpage">7420</span>, <span class="refDoi"> DOI: 10.1021/ac500938y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac500938y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=7413-7420&author=T.+N.+O%E2%80%99Connellauthor=J.+Ramsayauthor=S.+F.+Riethauthor=M.+J.+Shapiroauthor=J.+G.+Stroh&title=Solution-based+indirect+affinity+selection+mass+spectrometry%E2%80%93a+general+tool+for+high-throughput+screening+of+pharmaceutical+compound+libraries&doi=10.1021%2Fac500938y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fac500938y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac500938y%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DT.%2BN.%26aulast%3DRamsay%26aufirst%3DJ.%26aulast%3DRieth%26aufirst%3DS.%2BF.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DStroh%26aufirst%3DJ.%2BG.%26atitle%3DSolution-based%2520indirect%2520affinity%2520selection%2520mass%2520spectrometry%25E2%2580%2593a%2520general%2520tool%2520for%2520high-throughput%2520screening%2520of%2520pharmaceutical%2520compound%2520libraries%26jtitle%3DAnal.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D7413%26epage%3D7420%26doi%3D10.1021%2Fac500938y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowling, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malamas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellestad, G.</span></span> <span> </span><span class="NLM_article-title">Evaluation of fluorescence-based thermal shift assays for hit identification in drug discovery</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>332</i></span>,  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2004.04.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.ab.2004.04.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=15301960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmsVKhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=153-159&author=M.+C.+Loauthor=A.+Aulabaughauthor=G.+Jinauthor=R.+Cowlingauthor=J.+Bardauthor=M.+Malamasauthor=G.+Ellestad&title=Evaluation+of+fluorescence-based+thermal+shift+assays+for+hit+identification+in+drug+discovery&doi=10.1016%2Fj.ab.2004.04.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of fluorescence-based thermal shift assays for hit identification in drug discovery</span></div><div class="casAuthors">Lo, Mei-Chu; Aulabaugh, Ann; Jin, Guixian; Cowling, Rebecca; Bard, Jonathan; Malamas, Michael; Ellestad, George</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">153-159</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The fluorescence-based thermal shift assay is a general method for identification of inhibitors of target proteins from compd. libraries.  Using an environmentally sensitive fluorescent dye to monitor protein thermal unfolding, the ligand-binding affinity can be assessed from the shift of the unfolding temp. (ΔTm) obtained in the presence of ligands relative to that obtained in the absence of ligands.  In this article, we report that the thermal shift assay can be conducted in an inexpensive, com. available device for temp. control and fluorescence detection.  The binding affinities obtained from thermal shift assays are compared with the binding affinities measured by isothermal titrn. calorimetry and with the IC50 values from enzymic assays.  The potential pitfalls in the data anal. of thermal shift assays are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1qMM9vchBGrVg90H21EOLACvtfcHk0liZi8A0lGIBGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmsVKhu7g%253D&md5=bc10d980128340c79d7d2abc9a56463b</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.04.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.04.031%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DM.%2BC.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DG.%26aulast%3DCowling%26aufirst%3DR.%26aulast%3DBard%26aufirst%3DJ.%26aulast%3DMalamas%26aufirst%3DM.%26aulast%3DEllestad%26aufirst%3DG.%26atitle%3DEvaluation%2520of%2520fluorescence-based%2520thermal%2520shift%2520assays%2520for%2520hit%2520identification%2520in%2520drug%2520discovery%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D153%26epage%3D159%26doi%3D10.1016%2Fj.ab.2004.04.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pantoliano, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrella, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasnoski, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobanov, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myslik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carver, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salemme, F. R.</span></span> <span> </span><span class="NLM_article-title">High-density miniaturized thermal shift assays as a general strategy for drug discovery</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1177/108705710100600609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F108705710100600609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=11788061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVCntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2001&pages=429-440&author=M.+W.+Pantolianoauthor=E.+C.+Petrellaauthor=J.+D.+Kwasnoskiauthor=V.+S.+Lobanovauthor=J.+Myslikauthor=E.+Grafauthor=T.+Carverauthor=E.+Aselauthor=B.+A.+Springerauthor=P.+Laneauthor=F.+R.+Salemme&title=High-density+miniaturized+thermal+shift+assays+as+a+general+strategy+for+drug+discovery&doi=10.1177%2F108705710100600609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">High-density miniaturized thermal shift assays as a general strategy for drug discovery</span></div><div class="casAuthors">Pantoliano, Michael W.; Petrella, Eugene C.; Kwasnoski, Joseph D.; Lobanov, Victor S.; Myslik, James; Graf, Edward; Carver, Ted; Asel, Eric; Springer, Barry A.; Lane, Pamela; Salemme, F. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">429-440</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">More general and universally applicable drug discovery assay technologies are needed in order to keep pace with the recent advances in combinatorial chem. and genomics-based target generation.  Ligand-induced conformational stabilization of proteins is a well-understood phenomenon in which substrates, inhibitors, cofactors, and even other proteins provide enhanced stability to proteins on binding.  This phenomenon is based on the energetic coupling of the ligand-binding and protein-melting reactions.  In an attempt to harness these biophys. properties for drug discovery, fully automated instrumentation was designed and implemented to perform miniaturized fluorescence-based thermal shift assays in a microplate format for the high throughput screening of compd. libraries.  Validation of this process and instrumentation was achieved by investigating ligand binding to more than 100 protein targets.  The general applicability of the thermal shift screening strategy was found to be an important advantage because it circumvents the need to design and retool new assays with each new therapeutic target.  Moreover, the miniaturized thermal shift assay methodol. does not require any prior knowledge of a therapeutic target's function, making it ideally suited for the quant. high throughput drug screening and evaluation of targets derived from genomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp58y9R74HfiLVg90H21EOLACvtfcHk0lh-krSJkSby6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVCntg%253D%253D&md5=deef07e6683f632cd078214ad88d9999</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1177%2F108705710100600609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705710100600609%26sid%3Dliteratum%253Aachs%26aulast%3DPantoliano%26aufirst%3DM.%2BW.%26aulast%3DPetrella%26aufirst%3DE.%2BC.%26aulast%3DKwasnoski%26aufirst%3DJ.%2BD.%26aulast%3DLobanov%26aufirst%3DV.%2BS.%26aulast%3DMyslik%26aufirst%3DJ.%26aulast%3DGraf%26aufirst%3DE.%26aulast%3DCarver%26aufirst%3DT.%26aulast%3DAsel%26aufirst%3DE.%26aulast%3DSpringer%26aufirst%3DB.%2BA.%26aulast%3DLane%26aufirst%3DP.%26aulast%3DSalemme%26aufirst%3DF.%2BR.%26atitle%3DHigh-density%2520miniaturized%2520thermal%2520shift%2520assays%2520as%2520a%2520general%2520strategy%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2001%26volume%3D6%26spage%3D429%26epage%3D440%26doi%3D10.1177%2F108705710100600609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pellecchia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowburn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalvit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giralt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homans, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luchinat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oschkinat, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwalbe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegal, G.</span></span> <span> </span><span class="NLM_article-title">Perspectives on NMR in drug discovery: a technique comes of age</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">738</span>– <span class="NLM_lpage">745</span>, <span class="refDoi"> DOI: 10.1038/nrd2606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd2606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=19172689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVGgtL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=738-745&author=M.+Pellecchiaauthor=I.+Bertiniauthor=D.+Cowburnauthor=C.+Dalvitauthor=E.+Giraltauthor=W.+Jahnkeauthor=T.+L.+Jamesauthor=S.+W.+Homansauthor=H.+Kesslerauthor=C.+Luchinatauthor=B.+Meyerauthor=H.+Oschkinatauthor=J.+Pengauthor=H.+Schwalbeauthor=G.+Siegal&title=Perspectives+on+NMR+in+drug+discovery%3A+a+technique+comes+of+age&doi=10.1038%2Fnrd2606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Perspectives on NMR in drug discovery: a technique comes of age</span></div><div class="casAuthors">Pellecchia, Maurizio; Bertini, Ivano; Cowburn, David; Dalvit, Claudio; Giralt, Ernest; Jahnke, Wolfgang; James, Thomas L.; Homans, Steve W.; Kessler, Horst; Luchinat, Claudio; Meyer, Bernd; Oschkinat, Hartmut; Peng, Jeff; Schwalbe, Harald; Siegal, Gregg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">738-745</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the past decade, the ability of NMR spectroscopy to provide information on intermol. interactions that is valuable in drug discovery has been increasingly appreciated.  Pellecchia and colleagues provide their collective evaluation of the major applications of NMR in drug discovery, focusing on hit and lead generation, and critically analyze its current and potential utility.  In the past decade, the potential of harnessing the ability of NMR spectroscopy to monitor intermol. interactions as a tool for drug discovery has been increasingly appreciated in academia and industry.  In this Perspective, we highlight some of the major applications of NMR in drug discovery, focusing on hit and lead generation, and provide a crit. anal. of its current and potential utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgotirEZMnYbVg90H21EOLACvtfcHk0lh-krSJkSby6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVGgtL%252FK&md5=b88497140e0f2862c9dd434e548a56c5</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1038%2Fnrd2606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2606%26sid%3Dliteratum%253Aachs%26aulast%3DPellecchia%26aufirst%3DM.%26aulast%3DBertini%26aufirst%3DI.%26aulast%3DCowburn%26aufirst%3DD.%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DGiralt%26aufirst%3DE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJames%26aufirst%3DT.%2BL.%26aulast%3DHomans%26aufirst%3DS.%2BW.%26aulast%3DKessler%26aufirst%3DH.%26aulast%3DLuchinat%26aufirst%3DC.%26aulast%3DMeyer%26aufirst%3DB.%26aulast%3DOschkinat%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DSchwalbe%26aufirst%3DH.%26aulast%3DSiegal%26aufirst%3DG.%26atitle%3DPerspectives%2520on%2520NMR%2520in%2520drug%2520discovery%253A%2520a%2520technique%2520comes%2520of%2520age%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D738%26epage%3D745%26doi%3D10.1038%2Fnrd2606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michnick, S. W.</span></span> <span> </span><span class="NLM_article-title">Application of protein-fragment complementation assays in cell biology</span>. <i>BioTechniques</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">137</span>, 139, 141 passim <span class="refDoi"> DOI: 10.2144/000112396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.2144%2F000112396" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=137&author=I.+Remyauthor=S.+W.+Michnick&title=Application+of+protein-fragment+complementation+assays+in+cell+biology&doi=10.2144%2F000112396"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.2144%2F000112396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2144%252F000112396%26sid%3Dliteratum%253Aachs%26aulast%3DRemy%26aufirst%3DI.%26aulast%3DMichnick%26aufirst%3DS.%2BW.%26atitle%3DApplication%2520of%2520protein-fragment%2520complementation%2520assays%2520in%2520cell%2520biology%26jtitle%3DBioTechniques%26date%3D2007%26volume%3D42%26spage%3D137%26doi%3D10.2144%2F000112396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref150"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref150'); return false;" data-citation="" class="refNumLink">150</a></strong><div class="NLM_citation" id="cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westwick, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamerdin, J. E.</span></span> <span> </span><span class="NLM_article-title">Improving drug discovery with contextual assays and cellular systems analysis</span>. <i>Methods Mol. Biol. (N. Y., NY, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>756</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-160-4_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2F978-1-61779-160-4_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21870220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlansrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=756&publication_year=2011&pages=61-73&author=J.+K.+Westwickauthor=J.+E.+Lamerdin&title=Improving+drug+discovery+with+contextual+assays+and+cellular+systems+analysis&doi=10.1007%2F978-1-61779-160-4_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Improving drug discovery with contextual assays and cellular systems analysis</span></div><div class="casAuthors">Westwick, John K.; Lamerdin, Jane E.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">756</span>
        (<span class="NLM_cas:issue">Signal Transduction Protocols</span>),
    <span class="NLM_cas:pages">61-73</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Despite rapid growth in our knowledge of potential disease targets following completion of the first drafts of the human genome over 10 years ago, the success rate of new therapeutic discovery has been frustratingly low.  In addn. to the widely reported costs and single-digit success rate of the entire drug discovery and development process, it has recently been estd. that even the preliminary process of transitioning new targets to preclin. development succeeds in less than 3% of attempts.  At these early stages of development, poor understanding of therapeutic mechanisms and lack of compd. selectivity are often to blame for failed compds.  It is worth noting than the emerging class of nucleic acid-based therapeutics, including miRNA and RNAi, are likely to be even more prone to unexpected system-wide and off-target activities.  For all therapeutic approaches, it is clear that discovery strategies permitting the assessment of drug targets in their native context are required.  At the same time, these strategies need to retain the high throughput of current reductionist approaches to enable broad assessment of chem. space for small mol. and genetic therapeutics.  We describe here an integrated system based on high-content cellular anal. combined with system-wide pathway interrogation.  The platform can be applied to novel therapeutic target and drug candidate identification, and for providing detailed mechanistic and selectivity information at an early stage of development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqmwd5cwc7frVg90H21EOLACvtfcHk0li8IjTykIarKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlansrjN&md5=98aecd801b4ed3f9fdb3f91df5f182e2</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-160-4_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-160-4_3%26sid%3Dliteratum%253Aachs%26aulast%3DWestwick%26aufirst%3DJ.%2BK.%26aulast%3DLamerdin%26aufirst%3DJ.%2BE.%26atitle%3DImproving%2520drug%2520discovery%2520with%2520contextual%2520assays%2520and%2520cellular%2520systems%2520analysis%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2011%26volume%3D756%26spage%3D61%26epage%3D73%26doi%3D10.1007%2F978-1-61779-160-4_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref151"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref151'); return false;" data-citation="" class="refNumLink">151</a></strong><div class="NLM_citation" id="cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E. M.</span>; <span class="NLM_string-name">Hansen, T. M.</span>; <span class="NLM_string-name">Richardson, P. L.</span>; <span class="NLM_string-name">Wilcox, D. M.</span>; <span class="NLM_string-name">Addo, S. N.</span>; <span class="NLM_string-name">Tang, H.</span>; <span class="NLM_string-name">Sun, C.</span>; <span class="NLM_string-name">Petros, A.</span>; <span class="NLM_string-name">Panchal, S.</span>; <span class="NLM_string-name">Park, C.</span>; <span class="NLM_string-name">Nottoli, M. S.</span>; <span class="NLM_string-name">Patterson, M. J.</span>; <span class="NLM_string-name">Buchanan, G.</span>; <span class="NLM_string-name">Cheng, D.</span>; <span class="NLM_string-name">Davis, H. M.</span>; <span class="NLM_string-name">Calderwood, D. J.</span>; <span class="NLM_string-name">Elmore, S. W.</span>; <span class="NLM_string-name">Shen, Y.</span></span>, unpublished data.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Warder%2C+S.+E.+M.%3B+Hansen%2C+T.+M.%3B+Richardson%2C+P.+L.%3B+Wilcox%2C+D.+M.%3B+Addo%2C+S.+N.%3B+Tang%2C+H.%3B+Sun%2C+C.%3B+Petros%2C+A.%3B+Panchal%2C+S.%3B+Park%2C+C.%3B+Nottoli%2C+M.+S.%3B+Patterson%2C+M.+J.%3B+Buchanan%2C+G.%3B+Cheng%2C+D.%3B+Davis%2C+H.+M.%3B+Calderwood%2C+D.+J.%3B+Elmore%2C+S.+W.%3B+Shen%2C+Y.%2C+unpublished+data."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DWarder%26aufirst%3DS.%2BE.%2BM." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref152"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref152'); return false;" data-citation="" class="refNumLink">152</a></strong><div class="NLM_citation" id="cit152"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. F.</span></span> <span> </span><span class="NLM_article-title">Brd4 modulates the innate immune response through Mnk2-eIF4E pathway-dependent translational control of IkappaBalpha</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E3993</span>– <span class="NLM_lpage">E4001</span>, <span class="refDoi"> DOI: 10.1073/pnas.1700109114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.1700109114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E3993-E4001&author=Y.+Baoauthor=X.+Wuauthor=J.+Chenauthor=X.+Huauthor=F.+Zengauthor=J.+Chengauthor=H.+Jinauthor=X.+Linauthor=L.+F.+Chen&title=Brd4+modulates+the+innate+immune+response+through+Mnk2-eIF4E+pathway-dependent+translational+control+of+IkappaBalpha&doi=10.1073%2Fpnas.1700109114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1700109114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1700109114%26sid%3Dliteratum%253Aachs%26aulast%3DBao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DF.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBrd4%2520modulates%2520the%2520innate%2520immune%2520response%2520through%2520Mnk2-eIF4E%2520pathway-dependent%2520translational%2520control%2520of%2520IkappaBalpha%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE3993%26epage%3DE4001%26doi%3D10.1073%2Fpnas.1700109114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref153"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref153'); return false;" data-citation="" class="refNumLink">153</a></strong><div class="NLM_citation" id="cit153"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span>; <span class="NLM_string-name">Robinson, K. P. J.</span>; <span class="NLM_string-name">Oyer, J. A.</span>; <span class="NLM_string-name">Kifle, L. X.</span>; <span class="NLM_string-name">Kurnick, M. D.</span>; <span class="NLM_string-name">Clapham, B.</span>; <span class="NLM_string-name">Pliushchev, M. A.</span>; <span class="NLM_string-name">Sweis, R. X.</span>; <span class="NLM_string-name">Tang, H.</span>; <span class="NLM_string-name">McLoughlin, S. M.</span>; <span class="NLM_string-name">Jhala, M. A.</span>; <span class="NLM_string-name">Comess, K. M.</span>; <span class="NLM_string-name">Shannon, D. A.</span>; <span class="NLM_string-name">Galasinski, S. C.</span></span>, unpublished data.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Warder%2C+S.+E.%3B+Robinson%2C+K.+P.+J.%3B+Oyer%2C+J.+A.%3B+Kifle%2C+L.+X.%3B+Kurnick%2C+M.+D.%3B+Clapham%2C+B.%3B+Pliushchev%2C+M.+A.%3B+Sweis%2C+R.+X.%3B+Tang%2C+H.%3B+McLoughlin%2C+S.+M.%3B+Jhala%2C+M.+A.%3B+Comess%2C+K.+M.%3B+Shannon%2C+D.+A.%3B+Galasinski%2C+S.+C.%2C+unpublished+data."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DWarder%26aufirst%3DS.%2BE." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref154"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref154'); return false;" data-citation="" class="refNumLink">154</a></strong><div class="NLM_citation" id="cit154"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarvliden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogvadze, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgenstern, R.</span></span> <span> </span><span class="NLM_article-title">Multiple roles of microsomal glutathione transferase 1 in cellular protection: a mechanistic study</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1638</span>– <span class="NLM_lpage">1645</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2010.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.freeradbiomed.2010.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=20727966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKktLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=1638-1645&author=K.+Johanssonauthor=J.+Jarvlidenauthor=V.+Gogvadzeauthor=R.+Morgenstern&title=Multiple+roles+of+microsomal+glutathione+transferase+1+in+cellular+protection%3A+a+mechanistic+study&doi=10.1016%2Fj.freeradbiomed.2010.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple roles of microsomal glutathione transferase 1 in cellular protection: A mechanistic study</span></div><div class="casAuthors">Johansson, Katarina; Jaervliden, Julia; Gogvadze, Vladimir; Morgenstern, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1638-1645</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The aim of this study was to investigate the involvement of membrane-bound microsomal glutathione transferase 1 (MGST1) in cellular resistance against oxidative stress as well as its mechanism of protection.  MGST1 is ubiquitously expressed and predominantly located in the endoplasmic reticulum and outer mitochondrial membrane.  Utilizing MCF7 cells over-expressing MGST1 significant protection against agents that are known to induce lipid peroxidn. (e.g., cumene hydroperoxide and tert-butylhydroperoxide) and an end-product of lipid peroxidn. (e.g., 4-hydroxy-2-nonenal) was shown.  Furthermore, the results demonstrate that MGST1 protection can be enhanced by vitamin E when toxicity depends on oxidative stress, but not when direct alkylation is the dominant mechanism.  Mitochondria in MGST1-over-expressing cells were shown to be protected from oxidative insult as measured by calcium loading capacity and respiration.  MGST1 induces cellular resistance against cisplatin.  Here vitamin E was used to elucidate whether oxidative stress caused by cisplatin is significant for cell toxicity.  The results indicate that oxidative stress and induction of lipid peroxidn. are not the most prominent toxic mechanism of cisplatin in the cell system.  It is thus concluded that MGST1 protects cells (and mitochondria) by both conjugation and glutathione peroxidase functions.  A new protective mechanism against cisplatin is also indicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovM-cagT_i4LVg90H21EOLACvtfcHk0li8IjTykIarKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKktLnO&md5=b9cddef5cab3820279d67f46b19e93ea</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2010.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2010.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DK.%26aulast%3DJarvliden%26aufirst%3DJ.%26aulast%3DGogvadze%26aufirst%3DV.%26aulast%3DMorgenstern%26aufirst%3DR.%26atitle%3DMultiple%2520roles%2520of%2520microsomal%2520glutathione%2520transferase%25201%2520in%2520cellular%2520protection%253A%2520a%2520mechanistic%2520study%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2010%26volume%3D49%26spage%3D1638%26epage%3D1645%26doi%3D10.1016%2Fj.freeradbiomed.2010.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref155"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref155'); return false;" data-citation="" class="refNumLink">155</a></strong><div class="NLM_citation" id="cit155"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aniya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaizumi, N.</span></span> <span> </span><span class="NLM_article-title">Mitochondrial glutathione transferases involving a new function for membrane permeability transition pore regulation</span>. <i>Drug Metab. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">292</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.3109/03602532.2011.552913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.3109%2F03602532.2011.552913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21428695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslCisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=292-299&author=Y.+Aniyaauthor=N.+Imaizumi&title=Mitochondrial+glutathione+transferases+involving+a+new+function+for+membrane+permeability+transition+pore+regulation&doi=10.3109%2F03602532.2011.552913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit155R"><div class="casContent"><span class="casTitleNuber">155</span><div class="casTitle"><span class="NLM_cas:atitle">Mitochondrial glutathione transferases involving a new function for membrane permeability transition pore regulation</span></div><div class="casAuthors">Aniya, Yoko; Imaizumi, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">292-299</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The mitochondria in mammalian cells are a predominant resource of reactive oxygen species (ROS), which are produced during respiration-coupled oxidative metab. or various chem. stresses.  End-products from membrane-lipid peroxidn. caused by ROS are highly toxic, thereby their elimination/scavenging are protective of mitochondria and cells against oxidative damages.  In mitochondria, sol. (kappa, alpha, mu, pi, zeta) and membrane-bound glutathione transferases (GSTs) (MGST1) are distributed.  Mitochondrial GSTs display both glutathione transferase and peroxidase activities that detoxify such harmful products through glutathione (GSH) conjugation or GSH-mediated peroxide redn.  Some GST isoenzymes are induced by oxidative stress, an adaptation mechanism for the protection of cells from oxidative stress.  Membrane-bound MGST1 is activated through the thiol modification in oxidative conditions.  Protective action of MGST1 against oxidative stress has been confirmed using MCF7 cells highly expressed of MGST1.  In recent years, mitochondria have been recognized as a regulator of cell death via both apoptosis and necrosis, where oxidative stress-induced alteration of the membrane permeability is an important step.  Recent studies have shown that MGST1 in the inner mitochondrial membrane could interact with the mitochondrial permeability transition (MPT) regulator proteins, such as adenine nucleotide translocator (ANT) and/or cyclophilin D, and could contribute to oxidant-induced MPT pores.  Interaction of GST alpha with ANT has also been shown.  In this review, functions of the mitochondrial GSTs, including a new role for mitochondria-mediated cell death, are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwWno4AaI0r7Vg90H21EOLACvtfcHk0lgLIX99cxuFEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslCisbY%253D&md5=baa5ea91a6754eaeb1800a9dc702f3b9</span></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.3109%2F03602532.2011.552913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2011.552913%26sid%3Dliteratum%253Aachs%26aulast%3DAniya%26aufirst%3DY.%26aulast%3DImaizumi%26aufirst%3DN.%26atitle%3DMitochondrial%2520glutathione%2520transferases%2520involving%2520a%2520new%2520function%2520for%2520membrane%2520permeability%2520transition%2520pore%2520regulation%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2011%26volume%3D43%26spage%3D292%26epage%3D299%26doi%3D10.3109%2F03602532.2011.552913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref156"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref156'); return false;" data-citation="" class="refNumLink">156</a></strong><div class="NLM_citation" id="cit156"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yvone, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marton, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crenshaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winckler, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">High-throughput identification of genotype-specific cancer vulnerabilities in mixtures of barcoded tumor cell lines</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">423</span>, <span class="refDoi"> DOI: 10.1038/nbt.3460</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnbt.3460" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26928769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XivFKgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=419-423&author=C.+Yuauthor=A.+M.+Mannanauthor=G.+M.+Yvoneauthor=K.+N.+Rossauthor=Y.+L.+Zhangauthor=M.+A.+Martonauthor=B.+R.+Taylorauthor=A.+Crenshawauthor=J.+Z.+Gouldauthor=P.+Tamayoauthor=B.+A.+Weirauthor=A.+Tsherniakauthor=B.+Wongauthor=L.+A.+Garrawayauthor=A.+F.+Shamjiauthor=M.+A.+Palmerauthor=M.+A.+Foleyauthor=W.+Wincklerauthor=S.+L.+Schreiberauthor=A.+L.+Kungauthor=T.+R.+Golub&title=High-throughput+identification+of+genotype-specific+cancer+vulnerabilities+in+mixtures+of+barcoded+tumor+cell+lines&doi=10.1038%2Fnbt.3460"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput identification of genotype-specific cancer vulnerabilities in mixtures of barcoded tumor cell lines</span></div><div class="casAuthors">Yu, Channing; Mannan, Aristotle M.; Yvone, Griselda Metta; Ross, Kenneth N.; Zhang, Yan-Ling; Marton, Melissa A.; Taylor, Bradley R.; Crenshaw, Andrew; Gould, Joshua Z.; Tamayo, Pablo; Weir, Barbara A.; Tsherniak, Aviad; Wong, Bang; Garraway, Levi A.; Shamji, Alykhan F.; Palmer, Michelle A.; Foley, Michael A.; Winckler, Wendy; Schreiber, Stuart L.; Kung, Andrew L.; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">419-423</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hundreds of genetically characterized cell lines are available for the discovery of genotype-specific cancer vulnerabilities.  However, screening large nos. of compds. against large nos. of cell lines is currently impractical, and such expts. are often difficult to control.  Here we report a method called PRISM that allows pooled screening of mixts. of cancer cell lines by labeling each cell line with 24-nucleotide barcodes.  PRISM revealed the expected patterns of cell killing seen in conventional (unpooled) assays.  In a screen of 102 cell lines across 8,400 compds., PRISM led to the identification of BRD-7880 as a potent and highly specific inhibitor of aurora kinases B and C.  Cell line pools also efficiently formed tumors as xenografts, and PRISM recapitulated the expected pattern of erlotinib sensitivity in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvZWopzWWOhrVg90H21EOLACvtfcHk0lgLIX99cxuFEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivFKgsro%253D&md5=8c5206e6e3dd2fe19673acc9eed04956</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3460%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DC.%26aulast%3DMannan%26aufirst%3DA.%2BM.%26aulast%3DYvone%26aufirst%3DG.%2BM.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DZhang%26aufirst%3DY.%2BL.%26aulast%3DMarton%26aufirst%3DM.%2BA.%26aulast%3DTaylor%26aufirst%3DB.%2BR.%26aulast%3DCrenshaw%26aufirst%3DA.%26aulast%3DGould%26aufirst%3DJ.%2BZ.%26aulast%3DTamayo%26aufirst%3DP.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DPalmer%26aufirst%3DM.%2BA.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DHigh-throughput%2520identification%2520of%2520genotype-specific%2520cancer%2520vulnerabilities%2520in%2520mixtures%2520of%2520barcoded%2520tumor%2520cell%2520lines%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26spage%3D419%26epage%3D423%26doi%3D10.1038%2Fnbt.3460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref157"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref157'); return false;" data-citation="" class="refNumLink">157</a></strong><div class="NLM_citation" id="cit157"><span><span class="NLM_contrib-group">The QSTAR Consortium</span> <span> </span><span class="NLM_article-title">Using transcriptomics to guide lead optimization in
drug discovery projects: Lessons learned from the QSTAR project</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.12.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.drudis.2014.12.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25582842" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=505-513&author=The+QSTAR+Consortium&title=Using+transcriptomics+to+guide+lead+optimization+in%0Adrug+discovery+projects%3A+Lessons+learned+from+the+QSTAR+project&doi=10.1016%2Fj.drudis.2014.12.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.12.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.12.014%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUsing%2520transcriptomics%2520to%2520guide%2520lead%2520optimization%2520in%250Adrug%2520discovery%2520projects%253A%2520Lessons%2520learned%2520from%2520the%2520QSTAR%2520project%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D505%26epage%3D513%26doi%3D10.1016%2Fj.drudis.2014.12.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref158"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref158'); return false;" data-citation="" class="refNumLink">158</a></strong><div class="NLM_citation" id="cit158"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Awale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J. L.</span></span> <span> </span><span class="NLM_article-title">The polypharmacology browser: a web-based multi-fingerprint target prediction tool using ChEMBL bioactivity data</span>. <i>J. Cheminformatics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">11</span>, <span class="refDoi"> DOI: 10.1186/s13321-017-0199-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1186%2Fs13321-017-0199-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlsFGhtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=11&author=M.+Awaleauthor=J.+L.+Reymond&title=The+polypharmacology+browser%3A+a+web-based+multi-fingerprint+target+prediction+tool+using+ChEMBL+bioactivity+data&doi=10.1186%2Fs13321-017-0199-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">The polypharmacology browser: a web-based multi-fingerprint target prediction tool using ChEMBL bioactivity data</span></div><div class="casAuthors">Awale, Mahendra; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11/1-11/10</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Several web-based tools have been reported recently which predict the possible targets of a small mol. by similarity to compds. of known bioactivity using mol. fingerprints (fps), however predictions in each case rely on similarities computed from only one or two fps.  Considering that structural similarity and therefore the predicted targets strongly depend on the method used for comparison, it would be highly desirable to predict targets using a broader set of fps simultaneously.  Results: Herein, we present the polypharmacol. browser (PPB), a web-based platform which predicts possible targets for small mols. by searching for nearest neighbors using ten different fps describing compn., substructures, mol. shape and pharmacophores.  PPB searches through 4613 groups of at least 10 same target annotated bioactive mols. from ChEMBL and returns a list of predicted targets ranked by consensus voting scheme and p value.  A validation study across 670 drugs with up to 20 targets showed that combining the predictions from all 10 fps gives the best results, with on av. 50% of the known targets of a drug being correctly predicted with a hit rate of 25%.  Furthermore, when profiling a new inhibitor of the calcium channel TRPV6 against 24 targets taken from a safety screen panel, we obsd. inhibition in 5 out of 5 targets predicted by PPB and in 7 out of 18 targets not predicted by PPB.  The rate of correct (5/12) and incorrect (0/12) predictions for this compd. by PPB was comparable to that of other web-based prediction tools.  Conclusion: PPB offers a versatile platform for target prediction based on multi-fingerprint comparisons, and is freely accessible at www.gdb.unibe.ch as a valuable support for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtd4w_kv537bVg90H21EOLACvtfcHk0lh1CAeheNi0aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlsFGhtr0%253D&md5=20e3519d493380139f5cd61974017cec</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1186%2Fs13321-017-0199-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-017-0199-x%26sid%3Dliteratum%253Aachs%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DThe%2520polypharmacology%2520browser%253A%2520a%2520web-based%2520multi-fingerprint%2520target%2520prediction%2520tool%2520using%2520ChEMBL%2520bioactivity%2520data%26jtitle%3DJ.%2520Cheminformatics%26date%3D2017%26volume%3D9%26spage%3D11%26doi%3D10.1186%2Fs13321-017-0199-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref159"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref159'); return false;" data-citation="" class="refNumLink">159</a></strong><div class="NLM_citation" id="cit159"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagunin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stepanchikova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filimonov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poroikov, V.</span></span> <span> </span><span class="NLM_article-title">PASS: prediction of activity spectra for biologically active substances</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">747</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/16.8.747</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbioinformatics%2F16.8.747" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=11099264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVGms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2000&pages=747-748&author=A.+Laguninauthor=A.+Stepanchikovaauthor=D.+Filimonovauthor=V.+Poroikov&title=PASS%3A+prediction+of+activity+spectra+for+biologically+active+substances&doi=10.1093%2Fbioinformatics%2F16.8.747"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">PASS: prediction of activity spectra for biologically active substances</span></div><div class="casAuthors">Lagunin, Alexey; Stepanchikova, Alla; Filimonov, Dmitrii; Poroikov, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">747-748</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The concept of the biol. activity spectrum was introduced to describe the properties of biol. active substances.  The PASS (prediction of activity spectra for substances) software product, which predicts more than 300 pharmacol. effects and biochem. mechanisms on the basis of the structural formula of a substance, may be efficiently used to find new targets for some ligands and, conversely, to reveal new ligands for some biol. targets.  A WWW interface has been developed for the PASS software.  A WWW server for the online prediction of the biol. activity spectra of substances has been constructed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeV2yQu6YNBbVg90H21EOLACvtfcHk0lh1CAeheNi0aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVGms7k%253D&md5=f3b098eabf8c73b7285fead383a2a34f</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2F16.8.747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252F16.8.747%26sid%3Dliteratum%253Aachs%26aulast%3DLagunin%26aufirst%3DA.%26aulast%3DStepanchikova%26aufirst%3DA.%26aulast%3DFilimonov%26aufirst%3DD.%26aulast%3DPoroikov%26aufirst%3DV.%26atitle%3DPASS%253A%2520prediction%2520of%2520activity%2520spectra%2520for%2520biologically%2520active%2520substances%26jtitle%3DBioinformatics%26date%3D2000%26volume%3D16%26spage%3D747%26epage%3D748%26doi%3D10.1093%2Fbioinformatics%2F16.8.747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref160"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref160'); return false;" data-citation="" class="refNumLink">160</a></strong><div class="NLM_citation" id="cit160"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">TarFisDock: a web server for identifying drug targets with docking approach</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">W219</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgkl114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=16844997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=W219-224&author=H.+Liauthor=Z.+Gaoauthor=L.+Kangauthor=H.+Zhangauthor=K.+Yangauthor=K.+Yuauthor=X.+Luoauthor=W.+Zhuauthor=K.+Chenauthor=J.+Shenauthor=X.+Wangauthor=H.+Jiang&title=TarFisDock%3A+a+web+server+for+identifying+drug+targets+with+docking+approach&doi=10.1093%2Fnar%2Fgkl114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">TarFisDock: a web server for identifying drug targets with docking approach</span></div><div class="casAuthors">Li, Honglin; Gao, Zhenting; Kang, Ling; Zhang, Hailei; Yang, Kun; Yu, Kunqian; Luo, Xiaomin; Zhu, Weiliang; Chen, Kaixian; Shen, Jianhua; Wang, Xicheng; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W219-W224</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">TarFisDock is a web-based tool for automating the procedure of searching for small mol.-protein interactions over a large repertoire of protein structures.  It offers PDTD (potential drug target database), a target database contg. 698 protein structures covering 15 therapeutic areas and a reverse ligand-protein docking program.  In contrast to conventional ligand-protein docking, reverse ligand-protein docking aims to seek potential protein targets by screening an appropriate protein database.  The input file of this web server is the small mol. to be tested, in std. mol2 format; TarFisDock then searches for possible binding proteins for the given small mol. by use of a docking approach.  The ligand-protein interaction energy terms of the program DOCK are adopted for ranking the proteins.  To test the reliability of the TarFisDock server, we searched the PDTD for putative binding proteins for vitamin E and 4H-tamoxifen.  The top 2 and 10% candidates of vitamin E binding proteins identified by TarFisDock resp. cover 30 and 50% of reported targets verified or implicated by expts.; and 30 and 50% of exptl. confirmed targets for 4H-tamoxifen appear amongst the top 2 and 5% of the TarFisDock predicted candidates, resp.  Therefore, TarFisDock may be a useful tool for target identification, mechanism study of old drugs and probes discovered from natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFIwhF2-sUb7Vg90H21EOLACvtfcHk0lh1CAeheNi0aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1yit7s%253D&md5=39f644efb79cea93e654549f6e340c30</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl114%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DKang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DTarFisDock%253A%2520a%2520web%2520server%2520for%2520identifying%2520drug%2520targets%2520with%2520docking%2520approach%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW219%26epage%3D224%26doi%3D10.1093%2Fnar%2Fgkl114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref161"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref161'); return false;" data-citation="" class="refNumLink">161</a></strong><div class="NLM_citation" id="cit161"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armbruster, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernsberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0ljpV-CO2OSN-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref162"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref162'); return false;" data-citation="" class="refNumLink">162</a></strong><div class="NLM_citation" id="cit162"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">PharmMapper server: a web server for potential drug target identification using pharmacophore mapping approach</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">W609</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgkq300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=20430828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W609-614&author=X.+Liuauthor=S.+Ouyangauthor=B.+Yuauthor=Y.+Liuauthor=K.+Huangauthor=J.+Gongauthor=S.+Zhengauthor=Z.+Liauthor=H.+Liauthor=H.+Jiang&title=PharmMapper+server%3A+a+web+server+for+potential+drug+target+identification+using+pharmacophore+mapping+approach&doi=10.1093%2Fnar%2Fgkq300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">PharmMapper server: a web server for potential drug target identification using pharmacophore mapping approach</span></div><div class="casAuthors">Liu, Xiaofeng; Ouyang, Sisheng; Yu, Biao; Liu, Yabo; Huang, Kai; Gong, Jiayu; Zheng, Siyuan; Li, Zhihua; Li, Honglin; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W609-W614</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In silico drug target identification, which includes many distinct algorithms for finding disease genes and proteins, is the first step in the drug discovery pipeline.  When the 3D structures of the targets are available, the problem of target identification is usually converted to finding the best interaction mode between the potential target candidates and small mol. probes.  Pharmacophore, which is the spatial arrangement of features essential for a mol. to interact with a specific target receptor, is an alternative method for achieving this goal apart from mol. docking method.  PharmMapper server is a freely accessed web server designed to identify potential target candidates for the given small mols. (drugs, natural products or other newly discovered compds. with unidentified binding targets) using pharmacophore mapping approach.  PharmMapper hosts a large, inhouse repertoire of pharmacophore database (namely PharmTargetDB) annotated from all the targets information in TargetBank, BindingDB, DrugBank and potential drug target database, including over 7000 receptor-based pharmacophore models (covering over 1500 drug targets information).  PharmMapper automatically finds the best mapping poses of the query mol. against all the pharmacophore models in PharmTargetDB and lists the top N best-fitted hits with appropriate target annotations, as well as resp. mol.'s aligned poses are presented.  Benefited from the highly efficient and robust triangle hashing mapping method, PharmMapper bears high throughput ability and only costs 1 h averagely to screen the whole PharmTargetDB.  The protocol was successful in finding the proper targets among the top 300 pharmacophore candidates in the retrospective benchmarking test of tamoxifen.  PharmMapper is available at http://59.78.96.61/pharmmapper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL-Gaa6UM_zLVg90H21EOLACvtfcHk0ljpV-CO2OSN-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWjsL4%253D&md5=dadfc5a278243d22f2ea041914461e3a</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq300%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DPharmMapper%2520server%253A%2520a%2520web%2520server%2520for%2520potential%2520drug%2520target%2520identification%2520using%2520pharmacophore%2520mapping%2520approach%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW609%26epage%3D614%26doi%3D10.1093%2Fnar%2Fgkq300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref163"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref163'); return false;" data-citation="" class="refNumLink">163</a></strong><div class="NLM_citation" id="cit163"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikailov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span> <span> </span><span class="NLM_article-title">DRAR-CPI: a server for identifying drug repositioning potential and adverse drug reactions via the chemical-protein interactome</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">W492</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgkr299" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21558322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVOntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=W492-498&author=H.+Luoauthor=J.+Chenauthor=L.+Shiauthor=M.+Mikailovauthor=H.+Zhuauthor=K.+Wangauthor=L.+Heauthor=L.+Yang&title=DRAR-CPI%3A+a+server+for+identifying+drug+repositioning+potential+and+adverse+drug+reactions+via+the+chemical-protein+interactome&doi=10.1093%2Fnar%2Fgkr299"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">DRAR-CPI: a server for identifying drug repositioning potential and adverse drug reactions via the chemical-protein interactome</span></div><div class="casAuthors">Luo, Heng; Chen, Jian; Shi, Leming; Mikailov, Mike; Zhu, Huang; Wang, Kejian; He, Lin; Yang, Lun</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W492-W498</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Identifying new indications for existing drugs (drug repositioning) is an efficient way of maximizing their potential.  Adverse drug reaction (ADR) is one of the leading causes of death among hospitalized patients.  As both new indications and ADRs are caused by unexpected chem.-protein interactions on off-targets, it is reasonable to predict these interactions by mining the chem.-protein interactome (CPI).  Making such predictions has recently been facilitated by a web server named DRAR-CPI.  This server has a representative collection of drug mols. and targetable human proteins built up from our work in drug repositioning and ADR.  When a user submits a mol., the server will give the pos. or neg. assocn. scores between the user's mol. and our library drugs based on their interaction profiles towards the targets.  Users can thus predict the indications or ADRs of their mol. based on the assocn. scores towards our library drugs.  We have matched our predictions of drug-drug assocns. with those predicted via gene-expression profiles, achieving a matching rate as high as 74%.  We have also successfully predicted the connections between anti-psychotics and anti-infectives, indicating the underlying relevance of anti-psychotics in the potential treatment of infections, vice versa.  This server is freely available at http://cpi.bio-x.cn/drar/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrltOyRWjrcwLVg90H21EOLACvtfcHk0lhzG9jMYEaUPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosVOntLg%253D&md5=5964882fff9031297e60b228d2217ece</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr299%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DMikailov%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DL.%26atitle%3DDRAR-CPI%253A%2520a%2520server%2520for%2520identifying%2520drug%2520repositioning%2520potential%2520and%2520adverse%2520drug%2520reactions%2520via%2520the%2520chemical-protein%2520interactome%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DW492%26epage%3D498%26doi%3D10.1093%2Fnar%2Fgkr299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref164"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref164'); return false;" data-citation="" class="refNumLink">164</a></strong><div class="NLM_citation" id="cit164"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wipf, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X. Q.</span></span> <span> </span><span class="NLM_article-title">TargetHunter: an in silico target identification tool for predicting therapeutic potential of small organic molecules based on chemogenomic database</span>. <i>AAPS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1208/s12248-012-9449-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1208%2Fs12248-012-9449-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23292636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsF2hsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=395-406&author=L.+Wangauthor=C.+Maauthor=P.+Wipfauthor=H.+Liuauthor=W.+Suauthor=X.+Q.+Xie&title=TargetHunter%3A+an+in+silico+target+identification+tool+for+predicting+therapeutic+potential+of+small+organic+molecules+based+on+chemogenomic+database&doi=10.1208%2Fs12248-012-9449-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">TargetHunter: An In Silico Target Identification Tool for Predicting Therapeutic Potential of Small Organic Molecules Based on Chemogenomic Database</span></div><div class="casAuthors">Wang, Lirong; Ma, Chao; Wipf, Peter; Liu, Haibin; Su, Weiwei; Xie, Xiang-Qun</div><div class="citationInfo"><span class="NLM_cas:title">AAPS Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">395-406</span>CODEN:
                <span class="NLM_cas:coden">AJAOB6</span>;
        ISSN:<span class="NLM_cas:issn">1550-7416</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Target identification of the known bioactive compds. and novel synthetic analogs is a very important research field in medicinal chem., biochem., and pharmacol.  It is also a challenging and costly step towards chem. biol. and phenotypic screening.  In silico identification of potential biol. targets for chem. compds. offers an alternative avenue for the exploration of ligand-target interactions and biochem. mechanisms, as well as for investigation of drug repurposing.  Computational target fishing mines biol. annotated chem. databases and then maps compd. structures into chemogenomical space in order to predict the biol. targets.  We summarize the recent advances and applications in computational target fishing, such as chem. similarity searching, data mining/machine learning, panel docking, and the bioactivity spectral anal. for target identification.  We then described in detail a new web-based target prediction tool, TargetHunter (http://www.cbligand.org/TargetHunter).  This web portal implements a novel in silico target prediction algorithm, the Targets Assocd. with its MOst SImilar Counterparts, by exploring the largest chemogenomical databases, ChEMBL.  Prediction accuracy reached 91.1% from the top 3 guesses on a subset of high-potency compds. from the ChEMBL database, which outperformed a published algorithm, multiple-category models.  TargetHunter also features an embedded geog. tool, BioassayGeoMap, developed to allow the user easily to search for potential collaborators that can exptl. validate the predicted biol. target(s) or off target(s).  TargetHunter therefore provides a promising alternative to bridge the knowledge gap between biol. and chem., and significantly boost the productivity of chemogenomics researchers for in silico drug design and discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrApY5ImkDQYbVg90H21EOLACvtfcHk0lh_EQc3QbqYKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsF2hsLk%253D&md5=964f41bd7191d200529b2f1272b7061c</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1208%2Fs12248-012-9449-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-012-9449-z%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DWipf%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DSu%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%2BQ.%26atitle%3DTargetHunter%253A%2520an%2520in%2520silico%2520target%2520identification%2520tool%2520for%2520predicting%2520therapeutic%2520potential%2520of%2520small%2520organic%2520molecules%2520based%2520on%2520chemogenomic%2520database%26jtitle%3DAAPS%2520J.%26date%3D2013%26volume%3D15%26spage%3D395%26epage%3D406%26doi%3D10.1208%2Fs12248-012-9449-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref165"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref165'); return false;" data-citation="" class="refNumLink">165</a></strong><div class="NLM_citation" id="cit165"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ku, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">ChemMapper: a versatile web server for exploring pharmacology and chemical structure association based on molecular 3D similarity method</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1827</span>– <span class="NLM_lpage">1829</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbioinformatics%2Fbtt270" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23712658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVOjsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=1827-1829&author=J.+Gongauthor=C.+Caiauthor=X.+Liuauthor=X.+Kuauthor=H.+Jiangauthor=D.+Gaoauthor=H.+Li&title=ChemMapper%3A+a+versatile+web+server+for+exploring+pharmacology+and+chemical+structure+association+based+on+molecular+3D+similarity+method&doi=10.1093%2Fbioinformatics%2Fbtt270"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">ChemMapper: a versatile web server for exploring pharmacology and chemical structure association based on molecular 3D similarity method</span></div><div class="casAuthors">Gong, Jiayu; Cai, Chaoqian; Liu, Xiaofeng; Ku, Xin; Jiang, Hualiang; Gao, Daqi; Li, Honglin</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1827-1829</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: ChemMapper is an online platform to predict polypharmacol. effect and mode of action for small mols. based on 3D similarity computation.  ChemMapper collects >350 000 chem. structures with bioactivities and assocd. target annotations (as well as >3 000 000 non-annotated compds. for virtual screening).  Taking the user-provided chem. structure as the query, the top most similar compds. in terms of 3D similarity are returned with assocd. pharmacol. annotations.  ChemMapper is designed to provide versatile services in a variety of chemogenomics, drug repurposing, polypharmacol., novel bioactive compds. identification and scaffold hopping studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaSkcKTpsV3rVg90H21EOLACvtfcHk0lh_EQc3QbqYKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVOjsrbN&md5=b13c728b5b001074ab41315be24231e1</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt270%26sid%3Dliteratum%253Aachs%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DKu%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DChemMapper%253A%2520a%2520versatile%2520web%2520server%2520for%2520exploring%2520pharmacology%2520and%2520chemical%2520structure%2520association%2520based%2520on%2520molecular%25203D%2520similarity%2520method%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D1827%26epage%3D1829%26doi%3D10.1093%2Fbioinformatics%2Fbtt270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref166"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref166'); return false;" data-citation="" class="refNumLink">166</a></strong><div class="NLM_citation" id="cit166"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haque, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillos, M.</span></span> <span> </span><span class="NLM_article-title">HitPick: a web server for hit identification and target prediction of chemical screenings</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1910</span>– <span class="NLM_lpage">1912</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbioinformatics%2Fbtt303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23716196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFClu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=1910-1912&author=X.+Liuauthor=I.+Vogtauthor=T.+Haqueauthor=M.+Campillos&title=HitPick%3A+a+web+server+for+hit+identification+and+target+prediction+of+chemical+screenings&doi=10.1093%2Fbioinformatics%2Fbtt303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">HitPick: a web server for hit identification and target prediction of chemical screenings</span></div><div class="casAuthors">Liu, Xueping; Vogt, Ingo; Haque, Tanzeem; Campillos, Monica</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1910-1912</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: High-throughput phenotypic assays reveal information about the mols. that modulate biol. processes, such as a disease phenotype and a signaling pathway.  In these assays, the identification of hits along with their mol. targets is crit. to understand the chem. activities modulating the biol. system.  Here, we present HitPick, a web server for identification of hits in high-throughput chem. screenings and prediction of their mol. targets.  HitPick applies the B-score method for hit identification and a newly developed approach combining 1-nearest-neighbor (1NN) similarity searching and Laplacian-modified naive Bayesian target models to predict targets of identified hits.  The performance of the HitPick web server is presented and discussed.  Availability: The server can be accessed at http://mips.helmholtz-muenchen.de/proj/hitpick.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="89e4e6e7e0eae8a7eae8e4f9e0e5e5e6fac9e1ece5e4e1e6e5fdf3a4e4fcece7eae1ece7a7edeca7">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpofqf_952o_LVg90H21EOLACvtfcHk0lh_EQc3QbqYKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFClu7vJ&md5=7acb7f7fbd75fa1cc02bd653d888cec3</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt303%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DVogt%26aufirst%3DI.%26aulast%3DHaque%26aufirst%3DT.%26aulast%3DCampillos%26aufirst%3DM.%26atitle%3DHitPick%253A%2520a%2520web%2520server%2520for%2520hit%2520identification%2520and%2520target%2520prediction%2520of%2520chemical%2520screenings%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D1910%26epage%3D1912%26doi%3D10.1093%2Fbioinformatics%2Fbtt303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref167"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref167'); return false;" data-citation="" class="refNumLink">167</a></strong><div class="NLM_citation" id="cit167"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Identifying the macromolecular targets of de novo-designed chemical entities through self-organizing map consensus</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">4067</span>– <span class="NLM_lpage">4072</span>, <span class="refDoi"> DOI: 10.1073/pnas.1320001111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.1320001111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24591595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2msbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=4067-4072&author=D.+Rekerauthor=T.+Rodriguesauthor=P.+Schneiderauthor=G.+Schneider&title=Identifying+the+macromolecular+targets+of+de+novo-designed+chemical+entities+through+self-organizing+map+consensus&doi=10.1073%2Fpnas.1320001111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying the macromolecular targets of de novo-designed chemical entities through self-organizing map consensus</span></div><div class="casAuthors">Reker, Daniel; Rodrigues, Tiago; Schneider, Petra; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4067-4072</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">De novo mol. design and in silico prediction of polypharmacol. profiles are emerging research topics that will profoundly affect the future of drug discovery and chem. biol.  The goal is to identify the macromol. targets of new chem. agents.  Although several computational tools for predicting such targets are publicly available, none of these methods was explicitly designed to predict target engagement by de novo-designed mols.  Here the authors present the development and practical application of a unique technique, self-organizing map-based prediction of drug equivalence relationships (SPiDER), that merges the concepts of self-organizing maps, consensus scoring, and statistical anal. to successfully identify targets for both known drugs and computer-generated mol. scaffolds.  The authors discovered a potential off-target liability of fenofibrate-related compds., and in a comprehensive prospective application, the authors identified a multitarget-modulating profile of de novo designed mols.  These results demonstrate that SPiDER may be used to identify innovative compds. in chem. biol. and in the early stages of drug discovery, and help investigate the potential side effects of drugs and their repurposing options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD_gPUwcpea7Vg90H21EOLACvtfcHk0lhonNeW-dwlUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2msbg%253D&md5=5322c8755c4aeee029288015d8a55d69</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1320001111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1320001111%26sid%3Dliteratum%253Aachs%26aulast%3DReker%26aufirst%3DD.%26aulast%3DRodrigues%26aufirst%3DT.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DIdentifying%2520the%2520macromolecular%2520targets%2520of%2520de%2520novo-designed%2520chemical%2520entities%2520through%2520self-organizing%2520map%2520consensus%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D4067%26epage%3D4072%26doi%3D10.1073%2Fpnas.1320001111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref168"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref168'); return false;" data-citation="" class="refNumLink">168</a></strong><div class="NLM_citation" id="cit168"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gfeller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosdidier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissTargetPrediction: a web server for target prediction of bioactive small molecules</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">W32</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1093/nar/gku293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgku293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24792161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W32-38&author=D.+Gfellerauthor=A.+Grosdidierauthor=M.+Wirthauthor=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissTargetPrediction%3A+a+web+server+for+target+prediction+of+bioactive+small+molecules&doi=10.1093%2Fnar%2Fgku293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">SwissTargetPrediction: a web server for target prediction of bioactive small molecules</span></div><div class="casAuthors">Gfeller, David; Grosdidier, Aurelien; Wirth, Matthias; Daina, Antoine; Michielin, Olivier; Zoete, Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W32-W38</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bioactive small mols., such as drugs or metabolites, bind to proteins or other macro-mol. targets to modulate their activity, which in turn results in the obsd. phenotypic effects.  For this reason, mapping the targets of bioactive small mols. is a key step toward unraveling the mol. mechanisms underlying their bioactivity and predicting potential side effects or cross-reactivity.  Recently, large datasets of protein-small mol. interactions have become available, providing a unique source of information for the development of knowledge-based approaches to computationally identify new targets for uncharacterized mols. or secondary targets for known mols.  Here, the authors introduce SwissTargetPrediction, a web server to accurately predict the targets of bioactive mols. based on a combination of 2D and 3D similarity measures with known ligands.  Predictions can be carried out in five different organisms, and mapping predictions by homol. within and between different species is enabled for close paralogs and orthologs.  SwissTargetPrediction is accessible free of charge and without login requirement at http://www.swisstargetprediction.ch.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXsaXtGKzkj7Vg90H21EOLACvtfcHk0lhonNeW-dwlUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs73E&md5=77811c28978158e1e1aca0669a85b99c</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku293%26sid%3Dliteratum%253Aachs%26aulast%3DGfeller%26aufirst%3DD.%26aulast%3DGrosdidier%26aufirst%3DA.%26aulast%3DWirth%26aufirst%3DM.%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissTargetPrediction%253A%2520a%2520web%2520server%2520for%2520target%2520prediction%2520of%2520bioactive%2520small%2520molecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DW32%26epage%3D38%26doi%3D10.1093%2Fnar%2Fgku293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref169"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref169'); return false;" data-citation="" class="refNumLink">169</a></strong><div class="NLM_citation" id="cit169"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nickel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erehman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goede, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span> <span> </span><span class="NLM_article-title">SuperPred: update on drug classification and target prediction</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">W26</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1093/nar/gku477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgku477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24878925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W26-31&author=J.+Nickelauthor=B.+O.+Gohlkeauthor=J.+Erehmanauthor=P.+Banerjeeauthor=W.+W.+Rongauthor=A.+Goedeauthor=M.+Dunkelauthor=R.+Preissner&title=SuperPred%3A+update+on+drug+classification+and+target+prediction&doi=10.1093%2Fnar%2Fgku477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">SuperPred: update on drug classification and target prediction</span></div><div class="casAuthors">Nickel, Janette; Gohlke, Bjoern-Oliver; Erehman, Jevgeni; Banerjee, Priyanka; Rong, Wen Wei; Goede, Andrean; Dunkel, Mathias; Preissner, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W26-W31</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The SuperPred web server connects chem. similarity of drug-like compds. with mol. targets and the therapeutic approach based on the similar property principle.  Since the first release of this server, the no. of known compd.-target interactions has increased from 7000 to 665 000, which allows not only a better prediction quality but also the estn. of a confidence.  Apart from the addn. of quant. binding data and the statistical consideration of the similarity distribution in all drug classes, new approaches were implemented to improve the target prediction.  The 3D similarity as well as the occurrence of fragments and the concordance of physico-chem. properties is also taken into account.  In addn., the effect of different fingerprints on the prediction was examd.  The retrospective prediction of a drug class (ATC code of the WHO) allows the evaluation of methods and descriptors for a well-characterized set of approved drugs.  The prediction is improved by 7.5% to a total accuracy of 75.1%.  For query compds. with sufficient structural similarity, the web server allows prognoses about the medical indication area of novel compds. and to find new leads for known targets.  SuperPred is publicly available without registration at: http://prediction.charite.de.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWNMDss7U4k7Vg90H21EOLACvtfcHk0lhonNeW-dwlUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs7rF&md5=e23cf651ec0f4f7784104e6b198a014c</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku477%26sid%3Dliteratum%253Aachs%26aulast%3DNickel%26aufirst%3DJ.%26aulast%3DGohlke%26aufirst%3DB.%2BO.%26aulast%3DErehman%26aufirst%3DJ.%26aulast%3DBanerjee%26aufirst%3DP.%26aulast%3DRong%26aufirst%3DW.%2BW.%26aulast%3DGoede%26aufirst%3DA.%26aulast%3DDunkel%26aufirst%3DM.%26aulast%3DPreissner%26aufirst%3DR.%26atitle%3DSuperPred%253A%2520update%2520on%2520drug%2520classification%2520and%2520target%2520prediction%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DW26%26epage%3D31%26doi%3D10.1093%2Fnar%2Fgku477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref170"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref170'); return false;" data-citation="" class="refNumLink">170</a></strong><div class="NLM_citation" id="cit170"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span> <span> </span><span class="NLM_article-title">TarPred: a web application for predicting therapeutic and side effect targets of chemical compounds</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2049</span>– <span class="NLM_lpage">2051</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btv099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbioinformatics%2Fbtv099" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=2049-2051&author=X.+Liuauthor=Y.+Gaoauthor=J.+Pengauthor=Y.+Xuauthor=Y.+Wangauthor=N.+Zhouauthor=J.+Xingauthor=X.+Luoauthor=H.+Jiangauthor=M.+Zheng&title=TarPred%3A+a+web+application+for+predicting+therapeutic+and+side+effect+targets+of+chemical+compounds&doi=10.1093%2Fbioinformatics%2Fbtv099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtv099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtv099%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DN.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DM.%26atitle%3DTarPred%253A%2520a%2520web%2520application%2520for%2520predicting%2520therapeutic%2520and%2520side%2520effect%2520targets%2520of%2520chemical%2520compounds%26jtitle%3DBioinformatics%26date%3D2015%26volume%3D31%26spage%3D2049%26epage%3D2051%26doi%3D10.1093%2Fbioinformatics%2Fbtv099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref171"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref171'); return false;" data-citation="" class="refNumLink">171</a></strong><div class="NLM_citation" id="cit171"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artimo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonnalagedda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baratin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Castro, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flegel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasteiger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosdidier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioannidis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liechti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostaguir, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redaschi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenarios, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockinger, H.</span></span> <span> </span><span class="NLM_article-title">ExPASy: SIB bioinformatics resource portal</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">W597</span>– <span class="NLM_lpage">603</span>, <span class="refDoi"> DOI: 10.1093/nar/gks400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgks400" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=22661580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCqu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=W597-603&author=P.+Artimoauthor=M.+Jonnalageddaauthor=K.+Arnoldauthor=D.+Baratinauthor=G.+Csardiauthor=E.+de+Castroauthor=S.+Duvaudauthor=V.+Flegelauthor=A.+Fortierauthor=E.+Gasteigerauthor=A.+Grosdidierauthor=C.+Hernandezauthor=V.+Ioannidisauthor=D.+Kuznetsovauthor=R.+Liechtiauthor=S.+Morettiauthor=K.+Mostaguirauthor=N.+Redaschiauthor=G.+Rossierauthor=I.+Xenariosauthor=H.+Stockinger&title=ExPASy%3A+SIB+bioinformatics+resource+portal&doi=10.1093%2Fnar%2Fgks400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">ExPASy: SIB bioinformatics resource portal</span></div><div class="casAuthors">Artimo, Panu; Jonnalagedda, Manohar; Arnold, Konstantin; Baratin, Delphine; Csardi, Gabor; de Castro, Edouard; Duvaud, Severine; Flegel, Volker; Fortier, Arnaud; Gasteiger, Elisabeth; Grosdidier, Aurelien; Hernandez, Celine; Ioannidis, Vassilios; Kuznetsov, Dmitry; Liechti, Robin; Moretti, Sebastien; Mostaguir, Khaled; Redaschi, Nicole; Rossier, Gregoire; Xenarios, Ioannis; Stockinger, Heinz</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W597-W603</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ExPASy (http://www.expasy.org) has worldwide reputation as one of the main bioinformatics resources for proteomics.  It has now evolved, becoming an extensible and integrative portal accessing many scientific resources, databases and software tools in different areas of life sciences.  Scientists can henceforth access seamlessly a wide range of resources in many different domains, such as proteomics, genomics, phylogeny/evolution, systems biol., population genetics, transcriptomics, etc.  The individual resources (databases, web-based and downloadable software tools) are hosted in a  decentralized' way by different groups of the SIB Swiss Institute of Bioinformatics and partner institutions.  Specifically, a single web portal provides a common entry point to a wide range of resources developed and operated by different SIB groups and external institutions.  The portal features a search function across  selected' resources.  Addnl., the availability and usage of resources are monitored.  The portal is aimed for both expert users and people who are not familiar with a specific domain in life sciences.  The new web interface provides, in particular, visual guidance for newcomers to ExPASy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_eGs293-q2LVg90H21EOLACvtfcHk0lgkIlW-A4t2ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCqu7c%253D&md5=8fc906725e3985a75ae0d15c01c30fde</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks400%26sid%3Dliteratum%253Aachs%26aulast%3DArtimo%26aufirst%3DP.%26aulast%3DJonnalagedda%26aufirst%3DM.%26aulast%3DArnold%26aufirst%3DK.%26aulast%3DBaratin%26aufirst%3DD.%26aulast%3DCsardi%26aufirst%3DG.%26aulast%3Dde%2BCastro%26aufirst%3DE.%26aulast%3DDuvaud%26aufirst%3DS.%26aulast%3DFlegel%26aufirst%3DV.%26aulast%3DFortier%26aufirst%3DA.%26aulast%3DGasteiger%26aufirst%3DE.%26aulast%3DGrosdidier%26aufirst%3DA.%26aulast%3DHernandez%26aufirst%3DC.%26aulast%3DIoannidis%26aufirst%3DV.%26aulast%3DKuznetsov%26aufirst%3DD.%26aulast%3DLiechti%26aufirst%3DR.%26aulast%3DMoretti%26aufirst%3DS.%26aulast%3DMostaguir%26aufirst%3DK.%26aulast%3DRedaschi%26aufirst%3DN.%26aulast%3DRossier%26aufirst%3DG.%26aulast%3DXenarios%26aufirst%3DI.%26aulast%3DStockinger%26aufirst%3DH.%26atitle%3DExPASy%253A%2520SIB%2520bioinformatics%2520resource%2520portal%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DW597%26epage%3D603%26doi%3D10.1093%2Fnar%2Fgks400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref172"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref172'); return false;" data-citation="" class="refNumLink">172</a></strong><div class="NLM_citation" id="cit172"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lgkIlW-A4t2ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149'],'ref150':['cit150'],'ref151':['cit151'],'ref152':['cit152'],'ref153':['cit153'],'ref154':['cit154'],'ref155':['cit155'],'ref156':['cit156'],'ref157':['cit157'],'ref158':['cit158'],'ref159':['cit159'],'ref160':['cit160'],'ref161':['cit161'],'ref162':['cit162'],'ref163':['cit163'],'ref164':['cit164'],'ref165':['cit165'],'ref166':['cit166'],'ref167':['cit167'],'ref168':['cit168'],'ref169':['cit169'],'ref170':['cit170'],'ref171':['cit171'],'ref172':['cit172']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 32 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Rachel Friedman
Ohana, Sergiy Levin, Robin Hurst, Michael M. Rosenblatt, Kristopher Zimmerman, Thomas Machleidt, Keith V. Wood, <span class="NLM_string-name hlFld-ContribAuthor">Thomas A. Kirkland</span>. </span><span class="cited-content_cbyCitation_article-title">Streamlined Target Deconvolution Approach Utilizing a Single Photoreactive Chloroalkane Capture Tag. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (2)
                                     , 404-413. <a href="https://doi.org/10.1021/acschembio.0c00987" title="DOI URL">https://doi.org/10.1021/acschembio.0c00987</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00987%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DStreamlined%252BTarget%252BDeconvolution%252BApproach%252BUtilizing%252Ba%252BSingle%252BPhotoreactive%252BChloroalkane%252BCapture%252BTag%26aulast%3DFriedman%2BOhana%26aufirst%3DRachel%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D22122020%26date%3D22012021%26date%3D05022021%26volume%3D16%26issue%3D2%26spage%3D404%26epage%3D413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rin Miyajima, Koji Sakai, Yuki Otani, Takashi Wadatsu, Yasuyo Sakata, Yuki Nishikawa, Masaki Tanaka, Yu Yamashita, Mikayo Hayashi, Kazumi Kondo, <span class="NLM_string-name hlFld-ContribAuthor">Takashi Hayashi</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Tetrafunctional Probes Identify Target Receptors and Binding Sites of Small-Molecule Drugs from Living Systems. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (9)
                                     , 2364-2373. <a href="https://doi.org/10.1021/acschembio.0c00335" title="DOI URL">https://doi.org/10.1021/acschembio.0c00335</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00335%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DNovel%252BTetrafunctional%252BProbes%252BIdentify%252BTarget%252BReceptors%252Band%252BBinding%252BSites%252Bof%252BSmall-Molecule%252BDrugs%252Bfrom%252BLiving%252BSystems%26aulast%3DMiyajima%26aufirst%3DRin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29042020%26date%3D04082020%26date%3D20082020%26date%3D04082020%26volume%3D15%26issue%3D9%26spage%3D2364%26epage%3D2373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mudasir Nabi Peerzada, Parvez Khan, Nashrah Sharif Khan, Fernando Avecilla, Shadab Miyan Siddiqui, Md. Imtaiyaz Hassan, <span class="NLM_string-name hlFld-ContribAuthor">Amir Azam</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Development of Small-Molecule Arylaldoxime/5-Nitroimidazole Hybrids as Potent Inhibitors of MARK4: A Promising Approach for Target-Based Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (36)
                                     , 22759-22771. <a href="https://doi.org/10.1021/acsomega.0c01703" title="DOI URL">https://doi.org/10.1021/acsomega.0c01703</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c01703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c01703%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DDesign%252Band%252BDevelopment%252Bof%252BSmall-Molecule%252BArylaldoxime%25252F5-Nitroimidazole%252BHybrids%252Bas%252BPotent%252BInhibitors%252Bof%252BMARK4%25253A%252BA%252BPromising%252BApproach%252Bfor%252BTarget-Based%252BCancer%252BTherapy%26aulast%3DPeerzada%26aufirst%3DMudasir%2BNabi%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14042020%26date%3D17072020%26date%3D01092020%26volume%3D5%26issue%3D36%26spage%3D22759%26epage%3D22771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hiroyuki Takada, Akira Kaieda, Michiko Tawada, Tomoko Nagino, Katsunori Sasa, Tatsuo Oikawa, Akiko Oki, Tomoya Sameshima, Kazumasa Miyamoto, Makoto Miyamoto, Yuko Kokubu, Ryuichi Tozawa, Hidetoshi Sakurai, <span class="NLM_string-name hlFld-ContribAuthor">Bunnai Saito</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of 2,6-Disubstituted 3H-Imidazo[4,5-b]pyridines as Therapeutic Agents for Dysferlinopathies through Phenotypic Screening on Patient-Derived Induced Pluripotent Stem Cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (20)
                                     , 9175-9187. <a href="https://doi.org/10.1021/acs.jmedchem.9b01100" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01100%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252B2%25252C6-Disubstituted%252B3H-Imidazo%25255B4%25252C5-b%25255Dpyridines%252Bas%252BTherapeutic%252BAgents%252Bfor%252BDysferlinopathies%252Bthrough%252BPhenotypic%252BScreening%252Bon%252BPatient-Derived%252BInduced%252BPluripotent%252BStem%252BCells%26aulast%3DTakada%26aufirst%3DHiroyuki%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08072019%26date%3D07102019%26date%3D24092019%26volume%3D62%26issue%3D20%26spage%3D9175%26epage%3D9187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sayan Roy Chowdhury, Subrata Mondal, <span class="NLM_string-name hlFld-ContribAuthor">Parameswar Krishnan Iyer</span>. </span><span class="cited-content_cbyCitation_article-title">Blocking Oligomeric Insulin Amyloid Fibrillation via Perylenebisimides Containing Dipeptide Tentacles. </span><span class="cited-content_cbyCitation_journal-name">ACS Biomaterials Science & Engineering</span><span> <strong>2018,</strong> <em>4 </em>
                                    (12)
                                     , 4076-4083. <a href="https://doi.org/10.1021/acsbiomaterials.8b00927" title="DOI URL">https://doi.org/10.1021/acsbiomaterials.8b00927</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsbiomaterials.8b00927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsbiomaterials.8b00927%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Biomaterials%2520Science%2520%2526%2520Engineering%26atitle%3DBlocking%252BOligomeric%252BInsulin%252BAmyloid%252BFibrillation%252Bvia%252BPerylenebisimides%252BContaining%252BDipeptide%252BTentacles%26aulast%3DRoy%2BChowdhury%26aufirst%3DSayan%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D07082018%26date%3D27092018%26date%3D12102018%26date%3D27092018%26volume%3D4%26issue%3D12%26spage%3D4076%26epage%3D4083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jennie R.  Lill</span>, <span class="hlFld-ContribAuthor ">William R.  Mathews</span>, <span class="hlFld-ContribAuthor ">Christopher M.  Rose</span>, <span class="hlFld-ContribAuthor ">Markus  Schirle</span>. </span><span class="cited-content_cbyCitation_article-title">Proteomics in the pharmaceutical and biotechnology industry: a look to the next decade. </span><span class="cited-content_cbyCitation_journal-name">Expert Review of Proteomics</span><span> <strong>2021,</strong> <em> </em><a href="https://doi.org/10.1080/14789450.2021.1962300" title="DOI URL">https://doi.org/10.1080/14789450.2021.1962300</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14789450.2021.1962300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14789450.2021.1962300%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Review%2520of%2520Proteomics%26atitle%3DProteomics%252Bin%252Bthe%252Bpharmaceutical%252Band%252Bbiotechnology%252Bindustry%25253A%252Ba%252Blook%252Bto%252Bthe%252Bnext%252Bdecade%26aulast%3DLill%26aufirst%3DJennie%2BR.%26date%3D2021%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olívia Basso  Rocha</span>, <span class="hlFld-ContribAuthor ">Lívia  do Carmo Silva</span>, <span class="hlFld-ContribAuthor ">Marcos Antonio Batista  de Carvalho Júnior</span>, <span class="hlFld-ContribAuthor ">Amanda Alves  de Oliveira</span>, <span class="hlFld-ContribAuthor ">Célia Maria  de Almeida Soares</span>, <span class="hlFld-ContribAuthor ">Maristela  Pereira</span>. </span><span class="cited-content_cbyCitation_article-title">In vitro and in silico analysis reveals antifungal activity and potential targets of curcumin on Paracoccidioides spp.. </span><span class="cited-content_cbyCitation_journal-name">Brazilian Journal of Microbiology</span><span> <strong>2021,</strong> <em>17 </em><a href="https://doi.org/10.1007/s42770-021-00548-6" title="DOI URL">https://doi.org/10.1007/s42770-021-00548-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s42770-021-00548-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs42770-021-00548-6%26sid%3Dliteratum%253Aachs%26jtitle%3DBrazilian%2520Journal%2520of%2520Microbiology%26atitle%3DIn%252Bvitro%252Band%252Bin%252Bsilico%252Banalysis%252Breveals%252Bantifungal%252Bactivity%252Band%252Bpotential%252Btargets%252Bof%252Bcurcumin%252Bon%252BParacoccidioides%252Bspp.%26aulast%3DRocha%26aufirst%3DOl%25C3%25ADvia%2BBasso%26date%3D2021%26date%3D2021%26volume%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.  Aishwarya</span>, <span class="hlFld-ContribAuthor ">K.  Gunasekaran</span>, <span class="hlFld-ContribAuthor ">R.  Sagaya Jansi</span>, <span class="hlFld-ContribAuthor ">G.  Sangeetha</span>. </span><span class="cited-content_cbyCitation_article-title">From genomes to molecular dynamics – A bottom up approach in extrication of SARS CoV-2 main protease inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Computational Toxicology</span><span> <strong>2021,</strong> <em>18 </em>, 100156. <a href="https://doi.org/10.1016/j.comtox.2021.100156" title="DOI URL">https://doi.org/10.1016/j.comtox.2021.100156</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.comtox.2021.100156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.comtox.2021.100156%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Toxicology%26atitle%3DFrom%252Bgenomes%252Bto%252Bmolecular%252Bdynamics%252B%2525E2%252580%252593%252BA%252Bbottom%252Bup%252Bapproach%252Bin%252Bextrication%252Bof%252BSARS%252BCoV-2%252Bmain%252Bprotease%252Binhibitors%26aulast%3DAishwarya%26aufirst%3DS.%26date%3D2021%26volume%3D18%26spage%3D100156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aini  Vuorinen</span>, <span class="hlFld-ContribAuthor ">Isabel V.L.  Wilkinson</span>, <span class="hlFld-ContribAuthor ">Maria  Chatzopoulou</span>, <span class="hlFld-ContribAuthor ">Ben  Edwards</span>, <span class="hlFld-ContribAuthor ">Sarah E.  Squire</span>, <span class="hlFld-ContribAuthor ">Rebecca J.  Fairclough</span>, <span class="hlFld-ContribAuthor ">Noelia Araujo  Bazan</span>, <span class="hlFld-ContribAuthor ">Josh A.  Milner</span>, <span class="hlFld-ContribAuthor ">Daniel  Conole</span>, <span class="hlFld-ContribAuthor ">James R.  Donald</span>, <span class="hlFld-ContribAuthor ">Nandini  Shah</span>, <span class="hlFld-ContribAuthor ">Nicky J.  Willis</span>, <span class="hlFld-ContribAuthor ">Fernando Martinez  Vazquez</span>, <span class="hlFld-ContribAuthor ">Francis X.  Wilson</span>, <span class="hlFld-ContribAuthor ">Graham M.  Wynne</span>, <span class="hlFld-ContribAuthor ">Stephen G.  Davies</span>, <span class="hlFld-ContribAuthor ">Kay E.  Davies</span>, <span class="hlFld-ContribAuthor ">Angela J.  Russell</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Mechanism of Action studies of 4,6-diphenylpyrimidine-2-carbohydrazides as utrophin modulators for the treatment of Duchenne muscular dystrophy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>34 </em>, 113431. <a href="https://doi.org/10.1016/j.ejmech.2021.113431" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113431</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113431%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BMechanism%252Bof%252BAction%252Bstudies%252Bof%252B4%25252C6-diphenylpyrimidine-2-carbohydrazides%252Bas%252Butrophin%252Bmodulators%252Bfor%252Bthe%252Btreatment%252Bof%252BDuchenne%252Bmuscular%252Bdystrophy%26aulast%3DVuorinen%26aufirst%3DAini%26date%3D2021%26volume%3D34%26spage%3D113431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rebecca E.  Hughes</span>, <span class="hlFld-ContribAuthor ">Richard J.R.  Elliott</span>, <span class="hlFld-ContribAuthor ">John C.  Dawson</span>, <span class="hlFld-ContribAuthor ">Neil O.  Carragher</span>. </span><span class="cited-content_cbyCitation_article-title">High-content phenotypic and pathway profiling to advance drug discovery in diseases of unmet need. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (3)
                                     , 338-355. <a href="https://doi.org/10.1016/j.chembiol.2021.02.015" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.02.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.02.015%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DHigh-content%252Bphenotypic%252Band%252Bpathway%252Bprofiling%252Bto%252Badvance%252Bdrug%252Bdiscovery%252Bin%252Bdiseases%252Bof%252Bunmet%252Bneed%26aulast%3DHughes%26aufirst%3DRebecca%2BE.%26date%3D2021%26volume%3D28%26issue%3D3%26spage%3D338%26epage%3D355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Louis P.  Conway</span>, <span class="hlFld-ContribAuthor ">Weichao  Li</span>, <span class="hlFld-ContribAuthor ">Christopher G.  Parker</span>. </span><span class="cited-content_cbyCitation_article-title">Chemoproteomic-enabled phenotypic screening. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (3)
                                     , 371-393. <a href="https://doi.org/10.1016/j.chembiol.2021.01.012" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.01.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.01.012%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DChemoproteomic-enabled%252Bphenotypic%252Bscreening%26aulast%3DConway%26aufirst%3DLouis%2BP.%26date%3D2021%26volume%3D28%26issue%3D3%26spage%3D371%26epage%3D393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Uthpala  Seneviratne</span>, <span class="hlFld-ContribAuthor ">Zhen  Huang</span>, <span class="hlFld-ContribAuthor ">Christopher W.  am Ende</span>, <span class="hlFld-ContribAuthor ">Todd W.  Butler</span>, <span class="hlFld-ContribAuthor ">Leah  Cleary</span>, <span class="hlFld-ContribAuthor ">Erica  Dresselhaus</span>, <span class="hlFld-ContribAuthor ">Edelweiss  Evrard</span>, <span class="hlFld-ContribAuthor ">Ethan L.  Fisher</span>, <span class="hlFld-ContribAuthor ">Michael E.  Green</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Helal</span>, <span class="hlFld-ContribAuthor ">John M.  Humphrey</span>, <span class="hlFld-ContribAuthor ">Lorraine F.  Lanyon</span>, <span class="hlFld-ContribAuthor ">Michael  Marconi</span>, <span class="hlFld-ContribAuthor ">Paramita  Mukherjee</span>, <span class="hlFld-ContribAuthor ">Simone  Sciabola</span>, <span class="hlFld-ContribAuthor ">Claire M.  Steppan</span>, <span class="hlFld-ContribAuthor ">Emily K.  Sylvain</span>, <span class="hlFld-ContribAuthor ">Jamison B.  Tuttle</span>, <span class="hlFld-ContribAuthor ">Patrick R.  Verhoest</span>, <span class="hlFld-ContribAuthor ">Travis T.  Wager</span>, <span class="hlFld-ContribAuthor ">Longfei  Xie</span>, <span class="hlFld-ContribAuthor ">Gayathri  Ramaswamy</span>, <span class="hlFld-ContribAuthor ">Douglas S.  Johnson</span>, <span class="hlFld-ContribAuthor ">Martin  Pettersson</span>. </span><span class="cited-content_cbyCitation_article-title">Photoaffinity Labeling and Quantitative Chemical Proteomics Identify LXRβ as the Functional Target of Enhancers of Astrocytic apoE. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (2)
                                     , 148-157.e7. <a href="https://doi.org/10.1016/j.chembiol.2020.09.002" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.09.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.09.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DPhotoaffinity%252BLabeling%252Band%252BQuantitative%252BChemical%252BProteomics%252BIdentify%252BLXR%2525CE%2525B2%252Bas%252Bthe%252BFunctional%252BTarget%252Bof%252BEnhancers%252Bof%252BAstrocytic%252BapoE%26aulast%3DSeneviratne%26aufirst%3DUthpala%26date%3D2021%26volume%3D28%26issue%3D2%26spage%3D148%26epage%3D157.e7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jyoti  Kode</span>, <span class="hlFld-ContribAuthor ">Jeshma  Kovvuri</span>, <span class="hlFld-ContribAuthor ">Burri  Nagaraju</span>, <span class="hlFld-ContribAuthor ">Shailesh  Jadhav</span>, <span class="hlFld-ContribAuthor ">Madan  Barkume</span>, <span class="hlFld-ContribAuthor ">Subrata  Sen</span>, <span class="hlFld-ContribAuthor ">Nirmal Kumar  Kasinathan</span>, <span class="hlFld-ContribAuthor ">Pradip  Chaudhari</span>, <span class="hlFld-ContribAuthor ">Bhabani Shankar  Mohanty</span>, <span class="hlFld-ContribAuthor ">Jitendra  Gour</span>, <span class="hlFld-ContribAuthor ">Dilep Kumar  Sigalapalli</span>, <span class="hlFld-ContribAuthor ">C.  Ganesh Kumar</span>, <span class="hlFld-ContribAuthor ">Trupti  Pradhan</span>, <span class="hlFld-ContribAuthor ">Manisha  Banerjee</span>, <span class="hlFld-ContribAuthor ">Ahmed  Kamal</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, biological evaluation, and molecular docking analysis of phenstatin based indole linked chalcones as anticancer agents and tubulin polymerization inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>105 </em>, 104447. <a href="https://doi.org/10.1016/j.bioorg.2020.104447" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104447</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104447%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DSynthesis%25252C%252Bbiological%252Bevaluation%25252C%252Band%252Bmolecular%252Bdocking%252Banalysis%252Bof%252Bphenstatin%252Bbased%252Bindole%252Blinked%252Bchalcones%252Bas%252Banticancer%252Bagents%252Band%252Btubulin%252Bpolymerization%252Binhibitors%26aulast%3DKode%26aufirst%3DJyoti%26date%3D2020%26volume%3D105%26spage%3D104447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anutthaman  Parthasarathy</span>, <span class="hlFld-ContribAuthor ">Pavan K.  Mantravadi</span>, <span class="hlFld-ContribAuthor ">Karunakaran  Kalesh</span>. </span><span class="cited-content_cbyCitation_article-title">Detectives and helpers: Natural products as resources for chemical probes and compound libraries. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2020,</strong> <em>216 </em>, 107688. <a href="https://doi.org/10.1016/j.pharmthera.2020.107688" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2020.107688</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2020.107688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2020.107688%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DDetectives%252Band%252Bhelpers%25253A%252BNatural%252Bproducts%252Bas%252Bresources%252Bfor%252Bchemical%252Bprobes%252Band%252Bcompound%252Blibraries%26aulast%3DParthasarathy%26aufirst%3DAnutthaman%26date%3D2020%26volume%3D216%26spage%3D107688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao  Chen</span>, <span class="hlFld-ContribAuthor ">Yutong  Wang</span>, <span class="hlFld-ContribAuthor ">Nan  Ma</span>, <span class="hlFld-ContribAuthor ">Jing  Tian</span>, <span class="hlFld-ContribAuthor ">Yurou  Shao</span>, <span class="hlFld-ContribAuthor ">Bo  Zhu</span>, <span class="hlFld-ContribAuthor ">Yin Kwan  Wong</span>, <span class="hlFld-ContribAuthor ">Zhen  Liang</span>, <span class="hlFld-ContribAuthor ">Chang  Zou</span>, <span class="hlFld-ContribAuthor ">Jigang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Target identification of natural medicine with chemical proteomics approach: probe synthesis, target fishing and protein identification. </span><span class="cited-content_cbyCitation_journal-name">Signal Transduction and Targeted Therapy</span><span> <strong>2020,</strong> <em>5 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41392-020-0186-y" title="DOI URL">https://doi.org/10.1038/s41392-020-0186-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41392-020-0186-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41392-020-0186-y%26sid%3Dliteratum%253Aachs%26jtitle%3DSignal%2520Transduction%2520and%2520Targeted%2520Therapy%26atitle%3DTarget%252Bidentification%252Bof%252Bnatural%252Bmedicine%252Bwith%252Bchemical%252Bproteomics%252Bapproach%25253A%252Bprobe%252Bsynthesis%25252C%252Btarget%252Bfishing%252Band%252Bprotein%252Bidentification%26aulast%3DChen%26aufirst%3DXiao%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro F.  Dalberto</span>, <span class="hlFld-ContribAuthor ">Eduardo V.  de Souza</span>, <span class="hlFld-ContribAuthor ">Bruno L.  Abbadi</span>, <span class="hlFld-ContribAuthor ">Christiano E.  Neves</span>, <span class="hlFld-ContribAuthor ">Raoní S.  Rambo</span>, <span class="hlFld-ContribAuthor ">Alessandro S.  Ramos</span>, <span class="hlFld-ContribAuthor ">Fernanda S.  Macchi</span>, <span class="hlFld-ContribAuthor ">Pablo  Machado</span>, <span class="hlFld-ContribAuthor ">Cristiano V.  Bizarro</span>, <span class="hlFld-ContribAuthor ">Luiz A.  Basso</span>. </span><span class="cited-content_cbyCitation_article-title">Handling the Hurdles on the Way to Anti-tuberculosis Drug Development. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fchem.2020.586294" title="DOI URL">https://doi.org/10.3389/fchem.2020.586294</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2020.586294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2020.586294%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DHandling%252Bthe%252BHurdles%252Bon%252Bthe%252BWay%252Bto%252BAnti-tuberculosis%252BDrug%252BDevelopment%26aulast%3DDalberto%26aufirst%3DPedro%2BF.%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabien  Vincent</span>, <span class="hlFld-ContribAuthor ">Paula M.  Loria</span>, <span class="hlFld-ContribAuthor ">Andrea D.  Weston</span>, <span class="hlFld-ContribAuthor ">Claire M.  Steppan</span>, <span class="hlFld-ContribAuthor ">Regis  Doyonnas</span>, <span class="hlFld-ContribAuthor ">Yue-Ming  Wang</span>, <span class="hlFld-ContribAuthor ">Kristin L.  Rockwell</span>, <span class="hlFld-ContribAuthor ">Marie-Claire  Peakman</span>. </span><span class="cited-content_cbyCitation_article-title">Hit Triage and Validation in Phenotypic Screening: Considerations and Strategies. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (11)
                                     , 1332-1346. <a href="https://doi.org/10.1016/j.chembiol.2020.08.009" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.08.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.08.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DHit%252BTriage%252Band%252BValidation%252Bin%252BPhenotypic%252BScreening%25253A%252BConsiderations%252Band%252BStrategies%26aulast%3DVincent%26aufirst%3DFabien%26date%3D2020%26volume%3D27%26issue%3D11%26spage%3D1332%26epage%3D1346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabel V.L.  Wilkinson</span>, <span class="hlFld-ContribAuthor ">Georg C.  Terstappen</span>, <span class="hlFld-ContribAuthor ">Angela J.  Russell</span>. </span><span class="cited-content_cbyCitation_article-title">Combining experimental strategies for successful target deconvolution. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (11)
                                     , 1998-2005. <a href="https://doi.org/10.1016/j.drudis.2020.09.016" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.09.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.09.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.09.016%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DCombining%252Bexperimental%252Bstrategies%252Bfor%252Bsuccessful%252Btarget%252Bdeconvolution%26aulast%3DWilkinson%26aufirst%3DIsabel%2BV.L.%26date%3D2020%26volume%3D25%26issue%3D11%26spage%3D1998%26epage%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mudasir Nabi  Peerzada</span>, <span class="hlFld-ContribAuthor ">Parvez  Khan</span>, <span class="hlFld-ContribAuthor ">Nashrah Sharif  Khan</span>, <span class="hlFld-ContribAuthor ">Aysha  Gaur</span>, <span class="hlFld-ContribAuthor ">Fernando  Avecilla</span>, <span class="hlFld-ContribAuthor ">Md. Imtaiyaz  Hassan</span>, <span class="hlFld-ContribAuthor ">Amir  Azam</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of morpholine based hydroxylamine analogues: selective inhibitors of MARK4/Par-1d causing cancer cell death through apoptosis. </span><span class="cited-content_cbyCitation_journal-name">New Journal of Chemistry</span><span> <strong>2020,</strong> <em>44 </em>
                                    (38)
                                     , 16626-16637. <a href="https://doi.org/10.1039/D0NJ03474F" title="DOI URL">https://doi.org/10.1039/D0NJ03474F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0NJ03474F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0NJ03474F%26sid%3Dliteratum%253Aachs%26jtitle%3DNew%2520Journal%2520of%2520Chemistry%26atitle%3DIdentification%252Bof%252Bmorpholine%252Bbased%252Bhydroxylamine%252Banalogues%25253A%252Bselective%252Binhibitors%252Bof%252BMARK4%25252FPar-1d%252Bcausing%252Bcancer%252Bcell%252Bdeath%252Bthrough%252Bapoptosis%26aulast%3DPeerzada%26aufirst%3DMudasir%2BNabi%26date%3D2020%26date%3D2020%26volume%3D44%26issue%3D38%26spage%3D16626%26epage%3D16637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Breidert</span>, <span class="hlFld-ContribAuthor ">Pierre  Eftekhari</span>, <span class="hlFld-ContribAuthor ">François  Louis</span>, <span class="hlFld-ContribAuthor ">Claudia  Rotoiu</span>, <span class="hlFld-ContribAuthor ">Timo  Rath</span>, <span class="hlFld-ContribAuthor ">Markus F  Neurath</span>, <span class="hlFld-ContribAuthor ">Raja  Atreya</span>. </span><span class="cited-content_cbyCitation_article-title">Functional Molecular Network Analysis Enables Prediction of Response to Vedolizumab Therapy in Anti-TNF Refractory IBD Patients. </span><span class="cited-content_cbyCitation_journal-name">Crohn's & Colitis 360</span><span> <strong>2020,</strong> <em>2 </em>
                                    (2)
                                     <a href="https://doi.org/10.1093/crocol/otaa037" title="DOI URL">https://doi.org/10.1093/crocol/otaa037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/crocol/otaa037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fcrocol%2Fotaa037%26sid%3Dliteratum%253Aachs%26jtitle%3DCrohn%2527s%2520%2526%2520Colitis%2520360%26atitle%3DFunctional%252BMolecular%252BNetwork%252BAnalysis%252BEnables%252BPrediction%252Bof%252BResponse%252Bto%252BVedolizumab%252BTherapy%252Bin%252BAnti-TNF%252BRefractory%252BIBD%252BPatients%26aulast%3DBreidert%26aufirst%3DMatthias%26date%3D2020%26date%3D2020%26volume%3D2%26issue%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jitao David  Zhang</span>, <span class="hlFld-ContribAuthor ">Lisa  Sach-Peltason</span>, <span class="hlFld-ContribAuthor ">Christian  Kramer</span>, <span class="hlFld-ContribAuthor ">Ken  Wang</span>, <span class="hlFld-ContribAuthor ">Martin  Ebeling</span>. </span><span class="cited-content_cbyCitation_article-title">Multiscale modelling of drug mechanism and safety. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (3)
                                     , 519-534. <a href="https://doi.org/10.1016/j.drudis.2019.12.009" title="DOI URL">https://doi.org/10.1016/j.drudis.2019.12.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2019.12.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2019.12.009%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DMultiscale%252Bmodelling%252Bof%252Bdrug%252Bmechanism%252Band%252Bsafety%26aulast%3DZhang%26aufirst%3DJitao%2BDavid%26date%3D2020%26volume%3D25%26issue%3D3%26spage%3D519%26epage%3D534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José E. S.  Nunes</span>, <span class="hlFld-ContribAuthor ">Mario A.  Duque</span>, <span class="hlFld-ContribAuthor ">Talita F.  de Freitas</span>, <span class="hlFld-ContribAuthor ">Luiza  Galina</span>, <span class="hlFld-ContribAuthor ">Luis F. S. M.  Timmers</span>, <span class="hlFld-ContribAuthor ">Cristiano V.  Bizarro</span>, <span class="hlFld-ContribAuthor ">Pablo  Machado</span>, <span class="hlFld-ContribAuthor ">Luiz A.  Basso</span>, <span class="hlFld-ContribAuthor ">Rodrigo G.  Ducati</span>. </span><span class="cited-content_cbyCitation_article-title">Mycobacterium tuberculosis Shikimate Pathway Enzymes as Targets for the Rational Design of Anti-Tuberculosis Drugs. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (6)
                                     , 1259. <a href="https://doi.org/10.3390/molecules25061259" title="DOI URL">https://doi.org/10.3390/molecules25061259</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25061259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25061259%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMycobacterium%252Btuberculosis%252BShikimate%252BPathway%252BEnzymes%252Bas%252BTargets%252Bfor%252Bthe%252BRational%252BDesign%252Bof%252BAnti-Tuberculosis%252BDrugs%26aulast%3DNunes%26aufirst%3DJos%25C3%25A9%2BE.%2BS.%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D6%26spage%3D1259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabel V. L.  Wilkinson</span>, <span class="hlFld-ContribAuthor ">Kelly J.  Perkins</span>, <span class="hlFld-ContribAuthor ">Hannah  Dugdale</span>, <span class="hlFld-ContribAuthor ">Lee  Moir</span>, <span class="hlFld-ContribAuthor ">Aini  Vuorinen</span>, <span class="hlFld-ContribAuthor ">Maria  Chatzopoulou</span>, <span class="hlFld-ContribAuthor ">Sarah E.  Squire</span>, <span class="hlFld-ContribAuthor ">Sebastian  Monecke</span>, <span class="hlFld-ContribAuthor ">Alexander  Lomow</span>, <span class="hlFld-ContribAuthor ">Marcus  Geese</span>, <span class="hlFld-ContribAuthor ">Philip D.  Charles</span>, <span class="hlFld-ContribAuthor ">Peter  Burch</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Tinsley</span>, <span class="hlFld-ContribAuthor ">Graham M.  Wynne</span>, <span class="hlFld-ContribAuthor ">Stephen G.  Davies</span>, <span class="hlFld-ContribAuthor ">Francis X.  Wilson</span>, <span class="hlFld-ContribAuthor ">Fraydoon  Rastinejad</span>, <span class="hlFld-ContribAuthor ">Shabaz  Mohammed</span>, <span class="hlFld-ContribAuthor ">Kay E.  Davies</span>, <span class="hlFld-ContribAuthor ">Angela J.  Russell</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Proteomics and Phenotypic Profiling Identifies the Aryl Hydrocarbon Receptor as a Molecular Target of the Utrophin Modulator Ezutromid. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2020,</strong> <em>132 </em>
                                    (6)
                                     , 2441-2449. <a href="https://doi.org/10.1002/ange.201912392" title="DOI URL">https://doi.org/10.1002/ange.201912392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201912392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201912392%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DChemical%252BProteomics%252Band%252BPhenotypic%252BProfiling%252BIdentifies%252Bthe%252BAryl%252BHydrocarbon%252BReceptor%252Bas%252Ba%252BMolecular%252BTarget%252Bof%252Bthe%252BUtrophin%252BModulator%252BEzutromid%26aulast%3DWilkinson%26aufirst%3DIsabel%2BV.%2BL.%26date%3D2020%26date%3D2020%26volume%3D132%26issue%3D6%26spage%3D2441%26epage%3D2449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabel V. L.  Wilkinson</span>, <span class="hlFld-ContribAuthor ">Kelly J.  Perkins</span>, <span class="hlFld-ContribAuthor ">Hannah  Dugdale</span>, <span class="hlFld-ContribAuthor ">Lee  Moir</span>, <span class="hlFld-ContribAuthor ">Aini  Vuorinen</span>, <span class="hlFld-ContribAuthor ">Maria  Chatzopoulou</span>, <span class="hlFld-ContribAuthor ">Sarah E.  Squire</span>, <span class="hlFld-ContribAuthor ">Sebastian  Monecke</span>, <span class="hlFld-ContribAuthor ">Alexander  Lomow</span>, <span class="hlFld-ContribAuthor ">Marcus  Geese</span>, <span class="hlFld-ContribAuthor ">Philip D.  Charles</span>, <span class="hlFld-ContribAuthor ">Peter  Burch</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Tinsley</span>, <span class="hlFld-ContribAuthor ">Graham M.  Wynne</span>, <span class="hlFld-ContribAuthor ">Stephen G.  Davies</span>, <span class="hlFld-ContribAuthor ">Francis X.  Wilson</span>, <span class="hlFld-ContribAuthor ">Fraydoon  Rastinejad</span>, <span class="hlFld-ContribAuthor ">Shabaz  Mohammed</span>, <span class="hlFld-ContribAuthor ">Kay E.  Davies</span>, <span class="hlFld-ContribAuthor ">Angela J.  Russell</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Proteomics and Phenotypic Profiling Identifies the Aryl Hydrocarbon Receptor as a Molecular Target of the Utrophin Modulator Ezutromid. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2020,</strong> <em>59 </em>
                                    (6)
                                     , 2420-2428. <a href="https://doi.org/10.1002/anie.201912392" title="DOI URL">https://doi.org/10.1002/anie.201912392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201912392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201912392%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DChemical%252BProteomics%252Band%252BPhenotypic%252BProfiling%252BIdentifies%252Bthe%252BAryl%252BHydrocarbon%252BReceptor%252Bas%252Ba%252BMolecular%252BTarget%252Bof%252Bthe%252BUtrophin%252BModulator%252BEzutromid%26aulast%3DWilkinson%26aufirst%3DIsabel%2BV.%2BL.%26date%3D2020%26volume%3D59%26issue%3D6%26spage%3D2420%26epage%3D2428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adrien  Herledan</span>, <span class="hlFld-ContribAuthor ">Marine  Andres</span>, <span class="hlFld-ContribAuthor ">Aurore  Lejeune-Dodge</span>, <span class="hlFld-ContribAuthor ">Florence  Leroux</span>, <span class="hlFld-ContribAuthor ">Alexandre  Biela</span>, <span class="hlFld-ContribAuthor ">Catherine  Piveteau</span>, <span class="hlFld-ContribAuthor ">Sandrine  Warenghem</span>, <span class="hlFld-ContribAuthor ">Cyril  Couturier</span>, <span class="hlFld-ContribAuthor ">Benoit  Deprez</span>, <span class="hlFld-ContribAuthor ">Rebecca  Deprez-Poulain</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Target Engagement Using Coupled Cellular Thermal Shift Assay—Acoustic Reverse-Phase Protein Array. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2020,</strong> <em>25 </em>
                                    (2)
                                     , 207-214. <a href="https://doi.org/10.1177/2472555219897256" title="DOI URL">https://doi.org/10.1177/2472555219897256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555219897256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555219897256%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DDrug%252BTarget%252BEngagement%252BUsing%252BCoupled%252BCellular%252BThermal%252BShift%252BAssay%2525E2%252580%252594Acoustic%252BReverse-Phase%252BProtein%252BArray%26aulast%3DHerledan%26aufirst%3DAdrien%26date%3D2020%26date%3D2019%26volume%3D25%26issue%3D2%26spage%3D207%26epage%3D214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jim  Freeth</span>, <span class="hlFld-ContribAuthor ">Jo  Soden</span>. </span><span class="cited-content_cbyCitation_article-title">New Advances in Cell Microarray Technology to Expand Applications in Target Deconvolution and Off-Target Screening. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2020,</strong> <em>25 </em>
                                    (2)
                                     , 223-230. <a href="https://doi.org/10.1177/2472555219897567" title="DOI URL">https://doi.org/10.1177/2472555219897567</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555219897567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555219897567%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DNew%252BAdvances%252Bin%252BCell%252BMicroarray%252BTechnology%252Bto%252BExpand%252BApplications%252Bin%252BTarget%252BDeconvolution%252Band%252BOff-Target%252BScreening%26aulast%3DFreeth%26aufirst%3DJim%26date%3D2020%26date%3D2019%26volume%3D25%26issue%3D2%26spage%3D223%26epage%3D230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David C.  Swinney</span>, <span class="hlFld-ContribAuthor ">Jonathan A.  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in phenotypic drug discovery. </span><span class="cited-content_cbyCitation_journal-name">F1000Research</span><span> <strong>2020,</strong> <em>9 </em>, 944. <a href="https://doi.org/10.12688/f1000research.25813.1" title="DOI URL">https://doi.org/10.12688/f1000research.25813.1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.12688/f1000research.25813.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.12688%2Ff1000research.25813.1%26sid%3Dliteratum%253Aachs%26jtitle%3DF1000Research%26atitle%3DRecent%252Badvances%252Bin%252Bphenotypic%252Bdrug%252Bdiscovery%26aulast%3DSwinney%26aufirst%3DDavid%2BC.%26date%3D2020%26date%3D2020%26volume%3D9%26spage%3D944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesc  Sabanés Zariquiey</span>, <span class="hlFld-ContribAuthor ">João V.  de Souza</span>, <span class="hlFld-ContribAuthor ">Agnieszka K.  Bronowska</span>. </span><span class="cited-content_cbyCitation_article-title">Cosolvent Analysis Toolkit (CAT): a robust hotspot identification platform for cosolvent simulations of proteins to expand the druggable proteome. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2019,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-019-55394-2" title="DOI URL">https://doi.org/10.1038/s41598-019-55394-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-019-55394-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-019-55394-2%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DCosolvent%252BAnalysis%252BToolkit%252B%252528CAT%252529%25253A%252Ba%252Brobust%252Bhotspot%252Bidentification%252Bplatform%252Bfor%252Bcosolvent%252Bsimulations%252Bof%252Bproteins%252Bto%252Bexpand%252Bthe%252Bdruggable%252Bproteome%26aulast%3DSaban%25C3%25A9s%2BZariquiey%26aufirst%3DFrancesc%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dorothee  Childs</span>, <span class="hlFld-ContribAuthor ">Karsten  Bach</span>, <span class="hlFld-ContribAuthor ">Holger  Franken</span>, <span class="hlFld-ContribAuthor ">Simon  Anders</span>, <span class="hlFld-ContribAuthor ">Nils  Kurzawa</span>, <span class="hlFld-ContribAuthor ">Marcus  Bantscheff</span>, <span class="hlFld-ContribAuthor ">Mikhail M.  Savitski</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Huber</span>. </span><span class="cited-content_cbyCitation_article-title">Nonparametric Analysis of Thermal Proteome Profiles Reveals Novel Drug-binding Proteins*. </span><span class="cited-content_cbyCitation_journal-name">Molecular & Cellular Proteomics</span><span> <strong>2019,</strong> <em>18 </em>
                                    (12)
                                     , 2506-2515. <a href="https://doi.org/10.1074/mcp.TIR119.001481" title="DOI URL">https://doi.org/10.1074/mcp.TIR119.001481</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/mcp.TIR119.001481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fmcp.TIR119.001481%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520%2526%2520Cellular%2520Proteomics%26atitle%3DNonparametric%252BAnalysis%252Bof%252BThermal%252BProteome%252BProfiles%252BReveals%252BNovel%252BDrug-binding%252BProteins%252A%26aulast%3DChilds%26aufirst%3DDorothee%26date%3D2019%26volume%3D18%26issue%3D12%26spage%3D2506%26epage%3D2515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Florence  Leroux</span>, <span class="hlFld-ContribAuthor ">Damien  Bosc</span>, <span class="hlFld-ContribAuthor ">Terence  Beghyn</span>, <span class="hlFld-ContribAuthor ">Paul  Hermant</span>, <span class="hlFld-ContribAuthor ">Sandrine  Warenghem</span>, <span class="hlFld-ContribAuthor ">Valérie  Landry</span>, <span class="hlFld-ContribAuthor ">Virginie  Pottiez</span>, <span class="hlFld-ContribAuthor ">Valentin  Guillaume</span>, <span class="hlFld-ContribAuthor ">Julie  Charton</span>, <span class="hlFld-ContribAuthor ">Adrien  Herledan</span>, <span class="hlFld-ContribAuthor ">Sarah  Urata</span>, <span class="hlFld-ContribAuthor ">Wenguang  Liang</span>, <span class="hlFld-ContribAuthor ">Li  Sheng</span>, <span class="hlFld-ContribAuthor ">Wei-Jen  Tang</span>, <span class="hlFld-ContribAuthor ">Benoit  Deprez</span>, <span class="hlFld-ContribAuthor ">Rebecca  Deprez-Poulain</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of ebselen as a potent inhibitor of insulin degrading enzyme by a drug repurposing screening. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>179 </em>, 557-566. <a href="https://doi.org/10.1016/j.ejmech.2019.06.057" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.06.057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.06.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.06.057%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bebselen%252Bas%252Ba%252Bpotent%252Binhibitor%252Bof%252Binsulin%252Bdegrading%252Benzyme%252Bby%252Ba%252Bdrug%252Brepurposing%252Bscreening%26aulast%3DLeroux%26aufirst%3DFlorence%26date%3D2019%26volume%3D179%26spage%3D557%26epage%3D566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tim  Förster</span>, <span class="hlFld-ContribAuthor ">Erchang  Shang</span>, <span class="hlFld-ContribAuthor ">Kenshiro  Shimizu</span>, <span class="hlFld-ContribAuthor ">Emiko  Sanada</span>, <span class="hlFld-ContribAuthor ">Beate  Schölermann</span>, <span class="hlFld-ContribAuthor ">Mylene  Huebecker</span>, <span class="hlFld-ContribAuthor ">Gernot  Hahne</span>, <span class="hlFld-ContribAuthor ">Maria Pascual  López-Alberca</span>, <span class="hlFld-ContribAuthor ">Petra  Janning</span>, <span class="hlFld-ContribAuthor ">Nobumoto  Watanabe</span>, <span class="hlFld-ContribAuthor ">Sonja  Sievers</span>, <span class="hlFld-ContribAuthor ">Fabrizio  Giordanetto</span>, <span class="hlFld-ContribAuthor ">Takeshi  Shimizu</span>, <span class="hlFld-ContribAuthor ">Slava  Ziegler</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Osada</span>, <span class="hlFld-ContribAuthor ">Herbert  Waldmann</span>. </span><span class="cited-content_cbyCitation_article-title">2-Sulfonylpyrimidines Target the Kinesin HSET via Cysteine Alkylation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>2019 </em>
                                    (31-32)
                                     , 5486-5496. <a href="https://doi.org/10.1002/ejoc.201900586" title="DOI URL">https://doi.org/10.1002/ejoc.201900586</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201900586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201900586%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3D2-Sulfonylpyrimidines%252BTarget%252Bthe%252BKinesin%252BHSET%252Bvia%252BCysteine%252BAlkylation%26aulast%3DF%25C3%25B6rster%26aufirst%3DTim%26date%3D2019%26date%3D2019%26volume%3D2019%26issue%3D31-32%26spage%3D5486%26epage%3D5496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephanie S.  Kim</span>, <span class="hlFld-ContribAuthor ">Melanie L.  Aprahamian</span>, <span class="hlFld-ContribAuthor ">Steffen  Lindert</span>. </span><span class="cited-content_cbyCitation_article-title">Improving inverse docking target identification with
              Z
              ‐score selection. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2019,</strong> <em>93 </em>
                                    (6)
                                     , 1105-1116. <a href="https://doi.org/10.1111/cbdd.13453" title="DOI URL">https://doi.org/10.1111/cbdd.13453</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13453%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DImproving%252Binverse%252Bdocking%252Btarget%252Bidentification%252Bwith%252BZ%252B%2525E2%252580%252590score%252Bselection%26aulast%3DKim%26aufirst%3DStephanie%2BS.%26date%3D2019%26date%3D2019%26volume%3D93%26issue%3D6%26spage%3D1105%26epage%3D1116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Process roadmap for elucidation of small-molecule (SM) mechanism of action (MOA) and target identification.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Insufficiency of profiling panels to recapitulate the diversity of targets from a biological source. A well-designed phenotypic assay may identify small molecules that interact productively with any of a vast array of potential targets. Discrete profiling panels can only interrogate a small subset of the potential targets.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effects of structure on microtubule polymerization. (A) Effects of 204 compounds derived from a growth inhibition phenotypic screen on microtubule polymerization in vitro. Structural classes are denoted by color and linked together by lines. The shaded area denotes changes in microtubule assembly that are relatively weak, whereas compounds falling outside this range have a strong effect. The change in growth phase rate was calculated by taking the slope of the linear regression of the growth phase (5–15 min) and comparing it to the slope of the control. Data for the compounds reported in (B) and (C) are highlighted in red. (B) Structures of compound <b>1</b>, compound <b>2</b>, colchicine, and taxol. (C) Two representative compounds from the phenotypic screen were profiled for their ability to modulate microtubule assembly in vitro. At 10 μM, compound <b>1</b> (top left) reduced microtubule growth in a manner consistent with a known destabilizer, colchicine (bottom left). Conversely, a closely related analogue, Compound <b>2</b> (top right), may have had a growth enhancing affect similar to a known stabilizer, taxol (bottom right). DMSO (blue), 10 μM Compound (red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Dose–response L1000 (DR-L1000) to address SM cellular potency differences. (A) Strategy for data collection and integration. (B) CellTiter-Glo (CTG) ATP/viability assessment indicating compounds <b>1</b> and <b>2</b> exhibit significantly different potencies when profiled in cell line A but are equipotent in cell line B. (C) Heat-map for a gene set that correlates with cell line potencies. Cell lines 1 and 2 were treated with increasing doses (5 nM to 10 μM) of each compound. Red and blue represent increases and decreases in GEx, respectively. The extent of regulation (fold-change) varies directly with the color intensity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Reporter assay EC<sub>50</sub> values (left) and corresponding DR-L1000 plots for a representative gene (right). (B) (Top) gene set in common with two distinct MOAs of pathway inhibition; (bottom) gene sets distinct for two different MOAs. (C) Dose response for multiple genes originating from an individual SM treatment</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) BioMAP profiling platform. (B) Process for early triage of large sets of compounds derived from TDD or PDD campaigns.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Schematic that outlines pooled CRISPR screening paired with positive selection. A library of diverse sgRNAs is used to transform a clonal cell line into a pool of single gene targeted disruptions. Positive selection enriches mutant cells that acquired a novel trait distinct from the unmodified parental line and directly related to that particular selective challenge. Sequencing identifies targeting constructs as a surrogate of mutant genes in the enriched cell population.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. General schematic describing linked compounds for target ID/validation. (A) Linked versions of active compound via multiple vectors in order to produce phenotypically active intermediates. Site of linking determined by combination of compound–target structural information, SAR, and chemical tractability. (B) A few representative chemistries for introducing linked functionality. (C) Conversion of linked intermediates into target ID/validation probes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Schematic overview of activity based protein profiling (ABPP). Cells or tissue are dosed either with vehicle (DMSO) or compound at desired doses. Following abstraction of the target milieu, ABPP probes are applied to covalently bind to open active sites. The samples are then visualized by a reporter.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Bacteriophage display for small-molecule target ID (T7 bacteriophage are adapted from SIB (Swiss Institute of Bioinformatics), an open access resource<a onclick="showRef(event, 'ref171'); return false;" href="javascript:void(0);" class="ref ref171">(171)</a>). (A) Heterologous expression library creation process. (B) After cloning the cDNA library into T7 bacteriophage organisms, phages are applied to wells containing the immobilized small-molecule probe. After washing, phage particles most strongly associated with the compound are recovered, amplified, and the process repeated until no phage are shed during the binding and washing steps. (C) As an internal example to highlight the utility of the approach, a control biotinylated linker and probe based on JQ1<a onclick="showRef(event, 'ref172'); return false;" href="javascript:void(0);" class="ref ref172">(172)</a> were synthesized and used for preclearing and capture steps over six rounds of selection. (D) Bioanalysis image showing enrichment of specific DNA molecular weight bands from round 0 to round 6 after parent JQ1 compound elution each round. Fold enrichment refers to next generation DNA sequence analysis for the BRD2 gene. From 7000 significantly expressed sequences, BRD2 represented the largest increase from round 0 to round 6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Schematic overview of the glycan-mediated ligand-controlled click chemistry (“GLiCo-Click”) strategy. After the cells have been metabolically labeled with azidosugars, the chemical probe binds to its cell surface receptor and brings a bioorthogonal alkyne into close proximity to the receptor glycans, and this triggers the formation of a covalent cross-link between the alkyne and the azidosugar, thereby irreversibly equipping the target receptor with biotin. Biotinylated proteins can be analyzed by microscopy to track endocytosis or by chemoproteomics after cell lysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. EGFR was able to be enriched and observed by ligand receptor capture using a low receptor number and weak antibody. (A) Annotated structure of the TriCEPS ligand. (B) Ligand receptor capture and mass spectrometric detection from NCI-H358 cells, which contain a low 2000 receptor copy number per cell.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. General processes involved in ligand directed chemistries. (A) The molecular probe is recruited to the receptor by virtue of the ligand. Upon binding, the reactive group covalently binds to the protein in a proximity driven manner, ejecting the ligand in the process. (B) The molecular probe is again recruited to the receptor by its ligand, however, the reactive group forms a covalent bond, ejecting its own leaving group, while the ligand and pull down handle remain affixed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Antibodies with varying affinity to EGFR are all accurately detected and identified by using cell surface microarray technology. FCGR2B, low affinity immunoglobulin gamma Fc region receptor II-b, used as positive control for detection antibody. ZsGreen1, antibody for detection of GFP expression pattern.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Discovery of BET family of proteins as targets for an oncology program (A) Prioritization of CMPD-604 from a phenotypic screen (B) Profiling of CMPD-604 against select profiling panels and tumor cell lines (C) Probe synthesis, AC-MS target hypotheses, localization of binding site, and target engagement. (D) Translation of screening PCA into Alpha-LISA assay, confirmation of MOA, and in vivo detection of PD marker</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/medium/jm-2017-01921m_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Discovery of MGST1 as an unprecedented oncology target. CGS-0059 was identified from a CCL profiling campaign, initially based on a growth inhibition–GEx correlation. The SM was further prioritized based on a perceived unique MOA and subjected to target identification using generation of resistant clones and direct target enrichment. Engagement of CGS-0059 with MGST1 was supported by CETSA experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.7b01921/20181004/images/large/jm-2017-01921m_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01921&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i53">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26780" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26780" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 172 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedrani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span> <span> </span><span class="NLM_article-title">The discovery of first-in-class drugs: origins and evolution</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1038/nrd4336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd4336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25033734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=577-587&author=J.+Ederauthor=R.+Sedraniauthor=C.+Wiesmann&title=The+discovery+of+first-in-class+drugs%3A+origins+and+evolution&doi=10.1038%2Fnrd4336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of first-in-class drugs: origins and evolution</span></div><div class="casAuthors">Eder, Jorg; Sedrani, Richard; Wiesmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">577-587</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Anal. of the origins of new drugs approved by the US Food and Drug Administration (FDA) from 1999 to 2008 suggested that phenotypic screening strategies had been more productive than target-based approaches in the discovery of first-in-class small-mol. drugs.  However, given the relatively recent introduction of target-based approaches in the context of the long time frames of drug development, their full impact might not yet have become apparent.  Here, we present an anal. of the origins of all 113 first-in-class drugs approved by the FDA from 1999 to 2013, which shows that the majority (78) were discovered through target-based approaches (45 small-mol. drugs and 33 biologics).  In addn., of 33 drugs identified in the absence of a target hypothesis, 25 were found through a chemocentric approach in which compds. with known pharmacol. served as the starting point, with only eight coming from what we define here as phenotypic screening: testing a large no. of compds. in a target-agnostic assay that monitors phenotypic changes.  We also discuss the implications for drug discovery strategies, including viewing phenotypic screening as a novel discipline rather than as a neoclassical approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqthEelveRWN7Vg90H21EOLACvtfcHk0lhiMd4Eh6Zsjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I&md5=b2061d96a71893a329b5b654acec5420</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd4336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4336%26sid%3Dliteratum%253Aachs%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DWiesmann%26aufirst%3DC.%26atitle%3DThe%2520discovery%2520of%2520first-in-class%2520drugs%253A%2520origins%2520and%2520evolution%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D577%26epage%3D587%26doi%3D10.1038%2Fnrd4336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span> <span> </span><span class="NLM_article-title">Phenotypic screening in cancer drug discovery - past, present and future</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1038/nrd4366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd4366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25033736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=588-602&author=J.+G.+Moffatauthor=J.+Rudolphauthor=D.+Bailey&title=Phenotypic+screening+in+cancer+drug+discovery+-+past%2C+present+and+future&doi=10.1038%2Fnrd4366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening in cancer drug discovery - past, present and future</span></div><div class="casAuthors">Moffat, John G.; Rudolph, Joachim; Bailey, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">588-602</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in the use of phenotypic screens in drug discovery as an alternative to target-focused approaches.  Given that oncol. is currently the most active therapeutic area, and also one in which target-focused approaches have been particularly prominent in the past two decades, we investigated the contribution of phenotypic assays to oncol. drug discovery by analyzing the origins of all new small-mol. cancer drugs approved by the US Food and Drug Administration (FDA) over the past 15 years and those currently in clin. development.  Although the majority of these drugs originated from target-based discovery, we identified a significant no. whose discovery depended on phenotypic screening approaches.  We postulate that the contribution of phenotypic screening to cancer drug discovery has been hampered by a reliance on 'classical' nonspecific drug effects such as cytotoxicity and mitotic arrest, exacerbated by a paucity of mechanistically defined cellular models for therapeutically translatable cancer phenotypes.  However, tech. and biol. advances that enable such mechanistically informed phenotypic models have the potential to empower phenotypic drug discovery in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTWtoTIgWqvbVg90H21EOLACvtfcHk0lhiMd4Eh6Zsjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L&md5=0e8efed90e5bc4b47f959b83a6ab5b96</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd4366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4366%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DD.%26atitle%3DPhenotypic%2520screening%2520in%2520cancer%2520drug%2520discovery%2520-%2520past%252C%2520present%2520and%2520future%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D588%26epage%3D602%26doi%3D10.1038%2Fnrd4366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunotto, M.</span></span> <span> </span><span class="NLM_article-title">Opportunities and challenges in phenotypic drug discovery: an industry perspective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd.2017.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28685762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=531-543&author=J.+G.+Moffatauthor=F.+Vincentauthor=J.+A.+Leeauthor=J.+Ederauthor=M.+Prunotto&title=Opportunities+and+challenges+in+phenotypic+drug+discovery%3A+an+industry+perspective&doi=10.1038%2Fnrd.2017.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities and challenges in phenotypic drug discovery: an industry perspective</span></div><div class="casAuthors">Moffat, John G.; Vincent, Fabien; Lee, Jonathan A.; Eder, Jorg; Prunotto, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">531-543</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Phenotypic drug discovery (PDD) approaches do not rely on knowledge of the identity of a specific drug target or a hypothesis about its role in disease, in contrast to the target-based strategies that have been widely used in the pharmaceutical industry in the past three decades.  However, in recent years, there has been a resurgence in interest in PDD approaches based on their potential to address the incompletely understood complexity of diseases and their promise of delivering first-in-class drugs, as well as major advances in the tools for cell-based phenotypic screening.  Nevertheless, PDD approaches also have considerable challenges, such as hit validation and target deconvolution.  This article focuses on the lessons learned by researchers engaged in PDD in the pharmaceutical industry and considers the impact of 'omics' knowledge in defining a cellular disease phenotype in the era of precision medicine, introducing the concept of a chain of translatability.  We particularly aim to identify features and areas in which PDD can best deliver value to drug discovery portfolios and can contribute to the identification and the development of novel medicines, and to illustrate the challenges and uncertainties that are assocd. with PDD in order to help set realistic expectations with regard to its benefits and costs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrD5TsU8_kLVg90H21EOLACvtfcHk0lhiMd4Eh6Zsjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ&md5=d83196715c879a81d9eabcf544e303db</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.111%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DJ.%2BA.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DPrunotto%26aufirst%3DM.%26atitle%3DOpportunities%2520and%2520challenges%2520in%2520phenotypic%2520drug%2520discovery%253A%2520an%2520industry%2520perspective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D531%26epage%3D543%26doi%3D10.1038%2Fnrd.2017.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, J.</span></span> <span> </span><span class="NLM_article-title">How were new medicines discovered?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1038/nrd3480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd3480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21701501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=507-519&author=D.+C.+Swinneyauthor=J.+Anthony&title=How+were+new+medicines+discovered%3F&doi=10.1038%2Fnrd3480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">How were new medicines discovered?</span></div><div class="casAuthors">Swinney, David C.; Anthony, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">507-519</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Preclin. strategies that are used to identify potential drug candidates include target-based screening, phenotypic screening, modification of natural substances and biol.-based approaches.  To investigate whether some strategies have been more successful than others in the discovery of new drugs, we analyzed the discovery strategies and the mol. mechanism of action (MMOA) for new mol. entities and new biologics that were approved by the US Food and Drug Administration between 1999 and 2008.  Out of the 259 agents that were approved, 75 were first-in-class drugs with new MMOAs, and out of these, 50 (67%) were small mols. and 25 (33%) were biologics.  The results also show that the contribution of phenotypic screening to the discovery of first-in-class small-mol. drugs exceeded that of target-based approaches - with 28 and 17 of these drugs coming from the two approaches, resp. - in an era in which the major focus was on target-based approaches.  We postulate that a target-centric approach for first-in-class drugs, without consideration of an optimal MMOA, may contribute to the current high attrition rates and low productivity in pharmaceutical research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE8Ah1iNz76bVg90H21EOLACvtfcHk0ljAP4K4ITJYEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D&md5=7503620f8fbadb0a829f6ea2029936a2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd3480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3480%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26aulast%3DAnthony%26aufirst%3DJ.%26atitle%3DHow%2520were%2520new%2520medicines%2520discovered%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D507%26epage%3D519%26doi%3D10.1038%2Fnrd3480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pregel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitching, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nocka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyonnas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, M. C.</span></span> <span> </span><span class="NLM_article-title">Developing predictive assays: the phenotypic screening ″rule of 3″</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">293ps15</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aab1201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscitranslmed.aab1201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26109101" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=293ps15&author=F.+Vincentauthor=P.+Loriaauthor=M.+Pregelauthor=R.+Stantonauthor=L.+Kitchingauthor=K.+Nockaauthor=R.+Doyonnasauthor=C.+Steppanauthor=A.+Gilbertauthor=T.+Schroeterauthor=M.+C.+Peakman&title=Developing+predictive+assays%3A+the+phenotypic+screening+%E2%80%B3rule+of+3%E2%80%B3&doi=10.1126%2Fscitranslmed.aab1201"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aab1201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aab1201%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLoria%26aufirst%3DP.%26aulast%3DPregel%26aufirst%3DM.%26aulast%3DStanton%26aufirst%3DR.%26aulast%3DKitching%26aufirst%3DL.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DDoyonnas%26aufirst%3DR.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DGilbert%26aufirst%3DA.%26aulast%3DSchroeter%26aufirst%3DT.%26aulast%3DPeakman%26aufirst%3DM.%2BC.%26atitle%3DDeveloping%2520predictive%2520assays%253A%2520the%2520phenotypic%2520screening%2520%25E2%2580%25B3rule%2520of%25203%25E2%2580%25B3%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2015%26volume%3D7%26spage%3D293ps15%26doi%3D10.1126%2Fscitranslmed.aab1201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rees, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seashore-Ludlow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheah, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javaid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coletti, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodycombe, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soule, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hon, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefeld, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dancik, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Correlating chemical sensitivity and basal gene expression reveals mechanism of action</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1986</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.1986" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26656090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=109-116&author=M.+G.+Reesauthor=B.+Seashore-Ludlowauthor=J.+H.+Cheahauthor=D.+J.+Adamsauthor=E.+V.+Priceauthor=S.+Gillauthor=S.+Javaidauthor=M.+E.+Colettiauthor=V.+L.+Jonesauthor=N.+E.+Bodycombeauthor=C.+K.+Souleauthor=B.+Alexanderauthor=A.+Liauthor=P.+Montgomeryauthor=J.+D.+Kotzauthor=C.+S.+Honauthor=B.+Munozauthor=T.+Liefeldauthor=V.+Dancikauthor=D.+A.+Haberauthor=C.+B.+Clishauthor=J.+A.+Bittkerauthor=M.+Palmerauthor=B.+K.+Wagnerauthor=P.+A.+Clemonsauthor=A.+F.+Shamjiauthor=S.+L.+Schreiber&title=Correlating+chemical+sensitivity+and+basal+gene+expression+reveals+mechanism+of+action&doi=10.1038%2Fnchembio.1986"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Correlating chemical sensitivity and basal gene expression reveals mechanism of action</span></div><div class="casAuthors">Rees, Matthew G.; Seashore-Ludlow, Brinton; Cheah, Jaime H.; Adams, Drew J.; Price, Edmund V.; Gill, Shubhroz; Javaid, Sarah; Coletti, Matthew E.; Jones, Victor L.; Bodycombe, Nicole E.; Soule, Christian K.; Alexander, Benjamin; Li, Ava; Montgomery, Philip; Kotz, Joanne D.; Hon, C. Suk-Yee; Munoz, Benito; Liefeld, Ted; Dancik, Vlado; Haber, Daniel A.; Clish, Clary B.; Bittker, Joshua A.; Palmer, Michelle; Wagner, Bridget K.; Clemons, Paul A.; Shamji, Alykhan F.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">109-116</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Changes in cellular gene expression in response to small-mol. or genetic perturbations have yielded signatures that can connect unknown mechanisms of action (MoA) to ones previously established.  We hypothesized that differential basal gene expression could be correlated with patterns of small-mol. sensitivity across many cell lines to illuminate the actions of compds. whose MoA are unknown.  To test this idea, we correlated the sensitivity patterns of 481 compds. with ∼19,000 basal transcript levels across 823 different human cancer cell lines and identified selective outlier transcripts.  This process yielded many novel mechanistic insights, including the identification of activation mechanisms, cellular transporters and direct protein targets.  We found that ML239, originally identified in a phenotypic screen for selective cytotoxicity in breast cancer stem-like cells, most likely acts through activation of fatty acid desaturase 2 (FADS2).  These data and anal. tools are available to the research community through the Cancer Therapeutics Response Portal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTUYwiQeWDrVg90H21EOLACvtfcHk0ljAP4K4ITJYEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtb3K&md5=8e4b5c55bd8486b0b06f917ac937df7f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1986&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1986%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DM.%2BG.%26aulast%3DSeashore-Ludlow%26aufirst%3DB.%26aulast%3DCheah%26aufirst%3DJ.%2BH.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DPrice%26aufirst%3DE.%2BV.%26aulast%3DGill%26aufirst%3DS.%26aulast%3DJavaid%26aufirst%3DS.%26aulast%3DColetti%26aufirst%3DM.%2BE.%26aulast%3DJones%26aufirst%3DV.%2BL.%26aulast%3DBodycombe%26aufirst%3DN.%2BE.%26aulast%3DSoule%26aufirst%3DC.%2BK.%26aulast%3DAlexander%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DMontgomery%26aufirst%3DP.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DHon%26aufirst%3DC.%2BS.%26aulast%3DMunoz%26aufirst%3DB.%26aulast%3DLiefeld%26aufirst%3DT.%26aulast%3DDancik%26aufirst%3DV.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DCorrelating%2520chemical%2520sensitivity%2520and%2520basal%2520gene%2520expression%2520reveals%2520mechanism%2520of%2520action%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D109%26epage%3D116%26doi%3D10.1038%2Fnchembio.1986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swalley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoosear, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLellan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuierer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaustein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berenshteyn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacoste, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span> <span> </span><span class="NLM_article-title">SMN2 splice modulators enhance U1-pre-mRNA association and rescue SMA mice</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.1837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26030728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=511-517&author=J.+Palacinoauthor=S.+E.+Swalleyauthor=C.+Songauthor=A.+K.+Cheungauthor=L.+Shuauthor=X.+Zhangauthor=M.+Van+Hoosearauthor=Y.+Shinauthor=D.+N.+Chinauthor=C.+G.+Kellerauthor=M.+Beibelauthor=N.+A.+Renaudauthor=T.+M.+Smithauthor=M.+Salciusauthor=X.+Shiauthor=M.+Hildauthor=R.+Servaisauthor=M.+Jainauthor=L.+Dengauthor=C.+Bullockauthor=M.+McLellanauthor=S.+Schuiererauthor=L.+Murphyauthor=M.+J.+Blommersauthor=C.+Blausteinauthor=F.+Berenshteynauthor=A.+Lacosteauthor=J.+R.+Thomasauthor=G.+Romaauthor=G.+A.+Michaudauthor=B.+S.+Tsengauthor=J.+A.+Porterauthor=V.+E.+Myerauthor=J.+A.+Tallaricoauthor=L.+G.+Hamannauthor=D.+Curtisauthor=M.+C.+Fishmanauthor=W.+F.+Dietrichauthor=N.+A.+Dalesauthor=R.+Sivasankaran&title=SMN2+splice+modulators+enhance+U1-pre-mRNA+association+and+rescue+SMA+mice&doi=10.1038%2Fnchembio.1837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splice modulators enhance U1-pre-mRNA association and rescue SMA mice</span></div><div class="casAuthors">Palacino, James; Swalley, Susanne E.; Song, Cheng; Cheung, Atwood K.; Shu, Lei; Zhang, Xiaolu; Van Hoosear, Mailin; Shin, Youngah; Chin, Donovan N.; Keller, Caroline Gubser; Beibel, Martin; Renaud, Nicole A.; Smith, Thomas M.; Salcius, Michael; Shi, Xiaoying; Hild, Marc; Servais, Rebecca; Jain, Monish; Deng, Lin; Bullock, Caroline; McLellan, Michael; Schuierer, Sven; Murphy, Leo; Blommers, Marcel J. J.; Blaustein, Cecile; Berenshteyn, Frada; Lacoste, Arnaud; Thomas, Jason R.; Roma, Guglielmo; Michaud, Gregory A.; Tseng, Brian S.; Porter, Jeffery A.; Myer, Vic E.; Tallarico, John A.; Hamann, Lawrence G.; Curtis, Daniel; Fishman, Mark C.; Dietrich, William F.; Dales, Natalie A.; Sivasankaran, Rajeev</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">511-517</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), which results from the loss of expression of the survival of motor neuron-1 (SMN1) gene, represents the most common genetic cause of pediatric mortality.  A duplicate copy (SMN2) is inefficiently spliced, producing a truncated and unstable protein.  We describe herein a potent, orally active, small-mol. enhancer of SMN2 splicing that elevates full-length SMN protein and extends survival in a severe SMA mouse model.  We demonstrate that the mol. mechanism of action is via stabilization of the transient double-strand RNA structure formed by the SMN2 pre-mRNA and U1 small nuclear ribonucleic protein (snRNP) complex.  The binding affinity of U1 snRNP to the 5' splice site is increased in a sequence-selective manner, discrete from constitutive recognition.  This new mechanism demonstrates the feasibility of small mol.-mediated, sequence-selective splice modulation and the potential for leveraging this strategy in other splicing diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdMqLIC0lw17Vg90H21EOLACvtfcHk0lh8-saLgy8p8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN&md5=15e56c8b3f48feffa150e975a81633a0</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1837%26sid%3Dliteratum%253Aachs%26aulast%3DPalacino%26aufirst%3DJ.%26aulast%3DSwalley%26aufirst%3DS.%2BE.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVan%2BHoosear%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DKeller%26aufirst%3DC.%2BG.%26aulast%3DBeibel%26aufirst%3DM.%26aulast%3DRenaud%26aufirst%3DN.%2BA.%26aulast%3DSmith%26aufirst%3DT.%2BM.%26aulast%3DSalcius%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DBullock%26aufirst%3DC.%26aulast%3DMcLellan%26aufirst%3DM.%26aulast%3DSchuierer%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DL.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%26aulast%3DBlaustein%26aufirst%3DC.%26aulast%3DBerenshteyn%26aufirst%3DF.%26aulast%3DLacoste%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DJ.%2BR.%26aulast%3DRoma%26aufirst%3DG.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DTseng%26aufirst%3DB.%2BS.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DFishman%26aufirst%3DM.%2BC.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DDales%26aufirst%3DN.%2BA.%26aulast%3DSivasankaran%26aufirst%3DR.%26atitle%3DSMN2%2520splice%2520modulators%2520enhance%2520U1-pre-mRNA%2520association%2520and%2520rescue%2520SMA%2520mice%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D511%26epage%3D517%26doi%3D10.1038%2Fnchembio.1837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span> <span> </span><span class="NLM_article-title">Target identification for biologically active small molecules using chemical biology approaches</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1193</span>– <span class="NLM_lpage">1201</span>, <span class="refDoi"> DOI: 10.1007/s12272-016-0791-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2Fs12272-016-0791-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27387321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFertLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2016&pages=1193-1201&author=H.+Leeauthor=J.+W.+Lee&title=Target+identification+for+biologically+active+small+molecules+using+chemical+biology+approaches&doi=10.1007%2Fs12272-016-0791-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Target identification for biologically active small molecules using chemical biology approaches</span></div><div class="casAuthors">Lee, Heesu; Lee, Jae Wook</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1193-1201</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">A review.  The identification and validation of the targets of biol. active mols. is an important step in the field of chem. biol.  While recent advances in proteomic and genomic technol. have accelerated this identification process, the discovery of small mol. targets remains the most challenging step.  A general method for the identification of these small mol. targets has not yet been established.  To overcome the difficulty in target identification, new technol. derived from the fields of genomics, proteomics, and bioinformatics has been developed.  To date, pull-down methods using small mols. immobilized on a solid support followed by mass spectrometry have been the most successful approach.  Here, current procedures for target identification are discussed.  The authors also review the most recent target identification approaches and present several examples that illustrate advanced target identification technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEIcJ1_suxrbVg90H21EOLACvtfcHk0lh8-saLgy8p8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFertLrF&md5=f97036e221fd6e1265d58020b1478a5a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs12272-016-0791-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12272-016-0791-z%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26atitle%3DTarget%2520identification%2520for%2520biologically%2520active%2520small%2520molecules%2520using%2520chemical%2520biology%2520approaches%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2016%26volume%3D39%26spage%3D1193%26epage%3D1201%26doi%3D10.1007%2Fs12272-016-0791-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saxena, C.</span></span> <span> </span><span class="NLM_article-title">Identification of protein binding partners of small molecules using label-free methods</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1017</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1080/17460441.2016.1227316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1080%2F17460441.2016.1227316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27554241" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1017-1025&author=C.+Saxena&title=Identification+of+protein+binding+partners+of+small+molecules+using+label-free+methods&doi=10.1080%2F17460441.2016.1227316"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1080%2F17460441.2016.1227316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2016.1227316%26sid%3Dliteratum%253Aachs%26aulast%3DSaxena%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520protein%2520binding%2520partners%2520of%2520small%2520molecules%2520using%2520label-free%2520methods%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2016%26volume%3D11%26spage%3D1017%26epage%3D1025%26doi%3D10.1080%2F17460441.2016.1227316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span> <span> </span><span class="NLM_article-title">The impact of CRISPR-Cas9 on target identification and validation</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.12.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.drudis.2014.12.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25572406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=450-457&author=J.+D.+Moore&title=The+impact+of+CRISPR-Cas9+on+target+identification+and+validation&doi=10.1016%2Fj.drudis.2014.12.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of CRISPR-Cas9 on target identification and validation</span></div><div class="casAuthors">Moore, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">450-457</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The addn. of an RNA-guided nuclease, Cas9, to the gene editing toolbox has increased the accessibility of gene editing technologies by greatly simplifying the design of editing reagents.  Only a single 75-100 nucleotide RNA is required to guide Cas9 to the target gene of interest, which has meant that the established infrastructure of short-hairpin RNA interference screen could be readily adapted to genome-wide knock out screens.  Cas9-based editing technol. should streamline the generation of animal and cell-line models, make the generation of activity-dead mutations in target validation routine, and enable the discovery of a new generation of targets across therapeutic areas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbf_b_V0znILVg90H21EOLACvtfcHk0liqCgYgns-y6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFWkuw%253D%253D&md5=5d1fcdcf51f9511cc4dad583c65f958d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.12.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.12.016%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520impact%2520of%2520CRISPR-Cas9%2520on%2520target%2520identification%2520and%2520validation%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D450%26epage%3D457%26doi%3D10.1016%2Fj.drudis.2014.12.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">The power of sophisticated phenotypic screening and modern mechanism-of-action methods</span>. <i>Cell Chem Biol</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2015.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26933731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlSqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=3-9&author=B.+K.+Wagnerauthor=S.+L.+Schreiber&title=The+power+of+sophisticated+phenotypic+screening+and+modern+mechanism-of-action+methods&doi=10.1016%2Fj.chembiol.2015.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The Power of Sophisticated Phenotypic Screening and Modern Mechanism-of-Action Methods</span></div><div class="casAuthors">Wagner, Bridget K.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The enthusiasm for phenotypic screening as an approach for small-mol. discovery has increased dramatically over the last several years.  The recent increase in phenotype-based discoveries is in part due to advancements in phenotypic readouts in improved disease models that recapitulate clin. relevant biol. in cell culture.  Of course, a major historical barrier to using phenotypic assays in chem. biol. has been the challenge in detg. the mechanism of action (MoA) for compds. of interest.  With the combination of medically inspired phenotypic screening and the development of modern MoA methods, we can now start implementing this approach in chem. probe and drug discovery.  In this Perspective, we highlight recent advances in phenotypic readouts and MoA detn. by discussing several case studies in which both activities were required for understanding the chem. biol. involved and, in some cases, advancing toward clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVxdoheLgSbbVg90H21EOLACvtfcHk0liqCgYgns-y6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlSqsb4%253D&md5=e3219e156466fa5a32b2cb4bf6869a90</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DThe%2520power%2520of%2520sophisticated%2520phenotypic%2520screening%2520and%2520modern%2520mechanism-of-action%2520methods%26jtitle%3DCell%2520Chem%2520Biol%26date%3D2016%26volume%3D23%26spage%3D3%26epage%3D9%26doi%3D10.1016%2Fj.chembiol.2015.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pries, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Target identification for small bioactive molecules: finding the needle in the haystack</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2744</span>– <span class="NLM_lpage">2792</span>, <span class="refDoi"> DOI: 10.1002/anie.201208749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201208749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23418026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislSksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=2744-2792&author=S.+Zieglerauthor=V.+Priesauthor=C.+Hedbergauthor=H.+Waldmann&title=Target+identification+for+small+bioactive+molecules%3A+finding+the+needle+in+the+haystack&doi=10.1002%2Fanie.201208749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Target Identification for Small Bioactive Molecules: Finding the Needle in the Haystack</span></div><div class="casAuthors">Ziegler, Slava; Pries, Verena; Hedberg, Christian; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2744-2792</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Identification and confirmation of bioactive small-mol. targets is a crucial, often decisive step both in academic and pharmaceutical research.  Through the development and availability of several new exptl. techniques, target identification is, in principle, feasible, and the no. of successful examples steadily grows.  However, a generic methodol. that can successfully be applied in the majority of the cases has not yet been established.  Herein we summarize current methods for target identification of small mols., primarily for a chem. audience but also the biol. community, for example, the chemist or biologist attempting to identify the target of a given bioactive compd.  We describe the most frequently employed exptl. approaches for target identification and provide several representative examples illustrating the state-of-the-art.  Among the techniques currently available, protein affinity isolation using suitable small-mol. probes (pulldown) and subsequent mass spectrometric anal. of the isolated proteins appears to be most powerful and most frequently applied.  To provide guidance for rapid entry into the field and based on our own experience we propose a typical workflow for target identification, which centers on the application of chem. proteomics as the key step to generate hypotheses for potential target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYC9-7L3abMrVg90H21EOLACvtfcHk0liqCgYgns-y6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislSksLk%253D&md5=3a124205e3a57a53f479b889073bda76</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201208749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201208749%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DPries%26aufirst%3DV.%26aulast%3DHedberg%26aufirst%3DC.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DTarget%2520identification%2520for%2520small%2520bioactive%2520molecules%253A%2520finding%2520the%2520needle%2520in%2520the%2520haystack%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D2744%26epage%3D2792%26doi%3D10.1002%2Fanie.201208749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Drug-target interaction prediction: databases, web servers and computational models</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">696</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1093/bib/bbv066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbib%2Fbbv066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26283676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlsVSru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=696-712&author=X.+Chenauthor=C.+C.+Yanauthor=X.+Zhangauthor=X.+Zhangauthor=F.+Daiauthor=J.+Yinauthor=Y.+Zhang&title=Drug-target+interaction+prediction%3A+databases%2C+web+servers+and+computational+models&doi=10.1093%2Fbib%2Fbbv066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target interaction prediction: databases, web servers and computational models</span></div><div class="casAuthors">Chen, Xing; Yan, Chenggang Clarence; Zhang, Xiaotian; Zhang, Xu; Dai, Feng; Yin, Jian; Zhang, Yongdong</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">696-712</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1467-5463</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Identification of drug-target interactions is an important process in drug discovery.  Although high-throughput screening and other biol. assays are becoming available, exptl. methods for drug-target interaction identification remain to be extremely costly, time-consuming and challenging even nowadays.  Therefore, various computational models have been developed to predict potential drug-target assocns. on a large scale.  In this review, databases and web servers involved in drug-target identification and drug discovery are summarized.  In addn., we mainly introduced some state-of-the-art computational models for drug-target interactions prediction, including network-based method, machine learning-based method and so on.  Specially, for the machine learning-based method, much attention was paid to super-vised and semi-supervised models, which have essential difference in the adoption of neg. samples.  Although significant improvements for drug-target interaction prediction have been obtained by many effective computational models, both network-based and machine learning-based methods have their disadvantages, resp.  Furthermore, we discuss the future directions of the network-based drug discovery and network approach for personalized drug discovery based on personalized medicine, genome sequencing, tumor clone-based network and cancer hallmark-based network.  Finally, we discussed the new evaluation validation framework and the formulation of drug-target interactions prediction problem by more realistic regression formulation based on quant. bioactivity data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMvmzj7AuWvrVg90H21EOLACvtfcHk0lglhRNy4v63ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlsVSru7g%253D&md5=fa9d08bd0fdf4b039acac3baad7d8d60</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbv066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbv066%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DC.%2BC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DF.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDrug-target%2520interaction%2520prediction%253A%2520databases%252C%2520web%2520servers%2520and%2520computational%2520models%26jtitle%3DBriefings%2520Bioinf.%26date%3D2016%26volume%3D17%26spage%3D696%26epage%3D712%26doi%3D10.1093%2Fbib%2Fbbv066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kremer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habenberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choidas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kordes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholer, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterneckert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel inhibitor of the Hedgehog signaling pathway through cell-based compound discovery and target prediction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">13021</span>– <span class="NLM_lpage">13025</span>, <span class="refDoi"> DOI: 10.1002/anie.201707394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201707394" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=13021-13025&author=L.+Kremerauthor=C.+Schultz-Fademrechtauthor=M.+Baumannauthor=P.+Habenbergerauthor=A.+Choidasauthor=B.+Kleblauthor=S.+Kordesauthor=H.+R.+Scholerauthor=J.+Sterneckertauthor=S.+Zieglerauthor=G.+Schneiderauthor=H.+Waldmann&title=Discovery+of+a+novel+inhibitor+of+the+Hedgehog+signaling+pathway+through+cell-based+compound+discovery+and+target+prediction&doi=10.1002%2Fanie.201707394"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707394%26sid%3Dliteratum%253Aachs%26aulast%3DKremer%26aufirst%3DL.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DHabenberger%26aufirst%3DP.%26aulast%3DChoidas%26aufirst%3DA.%26aulast%3DKlebl%26aufirst%3DB.%26aulast%3DKordes%26aufirst%3DS.%26aulast%3DScholer%26aufirst%3DH.%2BR.%26aulast%3DSterneckert%26aufirst%3DJ.%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520inhibitor%2520of%2520the%2520Hedgehog%2520signaling%2520pathway%2520through%2520cell-based%2520compound%2520discovery%2520and%2520target%2520prediction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D13021%26epage%3D13025%26doi%3D10.1002%2Fanie.201707394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecassis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguiz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeons, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidah, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constam, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetsel, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.+P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lglhRNy4v63ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Somody, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKinnon, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windemuth, A.</span></span> <span> </span><span class="NLM_article-title">Structural coverage of the proteome for pharmaceutical applications</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1792</span>– <span class="NLM_lpage">1799</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.drudis.2017.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28843631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtl2ltLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1792-1799&author=J.+C.+Somodyauthor=S.+S.+MacKinnonauthor=A.+Windemuth&title=Structural+coverage+of+the+proteome+for+pharmaceutical+applications&doi=10.1016%2Fj.drudis.2017.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structural coverage of the proteome for pharmaceutical applications</span></div><div class="casAuthors">Somody, Joseph C.; MacKinnon, Stephen S.; Windemuth, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1792-1799</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Structure-based computational drug discovery efforts have traditionally focused on the structure of a single, well-known drug target.  Important applications, such as target deconvolution and the anal. of polypharmacol., require proteome-scale mol. docking and have been inaccessible to structure-based in silico approaches.  One important reason for this inaccessibility was that the structure of most proteins was not known.  Lately, this 'structure gap' has been closing rapidly, and proteome-scale mol. docking seems within reach.  Here, we survey the current state of structural coverage of the human genome and find that coverage is truly proteome-wide, both overall and in most pharmaceutically relevant categories of proteins.  The time is right for structure-based approaches to target deconvolution and polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozH-nC3SqUs7Vg90H21EOLACvtfcHk0lgA7xj9RqK2-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtl2ltLbL&md5=a3aae9e89ea541b6218b4f513d7218f3</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DSomody%26aufirst%3DJ.%2BC.%26aulast%3DMacKinnon%26aufirst%3DS.%2BS.%26aulast%3DWindemuth%26aufirst%3DA.%26atitle%3DStructural%2520coverage%2520of%2520the%2520proteome%2520for%2520pharmaceutical%2520applications%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D1792%26epage%3D1799%26doi%3D10.1016%2Fj.drudis.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savage, N.</span></span> <span> </span><span class="NLM_article-title">Proteomics: High-protein research</span>. <i>Nature (London, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>527</i></span>,  <span class="NLM_fpage">S6</span>– <span class="NLM_lpage">S7</span>, <span class="refDoi"> DOI: 10.1038/527S6a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2F527S6a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslOrtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=527&publication_year=2015&pages=S6-S7&author=N.+Savage&title=Proteomics%3A+High-protein+research&doi=10.1038%2F527S6a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics: High-protein research</span></div><div class="casAuthors">Savage, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">527</span>
        (<span class="NLM_cas:issue">7576</span>),
    <span class="NLM_cas:pages">S6-S7</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on how proteomics affects cell biol., the effort to catalog proteins going deeper in a push to make genetics research deliver practical benefits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqksnuhmBvFLVg90H21EOLACvtfcHk0lgA7xj9RqK2-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslOrtb7E&md5=10b10630a4c2b2114de73f58facade89</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2F527S6a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F527S6a%26sid%3Dliteratum%253Aachs%26aulast%3DSavage%26aufirst%3DN.%26atitle%3DProteomics%253A%2520High-protein%2520research%26jtitle%3DNature%2520%2528London%252C%2520U.%2520K.%2529%26date%3D2015%26volume%3D527%26spage%3DS6%26epage%3DS7%26doi%3D10.1038%2F527S6a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drissi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boisvert, F.-M.</span></span> <span> </span><span class="NLM_article-title">Proteomics methods for subcellular proteome analysis</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">5626</span>– <span class="NLM_lpage">5634</span>, <span class="refDoi"> DOI: 10.1111/febs.12502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1111%2Ffebs.12502" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=5626-5634&author=R.+Drissiauthor=M.-L.+Duboisauthor=F.-M.+Boisvert&title=Proteomics+methods+for+subcellular+proteome+analysis&doi=10.1111%2Ffebs.12502"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12502%26sid%3Dliteratum%253Aachs%26aulast%3DDrissi%26aufirst%3DR.%26aulast%3DDubois%26aufirst%3DM.-L.%26aulast%3DBoisvert%26aufirst%3DF.-M.%26atitle%3DProteomics%2520methods%2520for%2520subcellular%2520proteome%2520analysis%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D5626%26epage%3D5634%26doi%3D10.1111%2Ffebs.12502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goedken, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argiriadi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiamengo, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ihle, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merta, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalak, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murdock, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, J. W.</span></span> <span> </span><span class="NLM_article-title">Tricyclic covalent inhibitors selectively target Jak3 through an active site thiol</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">4573</span>– <span class="NLM_lpage">4589</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.595181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1074%2Fjbc.M114.595181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25552479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsF2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=4573-4589&author=E.+R.+Goedkenauthor=M.+A.+Argiriadiauthor=D.+L.+Banachauthor=B.+A.+Fiamengoauthor=S.+E.+Foleyauthor=K.+E.+Frankauthor=J.+S.+Georgeauthor=C.+M.+Harrisauthor=A.+D.+Hobsonauthor=D.+C.+Ihleauthor=D.+Marcotteauthor=P.+J.+Mertaauthor=M.+E.+Michalakauthor=S.+E.+Murdockauthor=M.+J.+Tomlinsonauthor=J.+W.+Voss&title=Tricyclic+covalent+inhibitors+selectively+target+Jak3+through+an+active+site+thiol&doi=10.1074%2Fjbc.M114.595181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Tricyclic Covalent Inhibitors Selectively Target Jak3 through an Active Site Thiol</span></div><div class="casAuthors">Goedken, Eric R.; Argiriadi, Maria A.; Banach, David L.; Fiamengo, Bryan A.; Foley, Sage E.; Frank, Kristine E.; George, Jonathan S.; Harris, Christopher M.; Hobson, Adrian D.; Ihle, David C.; Marcotte, Douglas; Merta, Philip J.; Michalak, Mark E.; Murdock, Sara E.; Tomlinson, Medha J.; Voss, Jeffrey W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4573-4589</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The action of Janus kinases (JAKs) is required for multiple cytokine signaling pathways, and as such, JAK inhibitors hold promise for treatment of autoimmune disorders, including rheumatoid arthritis, inflammatory bowel disease, and psoriasis.  However, due to high similarity in the active sites of the four members (Jak1, Jak2, Jak3, and Tyk2), developing selective inhibitors within this family is challenging.  We have designed and characterized substituted, tricyclic Jak3 inhibitors that selectively avoid inhibition of the other JAKs.  This is accomplished through a covalent interaction between an inhibitor contg. a terminal electrophile and an active site cysteine (Cys-909).  We found that these ATP competitive compds. are irreversible inhibitors of Jak3 enzyme activity in vitro.  They possess high selectivity against other kinases and can potently (IC50 < 100 nm) inhibit Jak3 activity in cell-based assays.  These results suggest irreversible inhibitors of this class may be useful selective agents, both as tools to probe Jak3 biol. and potentially as therapies for autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjt9jFjpWGbVg90H21EOLACvtfcHk0lj6LieVCna8UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsF2lsLY%253D&md5=4d99cf5b13fd05df5c4abc197c1fe284</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.595181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.595181%26sid%3Dliteratum%253Aachs%26aulast%3DGoedken%26aufirst%3DE.%2BR.%26aulast%3DArgiriadi%26aufirst%3DM.%2BA.%26aulast%3DBanach%26aufirst%3DD.%2BL.%26aulast%3DFiamengo%26aufirst%3DB.%2BA.%26aulast%3DFoley%26aufirst%3DS.%2BE.%26aulast%3DFrank%26aufirst%3DK.%2BE.%26aulast%3DGeorge%26aufirst%3DJ.%2BS.%26aulast%3DHarris%26aufirst%3DC.%2BM.%26aulast%3DHobson%26aufirst%3DA.%2BD.%26aulast%3DIhle%26aufirst%3DD.%2BC.%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DMichalak%26aufirst%3DM.%2BE.%26aulast%3DMurdock%26aufirst%3DS.%2BE.%26aulast%3DTomlinson%26aufirst%3DM.%2BJ.%26aulast%3DVoss%26aufirst%3DJ.%2BW.%26atitle%3DTricyclic%2520covalent%2520inhibitors%2520selectively%2520target%2520Jak3%2520through%2520an%2520active%2520site%2520thiol%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D4573%26epage%3D4589%26doi%3D10.1074%2Fjbc.M114.595181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, J. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vleet, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittelstadt, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomme, E. A. G.</span></span> <span> </span><span class="NLM_article-title">Potential functional and pathological side effects related to off-target pharmacological activity</span>. <i>J. Pharmacol. Toxicol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">108</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.vascn.2017.02.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.vascn.2017.02.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28216264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVGltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2017&pages=108-126&author=J.+J.+Lynchauthor=T.+R.+Van+Vleetauthor=S.+W.+Mittelstadtauthor=E.+A.+G.+Blomme&title=Potential+functional+and+pathological+side+effects+related+to+off-target+pharmacological+activity&doi=10.1016%2Fj.vascn.2017.02.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Potential functional and pathological side effects related to off-target pharmacological activity</span></div><div class="casAuthors">Lynch, James J., III; Van Vleet, Terry R.; Mittelstadt, Scott W.; Blomme, Eric A. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological and Toxicological Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108-126</span>CODEN:
                <span class="NLM_cas:coden">JPTMEZ</span>;
        ISSN:<span class="NLM_cas:issn">1056-8719</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Most pharmaceutical companies test their discovery-stage proprietary mols. in a battery of in vitro pharmacol. assays to try to det. off-target interactions.  During all phases of drug discovery and development, various questions arise regarding potential side effects assocd. with such off-target pharmacol. activity.  Here we present a scientific literature curation effort undertaken to det. and summarize the most likely functional and pathol. outcomes assocd. with interactions at 70 receptors, enzymes, ion channels and transporters with established links to adverse effects.  To that end, the scientific literature was reviewed using an online database, and the most commonly reported effects were summarized in tabular format.  The resultant table should serve as a practical guide for research scientists and clin. investigators for the prediction and interpretation of adverse side effects assocd. with mols. interacting with components of this screening battery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr73k3f-8KK17Vg90H21EOLACvtfcHk0lj6LieVCna8UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsVGltrg%253D&md5=86b82acd8981a020f19e236d2d4ecb16</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.vascn.2017.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.vascn.2017.02.020%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BJ.%26aulast%3DVan%2BVleet%26aufirst%3DT.%2BR.%26aulast%3DMittelstadt%26aufirst%3DS.%2BW.%26aulast%3DBlomme%26aufirst%3DE.%2BA.%2BG.%26atitle%3DPotential%2520functional%2520and%2520pathological%2520side%2520effects%2520related%2520to%2520off-target%2520pharmacological%2520activity%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2017%26volume%3D87%26spage%3D108%26epage%3D126%26doi%3D10.1016%2Fj.vascn.2017.02.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viklund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsblom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresaugues, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quanz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak-Reppel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irlbacher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queisser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haegebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjanacz, M.</span></span> <span> </span><span class="NLM_article-title">Novel class of potent and cellularly active inhibitors devalidates MTH1 as broad-spectrum cancer target</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1986</span>– <span class="NLM_lpage">1992</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1986-1992&author=M.+Ellermannauthor=A.+Eheimauthor=F.+Rahmauthor=J.+Viklundauthor=J.+Guentherauthor=M.+Anderssonauthor=U.+Ericssonauthor=R.+Forsblomauthor=T.+Ginmanauthor=J.+Lindstromauthor=C.+Silvanderauthor=L.+Tresauguesauthor=A.+Gieseauthor=S.+Bunseauthor=R.+Neuhausauthor=J.+Weiskeauthor=M.+Quanzauthor=A.+Glasauerauthor=K.+Nowak-Reppelauthor=B.+Baderauthor=H.+Irlbacherauthor=H.+Meyerauthor=N.+Queisserauthor=M.+Bauserauthor=A.+Haegebarthauthor=M.+Gorjanacz&title=Novel+class+of+potent+and+cellularly+active+inhibitors+devalidates+MTH1+as+broad-spectrum+cancer+target&doi=10.1021%2Facschembio.7b00370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Class of Potent and Cellularly Active Inhibitors Devalidates MTH1 as Broad-Spectrum Cancer Target</span></div><div class="casAuthors">Ellermann, Manuel; Eheim, Ashley; Rahm, Fredrik; Viklund, Jenny; Guenther, Judith; Andersson, Martin; Ericsson, Ulrika; Forsblom, Rickard; Ginman, Tobias; Lindstroem, Johan; Silvander, Camilla; Tresaugues, Lionel; Giese, Anja; Bunse, Stefanie; Neuhaus, Roland; Weiske, Joerg; Quanz, Maria; Glasauer, Andrea; Nowak-Reppel, Katrin; Bader, Benjamin; Irlbacher, Horst; Meyer, Hanna; Queisser, Nina; Bauser, Marcus; Haegebarth, Andrea; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1986-1992</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MTH1 is a hydrolase responsible for sanitization of oxidized purine nucleoside triphosphates to prevent their incorporation into replicating DNA.  Early tool compds. published in the literature inhibited the enzymic activity of MTH1 and subsequently induced cancer cell death; however recent studies have questioned the reported link between these two events.  Therefore, it is important to validate MTH1 as a cancer dependency with high quality chem. probes.  Here, we present BAY-707, a substrate-competitive, highly potent and selective inhibitor of MTH1, chem. distinct compared to those previously published.  Despite superior cellular target engagement and pharmacokinetic properties, inhibition of MTH1 with BAY-707 resulted in a clear lack of in vitro or in vivo anticancer efficacy either in mono- or in combination therapies.  Therefore, we conclude that MTH1 is dispensable for cancer cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHT2Ivo0Gx87Vg90H21EOLACvtfcHk0lj6LieVCna8UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsrbP&md5=69aa2247fae019cf695df8352e2ade18</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00370%26sid%3Dliteratum%253Aachs%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DEheim%26aufirst%3DA.%26aulast%3DRahm%26aufirst%3DF.%26aulast%3DViklund%26aufirst%3DJ.%26aulast%3DGuenther%26aufirst%3DJ.%26aulast%3DAndersson%26aufirst%3DM.%26aulast%3DEricsson%26aufirst%3DU.%26aulast%3DForsblom%26aufirst%3DR.%26aulast%3DGinman%26aufirst%3DT.%26aulast%3DLindstrom%26aufirst%3DJ.%26aulast%3DSilvander%26aufirst%3DC.%26aulast%3DTresaugues%26aufirst%3DL.%26aulast%3DGiese%26aufirst%3DA.%26aulast%3DBunse%26aufirst%3DS.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DQuanz%26aufirst%3DM.%26aulast%3DGlasauer%26aufirst%3DA.%26aulast%3DNowak-Reppel%26aufirst%3DK.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DIrlbacher%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DQueisser%26aufirst%3DN.%26aulast%3DBauser%26aufirst%3DM.%26aulast%3DHaegebarth%26aufirst%3DA.%26aulast%3DGorjanacz%26aufirst%3DM.%26atitle%3DNovel%2520class%2520of%2520potent%2520and%2520cellularly%2520active%2520inhibitors%2520devalidates%2520MTH1%2520as%2520broad-spectrum%2520cancer%2520target%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D1986%26epage%3D1992%26doi%3D10.1021%2Facschembio.7b00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blomme, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, Y.</span></span> <span> </span><span class="NLM_article-title">Toxicology strategies for drug discovery: present and future</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">504</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrestox.5b00407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.5b00407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOhu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=473-504&author=E.+A.+Blommeauthor=Y.+Will&title=Toxicology+strategies+for+drug+discovery%3A+present+and+future&doi=10.1021%2Facs.chemrestox.5b00407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicology Strategies for Drug Discovery: Present and Future</span></div><div class="casAuthors">Blomme, Eric A. G.; Will, Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-504</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Attrition due to nonclin. safety represents a major issue for the productivity of pharmaceutical research and development (R&D) organizations, esp. during the compd. optimization stages of drug discovery and the early stages of clin. development.  Focusing on decreasing nonclin. safety-related attrition is not a new concept, and various approaches have been experimented with over the last two decades.  Front-loading testing funnels in Discovery with in vitro toxicity assays designed to rapidly identify unfavorable mols. was the approach adopted by most pharmaceutical R&D organizations a few years ago.  However, this approach has also a non-negligible opportunity cost.  Hence, significant refinements to the "fail early, fail often" paradigm have been proposed recently to reflect the complexity of accurately categorizing compds. with early data points without taking into account other important contextual aspects, in particular efficacious systemic and tissue exposures.  This review provides an overview of toxicol. approaches and models that can be used in pharmaceutical Discovery at the series/lead identification and lead optimization stages to guide and inform chem. efforts, as well as a personal view on how to best use them to meet nonclin. safety-related attrition objectives consistent with a sustainable pharmaceutical R&D model.  The scope of this review is limited to small mols., as large mols. are assocd. with challenges that are quite different.  Finally, a perspective on how several emerging technologies may impact toxicity evaluation is also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnswsW0Ve2_rVg90H21EOLACvtfcHk0ljzuHjZR2Yk0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOhu7vI&md5=ec2ad8b3576b7ba00f750e3f93469a2f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.5b00407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.5b00407%26sid%3Dliteratum%253Aachs%26aulast%3DBlomme%26aufirst%3DE.%2BA.%26aulast%3DWill%26aufirst%3DY.%26atitle%3DToxicology%2520strategies%2520for%2520drug%2520discovery%253A%2520present%2520and%2520future%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2016%26volume%3D29%26spage%3D473%26epage%3D504%26doi%3D10.1021%2Facs.chemrestox.5b00407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marroquin, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamieson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, Y.</span></span> <span> </span><span class="NLM_article-title">Circumventing the Crabtree effect: replacing media glucose with galactose increases susceptibility of HepG2 cells to mitochondrial toxicants</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfm052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Ftoxsci%2Fkfm052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=17361016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsV2is7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2007&pages=539-547&author=L.+D.+Marroquinauthor=J.+Hynesauthor=J.+A.+Dykensauthor=J.+D.+Jamiesonauthor=Y.+Will&title=Circumventing+the+Crabtree+effect%3A+replacing+media+glucose+with+galactose+increases+susceptibility+of+HepG2+cells+to+mitochondrial+toxicants&doi=10.1093%2Ftoxsci%2Fkfm052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Circumventing the Crabtree effect: replacing media glucose with galactose increases susceptibility of HepG2 cells to mitochondrial toxicants</span></div><div class="casAuthors">Marroquin, Lisa D.; Hynes, James; Dykens, James A.; Jamieson, Joseph D.; Will, Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">539-547</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-6080</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Many highly proliferative cells generate almost all ATP via glycolysis despite abundant O2 and a normal complement of fully functional mitochondria, a circumstance known as the Crabtree effect.  Such anaerobically poised cells are resistant to xenobiotics that impair mitochondrial function, such as the inhibitors rotenone, antimycin, oligomycin, and compds. like carbonyl cyanide 4-(trifluoromethoxy)phenylhydrazone (FCCP), that uncouple the respiratory electron transfer system from phosphorylation.  These cells are also resistant to the toxicity of many drugs whose deleterious side effect profiles are either caused, or exacerbated, by impairment of mitochondrial function.  Drug-induced mitochondrial toxicity is shown by members of important drug classes, including the thiazolidinediones, statins, fibrates, antivirals, antibiotics, and anticancer agents.  To increase detection of drug-induced mitochondrial effects in a preclin. cell-based assay, HepG2 cells were forced to rely on mitochondrial oxidative phosphorylation rather than glycolysis by substituting galactose for glucose in the growth media.  Oxygen consumption doubles in galactose-grown HepG2 cells and their susceptibility to canonical mitochondrial toxicants correspondingly increases.  Similarly, toxicity of several drugs with known mitochondrial liabilities is more readily apparent in aerobically poised HepG2 cells compared to glucose-grown cells.  Some drugs were equally toxic to both glucose- and galactose-grown cells, suggesting that mitochondrial impairment is likely secondary to other cytotoxic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Zb0elenmNbVg90H21EOLACvtfcHk0ljzuHjZR2Yk0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsV2is7w%253D&md5=53d6a955ad0a942d1ced5043668bed19</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfm052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfm052%26sid%3Dliteratum%253Aachs%26aulast%3DMarroquin%26aufirst%3DL.%2BD.%26aulast%3DHynes%26aufirst%3DJ.%26aulast%3DDykens%26aufirst%3DJ.%2BA.%26aulast%3DJamieson%26aufirst%3DJ.%2BD.%26aulast%3DWill%26aufirst%3DY.%26atitle%3DCircumventing%2520the%2520Crabtree%2520effect%253A%2520replacing%2520media%2520glucose%2520with%2520galactose%2520increases%2520susceptibility%2520of%2520HepG2%2520cells%2520to%2520mitochondrial%2520toxicants%26jtitle%3DToxicol.%2520Sci.%26date%3D2007%26volume%3D97%26spage%3D539%26epage%3D547%26doi%3D10.1093%2Ftoxsci%2Fkfm052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span> <span> </span><span class="NLM_article-title">The Connectivity Map: a new tool for biomedical research</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1038/nrc2044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrc2044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=17186018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCht7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=54-60&author=J.+Lamb&title=The+Connectivity+Map%3A+a+new+tool+for+biomedical+research&doi=10.1038%2Fnrc2044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Connectivity Map: a new tool for biomedical research</span></div><div class="casAuthors">Lamb, Justin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">54-60</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The ultimate objective of biomedical research is to connect human diseases with the genes that underlie them and drugs that treat them.  But this remains a daunting task, and even the most inspired researchers still have to resort to laborious screens of genetic or chem. libraries.  What if at least some parts of this screening process could be systematized and centralized.  And hits found and hypotheses generated with something resembling an internet search engine.  These are the questions the Connectivity Map project set out to answer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3OVVSFxDDUbVg90H21EOLACvtfcHk0ljzuHjZR2Yk0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCht7nF&md5=8f23f79ae51eb97eb067f5357baef280</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnrc2044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2044%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DJ.%26atitle%3DThe%2520Connectivity%2520Map%253A%2520a%2520new%2520tool%2520for%2520biomedical%2520research%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D54%26epage%3D60%26doi%3D10.1038%2Fnrc2044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blat, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hieronymus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">1929</span>– <span class="NLM_lpage">1935</span>, <span class="refDoi"> DOI: 10.1126/science.1132939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1132939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=17008526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&author=J.+Lambauthor=E.+D.+Crawfordauthor=D.+Peckauthor=J.+W.+Modellauthor=I.+C.+Blatauthor=M.+J.+Wrobelauthor=J.+Lernerauthor=J.+P.+Brunetauthor=A.+Subramanianauthor=K.+N.+Rossauthor=M.+Reichauthor=H.+Hieronymusauthor=G.+Weiauthor=S.+A.+Armstrongauthor=S.+J.+Haggartyauthor=P.+A.+Clemonsauthor=R.+Weiauthor=S.+A.+Carrauthor=E.+S.+Landerauthor=T.+R.+Golub&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease&doi=10.1126%2Fscience.1132939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The connectivity map: Using gene-expression signatures to connect small molecules, genes, and disease</span></div><div class="casAuthors">Lamb, Justin; Crawford, Emily D.; Peck, David; Modell, Joshua W.; Blat, Irene C.; Wrobel, Matthew J.; Lerner, Jim; Brunet, Jean-Philippe; Subramanian, Aravind; Ross, Kenneth N.; Reich, Michael; Hieronymus, Haley; Wei, Guo; Armstrong, Scott A.; Haggarty, Stephen J.; Clemons, Paul A.; Wei, Ru; Carr, Steven A.; Lander, Eric S.; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">5795</span>),
    <span class="NLM_cas:pages">1929-1935</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">To pursue a systematic approach to the discovery of functional connections among diseases, genetic perturbation, and drug action, we have created the first installment of a ref. collection of gene-expression profiles from cultured human cells treated with bioactive small mols., together with pattern-matching software to mine these data.  We demonstrate that this "Connectivity Map" resource can be used to find connections among small mols. sharing a mechanism of action, chems. and physiol. processes, and diseases and drugs.  These results indicate the feasibility of the approach and suggest the value of a large-scale community Connectivity Map project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtu0M2BUO2w7Vg90H21EOLACvtfcHk0lizvc4DkHDXNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN&md5=1fad887ad515ffdd8064e526ff25a4db</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1126%2Fscience.1132939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1132939%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DE.%2BD.%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DModell%26aufirst%3DJ.%2BW.%26aulast%3DBlat%26aufirst%3DI.%2BC.%26aulast%3DWrobel%26aufirst%3DM.%2BJ.%26aulast%3DLerner%26aufirst%3DJ.%26aulast%3DBrunet%26aufirst%3DJ.%2BP.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DHieronymus%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DThe%2520Connectivity%2520Map%253A%2520using%2520gene-expression%2520signatures%2520to%2520connect%2520small%2520molecules%252C%2520genes%252C%2520and%2520disease%26jtitle%3DScience%26date%3D2006%26volume%3D313%26spage%3D1929%26epage%3D1935%26doi%3D10.1126%2Fscience.1132939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span> <span> </span><span class="NLM_article-title">A method for high-throughput gene expression signature analysis</span>. <i>Genome Biol</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">R61</span>, <span class="refDoi"> DOI: 10.1186/gb-2006-7-7-r61</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1186%2Fgb-2006-7-7-r61" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=R61&author=D.+Peckauthor=E.+D.+Crawfordauthor=K.+N.+Rossauthor=K.+Stegmaierauthor=T.+R.+Golubauthor=J.+Lamb&title=A+method+for+high-throughput+gene+expression+signature+analysis&doi=10.1186%2Fgb-2006-7-7-r61"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1186%2Fgb-2006-7-7-r61&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2006-7-7-r61%26sid%3Dliteratum%253Aachs%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DCrawford%26aufirst%3DE.%2BD.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLamb%26aufirst%3DJ.%26atitle%3DA%2520method%2520for%2520high-throughput%2520gene%2520expression%2520signature%2520analysis%26jtitle%3DGenome%2520Biol%26date%3D2006%26volume%3D7%26spage%3DR61%26doi%3D10.1186%2Fgb-2006-7-7-r61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niepel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhlich, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouillard, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma’ayan, A.</span></span> <span> </span><span class="NLM_article-title">LINCS Canvas Browser: interactive web app to query, browse and interrogate LINCS L1000 gene expression signatures</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">W449</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1093/nar/gku476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgku476" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W449-460&author=Q.+Duanauthor=C.+Flynnauthor=M.+Niepelauthor=M.+Hafnerauthor=J.+L.+Muhlichauthor=N.+F.+Fernandezauthor=A.+D.+Rouillardauthor=C.+M.+Tanauthor=E.+Y.+Chenauthor=T.+R.+Golubauthor=P.+K.+Sorgerauthor=A.+Subramanianauthor=A.+Ma%E2%80%99ayan&title=LINCS+Canvas+Browser%3A+interactive+web+app+to+query%2C+browse+and+interrogate+LINCS+L1000+gene+expression+signatures&doi=10.1093%2Fnar%2Fgku476"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku476%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DQ.%26aulast%3DFlynn%26aufirst%3DC.%26aulast%3DNiepel%26aufirst%3DM.%26aulast%3DHafner%26aufirst%3DM.%26aulast%3DMuhlich%26aufirst%3DJ.%2BL.%26aulast%3DFernandez%26aufirst%3DN.%2BF.%26aulast%3DRouillard%26aufirst%3DA.%2BD.%26aulast%3DTan%26aufirst%3DC.%2BM.%26aulast%3DChen%26aufirst%3DE.%2BY.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DMa%25E2%2580%2599ayan%26aufirst%3DA.%26atitle%3DLINCS%2520Canvas%2520Browser%253A%2520interactive%2520web%2520app%2520to%2520query%252C%2520browse%2520and%2520interrogate%2520LINCS%2520L1000%2520gene%2520expression%2520signatures%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DW449%26epage%3D460%26doi%3D10.1093%2Fnar%2Fgku476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsello, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natoli, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubelli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asiedu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahr, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadden, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orzechowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enache, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccioni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrcic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosains, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardlie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenside, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read-Button, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronco, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doench, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">A next generation Connectivity Map: L1000 platform and the first 1,000,000 profiles</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1437</span>– <span class="NLM_lpage">1452</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.10.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.cell.2017.10.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=29195078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452&author=A.+Subramanianauthor=R.+Narayanauthor=S.+M.+Corselloauthor=D.+D.+Peckauthor=T.+E.+Natoliauthor=X.+Luauthor=J.+Gouldauthor=J.+F.+Davisauthor=A.+A.+Tubelliauthor=J.+K.+Asieduauthor=D.+L.+Lahrauthor=J.+E.+Hirschmanauthor=Z.+Liuauthor=M.+Donahueauthor=B.+Julianauthor=M.+Khanauthor=D.+Waddenauthor=I.+C.+Smithauthor=D.+Lamauthor=A.+Liberzonauthor=C.+Toderauthor=M.+Bagulauthor=M.+Orzechowskiauthor=O.+M.+Enacheauthor=F.+Piccioniauthor=S.+A.+Johnsonauthor=N.+J.+Lyonsauthor=A.+H.+Bergerauthor=A.+F.+Shamjiauthor=A.+N.+Brooksauthor=A.+Vrcicauthor=C.+Flynnauthor=J.+Rosainsauthor=D.+Y.+Takedaauthor=R.+Huauthor=D.+Davisonauthor=J.+Lambauthor=K.+Ardlieauthor=L.+Hogstromauthor=P.+Greensideauthor=N.+S.+Grayauthor=P.+A.+Clemonsauthor=S.+Silverauthor=X.+Wuauthor=W.+N.+Zhaoauthor=W.+Read-Buttonauthor=X.+Wuauthor=S.+J.+Haggartyauthor=L.+V.+Roncoauthor=J.+S.+Boehmauthor=S.+L.+Schreiberauthor=J.+G.+Doenchauthor=J.+A.+Bittkerauthor=D.+E.+Rootauthor=B.+Wongauthor=T.+R.+Golub&title=A+next+generation+Connectivity+Map%3A+L1000+platform+and+the+first+1%2C000%2C000+profiles&doi=10.1016%2Fj.cell.2017.10.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles</span></div><div class="casAuthors">Subramanian, Aravind; Narayan, Rajiv; Corsello, Steven M.; Peck, David D.; Natoli, Ted E.; Lu, Xiaodong; Gould, Joshua; Davis, John F.; Tubelli, Andrew A.; Asiedu, Jacob K.; Lahr, David L.; Hirschman, Jodi E.; Liu, Zihan; Donahue, Melanie; Julian, Bina; Khan, Mariya; Wadden, David; Smith, Ian C.; Lam, Daniel; Liberzon, Arthur; Toder, Courtney; Bagul, Mukta; Orzechowski, Marek; Enache, Oana M.; Piccioni, Federica; Johnson, Sarah A.; Lyons, Nicholas J.; Berger, Alice H.; Shamji, Alykhan F.; Brooks, Angela N.; Vrcic, Anita; Flynn, Corey; Rosains, Jacqueline; Takeda, David Y.; Hu, Roger; Davison, Desiree; Lamb, Justin; Ardlie, Kristin; Hogstrom, Larson; Greenside, Peyton; Gray, Nathanael S.; Clemons, Paul A.; Silver, Serena; Wu, Xiaoyun; Zhao, Wen-Ning; Read-Button, Willis; Wu, Xiaohua; Haggarty, Stephen J.; Ronco, Lucienne V.; Boehm, Jesse S.; Schreiber, Stuart L.; Doench, John G.; Bittker, Joshua A.; Root, David E.; Wong, Bang; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1437-1452.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We previously piloted the concept of a Connectivity Map (CMap), whereby genes, drugs, and disease states are connected by virtue of common gene-expression signatures.  Here, we report more than a 1000-fold scale-up of the CMap as part of the NIH LINCS Consortium, made possible by a new, low-cost, high-throughput reduced representation expression profiling method that we term L1000.  We show that L1000 is highly reproducible, comparable to RNA sequencing, and suitable for computational inference of the expression levels of 81% of non-measured transcripts.  We further show that the expanded CMap can be used to discover mechanism of action of small mols., functionally annotate genetic variants of disease genes, and inform clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIWrWy5IxY-rVg90H21EOLACvtfcHk0lj65_Gfq_6vvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE&md5=d2b13f2b80fe94021ac03f6a8c859a4a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.10.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.10.049%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DNarayan%26aufirst%3DR.%26aulast%3DCorsello%26aufirst%3DS.%2BM.%26aulast%3DPeck%26aufirst%3DD.%2BD.%26aulast%3DNatoli%26aufirst%3DT.%2BE.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DGould%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DJ.%2BF.%26aulast%3DTubelli%26aufirst%3DA.%2BA.%26aulast%3DAsiedu%26aufirst%3DJ.%2BK.%26aulast%3DLahr%26aufirst%3DD.%2BL.%26aulast%3DHirschman%26aufirst%3DJ.%2BE.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDonahue%26aufirst%3DM.%26aulast%3DJulian%26aufirst%3DB.%26aulast%3DKhan%26aufirst%3DM.%26aulast%3DWadden%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DI.%2BC.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DLiberzon%26aufirst%3DA.%26aulast%3DToder%26aufirst%3DC.%26aulast%3DBagul%26aufirst%3DM.%26aulast%3DOrzechowski%26aufirst%3DM.%26aulast%3DEnache%26aufirst%3DO.%2BM.%26aulast%3DPiccioni%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DS.%2BA.%26aulast%3DLyons%26aufirst%3DN.%2BJ.%26aulast%3DBerger%26aufirst%3DA.%2BH.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DVrcic%26aufirst%3DA.%26aulast%3DFlynn%26aufirst%3DC.%26aulast%3DRosains%26aufirst%3DJ.%26aulast%3DTakeda%26aufirst%3DD.%2BY.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DDavison%26aufirst%3DD.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DHogstrom%26aufirst%3DL.%26aulast%3DGreenside%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DSilver%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%2BN.%26aulast%3DRead-Button%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRonco%26aufirst%3DL.%2BV.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DDoench%26aufirst%3DJ.%2BG.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DA%2520next%2520generation%2520Connectivity%2520Map%253A%2520L1000%2520platform%2520and%2520the%2520first%25201%252C000%252C000%2520profiles%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D1437%26epage%3D1452%26doi%3D10.1016%2Fj.cell.2017.10.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span> <span> </span><span class="NLM_article-title">Systematic quality control analysis of LINCS data</span>. <i>CPT: Pharmacometrics Syst. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1002/psp4.12107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fpsp4.12107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27796074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGgur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=588-598&author=L.+Chengauthor=L.+Li&title=Systematic+quality+control+analysis+of+LINCS+data&doi=10.1002%2Fpsp4.12107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Quality Control Analysis of LINCS Data</span></div><div class="casAuthors">Cheng, L.; Li, L.</div><div class="citationInfo"><span class="NLM_cas:title">CPT: Pharmacometrics & Systems Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">588-598</span>CODEN:
                <span class="NLM_cas:coden">CPSPBR</span>;
        ISSN:<span class="NLM_cas:issn">2163-8306</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Library of Integrated Cellular Signatures (LINCS) project provides comprehensive transcriptome profiling of human cell lines before and after chem. and genetic perturbations.  Its L1000 platform utilizes 978 landmark genes to infer the transcript levels of 14,292 genes computationally.  Here we conducted the L1000 data quality control anal. by using MCF7, PC3, and A375 cell lines as representative examples.  Before perturbations, a promising 80% correlation in transcriptome was obsd. between L1000- and Affymetrix HU133A-platforms.  After library-based shRNA perturbations, a moderate 30% of differentially expressed genes overlapped between any two selected controls viral vectors using the L1000 platform.  The mitogen-activated protein kinase, vascular endothelial growth factor, and T-cell receptor pathways were identified as the most significantly shared pathways between chem. and genetic perturbations in cancer cells.  In conclusion, L1000 platform is reliable in assessing transcriptome before perturbation.  Its response to perturbagens needs to be interpreted with caution.  A quality control anal. pipeline of L1000 is recommended before addressing biol. questions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLk1tBMwFMlLVg90H21EOLACvtfcHk0lj65_Gfq_6vvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGgur7O&md5=bfb45386df427049788bec6c33e6b5ac</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fpsp4.12107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsp4.12107%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DSystematic%2520quality%2520control%2520analysis%2520of%2520LINCS%2520data%26jtitle%3DCPT%253A%2520Pharmacometrics%2520Syst.%2520Pharmacol.%26date%3D2016%26volume%3D5%26spage%3D588%26epage%3D598%26doi%3D10.1002%2Fpsp4.12107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Wolf, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bondt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlmann, H. W.</span></span> <span> </span><span class="NLM_article-title">Transcriptional characterization of compounds: lessons learned from the public LINCS data</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1089/adt.2016.715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1089%2Fadt.2016.715" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=252-260&author=H.+De+Wolfauthor=A.+De+Bondtauthor=H.+Turnerauthor=H.+W.+Gohlmann&title=Transcriptional+characterization+of+compounds%3A+lessons+learned+from+the+public+LINCS+data&doi=10.1089%2Fadt.2016.715"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1089%2Fadt.2016.715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2016.715%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BWolf%26aufirst%3DH.%26aulast%3DDe%2BBondt%26aufirst%3DA.%26aulast%3DTurner%26aufirst%3DH.%26aulast%3DGohlmann%26aufirst%3DH.%2BW.%26atitle%3DTranscriptional%2520characterization%2520of%2520compounds%253A%2520lessons%2520learned%2520from%2520the%2520public%2520LINCS%2520data%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2016%26volume%3D14%26spage%3D252%26epage%3D260%26doi%3D10.1089%2Fadt.2016.715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span> <span> </span><span class="NLM_article-title">Compound signature detection on LINCS L1000 big data</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">714</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1039/C4MB00677A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2FC4MB00677A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25609570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtFGisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=714-722&author=C.+Liuauthor=J.+Suauthor=F.+Yangauthor=K.+Weiauthor=J.+Maauthor=X.+Zhou&title=Compound+signature+detection+on+LINCS+L1000+big+data&doi=10.1039%2FC4MB00677A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Compound signature detection on LINCS L1000 big data</span></div><div class="casAuthors">Liu, Chenglin; Su, Jing; Yang, Fei; Wei, Kun; Ma, Jinwen; Zhou, Xiaobo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">714-722</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The Library of Integrated Network-based Cellular Signatures (LINCS) L1000 big data provide gene expression profiles induced by over 10 000 compds., shRNAs, and kinase inhibitors using the L1000 platform.  We developed csNMF, a systematic compd. signature discovery pipeline covering from raw L1000 data processing to drug screening and mechanism generation.  The csNMF pipeline demonstrated better performance than the original L1000 pipeline.  The discovered compd. signatures of breast cancer were consistent with the LINCS KINOMEscan data and were clin. relevant.  The csNMF pipeline provided a novel and complete tool to expedite signature-based drug discovery leveraging the LINCS L1000 resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK2ETNNHh4ULVg90H21EOLACvtfcHk0lhOoQuv5acl3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtFGisg%253D%253D&md5=cb02442649e0bf33ca0c7cfd6d0603df</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1039%2FC4MB00677A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MB00677A%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DX.%26atitle%3DCompound%2520signature%2520detection%2520on%2520LINCS%2520L1000%2520big%2520data%26jtitle%3DMol.%2520BioSyst.%26date%3D2015%26volume%3D11%26spage%3D714%26epage%3D722%26doi%3D10.1039%2FC4MB00677A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filzen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudor, M.</span></span> <span> </span><span class="NLM_article-title">Representing high throughput expression profiles via perturbation barcodes reveals compound targets</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e1005335</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1371%2Fjournal.pcbi.1005335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28182661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCqsbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=e1005335&author=T.+M.+Filzenauthor=P.+S.+Kutchukianauthor=J.+D.+Hermesauthor=J.+Liauthor=M.+Tudor&title=Representing+high+throughput+expression+profiles+via+perturbation+barcodes+reveals+compound+targets&doi=10.1371%2Fjournal.pcbi.1005335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Representing high throughput expression profiles via perturbation barcodes reveals compound targets</span></div><div class="casAuthors">Filzen, Tracey M.; Kutchukian, Peter S.; Hermes, Jeffrey D.; Li, Jing; Tudor, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1005335/1-e1005335/19</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">High throughput mRNA expression profiling can be used to characterize the response of cell culture models to perturbations such as pharmacol. modulators and genetic perturbations.  As profiling campaigns expand in scope, it is important to homogenize, summarize, and analyze the resulting data in a manner that captures significant biol. signals in spite of various noise sources such as batch effects and stochastic variation.  We used the L1000 platform for large-scale profiling of 978 representative genes across thousands of compd. treatments.  Here, a method is described that uses deep learning techniques to convert the expression changes of the landmark genes into a perturbation barcode that reveals important features of the underlying data, performing better than the raw data in revealing important biol. insights.  The barcode captures compd. structure and target information, and predicts a compd.'s high throughput screening promiscuity, to a higher degree than the original data measurements, indicating that the approach uncovers underlying factors of the expression data that are otherwise entangled or masked by noise.  Furthermore, we demonstrate that visualizations derived from the perturbation barcode can be used to more sensitively assign functions to unknown compds. through a guilt-by-assocn. approach, which we use to predict and exptl. validate the activity of compds. on the MAPK pathway.  The demonstrated application of deep metric learning to large-scale chem. genetics projects highlights the utility of this and related approaches to the extn. of insights and testable hypotheses from big, sometimes noisy data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgD8qURxAqhbVg90H21EOLACvtfcHk0lhOoQuv5acl3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCqsbvP&md5=5f2f46c468401ba7bced65ee0bbc3244</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005335%26sid%3Dliteratum%253Aachs%26aulast%3DFilzen%26aufirst%3DT.%2BM.%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DHermes%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DTudor%26aufirst%3DM.%26atitle%3DRepresenting%2520high%2520throughput%2520expression%2520profiles%2520via%2520perturbation%2520barcodes%2520reveals%2520compound%2520targets%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3De1005335%26doi%3D10.1371%2Fjournal.pcbi.1005335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Senkowski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarvius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengqvist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nygren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kultima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fryknas, M.</span></span> <span> </span><span class="NLM_article-title">Large-scale gene expression profiling platform for identification of context-dependent drug responses in multicellular tumor spheroids</span>. <i>Cell Chem Biol</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1428</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2016.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27984028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOqsL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1428-1438&author=W.+Senkowskiauthor=M.+Jarviusauthor=J.+Rubinauthor=J.+Lengqvistauthor=M.+G.+Gustafssonauthor=P.+Nygrenauthor=K.+Kultimaauthor=R.+Larssonauthor=M.+Fryknas&title=Large-scale+gene+expression+profiling+platform+for+identification+of+context-dependent+drug+responses+in+multicellular+tumor+spheroids&doi=10.1016%2Fj.chembiol.2016.09.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Large-Scale Gene Expression Profiling Platform for Identification of Context-Dependent Drug Responses in Multicellular Tumor Spheroids</span></div><div class="casAuthors">Senkowski, Wojciech; Jarvius, Malin; Rubin, Jenny; Lengqvist, Johan; Gustafsson, Mats G.; Nygren, Peter; Kultima, Kim; Larsson, Rolf; Fryknaes, Maarten</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1428-1438</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Cancer cell lines grown as two-dimensional (2D) cultures have been an essential model for studying cancer biol. and anticancer drug discovery.  However, 2D cancer cell cultures have major limitations, as they do not closely mimic the heterogeneity and tissue context of in vivo tumors.  Developing three-dimensional (3D) cell cultures, such as multicellular tumor spheroids, has the potential to address some of these limitations.  Here, we combined a high-throughput gene expression profiling method with a tumor spheroid-based drug-screening assay to identify context-dependent treatment responses.  As a proof of concept, we examd. drug responses of quiescent cancer cells to oxidative phosphorylation (OXPHOS) inhibitors.  Use of multicellular tumor spheroids led to discovery that the mevalonate pathway is upregulated in quiescent cells during OXPHOS inhibition, and that OXPHOS inhibitors and mevalonate pathway inhibitors were synergistically toxic to quiescent spheroids.  This work illustrates how 3D cellular models yield functional and mechanistic insights not accessible via 2D cultures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj9b2oo56yvrVg90H21EOLACvtfcHk0ljHtrriQgBeQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOqsL7N&md5=95f3efebeac31078d368dfef55b9faf7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DSenkowski%26aufirst%3DW.%26aulast%3DJarvius%26aufirst%3DM.%26aulast%3DRubin%26aufirst%3DJ.%26aulast%3DLengqvist%26aufirst%3DJ.%26aulast%3DGustafsson%26aufirst%3DM.%2BG.%26aulast%3DNygren%26aufirst%3DP.%26aulast%3DKultima%26aufirst%3DK.%26aulast%3DLarsson%26aufirst%3DR.%26aulast%3DFryknas%26aufirst%3DM.%26atitle%3DLarge-scale%2520gene%2520expression%2520profiling%2520platform%2520for%2520identification%2520of%2520context-dependent%2520drug%2520responses%2520in%2520multicellular%2520tumor%2520spheroids%26jtitle%3DCell%2520Chem%2520Biol%26date%3D2016%26volume%3D23%26spage%3D1428%26epage%3D1438%26doi%3D10.1016%2Fj.chembiol.2016.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Silva, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruccoleri, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayne, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penhallow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockett, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemers, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross-Macdonald, P.</span></span> <span> </span><span class="NLM_article-title">Transcriptional profiling of the dose response: a more powerful approach for characterizing drug activities</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e1000512</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1000512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1371%2Fjournal.pcbi.1000512" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000512&author=R.+R.+Jiauthor=H.+de+Silvaauthor=Y.+Jinauthor=R.+E.+Bruccoleriauthor=J.+Caoauthor=A.+Heauthor=W.+Huangauthor=P.+S.+Kayneauthor=I.+M.+Neuhausauthor=K.+H.+Ottauthor=B.+Penhallowauthor=M.+I.+Cockettauthor=M.+G.+Neubauerauthor=N.+O.+Siemersauthor=P.+Ross-Macdonald&title=Transcriptional+profiling+of+the+dose+response%3A+a+more+powerful+approach+for+characterizing+drug+activities&doi=10.1371%2Fjournal.pcbi.1000512"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000512%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DR.%2BR.%26aulast%3Dde%2BSilva%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DBruccoleri%26aufirst%3DR.%2BE.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DKayne%26aufirst%3DP.%2BS.%26aulast%3DNeuhaus%26aufirst%3DI.%2BM.%26aulast%3DOtt%26aufirst%3DK.%2BH.%26aulast%3DPenhallow%26aufirst%3DB.%26aulast%3DCockett%26aufirst%3DM.%2BI.%26aulast%3DNeubauer%26aufirst%3DM.%2BG.%26aulast%3DSiemers%26aufirst%3DN.%2BO.%26aulast%3DRoss-Macdonald%26aufirst%3DP.%26atitle%3DTranscriptional%2520profiling%2520of%2520the%2520dose%2520response%253A%2520a%2520more%2520powerful%2520approach%2520for%2520characterizing%2520drug%2520activities%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000512%26doi%3D10.1371%2Fjournal.pcbi.1000512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drawnel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benmansour, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araujo Del Rosario, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen Zoffmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delobel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prummer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weibel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iacone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Certa, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunotto, M.</span></span> <span> </span><span class="NLM_article-title">Molecular phenotyping combines molecular information, biological relevance, and patient data to Improve productivity of early drug discovery</span>. <i>Cell Chem Biol</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">624</span>– <span class="NLM_lpage">634</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2017.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28434878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsVWitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=624-634&author=F.+M.+Drawnelauthor=J.+D.+Zhangauthor=E.+Kungauthor=N.+Aoyamaauthor=F.+Benmansourauthor=A.+Araujo+Del+Rosarioauthor=S.+Jensen+Zoffmannauthor=F.+Delobelauthor=M.+Prummerauthor=F.+Weibelauthor=C.+Carlsonauthor=B.+Ansonauthor=R.+Iaconeauthor=U.+Certaauthor=T.+Singerauthor=M.+Ebelingauthor=M.+Prunotto&title=Molecular+phenotyping+combines+molecular+information%2C+biological+relevance%2C+and+patient+data+to+Improve+productivity+of+early+drug+discovery&doi=10.1016%2Fj.chembiol.2017.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Phenotyping Combines Molecular Information, Biological Relevance, and Patient Data to Improve Productivity of Early Drug Discovery</span></div><div class="casAuthors">Drawnel, Faye Marie; Zhang, Jitao David; Kung, Erich; Aoyama, Natsuyo; Benmansour, Fethallah; Araujo Del Rosario, Andrea; Jensen Zoffmann, Sannah; Delobel, Frederic; Prummer, Michael; Weibel, Franziska; Carlson, Coby; Anson, Blake; Iacone, Roberto; Certa, Ulrich; Singer, Thomas; Ebeling, Martin; Prunotto, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">624-634.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Today, novel therapeutics are identified in an environment which is intrinsically different from the clin. context in which they are ultimately evaluated.  Using mol. phenotyping and an in vitro model of diabetic cardiomyopathy, we show that by quantifying pathway reporter gene expression, mol. phenotyping can cluster compds. based on pathway profiles and dissect assocns. between pathway activities and disease phenotypes simultaneously.  Mol. phenotyping was applicable to compds. with a range of binding specificities and triaged false positives derived from high-content screening assays.  The technique identified a class of calcium-signaling modulators that can reverse disease-regulated pathways and phenotypes, which was validated by structurally distinct compds. of relevant classes.  Our results advocate for application of mol. phenotyping in early drug discovery, promoting biol. relevance as a key selection criterion early in the drug development cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph9aTnyxvKj7Vg90H21EOLACvtfcHk0ljHtrriQgBeQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsVWitbw%253D&md5=757558351b5a00dcc0f3f107c8f61d89</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DDrawnel%26aufirst%3DF.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%2BD.%26aulast%3DKung%26aufirst%3DE.%26aulast%3DAoyama%26aufirst%3DN.%26aulast%3DBenmansour%26aufirst%3DF.%26aulast%3DAraujo%2BDel%2BRosario%26aufirst%3DA.%26aulast%3DJensen%2BZoffmann%26aufirst%3DS.%26aulast%3DDelobel%26aufirst%3DF.%26aulast%3DPrummer%26aufirst%3DM.%26aulast%3DWeibel%26aufirst%3DF.%26aulast%3DCarlson%26aufirst%3DC.%26aulast%3DAnson%26aufirst%3DB.%26aulast%3DIacone%26aufirst%3DR.%26aulast%3DCerta%26aufirst%3DU.%26aulast%3DSinger%26aufirst%3DT.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DPrunotto%26aufirst%3DM.%26atitle%3DMolecular%2520phenotyping%2520combines%2520molecular%2520information%252C%2520biological%2520relevance%252C%2520and%2520patient%2520data%2520to%2520Improve%2520productivity%2520of%2520early%2520drug%2520discovery%26jtitle%3DCell%2520Chem%2520Biol%26date%3D2017%26volume%3D24%26spage%3D624%26epage%3D634%26doi%3D10.1016%2Fj.chembiol.2017.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunkel, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plavec, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melrose, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosler, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hytopoulos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bateman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butcher, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E. L.</span></span> <span> </span><span class="NLM_article-title">Rapid structure-activity and selectivity analysis of kinase inhibitors by BioMAP analysis in complex human primary cell-based models</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1089/adt.2004.2.431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1089%2Fadt.2004.2.431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=15357924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntlGksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=431-441&author=E.+J.+Kunkelauthor=I.+Plavecauthor=D.+Nguyenauthor=J.+Melroseauthor=E.+S.+Roslerauthor=L.+T.+Kaoauthor=Y.+Wangauthor=E.+Hytopoulosauthor=A.+C.+Bishopauthor=R.+Batemanauthor=K.+M.+Shokatauthor=E.+C.+Butcherauthor=E.+L.+Berg&title=Rapid+structure-activity+and+selectivity+analysis+of+kinase+inhibitors+by+BioMAP+analysis+in+complex+human+primary+cell-based+models&doi=10.1089%2Fadt.2004.2.431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Structure-Activity and Selectivity Analysis of Kinase Inhibitors by BioMAP Analysis in Complex Human Primary Cell-Based Models</span></div><div class="casAuthors">Kunkel, Eric J.; Plavec, Ivan; Nguyen, Dat; Melrose, Jennifer; Rosler, Elen S.; Kao, Leon T.; Wang, Yuker; Hytopoulos, Evangelos; Bishop, Anthony C.; Bateman, Raynard; Shokat, Kevan M.; Butcher, Eugene C.; Berg, Ellen L.</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">431-441</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Rapid, quant. methods for characterizing the biol. activities of kinase inhibitors in complex human cell systems could allow the biol. consequences of differential target selectivity to be monitored early in development, improving the selection of drug candidates.  We have previously shown that Biol. Multiplexed Activity Profiling (BioMAP) permits rapid characterization of drug function based on statistical anal. of protein expression data sets from complex primary human cell based models of disease biol.  Here, using four such model systems contg. primary human endothelial cells and peripheral blood mononuclear cells in which multiple signaling pathways relevant to inflammation and immune responses are simultaneously activated, we demonstrate that BioMAP anal. can detect and distinguish a wide range of inhibitors directed against different kinase targets.  Using a panel of p38 mitogen-activated protein kinase antagonists as a test set, we show further that related compds. can be distinguished by unique features of the biol. responses they induce in complex systems, and can be classified according to their induction of shared (on-target) and secondary activities.  Statistical comparisons of quant. BioMAP profiles and anal. of profile features allow correlation of induced biol. effects with chem. structure and mapping of biol. responses to chem. series or substituents on a common scaffold.  Integration of automated BioMAP anal. for prioritization of hits and for structure-activity relation studies may improve and accelerate the design and selection of optimal therapeutic candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtlA37tRw7P7Vg90H21EOLACvtfcHk0lhY38pj27xrew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntlGksLY%253D&md5=eae8485e89ea69af97d5e60738bee209</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1089%2Fadt.2004.2.431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2004.2.431%26sid%3Dliteratum%253Aachs%26aulast%3DKunkel%26aufirst%3DE.%2BJ.%26aulast%3DPlavec%26aufirst%3DI.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DMelrose%26aufirst%3DJ.%26aulast%3DRosler%26aufirst%3DE.%2BS.%26aulast%3DKao%26aufirst%3DL.%2BT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHytopoulos%26aufirst%3DE.%26aulast%3DBishop%26aufirst%3DA.%2BC.%26aulast%3DBateman%26aufirst%3DR.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DButcher%26aufirst%3DE.%2BC.%26aulast%3DBerg%26aufirst%3DE.%2BL.%26atitle%3DRapid%2520structure-activity%2520and%2520selectivity%2520analysis%2520of%2520kinase%2520inhibitors%2520by%2520BioMAP%2520analysis%2520in%2520complex%2520human%2520primary%2520cell-based%2520models%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2004%26volume%3D2%26spage%3D431%26epage%3D441%26doi%3D10.1089%2Fadt.2004.2.431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polokoff, M. A.</span></span> <span> </span><span class="NLM_article-title">Building predictive models for mechanism-of-action classification from phenotypic assay data sets</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1260</span>– <span class="NLM_lpage">1269</span>, <span class="refDoi"> DOI: 10.1177/1087057113505324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F1087057113505324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24088371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWjsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1260-1269&author=E.+L.+Bergauthor=J.+Yangauthor=M.+A.+Polokoff&title=Building+predictive+models+for+mechanism-of-action+classification+from+phenotypic+assay+data+sets&doi=10.1177%2F1087057113505324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Building predictive models for mechanism-of-action classification from phenotypic assay data sets</span></div><div class="casAuthors">Berg, Ellen L.; Yang, Jian; Polokoff, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1260-1269, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Compd. mechanism-of-action information can be crit. for drug development decisions but is often challenging for phenotypic drug discovery programs.  One concern is that compds. selected by phenotypic screening will have a previously known but undesirable target mechanism.  Here we describe a useful method for assigning mechanism class to compds. and bioactive agents using an 84-feature signature from a panel of primary human cell systems (BioMAP systems).  For this approach, a ref. data set of well-characterized compds. was used to develop predictive models for 28 mechanism classes using support vector machines.  These mechanism classes encompass safety and efficacy-related mechanisms, include both target-specific and pathway-based classes, and cover the most common mechanisms identified in phenotypic screens, such as inhibitors of mitochondrial and microtubule function, histone deacetylase, and cAMP elevators.  Here we describe the performance and the application of these predictive models in a decision scheme for triaging phenotypic screening hits using a previously published data set of 309 environmental chems. tested as part of the Environmental Protection Agency's ToxCast program.  By providing quantified membership in specific mechanism classes, this approach is suitable for identification of off-target toxicity mechanisms as well as enabling target deconvolution of phenotypic drug discovery hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSbGfCpuChA7Vg90H21EOLACvtfcHk0lhY38pj27xrew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWjsLvP&md5=1177a3a1dbae60621bbc2249f0e8e0ac</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1177%2F1087057113505324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113505324%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DE.%2BL.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DPolokoff%26aufirst%3DM.%2BA.%26atitle%3DBuilding%2520predictive%2520models%2520for%2520mechanism-of-action%2520classification%2520from%2520phenotypic%2520assay%2520data%2520sets%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D1260%26epage%3D1269%26doi%3D10.1177%2F1087057113505324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E. L.</span></span> <span> </span><span class="NLM_article-title">Phenotypic chemical biology for predicting safety and efficacy</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2017.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.ddtec.2017.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28647086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC1cjhsFaitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=53-60&author=E.+L.+Berg&title=Phenotypic+chemical+biology+for+predicting+safety+and+efficacy&doi=10.1016%2Fj.ddtec.2017.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic chemical biology for predicting safety and efficacy</span></div><div class="casAuthors">Berg Ellen L</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Phenotypic assays using in vitro cell cultures to forecast compound effects in people are transforming pharmaceutical research and contribute to alternative methods for chemical safety testing.  How these assays are validated for human disease relevance is a critical factor for developing more predictive assays.  Chemical biology, using drugs as well as target-selective chemical probes, is a direct and efficient approach for establishing disease relevance.  Chemical probes can connect information across assays and associate targets to clinical effects.  When applied at scale, phenotypic chemical biology advances our understanding of drug and toxicity mechanisms enabling construction of disease outcome pathways.  To improve the physiological relevance of phenotypic assays, standardized testing of a curated set of phenotypic pathway probes can provide a higher level of validation for phenotypic assay best practices.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRF0mGW8Hm3ecvLc4ARHEmzfW6udTcc2eZAIPj4Miaqsbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjhsFaitg%253D%253D&md5=eb67d89d2f5ba017fe9609141cee7959</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2017.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2017.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DE.%2BL.%26atitle%3DPhenotypic%2520chemical%2520biology%2520for%2520predicting%2520safety%2520and%2520efficacy%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2017%26volume%3D23%26spage%3D53%26epage%3D60%26doi%3D10.1016%2Fj.ddtec.2017.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E. L.</span></span>, personal communication.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Berg%2C+E.+L.%2C+personal+communication."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcommun%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DE.%2BL." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jinek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">East, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudna, J.</span></span> <span> </span><span class="NLM_article-title">RNA-programmed genome editing in human cells</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">e00471</span>, <span class="refDoi"> DOI: 10.7554/eLife.00471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.7554%2FeLife.00471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23386978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00471&author=M.+Jinekauthor=A.+Eastauthor=A.+Chengauthor=S.+Linauthor=E.+Maauthor=J.+Doudna&title=RNA-programmed+genome+editing+in+human+cells&doi=10.7554%2FeLife.00471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">RNA-programmed genome editing in human cells</span></div><div class="casAuthors">Jinek, Martin; East, Alexandra; Cheng, Aaron; Lin, Steven; Ma, Enbo; Doudna, Jennifer</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00471/1-e00471/9</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Type II CRISPR immune systems in bacteria use a dual RNA-guided DNA endonuclease, Cas9, to cleave foreign DNA at specific sites.  We show here that Cas9 assembles with hybrid guide RNAs in human cells and can induce the formation of double-strand DNA breaks (DSBs) at a site complementary to the guide RNA sequence in genomic DNA.  This cleavage activity requires both Cas9 and the complementary binding of the guide RNA.  Expts. using exts. from transfected cells show that RNA expression and/or assembly into Cas9 is the limiting factor for Cas9-mediated DNA cleavage.  In addn., we find that extension of the RNA sequence at the 3' end enhances DNA targeting activity in vivo.  These results show that RNA-programmed genome editing is a facile strategy for introducing site-specific genetic changes in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvfsdoVCMm_bVg90H21EOLACvtfcHk0lgeqvQjKBfU8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWjsw%253D%253D&md5=5d3a051b052b4301fd5bf935d12e5f6c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.7554%2FeLife.00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00471%26sid%3Dliteratum%253Aachs%26aulast%3DJinek%26aufirst%3DM.%26aulast%3DEast%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DE.%26aulast%3DDoudna%26aufirst%3DJ.%26atitle%3DRNA-programmed%2520genome%2520editing%2520in%2520human%2520cells%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00471%26doi%3D10.7554%2FeLife.00471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goralski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaskill, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capota, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ting, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nijhawan, D.</span></span> <span> </span><span class="NLM_article-title">Anticancer sulfonamides target splicing by inducing RBM39 degradation via recruitment to DCAF15</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>356</i></span>,  <span class="NLM_fpage">eaal3755</span>, <span class="refDoi"> DOI: 10.1126/science.aal3755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.aal3755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28302793" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2017&pages=eaal3755&author=T.+Hanauthor=M.+Goralskiauthor=N.+Gaskillauthor=E.+Capotaauthor=J.+Kimauthor=T.+C.+Tingauthor=Y.+Xieauthor=N.+S.+Williamsauthor=D.+Nijhawan&title=Anticancer+sulfonamides+target+splicing+by+inducing+RBM39+degradation+via+recruitment+to+DCAF15&doi=10.1126%2Fscience.aal3755"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1126%2Fscience.aal3755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aal3755%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DT.%26aulast%3DGoralski%26aufirst%3DM.%26aulast%3DGaskill%26aufirst%3DN.%26aulast%3DCapota%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DTing%26aufirst%3DT.%2BC.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DWilliams%26aufirst%3DN.%2BS.%26aulast%3DNijhawan%26aufirst%3DD.%26atitle%3DAnticancer%2520sulfonamides%2520target%2520splicing%2520by%2520inducing%2520RBM39%2520degradation%2520via%2520recruitment%2520to%2520DCAF15%26jtitle%3DScience%26date%3D2017%26volume%3D356%26spage%3Deaal3755%26doi%3D10.1126%2Fscience.aal3755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghtaling, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, T. M.</span></span> <span> </span><span class="NLM_article-title">Using transcriptome sequencing to identify mechanisms of drug action and resistance</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1038/nchembio.779</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.779" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=22327403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XitFalsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=235-237&author=S.+A.+Wackerauthor=B.+R.+Houghtalingauthor=O.+Elementoauthor=T.+M.+Kapoor&title=Using+transcriptome+sequencing+to+identify+mechanisms+of+drug+action+and+resistance&doi=10.1038%2Fnchembio.779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Using transcriptome sequencing to identify mechanisms of drug action and resistance</span></div><div class="casAuthors">Wacker, Sarah A.; Houghtaling, Benjamin R.; Elemento, Olivier; Kapoor, Tarun M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">235-237</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Detg. mechanisms of drug action in human cells remains a major challenge.  Here we describe an approach in which multiple-drug-resistant clones are isolated and transcriptome sequencing is used to find mutations in each clone.  Further anal. of mutations common to more than one clone can identify a drug's physiol. target and indirect resistance mechanisms, as indicated by our proof-of-concept studies of the cytotoxic anticancer drugs BI 2536 and bortezomib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTYM1eLSgI47Vg90H21EOLACvtfcHk0lgeqvQjKBfU8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitFalsr0%253D&md5=d4a28e70029356c94f65c4895158b96d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.779%26sid%3Dliteratum%253Aachs%26aulast%3DWacker%26aufirst%3DS.%2BA.%26aulast%3DHoughtaling%26aufirst%3DB.%2BR.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DKapoor%26aufirst%3DT.%2BM.%26atitle%3DUsing%2520transcriptome%2520sequencing%2520to%2520identify%2520mechanisms%2520of%2520drug%2520action%2520and%2520resistance%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D235%26epage%3D237%26doi%3D10.1038%2Fnchembio.779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smurnyy, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWhinnie, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span> <span> </span><span class="NLM_article-title">DNA sequencing and CRISPR-Cas9 gene editing for target validation in mammalian cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">625</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.1550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24929529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=623-625&author=Y.+Smurnyyauthor=M.+Caiauthor=H.+Wuauthor=E.+McWhinnieauthor=J.+A.+Tallaricoauthor=Y.+Yangauthor=Y.+Feng&title=DNA+sequencing+and+CRISPR-Cas9+gene+editing+for+target+validation+in+mammalian+cells&doi=10.1038%2Fnchembio.1550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">DNA sequencing and CRISPR-Cas9 gene editing for target validation in mammalian cells</span></div><div class="casAuthors">Smurnyy, Yegor; Cai, Mi; Wu, Hua; McWhinnie, Elizabeth; Tallarico, John A.; Yang, Yi; Feng, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">623-625</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Identification and validation of drug-resistant mutations can provide important insights into the mechanism of action of a compd.  Here we demonstrate the feasibility of such an approach in mammalian cells using next-generation sequencing of drug-resistant clones and CRISPR-Cas9-mediated gene editing on two drug-target pairs, 6-thioguanine-HPRT1 and triptolide-ERCC3.  We showed that disrupting functional HPRT1 allele or introducing ERCC3 point mutations by gene editing can confer drug resistance in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDft4QWmx6YbVg90H21EOLACvtfcHk0lhDmgZKs_tcXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslGisb4%253D&md5=b05f570ebd84f2edb49c759214272abc</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1550%26sid%3Dliteratum%253Aachs%26aulast%3DSmurnyy%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMcWhinnie%26aufirst%3DE.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DY.%26atitle%3DDNA%2520sequencing%2520and%2520CRISPR-Cas9%2520gene%2520editing%2520for%2520target%2520validation%2520in%2520mammalian%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D623%26epage%3D625%26doi%3D10.1038%2Fnchembio.1550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLauchlan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">MELK is not necessary for the proliferation of basal-like breast cancer cells</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e22693</span>, <span class="refDoi"> DOI: 10.7554/eLife.26693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.7554%2FeLife.26693" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=e22693&author=H.+T.+Huangauthor=H.+S.+Seoauthor=T.+Zhangauthor=Y.+Wangauthor=B.+Jiangauthor=Q.+Liauthor=D.+L.+Buckleyauthor=B.+Nabetauthor=J.+M.+Robertsauthor=J.+Paulkauthor=S.+Dastjerdiauthor=G.+E.+Winterauthor=H.+McLauchlanauthor=J.+Moranauthor=J.+E.+Bradnerauthor=M.+J.+Eckauthor=S.+Dhe-Paganonauthor=J.+J.+Zhaoauthor=N.+S.+Gray&title=MELK+is+not+necessary+for+the+proliferation+of+basal-like+breast+cancer+cells&doi=10.7554%2FeLife.26693"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.7554%2FeLife.26693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.26693%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BT.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DMcLauchlan%26aufirst%3DH.%26aulast%3DMoran%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DMELK%2520is%2520not%2520necessary%2520for%2520the%2520proliferation%2520of%2520basal-like%2520breast%2520cancer%2520cells%26jtitle%3DeLife%26date%3D2017%26volume%3D6%26spage%3De22693%26doi%3D10.7554%2FeLife.26693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span> <span> </span><span class="NLM_article-title">Genetic screens in human cells using the CRISPR-Cas9 system</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1126/science.1246981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1246981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24336569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkslOq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=80-84&author=T.+Wangauthor=J.+J.+Weiauthor=D.+M.+Sabatiniauthor=E.+S.+Lander&title=Genetic+screens+in+human+cells+using+the+CRISPR-Cas9+system&doi=10.1126%2Fscience.1246981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic screens in human cells using the CRISPR-Cas9 system</span></div><div class="casAuthors">Wang, Tim; Wei, Jenny J.; Sabatini, David M.; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6166</span>),
    <span class="NLM_cas:pages">80-84</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The bacterial clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9 system for genome editing has greatly expanded the toolbox for mammalian genetics, enabling the rapid generation of isogenic cell lines and mice with modified alleles.  Here, we describe a pooled, loss-of-function genetic screening approach suitable for both pos. and neg. selection that uses a genome-scale lentiviral single-guide RNA (sgRNA) library.  SgRNA expression cassettes were stably integrated into the genome, which enabled a complex mutant pool to be tracked by massively parallel sequencing.  We used a library contg. 73,000 sgRNAs to generate knockout collections and performed screens in two human cell lines.  A screen for resistance to the nucleotide analog 6-thioguanine identified all expected members of the DNA mismatch repair pathway, whereas another for the DNA topoisomerase II (TOP2A) poison etoposide identified TOP2A, as expected, and also cyclin-dependent kinase 6, CDK6.  A neg. selection screen for essential genes identified numerous gene sets corresponding to fundamental processes.  Last, we show that sgRNA efficiency is assocd. with specific sequence motifs, enabling the prediction of more effective sgRNAs.  Collectively, these results establish Cas9/sgRNA screens as a powerful tool for systematic genetic anal. in mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKrKxZ3oDKMrVg90H21EOLACvtfcHk0lhDmgZKs_tcXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkslOq&md5=d69a508793812f82efd087d8ee1ed6f6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.1246981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1246981%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DJ.%2BJ.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DGenetic%2520screens%2520in%2520human%2520cells%2520using%2520the%2520CRISPR-Cas9%2520system%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D80%26epage%3D84%26doi%3D10.1126%2Fscience.1246981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jost, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlbeck, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krenning, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menchon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prota, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhmanova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanenbaum, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Combined CRISPRi/a-based chemical genetic screens reveal that Rigosertib Is a microtubule-destabilizing agent</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.09.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.molcel.2017.09.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28985505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Wqs77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2017&pages=210-223&author=M.+Jostauthor=Y.+Chenauthor=L.+A.+Gilbertauthor=M.+A.+Horlbeckauthor=L.+Krenningauthor=G.+Menchonauthor=A.+Raiauthor=M.+Y.+Choauthor=J.+J.+Sternauthor=A.+E.+Protaauthor=M.+Kampmannauthor=A.+Akhmanovaauthor=M.+O.+Steinmetzauthor=M.+E.+Tanenbaumauthor=J.+S.+Weissman&title=Combined+CRISPRi%2Fa-based+chemical+genetic+screens+reveal+that+Rigosertib+Is+a+microtubule-destabilizing+agent&doi=10.1016%2Fj.molcel.2017.09.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Combined CRISPRi/a-Based Chemical Genetic Screens Reveal that Rigosertib Is a Microtubule-Destabilizing Agent</span></div><div class="casAuthors">Jost, Marco; Chen, Yuwen; Gilbert, Luke A.; Horlbeck, Max A.; Krenning, Lenno; Menchon, Gregory; Rai, Ankit; Cho, Min Y.; Stern, Jacob J.; Prota, Andrea E.; Kampmann, Martin; Akhmanova, Anna; Steinmetz, Michel O.; Tanenbaum, Marvin E.; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-223.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Chem. libraries paired with phenotypic screens can now readily identify compds. with therapeutic potential.  A central limitation to exploiting these compds., however, has been in identifying their relevant cellular targets.  Here, we present a two-tiered CRISPR-mediated chem.-genetic strategy for target identification: combined genome-wide knockdown and overexpression screening as well as focused, comparative chem.-genetic profiling.  Application of these strategies to rigosertib, a drug in phase 3 clin. trials for high-risk myelodysplastic syndrome whose mol. target had remained controversial, pointed singularly to microtubules as rigosertib's target.  We showed that rigosertib indeed directly binds to and destabilizes microtubules using cell biol., in vitro, and structural approaches.  Finally, expression of tubulin with a structure-guided mutation in the rigosertib-binding pocket conferred resistance to rigosertib, establishing that rigosertib kills cancer cells by destabilizing microtubules.  These results demonstrate the power of our chem.-genetic screening strategies for pinpointing the physiol. relevant targets of chem. agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBsbZCZePrwLVg90H21EOLACvtfcHk0liM84S2W2p7Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Wqs77L&md5=115f89a6cc9eef8c3709c06b74109c51</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.09.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.09.012%26sid%3Dliteratum%253Aachs%26aulast%3DJost%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26aulast%3DHorlbeck%26aufirst%3DM.%2BA.%26aulast%3DKrenning%26aufirst%3DL.%26aulast%3DMenchon%26aufirst%3DG.%26aulast%3DRai%26aufirst%3DA.%26aulast%3DCho%26aufirst%3DM.%2BY.%26aulast%3DStern%26aufirst%3DJ.%2BJ.%26aulast%3DProta%26aufirst%3DA.%2BE.%26aulast%3DKampmann%26aufirst%3DM.%26aulast%3DAkhmanova%26aufirst%3DA.%26aulast%3DSteinmetz%26aufirst%3DM.%2BO.%26aulast%3DTanenbaum%26aufirst%3DM.%2BE.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DCombined%2520CRISPRi%252Fa-based%2520chemical%2520genetic%2520screens%2520reveal%2520that%2520Rigosertib%2520Is%2520a%2520microtubule-destabilizing%2520agent%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D68%26spage%3D210%26epage%3D223%26doi%3D10.1016%2Fj.molcel.2017.09.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deans, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgens, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okesli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlbeck, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glenn, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carette, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khosla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span> <span> </span><span class="NLM_article-title">Parallel shRNA and CRISPR-Cas9 screens enable antiviral drug target identification</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">366</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.2050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27018887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XksFCru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=361-366&author=R.+M.+Deansauthor=D.+W.+Morgensauthor=A.+Okesliauthor=S.+Pillayauthor=M.+A.+Horlbeckauthor=M.+Kampmannauthor=L.+A.+Gilbertauthor=A.+Liauthor=R.+Mateoauthor=M.+Smithauthor=J.+S.+Glennauthor=J.+E.+Caretteauthor=C.+Khoslaauthor=M.+C.+Bassik&title=Parallel+shRNA+and+CRISPR-Cas9+screens+enable+antiviral+drug+target+identification&doi=10.1038%2Fnchembio.2050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel shRNA and CRISPR-Cas9 screens enable antiviral drug target identification</span></div><div class="casAuthors">Deans, Richard M.; Morgens, David W.; Okesli, Ayse; Pillay, Sirika; Horlbeck, Max A.; Kampmann, Martin; Gilbert, Luke A.; Li, Amy; Mateo, Roberto; Smith, Mark; Glenn, Jeffrey S.; Carette, Jan E.; Khosla, Chaitan; Bassik, Michael C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">361-366</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Broad-spectrum antiviral drugs targeting host processes could potentially treat a wide range of viruses while reducing the likelihood of emergent resistance.  Despite great promise as therapeutics, such drugs remain largely elusive.  Here the authors used parallel genome-wide high-coverage short hairpin RNA (shRNA) and clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9 screens to identify the cellular target and mechanism of action of GSK983, a potent broad-spectrum antiviral with unexplained cytotoxicity.  The authors found that GSK983 blocked cell proliferation and dengue virus replication by inhibiting the pyrimidine biosynthesis enzyme dihydroorotate dehydrogenase (DHODH).  Guided by mechanistic insights from both genomic screens, the authors found that exogenous deoxycytidine markedly reduced GSK983 cytotoxicity but not antiviral activity, providing an attractive new approach to improve the therapeutic window of DHODH inhibitors against RNA viruses.  The authors' results highlight the distinct advantages and limitations of each screening method for identifying drug targets, and demonstrate the utility of parallel knockdown and knockout screens for comprehensive probing of drug activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1-QBjLboncbVg90H21EOLACvtfcHk0liM84S2W2p7Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksFCru7w%253D&md5=f1c93508518375e686dd87fd38c8e994</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2050%26sid%3Dliteratum%253Aachs%26aulast%3DDeans%26aufirst%3DR.%2BM.%26aulast%3DMorgens%26aufirst%3DD.%2BW.%26aulast%3DOkesli%26aufirst%3DA.%26aulast%3DPillay%26aufirst%3DS.%26aulast%3DHorlbeck%26aufirst%3DM.%2BA.%26aulast%3DKampmann%26aufirst%3DM.%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DMateo%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DM.%26aulast%3DGlenn%26aufirst%3DJ.%2BS.%26aulast%3DCarette%26aufirst%3DJ.%2BE.%26aulast%3DKhosla%26aufirst%3DC.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26atitle%3DParallel%2520shRNA%2520and%2520CRISPR-Cas9%2520screens%2520enable%2520antiviral%2520drug%2520target%2520identification%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D361%26epage%3D366%26doi%3D10.1038%2Fnchembio.2050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparbier, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beavis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolzenburg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgens, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neeson, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darcy, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voskoboinik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cebon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehner, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">CMTM6 maintains the expression of PD-L1 and regulates anti-tumour immunity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>549</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1038/nature23643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnature23643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28813417" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGktrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=549&publication_year=2017&pages=101-105&author=M.+L.+Burrauthor=C.+E.+Sparbierauthor=Y.+C.+Chanauthor=J.+C.+Williamsonauthor=K.+Woodsauthor=P.+A.+Beavisauthor=E.+Y.+N.+Lamauthor=M.+A.+Hendersonauthor=C.+C.+Bellauthor=S.+Stolzenburgauthor=O.+Gilanauthor=S.+Bloorauthor=T.+Nooriauthor=D.+W.+Morgensauthor=M.+C.+Bassikauthor=P.+J.+Neesonauthor=A.+Behrenauthor=P.+K.+Darcyauthor=S.+J.+Dawsonauthor=I.+Voskoboinikauthor=J.+A.+Trapaniauthor=J.+Cebonauthor=P.+J.+Lehnerauthor=M.+A.+Dawson&title=CMTM6+maintains+the+expression+of+PD-L1+and+regulates+anti-tumour+immunity&doi=10.1038%2Fnature23643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">CMTM6 maintains the expression of PD-L1 and regulates anti-tumour immunity</span></div><div class="casAuthors">Burr, Marian L.; Sparbier, Christina E.; Chan, Yih-Chih; Williamson, James C.; Woods, Katherine; Beavis, Paul A.; Lam, Enid Y. N.; Henderson, Melissa A.; Bell, Charles C.; Stolzenburg, Sabine; Gilan, Omer; Bloor, Stuart; Noori, Tahereh; Morgens, David W.; Bassik, Michael C.; Neeson, Paul J.; Behren, Andreas; Darcy, Phillip K.; Dawson, Sarah-Jane; Voskoboinik, Ilia; Trapani, Joseph A.; Cebon, Jonathan; Lehner, Paul J.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">549</span>
        (<span class="NLM_cas:issue">7670</span>),
    <span class="NLM_cas:pages">101-105</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells exploit the expression of the programmed death-1 (PD-1) ligand 1 (PD-L1) to subvert T-cell-mediated immunosurveillance.  The success of therapies that disrupt PD-L1-mediated tumor tolerance has highlighted the need to understand the mol. regulation of PD-L1 expression.  Here we identify the uncharacterized protein CMTM6 as a crit. regulator of PD-L1 in a broad range of cancer cells, by using a genome-wide CRISPR-Cas9 screen.  CMTM6 is a ubiquitously expressed protein that binds PD-L1 and maintains its cell surface expression.  CMTM6 is not required for PD-L1 maturation but co-localizes with PD-L1 at the plasma membrane and in recycling endosomes, where it prevents PD-L1 from being targeted for lysosome-mediated degrdn.  Using a quant. approach to profile the entire plasma membrane proteome, we find that CMTM6 displays specificity for PD-L1.  Notably, CMTM6 depletion decreases PD-L1 without compromising cell surface expression of MHC class I.  CMTM6 depletion, via the redn. of PD-L1, significantly alleviates the suppression of tumor-specific T cell activity in vitro and in vivo.  These findings provide insights into the biol. of PD-L1 regulation, identify a previously unrecognized master regulator of this crit. immune checkpoint and highlight a potential therapeutic target to overcome immune evasion by tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOaE7qlrP04LVg90H21EOLACvtfcHk0lgmMF5nlarQFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGktrvK&md5=c37c58dcfa75b3057fe4e9635789281d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature23643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature23643%26sid%3Dliteratum%253Aachs%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DSparbier%26aufirst%3DC.%2BE.%26aulast%3DChan%26aufirst%3DY.%2BC.%26aulast%3DWilliamson%26aufirst%3DJ.%2BC.%26aulast%3DWoods%26aufirst%3DK.%26aulast%3DBeavis%26aufirst%3DP.%2BA.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DHenderson%26aufirst%3DM.%2BA.%26aulast%3DBell%26aufirst%3DC.%2BC.%26aulast%3DStolzenburg%26aufirst%3DS.%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DBloor%26aufirst%3DS.%26aulast%3DNoori%26aufirst%3DT.%26aulast%3DMorgens%26aufirst%3DD.%2BW.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26aulast%3DNeeson%26aufirst%3DP.%2BJ.%26aulast%3DBehren%26aufirst%3DA.%26aulast%3DDarcy%26aufirst%3DP.%2BK.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DVoskoboinik%26aufirst%3DI.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26aulast%3DCebon%26aufirst%3DJ.%26aulast%3DLehner%26aufirst%3DP.%2BJ.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DCMTM6%2520maintains%2520the%2520expression%2520of%2520PD-L1%2520and%2520regulates%2520anti-tumour%2520immunity%26jtitle%3DNature%26date%3D2017%26volume%3D549%26spage%3D101%26epage%3D105%26doi%3D10.1038%2Fnature23643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koundakjian, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hultquist, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamothe-Molina, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krupzcak, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Beltran, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piechocka-Trocha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hacohen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, B. D.</span></span> <span> </span><span class="NLM_article-title">A genome-wide CRISPR screen identifies a restricted set of HIV host dependency factors</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">203</span>, <span class="refDoi"> DOI: 10.1038/ng.3741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fng.3741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27992415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWnsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=193-203&author=R.+J.+Parkauthor=T.+Wangauthor=D.+Koundakjianauthor=J.+F.+Hultquistauthor=P.+Lamothe-Molinaauthor=B.+Monelauthor=K.+Schumannauthor=H.+Yuauthor=K.+M.+Krupzcakauthor=W.+Garcia-Beltranauthor=A.+Piechocka-Trochaauthor=N.+J.+Kroganauthor=A.+Marsonauthor=D.+M.+Sabatiniauthor=E.+S.+Landerauthor=N.+Hacohenauthor=B.+D.+Walker&title=A+genome-wide+CRISPR+screen+identifies+a+restricted+set+of+HIV+host+dependency+factors&doi=10.1038%2Fng.3741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A genome-wide CRISPR screen identifies a restricted set of HIV host dependency factors</span></div><div class="casAuthors">Park, Ryan J.; Wang, Tim; Koundakjian, Dylan; Hultquist, Judd F.; Lamothe-Molina, Pedro; Monel, Blandine; Schumann, Kathrin; Yu, Haiyan; Krupzcak, Kevin M.; Garcia-Beltran, Wilfredo; Piechocka-Trocha, Alicja; Krogan, Nevan J.; Marson, Alexander; Sabatini, David M.; Lander, Eric S.; Hacohen, Nir; Walker, Bruce D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">193-203</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Host proteins are essential for HIV entry and replication and can be important nonviral therapeutic targets.  Large-scale RNA interference (RNAi)-based screens have identified nearly a thousand candidate host factors, but there is little agreement among studies and few factors have been validated.  Here we demonstrate that a genome-wide CRISPR-based screen identifies host factors in a physiol. relevant cell system.  We identify five factors, including the HIV co-receptors CD4 and CCR5, that are required for HIV infection yet are dispensable for cellular proliferation and viability.  Tyrosylprotein sulfotransferase 2 (TPST2) and solute carrier family 35 member B2 (SLC35B2) function in a common pathway to sulfate CCR5 on extracellular tyrosine residues, facilitating CCR5 recognition by the HIV envelope.  Activated leukocyte cell adhesion mol. (ALCAM) mediates cell aggregation, which is required for cell-to-cell HIV transmission.  We validated these pathways in primary human CD4+ T cells through Cas9-mediated knockout and antibody blockade.  Our findings indicate that HIV infection and replication rely on a limited set of host-dispensable genes and suggest that these pathways can be studied for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK_vf-hKNi6rVg90H21EOLACvtfcHk0lgmMF5nlarQFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWnsbbK&md5=42dac62c174b3f60e895d65d1f0273fd</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fng.3741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3741%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DKoundakjian%26aufirst%3DD.%26aulast%3DHultquist%26aufirst%3DJ.%2BF.%26aulast%3DLamothe-Molina%26aufirst%3DP.%26aulast%3DMonel%26aufirst%3DB.%26aulast%3DSchumann%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DKrupzcak%26aufirst%3DK.%2BM.%26aulast%3DGarcia-Beltran%26aufirst%3DW.%26aulast%3DPiechocka-Trocha%26aufirst%3DA.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26aulast%3DMarson%26aufirst%3DA.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DHacohen%26aufirst%3DN.%26aulast%3DWalker%26aufirst%3DB.%2BD.%26atitle%3DA%2520genome-wide%2520CRISPR%2520screen%2520identifies%2520a%2520restricted%2520set%2520of%2520HIV%2520host%2520dependency%2520factors%26jtitle%3DNat.%2520Genet.%26date%3D2017%26volume%3D49%26spage%3D193%26epage%3D203%26doi%3D10.1038%2Fng.3741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlbeck, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalta, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panning, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploegh, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Genome-scale CRISPR-mediated control of gene repression and activation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">661</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.cell.2014.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25307932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslelsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=647-661&author=L.%0AA.+Gilbertauthor=M.+A.+Horlbeckauthor=B.+Adamsonauthor=J.+E.+Villaltaauthor=Y.+Chenauthor=E.+H.+Whiteheadauthor=C.+Guimaraesauthor=B.+Panningauthor=H.+L.+Ploeghauthor=M.+C.+Bassikauthor=L.+S.+Qiauthor=M.+Kampmannauthor=J.+S.+Weissman&title=Genome-scale+CRISPR-mediated+control+of+gene+repression+and+activation&doi=10.1016%2Fj.cell.2014.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-scale CRISPR-mediated control of gene repression and activation</span></div><div class="casAuthors">Gilbert, Luke A.; Horlbeck, Max A.; Adamson, Britt; Villalta, Jacqueline E.; Chen, Yuwen; Whitehead, Evan H.; Guimaraes, Carla; Panning, Barbara; Ploegh, Hidde L.; Bassik, Michael C.; Qi, Lei S.; Kampmann, Martin; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">647-661</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">While the catalog of mammalian transcripts and their expression levels in different cell types and disease states is rapidly expanding, our understanding of transcript function lags behind.  We present a robust technol. enabling systematic investigation of the cellular consequences of repressing or inducing individual transcripts.  We identify rules for specific targeting of transcriptional repressors (CRISPRi), typically achieving 90%-99% knockdown with minimal off-target effects, and activators (CRISPRa) to endogenous genes via endonuclease-deficient Cas9.  Together they enable modulation of gene expression over a ∼1000-fold range.  Using these rules, we construct genome-scale CRISPRi and CRISPRa libraries, each of which we validate with two pooled screens.  Growth-based screens identify essential genes, tumor suppressors, and regulators of differentiation.  Screens for sensitivity to a cholera-diphtheria toxin provide broad insights into the mechanisms of pathogen entry, retrotranslocation and toxicity.  Our results establish CRISPRi and CRISPRa as powerful tools that provide rich and complementary information for mapping complex pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWZTeLYH9msrVg90H21EOLACvtfcHk0lgmMF5nlarQFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslelsrfJ&md5=3d7df984d13eb14b6f1b80f3b2e93b8d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26aulast%3DHorlbeck%26aufirst%3DM.%2BA.%26aulast%3DAdamson%26aufirst%3DB.%26aulast%3DVillalta%26aufirst%3DJ.%2BE.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWhitehead%26aufirst%3DE.%2BH.%26aulast%3DGuimaraes%26aufirst%3DC.%26aulast%3DPanning%26aufirst%3DB.%26aulast%3DPloegh%26aufirst%3DH.%2BL.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26aulast%3DQi%26aufirst%3DL.%2BS.%26aulast%3DKampmann%26aufirst%3DM.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DGenome-scale%2520CRISPR-mediated%2520control%2520of%2520gene%2520repression%2520and%2520activation%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D647%26epage%3D661%26doi%3D10.1016%2Fj.cell.2014.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thakore, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Ippolito, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabadi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gersbach, C. A.</span></span> <span> </span><span class="NLM_article-title">Highly specific epigenome editing by CRISPR-Cas9 repressors for silencing of distal regulatory elements</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1143</span>– <span class="NLM_lpage">1149</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnmeth.3630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26501517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslant7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1143-1149&author=P.+I.+Thakoreauthor=A.+M.+D%E2%80%99Ippolitoauthor=L.+Songauthor=A.+Safiauthor=N.+K.+Shivakumarauthor=A.+M.+Kabadiauthor=T.+E.+Reddyauthor=G.+E.+Crawfordauthor=C.+A.+Gersbach&title=Highly+specific+epigenome+editing+by+CRISPR-Cas9+repressors+for+silencing+of+distal+regulatory+elements&doi=10.1038%2Fnmeth.3630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Highly specific epigenome editing by CRISPR-Cas9 repressors for silencing of distal regulatory elements</span></div><div class="casAuthors">Thakore, Pratiksha I.; D'Ippolito, Anthony M.; Song, Lingyun; Safi, Alexias; Shivakumar, Nishkala K.; Kabadi, Ami M.; Reddy, Timothy E.; Crawford, Gregory E.; Gersbach, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1143-1149</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenome editing with the CRISPR (clustered, regularly interspaced, short palindromic repeats)-Cas9 platform is a promising technol. for modulating gene expression to direct cell phenotype and to dissect the causal epigenetic mechanisms of gene regulation.  Fusions of nuclease-inactive dCas9 to the Kruppel-assocd. box (KRAB) repressor (dCas9-KRAB) can silence target gene expression, but the genome-wide specificity and the extent of heterochromatin formation catalyzed by dCas9-KRAB are not known.  We targeted dCas9-KRAB to the HS2 enhancer, a distal regulatory element that orchestrates the expression of multiple globin genes, and obsd. highly specific induction of H3K9 trimethylation (H3K9me3) at the enhancer and decreased chromatin accessibility of both the enhancer and its promoter targets.  Targeted epigenetic modification of HS2 silenced the expression of multiple globin genes, with minimal off-target changes in global gene expression.  These results demonstrate that repression mediated by dCas9-KRAB is sufficiently specific to disrupt the activity of individual enhancers via local modification of the epigenome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1C8tZjvdyCbVg90H21EOLACvtfcHk0ljThr0_Z1d1SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslant7fO&md5=aca39b98762607cafc8f9568b2b8200e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3630%26sid%3Dliteratum%253Aachs%26aulast%3DThakore%26aufirst%3DP.%2BI.%26aulast%3DD%25E2%2580%2599Ippolito%26aufirst%3DA.%2BM.%26aulast%3DSong%26aufirst%3DL.%26aulast%3DSafi%26aufirst%3DA.%26aulast%3DShivakumar%26aufirst%3DN.%2BK.%26aulast%3DKabadi%26aufirst%3DA.%2BM.%26aulast%3DReddy%26aufirst%3DT.%2BE.%26aulast%3DCrawford%26aufirst%3DG.%2BE.%26aulast%3DGersbach%26aufirst%3DC.%2BA.%26atitle%3DHighly%2520specific%2520epigenome%2520editing%2520by%2520CRISPR-Cas9%2520repressors%2520for%2520silencing%2520of%2520distal%2520regulatory%2520elements%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1143%26epage%3D1149%26doi%3D10.1038%2Fnmeth.3630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grasso, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishihara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, P. S.</span></span> <span> </span><span class="NLM_article-title">Bioconjugation by native chemical tagging of C-H bonds</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">12994</span>– <span class="NLM_lpage">12997</span>, <span class="refDoi"> DOI: 10.1021/ja407739y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja407739y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjur3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=12994-12997&author=Q.+Zhouauthor=J.+Guiauthor=C.-M.+Panauthor=E.+Alboneauthor=X.+Chengauthor=E.+M.+Suhauthor=L.+Grassoauthor=Y.+Ishiharaauthor=P.+S.+Baran&title=Bioconjugation+by+native+chemical+tagging+of+C-H+bonds&doi=10.1021%2Fja407739y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Bioconjugation by Native Chemical Tagging of C-H Bonds</span></div><div class="casAuthors">Zhou, Qianghui; Gui, Jinghan; Pan, Chung-Mao; Albone, Earl; Cheng, Xin; Suh, Edward M.; Grasso, Luigi; Ishihara, Yoshihiro; Baran, Phil S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12994-12997</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A general C-H functionalization method for the tagging of natural products and pharmaceuticals is described.  An azide-contg. sulfinate reagent allows the appendage of azidoalkyl chains onto heteroaroms., the product of which can then be attached to a monoclonal antibody by a "click" reaction.  This strategy expands the breadth of bioactive small mols. that can be linked to macromols. in a manner that is beyond the scope of existing methods in bioconjugation to permit tagging of the "seemingly untaggable".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVwraFgGUte7Vg90H21EOLACvtfcHk0ljThr0_Z1d1SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjur3P&md5=dca0179112d7d412fa38073c986a0730</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja407739y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja407739y%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DGui%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DC.-M.%26aulast%3DAlbone%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DSuh%26aufirst%3DE.%2BM.%26aulast%3DGrasso%26aufirst%3DL.%26aulast%3DIshihara%26aufirst%3DY.%26aulast%3DBaran%26aufirst%3DP.%2BS.%26atitle%3DBioconjugation%2520by%2520native%2520chemical%2520tagging%2520of%2520C-H%2520bonds%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D12994%26epage%3D12997%26doi%3D10.1021%2Fja407739y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanoh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simizu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muroi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osada, H.</span></span> <span> </span><span class="NLM_article-title">Photo-cross-linked small-molecule affinity matrix for facilitating forward and reverse chemical genetics</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">3559</span>– <span class="NLM_lpage">3562</span>, <span class="refDoi"> DOI: 10.1002/anie.200462370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.200462370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsVyktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=3559-3562&author=N.+Kanohauthor=K.+Hondaauthor=S.+Simizuauthor=M.+Muroiauthor=H.+Osada&title=Photo-cross-linked+small-molecule+affinity+matrix+for+facilitating+forward+and+reverse+chemical+genetics&doi=10.1002%2Fanie.200462370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Photo-cross-linked small-molecule affinity matrix for facilitating forward and reverse chemical genetics</span></div><div class="casAuthors">Kanoh, Naoki; Honda, Kaori; Simizu, Siro; Muroi, Makoto; Osada, Hiroyuki</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3559-3562</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A photoaffinity reaction has been used to photo-cross-link small mols. on agarose beads to identify and purify the binding proteins.  The authors prepd. photoaffinity-linker-coated (PALC) agarose/Sepharose beads and demonstrated that the photo-cross-linked small-mol. agarose beads can be used to identify and purify the binding protein.  Complex small mols. such as FK506 and cyclosporin A could be introduced on the affinity matrix by using a uniform and straightforward procedure.  Although the present procedure can not be applied to small mols. that degrade upon UV irradn., technologies aimed at immobilizing ligands on an affinity matrix in a functional-group-independent manner are expected to facilitate the identification of mol. targets of small mols. in forward chem. genetics, and is also considered potentially useful for the confirmation of specificity between small mols. and proteins of interest in the field of reverse chem. genetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8ONPNgv4zlbVg90H21EOLACvtfcHk0ljs5CmYgxaOjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsVyktrY%253D&md5=82d1f6a2425c50dfd4cdd92bd9ed9fb5</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462370%26sid%3Dliteratum%253Aachs%26aulast%3DKanoh%26aufirst%3DN.%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DSimizu%26aufirst%3DS.%26aulast%3DMuroi%26aufirst%3DM.%26aulast%3DOsada%26aufirst%3DH.%26atitle%3DPhoto-cross-linked%2520small-molecule%2520affinity%2520matrix%2520for%2520facilitating%2520forward%2520and%2520reverse%2520chemical%2520genetics%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D3559%26epage%3D3562%26doi%3D10.1002%2Fanie.200462370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishiya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takashima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsutsumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okawa, K.</span></span> <span> </span><span class="NLM_article-title">A new efficient method of generating photoaffinity beads for drug target identification</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.bmcl.2017.01.021" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=834-840&author=Y.+Nishiyaauthor=T.+Hamadaauthor=M.+Abeauthor=M.+Takashimaauthor=K.+Tsutsumiauthor=K.+Okawa&title=A+new+efficient+method+of+generating+photoaffinity+beads+for+drug+target+identification&doi=10.1016%2Fj.bmcl.2017.01.021"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DNishiya%26aufirst%3DY.%26aulast%3DHamada%26aufirst%3DT.%26aulast%3DAbe%26aufirst%3DM.%26aulast%3DTakashima%26aufirst%3DM.%26aulast%3DTsutsumi%26aufirst%3DK.%26aulast%3DOkawa%26aufirst%3DK.%26atitle%3DA%2520new%2520efficient%2520method%2520of%2520generating%2520photoaffinity%2520beads%2520for%2520drug%2520target%2520identification%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D834%26epage%3D840%26doi%3D10.1016%2Fj.bmcl.2017.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedman Ohana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uyeda, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Encell, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Deciphering the cellular targets of bioactive compounds using a chloroalkane capture tag</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2316</span>– <span class="NLM_lpage">2324</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00351</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00351" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFGitLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2316-2324&author=R.+Friedman+Ohanaauthor=T.+A.+Kirklandauthor=C.+C.+Woodroofeauthor=S.+Levinauthor=H.+T.+Uyedaauthor=P.+Ottoauthor=R.+Hurstauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=L.+P.+Encellauthor=K.+V.+Wood&title=Deciphering+the+cellular+targets+of+bioactive+compounds+using+a+chloroalkane+capture+tag&doi=10.1021%2Facschembio.5b00351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Deciphering the Cellular Targets of Bioactive Compounds Using a Chloroalkane Capture Tag</span></div><div class="casAuthors">Friedman Ohana, Rachel; Kirkland, Thomas A.; Woodroofe, Carolyn C.; Levin, Sergiy; Uyeda, H. Tetsuo; Otto, Paul; Hurst, Robin; Robers, Matthew B.; Zimmerman, Kris; Encell, Lance P.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2316-2324</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phenotypic screening of compd. libraries is a significant trend in drug discovery, yet success can be hindered by difficulties in identifying the underlying cellular targets.  Current approaches rely on tethering bioactive compds. to a capture tag or surface to allow selective enrichment of interacting proteins for subsequent identification by mass spectrometry.  Such methods are often constrained by ineffective capture of low affinity and low abundance targets.  In addn., these methods are often not compatible with living cells and therefore cannot be used to verify the pharmacol. activity of the tethered compds.  The authors have developed a novel chloroalkane capture tag that minimally affects compd. potency in cultured cells, allowing binding interactions with the targets to occur under conditions relevant to the desired cellular phenotype.  Subsequent isolation of the interacting targets is achieved through rapid lysis and capture onto immobilized HaloTag protein.  Exchanging the chloroalkane tag for a fluorophore, the putative targets identified by mass spectrometry can be verified for direct binding to the compd. through resonance energy transfer.  Using the interaction between histone deacetylases (HDACs) and the inhibitor, Vorinostat (SAHA), as a model system, the authors were able to identify and verify all the known HDAC targets of SAHA as well as two previously undescribed targets, ADO and CPPED1.  The discovery of ADO as a target may provide mechanistic insight into a reported connection between SAHA and Huntington's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw7IYjjAcESrVg90H21EOLACvtfcHk0ljH_RNRkFfz1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFGitLfN&md5=85d569074a909033683ca519e4b44d00</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00351%26sid%3Dliteratum%253Aachs%26aulast%3DFriedman%2BOhana%26aufirst%3DR.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DUyeda%26aufirst%3DH.%2BT.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DHurst%26aufirst%3DR.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DEncell%26aufirst%3DL.%2BP.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DDeciphering%2520the%2520cellular%2520targets%2520of%2520bioactive%2520compounds%2520using%2520a%2520chloroalkane%2520capture%2520tag%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2316%26epage%3D2324%26doi%3D10.1021%2Facschembio.5b00351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedman Ohana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uyeda, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Encell, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Improved deconvolution of protein targets for bioactive compounds using a palladium cleavable chloroalkane capture tag</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2608</span>– <span class="NLM_lpage">2617</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00408</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00408" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKms7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2608-2617&author=R.+Friedman+Ohanaauthor=S.+Levinauthor=M.+G.+Woodauthor=K.+Zimmermanauthor=M.+L.+Dartauthor=M.+K.+Schwinnauthor=T.+A.+Kirklandauthor=R.+Hurstauthor=H.+T.+Uyedaauthor=L.+P.+Encellauthor=K.+V.+Wood&title=Improved+deconvolution+of+protein+targets+for+bioactive+compounds+using+a+palladium+cleavable+chloroalkane+capture+tag&doi=10.1021%2Facschembio.6b00408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Deconvolution of Protein Targets for Bioactive Compounds Using a Palladium Cleavable Chloroalkane Capture Tag</span></div><div class="casAuthors">Friedman Ohana, Rachel; Levin, Sergiy; Wood, Monika G.; Zimmerman, Kris; Dart, Melanie L.; Schwinn, Marie K.; Kirkland, Thomas A.; Hurst, Robin; Uyeda, H. Tetsuo; Encell, Lance P.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2608-2617</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The benefits provided by phenotypic screening of compd. libraries are often countered by difficulties in identifying the underlying cellular targets.  We recently described a new approach utilizing a chloroalkane capture tag, which can be chem. attached to bioactive compds. to facilitate the isolation of their resp. targets for subsequent identification by mass spectrometry.  The tag minimally affects compd. potency and membrane permeability, enabling target engagement inside cells.  Effective enrichment of these targets is achieved through selectivity in both their rapid capture onto immobilized HaloTag and their subsequent release by competitive elution.  Here, we describe a significant improvement to this method where selective elution was achieved through palladium-catalyzed cleavage of an allyl-carbamate linkage incorporated into the chloroalkane capture tag.  Selective tag cleavage provided robust release of captured targets exhibiting different modes of binding to the bioactive compd., including prolonged residence time and covalent interactions.  Using the kinase inhibitors ibrutinib and BIRB796 as model compds., we demonstrated the capability of this new method to identify both expected targets and "off-targets" exhibiting a range of binding affinities, cellular abundances, and binding characteristics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkjMO4VhhlNLVg90H21EOLACvtfcHk0ljH_RNRkFfz1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKms7rO&md5=dd52a1d47f74edb7b554ac985e3dbdc5</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00408%26sid%3Dliteratum%253Aachs%26aulast%3DFriedman%2BOhana%26aufirst%3DR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHurst%26aufirst%3DR.%26aulast%3DUyeda%26aufirst%3DH.%2BT.%26aulast%3DEncell%26aufirst%3DL.%2BP.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DImproved%2520deconvolution%2520of%2520protein%2520targets%2520for%2520bioactive%2520compounds%2520using%2520a%2520palladium%2520cleavable%2520chloroalkane%2520capture%2520tag%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2608%26epage%3D2617%26doi%3D10.1021%2Facschembio.6b00408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendez-Johnson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman Ohana, R.</span></span> <span> </span><span class="NLM_article-title">Target identification using cell permeable and cleavable chloroalkane derivatized small molecules</span>. <i>Methods Mol. Biol. (N. Y., NY, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1647</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-7201-2_6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2F978-1-4939-7201-2_6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28808997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC1cfns1Kguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1647&publication_year=2017&pages=91-108&author=J.+L.+Mendez-Johnsonauthor=D.+L.+Danielsauthor=M.+Urhauthor=R.+Friedman+Ohana&title=Target+identification+using+cell+permeable+and+cleavable+chloroalkane+derivatized+small+molecules&doi=10.1007%2F978-1-4939-7201-2_6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Target Identification Using Cell Permeable and Cleavable Chloroalkane Derivatized Small Molecules</span></div><div class="casAuthors">Mendez-Johnson Jacqui L; Daniels Danette L; Urh Marjeta; Friedman Ohana Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in molecular biology (Clifton, N.J.)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1647</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">91-108</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">An important aspect for gaining functional insight into the activity of small molecules revealed through phenotypic screening is the identification of their interacting proteins.  Yet, isolating and validating these interacting proteins remains difficult.  Here, we present a new approach utilizing a chloroalkane (CA) moiety capture handle, which can be chemically attached to small molecules to isolate their respective protein targets.  Derivatization of small molecules with the CA moiety has been shown to not significantly impact their cell permeability or potency, allowing for phenotypic validation of the derivatized small molecule prior to capture.  The retention of cell permeability also allows for treatment of live cells with the derivatized small molecule and the CA moiety enables rapid covalent capture onto HaloTag coated magnetic beads.  Additionally, several options are available for the elution of interacting proteins, including chemical cleavage of the CA moiety, competitive elution using excess unmodified small molecule, or sodium dodecyl sulfate (SDS) elution.  These features taken together yield a highly robust and efficient process for target identification, including capture of weak or low abundance interactors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQa3ZJ12fU6eXthwrmtH4OBfW6udTcc2eZTmud_0PU2W7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfns1Kguw%253D%253D&md5=b457db4ebabec8ddeb9bc0ef6213db98</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7201-2_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-7201-2_6%26sid%3Dliteratum%253Aachs%26aulast%3DMendez-Johnson%26aufirst%3DJ.%2BL.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DFriedman%2BOhana%26aufirst%3DR.%26atitle%3DTarget%2520identification%2520using%2520cell%2520permeable%2520and%2520cleavable%2520chloroalkane%2520derivatized%2520small%2520molecules%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2017%26volume%3D1647%26spage%3D91%26epage%3D108%26doi%3D10.1007%2F978-1-4939-7201-2_6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span> <span> </span><span class="NLM_article-title">Identifying cellular targets of small-molecule probes and drugs with biochemical enrichment and SILAC</span>. <i>Methods Mol. Biol. (N. Y., NY, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>803</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-364-6_9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2F978-1-61779-364-6_9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=22065222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlersLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=803&publication_year=2012&pages=129-140&author=S.+E.+Ongauthor=X.+Liauthor=M.+Schenoneauthor=S.+L.+Schreiberauthor=S.+A.+Carr&title=Identifying+cellular+targets+of+small-molecule+probes+and+drugs+with+biochemical+enrichment+and+SILAC&doi=10.1007%2F978-1-61779-364-6_9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying cellular targets of small-molecule probes and drugs with biochemical enrichment and SILAC</span></div><div class="casAuthors">Ong, Shao-En; Li, Xiaoyu; Schenone, Monica; Schreiber, Stuart L.; Carr, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">803</span>
        (<span class="NLM_cas:issue">Chemical Proteomics</span>),
    <span class="NLM_cas:pages">129-140</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Sequencing of the human genome in the last decade has not yet led to a concomitant increase in the nos. of novel drug targets.  While the pharmaceutical industry has invested heavily in improving drugs for existing protein targets, it has not tended toward a similar investment in exptl. approaches to identify cellular targets of drugs.  It is striking that the targets of numerous widely used FDA-approved drugs remain unknown.  The development of robust, unbiased methods for target identification would greatly enhance our understanding the mechanisms-of-action of small mols.  Cell-based phenotypic screens followed by unbiased target identification have the potential to identify novel combinations of small mols. and their protein targets, shed light on drug polypharmacol., and enable unbiased screening approaches to drug discovery.  Classical biochem. enrichment with immobilized small mols. has been used for over four decades but has been limited by issues concerning specificity and sensitivity.  The application of mass spectrometry-based quant. proteomics in combination with these affinity reagents has proven to be esp. useful in addressing these common issues in affinity purifn. expts.  We describe the use of SILAC in identifying proteins that bind small-mol. probes and drugs in a cellular context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-0ZARyhSotbVg90H21EOLACvtfcHk0lhsmnrQfzNahg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlersLjE&md5=4d95b8356fc6c4f9130465643e8c8e4b</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-364-6_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-364-6_9%26sid%3Dliteratum%253Aachs%26aulast%3DOng%26aufirst%3DS.%2BE.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26atitle%3DIdentifying%2520cellular%2520targets%2520of%2520small-molecule%2520probes%2520and%2520drugs%2520with%2520biochemical%2520enrichment%2520and%2520SILAC%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2012%26volume%3D803%26spage%3D129%26epage%3D140%26doi%3D10.1007%2F978-1-61779-364-6_9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murale, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haque, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span> <span> </span><span class="NLM_article-title">Photo-affinity labeling (PAL) in chemical proteomics: a handy tool to investigate protein-protein interactions (PPIs)</span>. <i>Proteome Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">14</span>, <span class="refDoi"> DOI: 10.1186/s12953-017-0123-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1186%2Fs12953-017-0123-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28652856" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=14&author=D.+P.+Muraleauthor=S.+C.+Hongauthor=M.+M.+Haqueauthor=J.+S.+Lee&title=Photo-affinity+labeling+%28PAL%29+in+chemical+proteomics%3A+a+handy+tool+to+investigate+protein-protein+interactions+%28PPIs%29&doi=10.1186%2Fs12953-017-0123-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1186%2Fs12953-017-0123-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12953-017-0123-3%26sid%3Dliteratum%253Aachs%26aulast%3DMurale%26aufirst%3DD.%2BP.%26aulast%3DHong%26aufirst%3DS.%2BC.%26aulast%3DHaque%26aufirst%3DM.%2BM.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26atitle%3DPhoto-affinity%2520labeling%2520%2528PAL%2529%2520in%2520chemical%2520proteomics%253A%2520a%2520handy%2520tool%2520to%2520investigate%2520protein-protein%2520interactions%2520%2528PPIs%2529%26jtitle%3DProteome%2520Sci.%26date%3D2017%26volume%3D15%26spage%3D14%26doi%3D10.1186%2Fs12953-017-0123-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marjanovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewandowski, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesbauer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ready, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span> <span> </span><span class="NLM_article-title">2-Aryl-5-carboxytetrazole as a new photoaffinity label for drug target identification</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">14609</span>– <span class="NLM_lpage">14615</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b06645</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b06645" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ylu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=14609-14615&author=A.+Hernerauthor=J.+Marjanovicauthor=T.+M.+Lewandowskiauthor=V.+Marinauthor=M.+Pattersonauthor=L.+Miesbauerauthor=D.+Readyauthor=J.+Williamsauthor=A.+Vasudevanauthor=Q.+Lin&title=2-Aryl-5-carboxytetrazole+as+a+new+photoaffinity+label+for+drug+target+identification&doi=10.1021%2Fjacs.6b06645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">2-Aryl-5-carboxytetrazole as a New Photoaffinity Label for Drug Target Identification</span></div><div class="casAuthors">Herner, Andras; Marjanovic, Jasmina; Lewandowski, Tracey M.; Marin, Violeta; Patterson, Melanie; Miesbauer, Laura; Ready, Damien; Williams, Jon; Vasudevan, Anil; Lin, Qing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">14609-14615</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Photoaffinity labels are powerful tools for dissecting ligand-protein interactions, and they have a broad utility in medicinal chem. and drug discovery.  Traditional photoaffinity labels work through nonspecific C-H/X-H bond insertion reactions with the protein of interest by the highly reactive photogenerated intermediate.  Herein, the authors report a new photoaffinity label, 2-aryl-5-carboxytetrazole (ACT), that interacts with the target protein via a unique mechanism in which the photogenerated carboxynitrile imine reacts with a proximal nucleophile near the target active site.  In two distinct case studies, the authors demonstrate that the attachment of ACT to a ligand does not significantly alter the binding affinity and specificity of the parent drug.  Compared with diazirine and benzophenone, two commonly used photoaffinity labels, in two case studies ACT showed higher photo-crosslinking yields toward their protein targets in vitro based on mass spectrometry anal.  In the in situ target identification studies, ACT successfully captured the desired targets with an efficiency comparable to the diazirine.  The authors expect that further development of this class of photoaffinity labels will lead to a broad range of applications across target identification, and validation and elucidation of the binding site in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4CPJmr7ODJLVg90H21EOLACvtfcHk0liIqRyTgCXvWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ylu73O&md5=266b5013741b7ed1863cdc49094ba840</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b06645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b06645%26sid%3Dliteratum%253Aachs%26aulast%3DHerner%26aufirst%3DA.%26aulast%3DMarjanovic%26aufirst%3DJ.%26aulast%3DLewandowski%26aufirst%3DT.%2BM.%26aulast%3DMarin%26aufirst%3DV.%26aulast%3DPatterson%26aufirst%3DM.%26aulast%3DMiesbauer%26aufirst%3DL.%26aulast%3DReady%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DJ.%26aulast%3DVasudevan%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DQ.%26atitle%3D2-Aryl-5-carboxytetrazole%2520as%2520a%2520new%2520photoaffinity%2520label%2520for%2520drug%2520target%2520identification%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D14609%26epage%3D14615%26doi%3D10.1021%2Fjacs.6b06645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Targeting protein kinases with selective and semipromiscuous covalent inhibitors</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>548</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-397918-6.00004-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2FB978-0-12-397918-6.00004-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25399643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Onu7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=548&publication_year=2014&pages=93-116&author=R.+M.+Millerauthor=J.+Taunton&title=Targeting+protein+kinases+with+selective+and+semipromiscuous+covalent+inhibitors&doi=10.1016%2FB978-0-12-397918-6.00004-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein kinases with selective and semipromiscuous covalent inhibitors</span></div><div class="casAuthors">Miller, Rand M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">548</span>
        (<span class="NLM_cas:issue">Protein Kinase Inhibitors in Research and Medicine</span>),
    <span class="NLM_cas:pages">93-116</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Protein kinase inhibitors are an important class of therapeutics.  In addn., selective kinase inhibitors can often reveal unexpected biol. insights, augmenting genetic approaches and playing a decisive role in preclin. target validation studies.  Nevertheless, developing protein kinase inhibitors with sufficient selectivity and pharmacodynamic potency presents significant challenges.  Targeting noncatalytic cysteines with covalent inhibitors is a powerful approach to address both challenges simultaneously.  Here, we describe our efforts to design irreversible and reversible electrophilic inhibitors with varying degrees of kinase selectivity.  Highly selective covalent inhibitors have been used to elucidate the roles of p90 ribosomal protein S6 kinases in animal models of atherosclerosis and diabetes.  By contrast, semipromiscuous covalent inhibitors have revealed new therapeutic targets in disease-causing parasites and have shown utility as chemoproteomic probes for interrogating kinase occupancy in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvwGf3R34QG7Vg90H21EOLACvtfcHk0liIqRyTgCXvWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Onu7nE&md5=7146d2377fbbc3d10cbf04ea0e304a77</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-397918-6.00004-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-397918-6.00004-5%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DTargeting%2520protein%2520kinases%2520with%2520selective%2520and%2520semipromiscuous%2520covalent%2520inhibitors%26jtitle%3DMethods%2520Enzymol.%26date%3D2014%26volume%3D548%26spage%3D93%26epage%3D116%26doi%3D10.1016%2FB978-0-12-397918-6.00004-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baruch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffery, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Enzyme activity–it’s all about image</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2003.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.tcb.2003.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=14729178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFegsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=29-35&author=A.+Baruchauthor=D.+A.+Jefferyauthor=M.+Bogyo&title=Enzyme+activity%E2%80%93it%E2%80%99s+all+about+image&doi=10.1016%2Fj.tcb.2003.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme activity - it's all about image</span></div><div class="casAuthors">Baruch, Amos; Jeffery, Douglas A.; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-35</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review and discussion.  Unraveling the functional roles of proteins is a major challenge facing the postgenome researcher.  Advances toward this goal have been made through the development of both chem. and biochem. tools for monitoring protein activity.  Recently, a myriad of fluorescence-based imaging tools have emerged for in vitro, in vivo, and whole animal applications.  These tools have provided methods to monitor the spatial and temporal distribution of proteins and bioorg. mols. dynamically.  Here, recent advances in chem. and biochem. techniques that allow the detection of enzymic activity within intact cells and in vivo are reviewed.  Such technologies have the potential to be integrated into drug-development programs to facilitate both the functional validation of pharmaceutical targets and the treatment of human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV67k9Zmph0bVg90H21EOLACvtfcHk0liczsek5d79jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFegsA%253D%253D&md5=9e28761f959869c5184c9294f584cec7</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2003.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2003.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DBaruch%26aufirst%3DA.%26aulast%3DJeffery%26aufirst%3DD.%2BA.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DEnzyme%2520activity%25E2%2580%2593it%25E2%2580%2599s%2520all%2520about%2520image%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2004%26volume%3D14%26spage%3D29%26epage%3D35%26doi%3D10.1016%2Fj.tcb.2003.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodani, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. J.</span></span> <span> </span><span class="NLM_article-title">Reaction-based small-molecule fluorescent probes for chemoselective bioimaging</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">973</span>– <span class="NLM_lpage">984</span>, <span class="refDoi"> DOI: 10.1038/nchem.1500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchem.1500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23174976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslahurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=973-984&author=J.+Chanauthor=S.+C.+Dodaniauthor=C.+J.+Chang&title=Reaction-based+small-molecule+fluorescent+probes+for+chemoselective+bioimaging&doi=10.1038%2Fnchem.1500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Reaction-based small-molecule fluorescent probes for chemoselective bioimaging</span></div><div class="casAuthors">Chan, Jefferson; Dodani, Sheel C.; Chang, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">973-984</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The dynamic chem. diversity of elements, ions and mols. that form the basis of life offers both a challenge and an opportunity for study.  Small-mol. fluorescent probes can make use of selective, bioorthogonal chemistries to report on specific analytes in cells and in more complex biol. specimens.  These probes offer powerful reagents to interrogate the physiol. and pathol. of reactive chem. species in their native environments with minimal perturbation to living systems.  This Review presents a survey of tools and tactics for using such probes to detect biol. important chem. analytes.  The authors highlight design criteria for effective chem. tools for use in biol. applications as well as gaps for future exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp136q67e3PjLVg90H21EOLACvtfcHk0liczsek5d79jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslahurnM&md5=2ab2ef20873f40a189b64fb5e2888998</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1500%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DDodani%26aufirst%3DS.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BJ.%26atitle%3DReaction-based%2520small-molecule%2520fluorescent%2520probes%2520for%2520chemoselective%2520bioimaging%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D973%26epage%3D984%26doi%3D10.1038%2Fnchem.1500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dubach, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinegoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumene Feruglio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">In vivo imaging of specific drug-target binding at subcellular resolution</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3946</span>, <span class="refDoi"> DOI: 10.1038/ncomms4946</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fncomms4946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24867710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVChs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3946&author=J.+M.+Dubachauthor=C.+Vinegoniauthor=R.+Mazitschekauthor=P.+Fumene+Feruglioauthor=L.+A.+Cameronauthor=R.+Weissleder&title=In+vivo+imaging+of+specific+drug-target+binding+at+subcellular+resolution&doi=10.1038%2Fncomms4946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo imaging of specific drug-target binding at subcellular resolution</span></div><div class="casAuthors">Dubach, J. M.; Vinegoni, C.; Mazitschek, R.; Fumene Feruglio, P.; Cameron, L. A.; Weissleder, R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3946</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The possibility of measuring binding of small-mol. drugs to desired targets in live cells could provide a better understanding of drug action.  However, current approaches mostly yield static data, require lysis or rely on indirect assays and thus often provide an incomplete understanding of drug action.  Here, we present a multiphoton fluorescence anisotropy microscopy live cell imaging technique to measure and map drug-target interaction in real time at subcellular resoln.  This approach is generally applicable using any fluorescently labeled drug and enables high-resoln. spatial and temporal mapping of bound and unbound drug distribution.  To illustrate our approach we measure intracellular target engagement of the chemotherapeutic Olaparib, a poly(ADP-ribose) polymerase inhibitor, in live cells and within a tumor in vivo.  These results are the first generalizable approach to directly measure drug-target binding in vivo and present a promising tool to enhance understanding of drug activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdsAoUwKS-W7Vg90H21EOLACvtfcHk0liczsek5d79jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVChs70%253D&md5=dac66bc6dc9df7b13ab0863f4f1adf02</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fncomms4946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4946%26sid%3Dliteratum%253Aachs%26aulast%3DDubach%26aufirst%3DJ.%2BM.%26aulast%3DVinegoni%26aufirst%3DC.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DFumene%2BFeruglio%26aufirst%3DP.%26aulast%3DCameron%26aufirst%3DL.%2BA.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DIn%2520vivo%2520imaging%2520of%2520specific%2520drug-target%2520binding%2520at%2520subcellular%2520resolution%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3946%26doi%3D10.1038%2Fncomms4946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Protein organic chemistry and applications for labeling and engineering in live-cell systems</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4088</span>– <span class="NLM_lpage">4106</span>, <span class="refDoi"> DOI: 10.1002/anie.201207089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201207089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivVKhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=4088-4106&author=Y.+Takaokaauthor=A.+Ojidaauthor=I.+Hamachi&title=Protein+organic+chemistry+and+applications+for+labeling+and+engineering+in+live-cell+systems&doi=10.1002%2Fanie.201207089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Organic Chemistry and Applications for Labeling and Engineering in Live-Cell Systems</span></div><div class="casAuthors">Takaoka, Yousuke; Ojida, Akio; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4088-4106</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The modification of proteins with synthetic probes is a powerful means of elucidating and engineering the functions of proteins both in vitro and in live cells or in vivo.  Herein the authors review recent progress in chem.-based protein modification methods and their application in protein engineering, with particular emphasis on the following four strategies: (1) the bioconjugation reactions of amino acids on the surfaces of natural proteins, mainly applied in test-tube settings; (2) the bioorthogonal reactions of proteins with non-natural functional groups; (3) the coupling of recognition and reactive sites using an enzyme or short peptide tag-probe pair for labeling natural amino acids; and (4) ligand-directed labeling chemistries for the selective labeling of endogenous proteins in living systems.  Overall, these techniques represent a useful set of tools for application in chem. biol., with the methods 2-4 in particular being applicable to crude (living) habitats.  Although still in its infancy, the use of org. chem. for the manipulation of endogenous proteins, with subsequent applications in living systems, represents a worthy challenge for many chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocsb_JlBFIH7Vg90H21EOLACvtfcHk0ljuAOI3lAyHgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivVKhsLY%253D&md5=9f7ef6846cc3e64f802254bd25d25941</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fanie.201207089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201207089%26sid%3Dliteratum%253Aachs%26aulast%3DTakaoka%26aufirst%3DY.%26aulast%3DOjida%26aufirst%3DA.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DProtein%2520organic%2520chemistry%2520and%2520applications%2520for%2520labeling%2520and%2520engineering%2520in%2520live-cell%2520systems%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D4088%26epage%3D4106%26doi%3D10.1002%2Fanie.201207089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galmozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joslyn, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallaro, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narvaiza, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Ligand and target discovery by fragment-based screening in human cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.cell.2016.12.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28111073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=527-541&author=C.+G.+Parkerauthor=A.+Galmozziauthor=Y.+Wangauthor=B.+E.+Correiaauthor=K.+Sasakiauthor=C.+M.+Joslynauthor=A.+S.+Kimauthor=C.+L.+Cavallaroauthor=R.+M.+Lawrenceauthor=S.+R.+Johnsonauthor=I.+Narvaizaauthor=E.+Saezauthor=B.+F.+Cravatt&title=Ligand+and+target+discovery+by+fragment-based+screening+in+human+cells&doi=10.1016%2Fj.cell.2016.12.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand and Target Discovery by Fragment-Based Screening in Human Cells</span></div><div class="casAuthors">Parker, Christopher G.; Galmozzi, Andrea; Wang, Yujia; Correia, Bruno E.; Sasaki, Kenji; Joslyn, Christopher M.; Kim, Arthur S.; Cavallaro, Cullen L.; Lawrence, R. Michael; Johnson, Stephen R.; Narvaiza, Inigo; Saez, Enrique; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">527-541.e29</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Advances in the synthesis and screening of small-mol. libraries have accelerated the discovery of chem. probes for studying biol. processes.  Still, only a small fraction of the human proteome has chem. ligands.  Here, we describe a platform that marries fragment-based ligand discovery with quant. chem. proteomics to map thousands of reversible small mol.-protein interactions directly in human cells, many of which can be site-specifically detd.  We show that fragment hits can be advanced to furnish selective ligands that affect the activity of proteins heretofore lacking chem. probes.  We further combine fragment-based chem. proteomics with phenotypic screening to identify small mols. that promote adipocyte differentiation by engaging the poorly characterized membrane protein PGRMC2.  Fragment-based screening in human cells thus provides an extensive proteome-wide map of protein ligandability and facilitates the coordinated discovery of bioactive small mols. and their mol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPk-YJwFlDwbVg90H21EOLACvtfcHk0ljuAOI3lAyHgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeiu7Y%253D&md5=835264291015037ab9b7e446992caba3</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DC.%2BG.%26aulast%3DGalmozzi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DSasaki%26aufirst%3DK.%26aulast%3DJoslyn%26aufirst%3DC.%2BM.%26aulast%3DKim%26aufirst%3DA.%2BS.%26aulast%3DCavallaro%26aufirst%3DC.%2BL.%26aulast%3DLawrence%26aufirst%3DR.%2BM.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DNarvaiza%26aufirst%3DI.%26aulast%3DSaez%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DLigand%2520and%2520target%2520discovery%2520by%2520fragment-based%2520screening%2520in%2520human%2520cells%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D527%26epage%3D541%26doi%3D10.1016%2Fj.cell.2016.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitorino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: applications to biomarker discovery, in vivo imaging and drug discovery</span>. <i>Am. J. PharmacoGenomics</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.2165/00129785-200404060-00004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.2165%2F00129785-200404060-00004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=15651898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslOmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=371-381&author=A.+B.+Bergerauthor=P.+M.+Vitorinoauthor=M.+Bogyo&title=Activity-based+protein+profiling%3A+applications+to+biomarker+discovery%2C+in+vivo+imaging+and+drug+discovery&doi=10.2165%2F00129785-200404060-00004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: applications to biomarker discovery, in vivo imaging and drug discovery</span></div><div class="casAuthors">Berger, Alicia B.; Vitorino, Phillip M.; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of PharmacoGenomics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">371-381</span>CODEN:
                <span class="NLM_cas:coden">AJPMC8</span>;
        ISSN:<span class="NLM_cas:issn">1175-2203</span>.
    
            (<span class="NLM_cas:orgname">Adis International Ltd.</span>)
        </div><div class="casAbstract">A review.  The genomic revolution has created a wealth of information regarding the fundamental genetic code that defines the inner workings of a cell.  However, it has become clear that analyzing genome sequences alone will not lead to new therapies to fight human disease.  Rather, an understanding of protein function within the context of complex cellular networks will be required to facilitate the discovery of novel drug targets and, subsequently, new therapies directed against them.  The past ten years has seen a dramatic increase in technologies that allow large-scale, systems-based methods for anal. of global biol. processes and disease states.  In the field of proteomics, several well-established methods persist as a means to resolve and analyze complex mixts. of proteins derived from cells and tissues.  However, the resolving power of these methods is often challenged by the diverse and dynamic nature of the proteome.  The field of activity-based proteomics, or chem. proteomics, was established in an attempt to focus proteomic efforts on subsets of physiol. important protein targets.  This new approach to proteomics is centered around the use of small mols. termed activity-based probes (ABPs) as a means to tag, enrich, and isolate, distinct sets of proteins based on their enzymic activity.  Chem. probes can be 'tuned' to react with defined enzymic targets through the use of chem. reactive warhead groups, fused to selective binding elements that control their overall specificity.  As a result, ABPs function as highly specific; mechanism-based reagents that provide a direct readout of enzymic activity within complex proteomes.  Modification of protein targets by an ABP facilitates their purifn. and isolation, thereby eliminating many of the confounding issues of dynamic range in protein abundance.  In this review, the authors outline recent advances in the field of chem. proteomics.  Specifically, the authors highlight how this technol. can be applied to advance the fields of biomarker discovery, in vivo imaging, and small mol. screening and drug target discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaxZGkCOsvc7Vg90H21EOLACvtfcHk0li0-iXZ12Lx5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslOmsg%253D%253D&md5=42186e406fcac17c088252644e216e0f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.2165%2F00129785-200404060-00004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F00129785-200404060-00004%26sid%3Dliteratum%253Aachs%26aulast%3DBerger%26aufirst%3DA.%2BB.%26aulast%3DVitorino%26aufirst%3DP.%2BM.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520applications%2520to%2520biomarker%2520discovery%252C%2520in%2520vivo%2520imaging%2520and%2520drug%2520discovery%26jtitle%3DAm.%2520J.%2520PharmacoGenomics%26date%3D2004%26volume%3D4%26spage%3D371%26epage%3D381%26doi%3D10.2165%2F00129785-200404060-00004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Z.-C.</span></span> <span> </span><span class="NLM_article-title">Target identification with quantitative activity based protein profiling (ABPP)</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1600212</span>– <span class="NLM_lpage">4</span>, <span class="refDoi"> DOI: 10.1002/pmic.201600212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fpmic.201600212" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=1600212-4&author=X.+Chenauthor=Y.+K.+Wongauthor=J.+Wangauthor=J.+Zhangauthor=Y.-M.+Leeauthor=H.-M.+Shenauthor=Q.+Linauthor=Z.-C.+Hua&title=Target+identification+with+quantitative+activity+based+protein+profiling+%28ABPP%29&doi=10.1002%2Fpmic.201600212"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Fpmic.201600212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.201600212%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWong%26aufirst%3DY.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DY.-M.%26aulast%3DShen%26aufirst%3DH.-M.%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DHua%26aufirst%3DZ.-C.%26atitle%3DTarget%2520identification%2520with%2520quantitative%2520activity%2520based%2520protein%2520profiling%2520%2528ABPP%2529%26jtitle%3DProteomics%26date%3D2017%26volume%3D17%26spage%3D1600212%26epage%3D4%26doi%3D10.1002%2Fpmic.201600212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pichler, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysiak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breinbauer, R.</span></span> <span> </span><span class="NLM_article-title">Target identification of covalently binding drugs by activity-based protein profiling (ABPP)</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3291</span>– <span class="NLM_lpage">3303</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.03.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.bmc.2016.03.050" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=3291-3303&author=C.+M.+Pichlerauthor=J.+Krysiakauthor=R.+Breinbauer&title=Target+identification+of+covalently+binding+drugs+by+activity-based+protein+profiling+%28ABPP%29&doi=10.1016%2Fj.bmc.2016.03.050"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.03.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.03.050%26sid%3Dliteratum%253Aachs%26aulast%3DPichler%26aufirst%3DC.%2BM.%26aulast%3DKrysiak%26aufirst%3DJ.%26aulast%3DBreinbauer%26aufirst%3DR.%26atitle%3DTarget%2520identification%2520of%2520covalently%2520binding%2520drugs%2520by%2520activity-based%2520protein%2520profiling%2520%2528ABPP%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D3291%26epage%3D3303%26doi%3D10.1016%2Fj.bmc.2016.03.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based proteomics of enzyme superfamilies: serine hydrolases as a case study</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">11051</span>– <span class="NLM_lpage">11055</span>, <span class="refDoi"> DOI: 10.1074/jbc.R109.097600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1074%2Fjbc.R109.097600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=20147750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktFGlsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=11051-11055&author=G.+M.+Simonauthor=B.+F.+Cravatt&title=Activity-based+proteomics+of+enzyme+superfamilies%3A+serine+hydrolases+as+a+case+study&doi=10.1074%2Fjbc.R109.097600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based Proteomics of Enzyme Superfamilies: Serine Hydrolases as a Case Study</span></div><div class="casAuthors">Simon, Gabriel M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">11051-11055</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Genome sequencing projects have uncovered thousands of uncharacterized enzymes in eukaryotic and prokaryotic organisms.  Deciphering the physiol. functions of enzymes requires tools to profile and perturb their activities in native biol. systems.  Activity-based protein profiling has emerged as a powerful chemoproteomic strategy to achieve these objectives through the use of chem. probes that target large swaths of enzymes that share active-site features.  Here, we review activity-based protein profiling and its implementation to annotate the enzymic proteome, with particular attention given to probes that target serine hydrolases, a diverse superfamily of enzymes replete with many uncharacterized members.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp04j7PfUvT_LVg90H21EOLACvtfcHk0ljpy4WMbs904g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktFGlsro%253D&md5=719e17677c5229574c9fc50006a16048</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R109.097600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R109.097600%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520proteomics%2520of%2520enzyme%2520superfamilies%253A%2520serine%2520hydrolases%2520as%2520a%2520case%2520study%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D11051%26epage%3D11055%26doi%3D10.1074%2Fjbc.R109.097600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: the serine hydrolases</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">14694</span>– <span class="NLM_lpage">14699</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.26.14694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.96.26.14694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10611275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=14694-14699&author=Y.+Liuauthor=M.+P.+Patricelliauthor=B.+F.+Cravatt&title=Activity-based+protein+profiling%3A+the+serine+hydrolases&doi=10.1073%2Fpnas.96.26.14694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: the serine hydrolases</span></div><div class="casAuthors">Liu, Yongsheng; Patricelli, Matthew P.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">14694-14699</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">With the postgenome era rapidly approaching, new strategies for the functional anal. of proteins are needed.  To date, proteomics efforts have primarily been confined to recording variations in protein level rather than activity.  The ability to profile classes of proteins on the basis of changes in their activity would greatly accelerate both the assignment of protein function and the identification of potential pharmaceutical targets.  Here, we describe the chem. synthesis and utility of an active-site directed probe for visualizing dynamics in the expression and function of an entire enzyme family, the serine hydrolases.  By reacting this probe, a biotinylated fluorophosphonate referred to as FP-biotin, with crude tissue exts., we quickly and with high sensitivity detect numerous serine hydrolases, many of which display tissue-restricted patterns of expression.  Addnl., we show that FP-biotin labels these proteins in an activity-dependent manner that can be followed kinetically, offering a powerful means to monitor dynamics simultaneously in both protein function and expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8i5m0pTTunrVg90H21EOLACvtfcHk0ljpy4WMbs904g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D&md5=d5d1a09328fc7715e9d30ca392733426</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.26.14694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.26.14694%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520the%2520serine%2520hydrolases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D14694%26epage%3D14699%26doi%3D10.1073%2Fpnas.96.26.14694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giang, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamp, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbaum, J. J.</span></span> <span> </span><span class="NLM_article-title">Direct visualization of serine hydrolase activities in complex proteomes using fluorescent active site-directed probes</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1002/1615-9861(200109)1:9<1067::AID-PROT1067>3.0.CO;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=11990500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsFOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=1067-1071&author=M.+P.+Patricelliauthor=D.+K.+Giangauthor=L.+M.+Stampauthor=J.+J.+Burbaum&title=Direct+visualization+of+serine+hydrolase+activities+in+complex+proteomes+using+fluorescent+active+site-directed+probes&doi=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Direct visualization of serine hydrolase activities in complex proteomes using fluorescent active site-directed probes</span></div><div class="casAuthors">Patricelli, Matthew P.; Giang, Dan K.; Stamp, Lisa M.; Burbaum, Jonathan J.</div><div class="citationInfo"><span class="NLM_cas:title">Proteomics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1067-1071</span>CODEN:
                <span class="NLM_cas:coden">PROTC7</span>;
        ISSN:<span class="NLM_cas:issn">1615-9853</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The field of biochem. is currently faced with the enormous challenge of assigning functional significance to more than thirty thousand predicted protein products encoded by the human genome.  In order to accomplish this daunting task, methods will be required that facilitate the global anal. of proteins in complex biol. systems.  Recently, methods have been described for simultaneously monitoring the activity of multiple enzymes in crude proteomes based on their reactivity with tagged chem. probes.  These activity based probes (ABPs) have used either radiochem. or biotin/avidin-based detection methods to allow consolidated visualization of numerous enzyme activities.  Here we report the synthesis and evaluation of fluorescent activity based probes for the serine hydrolase super-family of enzymes.  The fluorescent methods detailed herein provide superior throughput, sensitivity, and quant. accuracy when compared to previously described ABPs, and provide a straight-forward platform for high-throughput proteome anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrunfp6WitoIbVg90H21EOLACvtfcHk0lgJWXBk7McvvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsFOmtrs%253D&md5=5ed5063f140af14e24f1ab2444785eb4</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1615-9861%2528200109%25291%253A9%253C1067%253A%253AAID-PROT1067%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DGiang%26aufirst%3DD.%2BK.%26aulast%3DStamp%26aufirst%3DL.%2BM.%26aulast%3DBurbaum%26aufirst%3DJ.%2BJ.%26atitle%3DDirect%2520visualization%2520of%2520serine%2520hydrolase%2520activities%2520in%2520complex%2520proteomes%2520using%2520fluorescent%2520active%2520site-directed%2520probes%26jtitle%3DProteomics%26date%3D2001%26volume%3D1%26spage%3D1067%26epage%3D1071%26doi%3D10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Speers, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Profiling enzyme activities in vivo using click chemistry methods</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2004.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2004.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=15123248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1Cms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=535-546&author=A.+E.+Speersauthor=B.+F.+Cravatt&title=Profiling+enzyme+activities+in+vivo+using+click+chemistry+methods&doi=10.1016%2Fj.chembiol.2004.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling Enzyme Activities In Vivo Using Click Chemistry Methods</span></div><div class="casAuthors">Speers, Anna E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">535-546</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methods for profiling the activity of enzymes in vivo are needed to understand the role that these proteins and their endogenous regulators play in physiol. and pathol. processes.  Recently, we introduced a tag-free strategy for activity-based protein profiling (ABPP) that utilizes the copper(I)-catalyzed azide-alkyne cycloaddn. reaction ("click chem.") to analyze the functional state of enzymes in living cells and organisms.  Here, we report a detailed characterization of the reaction parameters that affect click chem.-based ABPP and identify conditions that maximize the speed, sensitivity, and bioorthogonality of this approach.  Using these optimized conditions, we compare the enzyme activity profiles of living and homogenized breast cancer cells, resulting in the identification of several enzymes that are labeled by activity-based probes in situ but not in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH0eSSbsaoDbVg90H21EOLACvtfcHk0lgJWXBk7McvvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1Cms78%253D&md5=467d291cac7fa5cd105cc45248c26eb5</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DSpeers%26aufirst%3DA.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProfiling%2520enzyme%2520activities%2520in%2520vivo%2520using%2520click%2520chemistry%2520methods%26jtitle%3DChem.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D535%26epage%3D546%26doi%3D10.1016%2Fj.chembiol.2004.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mileni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beidler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinney, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadagopan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liimatta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazerwith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiff, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamtekar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Becelaere, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of a highly selective FAAH inhibitor that reduces inflammatory pain</span>. <i>Chem. Biol. (Oxford, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">411</span>– <span class="NLM_lpage">420</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2009.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2009.02.013" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=411-420&author=K.+Ahnauthor=D.+S.+Johnsonauthor=M.+Mileniauthor=D.+Beidlerauthor=J.+Z.+Longauthor=M.+K.+McKinneyauthor=E.+Weerapanaauthor=N.+Sadagopanauthor=M.+Liimattaauthor=S.+E.+Smithauthor=S.+Lazerwithauthor=C.+Stiffauthor=S.+Kamtekarauthor=K.+Bhattacharyaauthor=Y.+Zhangauthor=S.+Swaneyauthor=K.+Van+Becelaereauthor=R.+C.+Stevensauthor=B.+F.+Cravatt&title=Discovery+and+characterization+of+a+highly+selective+FAAH+inhibitor+that+reduces+inflammatory+pain&doi=10.1016%2Fj.chembiol.2009.02.013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DD.%2BS.%26aulast%3DMileni%26aufirst%3DM.%26aulast%3DBeidler%26aufirst%3DD.%26aulast%3DLong%26aufirst%3DJ.%2BZ.%26aulast%3DMcKinney%26aufirst%3DM.%2BK.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DSadagopan%26aufirst%3DN.%26aulast%3DLiimatta%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DS.%2BE.%26aulast%3DLazerwith%26aufirst%3DS.%26aulast%3DStiff%26aufirst%3DC.%26aulast%3DKamtekar%26aufirst%3DS.%26aulast%3DBhattacharya%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSwaney%26aufirst%3DS.%26aulast%3DVan%2BBecelaere%26aufirst%3DK.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520a%2520highly%2520selective%2520FAAH%2520inhibitor%2520that%2520reduces%2520inflammatory%2520pain%26jtitle%3DChem.%2520Biol.%2520%2528Oxford%252C%2520U.%2520K.%2529%26date%3D2009%26volume%3D16%26spage%3D411%26epage%3D420%26doi%3D10.1016%2Fj.chembiol.2009.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saraf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolasa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lannoye, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brioni, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surowy, C. S.</span></span> <span> </span><span class="NLM_article-title">Fatty acid amide hydrolase inhibitors display broad selectivity and inhibit multiple carboxylesterases as off-targets</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1095</span>– <span class="NLM_lpage">1105</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2006.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.neuropharm.2006.11.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2007&pages=1095-1105&author=D.+Zhangauthor=A.+Sarafauthor=T.+Kolasaauthor=P.+Bhatiaauthor=G.+Z.+Zhengauthor=M.+Patelauthor=G.+S.+Lannoyeauthor=P.+Richardsonauthor=A.+Stewartauthor=J.+C.+Rogersauthor=J.+D.+Brioniauthor=C.+S.+Surowy&title=Fatty+acid+amide+hydrolase+inhibitors+display+broad+selectivity+and+inhibit+multiple+carboxylesterases+as+off-targets&doi=10.1016%2Fj.neuropharm.2006.11.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2006.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2006.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DSaraf%26aufirst%3DA.%26aulast%3DKolasa%26aufirst%3DT.%26aulast%3DBhatia%26aufirst%3DP.%26aulast%3DZheng%26aufirst%3DG.%2BZ.%26aulast%3DPatel%26aufirst%3DM.%26aulast%3DLannoye%26aufirst%3DG.%2BS.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DStewart%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DJ.%2BC.%26aulast%3DBrioni%26aufirst%3DJ.%2BD.%26aulast%3DSurowy%26aufirst%3DC.%2BS.%26atitle%3DFatty%2520acid%2520amide%2520hydrolase%2520inhibitors%2520display%2520broad%2520selectivity%2520and%2520inhibit%2520multiple%2520carboxylesterases%2520as%2520off-targets%26jtitle%3DNeuropharmacology%26date%3D2007%26volume%3D52%26spage%3D1095%26epage%3D1105%26doi%3D10.1016%2Fj.neuropharm.2006.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cognetta, A. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shpak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Kroeg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantae, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baggelaar, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vrij, F. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fezza, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Wel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soethoudt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mock, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Dulk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baak, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florea, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendriks, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Petrocellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overkleeft, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hankemeier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Zeeuw, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Marzo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maccarrone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kushner, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling reveals off-target proteins of the FAAH inhibitor BIA 10–2474</span>. <i>Science (Washington, DC, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>356</i></span>,  <span class="NLM_fpage">1084</span>– <span class="NLM_lpage">1087</span>, <span class="refDoi"> DOI: 10.1126/science.aaf7497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.aaf7497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28596366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt12jsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2017&pages=1084-1087&author=A.+C.+M.+van+Esbroeckauthor=A.+P.+A.+Janssenauthor=A.+B.+Cognettaauthor=D.+Ogasawaraauthor=G.+Shpakauthor=M.+van+der%0AKroegauthor=V.+Kantaeauthor=M.+P.+Baggelaarauthor=F.+M.+S.+de+Vrijauthor=H.+Dengauthor=M.+Allaraauthor=F.+Fezzaauthor=Z.+Linauthor=T.+van+der+Welauthor=M.+Soethoudtauthor=E.+D.+Mockauthor=H.+den+Dulkauthor=I.+L.+Baakauthor=B.+I.+Floreaauthor=G.+Hendriksauthor=L.+De+Petrocellisauthor=H.+S.+Overkleeftauthor=T.+Hankemeierauthor=C.+I.+De+Zeeuwauthor=V.+Di+Marzoauthor=M.+Maccarroneauthor=B.+F.+Cravattauthor=S.+A.+Kushnerauthor=M.+van+der%0AStelt&title=Activity-based+protein+profiling+reveals+off-target+proteins+of+the+FAAH+inhibitor+BIA+10%E2%80%932474&doi=10.1126%2Fscience.aaf7497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling reveals off-target proteins of the FAAH inhibitor BIA 10-2474</span></div><div class="casAuthors">van Esbroeck, Annelot C. M.; Janssen, Antonius P. A.; Cognetta, Armand B., III; Ogasawara, Daisuke; Shpak, Guy; van der Kroeg, Mark; Kantae, Vasudev; Baggelaar, Marc P.; de Vrij, Femke M. S.; Deng, Hui; Allara, Marco; Fezza, Filomena; Lin, Zhanmin; van der Wel, Tom; Soethoudt, Marjolein; Mock, Elliot D.; den Dulk, Hans; Baak, Ilse L.; Florea, Bogdan I.; Hendriks, Giel; De Petrocellis, Luciano; Overkleeft, Herman S.; Hankemeier, Thomas; De Zeeuw, Chris I.; Di Marzo, Vincenzo; Maccarrone, Mauro; Cravatt, Benjamin F.; Kushner, Steven A.; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">356</span>
        (<span class="NLM_cas:issue">6342</span>),
    <span class="NLM_cas:pages">1084-1087</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A recent phase 1 trial of the fatty acid amide hydrolase (FAAH) inhibitor BIA 10-2474 led to the death of one volunteer and produced mild-to-severe neurol. symptoms in four others.  Although the cause of the clin. neurotoxicity is unknown, it has been postulated, given the clin. safety profile of other tested FAAH inhibitors, that off-target activities of BIA 10-2474 may have played a role.  Here the authors use activity-based proteomic methods to det. the protein interaction landscape of BIA 10-2474 in human cells and tissues.  This anal. revealed that the drug inhibits several lipases that are not targeted by PF04457845, a highly selective and clin. tested FAAH inhibitor.  BIA 10-2474, but not PF04457845, produced substantial alterations in lipid networks in human cortical neurons, suggesting that promiscuous lipase inhibitors have the potential to cause metabolic dysregulation in the nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_cUEzfW-Nd7Vg90H21EOLACvtfcHk0lhsiMhEhdxZxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt12jsb0%253D&md5=6133b21e104942f7c6818ba1de0d73f4</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf7497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf7497%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3DCognetta%26aufirst%3DA.%2BB.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DShpak%26aufirst%3DG.%26aulast%3Dvan%2Bder%2BKroeg%26aufirst%3DM.%26aulast%3DKantae%26aufirst%3DV.%26aulast%3DBaggelaar%26aufirst%3DM.%2BP.%26aulast%3Dde%2BVrij%26aufirst%3DF.%2BM.%2BS.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DAllara%26aufirst%3DM.%26aulast%3DFezza%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3Dvan%2Bder%2BWel%26aufirst%3DT.%26aulast%3DSoethoudt%26aufirst%3DM.%26aulast%3DMock%26aufirst%3DE.%2BD.%26aulast%3Dden%2BDulk%26aufirst%3DH.%26aulast%3DBaak%26aufirst%3DI.%2BL.%26aulast%3DFlorea%26aufirst%3DB.%2BI.%26aulast%3DHendriks%26aufirst%3DG.%26aulast%3DDe%2BPetrocellis%26aufirst%3DL.%26aulast%3DOverkleeft%26aufirst%3DH.%2BS.%26aulast%3DHankemeier%26aufirst%3DT.%26aulast%3DDe%2BZeeuw%26aufirst%3DC.%2BI.%26aulast%3DDi%2BMarzo%26aufirst%3DV.%26aulast%3DMaccarrone%26aufirst%3DM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DKushner%26aufirst%3DS.%2BA.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DActivity-based%2520protein%2520profiling%2520reveals%2520off-target%2520proteins%2520of%2520the%2520FAAH%2520inhibitor%2520BIA%252010%25E2%2580%25932474%26jtitle%3DScience%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2017%26volume%3D356%26spage%3D1084%26epage%3D1087%26doi%3D10.1126%2Fscience.aaf7497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Disparate proteome reactivity profiles of carbon electrophiles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1038/nchembio.91</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.91" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=18488014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=405-407&author=E.+Weerapanaauthor=G.+M.+Simonauthor=B.+F.+Cravatt&title=Disparate+proteome+reactivity+profiles+of+carbon+electrophiles&doi=10.1038%2Fnchembio.91"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Disparate proteome reactivity profiles of carbon electrophiles</span></div><div class="casAuthors">Weerapana, Eranthie; Simon, Gabriel M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">405-407</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Insights into the proteome reactivity of electrophiles are crucial for designing activity-based probes for enzymes lacking cognate affinity labels.  Here, the authors show that different classes of carbon electrophiles exhibit markedly distinct amino acid labeling profiles in proteomes, ranging from selective reactivity with cysteine to adducts with several amino acids.  These data specify electrophilic chemotypes with restricted and permissive reactivity profiles to guide the design of next-generation functional proteomics probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI4Jh6351SYLVg90H21EOLACvtfcHk0lh9e7_5EmSpwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ejtL8%253D&md5=69eb0ae589c50f46a8695dc6731366e2</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.91&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.91%26sid%3Dliteratum%253Aachs%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DDisparate%2520proteome%2520reactivity%2520profiles%2520of%2520carbon%2520electrophiles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D405%26epage%3D407%26doi%3D10.1038%2Fnchembio.91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hacker, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazear, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Global profiling of lysine reactivity and ligandability in the human proteome</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1038/nchem.2826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchem.2826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=29168484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1181-1190&author=S.+M.+Hackerauthor=K.+M.+Backusauthor=M.+R.+Lazearauthor=S.+Forliauthor=B.+E.+Correiaauthor=B.+F.+Cravatt&title=Global+profiling+of+lysine+reactivity+and+ligandability+in+the+human+proteome&doi=10.1038%2Fnchem.2826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Global profiling of lysine reactivity and ligandability in the human proteome</span></div><div class="casAuthors">Hacker, Stephan M.; Backus, Keriann M.; Lazear, Michael R.; Forli, Stefano; Correia, Bruno E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nucleophilic amino acids make important contributions to protein function, including performing key roles in catalysis and serving as sites for post-translational modification.  Electrophilic groups that target amino-acid nucleophiles have been used to create covalent ligands and drugs, but have, so far, been mainly limited to cysteine and serine.  Here, the authors report a chem. proteomic platform for the global and quant. anal. of lysine residues in native biol. systems.  The authors have quantified, in total, >9000 lysines in human cell proteomes and have identified several hundred residues with heightened reactivity that are enriched at protein functional sites and can frequently be targeted by electrophilic small mols.  The authors also discovered lysine-reactive fragment electrophiles that inhibit enzymes by active site and allosteric mechanisms, as well as disrupt protein-protein interactions in transcriptional regulatory complexes, emphasizing the broad potential and diverse functional consequences of liganding lysine residues throughout the human proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRKvUVcT5YRrVg90H21EOLACvtfcHk0lh9e7_5EmSpwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN&md5=1dac8f079c1caffc8ae584291480fc61</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2826%26sid%3Dliteratum%253Aachs%26aulast%3DHacker%26aufirst%3DS.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DLazear%26aufirst%3DM.%2BR.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DGlobal%2520profiling%2520of%2520lysine%2520reactivity%2520and%2520ligandability%2520in%2520the%2520human%2520proteome%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1181%26epage%3D1190%26doi%3D10.1038%2Fnchem.2826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seo, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, K. S.</span></span> <span> </span><span class="NLM_article-title">Profiling protein thiol oxidation in tumor cells using sulfenic acid-specific antibodies</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">16163</span>– <span class="NLM_lpage">16168</span>, <span class="refDoi"> DOI: 10.1073/pnas.0903015106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.0903015106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=19805274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OjsbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=16163-16168&author=Y.+H.+Seoauthor=K.+S.+Carroll&title=Profiling+protein+thiol+oxidation+in+tumor+cells+using+sulfenic+acid-specific+antibodies&doi=10.1073%2Fpnas.0903015106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling protein thiol oxidation in tumor cells using sulfenic acid-specific antibodies</span></div><div class="casAuthors">Seo, Young Ho; Carroll, Kate S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">16163-16168, S16163/1-S16163/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hydrogen peroxide (H2O2) functions as a second messenger that can activate cell proliferation through chemoselective oxidn. of cysteine residues in signaling proteins.  The connection between H2O2 signaling, thiol oxidn., and activation of growth pathways has emerged as fertile ground for the development of strategies for cancer treatment.  Central to achieving this goal is the development of tools and assays that facilitate characterization of the mol. events assocd. with tumorigenesis and evaluation of patient response to therapy.  Here we report on the development of an immunochem. method for detecting sulfenic acid, the initial oxidn. product that results when a thiolate reacts with H2O2.  For this approach, the sulfenic acid is derivatized with a chem. tag to generate a unique epitope for recognition.  The elicited antibody is exquisitely specific, context-independent, and capable of visualizing sulfenic acid formation in cells.  Applying this approach to several systems, including cancer cell lines, shows it can be used to monitor differences in thiol redox status and reveals a diverse pattern of sulfenic acid modifications across different subtypes of breast tumors.  These studies demonstrate a general strategy for producing antibodies against a specific oxidn. state of cysteine and show the utility of these reagents for profiling thiol oxidn. assocd. with pathol. conditions such as breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4-72mmCOXrrVg90H21EOLACvtfcHk0lh9e7_5EmSpwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OjsbbE&md5=126a4c8a21227fb8ca7c74c86c9d28ba</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0903015106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0903015106%26sid%3Dliteratum%253Aachs%26aulast%3DSeo%26aufirst%3DY.%2BH.%26aulast%3DCarroll%26aufirst%3DK.%2BS.%26atitle%3DProfiling%2520protein%2520thiol%2520oxidation%2520in%2520tumor%2520cells%2520using%2520sulfenic%2520acid-specific%2520antibodies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D16163%26epage%3D16168%26doi%3D10.1073%2Fpnas.0903015106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piggott, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karuso, P.</span></span> <span> </span><span class="NLM_article-title">Identifying the cellular targets of natural products using T7 phage display</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">626</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1039/C5NP00128E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2FC5NP00128E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26964751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFegsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=626-636&author=A.+M.+Piggottauthor=P.+Karuso&title=Identifying+the+cellular+targets+of+natural+products+using+T7+phage+display&doi=10.1039%2FC5NP00128E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying the cellular targets of natural products using T7 phage display</span></div><div class="casAuthors">Piggott, Andrew M.; Karuso, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">626-636</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covering: up to the end of 2015While Nature continues to deliver a myriad of potent and structurally diverse biol. active small mols., the cellular targets and modes of action of these natural products are rarely identified, significantly hindering their development as new chemotherapeutic agents.  This article provides an introductory tutorial on the use of T7 phage display as a tool to rapidly identify the cellular targets of natural products and is aimed specifically at natural products chemists who may have only limited experience in mol. biol.  A brief overview of T7 phage display is provided, including its strengths, weaknesses, and the type of problems that can and cannot be tackled with this technol.  Affinity probe construction is reviewed, including linker design and natural product derivatisation strategies.  A detailed description of the T7 phage biopanning procedure is provided, with valuable tips for optimizing each step in the process, as well as advice for identifying and avoiding the most commonly encountered challenges and pitfalls along the way.  Finally, a brief discussion is provided on techniques for validating the cellular targets identified using T7 phage display.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW7KABYJ48i7Vg90H21EOLACvtfcHk0lgWhBsre0yfHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFegsLo%253D&md5=a070c3c88527b50b2591798731751b57</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1039%2FC5NP00128E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5NP00128E%26sid%3Dliteratum%253Aachs%26aulast%3DPiggott%26aufirst%3DA.%2BM.%26aulast%3DKaruso%26aufirst%3DP.%26atitle%3DIdentifying%2520the%2520cellular%2520targets%2520of%2520natural%2520products%2520using%2520T7%2520phage%2520display%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2016%26volume%3D33%26spage%3D626%26epage%3D636%26doi%3D10.1039%2FC5NP00128E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takakusagi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takakusagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, K.</span></span> <span> </span><span class="NLM_article-title">Use of phage display technology for the determination of the targets for small-molecule therapeutics</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1517/17460441003653155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1517%2F17460441003653155" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=361-389&author=Y.+Takakusagiauthor=K.+Takakusagiauthor=F.+Sugawaraauthor=K.+Sakaguchi&title=Use+of+phage+display+technology+for+the+determination+of+the+targets+for+small-molecule+therapeutics&doi=10.1517%2F17460441003653155"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1517%2F17460441003653155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441003653155%26sid%3Dliteratum%253Aachs%26aulast%3DTakakusagi%26aufirst%3DY.%26aulast%3DTakakusagi%26aufirst%3DK.%26aulast%3DSugawara%26aufirst%3DF.%26aulast%3DSakaguchi%26aufirst%3DK.%26atitle%3DUse%2520of%2520phage%2520display%2520technology%2520for%2520the%2520determination%2520of%2520the%2520targets%2520for%2520small-molecule%2520therapeutics%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D361%26epage%3D389%26doi%3D10.1517%2F17460441003653155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Licitra, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. O.</span></span> <span> </span><span class="NLM_article-title">A three-hybrid system for detecting small ligand-protein receptor interactions</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">12817</span>– <span class="NLM_lpage">12821</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.23.12817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.93.23.12817" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=12817-12821&author=E.+J.+Licitraauthor=J.+O.+Liu&title=A+three-hybrid+system+for+detecting+small+ligand-protein+receptor+interactions&doi=10.1073%2Fpnas.93.23.12817"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.23.12817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.23.12817%26sid%3Dliteratum%253Aachs%26aulast%3DLicitra%26aufirst%3DE.%2BJ.%26aulast%3DLiu%26aufirst%3DJ.%2BO.%26atitle%3DA%2520three-hybrid%2520system%2520for%2520detecting%2520small%2520ligand-protein%2520receptor%2520interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D12817%26epage%3D12821%26doi%3D10.1073%2Fpnas.93.23.12817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rezwan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auerbach, D.</span></span> <span> </span><span class="NLM_article-title">Yeast ″N″-hybrid systems for protein-protein and drug-protein interaction discovery</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2012.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.ymeth.2012.06.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=423-429&author=M.+Rezwanauthor=D.+Auerbach&title=Yeast+%E2%80%B3N%E2%80%B3-hybrid+systems+for+protein-protein+and+drug-protein+interaction+discovery&doi=10.1016%2Fj.ymeth.2012.06.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2012.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2012.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DRezwan%26aufirst%3DM.%26aulast%3DAuerbach%26aufirst%3DD.%26atitle%3DYeast%2520%25E2%2580%25B3N%25E2%2580%25B3-hybrid%2520systems%2520for%2520protein-protein%2520and%2520drug-protein%2520interaction%2520discovery%26jtitle%3DMethods%26date%3D2012%26volume%3D57%26spage%3D423%26epage%3D429%26doi%3D10.1016%2Fj.ymeth.2012.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santoni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabilloud, T.</span></span> <span> </span><span class="NLM_article-title">Membrane proteins and proteomics: un amour impossible?</span>. <i>Electrophoresis</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1054</span>– <span class="NLM_lpage">1070</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1522-2683(20000401)21:6<1054::AID-ELPS1054>3.0.CO;2-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1054%3A%3AAID-ELPS1054%3E3.0.CO%3B2-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10786880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3cXis12kurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=1054-1070&author=V.+Santoniauthor=M.+Molloyauthor=T.+Rabilloud&title=Membrane+proteins+and+proteomics%3A+un+amour+impossible%3F&doi=10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1054%3A%3AAID-ELPS1054%3E3.0.CO%3B2-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane proteins and proteomics: un amour impossible?</span></div><div class="casAuthors">Santoni, Veronique; Molloy, Mark; Rabilloud, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Electrophoresis</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1054-1070</span>CODEN:
                <span class="NLM_cas:coden">ELCTDN</span>;
        ISSN:<span class="NLM_cas:issn">0173-0835</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">A review with 136 refs.  Proteome anal. implies the ability to sep. proteins as a first step prior to characterization.  Thus, the overall performance of the anal. strongly depends on the performance of the sepn. tool, usually two-dimensional electrophoresis.  This review shows how two-dimensional electrophoresis performs with membrane proteins from bacteria or animal or vegetable cells and tissues, the recent progress in this field, and it examines future prospects in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbh4b0Ke4VKbVg90H21EOLACvtfcHk0liTfp_B8D1SjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXis12kurk%253D&md5=43a7c9679485adfbb68975dd126c982b</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1054%3A%3AAID-ELPS1054%3E3.0.CO%3B2-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291522-2683%252820000401%252921%253A6%253C1054%253A%253AAID-ELPS1054%253E3.0.CO%253B2-8%26sid%3Dliteratum%253Aachs%26aulast%3DSantoni%26aufirst%3DV.%26aulast%3DMolloy%26aufirst%3DM.%26aulast%3DRabilloud%26aufirst%3DT.%26atitle%3DMembrane%2520proteins%2520and%2520proteomics%253A%2520un%2520amour%2520impossible%253F%26jtitle%3DElectrophoresis%26date%3D2000%26volume%3D21%26spage%3D1054%26epage%3D1070%26doi%3D10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1054%3A%3AAID-ELPS1054%3E3.0.CO%3B2-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stockmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stairs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ireland-Zecchini, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindle, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeper, F. J.</span></span> <span> </span><span class="NLM_article-title">Development and evaluation of new cyclooctynes for cell surface glycan imaging in cancer cells</span>. <i>Chem Sci</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">932</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1039/c0sc00631a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2Fc0sc00631a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=22724056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC2srmslaqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=932-936&author=H.+Stockmannauthor=A.+A.+Nevesauthor=S.+Stairsauthor=H.+Ireland-Zecchiniauthor=K.+M.+Brindleauthor=F.+J.+Leeper&title=Development+and+evaluation+of+new+cyclooctynes+for+cell+surface+glycan+imaging+in+cancer+cells&doi=10.1039%2Fc0sc00631a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of new cyclooctynes for cell surface glycan imaging in cancer cells</span></div><div class="casAuthors">Stockmann Henning; Neves Andre A; Stairs Shaun; Ireland-Zecchini Heather; Brindle Kevin M; Leeper Finian J</div><div class="citationInfo"><span class="NLM_cas:title">Chemical science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">932-936</span>
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    </div><div class="casAbstract">Two reagents have been synthesized for selective labeling of cell surface azidoglycans, an unusually stable version of a dibenzocyclooctyne (TMDIBO) and a third-generation difluorinated cyclooctyne (DIFO3).  Both syntheses are efficient with minimal purification, and the dibenzocyclooctyne is stable under basic and acidic conditions.  Flow cytometric measurements with azidosugar labeled cancer cells, in which these reagents were linked to the fluorophore Alexa Fluor 647, gave a signal-to-background ratio of up to 35 with TMDIBO as compared to ≈10 for DIFO3 and ≈5 for a phosphine reagent.  TMDIBO-based probes should have applications in molecular imaging of cell surface glycans in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzfo0elEmHyUEksK8CslK8fW6udTcc2eZW2vIRgbzVbLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmslaqug%253D%253D&md5=f533b9dd572ce4182541603316f13661</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1039%2Fc0sc00631a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0sc00631a%26sid%3Dliteratum%253Aachs%26aulast%3DStockmann%26aufirst%3DH.%26aulast%3DNeves%26aufirst%3DA.%2BA.%26aulast%3DStairs%26aufirst%3DS.%26aulast%3DIreland-Zecchini%26aufirst%3DH.%26aulast%3DBrindle%26aufirst%3DK.%2BM.%26aulast%3DLeeper%26aufirst%3DF.%2BJ.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%2520new%2520cyclooctynes%2520for%2520cell%2520surface%2520glycan%2520imaging%2520in%2520cancer%2520cells%26jtitle%3DChem%2520Sci%26date%3D2011%26volume%3D2%26spage%3D932%26epage%3D936%26doi%3D10.1039%2Fc0sc00631a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neves, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmston, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pryor, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ireland-Zecchini, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeper, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindle, K. M.</span></span> <span> </span><span class="NLM_article-title">Imaging sialylated tumor cell glycans in vivo</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2528</span>– <span class="NLM_lpage">2537</span>, <span class="refDoi"> DOI: 10.1096/fj.10-178590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1096%2Ffj.10-178590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21493886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFersr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=2528-2537&author=A.+A.+Nevesauthor=H.+Stockmannauthor=R.+R.+Harmstonauthor=H.+J.+Pryorauthor=I.+S.+Alamauthor=H.+Ireland-Zecchiniauthor=D.+Y.+Lewisauthor=S.+K.+Lyonsauthor=F.+J.+Leeperauthor=K.+M.+Brindle&title=Imaging+sialylated+tumor+cell+glycans+in+vivo&doi=10.1096%2Ffj.10-178590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Imaging sialylated tumor cell glycans in vivo</span></div><div class="casAuthors">Neves, Andre A.; Stockmann, Henning; Harmston, Rebecca R.; Pryor, Helen J.; Alam, Israt S.; Ireland-Zecchini, Heather; Lewis, David Y.; Lyons, Scott K.; Leeper, Finian J.; Brindle, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2528-2537, 10.1096/fj.10-178590</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Cell surface glycans are involved in numerous physiol. processes that involve cell-cell interactions and migration, including lymphocyte trafficking and cancer metastasis.  We have used a bioorthogonal metabolic labeling strategy to detect cell surface glycans and demonstrate, for the first time, fluorescence and radionuclide imaging of sialylated glycans in a murine tumor model in vivo.  Peracetylated azido-labeled N-acetyl-mannosamine, injected i.p., was used as the metabolic precursor for the biosynthesis of 5-azidoneuraminic, or azidosialic acid.  Azidosialic acid-labeled cell surface glycans were then reacted, by Staudinger ligation, with a biotinylated phosphine injected i.p., and the biotin was detected by subsequent i.v. injection of a fluorescent or radiolabeled avidin deriv.  At 24 h after administration of NeutrAvidin, labeled with either a far-red fluorophore or 111In, there was a significant azido-labeled N-acetyl-mannosamine-dependent increase in tumor-to-tissue contrast, which was detected using optical imaging or single-photon-emission computed tomog. (SPECT), resp.  The technique has the potential to translate to the clinic, where, given the prognostic relevance of altered sialic acid expression in cancer, it could be used to monitor disease progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBcmMsnI9T9rVg90H21EOLACvtfcHk0lg5gFG4VKu8Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFersr0%253D&md5=93a49206ecec034ed100e40840c69edc</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1096%2Ffj.10-178590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.10-178590%26sid%3Dliteratum%253Aachs%26aulast%3DNeves%26aufirst%3DA.%2BA.%26aulast%3DStockmann%26aufirst%3DH.%26aulast%3DHarmston%26aufirst%3DR.%2BR.%26aulast%3DPryor%26aufirst%3DH.%2BJ.%26aulast%3DAlam%26aufirst%3DI.%2BS.%26aulast%3DIreland-Zecchini%26aufirst%3DH.%26aulast%3DLewis%26aufirst%3DD.%2BY.%26aulast%3DLyons%26aufirst%3DS.%2BK.%26aulast%3DLeeper%26aufirst%3DF.%2BJ.%26aulast%3DBrindle%26aufirst%3DK.%2BM.%26atitle%3DImaging%2520sialylated%2520tumor%2520cell%2520glycans%2520in%2520vivo%26jtitle%3DFASEB%2520J.%26date%3D2011%26volume%3D25%26spage%3D2528%26epage%3D2537%26doi%3D10.1096%2Ffj.10-178590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stockmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marin, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balut, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, A.</span></span> <span> </span><span class="NLM_article-title">Glycan-mediated, ligand-controlled click chemistry for drug-target identification</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1002/cbic.201500590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fcbic.201500590" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=150-154&author=H.+Stockmannauthor=V.+L.+Marinauthor=P.+Nimmerauthor=C.+M.+Balutauthor=D.+J.+Davidsonauthor=P.+L.+Richardsonauthor=A.+Vasudevan&title=Glycan-mediated%2C+ligand-controlled+click+chemistry+for+drug-target+identification&doi=10.1002%2Fcbic.201500590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201500590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201500590%26sid%3Dliteratum%253Aachs%26aulast%3DStockmann%26aufirst%3DH.%26aulast%3DMarin%26aufirst%3DV.%2BL.%26aulast%3DNimmer%26aufirst%3DP.%26aulast%3DBalut%26aufirst%3DC.%2BM.%26aulast%3DDavidson%26aufirst%3DD.%2BJ.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DVasudevan%26aufirst%3DA.%26atitle%3DGlycan-mediated%252C%2520ligand-controlled%2520click%2520chemistry%2520for%2520drug-target%2520identification%26jtitle%3DChemBioChem%26date%3D2016%26volume%3D17%26spage%3D150%26epage%3D154%26doi%3D10.1002%2Fcbic.201500590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Traceless affinity labeling of endogenous proteins for functional analysis in living cells</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1469</span>, <span class="refDoi"> DOI: 10.1021/ar200334r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar200334r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1Oltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=1460-1469&author=T.+Hayashiauthor=I.+Hamachi&title=Traceless+affinity+labeling+of+endogenous+proteins+for+functional+analysis+in+living+cells&doi=10.1021%2Far200334r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Traceless Affinity Labeling of Endogenous Proteins for Functional Analysis in Living Cells</span></div><div class="casAuthors">Hayashi, Takahiro; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1460-1469</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein labeling and imaging techniques have provided tremendous opportunities to study the structure, function, dynamics, and localization of individual proteins in the complex environment of living cells.  Mol. biol.-based approaches, such as GFP-fusion tags and monoclonal antibodies, have served as important tools for the visualization of individual proteins in cells.  Although these techniques continue to be valuable for live cell imaging, they have a no. of limitations that have only been addressed by recent progress in chem.-based approaches.  These chem. approaches benefit greatly from the smaller probe sizes that should result in fewer perturbations to proteins and to biol. systems as a whole.  Despite the research in this area, so far none of these labeling techniques permit labeling and imaging of selected endogenous proteins in living cells.  Researchers have widely used affinity labeling, in which the protein of interest is labeled by a reactive group attached to a ligand, to identify and characterize proteins.  Since the first report of affinity labeling in the early 1960s, efforts to fine-tune the chem. structures of both the reactive group and ligand have led to protein labeling with excellent target selectivity in the whole proteome of living cells.  Although the chem. probes used for affinity labeling generally inactivate target proteins, this strategy holds promise as a valuable tool for the labeling and imaging of endogenous proteins in living cells and by extension in living animals.  In this Account, we summarize traceless affinity labeling, a technique explored mainly in our lab.  In our overview of the different labeling techniques, we emphasize the challenge of designing chem. probes that allow for dissocn. of the affinity module (often a ligand) after the labeling reaction so that the labeled protein retains its native function.  This feature distinguishes the traceless labeling approach from the traditional affinity labeling method and allows for real-time monitoring of protein activity.  With the high target specificity and biocompatibility of this technique, we have achieved individual labeling and imaging of endogenously expressed proteins in samples of high biol. complexity.  We also highlight applications in which our current approach enabled the monitoring of important biol. events, such as ligand binding, in living cells.  These novel chem. labeling techniques are expected to provide a mol. toolbox for studying a wide variety of proteins and beyond in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSs46czrmjCLVg90H21EOLACvtfcHk0ljJ2pe9Rk0N5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot1Oltb4%253D&md5=0a02bcf8fdb9df79abc93021cab7f99c</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Far200334r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far200334r%26sid%3Dliteratum%253Aachs%26aulast%3DHayashi%26aufirst%3DT.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DTraceless%2520affinity%2520labeling%2520of%2520endogenous%2520proteins%2520for%2520functional%2520analysis%2520in%2520living%2520cells%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2012%26volume%3D45%26spage%3D1460%26epage%3D1469%26doi%3D10.1021%2Far200334r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsukiji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Ligand-directed tosyl chemistry for protein labeling in vivo</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">341</span>– <span class="NLM_lpage">343</span>, <span class="refDoi"> DOI: 10.1038/nchembio.157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=19330012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsl2gurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=341-343&author=S.+Tsukijiauthor=M.+Miyagawaauthor=Y.+Takaokaauthor=T.+Tamuraauthor=I.+Hamachi&title=Ligand-directed+tosyl+chemistry+for+protein+labeling+in+vivo&doi=10.1038%2Fnchembio.157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-directed tosyl chemistry for protein labeling in vivo</span></div><div class="casAuthors">Tsukiji, Shinya; Miyagawa, Masayoshi; Takaoka, Yousuke; Tamura, Tomonori; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">341-343</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here the authors describe a method for the site-selective attachment of synthetic mols. into specific 'endogenous' proteins in vivo using ligand-directed tosyl (LDT) chem.  This approach was applied not only for chem. labeling proteins in living cells, tissues and mice but also for constructing a biosensor directly inside cells without genetic engineering.  These data establish LDT chem. as a new tool for the study and manipulation of biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0kFQSeuTHX7Vg90H21EOLACvtfcHk0ljJ2pe9Rk0N5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsl2gurY%253D&md5=604943938355591f1f4f5e861c6a8278</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.157%26sid%3Dliteratum%253Aachs%26aulast%3DTsukiji%26aufirst%3DS.%26aulast%3DMiyagawa%26aufirst%3DM.%26aulast%3DTakaoka%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DLigand-directed%2520tosyl%2520chemistry%2520for%2520protein%2520labeling%2520in%2520vivo%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D341%26epage%3D343%26doi%3D10.1038%2Fnchembio.157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baek, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, I.</span></span> <span> </span><span class="NLM_article-title">A small molecule that binds to an ATPase domain of Hsc70 promotes membrane trafficking of mutant cystic fibrosis transmembrane conductance regulator</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">20267</span>– <span class="NLM_lpage">20276</span>, <span class="refDoi"> DOI: 10.1021/ja206762p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja206762p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGgsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=20267-20276&author=H.+J.+Choauthor=H.+Y.+Geeauthor=K.+H.+Baekauthor=S.+K.+Koauthor=J.+M.+Parkauthor=H.+Leeauthor=N.+D.+Kimauthor=M.+G.+Leeauthor=I.+Shin&title=A+small+molecule+that+binds+to+an+ATPase+domain+of+Hsc70+promotes+membrane+trafficking+of+mutant+cystic+fibrosis+transmembrane+conductance+regulator&doi=10.1021%2Fja206762p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule That Binds to an ATPase Domain of Hsc70 Promotes Membrane Trafficking of Mutant Cystic Fibrosis Transmembrane Conductance Regulator</span></div><div class="casAuthors">Cho, Hyungseoph J.; Gee, Heon Yung; Baek, Kyung-Hwa; Ko, Sung-Kyun; Park, Jong-Moon; Lee, Hookeun; Kim, Nam-Doo; Lee, Min Goo; Shin, Injae</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">20267-20276</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cystic fibrosis transmembrane conductance regulator (CFTR) is a cell-surface anion channel that permeates chloride and bicarbonate ions.  The most frequent mutation of CFTR that causes cystic fibrosis is the deletion of phenylalanine at position 508 (ΔF508), which leads to defects in protein folding and cellular trafficking to the plasma membrane.  The lack of the cell-surface CFTR results in a redn. in the lifespan due to chronic lung infection with progressive deterioration of lung function.  Hsc70 plays a crucial role in degrdn. of mutant CFTR by the ubiquitin-proteasome system.  To date, various Hsc70 inhibitors and transcription regulators have been tested to det. whether they correct the defective activity of mutant CFTR.  However, they exhibited limited or questionable effects on restoring the chloride channel activity in cystic fibrosis cells.  Herein, the authors show that a small mol. apoptozole (Az) has high cellular potency to promote membrane trafficking of mutant CFTR and its chloride channel activity in cystic fibrosis cells.  Results from affinity chromatog. and ATPase activity assay indicate that Az inhibits the ATPase activity of Hsc70 by binding to its ATPase domain.  In addn., a ligand-directed protein labeling and mol. modeling studies also suggest the binding of Az to an ATPase domain, in particular, an ATP-binding pocket.  It is proposed that Az suppresses ubiquitination of ΔF508-CFTR maybe by blocking interaction of the mutant with Hsc70 and CHIP, and, as a consequence, it enhances membrane trafficking of the mutant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhNslz_-hHxbVg90H21EOLACvtfcHk0li_tOG5cr9N8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGgsb3P&md5=574564da07a3f0c14e09cefac46408db</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fja206762p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja206762p%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%2BJ.%26aulast%3DGee%26aufirst%3DH.%2BY.%26aulast%3DBaek%26aufirst%3DK.%2BH.%26aulast%3DKo%26aufirst%3DS.%2BK.%26aulast%3DPark%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DShin%26aufirst%3DI.%26atitle%3DA%2520small%2520molecule%2520that%2520binds%2520to%2520an%2520ATPase%2520domain%2520of%2520Hsc70%2520promotes%2520membrane%2520trafficking%2520of%2520mutant%2520cystic%2520fibrosis%2520transmembrane%2520conductance%2520regulator%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D20267%26epage%3D20276%26doi%3D10.1021%2Fja206762p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkanda, J.</span></span> <span> </span><span class="NLM_article-title">Phosphopeptide-dependent labeling of 14–3-3 zeta proteins by fusicoccin-based fluorescent probes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">512</span>, <span class="refDoi"> DOI: 10.1002/anie.201106995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201106995" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=509-512&author=M.+Takahashiauthor=A.+Kawamuraauthor=N.+Katoauthor=T.+Nishiauthor=I.+Hamachiauthor=J.+Ohkanda&title=Phosphopeptide-dependent+labeling+of+14%E2%80%933-3+zeta+proteins+by+fusicoccin-based+fluorescent+probes&doi=10.1002%2Fanie.201106995"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1002%2Fanie.201106995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201106995%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DNishi%26aufirst%3DT.%26aulast%3DHamachi%26aufirst%3DI.%26aulast%3DOhkanda%26aufirst%3DJ.%26atitle%3DPhosphopeptide-dependent%2520labeling%2520of%252014%25E2%2580%25933-3%2520zeta%2520proteins%2520by%2520fusicoccin-based%2520fluorescent%2520probes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D509%26epage%3D512%26doi%3D10.1002%2Fanie.201106995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujishima, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Ligand-directed acyl imidazole chemistry for labeling of membrane-bound proteins on live cells</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">3961</span>– <span class="NLM_lpage">3964</span>, <span class="refDoi"> DOI: 10.1021/ja2108855</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja2108855" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XislSqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=3961-3964&author=S.+H.+Fujishimaauthor=R.+Yasuiauthor=T.+Mikiauthor=A.+Ojidaauthor=I.+Hamachi&title=Ligand-directed+acyl+imidazole+chemistry+for+labeling+of+membrane-bound+proteins+on+live+cells&doi=10.1021%2Fja2108855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Directed Acyl Imidazole Chemistry for Labeling of Membrane-Bound Proteins on Live Cells</span></div><div class="casAuthors">Fujishima, Sho-hei; Yasui, Ryosuke; Miki, Takayuki; Ojida, Akio; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3961-3964</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem.-based protein labeling in living cells is undoubtedly useful for understanding natural protein functions and for biol./pharmaceutical applications.  Here, the authors report a novel approach for endogenous membrane-bound protein labeling for both in vitro and live cell conditions.  A moderately reactive alkyloxyacyl imidazole (AI) assisted by ligand-binding affinity (ligand-directed AI (LDAI)) chem. allowed the authors to selectively modify natural proteins, such as dihydrofolate reductase (DHFR) and folate receptor (FR), neither of which could be efficiently labeled using the recently developed ligand-directed tosylate approach.  It was clear that LDAI selectively labeled a single Lys(K32) in DHFR, proximal to the ligand-binding pocket.  The authors also demonstrate that the fluorescein-labeled (endogenous, by LDAI) FR works as a fluorescent biosensor on the live KB cell surface, which allowed the authors to carry out unprecedented in situ kinetic anal. of ligand binding to FR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ZPOEzKbO1LVg90H21EOLACvtfcHk0li_tOG5cr9N8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislSqu7w%253D&md5=6ccb65da77cad1b62892b738f42e1c55</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1021%2Fja2108855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja2108855%26sid%3Dliteratum%253Aachs%26aulast%3DFujishima%26aufirst%3DS.%2BH.%26aulast%3DYasui%26aufirst%3DR.%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DOjida%26aufirst%3DA.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DLigand-directed%2520acyl%2520imidazole%2520chemistry%2520for%2520labeling%2520of%2520membrane-bound%2520proteins%2520on%2520live%2520cells%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D3961%26epage%3D3964%26doi%3D10.1021%2Fja2108855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujishima, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyonaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">LDAI-based chemical labeling of intact membrane proteins and its pulse-chase analysis under live cell conditions</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1013</span>– <span class="NLM_lpage">1022</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2014.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25126991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlymtLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1013-1022&author=T.+Mikiauthor=S.+H.+Fujishimaauthor=K.+Komatsuauthor=K.+Kuwataauthor=S.+Kiyonakaauthor=I.+Hamachi&title=LDAI-based+chemical+labeling+of+intact+membrane+proteins+and+its+pulse-chase+analysis+under+live+cell+conditions&doi=10.1016%2Fj.chembiol.2014.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">LDAI-Based Chemical Labeling of Intact Membrane Proteins and Its Pulse-Chase Analysis under Live Cell Conditions</span></div><div class="casAuthors">Miki, Takayuki; Fujishima, Sho-hei; Komatsu, Kazuhiro; Kuwata, Keiko; Kiyonaka, Shigeki; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1013-1022</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The functions of membrane proteins are tightly controlled by the dynamics such as protein trafficking and degrdn.  We demonstrated that ligand-directed acyl imidazole (LDAI) chem. is broadly applicable to selective chem. labeling of various types of membrane-bound proteins under live cell conditions without a need for any tag fragments.  The LDAI chem. enabled pulse-chase anal. of these proteins to det. the half-life, as well as their degrdn. pathways by the imaging, under almost natural cellular conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFbXNTJLffMbVg90H21EOLACvtfcHk0lij0ZzZo0_xPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlymtLjK&md5=1599f71635cc66d96ed8d9edf8f7f842</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DFujishima%26aufirst%3DS.%2BH.%26aulast%3DKomatsu%26aufirst%3DK.%26aulast%3DKuwata%26aufirst%3DK.%26aulast%3DKiyonaka%26aufirst%3DS.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DLDAI-based%2520chemical%2520labeling%2520of%2520intact%2520membrane%2520proteins%2520and%2520its%2520pulse-chase%2520analysis%2520under%2520live%2520cell%2520conditions%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1013%26epage%3D1022%26doi%3D10.1016%2Fj.chembiol.2014.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koshi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minato, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyonaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukiji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Chemical cell-surface receptor engineering using affinity-guided, multivalent organocatalysts</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">12220</span>– <span class="NLM_lpage">12228</span>, <span class="refDoi"> DOI: 10.1021/ja204422r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja204422r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=12220-12228&author=H.+Wangauthor=Y.+Koshiauthor=D.+Minatoauthor=H.+Nonakaauthor=S.+Kiyonakaauthor=Y.+Moriauthor=S.+Tsukijiauthor=I.+Hamachi&title=Chemical+cell-surface+receptor+engineering+using+affinity-guided%2C+multivalent+organocatalysts&doi=10.1021%2Fja204422r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Cell-Surface Receptor Engineering Using Affinity-Guided, Multivalent Organocatalysts</span></div><div class="casAuthors">Wang, Hangxiang; Koshi, Yoichiro; Minato, Daishiro; Nonaka, Hiroshi; Kiyonaka, Shigeki; Mori, Yasuo; Tsukiji, Shinya; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">12220-12228</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Catalysts hold promise as tools for chem. protein modification.  However, the application of catalysts or catalyst-mediated reactions to proteins has only recently begun to be addressed, mainly in in vitro systems.  By radically improving the affinity-guided DMAP (4-dimethylaminopyridine) (AGD) catalysts that the authors previously reported, here the authors have developed a new organocatalyst-based approach that allows specific chem. acylation of a receptor protein on the surface of live cells.  The catalysts consist of a set of multivalent DMAP groups (the acyl transfer catalyst) fused to a ligand specific to the target protein.  It was clearly demonstrated by in vitro expts. that the catalyst multivalency enables remarkable enhancement of protein acylation efficiency in the labeling of three different proteins: congerin II, a Src homol. 2 (SH2) domain, and FKBP12.  Using a multivalent AGD catalyst and optimized acyl donors contg. a chosen probe, the authors successfully achieved selective chem. labeling of bradykinin B2 receptor (B2R), a G-protein coupled receptor, on the live cell-surface.  Furthermore, the present tool allowed the authors to construct a membrane protein (B2R)-based fluorescent biosensor, the fluorescence of which is enhanced (tuned on) in response to the antagonist ligand binding.  The biosensor should be applicable to rapid and quant. screening and assay of potent drug candidates in the cellular context.  The design concept of the affinity-guided, multivalent catalysts should facilitate further development of diverse catalyst-based protein modification tools, providing new opportunities for org. chem. in biol. research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZGVvnwgJKrbVg90H21EOLACvtfcHk0lij0ZzZo0_xPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSksLs%253D&md5=870e8766a9ab6216298c91a01586e022</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1021%2Fja204422r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja204422r%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DKoshi%26aufirst%3DY.%26aulast%3DMinato%26aufirst%3DD.%26aulast%3DNonaka%26aufirst%3DH.%26aulast%3DKiyonaka%26aufirst%3DS.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DTsukiji%26aufirst%3DS.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DChemical%2520cell-surface%2520receptor%2520engineering%2520using%2520affinity-guided%252C%2520multivalent%2520organocatalysts%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D12220%26epage%3D12228%26doi%3D10.1021%2Fja204422r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nukadzuka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, M.</span></span> <span> </span><span class="NLM_article-title">Reactive group-embedded affinity labeling reagent for efficient intracellular protein labeling</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2888</span>– <span class="NLM_lpage">2894</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.02.059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.bmc.2017.02.059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2888-2894&author=Y.+Takaokaauthor=Y.+Nukadzukaauthor=M.+Ueda&title=Reactive+group-embedded+affinity+labeling+reagent+for+efficient+intracellular+protein+labeling&doi=10.1016%2Fj.bmc.2017.02.059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.02.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.02.059%26sid%3Dliteratum%253Aachs%26aulast%3DTakaoka%26aufirst%3DY.%26aulast%3DNukadzuka%26aufirst%3DY.%26aulast%3DUeda%26aufirst%3DM.%26atitle%3DReactive%2520group-embedded%2520affinity%2520labeling%2520reagent%2520for%2520efficient%2520intracellular%2520protein%2520labeling%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2888%26epage%3D2894%26doi%3D10.1016%2Fj.bmc.2017.02.059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koster, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqi, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marappan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, P.</span></span> <span> </span><span class="NLM_article-title">Capture compound mass spectrometry: a technology for the investigation of small molecule protein interactions</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1089/adt.2006.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1089%2Fadt.2006.039" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=381-390&author=H.+Kosterauthor=D.+P.+Littleauthor=P.+Luanauthor=R.+Mullerauthor=S.+M.+Siddiqiauthor=S.+Marappanauthor=P.+Yip&title=Capture+compound+mass+spectrometry%3A+a+technology+for+the+investigation+of+small+molecule+protein+interactions&doi=10.1089%2Fadt.2006.039"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1089%2Fadt.2006.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2006.039%26sid%3Dliteratum%253Aachs%26aulast%3DKoster%26aufirst%3DH.%26aulast%3DLittle%26aufirst%3DD.%2BP.%26aulast%3DLuan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DR.%26aulast%3DSiddiqi%26aufirst%3DS.%2BM.%26aulast%3DMarappan%26aufirst%3DS.%26aulast%3DYip%26aufirst%3DP.%26atitle%3DCapture%2520compound%2520mass%2520spectrometry%253A%2520a%2520technology%2520for%2520the%2520investigation%2520of%2520small%2520molecule%2520protein%2520interactions%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2007%26volume%3D5%26spage%3D381%26epage%3D390%26doi%3D10.1089%2Fadt.2006.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blex, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrey, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furkert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichhorst, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartho, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gyapon Quast, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marais, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hakelberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niquet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popp, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroll, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sefkow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koster, H.</span></span> <span> </span><span class="NLM_article-title">Targeting G Protein-coupled receptors by capture compound mass spectrometry: A case study with Sertindole</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1639</span>– <span class="NLM_lpage">1649</span>, <span class="refDoi"> DOI: 10.1002/cbic.201700152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fcbic.201700152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28557180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFaqtb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1639-1649&author=C.+Blexauthor=S.+Michaelisauthor=A.+K.+Schreyauthor=J.+Furkertauthor=J.+Eichhorstauthor=K.+Barthoauthor=F.+Gyapon+Quastauthor=A.+Maraisauthor=M.+Hakelbergauthor=U.+Gruberauthor=S.+Niquetauthor=O.+Poppauthor=F.+Krollauthor=M.+Sefkowauthor=R.+Schuleinauthor=M.+Dregerauthor=H.+Koster&title=Targeting+G+Protein-coupled+receptors+by+capture+compound+mass+spectrometry%3A+A+case+study+with+Sertindole&doi=10.1002%2Fcbic.201700152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting G Protein-Coupled Receptors by Capture Compound Mass Spectrometry: A Case Study with Sertindole</span></div><div class="casAuthors">Blex, Christian; Michaelis, Simon; Schrey, Anna K.; Furkert, Jens; Eichhorst, Jenny; Bartho, Kathrin; Gyapon Quast, Frederick; Marais, Anett; Hakelberg, Matthias; Gruber, Uschi; Niquet, Sylvia; Popp, Oliver; Kroll, Friedrich; Sefkow, Michael; Schuelein, Ralf; Dreger, Mathias; Koester, Hubert</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1639-1649</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Unbiased chemoproteomic profiling of small-mol. interactions with endogenous proteins is important for drug discovery.  For meaningful results, all protein classes have to be tractable, including G protein-coupled receptors (GPCRs).  These receptors are hardly tractable by affinity pulldown from lysates.  We report a capture compd. (CC)-based strategy to target and identify GPCRs directly from living cells.  We synthesized CCs with sertindole attached to the CC scaffold in different orientations to target the dopamine D2 receptor (DRD2) heterologously expressed in HEK 293 cells.  The structure-activity relationship of sertindole for DRD2 binding was reflected in the activities of the sertindole CCs in radioligand displacement, cell-based assays, and capture compd. mass spectrometry (CCMS).  The activity pattern was rationalized by mol. modeling.  The most-active CC showed activities very similar to that of unmodified sertindole.  A concn. of DRD2 in living cells well below 100 fmol used as an exptl. input was sufficient for unambiguous identification of captured DRD2 by mass spectrometry.  Our new CCMS workflow broadens the arsenal of chemoproteomic technologies to close a crit. gap for the comprehensive characterization of drug-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi26C37KrfrrVg90H21EOLACvtfcHk0licgyPTaydlZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFaqtb7F&md5=364e3679dce522cd7294e762a79748c9</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201700152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201700152%26sid%3Dliteratum%253Aachs%26aulast%3DBlex%26aufirst%3DC.%26aulast%3DMichaelis%26aufirst%3DS.%26aulast%3DSchrey%26aufirst%3DA.%2BK.%26aulast%3DFurkert%26aufirst%3DJ.%26aulast%3DEichhorst%26aufirst%3DJ.%26aulast%3DBartho%26aufirst%3DK.%26aulast%3DGyapon%2BQuast%26aufirst%3DF.%26aulast%3DMarais%26aufirst%3DA.%26aulast%3DHakelberg%26aufirst%3DM.%26aulast%3DGruber%26aufirst%3DU.%26aulast%3DNiquet%26aufirst%3DS.%26aulast%3DPopp%26aufirst%3DO.%26aulast%3DKroll%26aufirst%3DF.%26aulast%3DSefkow%26aufirst%3DM.%26aulast%3DSchulein%26aufirst%3DR.%26aulast%3DDreger%26aufirst%3DM.%26aulast%3DKoster%26aufirst%3DH.%26atitle%3DTargeting%2520G%2520Protein-coupled%2520receptors%2520by%2520capture%2520compound%2520mass%2520spectrometry%253A%2520A%2520case%2520study%2520with%2520Sertindole%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D1639%26epage%3D1649%26doi%3D10.1002%2Fcbic.201700152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sosnovtsev, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval-Jaime, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parra, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tin, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, K. Y.</span></span> <span> </span><span class="NLM_article-title">Identification of human junctional adhesion molecule 1 as a functional receptor for the Hom-1 calicivirus on human cells</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e00031-17</span>, <span class="refDoi"> DOI: 10.1128/mBio.00031-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1128%2FmBio.00031-17" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=e00031-17&author=S.+V.+Sosnovtsevauthor=C.+Sandoval-Jaimeauthor=G.+I.+Parraauthor=C.+M.+Tinauthor=R.+W.+Jonesauthor=J.+Sodenauthor=D.+Barnesauthor=J.+Freethauthor=A.+W.+Smithauthor=K.+Y.+Green&title=Identification+of+human+junctional+adhesion+molecule+1+as+a+functional+receptor+for+the+Hom-1+calicivirus+on+human+cells&doi=10.1128%2FmBio.00031-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1128%2FmBio.00031-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.00031-17%26sid%3Dliteratum%253Aachs%26aulast%3DSosnovtsev%26aufirst%3DS.%2BV.%26aulast%3DSandoval-Jaime%26aufirst%3DC.%26aulast%3DParra%26aufirst%3DG.%2BI.%26aulast%3DTin%26aufirst%3DC.%2BM.%26aulast%3DJones%26aufirst%3DR.%2BW.%26aulast%3DSoden%26aufirst%3DJ.%26aulast%3DBarnes%26aufirst%3DD.%26aulast%3DFreeth%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DA.%2BW.%26aulast%3DGreen%26aufirst%3DK.%2BY.%26atitle%3DIdentification%2520of%2520human%2520junctional%2520adhesion%2520molecule%25201%2520as%2520a%2520functional%2520receptor%2520for%2520the%2520Hom-1%2520calicivirus%2520on%2520human%2520cells%26jtitle%3DmBio%26date%3D2017%26volume%3D8%26spage%3De00031-17%26doi%3D10.1128%2FmBio.00031-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavstsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avril, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brazier, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jespersen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magistrado, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lusingu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theander, T. G.</span></span> <span> </span><span class="NLM_article-title">Severe malaria is associated with parasite binding to endothelial protein C receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>498</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">505</span>, <span class="refDoi"> DOI: 10.1038/nature12216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnature12216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23739325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXovVSrt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=498&publication_year=2013&pages=502-505&author=L.+Turnerauthor=T.+Lavstsenauthor=S.+S.+Bergerauthor=C.+W.+Wangauthor=J.+E.+Petersenauthor=M.+Avrilauthor=A.+J.+Brazierauthor=J.+Freethauthor=J.+S.+Jespersenauthor=M.+A.+Nielsenauthor=P.+Magistradoauthor=J.+Lusinguauthor=J.+D.+Smithauthor=M.+K.+Higginsauthor=T.+G.+Theander&title=Severe+malaria+is+associated+with+parasite+binding+to+endothelial+protein+C+receptor&doi=10.1038%2Fnature12216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Severe malaria is associated with parasite binding to endothelial protein C receptor</span></div><div class="casAuthors">Turner, Louise; Lavstsen, Thomas; Berger, Sanne S.; Wang, Christian W.; Petersen, Jens E. V.; Avril, Marion; Brazier, Andrew J.; Freeth, Jim; Jespersen, Jakob S.; Nielsen, Morten A.; Magistrado, Pamela; Lusingu, John; Smith, Joseph D.; Higgins, Matthew K.; Theander, Thor G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">498</span>
        (<span class="NLM_cas:issue">7455</span>),
    <span class="NLM_cas:pages">502-505</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sequestration of Plasmodium falciparum-infected erythrocytes in host blood vessels is a key triggering event in the pathogenesis of severe childhood malaria, which is responsible for about one million deaths every year.  Sequestration is mediated by specific interactions between members of the P. falciparum erythrocyte membrane protein 1 (PfEMP1) family and receptors on the endothelial lining.  Severe childhood malaria is assocd. with expression of specific PfEMP1 subtypes contg. domain cassettes (DCs) 8 and 13 (ref. 3), but the endothelial receptor for parasites expressing these proteins was unknown.  Here we identify endothelial protein C receptor (EPCR), which mediates the cytoprotective effects of activated protein C, as the endothelial receptor for DC8 and DC13 PfEMP1.  We show that EPCR binding is mediated through the amino-terminal cysteine-rich interdomain region (CIDRα1) of DC8 and group A PfEMP1 subfamilies, and that CIDRα1 interferes with protein C binding to EPCR.  This PfEMP1 adhesive property links P. falciparum cytoadhesion to a host receptor involved in anticoagulation and endothelial cytoprotective pathways, and has implications for understanding malaria pathol. and the development of new malaria interventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9Vvb39EmdnbVg90H21EOLACvtfcHk0lj4yLtln-u6rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXovVSrt7w%253D&md5=733b581c14de51378d761b25e133d45d</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1038%2Fnature12216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12216%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DL.%26aulast%3DLavstsen%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BS.%26aulast%3DWang%26aufirst%3DC.%2BW.%26aulast%3DPetersen%26aufirst%3DJ.%2BE.%26aulast%3DAvril%26aufirst%3DM.%26aulast%3DBrazier%26aufirst%3DA.%2BJ.%26aulast%3DFreeth%26aufirst%3DJ.%26aulast%3DJespersen%26aufirst%3DJ.%2BS.%26aulast%3DNielsen%26aufirst%3DM.%2BA.%26aulast%3DMagistrado%26aufirst%3DP.%26aulast%3DLusingu%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DM.%2BK.%26aulast%3DTheander%26aufirst%3DT.%2BG.%26atitle%3DSevere%2520malaria%2520is%2520associated%2520with%2520parasite%2520binding%2520to%2520endothelial%2520protein%2520C%2520receptor%26jtitle%3DNature%26date%3D2013%26volume%3D498%26spage%3D502%26epage%3D505%26doi%3D10.1038%2Fnature12216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrichsen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandercock, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Munoz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmentier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovanovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyland, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lamki, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leishman, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rust, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jermutus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valge-Archer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, R. W.</span></span> <span> </span><span class="NLM_article-title">Phenotypic screening reveals TNFR2 as a promising target for cancer immunotherapy</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">68278</span>– <span class="NLM_lpage">68291</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.11943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.18632%2Foncotarget.11943" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=68278-68291&author=G.+S.+Williamsauthor=B.+Mistryauthor=S.+Guillardauthor=J.+C.+Ulrichsenauthor=A.+M.+Sandercockauthor=J.+Wangauthor=A.+Gonzalez-Munozauthor=J.+Parmentierauthor=C.+Blackauthor=J.+Sodenauthor=J.+Freethauthor=J.+Jovanovicauthor=R.+Leylandauthor=R.+S.+Al-Lamkiauthor=A.+J.+Leishmanauthor=S.+J.+Rustauthor=R.+Stewartauthor=L.+Jermutusauthor=J.+R.+Bradleyauthor=V.+Bedianauthor=V.+Valge-Archerauthor=R.+Minterauthor=R.+W.+Wilkinson&title=Phenotypic+screening+reveals+TNFR2+as+a+promising+target+for+cancer+immunotherapy&doi=10.18632%2Foncotarget.11943"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.11943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.11943%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DG.%2BS.%26aulast%3DMistry%26aufirst%3DB.%26aulast%3DGuillard%26aufirst%3DS.%26aulast%3DUlrichsen%26aufirst%3DJ.%2BC.%26aulast%3DSandercock%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGonzalez-Munoz%26aufirst%3DA.%26aulast%3DParmentier%26aufirst%3DJ.%26aulast%3DBlack%26aufirst%3DC.%26aulast%3DSoden%26aufirst%3DJ.%26aulast%3DFreeth%26aufirst%3DJ.%26aulast%3DJovanovic%26aufirst%3DJ.%26aulast%3DLeyland%26aufirst%3DR.%26aulast%3DAl-Lamki%26aufirst%3DR.%2BS.%26aulast%3DLeishman%26aufirst%3DA.%2BJ.%26aulast%3DRust%26aufirst%3DS.%2BJ.%26aulast%3DStewart%26aufirst%3DR.%26aulast%3DJermutus%26aufirst%3DL.%26aulast%3DBradley%26aufirst%3DJ.%2BR.%26aulast%3DBedian%26aufirst%3DV.%26aulast%3DValge-Archer%26aufirst%3DV.%26aulast%3DMinter%26aufirst%3DR.%26aulast%3DWilkinson%26aufirst%3DR.%2BW.%26atitle%3DPhenotypic%2520screening%2520reveals%2520TNFR2%2520as%2520a%2520promising%2520target%2520for%2520cancer%2520immunotherapy%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D68278%26epage%3D68291%26doi%3D10.18632%2Foncotarget.11943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sandercock, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rust, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachsenmeier, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holoweckyj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herpers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jermutus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollingsworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minter, R.</span></span> <span> </span><span class="NLM_article-title">Identification of anti-tumour biologics using primary tumour models, 3-D phenotypic screening and image-based multi-parametric profiling</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">147</span>, <span class="refDoi"> DOI: 10.1186/s12943-015-0415-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1186%2Fs12943-015-0415-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26227951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FpsFWjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=147&author=A.+M.+Sandercockauthor=S.+Rustauthor=S.+Guillardauthor=K.+F.+Sachsenmeierauthor=N.+Holoweckyjauthor=C.+Hayauthor=M.+Flynnauthor=Q.+Huangauthor=K.+Yanauthor=B.+Herpersauthor=L.+S.+Priceauthor=J.+Sodenauthor=J.+Freethauthor=L.+Jermutusauthor=R.+Hollingsworthauthor=R.+Minter&title=Identification+of+anti-tumour+biologics+using+primary+tumour+models%2C+3-D+phenotypic+screening+and+image-based+multi-parametric+profiling&doi=10.1186%2Fs12943-015-0415-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of anti-tumour biologics using primary tumour models, 3-D phenotypic screening and image-based multi-parametric profiling</span></div><div class="casAuthors">Sandercock Alan M; Rust Steven; Guillard Sandrine; Jermutus Lutz; Minter Ralph; Sachsenmeier Kris F; Holoweckyj Nick; Hay Carl; Flynn Matt; Huang Qihui; Hollingsworth Robert; Yan Kuan; Herpers Bram; Price Leo S; Soden Jo; Freeth Jim</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">147</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Monolayer cultures of immortalised cell lines are a popular screening tool for novel anti-cancer therapeutics, but these methods can be a poor surrogate for disease states, and there is a need for drug screening platforms which are more predictive of clinical outcome.  In this study, we describe a phenotypic antibody screen using three-dimensional cultures of primary cells, and image-based multi-parametric profiling in PC-3 cells, to identify anti-cancer biologics against new therapeutic targets.  METHODS:  ScFv Antibodies and designed ankyrin repeat proteins (DARPins) were isolated using phage display selections against primary non-small cell lung carcinoma cells.  The selected molecules were screened for anti-proliferative and pro-apoptotic activity against primary cells grown in three-dimensional culture, and in an ultra-high content screen on a 3-D cultured cell line using multi-parametric profiling to detect treatment-induced phenotypic changes.  The targets of molecules of interest were identified using a cell-surface membrane protein array.  An anti-CUB domain containing protein 1 (CDCP1) antibody was tested for tumour growth inhibition in a patient-derived xenograft model, generated from a stage-IV non-small cell lung carcinoma, with and without cisplatin.  RESULTS:  Two primary non-small cell lung carcinoma cell models were established for antibody isolation and primary screening in anti-proliferative and apoptosis assays.  These assays identified multiple antibodies demonstrating activity in specific culture formats.  A subset of the DARPins was profiled in an ultra-high content multi-parametric screen, where 300 morphological features were measured per sample.  Machine learning was used to select features to classify treatment responses, then antibodies were characterised based on the phenotypes that they induced.  This method co-classified several DARPins that targeted CDCP1 into two sets with different phenotypes.  Finally, an anti-CDCP1 antibody significantly enhanced the efficacy of cisplatin in a patient-derived NSCLC xenograft model.  CONCLUSIONS:  Phenotypic profiling using complex 3-D cell cultures steers hit selection towards more relevant in vivo phenotypes, and may shed light on subtle mechanistic variations in drug candidates, enabling data-driven decisions for oncology target validation.  CDCP1 was identified as a potential target for cisplatin combination therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9qh7axt418RpyrDyM_m9cfW6udTcc2ebg9uwWjohE9bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FpsFWjtg%253D%253D&md5=9f6fa8799c7febcb8a9900c59729a0ab</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1186%2Fs12943-015-0415-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-015-0415-0%26sid%3Dliteratum%253Aachs%26aulast%3DSandercock%26aufirst%3DA.%2BM.%26aulast%3DRust%26aufirst%3DS.%26aulast%3DGuillard%26aufirst%3DS.%26aulast%3DSachsenmeier%26aufirst%3DK.%2BF.%26aulast%3DHoloweckyj%26aufirst%3DN.%26aulast%3DHay%26aufirst%3DC.%26aulast%3DFlynn%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DYan%26aufirst%3DK.%26aulast%3DHerpers%26aufirst%3DB.%26aulast%3DPrice%26aufirst%3DL.%2BS.%26aulast%3DSoden%26aufirst%3DJ.%26aulast%3DFreeth%26aufirst%3DJ.%26aulast%3DJermutus%26aufirst%3DL.%26aulast%3DHollingsworth%26aufirst%3DR.%26aulast%3DMinter%26aufirst%3DR.%26atitle%3DIdentification%2520of%2520anti-tumour%2520biologics%2520using%2520primary%2520tumour%2520models%252C%25203-D%2520phenotypic%2520screening%2520and%2520image-based%2520multi-parametric%2520profiling%26jtitle%3DMol.%2520Cancer%26date%3D2015%26volume%3D14%26spage%3D147%26doi%3D10.1186%2Fs12943-015-0415-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Identification of human UMP/CMP Kinase 1 as Doxorubicin binding target using protein microarray</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1015</span>, <span class="refDoi"> DOI: 10.1177/2472555217707704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F2472555217707704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28459633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKkt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1007-1015&author=S.+Chenauthor=X.+Wangauthor=X.+Yeauthor=D.+Maauthor=C.+Chenauthor=J.+Caiauthor=Y.+Fuauthor=X.+Chengauthor=Y.+Chenauthor=X.+Gongauthor=J.+Jin&title=Identification+of+human+UMP%2FCMP+Kinase+1+as+Doxorubicin+binding+target+using+protein+microarray&doi=10.1177%2F2472555217707704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Human UMP/CMP Kinase 1 as Doxorubicin Binding Target Using Protein Microarray</span></div><div class="casAuthors">Chen, Shuxian; Wang, Xu; Ye, Xianghui; Ma, Donghui; Chen, Caiwei; Cai, Junlong; Fu, Yongfeng; Cheng, Xunjia; Chen, Yun; Gong, Xiaohai; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1007-1015</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Doxorubicin (DOX) is a leading anthracycline drug with exceptional efficacy; however, little is known about the mol. mechanisms of its side effects, which include heart muscle damage, noncancerous cell death, and drug resistance.  A total of 17,950 human proteins expressed in HEK293 cells were screened and yielded 14 hits.  Competitive and binding expts. further verified the binding of DOX to UMP/CMP kinase 1 (CMPK1), and microscale thermophoresis showed that DOX binds to CMPK1 with a Kd of 1216 nM.  In addn., we obsd. that the binding of DOX to CMPK1 activated the phosphorylation of CMP, dCMP, and UMP.  A significant activation was obsd. at the concn. of 30 μM DOX and reached plateau at the concn. of DOX 30 μM, 150 μM, and 100 μM, resp.  DOX would add up stimulation of CMPK1 by DTT and overcome inhibition of CMPK1 by NaF, EDTA.  In summary, we showed that DOX might bind to the nonactive site of CMPK1 and regulate its activity with magnesium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGJWa1yckkXbVg90H21EOLACvtfcHk0ljgXzngVblHLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKkt7o%253D&md5=5fcbc3d47d6ae6bff18c2f7f7adec665</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1177%2F2472555217707704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217707704%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520human%2520UMP%252FCMP%2520Kinase%25201%2520as%2520Doxorubicin%2520binding%2520target%2520using%2520protein%2520microarray%26jtitle%3DSLAS%2520Discov%26date%3D2017%26volume%3D22%26spage%3D1007%26epage%3D1015%26doi%3D10.1177%2F2472555217707704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bassilana, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DaSilva, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosshans, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilmeringwetter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llamas, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigollier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourret, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramamurthy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbinski, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plonsky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruffner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sailer, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouwmeester, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finan, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelleher, J. F.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luchansky, S. J.</span></span> <span> </span><span class="NLM_article-title">Target identification for a Hedgehog pathway inhibitor reveals the receptor GPR39</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">349</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.1481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24633354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlGrt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=343-349&author=F.+Bassilanaauthor=A.+Carlsonauthor=J.+A.+DaSilvaauthor=B.+Grosshansauthor=S.+Vidalauthor=V.+Beckauthor=B.+Wilmeringwetterauthor=L.+A.+Llamasauthor=T.+B.+Showalterauthor=P.+Rigollierauthor=A.+Bourretauthor=A.+Ramamurthyauthor=X.+Wuauthor=F.+Harbinskiauthor=S.+Plonskyauthor=L.+Leeauthor=H.+Ruffnerauthor=P.+Grandiauthor=M.+Schirleauthor=J.+Jenkinsauthor=A.+W.+Sailerauthor=T.+Bouwmeesterauthor=J.+A.+Porterauthor=V.+Myerauthor=P.+M.+Finanauthor=J.+A.+Tallaricoauthor=J.+F.+Kelleherauthor=K.+Seuwenauthor=R.+K.+Jainauthor=S.+J.+Luchansky&title=Target+identification+for+a+Hedgehog+pathway+inhibitor+reveals+the+receptor+GPR39&doi=10.1038%2Fnchembio.1481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Target identification for a Hedgehog pathway inhibitor reveals the receptor GPR39</span></div><div class="casAuthors">Bassilana, Frederic; Carlson, Adam; Da Silva, Jennifer A.; Grosshans, Bianka; Vidal, Solange; Beck, Valerie; Wilmeringwetter, Barbara; Llamas, Luis A.; Showalter, Todd B.; Rigollier, Pascal; Bourret, Aaron; Ramamurthy, Arun; Wu, Xu; Harbinski, Fred; Plonsky, Samantha; Lee, Lac; Ruffner, Heinz; Grandi, Paola; Schirle, Markus; Jenkins, Jeremy; Sailer, Andreas W.; Bouwmeester, Tewis; Porter, Jeffrey A.; Myer, Vic; Finan, Peter M.; Tallarico, John A.; Kelleher, Joseph F., III; Seuwen, Klaus; Jain, Rishi K.; Luchansky, Sarah J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hedgehog (Hh) signaling dets. cell fate during development and can drive tumorigenesis.  We performed a screen for new compds. that can impinge on Hh signaling downstream of Smoothened (Smo).  A series of cyclohexyl-Me aminopyrimidine chemotype compds. ('CMAPs') were identified that could block pathway signaling in a Smo-independent manner.  In addn. to inhibiting Hh signaling, the compds. generated inositol phosphates through an unknown GPCR.  Correlation of GPCR mRNA expression levels with compd. activity across cell lines suggested the target to be the orphan receptor GPR39.  RNA interference or cDNA overexpression of GPR39 demonstrated that the receptor is necessary for compd. activity.  We propose a model in which CMAPs activate GPR39, which signals to the Gli transcription factors and blocks signaling.  In addn. to the discovery of GPR39 as a new target that impinges on Hh signaling, we report on small-mol. modulators of the receptor that will enable in vitro interrogation of GPR39 signaling in different cellular contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpauWdqJMnkc7Vg90H21EOLACvtfcHk0ljgXzngVblHLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlGrt7c%253D&md5=1b5bc844ae5698e1e69578eb6ac053cd</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1481%26sid%3Dliteratum%253Aachs%26aulast%3DBassilana%26aufirst%3DF.%26aulast%3DCarlson%26aufirst%3DA.%26aulast%3DDaSilva%26aufirst%3DJ.%2BA.%26aulast%3DGrosshans%26aufirst%3DB.%26aulast%3DVidal%26aufirst%3DS.%26aulast%3DBeck%26aufirst%3DV.%26aulast%3DWilmeringwetter%26aufirst%3DB.%26aulast%3DLlamas%26aufirst%3DL.%2BA.%26aulast%3DShowalter%26aufirst%3DT.%2BB.%26aulast%3DRigollier%26aufirst%3DP.%26aulast%3DBourret%26aufirst%3DA.%26aulast%3DRamamurthy%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHarbinski%26aufirst%3DF.%26aulast%3DPlonsky%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DL.%26aulast%3DRuffner%26aufirst%3DH.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%26aulast%3DSailer%26aufirst%3DA.%2BW.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DMyer%26aufirst%3DV.%26aulast%3DFinan%26aufirst%3DP.%2BM.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DKelleher%26aufirst%3DJ.%2BF.%26aulast%3DSeuwen%26aufirst%3DK.%26aulast%3DJain%26aufirst%3DR.%2BK.%26aulast%3DLuchansky%26aufirst%3DS.%2BJ.%26atitle%3DTarget%2520identification%2520for%2520a%2520Hedgehog%2520pathway%2520inhibitor%2520reveals%2520the%2520receptor%2520GPR39%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D343%26epage%3D349%26doi%3D10.1038%2Fnchembio.1481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnea, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strapps, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrada, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloss, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. J.</span></span> <span> </span><span class="NLM_article-title">The genetic design of signaling cascades to record receptor activation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1073/pnas.0710487105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.0710487105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=18165312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpvV2ktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=64-69&author=G.+Barneaauthor=W.+Strappsauthor=G.+Herradaauthor=Y.+Bermanauthor=J.+Ongauthor=B.+Klossauthor=R.+Axelauthor=K.+J.+Lee&title=The+genetic+design+of+signaling+cascades+to+record+receptor+activation&doi=10.1073%2Fpnas.0710487105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">The genetic design of signaling cascades to record receptor activation</span></div><div class="casAuthors">Barnea, Gilad; Strapps, Walter; Herrada, Gilles; Berman, Yemiliya; Ong, Jane; Kloss, Brian; Axel, Richard; Lee, Kevin J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-69</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The authors have developed an exptl. strategy to monitor protein interactions in a cell with a high degree of selectivity and sensitivity.  A transcription factor is tethered to a membrane-bound receptor with a linker that contains a cleavage site for a specific protease.  Activation of the receptor recruits a signaling protein fused to the protease that then cleaves and releases the transcription factor to activate reporter genes in the nucleus.  This strategy converts a transient interaction into a stable and amplifiable reporter gene signal to record the activation of a receptor without interference from endogenous signaling pathways.  The authors have developed this assay for three classes of receptors: G protein-coupled receptors, receptor tyrosine kinases, and steroid hormone receptors.  Finally, the authors use the assay to identify a ligand for the orphan receptor GPR1, suggesting a role for this receptor in the regulation of inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2ugB7pgEqlrVg90H21EOLACvtfcHk0lhCkdQajZmxBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpvV2ktA%253D%253D&md5=3b49dfdb2949d947e145084ac87bb343</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0710487105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0710487105%26sid%3Dliteratum%253Aachs%26aulast%3DBarnea%26aufirst%3DG.%26aulast%3DStrapps%26aufirst%3DW.%26aulast%3DHerrada%26aufirst%3DG.%26aulast%3DBerman%26aufirst%3DY.%26aulast%3DOng%26aufirst%3DJ.%26aulast%3DKloss%26aufirst%3DB.%26aulast%3DAxel%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DK.%2BJ.%26atitle%3DThe%2520genetic%2520design%2520of%2520signaling%2520cascades%2520to%2520record%2520receptor%2520activation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D64%26epage%3D69%26doi%3D10.1073%2Fpnas.0710487105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguere, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">369</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnsmb.3014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25895059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=362-369&author=W.+K.+Kroezeauthor=M.+F.+Sassanoauthor=X.+P.+Huangauthor=K.+Lansuauthor=J.+D.+McCorvyauthor=P.+M.+Giguereauthor=N.+Sciakyauthor=B.+L.+Roth&title=PRESTO-Tango+as+an+open-source+resource+for+interrogation+of+the+druggable+human+GPCRome&doi=10.1038%2Fnsmb.3014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span></div><div class="casAuthors">Kroeze, Wesley K.; Sassano, Maria F.; Huang, Xi-Ping; Lansu, Katherine; McCorvy, John D.; Giguere, Patrick M.; Sciaky, Noah; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">362-369</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are essential mediators of cellular signaling and are important targets of drug action.  Of the approx. 350 nonolfactory human GPCRs, more than 100 are still considered to be 'orphans' because their endogenous ligands remain unknown.  Here, we describe a unique open-source resource that allows interrogation of the druggable human GPCRome via a G protein-independent β-arrestin-recruitment assay.  We validate this unique platform at more than 120 nonorphan human GPCR targets, demonstrate its utility for discovering new ligands for orphan human GPCRs and describe a method (parallel receptorome expression and screening via transcriptional output, with transcriptional activation following arrestin translocation (PRESTO-Tango)) for the simultaneous and parallel interrogation of the entire human nonolfactory GPCRome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVbl7caV9gUrVg90H21EOLACvtfcHk0lhCkdQajZmxBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D&md5=2a78b74a5f1425fa9e8b79b730c16be6</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3014%26sid%3Dliteratum%253Aachs%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DGiguere%26aufirst%3DP.%2BM.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPRESTO-Tango%2520as%2520an%2520open-source%2520resource%2520for%2520interrogation%2520of%2520the%2520druggable%2520human%2520GPCRome%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D362%26epage%3D369%26doi%3D10.1038%2Fnsmb.3014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gee, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. E.</span></span> <span> </span><span class="NLM_article-title">Kirkwood-Buff theory of molecular and protein association, aggregation, and cellular crowding</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">165101</span>, <span class="refDoi"> DOI: 10.1063/1.3253299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1063%2F1.3253299" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=165101&author=M.+B.+Geeauthor=P.+E.+Smith&title=Kirkwood-Buff+theory+of+molecular+and+protein+association%2C+aggregation%2C+and+cellular+crowding&doi=10.1063%2F1.3253299"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1063%2F1.3253299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3253299%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DM.%2BB.%26aulast%3DSmith%26aufirst%3DP.%2BE.%26atitle%3DKirkwood-Buff%2520theory%2520of%2520molecular%2520and%2520protein%2520association%252C%2520aggregation%252C%2520and%2520cellular%2520crowding%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2009%26volume%3D131%26spage%3D165101%26doi%3D10.1063%2F1.3253299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bathula, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of novel 5-arylidenethiazolidinones with apoptotic properties via a three component reaction using piperidine as a bifunctional reagent</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">8053</span>– <span class="NLM_lpage">8063</span>, <span class="refDoi"> DOI: 10.1039/C6OB01257D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2FC6OB01257D" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=8053-8063&author=C.+Bathulaauthor=S.+Tripathiauthor=R.+Srinivasanauthor=K.+K.+Jhaauthor=A.+Ganguliauthor=G.+Chakrabartiauthor=S.+Singhauthor=P.+Munshiauthor=S.+Sen&title=Synthesis+of+novel+5-arylidenethiazolidinones+with+apoptotic+properties+via+a+three+component+reaction+using+piperidine+as+a+bifunctional+reagent&doi=10.1039%2FC6OB01257D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1039%2FC6OB01257D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6OB01257D%26sid%3Dliteratum%253Aachs%26aulast%3DBathula%26aufirst%3DC.%26aulast%3DTripathi%26aufirst%3DS.%26aulast%3DSrinivasan%26aufirst%3DR.%26aulast%3DJha%26aufirst%3DK.%2BK.%26aulast%3DGanguli%26aufirst%3DA.%26aulast%3DChakrabarti%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DMunshi%26aufirst%3DP.%26aulast%3DSen%26aufirst%3DS.%26atitle%3DSynthesis%2520of%2520novel%25205-arylidenethiazolidinones%2520with%2520apoptotic%2520properties%2520via%2520a%2520three%2520component%2520reaction%2520using%2520piperidine%2520as%2520a%2520bifunctional%2520reagent%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D8053%26epage%3D8063%26doi%3D10.1039%2FC6OB01257D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, S.</span></span> <span> </span><span class="NLM_article-title">Spiro[pyrrolidine-3, 3 -oxindole] as potent anti-breast cancer compounds: Their design, synthesis, biological evaluation and cellular target identification</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">32213</span>, <span class="refDoi"> DOI: 10.1038/srep32213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fsrep32213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27573798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCqt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=32213&author=S.+Hatiauthor=S.+Tripathyauthor=P.+K.+Duttaauthor=R.+Agarwalauthor=R.+Srinivasanauthor=A.+Singhauthor=S.+Singhauthor=S.+Sen&title=Spiro%5Bpyrrolidine-3%2C+3+-oxindole%5D+as+potent+anti-breast+cancer+compounds%3A+Their+design%2C+synthesis%2C+biological+evaluation+and+cellular+target+identification&doi=10.1038%2Fsrep32213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Spiro[pyrrolidine-3,3'-oxindole] as potent anti-breast cancer compounds: Their design, synthesis, biological evaluation and cellular target identification</span></div><div class="casAuthors">Hati, Santanu; Tripathy, Sayantan; Dutta, Pratip Kumar; Agarwal, Rahul; Srinivasan, Ramprasad; Singh, Ashutosh; Singh, Shailja; Sen, Subhabrata</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32213pp.</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design and synthesis of a library of compds. bearing spiro[pyrrolidine-3, 3'-oxindole] motifs I (Ar = C6H5, 4-CH3C6H4, thiophen-2-yl, etc.) that demonstrated exceptional inhibitory activity against the proliferation of MCF-7 breast cancer cells has been reported.  The synthesis involved a one pot Pictet Spengler-Oxidative ring contraction of tryptamine to the desired scaffolds and occurred in 1:1 THF and water with catalytic trifluoroacetic acid and stoichiometric N-bromosuccinimide as an oxidant.  Phenotypic profiling indicated that these mols. induce apoptotic cell death in MCF-7 cells.  Target deconvolution with most potent compd. I (Ar = C6H5) from the library, using chem. proteomics indicated histone deacetylase 2 (HDAC2) and prohibitin 2 as the potential cellular binding partners.  Mol. docking of compd. I (Ar = C6H5) with HDAC2 provided insights pertinent to putative binding interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo95MMqBQ5m37Vg90H21EOLACvtfcHk0lhmvQ7JIxqpoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCqt7vK&md5=a39920c7c7d8f6f3fe9eb7bda65ecc7f</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1038%2Fsrep32213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep32213%26sid%3Dliteratum%253Aachs%26aulast%3DHati%26aufirst%3DS.%26aulast%3DTripathy%26aufirst%3DS.%26aulast%3DDutta%26aufirst%3DP.%2BK.%26aulast%3DAgarwal%26aufirst%3DR.%26aulast%3DSrinivasan%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DSen%26aufirst%3DS.%26atitle%3DSpiro%255Bpyrrolidine-3%252C%25203%2520-oxindole%255D%2520as%2520potent%2520anti-breast%2520cancer%2520compounds%253A%2520Their%2520design%252C%2520synthesis%252C%2520biological%2520evaluation%2520and%2520cellular%2520target%2520identification%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D32213%26doi%3D10.1038%2Fsrep32213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franken, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweetman, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gade, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhard, F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span> <span> </span><span class="NLM_article-title">Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1567</span>– <span class="NLM_lpage">1593</span>, <span class="refDoi"> DOI: 10.1038/nprot.2015.101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnprot.2015.101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26379230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1567-1593&author=H.+Frankenauthor=T.+Mathiesonauthor=D.+Childsauthor=G.+M.+Sweetmanauthor=T.+Wernerauthor=I.+Togelauthor=C.+Doceauthor=S.+Gadeauthor=M.+Bantscheffauthor=G.+Drewesauthor=F.+B.+Reinhardauthor=W.+Huberauthor=M.+M.+Savitski&title=Thermal+proteome+profiling+for+unbiased+identification+of+direct+and+indirect+drug+targets+using+multiplexed+quantitative+mass+spectrometry&doi=10.1038%2Fnprot.2015.101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry</span></div><div class="casAuthors">Franken, Holger; Mathieson, Toby; Childs, Dorothee; Sweetman, Gavain M. A.; Werner, Thilo; Toegel, Ina; Doce, Carola; Gade, Stephan; Bantscheff, Marcus; Drewes, Gerard; Reinhard, Friedrich B. M.; Huber, Wolfgang; Savitski, Mikhail M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1567-1593</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The direct detection of drug-protein interactions in living cells is a major challenge in drug discovery research.  Recently, the authors introduced an approach termed thermal proteome profiling (TPP), which enables the monitoring of changes in protein thermal stability across the proteome using quant. mass spectrometry.  The authors detd. the intracellular thermal profiles for up to 7000 proteins, and by comparing profiles derived from cultured mammalian cells in the presence or absence of a drug it was possible to identify direct and indirect targets of drugs in living cells in an unbiased manner.  Here the authors demonstrate the complete workflow using the histone deacetylase inhibitor panobinostat.  The key to this approach is the use of isobaric tandem mass tag 10-plex (TMT10) reagents to label digested protein samples corresponding to each temp. point in the melting curve so that the samples can be analyzed by multiplexed quant. mass spectrometry.  Important steps in the bioinformatic anal. include data normalization, melting curve fitting and statistical significance detn. of compd. concn.-dependent changes in protein stability.  All anal. tools are made freely available as R and Python packages.  The workflow can be completed in 2 wk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS4LU9YprZqLVg90H21EOLACvtfcHk0lhz_dIevkD6CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzN&md5=d9c225616a35fc5dcc97db234452d7e1</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.101%26sid%3Dliteratum%253Aachs%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DChilds%26aufirst%3DD.%26aulast%3DSweetman%26aufirst%3DG.%2BM.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DTogel%26aufirst%3DI.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DGade%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DReinhard%26aufirst%3DF.%2BB.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26atitle%3DThermal%2520proteome%2520profiling%2520for%2520unbiased%2520identification%2520of%2520direct%2520and%2520indirect%2520drug%2520targets%2520using%2520multiplexed%2520quantitative%2520mass%2520spectrometry%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D1567%26epage%3D1593%26doi%3D10.1038%2Fnprot.2015.101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olek, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1055</span>– <span class="NLM_lpage">1057</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnmeth.3590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26389571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1055-1057&author=K.+V.+Huberauthor=K.+M.+Olekauthor=A.+C.+Mullerauthor=C.+S.+Tanauthor=K.+L.+Bennettauthor=J.+Colingeauthor=G.+Superti-Furga&title=Proteome-wide+drug+and+metabolite+interaction+mapping+by+thermal-stability+profiling&doi=10.1038%2Fnmeth.3590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span></div><div class="casAuthors">Huber, Kilian V. M.; Olek, Karin M.; Mueller, Andre C.; Tan, Chris Soon Heng; Bennett, Keiryn L.; Colinge, Jacques; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1055-1057</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal stabilization of proteins after ligand binding provides an efficient means to assess the binding of small mols. to proteins.  We show here that in combination with quant. mass spectrometry, the approach allows for the systematic survey of protein engagement by cellular metabolites and drugs.  We profiled the targets of the drugs methotrexate and (S)-crizotinib and the metabolite 2'3'-cGAMP in intact cells and identified the 2'3'-cGAMP cognate transmembrane receptor STING, involved in immune signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrow66ugdg1oLVg90H21EOLACvtfcHk0lhz_dIevkD6CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI&md5=619a2c08c4f151f4e3ab8bd8fc45c2b6</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3590%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DOlek%26aufirst%3DK.%2BM.%26aulast%3DMuller%26aufirst%3DA.%2BC.%26aulast%3DTan%26aufirst%3DC.%2BS.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DProteome-wide%2520drug%2520and%2520metabolite%2520interaction%2520mapping%2520by%2520thermal-stability%2520profiling%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1055%26epage%3D1057%26doi%3D10.1038%2Fnmeth.3590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reinhard, F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franken, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span> <span> </span><span class="NLM_article-title">Thermal proteome profiling monitors ligand interactions with cellular membrane proteins</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1129</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3652</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnmeth.3652" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26524241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1129-1131&author=F.+B.+Reinhardauthor=D.+Eberhardauthor=T.+Wernerauthor=H.+Frankenauthor=D.+Childsauthor=C.+Doceauthor=M.+F.+Savitskiauthor=W.+Huberauthor=M.+Bantscheffauthor=M.+M.+Savitskiauthor=G.+Drewes&title=Thermal+proteome+profiling+monitors+ligand+interactions+with+cellular+membrane+proteins&doi=10.1038%2Fnmeth.3652"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal proteome profiling monitors ligand interactions with cellular membrane proteins</span></div><div class="casAuthors">Reinhard, Friedrich B. M.; Eberhard, Dirk; Werner, Thilo; Franken, Holger; Childs, Dorothee; Doce, Carola; Savitski, Maria Faelth; Huber, Wolfgang; Bantscheff, Marcus; Savitski, Mikhail M.; Drewes, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1129-1131</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We extended thermal proteome profiling to detect transmembrane protein-small mol. interactions in cultured human cells.  When we assessed the effects of detergents on ATP-binding profiles, we obsd. shifts in denaturation temp. for ATP-binding transmembrane proteins.  We also obsd. cellular thermal shifts in pervanadate-induced T cell-receptor signaling, delineating the membrane target CD45 and components of the downstream pathway, and with drugs affecting the transmembrane transporters ATP1A1 and MDR1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo45i1e8jbFF7Vg90H21EOLACvtfcHk0lhz_dIevkD6CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7fO&md5=ec4651a2753f7969b16811faa8affa23</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3652&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3652%26sid%3Dliteratum%253Aachs%26aulast%3DReinhard%26aufirst%3DF.%2BB.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DChilds%26aufirst%3DD.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BF.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DThermal%2520proteome%2520profiling%2520monitors%2520ligand%2520interactions%2520with%2520cellular%2520membrane%2520proteins%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1129%26epage%3D1131%26doi%3D10.1038%2Fnmeth.3652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhard, F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franken, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovega, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span> <span> </span><span class="NLM_article-title">Tracking cancer drugs in living cells by thermal profiling of the proteome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span>,  <span class="NLM_fpage">1255784</span>, <span class="refDoi"> DOI: 10.1126/science.1255784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1255784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25278616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC2M7ntVKgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=1255784&author=M.+M.+Savitskiauthor=F.+B.+Reinhardauthor=H.+Frankenauthor=T.+Wernerauthor=M.+F.+Savitskiauthor=D.+Eberhardauthor=D.+Martinez+Molinaauthor=R.+Jafariauthor=R.+B.+Dovegaauthor=S.+Klaegerauthor=B.+Kusterauthor=P.+Nordlundauthor=M.+Bantscheffauthor=G.+Drewes&title=Tracking+cancer+drugs+in+living+cells+by+thermal+profiling+of+the+proteome&doi=10.1126%2Fscience.1255784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Tracking cancer drugs in living cells by thermal profiling of the proteome</span></div><div class="casAuthors">Savitski Mikhail M; Reinhard Friedrich B M; Franken Holger; Werner Thilo; Savitski Maria Falth; Eberhard Dirk; Bantscheff Marcus; Drewes Gerard; Martinez Molina Daniel; Jafari Rozbeh; Dovega Rebecca Bakszt; Klaeger Susan; Kuster Bernhard; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6205</span>),
    <span class="NLM_cas:pages">1255784</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The thermal stability of proteins can be used to assess ligand binding in living cells.  We have generalized this concept by determining the thermal profiles of more than 7000 proteins in human cells by means of mass spectrometry.  Monitoring the effects of small-molecule ligands on the profiles delineated more than 50 targets for the kinase inhibitor staurosporine.  We identified the heme biosynthesis enzyme ferrochelatase as a target of kinase inhibitors and suggest that its inhibition causes the phototoxicity observed with vemurafenib and alectinib.  Thermal shifts were also observed for downstream effectors of drug treatment.  In live cells, dasatinib induced shifts in BCR-ABL pathway proteins, including CRK/CRKL.  Thermal proteome profiling provides an unbiased measure of drug-target engagement and facilitates identification of markers for drug efficacy and toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS0ur3d8ba-Bu2mnZh2OgS0fW6udTcc2eZIvswJyIssYrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7ntVKgsg%253D%253D&md5=493a89ad107e656e0f0786ff5c8e5f63</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1126%2Fscience.1255784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1255784%26sid%3Dliteratum%253Aachs%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DReinhard%26aufirst%3DF.%2BB.%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DSavitski%26aufirst%3DM.%2BF.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DDovega%26aufirst%3DR.%2BB.%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DTracking%2520cancer%2520drugs%2520in%2520living%2520cells%2520by%2520thermal%2520profiling%2520of%2520the%2520proteome%26jtitle%3DScience%26date%3D2014%26volume%3D346%26spage%3D1255784%26doi%3D10.1126%2Fscience.1255784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundback, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundbackauthor=P.+Nordlundauthor=D.+Martinez+Molina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0li9J_Jsf6ENkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundback%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+Martinez+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span></div><div class="casAuthors">Martinez Molina Daniel; Jafari Rozbeh; Ignatushchenko Marina; Seki Takahiro; Larsson E Andreas; Dan Chen; Sreekumar Lekshmy; Cao Yihai; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophysical principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clinical targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRRn5K_mF1rcrHHuJhbyicZfW6udTcc2eYUHfshGhHA67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D&md5=a73bb49adb0d4f445223ea6f74fe3c10</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaal, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkers, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzpatrick, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span> <span> </span><span class="NLM_article-title">Thermal profiling reveals phenylalanine hydroxylase as an off-target of panobinostat</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">908</span>– <span class="NLM_lpage">910</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnchembio.2185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27669419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOmsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=908-910&author=I.+Becherauthor=T.+Wernerauthor=C.+Doceauthor=E.+A.+Zaalauthor=I.+Togelauthor=C.+A.+Khanauthor=A.+Ruegerauthor=M.+Muelbaierauthor=E.+Salzerauthor=C.+R.+Berkersauthor=P.+F.+Fitzpatrickauthor=M.+Bantscheffauthor=M.+M.+Savitski&title=Thermal+profiling+reveals+phenylalanine+hydroxylase+as+an+off-target+of+panobinostat&doi=10.1038%2Fnchembio.2185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal profiling reveals phenylalanine hydroxylase as an off-target of panobinostat</span></div><div class="casAuthors">Becher, Isabelle; Werner, Thilo; Doce, Carola; Zaal, Esther A.; Toegel, Ina; Khan, Crystal A.; Rueger, Anne; Muelbaier, Marcel; Salzer, Elsa; Berkers, Celia R.; Fitzpatrick, Paul F.; Bantscheff, Marcus; Savitski, Mikhail M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">908-910</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe a two-dimensional thermal proteome profiling strategy that can be combined with an orthogonal chemoproteomics approach to enable comprehensive target profiling of the marketed histone deacetylase inhibitor panobinostat.  The N-hydroxycinnamide moiety is identified as crit. for potent and tetrahydrobiopterin-competitive inhibition of phenylalanine hydroxylase leading to increases in phenylalanine and decreases in tyrosine levels.  These findings provide a rationale for adverse clin. observations and suggest repurposing of the drug for treatment of tyrosinemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ75ESSwULbbVg90H21EOLACvtfcHk0lhL5x1Wl6NdTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOmsLrJ&md5=f08addd4f25749aebd827dd99edf157b</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2185%26sid%3Dliteratum%253Aachs%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DZaal%26aufirst%3DE.%2BA.%26aulast%3DTogel%26aufirst%3DI.%26aulast%3DKhan%26aufirst%3DC.%2BA.%26aulast%3DRueger%26aufirst%3DA.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DSalzer%26aufirst%3DE.%26aulast%3DBerkers%26aufirst%3DC.%2BR.%26aulast%3DFitzpatrick%26aufirst%3DP.%2BF.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26atitle%3DThermal%2520profiling%2520reveals%2520phenylalanine%2520hydroxylase%2520as%2520an%2520off-target%2520of%2520panobinostat%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D908%26epage%3D910%26doi%3D10.1038%2Fnchembio.2185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St. Onge, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proctor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nile, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spagnuolo, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jitkova, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gronda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung-Ong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spear, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlecht, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duby, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heisler, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surendra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbanus, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebbia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lissina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeghouf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherfils, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trimble, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimmer, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankaitis, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nislow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaever, G.</span></span> <span> </span><span class="NLM_article-title">Mapping the cellular response to small molecules using chemogenomic fitness signatures</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>344</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1126/science.1250217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1126%2Fscience.1250217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24723613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlslSmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2014&pages=208-211&author=A.+Y.+Leeauthor=R.+P.+St.+Ongeauthor=M.+J.+Proctorauthor=I.+M.+Wallaceauthor=A.+H.+Nileauthor=P.+A.+Spagnuoloauthor=Y.+Jitkovaauthor=M.+Grondaauthor=Y.+Wuauthor=M.+K.+Kimauthor=K.+Cheung-Ongauthor=N.+P.+Torresauthor=E.+D.+Spearauthor=M.+K.+Hanauthor=U.+Schlechtauthor=S.+Sureshauthor=G.+Dubyauthor=L.+E.+Heislerauthor=A.+Surendraauthor=E.+Fungauthor=M.+L.+Urbanusauthor=M.+Gebbiaauthor=E.+Lissinaauthor=M.+Mirandaauthor=J.+H.+Chiangauthor=A.+M.+Aparicioauthor=M.+Zeghoufauthor=R.+W.+Davisauthor=J.+Cherfilsauthor=M.+Boutryauthor=C.+A.+Kaiserauthor=C.+L.+Cumminsauthor=W.+S.+Trimbleauthor=G.+W.+Brownauthor=A.+D.+Schimmerauthor=V.+A.+Bankaitisauthor=C.+Nislowauthor=G.+D.+Baderauthor=G.+Giaever&title=Mapping+the+cellular+response+to+small+molecules+using+chemogenomic+fitness+signatures&doi=10.1126%2Fscience.1250217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping the Cellular Response to Small Molecules Using Chemogenomic Fitness Signatures</span></div><div class="casAuthors">Lee, Anna Y.; St. Onge, Robert P.; Proctor, Michael J.; Wallace, Iain M.; Nile, Aaron H.; Spagnuolo, Paul A.; Jitkova, Yulia; Gronda, Marcela; Wu, Yan; Kim, Moshe K.; Cheung-Ong, Kahlin; Torres, Nikko P.; Spear, Eric D.; Han, Mitchell K. L.; Schlecht, Ulrich; Suresh, Sundari; Duby, Geoffrey; Heisler, Lawrence E.; Surendra, Anuradha; Fung, Eula; Urbanus, Malene L.; Gebbia, Marinella; Lissina, Elena; Miranda, Molly; Chiang, Jennifer H.; Aparicio, Ana Maria; Zeghouf, Mahel; Davis, Ronald W.; Cherfils, Jacqueline; Boutry, Marc; Kaiser, Chris A.; Cummins, Carolyn L.; Trimble, William S.; Brown, Grant W.; Schimmer, Aaron D.; Bankaitis, Vytas A.; Nislow, Corey; Bader, Gary D.; Giaever, Guri</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">6180</span>),
    <span class="NLM_cas:pages">208-211</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Genome-wide characterization of the in vivo cellular response to perturbation is fundamental to understanding how cells survive stress.  Identifying the proteins and pathways perturbed by small mols. affects biol. and medicine by revealing the mechanisms of drug action.  We used a yeast chemogenomics platform that quantifies the requirement for each gene for resistance to a compd. in vivo to profile 3250 small mols. in a systematic and unbiased manner.  We identified 317 compds. that specifically perturb the function of 121 genes and characterized the mechanism of specific compds.  Global anal. revealed that the cellular response to small mols. is limited and described by a network of 45 major chemogenomic signatures.  Our results provide a resource for the discovery of functional interactions among genes, chems., and biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo3_kXSDgt07Vg90H21EOLACvtfcHk0ljmE3ofNwEw-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlslSmtbw%253D&md5=543c8d40d54c94f789521ed29859d403</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1126%2Fscience.1250217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1250217%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DA.%2BY.%26aulast%3DSt.%2BOnge%26aufirst%3DR.%2BP.%26aulast%3DProctor%26aufirst%3DM.%2BJ.%26aulast%3DWallace%26aufirst%3DI.%2BM.%26aulast%3DNile%26aufirst%3DA.%2BH.%26aulast%3DSpagnuolo%26aufirst%3DP.%2BA.%26aulast%3DJitkova%26aufirst%3DY.%26aulast%3DGronda%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DM.%2BK.%26aulast%3DCheung-Ong%26aufirst%3DK.%26aulast%3DTorres%26aufirst%3DN.%2BP.%26aulast%3DSpear%26aufirst%3DE.%2BD.%26aulast%3DHan%26aufirst%3DM.%2BK.%26aulast%3DSchlecht%26aufirst%3DU.%26aulast%3DSuresh%26aufirst%3DS.%26aulast%3DDuby%26aufirst%3DG.%26aulast%3DHeisler%26aufirst%3DL.%2BE.%26aulast%3DSurendra%26aufirst%3DA.%26aulast%3DFung%26aufirst%3DE.%26aulast%3DUrbanus%26aufirst%3DM.%2BL.%26aulast%3DGebbia%26aufirst%3DM.%26aulast%3DLissina%26aufirst%3DE.%26aulast%3DMiranda%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DJ.%2BH.%26aulast%3DAparicio%26aufirst%3DA.%2BM.%26aulast%3DZeghouf%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DR.%2BW.%26aulast%3DCherfils%26aufirst%3DJ.%26aulast%3DBoutry%26aufirst%3DM.%26aulast%3DKaiser%26aufirst%3DC.%2BA.%26aulast%3DCummins%26aufirst%3DC.%2BL.%26aulast%3DTrimble%26aufirst%3DW.%2BS.%26aulast%3DBrown%26aufirst%3DG.%2BW.%26aulast%3DSchimmer%26aufirst%3DA.%2BD.%26aulast%3DBankaitis%26aufirst%3DV.%2BA.%26aulast%3DNislow%26aufirst%3DC.%26aulast%3DBader%26aufirst%3DG.%2BD.%26aulast%3DGiaever%26aufirst%3DG.%26atitle%3DMapping%2520the%2520cellular%2520response%2520to%2520small%2520molecules%2520using%2520chemogenomic%2520fitness%2520signatures%26jtitle%3DScience%26date%3D2014%26volume%3D344%26spage%3D208%26epage%3D211%26doi%3D10.1126%2Fscience.1250217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span> <span> </span><span class="NLM_article-title">Network-assisted target identification for haploinsufficiency and homozygous profiling screens</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e1005553</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1371%2Fjournal.pcbi.1005553" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=e1005553&author=S.+Wangauthor=J.+Peng&title=Network-assisted+target+identification+for+haploinsufficiency+and+homozygous+profiling+screens&doi=10.1371%2Fjournal.pcbi.1005553"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005553%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DJ.%26atitle%3DNetwork-assisted%2520target%2520identification%2520for%2520haploinsufficiency%2520and%2520homozygous%2520profiling%2520screens%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3De1005553%26doi%3D10.1371%2Fjournal.pcbi.1005553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, H. J.</span></span> <span> </span><span class="NLM_article-title">Advances in identification and validation of protein targets of natural products without chemical modification</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">719</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1039/C5NP00107B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1039%2FC5NP00107B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26964663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFegsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=719-730&author=J.+Changauthor=Y.+Kimauthor=H.+J.+Kwon&title=Advances+in+identification+and+validation+of+protein+targets+of+natural+products+without+chemical+modification&doi=10.1039%2FC5NP00107B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in identification and validation of protein targets of natural products without chemical modification</span></div><div class="casAuthors">Chang, J.; Kim, Y.; Kwon, H. J.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">719-730</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covering: up to Feb. 2016Identification of the target proteins of natural products is pivotal to understanding the mechanisms of action to develop natural products for use as mol. probes and potential therapeutic drugs.  Affinity chromatog. of immobilized natural products has been conventionally used to identify target proteins, and has yielded good results.  However, this method has limitations, in that labeling or tagging for immobilization and affinity purifn. often result in reduced or altered activity of the natural product.  New strategies have recently been developed and applied to identify the target proteins of natural products and synthetic small mols. without chem. modification of the natural product.  These direct and indirect methods for target identification of label-free natural products include drug affinity responsive target stability (DARTS), stability of proteins from rates of oxidn. (SPROX), cellular thermal shift assay (CETSA), thermal proteome profiling (TPP), and bioinformatics-based anal. of connectivity.  This review focuses on and reports case studies of the latest advances in target protein identification methods for label-free natural products.  The integration of newly developed technologies will provide new insights and highlight the value of natural products for use as biol. probes and new drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF67ZpeaXLxLVg90H21EOLACvtfcHk0liTizoLkaiUDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFegsL0%253D&md5=7cdabe40955c8c15971d68c29c92b99e</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1039%2FC5NP00107B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5NP00107B%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DKwon%26aufirst%3DH.%2BJ.%26atitle%3DAdvances%2520in%2520identification%2520and%2520validation%2520of%2520protein%2520targets%2520of%2520natural%2520products%2520without%2520chemical%2520modification%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2016%26volume%3D33%26spage%3D719%26epage%3D730%26doi%3D10.1039%2FC5NP00107B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoghegan, K. F.</span></span> <span> </span><span class="NLM_article-title">Emerging methods in chemoproteomics with relevance to drug discovery</span>. <i>Methods Mol. Biol. (N. Y., NY, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1513</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-6539-7_2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2F978-1-4939-6539-7_2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=27807827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1513&publication_year=2017&pages=11-22&author=C.+Nguyenauthor=G.+M.+Westauthor=K.+F.+Geoghegan&title=Emerging+methods+in+chemoproteomics+with+relevance+to+drug+discovery&doi=10.1007%2F978-1-4939-6539-7_2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging methods in chemoproteomics with relevance to drug discovery</span></div><div class="casAuthors">Nguyen, Chuong; West, Graham M.; Geoghegan, Kieran F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1513</span>
        (<span class="NLM_cas:issue">Cancer Gene Networks</span>),
    <span class="NLM_cas:pages">11-22</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A powerful interplay exists between the recognition of gene families, sensitive techniques in proteomics, and the interrogation of protein function using chem. probes.  The most prominent methods, such as affinity capture, activity-based protein profiling and photoaffinity labeling, are extensively reviewed in the literature.  Here we briefly review addnl. methods developed in the past 15 years.  These include "stability proteomics" methods such as proteomically analyzed cellular thermal shift assays and the use of chem. oxidn. as a probe of structure, the use of multiple bead-linked kinase inhibitors to analyze inhibitor specificities, and advances in the use of proteolysis-targeting chimeras for selective protein elimination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjKkAKWHyj8bVg90H21EOLACvtfcHk0liTizoLkaiUDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCksrw%253D&md5=e6db46559bc807e432ffa20101edd1ff</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-6539-7_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-6539-7_2%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DGeoghegan%26aufirst%3DK.%2BF.%26atitle%3DEmerging%2520methods%2520in%2520chemoproteomics%2520with%2520relevance%2520to%2520drug%2520discovery%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2017%26volume%3D1513%26spage%3D11%26epage%3D22%26doi%3D10.1007%2F978-1-4939-6539-7_2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rempel, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, M. L.</span></span> <span> </span><span class="NLM_article-title">Quantification of protein-ligand interactions by mass spectrometry, titration, and H/D exchange: PLIMSTEX</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">5252</span>– <span class="NLM_lpage">5253</span>, <span class="refDoi"> DOI: 10.1021/ja029460d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja029460d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFagtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=5252-5253&author=M.+M.+Zhuauthor=D.+L.+Rempelauthor=Z.+Duauthor=M.+L.+Gross&title=Quantification+of+protein-ligand+interactions+by+mass+spectrometry%2C+titration%2C+and+H%2FD+exchange%3A+PLIMSTEX&doi=10.1021%2Fja029460d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of protein-ligand interactions by mass spectrometry, titration, and H/D exchange: PLIMSTEX</span></div><div class="casAuthors">Zhu, Mei M.; Rempel, Don L.; Du, Zhaohui; Gross, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5252-5253</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand binding and the concomitant conformational change in the protein are of crucial importance in biophysics and drug design.  We report a novel method to quantify protein-ligand interactions in soln. by mass spectrometry, titrn., and H/D exchange (PLIMSTEX).  The approach can det. the conformational change, binding stoichiometry, and affinity in protein-ligand interactions including those that involve small mols., metal ions, and peptides.  Binding consts. obtained by PLIMSTEX for four model protein-ligand systems agree with K values measured by conventional methods.  At higher protein concn., the method can be used to det. quickly the binding stoichiometry and possibly the purity of proteins.  Taking advantage of concg. the protein on-column and desalting, we are able to use different concns. of proteins, buffer systems, salts, and pH in the exchange protocol.  High picomole quantities of proteins are sufficient, offering significantly better sensitivity than that of NMR and x-ray crystallog.  Automation could make PLIMSTEX a high throughput method for library screening, drug discovery, and proteomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppgAt2vdlBSLVg90H21EOLACvtfcHk0lj81n0HF-IXOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFagtro%253D&md5=6f30e9b7ae2c07ecd3be549b95ee4c68</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Fja029460d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja029460d%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DM.%2BM.%26aulast%3DRempel%26aufirst%3DD.%2BL.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DGross%26aufirst%3DM.%2BL.%26atitle%3DQuantification%2520of%2520protein-ligand%2520interactions%2520by%2520mass%2520spectrometry%252C%2520titration%252C%2520and%2520H%252FD%2520exchange%253A%2520PLIMSTEX%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D5252%26epage%3D5253%26doi%3D10.1021%2Fja029460d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani-Yoshihara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakariya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yogo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, T.</span></span> <span> </span><span class="NLM_article-title">CETSA quantitatively verifies in vivo target engagement of novel RIPK1 inhibitors in various biospecimens</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13000</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12513-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fs41598-017-12513-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=29026104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC1M7gtFSmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=13000&author=T.+Ishiiauthor=T.+Okaiauthor=M.+Iwatani-Yoshiharaauthor=M.+Mochizukiauthor=S.+Unnoauthor=M.+Kunoauthor=M.+Yoshikawaauthor=S.+Shibataauthor=M.+Nakakariyaauthor=T.+Yogoauthor=T.+Kawamoto&title=CETSA+quantitatively+verifies+in+vivo+target+engagement+of+novel+RIPK1+inhibitors+in+various+biospecimens&doi=10.1038%2Fs41598-017-12513-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">CETSA quantitatively verifies in vivo target engagement of novel RIPK1 inhibitors in various biospecimens</span></div><div class="casAuthors">Ishii Tsuyoshi; Iwatani-Yoshihara Misa; Mochizuki Manabu; Kawamoto Tomohiro; Okai Takuro; Unno Satoko; Kuno Masako; Yoshikawa Masato; Yogo Takatoshi; Shibata Sachio; Nakakariya Masanori</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13000</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The proof of target engagement (TE) is a key element for evaluating potential investment in drug development.  The cellular thermal shift assay (CETSA) is expected to facilitate direct measurement of intracellular TE at all stages of drug development.  However, there have been no reports of applying this technology to comprehensive animal and clinical studies.  This report demonstrates that CETSA can not only quantitatively evaluate the drug-TE in mouse peripheral blood, but also confirm TE in animal tissues exemplified by using the receptor interacting protein 1 kinase (RIPK1) lead compound we have developed.  Our established semi-automated system allows evaluation of the structure-activity relationship using native RIPK1 in culture cell lines, and also enables estimation of drug occupancy ratio in mouse peripheral blood mononuclear cells.  Moreover, optimized tissue homogenisation enables monitoring of the in vivo drug-TE in spleen and brain.  Our results indicate that CETSA methodology will provide an efficient tool for preclinical and clinical drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSVxZ2bIR2iP3O4gDb7JggCfW6udTcc2ea0alHBM-goSrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7gtFSmsw%253D%253D&md5=ae576418ac1d2ab3788ea241f74f59de</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12513-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12513-1%26sid%3Dliteratum%253Aachs%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DOkai%26aufirst%3DT.%26aulast%3DIwatani-Yoshihara%26aufirst%3DM.%26aulast%3DMochizuki%26aufirst%3DM.%26aulast%3DUnno%26aufirst%3DS.%26aulast%3DKuno%26aufirst%3DM.%26aulast%3DYoshikawa%26aufirst%3DM.%26aulast%3DShibata%26aufirst%3DS.%26aulast%3DNakakariya%26aufirst%3DM.%26aulast%3DYogo%26aufirst%3DT.%26aulast%3DKawamoto%26aufirst%3DT.%26atitle%3DCETSA%2520quantitatively%2520verifies%2520in%2520vivo%2520target%2520engagement%2520of%2520novel%2520RIPK1%2520inhibitors%2520in%2520various%2520biospecimens%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D13000%26doi%3D10.1038%2Fs41598-017-12513-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay: a novel biophysical assay for in situ drug target engagement and mechanistic biomarker studies</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1146%2Fannurev-pharmtox-010715-103715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26566155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCqs7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=141-161&author=D.+Martinez+Molinaauthor=P.+Nordlund&title=The+cellular+thermal+shift+assay%3A+a+novel+biophysical+assay+for+in+situ+drug+target+engagement+and+mechanistic+biomarker+studies&doi=10.1146%2Fannurev-pharmtox-010715-103715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">The Cellular Thermal Shift Assay: A Novel Biophysical Assay for In Situ Drug Target Engagement and Mechanistic Biomarker Studies</span></div><div class="casAuthors">Martinez Molina, Daniel; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-161</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A drug must engage its intended target to achieve its therapeutic effect.  However, conclusively measuring target engagement (TE) in situ is challenging.  This complicates preclin. development and is considered a key factor in the high rate of attrition in clin. trials.  Here, we discuss a recently developed, label-free, biophys. assay, the cellular thermal shift assay (CETSA), which facilitates the direct assessment of TE in cells and tissues at various stages of drug development.  CETSA also reveals biochem. events downstream of drug binding and therefore provides a promising means of establishing mechanistic biomarkers.  The implementation of proteome-wide CETSA using quant. mass spectrometry represents a novel strategy for defining off-target toxicity and polypharmacol. and for identifying downstream mechanistic biomarkers.  The first year of CETSA applications in the literature has focused on TE studies in cell culture systems and has confirmed the broad applicability of CETSA to many different target families.  The next phase of CETSA applications will likely encompass comprehensive animal and patient studies, and CETSA will likely serve as a very valuable tool in many stages of preclin. and clin. drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCOdmURGPcTbVg90H21EOLACvtfcHk0ljfyRUSO-A8rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCqs7nI&md5=092ac50d54d92ef9746a90af8c634e94</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103715%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%253A%2520a%2520novel%2520biophysical%2520assay%2520for%2520in%2520situ%2520drug%2520target%2520engagement%2520and%2520mechanistic%2520biomarker%2520studies%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26spage%3D141%26epage%3D161%26doi%3D10.1146%2Fannurev-pharmtox-010715-103715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waszkiewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallal, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creasy, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbert, M. A.</span></span> <span> </span><span class="NLM_article-title">A high-throughput dose-response cellular thermal shift assay for rapid screening of drug target engagement in living cells, exemplified using SMYD3 and IDO1</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1177/2472555217732014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F2472555217732014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28957646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=34-46&author=D.+E.+McNultyauthor=W.+G.+Bonnetteauthor=H.+Qiauthor=L.+Wangauthor=T.+F.+Hoauthor=A.+Waszkiewiczauthor=L.+A.+Kallalauthor=R.+P.+Nagarajanauthor=M.+Sternauthor=A.+M.+Quinnauthor=C.+L.+Creasyauthor=D.+S.+Suauthor=A.+P.+Gravesauthor=R.+S.+Annanauthor=S.+M.+Sweitzerauthor=M.+A.+Holbert&title=A+high-throughput+dose-response+cellular+thermal+shift+assay+for+rapid+screening+of+drug+target+engagement+in+living+cells%2C+exemplified+using+SMYD3+and+IDO1&doi=10.1177%2F2472555217732014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">A High-Throughput Dose-Response Cellular Thermal Shift Assay for Rapid Screening of Drug Target Engagement in Living Cells, Exemplified Using SMYD3 and IDO1</span></div><div class="casAuthors">McNulty, Dean E.; Bonnette, William G.; Qi, Hongwei; Wang, Liping; Ho, Thau F.; Waszkiewicz, Anna; Kallal, Lorena A.; Nagarajan, Raman P.; Stern, Melissa; Quinn, Amy M.; Creasy, Caretha L.; Su, Dai-Shi; Graves, Alan P.; Annan, Roland S.; Sweitzer, Sharon M.; Holbert, Marc A.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-46</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A persistent problem in early small-mol. drug discovery is the frequent lack of rank-order correlation between biochem. potencies derived from initial screens using purified proteins and the diminished potency and efficacy obsd. in subsequent disease-relevant cellular phenotypic assays.  The introduction of the cellular thermal shift assay (CETSA) has bridged this gap by enabling assessment of drug target engagement directly in live cells based on ligand-induced changes in protein thermal stability.  Initial success in applying CETSA across multiple drug target classes motivated our investigation into replacing the low-throughput, manually intensive Western blot readout with a quant., automated higher-throughput assay that would provide sufficient capacity to use CETSA as a primary hit qualification strategy.  We introduce a high-throughput dose-response cellular thermal shift assay (HTDR-CETSA), a single-pot homogenous assay adapted for high-d. microtiter plate format.  The assay features titratable BacMam expression of full-length target proteins fused to the DiscoverX 42 amino acid ePL tag in HeLa suspension cells, facilitating enzyme fragment complementation-based chemiluminescent quantification of ligand-stabilized sol. protein.  This simplified format can accommodate detn. of full-dose CETSA curves for hundreds of individual compds./analyst/day in replicates.  HTDR-CETSA data generated for substrate site and alternate binding mode inhibitors of the histone-lysine N-methyltransferase SMYD3 in HeLa suspension cells demonstrate excellent correlation with rank-order potencies obsd. in cellular mechanistic assays and direct translation to target engagement of endogenous Smyd3 in cancer-relevant cell lines.  We envision this workflow to be generically applicable to HTDR-CETSA screening spanning a wide variety of sol. intracellular protein target classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOf7ifeefGGbVg90H21EOLACvtfcHk0ljfyRUSO-A8rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKksrY%253D&md5=24b28b2bec845c423f4a3c446974f002</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1177%2F2472555217732014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217732014%26sid%3Dliteratum%253Aachs%26aulast%3DMcNulty%26aufirst%3DD.%2BE.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DWaszkiewicz%26aufirst%3DA.%26aulast%3DKallal%26aufirst%3DL.%2BA.%26aulast%3DNagarajan%26aufirst%3DR.%2BP.%26aulast%3DStern%26aufirst%3DM.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DSu%26aufirst%3DD.%2BS.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DAnnan%26aufirst%3DR.%2BS.%26aulast%3DSweitzer%26aufirst%3DS.%2BM.%26aulast%3DHolbert%26aufirst%3DM.%2BA.%26atitle%3DA%2520high-throughput%2520dose-response%2520cellular%2520thermal%2520shift%2520assay%2520for%2520rapid%2520screening%2520of%2520drug%2520target%2520engagement%2520in%2520living%2520cells%252C%2520exemplified%2520using%2520SMYD3%2520and%2520IDO1%26jtitle%3DSLAS%2520Discov%26date%3D2018%26volume%3D23%26spage%3D34%26epage%3D46%26doi%3D10.1177%2F2472555217732014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET–a novel BRET platform for the analysis of protein-protein interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+Mendezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET%E2%80%93a+novel+BRET+platform+for+the+analysis+of+protein-protein+interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DMendez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET%25E2%2580%2593a%2520novel%2520BRET%2520platform%2520for%2520the%2520analysis%2520of%2520protein-protein%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niles, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Target engagement and drug residence time can be observed in living cells with BRET</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">10091</span>, <span class="refDoi"> DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26631872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=10091&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.+Q.+Chengauthor=K.+V.+Wood&title=Target+engagement+and+drug+residence+time+can+be+observed+in+living+cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement and drug residence time can be observed in living cells with BRET</span></div><div class="casAuthors">Robers, Matthew B.; Dart, Melanie L.; Woodroofe, Carolyn C.; Zimprich, Chad A.; Kirkland, Thomas A.; Machleidt, Thomas; Kupcho, Kevin R.; Levin, Sergiy; Hartnett, James R.; Zimmerman, Kristopher; Niles, Andrew L.; Ohana, Rachel Friedman; Daniels, Danette L.; Slater, Michael; Wood, Monika G.; Cong, Mei; Cheng, Yi-Qiang; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10091</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The therapeutic action of drugs is predicated on their phys. engagement with cellular targets.  Here we describe a broadly applicable method using bioluminescence resonance energy transfer (BRET) to reveal the binding characteristics of a drug with selected targets within intact cells.  Cell-permeable fluorescent tracers are used in a competitive binding format to quantify drug engagement with the target proteins fused to Nanoluc luciferase.  The approach enabled us to profile isoenzyme-specific engagement and binding kinetics for a panel of histone deacetylase (HDAC) inhibitors.  Our anal. was directed particularly to the clin. approved prodrug FK228 (Istodax/Romidepsin) because of its unique and largely unexplained mechanism of sustained intracellular action.  Anal. of the binding kinetics by BRET revealed remarkably long intracellular residence times for FK228 at HDAC1, explaining the protracted intracellular behavior of this prodrug.  Our results demonstrate a novel application of BRET for assessing target engagement within the complex milieu of the intracellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFFk0LNiL_LVg90H21EOLACvtfcHk0lhHoEmBZ61URg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO&md5=723cc5dd7d229b1115da4c2c9d75c913</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.%2BQ.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520engagement%2520and%2520drug%2520residence%2520time%2520can%2520be%2520observed%2520in%2520living%2520cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D10091%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebakken, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddle, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frazee, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliason, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichling, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, K. W.</span></span> <span> </span><span class="NLM_article-title">Development and applications of a broad-coverage, TR-FRET-based kinase binding assay platform</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1177/1087057109339207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F1087057109339207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=19564447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=924-935&author=C.+S.+Lebakkenauthor=S.+M.+Riddleauthor=U.+Singhauthor=W.+J.+Frazeeauthor=H.+C.+Eliasonauthor=Y.+Gaoauthor=L.+J.+Reichlingauthor=B.+D.+Marksauthor=K.+W.+Vogel&title=Development+and+applications+of+a+broad-coverage%2C+TR-FRET-based+kinase+binding+assay+platform&doi=10.1177%2F1087057109339207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Development and applications of a broad-coverage, TR-FRET-based kinase binding assay platform</span></div><div class="casAuthors">Lebakken, Connie S.; Riddle, Steven M.; Singh, Upinder; Frazee, W. Jack; Eliason, Hildegard C.; Gao, Yi; Reichling, Laurie J.; Marks, Bryan D.; Vogel, Kurt W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">924-935</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The expansion of kinase assay technologies over the past decade has mirrored the growing interest in kinases as drug targets.  As a result, there is no shortage of convenient, fluorescence-based methods available to assay targets that span the kinome.  The authors recently reported on the development of a non-activity-based assay to characterize kinase inhibitors that depended on displacement of an Alexa Fluor 647 conjugate of staurosporine (a "tracer") from a particular kinase.  Kinase inhibitors were characterized by a change in fluorescence lifetime of the tracer when it was bound to a kinase relative to when it was displaced by an inhibitor.  Here, the authors report on improvements to this strategy by reconfiguring the assay in a time-resolved fluorescence resonance energy transfer (TR-FRET) format that simplifies instrumentation requirements and allows for the use of a substantially lower concn. of kinase than was required in the fluorescence-lifetime-based format.  The authors use this new assay to demonstrate several aspects of the binding assay format that are advantageous relative to traditional activity-based assays.  The TR-FRET binding format facilitates the assay of compds. against low-activity kinases, allows for the characterization of type II kinase inhibitors either using nonactivated kinases or by monitoring compd. potency over time, and ensures that the signal being detected is specific to the kinase of interest and not a contaminating kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqPbcMF0-PD7Vg90H21EOLACvtfcHk0lhHoEmBZ61URg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLnM&md5=f8df51adf90d821e480cf706213cc096</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1177%2F1087057109339207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109339207%26sid%3Dliteratum%253Aachs%26aulast%3DLebakken%26aufirst%3DC.%2BS.%26aulast%3DRiddle%26aufirst%3DS.%2BM.%26aulast%3DSingh%26aufirst%3DU.%26aulast%3DFrazee%26aufirst%3DW.%2BJ.%26aulast%3DEliason%26aufirst%3DH.%2BC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DReichling%26aufirst%3DL.%2BJ.%26aulast%3DMarks%26aufirst%3DB.%2BD.%26aulast%3DVogel%26aufirst%3DK.%2BW.%26atitle%3DDevelopment%2520and%2520applications%2520of%2520a%2520broad-coverage%252C%2520TR-FRET-based%2520kinase%2520binding%2520assay%2520platform%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D924%26epage%3D935%26doi%3D10.1177%2F1087057109339207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binkowski, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valley, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidugiris, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benink, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggers, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaubert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Encell, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Engineered luciferase reporter from a deep sea shrimp utilizing a novel imidazopyrazinone substrate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1848</span>– <span class="NLM_lpage">1857</span>, <span class="refDoi"> DOI: 10.1021/cb3002478</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3002478" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Smu7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1848-1857&author=M.+P.+Hallauthor=J.+Unchauthor=B.+F.+Binkowskiauthor=M.+P.+Valleyauthor=B.+L.+Butlerauthor=M.+G.+Woodauthor=P.+Ottoauthor=K.+Zimmermanauthor=G.+Vidugirisauthor=T.+Machleidtauthor=M.+B.+Robersauthor=H.+A.+Beninkauthor=C.+T.+Eggersauthor=M.+R.+Slaterauthor=P.+L.+Meisenheimerauthor=D.+H.+Klaubertauthor=F.+Fanauthor=L.+P.+Encellauthor=K.+V.+Wood&title=Engineered+luciferase+reporter+from+a+deep+sea+shrimp+utilizing+a+novel+imidazopyrazinone+substrate&doi=10.1021%2Fcb3002478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Engineered Luciferase Reporter from a Deep Sea Shrimp Utilizing a Novel Imidazopyrazinone Substrate</span></div><div class="casAuthors">Hall, Mary P.; Unch, James; Binkowski, Brock F.; Valley, Michael P.; Butler, Braeden L.; Wood, Monika G.; Otto, Paul; Zimmerman, Kristopher; Vidugiris, Gediminas; Machleidt, Thomas; Robers, Matthew B.; Benink, Helene A.; Eggers, Christopher T.; Slater, Michael R.; Meisenheimer, Poncho L.; Klaubert, Dieter H.; Fan, Frank; Encell, Lance P.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1848-1857</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bioluminescence methodologies have been extraordinarily useful due to their high sensitivity, broad dynamic range, and operational simplicity.  These capabilities have been realized largely through incremental adaptations of native enzymes and substrates, originating from luminous organisms of diverse evolutionary lineages.  We engineered both an enzyme and substrate in combination to create a novel bioluminescence system capable of more efficient light emission with superior biochem. and phys. characteristics.  Using a small luciferase subunit (19 kDa) from the deep sea shrimp Oplophorus gracilirostris, we have improved luminescence expression in mammalian cells ∼2.5 million-fold by merging optimization of protein structure with development of a novel imidazopyrazinone substrate (furimazine).  The new luciferase, NanoLuc, produces glow-type luminescence (signal half-life >2 h) with a specific activity ∼150-fold greater than that of either firefly (Photinus pyralis) or Renilla luciferases similarly configured for glow-type assays.  In mammalian cells, NanoLuc shows no evidence of post-translational modifications or subcellular partitioning.  The enzyme exhibits high phys. stability, retaining activity with incubation up to 55 °C or in culture medium for >15 h at 37 °C.  As a genetic reporter, NanoLuc may be configured for high sensitivity or for response dynamics by appending a degrdn. sequence to reduce intracellular accumulation.  Appending a signal sequence allows NanoLuc to be exported to the culture medium, where reporter expression can be measured without cell lysis.  Fusion onto other proteins allows luminescent assays of their metab. or localization within cells.  Reporter quantitation is achievable even at very low expression levels to facilitate more reliable coupling with endogenous cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlTT1m2z_bZLVg90H21EOLACvtfcHk0lhIbzNbwTWicQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Smu7%252FN&md5=0581a7e44935671471577ade1ff538eb</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Fcb3002478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3002478%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DM.%2BP.%26aulast%3DUnch%26aufirst%3DJ.%26aulast%3DBinkowski%26aufirst%3DB.%2BF.%26aulast%3DValley%26aufirst%3DM.%2BP.%26aulast%3DButler%26aufirst%3DB.%2BL.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DVidugiris%26aufirst%3DG.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DBenink%26aufirst%3DH.%2BA.%26aulast%3DEggers%26aufirst%3DC.%2BT.%26aulast%3DSlater%26aufirst%3DM.%2BR.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DKlaubert%26aufirst%3DD.%2BH.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DEncell%26aufirst%3DL.%2BP.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DEngineered%2520luciferase%2520reporter%2520from%2520a%2520deep%2520sea%2520shrimp%2520utilizing%2520a%2520novel%2520imidazopyrazinone%2520substrate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1848%26epage%3D1857%26doi%3D10.1021%2Fcb3002478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanyai, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">See, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, E. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfleger, K. D. G.</span></span> <span> </span><span class="NLM_article-title">Using nanoBRET and CRISPR/Cas9 to monitor proximity to a genome-edited protein in real-time</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3187</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-03486-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fs41598-017-03486-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=28600500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A280%3ADC%252BC1cnmvFSqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=3187&author=C.+W.+Whiteauthor=H.+K.+Vanyaiauthor=H.+B.+Seeauthor=E.+K.+M.+Johnstoneauthor=K.+D.+G.+Pfleger&title=Using+nanoBRET+and+CRISPR%2FCas9+to+monitor+proximity+to+a+genome-edited+protein+in+real-time&doi=10.1038%2Fs41598-017-03486-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Using nanoBRET and CRISPR/Cas9 to monitor proximity to a genome-edited protein in real-time</span></div><div class="casAuthors">White Carl W; See Heng B; Johnstone Elizabeth K M; Pfleger Kevin D G; White Carl W; Vanyai Hannah K; See Heng B; Johnstone Elizabeth K M; Pfleger Kevin D G; Vanyai Hannah K; Pfleger Kevin D G</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3187</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bioluminescence resonance energy transfer (BRET) has been a vital tool for understanding G protein-coupled receptor (GPCR) function.  It has been used to investigate GPCR-protein and/or -ligand interactions as well as GPCR oligomerisation.  However the utility of BRET is limited by the requirement that the fusion proteins, and in particular the donor, need to be exogenously expressed.  To address this, we have used CRISPR/Cas9-mediated homology-directed repair to generate protein-Nanoluciferase (Nluc) fusions under endogenous promotion, thus allowing investigation of proximity between the genome-edited protein and an exogenously expressed protein by BRET.  Here we report BRET monitoring of GPCR-mediated β-arrestin2 recruitment and internalisation where the donor luciferase was under endogenous promotion, in live cells and in real time.  We have investigated the utility of CRISPR/Cas9 genome editing to create genome-edited fusion proteins that can be used as BRET donors and propose that this strategy can be used to overcome the need for exogenous donor expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQGM460n2USOpQHPYJuLg8TfW6udTcc2eYsgW9zzG1k3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnmvFSqug%253D%253D&md5=dfeeabcf281a05fb5ee429955502dc85</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-03486-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-03486-2%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DC.%2BW.%26aulast%3DVanyai%26aufirst%3DH.%2BK.%26aulast%3DSee%26aufirst%3DH.%2BB.%26aulast%3DJohnstone%26aufirst%3DE.%2BK.%2BM.%26aulast%3DPfleger%26aufirst%3DK.%2BD.%2BG.%26atitle%3DUsing%2520nanoBRET%2520and%2520CRISPR%252FCas9%2520to%2520monitor%2520proximity%2520to%2520a%2520genome-edited%2520protein%2520in%2520real-time%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D3187%26doi%3D10.1038%2Fs41598-017-03486-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggers, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Encell, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binkowski, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">CRISPR-mediated tagging of endogenous proteins with a luminescent peptide</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00549</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00549" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKmur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=467-474&author=M.+K.+Schwinnauthor=T.+Machleidtauthor=K.+Zimmermanauthor=C.+T.+Eggersauthor=A.+S.+Dixonauthor=R.+Hurstauthor=M.+P.+Hallauthor=L.+P.+Encellauthor=B.+F.+Binkowskiauthor=K.+V.+Wood&title=CRISPR-mediated+tagging+of+endogenous+proteins+with+a+luminescent+peptide&doi=10.1021%2Facschembio.7b00549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">CRISPR-Mediated Tagging of Endogenous Proteins with a Luminescent Peptide</span></div><div class="casAuthors">Schwinn, Marie K.; Machleidt, Thomas; Zimmerman, Kris; Eggers, Christopher T.; Dixon, Andrew S.; Hurst, Robin; Hall, Mary P.; Encell, Lance P.; Binkowski, Brock F.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intracellular signaling pathways are mediated by changes in protein abundance and post-translational modifications.  A common approach for investigating signaling mechanisms and the effects induced by synthetic compds. is through overexpression of recombinant reporter genes.  Genome editing with CRISPR/Cas9 offers a means to better preserve native biol. by appending reporters directly onto the endogenous genes.  An optimal reporter for this purpose would be small to negligibly influence intracellular processes, be readily linked to the endogenous genes with minimal exptl. effort, and be sensitive enough to detect low expressing proteins.  HiBiT is a 1.3 kDa peptide (11 amino acids) capable of producing bright and quant. luminescence through high affinity complementation (KD = 700 pM) with an 18 kDa subunit derived from NanoLuc (LgBiT).  Using CRISPR/Cas9, we demonstrate that HiBiT can be rapidly and efficiently integrated into the genome to serve as a reporter tag for endogenous proteins.  Without requiring clonal isolation of the edited cells, we were able to quantify changes in abundance of the hypoxia inducible factor 1A (HIF1α) and several of its downstream transcriptional targets in response to various stimuli.  In combination with fluorescent antibodies, we further used HiBiT to directly correlate HIF1α levels with the hydroxyproline modification that mediates its degrdn.  These results demonstrate the ability to efficiently tag endogenous proteins with a small luminescent peptide, allowing sensitive quantitation of the response dynamics in their regulated expression and covalent modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzCZoXuuYB37Vg90H21EOLACvtfcHk0lhNowDpjnAmkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKmur7I&md5=0d02aa711b02f68a5a54b72b968f6292</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00549%26sid%3Dliteratum%253Aachs%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DEggers%26aufirst%3DC.%2BT.%26aulast%3DDixon%26aufirst%3DA.%2BS.%26aulast%3DHurst%26aufirst%3DR.%26aulast%3DHall%26aufirst%3DM.%2BP.%26aulast%3DEncell%26aufirst%3DL.%2BP.%26aulast%3DBinkowski%26aufirst%3DB.%2BF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DCRISPR-mediated%2520tagging%2520of%2520endogenous%2520proteins%2520with%2520a%2520luminescent%2520peptide%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D467%26epage%3D474%26doi%3D10.1021%2Facschembio.7b00549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell Chem Biol</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.+T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+Mullerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0lhNowDpjnAmkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.%2BT.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell%2520Chem%2520Biol%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evanko, D.</span></span> <span> </span><span class="NLM_article-title">Using ’bumps’ and ’holes’ to control protein activity</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1038/nmeth0306-154b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnmeth0306-154b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=16523567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFOrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=154-155&author=D.+Evanko&title=Using+%E2%80%99bumps%E2%80%99+and+%E2%80%99holes%E2%80%99+to+control+protein+activity&doi=10.1038%2Fnmeth0306-154b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Using 'bumps' and 'holes' to control protein activity</span></div><div class="casAuthors">Evanko, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">154-155</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The creation of orthogonal pairs of rapamycin analogs contg. engineered 'bumps' and protein binding partners with complementary 'holes' permits temporal control of multiple proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgaVLc10MT4bVg90H21EOLACvtfcHk0ljD28qPCFkzEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFOrur0%253D&md5=7ff17647d6554a6218f8f8858e9728b4</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1038%2Fnmeth0306-154b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth0306-154b%26sid%3Dliteratum%253Aachs%26aulast%3DEvanko%26aufirst%3DD.%26atitle%3DUsing%2520%25E2%2580%2599bumps%25E2%2580%2599%2520and%2520%25E2%2580%2599holes%25E2%2580%2599%2520to%2520control%2520protein%2520activity%26jtitle%3DNat.%2520Methods%26date%3D2006%26volume%3D3%26spage%3D154%26epage%3D155%26doi%3D10.1038%2Fnmeth0306-154b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deirmengian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Engineering unnatural nucleotide specificity for Rous sarcoma virus tyrosine kinase to uniquely label its direct substrates</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">3565</span>– <span class="NLM_lpage">3570</span>, <span class="refDoi"> DOI: 10.1073/pnas.94.8.3565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.94.8.3565" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=9108016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADyaK2sXis1amtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=3565-3570&author=K.+Shahauthor=Y.+Liuauthor=C.+Deirmengianauthor=K.+M.+Shokat&title=Engineering+unnatural+nucleotide+specificity+for+Rous+sarcoma+virus+tyrosine+kinase+to+uniquely+label+its+direct+substrates&doi=10.1073%2Fpnas.94.8.3565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering unnatural nucleotide specificity for Rous sarcoma virus tyrosine kinase to uniquely label its direct substrates</span></div><div class="casAuthors">Shah, Kavita; Liu, Yi; Deirmengian, Carl; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3565-3570</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein phosphorylation plays a central role in controlling many diverse signal transduction pathways in all cells.  Novel protein kinases are identified at a rapid rate using homol. cloning methods and genetic screens or selections; however identification of the direct substrates of kinases has proven elusive to genetic methods because of the tremendous redundancy and overlapping of substrate specificities among protein kinases.  We describe the development of a protein engineering-based method to identify the direct substrates of the prototypical protein tyrosine kinase v-Src, which controls fibroblast transformation by the Rous sarcoma virus.  To differentiate the substrates of v-Src from all other kinase substrates, we mutated the ATP binding site of v-Src such that the engineered v-Src uniquely accepted an ATP analog.  We show that the engineered v-Src kinase displayed catalytic efficiency with the ATP analog, N6-(cyclopentyl) ATP, which is similar to the wild-type kinase catalytic efficiency with ATP itself.  However, the N6-(cyclopentyl) ATP analog was not accepted by the wild-type kinase.  Furthermore, the engineered v-Src exhibited the same protein target specificity as wild-type v-Src despite the proximity of the re-engineered nucleotide binding site to the phosphoacceptor binding site.  The successful engineering of v-Src's active site to accept a unique nucleotide analog provides a unique handle by which the direct substrates of one kinase (v-Src) can be traced in the presence of any no. of cellular kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDWluj07-54LVg90H21EOLACvtfcHk0lgAcRn1FI-87A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1amtb4%253D&md5=ff97000a4e9b1c440f5d40ef784a64e8</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.8.3565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.8.3565%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDeirmengian%26aufirst%3DC.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DEngineering%2520unnatural%2520nucleotide%2520specificity%2520for%2520Rous%2520sarcoma%2520virus%2520tyrosine%2520kinase%2520to%2520uniquely%2520label%2520its%2520direct%2520substrates%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1997%26volume%3D94%26spage%3D3565%26epage%3D3570%26doi%3D10.1073%2Fpnas.94.8.3565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emami-Nemini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourrier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamarque, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinquet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohse, M. J.</span></span> <span> </span><span class="NLM_article-title">Time-resolved fluorescence ligand binding for G protein-coupled receptors</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1307</span>– <span class="NLM_lpage">1320</span>, <span class="refDoi"> DOI: 10.1038/nprot.2013.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnprot.2013.073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1307-1320&author=A.+Emami-Neminiauthor=T.+Rouxauthor=M.+Leblayauthor=E.+Bourrierauthor=L.+Lamarqueauthor=E.+Trinquetauthor=M.+J.+Lohse&title=Time-resolved+fluorescence+ligand+binding+for+G+protein-coupled+receptors&doi=10.1038%2Fnprot.2013.073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2013.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2013.073%26sid%3Dliteratum%253Aachs%26aulast%3DEmami-Nemini%26aufirst%3DA.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DLeblay%26aufirst%3DM.%26aulast%3DBourrier%26aufirst%3DE.%26aulast%3DLamarque%26aufirst%3DL.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26atitle%3DTime-resolved%2520fluorescence%2520ligand%2520binding%2520for%2520G%2520protein-coupled%2520receptors%26jtitle%3DNat.%2520Protoc.%26date%3D2013%26volume%3D8%26spage%3D1307%26epage%3D1320%26doi%3D10.1038%2Fnprot.2013.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsukiji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span> <span> </span><span class="NLM_article-title">Ligand-directed tosyl chemistry for in situ native protein labeling and engineering in living systems: from basic properties to applications</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2014.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.cbpa.2014.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25129055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVOhs7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=136-143&author=S.+Tsukijiauthor=I.+Hamachi&title=Ligand-directed+tosyl+chemistry+for+in+situ+native+protein+labeling+and+engineering+in+living+systems%3A+from+basic+properties+to+applications&doi=10.1016%2Fj.cbpa.2014.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-directed tosyl chemistry for in situ native protein labeling and engineering in living systems: from basic properties to applications</span></div><div class="casAuthors">Tsukiji, Shinya; Hamachi, Itaru</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">136-143</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ability to introduce any chem. probe to any endogenous target protein in its native environment, that is in cells and in vivo, is anticipated to provide various new exciting tools for biol. and biomedical research.  Although still at the prototype stage, the ligand-directed tosyl (LDT) chem. is a novel type of affinity labeling technique that we developed for such a dream.  This chem. allows for modifying native proteins by various chem. probes with high specificity in various biol. settings ranging from in vitro (in test tubes) to in living cells and in vivo.  Since the first report, the list of proteins that are successfully labeled by the LDT chem. has been increasing.  A growing no. of studies have demonstrated its utility to create semisynthetic proteins directly in cellular contexts.  The in situ generated semisynthetic proteins are applicable for various types of anal. and imaging of intracellular biol. processes.  In this review, we summarize the basic properties of the LDT chem. and its applications toward in situ engineering and anal. of native proteins in living systems.  Current limitations and future challenges of this area are also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWpiBaEqh9_7Vg90H21EOLACvtfcHk0lgAcRn1FI-87A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVOhs7nK&md5=0e1580b982702ef2074672524209f5e2</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2014.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2014.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DTsukiji%26aufirst%3DS.%26aulast%3DHamachi%26aufirst%3DI.%26atitle%3DLigand-directed%2520tosyl%2520chemistry%2520for%2520in%2520situ%2520native%2520protein%2520labeling%2520and%2520engineering%2520in%2520living%2520systems%253A%2520from%2520basic%2520properties%2520to%2520applications%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D136%26epage%3D143%26doi%3D10.1016%2Fj.cbpa.2014.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otrocka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundback, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seashore-Ludlow, B.</span></span> <span> </span><span class="NLM_article-title">In situ target engagement studies in adherent cells</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">942</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b01079</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b01079" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFaksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=942&author=H.+Axelssonauthor=H.+Almqvistauthor=M.+Otrockaauthor=M.+Vallinauthor=S.+Lundqvistauthor=P.+Hanssonauthor=U.+Karlssonauthor=T.+Lundbackauthor=B.+Seashore-Ludlow&title=In+situ+target+engagement+studies+in+adherent+cells&doi=10.1021%2Facschembio.7b01079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">In Situ Target Engagement Studies in Adherent Cells</span></div><div class="casAuthors">Axelsson, Hanna; Almqvist, Helena; Otrocka, Magdalena; Vallin, Michaela; Lundqvist, Sara; Hansson, Pia; Karlsson, Ulla; Lundbaeck, Thomas; Seashore-Ludlow, Brinton</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">942-950</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A prerequisite for successful drugs is effective binding of the desired target protein in the complex environment of a living system.  Drug-target engagement has typically been difficult to monitor in physiol. relevant models, and with current methods, esp., while maintaining spatial information.  One recent technique for quantifying drug-target engagement is the cellular thermal shift assay (CETSA), in which ligand-induced protein stabilization is measured after a heat challenge.  Here, we describe a CETSA protocol in live A431 cells for p38α (MAPK14), where remaining sol. protein is detected in situ, using high-content imaging in 384-well, microtiter plates.  We validate this assay concept using a no. of known p38α inhibitors and further demonstrate the potential of this technol. for chem. probe and drug discovery purposes by performing a small pilot screen for novel p38α binders.  Importantly, this protocol creates a workflow that is amenable to adherent cells in their native state and yields spatially resolved target engagement information measurable at the single-cell level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2DNlNjNsMnbVg90H21EOLACvtfcHk0liMuHTM4vljow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFaksr4%253D&md5=5d050e94828e9e5d6e32a92b70b837e4</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b01079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b01079%26sid%3Dliteratum%253Aachs%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DOtrocka%26aufirst%3DM.%26aulast%3DVallin%26aufirst%3DM.%26aulast%3DLundqvist%26aufirst%3DS.%26aulast%3DHansson%26aufirst%3DP.%26aulast%3DKarlsson%26aufirst%3DU.%26aulast%3DLundback%26aufirst%3DT.%26aulast%3DSeashore-Ludlow%26aufirst%3DB.%26atitle%3DIn%2520situ%2520target%2520engagement%2520studies%2520in%2520adherent%2520cells%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D942%26doi%3D10.1021%2Facschembio.7b01079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinegoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">Bioorthogonal imaging of aurora kinase A in live cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">6598</span>– <span class="NLM_lpage">6603</span>, <span class="refDoi"> DOI: 10.1002/anie.201200994</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201200994" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1Sqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=6598-6603&author=K.+S.+Yangauthor=G.+Budinauthor=T.+Reinerauthor=C.+Vinegoniauthor=R.+Weissleder&title=Bioorthogonal+imaging+of+aurora+kinase+A+in+live+cells&doi=10.1002%2Fanie.201200994"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Bioorthogonal Imaging of Aurora Kinase A in Live Cells</span></div><div class="casAuthors">Yang, Katherine S.; Budin, Ghyslain; Reiner, Thomas; Vinegoni, Claudio; Weissleder, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">6598-6603, S6598/1-S6598/32</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The first use of MLN8054-TCO (13) as an imaging agent for AKA visualization in live cells and as a tool for quantifying cellular AKA expression is reported here.  The authors show that endogenous AKA can be imaged in live cells by way of a two-step bioorthogonal reaction. As an imaging agent, MLN8054-TCO (13) in combination with CFDA-Tz demonstrated superior selectivity, fast reaction kinetics, and a sufficient target-to-back-ground ratio to enable AKA visualization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW_s6BEbgs27Vg90H21EOLACvtfcHk0liEk4z5gu1a6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1Sqtbk%253D&md5=a620de4622bbf073a0acf092dd778e9b</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1002%2Fanie.201200994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201200994%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DK.%2BS.%26aulast%3DBudin%26aufirst%3DG.%26aulast%3DReiner%26aufirst%3DT.%26aulast%3DVinegoni%26aufirst%3DC.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DBioorthogonal%2520imaging%2520of%2520aurora%2520kinase%2520A%2520in%2520live%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D6598%26epage%3D6603%26doi%3D10.1002%2Fanie.201200994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohler, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubach, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikula, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">Optimized near-IR fluorescent agents for in vivo imaging of Btk expression</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1513</span>– <span class="NLM_lpage">1518</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.5b00152</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.5b00152" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=1513-1518&author=E.+Kimauthor=K.+S.+Yangauthor=R.+H.+Kohlerauthor=J.+M.+Dubachauthor=H.+Mikulaauthor=R.+Weissleder&title=Optimized+near-IR+fluorescent+agents+for+in+vivo+imaging+of+Btk+expression&doi=10.1021%2Facs.bioconjchem.5b00152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Optimized Near-IR Fluorescent Agents for in Vivo Imaging of Btk Expression</span></div><div class="casAuthors">Kim, Eunha; Yang, Katherine S.; Kohler, Rainer H.; Dubach, John M.; Mikula, Hannes; Weissleder, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1513-1518</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bruton's tyrosine kinase (Btk) is intricately involved in anti-apoptotic signaling pathways in cancer and in regulating innate immune response.  A no. of Btk inhibitors are in development for use in treating B-cell malignancies and certain immunol. diseases.  To develop robust companion imaging diagnostics for in vivo use, we set out to explore the effects of red wavelength fluorochrome modifications of two highly potent irreversible Btk inhibitors, Ibrutinib and AVL-292.  Surprisingly, we found that subtle chem. differences in the fluorochrome had considerable effects on target localization.  Based on iterative designs, we developed a single optimized version with superb in vivo imaging characteristics enabling single cell Btk imaging in vivo.  This agent (Ibrutinib-SiR-COOH) is expected to be a valuable chem. tool in deciphering Btk biol. in cancer and host cells in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5fRkToD0SFLVg90H21EOLACvtfcHk0liEk4z5gu1a6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKrur0%253D&md5=78ad36d0e97044a4bcf239a60f957e35</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.5b00152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.5b00152%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DK.%2BS.%26aulast%3DKohler%26aufirst%3DR.%2BH.%26aulast%3DDubach%26aufirst%3DJ.%2BM.%26aulast%3DMikula%26aufirst%3DH.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DOptimized%2520near-IR%2520fluorescent%2520agents%2520for%2520in%2520vivo%2520imaging%2520of%2520Btk%2520expression%26jtitle%3DBioconjugate%2520Chem.%26date%3D2015%26volume%3D26%26spage%3D1513%26epage%3D1518%26doi%3D10.1021%2Facs.bioconjchem.5b00152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">Bioorthogonal small molecule imaging agents allow single-cell imaging of MET</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e81275</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0081275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1371%2Fjournal.pone.0081275" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e81275&author=E.+Kimauthor=K.+S.+Yangauthor=R.+Weissleder&title=Bioorthogonal+small+molecule+imaging+agents+allow+single-cell+imaging+of+MET&doi=10.1371%2Fjournal.pone.0081275"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0081275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0081275%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DK.%2BS.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DBioorthogonal%2520small%2520molecule%2520imaging%2520agents%2520allow%2520single-cell%2520imaging%2520of%2520MET%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De81275%26doi%3D10.1371%2Fjournal.pone.0081275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meimetis, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilderbrand, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">BODIPY-tetrazine derivatives as superbright bioorthogonal turn-on probes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6917</span>– <span class="NLM_lpage">6920</span>, <span class="refDoi"> DOI: 10.1002/anie.201301100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1002%2Fanie.201301100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=6917-6920&author=J.+C.+Carlsonauthor=L.+G.+Meimetisauthor=S.+A.+Hilderbrandauthor=R.+Weissleder&title=BODIPY-tetrazine+derivatives+as+superbright+bioorthogonal+turn-on+probes&doi=10.1002%2Fanie.201301100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">BODIPY-Tetrazine Derivatives as Superbright Bioorthogonal Turn-on Probes</span></div><div class="casAuthors">Carlson, Jonathan C. T.; Meimetis, Labros G.; Hilderbrand, Scott A.; Weissleder, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">6917-6920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">As an alternative way of designing more efficient turn-on probes, one might consider adapting through-bond energy transfer (TBET) for fluorescence quenching.  With these goals in mind, the authors synthesized a series of new bioorthogonal boron dipyrromethene (BODIPY) tetrazine derivs.  These structures enhance spatial donor-acceptor proximity, provide predictable donor-acceptor transition-dipole orientation, and afford the possibility of accessing alternate modes of fluorescence quenching.  Having advanced the understanding of the quenching mechanism, the authors sought to validate the utility of fluorogenic BODIPY-tetrazines for biol. imaging.  Both extracellular and intracellular trans-cyclooctenol-labeled targets were readily visualized, with excellent signal intensity, very low background, and with no washing steps required after addn. of the dye soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3itwIffX8-LVg90H21EOLACvtfcHk0liEk4z5gu1a6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCisrs%253D&md5=d12b9d6d6bec2aadf46e8a0138f27951</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1002%2Fanie.201301100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201301100%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DJ.%2BC.%26aulast%3DMeimetis%26aufirst%3DL.%2BG.%26aulast%3DHilderbrand%26aufirst%3DS.%2BA.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DBODIPY-tetrazine%2520derivatives%2520as%2520superbright%2520bioorthogonal%2520turn-on%2520probes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D6917%26epage%3D6920%26doi%3D10.1002%2Fanie.201301100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span> <span> </span><span class="NLM_article-title">A modular probe strategy for drug localization, target identification and target occupancy measurement on single cell level</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2541</span>– <span class="NLM_lpage">2550</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00346</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFais7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2541-2550&author=A.+Rutkowskaauthor=D.+W.+Thomsonauthor=J.+Vappianiauthor=T.+Wernerauthor=K.+M.+Muellerauthor=L.+Dittusauthor=J.+Krauseauthor=M.+Muelbaierauthor=G.+Bergaminiauthor=M.+Bantscheff&title=A+modular+probe+strategy+for+drug+localization%2C+target+identification+and+target+occupancy+measurement+on+single+cell+level&doi=10.1021%2Facschembio.6b00346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">A Modular Probe Strategy for Drug Localization, Target Identification and Target Occupancy Measurement on Single Cell Level</span></div><div class="casAuthors">Rutkowska, Anna; Thomson, Douglas W.; Vappiani, Johanna; Werner, Thilo; Mueller, Katrin M.; Dittus, Lars; Krause, Jana; Muelbaier, Marcel; Bergamini, Giovanna; Bantscheff, Marcus</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2541-2550</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Late stage failures of candidate drug mols. are frequently caused by off-target effects or inefficient target engagement in vivo.  In order to address these fundamental challenges in drug discovery, we developed a modular probe strategy based on bioorthogonal chem. that enables the attachment of multiple reporters to the same probe in cell exts. and live cells.  In a systematic evaluation, we identified the inverse electron demand Diels-Alder reaction between trans-cyclooctene labeled probe mols. and tetrazine-tagged reporters to be the most efficient bioorthogonal reaction for this strategy.  Bioorthogonal biotinylation of the probe allows the identification of drug targets in a chemoproteomics competition binding assay using quant. mass spectrometry.  Attachment of a fluorescent reporter enables monitoring of spatial localization of probes as well as drug-target colocalization studies.  Finally, direct target occupancy of unlabeled drugs can be detd. at single cell resoln. by competitive binding with fluorescently labeled probe mols.  The feasibility of the modular probe strategy is demonstrated with noncovalent PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodaDW6dNqXzrVg90H21EOLACvtfcHk0lg6AYqrhFoaMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFais7vF&md5=708fbb531d90863eca83989b9a9b85b4</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00346%26sid%3Dliteratum%253Aachs%26aulast%3DRutkowska%26aufirst%3DA.%26aulast%3DThomson%26aufirst%3DD.%2BW.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DMueller%26aufirst%3DK.%2BM.%26aulast%3DDittus%26aufirst%3DL.%26aulast%3DKrause%26aufirst%3DJ.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26atitle%3DA%2520modular%2520probe%2520strategy%2520for%2520drug%2520localization%252C%2520target%2520identification%2520and%2520target%2520occupancy%2520measurement%2520on%2520single%2520cell%2520level%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2541%26epage%3D2550%26doi%3D10.1021%2Facschembio.6b00346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Honer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martarello, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comley, R. A.</span></span> <span> </span><span class="NLM_article-title">Radioligand development for molecular imaging of the central nervous system with positron emission tomography</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1936</span>– <span class="NLM_lpage">1944</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.08.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.drudis.2014.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25173703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyhtrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1936-1944&author=M.+Honerauthor=L.+Gobbiauthor=L.+Martarelloauthor=R.+A.+Comley&title=Radioligand+development+for+molecular+imaging+of+the+central+nervous+system+with+positron+emission+tomography&doi=10.1016%2Fj.drudis.2014.08.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Radioligand development for molecular imaging of the central nervous system with positron emission tomography</span></div><div class="casAuthors">Honer, Michael; Gobbi, Luca; Martarello, Laurent; Comley, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1936-1944</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Positron emission tomog. (PET) is routinely used to support the development of drugs to treat neurol. and psychiatric disorders.  PET radioligands must not only be selective for the target of interest but must also possess a range of physicochem. and pharmacol. characteristics that allow them to be radiolabeled with short-lived positron-emitting isotopes, safely administered to humans, and for the degree of target binding to be quantified in vivo.  We review the ligand development process, including target selection, radioligand discovery (in vitro and preclin. evaluation), radiochem. and evaluation in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMOrjEdZ64wLVg90H21EOLACvtfcHk0lg6AYqrhFoaMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyhtrbE&md5=9ae3bb3030cdc3a701be1df568190371</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DHoner%26aufirst%3DM.%26aulast%3DGobbi%26aufirst%3DL.%26aulast%3DMartarello%26aufirst%3DL.%26aulast%3DComley%26aufirst%3DR.%2BA.%26atitle%3DRadioligand%2520development%2520for%2520molecular%2520imaging%2520of%2520the%2520central%2520nervous%2520system%2520with%2520positron%2520emission%2520tomography%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1936%26epage%3D1944%26doi%3D10.1016%2Fj.drudis.2014.08.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutilek, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smotrov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickbarg, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauvagnat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curran, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boinay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saradjian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsen, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandliker, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariv, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutel, B.</span></span> <span> </span><span class="NLM_article-title">Integration of affinity selection-mass spectrometry and functional cell-based assays to rapidly triage druggable target space within the NF-kappaB pathway</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1177/1087057116637353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F1087057116637353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26969322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XitV2qtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=608-619&author=V.+D.+Kutilekauthor=C.+L.+Andrewsauthor=M.+P.+Richardsauthor=Z.+Xuauthor=T.+Sunauthor=Y.+Chenauthor=A.+Hashkeauthor=N.+Smotrovauthor=R.+Fernandezauthor=E.+B.+Nickbargauthor=C.+Chamberlinauthor=B.+Sauvagnatauthor=P.+J.+Curranauthor=R.+Boinayauthor=P.+Saradjianauthor=S.+J.+Allenauthor=N.+Byrneauthor=N.+L.+Elsenauthor=R.+E.+Fordauthor=D.+L.+Hallauthor=M.+Kornienkoauthor=K.+W.+Rickertauthor=S.+Sharmaauthor=J.+M.+Shipmanauthor=K.+J.+Lumbauthor=K.+Colemanauthor=P.+J.+Dandlikerauthor=I.+Karivauthor=B.+Beutel&title=Integration+of+affinity+selection-mass+spectrometry+and+functional+cell-based+assays+to+rapidly+triage+druggable+target+space+within+the+NF-kappaB+pathway&doi=10.1177%2F1087057116637353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of affinity selection-mass spectrometry and functional cell-based assays to rapidly triage druggable target space within the NF-κB pathway</span></div><div class="casAuthors">Kutilek, Victoria D.; Andrews, Christine L.; Richards, Matthew P.; Xu, Zangwei; Sun, Tianxiao; Chen, Yiping; Hashke, Andrew; Smotrov, Nadya; Fernandez, Rafael; Nickbarg, Elliott B.; Chamberlin, Chad; Sauvagnat, Berengere; Curran, Patrick J.; Boinay, Ryan; Saradjian, Peter; Allen, Samantha J.; Byrne, Noel; Elsen, Nathaniel L.; Ford, Rachael E.; Hall, Dawn L.; Kornienko, Maria; Rickert, Keith W.; Sharma, Sujata; Shipman, Jennifer M.; Lumb, Kevin J.; Coleman, Kevin; Dandliker, Peter J.; Kariv, Ilona; Beutel, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">608-619</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The primary objective of early drug discovery is to assoc. druggable target space with a desired phenotype.  The inability to efficiently assoc. these often leads to failure early in the drug discovery process.  In this proof-of-concept study, the most tractable starting points for drug discovery within the NF-κB pathway model system were identified by integrating affinity selection-mass spectrometry (AS-MS) with functional cellular assays.  The AS-MS platform Automated Ligand Identification System (ALIS) was used to rapidly screen 15 NF-κB proteins in parallel against large-compd. libraries.  ALIS identified 382 target-selective compds. binding to 14 of the 15 proteins.  Without any chem. optimization, 22 of the 382 target-selective compds. exhibited a cellular phenotype consistent with the resp. target assocd. in ALIS.  Further studies on structurally related compds. distinguished two chem. series that exhibited a preliminary structure-activity relationship and confirmed target-driven cellular activity to NF-κB1/p105 and TRAF5, resp.  These two series represent new drug discovery opportunities for chem. optimization.  The results described herein demonstrate the power of combining ALIS with cell functional assays in a high-throughput, target-based approach to det. the most tractable drug discovery opportunities within a pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQAPIJwi2tULVg90H21EOLACvtfcHk0lhCZGYqjX3mMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitV2qtLjN&md5=d1130842a2d22e65c6903164dbddb341</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1177%2F1087057116637353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116637353%26sid%3Dliteratum%253Aachs%26aulast%3DKutilek%26aufirst%3DV.%2BD.%26aulast%3DAndrews%26aufirst%3DC.%2BL.%26aulast%3DRichards%26aufirst%3DM.%2BP.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DHashke%26aufirst%3DA.%26aulast%3DSmotrov%26aufirst%3DN.%26aulast%3DFernandez%26aufirst%3DR.%26aulast%3DNickbarg%26aufirst%3DE.%2BB.%26aulast%3DChamberlin%26aufirst%3DC.%26aulast%3DSauvagnat%26aufirst%3DB.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DBoinay%26aufirst%3DR.%26aulast%3DSaradjian%26aufirst%3DP.%26aulast%3DAllen%26aufirst%3DS.%2BJ.%26aulast%3DByrne%26aufirst%3DN.%26aulast%3DElsen%26aufirst%3DN.%2BL.%26aulast%3DFord%26aufirst%3DR.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BL.%26aulast%3DKornienko%26aufirst%3DM.%26aulast%3DRickert%26aufirst%3DK.%2BW.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DShipman%26aufirst%3DJ.%2BM.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DDandliker%26aufirst%3DP.%2BJ.%26aulast%3DKariv%26aufirst%3DI.%26aulast%3DBeutel%26aufirst%3DB.%26atitle%3DIntegration%2520of%2520affinity%2520selection-mass%2520spectrometry%2520and%2520functional%2520cell-based%2520assays%2520to%2520rapidly%2520triage%2520druggable%2520target%2520space%2520within%2520the%2520NF-kappaB%2520pathway%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D608%26epage%3D619%26doi%3D10.1177%2F1087057116637353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Annis, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazef, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, H. M.</span></span> <span> </span><span class="NLM_article-title">A general technique to rank protein-ligand binding affinities and determine allosteric versus direct binding site competition in compound mixtures</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">15495</span>– <span class="NLM_lpage">15503</span>, <span class="refDoi"> DOI: 10.1021/ja048365x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja048365x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptlOisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=15495-15503&author=D.+A.+Annisauthor=N.+Nazefauthor=C.+C.+Chuangauthor=M.+P.+Scottauthor=H.+M.+Nash&title=A+general+technique+to+rank+protein-ligand+binding+affinities+and+determine+allosteric+versus+direct+binding+site+competition+in+compound+mixtures&doi=10.1021%2Fja048365x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">A general technique to rank protein-ligand binding affinities and determine allosteric versus direct binding site competition in compound mixtures</span></div><div class="casAuthors">Annis, D. Allen; Nazef, Naim; Chuang, Cheng-Chi; Scott, Margaret Porter; Nash, Huw M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">15495-15503</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To realize the full potential of combinatorial chem.-based drug discovery, generic and efficient tools must be developed that apply the strengths of diversity-oriented chem. synthesis to the identification and optimization of lead compds. for disease-assocd. protein targets.  We report an affinity selection-mass spectrometry (AS-MS) method for protein-ligand affinity ranking and the classification of ligands by binding site.  The method incorporates the following steps: (1) an affinity selection stage, where protein-binding compds. are selected from pools of ligands in the presence of varying concns. of a competitor ligand, (2) a first chromatog. stage to sep. unbound ligands from protein-ligand complexes, and (3) a second chromatog. stage to dissoc. the ligands from the complexes for identification and quantification by MS.  The ability of the competitor ligand to displace a target-bound library member, as measured by MS, reveals the binding site classification and affinity ranking of the mixt. components.  The technique requires no radiolabel incorporation or direct biochem. assay, no modification or immobilization of the compds. or target protein, and all reaction components, including any buffers or cofactors required for protein stability, are free in soln.  We demonstrate the method for several compds. of wide structural variety against representatives of the most important protein classes in contemporary drug discovery, including novel ATP-competitive and allosteric inhibitors of the Akt-1 (PKB) and Zap-70 kinases, and previously undisclosed antagonists of the M2 muscarinic acetylcholine receptor, a G-protein coupled receptor (GPCR).  The theor. basis of the technique is analyzed math., allowing quant. estn. of binding affinities and, in the case of allosteric interaction, abs. detn. of binding cooperativity.  The method is readily applicable to high-throughput screening hit triage, combinatorial library-based affinity optimization, and developing structure-activity relationships among multiple ligands to a given receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpREa-teJ7tC7Vg90H21EOLACvtfcHk0lhCZGYqjX3mMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptlOisrk%253D&md5=bee110f0d9f40e75b5e22d120238a742</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Fja048365x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja048365x%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DChuang%26aufirst%3DC.%2BC.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DNash%26aufirst%3DH.%2BM.%26atitle%3DA%2520general%2520technique%2520to%2520rank%2520protein-ligand%2520binding%2520affinities%2520and%2520determine%2520allosteric%2520versus%2520direct%2520binding%2520site%2520competition%2520in%2520compound%2520mixtures%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D15495%26epage%3D15503%26doi%3D10.1021%2Fja048365x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Comess, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schurdak, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorbach, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trumbull, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCall, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutel, B. A.</span></span> <span> </span><span class="NLM_article-title">An ultraefficient affinity-based high-throughout screening process: application to bacterial cell wall biosynthesis enzyme MurF</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">743</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1177/1087057106289971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F1087057106289971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=16973923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1SgsL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=743-754&author=K.+M.+Comessauthor=M.+E.+Schurdakauthor=M.+J.+Voorbachauthor=M.+Coenauthor=J.+D.+Trumbullauthor=H.+Yangauthor=L.+Gaoauthor=H.+Tangauthor=X.+Chengauthor=C.+G.+Lernerauthor=J.+O.+McCallauthor=D.+J.+Burnsauthor=B.+A.+Beutel&title=An+ultraefficient+affinity-based+high-throughout+screening+process%3A+application+to+bacterial+cell+wall+biosynthesis+enzyme+MurF&doi=10.1177%2F1087057106289971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">An ultraefficient affinity-based high-throughout screening process: application to bacterial cell wall biosynthesis enzyme MurF</span></div><div class="casAuthors">Comess, Kenneth M.; Schurdak, Mark E.; Voorbach, Martin J.; Coen, Michael; Trumbull, Jonathan D.; Yang, Houjun; Gao, Lan; Tang, Hua; Cheng, Xueheng; Lerner, Claude G.; McCall, J. Owen; Burns, David J.; Beutel, Bruce A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">743-754</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The authors describe the discovery of a new class of inhibitors to an essential Streptococcus pneumoniae cell wall biosynthesis enzyme, MurF, by a novel affinity screening method.  The strategy involved screening very large mixts. of diverse small org. mols. against the protein target on the basis of equil. binding, followed by iterative ultrafiltration steps and ligand identification by mass spectrometry.  Hits from any affinity-based screening method often can be relatively nonselective ligands, sometimes referred to as "nuisance" or "promiscuous" compds.  Ligands selective in their binding affinity for the MurF target were readily identified through electronic subtraction of an empirically detd. subset of promiscuous compds. in the library without subsequent selectivity panels.  The complete strategy for discovery and identification of novel specific ligands can be applied to all sol. protein targets and a wide variety of ligand libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgdKbNMU3brVg90H21EOLACvtfcHk0ljdtooc_eHU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1SgsL7F&md5=949a2a17ddba4d6c001bad259a3c75e1</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1177%2F1087057106289971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106289971%26sid%3Dliteratum%253Aachs%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DSchurdak%26aufirst%3DM.%2BE.%26aulast%3DVoorbach%26aufirst%3DM.%2BJ.%26aulast%3DCoen%26aufirst%3DM.%26aulast%3DTrumbull%26aufirst%3DJ.%2BD.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DLerner%26aufirst%3DC.%2BG.%26aulast%3DMcCall%26aufirst%3DJ.%2BO.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DBeutel%26aufirst%3DB.%2BA.%26atitle%3DAn%2520ultraefficient%2520affinity-based%2520high-throughout%2520screening%2520process%253A%2520application%2520to%2520bacterial%2520cell%2520wall%2520biosynthesis%2520enzyme%2520MurF%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D743%26epage%3D754%26doi%3D10.1177%2F1087057106289971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieth, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroh, J. G.</span></span> <span> </span><span class="NLM_article-title">Solution-based indirect affinity selection mass spectrometry–a general tool for high-throughput screening of pharmaceutical compound libraries</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">7413</span>– <span class="NLM_lpage">7420</span>, <span class="refDoi"> DOI: 10.1021/ac500938y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac500938y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=7413-7420&author=T.+N.+O%E2%80%99Connellauthor=J.+Ramsayauthor=S.+F.+Riethauthor=M.+J.+Shapiroauthor=J.+G.+Stroh&title=Solution-based+indirect+affinity+selection+mass+spectrometry%E2%80%93a+general+tool+for+high-throughput+screening+of+pharmaceutical+compound+libraries&doi=10.1021%2Fac500938y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fac500938y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac500938y%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DT.%2BN.%26aulast%3DRamsay%26aufirst%3DJ.%26aulast%3DRieth%26aufirst%3DS.%2BF.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DStroh%26aufirst%3DJ.%2BG.%26atitle%3DSolution-based%2520indirect%2520affinity%2520selection%2520mass%2520spectrometry%25E2%2580%2593a%2520general%2520tool%2520for%2520high-throughput%2520screening%2520of%2520pharmaceutical%2520compound%2520libraries%26jtitle%3DAnal.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D7413%26epage%3D7420%26doi%3D10.1021%2Fac500938y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowling, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malamas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellestad, G.</span></span> <span> </span><span class="NLM_article-title">Evaluation of fluorescence-based thermal shift assays for hit identification in drug discovery</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>332</i></span>,  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2004.04.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.ab.2004.04.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=15301960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmsVKhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=153-159&author=M.+C.+Loauthor=A.+Aulabaughauthor=G.+Jinauthor=R.+Cowlingauthor=J.+Bardauthor=M.+Malamasauthor=G.+Ellestad&title=Evaluation+of+fluorescence-based+thermal+shift+assays+for+hit+identification+in+drug+discovery&doi=10.1016%2Fj.ab.2004.04.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of fluorescence-based thermal shift assays for hit identification in drug discovery</span></div><div class="casAuthors">Lo, Mei-Chu; Aulabaugh, Ann; Jin, Guixian; Cowling, Rebecca; Bard, Jonathan; Malamas, Michael; Ellestad, George</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">153-159</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The fluorescence-based thermal shift assay is a general method for identification of inhibitors of target proteins from compd. libraries.  Using an environmentally sensitive fluorescent dye to monitor protein thermal unfolding, the ligand-binding affinity can be assessed from the shift of the unfolding temp. (ΔTm) obtained in the presence of ligands relative to that obtained in the absence of ligands.  In this article, we report that the thermal shift assay can be conducted in an inexpensive, com. available device for temp. control and fluorescence detection.  The binding affinities obtained from thermal shift assays are compared with the binding affinities measured by isothermal titrn. calorimetry and with the IC50 values from enzymic assays.  The potential pitfalls in the data anal. of thermal shift assays are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1qMM9vchBGrVg90H21EOLACvtfcHk0ljdtooc_eHU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmsVKhu7g%253D&md5=bc10d980128340c79d7d2abc9a56463b</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.04.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.04.031%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DM.%2BC.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DG.%26aulast%3DCowling%26aufirst%3DR.%26aulast%3DBard%26aufirst%3DJ.%26aulast%3DMalamas%26aufirst%3DM.%26aulast%3DEllestad%26aufirst%3DG.%26atitle%3DEvaluation%2520of%2520fluorescence-based%2520thermal%2520shift%2520assays%2520for%2520hit%2520identification%2520in%2520drug%2520discovery%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D153%26epage%3D159%26doi%3D10.1016%2Fj.ab.2004.04.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pantoliano, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrella, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasnoski, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobanov, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myslik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carver, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salemme, F. R.</span></span> <span> </span><span class="NLM_article-title">High-density miniaturized thermal shift assays as a general strategy for drug discovery</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1177/108705710100600609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1177%2F108705710100600609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=11788061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVCntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2001&pages=429-440&author=M.+W.+Pantolianoauthor=E.+C.+Petrellaauthor=J.+D.+Kwasnoskiauthor=V.+S.+Lobanovauthor=J.+Myslikauthor=E.+Grafauthor=T.+Carverauthor=E.+Aselauthor=B.+A.+Springerauthor=P.+Laneauthor=F.+R.+Salemme&title=High-density+miniaturized+thermal+shift+assays+as+a+general+strategy+for+drug+discovery&doi=10.1177%2F108705710100600609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">High-density miniaturized thermal shift assays as a general strategy for drug discovery</span></div><div class="casAuthors">Pantoliano, Michael W.; Petrella, Eugene C.; Kwasnoski, Joseph D.; Lobanov, Victor S.; Myslik, James; Graf, Edward; Carver, Ted; Asel, Eric; Springer, Barry A.; Lane, Pamela; Salemme, F. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">429-440</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">More general and universally applicable drug discovery assay technologies are needed in order to keep pace with the recent advances in combinatorial chem. and genomics-based target generation.  Ligand-induced conformational stabilization of proteins is a well-understood phenomenon in which substrates, inhibitors, cofactors, and even other proteins provide enhanced stability to proteins on binding.  This phenomenon is based on the energetic coupling of the ligand-binding and protein-melting reactions.  In an attempt to harness these biophys. properties for drug discovery, fully automated instrumentation was designed and implemented to perform miniaturized fluorescence-based thermal shift assays in a microplate format for the high throughput screening of compd. libraries.  Validation of this process and instrumentation was achieved by investigating ligand binding to more than 100 protein targets.  The general applicability of the thermal shift screening strategy was found to be an important advantage because it circumvents the need to design and retool new assays with each new therapeutic target.  Moreover, the miniaturized thermal shift assay methodol. does not require any prior knowledge of a therapeutic target's function, making it ideally suited for the quant. high throughput drug screening and evaluation of targets derived from genomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp58y9R74HfiLVg90H21EOLACvtfcHk0ljIfD-380_Lug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVCntg%253D%253D&md5=deef07e6683f632cd078214ad88d9999</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1177%2F108705710100600609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705710100600609%26sid%3Dliteratum%253Aachs%26aulast%3DPantoliano%26aufirst%3DM.%2BW.%26aulast%3DPetrella%26aufirst%3DE.%2BC.%26aulast%3DKwasnoski%26aufirst%3DJ.%2BD.%26aulast%3DLobanov%26aufirst%3DV.%2BS.%26aulast%3DMyslik%26aufirst%3DJ.%26aulast%3DGraf%26aufirst%3DE.%26aulast%3DCarver%26aufirst%3DT.%26aulast%3DAsel%26aufirst%3DE.%26aulast%3DSpringer%26aufirst%3DB.%2BA.%26aulast%3DLane%26aufirst%3DP.%26aulast%3DSalemme%26aufirst%3DF.%2BR.%26atitle%3DHigh-density%2520miniaturized%2520thermal%2520shift%2520assays%2520as%2520a%2520general%2520strategy%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2001%26volume%3D6%26spage%3D429%26epage%3D440%26doi%3D10.1177%2F108705710100600609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pellecchia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowburn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalvit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giralt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homans, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luchinat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oschkinat, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwalbe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegal, G.</span></span> <span> </span><span class="NLM_article-title">Perspectives on NMR in drug discovery: a technique comes of age</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">738</span>– <span class="NLM_lpage">745</span>, <span class="refDoi"> DOI: 10.1038/nrd2606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnrd2606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=19172689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVGgtL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=738-745&author=M.+Pellecchiaauthor=I.+Bertiniauthor=D.+Cowburnauthor=C.+Dalvitauthor=E.+Giraltauthor=W.+Jahnkeauthor=T.+L.+Jamesauthor=S.+W.+Homansauthor=H.+Kesslerauthor=C.+Luchinatauthor=B.+Meyerauthor=H.+Oschkinatauthor=J.+Pengauthor=H.+Schwalbeauthor=G.+Siegal&title=Perspectives+on+NMR+in+drug+discovery%3A+a+technique+comes+of+age&doi=10.1038%2Fnrd2606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Perspectives on NMR in drug discovery: a technique comes of age</span></div><div class="casAuthors">Pellecchia, Maurizio; Bertini, Ivano; Cowburn, David; Dalvit, Claudio; Giralt, Ernest; Jahnke, Wolfgang; James, Thomas L.; Homans, Steve W.; Kessler, Horst; Luchinat, Claudio; Meyer, Bernd; Oschkinat, Hartmut; Peng, Jeff; Schwalbe, Harald; Siegal, Gregg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">738-745</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the past decade, the ability of NMR spectroscopy to provide information on intermol. interactions that is valuable in drug discovery has been increasingly appreciated.  Pellecchia and colleagues provide their collective evaluation of the major applications of NMR in drug discovery, focusing on hit and lead generation, and critically analyze its current and potential utility.  In the past decade, the potential of harnessing the ability of NMR spectroscopy to monitor intermol. interactions as a tool for drug discovery has been increasingly appreciated in academia and industry.  In this Perspective, we highlight some of the major applications of NMR in drug discovery, focusing on hit and lead generation, and provide a crit. anal. of its current and potential utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgotirEZMnYbVg90H21EOLACvtfcHk0ljIfD-380_Lug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVGgtL%252FK&md5=b88497140e0f2862c9dd434e548a56c5</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1038%2Fnrd2606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2606%26sid%3Dliteratum%253Aachs%26aulast%3DPellecchia%26aufirst%3DM.%26aulast%3DBertini%26aufirst%3DI.%26aulast%3DCowburn%26aufirst%3DD.%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DGiralt%26aufirst%3DE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJames%26aufirst%3DT.%2BL.%26aulast%3DHomans%26aufirst%3DS.%2BW.%26aulast%3DKessler%26aufirst%3DH.%26aulast%3DLuchinat%26aufirst%3DC.%26aulast%3DMeyer%26aufirst%3DB.%26aulast%3DOschkinat%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DSchwalbe%26aufirst%3DH.%26aulast%3DSiegal%26aufirst%3DG.%26atitle%3DPerspectives%2520on%2520NMR%2520in%2520drug%2520discovery%253A%2520a%2520technique%2520comes%2520of%2520age%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D738%26epage%3D745%26doi%3D10.1038%2Fnrd2606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michnick, S. W.</span></span> <span> </span><span class="NLM_article-title">Application of protein-fragment complementation assays in cell biology</span>. <i>BioTechniques</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">137</span>, 139, 141 passim <span class="refDoi"> DOI: 10.2144/000112396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.2144%2F000112396" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=137&author=I.+Remyauthor=S.+W.+Michnick&title=Application+of+protein-fragment+complementation+assays+in+cell+biology&doi=10.2144%2F000112396"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.2144%2F000112396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2144%252F000112396%26sid%3Dliteratum%253Aachs%26aulast%3DRemy%26aufirst%3DI.%26aulast%3DMichnick%26aufirst%3DS.%2BW.%26atitle%3DApplication%2520of%2520protein-fragment%2520complementation%2520assays%2520in%2520cell%2520biology%26jtitle%3DBioTechniques%26date%3D2007%26volume%3D42%26spage%3D137%26doi%3D10.2144%2F000112396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref150"><div class="reference"><strong class="refLabel"><a href="#ref150" class="rightTabRefNumLink">150</a></strong><div class="NLM_citation" id="rightTab-cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westwick, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamerdin, J. E.</span></span> <span> </span><span class="NLM_article-title">Improving drug discovery with contextual assays and cellular systems analysis</span>. <i>Methods Mol. Biol. (N. Y., NY, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>756</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-160-4_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1007%2F978-1-61779-160-4_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21870220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlansrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=756&publication_year=2011&pages=61-73&author=J.+K.+Westwickauthor=J.+E.+Lamerdin&title=Improving+drug+discovery+with+contextual+assays+and+cellular+systems+analysis&doi=10.1007%2F978-1-61779-160-4_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Improving drug discovery with contextual assays and cellular systems analysis</span></div><div class="casAuthors">Westwick, John K.; Lamerdin, Jane E.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">756</span>
        (<span class="NLM_cas:issue">Signal Transduction Protocols</span>),
    <span class="NLM_cas:pages">61-73</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Despite rapid growth in our knowledge of potential disease targets following completion of the first drafts of the human genome over 10 years ago, the success rate of new therapeutic discovery has been frustratingly low.  In addn. to the widely reported costs and single-digit success rate of the entire drug discovery and development process, it has recently been estd. that even the preliminary process of transitioning new targets to preclin. development succeeds in less than 3% of attempts.  At these early stages of development, poor understanding of therapeutic mechanisms and lack of compd. selectivity are often to blame for failed compds.  It is worth noting than the emerging class of nucleic acid-based therapeutics, including miRNA and RNAi, are likely to be even more prone to unexpected system-wide and off-target activities.  For all therapeutic approaches, it is clear that discovery strategies permitting the assessment of drug targets in their native context are required.  At the same time, these strategies need to retain the high throughput of current reductionist approaches to enable broad assessment of chem. space for small mol. and genetic therapeutics.  We describe here an integrated system based on high-content cellular anal. combined with system-wide pathway interrogation.  The platform can be applied to novel therapeutic target and drug candidate identification, and for providing detailed mechanistic and selectivity information at an early stage of development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqmwd5cwc7frVg90H21EOLACvtfcHk0ljsCSMqKkdBGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlansrjN&md5=98aecd801b4ed3f9fdb3f91df5f182e2</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-160-4_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-160-4_3%26sid%3Dliteratum%253Aachs%26aulast%3DWestwick%26aufirst%3DJ.%2BK.%26aulast%3DLamerdin%26aufirst%3DJ.%2BE.%26atitle%3DImproving%2520drug%2520discovery%2520with%2520contextual%2520assays%2520and%2520cellular%2520systems%2520analysis%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2011%26volume%3D756%26spage%3D61%26epage%3D73%26doi%3D10.1007%2F978-1-61779-160-4_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref151"><div class="reference"><strong class="refLabel"><a href="#ref151" class="rightTabRefNumLink">151</a></strong><div class="NLM_citation" id="rightTab-cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E. M.</span>; <span class="NLM_string-name">Hansen, T. M.</span>; <span class="NLM_string-name">Richardson, P. L.</span>; <span class="NLM_string-name">Wilcox, D. M.</span>; <span class="NLM_string-name">Addo, S. N.</span>; <span class="NLM_string-name">Tang, H.</span>; <span class="NLM_string-name">Sun, C.</span>; <span class="NLM_string-name">Petros, A.</span>; <span class="NLM_string-name">Panchal, S.</span>; <span class="NLM_string-name">Park, C.</span>; <span class="NLM_string-name">Nottoli, M. S.</span>; <span class="NLM_string-name">Patterson, M. J.</span>; <span class="NLM_string-name">Buchanan, G.</span>; <span class="NLM_string-name">Cheng, D.</span>; <span class="NLM_string-name">Davis, H. M.</span>; <span class="NLM_string-name">Calderwood, D. J.</span>; <span class="NLM_string-name">Elmore, S. W.</span>; <span class="NLM_string-name">Shen, Y.</span></span>, unpublished data.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Warder%2C+S.+E.+M.%3B+Hansen%2C+T.+M.%3B+Richardson%2C+P.+L.%3B+Wilcox%2C+D.+M.%3B+Addo%2C+S.+N.%3B+Tang%2C+H.%3B+Sun%2C+C.%3B+Petros%2C+A.%3B+Panchal%2C+S.%3B+Park%2C+C.%3B+Nottoli%2C+M.+S.%3B+Patterson%2C+M.+J.%3B+Buchanan%2C+G.%3B+Cheng%2C+D.%3B+Davis%2C+H.+M.%3B+Calderwood%2C+D.+J.%3B+Elmore%2C+S.+W.%3B+Shen%2C+Y.%2C+unpublished+data."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DWarder%26aufirst%3DS.%2BE.%2BM." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref152"><div class="reference"><strong class="refLabel"><a href="#ref152" class="rightTabRefNumLink">152</a></strong><div class="NLM_citation" id="rightTab-cit152"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. F.</span></span> <span> </span><span class="NLM_article-title">Brd4 modulates the innate immune response through Mnk2-eIF4E pathway-dependent translational control of IkappaBalpha</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E3993</span>– <span class="NLM_lpage">E4001</span>, <span class="refDoi"> DOI: 10.1073/pnas.1700109114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.1700109114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E3993-E4001&author=Y.+Baoauthor=X.+Wuauthor=J.+Chenauthor=X.+Huauthor=F.+Zengauthor=J.+Chengauthor=H.+Jinauthor=X.+Linauthor=L.+F.+Chen&title=Brd4+modulates+the+innate+immune+response+through+Mnk2-eIF4E+pathway-dependent+translational+control+of+IkappaBalpha&doi=10.1073%2Fpnas.1700109114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1700109114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1700109114%26sid%3Dliteratum%253Aachs%26aulast%3DBao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DF.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBrd4%2520modulates%2520the%2520innate%2520immune%2520response%2520through%2520Mnk2-eIF4E%2520pathway-dependent%2520translational%2520control%2520of%2520IkappaBalpha%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE3993%26epage%3DE4001%26doi%3D10.1073%2Fpnas.1700109114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref153"><div class="reference"><strong class="refLabel"><a href="#ref153" class="rightTabRefNumLink">153</a></strong><div class="NLM_citation" id="rightTab-cit153"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span>; <span class="NLM_string-name">Robinson, K. P. J.</span>; <span class="NLM_string-name">Oyer, J. A.</span>; <span class="NLM_string-name">Kifle, L. X.</span>; <span class="NLM_string-name">Kurnick, M. D.</span>; <span class="NLM_string-name">Clapham, B.</span>; <span class="NLM_string-name">Pliushchev, M. A.</span>; <span class="NLM_string-name">Sweis, R. X.</span>; <span class="NLM_string-name">Tang, H.</span>; <span class="NLM_string-name">McLoughlin, S. M.</span>; <span class="NLM_string-name">Jhala, M. A.</span>; <span class="NLM_string-name">Comess, K. M.</span>; <span class="NLM_string-name">Shannon, D. A.</span>; <span class="NLM_string-name">Galasinski, S. C.</span></span>, unpublished data.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Warder%2C+S.+E.%3B+Robinson%2C+K.+P.+J.%3B+Oyer%2C+J.+A.%3B+Kifle%2C+L.+X.%3B+Kurnick%2C+M.+D.%3B+Clapham%2C+B.%3B+Pliushchev%2C+M.+A.%3B+Sweis%2C+R.+X.%3B+Tang%2C+H.%3B+McLoughlin%2C+S.+M.%3B+Jhala%2C+M.+A.%3B+Comess%2C+K.+M.%3B+Shannon%2C+D.+A.%3B+Galasinski%2C+S.+C.%2C+unpublished+data."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DWarder%26aufirst%3DS.%2BE." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref154"><div class="reference"><strong class="refLabel"><a href="#ref154" class="rightTabRefNumLink">154</a></strong><div class="NLM_citation" id="rightTab-cit154"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarvliden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogvadze, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgenstern, R.</span></span> <span> </span><span class="NLM_article-title">Multiple roles of microsomal glutathione transferase 1 in cellular protection: a mechanistic study</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1638</span>– <span class="NLM_lpage">1645</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2010.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.freeradbiomed.2010.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=20727966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKktLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=1638-1645&author=K.+Johanssonauthor=J.+Jarvlidenauthor=V.+Gogvadzeauthor=R.+Morgenstern&title=Multiple+roles+of+microsomal+glutathione+transferase+1+in+cellular+protection%3A+a+mechanistic+study&doi=10.1016%2Fj.freeradbiomed.2010.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple roles of microsomal glutathione transferase 1 in cellular protection: A mechanistic study</span></div><div class="casAuthors">Johansson, Katarina; Jaervliden, Julia; Gogvadze, Vladimir; Morgenstern, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1638-1645</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The aim of this study was to investigate the involvement of membrane-bound microsomal glutathione transferase 1 (MGST1) in cellular resistance against oxidative stress as well as its mechanism of protection.  MGST1 is ubiquitously expressed and predominantly located in the endoplasmic reticulum and outer mitochondrial membrane.  Utilizing MCF7 cells over-expressing MGST1 significant protection against agents that are known to induce lipid peroxidn. (e.g., cumene hydroperoxide and tert-butylhydroperoxide) and an end-product of lipid peroxidn. (e.g., 4-hydroxy-2-nonenal) was shown.  Furthermore, the results demonstrate that MGST1 protection can be enhanced by vitamin E when toxicity depends on oxidative stress, but not when direct alkylation is the dominant mechanism.  Mitochondria in MGST1-over-expressing cells were shown to be protected from oxidative insult as measured by calcium loading capacity and respiration.  MGST1 induces cellular resistance against cisplatin.  Here vitamin E was used to elucidate whether oxidative stress caused by cisplatin is significant for cell toxicity.  The results indicate that oxidative stress and induction of lipid peroxidn. are not the most prominent toxic mechanism of cisplatin in the cell system.  It is thus concluded that MGST1 protects cells (and mitochondria) by both conjugation and glutathione peroxidase functions.  A new protective mechanism against cisplatin is also indicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovM-cagT_i4LVg90H21EOLACvtfcHk0ljsCSMqKkdBGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKktLnO&md5=b9cddef5cab3820279d67f46b19e93ea</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2010.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2010.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DK.%26aulast%3DJarvliden%26aufirst%3DJ.%26aulast%3DGogvadze%26aufirst%3DV.%26aulast%3DMorgenstern%26aufirst%3DR.%26atitle%3DMultiple%2520roles%2520of%2520microsomal%2520glutathione%2520transferase%25201%2520in%2520cellular%2520protection%253A%2520a%2520mechanistic%2520study%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2010%26volume%3D49%26spage%3D1638%26epage%3D1645%26doi%3D10.1016%2Fj.freeradbiomed.2010.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref155"><div class="reference"><strong class="refLabel"><a href="#ref155" class="rightTabRefNumLink">155</a></strong><div class="NLM_citation" id="rightTab-cit155"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aniya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaizumi, N.</span></span> <span> </span><span class="NLM_article-title">Mitochondrial glutathione transferases involving a new function for membrane permeability transition pore regulation</span>. <i>Drug Metab. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">292</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.3109/03602532.2011.552913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.3109%2F03602532.2011.552913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21428695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslCisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=292-299&author=Y.+Aniyaauthor=N.+Imaizumi&title=Mitochondrial+glutathione+transferases+involving+a+new+function+for+membrane+permeability+transition+pore+regulation&doi=10.3109%2F03602532.2011.552913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit155R"><div class="casContent"><span class="casTitleNuber">155</span><div class="casTitle"><span class="NLM_cas:atitle">Mitochondrial glutathione transferases involving a new function for membrane permeability transition pore regulation</span></div><div class="casAuthors">Aniya, Yoko; Imaizumi, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">292-299</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The mitochondria in mammalian cells are a predominant resource of reactive oxygen species (ROS), which are produced during respiration-coupled oxidative metab. or various chem. stresses.  End-products from membrane-lipid peroxidn. caused by ROS are highly toxic, thereby their elimination/scavenging are protective of mitochondria and cells against oxidative damages.  In mitochondria, sol. (kappa, alpha, mu, pi, zeta) and membrane-bound glutathione transferases (GSTs) (MGST1) are distributed.  Mitochondrial GSTs display both glutathione transferase and peroxidase activities that detoxify such harmful products through glutathione (GSH) conjugation or GSH-mediated peroxide redn.  Some GST isoenzymes are induced by oxidative stress, an adaptation mechanism for the protection of cells from oxidative stress.  Membrane-bound MGST1 is activated through the thiol modification in oxidative conditions.  Protective action of MGST1 against oxidative stress has been confirmed using MCF7 cells highly expressed of MGST1.  In recent years, mitochondria have been recognized as a regulator of cell death via both apoptosis and necrosis, where oxidative stress-induced alteration of the membrane permeability is an important step.  Recent studies have shown that MGST1 in the inner mitochondrial membrane could interact with the mitochondrial permeability transition (MPT) regulator proteins, such as adenine nucleotide translocator (ANT) and/or cyclophilin D, and could contribute to oxidant-induced MPT pores.  Interaction of GST alpha with ANT has also been shown.  In this review, functions of the mitochondrial GSTs, including a new role for mitochondria-mediated cell death, are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwWno4AaI0r7Vg90H21EOLACvtfcHk0ligNbxGwm3ZZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslCisbY%253D&md5=baa5ea91a6754eaeb1800a9dc702f3b9</span></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.3109%2F03602532.2011.552913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2011.552913%26sid%3Dliteratum%253Aachs%26aulast%3DAniya%26aufirst%3DY.%26aulast%3DImaizumi%26aufirst%3DN.%26atitle%3DMitochondrial%2520glutathione%2520transferases%2520involving%2520a%2520new%2520function%2520for%2520membrane%2520permeability%2520transition%2520pore%2520regulation%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2011%26volume%3D43%26spage%3D292%26epage%3D299%26doi%3D10.3109%2F03602532.2011.552913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref156"><div class="reference"><strong class="refLabel"><a href="#ref156" class="rightTabRefNumLink">156</a></strong><div class="NLM_citation" id="rightTab-cit156"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yvone, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marton, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crenshaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winckler, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">High-throughput identification of genotype-specific cancer vulnerabilities in mixtures of barcoded tumor cell lines</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">423</span>, <span class="refDoi"> DOI: 10.1038/nbt.3460</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnbt.3460" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=26928769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC28XivFKgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=419-423&author=C.+Yuauthor=A.+M.+Mannanauthor=G.+M.+Yvoneauthor=K.+N.+Rossauthor=Y.+L.+Zhangauthor=M.+A.+Martonauthor=B.+R.+Taylorauthor=A.+Crenshawauthor=J.+Z.+Gouldauthor=P.+Tamayoauthor=B.+A.+Weirauthor=A.+Tsherniakauthor=B.+Wongauthor=L.+A.+Garrawayauthor=A.+F.+Shamjiauthor=M.+A.+Palmerauthor=M.+A.+Foleyauthor=W.+Wincklerauthor=S.+L.+Schreiberauthor=A.+L.+Kungauthor=T.+R.+Golub&title=High-throughput+identification+of+genotype-specific+cancer+vulnerabilities+in+mixtures+of+barcoded+tumor+cell+lines&doi=10.1038%2Fnbt.3460"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput identification of genotype-specific cancer vulnerabilities in mixtures of barcoded tumor cell lines</span></div><div class="casAuthors">Yu, Channing; Mannan, Aristotle M.; Yvone, Griselda Metta; Ross, Kenneth N.; Zhang, Yan-Ling; Marton, Melissa A.; Taylor, Bradley R.; Crenshaw, Andrew; Gould, Joshua Z.; Tamayo, Pablo; Weir, Barbara A.; Tsherniak, Aviad; Wong, Bang; Garraway, Levi A.; Shamji, Alykhan F.; Palmer, Michelle A.; Foley, Michael A.; Winckler, Wendy; Schreiber, Stuart L.; Kung, Andrew L.; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">419-423</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hundreds of genetically characterized cell lines are available for the discovery of genotype-specific cancer vulnerabilities.  However, screening large nos. of compds. against large nos. of cell lines is currently impractical, and such expts. are often difficult to control.  Here we report a method called PRISM that allows pooled screening of mixts. of cancer cell lines by labeling each cell line with 24-nucleotide barcodes.  PRISM revealed the expected patterns of cell killing seen in conventional (unpooled) assays.  In a screen of 102 cell lines across 8,400 compds., PRISM led to the identification of BRD-7880 as a potent and highly specific inhibitor of aurora kinases B and C.  Cell line pools also efficiently formed tumors as xenografts, and PRISM recapitulated the expected pattern of erlotinib sensitivity in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvZWopzWWOhrVg90H21EOLACvtfcHk0ligNbxGwm3ZZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivFKgsro%253D&md5=8c5206e6e3dd2fe19673acc9eed04956</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3460%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DC.%26aulast%3DMannan%26aufirst%3DA.%2BM.%26aulast%3DYvone%26aufirst%3DG.%2BM.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DZhang%26aufirst%3DY.%2BL.%26aulast%3DMarton%26aufirst%3DM.%2BA.%26aulast%3DTaylor%26aufirst%3DB.%2BR.%26aulast%3DCrenshaw%26aufirst%3DA.%26aulast%3DGould%26aufirst%3DJ.%2BZ.%26aulast%3DTamayo%26aufirst%3DP.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DPalmer%26aufirst%3DM.%2BA.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DHigh-throughput%2520identification%2520of%2520genotype-specific%2520cancer%2520vulnerabilities%2520in%2520mixtures%2520of%2520barcoded%2520tumor%2520cell%2520lines%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26spage%3D419%26epage%3D423%26doi%3D10.1038%2Fnbt.3460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref157"><div class="reference"><strong class="refLabel"><a href="#ref157" class="rightTabRefNumLink">157</a></strong><div class="NLM_citation" id="rightTab-cit157"><span><span class="NLM_contrib-group">The QSTAR Consortium</span> <span> </span><span class="NLM_article-title">Using transcriptomics to guide lead optimization in
drug discovery projects: Lessons learned from the QSTAR project</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.12.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1016%2Fj.drudis.2014.12.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=25582842" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=505-513&author=The+QSTAR+Consortium&title=Using+transcriptomics+to+guide+lead+optimization+in%0Adrug+discovery+projects%3A+Lessons+learned+from+the+QSTAR+project&doi=10.1016%2Fj.drudis.2014.12.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.12.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.12.014%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUsing%2520transcriptomics%2520to%2520guide%2520lead%2520optimization%2520in%250Adrug%2520discovery%2520projects%253A%2520Lessons%2520learned%2520from%2520the%2520QSTAR%2520project%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D505%26epage%3D513%26doi%3D10.1016%2Fj.drudis.2014.12.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref158"><div class="reference"><strong class="refLabel"><a href="#ref158" class="rightTabRefNumLink">158</a></strong><div class="NLM_citation" id="rightTab-cit158"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Awale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J. L.</span></span> <span> </span><span class="NLM_article-title">The polypharmacology browser: a web-based multi-fingerprint target prediction tool using ChEMBL bioactivity data</span>. <i>J. Cheminformatics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">11</span>, <span class="refDoi"> DOI: 10.1186/s13321-017-0199-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1186%2Fs13321-017-0199-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlsFGhtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=11&author=M.+Awaleauthor=J.+L.+Reymond&title=The+polypharmacology+browser%3A+a+web-based+multi-fingerprint+target+prediction+tool+using+ChEMBL+bioactivity+data&doi=10.1186%2Fs13321-017-0199-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">The polypharmacology browser: a web-based multi-fingerprint target prediction tool using ChEMBL bioactivity data</span></div><div class="casAuthors">Awale, Mahendra; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11/1-11/10</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Several web-based tools have been reported recently which predict the possible targets of a small mol. by similarity to compds. of known bioactivity using mol. fingerprints (fps), however predictions in each case rely on similarities computed from only one or two fps.  Considering that structural similarity and therefore the predicted targets strongly depend on the method used for comparison, it would be highly desirable to predict targets using a broader set of fps simultaneously.  Results: Herein, we present the polypharmacol. browser (PPB), a web-based platform which predicts possible targets for small mols. by searching for nearest neighbors using ten different fps describing compn., substructures, mol. shape and pharmacophores.  PPB searches through 4613 groups of at least 10 same target annotated bioactive mols. from ChEMBL and returns a list of predicted targets ranked by consensus voting scheme and p value.  A validation study across 670 drugs with up to 20 targets showed that combining the predictions from all 10 fps gives the best results, with on av. 50% of the known targets of a drug being correctly predicted with a hit rate of 25%.  Furthermore, when profiling a new inhibitor of the calcium channel TRPV6 against 24 targets taken from a safety screen panel, we obsd. inhibition in 5 out of 5 targets predicted by PPB and in 7 out of 18 targets not predicted by PPB.  The rate of correct (5/12) and incorrect (0/12) predictions for this compd. by PPB was comparable to that of other web-based prediction tools.  Conclusion: PPB offers a versatile platform for target prediction based on multi-fingerprint comparisons, and is freely accessible at www.gdb.unibe.ch as a valuable support for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtd4w_kv537bVg90H21EOLACvtfcHk0lho_uEZ7XRUYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlsFGhtr0%253D&md5=20e3519d493380139f5cd61974017cec</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1186%2Fs13321-017-0199-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-017-0199-x%26sid%3Dliteratum%253Aachs%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DThe%2520polypharmacology%2520browser%253A%2520a%2520web-based%2520multi-fingerprint%2520target%2520prediction%2520tool%2520using%2520ChEMBL%2520bioactivity%2520data%26jtitle%3DJ.%2520Cheminformatics%26date%3D2017%26volume%3D9%26spage%3D11%26doi%3D10.1186%2Fs13321-017-0199-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref159"><div class="reference"><strong class="refLabel"><a href="#ref159" class="rightTabRefNumLink">159</a></strong><div class="NLM_citation" id="rightTab-cit159"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagunin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stepanchikova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filimonov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poroikov, V.</span></span> <span> </span><span class="NLM_article-title">PASS: prediction of activity spectra for biologically active substances</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">747</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/16.8.747</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbioinformatics%2F16.8.747" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=11099264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVGms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2000&pages=747-748&author=A.+Laguninauthor=A.+Stepanchikovaauthor=D.+Filimonovauthor=V.+Poroikov&title=PASS%3A+prediction+of+activity+spectra+for+biologically+active+substances&doi=10.1093%2Fbioinformatics%2F16.8.747"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">PASS: prediction of activity spectra for biologically active substances</span></div><div class="casAuthors">Lagunin, Alexey; Stepanchikova, Alla; Filimonov, Dmitrii; Poroikov, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">747-748</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The concept of the biol. activity spectrum was introduced to describe the properties of biol. active substances.  The PASS (prediction of activity spectra for substances) software product, which predicts more than 300 pharmacol. effects and biochem. mechanisms on the basis of the structural formula of a substance, may be efficiently used to find new targets for some ligands and, conversely, to reveal new ligands for some biol. targets.  A WWW interface has been developed for the PASS software.  A WWW server for the online prediction of the biol. activity spectra of substances has been constructed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeV2yQu6YNBbVg90H21EOLACvtfcHk0lho_uEZ7XRUYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVGms7k%253D&md5=f3b098eabf8c73b7285fead383a2a34f</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2F16.8.747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252F16.8.747%26sid%3Dliteratum%253Aachs%26aulast%3DLagunin%26aufirst%3DA.%26aulast%3DStepanchikova%26aufirst%3DA.%26aulast%3DFilimonov%26aufirst%3DD.%26aulast%3DPoroikov%26aufirst%3DV.%26atitle%3DPASS%253A%2520prediction%2520of%2520activity%2520spectra%2520for%2520biologically%2520active%2520substances%26jtitle%3DBioinformatics%26date%3D2000%26volume%3D16%26spage%3D747%26epage%3D748%26doi%3D10.1093%2Fbioinformatics%2F16.8.747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref160"><div class="reference"><strong class="refLabel"><a href="#ref160" class="rightTabRefNumLink">160</a></strong><div class="NLM_citation" id="rightTab-cit160"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">TarFisDock: a web server for identifying drug targets with docking approach</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">W219</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgkl114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=16844997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=W219-224&author=H.+Liauthor=Z.+Gaoauthor=L.+Kangauthor=H.+Zhangauthor=K.+Yangauthor=K.+Yuauthor=X.+Luoauthor=W.+Zhuauthor=K.+Chenauthor=J.+Shenauthor=X.+Wangauthor=H.+Jiang&title=TarFisDock%3A+a+web+server+for+identifying+drug+targets+with+docking+approach&doi=10.1093%2Fnar%2Fgkl114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">TarFisDock: a web server for identifying drug targets with docking approach</span></div><div class="casAuthors">Li, Honglin; Gao, Zhenting; Kang, Ling; Zhang, Hailei; Yang, Kun; Yu, Kunqian; Luo, Xiaomin; Zhu, Weiliang; Chen, Kaixian; Shen, Jianhua; Wang, Xicheng; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W219-W224</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">TarFisDock is a web-based tool for automating the procedure of searching for small mol.-protein interactions over a large repertoire of protein structures.  It offers PDTD (potential drug target database), a target database contg. 698 protein structures covering 15 therapeutic areas and a reverse ligand-protein docking program.  In contrast to conventional ligand-protein docking, reverse ligand-protein docking aims to seek potential protein targets by screening an appropriate protein database.  The input file of this web server is the small mol. to be tested, in std. mol2 format; TarFisDock then searches for possible binding proteins for the given small mol. by use of a docking approach.  The ligand-protein interaction energy terms of the program DOCK are adopted for ranking the proteins.  To test the reliability of the TarFisDock server, we searched the PDTD for putative binding proteins for vitamin E and 4H-tamoxifen.  The top 2 and 10% candidates of vitamin E binding proteins identified by TarFisDock resp. cover 30 and 50% of reported targets verified or implicated by expts.; and 30 and 50% of exptl. confirmed targets for 4H-tamoxifen appear amongst the top 2 and 5% of the TarFisDock predicted candidates, resp.  Therefore, TarFisDock may be a useful tool for target identification, mechanism study of old drugs and probes discovered from natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFIwhF2-sUb7Vg90H21EOLACvtfcHk0lho_uEZ7XRUYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1yit7s%253D&md5=39f644efb79cea93e654549f6e340c30</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl114%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DKang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DTarFisDock%253A%2520a%2520web%2520server%2520for%2520identifying%2520drug%2520targets%2520with%2520docking%2520approach%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW219%26epage%3D224%26doi%3D10.1093%2Fnar%2Fgkl114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref161"><div class="reference"><strong class="refLabel"><a href="#ref161" class="rightTabRefNumLink">161</a></strong><div class="NLM_citation" id="rightTab-cit161"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armbruster, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernsberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lhvUabYo2m6-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref162"><div class="reference"><strong class="refLabel"><a href="#ref162" class="rightTabRefNumLink">162</a></strong><div class="NLM_citation" id="rightTab-cit162"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">PharmMapper server: a web server for potential drug target identification using pharmacophore mapping approach</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">W609</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgkq300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=20430828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W609-614&author=X.+Liuauthor=S.+Ouyangauthor=B.+Yuauthor=Y.+Liuauthor=K.+Huangauthor=J.+Gongauthor=S.+Zhengauthor=Z.+Liauthor=H.+Liauthor=H.+Jiang&title=PharmMapper+server%3A+a+web+server+for+potential+drug+target+identification+using+pharmacophore+mapping+approach&doi=10.1093%2Fnar%2Fgkq300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">PharmMapper server: a web server for potential drug target identification using pharmacophore mapping approach</span></div><div class="casAuthors">Liu, Xiaofeng; Ouyang, Sisheng; Yu, Biao; Liu, Yabo; Huang, Kai; Gong, Jiayu; Zheng, Siyuan; Li, Zhihua; Li, Honglin; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W609-W614</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In silico drug target identification, which includes many distinct algorithms for finding disease genes and proteins, is the first step in the drug discovery pipeline.  When the 3D structures of the targets are available, the problem of target identification is usually converted to finding the best interaction mode between the potential target candidates and small mol. probes.  Pharmacophore, which is the spatial arrangement of features essential for a mol. to interact with a specific target receptor, is an alternative method for achieving this goal apart from mol. docking method.  PharmMapper server is a freely accessed web server designed to identify potential target candidates for the given small mols. (drugs, natural products or other newly discovered compds. with unidentified binding targets) using pharmacophore mapping approach.  PharmMapper hosts a large, inhouse repertoire of pharmacophore database (namely PharmTargetDB) annotated from all the targets information in TargetBank, BindingDB, DrugBank and potential drug target database, including over 7000 receptor-based pharmacophore models (covering over 1500 drug targets information).  PharmMapper automatically finds the best mapping poses of the query mol. against all the pharmacophore models in PharmTargetDB and lists the top N best-fitted hits with appropriate target annotations, as well as resp. mol.'s aligned poses are presented.  Benefited from the highly efficient and robust triangle hashing mapping method, PharmMapper bears high throughput ability and only costs 1 h averagely to screen the whole PharmTargetDB.  The protocol was successful in finding the proper targets among the top 300 pharmacophore candidates in the retrospective benchmarking test of tamoxifen.  PharmMapper is available at http://59.78.96.61/pharmmapper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL-Gaa6UM_zLVg90H21EOLACvtfcHk0lhvUabYo2m6-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWjsL4%253D&md5=dadfc5a278243d22f2ea041914461e3a</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq300%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DPharmMapper%2520server%253A%2520a%2520web%2520server%2520for%2520potential%2520drug%2520target%2520identification%2520using%2520pharmacophore%2520mapping%2520approach%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW609%26epage%3D614%26doi%3D10.1093%2Fnar%2Fgkq300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref163"><div class="reference"><strong class="refLabel"><a href="#ref163" class="rightTabRefNumLink">163</a></strong><div class="NLM_citation" id="rightTab-cit163"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikailov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span> <span> </span><span class="NLM_article-title">DRAR-CPI: a server for identifying drug repositioning potential and adverse drug reactions via the chemical-protein interactome</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">W492</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgkr299" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=21558322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVOntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=W492-498&author=H.+Luoauthor=J.+Chenauthor=L.+Shiauthor=M.+Mikailovauthor=H.+Zhuauthor=K.+Wangauthor=L.+Heauthor=L.+Yang&title=DRAR-CPI%3A+a+server+for+identifying+drug+repositioning+potential+and+adverse+drug+reactions+via+the+chemical-protein+interactome&doi=10.1093%2Fnar%2Fgkr299"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">DRAR-CPI: a server for identifying drug repositioning potential and adverse drug reactions via the chemical-protein interactome</span></div><div class="casAuthors">Luo, Heng; Chen, Jian; Shi, Leming; Mikailov, Mike; Zhu, Huang; Wang, Kejian; He, Lin; Yang, Lun</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W492-W498</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Identifying new indications for existing drugs (drug repositioning) is an efficient way of maximizing their potential.  Adverse drug reaction (ADR) is one of the leading causes of death among hospitalized patients.  As both new indications and ADRs are caused by unexpected chem.-protein interactions on off-targets, it is reasonable to predict these interactions by mining the chem.-protein interactome (CPI).  Making such predictions has recently been facilitated by a web server named DRAR-CPI.  This server has a representative collection of drug mols. and targetable human proteins built up from our work in drug repositioning and ADR.  When a user submits a mol., the server will give the pos. or neg. assocn. scores between the user's mol. and our library drugs based on their interaction profiles towards the targets.  Users can thus predict the indications or ADRs of their mol. based on the assocn. scores towards our library drugs.  We have matched our predictions of drug-drug assocns. with those predicted via gene-expression profiles, achieving a matching rate as high as 74%.  We have also successfully predicted the connections between anti-psychotics and anti-infectives, indicating the underlying relevance of anti-psychotics in the potential treatment of infections, vice versa.  This server is freely available at http://cpi.bio-x.cn/drar/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrltOyRWjrcwLVg90H21EOLACvtfcHk0lhvUabYo2m6-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosVOntLg%253D&md5=5964882fff9031297e60b228d2217ece</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr299%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DMikailov%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DL.%26atitle%3DDRAR-CPI%253A%2520a%2520server%2520for%2520identifying%2520drug%2520repositioning%2520potential%2520and%2520adverse%2520drug%2520reactions%2520via%2520the%2520chemical-protein%2520interactome%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DW492%26epage%3D498%26doi%3D10.1093%2Fnar%2Fgkr299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref164"><div class="reference"><strong class="refLabel"><a href="#ref164" class="rightTabRefNumLink">164</a></strong><div class="NLM_citation" id="rightTab-cit164"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wipf, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X. Q.</span></span> <span> </span><span class="NLM_article-title">TargetHunter: an in silico target identification tool for predicting therapeutic potential of small organic molecules based on chemogenomic database</span>. <i>AAPS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1208/s12248-012-9449-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1208%2Fs12248-012-9449-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23292636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsF2hsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=395-406&author=L.+Wangauthor=C.+Maauthor=P.+Wipfauthor=H.+Liuauthor=W.+Suauthor=X.+Q.+Xie&title=TargetHunter%3A+an+in+silico+target+identification+tool+for+predicting+therapeutic+potential+of+small+organic+molecules+based+on+chemogenomic+database&doi=10.1208%2Fs12248-012-9449-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">TargetHunter: An In Silico Target Identification Tool for Predicting Therapeutic Potential of Small Organic Molecules Based on Chemogenomic Database</span></div><div class="casAuthors">Wang, Lirong; Ma, Chao; Wipf, Peter; Liu, Haibin; Su, Weiwei; Xie, Xiang-Qun</div><div class="citationInfo"><span class="NLM_cas:title">AAPS Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">395-406</span>CODEN:
                <span class="NLM_cas:coden">AJAOB6</span>;
        ISSN:<span class="NLM_cas:issn">1550-7416</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Target identification of the known bioactive compds. and novel synthetic analogs is a very important research field in medicinal chem., biochem., and pharmacol.  It is also a challenging and costly step towards chem. biol. and phenotypic screening.  In silico identification of potential biol. targets for chem. compds. offers an alternative avenue for the exploration of ligand-target interactions and biochem. mechanisms, as well as for investigation of drug repurposing.  Computational target fishing mines biol. annotated chem. databases and then maps compd. structures into chemogenomical space in order to predict the biol. targets.  We summarize the recent advances and applications in computational target fishing, such as chem. similarity searching, data mining/machine learning, panel docking, and the bioactivity spectral anal. for target identification.  We then described in detail a new web-based target prediction tool, TargetHunter (http://www.cbligand.org/TargetHunter).  This web portal implements a novel in silico target prediction algorithm, the Targets Assocd. with its MOst SImilar Counterparts, by exploring the largest chemogenomical databases, ChEMBL.  Prediction accuracy reached 91.1% from the top 3 guesses on a subset of high-potency compds. from the ChEMBL database, which outperformed a published algorithm, multiple-category models.  TargetHunter also features an embedded geog. tool, BioassayGeoMap, developed to allow the user easily to search for potential collaborators that can exptl. validate the predicted biol. target(s) or off target(s).  TargetHunter therefore provides a promising alternative to bridge the knowledge gap between biol. and chem., and significantly boost the productivity of chemogenomics researchers for in silico drug design and discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrApY5ImkDQYbVg90H21EOLACvtfcHk0li2yH5a-hdA1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsF2hsLk%253D&md5=964f41bd7191d200529b2f1272b7061c</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1208%2Fs12248-012-9449-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-012-9449-z%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DWipf%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DSu%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%2BQ.%26atitle%3DTargetHunter%253A%2520an%2520in%2520silico%2520target%2520identification%2520tool%2520for%2520predicting%2520therapeutic%2520potential%2520of%2520small%2520organic%2520molecules%2520based%2520on%2520chemogenomic%2520database%26jtitle%3DAAPS%2520J.%26date%3D2013%26volume%3D15%26spage%3D395%26epage%3D406%26doi%3D10.1208%2Fs12248-012-9449-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref165"><div class="reference"><strong class="refLabel"><a href="#ref165" class="rightTabRefNumLink">165</a></strong><div class="NLM_citation" id="rightTab-cit165"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ku, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">ChemMapper: a versatile web server for exploring pharmacology and chemical structure association based on molecular 3D similarity method</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1827</span>– <span class="NLM_lpage">1829</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbioinformatics%2Fbtt270" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23712658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVOjsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=1827-1829&author=J.+Gongauthor=C.+Caiauthor=X.+Liuauthor=X.+Kuauthor=H.+Jiangauthor=D.+Gaoauthor=H.+Li&title=ChemMapper%3A+a+versatile+web+server+for+exploring+pharmacology+and+chemical+structure+association+based+on+molecular+3D+similarity+method&doi=10.1093%2Fbioinformatics%2Fbtt270"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">ChemMapper: a versatile web server for exploring pharmacology and chemical structure association based on molecular 3D similarity method</span></div><div class="casAuthors">Gong, Jiayu; Cai, Chaoqian; Liu, Xiaofeng; Ku, Xin; Jiang, Hualiang; Gao, Daqi; Li, Honglin</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1827-1829</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: ChemMapper is an online platform to predict polypharmacol. effect and mode of action for small mols. based on 3D similarity computation.  ChemMapper collects >350 000 chem. structures with bioactivities and assocd. target annotations (as well as >3 000 000 non-annotated compds. for virtual screening).  Taking the user-provided chem. structure as the query, the top most similar compds. in terms of 3D similarity are returned with assocd. pharmacol. annotations.  ChemMapper is designed to provide versatile services in a variety of chemogenomics, drug repurposing, polypharmacol., novel bioactive compds. identification and scaffold hopping studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaSkcKTpsV3rVg90H21EOLACvtfcHk0li2yH5a-hdA1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVOjsrbN&md5=b13c728b5b001074ab41315be24231e1</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt270%26sid%3Dliteratum%253Aachs%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DKu%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DChemMapper%253A%2520a%2520versatile%2520web%2520server%2520for%2520exploring%2520pharmacology%2520and%2520chemical%2520structure%2520association%2520based%2520on%2520molecular%25203D%2520similarity%2520method%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D1827%26epage%3D1829%26doi%3D10.1093%2Fbioinformatics%2Fbtt270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref166"><div class="reference"><strong class="refLabel"><a href="#ref166" class="rightTabRefNumLink">166</a></strong><div class="NLM_citation" id="rightTab-cit166"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haque, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillos, M.</span></span> <span> </span><span class="NLM_article-title">HitPick: a web server for hit identification and target prediction of chemical screenings</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1910</span>– <span class="NLM_lpage">1912</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbioinformatics%2Fbtt303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=23716196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFClu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=1910-1912&author=X.+Liuauthor=I.+Vogtauthor=T.+Haqueauthor=M.+Campillos&title=HitPick%3A+a+web+server+for+hit+identification+and+target+prediction+of+chemical+screenings&doi=10.1093%2Fbioinformatics%2Fbtt303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">HitPick: a web server for hit identification and target prediction of chemical screenings</span></div><div class="casAuthors">Liu, Xueping; Vogt, Ingo; Haque, Tanzeem; Campillos, Monica</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1910-1912</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: High-throughput phenotypic assays reveal information about the mols. that modulate biol. processes, such as a disease phenotype and a signaling pathway.  In these assays, the identification of hits along with their mol. targets is crit. to understand the chem. activities modulating the biol. system.  Here, we present HitPick, a web server for identification of hits in high-throughput chem. screenings and prediction of their mol. targets.  HitPick applies the B-score method for hit identification and a newly developed approach combining 1-nearest-neighbor (1NN) similarity searching and Laplacian-modified naive Bayesian target models to predict targets of identified hits.  The performance of the HitPick web server is presented and discussed.  Availability: The server can be accessed at http://mips.helmholtz-muenchen.de/proj/hitpick.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="1f727071767c7e317c7e726f767373706c5f777a73727770736b6532726a7a717c777a71317b7a31">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpofqf_952o_LVg90H21EOLACvtfcHk0li2yH5a-hdA1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFClu7vJ&md5=7acb7f7fbd75fa1cc02bd653d888cec3</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt303%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DVogt%26aufirst%3DI.%26aulast%3DHaque%26aufirst%3DT.%26aulast%3DCampillos%26aufirst%3DM.%26atitle%3DHitPick%253A%2520a%2520web%2520server%2520for%2520hit%2520identification%2520and%2520target%2520prediction%2520of%2520chemical%2520screenings%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D1910%26epage%3D1912%26doi%3D10.1093%2Fbioinformatics%2Fbtt303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref167"><div class="reference"><strong class="refLabel"><a href="#ref167" class="rightTabRefNumLink">167</a></strong><div class="NLM_citation" id="rightTab-cit167"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Identifying the macromolecular targets of de novo-designed chemical entities through self-organizing map consensus</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">4067</span>– <span class="NLM_lpage">4072</span>, <span class="refDoi"> DOI: 10.1073/pnas.1320001111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1073%2Fpnas.1320001111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24591595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2msbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=4067-4072&author=D.+Rekerauthor=T.+Rodriguesauthor=P.+Schneiderauthor=G.+Schneider&title=Identifying+the+macromolecular+targets+of+de+novo-designed+chemical+entities+through+self-organizing+map+consensus&doi=10.1073%2Fpnas.1320001111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying the macromolecular targets of de novo-designed chemical entities through self-organizing map consensus</span></div><div class="casAuthors">Reker, Daniel; Rodrigues, Tiago; Schneider, Petra; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4067-4072</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">De novo mol. design and in silico prediction of polypharmacol. profiles are emerging research topics that will profoundly affect the future of drug discovery and chem. biol.  The goal is to identify the macromol. targets of new chem. agents.  Although several computational tools for predicting such targets are publicly available, none of these methods was explicitly designed to predict target engagement by de novo-designed mols.  Here the authors present the development and practical application of a unique technique, self-organizing map-based prediction of drug equivalence relationships (SPiDER), that merges the concepts of self-organizing maps, consensus scoring, and statistical anal. to successfully identify targets for both known drugs and computer-generated mol. scaffolds.  The authors discovered a potential off-target liability of fenofibrate-related compds., and in a comprehensive prospective application, the authors identified a multitarget-modulating profile of de novo designed mols.  These results demonstrate that SPiDER may be used to identify innovative compds. in chem. biol. and in the early stages of drug discovery, and help investigate the potential side effects of drugs and their repurposing options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD_gPUwcpea7Vg90H21EOLACvtfcHk0lipuHaGeBqkpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2msbg%253D&md5=5322c8755c4aeee029288015d8a55d69</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1320001111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1320001111%26sid%3Dliteratum%253Aachs%26aulast%3DReker%26aufirst%3DD.%26aulast%3DRodrigues%26aufirst%3DT.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DIdentifying%2520the%2520macromolecular%2520targets%2520of%2520de%2520novo-designed%2520chemical%2520entities%2520through%2520self-organizing%2520map%2520consensus%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D4067%26epage%3D4072%26doi%3D10.1073%2Fpnas.1320001111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref168"><div class="reference"><strong class="refLabel"><a href="#ref168" class="rightTabRefNumLink">168</a></strong><div class="NLM_citation" id="rightTab-cit168"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gfeller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosdidier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissTargetPrediction: a web server for target prediction of bioactive small molecules</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">W32</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1093/nar/gku293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgku293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24792161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W32-38&author=D.+Gfellerauthor=A.+Grosdidierauthor=M.+Wirthauthor=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissTargetPrediction%3A+a+web+server+for+target+prediction+of+bioactive+small+molecules&doi=10.1093%2Fnar%2Fgku293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">SwissTargetPrediction: a web server for target prediction of bioactive small molecules</span></div><div class="casAuthors">Gfeller, David; Grosdidier, Aurelien; Wirth, Matthias; Daina, Antoine; Michielin, Olivier; Zoete, Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W32-W38</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bioactive small mols., such as drugs or metabolites, bind to proteins or other macro-mol. targets to modulate their activity, which in turn results in the obsd. phenotypic effects.  For this reason, mapping the targets of bioactive small mols. is a key step toward unraveling the mol. mechanisms underlying their bioactivity and predicting potential side effects or cross-reactivity.  Recently, large datasets of protein-small mol. interactions have become available, providing a unique source of information for the development of knowledge-based approaches to computationally identify new targets for uncharacterized mols. or secondary targets for known mols.  Here, the authors introduce SwissTargetPrediction, a web server to accurately predict the targets of bioactive mols. based on a combination of 2D and 3D similarity measures with known ligands.  Predictions can be carried out in five different organisms, and mapping predictions by homol. within and between different species is enabled for close paralogs and orthologs.  SwissTargetPrediction is accessible free of charge and without login requirement at http://www.swisstargetprediction.ch.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXsaXtGKzkj7Vg90H21EOLACvtfcHk0lipuHaGeBqkpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs73E&md5=77811c28978158e1e1aca0669a85b99c</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku293%26sid%3Dliteratum%253Aachs%26aulast%3DGfeller%26aufirst%3DD.%26aulast%3DGrosdidier%26aufirst%3DA.%26aulast%3DWirth%26aufirst%3DM.%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissTargetPrediction%253A%2520a%2520web%2520server%2520for%2520target%2520prediction%2520of%2520bioactive%2520small%2520molecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DW32%26epage%3D38%26doi%3D10.1093%2Fnar%2Fgku293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref169"><div class="reference"><strong class="refLabel"><a href="#ref169" class="rightTabRefNumLink">169</a></strong><div class="NLM_citation" id="rightTab-cit169"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nickel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erehman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goede, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span> <span> </span><span class="NLM_article-title">SuperPred: update on drug classification and target prediction</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">W26</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1093/nar/gku477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgku477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=24878925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W26-31&author=J.+Nickelauthor=B.+O.+Gohlkeauthor=J.+Erehmanauthor=P.+Banerjeeauthor=W.+W.+Rongauthor=A.+Goedeauthor=M.+Dunkelauthor=R.+Preissner&title=SuperPred%3A+update+on+drug+classification+and+target+prediction&doi=10.1093%2Fnar%2Fgku477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">SuperPred: update on drug classification and target prediction</span></div><div class="casAuthors">Nickel, Janette; Gohlke, Bjoern-Oliver; Erehman, Jevgeni; Banerjee, Priyanka; Rong, Wen Wei; Goede, Andrean; Dunkel, Mathias; Preissner, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W26-W31</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The SuperPred web server connects chem. similarity of drug-like compds. with mol. targets and the therapeutic approach based on the similar property principle.  Since the first release of this server, the no. of known compd.-target interactions has increased from 7000 to 665 000, which allows not only a better prediction quality but also the estn. of a confidence.  Apart from the addn. of quant. binding data and the statistical consideration of the similarity distribution in all drug classes, new approaches were implemented to improve the target prediction.  The 3D similarity as well as the occurrence of fragments and the concordance of physico-chem. properties is also taken into account.  In addn., the effect of different fingerprints on the prediction was examd.  The retrospective prediction of a drug class (ATC code of the WHO) allows the evaluation of methods and descriptors for a well-characterized set of approved drugs.  The prediction is improved by 7.5% to a total accuracy of 75.1%.  For query compds. with sufficient structural similarity, the web server allows prognoses about the medical indication area of novel compds. and to find new leads for known targets.  SuperPred is publicly available without registration at: http://prediction.charite.de.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWNMDss7U4k7Vg90H21EOLACvtfcHk0lipuHaGeBqkpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs7rF&md5=e23cf651ec0f4f7784104e6b198a014c</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku477%26sid%3Dliteratum%253Aachs%26aulast%3DNickel%26aufirst%3DJ.%26aulast%3DGohlke%26aufirst%3DB.%2BO.%26aulast%3DErehman%26aufirst%3DJ.%26aulast%3DBanerjee%26aufirst%3DP.%26aulast%3DRong%26aufirst%3DW.%2BW.%26aulast%3DGoede%26aufirst%3DA.%26aulast%3DDunkel%26aufirst%3DM.%26aulast%3DPreissner%26aufirst%3DR.%26atitle%3DSuperPred%253A%2520update%2520on%2520drug%2520classification%2520and%2520target%2520prediction%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DW26%26epage%3D31%26doi%3D10.1093%2Fnar%2Fgku477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref170"><div class="reference"><strong class="refLabel"><a href="#ref170" class="rightTabRefNumLink">170</a></strong><div class="NLM_citation" id="rightTab-cit170"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span> <span> </span><span class="NLM_article-title">TarPred: a web application for predicting therapeutic and side effect targets of chemical compounds</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2049</span>– <span class="NLM_lpage">2051</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btv099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fbioinformatics%2Fbtv099" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=2049-2051&author=X.+Liuauthor=Y.+Gaoauthor=J.+Pengauthor=Y.+Xuauthor=Y.+Wangauthor=N.+Zhouauthor=J.+Xingauthor=X.+Luoauthor=H.+Jiangauthor=M.+Zheng&title=TarPred%3A+a+web+application+for+predicting+therapeutic+and+side+effect+targets+of+chemical+compounds&doi=10.1093%2Fbioinformatics%2Fbtv099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtv099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtv099%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DN.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DM.%26atitle%3DTarPred%253A%2520a%2520web%2520application%2520for%2520predicting%2520therapeutic%2520and%2520side%2520effect%2520targets%2520of%2520chemical%2520compounds%26jtitle%3DBioinformatics%26date%3D2015%26volume%3D31%26spage%3D2049%26epage%3D2051%26doi%3D10.1093%2Fbioinformatics%2Fbtv099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref171"><div class="reference"><strong class="refLabel"><a href="#ref171" class="rightTabRefNumLink">171</a></strong><div class="NLM_citation" id="rightTab-cit171"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artimo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonnalagedda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baratin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Castro, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flegel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasteiger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosdidier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioannidis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liechti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostaguir, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redaschi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenarios, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockinger, H.</span></span> <span> </span><span class="NLM_article-title">ExPASy: SIB bioinformatics resource portal</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">W597</span>– <span class="NLM_lpage">603</span>, <span class="refDoi"> DOI: 10.1093/nar/gks400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1093%2Fnar%2Fgks400" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=22661580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCqu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=W597-603&author=P.+Artimoauthor=M.+Jonnalageddaauthor=K.+Arnoldauthor=D.+Baratinauthor=G.+Csardiauthor=E.+de+Castroauthor=S.+Duvaudauthor=V.+Flegelauthor=A.+Fortierauthor=E.+Gasteigerauthor=A.+Grosdidierauthor=C.+Hernandezauthor=V.+Ioannidisauthor=D.+Kuznetsovauthor=R.+Liechtiauthor=S.+Morettiauthor=K.+Mostaguirauthor=N.+Redaschiauthor=G.+Rossierauthor=I.+Xenariosauthor=H.+Stockinger&title=ExPASy%3A+SIB+bioinformatics+resource+portal&doi=10.1093%2Fnar%2Fgks400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">ExPASy: SIB bioinformatics resource portal</span></div><div class="casAuthors">Artimo, Panu; Jonnalagedda, Manohar; Arnold, Konstantin; Baratin, Delphine; Csardi, Gabor; de Castro, Edouard; Duvaud, Severine; Flegel, Volker; Fortier, Arnaud; Gasteiger, Elisabeth; Grosdidier, Aurelien; Hernandez, Celine; Ioannidis, Vassilios; Kuznetsov, Dmitry; Liechti, Robin; Moretti, Sebastien; Mostaguir, Khaled; Redaschi, Nicole; Rossier, Gregoire; Xenarios, Ioannis; Stockinger, Heinz</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W597-W603</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ExPASy (http://www.expasy.org) has worldwide reputation as one of the main bioinformatics resources for proteomics.  It has now evolved, becoming an extensible and integrative portal accessing many scientific resources, databases and software tools in different areas of life sciences.  Scientists can henceforth access seamlessly a wide range of resources in many different domains, such as proteomics, genomics, phylogeny/evolution, systems biol., population genetics, transcriptomics, etc.  The individual resources (databases, web-based and downloadable software tools) are hosted in a  decentralized' way by different groups of the SIB Swiss Institute of Bioinformatics and partner institutions.  Specifically, a single web portal provides a common entry point to a wide range of resources developed and operated by different SIB groups and external institutions.  The portal features a search function across  selected' resources.  Addnl., the availability and usage of resources are monitored.  The portal is aimed for both expert users and people who are not familiar with a specific domain in life sciences.  The new web interface provides, in particular, visual guidance for newcomers to ExPASy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_eGs293-q2LVg90H21EOLACvtfcHk0ljYsnpwZ2dpag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCqu7c%253D&md5=8fc906725e3985a75ae0d15c01c30fde</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks400%26sid%3Dliteratum%253Aachs%26aulast%3DArtimo%26aufirst%3DP.%26aulast%3DJonnalagedda%26aufirst%3DM.%26aulast%3DArnold%26aufirst%3DK.%26aulast%3DBaratin%26aufirst%3DD.%26aulast%3DCsardi%26aufirst%3DG.%26aulast%3Dde%2BCastro%26aufirst%3DE.%26aulast%3DDuvaud%26aufirst%3DS.%26aulast%3DFlegel%26aufirst%3DV.%26aulast%3DFortier%26aufirst%3DA.%26aulast%3DGasteiger%26aufirst%3DE.%26aulast%3DGrosdidier%26aufirst%3DA.%26aulast%3DHernandez%26aufirst%3DC.%26aulast%3DIoannidis%26aufirst%3DV.%26aulast%3DKuznetsov%26aufirst%3DD.%26aulast%3DLiechti%26aufirst%3DR.%26aulast%3DMoretti%26aufirst%3DS.%26aulast%3DMostaguir%26aufirst%3DK.%26aulast%3DRedaschi%26aufirst%3DN.%26aulast%3DRossier%26aufirst%3DG.%26aulast%3DXenarios%26aufirst%3DI.%26aulast%3DStockinger%26aufirst%3DH.%26atitle%3DExPASy%253A%2520SIB%2520bioinformatics%2520resource%2520portal%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DW597%26epage%3D603%26doi%3D10.1093%2Fnar%2Fgks400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref172"><div class="reference"><strong class="refLabel"><a href="#ref172" class="rightTabRefNumLink">172</a></strong><div class="NLM_citation" id="rightTab-cit172"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljMFudRxCxYTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01921&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-19%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01921" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01921" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                11MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a9a74f96b3c40","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
