
<html lang="en"     class="pb-page"  data-request-id="50508ddb-8a70-4140-ac33-b46bd82fefaa"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-14;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c00471;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of SHP2-D26 as a First, Potent, and Effective PROTAC Degrader of SHP2 Protein" /></meta><meta name="dc.Creator" content="Mingliang  Wang" /></meta><meta name="dc.Creator" content="Jianfeng  Lu" /></meta><meta name="dc.Creator" content="Mi  Wang" /></meta><meta name="dc.Creator" content="Chao-Yie  Yang" /></meta><meta name="dc.Creator" content="Shaomeng  Wang" /></meta><meta name="dc.Description" content="Src homology 2 domain-containing phosphatase 2 (SHP2) is an attractive therapeutic target for human cancers and other human diseases. Herein, we report our discovery of potent small-molecule SHP2 d..." /></meta><meta name="Description" content="Src homology 2 domain-containing phosphatase 2 (SHP2) is an attractive therapeutic target for human cancers and other human diseases. Herein, we report our discovery of potent small-molecule SHP2 d..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 21, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00471" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00471" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00471" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00471" /></link>
        
    
    

<title>Discovery of SHP2-D26 as a First, Potent, and Effective PROTAC Degrader of SHP2 Protein | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00471" /></meta><meta property="og:title" content="Discovery of SHP2-D26 as a First, Potent, and Effective PROTAC Degrader of SHP2 Protein" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0016.jpeg" /></meta><meta property="og:description" content="Src homology 2 domain-containing phosphatase 2 (SHP2) is an attractive therapeutic target for human cancers and other human diseases. Herein, we report our discovery of potent small-molecule SHP2 degraders whose design is based upon the proteolysis-targeting chimera (PROTAC) concept. This work has led to the discovery of potent and effective SHP2 degraders, exemplified by SHP2-D26. SHP2-D26 achieves DC50 values of 6.0 and 2.6 nM in esophageal cancer KYSE520 and acute myeloid leukemia MV4;11 cells, respectively, and is capable of reducing SHP2 protein levels by &gt;95% in cancer cells. SHP2-D26 is &gt;30-times more potent in inhibition of phosphorylation of extracellular signal-regulated kinase (ERK) and of cell growth than SHP099, a potent SHP2 inhibitor, in KYSE520 and MV4;11 cancer cell lines. This study demonstrates that induced SHP2 degradation is a very effective approach to inhibit the function of SHP2. Further optimization of these SHP2 degraders may lead to the development of a new class of therapies for cancers and other human diseases." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00471"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00471">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00471&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00471&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00471&amp;href=/doi/10.1021/acs.jmedchem.0c00471" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 7510-7528</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01000" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Featured Article</span><a href="/doi/10.1021/acs.jmedchem.9b01953" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.0c02108"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of SHP2-D26 as a First, Potent, and Effective PROTAC Degrader of SHP2 Protein</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Mingliang Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingliang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Internal Medicine, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingliang++Wang">Mingliang Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jianfeng Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianfeng Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Internal Medicine, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianfeng++Lu">Jianfeng Lu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mi Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mi Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Internal Medicine, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mi++Wang">Mi Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chao-Yie Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chao-Yie Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Internal Medicine, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chao-Yie++Yang">Chao-Yie Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5445-0109" title="Orcid link">http://orcid.org/0000-0002-5445-0109</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Shaomeng Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaomeng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Internal Medicine,  Pharmacology,  Medicinal Chemistry, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#fc8f949d939199929bbc8991959f94d2999889"><span class="__cf_email__" data-cfemail="81f2e9e0eeece4efe6c1f4ece8e2e9afe4e5f4">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaomeng++Wang">Shaomeng Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00471&amp;href=/doi/10.1021%2Facs.jmedchem.0c00471" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 7510–7528</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 21, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>20 March 2020</li><li><span class="item_label"><b>Published</b> online</span>21 May 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 July 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00471" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00471</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7510%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMingliang%2BWang%252C%2BJianfeng%2BLu%252C%2BMi%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D14%26contentID%3Dacs.jmedchem.0c00471%26title%3DDiscovery%2Bof%2BSHP2-D26%2Bas%2Ba%2BFirst%252C%2BPotent%252C%2Band%2BEffective%2BPROTAC%2BDegrader%2Bof%2BSHP2%2BProtein%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7528%26publicationDate%3DJuly%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00471"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">6489</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00471" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of SHP2-D26 as a First, Potent, and Effective PROTAC Degrader of SHP2 Protein&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Mingliang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Jianfeng&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Mi&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Chao-Yie&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Shaomeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;7510-7528&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00471&quot;},&quot;abstract&quot;:&quot;Src homology 2 domain-containing phosphatase 2 (SHP2) is an attractive therapeutic target for human cancers and other human diseases. Herein, we report our discovery of potent small-molecule SHP2 degraders whose design is based upon the proteolysis-targeting chimera (PROTAC) concept. This work has led to the discovery of potent and effective SHP2 degraders, exemplified by SHP2-D26. SHP2-D26 achieves DC50 values of 6.0 and 2.6 nM in esophageal cancer KYSE520 and acute myeloid leukemia MV4;11 cells, respectively, and is capable of reducing SHP2 protein levels by &gt;95% in cancer cells. SHP2-D26 is &gt;30-times more potent in inhibition of phosphorylation of extracellular signal-regulated kinase (ERK) and of cell growth than SHP099, a potent SHP2 inhibitor, in KYSE520 and MV4;11 cancer cell lines. This study demonstrates that induced SHP2 degradation is a very effective approach to inhibit the function of SHP2. Further optimization of these SHP2 degraders may lead to the development of a new class of therapies for ca&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00471&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00471" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00471&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00471" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00471&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00471" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00471&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00471&amp;href=/doi/10.1021/acs.jmedchem.0c00471" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00471" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00471" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00471%26sid%3Dliteratum%253Aachs%26pmid%3D32437146%26genre%3Darticle%26aulast%3DWang%26date%3D2020%26atitle%3DDiscovery%2Bof%2BSHP2-D26%2Bas%2Ba%2BFirst%252C%2BPotent%252C%2Band%2BEffective%2BPROTAC%2BDegrader%2Bof%2BSHP2%2BProtein%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D14%26spage%3D7510%26epage%3D7528%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/jmcmar.2020.63.issue-14/20200723/jmcmar.2020.63.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Src homology 2 domain-containing phosphatase 2 (SHP2) is an attractive therapeutic target for human cancers and other human diseases. Herein, we report our discovery of potent small-molecule SHP2 degraders whose design is based upon the proteolysis-targeting chimera (PROTAC) concept. This work has led to the discovery of potent and effective SHP2 degraders, exemplified by SHP2-D26. SHP2-D26 achieves DC<sub>50</sub> values of 6.0 and 2.6 nM in esophageal cancer KYSE520 and acute myeloid leukemia MV4;11 cells, respectively, and is capable of reducing SHP2 protein levels by >95% in cancer cells. SHP2-D26 is >30-times more potent in inhibition of phosphorylation of extracellular signal-regulated kinase (ERK) and of cell growth than SHP099, a potent SHP2 inhibitor, in KYSE520 and MV4;11 cancer cell lines. This study demonstrates that induced SHP2 degradation is a very effective approach to inhibit the function of SHP2. Further optimization of these SHP2 degraders may lead to the development of a new class of therapies for cancers and other human diseases.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span> Note</h4><p class="last">This paper was originally published ASAP on May 21, 2020. Due to a production error, part of Table 1 was missing. The corrected version was reposted on May 22, 2020.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61747" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61747" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Src homology 2 domain-containing phosphatase 2 (SHP2) is a protein tyrosine phosphatase. Mutations of SHP2 are prevalent in Noonan syndrome (50%) and LEOPARD syndrome (80%)<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> and activated mutations of SHP2 have also been identified in juvenile myelomonocytic leukemia (JMML, 35%), myelodysplastic syndrome (10%), B-cell acute lymphoblastic leukemia (7%), and acute myeloid leukemia (AML, 4%).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Somatic activating mutations in SHP2 have been associated with several types of solid tumors, including lung adenocarcinoma, colon cancer, neuroblastoma, glioblastoma, melanoma, hepatocellular carcinoma, prostate cancer, and breast cancer.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7 ref8">(4−8)</a> Accumulated evidence demonstrates that in cancer cells, SHP2 is involved in multiple signaling processes, such as RAS-ERK, JAK-STAT, PI3K-AKT, NF-κB, and mTOR pathways.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12 ref13">(9−13)</a> In the RAS-ERK pathway, SHP2 acts as a positive regulator at upstream to promote RAS-RAF-ERK kinase cascade signaling transduction. Therefore, SHP2 inhibition leads to dephosphorylation of ERK and suppression of the pro-oncogenic function of RAS-RAF-ERK pathway, resulting in cell growth inhibition and apoptosis induction in cancer cells.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Furthermore, SHP2 also participates in the programmed cell death pathway (PD-1/PD-L1) and inhibits T cell activation, thus contributing to immune evasion.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> Hence, SHP2 is a very attractive cancer therapeutic target.</div><div class="NLM_p">Due to the highly conserved and positively charged nature of its protein-tyrosine phosphatase (PTP) catalytic site, SHP2 has proved to be a difficult target in the discovery of small-molecule inhibitors.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> Previously reported SHP2 inhibitors have not shown satisfactory selectivity and/or cellular activity, and this has prevented their development as useful therapeutic agents.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22 ref23 ref24 ref25">(19−25)</a> A major breakthrough by Novartis scientists was the discovery of SHP099, a potent and allosteric SHP2 inhibitor, which was shown to selectively block SHP2 phosphatase activity and inhibit cancer cell growth <i>in vitro</i> and tumor growth in xenograft models in mice.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> Subsequently, additional allosteric SHP2 inhibitors including SHP389 were reported and several of them have been advanced into clinical development for the treatment of human cancers.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31">(28−31)</a></div><div class="NLM_p">Although allosteric SHP2 inhibitors have been shown to be effective in preclinical models of Kirsten rat sarcoma (KRAS) mutant human cancer, we hypothesized that efficient depletion of SHP2 protein may provide an alternative and perhaps even more effective strategy for inhibition of the SHP2 activity. Mainardi et al.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> demonstrated that SHP2 inactivation by CRISPR-Cas9 induces senescence and impairs tumor growth in xenograft models of KRAS mutant tumors. Furthermore, Ruess et al.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> revealed that knockout of the PTPN11 gene, which encodes SHP2, in KRAS mutant human ductal adenocarcinoma (PDAC) cells results in reduction in cell proliferation and PTPN11-knockout cells are uniquely susceptible to mitogen-activated protein kinase (MEK) inhibitors. Such findings provide evidence that depletion of the SHP2 protein in tumor cells could be an effective therapeutic strategy for human cancers, particularly those carrying a KRAS-mutation.</div><div class="NLM_p">In 2001, Deshaies and Crews formally demonstrated the proteolysis-targeting chimera (PROTAC) strategy to achieve targeted protein degradation.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> A PROTAC molecule is a bifunctional small molecule consisting of a ligand that binds to the target protein of interest and another ligand that recruits an E3 ligase system. The ligands are tethered together through a chemical linker.<a onclick="showRef(event, 'ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37">(35−37)</a> In recent years, the PROTAC approach has gained a momentum in the discovery and development of completely new classes of small-molecule therapeutic agents.<a onclick="showRef(event, 'ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45">(38−45)</a> Two PROTAC molecules targeting the androgen receptor and estrogen receptor, discovered by Arvinas scientists, have been advanced into clinical development for the treatment of human cancers.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div><div class="NLM_p last">In this paper, we report our design, synthesis, and evaluation of PROTAC SHP2 degraders. This study has resulted in the discovery of SHP2-D26, which induces rapid and efficient degradation of SHP2. This compound is also 10–100 times more potent than an allosteric SHP2 inhibitor in inhibition of ERK activity and cell growth in cancer cell lines. This study has laid a foundation for the development of a new class of therapeutics through targeted degradation of SHP2 protein.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63283" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63283" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the design of PROTAC SHP2 degraders, it is important to identify both an SHP2 inhibitor and an E3 ligase ligand as well as suitable tethering sites to link them. A number of cocrystal structures of the SHP2 protein in complexes with potent SHP2 allosteric inhibitors have been reported (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Examination of the cocrystal structures of SHP099 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EHR">5EHR</a>) and compound <b>3</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MD7">6MD7</a>) showed that all the atoms in these SHP2 inhibitors are in close contacts with the SHP2 protein. However, in the cocrystal structure of SHP389 (<b>2</b>) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MDC">6MDC</a>) in complex with SHP2, the cyclopropyl group of SHP389 is exposed to the solvent, suggesting that this site can be used for a tether in potential SHP2 degraders.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of three previously reported SHP2 inhibitors SHP099 (<b>1</b>), SHP389 (<b>2</b>) and <b>3</b>, and two designed SHP2 inhibitors (<b>4</b>, <b>5</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We designed compound <b>4</b> as a potential SHP2 inhibitor by combining a portion of SHP099 with a similar group found in compound <b>3</b> to facilitate our subsequent design and synthesis of PROTAC SHP2 degraders. The predicted binding model of <b>4</b> in a complex with SHP2 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B) suggested that the hydrogen bond interactions between SHP099 and T108, F113, and E250 in SHP2 are preserved in the interactions of compound <b>4</b> and SHP2. The sulfur atom in <b>4</b> reorients the 2-chloroaniline group in the binding site, while the chlorine atom in <b>4</b> interacts with the hydrophobic pocket formed by side chain atoms of L254, Q257, and Q495 in SHP2. The cation-aromatic interaction between SHP099 and R111 in SHP2 is also maintained between <b>4</b> and SHP2. Of significance, the amino group in <b>4</b> is exposed to the solvent, providing a potential tethering site for the design of PROTAC degraders.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Co-crystal structure of SHP2 in a complex with SHP099 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EHR">5EHR</a>), and (B and C) modeled structures of SHP2 in a complex with <b>4</b> or <b>5</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We synthesized compound <b>4</b> and determined its inhibition of SHP2 enzymatic activity (see <a class="ref internalNav" href="#notes1" aria-label="Figure S1">Figure S1</a>). Compound <b>4</b> was found to be a potent SHP2 inhibitor with IC<sub>50</sub> = 76.2 nM. In the same assay, SHP099 is slightly less potent, with IC<sub>50</sub> = 136.2 nM.</div><div class="NLM_p">To facilitate the convenient synthesis of PROTAC SHP2 degraders, we synthesized compound <b>5</b> by changing the amine group in <b>4</b> to an amide group and evaluated its inhibition of SHP2. Our data showed that <b>5</b> is a potent SHP2 inhibitor with IC<sub>50</sub> = 98.7 nM. The predicted binding model of compound <b>5</b> with SHP2 showed that the acetyl group extends further into the solvent exposed region (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C), making it a suitable site for tethering. We therefore employed compound <b>5</b> as the SHP2 inhibitor and its amide group as the tethering site for the design of PROTAC SHP2 degraders.</div><div class="NLM_p">The VHL-1/cullin 2 E3 ligase complex has been successfully used in the design of a large number of PROTAC degraders active against various proteins.<a onclick="showRef(event, 'ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57">(47−57)</a> Based upon the cocrystal structures of VHL-1 ligands in a complex with VHL-1 and on our previous studies of androgen receptor (AR) and estrogen receptor (ER) degraders,<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> we designed and synthesized a series of potential PROTAC SHP2 degraders using compound <b>5</b> and a potent and previously reported VHL ligand <b>6</b><a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Because the length of the linker plays a key role in the potency of PROTAC degrader molecules, we systematically varied the linker length in compounds <b>7</b>–<b>19</b> with the objective of determining the optimal linker length (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). We first evaluated these compounds by Western blotting in the KYSE520 esophageal cancer cell line, which was shown to be responsive to SHP2 inhibitors,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> for their ability to induce degradation of SHP2 protein at 0.1 μM and 1 μM and obtained the data summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref internalNav" href="#notes1" aria-label="Figure S2">Figure S2</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. PROTAC SHP2 Degraders Designed Using SHP2 Inhibitor <b>5</b> and a Previously Published Potent VHL-1 Ligand<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0012.gif" alt="" id="GRAPHIC-d7e468-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0013.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">All the data are averages of three independent experiments with a treatment time of 6 h.</p></div></div><div></div></div><div class="NLM_p">Our Western blotting data showed that compounds <b>7</b>–<b>12</b> with different linker lengths, at 0.1 or 1 μM, have little or no effect on the level of the SHP2 protein in the KYSE520 cell line. However, compound <b>13</b>, which has one additional methylene group in the linker when compared to <b>12</b>, reduces the SHP2 protein level by ∼80% at 0.1 and 1 μM. Increasing the linker length in <b>13</b> by one additional methylene yielded <b>14</b>, which reduces the SHP2 protein level by >90% at 0.1 and 1 μM. Compounds <b>15</b>–<b>19</b> with longer linkers than those in <b>14</b> are all potent and effective SHP2 degraders, capable of reducing the SHP2 protein level by >90% at 0.1 and 1 μM.</div><div class="NLM_p">Our previous studies showed that the linker composition in PROTAC molecules plays a key role in their degradative potencies and, accordingly, we performed further modifications of linker composition while keeping the linker length similar to that in <b>16</b>. The results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref internalNav" href="#notes1" aria-label="Figures S2 and S4">Figures S2 and S4</a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Optimization of the Linker Composition of SHP2 Degrader <b>16</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0014.gif" alt="" id="GRAPHIC-d7e528-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0015.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">All of the data are the average of three independent experiments with a treatment time of 6 h in KYSE520 cell lines and 24 h in MV4;11 cell lines.</p></div></div><div></div></div><div class="NLM_p">Western blotting results showed that compounds <b>20</b>–<b>22</b> with a polyethylene glycol (PEG) unit embedded in the linker are less potent than compound <b>16</b>. We replaced the amide bond in the middle of the linker in <b>16</b> with a positively charged piperazinyl group, which yielded compounds <b>23</b> and <b>24</b>, both of which reduce the SHP2 protein level by ∼80% at 0.1 μM and >95% at 1 μM in the KYSE520 cell line. Thus, while both <b>23</b> and <b>24</b> are very effective SHP2 degraders, they are less potent than <b>16</b>. To examine the effect of the positively charged piperazinyl group in <b>23</b> and <b>24</b>, we synthesized compound <b>25</b> containing a linker of 14 methylene groups. Compound <b>25</b> is less effective than either <b>23</b> or <b>24</b> in reducing the SHP2 protein level at both 0.1 μM and 1 μM in the KYSE520 cell line. Retention of the amide bond in the middle of the linker and introduction of a positively charged piperazinyl group resulted in compounds <b>26</b> and <b>27</b>, both of which achieve near-complete SHP2 degradation at both 100 nM and 1 μM. Introduction of a phenoxyl group in the middle of the linker led to compound <b>28</b>, which reduces SHP2 protein by >95% at both 0.1 μM and 1 μM. Compound <b>29</b>, containing a positively charged piperazinyl group in the central part of the linker, fails to induce any SHP2 degradation at 100 nM but reduces SHP2 protein by ∼80% at 1 μM.</div><div class="NLM_p">The acute myeloid leukemia MV4;11 cell line was also shown to be very responsive to SHP2 inhibitors.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> We next tested these compounds for their ability to reduce the SHP2 protein in the acute myeloid leukemia MV4;11 cell line at 100 nM with 24 h treatment time, obtaining the data summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. The data showed that <b>20</b>, <b>22</b>, <b>26</b>, <b>27</b>, and <b>28</b> all effectively reduce SHP2 protein by >95% at 100 nM, indicating that these compounds are potent and effective SHP2 degraders in the MV4;11 cell line. These linker modifications thus have led to a number of highly potent SHP2 degraders.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Further Evaluation of Compound <b>26</b> (SHP2-D26)</h3><div class="NLM_p">In addition to its excellent degradation potency in both the KYSE520 and MV4;11 cell lines, compound <b>26</b> (SHP2-D26) has good aqueous solubility. We further investigated SHP2-D26 for its potency and mechanism of action in these two cell lines.</div><div class="NLM_p">We examined the degradation of SHP2 by SHP2-D26 in a wide range of concentrations in KYSE520 and MV4;11 cell lines (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–D). Western blotting results showed that SHP2-D26 effectively reduces SHP2 protein in a dose-dependent manner. Quantification of the Western blotting data showed that the compound achieves DC<sub>50</sub> values (concentration needed to induce targeted protein degradation by 50%) of 6.0 and 2.6 nM in the KYSE520 and MV4;11 cell lines, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,D).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Further evaluation of SHP2-D26 for its degradation activity in the KYSE520 and MV4;11 cell lines. (A–D) Dose-dependent SHP2 degradation in the KYSE520 and MV4;11 cell lines, with SHP099 included as the inhibitor control. Cells were treated with SHP2-D26 for 12 h. SHP2 protein was examined by Western blotting, and the protein level was quantified by densitometry and normalized to the corresponding density of the GAPDH protein. (E,F). Degradation kinetics of SHP2-D26 in the KYSE520 and MV4;11 cell lines. Cells were treated with 100 nM of SHP2-D26 for different times. SHP2 protein was examined by Western blotting, and the GAPDH protein was used as the loading control. (G) KYSE520 cell line was treated with VHL-1 ligand <b>6</b> (10 μM), compound <b>4</b> (at 10 μM), MLN4924 (0.5 μM), or MG132 (3 μm) for 1 h and then treated with SHP-D26 (0.1 μM) for 3 h. The protein level of SHP2 was examined by Western blotting, and the GAPDH protein was used as the loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next evaluated the kinetics of SHP2-D26 in induction of SHP2 degradation in the KYSE520 and MV4;11 cell lines (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E,F). In KYSE520 cells, SHP2-D26 at 100 nM effectively reduces the SHP2 protein level within 4 h and achieves essentially complete SHP2 depletion with 8 h treatment (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). Similar kinetics was observed in the MV4;11 cell line (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). These data show that SHP2 degradation induced by SHP2-D26 is fairly rapid in both KYSE520 and MV4;11 cells.</div><div class="NLM_p">We examined if SHP2-D26 functions as a <i>bona fide</i> PROTAC degrader in the KYSE520 cell line. A VHL-1 ligand <b>6</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), compound <b>4</b> (an SHP2 inhibitor), MLN4924 (an E1 inhibitor), and MG132 (a proteasome inhibitor) all effectively block degradation of the SHP2 protein in KYSE520 cells (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G). Therefore, our data show that SHP2 degradation induced by SHP2-D26 requires its binding to VHL-1 and SHP2 proteins and is also neddylation- and proteasome-dependent, demonstrating that SHP2-D26 is a <i>bona fide</i> PROTAC SHP2 degrader.</div><div class="NLM_p">Since SHP2 protein is known to play an important role in the MAPK/ERK signaling pathway, we examined the impact of SHP2 degradation on the MAPK/ERK signaling pathway in the KYSE520 and MV4;11 cell lines, with SHP099 included as a control (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Western blotting showed that both SHP2-D26 and SHP099 dose-dependently inhibit phosphorylation of ERK in the KYSE520 and MV4;11 cell lines. However, in both cell lines, SHP2-D26 is much more potent than SHP099 in inhibition of pERK. Specifically, in the KYSE520 cell line, while 100 nM of SHP2-D26 is effective in reducing the level of pERK, >3,000 nM of the SHP2 inhibitor SHP099 is needed to do so. In the MV4;11 cell line, while 100 nM of SHP2-D26 completely inhibits phorsphorylated ERK (pERK), >3,000 nM of the SHP2 inhibitor SHP099 is required to achieve complete inhibition of pERK. Therefore, our data indicate that SHP2-D26 is >30-times more potent than SHP099 in inhibition of pERK in both cell lines.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>26</b> (SHP2-D26) suppresses the p-ERK pathway. KYSE520 or MV4;11 cells were treated as indicated with SHP2-D26 or SHP099 for 48 h. The protein levels of SHP2 (Bethyl Lab. A301–544), ERK (#9102, CST), and phospho-ERK (#4370, CST) determined by Western blotting. GAPDH was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We evaluated the cell growth inhibition of SHP2-D26, and three SHP2 inhibitors (SHP099, compounds <b>4</b> and <b>5</b>) in the KYSE520 and MV4;11 cell lines (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). In the KYSE520 cell line, SHP2-D26 achieves IC<sub>50</sub> values of 0.66 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). In comparison, SHP099, compounds <b>4</b> and <b>5</b> have IC<sub>50</sub> values of 18.2, 42.3, and 39.4 μM, respectively. Hence, SHP2-D26 is 28-, 64-, and 60-times more potent than SHP099 and compounds <b>4</b> and <b>5</b> in inhibition of cell growth in the KYSE520 cell line. In the MV4;11 cell line, SHP2-D26, SHP099, <b>4</b> and <b>5</b> have IC<sub>50</sub> values of 9.9 nM, 1.0 μM, 3.9 μM and 6.6 μM, respectively, in inhibition of cell growth. Thus, SHP2-D26 is 100-, > 400 and >600-times more potent than SHP099, compounds <b>4</b> and <b>5</b>, respectively in cell growth inhibition in the MV4;11 cell line.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cell growth inhibition in KYSE520 and MV4;11 cells treated with SHP2 degrader <b>26</b> (SHP2-D26) and SHP2 inhibitors SHP099, <b>4</b> and <b>5</b>. KYSE520 and MV4;11 cells were treated with indicated doses for 4 days, and cell viability was determined by a colorimetric WST-8 assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">CHEMISTRY</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26111" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26111" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The compounds in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> were synthesized from the VHL ligand (<b>6</b>) and the key intermediate (<b>42</b>). The synthesis of VHL ligand <b>6</b> follows a published procedure<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and is outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The commercially available (S)-1-(4-bromophenyl)ethan-1-amine (<b>30</b>) was protected by Boc<sub>2</sub>O, and a subsequent Heck coupling reaction with 4-methylthiazole afforded compound <b>31</b>. Removal of the Boc group under acidic conditions gave compound <b>32</b>, and amide coupling with compound <b>33</b> led to <b>34</b>, which after acidic deprotection afforded the VHL ligand (<b>6</b>).</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the VHL Ligand (<b>6</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, NaHCO<sub>3</sub>, EtOAc/H<sub>2</sub>O, 2 h; (b) 4-methylthiazole, Pd(OAc)<sub>2</sub>, KOAc, DMA, 90 °C, 12 h; (c) 4 N HCl dioxane/MeOH, rt, 4h; (d) HATU, DIPEA, DMF, 0 °C to RT, 12 h; and (e) TFA, DCM, rt, 4h.</p></p></figure><div class="NLM_p">The synthesis of the key intermediate (<b>40</b>) is shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The commercially available 2-chloro-3-fluoroaniline (<b>35</b>) was reacted with <i>tert</i>-butylthiol under basic conditions to produce compound <b>36</b>, which was heated in concentrated hydrochloric acid to remove the <i>tert</i>-butyl group, yielding compound <b>37</b>. A cross-coupling reaction of the intermediate <b>37</b> with 3-bromo-6-chloropyrazin-2-amine (<b>38</b>) was carried out in the presence of CuI and 1,10-phenanthroline as catalysts, and potassium phosphate (K<sub>3</sub>PO<sub>4</sub>) as a base in 1,4-dioxane at 90 °C for 16 h to obtain compound <b>39</b>, which was reacted with <i>tert</i>-butyl (4-methylpiperidin-4-yl)carbamate using DIPEA as a base in DMSO at 100 °C for 3 h to afford the key intermediate (<b>40</b>).</div><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the Key Intermediate (<b>40</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-butylthiol, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 120 °C, 36 h; (b) conc. HCl, 80 °C, 6 h; (c) K<sub>3</sub>PO<sub>4</sub>, CuI, 1,10-phenanthroline, dioxane, 90 °C, 16 h; and (d) DIPEA, DMSO, 100 °C, 3 h.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, compounds <b>7</b>–<b>19</b> were synthesized according to the following procedure. Using Boc-protected amine-terminated acids with different carbon chain lengths (<b>41a</b>–<b>41j</b>) as starting materials, the intermediates (<b>43a</b>–<b>43j</b>) were obtained by an amide condensation reaction followed by removal of the Boc protecting group under acidic conditions. The intermediates (<b>46a</b>–<b>46c</b>) were synthesized by the reaction of the key intermediate <b>40</b> with methyl-3-chloro-3-oxoproanoate esters with different carbon chain lengths (<b>44a</b>–<b>44c</b>) and subsequent hydrolysis of the methyl ester. Amide coupling between compound <b>43</b> and <b>46</b> followed by removal of the Boc protecting group afforded the final compounds (<b>7</b>–<b>19</b>) in high yields.</div><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>7</b>–<b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>6</b>, HATU, DIPEA, DMF, 0 °C to RT, 1 h; (b) 4 N HCl dioxane/MeOH, rt; (c) <b>40</b>, DIPEA, DCM, 0 °C, 2 h; (d) LiOH·H<sub>2</sub>O, THF, MeOH, H<sub>2</sub>O, RT, 3 h; (e) HATU, DIPEA, DMF, 0 °C to RT, 1 h; and (f) TFA, DCM, rt, 2 h.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, compounds <b>20</b>–<b>22</b> were treated in a similar way to produce compounds <b>7</b>–<b>19</b>. With a polyethylene glycol unit (PEG) embedded in their chains, Boc-protected amine-terminated acids (<b>47, 49</b>) were used as the starting material. The intermediates (<b>48, 50</b>) were obtained by amide condensation with the VHL ligand (<b>6</b>), and this was followed by removal of the Boc group. Employing compounds <b>48</b> and <b>50</b> as intermediates, compounds <b>20</b>–<b>22</b> were synthesized by condensation with the corresponding acid (<b>48b</b>) and subsequent removal of the Boc group.</div><figure id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>20</b>, <b>21</b>, and <b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>6</b>, HATU, DIPEA, DMF, 0 °C to rt, 1 h; (b) 4 N HCl dioxane, 2h; (c) <b>46b</b>, HATU, DIPEA, DMF, 0 °C to rt, 1 h; and (d) TFA, DCM, rt, 2h.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>23</b>–<b>25</b> is shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Because the starting material <b>40</b> is a non-nucleophilic amine, the formation of amide bonds in high yields with common coupling agents is difficult. After trying multiple reaction conditions, a combination of N,N,N,N-tetramethylchloroformamide hexafluorophosphate (TCFH) and <i>N</i>-methylimidazole (NMI) was found to be effective as a coupling reagent.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The intermediate <b>(52)</b> was obtained by amide condensation and subsequent removal of the Fmoc group. Following the substitution reaction of compound <b>52</b> with 11-bromoundecanoic acid or 12-bromododecanoic acid, (<b>54a</b>, <b>54b</b>) were obtained. Following amide condensation with VHL ligand <b>6</b> and Boc removal, the final compounds (<b>23</b>, <b>24</b>) were produced. Compound <b>25</b> was synthesized using the procedure described for the synthesis of compound <b>23</b> with 16-(benzyloxy)-16-oxohexadecanoic acid (<b>55</b>) as the starting material.</div><figure id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>23</b>, <b>24</b>, and <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TCFH, NMI, MeCN/THF, rt, 6 h; (b) piperidine, MeCN, rt, 2 h; (c) K<sub>2</sub>CO<sub>3</sub>, KI, DMF, 60 °C, 8 h; (d) HATU, DIPEA, DMF, 0 °C to rt, 1 h; (e) TFA, DCM, rt, 2h; and (f) NaOH, THF-H<sub>2</sub>O.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>, compounds <b>26</b>–<b>29</b> were synthesized using the following procedure. Intermediates <b>59a/59b</b> were synthesized by the substitution reaction of <i>tert</i>-butyl piperazine-1-carboxylate with methyl 9-bromononanoate or methyl 10-bromononanoate followed by hydrolysis with lithium hydroxide. The intermediates (<b>60a</b>, <b>60b</b>) were obtained by an amide condensation reaction of <b>59a</b> or <b>59b</b>, respectively, with VHL ligand <b>6</b> followed by removal of the Boc group. Amide condensation of <b>60a</b> or <b>60b</b> with <b>46b</b> followed by removal of the Boc group afforded compounds <b>26</b> and <b>27</b>. Compounds <b>28</b> and <b>29</b> were synthesized using the procedure described for the synthesis of compound <b>26</b> with <i>tert</i>-butyl (4-hydroxyphenyl)carbamate (<b>61</b>) and <i>tert</i>-butyl (4-(piperazin-1-yl)butyl)carbamate (<b>65</b>), respectively, as the starting materials.</div><figure id="sch6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>26</b>, <b>27, 28</b>, and <b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, NaI, MeCN, 60 °C, 16 h; (b) LiOH.H<sub>2</sub>O, THF, MeOH, H<sub>2</sub>O, rt, 3 h (c) <b>6</b>, HATU, DIPEA, DMF, 0 °C to rt, 1 h; (d) 4 N HCl dioxane, 2 h; (e) <b>46b</b>, HATU, DIPEA, DMF, 0 °C to rt, 1 h; (f) TFA, DCM, rt, 2 h.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02327" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02327" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have designed, synthesized, and evaluated a series of PROTAC SHP2 degraders. Our study has resulted in the discovery of a number of highly potent SHP2 degraders, we obtained a number of highly potent SHP2 degraders, exemplified by compound <b>26</b> (SHP2-D26). SHP2-D26 achieves DC<sub>50</sub> values of 6.0 nM and 2.6 nM in KYSE520 and MV4;11 cells, respectively, and >95% degradation at 30 nM in both cell lines. SHP2-D26 is much more potent and effective in inhibition of ERK phosphorylation and in inhibition of cell growth in KYSE520 and MV4;11 cells than three SHP2 inhibitors. This study provides the first proof-of-concept that targeted degradation of SHP2 is a very effective strategy in inhibition of SHP2 activity. Further optimization of SHP2-D26 may lead to the development of a potent and effective SHP2 degrader for the treatment of human cancers. In addition, a potent and effective SHP2 degrader may also have a great therapeutic potential for the treatment of certain human genetic disorders caused by SHP2 mutation and activation, such as Noonan syndrome and LEOPARD syndrome.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06711" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06711" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> General Information</h4><div class="NLM_p">All commercial reagents and solvents were used as supplied without further purification. Proton nuclear magnetic resonance (<sup>1</sup>H NMR) and carbon nuclear magnetic resonance (<sup>13</sup>C NMR) spectroscopy were performed on Bruker Advance 400 NMR spectrometers. <sup>1</sup>H NMR spectra are reported in parts per million (ppm) downfield from tetramethylsilane (TMS). All <sup>13</sup>C NMR spectra are reported in ppm and obtained with <sup>1</sup>H decoupling. In the spectral data reported, the format (δ) chemical shift (multiplicity, <i>J</i> values in Hz, integration) was used with the following abbreviations: s = singlet, brs = broad singlet, d = doublet, t = triplet, q = quartet, and m = multiplet. Electrospray ionization (ESI) mass spectral (MS) analysis was performed on a Thermo Scientific LCQ Fleet mass spectrometer. The final products were purified by reverse-phase HPLC (RP-HPLC) with solvent A (0.1% of TFSA in water) and solvent B (0.1% of TFA in CH<sub>3</sub>CN) as eluents with a flow rate of 45 mL/min. All final compounds have purity ≥95% as determined by Waters ACQUITY ultraperformance liquid chromatography (UPLC) using a reverse-phase column (SunFire, C18, 5 μm, 4.6 × 150 mm2) and a solvent gradient of solvent A (H<sub>2</sub>O with 0.1% of TFA) and solvent B (CH<sub>3</sub>CN with 0.1% of TFA).</div><div id="sec5_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> (2<i>S</i>,4<i>R</i>)-1-((S)-2-Amino-3,3-dimethylbutanoyl)-4-hydroxy-N-((S)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>6</b>)</h5><div class="NLM_p last">HATU (4.56 g, 12.0 mmol, 1.2 equiv) was added to a solution of compound <b>32</b> (2.55 g, 10.0 mmol, 1.0 equiv), which was synthesized according to the reported procedure,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> using (2<i>S</i>,4<i>R</i>)-1-((S)-2-((<i>tert</i>-butoxycarbonyl)amino)-3,3-dimethyl-butanoyl)-4-hydroxypyrrolidine-2-carboxylic acid (<b>33</b>) (3.45 g, 10.0 mmol, 1.0 equiv), and DIPEA (6.95 mL, 40.0 mmol, 4.0 equiv) in DMF (20 mL) at 0 °C under nitrogen. The mixture was stirred at ambient temperature for 12 h when LC-MS showed that the reaction was complete. The reaction mixture was quenched with H<sub>2</sub>O (100 mL) and extracted with EtOAc (100 mL × 2). The combined organic layers were washed with brine (200 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The organic solution was filtered and concentrated under reduced pressure to give a residue which was dissolved in DCM (30.0 mL), and TFA (15.0 mL) was added at 0 °C under nitrogen. The mixture was stirred at ambient temperature for 4 h, when LC-MS showed that the reaction was complete. The volatile components were removed on a rotary evaporator, and the residue was purified by reverse phase column chromatography to afford compound <b>6</b> as a TFA salt (3.12 g, 56%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.01 (s, 1H), 8.61 (d, <i>J</i> = 8.0 Hz, 1H), 8.20 (brs, 3H), 7.45–7.37 (m, 4H), 4.95–4.90 (m, 1H), 4.54 (t, <i>J</i> = 8.0 Hz, 1H), 4.30 (brs, 1H), 3.90–3.87 (m, 1H), 3.76–3.73 (m, 1H), 3.52–3.47 (m, 1H), 2.45 (s, 3H), 2.14–2.09 (m, 1H), 1.78–1.72 (m, 1H), 1.38 (d, J = 7.2 Hz, 3H), δ 1.02 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>23</sub>H<sub>33</sub>N<sub>4</sub>O<sub>3</sub>S [M + 1] <sup>+</sup>, 445.23; found, 445.44.</div></div><div id="sec5_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(3-(((S)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-(((S)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-3-oxopropyl)malonamide (<b>7</b>)</h5><div class="NLM_p last">HATU (25 mg, 0.066 mmol, 1.1 equiv) was added to a mixture of (2<i>S</i>,4<i>R</i>)-1-((S)-2-(3-aminopropanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((S)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide hydrochloride (<b>43a</b>) (33 mg, 0.06 mmol, 1.0 equiv), 3-((3-((3-amino-5-(4-((tert-butoxycarbonyl)amino)-4-methylpiperidin-1-yl)pyrazin-2-yl)-thio)-2-chlorophenyl)amino)-3-oxopropanoic acid (<b>46a</b>) (33 mg, 0.06 mmol, 1.0 equiv), and DIPEA (39 mg, 0.30 mmol, 5.0 equiv) in DMF (2 mL) at 0 °C under N<sub>2</sub>. The mixture was stirred at ambient temperature for 1 h. After being quenched with water (8 mL) and extracted with EtOAc (5 mL x 3), the organic layers were washed with brine (15 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was dissolved in DCM (3 mL), and trifluoracetic acid (1 mL) was added at 0 °C. The reaction was stirred for 2 h, and the solvent was removed <i>in vacuo</i>. The residue was purified by reverse-phase chromatography over a C18 column to yield <b>7</b> (37 mg, 58%) as a white powder. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.31 (s, 1H), 9.00 (s, 1H), 8.39 (d, <i>J</i> = 8.0 Hz, 1H), 8.32–8.29 (m, 1H), 7.97 (brs, 3H), 7.80 (d, <i>J</i> = 7.2 Hz, 1H), 7.66 (s, 1H), 7.45–7.37 (m, 4H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.40 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 4.95–4.85 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.43 (t, <i>J</i> = 8.0 Hz, 1H), 4.29 (s, 1H), 4.06–4.02 (m, 2H), 3.64–3.57 (m, 2H), 3.38 (s, 2H), 3.33–3.26 (m, 2H), 2.45 (s, 3H), 2.43–2.33 (m, 2H), 2.04–1.99 (m, 1H), 1.81–1.77 (m, 1H), 1.73–1.70 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 0.92 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>45</sub>H<sub>59</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 948.38; found, 948.31.</div></div><div id="sec5_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(4-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-4-oxobutyl)malonamide (<b>8</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43b</b> and <b>46a</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.34 (s, 1H), 9.00 (s, 1H), 8.39 (d, <i>J</i> = 8.0 Hz, 1H), 8.29–8.27 (m, 1H), 7.96 (brs, 3H), 7.88–7.81 (m, 2H), 7.66 (s, 1H), 7.45–7.37 (m, 4H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.40 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.21 (brs, 1H), 4.95–4.88 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.06–4.02 (m, 2H), 3.64–3.57 (m, 2H), 3.39 (s, 2H), 3.32–3.28 (m, 2H), 3.13–3.08 (m, 2H), 2.45 (s, 3H), 2.30–2.27 (m, 1H), 2.18–2.16 (m, 1H), 2.04–1.97 (m, 1H), 1.78–1.73 (m, 1H), 1.70–1.64 (m, 6H), 1.37 (s, 3H), 1.36 (s, 3H), 0.94 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>46</sub>H<sub>61</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 962.39; found, 962.33.</div></div><div id="sec5_1_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(5-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-5-oxopentyl)malonamide (<b>9</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43c</b> and <b>46a</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.33 (s, 1H), 9.00 (s, 1H), 8.39 (d, <i>J</i> = 8.0 Hz, 1H), 8.27–8.25 (m, 1H), 7.99 (brs, 3H), 7.83–7.81 (m, 2H), 7.66 (s, 1H), 7.45–7.37 (m, 4H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.40 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.17 (brs, 1H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.05–4.02 (m, 2H), 3.64–3.56 (m, 2H), 3.39 (s, 2H), 3.33–3.28 (m, 2H), 3.13–3.08 (m, 2H), 2.45 (s, 3H), 2.29–2.23 (m, 1H), 2.16–2.13 (m, 1H), 2.01–1.97 (m, 1H), 1.78–1.70 (m, 4H), 1.54–1.41 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>47</sub>H<sub>63</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 976.41; found, 976.45.</div></div><div id="sec5_1_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(6-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-6-oxohexyl)malonamide (<b>10</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43d</b> and <b>46a</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.33 (s, 1H), 9.00 (s, 1H), 8.39 (d, <i>J</i> = 8.0 Hz, 1H), 8.26–8.23 (m, 1H), 7.96 (brs, 3H), 7.81 (d, <i>J</i> = 8.8 Hz, 2H), 7.66 (s, 1H), 7.45–7.36 (m, 4H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.40 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.20 (brs, 1H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.06–4.02 (m, 2H), 3.62–3.56 (m, 2H), 3.38 (s, 2H), 3.33–3.28 (m, 2H), 3.11–3.06 (m, 2H), 2.45 (s, 3H), 2.26–2.23 (m, 1H), 2.14–2.10 (m, 1H), 2.01–1.98 (m, 1H), 1.78–1.67 (m, 5H), 1.54–1.41 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 1.27–1.23 (m, 2H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>48</sub>H<sub>65</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 990.42; found, 990.39.</div></div><div id="sec5_1_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(7-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-7-oxoheptyl)malonamide (<b>11</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43e</b> and <b>46a</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.32 (s, 1H), 9.00 (s, 1H), 8.38 (d, <i>J</i> = 8.0 Hz, 1H), 8.25–8.22 (m, 1H), 7.96 (brs, 3H), 7.82–7.79 (m, 2H), 7.66 (s, 1H), 7.47–7.36 (m, 4H), 7.15 (t, <i>J</i> = 8.0 Hz, 1H), 6.40 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.20 (brs, 1H), 4.95–4.88 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.07–4.02 (m, 2H), 3.63–3.57 (m, 2H), 3.38 (s, 2H), 3.33–3.26 (m, 2H), 3.12–3.07 (m, 2H), 2.45 (s, 3H), 2.29–2.21 (m, 1H), 2.15–2.08 (m, 1H), 2.04–1.98 (m, 1H), 1.82–1.70 (m, 5H), 1.54–1.41 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 1.28–1.19 (m, 4H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>49</sub>H<sub>67</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1004.44; found, 1004.37.</div></div><div id="sec5_1_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(8-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-8-oxooctyl)malonamide (<b>12</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43f</b> and <b>46a</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.33 (s, 1H), 9.00 (s, 1H), 8.38 (d, <i>J</i> = 8.0 Hz, 1H), 8.25–8.23 (m, 1H), 7.96 (brs, 3H), 7.82–7.78 (m, 2H), 7.66 (s, 1H), 7.45–7.35 (m, 4H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.40 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.20 (brs, 1H), 4.95–4.88 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.41 (t, <i>J</i> = 8.0 Hz, 1H), 4.27 (s, 1H), 4.06–4.02 (m, 2H), 3.63–3.57 (m, 2H), 3.38 (s, 2H), 3.33–3.26 (m, 2H), 3.12–3.07 (m, 2H), 2.45 (s, 3H), 2.28–2.21 (m, 1H), 2.14–2.07 (m, 1H), 2.04–1.98 (m, 1H), 1.82–1.70 (m, 5H), 1.51–1.42 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 1.25–1.14 (m, 6H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>50</sub>H<sub>69</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1018.46; found, 1018.41.</div></div><div id="sec5_1_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(9-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-9-oxononyl)malonamide (<b>13</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43g</b> and <b>46a</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.32 (s, 1H), 8.99 (s, 1H), 8.38 (d, <i>J</i> = 8.0 Hz, 1H), 8.26–8.23 (m, 1H), 7.97 (brs, 3H), 7.82–7.78 (m, 2H), 7.66 (s, 1H), 7.45–7.36 (m, 4H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.40 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 0.8 Hz, 1H), 6.20 (brs, 1H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.41 (t, <i>J</i> = 8.8 Hz, 1H), 4.27 (s, 1H), 4.05–4.02 (m, 2H), 3.63–3.57 (m, 2H), 3.38 (s, 2H), 3.33–3.26 (m, 2H), 3.12–3.07 (m, 2H), 2.45 (s, 3H), 2.28–2.21 (m, 1H), 2.13–1.98 (m, 2H), 1.82–1.70 (m, 5H), 1.47–1.40 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 1.28–1.17 (m, 8H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>51</sub>H<sub>71</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1032.47; found, 1032.42.</div></div><div id="sec5_1_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(10-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-10-oxodecyl)malonamide (<b>14</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43h</b> and <b>46a</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.33 (s, 1H), 9.00 (s, 1H), 8.39 (d, <i>J</i> = 8.0 Hz, 1H), 8.26–8.23 (m, 1H), 7.98 (brs, 3H), 7.83–7.78 (m, 2H), 7.67 (s, 1H), 7.47–7.33 (m, 4H), 7.15 (t, <i>J</i> = 8.0 Hz, 1H), 6.41 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.21 (brs, 1H), 4.96–4.88 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.06–4.03 (m, 2H), 3.64–3.58 (m, 2H), 3.39 (s, 2H), 3.34–3.27 (m, 2H), 3.13–3.08 (m, 2H), 2.46 (s, 3H), 2.29–2.22 (m, 1H), 2.14–1.99 (m, 2H), 1.82–1.71 (m, 5H), 1.51–1.39 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 1.27–1.19 (m, 10H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>52</sub>H<sub>73</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1046.49; found, 1046.45.</div></div><div id="sec5_1_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(11-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecyl)malonamide (<b>15</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43i</b> and <b>46a</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.31 (s, 1H), 8.99 (s, 1H), 8.36 (d, <i>J</i> = 8.0 Hz, 1H), 8.24–8.22 (m, 1H), 7.95 (brs, 3H), 7.82–7.75 (m, 2H), 7.66 (s, 1H), 7.44–7.37 (m, 4H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.41 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.18 (brs, 2H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.05–4.02 (m, 2H), 3.64–3.57 (m, 2H), 3.38 (s, 2H), 3.33–3.26 (m, 2H), 3.12–2.97 (m, 2H), 2.45 (s, 3H), 2.28–2.21 (m, 1H), 2.13–1.98 (m, 2H), 1.82–1.70 (m, 5H), 1.50–1.40 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 1.29–1.15 (m, 12H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>53</sub>H<sub>75</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1060.50; found, 1060.51.</div></div><div id="sec5_1_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N3-(12-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-12-oxododecyl)malonamide (<b>16</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43j</b> and <b>46a</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.30 (s, 1H), 8.98 (s, 1H), 8.36 (d, <i>J</i> = 8.0 Hz, 1H), 8.24–8.21 (m, 1H), 7.96 (brs, 3H), 7.83–7.75 (m, 2H), 7.66 (s, 1H), 7.44–7.37 (m, 4H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.41 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.15 (brs, 2H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.05–4.02 (m, 2H), 3.64–3.57 (m, 2H), 3.38 (s, 2H), 3.33–3.26 (m, 2H), 3.12–3.07 (m, 2H), 2.45 (s, 3H), 2.29–2.21 (m, 1H), 2.13–1.98 (m, 2H), 1.83–1.70 (m, 5H), 1.50–1.40 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 1.28–1.16 (m, 14H), 0.93 (s, 9H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 172.03, 170.60, 169.59, 167.18, 165.73, 158.25, 157.90, 155.73, 153.52, 151.48, 147.72, 144.65, 137.25, 135.70, 131.11, 129.67, 128.81, 127.20, 126.37, 120.29, 119.71, 117.59, 114.67, 113.76, 68.74, 58.53, 56.31, 56.23, 52.13, 47.68, 43.05, 37.72, 35.19, 34.88, 34.10, 28.97, 28.88, 28.75, 28.66, 26.43, 26.37, 25.43, 22.43, 21.96, 15.97; UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>54</sub>H<sub>77</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1074.52; found, 1074.38.</div></div><div id="sec5_1_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N4-(11-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecyl)succinimide (<b>17</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43i</b> and <b>46b</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.54 (s, 1H), 8.98 (s, 1H), 8.36 (d, <i>J</i> = 8.0 Hz, 1H), 7.97 (brs, 3H), 7.84–7.75 (m, 2H), 7.66 (s, 1H), 7.49–7.37 (m, 4H), 7.14 (t, <i>J</i> = 8.0 Hz, 1H), 6.43 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.18 (brs, 2H), 4.95–4.88 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.27 (s, 1H), 4.05–4.02 (m, 2H), 3.64–3.57 (m, 2H), 3.33–3.26 (m, 2H), 3.05–3.00 (m, 2H), 2.63–2.59 (m, 2H), 2.45 (s, 3H), 2.41–2.37 (m, 2H), 2.28–2.21 (m, 1H), 2.13–1.98 (m, 2H), 1.82–1.70 (m, 5H), 1.52–1.40 (m, 4H), 1.38 (s, 3H), 1.36 (s, 3H), 1.28–1.17 (m, 12H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>54</sub>H<sub>77</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1074.52; found, 1074.47.</div></div><div id="sec5_1_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N4-(12-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-12-oxododecyl)succinimide (<b>18</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43j</b> and <b>46b</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.54 (s, 1H), 8.98 (s, 1H), 8.36 (d, <i>J</i> = 8.0 Hz, 1H), 7.95 (brs, 3H), 7.84–7.75 (m, 2H), 7.66 (s, 1H), 7.48–7.37 (m, 5H), 7.14 (t, <i>J</i> = 8.0 Hz, 1H), 6.43 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.18 (brs, 2H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.05–4.02 (m, 2H), 3.63–3.57 (m, 2H), 3.33–3.26 (m, 2H), 3.05–3.00 (m, 2H), 2.63–2.59 (m, 2H), 2.45 (s, 3H), 2.41–2.37 (m, 2H), 2.28–2.21 (m, 1H), 2.13–1.98 (m, 2H), 1.82–1.70 (m, 5H), 1.52–1.41 (m, 4H), 1.37 (s, 3H), 1.36 (s, 3H), 1.27–1.16 (m, 14H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>55</sub>H<sub>79</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1088.53; found, 1088.49.</div></div><div id="sec5_1_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N5-(12-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-12-oxododecyl)glutaramide (<b>19</b>)</h5><div class="NLM_p last">This compound was prepared from <b>43j</b> and <b>46c</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.52 (s, 1H), 8.99 (s, 1H), 8.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.96 (brs, 3H), 7.80–7.75 (m, 2H), 7.67 (s, 1H), 7.45–7.37 (m, 4H), 7.15 (t, <i>J</i> = 8.0 Hz, 1H), 6.43 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.19 (brs, 2H), 4.95–4.89 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.41 (t, <i>J</i> = 8.0 Hz, 1H), 4.27 (s, 1H), 4.06–4.02 (m, 2H), 3.64–3.57 (m, 2H), 3.33–3.26 (m, 2H), 3.04–2.99 (m, 2H), 2.45 (s, 3H), 2.39–2.35 (m, 2H), 2.28–2.21 (m, 1H), 2.14–1.98 (m, 4H), 1.82–1.70 (m, 6H), 1.47–1.41 (m, 4H), 1.37 (s, 3H), 1.36 (s, 3H), 1.27–1.18 (m, 16H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>56</sub>H<sub>81</sub>ClN<sub>11</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1102.55; found, 1102.48.</div></div><div id="sec5_1_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N4-((R)-13-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidine-1-carbonyl)-14,14-dimethyl-11-oxo-3,6,9-trioxa-12-azapentadecyl)succinamide (<b>20</b>)</h5><div class="NLM_p last">This compound was prepared from <b>48</b> and <b>46b</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.29 (s, 1H), 9.00 (s, 1H), 8.45 (d, <i>J</i> = 8.0 Hz, 1H), 8.38 (t, <i>J</i> = 5.2 Hz, 1H), 8.07 (brs, 3H), 7.80 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.6 Hz, 1H), 7.66 (s, 1H), 7.45–7.33 (m, 5H), 7.15 (t, <i>J</i> = 8.0 Hz, 1H), 6.40 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.21 (brs, 1H), 4.94–4.87 (m, 1H), 4.54 (d, <i>J</i> = 8.8 Hz, 1H), 4.44 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.05–3.92 (m, 4H), 3.63–3.54 (m, 10H), 3.48–3.42 (m, 4H), 3.35–3.27 (m, 4H), 2.45 (s, 3H), 2.39–2.35 (m, 2H),2.08–1.99 (m, 1H), 1.80–1.69 (m, 5H), 1.38 (s, 3H), 1.36 (s, 3H), 0.94 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>51</sub>H<sub>71</sub>ClN<sub>11</sub>O<sub>9</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1080.46; found, 1080.21.</div></div><div id="sec5_1_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N4-((R)-14-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidine-1-carbonyl)-15,15-dimethyl-12-oxo-3,6,9-trioxa-13-azahexadecyl)succinamide (<b>21</b>)</h5><div class="NLM_p last">This compound was prepared from <b>50a</b> and <b>46b</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.28 (s, 1H), 9.00 (s, 1H), 8.40–8.36 (m, 2H), 8.06 (brs, 3H), 7.84 (d, <i>J</i> = 8.8 Hz, 1H), 7.80 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.6 Hz, 1H), 7.66 (s, 1H), 7.45–7.33 (m, 4H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.40 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.19 (brs, 1H), 4.95–4.88 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.27 (s, 1H), 4.05–4.00 (m, 2H), 3.63–3.55 (m, 4H), 3.51–3.42 (m, 12H), 3.34–3.26 (m, 4H), 2.57–2.52 (m, 1H), 2.45 (s, 3H), 2.40–2.32 (m, 3H), 2.07–1.99 (m, 1H), 1.80–1.69 (m, 5H), 1.38 (s, 3H), 1.36 (s, 3H), 0.94 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>52</sub>H<sub>73</sub>ClN<sub>11</sub>O<sub>9</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1094.47; found, 1094.35.</div></div><div id="sec5_1_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N4-((R)-16-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidine-1-carbonyl)-17,17-dimethyl-14-oxo-3,6,9,12-tetraoxa-15-azaoctadecyl)succinamide (<b>22</b>)</h5><div class="NLM_p last">This compound was prepared from <b>50b</b> and <b>46b</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.29 (s, 1H), 9.00 (s, 1H), 8.45 (d, <i>J</i> = 8.0 Hz, 1H), 8.38–8.36 (m, 1H), 7.96 (brs, 3H), 7.80 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.6 Hz, 1H), 7.66 (s, 1H), 7.46–7.33 (m, 5H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 6.39 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.21 (brs, 1H), 4.94–4.87 (m, 1H), 4.54 (d, <i>J</i> = 8.8 Hz, 1H), 4.43 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.05–4.02 (m, 2H), 3.96 (s, 2H), 3.62–3.53 (m, 14H), 3.47–3.42 (m, 4H), 3.33–3.26 (m, 4H), 2.45 (s, 3H), 2.38–2.33 (m, 2H), 2.07–2.02 (m, 1H), 1.87–1.70 (m, 5H), 1.38 (s, 3H), 1.36 (s, 3H), 0.94 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>53</sub>H<sub>75</sub>ClN<sub>11</sub>O<sub>10</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1124.48; found, 1124.35.</div></div><div id="sec5_1_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(11-(4-(2-((3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-2-oxoethyl)piperazin-1-yl)undecanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>23</b>)</h5><div class="NLM_p last">This compound was prepared from <b>54a</b> and <b>6</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.79 (s, 1H), 8.99 (s, 1H), 8.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.97 (brs, 3H), 7.83–7.78(m, 2H), 7.66 (s, 1H), 7.44–7.36 (m, 4H), 7.19 (t, <i>J</i> = 8.0 Hz, 1H), 6.42 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.18 (brs, 1H), 4.93–4.89 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.41 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.05–4.02 (m, 2H), 3.61–3.51 (m, 4H), 3.36–3.26 (m, 4H), 3.15–2.99 (m, 6H), 2.73–2.61 (m, 2H), 2.45 (s, 3H), 2.33–2.22 (m, 2H), 2.13–1.98 (m, 3H), 1.82–1.63 (m, 7H), 1.49–1.45 (m, 2H), 1.38 (s, 3H), 1.36 (s, 3H), 1.32–1.21 (m, 10H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>56</sub>H<sub>82</sub>ClN<sub>12</sub>O<sub>5</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1101.57; found, 1101.62.</div></div><div id="sec5_1_1_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(12-(4-(2-((3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-2-oxoethyl)piperazin-1-yl)dodecanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>24</b>)</h5><div class="NLM_p last">This compound was prepared from <b>54b</b> and <b>6</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.80 (s, 1H), 8.99 (s, 1H), 8.37 (d, <i>J</i> = 8.0 Hz, 1H), 7.97 (brs, 3H), 7.83–7.78(m, 2H), 7.66 (s, 1H), 7.45–7.36 (m, 4H), 7.20 (t, <i>J</i> = 8.0 Hz, 1H), 6.42 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.18 (brs, 1H), 4.95–4.87 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.41 (t, <i>J</i> = 8.0 Hz, 1H), 4.28 (s, 1H), 4.05–4.02 (m, 2H), 3.63–3.52 (m, 4H), 3.38–3.26 (m, 4H), 3.15–2.99 (m, 6H), 2.73–2.63 (m, 2H), 2.45 (s, 3H), 2.29–2.22 (m, 2H), 2.13–1.98 (m, 3H), 1.82–1.63 (m, 7H), 1.52–1.43 (m, 2H),1.38 (s, 3H), 1.36 (s, 3H), 1.32–1.21 (m, 12H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>57</sub>H<sub>84</sub>ClN<sub>12</sub>O<sub>5</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1115.58; found, 1115.53.</div></div><div id="sec5_1_1_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N16-((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)hexadecanediamide (<b>25</b>)</h5><div class="NLM_p last">This compound was prepared from <b>56</b> and <b>6</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.49 (s, 1H), 8.99 (s, 1H), 8.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.95 (brs, 3H), 7.81–7.78 (m, 2H), 7.66 (s, 1H), 7.45–7.34 (m, 4H), 7.15 (t, <i>J</i> = 8.0 Hz, 1H), 6.42 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.18 (brs, 2H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.42 (t, <i>J</i> = 8.0 Hz, 1H), 4.27 (s, 1H), 4.05–4.02 (m, 2H), 3.63–3.57 (m, 2H), 3.33–3.26 (m, 2H), 2.45 (s, 3H), 2.38–2.34 (m, 2H), 2.28–2.17 (m, 1H), 2.16–1.98 (m, 2H), 1.80–1.68 (m, 5H), 1.60–1.56 (m, 2H), 1.53–1.41 (m, 2H), 1.38 (s, 3H), 1.36 (s, 3H), 1.34–1.18 (m, 20H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>55</sub>H<sub>80</sub>ClN<sub>10</sub>O<sub>5</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1059.54; found, 1059.51.</div></div><div id="sec5_1_1_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(9-(4-(4-((3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-4-oxobutanoyl)piperazin-1-yl)nonanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>26</b>)</h5><div class="NLM_p last">This compound was prepared from <b>60a</b> and <b>46b</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.67 (s, 1H), 9.57 (s, 1H), 8.99 (s, 1H), 8.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.97 (brs, 3H), 7.79 (d, <i>J</i> = 8.8 Hz, 1H), 7.66 (s, 1H), 7.47–7.35 (m, 4H), 7.15 (t, <i>J</i> = 8.0 Hz, 1H), 6.41 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.19 (brs, 1H), 4.95–4.89 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.45–4.39 (m, 2H), 4.28 (s, 1H), 4.14–4.02 (m, 3H), 3.63–3.56 (m, 2H), 3.50–3.45 (m, 2H), 3.40–3.26 (m, 3H), 3.10–3.05 (m, 3H), 2.96–2.83 (m, 2H), 2.72–2.63 (m, 4H), 2.45 (s, 3H), 2.29–2.20 (m, 1H), 2.13–1.99 (m, 2H), 1.82–1.70 (m, 5H), 1.69–1.58 (m, 2H), 1.53–1.42 (m, 2H), 1.38 (s, 3H), 1.36 (s, 3H), 1.30–1.19 (m, 8H), 0.93 (s, 9H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 171.97, 170.87, 170.59, 170.18, 169.57, 158.28, 157.94, 155.78, 153.53, 151.51, 147.74, 144.63, 137.30, 135.95, 131.10, 129.69, 128.82, 126.95, 126.38, 122.27, 120.32, 117.80, 114.86, 113.77, 68.75, 58.54, 56.31, 55.55, 52.09, 50.92, 50.61, 47.69, 41.72, 38.15, 37.76, 35.23, 34.88, 34.12, 28.57, 28.51, 28.37, 27.30, 26.45, 25.90, 25.37, 23.13, 22.42, 21.98, 15.98; UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>56</sub>H<sub>80</sub>ClN<sub>12</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1115.54; found, 1115.29.</div></div><div id="sec5_1_1_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(10-(4-(4-((3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-4-oxobutanoyl)piperazin-1-yl)decanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>27</b>)</h5><div class="NLM_p last">This compound was prepared from <b>60b</b> and <b>46b</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.73 (s, 1H), 9.57 (s, 1H), 8.99 (s, 1H), 8.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.99 (brs, 3H), 7.79 (d, <i>J</i> = 8.8 Hz, 1H), 7.66 (s, 1H), 7.47–7.35 (m, 4H), 7.14 (t, <i>J</i> = 8.0 Hz, 1H), 6.42 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.19 (brs, 1H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.46–4.39 (m, 2H), 4.28 (s, 1H), 4.14–4.02 (m, 3H), 3.63–3.56 (m, 2H), 3.50–3.45 (m, 2H), 3.37–3.26 (m, 3H), 3.10–3.03 (m, 3H), 2.97–2.84 (m, 2H), 2.72–2.61 (m, 4H), 2.45 (s, 3H), 2.29–2.21 (m, 1H), 2.13–1.99 (m, 2H), 1.82–1.70 (m, 5H), 1.69–1.57 (m, 2H), 1.52–1.41 (m, 2H), 1.38 (s, 3H), 1.36 (s, 3H), 1.31–1.20 (m, 10H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>57</sub>H<sub>82</sub>ClN<sub>12</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1129.56; found, 1129.38.</div></div><div id="sec5_1_1_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N4-(4-((8-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-8-oxooctyl)oxy)phenyl)succinimide (<b>28</b>)</h5><div class="NLM_p last">This compound was prepared from <b>64</b> and <b>46b</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.84 (s, 1H), 9.60 (s, 1H), 8.99 (s, 1H), 8.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.95 (brs, 3H), 7.80 (d, <i>J</i> = 8.8 Hz, 1H), 7.66 (s, 1H), 7.49–7.34 (m, 7H), 7.14 (t, <i>J</i> = 8.0 Hz, 1H), 6.87–6.82 (m, 2H), 6.42 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.19 (brs, 1H), 4.95–4.88 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.45–4.40 (m, 1H), 4.27 (s, 1H), 4.05–4.02 (m, 2H), 3.91–3.88 (m, 2H), 3.63–3.57 (m, 2H), 3.33–3.26 (m, 2H), 3.17–3.08 (m, 1H), 2.73–2.69 (m, 2H), 2.63–2.59 (m, 2H), 2.45 (s, 3H), 2.29–2.22 (m, 1H), 2.14–1.98 (m, 2H), 1.82–1.64 (m, 6H), 1.54–1.44 (m, 2H), 1.52–1.41 (m, 2H), 1.38 (s, 3H), 1.36 (s, 3H), 1.31–1.15 (m, 6H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>57</sub>H<sub>75</sub>ClN<sub>11</sub>O<sub>7</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1124.50; found, 1124.56.</div></div><div id="sec5_1_1_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> N1-(3-((3-Amino-5-(4-amino-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)-N4-(4-(4-(5-(((R)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-5-oxopentyl)piperazin-1-yl)butyl)-succinamide (<b>29</b>)</h5><div class="NLM_p last">This compound was prepared from <b>68</b> and <b>46b</b> by a procedure similar to that used for compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.56 (s, 1H), 8.99 (s, 1H), 8.35 (d, <i>J</i> = 8.0 Hz, 1H), 8.02 (brs, 3H), 7.94–7.91 (m, 1H), 7.86 (d, <i>J</i> = 8.4 Hz, 1H), 7.80 (d, <i>J</i> = 8.8 Hz, 1H), 7.66 (s, 1H), 7.46–7.36 (m, 5H), 7.14 (t, <i>J</i> = 8.0 Hz, 1H), 6.87–6.82 (m, 2H), 6.43 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.17 (brs, 2H), 4.95–4.87 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.43–4.37 (m, 1H), 4.23 (s, 1H), 4.05–4.01 (m, 2H), 3.67–3.56 (m, 3H), 3.47–3.27 (m, 5H), 3.10–3.05 (m, 8H), 2.65–2.61 (m, 2H), 2.45 (s, 3H), 2.42–2.39 (m, 2H), 2.30–2.27 (m, 1H), 2.22–2.15 (m, 1H), 2.07–1.99 (m, 1H), 1.83–1.71 (m, 5H), 1.63–1.49 (m, 6H), 1.48–1.40 (m, 2H), 1.38 (s, 3H), 1.36 (s, 3H), 0.94 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>56</sub>H<sub>81</sub>ClN<sub>13</sub>O<sub>6</sub>S<sub>2</sub> [M + 1]<sup>+</sup>, 1124.50; found, 1124.47.</div></div><div id="sec5_1_1_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 3-(<i>tert</i>-Butylthio)-2-chloroaniline (<b>36</b>)</h5><div class="NLM_p last">Cesium carbonate (84.0 g, 257.8 mmol, 2.5 equiv) was added at rt to a solution of 2-chloro-3-fluoroaniline (15.0 g, 103.1 mmol) and <i>tert</i>-butylthiol (41.0 mL, 32.8 g, 360.9 mmol, 3.5 equiv) in anhydrous DMF (160 mL). The reaction mixture was heated to 120 °C and stirred for 36 h under N<sub>2</sub>. After cooling, the reaction mixture was diluted with EtOAc (300 mL), washed with H<sub>2</sub>O, brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated <i>in vacuo</i> to afford crude 3-(<i>tert</i>-butylthio)-2-chloroaniline (19.8 g, 91.8 mmol) which was used directly in the next step. UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>10</sub>H<sub>15</sub>ClNS [M + 1]<sup>+</sup>, 216.06; found, 216.10.</div></div><div id="sec5_1_1_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 3-Amino-2-chlorobenzenethiol Hydrochloride (<b>37</b>)</h5><div class="NLM_p last">3-(<i>tert</i>-Butylthio)-2-chloroaniline (19.5 g, 90.4 mmol) was suspended in conc. HCl (170 mL), and the reaction mixture was vigorously stirred for 6 h at 80 °C. After cooling down, the suspension was filtered, the white solids were washed with cold conc. HCl (15 mL) and hexane (30 mL), and dried under reduced pressure to give 3-amino-2-chlorobenzenethiol hydrochloride (13.8 g, 78%). UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>6</sub>H<sub>7</sub>ClNS [M + 1]<sup>+</sup>, 160.00; found, 160.10.</div></div><div id="sec5_1_1_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 3-((3-Amino-2-chlorophenyl)thio)-6-chloropyrazin-2-amine (<b>39</b>)</h5><div class="NLM_p last">Potassium phosphate (K<sub>3</sub>PO<sub>4</sub>, 20.7 g, 97.6 mmol, 2.6 equiv) was added to a solution of 3-bromo-6-chloropyrazin-2-amine (7.8 g, 37.5 mmol, 1.0 equiv) and 3-amino-2-chlorobenzenethiol hydrochloride (9.6 g, 48.8 mmol, 1.3 equiv) in dioxane (120 mL). The reaction mixture was degassed and stirred at rt for 15 min, then CuI (1.4 g, 7.5 mmol, 0.2 equiv) and 1,10-phenanthroline (2.7 g, 15.0 mmol, 0.4 equiv) were added. After being degassed three times, the mixture was stirred at 90 °C under dry N<sub>2</sub> for 16 h. The reaction mixture was cooled to rt, diluted with EtOAc (150 mL), and filtered through a pad of Celite followed by an EtOAc wash. The volatiles were removed under reduced pressure, and the residue was purified by silica gel chromatography eluting with 0–10% MeOH/DCM to afford 3-((3-amino-2-chlorophenyl)thio)-6-chloropyrazin-2-amine (6.6 g, 61%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 7.70 (s, 1H), 7.01–6.97 (m, 3H), 6.79 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.49 (dd, <i>J</i> = 7.6 Hz, <i>J</i> = 1.6 Hz, 1H), 5.51 (s, 2H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>10</sub>H<sub>9</sub>Cl<sub>2</sub>N<sub>4</sub>S [M + 1]<sup>+</sup>, 286.99; found, 287.05.</div></div><div id="sec5_1_1_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> <i>tert</i>-Butyl (1-(6-amino-5-((3-amino-2-chlorophenyl)thio)pyrazin-2-yl)-4-methylpiperidin-4-yl (<b>40</b>)</h5><div class="NLM_p last">DIPEA (6.5 mL, 4.9 g, 37.5 mmol, 3.0 equiv) (1.0 mL) was added at rt to a solution of 3-((3-amino-2-chlorophenyl)thio)-6-chloropyrazin-2-amine (3.6 g, 12.5 mmol, 1.0 equiv) and <i>tert</i>-butyl (4-methylpiperidin-4-yl)carbamate (5.4 g, 25.0 mmol, 2.0 equiv) in DMSO (50 mL). The reaction mixture was warmed to 100 °C and stirred for 3 h. After cooling, it was poured onto ice-cold water (200 mL). After extraction with EtOAc (100 mL × 2), the combined organic layers were washed with water, brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated under vacuum. The residue was purified by silica gelflash column chromatography with hexane:EtOAc (4:1–1:3) to afford <i>tert</i>-butyl (1-(6-amino-5-((3-amino-2-chlorophenyl)thio)pyrazin-2-yl)-4-methylpiperidin-4-yl as a white solid (4.4 g, 76% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 7.59 (s, 1H), 6.83 (t, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.62 (s, 1H), 6.55 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 5.99 (s, 2H); 5.81 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.6 Hz, 1H), 5.41 (s, 2H), 3.85–3.81 (m, 2H), 3.23–3.18 (m, 2H), 2.09–2.05 (m, 2H), 1.46–1.41 (m, 2H), 1.39 (s, 9H), 1.25 (s, 3H); C<sub>21</sub>H<sub>30</sub>ClN<sub>6</sub>O<sub>2</sub>S [M + 1] <sup>+</sup>, 465.18; found, 465.23.</div></div><div id="sec5_1_1_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> (2<i>S</i>,4<i>R</i>)-1-((S)-2-(3-Aminopropanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((S)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43a</b>)</h5><div class="NLM_p last">HATU (295 mg, 0.66 mmol, 1.1 equiv) was added to a solution of <b>6</b> (335 mg, 0. 60 mmol, 1 equiv), <b>41a</b> (125 mg, 0.66 mmol, 1.1 equiv), and DIEA (0.42 mL, 2.40 mmol, 4.0 equiv) in DMF (6 mL), and the resulting mixture was stirred at rt for 1 h. The solution was diluted with EtOAc and washed with H<sub>2</sub>O, saturated sodium bicarbonate aqueous solution, and brine and then dried over sodium sulfate. After removal of the solvent <i>in vacuo</i>, the residue was purified by HPLC and treated with 4 N HCl dioxane for 2 h to afford <b>43a</b> as a hydrochloride salt (235 mg 71%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.04 (s, 1H), 8.43 (d, <i>J</i> = 8.0 Hz, 1H), 8.23 (d, <i>J</i> = 8.8 Hz, 1H), 7.90 (s, 3H), 7.45–7.37 (m, 4H), 4.95–4.88 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.44–4.40 (m, 1H), 3.65–3.59 (m, 1H), 2.98–2.93 (m, 2H), 2.63–2.58 (m, 2H), 2.46 (s, 3H), 2.07–1.99 (m, 1H), 1.82–1.76 (m, 1H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.30–1.26 (m, 1H), 0.95 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>26</sub>H<sub>38</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 516.26; found, 516.19</div></div><div id="sec5_1_1_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> (2<i>S</i>,4<i>R</i>)-1-((S)-2-(4-Aminobutanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((S)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43b</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.09 (s, 1H), 8.43 (d, <i>J</i> = 8.0 Hz, 1H), 8.01 (s, 3H),7.95 (d, <i>J</i> = 8.8 Hz, 1H), 7.46–7.37 (m, 4H), 4.95–4.88 (m, 1H), 4.50 (d, <i>J</i> = 8.8 Hz, 1H), 4.45–4.41 (m, 1H), 4.32–4.26 (m, 1H), 3.63–3.59 (m, 1H), 2.79–2.71 (m, 2H), 2.63–2.58 (m, 2H), 2.46 (s, 3H), 2.30–2.21 (m, 2H), 2.05–1.99 (m, 1H), 1.82–1.73 (m, 3H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.31–1.26 (m, 1H), 0.94 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>27</sub>H<sub>40</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 530.28; found, 530.19.</div></div><div id="sec5_1_1_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> (2<i>S</i>,4<i>R</i>)-1-((S)-2-(5-Aminopentanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((S)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43c</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.17 (s, 1H), 8.45 (d, <i>J</i> = 8.0 Hz, 1H), 8.02 (brs, 3H), 7.87 (d, <i>J</i> = 8.8 Hz, 1H), 7.49–7.35 (m, 4H), 4.93–4.89 (m, 1H), 4.50 (d, <i>J</i> = 8.8 Hz, 1H), 4.44–4.39 (m, 1H), 4.28–4.26 (m, 1H), 3.65–3.59 (m, 1H), 2.80–2.68 (m, 2H), 2.47 (s, 3H), 2.32–2.17 (m, 2H), 2.09–1.97 (m, 1H), 1.82–1.75 (m, 1H), 1.59–1.48 (m, 4H), 1.43–1.41 (m, 1H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>28</sub>H<sub>42</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 544.30; found, 544.25.</div></div><div id="sec5_1_1_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(6-Aminohexanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43d</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.07 (s, 1H), 8.42 (d, <i>J</i> = 8.0 Hz, 1H), 7.93 (brs, 3H), 7.83 (d, <i>J</i> = 8.8 Hz, 1H), 7.53–7.35 (m, 4H), 4.95–4.89 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.44–4.40 (m, 1H), 4.29–4.27 (m, 1H), 3.65–3.59 (m, 1H), 2.76–2.70 (m, 2H), 2.47 (s, 3H), 2.28–2.09 (m, 2H), 2.07–1.99 (m, 1H), 1.82–1.75 (m, 1H), 1.59–1.43 (m, 5H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.30–1.22 (m, 2H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>29</sub>H<sub>44</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 558.31; found, 558.27</div></div><div id="sec5_1_1_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(7-Aminoheptanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43e</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.09 (s, 1H), 8.45 (d, <i>J</i> = 8.0 Hz, 1H), 7.96 (brs, 3H), 7.81 (d, <i>J</i> = 8.8 Hz, 1H), 7.49–7.34 (m, 4H), 4.95–4.88 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.44–4.40 (m, 1H), 4.30–4.26 (m, 1H), 3.61–3.56 (m, 1H), 2.75–2.70 (m, 2H), 2.46 (s, 3H), 2.28–2.09 (m, 2H), 2.05–2.00 (m, 1H), 1.81–1.74 (m, 1H), 1.54–1.42 (m, 5H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.31–1.21 (m, 4H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>30</sub>H<sub>46</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 572.33; found, 572.31.</div></div><div id="sec5_1_1_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(8-Aminooctanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43f</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.07 (s, 1H), 8.43 (d, <i>J</i> = 8.0 Hz, 1H), 7.94 (brs, 3H), 7.80 (d, <i>J</i> = 8.8 Hz, 1H), 7.46–7.37 (m, 4H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.44–4.40 (m, 1H), 4.29–4.25 (m, 1H), 3.61–3.56 (m, 1H), 2.76–2.71 (m, 2H), 2.46 (s, 3H), 2.28–2.08 (m, 2H), 2.04–1.99 (m, 1H), 1.81–1.75 (m, 1H), 1.53–1.42 (m, 5H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.31–1.26 (m, 6H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>31</sub>H<sub>48</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 586.34; found, 586.37.</div></div><div id="sec5_1_1_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(9-Aminononanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43g</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.10 (s, 1H), 8.42 (d, <i>J</i> = 8.0 Hz, 1H), 7.95 (brs, 3H), 7.80 (d, <i>J</i> = 8.8 Hz, 1H), 7.49–7.36 (m, 4H), 4.95–4.87 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.46–4.39 (m, 1H), 4.30–4.24 (m, 1H), 3.61–3.56 (m, 1H), 3.12–3.05 (m, 1H),2.75–2.70 (m, 2H), 2.46 (s, 3H), 2.28–2.07 (m, 2H), 2.04–1.99 (m, 1H), 1.82–1.76 (m, 1H), 1.55–1.41 (m, 5H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.31–1.25 (m, 8H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>32</sub>H<sub>50</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 600.36; found, 600.33.</div></div><div id="sec5_1_1_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(10-Aminodecanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43h</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.07 (s, 1H), 8.42 (d, <i>J</i> = 8.0 Hz, 1H), 7.92 (brs, 3H), 7.79 (d, <i>J</i> = 8.8 Hz, 1H), 7.48–7.37 (m, 4H), 4.95–4.87 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.42–4.39 (m, 1H), 4.29–4.25 (m, 1H), 3.63–3.56 (m, 1H), 3.14–3.08 (m, 1H), 2.76–2.69 (m, 2H), 2.46 (s, 3H), 2.28–2.06 (m, 2H), 2.03–1.98 (m, 1H), 1.81–1.75 (m, 1H), 1.55–1.42 (m, 5H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.31–1.24 (m, 10H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>33</sub>H<sub>52</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 614.37; found, 614.35.</div></div><div id="sec5_1_1_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (2<i>S</i>,4<i>R</i>)-1-((S)-2-(11-Aminoundecanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((S)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43i</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.02 (s, 1H), 8.39 (d, <i>J</i> = 8.0 Hz, 1H), 7.87 (brs, 3H), 7.79 (d, <i>J</i> = 8.8 Hz, 1H), 7.48–7.35 (m, 4H), 4.95–4.87 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.41–4.38 (m, 1H), 4.27–4.24 (m, 1H), 3.63–3.56 (m, 1H), 3.16–3.09 (m, 1H), 2.77–2.72 (m, 2H), 2.46 (s, 3H), 2.26–2.09 (m, 2H), 2.02–1.96 (m, 1H), 1.82–1.78 (m, 1H), 1.52–1.43 (m, 5H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.29–1.24 (m, 12H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>34</sub>H<sub>54</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 628.39; found, 628.31.</div></div><div id="sec5_1_1_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(12-Aminododecanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>43j</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 8.99 (s, 1H), 8.37 (d, <i>J</i> = 8.0 Hz, 1H), 7.79 (d, <i>J</i> = 8.8 Hz, 1H), 7.64 (brs, 3H), 7.45–7.37 (m, 4H), 4.95–4.89 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.43–4.39 (m, 1H), 4.29–4.26 (m, 1H), 3.63–3.57 (m, 1H), 3.16–3.09 (m, 1H), 2.80–2.72 (m, 2H), 2.45 (s, 3H), 2.33–1.98 (m, 3H), 1.82–1.75 (m, 1H), 1.51–1.45 (m, 5H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.30–1.24 (m, 14H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>35</sub>H<sub>56</sub>N<sub>5</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 642.40; found, 642.36.</div></div><div id="sec5_1_1_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> Methyl 3-((3-((3-Amino-5-(4-((tert-butoxycarbonyl)amino)-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-3-oxopropanoate (<b>45a</b>)</h5><div class="NLM_p last">Methyl 3-chloro-3-oxopropanoate (1.2 g, 8.9 mmol, 1.2 equiv) was added dropwise at 0 °C to a solution of <i>tert</i>-butyl (1-(6-amino-5-((3-amino-2-chlorophenyl)thio)pyrazin-2-yl)-4-methylpiperidin-4-yl (<b>40</b>) (3.4 g, 7.4 mmol, 1.0 equiv) and DIPEA (3.9 mL, 2.9 g, 22.2 mmol, 3.0 equiv) in DCM (50 mL). The reaction mixture was allowed to warm to rt and stirred for 1 h. It was poured into aq NaHCO<sub>3</sub> solution (50 mL). After extraction with DCM (30 mL × 2), the combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated under vacuum. The residue was used in the next step without further purification. UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>25</sub>H<sub>34</sub>ClN<sub>6</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>, 565.20; found, 565.33.</div></div><div id="sec5_1_1_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 3-((3-((3-Amino-5-(4-((tert-butoxycarbonyl)amino)-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-3-oxopropanoic Acid (<b>46a</b>)</h5><div class="NLM_p last">Lithium hydroxide monohydrate (932 mg, 22.2 mmol) was added at 0 °C to a solution of residue obtained from up and was dissolved in THF/MeOH/H<sub>2</sub>O (15 mL/10 mL/5 mL). The reaction mixture was allowed to warm to rt and stirred for 2 h. After quenching with 1 N HCl to pH ∼ 3, the resulting mixture was extracted with EtOAc (40 mL × 2). The organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and then concentrated under reduced pressure. The residue was then purified by silica gel column chromatography eluting with 0–5% MeOH/DCM to give 3-((3-((3-amino-5-(4-((tert-butoxycarbonyl)amino)-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-3-oxopropanoic acid (2.5 g, 61%, two steps) as a white-yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.72 (s, 1H), 9.85 (s, 1H), 7.62 (s, 1H), 7.58 (d, <i>J</i> = 7.6 Hz, 1H), 7.17 (t, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.63 (s, 1H), 6.42 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.11 (s, 2H); 5.81 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.6 Hz, 1H), 3.86–3.84 (m, 2H), 3.49 (s, 2H), 3.24–3.19 (m, 2H), 2.10–2.06 (m, 2H), 1.46–1.41 (m, 2H), 1.39 (s, 9H), 1.25 (s, 3H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>24</sub>H<sub>32</sub>ClN<sub>6</sub>O<sub>5</sub>S [M + 1] <sup>+</sup>, 551.18; found, 551.25.</div></div><div id="sec5_1_1_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 4-((3-((3-Amino-5-(4-((tert-butoxycarbonyl)amino)-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-4-oxobutanoic Acid (<b>46b</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>46a</b> using methyl 4-chloro-4-oxobutanoate as a starting material. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.14 (s, 1H), 9.55 (s, 1H), 7.62 (s, 1H), 7.44 (d, <i>J</i> = 7.6 Hz, 1H), 7.14 (t, <i>J</i> = 8.0 Hz,), 6.63 (s, 1H), 6.42 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.10 (s, 2H); 3.87–3.83 (m, 2H), 3.36 (s, 2H), 3.24–3.19 (m, 2H), 2.64–2.62 (m, 2H), 2.10–2.05 (m, 2H), 1.46–1.41 (m, 2H), 1.39 (s, 9H), 1.25 (s, 3H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>25</sub>H<sub>34</sub>ClN<sub>6</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>, 565.20; found, 565.11.</div></div><div id="sec5_1_1_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 5-((3-((3-Amino-5-(4-((tert-butoxycarbonyl)amino)-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-5-oxopentanoic Acid (<b>46c</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>46a</b> using methyl 5-chloro-5-oxopentanoate as the starting material. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.15 (s, 1H), 9.53 (s, 1H), 7.62 (s, 1H), 7.40 (d, <i>J</i> = 7.6 Hz, 1H), 7.14 (t, <i>J</i> = 8.0 Hz,), 6.63 (s, 1H), 6.43 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.13 (s, 2H); 3.88–3.84 (m, 2H), 3.25–3.19 (m, 2H), 2.43–2.39 (m, 2H), 2.31–2.27 (m, 2H), 2.10–2.05 (m, 2H), 1.84–1.77 (m, 2H), 1.73–1.68 (m, 1H), 1.45–1.40 (m, 2H), 1.39 (s, 9H), 1.25 (s, 3H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>26</sub>H<sub>36</sub>ClN<sub>6</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>, 579.22; found, 579.16.</div></div><div id="sec5_1_1_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> (2<i>R</i>,4<i>S</i>)-1-((R)-14-Amino-2-(<i>tert</i>-butyl)-4-oxo-6,9,12-trioxa-3-azatetradecanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>48</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.05 (s, 1H), 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 7.97 (brs, 3H), 7.47–7.34 (m, 4H), 6.58 (brs, 1H), 4.93–4.87 (m, 1H), 4.55 (d, <i>J</i> = 8.8 Hz, 1H), 4.47–4.42 (m, 1H), 4.31–4.27 (m, 1H), 3.98 (s, 2H), 3.63–3.56 (m, 12H), 2.98–2.94 (m, 1H), 2.46 (s, 3H), 2.09–2.04 (m, 1H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 0.94 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>31</sub>H<sub>48</sub>N<sub>5</sub>O<sub>7</sub>S [M + 1]<sup>+</sup>, 634.33; found, 634.31.</div></div><div id="sec5_1_1_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> (2<i>R</i>,4<i>S</i>)-1-((R)-1-Amino-14-(<i>tert</i>-butyl)-12-oxo-3,6,9-trioxa-13-azapentadecan-15-oyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>50a</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.11 (s, 1H), 8.44 (d, <i>J</i> = 8.0 Hz, 1H), 8.02 (brs, 3H), 7.89 (d, <i>J</i> = 8.0 Hz, 1H), 7.46–7.36 (m, 4H), 4.95–4.87 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.44–4.40 (m, 1H), 4.29–4.26 (m, 1H), 3.62–3.46 (m, 15H), 2.97–2.93 (m, 1H), 2.46 (s, 3H), 2.40–2.33 (m, 1H), 2.05–2.00 (m, 1H), 1.81–1.74 (m, 1H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 0.94 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>32</sub>H<sub>50</sub>N<sub>5</sub>O<sub>7</sub>S [M + 1]<sup>+</sup>, 648.34; found, 648.25.</div></div><div id="sec5_1_1_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> (2<i>R</i>,4<i>S</i>)-1-((R)-17-Amino-2-(<i>tert</i>-butyl)-4-oxo-6,9,12,15-tetraoxa-3-azaheptadecanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>50b</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.05 (s, 1H), 8.47 (d, <i>J</i> = 8.0 Hz, 1H), 7.96 (brs, 3H), 7.45–7.38 (m, 4H), 6.98 (brs, 1H), 4.94–4.87 (m, 1H), 4.54 (d, <i>J</i> = 8.8 Hz, 1H), 4.46–4.41 (m, 1H), 4.31–4.27 (m, 1H), 3.98 (s, 1H), 3.63–3.53 (m, 16H), 2.98–2.94 (m, 2H), 2.46 (s, 3H), 2.08–2.04 (m, 1H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 0.94 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>33</sub>H<sub>52</sub>N<sub>5</sub>O<sub>8</sub>S [M + 1] <sup>+</sup>, 678.35; found, 678.31.</div></div><div id="sec5_1_1_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> <i>tert</i>-Butyl (1-(6-Amino-5-((2-chloro-3-(2-(piperazin-1-yl)acetamido)phenyl)thio)pyrazin-2-yl)-4-methylpiperidin-4-yl)carbamate (<b>52</b>)</h5><div class="NLM_p last"><i>tert</i>-Butyl (1-(6-amino-5-((3-amino-2-chlorophenyl)thio)-pyrazin-2-yl)-4-methylpiperidin-4-yl)carbamate (<b>40</b>) (930 mg, 2.0 mmol, 1.0 equiv), 2-(4-(((9H-fluoren-9-yl)methoxy)carbonyl)piperazin-1-yl)acetic acid (<b>51</b>) (806 mg, 2.2 mmol, 1.1 equiv), and <i>N</i>-methylimidazole (574 mg, 7.0 mmol, 3.5 equiv) were dissolved in MeCN (8.0 mL) and THF (8.0 mL), and then TCFH (730 mg, 2.6 mmol, 1.3 equiv) was added in a single portion. The reaction was stirred until complete, judged by LC-MS. The reaction was then diluted with EtOAc (30 mL) and water (30 mL). The layers were separated and washed with water, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered and concentrated, the residue was dissolved in piperidine (2 mL) and MeCN (10 mL) after stirring at rt for 2 h, and the solvent was removed under vacuum and the residue was purified by pre-HPLC to afford <b>52</b> as a white solid (566 mg, 48% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.00 (s, 1H), 8.01 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 7.62 (s, 1H), 7.19 (t, <i>J</i> = 8.0 Hz, 1H), 6.63 (brs, 1H), 6.39 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.10 (brs, 1H), 3.37–3.29 (m, 2H), 3.27–3.17 (m, 2H), 3.16 (s, 1H), 2.87 (t, <i>J</i> = 4.8 Hz, 2H), 2.58–2.51 (m, 2H), 2.13–2.09 (m, 2H), 1.47–1.42 (m, 2H), 1.39 (s, 9H), 1.25 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>27</sub>H<sub>40</sub>ClN<sub>8</sub>O<sub>3</sub>S [M + 1] <sup>+</sup>, 591.26; found, 591.21.</div></div><div id="sec5_1_1_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 11-(4-(2-((3-((3-Amino-5-(4-((tert-butoxycarbonyl)amino)-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-2-oxoethyl)piperazin-1-yl)undecanoic Acid (<b>54a</b>)</h5><div class="NLM_p last">K<sub>2</sub>CO<sub>3</sub> (3.0 equiv) and KI (1.2 equiv) were added to a solution of the intermediate <b>52</b> (118 mg, 0.2 mmol) and 11-bromoundecanoic acid (1.2 equiv) in DMF (5.0 mL). After the mixture was stirred for 12 h at 60 °C, the reaction was diluted with EtOAc (30 mL) and water (30 mL). The layers were separated and washed with water, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered and concentrated, and the residue was purified by pre-HPLC to afford <b>54a</b> as a white solid (71 mg, 46% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.80 (s, 1H), 9.44 (brs, 1H), 7.81 (d, <i>J</i> = 8.0 Hz, 1H), 7.62 (s, 1H), 7.20 (t, <i>J</i> = 8.0 Hz, 1H), 6.64 (brs, 1H), 6.44 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.09 (brs, 1H), 3.87–3.83 (m, 2H), 3.55–3.50 (m, 2H), 3.38 (s, 1H), 3.25–3.20 (m, 2H), 3.16–3.03 (m, 6H), 2.73–2.65 (m, 2H), 2.19 (t, <i>J</i> = 7.6 Hz, 2H), 2.13–2.06 (m, 2H), 1.66–1.58 (m, 2H), 1.53–1.45 (m, 4H), 1.39 (s, 3H), 1.32–1.24 (m, 15H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>38</sub>H<sub>60</sub>ClN<sub>8</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>, 775.41; found, 775.32.</div></div><div id="sec5_1_1_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 12-(4-(2-((3-((3-Amino-5-(4-((tert-butoxycarbonyl)amino)-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-2-oxoethyl)piperazin-1-yl)dodecanoic Acid (<b>54b</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>54a</b> using 12-bromododecanoic acid as the starting material. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.78 (s, 1H), 9.36 (brs, 1H), 7.82 (d, <i>J</i> = 8.0 Hz, 1H), 7.62 (s, 1H), 7.20 (t, <i>J</i> = 8.0 Hz, 1H), 6.64 (brs, 1H), 6.44 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.09 (brs, 1H), 3.87–3.83 (m, 2H), 3.54–3.49 (m, 2H), 3.36 (s, 1H), 3.25–3.20 (m, 2H), 3.16–3.02 (m, 6H), 2.72–2.64 (m, 2H), 2.19 (t, <i>J</i> = 7.6 Hz, 2H), 2.12–2.05 (m, 2H), 1.68–1.59 (m, 2H), 1.52–1.42 (m, 4H), 1.39 (s, 9H), 1.33–1.21 (m, 17H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>39</sub>H<sub>62</sub>ClN<sub>8</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>, 789.42; found, 789.37.</div></div><div id="sec5_1_1_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 16-((3-((3-Amino-5-(4-((tert-butoxycarbonyl)amino)-4-methylpiperidin-1-yl)pyrazin-2-yl)thio)-2-chlorophenyl)amino)-16-oxohexadecanoic Acid (<b>56</b>)</h5><div class="NLM_p last">Compound <b>40</b> (558 mg, 1.20 mmol, 1.0 equiv), 16-(benzyloxy)-16-oxohexadecanoic acid (<b>55</b>) (542 mg, 1.44 mmol, 1.2 equiv), and <i>N</i>-methylimidazole (345 mg, 4.2 mmol, 3.5 equiv) were dissolved in MeCN (6.0 mL) and THF (6.0 mL), then TCFH (437 mg, 1.56 mmol, 1.3 equiv) was added in a single portion. The reaction was stirred until complete, judged by LC-MS. The reaction was then diluted with EtOAc (30 mL) and water (30 mL). The layers were separated and washed with water, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated; the residue was dissolved in THF (4 mL), MeOH (2 mL), and water (2 mL), and NaOH (192 mg, 4.8 mmol, 4.0 equiv) was added and stirred at rt for 8 h; the pH was adjusted to 6 and diluted with EtOAc (30 mL) and water (30 mL). The layers were separated and washed with water, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered and concentrated, and the residue was purified by pre-HPLC to afford <b>56</b> as a white solid (228 mg, 26% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 11.96 (s, 1H), 9.47 (s, 1H), 7.62 (s, 1H), 7.40 (d, <i>J</i> = 8.0 Hz, 1H), 7.62 (s, 1H), 7.14 (t, <i>J</i> = 8.0 Hz, 1H), 6.63 (brs, 1H), 6.42 (dd, <i>J</i> = 8.0 Hz, <i>J</i> = 1.2 Hz, 1H), 6.09 (brs, 2H), 3.88–3.83 (m, 2H), 3.25–3.19 (m, 2H), 2.38–2.32 (m, 2H), 2.18 (t, <i>J</i> = 7.6 Hz, 2H), 2.11–2.03 (m, 2H), 1.63–1.54 (m, 2H), 1.52–1.42 (m, 4H), 1.39 (s, 9H), 1.35–1.19 (m, 23H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>37</sub>H<sub>58</sub>ClN<sub>6</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>, 733.39; found, 733.41.</div></div><div id="sec5_1_1_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 9-(4-(tert-Butoxycarbonyl)piperazin-1-yl)nonanoic Acid (<b>59a</b>)</h5><div class="NLM_p last">A mixture of <i>tert</i>-butyl piperazine-1-carboxylate (<b>57</b>) (1.86 g, 10.0 mmol, 1 equiv), sodium iodide (1.49 g, 10.0 mmol, 1 equiv), potassium carbonate (4.14 g, 30.0 mmol, 3 equiv), and methyl 9-bromononanoate (3.02 g, 12.0 mmol, 1.2 equiv) in MeCN (60 mL) was stirred at 60 °C for 16 h. After removal of solvent by rotary evaporation, the resulting residue was diluted with DCM and insoluble inorganics were filtered off, and the liquid filtrate was concentrated in vacuo. The residue was dissolved in THF/MeOH/H<sub>2</sub>O (15 mL/9 mL/6 mL), and lithium hydroxide monohydrate (1.68 g, 40.0 mmol, 2 equiv) was added at 0 °C. The reaction mixture was allowed to warm to rt and stirred for 2 h. After quenching with 1 N HCl to pH ∼ 3, the resulting mixture was extracted with EtOAc (100 mL × 2). The organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and then concentrated under reduced pressure, and the residue was purified by Biotage silica gel column chromatography to afford <b>59a</b> as a white solid (1.43 g, 42%): <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 11.99 (s, 1H), 4.06–3.90 (m, 2H), 3.43–3.20 (m, 4H), 3.09–2.86 (m, 4H), 2.19 (t, J = 7.6 Hz, 2H), 1.73–1.65 (m, 2H), 1.52–1.47 (m, 2H), 1.41 (s, 9H), 1.27 (s, 8H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>18</sub>H<sub>35</sub>N<sub>2</sub>O<sub>4</sub> [M + 1] <sup>+</sup>, 343.26; found, 343.11.</div></div><div id="sec5_1_1_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 10-(4-(tert-Butoxycarbonyl)piperazin-1-yl)decanoic Acid (<b>59b</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>59a</b> using methyl 10-bromodecanoate as the starting material. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 11.98 (s, 1H), 4.07–3.92 (m, 2H), 3.47–3.34 (m, 2H), 3.26–3.13 (m, 2H), 3.09–2.98 (m, 2H), 2.96–2.85 (m, 2H), 2.18 (t, J = 7.6 Hz, 2H), 1.70–1.59 (m, 2H), 1.53–1.45 (m, 2H), 1.42 (s, 9H), 1.26 (s, 10H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>19</sub>H<sub>37</sub>N<sub>2</sub>O<sub>4</sub> [M + 1] <sup>+</sup>, 357.27; found, 357.16.</div></div><div id="sec5_1_1_52" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> (2<i>R</i>,4<i>S</i>)-1-((R)-3,3-Dimethyl-2-(9-(piperazin-1-yl)nonanamido)butanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>60a</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.18 (brs, 2H), 8.99 (s, 1H), 8.35 (d, <i>J</i> = 8.0 Hz, 1H), 7.76 (d, <i>J</i> = 8.8 Hz, 1H), 7.48–7.37 (m, 4H), 4.93–4.90 (m, 1H), 4.52 (d, <i>J</i> = 8.8 Hz, 1H), 4.43–4.39 (m, 1H), 4.30–4.26 (m, 1H), 3.64–3.56 (m, 3H), 3.51–3.21 (m, 6H), 3.16–3.09 (m, 3H), 2.45 (s, 3H), 2.33–1.99 (m, 3H), 1.84–1.77 (m, 1H), 1.66–1.57 (m, 2H), 1.51–1.45 (m, 2H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.30–1.24 (m, 8H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>36</sub>H<sub>57</sub>N<sub>6</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 669.42; found, 669.37.</div></div><div id="sec5_1_1_53" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> (2<i>R</i>,4<i>S</i>)-1-((R)-3,3-Dimethyl-2-(10-(piperazin-1-yl)decanamido)butanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>60b</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.24 (brs, 2H), 8.99 (s, 1H), 8.35 (d, <i>J</i> = 8.0 Hz, 1H), 7.76 (d, <i>J</i> = 8.8 Hz, 1H), 7.48–7.37 (m, 4H), 4.95–4.90 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.43–4.39 (m, 1H), 4.31–4.27 (m, 1H), 3.63–3.55 (m, 3H), 3.52–3.22 (m, 6H), 3.16–3.10 (m, 3H), 2.45 (s, 3H), 2.33–1.99 (m, 3H), 1.83–1.76 (m, 1H), 1.65–1.56 (m, 2H), 1.50–1.44 (m, 2H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.31–1.25 (m, 10H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>37</sub>H<sub>59</sub>N<sub>6</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>, 683.43; found, 683.41.</div></div><div id="sec5_1_1_54" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> 8-(4-((tert-Butoxycarbonyl)amino)phenoxy)octanoic Acid (<b>63</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>59a</b> using <i>tert</i>-butyl (4-hydroxyphenyl)carbamate and methyl 8-bromooctanoate as the starting material. UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>19</sub>H<sub>30</sub>NO<sub>5</sub> [M + 1] <sup>+</sup>, 352.21; found, 352.10.</div></div><div id="sec5_1_1_55" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(8-(4-Aminophenoxy)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>64</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.92 (brs, 2H), 9.00 (s, 1H), 8.36 (d, <i>J</i> = 8.0 Hz, 1H), 7.77 (d, <i>J</i> = 8.8 Hz, 1H), 7.48–7.37 (m, 4H), 7.29–7.26 (m, 2H), 7.05–7.02 (m, 2H), 4.95–4.88 (m, 1H), 4.51 (d, <i>J</i> = 8.8 Hz, 1H), 4.44–4.40 (m, 1H), 4.31–4.28 (m, 1H), 3.98–3.94 (m, 2H), 3.64–3.57 (m, 2H), 2.45 (s, 3H), 2.33–1.98 (m, 3H), 1.83–1.76 (m, 1H), 1.73–1.66 (m, 2H), 1.54–1.42 (m, 2H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 1.30–1.22 (m, 6H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>37</sub>H<sub>52</sub>N<sub>5</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>, 678.37; found, 678.41.</div></div><div id="sec5_1_1_56" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> 5-(4-(3-((tert-Butoxycarbonyl)amino)propyl)piperazin-1-yl)pentanoic Acid (<b>67</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>59a</b> using <i>tert</i>-butyl (3-(piperazin-1-yl)propyl)carbamate and ethyl 5-bromopentanoate as starting material. UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>17</sub>H<sub>34</sub>N<sub>3</sub>O<sub>4</sub> [M + 1]<sup>+</sup>, 344.25; found, 344.17.</div></div><div id="sec5_1_1_57" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> (2<i>R</i>,4<i>S</i>)-1-((R)-2-(5-(4-(3-Aminopropyl)piperazin-1-yl)pentanamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-((R)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide Hydrochloride (<b>68</b>)</h5><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>43a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 9.08 (brs, 1H), 8.40 (d, <i>J</i> = 8.0 Hz, 1H), 7.96 (brs, 3H), 7.73 (d, <i>J</i> = 8.8 Hz, 1H), 7.43–7.36 (m, 4H), 4.93–4.87 (m, 1H), 4.50 (d, <i>J</i> = 8.8 Hz, 1H), 4.42–4.38 (m, 1H), 4.30–4.26 (m, 1H), 3.62–3.54 (m, 1H), 3.51–3.21 (m, 6H), 3.16–3.10 (m, 3H), 2.45 (s, 3H), 2.38–1.96 (m, 7H), 1.83–1.76 (m, 1H), 1.65–1.56 (m, 2H), 1.52–1.41 (m, 6H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 0.93 (s, 9H); UPLC-MS (ESI<sup>+</sup>): calculated for C<sub>35</sub>H<sub>56</sub>N<sub>7</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>: 670.41; found, 670.27.</div></div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Cell Lines and Cell Culture</h3><div class="NLM_p last">The MV4;11 cell line was purchased from the American Type Culture Collection (ATCC), cultured in Iscove’s modified Dulbecco’s media (IMDM). Esophageal cancer cell lines KYSE70 and KYSE520 were purchased from DSMZ (Braunschweig, Germany), grown in RPMI 1640 (Invitrogen). All of the cells were supplemented with 10% fetal bovine serum (Invitrogen) at 37 °C in a humidified 5% CO2 incubator.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Cell Growth Assay</h3><div class="NLM_p last">Cell viability was evaluated with a WST-8 assay (Dojindo) following the manufacturer’s instructions. Briefly, cells were seeded in 96-well cell culture plates at a density of 10,000–20,000 cells/well in 200 μL for MV4;11 cell line or 2,000–3,000 for KYSE-520 cell line of culture medium containing serial dilution of testing compounds. After 4 days of treatment, cell growth was measured by a lactate dehydrogenase-based WST-8 assay (Dojindo Molecular Technologies) using a Tecan Infinite M-1000 multimode microplate reader (Tecan US, Morrisville, NC). The WST-8 reagent was added to each well, and cells were incubated for an additional 1–2 h and read at 450 nm. The readings were normalized to the vehicle-treated cells, and the IC<sub>50</sub> was calculated by nonlinear regression analysis using the GraphPad Prism 6 software.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Western Blot Analysis</h3><div class="NLM_p">Western blotting and quantification were performed with regular Western blot method or LI-COR Odyssey system. Treated cells were lysed by RIPA buffer supplemented with protease and phosphatase inhibitors. The cell lysates were separated by 4–12% SDS–PAGE gels and blotted into PVDF (polyvinylidene difluoride) membranes. Antibodies used in the study are indicated in the figure legends. The net protein bands and loading controls are calculated by deducting the background from the inverted band value. The final relative quantification values are the ratio of the net band to net loading control.</div><div class="NLM_p last">For the in vitro kinetics studies of SHP2 expression, cancer cells seeded in a 6-well plate overnight were treated with the compounds for another 2, 4, 8, 12, and 24 h. The treated cells were harvested, and the level of SHP2 protein was examined by blot analysis. GAPDH was used as a loading control.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Molecular Modeling</h3><div class="NLM_p last">The crystal structures of SHP2 with SHP099 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EHR">5EHR</a>),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> 9b (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XZR">5XZR</a>),<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> SHP099+SHP244 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BMU">6BMU</a>),<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> SHP504 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BMV">6BMV</a>),<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and SHP099+SHP844 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BMY">6BMY</a>)<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> were used to model the binding pose of compounds <b>4</b> and <b>5</b> with SHP2. SHP2 protein corrdinates from these crystal structures were extracted and protonated at pH 7.0 using the “protonate 3D” module in the MOE program (Molecular Operating Environment (MOE). Compounds <b>4</b> and <b>5</b> were created and optimized using MOE before performing an ensemble docking calculation using the GOLD program (version 5.2). Default parameters in the GOLD program were used in the docking calculation, and the PLP scoring function was used as the fitness function to rank the docked poses. The top ranked pose was used as the binding model.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i80"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00471" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41383" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41383" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00471?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00471</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Western blotting analysis of SHP2 proteins in KYSE520 and MV4;11 cells treated with the SHP2 degraders and <sup>1</sup>H NMR, <sup>13</sup>C NMR, and UPLC-MS spectra of representative compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular string files for all of the final target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Modeled structures of compound <b>4</b> in complex with SHP2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Modeled structures of compound <b>5</b> in complex with SHP2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_004.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_001.pdf">jm0c00471_si_001.pdf (1.41 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_002.csv">jm0c00471_si_002.csv (4.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_003.pdb">jm0c00471_si_003.pdb (711.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_004.pdb">jm0c00471_si_004.pdb (711.65 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00471" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14851" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14851" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaomeng Wang</span> - <span class="hlFld-Affiliation affiliation">Departments
of Internal Medicine, 
    Pharmacology, 
    Medicinal Chemistry, 
    , , and , Rogel Cancer
Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6c1f040d030109020b2c1901050f0442090819"><span class="__cf_email__" data-cfemail="5c2f343d333139323b1c2931353f3472393829">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingliang Wang</span> - <span class="hlFld-Affiliation affiliation">Departments
of Internal Medicine, , and , Rogel Cancer
Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianfeng Lu</span> - <span class="hlFld-Affiliation affiliation">Departments
of Internal Medicine, , and , Rogel Cancer
Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mi Wang</span> - <span class="hlFld-Affiliation affiliation">Departments
of Internal Medicine, , and , Rogel Cancer
Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao-Yie Yang</span> - <span class="hlFld-Affiliation affiliation">Departments
of Internal Medicine, , and , Rogel Cancer
Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5445-0109" title="Orcid link">http://orcid.org/0000-0002-5445-0109</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.W. and J.L. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The University of Michigan has filed a patent application on these SHP2 degraders, which has been licensed by Oncopia Therapeutics Inc. S. Wang, M. Wang, J. Lu, M. Wang are co-inventors on the patent application. The University of Michigan has received a research contract from Oncopia Therapeutics. S.W. is a co-founder of Oncopia, owns shares in Oncopia and is a paid consultant to Oncopia. The University of Michigan also owns shares in Oncopia.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i82">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05365" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05365" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This study was supported in part by a research contract from Oncopia Therapeutics Inc. and the Michigan Comprehensive Cancer Center (now Rogel Cancer Center) Core grant (P30 CA046592) from the National Cancer Institute (NCI), NIH.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">SHP2</td><td class="NLM_def"><p class="first last">Src homology 2 domain-containing phosphatase 2</p></td></tr><tr><td class="NLM_term">PTP</td><td class="NLM_def"><p class="first last">protein tyrosine phosphatase</p></td></tr><tr><td class="NLM_term">NS</td><td class="NLM_def"><p class="first last">Noonan Syndrome</p></td></tr><tr><td class="NLM_term">JMML</td><td class="NLM_def"><p class="first last">Juvenile myelomonocytic leukemia</p></td></tr><tr><td class="NLM_term">ERK</td><td class="NLM_def"><p class="first last">extracellular signal-regulated kinase</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)-methylene]-1H-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau</p></td></tr><tr><td class="NLM_term">TCFH</td><td class="NLM_def"><p class="first last">N,N,N,N-tetramethylchloroformamidinium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">NMI</td><td class="NLM_def"><p class="first last"><i>N</i>-methylimidazole</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">N,N-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1H-1,2,3-triazolo[4,5-<i>b</i>]-pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26112" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26112" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 62 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tartaglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehler, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zampino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Burgt, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crosby, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ion, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalidas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patton, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucherlapati, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelb, B. D.</span></span> <span> </span><span class="NLM_article-title">Mutations in PTPN11, encoding the protein tyrosine phosphatase SHP-2, cause Noonan syndrome</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">468</span>, <span class="refDoi"> DOI: 10.1038/ng772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fng772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=11704759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFGjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=465-468&author=M.+Tartagliaauthor=E.+L.+Mehlerauthor=R.+Goldbergauthor=G.+Zampinoauthor=H.+G.+Brunnerauthor=H.+Kremerauthor=I.+van+der+Burgtauthor=A.+H.+Crosbyauthor=A.+Ionauthor=S.+Jefferyauthor=K.+Kalidasauthor=M.+A.+Pattonauthor=R.+S.+Kucherlapatiauthor=B.+D.+Gelb&title=Mutations+in+PTPN11%2C+encoding+the+protein+tyrosine+phosphatase+SHP-2%2C+cause+Noonan+syndrome&doi=10.1038%2Fng772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in PTPN11, encoding the protein tyrosine phosphatase SHP-2, cause Noonan syndrome</span></div><div class="casAuthors">Tartaglia, Marco; Mehler, Ernest L.; Goldberg, Rosalie; Zampino, Giuseppe; Brunner, Han G.; Kremer, Hannie; van der Burgt, Ineke; Crosby, Andrew H.; Ion, Andra; Jeffery, Steve; Kalidas, Kamini; Patton, Michael A.; Kucherlapati, Raju S.; Gelb, Bruce D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">465-468</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Noonan syndrome (MIM 163950) is an autosomal dominant disorder characterized by dysmorphic facial features, proportionate short stature and heart disease (most commonly pulmonic stenosis and hypertrophic cardiomyopathy).  Webbed neck, chest deformity, cryptorchidism, mental retardation and bleeding diathesis also are frequently assocd. with this disease.  This syndrome is relatively common, with an estd. incidence of 1 in 1000-2500 live births.  It has been mapped to a 5-cM region (N-SH2) on chromosome 12q24.1, and genetic heterogeneity has also been documented.  Here we show that missense mutations in PTPN11 (MIM 176876)-a gene encoding the nonreceptor protein tyrosine phosphatase SHP-2, which contains two Src homol. 2 (SH2) domains-cause Noonan syndrome and account for more than 50% of the cases that we examd.  All PTPN11 missense mutations cluster in interacting portions of the amino N-SH2 domain and the phosphotyrosine phosphatase domains, which are involved in switching the protein between its inactive and active conformations.  An energetics-based structural anal. of two N-SH2 mutants indicates that in these mutants there may be a significant shift of the equil. favoring the active conformation.  This implies that they are gain-of-function changes and that the pathogenesis of Noonan syndrome arises from excessive SHP-2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKg7QISHW0bbVg90H21EOLACvtfcHk0lgsE2DlY0yxhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFGjt7w%253D&md5=c8225c08340f9446cd58c249de42e515</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fng772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng772%26sid%3Dliteratum%253Aachs%26aulast%3DTartaglia%26aufirst%3DM.%26aulast%3DMehler%26aufirst%3DE.%2BL.%26aulast%3DGoldberg%26aufirst%3DR.%26aulast%3DZampino%26aufirst%3DG.%26aulast%3DBrunner%26aufirst%3DH.%2BG.%26aulast%3DKremer%26aufirst%3DH.%26aulast%3Dvan%2Bder%2BBurgt%26aufirst%3DI.%26aulast%3DCrosby%26aufirst%3DA.%2BH.%26aulast%3DIon%26aufirst%3DA.%26aulast%3DJeffery%26aufirst%3DS.%26aulast%3DKalidas%26aufirst%3DK.%26aulast%3DPatton%26aufirst%3DM.%2BA.%26aulast%3DKucherlapati%26aufirst%3DR.%2BS.%26aulast%3DGelb%26aufirst%3DB.%2BD.%26atitle%3DMutations%2520in%2520PTPN11%252C%2520encoding%2520the%2520protein%2520tyrosine%2520phosphatase%2520SHP-2%252C%2520cause%2520Noonan%2520syndrome%26jtitle%3DNat.%2520Genet.%26date%3D2001%26volume%3D29%26spage%3D465%26epage%3D468%26doi%3D10.1038%2Fng772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kontaridis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span> <span> </span><span class="NLM_article-title">PTPN11 (Shp2) mutations in LEOPARD syndrome have dominant negative, not activating, effects</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">6785</span>– <span class="NLM_lpage">6792</span>, <span class="refDoi"> DOI: 10.1074/jbc.M513068200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1074%2Fjbc.M513068200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=16377799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFOgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=6785-6792&author=M.+I.+Kontaridisauthor=K.+D.+Swansonauthor=F.+S.+Davidauthor=D.+Barfordauthor=B.+G.+Neel&title=PTPN11+%28Shp2%29+mutations+in+LEOPARD+syndrome+have+dominant+negative%2C+not+activating%2C+effects&doi=10.1074%2Fjbc.M513068200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PTPN11 (Shp2) Mutations in LEOPARD Syndrome Have Dominant Negative, Not Activating, Effects</span></div><div class="casAuthors">Kontaridis, Maria I.; Swanson, Kenneth D.; David, Frank S.; Barford, David; Neel, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6785-6792</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Multiple lentigines/LEOPARD syndrome (LS) is a rare, autosomal dominant disorder characterized by Lentigines, ECG abnormalities, Ocular hypertelorism, Pulmonic valvular stenosis, Abnormalities of genitalia, Retardation of growth, and Deafness.  Like the more common Noonan syndrome (NS), LS is caused by germ line missense mutations in PTPN11, encoding the protein-tyrosine phosphatase Shp2.  Enzymol., structural, cell biol., and mouse genetic studies indicate that NS is caused by gain-of-function PTPN11 mutations.  Because NS and LS share several features, LS has been viewed as an NS variant.  We examd. a panel of LS mutants, including the two most common alleles.  Surprisingly, we found that in marked contrast to NS, LS mutants are catalytically defective and act as dominant neg. mutations that interfere with growth factor/Erk-mitogen-activated protein kinase-mediated signaling.  Mol. modeling and biochem. studies suggest that LS mutations contort the Shp2 catalytic domain and result in open, inactive forms of Shp2.  Our results establish that the pathogenesis of LS and NS is distinct and suggest that these disorders should be distinguished by mutational anal. rather than clin. presentation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAcRbFkVLdSLVg90H21EOLACvtfcHk0lgsE2DlY0yxhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvFOgtrw%253D&md5=83adc77383a03535fa7c8eb92c34ca5c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M513068200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M513068200%26sid%3Dliteratum%253Aachs%26aulast%3DKontaridis%26aufirst%3DM.%2BI.%26aulast%3DSwanson%26aufirst%3DK.%2BD.%26aulast%3DDavid%26aufirst%3DF.%2BS.%26aulast%3DBarford%26aufirst%3DD.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26atitle%3DPTPN11%2520%2528Shp2%2529%2520mutations%2520in%2520LEOPARD%2520syndrome%2520have%2520dominant%2520negative%252C%2520not%2520activating%252C%2520effects%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D6785%26epage%3D6792%26doi%3D10.1074%2Fjbc.M513068200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tartaglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niemeyer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fragale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buechner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hählen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelb, B. D.</span></span> <span> </span><span class="NLM_article-title">Somatic mutations in PTPN11 in juvenile myelomonocytic leukemia, myelodysplastic syndromes and acute myeloid leukemia</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">148</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1038/ng1156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fng1156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=12717436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFSltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2003&pages=148-150&author=M.+Tartagliaauthor=C.+M.+Niemeyerauthor=A.+Fragaleauthor=X.+Songauthor=J.+Buechnerauthor=A.+Jungauthor=K.+H%C3%A4hlenauthor=H.+Hasleauthor=J.+D.+Lichtauthor=B.+D.+Gelb&title=Somatic+mutations+in+PTPN11+in+juvenile+myelomonocytic+leukemia%2C+myelodysplastic+syndromes+and+acute+myeloid+leukemia&doi=10.1038%2Fng1156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Somatic mutations in PTPN11 in juvenile myelomonocytic leukemia, myelodysplastic syndromes and acute myeloid leukemia</span></div><div class="casAuthors">Tartaglia, Marco; Niemeyer, Charlotte M.; Fragale, Alessandra; Song, Xiaoling; Buechner, Jochen; Jung, Andreas; Haehlen, Karel; Hasle, Henrik; Licht, Jonathan D.; Gelb, Bruce D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-150</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report here that individuals with Noonan syndrome and juvenile myelomonocytic leukemia (JMML) have germline mutations in PTPN11 and that somatic mutations in PTPN11 account for 34% of non-syndromic JMML.  Furthermore, we found mutations in PTPN11 in a small percentage of individuals with myelodysplastic syndrome (MDS) and de novo acute myeloid leukemia (AML).  Functional analyses documented that the two most common mutations in PTPN11 assocd. with JMML caused a gain of function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCwFhlUQDJQLVg90H21EOLACvtfcHk0liHaRNjJg2O6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFSltrY%253D&md5=67d6900b46520ee461330bed95cbc890</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fng1156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng1156%26sid%3Dliteratum%253Aachs%26aulast%3DTartaglia%26aufirst%3DM.%26aulast%3DNiemeyer%26aufirst%3DC.%2BM.%26aulast%3DFragale%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DBuechner%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DA.%26aulast%3DH%25C3%25A4hlen%26aufirst%3DK.%26aulast%3DHasle%26aufirst%3DH.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DGelb%26aufirst%3DB.%2BD.%26atitle%3DSomatic%2520mutations%2520in%2520PTPN11%2520in%2520juvenile%2520myelomonocytic%2520leukemia%252C%2520myelodysplastic%2520syndromes%2520and%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Genet.%26date%3D2003%26volume%3D34%26spage%3D148%26epage%3D150%26doi%3D10.1038%2Fng1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bentires-Alj, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halmos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maris, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugarbaker, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velculescu, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelstein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span> <span> </span><span class="NLM_article-title">Activating mutations of the noonan syndrome-associated SHP2/PTPN11 gene in human solid tumors and adult acute myelogenous leukemia</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">8816</span>– <span class="NLM_lpage">8820</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-04-1923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1158%2F0008-5472.CAN-04-1923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=15604238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOms7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=8816-8820&author=M.+Bentires-Aljauthor=J.+G.+Paezauthor=F.+S.+Davidauthor=H.+Keilhackauthor=B.+Halmosauthor=K.+Naokiauthor=J.+M.+Marisauthor=A.+Richardsonauthor=A.+Bardelliauthor=D.+J.+Sugarbakerauthor=W.+G.+Richardsauthor=J.+Duauthor=L.+Girardauthor=J.+D.+Minnaauthor=M.+L.+Lohauthor=D.+E.+Fisherauthor=V.+E.+Velculescuauthor=B.+Vogelsteinauthor=M.+Meyersonauthor=W.+R.+Sellersauthor=B.+G.+Neel&title=Activating+mutations+of+the+noonan+syndrome-associated+SHP2%2FPTPN11+gene+in+human+solid+tumors+and+adult+acute+myelogenous+leukemia&doi=10.1158%2F0008-5472.CAN-04-1923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Activating Mutations of the Noonan Syndrome-Associated SHP2/PTPN11 Gene in Human Solid Tumors and Adult Acute Myelogenous Leukemia</span></div><div class="casAuthors">Bentires-Alj, Mohamed; Paez, J. Guillermo; David, Frank S.; Keilhack, Heike; Halmos, Balazs; Naoki, Katsuhiko; Maris, John M.; Richardson, Andrea; Bardelli, Alberto; Sugarbaker, David J.; Richards, William G.; Du, Jinyan; Girard, Luc; Minna, John D.; Loh, Mignon L.; Fisher, David E.; Velculescu, Victor E.; Vogelstein, Bert; Meyerson, Matthew; Sellers, William R.; Neel, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8816-8820</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The SH2 domain-contg. protein-tyrosine phosphatase PTPN11 (Shp2) is required for normal development and is an essential component of signaling pathways initiated by growth factors, cytokines, and extracellular matrix.  In many of these pathways, Shp2 acts upstream of Ras.  About 50% of patients with Noonan syndrome have germ-line PTPN11 gain of function mutations.  Assocns. between Noonan syndrome and an increased risk of some malignancies, notably leukemia and neuroblastoma, have been reported, and recent data indicate that somatic PTPN11 mutations occur in children with sporadic juvenile myelomonocytic leukemia, myelodysplasic syndrome, B-cell acute lymphoblastic leukemia, and acute myelogenous leukemia (AML).  Juvenile myelomonocytic leukemia patients without PTPN11 mutations have either homozygotic NF-1 deletion or activating RAS mutations.  Given the role of Shp2 in Ras activation and the frequent mutation of RAS in human tumors, these data raise the possibility that PTPN11 mutations play a broader role in cancer.  The authors asked whether PTPN11 mutations occur in other malignancies in which activating RAS mutations occur at low but significant frequency.  Sequencing of PTPN11 from 13 different human neoplasms including breast, lung, gastric, and neuroblastoma tumors and adult AML and acute lymphoblastic leukemia revealed 11 missense mutations.  Five are known mutations predicted to result in an activated form of Shp2, whereas six are new mutations.  Biochem. anal. confirmed that several of the new mutations result in increased Shp2 activity.  The data demonstrate that mutations in PTPN11 occur at low frequency in several human cancers, esp. neuroblastoma and AML, and suggest that Shp2 may be a novel target for antineoplastic therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3taKvDtHXBLVg90H21EOLACvtfcHk0liHaRNjJg2O6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOms7fK&md5=f6f9fb8c6e9d5260901a03081db259c7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-04-1923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-04-1923%26sid%3Dliteratum%253Aachs%26aulast%3DBentires-Alj%26aufirst%3DM.%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DDavid%26aufirst%3DF.%2BS.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DHalmos%26aufirst%3DB.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DMaris%26aufirst%3DJ.%2BM.%26aulast%3DRichardson%26aufirst%3DA.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DSugarbaker%26aufirst%3DD.%2BJ.%26aulast%3DRichards%26aufirst%3DW.%2BG.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DGirard%26aufirst%3DL.%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DLoh%26aufirst%3DM.%2BL.%26aulast%3DFisher%26aufirst%3DD.%2BE.%26aulast%3DVelculescu%26aufirst%3DV.%2BE.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26atitle%3DActivating%2520mutations%2520of%2520the%2520noonan%2520syndrome-associated%2520SHP2%252FPTPN11%2520gene%2520in%2520human%2520solid%2520tumors%2520and%2520adult%2520acute%2520myelogenous%2520leukemia%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D8816%26epage%3D8820%26doi%3D10.1158%2F0008-5472.CAN-04-1923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Cancer Genome Atlas Research Network</span> <span> </span><span class="NLM_article-title">Comprehensive genomic characterization
defines human glioblastoma genes and core pathways</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>455</i></span>,  <span class="NLM_fpage">1061</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1038/nature07385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fnature07385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18772890" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=455&publication_year=2008&pages=1061-1068&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization%0Adefines+human+glioblastoma+genes+and+core+pathways&doi=10.1038%2Fnature07385"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature07385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07385%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%2520genomic%2520characterization%250Adefines%2520human%2520glioblastoma%2520genes%2520and%2520core%2520pathways%26jtitle%3DNature%26date%3D2008%26volume%3D455%26spage%3D1061%26epage%3D1068%26doi%3D10.1038%2Fnature07385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalaitzidis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span> <span> </span><span class="NLM_article-title">The tyrosine phosphatase Shp2 (PTPN11) in cancer</span>. <i>Cancer Metastasis Rev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1007/s10555-008-9126-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1007%2Fs10555-008-9126-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18286234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1Ckur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=179-192&author=G.+Chanauthor=D.+Kalaitzidisauthor=B.+G.+Neel&title=The+tyrosine+phosphatase+Shp2+%28PTPN11%29+in+cancer&doi=10.1007%2Fs10555-008-9126-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The tyrosine phosphatase Shp2 (PTPN11) in cancer</span></div><div class="casAuthors">Chan, Gordon; Kalaitzidis, Demetrios; Neel, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer and Metastasis Reviews</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">179-192</span>CODEN:
                <span class="NLM_cas:coden">CMRED4</span>;
        ISSN:<span class="NLM_cas:issn">0167-7659</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Diverse cellular processes are regulated by tyrosyl phosphorylation, which is controlled by protein-tyrosine kinases (PTKs) and protein-tyrosine phosphatases (PTPs).  De-regulated tyrosyl phosphorylation, evoked by gain-of-function mutations and/or over-expression of PTKs, contributes to the pathogenesis of many cancers and other human diseases.  PTPs, because they oppose the action of PTKs, had been considered to be prime suspects for potential tumor suppressor genes.  Surprisingly, few, if any, tumor suppressor PTPs have been identified.  However, the Src homol.-2 domain-contg. phosphatase Shp2 (encoded by PTPN11) is a bona fide proto-oncogene.  Germline mutations in PTPN11 cause Noonan and LEOPARD syndromes, whereas somatic PTPN11 mutations occur in several types of hematol. malignancies, most notably juvenile myelomonocytic leukemia and, more rarely, in solid tumors.  Shp2 also is an essential component in several other oncogene signaling pathways.  Elucidation of the events underlying Shp2-evoked transformation may provide new insights into oncogenic mechanisms and novel targets for anti-cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbJ8OYsBzvg7Vg90H21EOLACvtfcHk0liHaRNjJg2O6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1Ckur4%253D&md5=fb514f60b97f8ff0dcc0dbd57cab2c9a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2Fs10555-008-9126-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10555-008-9126-y%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DG.%26aulast%3DKalaitzidis%26aufirst%3DD.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26atitle%3DThe%2520tyrosine%2520phosphatase%2520Shp2%2520%2528PTPN11%2529%2520in%2520cancer%26jtitle%3DCancer%2520Metastasis%2520Rev.%26date%3D2008%26volume%3D27%26spage%3D179%26epage%3D192%26doi%3D10.1007%2Fs10555-008-9126-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yomogita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatakeyama, M.</span></span> <span> </span><span class="NLM_article-title">Isolation of a distinct class of gain-of-function SHP-2 mutants with oncogenic RAS-like transforming activity from solid tumors</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3508</span>– <span class="NLM_lpage">3515</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1211019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fsj.onc.1211019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18223690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmslWmurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=3508-3515&author=D.+Miyamotoauthor=M.+Miyamotoauthor=A.+Takahashiauthor=Y.+Yomogitaauthor=H.+Higashiauthor=S.+Kondoauthor=M.+Hatakeyama&title=Isolation+of+a+distinct+class+of+gain-of-function+SHP-2+mutants+with+oncogenic+RAS-like+transforming+activity+from+solid+tumors&doi=10.1038%2Fsj.onc.1211019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of a distinct class of gain-of-function SHP-2 mutants with oncogenic RAS-like transforming activity from solid tumors</span></div><div class="casAuthors">Miyamoto, D.; Miyamoto, M.; Takahashi, A.; Yomogita, Y.; Higashi, H.; Kondo, S.; Hatakeyama, M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">3508-3515</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">SHP-2 protein tyrosine phosphatase plays an important role in activation of the RAS-dependent signaling.  Gain-of-function mutations in the PTPN11 gene, which encodes SHP-2, have been found in the leukemia-prone developmental disorder Noonan syndrome as well as sporadic childhood leukemias, indicating that SHP-2 is a bona fide human oncoprotein.  However, the role of SHP-2 mutations in non-hematol. malignancies remains obscure.  Here, we screened for PTPN11 mutations in primary solid tumors and identified a 1520C>A mutation that causes threonine-507 to lysine (T507K) substitution in the phosphatase domain of SHP-2 in a case of hepatocellular carcinoma.  T507K SHP-2 exhibited altered substrate specificity with slightly elevated basal phosphatase activity.  Upon expression in NIH3T3 cells, T507K SHP-2 induced transformed foci, which was not obsd. with wild type, Noonan-specific or leukemia-specific SHP-2.  Furthermore, NIH3T3 cells transformed by T507K SHP-2 showed anchorage-independent growth and developed tumors in nude mice.  These results indicate that quant. and/or qual. alteration in phosphatase activity dets. the transforming potential as well as target cell/tissue spectrum of individual SHP-2 mutants as oncoproteins.  Although rare in solid tumors, the identified T507K SHP-2 represents a distinct class of SHP-2 mutants with oncogenic RAS-like transforming activity, which could contribute to the development of solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwOMpOPEfD0rVg90H21EOLACvtfcHk0lgmElDJgbebrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmslWmurY%253D&md5=e0a0e2df71b6998a92b40bbba05ffef8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1211019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1211019%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DD.%26aulast%3DMiyamoto%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DA.%26aulast%3DYomogita%26aufirst%3DY.%26aulast%3DHigashi%26aufirst%3DH.%26aulast%3DKondo%26aufirst%3DS.%26aulast%3DHatakeyama%26aufirst%3DM.%26atitle%3DIsolation%2520of%2520a%2520distinct%2520class%2520of%2520gain-of-function%2520SHP-2%2520mutants%2520with%2520oncogenic%2520RAS-like%2520transforming%2520activity%2520from%2520solid%2520tumors%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D3508%26epage%3D3515%26doi%3D10.1038%2Fsj.onc.1211019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aceto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sausgruber, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinkhaus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidatzis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martiny-Baron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzarol, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Confalonieri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quarto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balwierz, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachkov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elledge, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Nimwegen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentires-Alj, M.</span></span> <span> </span><span class="NLM_article-title">Tyrosine phosphatase SHP2 promotes breast cancer progression and maintains tumor-initiating cells via activation of key transcription factors and a positive feedback signaling loop</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1038/nm.2645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fnm.2645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=22388088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1yisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=529-537&author=N.+Acetoauthor=N.+Sausgruberauthor=H.+Brinkhausauthor=D.+Gaidatzisauthor=G.+Martiny-Baronauthor=G.+Mazzarolauthor=S.+Confalonieriauthor=M.+Quartoauthor=G.+Huauthor=P.+J.+Balwierzauthor=M.+Pachkovauthor=S.+J.+Elledgeauthor=E.+van+Nimwegenauthor=M.+B.+Stadlerauthor=M.+Bentires-Alj&title=Tyrosine+phosphatase+SHP2+promotes+breast+cancer+progression+and+maintains+tumor-initiating+cells+via+activation+of+key+transcription+factors+and+a+positive+feedback+signaling+loop&doi=10.1038%2Fnm.2645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine phosphatase SHP2 promotes breast cancer progression and maintains tumor-initiating cells via activation of key transcription factors and a positive feedback signaling loop</span></div><div class="casAuthors">Aceto, Nicola; Sausgruber, Nina; Brinkhaus, Heike; Gaidatzis, Dimos; Martiny-Baron, Georg; Mazzarol, Giovanni; Confalonieri, Stefano; Quarto, Micaela; Hu, Guang; Balwierz, Piotr J.; Pachkov, Mikhail; Elledge, Stephen J.; van Nimwegen, Erik; Stadler, Michael B.; Bentires-Alj, Mohamed</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">529-537</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">New cancer therapies are likely to arise from an in-depth understanding of the signaling networks influencing tumor initiation, progression and metastasis.  We show a fundamental role for Src-homol. 2 domain-contg. phosphatase 2 (SHP2) in these processes in human epidermal growth factor receptor 2 (HER2)-pos. and triple-neg. breast cancers.  Knockdown of SHP2 eradicated breast tumor-initiating cells in xenograft models, and SHP2 depletion also prevented invasion in three-dimensional cultures and in a transductal invasion assay in vivo.  Notably, SHP2 knockdown in established breast tumors blocked their growth and reduced metastasis.  Mechanistically, SHP2 activated stemness-assocd. transcription factors, including v-myc myelocytomatosis viral oncogene homolog (c-Myc) and zinc finger E-box binding homeobox 1 (ZEB1), which resulted in the repression of let-7 microRNA and the expression of a set of 'SHP2 signature' genes.  We found these genes to be simultaneously activated in a large subset of human primary breast tumors that are assocd. with invasive behavior and poor prognosis.  These results provide new insights into the signaling cascades influencing tumor-initiating cells as well as a rationale for targeting SHP2 in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGBsjLtiOIWbVg90H21EOLACvtfcHk0lgmElDJgbebrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1yisrg%253D&md5=ea1d756b2f9d21771ca5f25d2dfda798</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnm.2645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2645%26sid%3Dliteratum%253Aachs%26aulast%3DAceto%26aufirst%3DN.%26aulast%3DSausgruber%26aufirst%3DN.%26aulast%3DBrinkhaus%26aufirst%3DH.%26aulast%3DGaidatzis%26aufirst%3DD.%26aulast%3DMartiny-Baron%26aufirst%3DG.%26aulast%3DMazzarol%26aufirst%3DG.%26aulast%3DConfalonieri%26aufirst%3DS.%26aulast%3DQuarto%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DG.%26aulast%3DBalwierz%26aufirst%3DP.%2BJ.%26aulast%3DPachkov%26aufirst%3DM.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26aulast%3Dvan%2BNimwegen%26aufirst%3DE.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DBentires-Alj%26aufirst%3DM.%26atitle%3DTyrosine%2520phosphatase%2520SHP2%2520promotes%2520breast%2520cancer%2520progression%2520and%2520maintains%2520tumor-initiating%2520cells%2520via%2520activation%2520of%2520key%2520transcription%2520factors%2520and%2520a%2520positive%2520feedback%2520signaling%2520loop%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D529%26epage%3D537%26doi%3D10.1038%2Fnm.2645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanafusa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasunaga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, E.</span></span> <span> </span><span class="NLM_article-title">Sprouty1 and Sprouty2 provide a control mechanism for the Ras/MAPK signalling pathway</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">850</span>– <span class="NLM_lpage">858</span>, <span class="refDoi"> DOI: 10.1038/ncb867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fncb867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=12402043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVKmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=850-858&author=H.+Hanafusaauthor=S.+Toriiauthor=T.+Yasunagaauthor=E.+Nishida&title=Sprouty1+and+Sprouty2+provide+a+control+mechanism+for+the+Ras%2FMAPK+signalling+pathway&doi=10.1038%2Fncb867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Sprouty1 and Sprouty2 provide a control mechanism for the Ras/MAPK signalling pathway</span></div><div class="casAuthors">Hanafusa, Hiroshi; Torii, Satoru; Yasunaga, Takayuki; Nishida, Eisuke</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">850-858</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sprouty (Spry) inhibits signaling by receptor tyrosine kinases; however, the mol. mechanism underlying this function has not been defined.  Here we show that after stimulation by growth factors Spry1 and Spry2 translocate to the plasma membrane and become phosphorylated on a conserved tyrosine.  Next, they bind to the adaptor protein Grb2 and inhibit the recruitment of the Grb2-Sos complex either to the fibroblast growth factor receptor (FGFR) docking adaptor protein FRS2 or to Shp2.  Membrane translocation of Spry is necessary for its phosphorylation, which is essential for its inhibitor activity.  A tyrosine-phosphorylated octapeptide derived from mouse Spry2 inhibits Grb2 from binding FRS2, Shp2 or mouse Spry2 in vitro and blocks activation of the extracellular-signal-regulated kinase (ERK) in cells stimulated by growth factor.  A non-phosphorylated Spry mutant cannot bind Grb2 and acts as a dominant neg., inducing prolonged activation of ERK in response to FGF and promoting the FGF-induced outgrowth of neurites in PC12 cells.  Our findings suggest that Spry functions in a neg. feedback mechanism in which its inhibitor activity is controlled rapidly and reversibly by post-translational mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpLO_810Tlg7Vg90H21EOLACvtfcHk0lgmElDJgbebrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVKmsw%253D%253D&md5=3f496bfc4a46a3e86d6ea1b8b489c04b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fncb867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb867%26sid%3Dliteratum%253Aachs%26aulast%3DHanafusa%26aufirst%3DH.%26aulast%3DTorii%26aufirst%3DS.%26aulast%3DYasunaga%26aufirst%3DT.%26aulast%3DNishida%26aufirst%3DE.%26atitle%3DSprouty1%2520and%2520Sprouty2%2520provide%2520a%2520control%2520mechanism%2520for%2520the%2520Ras%252FMAPK%2520signalling%2520pathway%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2002%26volume%3D4%26spage%3D850%26epage%3D858%26doi%3D10.1038%2Fncb867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agazie, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayman, M. J.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanism for a role of SHP2 in epidermal growth factor receptor signaling</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">7875</span>– <span class="NLM_lpage">7886</span>, <span class="refDoi"> DOI: 10.1128/MCB.23.21.7875-7886.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1128%2FMCB.23.21.7875-7886.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=14560030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosFOgtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=7875-7886&author=Y.+M.+Agazieauthor=M.+J.+Hayman&title=Molecular+mechanism+for+a+role+of+SHP2+in+epidermal+growth+factor+receptor+signaling&doi=10.1128%2FMCB.23.21.7875-7886.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanism for a role of SHP2 in epidermal growth factor receptor signaling</span></div><div class="casAuthors">Agazie, Yehenew M.; Hayman, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7875-7886</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Src homol. 2-contg. phosphotyrosine phosphatase (SHP2) is primarily a pos. effector of receptor tyrosine kinase signaling.  However, the mol. mechanism by which SHP2 effects its biol. function is unknown.  The authors provide evidence that defines the mol. mechanism and site of action of SHP2 in the epidermal growth factor-induced mitogenic pathway.  The authors demonstrate that SHP2 acts upstream of Ras and functions by increasing the half-life of activated Ras (GTP-Ras) in the cell by interfering with the process of Ras inactivation catalyzed by Ras GTPase-activating protein (RasGAP).  It does so by inhibition of tyrosine phosphorylation-dependent translocation of RasGAP to the plasma membrane, to its substrate (GTP-Ras) microdomain.  Inhibition is achieved through the dephosphorylation of RasGAP binding sites at the level of the plasma membrane.  The authors have identified Tyr 992 of the epidermal growth factor receptor (EGFR) to be one such site, since its mutation to Phe renders the EGFR refractory to the effect of dominant-neg. SHP2.  To the authors' knowledge, this is the first report to outline the site and mol. mechanism of action of SHP2 in EGFR signaling, which may also serve as a model to describe its role in other receptor tyrosine kinase signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA5SXQ45HAFrVg90H21EOLACvtfcHk0lhvE2UYW9IXIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosFOgtLo%253D&md5=23e83e2e6ea24b006daa799b15a95981</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.21.7875-7886.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.21.7875-7886.2003%26sid%3Dliteratum%253Aachs%26aulast%3DAgazie%26aufirst%3DY.%2BM.%26aulast%3DHayman%26aufirst%3DM.%2BJ.%26atitle%3DMolecular%2520mechanism%2520for%2520a%2520role%2520of%2520SHP2%2520in%2520epidermal%2520growth%2520factor%2520receptor%2520signaling%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D7875%26epage%3D7886%26doi%3D10.1128%2FMCB.23.21.7875-7886.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, C. K.</span></span> <span> </span><span class="NLM_article-title">Protein tyrosine phosphatases in the JAK/STAT pathway</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4925</span>– <span class="NLM_lpage">4932</span>, <span class="refDoi"> DOI: 10.2741/3051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.2741%2F3051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18508557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=4925-4932&author=D.+Xuauthor=C.+K.+Qu&title=Protein+tyrosine+phosphatases+in+the+JAK%2FSTAT+pathway&doi=10.2741%2F3051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Protein tyrosine phosphatases in the JAK/STAT pathway</span></div><div class="casAuthors">Xu, Dan; Qu, Cheng-Kui</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4925-4932</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  The Janus kinase (JAK)/signal transducer and activator of transcription (STAT) pathway is crucial in controlling cellular activities in response to extracellular cytokines.  Dysfunctions of the JAK/STAT pathway result in various hematopoietic and immune disorders.  The central events in regulating this pathway are tyrosine phosphorylation and dephosphorylation of the signaling components, which are carried out by protein tyrosine kinases and protein tyrosine phosphatases (PTP), resp.  Here, we review recent advances in the regulatory roles of PTPs, in particular, SHP2 phosphatase, in the JAK/STAT signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH1iYtqQyD_7Vg90H21EOLACvtfcHk0lhvE2UYW9IXIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOjtrs%253D&md5=d3f84b6306220659ef34e4bf43cab87e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2741%2F3051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F3051%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DD.%26aulast%3DQu%26aufirst%3DC.%2BK.%26atitle%3DProtein%2520tyrosine%2520phosphatases%2520in%2520the%2520JAK%252FSTAT%2520pathway%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2008%26volume%3D13%26spage%3D4925%26epage%3D4932%26doi%3D10.2741%2F3051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matozaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okazawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, H.</span></span> <span> </span><span class="NLM_article-title">Protein tyrosine phosphatase SHP-2: a proto-oncogene product that promotes Ras activation</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">1786</span>– <span class="NLM_lpage">1793</span>, <span class="refDoi"> DOI: 10.1111/j.1349-7006.2009.01257.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1111%2Fj.1349-7006.2009.01257.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=19622105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht12qs7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2009&pages=1786-1793&author=T.+Matozakiauthor=Y.+Murataauthor=Y.+Saitoauthor=H.+Okazawaauthor=H.+Ohnishi&title=Protein+tyrosine+phosphatase+SHP-2%3A+a+proto-oncogene+product+that+promotes+Ras+activation&doi=10.1111%2Fj.1349-7006.2009.01257.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Protein tyrosine phosphatase SHP-2: A protooncogene product that promotes Ras activation</span></div><div class="casAuthors">Matozaki, Takashi; Murata, Yoji; Saito, Yasuyuki; Okazawa, Hideki; Ohnishi, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1786-1793</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1347-9032</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  SHP-2 is a cytoplasmic protein tyrosine phosphatase (PTP) that contains two Src homol. 2 (SH2) domains.  Although PTPs are generally considered to be neg. regulators on the basis of their ability to oppose the effects of protein tyrosine kinases, SHP-2 is unusual in that it promotes the activation of the Ras-MAPK signaling pathway by receptors for various growth factors and cytokines.  The mol. basis for the activation of SHP-2 is also unique: In the basal state, the NH2-terminal SH2 domain of SHP-2 interacts with the PTP domain, resulting in autoinhibition of PTP activity; the binding of SHP-2 via its SH2 domains to tyrosine-phosphorylated growth factor receptors or docking proteins, however, results in disruption of this intramol. interaction, leading to exposure of the PTP domain and catalytic activation.  Indeed, SHP-2 proteins with artificial mutations in the NH2-terminal SH2 domain have been shown to act as dominant active mutants in vitro.  Such activating mutations of PTPN11 (human SHP-2 gene) were subsequently identified in individuals with Noonan syndrome, a human developmental disorder that is sometimes assocd. with juvenile myelomonocytic leukemia.  Furthermore, somatic mutations of PTPN11 were found to be assocd. with pediatric leukemia.  SHP-2 is also thought to participate in the development of other malignant disorders, but in a manner independent of such activating mutations.  Biochem. and functional studies of SHP-2 and genetic anal. of PTPN11 in human disorders have thus converged to provide new insight into the pathogenesis of cancer as well as potential new targets for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqagV6o3osbrVg90H21EOLACvtfcHk0lhvE2UYW9IXIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht12qs7vI&md5=d4f294641af7b4283f0ed30ce0208bee</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2009.01257.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2009.01257.x%26sid%3Dliteratum%253Aachs%26aulast%3DMatozaki%26aufirst%3DT.%26aulast%3DMurata%26aufirst%3DY.%26aulast%3DSaito%26aufirst%3DY.%26aulast%3DOkazawa%26aufirst%3DH.%26aulast%3DOhnishi%26aufirst%3DH.%26atitle%3DProtein%2520tyrosine%2520phosphatase%2520SHP-2%253A%2520a%2520proto-oncogene%2520product%2520that%2520promotes%2520Ras%2520activation%26jtitle%3DCancer%2520Sci.%26date%3D2009%26volume%3D100%26spage%3D1786%26epage%3D1793%26doi%3D10.1111%2Fj.1349-7006.2009.01257.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heir, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alamsahebpour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raught, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zadeh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohh, M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of SHP2-mediated dephosphorylation of Ras suppresses oncogenesis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8859</span>, <span class="refDoi"> DOI: 10.1038/ncomms9859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fncomms9859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=26617336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWksLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8859&author=S.+Bundaauthor=K.+Burrellauthor=P.+Heirauthor=L.+Zengauthor=A.+Alamsahebpourauthor=Y.+Kanoauthor=B.+Raughtauthor=Z.-Y.+Zhangauthor=G.+Zadehauthor=M.+Ohh&title=Inhibition+of+SHP2-mediated+dephosphorylation+of+Ras+suppresses+oncogenesis&doi=10.1038%2Fncomms9859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of SHP2-mediated dephosphorylation of Ras suppresses oncogenesis</span></div><div class="casAuthors">Bunda, Severa; Burrell, Kelly; Heir, Pardeep; Zeng, Lifan; Alamsahebpour, Amir; Kano, Yoshihito; Raught, Brian; Zhang, Zhong-Yin; Zadeh, Gelareh; Ohh, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8859</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ras is phosphorylated on a conserved tyrosine at position 32 within the switch I region via Src kinase.  This phosphorylation inhibits the binding of effector Raf while promoting the engagement of GTPase-activating protein (GAP) and GTP hydrolysis.  Here we identify SHP2 as the ubiquitously expressed tyrosine phosphatase that preferentially binds to and dephosphorylates Ras to increase its assocn. with Raf and activate downstream proliferative Ras/ERK/MAPK signalling.  In comparison to normal astrocytes, SHP2 activity is elevated in astrocytes isolated from glioblastoma multiforme (GBM)-prone H-Ras(12V) knock-in mice as well as in glioma cell lines and patient-derived GBM specimens exhibiting hyperactive Ras.  Pharmacol. inhibition of SHP2 activity attenuates cell proliferation, soft-agar colony formation and orthotopic GBM growth in NOD/SCID mice and decelerates the progression of low-grade astrocytoma to GBM in a spontaneous transgenic glioma mouse model.  These results identify SHP2 as a direct activator of Ras and a potential therapeutic target for cancers driven by a previously 'undruggable' oncogenic or hyperactive Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhWi25AJiphrVg90H21EOLACvtfcHk0liIdHC7XxkxjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWksLbP&md5=8524ad9cc6f2e85abaff6b07c698f464</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fncomms9859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9859%26sid%3Dliteratum%253Aachs%26aulast%3DBunda%26aufirst%3DS.%26aulast%3DBurrell%26aufirst%3DK.%26aulast%3DHeir%26aufirst%3DP.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DAlamsahebpour%26aufirst%3DA.%26aulast%3DKano%26aufirst%3DY.%26aulast%3DRaught%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DZ.-Y.%26aulast%3DZadeh%26aufirst%3DG.%26aulast%3DOhh%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520SHP2-mediated%2520dephosphorylation%2520of%2520Ras%2520suppresses%2520oncogenesis%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8859%26doi%3D10.1038%2Fncomms9859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chemnitz, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichols, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">June, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, J. L.</span></span> <span> </span><span class="NLM_article-title">SHP-1 and SHP-2 associate with immunoreceptor tyrosine-based switch motif of programmed death 1 upon primary human T cell stimulation, but only receptor ligation prevents T cell activation</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">945</span>– <span class="NLM_lpage">954</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.173.2.945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.4049%2Fjimmunol.173.2.945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=15240681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlsVGhurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2004&pages=945-954&author=J.+M.+Chemnitzauthor=R.+V.+Parryauthor=K.+E.+Nicholsauthor=C.+H.+Juneauthor=J.+L.+Riley&title=SHP-1+and+SHP-2+associate+with+immunoreceptor+tyrosine-based+switch+motif+of+programmed+death+1+upon+primary+human+T+cell+stimulation%2C+but+only+receptor+ligation+prevents+T+cell+activation&doi=10.4049%2Fjimmunol.173.2.945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">SHP-1 and SHP-2 Associate with Immunoreceptor Tyrosine-Based Switch Motif of Programmed Death 1 upon Primary Human T Cell Stimulation, but Only Receptor Ligation Prevents T Cell Activation</span></div><div class="casAuthors">Chemnitz, Jens M.; Parry, Richard V.; Nichols, Kim E.; June, Carl H.; Riley, James L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">945-954</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">To study the cis- and trans-acting factors that mediate programmed death 1 (PD-1) signaling in primary human CD4 T cells, the authors constructed a chimeric mol. consisting of the murine CD28 extracellular domain and human PD-1 cytoplasmic tail.  When introduced into CD4 T cells, this construct mimics the activity of endogenous PD-1 in terms of its ability to suppress T cell expansion and cytokine prodn.  The cytoplasmic tail of PD-1 contains two structural motifs, an ITIM and an immunoreceptor tyrosine-based switch motif (ITSM).  Mutation of the ITIM had little effect on PD-1 signaling or functional activity.  In contrast, mutation of the ITSM abrogated the ability of PD-1 to block cytokine synthesis and to limit T cell expansion.  Further biochem. analyses revealed that the ability of PD-1 to block T cell activation correlated with recruitment of Src homol. region 2 domain-contg. phosphatase-1 (SHP-1) and SHP-2, and not the adaptor Src homol. 2 domain-contg. mol. 1A, to the ITSM domain.  In TCR-stimulated T cells, SHP-2 assocd. with PD-1, even in the absence of PD-1 engagement.  Despite this interaction, the ability of PD-1 to block T cell activation required receptor ligation, suggesting that colocalization of PD-1 with CD3 and/or CD28 may be necessary for inhibition of T cell activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLiPMfGJ1GeLVg90H21EOLACvtfcHk0liIdHC7XxkxjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlsVGhurc%253D&md5=204c373c1a728fc94c10db9de7804f50</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.173.2.945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.173.2.945%26sid%3Dliteratum%253Aachs%26aulast%3DChemnitz%26aufirst%3DJ.%2BM.%26aulast%3DParry%26aufirst%3DR.%2BV.%26aulast%3DNichols%26aufirst%3DK.%2BE.%26aulast%3DJune%26aufirst%3DC.%2BH.%26aulast%3DRiley%26aufirst%3DJ.%2BL.%26atitle%3DSHP-1%2520and%2520SHP-2%2520associate%2520with%2520immunoreceptor%2520tyrosine-based%2520switch%2520motif%2520of%2520programmed%2520death%25201%2520upon%2520primary%2520human%2520T%2520cell%2520stimulation%252C%2520but%2520only%2520receptor%2520ligation%2520prevents%2520T%2520cell%2520activation%26jtitle%3DJ.%2520Immunol.%26date%3D2004%26volume%3D173%26spage%3D945%26epage%3D954%26doi%3D10.4049%2Fjimmunol.173.2.945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jie, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildener-Leapman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trivedi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferris, R. L.</span></span> <span> </span><span class="NLM_article-title">PD-1/SHP-2 inhibits Tc1/Th1 phenotypic responses and the activation of T cells in the tumor microenvironment</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">508</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-14-1215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1158%2F0008-5472.CAN-14-1215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=25480946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVenurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=508-518&author=J.+Liauthor=H.+B.+Jieauthor=Y.+Leiauthor=N.+Gildener-Leapmanauthor=S.+Trivediauthor=T.+Greenauthor=L.+P.+Kaneauthor=R.+L.+Ferris&title=PD-1%2FSHP-2+inhibits+Tc1%2FTh1+phenotypic+responses+and+the+activation+of+T+cells+in+the+tumor+microenvironment&doi=10.1158%2F0008-5472.CAN-14-1215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">PD-1/SHP-2 Inhibits Tc1/Th1 Phenotypic Responses and the Activation of T Cells in the Tumor Microenvironment</span></div><div class="casAuthors">Li, Jing; Jie, Hyun-Bae; Lei, Yu; Gildener-Leapman, Neil; Trivedi, Sumita; Green, Tony; Kane, Lawrence P.; Ferris, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">508-518</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">This study addresses the mechanisms through which PD-1 antibody therapy restores the ability of the immune system to recognize and attack tumors by skewing intratumoral T cells toward a more potent activation phenotype that is assocd. with enhanced signaling and a reversal of their immunosuppressed phenotype.  Immune rejection of tumors is mediated by IFNγ prodn. and T-cell cytolytic activity.  These processes are impeded by PD-1, a coinhibitory mol. expressed on T cells that is elevated in tumor-infiltrating lymphocytes (TIL).  PD-1 elevation may reflect T-cell exhaustion marked by decreased proliferation, prodn. of type I cytokines, and poor cytolytic activity.  Although anti-PD-1 antibodies enhance IFNγ secretion after stimulation of the T-cell receptor (TCR), the mechanistic link between PD-1 and its effects on T-cell help (Tc1/Th1 skewing) remains unclear.  In prospectively collected cancer tissues, we found that TIL exhibited dampened Tc1/Th1 skewing and activation compared with peripheral blood lymphocytes (PBL).  When PD-1 bound its ligand PD-L1, we obsd. a marked suppression of crit. TCR target genes and Th1 cytokines.  Conversely, PD-1 blockade reversed these suppressive effects of PD-1:PD-L1 ligation.  We also found that the TCR-regulated phosphatase SHP-2 was expressed higher in TIL than in PBL, tightly correlating with PD-1 expression and neg. regulation of TCR target genes.  Overall, these results defined a PD-1/SHP-2/STAT1/T-bet signaling axis mediating the suppressive effects of PD-1 on Th1 immunity at tumor sites.  Our findings argue that PD-1 or SHP-2 blockade will be sufficient to restore robust Th1 immunity and T-cell activation and thereby reverse immunosuppression in the tumor microenvironment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe6jAWHNatvLVg90H21EOLACvtfcHk0liIdHC7XxkxjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVenurY%253D&md5=695007f0715b5eee2b7e47deaf87ae23</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-1215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-1215%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJie%26aufirst%3DH.%2BB.%26aulast%3DLei%26aufirst%3DY.%26aulast%3DGildener-Leapman%26aufirst%3DN.%26aulast%3DTrivedi%26aufirst%3DS.%26aulast%3DGreen%26aufirst%3DT.%26aulast%3DKane%26aufirst%3DL.%2BP.%26aulast%3DFerris%26aufirst%3DR.%2BL.%26atitle%3DPD-1%252FSHP-2%2520inhibits%2520Tc1%252FTh1%2520phenotypic%2520responses%2520and%2520the%2520activation%2520of%2520T%2520cells%2520in%2520the%2520tumor%2520microenvironment%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D508%26epage%3D518%26doi%3D10.1158%2F0008-5472.CAN-14-1215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hui, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallweber, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasmal, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vale, R. D.</span></span> <span> </span><span class="NLM_article-title">T cell costimulatory receptor CD28 is a primary target for PD-1-mediated inhibition</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1428</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.1126/science.aaf1292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1126%2Fscience.aaf1292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=28280247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsVWhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1428-1433&author=E.+Huiauthor=J.+Cheungauthor=J.+Zhuauthor=X.+Suauthor=M.+J.+Taylorauthor=H.+A.+Wallweberauthor=D.+K.+Sasmalauthor=J.+Huangauthor=J.+M.+Kimauthor=I.+Mellmanauthor=R.+D.+Vale&title=T+cell+costimulatory+receptor+CD28+is+a+primary+target+for+PD-1-mediated+inhibition&doi=10.1126%2Fscience.aaf1292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">T cell costimulatory receptor CD28 is a primary target for PD-1-mediated inhibition</span></div><div class="casAuthors">Hui, Enfu; Cheung, Jeanne; Zhu, Jing; Su, Xiaolei; Taylor, Marcus J.; Wallweber, Heidi A.; Sasmal, Dibyendu K.; Huang, Jun; Kim, Jeong M.; Mellman, Ira; Vale, Ronald D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6332</span>),
    <span class="NLM_cas:pages">1428-1433</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">PD-1-targeted therapies have been a breakthrough for treating certain tumors and can rejuvenate T cells to unleash the anticancer immune response (see the Perspective by Clouthier and Ohashi).  It is widely believed that PD-1 suppresses signaling through the T cell receptor (TCR).  However, Hui et al. find instead that the TCR costimulatory receptor, CD28, is the primary target of PD-1 signaling.  Independently, Kamphorst et al. show that CD28 is required for PD-1 therapies to kill cancer cells efficiently and eliminate chronic viral infections in mice.  Lung cancer patients that responded to PD-1 therapy had more CD28+ T cells, which suggests that CD28 may predict treatment response.  Science, ibid p. 1428, p. 1423; see also p. 1373.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM_s4cf1lkxrVg90H21EOLACvtfcHk0lg6v1o1DtWjlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsVWhu7g%253D&md5=69750b99b68316ee035a0b7eeddfb9fa</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf1292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf1292%26sid%3Dliteratum%253Aachs%26aulast%3DHui%26aufirst%3DE.%26aulast%3DCheung%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DWallweber%26aufirst%3DH.%2BA.%26aulast%3DSasmal%26aufirst%3DD.%2BK.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%2BM.%26aulast%3DMellman%26aufirst%3DI.%26aulast%3DVale%26aufirst%3DR.%2BD.%26atitle%3DT%2520cell%2520costimulatory%2520receptor%2520CD28%2520is%2520a%2520primary%2520target%2520for%2520PD-1-mediated%2520inhibition%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1428%26epage%3D1433%26doi%3D10.1126%2Fscience.aaf1292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span> <span> </span><span class="NLM_article-title">Targeting protein tyrosine phosphatases for anticancer drug discovery</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1862</span>, <span class="refDoi"> DOI: 10.2174/138161210791209027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.2174%2F138161210791209027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=20337577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC3cXot1ClsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=1843-1862&author=L.+M.+Scottauthor=H.+R.+Lawrenceauthor=S.+M.+Sebtiauthor=N.+J.+Lawrenceauthor=J.+Wu&title=Targeting+protein+tyrosine+phosphatases+for+anticancer+drug+discovery&doi=10.2174%2F138161210791209027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein tyrosine phosphatases for anticancer drug discovery</span></div><div class="casAuthors">Scott, Latanya M.; Lawrence, Harshani R.; Sebti, Said M.; Lawrence, Nicholas J.; Wu, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1843-1862</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein tyrosine phosphatases (PTPs) are a diverse family of enzymes encoded by 107 genes in the human genome.  Together with protein tyrosine kinases (PTKs), PTPs regulate various cellular activities essential for the initiation and maintenance of malignant phenotypes.  While PTK inhibitors are now used routinely for cancer treatment, the PTP inhibitor development field is still in the discovery phase.  In this article, the suitability of targeting PTPs for novel anticancer drug discovery is discussed.  Examples are presented for PTPs that have been targeted for anticancer drug discovery as well as potential new PTP targets for novel anticancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHuBrNeSP0n7Vg90H21EOLACvtfcHk0lg6v1o1DtWjlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXot1ClsLY%253D&md5=17813d1fba0225fd5e54692b1159bc4a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F138161210791209027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161210791209027%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DL.%2BM.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%26atitle%3DTargeting%2520protein%2520tyrosine%2520phosphatases%2520for%2520anticancer%2520drug%2520discovery%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2010%26volume%3D16%26spage%3D1843%26epage%3D1862%26doi%3D10.2174%2F138161210791209027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butterworth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overduin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, A. J.</span></span> <span> </span><span class="NLM_article-title">Targeting protein tyrosine phosphatase SHP2 for therapeutic intervention</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1423</span>– <span class="NLM_lpage">1437</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.4155%2Ffmc.14.88" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=25329198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOhu7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1423-1437&author=S.+Butterworthauthor=M.+Overduinauthor=A.+J.+Barr&title=Targeting+protein+tyrosine+phosphatase+SHP2+for+therapeutic+intervention&doi=10.4155%2Ffmc.14.88"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein tyrosine phosphatase SHP2 for therapeutic intervention</span></div><div class="casAuthors">Butterworth, Sam; Overduin, Michael; Barr, Alastair J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1423-1437</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein tyrosine phosphatases have been the focus of considerable research efforts aimed at developing novel therapeutics; however, these targets are often characterized as being 'undruggable' due to the challenge of achieving selectivity, potency and cell permeability.  More recently, there has been renewed interest in developing inhibitors of the tyrosine phosphatase SHP2 (PTPN11) in the light of its broad role in cancer, specifically juvenile myelomonocytic leukemia, and recent studies that implicate SHP2 as a key factor in breast cancer progression.  Recent significant advances in the field of SHP2 inhibitor development raise the question: are we on the verge of a new era of protein tyrosine phosphatase-directed therapeutics This article critically appraises recent developments, assesses ongoing challenges and presents a perspective on possible future directions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEVdQ5GiIy9LVg90H21EOLACvtfcHk0lg6v1o1DtWjlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOhu7jO&md5=85c071c906c6c047d524460df5e0a9b5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.88%26sid%3Dliteratum%253Aachs%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DOverduin%26aufirst%3DM.%26aulast%3DBarr%26aufirst%3DA.%2BJ.%26atitle%3DTargeting%2520protein%2520tyrosine%2520phosphatase%2520SHP2%2520for%2520therapeutic%2520intervention%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D1423%26epage%3D1437%26doi%3D10.4155%2Ffmc.14.88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel shp2 protein tyrosine phosphatase inhibitor</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">562</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1124/mol.106.025536</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1124%2Fmol.106.025536" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=16717135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnslyjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2006&pages=562-570&author=L.+Chenauthor=S.+S.+Sungauthor=M.+L.+Yipauthor=H.+R.+Lawrenceauthor=Y.+Renauthor=W.+C.+Guidaauthor=S.+M.+Sebtiauthor=N.+J.+Lawrenceauthor=J.+Wu&title=Discovery+of+a+novel+shp2+protein+tyrosine+phosphatase+inhibitor&doi=10.1124%2Fmol.106.025536"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel Shp2 protein tyrosine phosphatase inhibitor</span></div><div class="casAuthors">Chen, Liwei; Sung, Shen-Shu; Yip, M. L. Richard; Lawrence, Harshani R.; Ren, Yuan; Guida, Wayne C.; Sebti, Said M.; Lawrence, Nicholas J.; Wu, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">562-570</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Shp2 is a non-receptor protein tyrosine phosphatase (PTP) encoded by the PTPN11 gene.  It is involved in growth factor-induced activation of mitogen-activated protein (MAP) kinases Erk1 and Erk2 (Erk1/2) and has been implicated in the pathogenicity of the oncogenic bacterium Helicobacter pylori.  Moreover, gain-of-function Shp2 mutations have been found in childhood leukemias and Noonan syndrome.  Thus, small mol. Shp2 PTP inhibitors are much needed reagents for evaluation of Shp2 as a therapeutic target and for chem. biol. studies of Shp2 function.  By screening the National Cancer Institute (NCI) Diversity Set chem. library, we identified 8-hydroxy-7-(6-sulfonaphthalen-2-yl)diazenyl-quinoline-5-sulfonic acid (NSC-87877) as a potent Shp2 PTP inhibitor.  Mol. modeling and site-directed mutagenesis studies suggested that NSC-87877 binds to the catalytic cleft of Shp2 PTP.  NSC-87877 cross-inhibited Shp1 in vitro, but it was selective for Shp2 over other PTPs (PTP1B, HePTP, DEP1, CD45, and LAR).  It is noteworthy that NSC-87877 inhibited epidermal growth factor (EGF)-induced activation of Shp2 PTP, Ras, and Erk1/2 in cell cultures but did not block EGF-induced Gab1 tyrosine phosphorylation or Gab1-Shp2 assocn.  Furthermore, NSC-87877 inhibited Erk1/2 activation by a Gab1-Shp2 chimera but did not affect the Shp2-independent Erk1/2 activation by phorbol 12-myristate 13-acetate.  These results identified NSC-87877 as the first PTP inhibitor capable of inhibiting Shp2 PTP in cell cultures without a detectable off-target effect.  Our study also provides the first pharmacol. evidence that Shp2 mediates EGF-induced Erk1/2 MAP kinase activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA4XVGUjw3W7Vg90H21EOLACvtfcHk0ljog1ycitvTjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnslyjsrk%253D&md5=a2ce03150ecbe6937876bb292ced30dc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fmol.106.025536&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.106.025536%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSung%26aufirst%3DS.%2BS.%26aulast%3DYip%26aufirst%3DM.%2BL.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520shp2%2520protein%2520tyrosine%2520phosphatase%2520inhibitor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2006%26volume%3D70%26spage%3D562%26epage%3D570%26doi%3D10.1124%2Fmol.106.025536" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hellmuth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosskopf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Würtele, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röder, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kries, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosario, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rademann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birchmeier, W.</span></span> <span> </span><span class="NLM_article-title">Specific inhibitors of the protein tyrosine phosphatase Shp2 identified by high-throughput docking</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">7275</span>– <span class="NLM_lpage">7280</span>, <span class="refDoi"> DOI: 10.1073/pnas.0710468105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1073%2Fpnas.0710468105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18480264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsFCqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=7275-7280&author=K.+Hellmuthauthor=S.+Grosskopfauthor=C.+T.+Lumauthor=M.+W%C3%BCrteleauthor=N.+R%C3%B6derauthor=J.+P.+von+Kriesauthor=M.+Rosarioauthor=J.+Rademannauthor=W.+Birchmeier&title=Specific+inhibitors+of+the+protein+tyrosine+phosphatase+Shp2+identified+by+high-throughput+docking&doi=10.1073%2Fpnas.0710468105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Specific inhibitors of the protein tyrosine phosphatase Shp2 identified by high-throughput docking</span></div><div class="casAuthors">Hellmuth, Klaus; Grosskopf, Stefanie; Lum, Ching Tung; Wuertele, Martin; Roeder, Nadine; von Kries, Jens Peter; Rosario, Marta; Rademann, Jorg; Birchmeier, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7275-7280</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The protein tyrosine phosphatase Shp2 is a pos. regulator of growth factor signaling.  Gain-of-function mutations in several types of leukemia define Shp2 as a bona fide oncogene.  We performed a high-throughput in silico screen for small-mol.-wt. compds. that bind the catalytic site of Shp2.  We have identified the phenylhydrazonopyrazolone sulfonate PHPS1 as a potent and cell-permeable inhibitor, which is specific for Shp2 over the closely related tyrosine phosphatases Shp1 and PTP1 B.  PHPS1 inhibits Shp2-dependent cellular events such as hepatocyte growth factor/scatter factor (HGF/SF)-induced epithelial cell scattering and branching morphogenesis.  PHPS1 also blocks Shp2-dependent downstream signaling, namely HGF/SF-induced sustained phosphorylation of the Erk1/2 MAP kinases and dephosphorylation of paxillin.  Furthermore, PHPS1 efficiently inhibits activation of Erk1/2 by the leukemia-assocd. Shp2 mutant, Shp2-E76K, and blocks the anchorage-independent growth of a variety of human tumor cell lines.  The PHPS compd. class is therefore suitable for further development of therapeutics for the treatment of Shp2-dependent diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGponh1kw_h7jrVg90H21EOLACvtfcHk0ljog1ycitvTjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsFCqsLs%253D&md5=39c7dda94e32134c305e135e4c48b9d1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0710468105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0710468105%26sid%3Dliteratum%253Aachs%26aulast%3DHellmuth%26aufirst%3DK.%26aulast%3DGrosskopf%26aufirst%3DS.%26aulast%3DLum%26aufirst%3DC.%2BT.%26aulast%3DW%25C3%25BCrtele%26aufirst%3DM.%26aulast%3DR%25C3%25B6der%26aufirst%3DN.%26aulast%3Dvon%2BKries%26aufirst%3DJ.%2BP.%26aulast%3DRosario%26aufirst%3DM.%26aulast%3DRademann%26aufirst%3DJ.%26aulast%3DBirchmeier%26aufirst%3DW.%26atitle%3DSpecific%2520inhibitors%2520of%2520the%2520protein%2520tyrosine%2520phosphatase%2520Shp2%2520identified%2520by%2520high-throughput%2520docking%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D7275%26epage%3D7280%26doi%3D10.1073%2Fpnas.0710468105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pireddu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szymanski, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, M. L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Src homology-2 domain containing protein tyrosine phosphatase-2 (Shp2) based on oxindole scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4948</span>– <span class="NLM_lpage">4956</span>, <span class="refDoi"> DOI: 10.1021/jm8002526</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8002526" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptl2hu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4948-4956&author=H.+R.+Lawrenceauthor=R.+Piredduauthor=L.+Chenauthor=Y.+Luoauthor=S.-S.+Sungauthor=A.+M.+Szymanskiauthor=M.+L.+R.+Yipauthor=W.+C.+Guidaauthor=S.+M.+Sebtiauthor=J.+Wuauthor=N.+J.+Lawrence&title=Inhibitors+of+Src+homology-2+domain+containing+protein+tyrosine+phosphatase-2+%28Shp2%29+based+on+oxindole+scaffolds&doi=10.1021%2Fjm8002526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Src Homology-2 Domain Containing Protein Tyrosine Phosphatase-2 (Shp2) Based on Oxindole Scaffolds</span></div><div class="casAuthors">Lawrence, Harshani R.; Pireddu, Roberta; Chen, Liwei; Luo, Yunting; Sung, Shen-Shu; Szymanski, Ann Marie; Yip, M. L. Richard; Guida, Wayne C.; Sebti, Said M.; Wu, Jie; Lawrence, Nicholas J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4948-4956</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening of the NCI diversity set of compds. has led to the identification of NSC-117199, which inhibits the protein tyrosine phosphatase (PTP) Shp2 with an IC50 of 47 μM.  A focused library incorporating an isatin scaffold was designed and evaluated for inhibition of Shp2 and Shp1 PTP activities.  Several compds. were identified that selectively inhibit Shp2 over Shp1 and PTP1B with low to submicromolar activity.  A model for the binding of the active compds. is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRqLAauyGNBbVg90H21EOLACvtfcHk0lj952XJAlur6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptl2hu7o%253D&md5=572227c5a8cfffb108388616bd36fdfe</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm8002526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8002526%26sid%3Dliteratum%253Aachs%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DPireddu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DSung%26aufirst%3DS.-S.%26aulast%3DSzymanski%26aufirst%3DA.%2BM.%26aulast%3DYip%26aufirst%3DM.%2BL.%2BR.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26atitle%3DInhibitors%2520of%2520Src%2520homology-2%2520domain%2520containing%2520protein%2520tyrosine%2520phosphatase-2%2520%2528Shp2%2529%2520based%2520on%2520oxindole%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4948%26epage%3D4956%26doi%3D10.1021%2Fjm8002526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabinger, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Salicylic acid based small molecule inhibitor for the oncogenic Src homology-2 domain containing protein tyrosine phosphatase-2 (SHP2)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2482</span>– <span class="NLM_lpage">2493</span>, <span class="refDoi"> DOI: 10.1021/jm901645u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901645u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFyhs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2482-2493&author=X.+Zhangauthor=Y.+Heauthor=S.+Liuauthor=Z.+Yuauthor=Z.+X.+Jiangauthor=Z.+Yangauthor=Y.+Dongauthor=S.+C.+Nabingerauthor=L.+Wuauthor=A.+M.+Gunawanauthor=L.+Wangauthor=R.+J.+Chanauthor=Z.+Y.+Zhang&title=Salicylic+acid+based+small+molecule+inhibitor+for+the+oncogenic+Src+homology-2+domain+containing+protein+tyrosine+phosphatase-2+%28SHP2%29&doi=10.1021%2Fjm901645u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Salicylic Acid Based Small Molecule Inhibitor for the Oncogenic Src Homology-2 Domain Containing Protein Tyrosine Phosphatase-2 (SHP2)</span></div><div class="casAuthors">Zhang, Xian; He, Yantao; Liu, Sijiu; Yu, Zhihong; Jiang, Zhong-Xing; Yang, Zhenyun; Dong, Yuanshu; Nabinger, Sarah C.; Wu, Li; Gunawan, Andrea M.; Wang, Lina; Chan, Rebecca J.; Zhang, Zhong-Yin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2482-2493</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Src homol.-2 domain contg. protein tyrosine phosphatase-2 (SHP2) plays a pivotal role in growth factor and cytokine signaling.  Gain-of-function SHP2 mutations are assocd. with Noonan syndrome, various kinds of leukemias, and solid tumors.  Thus, there is considerable interest in SHP2 as a potential target for anticancer and antileukemia therapy.  We report a salicylic acid based combinatorial library approach aimed at binding both active site and unique nearby subpockets for enhanced affinity and selectivity.  Screening of the library led to the identification of a SHP2 inhibitor II-B08 (compd. 9) with highly efficacious cellular activity.  Compd. 9 blocks growth factor stimulated ERK1/2 activation and hematopoietic progenitor proliferation, providing supporting evidence that chem. inhibition of SHP2 may be therapeutically useful for anticancer and antileukemia treatment.  X-ray crystallog. anal. of the structure of SHP2 in complex with 9 reveals mol. determinants that can be exploited for the acquisition of more potent and selective SHP2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC82T5JpCRArVg90H21EOLACvtfcHk0lj952XJAlur6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFyhs7k%253D&md5=6f87897eda38ea485dc08e189f1b5ca8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm901645u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901645u%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DZ.%2BX.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DNabinger%26aufirst%3DS.%2BC.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DGunawan%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChan%26aufirst%3DR.%2BJ.%26aulast%3DZhang%26aufirst%3DZ.%2BY.%26atitle%3DSalicylic%2520acid%2520based%2520small%2520molecule%2520inhibitor%2520for%2520the%2520oncogenic%2520Src%2520homology-2%2520domain%2520containing%2520protein%2520tyrosine%2520phosphatase-2%2520%2528SHP2%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2482%26epage%3D2493%26doi%3D10.1021%2Fjm901645u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, C.-K.</span></span> <span> </span><span class="NLM_article-title">Identification of cryptotanshinone as an inhibitor of oncogenic protein tyrosine phosphatase SHP2 (PTPN11)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7212</span>– <span class="NLM_lpage">7221</span>, <span class="refDoi"> DOI: 10.1021/jm400474r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400474r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7212-7221&author=W.+Liuauthor=B.+Yuauthor=G.+Xuauthor=W.-R.+Xuauthor=M.+L.+Lohauthor=L.-D.+Tangauthor=C.-K.+Qu&title=Identification+of+cryptotanshinone+as+an+inhibitor+of+oncogenic+protein+tyrosine+phosphatase+SHP2+%28PTPN11%29&doi=10.1021%2Fjm400474r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Cryptotanshinone as an Inhibitor of Oncogenic Protein Tyrosine Phosphatase SHP2 (PTPN11)</span></div><div class="casAuthors">Liu, Wei; Yu, Bing; Xu, Gang; Xu, Wei-Ren; Loh, Mignon L.; Tang, Li-Da; Qu, Cheng-Kui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7212-7221</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activating mutations of PTPN11 (encoding the SHP2 phosphatase) are assocd. with Noonan syndrome, childhood leukemias, and sporadic solid tumors.  Virtual screening combined with exptl. assays was performed to identify inhibitors of SHP2 from a database of natural products.  This effort led to the identification of cryptotanshinone as an inhibitor of SHP2.  Cryptotanshinone inhibited SHP2 with an IC50 of 22.50 μM.  Fluorescence titrn. expts. confirmed that it directly bound to SHP2.  Enzymic kinetic analyses showed that cryptotanshinone was a mixed-type and irreversible inhibitor.  This drug was further verified for its ability to block SHP2-mediated cell signaling and cellular functions.  Furthermore, mouse myeloid progenitors and patient leukemic cells with the activating mutation E76K in PTPN11 were found to be sensitive to this inhibitor.  Since cryptotanshinone is used to treat cardiovascular diseases in Asian countries, this drug has a potential to be used directly or to be further developed to treat PTPN11-assocd. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJYelS47QYHLVg90H21EOLACvtfcHk0lgF-aEovWsH-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurvN&md5=3bbbae68159ae4227bbf74d5573ca77a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm400474r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400474r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DW.-R.%26aulast%3DLoh%26aufirst%3DM.%2BL.%26aulast%3DTang%26aufirst%3DL.-D.%26aulast%3DQu%26aufirst%3DC.-K.%26atitle%3DIdentification%2520of%2520cryptotanshinone%2520as%2520an%2520inhibitor%2520of%2520oncogenic%2520protein%2520tyrosine%2520phosphatase%2520SHP2%2520%2528PTPN11%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7212%26epage%3D7221%26doi%3D10.1021%2Fjm400474r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mali, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.-Y.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of targeting the oncogenic SHP2 phosphatase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6594</span>– <span class="NLM_lpage">6609</span>, <span class="refDoi"> DOI: 10.1021/jm5006176</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006176" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFajtrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6594-6609&author=L.-F.+Zengauthor=R.-Y.+Zhangauthor=Z.-H.+Yuauthor=S.+Liauthor=L.+Wuauthor=A.+M.+Gunawanauthor=B.+S.+Laneauthor=R.+S.+Maliauthor=X.+Liauthor=R.+J.+Chanauthor=R.+Kapurauthor=C.+D.+Wellsauthor=Z.-Y.+Zhang&title=Therapeutic+potential+of+targeting+the+oncogenic+SHP2+phosphatase&doi=10.1021%2Fjm5006176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Potential of Targeting the Oncogenic SHP2 Phosphatase</span></div><div class="casAuthors">Zeng, Li-Fan; Zhang, Ruo-Yu; Yu, Zhi-Hong; Li, Sijiu; Wu, Li; Gunawan, Andrea M.; Lane, Brandon S.; Mali, Raghuveer S.; Li, Xingjun; Chan, Rebecca J.; Kapur, Reuben; Wells, Clark D.; Zhang, Zhong-Yin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6594-6609</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Src homol. 2 domain contg. protein tyrosine phosphatase-2 (SHP2) is an oncogenic phosphatase assocd. with various kinds of leukemia and solid tumors.  Thus, there is substantial interest in developing SHP2 inhibitors as potential anticancer and antileukemia agents.  Using a structure-guided and fragment-based library approach, we identified a novel hydroxyindole carboxylic acid-based SHP2 inhibitor (I), with an IC50 value of 200 nM and greater than 5-fold selectivity against 20 mammalian PTPs.  Structural and modeling studies reveal that the hydroxyindole carboxylic acid anchors the inhibitor to the SHP2 active site, while interactions of the oxalamide linker and the phenylthiophene tail with residues in the β5-β6 loop contribute to I's binding potency and selectivity.  Evidence suggests that I specifically attenuates the SHP2-dependent signaling inside the cell.  Moreover, I blocks growth factor mediated Erk1/2 and Akt activation and exhibits excellent antiproliferative activity in lung cancer and breast cancer as well as leukemia cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3ko-q4-PSXbVg90H21EOLACvtfcHk0lgF-aEovWsH-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFajtrvO&md5=ac8e5359387e614782c95c02f32fd97d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm5006176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006176%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.-F.%26aulast%3DZhang%26aufirst%3DR.-Y.%26aulast%3DYu%26aufirst%3DZ.-H.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DGunawan%26aufirst%3DA.%2BM.%26aulast%3DLane%26aufirst%3DB.%2BS.%26aulast%3DMali%26aufirst%3DR.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DR.%2BJ.%26aulast%3DKapur%26aufirst%3DR.%26aulast%3DWells%26aufirst%3DC.%2BD.%26aulast%3DZhang%26aufirst%3DZ.-Y.%26atitle%3DTherapeutic%2520potential%2520of%2520targeting%2520the%2520oncogenic%2520SHP2%2520phosphatase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6594%26epage%3D6609%26doi%3D10.1021%2Fjm5006176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chio, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, A. C.</span></span> <span> </span><span class="NLM_article-title">Targeting a cryptic allosteric site for selective inhibition of the oncogenic protein tyrosine phosphatase Shp2</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">504</span>, <span class="refDoi"> DOI: 10.1021/bi5013595</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi5013595" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrs7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=497-504&author=C.+M.+Chioauthor=C.+S.+Limauthor=A.+C.+Bishop&title=Targeting+a+cryptic+allosteric+site+for+selective+inhibition+of+the+oncogenic+protein+tyrosine+phosphatase+Shp2&doi=10.1021%2Fbi5013595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting a Cryptic Allosteric Site for Selective Inhibition of the Oncogenic Protein Tyrosine Phosphatase Shp2</span></div><div class="casAuthors">Chio, Cynthia M.; Lim, Christopher S.; Bishop, Anthony C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">497-504</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein tyrosine phosphatases (PTPs) have been the subject of considerable pharmaceutical-design efforts because of the ubiquitous connections between misregulation of PTP activity and human disease.  PTP-inhibitor discovery has been hampered, however, by the difficulty in identifying cell-permeable compds. that can selectively target PTP active sites, and no PTP inhibitors have progressed to the clinic.  The identification of allosteric sites on target PTPs therefore represents a potentially attractive soln. to the druggability problem of PTPs.  Here we report that the oncogenic PTP Shp2 contains an allosteric-inhibition site that renders the enzyme sensitive to potent and selective inhibition by cell-permeable biarsenical compds.  Because Shp2 contains no canonical tetracysteine biarsenical-binding motif, the enzyme's inhibitor-binding site is not readily predictable from its primary or three-dimensional structure.  Intriguingly, however, Shp2's PTP domain does contain a cysteine residue (C333) at a position that is removed from the active site and is occupied by proline in other classical PTPs.  We show that Shp2's unusual cysteine residue constitutes part of a Shp2-specific allosteric-inhibition site, and that Shp2's sensitivity to biarsenicals is dependent on the presence of the naturally occurring C333.  The determinative role of this residue in conferring inhibitor sensitivity is surprising because C333's side chain is inaccessible to solvent in Shp2 crystal structures.  The discovery of this cryptic Shp2 allosteric site may provide a means for targeting Shp2 activity with high specificity and suggests that buried-yet-targetable allosteric sites could be similarly uncovered in other protein families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWBChJ5ezZcbVg90H21EOLACvtfcHk0lgF-aEovWsH-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrs7rK&md5=824fcbb6a26460197422c7008f35ca2b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi5013595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi5013595%26sid%3Dliteratum%253Aachs%26aulast%3DChio%26aufirst%3DC.%2BM.%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DBishop%26aufirst%3DA.%2BC.%26atitle%3DTargeting%2520a%2520cryptic%2520allosteric%2520site%2520for%2520selective%2520inhibition%2520of%2520the%2520oncogenic%2520protein%2520tyrosine%2520phosphatase%2520Shp2%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D497%26epage%3D504%26doi%3D10.1021%2Fbi5013595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fekkes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Fortanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonakos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunenfelder, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Z. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Bonte, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenoir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palermo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slisz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, P. D.</span></span> <span> </span><span class="NLM_article-title">Allosteric inhibition of SHP2 phosphatase inhibits cancers driven by receptor tyrosine kinases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">148</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1038/nature18621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fnature18621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=27362227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVOns7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=148-152&author=Y.+N.+Chenauthor=M.+J.+LaMarcheauthor=H.+M.+Chanauthor=P.+Fekkesauthor=J.+Garcia-Fortanetauthor=M.+G.+Ackerauthor=B.+Antonakosauthor=C.+H.+Chenauthor=Z.+Chenauthor=V.+G.+Cookeauthor=J.+R.+Dobsonauthor=Z.+Dengauthor=F.+Feiauthor=B.+Firestoneauthor=M.+Fodorauthor=C.+Fridrichauthor=H.+Gaoauthor=D.+Grunenfelderauthor=H.+X.+Haoauthor=J.+Jacobauthor=S.+Hoauthor=K.+Hsiaoauthor=Z.+B.+Kangauthor=R.+Karkiauthor=M.+Katoauthor=J.+Larrowauthor=L.+R.+La+Bonteauthor=F.+Lenoirauthor=G.+Liuauthor=S.+Liuauthor=D.+Majumdarauthor=M.+J.+Meyerauthor=M.+Palermoauthor=L.+Perezauthor=M.+Puauthor=E.+Priceauthor=C.+Quinnauthor=S.+Shakyaauthor=M.+D.+Shultzauthor=J.+Sliszauthor=K.+Venkatesanauthor=P.+Wangauthor=M.+Warmuthauthor=S.+Williamsauthor=G.+Yangauthor=J.+Yuanauthor=J.+H.+Zhangauthor=P.+Zhuauthor=T.+Ramseyauthor=N.+J.+Keenauthor=W.+R.+Sellersauthor=T.+Stamsauthor=P.+D.+Fortin&title=Allosteric+inhibition+of+SHP2+phosphatase+inhibits+cancers+driven+by+receptor+tyrosine+kinases&doi=10.1038%2Fnature18621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric inhibition of SHP2 phosphatase inhibits cancers driven by receptor tyrosine kinases</span></div><div class="casAuthors">Chen, Ying-Nan P.; LaMarche, Matthew J.; Chan, Ho Man; Fekkes, Peter; Garcia-Fortanet, Jorge; Acker, Michael G.; Antonakos, Brandon; Chen, Christine Hiu-Tung; Chen, Zhouliang; Cooke, Vesselina G.; Dobson, Jason R.; Deng, Zhan; Fei, Feng; Firestone, Brant; Fodor, Michelle; Fridrich, Cary; Gao, Hui; Grunenfelder, Denise; Hao, Huai-Xiang; Jacob, Jaison; Ho, Samuel; Hsiao, Kathy; Kang, Zhao B.; Karki, Rajesh; Kato, Mitsunori; Larrow, Jay; La Bonte, Laura R.; Lenoir, Francois; Liu, Gang; Liu, Shumei; Majumdar, Dyuti; Meyer, Matthew J.; Palermo, Mark; Perez, Lawrence; Pu, Minying; Price, Edmund; Quinn, Christopher; Shakya, Subarna; Shultz, Michael D.; Slisz, Joanna; Venkatesan, Kavitha; Wang, Ping; Warmuth, Markus; Williams, Sarah; Yang, Guizhi; Yuan, Jing; Zhang, Ji-Hu; Zhu, Ping; Ramsey, Timothy; Keen, Nicholas J.; Sellers, William R.; Stams, Travis; Fortin, Pascal D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7610</span>),
    <span class="NLM_cas:pages">148-152</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The non-receptor protein tyrosine phosphatase SHP2, encoded by PTPN11, has an important role in signal transduction downstream of growth factor receptor signalling and was the first reported oncogenic tyrosine phosphatase.  Activating mutations of SHP2 have been assocd. with developmental pathologies such as Noonan syndrome and are found in multiple cancer types, including leukemia, lung and breast cancer and neuroblastoma.  SHP2 is ubiquitously expressed and regulates cell survival and proliferation primarily through activation of the RAS-ERK signalling pathway2,3.  It is also a key mediator of the programmed cell death 1 (PD-1) and B- and T-lymphocyte attenuator (BTLA) immune checkpoint pathways.  Redn. of SHP2 activity suppresses tumor cell growth and is a potential target of cancer therapy.  Here we report the discovery of a highly potent (IC50 = 0.071 μM), selective and orally bioavailable small-mol. SHP2 inhibitor, SHP099, that stabilizes SHP2 in an auto-inhibited conformation.  SHP099 concurrently binds to the interface of the N-terminal SH2, C-terminal SH2, and protein tyrosine phosphatase domains, thus inhibiting SHP2 activity through an allosteric mechanism.  SHP099 suppresses RAS-ERK signalling to inhibit the proliferation of receptor-tyrosine-kinase-driven human cancer cells in vitro and is efficacious in mouse tumor xenograft models.  Together, these data demonstrate that pharmacol. inhibition of SHP2 is a valid therapeutic approach for the treatment of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBs_e02S0Sp7Vg90H21EOLACvtfcHk0lhjcVnaU7bY-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVOns7bI&md5=ec8a4b612d5008a2519bd846d45ce019</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature18621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18621%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26aulast%3DChan%26aufirst%3DH.%2BM.%26aulast%3DFekkes%26aufirst%3DP.%26aulast%3DGarcia-Fortanet%26aufirst%3DJ.%26aulast%3DAcker%26aufirst%3DM.%2BG.%26aulast%3DAntonakos%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DCooke%26aufirst%3DV.%2BG.%26aulast%3DDobson%26aufirst%3DJ.%2BR.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DFei%26aufirst%3DF.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGrunenfelder%26aufirst%3DD.%26aulast%3DHao%26aufirst%3DH.%2BX.%26aulast%3DJacob%26aufirst%3DJ.%26aulast%3DHo%26aufirst%3DS.%26aulast%3DHsiao%26aufirst%3DK.%26aulast%3DKang%26aufirst%3DZ.%2BB.%26aulast%3DKarki%26aufirst%3DR.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DLarrow%26aufirst%3DJ.%26aulast%3DLa%2BBonte%26aufirst%3DL.%2BR.%26aulast%3DLenoir%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMajumdar%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DM.%2BJ.%26aulast%3DPalermo%26aufirst%3DM.%26aulast%3DPerez%26aufirst%3DL.%26aulast%3DPu%26aufirst%3DM.%26aulast%3DPrice%26aufirst%3DE.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DShakya%26aufirst%3DS.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DSlisz%26aufirst%3DJ.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DZhu%26aufirst%3DP.%26aulast%3DRamsey%26aufirst%3DT.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DFortin%26aufirst%3DP.%2BD.%26atitle%3DAllosteric%2520inhibition%2520of%2520SHP2%2520phosphatase%2520inhibits%2520cancers%2520driven%2520by%2520receptor%2520tyrosine%2520kinases%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D148%26epage%3D152%26doi%3D10.1038%2Fnature18621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia Fortanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fekkes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunenfelder, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Z. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaBonte, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenoir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palermo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric inhibition of SHP2: identification of a potent, selective, and orally efficacious phosphatase inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7773</span>– <span class="NLM_lpage">7782</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00680</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00680" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKju7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7773-7782&author=J.+Garcia+Fortanetauthor=C.+H.+Chenauthor=Y.+N.+Chenauthor=Z.+Chenauthor=Z.+Dengauthor=B.+Firestoneauthor=P.+Fekkesauthor=M.+Fodorauthor=P.+D.+Fortinauthor=C.+Fridrichauthor=D.+Grunenfelderauthor=S.+Hoauthor=Z.+B.+Kangauthor=R.+Karkiauthor=M.+Katoauthor=N.+Keenauthor=L.+R.+LaBonteauthor=J.+Larrowauthor=F.+Lenoirauthor=G.+Liuauthor=S.+Liuauthor=F.+Lombardoauthor=D.+Majumdarauthor=M.+J.+Meyerauthor=M.+Palermoauthor=L.+Perezauthor=M.+Puauthor=T.+Ramseyauthor=W.+R.+Sellersauthor=M.+D.+Shultzauthor=T.+Stamsauthor=C.+Towlerauthor=P.+Wangauthor=S.+L.+Williamsauthor=J.+H.+Zhangauthor=M.+J.+LaMarche&title=Allosteric+inhibition+of+SHP2%3A+identification+of+a+potent%2C+selective%2C+and+orally+efficacious+phosphatase+inhibitor&doi=10.1021%2Facs.jmedchem.6b00680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Inhibition of SHP2: Identification of a Potent, Selective, and Orally Efficacious Phosphatase Inhibitor</span></div><div class="casAuthors">Garcia Fortanet, Jorge; Chen, Christine Hiu-Tung; Chen, Ying-Nan P.; Chen, Zhouliang; Deng, Zhan; Firestone, Brant; Fekkes, Peter; Fodor, Michelle; Fortin, Pascal D.; Fridrich, Cary; Grunenfelder, Denise; Ho, Samuel; Kang, Zhao B.; Karki, Rajesh; Kato, Mitsunori; Keen, Nick; LaBonte, Laura R.; Larrow, Jay; Lenoir, Francois; Liu, Gang; Liu, Shumei; Lombardo, Franco; Majumdar, Dyuti; Meyer, Matthew J.; Palermo, Mark; Perez, Lawrence; Pu, Minying; Ramsey, Timothy; Sellers, William R.; Shultz, Michael D.; Stams, Travis; Towler, Christopher; Wang, Ping; Williams, Sarah L.; Zhang, Ji-Hu; LaMarche, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7773-7782</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2 is a nonreceptor protein tyrosine phosphatase (PTP) encoded by the PTPN11 gene involved in cell growth and differentiation via the MAPK signaling pathway.  SHP2 also purportedly plays an important role in the programmed cell death pathway (PD-1/PD-L1).  Because it is an oncoprotein assocd. with multiple cancer-related diseases, as well as a potential immunomodulator, controlling SHP2 activity is of significant therapeutic interest.  Recently in our labs., a small mol. inhibitor of SHP2 was identified as an allosteric modulator that stabilizes the autoinhibited conformation of SHP2.  A high throughput screen was performed to identify progressable chem. matter, and X-ray crystallog. revealed the location of binding in a previously undisclosed allosteric binding pocket.  Structure-based drug design was employed to optimize for SHP2 inhibition, and several new protein-ligand interactions were characterized.  These studies culminated in the discovery of 6-(4-amino-4-methylpiperidin-1-yl)-3-(2,3-dichlorophenyl)pyrazin-2-amine (SHP099, 1), a potent, selective, orally bioavailable, and efficacious SHP2 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6RT7zfZnvfLVg90H21EOLACvtfcHk0lhjcVnaU7bY-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKju7fP&md5=dfe43b5253e172576386f86797087f7b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00680%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%2BFortanet%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFekkes%26aufirst%3DP.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DFortin%26aufirst%3DP.%2BD.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DGrunenfelder%26aufirst%3DD.%26aulast%3DHo%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DZ.%2BB.%26aulast%3DKarki%26aufirst%3DR.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DLaBonte%26aufirst%3DL.%2BR.%26aulast%3DLarrow%26aufirst%3DJ.%26aulast%3DLenoir%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DMajumdar%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DM.%2BJ.%26aulast%3DPalermo%26aufirst%3DM.%26aulast%3DPerez%26aufirst%3DL.%26aulast%3DPu%26aufirst%3DM.%26aulast%3DRamsey%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DWilliams%26aufirst%3DS.%2BL.%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3DAllosteric%2520inhibition%2520of%2520SHP2%253A%2520identification%2520of%2520a%2520potent%252C%2520selective%252C%2520and%2520orally%2520efficacious%2520phosphatase%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7773%26epage%3D7782%26doi%3D10.1021%2Facs.jmedchem.6b00680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric inhibitors of SHP2 with therapeutic potential for cancer treatment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">10205</span>– <span class="NLM_lpage">10219</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01520</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01520" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVCks7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=10205-10219&author=J.+Xieauthor=X.+Siauthor=S.+Guauthor=M.+Wangauthor=J.+Shenauthor=H.+Liauthor=J.+Shenauthor=D.+Liauthor=Y.+Fangauthor=C.+Liuauthor=J.+Zhu&title=Allosteric+inhibitors+of+SHP2+with+therapeutic+potential+for+cancer+treatment&doi=10.1021%2Facs.jmedchem.7b01520"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Inhibitors of SHP2 with Therapeutic Potential for Cancer Treatment</span></div><div class="casAuthors">Xie, Jingjing; Si, Xiaojia; Gu, Shoulai; Wang, Mingliang; Shen, Jian; Li, Haoyan; Li, Dan; Fang, Yanjia; Liu, Cong; Zhu, Jidong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10205-10219</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2, a cytoplasmic protein tyrosine phosphatase encoded by the PTPN11 gene, is involved in multiple cell signaling processes including Ras/MAPK and Hippo/YAP pathways.  SHP2 has been shown to contribute to the progression of a no. of cancer types including leukemia, gastric, and breast cancers.  It also regulates T-cell activation by interacting with inhibitory immune checkpoint receptors such as the programmed cell death 1 (PD-1) and B- and T-lymphocyte attenuator (BTLA).  Thus, SHP2 inhibitors have drawn great attention by both inhibiting tumor cell proliferation and activating T cell immune responses toward cancer cells.  In this study, the authors report the identification of an allosteric SHP2 inhibitor I that locks SHP2 in a closed conformation by binding to the interface of the N-terminal SH2, C-terminal SH2, and phosphatase domains.  Compd. I suppresses MAPK signaling pathway and YAP transcriptional activity and shows antitumor activity in vivo.  The results indicate that allosteric inhibition of SHP2 could be a feasible approach for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriWHA25LKpH7Vg90H21EOLACvtfcHk0ljQZN6qvnWelg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVCks7%252FM&md5=4aaabafae3250476b7bf3687d83cffd3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01520%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DSi%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DJ.%26atitle%3DAllosteric%2520inhibitors%2520of%2520SHP2%2520with%2520therapeutic%2520potential%2520for%2520cancer%2520treatment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D10205%26epage%3D10219%26doi%3D10.1021%2Facs.jmedchem.7b01520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koltun, E. S.</span>; <span class="NLM_string-name">Aay, N.</span>; <span class="NLM_string-name">Buckl, A.</span>; <span class="NLM_string-name">Jogalekar, A. S.</span>; <span class="NLM_string-name">Kiss, G.</span>; <span class="NLM_string-name">Marquez, A.</span>; <span class="NLM_string-name">Mellem, K. T.</span>; <span class="NLM_string-name">Mordec, K.</span>; <span class="NLM_string-name">Saldajeno-Concar, M.</span>; <span class="NLM_string-name">Semko, C. M.</span></span> <span> </span><span class="NLM_article-title">RMC-4550, an allosteric inhibitor of SHP2: synthesis, structure, and anti-tumor activity</span>. Presented at the  <i>Annual Meeting of the American Association for Cancer Research (AACR)</i>. <span class="NLM_publisher-loc">Chicago, IL, Philadelphia (PA)</span>, <span class="NLM_year">2018</span>; Abstract 4878.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=E.+S.+Koltun&author=N.+Aay&author=A.+Buckl&author=A.+S.+Jogalekar&author=G.+Kiss&author=A.+Marquez&author=K.+T.+Mellem&author=K.+Mordec&author=M.+Saldajeno-Concar&author=C.+M.+Semko&title=RMC-4550%2C+an+allosteric+inhibitor+of+SHP2%3A+synthesis%2C+structure%2C+and+anti-tumor+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DKoltun%26aufirst%3DE.%2BS.%26atitle%3DRMC-4550%252C%2520an%2520allosteric%2520inhibitor%2520of%2520SHP2%253A%2520synthesis%252C%2520structure%252C%2520and%2520anti-tumor%2520activity%26jtitle%3DAnnual%2520Meeting%2520of%2520the%2520American%2520Association%2520for%2520Cancer%2520Research%2520%2528AACR%2529%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagdanoff, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dore, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hentemann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaBonte, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohseni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ntaganda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span> <span> </span><span class="NLM_article-title">Optimization of fused bicyclic allosteric SHP2 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1781</span>– <span class="NLM_lpage">1792</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01725</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01725" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1781-1792&author=J.+T.+Bagdanoffauthor=Z.+Chenauthor=M.+Ackerauthor=Y.+N.+Chenauthor=H.+Chanauthor=M.+Doreauthor=B.+Firestoneauthor=M.+Fodorauthor=J.+Fortanetauthor=M.+Hentemannauthor=M.+Katoauthor=R.+Koenigauthor=L.+R.+LaBonteauthor=S.+Liuauthor=M.+Mohseniauthor=R.+Ntagandaauthor=P.+Sarverauthor=T.+Smithauthor=M.+Sendzikauthor=T.+Stamsauthor=S.+Spenceauthor=C.+Towlerauthor=H.+Wangauthor=P.+Wangauthor=S.+L.+Williamsauthor=M.+J.+LaMarche&title=Optimization+of+fused+bicyclic+allosteric+SHP2+inhibitors&doi=10.1021%2Facs.jmedchem.8b01725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Fused Bicyclic Allosteric SHP2 Inhibitors</span></div><div class="casAuthors">Bagdanoff, Jeffrey T.; Chen, Zhouliang; Acker, Michael; Chen, Ying-Nan; Chan, Homan; Dore, Michael; Firestone, Brant; Fodor, Michelle; Fortanet, Jorge; Hentemann, Murphy; Kato, Mitsunori; Koenig, Robert; La Bonte, Laura R.; Liu, Shumei; Mohseni, Movarid; Ntaganda, Rukundo; Sarver, Patrick; Smith, Troy; Sendzik, Martin; Stams, Travis; Spence, Stan; Towler, Christopher; Wang, Hongyun; Wang, Ping; Williams, Sarah L.; La Marche, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1781-1792</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2 is a nonreceptor protein tyrosine phosphatase within the mitogen-activated protein kinase (MAPK) pathway controlling cell growth, differentiation, and oncogenic transformation.  SHP2 also participates in the programed cell death pathway (PD-1/PD-L1) governing immune surveillance.  Small-mol. inhibition of SHP2 has been widely investigated, including in our previous reports describing SHP099 (2), which binds to a tunnel-like allosteric binding site.  To broaden our approach to allosteric inhibition of SHP2, we conducted addnl. hit finding, evaluation, and structure-based scaffold morphing.  These studies, reported here in the first of two papers, led to the identification of multiple 5,6-fused bicyclic scaffolds that bind to the same allosteric tunnel as 2.  We demonstrate the structural diversity permitted by the tunnel pharmacophore and culminated in the identification of pyrazolopyrimidinones (e.g., SHP389, 1) that modulate MAPK signaling in vivo.  These studies also served as the basis for further scaffold morphing and optimization, detailed in the following manuscript.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra2OF_vrNnP7Vg90H21EOLACvtfcHk0ljQZN6qvnWelg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjtbo%253D&md5=e810a8a6bd1d1fb7bfadc5bc1448ff22</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01725%26sid%3Dliteratum%253Aachs%26aulast%3DBagdanoff%26aufirst%3DJ.%2BT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DAcker%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DDore%26aufirst%3DM.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DFortanet%26aufirst%3DJ.%26aulast%3DHentemann%26aufirst%3DM.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKoenig%26aufirst%3DR.%26aulast%3DLaBonte%26aufirst%3DL.%2BR.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMohseni%26aufirst%3DM.%26aulast%3DNtaganda%26aufirst%3DR.%26aulast%3DSarver%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DT.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DSpence%26aufirst%3DS.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DWilliams%26aufirst%3DS.%2BL.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3DOptimization%2520of%2520fused%2520bicyclic%2520allosteric%2520SHP2%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1781%26epage%3D1792%26doi%3D10.1021%2Facs.jmedchem.8b01725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagdanoff, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hentemann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaBonte, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohseni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span> <span> </span><span class="NLM_article-title">6-Amino-3-methylpyrimidinones as potent, selective, and orally efficacious SHP2 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1793</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01726</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01726" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1793-1802&author=P.+Sarverauthor=M.+Ackerauthor=J.+T.+Bagdanoffauthor=Z.+Chenauthor=Y.+N.+Chenauthor=H.+Chanauthor=B.+Firestoneauthor=M.+Fodorauthor=J.+Fortanetauthor=H.+Haoauthor=M.+Hentemannauthor=M.+Katoauthor=R.+Koenigauthor=L.+R.+LaBonteauthor=G.+Liuauthor=S.+Liuauthor=C.+Liuauthor=E.+McNeillauthor=M.+Mohseniauthor=M.+Sendzikauthor=T.+Stamsauthor=S.+Spenceauthor=V.+Tamezauthor=R.+Tichkuleauthor=C.+Towlerauthor=H.+Wangauthor=P.+Wangauthor=S.+L.+Williamsauthor=B.+Yuauthor=M.+J.+LaMarche&title=6-Amino-3-methylpyrimidinones+as+potent%2C+selective%2C+and+orally+efficacious+SHP2+inhibitors&doi=10.1021%2Facs.jmedchem.8b01726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">6-Amino-3-methylpyrimidinones as Potent, Selective, and Orally Efficacious SHP2 Inhibitors</span></div><div class="casAuthors">Sarver, Patrick; Acker, Michael; Bagdanoff, Jeffrey T.; Chen, Zhouliang; Chen, Ying-Nan; Chan, Homan; Firestone, Brant; Fodor, Michelle; Fortanet, Jorge; Hao, Huaixiang; Hentemann, Murphy; Koenig, Robert; Kato, Mitsunori; LaBonte, Laura R.; Liu, Gang; Liu, Shumei; Liu, Chen; McNeill, Eric; Mohseni, Morvarid; Sendzik, Martin; Stams, Travis; Spence, Stan; Tamez, Victoriano; Tichkule, Ritesh; Towler, Christopher; Wang, Hongyun; Wang, Ping; Williams, Sarah L.; Yu, Bing; LaMarche, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1793-1802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein tyrosine phosphatase SHP2 is an oncoprotein assocd. with cancer as well as a potential immune modulator because of its role in the programmed cell death PD-L1/PD-1 pathway.  In the preceding manuscript, we described the optimization of a fused, bicyclic screening hit for potency, selectivity, and physicochem. properties in order to further expand the chem. diversity of allosteric SHP2 inhibitors.  In this manuscript, we describe the further expansion of our approach, morphing the fused, bicyclic system into a novel monocyclic pyrimidinone scaffold through our understanding of SAR and use of structure-based design.  These studies led to the identification of SHP394 (1), an orally efficacious inhibitor of SHP2, with high lipophilic efficiency, improved potency, and enhanced pharmacokinetic properties.  We also report other pyrimidinone analogs with favorable pharmacokinetic and potency profiles.  Overall, this work improves upon our previously described allosteric inhibitors and exemplifies and extends the range of permissible chem. templates that inhibit SHP2 via the allosteric mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaRey6_uE0ULVg90H21EOLACvtfcHk0li7lXK3QTgRPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKisLs%253D&md5=29c357236b7ff51596385e3e256320a5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01726%26sid%3Dliteratum%253Aachs%26aulast%3DSarver%26aufirst%3DP.%26aulast%3DAcker%26aufirst%3DM.%26aulast%3DBagdanoff%26aufirst%3DJ.%2BT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DFortanet%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DH.%26aulast%3DHentemann%26aufirst%3DM.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKoenig%26aufirst%3DR.%26aulast%3DLaBonte%26aufirst%3DL.%2BR.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DMcNeill%26aufirst%3DE.%26aulast%3DMohseni%26aufirst%3DM.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DSpence%26aufirst%3DS.%26aulast%3DTamez%26aufirst%3DV.%26aulast%3DTichkule%26aufirst%3DR.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DWilliams%26aufirst%3DS.%2BL.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3D6-Amino-3-methylpyrimidinones%2520as%2520potent%252C%2520selective%252C%2520and%2520orally%2520efficacious%2520SHP2%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1793%26epage%3D1802%26doi%3D10.1021%2Facs.jmedchem.8b01726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mainardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulero-Sánchez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prahallad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimpenfort, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieftink, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinberg, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wit, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves-Ribeiro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadal, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernards, R.</span></span> <span> </span><span class="NLM_article-title">SHP2 is required for growth of KRAS-mutant non-small-cell lung cancer in vivo</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">961</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0023-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41591-018-0023-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=29808006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVelsbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=961-967&author=S.+Mainardiauthor=A.+Mulero-S%C3%A1nchezauthor=A.+Prahalladauthor=G.+Germanoauthor=A.+Bosmaauthor=P.+Krimpenfortauthor=C.+Lieftinkauthor=J.+D.+Steinbergauthor=N.+de+Witauthor=S.+Gon%C3%A7alves-Ribeiroauthor=E.+Nadalauthor=A.+Bardelliauthor=A.+Villanuevaauthor=R.+Bernards&title=SHP2+is+required+for+growth+of+KRAS-mutant+non-small-cell+lung+cancer+in+vivo&doi=10.1038%2Fs41591-018-0023-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">SHP2 is required for growth of KRAS-mutant non-small-cell lung cancer in vivo</span></div><div class="casAuthors">Mainardi, Sara; Mulero-Sanchez, Antonio; Prahallad, Anirudh; Germano, Giovanni; Bosma, Astrid; Krimpenfort, Paul; Lieftink, Cor; Steinberg, Jeffrey D.; de Wit, Niels; Goncalves-Ribeiro, Samuel; Nadal, Ernest; Bardelli, Alberto; Villanueva, Alberto; Bernards, Rene</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">961-967</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">RAS mutations are frequent in human cancer, esp. in pancreatic, colorectal and non-small-cell lung cancers (NSCLCs).  Inhibition of the RAS oncoproteins has proven difficult, and attempts to target downstream effectors have been hampered by the activation of compensatory resistance mechanisms.  It is also well established that KRAS-mutant tumors are insensitive to inhibition of upstream growth factor receptor signaling.  Thus, epidermal growth factor receptor antibody therapy is only effective in KRAS wild-type colon cancers.  Consistently, inhibition of SHP2 (also known as PTPN11), which links receptor tyrosine kinase signaling to the RAS-RAF-MEK-ERK pathway11,12, was shown to be ineffective in KRAS-mutant or BRAF-mutant cancer cell lines.  Tha data also indicate that SHP2 inhibition in KRAS-mutant NSCLC cells under normal cell culture conditions has little effect.  By contrast, SHP2 inhibition under growth factor-limiting conditions in vitro results in a senescence response.  In vivo, inhibition of SHP2 in KRAS-mutant NSCLC also provokes a senescence response, which is exacerbated by MEK inhibition.  Tha data identify SHP2 inhibition as an unexpected vulnerability of KRAS-mutant NSCLC cells that remains undetected in cell culture and can be exploited therapeutically.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLHUMCtBqtZrVg90H21EOLACvtfcHk0li7lXK3QTgRPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVelsbvL&md5=8e1ce1eb5d30d3985aafdec9d07f205d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0023-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0023-9%26sid%3Dliteratum%253Aachs%26aulast%3DMainardi%26aufirst%3DS.%26aulast%3DMulero-S%25C3%25A1nchez%26aufirst%3DA.%26aulast%3DPrahallad%26aufirst%3DA.%26aulast%3DGermano%26aufirst%3DG.%26aulast%3DBosma%26aufirst%3DA.%26aulast%3DKrimpenfort%26aufirst%3DP.%26aulast%3DLieftink%26aufirst%3DC.%26aulast%3DSteinberg%26aufirst%3DJ.%2BD.%26aulast%3Dde%2BWit%26aufirst%3DN.%26aulast%3DGon%25C3%25A7alves-Ribeiro%26aufirst%3DS.%26aulast%3DNadal%26aufirst%3DE.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DVillanueva%26aufirst%3DA.%26aulast%3DBernards%26aufirst%3DR.%26atitle%3DSHP2%2520is%2520required%2520for%2520growth%2520of%2520KRAS-mutant%2520non-small-cell%2520lung%2520cancer%2520in%2520vivo%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D961%26epage%3D967%26doi%3D10.1038%2Fs41591-018-0023-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruess, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heynen, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciecielski, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabacaoglu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Görgülü, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantes, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wörmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diakopoulos, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpathaki, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya-Aksoy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Laan, E. A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Alberca, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazaré, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erkan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopt, U. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sainz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birchmeier, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algül, H.</span></span> <span> </span><span class="NLM_article-title">Mutant KRAS-driven cancers depend on PTPN11/SHP2 phosphatase</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">954</span>– <span class="NLM_lpage">960</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0024-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41591-018-0024-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=29808009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVelsbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=954-960&author=D.+A.+Ruessauthor=G.+J.+Heynenauthor=K.+J.+Ciecielskiauthor=J.+Aiauthor=A.+Berningerauthor=D.+Kabacaogluauthor=K.+G%C3%B6rg%C3%BCl%C3%BCauthor=Z.+Dantesauthor=S.+M.+W%C3%B6rmannauthor=K.+N.+Diakopoulosauthor=A.+F.+Karpathakiauthor=M.+Kowalskaauthor=E.+Kaya-Aksoyauthor=L.+Songauthor=E.+A.+Z.+van+der+Laanauthor=M.+P.+L%C3%B3pez-Albercaauthor=M.+Nazar%C3%A9author=M.+Reichertauthor=D.+Saurauthor=M.+M.+Erkanauthor=U.+T.+Hoptauthor=B.+Sainzauthor=W.+Birchmeierauthor=R.+M.+Schmidauthor=M.+Lesinaauthor=H.+Alg%C3%BCl&title=Mutant+KRAS-driven+cancers+depend+on+PTPN11%2FSHP2+phosphatase&doi=10.1038%2Fs41591-018-0024-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Mutant KRAS-driven cancers depend on PTPN11/SHP2 phosphatase</span></div><div class="casAuthors">Ruess, Dietrich A.; Heynen, Guus J.; Ciecielski, Katrin J.; Ai, Jiaoyu; Berninger, Alexandra; Kabacaoglu, Derya; Goerguelue, Kivanc; Dantes, Zahra; Woermann, Sonja M.; Diakopoulos, Kalliope N.; Karpathaki, Angeliki F.; Kowalska, Marlena; Kaya-Aksoy, Ezgi; Song, Liang; van der Laan, Eveline A. Zeeuw; Lopez-Alberca, Maria P.; Nazare, Marc; Reichert, Maximilian; Saur, Dieter; Erkan, Mert M.; Hopt, Ulrich T.; Sainz, Jr, Bruno; Birchmeier, Walter; Schmid, Roland M.; Lesina, Marina; Alguel, Hana</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">954-960</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The ubiquitously expressed non-receptor protein tyrosine phosphatase SHP2, encoded by PTPN11, is involved in signal transduction downstream of multiple growth factor, cytokine and integrin receptors1.  Its requirement for complete RAS-MAPK activation and its role as a neg. regulator of JAK-STAT signaling have established SHP2 as an essential player in oncogenic signaling pathways1-7.  Recently, a novel potent allosteric SHP2 inhibitor was presented as a viable therapeutic option for receptor tyrosine kinase-driven cancers, but was shown to be ineffective in KRAS-mutant tumor cell lines in vitro8.  Here, we report a central and indispensable role for SHP2 in oncogenic KRAS-driven tumors.  Genetic deletion of Ptpn11 profoundly inhibited tumor development in mutant KRAS-driven murine models of pancreatic ductal adenocarcinoma and non-small-cell lung cancer.  We provide evidence for a crit. dependence of mutant KRAS on SHP2 during carcinogenesis.  Deletion or inhibition of SHP2 in established tumors delayed tumor progression but was not sufficient to achieve tumor regression.  However, SHP2 was necessary for resistance mechanisms upon blockade of MEK.  Synergy was obsd. when both SHP2 and MEK were targeted, resulting in sustained tumor growth control in murine and human patient-derived organoids and xenograft models of pancreatic ductal adenocarcinoma and non-small-cell lung cancer.  Our data indicate the clin. utility of dual SHP2/MEK inhibition as a targeted therapy approach for KRAS-mutant cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoepq09hU77abVg90H21EOLACvtfcHk0lijIlKk3YPnZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVelsbvF&md5=cc09897c2c4b8e32b035b991e5032f32</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0024-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0024-8%26sid%3Dliteratum%253Aachs%26aulast%3DRuess%26aufirst%3DD.%2BA.%26aulast%3DHeynen%26aufirst%3DG.%2BJ.%26aulast%3DCiecielski%26aufirst%3DK.%2BJ.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DBerninger%26aufirst%3DA.%26aulast%3DKabacaoglu%26aufirst%3DD.%26aulast%3DG%25C3%25B6rg%25C3%25BCl%25C3%25BC%26aufirst%3DK.%26aulast%3DDantes%26aufirst%3DZ.%26aulast%3DW%25C3%25B6rmann%26aufirst%3DS.%2BM.%26aulast%3DDiakopoulos%26aufirst%3DK.%2BN.%26aulast%3DKarpathaki%26aufirst%3DA.%2BF.%26aulast%3DKowalska%26aufirst%3DM.%26aulast%3DKaya-Aksoy%26aufirst%3DE.%26aulast%3DSong%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BLaan%26aufirst%3DE.%2BA.%2BZ.%26aulast%3DL%25C3%25B3pez-Alberca%26aufirst%3DM.%2BP.%26aulast%3DNazar%25C3%25A9%26aufirst%3DM.%26aulast%3DReichert%26aufirst%3DM.%26aulast%3DSaur%26aufirst%3DD.%26aulast%3DErkan%26aufirst%3DM.%2BM.%26aulast%3DHopt%26aufirst%3DU.%2BT.%26aulast%3DSainz%26aufirst%3DB.%26aulast%3DBirchmeier%26aufirst%3DW.%26aulast%3DSchmid%26aufirst%3DR.%2BM.%26aulast%3DLesina%26aufirst%3DM.%26aulast%3DAlg%25C3%25BCl%26aufirst%3DH.%26atitle%3DMutant%2520KRAS-driven%2520cancers%2520depend%2520on%2520PTPN11%252FSHP2%2520phosphatase%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D954%26epage%3D960%26doi%3D10.1038%2Fs41591-018-0024-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lijIlKk3YPnZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lijIlKk3YPnZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lhl0WNMSIQSNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) - past, present and future</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=Proteolysis+targeting+chimeras+%28PROTACs%29+-+past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eZcO2Qyg4mlFrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520-%2520past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lhl0WNMSIQSNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Enhancing antiproliferative activity and selectivity of a FLT-3 inhibitor by proteolysis targeting chimera conversion</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">16428</span>– <span class="NLM_lpage">16432</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b10320</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b10320" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2qtr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=16428-16432&author=G.+M.+Burslemauthor=J.+Songauthor=X.+Chenauthor=J.+Hinesauthor=C.+M.+Crews&title=Enhancing+antiproliferative+activity+and+selectivity+of+a+FLT-3+inhibitor+by+proteolysis+targeting+chimera+conversion&doi=10.1021%2Fjacs.8b10320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancing Antiproliferative Activity and Selectivity of a FLT-3 Inhibitor by Proteolysis Targeting Chimera Conversion</span></div><div class="casAuthors">Burslem, George M.; Song, Jayoung; Chen, Xin; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16428-16432</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The receptor tyrosine kinase FLT-3 is frequently mutated in acute myeloid leukemia; however, current small mol. inhibitors suffer from limited efficacy in the clinic.  Conversion of a FLT-3 inhibitor (quizartinib) into a proteolysis targeting chimera (PROTAC) results in a compd. that induces degrdn. of FLT-3 ITD mutant at low nanomolar concns.  Furthermore, the PROTAC is capable of inhibiting cell growth more potently than the warhead alone while inhibiting fewer off-target kinases.  This enhanced antiproliferative activity occurs, despite a slight redn. in the PROTAC's kinase inhibitory activity, via an increased level of apoptosis induction suggesting nonkinase roles for the FLT-3 ITD protein.  Addnl., the PROTAC is capable of inducing FLT-3 ITD degrdn. in vivo.  These results suggest that degrdn. of FLT-3 ITD may provide a useful method for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBxw_nAEQydLVg90H21EOLACvtfcHk0lhl0WNMSIQSNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2qtr7P&md5=52529d0c3f49052bd2a66c893e98e694</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b10320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b10320%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DEnhancing%2520antiproliferative%2520activity%2520and%2520selectivity%2520of%2520a%2520FLT-3%2520inhibitor%2520by%2520proteolysis%2520targeting%2520chimera%2520conversion%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D16428%26epage%3D16432%26doi%3D10.1021%2Fjacs.8b10320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmusson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricchiuto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Development of a novel B-cell lymphoma 6 (BCL6) PROTAC to provide insight into small molecule targeting of BCL6</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3141</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=3131-3141&author=W.+McCoullauthor=T.+Cheungauthor=E.+Andersonauthor=P.+Bartonauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=M.+P.+Castaldiauthor=H.+Chenauthor=E.+Chiarparinauthor=R.+J.+Carbajoauthor=E.+Codeauthor=S.+Cowanauthor=P.+R.+Daveyauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=N.+Gaoauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=A.+Kawatkarauthor=P.+D.+Kemmittauthor=E.+Leoauthor=D.+M.+Molinaauthor=N.+O%E2%80%99Connellauthor=P.+Petterutiauthor=T.+Rasmussonauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=P.+Ricchiutoauthor=G.+R.+Robbauthor=M.+Schenoneauthor=M.+J.+Waringauthor=M.+Zindaauthor=S.+Fawellauthor=D.+M.+Wilson&title=Development+of+a+novel+B-cell+lymphoma+6+%28BCL6%29+PROTAC+to+provide+insight+into+small+molecule+targeting+of+BCL6&doi=10.1021%2Facschembio.8b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Novel B-Cell Lymphoma 6 (BCL6) PROTAC To Provide Insight into Small Molecule Targeting of BCL6</span></div><div class="casAuthors">McCoull, William; Cheung, Tony; Anderson, Erica; Barton, Peter; Burgess, Jonathan; Byth, Kate; Cao, Qing; Castaldi, M. Paola; Chen, Huawei; Chiarparin, Elisabetta; Carbajo, Rodrigo J.; Code, Erin; Cowan, Suzanna; Davey, Paul R.; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gao, Ning; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kawatkar, Aarti; Kemmitt, Paul D.; Leo, Elisabetta; Molina, Daniel M.; O'Connell, Nichole; Petteruti, Philip; Rasmusson, Timothy; Raubo, Piotr; Rawlins, Philip B.; Ricchiuto, Piero; Robb, Graeme R.; Schenone, Monica; Waring, Michael J.; Zinda, Michael; Fawell, Stephen; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3131-3141</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) inhibition is a promising mechanism for treating hematol. cancers but high quality chem. probes are necessary to evaluate its therapeutic potential.  Here we report potent BCL6 inhibitors that demonstrate cellular target engagement and exhibit exquisite selectivity for BCL6 based on mass spectrometry analyses following chem. proteomic pull down.  Importantly, a proteolysis-targeting chimera, 4-((R)-25-Chloro-41-methyl-42-oxo-41,42,43,44-tetrahydro-5,8-dioxa-3-aza-4(6,8)-quinolina2(2,4)-pyrimidina-1(1,2)-piperazinacyclononaphane-14-yl)-N-(3-(2-(2-((2-((S)-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)propyl)-4-oxobutanamide trifluoroacetate, (PROTAC, 15) was also developed and shown to significantly degrade BCL6 in a no. of diffuse large B-cell lymphoma (DLBCL) cell lines, but neither BCL6 inhibition nor degrdn. selectively induced marked phenotypic response.  To investigate, we monitored PROTAC directed BCL6 degrdn. in DLBCL OCI-Ly1 cells by immunofluorescence and discovered a residual BCL6 population.  Anal. of subcellular fractions also showed incomplete BCL6 degrdn. in all fractions despite having measurable PROTAC concns., together providing a rationale for the weak antiproliferative response seen with both BCL6 inhibitor and degrader.  In summary, we have developed potent and selective BCL6 inhibitors and a BCL6 PROTAC that effectively degraded BCL6, but both modalities failed to induce a significant phenotypic response in DLBCL despite achieving cellular concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEe9aUjqcaHLVg90H21EOLACvtfcHk0liny5ey3_Eyvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF&md5=3d2c2d8acf143f05babee6ebdabddc8c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00698%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DDavey%26aufirst%3DP.%2BR.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKawatkar%26aufirst%3DA.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DRasmusson%26aufirst%3DT.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRicchiuto%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520PROTAC%2520to%2520provide%2520insight%2520into%2520small%2520molecule%2520targeting%2520of%2520BCL6%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D3131%26epage%3D3141%26doi%3D10.1021%2Facschembio.8b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of anaplastic lymphoma kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+induced+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0liny5ey3_Eyvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0lg44F5p1q4HCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0lg44F5p1q4HCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Development of dual and selective degraders of cyclin-dependent kinases 4 and 6</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6321</span>– <span class="NLM_lpage">6326</span>, <span class="refDoi"> DOI: 10.1002/anie.201901336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1002%2Fanie.201901336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtlKqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=6321-6326&author=B.+Jiangauthor=E.+S.+Wangauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Fischerauthor=T.+Zhangauthor=N.+S.+Gray&title=Development+of+dual+and+selective+degraders+of+cyclin-dependent+kinases+4+and+6&doi=10.1002%2Fanie.201901336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Dual and Selective Degraders of Cyclin-Dependent Kinases 4 and 6</span></div><div class="casAuthors">Jiang, Baishan; Wang, Eric S.; Donovan, Katherine A.; Liang, Yanke; Fischer, Eric S.; Zhang, Tinghu; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6321-6326</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases 4 and 6 (CDK4/6) are key regulators of the cell cycle, and there are FDA-approved CDK4/6 inhibitors for treating patients with metastatic breast cancer.  However, due to conservation of their ATP-binding sites, development of selective agents has remained elusive.  Here, we report imide-based degrader mols. capable of degrading both CDK4/6, or selectively degrading either CDK4 or CDK6.  We were also able to tune the activity of these mols. against Ikaros (IKZF1) and Aiolos (IKZF3), which are well-established targets of imide-based degraders.  We found that in mantle cell lymphoma cell lines, combined IKZF1/3 degrdn. with dual CDK4/6 degrdn. produced enhanced anti-proliferative effects compared to CDK4/6 inhibition, CDK4/6 degrdn., or IKZF1/3 degrdn.  In summary, we report here the first compds. capable of inducing selective degrdn. of CDK4 and CDK6 as tools to pharmacol. dissect their distinct biol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Vsqs-SM6hrVg90H21EOLACvtfcHk0lg44F5p1q4HCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtlKqsrw%253D&md5=39a6937b24abad0940d359020e2b2f74</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fanie.201901336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201901336%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520dual%2520and%2520selective%2520degraders%2520of%2520cyclin-dependent%2520kinases%25204%2520and%25206%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D6321%26epage%3D6326%26doi%3D10.1002%2Fanie.201901336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of SD-36 as a potent, selective, and efficacious PROTAC degrader of STAT3 protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11280</span>– <span class="NLM_lpage">11300</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01530</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01530" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11280-11300&author=H.+Zhouauthor=L.+Baiauthor=R.+Xuauthor=Y.+Zhaoauthor=J.+Chenauthor=D.+McEachernauthor=K.+Chinnaswamyauthor=B.+Wenauthor=L.+Daiauthor=P.+Kumarauthor=C.-Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=J.+L.+Meagherauthor=H.+Yiauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-based+discovery+of+SD-36+as+a+potent%2C+selective%2C+and+efficacious+PROTAC+degrader+of+STAT3+protein&doi=10.1021%2Facs.jmedchem.9b01530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span></div><div class="casAuthors">Zhou, Haibin; Bai, Longchuan; Xu, Renqi; Zhao, Yujun; Chen, Jianyong; McEachern, Donna; Chinnaswamy, Krishnapriya; Wen, Bo; Dai, Lipeng; Kumar, Praveen; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Meagher, Jennifer L.; Yi, Han; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11280-11300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases.  Despite twenty-years of persistent research efforts, targeting STAT3 has been very challenging.  The authors report herein the structure-based discovery of potent small-mol. STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept.  The authors first designed SI-109 as a potent, small-mol. inhibitor of the STAT3 SH2 domain.  Employing ligands for cereblon/Cullin 4A E3 ligase and SI-109, the authors obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36.  SD-36 induces rapid STAT3 degrdn. at low nanomolar concns. in cells and fails to degrader other STAT proteins.  SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3.  A single dose of SD-36 results in complete STAT3 protein degrdn. in xenograft tumor tissue and normal mouse tissues.  SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules.  SD-36 is a potent, selective and efficacious STAT3 degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIAWd1hiuqy7Vg90H21EOLACvtfcHk0ljJOlZR2FbR_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J&md5=7d0c656a2d96196a3b71829c9efe715a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01530%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%2520SD-36%2520as%2520a%2520potent%252C%2520selective%252C%2520and%2520efficacious%2520PROTAC%2520degrader%2520of%2520STAT3%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11280%26epage%3D11300%26doi%3D10.1021%2Facs.jmedchem.9b01530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">Arvinas’s PROTACs pass first safety and PK analysis</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">895</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00188-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fd41573-019-00188-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=895&author=A.+Mullard&title=Arvinas%E2%80%99s+PROTACs+pass+first+safety+and+PK+analysis&doi=10.1038%2Fd41573-019-00188-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">NIH launches open science Alzheimer initiative</span></div><div class="casAuthors">Mullard, Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">895</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKr7PwgMO_fLVg90H21EOLACvtfcHk0ljJOlZR2FbR_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs7zO&md5=2abfeaf7ac16a726ceff22cb0164347c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00188-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00188-4%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DArvinas%25E2%2580%2599s%2520PROTACs%2520pass%2520first%2520safety%2520and%2520PK%2520analysis%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D895%26doi%3D10.1038%2Fd41573-019-00188-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potjewyd, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rectenwald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolis, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herring, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Degradation of polycomb repressive complex 2 with an EED-targeted bivalent chemical degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1016%2Fj.chembiol.2019.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=31831267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlOmt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=47-56&author=F.+Potjewydauthor=A.+W.+Turnerauthor=J.+Beriauthor=J.+M.+Rectenwaldauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=D.+M.+Margolisauthor=K.+H.+Pearceauthor=L.+E.+Herringauthor=L.+I.+James&title=Degradation+of+polycomb+repressive+complex+2+with+an+EED-targeted+bivalent+chemical+degrader&doi=10.1016%2Fj.chembiol.2019.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of Polycomb Repressive Complex 2 with an EED-Targeted Bivalent Chemical Degrader</span></div><div class="casAuthors">Potjewyd, Frances; Turner, Anne-Marie W.; Beri, Joshua; Rectenwald, Justin M.; Norris-Drouin, Jacqueline L.; Cholensky, Stephanie H.; Margolis, David M.; Pearce, Kenneth H.; Herring, Laura E.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-56.e15</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein degrdn. via the use of bivalent chem. degraders provides an alternative strategy to block protein function and assess the biol. roles of putative drug targets.  This approach capitalizes on the advantages of small-mol. inhibitors while moving beyond the restrictions of traditional pharmacol.  Here, we report a chem. degrader (UNC6852) that targets polycomb repressive complex 2 (PRC2).  UNC6852 contains an EED226-derived ligand and a ligand for VHL which bind to the WD40 arom. cage of EED and CRL2VHL, resp., to induce proteasomal degrdn. of PRC2 components, EED, EZH2, and SUZ12.  Degrdn. of PRC2 with UNC6852 blocks the histone methyltransferase activity of EZH2, decreasing H3K27me3 levels in HeLa cells and diffuse large B cell lymphoma (DLBCL) cells contg. EZH2 gain-of-function mutations.  UNC6852 degrades both wild-type and mutant EZH2, and addnl. displays anti-proliferative effects in this cancer model system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF3p-HFdD9KLVg90H21EOLACvtfcHk0ljJOlZR2FbR_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlOmt7bM&md5=7d9ca2726bad4e40003310951edc09d4</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DPotjewyd%26aufirst%3DF.%26aulast%3DTurner%26aufirst%3DA.%2BW.%26aulast%3DBeri%26aufirst%3DJ.%26aulast%3DRectenwald%26aufirst%3DJ.%2BM.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26aulast%3DHerring%26aufirst%3DL.%2BE.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DDegradation%2520of%2520polycomb%2520repressive%2520complex%25202%2520with%2520an%2520EED-targeted%2520bivalent%2520chemical%2520degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D47%26epage%3D56%26doi%3D10.1016%2Fj.chembiol.2019.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruvilla, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-m.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogojina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghton, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hromas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span> <span> </span><span class="NLM_article-title">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1938</span>– <span class="NLM_lpage">1947</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0668-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41591-019-0668-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=31792461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1938-1947&author=S.+Khanauthor=X.+Zhangauthor=D.+Lvauthor=Q.+Zhangauthor=Y.+Heauthor=P.+Zhangauthor=X.+Liuauthor=D.+Thummuriauthor=Y.+Yuanauthor=J.+S.+Wiegandauthor=J.+Peiauthor=W.+Zhangauthor=A.+Sharmaauthor=C.+R.+McCurdyauthor=V.+M.+Kuruvillaauthor=N.+Baranauthor=A.+A.+Ferrandoauthor=Y.-m.+Kimauthor=A.+Rogojinaauthor=P.+J.+Houghtonauthor=G.+Huangauthor=R.+Hromasauthor=M.+Konoplevaauthor=G.+Zhengauthor=D.+Zhou&title=A+selective+BCL-XL+PROTAC+degrader+achieves+safe+and+potent+antitumor+activity&doi=10.1038%2Fs41591-019-0668-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span></div><div class="casAuthors">Khan, Sajid; Zhang, Xuan; Lv, Dongwen; Zhang, Qi; He, Yonghan; Zhang, Peiyi; Liu, Xingui; Thummuri, Dinesh; Yuan, Yaxia; Wiegand, Janet S.; Pei, Jing; Zhang, Weizhou; Sharma, Abhisheak; McCurdy, Christopher R.; Kuruvilla, Vinitha M.; Baran, Natalia; Ferrando, Adolfo A.; Kim, Yong-mi; Rogojina, Anna; Houghton, Peter J.; Huang, Guangcun; Hromas, Robert; Konopleva, Marina; Zheng, Guangrong; Zhou, Daohong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1938-1947</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">B-cell lymphoma extra large (BCL-XL) is a well-validated cancer target.  However, the on-target and dose-limiting thrombocytopenia limits the use of BCL-XL inhibitors, such as ABT263, as safe and effective anticancer agents.  To reduce the toxicity of ABT263, we converted it into DT2216, a BCL-XL proteolysis-targeting chimera (PROTAC), that targets BCL-XL to the Von Hippel-Lindau (VHL) E3 ligase for degrdn.  We found that DT2216 was more potent against various BCL-XL-dependent leukemia and cancer cells but considerably less toxic to platelets than ABT263 in vitro because VHL is poorly expressed in platelets.  In vivo, DT2216 effectively inhibits the growth of several xenograft tumors as a single agent or in combination with other chemotherapeutic agents, without causing appreciable thrombocytopenia.  These findings demonstrate the potential to use PROTAC technol. to reduce on-target drug toxicities and rescue the therapeutic potential of previously undruggable targets.  Furthermore, DT2216 may be developed as a safe first-in-class anticancer agent targeting BCL-XL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGposmmgxVNCRLVg90H21EOLACvtfcHk0lhHPM6L93_VVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO&md5=b07678c880569acac428ec479f002974</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0668-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0668-z%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWiegand%26aufirst%3DJ.%2BS.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DC.%2BR.%26aulast%3DKuruvilla%26aufirst%3DV.%2BM.%26aulast%3DBaran%26aufirst%3DN.%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DY.-m.%26aulast%3DRogojina%26aufirst%3DA.%26aulast%3DHoughton%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DHromas%26aufirst%3DR.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DD.%26atitle%3DA%2520selective%2520BCL-XL%2520PROTAC%2520degrader%2520achieves%2520safe%2520and%2520potent%2520antitumor%2520activity%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D1938%26epage%3D1947%26doi%3D10.1038%2Fs41591-019-0668-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class mitogen-activated protein kinase kinase 1/2 degrader</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10911</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01528</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01528" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10897-10911&author=J.+Weiauthor=J.+Huauthor=L.+Wangauthor=L.+Xieauthor=M.+S.+Jinauthor=X.+Chenauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+a+first-in-class+mitogen-activated+protein+kinase+kinase+1%2F2+degrader&doi=10.1021%2Facs.jmedchem.9b01528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader</span></div><div class="casAuthors">Wei, Jieli; Hu, Jianping; Wang, Li; Xie, Ling; Jin, Margaret S.; Chen, Xian; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10897-10911</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the "gatekeepers" of the ERK signaling output with redundant roles in controlling ERK activity.  Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs.  However, acquired resistance to MEK1/2 inhibitors has been obsd. in patients and new therapeutic strategies are needed to overcome the resistance.  Here, the authors report a first-in-class degrader of MEK1/2, MS432 (I), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells.  It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its neg. control MS432N and its effect was phenocopied by MEK1/2 knockdown.  Compd. I was highly selective for MEK1/2 in global proteomic profiling studies.  It was also bioavailable in mice and can be used for in vivo efficacy studies.  The authors provide two well-characterized chem. tools to the biomedical community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e5sg33oDYrVg90H21EOLACvtfcHk0lhHPM6L93_VVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK&md5=2c293b6b87990d6b814ec1dc88034f19</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520mitogen-activated%2520protein%2520kinase%2520kinase%25201%252F2%2520degrader%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10897%26epage%3D10911%26doi%3D10.1021%2Facs.jmedchem.9b01528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of SIAIS178 as an effective BCR-ABL degrader by recruiting von Hippel-Lindau (VHL) E3 ubiquitin ligase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9281</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9281-9298&author=Q.+Zhaoauthor=C.+Renauthor=L.+Liuauthor=J.+Chenauthor=Y.+Shaoauthor=N.+Sunauthor=R.+Sunauthor=Y.+Kongauthor=X.+Dingauthor=X.+Zhangauthor=Y.+Xuauthor=B.+Yangauthor=Q.+Yinauthor=X.+Yangauthor=B.+Jiang&title=Discovery+of+SIAIS178+as+an+effective+BCR-ABL+degrader+by+recruiting+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase&doi=10.1021%2Facs.jmedchem.9b01264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of SIAIS178 as an Effective BCR-ABL Degrader by Recruiting Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span></div><div class="casAuthors">Zhao, Quanju; Ren, Chaowei; Liu, Linyi; Chen, Jinju; Shao, Yubao; Sun, Ning; Sun, Renhong; Kong, Ying; Ding, Xinyu; Zhang, Xianfang; Xu, Youwei; Yang, Bei; Yin, Qianqian; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9281-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncogenic fusion protein BCR-ABL is the driving force of leukemogenesis in chronic myeloid leukemia (CML).  Despite great progress for CML treatment through application of tyrosine kinase inhibitors (TKIs) against BCR-ABL, long-term drug administration and clin. resistance continue to be an issue.  Herein, we described the design, synthesis, and evaluation of novel proteolysis-targeting chimeric (PROTAC) small mols. targeting BCR-ABL which connect dasatinib and VHL E3 ubiquitin ligase ligand by extensive optimization of linkers.  Our efforts have yielded SIAIS178 (19), which induces proper interaction between BCR-ABL and VHL ligase leading to effective degrdn. of BCR-ABL protein, achieves significant growth inhibition of BCR-ABL+ leukemic cells in vitro, and induces substantial tumor regression against K562 xenograft tumors in vivo.  In addn., SIAIS178 also degrades several clin. relevant resistance-conferring mutations.  Our data indicate that SIAIS178 as efficacious BCR-ABL degrader warrants extensive further investigation for the treatment of BCR-ABL+ leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLBFfujSx6ZbVg90H21EOLACvtfcHk0lhnGfHGuzllTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN&md5=50ace274dbc6ef3931f86504a1be93a7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01264%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DYin%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520SIAIS178%2520as%2520an%2520effective%2520BCR-ABL%2520degrader%2520by%2520recruiting%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9281%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.9b01264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tovell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shpiro, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crafter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of SGK3-PROTAC1, an isoform specific SGK3 kinase PROTAC degrader</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2024</span>– <span class="NLM_lpage">2034</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00505</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00505" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1KksLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2024-2034&author=H.+Tovellauthor=A.+Testaauthor=H.+Zhouauthor=N.+Shpiroauthor=C.+Crafterauthor=A.+Ciulliauthor=D.+R.+Alessi&title=Design+and+characterization+of+SGK3-PROTAC1%2C+an+isoform+specific+SGK3+kinase+PROTAC+degrader&doi=10.1021%2Facschembio.9b00505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of SGK3-PROTAC1, an Isoform Specific SGK3 Kinase PROTAC Degrader</span></div><div class="casAuthors">Tovell, Hannah; Testa, Andrea; Zhou, Houjiang; Shpiro, Natalia; Crafter, Claire; Ciulli, Alessio; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2024-2034</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SGK3 is a PX domain contg. protein kinase activated at endosomes downstream of Class 1 and 3 PI3K family members by growth factors and oncogenic mutations.  SGK3 plays a key role in mediating resistance of breast cancer cells to Class 1 PI3K or Akt inhibitors, by substituting for loss of Akt activity and restoring proliferative pathways such as mTORC1 signaling.  It is therefore crit. to develop tools to potently target SGK3, and obstruct its role in inhibitor resistance.  Here we describe the development of SGK3-PROTAC1 (I), a PROTAC conjugate of the 308-R SGK inhibitor with the VH032 VHL binding ligand, targeting SGK3 for degrdn.  0.3 μM I induced 50% degrdn. of endogenous SGK3 within 2 h, with maximal 80% degrdn. obsd. within 8 h, accompanied by a loss of phosphorylation of NDRG1, an SGK3 substrate.  I did not degrade closely related SGK1 and SGK2 isoforms that are nevertheless engaged and inhibited by 308-R.  Proteomic anal. revealed that SGK3 was the only cellular protein whose cellular levels were significantly reduced following treatment with I.  Low doses of I (0.1-0.3 μM) restored sensitivity of SGK3 dependent ZR-75-1 and CAMA-1 breast cancer cells to Akt (AZD5363) and PI3K (GDC0941) inhibitors, whereas the cis epimer analog incapable of binding to the VHL E3 ligase had no impact.  I suppressed proliferation of ZR-75-1 and CAMA-1 cancer cell lines treated with a PI3K inhibitor (GDC0941) more effectively than could be achieved by a conventional SGK isoform inhibitor (14H).  This work underscores the benefit of the PROTAC approach in targeting protein kinase signaling pathways with greater efficacy and selectivity than can be achieved with conventional inhibitors.  I will be an important reagent to explore the roles of the SGK3 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCZDBO0-RuDrVg90H21EOLACvtfcHk0lhnGfHGuzllTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1KksLnN&md5=83f84ea9aba235a129c4cc4919888c6f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00505%26sid%3Dliteratum%253Aachs%26aulast%3DTovell%26aufirst%3DH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DShpiro%26aufirst%3DN.%26aulast%3DCrafter%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520SGK3-PROTAC1%252C%2520an%2520isoform%2520specific%2520SGK3%2520kinase%2520PROTAC%2520degrader%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2024%26epage%3D2034%26doi%3D10.1021%2Facschembio.9b00505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedmueller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstabl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenhöfer-Wölfer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wöhrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechens, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen-Hughes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0294-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41589-019-0294-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=31178587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=672-680&author=W.+Farnabyauthor=M.+Koeglauthor=M.+J.+Royauthor=C.+Whitworthauthor=E.+Diersauthor=N.+Trainorauthor=D.+Zollmanauthor=S.+Steurerauthor=J.+Karolyi-Oezguerauthor=C.+Riedmuellerauthor=T.+Gmaschitzauthor=J.+Wachterauthor=C.+Dankauthor=M.+Galantauthor=B.+Sharpsauthor=K.+Rumpelauthor=E.+Traxlerauthor=T.+Gerstbergerauthor=R.+Schnitzerauthor=O.+Petermannauthor=P.+Grebauthor=H.+Weinstablauthor=G.+Baderauthor=A.+Zoephelauthor=A.+Weiss-Puxbaumauthor=K.+Ehrenh%C3%B6fer-W%C3%B6lferauthor=S.+W%C3%B6hrleauthor=G.+Boehmeltauthor=J.+Rinnenthalauthor=H.+Arnhofauthor=N.+Wiechensauthor=M.-Y.+Wuauthor=T.+Owen-Hughesauthor=P.+Ettmayerauthor=M.+Pearsonauthor=D.+B.+McConnellauthor=A.+Ciulli&title=BAF+complex+vulnerabilities+in+cancer+demonstrated+via+structure-based+PROTAC+design&doi=10.1038%2Fs41589-019-0294-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span></div><div class="casAuthors">Farnaby, William; Koegl, Manfred; Roy, Michael J.; Whitworth, Claire; Diers, Emelyne; Trainor, Nicole; Zollman, David; Steurer, Steffen; Karolyi-Oezguer, Jale; Riedmueller, Carina; Gmaschitz, Teresa; Wachter, Johannes; Dank, Christian; Galant, Michael; Sharps, Bernadette; Rumpel, Klaus; Traxler, Elisabeth; Gerstberger, Thomas; Schnitzer, Renate; Petermann, Oliver; Greb, Peter; Weinstabl, Harald; Bader, Gerd; Zoephel, Andreas; Weiss-Puxbaum, Alexander; Ehrenhofer-Wolfer, Katharina; Wohrle, Simon; Boehmelt, Guido; Rinnenthal, Joerg; Arnhof, Heribert; Wiechens, Nicola; Wu, Meng-Ying; Owen-Hughes, Tom; Ettmayer, Peter; Pearson, Mark; McConnell, Darryl B.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeting subunits of BAF/PBAF chromatin remodeling complexes has been proposed as an approach to exploit cancer vulnerabilities.  Here, we develop proteolysis targeting chimera (PROTAC) degraders of the BAF ATPase subunits SMARCA2 and SMARCA4 using a bromodomain ligand and recruitment of the E3 ubiquitin ligase VHL.  High-resoln. ternary complex crystal structures and biophys. investigation guided rational and efficient optimization toward ACBI1(I), a potent and cooperative degrader of SMARCA2, SMARCA4 and PBRM1.  I induced anti-proliferative effects and cell death caused by SMARCA2 depletion in SMARCA4 mutant cancer cells, and in acute myeloid leukemia cells dependent on SMARCA4 ATPase activity.  These findings exemplify a successful biophysics- and structure-based PROTAC design approach to degrade high profile drug targets, and pave the way toward new therapeutics for the treatment of tumors sensitive to the loss of BAF complex ATPases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEh4pux4Bs7Vg90H21EOLACvtfcHk0lhnGfHGuzllTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ&md5=e95fad12c727aedcf08aed9320dae905</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0294-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0294-6%26sid%3Dliteratum%253Aachs%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DDiers%26aufirst%3DE.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DZollman%26aufirst%3DD.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DRiedmueller%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWachter%26aufirst%3DJ.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DWeinstabl%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DEhrenh%25C3%25B6fer-W%25C3%25B6lfer%26aufirst%3DK.%26aulast%3DW%25C3%25B6hrle%26aufirst%3DS.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DWiechens%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DM.-Y.%26aulast%3DOwen-Hughes%26aufirst%3DT.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBAF%2520complex%2520vulnerabilities%2520in%2520cancer%2520demonstrated%2520via%2520structure-based%2520PROTAC%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D672%26epage%3D680%26doi%3D10.1038%2Fs41589-019-0294-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoppi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Iterative design and optimization of initially inactive proteolysis targeting chimeras (PROTACs) identify VZ185 as a potent, fast, and selective von Hippel-Lindau (VHL) based dual degrader probe of BRD9 and BRD7</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=699-726&author=V.+Zoppiauthor=S.+J.+Hughesauthor=C.+Maniaciauthor=A.+Testaauthor=T.+Gmaschitzauthor=C.+Wieshoferauthor=M.+Koeglauthor=K.+M.+Richingauthor=D.+L.+Danielsauthor=A.+Spallarossaauthor=A.+Ciulli&title=Iterative+design+and+optimization+of+initially+inactive+proteolysis+targeting+chimeras+%28PROTACs%29+identify+VZ185+as+a+potent%2C+fast%2C+and+selective+von+Hippel-Lindau+%28VHL%29+based+dual+degrader+probe+of+BRD9+and+BRD7&doi=10.1021%2Facs.jmedchem.8b01413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span></div><div class="casAuthors">Zoppi, Vittoria; Hughes, Scott J.; Maniaci, Chiara; Testa, Andrea; Gmaschitz, Teresa; Wieshofer, Corinna; Koegl, Manfred; Riching, Kristin M.; Daniels, Danette L.; Spallarossa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">699-726</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Developing PROTACs to redirect the ubiquitination activity of E3 ligases and potently degrade a target protein within cells can be a lengthy and unpredictable process, and it remains unclear whether any combination of E3 and target might be productive for degrdn.  The authors describe a probe-quality degrader for a ligase-target pair deemed unsuitable: the von Hippel-Lindau (VHL) and BRD9, a bromodomain-contg. subunit of the SWI/SNF chromatin remodeling complex BAF.  VHL-based degraders could be optimized from suboptimal compds. in two rounds by systematically varying conjugation patterns and linkers and monitoring cellular degrdn. activities, kinetic profiles, and ubiquitination, as well as ternary complex formation thermodn.  The emerged structure-activity relations guided the discovery of VZ185, a potent, fast, and selective degrader of BRD9 and of its close homolog BRD7.  The authors' findings qualify a new chem. tool for BRD7/9 knockdown and provide a roadmap for PROTAC development against seemingly incompatible target-ligase combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPnajotKvqLVg90H21EOLACvtfcHk0ljzT6Ak8S26BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN&md5=be7938c26807a2535b7f9b7de8ebfbb0</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01413%26sid%3Dliteratum%253Aachs%26aulast%3DZoppi%26aufirst%3DV.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIterative%2520design%2520and%2520optimization%2520of%2520initially%2520inactive%2520proteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520identify%2520VZ185%2520as%2520a%2520potent%252C%2520fast%252C%2520and%2520selective%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520based%2520dual%2520degrader%2520probe%2520of%2520BRD9%2520and%2520BRD7%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D699%26epage%3D726%26doi%3D10.1021%2Facs.jmedchem.8b01413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gechijian, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Functional TRIM24 degrader via conjugation of ineffectual bromodomain and VHL ligands</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0010-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41589-018-0010-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=29507391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=405-412&author=L.+N.+Gechijianauthor=D.+L.+Buckleyauthor=M.+A.+Lawlorauthor=J.+M.+Reyesauthor=J.+Paulkauthor=C.+J.+Ottauthor=G.+E.+Winterauthor=M.+A.+Erbauthor=T.+G.+Scottauthor=M.+Xuauthor=H.-S.+Seoauthor=S.+Dhe-Paganonauthor=N.+P.+Kwiatkowskiauthor=J.+A.+Perryauthor=J.+Qiauthor=N.+S.+Grayauthor=J.+E.+Bradner&title=Functional+TRIM24+degrader+via+conjugation+of+ineffectual+bromodomain+and+VHL+ligands&doi=10.1038%2Fs41589-018-0010-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Functional TRIM24 degrader via conjugation of ineffectual bromodomain and VHL ligands</span></div><div class="casAuthors">Gechijian, Lara N.; Buckley, Dennis L.; Lawlor, Matthew A.; Reyes, Jaime M.; Paulk, Joshiawa; Ott, Christopher J.; Winter, Georg E.; Erb, Michael A.; Scott, Thomas G.; Xu, Mousheng; Seo, Hyuk-Soo; Dhe-Paganon, Sirano; Kwiatkowski, Nicholas P.; Perry, Jennifer A.; Qi, Jun; Gray, Nathanael S.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">405-412</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The addressable pocket of a protein is often not functionally relevant in disease.  This is true for the multidomain, bromodomain-contg. transcriptional regulator TRIM24.  TRIM24 has been posited as a dependency in numerous cancers, yet potent and selective ligands for the TRIM24 bromodomain do not exert effective anti-proliferative responses.  We therefore repositioned these probes as targeting features for heterobifunctional protein degraders.  Recruitment of the VHL E3 ubiquitin ligase by dTRIM24 elicits potent and selective degrdn. of TRIM24.  Using dTRIM24 to probe TRIM24 function, we characterize the dynamic genome-wide consequences of TRIM24 loss on chromatin localization and gene control.  Further, we identify TRIM24 as a novel dependency in acute leukemia.  Pairwise study of TRIM24 degrdn. vs. bromodomain inhibition reveals enhanced anti-proliferative response from degrdn.  We offer dTRIM24 as a chem. probe of an emerging cancer dependency, and establish a path forward for numerous selective yet ineffectual ligands for proteins of therapeutic interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod-DnZMlk25LVg90H21EOLACvtfcHk0ljzT6Ak8S26BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFGmurg%253D&md5=f11dfc9a2edc34c632d6d9b6e886bdf4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0010-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0010-y%26sid%3Dliteratum%253Aachs%26aulast%3DGechijian%26aufirst%3DL.%2BN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DLawlor%26aufirst%3DM.%2BA.%26aulast%3DReyes%26aufirst%3DJ.%2BM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DKwiatkowski%26aufirst%3DN.%2BP.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DFunctional%2520TRIM24%2520degrader%2520via%2520conjugation%2520of%2520ineffectual%2520bromodomain%2520and%2520VHL%2520ligands%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D405%26epage%3D412%26doi%3D10.1038%2Fs41589-018-0010-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of von Hippel-Lindau-recruiting proteolysis targeting chimeras (PROTACs) of TANK-binding kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+characterization+of+von+Hippel-Lindau-recruiting+proteolysis+targeting+chimeras+%28PROTACs%29+of+TANK-binding+kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0ljTJYHSP2j3zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520von%2520Hippel-Lindau-recruiting%2520proteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520TANK-binding%2520kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of highly potent and efficient PROTAC degraders of androgen receptor (AR) by employing weak binding affinity VHL E3 ligase ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11218</span>– <span class="NLM_lpage">11231</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01393</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01393" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit12rtbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11218-11231&author=X.+Hanauthor=L.+Zhaoauthor=W.+Xiangauthor=C.+Qinauthor=B.+Miaoauthor=T.+Xuauthor=M.+Wangauthor=C.-Y.+Yangauthor=K.+Chinnaswamyauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+highly+potent+and+efficient+PROTAC+degraders+of+androgen+receptor+%28AR%29+by+employing+weak+binding+affinity+VHL+E3+ligase+ligands&doi=10.1021%2Facs.jmedchem.9b01393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Highly Potent and Efficient PROTAC Degraders of Androgen Receptor (AR) by Employing Weak Binding Affinity VHL E3 Ligase Ligands</span></div><div class="casAuthors">Han, Xin; Zhao, Lijie; Xiang, Weiguo; Qin, Chong; Miao, Bukeyan; Xu, Tianfeng; Wang, Mi; Yang, Chao-Yie; Chinnaswamy, Krishnapriya; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11218-11231</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Androgen receptor (AR) is a validated therapeutic target for the treatment of metastatic castration-resistant prostate cancer (mCRPC).  We report herein our design, synthesis, and biol. characterization of highly potent small-mol. proteolysis targeting chimera (PROTAC) AR degraders using a potent AR antagonist and E3 ligase ligands with weak binding affinities to VHL protein.  Our study resulted in the discovery of 11 (ARD-266), which effectively induces degrdn. of AR protein in AR-pos. (AR+) LNCaP, VCaP, and 22Rv1 prostate cancer cell lines with DC50 values of 0.2-1 nM.  ARD-266 is capable of reducing the AR protein level by >95% in these AR+ prostate cancer cell lines and effectively reduces AR-regulated gene expression suppression.  For the first time, we demonstrated that an E3 ligand with micromolar binding affinity to its E3 ligase complex can be successfully employed for the design of highly potent and efficient PROTAC degraders and this finding may have a significant implication for the field of PROTAC research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM96gs2yAlzLVg90H21EOLACvtfcHk0ljTJYHSP2j3zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit12rtbvE&md5=92e4f17c75b488749fced14476552c07</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01393%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520highly%2520potent%2520and%2520efficient%2520PROTAC%2520degraders%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520by%2520employing%2520weak%2520binding%2520affinity%2520VHL%2520E3%2520ligase%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11218%26epage%3D11231%26doi%3D10.1021%2Facs.jmedchem.9b01393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wurz, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellamaggiore, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span> <span> </span><span class="NLM_article-title">A “click chemistry platform” for the rapid synthesis of bispecific molecules for inducing protein degradation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01781</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01781" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Gls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=453-461&author=R.+P.+Wurzauthor=K.+Dellamaggioreauthor=H.+Douauthor=N.+Javierauthor=M.-C.+Loauthor=J.+D.+McCarterauthor=D.+Mohlauthor=C.+Sastriauthor=J.+R.+Lipfordauthor=V.+J.+Cee&title=A+%E2%80%9Cclick+chemistry+platform%E2%80%9D+for+the+rapid+synthesis+of+bispecific+molecules+for+inducing+protein+degradation&doi=10.1021%2Facs.jmedchem.6b01781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A "Click Chemistry Platform" for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation</span></div><div class="casAuthors">Wurz, Ryan P.; Dellamaggiore, Ken; Dou, Hannah; Javier, Noelle; Lo, Mei-Chu; McCarter, John D.; Mohl, Dane; Sastri, Christine; Lipford, J. Russell; Cee, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">453-461</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) are bispecific mols. contg. a target protein binder and an ubiquitin ligase binder connected by a linker.  By recruiting an ubiquitin ligase to a target protein, PROTACs promote ubiquitination and proteasomal degrdn. of the target protein.  The generation of effective PROTACs depends on the nature of the protein/ligase ligand pair, linkage site, linker length, and linker compn., all of which have been difficult to address in a systematic way.  Herein, the authors describe a "click chem." approach for the synthesis of PROTACs.  The authors demonstrate the utility of this approach with the bromodomain and extra terminal domain-4 (BRD4) ligand JQ-1 (3) and ligase binders targeting cereblon (CRBN) and Von Hippel-Lindau (VHL) proteins.  An AlphaScreen proximity assay was used to det. the ability of PROTACs to form the ternary ligase-PROTAC-target protein complex and a MSD assay to measure cellular degrdn. of the target protein promoted by PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpooapLJ2Zln7Vg90H21EOLACvtfcHk0ljTJYHSP2j3zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Gls7w%253D&md5=090c569ae1f079ab6504348aabec8909</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01781%26sid%3Dliteratum%253Aachs%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DDellamaggiore%26aufirst%3DK.%26aulast%3DDou%26aufirst%3DH.%26aulast%3DJavier%26aufirst%3DN.%26aulast%3DLo%26aufirst%3DM.-C.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DMohl%26aufirst%3DD.%26aulast%3DSastri%26aufirst%3DC.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26atitle%3DA%2520%25E2%2580%259Cclick%2520chemistry%2520platform%25E2%2580%259D%2520for%2520the%2520rapid%2520synthesis%2520of%2520bispecific%2520molecules%2520for%2520inducing%2520protein%2520degradation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D453%26epage%3D461%26doi%3D10.1021%2Facs.jmedchem.6b01781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+androgen+receptor+%28AR%29+for+the+treatment+of+prostate+cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0lj4TbRZrC0J-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520for%2520the%2520treatment%2520of%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.-Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lj4TbRZrC0J-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lj4TbRZrC0J-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beutner, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickey, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span> <span> </span><span class="NLM_article-title">TCFH-NMI: direct cccess to N-acyl imidazoliums for challenging amide bond formations</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4218</span>– <span class="NLM_lpage">4222</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.8b01591</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.8b01591" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ajsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=4218-4222&author=G.+L.+Beutnerauthor=I.+S.+Youngauthor=M.+L.+Daviesauthor=M.+R.+Hickeyauthor=H.+Parkauthor=J.+M.+Stevensauthor=Q.+Ye&title=TCFH-NMI%3A+direct+cccess+to+N-acyl+imidazoliums+for+challenging+amide+bond+formations&doi=10.1021%2Facs.orglett.8b01591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">TCFH-NMI: Direct Access to N-Acyl Imidazoliums for Challenging Amide Bond Formations</span></div><div class="casAuthors">Beutner, Gregory L.; Young, Ian S.; Davies, Merrill L.; Hickey, Matthew R.; Park, Hyunsoo; Stevens, Jason M.; Ye, Qingmei</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4218-4222</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Challenging couplings of hindered carboxylic acids with non-nucleophilic amines to form amide bonds can be accomplished in high yields, and in many cases, with complete retention of the adjacent stereogenic centers using the combination of N,N,N',N'-tetramethylchloroformamidinium hexafluorophosphate (TCFH) and N-methylimidazole (NMI).  This method allows for in situ generation of highly reactive acyl imidazolium ions, which have been demonstrated to be intermediates in the reaction.  The reagent delivers high reactivity similar to acid chlorides with the ease of use of modern uronium reagents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouk9CntOu0PrVg90H21EOLACvtfcHk0ligr_Y43STzzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ajsrrJ&md5=257505afd32cbac2425c2372facc0f42</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.8b01591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.8b01591%26sid%3Dliteratum%253Aachs%26aulast%3DBeutner%26aufirst%3DG.%2BL.%26aulast%3DYoung%26aufirst%3DI.%2BS.%26aulast%3DDavies%26aufirst%3DM.%2BL.%26aulast%3DHickey%26aufirst%3DM.%2BR.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DStevens%26aufirst%3DJ.%2BM.%26aulast%3DYe%26aufirst%3DQ.%26atitle%3DTCFH-NMI%253A%2520direct%2520cccess%2520to%2520N-acyl%2520imidazoliums%2520for%2520challenging%2520amide%2520bond%2520formations%26jtitle%3DOrg.%2520Lett.%26date%3D2018%26volume%3D20%26spage%3D4218%26epage%3D4222%26doi%3D10.1021%2Facs.orglett.8b01591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argintaru, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, H.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaRochelle, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiguchi, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span> <span> </span><span class="NLM_article-title">Dual allosteric inhibition of SHP2 phosphatase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">656</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00980</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00980" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjslOgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=647-656&author=M.+Fodorauthor=E.+Priceauthor=P.+Wangauthor=H.+Luauthor=A.+Argintaruauthor=Z.+Chenauthor=M.+Glickauthor=H.-X.+Haoauthor=M.+Katoauthor=R.+Koenigauthor=J.+R.+LaRochelleauthor=G.+Liuauthor=E.+McNeillauthor=D.+Majumdarauthor=G.+A.+Nishiguchiauthor=L.+B.+Perezauthor=G.+Parisauthor=C.+M.+Quinnauthor=T.+Ramseyauthor=M.+Sendzikauthor=M.+D.+Shultzauthor=S.+L.+Williamsauthor=T.+Stamsauthor=S.+C.+Blacklowauthor=M.+G.+Ackerauthor=M.+J.+LaMarche&title=Dual+allosteric+inhibition+of+SHP2+phosphatase&doi=10.1021%2Facschembio.7b00980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Dual allosteric inhibition of SHP2 phosphatase</span></div><div class="casAuthors">Fodor, Michelle; Price, Edmund; Wang, Ping; Lu, Hengyu; Argintaru, Andreea; Chen, Zhouliang; Glick, Meir; Hao, Huai-Xiang; Kato, Mitsunori; Koenig, Robert; La Rochelle, Jonathan R.; Liu, Gang; McNeill, Eric; Majumdar, Dyuti; Nishiguchi, Gisele A.; Perez, Lawrence B.; Paris, Gregory; Quinn, Christopher M.; Ramsey, Timothy; Sendzik, Martin; Shultz, Michael David; Williams, Sarah L.; Stams, Travis; Blacklow, Stephen C.; Acker, Michael G.; LaMarche, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">647-656</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2 is a cytoplasmic protein tyrosine phosphatase encoded by the PTPN11 gene and is involved in cell proliferation, differentiation, and survival.  Recently, we reported an allosteric mechanism of inhibition that stabilizes the auto-inhibited conformation of SHP2.  SHP099 (1) was identified and characterized as a moderately potent, orally bioavailable, allosteric small mol. inhibitor, which binds to a tunnel-like pocket formed by the confluence of three domains of SHP2.  In this report, we describe further screening strategies that enabled the identification of a second, distinct small mol. allosteric site.  SHP244 (2) was identified as a weak inhibitor of SHP2 with modest thermal stabilization of the enzyme.  X-ray crystallog. revealed that 2 binds and stabilizes the inactive, closed conformation of SHP2, at a distinct, previously unexplored binding site-a cleft formed at the interface of the N-terminal SH2 and PTP domains.  Derivatization of 2 using structure-based design resulted in an increase in SHP2 thermal stabilization, biochem. inhibition, and subsequent MAPK pathway modulation.  Downregulation of DUSP6 mRNA, a downstream MAPK pathway marker, was obsd. in KYSE-520 cancer cells.  Remarkably, simultaneous occupation of both allosteric sites by 1 and 2 was possible, as characterized by cooperative biochem. inhibition expts. and X-ray crystallog.  Combining an allosteric site 1 inhibitor with an allosteric site 2 inhibitor led to enhanced pharmacol. pathway inhibition in cells.  This work illustrates a rare example of dual allosteric targeted protein inhibition, demonstrates screening methodol. and tactics to identify allosteric inhibitors, and enables further interrogation of SHP2 in cancer and related pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWdw7pTS6sOLVg90H21EOLACvtfcHk0liULpkkjKgNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjslOgtg%253D%253D&md5=8e4de4c353b9fc196c81e1cb51e367df</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00980%26sid%3Dliteratum%253Aachs%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DPrice%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DArgintaru%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DHao%26aufirst%3DH.-X.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKoenig%26aufirst%3DR.%26aulast%3DLaRochelle%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DMcNeill%26aufirst%3DE.%26aulast%3DMajumdar%26aufirst%3DD.%26aulast%3DNishiguchi%26aufirst%3DG.%2BA.%26aulast%3DPerez%26aufirst%3DL.%2BB.%26aulast%3DParis%26aufirst%3DG.%26aulast%3DQuinn%26aufirst%3DC.%2BM.%26aulast%3DRamsey%26aufirst%3DT.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DWilliams%26aufirst%3DS.%2BL.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DAcker%26aufirst%3DM.%2BG.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3DDual%2520allosteric%2520inhibition%2520of%2520SHP2%2520phosphatase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D647%26epage%3D656%26doi%3D10.1021%2Facschembio.7b00980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Amy M. Barrios</span>. </span><span class="cited-content_cbyCitation_article-title">PTPs: Degrading the Undruggable. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (14)
                                     , 7508-7509. <a href="https://doi.org/10.1021/acs.jmedchem.0c01000" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01000</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01000%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPTPs%25253A%252BDegrading%252Bthe%252BUndruggable%26aulast%3DBarrios%26aufirst%3DAmy%2BM.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11062020%26date%3D02072020%26volume%3D63%26issue%3D14%26spage%3D7508%26epage%3D7509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Majid  Momeny</span>, <span class="hlFld-ContribAuthor ">Tiina  Arsiola</span>, <span class="hlFld-ContribAuthor ">Jukka  Westermarck</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer stem cell phosphatases. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2021,</strong> <em>478 </em>
                                    (14)
                                     , 2899-2920. <a href="https://doi.org/10.1042/BCJ20210254" title="DOI URL">https://doi.org/10.1042/BCJ20210254</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20210254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20210254%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DCancer%252Bstem%252Bcell%252Bphosphatases%26aulast%3DMomeny%26aufirst%3DMajid%26date%3D2021%26date%3D2021%26volume%3D478%26issue%3D14%26spage%3D2899%26epage%3D2920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Yunhua  Peng</span>, <span class="hlFld-ContribAuthor ">Wenyi  Wei</span>. </span><span class="cited-content_cbyCitation_article-title">Light-Controllable PROTACs for Temporospatial Control of Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.678077" title="DOI URL">https://doi.org/10.3389/fcell.2021.678077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.678077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.678077%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DLight-Controllable%252BPROTACs%252Bfor%252BTemporospatial%252BControl%252Bof%252BProtein%252BDegradation%26aulast%3DLiu%26aufirst%3DJing%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aimo  Kannt</span>, <span class="hlFld-ContribAuthor ">Ivan  Đikić</span>. </span><span class="cited-content_cbyCitation_article-title">Expanding the arsenal of E3 ubiquitin ligases for proximity-induced protein degradation. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 1014-1031. <a href="https://doi.org/10.1016/j.chembiol.2021.04.007" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.04.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.04.007%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DExpanding%252Bthe%252Barsenal%252Bof%252BE3%252Bubiquitin%252Bligases%252Bfor%252Bproximity-induced%252Bprotein%252Bdegradation%26aulast%3DKannt%26aufirst%3DAimo%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D1014%26epage%3D1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Guo</span>, <span class="hlFld-ContribAuthor ">Yaping  Xu</span>, <span class="hlFld-ContribAuthor ">Xiaowu  Dong</span>, <span class="hlFld-ContribAuthor ">Jianjun  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Cross the Undruggable Barrier, the Development of SHP2 Inhibitors: From Catalytic Site Inhibitors to Allosteric Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2021,</strong> <em>6 </em>
                                    (22)
                                     , 5504-5523. <a href="https://doi.org/10.1002/slct.202100186" title="DOI URL">https://doi.org/10.1002/slct.202100186</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202100186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202100186%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DCross%252Bthe%252BUndruggable%252BBarrier%25252C%252Bthe%252BDevelopment%252Bof%252BSHP2%252BInhibitors%25253A%252BFrom%252BCatalytic%252BSite%252BInhibitors%252Bto%252BAllosteric%252BInhibitors%26aulast%3DGuo%26aufirst%3DYu%26date%3D2021%26date%3D2021%26volume%3D6%26issue%3D22%26spage%3D5504%26epage%3D5523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D. Lucas  Kerr</span>, <span class="hlFld-ContribAuthor ">Franziska  Haderk</span>, <span class="hlFld-ContribAuthor ">Trever G.  Bivona</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric SHP2 inhibitors in cancer: Targeting the intersection of RAS, resistance, and the immune microenvironment. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>62 </em>, 1-12. <a href="https://doi.org/10.1016/j.cbpa.2020.11.007" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.11.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.11.007%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DAllosteric%252BSHP2%252Binhibitors%252Bin%252Bcancer%25253A%252BTargeting%252Bthe%252Bintersection%252Bof%252BRAS%25252C%252Bresistance%25252C%252Band%252Bthe%252Bimmune%252Bmicroenvironment%26aulast%3DKerr%26aufirst%3DD.%2BLucas%26date%3D2021%26volume%3D62%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangbo  Yang</span>, <span class="hlFld-ContribAuthor ">Zhijia  Wang</span>, <span class="hlFld-ContribAuthor ">Yuan  Pei</span>, <span class="hlFld-ContribAuthor ">Ning  Song</span>, <span class="hlFld-ContribAuthor ">Lei  Xu</span>, <span class="hlFld-ContribAuthor ">Bo  Feng</span>, <span class="hlFld-ContribAuthor ">Hanlin  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Xiaobei  Hu</span>, <span class="hlFld-ContribAuthor ">Xiaohui  Qiu</span>, <span class="hlFld-ContribAuthor ">Huijin  Feng</span>, <span class="hlFld-ContribAuthor ">Yaxi  Yang</span>, <span class="hlFld-ContribAuthor ">Yubo  Zhou</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Bing  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of thalidomide-based PROTAC small molecules as the highly efficient SHP2 degraders. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>218 </em>, 113341. <a href="https://doi.org/10.1016/j.ejmech.2021.113341" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113341</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113341%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bthalidomide-based%252BPROTAC%252Bsmall%252Bmolecules%252Bas%252Bthe%252Bhighly%252Befficient%252BSHP2%252Bdegraders%26aulast%3DYang%26aufirst%3DXiangbo%26date%3D2021%26volume%3D218%26spage%3D113341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julia P.  Vainonen</span>, <span class="hlFld-ContribAuthor ">Majid  Momeny</span>, <span class="hlFld-ContribAuthor ">Jukka  Westermarck</span>. </span><span class="cited-content_cbyCitation_article-title">Druggable cancer phosphatases. </span><span class="cited-content_cbyCitation_journal-name">Science Translational Medicine</span><span> <strong>2021,</strong> <em>13 </em>
                                    (588)
                                     , eabe2967. <a href="https://doi.org/10.1126/scitranslmed.abe2967" title="DOI URL">https://doi.org/10.1126/scitranslmed.abe2967</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/scitranslmed.abe2967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fscitranslmed.abe2967%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Translational%2520Medicine%26atitle%3DDruggable%252Bcancer%252Bphosphatases%26aulast%3DVainonen%26aufirst%3DJulia%2BP.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D588%26spage%3Deabe2967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lei  Dong</span>, <span class="hlFld-ContribAuthor ">Da  Han</span>, <span class="hlFld-ContribAuthor ">Xinyi  Meng</span>, <span class="hlFld-ContribAuthor ">Mengchuan  Xu</span>, <span class="hlFld-ContribAuthor ">Chuwen  Zheng</span>, <span class="hlFld-ContribAuthor ">Qin  Xia</span>. </span><span class="cited-content_cbyCitation_article-title">Activating Mutation of SHP2 Establishes a Tumorigenic Phonotype Through Cell-Autonomous and Non-Cell-Autonomous Mechanisms. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.630712" title="DOI URL">https://doi.org/10.3389/fcell.2021.630712</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.630712&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.630712%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DActivating%252BMutation%252Bof%252BSHP2%252BEstablishes%252Ba%252BTumorigenic%252BPhonotype%252BThrough%252BCell-Autonomous%252Band%252BNon-Cell-Autonomous%252BMechanisms%26aulast%3DDong%26aufirst%3DLei%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yihui  Song</span>, <span class="hlFld-ContribAuthor ">Min  Zhao</span>, <span class="hlFld-ContribAuthor ">Yahong  Wu</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">A multifunctional cross-validation high-throughput screening protocol enabling the discovery of new SHP2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (3)
                                     , 750-762. <a href="https://doi.org/10.1016/j.apsb.2020.10.021" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.10.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.10.021%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DA%252Bmultifunctional%252Bcross-validation%252Bhigh-throughput%252Bscreening%252Bprotocol%252Benabling%252Bthe%252Bdiscovery%252Bof%252Bnew%252BSHP2%252Binhibitors%26aulast%3DSong%26aufirst%3DYihui%26date%3D2021%26volume%3D11%26issue%3D3%26spage%3D750%26epage%3D762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meng  Liu</span>, <span class="hlFld-ContribAuthor ">Shan  Gao</span>, <span class="hlFld-ContribAuthor ">Reham M.  Elhassan</span>, <span class="hlFld-ContribAuthor ">Xuben  Hou</span>, <span class="hlFld-ContribAuthor ">Hao  Fang</span>. </span><span class="cited-content_cbyCitation_article-title">Strategies to overcome drug resistance using SHP2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>27 </em><a href="https://doi.org/10.1016/j.apsb.2021.03.037" title="DOI URL">https://doi.org/10.1016/j.apsb.2021.03.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2021.03.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2021.03.037%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DStrategies%252Bto%252Bovercome%252Bdrug%252Bresistance%252Busing%252BSHP2%252Binhibitors%26aulast%3DLiu%26aufirst%3DMeng%26date%3D2021%26volume%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yang  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaohe  Yang</span>, <span class="hlFld-ContribAuthor ">Yali  Wang</span>, <span class="hlFld-ContribAuthor ">Yueying  Yang</span>, <span class="hlFld-ContribAuthor ">Dejuan  Sun</span>, <span class="hlFld-ContribAuthor ">Hua  Li</span>, <span class="hlFld-ContribAuthor ">Lixia  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting SHP2 as a therapeutic strategy for inflammatory diseases. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>214 </em>, 113264. <a href="https://doi.org/10.1016/j.ejmech.2021.113264" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113264</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113264%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BSHP2%252Bas%252Ba%252Btherapeutic%252Bstrategy%252Bfor%252Binflammatory%252Bdiseases%26aulast%3DLiu%26aufirst%3DYang%26date%3D2021%26volume%3D214%26spage%3D113264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyan cheng</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (PROTAC) in drug discovery paradigm: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112981. <a href="https://doi.org/10.1016/j.ejmech.2020.112981" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112981%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bin%252Bdrug%252Bdiscovery%252Bparadigm%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26volume%3D210%26spage%3D112981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.  Maneiro</span>, <span class="hlFld-ContribAuthor ">E.  De Vita</span>, <span class="hlFld-ContribAuthor ">D.  Conole</span>, <span class="hlFld-ContribAuthor ">C.S.  Kounde</span>, <span class="hlFld-ContribAuthor ">Q.  Zhang</span>, <span class="hlFld-ContribAuthor ">E.W.  Tate</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs, molecular glues and bifunctionals from bench to bedside: Unlocking the clinical potential of catalytic drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/bs.pmch.2021.01.002" title="DOI URL">https://doi.org/10.1016/bs.pmch.2021.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2021.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2021.01.002%26sid%3Dliteratum%253Aachs%26atitle%3DPROTACs%25252C%252Bmolecular%252Bglues%252Band%252Bbifunctionals%252Bfrom%252Bbench%252Bto%252Bbedside%25253A%252BUnlocking%252Bthe%252Bclinical%252Bpotential%252Bof%252Bcatalytic%252Bdrugs%26aulast%3DManeiro%26aufirst%3DM.%26date%3D2021%26pub%3DElsevier%26date%3D2021%26volume%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Melissa J.  Buskes</span>, <span class="hlFld-ContribAuthor ">Maria-Jesus  Blanco</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of Cross-Coupling Reactions in Drug Discovery and Development. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (15)
                                     , 3493. <a href="https://doi.org/10.3390/molecules25153493" title="DOI URL">https://doi.org/10.3390/molecules25153493</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25153493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25153493%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DImpact%252Bof%252BCross-Coupling%252BReactions%252Bin%252BDrug%252BDiscovery%252Band%252BDevelopment%26aulast%3DBuskes%26aufirst%3DMelissa%2BJ.%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D15%26spage%3D3493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of three previously reported SHP2 inhibitors SHP099 (<b>1</b>), SHP389 (<b>2</b>) and <b>3</b>, and two designed SHP2 inhibitors (<b>4</b>, <b>5</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Co-crystal structure of SHP2 in a complex with SHP099 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EHR">5EHR</a>), and (B and C) modeled structures of SHP2 in a complex with <b>4</b> or <b>5</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Further evaluation of SHP2-D26 for its degradation activity in the KYSE520 and MV4;11 cell lines. (A–D) Dose-dependent SHP2 degradation in the KYSE520 and MV4;11 cell lines, with SHP099 included as the inhibitor control. Cells were treated with SHP2-D26 for 12 h. SHP2 protein was examined by Western blotting, and the protein level was quantified by densitometry and normalized to the corresponding density of the GAPDH protein. (E,F). Degradation kinetics of SHP2-D26 in the KYSE520 and MV4;11 cell lines. Cells were treated with 100 nM of SHP2-D26 for different times. SHP2 protein was examined by Western blotting, and the GAPDH protein was used as the loading control. (G) KYSE520 cell line was treated with VHL-1 ligand <b>6</b> (10 μM), compound <b>4</b> (at 10 μM), MLN4924 (0.5 μM), or MG132 (3 μm) for 1 h and then treated with SHP-D26 (0.1 μM) for 3 h. The protein level of SHP2 was examined by Western blotting, and the GAPDH protein was used as the loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>26</b> (SHP2-D26) suppresses the p-ERK pathway. KYSE520 or MV4;11 cells were treated as indicated with SHP2-D26 or SHP099 for 48 h. The protein levels of SHP2 (Bethyl Lab. A301–544), ERK (#9102, CST), and phospho-ERK (#4370, CST) determined by Western blotting. GAPDH was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cell growth inhibition in KYSE520 and MV4;11 cells treated with SHP2 degrader <b>26</b> (SHP2-D26) and SHP2 inhibitors SHP099, <b>4</b> and <b>5</b>. KYSE520 and MV4;11 cells were treated with indicated doses for 4 days, and cell viability was determined by a colorimetric WST-8 assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the VHL Ligand (<b>6</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, NaHCO<sub>3</sub>, EtOAc/H<sub>2</sub>O, 2 h; (b) 4-methylthiazole, Pd(OAc)<sub>2</sub>, KOAc, DMA, 90 °C, 12 h; (c) 4 N HCl dioxane/MeOH, rt, 4h; (d) HATU, DIPEA, DMF, 0 °C to RT, 12 h; and (e) TFA, DCM, rt, 4h.</p></p></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the Key Intermediate (<b>40</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-butylthiol, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 120 °C, 36 h; (b) conc. HCl, 80 °C, 6 h; (c) K<sub>3</sub>PO<sub>4</sub>, CuI, 1,10-phenanthroline, dioxane, 90 °C, 16 h; and (d) DIPEA, DMSO, 100 °C, 3 h.</p></p></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>7</b>–<b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>6</b>, HATU, DIPEA, DMF, 0 °C to RT, 1 h; (b) 4 N HCl dioxane/MeOH, rt; (c) <b>40</b>, DIPEA, DCM, 0 °C, 2 h; (d) LiOH·H<sub>2</sub>O, THF, MeOH, H<sub>2</sub>O, RT, 3 h; (e) HATU, DIPEA, DMF, 0 °C to RT, 1 h; and (f) TFA, DCM, rt, 2 h.</p></p></figure><figure data-id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>20</b>, <b>21</b>, and <b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>6</b>, HATU, DIPEA, DMF, 0 °C to rt, 1 h; (b) 4 N HCl dioxane, 2h; (c) <b>46b</b>, HATU, DIPEA, DMF, 0 °C to rt, 1 h; and (d) TFA, DCM, rt, 2h.</p></p></figure><figure data-id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>23</b>, <b>24</b>, and <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TCFH, NMI, MeCN/THF, rt, 6 h; (b) piperidine, MeCN, rt, 2 h; (c) K<sub>2</sub>CO<sub>3</sub>, KI, DMF, 60 °C, 8 h; (d) HATU, DIPEA, DMF, 0 °C to rt, 1 h; (e) TFA, DCM, rt, 2h; and (f) NaOH, THF-H<sub>2</sub>O.</p></p></figure><figure data-id="sch6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/medium/jm0c00471_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>26</b>, <b>27, 28</b>, and <b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-14/acs.jmedchem.0c00471/20200716/images/large/jm0c00471_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00471&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, NaI, MeCN, 60 °C, 16 h; (b) LiOH.H<sub>2</sub>O, THF, MeOH, H<sub>2</sub>O, rt, 3 h (c) <b>6</b>, HATU, DIPEA, DMF, 0 °C to rt, 1 h; (d) 4 N HCl dioxane, 2 h; (e) <b>46b</b>, HATU, DIPEA, DMF, 0 °C to rt, 1 h; (f) TFA, DCM, rt, 2 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i84">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45670" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45670" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 62 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tartaglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehler, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zampino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Burgt, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crosby, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ion, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalidas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patton, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucherlapati, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelb, B. D.</span></span> <span> </span><span class="NLM_article-title">Mutations in PTPN11, encoding the protein tyrosine phosphatase SHP-2, cause Noonan syndrome</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">468</span>, <span class="refDoi"> DOI: 10.1038/ng772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fng772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=11704759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFGjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=465-468&author=M.+Tartagliaauthor=E.+L.+Mehlerauthor=R.+Goldbergauthor=G.+Zampinoauthor=H.+G.+Brunnerauthor=H.+Kremerauthor=I.+van+der+Burgtauthor=A.+H.+Crosbyauthor=A.+Ionauthor=S.+Jefferyauthor=K.+Kalidasauthor=M.+A.+Pattonauthor=R.+S.+Kucherlapatiauthor=B.+D.+Gelb&title=Mutations+in+PTPN11%2C+encoding+the+protein+tyrosine+phosphatase+SHP-2%2C+cause+Noonan+syndrome&doi=10.1038%2Fng772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in PTPN11, encoding the protein tyrosine phosphatase SHP-2, cause Noonan syndrome</span></div><div class="casAuthors">Tartaglia, Marco; Mehler, Ernest L.; Goldberg, Rosalie; Zampino, Giuseppe; Brunner, Han G.; Kremer, Hannie; van der Burgt, Ineke; Crosby, Andrew H.; Ion, Andra; Jeffery, Steve; Kalidas, Kamini; Patton, Michael A.; Kucherlapati, Raju S.; Gelb, Bruce D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">465-468</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Noonan syndrome (MIM 163950) is an autosomal dominant disorder characterized by dysmorphic facial features, proportionate short stature and heart disease (most commonly pulmonic stenosis and hypertrophic cardiomyopathy).  Webbed neck, chest deformity, cryptorchidism, mental retardation and bleeding diathesis also are frequently assocd. with this disease.  This syndrome is relatively common, with an estd. incidence of 1 in 1000-2500 live births.  It has been mapped to a 5-cM region (N-SH2) on chromosome 12q24.1, and genetic heterogeneity has also been documented.  Here we show that missense mutations in PTPN11 (MIM 176876)-a gene encoding the nonreceptor protein tyrosine phosphatase SHP-2, which contains two Src homol. 2 (SH2) domains-cause Noonan syndrome and account for more than 50% of the cases that we examd.  All PTPN11 missense mutations cluster in interacting portions of the amino N-SH2 domain and the phosphotyrosine phosphatase domains, which are involved in switching the protein between its inactive and active conformations.  An energetics-based structural anal. of two N-SH2 mutants indicates that in these mutants there may be a significant shift of the equil. favoring the active conformation.  This implies that they are gain-of-function changes and that the pathogenesis of Noonan syndrome arises from excessive SHP-2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKg7QISHW0bbVg90H21EOLACvtfcHk0lgikkyVhrJ8rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFGjt7w%253D&md5=c8225c08340f9446cd58c249de42e515</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fng772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng772%26sid%3Dliteratum%253Aachs%26aulast%3DTartaglia%26aufirst%3DM.%26aulast%3DMehler%26aufirst%3DE.%2BL.%26aulast%3DGoldberg%26aufirst%3DR.%26aulast%3DZampino%26aufirst%3DG.%26aulast%3DBrunner%26aufirst%3DH.%2BG.%26aulast%3DKremer%26aufirst%3DH.%26aulast%3Dvan%2Bder%2BBurgt%26aufirst%3DI.%26aulast%3DCrosby%26aufirst%3DA.%2BH.%26aulast%3DIon%26aufirst%3DA.%26aulast%3DJeffery%26aufirst%3DS.%26aulast%3DKalidas%26aufirst%3DK.%26aulast%3DPatton%26aufirst%3DM.%2BA.%26aulast%3DKucherlapati%26aufirst%3DR.%2BS.%26aulast%3DGelb%26aufirst%3DB.%2BD.%26atitle%3DMutations%2520in%2520PTPN11%252C%2520encoding%2520the%2520protein%2520tyrosine%2520phosphatase%2520SHP-2%252C%2520cause%2520Noonan%2520syndrome%26jtitle%3DNat.%2520Genet.%26date%3D2001%26volume%3D29%26spage%3D465%26epage%3D468%26doi%3D10.1038%2Fng772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kontaridis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span> <span> </span><span class="NLM_article-title">PTPN11 (Shp2) mutations in LEOPARD syndrome have dominant negative, not activating, effects</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">6785</span>– <span class="NLM_lpage">6792</span>, <span class="refDoi"> DOI: 10.1074/jbc.M513068200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1074%2Fjbc.M513068200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=16377799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFOgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=6785-6792&author=M.+I.+Kontaridisauthor=K.+D.+Swansonauthor=F.+S.+Davidauthor=D.+Barfordauthor=B.+G.+Neel&title=PTPN11+%28Shp2%29+mutations+in+LEOPARD+syndrome+have+dominant+negative%2C+not+activating%2C+effects&doi=10.1074%2Fjbc.M513068200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PTPN11 (Shp2) Mutations in LEOPARD Syndrome Have Dominant Negative, Not Activating, Effects</span></div><div class="casAuthors">Kontaridis, Maria I.; Swanson, Kenneth D.; David, Frank S.; Barford, David; Neel, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6785-6792</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Multiple lentigines/LEOPARD syndrome (LS) is a rare, autosomal dominant disorder characterized by Lentigines, ECG abnormalities, Ocular hypertelorism, Pulmonic valvular stenosis, Abnormalities of genitalia, Retardation of growth, and Deafness.  Like the more common Noonan syndrome (NS), LS is caused by germ line missense mutations in PTPN11, encoding the protein-tyrosine phosphatase Shp2.  Enzymol., structural, cell biol., and mouse genetic studies indicate that NS is caused by gain-of-function PTPN11 mutations.  Because NS and LS share several features, LS has been viewed as an NS variant.  We examd. a panel of LS mutants, including the two most common alleles.  Surprisingly, we found that in marked contrast to NS, LS mutants are catalytically defective and act as dominant neg. mutations that interfere with growth factor/Erk-mitogen-activated protein kinase-mediated signaling.  Mol. modeling and biochem. studies suggest that LS mutations contort the Shp2 catalytic domain and result in open, inactive forms of Shp2.  Our results establish that the pathogenesis of LS and NS is distinct and suggest that these disorders should be distinguished by mutational anal. rather than clin. presentation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAcRbFkVLdSLVg90H21EOLACvtfcHk0liMSO3xNqDreQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvFOgtrw%253D&md5=83adc77383a03535fa7c8eb92c34ca5c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M513068200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M513068200%26sid%3Dliteratum%253Aachs%26aulast%3DKontaridis%26aufirst%3DM.%2BI.%26aulast%3DSwanson%26aufirst%3DK.%2BD.%26aulast%3DDavid%26aufirst%3DF.%2BS.%26aulast%3DBarford%26aufirst%3DD.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26atitle%3DPTPN11%2520%2528Shp2%2529%2520mutations%2520in%2520LEOPARD%2520syndrome%2520have%2520dominant%2520negative%252C%2520not%2520activating%252C%2520effects%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D6785%26epage%3D6792%26doi%3D10.1074%2Fjbc.M513068200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tartaglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niemeyer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fragale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buechner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hählen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelb, B. D.</span></span> <span> </span><span class="NLM_article-title">Somatic mutations in PTPN11 in juvenile myelomonocytic leukemia, myelodysplastic syndromes and acute myeloid leukemia</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">148</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1038/ng1156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fng1156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=12717436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFSltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2003&pages=148-150&author=M.+Tartagliaauthor=C.+M.+Niemeyerauthor=A.+Fragaleauthor=X.+Songauthor=J.+Buechnerauthor=A.+Jungauthor=K.+H%C3%A4hlenauthor=H.+Hasleauthor=J.+D.+Lichtauthor=B.+D.+Gelb&title=Somatic+mutations+in+PTPN11+in+juvenile+myelomonocytic+leukemia%2C+myelodysplastic+syndromes+and+acute+myeloid+leukemia&doi=10.1038%2Fng1156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Somatic mutations in PTPN11 in juvenile myelomonocytic leukemia, myelodysplastic syndromes and acute myeloid leukemia</span></div><div class="casAuthors">Tartaglia, Marco; Niemeyer, Charlotte M.; Fragale, Alessandra; Song, Xiaoling; Buechner, Jochen; Jung, Andreas; Haehlen, Karel; Hasle, Henrik; Licht, Jonathan D.; Gelb, Bruce D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-150</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report here that individuals with Noonan syndrome and juvenile myelomonocytic leukemia (JMML) have germline mutations in PTPN11 and that somatic mutations in PTPN11 account for 34% of non-syndromic JMML.  Furthermore, we found mutations in PTPN11 in a small percentage of individuals with myelodysplastic syndrome (MDS) and de novo acute myeloid leukemia (AML).  Functional analyses documented that the two most common mutations in PTPN11 assocd. with JMML caused a gain of function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCwFhlUQDJQLVg90H21EOLACvtfcHk0liMSO3xNqDreQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFSltrY%253D&md5=67d6900b46520ee461330bed95cbc890</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fng1156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng1156%26sid%3Dliteratum%253Aachs%26aulast%3DTartaglia%26aufirst%3DM.%26aulast%3DNiemeyer%26aufirst%3DC.%2BM.%26aulast%3DFragale%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DBuechner%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DA.%26aulast%3DH%25C3%25A4hlen%26aufirst%3DK.%26aulast%3DHasle%26aufirst%3DH.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DGelb%26aufirst%3DB.%2BD.%26atitle%3DSomatic%2520mutations%2520in%2520PTPN11%2520in%2520juvenile%2520myelomonocytic%2520leukemia%252C%2520myelodysplastic%2520syndromes%2520and%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Genet.%26date%3D2003%26volume%3D34%26spage%3D148%26epage%3D150%26doi%3D10.1038%2Fng1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bentires-Alj, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halmos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maris, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugarbaker, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velculescu, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelstein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span> <span> </span><span class="NLM_article-title">Activating mutations of the noonan syndrome-associated SHP2/PTPN11 gene in human solid tumors and adult acute myelogenous leukemia</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">8816</span>– <span class="NLM_lpage">8820</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-04-1923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1158%2F0008-5472.CAN-04-1923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=15604238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOms7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=8816-8820&author=M.+Bentires-Aljauthor=J.+G.+Paezauthor=F.+S.+Davidauthor=H.+Keilhackauthor=B.+Halmosauthor=K.+Naokiauthor=J.+M.+Marisauthor=A.+Richardsonauthor=A.+Bardelliauthor=D.+J.+Sugarbakerauthor=W.+G.+Richardsauthor=J.+Duauthor=L.+Girardauthor=J.+D.+Minnaauthor=M.+L.+Lohauthor=D.+E.+Fisherauthor=V.+E.+Velculescuauthor=B.+Vogelsteinauthor=M.+Meyersonauthor=W.+R.+Sellersauthor=B.+G.+Neel&title=Activating+mutations+of+the+noonan+syndrome-associated+SHP2%2FPTPN11+gene+in+human+solid+tumors+and+adult+acute+myelogenous+leukemia&doi=10.1158%2F0008-5472.CAN-04-1923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Activating Mutations of the Noonan Syndrome-Associated SHP2/PTPN11 Gene in Human Solid Tumors and Adult Acute Myelogenous Leukemia</span></div><div class="casAuthors">Bentires-Alj, Mohamed; Paez, J. Guillermo; David, Frank S.; Keilhack, Heike; Halmos, Balazs; Naoki, Katsuhiko; Maris, John M.; Richardson, Andrea; Bardelli, Alberto; Sugarbaker, David J.; Richards, William G.; Du, Jinyan; Girard, Luc; Minna, John D.; Loh, Mignon L.; Fisher, David E.; Velculescu, Victor E.; Vogelstein, Bert; Meyerson, Matthew; Sellers, William R.; Neel, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8816-8820</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The SH2 domain-contg. protein-tyrosine phosphatase PTPN11 (Shp2) is required for normal development and is an essential component of signaling pathways initiated by growth factors, cytokines, and extracellular matrix.  In many of these pathways, Shp2 acts upstream of Ras.  About 50% of patients with Noonan syndrome have germ-line PTPN11 gain of function mutations.  Assocns. between Noonan syndrome and an increased risk of some malignancies, notably leukemia and neuroblastoma, have been reported, and recent data indicate that somatic PTPN11 mutations occur in children with sporadic juvenile myelomonocytic leukemia, myelodysplasic syndrome, B-cell acute lymphoblastic leukemia, and acute myelogenous leukemia (AML).  Juvenile myelomonocytic leukemia patients without PTPN11 mutations have either homozygotic NF-1 deletion or activating RAS mutations.  Given the role of Shp2 in Ras activation and the frequent mutation of RAS in human tumors, these data raise the possibility that PTPN11 mutations play a broader role in cancer.  The authors asked whether PTPN11 mutations occur in other malignancies in which activating RAS mutations occur at low but significant frequency.  Sequencing of PTPN11 from 13 different human neoplasms including breast, lung, gastric, and neuroblastoma tumors and adult AML and acute lymphoblastic leukemia revealed 11 missense mutations.  Five are known mutations predicted to result in an activated form of Shp2, whereas six are new mutations.  Biochem. anal. confirmed that several of the new mutations result in increased Shp2 activity.  The data demonstrate that mutations in PTPN11 occur at low frequency in several human cancers, esp. neuroblastoma and AML, and suggest that Shp2 may be a novel target for antineoplastic therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3taKvDtHXBLVg90H21EOLACvtfcHk0liMSO3xNqDreQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOms7fK&md5=f6f9fb8c6e9d5260901a03081db259c7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-04-1923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-04-1923%26sid%3Dliteratum%253Aachs%26aulast%3DBentires-Alj%26aufirst%3DM.%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DDavid%26aufirst%3DF.%2BS.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DHalmos%26aufirst%3DB.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DMaris%26aufirst%3DJ.%2BM.%26aulast%3DRichardson%26aufirst%3DA.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DSugarbaker%26aufirst%3DD.%2BJ.%26aulast%3DRichards%26aufirst%3DW.%2BG.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DGirard%26aufirst%3DL.%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DLoh%26aufirst%3DM.%2BL.%26aulast%3DFisher%26aufirst%3DD.%2BE.%26aulast%3DVelculescu%26aufirst%3DV.%2BE.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26atitle%3DActivating%2520mutations%2520of%2520the%2520noonan%2520syndrome-associated%2520SHP2%252FPTPN11%2520gene%2520in%2520human%2520solid%2520tumors%2520and%2520adult%2520acute%2520myelogenous%2520leukemia%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D8816%26epage%3D8820%26doi%3D10.1158%2F0008-5472.CAN-04-1923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Cancer Genome Atlas Research Network</span> <span> </span><span class="NLM_article-title">Comprehensive genomic characterization
defines human glioblastoma genes and core pathways</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>455</i></span>,  <span class="NLM_fpage">1061</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1038/nature07385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fnature07385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18772890" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=455&publication_year=2008&pages=1061-1068&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization%0Adefines+human+glioblastoma+genes+and+core+pathways&doi=10.1038%2Fnature07385"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature07385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07385%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%2520genomic%2520characterization%250Adefines%2520human%2520glioblastoma%2520genes%2520and%2520core%2520pathways%26jtitle%3DNature%26date%3D2008%26volume%3D455%26spage%3D1061%26epage%3D1068%26doi%3D10.1038%2Fnature07385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalaitzidis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span> <span> </span><span class="NLM_article-title">The tyrosine phosphatase Shp2 (PTPN11) in cancer</span>. <i>Cancer Metastasis Rev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1007/s10555-008-9126-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1007%2Fs10555-008-9126-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18286234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1Ckur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=179-192&author=G.+Chanauthor=D.+Kalaitzidisauthor=B.+G.+Neel&title=The+tyrosine+phosphatase+Shp2+%28PTPN11%29+in+cancer&doi=10.1007%2Fs10555-008-9126-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The tyrosine phosphatase Shp2 (PTPN11) in cancer</span></div><div class="casAuthors">Chan, Gordon; Kalaitzidis, Demetrios; Neel, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer and Metastasis Reviews</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">179-192</span>CODEN:
                <span class="NLM_cas:coden">CMRED4</span>;
        ISSN:<span class="NLM_cas:issn">0167-7659</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Diverse cellular processes are regulated by tyrosyl phosphorylation, which is controlled by protein-tyrosine kinases (PTKs) and protein-tyrosine phosphatases (PTPs).  De-regulated tyrosyl phosphorylation, evoked by gain-of-function mutations and/or over-expression of PTKs, contributes to the pathogenesis of many cancers and other human diseases.  PTPs, because they oppose the action of PTKs, had been considered to be prime suspects for potential tumor suppressor genes.  Surprisingly, few, if any, tumor suppressor PTPs have been identified.  However, the Src homol.-2 domain-contg. phosphatase Shp2 (encoded by PTPN11) is a bona fide proto-oncogene.  Germline mutations in PTPN11 cause Noonan and LEOPARD syndromes, whereas somatic PTPN11 mutations occur in several types of hematol. malignancies, most notably juvenile myelomonocytic leukemia and, more rarely, in solid tumors.  Shp2 also is an essential component in several other oncogene signaling pathways.  Elucidation of the events underlying Shp2-evoked transformation may provide new insights into oncogenic mechanisms and novel targets for anti-cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbJ8OYsBzvg7Vg90H21EOLACvtfcHk0ljjxXddQ1IxrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1Ckur4%253D&md5=fb514f60b97f8ff0dcc0dbd57cab2c9a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2Fs10555-008-9126-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10555-008-9126-y%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DG.%26aulast%3DKalaitzidis%26aufirst%3DD.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26atitle%3DThe%2520tyrosine%2520phosphatase%2520Shp2%2520%2528PTPN11%2529%2520in%2520cancer%26jtitle%3DCancer%2520Metastasis%2520Rev.%26date%3D2008%26volume%3D27%26spage%3D179%26epage%3D192%26doi%3D10.1007%2Fs10555-008-9126-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yomogita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatakeyama, M.</span></span> <span> </span><span class="NLM_article-title">Isolation of a distinct class of gain-of-function SHP-2 mutants with oncogenic RAS-like transforming activity from solid tumors</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3508</span>– <span class="NLM_lpage">3515</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1211019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fsj.onc.1211019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18223690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmslWmurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=3508-3515&author=D.+Miyamotoauthor=M.+Miyamotoauthor=A.+Takahashiauthor=Y.+Yomogitaauthor=H.+Higashiauthor=S.+Kondoauthor=M.+Hatakeyama&title=Isolation+of+a+distinct+class+of+gain-of-function+SHP-2+mutants+with+oncogenic+RAS-like+transforming+activity+from+solid+tumors&doi=10.1038%2Fsj.onc.1211019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of a distinct class of gain-of-function SHP-2 mutants with oncogenic RAS-like transforming activity from solid tumors</span></div><div class="casAuthors">Miyamoto, D.; Miyamoto, M.; Takahashi, A.; Yomogita, Y.; Higashi, H.; Kondo, S.; Hatakeyama, M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">3508-3515</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">SHP-2 protein tyrosine phosphatase plays an important role in activation of the RAS-dependent signaling.  Gain-of-function mutations in the PTPN11 gene, which encodes SHP-2, have been found in the leukemia-prone developmental disorder Noonan syndrome as well as sporadic childhood leukemias, indicating that SHP-2 is a bona fide human oncoprotein.  However, the role of SHP-2 mutations in non-hematol. malignancies remains obscure.  Here, we screened for PTPN11 mutations in primary solid tumors and identified a 1520C>A mutation that causes threonine-507 to lysine (T507K) substitution in the phosphatase domain of SHP-2 in a case of hepatocellular carcinoma.  T507K SHP-2 exhibited altered substrate specificity with slightly elevated basal phosphatase activity.  Upon expression in NIH3T3 cells, T507K SHP-2 induced transformed foci, which was not obsd. with wild type, Noonan-specific or leukemia-specific SHP-2.  Furthermore, NIH3T3 cells transformed by T507K SHP-2 showed anchorage-independent growth and developed tumors in nude mice.  These results indicate that quant. and/or qual. alteration in phosphatase activity dets. the transforming potential as well as target cell/tissue spectrum of individual SHP-2 mutants as oncoproteins.  Although rare in solid tumors, the identified T507K SHP-2 represents a distinct class of SHP-2 mutants with oncogenic RAS-like transforming activity, which could contribute to the development of solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwOMpOPEfD0rVg90H21EOLACvtfcHk0liUJbt44PnmAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmslWmurY%253D&md5=e0a0e2df71b6998a92b40bbba05ffef8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1211019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1211019%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DD.%26aulast%3DMiyamoto%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DA.%26aulast%3DYomogita%26aufirst%3DY.%26aulast%3DHigashi%26aufirst%3DH.%26aulast%3DKondo%26aufirst%3DS.%26aulast%3DHatakeyama%26aufirst%3DM.%26atitle%3DIsolation%2520of%2520a%2520distinct%2520class%2520of%2520gain-of-function%2520SHP-2%2520mutants%2520with%2520oncogenic%2520RAS-like%2520transforming%2520activity%2520from%2520solid%2520tumors%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D3508%26epage%3D3515%26doi%3D10.1038%2Fsj.onc.1211019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aceto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sausgruber, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinkhaus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidatzis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martiny-Baron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzarol, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Confalonieri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quarto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balwierz, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachkov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elledge, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Nimwegen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentires-Alj, M.</span></span> <span> </span><span class="NLM_article-title">Tyrosine phosphatase SHP2 promotes breast cancer progression and maintains tumor-initiating cells via activation of key transcription factors and a positive feedback signaling loop</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1038/nm.2645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fnm.2645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=22388088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1yisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=529-537&author=N.+Acetoauthor=N.+Sausgruberauthor=H.+Brinkhausauthor=D.+Gaidatzisauthor=G.+Martiny-Baronauthor=G.+Mazzarolauthor=S.+Confalonieriauthor=M.+Quartoauthor=G.+Huauthor=P.+J.+Balwierzauthor=M.+Pachkovauthor=S.+J.+Elledgeauthor=E.+van+Nimwegenauthor=M.+B.+Stadlerauthor=M.+Bentires-Alj&title=Tyrosine+phosphatase+SHP2+promotes+breast+cancer+progression+and+maintains+tumor-initiating+cells+via+activation+of+key+transcription+factors+and+a+positive+feedback+signaling+loop&doi=10.1038%2Fnm.2645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine phosphatase SHP2 promotes breast cancer progression and maintains tumor-initiating cells via activation of key transcription factors and a positive feedback signaling loop</span></div><div class="casAuthors">Aceto, Nicola; Sausgruber, Nina; Brinkhaus, Heike; Gaidatzis, Dimos; Martiny-Baron, Georg; Mazzarol, Giovanni; Confalonieri, Stefano; Quarto, Micaela; Hu, Guang; Balwierz, Piotr J.; Pachkov, Mikhail; Elledge, Stephen J.; van Nimwegen, Erik; Stadler, Michael B.; Bentires-Alj, Mohamed</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">529-537</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">New cancer therapies are likely to arise from an in-depth understanding of the signaling networks influencing tumor initiation, progression and metastasis.  We show a fundamental role for Src-homol. 2 domain-contg. phosphatase 2 (SHP2) in these processes in human epidermal growth factor receptor 2 (HER2)-pos. and triple-neg. breast cancers.  Knockdown of SHP2 eradicated breast tumor-initiating cells in xenograft models, and SHP2 depletion also prevented invasion in three-dimensional cultures and in a transductal invasion assay in vivo.  Notably, SHP2 knockdown in established breast tumors blocked their growth and reduced metastasis.  Mechanistically, SHP2 activated stemness-assocd. transcription factors, including v-myc myelocytomatosis viral oncogene homolog (c-Myc) and zinc finger E-box binding homeobox 1 (ZEB1), which resulted in the repression of let-7 microRNA and the expression of a set of 'SHP2 signature' genes.  We found these genes to be simultaneously activated in a large subset of human primary breast tumors that are assocd. with invasive behavior and poor prognosis.  These results provide new insights into the signaling cascades influencing tumor-initiating cells as well as a rationale for targeting SHP2 in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGBsjLtiOIWbVg90H21EOLACvtfcHk0liUJbt44PnmAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1yisrg%253D&md5=ea1d756b2f9d21771ca5f25d2dfda798</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnm.2645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2645%26sid%3Dliteratum%253Aachs%26aulast%3DAceto%26aufirst%3DN.%26aulast%3DSausgruber%26aufirst%3DN.%26aulast%3DBrinkhaus%26aufirst%3DH.%26aulast%3DGaidatzis%26aufirst%3DD.%26aulast%3DMartiny-Baron%26aufirst%3DG.%26aulast%3DMazzarol%26aufirst%3DG.%26aulast%3DConfalonieri%26aufirst%3DS.%26aulast%3DQuarto%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DG.%26aulast%3DBalwierz%26aufirst%3DP.%2BJ.%26aulast%3DPachkov%26aufirst%3DM.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26aulast%3Dvan%2BNimwegen%26aufirst%3DE.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DBentires-Alj%26aufirst%3DM.%26atitle%3DTyrosine%2520phosphatase%2520SHP2%2520promotes%2520breast%2520cancer%2520progression%2520and%2520maintains%2520tumor-initiating%2520cells%2520via%2520activation%2520of%2520key%2520transcription%2520factors%2520and%2520a%2520positive%2520feedback%2520signaling%2520loop%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D529%26epage%3D537%26doi%3D10.1038%2Fnm.2645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanafusa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasunaga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, E.</span></span> <span> </span><span class="NLM_article-title">Sprouty1 and Sprouty2 provide a control mechanism for the Ras/MAPK signalling pathway</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">850</span>– <span class="NLM_lpage">858</span>, <span class="refDoi"> DOI: 10.1038/ncb867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fncb867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=12402043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVKmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=850-858&author=H.+Hanafusaauthor=S.+Toriiauthor=T.+Yasunagaauthor=E.+Nishida&title=Sprouty1+and+Sprouty2+provide+a+control+mechanism+for+the+Ras%2FMAPK+signalling+pathway&doi=10.1038%2Fncb867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Sprouty1 and Sprouty2 provide a control mechanism for the Ras/MAPK signalling pathway</span></div><div class="casAuthors">Hanafusa, Hiroshi; Torii, Satoru; Yasunaga, Takayuki; Nishida, Eisuke</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">850-858</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sprouty (Spry) inhibits signaling by receptor tyrosine kinases; however, the mol. mechanism underlying this function has not been defined.  Here we show that after stimulation by growth factors Spry1 and Spry2 translocate to the plasma membrane and become phosphorylated on a conserved tyrosine.  Next, they bind to the adaptor protein Grb2 and inhibit the recruitment of the Grb2-Sos complex either to the fibroblast growth factor receptor (FGFR) docking adaptor protein FRS2 or to Shp2.  Membrane translocation of Spry is necessary for its phosphorylation, which is essential for its inhibitor activity.  A tyrosine-phosphorylated octapeptide derived from mouse Spry2 inhibits Grb2 from binding FRS2, Shp2 or mouse Spry2 in vitro and blocks activation of the extracellular-signal-regulated kinase (ERK) in cells stimulated by growth factor.  A non-phosphorylated Spry mutant cannot bind Grb2 and acts as a dominant neg., inducing prolonged activation of ERK in response to FGF and promoting the FGF-induced outgrowth of neurites in PC12 cells.  Our findings suggest that Spry functions in a neg. feedback mechanism in which its inhibitor activity is controlled rapidly and reversibly by post-translational mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpLO_810Tlg7Vg90H21EOLACvtfcHk0lhG1HWX1rjp6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVKmsw%253D%253D&md5=3f496bfc4a46a3e86d6ea1b8b489c04b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fncb867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb867%26sid%3Dliteratum%253Aachs%26aulast%3DHanafusa%26aufirst%3DH.%26aulast%3DTorii%26aufirst%3DS.%26aulast%3DYasunaga%26aufirst%3DT.%26aulast%3DNishida%26aufirst%3DE.%26atitle%3DSprouty1%2520and%2520Sprouty2%2520provide%2520a%2520control%2520mechanism%2520for%2520the%2520Ras%252FMAPK%2520signalling%2520pathway%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2002%26volume%3D4%26spage%3D850%26epage%3D858%26doi%3D10.1038%2Fncb867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agazie, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayman, M. J.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanism for a role of SHP2 in epidermal growth factor receptor signaling</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">7875</span>– <span class="NLM_lpage">7886</span>, <span class="refDoi"> DOI: 10.1128/MCB.23.21.7875-7886.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1128%2FMCB.23.21.7875-7886.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=14560030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosFOgtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=7875-7886&author=Y.+M.+Agazieauthor=M.+J.+Hayman&title=Molecular+mechanism+for+a+role+of+SHP2+in+epidermal+growth+factor+receptor+signaling&doi=10.1128%2FMCB.23.21.7875-7886.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanism for a role of SHP2 in epidermal growth factor receptor signaling</span></div><div class="casAuthors">Agazie, Yehenew M.; Hayman, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7875-7886</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Src homol. 2-contg. phosphotyrosine phosphatase (SHP2) is primarily a pos. effector of receptor tyrosine kinase signaling.  However, the mol. mechanism by which SHP2 effects its biol. function is unknown.  The authors provide evidence that defines the mol. mechanism and site of action of SHP2 in the epidermal growth factor-induced mitogenic pathway.  The authors demonstrate that SHP2 acts upstream of Ras and functions by increasing the half-life of activated Ras (GTP-Ras) in the cell by interfering with the process of Ras inactivation catalyzed by Ras GTPase-activating protein (RasGAP).  It does so by inhibition of tyrosine phosphorylation-dependent translocation of RasGAP to the plasma membrane, to its substrate (GTP-Ras) microdomain.  Inhibition is achieved through the dephosphorylation of RasGAP binding sites at the level of the plasma membrane.  The authors have identified Tyr 992 of the epidermal growth factor receptor (EGFR) to be one such site, since its mutation to Phe renders the EGFR refractory to the effect of dominant-neg. SHP2.  To the authors' knowledge, this is the first report to outline the site and mol. mechanism of action of SHP2 in EGFR signaling, which may also serve as a model to describe its role in other receptor tyrosine kinase signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA5SXQ45HAFrVg90H21EOLACvtfcHk0lhG1HWX1rjp6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosFOgtLo%253D&md5=23e83e2e6ea24b006daa799b15a95981</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.21.7875-7886.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.21.7875-7886.2003%26sid%3Dliteratum%253Aachs%26aulast%3DAgazie%26aufirst%3DY.%2BM.%26aulast%3DHayman%26aufirst%3DM.%2BJ.%26atitle%3DMolecular%2520mechanism%2520for%2520a%2520role%2520of%2520SHP2%2520in%2520epidermal%2520growth%2520factor%2520receptor%2520signaling%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D7875%26epage%3D7886%26doi%3D10.1128%2FMCB.23.21.7875-7886.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, C. K.</span></span> <span> </span><span class="NLM_article-title">Protein tyrosine phosphatases in the JAK/STAT pathway</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4925</span>– <span class="NLM_lpage">4932</span>, <span class="refDoi"> DOI: 10.2741/3051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.2741%2F3051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18508557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=4925-4932&author=D.+Xuauthor=C.+K.+Qu&title=Protein+tyrosine+phosphatases+in+the+JAK%2FSTAT+pathway&doi=10.2741%2F3051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Protein tyrosine phosphatases in the JAK/STAT pathway</span></div><div class="casAuthors">Xu, Dan; Qu, Cheng-Kui</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4925-4932</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  The Janus kinase (JAK)/signal transducer and activator of transcription (STAT) pathway is crucial in controlling cellular activities in response to extracellular cytokines.  Dysfunctions of the JAK/STAT pathway result in various hematopoietic and immune disorders.  The central events in regulating this pathway are tyrosine phosphorylation and dephosphorylation of the signaling components, which are carried out by protein tyrosine kinases and protein tyrosine phosphatases (PTP), resp.  Here, we review recent advances in the regulatory roles of PTPs, in particular, SHP2 phosphatase, in the JAK/STAT signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH1iYtqQyD_7Vg90H21EOLACvtfcHk0lhG1HWX1rjp6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOjtrs%253D&md5=d3f84b6306220659ef34e4bf43cab87e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2741%2F3051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F3051%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DD.%26aulast%3DQu%26aufirst%3DC.%2BK.%26atitle%3DProtein%2520tyrosine%2520phosphatases%2520in%2520the%2520JAK%252FSTAT%2520pathway%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2008%26volume%3D13%26spage%3D4925%26epage%3D4932%26doi%3D10.2741%2F3051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matozaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okazawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, H.</span></span> <span> </span><span class="NLM_article-title">Protein tyrosine phosphatase SHP-2: a proto-oncogene product that promotes Ras activation</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">1786</span>– <span class="NLM_lpage">1793</span>, <span class="refDoi"> DOI: 10.1111/j.1349-7006.2009.01257.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1111%2Fj.1349-7006.2009.01257.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=19622105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht12qs7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2009&pages=1786-1793&author=T.+Matozakiauthor=Y.+Murataauthor=Y.+Saitoauthor=H.+Okazawaauthor=H.+Ohnishi&title=Protein+tyrosine+phosphatase+SHP-2%3A+a+proto-oncogene+product+that+promotes+Ras+activation&doi=10.1111%2Fj.1349-7006.2009.01257.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Protein tyrosine phosphatase SHP-2: A protooncogene product that promotes Ras activation</span></div><div class="casAuthors">Matozaki, Takashi; Murata, Yoji; Saito, Yasuyuki; Okazawa, Hideki; Ohnishi, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1786-1793</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1347-9032</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  SHP-2 is a cytoplasmic protein tyrosine phosphatase (PTP) that contains two Src homol. 2 (SH2) domains.  Although PTPs are generally considered to be neg. regulators on the basis of their ability to oppose the effects of protein tyrosine kinases, SHP-2 is unusual in that it promotes the activation of the Ras-MAPK signaling pathway by receptors for various growth factors and cytokines.  The mol. basis for the activation of SHP-2 is also unique: In the basal state, the NH2-terminal SH2 domain of SHP-2 interacts with the PTP domain, resulting in autoinhibition of PTP activity; the binding of SHP-2 via its SH2 domains to tyrosine-phosphorylated growth factor receptors or docking proteins, however, results in disruption of this intramol. interaction, leading to exposure of the PTP domain and catalytic activation.  Indeed, SHP-2 proteins with artificial mutations in the NH2-terminal SH2 domain have been shown to act as dominant active mutants in vitro.  Such activating mutations of PTPN11 (human SHP-2 gene) were subsequently identified in individuals with Noonan syndrome, a human developmental disorder that is sometimes assocd. with juvenile myelomonocytic leukemia.  Furthermore, somatic mutations of PTPN11 were found to be assocd. with pediatric leukemia.  SHP-2 is also thought to participate in the development of other malignant disorders, but in a manner independent of such activating mutations.  Biochem. and functional studies of SHP-2 and genetic anal. of PTPN11 in human disorders have thus converged to provide new insight into the pathogenesis of cancer as well as potential new targets for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqagV6o3osbrVg90H21EOLACvtfcHk0lgqZZ-4PNa-9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht12qs7vI&md5=d4f294641af7b4283f0ed30ce0208bee</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2009.01257.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2009.01257.x%26sid%3Dliteratum%253Aachs%26aulast%3DMatozaki%26aufirst%3DT.%26aulast%3DMurata%26aufirst%3DY.%26aulast%3DSaito%26aufirst%3DY.%26aulast%3DOkazawa%26aufirst%3DH.%26aulast%3DOhnishi%26aufirst%3DH.%26atitle%3DProtein%2520tyrosine%2520phosphatase%2520SHP-2%253A%2520a%2520proto-oncogene%2520product%2520that%2520promotes%2520Ras%2520activation%26jtitle%3DCancer%2520Sci.%26date%3D2009%26volume%3D100%26spage%3D1786%26epage%3D1793%26doi%3D10.1111%2Fj.1349-7006.2009.01257.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heir, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alamsahebpour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raught, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zadeh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohh, M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of SHP2-mediated dephosphorylation of Ras suppresses oncogenesis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8859</span>, <span class="refDoi"> DOI: 10.1038/ncomms9859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fncomms9859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=26617336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWksLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8859&author=S.+Bundaauthor=K.+Burrellauthor=P.+Heirauthor=L.+Zengauthor=A.+Alamsahebpourauthor=Y.+Kanoauthor=B.+Raughtauthor=Z.-Y.+Zhangauthor=G.+Zadehauthor=M.+Ohh&title=Inhibition+of+SHP2-mediated+dephosphorylation+of+Ras+suppresses+oncogenesis&doi=10.1038%2Fncomms9859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of SHP2-mediated dephosphorylation of Ras suppresses oncogenesis</span></div><div class="casAuthors">Bunda, Severa; Burrell, Kelly; Heir, Pardeep; Zeng, Lifan; Alamsahebpour, Amir; Kano, Yoshihito; Raught, Brian; Zhang, Zhong-Yin; Zadeh, Gelareh; Ohh, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8859</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ras is phosphorylated on a conserved tyrosine at position 32 within the switch I region via Src kinase.  This phosphorylation inhibits the binding of effector Raf while promoting the engagement of GTPase-activating protein (GAP) and GTP hydrolysis.  Here we identify SHP2 as the ubiquitously expressed tyrosine phosphatase that preferentially binds to and dephosphorylates Ras to increase its assocn. with Raf and activate downstream proliferative Ras/ERK/MAPK signalling.  In comparison to normal astrocytes, SHP2 activity is elevated in astrocytes isolated from glioblastoma multiforme (GBM)-prone H-Ras(12V) knock-in mice as well as in glioma cell lines and patient-derived GBM specimens exhibiting hyperactive Ras.  Pharmacol. inhibition of SHP2 activity attenuates cell proliferation, soft-agar colony formation and orthotopic GBM growth in NOD/SCID mice and decelerates the progression of low-grade astrocytoma to GBM in a spontaneous transgenic glioma mouse model.  These results identify SHP2 as a direct activator of Ras and a potential therapeutic target for cancers driven by a previously 'undruggable' oncogenic or hyperactive Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhWi25AJiphrVg90H21EOLACvtfcHk0lgqZZ-4PNa-9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWksLbP&md5=8524ad9cc6f2e85abaff6b07c698f464</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fncomms9859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9859%26sid%3Dliteratum%253Aachs%26aulast%3DBunda%26aufirst%3DS.%26aulast%3DBurrell%26aufirst%3DK.%26aulast%3DHeir%26aufirst%3DP.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DAlamsahebpour%26aufirst%3DA.%26aulast%3DKano%26aufirst%3DY.%26aulast%3DRaught%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DZ.-Y.%26aulast%3DZadeh%26aufirst%3DG.%26aulast%3DOhh%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520SHP2-mediated%2520dephosphorylation%2520of%2520Ras%2520suppresses%2520oncogenesis%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8859%26doi%3D10.1038%2Fncomms9859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chemnitz, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichols, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">June, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, J. L.</span></span> <span> </span><span class="NLM_article-title">SHP-1 and SHP-2 associate with immunoreceptor tyrosine-based switch motif of programmed death 1 upon primary human T cell stimulation, but only receptor ligation prevents T cell activation</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">945</span>– <span class="NLM_lpage">954</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.173.2.945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.4049%2Fjimmunol.173.2.945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=15240681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlsVGhurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2004&pages=945-954&author=J.+M.+Chemnitzauthor=R.+V.+Parryauthor=K.+E.+Nicholsauthor=C.+H.+Juneauthor=J.+L.+Riley&title=SHP-1+and+SHP-2+associate+with+immunoreceptor+tyrosine-based+switch+motif+of+programmed+death+1+upon+primary+human+T+cell+stimulation%2C+but+only+receptor+ligation+prevents+T+cell+activation&doi=10.4049%2Fjimmunol.173.2.945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">SHP-1 and SHP-2 Associate with Immunoreceptor Tyrosine-Based Switch Motif of Programmed Death 1 upon Primary Human T Cell Stimulation, but Only Receptor Ligation Prevents T Cell Activation</span></div><div class="casAuthors">Chemnitz, Jens M.; Parry, Richard V.; Nichols, Kim E.; June, Carl H.; Riley, James L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">945-954</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">To study the cis- and trans-acting factors that mediate programmed death 1 (PD-1) signaling in primary human CD4 T cells, the authors constructed a chimeric mol. consisting of the murine CD28 extracellular domain and human PD-1 cytoplasmic tail.  When introduced into CD4 T cells, this construct mimics the activity of endogenous PD-1 in terms of its ability to suppress T cell expansion and cytokine prodn.  The cytoplasmic tail of PD-1 contains two structural motifs, an ITIM and an immunoreceptor tyrosine-based switch motif (ITSM).  Mutation of the ITIM had little effect on PD-1 signaling or functional activity.  In contrast, mutation of the ITSM abrogated the ability of PD-1 to block cytokine synthesis and to limit T cell expansion.  Further biochem. analyses revealed that the ability of PD-1 to block T cell activation correlated with recruitment of Src homol. region 2 domain-contg. phosphatase-1 (SHP-1) and SHP-2, and not the adaptor Src homol. 2 domain-contg. mol. 1A, to the ITSM domain.  In TCR-stimulated T cells, SHP-2 assocd. with PD-1, even in the absence of PD-1 engagement.  Despite this interaction, the ability of PD-1 to block T cell activation required receptor ligation, suggesting that colocalization of PD-1 with CD3 and/or CD28 may be necessary for inhibition of T cell activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLiPMfGJ1GeLVg90H21EOLACvtfcHk0lgqZZ-4PNa-9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlsVGhurc%253D&md5=204c373c1a728fc94c10db9de7804f50</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.173.2.945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.173.2.945%26sid%3Dliteratum%253Aachs%26aulast%3DChemnitz%26aufirst%3DJ.%2BM.%26aulast%3DParry%26aufirst%3DR.%2BV.%26aulast%3DNichols%26aufirst%3DK.%2BE.%26aulast%3DJune%26aufirst%3DC.%2BH.%26aulast%3DRiley%26aufirst%3DJ.%2BL.%26atitle%3DSHP-1%2520and%2520SHP-2%2520associate%2520with%2520immunoreceptor%2520tyrosine-based%2520switch%2520motif%2520of%2520programmed%2520death%25201%2520upon%2520primary%2520human%2520T%2520cell%2520stimulation%252C%2520but%2520only%2520receptor%2520ligation%2520prevents%2520T%2520cell%2520activation%26jtitle%3DJ.%2520Immunol.%26date%3D2004%26volume%3D173%26spage%3D945%26epage%3D954%26doi%3D10.4049%2Fjimmunol.173.2.945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jie, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildener-Leapman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trivedi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferris, R. L.</span></span> <span> </span><span class="NLM_article-title">PD-1/SHP-2 inhibits Tc1/Th1 phenotypic responses and the activation of T cells in the tumor microenvironment</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">508</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-14-1215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1158%2F0008-5472.CAN-14-1215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=25480946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVenurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=508-518&author=J.+Liauthor=H.+B.+Jieauthor=Y.+Leiauthor=N.+Gildener-Leapmanauthor=S.+Trivediauthor=T.+Greenauthor=L.+P.+Kaneauthor=R.+L.+Ferris&title=PD-1%2FSHP-2+inhibits+Tc1%2FTh1+phenotypic+responses+and+the+activation+of+T+cells+in+the+tumor+microenvironment&doi=10.1158%2F0008-5472.CAN-14-1215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">PD-1/SHP-2 Inhibits Tc1/Th1 Phenotypic Responses and the Activation of T Cells in the Tumor Microenvironment</span></div><div class="casAuthors">Li, Jing; Jie, Hyun-Bae; Lei, Yu; Gildener-Leapman, Neil; Trivedi, Sumita; Green, Tony; Kane, Lawrence P.; Ferris, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">508-518</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">This study addresses the mechanisms through which PD-1 antibody therapy restores the ability of the immune system to recognize and attack tumors by skewing intratumoral T cells toward a more potent activation phenotype that is assocd. with enhanced signaling and a reversal of their immunosuppressed phenotype.  Immune rejection of tumors is mediated by IFNγ prodn. and T-cell cytolytic activity.  These processes are impeded by PD-1, a coinhibitory mol. expressed on T cells that is elevated in tumor-infiltrating lymphocytes (TIL).  PD-1 elevation may reflect T-cell exhaustion marked by decreased proliferation, prodn. of type I cytokines, and poor cytolytic activity.  Although anti-PD-1 antibodies enhance IFNγ secretion after stimulation of the T-cell receptor (TCR), the mechanistic link between PD-1 and its effects on T-cell help (Tc1/Th1 skewing) remains unclear.  In prospectively collected cancer tissues, we found that TIL exhibited dampened Tc1/Th1 skewing and activation compared with peripheral blood lymphocytes (PBL).  When PD-1 bound its ligand PD-L1, we obsd. a marked suppression of crit. TCR target genes and Th1 cytokines.  Conversely, PD-1 blockade reversed these suppressive effects of PD-1:PD-L1 ligation.  We also found that the TCR-regulated phosphatase SHP-2 was expressed higher in TIL than in PBL, tightly correlating with PD-1 expression and neg. regulation of TCR target genes.  Overall, these results defined a PD-1/SHP-2/STAT1/T-bet signaling axis mediating the suppressive effects of PD-1 on Th1 immunity at tumor sites.  Our findings argue that PD-1 or SHP-2 blockade will be sufficient to restore robust Th1 immunity and T-cell activation and thereby reverse immunosuppression in the tumor microenvironment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe6jAWHNatvLVg90H21EOLACvtfcHk0li2am5BFNRYyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVenurY%253D&md5=695007f0715b5eee2b7e47deaf87ae23</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-1215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-1215%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJie%26aufirst%3DH.%2BB.%26aulast%3DLei%26aufirst%3DY.%26aulast%3DGildener-Leapman%26aufirst%3DN.%26aulast%3DTrivedi%26aufirst%3DS.%26aulast%3DGreen%26aufirst%3DT.%26aulast%3DKane%26aufirst%3DL.%2BP.%26aulast%3DFerris%26aufirst%3DR.%2BL.%26atitle%3DPD-1%252FSHP-2%2520inhibits%2520Tc1%252FTh1%2520phenotypic%2520responses%2520and%2520the%2520activation%2520of%2520T%2520cells%2520in%2520the%2520tumor%2520microenvironment%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D508%26epage%3D518%26doi%3D10.1158%2F0008-5472.CAN-14-1215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hui, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallweber, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasmal, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vale, R. D.</span></span> <span> </span><span class="NLM_article-title">T cell costimulatory receptor CD28 is a primary target for PD-1-mediated inhibition</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1428</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.1126/science.aaf1292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1126%2Fscience.aaf1292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=28280247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsVWhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1428-1433&author=E.+Huiauthor=J.+Cheungauthor=J.+Zhuauthor=X.+Suauthor=M.+J.+Taylorauthor=H.+A.+Wallweberauthor=D.+K.+Sasmalauthor=J.+Huangauthor=J.+M.+Kimauthor=I.+Mellmanauthor=R.+D.+Vale&title=T+cell+costimulatory+receptor+CD28+is+a+primary+target+for+PD-1-mediated+inhibition&doi=10.1126%2Fscience.aaf1292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">T cell costimulatory receptor CD28 is a primary target for PD-1-mediated inhibition</span></div><div class="casAuthors">Hui, Enfu; Cheung, Jeanne; Zhu, Jing; Su, Xiaolei; Taylor, Marcus J.; Wallweber, Heidi A.; Sasmal, Dibyendu K.; Huang, Jun; Kim, Jeong M.; Mellman, Ira; Vale, Ronald D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6332</span>),
    <span class="NLM_cas:pages">1428-1433</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">PD-1-targeted therapies have been a breakthrough for treating certain tumors and can rejuvenate T cells to unleash the anticancer immune response (see the Perspective by Clouthier and Ohashi).  It is widely believed that PD-1 suppresses signaling through the T cell receptor (TCR).  However, Hui et al. find instead that the TCR costimulatory receptor, CD28, is the primary target of PD-1 signaling.  Independently, Kamphorst et al. show that CD28 is required for PD-1 therapies to kill cancer cells efficiently and eliminate chronic viral infections in mice.  Lung cancer patients that responded to PD-1 therapy had more CD28+ T cells, which suggests that CD28 may predict treatment response.  Science, ibid p. 1428, p. 1423; see also p. 1373.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM_s4cf1lkxrVg90H21EOLACvtfcHk0li2am5BFNRYyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsVWhu7g%253D&md5=69750b99b68316ee035a0b7eeddfb9fa</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf1292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf1292%26sid%3Dliteratum%253Aachs%26aulast%3DHui%26aufirst%3DE.%26aulast%3DCheung%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DWallweber%26aufirst%3DH.%2BA.%26aulast%3DSasmal%26aufirst%3DD.%2BK.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%2BM.%26aulast%3DMellman%26aufirst%3DI.%26aulast%3DVale%26aufirst%3DR.%2BD.%26atitle%3DT%2520cell%2520costimulatory%2520receptor%2520CD28%2520is%2520a%2520primary%2520target%2520for%2520PD-1-mediated%2520inhibition%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1428%26epage%3D1433%26doi%3D10.1126%2Fscience.aaf1292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span> <span> </span><span class="NLM_article-title">Targeting protein tyrosine phosphatases for anticancer drug discovery</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1862</span>, <span class="refDoi"> DOI: 10.2174/138161210791209027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.2174%2F138161210791209027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=20337577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC3cXot1ClsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=1843-1862&author=L.+M.+Scottauthor=H.+R.+Lawrenceauthor=S.+M.+Sebtiauthor=N.+J.+Lawrenceauthor=J.+Wu&title=Targeting+protein+tyrosine+phosphatases+for+anticancer+drug+discovery&doi=10.2174%2F138161210791209027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein tyrosine phosphatases for anticancer drug discovery</span></div><div class="casAuthors">Scott, Latanya M.; Lawrence, Harshani R.; Sebti, Said M.; Lawrence, Nicholas J.; Wu, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1843-1862</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein tyrosine phosphatases (PTPs) are a diverse family of enzymes encoded by 107 genes in the human genome.  Together with protein tyrosine kinases (PTKs), PTPs regulate various cellular activities essential for the initiation and maintenance of malignant phenotypes.  While PTK inhibitors are now used routinely for cancer treatment, the PTP inhibitor development field is still in the discovery phase.  In this article, the suitability of targeting PTPs for novel anticancer drug discovery is discussed.  Examples are presented for PTPs that have been targeted for anticancer drug discovery as well as potential new PTP targets for novel anticancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHuBrNeSP0n7Vg90H21EOLACvtfcHk0ljH4ud2AR1_VA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXot1ClsLY%253D&md5=17813d1fba0225fd5e54692b1159bc4a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F138161210791209027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161210791209027%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DL.%2BM.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%26atitle%3DTargeting%2520protein%2520tyrosine%2520phosphatases%2520for%2520anticancer%2520drug%2520discovery%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2010%26volume%3D16%26spage%3D1843%26epage%3D1862%26doi%3D10.2174%2F138161210791209027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butterworth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overduin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, A. J.</span></span> <span> </span><span class="NLM_article-title">Targeting protein tyrosine phosphatase SHP2 for therapeutic intervention</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1423</span>– <span class="NLM_lpage">1437</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.4155%2Ffmc.14.88" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=25329198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOhu7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1423-1437&author=S.+Butterworthauthor=M.+Overduinauthor=A.+J.+Barr&title=Targeting+protein+tyrosine+phosphatase+SHP2+for+therapeutic+intervention&doi=10.4155%2Ffmc.14.88"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein tyrosine phosphatase SHP2 for therapeutic intervention</span></div><div class="casAuthors">Butterworth, Sam; Overduin, Michael; Barr, Alastair J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1423-1437</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein tyrosine phosphatases have been the focus of considerable research efforts aimed at developing novel therapeutics; however, these targets are often characterized as being 'undruggable' due to the challenge of achieving selectivity, potency and cell permeability.  More recently, there has been renewed interest in developing inhibitors of the tyrosine phosphatase SHP2 (PTPN11) in the light of its broad role in cancer, specifically juvenile myelomonocytic leukemia, and recent studies that implicate SHP2 as a key factor in breast cancer progression.  Recent significant advances in the field of SHP2 inhibitor development raise the question: are we on the verge of a new era of protein tyrosine phosphatase-directed therapeutics This article critically appraises recent developments, assesses ongoing challenges and presents a perspective on possible future directions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEVdQ5GiIy9LVg90H21EOLACvtfcHk0ljH4ud2AR1_VA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOhu7jO&md5=85c071c906c6c047d524460df5e0a9b5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.88%26sid%3Dliteratum%253Aachs%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DOverduin%26aufirst%3DM.%26aulast%3DBarr%26aufirst%3DA.%2BJ.%26atitle%3DTargeting%2520protein%2520tyrosine%2520phosphatase%2520SHP2%2520for%2520therapeutic%2520intervention%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D1423%26epage%3D1437%26doi%3D10.4155%2Ffmc.14.88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel shp2 protein tyrosine phosphatase inhibitor</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">562</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1124/mol.106.025536</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1124%2Fmol.106.025536" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=16717135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnslyjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2006&pages=562-570&author=L.+Chenauthor=S.+S.+Sungauthor=M.+L.+Yipauthor=H.+R.+Lawrenceauthor=Y.+Renauthor=W.+C.+Guidaauthor=S.+M.+Sebtiauthor=N.+J.+Lawrenceauthor=J.+Wu&title=Discovery+of+a+novel+shp2+protein+tyrosine+phosphatase+inhibitor&doi=10.1124%2Fmol.106.025536"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel Shp2 protein tyrosine phosphatase inhibitor</span></div><div class="casAuthors">Chen, Liwei; Sung, Shen-Shu; Yip, M. L. Richard; Lawrence, Harshani R.; Ren, Yuan; Guida, Wayne C.; Sebti, Said M.; Lawrence, Nicholas J.; Wu, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">562-570</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Shp2 is a non-receptor protein tyrosine phosphatase (PTP) encoded by the PTPN11 gene.  It is involved in growth factor-induced activation of mitogen-activated protein (MAP) kinases Erk1 and Erk2 (Erk1/2) and has been implicated in the pathogenicity of the oncogenic bacterium Helicobacter pylori.  Moreover, gain-of-function Shp2 mutations have been found in childhood leukemias and Noonan syndrome.  Thus, small mol. Shp2 PTP inhibitors are much needed reagents for evaluation of Shp2 as a therapeutic target and for chem. biol. studies of Shp2 function.  By screening the National Cancer Institute (NCI) Diversity Set chem. library, we identified 8-hydroxy-7-(6-sulfonaphthalen-2-yl)diazenyl-quinoline-5-sulfonic acid (NSC-87877) as a potent Shp2 PTP inhibitor.  Mol. modeling and site-directed mutagenesis studies suggested that NSC-87877 binds to the catalytic cleft of Shp2 PTP.  NSC-87877 cross-inhibited Shp1 in vitro, but it was selective for Shp2 over other PTPs (PTP1B, HePTP, DEP1, CD45, and LAR).  It is noteworthy that NSC-87877 inhibited epidermal growth factor (EGF)-induced activation of Shp2 PTP, Ras, and Erk1/2 in cell cultures but did not block EGF-induced Gab1 tyrosine phosphorylation or Gab1-Shp2 assocn.  Furthermore, NSC-87877 inhibited Erk1/2 activation by a Gab1-Shp2 chimera but did not affect the Shp2-independent Erk1/2 activation by phorbol 12-myristate 13-acetate.  These results identified NSC-87877 as the first PTP inhibitor capable of inhibiting Shp2 PTP in cell cultures without a detectable off-target effect.  Our study also provides the first pharmacol. evidence that Shp2 mediates EGF-induced Erk1/2 MAP kinase activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA4XVGUjw3W7Vg90H21EOLACvtfcHk0ljJQ5Ezv8gqwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnslyjsrk%253D&md5=a2ce03150ecbe6937876bb292ced30dc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fmol.106.025536&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.106.025536%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSung%26aufirst%3DS.%2BS.%26aulast%3DYip%26aufirst%3DM.%2BL.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520shp2%2520protein%2520tyrosine%2520phosphatase%2520inhibitor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2006%26volume%3D70%26spage%3D562%26epage%3D570%26doi%3D10.1124%2Fmol.106.025536" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hellmuth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosskopf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Würtele, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röder, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kries, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosario, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rademann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birchmeier, W.</span></span> <span> </span><span class="NLM_article-title">Specific inhibitors of the protein tyrosine phosphatase Shp2 identified by high-throughput docking</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">7275</span>– <span class="NLM_lpage">7280</span>, <span class="refDoi"> DOI: 10.1073/pnas.0710468105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1073%2Fpnas.0710468105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=18480264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsFCqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=7275-7280&author=K.+Hellmuthauthor=S.+Grosskopfauthor=C.+T.+Lumauthor=M.+W%C3%BCrteleauthor=N.+R%C3%B6derauthor=J.+P.+von+Kriesauthor=M.+Rosarioauthor=J.+Rademannauthor=W.+Birchmeier&title=Specific+inhibitors+of+the+protein+tyrosine+phosphatase+Shp2+identified+by+high-throughput+docking&doi=10.1073%2Fpnas.0710468105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Specific inhibitors of the protein tyrosine phosphatase Shp2 identified by high-throughput docking</span></div><div class="casAuthors">Hellmuth, Klaus; Grosskopf, Stefanie; Lum, Ching Tung; Wuertele, Martin; Roeder, Nadine; von Kries, Jens Peter; Rosario, Marta; Rademann, Jorg; Birchmeier, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7275-7280</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The protein tyrosine phosphatase Shp2 is a pos. regulator of growth factor signaling.  Gain-of-function mutations in several types of leukemia define Shp2 as a bona fide oncogene.  We performed a high-throughput in silico screen for small-mol.-wt. compds. that bind the catalytic site of Shp2.  We have identified the phenylhydrazonopyrazolone sulfonate PHPS1 as a potent and cell-permeable inhibitor, which is specific for Shp2 over the closely related tyrosine phosphatases Shp1 and PTP1 B.  PHPS1 inhibits Shp2-dependent cellular events such as hepatocyte growth factor/scatter factor (HGF/SF)-induced epithelial cell scattering and branching morphogenesis.  PHPS1 also blocks Shp2-dependent downstream signaling, namely HGF/SF-induced sustained phosphorylation of the Erk1/2 MAP kinases and dephosphorylation of paxillin.  Furthermore, PHPS1 efficiently inhibits activation of Erk1/2 by the leukemia-assocd. Shp2 mutant, Shp2-E76K, and blocks the anchorage-independent growth of a variety of human tumor cell lines.  The PHPS compd. class is therefore suitable for further development of therapeutics for the treatment of Shp2-dependent diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGponh1kw_h7jrVg90H21EOLACvtfcHk0ljJQ5Ezv8gqwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsFCqsLs%253D&md5=39c7dda94e32134c305e135e4c48b9d1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0710468105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0710468105%26sid%3Dliteratum%253Aachs%26aulast%3DHellmuth%26aufirst%3DK.%26aulast%3DGrosskopf%26aufirst%3DS.%26aulast%3DLum%26aufirst%3DC.%2BT.%26aulast%3DW%25C3%25BCrtele%26aufirst%3DM.%26aulast%3DR%25C3%25B6der%26aufirst%3DN.%26aulast%3Dvon%2BKries%26aufirst%3DJ.%2BP.%26aulast%3DRosario%26aufirst%3DM.%26aulast%3DRademann%26aufirst%3DJ.%26aulast%3DBirchmeier%26aufirst%3DW.%26atitle%3DSpecific%2520inhibitors%2520of%2520the%2520protein%2520tyrosine%2520phosphatase%2520Shp2%2520identified%2520by%2520high-throughput%2520docking%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D7275%26epage%3D7280%26doi%3D10.1073%2Fpnas.0710468105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pireddu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szymanski, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, M. L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Src homology-2 domain containing protein tyrosine phosphatase-2 (Shp2) based on oxindole scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4948</span>– <span class="NLM_lpage">4956</span>, <span class="refDoi"> DOI: 10.1021/jm8002526</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8002526" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptl2hu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4948-4956&author=H.+R.+Lawrenceauthor=R.+Piredduauthor=L.+Chenauthor=Y.+Luoauthor=S.-S.+Sungauthor=A.+M.+Szymanskiauthor=M.+L.+R.+Yipauthor=W.+C.+Guidaauthor=S.+M.+Sebtiauthor=J.+Wuauthor=N.+J.+Lawrence&title=Inhibitors+of+Src+homology-2+domain+containing+protein+tyrosine+phosphatase-2+%28Shp2%29+based+on+oxindole+scaffolds&doi=10.1021%2Fjm8002526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Src Homology-2 Domain Containing Protein Tyrosine Phosphatase-2 (Shp2) Based on Oxindole Scaffolds</span></div><div class="casAuthors">Lawrence, Harshani R.; Pireddu, Roberta; Chen, Liwei; Luo, Yunting; Sung, Shen-Shu; Szymanski, Ann Marie; Yip, M. L. Richard; Guida, Wayne C.; Sebti, Said M.; Wu, Jie; Lawrence, Nicholas J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4948-4956</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening of the NCI diversity set of compds. has led to the identification of NSC-117199, which inhibits the protein tyrosine phosphatase (PTP) Shp2 with an IC50 of 47 μM.  A focused library incorporating an isatin scaffold was designed and evaluated for inhibition of Shp2 and Shp1 PTP activities.  Several compds. were identified that selectively inhibit Shp2 over Shp1 and PTP1B with low to submicromolar activity.  A model for the binding of the active compds. is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRqLAauyGNBbVg90H21EOLACvtfcHk0ljJQ5Ezv8gqwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptl2hu7o%253D&md5=572227c5a8cfffb108388616bd36fdfe</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm8002526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8002526%26sid%3Dliteratum%253Aachs%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DPireddu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DSung%26aufirst%3DS.-S.%26aulast%3DSzymanski%26aufirst%3DA.%2BM.%26aulast%3DYip%26aufirst%3DM.%2BL.%2BR.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26atitle%3DInhibitors%2520of%2520Src%2520homology-2%2520domain%2520containing%2520protein%2520tyrosine%2520phosphatase-2%2520%2528Shp2%2529%2520based%2520on%2520oxindole%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4948%26epage%3D4956%26doi%3D10.1021%2Fjm8002526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabinger, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Salicylic acid based small molecule inhibitor for the oncogenic Src homology-2 domain containing protein tyrosine phosphatase-2 (SHP2)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2482</span>– <span class="NLM_lpage">2493</span>, <span class="refDoi"> DOI: 10.1021/jm901645u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901645u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFyhs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2482-2493&author=X.+Zhangauthor=Y.+Heauthor=S.+Liuauthor=Z.+Yuauthor=Z.+X.+Jiangauthor=Z.+Yangauthor=Y.+Dongauthor=S.+C.+Nabingerauthor=L.+Wuauthor=A.+M.+Gunawanauthor=L.+Wangauthor=R.+J.+Chanauthor=Z.+Y.+Zhang&title=Salicylic+acid+based+small+molecule+inhibitor+for+the+oncogenic+Src+homology-2+domain+containing+protein+tyrosine+phosphatase-2+%28SHP2%29&doi=10.1021%2Fjm901645u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Salicylic Acid Based Small Molecule Inhibitor for the Oncogenic Src Homology-2 Domain Containing Protein Tyrosine Phosphatase-2 (SHP2)</span></div><div class="casAuthors">Zhang, Xian; He, Yantao; Liu, Sijiu; Yu, Zhihong; Jiang, Zhong-Xing; Yang, Zhenyun; Dong, Yuanshu; Nabinger, Sarah C.; Wu, Li; Gunawan, Andrea M.; Wang, Lina; Chan, Rebecca J.; Zhang, Zhong-Yin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2482-2493</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Src homol.-2 domain contg. protein tyrosine phosphatase-2 (SHP2) plays a pivotal role in growth factor and cytokine signaling.  Gain-of-function SHP2 mutations are assocd. with Noonan syndrome, various kinds of leukemias, and solid tumors.  Thus, there is considerable interest in SHP2 as a potential target for anticancer and antileukemia therapy.  We report a salicylic acid based combinatorial library approach aimed at binding both active site and unique nearby subpockets for enhanced affinity and selectivity.  Screening of the library led to the identification of a SHP2 inhibitor II-B08 (compd. 9) with highly efficacious cellular activity.  Compd. 9 blocks growth factor stimulated ERK1/2 activation and hematopoietic progenitor proliferation, providing supporting evidence that chem. inhibition of SHP2 may be therapeutically useful for anticancer and antileukemia treatment.  X-ray crystallog. anal. of the structure of SHP2 in complex with 9 reveals mol. determinants that can be exploited for the acquisition of more potent and selective SHP2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC82T5JpCRArVg90H21EOLACvtfcHk0lg_uxzSMtRMuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFyhs7k%253D&md5=6f87897eda38ea485dc08e189f1b5ca8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm901645u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901645u%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DZ.%2BX.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DNabinger%26aufirst%3DS.%2BC.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DGunawan%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChan%26aufirst%3DR.%2BJ.%26aulast%3DZhang%26aufirst%3DZ.%2BY.%26atitle%3DSalicylic%2520acid%2520based%2520small%2520molecule%2520inhibitor%2520for%2520the%2520oncogenic%2520Src%2520homology-2%2520domain%2520containing%2520protein%2520tyrosine%2520phosphatase-2%2520%2528SHP2%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2482%26epage%3D2493%26doi%3D10.1021%2Fjm901645u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, C.-K.</span></span> <span> </span><span class="NLM_article-title">Identification of cryptotanshinone as an inhibitor of oncogenic protein tyrosine phosphatase SHP2 (PTPN11)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7212</span>– <span class="NLM_lpage">7221</span>, <span class="refDoi"> DOI: 10.1021/jm400474r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400474r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7212-7221&author=W.+Liuauthor=B.+Yuauthor=G.+Xuauthor=W.-R.+Xuauthor=M.+L.+Lohauthor=L.-D.+Tangauthor=C.-K.+Qu&title=Identification+of+cryptotanshinone+as+an+inhibitor+of+oncogenic+protein+tyrosine+phosphatase+SHP2+%28PTPN11%29&doi=10.1021%2Fjm400474r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Cryptotanshinone as an Inhibitor of Oncogenic Protein Tyrosine Phosphatase SHP2 (PTPN11)</span></div><div class="casAuthors">Liu, Wei; Yu, Bing; Xu, Gang; Xu, Wei-Ren; Loh, Mignon L.; Tang, Li-Da; Qu, Cheng-Kui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7212-7221</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activating mutations of PTPN11 (encoding the SHP2 phosphatase) are assocd. with Noonan syndrome, childhood leukemias, and sporadic solid tumors.  Virtual screening combined with exptl. assays was performed to identify inhibitors of SHP2 from a database of natural products.  This effort led to the identification of cryptotanshinone as an inhibitor of SHP2.  Cryptotanshinone inhibited SHP2 with an IC50 of 22.50 μM.  Fluorescence titrn. expts. confirmed that it directly bound to SHP2.  Enzymic kinetic analyses showed that cryptotanshinone was a mixed-type and irreversible inhibitor.  This drug was further verified for its ability to block SHP2-mediated cell signaling and cellular functions.  Furthermore, mouse myeloid progenitors and patient leukemic cells with the activating mutation E76K in PTPN11 were found to be sensitive to this inhibitor.  Since cryptotanshinone is used to treat cardiovascular diseases in Asian countries, this drug has a potential to be used directly or to be further developed to treat PTPN11-assocd. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJYelS47QYHLVg90H21EOLACvtfcHk0lg_uxzSMtRMuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurvN&md5=3bbbae68159ae4227bbf74d5573ca77a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm400474r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400474r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DW.-R.%26aulast%3DLoh%26aufirst%3DM.%2BL.%26aulast%3DTang%26aufirst%3DL.-D.%26aulast%3DQu%26aufirst%3DC.-K.%26atitle%3DIdentification%2520of%2520cryptotanshinone%2520as%2520an%2520inhibitor%2520of%2520oncogenic%2520protein%2520tyrosine%2520phosphatase%2520SHP2%2520%2528PTPN11%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7212%26epage%3D7221%26doi%3D10.1021%2Fjm400474r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mali, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.-Y.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of targeting the oncogenic SHP2 phosphatase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6594</span>– <span class="NLM_lpage">6609</span>, <span class="refDoi"> DOI: 10.1021/jm5006176</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006176" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFajtrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6594-6609&author=L.-F.+Zengauthor=R.-Y.+Zhangauthor=Z.-H.+Yuauthor=S.+Liauthor=L.+Wuauthor=A.+M.+Gunawanauthor=B.+S.+Laneauthor=R.+S.+Maliauthor=X.+Liauthor=R.+J.+Chanauthor=R.+Kapurauthor=C.+D.+Wellsauthor=Z.-Y.+Zhang&title=Therapeutic+potential+of+targeting+the+oncogenic+SHP2+phosphatase&doi=10.1021%2Fjm5006176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Potential of Targeting the Oncogenic SHP2 Phosphatase</span></div><div class="casAuthors">Zeng, Li-Fan; Zhang, Ruo-Yu; Yu, Zhi-Hong; Li, Sijiu; Wu, Li; Gunawan, Andrea M.; Lane, Brandon S.; Mali, Raghuveer S.; Li, Xingjun; Chan, Rebecca J.; Kapur, Reuben; Wells, Clark D.; Zhang, Zhong-Yin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6594-6609</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Src homol. 2 domain contg. protein tyrosine phosphatase-2 (SHP2) is an oncogenic phosphatase assocd. with various kinds of leukemia and solid tumors.  Thus, there is substantial interest in developing SHP2 inhibitors as potential anticancer and antileukemia agents.  Using a structure-guided and fragment-based library approach, we identified a novel hydroxyindole carboxylic acid-based SHP2 inhibitor (I), with an IC50 value of 200 nM and greater than 5-fold selectivity against 20 mammalian PTPs.  Structural and modeling studies reveal that the hydroxyindole carboxylic acid anchors the inhibitor to the SHP2 active site, while interactions of the oxalamide linker and the phenylthiophene tail with residues in the β5-β6 loop contribute to I's binding potency and selectivity.  Evidence suggests that I specifically attenuates the SHP2-dependent signaling inside the cell.  Moreover, I blocks growth factor mediated Erk1/2 and Akt activation and exhibits excellent antiproliferative activity in lung cancer and breast cancer as well as leukemia cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3ko-q4-PSXbVg90H21EOLACvtfcHk0lg_uxzSMtRMuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFajtrvO&md5=ac8e5359387e614782c95c02f32fd97d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm5006176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006176%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.-F.%26aulast%3DZhang%26aufirst%3DR.-Y.%26aulast%3DYu%26aufirst%3DZ.-H.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DGunawan%26aufirst%3DA.%2BM.%26aulast%3DLane%26aufirst%3DB.%2BS.%26aulast%3DMali%26aufirst%3DR.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DR.%2BJ.%26aulast%3DKapur%26aufirst%3DR.%26aulast%3DWells%26aufirst%3DC.%2BD.%26aulast%3DZhang%26aufirst%3DZ.-Y.%26atitle%3DTherapeutic%2520potential%2520of%2520targeting%2520the%2520oncogenic%2520SHP2%2520phosphatase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6594%26epage%3D6609%26doi%3D10.1021%2Fjm5006176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chio, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, A. C.</span></span> <span> </span><span class="NLM_article-title">Targeting a cryptic allosteric site for selective inhibition of the oncogenic protein tyrosine phosphatase Shp2</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">504</span>, <span class="refDoi"> DOI: 10.1021/bi5013595</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi5013595" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrs7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=497-504&author=C.+M.+Chioauthor=C.+S.+Limauthor=A.+C.+Bishop&title=Targeting+a+cryptic+allosteric+site+for+selective+inhibition+of+the+oncogenic+protein+tyrosine+phosphatase+Shp2&doi=10.1021%2Fbi5013595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting a Cryptic Allosteric Site for Selective Inhibition of the Oncogenic Protein Tyrosine Phosphatase Shp2</span></div><div class="casAuthors">Chio, Cynthia M.; Lim, Christopher S.; Bishop, Anthony C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">497-504</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein tyrosine phosphatases (PTPs) have been the subject of considerable pharmaceutical-design efforts because of the ubiquitous connections between misregulation of PTP activity and human disease.  PTP-inhibitor discovery has been hampered, however, by the difficulty in identifying cell-permeable compds. that can selectively target PTP active sites, and no PTP inhibitors have progressed to the clinic.  The identification of allosteric sites on target PTPs therefore represents a potentially attractive soln. to the druggability problem of PTPs.  Here we report that the oncogenic PTP Shp2 contains an allosteric-inhibition site that renders the enzyme sensitive to potent and selective inhibition by cell-permeable biarsenical compds.  Because Shp2 contains no canonical tetracysteine biarsenical-binding motif, the enzyme's inhibitor-binding site is not readily predictable from its primary or three-dimensional structure.  Intriguingly, however, Shp2's PTP domain does contain a cysteine residue (C333) at a position that is removed from the active site and is occupied by proline in other classical PTPs.  We show that Shp2's unusual cysteine residue constitutes part of a Shp2-specific allosteric-inhibition site, and that Shp2's sensitivity to biarsenicals is dependent on the presence of the naturally occurring C333.  The determinative role of this residue in conferring inhibitor sensitivity is surprising because C333's side chain is inaccessible to solvent in Shp2 crystal structures.  The discovery of this cryptic Shp2 allosteric site may provide a means for targeting Shp2 activity with high specificity and suggests that buried-yet-targetable allosteric sites could be similarly uncovered in other protein families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWBChJ5ezZcbVg90H21EOLACvtfcHk0lgJbxpnjs9d5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrs7rK&md5=824fcbb6a26460197422c7008f35ca2b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi5013595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi5013595%26sid%3Dliteratum%253Aachs%26aulast%3DChio%26aufirst%3DC.%2BM.%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DBishop%26aufirst%3DA.%2BC.%26atitle%3DTargeting%2520a%2520cryptic%2520allosteric%2520site%2520for%2520selective%2520inhibition%2520of%2520the%2520oncogenic%2520protein%2520tyrosine%2520phosphatase%2520Shp2%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D497%26epage%3D504%26doi%3D10.1021%2Fbi5013595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fekkes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Fortanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonakos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunenfelder, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Z. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Bonte, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenoir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palermo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slisz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, P. D.</span></span> <span> </span><span class="NLM_article-title">Allosteric inhibition of SHP2 phosphatase inhibits cancers driven by receptor tyrosine kinases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">148</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1038/nature18621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fnature18621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=27362227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVOns7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=148-152&author=Y.+N.+Chenauthor=M.+J.+LaMarcheauthor=H.+M.+Chanauthor=P.+Fekkesauthor=J.+Garcia-Fortanetauthor=M.+G.+Ackerauthor=B.+Antonakosauthor=C.+H.+Chenauthor=Z.+Chenauthor=V.+G.+Cookeauthor=J.+R.+Dobsonauthor=Z.+Dengauthor=F.+Feiauthor=B.+Firestoneauthor=M.+Fodorauthor=C.+Fridrichauthor=H.+Gaoauthor=D.+Grunenfelderauthor=H.+X.+Haoauthor=J.+Jacobauthor=S.+Hoauthor=K.+Hsiaoauthor=Z.+B.+Kangauthor=R.+Karkiauthor=M.+Katoauthor=J.+Larrowauthor=L.+R.+La+Bonteauthor=F.+Lenoirauthor=G.+Liuauthor=S.+Liuauthor=D.+Majumdarauthor=M.+J.+Meyerauthor=M.+Palermoauthor=L.+Perezauthor=M.+Puauthor=E.+Priceauthor=C.+Quinnauthor=S.+Shakyaauthor=M.+D.+Shultzauthor=J.+Sliszauthor=K.+Venkatesanauthor=P.+Wangauthor=M.+Warmuthauthor=S.+Williamsauthor=G.+Yangauthor=J.+Yuanauthor=J.+H.+Zhangauthor=P.+Zhuauthor=T.+Ramseyauthor=N.+J.+Keenauthor=W.+R.+Sellersauthor=T.+Stamsauthor=P.+D.+Fortin&title=Allosteric+inhibition+of+SHP2+phosphatase+inhibits+cancers+driven+by+receptor+tyrosine+kinases&doi=10.1038%2Fnature18621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric inhibition of SHP2 phosphatase inhibits cancers driven by receptor tyrosine kinases</span></div><div class="casAuthors">Chen, Ying-Nan P.; LaMarche, Matthew J.; Chan, Ho Man; Fekkes, Peter; Garcia-Fortanet, Jorge; Acker, Michael G.; Antonakos, Brandon; Chen, Christine Hiu-Tung; Chen, Zhouliang; Cooke, Vesselina G.; Dobson, Jason R.; Deng, Zhan; Fei, Feng; Firestone, Brant; Fodor, Michelle; Fridrich, Cary; Gao, Hui; Grunenfelder, Denise; Hao, Huai-Xiang; Jacob, Jaison; Ho, Samuel; Hsiao, Kathy; Kang, Zhao B.; Karki, Rajesh; Kato, Mitsunori; Larrow, Jay; La Bonte, Laura R.; Lenoir, Francois; Liu, Gang; Liu, Shumei; Majumdar, Dyuti; Meyer, Matthew J.; Palermo, Mark; Perez, Lawrence; Pu, Minying; Price, Edmund; Quinn, Christopher; Shakya, Subarna; Shultz, Michael D.; Slisz, Joanna; Venkatesan, Kavitha; Wang, Ping; Warmuth, Markus; Williams, Sarah; Yang, Guizhi; Yuan, Jing; Zhang, Ji-Hu; Zhu, Ping; Ramsey, Timothy; Keen, Nicholas J.; Sellers, William R.; Stams, Travis; Fortin, Pascal D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7610</span>),
    <span class="NLM_cas:pages">148-152</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The non-receptor protein tyrosine phosphatase SHP2, encoded by PTPN11, has an important role in signal transduction downstream of growth factor receptor signalling and was the first reported oncogenic tyrosine phosphatase.  Activating mutations of SHP2 have been assocd. with developmental pathologies such as Noonan syndrome and are found in multiple cancer types, including leukemia, lung and breast cancer and neuroblastoma.  SHP2 is ubiquitously expressed and regulates cell survival and proliferation primarily through activation of the RAS-ERK signalling pathway2,3.  It is also a key mediator of the programmed cell death 1 (PD-1) and B- and T-lymphocyte attenuator (BTLA) immune checkpoint pathways.  Redn. of SHP2 activity suppresses tumor cell growth and is a potential target of cancer therapy.  Here we report the discovery of a highly potent (IC50 = 0.071 μM), selective and orally bioavailable small-mol. SHP2 inhibitor, SHP099, that stabilizes SHP2 in an auto-inhibited conformation.  SHP099 concurrently binds to the interface of the N-terminal SH2, C-terminal SH2, and protein tyrosine phosphatase domains, thus inhibiting SHP2 activity through an allosteric mechanism.  SHP099 suppresses RAS-ERK signalling to inhibit the proliferation of receptor-tyrosine-kinase-driven human cancer cells in vitro and is efficacious in mouse tumor xenograft models.  Together, these data demonstrate that pharmacol. inhibition of SHP2 is a valid therapeutic approach for the treatment of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBs_e02S0Sp7Vg90H21EOLACvtfcHk0lhUUdi3Sa5VSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVOns7bI&md5=ec8a4b612d5008a2519bd846d45ce019</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature18621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18621%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26aulast%3DChan%26aufirst%3DH.%2BM.%26aulast%3DFekkes%26aufirst%3DP.%26aulast%3DGarcia-Fortanet%26aufirst%3DJ.%26aulast%3DAcker%26aufirst%3DM.%2BG.%26aulast%3DAntonakos%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DCooke%26aufirst%3DV.%2BG.%26aulast%3DDobson%26aufirst%3DJ.%2BR.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DFei%26aufirst%3DF.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGrunenfelder%26aufirst%3DD.%26aulast%3DHao%26aufirst%3DH.%2BX.%26aulast%3DJacob%26aufirst%3DJ.%26aulast%3DHo%26aufirst%3DS.%26aulast%3DHsiao%26aufirst%3DK.%26aulast%3DKang%26aufirst%3DZ.%2BB.%26aulast%3DKarki%26aufirst%3DR.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DLarrow%26aufirst%3DJ.%26aulast%3DLa%2BBonte%26aufirst%3DL.%2BR.%26aulast%3DLenoir%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMajumdar%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DM.%2BJ.%26aulast%3DPalermo%26aufirst%3DM.%26aulast%3DPerez%26aufirst%3DL.%26aulast%3DPu%26aufirst%3DM.%26aulast%3DPrice%26aufirst%3DE.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DShakya%26aufirst%3DS.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DSlisz%26aufirst%3DJ.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DZhu%26aufirst%3DP.%26aulast%3DRamsey%26aufirst%3DT.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DFortin%26aufirst%3DP.%2BD.%26atitle%3DAllosteric%2520inhibition%2520of%2520SHP2%2520phosphatase%2520inhibits%2520cancers%2520driven%2520by%2520receptor%2520tyrosine%2520kinases%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D148%26epage%3D152%26doi%3D10.1038%2Fnature18621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia Fortanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fekkes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunenfelder, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Z. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaBonte, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenoir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palermo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric inhibition of SHP2: identification of a potent, selective, and orally efficacious phosphatase inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7773</span>– <span class="NLM_lpage">7782</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00680</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00680" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKju7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7773-7782&author=J.+Garcia+Fortanetauthor=C.+H.+Chenauthor=Y.+N.+Chenauthor=Z.+Chenauthor=Z.+Dengauthor=B.+Firestoneauthor=P.+Fekkesauthor=M.+Fodorauthor=P.+D.+Fortinauthor=C.+Fridrichauthor=D.+Grunenfelderauthor=S.+Hoauthor=Z.+B.+Kangauthor=R.+Karkiauthor=M.+Katoauthor=N.+Keenauthor=L.+R.+LaBonteauthor=J.+Larrowauthor=F.+Lenoirauthor=G.+Liuauthor=S.+Liuauthor=F.+Lombardoauthor=D.+Majumdarauthor=M.+J.+Meyerauthor=M.+Palermoauthor=L.+Perezauthor=M.+Puauthor=T.+Ramseyauthor=W.+R.+Sellersauthor=M.+D.+Shultzauthor=T.+Stamsauthor=C.+Towlerauthor=P.+Wangauthor=S.+L.+Williamsauthor=J.+H.+Zhangauthor=M.+J.+LaMarche&title=Allosteric+inhibition+of+SHP2%3A+identification+of+a+potent%2C+selective%2C+and+orally+efficacious+phosphatase+inhibitor&doi=10.1021%2Facs.jmedchem.6b00680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Inhibition of SHP2: Identification of a Potent, Selective, and Orally Efficacious Phosphatase Inhibitor</span></div><div class="casAuthors">Garcia Fortanet, Jorge; Chen, Christine Hiu-Tung; Chen, Ying-Nan P.; Chen, Zhouliang; Deng, Zhan; Firestone, Brant; Fekkes, Peter; Fodor, Michelle; Fortin, Pascal D.; Fridrich, Cary; Grunenfelder, Denise; Ho, Samuel; Kang, Zhao B.; Karki, Rajesh; Kato, Mitsunori; Keen, Nick; LaBonte, Laura R.; Larrow, Jay; Lenoir, Francois; Liu, Gang; Liu, Shumei; Lombardo, Franco; Majumdar, Dyuti; Meyer, Matthew J.; Palermo, Mark; Perez, Lawrence; Pu, Minying; Ramsey, Timothy; Sellers, William R.; Shultz, Michael D.; Stams, Travis; Towler, Christopher; Wang, Ping; Williams, Sarah L.; Zhang, Ji-Hu; LaMarche, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7773-7782</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2 is a nonreceptor protein tyrosine phosphatase (PTP) encoded by the PTPN11 gene involved in cell growth and differentiation via the MAPK signaling pathway.  SHP2 also purportedly plays an important role in the programmed cell death pathway (PD-1/PD-L1).  Because it is an oncoprotein assocd. with multiple cancer-related diseases, as well as a potential immunomodulator, controlling SHP2 activity is of significant therapeutic interest.  Recently in our labs., a small mol. inhibitor of SHP2 was identified as an allosteric modulator that stabilizes the autoinhibited conformation of SHP2.  A high throughput screen was performed to identify progressable chem. matter, and X-ray crystallog. revealed the location of binding in a previously undisclosed allosteric binding pocket.  Structure-based drug design was employed to optimize for SHP2 inhibition, and several new protein-ligand interactions were characterized.  These studies culminated in the discovery of 6-(4-amino-4-methylpiperidin-1-yl)-3-(2,3-dichlorophenyl)pyrazin-2-amine (SHP099, 1), a potent, selective, orally bioavailable, and efficacious SHP2 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6RT7zfZnvfLVg90H21EOLACvtfcHk0li7iHtJKfBrSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKju7fP&md5=dfe43b5253e172576386f86797087f7b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00680%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%2BFortanet%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFekkes%26aufirst%3DP.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DFortin%26aufirst%3DP.%2BD.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DGrunenfelder%26aufirst%3DD.%26aulast%3DHo%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DZ.%2BB.%26aulast%3DKarki%26aufirst%3DR.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DLaBonte%26aufirst%3DL.%2BR.%26aulast%3DLarrow%26aufirst%3DJ.%26aulast%3DLenoir%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DMajumdar%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DM.%2BJ.%26aulast%3DPalermo%26aufirst%3DM.%26aulast%3DPerez%26aufirst%3DL.%26aulast%3DPu%26aufirst%3DM.%26aulast%3DRamsey%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DWilliams%26aufirst%3DS.%2BL.%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3DAllosteric%2520inhibition%2520of%2520SHP2%253A%2520identification%2520of%2520a%2520potent%252C%2520selective%252C%2520and%2520orally%2520efficacious%2520phosphatase%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7773%26epage%3D7782%26doi%3D10.1021%2Facs.jmedchem.6b00680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric inhibitors of SHP2 with therapeutic potential for cancer treatment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">10205</span>– <span class="NLM_lpage">10219</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01520</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01520" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVCks7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=10205-10219&author=J.+Xieauthor=X.+Siauthor=S.+Guauthor=M.+Wangauthor=J.+Shenauthor=H.+Liauthor=J.+Shenauthor=D.+Liauthor=Y.+Fangauthor=C.+Liuauthor=J.+Zhu&title=Allosteric+inhibitors+of+SHP2+with+therapeutic+potential+for+cancer+treatment&doi=10.1021%2Facs.jmedchem.7b01520"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Inhibitors of SHP2 with Therapeutic Potential for Cancer Treatment</span></div><div class="casAuthors">Xie, Jingjing; Si, Xiaojia; Gu, Shoulai; Wang, Mingliang; Shen, Jian; Li, Haoyan; Li, Dan; Fang, Yanjia; Liu, Cong; Zhu, Jidong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10205-10219</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2, a cytoplasmic protein tyrosine phosphatase encoded by the PTPN11 gene, is involved in multiple cell signaling processes including Ras/MAPK and Hippo/YAP pathways.  SHP2 has been shown to contribute to the progression of a no. of cancer types including leukemia, gastric, and breast cancers.  It also regulates T-cell activation by interacting with inhibitory immune checkpoint receptors such as the programmed cell death 1 (PD-1) and B- and T-lymphocyte attenuator (BTLA).  Thus, SHP2 inhibitors have drawn great attention by both inhibiting tumor cell proliferation and activating T cell immune responses toward cancer cells.  In this study, the authors report the identification of an allosteric SHP2 inhibitor I that locks SHP2 in a closed conformation by binding to the interface of the N-terminal SH2, C-terminal SH2, and phosphatase domains.  Compd. I suppresses MAPK signaling pathway and YAP transcriptional activity and shows antitumor activity in vivo.  The results indicate that allosteric inhibition of SHP2 could be a feasible approach for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriWHA25LKpH7Vg90H21EOLACvtfcHk0li7iHtJKfBrSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVCks7%252FM&md5=4aaabafae3250476b7bf3687d83cffd3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01520%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DSi%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DJ.%26atitle%3DAllosteric%2520inhibitors%2520of%2520SHP2%2520with%2520therapeutic%2520potential%2520for%2520cancer%2520treatment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D10205%26epage%3D10219%26doi%3D10.1021%2Facs.jmedchem.7b01520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koltun, E. S.</span>; <span class="NLM_string-name">Aay, N.</span>; <span class="NLM_string-name">Buckl, A.</span>; <span class="NLM_string-name">Jogalekar, A. S.</span>; <span class="NLM_string-name">Kiss, G.</span>; <span class="NLM_string-name">Marquez, A.</span>; <span class="NLM_string-name">Mellem, K. T.</span>; <span class="NLM_string-name">Mordec, K.</span>; <span class="NLM_string-name">Saldajeno-Concar, M.</span>; <span class="NLM_string-name">Semko, C. M.</span></span> <span> </span><span class="NLM_article-title">RMC-4550, an allosteric inhibitor of SHP2: synthesis, structure, and anti-tumor activity</span>. Presented at the  <i>Annual Meeting of the American Association for Cancer Research (AACR)</i>. <span class="NLM_publisher-loc">Chicago, IL, Philadelphia (PA)</span>, <span class="NLM_year">2018</span>; Abstract 4878.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=E.+S.+Koltun&author=N.+Aay&author=A.+Buckl&author=A.+S.+Jogalekar&author=G.+Kiss&author=A.+Marquez&author=K.+T.+Mellem&author=K.+Mordec&author=M.+Saldajeno-Concar&author=C.+M.+Semko&title=RMC-4550%2C+an+allosteric+inhibitor+of+SHP2%3A+synthesis%2C+structure%2C+and+anti-tumor+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DKoltun%26aufirst%3DE.%2BS.%26atitle%3DRMC-4550%252C%2520an%2520allosteric%2520inhibitor%2520of%2520SHP2%253A%2520synthesis%252C%2520structure%252C%2520and%2520anti-tumor%2520activity%26jtitle%3DAnnual%2520Meeting%2520of%2520the%2520American%2520Association%2520for%2520Cancer%2520Research%2520%2528AACR%2529%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagdanoff, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dore, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hentemann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaBonte, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohseni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ntaganda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span> <span> </span><span class="NLM_article-title">Optimization of fused bicyclic allosteric SHP2 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1781</span>– <span class="NLM_lpage">1792</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01725</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01725" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1781-1792&author=J.+T.+Bagdanoffauthor=Z.+Chenauthor=M.+Ackerauthor=Y.+N.+Chenauthor=H.+Chanauthor=M.+Doreauthor=B.+Firestoneauthor=M.+Fodorauthor=J.+Fortanetauthor=M.+Hentemannauthor=M.+Katoauthor=R.+Koenigauthor=L.+R.+LaBonteauthor=S.+Liuauthor=M.+Mohseniauthor=R.+Ntagandaauthor=P.+Sarverauthor=T.+Smithauthor=M.+Sendzikauthor=T.+Stamsauthor=S.+Spenceauthor=C.+Towlerauthor=H.+Wangauthor=P.+Wangauthor=S.+L.+Williamsauthor=M.+J.+LaMarche&title=Optimization+of+fused+bicyclic+allosteric+SHP2+inhibitors&doi=10.1021%2Facs.jmedchem.8b01725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Fused Bicyclic Allosteric SHP2 Inhibitors</span></div><div class="casAuthors">Bagdanoff, Jeffrey T.; Chen, Zhouliang; Acker, Michael; Chen, Ying-Nan; Chan, Homan; Dore, Michael; Firestone, Brant; Fodor, Michelle; Fortanet, Jorge; Hentemann, Murphy; Kato, Mitsunori; Koenig, Robert; La Bonte, Laura R.; Liu, Shumei; Mohseni, Movarid; Ntaganda, Rukundo; Sarver, Patrick; Smith, Troy; Sendzik, Martin; Stams, Travis; Spence, Stan; Towler, Christopher; Wang, Hongyun; Wang, Ping; Williams, Sarah L.; La Marche, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1781-1792</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2 is a nonreceptor protein tyrosine phosphatase within the mitogen-activated protein kinase (MAPK) pathway controlling cell growth, differentiation, and oncogenic transformation.  SHP2 also participates in the programed cell death pathway (PD-1/PD-L1) governing immune surveillance.  Small-mol. inhibition of SHP2 has been widely investigated, including in our previous reports describing SHP099 (2), which binds to a tunnel-like allosteric binding site.  To broaden our approach to allosteric inhibition of SHP2, we conducted addnl. hit finding, evaluation, and structure-based scaffold morphing.  These studies, reported here in the first of two papers, led to the identification of multiple 5,6-fused bicyclic scaffolds that bind to the same allosteric tunnel as 2.  We demonstrate the structural diversity permitted by the tunnel pharmacophore and culminated in the identification of pyrazolopyrimidinones (e.g., SHP389, 1) that modulate MAPK signaling in vivo.  These studies also served as the basis for further scaffold morphing and optimization, detailed in the following manuscript.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra2OF_vrNnP7Vg90H21EOLACvtfcHk0licLT1RVHwLTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjtbo%253D&md5=e810a8a6bd1d1fb7bfadc5bc1448ff22</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01725%26sid%3Dliteratum%253Aachs%26aulast%3DBagdanoff%26aufirst%3DJ.%2BT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DAcker%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DDore%26aufirst%3DM.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DFortanet%26aufirst%3DJ.%26aulast%3DHentemann%26aufirst%3DM.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKoenig%26aufirst%3DR.%26aulast%3DLaBonte%26aufirst%3DL.%2BR.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMohseni%26aufirst%3DM.%26aulast%3DNtaganda%26aufirst%3DR.%26aulast%3DSarver%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DT.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DSpence%26aufirst%3DS.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DWilliams%26aufirst%3DS.%2BL.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3DOptimization%2520of%2520fused%2520bicyclic%2520allosteric%2520SHP2%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1781%26epage%3D1792%26doi%3D10.1021%2Facs.jmedchem.8b01725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagdanoff, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hentemann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaBonte, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohseni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span> <span> </span><span class="NLM_article-title">6-Amino-3-methylpyrimidinones as potent, selective, and orally efficacious SHP2 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1793</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01726</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01726" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1793-1802&author=P.+Sarverauthor=M.+Ackerauthor=J.+T.+Bagdanoffauthor=Z.+Chenauthor=Y.+N.+Chenauthor=H.+Chanauthor=B.+Firestoneauthor=M.+Fodorauthor=J.+Fortanetauthor=H.+Haoauthor=M.+Hentemannauthor=M.+Katoauthor=R.+Koenigauthor=L.+R.+LaBonteauthor=G.+Liuauthor=S.+Liuauthor=C.+Liuauthor=E.+McNeillauthor=M.+Mohseniauthor=M.+Sendzikauthor=T.+Stamsauthor=S.+Spenceauthor=V.+Tamezauthor=R.+Tichkuleauthor=C.+Towlerauthor=H.+Wangauthor=P.+Wangauthor=S.+L.+Williamsauthor=B.+Yuauthor=M.+J.+LaMarche&title=6-Amino-3-methylpyrimidinones+as+potent%2C+selective%2C+and+orally+efficacious+SHP2+inhibitors&doi=10.1021%2Facs.jmedchem.8b01726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">6-Amino-3-methylpyrimidinones as Potent, Selective, and Orally Efficacious SHP2 Inhibitors</span></div><div class="casAuthors">Sarver, Patrick; Acker, Michael; Bagdanoff, Jeffrey T.; Chen, Zhouliang; Chen, Ying-Nan; Chan, Homan; Firestone, Brant; Fodor, Michelle; Fortanet, Jorge; Hao, Huaixiang; Hentemann, Murphy; Koenig, Robert; Kato, Mitsunori; LaBonte, Laura R.; Liu, Gang; Liu, Shumei; Liu, Chen; McNeill, Eric; Mohseni, Morvarid; Sendzik, Martin; Stams, Travis; Spence, Stan; Tamez, Victoriano; Tichkule, Ritesh; Towler, Christopher; Wang, Hongyun; Wang, Ping; Williams, Sarah L.; Yu, Bing; LaMarche, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1793-1802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein tyrosine phosphatase SHP2 is an oncoprotein assocd. with cancer as well as a potential immune modulator because of its role in the programmed cell death PD-L1/PD-1 pathway.  In the preceding manuscript, we described the optimization of a fused, bicyclic screening hit for potency, selectivity, and physicochem. properties in order to further expand the chem. diversity of allosteric SHP2 inhibitors.  In this manuscript, we describe the further expansion of our approach, morphing the fused, bicyclic system into a novel monocyclic pyrimidinone scaffold through our understanding of SAR and use of structure-based design.  These studies led to the identification of SHP394 (1), an orally efficacious inhibitor of SHP2, with high lipophilic efficiency, improved potency, and enhanced pharmacokinetic properties.  We also report other pyrimidinone analogs with favorable pharmacokinetic and potency profiles.  Overall, this work improves upon our previously described allosteric inhibitors and exemplifies and extends the range of permissible chem. templates that inhibit SHP2 via the allosteric mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaRey6_uE0ULVg90H21EOLACvtfcHk0licLT1RVHwLTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKisLs%253D&md5=29c357236b7ff51596385e3e256320a5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01726%26sid%3Dliteratum%253Aachs%26aulast%3DSarver%26aufirst%3DP.%26aulast%3DAcker%26aufirst%3DM.%26aulast%3DBagdanoff%26aufirst%3DJ.%2BT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DFortanet%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DH.%26aulast%3DHentemann%26aufirst%3DM.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKoenig%26aufirst%3DR.%26aulast%3DLaBonte%26aufirst%3DL.%2BR.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DMcNeill%26aufirst%3DE.%26aulast%3DMohseni%26aufirst%3DM.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DSpence%26aufirst%3DS.%26aulast%3DTamez%26aufirst%3DV.%26aulast%3DTichkule%26aufirst%3DR.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DWilliams%26aufirst%3DS.%2BL.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3D6-Amino-3-methylpyrimidinones%2520as%2520potent%252C%2520selective%252C%2520and%2520orally%2520efficacious%2520SHP2%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1793%26epage%3D1802%26doi%3D10.1021%2Facs.jmedchem.8b01726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mainardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulero-Sánchez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prahallad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimpenfort, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieftink, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinberg, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wit, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves-Ribeiro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadal, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernards, R.</span></span> <span> </span><span class="NLM_article-title">SHP2 is required for growth of KRAS-mutant non-small-cell lung cancer in vivo</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">961</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0023-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41591-018-0023-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=29808006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVelsbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=961-967&author=S.+Mainardiauthor=A.+Mulero-S%C3%A1nchezauthor=A.+Prahalladauthor=G.+Germanoauthor=A.+Bosmaauthor=P.+Krimpenfortauthor=C.+Lieftinkauthor=J.+D.+Steinbergauthor=N.+de+Witauthor=S.+Gon%C3%A7alves-Ribeiroauthor=E.+Nadalauthor=A.+Bardelliauthor=A.+Villanuevaauthor=R.+Bernards&title=SHP2+is+required+for+growth+of+KRAS-mutant+non-small-cell+lung+cancer+in+vivo&doi=10.1038%2Fs41591-018-0023-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">SHP2 is required for growth of KRAS-mutant non-small-cell lung cancer in vivo</span></div><div class="casAuthors">Mainardi, Sara; Mulero-Sanchez, Antonio; Prahallad, Anirudh; Germano, Giovanni; Bosma, Astrid; Krimpenfort, Paul; Lieftink, Cor; Steinberg, Jeffrey D.; de Wit, Niels; Goncalves-Ribeiro, Samuel; Nadal, Ernest; Bardelli, Alberto; Villanueva, Alberto; Bernards, Rene</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">961-967</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">RAS mutations are frequent in human cancer, esp. in pancreatic, colorectal and non-small-cell lung cancers (NSCLCs).  Inhibition of the RAS oncoproteins has proven difficult, and attempts to target downstream effectors have been hampered by the activation of compensatory resistance mechanisms.  It is also well established that KRAS-mutant tumors are insensitive to inhibition of upstream growth factor receptor signaling.  Thus, epidermal growth factor receptor antibody therapy is only effective in KRAS wild-type colon cancers.  Consistently, inhibition of SHP2 (also known as PTPN11), which links receptor tyrosine kinase signaling to the RAS-RAF-MEK-ERK pathway11,12, was shown to be ineffective in KRAS-mutant or BRAF-mutant cancer cell lines.  Tha data also indicate that SHP2 inhibition in KRAS-mutant NSCLC cells under normal cell culture conditions has little effect.  By contrast, SHP2 inhibition under growth factor-limiting conditions in vitro results in a senescence response.  In vivo, inhibition of SHP2 in KRAS-mutant NSCLC also provokes a senescence response, which is exacerbated by MEK inhibition.  Tha data identify SHP2 inhibition as an unexpected vulnerability of KRAS-mutant NSCLC cells that remains undetected in cell culture and can be exploited therapeutically.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLHUMCtBqtZrVg90H21EOLACvtfcHk0licLT1RVHwLTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVelsbvL&md5=8e1ce1eb5d30d3985aafdec9d07f205d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0023-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0023-9%26sid%3Dliteratum%253Aachs%26aulast%3DMainardi%26aufirst%3DS.%26aulast%3DMulero-S%25C3%25A1nchez%26aufirst%3DA.%26aulast%3DPrahallad%26aufirst%3DA.%26aulast%3DGermano%26aufirst%3DG.%26aulast%3DBosma%26aufirst%3DA.%26aulast%3DKrimpenfort%26aufirst%3DP.%26aulast%3DLieftink%26aufirst%3DC.%26aulast%3DSteinberg%26aufirst%3DJ.%2BD.%26aulast%3Dde%2BWit%26aufirst%3DN.%26aulast%3DGon%25C3%25A7alves-Ribeiro%26aufirst%3DS.%26aulast%3DNadal%26aufirst%3DE.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DVillanueva%26aufirst%3DA.%26aulast%3DBernards%26aufirst%3DR.%26atitle%3DSHP2%2520is%2520required%2520for%2520growth%2520of%2520KRAS-mutant%2520non-small-cell%2520lung%2520cancer%2520in%2520vivo%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D961%26epage%3D967%26doi%3D10.1038%2Fs41591-018-0023-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruess, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heynen, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciecielski, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabacaoglu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Görgülü, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantes, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wörmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diakopoulos, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpathaki, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya-Aksoy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Laan, E. A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Alberca, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazaré, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erkan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopt, U. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sainz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birchmeier, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algül, H.</span></span> <span> </span><span class="NLM_article-title">Mutant KRAS-driven cancers depend on PTPN11/SHP2 phosphatase</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">954</span>– <span class="NLM_lpage">960</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0024-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41591-018-0024-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=29808009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVelsbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=954-960&author=D.+A.+Ruessauthor=G.+J.+Heynenauthor=K.+J.+Ciecielskiauthor=J.+Aiauthor=A.+Berningerauthor=D.+Kabacaogluauthor=K.+G%C3%B6rg%C3%BCl%C3%BCauthor=Z.+Dantesauthor=S.+M.+W%C3%B6rmannauthor=K.+N.+Diakopoulosauthor=A.+F.+Karpathakiauthor=M.+Kowalskaauthor=E.+Kaya-Aksoyauthor=L.+Songauthor=E.+A.+Z.+van+der+Laanauthor=M.+P.+L%C3%B3pez-Albercaauthor=M.+Nazar%C3%A9author=M.+Reichertauthor=D.+Saurauthor=M.+M.+Erkanauthor=U.+T.+Hoptauthor=B.+Sainzauthor=W.+Birchmeierauthor=R.+M.+Schmidauthor=M.+Lesinaauthor=H.+Alg%C3%BCl&title=Mutant+KRAS-driven+cancers+depend+on+PTPN11%2FSHP2+phosphatase&doi=10.1038%2Fs41591-018-0024-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Mutant KRAS-driven cancers depend on PTPN11/SHP2 phosphatase</span></div><div class="casAuthors">Ruess, Dietrich A.; Heynen, Guus J.; Ciecielski, Katrin J.; Ai, Jiaoyu; Berninger, Alexandra; Kabacaoglu, Derya; Goerguelue, Kivanc; Dantes, Zahra; Woermann, Sonja M.; Diakopoulos, Kalliope N.; Karpathaki, Angeliki F.; Kowalska, Marlena; Kaya-Aksoy, Ezgi; Song, Liang; van der Laan, Eveline A. Zeeuw; Lopez-Alberca, Maria P.; Nazare, Marc; Reichert, Maximilian; Saur, Dieter; Erkan, Mert M.; Hopt, Ulrich T.; Sainz, Jr, Bruno; Birchmeier, Walter; Schmid, Roland M.; Lesina, Marina; Alguel, Hana</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">954-960</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The ubiquitously expressed non-receptor protein tyrosine phosphatase SHP2, encoded by PTPN11, is involved in signal transduction downstream of multiple growth factor, cytokine and integrin receptors1.  Its requirement for complete RAS-MAPK activation and its role as a neg. regulator of JAK-STAT signaling have established SHP2 as an essential player in oncogenic signaling pathways1-7.  Recently, a novel potent allosteric SHP2 inhibitor was presented as a viable therapeutic option for receptor tyrosine kinase-driven cancers, but was shown to be ineffective in KRAS-mutant tumor cell lines in vitro8.  Here, we report a central and indispensable role for SHP2 in oncogenic KRAS-driven tumors.  Genetic deletion of Ptpn11 profoundly inhibited tumor development in mutant KRAS-driven murine models of pancreatic ductal adenocarcinoma and non-small-cell lung cancer.  We provide evidence for a crit. dependence of mutant KRAS on SHP2 during carcinogenesis.  Deletion or inhibition of SHP2 in established tumors delayed tumor progression but was not sufficient to achieve tumor regression.  However, SHP2 was necessary for resistance mechanisms upon blockade of MEK.  Synergy was obsd. when both SHP2 and MEK were targeted, resulting in sustained tumor growth control in murine and human patient-derived organoids and xenograft models of pancreatic ductal adenocarcinoma and non-small-cell lung cancer.  Our data indicate the clin. utility of dual SHP2/MEK inhibition as a targeted therapy approach for KRAS-mutant cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoepq09hU77abVg90H21EOLACvtfcHk0ljRRPp5mVQgnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVelsbvF&md5=cc09897c2c4b8e32b035b991e5032f32</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0024-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0024-8%26sid%3Dliteratum%253Aachs%26aulast%3DRuess%26aufirst%3DD.%2BA.%26aulast%3DHeynen%26aufirst%3DG.%2BJ.%26aulast%3DCiecielski%26aufirst%3DK.%2BJ.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DBerninger%26aufirst%3DA.%26aulast%3DKabacaoglu%26aufirst%3DD.%26aulast%3DG%25C3%25B6rg%25C3%25BCl%25C3%25BC%26aufirst%3DK.%26aulast%3DDantes%26aufirst%3DZ.%26aulast%3DW%25C3%25B6rmann%26aufirst%3DS.%2BM.%26aulast%3DDiakopoulos%26aufirst%3DK.%2BN.%26aulast%3DKarpathaki%26aufirst%3DA.%2BF.%26aulast%3DKowalska%26aufirst%3DM.%26aulast%3DKaya-Aksoy%26aufirst%3DE.%26aulast%3DSong%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BLaan%26aufirst%3DE.%2BA.%2BZ.%26aulast%3DL%25C3%25B3pez-Alberca%26aufirst%3DM.%2BP.%26aulast%3DNazar%25C3%25A9%26aufirst%3DM.%26aulast%3DReichert%26aufirst%3DM.%26aulast%3DSaur%26aufirst%3DD.%26aulast%3DErkan%26aufirst%3DM.%2BM.%26aulast%3DHopt%26aufirst%3DU.%2BT.%26aulast%3DSainz%26aufirst%3DB.%26aulast%3DBirchmeier%26aufirst%3DW.%26aulast%3DSchmid%26aufirst%3DR.%2BM.%26aulast%3DLesina%26aufirst%3DM.%26aulast%3DAlg%25C3%25BCl%26aufirst%3DH.%26atitle%3DMutant%2520KRAS-driven%2520cancers%2520depend%2520on%2520PTPN11%252FSHP2%2520phosphatase%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D954%26epage%3D960%26doi%3D10.1038%2Fs41591-018-0024-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0ljVAGxd2LLVrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0ljVAGxd2LLVrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0ljVAGxd2LLVrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) - past, present and future</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=Proteolysis+targeting+chimeras+%28PROTACs%29+-+past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eZywZgjH8VVnbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520-%2520past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lhhNQ0V_74zuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Enhancing antiproliferative activity and selectivity of a FLT-3 inhibitor by proteolysis targeting chimera conversion</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">16428</span>– <span class="NLM_lpage">16432</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b10320</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b10320" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2qtr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=16428-16432&author=G.+M.+Burslemauthor=J.+Songauthor=X.+Chenauthor=J.+Hinesauthor=C.+M.+Crews&title=Enhancing+antiproliferative+activity+and+selectivity+of+a+FLT-3+inhibitor+by+proteolysis+targeting+chimera+conversion&doi=10.1021%2Fjacs.8b10320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancing Antiproliferative Activity and Selectivity of a FLT-3 Inhibitor by Proteolysis Targeting Chimera Conversion</span></div><div class="casAuthors">Burslem, George M.; Song, Jayoung; Chen, Xin; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16428-16432</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The receptor tyrosine kinase FLT-3 is frequently mutated in acute myeloid leukemia; however, current small mol. inhibitors suffer from limited efficacy in the clinic.  Conversion of a FLT-3 inhibitor (quizartinib) into a proteolysis targeting chimera (PROTAC) results in a compd. that induces degrdn. of FLT-3 ITD mutant at low nanomolar concns.  Furthermore, the PROTAC is capable of inhibiting cell growth more potently than the warhead alone while inhibiting fewer off-target kinases.  This enhanced antiproliferative activity occurs, despite a slight redn. in the PROTAC's kinase inhibitory activity, via an increased level of apoptosis induction suggesting nonkinase roles for the FLT-3 ITD protein.  Addnl., the PROTAC is capable of inducing FLT-3 ITD degrdn. in vivo.  These results suggest that degrdn. of FLT-3 ITD may provide a useful method for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBxw_nAEQydLVg90H21EOLACvtfcHk0lhhNQ0V_74zuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2qtr7P&md5=52529d0c3f49052bd2a66c893e98e694</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b10320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b10320%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DEnhancing%2520antiproliferative%2520activity%2520and%2520selectivity%2520of%2520a%2520FLT-3%2520inhibitor%2520by%2520proteolysis%2520targeting%2520chimera%2520conversion%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D16428%26epage%3D16432%26doi%3D10.1021%2Fjacs.8b10320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmusson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricchiuto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Development of a novel B-cell lymphoma 6 (BCL6) PROTAC to provide insight into small molecule targeting of BCL6</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3141</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=3131-3141&author=W.+McCoullauthor=T.+Cheungauthor=E.+Andersonauthor=P.+Bartonauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=M.+P.+Castaldiauthor=H.+Chenauthor=E.+Chiarparinauthor=R.+J.+Carbajoauthor=E.+Codeauthor=S.+Cowanauthor=P.+R.+Daveyauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=N.+Gaoauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=A.+Kawatkarauthor=P.+D.+Kemmittauthor=E.+Leoauthor=D.+M.+Molinaauthor=N.+O%E2%80%99Connellauthor=P.+Petterutiauthor=T.+Rasmussonauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=P.+Ricchiutoauthor=G.+R.+Robbauthor=M.+Schenoneauthor=M.+J.+Waringauthor=M.+Zindaauthor=S.+Fawellauthor=D.+M.+Wilson&title=Development+of+a+novel+B-cell+lymphoma+6+%28BCL6%29+PROTAC+to+provide+insight+into+small+molecule+targeting+of+BCL6&doi=10.1021%2Facschembio.8b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Novel B-Cell Lymphoma 6 (BCL6) PROTAC To Provide Insight into Small Molecule Targeting of BCL6</span></div><div class="casAuthors">McCoull, William; Cheung, Tony; Anderson, Erica; Barton, Peter; Burgess, Jonathan; Byth, Kate; Cao, Qing; Castaldi, M. Paola; Chen, Huawei; Chiarparin, Elisabetta; Carbajo, Rodrigo J.; Code, Erin; Cowan, Suzanna; Davey, Paul R.; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gao, Ning; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kawatkar, Aarti; Kemmitt, Paul D.; Leo, Elisabetta; Molina, Daniel M.; O'Connell, Nichole; Petteruti, Philip; Rasmusson, Timothy; Raubo, Piotr; Rawlins, Philip B.; Ricchiuto, Piero; Robb, Graeme R.; Schenone, Monica; Waring, Michael J.; Zinda, Michael; Fawell, Stephen; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3131-3141</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) inhibition is a promising mechanism for treating hematol. cancers but high quality chem. probes are necessary to evaluate its therapeutic potential.  Here we report potent BCL6 inhibitors that demonstrate cellular target engagement and exhibit exquisite selectivity for BCL6 based on mass spectrometry analyses following chem. proteomic pull down.  Importantly, a proteolysis-targeting chimera, 4-((R)-25-Chloro-41-methyl-42-oxo-41,42,43,44-tetrahydro-5,8-dioxa-3-aza-4(6,8)-quinolina2(2,4)-pyrimidina-1(1,2)-piperazinacyclononaphane-14-yl)-N-(3-(2-(2-((2-((S)-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)propyl)-4-oxobutanamide trifluoroacetate, (PROTAC, 15) was also developed and shown to significantly degrade BCL6 in a no. of diffuse large B-cell lymphoma (DLBCL) cell lines, but neither BCL6 inhibition nor degrdn. selectively induced marked phenotypic response.  To investigate, we monitored PROTAC directed BCL6 degrdn. in DLBCL OCI-Ly1 cells by immunofluorescence and discovered a residual BCL6 population.  Anal. of subcellular fractions also showed incomplete BCL6 degrdn. in all fractions despite having measurable PROTAC concns., together providing a rationale for the weak antiproliferative response seen with both BCL6 inhibitor and degrader.  In summary, we have developed potent and selective BCL6 inhibitors and a BCL6 PROTAC that effectively degraded BCL6, but both modalities failed to induce a significant phenotypic response in DLBCL despite achieving cellular concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEe9aUjqcaHLVg90H21EOLACvtfcHk0lg_ORGoYQv7Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF&md5=3d2c2d8acf143f05babee6ebdabddc8c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00698%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DDavey%26aufirst%3DP.%2BR.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKawatkar%26aufirst%3DA.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DRasmusson%26aufirst%3DT.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRicchiuto%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520PROTAC%2520to%2520provide%2520insight%2520into%2520small%2520molecule%2520targeting%2520of%2520BCL6%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D3131%26epage%3D3141%26doi%3D10.1021%2Facschembio.8b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of anaplastic lymphoma kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+induced+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0lg_ORGoYQv7Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0liGX0_KxOHlXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0ljRuD7e_s6JDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Development of dual and selective degraders of cyclin-dependent kinases 4 and 6</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6321</span>– <span class="NLM_lpage">6326</span>, <span class="refDoi"> DOI: 10.1002/anie.201901336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1002%2Fanie.201901336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtlKqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=6321-6326&author=B.+Jiangauthor=E.+S.+Wangauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Fischerauthor=T.+Zhangauthor=N.+S.+Gray&title=Development+of+dual+and+selective+degraders+of+cyclin-dependent+kinases+4+and+6&doi=10.1002%2Fanie.201901336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Dual and Selective Degraders of Cyclin-Dependent Kinases 4 and 6</span></div><div class="casAuthors">Jiang, Baishan; Wang, Eric S.; Donovan, Katherine A.; Liang, Yanke; Fischer, Eric S.; Zhang, Tinghu; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6321-6326</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases 4 and 6 (CDK4/6) are key regulators of the cell cycle, and there are FDA-approved CDK4/6 inhibitors for treating patients with metastatic breast cancer.  However, due to conservation of their ATP-binding sites, development of selective agents has remained elusive.  Here, we report imide-based degrader mols. capable of degrading both CDK4/6, or selectively degrading either CDK4 or CDK6.  We were also able to tune the activity of these mols. against Ikaros (IKZF1) and Aiolos (IKZF3), which are well-established targets of imide-based degraders.  We found that in mantle cell lymphoma cell lines, combined IKZF1/3 degrdn. with dual CDK4/6 degrdn. produced enhanced anti-proliferative effects compared to CDK4/6 inhibition, CDK4/6 degrdn., or IKZF1/3 degrdn.  In summary, we report here the first compds. capable of inducing selective degrdn. of CDK4 and CDK6 as tools to pharmacol. dissect their distinct biol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Vsqs-SM6hrVg90H21EOLACvtfcHk0ljRuD7e_s6JDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtlKqsrw%253D&md5=39a6937b24abad0940d359020e2b2f74</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fanie.201901336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201901336%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520dual%2520and%2520selective%2520degraders%2520of%2520cyclin-dependent%2520kinases%25204%2520and%25206%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D6321%26epage%3D6326%26doi%3D10.1002%2Fanie.201901336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of SD-36 as a potent, selective, and efficacious PROTAC degrader of STAT3 protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11280</span>– <span class="NLM_lpage">11300</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01530</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01530" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11280-11300&author=H.+Zhouauthor=L.+Baiauthor=R.+Xuauthor=Y.+Zhaoauthor=J.+Chenauthor=D.+McEachernauthor=K.+Chinnaswamyauthor=B.+Wenauthor=L.+Daiauthor=P.+Kumarauthor=C.-Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=J.+L.+Meagherauthor=H.+Yiauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-based+discovery+of+SD-36+as+a+potent%2C+selective%2C+and+efficacious+PROTAC+degrader+of+STAT3+protein&doi=10.1021%2Facs.jmedchem.9b01530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span></div><div class="casAuthors">Zhou, Haibin; Bai, Longchuan; Xu, Renqi; Zhao, Yujun; Chen, Jianyong; McEachern, Donna; Chinnaswamy, Krishnapriya; Wen, Bo; Dai, Lipeng; Kumar, Praveen; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Meagher, Jennifer L.; Yi, Han; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11280-11300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases.  Despite twenty-years of persistent research efforts, targeting STAT3 has been very challenging.  The authors report herein the structure-based discovery of potent small-mol. STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept.  The authors first designed SI-109 as a potent, small-mol. inhibitor of the STAT3 SH2 domain.  Employing ligands for cereblon/Cullin 4A E3 ligase and SI-109, the authors obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36.  SD-36 induces rapid STAT3 degrdn. at low nanomolar concns. in cells and fails to degrader other STAT proteins.  SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3.  A single dose of SD-36 results in complete STAT3 protein degrdn. in xenograft tumor tissue and normal mouse tissues.  SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules.  SD-36 is a potent, selective and efficacious STAT3 degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIAWd1hiuqy7Vg90H21EOLACvtfcHk0ljYd4RwjU5WLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J&md5=7d0c656a2d96196a3b71829c9efe715a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01530%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%2520SD-36%2520as%2520a%2520potent%252C%2520selective%252C%2520and%2520efficacious%2520PROTAC%2520degrader%2520of%2520STAT3%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11280%26epage%3D11300%26doi%3D10.1021%2Facs.jmedchem.9b01530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">Arvinas’s PROTACs pass first safety and PK analysis</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">895</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00188-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fd41573-019-00188-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=895&author=A.+Mullard&title=Arvinas%E2%80%99s+PROTACs+pass+first+safety+and+PK+analysis&doi=10.1038%2Fd41573-019-00188-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">NIH launches open science Alzheimer initiative</span></div><div class="casAuthors">Mullard, Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">895</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKr7PwgMO_fLVg90H21EOLACvtfcHk0ljYd4RwjU5WLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs7zO&md5=2abfeaf7ac16a726ceff22cb0164347c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00188-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00188-4%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DArvinas%25E2%2580%2599s%2520PROTACs%2520pass%2520first%2520safety%2520and%2520PK%2520analysis%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D895%26doi%3D10.1038%2Fd41573-019-00188-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potjewyd, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rectenwald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolis, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herring, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Degradation of polycomb repressive complex 2 with an EED-targeted bivalent chemical degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1016%2Fj.chembiol.2019.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=31831267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlOmt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=47-56&author=F.+Potjewydauthor=A.+W.+Turnerauthor=J.+Beriauthor=J.+M.+Rectenwaldauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=D.+M.+Margolisauthor=K.+H.+Pearceauthor=L.+E.+Herringauthor=L.+I.+James&title=Degradation+of+polycomb+repressive+complex+2+with+an+EED-targeted+bivalent+chemical+degrader&doi=10.1016%2Fj.chembiol.2019.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of Polycomb Repressive Complex 2 with an EED-Targeted Bivalent Chemical Degrader</span></div><div class="casAuthors">Potjewyd, Frances; Turner, Anne-Marie W.; Beri, Joshua; Rectenwald, Justin M.; Norris-Drouin, Jacqueline L.; Cholensky, Stephanie H.; Margolis, David M.; Pearce, Kenneth H.; Herring, Laura E.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-56.e15</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein degrdn. via the use of bivalent chem. degraders provides an alternative strategy to block protein function and assess the biol. roles of putative drug targets.  This approach capitalizes on the advantages of small-mol. inhibitors while moving beyond the restrictions of traditional pharmacol.  Here, we report a chem. degrader (UNC6852) that targets polycomb repressive complex 2 (PRC2).  UNC6852 contains an EED226-derived ligand and a ligand for VHL which bind to the WD40 arom. cage of EED and CRL2VHL, resp., to induce proteasomal degrdn. of PRC2 components, EED, EZH2, and SUZ12.  Degrdn. of PRC2 with UNC6852 blocks the histone methyltransferase activity of EZH2, decreasing H3K27me3 levels in HeLa cells and diffuse large B cell lymphoma (DLBCL) cells contg. EZH2 gain-of-function mutations.  UNC6852 degrades both wild-type and mutant EZH2, and addnl. displays anti-proliferative effects in this cancer model system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF3p-HFdD9KLVg90H21EOLACvtfcHk0ljYd4RwjU5WLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlOmt7bM&md5=7d9ca2726bad4e40003310951edc09d4</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DPotjewyd%26aufirst%3DF.%26aulast%3DTurner%26aufirst%3DA.%2BW.%26aulast%3DBeri%26aufirst%3DJ.%26aulast%3DRectenwald%26aufirst%3DJ.%2BM.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26aulast%3DHerring%26aufirst%3DL.%2BE.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DDegradation%2520of%2520polycomb%2520repressive%2520complex%25202%2520with%2520an%2520EED-targeted%2520bivalent%2520chemical%2520degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D47%26epage%3D56%26doi%3D10.1016%2Fj.chembiol.2019.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruvilla, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-m.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogojina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghton, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hromas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span> <span> </span><span class="NLM_article-title">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1938</span>– <span class="NLM_lpage">1947</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0668-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41591-019-0668-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=31792461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1938-1947&author=S.+Khanauthor=X.+Zhangauthor=D.+Lvauthor=Q.+Zhangauthor=Y.+Heauthor=P.+Zhangauthor=X.+Liuauthor=D.+Thummuriauthor=Y.+Yuanauthor=J.+S.+Wiegandauthor=J.+Peiauthor=W.+Zhangauthor=A.+Sharmaauthor=C.+R.+McCurdyauthor=V.+M.+Kuruvillaauthor=N.+Baranauthor=A.+A.+Ferrandoauthor=Y.-m.+Kimauthor=A.+Rogojinaauthor=P.+J.+Houghtonauthor=G.+Huangauthor=R.+Hromasauthor=M.+Konoplevaauthor=G.+Zhengauthor=D.+Zhou&title=A+selective+BCL-XL+PROTAC+degrader+achieves+safe+and+potent+antitumor+activity&doi=10.1038%2Fs41591-019-0668-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span></div><div class="casAuthors">Khan, Sajid; Zhang, Xuan; Lv, Dongwen; Zhang, Qi; He, Yonghan; Zhang, Peiyi; Liu, Xingui; Thummuri, Dinesh; Yuan, Yaxia; Wiegand, Janet S.; Pei, Jing; Zhang, Weizhou; Sharma, Abhisheak; McCurdy, Christopher R.; Kuruvilla, Vinitha M.; Baran, Natalia; Ferrando, Adolfo A.; Kim, Yong-mi; Rogojina, Anna; Houghton, Peter J.; Huang, Guangcun; Hromas, Robert; Konopleva, Marina; Zheng, Guangrong; Zhou, Daohong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1938-1947</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">B-cell lymphoma extra large (BCL-XL) is a well-validated cancer target.  However, the on-target and dose-limiting thrombocytopenia limits the use of BCL-XL inhibitors, such as ABT263, as safe and effective anticancer agents.  To reduce the toxicity of ABT263, we converted it into DT2216, a BCL-XL proteolysis-targeting chimera (PROTAC), that targets BCL-XL to the Von Hippel-Lindau (VHL) E3 ligase for degrdn.  We found that DT2216 was more potent against various BCL-XL-dependent leukemia and cancer cells but considerably less toxic to platelets than ABT263 in vitro because VHL is poorly expressed in platelets.  In vivo, DT2216 effectively inhibits the growth of several xenograft tumors as a single agent or in combination with other chemotherapeutic agents, without causing appreciable thrombocytopenia.  These findings demonstrate the potential to use PROTAC technol. to reduce on-target drug toxicities and rescue the therapeutic potential of previously undruggable targets.  Furthermore, DT2216 may be developed as a safe first-in-class anticancer agent targeting BCL-XL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGposmmgxVNCRLVg90H21EOLACvtfcHk0lgr30dWZ5h2cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO&md5=b07678c880569acac428ec479f002974</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0668-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0668-z%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWiegand%26aufirst%3DJ.%2BS.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DC.%2BR.%26aulast%3DKuruvilla%26aufirst%3DV.%2BM.%26aulast%3DBaran%26aufirst%3DN.%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DY.-m.%26aulast%3DRogojina%26aufirst%3DA.%26aulast%3DHoughton%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DHromas%26aufirst%3DR.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DD.%26atitle%3DA%2520selective%2520BCL-XL%2520PROTAC%2520degrader%2520achieves%2520safe%2520and%2520potent%2520antitumor%2520activity%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D1938%26epage%3D1947%26doi%3D10.1038%2Fs41591-019-0668-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class mitogen-activated protein kinase kinase 1/2 degrader</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10911</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01528</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01528" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10897-10911&author=J.+Weiauthor=J.+Huauthor=L.+Wangauthor=L.+Xieauthor=M.+S.+Jinauthor=X.+Chenauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+a+first-in-class+mitogen-activated+protein+kinase+kinase+1%2F2+degrader&doi=10.1021%2Facs.jmedchem.9b01528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader</span></div><div class="casAuthors">Wei, Jieli; Hu, Jianping; Wang, Li; Xie, Ling; Jin, Margaret S.; Chen, Xian; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10897-10911</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the "gatekeepers" of the ERK signaling output with redundant roles in controlling ERK activity.  Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs.  However, acquired resistance to MEK1/2 inhibitors has been obsd. in patients and new therapeutic strategies are needed to overcome the resistance.  Here, the authors report a first-in-class degrader of MEK1/2, MS432 (I), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells.  It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its neg. control MS432N and its effect was phenocopied by MEK1/2 knockdown.  Compd. I was highly selective for MEK1/2 in global proteomic profiling studies.  It was also bioavailable in mice and can be used for in vivo efficacy studies.  The authors provide two well-characterized chem. tools to the biomedical community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e5sg33oDYrVg90H21EOLACvtfcHk0lgr30dWZ5h2cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK&md5=2c293b6b87990d6b814ec1dc88034f19</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520mitogen-activated%2520protein%2520kinase%2520kinase%25201%252F2%2520degrader%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10897%26epage%3D10911%26doi%3D10.1021%2Facs.jmedchem.9b01528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of SIAIS178 as an effective BCR-ABL degrader by recruiting von Hippel-Lindau (VHL) E3 ubiquitin ligase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9281</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9281-9298&author=Q.+Zhaoauthor=C.+Renauthor=L.+Liuauthor=J.+Chenauthor=Y.+Shaoauthor=N.+Sunauthor=R.+Sunauthor=Y.+Kongauthor=X.+Dingauthor=X.+Zhangauthor=Y.+Xuauthor=B.+Yangauthor=Q.+Yinauthor=X.+Yangauthor=B.+Jiang&title=Discovery+of+SIAIS178+as+an+effective+BCR-ABL+degrader+by+recruiting+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase&doi=10.1021%2Facs.jmedchem.9b01264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of SIAIS178 as an Effective BCR-ABL Degrader by Recruiting Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span></div><div class="casAuthors">Zhao, Quanju; Ren, Chaowei; Liu, Linyi; Chen, Jinju; Shao, Yubao; Sun, Ning; Sun, Renhong; Kong, Ying; Ding, Xinyu; Zhang, Xianfang; Xu, Youwei; Yang, Bei; Yin, Qianqian; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9281-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncogenic fusion protein BCR-ABL is the driving force of leukemogenesis in chronic myeloid leukemia (CML).  Despite great progress for CML treatment through application of tyrosine kinase inhibitors (TKIs) against BCR-ABL, long-term drug administration and clin. resistance continue to be an issue.  Herein, we described the design, synthesis, and evaluation of novel proteolysis-targeting chimeric (PROTAC) small mols. targeting BCR-ABL which connect dasatinib and VHL E3 ubiquitin ligase ligand by extensive optimization of linkers.  Our efforts have yielded SIAIS178 (19), which induces proper interaction between BCR-ABL and VHL ligase leading to effective degrdn. of BCR-ABL protein, achieves significant growth inhibition of BCR-ABL+ leukemic cells in vitro, and induces substantial tumor regression against K562 xenograft tumors in vivo.  In addn., SIAIS178 also degrades several clin. relevant resistance-conferring mutations.  Our data indicate that SIAIS178 as efficacious BCR-ABL degrader warrants extensive further investigation for the treatment of BCR-ABL+ leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLBFfujSx6ZbVg90H21EOLACvtfcHk0ljp39B6h_o3TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN&md5=50ace274dbc6ef3931f86504a1be93a7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01264%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DYin%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520SIAIS178%2520as%2520an%2520effective%2520BCR-ABL%2520degrader%2520by%2520recruiting%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9281%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.9b01264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tovell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shpiro, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crafter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of SGK3-PROTAC1, an isoform specific SGK3 kinase PROTAC degrader</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2024</span>– <span class="NLM_lpage">2034</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00505</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00505" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1KksLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2024-2034&author=H.+Tovellauthor=A.+Testaauthor=H.+Zhouauthor=N.+Shpiroauthor=C.+Crafterauthor=A.+Ciulliauthor=D.+R.+Alessi&title=Design+and+characterization+of+SGK3-PROTAC1%2C+an+isoform+specific+SGK3+kinase+PROTAC+degrader&doi=10.1021%2Facschembio.9b00505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of SGK3-PROTAC1, an Isoform Specific SGK3 Kinase PROTAC Degrader</span></div><div class="casAuthors">Tovell, Hannah; Testa, Andrea; Zhou, Houjiang; Shpiro, Natalia; Crafter, Claire; Ciulli, Alessio; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2024-2034</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SGK3 is a PX domain contg. protein kinase activated at endosomes downstream of Class 1 and 3 PI3K family members by growth factors and oncogenic mutations.  SGK3 plays a key role in mediating resistance of breast cancer cells to Class 1 PI3K or Akt inhibitors, by substituting for loss of Akt activity and restoring proliferative pathways such as mTORC1 signaling.  It is therefore crit. to develop tools to potently target SGK3, and obstruct its role in inhibitor resistance.  Here we describe the development of SGK3-PROTAC1 (I), a PROTAC conjugate of the 308-R SGK inhibitor with the VH032 VHL binding ligand, targeting SGK3 for degrdn.  0.3 μM I induced 50% degrdn. of endogenous SGK3 within 2 h, with maximal 80% degrdn. obsd. within 8 h, accompanied by a loss of phosphorylation of NDRG1, an SGK3 substrate.  I did not degrade closely related SGK1 and SGK2 isoforms that are nevertheless engaged and inhibited by 308-R.  Proteomic anal. revealed that SGK3 was the only cellular protein whose cellular levels were significantly reduced following treatment with I.  Low doses of I (0.1-0.3 μM) restored sensitivity of SGK3 dependent ZR-75-1 and CAMA-1 breast cancer cells to Akt (AZD5363) and PI3K (GDC0941) inhibitors, whereas the cis epimer analog incapable of binding to the VHL E3 ligase had no impact.  I suppressed proliferation of ZR-75-1 and CAMA-1 cancer cell lines treated with a PI3K inhibitor (GDC0941) more effectively than could be achieved by a conventional SGK isoform inhibitor (14H).  This work underscores the benefit of the PROTAC approach in targeting protein kinase signaling pathways with greater efficacy and selectivity than can be achieved with conventional inhibitors.  I will be an important reagent to explore the roles of the SGK3 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCZDBO0-RuDrVg90H21EOLACvtfcHk0ljp39B6h_o3TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1KksLnN&md5=83f84ea9aba235a129c4cc4919888c6f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00505%26sid%3Dliteratum%253Aachs%26aulast%3DTovell%26aufirst%3DH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DShpiro%26aufirst%3DN.%26aulast%3DCrafter%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520SGK3-PROTAC1%252C%2520an%2520isoform%2520specific%2520SGK3%2520kinase%2520PROTAC%2520degrader%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2024%26epage%3D2034%26doi%3D10.1021%2Facschembio.9b00505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedmueller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstabl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenhöfer-Wölfer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wöhrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechens, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen-Hughes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0294-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41589-019-0294-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=31178587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=672-680&author=W.+Farnabyauthor=M.+Koeglauthor=M.+J.+Royauthor=C.+Whitworthauthor=E.+Diersauthor=N.+Trainorauthor=D.+Zollmanauthor=S.+Steurerauthor=J.+Karolyi-Oezguerauthor=C.+Riedmuellerauthor=T.+Gmaschitzauthor=J.+Wachterauthor=C.+Dankauthor=M.+Galantauthor=B.+Sharpsauthor=K.+Rumpelauthor=E.+Traxlerauthor=T.+Gerstbergerauthor=R.+Schnitzerauthor=O.+Petermannauthor=P.+Grebauthor=H.+Weinstablauthor=G.+Baderauthor=A.+Zoephelauthor=A.+Weiss-Puxbaumauthor=K.+Ehrenh%C3%B6fer-W%C3%B6lferauthor=S.+W%C3%B6hrleauthor=G.+Boehmeltauthor=J.+Rinnenthalauthor=H.+Arnhofauthor=N.+Wiechensauthor=M.-Y.+Wuauthor=T.+Owen-Hughesauthor=P.+Ettmayerauthor=M.+Pearsonauthor=D.+B.+McConnellauthor=A.+Ciulli&title=BAF+complex+vulnerabilities+in+cancer+demonstrated+via+structure-based+PROTAC+design&doi=10.1038%2Fs41589-019-0294-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span></div><div class="casAuthors">Farnaby, William; Koegl, Manfred; Roy, Michael J.; Whitworth, Claire; Diers, Emelyne; Trainor, Nicole; Zollman, David; Steurer, Steffen; Karolyi-Oezguer, Jale; Riedmueller, Carina; Gmaschitz, Teresa; Wachter, Johannes; Dank, Christian; Galant, Michael; Sharps, Bernadette; Rumpel, Klaus; Traxler, Elisabeth; Gerstberger, Thomas; Schnitzer, Renate; Petermann, Oliver; Greb, Peter; Weinstabl, Harald; Bader, Gerd; Zoephel, Andreas; Weiss-Puxbaum, Alexander; Ehrenhofer-Wolfer, Katharina; Wohrle, Simon; Boehmelt, Guido; Rinnenthal, Joerg; Arnhof, Heribert; Wiechens, Nicola; Wu, Meng-Ying; Owen-Hughes, Tom; Ettmayer, Peter; Pearson, Mark; McConnell, Darryl B.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeting subunits of BAF/PBAF chromatin remodeling complexes has been proposed as an approach to exploit cancer vulnerabilities.  Here, we develop proteolysis targeting chimera (PROTAC) degraders of the BAF ATPase subunits SMARCA2 and SMARCA4 using a bromodomain ligand and recruitment of the E3 ubiquitin ligase VHL.  High-resoln. ternary complex crystal structures and biophys. investigation guided rational and efficient optimization toward ACBI1(I), a potent and cooperative degrader of SMARCA2, SMARCA4 and PBRM1.  I induced anti-proliferative effects and cell death caused by SMARCA2 depletion in SMARCA4 mutant cancer cells, and in acute myeloid leukemia cells dependent on SMARCA4 ATPase activity.  These findings exemplify a successful biophysics- and structure-based PROTAC design approach to degrade high profile drug targets, and pave the way toward new therapeutics for the treatment of tumors sensitive to the loss of BAF complex ATPases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEh4pux4Bs7Vg90H21EOLACvtfcHk0lhZRdFREYml7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ&md5=e95fad12c727aedcf08aed9320dae905</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0294-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0294-6%26sid%3Dliteratum%253Aachs%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DDiers%26aufirst%3DE.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DZollman%26aufirst%3DD.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DRiedmueller%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWachter%26aufirst%3DJ.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DWeinstabl%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DEhrenh%25C3%25B6fer-W%25C3%25B6lfer%26aufirst%3DK.%26aulast%3DW%25C3%25B6hrle%26aufirst%3DS.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DWiechens%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DM.-Y.%26aulast%3DOwen-Hughes%26aufirst%3DT.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBAF%2520complex%2520vulnerabilities%2520in%2520cancer%2520demonstrated%2520via%2520structure-based%2520PROTAC%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D672%26epage%3D680%26doi%3D10.1038%2Fs41589-019-0294-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoppi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Iterative design and optimization of initially inactive proteolysis targeting chimeras (PROTACs) identify VZ185 as a potent, fast, and selective von Hippel-Lindau (VHL) based dual degrader probe of BRD9 and BRD7</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=699-726&author=V.+Zoppiauthor=S.+J.+Hughesauthor=C.+Maniaciauthor=A.+Testaauthor=T.+Gmaschitzauthor=C.+Wieshoferauthor=M.+Koeglauthor=K.+M.+Richingauthor=D.+L.+Danielsauthor=A.+Spallarossaauthor=A.+Ciulli&title=Iterative+design+and+optimization+of+initially+inactive+proteolysis+targeting+chimeras+%28PROTACs%29+identify+VZ185+as+a+potent%2C+fast%2C+and+selective+von+Hippel-Lindau+%28VHL%29+based+dual+degrader+probe+of+BRD9+and+BRD7&doi=10.1021%2Facs.jmedchem.8b01413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span></div><div class="casAuthors">Zoppi, Vittoria; Hughes, Scott J.; Maniaci, Chiara; Testa, Andrea; Gmaschitz, Teresa; Wieshofer, Corinna; Koegl, Manfred; Riching, Kristin M.; Daniels, Danette L.; Spallarossa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">699-726</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Developing PROTACs to redirect the ubiquitination activity of E3 ligases and potently degrade a target protein within cells can be a lengthy and unpredictable process, and it remains unclear whether any combination of E3 and target might be productive for degrdn.  The authors describe a probe-quality degrader for a ligase-target pair deemed unsuitable: the von Hippel-Lindau (VHL) and BRD9, a bromodomain-contg. subunit of the SWI/SNF chromatin remodeling complex BAF.  VHL-based degraders could be optimized from suboptimal compds. in two rounds by systematically varying conjugation patterns and linkers and monitoring cellular degrdn. activities, kinetic profiles, and ubiquitination, as well as ternary complex formation thermodn.  The emerged structure-activity relations guided the discovery of VZ185, a potent, fast, and selective degrader of BRD9 and of its close homolog BRD7.  The authors' findings qualify a new chem. tool for BRD7/9 knockdown and provide a roadmap for PROTAC development against seemingly incompatible target-ligase combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPnajotKvqLVg90H21EOLACvtfcHk0lhZRdFREYml7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN&md5=be7938c26807a2535b7f9b7de8ebfbb0</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01413%26sid%3Dliteratum%253Aachs%26aulast%3DZoppi%26aufirst%3DV.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIterative%2520design%2520and%2520optimization%2520of%2520initially%2520inactive%2520proteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520identify%2520VZ185%2520as%2520a%2520potent%252C%2520fast%252C%2520and%2520selective%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520based%2520dual%2520degrader%2520probe%2520of%2520BRD9%2520and%2520BRD7%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D699%26epage%3D726%26doi%3D10.1021%2Facs.jmedchem.8b01413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gechijian, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Functional TRIM24 degrader via conjugation of ineffectual bromodomain and VHL ligands</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0010-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1038%2Fs41589-018-0010-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=29507391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=405-412&author=L.+N.+Gechijianauthor=D.+L.+Buckleyauthor=M.+A.+Lawlorauthor=J.+M.+Reyesauthor=J.+Paulkauthor=C.+J.+Ottauthor=G.+E.+Winterauthor=M.+A.+Erbauthor=T.+G.+Scottauthor=M.+Xuauthor=H.-S.+Seoauthor=S.+Dhe-Paganonauthor=N.+P.+Kwiatkowskiauthor=J.+A.+Perryauthor=J.+Qiauthor=N.+S.+Grayauthor=J.+E.+Bradner&title=Functional+TRIM24+degrader+via+conjugation+of+ineffectual+bromodomain+and+VHL+ligands&doi=10.1038%2Fs41589-018-0010-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Functional TRIM24 degrader via conjugation of ineffectual bromodomain and VHL ligands</span></div><div class="casAuthors">Gechijian, Lara N.; Buckley, Dennis L.; Lawlor, Matthew A.; Reyes, Jaime M.; Paulk, Joshiawa; Ott, Christopher J.; Winter, Georg E.; Erb, Michael A.; Scott, Thomas G.; Xu, Mousheng; Seo, Hyuk-Soo; Dhe-Paganon, Sirano; Kwiatkowski, Nicholas P.; Perry, Jennifer A.; Qi, Jun; Gray, Nathanael S.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">405-412</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The addressable pocket of a protein is often not functionally relevant in disease.  This is true for the multidomain, bromodomain-contg. transcriptional regulator TRIM24.  TRIM24 has been posited as a dependency in numerous cancers, yet potent and selective ligands for the TRIM24 bromodomain do not exert effective anti-proliferative responses.  We therefore repositioned these probes as targeting features for heterobifunctional protein degraders.  Recruitment of the VHL E3 ubiquitin ligase by dTRIM24 elicits potent and selective degrdn. of TRIM24.  Using dTRIM24 to probe TRIM24 function, we characterize the dynamic genome-wide consequences of TRIM24 loss on chromatin localization and gene control.  Further, we identify TRIM24 as a novel dependency in acute leukemia.  Pairwise study of TRIM24 degrdn. vs. bromodomain inhibition reveals enhanced anti-proliferative response from degrdn.  We offer dTRIM24 as a chem. probe of an emerging cancer dependency, and establish a path forward for numerous selective yet ineffectual ligands for proteins of therapeutic interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod-DnZMlk25LVg90H21EOLACvtfcHk0lifyt0Nx4G7xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFGmurg%253D&md5=f11dfc9a2edc34c632d6d9b6e886bdf4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0010-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0010-y%26sid%3Dliteratum%253Aachs%26aulast%3DGechijian%26aufirst%3DL.%2BN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DLawlor%26aufirst%3DM.%2BA.%26aulast%3DReyes%26aufirst%3DJ.%2BM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DKwiatkowski%26aufirst%3DN.%2BP.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DFunctional%2520TRIM24%2520degrader%2520via%2520conjugation%2520of%2520ineffectual%2520bromodomain%2520and%2520VHL%2520ligands%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D405%26epage%3D412%26doi%3D10.1038%2Fs41589-018-0010-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of von Hippel-Lindau-recruiting proteolysis targeting chimeras (PROTACs) of TANK-binding kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+characterization+of+von+Hippel-Lindau-recruiting+proteolysis+targeting+chimeras+%28PROTACs%29+of+TANK-binding+kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0lifyt0Nx4G7xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520von%2520Hippel-Lindau-recruiting%2520proteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520TANK-binding%2520kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of highly potent and efficient PROTAC degraders of androgen receptor (AR) by employing weak binding affinity VHL E3 ligase ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11218</span>– <span class="NLM_lpage">11231</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01393</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01393" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit12rtbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11218-11231&author=X.+Hanauthor=L.+Zhaoauthor=W.+Xiangauthor=C.+Qinauthor=B.+Miaoauthor=T.+Xuauthor=M.+Wangauthor=C.-Y.+Yangauthor=K.+Chinnaswamyauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+highly+potent+and+efficient+PROTAC+degraders+of+androgen+receptor+%28AR%29+by+employing+weak+binding+affinity+VHL+E3+ligase+ligands&doi=10.1021%2Facs.jmedchem.9b01393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Highly Potent and Efficient PROTAC Degraders of Androgen Receptor (AR) by Employing Weak Binding Affinity VHL E3 Ligase Ligands</span></div><div class="casAuthors">Han, Xin; Zhao, Lijie; Xiang, Weiguo; Qin, Chong; Miao, Bukeyan; Xu, Tianfeng; Wang, Mi; Yang, Chao-Yie; Chinnaswamy, Krishnapriya; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11218-11231</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Androgen receptor (AR) is a validated therapeutic target for the treatment of metastatic castration-resistant prostate cancer (mCRPC).  We report herein our design, synthesis, and biol. characterization of highly potent small-mol. proteolysis targeting chimera (PROTAC) AR degraders using a potent AR antagonist and E3 ligase ligands with weak binding affinities to VHL protein.  Our study resulted in the discovery of 11 (ARD-266), which effectively induces degrdn. of AR protein in AR-pos. (AR+) LNCaP, VCaP, and 22Rv1 prostate cancer cell lines with DC50 values of 0.2-1 nM.  ARD-266 is capable of reducing the AR protein level by >95% in these AR+ prostate cancer cell lines and effectively reduces AR-regulated gene expression suppression.  For the first time, we demonstrated that an E3 ligand with micromolar binding affinity to its E3 ligase complex can be successfully employed for the design of highly potent and efficient PROTAC degraders and this finding may have a significant implication for the field of PROTAC research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM96gs2yAlzLVg90H21EOLACvtfcHk0lizmPsF5UxORQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit12rtbvE&md5=92e4f17c75b488749fced14476552c07</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01393%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520highly%2520potent%2520and%2520efficient%2520PROTAC%2520degraders%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520by%2520employing%2520weak%2520binding%2520affinity%2520VHL%2520E3%2520ligase%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11218%26epage%3D11231%26doi%3D10.1021%2Facs.jmedchem.9b01393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wurz, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellamaggiore, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span> <span> </span><span class="NLM_article-title">A “click chemistry platform” for the rapid synthesis of bispecific molecules for inducing protein degradation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01781</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01781" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Gls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=453-461&author=R.+P.+Wurzauthor=K.+Dellamaggioreauthor=H.+Douauthor=N.+Javierauthor=M.-C.+Loauthor=J.+D.+McCarterauthor=D.+Mohlauthor=C.+Sastriauthor=J.+R.+Lipfordauthor=V.+J.+Cee&title=A+%E2%80%9Cclick+chemistry+platform%E2%80%9D+for+the+rapid+synthesis+of+bispecific+molecules+for+inducing+protein+degradation&doi=10.1021%2Facs.jmedchem.6b01781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A "Click Chemistry Platform" for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation</span></div><div class="casAuthors">Wurz, Ryan P.; Dellamaggiore, Ken; Dou, Hannah; Javier, Noelle; Lo, Mei-Chu; McCarter, John D.; Mohl, Dane; Sastri, Christine; Lipford, J. Russell; Cee, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">453-461</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) are bispecific mols. contg. a target protein binder and an ubiquitin ligase binder connected by a linker.  By recruiting an ubiquitin ligase to a target protein, PROTACs promote ubiquitination and proteasomal degrdn. of the target protein.  The generation of effective PROTACs depends on the nature of the protein/ligase ligand pair, linkage site, linker length, and linker compn., all of which have been difficult to address in a systematic way.  Herein, the authors describe a "click chem." approach for the synthesis of PROTACs.  The authors demonstrate the utility of this approach with the bromodomain and extra terminal domain-4 (BRD4) ligand JQ-1 (3) and ligase binders targeting cereblon (CRBN) and Von Hippel-Lindau (VHL) proteins.  An AlphaScreen proximity assay was used to det. the ability of PROTACs to form the ternary ligase-PROTAC-target protein complex and a MSD assay to measure cellular degrdn. of the target protein promoted by PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpooapLJ2Zln7Vg90H21EOLACvtfcHk0lizmPsF5UxORQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Gls7w%253D&md5=090c569ae1f079ab6504348aabec8909</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01781%26sid%3Dliteratum%253Aachs%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DDellamaggiore%26aufirst%3DK.%26aulast%3DDou%26aufirst%3DH.%26aulast%3DJavier%26aufirst%3DN.%26aulast%3DLo%26aufirst%3DM.-C.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DMohl%26aufirst%3DD.%26aulast%3DSastri%26aufirst%3DC.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26atitle%3DA%2520%25E2%2580%259Cclick%2520chemistry%2520platform%25E2%2580%259D%2520for%2520the%2520rapid%2520synthesis%2520of%2520bispecific%2520molecules%2520for%2520inducing%2520protein%2520degradation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D453%26epage%3D461%26doi%3D10.1021%2Facs.jmedchem.6b01781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+androgen+receptor+%28AR%29+for+the+treatment+of+prostate+cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0lizmPsF5UxORQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520for%2520the%2520treatment%2520of%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.-Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0liAwR941Xt_SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lhI6gJZ73TXnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beutner, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickey, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span> <span> </span><span class="NLM_article-title">TCFH-NMI: direct cccess to N-acyl imidazoliums for challenging amide bond formations</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4218</span>– <span class="NLM_lpage">4222</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.8b01591</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.8b01591" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ajsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=4218-4222&author=G.+L.+Beutnerauthor=I.+S.+Youngauthor=M.+L.+Daviesauthor=M.+R.+Hickeyauthor=H.+Parkauthor=J.+M.+Stevensauthor=Q.+Ye&title=TCFH-NMI%3A+direct+cccess+to+N-acyl+imidazoliums+for+challenging+amide+bond+formations&doi=10.1021%2Facs.orglett.8b01591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">TCFH-NMI: Direct Access to N-Acyl Imidazoliums for Challenging Amide Bond Formations</span></div><div class="casAuthors">Beutner, Gregory L.; Young, Ian S.; Davies, Merrill L.; Hickey, Matthew R.; Park, Hyunsoo; Stevens, Jason M.; Ye, Qingmei</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4218-4222</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Challenging couplings of hindered carboxylic acids with non-nucleophilic amines to form amide bonds can be accomplished in high yields, and in many cases, with complete retention of the adjacent stereogenic centers using the combination of N,N,N',N'-tetramethylchloroformamidinium hexafluorophosphate (TCFH) and N-methylimidazole (NMI).  This method allows for in situ generation of highly reactive acyl imidazolium ions, which have been demonstrated to be intermediates in the reaction.  The reagent delivers high reactivity similar to acid chlorides with the ease of use of modern uronium reagents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouk9CntOu0PrVg90H21EOLACvtfcHk0lhI6gJZ73TXnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ajsrrJ&md5=257505afd32cbac2425c2372facc0f42</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.8b01591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.8b01591%26sid%3Dliteratum%253Aachs%26aulast%3DBeutner%26aufirst%3DG.%2BL.%26aulast%3DYoung%26aufirst%3DI.%2BS.%26aulast%3DDavies%26aufirst%3DM.%2BL.%26aulast%3DHickey%26aufirst%3DM.%2BR.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DStevens%26aufirst%3DJ.%2BM.%26aulast%3DYe%26aufirst%3DQ.%26atitle%3DTCFH-NMI%253A%2520direct%2520cccess%2520to%2520N-acyl%2520imidazoliums%2520for%2520challenging%2520amide%2520bond%2520formations%26jtitle%3DOrg.%2520Lett.%26date%3D2018%26volume%3D20%26spage%3D4218%26epage%3D4222%26doi%3D10.1021%2Facs.orglett.8b01591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argintaru, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, H.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaRochelle, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiguchi, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaMarche, M. J.</span></span> <span> </span><span class="NLM_article-title">Dual allosteric inhibition of SHP2 phosphatase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">656</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00980</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00980" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjslOgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=647-656&author=M.+Fodorauthor=E.+Priceauthor=P.+Wangauthor=H.+Luauthor=A.+Argintaruauthor=Z.+Chenauthor=M.+Glickauthor=H.-X.+Haoauthor=M.+Katoauthor=R.+Koenigauthor=J.+R.+LaRochelleauthor=G.+Liuauthor=E.+McNeillauthor=D.+Majumdarauthor=G.+A.+Nishiguchiauthor=L.+B.+Perezauthor=G.+Parisauthor=C.+M.+Quinnauthor=T.+Ramseyauthor=M.+Sendzikauthor=M.+D.+Shultzauthor=S.+L.+Williamsauthor=T.+Stamsauthor=S.+C.+Blacklowauthor=M.+G.+Ackerauthor=M.+J.+LaMarche&title=Dual+allosteric+inhibition+of+SHP2+phosphatase&doi=10.1021%2Facschembio.7b00980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Dual allosteric inhibition of SHP2 phosphatase</span></div><div class="casAuthors">Fodor, Michelle; Price, Edmund; Wang, Ping; Lu, Hengyu; Argintaru, Andreea; Chen, Zhouliang; Glick, Meir; Hao, Huai-Xiang; Kato, Mitsunori; Koenig, Robert; La Rochelle, Jonathan R.; Liu, Gang; McNeill, Eric; Majumdar, Dyuti; Nishiguchi, Gisele A.; Perez, Lawrence B.; Paris, Gregory; Quinn, Christopher M.; Ramsey, Timothy; Sendzik, Martin; Shultz, Michael David; Williams, Sarah L.; Stams, Travis; Blacklow, Stephen C.; Acker, Michael G.; LaMarche, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">647-656</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2 is a cytoplasmic protein tyrosine phosphatase encoded by the PTPN11 gene and is involved in cell proliferation, differentiation, and survival.  Recently, we reported an allosteric mechanism of inhibition that stabilizes the auto-inhibited conformation of SHP2.  SHP099 (1) was identified and characterized as a moderately potent, orally bioavailable, allosteric small mol. inhibitor, which binds to a tunnel-like pocket formed by the confluence of three domains of SHP2.  In this report, we describe further screening strategies that enabled the identification of a second, distinct small mol. allosteric site.  SHP244 (2) was identified as a weak inhibitor of SHP2 with modest thermal stabilization of the enzyme.  X-ray crystallog. revealed that 2 binds and stabilizes the inactive, closed conformation of SHP2, at a distinct, previously unexplored binding site-a cleft formed at the interface of the N-terminal SH2 and PTP domains.  Derivatization of 2 using structure-based design resulted in an increase in SHP2 thermal stabilization, biochem. inhibition, and subsequent MAPK pathway modulation.  Downregulation of DUSP6 mRNA, a downstream MAPK pathway marker, was obsd. in KYSE-520 cancer cells.  Remarkably, simultaneous occupation of both allosteric sites by 1 and 2 was possible, as characterized by cooperative biochem. inhibition expts. and X-ray crystallog.  Combining an allosteric site 1 inhibitor with an allosteric site 2 inhibitor led to enhanced pharmacol. pathway inhibition in cells.  This work illustrates a rare example of dual allosteric targeted protein inhibition, demonstrates screening methodol. and tactics to identify allosteric inhibitors, and enables further interrogation of SHP2 in cancer and related pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWdw7pTS6sOLVg90H21EOLACvtfcHk0liKrskOdBRcSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjslOgtg%253D%253D&md5=8e4de4c353b9fc196c81e1cb51e367df</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00980%26sid%3Dliteratum%253Aachs%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DPrice%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DArgintaru%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DHao%26aufirst%3DH.-X.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKoenig%26aufirst%3DR.%26aulast%3DLaRochelle%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DMcNeill%26aufirst%3DE.%26aulast%3DMajumdar%26aufirst%3DD.%26aulast%3DNishiguchi%26aufirst%3DG.%2BA.%26aulast%3DPerez%26aufirst%3DL.%2BB.%26aulast%3DParis%26aufirst%3DG.%26aulast%3DQuinn%26aufirst%3DC.%2BM.%26aulast%3DRamsey%26aufirst%3DT.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DWilliams%26aufirst%3DS.%2BL.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DAcker%26aufirst%3DM.%2BG.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3DDual%2520allosteric%2520inhibition%2520of%2520SHP2%2520phosphatase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D647%26epage%3D656%26doi%3D10.1021%2Facschembio.7b00980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EHR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EHR','PDB','5EHR'); return false;">PDB: 5EHR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MD7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MD7','PDB','6MD7'); return false;">PDB: 6MD7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MDC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MDC','PDB','6MDC'); return false;">PDB: 6MDC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XZR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XZR','PDB','5XZR'); return false;">PDB: 5XZR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BMU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BMU','PDB','6BMU'); return false;">PDB: 6BMU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BMV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BMV','PDB','6BMV'); return false;">PDB: 6BMV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BMY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BMY','PDB','6BMY'); return false;">PDB: 6BMY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i80"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00471">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_05275"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00471?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00471</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Western blotting analysis of SHP2 proteins in KYSE520 and MV4;11 cells treated with the SHP2 degraders and <sup>1</sup>H NMR, <sup>13</sup>C NMR, and UPLC-MS spectra of representative compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular string files for all of the final target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_002.csv">CSV</a>)</p></li><li><p class="inline">Modeled structures of compound <b>4</b> in complex with SHP2 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Modeled structures of compound <b>5</b> in complex with SHP2 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_004.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_001.pdf">jm0c00471_si_001.pdf (1.41 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_002.csv">jm0c00471_si_002.csv (4.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_003.pdb">jm0c00471_si_003.pdb (711.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00471/suppl_file/jm0c00471_si_004.pdb">jm0c00471_si_004.pdb (711.65 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00471&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-14%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00471%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00471" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a900cfab6a2d0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
