
<html lang="en"     class="pb-page"  data-request-id="d40070cc-398b-480a-9911-f79126ab7e0b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.9b01528;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-23"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader" /></meta><meta name="dc.Creator" content="Jieli  Wei" /></meta><meta name="dc.Creator" content="Jianping  Hu" /></meta><meta name="dc.Creator" content="Li  Wang" /></meta><meta name="dc.Creator" content="Ling  Xie" /></meta><meta name="dc.Creator" content="Margaret S.  Jin" /></meta><meta name="dc.Creator" content="Xian  Chen" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the “gatekeepers” of the ERK signaling output with redundant roles in controlling ERK activity. Numerous inhibitors targeting MEK1/2 have been de..." /></meta><meta name="Description" content="MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the “gatekeepers” of the ERK signaling output with redundant roles in controlling ERK activity. Numerous inhibitors targeting MEK1/2 have been de..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 15, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01528" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01528" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01528" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01528" /></link>
        
    
    

<title>Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01528" /></meta><meta property="og:title" content="Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0014.jpeg" /></meta><meta property="og:description" content="MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the “gatekeepers” of the ERK signaling output with redundant roles in controlling ERK activity. Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs. However, acquired resistance to MEK1/2 inhibitors has been observed in patients, and new therapeutic strategies are needed to overcome the resistance. Here, we report a first-in-class degrader of MEK1/2, MS432 (23), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells. It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its negative control MS432N (24), and its effect was phenocopied by MEK1/2 knockdown. Compound 23 was highly selective for MEK1/2 in global proteomic profiling studies. It was also bioavailable in mice and can be used for in vivo efficacy studies. We provide two well-characterized chemical tools to the biomedical community." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01528"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01528">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01528&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01528&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01528&amp;href=/doi/10.1021/acs.jmedchem.9b01528" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 10897-10911</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01405" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01606" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jieli Wei</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jieli Wei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jieli++Wei">Jieli Wei</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jianping Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianping Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianping++Hu">Jianping Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Li Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Li Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Li++Wang">Li Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ling Xie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ling Xie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ling++Xie">Ling Xie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Margaret S. Jin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Margaret S. Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Margaret+S.++Jin">Margaret S. Jin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xian Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xian Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xian++Chen">Xian Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#690300070e4705001c29041a1a04470c0d1c"><span class="__cf_email__" data-cfemail="680201060f4604011d28051b1b05460d0c1d">[email protected]</span></a> (J.L.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Jin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#32585b535c1c585b5c725f41415f1c575647"><span class="__cf_email__" data-cfemail="02686b636c2c686b6c426f71716f2c676677">[email protected]</span></a> (J.J.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01528&amp;href=/doi/10.1021%2Facs.jmedchem.9b01528" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 10897–10911</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 15, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 September 2019</li><li><span class="item_label"><b>Published</b> online</span>15 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 December 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01528" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01528</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10897%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJieli%2BWei%252C%2BJianping%2BHu%252C%2BLi%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D23%26contentID%3Dacs.jmedchem.9b01528%26title%3DDiscovery%2Bof%2Ba%2BFirst-in-Class%2BMitogen-Activated%2BProtein%2BKinase%2BKinase%2B1%252F2%2BDegrader%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10911%26publicationDate%3DDecember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01528"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2862</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01528" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jieli&quot;,&quot;last_name&quot;:&quot;Wei&quot;},{&quot;first_name&quot;:&quot;Jianping&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Ling&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Margaret&quot;,&quot;last_name&quot;:&quot;S. Jin&quot;},{&quot;first_name&quot;:&quot;Xian&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10897-10911&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01528&quot;},&quot;abstract&quot;:&quot;MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the “gatekeepers” of the ERK signaling output with redundant roles in controlling ERK activity. Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs. However, acquired resistance to MEK1/2 inhibitors has been observed in patients, and new therapeutic strategies are needed to overcome the resistance. Here, we report a first-in-class degrader of MEK1/2, MS432 (23), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells. It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its negative control MS432N (24), and its effect was phenocopied by MEK1/2 knockdown. Compound 23 was highly selective for MEK1/2 in global proteomic profiling studies. It was also bioavailable in mice and can be used for in vivo efficacy studies. We provide two well-characterized chemical tools to the biomedical community.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01528&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01528" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01528&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01528" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01528&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01528" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01528&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01528&amp;href=/doi/10.1021/acs.jmedchem.9b01528" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01528" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01528" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26pmid%3D31730343%26genre%3Darticle%26aulast%3DWei%26date%3D2019%26atitle%3DDiscovery%2Bof%2Ba%2BFirst-in-Class%2BMitogen-Activated%2BProtein%2BKinase%2BKinase%2B1%252F2%2BDegrader%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D23%26spage%3D10897%26epage%3D10911%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=291297" title="Cell signaling">Cell signaling</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/jmcmar.2019.62.issue-23/20191212/jmcmar.2019.62.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the “gatekeepers” of the ERK signaling output with redundant roles in controlling ERK activity. Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs. However, acquired resistance to MEK1/2 inhibitors has been observed in patients, and new therapeutic strategies are needed to overcome the resistance. Here, we report a first-in-class degrader of MEK1/2, MS432 (<b>23</b>), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells. It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its negative control MS432N (<b>24</b>), and its effect was phenocopied by MEK1/2 knockdown. Compound <b>23</b> was highly selective for MEK1/2 in global proteomic profiling studies. It was also bioavailable in mice and can be used for in vivo efficacy studies. We provide two well-characterized chemical tools to the biomedical community.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14483" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14483" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The classic RAS-RAF-MEK-ERK signaling pathway, which is conserved in mammals, plays critical roles in multiple cellular processes, including cell cycle progression, cell proliferation, apoptosis, cell differentiation, cell metabolism, and cell migration.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Hyperactivation of ERK signaling due to mutations in genes encoding receptor tyrosine kinases (RTKs), RAS, BRAF, CRAF, MEK1 (also known as MAP2K1 (Mitogen-Activated Protein Kinase Kinase 1 or Dual Specificity Mitogen-Activated Protein Kinase Kinase 1)), or MEK2 (also known as MAP2K2 (Mitogen-Activated Protein Kinase Kinase 2 or Dual Specificity Mitogen-Activated Protein Kinase Kinase 2)) is associated with 30% of human cancers, including melanoma, colon cancer, thyroid cancer, ovarian cancer, breast cancer, prostate cancer, lung cancer, pancreatic cancer, bladder cancer, kidney cancer, leukemia, and lymphoma.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7 ref8 ref9">(4−9)</a> Deregulation of this pathway is mainly driven by mutations of <i>BRAF</i> and its activator <i>RAS</i>.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a><i>RAS</i> mutations, predominantly affecting G12, G13, and Q61, are associated with 27% of all human cancers. Although various inhibitors have been developed to modulate RAS activity,<a onclick="showRef(event, 'ref8 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref11">(8,11)</a> it still poses significant challenges to target RAS directly.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a><i>BRAF</i> mutations account for 8% of human tumors,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> for example, 50–60% of melanomas and 10% of colorectal cancers (CRCs).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> BRAF constitutive kinase activity is mainly induced by V600E (90%) and other V600K, V600D, or V600R missense mutations (10%).<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Notably, <i>BRAF</i><sup><i>V600E</i></sup> and <i>BRAF</i><sup><i>V600K</i></sup> are found in 70–80% and 5–12% of patients with melanoma, respectively.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Interestingly, <i>RAS</i> and <i>BRAF</i> mutations are exclusively not overlapped in cancer,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> suggesting molecular mechanisms of <i>RAS</i> or <i>RAF</i> mutations are complicated and the therapy strategies against them need to be more carefully designed. A substantial number of BRAF inhibitors have been developed, some of which have been approved by the FDA. However, drug resistance has been observed in patients with <i>BRAF</i> mutations.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">In contrast to the high frequency of <i>RAS</i> or <i>BRAF</i> mutation, MEK activating mutations have been rarely reported in cancer cases, with mutations across MEK1 or MEK2 peptides in melanoma, CRC, lung cancer, and ovarian cancer.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17">(14−17)</a> In addition, <i>MEK1</i> or <i>MEK2</i> mutations are induced by the acquired resistance against tumor treatment by BRAF or MEK inhibitors.<a onclick="showRef(event, 'ref4 ref18'); return false;" href="javascript:void(0);" class="ref ref4 ref18">(4,18)</a> In addition to RAF kinases, numerous other MAPKKKs (Mitogen-Activated Protein Kinase Kinase Kinases) can also trigger MEK phosphorylation. Therefore, MEK1/2 proteins are the crucial “gatekeepers” of ERK activity, while the latter stimulates feedback inhibitory regulation.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> MEK1/2 proteins are the only RAF substrates.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> In addition, MEK1/2 are the only MAPKKs (Mitogen-Activated Protein Kinase Kinases) that activate ERK1 and ERK2.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Thus, inhibitors targeting MEK proteins specifically suppress the ERK signaling output without affecting other signaling transduction cascades directly.</div><div class="NLM_p">A large number of highly potent and selective MEK inhibitors have been developed, most of which are allosteric, non-ATP competitive small-molecule inhibitors.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Among them, trametinib (<b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), cobimetinib (<b>2</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), and binimetinib (<b>3</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) have been approved by the FDA<a onclick="showRef(event, 'ref4 ref20'); return false;" href="javascript:void(0);" class="ref ref4 ref20">(4,20)</a> for the treatment of metastatic or unresectable melanoma with <i>BRAF</i><sup><i>V600E</i></sup> or <i>BRAF</i><sup><i>V600K</i></sup> or non-small-cell lung cancer (NSCLC) with <i>BRAF</i><sup><i>V600E</i></sup> in conjunction with BRAF inhibitors.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24">(21−24)</a> Moreover, a number of clinical and preclinical studies are ongoing to investigate the therapeutic potential of MEK inhibitors in combination with other targeted agents in different cancers. For example, EGFR+BRAF+MEK inhibition treatment is being evaluated in patients with <i>BRAF</i><sup><i>V600E</i></sup> CRC, and a combination therapy of trametinib with anti-PD1 or anti-PD-L1 antibody is being tested in the in vivo models of <i>KRAS</i>/<i>P53</i> mutant NSCLC.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> Despite the initial efficacy of MEK inhibitors in the treatment of melanoma, acquired resistance to the therapy has been observed in patients.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Thus, in addition to the different drug combination approaches being investigated, the development of new therapeutic strategies targeting the RAF-MEK-ERK signaling pathway is needed to prevent, delay, or overcome drug resistance.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative MEK1/2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Chemical-induced protein knockdown via the proteolysis targeting chimera (PROTAC) technology is a promising approach to target dysregulated proteins. This approach has been rapidly gaining momentum in the drug discovery field over the last several years.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> A PROTAC is a heterobifunctional small molecule composed of a ligand for the target protein, another ligand binding to an E3 ubiquitin ligase, and a linker to tether the two ligands together. Such a heterobifunctional small molecule induces the formation of a ternary complex composed of the protein of interest, PROTAC and an E3 ligase, leading to selective polyubiquitination of the target protein and its subsequent degradation by the proteasome.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> To date, the E3 ligases that have been successfully hijacked by PROTACs include von Hippel–Lindau (VHL), cereblon (CRBN), mouse double minute 2 homologue (MDM2), and cellular inhibitor of apoptosis protein 1 (cIAP).<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29−31)</a></div><div class="NLM_p">As dual-specificity protein kinases, mammalian MEK1 and MEK2 share 80% sequence identity and 86% identity in their catalytic domains.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> They are equally capable of phosphorylating ERK1 and ERK2 at the conserved threonine and then tyrosine residues within the TEY motif in the activation loop.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In addition to their redundant catalytic functions, noncatalytic functions of MEK1 and MEK2 have also been discovered. For example, MEK1 interacts with peroxisome proliferator-activated receptor γ (PPARγ) to mediate its nuclear export and regulates its transcriptional activity;<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> MEK1/2 proteins interact with KSR1 to drive BRAF-KSR1 heterodimerization and allosteric activation of BRAF.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Furthermore, it was reported that, instead of single depletion of MEK1 or MEK2, elimination of both MEK1 and MEK2 blocked <i>KRAS</i>-driven NSCLC development.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> On the basis of these literature results, we sought to develop a dual degrader of MEK1 and MEK2 using PROTAC technology to directly degrade both MEK1 and MEK2 as a potential therapeutic strategy for CRC and melanoma. An ERK degrader generated by an in-cell click-formed PROTAC (CLIPTAC) approach and a CRBN-recruiting BRAF degrader based on a ligand that binds the RAS binding domain of BRAF were reported recently.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> In addition, a strategy using procaspase-3 activating compound PAC-1 to induce MEK cleavage indirectly was applied to overcome acquired resistance in cancer cells.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> However, PROTACs of MEK1 and/or MEK2 have not been reported.</div><div class="NLM_p last">Here, we report the discovery and characterization of a first-in-class degrader of MEK1 and MEK2, MS432 (compound <b>23</b>), which is based on the MEK1/2 inhibitor PD0325901 (<b>4</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) and a known ligand of VHL E3 ligase.<a onclick="showRef(event, 'ref4 ref41'); return false;" href="javascript:void(0);" class="ref ref4 ref41">(4,41)</a> We also developed a very close analogue of compound <b>23</b>, MS432N (compound <b>24</b>), which cannot bind to VHL E3 ligase to induce MEK1/2 degradation, as a negative control for compound <b>23</b>. In <i>BRAF</i> mutant CRC and melanoma cell lines, we found that compound <b>23</b> potently reduced MEK1/2 protein levels and inhibited ERK signaling, and suppressed cancer cell proliferation much more effectively than did its negative control compound <b>24</b>. Notably, MEK knockdown phenocopied the effect of compound <b>23</b> in the tested cancer cell lines. Moreover, we demonstrated that compound <b>23</b> induced MEK degradation by hijacking VHL E3 ligase with a series of rescue experiments. Furthermore, our global proteomic analyses indicate that compound <b>23</b> is a highly selective MEK1/2 degrader. Finally, compound <b>23</b> displayed good plasma exposure in mice and can be used in the in vivo efficacy studies. Overall, this study has resulted in two well-characterized chemical tools for the research community to explore potential therapeutic strategies for CRC, melanoma, and other cancers.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61843" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61843" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Initial Design, Synthesis, and Biological Evaluation</h3><div class="NLM_p">PD0325901 (<b>4</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), a non-ATP competitive MEK inhibitor, potently inhibited ERK signaling and proliferation of tumor cells.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> This compound has advanced to phase II clinical trials for its study against several cancers.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Compound <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), an analogue of PD0325901, also effectively inhibited the catalytic activity of MEK1/2.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The cocrystal structure of compound <b>5</b> in complex with human MEK1 revealed that compound <b>5</b> is an allosteric MEK inhibitor, which binds an allosteric site close to the ATP pocket (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Compound <b>5</b> sits relatively deep in this allosteric binding pocket, and we did not observe an obvious solvent-exposed region of compound <b>5</b> in the cocrystal structure. However, we found a flexible glycine-rich P-loop of MEK1 around the dihydroxy propyl side chain of compound <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) and hypothesized that this flexible P-loop of MEK1 has dynamic conformations and may move away from the dihydroxy propyl side chain of compound <b>5</b>, resulting in the dihydroxy propyl side chain solvent exposed. Van Dort et al. recently discovered a series of dual inhibitors of MEK and PI3K, exemplified by compound <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), which contains the main portion of MEK inhibitor PD0325901 conjugated with PI3K inhibitor ZSTK474 via a linker.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> This result suggested that the dihydroxy propyl side chain of PD0325901 or compound <b>5</b> can be used as a handle to introduce a linker to an E3 ligase ligand. On the basis of this, we designed a set of putative MEK1/2 degraders <b>7</b>–<b>11</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C) by connecting the main portion of MEK inhibitor PD0325901 to a ligand of VHL or CRBN E3 ligase via different linkers.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Crystal structure of the ternary complex of compound <b>5</b> (cyan), ATP (green), and MEK1 (gray) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9J">1S9J</a>). Key H-bonds formed by <b>5</b> are illustrated by yellow dotted lines. The flexible glycine rich P-loop is illustrated in blue. The unstructured Asn221, Ser222, and Phe223 were modeled according to PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP1">3PP1</a> using Coot. (B) Structure of the dual PI3K-MEK1 inhibitor <b>6</b>. (C) Structures of compounds <b>7</b>–<b>11</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Syntheses of these compounds are outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The reaction of commercially available compound <b>12</b> with 2-hydroxyisoindoline-1,3-dione afforded compound <b>13</b>, which was converted to compound <b>14</b> by removing the protecting group. Condensation of compound <b>14</b> with commercially available 3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzoic acid yielded compound <b>15</b>, which was subsequently deprotected to afford compound <b>16</b>. The reaction of commercially available compound <b>17</b> with different linkers through condensation and subsequent removal of the protection group generated compounds <b>18</b>–<b>20</b>, which were then transformed to compounds <b>7</b>–<b>9</b> via reductive amination reactions with compound <b>16</b>. Reductive amination reactions of compounds <b>21</b> and <b>22</b>, which were prepared according to previously published procedures,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> with compound <b>16</b> furnished the desired compounds <b>10</b> and <b>11</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0012.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes for Preparing Compounds <b>7</b>–<b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DBU, DMF, 50 °C, 4 h; (b) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, MeOH, DCM, rt, 2 h; (c) HOAt, EDCI, NMM, DMSO, rt, overnight; (d) 3 M HCl, THF, rt, 6 h; (e) (1) HOAt, EDCI, NMM, DMSO, rt, overnight; (2) TFA, DCM, rt, 2 h; (f) NaBH<sub>3</sub>CN, DCM, MeOH, rt, overnight.</p></p></figure><div class="NLM_p">We next used HT-29, a <i>BRAF</i><sup><i>V600E</i></sup> mutant CRC cell line,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> to assess the effect of these compounds on MEK1/2 degradation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). As these compounds were designed to recruit VHL or CRBN E3 ligase, we first confirmed that both VHL and CRBN are expressed in HT-29 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">Figure S1</a>). We then employed Western blot to detect MEK1/2 protein levels in HT-29 cells with compounds <b>7</b>–<b>11</b> at a range of concentrations (0.3, 1, and 3 μM) for 24 h. We were pleased to find that compounds <b>7</b>–<b>9</b>, all of which contain VHL-1, a ligand of the E3 ligase VHL, can effectively reduce MEK1/2 protein levels (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Among them, compound <b>8</b>, which possesses a relatively long carbon linker, was the most effective MEK1/2 degrader (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Interestingly, compounds <b>10</b> and <b>11</b>, both of which contain the E3 ligase CRBN ligand pomalidomide, did not induce MEK1/2 degradation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of compounds <b>7</b>–<b>11</b> on MEK1/2 degradation in HT-29 cells. Cells were treated with DMSO or serial dilution of indicated compounds for 24 h. Compound concentrations are 0.3, 1, and 3 μM. MEK1/2 protein levels were determined by Western blots and normalized with α-tubulin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Design and Synthesis of MEK1/2 Degrader Compound <b>23</b> and Its Negative Control Compound <b>24</b></h3><div class="NLM_p">In addition to VHL-1, (<i>S</i>,<i>R</i>,<i>S</i>)-AHPC-Me (also known as VHL ligand 2) was reported to recruit VHL E3 ligase with higher affinity and has been utilized in the PROTAC field to effectively degrade the protein of interest.<a onclick="showRef(event, 'ref41 ref47'); return false;" href="javascript:void(0);" class="ref ref41 ref47">(41,47)</a> We therefore designed compound <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), which contains this higher affinity VHL ligand and the same linker as compound <b>8</b>. We also designed compound <b>24</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), a diastereoisomer of compound <b>23</b>, as a degrader negative control for compound <b>23</b>. The diastereoisomer of the VHL ligand in compound <b>24</b> is unable to bind VHL E3 ligase,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> while compounds <b>23</b> and <b>24</b> share the same MEK1/2 binding moiety and linker. Therefore, compound <b>24</b> was expected to display an inhibition effect toward MEK1/2 activity similar to that of compound <b>23</b>, but be incapable of inducing MEK1/2 degradation. The extremely high structural similarity between compounds <b>23</b> and <b>24</b> would make compound <b>24</b> an excellent negative control for compound <b>23</b>.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structures of compounds <b>23</b> and <b>24</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Synthesis of compounds <b>23</b> and <b>24</b> is depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The reaction of compounds <b>25</b> and <b>26</b>, both of which were prepared according to previously published procedures,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> with commercially available 11-((<i>tert</i>-butoxycarbonyl)amino)undecanoic acid and the subsequent deprotection provided compounds <b>27</b> and <b>28</b>. Reductive amination reactions of compounds <b>27</b> and <b>28</b> with compound <b>16</b> afforded compounds <b>23</b> and <b>24</b>.</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0013.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route for Preparing Compounds <b>23</b> and <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) HOAt, EDCI, NMM, DMSO, rt, overnight; (2) TFA, DCM, rt, 2 h; (b) NaBH<sub>3</sub>CN, DCM, MeOH, rt, overnight.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Inhibition of the Kinase Activity of MEK1 and MEK2 by Compounds <b>23</b> and <b>24</b> in Biochemical Assays</h3><div class="NLM_p">We first evaluated the effect of compounds <b>23</b> and <b>24</b> on inhibiting the kinase activity of MEK1 and MEK2 in the in vitro biochemical assays, which used a kinase-dead ERK mutant as the substrate of MEK1 or MEK2. Compounds <b>23</b> and <b>24</b> inhibited MEK1 in vitro with comparable potencies (<b>23</b>, IC<sub>50</sub> = 1500 ± 130 nM versus <b>24</b>, IC<sub>50</sub> = 1600 ± 30 nM), but they were about 17-fold less potent than the parental MEK inhibitor PD0325901 (IC<sub>50</sub> = 82 ± 2 nM) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). For MEK2, compound <b>23</b> was slightly more potent than compound <b>24</b> (<b>23</b>, IC<sub>50</sub> = 590 ± 25 nM versus <b>24</b>, IC<sub>50</sub> = 1500 ± 350 nM) and was approximately 4-fold less potent than PD0325901 (IC<sub>50</sub> = 110 ± 10 nM) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Overall, although compound <b>23</b> was less potent than the parental inhibitor PD0325901 at inhibiting the catalytic activity of MEK1 and MEK2 in vitro, we hypothesized that this potency for MEK1 and MEK2 is sufficient to lead to effective degradation of MEK1 and MEK2, based on a previous report that a weak inhibitor of p38α can result in an effective PROTAC of p38α.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In addition, compound <b>24</b> displayed in vitro potency similar to that of compound <b>23</b> for MEK1 and MEK2, providing evidence that compound <b>24</b> is an appropriate negative control for compound <b>23</b>.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compounds <b>23</b> and <b>24</b> inhibit the kinase activity of MEK1 and MEK2 in vitro. Inhibition value for each concentration point was determined by measuring MEK1/2 kinase activity on ERK phosphorylation and normalized with DMSO control point. IC<sub>50</sub> values of PD0325901 (A), compound <b>23</b> (B), and compound <b>24</b> (C) against MEK1 and MEK2 were analyzed in GraphPad Prism 8. For each concentration point, mean value ± SD from two independent experiments is shown in the curves.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. In Vitro Inhibition Potencies of Compounds <b>23</b> and <b>24</b> against MEK1 and MEK2 Kinases</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">kinase (IC<sub>50</sub>, nM)</th><th class="colsep0 rowsep0" align="center" char="±">PD0325901</th><th class="colsep0 rowsep0" align="center" char="±">compound <b>23</b></th><th class="colsep0 rowsep0" align="center" char="±">compound <b>24</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MEK1</td><td class="colsep0 rowsep0" align="char" char="±">82 ± 2</td><td class="colsep0 rowsep0" align="char" char="±">1500 ± 130</td><td class="colsep0 rowsep0" align="char" char="±">1600 ± 30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MEK2</td><td class="colsep0 rowsep0" align="char" char="±">110 ± 10</td><td class="colsep0 rowsep0" align="char" char="±">590 ± 25</td><td class="colsep0 rowsep0" align="char" char="±">1500 ± 350</td></tr></tbody></table></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Chemical-Induced Degradation of MEK1 and MEK2 in Cells</h3><div class="NLM_p">We next assessed the effects of compounds <b>23</b> and <b>24</b> on degradation of MEK1 and MEK2 and inhibition of the downstream signaling in four <i>BRAF</i><sup><i>V600E</i></sup> mutant cancer cell lines that have been widely used for evaluating BRAF or MEK inhibitors, HT-29 and COLO 205 (two CRC cell lines), and SK-MEL-28 and UACC 257 (two melanoma cell lines).<a onclick="showRef(event, 'ref46 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref46 ref49 ref50">(46,49,50)</a> As compound <b>23</b> was designed to recruit VHL E3 ligase, we first confirmed that VHL was expressed in these cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">Figure S1B</a>). We then examined MEK1/2 protein levels after 24 h treatment of compound <b>23</b> with different concentrations. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">S2A,B</a>, compound <b>23</b> effectively reduced MEK1/2 protein levels in these <i>BRAF</i><sup><i>V600E</i></sup> mutant CRC and melanoma cell lines in a concentration-dependent manner. In HT-29 cells, DC<sub>50</sub> (concentration that led to 50% degradation of the target protein) values for MEK1 and MEK2 were 31 ± 9 and 17 ± 2 nM, respectively (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). More than 80% of MEK1 and MEK2 protein levels were reduced by compound <b>23</b> at 0.5 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Similarly, compound <b>23</b> was highly potent in reducing MEK1 (DC<sub>50</sub> = 31 ± 1 nM) and MEK2 (DC<sub>50</sub> = 9.3 ± 5 nM) protein levels in SK-MEL-28 cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). Over 90% of MEK1 and MEK2 protein levels were reduced when SK-MEL-28 cells were treated with 0.3 μM compound <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). In addition, compound <b>23</b> was potent in reducing MEK1/2 protein levels in COLO 205 cells (DC<sub>50</sub> (MEK1) = 18 ± 7 nM, DC<sub>50</sub> (MEK2) = 11 ± 2 nM) and UACC257 cells (DC<sub>50</sub> (MEK1) = 56 ± 25 nM, DC<sub>50</sub> (MEK2) = 27 ± 19 nM) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">Figure S2</a>). In general, compound <b>23</b> was slightly more potent at degrading MEK2 than MEK1 in these cell lines. Interestingly, we did not observe an obvious “hook effect”<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> for compound <b>23</b> at up to 3 or 5 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">S2</a>). These results support our hypothesis that the relatively low in vitro potency of compound <b>23</b> for MEK1 (IC<sub>50</sub> = 1500 nM) and MEK2 (IC<sub>50</sub> = 590 nM) can still result in effective degradation of MEK1 and MEK2. Furthermore, we found that compound <b>23</b> effectively inhibited MEK phosphorylation (pMEK) at S218/222 mediated by RAF kinases and downstream ERK phosphorylation (pERK) at T202/Y204 catalyzed by MEK kinases<a onclick="showRef(event, 'ref4 ref52'); return false;" href="javascript:void(0);" class="ref ref4 ref52">(4,52)</a> in a concentration-dependent manner in these four cell lines (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">S2A,B</a>), but did not significantly change ERK protein levels in these cells. Collectively, our results indicate that compound <b>23</b> is a potent degrader of MEK1 and MEK2.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>23</b> reduces MEK1/2 protein levels and inhibits the downstream ERK signaling in a concentration-dependent manner. HT-29 (A) and SK-MEL-28 (B) cells were treated with DMSO or serial dilution of compound <b>23</b> for 24 h. The indicated protein levels were determined by Western blots and normalized with α-tubulin. DC<sub>50</sub> values of compound <b>23</b> for MEK1 and MEK2 in HT-29 (C) and SK-MEL-28 (D) cells were analyzed in GraphPad Prism 8. For each concentration point, mean value ± SD from three independent experiments is shown in the curves. The effects of PD0325901 and compounds <b>23</b> and <b>24</b> on reducing MEK1/2 protein levels and inhibiting the downstream ERK signaling were assessed in HT-29 (E) and SK-MEL-28 (F) cells treated with DMSO or different concentrations of indicated compounds for 24 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast to compound <b>23</b>, MEK1/2 inhibitor PD0325901 and the negative control compound <b>24</b>, up to 1 μM concentration, did not reduce MEK1 and MEK2 protein levels in both HT-29 and SK-MEL-28 cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E,F). Although PD0325901 inhibited ERK phosphorylation much more effectively than compound <b>23</b>, it was less potent than compound <b>23</b> at inhibiting MEK phosphorylation (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E,F), consistent with the previous report.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Furthermore, compound <b>23</b> was much more potent at inhibiting pMEK and pERK signals as compared to compound <b>24</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E,F), suggesting that the effect of compound <b>23</b> on inhibiting pMEK and pERK is mainly due to its MEK1/2 degradation activity, not its MEK1/2 inhibitory activity.</div><div class="NLM_p">We next performed time-course experiments to investigate the kinetics of MEK1/2 degradation and downstream signaling inhibition by compound <b>23</b>. We treated both HT-29 and SK-MEL-28 cells with 0.1 or 0.3 μM compound <b>23</b> for various short periods of time within 24 h. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, we found that MEK1 and MEK2 protein levels were reduced in both cell lines when treated with compound <b>23</b> for 2 h. Treatment with 0.3 μM compound <b>23</b> reduced over 50% MEK1 protein after 4 h and over 80% MEK1 protein after 8 h in both cell lines. Interestingly, we observed more degradation of MEK2 than that of MEK1 within 4 h in both cell lines treated with compound <b>23</b> at either 0.1 or 0.3 μM. We also found that MEK phosphorylation was inhibited after 2 h or longer treatment with compound <b>23</b>. In comparison, ERK phosphorylation was inhibited at later time points. Overall, MEK protein degradation and downstream signaling inhibition by compound <b>23</b> lasted for at least 24 h in these experiments. To determine whether compound <b>23</b> can induce MEK1/2 degradation for a longer period of time, we treated HT-29 cells with compound <b>23</b> for 6 days and evaluated MEK1 and MEK2 protein levels at various time points during the 6-day treatment (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">Figure S3</a>). Indeed, MEK1 and MEK2 degradation induced by compound <b>23</b> started after 3 h (the earliest time point in the study) and lasted for 6 days. Although PD0325901 inhibited pERK more effectively than compound <b>23</b> within the first 24 h, it induced very significant pERK rebound at the 48 and 96-h time points and less pERK rebound at the 72 and 144-h time points. This resembles acquired resistance to MEK1/2 inhibitors in cancer cells or clinical studies during a long-term treatment.<a onclick="showRef(event, 'ref4 ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref4 ref54 ref55 ref56">(4,54−56)</a> In contrast, compound <b>23</b> inhibited pMEK more potently and pERK more durably (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">Figure S3</a>). On the basis of these results, one might speculate that MEK1/2 degraders such as compound <b>23</b> could be more suitable to treat MEK1/2 inhibitor-resistant cancer cells. Therefore, further investigation into whether pharmacological degradation of MEK1/2 could provide a therapeutic strategy to overcome acquired resistance to MEK1/2 inhibition is warranted.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>23</b> induces degradation of MEK1 and MEK2 and suppresses the downstream ERK signaling in a time-dependent manner. HT-29 (A) and SK-MEL-28 (B) cells were treated with 0.1 or 0.3 μM compound <b>23</b> for 24 h. Cells were harvested at indicated time points within 24 h, and the indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next performed a set of rescue experiments in HT-29 and SK-MEL-28 cells to confirm that MEK1/2 degradation induced by compound <b>23</b> is dependent on the ubiquitin-proteasome system. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, pretreatment with PD0325901 (1 μM), which competes with compound <b>23</b> for binding MEK1/2, or the VHL ligand VH 032<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> (10 μM), which competes with compound <b>23</b> for binding VHL, significantly rescued the reduction of MEK1/2 protein levels induced by compound <b>23</b>. The pretreatments also blocked the effect of compound <b>23</b> on pMEK inhibition completely (HT-29) or partially (SK-MEL-28). As expected, pretreatment with PD0325901 effectively inhibited pERK formation, while pretreatment with VH 032 effectively rescued pERK formation. In addition, preincubation with proteasome inhibitor MG-132 (3 μM) or MLN4924 (1 μM), an inhibitor of NEDD8-activating enzyme (NAE) that mediates neddylation of cullin-RING E3 ubiquitin ligase (CRL) for CRL activation,<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> also successfully restored MEK1/2 protein levels and pMEK and pERK signals in both cell lines. These results, together with the effect of the negative control compound <b>24</b> on MEK1/2 protein levels and downstream signaling (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E,F), indicate that compound <b>23</b> degrades MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compound <b>23</b> induces MEK1/2 protein degradation through recruiting VHL E3 ubiquitin ligase and proteasome-dependent proteolysis. HT-29 and SK-MEL-28 cells were pretreated with DMSO, PD0325901 (1 μM), MG-132 (3 μM), MLN4924 (1 μM), or VH 032 (10 μM) for 2 h, followed by 10 h treatment with 0.3 μM compound <b>23</b> for rescue experiments. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Global Proteomic Profiling Studies</h3><div class="NLM_p">To assess the selectivity of compound <b>23</b>, we performed global proteomic profiling studies using label-free quantitative mass spectrometry in HT-29 cells treated with compound <b>23</b>, compound <b>24</b>, or DMSO. Among the 4085 quantified proteins, MEK1 and MEK2 protein levels were significantly reduced in terms of the relative abundance upon compound <b>23</b> treatment as compared to DMSO or compound <b>24</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A,B), while DMSO or the negative control compound <b>24</b> did not have a significant impact on the expression levels of any detected proteins (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C). The volcano plots in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A,B indicate that the protein levels of Cullin-9 and LENG8 were also affected by compound <b>23</b> treatment. However, we could not confirm the reduction of Cullin-9 and LENG8 protein levels by compound <b>23</b> via Western blotting (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">Figure S4</a>). Taken together, these results suggest that compound <b>23</b> is a highly selective degrader of MEK1 and MEK2.</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0009.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Global proteome analyses of HT-29 cells treated with compound <b>23</b>, compound <b>24</b>, or DMSO control. Cells were harvested for proteome analysis after 10 h treatment with 0.1% DMSO, 0.1 μM compound <b>23</b>, or 0.1 μM compound <b>24</b>. Volcano plots of the −log 10 (<i>P</i>-value) versus the log 2 ratio are displayed, with proteins outside the significance lines colored in pink or purple (FDR < 0.05). The <i>P</i>-values were calculated from the data of two technical replicates. (A) DMSO as compared to compound <b>23</b> treated samples; (B) compound <b>24</b> as compared to compound <b>23</b> treated samples; and (C) DMSO as compared to compound <b>24</b> treated samples.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Inhibition of Colorectal Cancer and Melanoma Cell Proliferation</h3><div class="NLM_p">We next evaluated the effect of compound <b>23</b> on inhibiting CRC and melanoma cell proliferation. Compound <b>23</b>, compound <b>24</b>, and PD0325901 were first tested in the four cancer cell lines (HT-29, SK-MEL-28, COLO 205, and UACC 257) used above by MTT assays to measure cell viability. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A,B and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">S5</a>, compound <b>23</b> effectively inhibited proliferation of these CRC and melanoma cells in a concentration-dependent manner, with GI<sub>50</sub> values ranging from 30 to 200 nM. The positive control, PD0325901, was very potent at inhibiting cell proliferation with GI<sub>50</sub> values between 0.2 and 4.2 nM. This result is consistent with the Western blot results that PD0325901 was more potent at inhibiting pERK as compared to compound <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E,F and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">S3</a>). Importantly, compound <b>24</b> did not display a pronounced antiproliferative effect in these cancer cell lines (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A,B and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">S5</a>). These results, together with the Western blot results that compound <b>23</b> effectively induced MEK1/2 degradation while compound <b>24</b> did not (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">S2</a>), suggest that the observed antiproliferative effects of compound <b>23</b> are mainly due to its degradation activity, but not its inhibitory activity toward MEK1/2 proteins.</div><figure id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0010.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compound <b>23</b> inhibits cell proliferation of HT-29 and SK-MEL-28 cells, phenocopied by MEK1/2 knockdown. MTT assay was conducted for HT-29 (A) and SK-MEL-28 (B) cells after 3 d treatment with DMSO or indicated compounds with serial dilutions. Data were analyzed in GraphPad Prism 8. For each concentration point, in duplicate, mean value ± SD from four independent experiments is shown in the curves. Clonogenic assay was performed in HT-29 (C) and SK-MEL-28 (D) cells for 14 d treatment with DMSO or indicated compounds with 10, 30, and 100 nM, in duplicate, respectively. Cells were then stained by crystal violet. The images here are representatives of three independent experiments. The shRNA-mediated MEK1/2 knockdown experiments were conducted in HT-29 (E) and SK-MEL-28 (F) cells. MTT assay was used to determine cell viability to assess the effect of MEK1/2 knockdown via shRNAs. For HT-29 and SK-MEL-28 cells, three and two independent experiments were performed for each treatment, respectively. Data were analyzed in GraphPad Prism 8 with two-way ANOVA to get <i>P</i>-values. For each time point, nine replicates, mean value ± SD from the independent experiments is shown in the plots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We further assessed the antiproliferative effect of compound <b>23</b> in HT-29 and SK-MEL-28 cells using a clonogenic assay,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> in which the cells were treated with compound <b>23</b>, PD0325901 (positive control), or compound <b>24</b> (negative control) for 14 days. Consistent with the MTT cell viability results, PD0325901 and compound <b>23</b>, but not compound <b>24</b>, significantly reduced cell colony formation in both cell lines (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>C,D). These results provide further evidence that the antiproliferative effects of compound <b>23</b> are mainly due to its MEK1/2 degradation activity, but not its MEK1/2 inhibitory activity. Collectively, our results indicate that compound <b>23</b> is a highly potent and selective MEK1/2 degrader with robust antiproliferative effects in <i>BRAF</i><sup><i>V600E</i></sup> CRC and melanoma cells.</div><div class="NLM_p last">We also conducted transient MEK1/2 knockdown with shRNAs in HT-29 and SK-MEL-28 cells to confirm that the MEK1/2 degradation activity of compound <b>23</b> is the main contributor to its antiproliferative effect. We first confirmed the specificity and effectiveness of MEK1 and MEK2 shRNAs via Western blotting (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">Figure S6</a>). We found that, in HT-29 cells, single knockdown of MEK1 or MEK2 suppressed cell growth significantly (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>E, <i>P</i> < 0.01), and double knockdown of MEK1 and MEK2 led to more profound inhibition of cell growth (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>E, <i>P</i> < 0.0001), suggesting that both MEK1 and MEK2 contribute to HT-29 cell growth. In SK-MEL-28 cells, single knockdown of MEK1 had no obvious effect on cell growth (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>F, ns), while MEK2 knockdown drastically decreased cell proliferation (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>F, <i>P</i> < 0.05), and MEK1/2 double knockdown exhibited an effect on cell proliferation (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>F, <i>P</i> < 0.01) similar to that of MEK2 knockdown. These results indicate that MEK1 and MEK2 have divergent roles in SK-MEL-28 cell growth, consistent with previously reported data.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Overall, knockdown of MEK1/2 via shRNAs phenocopied the antiproliferative effect of compound <b>23</b>, providing another piece of evidence that the on-target MEK1/2 degradation activity of compound <b>23</b> is the major contributor to its antiproliferative effect in cancer cells.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> In Vivo Mouse Pharmacokinetic (PK) Study</h3><div class="NLM_p">Last, we evaluated compound <b>23</b> in an in vivo mouse PK study. Following a single intraperitoneal (IP) injection of compound <b>23</b> to three mice, we observed that compound <b>23</b> displayed good plasma exposure with the maximum plasma concentration of 1400 nM detected at 0.5 h post dosing and plasma concentration of 710 nM at 8 h post dosing, which is approximately 3–20-fold higher than the GI<sub>50</sub> values of compound <b>23</b> in the tested cancer cell lines (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A,B, <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a>, and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">S5</a>). Additionally, compound <b>23</b> was well tolerated by the treated mice, and no clinical signs or adverse effects were observed. Thus, compound <b>23</b> is a well-characterized chemical tool and can be used for in vivo efficacy studies.</div><figure id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0011.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Plasma concentration of compound <b>23</b> following a single intraperitoneal administration for 8 h in male Swiss Albino mice. Data were analyzed in GraphPad Prism 8. For each time point, mean value ± SD from three mice is shown in the plot.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06758" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06758" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We discovered compound <b>23</b> as a first-in-class degrader of MEK1 and MEK2, the gatekeepers of ERK signaling. Compound <b>23</b> is a PD0325901-based VHL-recruiting small-molecule degrader of MEK1 and MEK2 with high potency and selectivity. We also developed compound <b>24</b> as a degrader negative control for compound <b>23</b>. Compound <b>24</b> is a diastereoisomer of compound <b>23</b>, thus sharing very high structural similarity with compound <b>23</b>. We show compounds <b>23</b> and <b>24</b> inhibited the catalytic activity of MEK1 and MEK2 in vitro with similar potencies. In cellular experiments, compound <b>23</b> potently induced the degradation of MEK1 and MEK2 in a concentration- and time-dependent manner with durable effect. In contrast, the negative control compound <b>24</b> did not decrease MEK1/2 protein levels in cells. Our rescue experiment results indicate that MEK1/2 degradation induced by compound <b>23</b> is mediated through recruiting VHL E3 ligase for MEK1/2 polyubiquitination and proteasome-dependent proteolysis. Our quantitative global proteomic profiling analyses revealed that compound <b>23</b> is a highly selective degrader of MEK1 and MEK2. Furthermore, we found that compound <b>23</b>, but not compound <b>24</b>, effectively inhibited CRC and melanoma cell proliferation, and knockdown of MEK1/2 via shRNAs phenocopied the antiproliferative effect of compound <b>23</b> in <i>BRAF</i><sup><i>V600E</i></sup> cancer cells. Moreover, compound <b>23</b> exhibited good plasma exposure in mice; thus it is suitable for in vivo efficacy studies. Overall, compounds <b>23</b> and <b>24</b> are a pair of well-characterized chemical tools for the research community to investigate the therapeutic potential of targeting MEK1 and MEK2. Further optimization of compound <b>23</b> into drug candidates may lead to novel, effective therapeutics for the treatment of CRC, melanoma, and other cancers.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21507" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21507" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry General Procedures</h3><div class="NLM_p">An Agilent 1200 series system with DAD detector and a 2.1 mm × 150 mm Zorbax 300SB-C18 5 μm column with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.4 mL/min for chromatography were used to obtain high-performance liquid chromatography (HPLC) spectra for all compounds. The gradient program was as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). A Waters Acquity I-Class Ultraperformance liquid chromatography (UPLC) system with a PDA detector was used to generate UPLC spectra for all compounds. Chromatography was performed using a 2.1 × 30 mm ACQUITY UPLC BEH C18 1.7 μm column with water containing 3% acetonitrile and 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.8 mL/min. The gradient program was as follows: 1–99% B (1–1.5 min) and 99–1% B (1.5–2.5 min). High-resolution mass spectra (HRMS) data were obtained in positive ion mode using an Agilent G1969A API-TOF with an electrospray ionization (ESI) source. Nuclear magnetic resonance (NMR) spectra were obtained on a Bruker DRX-600 spectrometer with 600 MHz for proton (<sup>1</sup>H NMR) or a Bruker DXI 800 MHz spectrometer with 800 MHz for proton (<sup>1</sup>H NMR) or 200 MHz for carbon (<sup>13</sup>C NMR); chemical shifts are reported in ppm (δ). Preparative HPLC was performed using an Agilent Prep 1200 series with UV detector set to 220 nm. Samples were injected into a Phenomenex Luna 75 × 30 mm, 5 μm, C18 column at room temperature. The flow rate was 40 mL/min. A linear gradient was used with 10% (or 50%) of MeOH (A) in H<sub>2</sub>O (with 0.1% TFA) (B) to 100% of MeOH (A). HPLC and UPLC were used to establish the purity of target compounds. All final compounds had >95% purity using the HPLC and UPLC methods described above.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-16-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,14-dioxo-3-oxa-2,7,15-triazaheptadecan-17-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>7</b>)</h4><div class="NLM_p last">To a solution of compound <b>18</b> (0.1 g, 0.2 mmol) and compound <b>16</b> (0.1 g, 0.2 mmol) in DCM (2 mL) and MeOH (1 mL) was added NaBH<sub>3</sub>CN (19 mg, 0.3 mmol). The reaction mixture was stirred at room temperature overnight. The reaction was monitored by UPLC. Upon completion, the reaction mixture was purified by preparative HPLC to give compound <b>7</b> (0.1 g, 0.1 mmol, 50% yield) as a white solid. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.94 (d, <i>J</i> = 6.0 Hz, 1H), 7.51–7.45 (m, 3H), 7.44–7.39 (m, 3H), 7.38 (d, <i>J</i> = 8.8 Hz, 1H), 7.07 (q, <i>J</i> = 8.9 Hz, 1H), 6.63 (td, <i>J</i> = 8.6, 4.0 Hz, 1H), 4.67–4.63 (m, 1H), 4.58–4.55 (m, 1H), 4.53 (d, <i>J</i> = 10.4 Hz, 1H), 4.51–4.49 (m, 1H), 4.36 (d, <i>J</i> = 15.4 Hz, 1H), 4.05 (t, <i>J</i> = 5.1 Hz, 2H), 3.90 (d, <i>J</i> = 11.0 Hz, 1H), 3.81 (dd, <i>J</i> = 10.9, 4.0 Hz, 1H), 3.23 (t, <i>J</i> = 5.9 Hz, 2H), 3.10–2.99 (m, 2H), 2.48 (s, 3H), 2.31 (dt, <i>J</i> = 14.9, 7.6 Hz, 1H), 2.28–2.25 (m, 1H), 2.25–2.19 (m, 1H), 2.11–2.06 (m, 1H), 2.05–1.99 (m, 2H), 1.73 (p, <i>J</i> = 7.8 Hz, 2H), 1.61 (dt, <i>J</i> = 13.7, 6.9 Hz, 2H), 1.42 (p, <i>J</i> = 7.1 Hz, 2H), 1.37 (q, <i>J</i> = 7.2 Hz, 2H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.17 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>45</sub>H<sub>56</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>, 1006.3004; found, 1006.2983.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,18-dioxo-3-oxa-2,7,19-triazahenicosan-21-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>8</b>)</h4><div class="NLM_p last">Using the above procedure for the synthesis of compound <b>7</b>, compound <b>8</b> was obtained as a white solid (51% yield). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.94 (s, 1H), 7.50–7.44 (m, 3H), 7.44–7.39 (m, 3H), 7.38 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.07 (td, <i>J</i> = 9.2, 6.9 Hz, 1H), 6.64 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 4.64 (s, 1H), 4.59–4.55 (m, 1H), 4.54 (d, <i>J</i> = 12.6 Hz, 1H), 4.51–4.49 (m, 1H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.13–4.03 (m, 2H), 3.90 (d, <i>J</i> = 11.0 Hz, 1H), 3.81 (dd, <i>J</i> = 11.0, 3.9 Hz, 1H), 3.24 (t, <i>J</i> = 5.8 Hz, 2H), 3.10–2.97 (m, 2H), 2.48 (s, 3H), 2.32–2.26 (m, 1H), 2.26–2.19 (m, 2H), 2.11–2.06 (m, 1H), 2.05–1.99 (m, 2H), 1.71 (p, <i>J</i> = 7.7 Hz, 2H), 1.59 (tq, <i>J</i> = 14.3, 6.9 Hz, 2H), 1.44–1.35 (m, 2H), 1.32–1.29 (m, 6H), 1.29–1.26 (m, 4H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.20 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>49</sub>H<sub>64</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>, 1062.3630; found, 1062.4355.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,18-dioxo-3,10,13,16-tetraoxa-2,7,19-triazahenicosan-21-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>9</b>)</h4><div class="NLM_p last">Using the above procedure for the synthesis of compound <b>7</b>, compound <b>9</b> was obtained as a white solid (45% yield). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.99 (s, 1H), 7.49–7.45 (m, 3H), 7.42 (d, <i>J</i> = 8.2 Hz, 2H), 7.41–7.38 (m, 1H), 7.37 (dt, <i>J</i> = 8.4, 1.3 Hz, 1H), 7.05 (td, <i>J</i> = 9.2, 7.0 Hz, 1H), 6.64 (td, <i>J</i> = 8.8, 4.2 Hz, 1H), 4.68 (s, 1H), 4.56 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 4.53 (d, <i>J</i> = 11.4 Hz, 1H), 4.51–4.48 (m, 1H), 4.39 (d, <i>J</i> = 15.5 Hz, 1H), 4.14–4.00 (m, 4H), 3.88 (d, <i>J</i> = 11.1 Hz, 1H), 3.81 (dd, <i>J</i> = 10.9, 3.8 Hz, 1H), 3.77 (dd, <i>J</i> = 5.7, 4.3 Hz, 2H), 3.71–3.65 (m, 2H), 3.63–3.58 (m, 4H), 3.58–3.55 (m, 2H), 3.30–3.23 (m, 4H), 2.48 (s, 3H), 2.24 (ddt, <i>J</i> = 13.2, 7.5, 1.8 Hz, 1H), 2.11–2.06 (m, 1H), 2.05–2.00 (m, 2H), 1.04 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.23 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>46</sub>H<sub>58</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>9</sub>S<sup>+</sup>, 1068.3008; found, 1068.2927.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>N</i>-(3-((2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>10</b>)</h4><div class="NLM_p last">Using the above procedure for the synthesis of compound <b>7</b>, compound <b>10</b> was obtained as a white solid (42% yield). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.46 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.38 (dd, <i>J</i> = 10.7, 2.0 Hz, 1H), 7.35 (dd, <i>J</i> = 7.7, 2.3 Hz, 1H), 7.26 (dt, <i>J</i> = 8.3, 1.3 Hz, 1H), 6.96 (d, <i>J</i> = 7.0 Hz, 1H), 6.95–6.88 (m, 2H), 6.57 (td, <i>J</i> = 8.8, 4.7 Hz, 1H), 4.98–4.93 (m, 1H), 4.15–4.07 (m, 2H), 3.93–3.83 (m, 2H), 3.77 (ddd, <i>J</i> = 6.0, 4.5, 1.9 Hz, 2H), 3.45 (dd, <i>J</i> = 5.7, 4.2 Hz, 2H), 3.34 (t, <i>J</i> = 5.2 Hz, 4H), 2.82 (ddd, <i>J</i> = 17.5, 13.8, 5.3 Hz, 1H), 2.73–2.68 (m, 1H), 2.60 (qd, <i>J</i> = 13.1, 4.5 Hz, 1H), 2.09–2.03 (m, 2H), 2.02–1.97 (m, 1H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.19 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>33</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>7</sub><sup>+</sup>, 809.1402; found, 809.1627.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>N</i>-(3-((2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>11</b>)</h4><div class="NLM_p last">Using the above procedure for the synthesis of compound <b>7</b>, compound <b>11</b> was obtained as a white solid (40% yield). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.54 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.37–7.34 (m, 1H), 7.32 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.29 (dt, <i>J</i> = 8.6, 1.2 Hz, 1H), 7.09 (d, <i>J</i> = 8.5 Hz, 1H), 7.06–7.00 (m, 2H), 6.58 (td, <i>J</i> = 8.7, 4.6 Hz, 1H), 4.95–4.91 (m, 1H), 4.19–4.02 (m, 2H), 3.77 (td, <i>J</i> = 5.7, 2.3 Hz, 2H), 3.37 (t, <i>J</i> = 5.8 Hz, 2H), 3.33 (t, <i>J</i> = 5.9 Hz, 2H), 2.79 (ddd, <i>J</i> = 17.5, 13.9, 5.4 Hz, 1H), 2.67 (ddd, <i>J</i> = 17.5, 4.6, 2.6 Hz, 1H), 2.52 (qd, <i>J</i> = 13.1, 4.5 Hz, 1H), 2.11–2.00 (m, 2H), 1.97–1.91 (m, 1H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.17 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>29</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>, 765.1140; found, 765.1063.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 2-(2-(1,3-Dioxolan-2-yl)ethoxy)isoindoline-1,3-dione (<b>13</b>)</h4><div class="NLM_p last">To a solution of compound <b>12</b> (1 g, 5.5 mmol) in DMF (5 mL) were added 2-hydroxyisoindoline-1,3-dione (0.9 g, 5.5 mmol) and DBU (1.6 mL, 11.0 mmol). The reaction mixture was heated to 50 °C for 4 h. The reaction was monitored by UPLC. Upon completion, the reaction mixture was purified by reverse column chromatography to give compound <b>13</b> (1.2 g, 4.5 mmol, 82% yield) as a yellow solid. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.36–7.40 (m, 4H), 5.10 (t, <i>J</i> = 4.7 Hz, 1H), 4.31 (t, <i>J</i> = 6.6 Hz, 2H), 3.98–3.93 (m, 2H), 3.87–3.82 (m, 2H), 2.09 (td, <i>J</i> = 6.6, 4.6 Hz, 2H). UPLC > 95%, <i>t</i><sub>R</sub> = 1.12 min. MS (ESI) [M + H]<sup>+</sup> = 264.1638.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>O</i>-(2-(1,3-Dioxolan-2-yl)ethyl)hydroxylamine (<b>14</b>)</h4><div class="NLM_p last">A solution of compound <b>13</b> (1.2 g, 4.5 mmol) and hydrazine monohydrate (0.66 mL, 13.5 mmol) in MeOH (4 mL) and DCM (2 mL) was stirred at room temperature for 2 h. The reaction was monitored by UPLC. Upon completion, the solvent was evaporated, water (20 mL) was added to the residue, and then extracted with DCM (20 mL × 3). The combined organic layers were washed with 5% NaHCO<sub>3</sub> solution and brine, then dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to obtain compound <b>14</b> (0.53 g, 4.0 mmol, 89% yield) as a colorless liquid without further purification.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>N</i>-(2-(1,3-Dioxolan-2-yl)ethoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>15</b>)</h4><div class="NLM_p last">To a solution of compound <b>14</b> (0.53 g, 4.0 mmol) and 3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzoic acid (1.6 g, 4.0 mmol) in DMSO (5 mL) were added HOAt (0.8 g, 6.0 mmol), EDCI (1.2 g, 6.0 mmol), and 4-methylmorpholine (1.3 mL, 12.0 mmol). The reaction mixture was stirred at room temperature overnight. The reaction was monitored by UPLC. Upon completion, the reaction mixture was purified by reverse column chromatography to give compound <b>15</b> (1.5 g, 3.0 mmol, 75% yield) as a white solid. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.45 (dd, <i>J</i> = 10.6, 1.9 Hz, 1H), 7.38 (t, <i>J</i> = 7.0 Hz, 1H), 7.35 (dt, <i>J</i> = 8.5, 1.5 Hz, 1H), 7.03 (td, <i>J</i> = 9.2, 7.0 Hz, 1H), 6.60 (td, <i>J</i> = 8.7, 4.2 Hz, 1H), 4.97 (t, <i>J</i> = 4.9 Hz, 1H), 3.96 (t, <i>J</i> = 6.7 Hz, 2H), 3.95–3.90 (m, 2H), 3.86–3.78 (m, 2H), 1.97 (td, <i>J</i> = 6.7, 4.7 Hz, 2H). UPLC > 95%, <i>t</i><sub>R</sub> = 1.95 min. MS (ESI) [M – H]<sup>+</sup> = 507.0261.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 3,4-Difluoro-2-((2-fluoro-4-iodophenyl)amino)-<i>N</i>-(3-oxopropoxy)benzamide (<b>16</b>)</h4><div class="NLM_p last">To a solution of compound <b>15</b> (0.5 g, 1.0 mmol) in THF (3 mL) was added 3 M HCl (5 mL). The reaction mixture was stirred at room temperature for 6 h. The reaction was monitored by UPLC. Upon completion, the reaction mixture was cooled to 0 °C, neutralized with saturated NaHCO<sub>3</sub> to pH 7–8, and then extracted with EtOAc (30 mL × 3) to obtain compound <b>16</b> (0.37 g, 0.8 mmol, 80% yield) as a colorless liquid without further purification.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(7-Aminoheptanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>18</b>)</h4><div class="NLM_p last">To a solution of compound <b>17</b> (0.5 g, 1.2 mmol) and 7-((<i>tert</i>-butoxycarbonyl)amino)heptanoic acid (0.3 g, 1.2 mmol) in DMSO (5 mL) were added HOAt (0.2 g, 1.8 mmol), EDCI (0.4 g, 1.8 mmol), and 4-methylmorpholine (0.4 mL, 3.6 mmol). The reaction mixture was stirred at room temperature overnight. The reaction was monitored by UPLC. Upon completion, the reaction mixture was purified by reverse column chromatography to give the desired intermediate (0.7 g, 1.1 mmol, 92% yield) as a white solid. This intermediate (0.7 g, 1.1 mmol) then was dissolved in DCM (2 mL), and trifluoroacetic acid (2 mL) was added. The reaction mixture was stirred at room temperature for 2 h. The reaction was monitored by UPLC. Once completion, the solvent was evaporated, and the residue was purified by reverse column chromatography to obtain compound <b>18</b> (0.6 g, 1.0 mmol, 91% yield) as a white solid. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.80 (s, 1H), 7.56 (dd, <i>J</i> = 8.2, 4.1 Hz, 2H), 7.51 (d, <i>J</i> = 7.9 Hz, 2H), 4.65–4.60 (m, 1H), 4.58–4.56 (m, 1H), 4.55–4.52 (m, 1H), 4.52–4.49 (m, 1H), 4.41 (d, <i>J</i> = 15.7 Hz, 1H), 3.91 (d, <i>J</i> = 11.0 Hz, 1H), 3.81 (dd, <i>J</i> = 11.0, 3.9 Hz, 1H), 2.92 (t, <i>J</i> = 7.7 Hz, 2H), 2.59 (s, 3H), 2.35–2.27 (m, 2H), 2.27–2.22 (m, 1H), 2.12–2.04 (m, 1H), 1.70–1.60 (m, 4H), 1.51–1.34 (m, 4H), 1.04 (s, 9H).</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,18-dioxo-3-oxa-2,7,19-triazahenicosan-21-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>23</b>)</h4><div class="NLM_p last">To a solution of compound <b>27</b> (0.3 g, 0.4 mmol) and compound <b>16</b> (0.2 g, 0.4 mmol) in DCM (2 mL) and MeOH (1 mL) was added NaBH<sub>3</sub>CN (38 mg, 0.6 mmol). The reaction mixture was stirred at room temperature overnight. The reaction was monitored by UPLC. Upon completion, the reaction mixture was purified by preparative HPLC to give compound <b>23</b> (0.2 g, 0.2 mmol, 50% yield) as a white solid. <sup>1</sup>H NMR (800 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.97 (s, 1H), 7.43 (dq, <i>J</i> = 17.7, 10.2 Hz, 6H), 7.37 (d, <i>J</i> = 8.6 Hz, 1H), 7.06 (q, <i>J</i> = 8.5 Hz, 1H), 6.64 (td, <i>J</i> = 8.8, 4.0 Hz, 1H), 5.00 (q, <i>J</i> = 6.9 Hz, 1H), 4.62 (s, 1H), 4.57 (t, <i>J</i> = 8.3 Hz, 1H), 4.43 (d, <i>J</i> = 5.2 Hz, 1H), 4.08 (t, <i>J</i> = 5.2 Hz, 2H), 3.88 (d, <i>J</i> = 11.1 Hz, 1H), 3.75 (dd, <i>J</i> = 11.1, 3.9 Hz, 1H), 3.23 (q, <i>J</i> = 9.5, 7.7 Hz, 2H), 3.04 (q, <i>J</i> = 7.1, 6.3 Hz, 2H), 2.49 (s, 3H), 2.29 (dt, <i>J</i> = 15.0, 7.7 Hz, 1H), 2.22 (ddd, <i>J</i> = 27.3, 13.7, 8.1 Hz, 2H), 2.03 (p, <i>J</i> = 5.5 Hz, 2H), 1.95 (ddd, <i>J</i> = 13.3, 9.1, 4.5 Hz, 1H), 1.72 (p, <i>J</i> = 7.9 Hz, 2H), 1.63–1.55 (m, 2H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H), 1.38 (q, <i>J</i> = 7.6 Hz, 2H), 1.33–1.25 (m, 10H), 1.04 (s, 9H). <sup>13</sup>C NMR (201 MHz, methanol-<i>d</i><sub>4</sub>) δ 175.83, 173.02, 172.09, 161.51, 155.12, 154.91, 153.86, 153.67, 152.98, 148.34, 145.63, 134.44, 133.62, 132.28, 131.01, 130.29 (2 × C), 127.45 (2 × C), 127.30, 125.58, 125.11, 120.84, 119.88, 111.42, 82.36, 77.56, 70.75, 60.37, 58.78, 57.77, 49.93, 38.57, 36.47, 36.30, 30.30, 30.23 (2 × C), 30.14, 29.99, 27.38, 27.21, 26.86 (4 × C), 26.81, 25.50, 22.19, 15.37. HPLC > 99%, <i>t</i><sub>R</sub> = 4.58 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>50</sub>H<sub>66</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>, 1076.3787; found, 1076.3871.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (2<i>R</i>,4<i>S</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,18-dioxo-3-oxa-2,7,19-triazahenicosan-21-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>24</b>)</h4><div class="NLM_p last">Using the above procedure for the synthesis of compound <b>23</b>, compound <b>24</b> was obtained as a white solid (51% yield). <sup>1</sup>H NMR (800 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.98 (s, 1H), 7.52 (d, <i>J</i> = 7.9 Hz, 2H), 7.46–7.42 (m, 3H), 7.41 (dd, <i>J</i> = 8.8, 5.1 Hz, 1H), 7.35 (d, <i>J</i> = 8.6 Hz, 1H), 7.05 (q, <i>J</i> = 8.6 Hz, 1H), 6.62 (td, <i>J</i> = 8.7, 4.1 Hz, 1H), 5.02 (q, <i>J</i> = 7.9, 7.1 Hz, 1H), 4.55 (t, <i>J</i> = 7.4 Hz, 1H), 4.49 (s, 1H), 4.45 (p, <i>J</i> = 4.5 Hz, 1H), 4.08 (t, <i>J</i> = 5.1 Hz, 2H), 3.94 (dd, <i>J</i> = 10.8, 4.9 Hz, 1H), 3.69 (dd, <i>J</i> = 10.9, 3.5 Hz, 1H), 3.23 (t, <i>J</i> = 5.8 Hz, 2H), 3.02 (t, <i>J</i> = 7.8 Hz, 2H), 2.49 (s, 3H), 2.28 (dt, <i>J</i> = 15.1, 7.9 Hz, 1H), 2.23–2.16 (m, 2H), 2.10 (dt, <i>J</i> = 12.5, 5.8 Hz, 1H), 2.02 (dd, <i>J</i> = 11.0, 5.6 Hz, 2H), 1.70 (p, <i>J</i> = 7.8 Hz, 2H), 1.59–1.51 (m, 2H), 1.45 (d, <i>J</i> = 7.0 Hz, 3H), 1.36 (p, <i>J</i> = 7.5 Hz, 2H), 1.30–1.26 (m, 4H), 1.25–1.21 (m, 6H), 1.05 (s, 9H). <sup>13</sup>C NMR (201 MHz, methanol-<i>d</i><sub>4</sub>) δ 176.21, 173.27, 172.01, 161.18, 155.11, 154.89, 153.86, 153.66, 152.99, 148.14, 145.45, 134.43, 133.71, 132.26, 130.78, 130.21 (2 × C), 128.13, 127.66 (2 × C), 125.60, 125.10, 120.84, 119.82, 111.41, 82.37, 77.59, 70.31, 60.52, 59.32, 56.68, 49.71, 38.76, 36.37, 35.49, 30.27, 30.23, 30.21, 30.15, 29.98, 27.37, 27.21, 26.84 (4 × C), 26.70, 25.50, 22.41, 15.43. HPLC > 99%, <i>t</i><sub>R</sub> = 4.52 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>50</sub>H<sub>66</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>, 1076.3787; found, 1076.4045.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(11-Aminoundecanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>27</b>)</h4><div class="NLM_p last">To a solution of compound <b>25</b> (0.5 g, 1.1 mmol) and 11-((<i>tert</i>-butoxycarbonyl)amino)undecanoic acid (0.33 g, 1.1 mmol) in DMSO (5 mL) were added HOAt (0.2 g, 1.6 mmol), EDCI (0.3 g, 1.6 mmol), and 4-methylmorpholine (0.36 mL, 3.3 mmol). The reaction mixture was stirred at room temperature overnight. The reaction was monitored by UPLC. Upon completion, the reaction mixture was purified by reverse column chromatography to give the desired intermediate (0.73 g, 1.0 mmol, 91% yield) as a white solid. This intermediate (0.73 g, 1.0 mmol) then was dissolved in DCM (2 mL), and trifluoroacetic acid (2 mL) was added. The reaction mixture was stirred at room temperature for 2 h. The reaction was monitored by UPLC. Once completion, the solvent was evaporated, and the residue was purified by reverse column chromatography to obtain compound <b>27</b> (0.57 g, 0.9 mmol, 90% yield) as a white solid. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.94 (s, 1H), 7.52–7.38 (m, 4H), 5.00 (q, <i>J</i> = 7.4 Hz, 1H), 4.63 (s, 1H), 4.60–4.53 (m, 1H), 4.44 (dt, <i>J</i> = 4.4, 2.3 Hz, 1H), 3.88 (dt, <i>J</i> = 11.2, 1.8 Hz, 1H), 3.75 (dd, <i>J</i> = 11.0, 4.0 Hz, 1H), 3.31 (p, <i>J</i> = 1.7 Hz, 2H), 2.91 (t, <i>J</i> = 7.7 Hz, 2H), 2.30 (ddd, <i>J</i> = 14.1, 8.3, 7.0 Hz, 1H), 2.27–2.22 (m, 1H), 2.22–2.17 (m, 1H), 1.96 (ddd, <i>J</i> = 13.3, 9.0, 4.6 Hz, 1H), 1.68–1.56 (m, 4H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H), 1.43–1.29 (m, 13H), 1.04 (s, 9H). UPLC > 95%, <i>t</i><sub>R</sub> = 1.14 min. MS (ESI) [M + H]<sup>+</sup> = 628.4438.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Kinase Inhibition Assay</h3><div class="NLM_p last">The inhibition potencies of compounds against MEK1 and MEK2 kinases were determined using the HotSpot kinase assay by Reaction Biology Co. This assay measures MEK kinase activity on ERK phosphorylation. Briefly, after the compounds were incubated with the kinase reaction mixture of MEK and ERK proteins for 20 min at rt, <sup>33</sup>P-ATP (specific activity 10 μCi/μL) was delivered into the reaction mixture to initiate the reaction for incubation for 2 h at rt. Radioactivity was then detected by filter-binding method. Kinase activity data were expressed as the percent remaining kinase activity in samples as compared to DMSO reactions. Purified kinase proteins, MEK1 (PV3303, Thermo Fisher Scientific) at 100 nM, MEK2 (PV3615, Thermo Fisher Scientific) at 150 nM, and ERK kinase-dead mutant K52R (Reaction Biology) at 5 μM, were used in the reactions. IC<sub>50</sub> values were determined using 10-concentration 3-fold serial dilution (top concentrations for PD0325901, compound <b>23</b>, and compound <b>24</b> were 3, 30, and 30 μM, respectively) with DMSO as control point in two independent experiments.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Cell Culture</h3><div class="NLM_p last">Cells were cultivated in DMEM or RPMI-1640 medium supplemented with 10% FBS, 100 units/mL of penicillin, and 100 μg/mL of streptomycin. Mycoplasma elimination using a LookOut Mycoplasma Elimination Kit (MP0030, Sigma-Aldrich) was conducted before the cells were used for experiments.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Western Blot</h3><div class="NLM_p last">Cells were lysed on ice for 30 min with the lysis buffer (50 mM Tris pH 7.4, 1% IGEPAL CA-630, 150 mM NaCl, 1 mM EDTA, and 1 mM AESBF), supplemented with protease and phosphatase inhibitor cocktail (A32961, Thermo Fisher Scientific). The sample was centrifuged at 12 000<i>g</i> for 10 min at 4 °C to get supernatant as cell lysate. Protein concentrations were quantified using the Pierce rapid gold BCA protein assay kit. The primary antibodies used were MEK1 (2352, CST), MEK2 (9147, CST), pMEK (9121, CST), ERK (4696, CST), pERK (4370, CST), α-tubulin (T6074, Sigma-Aldrich), VHL (68547, CST), CRBN (HPA045910, Sigma-Aldrich), Cullin-9 (A300-098A, Bethyl Laboratories), and LENG8 (A304-947A, Bethyl Laboratories). Fluorescence-labeled secondary antibodies (IRDye 680, 800, LI-COR) and the OdysseyCLx imaging system (LI-COR) were used to obtain protein signals, which were then analyzed by Image Studio Lite software (LI-COR). DC<sub>50</sub> values were obtained with GraphPad Prism 8 from the data of three independent experiments.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Proteomics Studies</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Protein Extraction and Digestion</h4><div class="NLM_p last">Cells with indicated treatments were harvested and lysed in lysis buffer (6 M guanidine hydrochloride, 100 mM Tris-HCl pH 8.0). Sonication (5 s on, 5 s off, 2 × 30 s) was performed to shear genome DNA. Lysate was centrifugated for 30 min at 3500<i>g</i> at 4 °C, and the supernatant was transferred to a clean tube, diluted 50% with Milli-Q H<sub>2</sub>O, then precipitated by addition of cold acetone (4 times the original volume), and placed at −20 °C overnight. Precipitated proteins were brought down at 20 000<i>g</i> at 4 °C for 10 min, and washed with cold acetone after the supernatant was discarded. The pellet was air-dried at rt for 5 min and solubilized with 50 mM Tris-HCl (pH 8.0), 8 M urea. Protein concentration was determined (BCA assay), and protein was reduced with 5 mM DTT (dithiothreitol), alkylated with 15 mM IAA (iodoacetamide) in the dark, and then diluted with buffer 25 mM Tris (pH 8.0) and 1 mM CaCl<sub>2</sub> (3 times the original volume). The final urea concentration was 2 M. Trypsin was added into protein solution with a 1:100 ratio (trypsin:protein), and digested 12–16 h or overnight at rt.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Mass Spectrometry Analysis</h4><div class="NLM_p">Peptides were cleaned up by C18 stage tips, and the concentration was determined (Peptide assay, 23275, Thermo Fisher Scientific). The clean peptides were dissolved in 0.1% formic acid and analyzed on a Q-Exactive HF-X coupled with an Easy nanoLC 1200 (Thermo Fisher Scientific, San Jose, CA). Next, 0.5 μg of peptides were loaded onto an Acclain PepMap RSLC C18 column (250 mm × 75 μm ID, C18, 2 μm, Thermo Fisher Scientific). Analytical separation of all peptides was achieved with 130 min gradient. A linear gradient of 5–30% buffer B over 110 min was executed at a 300 nL/min flow rate followed by a ramp to 100% B in 5 min, and 15 min wash with 100% B, where buffer A was aqueous 0.1% formic acid, and buffer B was 80% acetonitrile and 0.1% formic acid.</div><div class="NLM_p last">LC–MS experiments were performed in a data-dependent mode with full MS (externally calibrated to a mass accuracy of <5 ppm and a resolution of 60 000 at <i>m</i>/<i>z</i> 200) followed by high energy collision-activated dissociation-MS/MS of the top 20 most intense ions with a resolution of 15 000 at <i>m</i>/<i>z</i> 200. High energy collision-activated dissociation-MS/MS was used to dissociate peptides at a normalized collision energy of 27 eV in the presence of nitrogen bath gas atoms. Dynamic exclusion was 30.0 s. Each sample was subjected to two technical LC–MS replicates.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> MS Data Analysis</h4><div class="NLM_p last">Mass spectra processing and peptide identification were performed on the Andromeda search engine in MaxQuant software (Version 1.6.0.16) against the human UniProt database (UP000005640). All searches were conducted with a defined modification of cysteine carbamidomethylation, with methionine oxidation and protein amino-terminal acetylation as dynamic modifications. Peptides were confidently identified using a target-decoy approach with a peptide false discovery rate (FDR) of 1% and a protein FDR of 1%. A minimum peptide length of 7 amino acids was required, maximally two missed cleavages were allowed, initial mass deviation for precursor ion was up to 7 ppm, and the maximum allowed mass deviation for fragment ions was 0.5 Da. Data processing and statistical analysis were performed on Perseus (Version 1.6.0.7). Protein quantitation was performed on duplicate runs, and a two-sample <i>t</i> test statistics was used to report statistically significant expression fold-changes.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Cell Viability Assay</h3><div class="NLM_p last">Cells (2000 cells per well) were seeded into 96-well microplates. After 20 h, cells were treated with 0.1% DMSO or indicated serial dilutions of compounds in duplicate or triplicate for 3 days. Cell viability was tested using MTT (M6494, Thermo Fisher Scientific) or WST-8 reagent (CK04, Dojindo). Briefly, 12 mM MTT was prepared in PBS, or WST-8 reagent was warmed to room temperature. Twenty microliters of MTT or CCK-8 was then added to each well, and the plates were kept in an incubator at 37 °C for 3 h in the dark. For the WST-8 assay, signal was obtained after the incubation. For the MTT assay, cell medium was replaced with 200 μL of DMSO after the incubation, and then cell plates were kept at 37 °C for another 30 min. Absorbance signals for MTT and WST-8 were read at 540 and 450 nm, respectively, with 690 nm as reference performed with Infinite F PLEX plate reader (TECAN, Morrisville, NC). GI<sub>50</sub> values were analyzed using GraphPad Prism 8 from the data of at least three independent experiments.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Clonogenic Assay</h3><div class="NLM_p last">HT-29 or SK-MEL-28 cells (300 cells per well) were seeded in 12-well plates. After 20 h, cells were treated with DMSO or indicated concentrations of compounds in duplicate for 2 weeks. Medium containing indicated compounds was changed every 3 days. Cells were then fixed and stained with the solution containing 6% (v/v) glutaraldehyde (G5882, Sigma-Aldrich) and 0.5% (w/v) crystal violet for 1 h. To remove the background color, the plates with stained cells were immersed in running water until clear colonies were observed. The plates were dried at rt. The staining images were obtained with Epson Perfection V600 Photo.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> shRNA-Mediated MEK Knockdown</h3><div class="NLM_p last">HEK293T cells were transfected with PMD (VSVG)/pCMVΔ8.2/pLKO.1 plasmids for lentivirus packaging. Forty-eight hours later, viruses were harvested in the medium and filtered. Next, 1.5 mL of viruses with 10 μg/mL of Polybrene (TR-1003, Sigma-Aldrich) were added into 6 cm dishes to infect HT-29 or SK-MEL-28 cells. After 24 h, medium was changed into fresh full medium with 2 μg/mL of puromycin (P8833, Sigma-Aldrich) for 48 h for selection, and then 2000 cells of shControl, shMEK1 (TRCN0000002329, Sigma-Aldrich), shMEK2 (TRCN0000195037, Sigma-Aldrich), or shMEK1/2 cells, in nonuplicate, were seeded into 96-well plates for 4 d growth. MTT assay was performed every day to obtain cell viability signals. Meanwhile, knockdown efficiency of each shRNA was examined by Western blot.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Mouse PK Study</h3><div class="NLM_p last">A standard in vivo PK study was conducted for compound <b>23</b> using three male Swiss Albino mice. The mice were administered intraperitoneally with solution formulation of compound <b>23</b> at a 50 mg/kg dose. Sixty microliters of blood samples was collected from each mouse at 0.5, 2, and 8 h. Plasma was harvested by centrifugation of blood and stored at −70 ± 10 °C until analysis. Pharmacokinetic analysis was performed using the NCA module of Phoenix WinNonlin (Version 7.0). Plasma samples were quantified by fit-for-purpose LC–MS/MS method (LLOQ: 5.02 ng/mL for plasma). The formulation of compound <b>23</b> was 5% NMP, 5% Solutol HS-15, and 90% normal saline.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Statistic Methods</h3><div class="NLM_p last">For all data, the number of biologically independent experiments and technical replicates, error bars, and <i>P</i>-values are described in figure legends, respectively. At least two independent experiments were conducted for all biological studies. The proteomics study was conducted in duplicate. Two-tailed Student’s <i>t</i> tests and two-way ANOVA were used for the indicated analyses, respectively, <i>P</i> ≥ 0.05, ns; 0.01 < <i>P</i> < 0.05, *; 0.001 < <i>P</i> < 0.01, **; <i>P</i> < 0.001, ***.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01528" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30203" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30203" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01528?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01528</a>. <ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings of <b>7</b> and <b>8</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Detection of CRBN expression in HT-29 cells and VHL expression in CRC and melanoma cell lines; time-dependent degradation of MEK1 and MEK2 by compound <b>23</b> in COLO 205 and UACC257 cell lines; Western blot analyses of long-term treatment of PD0325901 and compound <b>23</b> in HT-29 cells; Western blot validation of proteomics studies in HT-29 cells; antiproliferative effects of compound <b>23</b> in COLO 205 and UACC257 cell lines; shRNA-mediated MEK1/2 protein knockdown data in HT-29 and SK-MEL-28 cell lines; and <sup>1</sup>H, <sup>13</sup>C, HPLC, and MS spectra of compounds <b>23</b> and <b>24</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of <b>7</b>–<b>11</b>, <b>23</b>, and <b>24</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_001.csv">jm9b01528_si_001.csv (0.3 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf">jm9b01528_si_002.pdf (1.9 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_003.csv">jm9b01528_si_003.csv (1.01 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01528" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84308" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84308" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#016b686f662f6d6874416c72726c2f646574"><span class="__cf_email__" data-cfemail="741e1d1a135a181d0134190707195a111001">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1a70737b74347073745a77696977347f7e6f"><span class="__cf_email__" data-cfemail="6e04070f00400407002e031d1d03400b0a1b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jieli Wei</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianping Hu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ling Xie</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Margaret S. Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xian Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.W. and J.H. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3172-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16814" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16814" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">J.J. acknowledges the support by an endowed professorship by the Icahn School of Medicine at Mount Sinai. We thank Drs. Poulikos Poulikakos, Ernesto Guccione, and E. Premkumar Reddy for providing HT-29, SK-MEL-28, UACC257, and COLO 205 cell lines.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">MEK1/2 (MAP2K1/2)</td><td class="NLM_def"><p class="first last">Mitogen-Activated Protein Kinase Kinase 1/2</p></td></tr><tr><td class="NLM_term">RTK</td><td class="NLM_def"><p class="first last">receptor tyrosine kinase</p></td></tr><tr><td class="NLM_term">MAPKKK</td><td class="NLM_def"><p class="first last">Mitogen-Activated Protein Kinase Kinase Kinase</p></td></tr><tr><td class="NLM_term">MAPKK</td><td class="NLM_def"><p class="first last">Mitogen-Activated Protein Kinase Kinase</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">non-small-cell lung cancer</p></td></tr><tr><td class="NLM_term">CRC</td><td class="NLM_def"><p class="first last">colorectal cancers</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">MDM2</td><td class="NLM_def"><p class="first last">mouse double minute 2 homologue</p></td></tr><tr><td class="NLM_term">cIAP</td><td class="NLM_def"><p class="first last">cellular inhibitor of apoptosis protein 1</p></td></tr><tr><td class="NLM_term">PPARγ</td><td class="NLM_def"><p class="first last">peroxisome proliferator-activated receptor γ</p></td></tr><tr><td class="NLM_term">CLIPTAC</td><td class="NLM_def"><p class="first last">in-cell click-formed PROTAC</p></td></tr><tr><td class="NLM_term">CRL</td><td class="NLM_def"><p class="first last">cullin-RING E3 ubiquitin ligase</p></td></tr><tr><td class="NLM_term">NAE</td><td class="NLM_def"><p class="first last">NEDD8-activating enzyme</p></td></tr><tr><td class="NLM_term">KSR1</td><td class="NLM_def"><p class="first last">kinase suppressor of Ras 1</p></td></tr><tr><td class="NLM_term">LENG8</td><td class="NLM_def"><p class="first last">leukocyte receptor cluster member 8</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">IP</td><td class="NLM_def"><p class="first last">intraperitoneal</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62088" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62088" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">MEK1/2 dual-specificity protein kinases: structure and regulation</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>417</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2011.11.145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.bbrc.2011.11.145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=22177953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1Oqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2012&pages=5-10&author=R.+Roskoski&title=MEK1%2F2+dual-specificity+protein+kinases%3A+structure+and+regulation&doi=10.1016%2Fj.bbrc.2011.11.145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1/2 dual-specificity protein kinases: Structure and regulation</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">417</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  MEK1 and MEK2 are related protein kinases that participate in the RAS-RAF-MEK-ERK signal transduction cascade.  This cascade participates in the regulation of a large variety of processes including apoptosis, cell cycle progression, cell migration, differentiation, metab., and proliferation.  Moreover, oncogenic mutations in RAS or B-RAF are responsible for a large proportion of human cancers.  MEK1 is activated by phosphorylation of Ser-218 and Ser-222 in its activation segment as catalyzed by RAF kinases in an intricate process that involves a KSR scaffold.  In addn. to functioning as a scaffold, the kinase activity of KSR is also required for MEK activation.  MEK1 regulation is unusual in that Ser-212 phosphorylation in its activation segment is inhibitory.  Moreover, active ERK kinase catalyzes a feedback inhibitory phosphorylation of MEK1 Thr-292 that serves to down-regulate the pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRX9pGR2iVKbVg90H21EOLACvtfcHk0lhDp7oMQiXbwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1Oqtg%253D%253D&md5=4bdfbf6ebaa90055a314922fa1056929</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2011.11.145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2011.11.145%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DMEK1%252F2%2520dual-specificity%2520protein%2520kinases%253A%2520structure%2520and%2520regulation%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2012%26volume%3D417%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.bbrc.2011.11.145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scholl, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumesic, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barragan, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bissonauth, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khavari, P. A.</span></span> <span> </span><span class="NLM_article-title">Mek1/2 MAPK kinases are essential for mammalian development, homeostasis, and Raf-induced hyperplasia</span>. <i>Dev. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">629</span>, <span class="refDoi"> DOI: 10.1016/j.devcel.2007.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.devcel.2007.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=17419998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVOhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=615-629&author=F.+A.+Schollauthor=P.+A.+Dumesicauthor=D.+I.+Barraganauthor=K.+Haradaauthor=V.+Bissonauthauthor=J.+Charronauthor=P.+A.+Khavari&title=Mek1%2F2+MAPK+kinases+are+essential+for+mammalian+development%2C+homeostasis%2C+and+Raf-induced+hyperplasia&doi=10.1016%2Fj.devcel.2007.03.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mek1/2 MAPK kinases are essential for mammalian development, homeostasis, and Raf-induced hyperplasia</span></div><div class="casAuthors">Scholl, Florence A.; Dumesic, Phillip A.; Barragan, Deborah I.; Harada, Kazutoshi; Bissonauth, Vickram; Charron, Jean; Khavari, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">615-629</span>CODEN:
                <span class="NLM_cas:coden">DCEEBE</span>;
        ISSN:<span class="NLM_cas:issn">1534-5807</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The p42/p44 mitogen-activated protein kinase (MAPK) cascade includes Ras, Raf, Mek, and Erk MAPK.  To det. the effect of a full knockout at a single level of this signaling pathway in mammals, and to investigate functional redundancy between Mek1 and Mek2, we disrupted these genes in murine and human epidermis.  Loss of either protein alone produced no phenotype, whereas combined Mek1/2 deletion in development or adulthood abolished Erk1/2 phosphorylation and led to hypoproliferation, apoptosis, skin barrier defects, and death.  Conversely, a single copy of either allele was sufficient for normal development.  Combined Mek1/2 loss also abolished Raf-induced hyperproliferation.  Human tissue deficient in either Mek isoform was normal, whereas loss of both proteins led to hypoplasia, which was rescued by active Erk2 expression.  These data indicate that Mek1/2 are functionally redundant in the epidermis, where they act as a linear relay in the MAPK pathway to mediate development and homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogiBLCMiwCarVg90H21EOLACvtfcHk0lhDp7oMQiXbwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVOhsLY%253D&md5=739b373ceee916e03504d56745f87444</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2007.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2007.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DScholl%26aufirst%3DF.%2BA.%26aulast%3DDumesic%26aufirst%3DP.%2BA.%26aulast%3DBarragan%26aufirst%3DD.%2BI.%26aulast%3DHarada%26aufirst%3DK.%26aulast%3DBissonauth%26aufirst%3DV.%26aulast%3DCharron%26aufirst%3DJ.%26aulast%3DKhavari%26aufirst%3DP.%2BA.%26atitle%3DMek1%252F2%2520MAPK%2520kinases%2520are%2520essential%2520for%2520mammalian%2520development%252C%2520homeostasis%252C%2520and%2520Raf-induced%2520hyperplasia%26jtitle%3DDev.%2520Cell%26date%3D2007%26volume%3D12%26spage%3D615%26epage%3D629%26doi%3D10.1016%2Fj.devcel.2007.03.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. T.</span></span> <span> </span><span class="NLM_article-title">MAPK signal pathways in the regulation of cell proliferation in mammalian cells</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1038/sj.cr.7290105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fsj.cr.7290105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=11942415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BD383gvV2qsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=9-18&author=W.+Zhangauthor=H.+T.+Liu&title=MAPK+signal+pathways+in+the+regulation+of+cell+proliferation+in+mammalian+cells&doi=10.1038%2Fsj.cr.7290105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">MAPK signal pathways in the regulation of cell proliferation in mammalian cells</span></div><div class="casAuthors">Zhang Wei; Liu Hui Tu</div><div class="citationInfo"><span class="NLM_cas:title">Cell research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-18</span>
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    </div><div class="casAbstract">MAPK families play an important role in complex cellular programs like proliferation, differentiation, development, transformation, and apoptosis.  At least three MAPK families have been characterized: extracellular signal-regulated kinase (ERK), Jun kinase (JNK/SAPK) and p38 MAPK.  The above effects are fulfilled by regulation of cell cycle engine and other cell proliferation related proteins.  In this paper we discussed their functions and cooperation with other signal pathways in regulation of cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQjYqac378pM4HWj6b1v8uTfW6udTcc2eZ4LxWNHSL7k7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD383gvV2qsA%253D%253D&md5=a466a156756e0afbbd63fbd7f9ec88e8</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.cr.7290105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.cr.7290105%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BT.%26atitle%3DMAPK%2520signal%2520pathways%2520in%2520the%2520regulation%2520of%2520cell%2520proliferation%2520in%2520mammalian%2520cells%26jtitle%3DCell%2520Res.%26date%3D2002%26volume%3D12%26spage%3D9%26epage%3D18%26doi%3D10.1038%2Fsj.cr.7290105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caunt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. J.</span></span> <span> </span><span class="NLM_article-title">MEK1 and MEK2 inhibitors and cancer therapy: the long and winding road</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1038/nrc4000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnrc4000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26399658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGhsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=577-592&author=C.+J.+Cauntauthor=M.+J.+Saleauthor=P.+D.+Smithauthor=S.+J.+Cook&title=MEK1+and+MEK2+inhibitors+and+cancer+therapy%3A+the+long+and+winding+road&doi=10.1038%2Fnrc4000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1 and MEK2 inhibitors and cancer therapy: the long and winding road</span></div><div class="casAuthors">Caunt, Christopher J.; Sale, Matthew J.; Smith, Paul D.; Cook, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">577-592</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The role of the ERK signaling pathway in cancer is thought to be most prominent in tumors in which mutations in the receptor tyrosine kinases RAS, BRAF, CRAF, MEK1 or MEK2 drive growth factor-independent ERK1 and ERK2 activation and thence inappropriate cell proliferation and survival.  New drugs that inhibit RAF or MEK1 and MEK2 have recently been approved or are currently undergoing late-stage clin. evaluation.  In this Review, the authors consider the ERK pathway, focusing particularly on the role of MEK1 and MEK2, the 'gatekeepers' of ERK1/2 activity.  The authors discuss their validation as drug targets, the merits of targeting MEK1 and MEK2 vs. BRAF and the mechanisms of action of different inhibitors of MEK1 and MEK2.  The authors also consider how some of the systems-level properties (intrapathway regulatory loops and wider signaling network connections) of the ERK pathway present a challenge for the success of MEK1 and MEK2 inhibitors, discuss mechanisms of resistance to these inhibitors, and review their clin. progress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnTcSz8CvRcLVg90H21EOLACvtfcHk0lhMjgyVGzMjzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGhsb%252FO&md5=b1fd44051024a59608dd61310a437c65</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc4000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc4000%26sid%3Dliteratum%253Aachs%26aulast%3DCaunt%26aufirst%3DC.%2BJ.%26aulast%3DSale%26aufirst%3DM.%2BJ.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26atitle%3DMEK1%2520and%2520MEK2%2520inhibitors%2520and%2520cancer%2520therapy%253A%2520the%2520long%2520and%2520winding%2520road%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D577%26epage%3D592%26doi%3D10.1038%2Fnrc4000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoshino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuruo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ari-i, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, M.</span></span> <span> </span><span class="NLM_article-title">Constitutive activation of the 41-/43-kDa mitogen-activated protein kinase signaling pathway in human tumors</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">813</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1202367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fsj.onc.1202367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=9989833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADyaK1MXht1eht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=813-822&author=R.+Hoshinoauthor=Y.+Chataniauthor=T.+Yamoriauthor=T.+Tsuruoauthor=H.+Okaauthor=O.+Yoshidaauthor=Y.+Shimadaauthor=S.+Ari-iauthor=H.+Wadaauthor=J.+Fujimotoauthor=M.+Kohno&title=Constitutive+activation+of+the+41-%2F43-kDa+mitogen-activated+protein+kinase+signaling+pathway+in+human+tumors&doi=10.1038%2Fsj.onc.1202367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Constitutive activation of the 41-/43-kDa mitogen-activated protein kinase signaling pathway in human tumors</span></div><div class="casAuthors">Hoshino, Rika; Chatani, Yuji; Yamori, Takao; Tsuruo, Takashi; Oka, Hiroya; Yoshida, Osamu; Shimada, Yutaka; Ari-i, Shigeki; Wada, Hiromi; Fujimoto, Jiro; Kohno, Michiaki</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">813-822</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">The 41-kDa and 43-kDa mitogen-activated protein (MAP) kinases play a pivotal role in the mitogenic signal transduction pathway and are essential components of the MAP kinase cascade, which includes MAP kinase kinase (MEK) and Raf-1.  As aberrant activation of signal transducing mols. such as Ras and Raf-1 has been linked with cancer, the authors examd. whether constitutive activation of the 41-/43-kDa MAP kinases is assocd. with the neoplastic phenotype of 138 tumor cell lines and 102 primary tumors derived from various human organs.  Constitutive activation of the MAP kinases was obsd. in 50 tumor cell lines (36.2%) in a rather tissue-specific manner: cell lines derived from pancreas, colon, lung, ovary and kidney showed esp. high frequencies with a high degree of MAP kinase activation, while those derived from brain, esophagus, stomach, liver and of hematopoietic origin showed low frequencies with a limited degree of MAP kinase activation.  The authors also detected constitutive activation of the 41-/43-kDa MAP kinases in a relatively large no. of primary human tumors derived from kidney, colon and lung tissues but not from liver tissue.  Many tumor cells, in which point mutations of ras genes were detected, showed constitutive activation of MAP kinases, however, there were also many exceptions to this observation.  In contrast, the activation of the 41-/43-kDa MAP kinases was accompanied by the activation of Raf-1 in the majority of tumor cells and was completely assocd. with the activation of MEK and p90rsk in all the tumor cells examd.  These results suggest that the constitutive activation of 41-/43-kDa MAP kinases in tumor cells is not due to the disorder of MAP kinases themselves, but is due to the disorder of Raf-1, Ras, or some other signaling mols. upstream of Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRICRJXnIoerVg90H21EOLACvtfcHk0lhMjgyVGzMjzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXht1eht70%253D&md5=719a702e589f259a22f53f7a2be0e94a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202367%26sid%3Dliteratum%253Aachs%26aulast%3DHoshino%26aufirst%3DR.%26aulast%3DChatani%26aufirst%3DY.%26aulast%3DYamori%26aufirst%3DT.%26aulast%3DTsuruo%26aufirst%3DT.%26aulast%3DOka%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DO.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DAri-i%26aufirst%3DS.%26aulast%3DWada%26aufirst%3DH.%26aulast%3DFujimoto%26aufirst%3DJ.%26aulast%3DKohno%26aufirst%3DM.%26atitle%3DConstitutive%2520activation%2520of%2520the%252041-%252F43-kDa%2520mitogen-activated%2520protein%2520kinase%2520signaling%2520pathway%2520in%2520human%2520tumors%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D813%26epage%3D822%26doi%3D10.1038%2Fsj.onc.1202367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovly, C. M.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of receptor tyrosine kinase activation in cancer</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1186/s12943-018-0782-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1186%2Fs12943-018-0782-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29304823" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1-13&author=Z.+Duauthor=C.+M.+Lovly&title=Mechanisms+of+receptor+tyrosine+kinase+activation+in+cancer&doi=10.1186%2Fs12943-018-0782-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0782-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0782-4%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DLovly%26aufirst%3DC.%2BM.%26atitle%3DMechanisms%2520of%2520receptor%2520tyrosine%2520kinase%2520activation%2520in%2520cancer%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D1%26epage%3D13%26doi%3D10.1186%2Fs12943-018-0782-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Medarde, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, E.</span></span> <span> </span><span class="NLM_article-title">Ras in cancer and developmental diseases</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">344</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1177/1947601911411084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1177%2F1947601911411084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=21779504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWlsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=344-358&author=A.+Fernandez-Medardeauthor=E.+Santos&title=Ras+in+cancer+and+developmental+diseases&doi=10.1177%2F1947601911411084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Ras in cancer and developmental diseases</span></div><div class="casAuthors">Fernandez-Medarde, Alberto; Santos, Eugenio</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">344-358</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Somatic, gain-of-function mutations in ras genes were the first specific genetic alterations identified in human cancer about 3 decades ago.  Studies during the last quarter century have characterized the Ras proteins as essential components of signaling networks controlling cellular proliferation, differentiation, or survival.  The oncogenic mutations of the H-ras, N-ras, or K-ras genes frequently found in human tumors are known to throw off balance the normal outcome of those signaling pathways, thus leading to tumor development.  Oncogenic mutations in a no. of other upstream or downstream components of Ras signaling pathways (including membrane RTKs or cytosolic kinases) have been detected more recently in assocn. with a variety of cancers.  Interestingly, the oncogenic Ras mutations and the mutations in other components of Ras/MAPK signaling pathways appear to be mutually exclusive events in most tumors, indicating that deregulation of Ras-dependent signaling is the essential requirement for tumorigenesis.  In contrast to sporadic tumors, sep. studies have identified germline mutations in Ras and various other components of Ras signaling pathways that occur in specific assocn. with a no. of different familial, developmental syndromes frequently sharing common phenotypic cardiofaciocutaneous features.  Finally, even without being a causative force, defective Ras signaling has been cited as a contributing factor to many other human illnesses, including diabetes and immunol. and inflammatory disorders.  We aim this review at summarizing and updating current knowledge on the contribution of Ras mutations and altered Ras signaling to development of various tumoral and nontumoral pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC8FC347lm6bVg90H21EOLACvtfcHk0lhMjgyVGzMjzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWlsbs%253D&md5=048ad92dd5959250e4a2b8e0319ff225</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1177%2F1947601911411084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911411084%26sid%3Dliteratum%253Aachs%26aulast%3DFernandez-Medarde%26aufirst%3DA.%26aulast%3DSantos%26aufirst%3DE.%26atitle%3DRas%2520in%2520cancer%2520and%2520developmental%2520diseases%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D344%26epage%3D358%26doi%3D10.1177%2F1947601911411084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karoulia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavathiotis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulikakos, P. I.</span></span> <span> </span><span class="NLM_article-title">New perspectives for targeting RAF kinase in human cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">676</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.79</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnrc.2017.79" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=28984291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqt7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=676-691&author=Z.+Karouliaauthor=E.+Gavathiotisauthor=P.+I.+Poulikakos&title=New+perspectives+for+targeting+RAF+kinase+in+human+cancer&doi=10.1038%2Fnrc.2017.79"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">New perspectives for targeting RAF kinase in human cancer</span></div><div class="casAuthors">Karoulia, Zoi; Gavathiotis, Evripidis; Poulikakos, Poulikos I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">676-691</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The discovery that a subset of human tumors is dependent on mutationally deregulated BRAF kinase intensified the development of RAF inhibitors to be used as potential therapeutics.  The US Food and Drug Administration (FDA)-approved second-generation RAF inhibitors vemurafenib and dabrafenib have elicited remarkable responses and improved survival of patients with BRAF-V600E/K melanoma, but their effectiveness is limited by resistance.  Beyond melanoma, current clin. RAF inhibitors show modest efficacy when used for colorectal and thyroid BRAF-V600E tumors or for tumors harboring BRAF alterations other than the V600 mutation.  Accumulated exptl. and clin. evidence indicates that the complex biochem. mechanisms of RAF kinase signalling account both for the effectiveness of RAF inhibitors and for the various mechanisms of tumor resistance to them.  Recently, a no. of next-generation RAF inhibitors, with diverse structural and biochem. properties, have entered preclin. and clin. development.  In this Review, we discuss the current understanding of RAF kinase regulation, mechanisms of inhibitor action and related clin. resistance to these drugs.  The recent elucidation of crit. structural and biochem. aspects of RAF inhibitor action, combined with the availability of a no. of structurally diverse RAF inhibitors currently in preclin. and clin. development, will enable the design of more effective RAF inhibitors and RAF-inhibitor-based therapeutic strategies, tailored to different clin. contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEUrKAljS7SrVg90H21EOLACvtfcHk0ljpACoohgGLfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqt7fN&md5=7e8ee4222467a2d5c12b23ccce99fc62</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.79%26sid%3Dliteratum%253Aachs%26aulast%3DKaroulia%26aufirst%3DZ.%26aulast%3DGavathiotis%26aufirst%3DE.%26aulast%3DPoulikakos%26aufirst%3DP.%2BI.%26atitle%3DNew%2520perspectives%2520for%2520targeting%2520RAF%2520kinase%2520in%2520human%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D676%26epage%3D691%26doi%3D10.1038%2Fnrc.2017.79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimaldi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Festino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanella, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strudel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascierto, P. A.</span></span> <span> </span><span class="NLM_article-title">MEK inhibitors in the treatment of metastatic melanoma and solid tumors</span>. <i>Am. J. Clin. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1007/s40257-017-0292-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1007%2Fs40257-017-0292-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=28537004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BC1cngsFCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=745-754&author=A.+M.+Grimaldiauthor=E.+Simeoneauthor=L.+Festinoauthor=V.+Vanellaauthor=M.+Strudelauthor=P.+A.+Ascierto&title=MEK+inhibitors+in+the+treatment+of+metastatic+melanoma+and+solid+tumors&doi=10.1007%2Fs40257-017-0292-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">MEK Inhibitors in the Treatment of Metastatic Melanoma and Solid Tumors</span></div><div class="casAuthors">Grimaldi Antonio M; Simeone Ester; Festino Lucia; Vanella Vito; Strudel Martina; Ascierto Paolo A</div><div class="citationInfo"><span class="NLM_cas:title">American journal of clinical dermatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">745-754</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The mitogen-activated protein kinase (MAPK) cascade is an intracellular signaling pathway involved in the regulation of cellular proliferation and the survival of tumor cells.  Several different mutations, involving BRAF or NRAS, exert an oncogenic effect by activating the MAPK pathway, resulting in an increase in cellular proliferation.  These mutations have become targets for new therapeutic strategies in melanoma and other cancers.  Selective MEK inhibitors have the ability to inhibit growth and induce cell death in BRAF- and NRAS-mutant melanoma cell lines.  MEK inhibitor therapy in combination with a BRAF inhibitor is more effective and less toxic than treatment with a BRAF inhibitor alone, and has become the standard of care for patients with BRAF-mutated melanoma.  Trametinib was the first MEK inhibitor approved for the treatment of BRAF-mutated metastatic melanoma not previously treated with BRAF inhibitors, and is also approved in combination with the BRAF inhibitor dabrafenib.  Furthermore, cobimetinib is another MEK inhibitor approved for the treatment of BRAF-mutated metastatic melanoma in combination with a BRAF inhibitor, vemurafenib.  The MEK inhibitor binimetinib in combination with the BRAF inhibitor encorafenib is in clinical development.  The addition of an anti-PD-1/PD-L1 agent, such as pembrolizumab, durvalumab or atezolizumab, to combined BRAF and MEK inhibition has shown considerable promise, with several trials ongoing in metastatic melanoma.  Binimetinib has also shown efficacy in NRAS-mutated melanoma patients.  Future possibilities for MEK inhibitors in advanced melanoma, as well as other solid tumors, include their use in combination with other targeted therapies (e.g. anti-CDK4/6 inhibitors) and/or various immune-modulating antibodies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTRMQVeZbuZD8RAV4D_Ins_fW6udTcc2eZ0uvnfMLS5hrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cngsFCgtA%253D%253D&md5=23b16eb79f7203a8acd54e8c9249357d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs40257-017-0292-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40257-017-0292-y%26sid%3Dliteratum%253Aachs%26aulast%3DGrimaldi%26aufirst%3DA.%2BM.%26aulast%3DSimeone%26aufirst%3DE.%26aulast%3DFestino%26aufirst%3DL.%26aulast%3DVanella%26aufirst%3DV.%26aulast%3DStrudel%26aufirst%3DM.%26aulast%3DAscierto%26aufirst%3DP.%2BA.%26atitle%3DMEK%2520inhibitors%2520in%2520the%2520treatment%2520of%2520metastatic%2520melanoma%2520and%2520solid%2520tumors%26jtitle%3DAm.%2520J.%2520Clin.%2520Dermatol.%26date%3D2017%26volume%3D18%26spage%3D745%26epage%3D754%26doi%3D10.1007%2Fs40257-017-0292-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. I.</span></span> <span> </span><span class="NLM_article-title">MEK1/2 Inhibitors: molecular activity and resistance mechanisms</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">849</span>– <span class="NLM_lpage">862</span>, <span class="refDoi"> DOI: 10.1053/j.seminoncol.2015.09.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1053%2Fj.seminoncol.2015.09.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26615130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=849-862&author=P.+K.+Wuauthor=J.+I.+Park&title=MEK1%2F2+Inhibitors%3A+molecular+activity+and+resistance+mechanisms&doi=10.1053%2Fj.seminoncol.2015.09.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1/2 Inhibitors: Molecular Activity and Resistance Mechanisms</span></div><div class="casAuthors">Wu, Pui-Kei; Park, Jong-In</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">849-862</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Aberrant activation of the three-layered protein kinase cascade, Raf/MEK/ERK, is often detected in human cancer, which is mainly attributed to the oncogenic alterations of RAF, or its upstream activators RAS or cell surface receptor tyrosine kinases.  Deregulated activity of the Raf/MEK/ERK pathway drives uncontrolled tumor cell proliferation and survival, thus providing a rational therapeutic target for the treatment of many cancers.  While Raf, MEK1/2, and ERK1/2 are equally important targets for the design of therapeutic small mol. wt. inhibitors, the effort to develop MEK1/2-specific inhibitors has been greatly successful.  Particularly, MEK1/2 have been relatively advantageous for the design of highly selective ATP-noncompetitive inhibitors.  Indeed, a plethora of highly selective and potent MEK1/2 inhibitors are now available and many of those inhibitors have been evaluated for their therapeutic potential.  Herein, we review different MEK1/2 inhibitors that have been studied for their inhibitory mechanisms and therapeutic potential in cancer.  Some of the key structural features of MEK1/2 that are important for the efficacy of these inhibitors are also discussed.  In addn., we discuss current challenges and future prospective in using these advanced MEK1/2 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCuCxmKMCT7bVg90H21EOLACvtfcHk0ljpACoohgGLfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWks7fF&md5=444f0ee51369a9e9e8baa29ddd8d90a5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1053%2Fj.seminoncol.2015.09.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.seminoncol.2015.09.023%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%2BK.%26aulast%3DPark%26aufirst%3DJ.%2BI.%26atitle%3DMEK1%252F2%2520Inhibitors%253A%2520molecular%2520activity%2520and%2520resistance%2520mechanisms%26jtitle%3DSemin.%2520Oncol.%26date%3D2015%26volume%3D42%26spage%3D849%26epage%3D862%26doi%3D10.1053%2Fj.seminoncol.2015.09.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, K. L.</span></span> <span> </span><span class="NLM_article-title">RAS isoforms and mutations in cancer at a glance</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">1287</span>– <span class="NLM_lpage">1292</span>, <span class="refDoi"> DOI: 10.1242/jcs.182873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1242%2Fjcs.182873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26985062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKltr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2016&pages=1287-1292&author=G.+A.+Hobbsauthor=C.+J.+Derauthor=K.+L.+Rossman&title=RAS+isoforms+and+mutations+in+cancer+at+a+glance&doi=10.1242%2Fjcs.182873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">RAS isoforms and mutations in cancer at a glance</span></div><div class="casAuthors">Hobbs, G. Aaron; Der, Channing J.; Rossman, Kent L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1287-1292</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">RAS proteins (KRAS4A, KRAS4B, NRAS and HRAS) function as GDP-GTP-regulated binary on-off switches, which regulate cytoplasmic signaling networks that control diverse normal cellular processes.  Gain-of-function missense mutations in RAS genes are found in ∼ 25% of human cancers, prompting interest in identifying anti-RAS therapeutic strategies for cancer treatment.  However, despite more than three decades of intense effort, no anti-RAS therapies have reached clin. application.  Contributing to this failure has been an underestimation of the complexities of RAS.  First, there is now appreciation that the four human RAS proteins are not functionally identical.  Second, with >130 different missense mutations found in cancer, there is an emerging view that there are mutation-specific consequences on RAS structure, biochem. and biol., and mutation-selective therapeutic strategies are needed.  In this Cell Science at a Glance article and accompanying poster, we provide a snapshot of the differences between RAS isoforms and mutations, as well as the current status of anti-RAS drug-discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNufX89IRJk7Vg90H21EOLACvtfcHk0ljpACoohgGLfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKltr7I&md5=bf50d212b900fb437cad2b650a9dc07a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1242%2Fjcs.182873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.182873%26sid%3Dliteratum%253Aachs%26aulast%3DHobbs%26aufirst%3DG.%2BA.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26aulast%3DRossman%26aufirst%3DK.%2BL.%26atitle%3DRAS%2520isoforms%2520and%2520mutations%2520in%2520cancer%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2016%26volume%3D129%26spage%3D1287%26epage%3D1292%26doi%3D10.1242%2Fjcs.182873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span> <span> </span><span class="NLM_article-title">Targeting ERK, an Achilles’ heel of the MAPK pathway, in cancer therapy</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2018.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.apsb.2018.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30109180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ot1CmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=552-562&author=F.+Liuauthor=X.+Yangauthor=M.+Gengauthor=M.+Huang&title=Targeting+ERK%2C+an+Achilles%E2%80%99+heel+of+the+MAPK+pathway%2C+in+cancer+therapy&doi=10.1016%2Fj.apsb.2018.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting ERK, an Achilles' Heel of the MAPK pathway, in cancer therapy</span></div><div class="casAuthors">Liu Feifei; Yang Xiaotong; Geng Meiyu; Huang Min</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">552-562</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">The mitogen-activated protein kinases (MAPK) pathway, often known as the RAS-RAF-MEK-ERK signal cascade, functions to transmit upstream signals to its downstream effectors to regulate physiological process such as cell proliferation, differentiation, survival and death.  As the most frequently mutated signaling pathway in human cancer, targeting the MAPK pathway has long been considered a promising strategy for cancer therapy.  Substantial efforts in the past decades have led to the clinical success of BRAF and MEK inhibitors.  However, the clinical benefits of these inhibitors are compromised by the frequently occurring acquired resistance due to cancer heterogeneity and genomic instability.  This review briefly introduces the key protein kinases involved in this pathway as well as their activation mechanisms.  We also generalize the correlations between mutations of MAPK members and human cancers, followed by a summarization of progress made on the development of small molecule MAPK kinases inhibitors.  In particular, this review highlights the potential advantages of ERK inhibitors in overcoming resistance to upstream targets and proposes that targeting ERK kinase may hold a promising prospect for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-a-G-lYtjXxdUSIfHIn4FfW6udTcc2eYKl1b6IwvPMrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ot1CmsA%253D%253D&md5=37768ff6015b0792e009e7b53a357d65</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2018.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2018.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DM.%26atitle%3DTargeting%2520ERK%252C%2520an%2520Achilles%25E2%2580%2599%2520heel%2520of%2520the%2520MAPK%2520pathway%252C%2520in%2520cancer%2520therapy%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2018%26volume%3D8%26spage%3D552%26epage%3D562%26doi%3D10.1016%2Fj.apsb.2018.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. S.</span></span> <span> </span><span class="NLM_article-title">BRAF and MEK inhibitors: use and resistance in BRAF-mutated cancers</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1007/s40265-018-0884-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1007%2Fs40265-018-0884-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29488071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFKns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=549-566&author=J.+N.+Sanchezauthor=T.+Wangauthor=M.+S.+Cohen&title=BRAF+and+MEK+inhibitors%3A+use+and+resistance+in+BRAF-mutated+cancers&doi=10.1007%2Fs40265-018-0884-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">BRAF and MEK Inhibitors: Use and Resistance in BRAF-Mutated Cancers</span></div><div class="casAuthors">Sanchez, Jaquelyn N.; Wang, Ton; Cohen, Mark S.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">549-566</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">The mitogen activated protein kinase/extracellular signal-related kinase (MAPK/ERK) signaling pathway serves an integral role in growth, proliferation, differentiation, migration, and survival of all mammalian cells.  Aberrant signaling of this pathway is often obsd. in several types of hematol. and solid malignancies.  The most frequent insult to this signaling cascade, leading to its constitutive activation, is to the serine/threonine kinase rapidly accelerating fibrosarcoma (RAF).  Considering this, the development and approval of various small-mol. inhibitors targeting the MAPK/ERK pathway has become a mainstay of treatment as either mono- or combination therapy in these cancers.  Although effective initially, a major clin. barrier with these inhibitors is the relapse of patients due to drug resistance.  Knowledge of the mechanisms of resistance to these drugs is still premature, highlighting the need for a more in-depth understanding of how patients become insensitive to these pharmacol. interventions.  Herein, we will succinctly summarize the milestones in the approval of select MAPK/ERK pathway inhibitors, their use in patients, and major modes of resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW2IJqDOsQsbVg90H21EOLACvtfcHk0ljo_A8dHmFM3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFKns7c%253D&md5=82a87a206e6544803ec6799ad42b1126</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs40265-018-0884-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-018-0884-8%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DJ.%2BN.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DCohen%26aufirst%3DM.%2BS.%26atitle%3DBRAF%2520and%2520MEK%2520inhibitors%253A%2520use%2520and%2520resistance%2520in%2520BRAF-mutated%2520cancers%26jtitle%3DDrugs%26date%3D2018%26volume%3D78%26spage%3D549%26epage%3D566%26doi%3D10.1007%2Fs40265-018-0884-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murugan, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, M.</span></span> <span> </span><span class="NLM_article-title">MEK1 mutations, but not ERK2 mutations, occur in melanomas and colon carcinomas, but none in thyroid carcinomas</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2122</span>– <span class="NLM_lpage">2124</span>, <span class="refDoi"> DOI: 10.4161/cc.8.13.8710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.4161%2Fcc.8.13.8710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=19411838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovFyqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=2122-2124&author=A.+K.+Muruganauthor=J.+Dongauthor=J.+Xieauthor=M.+Xing&title=MEK1+mutations%2C+but+not+ERK2+mutations%2C+occur+in+melanomas+and+colon+carcinomas%2C+but+none+in+thyroid+carcinomas&doi=10.4161%2Fcc.8.13.8710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1 mutations, but not ERK2 mutations, occur in melanomas and colon carcinomas, but none in thyroid carcinomas</span></div><div class="casAuthors">Murugan, Avaniyapuram Kannan; Dong, Jianli; Xie, Jingwu; Xing, Mingzhao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2122-2124</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Mitogen-activated protein kinase (MAPK) signaling pathway plays an important role in the pathogenesis of melanoma, colon cancer and thyroid cancer, which commonly harbor RAS and BRAF mutations.  However, mutations in exon 2 of MEK1 and exon 7 of ERK2 have not been investigated in these cancers although they are occasionally found in some other cancers or cell lines.  In this study, we performed mutational anal. to search for these mutations in 185 samples, including 167 tumor samples and 18 cell lines of melanoma, colon cancer and thyroid cancer.  We found one MEK1 mutation in 1 of 37 (3%) melanoma tumor samples and another MEK1 mutation in 1 of 45 (2.2%) colon cancer samples.  We did not find any MEK1 mutation in 99 thyroid tumor samples and 12 thyroid cancer cell lines.  We also did not find any ERK2 mutation in melanoma, colon cancer and thyroid cancer.  We thus report for the first time a low prevalence of MEK1 mutations in melanoma and colon cancer.  Both of the two mutants were demonstrated to be activating in the MAPK signaling pathway and may therefore provide potential target for effective therapy in cases of melanomas and colon cancer harboring these mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbwTT0xH_R_7Vg90H21EOLACvtfcHk0ljo_A8dHmFM3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovFyqtQ%253D%253D&md5=ad3811165466cf85870772386494bd68</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4161%2Fcc.8.13.8710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.8.13.8710%26sid%3Dliteratum%253Aachs%26aulast%3DMurugan%26aufirst%3DA.%2BK.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DXing%26aufirst%3DM.%26atitle%3DMEK1%2520mutations%252C%2520but%2520not%2520ERK2%2520mutations%252C%2520occur%2520in%2520melanomas%2520and%2520colon%2520carcinomas%252C%2520but%2520none%2520in%2520thyroid%2520carcinomas%26jtitle%3DCell%2520Cycle%26date%3D2009%26volume%3D8%26spage%3D2122%26epage%3D2124%26doi%3D10.4161%2Fcc.8.13.8710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolaev, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimoldi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iseli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valsesia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robyr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harshman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guipponi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukach, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muehlethaler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speiser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckmann, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenarios, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halazonetis, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jongeneel, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonarakis, S. E.</span></span> <span> </span><span class="NLM_article-title">Exome sequencing identifies recurrent somatic MAP2K1 and MAP2K2 mutations in melanoma</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">139</span>, <span class="refDoi"> DOI: 10.1038/ng.1026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fng.1026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1OltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=133-139&author=S.+I.+Nikolaevauthor=D.+Rimoldiauthor=C.+Iseliauthor=A.+Valsesiaauthor=D.+Robyrauthor=C.+Gehrigauthor=K.+Harshmanauthor=M.+Guipponiauthor=O.+Bukachauthor=V.+Zoeteauthor=O.+Michielinauthor=K.+Muehlethalerauthor=D.+Speiserauthor=J.+S.+Beckmannauthor=I.+Xenariosauthor=T.+D.+Halazonetisauthor=C.+V.+Jongeneelauthor=B.+J.+Stevensonauthor=S.+E.+Antonarakis&title=Exome+sequencing+identifies+recurrent+somatic+MAP2K1+and+MAP2K2+mutations+in+melanoma&doi=10.1038%2Fng.1026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exome sequencing identifies recurrent somatic MAP2K1 and MAP2K2 mutations in melanoma</span></div><div class="casAuthors">Nikolaev, Sergey I.; Rimoldi, Donata; Iseli, Christian; Valsesia, Armand; Robyr, Daniel; Gehrig, Corinne; Harshman, Keith; Guipponi, Michel; Bukach, Olesya; Zoete, Vincent; Michielin, Olivier; Muehlethaler, Katja; Speiser, Daniel; Beckmann, Jacques S.; Xenarios, Ioannis; Halazonetis, Thanos D.; Jongeneel, C. Victor; Stevenson, Brian J.; Antonarakis, Stylianos E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-139</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We performed exome sequencing to detect somatic mutations in protein-coding regions in seven melanoma cell lines and donor-matched germline cells.  All melanoma samples had high nos. of somatic mutations, which showed the hallmark of UV-induced DNA repair.  Such a hallmark was absent in tumor sample-specific mutations in two metastases derived from the same individual.  Two melanomas with non-canonical BRAF mutations harbored gain-of-function MAP2K1 and MAP2K2 (MEK1 and MEK2, resp.) mutations, resulting in constitutive ERK phosphorylation and higher resistance to MEK inhibitors.  Screening a larger cohort of individuals with melanoma revealed the presence of recurring somatic MAP2K1 and MAP2K2 mutations, which occurred at an overall frequency of 8%.  Furthermore, missense and nonsense somatic mutations were frequently found in three candidate melanoma genes, FAT4, LRP1B and DSC1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqoz4Pb6v6W7Vg90H21EOLACvtfcHk0lh5kO8Wb39i0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1OltbbJ&md5=9888253381503600ced0c828a097df48</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fng.1026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.1026%26sid%3Dliteratum%253Aachs%26aulast%3DNikolaev%26aufirst%3DS.%2BI.%26aulast%3DRimoldi%26aufirst%3DD.%26aulast%3DIseli%26aufirst%3DC.%26aulast%3DValsesia%26aufirst%3DA.%26aulast%3DRobyr%26aufirst%3DD.%26aulast%3DGehrig%26aufirst%3DC.%26aulast%3DHarshman%26aufirst%3DK.%26aulast%3DGuipponi%26aufirst%3DM.%26aulast%3DBukach%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DMuehlethaler%26aufirst%3DK.%26aulast%3DSpeiser%26aufirst%3DD.%26aulast%3DBeckmann%26aufirst%3DJ.%2BS.%26aulast%3DXenarios%26aufirst%3DI.%26aulast%3DHalazonetis%26aufirst%3DT.%2BD.%26aulast%3DJongeneel%26aufirst%3DC.%2BV.%26aulast%3DStevenson%26aufirst%3DB.%2BJ.%26aulast%3DAntonarakis%26aufirst%3DS.%2BE.%26atitle%3DExome%2520sequencing%2520identifies%2520recurrent%2520somatic%2520MAP2K1%2520and%2520MAP2K2%2520mutations%2520in%2520melanoma%26jtitle%3DNat.%2520Genet.%26date%3D2012%26volume%3D44%26spage%3D133%26epage%3D139%26doi%3D10.1038%2Fng.1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arcila, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drilon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borsu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kris, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladanyi, M.</span></span> <span> </span><span class="NLM_article-title">MAP2K1 (MEK1) mutations define a distinct subset of lung adenocarcinoma associated with smoking</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1935</span>– <span class="NLM_lpage">1943</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-2124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1158%2F1078-0432.CCR-14-2124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=25351745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsl2gtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1935-1943&author=M.+E.+Arcilaauthor=A.+Drilonauthor=B.+E.+Sylvesterauthor=C.+M.+Lovlyauthor=L.+Borsuauthor=B.+Revaauthor=M.+G.+Krisauthor=D.+B.+Solitauthor=M.+Ladanyi&title=MAP2K1+%28MEK1%29+mutations+define+a+distinct+subset+of+lung+adenocarcinoma+associated+with+smoking&doi=10.1158%2F1078-0432.CCR-14-2124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">MAP2K1 (MEK1) Mutations Define a Distinct Subset of Lung Adenocarcinoma Associated with Smoking</span></div><div class="casAuthors">Arcila, Maria E.; Drilon, Alexander; Sylvester, Brooke E.; Lovly, Christine M.; Borsu, Laetitia; Reva, Boris; Kris, Mark G.; Solit, David B.; Ladanyi, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1935-1943</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Genetic alterations affecting the MAPK/ERK pathway are common in lung adenocarcinoma (LAD).  Early steps of the signaling pathway are most often affected with EGFR, KRAS, and BRAF mutations encompassing more than 70% of all alterations.  Somatic mutations in MEK1, located downstream of BRAF, are rare and remain poorly defined as a distinct mol. subset.  Exptl. Design: Tumors harboring MEK1 mutations were identified through targeted screening of a large LAD cohort concurrently interrogated for recurrent mutations in MEK1, EGFR, KRAS, BRAF, ERBB2/HER2, NRAS, PIK3CA, and AKT.  Addnl. cases were identified through a search of publically available cancer genomic datasets.  Mutations were correlated with patient characteristics and treatment outcomes.  Overall survival was compared with stage-matched patients with KRAS- and EGFR-mutant LADs.  Results: We identified 36 MEK1-mutated cases among 6,024 LAD (0.6%; 95% confidence interval, 0.42-0.85).  The majority of patients were smokers (97%, n = 35/36).  There was no assocn. with age, sex, race, or stage.  The most common mutations were K57N (64%, 23/36) followed by Q56P (19%, 7/36), all mutually exclusive with other driver mutations in the targeted panel.  Transversions G:C>T:A were predominant (89%, 31/35), in keeping with smoking-assocd. DNA damage.  Addnl. less common somatic mutations were identified in the kinase domain, all of which are predicted to converge into a single interaction area based on in silico 3D modeling.  Conclusions: MEK1 mutations define a distinct subset of lung cancers (∼1%) with potential sensitivity to MEK inhibitors.  Mutations are predominantly transversions, in keeping with a strong assocn. with smoking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEJ9tmIMr9DbVg90H21EOLACvtfcHk0lh5kO8Wb39i0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsl2gtb8%253D&md5=92aa864d442da526e85b7ecc536130f6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-2124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-2124%26sid%3Dliteratum%253Aachs%26aulast%3DArcila%26aufirst%3DM.%2BE.%26aulast%3DDrilon%26aufirst%3DA.%26aulast%3DSylvester%26aufirst%3DB.%2BE.%26aulast%3DLovly%26aufirst%3DC.%2BM.%26aulast%3DBorsu%26aufirst%3DL.%26aulast%3DReva%26aufirst%3DB.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DLadanyi%26aufirst%3DM.%26atitle%3DMAP2K1%2520%2528MEK1%2529%2520mutations%2520define%2520a%2520distinct%2520subset%2520of%2520lung%2520adenocarcinoma%2520associated%2520with%2520smoking%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1935%26epage%3D1943%26doi%3D10.1158%2F1078-0432.CCR-14-2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grisham, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Won, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLair, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogomolniy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soumerai, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyman, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Socci, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershenson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spriggs, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aghajanian, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, G.</span></span> <span> </span><span class="NLM_article-title">Extreme outlier analysis identifies occult mitogen-activated protein kinase pathway mutations in patients with low-grade serous ovarian cancer</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">4099</span>– <span class="NLM_lpage">4105</span>, <span class="refDoi"> DOI: 10.1200/JCO.2015.62.4726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1200%2FJCO.2015.62.4726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26324360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVynsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=4099-4105&author=R.+N.+Grishamauthor=B.+E.+Sylvesterauthor=H.+Wonauthor=G.+McDermottauthor=D.+DeLairauthor=R.+Ramirezauthor=Z.+Yaoauthor=R.+Shenauthor=F.+Daoauthor=F.+Bogomolniyauthor=V.+Makkerauthor=E.+Salaauthor=T.+E.+Soumeraiauthor=D.+M.+Hymanauthor=N.+D.+Socciauthor=A.+Vialeauthor=D.+M.+Gershensonauthor=J.+Farleyauthor=D.+A.+Levineauthor=N.+Rosenauthor=M.+F.+Bergerauthor=D.+R.+Spriggsauthor=C.+A.+Aghajanianauthor=D.+B.+Solitauthor=G.+Iyer&title=Extreme+outlier+analysis+identifies+occult+mitogen-activated+protein+kinase+pathway+mutations+in+patients+with+low-grade+serous+ovarian+cancer&doi=10.1200%2FJCO.2015.62.4726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Extreme outlier analysis identifies occult mitogen-activated protein kinase pathway mutations in patients with low-grade serous ovarian cancer</span></div><div class="casAuthors">Grisham, Rachel N.; Sylvester, Brooke E.; Won, Helen; McDermott, Gregory; DeLair, Deborah; Ramirez, Ricardo; Yao, Zhan; Shen, Ronglai; Dao, Fanny; Bogomolniy, Faina; Makker, Vicky; Sala, Evis; Soumerai, Tara E.; Hyman, David M.; Socci, Nicholas D.; Viale, Agnes; Gershenson, David M.; Farley, John; Levine, Douglas A.; Rosen, Neal; Berger, Michael F.; Spriggs, David R.; Aghajanian, Carol A.; Solit, David B.; Iyer, Gopa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4099-4105</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose No effective systemic therapy exists for patients with metastatic low-grade serous (LGS) ovarian cancers.  BRAF and KRAS mutations are common in serous borderline (SB) and LGS ovarian cancers, and MEK inhibition has been shown to induce tumor regression in a minority of patients; however, no correlation has been obsd. between mutation status and clin. response.  With the goal of identifying biomarkers of sensitivity to MEK inhibitor treatment, we performed an outlier anal. of a patient who experienced a complete, durable, and ongoing (> 5 years) response to selumetinib, a non-ATP competitive MEK inhibitor.  Patients and Methods Next-generation sequencing was used to analyze this patient's tumor as well as an addnl. 28 SB/LGS tumors.  Functional characterization of an identified novel alteration of interest was performed.  Results Anal. of the extraordinary responder's tumor identified a 15-nucleotide deletion in the neg. regulatory helix of the MAP2K1 gene encoding for MEK1.  Functional characterization demonstrated that this mutant induced extracellular signal-regulated kinase pathway activation, promoted anchorage-independent growth and tumor formation in mice, and retained sensitivity to selumetinib.  Anal. of addnl. LGS/SB tumors identified mutations predicted to induce extracellular signal-regulated kinase pathway activation in 82% (23 of 28), including two patients with BRAF fusions, one of whom achieved an ongoing complete response to MEK inhibitor-based combination therapy.  Conclusion Alterations affecting the mitogen-activated protein kinase pathway are present in the majority of patients with LGS ovarian cancer.  Next-generation sequencing anal. revealed deletions and fusions that are not detected by older sequencing approaches.  These findings, coupled with the observation that a subset of patients with recurrent LGS ovarian cancer experienced dramatic and durable responses to MEK inhibitor therapy, support addnl. clin. studies of MEK inhibitors in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXv3nN7rbAh7Vg90H21EOLACvtfcHk0lh5kO8Wb39i0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVynsb4%253D&md5=3080b9e163347888b3db5f333ea50b05</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1200%2FJCO.2015.62.4726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2015.62.4726%26sid%3Dliteratum%253Aachs%26aulast%3DGrisham%26aufirst%3DR.%2BN.%26aulast%3DSylvester%26aufirst%3DB.%2BE.%26aulast%3DWon%26aufirst%3DH.%26aulast%3DMcDermott%26aufirst%3DG.%26aulast%3DDeLair%26aufirst%3DD.%26aulast%3DRamirez%26aufirst%3DR.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DDao%26aufirst%3DF.%26aulast%3DBogomolniy%26aufirst%3DF.%26aulast%3DMakker%26aufirst%3DV.%26aulast%3DSala%26aufirst%3DE.%26aulast%3DSoumerai%26aufirst%3DT.%2BE.%26aulast%3DHyman%26aufirst%3DD.%2BM.%26aulast%3DSocci%26aufirst%3DN.%2BD.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DGershenson%26aufirst%3DD.%2BM.%26aulast%3DFarley%26aufirst%3DJ.%26aulast%3DLevine%26aufirst%3DD.%2BA.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DBerger%26aufirst%3DM.%2BF.%26aulast%3DSpriggs%26aufirst%3DD.%2BR.%26aulast%3DAghajanian%26aufirst%3DC.%2BA.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DIyer%26aufirst%3DG.%26atitle%3DExtreme%2520outlier%2520analysis%2520identifies%2520occult%2520mitogen-activated%2520protein%2520kinase%2520pathway%2520mutations%2520in%2520patients%2520with%2520low-grade%2520serous%2520ovarian%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D33%26spage%3D4099%26epage%3D4105%26doi%3D10.1200%2FJCO.2015.62.4726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kakadia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarlagadda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundranda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naraev, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mina, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimbel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoud, F.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of resistance to BRAF and MEK inhibitors and clinical update of US Food and Drug Administration-approved targeted therapy in advanced melanoma</span>. <i>OncoTargets Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">7095</span>– <span class="NLM_lpage">7107</span>, <span class="refDoi"> DOI: 10.2147/OTT.S182721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.2147%2FOTT.S182721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30410366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFSgurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2018&pages=7095-7107&author=S.+Kakadiaauthor=N.+Yarlagaddaauthor=R.+Awadauthor=M.+Kundrandaauthor=J.+Niuauthor=B.+Naraevauthor=L.+Minaauthor=T.+Dragovichauthor=M.+Gimbelauthor=F.+Mahmoud&title=Mechanisms+of+resistance+to+BRAF+and+MEK+inhibitors+and+clinical+update+of+US+Food+and+Drug+Administration-approved+targeted+therapy+in+advanced+melanoma&doi=10.2147%2FOTT.S182721"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of resistance to BRAF and MEK inhibitors and clinical update of US Food and Drug Administration-approved targeted therapy in advanced melanoma</span></div><div class="casAuthors">Kakadia, Sunilkumar; Yarlagadda, Naveen; Awad, Ramez; Kundranda, Madappa; Niu, Jiaxin; Naraev, Boris; Mina, Lida; Dragovich, Tomislav; Gimbel, Mark; Mahmoud, Fade</div><div class="citationInfo"><span class="NLM_cas:title">OncoTargets and Therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7095-7107</span>CODEN:
                <span class="NLM_cas:coden">OTNHAZ</span>;
        ISSN:<span class="NLM_cas:issn">1178-6930</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Approx. 50% of melanomas harbor an activating BRAF mutation.  Combined BRAF and MEK inhibitors such as dabrafenib and trametinib, vemurafenib and cobimetinib, and encorafenib and binimetinib are US Food and Drug Administration (FDA)-approved to treat patients with BRAFV600-mutated advanced melanoma.  Both genetic and epigenetic alterations play a major role in resistance to BRAF inhibitors by reactivation of the MAPK and/or the PI3K-Akt pathways.  The role of BRAF inhibitors in modulating the immunomicroenvironment and perhaps enhancing the efficacy of checkpoint inhibitors is gaining interest.  This article provides a comprehensive review of mechanisms of resistance to BRAF and MEK inhibitors in melanoma and summarizes landmark trials that led to the FDA approval of BRAF and MEK inhibitors in metastatic melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYyD-wH6vxpbVg90H21EOLACvtfcHk0lg8vlrBQCm1MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFSgurnK&md5=57412f516c0e99e1846defd03492c71f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2147%2FOTT.S182721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FOTT.S182721%26sid%3Dliteratum%253Aachs%26aulast%3DKakadia%26aufirst%3DS.%26aulast%3DYarlagadda%26aufirst%3DN.%26aulast%3DAwad%26aufirst%3DR.%26aulast%3DKundranda%26aufirst%3DM.%26aulast%3DNiu%26aufirst%3DJ.%26aulast%3DNaraev%26aufirst%3DB.%26aulast%3DMina%26aufirst%3DL.%26aulast%3DDragovich%26aufirst%3DT.%26aulast%3DGimbel%26aufirst%3DM.%26aulast%3DMahmoud%26aufirst%3DF.%26atitle%3DMechanisms%2520of%2520resistance%2520to%2520BRAF%2520and%2520MEK%2520inhibitors%2520and%2520clinical%2520update%2520of%2520US%2520Food%2520and%2520Drug%2520Administration-approved%2520targeted%2520therapy%2520in%2520advanced%2520melanoma%26jtitle%3DOncoTargets%2520Ther.%26date%3D2018%26volume%3D11%26spage%3D7095%26epage%3D7107%26doi%3D10.2147%2FOTT.S182721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matallanas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birtwistle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zebisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kriegsheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolch, W.</span></span> <span> </span><span class="NLM_article-title">Raf family kinases: old dogs have learned new tricks</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1177/1947601911407323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1177%2F1947601911407323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=21779496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=232-260&author=D.+Matallanasauthor=M.+Birtwistleauthor=D.+Romanoauthor=A.+Zebischauthor=J.+Rauchauthor=A.+von+Kriegsheimauthor=W.+Kolch&title=Raf+family+kinases%3A+old+dogs+have+learned+new+tricks&doi=10.1177%2F1947601911407323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Raf family kinases: old dogs have learned new tricks</span></div><div class="casAuthors">Matallanas, David; Birtwistle, Marc; Romano, David; Zebisch, Armin; Rauch, Jens; von Kriegsheim, Alexander; Kolch, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">232-260</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  First identified in the early 1980s as retroviral oncogenes, the Raf proteins have been the objects of intense research.  The discoveries 10 yr later that the Raf family members (Raf-1, B-Raf, and A-Raf) are bona fide Ras effectors and upstream activators of the ubiquitous ERK kinase pathway increased the interest in these proteins primarily because of the central role that this cascade plays in cancer development.  The important role of Raf in cancer was corroborated in 2002 with the discovery of B-Raf genetic mutations in a large no. of tumors.  This led to intensified drug development efforts to target Raf signaling in cancer.  This work yielded not only recent clin. successes but also surprising insights into the regulation of Raf proteins by homodimerization and heterodimerization.  Surprising insights also came from the hunt for new Raf targets.  Although MEK remains the only widely accepted Raf substrate, new kinase-independent roles for Raf proteins have emerged.  These include the regulation of apoptosis by suppressing the activity of pro-apoptotic kinases ASK1 and MST2 and the regulation of cell motility and differentiation by controlling the activity of Rok-α.  Here, the authors discuss the regulation of Raf proteins and their role in cancer, with special focus on the interacting proteins that modulate Raf signaling.  The authors also describe the new pathways controlled by Raf proteins and summarize the successes and failures in the development of efficient anticancer therapies targeting Raf.  Finally, the authors also argue for the necessity of more systemic approaches to obtain a better understanding of how the Ras-Raf signaling network generates biol. specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0NzXqhz42aLVg90H21EOLACvtfcHk0lg8vlrBQCm1MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7k%253D&md5=4dba3fd7949b1c67fe5a0ecb1bec068a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1947601911407323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911407323%26sid%3Dliteratum%253Aachs%26aulast%3DMatallanas%26aufirst%3DD.%26aulast%3DBirtwistle%26aufirst%3DM.%26aulast%3DRomano%26aufirst%3DD.%26aulast%3DZebisch%26aufirst%3DA.%26aulast%3DRauch%26aufirst%3DJ.%26aulast%3Dvon%2BKriegsheim%26aufirst%3DA.%26aulast%3DKolch%26aufirst%3DW.%26atitle%3DRaf%2520family%2520kinases%253A%2520old%2520dogs%2520have%2520learned%2520new%2520tricks%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D232%26epage%3D260%26doi%3D10.1177%2F1947601911407323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span>FDA approves
encorafenib and binimetinib in combination for unresectable or metastatic
melanoma with BRAF mutations (accessed June 27, 2018). fda.gov, <span class="NLM_year">2018</span>; <a href="https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf" class="extLink">https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FDA+approves%0Aencorafenib+and+binimetinib+in+combination+for+unresectable+or+metastatic%0Amelanoma+with+BRAF+mutations+%28accessed+June+27%2C+2018%29.+fda.gov%2C+2018%3B+https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fresources-information-approved-drugs%2Ffda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karaszewska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackiewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroiakovski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichinitser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dummer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grange, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarion-Sileni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drucis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajsova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauschild, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorigan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crist, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadendorf, D.</span></span> <span> </span><span class="NLM_article-title">Improved overall survival in melanoma with combined dabrafenib and trametinib</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>372</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1412690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1056%2FNEJMoa1412690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=25399551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFSqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2015&pages=30-39&author=C.+Robertauthor=B.+Karaszewskaauthor=J.+Schachterauthor=P.+Rutkowskiauthor=A.+Mackiewiczauthor=D.+Stroiakovskiauthor=M.+Lichinitserauthor=R.+Dummerauthor=F.+Grangeauthor=L.+Mortierauthor=V.+Chiarion-Sileniauthor=K.+Drucisauthor=I.+Krajsovaauthor=A.+Hauschildauthor=P.+Loriganauthor=P.+Wolterauthor=G.+V.+Longauthor=K.+Flahertyauthor=P.+Nathanauthor=A.+Ribasauthor=A.+M.+Martinauthor=P.+Sunauthor=W.+Cristauthor=J.+Legosauthor=S.+D.+Rubinauthor=S.+M.+Littleauthor=D.+Schadendorf&title=Improved+overall+survival+in+melanoma+with+combined+dabrafenib+and+trametinib&doi=10.1056%2FNEJMoa1412690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Improved overall survival in melanoma with combined dabrafenib and trametinib</span></div><div class="casAuthors">Robert, Caroline; Karaszewska, Boguslawa; Schachter, Jacob; Rutkowski, Piotr; Mackiewicz, Andrzej; Stroiakovski, Daniil; Lichinitser, Michael; Dummer, Reinhard; Grange, Florent; Mortier, Laurent; Chiarion-Sileni, Vanna; Drucis, Kamil; Krajsova, Ivana; Hauschild, Axel; Lorigan, Paul; Wolter, Pascal; Long, Georgina V.; Flaherty, Keith; Nathan, Paul; Ribas, Antoni; Martin, Anne-Marie; Sun, Peng; Crist, Wendy; Legos, Jeff; Rubin, Stephen D.; Little, Shonda M.; Schadendorf, Dirk</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-39/1-30-39/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: The BRAF inhibitors vemurafenib and dabrafenib have shown efficacy as monotherapies in patients with previously untreated metastatic melanoma with BRAF V600E or V600K mutations.  Combining dabrafenib and the MEK inhibitor trametinib, as compared with dabrafenib alone, enhanced antitumor activity in this population of patients.  Methods: In this open-label, phase 3 trial, we randomly assigned 704 patients with metastatic melanoma with a BRAF V600 mutation to receive either a combination of dabrafenib (150 mg twice daily) and trametinib (2 mg once daily) or vemurafenib (960 mg twice daily) orally as first-line therapy.  The primary end point was overall survival.  Results: At the preplanned interim overall survival anal., which was performed after 77% of the total no. of expected events occurred, the overall survival rate at 12 mo was 72% (95% confidence interval [CI], 67 to 77) in the combination-therapy group and 65% (95% CI, 59 to 70) in the vemurafenib group (hazard ratio for death in the combination-therapy group, 0.69; 95% CI, 0.53 to 0.89; P = 0.005).  The prespecified interim stopping boundary was crossed, and the study was stopped for efficacy in July 2014.  Median progression-free survival was 11.4 mo in the combination therapy group and 7.3 mo in the vemurafenib group (hazard ratio, 0.56; 95% CI, 0.46 to 0.69; P<0.001).  The objective response rate was 64% in the combination therapy group and 51% in the vemurafenib group (P<0.001).  Rates of severe adverse events and study-drug discontinuations were similar in the two groups.  Cutaneous squamous-cell carcinoma and keratoacanthoma occurred in 1% of patients in the combination-therapy group and 18% of those in the vemurafenib group.  Conclusions: Dabrafenib plus trametinib, as compared with vemurafenib monotherapy, significantly improved overall survival in previously untreated patients with metastatic melanoma with BRAF V600E or V600K mutations, without increased overall toxicity. (Funded by GlaxoSmithKline; ClinicalTrials.gov no., NCT01597908.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5of4j14YrP7Vg90H21EOLACvtfcHk0lg8vlrBQCm1MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFSqsb0%253D&md5=9d6a8f49bbd196f9463ec15958eb15d1</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1412690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1412690%26sid%3Dliteratum%253Aachs%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DKaraszewska%26aufirst%3DB.%26aulast%3DSchachter%26aufirst%3DJ.%26aulast%3DRutkowski%26aufirst%3DP.%26aulast%3DMackiewicz%26aufirst%3DA.%26aulast%3DStroiakovski%26aufirst%3DD.%26aulast%3DLichinitser%26aufirst%3DM.%26aulast%3DDummer%26aufirst%3DR.%26aulast%3DGrange%26aufirst%3DF.%26aulast%3DMortier%26aufirst%3DL.%26aulast%3DChiarion-Sileni%26aufirst%3DV.%26aulast%3DDrucis%26aufirst%3DK.%26aulast%3DKrajsova%26aufirst%3DI.%26aulast%3DHauschild%26aufirst%3DA.%26aulast%3DLorigan%26aufirst%3DP.%26aulast%3DWolter%26aufirst%3DP.%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DFlaherty%26aufirst%3DK.%26aulast%3DNathan%26aufirst%3DP.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DA.%2BM.%26aulast%3DSun%26aufirst%3DP.%26aulast%3DCrist%26aufirst%3DW.%26aulast%3DLegos%26aufirst%3DJ.%26aulast%3DRubin%26aufirst%3DS.%2BD.%26aulast%3DLittle%26aufirst%3DS.%2BM.%26aulast%3DSchadendorf%26aufirst%3DD.%26atitle%3DImproved%2520overall%2520survival%2520in%2520melanoma%2520with%2520combined%2520dabrafenib%2520and%2520trametinib%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2015%26volume%3D372%26spage%3D30%26epage%3D39%26doi%3D10.1056%2FNEJMoa1412690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroyakovskiy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levchenko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Braud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jouary, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauschild, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grob, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarion-Sileni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebbe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millward, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arance, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondarenko, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haanen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Utikal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraresi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovalenko, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Probachai, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadendorf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMarini, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irani, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legos, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mookerjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, K.</span></span> <span> </span><span class="NLM_article-title">Dabrafenib and trametinib versus dabrafenib and placebo for Val600 BRAF-mutant melanoma: a multicentre, double-blind, phase 3 randomised controlled trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>386</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(15)60898-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2FS0140-6736%2815%2960898-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26037941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpsVyrt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2015&pages=444-451&author=G.+V.+Longauthor=D.+Stroyakovskiyauthor=H.+Gogasauthor=E.+Levchenkoauthor=F.+de+Braudauthor=J.+Larkinauthor=C.+Garbeauthor=T.+Jouaryauthor=A.+Hauschildauthor=J.+J.+Grobauthor=V.+Chiarion-Sileniauthor=C.+Lebbeauthor=M.+Mandalaauthor=M.+Millwardauthor=A.+Aranceauthor=I.+Bondarenkoauthor=J.+B.+Haanenauthor=J.+Hanssonauthor=J.+Utikalauthor=V.+Ferraresiauthor=N.+Kovalenkoauthor=P.+Mohrauthor=V.+Probachaiauthor=D.+Schadendorfauthor=P.+Nathanauthor=C.+Robertauthor=A.+Ribasauthor=D.+J.+DeMariniauthor=J.+G.+Iraniauthor=S.+Swannauthor=J.+J.+Legosauthor=F.+Jinauthor=B.+Mookerjeeauthor=K.+Flaherty&title=Dabrafenib+and+trametinib+versus+dabrafenib+and+placebo+for+Val600+BRAF-mutant+melanoma%3A+a+multicentre%2C+double-blind%2C+phase+3+randomised+controlled+trial&doi=10.1016%2FS0140-6736%2815%2960898-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Dabrafenib and trametinib versus dabrafenib and placebo for Val600 BRAF-mutant melanoma: a multicentre, double-blind, phase 3 randomised controlled trial</span></div><div class="casAuthors">Long, Georgina V.; Stroyakovskiy, Daniil; Gogas, Helen; Levchenko, Evgeny; de Braud, Filippo; Larkin, James; Garbe, Claus; Jouary, Thomas; Hauschild, Axel; Grob, Jean-Jacques; Chiarion-Sileni, Vanna; Lebbe, Celeste; Mandala, Mario; Millward, Michael; Arance, Ana; Bondarenko, Igor; Haanen, John B. A. G.; Hansson, Johan; Utikal, Jochen; Ferraresi, Virginia; Kovalenko, Nadezhda; Mohr, Peter; Probachai, Volodymr; Schadendorf, Dirk; Nathan, Paul; Robert, Caroline; Ribas, Antoni; De Marini, Douglas J.; Irani, Jhangir G.; Swann, Suzanne; Legos, Jeffrey J.; Jin, Fan; Mookerjee, Bijoyesh; Flaherty, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">9992</span>),
    <span class="NLM_cas:pages">444-451</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Previously, a study of ours showed that the combination of dabrafenib and trametinib improves progression-free survival compared with dabrafenib and placebo in patients with BRAF Val600Lys/Glu mutation-pos. metastatic melanoma.  The study was continued to assess the secondary endpoint of overall survival, which we report in this Article.  We did this double-blind phase 3 study at 113 sites in 14 countries.  We enrolled previously untreated patients with BRAF Val600Glu or Val600Lys mutation-pos. unresectable stage IIIC or stage IV melanoma.  Participants were computer-randomised (1:1) to receive a combination of dabrafenib (150 mg orally twice daily) and trametinib (2 mg orally once daily), or dabrafenib and placebo.  The primary endpoint was progression-free survival and overall survival was a secondary endpoint.  This study is registered with ClinicalTrials.gov, no. NCT01584648.  Between May 4, 2012, and Nov 30, 2012, we screened 947 patients for eligibility, of whom 423 were randomly assigned to receive dabrafenib and trametinib (n=211) or dabrafenib only (n=212).  The final data cutoff was Jan 12, 2015, at which time 222 patients had died.  Median overall survival was 25·1 mo (95% CI 19·2-not reached) in the dabrafenib and trametinib group vs. 18·7 mo (15·2-23·7) in the dabrafenib only group (hazard ratio [HR] 0·71, 95% CI 0·55-0·92; p=0·0107).  Overall survival was 74% at 1 yr and 51% at 2 years in the dabrafenib and trametinib group vs. 68% and 42%, resp., in the dabrafenib only group.  Based on 301 events, median progression-free survival was 11·0 mo (95% CI 8·0-13·9) in the dabrafenib and trametinib group and 8·8 mo (5·9-9·3) in the dabrafenib only group (HR 0·67, 95% CI 0·53-0·84; p=0·0004; unadjusted for multiple testing).  Treatment-related adverse events occurred in 181 (87%) of 209 patients in the dabrafenib and trametinib group and 189 (90%) of 211 patients in the dabrafenib only group; the most common was pyrexia (108 patients, 52%) in the dabrafenib and trametinib group, and hyperkeratosis (70 patients, 33%) in the dabrafenib only group.  Grade 3 or 4 adverse events occurred in 67 (32%) patients in the dabrafenib and trametinib group and 66 (31%) patients in the dabrafenib only group.  The improvement in overall survival establishes the combination of dabrafenib and trametinib as the std. targeted treatment for BRAF Val600 mutation-pos. melanoma.  Studies assessing dabrafenib and trametinib in combination with immunotherapies are ongoing.GlaxoSmithKline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeIg9L7TouB7Vg90H21EOLACvtfcHk0lgqQdkGC6drrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpsVyrt7k%253D&md5=3de7b0a659267f73299e8111ff2a09bb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2815%2960898-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252815%252960898-4%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DStroyakovskiy%26aufirst%3DD.%26aulast%3DGogas%26aufirst%3DH.%26aulast%3DLevchenko%26aufirst%3DE.%26aulast%3Dde%2BBraud%26aufirst%3DF.%26aulast%3DLarkin%26aufirst%3DJ.%26aulast%3DGarbe%26aufirst%3DC.%26aulast%3DJouary%26aufirst%3DT.%26aulast%3DHauschild%26aufirst%3DA.%26aulast%3DGrob%26aufirst%3DJ.%2BJ.%26aulast%3DChiarion-Sileni%26aufirst%3DV.%26aulast%3DLebbe%26aufirst%3DC.%26aulast%3DMandala%26aufirst%3DM.%26aulast%3DMillward%26aufirst%3DM.%26aulast%3DArance%26aufirst%3DA.%26aulast%3DBondarenko%26aufirst%3DI.%26aulast%3DHaanen%26aufirst%3DJ.%2BB.%26aulast%3DHansson%26aufirst%3DJ.%26aulast%3DUtikal%26aufirst%3DJ.%26aulast%3DFerraresi%26aufirst%3DV.%26aulast%3DKovalenko%26aufirst%3DN.%26aulast%3DMohr%26aufirst%3DP.%26aulast%3DProbachai%26aufirst%3DV.%26aulast%3DSchadendorf%26aufirst%3DD.%26aulast%3DNathan%26aufirst%3DP.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DDeMarini%26aufirst%3DD.%2BJ.%26aulast%3DIrani%26aufirst%3DJ.%2BG.%26aulast%3DSwann%26aufirst%3DS.%26aulast%3DLegos%26aufirst%3DJ.%2BJ.%26aulast%3DJin%26aufirst%3DF.%26aulast%3DMookerjee%26aufirst%3DB.%26aulast%3DFlaherty%26aufirst%3DK.%26atitle%3DDabrafenib%2520and%2520trametinib%2520versus%2520dabrafenib%2520and%2520placebo%2520for%2520Val600%2520BRAF-mutant%2520melanoma%253A%2520a%2520multicentre%252C%2520double-blind%252C%2520phase%25203%2520randomised%2520controlled%2520trial%26jtitle%3DLancet%26date%3D2015%26volume%3D386%26spage%3D444%26epage%3D451%26doi%3D10.1016%2FS0140-6736%2815%2960898-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grob, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadendorf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legenne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M. A.</span></span> <span> </span><span class="NLM_article-title">Factors predictive of response, disease progression, and overall survival after dabrafenib and trametinib combination treatment: a pooled analysis of individual patient data from randomised trials</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1743</span>– <span class="NLM_lpage">1754</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(16)30578-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2FS1470-2045%2816%2930578-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=27864013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVymt77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=1743-1754&author=G.+V.+Longauthor=J.+J.+Grobauthor=P.+Nathanauthor=A.+Ribasauthor=C.+Robertauthor=D.+Schadendorfauthor=S.+R.+Laneauthor=C.+Makauthor=P.+Legenneauthor=K.+T.+Flahertyauthor=M.+A.+Davies&title=Factors+predictive+of+response%2C+disease+progression%2C+and+overall+survival+after+dabrafenib+and+trametinib+combination+treatment%3A+a+pooled+analysis+of+individual+patient+data+from+randomised+trials&doi=10.1016%2FS1470-2045%2816%2930578-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Factors predictive of response, disease progression, and overall survival after dabrafenib and trametinib combination treatment: a pooled analysis of individual patient data from randomised trials</span></div><div class="casAuthors">Long, Georgina V.; Grob, Jean-Jacques; Nathan, Paul; Ribas, Antoni; Robert, Caroline; Schadendorf, Dirk; Lane, Stephen R.; Mak, Carmen; Legenne, Philippe; Flaherty, Keith T.; Davies, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1743-1754</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dabrafenib plus trametinib treatment provides significant benefits over BRAF-inhibitor monotherapy in patients with BRAFV600E-mutant or BRAFV600K-mutant advanced melanoma; however, in many patients the disease progresses, leading to death.  With many treatment options available, understanding clin. factors that predict long-term response and survival for treatments is important for optimization of patient management.  We aimed to identify clin. factors assocd. with long-term response and survival using pooled data from randomized trials of dabrafenib plus trametinib in patients with metastatic BRAF-mutant melanoma.  We did a retrospective individual data anal. based on all published randomized trials that included treatment-naive patients with BRAFV600E-mutant or BRAFV600K-mutant metastatic melanoma who received the approved dose of dabrafenib 150 mg twice daily plus trametinib 2 mg once daily.  Data were pooled from patients in the BRF113220 (part C; March 26, 2010, to Jan 15, 2015), COMBI-d (May 4, 2012, to Jan 12, 2015), and COMBI-v (June 4, 2012, to March 13, 2015) randomized trials.  Patients with untreated brain metastases were not permitted to enroll in these trials.  Baseline factors, identified a priori based on known melanoma clin. or prognostic characteristics, were analyzed for assocn. with progression-free survival and overall survival using univariate and multivariate analyses and assessed for hierarchical effect on outcomes using regression tree analyses.  We also analyzed factors identified after baseline, on treatment, and at progression, for assocns. with survival after progression.  The trials included in this anal. are registered with ClinicalTrials.gov: BRF113220, no. NCT01072175; COMBI-d, no. NCT01584648; COMBI-v, no. NCT01597908.617 patients were included in this anal. with a median follow-up of 20·0 mo (range 0-48·0, IQR 10·1-24·8); 396 patients had progression events (ie, disease progression or death) and 290 patients had died.  Median progression-free survival (11·1 mo [95% CI 9·7-12·9]), median overall survival (25·6 mo [23·1-34·3]), 1-yr progression-free survival (48% [44-52]) and overall survival (74% [71-78]), and 2-yr progression-free survival (30% [26-34]) and overall survival (53% [49-57]) were consistent with those in the individual trials.  Patients with normal lactate dehydrogenase (LDH) concn. and fewer than three organ sites contg. metastases (n=237) had the longest 1-yr progression-free survival (68% [95% CI 62-74]) and overall survival (90% [87-94]) and 2-yr progression-free survival (46% [40-54]) and overall survival (75% [70-81]), whereas patients with LDH concn. at least two times the upper limit of normal (n=70) had the shortest 1-yr progression-free survival (8% [3-19]) and overall survival (40% [29-55]) and 2-yr progression-free survival (2% [0-13]) and overall survival (7% [3-19]).  Of patients with disease progression (n=379), survival after progression was longest in those with progression in baseline or new non-CNS lesions (n=205; median 10·0 mo [95% CI 7·9-12·0]) and shortest in those with new CNS lesions or concurrent progression in baseline and new lesions (n=171; median 4·0 mo [3·5-4·9]).  Several patient and clin. characteristics at and after baseline are assocd. with outcomes with dabrafenib plus trametinib, and durable benefit is possible with targeted treatment in defined patient subsets.Novartis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_v9J1131FDLVg90H21EOLACvtfcHk0lgqQdkGC6drrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVymt77I&md5=8f256406a53b9ea7b258ad173b56a7ca</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2816%2930578-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252816%252930578-2%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DGrob%26aufirst%3DJ.%2BJ.%26aulast%3DNathan%26aufirst%3DP.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DSchadendorf%26aufirst%3DD.%26aulast%3DLane%26aufirst%3DS.%2BR.%26aulast%3DMak%26aufirst%3DC.%26aulast%3DLegenne%26aufirst%3DP.%26aulast%3DFlaherty%26aufirst%3DK.%2BT.%26aulast%3DDavies%26aufirst%3DM.%2BA.%26atitle%3DFactors%2520predictive%2520of%2520response%252C%2520disease%2520progression%252C%2520and%2520overall%2520survival%2520after%2520dabrafenib%2520and%2520trametinib%2520combination%2520treatment%253A%2520a%2520pooled%2520analysis%2520of%2520individual%2520patient%2520data%2520from%2520randomised%2520trials%26jtitle%3DLancet%2520Oncol.%26date%3D2016%26volume%3D17%26spage%3D1743%26epage%3D1754%26doi%3D10.1016%2FS1470-2045%2816%2930578-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span>FDA grants
regular approval to dabrafenib and trametinib combination for metastatic
NSCLC with BRAF V600E mutation (accessed June 22, 2017). fda.gov, <span class="NLM_year">2017</span>; <a href="https://www.fda.gov/drugs/resources-information-approved-drugs/fda-grants-regular-approval-dabrafenib-and-trametinib-combination-metastatic-nsclc-braf-v600e" class="extLink">https://www.fda.gov/drugs/resources-information-approved-drugs/fda-grants-regular-approval-dabrafenib-and-trametinib-combination-metastatic-nsclc-braf-v600e</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FDA+grants%0Aregular+approval+to+dabrafenib+and+trametinib+combination+for+metastatic%0ANSCLC+with+BRAF+V600E+mutation+%28accessed+June+22%2C+2017%29.+fda.gov%2C+2017%3B+https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fresources-information-approved-drugs%2Ffda-grants-regular-approval-dabrafenib-and-trametinib-combination-metastatic-nsclc-braf-v600e."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corcoran, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atreya, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellens, J. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendell, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollebecque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRee, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muro, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Dwyer, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brase, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mookerjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangwala, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Cutsem, E.</span></span> <span> </span><span class="NLM_article-title">Combined BRAF, EGFR, and MEK inhibition in patients with <i>BRAF</i><sup><i>V600E</i></sup>-mutant colorectal cancer</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">443</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-17-1226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1158%2F2159-8290.CD-17-1226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29431699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1emtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=428-443&author=R.+B.+Corcoranauthor=T.+Andreauthor=C.+E.+Atreyaauthor=J.+H.+M.+Schellensauthor=T.+Yoshinoauthor=J.+C.+Bendellauthor=A.+Hollebecqueauthor=A.+J.+McReeauthor=S.+Sienaauthor=G.+Middletonauthor=K.+Muroauthor=M.+S.+Gordonauthor=J.+Taberneroauthor=R.+Yaegerauthor=P.+J.+O%E2%80%99Dwyerauthor=Y.+Humbletauthor=F.+De+Vosauthor=A.+S.+Jungauthor=J.+C.+Braseauthor=S.+Jaegerauthor=S.+Bettingerauthor=B.+Mookerjeeauthor=F.+Rangwalaauthor=E.+Van+Cutsem&title=Combined+BRAF%2C+EGFR%2C+and+MEK+inhibition+in+patients+with+BRAFV600E-mutant+colorectal+cancer&doi=10.1158%2F2159-8290.CD-17-1226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Combined BRAF, EGFR, and MEK Inhibition in Patients with BRAFV600E-Mutant Colorectal Cancer</span></div><div class="casAuthors">Corcoran, Ryan B.; Andre, Thierry; Atreya, Chloe E.; Schellens, Jan H. M.; Yoshino, Takayuki; Bendell, Johanna C.; Hollebecque, Antoine; McRee, Autumn J.; Siena, Salvatore; Middleton, Gary; Muro, Kei; Gordon, Michael S.; Tabernero, Josep; Yaeger, Rona; O'Dwyer, Peter J.; Humblet, Yves; De Vos, Filip; Jung, A. Scott; Brase, Jan C.; Jaeger, Savina; Bettinger, Severine; Mookerjee, Bijoyesh; Rangwala, Fatima; Van Cutsem, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">428-443</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although BRAF inhibitor monotherapy yields response rates >50% in BRAFV600-mutant melanoma, only approx. 5% of patients with BRAFV600E colorectal cancer respond.  Preclin. studies suggest that the lack of efficacy in BRAFV600E colorectal cancer is due to adaptive feedback reactivation of MAPK signaling, often mediated by EGFR.  This clin. trial evaluated BRAF and EGFR inhibition with dabrafenib (D) + panitumumab (P) ± MEK inhibition with trametinib (T) to achieve greater MAPK suppression and improved efficacy in 142 patients with BRAFV600E colorectal cancer.  Confirmed response rates for D+P, D+T+P, and T+P were 10%, 21%, and 0%, resp.  Pharmacodynamic anal. of paired pretreatment and on-treatment biopsies found that efficacy of D+T+P correlated with increased MAPK suppression.  Serial cell-free DNA anal. revealed addnl. correlates of response and emergence of KRAS and NRAS mutations on disease progression.  Thus, targeting adaptive feedback pathways in BRAFV600E colorectal cancer can improve efficacy, but MAPK reactivation remains an important primary and acquired resistance mechanism.  Significance: This trial demonstrates that combined BRAF + EGFR + MEK inhibition is tolerable, with promising activity in patients with BRAFV600E colorectal cancer.  Our findings highlight the MAPK pathway as a crit. target in BRAFV600E colorectal cancer and the need to optimize strategies inhibiting this pathway to overcome both primary and acquired resistance.  Cancer Discov; 8(4); 428-43. ©2018 AACR.  See related commentary by Janku, p. 389.  See related article by Hazar-Rethinam et al., p. 417.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJr18ksu0glrVg90H21EOLACvtfcHk0lgtIukGyDR8SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1emtLY%253D&md5=ea2b2a5036b06d4858d8deca9513e97f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-1226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-1226%26sid%3Dliteratum%253Aachs%26aulast%3DCorcoran%26aufirst%3DR.%2BB.%26aulast%3DAndre%26aufirst%3DT.%26aulast%3DAtreya%26aufirst%3DC.%2BE.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DYoshino%26aufirst%3DT.%26aulast%3DBendell%26aufirst%3DJ.%2BC.%26aulast%3DHollebecque%26aufirst%3DA.%26aulast%3DMcRee%26aufirst%3DA.%2BJ.%26aulast%3DSiena%26aufirst%3DS.%26aulast%3DMiddleton%26aufirst%3DG.%26aulast%3DMuro%26aufirst%3DK.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DTabernero%26aufirst%3DJ.%26aulast%3DYaeger%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Dwyer%26aufirst%3DP.%2BJ.%26aulast%3DHumblet%26aufirst%3DY.%26aulast%3DDe%2BVos%26aufirst%3DF.%26aulast%3DJung%26aufirst%3DA.%2BS.%26aulast%3DBrase%26aufirst%3DJ.%2BC.%26aulast%3DJaeger%26aufirst%3DS.%26aulast%3DBettinger%26aufirst%3DS.%26aulast%3DMookerjee%26aufirst%3DB.%26aulast%3DRangwala%26aufirst%3DF.%26aulast%3DVan%2BCutsem%26aufirst%3DE.%26atitle%3DCombined%2520BRAF%252C%2520EGFR%252C%2520and%2520MEK%2520inhibition%2520in%2520patients%2520with%2520BRAFV600E-mutant%2520colorectal%2520cancer%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D428%26epage%3D443%26doi%3D10.1158%2F2159-8290.CD-17-1226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eoh, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanmamed, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaftan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitrakopoulou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbst, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, J. S.</span></span> <span> </span><span class="NLM_article-title">The combination of MEK inhibitor with immunomodulatory antibodies targeting programmed death 1 and programmed death ligand 1 results in prolonged survival in <i>Kras/p53</i>-driven lung cancer</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1060</span>, <span class="refDoi"> DOI: 10.1016/j.jtho.2019.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.jtho.2019.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30771521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BB3cfkt1SgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1046-1060&author=J.+W.+Leeauthor=Y.+Zhangauthor=K.+J.+Eohauthor=R.+Sharmaauthor=M.+F.+Sanmamedauthor=J.+Wuauthor=J.+Choiauthor=H.+S.+Parkauthor=A.+Iwasakiauthor=E.+Kaftanauthor=L.+Chenauthor=V.+Papadimitrakopoulouauthor=R.+S.+Herbstauthor=J.+S.+Koo&title=The+combination+of+MEK+inhibitor+with+immunomodulatory+antibodies+targeting+programmed+death+1+and+programmed+death+ligand+1+results+in+prolonged+survival+in+Kras%2Fp53-driven+lung+cancer&doi=10.1016%2Fj.jtho.2019.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Combination of MEK Inhibitor With Immunomodulatory Antibodies Targeting Programmed Death 1 and Programmed Death Ligand 1 Results in Prolonged Survival in Kras/p53-Driven Lung Cancer</span></div><div class="casAuthors">Lee Jong Woo; Sharma Roshan; Wu Jenny; Choi Justin; Kaftan Edward; Herbst Roy S; Zhang Yu; Sanmamed Miguel F; Chen Lieping; Eoh Kyung Jin; Park Hee Sun; Iwasaki Akiko; Papadimitrakopoulou Vali; Koo Ja Seok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1046-1060</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  This study aimed to characterize the tumor-infiltrating immune cells population in Kras/tumor protein 53 (Trp53)-driven lung tumors and to evaluate the combinatorial antitumor effect with MEK inhibitor (MEKi), trametinib, and immunomodulatory monoclonal antibodies (mAbs) targeting either programmed death -1 (PD-1) or programmed cell death ligand 1 (PD-L1) in vivo.  METHODS:  Trp53(FloxFlox);Kras(G12D/+);Rosa26(LSL-Luciferase/LSL-Luciferase) (PKL) genetically engineered mice were used to develop autochthonous lung tumors with intratracheal delivery of adenoviral Cre recombinase.  Using these tumor-bearing lungs, tumor-infiltrating immune cells were characterized by both mass cytometry and flow cytometry.  PKL-mediated immunocompetent syngeneic and transgenic lung cancer mouse models were treated with MEKi alone as well as in combination with either anti-PD-1 or anti-PD-L1 mAbs.  Tumor growth and survival outcome were assessed.  Finally, immune cell populations within spleens and tumors were evaluated by flow cytometry and immunohistochemistry.  RESULTS:  Myeloid-derived suppressor cells (MDSCs) were significantly augmented in PKL-driven lung tumors compared to normal lungs of tumor-free mice.  PD-L1 expression appeared to be highly positive in both lung tumor cells and, particularly MDSCs.  The combinatory administration of MEKi with either anti-PD-1 or anti-PD-L1 mAbs synergistically increased antitumor response and survival outcome compared with single-agent therapy in both the PKL-mediated syngeneic and transgenic lung cancer models.  Theses combinational treatments resulted in significant increases of tumor-infiltrating CD8(+) and CD4(+) T cells, whereas attenuation of CD11b(+)/Gr-1(high) MDSCs, in particular, Ly6G(high) polymorphonuclear-MDSCs in the syngeneic model.  CONCLUSIONS:  These findings suggest a potential therapeutic approach for untargetable Kras/p53-driven lung cancers with synergy between targeted therapy using MEKi and immunotherapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQYgLfJG8xGHBWfpMXQLRstfW6udTcc2eYl9EzRm9cSL7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfkt1SgtA%253D%253D&md5=59df9199c1c21e5fe8627a4b270f7c90</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.jtho.2019.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtho.2019.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DEoh%26aufirst%3DK.%2BJ.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DSanmamed%26aufirst%3DM.%2BF.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DH.%2BS.%26aulast%3DIwasaki%26aufirst%3DA.%26aulast%3DKaftan%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DPapadimitrakopoulou%26aufirst%3DV.%26aulast%3DHerbst%26aufirst%3DR.%2BS.%26aulast%3DKoo%26aufirst%3DJ.%2BS.%26atitle%3DThe%2520combination%2520of%2520MEK%2520inhibitor%2520with%2520immunomodulatory%2520antibodies%2520targeting%2520programmed%2520death%25201%2520and%2520programmed%2520death%2520ligand%25201%2520results%2520in%2520prolonged%2520survival%2520in%2520Kras%252Fp53-driven%2520lung%2520cancer%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2019%26volume%3D14%26spage%3D1046%26epage%3D1060%26doi%3D10.1016%2Fj.jtho.2019.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pupo, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahheydari, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tembe, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saw, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kefford, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizos, H.</span></span> <span> </span><span class="NLM_article-title">Increased MAPK reactivation in early resistance to dabrafenib/trametinib combination therapy of BRAF-mutant metastatic melanoma</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1038/ncomms6694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fncomms6694" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1-9&author=G.+V.+Longauthor=C.+Fungauthor=A.+M.+Menziesauthor=G.+M.+Pupoauthor=M.+S.+Carlinoauthor=J.+Hymanauthor=H.+Shahheydariauthor=V.+Tembeauthor=J.+F.+Thompsonauthor=R.+P.+Sawauthor=J.+Howleauthor=N.+K.+Haywardauthor=P.+Johanssonauthor=R.+A.+Scolyerauthor=R.+F.+Keffordauthor=H.+Rizos&title=Increased+MAPK+reactivation+in+early+resistance+to+dabrafenib%2Ftrametinib+combination+therapy+of+BRAF-mutant+metastatic+melanoma&doi=10.1038%2Fncomms6694"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fncomms6694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6694%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DFung%26aufirst%3DC.%26aulast%3DMenzies%26aufirst%3DA.%2BM.%26aulast%3DPupo%26aufirst%3DG.%2BM.%26aulast%3DCarlino%26aufirst%3DM.%2BS.%26aulast%3DHyman%26aufirst%3DJ.%26aulast%3DShahheydari%26aufirst%3DH.%26aulast%3DTembe%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DJ.%2BF.%26aulast%3DSaw%26aufirst%3DR.%2BP.%26aulast%3DHowle%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DN.%2BK.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DKefford%26aufirst%3DR.%2BF.%26aulast%3DRizos%26aufirst%3DH.%26atitle%3DIncreased%2520MAPK%2520reactivation%2520in%2520early%2520resistance%2520to%2520dabrafenib%252Ftrametinib%2520combination%2520therapy%2520of%2520BRAF-mutant%2520metastatic%2520melanoma%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D1%26epage%3D9%26doi%3D10.1038%2Fncomms6694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriceau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hugo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koya, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samatar, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanlou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchalski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seifert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlman, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, R. S.</span></span> <span> </span><span class="NLM_article-title">Tunable-combinatorial mechanisms of acquired resistance limit the efficacy of BRAF/MEK cotargeting but result in melanoma drug addiction</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2014.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ccell.2014.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=25600339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=240-256&author=G.+Moriceauauthor=W.+Hugoauthor=A.+Hongauthor=H.+Shiauthor=X.+Kongauthor=C.+C.+Yuauthor=R.+C.+Koyaauthor=A.+A.+Samatarauthor=N.+Khanlouauthor=J.+Braunauthor=K.+Ruchalskiauthor=H.+Seifertauthor=J.+Larkinauthor=K.+B.+Dahlmanauthor=D.+B.+Johnsonauthor=A.+Algaziauthor=J.+A.+Sosmanauthor=A.+Ribasauthor=R.+S.+Lo&title=Tunable-combinatorial+mechanisms+of+acquired+resistance+limit+the+efficacy+of+BRAF%2FMEK+cotargeting+but+result+in+melanoma+drug+addiction&doi=10.1016%2Fj.ccell.2014.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Tunable-Combinatorial Mechanisms of Acquired Resistance Limit the Efficacy of BRAF/MEK Cotargeting but Result in Melanoma Drug Addiction</span></div><div class="casAuthors">Moriceau, Gatien; Hugo, Willy; Hong, Aayoung; Shi, Hubing; Kong, Xiangju; Yu, Clarissa C.; Koya, Richard C.; Samatar, Ahmed A.; Khanlou, Negar; Braun, Jonathan; Ruchalski, Kathleen; Seifert, Heike; Larkin, James; Dahlman, Kimberly B.; Johnson, Douglas B.; Algazi, Alain; Sosman, Jeffrey A.; Ribas, Antoni; Lo, Roger S.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">240-256</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Combined BRAF- and MEK-targeted therapy improves upon BRAF inhibitor (BRAFi) therapy but is still beset by acquired resistance.  We show that melanomas acquire resistance to combined BRAF and MEK inhibition by augmenting or combining mechanisms of single-agent BRAFi resistance.  These double-drug resistance-assocd. genetic configurations significantly altered mol. interactions underlying MAPK pathway reactivation. V600EBRAF, expressed at supraphysiol. levels because of V600EBRAF ultra-amplification, dimerized with and activated CRAF.  In addn., MEK mutants enhanced interaction with overexpressed V600EBRAF via a regulatory interface at R662 of V600EBRAF.  Importantly, melanoma cell lines selected for resistance to BRAFi+MEKi, but not those to BRAFi alone, displayed robust drug addiction, providing a potentially exploitable therapeutic opportunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWdJyJ2wlmALVg90H21EOLACvtfcHk0ljIlWk_zSNkqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKgtrg%253D&md5=42788605b6c22cf70ff49304bf65ca19</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2014.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2014.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DMoriceau%26aufirst%3DG.%26aulast%3DHugo%26aufirst%3DW.%26aulast%3DHong%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DC.%2BC.%26aulast%3DKoya%26aufirst%3DR.%2BC.%26aulast%3DSamatar%26aufirst%3DA.%2BA.%26aulast%3DKhanlou%26aufirst%3DN.%26aulast%3DBraun%26aufirst%3DJ.%26aulast%3DRuchalski%26aufirst%3DK.%26aulast%3DSeifert%26aufirst%3DH.%26aulast%3DLarkin%26aufirst%3DJ.%26aulast%3DDahlman%26aufirst%3DK.%2BB.%26aulast%3DJohnson%26aufirst%3DD.%2BB.%26aulast%3DAlgazi%26aufirst%3DA.%26aulast%3DSosman%26aufirst%3DJ.%2BA.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DLo%26aufirst%3DR.%2BS.%26atitle%3DTunable-combinatorial%2520mechanisms%2520of%2520acquired%2520resistance%2520limit%2520the%2520efficacy%2520of%2520BRAF%252FMEK%2520cotargeting%2520but%2520result%2520in%2520melanoma%2520drug%2520addiction%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D240%26epage%3D256%26doi%3D10.1016%2Fj.ccell.2014.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0ljIlWk_zSNkqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>474</i></span>,  <span class="NLM_fpage">1127</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1042/BCJ20160762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1042%2FBCJ20160762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=28298557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1127-1147&author=I.+Collinsauthor=H.+Wangauthor=J.+J.+Caldwellauthor=R.+Chopra&title=Chemical+approaches+to+targeted+protein+degradation+through+modulation+of+the+ubiquitin-proteasome+pathway&doi=10.1042%2FBCJ20160762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span></div><div class="casAuthors">Collins, Ian; Wang, Hannah; Caldwell, John J.; Chopra, Raj</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1127-1147</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Manipulation of the ubiquitin-proteasome system to achieve targeted degrdn. of proteins within cells using chem. tools and drugs has the potential to transform pharmacol. and therapeutic approaches in cancer and other diseases.  An increased understanding of the mol. mechanism of thalidomide and its analogs following their clin. use has unlocked small-mol. modulation of the substrate specificity of the E3 ligase cereblon (CRBN), which in turn has resulted in the advancement of new immunomodulatory drugs (IMiDs) into the clinic.  The degrdn. of multiple context-specific proteins by these pleiotropic small mols. provides a means to uncover new cell biol. and to generate future drug mols. against currently undruggable targets.  In parallel, the development of larger bifunctional mols. that bring together highly specific protein targets in complexes with CRBN, von Hippel-Lindau, or other E3 ligases to promote ubiquitin-dependent degrdn. has progressed to generate selective chem. compds. with potent effects in cells and in vivo models, providing valuable tools for biol. target validation and with future potential for therapeutic use.  In this review, we survey recent breakthroughs achieved in these two complementary methods and the discovery of new modes of direct and indirect engagement of target proteins with the proteasome.  We discuss the exptl. characterization that validates the use of mols. that promote protein degrdn. as chem. tools, the preclin. and clin. examples disclosed to date, and the future prospects for this exciting area of chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkoPeMHzjTbVg90H21EOLACvtfcHk0ljIlWk_zSNkqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D&md5=8c9ccac22f5088d776b81203dc8ebf2b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160762%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DChemical%2520approaches%2520to%2520targeted%2520protein%2520degradation%2520through%2520modulation%2520of%2520the%2520ubiquitin-proteasome%2520pathway%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26spage%3D1127%26epage%3D1147%26doi%3D10.1042%2FBCJ20160762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lihwEYgF0pNxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span> <span> </span><span class="NLM_article-title">Cloning and characterization of two distinct human extracellular signal-regulated kinase activator kinases, MEK1 and MEK2</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">11435</span>– <span class="NLM_lpage">11439</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=8388392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADyaK3sXks1KmsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=11435-11439&author=C.+F.+Zhengauthor=K.+L.+Guan&title=Cloning+and+characterization+of+two+distinct+human+extracellular+signal-regulated+kinase+activator+kinases%2C+MEK1+and+MEK2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Cloning and characterization of two distinct human extracellular signal-regulated kinase activator kinases, MEK1 and MEK2</span></div><div class="casAuthors">Zheng, Chao Feng; Guan, Kun Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">11435-9</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Mitogen-induced signal transduction is mediated by a cascade of protein phosphorylation and dephosphorylation.  One of the immediate responses of mitogen stimulation is the activation of a family of protein kinases known as mitogen-activated protein kinases or extracellular signal-regulated kinase (ERK).  MEK (MAP kinase or ERK kinase) is the immediate upstream activator kinase of ERK.  Two cDNAs, MEK1 and MEK2, were cloned and sequenced.  MEK1 and MEK2 encode 393 and 400 amino acid residues, resp.  The human MEK1 shares 99% amino acid sequence identity with the murine MEK1 and 80% with human MEK2.  Both MEK1 and MEK2 were expressed in Escherichia coli and shown to be able to activate recombinant human ERK1 in vitro.  The purified MEK2 protein stimulated threonine and tyrosine phosphorylation on ERK1 and concomitantly activated ERK1 kinase activity more than 100-fold.  The recombinant MEK2 showed lower activity as an ERK activator as compared with MEK purified from tissue.  However, the recombinant MEK2 can be activated by serum-stimulated cell ext. in vitro.  MEKs, in a manner similar to ERKs, are likely to consist of a family of related proteins playing crit. roles in signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmDkRVD-cBq7Vg90H21EOLACvtfcHk0lihwEYgF0pNxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1KmsLk%253D&md5=7c149c9378198dd108ce2736e3c55138</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DC.%2BF.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26atitle%3DCloning%2520and%2520characterization%2520of%2520two%2520distinct%2520human%2520extracellular%2520signal-regulated%2520kinase%2520activator%2520kinases%252C%2520MEK1%2520and%2520MEK2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D11435%26epage%3D11439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fremin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meloche, S.</span></span> <span> </span><span class="NLM_article-title">From basic research to clinical development of MEK1/2 inhibitors for cancer therapy</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1186/1756-8722-3-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1186%2F1756-8722-3-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=20051128" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=1-11&author=C.+Freminauthor=S.+Meloche&title=From+basic+research+to+clinical+development+of+MEK1%2F2+inhibitors+for+cancer+therapy&doi=10.1186%2F1756-8722-3-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2F1756-8722-3-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8722-3-8%26sid%3Dliteratum%253Aachs%26aulast%3DFremin%26aufirst%3DC.%26aulast%3DMeloche%26aufirst%3DS.%26atitle%3DFrom%2520basic%2520research%2520to%2520clinical%2520development%2520of%2520MEK1%252F2%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2010%26volume%3D3%26spage%3D1%26epage%3D11%26doi%3D10.1186%2F1756-8722-3-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Targeting ERK1/2 protein-serine/threonine kinases in human cancers</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">168</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2019.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.phrs.2019.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30794926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktFagurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2019&pages=151-168&author=R.+Roskoski&title=Targeting+ERK1%2F2+protein-serine%2Fthreonine+kinases+in+human+cancers&doi=10.1016%2Fj.phrs.2019.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting ERK1/2 protein-serine/threonine kinases in human cancers</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">151-168</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  ERK1 and ERK2 are key protein kinases that contribute to the Ras-Raf-MEK-ERK MAP kinase signalling module.  This pathway participates in the control of numerous processes including apoptosis, cell proliferation, the immune response, nervous system function, and RNA synthesis and processing.  MEK1/2 activate human ERK1/2 by first catalyzing the phosphorylation of Y204/187 and then T202/185, both residues of which occur within the activation segment.  The phosphorylation of both residues is required for enzyme activation.  The only Raf substrates are MEK1/2 and the only MEK1/2 substrates are ERK1/2.  In contrast, ERK1/2 catalyze the phosphorylation of many cytoplasmic and nuclear substrates including transcription factors and regulatory mols.  The linear MAP kinase pathway branches extensively at the ERK1/2 node.  ERK1/2 are proline-directed kinases that preferentially catalyze the phosphorylation of substrates contg. a PxS/TP sequence.  The dephosphorylation and inactivation of ERK1/2 is catalyzed by dual specificity phosphatases, protein-tyrosine specific phosphatases, and protein-serine/threonine phosphatases.  The combined functions of kinases and phosphatases make the overall process reversible.  To provide an idea of the complexities involved in these reactions, somatic cell cycle progression involves the strict timing of more than 32,000 phosphorylation and dephosphorylation events as detd. by mass spectrometry.  The MAP kinase cascade is perhaps the most important oncogenic driver of human cancers and the blockade of this signalling module by targeted inhibitors is an important anti-tumor strategy.  Although numerous cancers are driven by MAP kinase pathway activation, thus far the only orally effective approved drugs that target this signaling module are used for the treatment of BRAF-mutant melanomas.  The best treatments include the combination of B-Raf and MEK inhibitors (dabrafenib and trametinib, encorafenib and binimetinib, vemurafenib and cobimetanib).  However, resistance to these antagonists occurs within one year and addnl. treatment options are necessary.  Owing to the large variety of malignancies that are driven by dysregulation of the MAP kinase pathway, addnl. tumor types should be amenable to MAP kinase pathway inhibitor therapy.  In addn. to new B-Raf and MEK inhibitors, the addn. of ERK inhibitors should prove helpful.  Ulixertinib, MK-8353, and GDC-0994 are orally effective, potent, and specific inhibitors of ERK1/2 that are in early clin. trials for the treatment of various advanced/metastatic solid tumors.  These agents are effective against cell lines that are resistant to B-Raf and MEK1/2 inhibitor therapy.  Although MK-8353 does not directly inhibit MEK1/2, it decreases the phosphorylation of ERK1/2 as well as the phosphorylation of RSK, an ERK1/2 substrate.  The decrease in RSK phosphorylation appears to be a result of ERK inhibition and the decrease in ERK1/2 phosphorylation is related to the inability of MEK to catalyze the phosphorylation of the ERK-MK-8353 complex; these decreases characterize the ERK dual mechanism inhibition paradigm.  Addnl. work will be required to det. whether ERK inhibitors will be successful in the clinic and are able to forestall the development of drug resistance of the MAP kinase pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJOqTxpKRfHLVg90H21EOLACvtfcHk0lhXQIfsbSFLYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktFagurs%253D&md5=75743a80b77c8f1b49dc59de7bbc2b7f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2019.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2019.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DTargeting%2520ERK1%252F2%2520protein-serine%252Fthreonine%2520kinases%2520in%2520human%2520cancers%26jtitle%3DPharmacol.%2520Res.%26date%3D2019%26volume%3D142%26spage%3D151%26epage%3D168%26doi%3D10.1016%2Fj.phrs.2019.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgermeister, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuderland, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanoch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liscovitch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seger, R.</span></span> <span> </span><span class="NLM_article-title">Interaction with MEK causes nuclear export and downregulation of peroxisome proliferator-activated receptor gamma</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">803</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1128/MCB.00601-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1128%2FMCB.00601-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=17101779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=803-817&author=E.+Burgermeisterauthor=D.+Chuderlandauthor=T.+Hanochauthor=M.+Meyerauthor=M.+Liscovitchauthor=R.+Seger&title=Interaction+with+MEK+causes+nuclear+export+and+downregulation+of+peroxisome+proliferator-activated+receptor+gamma&doi=10.1128%2FMCB.00601-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction with MEK causes nuclear export and downregulation of peroxisome proliferator-activated receptor</span></div><div class="casAuthors">Burgermeister, Elke; Chuderland, Dana; Hanoch, Tamar; Meyer, Markus; Liscovitch, Mordechai; Seger, Rony</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">803-817</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mitogen-activated protein kinase (MAPK)/extracellular signal-regulated kinase (ERK) cascade plays a central role in intracellular signaling by many extracellular stimuli.  One target of the ERK cascade is peroxisome proliferator-activated receptor γ (PPARγ), a nuclear receptor that promotes differentiation and apoptosis.  It was previously demonstrated that PPARγ activity is attenuated upon mitogenic stimulation due to phosphorylation of its Ser84 by ERKs.  Here we show that stimulation by tetradecanoyl phorbol acetate (TPA) attenuates PPARγ's activity in a MEK-dependent manner, even when Ser84 is mutated to Ala.  To elucidate the mechanism of attenuation, we found that PPARγ directly interacts with MEKs, which are the activators of ERKs, but not with ERKs themselves, both in vivo and in vitro.  This interaction is facilitated by MEKs' phosphorylation and is mediated by the basic D domain of MEK1 and the AF2 domain of PPARγ.  Immunofluorescence microscopy and subcellular fractionation revealed that MEK1 exports PPARγ from the nucleus, and this finding was supported by small interfering RNA knockdown of MEK1 and use of a cell-permeable interaction-blocking peptide, which prevented TPA-induced export of PPARγ from the nucleus.  Thus, we show here a novel mode of downregulation of PPARγ by its MEK-dependent redistribution from the nucleus to the cytosol.  This unanticipated role for the stimulation-induced nuclear shuttling of MEKs shows that MEKs can regulate addnl. signaling components besides the ERK cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyBt7HomvWmLVg90H21EOLACvtfcHk0lhXQIfsbSFLYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGht7s%253D&md5=60b9cd18e1c5ac33122ea01c109fcd02</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1128%2FMCB.00601-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00601-06%26sid%3Dliteratum%253Aachs%26aulast%3DBurgermeister%26aufirst%3DE.%26aulast%3DChuderland%26aufirst%3DD.%26aulast%3DHanoch%26aufirst%3DT.%26aulast%3DMeyer%26aufirst%3DM.%26aulast%3DLiscovitch%26aufirst%3DM.%26aulast%3DSeger%26aufirst%3DR.%26atitle%3DInteraction%2520with%2520MEK%2520causes%2520nuclear%2520export%2520and%2520downregulation%2520of%2520peroxisome%2520proliferator-activated%2520receptor%2520gamma%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D803%26epage%3D817%26doi%3D10.1128%2FMCB.00601-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahmi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maisonneuve, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marullo, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thevakumaran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurinov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, M.</span></span> <span> </span><span class="NLM_article-title">MEK drives BRAF activation through allosteric control of KSR proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>554</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1038/nature25478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnature25478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29433126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1SltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=554&publication_year=2018&pages=549-553&author=H.+Lavoieauthor=M.+Sahmiauthor=P.+Maisonneuveauthor=S.+A.+Marulloauthor=N.+Thevakumaranauthor=T.+Jinauthor=I.+Kurinovauthor=F.+Sicheriauthor=M.+Therrien&title=MEK+drives+BRAF+activation+through+allosteric+control+of+KSR+proteins&doi=10.1038%2Fnature25478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">MEK drives BRAF activation through allosteric control of KSR proteins</span></div><div class="casAuthors">Lavoie, Hugo; Sahmi, Malha; Maisonneuve, Pierre; Marullo, Sara A.; Thevakumaran, Neroshan; Jin, Ting; Kurinov, Igor; Sicheri, Frank; Therrien, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">554</span>
        (<span class="NLM_cas:issue">7693</span>),
    <span class="NLM_cas:pages">549-553</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">RAF family kinases have prominent roles in cancer.  Their activation is dependent on dimerization of their kinase domains, which has emerged as a hindrance for drug development.  In mammals, RAF family kinases include three catalytically competent enzymes (ARAF, BRAF and CRAF) and two pseudokinases (KSR1 and KSR2) that have been described as scaffolds owing to their apparent ability to bridge RAF isoforms and their substrate, mitogen-activated protein kinase kinase (MEK).  Kinase suppressor of Ras (KSR) pseudokinases were also shown to dimerize with kinase-competent RAFs to stimulate catalysis allosterically.  Although GTP-bound RAS can modulate the dimerization of RAF isoforms by engaging their RAS-binding domains, KSR1 and KSR2 lack an RAS-binding domain and therefore the regulatory principles underlying their dimerization with other RAF family members remain unknown.  Here we show that the selective heterodimerization of BRAF with KSR1 is specified by direct contacts between the amino-terminal regulatory regions of each protein, comprising in part a novel domain called BRS in BRAF and the coiled-coil-sterile α motif (CC-SAM) domain in KSR1.  We also discovered that MEK binding to the kinase domain of KSR1 asym. drives BRAF-KSR1 heterodimerization, resulting in the concomitant stimulation of BRAF catalytic activity towards free MEK mols.  These findings demonstrate that KSR-MEK complexes allosterically activate BRAF through the action of N-terminal regulatory region and kinase domain contacts and challenge the accepted role of KSR as a scaffold for MEK recruitment to RAF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprMdHlsp1vWbVg90H21EOLACvtfcHk0lhXQIfsbSFLYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1SltLY%253D&md5=8b50458212b822a6c9caefda2b08aa85</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnature25478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25478%26sid%3Dliteratum%253Aachs%26aulast%3DLavoie%26aufirst%3DH.%26aulast%3DSahmi%26aufirst%3DM.%26aulast%3DMaisonneuve%26aufirst%3DP.%26aulast%3DMarullo%26aufirst%3DS.%2BA.%26aulast%3DThevakumaran%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DT.%26aulast%3DKurinov%26aufirst%3DI.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DTherrien%26aufirst%3DM.%26atitle%3DMEK%2520drives%2520BRAF%2520activation%2520through%2520allosteric%2520control%2520of%2520KSR%2520proteins%26jtitle%3DNature%26date%3D2018%26volume%3D554%26spage%3D549%26epage%3D553%26doi%3D10.1038%2Fnature25478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blasco, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francoz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santamaria, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canamero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubus, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baccarini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">c-Raf, but not B-Raf, is essential for development of K-Ras oncogene-driven non-small cell lung carcinoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">652</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2011.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ccr.2011.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=21514245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFGls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=652-663&author=R.+B.+Blascoauthor=S.+Francozauthor=D.+Santamariaauthor=M.+Canameroauthor=P.+Dubusauthor=J.+Charronauthor=M.+Baccariniauthor=M.+Barbacid&title=c-Raf%2C+but+not+B-Raf%2C+is+essential+for+development+of+K-Ras+oncogene-driven+non-small+cell+lung+carcinoma&doi=10.1016%2Fj.ccr.2011.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">c-Raf, but Not B-Raf, Is Essential for Development of K-Ras Oncogene-Driven Non-Small Cell Lung Carcinoma</span></div><div class="casAuthors">Blasco, Rafael B.; Francoz, Sarah; Santamaria, David; Canamero, Marta; Dubus, Pierre; Charron, Jean; Baccarini, Manuela; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">652-663</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We have investigated the role of individual members of the Raf/Mek/Erk cascade in the onset of K-Ras oncogene-driven non-small cell lung carcinoma (NSCLC).  Ablation of Erk1 or Erk2 in K-Ras oncogene-expressing lung cells had no significant effect due to compensatory activities.  Yet, elimination of both Erk kinases completely blocked tumor development.  Similar results were obtained with Mek kinases.  Ablation of B-Raf had no significant effect on tumor development.  However, c-Raf expression was absolutely essential for the onset of NSCLC.  Interestingly, concomitant elimination of c-Raf and B-Raf in adult mice had no deleterious consequences for normal homeostasis.  These results indicate that c-Raf plays a unique role in mediating K-Ras signaling and makes it a suitable target for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCWweCotG3S7Vg90H21EOLACvtfcHk0liNZDw-DFM-eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFGls7c%253D&md5=11e9470b124d1c3ba6afe7e930f8438f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DBlasco%26aufirst%3DR.%2BB.%26aulast%3DFrancoz%26aufirst%3DS.%26aulast%3DSantamaria%26aufirst%3DD.%26aulast%3DCanamero%26aufirst%3DM.%26aulast%3DDubus%26aufirst%3DP.%26aulast%3DCharron%26aufirst%3DJ.%26aulast%3DBaccarini%26aufirst%3DM.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3Dc-Raf%252C%2520but%2520not%2520B-Raf%252C%2520is%2520essential%2520for%2520development%2520of%2520K-Ras%2520oncogene-driven%2520non-small%2520cell%2520lung%2520carcinoma%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D19%26spage%3D652%26epage%3D663%26doi%3D10.1016%2Fj.ccr.2011.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebraud, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Protein degradation by in-cell self-assembly of proteolysis targeting chimeras</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">934</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.6b00280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.6b00280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=927-934&author=H.+Lebraudauthor=D.+J.+Wrightauthor=C.+N.+Johnsonauthor=T.+D.+Heightman&title=Protein+degradation+by+in-cell+self-assembly+of+proteolysis+targeting+chimeras&doi=10.1021%2Facscentsci.6b00280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Degradation by In-Cell Self-Assembly of Proteolysis Targeting Chimeras</span></div><div class="casAuthors">Lebraud, Honorine; Wright, David J.; Johnson, Christopher N.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">927-934</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective degrdn. of proteins by proteolysis targeting chimeras (PROTACs) offers a promising potential alternative to protein inhibition for therapeutic intervention.  Current PROTAC mols. incorporate a ligand for the target protein, a linker, and an E3 ubiquitin ligase recruiting group, which bring together target protein and ubiquitinating machinery.  Such hetero-bifunctional mols. require significant linker optimization and possess high mol. wt., which can limit cellular permeation, soly., and other drug-like properties.  We show here that the hetero-bifunctional mol. can be formed intracellularly by bio-orthogonal click combination of two smaller precursors.  We designed a tetrazine tagged thalidomide deriv. which reacts rapidly with a trans-cyclo-octene tagged ligand of the target protein in cells to form a cereblon E3 ligase recruiting PROTAC mol.  The in-cell click-formed proteolysis targeting chimeras (CLIPTACs) were successfully used to degrade two key oncol. targets, BRD4 and ERK1/2.  ERK1/2 degrdn. was achieved using a CLIPTAC based on a covalent inhibitor.  We expect this approach to be readily extendable to other inhibitor-protein systems because the tagged E3 ligase recruiter is capable of undergoing the click reaction with a suitably tagged ligand of any protein of interest to elicit its degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRnQIoT7hbt7Vg90H21EOLACvtfcHk0liNZDw-DFM-eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ&md5=4cc9b4aaba6ee03cd2d4ea97a72728a5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.6b00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.6b00280%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DWright%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DProtein%2520degradation%2520by%2520in-cell%2520self-assembly%2520of%2520proteolysis%2520targeting%2520chimeras%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2016%26volume%3D2%26spage%3D927%26epage%3D934%26doi%3D10.1021%2Facscentsci.6b00280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gou, S.</span></span> <span> </span><span class="NLM_article-title">Pomalidomide hybrids act as proteolysis targeting chimeras: synthesis, anticancer activity and B-Raf degradation</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2019.03.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.bioorg.2019.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30901674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1ehsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2019&pages=191-199&author=H.+Chenauthor=F.+Chenauthor=S.+Peiauthor=S.+Gou&title=Pomalidomide+hybrids+act+as+proteolysis+targeting+chimeras%3A+synthesis%2C+anticancer+activity+and+B-Raf+degradation&doi=10.1016%2Fj.bioorg.2019.03.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Pomalidomide hybrids act as proteolysis targeting chimeras: Synthesis, anticancer activity and B-Raf degradation</span></div><div class="casAuthors">Chen, Hong; Chen, Feihong; Pei, Sinan; Gou, Shaohua</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">191-199</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As the first intracellular signaling mol. and the most frequently mutated oncogene, B-Raf represents an important target in cancer therapy.  Here we report several pomalidomide hybrids acting as proteolysis targeting chimeras (PROTACs) for the degrdn. of B-Raf.  Due to its high expression of B-Raf, MCF-7 cells are sensitive to these compds.  Among them, compd. 2 can effectively kill cancer cells via inducing cells apoptosis.  As a B-Raf degrader, compd. 2 can accelerate the degrdn. of B-Raf by recruiting ubiquitin-proteasome system, and further affects the expression of Mcl-1, a downstream protein of B-Raf.  The anticancer mechanism of compd. 2 is quite different from its mother compd. and cancer cells seem to be more sensitive to the degrader, hinting that degrdn. of B-Raf by PROTAC is a potential way for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQqYbDk5Vs5LVg90H21EOLACvtfcHk0liNZDw-DFM-eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXls1ehsLY%253D&md5=0d717c1e43a72d5213857ed47dd24a57</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2019.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2019.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DPei%26aufirst%3DS.%26aulast%3DGou%26aufirst%3DS.%26atitle%3DPomalidomide%2520hybrids%2520act%2520as%2520proteolysis%2520targeting%2520chimeras%253A%2520synthesis%252C%2520anticancer%2520activity%2520and%2520B-Raf%2520degradation%26jtitle%3DBioorg.%2520Chem.%26date%3D2019%26volume%3D87%26spage%3D191%26epage%3D199%26doi%3D10.1016%2Fj.bioorg.2019.03.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Overcoming resistance to targeted anticancer therapies through small-molecule-mediated MEK degradation</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">996</span>– <span class="NLM_lpage">1005</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.chembiol.2018.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29909991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCqtLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=996-1005&author=J.+Pehauthor=M.+W.+Boudreauauthor=H.+M.+Smithauthor=P.+J.+Hergenrother&title=Overcoming+resistance+to+targeted+anticancer+therapies+through+small-molecule-mediated+MEK+degradation&doi=10.1016%2Fj.chembiol.2018.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming Resistance to Targeted Anticancer Therapies through Small-Molecule-Mediated MEK Degradation</span></div><div class="casAuthors">Peh, Jessie; Boudreau, Matthew W.; Smith, Hannah M.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">996-1005.e4</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The discovery of mutant or fusion kinases that drive oncogenesis, and the subsequent approval of specific inhibitors for these enzymes, has been instrumental in the management of some cancers.  However, acquired resistance remains a significant problem in the clinic, limiting the long-term effectiveness of most of these drugs.  Here we demonstrate a general strategy to overcome this resistance through drug-induced MEK cleavage (via direct procaspase-3 activation) combined with targeted kinase inhibition.  This combination effect is shown to be general across diverse tumor histologies (melanoma, lung cancer, and leukemia) and driver mutations (mutant BRAF or EGFR, fusion kinases EML4-ALK and BCR-ABL).  Caspase-3-mediated degrdn. of MEK kinases results in sustained pathway inhibition and substantially delayed or eliminated resistance in cancer cells in a manner far superior to combinations with MEK inhibitors.  These data suggest the generality of drug-mediated MEK kinase cleavage as a therapeutic strategy to prevent resistance to targeted anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopoZHGk_4QDbVg90H21EOLACvtfcHk0liNZDw-DFM-eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCqtLbN&md5=83d3b1df9388e9de083ff139e452b2f6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DPeh%26aufirst%3DJ.%26aulast%3DBoudreau%26aufirst%3DM.%2BW.%26aulast%3DSmith%26aufirst%3DH.%2BM.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DOvercoming%2520resistance%2520to%2520targeted%2520anticancer%2520therapies%2520through%2520small-molecule-mediated%2520MEK%2520degradation%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D996%26epage%3D1005%26doi%3D10.1016%2Fj.chembiol.2018.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.+Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lilK54TSE4ukQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span> <span> </span><span class="NLM_article-title">Current development status of MEK inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.3390/molecules22101551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.3390%2Fmolecules22101551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1-20&author=Y.+Chengauthor=H.+Tian&title=Current+development+status+of+MEK+inhibitors&doi=10.3390%2Fmolecules22101551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.3390%2Fmolecules22101551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules22101551%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DH.%26atitle%3DCurrent%2520development%2520status%2520of%2520MEK%2520inhibitors%26jtitle%3DMolecules%26date%3D2017%26volume%3D22%26spage%3D1%26epage%3D20%26doi%3D10.3390%2Fmolecules22101551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohren, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuffa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spessard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banotai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudley, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasemann, C. A.</span></span> <span> </span><span class="NLM_article-title">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1192</span>– <span class="NLM_lpage">1197</span>, <span class="refDoi"> DOI: 10.1038/nsmb859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnsmb859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=15543157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1192-1197&author=J.+F.+Ohrenauthor=H.+Chenauthor=A.+Pavlovskyauthor=C.+Whiteheadauthor=E.+Zhangauthor=P.+Kuffaauthor=C.+Yanauthor=P.+McConnellauthor=C.+Spessardauthor=C.+Banotaiauthor=W.+T.+Muellerauthor=A.+Delaneyauthor=C.+Omerauthor=J.+Sebolt-Leopoldauthor=D.+T.+Dudleyauthor=I.+K.+Leungauthor=C.+Flammeauthor=J.+Warmusauthor=M.+Kaufmanauthor=S.+Barrettauthor=H.+Tecleauthor=C.+A.+Hasemann&title=Structures+of+human+MAP+kinase+kinase+1+%28MEK1%29+and+MEK2+describe+novel+noncompetitive+kinase+inhibition&doi=10.1038%2Fnsmb859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span></div><div class="casAuthors">Ohren, Jeffrey F.; Chen, Huifen; Pavlovsky, Alexander; Whitehead, Christopher; Zhang, Erli; Kuffa, Peter; Yan, Chunhong; McConnell, Patrick; Spessard, Cindy; Banotai, Craig; Mueller, W. Thomas; Delaney, Amy; Omer, Charles; Sebolt-Leopold, Judith; Dudley, David T.; Leung, Iris K.; Flamme, Cathlin; Warmus, Joseph; Kaufman, Michael; Barrett, Stephen; Tecle, Haile; Hasemann, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1192-1197</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MEK1 and MEK2 kinases are closely related, dual-specificity tyrosine/threonine protein kinases found in the Ras/Raf/MEK/ERK MAP kinase (MAPK) signaling pathway.  Approx. 30% of all human cancers have a constitutively activated MAPK pathway, and constitutive activation of MEK1 results in cellular transformation.  Here, the authors present the x-ray crystal structures of human MEK1 and MEK2 kinases, each detd. as a ternary complex with MgATP and an inhibitor to a resoln. of 2.4 and 3.2 Å, resp.  The structures revealed that MEK1 and MEK2 each had a unique inhibitor-binding pocket adjacent to the MgATP-binding site.  The presence of the potent inhibitors induced several conformational changes in the unphosphorylated MEK1 and MEK2 kinases that locked them into a closed but catalytically inactive species.  Thus, the structures reported here reveal a novel, noncompetitive mechanism for protein kinase inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIY9begVVfW7Vg90H21EOLACvtfcHk0lilK54TSE4ukQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM&md5=79ebbc2e3a9cdd3fbecb8bb35ea9dba3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnsmb859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb859%26sid%3Dliteratum%253Aachs%26aulast%3DOhren%26aufirst%3DJ.%2BF.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DPavlovsky%26aufirst%3DA.%26aulast%3DWhitehead%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DKuffa%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DMcConnell%26aufirst%3DP.%26aulast%3DSpessard%26aufirst%3DC.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DMueller%26aufirst%3DW.%2BT.%26aulast%3DDelaney%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DC.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DDudley%26aufirst%3DD.%2BT.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DFlamme%26aufirst%3DC.%26aulast%3DWarmus%26aufirst%3DJ.%26aulast%3DKaufman%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DS.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DHasemann%26aufirst%3DC.%2BA.%26atitle%3DStructures%2520of%2520human%2520MAP%2520kinase%2520kinase%25201%2520%2528MEK1%2529%2520and%2520MEK2%2520describe%2520novel%2520noncompetitive%2520kinase%2520inhibition%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1192%26epage%3D1197%26doi%3D10.1038%2Fnsmb859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Dort, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nino, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanks, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galban, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, B. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of bifunctional oncogenic target inhibitors against allosteric mitogen-activated protein kinase (MEK1) and phosphatidylinositol 3-kinase (PI3K)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2512</span>– <span class="NLM_lpage">2522</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs1yqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2512-2522&author=M.+E.+Van%0ADortauthor=H.+Hongauthor=H.+Wangauthor=C.+A.+Ninoauthor=R.+L.+Lombardiauthor=A.+E.+Blanksauthor=S.+Galbanauthor=B.+D.+Ross&title=Discovery+of+bifunctional+oncogenic+target+inhibitors+against+allosteric+mitogen-activated+protein+kinase+%28MEK1%29+and+phosphatidylinositol+3-kinase+%28PI3K%29&doi=10.1021%2Facs.jmedchem.5b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bifunctional Oncogenic Target Inhibitors against Allosteric Mitogen-Activated Protein Kinase (MEK1) and Phosphatidylinositol 3-Kinase (PI3K)</span></div><div class="casAuthors">Van Dort, Marcian E.; Hong, Hao; Wang, Hanxiao; Nino, Charles A.; Lombardi, Rachel L.; Blanks, Avery E.; Galban, Stefanie; Ross, Brian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2512-2522</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of a series of single entity, bifunctional MEK1/PI3K inhibitors achieved by covalent linking of structural analogs of the ATP-competitive PI3K inhibitor ZSTK474 and the ATP-noncompetitive MEK inhibitor PD0325901 is described.  Inhibitors displayed potent in vitro inhibition of MEK1 (0.015 < IC50 (nM) < 56.7) and PI3K (54 < IC50 (nM) < 341) in enzymic inhibition assays.  Concurrent MEK1 and PI3K inhibition was demonstrated with inhibitors 9 and 14 in two tumor cell lines (A549, D54).  Inhibitors produced dose-dependent decreased cell viability similar to the combined administration of equiv. doses of ZSTK474 and PD0325901.  In vivo efficacy of 14 following oral administration was demonstrated in D54 glioma and A549 lung tumor bearing mice.  Compd. 14 showed a 95% and 67% inhibition of tumor ERK1/2 and Akt phosphorylation, resp., at 2 h postadministration by Western blot anal., confirming the bioavailability and efficacy of this bifunctional inhibitor strategy toward combined MEK1/PI3K inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxtplBF0UZ7bVg90H21EOLACvtfcHk0lgjqtAkoLwclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs1yqsL4%253D&md5=1bbc4eef05d0c4bab6a21e7a5db2790e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01655%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDort%26aufirst%3DM.%2BE.%26aulast%3DHong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DNino%26aufirst%3DC.%2BA.%26aulast%3DLombardi%26aufirst%3DR.%2BL.%26aulast%3DBlanks%26aufirst%3DA.%2BE.%26aulast%3DGalban%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DB.%2BD.%26atitle%3DDiscovery%2520of%2520bifunctional%2520oncogenic%2520target%2520inhibitors%2520against%2520allosteric%2520mitogen-activated%2520protein%2520kinase%2520%2528MEK1%2529%2520and%2520phosphatidylinositol%25203-kinase%2520%2528PI3K%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2512%26epage%3D2522%26doi%3D10.1021%2Facs.jmedchem.5b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lgjqtAkoLwclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span> <span> </span><span class="NLM_article-title">MEK inhibitors induce apoptosis via FoxO3a-dependent PUMA induction in colorectal cancer cells</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1038/s41389-018-0078-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fs41389-018-0078-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29367650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFChsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=1-9&author=L.+Linauthor=D.+Dingauthor=Y.+Jiangauthor=Y.+Liauthor=S.+Li&title=MEK+inhibitors+induce+apoptosis+via+FoxO3a-dependent+PUMA+induction+in+colorectal+cancer+cells&doi=10.1038%2Fs41389-018-0078-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Atypical GATA transcription factor TRPS1 represses gene expression by recruiting CHD4/NuRD(MTA2) and suppresses cell migration and invasion by repressing TP63 expression</span></div><div class="casAuthors">Wang, Yuzhi; Lin, Xue; Gong, Xue; Wu, Lele; Zhang, Jun; Liu, Weiguang; Li, Jian; Chen, Liming</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">ONCOCT</span>;
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Transcriptional repressor GATA binding 1 (TRPS1), an atypical GATA transcription factor, functions as a transcriptional repressor and is also implicated in human cancers.  However, the underlying mechanism of TRPS1 contributing to malignancy remains obscure.  In the current study, we report that TRPS1 recognizes both gene proximal and distal transcription start site (TSS) sequences to repress gene expression.  Co-IP mass spectrometry and biochem. studies showed that TRPS1 binds to CHD4/NuRD(MTA2).  Genome-wide and mol. studies revealed that CHD4/NuRD(MTA2) is required for TRPS1 transcriptional repression.  Mech., TRPS1 and CHD4/NuRD(MTA2) form precision-guided transcriptional repression machinery in which TRPS1 guides the machinery to specific target sites by recognizing GATA elements, and CHD4/NuRD(MTA2) represses the transcription of target genes.  Furthermore, TP63 was identified and validated to be a direct target of TRPS1-CHD4/NuRD(MTA2) complex, which represses TP63 expression by involving decommission of TP63 enhancer in the described precision-guided manner, leading to a redn. of the ΔNp63 level and contributing to migration and invasion of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHKAZL8GecY7Vg90H21EOLACvtfcHk0lgjqtAkoLwclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFChsrfK&md5=34a4014288e6d1504c4da2ad214d67ad</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41389-018-0078-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-018-0078-y%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26atitle%3DMEK%2520inhibitors%2520induce%2520apoptosis%2520via%2520FoxO3a-dependent%2520PUMA%2520induction%2520in%2520colorectal%2520cancer%2520cells%26jtitle%3DOncogenesis%26date%3D2018%26volume%3D7%26spage%3D1%26epage%3D9%26doi%3D10.1038%2Fs41389-018-0078-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lgjqtAkoLwclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lgzmr_n54ej5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratilas, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span> <span> </span><span class="NLM_article-title">BRAF mutation predicts sensitivity to MEK inhibition</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>439</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1038/nature04304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnature04304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=16273091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvF2mtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=439&publication_year=2006&pages=358-362&author=D.+B.+Solitauthor=L.+A.+Garrawayauthor=C.+A.+Pratilasauthor=A.+Sawaiauthor=G.+Getzauthor=A.+Bassoauthor=Q.+Yeauthor=J.+M.+Loboauthor=Y.+Sheauthor=I.+Osmanauthor=T.+R.+Golubauthor=J.+Sebolt-Leopoldauthor=W.+R.+Sellersauthor=N.+Rosen&title=BRAF+mutation+predicts+sensitivity+to+MEK+inhibition&doi=10.1038%2Fnature04304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">BRAF mutation predicts sensitivity to MEK inhibition</span></div><div class="casAuthors">Solit, David B.; Garraway, Levi A.; Pratilas, Christine A.; Sawai, Ayana; Getz, Gad; Basso, Andrea; Ye, Qing; Lobo, Jose M.; She, Yuhong; Osman, Iman; Golub, Todd R.; Sebolt-Leopold, Judith; Sellers, William R.; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">439</span>
        (<span class="NLM_cas:issue">7074</span>),
    <span class="NLM_cas:pages">358-362</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The kinase pathway comprising RAS, RAF, mitogen-activated protein kinase kinase (MEK) and extracellular signal regulated kinase (ERK) is activated in most human tumors, often through gain-of-function mutations of RAS and RAF family members.  Using small-mol. inhibitors of MEK and an integrated genetic and pharmacol. anal., we find that mutation of BRAF is assocd. with enhanced and selective sensitivity to MEK inhibition when compared to either 'wild-type' cells or cells harboring a RAS mutation.  This MEK dependency was obsd. in BRAF mutant cells regardless of tissue lineage, and correlated with both downregulation of cyclin D1 protein expression and the induction of G1 arrest.  Pharmacol. MEK inhibition completely abrogated tumor growth in BRAF mutant xenografts, whereas RAS mutant tumors were only partially inhibited.  These data suggest an exquisite dependency on MEK activity in BRAF mutant tumors, and offer a rational therapeutic strategy for this genetically defined tumor subtype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpBpONOvd0vbVg90H21EOLACvtfcHk0lgzmr_n54ej5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvF2mtw%253D%253D&md5=37c8976aa5bb371ef5580aab2a91f89e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature04304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04304%26sid%3Dliteratum%253Aachs%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26aulast%3DSawai%26aufirst%3DA.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DBasso%26aufirst%3DA.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DLobo%26aufirst%3DJ.%2BM.%26aulast%3DShe%26aufirst%3DY.%26aulast%3DOsman%26aufirst%3DI.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DBRAF%2520mutation%2520predicts%2520sensitivity%2520to%2520MEK%2520inhibition%26jtitle%3DNature%26date%3D2006%26volume%3D439%26spage%3D358%26epage%3D362%26doi%3D10.1038%2Fnature04304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asundi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacap, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span> <span> </span><span class="NLM_article-title">MAPK pathway inhibition enhances the efficacy of an anti-endothelin B receptor drug conjugate by inducing target expression in melanoma</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0446</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1158%2F1535-7163.MCT-13-0446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=24651527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpt1Ojsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1599-1610&author=J.+Asundiauthor=J.+A.+Lacapauthor=S.+Clarkauthor=M.+Nanniniauthor=L.+Rothauthor=P.+Polakis&title=MAPK+pathway+inhibition+enhances+the+efficacy+of+an+anti-endothelin+B+receptor+drug+conjugate+by+inducing+target+expression+in+melanoma&doi=10.1158%2F1535-7163.MCT-13-0446"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MAPK Pathway Inhibition Enhances the Efficacy of an Anti-Endothelin B Receptor Drug Conjugate by Inducing Target Expression in Melanoma</span></div><div class="casAuthors">Asundi, Jyoti; Lacap, Jennifer A.; Clark, Suzanna; Nannini, Michelle; Roth, Leslie; Polakis, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1599-1610</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Therapies targeting the mitogen-activated protein (MAP) kinase pathway in melanoma have produced significant clin. responses; however, duration of response is limited by acquisition of drug resistance.  Rational drug combinations may improve outcomes in this setting.  We assessed the therapeutic combination of an antibody-drug conjugate (ADC) targeting the endothelin B receptor (EDNRB) with small-mol. inhibitors of the MAP kinase signaling pathway in melanoma.  Cell lines and tumor models contg. either mutant BRAF or NRAS, or wild-type for both, were exposed to small-mol. inhibitors of BRAF and MEK.  Expression of EDNRB was analyzed and the therapeutic impact of combining the anti-EDNRB ADC with the BRAF and MEK inhibitors was assessed.  Increased expression of EDNRB in response to inhibition of BRAF and/or MEK was obsd. and augmented the antitumor activity of the ADC.  Enhanced target expression and ADC antitumor activity were realized irresp. of the response of the tumor model to the BRAF or MEK inhibitors alone and could be achieved in melanoma with mutant NRAS, BRAF, or neither mutation.  Cells that acquired resistance to BRAF inhibition through long-term culture retained drug-induced elevated levels of EDNRB expression.  Expression of EDNRB was not enhanced in normal human melanocytes by inhibition of BRAF and the combination of the ADC with MAPK inhibitors was well-tolerated in mice.  The anti-EDNRB ADC combines well with BRAF and MEK inhibitors and could have therapeutic use in the majority of human melanoma cases.  Mol Cancer Ther; 13(6); 1599-610. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6AdZ-w8sQgLVg90H21EOLACvtfcHk0lgzmr_n54ej5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpt1Ojsb0%253D&md5=91e2a8f9a26da15c08f752429f8cdd55</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0446%26sid%3Dliteratum%253Aachs%26aulast%3DAsundi%26aufirst%3DJ.%26aulast%3DLacap%26aufirst%3DJ.%2BA.%26aulast%3DClark%26aufirst%3DS.%26aulast%3DNannini%26aufirst%3DM.%26aulast%3DRoth%26aufirst%3DL.%26aulast%3DPolakis%26aufirst%3DP.%26atitle%3DMAPK%2520pathway%2520inhibition%2520enhances%2520the%2520efficacy%2520of%2520an%2520anti-endothelin%2520B%2520receptor%2520drug%2520conjugate%2520by%2520inducing%2520target%2520expression%2520in%2520melanoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D1599%26epage%3D1610%26doi%3D10.1158%2F1535-7163.MCT-13-0446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACs: An emerging and promising approach for the development of targeted therapy drugs</span>. <i>EBioMedicine</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1016/j.ebiom.2018.09.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ebiom.2018.09.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30224312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BB3c3pslWjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=553-562&author=S.+Anauthor=L.+Fu&title=Small-molecule+PROTACs%3A+An+emerging+and+promising+approach+for+the+development+of+targeted+therapy+drugs&doi=10.1016%2Fj.ebiom.2018.09.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule PROTACs: An emerging and promising approach for the development of targeted therapy drugs</span></div><div class="casAuthors">An Sainan; Fu Liwu</div><div class="citationInfo"><span class="NLM_cas:title">EBioMedicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">553-562</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">There are several challenges towards the development and clinical use of small molecule inhibitors, which are currently the main type of targeted therapies towards intracellular proteins.  PROteolysis-TArgeting Chimeras (PROTACs) exploit the intracellular ubiquitin-proteasome system to selectively degrade target proteins.  Recently, small-molecule PROTACs with high potency have been frequently reported.  In this review, we summarize the emerging characteristics of small-molecule PROTACs, such as inducing a rapid, profound and sustained degradation, inducing a robust inhibition of downstream signals, displaying enhanced target selectivity, and overcoming resistance to small molecule inhibitors.  In tumor xenografts, small-molecule PROTACs can significantly attenuate tumor progression.  In addition, we also introduce recent developments of the PROTAC technology such as homo-PROTACs.  The outstanding advantages over traditional small-molecule drugs and the promising preclinical data suggest that small-molecule PROTAC technology has the potential to greatly promote the development of targeted therapy drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSW7YU6u8U3fIFEOhA_Im3cfW6udTcc2eZWMcVL5eer27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3pslWjtQ%253D%253D&md5=415a5c492059c464dede49de6375bdba</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.ebiom.2018.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ebiom.2018.09.005%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DL.%26atitle%3DSmall-molecule%2520PROTACs%253A%2520An%2520emerging%2520and%2520promising%2520approach%2520for%2520the%2520development%2520of%2520targeted%2520therapy%2520drugs%26jtitle%3DEBioMedicine%26date%3D2018%26volume%3D36%26spage%3D553%26epage%3D562%26doi%3D10.1016%2Fj.ebiom.2018.09.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">ERK1/2 MAP kinases: structure, function, and regulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2012.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.phrs.2012.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=22569528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC38XovVygs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2012&pages=105-143&author=R.+Roskoski&title=ERK1%2F2+MAP+kinases%3A+structure%2C+function%2C+and+regulation&doi=10.1016%2Fj.phrs.2012.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">ERK1/2 MAP kinases: Structure, function, and regulation</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-143</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  ERK1 and ERK2 are related protein-serine/threonine kinases that participate in the Ras-Raf-MEK-ERK signal transduction cascade.  This cascade participates in the regulation of a large variety of processes including cell adhesion, cell cycle progression, cell migration, cell survival, differentiation, metab., proliferation, and transcription.  MEK1/2 catalyze the phosphorylation of human ERK1/2 at Tyr204/187 and then Thr202/185.  The phosphorylation of both tyrosine and threonine is required for enzyme activation.  Whereas the Raf kinase and MEK families have narrow substrate specificity, ERK1/2 catalyze the phosphorylation of hundreds of cytoplasmic and nuclear substrates including regulatory mols. and transcription factors.  ERK1/2 are proline-directed kinases that preferentially catalyze the phosphorylation of substrates contg. a Pro-Xxx-Ser/Thr-Pro sequence.  Besides this primary structure requirement, many ERK1/2 substrates possess a D-docking site, an F-docking site, or both.  A variety of scaffold proteins including KSR1/2, IQGAP1, MP1, β-Arrestin1/2 participate in the regulation of the ERK1/2 MAP kinase cascade.  The regulatory dephosphorylation of ERK1/2 is mediated by protein-tyrosine specific phosphatases, protein-serine/threonine phosphatases, and dual specificity phosphatases.  The combination of kinases and phosphatases make the overall process reversible.  The ERK1/2 catalyzed phosphorylation of nuclear transcription factors including those of Ets, Elk, and c-Fos represents an important function and requires the translocation of ERK1/2 into the nucleus by active and passive processes involving the nuclear pore.  These transcription factors participate in the immediate early gene response.  The activity of the Ras-Raf-MEK-ERK cascade is increased in about one-third of all human cancers, and inhibition of components of this cascade by targeted inhibitors represents an important anti-tumor strategy.  Thus far, however, only inhibition of mutant B-Raf (Val600Glu) has been found to be therapeutically efficacious.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUVgAuIw2CcLVg90H21EOLACvtfcHk0liyZXFWJX7I4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovVygs7k%253D&md5=ee168ded480a45f8995278f06bef6440</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2012.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2012.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DERK1%252F2%2520MAP%2520kinases%253A%2520structure%252C%2520function%252C%2520and%2520regulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2012%26volume%3D66%26spage%3D105%26epage%3D143%26doi%3D10.1016%2Fj.phrs.2012.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lito, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saborowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastenhuber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morikami, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span> <span> </span><span class="NLM_article-title">Disruption of CRAF-mediated MEK activation is required for effective MEK inhibition in KRAS mutant tumors</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">697</span>– <span class="NLM_lpage">710</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ccr.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=24746704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVCht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=697-710&author=P.+Litoauthor=A.+Saborowskiauthor=J.+Yueauthor=M.+Solomonauthor=E.+Josephauthor=S.+Gadalauthor=M.+Saborowskiauthor=E.+Kastenhuberauthor=C.+Fellmannauthor=K.+Oharaauthor=K.+Morikamiauthor=T.+Miuraauthor=C.+Lukacsauthor=N.+Ishiiauthor=S.+Loweauthor=N.+Rosen&title=Disruption+of+CRAF-mediated+MEK+activation+is+required+for+effective+MEK+inhibition+in+KRAS+mutant+tumors&doi=10.1016%2Fj.ccr.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Disruption of CRAF-Mediated MEK Activation Is Required for Effective MEK Inhibition in KRAS Mutant Tumors</span></div><div class="casAuthors">Lito, Piro; Saborowski, Anna; Yue, Jingyin; Solomon, Martha; Joseph, Eric; Gadal, Sunyana; Saborowski, Michael; Kastenhuber, Edward; Fellmann, Christof; Ohara, Kazuhiro; Morikami, Kenji; Miura, Takaaki; Lukacs, Christine; Ishii, Nobuya; Lowe, Scott; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-710</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MEK inhibitors are clin. active in BRAFV600E melanomas but only marginally so in KRAS mutant tumors.  Here, we found that MEK inhibitors suppress ERK signaling more potently in BRAFV600E, than in KRAS mutant tumors.  To understand this, we performed an RNAi screen in a KRAS mutant model and found that CRAF knockdown enhanced MEK inhibition.  MEK activated by CRAF was less susceptible to MEK inhibitors than when activated by BRAFV600E.  MEK inhibitors induced RAF-MEK complexes in KRAS mutant models, and disrupting such complexes enhanced inhibition of CRAF-dependent ERK signaling.  Newer MEK inhibitors target MEK catalytic activity and also impair its reactivation by CRAF, either by disrupting RAF-MEK complexes or by interacting with Ser 222 to prevent MEK phosphorylation by RAF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrm_fqelH7I0bVg90H21EOLACvtfcHk0liyZXFWJX7I4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVCht7c%253D&md5=1299892e2941b79fa511a47347557c6e</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DLito%26aufirst%3DP.%26aulast%3DSaborowski%26aufirst%3DA.%26aulast%3DYue%26aufirst%3DJ.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DJoseph%26aufirst%3DE.%26aulast%3DGadal%26aufirst%3DS.%26aulast%3DSaborowski%26aufirst%3DM.%26aulast%3DKastenhuber%26aufirst%3DE.%26aulast%3DFellmann%26aufirst%3DC.%26aulast%3DOhara%26aufirst%3DK.%26aulast%3DMorikami%26aufirst%3DK.%26aulast%3DMiura%26aufirst%3DT.%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DIshii%26aufirst%3DN.%26aulast%3DLowe%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DDisruption%2520of%2520CRAF-mediated%2520MEK%2520activation%2520is%2520required%2520for%2520effective%2520MEK%2520inhibition%2520in%2520KRAS%2520mutant%2520tumors%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D697%26epage%3D710%26doi%3D10.1016%2Fj.ccr.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Little, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. J.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of acquired resistance to ERK1/2 pathway inhibitors</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1207</span>– <span class="NLM_lpage">1215</span>, <span class="refDoi"> DOI: 10.1038/onc.2012.160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fonc.2012.160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=22562245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsFOqt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=1207-1215&author=A.+S.+Littleauthor=P.+D.+Smithauthor=S.+J.+Cook&title=Mechanisms+of+acquired+resistance+to+ERK1%2F2+pathway+inhibitors&doi=10.1038%2Fonc.2012.160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of acquired resistance to ERK1/2 pathway inhibitors</span></div><div class="casAuthors">Little, A. S.; Smith, P. D.; Cook, S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1207-1215</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The ERK1/2 (extracellular signal-regulated kinase 1 and 2) pathway, comprising the protein kinases RAF (v-raf-1 murine leukemia viral oncogene homolog 1), MEK1/2 (mitogen-activated protein kinase or ERK kinase 1 and 2) and ERK1/2 is frequently de-regulated in human cancers, due to mutations in RAS or BRAF (v-raf-1 murine leukemia viral oncogene homolog B1).  New, highly selective inhibitors of BRAF and MEK1/2 have shown promise in clin. trials, including in previously intractable diseases such as melanoma.  However, drug-resistant tumor cells invariably emerge leading to disease progression.  It is important to understand the mechanisms underlying such acquired resistance since this may lead to the development of rational strategies either to delay its onset or to overcome it once established.  It also offers unique insights into the plasticity of signalling pathways, which may in turn inform our understanding of the basic biol. of these pathways and lead to the validation of new drug targets.  Several recent reports have identified diverse mechanisms of acquired resistance to MEK1/2 or BRAF inhibitors.  In this article, we review these studies, discuss the different mechanisms, identify common themes and consider their therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPrc4YOQY4zrVg90H21EOLACvtfcHk0lhXKGsbXMTaDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsFOqt7Y%253D&md5=0d604795d8d70e7725d7d6d6cc1d9edb</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fonc.2012.160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2012.160%26sid%3Dliteratum%253Aachs%26aulast%3DLittle%26aufirst%3DA.%2BS.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26atitle%3DMechanisms%2520of%2520acquired%2520resistance%2520to%2520ERK1%252F2%2520pathway%2520inhibitors%26jtitle%3DOncogene%26date%3D2013%26volume%3D32%26spage%3D1207%26epage%3D1215%26doi%3D10.1038%2Fonc.2012.160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Little, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balmanno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hampson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. J.</span></span> <span> </span><span class="NLM_article-title">Amplification of the driving oncogene, KRAS or BRAF, underpins acquired resistance to MEK1/2 inhibitors in colorectal cancer cells</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1126/scisignal.2001752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1126%2Fscisignal.2001752" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=1-16&author=A.+S.+Littleauthor=K.+Balmannoauthor=M.+J.+Saleauthor=S.+Newmanauthor=J.+R.+Dryauthor=M.+Hampsonauthor=P.+A.+Edwardsauthor=P.+D.+Smithauthor=S.+J.+Cook&title=Amplification+of+the+driving+oncogene%2C+KRAS+or+BRAF%2C+underpins+acquired+resistance+to+MEK1%2F2+inhibitors+in+colorectal+cancer+cells&doi=10.1126%2Fscisignal.2001752"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2001752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2001752%26sid%3Dliteratum%253Aachs%26aulast%3DLittle%26aufirst%3DA.%2BS.%26aulast%3DBalmanno%26aufirst%3DK.%26aulast%3DSale%26aufirst%3DM.%2BJ.%26aulast%3DNewman%26aufirst%3DS.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DHampson%26aufirst%3DM.%26aulast%3DEdwards%26aufirst%3DP.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26atitle%3DAmplification%2520of%2520the%2520driving%2520oncogene%252C%2520KRAS%2520or%2520BRAF%252C%2520underpins%2520acquired%2520resistance%2520to%2520MEK1%252F2%2520inhibitors%2520in%2520colorectal%2520cancer%2520cells%26jtitle%3DSci.%2520Signaling%26date%3D2011%26volume%3D4%26spage%3D1%26epage%3D16%26doi%3D10.1126%2Fscisignal.2001752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welsh, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizos, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span> <span> </span><span class="NLM_article-title">Resistance to combination BRAF and MEK inhibition in metastatic melanoma: where to next?</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/j.ejca.2016.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ejca.2016.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=27232329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsVWqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2016&pages=76-85&author=S.+J.+Welshauthor=H.+Rizosauthor=R.+A.+Scolyerauthor=G.+V.+Long&title=Resistance+to+combination+BRAF+and+MEK+inhibition+in+metastatic+melanoma%3A+where+to+next%3F&doi=10.1016%2Fj.ejca.2016.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to combination BRAF and MEK inhibition in metastatic melanoma: Where to next?</span></div><div class="casAuthors">Welsh, Sarah J.; Rizos, Helen; Scolyer, Richard A.; Long, Georgina V.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76-85</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Treatment of BRAF-mutant metastatic melanoma with mitogen-activated protein kinase (MAPK) pathway targeted therapies (BRAF/MEK inhibitors) and immune checkpoint inhibitors has revolutionized management and improved outcomes for patients with advanced stage disease.  However, acquired resistance to MAPK inhibitor therapy develops in the majority of patients at approx. 12 mo and multiple mechanisms lead to resistance.  Understanding the mechanisms of resistance is therefore crit. for the development of more effective therapeutic strategies in BRAF-mutant melanoma.  Recently, several distinct mechanisms of resistance to BRAF-inhibition have been proposed based on data obtained in exptl. melanoma cell models and small series of human tumor samples.  These include reactivation of the MAPK pathway resulting in continued extracellular signal-regulated kinase activation and activation of parallel signalling pathways including the PI3K-mTOR (phosphoinositide 3-kinase-mammalian target of rapamycin) pathway.  Alterations in how the cells of the immune system respond to melanoma cells treated with targeted therapy may also influence response and progression.  In this review, we discuss these mechanisms and identify potential therapeutic strategies to overcome resistance which, in turn, will lead to improved outcomes for patients with metastatic melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl-VbAROwGALVg90H21EOLACvtfcHk0lhXKGsbXMTaDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsVWqsro%253D&md5=34774092eb983ec5c48c247fd49c6887</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2016.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2016.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWelsh%26aufirst%3DS.%2BJ.%26aulast%3DRizos%26aufirst%3DH.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DLong%26aufirst%3DG.%2BV.%26atitle%3DResistance%2520to%2520combination%2520BRAF%2520and%2520MEK%2520inhibition%2520in%2520metastatic%2520melanoma%253A%2520where%2520to%2520next%253F%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2016%26volume%3D62%26spage%3D76%26epage%3D85%26doi%3D10.1016%2Fj.ejca.2016.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with <i>in vitro</i> nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lhXKGsbXMTaDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0ljGo_TyCaanOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franken, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodermond, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stap, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haveman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Bree, C.</span></span> <span> </span><span class="NLM_article-title">Clonogenic assay of cells <i>in vitro</i></span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">2315</span>– <span class="NLM_lpage">2319</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnprot.2006.339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=17406473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGjtLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=2315-2319&author=N.+A.+Frankenauthor=H.+M.+Rodermondauthor=J.+Stapauthor=J.+Havemanauthor=C.+van+Bree&title=Clonogenic+assay+of+cells+in+vitro&doi=10.1038%2Fnprot.2006.339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Clonogenic assay of cells in vitro</span></div><div class="casAuthors">Franken, Nicolaas A. P.; Rodermond, Hans M.; Stap, Jan; Haveman, Jaap; van Bree, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2315-2319</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clonogenic assay or colony formation assay is an in vitro cell survival assay based on the ability of a single cell to grow into a colony.  The colony is defined to consist of at least 50 cells.  The assay essentially tests every cell in the population for its ability to undergo "unlimited" division.  Clonogenic assay is the method of choice to det. cell reproductive death after treatment with ionizing radiation, but can also be used to det. the effectiveness of other cytotoxic agents.  Only a fraction of seeded cells retains the capacity to produce colonies.  Before or after treatment, cells are seeded out in appropriate dilns. to form colonies in 1-3 wk.  Colonies are fixed with glutaraldehyde (6.0% vol./vol.), stained with crystal violet (0.5% w/v) and counted using a stereomicroscope.  A method for the anal. of radiation dose-survival curves is included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGove0egxayKRrVg90H21EOLACvtfcHk0ljGo_TyCaanOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGjtLnI&md5=5edcb3202aa9088b16d27608ae00929b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.339%26sid%3Dliteratum%253Aachs%26aulast%3DFranken%26aufirst%3DN.%2BA.%26aulast%3DRodermond%26aufirst%3DH.%2BM.%26aulast%3DStap%26aufirst%3DJ.%26aulast%3DHaveman%26aufirst%3DJ.%26aulast%3Dvan%2BBree%26aufirst%3DC.%26atitle%3DClonogenic%2520assay%2520of%2520cells%2520in%2520vitro%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D2315%26epage%3D2319%26doi%3D10.1038%2Fnprot.2006.339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykema, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherba, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furge, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duesbery, N. S.</span></span> <span> </span><span class="NLM_article-title">MEK2 is sufficient but not necessary for proliferation and anchorage-independent growth of SK-MEL-28 melanoma cells</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0017165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1371%2Fjournal.pone.0017165" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1-15&author=C.+S.+Leeauthor=K.+J.+Dykemaauthor=D.+M.+Hawkinsauthor=D.+M.+Cherbaauthor=C.+P.+Webbauthor=K.+A.+Furgeauthor=N.+S.+Duesbery&title=MEK2+is+sufficient+but+not+necessary+for+proliferation+and+anchorage-independent+growth+of+SK-MEL-28+melanoma+cells&doi=10.1371%2Fjournal.pone.0017165"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0017165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0017165%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BS.%26aulast%3DDykema%26aufirst%3DK.%2BJ.%26aulast%3DHawkins%26aufirst%3DD.%2BM.%26aulast%3DCherba%26aufirst%3DD.%2BM.%26aulast%3DWebb%26aufirst%3DC.%2BP.%26aulast%3DFurge%26aufirst%3DK.%2BA.%26aulast%3DDuesbery%26aufirst%3DN.%2BS.%26atitle%3DMEK2%2520is%2520sufficient%2520but%2520not%2520necessary%2520for%2520proliferation%2520and%2520anchorage-independent%2520growth%2520of%2520SK-MEL-28%2520melanoma%2520cells%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3D1%26epage%3D15%26doi%3D10.1371%2Fjournal.pone.0017165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jing Liu, He Chen, Yi Liu, Yudao Shen, Fanye Meng, H. Ümit Kaniskan, Jian Jin, <span class="NLM_string-name hlFld-ContribAuthor">Wenyi Wei</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer Selective Target Degradation by Folate-Caged PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (19)
                                     , 7380-7387. <a href="https://doi.org/10.1021/jacs.1c00451" title="DOI URL">https://doi.org/10.1021/jacs.1c00451</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.1c00451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.1c00451%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DCancer%252BSelective%252BTarget%252BDegradation%252Bby%252BFolate-Caged%252BPROTACs%26aulast%3DLiu%26aufirst%3DJing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D13012021%26date%3D10052021%26volume%3D143%26issue%3D19%26spage%3D7380%26epage%3D7387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jianping Hu, Jieli Wei, Hyerin Yim, Li Wang, Ling Xie, Margaret S. Jin, Md Kabir, Lihuai Qin, Xian Chen, Jing Liu, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Potent and Selective Mitogen-Activated Protein Kinase Kinase 1/2 (MEK1/2) Heterobifunctional Small-molecule Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15883-15905. <a href="https://doi.org/10.1021/acs.jmedchem.0c01609" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01609</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01609%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPotent%252Band%252BSelective%252BMitogen-Activated%252BProtein%252BKinase%252BKinase%252B1%25252F2%252B%252528MEK1%25252F2%252529%252BHeterobifunctional%252BSmall-molecule%252BDegraders%26aulast%3DHu%26aufirst%3DJianping%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14092020%26date%3D07122020%26volume%3D63%26issue%3D24%26spage%3D15883%26epage%3D15905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Liqun Chen, Yanke Chen, Chunyan Zhang, Bingyang Jiao, Sheng Liang, Qiong Tan, Hongyu Chai, Weihua Yu, Yongzheng Qian, Hui Yang, Wuyi Yao, Jianguo Yu, Ying Luo, Michael Plewe, Jialiang Wang, Xiao-Ran Han, <span class="NLM_string-name hlFld-ContribAuthor">Jing Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of First-In-Class Potent and Selective Tropomyosin Receptor Kinase Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14562-14575. <a href="https://doi.org/10.1021/acs.jmedchem.0c01342" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01342</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01342%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BFirst-In-Class%252BPotent%252Band%252BSelective%252BTropomyosin%252BReceptor%252BKinase%252BDegraders%26aulast%3DChen%26aufirst%3DLiqun%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07082020%26date%3D15102020%26volume%3D63%26issue%3D23%26spage%3D14562%26epage%3D14575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Guozhen Gao, Xufen Yu, Huensuk Kim, Li Wang, Ling Xie, Megan Schwarz, Xian Chen, Ernesto Guccione, Jing Liu, Mark T. Bedford, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9977-9989. <a href="https://doi.org/10.1021/acs.jmedchem.0c01111" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01111</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01111%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BFirst-in-Class%252BProtein%252BArginine%252BMethyltransferase%252B5%252B%252528PRMT5%252529%252BDegraders%26aulast%3DShen%26aufirst%3DYudao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29062020%26date%3D21082020%26date%3D11082020%26volume%3D63%26issue%3D17%26spage%3D9977%26epage%3D9989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mingliang Wang, Jianfeng Lu, Mi Wang, Chao-Yie Yang, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of SHP2-D26 as a First, Potent, and Effective PROTAC Degrader of SHP2 Protein. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (14)
                                     , 7510-7528. <a href="https://doi.org/10.1021/acs.jmedchem.0c00471" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00471</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00471%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSHP2-D26%252Bas%252Ba%252BFirst%25252C%252BPotent%25252C%252Band%252BEffective%252BPROTAC%252BDegrader%252Bof%252BSHP2%252BProtein%26aulast%3DWang%26aufirst%3DMingliang%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20032020%26date%3D21052020%26volume%3D63%26issue%3D14%26spage%3D7510%26epage%3D7528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Yunhua  Peng</span>, <span class="hlFld-ContribAuthor ">Wenyi  Wei</span>. </span><span class="cited-content_cbyCitation_article-title">Light-Controllable PROTACs for Temporospatial Control of Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.678077" title="DOI URL">https://doi.org/10.3389/fcell.2021.678077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.678077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.678077%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DLight-Controllable%252BPROTACs%252Bfor%252BTemporospatial%252BControl%252Bof%252BProtein%252BDegradation%26aulast%3DLiu%26aufirst%3DJing%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Yu</span>, <span class="hlFld-ContribAuthor ">Ming  Cai</span>, <span class="hlFld-ContribAuthor ">Liang  Shao</span>, <span class="hlFld-ContribAuthor ">Jihong  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Protein Kinases Degradation by PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.679120" title="DOI URL">https://doi.org/10.3389/fchem.2021.679120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.679120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.679120%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DTargeting%252BProtein%252BKinases%252BDegradation%252Bby%252BPROTACs%26aulast%3DYu%26aufirst%3DFei%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandon  Dale</span>, <span class="hlFld-ContribAuthor ">Meng  Cheng</span>, <span class="hlFld-ContribAuthor ">Kwang-Su  Park</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Yue  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Advancing targeted protein degradation for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2021,</strong> <em>170 </em><a href="https://doi.org/10.1038/s41568-021-00365-x" title="DOI URL">https://doi.org/10.1038/s41568-021-00365-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41568-021-00365-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41568-021-00365-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DAdvancing%252Btargeted%252Bprotein%252Bdegradation%252Bfor%252Bcancer%252Btherapy%26aulast%3DDale%26aufirst%3DBrandon%26date%3D2021%26date%3D2021%26volume%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao  Wang</span>, <span class="hlFld-ContribAuthor ">Han  Wang</span>, <span class="hlFld-ContribAuthor ">Cangxin  Zheng</span>, <span class="hlFld-ContribAuthor ">Zhenming  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaozuo  Gao</span>, <span class="hlFld-ContribAuthor ">Fengrong  Xu</span>, <span class="hlFld-ContribAuthor ">Yan  Niu</span>, <span class="hlFld-ContribAuthor ">Liangren  Zhang</span>, <span class="hlFld-ContribAuthor ">Ping  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Research progress of MEK1/2 inhibitors and degraders in the treatment of cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>218 </em>, 113386. <a href="https://doi.org/10.1016/j.ejmech.2021.113386" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113386</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113386%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DResearch%252Bprogress%252Bof%252BMEK1%25252F2%252Binhibitors%252Band%252Bdegraders%252Bin%252Bthe%252Btreatment%252Bof%252Bcancer%26aulast%3DWang%26aufirst%3DChao%26date%3D2021%26volume%3D218%26spage%3D113386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyan cheng</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (PROTAC) in drug discovery paradigm: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112981. <a href="https://doi.org/10.1016/j.ejmech.2020.112981" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112981%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bin%252Bdrug%252Bdiscovery%252Bparadigm%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26volume%3D210%26spage%3D112981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuhua  Zheng</span>, <span class="hlFld-ContribAuthor ">Wensi  Tao</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Cullin-RING E3 Ligases for Radiosensitization: From NEDDylation Inhibition to PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2020,</strong> <em>10 </em><a href="https://doi.org/10.3389/fonc.2020.01517" title="DOI URL">https://doi.org/10.3389/fonc.2020.01517</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2020.01517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2020.01517%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DTargeting%252BCullin-RING%252BE3%252BLigases%252Bfor%252BRadiosensitization%25253A%252BFrom%252BNEDDylation%252BInhibition%252Bto%252BPROTACs%26aulast%3DZheng%26aufirst%3DShuhua%26date%3D2020%26date%3D2020%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative MEK1/2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Crystal structure of the ternary complex of compound <b>5</b> (cyan), ATP (green), and MEK1 (gray) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9J">1S9J</a>). Key H-bonds formed by <b>5</b> are illustrated by yellow dotted lines. The flexible glycine rich P-loop is illustrated in blue. The unstructured Asn221, Ser222, and Phe223 were modeled according to PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP1">3PP1</a> using Coot. (B) Structure of the dual PI3K-MEK1 inhibitor <b>6</b>. (C) Structures of compounds <b>7</b>–<b>11</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0012.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes for Preparing Compounds <b>7</b>–<b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DBU, DMF, 50 °C, 4 h; (b) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, MeOH, DCM, rt, 2 h; (c) HOAt, EDCI, NMM, DMSO, rt, overnight; (d) 3 M HCl, THF, rt, 6 h; (e) (1) HOAt, EDCI, NMM, DMSO, rt, overnight; (2) TFA, DCM, rt, 2 h; (f) NaBH<sub>3</sub>CN, DCM, MeOH, rt, overnight.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of compounds <b>7</b>–<b>11</b> on MEK1/2 degradation in HT-29 cells. Cells were treated with DMSO or serial dilution of indicated compounds for 24 h. Compound concentrations are 0.3, 1, and 3 μM. MEK1/2 protein levels were determined by Western blots and normalized with α-tubulin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structures of compounds <b>23</b> and <b>24</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0013.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route for Preparing Compounds <b>23</b> and <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) HOAt, EDCI, NMM, DMSO, rt, overnight; (2) TFA, DCM, rt, 2 h; (b) NaBH<sub>3</sub>CN, DCM, MeOH, rt, overnight.</p></p></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compounds <b>23</b> and <b>24</b> inhibit the kinase activity of MEK1 and MEK2 in vitro. Inhibition value for each concentration point was determined by measuring MEK1/2 kinase activity on ERK phosphorylation and normalized with DMSO control point. IC<sub>50</sub> values of PD0325901 (A), compound <b>23</b> (B), and compound <b>24</b> (C) against MEK1 and MEK2 were analyzed in GraphPad Prism 8. For each concentration point, mean value ± SD from two independent experiments is shown in the curves.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>23</b> reduces MEK1/2 protein levels and inhibits the downstream ERK signaling in a concentration-dependent manner. HT-29 (A) and SK-MEL-28 (B) cells were treated with DMSO or serial dilution of compound <b>23</b> for 24 h. The indicated protein levels were determined by Western blots and normalized with α-tubulin. DC<sub>50</sub> values of compound <b>23</b> for MEK1 and MEK2 in HT-29 (C) and SK-MEL-28 (D) cells were analyzed in GraphPad Prism 8. For each concentration point, mean value ± SD from three independent experiments is shown in the curves. The effects of PD0325901 and compounds <b>23</b> and <b>24</b> on reducing MEK1/2 protein levels and inhibiting the downstream ERK signaling were assessed in HT-29 (E) and SK-MEL-28 (F) cells treated with DMSO or different concentrations of indicated compounds for 24 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>23</b> induces degradation of MEK1 and MEK2 and suppresses the downstream ERK signaling in a time-dependent manner. HT-29 (A) and SK-MEL-28 (B) cells were treated with 0.1 or 0.3 μM compound <b>23</b> for 24 h. Cells were harvested at indicated time points within 24 h, and the indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compound <b>23</b> induces MEK1/2 protein degradation through recruiting VHL E3 ubiquitin ligase and proteasome-dependent proteolysis. HT-29 and SK-MEL-28 cells were pretreated with DMSO, PD0325901 (1 μM), MG-132 (3 μM), MLN4924 (1 μM), or VH 032 (10 μM) for 2 h, followed by 10 h treatment with 0.3 μM compound <b>23</b> for rescue experiments. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0009.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Global proteome analyses of HT-29 cells treated with compound <b>23</b>, compound <b>24</b>, or DMSO control. Cells were harvested for proteome analysis after 10 h treatment with 0.1% DMSO, 0.1 μM compound <b>23</b>, or 0.1 μM compound <b>24</b>. Volcano plots of the −log 10 (<i>P</i>-value) versus the log 2 ratio are displayed, with proteins outside the significance lines colored in pink or purple (FDR < 0.05). The <i>P</i>-values were calculated from the data of two technical replicates. (A) DMSO as compared to compound <b>23</b> treated samples; (B) compound <b>24</b> as compared to compound <b>23</b> treated samples; and (C) DMSO as compared to compound <b>24</b> treated samples.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0010.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compound <b>23</b> inhibits cell proliferation of HT-29 and SK-MEL-28 cells, phenocopied by MEK1/2 knockdown. MTT assay was conducted for HT-29 (A) and SK-MEL-28 (B) cells after 3 d treatment with DMSO or indicated compounds with serial dilutions. Data were analyzed in GraphPad Prism 8. For each concentration point, in duplicate, mean value ± SD from four independent experiments is shown in the curves. Clonogenic assay was performed in HT-29 (C) and SK-MEL-28 (D) cells for 14 d treatment with DMSO or indicated compounds with 10, 30, and 100 nM, in duplicate, respectively. Cells were then stained by crystal violet. The images here are representatives of three independent experiments. The shRNA-mediated MEK1/2 knockdown experiments were conducted in HT-29 (E) and SK-MEL-28 (F) cells. MTT assay was used to determine cell viability to assess the effect of MEK1/2 knockdown via shRNAs. For HT-29 and SK-MEL-28 cells, three and two independent experiments were performed for each treatment, respectively. Data were analyzed in GraphPad Prism 8 with two-way ANOVA to get <i>P</i>-values. For each time point, nine replicates, mean value ± SD from the independent experiments is shown in the plots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/medium/jm9b01528_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0011.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Plasma concentration of compound <b>23</b> following a single intraperitoneal administration for 8 h in male Swiss Albino mice. Data were analyzed in GraphPad Prism 8. For each time point, mean value ± SD from three mice is shown in the plot.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01528/20191205/images/large/jm9b01528_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01528&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05112" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05112" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">MEK1/2 dual-specificity protein kinases: structure and regulation</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>417</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2011.11.145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.bbrc.2011.11.145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=22177953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1Oqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2012&pages=5-10&author=R.+Roskoski&title=MEK1%2F2+dual-specificity+protein+kinases%3A+structure+and+regulation&doi=10.1016%2Fj.bbrc.2011.11.145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1/2 dual-specificity protein kinases: Structure and regulation</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">417</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  MEK1 and MEK2 are related protein kinases that participate in the RAS-RAF-MEK-ERK signal transduction cascade.  This cascade participates in the regulation of a large variety of processes including apoptosis, cell cycle progression, cell migration, differentiation, metab., and proliferation.  Moreover, oncogenic mutations in RAS or B-RAF are responsible for a large proportion of human cancers.  MEK1 is activated by phosphorylation of Ser-218 and Ser-222 in its activation segment as catalyzed by RAF kinases in an intricate process that involves a KSR scaffold.  In addn. to functioning as a scaffold, the kinase activity of KSR is also required for MEK activation.  MEK1 regulation is unusual in that Ser-212 phosphorylation in its activation segment is inhibitory.  Moreover, active ERK kinase catalyzes a feedback inhibitory phosphorylation of MEK1 Thr-292 that serves to down-regulate the pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRX9pGR2iVKbVg90H21EOLACvtfcHk0lj_SREKf6WqaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1Oqtg%253D%253D&md5=4bdfbf6ebaa90055a314922fa1056929</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2011.11.145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2011.11.145%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DMEK1%252F2%2520dual-specificity%2520protein%2520kinases%253A%2520structure%2520and%2520regulation%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2012%26volume%3D417%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.bbrc.2011.11.145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scholl, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumesic, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barragan, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bissonauth, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khavari, P. A.</span></span> <span> </span><span class="NLM_article-title">Mek1/2 MAPK kinases are essential for mammalian development, homeostasis, and Raf-induced hyperplasia</span>. <i>Dev. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">629</span>, <span class="refDoi"> DOI: 10.1016/j.devcel.2007.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.devcel.2007.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=17419998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVOhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=615-629&author=F.+A.+Schollauthor=P.+A.+Dumesicauthor=D.+I.+Barraganauthor=K.+Haradaauthor=V.+Bissonauthauthor=J.+Charronauthor=P.+A.+Khavari&title=Mek1%2F2+MAPK+kinases+are+essential+for+mammalian+development%2C+homeostasis%2C+and+Raf-induced+hyperplasia&doi=10.1016%2Fj.devcel.2007.03.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mek1/2 MAPK kinases are essential for mammalian development, homeostasis, and Raf-induced hyperplasia</span></div><div class="casAuthors">Scholl, Florence A.; Dumesic, Phillip A.; Barragan, Deborah I.; Harada, Kazutoshi; Bissonauth, Vickram; Charron, Jean; Khavari, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">615-629</span>CODEN:
                <span class="NLM_cas:coden">DCEEBE</span>;
        ISSN:<span class="NLM_cas:issn">1534-5807</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The p42/p44 mitogen-activated protein kinase (MAPK) cascade includes Ras, Raf, Mek, and Erk MAPK.  To det. the effect of a full knockout at a single level of this signaling pathway in mammals, and to investigate functional redundancy between Mek1 and Mek2, we disrupted these genes in murine and human epidermis.  Loss of either protein alone produced no phenotype, whereas combined Mek1/2 deletion in development or adulthood abolished Erk1/2 phosphorylation and led to hypoproliferation, apoptosis, skin barrier defects, and death.  Conversely, a single copy of either allele was sufficient for normal development.  Combined Mek1/2 loss also abolished Raf-induced hyperproliferation.  Human tissue deficient in either Mek isoform was normal, whereas loss of both proteins led to hypoplasia, which was rescued by active Erk2 expression.  These data indicate that Mek1/2 are functionally redundant in the epidermis, where they act as a linear relay in the MAPK pathway to mediate development and homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogiBLCMiwCarVg90H21EOLACvtfcHk0lj_SREKf6WqaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVOhsLY%253D&md5=739b373ceee916e03504d56745f87444</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2007.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2007.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DScholl%26aufirst%3DF.%2BA.%26aulast%3DDumesic%26aufirst%3DP.%2BA.%26aulast%3DBarragan%26aufirst%3DD.%2BI.%26aulast%3DHarada%26aufirst%3DK.%26aulast%3DBissonauth%26aufirst%3DV.%26aulast%3DCharron%26aufirst%3DJ.%26aulast%3DKhavari%26aufirst%3DP.%2BA.%26atitle%3DMek1%252F2%2520MAPK%2520kinases%2520are%2520essential%2520for%2520mammalian%2520development%252C%2520homeostasis%252C%2520and%2520Raf-induced%2520hyperplasia%26jtitle%3DDev.%2520Cell%26date%3D2007%26volume%3D12%26spage%3D615%26epage%3D629%26doi%3D10.1016%2Fj.devcel.2007.03.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. T.</span></span> <span> </span><span class="NLM_article-title">MAPK signal pathways in the regulation of cell proliferation in mammalian cells</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1038/sj.cr.7290105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fsj.cr.7290105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=11942415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BD383gvV2qsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=9-18&author=W.+Zhangauthor=H.+T.+Liu&title=MAPK+signal+pathways+in+the+regulation+of+cell+proliferation+in+mammalian+cells&doi=10.1038%2Fsj.cr.7290105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">MAPK signal pathways in the regulation of cell proliferation in mammalian cells</span></div><div class="casAuthors">Zhang Wei; Liu Hui Tu</div><div class="citationInfo"><span class="NLM_cas:title">Cell research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-18</span>
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    </div><div class="casAbstract">MAPK families play an important role in complex cellular programs like proliferation, differentiation, development, transformation, and apoptosis.  At least three MAPK families have been characterized: extracellular signal-regulated kinase (ERK), Jun kinase (JNK/SAPK) and p38 MAPK.  The above effects are fulfilled by regulation of cell cycle engine and other cell proliferation related proteins.  In this paper we discussed their functions and cooperation with other signal pathways in regulation of cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQjYqac378pM4HWj6b1v8uTfW6udTcc2ebzfBgByELo6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD383gvV2qsA%253D%253D&md5=a466a156756e0afbbd63fbd7f9ec88e8</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.cr.7290105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.cr.7290105%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BT.%26atitle%3DMAPK%2520signal%2520pathways%2520in%2520the%2520regulation%2520of%2520cell%2520proliferation%2520in%2520mammalian%2520cells%26jtitle%3DCell%2520Res.%26date%3D2002%26volume%3D12%26spage%3D9%26epage%3D18%26doi%3D10.1038%2Fsj.cr.7290105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caunt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. J.</span></span> <span> </span><span class="NLM_article-title">MEK1 and MEK2 inhibitors and cancer therapy: the long and winding road</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1038/nrc4000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnrc4000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26399658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGhsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=577-592&author=C.+J.+Cauntauthor=M.+J.+Saleauthor=P.+D.+Smithauthor=S.+J.+Cook&title=MEK1+and+MEK2+inhibitors+and+cancer+therapy%3A+the+long+and+winding+road&doi=10.1038%2Fnrc4000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1 and MEK2 inhibitors and cancer therapy: the long and winding road</span></div><div class="casAuthors">Caunt, Christopher J.; Sale, Matthew J.; Smith, Paul D.; Cook, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">577-592</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The role of the ERK signaling pathway in cancer is thought to be most prominent in tumors in which mutations in the receptor tyrosine kinases RAS, BRAF, CRAF, MEK1 or MEK2 drive growth factor-independent ERK1 and ERK2 activation and thence inappropriate cell proliferation and survival.  New drugs that inhibit RAF or MEK1 and MEK2 have recently been approved or are currently undergoing late-stage clin. evaluation.  In this Review, the authors consider the ERK pathway, focusing particularly on the role of MEK1 and MEK2, the 'gatekeepers' of ERK1/2 activity.  The authors discuss their validation as drug targets, the merits of targeting MEK1 and MEK2 vs. BRAF and the mechanisms of action of different inhibitors of MEK1 and MEK2.  The authors also consider how some of the systems-level properties (intrapathway regulatory loops and wider signaling network connections) of the ERK pathway present a challenge for the success of MEK1 and MEK2 inhibitors, discuss mechanisms of resistance to these inhibitors, and review their clin. progress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnTcSz8CvRcLVg90H21EOLACvtfcHk0lihd8hd3FPLag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGhsb%252FO&md5=b1fd44051024a59608dd61310a437c65</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc4000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc4000%26sid%3Dliteratum%253Aachs%26aulast%3DCaunt%26aufirst%3DC.%2BJ.%26aulast%3DSale%26aufirst%3DM.%2BJ.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26atitle%3DMEK1%2520and%2520MEK2%2520inhibitors%2520and%2520cancer%2520therapy%253A%2520the%2520long%2520and%2520winding%2520road%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D577%26epage%3D592%26doi%3D10.1038%2Fnrc4000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoshino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuruo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ari-i, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, M.</span></span> <span> </span><span class="NLM_article-title">Constitutive activation of the 41-/43-kDa mitogen-activated protein kinase signaling pathway in human tumors</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">813</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1202367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fsj.onc.1202367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=9989833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADyaK1MXht1eht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=813-822&author=R.+Hoshinoauthor=Y.+Chataniauthor=T.+Yamoriauthor=T.+Tsuruoauthor=H.+Okaauthor=O.+Yoshidaauthor=Y.+Shimadaauthor=S.+Ari-iauthor=H.+Wadaauthor=J.+Fujimotoauthor=M.+Kohno&title=Constitutive+activation+of+the+41-%2F43-kDa+mitogen-activated+protein+kinase+signaling+pathway+in+human+tumors&doi=10.1038%2Fsj.onc.1202367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Constitutive activation of the 41-/43-kDa mitogen-activated protein kinase signaling pathway in human tumors</span></div><div class="casAuthors">Hoshino, Rika; Chatani, Yuji; Yamori, Takao; Tsuruo, Takashi; Oka, Hiroya; Yoshida, Osamu; Shimada, Yutaka; Ari-i, Shigeki; Wada, Hiromi; Fujimoto, Jiro; Kohno, Michiaki</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">813-822</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">The 41-kDa and 43-kDa mitogen-activated protein (MAP) kinases play a pivotal role in the mitogenic signal transduction pathway and are essential components of the MAP kinase cascade, which includes MAP kinase kinase (MEK) and Raf-1.  As aberrant activation of signal transducing mols. such as Ras and Raf-1 has been linked with cancer, the authors examd. whether constitutive activation of the 41-/43-kDa MAP kinases is assocd. with the neoplastic phenotype of 138 tumor cell lines and 102 primary tumors derived from various human organs.  Constitutive activation of the MAP kinases was obsd. in 50 tumor cell lines (36.2%) in a rather tissue-specific manner: cell lines derived from pancreas, colon, lung, ovary and kidney showed esp. high frequencies with a high degree of MAP kinase activation, while those derived from brain, esophagus, stomach, liver and of hematopoietic origin showed low frequencies with a limited degree of MAP kinase activation.  The authors also detected constitutive activation of the 41-/43-kDa MAP kinases in a relatively large no. of primary human tumors derived from kidney, colon and lung tissues but not from liver tissue.  Many tumor cells, in which point mutations of ras genes were detected, showed constitutive activation of MAP kinases, however, there were also many exceptions to this observation.  In contrast, the activation of the 41-/43-kDa MAP kinases was accompanied by the activation of Raf-1 in the majority of tumor cells and was completely assocd. with the activation of MEK and p90rsk in all the tumor cells examd.  These results suggest that the constitutive activation of 41-/43-kDa MAP kinases in tumor cells is not due to the disorder of MAP kinases themselves, but is due to the disorder of Raf-1, Ras, or some other signaling mols. upstream of Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRICRJXnIoerVg90H21EOLACvtfcHk0lihd8hd3FPLag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXht1eht70%253D&md5=719a702e589f259a22f53f7a2be0e94a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202367%26sid%3Dliteratum%253Aachs%26aulast%3DHoshino%26aufirst%3DR.%26aulast%3DChatani%26aufirst%3DY.%26aulast%3DYamori%26aufirst%3DT.%26aulast%3DTsuruo%26aufirst%3DT.%26aulast%3DOka%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DO.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DAri-i%26aufirst%3DS.%26aulast%3DWada%26aufirst%3DH.%26aulast%3DFujimoto%26aufirst%3DJ.%26aulast%3DKohno%26aufirst%3DM.%26atitle%3DConstitutive%2520activation%2520of%2520the%252041-%252F43-kDa%2520mitogen-activated%2520protein%2520kinase%2520signaling%2520pathway%2520in%2520human%2520tumors%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D813%26epage%3D822%26doi%3D10.1038%2Fsj.onc.1202367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovly, C. M.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of receptor tyrosine kinase activation in cancer</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1186/s12943-018-0782-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1186%2Fs12943-018-0782-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29304823" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1-13&author=Z.+Duauthor=C.+M.+Lovly&title=Mechanisms+of+receptor+tyrosine+kinase+activation+in+cancer&doi=10.1186%2Fs12943-018-0782-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0782-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0782-4%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DLovly%26aufirst%3DC.%2BM.%26atitle%3DMechanisms%2520of%2520receptor%2520tyrosine%2520kinase%2520activation%2520in%2520cancer%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D1%26epage%3D13%26doi%3D10.1186%2Fs12943-018-0782-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Medarde, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, E.</span></span> <span> </span><span class="NLM_article-title">Ras in cancer and developmental diseases</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">344</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1177/1947601911411084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1177%2F1947601911411084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=21779504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWlsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=344-358&author=A.+Fernandez-Medardeauthor=E.+Santos&title=Ras+in+cancer+and+developmental+diseases&doi=10.1177%2F1947601911411084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Ras in cancer and developmental diseases</span></div><div class="casAuthors">Fernandez-Medarde, Alberto; Santos, Eugenio</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">344-358</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Somatic, gain-of-function mutations in ras genes were the first specific genetic alterations identified in human cancer about 3 decades ago.  Studies during the last quarter century have characterized the Ras proteins as essential components of signaling networks controlling cellular proliferation, differentiation, or survival.  The oncogenic mutations of the H-ras, N-ras, or K-ras genes frequently found in human tumors are known to throw off balance the normal outcome of those signaling pathways, thus leading to tumor development.  Oncogenic mutations in a no. of other upstream or downstream components of Ras signaling pathways (including membrane RTKs or cytosolic kinases) have been detected more recently in assocn. with a variety of cancers.  Interestingly, the oncogenic Ras mutations and the mutations in other components of Ras/MAPK signaling pathways appear to be mutually exclusive events in most tumors, indicating that deregulation of Ras-dependent signaling is the essential requirement for tumorigenesis.  In contrast to sporadic tumors, sep. studies have identified germline mutations in Ras and various other components of Ras signaling pathways that occur in specific assocn. with a no. of different familial, developmental syndromes frequently sharing common phenotypic cardiofaciocutaneous features.  Finally, even without being a causative force, defective Ras signaling has been cited as a contributing factor to many other human illnesses, including diabetes and immunol. and inflammatory disorders.  We aim this review at summarizing and updating current knowledge on the contribution of Ras mutations and altered Ras signaling to development of various tumoral and nontumoral pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC8FC347lm6bVg90H21EOLACvtfcHk0ljsk-RwPO69uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWlsbs%253D&md5=048ad92dd5959250e4a2b8e0319ff225</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1177%2F1947601911411084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911411084%26sid%3Dliteratum%253Aachs%26aulast%3DFernandez-Medarde%26aufirst%3DA.%26aulast%3DSantos%26aufirst%3DE.%26atitle%3DRas%2520in%2520cancer%2520and%2520developmental%2520diseases%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D344%26epage%3D358%26doi%3D10.1177%2F1947601911411084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karoulia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavathiotis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulikakos, P. I.</span></span> <span> </span><span class="NLM_article-title">New perspectives for targeting RAF kinase in human cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">676</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.79</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnrc.2017.79" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=28984291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqt7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=676-691&author=Z.+Karouliaauthor=E.+Gavathiotisauthor=P.+I.+Poulikakos&title=New+perspectives+for+targeting+RAF+kinase+in+human+cancer&doi=10.1038%2Fnrc.2017.79"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">New perspectives for targeting RAF kinase in human cancer</span></div><div class="casAuthors">Karoulia, Zoi; Gavathiotis, Evripidis; Poulikakos, Poulikos I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">676-691</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The discovery that a subset of human tumors is dependent on mutationally deregulated BRAF kinase intensified the development of RAF inhibitors to be used as potential therapeutics.  The US Food and Drug Administration (FDA)-approved second-generation RAF inhibitors vemurafenib and dabrafenib have elicited remarkable responses and improved survival of patients with BRAF-V600E/K melanoma, but their effectiveness is limited by resistance.  Beyond melanoma, current clin. RAF inhibitors show modest efficacy when used for colorectal and thyroid BRAF-V600E tumors or for tumors harboring BRAF alterations other than the V600 mutation.  Accumulated exptl. and clin. evidence indicates that the complex biochem. mechanisms of RAF kinase signalling account both for the effectiveness of RAF inhibitors and for the various mechanisms of tumor resistance to them.  Recently, a no. of next-generation RAF inhibitors, with diverse structural and biochem. properties, have entered preclin. and clin. development.  In this Review, we discuss the current understanding of RAF kinase regulation, mechanisms of inhibitor action and related clin. resistance to these drugs.  The recent elucidation of crit. structural and biochem. aspects of RAF inhibitor action, combined with the availability of a no. of structurally diverse RAF inhibitors currently in preclin. and clin. development, will enable the design of more effective RAF inhibitors and RAF-inhibitor-based therapeutic strategies, tailored to different clin. contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEUrKAljS7SrVg90H21EOLACvtfcHk0ljsk-RwPO69uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqt7fN&md5=7e8ee4222467a2d5c12b23ccce99fc62</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.79%26sid%3Dliteratum%253Aachs%26aulast%3DKaroulia%26aufirst%3DZ.%26aulast%3DGavathiotis%26aufirst%3DE.%26aulast%3DPoulikakos%26aufirst%3DP.%2BI.%26atitle%3DNew%2520perspectives%2520for%2520targeting%2520RAF%2520kinase%2520in%2520human%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D676%26epage%3D691%26doi%3D10.1038%2Fnrc.2017.79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimaldi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Festino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanella, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strudel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascierto, P. A.</span></span> <span> </span><span class="NLM_article-title">MEK inhibitors in the treatment of metastatic melanoma and solid tumors</span>. <i>Am. J. Clin. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1007/s40257-017-0292-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1007%2Fs40257-017-0292-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=28537004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BC1cngsFCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=745-754&author=A.+M.+Grimaldiauthor=E.+Simeoneauthor=L.+Festinoauthor=V.+Vanellaauthor=M.+Strudelauthor=P.+A.+Ascierto&title=MEK+inhibitors+in+the+treatment+of+metastatic+melanoma+and+solid+tumors&doi=10.1007%2Fs40257-017-0292-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">MEK Inhibitors in the Treatment of Metastatic Melanoma and Solid Tumors</span></div><div class="casAuthors">Grimaldi Antonio M; Simeone Ester; Festino Lucia; Vanella Vito; Strudel Martina; Ascierto Paolo A</div><div class="citationInfo"><span class="NLM_cas:title">American journal of clinical dermatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">745-754</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The mitogen-activated protein kinase (MAPK) cascade is an intracellular signaling pathway involved in the regulation of cellular proliferation and the survival of tumor cells.  Several different mutations, involving BRAF or NRAS, exert an oncogenic effect by activating the MAPK pathway, resulting in an increase in cellular proliferation.  These mutations have become targets for new therapeutic strategies in melanoma and other cancers.  Selective MEK inhibitors have the ability to inhibit growth and induce cell death in BRAF- and NRAS-mutant melanoma cell lines.  MEK inhibitor therapy in combination with a BRAF inhibitor is more effective and less toxic than treatment with a BRAF inhibitor alone, and has become the standard of care for patients with BRAF-mutated melanoma.  Trametinib was the first MEK inhibitor approved for the treatment of BRAF-mutated metastatic melanoma not previously treated with BRAF inhibitors, and is also approved in combination with the BRAF inhibitor dabrafenib.  Furthermore, cobimetinib is another MEK inhibitor approved for the treatment of BRAF-mutated metastatic melanoma in combination with a BRAF inhibitor, vemurafenib.  The MEK inhibitor binimetinib in combination with the BRAF inhibitor encorafenib is in clinical development.  The addition of an anti-PD-1/PD-L1 agent, such as pembrolizumab, durvalumab or atezolizumab, to combined BRAF and MEK inhibition has shown considerable promise, with several trials ongoing in metastatic melanoma.  Binimetinib has also shown efficacy in NRAS-mutated melanoma patients.  Future possibilities for MEK inhibitors in advanced melanoma, as well as other solid tumors, include their use in combination with other targeted therapies (e.g. anti-CDK4/6 inhibitors) and/or various immune-modulating antibodies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTRMQVeZbuZD8RAV4D_Ins_fW6udTcc2eapoaskXQ7nNLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cngsFCgtA%253D%253D&md5=23b16eb79f7203a8acd54e8c9249357d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs40257-017-0292-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40257-017-0292-y%26sid%3Dliteratum%253Aachs%26aulast%3DGrimaldi%26aufirst%3DA.%2BM.%26aulast%3DSimeone%26aufirst%3DE.%26aulast%3DFestino%26aufirst%3DL.%26aulast%3DVanella%26aufirst%3DV.%26aulast%3DStrudel%26aufirst%3DM.%26aulast%3DAscierto%26aufirst%3DP.%2BA.%26atitle%3DMEK%2520inhibitors%2520in%2520the%2520treatment%2520of%2520metastatic%2520melanoma%2520and%2520solid%2520tumors%26jtitle%3DAm.%2520J.%2520Clin.%2520Dermatol.%26date%3D2017%26volume%3D18%26spage%3D745%26epage%3D754%26doi%3D10.1007%2Fs40257-017-0292-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. I.</span></span> <span> </span><span class="NLM_article-title">MEK1/2 Inhibitors: molecular activity and resistance mechanisms</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">849</span>– <span class="NLM_lpage">862</span>, <span class="refDoi"> DOI: 10.1053/j.seminoncol.2015.09.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1053%2Fj.seminoncol.2015.09.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26615130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=849-862&author=P.+K.+Wuauthor=J.+I.+Park&title=MEK1%2F2+Inhibitors%3A+molecular+activity+and+resistance+mechanisms&doi=10.1053%2Fj.seminoncol.2015.09.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1/2 Inhibitors: Molecular Activity and Resistance Mechanisms</span></div><div class="casAuthors">Wu, Pui-Kei; Park, Jong-In</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">849-862</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Aberrant activation of the three-layered protein kinase cascade, Raf/MEK/ERK, is often detected in human cancer, which is mainly attributed to the oncogenic alterations of RAF, or its upstream activators RAS or cell surface receptor tyrosine kinases.  Deregulated activity of the Raf/MEK/ERK pathway drives uncontrolled tumor cell proliferation and survival, thus providing a rational therapeutic target for the treatment of many cancers.  While Raf, MEK1/2, and ERK1/2 are equally important targets for the design of therapeutic small mol. wt. inhibitors, the effort to develop MEK1/2-specific inhibitors has been greatly successful.  Particularly, MEK1/2 have been relatively advantageous for the design of highly selective ATP-noncompetitive inhibitors.  Indeed, a plethora of highly selective and potent MEK1/2 inhibitors are now available and many of those inhibitors have been evaluated for their therapeutic potential.  Herein, we review different MEK1/2 inhibitors that have been studied for their inhibitory mechanisms and therapeutic potential in cancer.  Some of the key structural features of MEK1/2 that are important for the efficacy of these inhibitors are also discussed.  In addn., we discuss current challenges and future prospective in using these advanced MEK1/2 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCuCxmKMCT7bVg90H21EOLACvtfcHk0lgZCNI0nemcog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWks7fF&md5=444f0ee51369a9e9e8baa29ddd8d90a5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1053%2Fj.seminoncol.2015.09.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.seminoncol.2015.09.023%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%2BK.%26aulast%3DPark%26aufirst%3DJ.%2BI.%26atitle%3DMEK1%252F2%2520Inhibitors%253A%2520molecular%2520activity%2520and%2520resistance%2520mechanisms%26jtitle%3DSemin.%2520Oncol.%26date%3D2015%26volume%3D42%26spage%3D849%26epage%3D862%26doi%3D10.1053%2Fj.seminoncol.2015.09.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, K. L.</span></span> <span> </span><span class="NLM_article-title">RAS isoforms and mutations in cancer at a glance</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">1287</span>– <span class="NLM_lpage">1292</span>, <span class="refDoi"> DOI: 10.1242/jcs.182873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1242%2Fjcs.182873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26985062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKltr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2016&pages=1287-1292&author=G.+A.+Hobbsauthor=C.+J.+Derauthor=K.+L.+Rossman&title=RAS+isoforms+and+mutations+in+cancer+at+a+glance&doi=10.1242%2Fjcs.182873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">RAS isoforms and mutations in cancer at a glance</span></div><div class="casAuthors">Hobbs, G. Aaron; Der, Channing J.; Rossman, Kent L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1287-1292</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">RAS proteins (KRAS4A, KRAS4B, NRAS and HRAS) function as GDP-GTP-regulated binary on-off switches, which regulate cytoplasmic signaling networks that control diverse normal cellular processes.  Gain-of-function missense mutations in RAS genes are found in ∼ 25% of human cancers, prompting interest in identifying anti-RAS therapeutic strategies for cancer treatment.  However, despite more than three decades of intense effort, no anti-RAS therapies have reached clin. application.  Contributing to this failure has been an underestimation of the complexities of RAS.  First, there is now appreciation that the four human RAS proteins are not functionally identical.  Second, with >130 different missense mutations found in cancer, there is an emerging view that there are mutation-specific consequences on RAS structure, biochem. and biol., and mutation-selective therapeutic strategies are needed.  In this Cell Science at a Glance article and accompanying poster, we provide a snapshot of the differences between RAS isoforms and mutations, as well as the current status of anti-RAS drug-discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNufX89IRJk7Vg90H21EOLACvtfcHk0lgZCNI0nemcog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKltr7I&md5=bf50d212b900fb437cad2b650a9dc07a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1242%2Fjcs.182873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.182873%26sid%3Dliteratum%253Aachs%26aulast%3DHobbs%26aufirst%3DG.%2BA.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26aulast%3DRossman%26aufirst%3DK.%2BL.%26atitle%3DRAS%2520isoforms%2520and%2520mutations%2520in%2520cancer%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2016%26volume%3D129%26spage%3D1287%26epage%3D1292%26doi%3D10.1242%2Fjcs.182873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span> <span> </span><span class="NLM_article-title">Targeting ERK, an Achilles’ heel of the MAPK pathway, in cancer therapy</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2018.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.apsb.2018.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30109180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ot1CmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=552-562&author=F.+Liuauthor=X.+Yangauthor=M.+Gengauthor=M.+Huang&title=Targeting+ERK%2C+an+Achilles%E2%80%99+heel+of+the+MAPK+pathway%2C+in+cancer+therapy&doi=10.1016%2Fj.apsb.2018.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting ERK, an Achilles' Heel of the MAPK pathway, in cancer therapy</span></div><div class="casAuthors">Liu Feifei; Yang Xiaotong; Geng Meiyu; Huang Min</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">552-562</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">The mitogen-activated protein kinases (MAPK) pathway, often known as the RAS-RAF-MEK-ERK signal cascade, functions to transmit upstream signals to its downstream effectors to regulate physiological process such as cell proliferation, differentiation, survival and death.  As the most frequently mutated signaling pathway in human cancer, targeting the MAPK pathway has long been considered a promising strategy for cancer therapy.  Substantial efforts in the past decades have led to the clinical success of BRAF and MEK inhibitors.  However, the clinical benefits of these inhibitors are compromised by the frequently occurring acquired resistance due to cancer heterogeneity and genomic instability.  This review briefly introduces the key protein kinases involved in this pathway as well as their activation mechanisms.  We also generalize the correlations between mutations of MAPK members and human cancers, followed by a summarization of progress made on the development of small molecule MAPK kinases inhibitors.  In particular, this review highlights the potential advantages of ERK inhibitors in overcoming resistance to upstream targets and proposes that targeting ERK kinase may hold a promising prospect for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-a-G-lYtjXxdUSIfHIn4FfW6udTcc2eYRLhsCUOTvKbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ot1CmsA%253D%253D&md5=37768ff6015b0792e009e7b53a357d65</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2018.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2018.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DM.%26atitle%3DTargeting%2520ERK%252C%2520an%2520Achilles%25E2%2580%2599%2520heel%2520of%2520the%2520MAPK%2520pathway%252C%2520in%2520cancer%2520therapy%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2018%26volume%3D8%26spage%3D552%26epage%3D562%26doi%3D10.1016%2Fj.apsb.2018.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. S.</span></span> <span> </span><span class="NLM_article-title">BRAF and MEK inhibitors: use and resistance in BRAF-mutated cancers</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1007/s40265-018-0884-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1007%2Fs40265-018-0884-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29488071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFKns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=549-566&author=J.+N.+Sanchezauthor=T.+Wangauthor=M.+S.+Cohen&title=BRAF+and+MEK+inhibitors%3A+use+and+resistance+in+BRAF-mutated+cancers&doi=10.1007%2Fs40265-018-0884-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">BRAF and MEK Inhibitors: Use and Resistance in BRAF-Mutated Cancers</span></div><div class="casAuthors">Sanchez, Jaquelyn N.; Wang, Ton; Cohen, Mark S.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">549-566</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">The mitogen activated protein kinase/extracellular signal-related kinase (MAPK/ERK) signaling pathway serves an integral role in growth, proliferation, differentiation, migration, and survival of all mammalian cells.  Aberrant signaling of this pathway is often obsd. in several types of hematol. and solid malignancies.  The most frequent insult to this signaling cascade, leading to its constitutive activation, is to the serine/threonine kinase rapidly accelerating fibrosarcoma (RAF).  Considering this, the development and approval of various small-mol. inhibitors targeting the MAPK/ERK pathway has become a mainstay of treatment as either mono- or combination therapy in these cancers.  Although effective initially, a major clin. barrier with these inhibitors is the relapse of patients due to drug resistance.  Knowledge of the mechanisms of resistance to these drugs is still premature, highlighting the need for a more in-depth understanding of how patients become insensitive to these pharmacol. interventions.  Herein, we will succinctly summarize the milestones in the approval of select MAPK/ERK pathway inhibitors, their use in patients, and major modes of resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW2IJqDOsQsbVg90H21EOLACvtfcHk0lgZCNI0nemcog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFKns7c%253D&md5=82a87a206e6544803ec6799ad42b1126</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs40265-018-0884-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-018-0884-8%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DJ.%2BN.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DCohen%26aufirst%3DM.%2BS.%26atitle%3DBRAF%2520and%2520MEK%2520inhibitors%253A%2520use%2520and%2520resistance%2520in%2520BRAF-mutated%2520cancers%26jtitle%3DDrugs%26date%3D2018%26volume%3D78%26spage%3D549%26epage%3D566%26doi%3D10.1007%2Fs40265-018-0884-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murugan, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, M.</span></span> <span> </span><span class="NLM_article-title">MEK1 mutations, but not ERK2 mutations, occur in melanomas and colon carcinomas, but none in thyroid carcinomas</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2122</span>– <span class="NLM_lpage">2124</span>, <span class="refDoi"> DOI: 10.4161/cc.8.13.8710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.4161%2Fcc.8.13.8710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=19411838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovFyqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=2122-2124&author=A.+K.+Muruganauthor=J.+Dongauthor=J.+Xieauthor=M.+Xing&title=MEK1+mutations%2C+but+not+ERK2+mutations%2C+occur+in+melanomas+and+colon+carcinomas%2C+but+none+in+thyroid+carcinomas&doi=10.4161%2Fcc.8.13.8710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1 mutations, but not ERK2 mutations, occur in melanomas and colon carcinomas, but none in thyroid carcinomas</span></div><div class="casAuthors">Murugan, Avaniyapuram Kannan; Dong, Jianli; Xie, Jingwu; Xing, Mingzhao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2122-2124</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Mitogen-activated protein kinase (MAPK) signaling pathway plays an important role in the pathogenesis of melanoma, colon cancer and thyroid cancer, which commonly harbor RAS and BRAF mutations.  However, mutations in exon 2 of MEK1 and exon 7 of ERK2 have not been investigated in these cancers although they are occasionally found in some other cancers or cell lines.  In this study, we performed mutational anal. to search for these mutations in 185 samples, including 167 tumor samples and 18 cell lines of melanoma, colon cancer and thyroid cancer.  We found one MEK1 mutation in 1 of 37 (3%) melanoma tumor samples and another MEK1 mutation in 1 of 45 (2.2%) colon cancer samples.  We did not find any MEK1 mutation in 99 thyroid tumor samples and 12 thyroid cancer cell lines.  We also did not find any ERK2 mutation in melanoma, colon cancer and thyroid cancer.  We thus report for the first time a low prevalence of MEK1 mutations in melanoma and colon cancer.  Both of the two mutants were demonstrated to be activating in the MAPK signaling pathway and may therefore provide potential target for effective therapy in cases of melanomas and colon cancer harboring these mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbwTT0xH_R_7Vg90H21EOLACvtfcHk0liS0eVnBDzpvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovFyqtQ%253D%253D&md5=ad3811165466cf85870772386494bd68</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4161%2Fcc.8.13.8710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.8.13.8710%26sid%3Dliteratum%253Aachs%26aulast%3DMurugan%26aufirst%3DA.%2BK.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DXing%26aufirst%3DM.%26atitle%3DMEK1%2520mutations%252C%2520but%2520not%2520ERK2%2520mutations%252C%2520occur%2520in%2520melanomas%2520and%2520colon%2520carcinomas%252C%2520but%2520none%2520in%2520thyroid%2520carcinomas%26jtitle%3DCell%2520Cycle%26date%3D2009%26volume%3D8%26spage%3D2122%26epage%3D2124%26doi%3D10.4161%2Fcc.8.13.8710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolaev, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimoldi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iseli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valsesia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robyr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harshman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guipponi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukach, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muehlethaler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speiser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckmann, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenarios, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halazonetis, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jongeneel, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonarakis, S. E.</span></span> <span> </span><span class="NLM_article-title">Exome sequencing identifies recurrent somatic MAP2K1 and MAP2K2 mutations in melanoma</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">139</span>, <span class="refDoi"> DOI: 10.1038/ng.1026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fng.1026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1OltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=133-139&author=S.+I.+Nikolaevauthor=D.+Rimoldiauthor=C.+Iseliauthor=A.+Valsesiaauthor=D.+Robyrauthor=C.+Gehrigauthor=K.+Harshmanauthor=M.+Guipponiauthor=O.+Bukachauthor=V.+Zoeteauthor=O.+Michielinauthor=K.+Muehlethalerauthor=D.+Speiserauthor=J.+S.+Beckmannauthor=I.+Xenariosauthor=T.+D.+Halazonetisauthor=C.+V.+Jongeneelauthor=B.+J.+Stevensonauthor=S.+E.+Antonarakis&title=Exome+sequencing+identifies+recurrent+somatic+MAP2K1+and+MAP2K2+mutations+in+melanoma&doi=10.1038%2Fng.1026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exome sequencing identifies recurrent somatic MAP2K1 and MAP2K2 mutations in melanoma</span></div><div class="casAuthors">Nikolaev, Sergey I.; Rimoldi, Donata; Iseli, Christian; Valsesia, Armand; Robyr, Daniel; Gehrig, Corinne; Harshman, Keith; Guipponi, Michel; Bukach, Olesya; Zoete, Vincent; Michielin, Olivier; Muehlethaler, Katja; Speiser, Daniel; Beckmann, Jacques S.; Xenarios, Ioannis; Halazonetis, Thanos D.; Jongeneel, C. Victor; Stevenson, Brian J.; Antonarakis, Stylianos E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-139</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We performed exome sequencing to detect somatic mutations in protein-coding regions in seven melanoma cell lines and donor-matched germline cells.  All melanoma samples had high nos. of somatic mutations, which showed the hallmark of UV-induced DNA repair.  Such a hallmark was absent in tumor sample-specific mutations in two metastases derived from the same individual.  Two melanomas with non-canonical BRAF mutations harbored gain-of-function MAP2K1 and MAP2K2 (MEK1 and MEK2, resp.) mutations, resulting in constitutive ERK phosphorylation and higher resistance to MEK inhibitors.  Screening a larger cohort of individuals with melanoma revealed the presence of recurring somatic MAP2K1 and MAP2K2 mutations, which occurred at an overall frequency of 8%.  Furthermore, missense and nonsense somatic mutations were frequently found in three candidate melanoma genes, FAT4, LRP1B and DSC1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqoz4Pb6v6W7Vg90H21EOLACvtfcHk0liS0eVnBDzpvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1OltbbJ&md5=9888253381503600ced0c828a097df48</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fng.1026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.1026%26sid%3Dliteratum%253Aachs%26aulast%3DNikolaev%26aufirst%3DS.%2BI.%26aulast%3DRimoldi%26aufirst%3DD.%26aulast%3DIseli%26aufirst%3DC.%26aulast%3DValsesia%26aufirst%3DA.%26aulast%3DRobyr%26aufirst%3DD.%26aulast%3DGehrig%26aufirst%3DC.%26aulast%3DHarshman%26aufirst%3DK.%26aulast%3DGuipponi%26aufirst%3DM.%26aulast%3DBukach%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DMuehlethaler%26aufirst%3DK.%26aulast%3DSpeiser%26aufirst%3DD.%26aulast%3DBeckmann%26aufirst%3DJ.%2BS.%26aulast%3DXenarios%26aufirst%3DI.%26aulast%3DHalazonetis%26aufirst%3DT.%2BD.%26aulast%3DJongeneel%26aufirst%3DC.%2BV.%26aulast%3DStevenson%26aufirst%3DB.%2BJ.%26aulast%3DAntonarakis%26aufirst%3DS.%2BE.%26atitle%3DExome%2520sequencing%2520identifies%2520recurrent%2520somatic%2520MAP2K1%2520and%2520MAP2K2%2520mutations%2520in%2520melanoma%26jtitle%3DNat.%2520Genet.%26date%3D2012%26volume%3D44%26spage%3D133%26epage%3D139%26doi%3D10.1038%2Fng.1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arcila, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drilon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borsu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kris, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladanyi, M.</span></span> <span> </span><span class="NLM_article-title">MAP2K1 (MEK1) mutations define a distinct subset of lung adenocarcinoma associated with smoking</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1935</span>– <span class="NLM_lpage">1943</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-2124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1158%2F1078-0432.CCR-14-2124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=25351745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsl2gtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1935-1943&author=M.+E.+Arcilaauthor=A.+Drilonauthor=B.+E.+Sylvesterauthor=C.+M.+Lovlyauthor=L.+Borsuauthor=B.+Revaauthor=M.+G.+Krisauthor=D.+B.+Solitauthor=M.+Ladanyi&title=MAP2K1+%28MEK1%29+mutations+define+a+distinct+subset+of+lung+adenocarcinoma+associated+with+smoking&doi=10.1158%2F1078-0432.CCR-14-2124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">MAP2K1 (MEK1) Mutations Define a Distinct Subset of Lung Adenocarcinoma Associated with Smoking</span></div><div class="casAuthors">Arcila, Maria E.; Drilon, Alexander; Sylvester, Brooke E.; Lovly, Christine M.; Borsu, Laetitia; Reva, Boris; Kris, Mark G.; Solit, David B.; Ladanyi, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1935-1943</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Genetic alterations affecting the MAPK/ERK pathway are common in lung adenocarcinoma (LAD).  Early steps of the signaling pathway are most often affected with EGFR, KRAS, and BRAF mutations encompassing more than 70% of all alterations.  Somatic mutations in MEK1, located downstream of BRAF, are rare and remain poorly defined as a distinct mol. subset.  Exptl. Design: Tumors harboring MEK1 mutations were identified through targeted screening of a large LAD cohort concurrently interrogated for recurrent mutations in MEK1, EGFR, KRAS, BRAF, ERBB2/HER2, NRAS, PIK3CA, and AKT.  Addnl. cases were identified through a search of publically available cancer genomic datasets.  Mutations were correlated with patient characteristics and treatment outcomes.  Overall survival was compared with stage-matched patients with KRAS- and EGFR-mutant LADs.  Results: We identified 36 MEK1-mutated cases among 6,024 LAD (0.6%; 95% confidence interval, 0.42-0.85).  The majority of patients were smokers (97%, n = 35/36).  There was no assocn. with age, sex, race, or stage.  The most common mutations were K57N (64%, 23/36) followed by Q56P (19%, 7/36), all mutually exclusive with other driver mutations in the targeted panel.  Transversions G:C>T:A were predominant (89%, 31/35), in keeping with smoking-assocd. DNA damage.  Addnl. less common somatic mutations were identified in the kinase domain, all of which are predicted to converge into a single interaction area based on in silico 3D modeling.  Conclusions: MEK1 mutations define a distinct subset of lung cancers (∼1%) with potential sensitivity to MEK inhibitors.  Mutations are predominantly transversions, in keeping with a strong assocn. with smoking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEJ9tmIMr9DbVg90H21EOLACvtfcHk0liS0eVnBDzpvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsl2gtb8%253D&md5=92aa864d442da526e85b7ecc536130f6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-2124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-2124%26sid%3Dliteratum%253Aachs%26aulast%3DArcila%26aufirst%3DM.%2BE.%26aulast%3DDrilon%26aufirst%3DA.%26aulast%3DSylvester%26aufirst%3DB.%2BE.%26aulast%3DLovly%26aufirst%3DC.%2BM.%26aulast%3DBorsu%26aufirst%3DL.%26aulast%3DReva%26aufirst%3DB.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DLadanyi%26aufirst%3DM.%26atitle%3DMAP2K1%2520%2528MEK1%2529%2520mutations%2520define%2520a%2520distinct%2520subset%2520of%2520lung%2520adenocarcinoma%2520associated%2520with%2520smoking%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1935%26epage%3D1943%26doi%3D10.1158%2F1078-0432.CCR-14-2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grisham, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Won, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLair, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogomolniy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soumerai, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyman, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Socci, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershenson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spriggs, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aghajanian, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, G.</span></span> <span> </span><span class="NLM_article-title">Extreme outlier analysis identifies occult mitogen-activated protein kinase pathway mutations in patients with low-grade serous ovarian cancer</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">4099</span>– <span class="NLM_lpage">4105</span>, <span class="refDoi"> DOI: 10.1200/JCO.2015.62.4726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1200%2FJCO.2015.62.4726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26324360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVynsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=4099-4105&author=R.+N.+Grishamauthor=B.+E.+Sylvesterauthor=H.+Wonauthor=G.+McDermottauthor=D.+DeLairauthor=R.+Ramirezauthor=Z.+Yaoauthor=R.+Shenauthor=F.+Daoauthor=F.+Bogomolniyauthor=V.+Makkerauthor=E.+Salaauthor=T.+E.+Soumeraiauthor=D.+M.+Hymanauthor=N.+D.+Socciauthor=A.+Vialeauthor=D.+M.+Gershensonauthor=J.+Farleyauthor=D.+A.+Levineauthor=N.+Rosenauthor=M.+F.+Bergerauthor=D.+R.+Spriggsauthor=C.+A.+Aghajanianauthor=D.+B.+Solitauthor=G.+Iyer&title=Extreme+outlier+analysis+identifies+occult+mitogen-activated+protein+kinase+pathway+mutations+in+patients+with+low-grade+serous+ovarian+cancer&doi=10.1200%2FJCO.2015.62.4726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Extreme outlier analysis identifies occult mitogen-activated protein kinase pathway mutations in patients with low-grade serous ovarian cancer</span></div><div class="casAuthors">Grisham, Rachel N.; Sylvester, Brooke E.; Won, Helen; McDermott, Gregory; DeLair, Deborah; Ramirez, Ricardo; Yao, Zhan; Shen, Ronglai; Dao, Fanny; Bogomolniy, Faina; Makker, Vicky; Sala, Evis; Soumerai, Tara E.; Hyman, David M.; Socci, Nicholas D.; Viale, Agnes; Gershenson, David M.; Farley, John; Levine, Douglas A.; Rosen, Neal; Berger, Michael F.; Spriggs, David R.; Aghajanian, Carol A.; Solit, David B.; Iyer, Gopa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4099-4105</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose No effective systemic therapy exists for patients with metastatic low-grade serous (LGS) ovarian cancers.  BRAF and KRAS mutations are common in serous borderline (SB) and LGS ovarian cancers, and MEK inhibition has been shown to induce tumor regression in a minority of patients; however, no correlation has been obsd. between mutation status and clin. response.  With the goal of identifying biomarkers of sensitivity to MEK inhibitor treatment, we performed an outlier anal. of a patient who experienced a complete, durable, and ongoing (> 5 years) response to selumetinib, a non-ATP competitive MEK inhibitor.  Patients and Methods Next-generation sequencing was used to analyze this patient's tumor as well as an addnl. 28 SB/LGS tumors.  Functional characterization of an identified novel alteration of interest was performed.  Results Anal. of the extraordinary responder's tumor identified a 15-nucleotide deletion in the neg. regulatory helix of the MAP2K1 gene encoding for MEK1.  Functional characterization demonstrated that this mutant induced extracellular signal-regulated kinase pathway activation, promoted anchorage-independent growth and tumor formation in mice, and retained sensitivity to selumetinib.  Anal. of addnl. LGS/SB tumors identified mutations predicted to induce extracellular signal-regulated kinase pathway activation in 82% (23 of 28), including two patients with BRAF fusions, one of whom achieved an ongoing complete response to MEK inhibitor-based combination therapy.  Conclusion Alterations affecting the mitogen-activated protein kinase pathway are present in the majority of patients with LGS ovarian cancer.  Next-generation sequencing anal. revealed deletions and fusions that are not detected by older sequencing approaches.  These findings, coupled with the observation that a subset of patients with recurrent LGS ovarian cancer experienced dramatic and durable responses to MEK inhibitor therapy, support addnl. clin. studies of MEK inhibitors in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXv3nN7rbAh7Vg90H21EOLACvtfcHk0ljqxritmvqA0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVynsb4%253D&md5=3080b9e163347888b3db5f333ea50b05</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1200%2FJCO.2015.62.4726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2015.62.4726%26sid%3Dliteratum%253Aachs%26aulast%3DGrisham%26aufirst%3DR.%2BN.%26aulast%3DSylvester%26aufirst%3DB.%2BE.%26aulast%3DWon%26aufirst%3DH.%26aulast%3DMcDermott%26aufirst%3DG.%26aulast%3DDeLair%26aufirst%3DD.%26aulast%3DRamirez%26aufirst%3DR.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DDao%26aufirst%3DF.%26aulast%3DBogomolniy%26aufirst%3DF.%26aulast%3DMakker%26aufirst%3DV.%26aulast%3DSala%26aufirst%3DE.%26aulast%3DSoumerai%26aufirst%3DT.%2BE.%26aulast%3DHyman%26aufirst%3DD.%2BM.%26aulast%3DSocci%26aufirst%3DN.%2BD.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DGershenson%26aufirst%3DD.%2BM.%26aulast%3DFarley%26aufirst%3DJ.%26aulast%3DLevine%26aufirst%3DD.%2BA.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DBerger%26aufirst%3DM.%2BF.%26aulast%3DSpriggs%26aufirst%3DD.%2BR.%26aulast%3DAghajanian%26aufirst%3DC.%2BA.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DIyer%26aufirst%3DG.%26atitle%3DExtreme%2520outlier%2520analysis%2520identifies%2520occult%2520mitogen-activated%2520protein%2520kinase%2520pathway%2520mutations%2520in%2520patients%2520with%2520low-grade%2520serous%2520ovarian%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D33%26spage%3D4099%26epage%3D4105%26doi%3D10.1200%2FJCO.2015.62.4726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kakadia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarlagadda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundranda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naraev, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mina, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimbel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoud, F.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of resistance to BRAF and MEK inhibitors and clinical update of US Food and Drug Administration-approved targeted therapy in advanced melanoma</span>. <i>OncoTargets Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">7095</span>– <span class="NLM_lpage">7107</span>, <span class="refDoi"> DOI: 10.2147/OTT.S182721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.2147%2FOTT.S182721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30410366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFSgurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2018&pages=7095-7107&author=S.+Kakadiaauthor=N.+Yarlagaddaauthor=R.+Awadauthor=M.+Kundrandaauthor=J.+Niuauthor=B.+Naraevauthor=L.+Minaauthor=T.+Dragovichauthor=M.+Gimbelauthor=F.+Mahmoud&title=Mechanisms+of+resistance+to+BRAF+and+MEK+inhibitors+and+clinical+update+of+US+Food+and+Drug+Administration-approved+targeted+therapy+in+advanced+melanoma&doi=10.2147%2FOTT.S182721"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of resistance to BRAF and MEK inhibitors and clinical update of US Food and Drug Administration-approved targeted therapy in advanced melanoma</span></div><div class="casAuthors">Kakadia, Sunilkumar; Yarlagadda, Naveen; Awad, Ramez; Kundranda, Madappa; Niu, Jiaxin; Naraev, Boris; Mina, Lida; Dragovich, Tomislav; Gimbel, Mark; Mahmoud, Fade</div><div class="citationInfo"><span class="NLM_cas:title">OncoTargets and Therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7095-7107</span>CODEN:
                <span class="NLM_cas:coden">OTNHAZ</span>;
        ISSN:<span class="NLM_cas:issn">1178-6930</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Approx. 50% of melanomas harbor an activating BRAF mutation.  Combined BRAF and MEK inhibitors such as dabrafenib and trametinib, vemurafenib and cobimetinib, and encorafenib and binimetinib are US Food and Drug Administration (FDA)-approved to treat patients with BRAFV600-mutated advanced melanoma.  Both genetic and epigenetic alterations play a major role in resistance to BRAF inhibitors by reactivation of the MAPK and/or the PI3K-Akt pathways.  The role of BRAF inhibitors in modulating the immunomicroenvironment and perhaps enhancing the efficacy of checkpoint inhibitors is gaining interest.  This article provides a comprehensive review of mechanisms of resistance to BRAF and MEK inhibitors in melanoma and summarizes landmark trials that led to the FDA approval of BRAF and MEK inhibitors in metastatic melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYyD-wH6vxpbVg90H21EOLACvtfcHk0ljqxritmvqA0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFSgurnK&md5=57412f516c0e99e1846defd03492c71f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2147%2FOTT.S182721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FOTT.S182721%26sid%3Dliteratum%253Aachs%26aulast%3DKakadia%26aufirst%3DS.%26aulast%3DYarlagadda%26aufirst%3DN.%26aulast%3DAwad%26aufirst%3DR.%26aulast%3DKundranda%26aufirst%3DM.%26aulast%3DNiu%26aufirst%3DJ.%26aulast%3DNaraev%26aufirst%3DB.%26aulast%3DMina%26aufirst%3DL.%26aulast%3DDragovich%26aufirst%3DT.%26aulast%3DGimbel%26aufirst%3DM.%26aulast%3DMahmoud%26aufirst%3DF.%26atitle%3DMechanisms%2520of%2520resistance%2520to%2520BRAF%2520and%2520MEK%2520inhibitors%2520and%2520clinical%2520update%2520of%2520US%2520Food%2520and%2520Drug%2520Administration-approved%2520targeted%2520therapy%2520in%2520advanced%2520melanoma%26jtitle%3DOncoTargets%2520Ther.%26date%3D2018%26volume%3D11%26spage%3D7095%26epage%3D7107%26doi%3D10.2147%2FOTT.S182721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matallanas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birtwistle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zebisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kriegsheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolch, W.</span></span> <span> </span><span class="NLM_article-title">Raf family kinases: old dogs have learned new tricks</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1177/1947601911407323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1177%2F1947601911407323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=21779496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=232-260&author=D.+Matallanasauthor=M.+Birtwistleauthor=D.+Romanoauthor=A.+Zebischauthor=J.+Rauchauthor=A.+von+Kriegsheimauthor=W.+Kolch&title=Raf+family+kinases%3A+old+dogs+have+learned+new+tricks&doi=10.1177%2F1947601911407323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Raf family kinases: old dogs have learned new tricks</span></div><div class="casAuthors">Matallanas, David; Birtwistle, Marc; Romano, David; Zebisch, Armin; Rauch, Jens; von Kriegsheim, Alexander; Kolch, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">232-260</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  First identified in the early 1980s as retroviral oncogenes, the Raf proteins have been the objects of intense research.  The discoveries 10 yr later that the Raf family members (Raf-1, B-Raf, and A-Raf) are bona fide Ras effectors and upstream activators of the ubiquitous ERK kinase pathway increased the interest in these proteins primarily because of the central role that this cascade plays in cancer development.  The important role of Raf in cancer was corroborated in 2002 with the discovery of B-Raf genetic mutations in a large no. of tumors.  This led to intensified drug development efforts to target Raf signaling in cancer.  This work yielded not only recent clin. successes but also surprising insights into the regulation of Raf proteins by homodimerization and heterodimerization.  Surprising insights also came from the hunt for new Raf targets.  Although MEK remains the only widely accepted Raf substrate, new kinase-independent roles for Raf proteins have emerged.  These include the regulation of apoptosis by suppressing the activity of pro-apoptotic kinases ASK1 and MST2 and the regulation of cell motility and differentiation by controlling the activity of Rok-α.  Here, the authors discuss the regulation of Raf proteins and their role in cancer, with special focus on the interacting proteins that modulate Raf signaling.  The authors also describe the new pathways controlled by Raf proteins and summarize the successes and failures in the development of efficient anticancer therapies targeting Raf.  Finally, the authors also argue for the necessity of more systemic approaches to obtain a better understanding of how the Ras-Raf signaling network generates biol. specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0NzXqhz42aLVg90H21EOLACvtfcHk0ljqxritmvqA0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWls7k%253D&md5=4dba3fd7949b1c67fe5a0ecb1bec068a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1947601911407323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911407323%26sid%3Dliteratum%253Aachs%26aulast%3DMatallanas%26aufirst%3DD.%26aulast%3DBirtwistle%26aufirst%3DM.%26aulast%3DRomano%26aufirst%3DD.%26aulast%3DZebisch%26aufirst%3DA.%26aulast%3DRauch%26aufirst%3DJ.%26aulast%3Dvon%2BKriegsheim%26aufirst%3DA.%26aulast%3DKolch%26aufirst%3DW.%26atitle%3DRaf%2520family%2520kinases%253A%2520old%2520dogs%2520have%2520learned%2520new%2520tricks%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D232%26epage%3D260%26doi%3D10.1177%2F1947601911407323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span>FDA approves
encorafenib and binimetinib in combination for unresectable or metastatic
melanoma with BRAF mutations (accessed June 27, 2018). fda.gov, <span class="NLM_year">2018</span>; <a href="https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf" class="extLink">https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FDA+approves%0Aencorafenib+and+binimetinib+in+combination+for+unresectable+or+metastatic%0Amelanoma+with+BRAF+mutations+%28accessed+June+27%2C+2018%29.+fda.gov%2C+2018%3B+https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fresources-information-approved-drugs%2Ffda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karaszewska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackiewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroiakovski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichinitser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dummer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grange, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarion-Sileni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drucis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajsova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauschild, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorigan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crist, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadendorf, D.</span></span> <span> </span><span class="NLM_article-title">Improved overall survival in melanoma with combined dabrafenib and trametinib</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>372</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1412690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1056%2FNEJMoa1412690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=25399551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFSqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2015&pages=30-39&author=C.+Robertauthor=B.+Karaszewskaauthor=J.+Schachterauthor=P.+Rutkowskiauthor=A.+Mackiewiczauthor=D.+Stroiakovskiauthor=M.+Lichinitserauthor=R.+Dummerauthor=F.+Grangeauthor=L.+Mortierauthor=V.+Chiarion-Sileniauthor=K.+Drucisauthor=I.+Krajsovaauthor=A.+Hauschildauthor=P.+Loriganauthor=P.+Wolterauthor=G.+V.+Longauthor=K.+Flahertyauthor=P.+Nathanauthor=A.+Ribasauthor=A.+M.+Martinauthor=P.+Sunauthor=W.+Cristauthor=J.+Legosauthor=S.+D.+Rubinauthor=S.+M.+Littleauthor=D.+Schadendorf&title=Improved+overall+survival+in+melanoma+with+combined+dabrafenib+and+trametinib&doi=10.1056%2FNEJMoa1412690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Improved overall survival in melanoma with combined dabrafenib and trametinib</span></div><div class="casAuthors">Robert, Caroline; Karaszewska, Boguslawa; Schachter, Jacob; Rutkowski, Piotr; Mackiewicz, Andrzej; Stroiakovski, Daniil; Lichinitser, Michael; Dummer, Reinhard; Grange, Florent; Mortier, Laurent; Chiarion-Sileni, Vanna; Drucis, Kamil; Krajsova, Ivana; Hauschild, Axel; Lorigan, Paul; Wolter, Pascal; Long, Georgina V.; Flaherty, Keith; Nathan, Paul; Ribas, Antoni; Martin, Anne-Marie; Sun, Peng; Crist, Wendy; Legos, Jeff; Rubin, Stephen D.; Little, Shonda M.; Schadendorf, Dirk</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-39/1-30-39/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: The BRAF inhibitors vemurafenib and dabrafenib have shown efficacy as monotherapies in patients with previously untreated metastatic melanoma with BRAF V600E or V600K mutations.  Combining dabrafenib and the MEK inhibitor trametinib, as compared with dabrafenib alone, enhanced antitumor activity in this population of patients.  Methods: In this open-label, phase 3 trial, we randomly assigned 704 patients with metastatic melanoma with a BRAF V600 mutation to receive either a combination of dabrafenib (150 mg twice daily) and trametinib (2 mg once daily) or vemurafenib (960 mg twice daily) orally as first-line therapy.  The primary end point was overall survival.  Results: At the preplanned interim overall survival anal., which was performed after 77% of the total no. of expected events occurred, the overall survival rate at 12 mo was 72% (95% confidence interval [CI], 67 to 77) in the combination-therapy group and 65% (95% CI, 59 to 70) in the vemurafenib group (hazard ratio for death in the combination-therapy group, 0.69; 95% CI, 0.53 to 0.89; P = 0.005).  The prespecified interim stopping boundary was crossed, and the study was stopped for efficacy in July 2014.  Median progression-free survival was 11.4 mo in the combination therapy group and 7.3 mo in the vemurafenib group (hazard ratio, 0.56; 95% CI, 0.46 to 0.69; P<0.001).  The objective response rate was 64% in the combination therapy group and 51% in the vemurafenib group (P<0.001).  Rates of severe adverse events and study-drug discontinuations were similar in the two groups.  Cutaneous squamous-cell carcinoma and keratoacanthoma occurred in 1% of patients in the combination-therapy group and 18% of those in the vemurafenib group.  Conclusions: Dabrafenib plus trametinib, as compared with vemurafenib monotherapy, significantly improved overall survival in previously untreated patients with metastatic melanoma with BRAF V600E or V600K mutations, without increased overall toxicity. (Funded by GlaxoSmithKline; ClinicalTrials.gov no., NCT01597908.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5of4j14YrP7Vg90H21EOLACvtfcHk0lhQqC07IX0Ftg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFSqsb0%253D&md5=9d6a8f49bbd196f9463ec15958eb15d1</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1412690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1412690%26sid%3Dliteratum%253Aachs%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DKaraszewska%26aufirst%3DB.%26aulast%3DSchachter%26aufirst%3DJ.%26aulast%3DRutkowski%26aufirst%3DP.%26aulast%3DMackiewicz%26aufirst%3DA.%26aulast%3DStroiakovski%26aufirst%3DD.%26aulast%3DLichinitser%26aufirst%3DM.%26aulast%3DDummer%26aufirst%3DR.%26aulast%3DGrange%26aufirst%3DF.%26aulast%3DMortier%26aufirst%3DL.%26aulast%3DChiarion-Sileni%26aufirst%3DV.%26aulast%3DDrucis%26aufirst%3DK.%26aulast%3DKrajsova%26aufirst%3DI.%26aulast%3DHauschild%26aufirst%3DA.%26aulast%3DLorigan%26aufirst%3DP.%26aulast%3DWolter%26aufirst%3DP.%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DFlaherty%26aufirst%3DK.%26aulast%3DNathan%26aufirst%3DP.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DA.%2BM.%26aulast%3DSun%26aufirst%3DP.%26aulast%3DCrist%26aufirst%3DW.%26aulast%3DLegos%26aufirst%3DJ.%26aulast%3DRubin%26aufirst%3DS.%2BD.%26aulast%3DLittle%26aufirst%3DS.%2BM.%26aulast%3DSchadendorf%26aufirst%3DD.%26atitle%3DImproved%2520overall%2520survival%2520in%2520melanoma%2520with%2520combined%2520dabrafenib%2520and%2520trametinib%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2015%26volume%3D372%26spage%3D30%26epage%3D39%26doi%3D10.1056%2FNEJMoa1412690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroyakovskiy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levchenko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Braud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jouary, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauschild, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grob, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarion-Sileni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebbe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millward, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arance, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondarenko, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haanen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Utikal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraresi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovalenko, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Probachai, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadendorf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMarini, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irani, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legos, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mookerjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, K.</span></span> <span> </span><span class="NLM_article-title">Dabrafenib and trametinib versus dabrafenib and placebo for Val600 BRAF-mutant melanoma: a multicentre, double-blind, phase 3 randomised controlled trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>386</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(15)60898-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2FS0140-6736%2815%2960898-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=26037941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpsVyrt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2015&pages=444-451&author=G.+V.+Longauthor=D.+Stroyakovskiyauthor=H.+Gogasauthor=E.+Levchenkoauthor=F.+de+Braudauthor=J.+Larkinauthor=C.+Garbeauthor=T.+Jouaryauthor=A.+Hauschildauthor=J.+J.+Grobauthor=V.+Chiarion-Sileniauthor=C.+Lebbeauthor=M.+Mandalaauthor=M.+Millwardauthor=A.+Aranceauthor=I.+Bondarenkoauthor=J.+B.+Haanenauthor=J.+Hanssonauthor=J.+Utikalauthor=V.+Ferraresiauthor=N.+Kovalenkoauthor=P.+Mohrauthor=V.+Probachaiauthor=D.+Schadendorfauthor=P.+Nathanauthor=C.+Robertauthor=A.+Ribasauthor=D.+J.+DeMariniauthor=J.+G.+Iraniauthor=S.+Swannauthor=J.+J.+Legosauthor=F.+Jinauthor=B.+Mookerjeeauthor=K.+Flaherty&title=Dabrafenib+and+trametinib+versus+dabrafenib+and+placebo+for+Val600+BRAF-mutant+melanoma%3A+a+multicentre%2C+double-blind%2C+phase+3+randomised+controlled+trial&doi=10.1016%2FS0140-6736%2815%2960898-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Dabrafenib and trametinib versus dabrafenib and placebo for Val600 BRAF-mutant melanoma: a multicentre, double-blind, phase 3 randomised controlled trial</span></div><div class="casAuthors">Long, Georgina V.; Stroyakovskiy, Daniil; Gogas, Helen; Levchenko, Evgeny; de Braud, Filippo; Larkin, James; Garbe, Claus; Jouary, Thomas; Hauschild, Axel; Grob, Jean-Jacques; Chiarion-Sileni, Vanna; Lebbe, Celeste; Mandala, Mario; Millward, Michael; Arance, Ana; Bondarenko, Igor; Haanen, John B. A. G.; Hansson, Johan; Utikal, Jochen; Ferraresi, Virginia; Kovalenko, Nadezhda; Mohr, Peter; Probachai, Volodymr; Schadendorf, Dirk; Nathan, Paul; Robert, Caroline; Ribas, Antoni; De Marini, Douglas J.; Irani, Jhangir G.; Swann, Suzanne; Legos, Jeffrey J.; Jin, Fan; Mookerjee, Bijoyesh; Flaherty, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">9992</span>),
    <span class="NLM_cas:pages">444-451</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Previously, a study of ours showed that the combination of dabrafenib and trametinib improves progression-free survival compared with dabrafenib and placebo in patients with BRAF Val600Lys/Glu mutation-pos. metastatic melanoma.  The study was continued to assess the secondary endpoint of overall survival, which we report in this Article.  We did this double-blind phase 3 study at 113 sites in 14 countries.  We enrolled previously untreated patients with BRAF Val600Glu or Val600Lys mutation-pos. unresectable stage IIIC or stage IV melanoma.  Participants were computer-randomised (1:1) to receive a combination of dabrafenib (150 mg orally twice daily) and trametinib (2 mg orally once daily), or dabrafenib and placebo.  The primary endpoint was progression-free survival and overall survival was a secondary endpoint.  This study is registered with ClinicalTrials.gov, no. NCT01584648.  Between May 4, 2012, and Nov 30, 2012, we screened 947 patients for eligibility, of whom 423 were randomly assigned to receive dabrafenib and trametinib (n=211) or dabrafenib only (n=212).  The final data cutoff was Jan 12, 2015, at which time 222 patients had died.  Median overall survival was 25·1 mo (95% CI 19·2-not reached) in the dabrafenib and trametinib group vs. 18·7 mo (15·2-23·7) in the dabrafenib only group (hazard ratio [HR] 0·71, 95% CI 0·55-0·92; p=0·0107).  Overall survival was 74% at 1 yr and 51% at 2 years in the dabrafenib and trametinib group vs. 68% and 42%, resp., in the dabrafenib only group.  Based on 301 events, median progression-free survival was 11·0 mo (95% CI 8·0-13·9) in the dabrafenib and trametinib group and 8·8 mo (5·9-9·3) in the dabrafenib only group (HR 0·67, 95% CI 0·53-0·84; p=0·0004; unadjusted for multiple testing).  Treatment-related adverse events occurred in 181 (87%) of 209 patients in the dabrafenib and trametinib group and 189 (90%) of 211 patients in the dabrafenib only group; the most common was pyrexia (108 patients, 52%) in the dabrafenib and trametinib group, and hyperkeratosis (70 patients, 33%) in the dabrafenib only group.  Grade 3 or 4 adverse events occurred in 67 (32%) patients in the dabrafenib and trametinib group and 66 (31%) patients in the dabrafenib only group.  The improvement in overall survival establishes the combination of dabrafenib and trametinib as the std. targeted treatment for BRAF Val600 mutation-pos. melanoma.  Studies assessing dabrafenib and trametinib in combination with immunotherapies are ongoing.GlaxoSmithKline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeIg9L7TouB7Vg90H21EOLACvtfcHk0lhQqC07IX0Ftg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpsVyrt7k%253D&md5=3de7b0a659267f73299e8111ff2a09bb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2815%2960898-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252815%252960898-4%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DStroyakovskiy%26aufirst%3DD.%26aulast%3DGogas%26aufirst%3DH.%26aulast%3DLevchenko%26aufirst%3DE.%26aulast%3Dde%2BBraud%26aufirst%3DF.%26aulast%3DLarkin%26aufirst%3DJ.%26aulast%3DGarbe%26aufirst%3DC.%26aulast%3DJouary%26aufirst%3DT.%26aulast%3DHauschild%26aufirst%3DA.%26aulast%3DGrob%26aufirst%3DJ.%2BJ.%26aulast%3DChiarion-Sileni%26aufirst%3DV.%26aulast%3DLebbe%26aufirst%3DC.%26aulast%3DMandala%26aufirst%3DM.%26aulast%3DMillward%26aufirst%3DM.%26aulast%3DArance%26aufirst%3DA.%26aulast%3DBondarenko%26aufirst%3DI.%26aulast%3DHaanen%26aufirst%3DJ.%2BB.%26aulast%3DHansson%26aufirst%3DJ.%26aulast%3DUtikal%26aufirst%3DJ.%26aulast%3DFerraresi%26aufirst%3DV.%26aulast%3DKovalenko%26aufirst%3DN.%26aulast%3DMohr%26aufirst%3DP.%26aulast%3DProbachai%26aufirst%3DV.%26aulast%3DSchadendorf%26aufirst%3DD.%26aulast%3DNathan%26aufirst%3DP.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DDeMarini%26aufirst%3DD.%2BJ.%26aulast%3DIrani%26aufirst%3DJ.%2BG.%26aulast%3DSwann%26aufirst%3DS.%26aulast%3DLegos%26aufirst%3DJ.%2BJ.%26aulast%3DJin%26aufirst%3DF.%26aulast%3DMookerjee%26aufirst%3DB.%26aulast%3DFlaherty%26aufirst%3DK.%26atitle%3DDabrafenib%2520and%2520trametinib%2520versus%2520dabrafenib%2520and%2520placebo%2520for%2520Val600%2520BRAF-mutant%2520melanoma%253A%2520a%2520multicentre%252C%2520double-blind%252C%2520phase%25203%2520randomised%2520controlled%2520trial%26jtitle%3DLancet%26date%3D2015%26volume%3D386%26spage%3D444%26epage%3D451%26doi%3D10.1016%2FS0140-6736%2815%2960898-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grob, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadendorf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legenne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M. A.</span></span> <span> </span><span class="NLM_article-title">Factors predictive of response, disease progression, and overall survival after dabrafenib and trametinib combination treatment: a pooled analysis of individual patient data from randomised trials</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1743</span>– <span class="NLM_lpage">1754</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(16)30578-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2FS1470-2045%2816%2930578-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=27864013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVymt77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=1743-1754&author=G.+V.+Longauthor=J.+J.+Grobauthor=P.+Nathanauthor=A.+Ribasauthor=C.+Robertauthor=D.+Schadendorfauthor=S.+R.+Laneauthor=C.+Makauthor=P.+Legenneauthor=K.+T.+Flahertyauthor=M.+A.+Davies&title=Factors+predictive+of+response%2C+disease+progression%2C+and+overall+survival+after+dabrafenib+and+trametinib+combination+treatment%3A+a+pooled+analysis+of+individual+patient+data+from+randomised+trials&doi=10.1016%2FS1470-2045%2816%2930578-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Factors predictive of response, disease progression, and overall survival after dabrafenib and trametinib combination treatment: a pooled analysis of individual patient data from randomised trials</span></div><div class="casAuthors">Long, Georgina V.; Grob, Jean-Jacques; Nathan, Paul; Ribas, Antoni; Robert, Caroline; Schadendorf, Dirk; Lane, Stephen R.; Mak, Carmen; Legenne, Philippe; Flaherty, Keith T.; Davies, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1743-1754</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dabrafenib plus trametinib treatment provides significant benefits over BRAF-inhibitor monotherapy in patients with BRAFV600E-mutant or BRAFV600K-mutant advanced melanoma; however, in many patients the disease progresses, leading to death.  With many treatment options available, understanding clin. factors that predict long-term response and survival for treatments is important for optimization of patient management.  We aimed to identify clin. factors assocd. with long-term response and survival using pooled data from randomized trials of dabrafenib plus trametinib in patients with metastatic BRAF-mutant melanoma.  We did a retrospective individual data anal. based on all published randomized trials that included treatment-naive patients with BRAFV600E-mutant or BRAFV600K-mutant metastatic melanoma who received the approved dose of dabrafenib 150 mg twice daily plus trametinib 2 mg once daily.  Data were pooled from patients in the BRF113220 (part C; March 26, 2010, to Jan 15, 2015), COMBI-d (May 4, 2012, to Jan 12, 2015), and COMBI-v (June 4, 2012, to March 13, 2015) randomized trials.  Patients with untreated brain metastases were not permitted to enroll in these trials.  Baseline factors, identified a priori based on known melanoma clin. or prognostic characteristics, were analyzed for assocn. with progression-free survival and overall survival using univariate and multivariate analyses and assessed for hierarchical effect on outcomes using regression tree analyses.  We also analyzed factors identified after baseline, on treatment, and at progression, for assocns. with survival after progression.  The trials included in this anal. are registered with ClinicalTrials.gov: BRF113220, no. NCT01072175; COMBI-d, no. NCT01584648; COMBI-v, no. NCT01597908.617 patients were included in this anal. with a median follow-up of 20·0 mo (range 0-48·0, IQR 10·1-24·8); 396 patients had progression events (ie, disease progression or death) and 290 patients had died.  Median progression-free survival (11·1 mo [95% CI 9·7-12·9]), median overall survival (25·6 mo [23·1-34·3]), 1-yr progression-free survival (48% [44-52]) and overall survival (74% [71-78]), and 2-yr progression-free survival (30% [26-34]) and overall survival (53% [49-57]) were consistent with those in the individual trials.  Patients with normal lactate dehydrogenase (LDH) concn. and fewer than three organ sites contg. metastases (n=237) had the longest 1-yr progression-free survival (68% [95% CI 62-74]) and overall survival (90% [87-94]) and 2-yr progression-free survival (46% [40-54]) and overall survival (75% [70-81]), whereas patients with LDH concn. at least two times the upper limit of normal (n=70) had the shortest 1-yr progression-free survival (8% [3-19]) and overall survival (40% [29-55]) and 2-yr progression-free survival (2% [0-13]) and overall survival (7% [3-19]).  Of patients with disease progression (n=379), survival after progression was longest in those with progression in baseline or new non-CNS lesions (n=205; median 10·0 mo [95% CI 7·9-12·0]) and shortest in those with new CNS lesions or concurrent progression in baseline and new lesions (n=171; median 4·0 mo [3·5-4·9]).  Several patient and clin. characteristics at and after baseline are assocd. with outcomes with dabrafenib plus trametinib, and durable benefit is possible with targeted treatment in defined patient subsets.Novartis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_v9J1131FDLVg90H21EOLACvtfcHk0lg24V8Ox5OZng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVymt77I&md5=8f256406a53b9ea7b258ad173b56a7ca</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2816%2930578-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252816%252930578-2%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DGrob%26aufirst%3DJ.%2BJ.%26aulast%3DNathan%26aufirst%3DP.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DSchadendorf%26aufirst%3DD.%26aulast%3DLane%26aufirst%3DS.%2BR.%26aulast%3DMak%26aufirst%3DC.%26aulast%3DLegenne%26aufirst%3DP.%26aulast%3DFlaherty%26aufirst%3DK.%2BT.%26aulast%3DDavies%26aufirst%3DM.%2BA.%26atitle%3DFactors%2520predictive%2520of%2520response%252C%2520disease%2520progression%252C%2520and%2520overall%2520survival%2520after%2520dabrafenib%2520and%2520trametinib%2520combination%2520treatment%253A%2520a%2520pooled%2520analysis%2520of%2520individual%2520patient%2520data%2520from%2520randomised%2520trials%26jtitle%3DLancet%2520Oncol.%26date%3D2016%26volume%3D17%26spage%3D1743%26epage%3D1754%26doi%3D10.1016%2FS1470-2045%2816%2930578-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span>FDA grants
regular approval to dabrafenib and trametinib combination for metastatic
NSCLC with BRAF V600E mutation (accessed June 22, 2017). fda.gov, <span class="NLM_year">2017</span>; <a href="https://www.fda.gov/drugs/resources-information-approved-drugs/fda-grants-regular-approval-dabrafenib-and-trametinib-combination-metastatic-nsclc-braf-v600e" class="extLink">https://www.fda.gov/drugs/resources-information-approved-drugs/fda-grants-regular-approval-dabrafenib-and-trametinib-combination-metastatic-nsclc-braf-v600e</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FDA+grants%0Aregular+approval+to+dabrafenib+and+trametinib+combination+for+metastatic%0ANSCLC+with+BRAF+V600E+mutation+%28accessed+June+22%2C+2017%29.+fda.gov%2C+2017%3B+https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fresources-information-approved-drugs%2Ffda-grants-regular-approval-dabrafenib-and-trametinib-combination-metastatic-nsclc-braf-v600e."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corcoran, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atreya, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellens, J. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendell, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollebecque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRee, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muro, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Dwyer, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brase, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mookerjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangwala, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Cutsem, E.</span></span> <span> </span><span class="NLM_article-title">Combined BRAF, EGFR, and MEK inhibition in patients with <i>BRAF</i><sup><i>V600E</i></sup>-mutant colorectal cancer</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">443</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-17-1226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1158%2F2159-8290.CD-17-1226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29431699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1emtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=428-443&author=R.+B.+Corcoranauthor=T.+Andreauthor=C.+E.+Atreyaauthor=J.+H.+M.+Schellensauthor=T.+Yoshinoauthor=J.+C.+Bendellauthor=A.+Hollebecqueauthor=A.+J.+McReeauthor=S.+Sienaauthor=G.+Middletonauthor=K.+Muroauthor=M.+S.+Gordonauthor=J.+Taberneroauthor=R.+Yaegerauthor=P.+J.+O%E2%80%99Dwyerauthor=Y.+Humbletauthor=F.+De+Vosauthor=A.+S.+Jungauthor=J.+C.+Braseauthor=S.+Jaegerauthor=S.+Bettingerauthor=B.+Mookerjeeauthor=F.+Rangwalaauthor=E.+Van+Cutsem&title=Combined+BRAF%2C+EGFR%2C+and+MEK+inhibition+in+patients+with+BRAFV600E-mutant+colorectal+cancer&doi=10.1158%2F2159-8290.CD-17-1226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Combined BRAF, EGFR, and MEK Inhibition in Patients with BRAFV600E-Mutant Colorectal Cancer</span></div><div class="casAuthors">Corcoran, Ryan B.; Andre, Thierry; Atreya, Chloe E.; Schellens, Jan H. M.; Yoshino, Takayuki; Bendell, Johanna C.; Hollebecque, Antoine; McRee, Autumn J.; Siena, Salvatore; Middleton, Gary; Muro, Kei; Gordon, Michael S.; Tabernero, Josep; Yaeger, Rona; O'Dwyer, Peter J.; Humblet, Yves; De Vos, Filip; Jung, A. Scott; Brase, Jan C.; Jaeger, Savina; Bettinger, Severine; Mookerjee, Bijoyesh; Rangwala, Fatima; Van Cutsem, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">428-443</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although BRAF inhibitor monotherapy yields response rates >50% in BRAFV600-mutant melanoma, only approx. 5% of patients with BRAFV600E colorectal cancer respond.  Preclin. studies suggest that the lack of efficacy in BRAFV600E colorectal cancer is due to adaptive feedback reactivation of MAPK signaling, often mediated by EGFR.  This clin. trial evaluated BRAF and EGFR inhibition with dabrafenib (D) + panitumumab (P) ± MEK inhibition with trametinib (T) to achieve greater MAPK suppression and improved efficacy in 142 patients with BRAFV600E colorectal cancer.  Confirmed response rates for D+P, D+T+P, and T+P were 10%, 21%, and 0%, resp.  Pharmacodynamic anal. of paired pretreatment and on-treatment biopsies found that efficacy of D+T+P correlated with increased MAPK suppression.  Serial cell-free DNA anal. revealed addnl. correlates of response and emergence of KRAS and NRAS mutations on disease progression.  Thus, targeting adaptive feedback pathways in BRAFV600E colorectal cancer can improve efficacy, but MAPK reactivation remains an important primary and acquired resistance mechanism.  Significance: This trial demonstrates that combined BRAF + EGFR + MEK inhibition is tolerable, with promising activity in patients with BRAFV600E colorectal cancer.  Our findings highlight the MAPK pathway as a crit. target in BRAFV600E colorectal cancer and the need to optimize strategies inhibiting this pathway to overcome both primary and acquired resistance.  Cancer Discov; 8(4); 428-43. ©2018 AACR.  See related commentary by Janku, p. 389.  See related article by Hazar-Rethinam et al., p. 417.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJr18ksu0glrVg90H21EOLACvtfcHk0lg24V8Ox5OZng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1emtLY%253D&md5=ea2b2a5036b06d4858d8deca9513e97f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-1226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-1226%26sid%3Dliteratum%253Aachs%26aulast%3DCorcoran%26aufirst%3DR.%2BB.%26aulast%3DAndre%26aufirst%3DT.%26aulast%3DAtreya%26aufirst%3DC.%2BE.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DYoshino%26aufirst%3DT.%26aulast%3DBendell%26aufirst%3DJ.%2BC.%26aulast%3DHollebecque%26aufirst%3DA.%26aulast%3DMcRee%26aufirst%3DA.%2BJ.%26aulast%3DSiena%26aufirst%3DS.%26aulast%3DMiddleton%26aufirst%3DG.%26aulast%3DMuro%26aufirst%3DK.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DTabernero%26aufirst%3DJ.%26aulast%3DYaeger%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Dwyer%26aufirst%3DP.%2BJ.%26aulast%3DHumblet%26aufirst%3DY.%26aulast%3DDe%2BVos%26aufirst%3DF.%26aulast%3DJung%26aufirst%3DA.%2BS.%26aulast%3DBrase%26aufirst%3DJ.%2BC.%26aulast%3DJaeger%26aufirst%3DS.%26aulast%3DBettinger%26aufirst%3DS.%26aulast%3DMookerjee%26aufirst%3DB.%26aulast%3DRangwala%26aufirst%3DF.%26aulast%3DVan%2BCutsem%26aufirst%3DE.%26atitle%3DCombined%2520BRAF%252C%2520EGFR%252C%2520and%2520MEK%2520inhibition%2520in%2520patients%2520with%2520BRAFV600E-mutant%2520colorectal%2520cancer%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D428%26epage%3D443%26doi%3D10.1158%2F2159-8290.CD-17-1226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eoh, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanmamed, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaftan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitrakopoulou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbst, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, J. S.</span></span> <span> </span><span class="NLM_article-title">The combination of MEK inhibitor with immunomodulatory antibodies targeting programmed death 1 and programmed death ligand 1 results in prolonged survival in <i>Kras/p53</i>-driven lung cancer</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1060</span>, <span class="refDoi"> DOI: 10.1016/j.jtho.2019.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.jtho.2019.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30771521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BB3cfkt1SgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1046-1060&author=J.+W.+Leeauthor=Y.+Zhangauthor=K.+J.+Eohauthor=R.+Sharmaauthor=M.+F.+Sanmamedauthor=J.+Wuauthor=J.+Choiauthor=H.+S.+Parkauthor=A.+Iwasakiauthor=E.+Kaftanauthor=L.+Chenauthor=V.+Papadimitrakopoulouauthor=R.+S.+Herbstauthor=J.+S.+Koo&title=The+combination+of+MEK+inhibitor+with+immunomodulatory+antibodies+targeting+programmed+death+1+and+programmed+death+ligand+1+results+in+prolonged+survival+in+Kras%2Fp53-driven+lung+cancer&doi=10.1016%2Fj.jtho.2019.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Combination of MEK Inhibitor With Immunomodulatory Antibodies Targeting Programmed Death 1 and Programmed Death Ligand 1 Results in Prolonged Survival in Kras/p53-Driven Lung Cancer</span></div><div class="casAuthors">Lee Jong Woo; Sharma Roshan; Wu Jenny; Choi Justin; Kaftan Edward; Herbst Roy S; Zhang Yu; Sanmamed Miguel F; Chen Lieping; Eoh Kyung Jin; Park Hee Sun; Iwasaki Akiko; Papadimitrakopoulou Vali; Koo Ja Seok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1046-1060</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  This study aimed to characterize the tumor-infiltrating immune cells population in Kras/tumor protein 53 (Trp53)-driven lung tumors and to evaluate the combinatorial antitumor effect with MEK inhibitor (MEKi), trametinib, and immunomodulatory monoclonal antibodies (mAbs) targeting either programmed death -1 (PD-1) or programmed cell death ligand 1 (PD-L1) in vivo.  METHODS:  Trp53(FloxFlox);Kras(G12D/+);Rosa26(LSL-Luciferase/LSL-Luciferase) (PKL) genetically engineered mice were used to develop autochthonous lung tumors with intratracheal delivery of adenoviral Cre recombinase.  Using these tumor-bearing lungs, tumor-infiltrating immune cells were characterized by both mass cytometry and flow cytometry.  PKL-mediated immunocompetent syngeneic and transgenic lung cancer mouse models were treated with MEKi alone as well as in combination with either anti-PD-1 or anti-PD-L1 mAbs.  Tumor growth and survival outcome were assessed.  Finally, immune cell populations within spleens and tumors were evaluated by flow cytometry and immunohistochemistry.  RESULTS:  Myeloid-derived suppressor cells (MDSCs) were significantly augmented in PKL-driven lung tumors compared to normal lungs of tumor-free mice.  PD-L1 expression appeared to be highly positive in both lung tumor cells and, particularly MDSCs.  The combinatory administration of MEKi with either anti-PD-1 or anti-PD-L1 mAbs synergistically increased antitumor response and survival outcome compared with single-agent therapy in both the PKL-mediated syngeneic and transgenic lung cancer models.  Theses combinational treatments resulted in significant increases of tumor-infiltrating CD8(+) and CD4(+) T cells, whereas attenuation of CD11b(+)/Gr-1(high) MDSCs, in particular, Ly6G(high) polymorphonuclear-MDSCs in the syngeneic model.  CONCLUSIONS:  These findings suggest a potential therapeutic approach for untargetable Kras/p53-driven lung cancers with synergy between targeted therapy using MEKi and immunotherapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQYgLfJG8xGHBWfpMXQLRstfW6udTcc2ebS7a9FIVPWhbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfkt1SgtA%253D%253D&md5=59df9199c1c21e5fe8627a4b270f7c90</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.jtho.2019.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtho.2019.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DEoh%26aufirst%3DK.%2BJ.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DSanmamed%26aufirst%3DM.%2BF.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DH.%2BS.%26aulast%3DIwasaki%26aufirst%3DA.%26aulast%3DKaftan%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DPapadimitrakopoulou%26aufirst%3DV.%26aulast%3DHerbst%26aufirst%3DR.%2BS.%26aulast%3DKoo%26aufirst%3DJ.%2BS.%26atitle%3DThe%2520combination%2520of%2520MEK%2520inhibitor%2520with%2520immunomodulatory%2520antibodies%2520targeting%2520programmed%2520death%25201%2520and%2520programmed%2520death%2520ligand%25201%2520results%2520in%2520prolonged%2520survival%2520in%2520Kras%252Fp53-driven%2520lung%2520cancer%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2019%26volume%3D14%26spage%3D1046%26epage%3D1060%26doi%3D10.1016%2Fj.jtho.2019.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pupo, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahheydari, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tembe, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saw, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kefford, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizos, H.</span></span> <span> </span><span class="NLM_article-title">Increased MAPK reactivation in early resistance to dabrafenib/trametinib combination therapy of BRAF-mutant metastatic melanoma</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1038/ncomms6694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fncomms6694" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1-9&author=G.+V.+Longauthor=C.+Fungauthor=A.+M.+Menziesauthor=G.+M.+Pupoauthor=M.+S.+Carlinoauthor=J.+Hymanauthor=H.+Shahheydariauthor=V.+Tembeauthor=J.+F.+Thompsonauthor=R.+P.+Sawauthor=J.+Howleauthor=N.+K.+Haywardauthor=P.+Johanssonauthor=R.+A.+Scolyerauthor=R.+F.+Keffordauthor=H.+Rizos&title=Increased+MAPK+reactivation+in+early+resistance+to+dabrafenib%2Ftrametinib+combination+therapy+of+BRAF-mutant+metastatic+melanoma&doi=10.1038%2Fncomms6694"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fncomms6694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6694%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DFung%26aufirst%3DC.%26aulast%3DMenzies%26aufirst%3DA.%2BM.%26aulast%3DPupo%26aufirst%3DG.%2BM.%26aulast%3DCarlino%26aufirst%3DM.%2BS.%26aulast%3DHyman%26aufirst%3DJ.%26aulast%3DShahheydari%26aufirst%3DH.%26aulast%3DTembe%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DJ.%2BF.%26aulast%3DSaw%26aufirst%3DR.%2BP.%26aulast%3DHowle%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DN.%2BK.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DKefford%26aufirst%3DR.%2BF.%26aulast%3DRizos%26aufirst%3DH.%26atitle%3DIncreased%2520MAPK%2520reactivation%2520in%2520early%2520resistance%2520to%2520dabrafenib%252Ftrametinib%2520combination%2520therapy%2520of%2520BRAF-mutant%2520metastatic%2520melanoma%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D1%26epage%3D9%26doi%3D10.1038%2Fncomms6694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriceau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hugo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koya, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samatar, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanlou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchalski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seifert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlman, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, R. S.</span></span> <span> </span><span class="NLM_article-title">Tunable-combinatorial mechanisms of acquired resistance limit the efficacy of BRAF/MEK cotargeting but result in melanoma drug addiction</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2014.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ccell.2014.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=25600339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=240-256&author=G.+Moriceauauthor=W.+Hugoauthor=A.+Hongauthor=H.+Shiauthor=X.+Kongauthor=C.+C.+Yuauthor=R.+C.+Koyaauthor=A.+A.+Samatarauthor=N.+Khanlouauthor=J.+Braunauthor=K.+Ruchalskiauthor=H.+Seifertauthor=J.+Larkinauthor=K.+B.+Dahlmanauthor=D.+B.+Johnsonauthor=A.+Algaziauthor=J.+A.+Sosmanauthor=A.+Ribasauthor=R.+S.+Lo&title=Tunable-combinatorial+mechanisms+of+acquired+resistance+limit+the+efficacy+of+BRAF%2FMEK+cotargeting+but+result+in+melanoma+drug+addiction&doi=10.1016%2Fj.ccell.2014.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Tunable-Combinatorial Mechanisms of Acquired Resistance Limit the Efficacy of BRAF/MEK Cotargeting but Result in Melanoma Drug Addiction</span></div><div class="casAuthors">Moriceau, Gatien; Hugo, Willy; Hong, Aayoung; Shi, Hubing; Kong, Xiangju; Yu, Clarissa C.; Koya, Richard C.; Samatar, Ahmed A.; Khanlou, Negar; Braun, Jonathan; Ruchalski, Kathleen; Seifert, Heike; Larkin, James; Dahlman, Kimberly B.; Johnson, Douglas B.; Algazi, Alain; Sosman, Jeffrey A.; Ribas, Antoni; Lo, Roger S.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">240-256</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Combined BRAF- and MEK-targeted therapy improves upon BRAF inhibitor (BRAFi) therapy but is still beset by acquired resistance.  We show that melanomas acquire resistance to combined BRAF and MEK inhibition by augmenting or combining mechanisms of single-agent BRAFi resistance.  These double-drug resistance-assocd. genetic configurations significantly altered mol. interactions underlying MAPK pathway reactivation. V600EBRAF, expressed at supraphysiol. levels because of V600EBRAF ultra-amplification, dimerized with and activated CRAF.  In addn., MEK mutants enhanced interaction with overexpressed V600EBRAF via a regulatory interface at R662 of V600EBRAF.  Importantly, melanoma cell lines selected for resistance to BRAFi+MEKi, but not those to BRAFi alone, displayed robust drug addiction, providing a potentially exploitable therapeutic opportunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWdJyJ2wlmALVg90H21EOLACvtfcHk0lgBfsSgND_7fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKgtrg%253D&md5=42788605b6c22cf70ff49304bf65ca19</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2014.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2014.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DMoriceau%26aufirst%3DG.%26aulast%3DHugo%26aufirst%3DW.%26aulast%3DHong%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DC.%2BC.%26aulast%3DKoya%26aufirst%3DR.%2BC.%26aulast%3DSamatar%26aufirst%3DA.%2BA.%26aulast%3DKhanlou%26aufirst%3DN.%26aulast%3DBraun%26aufirst%3DJ.%26aulast%3DRuchalski%26aufirst%3DK.%26aulast%3DSeifert%26aufirst%3DH.%26aulast%3DLarkin%26aufirst%3DJ.%26aulast%3DDahlman%26aufirst%3DK.%2BB.%26aulast%3DJohnson%26aufirst%3DD.%2BB.%26aulast%3DAlgazi%26aufirst%3DA.%26aulast%3DSosman%26aufirst%3DJ.%2BA.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DLo%26aufirst%3DR.%2BS.%26atitle%3DTunable-combinatorial%2520mechanisms%2520of%2520acquired%2520resistance%2520limit%2520the%2520efficacy%2520of%2520BRAF%252FMEK%2520cotargeting%2520but%2520result%2520in%2520melanoma%2520drug%2520addiction%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D240%26epage%3D256%26doi%3D10.1016%2Fj.ccell.2014.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0liz72UA1fywZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>474</i></span>,  <span class="NLM_fpage">1127</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1042/BCJ20160762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1042%2FBCJ20160762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=28298557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1127-1147&author=I.+Collinsauthor=H.+Wangauthor=J.+J.+Caldwellauthor=R.+Chopra&title=Chemical+approaches+to+targeted+protein+degradation+through+modulation+of+the+ubiquitin-proteasome+pathway&doi=10.1042%2FBCJ20160762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span></div><div class="casAuthors">Collins, Ian; Wang, Hannah; Caldwell, John J.; Chopra, Raj</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1127-1147</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Manipulation of the ubiquitin-proteasome system to achieve targeted degrdn. of proteins within cells using chem. tools and drugs has the potential to transform pharmacol. and therapeutic approaches in cancer and other diseases.  An increased understanding of the mol. mechanism of thalidomide and its analogs following their clin. use has unlocked small-mol. modulation of the substrate specificity of the E3 ligase cereblon (CRBN), which in turn has resulted in the advancement of new immunomodulatory drugs (IMiDs) into the clinic.  The degrdn. of multiple context-specific proteins by these pleiotropic small mols. provides a means to uncover new cell biol. and to generate future drug mols. against currently undruggable targets.  In parallel, the development of larger bifunctional mols. that bring together highly specific protein targets in complexes with CRBN, von Hippel-Lindau, or other E3 ligases to promote ubiquitin-dependent degrdn. has progressed to generate selective chem. compds. with potent effects in cells and in vivo models, providing valuable tools for biol. target validation and with future potential for therapeutic use.  In this review, we survey recent breakthroughs achieved in these two complementary methods and the discovery of new modes of direct and indirect engagement of target proteins with the proteasome.  We discuss the exptl. characterization that validates the use of mols. that promote protein degrdn. as chem. tools, the preclin. and clin. examples disclosed to date, and the future prospects for this exciting area of chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkoPeMHzjTbVg90H21EOLACvtfcHk0liz72UA1fywZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D&md5=8c9ccac22f5088d776b81203dc8ebf2b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160762%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DChemical%2520approaches%2520to%2520targeted%2520protein%2520degradation%2520through%2520modulation%2520of%2520the%2520ubiquitin-proteasome%2520pathway%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26spage%3D1127%26epage%3D1147%26doi%3D10.1042%2FBCJ20160762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0liz72UA1fywZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span> <span> </span><span class="NLM_article-title">Cloning and characterization of two distinct human extracellular signal-regulated kinase activator kinases, MEK1 and MEK2</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">11435</span>– <span class="NLM_lpage">11439</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=8388392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADyaK3sXks1KmsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=11435-11439&author=C.+F.+Zhengauthor=K.+L.+Guan&title=Cloning+and+characterization+of+two+distinct+human+extracellular+signal-regulated+kinase+activator+kinases%2C+MEK1+and+MEK2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Cloning and characterization of two distinct human extracellular signal-regulated kinase activator kinases, MEK1 and MEK2</span></div><div class="casAuthors">Zheng, Chao Feng; Guan, Kun Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">11435-9</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Mitogen-induced signal transduction is mediated by a cascade of protein phosphorylation and dephosphorylation.  One of the immediate responses of mitogen stimulation is the activation of a family of protein kinases known as mitogen-activated protein kinases or extracellular signal-regulated kinase (ERK).  MEK (MAP kinase or ERK kinase) is the immediate upstream activator kinase of ERK.  Two cDNAs, MEK1 and MEK2, were cloned and sequenced.  MEK1 and MEK2 encode 393 and 400 amino acid residues, resp.  The human MEK1 shares 99% amino acid sequence identity with the murine MEK1 and 80% with human MEK2.  Both MEK1 and MEK2 were expressed in Escherichia coli and shown to be able to activate recombinant human ERK1 in vitro.  The purified MEK2 protein stimulated threonine and tyrosine phosphorylation on ERK1 and concomitantly activated ERK1 kinase activity more than 100-fold.  The recombinant MEK2 showed lower activity as an ERK activator as compared with MEK purified from tissue.  However, the recombinant MEK2 can be activated by serum-stimulated cell ext. in vitro.  MEKs, in a manner similar to ERKs, are likely to consist of a family of related proteins playing crit. roles in signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmDkRVD-cBq7Vg90H21EOLACvtfcHk0liz72UA1fywZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1KmsLk%253D&md5=7c149c9378198dd108ce2736e3c55138</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DC.%2BF.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26atitle%3DCloning%2520and%2520characterization%2520of%2520two%2520distinct%2520human%2520extracellular%2520signal-regulated%2520kinase%2520activator%2520kinases%252C%2520MEK1%2520and%2520MEK2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D11435%26epage%3D11439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fremin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meloche, S.</span></span> <span> </span><span class="NLM_article-title">From basic research to clinical development of MEK1/2 inhibitors for cancer therapy</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1186/1756-8722-3-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1186%2F1756-8722-3-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=20051128" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=1-11&author=C.+Freminauthor=S.+Meloche&title=From+basic+research+to+clinical+development+of+MEK1%2F2+inhibitors+for+cancer+therapy&doi=10.1186%2F1756-8722-3-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2F1756-8722-3-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8722-3-8%26sid%3Dliteratum%253Aachs%26aulast%3DFremin%26aufirst%3DC.%26aulast%3DMeloche%26aufirst%3DS.%26atitle%3DFrom%2520basic%2520research%2520to%2520clinical%2520development%2520of%2520MEK1%252F2%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2010%26volume%3D3%26spage%3D1%26epage%3D11%26doi%3D10.1186%2F1756-8722-3-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Targeting ERK1/2 protein-serine/threonine kinases in human cancers</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">168</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2019.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.phrs.2019.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30794926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktFagurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2019&pages=151-168&author=R.+Roskoski&title=Targeting+ERK1%2F2+protein-serine%2Fthreonine+kinases+in+human+cancers&doi=10.1016%2Fj.phrs.2019.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting ERK1/2 protein-serine/threonine kinases in human cancers</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">151-168</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  ERK1 and ERK2 are key protein kinases that contribute to the Ras-Raf-MEK-ERK MAP kinase signalling module.  This pathway participates in the control of numerous processes including apoptosis, cell proliferation, the immune response, nervous system function, and RNA synthesis and processing.  MEK1/2 activate human ERK1/2 by first catalyzing the phosphorylation of Y204/187 and then T202/185, both residues of which occur within the activation segment.  The phosphorylation of both residues is required for enzyme activation.  The only Raf substrates are MEK1/2 and the only MEK1/2 substrates are ERK1/2.  In contrast, ERK1/2 catalyze the phosphorylation of many cytoplasmic and nuclear substrates including transcription factors and regulatory mols.  The linear MAP kinase pathway branches extensively at the ERK1/2 node.  ERK1/2 are proline-directed kinases that preferentially catalyze the phosphorylation of substrates contg. a PxS/TP sequence.  The dephosphorylation and inactivation of ERK1/2 is catalyzed by dual specificity phosphatases, protein-tyrosine specific phosphatases, and protein-serine/threonine phosphatases.  The combined functions of kinases and phosphatases make the overall process reversible.  To provide an idea of the complexities involved in these reactions, somatic cell cycle progression involves the strict timing of more than 32,000 phosphorylation and dephosphorylation events as detd. by mass spectrometry.  The MAP kinase cascade is perhaps the most important oncogenic driver of human cancers and the blockade of this signalling module by targeted inhibitors is an important anti-tumor strategy.  Although numerous cancers are driven by MAP kinase pathway activation, thus far the only orally effective approved drugs that target this signaling module are used for the treatment of BRAF-mutant melanomas.  The best treatments include the combination of B-Raf and MEK inhibitors (dabrafenib and trametinib, encorafenib and binimetinib, vemurafenib and cobimetanib).  However, resistance to these antagonists occurs within one year and addnl. treatment options are necessary.  Owing to the large variety of malignancies that are driven by dysregulation of the MAP kinase pathway, addnl. tumor types should be amenable to MAP kinase pathway inhibitor therapy.  In addn. to new B-Raf and MEK inhibitors, the addn. of ERK inhibitors should prove helpful.  Ulixertinib, MK-8353, and GDC-0994 are orally effective, potent, and specific inhibitors of ERK1/2 that are in early clin. trials for the treatment of various advanced/metastatic solid tumors.  These agents are effective against cell lines that are resistant to B-Raf and MEK1/2 inhibitor therapy.  Although MK-8353 does not directly inhibit MEK1/2, it decreases the phosphorylation of ERK1/2 as well as the phosphorylation of RSK, an ERK1/2 substrate.  The decrease in RSK phosphorylation appears to be a result of ERK inhibition and the decrease in ERK1/2 phosphorylation is related to the inability of MEK to catalyze the phosphorylation of the ERK-MK-8353 complex; these decreases characterize the ERK dual mechanism inhibition paradigm.  Addnl. work will be required to det. whether ERK inhibitors will be successful in the clinic and are able to forestall the development of drug resistance of the MAP kinase pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJOqTxpKRfHLVg90H21EOLACvtfcHk0lgJMO9wtAk-Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktFagurs%253D&md5=75743a80b77c8f1b49dc59de7bbc2b7f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2019.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2019.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DTargeting%2520ERK1%252F2%2520protein-serine%252Fthreonine%2520kinases%2520in%2520human%2520cancers%26jtitle%3DPharmacol.%2520Res.%26date%3D2019%26volume%3D142%26spage%3D151%26epage%3D168%26doi%3D10.1016%2Fj.phrs.2019.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgermeister, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuderland, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanoch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liscovitch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seger, R.</span></span> <span> </span><span class="NLM_article-title">Interaction with MEK causes nuclear export and downregulation of peroxisome proliferator-activated receptor gamma</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">803</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1128/MCB.00601-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1128%2FMCB.00601-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=17101779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=803-817&author=E.+Burgermeisterauthor=D.+Chuderlandauthor=T.+Hanochauthor=M.+Meyerauthor=M.+Liscovitchauthor=R.+Seger&title=Interaction+with+MEK+causes+nuclear+export+and+downregulation+of+peroxisome+proliferator-activated+receptor+gamma&doi=10.1128%2FMCB.00601-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction with MEK causes nuclear export and downregulation of peroxisome proliferator-activated receptor</span></div><div class="casAuthors">Burgermeister, Elke; Chuderland, Dana; Hanoch, Tamar; Meyer, Markus; Liscovitch, Mordechai; Seger, Rony</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">803-817</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mitogen-activated protein kinase (MAPK)/extracellular signal-regulated kinase (ERK) cascade plays a central role in intracellular signaling by many extracellular stimuli.  One target of the ERK cascade is peroxisome proliferator-activated receptor γ (PPARγ), a nuclear receptor that promotes differentiation and apoptosis.  It was previously demonstrated that PPARγ activity is attenuated upon mitogenic stimulation due to phosphorylation of its Ser84 by ERKs.  Here we show that stimulation by tetradecanoyl phorbol acetate (TPA) attenuates PPARγ's activity in a MEK-dependent manner, even when Ser84 is mutated to Ala.  To elucidate the mechanism of attenuation, we found that PPARγ directly interacts with MEKs, which are the activators of ERKs, but not with ERKs themselves, both in vivo and in vitro.  This interaction is facilitated by MEKs' phosphorylation and is mediated by the basic D domain of MEK1 and the AF2 domain of PPARγ.  Immunofluorescence microscopy and subcellular fractionation revealed that MEK1 exports PPARγ from the nucleus, and this finding was supported by small interfering RNA knockdown of MEK1 and use of a cell-permeable interaction-blocking peptide, which prevented TPA-induced export of PPARγ from the nucleus.  Thus, we show here a novel mode of downregulation of PPARγ by its MEK-dependent redistribution from the nucleus to the cytosol.  This unanticipated role for the stimulation-induced nuclear shuttling of MEKs shows that MEKs can regulate addnl. signaling components besides the ERK cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyBt7HomvWmLVg90H21EOLACvtfcHk0lgJMO9wtAk-Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGht7s%253D&md5=60b9cd18e1c5ac33122ea01c109fcd02</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1128%2FMCB.00601-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00601-06%26sid%3Dliteratum%253Aachs%26aulast%3DBurgermeister%26aufirst%3DE.%26aulast%3DChuderland%26aufirst%3DD.%26aulast%3DHanoch%26aufirst%3DT.%26aulast%3DMeyer%26aufirst%3DM.%26aulast%3DLiscovitch%26aufirst%3DM.%26aulast%3DSeger%26aufirst%3DR.%26atitle%3DInteraction%2520with%2520MEK%2520causes%2520nuclear%2520export%2520and%2520downregulation%2520of%2520peroxisome%2520proliferator-activated%2520receptor%2520gamma%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D803%26epage%3D817%26doi%3D10.1128%2FMCB.00601-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahmi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maisonneuve, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marullo, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thevakumaran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurinov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, M.</span></span> <span> </span><span class="NLM_article-title">MEK drives BRAF activation through allosteric control of KSR proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>554</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1038/nature25478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnature25478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29433126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1SltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=554&publication_year=2018&pages=549-553&author=H.+Lavoieauthor=M.+Sahmiauthor=P.+Maisonneuveauthor=S.+A.+Marulloauthor=N.+Thevakumaranauthor=T.+Jinauthor=I.+Kurinovauthor=F.+Sicheriauthor=M.+Therrien&title=MEK+drives+BRAF+activation+through+allosteric+control+of+KSR+proteins&doi=10.1038%2Fnature25478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">MEK drives BRAF activation through allosteric control of KSR proteins</span></div><div class="casAuthors">Lavoie, Hugo; Sahmi, Malha; Maisonneuve, Pierre; Marullo, Sara A.; Thevakumaran, Neroshan; Jin, Ting; Kurinov, Igor; Sicheri, Frank; Therrien, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">554</span>
        (<span class="NLM_cas:issue">7693</span>),
    <span class="NLM_cas:pages">549-553</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">RAF family kinases have prominent roles in cancer.  Their activation is dependent on dimerization of their kinase domains, which has emerged as a hindrance for drug development.  In mammals, RAF family kinases include three catalytically competent enzymes (ARAF, BRAF and CRAF) and two pseudokinases (KSR1 and KSR2) that have been described as scaffolds owing to their apparent ability to bridge RAF isoforms and their substrate, mitogen-activated protein kinase kinase (MEK).  Kinase suppressor of Ras (KSR) pseudokinases were also shown to dimerize with kinase-competent RAFs to stimulate catalysis allosterically.  Although GTP-bound RAS can modulate the dimerization of RAF isoforms by engaging their RAS-binding domains, KSR1 and KSR2 lack an RAS-binding domain and therefore the regulatory principles underlying their dimerization with other RAF family members remain unknown.  Here we show that the selective heterodimerization of BRAF with KSR1 is specified by direct contacts between the amino-terminal regulatory regions of each protein, comprising in part a novel domain called BRS in BRAF and the coiled-coil-sterile α motif (CC-SAM) domain in KSR1.  We also discovered that MEK binding to the kinase domain of KSR1 asym. drives BRAF-KSR1 heterodimerization, resulting in the concomitant stimulation of BRAF catalytic activity towards free MEK mols.  These findings demonstrate that KSR-MEK complexes allosterically activate BRAF through the action of N-terminal regulatory region and kinase domain contacts and challenge the accepted role of KSR as a scaffold for MEK recruitment to RAF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprMdHlsp1vWbVg90H21EOLACvtfcHk0ljdZSCFA0HfaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1SltLY%253D&md5=8b50458212b822a6c9caefda2b08aa85</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnature25478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25478%26sid%3Dliteratum%253Aachs%26aulast%3DLavoie%26aufirst%3DH.%26aulast%3DSahmi%26aufirst%3DM.%26aulast%3DMaisonneuve%26aufirst%3DP.%26aulast%3DMarullo%26aufirst%3DS.%2BA.%26aulast%3DThevakumaran%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DT.%26aulast%3DKurinov%26aufirst%3DI.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DTherrien%26aufirst%3DM.%26atitle%3DMEK%2520drives%2520BRAF%2520activation%2520through%2520allosteric%2520control%2520of%2520KSR%2520proteins%26jtitle%3DNature%26date%3D2018%26volume%3D554%26spage%3D549%26epage%3D553%26doi%3D10.1038%2Fnature25478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blasco, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francoz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santamaria, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canamero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubus, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baccarini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">c-Raf, but not B-Raf, is essential for development of K-Ras oncogene-driven non-small cell lung carcinoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">652</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2011.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ccr.2011.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=21514245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFGls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=652-663&author=R.+B.+Blascoauthor=S.+Francozauthor=D.+Santamariaauthor=M.+Canameroauthor=P.+Dubusauthor=J.+Charronauthor=M.+Baccariniauthor=M.+Barbacid&title=c-Raf%2C+but+not+B-Raf%2C+is+essential+for+development+of+K-Ras+oncogene-driven+non-small+cell+lung+carcinoma&doi=10.1016%2Fj.ccr.2011.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">c-Raf, but Not B-Raf, Is Essential for Development of K-Ras Oncogene-Driven Non-Small Cell Lung Carcinoma</span></div><div class="casAuthors">Blasco, Rafael B.; Francoz, Sarah; Santamaria, David; Canamero, Marta; Dubus, Pierre; Charron, Jean; Baccarini, Manuela; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">652-663</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We have investigated the role of individual members of the Raf/Mek/Erk cascade in the onset of K-Ras oncogene-driven non-small cell lung carcinoma (NSCLC).  Ablation of Erk1 or Erk2 in K-Ras oncogene-expressing lung cells had no significant effect due to compensatory activities.  Yet, elimination of both Erk kinases completely blocked tumor development.  Similar results were obtained with Mek kinases.  Ablation of B-Raf had no significant effect on tumor development.  However, c-Raf expression was absolutely essential for the onset of NSCLC.  Interestingly, concomitant elimination of c-Raf and B-Raf in adult mice had no deleterious consequences for normal homeostasis.  These results indicate that c-Raf plays a unique role in mediating K-Ras signaling and makes it a suitable target for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCWweCotG3S7Vg90H21EOLACvtfcHk0ljdZSCFA0HfaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFGls7c%253D&md5=11e9470b124d1c3ba6afe7e930f8438f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DBlasco%26aufirst%3DR.%2BB.%26aulast%3DFrancoz%26aufirst%3DS.%26aulast%3DSantamaria%26aufirst%3DD.%26aulast%3DCanamero%26aufirst%3DM.%26aulast%3DDubus%26aufirst%3DP.%26aulast%3DCharron%26aufirst%3DJ.%26aulast%3DBaccarini%26aufirst%3DM.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3Dc-Raf%252C%2520but%2520not%2520B-Raf%252C%2520is%2520essential%2520for%2520development%2520of%2520K-Ras%2520oncogene-driven%2520non-small%2520cell%2520lung%2520carcinoma%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D19%26spage%3D652%26epage%3D663%26doi%3D10.1016%2Fj.ccr.2011.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebraud, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Protein degradation by in-cell self-assembly of proteolysis targeting chimeras</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">934</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.6b00280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.6b00280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=927-934&author=H.+Lebraudauthor=D.+J.+Wrightauthor=C.+N.+Johnsonauthor=T.+D.+Heightman&title=Protein+degradation+by+in-cell+self-assembly+of+proteolysis+targeting+chimeras&doi=10.1021%2Facscentsci.6b00280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Degradation by In-Cell Self-Assembly of Proteolysis Targeting Chimeras</span></div><div class="casAuthors">Lebraud, Honorine; Wright, David J.; Johnson, Christopher N.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">927-934</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective degrdn. of proteins by proteolysis targeting chimeras (PROTACs) offers a promising potential alternative to protein inhibition for therapeutic intervention.  Current PROTAC mols. incorporate a ligand for the target protein, a linker, and an E3 ubiquitin ligase recruiting group, which bring together target protein and ubiquitinating machinery.  Such hetero-bifunctional mols. require significant linker optimization and possess high mol. wt., which can limit cellular permeation, soly., and other drug-like properties.  We show here that the hetero-bifunctional mol. can be formed intracellularly by bio-orthogonal click combination of two smaller precursors.  We designed a tetrazine tagged thalidomide deriv. which reacts rapidly with a trans-cyclo-octene tagged ligand of the target protein in cells to form a cereblon E3 ligase recruiting PROTAC mol.  The in-cell click-formed proteolysis targeting chimeras (CLIPTACs) were successfully used to degrade two key oncol. targets, BRD4 and ERK1/2.  ERK1/2 degrdn. was achieved using a CLIPTAC based on a covalent inhibitor.  We expect this approach to be readily extendable to other inhibitor-protein systems because the tagged E3 ligase recruiter is capable of undergoing the click reaction with a suitably tagged ligand of any protein of interest to elicit its degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRnQIoT7hbt7Vg90H21EOLACvtfcHk0ljdZSCFA0HfaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ&md5=4cc9b4aaba6ee03cd2d4ea97a72728a5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.6b00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.6b00280%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DWright%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DProtein%2520degradation%2520by%2520in-cell%2520self-assembly%2520of%2520proteolysis%2520targeting%2520chimeras%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2016%26volume%3D2%26spage%3D927%26epage%3D934%26doi%3D10.1021%2Facscentsci.6b00280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gou, S.</span></span> <span> </span><span class="NLM_article-title">Pomalidomide hybrids act as proteolysis targeting chimeras: synthesis, anticancer activity and B-Raf degradation</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2019.03.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.bioorg.2019.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30901674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1ehsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2019&pages=191-199&author=H.+Chenauthor=F.+Chenauthor=S.+Peiauthor=S.+Gou&title=Pomalidomide+hybrids+act+as+proteolysis+targeting+chimeras%3A+synthesis%2C+anticancer+activity+and+B-Raf+degradation&doi=10.1016%2Fj.bioorg.2019.03.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Pomalidomide hybrids act as proteolysis targeting chimeras: Synthesis, anticancer activity and B-Raf degradation</span></div><div class="casAuthors">Chen, Hong; Chen, Feihong; Pei, Sinan; Gou, Shaohua</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">191-199</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As the first intracellular signaling mol. and the most frequently mutated oncogene, B-Raf represents an important target in cancer therapy.  Here we report several pomalidomide hybrids acting as proteolysis targeting chimeras (PROTACs) for the degrdn. of B-Raf.  Due to its high expression of B-Raf, MCF-7 cells are sensitive to these compds.  Among them, compd. 2 can effectively kill cancer cells via inducing cells apoptosis.  As a B-Raf degrader, compd. 2 can accelerate the degrdn. of B-Raf by recruiting ubiquitin-proteasome system, and further affects the expression of Mcl-1, a downstream protein of B-Raf.  The anticancer mechanism of compd. 2 is quite different from its mother compd. and cancer cells seem to be more sensitive to the degrader, hinting that degrdn. of B-Raf by PROTAC is a potential way for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQqYbDk5Vs5LVg90H21EOLACvtfcHk0ljdZSCFA0HfaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXls1ehsLY%253D&md5=0d717c1e43a72d5213857ed47dd24a57</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2019.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2019.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DPei%26aufirst%3DS.%26aulast%3DGou%26aufirst%3DS.%26atitle%3DPomalidomide%2520hybrids%2520act%2520as%2520proteolysis%2520targeting%2520chimeras%253A%2520synthesis%252C%2520anticancer%2520activity%2520and%2520B-Raf%2520degradation%26jtitle%3DBioorg.%2520Chem.%26date%3D2019%26volume%3D87%26spage%3D191%26epage%3D199%26doi%3D10.1016%2Fj.bioorg.2019.03.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Overcoming resistance to targeted anticancer therapies through small-molecule-mediated MEK degradation</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">996</span>– <span class="NLM_lpage">1005</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.chembiol.2018.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29909991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCqtLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=996-1005&author=J.+Pehauthor=M.+W.+Boudreauauthor=H.+M.+Smithauthor=P.+J.+Hergenrother&title=Overcoming+resistance+to+targeted+anticancer+therapies+through+small-molecule-mediated+MEK+degradation&doi=10.1016%2Fj.chembiol.2018.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming Resistance to Targeted Anticancer Therapies through Small-Molecule-Mediated MEK Degradation</span></div><div class="casAuthors">Peh, Jessie; Boudreau, Matthew W.; Smith, Hannah M.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">996-1005.e4</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The discovery of mutant or fusion kinases that drive oncogenesis, and the subsequent approval of specific inhibitors for these enzymes, has been instrumental in the management of some cancers.  However, acquired resistance remains a significant problem in the clinic, limiting the long-term effectiveness of most of these drugs.  Here we demonstrate a general strategy to overcome this resistance through drug-induced MEK cleavage (via direct procaspase-3 activation) combined with targeted kinase inhibition.  This combination effect is shown to be general across diverse tumor histologies (melanoma, lung cancer, and leukemia) and driver mutations (mutant BRAF or EGFR, fusion kinases EML4-ALK and BCR-ABL).  Caspase-3-mediated degrdn. of MEK kinases results in sustained pathway inhibition and substantially delayed or eliminated resistance in cancer cells in a manner far superior to combinations with MEK inhibitors.  These data suggest the generality of drug-mediated MEK kinase cleavage as a therapeutic strategy to prevent resistance to targeted anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopoZHGk_4QDbVg90H21EOLACvtfcHk0lhb6HSEULbn9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCqtLbN&md5=83d3b1df9388e9de083ff139e452b2f6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DPeh%26aufirst%3DJ.%26aulast%3DBoudreau%26aufirst%3DM.%2BW.%26aulast%3DSmith%26aufirst%3DH.%2BM.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DOvercoming%2520resistance%2520to%2520targeted%2520anticancer%2520therapies%2520through%2520small-molecule-mediated%2520MEK%2520degradation%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D996%26epage%3D1005%26doi%3D10.1016%2Fj.chembiol.2018.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.+Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lhb6HSEULbn9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span> <span> </span><span class="NLM_article-title">Current development status of MEK inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.3390/molecules22101551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.3390%2Fmolecules22101551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1-20&author=Y.+Chengauthor=H.+Tian&title=Current+development+status+of+MEK+inhibitors&doi=10.3390%2Fmolecules22101551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.3390%2Fmolecules22101551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules22101551%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DH.%26atitle%3DCurrent%2520development%2520status%2520of%2520MEK%2520inhibitors%26jtitle%3DMolecules%26date%3D2017%26volume%3D22%26spage%3D1%26epage%3D20%26doi%3D10.3390%2Fmolecules22101551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohren, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuffa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spessard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banotai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudley, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasemann, C. A.</span></span> <span> </span><span class="NLM_article-title">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1192</span>– <span class="NLM_lpage">1197</span>, <span class="refDoi"> DOI: 10.1038/nsmb859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnsmb859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=15543157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1192-1197&author=J.+F.+Ohrenauthor=H.+Chenauthor=A.+Pavlovskyauthor=C.+Whiteheadauthor=E.+Zhangauthor=P.+Kuffaauthor=C.+Yanauthor=P.+McConnellauthor=C.+Spessardauthor=C.+Banotaiauthor=W.+T.+Muellerauthor=A.+Delaneyauthor=C.+Omerauthor=J.+Sebolt-Leopoldauthor=D.+T.+Dudleyauthor=I.+K.+Leungauthor=C.+Flammeauthor=J.+Warmusauthor=M.+Kaufmanauthor=S.+Barrettauthor=H.+Tecleauthor=C.+A.+Hasemann&title=Structures+of+human+MAP+kinase+kinase+1+%28MEK1%29+and+MEK2+describe+novel+noncompetitive+kinase+inhibition&doi=10.1038%2Fnsmb859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span></div><div class="casAuthors">Ohren, Jeffrey F.; Chen, Huifen; Pavlovsky, Alexander; Whitehead, Christopher; Zhang, Erli; Kuffa, Peter; Yan, Chunhong; McConnell, Patrick; Spessard, Cindy; Banotai, Craig; Mueller, W. Thomas; Delaney, Amy; Omer, Charles; Sebolt-Leopold, Judith; Dudley, David T.; Leung, Iris K.; Flamme, Cathlin; Warmus, Joseph; Kaufman, Michael; Barrett, Stephen; Tecle, Haile; Hasemann, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1192-1197</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MEK1 and MEK2 kinases are closely related, dual-specificity tyrosine/threonine protein kinases found in the Ras/Raf/MEK/ERK MAP kinase (MAPK) signaling pathway.  Approx. 30% of all human cancers have a constitutively activated MAPK pathway, and constitutive activation of MEK1 results in cellular transformation.  Here, the authors present the x-ray crystal structures of human MEK1 and MEK2 kinases, each detd. as a ternary complex with MgATP and an inhibitor to a resoln. of 2.4 and 3.2 Å, resp.  The structures revealed that MEK1 and MEK2 each had a unique inhibitor-binding pocket adjacent to the MgATP-binding site.  The presence of the potent inhibitors induced several conformational changes in the unphosphorylated MEK1 and MEK2 kinases that locked them into a closed but catalytically inactive species.  Thus, the structures reported here reveal a novel, noncompetitive mechanism for protein kinase inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIY9begVVfW7Vg90H21EOLACvtfcHk0lhb6HSEULbn9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM&md5=79ebbc2e3a9cdd3fbecb8bb35ea9dba3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnsmb859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb859%26sid%3Dliteratum%253Aachs%26aulast%3DOhren%26aufirst%3DJ.%2BF.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DPavlovsky%26aufirst%3DA.%26aulast%3DWhitehead%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DKuffa%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DMcConnell%26aufirst%3DP.%26aulast%3DSpessard%26aufirst%3DC.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DMueller%26aufirst%3DW.%2BT.%26aulast%3DDelaney%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DC.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DDudley%26aufirst%3DD.%2BT.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DFlamme%26aufirst%3DC.%26aulast%3DWarmus%26aufirst%3DJ.%26aulast%3DKaufman%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DS.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DHasemann%26aufirst%3DC.%2BA.%26atitle%3DStructures%2520of%2520human%2520MAP%2520kinase%2520kinase%25201%2520%2528MEK1%2529%2520and%2520MEK2%2520describe%2520novel%2520noncompetitive%2520kinase%2520inhibition%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1192%26epage%3D1197%26doi%3D10.1038%2Fnsmb859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Dort, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nino, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanks, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galban, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, B. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of bifunctional oncogenic target inhibitors against allosteric mitogen-activated protein kinase (MEK1) and phosphatidylinositol 3-kinase (PI3K)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2512</span>– <span class="NLM_lpage">2522</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs1yqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2512-2522&author=M.+E.+Van%0ADortauthor=H.+Hongauthor=H.+Wangauthor=C.+A.+Ninoauthor=R.+L.+Lombardiauthor=A.+E.+Blanksauthor=S.+Galbanauthor=B.+D.+Ross&title=Discovery+of+bifunctional+oncogenic+target+inhibitors+against+allosteric+mitogen-activated+protein+kinase+%28MEK1%29+and+phosphatidylinositol+3-kinase+%28PI3K%29&doi=10.1021%2Facs.jmedchem.5b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bifunctional Oncogenic Target Inhibitors against Allosteric Mitogen-Activated Protein Kinase (MEK1) and Phosphatidylinositol 3-Kinase (PI3K)</span></div><div class="casAuthors">Van Dort, Marcian E.; Hong, Hao; Wang, Hanxiao; Nino, Charles A.; Lombardi, Rachel L.; Blanks, Avery E.; Galban, Stefanie; Ross, Brian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2512-2522</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of a series of single entity, bifunctional MEK1/PI3K inhibitors achieved by covalent linking of structural analogs of the ATP-competitive PI3K inhibitor ZSTK474 and the ATP-noncompetitive MEK inhibitor PD0325901 is described.  Inhibitors displayed potent in vitro inhibition of MEK1 (0.015 < IC50 (nM) < 56.7) and PI3K (54 < IC50 (nM) < 341) in enzymic inhibition assays.  Concurrent MEK1 and PI3K inhibition was demonstrated with inhibitors 9 and 14 in two tumor cell lines (A549, D54).  Inhibitors produced dose-dependent decreased cell viability similar to the combined administration of equiv. doses of ZSTK474 and PD0325901.  In vivo efficacy of 14 following oral administration was demonstrated in D54 glioma and A549 lung tumor bearing mice.  Compd. 14 showed a 95% and 67% inhibition of tumor ERK1/2 and Akt phosphorylation, resp., at 2 h postadministration by Western blot anal., confirming the bioavailability and efficacy of this bifunctional inhibitor strategy toward combined MEK1/PI3K inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxtplBF0UZ7bVg90H21EOLACvtfcHk0ljb_6nL9tRy3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs1yqsL4%253D&md5=1bbc4eef05d0c4bab6a21e7a5db2790e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01655%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDort%26aufirst%3DM.%2BE.%26aulast%3DHong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DNino%26aufirst%3DC.%2BA.%26aulast%3DLombardi%26aufirst%3DR.%2BL.%26aulast%3DBlanks%26aufirst%3DA.%2BE.%26aulast%3DGalban%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DB.%2BD.%26atitle%3DDiscovery%2520of%2520bifunctional%2520oncogenic%2520target%2520inhibitors%2520against%2520allosteric%2520mitogen-activated%2520protein%2520kinase%2520%2528MEK1%2529%2520and%2520phosphatidylinositol%25203-kinase%2520%2528PI3K%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2512%26epage%3D2522%26doi%3D10.1021%2Facs.jmedchem.5b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0ljb_6nL9tRy3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span> <span> </span><span class="NLM_article-title">MEK inhibitors induce apoptosis via FoxO3a-dependent PUMA induction in colorectal cancer cells</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1038/s41389-018-0078-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fs41389-018-0078-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29367650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFChsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=1-9&author=L.+Linauthor=D.+Dingauthor=Y.+Jiangauthor=Y.+Liauthor=S.+Li&title=MEK+inhibitors+induce+apoptosis+via+FoxO3a-dependent+PUMA+induction+in+colorectal+cancer+cells&doi=10.1038%2Fs41389-018-0078-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Atypical GATA transcription factor TRPS1 represses gene expression by recruiting CHD4/NuRD(MTA2) and suppresses cell migration and invasion by repressing TP63 expression</span></div><div class="casAuthors">Wang, Yuzhi; Lin, Xue; Gong, Xue; Wu, Lele; Zhang, Jun; Liu, Weiguang; Li, Jian; Chen, Liming</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">ONCOCT</span>;
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Transcriptional repressor GATA binding 1 (TRPS1), an atypical GATA transcription factor, functions as a transcriptional repressor and is also implicated in human cancers.  However, the underlying mechanism of TRPS1 contributing to malignancy remains obscure.  In the current study, we report that TRPS1 recognizes both gene proximal and distal transcription start site (TSS) sequences to repress gene expression.  Co-IP mass spectrometry and biochem. studies showed that TRPS1 binds to CHD4/NuRD(MTA2).  Genome-wide and mol. studies revealed that CHD4/NuRD(MTA2) is required for TRPS1 transcriptional repression.  Mech., TRPS1 and CHD4/NuRD(MTA2) form precision-guided transcriptional repression machinery in which TRPS1 guides the machinery to specific target sites by recognizing GATA elements, and CHD4/NuRD(MTA2) represses the transcription of target genes.  Furthermore, TP63 was identified and validated to be a direct target of TRPS1-CHD4/NuRD(MTA2) complex, which represses TP63 expression by involving decommission of TP63 enhancer in the described precision-guided manner, leading to a redn. of the ΔNp63 level and contributing to migration and invasion of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHKAZL8GecY7Vg90H21EOLACvtfcHk0ljb_6nL9tRy3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFChsrfK&md5=34a4014288e6d1504c4da2ad214d67ad</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41389-018-0078-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-018-0078-y%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26atitle%3DMEK%2520inhibitors%2520induce%2520apoptosis%2520via%2520FoxO3a-dependent%2520PUMA%2520induction%2520in%2520colorectal%2520cancer%2520cells%26jtitle%3DOncogenesis%26date%3D2018%26volume%3D7%26spage%3D1%26epage%3D9%26doi%3D10.1038%2Fs41389-018-0078-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0ljFagz6G7WhDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0ljFagz6G7WhDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratilas, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span> <span> </span><span class="NLM_article-title">BRAF mutation predicts sensitivity to MEK inhibition</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>439</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1038/nature04304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnature04304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=16273091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvF2mtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=439&publication_year=2006&pages=358-362&author=D.+B.+Solitauthor=L.+A.+Garrawayauthor=C.+A.+Pratilasauthor=A.+Sawaiauthor=G.+Getzauthor=A.+Bassoauthor=Q.+Yeauthor=J.+M.+Loboauthor=Y.+Sheauthor=I.+Osmanauthor=T.+R.+Golubauthor=J.+Sebolt-Leopoldauthor=W.+R.+Sellersauthor=N.+Rosen&title=BRAF+mutation+predicts+sensitivity+to+MEK+inhibition&doi=10.1038%2Fnature04304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">BRAF mutation predicts sensitivity to MEK inhibition</span></div><div class="casAuthors">Solit, David B.; Garraway, Levi A.; Pratilas, Christine A.; Sawai, Ayana; Getz, Gad; Basso, Andrea; Ye, Qing; Lobo, Jose M.; She, Yuhong; Osman, Iman; Golub, Todd R.; Sebolt-Leopold, Judith; Sellers, William R.; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">439</span>
        (<span class="NLM_cas:issue">7074</span>),
    <span class="NLM_cas:pages">358-362</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The kinase pathway comprising RAS, RAF, mitogen-activated protein kinase kinase (MEK) and extracellular signal regulated kinase (ERK) is activated in most human tumors, often through gain-of-function mutations of RAS and RAF family members.  Using small-mol. inhibitors of MEK and an integrated genetic and pharmacol. anal., we find that mutation of BRAF is assocd. with enhanced and selective sensitivity to MEK inhibition when compared to either 'wild-type' cells or cells harboring a RAS mutation.  This MEK dependency was obsd. in BRAF mutant cells regardless of tissue lineage, and correlated with both downregulation of cyclin D1 protein expression and the induction of G1 arrest.  Pharmacol. MEK inhibition completely abrogated tumor growth in BRAF mutant xenografts, whereas RAS mutant tumors were only partially inhibited.  These data suggest an exquisite dependency on MEK activity in BRAF mutant tumors, and offer a rational therapeutic strategy for this genetically defined tumor subtype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpBpONOvd0vbVg90H21EOLACvtfcHk0ljFagz6G7WhDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvF2mtw%253D%253D&md5=37c8976aa5bb371ef5580aab2a91f89e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature04304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04304%26sid%3Dliteratum%253Aachs%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26aulast%3DSawai%26aufirst%3DA.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DBasso%26aufirst%3DA.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DLobo%26aufirst%3DJ.%2BM.%26aulast%3DShe%26aufirst%3DY.%26aulast%3DOsman%26aufirst%3DI.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DBRAF%2520mutation%2520predicts%2520sensitivity%2520to%2520MEK%2520inhibition%26jtitle%3DNature%26date%3D2006%26volume%3D439%26spage%3D358%26epage%3D362%26doi%3D10.1038%2Fnature04304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asundi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacap, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span> <span> </span><span class="NLM_article-title">MAPK pathway inhibition enhances the efficacy of an anti-endothelin B receptor drug conjugate by inducing target expression in melanoma</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0446</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1158%2F1535-7163.MCT-13-0446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=24651527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpt1Ojsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1599-1610&author=J.+Asundiauthor=J.+A.+Lacapauthor=S.+Clarkauthor=M.+Nanniniauthor=L.+Rothauthor=P.+Polakis&title=MAPK+pathway+inhibition+enhances+the+efficacy+of+an+anti-endothelin+B+receptor+drug+conjugate+by+inducing+target+expression+in+melanoma&doi=10.1158%2F1535-7163.MCT-13-0446"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MAPK Pathway Inhibition Enhances the Efficacy of an Anti-Endothelin B Receptor Drug Conjugate by Inducing Target Expression in Melanoma</span></div><div class="casAuthors">Asundi, Jyoti; Lacap, Jennifer A.; Clark, Suzanna; Nannini, Michelle; Roth, Leslie; Polakis, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1599-1610</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Therapies targeting the mitogen-activated protein (MAP) kinase pathway in melanoma have produced significant clin. responses; however, duration of response is limited by acquisition of drug resistance.  Rational drug combinations may improve outcomes in this setting.  We assessed the therapeutic combination of an antibody-drug conjugate (ADC) targeting the endothelin B receptor (EDNRB) with small-mol. inhibitors of the MAP kinase signaling pathway in melanoma.  Cell lines and tumor models contg. either mutant BRAF or NRAS, or wild-type for both, were exposed to small-mol. inhibitors of BRAF and MEK.  Expression of EDNRB was analyzed and the therapeutic impact of combining the anti-EDNRB ADC with the BRAF and MEK inhibitors was assessed.  Increased expression of EDNRB in response to inhibition of BRAF and/or MEK was obsd. and augmented the antitumor activity of the ADC.  Enhanced target expression and ADC antitumor activity were realized irresp. of the response of the tumor model to the BRAF or MEK inhibitors alone and could be achieved in melanoma with mutant NRAS, BRAF, or neither mutation.  Cells that acquired resistance to BRAF inhibition through long-term culture retained drug-induced elevated levels of EDNRB expression.  Expression of EDNRB was not enhanced in normal human melanocytes by inhibition of BRAF and the combination of the ADC with MAPK inhibitors was well-tolerated in mice.  The anti-EDNRB ADC combines well with BRAF and MEK inhibitors and could have therapeutic use in the majority of human melanoma cases.  Mol Cancer Ther; 13(6); 1599-610. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6AdZ-w8sQgLVg90H21EOLACvtfcHk0liX_xoL5EUhdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpt1Ojsb0%253D&md5=91e2a8f9a26da15c08f752429f8cdd55</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0446%26sid%3Dliteratum%253Aachs%26aulast%3DAsundi%26aufirst%3DJ.%26aulast%3DLacap%26aufirst%3DJ.%2BA.%26aulast%3DClark%26aufirst%3DS.%26aulast%3DNannini%26aufirst%3DM.%26aulast%3DRoth%26aufirst%3DL.%26aulast%3DPolakis%26aufirst%3DP.%26atitle%3DMAPK%2520pathway%2520inhibition%2520enhances%2520the%2520efficacy%2520of%2520an%2520anti-endothelin%2520B%2520receptor%2520drug%2520conjugate%2520by%2520inducing%2520target%2520expression%2520in%2520melanoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D1599%26epage%3D1610%26doi%3D10.1158%2F1535-7163.MCT-13-0446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACs: An emerging and promising approach for the development of targeted therapy drugs</span>. <i>EBioMedicine</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1016/j.ebiom.2018.09.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ebiom.2018.09.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=30224312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A280%3ADC%252BB3c3pslWjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=553-562&author=S.+Anauthor=L.+Fu&title=Small-molecule+PROTACs%3A+An+emerging+and+promising+approach+for+the+development+of+targeted+therapy+drugs&doi=10.1016%2Fj.ebiom.2018.09.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule PROTACs: An emerging and promising approach for the development of targeted therapy drugs</span></div><div class="casAuthors">An Sainan; Fu Liwu</div><div class="citationInfo"><span class="NLM_cas:title">EBioMedicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">553-562</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">There are several challenges towards the development and clinical use of small molecule inhibitors, which are currently the main type of targeted therapies towards intracellular proteins.  PROteolysis-TArgeting Chimeras (PROTACs) exploit the intracellular ubiquitin-proteasome system to selectively degrade target proteins.  Recently, small-molecule PROTACs with high potency have been frequently reported.  In this review, we summarize the emerging characteristics of small-molecule PROTACs, such as inducing a rapid, profound and sustained degradation, inducing a robust inhibition of downstream signals, displaying enhanced target selectivity, and overcoming resistance to small molecule inhibitors.  In tumor xenografts, small-molecule PROTACs can significantly attenuate tumor progression.  In addition, we also introduce recent developments of the PROTAC technology such as homo-PROTACs.  The outstanding advantages over traditional small-molecule drugs and the promising preclinical data suggest that small-molecule PROTAC technology has the potential to greatly promote the development of targeted therapy drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSW7YU6u8U3fIFEOhA_Im3cfW6udTcc2ebQMt7AJCJ-47ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3pslWjtQ%253D%253D&md5=415a5c492059c464dede49de6375bdba</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.ebiom.2018.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ebiom.2018.09.005%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DL.%26atitle%3DSmall-molecule%2520PROTACs%253A%2520An%2520emerging%2520and%2520promising%2520approach%2520for%2520the%2520development%2520of%2520targeted%2520therapy%2520drugs%26jtitle%3DEBioMedicine%26date%3D2018%26volume%3D36%26spage%3D553%26epage%3D562%26doi%3D10.1016%2Fj.ebiom.2018.09.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">ERK1/2 MAP kinases: structure, function, and regulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2012.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.phrs.2012.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=22569528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC38XovVygs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2012&pages=105-143&author=R.+Roskoski&title=ERK1%2F2+MAP+kinases%3A+structure%2C+function%2C+and+regulation&doi=10.1016%2Fj.phrs.2012.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">ERK1/2 MAP kinases: Structure, function, and regulation</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-143</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  ERK1 and ERK2 are related protein-serine/threonine kinases that participate in the Ras-Raf-MEK-ERK signal transduction cascade.  This cascade participates in the regulation of a large variety of processes including cell adhesion, cell cycle progression, cell migration, cell survival, differentiation, metab., proliferation, and transcription.  MEK1/2 catalyze the phosphorylation of human ERK1/2 at Tyr204/187 and then Thr202/185.  The phosphorylation of both tyrosine and threonine is required for enzyme activation.  Whereas the Raf kinase and MEK families have narrow substrate specificity, ERK1/2 catalyze the phosphorylation of hundreds of cytoplasmic and nuclear substrates including regulatory mols. and transcription factors.  ERK1/2 are proline-directed kinases that preferentially catalyze the phosphorylation of substrates contg. a Pro-Xxx-Ser/Thr-Pro sequence.  Besides this primary structure requirement, many ERK1/2 substrates possess a D-docking site, an F-docking site, or both.  A variety of scaffold proteins including KSR1/2, IQGAP1, MP1, β-Arrestin1/2 participate in the regulation of the ERK1/2 MAP kinase cascade.  The regulatory dephosphorylation of ERK1/2 is mediated by protein-tyrosine specific phosphatases, protein-serine/threonine phosphatases, and dual specificity phosphatases.  The combination of kinases and phosphatases make the overall process reversible.  The ERK1/2 catalyzed phosphorylation of nuclear transcription factors including those of Ets, Elk, and c-Fos represents an important function and requires the translocation of ERK1/2 into the nucleus by active and passive processes involving the nuclear pore.  These transcription factors participate in the immediate early gene response.  The activity of the Ras-Raf-MEK-ERK cascade is increased in about one-third of all human cancers, and inhibition of components of this cascade by targeted inhibitors represents an important anti-tumor strategy.  Thus far, however, only inhibition of mutant B-Raf (Val600Glu) has been found to be therapeutically efficacious.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUVgAuIw2CcLVg90H21EOLACvtfcHk0liX_xoL5EUhdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovVygs7k%253D&md5=ee168ded480a45f8995278f06bef6440</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2012.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2012.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DERK1%252F2%2520MAP%2520kinases%253A%2520structure%252C%2520function%252C%2520and%2520regulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2012%26volume%3D66%26spage%3D105%26epage%3D143%26doi%3D10.1016%2Fj.phrs.2012.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lito, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saborowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastenhuber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morikami, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span> <span> </span><span class="NLM_article-title">Disruption of CRAF-mediated MEK activation is required for effective MEK inhibition in KRAS mutant tumors</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">697</span>– <span class="NLM_lpage">710</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ccr.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=24746704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVCht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=697-710&author=P.+Litoauthor=A.+Saborowskiauthor=J.+Yueauthor=M.+Solomonauthor=E.+Josephauthor=S.+Gadalauthor=M.+Saborowskiauthor=E.+Kastenhuberauthor=C.+Fellmannauthor=K.+Oharaauthor=K.+Morikamiauthor=T.+Miuraauthor=C.+Lukacsauthor=N.+Ishiiauthor=S.+Loweauthor=N.+Rosen&title=Disruption+of+CRAF-mediated+MEK+activation+is+required+for+effective+MEK+inhibition+in+KRAS+mutant+tumors&doi=10.1016%2Fj.ccr.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Disruption of CRAF-Mediated MEK Activation Is Required for Effective MEK Inhibition in KRAS Mutant Tumors</span></div><div class="casAuthors">Lito, Piro; Saborowski, Anna; Yue, Jingyin; Solomon, Martha; Joseph, Eric; Gadal, Sunyana; Saborowski, Michael; Kastenhuber, Edward; Fellmann, Christof; Ohara, Kazuhiro; Morikami, Kenji; Miura, Takaaki; Lukacs, Christine; Ishii, Nobuya; Lowe, Scott; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-710</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MEK inhibitors are clin. active in BRAFV600E melanomas but only marginally so in KRAS mutant tumors.  Here, we found that MEK inhibitors suppress ERK signaling more potently in BRAFV600E, than in KRAS mutant tumors.  To understand this, we performed an RNAi screen in a KRAS mutant model and found that CRAF knockdown enhanced MEK inhibition.  MEK activated by CRAF was less susceptible to MEK inhibitors than when activated by BRAFV600E.  MEK inhibitors induced RAF-MEK complexes in KRAS mutant models, and disrupting such complexes enhanced inhibition of CRAF-dependent ERK signaling.  Newer MEK inhibitors target MEK catalytic activity and also impair its reactivation by CRAF, either by disrupting RAF-MEK complexes or by interacting with Ser 222 to prevent MEK phosphorylation by RAF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrm_fqelH7I0bVg90H21EOLACvtfcHk0ljnD4l4WyCFXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVCht7c%253D&md5=1299892e2941b79fa511a47347557c6e</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DLito%26aufirst%3DP.%26aulast%3DSaborowski%26aufirst%3DA.%26aulast%3DYue%26aufirst%3DJ.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DJoseph%26aufirst%3DE.%26aulast%3DGadal%26aufirst%3DS.%26aulast%3DSaborowski%26aufirst%3DM.%26aulast%3DKastenhuber%26aufirst%3DE.%26aulast%3DFellmann%26aufirst%3DC.%26aulast%3DOhara%26aufirst%3DK.%26aulast%3DMorikami%26aufirst%3DK.%26aulast%3DMiura%26aufirst%3DT.%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DIshii%26aufirst%3DN.%26aulast%3DLowe%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DDisruption%2520of%2520CRAF-mediated%2520MEK%2520activation%2520is%2520required%2520for%2520effective%2520MEK%2520inhibition%2520in%2520KRAS%2520mutant%2520tumors%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D697%26epage%3D710%26doi%3D10.1016%2Fj.ccr.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Little, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. J.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of acquired resistance to ERK1/2 pathway inhibitors</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1207</span>– <span class="NLM_lpage">1215</span>, <span class="refDoi"> DOI: 10.1038/onc.2012.160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fonc.2012.160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=22562245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsFOqt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=1207-1215&author=A.+S.+Littleauthor=P.+D.+Smithauthor=S.+J.+Cook&title=Mechanisms+of+acquired+resistance+to+ERK1%2F2+pathway+inhibitors&doi=10.1038%2Fonc.2012.160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of acquired resistance to ERK1/2 pathway inhibitors</span></div><div class="casAuthors">Little, A. S.; Smith, P. D.; Cook, S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1207-1215</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The ERK1/2 (extracellular signal-regulated kinase 1 and 2) pathway, comprising the protein kinases RAF (v-raf-1 murine leukemia viral oncogene homolog 1), MEK1/2 (mitogen-activated protein kinase or ERK kinase 1 and 2) and ERK1/2 is frequently de-regulated in human cancers, due to mutations in RAS or BRAF (v-raf-1 murine leukemia viral oncogene homolog B1).  New, highly selective inhibitors of BRAF and MEK1/2 have shown promise in clin. trials, including in previously intractable diseases such as melanoma.  However, drug-resistant tumor cells invariably emerge leading to disease progression.  It is important to understand the mechanisms underlying such acquired resistance since this may lead to the development of rational strategies either to delay its onset or to overcome it once established.  It also offers unique insights into the plasticity of signalling pathways, which may in turn inform our understanding of the basic biol. of these pathways and lead to the validation of new drug targets.  Several recent reports have identified diverse mechanisms of acquired resistance to MEK1/2 or BRAF inhibitors.  In this article, we review these studies, discuss the different mechanisms, identify common themes and consider their therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPrc4YOQY4zrVg90H21EOLACvtfcHk0ljnD4l4WyCFXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsFOqt7Y%253D&md5=0d604795d8d70e7725d7d6d6cc1d9edb</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fonc.2012.160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2012.160%26sid%3Dliteratum%253Aachs%26aulast%3DLittle%26aufirst%3DA.%2BS.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26atitle%3DMechanisms%2520of%2520acquired%2520resistance%2520to%2520ERK1%252F2%2520pathway%2520inhibitors%26jtitle%3DOncogene%26date%3D2013%26volume%3D32%26spage%3D1207%26epage%3D1215%26doi%3D10.1038%2Fonc.2012.160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Little, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balmanno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hampson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. J.</span></span> <span> </span><span class="NLM_article-title">Amplification of the driving oncogene, KRAS or BRAF, underpins acquired resistance to MEK1/2 inhibitors in colorectal cancer cells</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1126/scisignal.2001752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1126%2Fscisignal.2001752" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=1-16&author=A.+S.+Littleauthor=K.+Balmannoauthor=M.+J.+Saleauthor=S.+Newmanauthor=J.+R.+Dryauthor=M.+Hampsonauthor=P.+A.+Edwardsauthor=P.+D.+Smithauthor=S.+J.+Cook&title=Amplification+of+the+driving+oncogene%2C+KRAS+or+BRAF%2C+underpins+acquired+resistance+to+MEK1%2F2+inhibitors+in+colorectal+cancer+cells&doi=10.1126%2Fscisignal.2001752"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2001752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2001752%26sid%3Dliteratum%253Aachs%26aulast%3DLittle%26aufirst%3DA.%2BS.%26aulast%3DBalmanno%26aufirst%3DK.%26aulast%3DSale%26aufirst%3DM.%2BJ.%26aulast%3DNewman%26aufirst%3DS.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DHampson%26aufirst%3DM.%26aulast%3DEdwards%26aufirst%3DP.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26atitle%3DAmplification%2520of%2520the%2520driving%2520oncogene%252C%2520KRAS%2520or%2520BRAF%252C%2520underpins%2520acquired%2520resistance%2520to%2520MEK1%252F2%2520inhibitors%2520in%2520colorectal%2520cancer%2520cells%26jtitle%3DSci.%2520Signaling%26date%3D2011%26volume%3D4%26spage%3D1%26epage%3D16%26doi%3D10.1126%2Fscisignal.2001752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welsh, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizos, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span> <span> </span><span class="NLM_article-title">Resistance to combination BRAF and MEK inhibition in metastatic melanoma: where to next?</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/j.ejca.2016.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1016%2Fj.ejca.2016.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=27232329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsVWqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2016&pages=76-85&author=S.+J.+Welshauthor=H.+Rizosauthor=R.+A.+Scolyerauthor=G.+V.+Long&title=Resistance+to+combination+BRAF+and+MEK+inhibition+in+metastatic+melanoma%3A+where+to+next%3F&doi=10.1016%2Fj.ejca.2016.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to combination BRAF and MEK inhibition in metastatic melanoma: Where to next?</span></div><div class="casAuthors">Welsh, Sarah J.; Rizos, Helen; Scolyer, Richard A.; Long, Georgina V.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76-85</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Treatment of BRAF-mutant metastatic melanoma with mitogen-activated protein kinase (MAPK) pathway targeted therapies (BRAF/MEK inhibitors) and immune checkpoint inhibitors has revolutionized management and improved outcomes for patients with advanced stage disease.  However, acquired resistance to MAPK inhibitor therapy develops in the majority of patients at approx. 12 mo and multiple mechanisms lead to resistance.  Understanding the mechanisms of resistance is therefore crit. for the development of more effective therapeutic strategies in BRAF-mutant melanoma.  Recently, several distinct mechanisms of resistance to BRAF-inhibition have been proposed based on data obtained in exptl. melanoma cell models and small series of human tumor samples.  These include reactivation of the MAPK pathway resulting in continued extracellular signal-regulated kinase activation and activation of parallel signalling pathways including the PI3K-mTOR (phosphoinositide 3-kinase-mammalian target of rapamycin) pathway.  Alterations in how the cells of the immune system respond to melanoma cells treated with targeted therapy may also influence response and progression.  In this review, we discuss these mechanisms and identify potential therapeutic strategies to overcome resistance which, in turn, will lead to improved outcomes for patients with metastatic melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl-VbAROwGALVg90H21EOLACvtfcHk0ljnD4l4WyCFXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsVWqsro%253D&md5=34774092eb983ec5c48c247fd49c6887</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2016.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2016.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWelsh%26aufirst%3DS.%2BJ.%26aulast%3DRizos%26aufirst%3DH.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DLong%26aufirst%3DG.%2BV.%26atitle%3DResistance%2520to%2520combination%2520BRAF%2520and%2520MEK%2520inhibition%2520in%2520metastatic%2520melanoma%253A%2520where%2520to%2520next%253F%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2016%26volume%3D62%26spage%3D76%26epage%3D85%26doi%3D10.1016%2Fj.ejca.2016.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with <i>in vitro</i> nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lh_Y_Xs09jOqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0lh_Y_Xs09jOqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franken, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodermond, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stap, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haveman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Bree, C.</span></span> <span> </span><span class="NLM_article-title">Clonogenic assay of cells <i>in vitro</i></span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">2315</span>– <span class="NLM_lpage">2319</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1038%2Fnprot.2006.339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=17406473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGjtLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=2315-2319&author=N.+A.+Frankenauthor=H.+M.+Rodermondauthor=J.+Stapauthor=J.+Havemanauthor=C.+van+Bree&title=Clonogenic+assay+of+cells+in+vitro&doi=10.1038%2Fnprot.2006.339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Clonogenic assay of cells in vitro</span></div><div class="casAuthors">Franken, Nicolaas A. P.; Rodermond, Hans M.; Stap, Jan; Haveman, Jaap; van Bree, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2315-2319</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clonogenic assay or colony formation assay is an in vitro cell survival assay based on the ability of a single cell to grow into a colony.  The colony is defined to consist of at least 50 cells.  The assay essentially tests every cell in the population for its ability to undergo "unlimited" division.  Clonogenic assay is the method of choice to det. cell reproductive death after treatment with ionizing radiation, but can also be used to det. the effectiveness of other cytotoxic agents.  Only a fraction of seeded cells retains the capacity to produce colonies.  Before or after treatment, cells are seeded out in appropriate dilns. to form colonies in 1-3 wk.  Colonies are fixed with glutaraldehyde (6.0% vol./vol.), stained with crystal violet (0.5% w/v) and counted using a stereomicroscope.  A method for the anal. of radiation dose-survival curves is included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGove0egxayKRrVg90H21EOLACvtfcHk0lj40w7X_miwiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGjtLnI&md5=5edcb3202aa9088b16d27608ae00929b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.339%26sid%3Dliteratum%253Aachs%26aulast%3DFranken%26aufirst%3DN.%2BA.%26aulast%3DRodermond%26aufirst%3DH.%2BM.%26aulast%3DStap%26aufirst%3DJ.%26aulast%3DHaveman%26aufirst%3DJ.%26aulast%3Dvan%2BBree%26aufirst%3DC.%26atitle%3DClonogenic%2520assay%2520of%2520cells%2520in%2520vitro%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D2315%26epage%3D2319%26doi%3D10.1038%2Fnprot.2006.339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykema, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherba, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furge, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duesbery, N. S.</span></span> <span> </span><span class="NLM_article-title">MEK2 is sufficient but not necessary for proliferation and anchorage-independent growth of SK-MEL-28 melanoma cells</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0017165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;key=10.1371%2Fjournal.pone.0017165" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1-15&author=C.+S.+Leeauthor=K.+J.+Dykemaauthor=D.+M.+Hawkinsauthor=D.+M.+Cherbaauthor=C.+P.+Webbauthor=K.+A.+Furgeauthor=N.+S.+Duesbery&title=MEK2+is+sufficient+but+not+necessary+for+proliferation+and+anchorage-independent+growth+of+SK-MEL-28+melanoma+cells&doi=10.1371%2Fjournal.pone.0017165"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0017165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0017165%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BS.%26aulast%3DDykema%26aufirst%3DK.%2BJ.%26aulast%3DHawkins%26aufirst%3DD.%2BM.%26aulast%3DCherba%26aufirst%3DD.%2BM.%26aulast%3DWebb%26aufirst%3DC.%2BP.%26aulast%3DFurge%26aufirst%3DK.%2BA.%26aulast%3DDuesbery%26aufirst%3DN.%2BS.%26atitle%3DMEK2%2520is%2520sufficient%2520but%2520not%2520necessary%2520for%2520proliferation%2520and%2520anchorage-independent%2520growth%2520of%2520SK-MEL-28%2520melanoma%2520cells%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3D1%26epage%3D15%26doi%3D10.1371%2Fjournal.pone.0017165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9J','PDB','1S9J'); return false;">PDB: 1S9J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP1','PDB','3PP1'); return false;">PDB: 3PP1</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i42"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01528">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_45036"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01528?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01528</a>. <ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings of <b>7</b> and <b>8</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_001.csv">CSV</a>)</p></li><li><p class="inline">Detection of CRBN expression in HT-29 cells and VHL expression in CRC and melanoma cell lines; time-dependent degradation of MEK1 and MEK2 by compound <b>23</b> in COLO 205 and UACC257 cell lines; Western blot analyses of long-term treatment of PD0325901 and compound <b>23</b> in HT-29 cells; Western blot validation of proteomics studies in HT-29 cells; antiproliferative effects of compound <b>23</b> in COLO 205 and UACC257 cell lines; shRNA-mediated MEK1/2 protein knockdown data in HT-29 and SK-MEL-28 cell lines; and <sup>1</sup>H, <sup>13</sup>C, HPLC, and MS spectra of compounds <b>23</b> and <b>24</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of <b>7</b>–<b>11</b>, <b>23</b>, and <b>24</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_001.csv">jm9b01528_si_001.csv (0.3 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_002.pdf">jm9b01528_si_002.pdf (1.9 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01528/suppl_file/jm9b01528_si_003.csv">jm9b01528_si_003.csv (1.01 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01528%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-23" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01528" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a95a1ddd33c40","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
