
<html lang="en"     class="pb-page"  data-request-id="96d86bde-4a32-4f76-8678-47cd784ddf1a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.7b01478;website:website:acspubs;issue:issue:10.1021/jmcmar.2018.61.issue-1;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="The Design and Development of a Potent and Selective Novel Diprolyl Derivative That Binds to the N-Domain of Angiotensin-I Converting Enzyme" /></meta><meta name="dc.Creator" content="Stephen  Fienberg" /></meta><meta name="dc.Creator" content="Gyles E.  Cozier" /></meta><meta name="dc.Creator" content="K. Ravi  Acharya" /></meta><meta name="dc.Creator" content="Kelly  Chibale" /></meta><meta name="dc.Creator" content="Edward D.  Sturrock" /></meta><meta name="dc.Description" content="Angiotensin-I converting enzyme (ACE) is a zinc metalloprotease consisting of two catalytic domains (N- and C-). Most clinical ACE inhibitor(s) (ACEi) have been shown to inhibit both domains nonsel..." /></meta><meta name="Description" content="Angiotensin-I converting enzyme (ACE) is a zinc metalloprotease consisting of two catalytic domains (N- and C-). Most clinical ACE inhibitor(s) (ACEi) have been shown to inhibit both domains nonsel..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 21, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01478" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01478" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01478" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01478" /></link>
        
    
    

<title>The Design and Development of a Potent and Selective Novel Diprolyl Derivative That Binds to the N-Domain of Angiotensin-I Converting Enzyme | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01478" /></meta><meta property="og:title" content="The Design and Development of a Potent and Selective Novel Diprolyl Derivative That Binds to the N-Domain of Angiotensin-I Converting Enzyme" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0018.jpeg" /></meta><meta property="og:description" content="Angiotensin-I converting enzyme (ACE) is a zinc metalloprotease consisting of two catalytic domains (N- and C-). Most clinical ACE inhibitor(s) (ACEi) have been shown to inhibit both domains nonselectively, resulting in adverse effects such as cough and angioedema. Selectively inhibiting the individual domains is likely to reduce these effects and potentially treat fibrosis in addition to hypertension. ACEi from the GVK Biosciences database were inspected for possible N-domain selective binding patterns. From this set, a diprolyl chemical series was modeled using docking simulations. The series was expanded based on key target interactions involving residues known to impart N-domain selectivity. In total, seven diprolyl compounds were synthesized and tested for N-domain selective ACE inhibition. One compound with an aspartic acid in the P2 position (compound 16) displayed potent inhibition (Ki = 11.45 nM) and was 84-fold more selective toward the N-domain. A high-resolution crystal structure of compound 16 in complex with the N-domain revealed the molecular basis for the observed selectivity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01478"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01478">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01478&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01478&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01478&amp;href=/doi/10.1021/acs.jmedchem.7b01478" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 344-359</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01427" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01552" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">The Design and Development of a Potent and Selective Novel Diprolyl Derivative That Binds to the N-Domain of Angiotensin-I Converting Enzyme</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen++Fienberg">Stephen Fienberg</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gyles+E.++Cozier">Gyles E. Cozier</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=K.+Ravi++Acharya">K. Ravi Acharya</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kelly++Chibale">Kelly Chibale</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-1327-4727" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Edward+D.++Sturrock">Edward D. Sturrock</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">â </span> <span class="aff-text">Department of Chemistry, University of Cape Town, Rondebosch 7701, South Africa</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">â¡</span> <span class="aff-text">Department of Biology and Biochemistry, University of Bath, Claverton Down, Bath BA2 7AY, U.K.</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">Â§</span> <span class="aff-text">Institute of Infectious Disease and Molecular Medicine, University of Cape Town, Observatory 7925, South Africa</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">â¥</span> <span class="aff-text">Department of Integrative Biomedical Sciences, University of Cape Town, Observatory 7925, South Africa</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">â¥</span> <span class="aff-text">South African Medical Research Council Drug Discovery and Development Research Unit, University of Cape Town, Rondebosch 7701, South Africa</span></div><div class="corresp-info"><strong>*</strong>For K.C.: phone, +27 21 650-2553; fax, +27 21 650-4521; E-mail, <a href="/cdn-cgi/l/email-protection#8bc0eee7e7f2a5c8e3e2e9eae7eecbfee8ffa5eae8a5f1ea"><span class="__cf_email__" data-cfemail="19527c757560375a71707b78757c596c7a6d37787a376378">[emailÂ protected]</span></a></div><div class="corresp-info"><strong>*</strong>For E.D.S.: phone, +27 21 406-6312; fax, +27 21 406-6479; E-mail, <a href="/cdn-cgi/l/email-protection#2663425147544208755253545449454d66534552084745085c47"><span class="__cf_email__" data-cfemail="7732130016051359240302050518141c37021403591614590d16">[emailÂ protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01478&amp;href=/doi/10.1021%2Facs.jmedchem.7b01478" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 344â359</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 5, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>12 October 2017</li><li><span class="item_label"><b>Published</b> online</span>21 December 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 January 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01478" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01478</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D344%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DStephen%2BFienberg%252C%2BGyles%2BE.%2BCozier%252C%2BK.%2BRavi%2BAcharya%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D1%26contentID%3Dacs.jmedchem.7b01478%26title%3DThe%2BDesign%2Band%2BDevelopment%2Bof%2Ba%2BPotent%2Band%2BSelective%2BNovel%2BDiprolyl%2BDerivative%2BThat%2BBinds%2Bto%2Bthe%2BN-Domain%2Bof%2BAngiotensin-I%2BConverting%2BEnzyme%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D359%26publicationDate%3DJanuary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01478"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1060</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01478" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;The Design and Development of a Potent and Selective Novel Diprolyl Derivative That Binds to the N-Domain of Angiotensin-I Converting Enzyme&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;Fienberg&quot;},{&quot;first_name&quot;:&quot;Gyles&quot;,&quot;last_name&quot;:&quot;E. Cozier&quot;},{&quot;first_name&quot;:&quot;K.&quot;,&quot;last_name&quot;:&quot;Ravi Acharya&quot;},{&quot;first_name&quot;:&quot;Kelly&quot;,&quot;last_name&quot;:&quot;Chibale&quot;},{&quot;first_name&quot;:&quot;Edward&quot;,&quot;last_name&quot;:&quot;D. Sturrock&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;344-359&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01478&quot;},&quot;abstract&quot;:&quot;Angiotensin-I converting enzyme (ACE) is a zinc metalloprotease consisting of two catalytic domains (N- and C-). Most clinical ACE inhibitor(s) (ACEi) have been shown to inhibit both domains nonselectively, resulting in adverse effects such as cough and angioedema. Selectively inhibiting the individual domains is likely to reduce these effects and potentially treat fibrosis in addition to hypertension. ACEi from the GVK Biosciences database were inspected for possible N-domain selective binding patterns. From this set, a diprolyl chemical series was modeled using docking simulations. The series was expanded based on key target interactions involving residues known to impart N-domain selectivity. In total, seven diprolyl compounds were synthesized and tested for N-domain selective ACE inhibition. One compound with an aspartic acid in the P2 position (compound 16) displayed potent inhibition (Ki = 11.45 nM) and was 84-fold more selective toward the N-domain. A high-resolution crystal structure of compound 16 in&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01478&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01478" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01478&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01478" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01478&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01478" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01478&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01478&amp;href=/doi/10.1021/acs.jmedchem.7b01478" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01478" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01478" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01478%26sid%3Dliteratum%253Aachs%26pmid%3D29206036%26genre%3Darticle%26aulast%3DFienberg%26date%3D2018%26atitle%3DThe%2BDesign%2Band%2BDevelopment%2Bof%2Ba%2BPotent%2Band%2BSelective%2BNovel%2BDiprolyl%2BDerivative%2BThat%2BBinds%2Bto%2Bthe%2BN-Domain%2Bof%2BAngiotensin-I%2BConverting%2BEnzyme%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D1%26spage%3D344%26epage%3D359%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291012" title="Organic acids">Organic acids</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/jmcmar.2018.61.issue-1/20180111/jmcmar.2018.61.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Angiotensin-I converting enzyme (ACE) is a zinc metalloprotease consisting of two catalytic domains (N- and C-). Most clinical ACE inhibitor(s) (ACEi) have been shown to inhibit both domains nonselectively, resulting in adverse effects such as cough and angioedema. Selectively inhibiting the individual domains is likely to reduce these effects and potentially treat fibrosis in addition to hypertension. ACEi from the GVK Biosciences database were inspected for possible N-domain selective binding patterns. From this set, a diprolyl chemical series was modeled using docking simulations. The series was expanded based on key target interactions involving residues known to impart N-domain selectivity. In total, seven diprolyl compounds were synthesized and tested for N-domain selective ACE inhibition. One compound with an aspartic acid in the P<sub>2</sub> position (compound <b>16</b>) displayed potent inhibition (<i>K</i><sub>i</sub> = 11.45 nM) and was 84-fold more selective toward the N-domain. A high-resolution crystal structure of compound <b>16</b> in complex with the N-domain revealed the molecular basis for the observed selectivity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36858" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36858" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Angiotensin-I converting enzyme (ACE; EC 3.4.15.1) is a zinc dipeptidyl carboxypeptidase that cleaves the inactive decapeptide angiotensin-I (Ang-I) into the vasopressor angiotensin-II (Ang-II) and degrades the vasodilator bradykinin (BK). The importance of these activities is emphasized by the extensive use of ACE inhibitor(s) (ACEi) for the treatment of hypertension and cardiovascular disease. The ACEi captopril<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and lisinopril<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> were designed in the 1970s and later extended by others. Currently, more than 17 ACEi have been approved for clinical use.</div><div class="NLM_p">Captopril is a peptidomimetic compound designed to mimic the sequence of peptides isolated from the <i>Bothrops jaracara</i> venom, which were known to inhibit the ACE-catalyzed dipeptidyl cleavage of Ang-I. Since the elucidation of the RAAS, many more peptide substrates of ACE have been identified and have been shown to play a role in many more signaling pathways.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">At the time of the development of captopril, ACE was assumed to share mechanistic similarities with carboxypeptidase A, a zinc metalloprotease of which the structure was known.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Following the cloning of somatic ACE, it was shown to consist of two homologous catalytic domains known as the N- and C-domains. Apart from structural differences, the two domains have been shown to possess different kinetic profiles when hydrolyzing their peptide substrate.</div><div class="NLM_p">Subsequent studies have shown the C-domain to be responsible for the majority of Ang-I hydrolysis, while the other major substrate, bradykinin, has been shown to be hydrolyzed at a similar rate by both the N- and C-domains.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> For the most part, the two domains of ACE have been shown to act independently of each other with the total activity of somatic ACE (sACE) equaling the sum of activities for the individual domains.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> However, negative cooperativity between the two domains has been demonstrated with the individual inhibition of just one domain<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and a series of mutant ACE constructs, changing the order of these catalytic domains.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">The tetrapeptide AcSDKP was discovered around the same time that the two domains of ACE were elucidated.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The presence of AcSDKP has been shown to reduce the deposition of collagen in hematopoetic stem cells of both cardiac and pulmonary tissues.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Excess collagen deposition in these tissues has been identified as a mechanism for the pathogenesis of fibrosis. AcSDKP has been shown to be exclusively hydrolyzed by the N-domain of ACE.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> N-Domain selective ACE inhibition has been identified as a possible treatment for both cardiac and pulmonary fibrosis without the side effects of nonselective inhibition of both domains of ACE.</div><div class="NLM_p">Individual N- and C-domain ACE constructs have been crystallized with a variety of different inhibitors. These crystal structures have clearly elucidated the core interactions responsible for potent ACE inhibition.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a></div><div class="NLM_p">The N- and C-domains share 65% sequence homology with each other.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Many of the key interactions are shared between the two domains with most inhibitors showing similar <i>K</i><sub>i</sub> values. Both substrates and most ACEi bind to the ACE active site via a chelation interaction with the catalytic zinc metal, an H-bond between the catalytic glutamic acid, and the amide nitrogen of the zinc binding amide and a polar salt-bridge interaction between the charged terminal carboxylic acid and three conserved residues in the S<sub>2</sub>â² subsite (Gln259, Lys489, and Tyr498 in the N-domain, conserved as Gln281, Lys511, and Tyr520 in the C-domain). The P<sub>1</sub>â² backbone carbonyl interacts with the conserved His331 and His491 in the N-domain (His353 and His513 in the C-domain). A conserved tyrosine (Tyr501 and Tyr523 in the N- and C-domains, respectively) can interact with either the P<sub>1</sub>â² peptide carbonyl or the zinc binding group (ZBG). Lipophilic interactions between nonpolar groups and the nonpolar subsites indicated in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> add to the potency of the inhibitor. These are mostly conserved between the two domains. However, the C-domain S<sub>2</sub>â² is more lipophilic and favors P<sub>2</sub>â² bicyclic groups, largely due to the replacement of Ser257 and Thr358 in the N-domain with Val379 and Val380 in the C-domain. The S<sub>1</sub> subsite is largely similar except for Thr496 in the N-domain being replaced with Val518 in the C-domain, making this subsite more hydrophobic in the C-domain.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic outline of the ACE binding site occupied by a typical ACEi drug. ACE inhibitors bind to the catalytic region of the active sites of both domains via a chelation interaction with the zinc atom and a hydrogen bond with the catalytic Glu residue. Salt-bridge and hydrogen bonding interactions with the respective Gln, Lys, and Tyr residues in the S<sub>2</sub>â² subsite hold in place the terminal P<sub>2</sub>â² carboxylic which is attached to a cyclic group. Hydrophobic side chain mimicking P<sub>1</sub> and P<sub>1</sub>â² groups interact with the respective hydrophobic S<sub>1</sub>â² and S<sub>2</sub>â² subsites. Different regions of the active site are color coded according to polarity as indicated in the legend.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There are various conserved residues in the S<sub>2</sub>â² subsite, and a conserved interaction between the backbone S<sub>2</sub>â² carbonyl with the backbone nitrogen of Ala334 in the N-domain (Ala356 in the C-domain). However, there are two key differences between the two binding sites with the N-domain to C-domain substitutions of Arg381/Glu403 and Tyr369/Phe391. Both of these substitutions could influence ligand binding. The substitution of a positive charge for a negative one, and a hydrophilic residue for a hydrophobic one provides an ideal set of interactions to be targeted for the design of a domain selective ACEi.</div><div class="NLM_p">Both of these substitutions were shown to impart N-domain selectivity in the case of the phosphinic tetrapeptide [(3<i>S</i>)-3-amino-4-(((1<i>R</i>)-1-(((<i>R</i>)-1-(((<i>R</i>)-1-amino-1-oxopropan-2-yl)amino)-1-oxopropan-2-yl)(hydroxy)phosphoryl)-2-phenylethyl)amino)-4-oxobutanoic acid] (RXP407),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> the first example of an N-domain selective ACEi. It contains all the core ACEi moieties with the addition of an aspartic acid in the P<sub>2</sub> position. RXP407 was shown to inhibit the N-domain with a 2500-fold selectivity over the C-domain. This selectivity was explained via a P<sub>2</sub> salt bridge with Arg381 and an H-bond partner Tyr369 with the N-domain. Substituting the positively charged Arg381 and the H-bond donor Tyr369 with the negatively charged Glu403 and the hydrophobic Phe391 in the C-domain could explain the dramatic disruption of binding observed in the C-domain.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">While the N-domain selective ACE inhibition observed for RXP407 proved to be an excellent proof-of-concept, the compound is too large and peptidic to be considered drug-like. Despite its shortcomings, RXP407 provides a template upon which to base the design of a drug-like N-domain selective ACEi. Designing an ACEi specific to just one domain of ACE requires careful consideration of the small structural differences in the respective domain active sites, such as the Arg381 to Glu403 and Tyr369 to Phe391 substitutions. This objective warrants a structure-based approach.</div><div class="NLM_p">Many high-resolution crystal structures detailing the interactions between this target and a myriad of different inhibitors have now been reported.<a onclick="showRef(event, 'ref12 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref16">(12, 16)</a> A combination of crystal structure data, a large collection of ACEi from decades of ACEi development, modern computer hardware, and state-of-the-art docking software have provided an ideal opportunity to develop novel N-domain selective ACEi.</div><div class="NLM_p last">In the search for drug-like N-domain selective ACEi, a large library of known ACEi was screened for a versatile P<sub>2</sub> group with the potential to probe the structureâactivity relationship (SAR) around the residue substitutions found in the S<sub>2</sub> subsite. Here we report an integrated approach to the identification of a novel drug-like N-domain selective ACEi. The approach entailed database mining, docking simulations, hit identification, synthesis, biological evaluation, and high-resolution crystal structure studies.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81205" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81205" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In an effort to identify new starting point leads for a novel N-domain selective ACEi, we screened the comprehensive Gunupati Venkata Krishna (GVK) Biosciences database, containing inhibitors against most biological targets and their associated SAR data.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(23)</a> The GVK database contained 24958 compounds with data from biological studies performed on human sACE. Within this set, 1832 unique compounds were found to contain a carboxylic acid ZBG with IC<sub>50</sub> inhibition data.</div><div class="NLM_p">Manual filtering was used to search for polar P<sub>1</sub>/P<sub>2</sub> moieties. Within this carboxylic acid ZBG subset, 13 compounds (<b>1</b>â<b>13</b>; <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) fit the filtering criteria of a polar group in the P<sub>1</sub>/P<sub>2</sub> position. This set of compounds contained four chemical series (series 1â4) defined by a shared chemical scaffold. Compounds not belonging to one of these scaffolds were left ungrouped. Once these promising compounds had been identified, their binding pose was predicted using constrained Glide docking.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(24)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Thirteen Compounds Identified from the GVK Database for Their Polar P<sub>2</sub> Groups<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0010.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>Table a</sup><p class="last">These compounds were docked to assess their potential for selectively inhibiting the N-domain. IC<sub>50</sub> values given are those reported in the GVK database. The IC<sub>50</sub> values of compounds <b>1</b> and <b>3</b> are marked with an asterisk (*) to indicate that these values were reported in the database but were later found to correspond to different compounds in the cited reference.</p></div></div><div></div></div><div class="NLM_p">The ability of Glide to evaluate metalâligand interactions is inadequate for the accurate prediction of an ACEi binding pose. This has been previously reported<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(25)</a> and is due to the prohibitively expensive quantum mechanical (QM) calculation required to accurately model the force field around transition metals. Consequently, Glide approximates the chelation interaction between a metal and a ligand using a template of precalculated interactions between each metal and its chelating species. Such an approximation does not account for the unique electrostatic environment of each metal inside each protein. The use of a precalculated generalized template for each metalâchelator interaction is problematic with ligands containing multiple metal-binding moieties of the same species. Most ACEi compounds fall into this category with two carboxylic acids. The use of chelation templates often fails to assign the correct metal binding group when presented with a choice of two carboxylic acids. Hence Glide could not efficiently recreate the native pose of a metal-chelating ACEi in the binding site without the use of docking constraints.</div><div class="NLM_p">In this case, five docking constraints were created in a docking grid centered around the Lisinopril ligand in the C-domain structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1O86">1O86</a><a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and RXP407 in the N-domain structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXQ">3NXQ</a>.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> These grids were assigned three positional constraints requiring the docked ligand to place either a carbon or nitrogen atom in one of three 1 Ã radius spheres indicated in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A. Interaction constraints were then created for a chelation interaction between the ZBG and the zinc metal (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). H-bond constraints were set between the terminal carboxylic acid and the H-bond donor Lys and Gln residues in the S<sub>2</sub>â² subsite (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. An illustration of the positional (A), metal chelation (B), and H-bonding (C) docking constraints created and implemented under various permutations when docking ligands against both the N- and C-domains of ACE. Positional constraints (A) were set holding either a carbon or nitrogen atom in one of the 1 Ã constraint spheres. The metal chelation constraint was created between the zinc atom and a carboxylic acid ZBG from the ligand. The H-bonding constraints were created between both the Lys489/511 and Gln259/281 donors from N-/C-domain and an H-bond acceptor from the P<sub>2</sub>â² group on the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Three different constraint conditions were created from different permutations of these five docking constraints, C1, C2, and C3 (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Different constraint conditions were employed due to the different structural requirements of each chemical series. In each case, one of the three constraint conditions enabled Glide to assign the correct ZBG and find a plausible docking pose (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Three Different Permutations of the Docking Constraints Employed to Help Find Plausible Docking Poses of the Given ACEi</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">condition</th><th class="colsep0 rowsep0" align="center">constraints</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">C1</td><td class="colsep0 rowsep0" align="left">carboxylicÂ acidâZn chelation, H-bond with Lys489/511 or GlnGln259/281</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">C2</td><td class="colsep0 rowsep0" align="left">carboxylicÂ acidâZnÂ chelation, amide/amine N in constraint sphereÂ 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">C3</td><td class="colsep0 rowsep0" align="left">carboxylicÂ acidâZn chelation, amide/amine N in constraint sphereÂ 3, amineÂ N, or alkyl C in constraint sphereÂ 1</td></tr></tbody></table></div></div><div class="NLM_p">All compounds were successfully docked with a plausible pose into at least one of the two active sites using one of the three constraint conditions (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). These details were then used to assess the potential of a compound to selectively inhibit the N-domain of ACE. This assessment of domain selectivity initially considered the distance from the ligand to a N atom from Arg381 and an O from Tyr369 in the N-domain and the nearest aromatic C atom from Phe391 and an O atom of Glu403 in the C-domain.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Summary of Constraint Conditions Used to Dock Each Compound in the Respective Catalytic Domains<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0011.gif" alt="" id="gr12" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0012.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup>Table a</sup><p class="last">The distances are provided between the nearest ligand atom and an N from Arg381 or an O from Glu403 and the atom from the ligand indicated in brackets.</p></div></div><div></div></div><div class="NLM_p"><a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> shows example representations of three compounds docked into the N- and C-domains. Compound <b>1</b> of series 1 displayed the ideal interactions with Tyr369 and Arg381 of the N-domain in the S<sub>2</sub> subsite. The other two compounds in this series demonstrated the versatility of this series with the modification to its P<sub>2</sub> group occurring in the last stage of its synthesis. Variability in the P<sub>2</sub> position is ideal for probing the SAR of the S<sub>2</sub> subsite and possible causes of N-domain selective ACE inhibition. Series 2 and 3 provided limited opportunities for probing S<sub>2</sub> interactions apart from a basic Lys group. Series 4 consisted of a pair of natural products with a promising P<sub>1</sub> acidic group. Despite the strong sACE inhibition, these two compounds appeared to be too short to make strong contacts with key S<sub>2</sub> residues that have been shown to influence N-domain selectivity, although a weak, longer range H-bond with Tyr369 is possible. The combination of the shorter length and limited synthetic versatility of a natural product left little opportunity to explore the P<sub>1</sub>/P<sub>2</sub> SAR of ACE using this series. An evaluation of the binding poses of the remaining three ungrouped compounds showed little opportunity for P<sub>1</sub>/P<sub>2</sub> SAR studies.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Poses of <b>1</b> (A,B), <b>4</b> (C,D), and <b>10</b> (E,F) docked into the N- (cyan; left) and C-domains (green; right). The two key ACEi binding features, a carboxylic acid Zn chelation and a second carboxylic acid acting as an H-bond acceptor in the S<sub>2</sub>â² subsite, are evident. Compound <b>1</b> in the N-domain has a twisted peptide bond for the P<sub>1</sub> proline and interactions between the carboxylic acid P<sub>2</sub> group with Tyr369 and Arg381, while in the C-domain <b>1</b> contains a trans proline and the P<sub>2</sub> carboxylic acid has no interactions. The bicylic link in compound <b>4</b> is too long and rigid to allow for a conventional orientation in either domain and causes the P<sub>2</sub>â² and ZBG carboxylic acid groups to bind in different positions in the N- and C-domains, respectively. This, along with the flexible alkyl chain, allows for a large variation in the position of the P<sub>2</sub> group. Compound <b>10</b> is too short to form interactions from its P<sub>2</sub> carboxylic group with Arg381 or Tyr369 in the N-domain; instead, it coordinates with the zinc ion. In the C-domain, there are no interactions with this group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">From the docking results, the diprolyl series 1 with its variable P<sub>2</sub> moiety, published by Greenlee et al.,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(17)</a> appeared to be an ideal scaffold to make contact with key residues in the S<sub>2</sub> subsite. In addition, the P<sub>2</sub> acid found in <b>1</b> made it an ideal candidate for N-domain selectivity as it docked with a salt-bridge interaction with Arg381 and formed an H-bond with Tyr369, while its prime groups made the conventional interactions of an ACEi scaffold. Compounds <b>1</b> and <b>3</b> were found to be incorrectly recorded entries of compounds of <b>2</b> and <b>15</b> published by Greenlee et al. (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Despite this error, the diprolyl series presents a promising scaffold with which to probe the P<sub>2</sub> SAR as a large range of amino acids and amino acid analogues can be added in the final stage of the synthesis via the formation of an amide bond. The synthesis of compound <b>1</b> requires the introduction of a urea group to the diprolyl scaffold. This is a departure from the synthetic scheme prescribed by Greenlee et al. However, compound <b>3</b> and several novel compounds can be synthesized using this method to attach different carboxylic acid containing monomers to the diprolyl scaffold.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Comparison of the Structures Found in the GVK Database and the Correct Structure Matching the Data Provided by Greenlee et al.</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0013.gif" alt="" id="GRAPHIC-d42e759-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0014.gif" alt="" id="gr14" /></img><div></div></div><div class="NLM_p">Greenlee et al. synthesized seven compounds in the diprolyl series. Of the seven compounds, three (<b>2</b>, <b>14</b>, and <b>15</b>) contained a terminal P<sub>2</sub> moiety which was free to interact with the Arg381/Glu403 and Try369/Phe391 residues in the N- and C-domains, respectively. A library of P<sub>2</sub> diprolyl variants was therefore modeled.</div><div class="NLM_p">The diprolyl series was expanded with a diverse range of amino acids and monomer carboxylic acid containing groups (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). These groups were chosen for their ability to probe the P<sub>2</sub> SAR with a carboxylic acid in a variety of positions and orientations. These compounds were docked using the fixed core method. The backbone was fixed in the most plausible pose from the first round of docking with flexibility only permitted in the P<sub>2</sub> position. The resulting poses were then minimized. The final poses can be easily visualized in the 2D ligplot diagrams provided in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_002.pdf" class="ext-link">Figure SI1</a>.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Summary of Docking Poses after MM-GBSA Minimization and Binding Energy Calculations and Domain Selectivity Predictions<a class="ref internalNav" href="#tbl5-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0015.gif" alt="" id="GRAPHIC-d42e800-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0016.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl5-fn1"><div class="footnote" id="tbl5-fn1"><sup>Table a</sup><p class="last">Each binding pose was assessed by measuring the distance between the ligand and atom indicated in brackets and a N atom from Arg381, an O-atom from Tyr369 in the N-domain with the distances between the nearest aromatic C atom from Phe391, and an O atom of Glu403 from the C-domain. Salt-bridge interactions and raised energy conformations are also noted along with each measurement. The MM-GBSA binding energy is also given as a quantitative metric with which to analyse each pose. Each pose is then judged as nonselective (NS) or N-domain selective (N). Compounds which were selected for synthesis have been marked with an asterisk (*).</p></div></div><div></div></div><div class="NLM_p">When reviewing the docked poses, the position of the P<sub>2</sub> group was primarily considered due to the rest of the diprolyl scaffold having conserved interactions. The distance between this group and the Tyr369/Arg381 and Phe391/Glu403 residues in the respective N- and C-domains were measured. Favorable H-bond and salt-bridge interactions with Tyr369/Arg381 were noted, and any additional interactions, both favorable and unfavorable toward the P<sub>2</sub> group, were considered.</div><div class="NLM_p"><a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> shows representative examples of three compounds docked into the N- and C-domains. The P<sub>2</sub> group of compound <b>16</b> shows a network of interactions consisting of a salt bridge with Arg381 and an H-bond with Tyr369 in the N-domain. In contrast, when docked into the C-domain, the charged P<sub>2</sub> group rotates away from the hydrophobic Phe391 and only forms a single interaction, albeit a strong H-bond with Arg522, the residue in the N-domain in the position equivalent to Arg500. This movement of the P<sub>2</sub> group of compound <b>16</b> between the N- and C-domains resulted in an orientation in the S<sub>3</sub> subsite that is typical of compounds that were either too long for the S<sub>2</sub> subsite or were not able to form favorable interactions with Tyr369/Phe391 and/or Arg381/Glu403.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Representative samples of ligand poses from the extended set of diprolyl compounds. (A,B), (C,D), and (E,F) show <b>16</b>, <b>17</b>, and <b>21</b>, respectively, docked in the N- (cyan; left) and C-domains (green; right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The P<sub>2</sub> group of compound <b>17</b> is one alkyl carbon longer than that of <b>16.</b> The most obvious effect of this difference is the change, for both the N- and C-domains, from a trans proline in the P<sub>1</sub> position of <b>16</b> to a cis proline in <b>17</b>. In the trans orientation, the P<sub>2</sub> backbone carbonyl interacts with the backbone nitrogen of Ala334 and Ala356 for the N- and C-domains, respectively. In contrast, with a cis proline, the P<sub>2</sub> backbone carbonyl interacts with Thr496 in the N-domain, which is not conserved in the C-domain, and the interaction is instead with Arg522. This cis proline is also observed for <b>22</b> with both the N- and C-domains, <b>19</b> with the N-domain, and <b>2</b>, <b>3</b>, and <b>15</b> with the C-domain. For the N-domain, the change to a cis proline with <b>17</b> only has small effect on the position of the P<sub>2</sub> carboxyl group, but the orientation indicates potentially stronger interactions with Tyr369 and Arg381. In addition, there is a large change in the position of the P<sub>2</sub> backbone nitrogen from interacting with the backbone carbonyl of Ala334 for <b>16</b> to H-bonds with Glu389 and Tyr501 for <b>17</b>. A comparison of <b>16</b> and <b>17</b> with the C-domain shows a shift of the P<sub>2</sub> carboxyl group closer to Phe391 for the docked <b>17</b>, and this causes the loss of interaction with Arg522 seen for <b>16</b>. Similar to the N-domain, there is a large change in position of the P<sub>2</sub> backbone nitrogen between <b>16</b> and <b>17</b> in the C-domain, which interacts with Ala356 for <b>16</b>, but Glu411 for <b>17</b>.</div><div class="NLM_p">Compound <b>21</b> is an example where the P<sub>2</sub> group is not an amino acid, and possibly a consequence of this is that the P<sub>1</sub> proline backbone is twisted in both the N- and C-domains and is therefore neither cis nor trans. In the N-domain, the P<sub>2</sub> carboxyl group forms the desired interactions with Tyr369 and Arg381, whereas in the C-domain, this group twists away from the hydrophobic Phe391 but does form an H-bond with Arg522. There is also a difference with the P<sub>2</sub> backbone carbonyl which interacts with Thr496 in the N-domain (not conserved in the C-domain) but Tyr523 in the C-domain.</div><div class="NLM_p">In addition to the qualitative assessment of the binding pose, a quantitative molecular mechanics-generalized Born surface area (MM-GBSA) binding energy calculation was performed to provide a metric with which to assess the binding prediction. The combination of the qualitative binding assessment and the quantitative MM-GBSA binding energy calculation for each ligand in each domain helped to form a binding prediction.</div><div class="NLM_p">Upon the formulation of the binding predictions, all compounds were deemed to be specific for either the N-domain (N) or C-domain (C) or nonselective (NS) (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). Considering these selectivity predictions and availability of starting materials, a representative set of seven compounds (indicated with an asterisk in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>) was selected for synthesis. These compounds were chosen for proof-of concept studies to test the hypothesis that an acid group in the P<sub>2</sub> position is crucial for N-domain selective ACE inhibition.</div><div id="sec2-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Chemistry</h3><div class="NLM_p">The diprolyl series was synthesized in six steps as depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. For reaction i, <i>N-</i>Boc pyrrolidine aldehyde was mixed with Ala-Pro in a MeOH/NH<sub>4</sub><sup>+</sup>Cl<sup>â</sup> solution at 25 Â°C for 36 h to produce <b>27</b> with a yield of 36%. This compound was then oxidized in an acidic HCl/MeOH solution at 25 Â°C for 2 days. After a quick workup, the product was dissolved in a solution of (Boc)<sub>2</sub>O and Et<sub>3</sub>N/MeOH to reprotect any Boc-amine groups hydrolyzed during a side reaction to produce <b>28</b> (72%), which was then loaded onto an Amberlyst 15 resin and stirred in MeOH at 60 Â°C for 10 days. Compound <b>29</b> was then eluted from the resin using HCl in dioxane (40%). The RCO<sub>2</sub>H group was then attached to <b>29</b> using triphenylphosphine (T3P) in a basic Et<sub>3</sub>N solution at 0â25 Â°C for 12 h to form <b>30_x</b>, with <b>x</b> denoting the final compound to be synthesized after the deprotection of <b>30_x</b> (for example, <b>30_2</b> was deprotected into compound <b>2</b>). The diastereomeric mixture of <b>30_x</b> was then separated via HPLC using a chiral column. The separated diastereomers were then de-esterified using LiOH in THF/H<sub>2</sub>O and Boc deprotected using HCl/dioxane if the compound still contained a Boc protected amine.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Diprolyl Series<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(i) TMSCN, MeOH/NH<sub>4</sub><sup>+</sup>Cl<sup>â</sup>, 25 Â°C, 36 h, 36%; (ii) (a) HCl in MeOH, 25 Â°C, 2 d, (b) (Boc)<sub>2</sub>O, Et<sub>3</sub>N/MeOH, 72%; (iii) (a) Amberlyst 15, MeOH, 60 Â°C, 10 days, (b) HCl/dioxane 25 Â°C, 12 h, 40%; (iv) (a) RCO<sub>2</sub>H, T3P, Et<sub>3</sub>N, DCM, 0â25 Â°C, (b) chiral HPLC diastereomer separation; (vi) (a) LiOH, THF/H<sub>2</sub>O, (b) HCl/dioxane.</p></p></figure></div><div id="sec2-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Biological Evaluation</h3><div class="NLM_p">Docking and MM-GBSA simulations predicted compounds <b>2</b> and <b>14</b> to be strong inhibitors of both ACE domains and <b>15</b> to potentially have some C-domain specificity, while <b>3</b>, <b>16</b>, <b>17</b>, and <b>18</b> were predicted to show a degree of selectivity toward the N-domain. Only <b>3</b>, <b>16</b>, and <b>17</b> met both the qualitative and quantitative criteria for selectivity, albeit the difference between the MM-GBSA binding energy for the N- and C-domains for <b>3</b> was small.</div><div class="NLM_p">The most bioactive diastereomer (SSSS) of each diprolyl compound was identified via ACE inhibition assays. In each case, the more bioactive diastereomer inhibited both domains of ACE with an IC<sub>50</sub> 2 orders of magnitude lower than the less bioactive (RSSS) diastereomer, in line with the findings of Greenlee et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(17)</a> The <i>K</i><sub>i</sub> value of the more bioactive diastereomer of the seven synthesized diprolyl compounds was then evaluated against both domains via a fluorimetric competitive inhibition assay (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). All compounds inhibited both domains of ACE in the nanomolar range. Compounds <b>3</b>, <b>14</b>, and <b>18</b> showed very low nanomolar inhibition of both domains with a small degree of C-domain selectivity. Of the four compounds predicted to selectively inhibit the N-domain of ACE, only <b>16</b> was specific toward the N-domain with a similar affinity to <b>3</b>, <b>14</b>, and <b>18</b> but an 84-fold lower affinity for the C-domain. In contrast, compound <b>17</b> had the same high affinity for both domains showing no specificity. Compounds <b>2</b> and <b>15</b> were lower affinity inhibitors for both domains, with <b>2</b> showing a small level of C-domain specificity.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Summary of <i>K</i><sub>i</sub> Values for Each Compound Synthesised and the Observed N-Domain Selectivity Factor<a class="ref internalNav" href="#tbl6-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="Â±">N-domain K<sub>i</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="Â±">C-domain <i>K</i><sub>i</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char=".">N-domain selectivity factor</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char="Â±">279.48Â Â±Â 46.93</td><td class="colsep0 rowsep0" align="char" char="Â±">44.38Â Â±Â 3.58</td><td class="colsep0 rowsep0" align="char" char=".">0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char="Â±">3.37Â Â±Â 0.55</td><td class="colsep0 rowsep0" align="char" char="Â±">0.22Â Â±Â 0.02</td><td class="colsep0 rowsep0" align="char" char=".">0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char="Â±">4.45Â Â±Â 1.15</td><td class="colsep0 rowsep0" align="char" char="Â±">0.31Â Â±Â 0.11</td><td class="colsep0 rowsep0" align="char" char=".">0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char="Â±">70.35Â Â±Â 9.73</td><td class="colsep0 rowsep0" align="char" char="Â±">167.80Â Â±Â 31.33</td><td class="colsep0 rowsep0" align="char" char=".">2.39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char="Â±">11.45Â Â±Â 1.37</td><td class="colsep0 rowsep0" align="char" char="Â±">961.02Â Â±Â 153.00</td><td class="colsep0 rowsep0" align="char" char=".">83.93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char="Â±">9.63Â Â±Â 1.56</td><td class="colsep0 rowsep0" align="char" char="Â±">17.39Â Â±Â 2.38</td><td class="colsep0 rowsep0" align="char" char=".">1.81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char="Â±">16.86Â Â±Â 1.24</td><td class="colsep0 rowsep0" align="char" char="Â±">2.29Â Â±Â 0.27</td><td class="colsep0 rowsep0" align="char" char=".">0.14</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl6-fn1"><div class="footnote" id="tbl6-fn1"><sup>a</sup><p class="last">N-Domain selectivity is calculated as the C-domain <i>K</i><sub>i</sub>/N-domain <i>K</i><sub>i</sub>. All <i>K</i><sub>i</sub> values are reported with a 95% confidence interval.</p></div></div></div><div class="NLM_p last">The design and docking procedures have produced high affinity inhibitors of ACE. The data also shows that introducing the P<sub>2</sub> carboxylic acid group can give the desired N-domain specificity as seen with compound <b>16</b>. It is not immediately obvious from the docking results why compound <b>17</b> has a higher affinity for the C-domain than <b>16</b> and therefore why <b>17</b> loses N-domain specificity. Compound <b>17</b> docked with the cis proline conformation. Although this causes the loss of the H-bond with Arg522, and there are no additional interactions, the cis proline conformation may allow the conserved interactions to be stronger. Further SAR and structural studies are needed to fully understand these subtle differences.</div></div><div id="sec2-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> X-ray Crystallography</h3><div class="NLM_p">Two structures (A and B) of the N-domain of ACE in complex with compound <b>16</b> are presented in this paper (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_002.pdf" class="ext-link">Table SI1</a>), both of which crystallized in the usual <i>P</i>1 space group previously seen for the N-domain of ACE. These two structures have a larger unit cell (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_002.pdf" class="ext-link">Table SI1</a>) than what is typical of the ACE N-domain, with four chains in the asymmetric unit, similar to the structure of the N-domain in complex with amyloid-Î² 4â10 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AM8">5AM8</a>). However, the overall structure of all chains in the unit cells of both structures are similar to each other and previously reported structures. Residues 10â23 and 74â90 form part of the proposed N-terminal hinge region and often show flexibility in crystal structures with increased temperature factors (<i>B</i>-factors) and poor electron density. This is seen to some degree in every chain of both reported structures but to a greater extent in chain B of each structure. In addition, chain B of structure A shows a particularly high degree of flexibility with some evidence of a second conformation apparent in the <i>mFo</i>â<i>DFc</i> electron density map of this region. The electron density for this second conformation is not sufficient to model the structure, but to account for this, the first conformation of this region has been assigned a 0.8 occupancy. Another difference between the chains in the asymmetric unit is that extra 7 (structure A) and 8 (structure B) residues can be resolved in Chain D at the C-terminus due to the proximity and interaction with a symmetry-related molecule. In both structures, there is a region of positive density in the <i>mFo</i>â<i>DFc</i> difference map adjacent to residue Asp177 of chain D. This cannot be properly accounted for by waters or any other constituent of the purification or crystallization conditions and has therefore been left unmodeled. A further significant difference between the molecules is seen with Arg381 as described in detail below.</div><div class="NLM_p">The active site of the N-domain/compound <b>16</b> complex structure shows clear electron density for the ligand in every molecule of the asymmetric unit in both structures (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). Compound <b>16</b> binds to the active site in the expected position and orientation coordinating the zinc metal with its carboxylic acid ZBG, while the remainder of the ligand occupies the S<sub>2</sub>â², S<sub>1</sub>â², S<sub>1</sub>, and S<sub>2</sub> subsites. The C-terminus mimic of <b>16</b> occupies the usual position in the S<sub>2</sub>â² subsite forming strong hydrogen bonding and salt-bridge interactions with Gln259, Lys489, and Tyr498 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B). These residues are conserved in the C-domain. The P<sub>1</sub>â² backbone carbonyl forms hydrogen bonds with the His331 and His491 side chains and the P<sub>1</sub>â² backbone nitrogen with the backbone carbonyl of Ala332. These residues are also conserved in the C-domain, and the interactions are typical in most structures of the N-and C-domains in complex with different ligands. The ZBG of <b>16</b> also interacts with the C-domain conserved residues Glu362 and Tyr501. The S<sub>1</sub> subsite contains the Thr496 to Val518 substitution between the N- and C-domains, making the S<sub>1</sub> subsite more hydrophobic in the C-domain. This would most likely favor the pseudoproline P<sub>1</sub> group of <b>16</b> and therefore cannot account for the difference in affinity toward <b>16</b>.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Schematic representation of compound <b>16</b> bound to the N-domain of ACE overlaid with the 2<i>mFo</i>â<i>DFc</i> (blue, contoured at 1Ï level) and <i>mFo</i>â<i>DFc</i> (green, contoured at 3Ï level) electron density omit maps. The zinc ion is shown as a green sphere with the coordinating side chains shown as sticks. (B) Ligplot representation of the binding site interactions of compound <b>16</b> in complex with ACE N-domain. The Arg381 conformation that could possibly provide water mediated interactions with <b>16</b> is shown for completeness, although this is not a major form.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The interactions between <b>16</b> and the S<sub>2</sub>â², S<sub>1</sub>â², and S<sub>1</sub> subsites involve largely conserved residues. The observed specificity for the N-domain must therefore arise from the interactions between the P<sub>2</sub> group and residues in the S<sub>2</sub> subsite. Because the binding site cavity begins to widen near the S<sub>2</sub> subsite, there are numerous residues that could interact with the ligand depending on the orientation and length of the P<sub>2</sub> moiety. As already mentioned, Tyr369 and Arg381 in the N-domain are the only two residues not conserved in the C-domain that could realistically provide different interactions with <b>16</b>. These residues are replaced by Phe391 and Glu403 respectively in analogous positions in the C-domain. As previously mentioned, interactions with Tyr369 and Arg381 were targeted due to the interactions observed with RXP407 in the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXQ">3NXQ</a> N-domain complex structure. In all the molecules of the asymmetric unit for both structures, Arg381 is poorly defined beyond the CÎ± (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>AâC). The weak density suggests a high degree of flexibility, with two orientations partly explaining some of this flexibility. However, neither conformation interacts directly with compound <b>16</b>. There is a varying amount of electron density visible for these conformations in the different chains, but each conformation is visible to some extent in at least one chain of both structures. This collective observation across the eight chains was used to model these alternative conformations into weak density with a high degree of certainty (a true experimentally observed âdynamic pictureâ emerged from the crystal structures with multiple copies in the asymmetric unit of the crystal lattice) and is the reason for including two versions of the structure. One of these conformations allows for a water mediated interaction between Arg381 and the P<sub>2</sub> carboxylic acid, with the water bridging both oxygens of the carboxylic acid. However, due to the high degree of flexibility of Arg381 observed in these structures, it is unlikely that interactions with this residue contributes significantly to the N-domain specificity of <b>16</b>. The second conformation orientates away from the bound ligand, and there is variation in this conformation across the different chains in the asymmetric units, further confirming the flexibility of this residue.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Close-up views of the P<sub>2</sub> carboxylic acid binding site. The 2<i>mFo</i>â<i>DFc</i> electron density map (contoured at 1Ï level) is shown in blue, and the zinc ion and relevant water molecules are shown as green and red spheres, respectively. (AâC) Schematic representation showing the possible alternate conformations of Arg381 in three molecules of the complex structure with ACE N-domain. This highlights the weak density for Arg381. (D) Schematic representation showing the strong interaction between Tyr369 and the P<sub>2</sub> carboxylic acid of compound <b>16</b> and the water mediated interaction with Arg500.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast to Arg381, the crystal structure shows a strong H-bond interaction between the P<sub>2</sub> carboxylic acid of <b>16</b> and the hydroxyl group of Tyr369 (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>D). The equivalent Phe391 residue in the C-domain lacks the hydroxyl hydrogen bond donor group while creating a hydrophobic environment that is not suitable for charged moieties. Therefore, it is likely that the interaction between <b>16</b> and Tyr369 is a major contributing factor for the N-domain specificity of this compound.</div><div class="NLM_p last">The P<sub>2</sub> carboxylic acid also has water-mediated interactions with Arg500 (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>D), contributing to the high affinity of <b>16</b> for the N-domain. The docking results predicted this as a long-range (3.4 Ã) direct interaction and an H-bond between <b>16</b> with the equivalent residue Arg522 of the C-domain. Therefore, the interaction seen with Arg500 of the N-domain may not contribute to the specificity of <b>16</b>.</div></div><div id="sec2-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Comparison of Crystallography and Docking Results</h3><div class="NLM_p">The crystal structure of compound <b>16</b> in the N-domain overlays very closely with the predicted docked pose (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). An RMSD of 1.182 Ã was measured between the predicted and crystallized poses. The diprolyl scaffold shows a strong agreement with just a small difference in the orientation but not the position of the P<sub>2</sub> aspartate. This close overlay strongly validates the constrained docking methodology. The only notable difference between the predicted and crystallized structures was the lack of a salt bridge between the P<sub>2</sub> carboxylic acid and Arg381. This is a consequence of docking the ligand against the crystal structure of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXQ">3NXQ</a>, where Arg381 is oriented inward to make a salt-bridge contact with the P<sub>2</sub> carboxylic acid of the larger RXP407 ligand. The choice of the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXQ">3NXQ</a> structure is justified for this purpose to maximize the identification of compounds that could potentially interact with Arg381.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) An overlay of the crystal (orange) and docked (cyan) structures of <b>16</b> in the N-domain of ACE. The zinc ions in the crystal and docked structures are shown as green and gray spheres, respectively. The two poses of the ligand are closely correlated (RMSD = 1.182 Ã), while Arg381 is seen to be highly flexible, with two conformations of this flexibility being visible to varying extents in the different chains of the asymmetric units. (B) An overlay of compound <b>16</b>âN-domain crystal structure (orange) and <b>16</b> docked in to the C-domain (green). The zinc ions in the crystal and docked structures are shown as green and gray spheres, respectively. The major portion of the scaffolds overlay between the two structures, while a shift is observed for P<sub>2</sub> carboxylic acid group, showing interactions with Tyr369 and Arg522 for the N- and C-domains, respectively. In addition, there is a water mediated interaction with Arg500 in the N-domain.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Overlaying the docked C-domain pose of <b>16</b> with the N-domain crystal structure (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B) illustrates the change in orientation of the P<sub>2</sub> group that is possibly required to avoid an unfavorable interaction with Phe391 in the C-domain. This may also be driven by the favorable direct interaction with Arg522 that resulted from this change. In the N-domain crystal structure, the interaction with the equivalent Arg500 is water mediated.</div><div class="NLM_p last">The docking results for <b>16</b> are therefore consistent with the results from the crystal structure and the biological assay with <b>16</b> having a greater number of interactions with the N-domain than with the C-domain, particularly the strong H-bond with Tyr369, thereby explaining the specificity for the N-domain.</div></div><div id="sec2-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Comparison of N-Domain Specific Inhibitor Structures</h3><div class="NLM_p">The work presented here investigated whether N-domain specificity could be achieved by targeting interactions with Tyr369 and Arg381 in the N-domain S<sub>2</sub> subsite. These residues were identified for their important interactions contributing to the N-domain specificity of RXP407.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> A comparison of crystal structures of the N-domain in complex with compound <b>16</b> and RXP407 (depicted as an overlay in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>A) show the compounds occupying very similar positions with the majority of interactions conserved. In the S<sub>2</sub> subsite, <b>16</b> retains the Tyr369 interaction, but Arg381 at best only provides a transient water-mediated interaction. This is consistent with the 2500-fold N-domain selectivity of RXP407 compared with compound <b>16</b> showing an 84-fold N-domain selectivity. Compared to RXP407, the P<sub>2</sub> carboxylic acid of <b>16</b> is shifted approximately 0.5 Ã away from the Arg381 location due to differences in ligand structure. This small shift appears to be sufficient to prevent a strong interaction with Arg381, indicating the sensitivity of the binding to slight changes in ligand structure.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Overlays of the ACE N-domain complex structures of compound <b>16</b> (orange) with (A) RXP407 (cyan) and (B) 33RE (lilac). The zinc ions are shown as a green sphere for the compound <b>16</b> structure and gray spheres for the RXP407 and 33RE structures. Water molecules involved in mediated interactions between <b>16</b> and the binding site are shown as red spheres. The water molecule marked * is conserved in the RXP407 structure in an almost identical position. A water molecule in the 33RE structure that performs a similar mediated interaction is shown as a pink sphere.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">33RE is another N-domain specific inhibitor, with identical groups to RXP407 in the P<sub>2</sub>â², P<sub>1</sub>â², and P<sub>1</sub> positions, but with the P<sub>2</sub> group including the substitution of the carboxylic acid for a tetrazole group. 33RE has a 1000-fold specificity toward the N-domain. An overlay of the 33RE structure<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(26)</a> with that of <b>16</b> shows the P<sub>2</sub> groups occupy a similar position (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>B). The tetrazole of 33RE interacts with Tyr369, similar to both RXP407 and <b>16</b> but without a direct interaction with Arg381. It is interesting to note that due to this lack of interaction, the Arg381 in the 33RE structure overlays with the second conformation of Arg381 in the <b>16</b>-complex structure. Comparison of the P<sub>2</sub> group interaction does not explain why 33RE shows a greater specificity for the N-domain than observed with <b>16</b>.</div><div class="NLM_p last">Previous analysis of N-domain specificity shown by RXP407 and 33RE<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(27)</a> indicated that a P<sub>2</sub>â² carboxylic acid in place of an amide group greatly reduces, or even abolishes, the N-domain specificity. Both RXP407 and 33RE contain P<sub>2</sub>â² amide groups, whereas <b>16</b> has a carboxylic acid. It is possible that changing this P<sub>2</sub>â² carboxylic acid of <b>16</b> to an amide may greatly increase the specificity for the N-domain. In addition, it is conceivable that compounds <b>3</b>, <b>17</b>, and <b>18</b>, predicted to be N-domain selective but shown to be nonspecific or even having some C-domain specificity, are having their P<sub>2</sub> N-domain selectivity masked by the P<sub>2</sub>â² carboxylic acid. Synthesis of P<sub>2</sub>â² amide analogues of these compounds would therefore be of considerable interest.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61720" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61720" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have investigated a series of diprolyl compounds for specificity toward the N-domain catalytic site of ACE by screening the GVK database for potentially N-domain selective P<sub>2</sub> chemical groups. The series was expanded with a diverse range of amino acids and monomer carboxylic acid containing groups to explore the S<sub>2</sub> subsite. These compounds were assessed using molecular docking and MM-GBSA to estimate their relative binding affinity. Nine compounds with different P<sub>2</sub> groups were predicted to bind selectively to the N-domain. A representative set of seven compounds, four of which were predicted to be N-domain selective, one C-domain specific and two nonspecific, were synthesized using an six-step synthetic route. While six of the compounds showed highly potent N- and C-domain inhibition, only the P<sub>2</sub> Asp derivative (<b>16</b>) was N-domain-selective and bound with an 84-fold higher affinity than the C-domain.</div><div class="NLM_p">The high-resolution crystal structures of compound <b>16</b> in complex with the N-domain showed that the ligand position, orientation, and interactions largely agreed with the docking results. The only significant difference was the lack of a salt bridge with Arg381. This residue was shown to be highly flexible, forming only a transient water mediated interaction with the P<sub>2</sub> carboxylic acid. Instead, hydrogen bonding of <b>16</b> with Tyr369 is likely to be the interaction responsible for N-domain selectivity. The close correlation between the predicted and crystallized pose of <b>16</b> reaffirms accuracy of the pose prediction, while the movement of Arg381 highlights the shortcomings of a fixed receptor prediction.</div><div class="NLM_p last">The discovery of the diprolyl series has provided a more drug-like scaffold with which to probe the S<sub>2</sub> subsite of ACE for N-domain selectivity. While the discovery of <b>16</b>, a novel N-domain selective ACEi is encouraging, it highlights the need for a more extensive SAR study probing the S<sub>2</sub> subsite to better define the chemical space responsible for N-domain selective ACE inhibition. A previous study showed a complete loss in N-domain specificity caused by a change from a P<sub>2</sub>â² amide to carboxylic acid. That SAR masked the contribution of the P<sub>2</sub> group.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The present study has identified the first drug-like N-domain selective ACEi. Compound <b>16</b> serves as an excellent starting point for further SAR studies which could realistically develop into a viable N-domain selective ACEi and achieve the original goal of treating tissue fibrosis without affecting blood pressure.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21142" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21142" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Computational</h3><div class="NLM_p">All computational predictions were carried out using desktop workstations running the CentOS 6.5 OS using the Glide utility included in the Schrodinger 2014-1 suite release. PDB entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXQ">3NXQ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1O86">1O86</a> of the respective N- and C-domains were prepared using the Maestro prepwizard to complete the structure with the addition of bond orders and missing side chains. The nonpeptide ligands were edited manually to reflect their correct structure. Nonbound waters were removed, and in the case of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXQ">3NXQ</a> the less complete chain B was removed. The automated optimize and Impref protocols were then run to refine the structures. Glide docking grids of default length were created centered on the native Lisinopril and RXP407 ligands, each with one metal chelation constraint, three positional constraints, and two S<sub>2</sub>â² H-bond constraints as indicated in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>. Docking simulations were performed iteratively using the Glide SP setting with constraint conditions CS1âCS3 enforced until a plausible docking pose was found.</div><div class="NLM_p last">Binding energy was calculated using the Prime MM-GBSA minimization and binding energy calculation package provided with the Schrodinger suite. MM-GBSA calculations were performed using the VSGB solvent model. The minimization was performed with flexibility tolerated for all protein atoms within a 15 Ã radius of the ligand.</div></div><div id="sec4-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis</h3><div id="sec4-2-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Reagents and Solvents</h4><div class="NLM_p last">All chemical reagents and anhydrous solvents used during the synthesis of the Enalaprilat analogue were purchased from Sigma-Aldrich. HPLC grade solvents were purchased from Sigma-Aldrich, Merck, and Microsep</div></div><div id="sec4-2-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Chromatography</h4><div class="NLM_p last">Thin layer chromatography (TLC) reaction monitoring was performed using Merck F<sub>254</sub> aluminum-backed silica gel 60 plates. Spots were visualized with either ultraviolet (UV) light (254/366 nm), anisaldehyde, or ninhydrin stains. Reaction products were all purified via column chromatography using Merck Kieselgel 60:70â230 mesh via gravitational column chromatography and flash chromatography.</div></div><div id="sec4-2-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> HPLC</h4><div class="NLM_p">Diastereomer separation was performed using a thermo separation system comprising an Analytical Technologies P2230 HPLC pump with an automated sampler and Analytical Technologies UV 2230 variable wavelength detector set to detect absorbance at 214 nm. The stationary phase column used was a Gemini NX-C18 3.5 Î¼m (4.6 mm Ã 50 mm) (Phenomenex, Torrance, CA).</div><div class="NLM_p last">Peak purity of each compound was determined using preparatory HPLC with a thermo separation system comprising a Spectra series P200 pump, an AS100 automated sampler, and a UV 100 variable wave detector. The UV detector was set to monitor peak absorption at 214 nm. A Waters X-bridge C18 5.0 Î¼m (4.6 mm Ã 150 mm) (Phenomenex, Torrance, CA) column stationary phase used was fitted to a Supelguard  Ascentis  C18 guardcartridge (2 cm Ã 40 mm, 3 Î¼m) (Supelco Analytical, Bellefonto, PA).</div></div><div id="sec4-2-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> LC-MS</h4><div class="NLM_p last">Liquid chromatography mass spectrometer (LC-MS) analysis on each compound was performed using an Agilent 1260 Infinity binary pump, Agilent 1260 Infinity diode array detector (DAD), Agilent 1290 Infinity column compartment, Agilent 1260 Infinity standard autosampler, and an Agilent 6120 quadruple (single) mass spectrometer equipped with an APCI and ESI multimode ionization source. Compound purity was determined using an Agilent LC-MS with a Kinetex Core C18 2.6 Î¼m column (50 mm Ã 3 mm).</div></div><div id="sec4-2-5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> NMR</h4><div class="NLM_p last">All NMR spectra were recorded a Bruker Ultrashield-Plus spectrometer (<sup>1</sup>H 400 MHz; <sup>13</sup>C 100 MHz), with compounds dissolved in either deuterated methanol (MeOD-<i>d</i><sub>4</sub>) or deuterium oxide (D<sub>2</sub>O) solvents.</div></div><div id="sec4-2-6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Inhibitions Assays</h4><div class="NLM_p">The Ndom389 and tACE Î36NJ constructs of the respective N- and C-domain samples were diluted to 10 and 5 nM respectively from frozen stock solutions. Then 50 mM stock solutions of each compound were prepared in deionized water. Aliquots of these stock solutions were then diluted to 10 mM with deionized H<sub>2</sub>O, followed by dilution into a phosphate buffer (100 mM KHPO<sub>4</sub>, pH 8.3, 300 mM NaCl, 10 Î¼M ZnSO<sub>4</sub>, 1 mg/mL albumin).</div><div class="NLM_p last">Ten-fold dilution series running from 100 Î¼M to 1 nM of each compound were prepared in buffer. Then 40 Î¼L of each solution in the series was mixed with 40 Î¼L of the enzyme in buffer. A 10 Î¼M solution of Lisinopril in buffer was used for positive control, while buffer was used for the negative control. The mixtures were incubated at room temperature for 30 min, after which triplicate 20 Î¼L aliquots of each mixture were loaded into three individual wells of a 96-well plate. Then 30 Î¼L of 1 mM Z-FHL substrate (Bachem Ltd., Bubenhof, Switzerland) was added to each well. The plate was then incubated at 40 Â°C for 15 min in a shaker. The reactions were then stopped and the product was derivatized with the addition of 190 Î¼L of a base and <i>O</i>-phthaldehyde solution (0.28 M NaOH and 7 mM <i>O</i>-pthaldehyde) to each well followed by further incubation on a shaker at room temperature for 10 min. The wells were then quenched with 25 Î¼L of 3 M HCl, and the plate was read on a fluorimeter (Varian Inc., Mulgrave, Victoria, Australia) with an excitation wavelength at 360 nm and an emission wavelength at 485 nm. After approximating the IC<sub>50</sub> values from the 10-fold dilution series of each compound, nine solutions of a 2-fold dilution series spanning the approximated IC<sub>50</sub> were prepared. Activity curves were plotted to determine the IC<sub>50</sub> of each compound. For better inhibition comparisons between the two domains of ACE, the <i>K</i><sub>i</sub> values were calculated using the equation given below:<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/jm-2017-01478z_m001.gif" alt="" /></img></span></div></div></div><div id="sec4-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Characterization</h3><div id="sec4-3-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> General Ester Deprotection Method</h4><div class="NLM_p last">Five equivalents of a 4.0 M aqueous solution of LiOH were added to a 0.1 M solution of the methyl ester protected form of each diprolyl compound in a THF/MeOH/H<sub>2</sub>O (3:2:1) solution. The mixture was stirred for 2 h at room temperature. The mixture was then diluted with 5 mL of water then neutralized by adding 1 N HCl dropwise. The compound was either dried and purified via Prep-HPLC or the crude product was Boc-deprotected.</div></div><div id="sec4-3-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Boc Deprotection Method</h4><div class="NLM_p last">Crude Boc protected compound was dissolved in chilled 4 M HCl/dioxane under N<sub>2</sub> at 0 Â°C. The mixture was stirred for 30 min at 0 Â°C and monitored via TLC. Upon completion, the solvent was evaporated and the residue was washed with diethyl ether. Purities of all reported compounds with the exception of <b>3</b> (93.79%) were shown to be greater than 95% by HPLC chromatograms on an Agilent 1260 LCMS system.</div></div><div id="sec4-3-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> [((<i>S</i>)-Carboxy((<i>S</i>)-1-glycylpyrrolidin-2-yl)methyl)-<span class="smallcaps smallerCapital">l</span>-alanyl-<span class="smallcaps smallerCapital">l</span>-proline] (<b>2</b>)</h4><div class="NLM_p last">The general ester deprotection was performed on 90 mg of <b>30_2</b>, after which the crude residue was Boc deprotected via the general Boc deprotection method, after which it was purified via HPLC to yield 65 mg (97%) of <b>2</b> as a sticky yellow solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) Î´ 4.42 (m, 2H), 4.29 (q, <i>J</i> = 6.9 Hz), 3.95â3.84 (m, 3H), 3.54 (m, 2H), 3.48â3.33 (m, 2H), 2.25 (s, 1H), 2.15 (m, 1H), 2.04 (m, 1H), 1.95 (m, 4H), 1.82 (m, 1H), 1.50 (d, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O): Î´ 177.1, 176.2, 174.7, 170.9, 70.5, 63.1, 56.3, 55.5, 52.3, 51.5, 42.9, 31.8, 26.9, 25.9, 24.1, 23.5. LC-ESI-MS (+ve ion mode): 371 [M + H]<sup>+</sup>. HPLC purity 97.4%, <i>t</i><sub>r</sub> = 5.66 min.</div></div><div id="sec4-3-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> [((<i>S</i>)-((<i>S</i>)-1-(4-Aminobenzoyl)pyrrolidin-2-yl)(carboxy)methyl)-<span class="smallcaps smallerCapital">l</span>-alanyl-<span class="smallcaps smallerCapital">l</span>-proline] (<b>3</b>)</h4><div class="NLM_p last">The general ester deprotection was performed on 50 mg of <b>30_3</b>, after which the crude residue was Boc deprotected via the general Boc deprotection method, after which it was purified via Prep HPLC to yield 30 mg (78%) of <b>3</b> as a sticky yellow solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) Î´ 7.61â7.51 (m, 2H), 7.27 (dd, <i>J</i> = 8.7, 2.9 Hz, 2H), 4.51 (t, <i>J</i> = 8.5 Hz, 1H), 4.33 (q, <i>J</i> = 6.9 Hz, 1H), 4.29 (m, 1H), 3.96 (m, 1H), 3.62â3.36 (m, 4H), 2.21 (m, 2H), 2.13â2.00 (m, 1H), 1.98â1.79 (m, 4H), 1.66 (m, 1H), 1.50 (d, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O): Î´ 176.3, 174.8, 174.1, 168.3, 133.9, 128.1, 127.7 (2C), 126.8 (2C), 67.5, 66.1, 60.7, 58.2, 52.7, 48.9, 28.1, 26.4, 23.7, 22.1, 21.1. LC-ESI-MS (+ve ion mode): 433 [M + H]<sup>+</sup>. HPLC purity 93.79%, <i>t</i><sub>r</sub> = 4.95 min.</div></div><div id="sec4-3-5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> [((<i>S</i>)-((<i>S</i>)-1-Benzoylpyrrolidin-2-yl)(carboxy)methyl)-<span class="smallcaps smallerCapital">l</span>-alanyl-<span class="smallcaps smallerCapital">l</span>-proline] (<b>14</b>)</h4><div class="NLM_p last">The general ester deprotection was performed on 45 mg of <b>30_14</b> and then purified via prep HPLC to obtain 26 mg (62%) of <b>14</b> as a sticky white-yellow solid. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) Î´ 7.62â7.55 (m, 2H,), 7.55â7.44 (m, 3H, H-12), 4.72 (t, <i>J</i> = 8.7 Hz, 1H), 4.52â4.47 (m, 1H), 4.37, (d, <i>J</i> = 1.9 Hz, 1H), 4.23 (q, <i>J</i> = 6.8 Hz, 1H), 3.70 (m, 1H), 3.60 (m, 2H), 3.50 (d, 1H), 2.44â2.25 (m, 3H, H-2), 2.12â1.93 (m, 4H), 1.87â1.73 (m, 1H), 1.64 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>): Î´ 175.9, 175.1, 173.2, 168.2, 132.3, 129.4, 127.9 (2C), 126.5 (2C), 66.3, 64.2, 59.5, 58.3, 50.5, 49.3, 27.2, 25.3, 23.4, 22.7, 22.1. LC-ESI-MS (+ve ion mode): 418 [M + H]<sup>+</sup>. HPLC purity 98%, <i>t</i><sub>r</sub> = 8.26 min.</div></div><div id="sec4-3-6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> [((<i>S</i>)-((<i>S</i>)-1-(<span class="smallcaps smallerCapital">l</span>-Phenylalanyl)pyrrolidin-2-yl)(carboxy)methyl)-<span class="smallcaps smallerCapital">l</span>-alanyl-<span class="smallcaps smallerCapital">l</span>-proline] (<b>15</b>)</h4><div class="NLM_p last">The general ester deprotection was performed on 50 mg of <b>30_15</b>, after which the crude residue was Boc deprotected via the general Boc deprotection method, after which it was purified vi prep-HPLC to yield 38 mg (97%) of <b>15</b> as a sticky yellow solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) Î´ 7.32 (m, 3H), 7.25 (m, 2H), 4.51 (t, <i>J</i> = 7.6 Hz, 1H), 4.40â4.30 (m, 3H, H-4), 3.82 (d, <i>J</i> = 4.4 Hz, 1H), 3.64 (d, 2H), 3.54â3.39 (m, 2H), 3.23 (dd, <i>J</i> = 14.7, 5.1 Hz, 1H), 3.06 (dd, <i>J</i> = 14.8, 7.7 Hz, 1H), 2.21 (m, 1H), 2.07 (m, 1H), 2.00â1.81 (m, 6H), 1.48â1.36 (m, 3H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O): Î´ 177.1, 175.3, 174.8, 171.2, 136.7, 129.1 (2C), 128.5 (2C), 127.2, 68.3, 67.5, 62.1, 57.3, 53.6, 51.9, 49.2, 40.1, 29.3, 25.7, 24.1, 22.8, 22.1. LC-ESI-MS (+ve ion mode): 461 [M + H]<sup>+</sup>. HPLC purity 97.4%, <i>t</i><sub>r</sub> = 5.52 min.</div></div><div id="sec4-3-7" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> [((<i>S</i>)-((<i>S</i>)-1-(<span class="smallcaps smallerCapital">l</span>-Aspartyl)pyrrolidin-2-yl)(carboxy)methyl)-<span class="smallcaps smallerCapital">l</span>-alanyl-<span class="smallcaps smallerCapital">l</span>-proline] (<b>16</b>)</h4><div class="NLM_p last">The general ester deprotection was performed on 230 mg of <b>30_16</b>, after which the crude residue was Boc deprotected via the general Boc deprotection method, after which it was purified via prep-HPLC to yield 85 mg (49%) of <b>16</b> as a sticky yellow-brown solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) Î´ 4.60â4.50 (m, 1H), 4.48â4.35 (m, 2H), 4.33â4.24 (m, 1H), 3.85 (d, <i>J</i> = 2.1 Hz), 3.70â3.48 (m, 3H), 3.47â3.35 (m, 1H), 3.03â2.91 (m, 1H), 2.84â2.71 (m, 1H), 2.23â2.14 (m, 2H), 2.13â1.75 (m, 6H), 1.59â1.35 (d, <i>J</i> = 6.1 Hz, 3H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O): Î´ 177.1, 176.5, 175.9, 174.2 170.6, 70.9, 68.5, 59.5, 58.2, 53.8, 50.2, 49.1, 41.9, 31.1, 28.3, 27.1, 24.1, 23.5. LC-ESI-MS (+ve ion mode): 429 [M + H]<sup>+</sup>. HPLC purity 97.55%, <i>t</i><sub>r</sub> = 4.76 min.</div></div><div id="sec4-3-8" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> [((<i>S</i>)-((<i>S</i>)-1-(<span class="smallcaps smallerCapital">l</span>-Glutamyl)pyrrolidin-2-yl)(carboxy)methyl)-<span class="smallcaps smallerCapital">l</span>-alanyl-<span class="smallcaps smallerCapital">l</span>-proline] (<b>17</b>)</h4><div class="NLM_p last">The general ester deprotection was performed on 80 mg of <b>30_17</b>, after which the crude residue was Boc deprotected via the general Boc deprotection method, after which it was purified to yield 6 mg (10%) of <b>17</b> as a sticky yellow-brown solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) Î´ 4.40â4.23 (m, 4H), 3.76 (d, <i>J</i> = 2.0 Hz, 1H), 3.65 (m, 1H), 3.53 (m, 2H), 3.39 (m, 1H), 2.54 (t, <i>J</i> = 7.5 Hz, 2H), 2.22 (m, 2H), 2.13 (m 2H), 1.95 (m, 5H), 1.77 (q, 1H), 1.49 (d, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O): Î´ 178.2, 177.4, 175.5, 174.9 171.4, 72.3, 68.9, 61.5, 58.1, 51.9, 51.2, 48.3, 41.9, 31.1, 29.7, 28.1, 26.4, 24.2, 22.9. LC-ESI-MS (+ve ion mode): 443 [M + H]<sup>+</sup>. HPLC purity 98.95%, <i>t</i><sub>r</sub> = 4.76 min.</div></div><div id="sec4-3-9" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> [((<i>S</i>)-Carboxy((<i>S</i>)-1-(4-carboxybenzoyl)pyrrolidin-2-yl)methyl)-<span class="smallcaps smallerCapital">l</span>-alanyl-<span class="smallcaps smallerCapital">l</span>-proline] (<b>18</b>)</h4><div class="NLM_p last">The general ester deprotection was performed on 30 mg of <b>30_18</b> and then purified via prep HPLC to obtain 23 mg (62%) of <b>18</b> as a sticky white-yellow solid. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) Î´ 8.07 (d, <i>J</i> = 7.8 Hz, 2H), 7.65 (dd, <i>J</i> = 8.2, 2.9 Hz, 2H), 4.71â4.61 (t, <i>J</i> = 8.2 Hz), 4.40 (q, <i>J</i> = 6.9 Hz, 1H), 4.12 (m, 1H), 3.96 (m, 1H), 3.62 (m, 2H), 3.43 (m, 2H), 2.32 (m, 3H), 1.98 (m, 3H), 1.86 (m, 1H), 1.74 (m, 1H), 1.58 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>): Î´ 175.9, 175.1, 174.5, 171.3, 167.2, 134.3, 128.8, 128.1 (2C), 126.3 (2C), 67.1, 66.2, 59.3, 57.9, 51.5, 49.1, 27.3, 25.8, 23.1, 22.5, 21.3. LC-ESI-MS (+ve ion mode): 462 [M + H]<sup>+</sup>. HPLC purity 97%, <i>t</i><sub>r</sub> = 7.31 min.</div></div></div><div id="sec4-3-10" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Protein Purification and X-ray Crystallography</h3><div class="NLM_p">The N-domain of ACE was expressed and purified to homogeneity from CHO (Chinese hamster ovary) cells.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Compound <b>16</b> in complex with Ndom389 (minimally glycosylated ACE N-domain) was prepared by incubating a 4:1 v/v ratio of 5 mg mL<sup>â1</sup> Ndom389 (in 50 mM Hepes, pH 7.5, 0.1 mM PMSF) and 20 mM <b>16</b> at room temperature for 1 h. The hanging drop co-crystallization used 1 Î¼L of the complex mixed with an equal volume of reservoir solution (30% PEG 550 MME/PEG 20000, 0.1 M Tris/Bicine pH 8.5, and 60 mM divalent cations, Molecular Dimensions Morpheus A9).</div><div class="NLM_p last">X-ray diffraction data were collected from two crystals on station IO4-1 at the Diamond Light Source (Didcot, UK). Crystals were kept at constant temperature (100 K) under the liquid nitrogen jet during data collection. Images were collected using a PILATUS-6M-F detector (Dectris, Switzerland). Raw data images were indexed and integrated with DIALS<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(28)</a> and then scaled using AIMLESS<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(29)</a> from the CCP4 suite.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(30)</a> Initial phases were obtained by molecular replacement with PHASER<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(31)</a> using PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXQ">3NXQ</a><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> as the search model. Further refinement was initially carried out using REFMAC5<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(32)</a> and then Phenix,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(33)</a> with COOT<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(34)</a> used for rounds of manual model building. Ligand and water molecules were added based on electron density in the <i>Fo</i>â<i>Fc</i> difference map. MolProbity<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(35)</a> was used to help validate the structures. Crystallographic data statistics are summarized in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_002.pdf" class="ext-link">Table S1</a>. All figures showing the crystal structures were generated using CCP4mg.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(36)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01478">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02260" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02260" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01478" class="ext-link">10.1021/acs.jmedchem.7b01478</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Crystallographic information, Ligplot 2D interaction maps of docked compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_001.csv">jm7b01478_si_001.csv (2.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_002.pdf">jm7b01478_si_002.pdf (1.83 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB code for N-domain of ACE bound to <b>16</b> is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EN5">6EN5</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01478" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22307" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22307" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kelly Chibale</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Cape Town, Rondebosch 7701, South Africa</span>;Â 
    <span class="hlFld-Affiliation affiliation">Institute
of Infectious Disease and Molecular Medicine, University of Cape Town, Observatory 7925, South Africa</span>;Â 
    <span class="hlFld-Affiliation affiliation">South
African Medical Research Council Drug Discovery and Development Research
Unit, University of Cape Town, Rondebosch 7701, South Africa</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1327-4727" title="Orcid link">http://orcid.org/0000-0002-1327-4727</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c68da3aaaabfe885aeafa4a7aaa386b3a5b2e8a7a5e8bca7"><span class="__cf_email__" data-cfemail="6e250b020217402d06070c0f020b2e1b0d1a400f0d40140f">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edward D. Sturrock</span> - <span class="hlFld-Affiliation affiliation">Institute
of Infectious Disease and Molecular Medicine, University of Cape Town, Observatory 7925, South Africa</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Integrative Biomedical Sciences, University
of Cape Town, Observatory 7925, South Africa</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c782a3b0a6b5a3e994b3b2b5b5a8a4ac87b2a4b3e9a6a4e9bda6"><span class="__cf_email__" data-cfemail="2a6f4e5d4b584e04795e5f58584549416a5f495e044b4904504b">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen Fienberg</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Cape Town, Rondebosch 7701, South Africa</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gyles E. Cozier</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biochemistry, University
of Bath, Claverton Down, Bath BA2 7AY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">K. Ravi Acharya</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biochemistry, University
of Bath, Claverton Down, Bath BA2 7AY, U.K.</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80000" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80000" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The University of Cape Town, South African Medical Research Council (SAMRC), National Research Foundation (NRF) Competitive Programme for Rated Researchers, and South African Research Chairs Initiative of the Department of Science and Technology, administered through the NRF, are gratefully acknowledged for support. K.R.A. thanks the Medical Research Council (UK) for support (grant no. MR/M026647/1) and the scientists at station IO4-1, Diamond Light Source, Didcot, Oxon (UK), for their support during X-ray diffraction data collection. K.R.A. and E.D.S. also thank the University of Cape Town (South Africa) and University of Bath (UK), respectively, for Visiting Professorships.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i39" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i39"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="DEF-LIST-d42e2556-autogenerated"><div id="ac_i40" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i40"> Abbreviations Used</h2><tr><td class="NLM_term">ACE</td><td class="NLM_def"><p class="first last">angiotensin converting enzyme</p></td></tr><tr><td class="NLM_term">ACEi</td><td class="NLM_def"><p class="first last">angiotensin converting enzyme inhibitor(s)</p></td></tr><tr><td class="NLM_term">Ang-I</td><td class="NLM_def"><p class="first last">angiotensin-I</p></td></tr><tr><td class="NLM_term">Ang-II</td><td class="NLM_def"><p class="first last">angiotensin-II</p></td></tr><tr><td class="NLM_term">BK</td><td class="NLM_def"><p class="first last">bradykinin</p></td></tr><tr><td class="NLM_term">GVK</td><td class="NLM_def"><p class="first last">Gunupati Venkata Krishna</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high performance liquid chromatography</p></td></tr><tr><td class="NLM_term">LC-MS</td><td class="NLM_def"><p class="first last">liquid chromatography mass spectrometry</p></td></tr><tr><td class="NLM_term">MM-GBSA</td><td class="NLM_def"><p class="first last">molecular mechanics generalized Born surface area</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">NS</td><td class="NLM_def"><p class="first last">nonselective</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">QM</td><td class="NLM_def"><p class="first last">quantum mechanics</p></td></tr><tr><td class="NLM_term">RAAS</td><td class="NLM_def"><p class="first last">renin aldosterone angiotensin system</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">sACE</td><td class="NLM_def"><p class="first last">somatic angiotensin converting enzyme</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structureâactivity relationship</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin layer chromatography</p></td></tr><tr><td class="NLM_term">T3P</td><td class="NLM_def"><p class="first last">triphenylphosphine</p></td></tr><tr><td class="NLM_term">UV</td><td class="NLM_def"><p class="first last">ultraviolet</p></td></tr><tr><td class="NLM_term">ZBG</td><td class="NLM_def"><p class="first last">zinc binding group</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16824" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16824" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 36 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Cushman, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ondetti, M.</span><span> </span><span class="NLM_article-title">Design of potent competitive inhibitors of angiotensin-converting enzyme. Carboxyalkanoyl and mercaptoalkanoyl amino acids</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">5484</span><span class="NLM_x">â</span> <span class="NLM_lpage">5491</span><span class="refDoi">Â DOI: 10.1021/bi00644a014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00644a014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaE1cXhvVSqsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1977&pages=5484-5491&author=D.+W.+Cushmanauthor=H.+Cheungauthor=E.+Saboauthor=M.+Ondetti&title=Design+of+potent+competitive+inhibitors+of+angiotensin-converting+enzyme.+Carboxyalkanoyl+and+mercaptoalkanoyl+amino+acids&doi=10.1021%2Fbi00644a014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Design of potent competitive inhibitors of angiotensin-converting enzyme.  Carboxyalkanoyl and mercaptoalkanoyl amino acids</span></div><div class="casAuthors">Cushman, D. W.; Cheung, H. S.; Sabo, E. F.; Ondetti, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5484-91</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">A hypothetical model of the active site of angiotensin-coverting enzyme (I) was utilized to guide the design and synthesis of specific inhibitors.  By analogy to bovine carboxypeptidase A, the active site of I was proposed to contain 3 important groups that participate in binding of peptide substrates: a carboxyl-binding group, a group with affinity for the C-terminal peptide bond, and a tightly bound Zn2+ that could coordinate with the carbonyl of the penultimate (scissiie) peptide bond.  According to the model, a succinyl amino acid could interact with each of these binding groups via its amino acid carboxyl, amide bond, and succinyl carboxyl, resp., and thus act as a specific competitive inhibitor of the enzyme.  Succinyl-L-proline was such an inhibitor (I50 = 330 Î¼M), and attempts to optimize its interaction with the active site of the enzyme as proposed in the model led to the synthesis of D-2-methylsuccinyl-L-proline (R,S) (Ki = 2.5 Î¼M), and D-2-methylglutaryl-L-proline (R,S) (Ki = 0.8 Î¼M).  Replacement of the succinyl carboxyl group of these compds. by a SH group led to a series of extremely potent competitive inhibitors of I, including 3-mercaptopropranoyl-L-proline (SQ 13,863, Ki = 0.012 Î¼M) and D-3-mercapto-2-methylpropranoyl-L-proline (S,S) (SQ 14,225, Ki = 0.0017 Î¼M).  These compds. are also potent orally active inhibitors of I and have great potential as antihypertensive agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWc6ldKEo4oLVg90H21EOLACvtfcHk0lhyh29Rz1PI2Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXhvVSqsw%253D%253D&md5=e575497901ccc7c7370ca7dec99882e7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fbi00644a014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00644a014%26sid%3Dliteratum%253Aachs%26aulast%3DCushman%26aufirst%3DD.%2BW.%26aulast%3DCheung%26aufirst%3DH.%26aulast%3DSabo%26aufirst%3DE.%26aulast%3DOndetti%26aufirst%3DM.%26atitle%3DDesign%2520of%2520potent%2520competitive%2520inhibitors%2520of%2520angiotensin-converting%2520enzyme.%2520Carboxyalkanoyl%2520and%2520mercaptoalkanoyl%2520amino%2520acids%26jtitle%3DBiochemistry%26date%3D1977%26volume%3D16%26spage%3D5484%26epage%3D5491%26doi%3D10.1021%2Fbi00644a014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Patchett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tristram, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyvratt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taub, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ten Broeke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ondeyka, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorsett, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenlee, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohr, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffsommer, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshua, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruyle, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothrock, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aster, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maycock, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirschmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulm, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassil, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, C. A.</span><span> </span><span class="NLM_article-title">A new class of angiotensin-converting enzyme inhibitors</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">280</span><span class="NLM_x">â</span> <span class="NLM_lpage">283</span><span class="refDoi">Â DOI: 10.1038/288280a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1038%2F288280a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=6253826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaL3MXktlGhtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=1980&pages=280-283&author=A.+Patchettauthor=E.+Harrisauthor=E.+Tristramauthor=M.+Wyvrattauthor=M.+Wuauthor=D.+Taubauthor=E.+Petersonauthor=T.+Ikelerauthor=J.+Ten+Broekeauthor=L.+Payneauthor=D.+L.+Ondeykaauthor=E.+D.+Thorsettauthor=W.+J.+Greenleeauthor=N.+S.+Lohrauthor=R.+D.+Hoffsommerauthor=H.+Joshuaauthor=W.+V.+Ruyleauthor=J.+W.+Rothrockauthor=S.+D.+Asterauthor=A.+L.+Maycockauthor=F.+M.+Robinsonauthor=R.+Hirschmannauthor=C.+S.+Sweetauthor=E.+H.+Ulmauthor=D.+M.+Grossauthor=T.+C.+Vassilauthor=C.+A.+Stone&title=A+new+class+of+angiotensin-converting+enzyme+inhibitors&doi=10.1038%2F288280a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A new class of angiotensin-converting enzyme inhibitors</span></div><div class="casAuthors">Patchett, A. A.; Harris, E.; Tristram, E. W.; Wyvratt, M. J.; Wu, M. T.; Taub, D.; Peterson, E. R.; Ikeler, T. J.; Ten Broeke, J.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">5788</span>),
    <span class="NLM_cas:pages">280-3</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Substituted N-carboxymethyldipeptides were evaluated for their ability to antagonize the pressor effect of angiotensin I (I)  [9041-90-1] in anesthetized rats and dogs (100 and 600 ng/kg, i.v., resp.); Ph(CH2)2CH(CO2H)-Ala-Pro (II)  [76420-72-9], Ph(CH2)2CH(CO2Et)-Ala-Pro maleate (III)  [75847-73-3] and Ph(CH2)2CH(CO2H)-Lys-Pro (IV)  [77726-95-5] were highly active by the i.v. route in rats and dogs, except for III in dogs, but all were orally active in rats and dogs.  II, III, and IV showed high potency in inhibiting the blood pressure response to I and all had a long duration of action; high angiotensin-converting enzyme (EC 3.4.15.1) (V)  [9015-82-1] inhibitory activities at doses as low as 2.3 Î¼g/kg were obsd. after i.v. administration and after oral administration, the compds. showed good activities with duration of action of â¥6 h at doses of 0.3 mg/kg.  Thus, V inhibitors have important design implications for the synthesis of inhibitors of other metallodipeptidyl carboxypeptidases and possibly of metalloendopeptidases more generally.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2vFzl18NAf7Vg90H21EOLACvtfcHk0ljWCWH8gYdRSQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXktlGhtLY%253D&md5=3a395d264e4e5c14d0cca0bcb6065a73</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F288280a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F288280a0%26sid%3Dliteratum%253Aachs%26aulast%3DPatchett%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DE.%26aulast%3DTristram%26aufirst%3DE.%26aulast%3DWyvratt%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DTaub%26aufirst%3DD.%26aulast%3DPeterson%26aufirst%3DE.%26aulast%3DIkeler%26aufirst%3DT.%26aulast%3DTen%2BBroeke%26aufirst%3DJ.%26aulast%3DPayne%26aufirst%3DL.%26aulast%3DOndeyka%26aufirst%3DD.%2BL.%26aulast%3DThorsett%26aufirst%3DE.%2BD.%26aulast%3DGreenlee%26aufirst%3DW.%2BJ.%26aulast%3DLohr%26aufirst%3DN.%2BS.%26aulast%3DHoffsommer%26aufirst%3DR.%2BD.%26aulast%3DJoshua%26aufirst%3DH.%26aulast%3DRuyle%26aufirst%3DW.%2BV.%26aulast%3DRothrock%26aufirst%3DJ.%2BW.%26aulast%3DAster%26aufirst%3DS.%2BD.%26aulast%3DMaycock%26aufirst%3DA.%2BL.%26aulast%3DRobinson%26aufirst%3DF.%2BM.%26aulast%3DHirschmann%26aufirst%3DR.%26aulast%3DSweet%26aufirst%3DC.%2BS.%26aulast%3DUlm%26aufirst%3DE.%2BH.%26aulast%3DGross%26aufirst%3DD.%2BM.%26aulast%3DVassil%26aufirst%3DT.%2BC.%26aulast%3DStone%26aufirst%3DC.%2BA.%26atitle%3DA%2520new%2520class%2520of%2520angiotensin-converting%2520enzyme%2520inhibitors%26jtitle%3DNature%26date%3D1980%26volume%3D288%26spage%3D280%26epage%3D283%26doi%3D10.1038%2F288280a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Bakhle, Y.</span><span> </span><span class="NLM_article-title">Conversion of angiotensin I to angiotensin II by cell-free extracts of dog lung</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1968</span><span class="NLM_x">, </span> <span class="NLM_volume">220</span><span class="NLM_x">, </span> <span class="NLM_fpage">919</span><span class="NLM_x">â</span> <span class="NLM_lpage">921</span><span class="refDoi">Â DOI: 10.1038/220919a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1038%2F220919a0" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=220&publication_year=1968&pages=919-921&author=Y.+Bakhle&title=Conversion+of+angiotensin+I+to+angiotensin+II+by+cell-free+extracts+of+dog+lung&doi=10.1038%2F220919a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F220919a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F220919a0%26sid%3Dliteratum%253Aachs%26aulast%3DBakhle%26aufirst%3DY.%26atitle%3DConversion%2520of%2520angiotensin%2520I%2520to%2520angiotensin%2520II%2520by%2520cell-free%2520extracts%2520of%2520dog%2520lung%26jtitle%3DNature%26date%3D1968%26volume%3D220%26spage%3D919%26epage%3D921%26doi%3D10.1038%2F220919a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Byers, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfenden, R.</span><span> </span><span class="NLM_article-title">Binding of the by-product analog benzylsuccinic acid by carboxypeptidase A</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2070</span><span class="NLM_x">â</span> <span class="NLM_lpage">2078</span><span class="refDoi">Â DOI: 10.1021/bi00735a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00735a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1973&pages=2070-2078&author=L.+D.+Byersauthor=R.+Wolfenden&title=Binding+of+the+by-product+analog+benzylsuccinic+acid+by+carboxypeptidase+A&doi=10.1021%2Fbi00735a008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fbi00735a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00735a008%26sid%3Dliteratum%253Aachs%26aulast%3DByers%26aufirst%3DL.%2BD.%26aulast%3DWolfenden%26aufirst%3DR.%26atitle%3DBinding%2520of%2520the%2520by-product%2520analog%2520benzylsuccinic%2520acid%2520by%2520carboxypeptidase%2520A%26jtitle%3DBiochemistry%26date%3D1973%26volume%3D12%26spage%3D2070%26epage%3D2078%26doi%3D10.1021%2Fbi00735a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Fuchs, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaud, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capecchi, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernstein, K. E.</span><span> </span><span class="NLM_article-title">Angiotensin-converting enzyme C-terminal catalytic domain is the main site of angiotensin I cleavage in vivo</span> <span class="citation_source-journal">Hypertension</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">â</span> <span class="NLM_lpage">274</span><span class="refDoi">Â DOI: 10.1161/HYPERTENSIONAHA.107.097865</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1161%2FHYPERTENSIONAHA.107.097865" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=18158355" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt1OjtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=267-274&author=S.+Fuchsauthor=H.+D.+Xiaoauthor=C.+Hubertauthor=A.+Michaudauthor=D.+J.+Campbellauthor=J.+W.+Adamsauthor=M.+R.+Capecchiauthor=P.+Corvolauthor=K.+E.+Bernstein&title=Angiotensin-converting+enzyme+C-terminal+catalytic+domain+is+the+main+site+of+angiotensin+I+cleavage+in+vivo&doi=10.1161%2FHYPERTENSIONAHA.107.097865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Angiotensin-Converting Enzyme C-Terminal Catalytic Domain Is the Main Site of Angiotensin I Cleavage In Vivo</span></div><div class="casAuthors">Fuchs, Sebastien; Xiao, Hong D.; Hubert, Christine; Michaud, Annie; Campbell, Duncan J.; Adams, Jonathan W.; Capecchi, Mario R.; Corvol, Pierre; Bernstein, Kenneth E.</div><div class="citationInfo"><span class="NLM_cas:title">Hypertension</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">267-274</span>CODEN:
                <span class="NLM_cas:coden">HPRTDN</span>;
        ISSN:<span class="NLM_cas:issn">0194-911X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Angiotensin-converting enzyme (ACE) plays a central role in the prodn. of the vasoconstrictor angiotensin II.  ACE is a single polypeptide, but it contains 2 homologous and independent catalytic domains, each of which binds zinc.  To understand the in vivo role of these 2 domains, we used gene targeting to create mice with point mutations in the ACE C-domain zinc-binding motif.  Such mice, termed ACE13/13, produce a full-length ACE protein with tissue expression identical to wild-type mice.  Anal. of ACE13/13 mice showed that they produce ACE having only N-domain catalytic activity, as detd. by the hydrolysis of domain specific substrates and by chloride sensitivity.  ACE13/13 mice have blood pressure and blood angiotensin II levels similar to wild-type mice.  However, plasma renin concn. is increased 2.6-fold and blood angiotensin I levels are increased 7.5-fold.  Bradykinin peptide levels are not different from wild-type levels.  ACE13/13 mice have a reduced increase of blood pressure after i.v. infusion of angiotensin I.  ACE13/13 mice have a normal renal structure, but they are not able to conc. urine after dehydration as effectively as wild-type mice.  This study shows that the C-domain of ACE is the predominant site of angiotensin I cleavage in vivo.  Although mice lacking C-domain activity have normal physiol. under lab. conditions, they respond less well to the stress of dehydration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPzpHqq5luNbVg90H21EOLACvtfcHk0liGHiNzDw9Lpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt1OjtQ%253D%253D&md5=9cd89bdcb7fa8ca910583f4b952fcdf0</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1161%2FHYPERTENSIONAHA.107.097865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FHYPERTENSIONAHA.107.097865%26sid%3Dliteratum%253Aachs%26aulast%3DFuchs%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DH.%2BD.%26aulast%3DHubert%26aufirst%3DC.%26aulast%3DMichaud%26aufirst%3DA.%26aulast%3DCampbell%26aufirst%3DD.%2BJ.%26aulast%3DAdams%26aufirst%3DJ.%2BW.%26aulast%3DCapecchi%26aufirst%3DM.%2BR.%26aulast%3DCorvol%26aufirst%3DP.%26aulast%3DBernstein%26aufirst%3DK.%2BE.%26atitle%3DAngiotensin-converting%2520enzyme%2520C-terminal%2520catalytic%2520domain%2520is%2520the%2520main%2520site%2520of%2520angiotensin%2520I%2520cleavage%2520in%2520vivo%26jtitle%3DHypertension%26date%3D2008%26volume%3D51%26spage%3D267%26epage%3D274%26doi%3D10.1161%2FHYPERTENSIONAHA.107.097865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Wei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alhenc-Gelas, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clauser, E.</span><span> </span><span class="NLM_article-title">The two homologous domains of human angiotensin I-converting enzyme are both catalytically active</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">266</span><span class="NLM_x">, </span> <span class="NLM_fpage">9002</span><span class="NLM_x">â</span> <span class="NLM_lpage">9008</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1851160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaK3MXks1OntLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=266&publication_year=1991&pages=9002-9008&author=L.+Weiauthor=F.+Alhenc-Gelasauthor=P.+Corvolauthor=E.+Clauser&title=The+two+homologous+domains+of+human+angiotensin+I-converting+enzyme+are+both+catalytically+active"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The two homologous domains of human angiotensin I-converting enzyme are both catalytically active</span></div><div class="casAuthors">Wei, Lei; Alhenc-Gelas, Francois; Corvol, Pierre; Clauser, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">266</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">9002-8</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Mol. cloning of human endothelial angiotensin I-covering enzyme (kinase II; EC 3.4.15.1) (ACE) has recently shown that the enzyme contains 2 large homol. domains (called here the N and C domains), each bearing a putative active site, identified by sequence comparisons with the active sites of other zinc metallopeptidases.  However, the previous expts. with zinc or competitive ACE inhibitors suggested a single active site in ACE.  To establish whether both domains of ACE are enzymically active, a series of ACE mutants, each contg. only 1 intact domain, were constructed by deletion or point mutations of putative crit. residues of the other domain, and expressed in heterologous Chinese hamster ovary cells.  Both domains are enzymically active and cleave the C-terminal dipeptide of hippuryl-His-Leu or angiotensin I.  Moreover, both domains have an abs. zinc requirement for activity, are activated by chloride and are sensitive to competitive ACE inhibitors, and appear to function independently.  However, the 2 domains display different catalytic consts. and different patterns of chloride activation.  At high chloride concns., the C domain hydrolyzes the 2 substrates tested faster than does the N domain.  His-361,365 and His-959,963 are established as essential residues in the N and C domains, resp., most likely involved in zinc binding, and Glu-362 in the N domain and Glu-960 in the C domain are essential catalytic residues.  These observations provide strong evidence that ACE possesses 2 independent catalytic domains and suggest that they may have different functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquqPiPJev2abVg90H21EOLACvtfcHk0liGHiNzDw9Lpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXks1OntLY%253D&md5=907f83aa628bc22968428d5d43d804ff</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DL.%26aulast%3DAlhenc-Gelas%26aufirst%3DF.%26aulast%3DCorvol%26aufirst%3DP.%26aulast%3DClauser%26aufirst%3DE.%26atitle%3DThe%2520two%2520homologous%2520domains%2520of%2520human%2520angiotensin%2520I-converting%2520enzyme%2520are%2520both%2520catalytically%2520active%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1991%26volume%3D266%26spage%3D9002%26epage%3D9008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Binevski, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sizova, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozdnev, V. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kost, O. A.</span><span> </span><span class="NLM_article-title">Evidence for the negative cooperativity of the two active sites within bovine somatic angiotensin-converting enzyme</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">550</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">â</span> <span class="NLM_lpage">88</span><span class="refDoi">Â DOI: 10.1016/S0014-5793(03)00825-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1016%2FS0014-5793%2803%2900825-1" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2003&pages=84-88&author=P.+V.+Binevskiauthor=E.+A.+Sizovaauthor=V.+F.+Pozdnevauthor=O.+A.+Kost&title=Evidence+for+the+negative+cooperativity+of+the+two+active+sites+within+bovine+somatic+angiotensin-converting+enzyme&doi=10.1016%2FS0014-5793%2803%2900825-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2803%2900825-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252803%252900825-1%26sid%3Dliteratum%253Aachs%26aulast%3DBinevski%26aufirst%3DP.%2BV.%26aulast%3DSizova%26aufirst%3DE.%2BA.%26aulast%3DPozdnev%26aufirst%3DV.%2BF.%26aulast%3DKost%26aufirst%3DO.%2BA.%26atitle%3DEvidence%2520for%2520the%2520negative%2520cooperativity%2520of%2520the%2520two%2520active%2520sites%2520within%2520bovine%2520somatic%2520angiotensin-converting%2520enzyme%26jtitle%3DFEBS%2520Lett.%26date%3D2003%26volume%3D550%26spage%3D84%26epage%3D88%26doi%3D10.1016%2FS0014-5793%2803%2900825-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Woodman, Z. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwager, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redelinghuys, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmona, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehlers, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">The N domain of somatic angiotensin-converting enzyme negatively regulates ectodomain shedding and catalytic activity</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">389</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">â</span> <span class="NLM_lpage">744</span><span class="refDoi">Â DOI: 10.1042/BJ20050187</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1042%2FBJ20050187" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2005&pages=739-744&author=Z.+L.+Woodmanauthor=S.+L.+Schwagerauthor=P.+Redelinghuysauthor=A.+K.+Carmonaauthor=M.+R.+Ehlersauthor=E.+D.+Sturrock&title=The+N+domain+of+somatic+angiotensin-converting+enzyme+negatively+regulates+ectodomain+shedding+and+catalytic+activity&doi=10.1042%2FBJ20050187"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2FBJ20050187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20050187%26sid%3Dliteratum%253Aachs%26aulast%3DWoodman%26aufirst%3DZ.%2BL.%26aulast%3DSchwager%26aufirst%3DS.%2BL.%26aulast%3DRedelinghuys%26aufirst%3DP.%26aulast%3DCarmona%26aufirst%3DA.%2BK.%26aulast%3DEhlers%26aufirst%3DM.%2BR.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DThe%2520N%2520domain%2520of%2520somatic%2520angiotensin-converting%2520enzyme%2520negatively%2520regulates%2520ectodomain%2520shedding%2520and%2520catalytic%2520activity%26jtitle%3DBiochem.%2520J.%26date%3D2005%26volume%3D389%26spage%3D739%26epage%3D744%26doi%3D10.1042%2FBJ20050187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Lenfant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wdzieczak-Bakala, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guittet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prome, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sotty, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frindel, E.</span><span> </span><span class="NLM_article-title">Inhibitor of hematopoietic pluripotent stem cell proliferation: purification and determination of its structure</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">â</span> <span class="NLM_lpage">782</span><span class="refDoi">Â DOI: 10.1073/pnas.86.3.779</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1073%2Fpnas.86.3.779" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=2915977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaL1MXhslWnsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=779-782&author=M.+Lenfantauthor=J.+Wdzieczak-Bakalaauthor=E.+Guittetauthor=J.+C.+Promeauthor=D.+Sottyauthor=E.+Frindel&title=Inhibitor+of+hematopoietic+pluripotent+stem+cell+proliferation%3A+purification+and+determination+of+its+structure&doi=10.1073%2Fpnas.86.3.779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor of hematopoietic pluripotent stem cell proliferation:  purification and determination of its structure</span></div><div class="casAuthors">Lenfant, Maryse; Wdzieczak-Bakala, Joanna; Guittet, Eric; Prome, Jean Claude; Sotty, Dominique; Frindel, Emilia</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">779-82</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">A 5-step purifn. procedure is reported here that led to the isolation from fetal calf bone marrow ext. of a tetrapeptide, Ac-Ser-Asp-Lys-Pro (mol. wt. 487), exerting a high inhibitory activity on the proliferation of hematopoietic pluripotent stem cells [defined here as spleen colony-forming units (CFU-S)].  The structure of this mol. was established from amino acid anal., fast atom bombardment mass spectrometry, and 1H NMR spectral data.  This structure was confirmed by comparison with the corresponding synthetic mol., which presents identical physicochem. characteristics and biol. properties.  Natural and synthetic peptides administered to mice (at a dose of 100 ng per mouse) after one injection of cytosine arabinonucleoside prevent CFU-S recruitment into DNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptPOPzmuaaarVg90H21EOLACvtfcHk0lh6eCupNvEPdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhslWnsbc%253D&md5=0b603b2b751d1233d1624e0e42e2f3a5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.86.3.779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.86.3.779%26sid%3Dliteratum%253Aachs%26aulast%3DLenfant%26aufirst%3DM.%26aulast%3DWdzieczak-Bakala%26aufirst%3DJ.%26aulast%3DGuittet%26aufirst%3DE.%26aulast%3DProme%26aufirst%3DJ.%2BC.%26aulast%3DSotty%26aufirst%3DD.%26aulast%3DFrindel%26aufirst%3DE.%26atitle%3DInhibitor%2520of%2520hematopoietic%2520pluripotent%2520stem%2520cell%2520proliferation%253A%2520purification%2520and%2520determination%2520of%2520its%2520structure%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1989%26volume%3D86%26spage%3D779%26epage%3D782%26doi%3D10.1073%2Fpnas.86.3.779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Robinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenfant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wdzieczak-Bakala, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melville, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riches, A.</span><span> </span><span class="NLM_article-title">The mechanism of action of the tetrapeptide acetyl-N-Ser-Asp-Lys-Pro (AcSDKP) in the control of haematopoietic stem cell proliferation</span> <span class="citation_source-journal">Cell Proliferation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">â</span> <span class="NLM_lpage">632</span><span class="refDoi">Â DOI: 10.1111/j.1365-2184.1992.tb01464.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1111%2Fj.1365-2184.1992.tb01464.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1457609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaK3sXktVWmu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1992&pages=623-632&author=S.+Robinsonauthor=M.+Lenfantauthor=J.+Wdzieczak-Bakalaauthor=J.+Melvilleauthor=A.+Riches&title=The+mechanism+of+action+of+the+tetrapeptide+acetyl-N-Ser-Asp-Lys-Pro+%28AcSDKP%29+in+the+control+of+haematopoietic+stem+cell+proliferation&doi=10.1111%2Fj.1365-2184.1992.tb01464.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The mechanism of action of the tetrapeptide acetyl-N-Ser-Asp-Lys-Pro (AcSDKP) in the control of hematopoietic stem cell proliferation</span></div><div class="casAuthors">Robinson, S.; Lenfant, M.; Wdzieczak-Babala, J.; Melville, J.; Riches, A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Proliferation</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">623-32</span>CODEN:
                <span class="NLM_cas:coden">CPROEM</span>;
        ISSN:<span class="NLM_cas:issn">0960-7722</span>.
    </div><div class="casAbstract">The mechanism of action of the hemoregulatory tetrapeptide acetyl-N-Ser-Asp-Lys-Pro (AcSDKP, Mr = 487 amu) was investigated using an in vitro assay of a murine high proliferative potential colony-forming cell (HPP-CFC) which responds to proliferation regulators of the hematopoietic stem cell population.  AcSDKP had no direct inhibitory effect on the no. or the proportion in S phase, of the committed granulocyte-macrophage progenitor (GM-CFC), or cycling HPP-CFC populations.  However, AcSDKP blocked the action of a stimulator of hematopoietic stem cell proliferation, preventing the switching of quiescent HPP-CFC into cell cycle.  It would appear that AcSDKP exerts its inhibitory hemoregulatory role indirectly, by preventing stimulator action on hematopoietic stem cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVaPl7Jppd97Vg90H21EOLACvtfcHk0lh6eCupNvEPdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktVWmu7s%253D&md5=3c53c736b3b8d9ba339351a335dd3b68</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2184.1992.tb01464.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2184.1992.tb01464.x%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DLenfant%26aufirst%3DM.%26aulast%3DWdzieczak-Bakala%26aufirst%3DJ.%26aulast%3DMelville%26aufirst%3DJ.%26aulast%3DRiches%26aufirst%3DA.%26atitle%3DThe%2520mechanism%2520of%2520action%2520of%2520the%2520tetrapeptide%2520acetyl-N-Ser-Asp-Lys-Pro%2520%2528AcSDKP%2529%2520in%2520the%2520control%2520of%2520haematopoietic%2520stem%2520cell%2520proliferation%26jtitle%3DCell%2520Proliferation%26date%3D1992%26volume%3D25%26spage%3D623%26epage%3D632%26doi%3D10.1111%2Fj.1365-2184.1992.tb01464.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Rousseau, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaud, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauvet, M.-T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenfant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span> </span><span class="NLM_article-title">The hemoregulatory peptide N-acetyl-Ser-Asp-Lys-Pro is a natural and specific substrate of the N-terminal active site of human angiotensin-converting enzyme</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">270</span><span class="NLM_x">, </span> <span class="NLM_fpage">3656</span><span class="NLM_x">â</span> <span class="NLM_lpage">3661</span><span class="refDoi">Â DOI: 10.1074/jbc.270.8.3656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1074%2Fjbc.270.8.3656" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=3656-3661&author=A.+Rousseauauthor=A.+Michaudauthor=M.-T.+Chauvetauthor=M.+Lenfantauthor=P.+Corvol&title=The+hemoregulatory+peptide+N-acetyl-Ser-Asp-Lys-Pro+is+a+natural+and+specific+substrate+of+the+N-terminal+active+site+of+human+angiotensin-converting+enzyme&doi=10.1074%2Fjbc.270.8.3656"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.270.8.3656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.270.8.3656%26sid%3Dliteratum%253Aachs%26aulast%3DRousseau%26aufirst%3DA.%26aulast%3DMichaud%26aufirst%3DA.%26aulast%3DChauvet%26aufirst%3DM.-T.%26aulast%3DLenfant%26aufirst%3DM.%26aulast%3DCorvol%26aufirst%3DP.%26atitle%3DThe%2520hemoregulatory%2520peptide%2520N-acetyl-Ser-Asp-Lys-Pro%2520is%2520a%2520natural%2520and%2520specific%2520substrate%2520of%2520the%2520N-terminal%2520active%2520site%2520of%2520human%2520angiotensin-converting%2520enzyme%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D3656%26epage%3D3661%26doi%3D10.1074%2Fjbc.270.8.3656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Natesh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwager, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, K. R.</span><span> </span><span class="NLM_article-title">Crystal structure of the human angiotensin-converting enzymeâlisinopril complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">421</span><span class="NLM_x">, </span> <span class="NLM_fpage">551</span><span class="NLM_x">â</span> <span class="NLM_lpage">554</span><span class="refDoi">Â DOI: 10.1038/nature01370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1038%2Fnature01370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=12540854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFSqtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2003&pages=551-554&author=R.+Nateshauthor=S.+L.+Schwagerauthor=E.+D.+Sturrockauthor=K.+R.+Acharya&title=Crystal+structure+of+the+human+angiotensin-converting+enzyme%E2%80%93lisinopril+complex&doi=10.1038%2Fnature01370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human angiotensin-converting enzyme-lisinopril complex</span></div><div class="casAuthors">Natesh, Ramanathan; Schwager, Sylva L. U.; Sturrock, Edward D.; Acharya, K. Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">6922</span>),
    <span class="NLM_cas:pages">551-554</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Angiotensin-converting enzyme (ACE) has a crit. role in cardiovascular function by cleaving the carboxy terminal His-Leu dipeptide from angiotensin I to produce a potent vasopressor octapeptide, angiotensin II.  Inhibitors of ACE are a first line of therapy for hypertension, heart failure, myocardial infarction and diabetic nephropathy.  Notably, these inhibitors were developed without knowledge of the structure of human ACE, but were instead designed on the basis of an assumed mechanistic homol. with carboxypeptidase A.  Here we present the x-ray structure of human testicular ACE and its complex with one of the most widely used inhibitors, lisinopril (N2-[(S)-1-carboxy-3-phenylpropyl]-l-lysyl-l-proline; also known as Prinivil or Zestril), at 2.0 Ã resoln.  Anal. of the three-dimensional structure of ACE shows that it bears little similarity to that of carboxypeptidase A, but instead resembles neurolysin and Pyrococcus furiosus carboxypeptidase-zinc metallopeptidases with no detectable sequence similarity to ACE.  The structure provides an opportunity to design domain-selective ACE inhibitors that may exhibit new pharmacol. profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxHYbWU8PmP7Vg90H21EOLACvtfcHk0lh6eCupNvEPdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFSqtg%253D%253D&md5=84f2f474ac20541b0352691128782b1d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature01370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01370%26sid%3Dliteratum%253Aachs%26aulast%3DNatesh%26aufirst%3DR.%26aulast%3DSchwager%26aufirst%3DS.%2BL.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520angiotensin-converting%2520enzyme%25E2%2580%2593lisinopril%2520complex%26jtitle%3DNature%26date%3D2003%26volume%3D421%26spage%3D551%26epage%3D554%26doi%3D10.1038%2Fnature01370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">KrÃ¶ger, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douglas, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâNeill, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dive, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">Investigating the domain specificity of phosphinic inhibitors RXPA380 and RXP407 in angiotensin-converting enzyme</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">8405</span><span class="NLM_x">â</span> <span class="NLM_lpage">8412</span><span class="refDoi">Â DOI: 10.1021/bi9011226</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9011226" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFyju7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8405-8412&author=W.+L.+Kr%C3%B6gerauthor=R.+G.+Douglasauthor=H.+G.+O%E2%80%99Neillauthor=V.+Diveauthor=E.+D.+Sturrock&title=Investigating+the+domain+specificity+of+phosphinic+inhibitors+RXPA380+and+RXP407+in+angiotensin-converting+enzyme&doi=10.1021%2Fbi9011226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Investigating the Domain Specificity of Phosphinic Inhibitors RXPA380 and RXP407 in Angiotensin-Converting Enzyme</span></div><div class="casAuthors">Kroger, Wendy L.; Douglas, Ross G.; O'Neill, Hester G.; Dive, Vincent; Sturrock, Edward D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">8405-8412</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human somatic angiotensin-converting enzyme (ACE) is a membrane-bound dipeptidyl carboxypeptidase that contains two extracellular domains (N and C).  Although highly homologous, they exhibit different substrate and inhibition profiles.  The phosphinic inhibitors RXPA380 and RXP407 are highly selective for the C- and N-domains, resp.  A no. of residues, implicated by structural data, are likely to contribute to this selectivity.  However, the extent to which these different interactions are responsible for domain selectivity is unclear.  In this study, a series of C- and N-domain mutants contg. conversions to corresponding domain residues were used to scrutinize the contribution of these residues to selective inhibitor binding.  Enzyme kinetic analyses of the purified mutants indicated that the RXPA380 C-selectivity is particularly reliant on the interaction between the P2 substituent and Phe 391 (testis ACE numbering).  Moreover, a C-domain mutant in which Phe 391 has been changed to a Tyr residue, in addn. to contg. an N-domain S2' pocket (S2'F/Y), displayed the greatest shift toward a more N-domain-like Ki.  None of the single mutations within the N-domain caused a large shift in RXP407's affinity for these enzymes.  However, the double mutant contg. the Tyr 369 to Phe change as well as Arg 381 to Glu displayed a 100-fold decrease in binding affinity, confirming that the S2 pocket plays a major role in RXP407 selectivity.  Taken together, these data advance our understanding regarding the mol. basis for the remarkable ACE domain selectivity exhibited by these inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdh4e6bOYw2LVg90H21EOLACvtfcHk0li2nXcSfagJBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFyju7w%253D&md5=7a32a2e7ed0ba9199a43c5b79305fc53</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi9011226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9011226%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25B6ger%26aufirst%3DW.%2BL.%26aulast%3DDouglas%26aufirst%3DR.%2BG.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DH.%2BG.%26aulast%3DDive%26aufirst%3DV.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DInvestigating%2520the%2520domain%2520specificity%2520of%2520phosphinic%2520inhibitors%2520RXPA380%2520and%2520RXP407%2520in%2520angiotensin-converting%2520enzyme%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D8405%26epage%3D8412%26doi%3D10.1021%2Fbi9011226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Soubrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alhenc-Gelas, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allegrini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tregear, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span> </span><span class="NLM_article-title">Two putative active centers in human angiotensin I-converting enzyme revealed by molecular cloning</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">9386</span><span class="NLM_x">â</span> <span class="NLM_lpage">9390</span><span class="refDoi">Â DOI: 10.1073/pnas.85.24.9386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1073%2Fpnas.85.24.9386" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=1988&pages=9386-9390&author=F.+Soubrierauthor=F.+Alhenc-Gelasauthor=C.+Hubertauthor=J.+Allegriniauthor=M.+Johnauthor=G.+Tregearauthor=P.+Corvol&title=Two+putative+active+centers+in+human+angiotensin+I-converting+enzyme+revealed+by+molecular+cloning&doi=10.1073%2Fpnas.85.24.9386"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.85.24.9386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.85.24.9386%26sid%3Dliteratum%253Aachs%26aulast%3DSoubrier%26aufirst%3DF.%26aulast%3DAlhenc-Gelas%26aufirst%3DF.%26aulast%3DHubert%26aufirst%3DC.%26aulast%3DAllegrini%26aufirst%3DJ.%26aulast%3DJohn%26aufirst%3DM.%26aulast%3DTregear%26aufirst%3DG.%26aulast%3DCorvol%26aufirst%3DP.%26atitle%3DTwo%2520putative%2520active%2520centers%2520in%2520human%2520angiotensin%2520I-converting%2520enzyme%2520revealed%2520by%2520molecular%2520cloning%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1988%26volume%3D85%26spage%3D9386%26epage%3D9390%26doi%3D10.1073%2Fpnas.85.24.9386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Dive, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cotton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yiotakis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaud, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassiliou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiracek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vazeux, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauvet, M.-T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuniasse, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span> </span><span class="NLM_article-title">RXP 407, a phosphinic peptide, is a potent inhibitor of angiotensin I converting enzyme able to differentiate between its two active sites</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">4330</span><span class="NLM_x">â</span> <span class="NLM_lpage">4335</span><span class="refDoi">Â DOI: 10.1073/pnas.96.8.4330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1073%2Fpnas.96.8.4330" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=4330-4335&author=V.+Diveauthor=J.+Cottonauthor=A.+Yiotakisauthor=A.+Michaudauthor=S.+Vassiliouauthor=J.+Jiracekauthor=G.+Vazeuxauthor=M.-T.+Chauvetauthor=P.+Cuniasseauthor=P.+Corvol&title=RXP+407%2C+a+phosphinic+peptide%2C+is+a+potent+inhibitor+of+angiotensin+I+converting+enzyme+able+to+differentiate+between+its+two+active+sites&doi=10.1073%2Fpnas.96.8.4330"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.8.4330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.8.4330%26sid%3Dliteratum%253Aachs%26aulast%3DDive%26aufirst%3DV.%26aulast%3DCotton%26aufirst%3DJ.%26aulast%3DYiotakis%26aufirst%3DA.%26aulast%3DMichaud%26aufirst%3DA.%26aulast%3DVassiliou%26aufirst%3DS.%26aulast%3DJiracek%26aufirst%3DJ.%26aulast%3DVazeux%26aufirst%3DG.%26aulast%3DChauvet%26aufirst%3DM.-T.%26aulast%3DCuniasse%26aufirst%3DP.%26aulast%3DCorvol%26aufirst%3DP.%26atitle%3DRXP%2520407%252C%2520a%2520phosphinic%2520peptide%252C%2520is%2520a%2520potent%2520inhibitor%2520of%2520angiotensin%2520I%2520converting%2520enzyme%2520able%2520to%2520differentiate%2520between%2520its%2520two%2520active%2520sites%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D4330%26epage%3D4335%26doi%3D10.1073%2Fpnas.96.8.4330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Anthony, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corradi, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwager, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redelinghuys, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiadis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dive, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">The N domain of human angiotensin-I-converting enzyme: the role of N domain glycosylation and the crystal structure in complex with an N domain-specific phosphinic inhibitor, RXP407</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">35685</span><span class="NLM_x">â</span> <span class="NLM_lpage">35693</span><span class="refDoi">Â DOI: 10.1074/jbc.M110.167866</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1074%2Fjbc.M110.167866" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=35685-35693&author=C.+S.+Anthonyauthor=H.+R.+Corradiauthor=S.+L.+Schwagerauthor=P.+Redelinghuysauthor=D.+Georgiadisauthor=V.+Diveauthor=K.+R.+Acharyaauthor=E.+D.+Sturrock&title=The+N+domain+of+human+angiotensin-I-converting+enzyme%3A+the+role+of+N+domain+glycosylation+and+the+crystal+structure+in+complex+with+an+N+domain-specific+phosphinic+inhibitor%2C+RXP407&doi=10.1074%2Fjbc.M110.167866"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.167866&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.167866%26sid%3Dliteratum%253Aachs%26aulast%3DAnthony%26aufirst%3DC.%2BS.%26aulast%3DCorradi%26aufirst%3DH.%2BR.%26aulast%3DSchwager%26aufirst%3DS.%2BL.%26aulast%3DRedelinghuys%26aufirst%3DP.%26aulast%3DGeorgiadis%26aufirst%3DD.%26aulast%3DDive%26aufirst%3DV.%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DThe%2520N%2520domain%2520of%2520human%2520angiotensin-I-converting%2520enzyme%253A%2520the%2520role%2520of%2520N%2520domain%2520glycosylation%2520and%2520the%2520crystal%2520structure%2520in%2520complex%2520with%2520an%2520N%2520domain-specific%2520phosphinic%2520inhibitor%252C%2520RXP407%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D35685%26epage%3D35693%26doi%3D10.1074%2Fjbc.M110.167866" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Greenlee, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allibone, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlow, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patchett, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulm, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassil, T. C.</span><span> </span><span class="NLM_article-title">Angiotensin-converting enzyme inhibitors: synthesis and biological activity of acyltripeptide analogs of enalapril</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">434</span><span class="NLM_x">â</span> <span class="NLM_lpage">442</span><span class="refDoi">Â DOI: 10.1021/jm00382a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00382a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=434-442&author=W.+J.+Greenleeauthor=P.+L.+Alliboneauthor=D.+S.+Perlowauthor=A.+A.+Patchettauthor=E.+H.+Ulmauthor=T.+C.+Vassil&title=Angiotensin-converting+enzyme+inhibitors%3A+synthesis+and+biological+activity+of+acyltripeptide+analogs+of+enalapril&doi=10.1021%2Fjm00382a008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm00382a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00382a008%26sid%3Dliteratum%253Aachs%26aulast%3DGreenlee%26aufirst%3DW.%2BJ.%26aulast%3DAllibone%26aufirst%3DP.%2BL.%26aulast%3DPerlow%26aufirst%3DD.%2BS.%26aulast%3DPatchett%26aufirst%3DA.%2BA.%26aulast%3DUlm%26aufirst%3DE.%2BH.%26aulast%3DVassil%26aufirst%3DT.%2BC.%26atitle%3DAngiotensin-converting%2520enzyme%2520inhibitors%253A%2520synthesis%2520and%2520biological%2520activity%2520of%2520acyltripeptide%2520analogs%2520of%2520enalapril%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D434%26epage%3D442%26doi%3D10.1021%2Fjm00382a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Stanton, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watthey, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babiarz, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomaselli, H. C.</span><span> </span><span class="NLM_article-title">Angiotensin converting enzyme inhibitors: structure-activity profile of 1-benzazepin-2-one derivatives</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1603</span><span class="NLM_x">â</span> <span class="NLM_lpage">1606</span><span class="refDoi">Â DOI: 10.1021/jm00149a010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00149a010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=1603-1606&author=J.+L.+Stantonauthor=J.+W.+Wattheyauthor=M.+N.+Desaiauthor=B.+M.+Finnauthor=J.+E.+Babiarzauthor=H.+C.+Tomaselli&title=Angiotensin+converting+enzyme+inhibitors%3A+structure-activity+profile+of+1-benzazepin-2-one+derivatives&doi=10.1021%2Fjm00149a010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm00149a010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00149a010%26sid%3Dliteratum%253Aachs%26aulast%3DStanton%26aufirst%3DJ.%2BL.%26aulast%3DWatthey%26aufirst%3DJ.%2BW.%26aulast%3DDesai%26aufirst%3DM.%2BN.%26aulast%3DFinn%26aufirst%3DB.%2BM.%26aulast%3DBabiarz%26aufirst%3DJ.%2BE.%26aulast%3DTomaselli%26aufirst%3DH.%2BC.%26atitle%3DAngiotensin%2520converting%2520enzyme%2520inhibitors%253A%2520structure-activity%2520profile%2520of%25201-benzazepin-2-one%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D1603%26epage%3D1606%26doi%3D10.1021%2Fjm00149a010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Ksander, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erion, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diefenbacher, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Chehabi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levens, N.</span><span> </span><span class="NLM_article-title">Dual angiotensin converting enzyme/thromboxane synthase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">1823</span><span class="NLM_x">â</span> <span class="NLM_lpage">1832</span><span class="refDoi">Â DOI: 10.1021/jm00038a011</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00038a011" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=1823-1832&author=G.+M.+Ksanderauthor=M.+Erionauthor=A.+M.+Yuanauthor=C.+G.+Diefenbacherauthor=L.+El-Chehabiauthor=D.+Coteauthor=N.+Levens&title=Dual+angiotensin+converting+enzyme%2Fthromboxane+synthase+inhibitors&doi=10.1021%2Fjm00038a011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm00038a011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00038a011%26sid%3Dliteratum%253Aachs%26aulast%3DKsander%26aufirst%3DG.%2BM.%26aulast%3DErion%26aufirst%3DM.%26aulast%3DYuan%26aufirst%3DA.%2BM.%26aulast%3DDiefenbacher%26aufirst%3DC.%2BG.%26aulast%3DEl-Chehabi%26aufirst%3DL.%26aulast%3DCote%26aufirst%3DD.%26aulast%3DLevens%26aufirst%3DN.%26atitle%3DDual%2520angiotensin%2520converting%2520enzyme%252Fthromboxane%2520synthase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D1823%26epage%3D1832%26doi%3D10.1021%2Fjm00038a011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Gruenfeld, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebetino, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browne, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gude, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huebner, C. F.</span><span> </span><span class="NLM_article-title">Angiotensin converting enzyme inhibitors: 1-glutarylindoline-2-carboxylic acid derivatives</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1277</span><span class="NLM_x">â</span> <span class="NLM_lpage">1282</span><span class="refDoi">Â DOI: 10.1021/jm00363a012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00363a012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1983&pages=1277-1282&author=N.+Gruenfeldauthor=J.+L.+Stantonauthor=A.+M.+Yuanauthor=F.+H.+Ebetinoauthor=L.+J.+Browneauthor=C.+Gudeauthor=C.+F.+Huebner&title=Angiotensin+converting+enzyme+inhibitors%3A+1-glutarylindoline-2-carboxylic+acid+derivatives&doi=10.1021%2Fjm00363a012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm00363a012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00363a012%26sid%3Dliteratum%253Aachs%26aulast%3DGruenfeld%26aufirst%3DN.%26aulast%3DStanton%26aufirst%3DJ.%2BL.%26aulast%3DYuan%26aufirst%3DA.%2BM.%26aulast%3DEbetino%26aufirst%3DF.%2BH.%26aulast%3DBrowne%26aufirst%3DL.%2BJ.%26aulast%3DGude%26aufirst%3DC.%26aulast%3DHuebner%26aufirst%3DC.%2BF.%26atitle%3DAngiotensin%2520converting%2520enzyme%2520inhibitors%253A%25201-glutarylindoline-2-carboxylic%2520acid%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1983%26volume%3D26%26spage%3D1277%26epage%3D1282%26doi%3D10.1021%2Fjm00363a012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Ksander, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diefenbacher, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, J. L.</span><span> </span><span class="NLM_article-title">Angiotensin converting enzyme inhibitors: N-substituted D-glutamic acid. gamma.-dipeptides</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1606</span><span class="NLM_x">â</span> <span class="NLM_lpage">1611</span><span class="refDoi">Â DOI: 10.1021/jm00149a011</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00149a011" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=1606-1611&author=G.+M.+Ksanderauthor=A.+M.+Yuanauthor=C.+G.+Diefenbacherauthor=J.+L.+Stanton&title=Angiotensin+converting+enzyme+inhibitors%3A+N-substituted+D-glutamic+acid.+gamma.-dipeptides&doi=10.1021%2Fjm00149a011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm00149a011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00149a011%26sid%3Dliteratum%253Aachs%26aulast%3DKsander%26aufirst%3DG.%2BM.%26aulast%3DYuan%26aufirst%3DA.%2BM.%26aulast%3DDiefenbacher%26aufirst%3DC.%2BG.%26aulast%3DStanton%26aufirst%3DJ.%2BL.%26atitle%3DAngiotensin%2520converting%2520enzyme%2520inhibitors%253A%2520N-substituted%2520D-glutamic%2520acid.%2520gamma.-dipeptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D1606%26epage%3D1611%26doi%3D10.1021%2Fjm00149a011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Aoyagi, Y.</span><span> </span><span class="NLM_article-title">An angiotensin-I converting enzyme inhibitor from buckwheat (Fagopyrum esculentum Moench) flour</span> <span class="citation_source-journal">Phytochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">618</span><span class="NLM_x">â</span> <span class="NLM_lpage">621</span><span class="refDoi">Â DOI: 10.1016/j.phytochem.2005.12.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1016%2Fj.phytochem.2005.12.022" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=618-621&author=Y.+Aoyagi&title=An+angiotensin-I+converting+enzyme+inhibitor+from+buckwheat+%28Fagopyrum+esculentum+Moench%29+flour&doi=10.1016%2Fj.phytochem.2005.12.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.phytochem.2005.12.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytochem.2005.12.022%26sid%3Dliteratum%253Aachs%26aulast%3DAoyagi%26aufirst%3DY.%26atitle%3DAn%2520angiotensin-I%2520converting%2520enzyme%2520inhibitor%2520from%2520buckwheat%2520%2528Fagopyrum%2520esculentum%2520Moench%2529%2520flour%26jtitle%3DPhytochemistry%26date%3D2006%26volume%3D67%26spage%3D618%26epage%3D621%26doi%3D10.1016%2Fj.phytochem.2005.12.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit17"><span> <span class="citation_source-book">GOSTAR Drug Database and Clinical Candidate Database</span>; <span class="NLM_publisher-name">Excelra Knowledge Solutions</span>: <span class="NLM_publisher-loc">Hyderabad, India</span>, Jul 22,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span><a href="https://www.gostardb.com/index.jsp" class="extLink">https://www.gostardb.com/index.jsp</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GOSTAR+Drug+Database+and+Clinical+Candidate+Database%3B+Excelra+Knowledge+Solutions%3A+Hyderabad%2C+India%2C+Jul+22%2C+2013%3B+https%3A%2F%2Fwww.gostardb.com%2Findex.jsp."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGOSTAR%2520Drug%2520Database%2520and%2520Clinical%2520Candidate%2520Database%26pub%3DExcelra%2520Knowledge%2520Solutions%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">â</span> <span class="NLM_lpage">1749</span><span class="refDoi">Â DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Ã in nearly half of the cases and are greater than 2 Ã in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0ljo_9C_hMeWhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelver, W. H.</span><span> </span><span class="NLM_article-title">Docking studies of matrix metalloproteinase inhibitors: zinc parameter optimization to improve the binding free energy prediction</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">â</span> <span class="NLM_lpage">126</span><span class="refDoi">Â DOI: 10.1016/S1093-3263(03)00153-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1016%2FS1093-3263%2803%2900153-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=12932782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVeisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=115-126&author=X.+Huauthor=W.+H.+Shelver&title=Docking+studies+of+matrix+metalloproteinase+inhibitors%3A+zinc+parameter+optimization+to+improve+the+binding+free+energy+prediction&doi=10.1016%2FS1093-3263%2803%2900153-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Docking studies of matrix metalloproteinase inhibitors: zinc parameter optimization to improve the binding free energy prediction</span></div><div class="casAuthors">Hu, Xin; Shelver, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-126</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Docking of metalloproteinase inhibitors remains a challenge due to the zinc multiple coordination geometries and the lack of appropriate force field parameters to model the metal/ligand interactions.  In this study, we explore the docking accuracy and scoring reliability for the docking of matrix metalloproteinase (MMP) inhibitors using AutoDock 3.0.  Potential problems assocd. with zinc ion were investigated by docking 16 matrix metalloproteinase ligands to their crystal structures.  A good coordination between the zinc binding group (ZBG) and the zinc was shown to be a prerequisite for the ligand to fit the binding site.  A simplex optimization of zinc parameters, including zinc radius, well depth, and zinc charges, was performed utilizing the 14 MMP complexes with good docking.  The use of optimized zinc parameters (zinc radius: 0.87 A; well depth: 0.35 kcal/mol; and zinc charges: +0.95 e) shows improvement in both docking accuracy at the zinc binding site and the prediction of binding free energies.  Although further improvement in the docking procedure, particularly the scoring function is needed, optimization of zinc parameters provides an efficient way to improve the performance of AutoDock as a drug discovery tool.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPGJeKzUXd9LVg90H21EOLACvtfcHk0ljo_9C_hMeWhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVeisb8%253D&md5=0f33250fe5f7e84bf3a20c1643e383b5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2803%2900153-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252803%252900153-0%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DShelver%26aufirst%3DW.%2BH.%26atitle%3DDocking%2520studies%2520of%2520matrix%2520metalloproteinase%2520inhibitors%253A%2520zinc%2520parameter%2520optimization%2520to%2520improve%2520the%2520binding%2520free%2520energy%2520prediction%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2003%26volume%3D22%26spage%3D115%26epage%3D126%26doi%3D10.1016%2FS1093-3263%2803%2900153-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Douglas, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuyer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubbe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chibale, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">Fragment-based design for the development of N-domain-selective angiotensin-1-converting enzyme inhibitors</span> <span class="citation_source-journal">Clin. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">â</span> <span class="NLM_lpage">313</span><span class="refDoi">Â DOI: 10.1042/CS20130403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1042%2FCS20130403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=24015848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CktL3M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2014&pages=305-313&author=R.+G.+Douglasauthor=R.+K.+Sharmaauthor=G.+Masuyerauthor=L.+Lubbeauthor=I.+Zamoraauthor=K.+R.+Acharyaauthor=K.+Chibaleauthor=E.+D.+Sturrock&title=Fragment-based+design+for+the+development+of+N-domain-selective+angiotensin-1-converting+enzyme+inhibitors&doi=10.1042%2FCS20130403"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based design for the development of N-domain-selective angiotensin-1-converting enzyme inhibitors</span></div><div class="casAuthors">Douglas, Ross G.; Sharma, Rajni K.; Masuyer, Geoffrey; Lubbe, Lizelle; Zamora, Ismael; Acharya, K. Ravi; Chibale, Kelly; Sturrock, Edward D.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-313</span>CODEN:
                <span class="NLM_cas:coden">CSCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1470-8736</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">ACE (angiotensin-1-converting enzyme) is a zinc metallopeptidase that plays a prominent role in blood pressure regulation and electrolyte homeostasis.  ACE consists of two homologous domains that despite similarities of sequence and topol. display differences in substrate processing and inhibitor binding.  The design of inhibitors that selectively inhibit the N-domain (N-selective) could be useful in treating conditions of tissue injury and fibrosis due to build-up of N-domain-specific substrate Ac-SDKP (N-acetyl-Ser-Asp-Lys-Pro).  Using a receptor-based SHOP (scaffold hopping) approach with N-selective inhibitor RXP407, a shortlist of scaffolds that consisted of modified RXP407 backbones with novel chemotypes was generated.  These scaffolds were selected on the basis of enhanced predicted interaction energies with N-domain residues that differed from their C-domain counterparts.  One scaffold was synthesized and inhibitory binding tested using a fluorogenic ACE assay.  A mol. incorporating a tetrazole moiety in the P2 position (compd. 33RE) displayed potent inhibition (Ki=11.21Â±0.74 nM) and was 927-fold more selective for the N-domain than the C-domain.  A crystal structure of compd. 33RE in complex with the N-domain revealed its mode of binding through arom. stacking with His388 and a direct hydrogen bond with the hydroxy group of the N-domain specific Tyr369.  This work further elucidates the mol. basis for N-domain selective inhibition and assists in the design of novel N-selective ACE inhibitors that could be employed in treatment of fibrosis disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra1vuAtVm_07Vg90H21EOLACvtfcHk0ljo_9C_hMeWhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CktL3M&md5=b74b9a12946c6235f20d766eddea846f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1042%2FCS20130403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FCS20130403%26sid%3Dliteratum%253Aachs%26aulast%3DDouglas%26aufirst%3DR.%2BG.%26aulast%3DSharma%26aufirst%3DR.%2BK.%26aulast%3DMasuyer%26aufirst%3DG.%26aulast%3DLubbe%26aufirst%3DL.%26aulast%3DZamora%26aufirst%3DI.%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26aulast%3DChibale%26aufirst%3DK.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DFragment-based%2520design%2520for%2520the%2520development%2520of%2520N-domain-selective%2520angiotensin-1-converting%2520enzyme%2520inhibitors%26jtitle%3DClin.%2520Sci.%26date%3D2014%26volume%3D126%26spage%3D305%26epage%3D313%26doi%3D10.1042%2FCS20130403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Lubbe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sewell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">The influence of angiotensin converting enzyme mutations on the kinetics and dynamics of N-domain selective inhibition</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">3941</span><span class="NLM_x">â</span> <span class="NLM_lpage">3961</span><span class="refDoi">Â DOI: 10.1111/febs.13900</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1111%2Ffebs.13900" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2016&pages=3941-3961&author=L.+Lubbeauthor=B.+T.+Sewellauthor=E.+D.+Sturrock&title=The+influence+of+angiotensin+converting+enzyme+mutations+on+the+kinetics+and+dynamics+of+N-domain+selective+inhibition&doi=10.1111%2Ffebs.13900"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1111%2Ffebs.13900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.13900%26sid%3Dliteratum%253Aachs%26aulast%3DLubbe%26aufirst%3DL.%26aulast%3DSewell%26aufirst%3DB.%2BT.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DThe%2520influence%2520of%2520angiotensin%2520converting%2520enzyme%2520mutations%2520on%2520the%2520kinetics%2520and%2520dynamics%2520of%2520N-domain%2520selective%2520inhibition%26jtitle%3DFEBS%2520J.%26date%3D2016%26volume%3D283%26spage%3D3941%26epage%3D3961%26doi%3D10.1111%2Ffebs.13900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Waterman, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gildea, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkhurst, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewster, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, G.</span><span> </span><span class="NLM_article-title">Diffraction-geometry refinement in the DIALS framework</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">558</span><span class="NLM_x">â</span> <span class="NLM_lpage">575</span><span class="refDoi">Â DOI: 10.1107/S2059798316002187</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS2059798316002187" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFSkurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=558-575&author=D.+G.+Watermanauthor=G.+Winterauthor=R.+J.+Gildeaauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=N.+K.+Sauterauthor=G.+Evans&title=Diffraction-geometry+refinement+in+the+DIALS+framework&doi=10.1107%2FS2059798316002187"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Diffraction-geometry refinement in the DIALS framework</span></div><div class="casAuthors">Waterman, David G.; Winter, Graeme; Gildea, Richard J.; Parkhurst, James M.; Brewster, Aaron S.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">558-575</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Rapid data collection and modern computing resources provide the opportunity to revisit the task of optimizing the model of diffraction geometry prior to integration.  A comprehensive description is given of new software that builds upon established methods by performing a single global refinement procedure, utilizing a smoothly varying model of the crystal lattice where appropriate.  This global refinement technique extends to multiple data sets, providing useful constraints to handle the problem of correlated parameters, particularly for small wedges of data.  Examples of advanced uses of the software are given and the design is explained in detail, with particular emphasis on the flexibility and extensibility it entails.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowmCZHkbbl6bVg90H21EOLACvtfcHk0lgXCnZAoYlJkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFSkurs%253D&md5=0041260941cd2e0581c8a1db0b5add97</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS2059798316002187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316002187%26sid%3Dliteratum%253Aachs%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDiffraction-geometry%2520refinement%2520in%2520the%2520DIALS%2520framework%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2016%26volume%3D72%26spage%3D558%26epage%3D575%26doi%3D10.1107%2FS2059798316002187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">â</span> <span class="NLM_lpage">1214</span><span class="refDoi">Â DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgXCnZAoYlJkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Collaborative Computational Project 4</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760&author=Collaborative+Computational+Project+4&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">â</span> <span class="NLM_lpage">674</span><span class="refDoi">Â DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ÎF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgXCnZAoYlJkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">â</span> <span class="NLM_lpage">255</span><span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhp8stujf_XYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">BunkÃ³czi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">â</span> <span class="NLM_lpage">221</span><span class="refDoi">Â DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhp8stujf_XYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">â</span> <span class="NLM_lpage">2132</span><span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lifaNYwR5-sJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">â</span> <span class="NLM_lpage">21</span><span class="refDoi">Â DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lifaNYwR5-sJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">McNicholas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noble, M.</span><span> </span><span class="NLM_article-title">Presenting your structures: the CCP4mg molecular-graphics software</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">386</span><span class="NLM_x">â</span> <span class="NLM_lpage">394</span><span class="refDoi">Â DOI: 10.1107/S0907444911007281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444911007281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=21460457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=386-394&author=S.+McNicholasauthor=E.+Pottertonauthor=K.+Wilsonauthor=M.+Noble&title=Presenting+your+structures%3A+the+CCP4mg+molecular-graphics+software&doi=10.1107%2FS0907444911007281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Presenting your structures: The CCP4mg molecular-graphics software</span></div><div class="casAuthors">McNicholas, S.; Potterton, E.; Wilson, K. S.; Noble, M. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">386-394</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">CCP4mg is a mol.-graphics program that is designed to give rapid access to both straightforward and complex static and dynamic representations of macromol. structures.  It has recently been updated with a new interface that provides more sophisticated atom-selection options and a wizard to facilitate the generation of complex scenes.  These scenes may contain a mixt. of coordinate-derived and abstr. graphical objects, including text objects, arbitrary vectors, geometric objects and imported images, which can enhance a picture and eliminate the need for subsequent editing.  Scene descriptions can be saved to file and transferred to other mols.  Here, the substantially enhanced version 2 of the program, with a new underlying GUI toolkit, is described.  A built-in rendering module produces publication-quality images.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdtflrxrILlbVg90H21EOLACvtfcHk0lifaNYwR5-sJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqurw%253D&md5=d96403a39bc723a66f11a730f81f76c3</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007281%26sid%3Dliteratum%253Aachs%26aulast%3DMcNicholas%26aufirst%3DS.%26aulast%3DPotterton%26aufirst%3DE.%26aulast%3DWilson%26aufirst%3DK.%26aulast%3DNoble%26aufirst%3DM.%26atitle%3DPresenting%2520your%2520structures%253A%2520the%2520CCP4mg%2520molecular-graphics%2520software%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D386%26epage%3D394%26doi%3D10.1107%2FS0907444911007281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref20':['cit20'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Gyles
E. Cozier, Lauren B. Arendse, Sylva L. Schwager, Edward D. Sturrock, <span class="NLM_string-name hlFld-ContribAuthor">K. Ravi Acharya</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Basis for Multiple Omapatrilat Binding Sites within the ACE C-Domain: Implications for Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (22)
                                     , 10141-10154. <a href="https://doi.org/10.1021/acs.jmedchem.8b01309" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01309</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01309%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMolecular%252BBasis%252Bfor%252BMultiple%252BOmapatrilat%252BBinding%252BSites%252Bwithin%252Bthe%252BACE%252BC-Domain%25253A%252BImplications%252Bfor%252BDrug%252BDesign%26aulast%3DCozier%26aufirst%3DGyles%2BE.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D19082018%26date%3D07112018%26date%3D29102018%26volume%3D61%26issue%3D22%26spage%3D10141%26epage%3D10154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julio  Caballero</span>. </span><span class="cited-content_cbyCitation_article-title">The latest automated docking technologies for novel drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>16 </em>
                                    (6)
                                     , 625-645. <a href="https://doi.org/10.1080/17460441.2021.1858793" title="DOI URL">https://doi.org/10.1080/17460441.2021.1858793</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1858793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1858793%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Blatest%252Bautomated%252Bdocking%252Btechnologies%252Bfor%252Bnovel%252Bdrug%252Bdiscovery%26aulast%3DCaballero%26aufirst%3DJulio%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D6%26spage%3D625%26epage%3D645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lizelle  Lubbe</span>, <span class="hlFld-ContribAuthor ">GylesÂ E.  Cozier</span>, <span class="hlFld-ContribAuthor ">Delia  Oosthuizen</span>, <span class="hlFld-ContribAuthor ">K.Â Ravi  Acharya</span>, <span class="hlFld-ContribAuthor ">EdwardÂ D.  Sturrock</span>. </span><span class="cited-content_cbyCitation_article-title">ACE2 and ACE: structure-based insights into mechanism, regulation and receptor recognition by SARS-CoV. </span><span class="cited-content_cbyCitation_journal-name">Clinical Science</span><span> <strong>2020,</strong> <em>134 </em>
                                    (21)
                                     , 2851-2871. <a href="https://doi.org/10.1042/CS20200899" title="DOI URL">https://doi.org/10.1042/CS20200899</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/CS20200899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FCS20200899%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Science%26atitle%3DACE2%252Band%252BACE%25253A%252Bstructure-based%252Binsights%252Binto%252Bmechanism%25252C%252Bregulation%252Band%252Breceptor%252Brecognition%252Bby%252BSARS-CoV%26aulast%3DLubbe%26aufirst%3DLizelle%26date%3D2020%26date%3D2020%26volume%3D134%26issue%3D21%26spage%3D2851%26epage%3D2871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergio  Alfaro</span>, <span class="hlFld-ContribAuthor ">Carlos  Navarro-Retamal</span>, <span class="hlFld-ContribAuthor ">Julio  Caballero</span>. </span><span class="cited-content_cbyCitation_article-title">Transforming Non-Selective Angiotensin-Converting Enzyme Inhibitors in C- and N-domain Selective Inhibitors by Using Computational Tools. </span><span class="cited-content_cbyCitation_journal-name">Mini-Reviews in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (14)
                                     , 1436-1446. <a href="https://doi.org/10.2174/1389557520666191224113830" title="DOI URL">https://doi.org/10.2174/1389557520666191224113830</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1389557520666191224113830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1389557520666191224113830%26sid%3Dliteratum%253Aachs%26jtitle%3DMini-Reviews%2520in%2520Medicinal%2520Chemistry%26atitle%3DTransforming%252BNon-Selective%252BAngiotensin-Converting%252BEnzyme%252BInhibitors%252Bin%252BC-%252Band%252BN-domain%252BSelective%252BInhibitors%252Bby%252BUsing%252BComputational%252BTools%26aulast%3DAlfaro%26aufirst%3DSergio%26date%3D2020%26volume%3D20%26issue%3D14%26spage%3D1436%26epage%3D1446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gyles E.  Cozier</span>, <span class="hlFld-ContribAuthor ">Lizelle  Lubbe</span>, <span class="hlFld-ContribAuthor ">Edward D.  Sturrock</span>, <span class="hlFld-ContribAuthor ">K. Ravi  Acharya</span>. </span><span class="cited-content_cbyCitation_article-title">ACE-domain selectivity extends beyond direct interacting residues at the active site. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2020,</strong> <em>477 </em>
                                    (7)
                                     , 1241-1259. <a href="https://doi.org/10.1042/BCJ20200060" title="DOI URL">https://doi.org/10.1042/BCJ20200060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20200060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20200060%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DACE-domain%252Bselectivity%252Bextends%252Bbeyond%252Bdirect%252Binteracting%252Bresidues%252Bat%252Bthe%252Bactive%252Bsite%26aulast%3DCozier%26aufirst%3DGyles%2BE.%26date%3D2020%26date%3D2020%26volume%3D477%26issue%3D7%26spage%3D1241%26epage%3D1259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julio  Caballero</span>. </span><span class="cited-content_cbyCitation_article-title">Considerations for Docking of Selective Angiotensin-Converting Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (2)
                                     , 295. <a href="https://doi.org/10.3390/molecules25020295" title="DOI URL">https://doi.org/10.3390/molecules25020295</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25020295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25020295%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DConsiderations%252Bfor%252BDocking%252Bof%252BSelective%252BAngiotensin-Converting%252BEnzyme%252BInhibitors%26aulast%3DCaballero%26aufirst%3DJulio%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D2%26spage%3D295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Md Yousof  Ali</span>, <span class="hlFld-ContribAuthor ">Su Hui  Seong</span>, <span class="hlFld-ContribAuthor ">Hyun Ah  Jung</span>, <span class="hlFld-ContribAuthor ">Jae Sue  Choi</span>. </span><span class="cited-content_cbyCitation_article-title">Angiotensin-I-Converting Enzyme Inhibitory Activity of Coumarins from Angelica decursiva. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (21)
                                     , 3937. <a href="https://doi.org/10.3390/molecules24213937" title="DOI URL">https://doi.org/10.3390/molecules24213937</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24213937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24213937%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DAngiotensin-I-Converting%252BEnzyme%252BInhibitory%252BActivity%252Bof%252BCoumarins%252Bfrom%252BAngelica%252Bdecursiva%26aulast%3DAli%26aufirst%3DMd%2BYousof%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D21%26spage%3D3937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lauren B.  Arendse</span>, <span class="hlFld-ContribAuthor ">A. H. Jan  Danser</span>, <span class="hlFld-ContribAuthor ">Marko  Poglitsch</span>, <span class="hlFld-ContribAuthor ">Rhian M.  Touyz</span>, <span class="hlFld-ContribAuthor ">John C.  Burnett</span>, <span class="hlFld-ContribAuthor ">Catherine  Llorens-Cortes</span>, <span class="hlFld-ContribAuthor ">Mario R.  Ehlers</span>, <span class="hlFld-ContribAuthor ">Edward D.  Sturrock</span>, . </span><span class="cited-content_cbyCitation_article-title">Novel Therapeutic Approaches Targeting the Renin-Angiotensin System and Associated Peptides in Hypertension and Heart Failure. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Reviews</span><span> <strong>2019,</strong> <em>71 </em>
                                    (4)
                                     , 539-570. <a href="https://doi.org/10.1124/pr.118.017129" title="DOI URL">https://doi.org/10.1124/pr.118.017129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/pr.118.017129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fpr.118.017129%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Reviews%26atitle%3DNovel%252BTherapeutic%252BApproaches%252BTargeting%252Bthe%252BRenin-Angiotensin%252BSystem%252Band%252BAssociated%252BPeptides%252Bin%252BHypertension%252Band%252BHeart%252BFailure%26aulast%3DArendse%26aufirst%3DLauren%2BB.%26date%3D2019%26date%3D2019%26volume%3D71%26issue%3D4%26spage%3D539%26epage%3D570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">MÃ¡ja  PolakoviÄovÃ¡</span>, <span class="hlFld-ContribAuthor ">Josef  JampÃ­lek</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in Structural Biology of ACE and Development of Domain Selective ACE-inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry</span><span> <strong>2019,</strong> <em>15 </em>
                                    (6)
                                     , 574-587. <a href="https://doi.org/10.2174/1573406415666190514081132" title="DOI URL">https://doi.org/10.2174/1573406415666190514081132</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1573406415666190514081132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1573406415666190514081132%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%26atitle%3DAdvances%252Bin%252BStructural%252BBiology%252Bof%252BACE%252Band%252BDevelopment%252Bof%252BDomain%252BSelective%252BACE-inhibitors%26aulast%3DPolakovi%25C4%258Dov%25C3%25A1%26aufirst%3DM%25C3%25A1ja%26date%3D2019%26volume%3D15%26issue%3D6%26spage%3D574%26epage%3D587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lizelle  Lubbe</span>, <span class="hlFld-ContribAuthor ">Edward D  Sturrock</span>. </span><span class="cited-content_cbyCitation_article-title">Interacting cogs in the machinery of the renin angiotensin system. </span><span class="cited-content_cbyCitation_journal-name">Biophysical Reviews</span><span> <strong>2019,</strong> <em>11 </em>
                                    (4)
                                     , 583-589. <a href="https://doi.org/10.1007/s12551-019-00555-w" title="DOI URL">https://doi.org/10.1007/s12551-019-00555-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12551-019-00555-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12551-019-00555-w%26sid%3Dliteratum%253Aachs%26jtitle%3DBiophysical%2520Reviews%26atitle%3DInteracting%252Bcogs%252Bin%252Bthe%252Bmachinery%252Bof%252Bthe%252Brenin%252Bangiotensin%252Bsystem%26aulast%3DLubbe%26aufirst%3DLizelle%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D4%26spage%3D583%26epage%3D589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erik  Andrade-Jorge</span>, <span class="hlFld-ContribAuthor ">Jessica E.  RodrÃ­guez</span>, <span class="hlFld-ContribAuthor ">JosÃ©  Bribiesca-Carlos</span>, <span class="hlFld-ContribAuthor ">Itzell A.  Gallardo-OrtÃ­z</span>, <span class="hlFld-ContribAuthor ">JosÃ© G.  Trujillo-Ferrara</span>, <span class="hlFld-ContribAuthor ">Rafael  Villalobos-Molina</span>. </span><span class="cited-content_cbyCitation_article-title">Novel phthalamide derivatives as antihypertensive agents: rapid and clean synthesis, in silico and in vivo evaluation. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2019,</strong> <em>28 </em>
                                    (5)
                                     , 681-695. <a href="https://doi.org/10.1007/s00044-019-02327-3" title="DOI URL">https://doi.org/10.1007/s00044-019-02327-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-019-02327-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-019-02327-3%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DNovel%252Bphthalamide%252Bderivatives%252Bas%252Bantihypertensive%252Bagents%25253A%252Brapid%252Band%252Bclean%252Bsynthesis%25252C%252Bin%252Bsilico%252Band%252Bin%252Bvivo%252Bevaluation%26aulast%3DAndrade-Jorge%26aufirst%3DErik%26date%3D2019%26date%3D2019%26volume%3D28%26issue%3D5%26spage%3D681%26epage%3D695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhensheng  Jiang</span>, <span class="hlFld-ContribAuthor ">Qidong  You</span>, <span class="hlFld-ContribAuthor ">Xiaojin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Medicinal chemistry of metal chelating fragments in metalloenzyme active sites: A perspective. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>165 </em>, 172-197. <a href="https://doi.org/10.1016/j.ejmech.2019.01.018" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.018%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMedicinal%252Bchemistry%252Bof%252Bmetal%252Bchelating%252Bfragments%252Bin%252Bmetalloenzyme%252Bactive%252Bsites%25253A%252BA%252Bperspective%26aulast%3DJiang%26aufirst%3DZhensheng%26date%3D2019%26volume%3D165%26spage%3D172%26epage%3D197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Franziska  HÃ¤rtner</span>, <span class="hlFld-ContribAuthor ">Miguel A.  Andrade-Navarro</span>, <span class="hlFld-ContribAuthor ">Gregorio  Alanis-Lobato</span>. </span><span class="cited-content_cbyCitation_article-title">Geometric characterisation of disease modules. </span><span class="cited-content_cbyCitation_journal-name">Applied Network Science</span><span> <strong>2018,</strong> <em>3 </em>
                                    (1)
                                     <a href="https://doi.org/10.1007/s41109-018-0066-3" title="DOI URL">https://doi.org/10.1007/s41109-018-0066-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s41109-018-0066-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs41109-018-0066-3%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520Network%2520Science%26atitle%3DGeometric%252Bcharacterisation%252Bof%252Bdisease%252Bmodules%26aulast%3DH%25C3%25A4rtner%26aufirst%3DFranziska%26date%3D2018%26date%3D2018%26volume%3D3%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic outline of the ACE binding site occupied by a typical ACEi drug. ACE inhibitors bind to the catalytic region of the active sites of both domains via a chelation interaction with the zinc atom and a hydrogen bond with the catalytic Glu residue. Salt-bridge and hydrogen bonding interactions with the respective Gln, Lys, and Tyr residues in the S<sub>2</sub>â² subsite hold in place the terminal P<sub>2</sub>â² carboxylic which is attached to a cyclic group. Hydrophobic side chain mimicking P<sub>1</sub> and P<sub>1</sub>â² groups interact with the respective hydrophobic S<sub>1</sub>â² and S<sub>2</sub>â² subsites. Different regions of the active site are color coded according to polarity as indicated in the legend.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. An illustration of the positional (A), metal chelation (B), and H-bonding (C) docking constraints created and implemented under various permutations when docking ligands against both the N- and C-domains of ACE. Positional constraints (A) were set holding either a carbon or nitrogen atom in one of the 1 Ã constraint spheres. The metal chelation constraint was created between the zinc atom and a carboxylic acid ZBG from the ligand. The H-bonding constraints were created between both the Lys489/511 and Gln259/281 donors from N-/C-domain and an H-bond acceptor from the P<sub>2</sub>â² group on the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Poses of <b>1</b> (A,B), <b>4</b> (C,D), and <b>10</b> (E,F) docked into the N- (cyan; left) and C-domains (green; right). The two key ACEi binding features, a carboxylic acid Zn chelation and a second carboxylic acid acting as an H-bond acceptor in the S<sub>2</sub>â² subsite, are evident. Compound <b>1</b> in the N-domain has a twisted peptide bond for the P<sub>1</sub> proline and interactions between the carboxylic acid P<sub>2</sub> group with Tyr369 and Arg381, while in the C-domain <b>1</b> contains a trans proline and the P<sub>2</sub> carboxylic acid has no interactions. The bicylic link in compound <b>4</b> is too long and rigid to allow for a conventional orientation in either domain and causes the P<sub>2</sub>â² and ZBG carboxylic acid groups to bind in different positions in the N- and C-domains, respectively. This, along with the flexible alkyl chain, allows for a large variation in the position of the P<sub>2</sub> group. Compound <b>10</b> is too short to form interactions from its P<sub>2</sub> carboxylic group with Arg381 or Tyr369 in the N-domain; instead, it coordinates with the zinc ion. In the C-domain, there are no interactions with this group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Representative samples of ligand poses from the extended set of diprolyl compounds. (A,B), (C,D), and (E,F) show <b>16</b>, <b>17</b>, and <b>21</b>, respectively, docked in the N- (cyan; left) and C-domains (green; right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Diprolyl Series<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(i) TMSCN, MeOH/NH<sub>4</sub><sup>+</sup>Cl<sup>â</sup>, 25 Â°C, 36 h, 36%; (ii) (a) HCl in MeOH, 25 Â°C, 2 d, (b) (Boc)<sub>2</sub>O, Et<sub>3</sub>N/MeOH, 72%; (iii) (a) Amberlyst 15, MeOH, 60 Â°C, 10 days, (b) HCl/dioxane 25 Â°C, 12 h, 40%; (iv) (a) RCO<sub>2</sub>H, T3P, Et<sub>3</sub>N, DCM, 0â25 Â°C, (b) chiral HPLC diastereomer separation; (vi) (a) LiOH, THF/H<sub>2</sub>O, (b) HCl/dioxane.</p></p></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Schematic representation of compound <b>16</b> bound to the N-domain of ACE overlaid with the 2<i>mFo</i>â<i>DFc</i> (blue, contoured at 1Ï level) and <i>mFo</i>â<i>DFc</i> (green, contoured at 3Ï level) electron density omit maps. The zinc ion is shown as a green sphere with the coordinating side chains shown as sticks. (B) Ligplot representation of the binding site interactions of compound <b>16</b> in complex with ACE N-domain. The Arg381 conformation that could possibly provide water mediated interactions with <b>16</b> is shown for completeness, although this is not a major form.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Close-up views of the P<sub>2</sub> carboxylic acid binding site. The 2<i>mFo</i>â<i>DFc</i> electron density map (contoured at 1Ï level) is shown in blue, and the zinc ion and relevant water molecules are shown as green and red spheres, respectively. (AâC) Schematic representation showing the possible alternate conformations of Arg381 in three molecules of the complex structure with ACE N-domain. This highlights the weak density for Arg381. (D) Schematic representation showing the strong interaction between Tyr369 and the P<sub>2</sub> carboxylic acid of compound <b>16</b> and the water mediated interaction with Arg500.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) An overlay of the crystal (orange) and docked (cyan) structures of <b>16</b> in the N-domain of ACE. The zinc ions in the crystal and docked structures are shown as green and gray spheres, respectively. The two poses of the ligand are closely correlated (RMSD = 1.182 Ã), while Arg381 is seen to be highly flexible, with two conformations of this flexibility being visible to varying extents in the different chains of the asymmetric units. (B) An overlay of compound <b>16</b>âN-domain crystal structure (orange) and <b>16</b> docked in to the C-domain (green). The zinc ions in the crystal and docked structures are shown as green and gray spheres, respectively. The major portion of the scaffolds overlay between the two structures, while a shift is observed for P<sub>2</sub> carboxylic acid group, showing interactions with Tyr369 and Arg522 for the N- and C-domains, respectively. In addition, there is a water mediated interaction with Arg500 in the N-domain.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/medium/jm-2017-01478z_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Overlays of the ACE N-domain complex structures of compound <b>16</b> (orange) with (A) RXP407 (cyan) and (B) 33RE (lilac). The zinc ions are shown as a green sphere for the compound <b>16</b> structure and gray spheres for the RXP407 and 33RE structures. Water molecules involved in mediated interactions between <b>16</b> and the binding site are shown as red spheres. The water molecule marked * is conserved in the RXP407 structure in an almost identical position. A water molecule in the 33RE structure that performs a similar mediated interaction is shown as a pink sphere.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01478/20180110/images/large/jm-2017-01478z_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01478&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80257" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80257" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 36 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Cushman, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ondetti, M.</span><span> </span><span class="NLM_article-title">Design of potent competitive inhibitors of angiotensin-converting enzyme. Carboxyalkanoyl and mercaptoalkanoyl amino acids</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">5484</span><span class="NLM_x">â</span> <span class="NLM_lpage">5491</span><span class="refDoi">Â DOI: 10.1021/bi00644a014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00644a014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaE1cXhvVSqsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1977&pages=5484-5491&author=D.+W.+Cushmanauthor=H.+Cheungauthor=E.+Saboauthor=M.+Ondetti&title=Design+of+potent+competitive+inhibitors+of+angiotensin-converting+enzyme.+Carboxyalkanoyl+and+mercaptoalkanoyl+amino+acids&doi=10.1021%2Fbi00644a014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Design of potent competitive inhibitors of angiotensin-converting enzyme.  Carboxyalkanoyl and mercaptoalkanoyl amino acids</span></div><div class="casAuthors">Cushman, D. W.; Cheung, H. S.; Sabo, E. F.; Ondetti, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5484-91</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">A hypothetical model of the active site of angiotensin-coverting enzyme (I) was utilized to guide the design and synthesis of specific inhibitors.  By analogy to bovine carboxypeptidase A, the active site of I was proposed to contain 3 important groups that participate in binding of peptide substrates: a carboxyl-binding group, a group with affinity for the C-terminal peptide bond, and a tightly bound Zn2+ that could coordinate with the carbonyl of the penultimate (scissiie) peptide bond.  According to the model, a succinyl amino acid could interact with each of these binding groups via its amino acid carboxyl, amide bond, and succinyl carboxyl, resp., and thus act as a specific competitive inhibitor of the enzyme.  Succinyl-L-proline was such an inhibitor (I50 = 330 Î¼M), and attempts to optimize its interaction with the active site of the enzyme as proposed in the model led to the synthesis of D-2-methylsuccinyl-L-proline (R,S) (Ki = 2.5 Î¼M), and D-2-methylglutaryl-L-proline (R,S) (Ki = 0.8 Î¼M).  Replacement of the succinyl carboxyl group of these compds. by a SH group led to a series of extremely potent competitive inhibitors of I, including 3-mercaptopropranoyl-L-proline (SQ 13,863, Ki = 0.012 Î¼M) and D-3-mercapto-2-methylpropranoyl-L-proline (S,S) (SQ 14,225, Ki = 0.0017 Î¼M).  These compds. are also potent orally active inhibitors of I and have great potential as antihypertensive agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWc6ldKEo4oLVg90H21EOLACvtfcHk0lgO2fSrdlt3Og"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXhvVSqsw%253D%253D&md5=e575497901ccc7c7370ca7dec99882e7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fbi00644a014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00644a014%26sid%3Dliteratum%253Aachs%26aulast%3DCushman%26aufirst%3DD.%2BW.%26aulast%3DCheung%26aufirst%3DH.%26aulast%3DSabo%26aufirst%3DE.%26aulast%3DOndetti%26aufirst%3DM.%26atitle%3DDesign%2520of%2520potent%2520competitive%2520inhibitors%2520of%2520angiotensin-converting%2520enzyme.%2520Carboxyalkanoyl%2520and%2520mercaptoalkanoyl%2520amino%2520acids%26jtitle%3DBiochemistry%26date%3D1977%26volume%3D16%26spage%3D5484%26epage%3D5491%26doi%3D10.1021%2Fbi00644a014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Patchett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tristram, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyvratt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taub, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ten Broeke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ondeyka, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorsett, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenlee, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohr, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffsommer, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshua, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruyle, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothrock, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aster, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maycock, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirschmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulm, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassil, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, C. A.</span><span> </span><span class="NLM_article-title">A new class of angiotensin-converting enzyme inhibitors</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">280</span><span class="NLM_x">â</span> <span class="NLM_lpage">283</span><span class="refDoi">Â DOI: 10.1038/288280a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1038%2F288280a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=6253826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaL3MXktlGhtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=1980&pages=280-283&author=A.+Patchettauthor=E.+Harrisauthor=E.+Tristramauthor=M.+Wyvrattauthor=M.+Wuauthor=D.+Taubauthor=E.+Petersonauthor=T.+Ikelerauthor=J.+Ten+Broekeauthor=L.+Payneauthor=D.+L.+Ondeykaauthor=E.+D.+Thorsettauthor=W.+J.+Greenleeauthor=N.+S.+Lohrauthor=R.+D.+Hoffsommerauthor=H.+Joshuaauthor=W.+V.+Ruyleauthor=J.+W.+Rothrockauthor=S.+D.+Asterauthor=A.+L.+Maycockauthor=F.+M.+Robinsonauthor=R.+Hirschmannauthor=C.+S.+Sweetauthor=E.+H.+Ulmauthor=D.+M.+Grossauthor=T.+C.+Vassilauthor=C.+A.+Stone&title=A+new+class+of+angiotensin-converting+enzyme+inhibitors&doi=10.1038%2F288280a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A new class of angiotensin-converting enzyme inhibitors</span></div><div class="casAuthors">Patchett, A. A.; Harris, E.; Tristram, E. W.; Wyvratt, M. J.; Wu, M. T.; Taub, D.; Peterson, E. R.; Ikeler, T. J.; Ten Broeke, J.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">5788</span>),
    <span class="NLM_cas:pages">280-3</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Substituted N-carboxymethyldipeptides were evaluated for their ability to antagonize the pressor effect of angiotensin I (I)  [9041-90-1] in anesthetized rats and dogs (100 and 600 ng/kg, i.v., resp.); Ph(CH2)2CH(CO2H)-Ala-Pro (II)  [76420-72-9], Ph(CH2)2CH(CO2Et)-Ala-Pro maleate (III)  [75847-73-3] and Ph(CH2)2CH(CO2H)-Lys-Pro (IV)  [77726-95-5] were highly active by the i.v. route in rats and dogs, except for III in dogs, but all were orally active in rats and dogs.  II, III, and IV showed high potency in inhibiting the blood pressure response to I and all had a long duration of action; high angiotensin-converting enzyme (EC 3.4.15.1) (V)  [9015-82-1] inhibitory activities at doses as low as 2.3 Î¼g/kg were obsd. after i.v. administration and after oral administration, the compds. showed good activities with duration of action of â¥6 h at doses of 0.3 mg/kg.  Thus, V inhibitors have important design implications for the synthesis of inhibitors of other metallodipeptidyl carboxypeptidases and possibly of metalloendopeptidases more generally.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2vFzl18NAf7Vg90H21EOLACvtfcHk0lgdwFWZeGxfNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXktlGhtLY%253D&md5=3a395d264e4e5c14d0cca0bcb6065a73</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F288280a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F288280a0%26sid%3Dliteratum%253Aachs%26aulast%3DPatchett%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DE.%26aulast%3DTristram%26aufirst%3DE.%26aulast%3DWyvratt%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DTaub%26aufirst%3DD.%26aulast%3DPeterson%26aufirst%3DE.%26aulast%3DIkeler%26aufirst%3DT.%26aulast%3DTen%2BBroeke%26aufirst%3DJ.%26aulast%3DPayne%26aufirst%3DL.%26aulast%3DOndeyka%26aufirst%3DD.%2BL.%26aulast%3DThorsett%26aufirst%3DE.%2BD.%26aulast%3DGreenlee%26aufirst%3DW.%2BJ.%26aulast%3DLohr%26aufirst%3DN.%2BS.%26aulast%3DHoffsommer%26aufirst%3DR.%2BD.%26aulast%3DJoshua%26aufirst%3DH.%26aulast%3DRuyle%26aufirst%3DW.%2BV.%26aulast%3DRothrock%26aufirst%3DJ.%2BW.%26aulast%3DAster%26aufirst%3DS.%2BD.%26aulast%3DMaycock%26aufirst%3DA.%2BL.%26aulast%3DRobinson%26aufirst%3DF.%2BM.%26aulast%3DHirschmann%26aufirst%3DR.%26aulast%3DSweet%26aufirst%3DC.%2BS.%26aulast%3DUlm%26aufirst%3DE.%2BH.%26aulast%3DGross%26aufirst%3DD.%2BM.%26aulast%3DVassil%26aufirst%3DT.%2BC.%26aulast%3DStone%26aufirst%3DC.%2BA.%26atitle%3DA%2520new%2520class%2520of%2520angiotensin-converting%2520enzyme%2520inhibitors%26jtitle%3DNature%26date%3D1980%26volume%3D288%26spage%3D280%26epage%3D283%26doi%3D10.1038%2F288280a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Bakhle, Y.</span><span> </span><span class="NLM_article-title">Conversion of angiotensin I to angiotensin II by cell-free extracts of dog lung</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1968</span><span class="NLM_x">, </span> <span class="NLM_volume">220</span><span class="NLM_x">, </span> <span class="NLM_fpage">919</span><span class="NLM_x">â</span> <span class="NLM_lpage">921</span><span class="refDoi">Â DOI: 10.1038/220919a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1038%2F220919a0" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=220&publication_year=1968&pages=919-921&author=Y.+Bakhle&title=Conversion+of+angiotensin+I+to+angiotensin+II+by+cell-free+extracts+of+dog+lung&doi=10.1038%2F220919a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F220919a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F220919a0%26sid%3Dliteratum%253Aachs%26aulast%3DBakhle%26aufirst%3DY.%26atitle%3DConversion%2520of%2520angiotensin%2520I%2520to%2520angiotensin%2520II%2520by%2520cell-free%2520extracts%2520of%2520dog%2520lung%26jtitle%3DNature%26date%3D1968%26volume%3D220%26spage%3D919%26epage%3D921%26doi%3D10.1038%2F220919a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Byers, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfenden, R.</span><span> </span><span class="NLM_article-title">Binding of the by-product analog benzylsuccinic acid by carboxypeptidase A</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2070</span><span class="NLM_x">â</span> <span class="NLM_lpage">2078</span><span class="refDoi">Â DOI: 10.1021/bi00735a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00735a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1973&pages=2070-2078&author=L.+D.+Byersauthor=R.+Wolfenden&title=Binding+of+the+by-product+analog+benzylsuccinic+acid+by+carboxypeptidase+A&doi=10.1021%2Fbi00735a008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fbi00735a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00735a008%26sid%3Dliteratum%253Aachs%26aulast%3DByers%26aufirst%3DL.%2BD.%26aulast%3DWolfenden%26aufirst%3DR.%26atitle%3DBinding%2520of%2520the%2520by-product%2520analog%2520benzylsuccinic%2520acid%2520by%2520carboxypeptidase%2520A%26jtitle%3DBiochemistry%26date%3D1973%26volume%3D12%26spage%3D2070%26epage%3D2078%26doi%3D10.1021%2Fbi00735a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Fuchs, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaud, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capecchi, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernstein, K. E.</span><span> </span><span class="NLM_article-title">Angiotensin-converting enzyme C-terminal catalytic domain is the main site of angiotensin I cleavage in vivo</span> <span class="citation_source-journal">Hypertension</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">â</span> <span class="NLM_lpage">274</span><span class="refDoi">Â DOI: 10.1161/HYPERTENSIONAHA.107.097865</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1161%2FHYPERTENSIONAHA.107.097865" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=18158355" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt1OjtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=267-274&author=S.+Fuchsauthor=H.+D.+Xiaoauthor=C.+Hubertauthor=A.+Michaudauthor=D.+J.+Campbellauthor=J.+W.+Adamsauthor=M.+R.+Capecchiauthor=P.+Corvolauthor=K.+E.+Bernstein&title=Angiotensin-converting+enzyme+C-terminal+catalytic+domain+is+the+main+site+of+angiotensin+I+cleavage+in+vivo&doi=10.1161%2FHYPERTENSIONAHA.107.097865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Angiotensin-Converting Enzyme C-Terminal Catalytic Domain Is the Main Site of Angiotensin I Cleavage In Vivo</span></div><div class="casAuthors">Fuchs, Sebastien; Xiao, Hong D.; Hubert, Christine; Michaud, Annie; Campbell, Duncan J.; Adams, Jonathan W.; Capecchi, Mario R.; Corvol, Pierre; Bernstein, Kenneth E.</div><div class="citationInfo"><span class="NLM_cas:title">Hypertension</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">267-274</span>CODEN:
                <span class="NLM_cas:coden">HPRTDN</span>;
        ISSN:<span class="NLM_cas:issn">0194-911X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Angiotensin-converting enzyme (ACE) plays a central role in the prodn. of the vasoconstrictor angiotensin II.  ACE is a single polypeptide, but it contains 2 homologous and independent catalytic domains, each of which binds zinc.  To understand the in vivo role of these 2 domains, we used gene targeting to create mice with point mutations in the ACE C-domain zinc-binding motif.  Such mice, termed ACE13/13, produce a full-length ACE protein with tissue expression identical to wild-type mice.  Anal. of ACE13/13 mice showed that they produce ACE having only N-domain catalytic activity, as detd. by the hydrolysis of domain specific substrates and by chloride sensitivity.  ACE13/13 mice have blood pressure and blood angiotensin II levels similar to wild-type mice.  However, plasma renin concn. is increased 2.6-fold and blood angiotensin I levels are increased 7.5-fold.  Bradykinin peptide levels are not different from wild-type levels.  ACE13/13 mice have a reduced increase of blood pressure after i.v. infusion of angiotensin I.  ACE13/13 mice have a normal renal structure, but they are not able to conc. urine after dehydration as effectively as wild-type mice.  This study shows that the C-domain of ACE is the predominant site of angiotensin I cleavage in vivo.  Although mice lacking C-domain activity have normal physiol. under lab. conditions, they respond less well to the stress of dehydration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPzpHqq5luNbVg90H21EOLACvtfcHk0lgc-hQtc3mYpQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt1OjtQ%253D%253D&md5=9cd89bdcb7fa8ca910583f4b952fcdf0</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1161%2FHYPERTENSIONAHA.107.097865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FHYPERTENSIONAHA.107.097865%26sid%3Dliteratum%253Aachs%26aulast%3DFuchs%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DH.%2BD.%26aulast%3DHubert%26aufirst%3DC.%26aulast%3DMichaud%26aufirst%3DA.%26aulast%3DCampbell%26aufirst%3DD.%2BJ.%26aulast%3DAdams%26aufirst%3DJ.%2BW.%26aulast%3DCapecchi%26aufirst%3DM.%2BR.%26aulast%3DCorvol%26aufirst%3DP.%26aulast%3DBernstein%26aufirst%3DK.%2BE.%26atitle%3DAngiotensin-converting%2520enzyme%2520C-terminal%2520catalytic%2520domain%2520is%2520the%2520main%2520site%2520of%2520angiotensin%2520I%2520cleavage%2520in%2520vivo%26jtitle%3DHypertension%26date%3D2008%26volume%3D51%26spage%3D267%26epage%3D274%26doi%3D10.1161%2FHYPERTENSIONAHA.107.097865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Wei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alhenc-Gelas, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clauser, E.</span><span> </span><span class="NLM_article-title">The two homologous domains of human angiotensin I-converting enzyme are both catalytically active</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">266</span><span class="NLM_x">, </span> <span class="NLM_fpage">9002</span><span class="NLM_x">â</span> <span class="NLM_lpage">9008</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1851160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaK3MXks1OntLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=266&publication_year=1991&pages=9002-9008&author=L.+Weiauthor=F.+Alhenc-Gelasauthor=P.+Corvolauthor=E.+Clauser&title=The+two+homologous+domains+of+human+angiotensin+I-converting+enzyme+are+both+catalytically+active"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The two homologous domains of human angiotensin I-converting enzyme are both catalytically active</span></div><div class="casAuthors">Wei, Lei; Alhenc-Gelas, Francois; Corvol, Pierre; Clauser, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">266</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">9002-8</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Mol. cloning of human endothelial angiotensin I-covering enzyme (kinase II; EC 3.4.15.1) (ACE) has recently shown that the enzyme contains 2 large homol. domains (called here the N and C domains), each bearing a putative active site, identified by sequence comparisons with the active sites of other zinc metallopeptidases.  However, the previous expts. with zinc or competitive ACE inhibitors suggested a single active site in ACE.  To establish whether both domains of ACE are enzymically active, a series of ACE mutants, each contg. only 1 intact domain, were constructed by deletion or point mutations of putative crit. residues of the other domain, and expressed in heterologous Chinese hamster ovary cells.  Both domains are enzymically active and cleave the C-terminal dipeptide of hippuryl-His-Leu or angiotensin I.  Moreover, both domains have an abs. zinc requirement for activity, are activated by chloride and are sensitive to competitive ACE inhibitors, and appear to function independently.  However, the 2 domains display different catalytic consts. and different patterns of chloride activation.  At high chloride concns., the C domain hydrolyzes the 2 substrates tested faster than does the N domain.  His-361,365 and His-959,963 are established as essential residues in the N and C domains, resp., most likely involved in zinc binding, and Glu-362 in the N domain and Glu-960 in the C domain are essential catalytic residues.  These observations provide strong evidence that ACE possesses 2 independent catalytic domains and suggest that they may have different functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquqPiPJev2abVg90H21EOLACvtfcHk0lgc-hQtc3mYpQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXks1OntLY%253D&md5=907f83aa628bc22968428d5d43d804ff</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DL.%26aulast%3DAlhenc-Gelas%26aufirst%3DF.%26aulast%3DCorvol%26aufirst%3DP.%26aulast%3DClauser%26aufirst%3DE.%26atitle%3DThe%2520two%2520homologous%2520domains%2520of%2520human%2520angiotensin%2520I-converting%2520enzyme%2520are%2520both%2520catalytically%2520active%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1991%26volume%3D266%26spage%3D9002%26epage%3D9008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Binevski, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sizova, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozdnev, V. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kost, O. A.</span><span> </span><span class="NLM_article-title">Evidence for the negative cooperativity of the two active sites within bovine somatic angiotensin-converting enzyme</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">550</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">â</span> <span class="NLM_lpage">88</span><span class="refDoi">Â DOI: 10.1016/S0014-5793(03)00825-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1016%2FS0014-5793%2803%2900825-1" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2003&pages=84-88&author=P.+V.+Binevskiauthor=E.+A.+Sizovaauthor=V.+F.+Pozdnevauthor=O.+A.+Kost&title=Evidence+for+the+negative+cooperativity+of+the+two+active+sites+within+bovine+somatic+angiotensin-converting+enzyme&doi=10.1016%2FS0014-5793%2803%2900825-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2803%2900825-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252803%252900825-1%26sid%3Dliteratum%253Aachs%26aulast%3DBinevski%26aufirst%3DP.%2BV.%26aulast%3DSizova%26aufirst%3DE.%2BA.%26aulast%3DPozdnev%26aufirst%3DV.%2BF.%26aulast%3DKost%26aufirst%3DO.%2BA.%26atitle%3DEvidence%2520for%2520the%2520negative%2520cooperativity%2520of%2520the%2520two%2520active%2520sites%2520within%2520bovine%2520somatic%2520angiotensin-converting%2520enzyme%26jtitle%3DFEBS%2520Lett.%26date%3D2003%26volume%3D550%26spage%3D84%26epage%3D88%26doi%3D10.1016%2FS0014-5793%2803%2900825-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Woodman, Z. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwager, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redelinghuys, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmona, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehlers, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">The N domain of somatic angiotensin-converting enzyme negatively regulates ectodomain shedding and catalytic activity</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">389</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">â</span> <span class="NLM_lpage">744</span><span class="refDoi">Â DOI: 10.1042/BJ20050187</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1042%2FBJ20050187" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2005&pages=739-744&author=Z.+L.+Woodmanauthor=S.+L.+Schwagerauthor=P.+Redelinghuysauthor=A.+K.+Carmonaauthor=M.+R.+Ehlersauthor=E.+D.+Sturrock&title=The+N+domain+of+somatic+angiotensin-converting+enzyme+negatively+regulates+ectodomain+shedding+and+catalytic+activity&doi=10.1042%2FBJ20050187"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2FBJ20050187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20050187%26sid%3Dliteratum%253Aachs%26aulast%3DWoodman%26aufirst%3DZ.%2BL.%26aulast%3DSchwager%26aufirst%3DS.%2BL.%26aulast%3DRedelinghuys%26aufirst%3DP.%26aulast%3DCarmona%26aufirst%3DA.%2BK.%26aulast%3DEhlers%26aufirst%3DM.%2BR.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DThe%2520N%2520domain%2520of%2520somatic%2520angiotensin-converting%2520enzyme%2520negatively%2520regulates%2520ectodomain%2520shedding%2520and%2520catalytic%2520activity%26jtitle%3DBiochem.%2520J.%26date%3D2005%26volume%3D389%26spage%3D739%26epage%3D744%26doi%3D10.1042%2FBJ20050187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Lenfant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wdzieczak-Bakala, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guittet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prome, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sotty, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frindel, E.</span><span> </span><span class="NLM_article-title">Inhibitor of hematopoietic pluripotent stem cell proliferation: purification and determination of its structure</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">â</span> <span class="NLM_lpage">782</span><span class="refDoi">Â DOI: 10.1073/pnas.86.3.779</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1073%2Fpnas.86.3.779" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=2915977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaL1MXhslWnsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=779-782&author=M.+Lenfantauthor=J.+Wdzieczak-Bakalaauthor=E.+Guittetauthor=J.+C.+Promeauthor=D.+Sottyauthor=E.+Frindel&title=Inhibitor+of+hematopoietic+pluripotent+stem+cell+proliferation%3A+purification+and+determination+of+its+structure&doi=10.1073%2Fpnas.86.3.779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor of hematopoietic pluripotent stem cell proliferation:  purification and determination of its structure</span></div><div class="casAuthors">Lenfant, Maryse; Wdzieczak-Bakala, Joanna; Guittet, Eric; Prome, Jean Claude; Sotty, Dominique; Frindel, Emilia</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">779-82</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">A 5-step purifn. procedure is reported here that led to the isolation from fetal calf bone marrow ext. of a tetrapeptide, Ac-Ser-Asp-Lys-Pro (mol. wt. 487), exerting a high inhibitory activity on the proliferation of hematopoietic pluripotent stem cells [defined here as spleen colony-forming units (CFU-S)].  The structure of this mol. was established from amino acid anal., fast atom bombardment mass spectrometry, and 1H NMR spectral data.  This structure was confirmed by comparison with the corresponding synthetic mol., which presents identical physicochem. characteristics and biol. properties.  Natural and synthetic peptides administered to mice (at a dose of 100 ng per mouse) after one injection of cytosine arabinonucleoside prevent CFU-S recruitment into DNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptPOPzmuaaarVg90H21EOLACvtfcHk0lhR87MJpoz85A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhslWnsbc%253D&md5=0b603b2b751d1233d1624e0e42e2f3a5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.86.3.779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.86.3.779%26sid%3Dliteratum%253Aachs%26aulast%3DLenfant%26aufirst%3DM.%26aulast%3DWdzieczak-Bakala%26aufirst%3DJ.%26aulast%3DGuittet%26aufirst%3DE.%26aulast%3DProme%26aufirst%3DJ.%2BC.%26aulast%3DSotty%26aufirst%3DD.%26aulast%3DFrindel%26aufirst%3DE.%26atitle%3DInhibitor%2520of%2520hematopoietic%2520pluripotent%2520stem%2520cell%2520proliferation%253A%2520purification%2520and%2520determination%2520of%2520its%2520structure%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1989%26volume%3D86%26spage%3D779%26epage%3D782%26doi%3D10.1073%2Fpnas.86.3.779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Robinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenfant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wdzieczak-Bakala, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melville, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riches, A.</span><span> </span><span class="NLM_article-title">The mechanism of action of the tetrapeptide acetyl-N-Ser-Asp-Lys-Pro (AcSDKP) in the control of haematopoietic stem cell proliferation</span> <span class="citation_source-journal">Cell Proliferation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">â</span> <span class="NLM_lpage">632</span><span class="refDoi">Â DOI: 10.1111/j.1365-2184.1992.tb01464.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1111%2Fj.1365-2184.1992.tb01464.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1457609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaK3sXktVWmu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1992&pages=623-632&author=S.+Robinsonauthor=M.+Lenfantauthor=J.+Wdzieczak-Bakalaauthor=J.+Melvilleauthor=A.+Riches&title=The+mechanism+of+action+of+the+tetrapeptide+acetyl-N-Ser-Asp-Lys-Pro+%28AcSDKP%29+in+the+control+of+haematopoietic+stem+cell+proliferation&doi=10.1111%2Fj.1365-2184.1992.tb01464.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The mechanism of action of the tetrapeptide acetyl-N-Ser-Asp-Lys-Pro (AcSDKP) in the control of hematopoietic stem cell proliferation</span></div><div class="casAuthors">Robinson, S.; Lenfant, M.; Wdzieczak-Babala, J.; Melville, J.; Riches, A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Proliferation</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">623-32</span>CODEN:
                <span class="NLM_cas:coden">CPROEM</span>;
        ISSN:<span class="NLM_cas:issn">0960-7722</span>.
    </div><div class="casAbstract">The mechanism of action of the hemoregulatory tetrapeptide acetyl-N-Ser-Asp-Lys-Pro (AcSDKP, Mr = 487 amu) was investigated using an in vitro assay of a murine high proliferative potential colony-forming cell (HPP-CFC) which responds to proliferation regulators of the hematopoietic stem cell population.  AcSDKP had no direct inhibitory effect on the no. or the proportion in S phase, of the committed granulocyte-macrophage progenitor (GM-CFC), or cycling HPP-CFC populations.  However, AcSDKP blocked the action of a stimulator of hematopoietic stem cell proliferation, preventing the switching of quiescent HPP-CFC into cell cycle.  It would appear that AcSDKP exerts its inhibitory hemoregulatory role indirectly, by preventing stimulator action on hematopoietic stem cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVaPl7Jppd97Vg90H21EOLACvtfcHk0lhR87MJpoz85A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktVWmu7s%253D&md5=3c53c736b3b8d9ba339351a335dd3b68</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2184.1992.tb01464.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2184.1992.tb01464.x%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DLenfant%26aufirst%3DM.%26aulast%3DWdzieczak-Bakala%26aufirst%3DJ.%26aulast%3DMelville%26aufirst%3DJ.%26aulast%3DRiches%26aufirst%3DA.%26atitle%3DThe%2520mechanism%2520of%2520action%2520of%2520the%2520tetrapeptide%2520acetyl-N-Ser-Asp-Lys-Pro%2520%2528AcSDKP%2529%2520in%2520the%2520control%2520of%2520haematopoietic%2520stem%2520cell%2520proliferation%26jtitle%3DCell%2520Proliferation%26date%3D1992%26volume%3D25%26spage%3D623%26epage%3D632%26doi%3D10.1111%2Fj.1365-2184.1992.tb01464.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Rousseau, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaud, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauvet, M.-T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenfant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span> </span><span class="NLM_article-title">The hemoregulatory peptide N-acetyl-Ser-Asp-Lys-Pro is a natural and specific substrate of the N-terminal active site of human angiotensin-converting enzyme</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">270</span><span class="NLM_x">, </span> <span class="NLM_fpage">3656</span><span class="NLM_x">â</span> <span class="NLM_lpage">3661</span><span class="refDoi">Â DOI: 10.1074/jbc.270.8.3656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1074%2Fjbc.270.8.3656" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=3656-3661&author=A.+Rousseauauthor=A.+Michaudauthor=M.-T.+Chauvetauthor=M.+Lenfantauthor=P.+Corvol&title=The+hemoregulatory+peptide+N-acetyl-Ser-Asp-Lys-Pro+is+a+natural+and+specific+substrate+of+the+N-terminal+active+site+of+human+angiotensin-converting+enzyme&doi=10.1074%2Fjbc.270.8.3656"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.270.8.3656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.270.8.3656%26sid%3Dliteratum%253Aachs%26aulast%3DRousseau%26aufirst%3DA.%26aulast%3DMichaud%26aufirst%3DA.%26aulast%3DChauvet%26aufirst%3DM.-T.%26aulast%3DLenfant%26aufirst%3DM.%26aulast%3DCorvol%26aufirst%3DP.%26atitle%3DThe%2520hemoregulatory%2520peptide%2520N-acetyl-Ser-Asp-Lys-Pro%2520is%2520a%2520natural%2520and%2520specific%2520substrate%2520of%2520the%2520N-terminal%2520active%2520site%2520of%2520human%2520angiotensin-converting%2520enzyme%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D3656%26epage%3D3661%26doi%3D10.1074%2Fjbc.270.8.3656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Natesh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwager, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, K. R.</span><span> </span><span class="NLM_article-title">Crystal structure of the human angiotensin-converting enzymeâlisinopril complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">421</span><span class="NLM_x">, </span> <span class="NLM_fpage">551</span><span class="NLM_x">â</span> <span class="NLM_lpage">554</span><span class="refDoi">Â DOI: 10.1038/nature01370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1038%2Fnature01370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=12540854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFSqtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2003&pages=551-554&author=R.+Nateshauthor=S.+L.+Schwagerauthor=E.+D.+Sturrockauthor=K.+R.+Acharya&title=Crystal+structure+of+the+human+angiotensin-converting+enzyme%E2%80%93lisinopril+complex&doi=10.1038%2Fnature01370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human angiotensin-converting enzyme-lisinopril complex</span></div><div class="casAuthors">Natesh, Ramanathan; Schwager, Sylva L. U.; Sturrock, Edward D.; Acharya, K. Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">6922</span>),
    <span class="NLM_cas:pages">551-554</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Angiotensin-converting enzyme (ACE) has a crit. role in cardiovascular function by cleaving the carboxy terminal His-Leu dipeptide from angiotensin I to produce a potent vasopressor octapeptide, angiotensin II.  Inhibitors of ACE are a first line of therapy for hypertension, heart failure, myocardial infarction and diabetic nephropathy.  Notably, these inhibitors were developed without knowledge of the structure of human ACE, but were instead designed on the basis of an assumed mechanistic homol. with carboxypeptidase A.  Here we present the x-ray structure of human testicular ACE and its complex with one of the most widely used inhibitors, lisinopril (N2-[(S)-1-carboxy-3-phenylpropyl]-l-lysyl-l-proline; also known as Prinivil or Zestril), at 2.0 Ã resoln.  Anal. of the three-dimensional structure of ACE shows that it bears little similarity to that of carboxypeptidase A, but instead resembles neurolysin and Pyrococcus furiosus carboxypeptidase-zinc metallopeptidases with no detectable sequence similarity to ACE.  The structure provides an opportunity to design domain-selective ACE inhibitors that may exhibit new pharmacol. profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxHYbWU8PmP7Vg90H21EOLACvtfcHk0lhR87MJpoz85A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFSqtg%253D%253D&md5=84f2f474ac20541b0352691128782b1d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature01370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01370%26sid%3Dliteratum%253Aachs%26aulast%3DNatesh%26aufirst%3DR.%26aulast%3DSchwager%26aufirst%3DS.%2BL.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520angiotensin-converting%2520enzyme%25E2%2580%2593lisinopril%2520complex%26jtitle%3DNature%26date%3D2003%26volume%3D421%26spage%3D551%26epage%3D554%26doi%3D10.1038%2Fnature01370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">KrÃ¶ger, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douglas, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâNeill, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dive, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">Investigating the domain specificity of phosphinic inhibitors RXPA380 and RXP407 in angiotensin-converting enzyme</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">8405</span><span class="NLM_x">â</span> <span class="NLM_lpage">8412</span><span class="refDoi">Â DOI: 10.1021/bi9011226</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9011226" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFyju7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8405-8412&author=W.+L.+Kr%C3%B6gerauthor=R.+G.+Douglasauthor=H.+G.+O%E2%80%99Neillauthor=V.+Diveauthor=E.+D.+Sturrock&title=Investigating+the+domain+specificity+of+phosphinic+inhibitors+RXPA380+and+RXP407+in+angiotensin-converting+enzyme&doi=10.1021%2Fbi9011226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Investigating the Domain Specificity of Phosphinic Inhibitors RXPA380 and RXP407 in Angiotensin-Converting Enzyme</span></div><div class="casAuthors">Kroger, Wendy L.; Douglas, Ross G.; O'Neill, Hester G.; Dive, Vincent; Sturrock, Edward D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">8405-8412</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human somatic angiotensin-converting enzyme (ACE) is a membrane-bound dipeptidyl carboxypeptidase that contains two extracellular domains (N and C).  Although highly homologous, they exhibit different substrate and inhibition profiles.  The phosphinic inhibitors RXPA380 and RXP407 are highly selective for the C- and N-domains, resp.  A no. of residues, implicated by structural data, are likely to contribute to this selectivity.  However, the extent to which these different interactions are responsible for domain selectivity is unclear.  In this study, a series of C- and N-domain mutants contg. conversions to corresponding domain residues were used to scrutinize the contribution of these residues to selective inhibitor binding.  Enzyme kinetic analyses of the purified mutants indicated that the RXPA380 C-selectivity is particularly reliant on the interaction between the P2 substituent and Phe 391 (testis ACE numbering).  Moreover, a C-domain mutant in which Phe 391 has been changed to a Tyr residue, in addn. to contg. an N-domain S2' pocket (S2'F/Y), displayed the greatest shift toward a more N-domain-like Ki.  None of the single mutations within the N-domain caused a large shift in RXP407's affinity for these enzymes.  However, the double mutant contg. the Tyr 369 to Phe change as well as Arg 381 to Glu displayed a 100-fold decrease in binding affinity, confirming that the S2 pocket plays a major role in RXP407 selectivity.  Taken together, these data advance our understanding regarding the mol. basis for the remarkable ACE domain selectivity exhibited by these inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdh4e6bOYw2LVg90H21EOLACvtfcHk0ljl55BTgD0fGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFyju7w%253D&md5=7a32a2e7ed0ba9199a43c5b79305fc53</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi9011226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9011226%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25B6ger%26aufirst%3DW.%2BL.%26aulast%3DDouglas%26aufirst%3DR.%2BG.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DH.%2BG.%26aulast%3DDive%26aufirst%3DV.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DInvestigating%2520the%2520domain%2520specificity%2520of%2520phosphinic%2520inhibitors%2520RXPA380%2520and%2520RXP407%2520in%2520angiotensin-converting%2520enzyme%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D8405%26epage%3D8412%26doi%3D10.1021%2Fbi9011226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Soubrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alhenc-Gelas, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allegrini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tregear, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span> </span><span class="NLM_article-title">Two putative active centers in human angiotensin I-converting enzyme revealed by molecular cloning</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">9386</span><span class="NLM_x">â</span> <span class="NLM_lpage">9390</span><span class="refDoi">Â DOI: 10.1073/pnas.85.24.9386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1073%2Fpnas.85.24.9386" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=1988&pages=9386-9390&author=F.+Soubrierauthor=F.+Alhenc-Gelasauthor=C.+Hubertauthor=J.+Allegriniauthor=M.+Johnauthor=G.+Tregearauthor=P.+Corvol&title=Two+putative+active+centers+in+human+angiotensin+I-converting+enzyme+revealed+by+molecular+cloning&doi=10.1073%2Fpnas.85.24.9386"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.85.24.9386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.85.24.9386%26sid%3Dliteratum%253Aachs%26aulast%3DSoubrier%26aufirst%3DF.%26aulast%3DAlhenc-Gelas%26aufirst%3DF.%26aulast%3DHubert%26aufirst%3DC.%26aulast%3DAllegrini%26aufirst%3DJ.%26aulast%3DJohn%26aufirst%3DM.%26aulast%3DTregear%26aufirst%3DG.%26aulast%3DCorvol%26aufirst%3DP.%26atitle%3DTwo%2520putative%2520active%2520centers%2520in%2520human%2520angiotensin%2520I-converting%2520enzyme%2520revealed%2520by%2520molecular%2520cloning%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1988%26volume%3D85%26spage%3D9386%26epage%3D9390%26doi%3D10.1073%2Fpnas.85.24.9386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Dive, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cotton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yiotakis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaud, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassiliou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiracek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vazeux, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauvet, M.-T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuniasse, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corvol, P.</span><span> </span><span class="NLM_article-title">RXP 407, a phosphinic peptide, is a potent inhibitor of angiotensin I converting enzyme able to differentiate between its two active sites</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">4330</span><span class="NLM_x">â</span> <span class="NLM_lpage">4335</span><span class="refDoi">Â DOI: 10.1073/pnas.96.8.4330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1073%2Fpnas.96.8.4330" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=4330-4335&author=V.+Diveauthor=J.+Cottonauthor=A.+Yiotakisauthor=A.+Michaudauthor=S.+Vassiliouauthor=J.+Jiracekauthor=G.+Vazeuxauthor=M.-T.+Chauvetauthor=P.+Cuniasseauthor=P.+Corvol&title=RXP+407%2C+a+phosphinic+peptide%2C+is+a+potent+inhibitor+of+angiotensin+I+converting+enzyme+able+to+differentiate+between+its+two+active+sites&doi=10.1073%2Fpnas.96.8.4330"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.8.4330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.8.4330%26sid%3Dliteratum%253Aachs%26aulast%3DDive%26aufirst%3DV.%26aulast%3DCotton%26aufirst%3DJ.%26aulast%3DYiotakis%26aufirst%3DA.%26aulast%3DMichaud%26aufirst%3DA.%26aulast%3DVassiliou%26aufirst%3DS.%26aulast%3DJiracek%26aufirst%3DJ.%26aulast%3DVazeux%26aufirst%3DG.%26aulast%3DChauvet%26aufirst%3DM.-T.%26aulast%3DCuniasse%26aufirst%3DP.%26aulast%3DCorvol%26aufirst%3DP.%26atitle%3DRXP%2520407%252C%2520a%2520phosphinic%2520peptide%252C%2520is%2520a%2520potent%2520inhibitor%2520of%2520angiotensin%2520I%2520converting%2520enzyme%2520able%2520to%2520differentiate%2520between%2520its%2520two%2520active%2520sites%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D4330%26epage%3D4335%26doi%3D10.1073%2Fpnas.96.8.4330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Anthony, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corradi, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwager, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redelinghuys, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiadis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dive, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">The N domain of human angiotensin-I-converting enzyme: the role of N domain glycosylation and the crystal structure in complex with an N domain-specific phosphinic inhibitor, RXP407</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">35685</span><span class="NLM_x">â</span> <span class="NLM_lpage">35693</span><span class="refDoi">Â DOI: 10.1074/jbc.M110.167866</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1074%2Fjbc.M110.167866" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=35685-35693&author=C.+S.+Anthonyauthor=H.+R.+Corradiauthor=S.+L.+Schwagerauthor=P.+Redelinghuysauthor=D.+Georgiadisauthor=V.+Diveauthor=K.+R.+Acharyaauthor=E.+D.+Sturrock&title=The+N+domain+of+human+angiotensin-I-converting+enzyme%3A+the+role+of+N+domain+glycosylation+and+the+crystal+structure+in+complex+with+an+N+domain-specific+phosphinic+inhibitor%2C+RXP407&doi=10.1074%2Fjbc.M110.167866"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.167866&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.167866%26sid%3Dliteratum%253Aachs%26aulast%3DAnthony%26aufirst%3DC.%2BS.%26aulast%3DCorradi%26aufirst%3DH.%2BR.%26aulast%3DSchwager%26aufirst%3DS.%2BL.%26aulast%3DRedelinghuys%26aufirst%3DP.%26aulast%3DGeorgiadis%26aufirst%3DD.%26aulast%3DDive%26aufirst%3DV.%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DThe%2520N%2520domain%2520of%2520human%2520angiotensin-I-converting%2520enzyme%253A%2520the%2520role%2520of%2520N%2520domain%2520glycosylation%2520and%2520the%2520crystal%2520structure%2520in%2520complex%2520with%2520an%2520N%2520domain-specific%2520phosphinic%2520inhibitor%252C%2520RXP407%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D35685%26epage%3D35693%26doi%3D10.1074%2Fjbc.M110.167866" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Greenlee, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allibone, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlow, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patchett, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulm, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassil, T. C.</span><span> </span><span class="NLM_article-title">Angiotensin-converting enzyme inhibitors: synthesis and biological activity of acyltripeptide analogs of enalapril</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">434</span><span class="NLM_x">â</span> <span class="NLM_lpage">442</span><span class="refDoi">Â DOI: 10.1021/jm00382a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00382a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=434-442&author=W.+J.+Greenleeauthor=P.+L.+Alliboneauthor=D.+S.+Perlowauthor=A.+A.+Patchettauthor=E.+H.+Ulmauthor=T.+C.+Vassil&title=Angiotensin-converting+enzyme+inhibitors%3A+synthesis+and+biological+activity+of+acyltripeptide+analogs+of+enalapril&doi=10.1021%2Fjm00382a008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm00382a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00382a008%26sid%3Dliteratum%253Aachs%26aulast%3DGreenlee%26aufirst%3DW.%2BJ.%26aulast%3DAllibone%26aufirst%3DP.%2BL.%26aulast%3DPerlow%26aufirst%3DD.%2BS.%26aulast%3DPatchett%26aufirst%3DA.%2BA.%26aulast%3DUlm%26aufirst%3DE.%2BH.%26aulast%3DVassil%26aufirst%3DT.%2BC.%26atitle%3DAngiotensin-converting%2520enzyme%2520inhibitors%253A%2520synthesis%2520and%2520biological%2520activity%2520of%2520acyltripeptide%2520analogs%2520of%2520enalapril%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D434%26epage%3D442%26doi%3D10.1021%2Fjm00382a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Stanton, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watthey, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babiarz, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomaselli, H. C.</span><span> </span><span class="NLM_article-title">Angiotensin converting enzyme inhibitors: structure-activity profile of 1-benzazepin-2-one derivatives</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1603</span><span class="NLM_x">â</span> <span class="NLM_lpage">1606</span><span class="refDoi">Â DOI: 10.1021/jm00149a010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00149a010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=1603-1606&author=J.+L.+Stantonauthor=J.+W.+Wattheyauthor=M.+N.+Desaiauthor=B.+M.+Finnauthor=J.+E.+Babiarzauthor=H.+C.+Tomaselli&title=Angiotensin+converting+enzyme+inhibitors%3A+structure-activity+profile+of+1-benzazepin-2-one+derivatives&doi=10.1021%2Fjm00149a010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm00149a010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00149a010%26sid%3Dliteratum%253Aachs%26aulast%3DStanton%26aufirst%3DJ.%2BL.%26aulast%3DWatthey%26aufirst%3DJ.%2BW.%26aulast%3DDesai%26aufirst%3DM.%2BN.%26aulast%3DFinn%26aufirst%3DB.%2BM.%26aulast%3DBabiarz%26aufirst%3DJ.%2BE.%26aulast%3DTomaselli%26aufirst%3DH.%2BC.%26atitle%3DAngiotensin%2520converting%2520enzyme%2520inhibitors%253A%2520structure-activity%2520profile%2520of%25201-benzazepin-2-one%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D1603%26epage%3D1606%26doi%3D10.1021%2Fjm00149a010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Ksander, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erion, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diefenbacher, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Chehabi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levens, N.</span><span> </span><span class="NLM_article-title">Dual angiotensin converting enzyme/thromboxane synthase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">1823</span><span class="NLM_x">â</span> <span class="NLM_lpage">1832</span><span class="refDoi">Â DOI: 10.1021/jm00038a011</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00038a011" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=1823-1832&author=G.+M.+Ksanderauthor=M.+Erionauthor=A.+M.+Yuanauthor=C.+G.+Diefenbacherauthor=L.+El-Chehabiauthor=D.+Coteauthor=N.+Levens&title=Dual+angiotensin+converting+enzyme%2Fthromboxane+synthase+inhibitors&doi=10.1021%2Fjm00038a011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm00038a011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00038a011%26sid%3Dliteratum%253Aachs%26aulast%3DKsander%26aufirst%3DG.%2BM.%26aulast%3DErion%26aufirst%3DM.%26aulast%3DYuan%26aufirst%3DA.%2BM.%26aulast%3DDiefenbacher%26aufirst%3DC.%2BG.%26aulast%3DEl-Chehabi%26aufirst%3DL.%26aulast%3DCote%26aufirst%3DD.%26aulast%3DLevens%26aufirst%3DN.%26atitle%3DDual%2520angiotensin%2520converting%2520enzyme%252Fthromboxane%2520synthase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D1823%26epage%3D1832%26doi%3D10.1021%2Fjm00038a011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Gruenfeld, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebetino, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browne, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gude, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huebner, C. F.</span><span> </span><span class="NLM_article-title">Angiotensin converting enzyme inhibitors: 1-glutarylindoline-2-carboxylic acid derivatives</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1277</span><span class="NLM_x">â</span> <span class="NLM_lpage">1282</span><span class="refDoi">Â DOI: 10.1021/jm00363a012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00363a012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1983&pages=1277-1282&author=N.+Gruenfeldauthor=J.+L.+Stantonauthor=A.+M.+Yuanauthor=F.+H.+Ebetinoauthor=L.+J.+Browneauthor=C.+Gudeauthor=C.+F.+Huebner&title=Angiotensin+converting+enzyme+inhibitors%3A+1-glutarylindoline-2-carboxylic+acid+derivatives&doi=10.1021%2Fjm00363a012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm00363a012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00363a012%26sid%3Dliteratum%253Aachs%26aulast%3DGruenfeld%26aufirst%3DN.%26aulast%3DStanton%26aufirst%3DJ.%2BL.%26aulast%3DYuan%26aufirst%3DA.%2BM.%26aulast%3DEbetino%26aufirst%3DF.%2BH.%26aulast%3DBrowne%26aufirst%3DL.%2BJ.%26aulast%3DGude%26aufirst%3DC.%26aulast%3DHuebner%26aufirst%3DC.%2BF.%26atitle%3DAngiotensin%2520converting%2520enzyme%2520inhibitors%253A%25201-glutarylindoline-2-carboxylic%2520acid%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1983%26volume%3D26%26spage%3D1277%26epage%3D1282%26doi%3D10.1021%2Fjm00363a012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Ksander, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diefenbacher, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, J. L.</span><span> </span><span class="NLM_article-title">Angiotensin converting enzyme inhibitors: N-substituted D-glutamic acid. gamma.-dipeptides</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1606</span><span class="NLM_x">â</span> <span class="NLM_lpage">1611</span><span class="refDoi">Â DOI: 10.1021/jm00149a011</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00149a011" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=1606-1611&author=G.+M.+Ksanderauthor=A.+M.+Yuanauthor=C.+G.+Diefenbacherauthor=J.+L.+Stanton&title=Angiotensin+converting+enzyme+inhibitors%3A+N-substituted+D-glutamic+acid.+gamma.-dipeptides&doi=10.1021%2Fjm00149a011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm00149a011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00149a011%26sid%3Dliteratum%253Aachs%26aulast%3DKsander%26aufirst%3DG.%2BM.%26aulast%3DYuan%26aufirst%3DA.%2BM.%26aulast%3DDiefenbacher%26aufirst%3DC.%2BG.%26aulast%3DStanton%26aufirst%3DJ.%2BL.%26atitle%3DAngiotensin%2520converting%2520enzyme%2520inhibitors%253A%2520N-substituted%2520D-glutamic%2520acid.%2520gamma.-dipeptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D1606%26epage%3D1611%26doi%3D10.1021%2Fjm00149a011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Aoyagi, Y.</span><span> </span><span class="NLM_article-title">An angiotensin-I converting enzyme inhibitor from buckwheat (Fagopyrum esculentum Moench) flour</span> <span class="citation_source-journal">Phytochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">618</span><span class="NLM_x">â</span> <span class="NLM_lpage">621</span><span class="refDoi">Â DOI: 10.1016/j.phytochem.2005.12.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1016%2Fj.phytochem.2005.12.022" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=618-621&author=Y.+Aoyagi&title=An+angiotensin-I+converting+enzyme+inhibitor+from+buckwheat+%28Fagopyrum+esculentum+Moench%29+flour&doi=10.1016%2Fj.phytochem.2005.12.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.phytochem.2005.12.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytochem.2005.12.022%26sid%3Dliteratum%253Aachs%26aulast%3DAoyagi%26aufirst%3DY.%26atitle%3DAn%2520angiotensin-I%2520converting%2520enzyme%2520inhibitor%2520from%2520buckwheat%2520%2528Fagopyrum%2520esculentum%2520Moench%2529%2520flour%26jtitle%3DPhytochemistry%26date%3D2006%26volume%3D67%26spage%3D618%26epage%3D621%26doi%3D10.1016%2Fj.phytochem.2005.12.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit17"><span> <span class="citation_source-book">GOSTAR Drug Database and Clinical Candidate Database</span>; <span class="NLM_publisher-name">Excelra Knowledge Solutions</span>: <span class="NLM_publisher-loc">Hyderabad, India</span>, Jul 22,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span><a href="https://www.gostardb.com/index.jsp" class="extLink">https://www.gostardb.com/index.jsp</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GOSTAR+Drug+Database+and+Clinical+Candidate+Database%3B+Excelra+Knowledge+Solutions%3A+Hyderabad%2C+India%2C+Jul+22%2C+2013%3B+https%3A%2F%2Fwww.gostardb.com%2Findex.jsp."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGOSTAR%2520Drug%2520Database%2520and%2520Clinical%2520Candidate%2520Database%26pub%3DExcelra%2520Knowledge%2520Solutions%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">â</span> <span class="NLM_lpage">1749</span><span class="refDoi">Â DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Ã in nearly half of the cases and are greater than 2 Ã in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lgSvwTnfDoJ4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelver, W. H.</span><span> </span><span class="NLM_article-title">Docking studies of matrix metalloproteinase inhibitors: zinc parameter optimization to improve the binding free energy prediction</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">â</span> <span class="NLM_lpage">126</span><span class="refDoi">Â DOI: 10.1016/S1093-3263(03)00153-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1016%2FS1093-3263%2803%2900153-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=12932782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVeisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=115-126&author=X.+Huauthor=W.+H.+Shelver&title=Docking+studies+of+matrix+metalloproteinase+inhibitors%3A+zinc+parameter+optimization+to+improve+the+binding+free+energy+prediction&doi=10.1016%2FS1093-3263%2803%2900153-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Docking studies of matrix metalloproteinase inhibitors: zinc parameter optimization to improve the binding free energy prediction</span></div><div class="casAuthors">Hu, Xin; Shelver, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-126</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Docking of metalloproteinase inhibitors remains a challenge due to the zinc multiple coordination geometries and the lack of appropriate force field parameters to model the metal/ligand interactions.  In this study, we explore the docking accuracy and scoring reliability for the docking of matrix metalloproteinase (MMP) inhibitors using AutoDock 3.0.  Potential problems assocd. with zinc ion were investigated by docking 16 matrix metalloproteinase ligands to their crystal structures.  A good coordination between the zinc binding group (ZBG) and the zinc was shown to be a prerequisite for the ligand to fit the binding site.  A simplex optimization of zinc parameters, including zinc radius, well depth, and zinc charges, was performed utilizing the 14 MMP complexes with good docking.  The use of optimized zinc parameters (zinc radius: 0.87 A; well depth: 0.35 kcal/mol; and zinc charges: +0.95 e) shows improvement in both docking accuracy at the zinc binding site and the prediction of binding free energies.  Although further improvement in the docking procedure, particularly the scoring function is needed, optimization of zinc parameters provides an efficient way to improve the performance of AutoDock as a drug discovery tool.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPGJeKzUXd9LVg90H21EOLACvtfcHk0lgSvwTnfDoJ4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVeisb8%253D&md5=0f33250fe5f7e84bf3a20c1643e383b5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2803%2900153-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252803%252900153-0%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DShelver%26aufirst%3DW.%2BH.%26atitle%3DDocking%2520studies%2520of%2520matrix%2520metalloproteinase%2520inhibitors%253A%2520zinc%2520parameter%2520optimization%2520to%2520improve%2520the%2520binding%2520free%2520energy%2520prediction%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2003%26volume%3D22%26spage%3D115%26epage%3D126%26doi%3D10.1016%2FS1093-3263%2803%2900153-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Douglas, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuyer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubbe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chibale, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">Fragment-based design for the development of N-domain-selective angiotensin-1-converting enzyme inhibitors</span> <span class="citation_source-journal">Clin. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">â</span> <span class="NLM_lpage">313</span><span class="refDoi">Â DOI: 10.1042/CS20130403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1042%2FCS20130403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=24015848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CktL3M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2014&pages=305-313&author=R.+G.+Douglasauthor=R.+K.+Sharmaauthor=G.+Masuyerauthor=L.+Lubbeauthor=I.+Zamoraauthor=K.+R.+Acharyaauthor=K.+Chibaleauthor=E.+D.+Sturrock&title=Fragment-based+design+for+the+development+of+N-domain-selective+angiotensin-1-converting+enzyme+inhibitors&doi=10.1042%2FCS20130403"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based design for the development of N-domain-selective angiotensin-1-converting enzyme inhibitors</span></div><div class="casAuthors">Douglas, Ross G.; Sharma, Rajni K.; Masuyer, Geoffrey; Lubbe, Lizelle; Zamora, Ismael; Acharya, K. Ravi; Chibale, Kelly; Sturrock, Edward D.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-313</span>CODEN:
                <span class="NLM_cas:coden">CSCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1470-8736</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">ACE (angiotensin-1-converting enzyme) is a zinc metallopeptidase that plays a prominent role in blood pressure regulation and electrolyte homeostasis.  ACE consists of two homologous domains that despite similarities of sequence and topol. display differences in substrate processing and inhibitor binding.  The design of inhibitors that selectively inhibit the N-domain (N-selective) could be useful in treating conditions of tissue injury and fibrosis due to build-up of N-domain-specific substrate Ac-SDKP (N-acetyl-Ser-Asp-Lys-Pro).  Using a receptor-based SHOP (scaffold hopping) approach with N-selective inhibitor RXP407, a shortlist of scaffolds that consisted of modified RXP407 backbones with novel chemotypes was generated.  These scaffolds were selected on the basis of enhanced predicted interaction energies with N-domain residues that differed from their C-domain counterparts.  One scaffold was synthesized and inhibitory binding tested using a fluorogenic ACE assay.  A mol. incorporating a tetrazole moiety in the P2 position (compd. 33RE) displayed potent inhibition (Ki=11.21Â±0.74 nM) and was 927-fold more selective for the N-domain than the C-domain.  A crystal structure of compd. 33RE in complex with the N-domain revealed its mode of binding through arom. stacking with His388 and a direct hydrogen bond with the hydroxy group of the N-domain specific Tyr369.  This work further elucidates the mol. basis for N-domain selective inhibition and assists in the design of novel N-selective ACE inhibitors that could be employed in treatment of fibrosis disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra1vuAtVm_07Vg90H21EOLACvtfcHk0lgSvwTnfDoJ4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CktL3M&md5=b74b9a12946c6235f20d766eddea846f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1042%2FCS20130403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FCS20130403%26sid%3Dliteratum%253Aachs%26aulast%3DDouglas%26aufirst%3DR.%2BG.%26aulast%3DSharma%26aufirst%3DR.%2BK.%26aulast%3DMasuyer%26aufirst%3DG.%26aulast%3DLubbe%26aufirst%3DL.%26aulast%3DZamora%26aufirst%3DI.%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26aulast%3DChibale%26aufirst%3DK.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DFragment-based%2520design%2520for%2520the%2520development%2520of%2520N-domain-selective%2520angiotensin-1-converting%2520enzyme%2520inhibitors%26jtitle%3DClin.%2520Sci.%26date%3D2014%26volume%3D126%26spage%3D305%26epage%3D313%26doi%3D10.1042%2FCS20130403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Lubbe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sewell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturrock, E. D.</span><span> </span><span class="NLM_article-title">The influence of angiotensin converting enzyme mutations on the kinetics and dynamics of N-domain selective inhibition</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">3941</span><span class="NLM_x">â</span> <span class="NLM_lpage">3961</span><span class="refDoi">Â DOI: 10.1111/febs.13900</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1111%2Ffebs.13900" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2016&pages=3941-3961&author=L.+Lubbeauthor=B.+T.+Sewellauthor=E.+D.+Sturrock&title=The+influence+of+angiotensin+converting+enzyme+mutations+on+the+kinetics+and+dynamics+of+N-domain+selective+inhibition&doi=10.1111%2Ffebs.13900"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1111%2Ffebs.13900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.13900%26sid%3Dliteratum%253Aachs%26aulast%3DLubbe%26aufirst%3DL.%26aulast%3DSewell%26aufirst%3DB.%2BT.%26aulast%3DSturrock%26aufirst%3DE.%2BD.%26atitle%3DThe%2520influence%2520of%2520angiotensin%2520converting%2520enzyme%2520mutations%2520on%2520the%2520kinetics%2520and%2520dynamics%2520of%2520N-domain%2520selective%2520inhibition%26jtitle%3DFEBS%2520J.%26date%3D2016%26volume%3D283%26spage%3D3941%26epage%3D3961%26doi%3D10.1111%2Ffebs.13900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Waterman, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gildea, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkhurst, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewster, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, G.</span><span> </span><span class="NLM_article-title">Diffraction-geometry refinement in the DIALS framework</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">558</span><span class="NLM_x">â</span> <span class="NLM_lpage">575</span><span class="refDoi">Â DOI: 10.1107/S2059798316002187</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS2059798316002187" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFSkurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=558-575&author=D.+G.+Watermanauthor=G.+Winterauthor=R.+J.+Gildeaauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=N.+K.+Sauterauthor=G.+Evans&title=Diffraction-geometry+refinement+in+the+DIALS+framework&doi=10.1107%2FS2059798316002187"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Diffraction-geometry refinement in the DIALS framework</span></div><div class="casAuthors">Waterman, David G.; Winter, Graeme; Gildea, Richard J.; Parkhurst, James M.; Brewster, Aaron S.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">558-575</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Rapid data collection and modern computing resources provide the opportunity to revisit the task of optimizing the model of diffraction geometry prior to integration.  A comprehensive description is given of new software that builds upon established methods by performing a single global refinement procedure, utilizing a smoothly varying model of the crystal lattice where appropriate.  This global refinement technique extends to multiple data sets, providing useful constraints to handle the problem of correlated parameters, particularly for small wedges of data.  Examples of advanced uses of the software are given and the design is explained in detail, with particular emphasis on the flexibility and extensibility it entails.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowmCZHkbbl6bVg90H21EOLACvtfcHk0lgxZJ82y6ye0w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFSkurs%253D&md5=0041260941cd2e0581c8a1db0b5add97</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS2059798316002187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316002187%26sid%3Dliteratum%253Aachs%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDiffraction-geometry%2520refinement%2520in%2520the%2520DIALS%2520framework%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2016%26volume%3D72%26spage%3D558%26epage%3D575%26doi%3D10.1107%2FS2059798316002187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">â</span> <span class="NLM_lpage">1214</span><span class="refDoi">Â DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgxZJ82y6ye0w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Collaborative Computational Project 4</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760&author=Collaborative+Computational+Project+4&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">â</span> <span class="NLM_lpage">674</span><span class="refDoi">Â DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ÎF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgxZJ82y6ye0w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">â</span> <span class="NLM_lpage">255</span><span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0liOFEomgV56ag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">BunkÃ³czi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">â</span> <span class="NLM_lpage">221</span><span class="refDoi">Â DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liOFEomgV56ag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">â</span> <span class="NLM_lpage">2132</span><span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhborFXP4sgdQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">â</span> <span class="NLM_lpage">21</span><span class="refDoi">Â DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lhborFXP4sgdQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">McNicholas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noble, M.</span><span> </span><span class="NLM_article-title">Presenting your structures: the CCP4mg molecular-graphics software</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">386</span><span class="NLM_x">â</span> <span class="NLM_lpage">394</span><span class="refDoi">Â DOI: 10.1107/S0907444911007281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=10.1107%2FS0907444911007281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=21460457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=386-394&author=S.+McNicholasauthor=E.+Pottertonauthor=K.+Wilsonauthor=M.+Noble&title=Presenting+your+structures%3A+the+CCP4mg+molecular-graphics+software&doi=10.1107%2FS0907444911007281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Presenting your structures: The CCP4mg molecular-graphics software</span></div><div class="casAuthors">McNicholas, S.; Potterton, E.; Wilson, K. S.; Noble, M. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">386-394</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">CCP4mg is a mol.-graphics program that is designed to give rapid access to both straightforward and complex static and dynamic representations of macromol. structures.  It has recently been updated with a new interface that provides more sophisticated atom-selection options and a wizard to facilitate the generation of complex scenes.  These scenes may contain a mixt. of coordinate-derived and abstr. graphical objects, including text objects, arbitrary vectors, geometric objects and imported images, which can enhance a picture and eliminate the need for subsequent editing.  Scene descriptions can be saved to file and transferred to other mols.  Here, the substantially enhanced version 2 of the program, with a new underlying GUI toolkit, is described.  A built-in rendering module produces publication-quality images.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdtflrxrILlbVg90H21EOLACvtfcHk0lhborFXP4sgdQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqurw%253D&md5=d96403a39bc723a66f11a730f81f76c3</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007281%26sid%3Dliteratum%253Aachs%26aulast%3DMcNicholas%26aufirst%3DS.%26aulast%3DPotterton%26aufirst%3DE.%26aulast%3DWilson%26aufirst%3DK.%26aulast%3DNoble%26aufirst%3DM.%26atitle%3DPresenting%2520your%2520structures%253A%2520the%2520CCP4mg%2520molecular-graphics%2520software%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D386%26epage%3D394%26doi%3D10.1107%2FS0907444911007281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1O86" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1O86','PDB','1O86'); return false;">PDB: 1O86</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXQ','PDB','3NXQ'); return false;">PDB: 3NXQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AM8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AM8','PDB','5AM8'); return false;">PDB: 5AM8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EN5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EN5','PDB','6EN5'); return false;">PDB: 6EN5</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i36"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01478">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_17455"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01478">10.1021/acs.jmedchem.7b01478</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Crystallographic information, Ligplot 2D interaction maps of docked compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_001.csv">jm7b01478_si_001.csv (2.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01478/suppl_file/jm7b01478_si_002.pdf">jm7b01478_si_002.pdf (1.83 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB code for N-domain of ACE bound to <b>16</b> is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EN5">6EN5</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01478&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01478%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-1%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01478" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a9bedcd4e22d8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
