
<html lang="en"     class="pb-page"  data-request-id="752532a8-0b65-492b-8153-c6d0f00210b4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b01961;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-10"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Synthesis and Structure–Activity Relationships of Arylsulfonamides as AIMP2-DX2 Inhibitors for the Development of a Novel Anticancer Therapy" /></meta><meta name="dc.Creator" content="Aneesh  Sivaraman" /></meta><meta name="dc.Creator" content="Dae Gyu  Kim" /></meta><meta name="dc.Creator" content="Deepak  Bhattarai" /></meta><meta name="dc.Creator" content="Minkyoung  Kim" /></meta><meta name="dc.Creator" content="Hwa Young  Lee" /></meta><meta name="dc.Creator" content="Semi  Lim" /></meta><meta name="dc.Creator" content="Jiwon  Kong" /></meta><meta name="dc.Creator" content="Ja-il  Goo" /></meta><meta name="dc.Creator" content="Seunghwan  Shim" /></meta><meta name="dc.Creator" content="Seungbeom  Lee" /></meta><meta name="dc.Creator" content="Young-Ger  Suh" /></meta><meta name="dc.Creator" content="Yongseok  Choi" /></meta><meta name="dc.Creator" content="Sunghoon  Kim" /></meta><meta name="dc.Creator" content="Kyeong  Lee" /></meta><meta name="dc.Description" content="AIMP2-DX2, a splicing variant of AIMP2, is up-regulated in lung cancer, possesses oncogenic activity, and results in tumorigenesis. Specifically inhibiting the interaction between AIMP2-DX2 and HSP..." /></meta><meta name="Description" content="AIMP2-DX2, a splicing variant of AIMP2, is up-regulated in lung cancer, possesses oncogenic activity, and results in tumorigenesis. Specifically inhibiting the interaction between AIMP2-DX2 and HSP..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 21, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01961" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01961" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01961" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01961" /></link>
        
    
    

<title>Synthesis and Structure–Activity Relationships of Arylsulfonamides as AIMP2-DX2 Inhibitors for the Development of a Novel Anticancer Therapy | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01961" /></meta><meta property="og:title" content="Synthesis and Structure–Activity Relationships of Arylsulfonamides as AIMP2-DX2 Inhibitors for the Development of a Novel Anticancer Therapy" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0021.jpeg" /></meta><meta property="og:description" content="AIMP2-DX2, a splicing variant of AIMP2, is up-regulated in lung cancer, possesses oncogenic activity, and results in tumorigenesis. Specifically inhibiting the interaction between AIMP2-DX2 and HSP70 to suppress AIMP2-DX2-dependent cancers with small molecules is considered a promising avenue for cancer therapeutics. Optimization of hit BC-DXI-04 (IC50 = 40.1 μM) provided new potent sulfonamide based AIMP2-DX2 inhibitors. Among these, BC-DXI-843 showed improved inhibition against AIMP2-DX2 (IC50 = 0.92 μM) with more than 100-fold selectivity over AIMP2 in a luciferase assay. Several binding assays indicated that this compound effectively induces cancer cell apoptosis by specifically interrupting the interaction between DX2 and HSP70, which leads to the degradation of DX2 via Siah1-mediated ubiquitination. More importantly, BC-DXI-843 demonstrated in vivo efficacy in a tumor xenograft mouse model (H460 cells) at a dosage of 50 mg/kg, suggesting it as a promising lead for development of novel therapeutics targeting AIMP2-DX2 in lung cancer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01961"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01961">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01961&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01961&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01961&amp;href=/doi/10.1021/acs.jmedchem.9b01961" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 5139-5158</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01924" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b02094" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Synthesis and Structure–Activity Relationships of Arylsulfonamides as AIMP2-DX2 Inhibitors for the Development of a Novel Anticancer Therapy</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Aneesh Sivaraman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aneesh Sivaraman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aneesh++Sivaraman">Aneesh Sivaraman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dae Gyu Kim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dae Gyu Kim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Bioconvergence Research Center, College of Pharmacy & College of Medicine, Gangnam Severance Hospital, Yonsei University, Incheon 21983, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dae+Gyu++Kim">Dae Gyu Kim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Deepak Bhattarai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Deepak Bhattarai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Deepak++Bhattarai">Deepak Bhattarai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Minkyoung Kim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Minkyoung Kim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Minkyoung++Kim">Minkyoung Kim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hwa Young Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hwa Young Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hwa+Young++Lee">Hwa Young Lee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Semi Lim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Semi Lim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Bioconvergence Research Center, College of Pharmacy & College of Medicine, Gangnam Severance Hospital, Yonsei University, Incheon 21983, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Semi++Lim">Semi Lim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiwon Kong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiwon Kong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Bioconvergence Research Center, College of Pharmacy & College of Medicine, Gangnam Severance Hospital, Yonsei University, Incheon 21983, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiwon++Kong">Jiwon Kong</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3990-5619" title="Orcid link">http://orcid.org/0000-0003-3990-5619</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ja-il Goo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ja-il Goo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biotechnology, Korea University, Seoul 02841, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ja-il++Goo">Ja-il Goo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Seunghwan Shim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Seunghwan Shim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biotechnology, Korea University, Seoul 02841, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Seunghwan++Shim">Seunghwan Shim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Seungbeom Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Seungbeom Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, CHA University, Gyeonggi-do 11160, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Seungbeom++Lee">Seungbeom Lee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Young-Ger Suh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Young-Ger Suh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, CHA University, Gyeonggi-do 11160, Republic of Korea</div><div class="loa-info-affiliations-info">College of Pharmacy, Seoul National University, Seoul 08826, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Young-Ger++Suh">Young-Ger Suh</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1799-8607" title="Orcid link">http://orcid.org/0000-0003-1799-8607</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yongseok Choi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yongseok Choi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biotechnology, Korea University, Seoul 02841, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yongseok++Choi">Yongseok Choi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sunghoon Kim</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sunghoon Kim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Bioconvergence Research Center, College of Pharmacy & College of Medicine, Gangnam Severance Hospital, Yonsei University, Incheon 21983, Republic of Korea</div></div><span class="conrtib-corresp"><strong>*</strong>S.K..: email, <a href="/cdn-cgi/l/email-protection#3f4c4a51585456527f4c514a115e5c11544d"><span class="__cf_email__" data-cfemail="3e4d4b50595557537e4d504b105f5d10554c">[email protected]</span></a>; phone, +82-2-880-8180.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sunghoon++Kim">Sunghoon Kim</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1570-3230" title="Orcid link">http://orcid.org/0000-0002-1570-3230</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Kyeong Lee</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kyeong Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</div></div><span class="conrtib-corresp"><strong>*</strong>K.L.: email, <a href="/cdn-cgi/l/email-protection#8be0eaf2e7eeeecbefe4e5ececfee0a5eeeffe"><span class="__cf_email__" data-cfemail="600b01190c050520040f0e0707150b4e050415">[email protected]</span></a>; phone, +82-31-961-5214.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kyeong++Lee">Kyeong Lee</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5455-9956" title="Orcid link">http://orcid.org/0000-0002-5455-9956</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01961&amp;href=/doi/10.1021%2Facs.jmedchem.9b01961" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 5139–5158</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 21, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 December 2019</li><li><span class="item_label"><b>Published</b> online</span>21 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01961" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01961</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5139%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAneesh%2BSivaraman%252C%2BDae%2BGyu%2BKim%252C%2BDeepak%2BBhattarai%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D10%26contentID%3Dacs.jmedchem.9b01961%26title%3DSynthesis%2Band%2BStructure%25E2%2580%2593Activity%2BRelationships%2Bof%2BArylsulfonamides%2Bas%2BAIMP2-DX2%2BInhibitors%2Bfor%2Bthe%2BDevelopment%2Bof%2Ba%2BNovel%2BAnticancer%2BTherapy%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5158%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01961"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1507</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01961" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Synthesis and Structure–Activity Relationships of Arylsulfonamides as AIMP2-DX2 Inhibitors for the Development of a Novel Anticancer Therapy&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Aneesh&quot;,&quot;last_name&quot;:&quot;Sivaraman&quot;},{&quot;first_name&quot;:&quot;Dae&quot;,&quot;last_name&quot;:&quot;Gyu Kim&quot;},{&quot;first_name&quot;:&quot;Deepak&quot;,&quot;last_name&quot;:&quot;Bhattarai&quot;},{&quot;first_name&quot;:&quot;Minkyoung&quot;,&quot;last_name&quot;:&quot;Kim&quot;},{&quot;first_name&quot;:&quot;Hwa&quot;,&quot;last_name&quot;:&quot;Young Lee&quot;},{&quot;first_name&quot;:&quot;Semi&quot;,&quot;last_name&quot;:&quot;Lim&quot;},{&quot;first_name&quot;:&quot;Jiwon&quot;,&quot;last_name&quot;:&quot;Kong&quot;},{&quot;first_name&quot;:&quot;Ja-il&quot;,&quot;last_name&quot;:&quot;Goo&quot;},{&quot;first_name&quot;:&quot;Seunghwan&quot;,&quot;last_name&quot;:&quot;Shim&quot;},{&quot;first_name&quot;:&quot;Seungbeom&quot;,&quot;last_name&quot;:&quot;Lee&quot;},{&quot;first_name&quot;:&quot;Young-Ger&quot;,&quot;last_name&quot;:&quot;Suh&quot;},{&quot;first_name&quot;:&quot;Yongseok&quot;,&quot;last_name&quot;:&quot;Choi&quot;},{&quot;first_name&quot;:&quot;Sunghoon&quot;,&quot;last_name&quot;:&quot;Kim&quot;},{&quot;first_name&quot;:&quot;Kyeong&quot;,&quot;last_name&quot;:&quot;Lee&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;5139-5158&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01961&quot;},&quot;abstract&quot;:&quot;AIMP2-DX2, a splicing variant of AIMP2, is up-regulated in lung cancer, possesses oncogenic activity, and results in tumorigenesis. Specifically inhibiting the interaction between AIMP2-DX2 and HSP70 to suppress AIMP2-DX2-dependent cancers with small molecules is considered a promising avenue for cancer therapeutics. Optimization of hit BC-DXI-04 (IC50 = 40.1 μM) provided new potent sulfonamide based AIMP2-DX2 inhibitors. Among these, BC-DXI-843 showed improved inhibition against AIMP2-DX2 (IC50 = 0.92 μM) with more than 100-fold selectivity over AIMP2 in a luciferase assay. Several binding assays indicated that this compound effectively induces cancer cell apoptosis by specifically interrupting the interaction between DX2 and HSP70, which leads to the degradation of DX2 via Siah1-mediated ubiquitination. More importantly, BC-DXI-843 demonstrated in vivo efficacy in a tumor xenograft mouse model (H460 cells) at a dosage of 50 mg/kg, suggesting it as a promising lead for development of novel therapeutics targe&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01961&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01961" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01961&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01961" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01961&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01961" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01961&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01961&amp;href=/doi/10.1021/acs.jmedchem.9b01961" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01961" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01961" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01961%26sid%3Dliteratum%253Aachs%26pmid%3D32315177%26genre%3Darticle%26aulast%3DSivaraman%26date%3D2020%26atitle%3DSynthesis%2Band%2BStructure%25E2%2580%2593Activity%2BRelationships%2Bof%2BArylsulfonamides%2Bas%2BAIMP2-DX2%2BInhibitors%2Bfor%2Bthe%2BDevelopment%2Bof%2Ba%2BNovel%2BAnticancer%2BTherapy%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D10%26spage%3D5139%26epage%3D5158%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291651" title="Anions">Anions</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/jmcmar.2020.63.issue-10/20200528/jmcmar.2020.63.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">AIMP2-DX2, a splicing variant of AIMP2, is up-regulated in lung cancer, possesses oncogenic activity, and results in tumorigenesis. Specifically inhibiting the interaction between AIMP2-DX2 and HSP70 to suppress AIMP2-DX2-dependent cancers with small molecules is considered a promising avenue for cancer therapeutics. Optimization of hit <b>BC-DXI-04</b> (IC<sub>50</sub> = 40.1 μM) provided new potent sulfonamide based AIMP2-DX2 inhibitors. Among these, <b>BC-DXI-843</b> showed improved inhibition against AIMP2-DX2 (IC<sub>50</sub> = 0.92 μM) with more than 100-fold selectivity over AIMP2 in a luciferase assay. Several binding assays indicated that this compound effectively induces cancer cell apoptosis by specifically interrupting the interaction between DX2 and HSP70, which leads to the degradation of DX2 via Siah1-mediated ubiquitination. More importantly, <b>BC-DXI-843</b> demonstrated <i>in vivo</i> efficacy in a tumor xenograft mouse model (H460 cells) at a dosage of 50 mg/kg, suggesting it as a promising lead for development of novel therapeutics targeting AIMP2-DX2 in lung cancer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Aminoacyl-tRNA synthetases (ARSs) are a family of 20 essential enzymes (one for each amino acid) that play an important role in conjugating amino acids to their cognate tRNAs during protein synthesis. In higher eukaryotic systems, nine different ARSs form a multisynthetase complex (MSC) with three auxiliary factor ARS-interacting multifunctional proteins (AIMPs). Although AIMP1–3 provide scaffolding for the assembly and integrity of the MSC, they also dissociate from the MSC to perform diverse regulatory functions that are not directly related to protein synthesis.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> Among them, AIMP2/p38 helps control the fate of a cell and performs various functions in response to stress signals. It displays antiproliferative activity by enhancing the growth-arresting signal of TGF-β, and it also promotes cell death by activating p53 and the apoptotic signal of TNF-α. The antiproliferative and proapoptotic roles of AIMP2 have been supported by <i>in vivo</i> mouse models. <i>Aimp2</i>-heterozygous mice, when compared with wild-type mice, are more vulnerable to tumorigenesis in lung, colon, and skin carcinogenesis models, suggesting that AIMP2 is a haploinsufficient tumor suppressor with a unique mechanism.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9">(5−9)</a></div><div class="NLM_p">The gene encoding AIMP2 consists of four exons and is subject to alternative splicing. A variant of AIMP2, known as AIMP2-DX2 (DX2 from here onward), is created by deleting exon 2 via alternative splicing and has been detected in various cancer cell lines and tissues. Although this variant has lost the ability to associate with the MSC, it can still inhibit the tumor-suppressive function of the native AIMP2 through competitive interactions with p53, FBP, and TRAF2. Furthermore, the specific interaction of DX2 with p14 and the subsequent inactivation of p14 are critical for tumor growth in KRAS-driven lung cancers. An increased ratio of DX2 versus AIMP2 shows a positive correlation with poor clinical outcome of lung cancer as well as chemotherapy resistance in ovarian cancer. siRNA or chemical inhibition of DX2 expression reduces tumor growth, indicating its potential as a therapeutic target.<a onclick="showRef(event, 'ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13 ref14">(10−14)</a></div><div class="NLM_p">More recently, we reported that the 70 kDa heat shock proteins (HSP70s) are critical determinants of DX2 levels in cells. A positive correlation between HSP70 and DX2 levels has been shown in various lung cancer cell lines and patient tissues. The binding of the substrate-binding domain (SBD) of HSP70 to the flexible N-terminal region (NFR) and the GST-N-terminal domain (GST-N) of DX2 protect it from Siah1-mediated ubiquitination. Moreover, preventing HSP70 from binding to DX2 with mutational and chemical inhibition destabilizes DX2, suggesting HSP70 is functionally significant in maintaining DX2 levels in cells. Thus, inhibiting the protein–protein interaction (PPI) between AIMP2-DX2 and HSP70 may lead to a therapeutic strategy against cancer (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic model for DX2-dependent cancer progression. DX2 is degraded by Siah1-mediated ubiquitination (upper). But in the presence of HSP70, DX2 binds with HSP70 and is protected from Siah1, stabilizing DX2 protein levels. Stabilized DX2 reduces p53 and p14 protein levels upon UV exposure and oncogenic signals, respectively. Stabilized DX2 also increases FBP and TRAF2 upon TGFβ and TNF-α transduction, respectively, resulting in enhanced cancer progression (bottom). Ub = ubiquitin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Among the previously identified small molecules targeting DX2,<a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19 ref20">(15−20)</a><b>BC-DXI-01</b> suppresses DX2 cellular levels through selective degradation of DX2 mRNA transcripts, leading to inhibition of DX2 activity and tumor suppression.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a><b>SLCB050</b> specifically inhibits the interaction between DX2 and p14ARF, suppressing proliferation of p14ARF-positive lung cancer cells.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> The sulfonamide <b>BC-DXI-04</b> (<b>1a</b>) was identified as a screening hit of DX2 that decreases the proliferation of lung cancer cells in a concentration-dependent manner.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Using one of its synthetic derivatives (<b>BC-DXI-495</b>, <b>2</b>) as a chemical probe, the mode of action was elucidated. These compounds destabilize DX2 by inhibiting its interaction with HSP70.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Herein, we report the synthesis and structure–activity relationships of a series of arylsulfonamides (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), which led to the novel compound <b>BC-DXI-843</b> (<b>20d</b>). It modulates the HSP70-DX2 interaction with greater efficacy than the starting compound. We also demonstrate the tumor-suppressive function of <b>BC-DXI-843</b> in a DX2-dependent manner via the inhibition of two protein interactions.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of known sulfonamide based small molecule inhibitors and strategy for SAR exploration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18305" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18305" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">The synthetic approaches used to generate the arylsulfonamides are described in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch6" aria-label="6">6</a>. The hit compound and its morpholine derivative were synthesized following the procedure shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. <span class="smallcaps smallerCapital">l</span>-Leucine was coupled with <i>p</i>-toluenesulfonyl chloride in basic medium to generate the sulfonamide <b>4a</b>, which yielded derivatives <b>1a</b> (<b>BC-DXI-04</b>) and <b>5a</b> under standard amide coupling conditions. Using similar conditions, the enantiomers, <b>1b</b> and <b>5b</b>, were synthesized from <span class="smallcaps smallerCapital">d</span>-leucine (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) <i>p</i>-toluenesulfonyl chloride, NaOH, H<sub>2</sub>O, rt, 12 h, 52% and 58%; (b) amine, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, 21–91%.</p></p></figure><div class="NLM_p">Anilines <b>8a</b>–<b>e</b> were prepared from 4-fluoronitrobenzene (<b>6</b>) by using nucleophilic aromatic substitution followed by reduction with Pd/C and H<sub>2</sub> as previously described.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> Subsequent condensation with intermediate <b>4a</b> under the standard conditions yielded amide derivatives <b>9a</b>–<b>e</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). <span class="smallcaps smallerCapital">l</span>-Leucine benzyl ester was initially coupled with <i>p</i>-toluic acid to generate intermediate <b>11</b>, which was subsequently deprotected under hydrogenation and further coupled with 4-morpholinoaniline using the aforementioned conditions to yield the diamide derivative <b>12</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0008.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) amine, K<sub>2</sub>CO<sub>3</sub>, DMF, 70°C, 5–6 h, 83–93%; (b) H<sub>2</sub>, Pd/C, MeOH, rt, 2–3 h, 94–99%; (c) amine, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, 21–91%.</p></p></figure><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0009.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) <i>p</i>-toluic acid, EDCI·HCl, HOBt, TEA, DCM, rt, 12 h, 18–94%; (b) H<sub>2</sub>, Pd/C, MeOH, 2 h; (c) 4-morpholinoaniline, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, 24%.</p></p></figure><div class="NLM_p">The arylsulfonamide derivatives <b>14a</b>–<b>f</b> were prepared following the procedures shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Various arylsulfonyl chlorides were coupled with <span class="smallcaps smallerCapital">l</span>-leucine under basic conditions yielding the intermediates <b>13a</b>–<b>f</b>. Under the standard amide coupling conditions, these acids yielded compounds <b>14a</b>–<b>f</b>. <i>p-</i>Toluene sulfonamides, <b>16a</b>–<b>k</b>, of various <span class="smallcaps smallerCapital">l</span>-amino acids were prepared as shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Compounds <b>17a</b>–<b>f</b>, <b>17i</b>, and <b>2</b> (<b>BC-DXI-495</b>) were obtained from their respective acids under the aforementioned conditions. The tyrosine derivative <b>17j</b> was prepared by the deprotection of the O-sulfonated compound under basic conditions, and derivatives <b>17g</b> and <b>17h</b> were prepared through the deprotection of the benzyl groups from their respective benzyl ester by hydrogenation.</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0010.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) arylsulfonyl chloride, NaOH, H<sub>2</sub>O, rt, 12 h, 14–59%; (b) 4-morpholinoaniline, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, 31–80%.</p></p></figure><figure id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0011.jpeg" id="GRAPHIC-d7e709-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) <i>p</i>-toluenesulfonyl chloride, NaOH, H<sub>2</sub>O, rt, 12 h, 13–92%; (b) 4-morpholinoaniline, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, and then (c) 5 M NaOH, EtOH, reflux, 4 h for <b>17j</b> and H<sub>2</sub>, Pd/C, MeOH, rt, 3 h for <b>17g</b> and <b>17h</b>, 31–68%.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>, various <span class="smallcaps smallerCapital">l</span>-tryptophan sulfonamides (<b>16k</b>, <b>18a</b>–<b>c</b>) were prepared using the established conditions. Subsequent condensation with various amines under the standard conditions generated derivatives <b>19a</b>,<b>b</b>, <b>20a</b>–<b>d</b>, and <b>21a</b>–<b>f</b>. The thiazole amines (<b>d</b>, <b>e</b>, and <b>f</b>) were prepared in one-step with the corresponding phenacyl bromides and thiourea using previously described conditions.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a></div><figure id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0012.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) arylsulfonyl chloride, NaOH, H<sub>2</sub>O, rt, 12 h, 40–54%; (b) amine, EDCI·HCl, HOBt, TEA, DCM, rt, 3–4 h, 31–92%; (c) amine, EDCI·HCl, HOBt, DIPEA, DMF, 60 °C, 16 h, for <b>20d</b>–<b>21f</b>, 31–75%.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Structure–Activity Relationship Exploration</h3><div class="NLM_p">High-throughput screening (HTS) involving our in-house chemical library identified a substituted sulfonamide, <b>BC-DXI-04</b> (<b>1a</b>), that suppresses intracellular levels of DX2 and inhibits the proliferation of cancer cells with moderate activity and good selectivity. It demonstrated 55.80% inhibition at 40 μM in a DX2-luciferase assay, which is an assay for determining the level of luciferase-tagged DX2 via detection of luminescence signal. The IC<sub>50</sub> and EC<sub>50</sub> values measured by DX2-luciferase and cell viability assay were 40.1 μM and 40.8 μM, respectively. On the basis of this hit, a series of arylsulfonamides were screened using a nanoluciferase assay at 40 μM. To define the key structural requirements for the biological activity of <b>BC-DXI-04</b>, our optimization strategy was focused on four discrete areas: (A) the effect of the chiral center and the requirement of a sulfonamide linker, (B) the 1,4,7-trioxa-10-azacyclododecane moiety, which could be replaced by an arylamine containing various substituents, (C) substitution effects on the phenyl ring of the sulfonamide part, and (D) variations in the amino acid chain (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><div class="NLM_p">Initial replacement of the 1,4,7-trioxa-10-azacyclododecane moiety with morpholine yielded a simplified derivative, <b>5a</b>, with a comparable inhibition of 57.30% in the DX2-luciferase assay (part B). In parallel, we also synthesized enantiomers (<span class="smallcaps smallerCapital">d</span>-isomer) of these compounds, <b>1b</b> and <b>5b</b>, to understand the effect of the chiral center on their activity (part A). As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the inhibition values of these isomers dramatically decreased, suggesting that the presence of an <span class="smallcaps smallerCapital">l</span>-configuration is essential for their activity. Furthermore, we replaced the morpholine group of <b>5a</b> with 4-morpholinoaniline, resulting in an equipotent molecule <b>9a</b>, (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) with 56.90% inhibition. Considering that the aryl linker could be a useful handle for optimizing the properties of the molecule, a detailed SAR study of this molecule (<b>9a</b>) was designed.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. DX2-Inhibitory Activity of the Sulfonamide Derivatives <b>1a</b>,<b>b</b> and <b>5a</b>,<b>b</b> Using a DX2-Nanoluciferase Assay</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0013.gif" alt="" id="gr8" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0014.gif" alt="" id="gr9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The change of the nanoluciferase-tagged DX2 level upon treating chemicals was determined by detecting the luminescence signal.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. DX2-Inhibitory Activity of the Sulfonamide Derivatives <b>9a</b>–<b>e</b>, <b>12</b>, and <b>14a</b>–<b>f</b> Using a Nanoluciferase Assay</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0015.gif" alt="" id="gr10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0016.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The change of the nanoluciferase-tagged DX2 level upon treating chemicals was determined by detecting the luminescence signal.</p></div></div><div></div></div><div class="NLM_p">We explored the effects of various substitutions on the phenyl ring of the amide chain and the sulfonamide moiety on inhibitory activity as shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Continuing the SAR study in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, replacing morpholine with pyrrolidine and piperidine moieties resulted in the less potent derivatives <b>9b</b>–<b>d</b>. Substituting with thiomorpholine 1,1-dioxide also decreased the inhibitory activity as in <b>9e</b>. Further modifications to the amide chain were planned in the later stages. The diamide analog, <b>12</b>, of compound <b>9a</b> exhibited significantly diminished inhibitory effects (30.20% inhibition), suggesting that the sulfonamide moiety is essential for the activity. We further explored the effects of substituents on the phenyl ring of the sulfonamide part on the inhibitory activity. When the methyl group was removed, the resulting compound <b>14a</b> exhibited a loss of potency (14.46% inhibition). The effects of electron withdrawing and donating groups were also examined (<b>14b</b>–<b>f</b>). However, except for 4-bromo, all other substitutions resulted in a significant loss in activity and the 4-bromobenzenesulfonamide derivative <b>14c</b> demonstrated a potency similar to <b>9a</b> with 54.55% inhibition in the DX2-luciferase assay.</div><div class="NLM_p">The impact of the amino acid chain on the inhibitory effects was then evaluated as shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. A series of <span class="smallcaps smallerCapital">l</span>-amino acid derivatives (<b>17a</b>–<b>j</b>, <b>2</b>) were synthesized and tested. Compounds <b>17a</b>–<b>j</b> showed significantly diminished inhibitory effects when compared with <b>9a</b>. However, the tryptophan derivative <b>2</b> (<b>BC-DXI-495</b>) showed significantly increased activity (67.10% inhibition) and is the most potent DX2 inhibitor synthesized in this series so far. <b>BC-DXI-495</b>, a representative compound in this series, was also used to understand their mode of action. It exerts tumor-suppressive activity by specifically inhibiting the tumor-promoting interaction between DX2 and HSP70.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. DX2-Inhibitory Activity of the Sulfonamide Derivatives <b>17a</b>–<b>j</b> and <b>2</b> Using a Nanoluciferase Assay</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0017.gif" alt="" id="gr12" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0018.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">The change of the nanoluciferase-tagged DX2 level upon treating chemicals was determined by detecting the luminescence signal.</p></div></div><div></div></div><div class="NLM_p">Further structural modifications of <b>BC-DXI-495</b> were explored (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The orientation effect of the methyl group on the phenyl ring was initially investigated. The <i>ortho</i> and <i>meta</i> derivatives <b>19a</b> and <b>19b</b> resulted in reduced potency with 59.23% and 40.78% inhibition, respectively, which suggests the importance of a <i>para</i>-methyl group. Further optimization focused on amide derivatives (part B) with 4-methyl (from <b>2</b>) and 4-bromo (from <b>14c</b>) sulfonamide moieties. Decreased potency was observed for the <b>20a</b> and <b>21a</b> derivatives without morpholine substitution on the phenyl ring. The introduction of thiophen-2-ylmethanamine resulted in decreased potency for the 4-methylbenzenesulfonamide derivative <b>20b</b>, while a moderate effect (42.99% inhibition) was observed with the 4-bromobenzenesulfonamide derivative <b>21b</b>. Similar results were obtained for the five-membered heteroarylamine derivatives <b>20c</b> and <b>21c</b>. Better potencies observed with the heterocycles compared to phenyl ring suggested an opportunity to introduce heteroarylamines. In addition, loss of potency with removal of the morpholine group (<b>20a</b> and <b>21a</b>) indicated the requirement of bulky amino groups. Introduction of 4-(4-methoxyphenyl)thiazol-2-amine provided the 4-methylbenzenesulfonamide derivative <b>20d</b>, which exhibited the best inhibitory results (72.56% inhibition), and the 4-bromobenzenesulfonamide analog <b>21d</b>, showing moderately good results (45.43% inhibition). We also examined the orientation effects of methoxy substitution on the benzene ring for the 4-bromobenzenesulfonamide derivative. The inhibitory effects were abolished for the synthesized molecules <b>21e</b> and <b>21f</b>, which suggests that the <i>p</i>-toluenesulfonamide and 4-(4-methoxyphenyl)thiazol-2-amine moieties are essential for inhibiting DX2 as exemplified by <b>20d</b> (<b>BC-DXI-843</b>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. DX2-Inhibitory Activity of the Sulfonamide Derivatives <b>19a</b>,<b>b</b>, <b>20a</b>–<b>d</b>, and <b>21a</b>–<b>f</b> Using a Nanoluciferase Assay</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0019.gif" alt="" id="gr14" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0020.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">The change of the nanoluciferase-tagged DX2 level upon treating chemicals was determined by detecting the luminescence signal.</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> <b>BC-DXI-843</b> Specifically Inhibits AIMP2-DX2</h3><div class="NLM_p">Driven by the improved inhibitory effect of <b>BC-DXI-843</b>, we assessed the selectivity profile of candidates to identify lead compounds. Six novel derivatives (<b>BC-DXI-04</b>, <b>5a</b>, <b>9a</b>, <b>14c</b>, <b>BC-DXI-495</b>, and <b>BC-DXI-843</b>) showing over 50% inhibition were subjected to secondary screening using nanoluciferase-DX2 and -AIMP2. Among the screened compounds, <b>BC-DXI-843</b> demonstrated an improved DX2 inhibition with better AIMP2 vs DX2 selectivity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Therefore, this compound was selected for further evaluation. To generate IC<sub>50</sub> values, A549 cells expressing nanoluciferase-DX2 or nanoluciferase-AIMP2 were treated with <b>BC-DXI-843</b> in a dose-dependent manner for 4 h, and the luminescence was measured. The IC<sub>50</sub> for inhibition of DX2 and AIMP2 shows more than 100-fold selectivity (DX2 IC<sub>50</sub> = 0.92 μM and AIMP2 IC<sub>50</sub> > 100 μM, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). <b>BC-DXI-495</b> was previously reported to be a DX2 inhibitor, and its IC<sub>50</sub> for DX2 degradation was 4.20 μM.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> To observe any improvement of <b>BC-DXI-843</b> over the previously reported compounds <b>BC-DXI-04</b> and <b>BC-DXI-495</b>, we compared their IC<sub>50</sub> values regarding inhibition of AIMP2 and DX2 expression and their EC<sub>50</sub> values in lung cancer and normal cells (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). The IC<sub>50</sub> of the compounds with respect to DX2 gradually decreased according to the modification of the compound but did not affect AIMP2 levels. The EC<sub>50</sub> measurements trended similarly to the IC<sub>50</sub> measurements; they gradually decreased. But the EC<sub>50</sub> values generated by normal lung cells were all 100-fold higher than those generated by cancer cells (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, right). To determine the mode of action of <b>BC-DXI-843</b>, we first examined the interaction between <b>BC-DXI-843</b> and DX2 using a fluorescence-based equilibrium binding assay. The binding affinity (<i>K</i><sub>d</sub>) was approximately 3.07 μM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Previously it was found that the analog <b>BC-DXI-495</b> was bound to the hydrophobic pocket of DX2, which comprises L80, T82, F116, T117, and K129 residues. This hydrophobic pocket is proximal to the binding surface of HSP70, suggesting that when <b>BC-DXI-495</b> binds to DX2, it may interfere with DX2/HSP70 binding.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> To characterize this relationship further, a docking study was performed with <b>BC-DXI-843</b> that focused on the hydrophobic pocket identified in the previous study. This study used our in-house homology model of DX2 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). In addition, <b>BC-DXI-843</b> docking was analyzed through the binding configuration of five important residues in the hydrophobic pocket. According to the results, <b>BC-DXI-843</b> was positioned very close to L80, T82, F116, T117, and K129. Interestingly, <b>BC-DXI-843</b> forms two hydrogen bonds with the hydrophobic pocket via T82 and D52 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). To validate the significance of T82 and D52 to the interaction between <b>BC-DXI-843</b> and DX2, we substituted several residues surrounding the hydrophobic pocket, including these two residues, to alanine and conducted the functional assay. When either D52 or T82 was mutated, <b>BC-DXI-843</b> could not interfere with the binding of DX2 to HSP70 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S1A</a>) and induce the degradation of DX2 protein (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S1B</a>). This confirmed the significance of the hydrogen bonds formed between <b>BC-DXI-843</b> and the D52 and T82 amino acid residues of DX2. Therefore, the improved biological activity of <b>BC-DXI-843</b> may be explained by the formation of these hydrogen bonds.</div><figure id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0004.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <b>BC-DXI-843</b> is a specific inhibitor of DX2. (A) DX2 and AIMP2 levels were measured using nanoluciferase-DX2 and AIMP2, respectively, after treatment with each of the six compounds (40 μM, 4 h). The experiment was independently repeated three times with error bars denoting SD. The presented value was the inhibited luciferase activity upon treatment with each chemicals compared to DMSO as a control. (B) A549 cells expressing nanoluciferase-DX2 or AIMP2 were treated with <b>BC-DXI-843</b> in a dose-dependent manner for 4 h, and the resulting luminescence was measured. IC<sub>50</sub> values for the inhibition of DX2 and AIMP2 were calculated as shown. The experiment was independently repeated three times with error bars denoting SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Comparison of the Inhibition Efficacy on DX2 and AIMP2 Protein Levels and Cell Proliferation with <b>BC-DXI-04</b>, <b>-495</b>, and <b>-843</b><a class="ref internalNav" href="#tbl5-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t5fn1" aria-label="b">b</a></th><th class="rowsep1 colsep0" colspan="2" align="center">EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t5fn1" aria-label="b">b</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">DX2</th><th class="colsep0 rowsep0" align="center">AIMP2</th><th class="colsep0 rowsep0" align="center">A549</th><th class="colsep0 rowsep0" align="center">WI-26</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BC-DXI-04</b></td><td class="colsep0 rowsep0" align="left">40.1</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">40.8</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BC-DXI-495</b></td><td class="colsep0 rowsep0" align="left">4.20</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">14.2</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BC-DXI-843</b></td><td class="colsep0 rowsep0" align="left">0.92</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">1.20</td><td class="colsep0 rowsep0" align="left">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl5-fn1"><div class="footnote" id="tbl5-fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> and EC<sub>50</sub> indicate 50% inhibition concentration for protein expression and cell viability, respectively.</p></div><div class="footnote" id="t5fn1"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> and EC<sub>50</sub> were determined by DX2- or AIMP2-nanoluciferase and cell viability assay, respectively</p></div></div></div><figure id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0005.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Determining the binding mode of <b>BC-DXI-843</b> to DX2. (A) Fluorescence-based binding titration measurement of <b>BC-DXI-843</b> to DX2. Protein samples were excited at 280 nm, and the decrease in fluorescence emission upon binding was measured at 335 nm. (B) Docking pose with the binding site of our in-house homology model of DX2 and <b>BC-DXI-843</b> (red). (C) Close-up view of <b>BC-DXI-843</b> with the relevant interaction residues (D52, T82) and distances between them and <b>BC-DXI-843</b>’s closest atoms. <b>BC-DXI-843</b> was also found to be docked close to the residues in the hydrophobic pocket. The structure of DX2-GST was obtained from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A34">5A34</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> <b>BC-DXI-843</b> Specifically Induces the Degradation of DX2 via Ubiquitination</h3><div class="NLM_p last">To confirm that <b>BC-DXI-843</b> is a DX2-specific inhibitor, we compared the inhibition of endogenous DX2 protein expression in H460 cells using immunoblotting after treating the cells with <b>BC-DXI-843</b> or <b>BC-DXI-495</b>. Both compounds exhibited dose-dependent inhibition of DX2 protein levels, and <b>BC-DXI-843</b> efficiently induced degradation of DX2 at a much lower concentration (1.20 μM) when compared to <b>BC-DXI-495</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S1C</a>). This implies that <b>BC-DXI-843</b> is a more potent regulator of DX2, and it regulates DX2 levels by specifically inducing its degradation. However, AIMP2 was not affected in any of the test conditions by <b>BC-DXI-843</b> or <b>BC-DXI-495</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S1C</a>). We also confirmed that BC-DXI-843 has no effect on the level of DX2 mRNA (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S1C</a>), suggesting the specific inhibitory effect on protein level of DX2. Since the degradation of DX2 is mediated by ubiquitination,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> we evaluated the amount of ubiquitinated DX2 after treatment with <b>BC-DXI-843</b> in a dose-dependent manner. <b>BC-DXI-843</b> induced the ubiquitination of endogenous DX2, and the amount of ubiquitinated DX2 was saturated at 1.20 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S1D</a>). These results are similar to the upper results in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S1C</a>, suggesting that <b>BC-DXI-843</b> leads to the degradation of DX2 via ubiquitination.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> <b>BC-DXI-843</b> Inhibits DX2 from Binding to HSP70</h3><div class="NLM_p last">It has been reported that DX2 is stabilized when bound to HSP70.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Therefore, we tested whether <b>BC-DXI-843</b> interferes with the interaction between DX2 and HSP70. First, we confirmed that <b>BC-DXI-843</b> interferes with DX2 and HSP70 binding by performing an <i>in vitro</i> pull-down assay using purified DX2 and HSP70 proteins in the presence of <b>BC-DXI-843</b>. <b>BC-DXI-843</b> interfered with DX2 and HSP70 binding at a dose similar to that which caused degradation of DX2 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S2A</a>). Next, we determined whether <b>BC-DXI-843</b>-mediated inhibition is specific to DX2 and HSP70 binding using a nanoluciferase-based complementation assay. Luminescence generated via DX2 and HSP70 binding was detected in the absence of <b>BC-DXI-843</b>, but the signal decreased in a dose-dependent manner in the presence of <b>BC-DXI-843</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S2B</a>, red). However, a dose-dependent decline was not observed in another binding pair, PRKACA and PRKAR2A (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S2B</a>, gray), suggesting that <b>BC-DXI-843</b> specifically inhibits DX2 and HSP70 binding. HSP70 is reported to block Siah1 from interacting with DX2, resulting in the inhibition of Siah1-mediated ubiquitination.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Therefore, we evaluated whether <b>BC-DXI-843</b> could affect the relationship between HSP70, Siah1, and DX2. Immunoprecipitation of DX2 in the presence of <b>BC-DXI-843</b> revealed diminished and enhanced binding of HSP70 and Siah1 to DX2, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">Figure S2C</a>). These results demonstrate that <b>BC-DXI-843</b> specifically inhibits HSP70 and DX2 binding, which causes ubiquitin-dependent degradation of DX2 by Siah1.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> <b>BC-DXI-843</b> Inhibits DX2-Dependent Cancer Cell Proliferation</h3><div class="NLM_p">Since DX2 is a significant oncogenic protein,<a onclick="showRef(event, 'ref11 ref12 ref17'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref17">(11,12,17)</a> we examined whether <b>BC-DXI-843</b>-mediated degradation of DX2 could inhibit cancer cell progression. A549 lung cancer cells and WI-26 normal cell lines were treated with <b>BC-DXI-843</b> in a dose-dependent manner, and EC<sub>50</sub> values for cell viability were determined. The EC<sub>50</sub> in A549 cells was 1.20 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, red), which was similar to the IC<sub>50</sub> for inhibition of DX2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). However, no inhibition of WI-26 cells was observed (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, blue), suggesting that <b>BC-DXI-843</b> specifically reduces the viability of cancer cells. Since DX2 levels in A549 cells were higher than in WI-26 cells, the above results suggest a relationship between <b>BC-DXI-843</b> and DX2 levels. Next, we demonstrated that the mechanism of action for <b>BC-DXI-843</b> is dependent on DX2 by using a DX2-inducible cell line. The DX2-induced cell line showed lower cell viability than control cells after treatment with <b>BC-DXI-843</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B), demonstrating that the level of DX2 is critical to the activity of <b>BC-DXI-843</b>. We further characterized the effect of <b>BC-DXI-843</b> on cell viability using a 3D cell culture system. Spheroids formed with H460 cells decreased in size after treatment with <b>BC-DXI-843</b> in a dose-dependent manner, although the functional concentration was higher than in the 2D cell culture condition (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> To further test the <i>in vivo</i> efficacy of <b>BC-DXI-843</b>, we xenografted H460 cells into the backs of mice, and <b>BC-DXI-843</b> (50 mg/kg) was intraperitoneally injected every other day for 15 days. The embedded tumor volume gradually declined (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B) after <b>BC-DXI-843</b> administration, but no changes in body weight were observed (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). The weight of the excised tumors after sacrifice had decreased in mice treated with <b>BC-DXI-843</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). These data suggest that <b>BC-DXI-843</b> decreases DX2-dependent cancer cell viability.</div><figure id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0006.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <b>BC-DXI-843</b> suppresses cancer cell proliferation in a DX2-dependent manner. (A) <b>BC-DXI-843</b>-specific inhibition of cancer cell proliferation. A549 cancer cells and WI-26 normal cells were treated with <b>BC-DXI-843</b> as indicated for 72 h, and cell viability was determined with an MTT assay. The EC<sub>50</sub>, 50% inhibiting concentration of cell viability, is shown on the right. N.D. = not determined. (B) Suppression of DX2-mediated cancer cell proliferation by <b>BC-DXI-843</b>. Empty vector (EV)- or DX2-inducible A549 cells (Ind-EV and Ind-DX2, respectively) were treated with doxycycline (Dox) and <b>BC-DXI-843</b> (<b>843</b>) as indicated, and cell viability was determined in the same manner as described above. (C) Inhibitory effect of <b>BC-DXI-843</b> on cancer cell spheroids formed in a 3D culture system. Several doses of <b>BC-DXI-843</b> were added to H460 spheroids over the course of 72 h. Cell toxicity was determined by a CellTiter-Glo 3D assay, and representative photos of the spheroids are shown. (A–C) All experiments were independently repeated three times with error bars denoting SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0007.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <i>In vivo</i> antitumor effect of <b>BC-DXI-843</b>. (A–D) Mice (<i>n</i> = 4) subcutaneously injected with H460 cells were intraperitoneally administered <b>BC-DXI-843</b> (50 mg/kg) every other day for 15 days. The sizes of the embedded tumors (B) and body weights (D) were measured throughout the experiment. After sacrifice, the weights of the harvested tumors were determined (C) and photos of whole-body bearing the tumor and the excised tumors were taken (A).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02311" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02311" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">DX2, a splicing variant of AIMP2, is highly expressed in various cancers, including histologically small cell lung cancer. Therefore, inhibiting DX2 with small molecules is a promising therapeutic treatment for cancers expressing DX2. In this study, a series of arylsulfonamides was synthesized and evaluated as potent PPI inhibitors that specifically interrupt the interaction between DX2 and HSP70. Structure–activity relationship studies revealed that the <span class="smallcaps smallerCapital">l</span>-configuration at the chiral center and the arylsulfonamide moiety are essential for anticancer activity. The replacement of <span class="smallcaps smallerCapital">l</span>-leucine by <span class="smallcaps smallerCapital">l</span>-tryptophan and the incorporation of a substituted arylamine group are critical in improving antiproliferation efficacy in cancer cells.</div><div class="NLM_p">Thus, we identified a series of potent anticancer compounds. The most potent compound, <b>BC-DXI-843</b>, exhibited an IC<sub>50</sub> of 0.92 μM on DX2 with more than 100-fold selectivity over AIMP2. <b>BC-DXI-843</b> also showed an EC<sub>50</sub> value of 1.20 μM on cancer cell viability but had no effect on normal cells. A docking study revealed that <b>BC-DXI-843</b> exhibits favorable binding characteristics with key residues in a hydrophobic pocket of AIMP2-DX2, which may interfere with the interaction between DX2 and HSP70.</div><div class="NLM_p last">The biological activity and mechanism of action of <b>BC-DXI-843</b> were analyzed. <b>BC-DXI-843</b> specifically decreased endogenous DX2 protein levels by inducing DX2 degradation via Siah1, an E3 ligase specific to DX2. In comparison with <b>BC-DXI-04</b> and <b>BC-DXI-495</b>, small molecules previously reported to decrease DX2 levels, <b>BC-DXI-843</b> showed approximately 40- and 5-fold improvements in potency, respectively. <b>BC-DXI-843</b> specifically interrupts the interaction between DX2 and HSP70 according to several binding assays. This proves that the binding of the two proteins is significant to DX2 stabilization. Furthermore, <b>BC-DXI-843</b> significantly suppressed the progression of cancer cells <i>in vitro</i> and <i>in vivo</i> in a DX2-dependent manner, which emphasizes the tumor-promoting function of DX2. The proliferation of normal lung cells and the bodyweight of mice were not affected after treatment with <b>BC-DXI-843</b>, implying that there are no significant side effects associated with <b>BC-DXI-843</b>. Thus, this study identified <b>BC-DXI-843</b> as a potential lead for the development of novel therapeutics that could target DX2 and combat lung cancer, especially SCLC.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85880" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85880" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry. General Procedures</h3><div class="NLM_p last">All commercial chemicals were reagent-grade and were used without further purification. Solvents were dried with standard procedures. All reactions were carried out under an atmosphere of dried argon in flame-dried glassware. The proton nuclear magnetic resonance (<sup>1</sup>H NMR) spectra were determined on a Varian 400 MHz spectrometer (Varian Medical Systems, Inc., Palo Alto, CA, USA). <sup>13</sup>C NMR spectra were recorded on a Varian 100 MHz spectrometer. The chemical shifts are provided in parts per million (ppm) downfield with coupling constants in hertz (Hz). The mass spectra were recorded using high-resolution mass spectrometry (HRMS) (electron ionization MS) on a JMS-700 mass spectrometer (Jeol, Japan) or by HRMS (electrospray ionization MS) on a G2 QTOF mass spectrometer. The products from all reactions were purified by flash column chromatography using silica gel 60 (230–400 mesh Kieselgel 60). Additionally, thin-layer chromatography on 0.25 mm silica plates (E. Merck; silica gel 60 F254) was used to monitor reactions. Final product purity was determined by reversed phase high-pressure liquid chromatography (RP-HPLC) using a Waters Corp. HPLC system equipped with an ultraviolet (UV) detector set at 254 nm. The mobile phases used were (A) H<sub>2</sub>O containing 0.05% TFA and (B) CH<sub>3</sub>CN. HPLC employed a YMC Hydrosphere C18 (HS-302) column (5 μm particle size, 12 nm pore size) that was 4.6 mm in diameter ×150 mm in size with a flow rate of 1.0 mL/min. The compound purity was assessed using either a gradient of 25% B to 100% B in 30 min (method A) or a gradient of 5% B to 100% B in 30 min (method B). The purity of all biologically evaluated compounds was ≥95% in method A or in method B. Melting points were measured on a Fisherbrand digital melting point apparatus.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Procedure for the Preparation of Sulfonamide</h3><div class="NLM_p last">Arylsulfonyl chloride (1 equiv) was added to a stirred solution of amino acid (1 equiv) in 1 M aqueous sodium hydroxide solution (20 volumes), and the mixture was stirred at rt for 8 h. The reaction mass was extracted with <i>n</i>-hexane, and the aqueous layer was acidified with 1 N HCl up to pH 4 at 0 °C. The resulting white precipitate was filtered, washed with cold water, and collected.</div></div><div id="sec4_1_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> (<i>S</i>)-4-Methyl-2-(4-methylphenylsulfonamido)pentanoic Acid (<b>4a</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>3a</b> and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (1.12 g, 52%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.72 (brs, 1H), 8.03 (d, <i>J</i> = 8.4 Hz, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 7.35 (d, <i>J</i> = 8.0 Hz, 2H), 3.64–3.40 (m, 1H), 2.36 (s, 3H), 1.55–1.45 (m, 1H), 1.40–1.25 (m, 2H), 0.79 (d, <i>J</i> = 6.4 Hz, 3H), 0.68 (d, <i>J</i> = 6.4 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>13</sub>H<sub>20</sub>NO<sub>4</sub>S calcd 286.1113, found 286.1113.</div></div><div id="sec4_1_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> (<i>R</i>)-4-Methyl-2-(4-methylphenylsulfonamido)pentanoic Acid (<b>4b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>3b</b> and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (1.26 g, 58%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.72 (brs, 1H), 8.03 (d, <i>J</i> = 8.4 Hz, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 7.35 (d, <i>J</i> = 8.0 Hz, 2H), 3.64–3.40 (m, 1H), 2.36 (s, 3H), 1.55–1.45 (m, 1H), 1.40–1.25 (m, 2H), 0.79 (d, <i>J</i> = 6.4 Hz, 3H), 0.68 (d, <i>J</i> = 6.4 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>13</sub>H<sub>20</sub>NO<sub>4</sub>S calcd 286.1113, found 286.1107.</div></div><div id="sec4_1_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> (<i>S</i>)-4-Methyl-2-(phenylsulfonamido)pentanoic Acid (<b>13a</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>3a</b> and benzenesulfonyl chloride according to the general procedure as a colorless solid (0.68 g, 33%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.86 (d, <i>J</i> = 8.0 Hz, 2H), 7.58 (t, <i>J</i> = 7.2 Hz, 1H), 7.50 (t, <i>J</i> = 7.6 Hz, 2H), 5.10 (d, <i>J</i> = 1.2 Hz, 1H), 3.98–3.92 (m, 1H), 1.76–1.74 (m, 1H), 1.54–1.51 (m, 2H), 0.90 (d, <i>J</i> = 6.8 Hz, 3H), 0.83 (d, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>12</sub>H<sub>18</sub>NO<sub>4</sub>S calcd 272.0957, found 272.0944.</div></div><div id="sec4_1_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> (<i>S</i>)-2-(4-Fluorophenylsulfonamido)-4-methylpentanoic Acid (<b>13b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>3a</b> and 4-fluorobenzenesulfonyl chloride according to the general procedure as an off white solid (0.95 g, 43%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.89–7.82 (m, 2H), 7.16–7.09 (m, 2H), 5.05 (d, <i>J</i> = 1.2 Hz, 1H), 3.98–3.92 (m, 1H), 1.82–1.75 (m, 1H), 1.59–1.47 (m, 2H), 0.92 (d, <i>J</i> = 5.2 Hz, 3H), 0.87 (d, <i>J</i> = 5.2 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>12</sub>H<sub>17</sub>FNO<sub>4</sub>S calcd 290.0862, found 290.0851.</div></div><div id="sec4_1_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> (<i>S</i>)-2-(4-Bromophenylsulfonamido)-4-methylpentanoic Acid (<b>13c</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>3a</b> and 4-bromobenzenesulfonyl chloride according to the general procedure as a light yellow solid (1.57 g, 59%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.72 (d, <i>J</i> = 8.8 Hz, 2H), 7.63 (d, <i>J</i> = 8.8 Hz, 2H), 5.10 (d, <i>J</i> = 1.2 Hz, 1H), 4.01–3.95 (m, 1H), 1.83–1.76 (m, 1H), 1.61–1.53 (m, 2H), 0.91 (d, <i>J</i> = 6.8 Hz, 3H), 0.88 (d, <i>J</i> = 6.4 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>12</sub>H<sub>17</sub>BrNO<sub>4</sub>S calcd 350.0062, found 350.0055.</div></div><div id="sec4_1_2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> (<i>S</i>)-2-(4-Methoxyphenylsulfonamido)-4-methylpentanoic Acid (<b>13d</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>3a</b> and 4-methoxybenzenesulfonyl chloride according to the general procedure as a colorless solid (1.15 g, 48%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.22 (brs, 1H), 7.97 (d, <i>J</i> = 5.6 Hz, 1H), 7.74 (d, <i>J</i> = 8.0 Hz, 2H), 6.96 (d, <i>J</i> = 8.0 Hz, 2H), 3.82 (s, 3H), 3.63–3.59 (m, 1H), 1.58–1.53 (m, 1H), 1.39–1.34 (m, 2H), 0.80 (d, <i>J</i> = 6.4 Hz, 3H), 0.69 (d, <i>J</i> = 6.4 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>13</sub>H<sub>20</sub>NO<sub>5</sub>S calcd 302.1062, found 302.1055.</div></div><div id="sec4_1_2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> (<i>S</i>)-2-(Biphenyl-4-ylsulfonamido)-4-methylpentanoic Acid (<b>13e</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>3a</b> and 4-phenylbenzenesulfonyl chloride according to the general procedure as a pink solid (0.37 g, 14%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.88 (d, <i>J</i> = 8.2 Hz, 2H), 7.71–7.65 (m, 2H), 7.63–7.57 (m, 2H), 7.53–7.40 (m, 3H), 4.97 (d, <i>J</i> = 9.3 Hz, 1H), 4.02–3.96 (m, 1H), 1.85–1.73 (m, 1H), 1.56–1.40 (m, 2H), 0.89 (d, <i>J</i> = 6.6 Hz, 3H), 0.85 (d, <i>J</i> = 6.6 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>18</sub>H<sub>22</sub>NO<sub>4</sub>S calcd 348.1270, found 348.1270.</div></div><div id="sec4_1_2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> (<i>S</i>)-4-Methyl-2-(4-nitrophenylsulfonamido)pentanoic Acid (<b>13f</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>3a</b> and 4-nitrobenzenesulfonyl chloride according to the general procedure as a light yellow solid (0.43 g, 18%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.67 (brs, 1H), 8.5 (s, 1H), 8.40 (d, <i>J</i> = 8.0 Hz, 2H), 8.01 (d, <i>J</i> = 8.0 Hz, 2H), 3.78–3.34 (m, 1H), 1.61–1.58 (m, 1H), 1.45–1.41 (m, 2H), 0.84 (d, <i>J</i> = 6.4 Hz, 3H), 0.75 (d, <i>J</i> = 6.4 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>12</sub>H<sub>17</sub>N<sub>2</sub>O<sub>6</sub>S calcd 317.0807, found 317.0808.</div></div><div id="sec4_1_2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 2-(4-Methylphenylsulfonamido)acetic Acid (<b>16a</b>)</h3><div class="NLM_p last">The title compound was prepared from glycine (<b>15a</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (1.03 g, 34%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.72 (brs, 1H), 7.94 (t, <i>J</i> = 6.0 Hz, 1H), 7.67 (d, <i>J</i> = 8.4 Hz, 2H), 7.37 (d, <i>J</i> = 8.4 Hz, 2H), 3.54 (d, <i>J</i> = 6.0 Hz, 2H), 2.35 (s, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>9</sub>H<sub>12</sub>NO<sub>4</sub>S calcd 230.0487, found 230.0486.</div></div><div id="sec4_1_2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> (<i>S</i>)-2-(4-Methylphenylsulfonamido)propanoic Acid (<b>16b</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-alanine (<b>15b</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (1.50 g, 55%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.62 (brs, 1H), 8.04 (d, <i>J</i> = 8.4 Hz, 1H) (s, 1H), 7.66 (d, <i>J</i> = 8.0 Hz, 2H), 7.36 (d, <i>J</i> = 8.0 Hz, 2H), 3.76–3.68 (m, 1H), 2.37 (s, 3H), 1.12 (d, <i>J</i> = 7.2 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>10</sub>H<sub>14</sub>NO<sub>4</sub>S calcd 244.0644, found 244.0638.</div></div><div id="sec4_1_2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> (<i>S</i>)-3-Hydroxy-2-(4-methylphenylsulfonamido)propanoic Acid (<b>16c</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-serine (<b>15c</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (0.57 g, 23%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 7.89 (d, <i>J</i> = 8.4 Hz, 1H), 7.67 (d, <i>J</i> = 8.4 Hz, 2H), 7.33 (d, <i>J</i> = 8.4 Hz, 2H), 4.95 (brs, 1H), 3.73–3.68 (m, 1H), 3.50–3.40 (m, 2H), 2.37 (s, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>10</sub>H<sub>14</sub>NO<sub>5</sub>S calcd 260.0593, found 260.0585.</div></div><div id="sec4_1_2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> (<i>S</i>)-3-Methyl-2-(4-methylphenylsulfonamido)butanoic Acid (<b>16d</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-valine (<b>15d</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (0.58 g, 25%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.55 (s, 1H), 7.91 (d, <i>J</i> = 9.6 Hz, 1H), 7.65 (d, <i>J</i> = 8.4 Hz, 2H), 7.34 (d, <i>J</i> = 8.4 Hz, 2H), 3.50–3.44 (m, 1H), 2.34 (s, 3H), 1.92–1.85 (m, 1H), 0.81 (d, <i>J</i> = 6.8 Hz, 3H), 077 (d, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>12</sub>H<sub>18</sub>NO<sub>4</sub>S calcd 272.0957, found 272.0950.</div></div><div id="sec4_1_2_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (2<i>S</i>,3<i>S</i>)-3-Hydroxy-2-(4-methylphenylsulfonamido)butanoic Acid (<b>16e</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-(−)-threonine (<b>15e</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (0.94 g, 41%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.67 (d, <i>J</i> = 8.0 Hz, 2H), 7.51 (d, <i>J</i> = 9.2 Hz, 1H), 7.33 (d, <i>J</i> = 8.0 Hz, 2H), 3.96–3.93 (m, 1H), 3.63–3.61 (m, 1H), 2.28 (s, 3H), 1.00 (d, <i>J</i> = 6.4 Hz, 3H). HRMS (ESI); [M + H]<sup>+</sup> C<sub>11</sub>H<sub>16</sub>NO<sub>5</sub>S calcd 274.0749, found 274.0749.</div></div><div id="sec4_1_2_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> (<i>S</i>)-2-(4-Methylphenylsulfonamido)-4-(methylthio)butanoic Acid (<b>16f</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-methionine (<b>15f</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (0.26 g, 13%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.08 (d, <i>J</i> = 8.0 Hz, 1H), 7.65 (d, <i>J</i> = 8.4 Hz, 2H), 7.35 (d, <i>J</i> = 8.4 Hz, 2H), 3.81–3.79 (m, 1H), 2.44–2.24 (m, 5H), 1.91 (s, 3H), 1.86–1.64 (m, 2H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>12</sub>H<sub>18</sub>NO<sub>4</sub>S<sub>2</sub> calcd 304.0677, found 304.0677.</div></div><div id="sec4_1_2_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> (<i>S</i>)-4-(Benzyloxy)-2-(4-methylphenylsulfonamido)-4-oxobutanoic Acid (<b>16g</b>)</h3><div class="NLM_p last">The title compound was prepared from 5-benzyl <span class="smallcaps smallerCapital">l</span>-aspartate (<b>15g</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (1.10 g, 66%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.73 (d, <i>J</i> = 8.4 Hz, 2H), 7.39–7.25 (m, 7H), 5.64 (d, <i>J</i> = 8.0 Hz, 1H), 5.04 (s, 2H), 4.21–4.17 (m, 1H), 3.04 (dd, <i>J</i> = 17.4, 4.2 Hz, 1H), 2.87 (dd, <i>J</i> = 17.6, 5.2 Hz, 1H), 2.39 (s, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>18</sub>H<sub>20</sub>NO<sub>6</sub>S calcd 378.1011, found378.0995.</div></div><div id="sec4_1_2_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> (<i>S</i>)-5-(Benzyloxy)-2-(4-methylphenylsulfonamido)-5-oxopentanoic Acid (<b>16h</b>)</h3><div class="NLM_p last">The title compound was prepared from 5-benzyl <span class="smallcaps smallerCapital">l</span>-glutamate (<b>15h</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (1.15 g, 70%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.68 (d, <i>J</i> = 8.4 Hz, 2H), 7.36–7.30 (m, 5H), 7.22 (d, <i>J</i> = 8.4 Hz, 2H), 5.67 (d, <i>J</i> = 9.2 Hz, 1H), 5.07 (s, 2H), 4.01–3.96 (m, 1H), 2.51–2.38 (m, 2H), 2.34 (s, 3H), 2.17–2.12 (m, 1H), 1.93–1.88 (m, 1H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>19</sub>H<sub>22</sub>NO<sub>6</sub>S calcd 392.1168, found 392.1168.</div></div><div id="sec4_1_2_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (<i>S</i>)-2-(4-Methylphenylsulfonamido)-3-phenylpropanoic Acid (<b>16i</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-phenylalanine (<b>15i</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (0.60 g, 31%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.72 (s, 1H), 8.18 (d, J = 9.2 Hz, 1H), 7.44 (d, J = 8.4 Hz, 2H), 7.21–7.16 (m, 5H), 7.10–7.08 (m, 2H), 3.85–3.76 (m, 1H), 2.94–2.89 (m, 1H), 2.72–2.67 (m, 1H), 2.34 (s, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>16</sub>H<sub>18</sub>NO<sub>4</sub>S calcd 320.0957, found 320.0949.</div></div><div id="sec4_1_2_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> (<i>S</i>)-2-(4-Methylphenylsulfonamido)-3-(4-(tosyloxy)phenyl)propanoic Acid (<b>16j</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-tyrosine (<b>15j</b>) and <i>p</i>-toluenesulfonyl chloride according to the general procedure as a colorless solid (0.81 g, 30%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.17 (d, <i>J</i> = 9.2 Hz, 1H), 7.69 (d, <i>J</i> = 8.4 Hz, 2H), 7.48–7.41 (m, 4H), 7.21 (d, <i>J</i> = 8.0 Hz, 2H), 7.09 (d, <i>J</i> = 8.8 Hz, 1H), 6.81 (d, <i>J</i> = 8.8 Hz, 2H), 3.83–3.77 (m, 1H), 3.07–3.03 (m, 1H), 2.69–2.63 (m, 1H), 2.33 (s, 3H), 2.28 (s, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>23</sub>H<sub>24</sub>NO<sub>7</sub>S<sub>2</sub> calcd 490.0994, found 490.0992.</div></div><div id="sec4_1_2_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> (<i>S</i>)-3-(1<i>H</i>-Indol-3-yl)-2-(4-methylphenylsulfonamido)propanoic Acid (<b>16k</b>)</h3><div class="NLM_p last">See ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>.</div></div><div id="sec4_1_2_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> (<i>S</i>)-3-(1<i>H</i>-Indol-3-yl)-2-(2-methylphenylsulfonamido)propanoic Acid (<b>18a</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-tryptophan (<b>15k</b>) and <i>o</i>-toluenesulfonyl chloride according to the general procedure as a light brown solid (0.70 g, 40%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.61 (brs, 1 H), 10.78 (s, 1H), 8.26 (d, <i>J</i> = 8.4 Hz, 1H), 7.57 (d, <i>J</i> = 7.6 Hz, 2H), 7.35 (t, <i>J</i> = 8.0 Hz, 1H), 7.29 (t, <i>J</i> = 8.0 Hz, 1H), 7.29 (d, <i>J</i> = 8.0 Hz, 2H), 7.03 (d, <i>J</i> = 8.0 Hz, 2H), 6.89 (t, <i>J</i> = 7.6 Hz, 1H), 3.83 (q, <i>J</i> = 7.6 Hz, 1H), 3.05 (dd, <i>J</i> = 14.5, 6.7 Hz, 1H), 2.89 (dd, <i>J</i> = 14.5, 7.8 Hz, 1H), 2.43 (s, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>18</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub>S calcd 359.1066, found 359.1066.</div></div><div id="sec4_1_2_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> (<i>S</i>)-3-(1<i>H</i>-Indol-3-yl)-2-(3-methylphenylsulfonamido)propanoic Acid (<b>18b</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-tryptophan (<b>15k</b>) and <i>m</i>-toluenesulfonyl chloride according to the general procedure as a light brown solid (0.750 g, 43%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.61 (brs, 1 H), 10.79 (s, 1H), 8.19 (d, <i>J</i> = 8.4 Hz, 1H), 7.39 (s, 2H), 7.30–7.25 (m, 4H), 7.05–7.01 (m, 2H), 6.92 (t, <i>J</i> = 8.0 Hz, 1H), 3.90 (q, <i>J</i> = 7.6 Hz, 1H), 3.03 (dd, <i>J</i> = 14.5, 6.7 Hz, 1H), 2.82 (dd, <i>J</i> = 14.5, 7.8 Hz, 1H), 2.26 (s, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>18</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub>S calcd 359.1066, found 359.1066.</div></div><div id="sec4_1_2_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> (<i>S</i>)-2-(4-Bromophenylsulfonamido)-3-(1<i>H</i>-indol-3-yl)propanoic Acid (<b>18c</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-tryptophan (<b>15k</b>) and 4-bromobenzenesulfonyl chloride according to the general procedure as a colorless solid (0.82 g, 54%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.61 (brs, 1 H), 10.78 (s, 1H), 8.38 (d, <i>J</i> = 8.4 Hz, 1H), 7.51 (d, <i>J</i> = 8.0 Hz, 2H), 7.40 (t, <i>J</i> = 8.0 Hz, 2H), 7.31 (d, <i>J</i> = 7.2 Hz, 1H), 7.29 (d, <i>J</i> = 8.0 Hz, 1H), 7.07 (s, 1H), 7.03 (d, <i>J</i> = 8.0 Hz, 1H), 6.92 (t, <i>J</i> = 7.6 Hz, 1H), 3.92 (q, <i>J</i> = 7.6 Hz, 1H), 3.05 (dd, <i>J</i> = 14.5, 6.7 Hz, 1H), 2.89 (dd, <i>J</i> = 14.5, 7.8 Hz, 1H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>17</sub>H<sub>16</sub>BrN<sub>2</sub>O<sub>4</sub>S calcd 423.0014, found 424.0009.</div></div><div id="sec4_1_2_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> (<i>S</i>)-Benzyl 4-Methyl-2-(4-methylbenzamido)pentanoate (<b>11</b>)</h3><div class="NLM_p last">The title compound was prepared from <span class="smallcaps smallerCapital">l</span>-leucine benzylester (<b>10</b>) and <i>p</i>-toluenebenzoic acid following general procedure for amide coupling reaction as a colorless solid (0.76 g, 94%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.68 (d, <i>J</i> = 8.0 Hz, 2H), 7.39–7.32 (m, 5H), 7.22 (d, <i>J</i> = 8.4 Hz, 2H), 6.51 (d, <i>J</i> = 7.6 Hz, 1H), 5.22–5.16(m, 2H), 4.93–4.87 (m, 1H), 2.39 (s, 3H), 1.80–1.62 (m, 3H), 0.97 (d, <i>J</i> = 6.0 Hz, 3H), 0.95 (d, <i>J</i> = 6.0 Hz, 3H). HRMS (ESI): [M + H]<sup>+</sup> C<sub>21</sub>H<sub>26</sub>NO<sub>3</sub>S calcd 340.1913, found 340.1913.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> General Procedure for Amide Coupling</h3><div class="NLM_p last">EDCI·HCl (1.1 equiv) and HOBt (1.1 equiv) were added to a stirred solution of carboxylic acid (1 equiv), amine (1 equiv), and TEA (2.5 equiv) in DCM at room temperature, and the reaction mixture was stirred at the same temperature for 18 h. The solvent was evaporated, and the residue was treated with saturated sodium bicarbonate and extracted with DCM. The combined organic layer was then washed with brine, dried over magnesium sulfate, filtered, and concentrated under reduced pressure. The residue was purified through flash column chromatography (hexane/ethyl acetate).</div></div><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> (<i>S</i>)-4-Methyl-<i>N</i>-(4-methyl-1-oxo-1-(1,4,7-trioxa-10-azacyclododecan-10-yl)pentan-2-yl)benzenesulfonamide (<b>1a</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>4a</b> and 1,4,7-trioxa-10-azacyclododecane according to the general procedure as a colorless solid (0.14 g, 31%). Mp: 78–80 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.71 (d, <i>J</i> = 8.0 Hz, 2H), 7.27 (d, <i>J</i> = 8.0 Hz, 2H), 5.59 (d, <i>J</i> = 10.0 Hz, 1H), 4.25–4.19 (m, 1H), 3.78–3.71 (m, 2H), 3.69–3.37 (m, 11H), 3.22–3.12 (m, 3H), 2.40 (s, 3H), 1.98–1.91 (m, 1H), 1.49–1.33 (m, 2H), 0.91 (d, <i>J</i> = 4.4 Hz, 3H) 0.89 (d, <i>J</i> = 4.4 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 172.37, 143.28, 137.20, 129.43, 127.30, 71.44, 70.49, 69.80, 69.68, 69.22, 68.93, 51.64, 49.99, 48.96, 42.69, 24.22, 23.52, 21.51, 21.05, 20.93. HRMS (ESI): [M + H]<sup>+</sup> C<sub>21</sub>H<sub>35</sub>N<sub>2</sub>O<sub>6</sub>S calcd 443.2216, found 443.2216; purity ≥99.29% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 13.35 min).</div></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> (<i>R</i>)-4-Methyl-<i>N</i>-(4-methyl-1-oxo-1-(1,4,7-trioxa-10-azacyclododecan-10-yl)pentan-2-yl)benzenesulfonamide (<b>1b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>4b</b> and 1,4,7-trioxa-10-azacyclododecane according to the general procedure as a colorless solid (0.09 g, 21%). Mp: 69–71 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.71 (d, <i>J</i> = 7.6 Hz, 2H), 7.27 (d, <i>J</i> = 7.6 Hz, 2H), 5.59 (d, <i>J</i> = 9.6 Hz, 1H), 4.25–4.19 (m, 1H), 3.78–3.7 (m, 2H), 3.61–3.39 (m, 11H), 3.23–3.12 (m, 3H), 2.40 (s, 3H), 1.97–1.90 (m, 1H), 1.49–1.33 (m, 2H), 0.92 (d, <i>J</i> = 4.2 Hz, 3H), 0.89 (d, <i>J</i> = 5.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 172.33, 143.25, 137.19, 129.39, 127.26, 71.41, 70.46, 69.77, 69.66, 69.18, 68.89, 51.60, 49.96, 48.93, 42.64, 24.18, 23.49, 21.49, 20.90. HRMS (ESI): [M + H]<sup>+</sup> C<sub>21</sub>H<sub>35</sub>N<sub>2</sub>O<sub>6</sub>S calcd 443.2216, found 443.2208; purity ≥98.18% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 13.33 min).</div></div><div id="sec4_1_3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> (<i>S</i>)-4-Methyl-<i>N</i>-(4-methyl-1-morpholino-1-oxopentan-2-yl)benzenesulfonamide (<b>5a</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>4a</b> and morpholine according to the general procedure as a colorless solid (0.09 g, 74%). Mp: 162–164 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.70 (d, <i>J</i> = 7.6 Hz, 2H), 7.29 (d, <i>J</i> = 7.6 Hz, 2H), 5.60 (d, <i>J</i> = 9.6 Hz, 1H), 4.04–4.01 (m, 1H), 3.61–3.51 (m, 2H), 3.41–3.36 (m, 1H), 3.31–3.14 (m, 5H), 2.42 (s, 3H), 2.00–1.93 (m, 1H), 1.50–1.43 (m, 1H), 1.17–1.11 (m, 1H), 0.95 (d, <i>J</i> = 6.8 Hz, 3H), 0.92 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.91, 143.56, 136.56, 129.46, 127.53, 66.36, 66.04, 51.04, 45.41, 42.55, 42.25, 24.07, 23.27, 21.50, 21.11. HRMS (ESI): [M + H]<sup>+</sup> C<sub>17</sub>H<sub>27</sub>N<sub>2</sub>O<sub>4</sub>S calcd 355.1692, found 355.1692; purity ≥98.57% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 12.34 min).</div></div><div id="sec4_1_3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> (<i>R</i>)-4-Methyl-<i>N</i>-(4-methyl-1-morpholino-1-oxopentan-2-yl)benzenesulfonamide (<b>5b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>4b</b> and morpholine according to the general procedure as a colorless solid (0.10 g, 91%). Mp: 163–165 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.70 (d, <i>J</i> = 7.6 Hz, 2H), 7.29 (d, <i>J</i> = 7.6 Hz, 2H), 5.61 (d, <i>J</i> = 9.6 Hz, 1H), 4.06–4.01 (m, 1H), 3.60–3.50 (m, 2H), 3.41–3.36 (m, 1H), 3.31–3.14 (m, 5H), 2.42 (s, 3H), 2.00–1.93 (m, 1H), 1.50–1.43 (m, 1H), 1.17–1.11 (m, 1H), 0.95 (d, <i>J</i> = 6.8 Hz, 3H), 0.92 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.90, 143.55, 136.57, 129.45, 127.52, 66.39, 66.04, 51.02, 45.41, 42.53, 42.24, 24.06, 23.26, 21.50, 21.11. HRMS (ESI): [M + H]<sup>+</sup> C<sub>17</sub>H<sub>27</sub>N<sub>2</sub>O<sub>4</sub>S calcd 355.1692, found 355.1692; purity ≥97.81% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 12.35 min).</div></div><div id="sec4_1_3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> (<i>S</i>)-4-Methyl-2-(4-methylphenylsulfonamido)-N-(4-morpholinophenyl)pentanamide (<b>9a</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>4a</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.04 g, 25%). Mp: 176–178 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.76 (d, <i>J</i> = 8.0 Hz, 2H), 7.69 (s, 1H), 7.28–7.16 (m, 4H), 6.83 (d, <i>J</i> = 8.8 Hz, 2H), 5.03 (d, <i>J</i> = 7.6 Hz, 1H), 3.85 (t, <i>J</i> = 4.8 Hz, 4H), 3.78–3.73 (m, 1H), 3.08 (t, <i>J</i> = 4.8 Hz, 4H), 2.33 (s, 3H), 1.70–1.60 (m, 2H), 1.52–1.46 (m, 1H), 0.86 (d, <i>J</i> = 6.0 Hz, 3H), 0.70 (d, <i>J</i> = 5.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.30, 148.49, 144.06, 136.20, 129.80, 129.75, 127.32, 121.52, 116.03, 66.84, 56.16, 49.64, 42.08, 24.39, 22.89, 21.51, 21.50, 21.31. HRMS (ESI): [M + H]<sup>+</sup> C<sub>23</sub>H<sub>32</sub>N<sub>3</sub>O<sub>4</sub>S calcd 446.2114, found 446.2115; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 11.48 min).</div></div><div id="sec4_1_3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> (<i>S</i>)-4-Methyl-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-(pyrrolidin-1-yl)phenyl)pentanamide (<b>9b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>4a</b> and 4-(pyrrolidin-1-yl)aniline (<b>8b</b>) according to the general procedure as a colorless solid (0.04 g, 24%). Mp: 183–185 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.75 (d, <i>J</i> = 7.6 Hz, 2H), 7.42 (s, 1H), 7.27 (d, <i>J</i> = 8.0 Hz, 2H), 7.08 (d, <i>J</i> = 8.8 Hz, 2H), 6.45 (d, <i>J</i> = 8.8 Hz, 2H), 5.09 (d, <i>J</i> = 7.6 Hz, 1H), 3.78–3.73 (m, 1H), 3.24 (t, <i>J</i> = 6.4 Hz, 4H), 2.43 (s, 3H), 2.03–1.96 (m, 4H), 1.64–1.47 (m, 3H), 0.88 (d, <i>J</i> = 5.6 Hz, 3H), 0.75 (d, <i>J</i> = 5.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.11, 145.65, 143.87, 136.43, 129.76, 127.32, 125.44, 122.34, 111.46, 56.14, 47.75, 42.32, 25.41, 24.38, 22.94, 21.51, 21.39. HRMS (ESI): [M + H]<sup>+</sup> C<sub>23</sub>H<sub>32</sub>N<sub>3</sub>O<sub>3</sub>S calcd 430.2164, found 430.2164; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 11.83 min).</div></div><div id="sec4_1_3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> (<i>S</i>)-4-Methyl-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-(piperidin-1-yl)phenyl)pentanamide (<b>9c</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>4a</b> and 4-(piperidin-1-yl)aniline (<b>8c</b>) according to the general procedure as a colorless solid (0.06 g, 40%). Mp: 178–180 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.75 (d, <i>J</i> = 8.0 Hz, 2H), 7.59 (s, 1H), 7.25 (d, <i>J</i> = 8.0 Hz, 2H), 7.16 (d, <i>J</i> = 8.8 Hz, 2H), 6.84 (d, <i>J</i> = 8.8 Hz, 2H), 5.10 (d, <i>J</i> = 7.2 Hz, 1H), 3.77–3.74 (m, 1H), 3.09 (t, <i>J</i> = 5.6 Hz, 4H), 2.32 (s, 3H), 1.72–1.68 (m, 4H), 1.67–1.46 (m, 5H), 0.86 (d, <i>J</i> = 6.0 Hz, 3H), 0.73 (d, <i>J</i> = 5.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.24, 149.57, 144.01, 136.30, 129.80, 128.91, 127.32, 121.48, 116.81, 56.19, 50.97, 42.18, 25.79, 24.40, 24.20, 22.93, 21.52, 21.50, 21.37. HRMS (ESI): [M + H]<sup>+</sup> C<sub>24</sub>H<sub>34</sub>N<sub>3</sub>O<sub>3</sub>S calcd 444.2321, found 444.2321; purity ≥99.99% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 18.20 min).</div></div><div id="sec4_1_3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> (<i>S</i>)-4-Methyl-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-(4-methylpiperidin-1-yl)phenyl)pentanamide (<b>9d</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>4a</b> and 4-(4-methylpiperidin-1-yl)aniline (<b>8d</b>) according to the general procedure as a colorless solid (0.11 g, 72%). Mp: 184–186 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.76 (d, <i>J</i> = 8.0 Hz, 2H), 7.51 (s, 1H), 7.27 (d, <i>J</i> = 8.0 Hz, 2H), 7.17 (d, <i>J</i> = 8.8 Hz, 2H), 6.85 (d, <i>J</i> = 8.8 Hz, 2H), 4.98 (d, <i>J</i> = 8.0 Hz, 1H), 3.77–3.72 (m, 1H), 3.59–3.56 (m, 2H), 2.65–2.58 (m, 2H), 2.35 (s, 3H), 1.73–1.70 (m, 2H), 1.62–1.56 (m, 2H), 1.50–1.45 (m, 2H), 1.37–1.25 (m, 2H), 0.97 (d, <i>J</i> = 6.4 Hz, 3H), 0.84 (d, <i>J</i> = 6.0 Hz, 3H), 0.71 (d, <i>J</i> = 5.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.10, 149.23, 144.04, 136.22, 129.80, 128.83, 127.32, 121.44, 116.76, 56.15, 50.32, 42.18, 34.02, 30.61, 24.39, 22.91, 21.87, 21.51, 21.35. HRMS (ESI): [M + H]<sup>+</sup> C<sub>25</sub>H<sub>36</sub>N<sub>3</sub>O<sub>3</sub>S calcd 458.2477, found 458.2477; purity ≥99.32% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 19.59 min).</div></div><div id="sec4_1_3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> (<i>S</i>)-4-Methyl-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-thiomorpholine-1,1-dioxophenyl)pentanamide (<b>9e</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>4a</b> and 4-(4-aminophenyl)thiomorpholine 1,1-dioxide (<b>8e</b>) according to the general procedure as a colorless solid (0.12 g, 70%). Mp: 195–197 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.90 (s, 1H), 7.77 (d, <i>J</i> = 8.0 Hz, 2H), 7.33 (d, <i>J</i> = 8.8 Hz, 2H), 7.28 (d, <i>J</i> = 8.0 Hz, 2H), 6.86 (d, <i>J</i> = 8.8 Hz, 2H), 4.97 (d, <i>J</i> = 6.8 Hz, 1H), 3.82–3.76 (m, 5H), 3.10 (t, <i>J</i> = 5.2 Hz, 4H), 2.37 (s, 3H), 1.67–1.64(m, 1H), 1.55–1.45 (m, 2H), 0.84 (d, <i>J</i> = 6.4 Hz, 3H), 0.65 (d, <i>J</i> = 6.4 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.51, 144.80, 144.19, 136.02, 130.78, 129.85, 127.36, 121.86, 117.00, 56.12, 50.53, 48.10, 41.93, 24.40, 22.89, 21.54, 21.21. HRMS (ESI): [M + H]<sup>+</sup> C<sub>23</sub>H<sub>32</sub>N<sub>3</sub>O<sub>5</sub>S<sub>2</sub> calcd 494.1783, found 494.1783; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 13.21 min).</div></div><div id="sec4_1_3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> (<i>S</i>)-4-Methyl-<i>N</i>-(4-methyl-1-(4-morpholinophenylamino)-1-oxopentan-2-yl)benzamide (<b>12</b>)</h3><div class="NLM_p last">Palladium charcoal was added to the solution of <b>11</b> in methanol, and the reaction solution was stirred at room temperature under H<sub>2</sub> pressure for 3 h. After deprotection of the benzyl group from the starting material indicated by TLC, the reaction mass was filtered through Celite and concentrated. The residue was used without further purification. The title compound was prepared from the above acid and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.06 g, 24% over 2 steps). Mp: 208–210 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.73 (s, 1H), 7.69 (d, <i>J</i> = 8.0 Hz, 2H), 7.42 (d, <i>J</i> = 8.0 Hz, 2H), 7.22 (d, <i>J</i> = 8.8 Hz, 2H), 6.84–6.80 (m, 3H) 4.90–4.86 (m, 1H), 3.84 (t, <i>J</i> = 4.8 Hz, 4H), 3.08 (t, <i>J</i> = 4.8 Hz, 4H), 2.39 (s, 3H), 1.90–1.88 (m, 1H), 1.86–1.71 (m, 2H), 0.97 (d, <i>J</i> = 6.4 Hz, 3H), 0.90 (d, <i>J</i> = 6.4 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 170.73, 167.78, 140.09, 142.19, 131.08, 130.95, 129.19, 127.23, 116.18, 66.87, 53.05, 49.79, 41.41, 24.98, 22.93, 22.27, 21.44. HRMS (ESI): [M + H]<sup>+</sup> C<sub>24</sub>H<sub>32</sub>N<sub>3</sub>O<sub>3</sub> calcd 410.2444, found 410.2439; purity ≥99.74% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 11.43 min).</div></div><div id="sec4_1_3_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> (<i>S</i>)-4-Methyl-<i>N</i>-(4-morpholinophenyl)-2-(phenylsulfonamido)pentanamide (<b>14a</b>)</h3><div class="NLM_p last">The title compound was prepared from 1<b>3a</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.11 g, 69%). Mp: 217–219 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.89 (d, <i>J</i> = 7.6 Hz, 2H), 7.75 (s, 1H), 7.56 (t, <i>J</i> = 7.6 Hz, 1H), 7.49 (t, <i>J</i> = 7.6 Hz, 2H), 7.24 (d, <i>J</i> = 8.8 Hz, 2H), 6.83 (d, <i>J</i> = 8.8 Hz, 2H), 5.07 (d, <i>J</i> = 7.2 Hz, 1H), 3.85 (t, <i>J</i> = 4.8 Hz, 4H), 3.81–3.77 (m, 1H), 3.10 (t, <i>J</i> = 4.8 Hz, 4H), 1.66–1.46 (m, 3H), 0.85 (d, <i>J</i> = 6.0 Hz, 3H), 0.69 (d, <i>J</i> = 6.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.09, 147.56, 138.16, 132.15, 128.58, 128.25, 126.26, 120.56, 11.05, 65.83, 55.11, 48.60, 41.12, 28.68, 23.39, 21.89, 20.26. HRMS (ESI): [M + H]<sup>+</sup> C<sub>22</sub>H<sub>30</sub>N<sub>3</sub>O<sub>4</sub>S calcd 432.1957, found 432.1956; purity ≥98.65% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 10.31 min).</div></div><div id="sec4_1_3_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> (<i>S</i>)-2-(4-Fluorophenylsulfonamido)-4-methyl-<i>N</i>-(4-morpholinophenyl)pentanamide (<b>14b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>13b</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.05 g, 31%). Mp: 168–170 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.91–7.87 (m, 2H), 7.65 (s, 1H), 7.22 (d, <i>J</i> = 8.8 Hz, 2H), 7.13 (t, <i>J</i> = 8.4 Hz, 2H), 6.82 (d <i>J</i> = 8.8 Hz, 2H), 5.26 (d, <i>J</i> = 8.0 Hz, 1H), 3.82 (t, <i>J</i> = 4.8 Hz, 4H), 3.80–3.77 (m, 1H), 3.10 (t, <i>J</i> = 4.8 Hz, 4H), 1.63–1.57 (m, 2H), 1.53–1.48 (m, 1H), 0.87 (d, <i>J</i> = 6.4 Hz, 3H), 0.74 (d, <i>J</i> = 6.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.28, 166.50, 163.96, 148.64, 135.48, 135.45, 130.08, 129.99, 129.43, 121.61, 116.52, 116.29, 116.06, 66.81, 56.20, 49.55, 42.12, 24.41, 22.87, 21.35. HRMS (ESI): [M + H]<sup>+</sup> C<sub>22</sub>H<sub>29</sub>FN<sub>3</sub>O<sub>4</sub>S calcd 450.1863, found 450.1863; purity ≥99.15% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 11.15 min).</div></div><div id="sec4_1_3_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> (<i>S</i>)-2-(4-Bromophenylsulfonamido)-4-methyl-<i>N</i>-(4-morpholinophenyl)pentanamide (<b>14c</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>13c</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.10 g, 70%). Mp: 185–187 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.72 (d, <i>J</i> = 8.4 Hz, 2H), 7.59 (d, <i>J</i> = 8.4 Hz, 2H), 7.56 (s, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 2H), 6.82 (d, <i>J</i> = 8.4 Hz, 2H), 5.38 (d, <i>J</i> = 7.6 Hz, 1H), 3.86 (t, <i>J</i> = 4.6 Hz, 4H), 3.82–3.77 (m, 1H), 3.12 (t, <i>J</i> = 5.0 Hz, 4H), 1.67–1.50 (m, 3H), 0.88 (d, <i>J</i> = 6.4 Hz, 3H), 0.78 (d, <i>J</i> = 6.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.18, 148.70, 138.50, 132.41, 129.25, 128.74, 128.12, 121.70, 116.07, 66.82, 56.26, 49.53, 42.12, 24.41, 22.87, 21.43. HRMS (ESI): [M + H]<sup>+</sup> C<sub>22</sub>H<sub>29</sub>BrN<sub>3</sub>O<sub>4</sub>S calcd 510.1062, found 510.1064; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 12.75 min).</div></div><div id="sec4_1_3_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> (<i>S</i>)-2-(4-Methoxyphenylsulfonamido)-4-methyl-<i>N</i>-(4-morpholinophenyl)pentanamide (<b>14d</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>13d</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.12 g, 76%). Mp: 192–194 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.81 (d, <i>J</i> = 8.8 Hz, 2H), 7.74 (s, 1H), 7.25 (d, <i>J</i> = 8.4 Hz, 2H), 6.92 (d, <i>J</i> = 8.8 Hz, 2H), 6.83 (d, <i>J</i> = 8.4 Hz, 2H), 5.01 (d, <i>J</i> = 6.8 Hz, 1H), 3.85 (t, <i>J</i> = 4.8 Hz, 4H), 3.79 (, 3H), 3.77–3.71 (m, 1H), 3.10 (t, <i>J</i> = 4.8 Hz, 4H), 1.66–1.60 (m, 2H), 1.51–1.47 (m, 1H), 0.87 (d, <i>J</i> = 6.4 Hz, 3H), 0.71 (d, <i>J</i> = 5.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.37, 163.23, 148.48, 130.60, 129.80, 129.50, 121.47, 116.04, 114.36, 66.84, 56.24, 55.62, 49.63, 42.08, 24.41, 22.91, 21.32. HRMS (ESI): [M + H]<sup>+</sup> C<sub>23</sub>H<sub>32</sub>N<sub>3</sub>O<sub>5</sub>S calcd 462.2063, found 462.2071; purity ≥99.16% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 10.57 min).</div></div><div id="sec4_1_3_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> (<i>S</i>)-2-(Biphenyl-4-ylsulfonamido)-4-methyl-<i>N</i>-(4-morpholinophenyl)pentanamide (<b>14e</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>13e</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.12 g, 80%). Mp: 213-215 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.93 (d, <i>J</i> = 8.0 Hz, 2H), 7.67 (s, 1H), 7.63 (d, <i>J</i> = 8.4 Hz, 2H), 7.46–7.39 (m, 5H), 7.18 (d, <i>J</i> = 8.8 Hz, 2H), 6.74 (d, <i>J</i> = 8.4 Hz, 2H), 5.27 (d, <i>J</i> = 8.0 Hz, 1H), 3.87–382 (m, 5H), 3.03 (t, <i>J</i> = 4.8 Hz, 4H), 1.66–1.60 (m, 2H), 1.56–1.51 (m, 1H), 0.88 (d, <i>J</i> = 5.6 Hz, 3H), 0.76 (d, <i>J</i> = 5.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.19, 148.44, 146.01, 139.10, 137.78, 129.54, 128.95, 128.50, 127.80, 127.78, 127.33, 121.45, 115.97, 66.82, 56.31, 49.48, 42.14, 24.44, 22.91, 21.40. HRMS (ESI): [M + H]<sup>+</sup> C<sub>28</sub>H<sub>34</sub>N<sub>3</sub>O<sub>4</sub>S calcd 508.2270, found 508.2271; purity ≥98.94% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 15.08 min).</div></div><div id="sec4_1_3_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> (<i>S</i>)-4-Methyl-<i>N</i>-(4-morpholinophenyl)-2-(4-nitrophenylsulfonamido)pentanamide (<b>14f</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>13g</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as yellowish solid (0.03g, 21%). Mp: 215–217 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.22 (d, <i>J</i> = 8.8 Hz, 2H), 8.02 (d, <i>J</i> = 8.8 Hz, 2H), 7.32 (s, 1H), 7.122 (d, <i>J</i> = 8.8 Hz, 2H), 6.78 (d, <i>J</i> = 8.8 Hz, 2H), 5.55 (d, <i>J</i> = 7.2 Hz, 1H), 3.91–3.84 (m, 5H), 3.09 (t, <i>J</i> = 4.8 Hz, 4H), 1.76–1.70 (m, 1H), 1.61–1.51 (m, 2H), 0.93 (d, <i>J</i> = 6.8 Hz, 3H), 0.87 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>,) δ 167.51, 148.93, 147.89, 144.35, 127.77, 127.44, 123.26, 120.29, 114.95, 65.78, 55.45, 48.33, 41.19, 28.69, 23.40, 21.81, 20.56. HRMS (ESI): [M + H]<sup>+</sup> C<sub>22</sub>H<sub>29</sub>N<sub>4</sub>O<sub>6</sub>S calcd 477.1808, found 477.1808; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 11.86 min).</div></div><div id="sec4_1_3_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 2-(4-Methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)acetamide (<b>17a</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16a</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.09 g, 55%). Mp: 182–184 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.96 (s, 1H), 7.76 (d, <i>J</i> = 8.4 Hz, 2H), 7.35–7.32 (m, 4H), 6.85 (d, <i>J</i> = 8.8 Hz, 2H), 5.31 (t, <i>J</i> = 6.8 Hz, 1H), 3.85 (t, <i>J</i> = 4.8 Hz, 4H), 3.69 (d, <i>J</i> = 6.4 Hz, 2H), 3.11 (t, <i>J</i> = 4.8 Hz, 4H), 2.42 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 165.61, 148.65, 144.40, 135.40, 130.05, 129.43, 127.27, 121.54, 116.10, 66.84, 49.53, 46.49, 21.56. HRMS (ESI): [M + H]<sup>+</sup> C<sub>19</sub>H<sub>24</sub>N<sub>3</sub>O<sub>4</sub>S calcd 390.1488, found 390.1485; purity ≥99.8% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 17.55 min).</div></div><div id="sec4_1_3_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> (<i>S</i>)-2-(4-Methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)propanamide (<b>17b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16b</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.10 g, 60%). Mp: 215–217 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.94 (s, 1H), 7.77 (d, <i>J</i> = 8.4 Hz, 2H), 7.32–7.30 (m, 4H), 6.85 (d, <i>J</i> = 8.4 Hz, 2H), 5.04 (d, <i>J</i> = 7.6 Hz, 1H), 3.89–3.85 (m, 5H), 3.11 (t, <i>J</i> = 5.2 Hz, 4H), 2.40(s, 3H), 1.31 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.09, 148.50, 144.22, 136.09, 129.96, 129.76, 127.23, 121.44, 116.07, 66.83, 53.15, 49.61, 21.54, 18.86. HRMS (ESI): [M + H]<sup>+</sup> C<sub>20</sub>H<sub>26</sub>N<sub>3</sub>O<sub>4</sub>S calcd 404.1644, found 404.1643; purity ≥99.99% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 17.87 min).</div></div><div id="sec4_1_3_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> (<i>S</i>)-3-Hydroxy-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)propanamide (<b>17c</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16c</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.10 g, 61%). Mp: 208–210 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.40 (s, 1H), 7.78 (d, <i>J</i> = 8.0 Hz, 2H), 7.37–7.32 (m, 4H), 6.86 (d, <i>J</i> = 8.4 Hz, 2H), 5.70 (d, <i>J</i> = 6.8 Hz, 1H), 4.10 (d, <i>J</i> = 10.4 Hz, 1H), 3.85–3.87 (m, 5H), 3.39–3.36 (m, 1H), 3.12 (t, <i>J</i> = 4.8 Hz, 4H), 2.43 (s, 3H), 2.30 (s, 1H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 167.28, 148.67, 144.43, 135.87, 130.89, 129.50, 127.24, 121.49, 116.06, 66.83, 62.12, 57.57, 49.54. HRMS (ESI): [M + H]<sup>+</sup> C<sub>20</sub>H<sub>26</sub>N<sub>3</sub>O<sub>5</sub>S calcd 420.1593, found 420.1593; purity ≥95.59% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 15.24 min).</div></div><div id="sec4_1_3_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> (<i>S</i>)-3-Methyl-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)butanamide (<b>17d</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16d</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.10 g, 63%). Mp: 240–242 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.74 (d, <i>J</i> = 7.2 Hz, 2H), 7.51 (s, 1H), 7.25 (d, <i>J</i> = 8.0 Hz, 2H), 7.19 (d, <i>J</i> = 8.0 Hz, 2H), 6.82 (d, <i>J</i> = 8.8 Hz, 2H), 5.15 (d, <i>J</i> = 6.8 Hz, 1H), 3.85 (t, <i>J</i> = 4.4 Hz, 4H), 3.55(t, <i>J</i> = 5.6 Hz, 1H), 3.10 (t, <i>J</i> = 4.8 Hz, 4H), 2.35 (s, 3H), 2.21–2.18 (m, 1H), 0.90 (d, <i>J</i> = 6.8 Hz, 3H), 0.83 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 168.29, 148.64, 144.05, 136.08, 129.76, 129.45, 127.38, 121.62, 116.06, 66.83, 62.81, 49.63, 31.24, 21.53, 19.12, 17.36. HRMS (ESI): [M + H]<sup>+</sup> C<sub>22</sub>H<sub>30</sub>N<sub>3</sub>O<sub>4</sub>S calcd 432.1957, found 432.1946; purity ≥99.99% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 21.16 min).</div></div><div id="sec4_1_3_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> (2<i>S</i>,3<i>S</i>)-3-Hydroxy-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)butanamide (<b>17e</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16e</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.07 g, 42%). Mp: 202–203 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.42 (s, 1H), 7.78 (d, <i>J</i> = 7.2 Hz, 2H), 7.32–7.26 (m, 4H), 6.84 (d, <i>J</i> = 7.2 Hz, 2H), 5.80 (d, <i>J</i> = 5.6 Hz, 1H), 4.43 (brs, 1H), 3.85 (t, <i>J</i> = 4.4 Hz, 4H), 3.75 (d, <i>J</i> = 5.6 Hz, 1H), 3.11 (t, <i>J</i> = 4.4 Hz, 4H), 2.79 (s, 1H), 2.41 (s, 3H), 0.94 (d, <i>J</i> = 5.2 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.37, 148.59, 144.18, 136.06, 129.93, 129.59, 127.22, 121.47, 116.06, 67.64, 66.82, 60.80, 49.56, 21.55, 18.09. HRMS (ESI): [M + H]<sup>+</sup> C<sub>21</sub>H<sub>28</sub>N<sub>3</sub>O<sub>5</sub>S calcd 434.1750, found 434.1745; purity ≥98.74% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 16.15 min).</div></div><div id="sec4_1_3_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> (<i>S</i>)-2-(4-Methylphenylsulfonamido)-4-(methylthio)-<i>N</i>-(4-morpholinophenyl)butanamide (<b>17f</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16f</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.07 g, 45%). Mp: 200–202 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.92 (s, 1H), 7.77 (d, <i>J</i> = 8.0 Hz, 2H), 7.30–7.26 (m, 4H), 6.84 (d, <i>J</i> = 8.4 Hz, 2H), 5.84 (d, <i>J</i> = 8.4 Hz, 1H), 4.03–3.98 (m, 1H), 3.85 (t, <i>J</i> = 4.8 Hz, 4H), 3.11 (t, <i>J</i> = 4.4 Hz, 4H), 2.53–2.47 (m, 1H), 2.44–2.41 (m, 1H), 2.39 (s, 3H), 2.03 (s, 3H), 1.99–1.95 (m, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 168.19, 148.61, 144.17, 136.25, 129.90, 129.55, 127.30, 121.48, 116.06, 66.83, 56.59, 49.58, 31.17, 30.04, 21.56, 15.16. HRMS (ESI): [M + H]<sup>+</sup> C<sub>22</sub>H<sub>30</sub>N<sub>3</sub>O<sub>4</sub>S<sub>2</sub> calcd 464.1678, found 464.1669; purity ≥99.06% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 21.91 min).</div></div><div id="sec4_1_3_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> (<i>S</i>)-3-(4-Methylphenylsulfonamido)-4-(4-morpholinophenylamino)-4-oxobutanoic Acid (<b>17g</b>)</h3><div class="NLM_p last">A general amide coupling reaction was used to prepare intermediate (<i>S</i>)-benzyl 3-(4-methylphenylsulfonamido)-4-(4-morpholinophenylamino)-4-oxobutanoate from <b>16g</b> and 4-morpholinoaniline (<b>8a</b>) as a colorless solid. To the solution of the above prepared intermediate (1 equiv) in MeOH/EA palladium/charcoal (0.1 equiv) was added, and the reaction mixture was stirred at room temperature under H<sub>2</sub> pressure for 3 h. The reaction solution was filtered through Celite and the solvent was evaporated to yield the title compound as a colorless solid (0.13 g, 55% over 2 steps). Mp: 166–168 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.77 (s, 1H), 7.65–7.20 (m, 4H), 7.21 (d, <i>J</i> = 4.0 Hz, 2H), 6.82 (d, <i>J</i> = 8.8 Hz, 2H), 4.17 (t, <i>J</i> = 7.0 Hz, 1H), 3.72 (t, <i>J</i> = 4.4 Hz, 4H), 3.02 (t, <i>J</i> = 4.6 Hz, 4H), 2.57 (dd, <i>J</i> = 16.2, 6.6 Hz, 1H), 2.35 (dd, <i>J</i> = 16.0, 7.6 Hz, 1H), 2.25 (s, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.53, 167.93, 147.78, 142.88, 138.51, 131.19, 129.66, 127.05, 120.99, 115.59, 66.53, 54.29, 49.40, 38.09, 21.38. HRMS (ESI): [M + H]<sup>+</sup> C<sub>21</sub>H<sub>26</sub>N<sub>3</sub>O<sub>6</sub>S calcd 448.1542, found 448.1541; purity ≥99.99% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 16.29 min).</div></div><div id="sec4_1_3_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> (<i>S</i>)-4-(4-Methylphenylsulfonamido)-5-(4-morpholinophenylamino)-5-oxopentanoic Acid (<b>17h</b>)</h3><div class="NLM_p last">A general amide coupling reaction was used to prepare intermediate (<i>S</i>)-benzyl 4-(4-methylphenylsulfonamido)-5-(4-morpholinophenylamino)-5-oxopentanoate from <b>16h</b> and 4-morpholinoaniline (<b>8a)</b> as a colorless solid. To the solution of above prepared intermediate (1 equiv) in MeOH/EA palladium/charcoal (0.1 equiv) was added, and the reaction mixture was stirred at room temperature under H<sub>2</sub> pressure for 3 h. The reaction solution was filtered through Celite, and the solvent was evaporated to yield the title compound as white solid (0.12 g, 52%). Mp: 139–141 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.67 (s, 1H), 7.65 (d, <i>J</i> = 8.4 Hz, 2H), 7.25 (d, <i>J</i> = 7.6 Hz, 2H), 7.19 (d, <i>J</i> = 8.8 Hz, 2H), 6.84 (d, <i>J</i> = 8.4 Hz, 2H), 3.83–3.79 (m, 1H), 3.72 (t, <i>J</i> = 4.6 Hz, 4H), 3.02 (t, <i>J</i> = 4.8 Hz, 4H), 2.26 (s, 3H), 2.20–2.11 (m, 2H), 1.80–1.70 (m, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 174.06, 168.74, 147.88, 142.92, 138.48, 130.98, 129.71, 126.97, 121.09, 115.68, 66.53, 56.73, 49.37, 30.53, 28.58, 21.37. HRMS (ESI): [M + H]<sup>+</sup> C<sub>22</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S calcd 462.1699, found 462.1697; purity ≥99.14% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 16.43 min).</div></div><div id="sec4_1_3_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> (<i>S</i>)-2-(4-Methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)-3-phenylpropanamide (<b>17i</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16i</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.10 g, 65%). Mp: 223–225 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.83 (s, 1H), 7.57 (d, <i>J</i> = 7.6 Hz, 2H), 7.26–7.17 (m, 7H), 6.97 (d, <i>J</i> = 6.8 Hz, 2H), 6.83 (d, <i>J</i> = 8.4 Hz, 2H), 5.03 (d, <i>J</i> = 6.8 Hz, 1H), 3.98–3.93 (m, 1H), 3.85 (t, <i>J</i> = 4.4 Hz, 4H), 3.10 (t, <i>J</i> = 4.4 Hz, 4H), 3.09–3.04 (m, 1H), 2.969–2.91 (m, 1H), 2.40 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 168.00, 148.59, 144.04, 135.53, 135.22, 129.88, 129.49, 129.21, 128.99, 127.31, 127.19, 121.69, 116.05, 66.84, 58.56, 49.59, 38.41, 21.57. HRMS (ESI): [M + H]<sup>+</sup> C<sub>26</sub>H<sub>30</sub>N<sub>3</sub>O<sub>4</sub>S calcd 480.1957, found 480.1956; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 12.01 min).</div></div><div id="sec4_1_3_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> (<i>S</i>)-3-(4-Hydroxyphenyl)-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)propanamide (<b>17j</b>)</h3><div class="NLM_p last">A general amide coupling reaction was used to prepare intermediate (<i>S</i>)-4-(2-(4-methylphenylsulfonamido)-3-(4-morpholinophenylamino)-3-oxopropyl)phenyl 4-methylbenzenesulfonate from <b>16j</b> and 4-morpholinoaniline (<b>8a</b>) as a colorless solid. To the solution of the above prepared intermediate (1 equiv) in ethanol was added sodium hydroxide (5 equiv), and the solution was refluxed for 4 h. The reaction mixture was cooled down to room temperature, and the ethanol was evaporated under reduced pressure. The residue was acidified with 1.5 N HCl and extracted with ethyl acetate. The combined organic fraction was dried over magnesium sulfate, filtered, and concentrated. The residue was purified through flash column chromatography eluting with 0–5% MeOH in DCM to yield the title compound as a white solid (0.03 g, 40%). Mp: 224–226 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.61 (s, 1H), 9.18 (s, 1H), 8.05 (s, 1H), 7.49 (d, <i>J</i> = 8.0 Hz, 2H), 7.15 (d, <i>J</i> = 8.4 Hz, 4H), 6.93 (d, <i>J</i> = 7.6 Hz, 2H), 6.82 (d, <i>J</i> = 9.2 Hz, 2H), 6.57 (d, <i>J</i> = 8.8 Hz, 2H), 3.98–3.85 (m, 1H), 3.72 (t, <i>J</i> = 4.6 Hz, 4H), 3.01 (t, <i>J</i> = 4.6 Hz, 4H), 2.78–2.73 (m, 1H), 2.67–2.59 (m, 1H), 2.33 (s, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.85, 156.33, 147.79, 142.60, 138.63, 131.05, 130.62, 129.51, 127.39, 126.84, 121.06, 115.65, 115.25, 66.53, 59.01, 49.37, 21.37. HRMS (ESI): [M + H]<sup>+</sup> C<sub>26</sub>H<sub>30</sub>N<sub>3</sub>O<sub>5</sub>S calcd 496.1906, found 496.1898; purity ≥99.99% (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 19.93 min).</div></div><div id="sec4_1_3_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> (<i>S</i>)-3-(1<i>H</i>-Indol-2-yl)-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)propanamide (<b>BC-DXI-495</b>, <b>2</b>)</h3><div class="NLM_p last">See ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>.</div></div><div id="sec4_1_3_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> (<i>S</i>)-3-(1<i>H</i>-Indol-3-yl)-2-(2-methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)propanamide (<b>19a</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>18a</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.08 g, 60%). Mp: 125–127 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.08 (s, 1H), 7.88 (d, <i>J</i> = 8.0 Hz, 1H), 7.71 (s, 1H), 7.34 (d, <i>J</i> = 7.2 Hz, 3H), 7.24–7.18 (m, 4H), 7.05–6.96 (m, 3H), 6.82 (d, <i>J</i> = 8.8 Hz, 2H), 5.23 (d, <i>J</i> = 5.2 Hz, 1H), 3.98–3.95 (m, 1H), 3.85 (t, <i>J</i> = 4.8 Hz, 4H), 3.23 (d, <i>J</i> = 6.8 Hz, 2H), 3.10 (t, <i>J</i> = 4.8 Hz, 4H), 2.24 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 168.65, 148.55, 136.99, 136.34, 135.89, 133.19, 132.64, 129.72, 129.53, 126.72, 126.09, 123.49, 122.62, 121.72, 120.15, 118.41, 116.06, 111.32, 109.32, 66.84, 57.47, 49.49, 29.03, 19.97. HRMS (ESI): [M + H]<sup>+</sup> C<sub>28</sub>H<sub>31</sub>N<sub>4</sub>O<sub>4</sub>S calcd 519.2066, found 519.2066; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 11.94 min).</div></div><div id="sec4_1_3_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> (<i>S</i>)-3-(1<i>H</i>-Indol-3-yl)-2-(3-methylphenylsulfonamido)-<i>N</i>-(4-morpholinophenyl)propanamide (<b>19b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>18b</b> and 4-morpholinoaniline (<b>8a</b>) according to the general procedure as a colorless solid (0.07 g, 55%). Mp: 210–212 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.77 (s, 1H), 9.73 (s, 1H), 8.14 (s, 1H), 7.46–7.40 (m, 3H), 7.29–7.21 (m, 5H), 7.07–7.01 (m, 2H), 6.94 (d, <i>J</i> = 8.0 Hz, 1H), 6.82 (d, <i>J</i> = 8.8 Hz, 2H), 4.10–4.05 (m,1H), 3.72 (t, <i>J</i> = 4.4 Hz, 4H), 3.07–3.05 (m, 1H), 3.02 (t, <i>J</i> = 4.4 Hz, 4H), 2.87–2.83 (m, 1H), 2.16 (s, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.23, 147.81, 141.40, 138.67, 136.36, 132.98, 131.16, 128.91, 127.58, 127.04, 124.30, 123.94, 121.26, 121.00, 118.71, 118.63, 115.62, 111.69, 109.59, 66.53, 58.08, 49.36, 29.08, 21.15. HRMS (ESI): [M + H]<sup>+</sup> C<sub>28</sub>H<sub>31</sub>N<sub>4</sub>O<sub>4</sub>S calcd 519.2066, found 519.2066; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 11.98 min).</div></div><div id="sec4_1_3_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> (<i>S</i>)-3-(1<i>H</i>-Indol-3-yl)-2-(4-methylphenylsulfonamido)-<i>N</i>-phenylpropanamide (<b>20a</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16k</b> and aniline according to the general procedure as a colorless solid (0.04 g, 35%). Mp: 186–188 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.14 (s, 1H), 8.10 (s, 1H), 7.45 (d, <i>J</i> = 8.4 Hz, 2H), 7.34 (d, <i>J</i> = 7.6 Hz, 2H), 7.30–7.24 (m, 4H), 7.17–7.11 (m, 2H), 7.09–6.92 (m, 3H), 6.91 (s, 1H), 5.19 (d, <i>J</i> = 5.6 Hz, 1H), 4.05–4.01 (m, 1H), 3.20–3.17 (m, 2H), 2.29 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 168.86, 144.03, 137.00, 136.33, 134.91, 129.63, 128.89, 126.95, 126.69, 124.71, 123.49, 120.18, 119.99, 118.41, 111.35, 109.13, 57.43, 28.45, 21.51. HRMS (ESI): [M + H]<sup>+</sup> C<sub>24</sub>H<sub>24</sub>N<sub>3</sub>O<sub>3</sub>S calcd 434.1538, found 434.1538; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 16.45 min).</div></div><div id="sec4_1_3_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> (<i>S</i>)-2-(4-Bromophenylsulfonamido)-3-(1<i>H</i>-indol-3-yl)-<i>N</i>-phenylpropanamide (<b>21a</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>18c</b> and aniline according to the general procedure as a colorless solid (0.08 g, 40%). Mp: 215–217 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.78 (s, 1H), 9.98 (s, 1H), 8.40 (s, 1H), 7.40–7.39 (m, 5H), 7.36 (d, <i>J</i> = 8.4 Hz, 2H), 7.30–7.23 (m, 3H), 7.10 (s, 1H), 7.06–7.01 (m, 2H), 6.93 (t, <i>J</i> = 7.2 Hz, 1H), 4.16–4.13 (m, 1H), 3.05 (dd, <i>J</i> = 14.2, 6 Hz, 1H), 2.87 (dd,, <i>J</i> = 14.7, 8 Hz, 1H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.75, 140.44, 138.78, 136.38, 132.12, 131.86, 128.97, 128.55, 127.44, 126.22, 124.49,123.95, 121.27, 119.92, 118.73, 118.62, 111.76, 109.38, 58.08, 28.95. HRMS (ESI): [M + H]<sup>+</sup> C<sub>23</sub>H<sub>21</sub>BrN<sub>3</sub>O<sub>3</sub>S calcd 498.0487, found 498.0487; purity ≥96.56% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 19.59 min).</div></div><div id="sec4_1_3_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> (<i>S</i>)-3-(1<i>H</i>-Indol-3-yl)-2-(4-methylphenylsulfonamido)-<i>N</i>-(thiophen-2-ylmethyl)propanamide (<b>20b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16k</b> and thiophen-2-ylmethanamine according to the general procedure as a colorless solid (0.09 g, 66%). Mp: 158–160 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.20 (s, 1H), 7.41 (d, <i>J</i> = 8.4 Hz, 2H), 7.28–7.24 (m, 2H), 7.16–7.11 (m, 2H), 7.01–6.88 (m, 3H), 6.87 (s, 1H), 6.81 (s, 1H), 6.74 (s, 1H), 5.13 (d, <i>J</i> = 6.0 Hz, 1H), 4.48–4.45 (m, 2H), 3.94–3.91 (m, 1H), 3.11–3.09 (m, 2H), 2.31 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 170.55, 143.79, 140.17, 136.31, 135.18, 129.55, 126.85, 126.76, 126.09, 125.12, 123.52, 122.35, 119.86, 118.41, 111.30, 109.03, 56.85, 38.35, 28.44, 21.55. HRMS (ESI): [M + H]<sup>+</sup> C<sub>23</sub>H<sub>24</sub>N<sub>3</sub>O<sub>3</sub>S<sub>2</sub> calcd 454.1259, found 454.1270; purity ≥99.41% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 15.40 min).</div></div><div id="sec4_1_3_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> (<i>S</i>)-2-(4-Bromophenylsulfonamido)-3-(1<i>H</i>-indol-3-yl)-<i>N</i>-(thiophen-2-ylmethyl)propanamide (<b>21b</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>18c</b> and thiophen-2-ylmethanamine according to the general procedure as a colorless solid (0.08 g, 60%). Mp: 170–170 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.73 (s, 1H), 8.62 (t, <i>J</i> = 6.0 Hz, 1H), 8.22 (s, 1H), 7.42–7.33 (m, 6H), 7.27 (d, <i>J</i> = 8.4 Hz, 1H), 7.05–7.01 (m, 2H), 6.94–6.88 (m, 2H), 6.86 (d, <i>J</i> = 2.4 Hz, 1H), 4.30–4.26 (m, 2H), 3.99–3.95 (m, 1H), 2.99 (dd, <i>J</i> = 14.2, 6.0 Hz, 1H), 2.80 (dd,, <i>J</i> = 14.7, 8 Hz, 1H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.92, 142.19, 140.45, 136.42, 131.74, 128.36, 127.33, 127.03, 126.07, 125.71, 125.41, 124.60, 121.16, 118.54, 111.17, 109.41, 51.31, 37.66, 29.12. HRMS (ESI): [M + H]<sup>+</sup> C<sub>22</sub>H<sub>21</sub>BrN<sub>3</sub>O<sub>3</sub>S<sub>2</sub> calcd 518.0208, found 518.0223; purity ≥98.48% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 16.60 min).</div></div><div id="sec4_1_3_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> (<i>S</i>)-3-(1<i>H</i>-Indol-3-yl)-2-(4-methylphenylsulfonamido)-<i>N</i>-(4-methylthiazol-2-yl)propanamide (<b>20c</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16k</b> and 4-methylthiazol-2-amine according to the general procedure as a colorless solid (0.08 g, 60%). Mp: 299–301 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.11 (s, 1H), 10.84 (s, 1H), 8.24 (s, 1H), 7.44 (d, <i>J</i> = 8.4 Hz, 2H), 7.38 (d, <i>J</i> = 7.6 Hz, 1H), 7.27 (d, <i>J</i> = 8.0 Hz, 1H), 7.06–7.00 (m, 4H), 6.89 (t, <i>J</i> = 7.6 Hz, 1H), 6.71 (s, 1H), 4.24–4.19 (m, 1H), 3.08–3.02 (m, 1H), 2.87–2.81 (m, 1H), 2.23 (s, 3H), 2.21 (s, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.12, 157.14, 147.01, 142.66, 137.91, 136.39, 129.39, 127.41, 126.75, 124.41, 121.21, 118.74, 118.57, 111.67, 108.93, 108.29, 57.02, 28.79, 21.33, 17.28. HRMS (ESI): [M + H]<sup>+</sup> C<sub>22</sub>H<sub>23</sub>N<sub>4</sub>O<sub>3</sub>S<sub>2</sub> calcd 455.1212, found 455.1234; purity ≥99.16% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 14.99 min).</div></div><div id="sec4_1_3_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> (<i>S</i>)-2-(4-Bromophenylsulfonamido)-3-(1<i>H</i>-indol-3-yl)-<i>N</i>-(4-methylthiazol-2-yl)propanamide (<b>21c</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>18c</b> and 4-methylthiazol-2-amine according to the general procedure as a colorless solid (0.10 g, 92%). Mp: 303–305 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.21 (s, 1H), 10.85 (s, 1H), 8.53 (d, <i>J</i> = 8.5 Hz, 1H), 7.44 (d, <i>J</i> = 8 Hz, 1H), 7.44–7.36 (m, 4H), 7.27 (d, <i>J</i> = 8 Hz, 1H), 7.07–7.02 (m, 2H), 6.90 (t, <i>J</i> = 8.4 Hz, 1H), 6.73 (s, 1H), 4.29–4.25 (m, 1H), 3.07 (dd, <i>J</i> = 14.2, 6 Hz, 1H), 2.87 (dd,, <i>J</i> = 14.7, 8 Hz, 1H), 2.25 (s, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.89, 157.01, 147.10, 139.96, 136.39, 131.85, 128.49, 127.30, 126.29, 124.64, 121.24, 118.77, 118.58, 111.76, 108.84, 104.45, 56.98, 28.76, 17.31. HRMS (ESI): [M + H]<sup>+</sup> C<sub>21</sub>H<sub>20</sub>BrN<sub>4</sub>O<sub>3</sub>S<sub>2</sub> calcd 519.0160, found 519.0176; purity ≥98.96% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 16.20 min).</div></div><div id="sec4_1_3_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Synthesis of Compounds <b>20d</b>, <b>21d</b>, <b>21e</b>, and <b>21f</b></h3><div class="NLM_p last">EDCI·HCl (1.1 equiv) and HOBt (1.1 equiv) were added to a stirred solution of carboxylic acid (1 equiv), amine (1 equiv) and DIPEA (2.5 equiv) in DMF at room temperature, and the reaction mixture was stirred at 60 °C for 16 h. The reaction mass was cooled to room temperature, acidified with 1.5 N HCl and extracted with ethyl acetate. The combined organic layer was then washed with saturated sodium bicarbonate, brine and concentrated under reduced pressure. The residue was purified through flash column chromatography (hexane/ethyl acetate).</div></div><div id="sec4_1_3_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> (<i>S</i>)-3-(1<i>H</i>-Indol-3-yl)-<i>N</i>-(4-(4-methoxyphenyl)thiazol-2-yl)-2-(4-methylphenylsulfonamido)propanamide (<b>BC-DXI-843</b>, <b>20d</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>16k</b> and 4-(4-methoxyphenyl)thiazol-2-amine as a colorless solid (0.13 g, 65%). Mp: 249–251 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.27 (s, 1H), 10.77 (s, 1H), 8.21 (d, <i>J</i> = 8.4 Hz, 1H), 7.79 (d, <i>J</i> = 8.4 Hz, 2H), 7.45 (d, <i>J</i> = 8.0 Hz, 2H), 7.41 (d, <i>J</i> = 8.0 Hz, 2H), 7.27 (d, <i>J</i> = 8.0 Hz, 1H), 7.06–6.97 (m, 6H), 6.90 (t, <i>J</i> = 7.6 Hz, 1H), 4.33–4.27 (m, 1H), 3.79 (s, 3H), 3.11–3.05 (m, 1H), 2.88–2.83 (m, 1H), 2.18 (s, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.39, 159.41, 157.63, 149.14, 142.68, 137.90, 136.42, 129.41, 127.56, 127.43, 127.41, 126.79, 124.48, 121.26, 118.82, 118.61, 114.55, 111.66, 108.95, 106.69, 56.88, 55.56, 28.80, 21.33. HRMS (ESI): [M + H]<sup>+</sup> C<sub>28</sub>H<sub>27</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub> calcd 547.1474, found 547.1474; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 19.53 min).</div></div><div id="sec4_1_3_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> (<i>S</i>)-2-(4-Bromophenylsulfonamido)-3-(1<i>H</i>-indol-3-yl)-<i>N</i>-(4-(4-methoxyphenyl)thiazol-2-yl)propanamide (<b>21d</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>18c</b> and 4-(4-methoxyphenyl)thiazol-2-amine as a colorless solid (0.12 g, 62%). Mp: 260–262 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.40 (s, 1H), 10.77 (s, 1H), 8.48 (s, 1H), 7.81 (d, <i>J</i> = 8.8 Hz, 2H), 7.48–7.37 (m, 5H), 7.27 (d, <i>J</i> = 8.0 Hz, 1H), 7.08–6.97 (m, 5H), 6.92 (t, <i>J</i> = 8.0 Hz, 1H), 4.34–4.31 (m, 1H), 3.79 (s, 3H), 3.12–3.07 (m, 1H), 2.91–2.85 (m, 1H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.34, 159.41, 157.59, 149.21, 139.95, 136.41, 131.85, 128.51, 127.56, 127.44, 127.31, 126.32, 124.69, 121.28, 118.84, 118.60, 114.53, 111.75, 108.86, 106.83, 56.84, 54.41, 28.77. HRMS (ESI): [M + H]<sup>+</sup> C<sub>27</sub>H<sub>24</sub>BrN<sub>4</sub>O<sub>4</sub>S<sub>2</sub> calcd 611.0422, found 611.0422; purity ≥99.88% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 20.42 min).</div></div><div id="sec4_1_3_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> (<i>S</i>)-2-(4-Bromophenylsulfonamido)-<i>N</i>-(4-(2,4-dimethoxyphenyl)thiazol-2-yl)-3-(1<i>H</i>-indol-3-yl)propanamide (<b>21e</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>18c</b> and 4-(2,4-dimethoxyphenyl)thiazol-2-amine as a colorless solid (0.05 g, 31%). Mp: 165–168 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 1H), 10.76 (s, 1H), 8.48 (s, 1H), 7.81 (d, <i>J</i> = 8.8 Hz, 2H), 7.96 (d, <i>J</i> = 8.8 Hz, 1H), 7.47 (s, 2H), 7.46–7.40 (m, 4H), 7.37 (d, <i>J</i> = 9.2 Hz, 1H), 7.07–7.02 (m, 2H), 6.92 (d, <i>J</i> = 8.8 Hz, 1H), 6.89–6.60 (m, 2H), 4.34–4.30 (m, 1H), 3.90 (s, 3H), 3.80 (s, 3H), 3.12–3.07 (m, 1H), 2.91–2.85 (m, 1H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.29, 160.41, 158.19, 155.95, 145.30, 139.98, 136.41, 131.82, 130.11, 128.416, 127.31, 126.32, 124.73, 121.28, 118.85, 118.60, 116.26, 111.76, 110.21, 108.88, 105.49, 99.16, 56.86, 55.97, 55.72, 28.77. HRMS (ESI): [M + H]<sup>+</sup> C<sub>28</sub>H<sub>26</sub>BrN<sub>4</sub>O<sub>5</sub>S<sub>2</sub> calcd 641.0528, found 641.0524; purity ≥99.99% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 21.01 min).</div></div><div id="sec4_1_3_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> (<i>S</i>)-2-(4-Bromophenylsulfonamido)-3-(1<i>H</i>-indol-3-yl)-<i>N</i>-(4-(2-methoxyphenyl)thiazol-2-yl)propanamide (<b>21f</b>)</h3><div class="NLM_p last">The title compound was prepared from <b>18c</b> and 4-(2-methoxyphenyl)thiazol-2-amine as a colorless solid (0.09 g, 75%). Mp: 196–198 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.39 (s, 1H), 10.76 (s, 1H), 8.48 (s, 1H), 7.64 (s, 1H), 7.47 (d, <i>J</i> = 7.6 Hz, 1H), 7.41–7.11 (m, 6H), 7.08–7.01 (m, 5H), 6.91(t, <i>J</i> = 7.6 Hz, 1H), 4.34–4.30 (m, 1H), 3.91 (s, 3H), 3.12–3.07 (m, 1H), 2.91–2.85 (m, 1H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.38, 157.06, 156.11, 145.24, 139.98, 136.43, 131.83, 129.27, 128.48, 127.32, 126.34, 124.74, 122.99, 121.29, 120.88, 118.84, 118.62, 112.52, 112.12, 111.77, 108.88, 56.88, 55.91, 28.78. HRMS (ESI): [M + H]<sup>+</sup> C<sub>27</sub>H<sub>24</sub>BrN<sub>4</sub>O<sub>4</sub>S<sub>2</sub> calcd 611.0422, found 611.0404; purity ≥99.15% (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 21.07 min).</div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Biology. Cell Culture and Materials</h3><div class="NLM_p last">A549, H460, and CHO cells were cultured in RPMI media supplemented with 10% FBS and 1% penicillin/streptomycin in 5% CO<sub>2</sub> at 37 °C. WI-26 cells were cultivated in DMEM media using the same conditions as above. All cell lines, antibodies specific for DX2 and AIMP2, and purified DX2 proteins were kindly provided by Biobank of Biocon (Medicinal Bioconvergence Research Center, Seoul National University). Antibodies specific for Siah1 (no. ab2237) and actin (no. A1978) were purchased from Abcam and Sigma, respectively. Anti-ubiquitin (no. sc-8017) and anti-HSP70 (no. sc-29352) antibodies were purchased from Santa Cruz Biotechnology. MG-132 (no. 474790) and doxycycline (no. D3447) were purchased from Millipore and Sigma, respectively.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Screening</h3><div class="NLM_p last">A549 cells (4 × 10<sup>4</sup> cells/mL) expressing nanoluciferase-tagged DX2 were seeded in 96-well, flat-bottom white plates (no. 3903 Corning) and incubated for 12 h. The cells were treated with the synthetic compounds (40 μM), individually, for 4 h, and luminescence was measured according to the manufacturer’s protocol (Promega). A549 cells were treated with six compounds (40 μM), individually, and each showed over 50% inhibition of DX2 using the same conditions as above; nanoluciferase-tagged AIMP2 was introduced ectopically for negative screening. The compound showing the highest and lowest inhibition of DX2 and AIMP2, respectively (<b>BC-DXI-843</b>), was identified as a lead. All experiments were independently repeated three times.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> <i>In Vitro</i> Pull-down Assay</h3><div class="NLM_p last">DX2-His 6 (0.1 mg) and HSP70 (0.25 mg) were mixed at an equal molar ratio (1:1) in 50 mM Tris-HCl (pH 7.4) lysis buffer containing 100 mM NaCl, 0.5% Triton X-100, 10% glycerol, 1 mM EDTA, and protease inhibitors (Calbiochem). <b>BC-DXI-843</b> was then added in a dose-dependent manner and incubated for 4 h. After incubation, DX2-His 6 proteins were precipitated with Ni-IDA beads (ELPIS-BIOTECH) and washed with cold lysis buffer three times. HSP70 that co-precipitated with DX2-His 6 was separated and detected by SDS–PAGE and coomassie staining, respectively.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> Complementation Assay Using Nanoluciferase Binary Technology</h3><div class="NLM_p last">DX2 and HSP70 were cloned into pBiT1.1-N[TK/LgBiT] and pBiT2.1-N[TK/SmBiT] to express LgBiT-tagged DX2 and SmBiT-tagged HSP70, respectively. A549 cells were ectopically induced with LgBiT-DX2 and SmBiT-HSP70 and then treated with <b>BC-DXI-843</b> in a dose-dependent manner for 4 h. After incubation, luciferase activity was measured with a nanoluciferase assay system following the manufacturer’s protocol (Promega). For a negative binding control, PRKACA (protein kinase A catalytic subunit) and PKRAR2A (protein kinase A type 2A regulatory subunit) were cloned into pBiT1.1-N[TK/LgBiT] and pBiT2.1-N[TK/SmBiT], respectively, following the same experimental procedures as above. The experiments were independently repeated three times.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Immunoprecipitation</h3><div class="NLM_p last">The cells were lysed with 50 mM Tris-HCl (pH 7.4) lysis buffer containing 100 mM NaCl, 0.5% Triton X-100, 0.1% SDS, 10% glycerol, 1 mM EDTA, and protease inhibitors (Calbiochem). Total cell extracts were mixed with a DX2-specific antibody and incubated with agarose A for 12 h. After incubation, co-precipitates with agarose A were washed with lysis buffer without SDS and subjected to SDS–PAGE and immunoblotting using specific antibodies against the proteins of interest. To monitor the ubiquitination of DX2, the cells were cultured in media supplemented with MG-132 (50 μM) following the same experimental procedure described above.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Cell Viability Assay</h3><div class="NLM_p last">A549 (1.5 × 10<sup>4</sup> cells/mL) or WI-26 (4 × 10<sup>4</sup> cells/mL) cells were cultured in 96-well, flat-bottom plates for 24 h. Diluted compounds in serum-free media were added to the cells and incubated for 72 h. To test each compound’s DX2 dependency, stable cells expressing DX2 upon induction with DMSO (Sigma) were cultured in 96-well plates and treated with each compound. An amount of 10 μL of MTT solution (5 mg/mL, Sigma) was added to the cells and incubated for 1.5 h at 37 °C. After discarding the culture media with the MTT solution, the precipitated formazan crystals in each well were dissolved with 100 μL of DMSO (Duchefa). Absorbance was measured at 560 nm using a microplate reader (Sunrise, TECAN). All experiments were independently repeated three times.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> 3D Cell Culture Assay</h3><div class="NLM_p last">H460 cells cultivated in 2D culture conditions were detached using Accumax (EMD Millipore) for 3D cell culture. Cells were counted with a disposable hemocytometer, C-Chip (INCYTO), and diluted in media supplemented with 1% penicillin/streptomycin and 10% FBS to a concentration of 5000 cells/100 μL. The cells were seeded onto ultralow attachment, 96-well, 3D-culture plates (Corning). After incubation for 48 h, spheroid, 3D-cultured cancer cells were treated with each compound for 72 h. Cell viability was determined using the CellTiter-Glo 3D cell viability assay (Promega) according to the manufacturer’s instructions.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Xenograft</h3><div class="NLM_p last">H460 cells (1 × 10<sup>7</sup>) were subcutaneously inoculated into the back of 7-week-old female BALB/cSLC-nu/nu mice (Central Lab. Animal Inc., Korea) (<i>n</i> = 4/group). <b>BC-DXI-843</b> (50 mg/kg) was intraperitoneally injected every other day for 15 days. The embedded tumor volume and body weight were measured five and four times, respectively, over the experimental period. After 15 days, all mice were sacrificed, and the size and weight of the excised tumors were measured after photos of the mice were taken. Animal experiments complied with the University Animal Care and Use Committee guidelines at Seoul National University.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Docking and System Preparation for Computational Modeling</h3><div class="NLM_p last">Modeling and initial docking calculations were performed using the Schrödinger Suite molecular modeling package (version 2017-3). The default parameters were used unless otherwise noted. The homology structure that was built in our previous study was prepared using the Protein Preparation Wizard.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> First, force field atom types and bond orders were assigned, missing atoms were added, and tautomer/ionization states were assigned so that their tautomers were adjusted to optimize the hydrogen bond. Constrained energy minimization was then performed. Typically, docking sites were explored and characterized using the Site Map<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> tool. On the basis of previous binding studies using NMR, X-ray, and site-directed mutagenesis, we prepared Grid box and docked using Glide.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a></div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Fluorescence-Based Equilibrium Binding Assay</h3><div class="NLM_p last">All titration experiments were conducted at 20 °C using a Jasco FP 6500 spectrofluorometer (Easton, MD, USA). Purified DX2 protein was equilibrated with various concentrations of ligands before measuring fluorescence emission. Ligand stock solutions were titrated into a protein sample dissolved in phosphate buffer (pH 7.4) containing 137 mM NaCl, 2.7 mM KCl, 10 mM Na<sub>2</sub>HPO<sub>4</sub>, and 2 mM KH<sub>2</sub>PO<sub>4</sub>. Protein samples were excited at 280 nm, and the decrease in fluorescence emission upon ligand binding was measured at 335 nm as a function of ligand concentration. All titration data were fit to a hyperbolic binding equation to obtain <i>K</i><sub>d</sub> values.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> RT-PCR</h3><div class="NLM_p last">Total RNA extracted by using an RNeasy Mini Kit (Qiagen) was subjected to RT-PCR with dNTP, random hexamer, and Moloney murine leukemia virus (MMLV). The same volume of cDNA was used in a PCR assay to detect the mRNA expression of DX2, AIMP2, and actin using the following specific primers: DX2, CTGGCCACGTGCAGGATTACGGGG and AAGTGAATCCCAGCTGATAG; AIMP2, ATGCCGATGTACCAGGTAAAG and CTTAAGGAGCTTGAGGGCCGT; actin, CCTTCCTGGGCATGGAGTCCT and GGAGCAATGATCTTGATCTT.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Statistics</h3><div class="NLM_p last">Statistical tests were performed with Prism (GraphPad). All error bars represent standard deviation (SD). For calculation of IC<sub>50</sub>, EC<sub>50</sub>, and <i>K</i><sub>d</sub> values, we used Prism (GraphPad).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i95"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01961" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78736" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78736" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01961?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01961</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Determining the significant residues for binding of <b>BC-DXI-843</b> to DX2 (Figure S1A,B); DX2 degradation caused by <b>BC-DXI-843</b> (Figure S1C,D); <b>BC-DXI-843</b>-dependent inhibition of DX2 and HSP70 binding (Figure S2); <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of the final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf">jm9b01961_si_001.pdf (9.32 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_002.pdb">jm9b01961_si_002.pdb (7.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_003.csv">jm9b01961_si_003.csv (5.87 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01961" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sunghoon Kim</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Bioconvergence Research Center, College of Pharmacy & College
of Medicine, Gangnam Severance Hospital, Yonsei University, Incheon 21983, Republic of Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1570-3230" title="Orcid link">http://orcid.org/0000-0002-1570-3230</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#daa9afb4bdb1b3b79aa9b4aff4bbb9f4b1a8"><span class="__cf_email__" data-cfemail="6e1d1b00090507032e1d001b400f0d40051c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kyeong Lee</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5455-9956" title="Orcid link">http://orcid.org/0000-0002-5455-9956</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b3d8d2cadfd6d6f3d7dcddd4d4c6d89dd6d7c6"><span class="__cf_email__" data-cfemail="711a10081d141431151e1f1616041a5f141504">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aneesh Sivaraman</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dae Gyu Kim</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Bioconvergence Research Center, College of Pharmacy & College
of Medicine, Gangnam Severance Hospital, Yonsei University, Incheon 21983, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Deepak Bhattarai</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Minkyoung Kim</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hwa Young Lee</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Dongguk University—Seoul, Goyang 10326, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Semi Lim</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Bioconvergence Research Center, College of Pharmacy & College
of Medicine, Gangnam Severance Hospital, Yonsei University, Incheon 21983, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiwon Kong</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Bioconvergence Research Center, College of Pharmacy & College
of Medicine, Gangnam Severance Hospital, Yonsei University, Incheon 21983, Republic of Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3990-5619" title="Orcid link">http://orcid.org/0000-0003-3990-5619</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ja-il Goo</span> - <span class="hlFld-Affiliation affiliation">Department
of Biotechnology, Korea University, Seoul 02841, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Seunghwan Shim</span> - <span class="hlFld-Affiliation affiliation">Department
of Biotechnology, Korea University, Seoul 02841, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Seungbeom Lee</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, CHA University, Gyeonggi-do 11160, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Young-Ger Suh</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, CHA University, Gyeonggi-do 11160, Republic of Korea</span>; 
    <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Seoul National University, Seoul 08826, Republic of Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1799-8607" title="Orcid link">http://orcid.org/0000-0003-1799-8607</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yongseok Choi</span> - <span class="hlFld-Affiliation affiliation">Department
of Biotechnology, Korea University, Seoul 02841, Republic of Korea</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.S. and D.G.K. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i97">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24331" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24331" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This study was funded with a National Research Foundation of Korea (NRF) grant funded by the Korea Government (MSIT) (Grants NRF 2018R1A5A2023127 and NRF-M3A6A4-2010-0029785).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AIMP2</td><td class="NLM_def"><p class="first last">aminoacyl-tRNA synthetase-interacting multifunctional protein 2</p></td></tr><tr><td class="NLM_term">HSP70s</td><td class="NLM_def"><p class="first last">70 kDa heat shock proteins</p></td></tr><tr><td class="NLM_term">p53</td><td class="NLM_def"><p class="first last">phosphoprotein p53</p></td></tr><tr><td class="NLM_term">TGF-β</td><td class="NLM_def"><p class="first last">transforming growth factor-β</p></td></tr><tr><td class="NLM_term">TNF-α</td><td class="NLM_def"><p class="first last">tumor necrosis factor-α</p></td></tr><tr><td class="NLM_term">FBP</td><td class="NLM_def"><p class="first last">fuse-binding protein</p></td></tr><tr><td class="NLM_term">TRAF2</td><td class="NLM_def"><p class="first last">TNF receptor-associated factor 2</p></td></tr><tr><td class="NLM_term">GST</td><td class="NLM_def"><p class="first last">Glutathione <i>S</i>-transferase</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EDC·HCl</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimidehydrochloride</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">mp</td><td class="NLM_def"><p class="first last">melting point.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i99">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61372" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61372" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 29 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span> <span> </span><span class="NLM_article-title">Aminoacyl-tRNA synthetases and tumorigenesis: more than housekeeping</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">708</span>– <span class="NLM_lpage">718</span>, <span class="refDoi"> DOI: 10.1038/nrc3124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1038%2Fnrc3124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=21941282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ajurrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=708-718&author=S.+Kimauthor=S.+Youauthor=D.+Hwang&title=Aminoacyl-tRNA+synthetases+and+tumorigenesis%3A+more+than+housekeeping&doi=10.1038%2Fnrc3124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoacyl-tRNA synthetases and tumorigenesis: more than housekeeping</span></div><div class="casAuthors">Kim, Sung-Hoon; You, Sung-Yong; Hwang, Dae-Hee</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">708-718</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past decade, the identification of cancer-assocd. factors has been a subject of primary interest not only for understanding the basic mechanisms of tumorigenesis but also for discovering the assocd. therapeutic targets.  However, aminoacyl-tRNA synthetases (ARSs) have been overlooked, mostly because many assumed that they were simply 'housekeepers' that were involved in protein synthesis.  Mammalian ARSs have evolved many addnl. domains that are not necessarily linked to their catalytic activities.  With these domains, they interact with diverse regulatory factors.  In addn., the expression of some ARSs is dynamically changed depending on various cellular types and stresses.  This Anal. article addresses the potential pathophysiol. implications of ARSs in tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLu4m0EWOj57Vg90H21EOLACvtfcHk0ljgyvwcVm_v8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ajurrO&md5=cfc0b098147984cf9728af848c637020</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc3124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3124%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DYou%26aufirst%3DS.%26aulast%3DHwang%26aufirst%3DD.%26atitle%3DAminoacyl-tRNA%2520synthetases%2520and%2520tumorigenesis%253A%2520more%2520than%2520housekeeping%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2011%26volume%3D11%26spage%3D708%26epage%3D718%26doi%3D10.1038%2Fnrc3124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Aminoacyl- tRNA synthetases as therapeutic targets</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">650</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0026-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1038%2Fs41573-019-0026-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=31073243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlajtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=629-650&author=N.+H.+Kwonauthor=P.+L.+Foxauthor=S.+Kim&title=Aminoacyl-+tRNA+synthetases+as+therapeutic+targets&doi=10.1038%2Fs41573-019-0026-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoacyl-tRNA synthetases as therapeutic targets</span></div><div class="casAuthors">Kwon, Nam Hoon; Fox, Paul L.; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">629-650</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Aminoacyl-tRNA synthetases (ARSs) are essential enzymes for protein synthesis with evolutionarily conserved enzymic mechanisms.  Despite their similarity across organisms, scientists have been able to generate effective anti-infective agents based on the structural differences in the catalytic clefts of ARSs from pathogens and humans.  However, recent genomic, proteomic and functionomic advances have unveiled unexpected disease-assocd. mutations and altered expression, secretion and interactions in human ARSs, revealing hidden biol. functions beyond their catalytic roles in protein synthesis.  These studies have also brought to light their potential as a rich and unexplored source for new therapeutic targets and agents through multiple avenues, including direct targeting of the catalytic sites, controlling disease-assocd. protein-protein interactions and developing novel biologics from the secreted ARS proteins or their parts.  This Review addresses the emerging biol. and therapeutic applications of human ARSs in diseases including autoimmune and rare diseases, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1lM6NQW2SDLVg90H21EOLACvtfcHk0ljgyvwcVm_v8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlajtrY%253D&md5=dd9c680cd873d35c8ab10350cd7fb044</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0026-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0026-3%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DFox%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAminoacyl-%2520tRNA%2520synthetases%2520as%2520therapeutic%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D629%26epage%3D650%26doi%3D10.1038%2Fs41573-019-0026-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Hierarchical network between the components of the multi-tRNA synthetase complex: implications for complex formation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">38663</span>– <span class="NLM_lpage">38667</span>, <span class="refDoi"> DOI: 10.1074/jbc.M605211200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1074%2Fjbc.M605211200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=17062567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12msr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=38663-38667&author=J.+M.+Hanauthor=M.+J.+Leeauthor=S.+G.+Parkauthor=S.+H.+Leeauthor=E.+Razinauthor=E.+C.+Choiauthor=S.+Kim&title=Hierarchical+network+between+the+components+of+the+multi-tRNA+synthetase+complex%3A+implications+for+complex+formation&doi=10.1074%2Fjbc.M605211200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Hierarchical Network between the Components of the Multi-tRNA Synthetase Complex: Implications for complex formation</span></div><div class="casAuthors">Han, Jung Min; Lee, Min Ji; Park, Sang Gyu; Lee, Sun Hee; Razin, Ehud; Choi, Eung-Chil; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">38663-38667</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The macromol. tRNA synthetase complex consists of nine different enzymes and three non-enzymic factors.  This complex was recently shown to be a novel signalosome, since many of its components are involved in signaling pathways in addn. to their catalytic roles in protein synthesis.  The structural organization and dynamic relationships of the components of the complex are not well understood.  Here we performed a systematic depletion anal. to det. the effects of structural intimacy and the turnover of the components.  The results showed that the stability of some components depended on their neighbors.  Lysyl-tRNA synthetase was most independent of other components for its stability whereas it was most required for the stability of other components.  Arginyl- and methionyl-tRNA synthetases had the opposite characteristics.  Thus, the systematic depletion of the components revealed the functional reason for the complex formation and the assembly pattern of these multi-functional enzymes and their assocd. factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJHgRwZP0CkrVg90H21EOLACvtfcHk0ljgyvwcVm_v8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12msr3P&md5=14ee04049d062216dcecca3cc66f6137</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M605211200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M605211200%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DS.%2BG.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DRazin%26aufirst%3DE.%26aulast%3DChoi%26aufirst%3DE.%2BC.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DHierarchical%2520network%2520between%2520the%2520components%2520of%2520the%2520multi-tRNA%2520synthetase%2520complex%253A%2520implications%2520for%2520complex%2520formation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D38663%26epage%3D38667%26doi%3D10.1074%2Fjbc.M605211200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Aminoacyl-tRNA synthetase-interacting multifunctional proteins (AIMPs): A triad for cellular homeostasis</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">296</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1002/iub.324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1002%2Fiub.324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=20306515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVCkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=296-302&author=S.+G.+Parkauthor=E.+C.+Choiauthor=S.+Kim&title=Aminoacyl-tRNA+synthetase-interacting+multifunctional+proteins+%28AIMPs%29%3A+A+triad+for+cellular+homeostasis&doi=10.1002%2Fiub.324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoacyl-tRNA synthetase-interacting multifunctional proteins (AIMPs): a triad for cellular homeostasis</span></div><div class="casAuthors">Park, Sang Gyu; Choi, Eung-Chil; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">296-302</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">A review.  Aminoacyl-tRNA synthetases (ARSs) are highly conserved for efficient and precise translation of genetic codes.  In higher eukaryotic systems, several different ARSs including glutamyl-, prolyl-, isoleucyl-, leucyl-, methionyl-, glutaminyl-, lysyl-, arginyl-, and aspartyl-tRNA synthetases form a macromol. protein complex with 3 nonenzymic protein cofactors (AIMP1/p43, AIMP2/p38, and AIMP3/p18).  Although the structure and functional implications for this complex formation are not completely understood, rapidly accumulating evidences suggest that this complex would work as a mol. hub linked to the multiple signaling pathways that involve the components of enzymes and cofactors.  Here, the roles of 3 nonenzymic protein components of the multi-tRNA synthetase complex in the assembly of the components and in cell regulation are addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqVLxruiaOp7Vg90H21EOLACvtfcHk0lg82BjqOIqrNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVCkt7Y%253D&md5=5aaec0e7bb1bbf74b649771bd91fcbb7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fiub.324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.324%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DS.%2BG.%26aulast%3DChoi%26aufirst%3DE.%2BC.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAminoacyl-tRNA%2520synthetase-interacting%2520multifunctional%2520proteins%2520%2528AIMPs%2529%253A%2520A%2520triad%2520for%2520cellular%2520homeostasis%26jtitle%3DIUBMB%2520Life%26date%3D2010%26volume%3D62%26spage%3D296%26epage%3D302%26doi%3D10.1002%2Fiub.324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">AIMP2/p38, the scaffold for the multi-tRNA synthetase complex, responds to genotoxic stresses via p53</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">11206</span>– <span class="NLM_lpage">11211</span>, <span class="refDoi"> DOI: 10.1073/pnas.0800297105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1073%2Fpnas.0800297105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=18695251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVShtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=11206-11211&author=J.+M.+Hanauthor=B.+J.+Parkauthor=S.+G.+Parkauthor=Y.+S.+Ohauthor=S.+J.+Choiauthor=S.+W.+Leeauthor=S.+K.+Hwangauthor=S.+H.+Changauthor=M.+H.+Choauthor=S.+Kim&title=AIMP2%2Fp38%2C+the+scaffold+for+the+multi-tRNA+synthetase+complex%2C+responds+to+genotoxic+stresses+via+p53&doi=10.1073%2Fpnas.0800297105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">AIMP2/p38, the scaffold for the multi-tRNA synthetase complex, responds to genotoxic stresses via p53</span></div><div class="casAuthors">Han, Jung Min; Park, Bum-Joon; Park, Sang Gyu; Oh, Young Sun; Choi, So Jung; Lee, Sang Won; Hwang, Soon-Kyung; Chang, Seung-Hee; Cho, Myung-Haing; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">11206-11211</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">AIMP2/p38 is a scaffolding protein required for the assembly of the macromol. tRNA synthetase complex.  Here, we describe a previously unknown function for AIMP2 as a pos. regulator of p53 in response to genotoxic stresses.  Depletion of AIMP2 increased resistance to DNA damage-induced apoptosis, and introduction of AIMP2 into AIMP2-deficient cells restored the susceptibility to apoptosis.  Upon DNA damage, AIMP2 was phosphorylated, dissocd. from the multi-tRNA synthetase complex, and translocated into the nuclei of cells.  AIMP2 directly interacts with p53, thereby preventing MDM2-mediated ubiquitination and degrdn. of p53.  Mutations in AIMP2, affecting its interaction with p53, hampered its ability to activate p53.  Nutlin-3 recovered the level of p53 and the susceptibility to UV-induced cell death in AIMP2-deficient cells.  This work demonstrates that AIMP2, a component of the translational machinery, functions as proapoptotic factor via p53 in response to DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3ezVoQ1fGhbVg90H21EOLACvtfcHk0lg82BjqOIqrNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVShtbnE&md5=bd708b2ee1ae05ba7bb81a6e7d034bda</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0800297105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0800297105%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DPark%26aufirst%3DS.%2BG.%26aulast%3DOh%26aufirst%3DY.%2BS.%26aulast%3DChoi%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BW.%26aulast%3DHwang%26aufirst%3DS.%2BK.%26aulast%3DChang%26aufirst%3DS.%2BH.%26aulast%3DCho%26aufirst%3DM.%2BH.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAIMP2%252Fp38%252C%2520the%2520scaffold%2520for%2520the%2520multi-tRNA%2520synthetase%2520complex%252C%2520responds%2520to%2520genotoxic%2520stresses%2520via%2520p53%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D11206%26epage%3D11211%26doi%3D10.1073%2Fpnas.0800297105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Downregulation of FUSE-binding protein and c-myc by tRNA synthetase cofactor p38 is required for lung cell differentiation</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">330</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/ng1182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1038%2Fng1182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=12819782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFSrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2003&pages=330-336&author=M.+J.+Kimauthor=B.+J.+Parkauthor=Y.+S.+Kangauthor=H.+J.+Kimauthor=J.+H.+Parkauthor=J.+W.+Kangauthor=S.+W.+Leeauthor=J.+M.+Hanauthor=H.+W.+Leeauthor=S.+Kim&title=Downregulation+of+FUSE-binding+protein+and+c-myc+by+tRNA+synthetase+cofactor+p38+is+required+for+lung+cell+differentiation&doi=10.1038%2Fng1182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of FUSE-binding protein and c-myc by tRNA synthetase cofactor p38 is required for lung cell differentiation</span></div><div class="casAuthors">Kim, Min Jung; Park, Bum-Joon; Kang, Young-Sun; Kim, Hyoung June; Park, Jae-Hyun; Kang, Jung Woo; Lee, Sang Won; Han, Jung Min; Lee, Han-Woong; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">330-336</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P38 is assocd. with a macromol. tRNA synthetase complex.  It has an essential role as a scaffold for the complex, and genetic disruption of p38 in mice causes neonatal lethality.  Here we investigated the mol. mechanisms underlying lethality of p38-mutant mice. p38-deficient mice showed defects in lung differentiation and respiratory distress syndrome. p38 was found to interact with FUSE-binding protein (FBP), a transcriptional activator of c-myc.  Binding of p38 stimulated ubiquitination and degrdn. of FBP, leading to downregulation of c-myc, which is required for differentiation of functional alveolar type II cells.  Transforming growth factor-β (TGF-β) induced p38 expression and promoted its translocation to nuclei for the regulation of FBP and c-myc.  Thus, this work identified a new activity of p38 as a mediator of TGF-β signaling and its functional importance in the control of c-myc during lung differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhjxINrMthVbVg90H21EOLACvtfcHk0lg82BjqOIqrNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFSrs7g%253D&md5=9523a4243e6696a071ac65ccf790f7b1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fng1182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng1182%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DKang%26aufirst%3DY.%2BS.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DPark%26aufirst%3DJ.%2BH.%26aulast%3DKang%26aufirst%3DJ.%2BW.%26aulast%3DLee%26aufirst%3DS.%2BW.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DH.%2BW.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DDownregulation%2520of%2520FUSE-binding%2520protein%2520and%2520c-myc%2520by%2520tRNA%2520synthetase%2520cofactor%2520p38%2520is%2520required%2520for%2520lung%2520cell%2520differentiation%26jtitle%3DNat.%2520Genet.%26date%3D2003%26volume%3D34%26spage%3D330%26epage%3D336%26doi%3D10.1038%2Fng1182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Um, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">AIMP2 promotes TNFalpha-dependent apoptosis via ubiquitin-mediated degradation of TRAF2</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">2710</span>– <span class="NLM_lpage">2715</span>, <span class="refDoi"> DOI: 10.1242/jcs.049767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1242%2Fjcs.049767" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=19584093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGhsLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2009&pages=2710-2715&author=J.+W.+Choiauthor=D.+G.+Kimauthor=M.+C.+Parkauthor=J.+Y.+Umauthor=J.+M.+Hanauthor=S.+G.+Parkauthor=E.+C.+Choiauthor=S.+Kim&title=AIMP2+promotes+TNFalpha-dependent+apoptosis+via+ubiquitin-mediated+degradation+of+TRAF2&doi=10.1242%2Fjcs.049767"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">AIMP2 promotes TNFα-dependent apoptosis via ubiquitin-mediated degradation of TRAF2</span></div><div class="casAuthors">Choi, Jin Woo; Kim, Dae Gyu; Park, Min Chul; Um, Jung Yeon; Han, Jung Min; Park, Sang Gyu; Choi, Eung-Chil; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2710-2715</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">AIMP2 (aminoacyl-tRNA synthetase interacting multifunctional protein 2; also known as JTV-1) was first identified as p38 in a macromol. protein complex that consisted of nine different aminoacyl-tRNA synthetases and two other auxiliary factors.  AIMP2 also plays pivotal roles in the regulation of cell proliferation and death.  Although AIMP2 was previously shown to augment TNFα-induced cell death, its working mechanism in this signal pathway was not understood.  Here, we investigate the functional significance and mode of action of AIMP2 in TNFα signaling.  TNFα-induced cell death was compromised in AIMP2-deficient or -suppressed cells and exogenous supplementation of AIMP2 augmented apoptotic sensitivity to TNFα signaling.  This activity was confirmed by the AIMP2-dependent increase of IκB and suppression of NFκB.  We found binding of AIMP2 to TRAF2, a key player in the TNFα signaling pathway.  AIMP2 augmented the assocn. of an E3 ubiquitin ligase, c-IAP1, with TRAF2, causing ubiquitin-dependent degrdn. of TRAF2.  These findings suggest that AIMP2 can mediate the pro-apoptotic activity of TNFα via the downregulation of TRAF2 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVp478RpzlLrVg90H21EOLACvtfcHk0ljcIv-mdNiM3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGhsLzO&md5=cf685c704a7d95b85637f3c6642ccef1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1242%2Fjcs.049767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.049767%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DPark%26aufirst%3DM.%2BC.%26aulast%3DUm%26aufirst%3DJ.%2BY.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DPark%26aufirst%3DS.%2BG.%26aulast%3DChoi%26aufirst%3DE.%2BC.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAIMP2%2520promotes%2520TNFalpha-dependent%2520apoptosis%2520via%2520ubiquitin-mediated%2520degradation%2520of%2520TRAF2%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2009%26volume%3D122%26spage%3D2710%26epage%3D2715%26doi%3D10.1242%2Fjcs.049767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yum, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jo, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y. Y.</span></span> <span> </span><span class="NLM_article-title">AIMP2 controls intestinal stem cell compartments and tumorigenesis by modulating Wnt/beta-catenin signaling</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">4559</span>– <span class="NLM_lpage">4568</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-3357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1158%2F0008-5472.CAN-15-3357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=27262173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gmt7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=4559-4568&author=M.+K.+Yumauthor=J.+S.+Kangauthor=A.+E.+Leeauthor=Y.+W.+Joauthor=J.+Y.+Seoauthor=H.+A.+Kimauthor=Y.+Y.+Kimauthor=J.+Seongauthor=E.+B.+Leeauthor=J.+H.+Kimauthor=J.+M.+Hanauthor=S.+Kimauthor=Y.+Y.+Kong&title=AIMP2+controls+intestinal+stem+cell+compartments+and+tumorigenesis+by+modulating+Wnt%2Fbeta-catenin+signaling&doi=10.1158%2F0008-5472.CAN-15-3357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">AIMP2 Controls Intestinal Stem Cell Compartments and Tumorigenesis by Modulating Wnt/β-Catenin Signaling</span></div><div class="casAuthors">Yum, Min Kyu; Kang, Jong-Seol; Lee, Al-Eum; Jo, Young-Woo; Seo, Ji-Yun; Kim, Hyun-A.; Kim, Yoon-Young; Seong, Jinwoo; Lee, Eun Byul; Kim, Ji-Hoon; Han, Jung Min; Kim, Sunghoon; Kong, Young-Yun</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4559-4568</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Wnt/β-catenin (CTNNB1) signaling is crucial for the proliferation and maintenance of intestinal stem cells (ISC), but excessive activation leads to ISC expansion and eventually colorectal cancer.  Thus, neg. regulators are required to maintain optimal levels of Wnt/β-catenin signaling.  Aminoacyl-tRNA synthetase-interacting multifunctional proteins (AIMP) function in protein synthesis, but have also been implicated in signaling cascades affecting angiogenesis, immunity, and apoptosis.  In this study, we investigated the relationship between AIMP2 and Wnt/β-catenin signaling in a murine model of intestinal homeostasis and tumorigenesis.  Hemizygous deletion of Aimp2 resulted in enhanced Wnt/β-catenin signaling, increased proliferation of cryptic epithelial cells, and expansion of ISC compartments.  In an ApcMin/+ background, Aimp2 hemizygosity increased adenoma formation.  Mechanistically, AIMP2 disrupted the interaction between AXIN and Dishevelled-1 (DVL1) to inhibit Wnt/β-catenin signaling by competing with AXIN.  Furthermore, AIMP2 inhibited intestinal organoid formation and growth by suppressing Wnt/β-catenin signaling in an Aimp2 gene dosage-dependent manner.  Collectively, our results showed that AIMP2 acts as a haploinsufficient tumor suppressor that fine-tunes Wnt/β-catenin signaling in the intestine, illuminating the regulation of ISC abundance and activity.  Cancer Res; 76(15); 4559-68. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw0FydKmEcA7Vg90H21EOLACvtfcHk0ljcIv-mdNiM3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gmt7fI&md5=407c3f1529faaa458c534cfc8d1838a1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-3357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-3357%26sid%3Dliteratum%253Aachs%26aulast%3DYum%26aufirst%3DM.%2BK.%26aulast%3DKang%26aufirst%3DJ.%2BS.%26aulast%3DLee%26aufirst%3DA.%2BE.%26aulast%3DJo%26aufirst%3DY.%2BW.%26aulast%3DSeo%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DH.%2BA.%26aulast%3DKim%26aufirst%3DY.%2BY.%26aulast%3DSeong%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DE.%2BB.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DY.%2BY.%26atitle%3DAIMP2%2520controls%2520intestinal%2520stem%2520cell%2520compartments%2520and%2520tumorigenesis%2520by%2520modulating%2520Wnt%252Fbeta-catenin%2520signaling%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D4559%26epage%3D4568%26doi%3D10.1158%2F0008-5472.CAN-15-3357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Won, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. W.</span></span> <span> </span><span class="NLM_article-title">Selective regression of cancer cells expressing a splicing variant of AIMP2 through targeted RNA replacement by <i>trans</i>-splicing ribozyme</span>. <i>J. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.jbiotec.2012.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1016%2Fj.jbiotec.2012.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=22285955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVGlu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2012&pages=44-49&author=Y.+S.+Wonauthor=S.+W.+Lee&title=Selective+regression+of+cancer+cells+expressing+a+splicing+variant+of+AIMP2+through+targeted+RNA+replacement+by+trans-splicing+ribozyme&doi=10.1016%2Fj.jbiotec.2012.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Selective regression of cancer cells expressing a splicing variant of AIMP2 through targeted RNA replacement by trans-splicing ribozyme</span></div><div class="casAuthors">Won, You-Sub; Lee, Seong-Wook</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">44-49</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">AIMP2/p38 is a scaffolding protein crit. for the assembly of the macromol. tRNA synthetase complex.  Moreover, AIMP2 harbors anti-proliferative activity and promotes cell death as a proapototic factor.  A splicing variant of AIMP2 lacking exon 2 (AIMP2-DX2) is specifically generated by an alternative splicing process and is highly expressed in human lung cancer cells and the tissues of cancer patients.  AIMP2-DX2 induces tumorigenesis by compromising the tumor suppressor function of normal AIMP2.  Here, we describe a novel approach to cancer therapy that is based on trans-splicing ribozyme-mediated replacement of specific RNAs.  We developed a specific ribozyme that can target and replace the splicing variant AIMP2-DX2 RNA with a new transcript selectively exerting therapeutic activity in AIMP2-DX2-expressing lung cancer cells.  The RNA replacement was employed via a high-fidelity trans-splicing reaction with the targeted residue of the AIMP2-DX2 transcript, but not with normal AIMP2 RNA, in the cells.  Noticeably, the ribozyme could selectively deliver the activity of a suicide gene into the AIMP2-DX2 RNA expressing lung cancer cells and thereby specifically and effectively retard the growth of the cancer cells with prodrug treatment.  Therefore, the AIMP2-DX2 RNA-targeting trans-splicing ribozyme could be a useful genetic agent for efficient therapy targeting lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDbUoFKEVvz7Vg90H21EOLACvtfcHk0ljcIv-mdNiM3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVGlu7o%253D&md5=39c8cce87457e98580f54cdce9dd0947</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2012.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2012.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DWon%26aufirst%3DY.%2BS.%26aulast%3DLee%26aufirst%3DS.%2BW.%26atitle%3DSelective%2520regression%2520of%2520cancer%2520cells%2520expressing%2520a%2520splicing%2520variant%2520of%2520AIMP2%2520through%2520targeted%2520RNA%2520replacement%2520by%2520trans-splicing%2520ribozyme%26jtitle%3DJ.%2520Biotechnol.%26date%3D2012%26volume%3D158%26spage%3D44%26epage%3D49%26doi%3D10.1016%2Fj.jbiotec.2012.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span>; <span class="NLM_string-name">Choi, J. W.</span></span> <span> </span><span class="NLM_article-title">AIMP2-DX2 Gene and its Uses</span>. <span class="NLM_patent">US 2011/0136119A1</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=S.+Kim&author=J.+W.+Choi&title=AIMP2-DX2+Gene+and+its+Uses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAIMP2-DX2%2520Gene%2520and%2520its%2520Uses%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Cancer-associated splicing variant of tumor suppressor AIMP2/p38: pathological implication in tumorigenesis</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e1001351</span> <span class="refDoi"> DOI: 10.1371/journal.pgen.1001351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1371%2Fjournal.pgen.1001351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=21483803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvVeqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&author=J.+W.+Choiauthor=D.+G.+Kimauthor=A.+E.+Leeauthor=H.+R.+Kimauthor=J.+Y.+Leeauthor=N.+H.+Kwonauthor=Y.+K.+Shinauthor=S.+K.+Hwangauthor=S.+H.+Changauthor=M.+H.+Choauthor=Y.+L.+Choiauthor=J.+Kimauthor=S.+H.+Ohauthor=B.+Kimauthor=S.+Y.+Kimauthor=H.+S.+Jeonauthor=J.+Y.+Parkauthor=H.+P.+Kangauthor=B.+J.+Parkauthor=J.+M.+Hanauthor=S.+Kim&title=Cancer-associated+splicing+variant+of+tumor+suppressor+AIMP2%2Fp38%3A+pathological+implication+in+tumorigenesis&doi=10.1371%2Fjournal.pgen.1001351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer-associated splicing variant of tumor suppressor AIMP2/p38: pathological implication in tumorigenesis</span></div><div class="casAuthors">Choi, Jin Woo; Kim, Dae Gyu; Lee, Al-Eum; Kim, Hye Rim; Lee, Jin Young; Kwon, Nam Hoon; Shin, Young Kee; Hwang, Soon-Kyung; Chang, Seung-Hee; Cho, Myung-Haing; Choi, Yoon-La; Kim, Jhingook; Oh, Seung Hyun; Kim, Bora; Kim, Soo-Youl; Jeon, Hyo-Sung; Park, Jae Yong; Kang, Hyunseok Peter; Park, Bum Joon; Han, Jung Min; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1001351</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Although ARS-interacting multi-functional protein 2 (AIMP2, also named as MSC p38) was first found as a component for a macromol. tRNA synthetase complex, it was recently discovered to dissoc. from the complex and work as a potent tumor suppressor.  Upon DNA damage, AIMP2 promotes apoptosis through the protective interaction with p53.  However, it was not demonstrated whether AIMP2 was indeed pathol. linked to human cancer.  In this work, we found that a splicing variant of AIMP2 lacking exon 2 (AIMP2-DX2) is highly expressed by alternative splicing in human lung cancer cells and patient's tissues.  AIMP2-DX2 compromised pro-apoptotic activity of normal AIMP2 through the competitive binding to p53.  The cells with higher level of AIMP2-DX2 showed higher propensity to form anchorage-independent colonies and increased resistance to cell death.  Mice constitutively expressing this variant showed increased susceptibility to carcinogen-induced lung tumorigenesis.  The expression ratio of AIMP2-DX2 to normal AIMP2 was increased according to lung cancer stage and showed a pos. correlation with the survival of patients.  Thus, this work identified an oncogenic splicing variant of a tumor suppressor, AIMP2/p38, and suggests its potential for anti-cancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptcSQBMoogs7Vg90H21EOLACvtfcHk0lip7JC60rrMOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvVeqsrY%253D&md5=ae6806bab0a6e632c31e03ff5f0241bf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1001351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1001351%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DLee%26aufirst%3DA.%2BE.%26aulast%3DKim%26aufirst%3DH.%2BR.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DShin%26aufirst%3DY.%2BK.%26aulast%3DHwang%26aufirst%3DS.%2BK.%26aulast%3DChang%26aufirst%3DS.%2BH.%26aulast%3DCho%26aufirst%3DM.%2BH.%26aulast%3DChoi%26aufirst%3DY.%2BL.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DOh%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DJeon%26aufirst%3DH.%2BS.%26aulast%3DPark%26aufirst%3DJ.%2BY.%26aulast%3DKang%26aufirst%3DH.%2BP.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DCancer-associated%2520splicing%2520variant%2520of%2520tumor%2520suppressor%2520AIMP2%252Fp38%253A%2520pathological%2520implication%2520in%2520tumorigenesis%26jtitle%3DPLoS%2520Genet.%26date%3D2011%26volume%3D7%26doi%3D10.1371%2Fjournal.pgen.1001351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Splicing variant of AIMP2 as an effective target against chemoresistant ovarian cancer</span>. <i>J. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">164</span>– <span class="NLM_lpage">173</span>, <span class="refDoi"> DOI: 10.1093/jmcb/mjs018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1093%2Fjmcb%2Fmjs018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=22532625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12rtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=164-173&author=J.+W.+Choiauthor=J.+W.+Leeauthor=J.+K.+Kimauthor=H.+K.+Jeonauthor=J.+J.+Choiauthor=D.+G.+Kimauthor=B.+G.+Kimauthor=D.+H.+Namauthor=H.+J.+Kimauthor=S.+H.+Yunauthor=S.+Kim&title=Splicing+variant+of+AIMP2+as+an+effective+target+against+chemoresistant+ovarian+cancer&doi=10.1093%2Fjmcb%2Fmjs018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Splicing variant of AIMP2 as an effective target against chemoresistant ovarian cancer</span></div><div class="casAuthors">Choi, Jin Woo; Lee, Jeong-Won; Kim, Jun Ki; Jeon, Hye-Kyung; Choi, Jung-Joo; Kim, Dae Gyu; Kim, Byoung-Gie; Nam, Do-Hyun; Kim, Hong Joong; Yun, Seok Hyun; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">164-173, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Chemoresistance is a main cause for the failure of cancer management and intensive investigation is on-going to control chemoresistant (CR) cancers.  Although NF-κB has been suggested as one of the potential targets to alleviate chemoresistance of epithelial ovarian cancer (EOC), direct targeting of NF-κB may result in an unexpected effect due to the complex regulatory network via NF-κB.  Here we show that AIMP2-DX2, a splicing variant of tumor suppressor AIMP2, can be a therapeutic target to control CR EOC.  AIMP2-DX2 was often highly expressed in CR EOC both in vitro and in vivo.  AIMP2-DX2 compromised the tumor necrosis factor alpha-dependent pro-apoptotic activity of AIMP2 via the competitive inhibition of AIMP2 binding to TRAF2 that plays a pivotal role in the regulation of NF-κB.  The direct delivery of siRNA against AIMP2-DX2 into abdominal metastatic tumors of ovarian cancer using a microneedle converged on microendoscopy significantly suppressed the growth rate of tumors.  The treated cancer tissues showed an enhanced apoptosis and the decreased TRAF2 level.  Thus, we suggest that the downregulation of AIMP2-DX2 can be a potent adjuvant therapeutic approach for CR EOC that resulted from an aberrant activity of NF-κB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPbT1cw7EKMbVg90H21EOLACvtfcHk0lip7JC60rrMOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12rtbbL&md5=eca0bcc749a50e4bb3ea5a0319f86ca0</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjs018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjs018%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BW.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DJ.%2BK.%26aulast%3DJeon%26aufirst%3DH.%2BK.%26aulast%3DChoi%26aufirst%3DJ.%2BJ.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DKim%26aufirst%3DB.%2BG.%26aulast%3DNam%26aufirst%3DD.%2BH.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DYun%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DSplicing%2520variant%2520of%2520AIMP2%2520as%2520an%2520effective%2520target%2520against%2520chemoresistant%2520ovarian%2520cancer%26jtitle%3DJ.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D4%26spage%3D164%26epage%3D173%26doi%3D10.1093%2Fjmcb%2Fmjs018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. S.</span></span> <span> </span><span class="NLM_article-title">Ratio of autoantibodies of tumor suppressor AIMP2and its oncogenic variant is associated with clinical outcome in lung cancer</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1354</span>, <span class="refDoi"> DOI: 10.7150/jca.18450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.7150%2Fjca.18450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=28638448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A280%3ADC%252BC1cjgs1GktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1347-1354&author=J.+Y.+Jungauthor=E.+Y.+Kimauthor=A.+Kimauthor=J.+Changauthor=N.+H.+Kwonauthor=Y.+Moonauthor=E.+J.+Kangauthor=J.+S.+Sungauthor=H.+Shimauthor=S.+Kimauthor=Y.+S.+Chang&title=Ratio+of+autoantibodies+of+tumor+suppressor+AIMP2and+its+oncogenic+variant+is+associated+with+clinical+outcome+in+lung+cancer&doi=10.7150%2Fjca.18450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Ratio of Autoantibodies of Tumor Suppressor AIMP2 and Its Oncogenic Variant Is Associated with Clinical Outcome in Lung Cancer</span></div><div class="casAuthors">Jung Ji Ye; Kim Eun Young; Chang Joon; Kim Arum; Chang Yoon Soo; Kwon Nam Hoon; Moon Youngji; Kang Eun Joo; Sung Jun Sik; Kim Sunghoon; Shim Hyunbo; Kim Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1347-1354</span>
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    </div><div class="casAbstract">Aminoacyl-tRNA synthetase-interacting multi-functional protein 2 (AIMP2) works as potent tumor suppressor, while its splicing variant lacking exon 2 (AIMP2-DX2) competes with AIMP2 for binding to target proteins and compromises its anti-tumor activity.  Assuming that AIMP2 and its variant AIMP2-DX2 could be released out to human sera in pathological condition, we investigated the diagnostic and prognostic usefulness of autoantibodies against AIMP2 and AIMP2-DX2 by measuring their serum levels in 80 normal and lung cancer samples that were matched in age, gender and smoking status.  The area under the curve of AIMP2-DX2, AIMP2, and AIMP2-DX2/AIMP2 autoantibody ratio was low (0.416, 0.579, and 0.357, respectively), suggesting limited diagnostic value.  A total of 165 lung cancer patients were classified into low and high AIMP2-DX2, AIMP2, and AIMP2-DX2/AIMP2 based on the median expression of each parameter.  The high AIMP2-DX2 group was older and had larger tumors (>3 cm) than the low AIMP2-DX2 group.  The high AIMP2-DX2/AIMP2 group had higher CYFRA-21 levels and significantly shorter overall survival than the low AIMP2-DX2/AIMP2 group (18.6 vs. 48.9 months, P = 0.021, Log Rank Test).  Taken together, autoantibodies against AIMP2-DX2 and AIMP2 are detectable in the human blood and the increased ratio of AIMP2-DX2/AIMP2 is related to poor clinical outcome of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2VzkwiviqdgMthz5_Rx5rfW6udTcc2eaeu8-5iYAMELntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjgs1GktQ%253D%253D&md5=5b2d609d5b5db98ab6850f1ba81dbb3c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.7150%2Fjca.18450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.18450%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DE.%2BY.%26aulast%3DKim%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DMoon%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DE.%2BJ.%26aulast%3DSung%26aufirst%3DJ.%2BS.%26aulast%3DShim%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DY.%2BS.%26atitle%3DRatio%2520of%2520autoantibodies%2520of%2520tumor%2520suppressor%2520AIMP2and%2520its%2520oncogenic%2520variant%2520is%2520associated%2520with%2520clinical%2520outcome%2520in%2520lung%2520cancer%26jtitle%3DJ.%2520Cancer%26date%3D2017%26volume%3D8%26spage%3D1347%26epage%3D1354%26doi%3D10.7150%2Fjca.18450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Oncogenic mutation of AIMP2/p38 inhibits its tumor-suppressive interaction with Smurf2</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3422</span>– <span class="NLM_lpage">3436</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-3255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1158%2F0008-5472.CAN-15-3255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=27197155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVyktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3422-3436&author=D.+G.+Kimauthor=J.+Y.+Leeauthor=J.+H.+Leeauthor=H.+Y.+Choauthor=B.+S.+Kangauthor=S.+Y.+Jangauthor=M.+H.+Kimauthor=M.+Guoauthor=J.+M.+Hanauthor=S.+J.+Kimauthor=S.+Kim&title=Oncogenic+mutation+of+AIMP2%2Fp38+inhibits+its+tumor-suppressive+interaction+with+Smurf2&doi=10.1158%2F0008-5472.CAN-15-3255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic Mutation of AIMP2/p38 Inhibits Its Tumor-Suppressive Interaction with Smurf2</span></div><div class="casAuthors">Kim, Dae Gyu; Lee, Jin Young; Lee, Ji-Hyun; Cho, Ha Yeon; Kang, Beom Sik; Jang, Song-Yee; Kim, Myung Hee; Guo, Min; Han, Jung Min; Kim, Seong-Jin; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3422-3436</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">AIMP2/p38 is a multifunctional tumor suppressor that normally resides in the cytosol as a scaffold protein of the multi-tRNA synthetase complex (MSC).  One of the tumor-suppressive functions of AIMP2 is to facilitate ubiquitin-mediated degrdn. of FUSE-binding protein (FBP, FUBP1), a transcriptional activator of c-Myc.  However, the mechanism by which AIMP2 functions within this pathway and its significance in tumorigenesis are uncertain.  Here, we report that Smurf2 is responsible for AIMP2-mediated ubiquitination of FBP, and a mutation in AIMP2 that inhibited its nuclear interaction with Smurf2 enhanced cellular transformation and tumorigenesis in vivo.  Treatment of HeLa cells with TGFβ resulted in the phosphorylation of AIMP2 on S156, a residue that is exposed on the embedded GST domain of AIMP2.  We further found that phospho-AIMP2 dissocd. from the MSC and translocated to the nucleus, where it bound to Smurf2, enhancing ubiquitination of FBP.  AIMP2 also inhibited nuclear export of Smurf2 to sustain TGFβ signaling.  Collectively, these findings present a novel tumor-suppressive interaction between AIMP2 and Smurf2 and suggest that the disruption of this interaction can lead to oncogenic transformation.  Cancer Res; 76(11); 3422-36. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0ojyRQmzBfrVg90H21EOLACvtfcHk0liMtlnDbSzx_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVyktbo%253D&md5=0f720da36bc561c9f0a52117d4520468</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-3255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-3255%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DH.%2BY.%26aulast%3DKang%26aufirst%3DB.%2BS.%26aulast%3DJang%26aufirst%3DS.%2BY.%26aulast%3DKim%26aufirst%3DM.%2BH.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DOncogenic%2520mutation%2520of%2520AIMP2%252Fp38%2520inhibits%2520its%2520tumor-suppressive%2520interaction%2520with%2520Smurf2%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3422%26epage%3D3436%26doi%3D10.1158%2F0008-5472.CAN-15-3255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span>; <span class="NLM_string-name">Kim, D. G.</span>; <span class="NLM_string-name">Lim, S.</span></span> <span> </span><span class="NLM_article-title">Method for Screening Anticancer Agent Inhibiting Binding of AIMP2 - DX2 and HSP70</span>. <span class="NLM_patent">US 20190033295A1</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=S.+Kim&author=D.+G.+Kim&author=S.+Lim&title=Method+for+Screening+Anticancer+Agent+Inhibiting+Binding+of+AIMP2+-+DX2+and+HSP70"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26atitle%3DMethod%2520for%2520Screening%2520Anticancer%2520Agent%2520Inhibiting%2520Binding%2520of%2520AIMP2%2520-%2520DX2%2520and%2520HSP70%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushtaq, A. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattarai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaraman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Targeting the interaction of AIMP2-DX2 with HSP70 suppresses cancer development</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0415-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1038%2Fs41589-019-0415-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=31792442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=31-41&issue=1&author=S.+Limauthor=H.+Y.+Choauthor=D.+G.+Kimauthor=Y.+Rohauthor=S.+Sonauthor=A.+U.+Mushtaqauthor=M.+Kimauthor=D.+Bhattaraiauthor=A.+Sivaramanauthor=Y.+Leeauthor=J.+Leeauthor=W.+S.+Yangauthor=H.+K.+Kimauthor=M.+H.+Kimauthor=K.+Leeauthor=Y.+H.+Jeonauthor=S.+Kim&title=Targeting+the+interaction+of+AIMP2-DX2+with+HSP70+suppresses+cancer+development&doi=10.1038%2Fs41589-019-0415-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the interaction of AIMP2-DX2 with HSP70 suppresses cancer development</span></div><div class="casAuthors">Lim, Semi; Cho, Hye Young; Kim, Dae Gyu; Roh, Younah; Son, Se-Young; Mushtaq, Ameeq Ul; Kim, Minkyoung; Bhattarai, Deepak; Sivaraman, Aneesh; Lee, Youngjin; Lee, Jihye; Yang, Won Suk; Kim, Hoi Kyoung; Kim, Myung Hee; Lee, Kyeong; Jeon, Young Ho; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-41</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A tumorigenic factor, AIMP2 lacking exon 2 (AIMP2-DX2), is often upregulated in many cancers.  However, how its cellular level is detd. is not understood.  Here, we report heat-shock protein HSP70 as a crit. determinant for the level of AIMP2-DX2.  Interaction of the two factors was identified by interactome anal. and structurally detd. by X-ray crystallog. and NMR analyses.  HSP70 recognizes the amino (N)-terminal flexible region, as well as the glutathione S-transferase domain of AIMP2-DX2, via its substrate-binding domain, thus blocking the Siah1-dependent ubiquitination of AIMP2-DX2.  AIMP2-DX2-induced cell transformation and cancer progression in vivo was further augmented by HSP70.  A pos. correlation between HSP70 and AIMP2-DX2 levels was shown in various lung cancer cell lines and patient tissues.  Chem. intervention in the AIMP2-DX2-HSP70 interaction suppressed cancer cell growth in vitro and in vivo.  Thus, this work demonstrates the importance of the interaction between AIMP2-DX2 and HSP70 on tumor progression and its therapeutic potential against cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4li81ZRTJbrVg90H21EOLACvtfcHk0liMtlnDbSzx_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrzE&md5=14f219d03385a942300f8e0e3ed7f6a4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0415-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0415-2%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DH.%2BY.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DRoh%26aufirst%3DY.%26aulast%3DSon%26aufirst%3DS.%26aulast%3DMushtaq%26aufirst%3DA.%2BU.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DBhattarai%26aufirst%3DD.%26aulast%3DSivaraman%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DW.%2BS.%26aulast%3DKim%26aufirst%3DH.%2BK.%26aulast%3DKim%26aufirst%3DM.%2BH.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DJeon%26aufirst%3DY.%2BH.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DTargeting%2520the%2520interaction%2520of%2520AIMP2-DX2%2520with%2520HSP70%2520suppresses%2520cancer%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26issue%3D1%26spage%3D31%26epage%3D41%26doi%3D10.1038%2Fs41589-019-0415-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Chemical suppression of an oncogenic splicing variant of AIMP2 induces tumour regression</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>454</i></span>,  <span class="NLM_fpage">411</span>– <span class="NLM_lpage">416</span>, <span class="refDoi"> DOI: 10.1042/BJ20130550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1042%2FBJ20130550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=23815603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGmt7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=454&publication_year=2013&pages=411-416&author=H.+S.+Leeauthor=D.+G.+Kimauthor=Y.+S.+Ohauthor=N.+H.+Kwonauthor=J.+Y.+Leeauthor=D.+Kimauthor=S.+H.+Parkauthor=J.+H.+Songauthor=S.+Leeauthor=J.+M.+Hanauthor=B.+J.+Parkauthor=J.+Leeauthor=S.+Kim&title=Chemical+suppression+of+an+oncogenic+splicing+variant+of+AIMP2+induces+tumour+regression&doi=10.1042%2FBJ20130550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical suppression of an oncogenic splicing variant of AIMP2 induces tumour regression</span></div><div class="casAuthors">Lee, Hee Sook; Kim, Dae Gyu; Oh, Young Sun; Kwon, Nam Hoon; Lee, Jin Young; Kim, Doyeun; Park, Song-Hwa; Song, Jong-Hwan; Lee, Sunkyung; Han, Jung Min; Park, Bum-Joon; Lee, Jongkook; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">454</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">411-416</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">AIMP2 (aminoacyl-tRNA synthetase-interacting multifunctional protein 2) is a potent tumor suppressor that induces apoptosis in response to various oncogenic signals.  AIMP2-DX2, an exon2-deleted splicing variant of AIMP2, is up-regulated in lung cancer and competitively suppresses the pro-apoptotic activity of AIMP2, resulting in tumorigenesis.  In the present study we report that BC-DXI01, a synthetic compd., specifically reduces the cellular levels of AIMP2-DX2 through selective degrdn. of the AIMP2-DX2 mRNA transcript.  We found that BC-DXI01-mediated cell death pos. correlates with AIMP2-DX2 expression in the lung cancer cell lines tested.  Administration of BC-DXI01 in a AIMP2-DX2-driven tumor xenograft mice model led to reduced tumor sizes and vols. of up to 60% in comparison with vehicle-treated mice group, consistent with decreases in AIMP2-DX2 transcript and protein levels.  Taken together, our findings suggest that tumorigenic activity of AIMP2-DX2 can be controlled by the small chem. BC-DXI01, which can selectively suppress the AIMP2-DX2 mRNA transcript.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouKh0xvh3D4bVg90H21EOLACvtfcHk0liMtlnDbSzx_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGmt7jN&md5=9b4b85449eab8ca18a041fff00afb905</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1042%2FBJ20130550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20130550%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BS.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DOh%26aufirst%3DY.%2BS.%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DPark%26aufirst%3DS.%2BH.%26aulast%3DSong%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DChemical%2520suppression%2520of%2520an%2520oncogenic%2520splicing%2520variant%2520of%2520AIMP2%2520induces%2520tumour%2520regression%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D454%26spage%3D411%26epage%3D416%26doi%3D10.1042%2FBJ20130550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span> <span> </span><span class="NLM_article-title">Inhibiting DX2-p14/ARF interaction exerts antitumor effects in lung cancer and delays tumor progression</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">4791</span>– <span class="NLM_lpage">4804</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-1025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1158%2F0008-5472.CAN-15-1025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=27302160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtleiu7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=4791-4804&author=A.+Y.+Ohauthor=Y.+S.+Jungauthor=J.+Kimauthor=J.+H.+Leeauthor=J.+H.+Choauthor=H.+Y.+Chunauthor=S.+Parkauthor=H.+Parkauthor=S.+Limauthor=N.+C.+Haauthor=J.+S.+Parkauthor=C.+S.+Parkauthor=G.+Y.+Songauthor=B.+J.+Park&title=Inhibiting+DX2-p14%2FARF+interaction+exerts+antitumor+effects+in+lung+cancer+and+delays+tumor+progression&doi=10.1158%2F0008-5472.CAN-15-1025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting DX2-p14/ARF Interaction Exerts Antitumor Effects in Lung Cancer and Delays Tumor Progression</span></div><div class="casAuthors">Oh, Ah-Young; Jung, Youn Sang; Kim, Jiseon; Lee, Jee-Hyun; Cho, Jung-Hyun; Chun, Ho-Young; Park, Soyoung; Park, Hyunchul; Lim, Sikeun; Ha, Nam-Chul; Park, Jong Sook; Park, Choon-Sik; Song, Gyu-Yong; Park, Bum-Joon</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4791-4804</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The aminoacyl tRNA synthetase complex-interacting multifunctional protein 2 (AIMP2) splice variant designated DX2 is induced by cigarette smoke carcinogens and is often detected in human lung cancer specimens.  However, the function of DX2 in lung carcinogenesis is obscure.  In this study, we found that DX2 expression was induced by oncogenes in human lung cancer tissues and cells.  DX2 prevented oncogene-induced apoptosis and senescence and promoted drug resistance by directly binding to and inhibiting p14/ARF.  Through chem. screening, we identified SLCB050, a novel compd. that blocks the interaction between DX2 and p14/ARF in vitro and in vivo.  SLCB050 reduced the viability of human lung cancer cells, esp. small cell lung cancer cells, in a p14/ARF-dependent manner.  Moreover, in a mouse model of K-Ras-driven lung tumorigenesis, ectopic expression of DX2 induced small cell and non-small cell lung cancers, both of which could be suppressed by SLCB050 treatment.  Taken together, our findings show how DX2 promotes lung cancer progression and how its activity may be thwarted as a strategy to treat patients with lung cancers exhibiting elevated DX2 levels.  Cancer Res; 76(16); 4791-804. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomT7AJTx-0NbVg90H21EOLACvtfcHk0lhswcrUp_Bi4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtleiu7zI&md5=5496a0d901d7457292140cfc22e621b2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-1025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-1025%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DA.%2BY.%26aulast%3DJung%26aufirst%3DY.%2BS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DChun%26aufirst%3DH.%2BY.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DHa%26aufirst%3DN.%2BC.%26aulast%3DPark%26aufirst%3DJ.%2BS.%26aulast%3DPark%26aufirst%3DC.%2BS.%26aulast%3DSong%26aufirst%3DG.%2BY.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26atitle%3DInhibiting%2520DX2-p14%252FARF%2520interaction%2520exerts%2520antitumor%2520effects%2520in%2520lung%2520cancer%2520and%2520delays%2520tumor%2520progression%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D4791%26epage%3D4804%26doi%3D10.1158%2F0008-5472.CAN-15-1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span>; <span class="NLM_string-name">Song, G. Y.</span>; <span class="NLM_string-name">Oh, A. Y.</span>; <span class="NLM_string-name">Lee, J. H.</span>; <span class="NLM_string-name">Her, J. H.</span></span> <span> </span><span class="NLM_article-title">Compound for Inhibiting Binding between DX2 Protein and p14/ARF Protein, and Pharmaceutical Composition for Treating or Preventing Cancer Disease Containing same as Effective Ingredient</span>. <span class="NLM_patent">US10280178B2</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=B.+J.+Park&author=G.+Y.+Song&author=A.+Y.+Oh&author=J.+H.+Lee&author=J.+H.+Her&title=Compound+for+Inhibiting+Binding+between+DX2+Protein+and+p14%2FARF+Protein%2C+and+Pharmaceutical+Composition+for+Treating+or+Preventing+Cancer+Disease+Containing+same+as+Effective+Ingredient"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DB.%2BJ.%26atitle%3DCompound%2520for%2520Inhibiting%2520Binding%2520between%2520DX2%2520Protein%2520and%2520p14%252FARF%2520Protein%252C%2520and%2520Pharmaceutical%2520Composition%2520for%2520Treating%2520or%2520Preventing%2520Cancer%2520Disease%2520Containing%2520same%2520as%2520Effective%2520Ingredient%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span>; <span class="NLM_string-name">Kwon, N. H.</span>; <span class="NLM_string-name">Kim, D. G.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical Composition for Preventing or Treating Cancer</span>. <span class="NLM_patent">US9539236B2</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=S.+Kim&author=N.+H.+Kwon&author=D.+G.+Kim&title=Pharmaceutical+Composition+for+Preventing+or+Treating+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26atitle%3DPharmaceutical%2520Composition%2520for%2520Preventing%2520or%2520Treating%2520Cancer%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, P.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel 7-amino-[1,2,4]triazolo[4,3-f]pteridinone, and 7-aminotetrazolo[1,5-f]pteridinone derivative as potent antitumor agents</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">690</span>– <span class="NLM_lpage">709</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.12.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1016%2Fj.ejmech.2018.12.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=30572179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFygur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2019&pages=690-709&author=Y.+Houauthor=L.+Zhuauthor=Z.+Liauthor=Q.+Shenauthor=Q.+Xuauthor=W.+Liauthor=Y.+Liuauthor=P.+Gong&title=Design%2C+synthesis+and+biological+evaluation+of+novel+7-amino-%5B1%2C2%2C4%5Dtriazolo%5B4%2C3-f%5Dpteridinone%2C+and+7-aminotetrazolo%5B1%2C5-f%5Dpteridinone+derivative+as+potent+antitumor+agents&doi=10.1016%2Fj.ejmech.2018.12.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel 7-amino-[1,2,4]triazolo[4,3-f]pteridinone, and 7-aminotetrazolo[1,5-f]pteridinone derivative as potent antitumor agents</span></div><div class="casAuthors">Hou, Yunlei; Zhu, Liangyu; Li, Zhiwei; Shen, Qi; Xu, Qiaoling; Li, Wei; Liu, Yajing; Gong, Ping</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">690-709</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">To develop novel therapeutic agents with anticancer activities, two series of novel 7-amino-[1,2,4]triazolo[4,3-f]pteridinone, and 7-aminotetrazolo[1,5-f]pteridinone derivs. were designed and synthesized.  All compds. were tested for anti-proliferative activities against five cancer cell lines.  The structure-activity relationships (SARs) studies were conducted through the variation in two regions, the moiety of A ring and the terminal aniline B on pteridinone core. 1-Methyl-1,2,4-triazole deriv. I with 2,6-dimethylpiperazine showed the most potent antiproliferative activity against A549, PC-3, HCT116, MCF-7 and MDA-MB-231 cell lines with IC50 values of 0.16 μM, 0.30 μM, 0.51 μM, 0.30 μM, and 0.70 μM, resp.  Combined with the results of the mol. docking and enzymic studies, the PLK1 was very likely to be one of the drug targets of compd. I.  Furthermore, to clarify the anticancer mechanism of compd. I, further explorations in the bioactivity were conducted.  The results showed that compd. I obviously inhibited proliferation of A549 cell lines, induced a great decrease in mitochondrial membrane potential leading to apoptosis of cancer cells, suppressed the migration of tumor cells, and arrested G1 phase of A549 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj6IslFQC9NrVg90H21EOLACvtfcHk0lhswcrUp_Bi4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFygur7F&md5=e95279a0b95740db3d9ef69892325ab9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.12.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.12.009%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DP.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%25207-amino-%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-f%255Dpteridinone%252C%2520and%25207-aminotetrazolo%255B1%252C5-f%255Dpteridinone%2520derivative%2520as%2520potent%2520antitumor%2520agents%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D163%26spage%3D690%26epage%3D709%26doi%3D10.1016%2Fj.ejmech.2018.12.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span>; <span class="NLM_string-name">Rajagopalan, S.</span>; <span class="NLM_string-name">Belavagi, N.</span>; <span class="NLM_string-name">Ramachandra, M.</span></span> <span> </span><span class="NLM_article-title">2,4-Diaminopyrimidine Derivatives as Protein Kinase Inhibitors</span>. <span class="NLM_patent">WO 2012/059932Al</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=S.+Sengupta&author=S.+Rajagopalan&author=N.+Belavagi&author=M.+Ramachandra&title=2%2C4-Diaminopyrimidine+Derivatives+as+Protein+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSengupta%26aufirst%3DS.%26atitle%3D2%252C4-Diaminopyrimidine%2520Derivatives%2520as%2520Protein%2520Kinase%2520Inhibitors%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span> <span> </span><span class="NLM_article-title">Identification of novel 2-aminothiazole conjugated nitrofuran as antitubercular and antibacterial agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3669</span>– <span class="NLM_lpage">3674</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.05.088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1016%2Fj.bmcl.2016.05.088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=27289321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1ynu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=3669-3674&author=K.+Ranauthor=C.+Gaoauthor=H.+Dengauthor=Q.+Leiauthor=X.+Youauthor=N.+Wangauthor=Y.+Shiauthor=Z.+Liuauthor=W.+Weiauthor=C.+Pengauthor=L.+Xiongauthor=K.+Xiaoauthor=L.+Yu&title=Identification+of+novel+2-aminothiazole+conjugated+nitrofuran+as+antitubercular+and+antibacterial+agents&doi=10.1016%2Fj.bmcl.2016.05.088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel 2-aminothiazole conjugated nitrofuran as antitubercular and antibacterial agents</span></div><div class="casAuthors">Ran, Kai; Gao, Chao; Deng, Hongxia; Lei, Qian; You, Xinyu; Wang, Ningyu; Shi, Yaojie; Liu, Zhihao; Wei, Wei; Peng, Cuiting; Xiong, Lu; Xiao, Kunjie; Yu, Luoting</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3669-3674</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The emergence of antibiotic resistant pathogens is an ongoing main problem in the therapy of bacterial infections.  In order to develop promising antitubercular and antibacterial lead compds., the authors designed and synthesized a new series of derivs. of 2-aminothiazole conjugated nitrofuran with activities against both Mycobacterium tuberculosis and Staphylococcus aureus.  Eight compds. emerged as promising antitubercular agents.  Structure-activity relationships (SARs) were discussed and showed that the derivs. substituted at the position-3 of benzene of 5-nitro-N-(4-phenylthiazol-2-yl)furan-2-carboxamide exhibited superior potency.  The most potent compd. I, substituted with benzamide at this position, displayed min. inhibitory concns. (MICs) of 0.27 μg/mL against Mtb H37Ra and 1.36 μg/mL against S. aureus.  Furthermore, compd. I had no obvious cytotoxicity to normal Vero cells (IC50 = 50.2 μM).  The results suggest that the novel scaffolds of aminothiazole conjugated nitrofuran would be a promising class of potent antitubercular and antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvVRT2hzqQSbVg90H21EOLACvtfcHk0lh5lPBNdb69nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1ynu7Y%253D&md5=838ecf4c24b220659817d0abeb00697d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.05.088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.05.088%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DLei%26aufirst%3DQ.%26aulast%3DYou%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DXiong%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DL.%26atitle%3DIdentification%2520of%2520novel%25202-aminothiazole%2520conjugated%2520nitrofuran%2520as%2520antitubercular%2520and%2520antibacterial%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D3669%26epage%3D3674%26doi%3D10.1016%2Fj.bmcl.2016.05.088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jyh Huang, J.</span>; <span class="NLM_string-name">Chuang, S. H.</span>; <span class="NLM_string-name">Lee, Y. E.</span>; <span class="NLM_string-name">Huang, Y.</span>; <span class="NLM_string-name">Lau, J.</span></span> <span> </span><span class="NLM_article-title">Modulators of Hec1 Activity and Methods Therefor</span>. <span class="NLM_patent">US 2012/059932Al</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+Jyh+Huang&author=S.+H.+Chuang&author=Y.+E.+Lee&author=Y.+Huang&author=J.+Lau&title=Modulators+of+Hec1+Activity+and+Methods+Therefor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJyh%2BHuang%26aufirst%3DJ.%26atitle%3DModulators%2520of%2520Hec1%2520Activity%2520and%2520Methods%2520Therefor%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goughnour, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paek, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seol, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Novel morphologic and genetic analysis of cancer cells in a 3D microenvironment identifies STAT3 as a regulator of tumor permeability barrier function</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">1044</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-14-2611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1158%2F0008-5472.CAN-14-2611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=26676754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslCltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=1044-1054&author=M.+C.+Parkauthor=H.+Jeongauthor=S.+H.+Sonauthor=Y.+Kimauthor=D.+Hanauthor=P.+C.+Goughnourauthor=T.+Kangauthor=N.+H.+Kwonauthor=H.+E.+Moonauthor=S.+H.+Paekauthor=D.+Hwangauthor=H.+J.+Seolauthor=D.+H.+Namauthor=S.+Kim&title=Novel+morphologic+and+genetic+analysis+of+cancer+cells+in+a+3D+microenvironment+identifies+STAT3+as+a+regulator+of+tumor+permeability+barrier+function&doi=10.1158%2F0008-5472.CAN-14-2611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Morphologic and Genetic Analysis of Cancer Cells in a 3D Microenvironment Identifies STAT3 as a Regulator of Tumor Permeability Barrier Function</span></div><div class="casAuthors">Park, Min Chul; Jeong, Hyobin; Son, Sung Hwa; Kim, YounHa; Han, Daeyoung; Goughnour, Peter C.; Kang, Taehee; Kwon, Nam Hoon; Moon, Hyo Eun; Paek, Sun Ha; Hwang, Daehee; Seol, Ho Jun; Nam, Do-Hyun; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1044-1054</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Tumor permeability is a crit. determinant of drug delivery and sensitivity, but systematic methods to identify factors that perform permeability barrier functions in the tumor microenvironment are not yet available.  Multicellular tumor spheroids have become tractable in vitro models to study the impact of a three-dimensional (3D) environment on cellular behavior.  In this study, we characterized the spheroid-forming potential of cancer cells and correlated the resulting spheroid morphologies with genetic information to identify conserved cellular processes assocd. with spheroid structure.  Spheroids generated from 100 different cancer cell lines were classified into four distinct groups based on morphol.  In particular, round and compact spheroids exhibited highly hypoxic inner cores and permeability barriers against anticancer drugs.  Through systematic and correlative anal., we reveal JAK-STAT signaling as one of the signature pathways activated in round spheroids.  Accordingly, STAT3 inhibition in spheroids generated from the established cancer cells and primary glioblastoma patient-derived cells altered the rounded morphol. and increased drug sensitivity.  Furthermore, combined administration of the STAT3 inhibitor and 5-fluorouracil to a mouse xenograft model markedly reduced tumor growth compared with monotherapy.  Collectively, our findings demonstrate the ability to integrate 3D culture and genetic profiling to det. the factors underlying the integrity of the permeability barrier in the tumor microenvironment, and may help to identify and exploit novel mechanisms of drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0yDgPJn8uBbVg90H21EOLACvtfcHk0lh5lPBNdb69nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslCltLw%253D&md5=db43b2732ec9282c4e875cda92e2ab52</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-2611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-2611%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DM.%2BC.%26aulast%3DJeong%26aufirst%3DH.%26aulast%3DSon%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DD.%26aulast%3DGoughnour%26aufirst%3DP.%2BC.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DMoon%26aufirst%3DH.%2BE.%26aulast%3DPaek%26aufirst%3DS.%2BH.%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DSeol%26aufirst%3DH.%2BJ.%26aulast%3DNam%26aufirst%3DD.%2BH.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DNovel%2520morphologic%2520and%2520genetic%2520analysis%2520of%2520cancer%2520cells%2520in%2520a%25203D%2520microenvironment%2520identifies%2520STAT3%2520as%2520a%2520regulator%2520of%2520tumor%2520permeability%2520barrier%2520function%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D1044%26epage%3D1054%26doi%3D10.1158%2F0008-5472.CAN-14-2611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+M.+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Sastry G Madhavi; Adzhigirey Matvey; Day Tyler; Annabhimoju Ramakrishna; Sherman Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepared prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove atomic clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addition, ligands must be prepared to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system preparation is generally accepted in the field, an extensive study of the preparation steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in preparing a system for virtual screening.  We first explore a large number of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper preparation and that neglecting certain steps of the preparation process produces a systematic degradation in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the preparation that impact database enrichment.  While the work presented here was performed with the Protein Preparation Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQd2Qiai-OqeXqj_yg7twvpfW6udTcc2eY6ch86zTJ7M7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D&md5=9eb1a820e121aca2742ed53f24481ace</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0liIznq6VJYDCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0liIznq6VJYDCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0liHoOravOfSAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Felagot A. Abebe, Megan D. Hopkins, Suraj N. Vodnala, Robert J. Sheaff, <span class="NLM_string-name hlFld-ContribAuthor">Angus A. Lamar</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Rapid In Vitro Screening Assay Using Metabolic Inhibitors to Detect Highly Selective Anticancer Agents. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (28)
                                     , 18333-18343. <a href="https://doi.org/10.1021/acsomega.1c02203" title="DOI URL">https://doi.org/10.1021/acsomega.1c02203</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.1c02203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.1c02203%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DDevelopment%252Bof%252Ba%252BRapid%252BIn%252BVitro%252BScreening%252BAssay%252BUsing%252BMetabolic%252BInhibitors%252Bto%252BDetect%252BHighly%252BSelective%252BAnticancer%252BAgents%26aulast%3DAbebe%26aufirst%3DFelagot%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D26042021%26date%3D23062021%26date%3D02072021%26volume%3D6%26issue%3D28%26spage%3D18333%26epage%3D18343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">András Gy.  Németh</span>, <span class="hlFld-ContribAuthor ">Bence  Marlok</span>, <span class="hlFld-ContribAuthor ">Attila  Domján</span>, <span class="hlFld-ContribAuthor ">Qinghe  Gao</span>, <span class="hlFld-ContribAuthor ">Xinya  Han</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>, <span class="hlFld-ContribAuthor ">Péter  Ábrányi‐Balogh</span>. </span><span class="cited-content_cbyCitation_article-title">Convenient Multicomponent One‐Pot Synthesis of 2‐Iminothiazolines and 2‐Aminothiazoles Using Elemental Sulfur Under Aqueous Conditions. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>2021 </em>
                                    (25)
                                     , 3587-3597. <a href="https://doi.org/10.1002/ejoc.202100548" title="DOI URL">https://doi.org/10.1002/ejoc.202100548</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202100548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202100548%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DConvenient%252BMulticomponent%252BOne%2525E2%252580%252590Pot%252BSynthesis%252Bof%252B2%2525E2%252580%252590Iminothiazolines%252Band%252B2%2525E2%252580%252590Aminothiazoles%252BUsing%252BElemental%252BSulfur%252BUnder%252BAqueous%252BConditions%26aulast%3DN%25C3%25A9meth%26aufirst%3DAndr%25C3%25A1s%2BGy.%26date%3D2021%26date%3D2021%26volume%3D2021%26issue%3D25%26spage%3D3587%26epage%3D3597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Soong-Hyun  Kim</span>, <span class="hlFld-ContribAuthor ">Seri  Bae</span>, <span class="hlFld-ContribAuthor ">Minsoo  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Development of Aminoacyl-tRNA Synthetase Inhibitors for Human Diseases: A Future Perspective. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2020,</strong> <em>10 </em>
                                    (12)
                                     , 1625. <a href="https://doi.org/10.3390/biom10121625" title="DOI URL">https://doi.org/10.3390/biom10121625</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom10121625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom10121625%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DRecent%252BDevelopment%252Bof%252BAminoacyl-tRNA%252BSynthetase%252BInhibitors%252Bfor%252BHuman%252BDiseases%25253A%252BA%252BFuture%252BPerspective%26aulast%3DKim%26aufirst%3DSoong-Hyun%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D12%26spage%3D1625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Zhou</span>, <span class="hlFld-ContribAuthor ">Bao  Sun</span>, <span class="hlFld-ContribAuthor ">Anzheng  Nie</span>, <span class="hlFld-ContribAuthor ">Dongsheng  Yu</span>, <span class="hlFld-ContribAuthor ">Meng  Bian</span>. </span><span class="cited-content_cbyCitation_article-title">Roles of Aminoacyl-tRNA Synthetases in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fcell.2020.599765" title="DOI URL">https://doi.org/10.3389/fcell.2020.599765</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2020.599765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2020.599765%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DRoles%252Bof%252BAminoacyl-tRNA%252BSynthetases%252Bin%252BCancer%26aulast%3DZhou%26aufirst%3DZheng%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic model for DX2-dependent cancer progression. DX2 is degraded by Siah1-mediated ubiquitination (upper). But in the presence of HSP70, DX2 binds with HSP70 and is protected from Siah1, stabilizing DX2 protein levels. Stabilized DX2 reduces p53 and p14 protein levels upon UV exposure and oncogenic signals, respectively. Stabilized DX2 also increases FBP and TRAF2 upon TGFβ and TNF-α transduction, respectively, resulting in enhanced cancer progression (bottom). Ub = ubiquitin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of known sulfonamide based small molecule inhibitors and strategy for SAR exploration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) <i>p</i>-toluenesulfonyl chloride, NaOH, H<sub>2</sub>O, rt, 12 h, 52% and 58%; (b) amine, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, 21–91%.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0008.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) amine, K<sub>2</sub>CO<sub>3</sub>, DMF, 70°C, 5–6 h, 83–93%; (b) H<sub>2</sub>, Pd/C, MeOH, rt, 2–3 h, 94–99%; (c) amine, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, 21–91%.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0009.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) <i>p</i>-toluic acid, EDCI·HCl, HOBt, TEA, DCM, rt, 12 h, 18–94%; (b) H<sub>2</sub>, Pd/C, MeOH, 2 h; (c) 4-morpholinoaniline, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, 24%.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0010.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) arylsulfonyl chloride, NaOH, H<sub>2</sub>O, rt, 12 h, 14–59%; (b) 4-morpholinoaniline, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, 31–80%.</p></p></figure><figure data-id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0011.jpeg" id="rightTab-GRAPHIC-d7e709-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) <i>p</i>-toluenesulfonyl chloride, NaOH, H<sub>2</sub>O, rt, 12 h, 13–92%; (b) 4-morpholinoaniline, EDCI·HCl, HOBt, TEA, DCM, rt, 18 h, and then (c) 5 M NaOH, EtOH, reflux, 4 h for <b>17j</b> and H<sub>2</sub>, Pd/C, MeOH, rt, 3 h for <b>17g</b> and <b>17h</b>, 31–68%.</p></p></figure><figure data-id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0012.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and condition: (a) arylsulfonyl chloride, NaOH, H<sub>2</sub>O, rt, 12 h, 40–54%; (b) amine, EDCI·HCl, HOBt, TEA, DCM, rt, 3–4 h, 31–92%; (c) amine, EDCI·HCl, HOBt, DIPEA, DMF, 60 °C, 16 h, for <b>20d</b>–<b>21f</b>, 31–75%.</p></p></figure><figure data-id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0004.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <b>BC-DXI-843</b> is a specific inhibitor of DX2. (A) DX2 and AIMP2 levels were measured using nanoluciferase-DX2 and AIMP2, respectively, after treatment with each of the six compounds (40 μM, 4 h). The experiment was independently repeated three times with error bars denoting SD. The presented value was the inhibited luciferase activity upon treatment with each chemicals compared to DMSO as a control. (B) A549 cells expressing nanoluciferase-DX2 or AIMP2 were treated with <b>BC-DXI-843</b> in a dose-dependent manner for 4 h, and the resulting luminescence was measured. IC<sub>50</sub> values for the inhibition of DX2 and AIMP2 were calculated as shown. The experiment was independently repeated three times with error bars denoting SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0005.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Determining the binding mode of <b>BC-DXI-843</b> to DX2. (A) Fluorescence-based binding titration measurement of <b>BC-DXI-843</b> to DX2. Protein samples were excited at 280 nm, and the decrease in fluorescence emission upon binding was measured at 335 nm. (B) Docking pose with the binding site of our in-house homology model of DX2 and <b>BC-DXI-843</b> (red). (C) Close-up view of <b>BC-DXI-843</b> with the relevant interaction residues (D52, T82) and distances between them and <b>BC-DXI-843</b>’s closest atoms. <b>BC-DXI-843</b> was also found to be docked close to the residues in the hydrophobic pocket. The structure of DX2-GST was obtained from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A34">5A34</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0006.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <b>BC-DXI-843</b> suppresses cancer cell proliferation in a DX2-dependent manner. (A) <b>BC-DXI-843</b>-specific inhibition of cancer cell proliferation. A549 cancer cells and WI-26 normal cells were treated with <b>BC-DXI-843</b> as indicated for 72 h, and cell viability was determined with an MTT assay. The EC<sub>50</sub>, 50% inhibiting concentration of cell viability, is shown on the right. N.D. = not determined. (B) Suppression of DX2-mediated cancer cell proliferation by <b>BC-DXI-843</b>. Empty vector (EV)- or DX2-inducible A549 cells (Ind-EV and Ind-DX2, respectively) were treated with doxycycline (Dox) and <b>BC-DXI-843</b> (<b>843</b>) as indicated, and cell viability was determined in the same manner as described above. (C) Inhibitory effect of <b>BC-DXI-843</b> on cancer cell spheroids formed in a 3D culture system. Several doses of <b>BC-DXI-843</b> were added to H460 spheroids over the course of 72 h. Cell toxicity was determined by a CellTiter-Glo 3D assay, and representative photos of the spheroids are shown. (A–C) All experiments were independently repeated three times with error bars denoting SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/medium/jm9b01961_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0007.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <i>In vivo</i> antitumor effect of <b>BC-DXI-843</b>. (A–D) Mice (<i>n</i> = 4) subcutaneously injected with H460 cells were intraperitoneally administered <b>BC-DXI-843</b> (50 mg/kg) every other day for 15 days. The sizes of the embedded tumors (B) and body weights (D) were measured throughout the experiment. After sacrifice, the weights of the harvested tumors were determined (C) and photos of whole-body bearing the tumor and the excised tumors were taken (A).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.9b01961/20200520/images/large/jm9b01961_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01961&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i99">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37200" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37200" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 29 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span> <span> </span><span class="NLM_article-title">Aminoacyl-tRNA synthetases and tumorigenesis: more than housekeeping</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">708</span>– <span class="NLM_lpage">718</span>, <span class="refDoi"> DOI: 10.1038/nrc3124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1038%2Fnrc3124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=21941282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ajurrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=708-718&author=S.+Kimauthor=S.+Youauthor=D.+Hwang&title=Aminoacyl-tRNA+synthetases+and+tumorigenesis%3A+more+than+housekeeping&doi=10.1038%2Fnrc3124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoacyl-tRNA synthetases and tumorigenesis: more than housekeeping</span></div><div class="casAuthors">Kim, Sung-Hoon; You, Sung-Yong; Hwang, Dae-Hee</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">708-718</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past decade, the identification of cancer-assocd. factors has been a subject of primary interest not only for understanding the basic mechanisms of tumorigenesis but also for discovering the assocd. therapeutic targets.  However, aminoacyl-tRNA synthetases (ARSs) have been overlooked, mostly because many assumed that they were simply 'housekeepers' that were involved in protein synthesis.  Mammalian ARSs have evolved many addnl. domains that are not necessarily linked to their catalytic activities.  With these domains, they interact with diverse regulatory factors.  In addn., the expression of some ARSs is dynamically changed depending on various cellular types and stresses.  This Anal. article addresses the potential pathophysiol. implications of ARSs in tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLu4m0EWOj57Vg90H21EOLACvtfcHk0liqjncopXC6BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ajurrO&md5=cfc0b098147984cf9728af848c637020</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc3124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3124%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DYou%26aufirst%3DS.%26aulast%3DHwang%26aufirst%3DD.%26atitle%3DAminoacyl-tRNA%2520synthetases%2520and%2520tumorigenesis%253A%2520more%2520than%2520housekeeping%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2011%26volume%3D11%26spage%3D708%26epage%3D718%26doi%3D10.1038%2Fnrc3124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Aminoacyl- tRNA synthetases as therapeutic targets</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">650</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0026-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1038%2Fs41573-019-0026-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=31073243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlajtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=629-650&author=N.+H.+Kwonauthor=P.+L.+Foxauthor=S.+Kim&title=Aminoacyl-+tRNA+synthetases+as+therapeutic+targets&doi=10.1038%2Fs41573-019-0026-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoacyl-tRNA synthetases as therapeutic targets</span></div><div class="casAuthors">Kwon, Nam Hoon; Fox, Paul L.; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">629-650</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Aminoacyl-tRNA synthetases (ARSs) are essential enzymes for protein synthesis with evolutionarily conserved enzymic mechanisms.  Despite their similarity across organisms, scientists have been able to generate effective anti-infective agents based on the structural differences in the catalytic clefts of ARSs from pathogens and humans.  However, recent genomic, proteomic and functionomic advances have unveiled unexpected disease-assocd. mutations and altered expression, secretion and interactions in human ARSs, revealing hidden biol. functions beyond their catalytic roles in protein synthesis.  These studies have also brought to light their potential as a rich and unexplored source for new therapeutic targets and agents through multiple avenues, including direct targeting of the catalytic sites, controlling disease-assocd. protein-protein interactions and developing novel biologics from the secreted ARS proteins or their parts.  This Review addresses the emerging biol. and therapeutic applications of human ARSs in diseases including autoimmune and rare diseases, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1lM6NQW2SDLVg90H21EOLACvtfcHk0liqjncopXC6BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlajtrY%253D&md5=dd9c680cd873d35c8ab10350cd7fb044</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0026-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0026-3%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DFox%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAminoacyl-%2520tRNA%2520synthetases%2520as%2520therapeutic%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D629%26epage%3D650%26doi%3D10.1038%2Fs41573-019-0026-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Hierarchical network between the components of the multi-tRNA synthetase complex: implications for complex formation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">38663</span>– <span class="NLM_lpage">38667</span>, <span class="refDoi"> DOI: 10.1074/jbc.M605211200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1074%2Fjbc.M605211200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=17062567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12msr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=38663-38667&author=J.+M.+Hanauthor=M.+J.+Leeauthor=S.+G.+Parkauthor=S.+H.+Leeauthor=E.+Razinauthor=E.+C.+Choiauthor=S.+Kim&title=Hierarchical+network+between+the+components+of+the+multi-tRNA+synthetase+complex%3A+implications+for+complex+formation&doi=10.1074%2Fjbc.M605211200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Hierarchical Network between the Components of the Multi-tRNA Synthetase Complex: Implications for complex formation</span></div><div class="casAuthors">Han, Jung Min; Lee, Min Ji; Park, Sang Gyu; Lee, Sun Hee; Razin, Ehud; Choi, Eung-Chil; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">38663-38667</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The macromol. tRNA synthetase complex consists of nine different enzymes and three non-enzymic factors.  This complex was recently shown to be a novel signalosome, since many of its components are involved in signaling pathways in addn. to their catalytic roles in protein synthesis.  The structural organization and dynamic relationships of the components of the complex are not well understood.  Here we performed a systematic depletion anal. to det. the effects of structural intimacy and the turnover of the components.  The results showed that the stability of some components depended on their neighbors.  Lysyl-tRNA synthetase was most independent of other components for its stability whereas it was most required for the stability of other components.  Arginyl- and methionyl-tRNA synthetases had the opposite characteristics.  Thus, the systematic depletion of the components revealed the functional reason for the complex formation and the assembly pattern of these multi-functional enzymes and their assocd. factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJHgRwZP0CkrVg90H21EOLACvtfcHk0liqjncopXC6BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12msr3P&md5=14ee04049d062216dcecca3cc66f6137</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M605211200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M605211200%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DS.%2BG.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DRazin%26aufirst%3DE.%26aulast%3DChoi%26aufirst%3DE.%2BC.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DHierarchical%2520network%2520between%2520the%2520components%2520of%2520the%2520multi-tRNA%2520synthetase%2520complex%253A%2520implications%2520for%2520complex%2520formation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D38663%26epage%3D38667%26doi%3D10.1074%2Fjbc.M605211200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Aminoacyl-tRNA synthetase-interacting multifunctional proteins (AIMPs): A triad for cellular homeostasis</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">296</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1002/iub.324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1002%2Fiub.324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=20306515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVCkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=296-302&author=S.+G.+Parkauthor=E.+C.+Choiauthor=S.+Kim&title=Aminoacyl-tRNA+synthetase-interacting+multifunctional+proteins+%28AIMPs%29%3A+A+triad+for+cellular+homeostasis&doi=10.1002%2Fiub.324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoacyl-tRNA synthetase-interacting multifunctional proteins (AIMPs): a triad for cellular homeostasis</span></div><div class="casAuthors">Park, Sang Gyu; Choi, Eung-Chil; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">296-302</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">A review.  Aminoacyl-tRNA synthetases (ARSs) are highly conserved for efficient and precise translation of genetic codes.  In higher eukaryotic systems, several different ARSs including glutamyl-, prolyl-, isoleucyl-, leucyl-, methionyl-, glutaminyl-, lysyl-, arginyl-, and aspartyl-tRNA synthetases form a macromol. protein complex with 3 nonenzymic protein cofactors (AIMP1/p43, AIMP2/p38, and AIMP3/p18).  Although the structure and functional implications for this complex formation are not completely understood, rapidly accumulating evidences suggest that this complex would work as a mol. hub linked to the multiple signaling pathways that involve the components of enzymes and cofactors.  Here, the roles of 3 nonenzymic protein components of the multi-tRNA synthetase complex in the assembly of the components and in cell regulation are addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqVLxruiaOp7Vg90H21EOLACvtfcHk0lj6gtN2BrxYAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVCkt7Y%253D&md5=5aaec0e7bb1bbf74b649771bd91fcbb7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fiub.324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.324%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DS.%2BG.%26aulast%3DChoi%26aufirst%3DE.%2BC.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAminoacyl-tRNA%2520synthetase-interacting%2520multifunctional%2520proteins%2520%2528AIMPs%2529%253A%2520A%2520triad%2520for%2520cellular%2520homeostasis%26jtitle%3DIUBMB%2520Life%26date%3D2010%26volume%3D62%26spage%3D296%26epage%3D302%26doi%3D10.1002%2Fiub.324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">AIMP2/p38, the scaffold for the multi-tRNA synthetase complex, responds to genotoxic stresses via p53</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">11206</span>– <span class="NLM_lpage">11211</span>, <span class="refDoi"> DOI: 10.1073/pnas.0800297105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1073%2Fpnas.0800297105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=18695251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVShtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=11206-11211&author=J.+M.+Hanauthor=B.+J.+Parkauthor=S.+G.+Parkauthor=Y.+S.+Ohauthor=S.+J.+Choiauthor=S.+W.+Leeauthor=S.+K.+Hwangauthor=S.+H.+Changauthor=M.+H.+Choauthor=S.+Kim&title=AIMP2%2Fp38%2C+the+scaffold+for+the+multi-tRNA+synthetase+complex%2C+responds+to+genotoxic+stresses+via+p53&doi=10.1073%2Fpnas.0800297105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">AIMP2/p38, the scaffold for the multi-tRNA synthetase complex, responds to genotoxic stresses via p53</span></div><div class="casAuthors">Han, Jung Min; Park, Bum-Joon; Park, Sang Gyu; Oh, Young Sun; Choi, So Jung; Lee, Sang Won; Hwang, Soon-Kyung; Chang, Seung-Hee; Cho, Myung-Haing; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">11206-11211</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">AIMP2/p38 is a scaffolding protein required for the assembly of the macromol. tRNA synthetase complex.  Here, we describe a previously unknown function for AIMP2 as a pos. regulator of p53 in response to genotoxic stresses.  Depletion of AIMP2 increased resistance to DNA damage-induced apoptosis, and introduction of AIMP2 into AIMP2-deficient cells restored the susceptibility to apoptosis.  Upon DNA damage, AIMP2 was phosphorylated, dissocd. from the multi-tRNA synthetase complex, and translocated into the nuclei of cells.  AIMP2 directly interacts with p53, thereby preventing MDM2-mediated ubiquitination and degrdn. of p53.  Mutations in AIMP2, affecting its interaction with p53, hampered its ability to activate p53.  Nutlin-3 recovered the level of p53 and the susceptibility to UV-induced cell death in AIMP2-deficient cells.  This work demonstrates that AIMP2, a component of the translational machinery, functions as proapoptotic factor via p53 in response to DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3ezVoQ1fGhbVg90H21EOLACvtfcHk0lj6gtN2BrxYAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVShtbnE&md5=bd708b2ee1ae05ba7bb81a6e7d034bda</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0800297105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0800297105%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DPark%26aufirst%3DS.%2BG.%26aulast%3DOh%26aufirst%3DY.%2BS.%26aulast%3DChoi%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BW.%26aulast%3DHwang%26aufirst%3DS.%2BK.%26aulast%3DChang%26aufirst%3DS.%2BH.%26aulast%3DCho%26aufirst%3DM.%2BH.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAIMP2%252Fp38%252C%2520the%2520scaffold%2520for%2520the%2520multi-tRNA%2520synthetase%2520complex%252C%2520responds%2520to%2520genotoxic%2520stresses%2520via%2520p53%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D11206%26epage%3D11211%26doi%3D10.1073%2Fpnas.0800297105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Downregulation of FUSE-binding protein and c-myc by tRNA synthetase cofactor p38 is required for lung cell differentiation</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">330</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/ng1182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1038%2Fng1182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=12819782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFSrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2003&pages=330-336&author=M.+J.+Kimauthor=B.+J.+Parkauthor=Y.+S.+Kangauthor=H.+J.+Kimauthor=J.+H.+Parkauthor=J.+W.+Kangauthor=S.+W.+Leeauthor=J.+M.+Hanauthor=H.+W.+Leeauthor=S.+Kim&title=Downregulation+of+FUSE-binding+protein+and+c-myc+by+tRNA+synthetase+cofactor+p38+is+required+for+lung+cell+differentiation&doi=10.1038%2Fng1182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of FUSE-binding protein and c-myc by tRNA synthetase cofactor p38 is required for lung cell differentiation</span></div><div class="casAuthors">Kim, Min Jung; Park, Bum-Joon; Kang, Young-Sun; Kim, Hyoung June; Park, Jae-Hyun; Kang, Jung Woo; Lee, Sang Won; Han, Jung Min; Lee, Han-Woong; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">330-336</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P38 is assocd. with a macromol. tRNA synthetase complex.  It has an essential role as a scaffold for the complex, and genetic disruption of p38 in mice causes neonatal lethality.  Here we investigated the mol. mechanisms underlying lethality of p38-mutant mice. p38-deficient mice showed defects in lung differentiation and respiratory distress syndrome. p38 was found to interact with FUSE-binding protein (FBP), a transcriptional activator of c-myc.  Binding of p38 stimulated ubiquitination and degrdn. of FBP, leading to downregulation of c-myc, which is required for differentiation of functional alveolar type II cells.  Transforming growth factor-β (TGF-β) induced p38 expression and promoted its translocation to nuclei for the regulation of FBP and c-myc.  Thus, this work identified a new activity of p38 as a mediator of TGF-β signaling and its functional importance in the control of c-myc during lung differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhjxINrMthVbVg90H21EOLACvtfcHk0lj6gtN2BrxYAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFSrs7g%253D&md5=9523a4243e6696a071ac65ccf790f7b1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fng1182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng1182%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DKang%26aufirst%3DY.%2BS.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DPark%26aufirst%3DJ.%2BH.%26aulast%3DKang%26aufirst%3DJ.%2BW.%26aulast%3DLee%26aufirst%3DS.%2BW.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DH.%2BW.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DDownregulation%2520of%2520FUSE-binding%2520protein%2520and%2520c-myc%2520by%2520tRNA%2520synthetase%2520cofactor%2520p38%2520is%2520required%2520for%2520lung%2520cell%2520differentiation%26jtitle%3DNat.%2520Genet.%26date%3D2003%26volume%3D34%26spage%3D330%26epage%3D336%26doi%3D10.1038%2Fng1182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Um, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">AIMP2 promotes TNFalpha-dependent apoptosis via ubiquitin-mediated degradation of TRAF2</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">2710</span>– <span class="NLM_lpage">2715</span>, <span class="refDoi"> DOI: 10.1242/jcs.049767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1242%2Fjcs.049767" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=19584093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGhsLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2009&pages=2710-2715&author=J.+W.+Choiauthor=D.+G.+Kimauthor=M.+C.+Parkauthor=J.+Y.+Umauthor=J.+M.+Hanauthor=S.+G.+Parkauthor=E.+C.+Choiauthor=S.+Kim&title=AIMP2+promotes+TNFalpha-dependent+apoptosis+via+ubiquitin-mediated+degradation+of+TRAF2&doi=10.1242%2Fjcs.049767"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">AIMP2 promotes TNFα-dependent apoptosis via ubiquitin-mediated degradation of TRAF2</span></div><div class="casAuthors">Choi, Jin Woo; Kim, Dae Gyu; Park, Min Chul; Um, Jung Yeon; Han, Jung Min; Park, Sang Gyu; Choi, Eung-Chil; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2710-2715</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">AIMP2 (aminoacyl-tRNA synthetase interacting multifunctional protein 2; also known as JTV-1) was first identified as p38 in a macromol. protein complex that consisted of nine different aminoacyl-tRNA synthetases and two other auxiliary factors.  AIMP2 also plays pivotal roles in the regulation of cell proliferation and death.  Although AIMP2 was previously shown to augment TNFα-induced cell death, its working mechanism in this signal pathway was not understood.  Here, we investigate the functional significance and mode of action of AIMP2 in TNFα signaling.  TNFα-induced cell death was compromised in AIMP2-deficient or -suppressed cells and exogenous supplementation of AIMP2 augmented apoptotic sensitivity to TNFα signaling.  This activity was confirmed by the AIMP2-dependent increase of IκB and suppression of NFκB.  We found binding of AIMP2 to TRAF2, a key player in the TNFα signaling pathway.  AIMP2 augmented the assocn. of an E3 ubiquitin ligase, c-IAP1, with TRAF2, causing ubiquitin-dependent degrdn. of TRAF2.  These findings suggest that AIMP2 can mediate the pro-apoptotic activity of TNFα via the downregulation of TRAF2 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVp478RpzlLrVg90H21EOLACvtfcHk0lhqhv50th3PCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGhsLzO&md5=cf685c704a7d95b85637f3c6642ccef1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1242%2Fjcs.049767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.049767%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DPark%26aufirst%3DM.%2BC.%26aulast%3DUm%26aufirst%3DJ.%2BY.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DPark%26aufirst%3DS.%2BG.%26aulast%3DChoi%26aufirst%3DE.%2BC.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAIMP2%2520promotes%2520TNFalpha-dependent%2520apoptosis%2520via%2520ubiquitin-mediated%2520degradation%2520of%2520TRAF2%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2009%26volume%3D122%26spage%3D2710%26epage%3D2715%26doi%3D10.1242%2Fjcs.049767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yum, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jo, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y. Y.</span></span> <span> </span><span class="NLM_article-title">AIMP2 controls intestinal stem cell compartments and tumorigenesis by modulating Wnt/beta-catenin signaling</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">4559</span>– <span class="NLM_lpage">4568</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-3357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1158%2F0008-5472.CAN-15-3357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=27262173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gmt7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=4559-4568&author=M.+K.+Yumauthor=J.+S.+Kangauthor=A.+E.+Leeauthor=Y.+W.+Joauthor=J.+Y.+Seoauthor=H.+A.+Kimauthor=Y.+Y.+Kimauthor=J.+Seongauthor=E.+B.+Leeauthor=J.+H.+Kimauthor=J.+M.+Hanauthor=S.+Kimauthor=Y.+Y.+Kong&title=AIMP2+controls+intestinal+stem+cell+compartments+and+tumorigenesis+by+modulating+Wnt%2Fbeta-catenin+signaling&doi=10.1158%2F0008-5472.CAN-15-3357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">AIMP2 Controls Intestinal Stem Cell Compartments and Tumorigenesis by Modulating Wnt/β-Catenin Signaling</span></div><div class="casAuthors">Yum, Min Kyu; Kang, Jong-Seol; Lee, Al-Eum; Jo, Young-Woo; Seo, Ji-Yun; Kim, Hyun-A.; Kim, Yoon-Young; Seong, Jinwoo; Lee, Eun Byul; Kim, Ji-Hoon; Han, Jung Min; Kim, Sunghoon; Kong, Young-Yun</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4559-4568</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Wnt/β-catenin (CTNNB1) signaling is crucial for the proliferation and maintenance of intestinal stem cells (ISC), but excessive activation leads to ISC expansion and eventually colorectal cancer.  Thus, neg. regulators are required to maintain optimal levels of Wnt/β-catenin signaling.  Aminoacyl-tRNA synthetase-interacting multifunctional proteins (AIMP) function in protein synthesis, but have also been implicated in signaling cascades affecting angiogenesis, immunity, and apoptosis.  In this study, we investigated the relationship between AIMP2 and Wnt/β-catenin signaling in a murine model of intestinal homeostasis and tumorigenesis.  Hemizygous deletion of Aimp2 resulted in enhanced Wnt/β-catenin signaling, increased proliferation of cryptic epithelial cells, and expansion of ISC compartments.  In an ApcMin/+ background, Aimp2 hemizygosity increased adenoma formation.  Mechanistically, AIMP2 disrupted the interaction between AXIN and Dishevelled-1 (DVL1) to inhibit Wnt/β-catenin signaling by competing with AXIN.  Furthermore, AIMP2 inhibited intestinal organoid formation and growth by suppressing Wnt/β-catenin signaling in an Aimp2 gene dosage-dependent manner.  Collectively, our results showed that AIMP2 acts as a haploinsufficient tumor suppressor that fine-tunes Wnt/β-catenin signaling in the intestine, illuminating the regulation of ISC abundance and activity.  Cancer Res; 76(15); 4559-68. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw0FydKmEcA7Vg90H21EOLACvtfcHk0lhqhv50th3PCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gmt7fI&md5=407c3f1529faaa458c534cfc8d1838a1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-3357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-3357%26sid%3Dliteratum%253Aachs%26aulast%3DYum%26aufirst%3DM.%2BK.%26aulast%3DKang%26aufirst%3DJ.%2BS.%26aulast%3DLee%26aufirst%3DA.%2BE.%26aulast%3DJo%26aufirst%3DY.%2BW.%26aulast%3DSeo%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DH.%2BA.%26aulast%3DKim%26aufirst%3DY.%2BY.%26aulast%3DSeong%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DE.%2BB.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DY.%2BY.%26atitle%3DAIMP2%2520controls%2520intestinal%2520stem%2520cell%2520compartments%2520and%2520tumorigenesis%2520by%2520modulating%2520Wnt%252Fbeta-catenin%2520signaling%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D4559%26epage%3D4568%26doi%3D10.1158%2F0008-5472.CAN-15-3357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Won, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. W.</span></span> <span> </span><span class="NLM_article-title">Selective regression of cancer cells expressing a splicing variant of AIMP2 through targeted RNA replacement by <i>trans</i>-splicing ribozyme</span>. <i>J. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.jbiotec.2012.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1016%2Fj.jbiotec.2012.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=22285955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVGlu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2012&pages=44-49&author=Y.+S.+Wonauthor=S.+W.+Lee&title=Selective+regression+of+cancer+cells+expressing+a+splicing+variant+of+AIMP2+through+targeted+RNA+replacement+by+trans-splicing+ribozyme&doi=10.1016%2Fj.jbiotec.2012.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Selective regression of cancer cells expressing a splicing variant of AIMP2 through targeted RNA replacement by trans-splicing ribozyme</span></div><div class="casAuthors">Won, You-Sub; Lee, Seong-Wook</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">44-49</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">AIMP2/p38 is a scaffolding protein crit. for the assembly of the macromol. tRNA synthetase complex.  Moreover, AIMP2 harbors anti-proliferative activity and promotes cell death as a proapototic factor.  A splicing variant of AIMP2 lacking exon 2 (AIMP2-DX2) is specifically generated by an alternative splicing process and is highly expressed in human lung cancer cells and the tissues of cancer patients.  AIMP2-DX2 induces tumorigenesis by compromising the tumor suppressor function of normal AIMP2.  Here, we describe a novel approach to cancer therapy that is based on trans-splicing ribozyme-mediated replacement of specific RNAs.  We developed a specific ribozyme that can target and replace the splicing variant AIMP2-DX2 RNA with a new transcript selectively exerting therapeutic activity in AIMP2-DX2-expressing lung cancer cells.  The RNA replacement was employed via a high-fidelity trans-splicing reaction with the targeted residue of the AIMP2-DX2 transcript, but not with normal AIMP2 RNA, in the cells.  Noticeably, the ribozyme could selectively deliver the activity of a suicide gene into the AIMP2-DX2 RNA expressing lung cancer cells and thereby specifically and effectively retard the growth of the cancer cells with prodrug treatment.  Therefore, the AIMP2-DX2 RNA-targeting trans-splicing ribozyme could be a useful genetic agent for efficient therapy targeting lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDbUoFKEVvz7Vg90H21EOLACvtfcHk0lhqhv50th3PCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVGlu7o%253D&md5=39c8cce87457e98580f54cdce9dd0947</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2012.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2012.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DWon%26aufirst%3DY.%2BS.%26aulast%3DLee%26aufirst%3DS.%2BW.%26atitle%3DSelective%2520regression%2520of%2520cancer%2520cells%2520expressing%2520a%2520splicing%2520variant%2520of%2520AIMP2%2520through%2520targeted%2520RNA%2520replacement%2520by%2520trans-splicing%2520ribozyme%26jtitle%3DJ.%2520Biotechnol.%26date%3D2012%26volume%3D158%26spage%3D44%26epage%3D49%26doi%3D10.1016%2Fj.jbiotec.2012.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span>; <span class="NLM_string-name">Choi, J. W.</span></span> <span> </span><span class="NLM_article-title">AIMP2-DX2 Gene and its Uses</span>. <span class="NLM_patent">US 2011/0136119A1</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=S.+Kim&author=J.+W.+Choi&title=AIMP2-DX2+Gene+and+its+Uses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26atitle%3DAIMP2-DX2%2520Gene%2520and%2520its%2520Uses%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Cancer-associated splicing variant of tumor suppressor AIMP2/p38: pathological implication in tumorigenesis</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e1001351</span> <span class="refDoi"> DOI: 10.1371/journal.pgen.1001351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1371%2Fjournal.pgen.1001351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=21483803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvVeqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&author=J.+W.+Choiauthor=D.+G.+Kimauthor=A.+E.+Leeauthor=H.+R.+Kimauthor=J.+Y.+Leeauthor=N.+H.+Kwonauthor=Y.+K.+Shinauthor=S.+K.+Hwangauthor=S.+H.+Changauthor=M.+H.+Choauthor=Y.+L.+Choiauthor=J.+Kimauthor=S.+H.+Ohauthor=B.+Kimauthor=S.+Y.+Kimauthor=H.+S.+Jeonauthor=J.+Y.+Parkauthor=H.+P.+Kangauthor=B.+J.+Parkauthor=J.+M.+Hanauthor=S.+Kim&title=Cancer-associated+splicing+variant+of+tumor+suppressor+AIMP2%2Fp38%3A+pathological+implication+in+tumorigenesis&doi=10.1371%2Fjournal.pgen.1001351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer-associated splicing variant of tumor suppressor AIMP2/p38: pathological implication in tumorigenesis</span></div><div class="casAuthors">Choi, Jin Woo; Kim, Dae Gyu; Lee, Al-Eum; Kim, Hye Rim; Lee, Jin Young; Kwon, Nam Hoon; Shin, Young Kee; Hwang, Soon-Kyung; Chang, Seung-Hee; Cho, Myung-Haing; Choi, Yoon-La; Kim, Jhingook; Oh, Seung Hyun; Kim, Bora; Kim, Soo-Youl; Jeon, Hyo-Sung; Park, Jae Yong; Kang, Hyunseok Peter; Park, Bum Joon; Han, Jung Min; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1001351</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Although ARS-interacting multi-functional protein 2 (AIMP2, also named as MSC p38) was first found as a component for a macromol. tRNA synthetase complex, it was recently discovered to dissoc. from the complex and work as a potent tumor suppressor.  Upon DNA damage, AIMP2 promotes apoptosis through the protective interaction with p53.  However, it was not demonstrated whether AIMP2 was indeed pathol. linked to human cancer.  In this work, we found that a splicing variant of AIMP2 lacking exon 2 (AIMP2-DX2) is highly expressed by alternative splicing in human lung cancer cells and patient's tissues.  AIMP2-DX2 compromised pro-apoptotic activity of normal AIMP2 through the competitive binding to p53.  The cells with higher level of AIMP2-DX2 showed higher propensity to form anchorage-independent colonies and increased resistance to cell death.  Mice constitutively expressing this variant showed increased susceptibility to carcinogen-induced lung tumorigenesis.  The expression ratio of AIMP2-DX2 to normal AIMP2 was increased according to lung cancer stage and showed a pos. correlation with the survival of patients.  Thus, this work identified an oncogenic splicing variant of a tumor suppressor, AIMP2/p38, and suggests its potential for anti-cancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptcSQBMoogs7Vg90H21EOLACvtfcHk0li5ENh3-IUnFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvVeqsrY%253D&md5=ae6806bab0a6e632c31e03ff5f0241bf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1001351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1001351%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DLee%26aufirst%3DA.%2BE.%26aulast%3DKim%26aufirst%3DH.%2BR.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DShin%26aufirst%3DY.%2BK.%26aulast%3DHwang%26aufirst%3DS.%2BK.%26aulast%3DChang%26aufirst%3DS.%2BH.%26aulast%3DCho%26aufirst%3DM.%2BH.%26aulast%3DChoi%26aufirst%3DY.%2BL.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DOh%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DJeon%26aufirst%3DH.%2BS.%26aulast%3DPark%26aufirst%3DJ.%2BY.%26aulast%3DKang%26aufirst%3DH.%2BP.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DCancer-associated%2520splicing%2520variant%2520of%2520tumor%2520suppressor%2520AIMP2%252Fp38%253A%2520pathological%2520implication%2520in%2520tumorigenesis%26jtitle%3DPLoS%2520Genet.%26date%3D2011%26volume%3D7%26doi%3D10.1371%2Fjournal.pgen.1001351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Splicing variant of AIMP2 as an effective target against chemoresistant ovarian cancer</span>. <i>J. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">164</span>– <span class="NLM_lpage">173</span>, <span class="refDoi"> DOI: 10.1093/jmcb/mjs018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1093%2Fjmcb%2Fmjs018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=22532625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12rtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=164-173&author=J.+W.+Choiauthor=J.+W.+Leeauthor=J.+K.+Kimauthor=H.+K.+Jeonauthor=J.+J.+Choiauthor=D.+G.+Kimauthor=B.+G.+Kimauthor=D.+H.+Namauthor=H.+J.+Kimauthor=S.+H.+Yunauthor=S.+Kim&title=Splicing+variant+of+AIMP2+as+an+effective+target+against+chemoresistant+ovarian+cancer&doi=10.1093%2Fjmcb%2Fmjs018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Splicing variant of AIMP2 as an effective target against chemoresistant ovarian cancer</span></div><div class="casAuthors">Choi, Jin Woo; Lee, Jeong-Won; Kim, Jun Ki; Jeon, Hye-Kyung; Choi, Jung-Joo; Kim, Dae Gyu; Kim, Byoung-Gie; Nam, Do-Hyun; Kim, Hong Joong; Yun, Seok Hyun; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">164-173, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Chemoresistance is a main cause for the failure of cancer management and intensive investigation is on-going to control chemoresistant (CR) cancers.  Although NF-κB has been suggested as one of the potential targets to alleviate chemoresistance of epithelial ovarian cancer (EOC), direct targeting of NF-κB may result in an unexpected effect due to the complex regulatory network via NF-κB.  Here we show that AIMP2-DX2, a splicing variant of tumor suppressor AIMP2, can be a therapeutic target to control CR EOC.  AIMP2-DX2 was often highly expressed in CR EOC both in vitro and in vivo.  AIMP2-DX2 compromised the tumor necrosis factor alpha-dependent pro-apoptotic activity of AIMP2 via the competitive inhibition of AIMP2 binding to TRAF2 that plays a pivotal role in the regulation of NF-κB.  The direct delivery of siRNA against AIMP2-DX2 into abdominal metastatic tumors of ovarian cancer using a microneedle converged on microendoscopy significantly suppressed the growth rate of tumors.  The treated cancer tissues showed an enhanced apoptosis and the decreased TRAF2 level.  Thus, we suggest that the downregulation of AIMP2-DX2 can be a potent adjuvant therapeutic approach for CR EOC that resulted from an aberrant activity of NF-κB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPbT1cw7EKMbVg90H21EOLACvtfcHk0li5ENh3-IUnFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12rtbbL&md5=eca0bcc749a50e4bb3ea5a0319f86ca0</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjs018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjs018%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BW.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DJ.%2BK.%26aulast%3DJeon%26aufirst%3DH.%2BK.%26aulast%3DChoi%26aufirst%3DJ.%2BJ.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DKim%26aufirst%3DB.%2BG.%26aulast%3DNam%26aufirst%3DD.%2BH.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DYun%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DSplicing%2520variant%2520of%2520AIMP2%2520as%2520an%2520effective%2520target%2520against%2520chemoresistant%2520ovarian%2520cancer%26jtitle%3DJ.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D4%26spage%3D164%26epage%3D173%26doi%3D10.1093%2Fjmcb%2Fmjs018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. S.</span></span> <span> </span><span class="NLM_article-title">Ratio of autoantibodies of tumor suppressor AIMP2and its oncogenic variant is associated with clinical outcome in lung cancer</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1354</span>, <span class="refDoi"> DOI: 10.7150/jca.18450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.7150%2Fjca.18450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=28638448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A280%3ADC%252BC1cjgs1GktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1347-1354&author=J.+Y.+Jungauthor=E.+Y.+Kimauthor=A.+Kimauthor=J.+Changauthor=N.+H.+Kwonauthor=Y.+Moonauthor=E.+J.+Kangauthor=J.+S.+Sungauthor=H.+Shimauthor=S.+Kimauthor=Y.+S.+Chang&title=Ratio+of+autoantibodies+of+tumor+suppressor+AIMP2and+its+oncogenic+variant+is+associated+with+clinical+outcome+in+lung+cancer&doi=10.7150%2Fjca.18450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Ratio of Autoantibodies of Tumor Suppressor AIMP2 and Its Oncogenic Variant Is Associated with Clinical Outcome in Lung Cancer</span></div><div class="casAuthors">Jung Ji Ye; Kim Eun Young; Chang Joon; Kim Arum; Chang Yoon Soo; Kwon Nam Hoon; Moon Youngji; Kang Eun Joo; Sung Jun Sik; Kim Sunghoon; Shim Hyunbo; Kim Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1347-1354</span>
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    </div><div class="casAbstract">Aminoacyl-tRNA synthetase-interacting multi-functional protein 2 (AIMP2) works as potent tumor suppressor, while its splicing variant lacking exon 2 (AIMP2-DX2) competes with AIMP2 for binding to target proteins and compromises its anti-tumor activity.  Assuming that AIMP2 and its variant AIMP2-DX2 could be released out to human sera in pathological condition, we investigated the diagnostic and prognostic usefulness of autoantibodies against AIMP2 and AIMP2-DX2 by measuring their serum levels in 80 normal and lung cancer samples that were matched in age, gender and smoking status.  The area under the curve of AIMP2-DX2, AIMP2, and AIMP2-DX2/AIMP2 autoantibody ratio was low (0.416, 0.579, and 0.357, respectively), suggesting limited diagnostic value.  A total of 165 lung cancer patients were classified into low and high AIMP2-DX2, AIMP2, and AIMP2-DX2/AIMP2 based on the median expression of each parameter.  The high AIMP2-DX2 group was older and had larger tumors (>3 cm) than the low AIMP2-DX2 group.  The high AIMP2-DX2/AIMP2 group had higher CYFRA-21 levels and significantly shorter overall survival than the low AIMP2-DX2/AIMP2 group (18.6 vs. 48.9 months, P = 0.021, Log Rank Test).  Taken together, autoantibodies against AIMP2-DX2 and AIMP2 are detectable in the human blood and the increased ratio of AIMP2-DX2/AIMP2 is related to poor clinical outcome of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2VzkwiviqdgMthz5_Rx5rfW6udTcc2eYd0DVyB_55U7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjgs1GktQ%253D%253D&md5=5b2d609d5b5db98ab6850f1ba81dbb3c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.7150%2Fjca.18450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.18450%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DE.%2BY.%26aulast%3DKim%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DMoon%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DE.%2BJ.%26aulast%3DSung%26aufirst%3DJ.%2BS.%26aulast%3DShim%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DY.%2BS.%26atitle%3DRatio%2520of%2520autoantibodies%2520of%2520tumor%2520suppressor%2520AIMP2and%2520its%2520oncogenic%2520variant%2520is%2520associated%2520with%2520clinical%2520outcome%2520in%2520lung%2520cancer%26jtitle%3DJ.%2520Cancer%26date%3D2017%26volume%3D8%26spage%3D1347%26epage%3D1354%26doi%3D10.7150%2Fjca.18450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Oncogenic mutation of AIMP2/p38 inhibits its tumor-suppressive interaction with Smurf2</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3422</span>– <span class="NLM_lpage">3436</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-3255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1158%2F0008-5472.CAN-15-3255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=27197155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVyktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3422-3436&author=D.+G.+Kimauthor=J.+Y.+Leeauthor=J.+H.+Leeauthor=H.+Y.+Choauthor=B.+S.+Kangauthor=S.+Y.+Jangauthor=M.+H.+Kimauthor=M.+Guoauthor=J.+M.+Hanauthor=S.+J.+Kimauthor=S.+Kim&title=Oncogenic+mutation+of+AIMP2%2Fp38+inhibits+its+tumor-suppressive+interaction+with+Smurf2&doi=10.1158%2F0008-5472.CAN-15-3255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic Mutation of AIMP2/p38 Inhibits Its Tumor-Suppressive Interaction with Smurf2</span></div><div class="casAuthors">Kim, Dae Gyu; Lee, Jin Young; Lee, Ji-Hyun; Cho, Ha Yeon; Kang, Beom Sik; Jang, Song-Yee; Kim, Myung Hee; Guo, Min; Han, Jung Min; Kim, Seong-Jin; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3422-3436</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">AIMP2/p38 is a multifunctional tumor suppressor that normally resides in the cytosol as a scaffold protein of the multi-tRNA synthetase complex (MSC).  One of the tumor-suppressive functions of AIMP2 is to facilitate ubiquitin-mediated degrdn. of FUSE-binding protein (FBP, FUBP1), a transcriptional activator of c-Myc.  However, the mechanism by which AIMP2 functions within this pathway and its significance in tumorigenesis are uncertain.  Here, we report that Smurf2 is responsible for AIMP2-mediated ubiquitination of FBP, and a mutation in AIMP2 that inhibited its nuclear interaction with Smurf2 enhanced cellular transformation and tumorigenesis in vivo.  Treatment of HeLa cells with TGFβ resulted in the phosphorylation of AIMP2 on S156, a residue that is exposed on the embedded GST domain of AIMP2.  We further found that phospho-AIMP2 dissocd. from the MSC and translocated to the nucleus, where it bound to Smurf2, enhancing ubiquitination of FBP.  AIMP2 also inhibited nuclear export of Smurf2 to sustain TGFβ signaling.  Collectively, these findings present a novel tumor-suppressive interaction between AIMP2 and Smurf2 and suggest that the disruption of this interaction can lead to oncogenic transformation.  Cancer Res; 76(11); 3422-36. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0ojyRQmzBfrVg90H21EOLACvtfcHk0lieynlO98ZkZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVyktbo%253D&md5=0f720da36bc561c9f0a52117d4520468</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-3255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-3255%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DH.%2BY.%26aulast%3DKang%26aufirst%3DB.%2BS.%26aulast%3DJang%26aufirst%3DS.%2BY.%26aulast%3DKim%26aufirst%3DM.%2BH.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DOncogenic%2520mutation%2520of%2520AIMP2%252Fp38%2520inhibits%2520its%2520tumor-suppressive%2520interaction%2520with%2520Smurf2%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3422%26epage%3D3436%26doi%3D10.1158%2F0008-5472.CAN-15-3255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span>; <span class="NLM_string-name">Kim, D. G.</span>; <span class="NLM_string-name">Lim, S.</span></span> <span> </span><span class="NLM_article-title">Method for Screening Anticancer Agent Inhibiting Binding of AIMP2 - DX2 and HSP70</span>. <span class="NLM_patent">US 20190033295A1</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=S.+Kim&author=D.+G.+Kim&author=S.+Lim&title=Method+for+Screening+Anticancer+Agent+Inhibiting+Binding+of+AIMP2+-+DX2+and+HSP70"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26atitle%3DMethod%2520for%2520Screening%2520Anticancer%2520Agent%2520Inhibiting%2520Binding%2520of%2520AIMP2%2520-%2520DX2%2520and%2520HSP70%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushtaq, A. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattarai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaraman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Targeting the interaction of AIMP2-DX2 with HSP70 suppresses cancer development</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0415-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1038%2Fs41589-019-0415-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=31792442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=31-41&issue=1&author=S.+Limauthor=H.+Y.+Choauthor=D.+G.+Kimauthor=Y.+Rohauthor=S.+Sonauthor=A.+U.+Mushtaqauthor=M.+Kimauthor=D.+Bhattaraiauthor=A.+Sivaramanauthor=Y.+Leeauthor=J.+Leeauthor=W.+S.+Yangauthor=H.+K.+Kimauthor=M.+H.+Kimauthor=K.+Leeauthor=Y.+H.+Jeonauthor=S.+Kim&title=Targeting+the+interaction+of+AIMP2-DX2+with+HSP70+suppresses+cancer+development&doi=10.1038%2Fs41589-019-0415-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the interaction of AIMP2-DX2 with HSP70 suppresses cancer development</span></div><div class="casAuthors">Lim, Semi; Cho, Hye Young; Kim, Dae Gyu; Roh, Younah; Son, Se-Young; Mushtaq, Ameeq Ul; Kim, Minkyoung; Bhattarai, Deepak; Sivaraman, Aneesh; Lee, Youngjin; Lee, Jihye; Yang, Won Suk; Kim, Hoi Kyoung; Kim, Myung Hee; Lee, Kyeong; Jeon, Young Ho; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-41</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A tumorigenic factor, AIMP2 lacking exon 2 (AIMP2-DX2), is often upregulated in many cancers.  However, how its cellular level is detd. is not understood.  Here, we report heat-shock protein HSP70 as a crit. determinant for the level of AIMP2-DX2.  Interaction of the two factors was identified by interactome anal. and structurally detd. by X-ray crystallog. and NMR analyses.  HSP70 recognizes the amino (N)-terminal flexible region, as well as the glutathione S-transferase domain of AIMP2-DX2, via its substrate-binding domain, thus blocking the Siah1-dependent ubiquitination of AIMP2-DX2.  AIMP2-DX2-induced cell transformation and cancer progression in vivo was further augmented by HSP70.  A pos. correlation between HSP70 and AIMP2-DX2 levels was shown in various lung cancer cell lines and patient tissues.  Chem. intervention in the AIMP2-DX2-HSP70 interaction suppressed cancer cell growth in vitro and in vivo.  Thus, this work demonstrates the importance of the interaction between AIMP2-DX2 and HSP70 on tumor progression and its therapeutic potential against cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4li81ZRTJbrVg90H21EOLACvtfcHk0lieynlO98ZkZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrzE&md5=14f219d03385a942300f8e0e3ed7f6a4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0415-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0415-2%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DH.%2BY.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DRoh%26aufirst%3DY.%26aulast%3DSon%26aufirst%3DS.%26aulast%3DMushtaq%26aufirst%3DA.%2BU.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DBhattarai%26aufirst%3DD.%26aulast%3DSivaraman%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DW.%2BS.%26aulast%3DKim%26aufirst%3DH.%2BK.%26aulast%3DKim%26aufirst%3DM.%2BH.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DJeon%26aufirst%3DY.%2BH.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DTargeting%2520the%2520interaction%2520of%2520AIMP2-DX2%2520with%2520HSP70%2520suppresses%2520cancer%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26issue%3D1%26spage%3D31%26epage%3D41%26doi%3D10.1038%2Fs41589-019-0415-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Chemical suppression of an oncogenic splicing variant of AIMP2 induces tumour regression</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>454</i></span>,  <span class="NLM_fpage">411</span>– <span class="NLM_lpage">416</span>, <span class="refDoi"> DOI: 10.1042/BJ20130550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1042%2FBJ20130550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=23815603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGmt7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=454&publication_year=2013&pages=411-416&author=H.+S.+Leeauthor=D.+G.+Kimauthor=Y.+S.+Ohauthor=N.+H.+Kwonauthor=J.+Y.+Leeauthor=D.+Kimauthor=S.+H.+Parkauthor=J.+H.+Songauthor=S.+Leeauthor=J.+M.+Hanauthor=B.+J.+Parkauthor=J.+Leeauthor=S.+Kim&title=Chemical+suppression+of+an+oncogenic+splicing+variant+of+AIMP2+induces+tumour+regression&doi=10.1042%2FBJ20130550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical suppression of an oncogenic splicing variant of AIMP2 induces tumour regression</span></div><div class="casAuthors">Lee, Hee Sook; Kim, Dae Gyu; Oh, Young Sun; Kwon, Nam Hoon; Lee, Jin Young; Kim, Doyeun; Park, Song-Hwa; Song, Jong-Hwan; Lee, Sunkyung; Han, Jung Min; Park, Bum-Joon; Lee, Jongkook; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">454</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">411-416</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">AIMP2 (aminoacyl-tRNA synthetase-interacting multifunctional protein 2) is a potent tumor suppressor that induces apoptosis in response to various oncogenic signals.  AIMP2-DX2, an exon2-deleted splicing variant of AIMP2, is up-regulated in lung cancer and competitively suppresses the pro-apoptotic activity of AIMP2, resulting in tumorigenesis.  In the present study we report that BC-DXI01, a synthetic compd., specifically reduces the cellular levels of AIMP2-DX2 through selective degrdn. of the AIMP2-DX2 mRNA transcript.  We found that BC-DXI01-mediated cell death pos. correlates with AIMP2-DX2 expression in the lung cancer cell lines tested.  Administration of BC-DXI01 in a AIMP2-DX2-driven tumor xenograft mice model led to reduced tumor sizes and vols. of up to 60% in comparison with vehicle-treated mice group, consistent with decreases in AIMP2-DX2 transcript and protein levels.  Taken together, our findings suggest that tumorigenic activity of AIMP2-DX2 can be controlled by the small chem. BC-DXI01, which can selectively suppress the AIMP2-DX2 mRNA transcript.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouKh0xvh3D4bVg90H21EOLACvtfcHk0lieynlO98ZkZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGmt7jN&md5=9b4b85449eab8ca18a041fff00afb905</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1042%2FBJ20130550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20130550%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BS.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DOh%26aufirst%3DY.%2BS.%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DPark%26aufirst%3DS.%2BH.%26aulast%3DSong%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DChemical%2520suppression%2520of%2520an%2520oncogenic%2520splicing%2520variant%2520of%2520AIMP2%2520induces%2520tumour%2520regression%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D454%26spage%3D411%26epage%3D416%26doi%3D10.1042%2FBJ20130550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span> <span> </span><span class="NLM_article-title">Inhibiting DX2-p14/ARF interaction exerts antitumor effects in lung cancer and delays tumor progression</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">4791</span>– <span class="NLM_lpage">4804</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-1025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1158%2F0008-5472.CAN-15-1025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=27302160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtleiu7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=4791-4804&author=A.+Y.+Ohauthor=Y.+S.+Jungauthor=J.+Kimauthor=J.+H.+Leeauthor=J.+H.+Choauthor=H.+Y.+Chunauthor=S.+Parkauthor=H.+Parkauthor=S.+Limauthor=N.+C.+Haauthor=J.+S.+Parkauthor=C.+S.+Parkauthor=G.+Y.+Songauthor=B.+J.+Park&title=Inhibiting+DX2-p14%2FARF+interaction+exerts+antitumor+effects+in+lung+cancer+and+delays+tumor+progression&doi=10.1158%2F0008-5472.CAN-15-1025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting DX2-p14/ARF Interaction Exerts Antitumor Effects in Lung Cancer and Delays Tumor Progression</span></div><div class="casAuthors">Oh, Ah-Young; Jung, Youn Sang; Kim, Jiseon; Lee, Jee-Hyun; Cho, Jung-Hyun; Chun, Ho-Young; Park, Soyoung; Park, Hyunchul; Lim, Sikeun; Ha, Nam-Chul; Park, Jong Sook; Park, Choon-Sik; Song, Gyu-Yong; Park, Bum-Joon</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4791-4804</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The aminoacyl tRNA synthetase complex-interacting multifunctional protein 2 (AIMP2) splice variant designated DX2 is induced by cigarette smoke carcinogens and is often detected in human lung cancer specimens.  However, the function of DX2 in lung carcinogenesis is obscure.  In this study, we found that DX2 expression was induced by oncogenes in human lung cancer tissues and cells.  DX2 prevented oncogene-induced apoptosis and senescence and promoted drug resistance by directly binding to and inhibiting p14/ARF.  Through chem. screening, we identified SLCB050, a novel compd. that blocks the interaction between DX2 and p14/ARF in vitro and in vivo.  SLCB050 reduced the viability of human lung cancer cells, esp. small cell lung cancer cells, in a p14/ARF-dependent manner.  Moreover, in a mouse model of K-Ras-driven lung tumorigenesis, ectopic expression of DX2 induced small cell and non-small cell lung cancers, both of which could be suppressed by SLCB050 treatment.  Taken together, our findings show how DX2 promotes lung cancer progression and how its activity may be thwarted as a strategy to treat patients with lung cancers exhibiting elevated DX2 levels.  Cancer Res; 76(16); 4791-804. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomT7AJTx-0NbVg90H21EOLACvtfcHk0ljdXGR8KdaP_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtleiu7zI&md5=5496a0d901d7457292140cfc22e621b2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-1025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-1025%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DA.%2BY.%26aulast%3DJung%26aufirst%3DY.%2BS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DChun%26aufirst%3DH.%2BY.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DHa%26aufirst%3DN.%2BC.%26aulast%3DPark%26aufirst%3DJ.%2BS.%26aulast%3DPark%26aufirst%3DC.%2BS.%26aulast%3DSong%26aufirst%3DG.%2BY.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26atitle%3DInhibiting%2520DX2-p14%252FARF%2520interaction%2520exerts%2520antitumor%2520effects%2520in%2520lung%2520cancer%2520and%2520delays%2520tumor%2520progression%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D4791%26epage%3D4804%26doi%3D10.1158%2F0008-5472.CAN-15-1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span>; <span class="NLM_string-name">Song, G. Y.</span>; <span class="NLM_string-name">Oh, A. Y.</span>; <span class="NLM_string-name">Lee, J. H.</span>; <span class="NLM_string-name">Her, J. H.</span></span> <span> </span><span class="NLM_article-title">Compound for Inhibiting Binding between DX2 Protein and p14/ARF Protein, and Pharmaceutical Composition for Treating or Preventing Cancer Disease Containing same as Effective Ingredient</span>. <span class="NLM_patent">US10280178B2</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=B.+J.+Park&author=G.+Y.+Song&author=A.+Y.+Oh&author=J.+H.+Lee&author=J.+H.+Her&title=Compound+for+Inhibiting+Binding+between+DX2+Protein+and+p14%2FARF+Protein%2C+and+Pharmaceutical+Composition+for+Treating+or+Preventing+Cancer+Disease+Containing+same+as+Effective+Ingredient"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DB.%2BJ.%26atitle%3DCompound%2520for%2520Inhibiting%2520Binding%2520between%2520DX2%2520Protein%2520and%2520p14%252FARF%2520Protein%252C%2520and%2520Pharmaceutical%2520Composition%2520for%2520Treating%2520or%2520Preventing%2520Cancer%2520Disease%2520Containing%2520same%2520as%2520Effective%2520Ingredient%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span>; <span class="NLM_string-name">Kwon, N. H.</span>; <span class="NLM_string-name">Kim, D. G.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical Composition for Preventing or Treating Cancer</span>. <span class="NLM_patent">US9539236B2</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=S.+Kim&author=N.+H.+Kwon&author=D.+G.+Kim&title=Pharmaceutical+Composition+for+Preventing+or+Treating+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26atitle%3DPharmaceutical%2520Composition%2520for%2520Preventing%2520or%2520Treating%2520Cancer%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, P.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel 7-amino-[1,2,4]triazolo[4,3-f]pteridinone, and 7-aminotetrazolo[1,5-f]pteridinone derivative as potent antitumor agents</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">690</span>– <span class="NLM_lpage">709</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.12.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1016%2Fj.ejmech.2018.12.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=30572179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFygur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2019&pages=690-709&author=Y.+Houauthor=L.+Zhuauthor=Z.+Liauthor=Q.+Shenauthor=Q.+Xuauthor=W.+Liauthor=Y.+Liuauthor=P.+Gong&title=Design%2C+synthesis+and+biological+evaluation+of+novel+7-amino-%5B1%2C2%2C4%5Dtriazolo%5B4%2C3-f%5Dpteridinone%2C+and+7-aminotetrazolo%5B1%2C5-f%5Dpteridinone+derivative+as+potent+antitumor+agents&doi=10.1016%2Fj.ejmech.2018.12.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel 7-amino-[1,2,4]triazolo[4,3-f]pteridinone, and 7-aminotetrazolo[1,5-f]pteridinone derivative as potent antitumor agents</span></div><div class="casAuthors">Hou, Yunlei; Zhu, Liangyu; Li, Zhiwei; Shen, Qi; Xu, Qiaoling; Li, Wei; Liu, Yajing; Gong, Ping</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">690-709</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">To develop novel therapeutic agents with anticancer activities, two series of novel 7-amino-[1,2,4]triazolo[4,3-f]pteridinone, and 7-aminotetrazolo[1,5-f]pteridinone derivs. were designed and synthesized.  All compds. were tested for anti-proliferative activities against five cancer cell lines.  The structure-activity relationships (SARs) studies were conducted through the variation in two regions, the moiety of A ring and the terminal aniline B on pteridinone core. 1-Methyl-1,2,4-triazole deriv. I with 2,6-dimethylpiperazine showed the most potent antiproliferative activity against A549, PC-3, HCT116, MCF-7 and MDA-MB-231 cell lines with IC50 values of 0.16 μM, 0.30 μM, 0.51 μM, 0.30 μM, and 0.70 μM, resp.  Combined with the results of the mol. docking and enzymic studies, the PLK1 was very likely to be one of the drug targets of compd. I.  Furthermore, to clarify the anticancer mechanism of compd. I, further explorations in the bioactivity were conducted.  The results showed that compd. I obviously inhibited proliferation of A549 cell lines, induced a great decrease in mitochondrial membrane potential leading to apoptosis of cancer cells, suppressed the migration of tumor cells, and arrested G1 phase of A549 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj6IslFQC9NrVg90H21EOLACvtfcHk0ljdXGR8KdaP_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFygur7F&md5=e95279a0b95740db3d9ef69892325ab9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.12.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.12.009%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DP.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%25207-amino-%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-f%255Dpteridinone%252C%2520and%25207-aminotetrazolo%255B1%252C5-f%255Dpteridinone%2520derivative%2520as%2520potent%2520antitumor%2520agents%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D163%26spage%3D690%26epage%3D709%26doi%3D10.1016%2Fj.ejmech.2018.12.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span>; <span class="NLM_string-name">Rajagopalan, S.</span>; <span class="NLM_string-name">Belavagi, N.</span>; <span class="NLM_string-name">Ramachandra, M.</span></span> <span> </span><span class="NLM_article-title">2,4-Diaminopyrimidine Derivatives as Protein Kinase Inhibitors</span>. <span class="NLM_patent">WO 2012/059932Al</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=S.+Sengupta&author=S.+Rajagopalan&author=N.+Belavagi&author=M.+Ramachandra&title=2%2C4-Diaminopyrimidine+Derivatives+as+Protein+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSengupta%26aufirst%3DS.%26atitle%3D2%252C4-Diaminopyrimidine%2520Derivatives%2520as%2520Protein%2520Kinase%2520Inhibitors%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span> <span> </span><span class="NLM_article-title">Identification of novel 2-aminothiazole conjugated nitrofuran as antitubercular and antibacterial agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3669</span>– <span class="NLM_lpage">3674</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.05.088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1016%2Fj.bmcl.2016.05.088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=27289321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1ynu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=3669-3674&author=K.+Ranauthor=C.+Gaoauthor=H.+Dengauthor=Q.+Leiauthor=X.+Youauthor=N.+Wangauthor=Y.+Shiauthor=Z.+Liuauthor=W.+Weiauthor=C.+Pengauthor=L.+Xiongauthor=K.+Xiaoauthor=L.+Yu&title=Identification+of+novel+2-aminothiazole+conjugated+nitrofuran+as+antitubercular+and+antibacterial+agents&doi=10.1016%2Fj.bmcl.2016.05.088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel 2-aminothiazole conjugated nitrofuran as antitubercular and antibacterial agents</span></div><div class="casAuthors">Ran, Kai; Gao, Chao; Deng, Hongxia; Lei, Qian; You, Xinyu; Wang, Ningyu; Shi, Yaojie; Liu, Zhihao; Wei, Wei; Peng, Cuiting; Xiong, Lu; Xiao, Kunjie; Yu, Luoting</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3669-3674</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The emergence of antibiotic resistant pathogens is an ongoing main problem in the therapy of bacterial infections.  In order to develop promising antitubercular and antibacterial lead compds., the authors designed and synthesized a new series of derivs. of 2-aminothiazole conjugated nitrofuran with activities against both Mycobacterium tuberculosis and Staphylococcus aureus.  Eight compds. emerged as promising antitubercular agents.  Structure-activity relationships (SARs) were discussed and showed that the derivs. substituted at the position-3 of benzene of 5-nitro-N-(4-phenylthiazol-2-yl)furan-2-carboxamide exhibited superior potency.  The most potent compd. I, substituted with benzamide at this position, displayed min. inhibitory concns. (MICs) of 0.27 μg/mL against Mtb H37Ra and 1.36 μg/mL against S. aureus.  Furthermore, compd. I had no obvious cytotoxicity to normal Vero cells (IC50 = 50.2 μM).  The results suggest that the novel scaffolds of aminothiazole conjugated nitrofuran would be a promising class of potent antitubercular and antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvVRT2hzqQSbVg90H21EOLACvtfcHk0liGA5uAPXBa-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1ynu7Y%253D&md5=838ecf4c24b220659817d0abeb00697d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.05.088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.05.088%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DLei%26aufirst%3DQ.%26aulast%3DYou%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DXiong%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DL.%26atitle%3DIdentification%2520of%2520novel%25202-aminothiazole%2520conjugated%2520nitrofuran%2520as%2520antitubercular%2520and%2520antibacterial%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D3669%26epage%3D3674%26doi%3D10.1016%2Fj.bmcl.2016.05.088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jyh Huang, J.</span>; <span class="NLM_string-name">Chuang, S. H.</span>; <span class="NLM_string-name">Lee, Y. E.</span>; <span class="NLM_string-name">Huang, Y.</span>; <span class="NLM_string-name">Lau, J.</span></span> <span> </span><span class="NLM_article-title">Modulators of Hec1 Activity and Methods Therefor</span>. <span class="NLM_patent">US 2012/059932Al</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+Jyh+Huang&author=S.+H.+Chuang&author=Y.+E.+Lee&author=Y.+Huang&author=J.+Lau&title=Modulators+of+Hec1+Activity+and+Methods+Therefor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJyh%2BHuang%26aufirst%3DJ.%26atitle%3DModulators%2520of%2520Hec1%2520Activity%2520and%2520Methods%2520Therefor%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goughnour, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paek, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seol, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Novel morphologic and genetic analysis of cancer cells in a 3D microenvironment identifies STAT3 as a regulator of tumor permeability barrier function</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">1044</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-14-2611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1158%2F0008-5472.CAN-14-2611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=26676754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslCltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=1044-1054&author=M.+C.+Parkauthor=H.+Jeongauthor=S.+H.+Sonauthor=Y.+Kimauthor=D.+Hanauthor=P.+C.+Goughnourauthor=T.+Kangauthor=N.+H.+Kwonauthor=H.+E.+Moonauthor=S.+H.+Paekauthor=D.+Hwangauthor=H.+J.+Seolauthor=D.+H.+Namauthor=S.+Kim&title=Novel+morphologic+and+genetic+analysis+of+cancer+cells+in+a+3D+microenvironment+identifies+STAT3+as+a+regulator+of+tumor+permeability+barrier+function&doi=10.1158%2F0008-5472.CAN-14-2611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Morphologic and Genetic Analysis of Cancer Cells in a 3D Microenvironment Identifies STAT3 as a Regulator of Tumor Permeability Barrier Function</span></div><div class="casAuthors">Park, Min Chul; Jeong, Hyobin; Son, Sung Hwa; Kim, YounHa; Han, Daeyoung; Goughnour, Peter C.; Kang, Taehee; Kwon, Nam Hoon; Moon, Hyo Eun; Paek, Sun Ha; Hwang, Daehee; Seol, Ho Jun; Nam, Do-Hyun; Kim, Sunghoon</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1044-1054</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Tumor permeability is a crit. determinant of drug delivery and sensitivity, but systematic methods to identify factors that perform permeability barrier functions in the tumor microenvironment are not yet available.  Multicellular tumor spheroids have become tractable in vitro models to study the impact of a three-dimensional (3D) environment on cellular behavior.  In this study, we characterized the spheroid-forming potential of cancer cells and correlated the resulting spheroid morphologies with genetic information to identify conserved cellular processes assocd. with spheroid structure.  Spheroids generated from 100 different cancer cell lines were classified into four distinct groups based on morphol.  In particular, round and compact spheroids exhibited highly hypoxic inner cores and permeability barriers against anticancer drugs.  Through systematic and correlative anal., we reveal JAK-STAT signaling as one of the signature pathways activated in round spheroids.  Accordingly, STAT3 inhibition in spheroids generated from the established cancer cells and primary glioblastoma patient-derived cells altered the rounded morphol. and increased drug sensitivity.  Furthermore, combined administration of the STAT3 inhibitor and 5-fluorouracil to a mouse xenograft model markedly reduced tumor growth compared with monotherapy.  Collectively, our findings demonstrate the ability to integrate 3D culture and genetic profiling to det. the factors underlying the integrity of the permeability barrier in the tumor microenvironment, and may help to identify and exploit novel mechanisms of drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0yDgPJn8uBbVg90H21EOLACvtfcHk0liGA5uAPXBa-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslCltLw%253D&md5=db43b2732ec9282c4e875cda92e2ab52</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-2611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-2611%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DM.%2BC.%26aulast%3DJeong%26aufirst%3DH.%26aulast%3DSon%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DD.%26aulast%3DGoughnour%26aufirst%3DP.%2BC.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DKwon%26aufirst%3DN.%2BH.%26aulast%3DMoon%26aufirst%3DH.%2BE.%26aulast%3DPaek%26aufirst%3DS.%2BH.%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DSeol%26aufirst%3DH.%2BJ.%26aulast%3DNam%26aufirst%3DD.%2BH.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DNovel%2520morphologic%2520and%2520genetic%2520analysis%2520of%2520cancer%2520cells%2520in%2520a%25203D%2520microenvironment%2520identifies%2520STAT3%2520as%2520a%2520regulator%2520of%2520tumor%2520permeability%2520barrier%2520function%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D1044%26epage%3D1054%26doi%3D10.1158%2F0008-5472.CAN-14-2611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+M.+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Sastry G Madhavi; Adzhigirey Matvey; Day Tyler; Annabhimoju Ramakrishna; Sherman Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepared prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove atomic clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addition, ligands must be prepared to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system preparation is generally accepted in the field, an extensive study of the preparation steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in preparing a system for virtual screening.  We first explore a large number of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper preparation and that neglecting certain steps of the preparation process produces a systematic degradation in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the preparation that impact database enrichment.  While the work presented here was performed with the Protein Preparation Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQd2Qiai-OqeXqj_yg7twvpfW6udTcc2eZwUUHYCEO3Jbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D&md5=9eb1a820e121aca2742ed53f24481ace</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0lh4aJGRnS52pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lh4aJGRnS52pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lh4aJGRnS52pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A34" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A34','PDB','5A34'); return false;">PDB: 5A34</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i95"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01961">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_41888"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01961?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01961</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Determining the significant residues for binding of <b>BC-DXI-843</b> to DX2 (Figure S1A,B); DX2 degradation caused by <b>BC-DXI-843</b> (Figure S1C,D); <b>BC-DXI-843</b>-dependent inhibition of DX2 and HSP70 binding (Figure S2); <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of the final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_001.pdf">jm9b01961_si_001.pdf (9.32 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_002.pdb">jm9b01961_si_002.pdb (7.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01961/suppl_file/jm9b01961_si_003.csv">jm9b01961_si_003.csv (5.87 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01961&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01961%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-10" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01961" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677ab2ed6d41d1db","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
