
<html lang="en"     class="pb-page"  data-request-id="879d1bc7-b7a1-40fd-908c-e088643bd218"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.5b00586;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2015.58.issue-23;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="New Frontiers in Druggability" /></meta><meta name="dc.Creator" content="Dima  Kozakov" /></meta><meta name="dc.Creator" content="David R.  Hall" /></meta><meta name="dc.Creator" content="Raeanne L.  Napoleon" /></meta><meta name="dc.Creator" content="Christine  Yueh" /></meta><meta name="dc.Creator" content="Adrian  Whitty" /></meta><meta name="dc.Creator" content="Sandor  Vajda" /></meta><meta name="dc.Description" content="A powerful early approach to evaluating the druggability of proteins involved determining the hit rate in NMR-based screening of a library of small compounds. Here, we show that a computational ana..." /></meta><meta name="Description" content="A powerful early approach to evaluating the druggability of proteins involved determining the hit rate in NMR-based screening of a library of small compounds. Here, we show that a computational ana..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 11, 2015" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b00586" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b00586" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b00586" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b00586" /></link>
        
    
    

<title>New Frontiers in Druggability | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b00586" /></meta><meta property="og:title" content="New Frontiers in Druggability" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0003.jpeg" /></meta><meta property="og:description" content="A powerful early approach to evaluating the druggability of proteins involved determining the hit rate in NMR-based screening of a library of small compounds. Here, we show that a computational analog of this method, based on mapping proteins using small molecules as probes, can reliably reproduce druggability results from NMR-based screening and can provide a more meaningful assessment in cases where the two approaches disagree. We apply the method to a large set of proteins. The results show that, because the method is based on the biophysics of binding rather than on empirical parametrization, meaningful information can be gained about classes of proteins and classes of compounds beyond those resembling validated targets and conventionally druglike ligands. In particular, the method identifies targets that, while not druggable by druglike compounds, may become druggable using compound classes such as macrocycles or other large molecules beyond the rule-of-five limit." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b00586"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b00586">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b00586&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b00586&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b00586&amp;href=/doi/10.1021/acs.jmedchem.5b00586" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 9063-9088</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.5b00669" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">New Frontiers in Druggability</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dima++Kozakov">Dima Kozakov</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+R.++Hall">David R. Hall</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Raeanne+L.++Napoleon">Raeanne L. Napoleon</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christine++Yueh">Christine Yueh</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Adrian++Whitty">Adrian Whitty</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sandor++Vajda">Sandor Vajda</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Applied Mathematics & Statistics, Stony Brook University, Stony Brook, New York 11794, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Acpharis Inc., Holliston, Massachusetts 01746, United States</span></div><div class="aff-info" id="aff3"><span class="aff-text"><sup>§</sup>Department of Chemistry and <sup>∥</sup>Department of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</span></div><div class="corresp-info"><strong>*</strong>(D.K.) Phone: 617-353-4842; E-mail: <a href="/cdn-cgi/l/email-protection#610c080500122103144f040514"><span class="__cf_email__" data-cfemail="caa7a3aeabb98aa8bfe4afaebf">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>(A.W.) Phone: 617-353-2488; E-mail: <a href="/cdn-cgi/l/email-protection#7e0916170a0a073e1c0b501b1a0b"><span class="__cf_email__" data-cfemail="bbccd3d2cfcfc2fbd9ce95dedfce">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>(S.V.) Phone: 617-353-4757; E-mail: <a href="/cdn-cgi/l/email-protection#1167707b75705173643f747564"><span class="__cf_email__" data-cfemail="0375626967624361762d666776">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b00586&amp;href=/doi/10.1021%2Facs.jmedchem.5b00586" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 9063–9088</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 31, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 April 2015</li><li><span class="item_label"><b>Published</b> online</span>11 August 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 December 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b00586" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00586</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9063%26pageCount%3D26%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDima%2BKozakov%252C%2BDavid%2BR.%2BHall%252C%2BRaeanne%2BL.%2BNapoleon%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D58%26issueNum%3D23%26contentID%3Dacs.jmedchem.5b00586%26title%3DNew%2BFrontiers%2Bin%2BDruggability%26numPages%3D26%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9088%26publicationDate%3DDecember%2B2015">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b00586"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">8961</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">60</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b00586" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;New Frontiers in Druggability&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dima&quot;,&quot;last_name&quot;:&quot;Kozakov&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;R. Hall&quot;},{&quot;first_name&quot;:&quot;Raeanne&quot;,&quot;last_name&quot;:&quot;L. Napoleon&quot;},{&quot;first_name&quot;:&quot;Christine&quot;,&quot;last_name&quot;:&quot;Yueh&quot;},{&quot;first_name&quot;:&quot;Adrian&quot;,&quot;last_name&quot;:&quot;Whitty&quot;},{&quot;first_name&quot;:&quot;Sandor&quot;,&quot;last_name&quot;:&quot;Vajda&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;9063-9088&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b00586&quot;},&quot;abstract&quot;:&quot;A powerful early approach to evaluating the druggability of proteins involved determining the hit rate in NMR-based screening of a library of small compounds. Here, we show that a computational analog of this method, based on mapping proteins using small molecules as probes, can reliably reproduce druggability results from NMR-based screening and can provide a more meaningful assessment in cases where the two approaches disagree. We apply the method to a large set of proteins. The results show that, because the method is based on the biophysics of binding rather than on empirical parametrization, meaningful information can be gained about classes of proteins and classes of compounds beyond those resembling validated targets and conventionally druglike ligands. In particular, the method identifies targets that, while not druggable by druglike compounds, may become druggable using compound classes such as macrocycles or other large molecules beyond the rule-of-five limit.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b00586&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00586" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b00586&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00586" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b00586&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00586" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b00586&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00586&amp;href=/doi/10.1021/acs.jmedchem.5b00586" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b00586" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b00586" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b00586%26sid%3Dliteratum%253Aachs%26pmid%3D26230724%26genre%3Darticle%26aulast%3DKozakov%26date%3D2015%26atitle%3DNew%2BFrontiers%2Bin%2BDruggability%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D23%26spage%3D9063%26epage%3D9088%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291967" title="Cluster chemistry">Cluster chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292601" title="Probes">Probes</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/jmcmar.2015.58.issue-23/20151210/jmcmar.2015.58.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A powerful early approach to evaluating the druggability of proteins involved determining the hit rate in NMR-based screening of a library of small compounds. Here, we show that a computational analog of this method, based on mapping proteins using small molecules as probes, can reliably reproduce druggability results from NMR-based screening and can provide a more meaningful assessment in cases where the two approaches disagree. We apply the method to a large set of proteins. The results show that, because the method is based on the biophysics of binding rather than on empirical parametrization, meaningful information can be gained about classes of proteins and classes of compounds beyond those resembling validated targets and conventionally druglike ligands. In particular, the method identifies targets that, while not druggable by druglike compounds, may become druggable using compound classes such as macrocycles or other large molecules beyond the rule-of-five limit.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16607" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16607" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Among the distinct proteins encoded by the human proteome, it had been estimated that only about 3000 belong to structural families for which small molecule (i.e., synthetic organic) drugs have been developed.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Many biologically compelling drug targets belong to other protein families that lack such empirical proof of principle that they can be inhibited by small molecule drugs. Consequently, approaches to assessing the druggability of potential drug targets, i.e., the likelihood of being able to identify a druglike small ligand that can modulate the activity of the target, have emerged as an important tool for target-based drug discovery. We note that throughout this perspective, we call a small compound druglike if it satisfies Lipinski’s rule of five (no more than five hydrogen-bond donors, no more than 10 hydrogen-bond acceptors, a molecular mass less than 500 g/mol, and an octanol–water partition coefficient not greater than 5).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> There are two major approaches to determining the druggability of a protein. One is based on empirical analysis of known target/drug associations,<a onclick="showRef(event, 'ref1 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref3 ref4">(1, 3, 4)</a> which can be applied only when the protein in question belongs to a well-studied family that includes close homologues for which reliable druggability information is available. The other is based solely on analysis of the structure of the target protein.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17">(5-17)</a> For novel targets, from families for which no drugs or advanced drug candidates are known, one has to rely on the latter method of structure-based predictive models. This approach was pioneered by Hajduk et al.,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> who re-examined the results of NMR-based fragment screening for a diverse set of 23 proteins and observed a high correlation between the fragment hit rate and the ability to identify high-affinity (<i>K</i><sub>d</sub> < 300 nM) druglike ligands. They used these data to derive a regression-based model to predict the likely fragment hit rate for untested targets, based on structural properties of the protein binding site such as total and apolar surface area and pocket compactness, generating for each a predicted druggability score, defined as the logarithm of the predicted NMR hit rate. The regression formula provided a good approximation of the original hit rates and, as we will show, resulted in fairly accurate prediction of druggability. Another important contribution came from Cheng et al.,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> who used the curvature and the lipophilic surface area of pockets to predict the maximal affinity achievable by passively absorbed drugs. Despite its simplicity, the model also provided good discrimination of druggable binding sites from those labeled as difficult or undruggable, but it was tested only on a limited number of proteins.</div><div class="NLM_p">To further improve druggability prediction, a variety of methods have been published in the last 6 years, with most of them trained on the growing number of protein targets with some level of druggability information available.<a onclick="showRef(event, 'ref5 ref10 ref11 ref12 ref14 ref16'); return false;" href="javascript:void(0);" class="ref ref5 ref10 ref11 ref12 ref14 ref16">(5, 10-12, 14, 16)</a> For method validation, Schmidtke and Barril compiled a large set of protein targets with associated degrees of druggability.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Krasowski et al.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> added further proteins to the training set and developed the DrugPred druggability score. All of these methods are based on the analysis of various pocket properties, including hydrophobicity, size, compactness, hydrogen-bond donor and -acceptor surface areas, and amino acid composition in pockets. Thus, evaluating the druggability of proteins with respect to conventional druglike compounds is now a mature technology that can perform such assessments with some reliability. However, this technology has considerable limitations. Apart from the NMR-based fragment screening by Hajduk et al.,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> all methods correlate pocket properties with druggability data available for traditional targets and therefore, in essence, simply evaluate whether the novel target in question contains a binding pocket that in some sense resembles pockets on known druggable targets that bind conventional druglike ligands. Moreover, the methods provide a single druggability scale that attempts to capture both the expected affinity that can be achieved as well as the druglikeness, and, implicitly, the potential for oral biovalability, of ligands that will potentially bind to the site. A general shortcoming of all of these methods, therefore, is that they are poorly equipped to evaluate the druggability of proteins with surface topologies that substantially differ from the relatively small number of protein families for which drugs or high-affinity druglike ligands exist, such as protein–protein interaction (PPI) targets, or to assess the potential of any target to bind ligands that do not conform to conventional definitions of druglikeness. Moreover, because these methods are parametrized rather than being based on biophysical principles and return only a monolithic measure of druggability, they provide little insight into why a particular target is or is not druggable and therefore into the prospects that strong binding might be achievable by an appropriately chosen nontraditional drug chemotype such as a natural product, a macrocyclic compound,<a onclick="showRef(event, 'ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21">(18-21)</a> a covalent inhibitor,<a onclick="showRef(event, 'ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24">(22-24)</a> or a peptide-derived foldamer.<a onclick="showRef(event, 'ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27 ref28">(25-28)</a></div><div class="NLM_p">It is additionally noteworthy that, among the above methods, only Seco et al.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> explicitly take into account the potential for conformational flexibility of the protein to affect the druggability of a binding site. The other methods base their analysis on static X-ray crystal structures. Several of the studies did, however, test the robustness of their results to variations in protein conformation, by comparing the results obtained after computationally perturbing the structure<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> or by comparing the results obtained for different crystal structures of the same target protein,<a onclick="showRef(event, 'ref5 ref8 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref5 ref8 ref11 ref12">(5, 8, 11, 12)</a> and generally reported that the ability to identify druggable sites was not greatly sensitive to such conformational effects. Other work has suggested, however, that conformational adaptivity at the binding site can substantially affect druggability, in cases where the alternative conformations that are sampled through thermal motions persist for a sufficient period of time.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p">In this perspective, we first discuss the advantages of the fragment screening approach used by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and introduce an analog of the method that uses a computational algorithm rather than experimental NMR data for fragment screening. The computational approach was used to evaluate some of the historical druggability results published during the past decade, including the 23 proteins considered in the NMR-based screening study<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> and the additional set of 72 proteins previously studied by the regression model, based on the NMR results and developed by the same authors.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> We also studied the protein set selected by Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and some of the more contentious results from the DrugPred algorithm of Krasowski et al.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> As will be shown, the method can reliably reproduce earlier druggability results. Moreover, evaluation of the small number of cases in which the original assessment was uncertain or was subsequently proved to be incorrect shows that the method presented here can, in fact, provide an even more meaningful assessment of druggability. We additionally show that biophysics-based computational approaches (as opposed to empirical approaches that are parametrized against a particular training set of protein–ligand complexes) are now sufficiently reliable that they can be used to address the potential for highly challenging or atypical drug targets, such as protein–protein interfaces, to be addressed using non-canonical drug chemotypes such as synthetic macrocycles. These methods, therefore, point the way to expanding the range of drug targets that can be addressed by using chemotypes that are known to be pharmaceutically relevant but that fall outside of conventional druglike chemical space. We also provide detailed discussion of the druggability of six targets of high pharmaceutical interest for which the prospects of success have been controversial.</div><div id="sec1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i2" class="anchor-spacer"></div><h3 class="article-section__title" id="_i2"> Experimental Fragment-Based Analysis of Druggability</h3><div class="NLM_p">As already mentioned, the structure-based analysis of druggability was introduced in a landmark paper by Hajduk and co-workers,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> who re-examined the results of NMR-based fragment screening experiments and observed a high correlation between the fragment hit rate and the ability to identify high-affinity druglike inhibitors. This early method has two major advantages over more recent approaches. First, by determining the binding locations of fragment-sized compounds, the screening based method automatically identifies the binding site. In contrast, the prediction methods described in the <a class="ref internalNav" href="#sec1" aria-label="Introduction">Introduction</a> are based on the analysis of pocket properties, primarily size, shape, complexity, and hydrophobicity, and thus have first to select the binding site residues. As noticed by Rarey and co-workers,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> pocket descriptors depend heavily on correctly selecting the region of the protein surface that is most likely to form a binding site. This is a nontrivial task for a ligand-free protein and is also complicated by potential conformational differences between ligand-bound and -free structures. Second, as Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> were aware, their method actually identifies binding hot spots, i.e., regions of the binding site that contribute a disproportionate amount of the binding free energy<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> and hence that are primarily responsible for ligand binding, in contrast to other methods that base their predictions on global pocket properties without regard for the extent to which sites within the pocket can contribute significant binding energy.</div><div class="NLM_p last">The concept of hot spots was originally introduced in the context of mutating interface residues to alanine in protein–protein or protein–peptide interfaces.<a onclick="showRef(event, 'ref30 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref32 ref33 ref34">(30, 32-34)</a> On the basis of this method, a residue is considered to be a hot spot if its mutation to alanine gives rise to a substantial drop in binding affinity. The concept was extended to the binding of small molecules by Ringe and co-workers, who used X-ray crystallography to determine the structure of the target protein soaked in aqueous solutions of 6–8 organic solvents.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> By superimposing the structures in these so-called multiple solvent crystal structure (MSCS) experiments, regions that bind multiple different probes were detected.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> While individual probes were able to bind at a number of locations, their clusters indicated binding hot spots. In their NMR-based experiments, Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> incubated the target protein with a series of small organic probes, generically referred to as fragments, and used the perturbations in residue chemical shifts to identify residues that participate in small molecule binding, thereby also finding the binding hot spots. While results of the MSCS method requiring the determination of X-ray structures in multiple solvents were reported only for a few proteins,<a onclick="showRef(event, 'ref36 ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38 ref39 ref40">(36-40)</a> Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> analyzed the results of NMR-based fragment screening for 23 proteins. These results convincingly demonstrated that the small fragment probe ligands cluster at hot spots and that the hit rate predicts the druggability of the site,<a onclick="showRef(event, 'ref7 ref35'); return false;" href="javascript:void(0);" class="ref ref7 ref35">(7, 35)</a> thereby initiating the development of structure-based druggability conditions.</div></div><div id="sec1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Computational Fragment Screening by FTMap</h3><div class="NLM_p">On the basis of the above results from the Hajduk<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and Ringe<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> groups, NMR and X-ray based fragment screening provides information on druggability and is now broadly used in pharmaceutical research.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41, 42)</a> However, the experiments are expensive and may be hampered by the requirement that the target protein be expressed and purified in relatively large quantities for experimental analysis.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Thus, it is desirable to replace these experimental methods with <i>in silico</i> approaches to evaluating druggability, particularly when exploring novel targets. Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> accomplished this goal by constructing a regression equation to express the NMR hit rates in terms of pocket properties. In this perspective, we describe druggability conditions based on direct simulation of the X-ray and NMR-based screening experiments. In fact, modeling the weakly specific binding of small and rigid ligands to proteins is not very difficult and hence computational methods offer attractive alternatives.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43, 44)</a> We use computational solvent mapping, implemented as the FTMap algorithm.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The method places small molecular probes of various sizes and shapes on a dense grid around the protein, finds favorable positions using empirical energy functions, refines the bound positions while accounting for side chain and probe flexibility, clusters the conformations, and ranks the clusters on the basis of the average energy. All ligands and crystallographic water molecules are removed prior to mapping, and the probes are initially distributed over the entire protein surface without any assumption as to the location of the binding site(s). As has been extensively validated, the regions that bind multiple low-energy probe clusters, called consensus cluster (CC) sites, identify the locations of the binding energy hot spots.<a onclick="showRef(event, 'ref40 ref45 ref46 ref47 ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref40 ref45 ref46 ref47 ref48 ref49 ref50">(40, 45-50)</a> The CCs are ranked on the basis of the number of probe clusters contained, which we have shown corresponds to relative energetic importance,<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> and are denoted as CC1, CC2, and so on. The CC with the highest number of probe clusters in the binding site of interest is defined as the primary hot spot, whereas other CCs with fewer probe clusters are secondary hot spots.</div><div class="NLM_p last">FTMap has been implemented as a server (<a href="http://ftmap.bu.edu/" class="extLink">http://ftmap.bu.edu/</a>), which currently uses 16 small molecules as probes (ethanol, isopropanol, isobutanol, acetone, acetaldehyde, dimethyl ether, cyclohexane, ethane, acetonitrile, urea, methylamine, phenol, benzaldehyde, benzene, acetamide, and <i>N</i>,<i>N</i>-dimethylformamide). The server and its potential applications were recently discussed in detail.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> We explored the use of substantially larger probe libraries, but, since this did not affect druggability predictions, we returned to the 16 compounds whose aqueous solutions have been used previously in mapping experiments based on NMR<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> or X-ray techniques.<a onclick="showRef(event, 'ref35 ref36 ref37 ref38 ref54'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38 ref54">(35-38, 54)</a> Since most probes include both polar and nonpolar moieties, our mapping results show that the highest number of probes cluster in pockets that have a mosaic-like pattern of hydrophobic and polar regions, enabling the binding of many different compounds in a variety of poses.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Druggability Analysis by FTMap</h3><div class="NLM_p">As will be described, we have mapped and analyzed over 150 ligand–protein complexes as well as the corresponding ligand-free proteins. To establish objective FTMap-based criteria for evaluating the druggability of proteins, we set out to benchmark our results by using the 16 probe types used in the FTMap server. For most target proteins, we mapped both the highest resolution ligand-free and -bound structures available in the Protein Data Bank (PDB).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> As explained below, generally, druggability can best be determined by mapping the ligand-free structure; although analysis of ligand-bound structure provides additional information, in most cases it is used only to indicate the binding location of a known ligand. On the basis of the analysis of this large set of proteins, we have observed that the hot spots of druggable proteins satisfy conditions on (1) strength, (2) connectivity or compactness, and (3) the maximum dimension of the hot spot region. We describe the origin of these conditions in turn.</div><div class="NLM_p">As already mentioned, Hajduk et al. observed that the hit rate in fragment screening is a predictor of druggability.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In agreement with this observation, we have previously established that, when using 16 probes for the mapping, the sites that are known to be druggable invariably contain a strong hot spot containing 16 or more probe clusters.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Indeed, among the 121 drug targets considered in earlier studies by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and Cheng et al.,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and also examined by FTMap here, only in a single case was there a druggable target with known, potent ligands that had fewer than 16 probe clusters in the strongest hot spot at the drug binding site (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Figure S1</a>), whereas we found 21 such targets among those that experimental evidence suggests are not druggable (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Figure S2</a>). Since FTMap clusters probe positions using pairwise 4 Å root-mean-square deviation (RMSD) as the clustering radius, the probe clusters are relatively small; thus, a consensus cluster (CC) can include several clusters of the same probe type. Thus, the requirement of 16 probe clusters does not imply that a druggable site has to bind all 16 different probes, although very frequently this is the case. FTMap analysis of the targets considered herein indicates that the requirement for a hot spot of at least 16 probe clusters is the most important condition for druggability: without it, high-affinity binding is essentially impossible (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Figure S1</a>).</div><div class="NLM_p">The second condition, requiring connectivity or compactness, is based on the observation that a druggable site should also include one or more secondary hot spots that are close enough to the primary hot spot to be reached by a drug-sized molecule.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> For the current work, we attempted to establish a quantitative threshold for how close these additional hot spots must be to support the strong binding of a druglike ligand so that we could include this measure as part of an objective metric for site druggability. We therefore analyzed the separation between hot spots seen for a large set of druggable sites that bind druglike ligands with molecular weight not exceeding 500 g/mol, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Figure S3</a>. The results show that the mean distance between the centers of CCs in druggable sites (i.e., the points with the highest density of probe atoms within the consensus clusters) is between 5 and 6 Å. However, to avoid missing any druggable sites, we designated 8 Å as the maximum allowed distance between connected hot spots, which captures over 99% of the druggable sites. In evaluating druggability, we used this criterion for hot spot proximity to characterize the ensemble of hot spots that constitute a particular binding site. Specifically, we first selected the primary hot spot in a given site (not necessarily CC1) to form the kernel of the ensemble. The ensemble was then expanded by iteratively adding all secondary hot spots within 8 Å from any hot spot already in the ensemble until no further expansion was possible. In this process, secondary hot spots with less than three probe clusters were ignored.</div><div class="NLM_p">The third criterion involving the size of the hot spot region is based on the observation that a binding site can be too small to be easily druggable. It is well-known that very small binding sites, and the fragment-sized ligands that bind to them, tend to be promiscuous.<a onclick="showRef(event, 'ref56 ref57 ref58 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref56 ref57 ref58 ref59 ref60">(56-60)</a> Indeed, this fact is implicit in the observations that experimental fragment libraries containing only 1000–2000 compounds typically provide multiple hits when screened against essentially any druggable target and that a druggable site is characterized by the observation of a high fragment hit rate, indicating that each site can bind many different fragment structures.<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56, 57)</a> Thus, while a small site containing one or more strong CCs can be sufficient to enable strong ligand binding, for the binding to be selective, it appears to be also necessary that the site be sufficiently large and topologically complex that a ligand that is complementary to this site is unlikely to bind as strongly to unrelated sites.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> As the simplest way to capture the extent of the binding site, or at least that portion of the binding site that is relevant for the generation of binding energy with the ligand, we simply report the longest dimension of the CC ensemble, which we defined as the distance between the two most widely separated probe atoms across the ensemble of probe clusters. In the following analysis, we found that the maximum extent of the binding site should exceed 10 Å for a site to be considered druggable. Indeed, among the druggable targets that have known druglike ligands, we found only three violations of this rule (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Figure S4</a>).</div><div class="NLM_p">On the basis of the three properties described above, we classify target sites on proteins into four broad classes of druggability (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Classification of Protein Druggability Based on FTMap Results</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">druggability class</th><th class="colsep0 rowsep0" align="center">strength (S) (no. of probe clusters)</th><th class="colsep0 rowsep0" align="center">center-to-center distance (CD), Å</th><th class="colsep0 rowsep0" align="center">maximum dimension (MD), Å</th><th class="colsep0 rowsep0" align="center">druggability subclass</th><th class="colsep0 rowsep0" align="center">notation</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">druggable</td><td class="colsep0 rowsep0" align="left">S ≥ 16</td><td class="colsep0 rowsep0" align="left">CD < 8</td><td class="colsep0 rowsep0" align="left">MD ≥ 10</td><td class="colsep0 rowsep0" align="left">druggable using druglike compounds</td><td class="colsep0 rowsep0" align="left">D</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">not druggable</td><td class="colsep0 rowsep0" align="left">S < 13</td><td class="colsep0 rowsep0" align="left">any CD</td><td class="colsep0 rowsep0" align="left">any MD</td><td class="colsep0 rowsep0" align="left">not druggable due to weak hot spots</td><td class="colsep0 rowsep0" rowspan="2" align="left">N</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">not druggable</td><td class="colsep0 rowsep0" align="left">any S</td><td class="colsep0 rowsep0" align="left">any CD</td><td class="colsep0 rowsep0" align="left">MD < 7</td><td class="colsep0 rowsep0" align="left">not druggable due to small hot spot ensemble</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">non-canonically druggable–large</td><td class="colsep0 rowsep0" align="left">S ≥ 16</td><td class="colsep0 rowsep0" align="left">CD ≥ 8</td><td class="colsep0 rowsep0" align="left">MD ≥ 10</td><td class="colsep0 rowsep0" align="left">druggable only by large chemotype such as macrocycle or foldamer</td><td class="colsep0 rowsep0" align="left">D*<sub>L</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">non-canonically druggable–small</td><td class="colsep0 rowsep0" align="left">S ≥ 16</td><td class="colsep0 rowsep0" align="left">CD < 8</td><td class="colsep0 rowsep0" align="left">7 ≤ MD < 10</td><td class="colsep0 rowsep0" align="left">druggable only by peptide, macrocycle, or charged compound</td><td class="colsep0 rowsep0" align="left">D*<sub>S</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">borderline druggable</td><td class="colsep0 rowsep0" align="left">13 ≤ S < 16</td><td class="colsep0 rowsep0" align="left">CD < 8</td><td class="colsep0 rowsep0" align="left">MD ≥ 10</td><td class="colsep0 rowsep0" align="left">at most micromolar affinity using druglike compounds</td><td class="colsep0 rowsep0" align="left">B</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">borderline non-canonically druggable–large</td><td class="colsep0 rowsep0" align="left">13 ≤ S < 16</td><td class="colsep0 rowsep0" align="left">CD ≥ 8</td><td class="colsep0 rowsep0" align="left">MD ≥ 10</td><td class="colsep0 rowsep0" align="left">micromolar affinity by large chemotype such as macrocycle or foldamer</td><td class="colsep0 rowsep0" align="left">B*<sub>L</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">borderline non-canonically druggable–small</td><td class="colsep0 rowsep0" align="left">13 ≤ S < 16</td><td class="colsep0 rowsep0" align="left">CD < 8</td><td class="colsep0 rowsep0" align="left">7 ≤ MD < 10</td><td class="colsep0 rowsep0" align="left">micromolar affinity by peptide, macrocycle, or charged compound</td><td class="colsep0 rowsep0" align="left">B*<sub>S</sub></td></tr></tbody></table></div></div><div id="sec1_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6">(1) Druggable</h4><div class="NLM_p last">A binding site is categorized as druggable with respect to conventional, druglike (e.g., rule-of-five compliant)<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> compounds if it meets the following criteria: (a) it includes a strong primary hot spot with at least 16 probe clusters, denoted S ≥ 16. (b) One or more secondary hot spots are connected to the primary hot spot with a center-to-center distance (CD) < 8 Å. (c) The maximum dimension (MD) of the connected hot spot ensemble is at least 10 Å, i.e., MD ≥ 10 Å (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div></div><div id="sec1_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7">(2) Not Druggable</h4><div class="NLM_p last">If the primary hot spot is very weak (S < 13) or if the hot spot ensemble is very small (MD < 7 Å), then a site is not druggable irrespective of the compounds used.</div></div><div id="sec1_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8">(3) Non-canonically Druggable–Large</h4><div class="NLM_p last">If a sufficient number of sufficiently strong hot spots are present (S ≥ 16) but the hot spot ensemble is too sparse, requiring a center-to-center distance exceeding 8 Å to reach a secondary hot spot from the primary one, then small druglike compounds cannot bind with sufficient affinity. However, the evidence indicates that such sites generally can bind strongly to larger but still pharmaceutically relevant compounds such as macrocycles<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61, 62)</a> or peptide/peptidomimetic foldamers.<a onclick="showRef(event, 'ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref63 ref64">(63, 64)</a> As will be discussed, it is possible that a subset of hot spots can satisfy the conditions CD < 8 Å, S ≥ 16, and MD ≥ 10 Å but that there exist additional strong hot spots beyond the ensemble. In such cases, the target is druggable using druglike molecules, but there is the potential for binding affinity to be improved if the size of the compounds is increased to reach all hot spots, although doing so will likely require using compounds that are large and so lie outside of conventional druglike chemical space.</div></div><div id="sec1_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9">(4) Non-canonically Druggable–Small</h4><div class="NLM_p">Another class of sites is encountered when the hot spot ensemble is small but not very small (7 Å ≤ MD < 10 Å). Such sites may be druggable using ligands with a charged group that is not fully desolvated upon binding. Alternatively, druggability can be achieved by large and thus not druglike compounds (peptides,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> peptidomimetics,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> or macrocycles<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a>) that extend beyond the small hot spot ensemble to interact with large surface regions without any strong hot spot. As will be demonstrated, while such interactions are weak, they still contribute to the binding free energy.</div><div class="NLM_p">It is important to note that druggability is not an on–off property. In particular, for categories (1), (3), and (4), a site where the primary hot spot is somewhat weaker than the threshold (S ≤ 13 < 16), we use the term borderline druggable (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). For such sites, ligands can be found that bind with micromolar affinity, but attempts to increase potency usually fail. More generally, the strengths and arrangement of hot spots provide information beyond the simple four categories above. As an example, we discuss here the analysis of druggability for the FK506 binding protein (FKBP), which binds 8-deethyl-8-[but-3-enyl]-ascomycin (FK506), a macrocyclic compound with a molecular weight (MW) of 804 g/mol (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Later, we will also study ABT737, a well-known regulator of apopotosis by inhibiting the interaction between the Bcl-xL protein and BAK peptide (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B).<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a><a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C shows the results of mapping a ligand-free structure of FKBP (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn">2ppn</a>). As mentioned, each hot spot is identified by a consensus cluster (CC). In all of our figures, each probe cluster contained in a CC is represented by the single lowest energy probe structure to avoid the overcrowding of the figure. All probes in a CC have the same color, using the color code as follows: CC1, cyan; CC2, magenta; CC3, yellow; CC4, salmon; CC5, white; CC6, blue; CC7, orange; and CC8, green. <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C also shows the location of the bound FK506 (taken from the structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj">1fkj</a>) superimposed for reference. The ligand binding site contains three connected hot spots. These were the top-ranked hot spot, CC1, which contained 18 probe clusters, plus two secondary CCs, CC3 (14 probe clusters) and CC6 (7 probe clusters). The center-to-center distance (CD) between CC1 and CC3 is 6.3 Å, and the maximum dimension of the ensemble with the two hot spots is 10.6 Å. Thus, according to our conditions, the site would be druggable when using a drug that binds only to these two hot spots. However, since CC1 is not much stronger than the required S = 16, and the MD value is close to the required threshold of 10 Å, the affinity would be moderate. Since the CD distance between CC1 and CC6 is 7.2 Å, CC6 can be also added to the hot spot example used for the binding. Adding the third hot spot CC6 (7 probe clusters) increases MD to 12.3 Å. According to <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, all three hot spots in this extended ensemble still can be reached using druglike compounds. In fact, FKBP has been cocrystallized with smaller but still potent inhibitors. Examples are PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkh">1fkh</a> (MW = 453, <i>K</i><sub>i</sub> = 7 nM)<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> and PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f40">1f40</a> (MW = 360.5, <i>K</i><sub>i</sub> = 7.5 nM).<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> However, the inhibitors that reach the three hot spots are fairly long, and connecting the two far ends by forming a macrocycle in FK506 improves the affinity to <i>K</i><sub>i</sub> = 0.2 nM.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> This example also demonstrates that, while the analysis of druggability could be simplified by integrating the three different requirements into a single druggability measure, considering the three requirements separately can more explicitly show problems that may lead to the lack of druggability, as well as explore opportunities for using non-canonical compounds as drugs.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Using hot spot properties to assess druggability. (A) Structure of FK506, considered in the example of druggability analysis by FTMap. (B) Structure of ABT737, a well-known example of protein–protein interaction inhibitors. (C) Mapping the ligand-free FK506 binding protein (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn">2ppn</a>, shown as surface), with bound FK506 (green sticks, from the structure with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj">1fkj</a>) superimposed for reference. The consensus clusters shown are CC1 (cyan, 18 probe clusters), CC3 (yellow, 14 probe clusters), and CC6 (blue, 7 probe clusters). The center-to-center distance (CD) between CC1 and CC3 is 6.3 Å, and the maximum dimension of the ensemble of the two hot spots is 10.6 Å. Thus, the subset of these two hot spots is druggable on its own but would bind small ligands with only moderate potency. Using all three hot spots increases MD to 12.3 Å and allows for high-affinity binding. (D) Distribution of the parameters used for assessing druggability of the 23 protein targets in the Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> data set. (E) Top view of the distribution shown in panel D to show the separation of different druggability classes. Note that the non-canonically druggable–large (D*<sub>L</sub>) category is defined by the large (CD ≥ 8 Å) distance between the centers of the hot spots. (F) Right view of the distribution shown in panel D. The non-canonically druggable–small (D*<sub>S</sub>) category is defined by the small (MD < 10 Å) dimension of the entire hot spot ensemble.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">If several structures of a target are available, then an important question is which structure to use for the analysis. Frequently, druggability needs to be established before discovering any ligand, motivating preference for unbound structures. If multiple unbound structures are available, then it is useful to map all and to consider the one that results in the highest level of druggability, assuming that ligands can be optimized to take advantage of the available hot spots. While ligand binding may affect their location and relative strength,<a onclick="showRef(event, 'ref70 ref71'); return false;" href="javascript:void(0);" class="ref ref70 ref71">(70, 71)</a> hot spots are generally less sensitive to conformational changes than binding sites are,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and the analysis of the ligand-free structures provides valid druggability assessment for most targets. However, the binding sites in the unliganded structures of some proteins may be completely closed, or ligand binding may create new hot spots; hence, it is generally advisible to also map ligand-bound structures when they become available in the process of drug discovery. This additional information frequently shows if there is a choice between a large and hence nondruglike ligand that reaches all major hot spots or a smaller and less potent druglike ligand that can take advantage of only the main hot spot and some of the weaker hot spots around it.</div></div></div><div id="sec1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Druggability Results from NMR-Based Screening versus Predicted by FTMap</h3><div class="NLM_p"><a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> shows the analysis of the 23 proteins in the test set from Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> using the three FTMap criteria. In two cases, PTP1B (T19 in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) and survivin (T22), the protein was found to contain two distinct binding sites that were characterized separately, resulting in a total of 25 binding sites. Druggability results are also shown graphically in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>D–F. <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> lists the short names of the targets and specifies their binding sites, as given by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> While we quote the Protein Data Bank (PDB) entry as given in the original publication,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> for our analysis, we mapped both the highest resolution ligand-free and -bound structures currently available in the PDB.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> As discussed, in most cases, druggability can be determined by mapping the ligand-free structure alone, and the ligand-bound structure is needed only to judge whether the hot spots are located in the appropriate binding site for the purpose of retrospective validation with respect to experimentally identified ligands. For the protein PAK4 (T16 in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>), no structure was found in the PDB; hence, the druggability of this target is not discussed. The affinities shown in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> are the ones given by Hajduk et al.,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> but we also list (in italics) if more potent compounds have subsequently been reported. <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S1</a> is a copy of <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, but its footnotes provide detailed information on targets and their affinities, with references. Additional references may not be listed if the value is given in PDBbind,<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> BindingDB,<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> or BindingMOAD.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a><a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> also shows the experimental and predicted druggability scores (DS) calculated by Hajduk et al.,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> which represent the logarithm of the actual or predicted hit rate in an NMR-based fragment screen. These authors observed that most of the targets for which high-affinity, druglike leads could be identified had DS values greater than −1.0, whereas DS values below −1.5 indicated proteins without such ligands. Consequently, a protein pocket with DS > −1.0 was considered to have high druggability, whereas low druggability was assigned for targets with DS < −1.5. The DS value may be close to −2.0 for very undruggable proteins. Protein pockets with intermediate values (−1.5 < DS < −1.0) were assigned as having moderate druggability.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Druggability Results from NMR-Based Screening versus Those Predicted by FTMap</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">PDB</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">druggability score (DS)<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></th><th class="rowsep1 colsep0" colspan="2" align="center">distances, Å</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">target</th><th class="colsep0 rowsep0" align="center">site</th><th class="colsep0 rowsep0" align="center">structure<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></th><th class="colsep0 rowsep0" align="center">U</th><th class="colsep0 rowsep0" align="center">B</th><th class="colsep0 rowsep0" align="center">affinity (nM)</th><th class="colsep0 rowsep0" align="center">consensus sites</th><th class="colsep0 rowsep0" align="center">D<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">exp.</th><th class="colsep0 rowsep0" align="center">pred.</th><th class="colsep0 rowsep0" align="center">to further CCs</th><th class="colsep0 rowsep0" align="center">max. dim.</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">AK</td><td class="colsep0 rowsep0" align="left">Adenosine</td><td class="colsep0 rowsep0" align="left">Internal</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bx4">1bx4</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2i6b">2i6b</a></td><td class="colsep0 rowsep0" align="left"><10</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bx4">1bx4</a>: 1(19), 8(4)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.66</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.42</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bx4">1bx4</a>: 7.3</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bx4">1bx4</a>: 11.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>68</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2a6b">2a6b</a>: 1(21), 6(6), 7(5)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2a6b">2a6b</a>: 6.5, 6.8</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2a6b">2a6b</a>: 11.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">Akt-PH</td><td class="colsep0 rowsep0" align="left">IP3</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1h10">1h10</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1unr">1unr</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1h10">1h10</a></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1unr">1unr</a>: 3(14), 4(10)</td><td class="colsep0 rowsep0" rowspan="2" align="left">B</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.91</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.98</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1unr">1unr</a>: 7.8</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1unr">1unr</a>: 14.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>6300</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1h10">1h10</a>: 5(9), 6(8)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1h10">1h10</a>: 7.0</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1h10">1h10</a>: 11.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">Bcl-xL</td><td class="colsep0 rowsep0" align="left">Bak</td><td class="colsep0 rowsep0" align="left">Internal</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1r2d">1r2d</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2yxj">2yxj</a></td><td class="colsep0 rowsep0" align="left"><200</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1r2d">1r2d</a>: 1(18), 2(17), 5(9)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.11</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.64</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1r2d">1r2d</a>: 7.4, 5.8,</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1r2d">1r2d</a>: 12.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>60</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2yxj">2yxj</a>: 1(28), 2(25), 4(9), 5(8)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2yxj">2yxj</a>: 6.3, 7.7, 7.0</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2yxj">2yxj</a>: 16.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">XIAP Bir3</td><td class="colsep0 rowsep0" align="left">Pept.</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g3f">1g3f</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2opz">2opz</a></td><td class="colsep0 rowsep0" align="left"><i>28</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2opz">2opz</a>: 1(23), 7(5)</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">–1.03</td><td class="colsep0 rowsep0" align="left">–0.72</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2opz">2opz</a>: 3.1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2opz">2opz</a>: 10.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">5</td><td class="colsep0 rowsep0" rowspan="2" align="left">CMPK</td><td class="colsep0 rowsep0" rowspan="2" align="left">CMP</td><td class="colsep0 rowsep0" rowspan="2" align="left">Internal</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cke">1cke</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cmk">2cmk</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cke">1cke</a>: 1(20), 3(10)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.13</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.81</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cke">1cke</a>: 5.3</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cke">1cke</a>: 14.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cmk">2cmk</a>: 1(19), 4(10)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cmk">2cmk</a>: 4.7</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cmk">2cmk</a>: 11.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">E2-31</td><td class="colsep0 rowsep0" align="left">DNA</td><td class="colsep0 rowsep0" align="left">Internal</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dhm">1dhm</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jj4">1jj4</a></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dhm">1dhm</a>: 5(12), 6(7)</td><td class="colsep0 rowsep0" rowspan="2" align="left">N</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.71</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.72</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dhm">1dhm</a>: 5.4</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dhm">1dhm</a>: 9.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>60 000</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jj4">1jj4</a>: 4(11)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jj4">1jj4</a>: −</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jj4">1jj4</a>: 4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">ErmAM</td><td class="colsep0 rowsep0" align="left">SAH</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qam">1qam</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qam">1qam</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qan">1qan</a></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qam">1qam</a>: 1(25), 3(16)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.01</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.87</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qam">1qam</a>: 5.6</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qam">1qam</a>: 11.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>1100</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qan">1qan</a>: 2(15), 3(12), 5(8)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qan">1qan</a>: 5.3, 7.8</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qan">1qan</a>: 15.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">FBP</td><td class="colsep0 rowsep0" align="left">DNA</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1j4w">1j4w</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1j4w">1j4w</a></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1j4w">1j4w</a>: 1(18), 2(15)</td><td class="colsep0 rowsep0" align="left">D*<sub>S</sub></td><td class="colsep0 rowsep0" align="left">–1.61</td><td class="colsep0 rowsep0" align="left">–1.04</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1j4w">1j4w</a>: 5.1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1j4w">1j4w</a>: 9.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">FKBP</td><td class="colsep0 rowsep0" align="left">FK506</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj">1fkj</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn">2ppn</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj">1fkj</a></td><td class="colsep0 rowsep0" align="left"><10</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn">2ppn</a>: 1(18), 3(14), 6(7)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.03</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.24</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn">2ppn</a>: 6.3, 7.2</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn">2ppn</a>: 11.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>2.3</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj">1fkj</a>: 1(34), 2(16)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj">1fkj</a>: 7.9</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj">1fkj</a>: 10.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">HI-0065</td><td class="colsep0 rowsep0" align="left">ADP</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fl9">1fl9</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fl9">1fl9</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1htw">1htw</a></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fl9">1fl9</a>: 3(13), 5(8)</td><td class="colsep0 rowsep0" rowspan="2" align="left">B</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.82</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.28</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fl9">1fl9</a>: 6.2</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fl9">1fl9</a>: 11.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>10 000</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1htw">1htw</a>: 2(19)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1htw">1htw</a>: −</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1htw">1htw</a>: 7.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">11</td><td class="colsep0 rowsep0" rowspan="2" align="left">LCK</td><td class="colsep0 rowsep0" rowspan="2" align="left">PTyr</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lkl">1lkl</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3hck">3hck</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lkl">1lkl</a></td><td class="colsep0 rowsep0" rowspan="2" align="left">300</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3hck">3hck</a>: 3(18), 5(9), 6(7)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.21</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.07</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3hck">3hck</a>: 6.6. 7.6</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3hck">3hck</a>: 11.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lkl">1lkl</a>: 6(9), 8(3)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lkl">1lkl</a>: 6.2</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lkl">1lkl</a>: 9.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">LFA</td><td class="colsep0 rowsep0" align="left">IDAS</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rd4">1rd4</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lfa">1lfa</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rd4">1rd4</a></td><td class="colsep0 rowsep0" align="left"><50</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lfa">1lfa</a>: 1(31), 4(12)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.40</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.35</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lfa">1lfa</a>: 5.6</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lfa">1lfa</a>: 11.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>18.3</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rd4">1rd4</a>: 1(18), 2(13), 3(12), 7(6), 8(6)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rd4">1rd4</a>: 5.7, 6.0, 4.7, 3.7</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rd4">1rd4</a>: 13.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">13</td><td class="colsep0 rowsep0" rowspan="2" align="left">MDM2</td><td class="colsep0 rowsep0" rowspan="2" align="left">p53</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ycr">1ycr</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1z1m">1z1m</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rv1">1rv1</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><100</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1z1m">1z1m</a>: 1(21), 2(21)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.49</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.35</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1z1m">1z1m</a>: 5.6</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1z1m">1z1m</a>: 10.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rv1">1rv1</a>: 1(21), 2(20), 3(19)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rv1">1rv1</a>: 5.7, 4.7</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rv1">1rv1</a>: 14.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">14</td><td class="colsep0 rowsep0" rowspan="2" align="left">MurA</td><td class="colsep0 rowsep0" rowspan="2" align="left">UDPNAG</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1uae">1uae</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ejd">1ejd</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1uae">1uae</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><i>250</i></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ejd">1ejd</a>: 2(22), 4(10), 5(8)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.38</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.44</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ejd">1ejd</a>: 4.3, 0.8, 5.9, 7.2</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ejd">1ejd</a>: 13.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1uae">1uae</a>: 1(21), 2(12), 3(11), 4(8)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1uae">1uae</a>: 16.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">15</td><td class="colsep0 rowsep0" rowspan="2" align="left">MurI</td><td class="colsep0 rowsep0" rowspan="2" align="left">Glu</td><td class="colsep0 rowsep0" rowspan="2" align="left">Internal</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b73">1b73</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b74">1b74</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><20</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b73">1b73</a>: 1(30), 2(14), 3(12), 4(11), 5(9), 6(8), 7(7)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.93</td><td class="colsep0 rowsep0" rowspan="2" align="left">–2.00</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b73">1b73</a>: 5.3, 7.5, 5.3, 7.1, 3.0, 4.9</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b73">1b73</a>: 15.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b74">1b74</a>: 1(20), 3(14), 4(12), 6(9)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b74">1b74</a>: 4.0, 7.6, 3.8,</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b74">1b74</a>: 14.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">PAK4</td><td class="colsep0 rowsep0" align="left">ATP</td><td class="colsep0 rowsep0" align="left">Inter.</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">–0.78</td><td class="colsep0 rowsep0" align="left">–0.63</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">17</td><td class="colsep0 rowsep0" rowspan="2" align="left">Pin1 WW domain</td><td class="colsep0 rowsep0" rowspan="2" align="left">Pept.</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8h">1i8h</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i6c">1i6c</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8g">1i8g</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8h">1i8h</a></td><td class="colsep0 rowsep0" rowspan="2" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i6c">1i6c</a>: 1(28), 2(18), 4(13)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D*<sub>L</sub></td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.94</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.49</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i6c">1i6c</a>: 8.0, 4.6</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i6c">1i6c</a>: 14.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8h">1i8h</a>: 1(20), 6(11), 7(4)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8h">1i8h</a>: 6.3, 5.6</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8h">1i8h</a>: 13.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">18</td><td class="colsep0 rowsep0" rowspan="2" align="left">PDZ–PSD95</td><td class="colsep0 rowsep0" rowspan="2" align="left">Pept.</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu0">1iu0</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu2">1iu2</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rgr">1rgr</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><300</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu2">1iu2</a>: 1(22), 5(11)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D*<sub>L</sub></td><td class="colsep0 rowsep0" rowspan="2" align="left">–2.00</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.99</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu2">1iu2</a>: 9.8</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu2">1iu2</a>: 12.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rgr">1rgr</a>: 1(22), 6(7)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rgr">1rgr</a>: 5.9</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rgr">1rgr</a>: 8.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">19</td><td class="colsep0 rowsep0" rowspan="2" align="left">PTP1B</td><td class="colsep0 rowsep0" rowspan="2" align="left">Cat. pY</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ph0">1ph0</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a></td><td class="colsep0 rowsep0" rowspan="2" align="left">1pty</td><td class="colsep0 rowsep0" rowspan="2" align="left"><10</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a>: 1(20), 2(16)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.68</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.15</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a>: 6.8</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a>: 11.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a>: 1(13), 7(7)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a>: 5.3</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a>: 9.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">19</td><td class="colsep0 rowsep0" rowspan="2" align="left">PTP1B</td><td class="colsep0 rowsep0" rowspan="2" align="left">noncat. pY</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ph0">1ph0</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a></td><td class="colsep0 rowsep0" rowspan="2" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a>: 4(9), 7(7), 8(7)</td><td class="colsep0 rowsep0" rowspan="2" align="left">N</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.77</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.66</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a>: 5.0, 6.2</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a>: 11.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a>: 8(7), 9(7)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a>: 5.9</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a>: 10.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">SARS N-term</td><td class="colsep0 rowsep0" align="left">RNA</td><td class="colsep0 rowsep0" align="left">Internal</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ssk">1ssk</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>1000<br /></br><i>∼ 10</i><sup>6</sup></td><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">–1.93</td><td class="colsep0 rowsep0" align="left">–1.92</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">21</td><td class="colsep0 rowsep0" rowspan="2" align="left">SCD</td><td class="colsep0 rowsep0" rowspan="2" align="left">Subst.</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g4k">1g4k</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cqr">1cqr</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g4k">1g4k</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><10</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cqr">1cqr</a>: 1(26), 5(7)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.09</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.55</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cqr">1cqr</a>: 4.7</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cqr">1cqr</a>: 12.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g4k">1g4k</a>: 1(20), 2(17), 5(9), 7(4)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g4k">1g4k</a>: 4.6, 4.8, 3.2</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g4k">1g4k</a>: 13.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">22</td><td class="colsep0 rowsep0" rowspan="2" align="left">Survivin</td><td class="colsep0 rowsep0" rowspan="2" align="left">Bir3</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uih">3uih</a></td><td class="colsep0 rowsep0" rowspan="2" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a>: 7(5)</td><td class="colsep0 rowsep0" rowspan="2" align="left">N</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.97</td><td class="colsep0 rowsep0" rowspan="2" align="left">–1.99</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a>: −</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a>: 4.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uih">3uih</a>: 8(5)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uih">3uih</a>: −</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uih">3uih</a>: 4.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">Survivin</td><td class="colsep0 rowsep0" align="left">other</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><300</td><td class="colsep0 rowsep0" align="left">1(24), 2(24)</td><td class="colsep0 rowsep0" align="left">D*<sub>L</sub></td><td class="colsep0 rowsep0" align="left">–0.45</td><td class="colsep0 rowsep0" align="left">–0.35</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a>: 8.3</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a>: 13.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">23</td><td class="colsep0 rowsep0" rowspan="2" align="left">UK</td><td class="colsep0 rowsep0" rowspan="2" align="left">Pept.</td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fv9">1fv9</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2o8t">2o8t</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fv9">1fv9</a></td><td class="colsep0 rowsep0" rowspan="2" align="left"><50</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2o8t">2o8t</a>: 1(29), 2(26)</td><td class="colsep0 rowsep0" rowspan="2" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.40</td><td class="colsep0 rowsep0" rowspan="2" align="left">–0.81</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2o8t">2o8t</a>: 6.0</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2o8t">2o8t</a>: 17.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fv9">1fv9</a>: 1(18), 2(15), 5(9)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fv9">1fv9</a>: 4.3, 2.1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fv9">1fv9</a>: 13.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Druggability class based on FTMap analysis, as described in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</p></div></div></div><div class="NLM_p">According to the FTMap results, 14 of the 25 distinct binding sites in this test set satisfy all three criteria for druggability: i.e., the top-ranking CC in the cluster contains at least 16 probe clusters, the ensemble contains at least two CCs with a center-to-center distance of less than 8 Å, and the maximum dimension of the ensemble is at least 10 Å. These 14 sites are therefore classified as druggable in our analysis, indicating a strong likelihood of finding a druglike small molecule ligand that binds strongly to the site. Such ligands are known for 12 of these 14 targets (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S1</a>). The exceptions are cytidine monophosphate kinase (CMPK, T5), with only micromolar ligands identified so far,<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> and rRNA methyltransferase ErmC (ErmAM, T7), for which the best known inhibitor has IC<sub>50</sub> ∼ 1100 nM.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> Two targets, Akt-PH (T2) and HI-0065 (T10), are predicted as being borderline druggable and have micromolar ligands.<a onclick="showRef(event, 'ref77 ref78'); return false;" href="javascript:void(0);" class="ref ref77 ref78">(77, 78)</a> Four sites (E2-31, the noncatalytic phosphotyrosyl binding site of PTP1B, the N-terminal domain of SARS, and the Bir3 domain of survivin) are classified as not druggable, in each case because of a weak main hot spot (S < 13). According to FTMap, three targets (the WW domain of Pin1, PDZ–PSD95, and the “other” site of survivin) belong to the D*<sub>L</sub> category; thus, achieving substantial potency requires peptide-like or macrocyclic ligands. PZD-PSD95 (T18) was considered not to be druggable by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> (DS = −2.00), but FTMap identifies two hot spots in the site that are 9.8 Å apart (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A); hence, based on the FTMap results, this site can be targeted by peptides or macrocycles that are large enough to reach the two hot spots.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> In fact, PZD-PSD95 binds cyclic peptides with IC<sub>50</sub> ∼ 1000 nM in a solvent-exposed shallow pocket.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> The two main hot spots are also too far apart in the peptide binding site of Pin1 WW domain (T17). We note that, based on the PDB code (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ih8">1ih8</a>) given by Hajduk et al.,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> we consider here the WW domain of Pin1 that has only peptides as known ligands<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> rather than the peptidyl prolyl cis/trans isomerase (PPIase) domain that is the target of more recent studies.<a onclick="showRef(event, 'ref81 ref82 ref83'); return false;" href="javascript:void(0);" class="ref ref81 ref82 ref83">(81-83)</a> Survivin (T22 in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) has two binding sites of interest. As shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B, in contrast to XIAP Bir3, the Smac peptide binding site of this Bir3 domain includes only CC7 with 5 probe clusters and hence is not druggable. However, the “other” site in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, close to the dimer interface, includes two strong hot spots that are 8.3 Å apart, almost within the reach of druglike ligands. Inhibitors that binds to both hot spots have <i>K</i><sub>d</sub> values as low as 37 nM.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> Finally, based on the FTMap results, one target, FUSE-binding protein (FBP, T8 in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>), is in the D*<sub>S</sub> category due to the small size of the hot spot ensemble. In fact, the only known inhibitors of DNA binding to FBP are charged and of high micromolar affinity.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Hot spots in two protein–protein interaction targets, belonging to the druggability class D*<sub>L</sub>. (A) Mapping of the ligand free PDZ-1 domain of PSD-95 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu2">1iu2</a>, shown as surface), with a cyclic peptide (from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rgr">1rgr</a>, shown as yellow stick model) superimposed for reference. The two hot spots in the interface, CC1 (cyan, 22 probe clusters) and CC5 (white, 11 probe clusters), are 9.8 Å from each other. (B) Mapping of the ligand-free survivin dimer (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a>, shown as cyan and green cartoon), with the Smac/DIABLO peptide (from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uih">3uih</a>, shown as yellow cartoon) superimposed for reference. The only hot spot in the survivin/Smac interface is CC7 (orange, 5 probe clusters); thus, the peptide binding site is not druggable. A novel site is defined by the strong hot spots CC1 (cyan, 24 probe clusters) and CC2 (magenta, also 24 probe clusters) at the dimerization interface. Since the center-to-center distance between CC1 and CC2 is 8.3 Å, ligands that bind to both are over the 500 g/mol MW limit.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The FTMap results generally agree well with the assessment based on experimental and predicted druggability scores given by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> as well as with the actual success or failure to identify a high-affinity druglike ligand that binds to the site in question. We have substantive differences with the DS value from Hajduk et al. only for 3 of the 25 sites. Moreover, the available experimental data suggest that, in these three cases, the druggability assignment based on FTMap is, in fact, the correct one. Specifically, the DNA binding site of the human papillomavirus (HPV) E2-31 protein (Target T6 in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) is considered to be druggable by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> but not by FTMap. The best inhibitor of the HPV E2 DNA binding site in the literature has an IC<sub>50</sub> value of approximately 10 μM,<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> and due to the consistent lack of success in finding more potent ligands for this site, efforts have been largely redirected to target the DNA rather than the protein.<a onclick="showRef(event, 'ref87 ref88'); return false;" href="javascript:void(0);" class="ref ref87 ref88">(87, 88)</a> Moreover, the majority of DNA binding sites of proteins have limited druggability.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> FTMap also differs from the DS method in predicting that the adenosine diphosphate (ADP) binding site of the hypothetical bacterial ATPase HI-0065, also known as YjeE (T10 in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), is at most borderline druggable. HI-0065 was targeted by the Abbott group,<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> but the best compound identified by NMR-based screening and optimization had a <i>K</i><sub>d</sub> value of about 5 μM. We note that both HPV E2-31 and HI-0065 have strong binding hot spots close to, but not overlapping with, the ligand binding site, which may have resulted in a misleadingly high NMR fragment hit rates. The third deviation between FTMap and the earlier results occurs for the glutamate racemase MurI that was rated as not druggable by Hajduk et al.,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> although they reported that 4-substituted <span class="smallcaps smallerCapital">d</span>-glutamic acid analogues (not considered druglike) bind with nanomolar affinity.<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> It is not clear why the NMR-based screening did not result in hits. According to FTMap, this target is druggable, emphasizing that our condition does not directly account for the lack of druglikeness due to the need for a charged ligand, unless the hot spot region is too small. In this particular case, we note that a close homologue, glutamate racemase from <i>Helicobacter pylori</i>, has potent and druglike noncompetitive inhibitors that bind to the corresponding site.<a onclick="showRef(event, 'ref90 ref91'); return false;" href="javascript:void(0);" class="ref ref90 ref91">(90, 91)</a></div><div class="NLM_p last">Comparison of FTMap results obtained using the ligand-bound structure rather than the unbound structure of the protein (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) shows that small differences between the structures can affect the strength and location of hot spots but that such changes impact druggability assessment only for three targets, HI-0065, LCK, and PTP1B, all having strongly charged ligands with a phosphate group (see footnotes to <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S1</a>). It appears that the binding of such ligands tightens the binding pocket, which then binds fewer probe clusters. Thus, the mapping of ligand-free proteins generally provides valid druggability information, emphasizing that druggability is an inherent property of a protein. Nevertheless, as will be further discussed, it may be useful to also map ligand-bound protein structures if available, since, in some cases, ligand binding may open additional subsites and create additional hot spots, favoring druggability.</div></div><div id="sec1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Validation of FTMap Druggability Assessment against Extended 72 Protein Set of Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></h3><div class="NLM_p">In addition to analyzing the 23 drug targets for which experimental NMR fragment screening results were available, Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> calculated druggability indices for a set of 72 proteins based on cocrystal structures with ligands. They did this by constructing a regression equation to calculate a druggability score based on geometric features of the binding site, including the total and apolar surface area, compactness, and the first and third principal moments of the pocket, which they parametrized using the results of NMR fragment screening of the original 23 protein test set. The 72 protein set includes 35 proteins for which high-affinity, druglike molecules have been reported plus 37 proteins for which the known ligands were either not high affinity or not druglike. <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S2</a> compares the results of FTMap analysis of this extended 72 protein set to the druggability scores (DS) reported by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">For each of these 72 targets, we mapped the highest resolution ligand-free structure available in the PDB. Bound structures are listed only to specify the binding site of interest. Among the 35 proteins that were considered by Hajduk et al. to have druggability, the FTMap results disagree with the high DS value only for two targets, cytochrome c peroxidase (Target T17) and leghemoglobin (Target T32). Target T17, given by PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ccg">1ccg</a>,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> is actually the His175Gly mutant of cytochrome c peroxidase, which binds imidazole in a cavity created by the removal of His175 side chain, but it does so with a <i>K</i><sub>d</sub> value of only 2.7 mM.<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> FTMap finds this imidazole binding site to be not druggable. Although a number of strong hot spots exist in the large cavity of the protein, these do not overlap with the bound imidazole position. The problem with leghemoglobin (T32) is that the binding site is formed in the process of ligand binding; hence, only the bound structure, which was studied by Hajduk et al.,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> reveals the druggability of this target. Since we generally consider the mapping of the ligand-free structure to be more predictive of druggability, this may appear to be a shortcoming of our approach. However, leghemoglobin is not a drug target, and its binding pocket is extremely small, accommodating just a nicotinate with molecular weight 122 g/mol.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> As we have discussed, such small ligands are unlikely to be drugs, and, in fact, the binding of nicotinate is weak (<i>K</i><sub>d</sub> = 1400 nM).<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a></div><div class="NLM_p">Among the remaining 37 structures that did not have high-affinity druglike ligands, the agreement between FTMap and DS druggability predictions was also generally good; however, we again see some differences. Phosphoserine phosphatase (T36) is rated as being druggable by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> (DS = −0.808), but it is only borderline druggable by FTMap. No potent inhibitor is indicated by Hajduk et al.,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and such a compound has not been reported in the subsequent literature. In the apo structure of flavodoxin (T37, DS = −0.857), the side chains of W57 and Y94 completely close down the flavin mononucleotide binding site;<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> hence, FTMap does not find a hot spot there. As mentioned, proteins for which the druggability of the bound conformation is so greatly different from that of the apoprotein are very rare; we have found only two cases (leghemoglobin and flavodoxin) among the more than 150 proteins we have studied, and these two are not drug targets. FTMap confirms druggability for most other targets with DS > −1.2 (Targets T38 through T45), and, apart from inositol monophosphatase (T44), these proteins indeed have known potent ligands, as does sialidase (T46), with DS = −1.201 (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S2</a>). Targets PI phospholipase (T47) and enterotoxin (T48), with lower DS values, we rate as borderline druggable, and phosphofructokinase (T49), as not druggable. Consistent with these ratings, none of these proteins has any known potent ligand. The druggability of targets with DS values between −1.334 and −1.5 is somewhat undefined by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> FTMap shows substantial variation in this range, and for several proteins, our results are supported by binding data. In particular, protein kinase Cδ (PKCδ, T50 in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S2</a>) is in the D*<sub>S</sub> category, and it binds phorbol-12,13-dibutyrate, a nondruglike compound with MW = 504.6 g/mol, with nanomolar <i>K</i><sub>d</sub>.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> FTMap confirms the druggability of the GTP binding site of Ras p21 (T52 in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S2</a>). Guanylate kinase (T53) is not druggable, whereas based on FTMap, chloramphenicol acetyltransferase (T54) is in the D*<sub>S</sub> category and indeed is known to bind fusidic acid with <i>K</i><sub>i</sub> = 5400 nM.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> FTMap predicts seven targets to be druggable (aldehyde ferredoxin oxidoreductase, galactose binding protein, cytidine deaminase, trichosanthin, achromobacter protease, diaminopimelic acid dehydrogenase, and glutamine methyltransferase) that were assigned a low druggability index (DS < −1.5) by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In three cases, the FTMap assignment is supported by known affinity data. These three are the galactose binding protein (T61), which binds galactose with <i>K</i><sub>d</sub> = 400 nM,<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> cytidine deaminase (T62), which has potent inhibitors,<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> and achromobacter protease (T64), which has inhibitors with <i>K</i><sub>i</sub> = 6.5 nM.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> We could not find data confirming druggability of the remaining four targets, aldehyde ferredoxin oxidoreductase (T60), trichosanthin (T63), diaminopimelic acid dehydrogenase (T66), and glutamine methyltransferase (T71), but in three cases, T60, T63, and T66, the proteins are known to bind (nondruglike) small substrates or cofactors with relatively high affinity (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S2</a>). In the same range (i.e., with DS < −1.5), we predict four targets in the D*<sub>S</sub> category. Two of these, periplasmic glucose/galactose receptor (T69) and IP phosphatase (T70), are known to bind charged ligands with nanomolar affinity (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S2</a>). We have no information on the other two, deoxynucleoside monophosphate kinase (T56) and glucoamylase (T58). Finally, <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S2</a> has one target, <sup>pp60</sup>Src SH2 domain (T57), with DS < −1.5, that FTMap places in the D*<sub>L</sub> category. This domain is known to bind a peptidomimetic ligand with IC<sub>50</sub> = 1800 nM.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a></div><div class="NLM_p last">As shown, we were able to find binding data for a substantial fraction of targets that were predicted to be druggable by FTMap but that had DS < −1.5 assigned by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> This poses the question of why such low DS values were predicted. The most likely reason is that the regression formula for DS is based on the properties of pockets rather than of the hot spots, and this may underestimate potential binding capacity for pockets that have a somewhat unusual shape. Indeed, targets T55, T60, and T61 all have completely buried ligand binding sites with no solvent accessibility in the ligand-bound structures considered by Hajduk et al.,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> which likely contributed to their low DS values.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> However, FTMap can still place probes in such internal cavities, and the proteins are likely to have alternative conformations with more open pockets. Trichosanthin (T63) and glutamine methyltransferase (T71) have deep and narrow binding pockets, whereas the binding site in diaminopimelic acid dehydrogenase (T66) is largely solvent-exposed, and it seems that the simple regression formula by Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> underestimates the druggability for such unusually shaped pockets.</div></div><div id="sec1_5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Validation of FTMap Druggability Assessment against the Test Set of Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></h3><div class="NLM_p">A biophysical model developed by Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> accounts for lipophilic surface area and curvature of binding sites to compute a score that corresponds to the maximal affinity predicted for a passively absorbed oral drug (MAP<sub>POD</sub>). MAP<sub>POD</sub> is based on the hypothesis that the maximal achievable binding free energy is largely due to desolvation, and it models the desolvation contributions from target and ligand in terms of nonpolar surface areas with coefficients that depend on surface curvature. The expression does not have an explicit polar desolvation term, since it is assumed that the electrostatic interactions and the desolvation energies largely cancel each other.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> This model was used to assess the druggability of 27 binding sites. The value of this test set is that all sites are of pharmaceutical interest, in contrast to later test sets that included many proteins without this property. The 27 target sites were classified as druggable (17), difficult (6), or undruggable (4). Nearly all targets classified as druggable or difficult had associated orally bioavailable, marketed drugs, whereas the undruggable targets did not, despite considerable efforts within the industry.</div><div class="NLM_p">In <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S3</a>, we compare the FTMap results to the maximal affinity (MAP<sub>POD</sub>) values predicted by Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> that were recently reviewed.<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> Both ligand-free and -bound structures were mapped, but, as mentioned, we consider the unbound results to be more relevant for assessing druggability. The agreement with the results of Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> is generally good. For the 23 druggable or difficult targets, we have different results only for the binding site of COX-2 (Target T6 in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S3</a>), which FTMap places into the D*<sub>S</sub> category due to the small size of the hot spot ensemble, predicting that potent ligands binding to this site must have a charged or highly polar group not fully desolvated upon binding. Indeed, all strong COX-2 inhibitors seem to have either a carboxyl or a terminal sulfonamide group.<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> examined six targets that they considered to be difficult, namely, ACE-1 (T1), HIV RT nucleotide site (T13), IMPDH (T16), neuraminidase (T18), penicillin binding protein (T22), and thrombin (T23), because they required highly polar molecules that were either processed from prodrugs or relied on active transport mechanisms. Thrombin (T23) is predicted to be druggable both by Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and by FTMap and now has direct (i.e., not prodrug) oral inhibitors.<a onclick="showRef(event, 'ref103 ref104'); return false;" href="javascript:void(0);" class="ref ref103 ref104">(103, 104)</a> With 15 probe clusters, the penicillin binding protein (T22) is borderline druggable by FTMap, and ligand binding substantially strengthens the hot spot. This protein binds piperacillin,<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> an approved but not orally bioavailable drug (MW= 517.5 g/mol), with an IC<sub>50</sub> of 166 nM.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> FTMap predicts that ACE-1 (T1), HIV RT nucleotide site (T13), IMPDH (T16), and neuraminidase (T18) are all druggable, although the primary hot spot in IMPDH is relatively weak, with only 17 probe clusters, and ACE-1 and neuraminidase have strong but small hot spots (MD < 11.0). Both ACE-1 and neuraminidase have approved drugs that include a carboxyl group but are not prodrugs, for example, <i>N</i>2-[(1<i>S</i>)-1-carboxy-3-phenylpropyl]-<span class="smallcaps smallerCapital">l</span>-lysyl-<span class="smallcaps smallerCapital">l</span>-proline (lisinopril)<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> for ACE-1 and (2<i>R</i>,3<i>R</i>,4<i>S</i>)-3-acetamido-4-(diaminomethylideneamino)-2-[(1<i>R</i>,2<i>R</i>)-1,2,3-trihydroxypropyl]-3,4-dihydro-2<i>H</i>-pyran-6-carboxylic acid (zanamivir)<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> for influenza neuraminidase. These results confirm that ACE-1 and neuraminidase are correctly classified as druggable, but they also show that a small hot spot is likely to require a charged ligand. Two targets, Factor Xa (T10) and HMG CoA reductase (T15), were placed on the border of the druggable region by Cheng et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Factor Xa is categorized by FTMap as D*<sub>L</sub>, and it has druglike inhibitors close to the upper limit on size. For example, the inhibitor methyl(2<i>R</i>,3<i>R</i>)-3-([[3′-(aminomethyl)biphenyl-4-yl]carbonyl]amino)-2-(3-carbamimidoylbenzyl)butanoate (RPR128515),<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> bound in the structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ezq">1ezq</a>, has a molecular weight of 458.6 g/mol.<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> Both Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and FTMap underestimate the druggability of HMG CoA reductase, which has potent statin inhibitors,<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> but is considered to be only borderline druggable by FTMap (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S3</a>). This is due to the fact that statins occupy only a portion of the large ligand binding site. While this is enough to block access of the substrate to the active site of the enzyme, the unoccupied region also includes a number of hot spots, thus resulting in fewer probe clusters overlapping with the inhibitor.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a></div><div class="NLM_p last">Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> classify four targets as being not druggable: cathepsin K (T24), Caspase 1 (T25), the active site of HIV integrase (T26), and PTP1B. Cathepsin K (T24) and the catalytic site of the HIV integrase are predicted to be druggable by FTMap. This finding is supported by data that became available after the publication of the analysis by Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Cathepsin K has acrylic cyanamide-based inhibitors with <i>K</i><sub>i</sub> values as low as 0.003 nM.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> The osteoporosis drug <i>N</i>-(1-cyanocyclopropyl)-4-fluoro-<i>N</i>2-[(1<i>S</i>)-2,2,2-trifluoro-1-[4′-(methylsulfonyl)biphenyl-4-yl]ethyl]-<span class="smallcaps smallerCapital">l</span>-leucinamide (Odanacatib),<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> in Phase III clinical trial by Merck, is a potent cathepsin K inhibitor, and current studies focus on safety issues rather than affinity or oral bioavailability.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> For HIV integrase, there exist several FDA approved orally bioavalable inhibitors, such as <i>N</i>-(4-fluorobenzyl)-5-hydroxy-1-methyl-2-(2-[[(5-methyl-1,3,4-oxadiazol-2-yl)carbonyl]amino]-2-propanyl)-6-oxo-1,6-dihydro-4-pyrimidinecarboxamide (raltegravir),<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> with low nanomolar IC<sub>50</sub> values. These newly druggable sites emphasize the importance of assessing druggability using biophysical rather than purely statistical methods. However, we agree with Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> that the other two targets are very difficult. FTMap results categorize Caspase 1 as borderline druggable at best due to the weak hot spots and the small size of the hot spot region. Indeed, although there are peptidomimetic inhibitors containing a carboxyl group with <i>K</i><sub>i</sub> values as low as 5 nM, their oral bioavailability is limited,<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> and based on our mapping results, it is unlikely that more druglike inhibitors will be found. The druggability of PTP1B will be discussed in detail later in this perspective and hence is not considered here.</div></div><div id="sec1_5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Validation of FTMap Druggability Assessment against DrugPred</h3><div class="NLM_p last">Krasowski et al.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> compiled a nonredundant data set containing crystal structures of 71 druggable and 44 less druggable proteins. This data set was used to train a structure-based druggability predictor called DrugPred. We did not apply FTMap to this entire set because, in contrast to the set by Cheng et al.,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> it includes many proteins that have not been targets of substantive screening campaigns. However, we mapped the targets for which DrugPred clearly failed. These included six targets (DPP-IV, urokinase plasminogen activator, factor Xa, human thymidine phosphorylase, thiamine pyrophosphokinase, and thymidine kinase) that were included as druggable in either training or validation sets by Krasowski et al.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> but were predicted to be undruggable by DrugPred and two proteins (peptide deformylase and CDP-<span class="smallcaps smallerCapital">d</span>-glucose synthase) that were included as undruggable but were predicted to be druggable (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S4</a>). These analyses were performed to obtain more information on the likely druggability of these targets and to explore whether the FTMap results differ from those of DrudPred. Of the six targets that were considered to have false negative predictions,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> FTMap correctly predicts two (DPP-IV and urokinase plasminogen activator) to be druggable. As already shown, factor Xa is categorized as D*<sub>L</sub>, but it still binds druglike ligands, albeit at the upper limit of molecular weight.<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> Human thymidine phosphorylase is classified as B*<sub>S</sub>, and it is known to require charged ligands or prodrugs for nanomolar affinity.<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> The two remaining false negatives (thiamine pyrophosphokinase and thymidine kinase) are predicted not to be druggable also by FTMap. While we did not find any evidence that thiamine pyrophosphokinase has potent inhibitors, the thymidine kinase of the herpes simplex virus has inhibitors with <i>K</i><sub>i</sub> values as low as 5 nM.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> According to FTMap, the thymidine kinase binding site has only weak hot spots both in apo and holo structures (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S4</a>), making this target the sole real failure of our method. The apparent reason is that the thymidine kinase inhibitors are planar molecules that bind in an extremely narrow and fully buried pocket that can accommodate only the few probes that are also planar. This problem is very rare, and at this point, we have no solution to avoid it. The last two targets in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S4</a> were considered to be false positives by Krasowski et al.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and were also predicted to be druggable by FTMap. Of these, peptide deformylase now has low nanomolar inhibitors.<a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a> CDP-<span class="smallcaps smallerCapital">d</span>-glucose synthase binds cytidine-5′-triphosphate with <i>K</i><sub>i</sub> = 35 μM,<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a> but it has not been considered to be a drug target and hence its druggability has not been established experimentally.</div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Targetting Conventionally Undruggable Targets with Non-canonical Drug Chemotypes</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28283" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28283" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Because the FTMAP algorithm evaluates the binding potential of protein surface sites based on structural and physicochemical principles, rather than by empirical parametrization using a particular test set of target proteins, the method has the potential to provide information on the druggability of sites that differ in structure or properties from those that previously have been successfully targeted in drug discovery efforts. We therefore evaluated whether the FTMap method can shed light on precisely why sites that were not assessed to be druggable failed to qualify. This information is potentially useful in deciding whether the site might be targeted by a non-canonical drug chemotype, such as a peptide, a macrocycle, or a covalent inhibitor, or whether the target is likely to be intractable to all approaches currently available for small molecule drug discovery. Of the targets in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> that were rated as not druggable, four of them, E2-31, the noncatalytic site of PTP1B, the SARS N-terminal domain, and the bir3 site of survivin, simply lack a hot spot of sufficient strength to serve as a strong anchor site for ligand binding. Such targets are unlikely to bind strongly to any small ligand, so we consider them likely to be undruggable by any such means. However, PDZ-PSD95 and the WW domain of Pin1 have strong hot spots that are positioned too far apart to be easily spanned by a ligand of typical drug size, and in the “other” site of survivin, the separation of hot spots also slightly exceeds 8 Å. In these cases, one might expect that these sites will not be conventionally druggable but might bind strongly to a somewhat larger ligand such as a peptide-derived foldamer or a large macrocycle; hence, they are classified as non-canonically druggable–large, denoted D*<sub>L</sub>. Other examples include the <sup>pp60</sup>Src SH2 domain, elongation factor TU, the catalytic domain of protein phosphatase-1, and the site on interleukin-2 (IL-2) that binds the IL-2 receptor α-chain. In each of these cases, FTMap analysis indicates that the binding site contains a sufficient number of strong hot spots to support high-affinity binding of a ligand, but these hot spots are spaced too far apart to be encompassed by a ligand the size of a typical drug; thus, the site requires a larger chemotype to achieve strong binding (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Tables S1–S6</a> for details). Furthermore, we have recently reported and further discuss below that the binding sites on proteins binding natural product macrocycles of MW > 600 g/mol differ from conventional drug binding sites in having hot spots that are spaced farther apart, supporting the notion that large ligands such as large macrocycles can bind to sites that have sufficiently strong hot spots but fail the standard druggability criterion due to the spacing between the hot spots being too large.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Druggability of Proteins Binding Macrocyclic Compounds</h3><div class="NLM_p">It is generally recognized that macrocyclic compounds can be useful to expand the class of druggable targets and provide additional options against DNA, RNA, and carbohydrate binding proteins and protein–protein interactions.<a onclick="showRef(event, 'ref18 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref18 ref20 ref21">(18, 20, 21)</a> In <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>, we show druggability results for a number of proteins that bind macrocyclic natural products with high affinity (except for actin, where the binding is at best micromolar). Additional details on individual targets are given in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S5</a>. These targets include the already discussed FK506 binding protein (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C), which FTMap indicates is druggable even using druglike compounds, as confirmed by published experimental data,<a onclick="showRef(event, 'ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref68 ref69">(68, 69)</a> albeit not with the very high affinity of the macrocyclic FK506 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> We find a similar result for a number of other targets in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>. Cyclophilin A, pancreatic elastase, Chitinase, and HSP90 are all predicted to be conventionally druggable, and, indeed, in addition to their potent nanomolar macrocyclic inhibitors, these proteins are known to also have somewhat less potent nonmacrocyclic inhibitors (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S5</a>). However, in the case of three proteins, elongation factor TU, protein phosphatase 1, and the biotin carboxylase (BC) domain of human acetyl-CoA carboxylase 2, the hot spots are too far from each other to render the sites conventionally druggable; hence, these sites can be targeted only using larger compounds such as macrocycles or peptide foldamers.<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> According to FTMap, actin is not druggable by any means. In fact, although the protein has been cocrystallized with several large macrocycles and the compounds have biological activity, the binding is very weak in all cases (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S5</a>). These results support our previous suggestion<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> that sites that fall outside the FTMap criteria for conventional druggability for the sole reason that the hot spots are spaced too far apart might be good targets for macrocyclic compounds or other larger ligands. For this reason, we classify such sites as non-canonically druggable, D*<sub>L</sub> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), to indicate that they are not druggable using conventional druglike compounds but that they could be druggable using other, larger but still pharmaceutically relevant, drug chemotypes.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Druggability of Proteins Binding Macrocyclic Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">PDB</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">protein name</th><th class="colsep0 rowsep0" align="center">U</th><th class="colsep0 rowsep0" align="center">B</th><th class="colsep0 rowsep0" align="center">hot spots</th><th class="colsep0 rowsep0" align="center">D<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">comment</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">Cyclophilin A</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cpl">2cpl</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cwa">1cwa</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1sfa">1sfa</a></td><td class="colsep0 rowsep0" align="left">1(23), 3(15), 5(6)</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">CC1 and CC5 are overlapping, CC5–CC3 = 5.8 Å, MD = 14.0 Å, druggable by druglike compounds</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">FKBP</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn">2ppn</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2tke">2tke</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2dg3">2dg3</a></td><td class="colsep0 rowsep0" align="left">1(18), 3(14), 6(7), 7(6)</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">CC1, CC3, and CC7 are close and make it already druggable by druglike compounds (MD = 10.5 Å); CC1–CC6 = 7.2 Å</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">Pancreatic elastase</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lvy">1lvy</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1okx">1okx</a></td><td class="colsep0 rowsep0" align="left">2(19), 5(8), 6(6), 7(5)</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">CC2 is elongated, MD = 9.5 Å (borderline), but CC2–CC7 = 6 Å, CC6–CC4= 5.3 Å, CC2–CC6 = 7.2 Å, thus all are connected</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">Elongation factor TU</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1efc">1efc</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dt8">1dt8</a></td><td class="colsep0 rowsep0" align="left">1(21), 4(12), 6(5)</td><td class="colsep0 rowsep0" align="left">D*<sub>L</sub></td><td class="colsep0 rowsep0" align="left">CC1 and CC6 are close but too small (9.0 Å); CC1–CC4 = 11.5 Å, too far. Druggable only using a long ligand (peptide) or a macrocycle</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">Protein phosphatase 1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3egg">3egg</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3e7a">3e7a</a></td><td class="colsep0 rowsep0" align="left">2(16), 6(7)</td><td class="colsep0 rowsep0" align="left">D*<sub>L</sub></td><td class="colsep0 rowsep0" align="left">CC2–CC6 = 12.9 Å. Too far, druggable only using either a macrocycle or a very elongated ligand</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">Type I signal peptidase</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kn9">1kn9</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1t7d">1t7d</a></td><td class="colsep0 rowsep0" align="left">2(18)</td><td class="colsep0 rowsep0" align="left">D*<sub>S</sub></td><td class="colsep0 rowsep0" align="left">CC2 is elongated, but MD = 10.0 Å. Small and weak hot spot</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">Chitinase</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1guv">1guv</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1waw">1waw</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1wb0">1wb0</a></td><td class="colsep0 rowsep0" align="left">1(23), 4(10), 5(6), 6(6)</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">Overlapping hot spots, total MD = 12.6 Å. Also druggable using druglike ligands</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">Actin</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1nwk">1nwk</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qz5">1qz5</a></td><td class="colsep0 rowsep0" align="left">3(14)</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">Inherently not druggable, additional surface area is needed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2q0u">2q0u</a></td><td class="colsep0 rowsep0" align="left">2(16), 7(4)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CC2–CC7 = 8.0 Å, too weak hot spots, at most borderline</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2q0r">2q0r</a></td><td class="colsep0 rowsep0" align="left">10(2)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Inherently not druggable, additional surface area is needed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">AcetylCoA Carboxylase</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3glk">3glk</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gid">3gid</a></td><td class="colsep0 rowsep0" align="left">3(15), 4(9), 5(8), 6(6)</td><td class="colsep0 rowsep0" align="left">B*<sub>L</sub></td><td class="colsep0 rowsep0" align="left">CC3 and CC6 are close, but MD = 9.0 Å only. CC5 is close to CC4, but the distance between CC3 and CC5 is 9.0 Å, too large. Druggable only using a macrocycle or very long compound</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">HSP90 (closed)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yer">1yer</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yet">1yet</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3inw">3inw</a></td><td class="colsep0 rowsep0" align="left">1(23), 3(10), 5(8), 6(8)</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">CC1, CC3, and CC5 provide druggability, MD = 14.5 Å; CC1–CC6 is 8.9 Å, too far; macrocycle is useful, but not necessary</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HSP90 (open)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yes">1yes</a></td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yet">1yet</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3inw">3inw</a></td><td class="colsep0 rowsep0" align="left">1(22), 2(13), 3(12), 5(9)</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">CC1, CC2, and CC5 provide druggability, MD = 11.4 Å; CC2–CC3 = 7.67 Å, also part of the hot spot ensemble</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Druggability class based on FTMap analysis, as described in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Druggability of Protein–Protein Interaction Targets</h3><div class="NLM_p last">A substantial number of PPI interfaces are biologically compelling targets for drug discovery, and in some cases, small molecules are known to inhibit the interactions of two proteins with moderate to high potency.<a onclick="showRef(event, 'ref121 ref122'); return false;" href="javascript:void(0);" class="ref ref121 ref122">(121, 122)</a> The druggability of such targets is still poorly understood, however, and PPI target sites are generally considered to be highly challenging if not undruggable. The problem in assessing the druggability of PPI targets is that protein–protein interfaces generally have much smaller pockets than traditional drug targets, and it is difficult to determine which one, if any, will be able to bind a small compound with sufficient affinity to modulate the interaction at pharmacologically relevant doses.<a onclick="showRef(event, 'ref123'); return false;" href="javascript:void(0);" class="ref ref123">(123)</a> We recently showed that FTMap can identify the most important ligandable interface binding sites based on analysis of the unbound protein structure.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Here, we review these results in the light of our current and more extensive druggability analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S6</a>). PPI targets T1–T8 of <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S6</a> (Bcl-xL, XIAP Bir3, MDM2, Pin1, PDZ–PSD95, the survivin Bir3 domain, the “other” site on survivin, and urokinase) are already in the Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> data set of <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, and we have previously studied the druggability of targets T9–T12 (IL-2, HPV-11 E2, TNF-α, and stromelysin 1 catalytic domain).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Targets T13–T15 (ZipA, K-Ras, and HIV integrase) will be described in more detail in the next section. On the basis of the mapping results collected in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S6</a>, five of the 15 PPI targets (Bcl-xL, XIAP Bir3, MDM2, urokinase, and HPV-11 E2) appear to be druggable using druglike compounds. However, the binding of small ligands in protein–protein interfaces may introduce conformational changes that affect the location and relative strength of the hot spots. While these changes do not affect the predicted druggability, they may impact the class of compounds required for strong binding. For example, in the ligand-free structure of Bcl-xL, the three hot spots are close to each other, indicating druggability. Indeed, it has been reported that fragments of known inhibitors all bind in the pocket that binds L578 of the BAK peptide.<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a> Upon binding of the inhibitor ABT737, shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B,<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> the hot spots move farther apart (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S6</a>). Nevertheless, Bcl-xL is assessed to be druggable also on the basis of mapping the ligand-bound structure, since the distance from CC1 to CC5 is 6.3 Å, from CC5 to CC4 is 7.7 Å, and from CC4 to CC2 is 7.0 Å. However, to achieve high potency, ABT737 must extend from CC1 all the way to CC2, a distance of 11.3 Å, and this requires long molecules with MW > 650 g/mol.<a onclick="showRef(event, 'ref48 ref126 ref127'); return false;" href="javascript:void(0);" class="ref ref48 ref126 ref127">(48, 126, 127)</a> In XIAP Bir3, the two hot spots are close to each other, and several potent monomeric inhibitors have been developed.<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> However, the hot spot region is quite small (MD = 10.6 Å); thus, XIAP Bir3 is close to the D*<sub>S</sub> category. Accordingly, the inhibitors are druglike but relatively large. For example, the inhibitor (2<i>S</i>)-1-[(2<i>S</i>)-2-cyclohexyl-2-[[(2<i>S</i>)-2-(methylamino)propanoyl]amino]acetyl]-<i>N</i>-(4-phenylthiadiazol-5-yl)pyrrolidine-2-carboxamide (GDC-0152)<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> has a MW of 498.2, and recently macrocyclic inhibitors have also been developed.<a onclick="showRef(event, 'ref129'); return false;" href="javascript:void(0);" class="ref ref129">(129)</a> Mapping results for MDM2 are similar: the protein has strong hot spots close to each other, but the region is small (MD = 10.6 Å), and the inhibitors in the Nutlin family<a onclick="showRef(event, 'ref130'); return false;" href="javascript:void(0);" class="ref ref130">(130)</a> and related compounds spread beyond the hot spots, resulting in MW of over 600 g/mol.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a> However, MD exceeds the threshold of 10.0 Å, and more recent optimizations succeeded to yield inhibitors with MW below 600 g/mol and even slightly below 500 g/mol.<a onclick="showRef(event, 'ref132 ref133'); return false;" href="javascript:void(0);" class="ref ref132 ref133">(132, 133)</a> Urokinase is druggable, and druglike nanomolar inhibitors are known.<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a> HPV-11 E2 is also druggable, but the hot spot region is small (MD = 10.8 Å), and known inhibitors with high nanomolar affinity have MW > 500 g/mol.<a onclick="showRef(event, 'ref135'); return false;" href="javascript:void(0);" class="ref ref135">(135)</a> On the basis of FTMap, three targets (the Smac peptide binding site of survivin, the FtsZ interface of ZipA, and the receptor binding interface of TNF-α) are not druggable with noncovalent inhibitors under any circumstances.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In fact, all three interactions have been targets of extensive screening campaigns that did not produce any potent compounds. The SOS-binding interface of KRAS, which is discussed in greater detail below, is borderline druggable, and the best noncovalent inhibitor found so far has <i>K</i><sub>d</sub> = 190 μM.<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> Three targets in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S6</a> (the WW domain of Pin1, PDZ-PSD95, the interface of IL-2 with IL-2Rα) contain strong hot spots but with spacing that exceeds 8 Å; thus, they fall into the non-canonically druggable–large (D*<sub>L</sub>) category and are druggable only using large (peptide-like or macrocyclic) ligands. Since the 8.3 Å distance between the two strong hot spots in the “other” site of survivin barely puts this target into the D*<sub>L</sub> category, relatively small (although not <500 g/mol) potent ligands have been developed.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> In the unbound structure of the stromelysin 1 catalytic domain, the ligand binding pocket is partially closed, and since the hot spot ensemble is small, FTMap predicts druggability in the D*<sub>S</sub> category. We also mapped the structure complexed with the inhibitor 5-methyl-5-(4-phenoxyphenyl)pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,5<i>H</i>)-trione (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g4k">1g4k</a>),<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a> which showed a more extended hot spot region and hence suggested a higher level of druggability. However, it appears that the most potent inhibitor still has only <i>K</i><sub>i</sub> = 192 nM.<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a> Although we have only a limited number of examples, the ones considered here suggest that the even the druggable PPI targets frequently require the use of compounds lying outside of traditional druglike chemical space.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Druggability Analysis of Selected High-Value Targets</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43038" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43038" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this final section, we apply our analysis to a number of targets that have been of high pharmaceutical interest and for which the question of druggability is not fully settled. Most of the targets considered in this section are complex and undergo significant conformational change upon ligand binding, which may affect the strength and location of hot spots. Therefore, it is important to recall that, based on our experience drawn from the analysis of more than 150 proteins, as described above, for druggability it is necessary to identify appropriate hot spots in ligand-free protein structures, and it is the hot spot properties of the unliganded proteins that provide the most valuable information for establishing druggability. As discussed, the analysis of ligand-bound structures can be useful, however, for understanding the changes in the relative importance of the binding energy hot spots upon ligand binding. In addition, as shown for PTP1B, the analysis of hot spots can also indicate that a target that can bind small ligands with high affinity can nevertheless be undruggable because any such ligands will not be able to satisfy additional requirements for specificity and/or for membrane permeability. We additionally note that, even for targets with good druggability, lead and drug discovery can be challenging when pursued by particular approaches. For example, although high druggability generally correlates with the observation of a high hit rate in fragment-based screening,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> whether the resulting fragment hits will have sufficiently robust binding modes to serve as good starting points for lead discovery depends additionally on how compact or diffuse the main hot spot is and how fully the fragment occupies it.<a onclick="showRef(event, 'ref138'); return false;" href="javascript:void(0);" class="ref ref138">(138)</a> This topic is beyond the scope of the current perspective, however, where we are concerned with whether high-affinity binding can occur rather than with differences between druggable sites in the technical ease or difficulty in identifying a good ligand.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cell Division Protein ZipA</h3><div class="NLM_p">The ZipA/FtsZ protein–protein interaction has been considered to be a potential target for antibacterial agents.<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> We have mapped three different structures of ZipA: unliganded, cocrystallized with a peptide fragment of FtsZ,<a onclick="showRef(event, 'ref139'); return false;" href="javascript:void(0);" class="ref ref139">(139)</a> and cocrystallized with the indoloquinolizin inhibitor DB01967 (<b>1</b>), shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A.<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(140)</a> In all three structures, FTMap finds only three weak hot spots at the protein–peptide interface (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> and <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C). Although ZipA has strong hot spots elsewhere, these are at least 11 Å distant from the interface. Thus, we conclude that the interface between ZipA and FtsZ is not a druggable target. In fact, the <i>K</i><sub>d</sub> of the interaction between ZipA and a 17-mer peptide of FtsZ has been measured to be ∼7 μM,<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a> and efforts to identify and optimize small molecule PPI inhibitors for this system, including HTS of 250 000 compounds, resulted only in inhibitors with <i>K</i><sub>d</sub> ≥ 12 μM.<a onclick="showRef(event, 'ref140 ref142 ref143 ref144'); return false;" href="javascript:void(0);" class="ref ref140 ref142 ref143 ref144">(140, 142-144)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Druggability assessment for important targets. (A) Structure of the weak inhibitor <b>1</b> of the ZipA/FtsZ protein–protein interaction. (B) Structure of the PTP1B inhibitor <b>2</b>. (C) Mapping of the ligand free ZipA protein (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f46">1f46</a>), with inhibitor <b>1</b> (from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1s1s">1s1s</a>) superimposed for reference. The consensus clusters shown are CC8 (green, 6 probe clusters), CC9 (blue, 5 clusters), and CC10 (orange, 3 clusters). We also show the strongest hot spot CC1 (cyan, 22 clusters), which is located outside the ZipA/FtsZ interface. CC4 (salmon, 9 clusters) is also too far from the interface. (D) Mapping of the apurinic/apyrimidinic endonuclease 1 (APE1) structure from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1de9">1de9</a>, cocrystallized with abasic DNA (fragment of the DNA is shown as blue sticks for reference), and an Mn<sup>2+</sup> ion (cyan sphere). As always, the DNA and the ion were removed prior to the mapping. The hot spots shown are CC1 (cyan, 16 clusters), CC2 (magenta, 16 clusters), and CC3 (yellow, 13 clusters). (E) Mapping of the ligand-free protein tyrosine phosphatase 1B (PTP1B, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a>), with two bound phosphotyrosine (pTyr) molecules superimposed for reference (from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a>). The hot spots shown are CC1 (cyan, 20 clusters) at the catalytic PTyr site, CC4 (salmon, 9 clusters), CC7 (orange, 7 clusters), and CC8 (green, 7 clusters) close to the site that binds the noncatalytic pTyr. The hot spots CC2 (magenta, 16 clusters) and CC7 (orange, 7 clusters) are closed in any ligand-bound structure. (F) Same as panel E but with the bound structure of the potent inhibitor <b>2</b> shown for reference from the structure with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1q6t">1q6t</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Detailed Analysis of Important Targets</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">PDB</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">target</th><th class="colsep0 rowsep0" align="center">site(s)</th><th class="colsep0 rowsep0" align="center">U</th><th class="colsep0 rowsep0" align="center">B</th><th class="colsep0 rowsep0" align="center">consensus sites</th><th class="colsep0 rowsep0" align="center">D<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">comment</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">1</td><td class="colsep0 rowsep0" rowspan="3" align="left">ZipA</td><td class="colsep0 rowsep0" rowspan="3" align="left">FtsZ peptide binding site</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f46">1f46</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">8(6), 9(5), 11(3)</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" rowspan="3" align="left">All hot spots in the ZipA:FtsZ interface are very weak. There is a strong hot spots, 1(22) and 2(16), but are more than 10 Å from any hot spot in the interface</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f47">1f47</a></td><td class="colsep0 rowsep0" align="left">5(13), 6(6), 7(3)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1s1s">1s1s</a></td><td class="colsep0 rowsep0" align="left">4(10), 8(4), 10(2)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">2</td><td class="colsep0 rowsep0" rowspan="4" align="left">APE-1</td><td class="colsep0 rowsep0" rowspan="4" align="left">A: PO3 of abasic nucleotide; B: sugar of abasic nucleotide; U: PO3 of two nucl. upstream; Y: at Y128</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bix">1bix</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">A: 1(18), B: 2(16); U: 5(10)</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">For the ensemble formed by CC1 and CC2, MD = 11.6 Å, druggable but weak; CC5 from CC2 = 8 Å, CC5 is closed in DNA bound structures</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e9n">1e9n</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">A: 4(15), B: 2(17), U: 6(4)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1de9">1de9</a></td><td class="colsep0 rowsep0" align="left">A: 1(16), B: 2(16), Y: 3(13)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">With bound DNA and Mn<sup>2+</sup>; CC3 is close to CC1, not present in the DNA-free structures, MD = 14.5 Å</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dew">1dew</a></td><td class="colsep0 rowsep0" align="left">A: 3(14), B: 1(22), Y: 7(5)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">With bound DNA, no Mn<sup>2+</sup>, MD = 14.4 Å</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="10" align="left">3</td><td class="colsep0 rowsep0" rowspan="10" align="left">PTP1B</td><td class="colsep0 rowsep0" rowspan="10" align="left">A site: catalytic pTyr; B site: noncatalytic pTyr; C site: Tyr46, Arg47, and Asp48</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">A: 1(20), 2(16); B: 4(9), 7(7), 8(7)</td><td class="colsep0 rowsep0" rowspan="10" align="left">D</td><td class="colsep0 rowsep0" rowspan="2" align="left">A and B sites are too far (9.0 Å), but A is druggable on its own</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pa1">1pa1</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">A: 1(21), 2(16); B: 8(4), 10(4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a></td><td class="colsep0 rowsep0" align="left">A: 1(13), 7(7); B: 8(7), 9(7)</td><td class="colsep0 rowsep0" align="left">Co-crystallized with 2 pTyr amino acids; A site contracts upon pTtyr binding due to the move of loop 180–184</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2veu">2veu</a></td><td class="colsep0 rowsep0" align="left">A: 1(16), 10(3); B: 3(12), 6(7), 9(4)</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 100 nM, not selective; inhibitor binds only at A and C, A–B = 9.1 Å is too far, MW = 604.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4i8n">4i8n</a></td><td class="colsep0 rowsep0" align="left">A: 1(26); B: 3(10), 8(6)</td><td class="colsep0 rowsep0" align="left">Binds at A and the weaker hot spot of B, not selective; A–B= 6.4 Å, MW = 483.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vew">2vew</a></td><td class="colsep0 rowsep0" align="left">A: 1(25); B: 4(9), 6(7)</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 64 nM, binds A and the weaker hot spot of B and part of C, A–B = 6.6 Å, MW = 584.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2fjn">2fjn</a></td><td class="colsep0 rowsep0" align="left">A: 1(15), 6(9); B: 5(11), 7(7)</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 39 nM, A and one hot spot of B and part of C, A–B= 7.0 Å, MW = 603.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2qbp">2qbp</a></td><td class="colsep0 rowsep0" align="left">A: 1(18), 5(8); B: 4(10), 9(5)</td><td class="colsep0 rowsep0" align="left"><i>K</i><sub>i</sub> = 4 nM, both A and B; both hot spots in B are used, A–B = 7.7 Å, MW = 609.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1q6t">1q6t</a></td><td class="colsep0 rowsep0" align="left">A: 1(20), 10(3); B: 5(11), 9(5)</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 7 nM, pTyr analogue, both A and B and some C, A–B= 7.7 Å, MW = 826.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pxh">1pxh</a></td><td class="colsep0 rowsep0" align="left">A: 1(20), 9(5); B: 5(7), 7(7)</td><td class="colsep0 rowsep0" align="left"><i>K</i><sub>i</sub> = 1.4 nM, binds at A and weaker hot spot of B and C, A–B= 6.6 Å, MW = 873.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">4</td><td class="colsep0 rowsep0" rowspan="3" align="left">14-3-3 protein</td><td class="colsep0 rowsep0" rowspan="3" align="left">S1: pS/T binding; S2: pS/T – 1; S3: pS/T + 1; S4: fusicoccin; S5: pS/T-X-COOH</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yz5">1yz5</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">S1: 1(21); S2: 4(12); S3: 7(5); S4: 2(16), 5(11)</td><td class="colsep0 rowsep0" rowspan="3" align="left">D*<sub>L</sub></td><td class="colsep0 rowsep0" align="left">CC1–CC4 = 8.2 Å, all other hot spots are even farther: CC1–CC2 = 10.7 Å, CC1–CC5 = 9.7 Å</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p1n">3p1n</a></td><td class="colsep0 rowsep0" align="left">S2: 2(14); S3: 1(29); S4: 3(13); S5; 5(10)</td><td class="colsep0 rowsep0" align="left">Bound phosphate group moves the side chain of Arg129 and closes down the pS/T pocket; S3 is strong</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dhu">4dhu</a></td><td class="colsep0 rowsep0" align="left">S2: 4(13), 8(3); S3: 5(11); S4: 1(18), 3(160), 7(5)</td><td class="colsep0 rowsep0" align="left">The small inhibitor places its phosphate group into the pS/T pocket (S1 site), which becomes closed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="10" align="left">5</td><td class="colsep0 rowsep0" rowspan="10" align="left">KRAS</td><td class="colsep0 rowsep0" rowspan="6" align="left">G: GTP/GDP; S1: Site 1; S2: Site 2 (SOS interface); S3: Site 3; S4: Site 4; S5: Site 5; S6: not in Raf or SOS interface</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gft">3gft</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">G: 1(24), 7(6), 9(5); S2: 2(13); S3: 4(9), 7(6); S5: 6(7), 8(6)</td><td class="colsep0 rowsep0" rowspan="10" align="left">B*<sub>S</sub></td><td class="colsep0 rowsep0" align="left">Inhibitor binding site S2 is a weak and isolated hot spot, MD = 7.0 Å</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2pmx">2pmx</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">G: 1(25), 3(11), 7(5); S2: 2(12), S3: 8(5); S5: 6(9)</td><td class="colsep0 rowsep0" align="left">S2 is slightly weaker, but still there</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epr">4epr</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">G: 1(26), 4(10), 5(9); S3: 7(6); S5: 3(11)</td><td class="colsep0 rowsep0" align="left">S2 does not include any hot spot</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dst">4dst</a></td><td class="colsep0 rowsep0" align="left">G: 1(19), 5(6), 8(3); S2: 3(17), S3: 6(5), 7(4); S5: 2(19)</td><td class="colsep0 rowsep0" align="left">S2 stronger, but still isolated, MD = 6.7 Å</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epw">4epw</a></td><td class="colsep0 rowsep0" align="left">G: 2(17), 4(12); S1: 7(5), S2: 1(21), 8(3); S5: 3(14), 5(9)</td><td class="colsep0 rowsep0" align="left">S2 becomes the strongest hot spot region, MD = 10.9 Å, but the secondary hot spot is weak, <i>K</i><sub>d</sub> = 420 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epy">4epy</a></td><td class="colsep0 rowsep0" align="left">G: 2(16), 3(12), 4(10), 5(8), S2: 1(20), 8(5); S3: 10(3); S5: 6(7), 9(4)</td><td class="colsep0 rowsep0" align="left">There are no hot spots for further extension. Adding different groups to the end of the inhibitor, the best <i>K</i><sub>d</sub> value achieved is 190 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">Mapping with masked GTP site, i.e., F site is not available to probes</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gft">3gft</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">S2: 4(14), S6: 1(19), S3: 7(5), S5: 2(18)</td><td class="colsep0 rowsep0" align="left">Strongest site is the one identified by MSCS; S2 site is essentially invariant, remains weak</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dst">4dst</a></td><td class="colsep0 rowsep0" align="left">S2: 2(17), S3: 5(11), 6(9); S5: 1(23)</td><td class="colsep0 rowsep0" align="left">Almost no change relative to free GTP site</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epw">4epw</a></td><td class="colsep0 rowsep0" align="left">S2: 1(17), 4(13); S3: 3(14), S5: 2(15)</td><td class="colsep0 rowsep0" align="left">The secondary site in S2 appears to be stronger, but S2 is still only borderline druggable</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epy">4epy</a></td><td class="colsep0 rowsep0" align="left">S2: 1(16), 3(14); S3: 2(16), 7(9), S5: 4(11)</td><td class="colsep0 rowsep0" align="left">The same as <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epw">4epw</a>, MD = 11.4 Å</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">6</td><td class="colsep0 rowsep0" rowspan="4" align="left">HIV Integ- rase</td><td class="colsep0 rowsep0" rowspan="4" align="left">LEDGF/p75 site</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bi4">1bi4</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1(18), 5(9)</td><td class="colsep0 rowsep0" align="left">D*<sub>S</sub></td><td class="colsep0 rowsep0" align="left">Ligand binding site is a deep pocket, hot spot at the bottom; rather small hot spot ensemble, MD = 9.6 Å, predicted to require charged ligand for potency</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3avn">3avn</a></td><td class="colsep0 rowsep0" align="left">1(20)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Bound to LEDGF peptide; elongated hot spot, but still only MD = 9.2 Å</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2b4j">2b4j</a></td><td class="colsep0 rowsep0" align="left">1(19)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Bound to LEDGF, MD = 8.9 Å</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3lpu">3lpu</a></td><td class="colsep0 rowsep0" align="left">3(8), 9(3)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Inhibitor bound, MD = 9.5 Å, related inhibitor with EC<sub>50</sub> = 69 nM, all ligands are carboxylic acid derivatives</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Druggability class based on FTMap analysis, as described in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</p></div></div></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Apurinic/Apyrimidinic (AP) Endonuclease 1 (APE1)</h3><div class="NLM_p last">APE1 is an essential protein that operates in the base excision repair (BER) pathway and is responsible for ≥95% of the total AP endonuclease activity in human cells. BER is a major pathway that copes with DNA damage induced by several cancer causing agents, including ionizing radiation and Temozolomide. Overexpression of APE1 and enhanced AP endonuclease activity have been linked to increased resistance of tumor cells to treatment with monofunctional alkylators, implicating inhibition of APE1 as a valid strategy for cancer therapy.<a onclick="showRef(event, 'ref145'); return false;" href="javascript:void(0);" class="ref ref145">(145)</a> We analyzed two different unbound structures of APE1 by FTMap (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). The results show two main hot spots at the locations that bind the phosphate group (site A) of the abasic nucleotide, close to the divalent metal ion in the active site, and the sugar ring (site B) of the same nucleotide (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). In ligand-free structures, there is also a weak hot spot (site U) at the phosphate group of the nucleotide two positions upstream from the abasic group, but the site is far (8 Å) from site B and closes down upon DNA binding. The mapping of DNA bound structures reveals another hot spot (site Y), formed by turning the side chain of Tyr128 out of the active site. This site is somewhat stronger when a bound Mn<sup>2+</sup> ion is present (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D). However, the two main hot spots remain weak, and although utilizing the latter hot spot may provide additional potency, there is nothing else near the small active site. Thus, we consider the active site of APE-1 to be druggable, but very weak, close to borderline druggability. In spite of recent hopes,<a onclick="showRef(event, 'ref146'); return false;" href="javascript:void(0);" class="ref ref146">(146)</a> information available in the literature fully confirms this prediction. As described in an excellent review,<a onclick="showRef(event, 'ref147'); return false;" href="javascript:void(0);" class="ref ref147">(147)</a> the reproducibility and biological value of a few potent compounds reported in early studies remain questionable and therefore APE1 was targeted in a number of more recent screens, which collectively returned over 50 compounds. However, even the best inhibitors had only low micromolar IC<sub>50</sub> values. Almost all compounds share a relatively small size, an aromatic hydrophobic central core that most likely binds at site B, and a negative ionizable group, frequently a carboxylate, which is thus predicted to bind at site A and to interact with the metal. On the basis of the weak druggability of the active site. we believe it will be difficult to get much higher potency, and APE1 selectivity may also become an issue.<a onclick="showRef(event, 'ref147'); return false;" href="javascript:void(0);" class="ref ref147">(147)</a> However, it is possible that larger compounds, primarily macrocycles, will be able to take advantage of the additional U and Y sites, improving both potency and selectivity.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Protein Tyrosine Phosphatase 1B (PTP1B)</h3><div class="NLM_p last">PTP1B is a highly validated drug target with a very long history of failures to identify potent and selective inhibitors. The enzyme acts as a negative regulator in insulin signaling pathways. Inhibition of PTP1B activity exhibits potential for enhancing insulin action by prolonging the phosphorylated state of the insulin receptor.<a onclick="showRef(event, 'ref148'); return false;" href="javascript:void(0);" class="ref ref148">(148)</a> A large variety of potent PTP1B inhibitors have been developed, such as <b>2</b> shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B with IC<sub>50</sub> = 7 nM,<a onclick="showRef(event, 'ref149'); return false;" href="javascript:void(0);" class="ref ref149">(149)</a> and it is known that the primary challenges are cell permeability and selectivity.<a onclick="showRef(event, 'ref150'); return false;" href="javascript:void(0);" class="ref ref150">(150)</a> The challenge of permeability is a consequence of the highly polar nature of the active site. Achieving selectivity is difficult because PTP1B and the most highly related tyrosine phosphatase, T-cell PTP (TCPTP), share 72% sequence identity overall and 94% identity for the active site residues. We analyzed two different unbound PTP1B structures by FTMap (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). The results show that top two hot spots in the ligand-free PTP1B structures are located at the catalytic site (A site), with additional, weaker hot spots at a second, noncatalytic site for phosphotyrosine binding (B site) (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> and <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>E). We agree with Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> that the A site is druggable on its own, i.e., it binds potent (albeit charged) ligands. As seen for other ligands with a phosphate group, the structure of PTP1B in complex with phosphotyrosines (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a>) shows that the binding changes the conformation of the loop comprising residues 180–184, which tightens the pocket, but the site can open up to accommodate larger inhibitors. All inhibitors must interact with the charged A site because it is the main source of binding affinity. Attempts have been made to replace the phosphate group with 1,1-dioxido-3-oxoisothiazolidin in site A and expand the inhibitor toward residues Tyr46, Arg47, and Asp 48, defined as the C site, without any strong hot spots. Such inhibitors (e.g., PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2veu">2veu</a>) have IC<sub>50</sub> values between 100 and 2900 nM, but MW > 600 g/mol.<a onclick="showRef(event, 'ref151'); return false;" href="javascript:void(0);" class="ref ref151">(151)</a> Alternatively, a number of inhibitors have been developed that have the phosphate group in site A but that expand into the less polar B site, thereby improving both affinity and cell permeability. To reach even the weaker hot spot at the B site, around 7 Å away from the A site, requires a molecule with MW > 500 g/mol. In addition, the residues that differ between PTP1B and TCPTP (Phe52Tyr and Ala27Ser) lie at the distal end of the B site. Thus, in order to improve selectivity, inhibitors first have to bridge from the A site to the first hot spot of the B site (7.7 Å) and then continue to the second, stronger hot spot of the latter site (another 6 Å). Adding interactions with the C site further improves binding, and such compounds reach an IC<sub>50</sub> of 5 nM and 7-fold selectivity over TCPTP.<a onclick="showRef(event, 'ref149'); return false;" href="javascript:void(0);" class="ref ref149">(149)</a> However, inhibitor <b>2</b> of this type from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1q6t">1q6t</a>,<a onclick="showRef(event, 'ref149'); return false;" href="javascript:void(0);" class="ref ref149">(149)</a> shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B, still has the charged phosphate group and the high MW of 826.76 g/mol. Thus, we conclude that PTP1B is formally druggable, in the sense that very potent inhibitors can be and have been developed. However, the analysis of the hot spot strengths and locations, and the residue differences between PTP1B and its close homologue TCPTP, show that the requirements of high affinity, selectivity, and cell permeability cannot be simultaneously satisfied by an ordinary druglike compound.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 14-3-3 Protein</h3><div class="NLM_p">The 14-3-3 family of phosphoserine/threonine-recognition proteins engage multiple nodes in signaling networks that control diverse physiological and pathophysiological functions and have emerged as promising therapeutic targets for such diseases as cancer and neurodegenerative disorders.<a onclick="showRef(event, 'ref152'); return false;" href="javascript:void(0);" class="ref ref152">(152)</a> 14-3-3 proteins have been cocrystallized with several phosphoserine or phosphothreonine (pS/T) containing peptides.<a onclick="showRef(event, 'ref153'); return false;" href="javascript:void(0);" class="ref ref153">(153)</a><a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A shows such a phosphopeptide, ERα 591–595 (<b>3</b>), with phosphothreonine at position 594, and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B shows the compound named fusicoccin (<b>4</b>), originally identified as a fungal toxin, that stabilizes the binding of <b>3</b>.<a onclick="showRef(event, 'ref154'); return false;" href="javascript:void(0);" class="ref ref154">(154)</a> The 14-3-3 protein forms a weak homodimer, but each monomer can bind <b>3</b> independently in a well-defined elongated site with a deep pS/T binding cavity, together with <b>4</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>D). The protein has been recently cocrystallized with the inhibitor PubChem17588196 (<b>5</b>), shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C.<a onclick="showRef(event, 'ref155'); return false;" href="javascript:void(0);" class="ref ref155">(155)</a><a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>E shows the resulting complex.<a onclick="showRef(event, 'ref155'); return false;" href="javascript:void(0);" class="ref ref155">(155)</a> Although the IC<sub>50</sub> value of <b>5</b> is only 5 μM, it was shown to disrupt the interaction of 14-3-3 with aminopeptidase N in a cellular assay.<a onclick="showRef(event, 'ref155'); return false;" href="javascript:void(0);" class="ref ref155">(155)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Druggability assessment for 14-3-3 protein. (A) Structure of the phosphothreonine peptide <b>3</b> that was cocystallyzed with 14-3-3. (B) Structure of fusicoccin (<b>4</b>), a compound that binds to 14-3-3 and stabilizes the binding of <b>3</b>. (C) Structure of inhibitor <b>5</b>. (D) Mapping of the ligand free structure of 14-3-3 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yz5">1yz5</a>). For reference we superimpose the bound compounds <b>3</b> and <b>4</b>, shown as green and yellow sticks, respectively, from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jdd">4jdd</a>. The hot spots shown are CC1 (cyan, 21 clusters) at the deep pocket that binds <b>3</b>, CC2 (magenta, 16 clusters), and CC5 (white, 11 clusters) at the region that binds <b>4</b>. Two more hot spots are close to <b>3</b>: CC4 (salmon, 12 clusters) and CC7 (orange, 5 clusters). (E) Same as panel D but with inhibitor <b>5</b> (magenta).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As shown in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>D, the strongest hot spot of the ligand-free 14-3-3 protein is in the pS/T binding pocket, with additional hot spots, both at the side chains adjacent to the bound <b>3</b>, and in other regions in the binding site. The binding of the phosphate is clearly an important part of the interactions with peptides and small inhibitors. However, the binding of the phosphate group, both in the peptides and the inhibitors, attracts the side chain of Arg129 into the pS/T binding pocket, eliminating the hot spot at that location (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). This example emphasizes that the analysis of ligand-free protein structure, which here predicts the site to be druggable, provides useful druggability information, whereas restricting consideration to pocket properties in ligand-bound structures can be misleading when the protein binds charged ligands. Indeed, the inhibitors are located at the strongest hot spot, CC1, of the unliganded protein, rather than those present in the ligand-bound structure (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>E). The hot spot closest to CC1 is CC5, but even that distance is 8.2 Å; all other hot spots are even farther and are not used by the small inhibitor. However, the S2 and S3 sites at the pS/T – 1 and pS/T + 1 positions, respectively, remain relatively strong in all structures, suggesting that higher potency but larger inhibitors can most likely be found.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> KRAS</h3><div class="NLM_p">The Ras (rat sarcoma) protein family members are all low-molecular-weight GTP-binding proteins that play a role in regulating cell differentiation, proliferation, and survival. KRAS, a key member of the Ras family, is an attractive cancer target, as frequent point mutations in the KRAS gene render the protein constitutively active.<a onclick="showRef(event, 'ref156'); return false;" href="javascript:void(0);" class="ref ref156">(156)</a> However, even though the structure and function of RAS have been studied intensively for nearly 30 years, it has remained elusive as a drug target.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The most obvious target site would be the deep pocket where GTP and GDP bind, but these nucleotides bind with nanomolar affinity<a onclick="showRef(event, 'ref157'); return false;" href="javascript:void(0);" class="ref ref157">(157)</a> and are present at very high concentrations in cells. Therefore, we mapped the protein both with the GTP/GDP site free and also with the site masked to prevent probe binding (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). There have been many attempts to find alternative sites to target, with recent successes by two groups, who reported small molecules binding to a pocket in the surface of KRAS that interacts with the SOS protein,<a onclick="showRef(event, 'ref136 ref158'); return false;" href="javascript:void(0);" class="ref ref136 ref158">(136, 158)</a> referred to as Site 2.<a onclick="showRef(event, 'ref159'); return false;" href="javascript:void(0);" class="ref ref159">(159)</a> Studying KRAS is complicated by the fact that the protein acts as a molecular switch with complex dynamics between multiple conformational states.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> We analyzed three different unbound structures by FTMap. As shown in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>, in mapping KRAS the GTP/GDP site dominates, if the site is not masked, with several strong hot spots at this location. In two of the three unbound KRAS structures, Site 2 in the KRAS/SOS interface had the second strongest hot spots. However, the hot spot has at most 13 probe clusters and has a small size (MD = 7.0 Å); hence, on the basis of the FTMap criteria, this site is classified as B*<sub>S</sub>. Nevertheless, the KRAS/SOS interface has been targeted in recent campaigns, and two groups independently identified series of druglike inhibitors binding at Site 2,<a onclick="showRef(event, 'ref136 ref158'); return false;" href="javascript:void(0);" class="ref ref136 ref158">(136, 158)</a> such as CHEMBL2086796 (<b>6</b>)<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A. To show the hot spot in the unliganded KRAS structure, we superimpose a small ligand, benzamidine (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C). As the size of ligands increases, the number of probes in the main hot spot increases to 20, and second hot spot also opens in the structure cocrystallized with <b>6</b>, but it is very weak and includes at most 5 probe clusters (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D). In addition, the maximum dimension of the hot spot region increases to 10.9 Å, and on the basis of these values, the site would be considered druggable. However, as our analyses of other targets shows, the mapping of the ligand-free structure usually provides more valid assessment of druggability that the mapping of the ligand-bound state, and on the basis of the former, the interface is only borderline druggable (B*<sub>S</sub>). This prediction is supported by the fact that, in spite of major efforts, the best-known noncovalent inhibitor binding at this site has a <i>K</i><sub>d</sub> value of 190 μM and is positively charged at neutral pH.<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> The hot spots are even weaker if the GTP/GDP site is masked (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>), so the weakness of the hot spots at Site 2 is not due to the binding of probes elsewhere.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Druggability assessment for KRAS. (A) Structure of <b>6</b>, one of the highest affinity inhibitors binding to KRAS at the interface with SOS. (B) Structure of inhibitor <b>7</b> binding covalently to KRAS(G12C). (C) Mapping of the ligand-free KRAS (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gft">3gft</a>). The cavities with probes bound are the GTP/GDP binding site, with part of CC1 (cyan, 24 clusters) visible, and the targeted hot spot CC2 (magenta, 13 clusters) in Site 2 located in the KRAS/SOS interface. For reference, we superimpose a small ligand, benzamidine (shown as yellow sticks, from the structure with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dso">4dso</a>), binding to Site 2. (D) Mapping of the KRAS structure from its complex with inhibitor <b>6</b> from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epy">4epy</a> (<i>K</i><sub>d</sub> = 340 μM, superimposed as magenta sticks). Due to the binding of <b>6</b>, the strongest hot spot CC1 (cyan, 20 clusters) is now in the KRAS/SOS interface, and the interface includes a second weak hot spot CC8 (green, 7 clusters) located close to CC1. (E) The mapping of the ligand-free KRAS (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gft">3gft</a>) places CC4 (salmon, 9 clusters) at the S3 site, in the pocket binding the KRAS(G12C) inhibitor <b>7</b> that allosterically controls GTP affinity and effector interactions. Inhibitor <b>7</b> from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4lv6">4lv6</a> is superimposed for reference and is shown as green sticks. Notice that in this closed conformation of KRAS the loop 58–65 would clash with <b>7</b> and thus prevents its binding. For reference, the figure also shows CC2 (magenta) and the benzamidine ligand (yellow) on the other side of the protein. (F) Homology model of KRAS based on a HRAS structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4q21">4q21</a>) to model the open state. In this structure, the opening of the 58–65 loop creates a stronger hot spot, CC2 (magenta, 16 clusters). Inhibitor <b>7</b> from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4lv6">4lv6</a> is shown again for reference. Although the site in this open structure binds 16 probe clusters, the maximum dimension of CC2 is only 5.1 Å; thus, the site is not druggable without the covalent interaction with G12C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We note that the mapping of the ligand-free KRAS structure places CC4 (salmon, 9 clusters) at the S3 site, in the pocket that binds the recently developed covalent KRAS(G12C) inhibitors such as SCHEMBL15337743 (<b>7</b>), shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B.<a onclick="showRef(event, 'ref160'); return false;" href="javascript:void(0);" class="ref ref160">(160)</a> In this closed conformation of KRAS, the loop 58–65 would clash with the allosteric inhibitor <b>7</b> and thus prevent binding (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>E). In spite of this unfavorable geometry, FTMap places a hot spot at the allosteric site. To show that the opening of the site would allow for the binding of <b>7</b>, we constructed a homology model of the KRAS open state based on an open structure of HRAS. <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>F shows the mapping of the resulting structure with <b>7</b> superimposed. As expected, the opening of the 58–65 loop creates a stronger hot spot, CC2 (magenta, 16 clusters). However, the maximum dimension of this CC2 is only 5.1 Å; thus, the site is not druggable without covalent protein–ligand interaction.<a onclick="showRef(event, 'ref160'); return false;" href="javascript:void(0);" class="ref ref160">(160)</a></div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> HIV Integrase, LEDGF/p75 Interaction Site</h3><div class="NLM_p last">Integration of HIV-1 DNA is required to maintain the viral DNA in the infected cell; hence, the integrase (IN) is an attractive target for HIV treatment. Two different mechanisms have been explored for IN inhibition. The design and discovery of IN inhibitors were first focused at targeting the catalytic site of IN, with the goal of achieving an effect on strand transfer.<a onclick="showRef(event, 'ref161'); return false;" href="javascript:void(0);" class="ref ref161">(161)</a> Cheng et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> considered this site to be undruggable, whereas, based on FTMap, we predicted it to be druggable (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">Table S3</a>). In fact, since the publication of the work by Cheng et al.,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> several compounds targeting the active site, called integrase strand transfer inhibitors (INSTIs), have been approved.<a onclick="showRef(event, 'ref162'); return false;" href="javascript:void(0);" class="ref ref162">(162)</a> Since INSTI resistance evolves readily, a proposed alternative approach is to disrupt the interaction between IN and the transcriptional coactivator protein LEDGF/p75, which results in inhibition through an allosteric mechanism.<a onclick="showRef(event, 'ref161 ref162'); return false;" href="javascript:void(0);" class="ref ref161 ref162">(161, 162)</a> According to FTMap, the interface region of IN has a strong hot spot, but the hot spot ensemble is small (MD = 9.6 Å) and thus we classify the site as non-canonically druggable–small (D*<sub>S</sub>), requiring charged inhibitors. In agreement with this prediction, the recently discovered potent inhibitors called LEDGINs include a carboxyl group.<a onclick="showRef(event, 'ref163'); return false;" href="javascript:void(0);" class="ref ref163">(163)</a> An example of this class is (2<i>S</i>)-2-(6-chloro-2-methyl-4-phenylquinolin-3-yl)pentanoic acid (PubChem45281242),<a onclick="showRef(event, 'ref161'); return false;" href="javascript:void(0);" class="ref ref161">(161)</a> which binds to IN with EC<sub>50</sub> = 69 nM. Note that the binding of the charged ligands reduces the hot spot, similarly to the 14-3-3 protein, again emphasizing that the ligand-free structure is likely to provide more accurate druggability prediction.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48028" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48028" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We show here that the druggability of binding sites on proteins can be evaluated with good reliability based on fragment hit rates, whether obtained experimentally (e.g., by NMR) or computationally using the FTMap algorithm, and establish clear FTMap-based criteria for druggability. Importantly, because the FTMap method is based on the biophysics of binding rather than on empirical parametrization to a particular training set of protein–drug complexes, meaningful information can be gained about classes of proteins and classes of compounds beyond those resembling validated targets and conventionally druglike ligands. In particular, the information provided by the mapping identifies targets that, while not druggable by druglike ligands, may become druggable using pharmaceutically attractive compound classes such as macrocycles or other large molecules that fall outside conventional definitions of druglikeness such as the rule-of-five. Application of the method to ZipA and PTP-1B show why these targets are not druggable. In contrast, results for other targets of high current biomedical interest, including apurinic/apyrimidinic endonuclease 1 (APE1), the SOS interaction site of KRAS, the phosphoserine/threonine-recognition protein 14-3-3, and the LEDGF/p75 interaction site of HIV integrase, show that, in these cases, it may be necessary to consider ligands lying outside conventional druglike chemical space and provide guidance as to what compound classes are most likely to succeed. Overall, due to the progress that has been made in recent years across multiple research groups and institutions, we believe that evaluation of protein druggability has entered an era in which, for the majority of cases, the prospects for identifying a pharmaceutically useful small molecule ligand can now be ascertained with good confidence based only on the X-ray crystal structure of the unliganded protein. These advances substantially lower the technical risk associated with launching a drug discovery project involving a novel protein target and provide valuable information concerning which approaches and chemotypes are most likely to succeed against a given site.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.5b00586">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37803" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37803" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b00586" class="ext-link">10.1021/acs.jmedchem.5b00586</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional druggability results and histograms (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf" class="ext-link">PDF</a>).</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf">jm5b00586_si_001.pdf (821.81 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b00586" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66632" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66632" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dima Kozakov</span> - <span class="hlFld-Affiliation affiliation">Department
of Applied Mathematics & Statistics, Stony Brook University, Stony
Brook, New York 11794, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a4c9cdc0c5d7e4c6d18ac1c0d1"><span class="__cf_email__" data-cfemail="6b06020f0a182b091e450e0f1e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adrian Whitty</span> - <span class="hlFld-Affiliation affiliation">Department
of Applied Mathematics & Statistics, Stony Brook University, Stony
Brook, New York 11794, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Acpharis
Inc., Holliston, Massachusetts 01746, United States</span>; 
    <span class="hlFld-Affiliation affiliation">§Department of Chemistry and ∥Department of
Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f0879899848489b09285de959485"><span class="__cf_email__" data-cfemail="89fee1e0fdfdf0c9ebfca7ecedfc">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sandor Vajda</span> - <span class="hlFld-Affiliation affiliation">Department
of Applied Mathematics & Statistics, Stony Brook University, Stony
Brook, New York 11794, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Acpharis
Inc., Holliston, Massachusetts 01746, United States</span>; 
    <span class="hlFld-Affiliation affiliation">§Department of Chemistry and ∥Department of
Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c4b2a5aea0a584a6b1eaa1a0b1"><span class="__cf_email__" data-cfemail="dfa9beb5bbbe9fbdaaf1babbaa">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David R. Hall</span> - <span class="hlFld-Affiliation affiliation">Acpharis
Inc., Holliston, Massachusetts 01746, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raeanne L. Napoleon</span> - <span class="hlFld-Affiliation affiliation">Department
of Applied Mathematics & Statistics, Stony Brook University, Stony
Brook, New York 11794, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Acpharis
Inc., Holliston, Massachusetts 01746, United States</span>; 
    <span class="hlFld-Affiliation affiliation">§Department of Chemistry and ∥Department of
Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christine Yueh</span> - <span class="hlFld-Affiliation affiliation">Department
of Applied Mathematics & Statistics, Stony Brook University, Stony
Brook, New York 11794, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Acpharis
Inc., Holliston, Massachusetts 01746, United States</span>; 
    <span class="hlFld-Affiliation affiliation">§Department of Chemistry and ∥Department of
Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): D.R.H. is a full-time employee of Acpharis, Inc. The company offers software similar to the FTMap program that was used in this perspective. D.K. and S.V. own Acpharis stock. However, the FTMap software and server are free for use.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20244" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20244" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Dima Kozakov</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=BIO-d414e5261-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Dima Kozakov</b> received an M.S. in Applied Mathematics and Physics from the Moscow Institute of Physics and Technology and a Ph.D. in Biomedical Engineering from Boston University. Currently, he is Assistant Professor of Applied Mathematics & Statistics at Stony Brook University. Dr. Kozakov has been working on the development of algorithms and software for molecular modeling and drug design. He is the primary architect of ClusPro, Version 2.0, protein docking server, currently the best automatic server according to latest round of evaluations of the blind protein docking assessment CAPRI. ClusPro has more than 10 000 users and has run more than 100 000 jobs in the last 3 years. Dr. Kozakov is also the primary developer of the FTMap approach, which was used in this work. The FTMap server currently has more than 2000 users.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">David R. Hall</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=BIO-d414e5266-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>David R. Hall</b> received a Ph.D. in Biomedical Engineering from Boston University in 2012. Since 2012, he has led Acpharis, a computer-aided drug design company specializing in computational solvent mapping and protein–protein docking. In this role, he has led a long-term collaboration with Biogen on evaluating the binding sites of proteins using computational solvent mapping.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Raeanne L. Napoleon</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=BIO-d414e5271-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Raeanne L. Napoleon</b> received a Ph.D. in Theoretical Physical Chemistry from Boston University in 2012. She received her B.S. (<i>cum laude</i>) from the University of the Sciences in Philadelphia, PA, in 2005. She is currently Assistant Professor of Chemistry at Santa Barbara City College, where she focuses on curriculum development in STEM education, with an emphasis on both chemistry and nanoscience disciplines, and promoting women in STEM. She also continues to work on computational simulations of protein systems and interfacial condensed phase simulations.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Christine Yueh</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=BIO-d414e5279-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Christine Yueh</b> received a B.S. in Chemical Engineering from UC Santa Barbara in 2010. She is currently a Ph.D. candidate in the Biomedical Engineering program at Boston University, where her research involves using computational modeling to study protein–protein and protein–small ligand interactions.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Adrian Whitty</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=BIO-d414e5284-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Adrian Whitty</b> received a B.Sc. in Chemistry from King’s College London in 1985, followed by M.S. and Ph.D. degrees in Chemistry from the University of Illinois at Chicago. After completing postdoctoral work with William P. Jencks FRS at Brandeis University, he joined the biopharmaceutical company Biogen, where he worked in the Department of Drug Discovery, rising to the position of Director of Physical Biochemistry. In 2008, he moved to Boston University, where he is an Associate Professor in the Department of Chemistry. His research interests include protein–protein and protein–ligand binding, especially as they are related to the discovery of small molecule inhibitors of protein–protein interactions, and also the quantitative, mechanistic analysis of growth factor receptor activation and signaling.</p></figure></div><div class="bio" rid="ath6"><figure id="ath6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Sandor Vajda</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=BIO-d414e5289-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Sandor Vajda</b> received an M.S. in Applied Mathematics and Ph.D. in chemistry in Budapest, Hungary. He held research positions in the Department of Engineering, University of Warwick, England, and the Department of Chemistry, Princeton University. He was a faculty member in Hungary and at the Mount Sinai School of Medicine, New York, NY. Currently, he is Professor of Biomedical Engineering and Chemistry at Boston University. He also directs the Biomolecular Engineering Research Center at BU. Dr. Vajda has been active in method development for modeling biological macromolecules, with emphasis on molecular interactions and drug design. In 2004, he and his graduate students founded SolMap Pharmaceuticals, a start-up company focusing on fragment-based drug design. SolMap was acquired by Forma Therapeutics, Cambridge, MA, in 2008.</p></figure></div><div class="ack" id="ACK-d414e5310-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54542" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54542" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This investigation was supported by grant nos. GM064700 to S.V. and GM094551 to A.W. and S.V. from the National Institute of General Medical Sciences.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i36" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i36"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i37" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i37"> Abbreviations Used</h2><tr><td class="NLM_term">PTP-1B</td><td class="NLM_def"><p class="first last">protein tyrosine phosphatase-1B</p></td></tr><tr><td class="NLM_term">APE1</td><td class="NLM_def"><p class="first last">apurinic/apyrimidinic endonuclease 1</p></td></tr><tr><td class="NLM_term">SOS</td><td class="NLM_def"><p class="first last">son of sevenless</p></td></tr><tr><td class="NLM_term">LEDGF/p75</td><td class="NLM_def"><p class="first last">lens epithelium-derived growth factor</p></td></tr><tr><td class="NLM_term">KRAS</td><td class="NLM_def"><p class="first last">Kirsten rat sarcoma viral oncogene homologue</p></td></tr><tr><td class="NLM_term">MSCS</td><td class="NLM_def"><p class="first last">multiple solvent crystal structures</p></td></tr><tr><td class="NLM_term">CC</td><td class="NLM_def"><p class="first last">consensus cluster</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">CD</td><td class="NLM_def"><p class="first last">center-to-center distance</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">maximum distance</p></td></tr><tr><td class="NLM_term">DS</td><td class="NLM_def"><p class="first last">druggability score</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10665" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10665" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 163 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-30&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0liS9L3lnveCIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D30%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lizlKavH7ZOnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Russ, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lampel, S.</span><span> </span><span class="NLM_article-title">The druggable genome: An update</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1607</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03666-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS1359-6446%2805%2903666-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16376820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A280%3ADC%252BD2Mnos1KntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=1607-10&author=A.+P.+Russauthor=S.+Lampel&title=The+druggable+genome%3A+An+update&doi=10.1016%2FS1359-6446%2805%2903666-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The druggable genome: an update</span></div><div class="casAuthors">Russ Andreas P; Lampel Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">1607-10</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Annotating the druggable genome estimates the potential maximum size of the playing field for current small-molecule drug design but It does not consider biologicals or future breakthroughs in medicinal chemistry or biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTZBV0v0n_yXIMcDoIzZtNkfW6udTcc2ea4GOCvuxf4-bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2Mnos1KntQ%253D%253D&md5=dc4dbf7d5220b5642cca6152b8c37d09</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903666-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903666-4%26sid%3Dliteratum%253Aachs%26aulast%3DRuss%26aufirst%3DA.%2BP.%26aulast%3DLampel%26aufirst%3DS.%26atitle%3DThe%2520druggable%2520genome%253A%2520An%2520update%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D1607%26epage%3D10%26doi%3D10.1016%2FS1359-6446%2805%2903666-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Johns, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russ, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H. A.</span><span> </span><span class="NLM_article-title">Current drug targets and the druggable genome</span> <span class="citation_source-journal">Chemical Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_fpage">320</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1017/CBO9781139021500.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1017%2FCBO9781139021500.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=320-32&author=M.+A.+Johnsauthor=A.+Russauthor=H.+A.+Fu&title=Current+drug+targets+and+the+druggable+genome&doi=10.1017%2FCBO9781139021500.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1017%2FCBO9781139021500.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FCBO9781139021500.029%26sid%3Dliteratum%253Aachs%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DRuss%26aufirst%3DA.%26aulast%3DFu%26aufirst%3DH.%2BA.%26atitle%3DCurrent%2520drug%2520targets%2520and%2520the%2520druggable%2520genome%26jtitle%3DChemical%2520Genomics%26date%3D2012%26spage%3D320%26epage%3D32%26doi%3D10.1017%2FCBO9781139021500.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiorov, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornell, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Y. D.</span><span> </span><span class="NLM_article-title">Drug-like density: A method of quantifying the ″bindability″ of a protein target based on a very large set of pockets and drug-like ligands from the Protein Data Bank</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2029</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span><span class="refDoi"> DOI: 10.1021/ci100312t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100312t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtleiur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=2029-40&author=R.+P.+Sheridanauthor=V.+N.+Maiorovauthor=M.+K.+Hollowayauthor=W.+D.+Cornellauthor=Y.+D.+Gao&title=Drug-like+density%3A+A+method+of+quantifying+the+%E2%80%B3bindability%E2%80%B3+of+a+protein+target+based+on+a+very+large+set+of+pockets+and+drug-like+ligands+from+the+Protein+Data+Bank&doi=10.1021%2Fci100312t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-like Density: A Method of Quantifying the "Bindability" of a Protein Target Based on a Very Large Set of Pockets and Drug-like Ligands from the Protein Data Bank</span></div><div class="casAuthors">Sheridan, Robert P.; Maiorov, Vladimir N.; Holloway, M. Katharine; Cornell, Wendy D.; Gao, Ying-Duo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2029-2040</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One approach to estg. the "chem. tractability" of a candidate protein target where we know the at. resoln. structure is to examine the phys. properties of potential binding sites.  A no. of other workers have addressed this issue.  We characterize ∼290 000 "pockets" from ∼42 000 protein crystal structures in terms of a three parameter "pocket space": vol., buriedness, and hydrophobicity.  A metric DLID (drug-like d.) measures how likely a pocket is to bind a drug-like mol.  This is calcd. from the count of other pockets in its local neighborhood in pocket space that contain drug-like cocrystd. ligands and the count of total pockets in the neighborhood.  Surprisingly, despite being defined locally, a global trend in DLID can be predicted by a simple linear regression on log(vol.), buriedness, and hydrophobicity.  Two levels of simplification are necessary to relate the DLID of individual pockets to "targets": taking the best DLID per Protein Data Bank (PDB) entry (because any given crystal structure can have many pockets), and taking the median DLID over all PDB entries for the same target (because different crystal structures of the same protein can vary because of artifacts and real conformational changes).  We can show that median DLIDs for targets that are detectably homologous in sequence are reasonably similar and that median DLIDs correlate with the "druggability" est. of Cheng et al. (Nature Biotechnol.2007, 25, 71-75).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY1WHjyA8uarVg90H21EOLACvtfcHk0lizlKavH7ZOnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtleiur3O&md5=c7fef497000a2830323d623c19c107f9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fci100312t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100312t%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DMaiorov%26aufirst%3DV.%2BN.%26aulast%3DHolloway%26aufirst%3DM.%2BK.%26aulast%3DCornell%26aufirst%3DW.%2BD.%26aulast%3DGao%26aufirst%3DY.%2BD.%26atitle%3DDrug-like%2520density%253A%2520A%2520method%2520of%2520quantifying%2520the%2520%25E2%2580%25B3bindability%25E2%2580%25B3%2520of%2520a%2520protein%2520target%2520based%2520on%2520a%2520very%2520large%2520set%2520of%2520pockets%2520and%2520drug-like%2520ligands%2520from%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D2029%26epage%3D40%26doi%3D10.1021%2Fci100312t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span> </span><span class="NLM_article-title">Predicting protein druggability</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1675</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03624-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS1359-6446%2805%2903624-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16376828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCrtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=1675-82&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=C.+Tse&title=Predicting+protein+druggability&doi=10.1016%2FS1359-6446%2805%2903624-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting protein druggability</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Tse, Christin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">23/24</span>),
    <span class="NLM_cas:pages">1675-1682</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  New exptl. and computational approaches promise to rapidly and reliably identify druggable, disease-modifying targets and thereby streamline the drug discovery process.  The ability to predict whether a particular protein can bind with high affinity and specificity to small, drug-like compds. based solely on its 3D structure has been a longstanding goal of structural biologists and computational scientists.  The promise is that an accurate prediction of protein druggability can capitalize on the huge investments already made in structural genomics initiatives by identifying highly druggable proteins and using this information in target identification and validation campaigns.  Here we discuss the potential utility of tools that characterize protein targets and describe strategies for the optimal integration of protein druggability data with bioinformatic approaches to target selection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodx4xzYCqOWLVg90H21EOLACvtfcHk0ljBp_RRIa6TaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCrtrnN&md5=217b1b92ddb845506b14bded4fb56549</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903624-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903624-X%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DTse%26aufirst%3DC.%26atitle%3DPredicting%2520protein%2520druggability%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D1675%26epage%3D82%26doi%3D10.1016%2FS1359-6446%2805%2903624-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from NMR-based screening data</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2518</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-25&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+NMR-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0ljBp_RRIa6TaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520NMR-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D25%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Cheng, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smyth, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soulard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caffrey, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzberg, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, E. S.</span><span> </span><span class="NLM_article-title">Structure-based maximal affinity model predicts small-molecule druggability</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1038/nbt1273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnbt1273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17211405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1Grtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=71-5&author=A.+C.+Chengauthor=R.+G.+Colemanauthor=K.+T.+Smythauthor=Q.+Caoauthor=P.+Soulardauthor=D.+R.+Caffreyauthor=A.+C.+Salzbergauthor=E.+S.+Huang&title=Structure-based+maximal+affinity+model+predicts+small-molecule+druggability&doi=10.1038%2Fnbt1273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based maximal affinity model predicts small-molecule druggability</span></div><div class="casAuthors">Cheng, Alan C.; Coleman, Ryan G.; Smyth, Kathleen T.; Cao, Qing; Soulard, Patricia; Caffrey, Daniel R.; Salzberg, Anna C.; Huang, Enoch S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-75</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lead generation is a major hurdle in small-mol. drug discovery, with an estd. 60% of projects failing from lack of lead matter or difficulty in optimizing leads for drug-like properties.  It would be valuable to identify these less-druggable targets before incurring substantial expenditure and effort.  Here we show that a model-based approach using basic biophys. principles yields good prediction of druggability based solely on the crystal structure of the target binding site.  We quant. est. the maximal affinity achievable by a drug-like mol., and we show that these calcd. values correlate with drug discovery outcomes.  We exptl. test two predictions using high-throughput screening of a diverse compd. collection.  The collective results highlight the utility of our approach as well as strategies for tackling difficult targets.  Both assays are simple, suitable for rapid screening of a large no. of medicinal libraries, and amenable for automation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomQJn2NNwCFLVg90H21EOLACvtfcHk0linbynZ4JyVTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1Grtg%253D%253D&md5=e971faf87bf4b91602bf592b9296325b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt1273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1273%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DA.%2BC.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSmyth%26aufirst%3DK.%2BT.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DSoulard%26aufirst%3DP.%26aulast%3DCaffrey%26aufirst%3DD.%2BR.%26aulast%3DSalzberg%26aufirst%3DA.%2BC.%26aulast%3DHuang%26aufirst%3DE.%2BS.%26atitle%3DStructure-based%2520maximal%2520affinity%2520model%2520predicts%2520small-molecule%2520druggability%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D71%26epage%3D5%26doi%3D10.1038%2Fnbt1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Seco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability index from first principles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1021/jm801385d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801385d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2363-71&author=J.+Secoauthor=F.+J.+Luqueauthor=X.+Barril&title=Binding+site+detection+and+druggability+index+from+first+principles&doi=10.1021%2Fjm801385d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Detection and Druggability Index from First Principles</span></div><div class="casAuthors">Seco, Jesus; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In drug discovery, it is essential to identify binding sites on protein surfaces that drug-like mols. could exploit to exert a biol. effect.  Both X-ray crystallog. and NMR expts. have demonstrated that org. solvents bind precisely at these locations.  We show that this effect is reproduced using mol. dynamics with a binary solvent.  Furthermore, anal. of the simulations give direct access to interaction free energies between the protein and small org. mols., which can be used to detect binding sites and to predict the maximal affinity that a drug-like mol. could attain for them.  On a set of pharmacol. relevant proteins, we obtain good predictions for druggable sites as well as for protein-protein and low affinity binding sites.  This is the first druggability index not based on surface descriptors and, being independent of a training set, is particularly indicated to study unconventional targets such as protein-protein interactions or allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAgMI9Qer6rVg90H21EOLACvtfcHk0linbynZ4JyVTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D&md5=b766cbadeba11b23a1de44b27aa794b4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm801385d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801385d%26sid%3Dliteratum%253Aachs%26aulast%3DSeco%26aufirst%3DJ.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520index%2520from%2520first%2520principles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2363%26epage%3D71%26doi%3D10.1021%2Fjm801385d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-89&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0linbynZ4JyVTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D89%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Schmidtke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Understanding and predicting druggability. A high-throughput method for detection of drug binding sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5858</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1021/jm100574m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100574m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ektLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5858-67&author=P.+Schmidtkeauthor=X.+Barril&title=Understanding+and+predicting+druggability.+A+high-throughput+method+for+detection+of+drug+binding+sites&doi=10.1021%2Fjm100574m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding and Predicting Druggability. A High-Throughput Method for Detection of Drug Binding Sites</span></div><div class="casAuthors">Schmidtke, Peter; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5858-5867</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Druggability predictions are important to avoid intractable targets and to focus drug discovery efforts on sites offering better prospects.  However, few druggability prediction tools have been released and none has been extensively tested.  Here, a set of druggable and nondruggable cavities has been compiled in a collaborative platform (http://fpocket.sourceforge.net/dcd) that can be used, contributed, and curated by the community.  Druggable binding sites are often oversimplified as closed, hydrophobic cavities, but data set anal. reveals that polar groups in druggable binding sites have properties that enable them to play a decisive role in ligand recognition.  Finally, the data set has been used in conjunction with the open source fpocket suite to train and validate a logistic model.  State of the art performance was achieved for predicting druggability on known binding sites and on virtual screening expts. where druggable pockets are retrieved from a pool of decoys.  The algorithm is free, extremely fast, and can effectively be used to automatically sieve through massive collections of structures (http://fpocket.sourceforge.net).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos1ixAU42dJrVg90H21EOLACvtfcHk0lgAkp3TktpLmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ektLo%253D&md5=d9fc5c4d52a6e0f032b8d5c021a6cd89</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm100574m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100574m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidtke%26aufirst%3DP.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DUnderstanding%2520and%2520predicting%2520druggability.%2520A%2520high-throughput%2520method%2520for%2520detection%2520of%2520drug%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5858%26epage%3D67%26doi%3D10.1021%2Fjm100574m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Krasowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muthas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenk, R.</span><span> </span><span class="NLM_article-title">Drugpred: A structure-based approach to predict protein druggability developed using an extensive nonredundant data set</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2829</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span><span class="refDoi"> DOI: 10.1021/ci200266d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200266d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlShtLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2829-42&author=A.+Krasowskiauthor=D.+Muthasauthor=A.+Sarkarauthor=S.+Schmittauthor=R.+Brenk&title=Drugpred%3A+A+structure-based+approach+to+predict+protein+druggability+developed+using+an+extensive+nonredundant+data+set&doi=10.1021%2Fci200266d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DrugPred: A Structure-Based Approach To Predict Protein Druggability Developed Using an Extensive Nonredundant Data Set</span></div><div class="casAuthors">Krasowski, Agata; Muthas, Daniel; Sarkar, Aurijit; Schmitt, Stefan; Brenk, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2829-2842</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Judging if a protein is able to bind orally available mols. with high affinity, i.e. if a protein is druggable, is an important step in target assessment.  In order to derive a structure-based method to predict protein druggability, a comprehensive, nonredundant data set contg. crystal structures of 71 druggable and 44 less druggable proteins was compiled by literature search and data mining.  This data set was subsequently used to train a structure-based druggability predictor (DrugPred) using partial least-squares projection to latent structures discriminant anal. (PLS-DA).  DrugPred performed well in discriminating druggable from less druggable binding sites for both internal and external predictions.  The method is robust against conformational changes in the binding site and outperforms previously published methods.  The superior performance of DrugPred is likely due to the size and compn. of the training set which, in contrast to most previously developed methods, only contains cavities that have evolved to bind a natural ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLISVIZ5UnXrVg90H21EOLACvtfcHk0lgAkp3TktpLmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlShtLnN&md5=a352186705196d6c58b4a590e2848896</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci200266d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200266d%26sid%3Dliteratum%253Aachs%26aulast%3DKrasowski%26aufirst%3DA.%26aulast%3DMuthas%26aufirst%3DD.%26aulast%3DSarkar%26aufirst%3DA.%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DBrenk%26aufirst%3DR.%26atitle%3DDrugpred%253A%2520A%2520structure-based%2520approach%2520to%2520predict%2520protein%2520druggability%2520developed%2520using%2520an%2520extensive%2520nonredundant%2520data%2520set%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2829%26epage%3D42%26doi%3D10.1021%2Fci200266d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Fauman, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, E. S.</span><span> </span><span class="NLM_article-title">Structure-based druggability assessment - identifying suitable targets for small molecule therapeutics</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2011.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.cbpa.2011.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21704549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFCltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2011&pages=463-8&author=E.+B.+Faumanauthor=B.+K.+Raiauthor=E.+S.+Huang&title=Structure-based+druggability+assessment+-+identifying+suitable+targets+for+small+molecule+therapeutics&doi=10.1016%2Fj.cbpa.2011.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based druggability assessment - identifying suitable targets for small molecule therapeutics</span></div><div class="casAuthors">Fauman, Eric B.; Rai, Brajesh K.; Huang, Enoch S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">463-468</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A target is druggable if it can be modulated in vivo by a drug-like mol.  The general properties of oral drugs are summarized by the rule of 5' which specifies parameters related to size and lipophilicity.  Structure-based target druggability assessment consists of predicting ligand-binding sites on the protein that are complementary to these drug-like properties.  Automated identification of ligand-binding sites can use geometrical considerations alone or include specific physicochem. properties of the protein surface.  Features of a pocket's size and shape, together with measures of its hydrophobicity, are most informative in identifying suitable drug-binding pockets.  The recent availability of several validation sets of druggable vs. undruggable targets has helped fuel the development of more elaborate methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzQbgkbe44IrVg90H21EOLACvtfcHk0lgAkp3TktpLmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFCltrw%253D&md5=876a5083e4ae685d55eee3295e924cfd</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2011.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2011.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DFauman%26aufirst%3DE.%2BB.%26aulast%3DRai%26aufirst%3DB.%2BK.%26aulast%3DHuang%26aufirst%3DE.%2BS.%26atitle%3DStructure-based%2520druggability%2520assessment%2520-%2520identifying%2520suitable%2520targets%2520for%2520small%2520molecule%2520therapeutics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D15%26spage%3D463%26epage%3D8%26doi%3D10.1016%2Fj.cbpa.2011.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Dogsitescorer: A web server for automatic binding site prediction, analysis and druggability assessment</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">2074</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1093/bioinformatics/bts310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fbioinformatics%2Fbts310" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22628523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFSlsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=2074-5&author=A.+Volkamerauthor=D.+Kuhnauthor=F.+Rippmannauthor=M.+Rarey&title=Dogsitescorer%3A+A+web+server+for+automatic+binding+site+prediction%2C+analysis+and+druggability+assessment&doi=10.1093%2Fbioinformatics%2Fbts310"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">DoGSiteScorer: a web server for automatic binding site prediction, analysis and druggability assessment</span></div><div class="casAuthors">Volkamer, Andrea; Kuhn, Daniel; Rippmann, Friedrich; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2074-2075</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Many drug discovery projects fail because the underlying target is finally found to be undruggable.  Progress in structure elucidation of proteins now opens up a route to automatic structure-based target assessment.  DoGSiteScorer is a newly developed automatic tool combining pocket prediction, characterization and druggability estn. and is now available through a web server.  Availability: The DoGSiteScorer web server is freely available for academic use at http://dogsite.zbh.uni-hamburg.de Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="2c5e4d5e49556c564e440259424501444d414e595e4b02484902">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN-lCqMEbql7Vg90H21EOLACvtfcHk0lj0JQS61AWBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFSlsbzE&md5=747f225234540d2a009fcb0d5b077d40</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts310%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DDogsitescorer%253A%2520A%2520web%2520server%2520for%2520automatic%2520binding%2520site%2520prediction%252C%2520analysis%2520and%2520druggability%2520assessment%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3D2074%26epage%3D5%26doi%3D10.1093%2Fbioinformatics%2Fbts310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Nisius, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sha, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span> </span><span class="NLM_article-title">Structure-based computational analysis of protein binding sites for function and druggability prediction</span> <span class="citation_source-journal">J. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1016/j.jbiotec.2011.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jbiotec.2011.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22197384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2012&pages=123-34&author=B.+Nisiusauthor=F.+Shaauthor=H.+Gohlke&title=Structure-based+computational+analysis+of+protein+binding+sites+for+function+and+druggability+prediction&doi=10.1016%2Fj.jbiotec.2011.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based computational analysis of protein binding sites for function and druggability prediction</span></div><div class="casAuthors">Nisius, Britta; Sha, Fan; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">123-134</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein binding sites are the places where mol. interactions occur.  Thus, the anal. of protein binding sites is of crucial importance to understand the biol. processes proteins are involved in.  Herein, we focus on the computational anal. of protein binding sites and present structure-based methods that enable function prediction for orphan proteins and prediction of target druggability.  We present the general ideas behind these methods, with a special emphasis on the scopes and limitations of these methods and their validation.  Addnl., we present some successful applications of computational binding site anal. to emphasize the practical importance of these methods for biotechnol./bioeconomy and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXzgcXRf62cbVg90H21EOLACvtfcHk0lj0JQS61AWBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahurs%253D&md5=9d8c5a3a41abd1ce0444ed23c3b3f7a9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2011.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2011.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DNisius%26aufirst%3DB.%26aulast%3DSha%26aufirst%3DF.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DStructure-based%2520computational%2520analysis%2520of%2520protein%2520binding%2520sites%2520for%2520function%2520and%2520druggability%2520prediction%26jtitle%3DJ.%2520Biotechnol.%26date%3D2012%26volume%3D159%26spage%3D123%26epage%3D34%26doi%3D10.1016%2Fj.jbiotec.2011.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Perola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, J.</span><span> </span><span class="NLM_article-title">Development of a rule-based method for the assessment of protein druggability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1027</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1021/ci200613b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200613b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XksVyls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1027-38&author=E.+Perolaauthor=L.+Hermanauthor=J.+Weiss&title=Development+of+a+rule-based+method+for+the+assessment+of+protein+druggability&doi=10.1021%2Fci200613b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Rule-Based Method for the Assessment of Protein Druggability</span></div><div class="casAuthors">Perola, Emanuele; Herman, Lee; Weiss, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1027-1038</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Target selection is a crit. step in the majority of modern drug discovery programs.  The viability of a drug target depends on two components: biol. relevance and chem. tractability.  The concept of druggability was introduced to describe the second component, and it is defined as the ability of a target to bind a drug-like mol. with a therapeutically useful level of affinity.  To investigate the rules that govern druggability, we developed an algorithm to isolate and characterize the binding pockets of protein targets.  Using this algorithm, we performed a comparative anal. between the relevant pockets of 60 targets of approved drugs and a diverse set of 440 ligand-binding pockets.  As a result, we defined a preferred property space for druggable pockets based on five key properties (vol., depth, enclosure, percentage of charged residues and hydrophobicity), and we represented it with a set of simple rules.  These rules may be applicable in the future to evaluate the chem. tractability of prospective targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaRGoeAUMXUbVg90H21EOLACvtfcHk0lj0JQS61AWBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksVyls7w%253D&md5=49a4c07150749f6d09a062365512b8c2</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci200613b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200613b%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DHerman%26aufirst%3DL.%26aulast%3DWeiss%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520a%2520rule-based%2520method%2520for%2520the%2520assessment%2520of%2520protein%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1027%26epage%3D38%26doi%3D10.1021%2Fci200613b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grombacher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Combining global and local measures for structure-based druggability predictions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">360</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1021/ci200454v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200454v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFylurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=360-72&author=A.+Volkamerauthor=D.+Kuhnauthor=T.+Grombacherauthor=F.+Rippmannauthor=M.+Rarey&title=Combining+global+and+local+measures+for+structure-based+druggability+predictions&doi=10.1021%2Fci200454v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Global and Local Measures for Structure-Based Druggability Predictions</span></div><div class="casAuthors">Volkamer, Andrea; Kuhn, Daniel; Grombacher, Thomas; Rippmann, Friedrich; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">360-372</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting druggability and prioritizing certain disease modifying targets for the drug development process is of high practical relevance in pharmaceutical research.  DoGSiteScorer is a fully automatic algorithm for pocket and druggability prediction.  Besides consideration of global properties of the pocket, also local similarities shared between pockets are reflected.  Druggability scores are predicted by a support vector machine (SVM), trained, and tested on the druggability data set (DD) and its nonredundant version (NRDD).  The DD consists of 1069 targets with assigned druggable, difficult, and undruggable classes.  In 90% of the NRDD, the SVM model based on global descriptors correctly classifies a target as either druggable or undruggable.  Nevertheless, global properties suffer from binding site changes due to ligand binding and from the pocket boundary definition.  Therefore, local pocket properties are addnl. investigated in terms of a nearest neighbor search.  Local similarities are described by distance dependent histograms between atom pairs.  In 88% of the DD pocket set, the nearest neighbor and the structure itself conform with their druggability type.  A discriminant feature between druggable and undruggable pockets is having less short-range hydrophilic-hydrophilic pairs and more short-range lipophilic-lipophilic pairs.  Our findings for global pocket descriptors coincide with previously published methods affirming that size, shape, and hydrophobicity are important global pocket descriptors for automatic druggability prediction.  Nevertheless, the variety of pocket shapes and their flexibility upon ligand binding limit the automatic projection of druggable features onto descriptors.  Incorporating local pocket properties is another step toward a reliable descriptor-based druggability prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRnTVZG_4qNLVg90H21EOLACvtfcHk0lg0Paom6y5TFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFylurrP&md5=ec276048e78e1297610e3d4b756ab6e8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci200454v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200454v%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DGrombacher%26aufirst%3DT.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DCombining%2520global%2520and%2520local%2520measures%2520for%2520structure-based%2520druggability%2520predictions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D360%26epage%3D72%26doi%3D10.1021%2Fci200454v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Driggers, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hale, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terrett, N. K.</span><span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery--an underexploited structural class</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span><span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-24&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery%2D%2Dan+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lg0Paom6y5TFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery--an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D24%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Wessjohann, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruijter, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Rivera, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, W.</span><span> </span><span class="NLM_article-title">What can a chemist learn from nature’s macrocycles?--A brief, conceptual view</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span><span class="refDoi"> DOI: 10.1007/s11030-005-1314-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs11030-005-1314-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15789564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslels7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=171-86&author=L.+A.+Wessjohannauthor=E.+Ruijterauthor=D.+Garcia-Riveraauthor=W.+Brandt&title=What+can+a+chemist+learn+from+nature%E2%80%99s+macrocycles%3F%2D%2DA+brief%2C+conceptual+view&doi=10.1007%2Fs11030-005-1314-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">What can a chemist learn from nature's macrocycles? - A brief, conceptual view</span></div><div class="casAuthors">Wessjohann, Ludger A.; Ruijter, Eelco; Garcia-Rivera, Daniel; Brandt, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">171-186</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Macrocyclic natural products often display remarkable biol. activities, and many of these compds. (or their derivs.) are used as drugs.  The chem. diversity of these compds. is immense and may provide inspiration for innovative drug design.  Therefore, a database of naturally occurring macrocycles was analyzed for ring size, mol. wt. distribution, and the frequency of some common substructural motifs.  The underlying principles of the chem. diversity are reviewed in terms of biosynthetic origin and nature's strategies for diversity and complexity generation in relation to the structural diversity and similarities found in the macrocycle database.  Finally, it is suggested that synthetic chemists should use not only nature's mols., but also nature's strategies as a source of inspiration.  To illustrate this, the biosynthesis of macrocycles by non-ribosomal peptide synthetases and terpene and polyketide cyclases, as well as recent advances of these strategies in an integrated synthesis/biotechnol. approach are briefly reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVB4g1QplcWLVg90H21EOLACvtfcHk0lg0Paom6y5TFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslels7w%253D&md5=88df3ca095b446e3781369708ccddfd2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs11030-005-1314-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-005-1314-x%26sid%3Dliteratum%253Aachs%26aulast%3DWessjohann%26aufirst%3DL.%2BA.%26aulast%3DRuijter%26aufirst%3DE.%26aulast%3DGarcia-Rivera%26aufirst%3DD.%26aulast%3DBrandt%26aufirst%3DW.%26atitle%3DWhat%2520can%2520a%2520chemist%2520learn%2520from%2520nature%25E2%2580%2599s%2520macrocycles%253F--A%2520brief%252C%2520conceptual%2520view%26jtitle%3DMol.%2520Diversity%26date%3D2005%26volume%3D9%26spage%3D171%26epage%3D86%26doi%3D10.1007%2Fs11030-005-1314-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Marsault, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. L.</span><span> </span><span class="NLM_article-title">Macrocycles are great cycles: Applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1961</span><span class="NLM_x">–</span> <span class="NLM_lpage">2004</span><span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+Applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lg0Paom6y5TFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520Applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Mallinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1409</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-38&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lg3U2MJppDK8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D38%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Kalgutkar, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvie, D. K.</span><span> </span><span class="NLM_article-title">Drug discovery for a new generation of covalent drugs</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1517/17460441.2012.688744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1517%2F17460441.2012.688744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22607458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFWmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=561-81&author=A.+S.+Kalgutkarauthor=D.+K.+Dalvie&title=Drug+discovery+for+a+new+generation+of+covalent+drugs&doi=10.1517%2F17460441.2012.688744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery for a new generation of covalent drugs</span></div><div class="casAuthors">Kalgutkar, Amit S.; Dalvie, Deepak K.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">561-581</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The design of target-specific covalent inhibitors is conceptually attractive because of increased biochem. efficiency through covalency and increased duration of action that outlasts the pharmacokinetics of the agent.  Although many covalent inhibitors have been approved or are in advanced clin. trials to treat indications such as cancer and hepatitis C, there is a general tendency to avoid them as drug candidates because of concerns regarding immune-mediated toxicity that can arise from indiscriminate reactivity with off-target proteins.Areas covered: The review examines potential reason(s) for the excellent safety record of marketed covalent agents and advanced clin. candidates for emerging therapeutic targets.  A significant emphasis is placed on proteomic techniques and chem./biochem. reactivity assays that aim to provide a systematic rank ordering of pharmacol. selectivity relative to off-target protein reactivity of covalent inhibitors.Expert opinion: While tactics to examine selective covalent modification of the pharmacol. target are broadly applicable in drug discovery, it is unclear whether the output from such studies can prospectively predict idiosyncratic immune-mediated drug toxicity.  Opinions regarding an acceptable threshold of protein reactivity/body burden for a toxic electrophile and a non-toxic electrophilic covalent drug have not been defined.  Increasing confidence in proteomic and chem./biochem. reactivity screens will require a retrospective side-by-side profiling of marketed covalent drugs and electrophiles known to cause deleterious toxic effects via non-selective covalent binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowmXy20mDTPbVg90H21EOLACvtfcHk0lg3U2MJppDK8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFWmt7k%253D&md5=b7fc8e60b263dda5765dd8380f52df17</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.688744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.688744%26sid%3Dliteratum%253Aachs%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DDalvie%26aufirst%3DD.%2BK.%26atitle%3DDrug%2520discovery%2520for%2520a%2520new%2520generation%2520of%2520covalent%2520drugs%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D561%26epage%3D81%26doi%3D10.1517%2F17460441.2012.688744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-17&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0lg3U2MJppDK8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D17%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Way, J. C.</span><span> </span><span class="NLM_article-title">Covalent modification as a strategy to block protein-protein interactions with small-molecule drugs</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1016/S1367-5931(99)00049-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS1367-5931%2899%2900049-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10679371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtlygtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2000&pages=40-6&author=J.+C.+Way&title=Covalent+modification+as+a+strategy+to+block+protein-protein+interactions+with+small-molecule+drugs&doi=10.1016%2FS1367-5931%2899%2900049-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent modification as a strategy to block protein-protein interactions with small-molecule drugs</span></div><div class="casAuthors">Way, Jeffrey C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-46</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 42 refs.  It is generally difficult to block protein-protein interactions with small-mol. drugs.  A novel pharmaceutical development strategy to block protein interactions is emerging: targeted covalent modification to sterically block interactions.  By this approach, compds. first interact non-covalently with a specific target protein.  This interaction juxtaposes a weakly reactive group of the drug with a target amino acid sidechain, which then react by virtue of their high local concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAxQTnoYgzeLVg90H21EOLACvtfcHk0ljf3fduMjE1zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtlygtrs%253D&md5=b5b65c074e838ddc2410435cb2f2eefc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2899%2900049-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252899%252900049-6%26sid%3Dliteratum%253Aachs%26aulast%3DWay%26aufirst%3DJ.%2BC.%26atitle%3DCovalent%2520modification%2520as%2520a%2520strategy%2520to%2520block%2520protein-protein%2520interactions%2520with%2520small-molecule%2520drugs%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2000%26volume%3D4%26spage%3D40%26epage%3D6%26doi%3D10.1016%2FS1367-5931%2899%2900049-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Edwards, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, A. J.</span><span> </span><span class="NLM_article-title">Helix-mediated protein--protein interactions as targets for intervention using foldamers</span> <span class="citation_source-journal">Amino Acids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">743</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span><span class="refDoi"> DOI: 10.1007/s00726-011-0880-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs00726-011-0880-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2011&pages=743-54&author=T.+A.+Edwardsauthor=A.+J.+Wilson&title=Helix-mediated+protein%2D%2Dprotein+interactions+as+targets+for+intervention+using+foldamers&doi=10.1007%2Fs00726-011-0880-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs00726-011-0880-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-011-0880-8%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DT.%2BA.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DHelix-mediated%2520protein--protein%2520interactions%2520as%2520targets%2520for%2520intervention%2520using%2520foldamers%26jtitle%3DAmino%2520Acids%26date%3D2011%26volume%3D41%26spage%3D743%26epage%3D54%26doi%3D10.1007%2Fs00726-011-0880-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Murray, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellman, S. H.</span><span> </span><span class="NLM_article-title">Targeting protein-protein interactions: Lessons from p53/MDM2</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">657</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span><span class="refDoi"> DOI: 10.1002/bip.20741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2Fbip.20741" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2007&pages=657-86&author=J.+K.+Murrayauthor=S.+H.+Gellman&title=Targeting+protein-protein+interactions%3A+Lessons+from+p53%2FMDM2&doi=10.1002%2Fbip.20741"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fbip.20741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20741%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DTargeting%2520protein-protein%2520interactions%253A%2520Lessons%2520from%2520p53%252FMDM2%26jtitle%3DBiopolymers%26date%3D2007%26volume%3D88%26spage%3D657%26epage%3D86%26doi%3D10.1002%2Fbip.20741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Raj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arora, P. S.</span><span> </span><span class="NLM_article-title">Plucking the high hanging fruit: A systematic approach for targeting protein-protein interactions</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">4051</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.bmc.2012.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmc.2012.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23267671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2rsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4051-7&author=M.+Rajauthor=B.+N.+Bullockauthor=P.+S.+Arora&title=Plucking+the+high+hanging+fruit%3A+A+systematic+approach+for+targeting+protein-protein+interactions&doi=10.1016%2Fj.bmc.2012.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Plucking the high hanging fruit: A systematic approach for targeting protein-protein interactions</span></div><div class="casAuthors">Raj, Monika; Bullock, Brooke N.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4051-4057</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Development of specific ligands for protein targets that help decode the complexities of protein-protein interaction networks is a key goal for the field of chem. biol.  Despite the emergence of powerful in silico and exptl. high-throughput screening strategies, the discovery of synthetic ligands that selectively modulate protein-protein interactions remains a challenge for bioorg. and medicinal chemists.  This Perspective discusses emerging principles for the rational design of PPI inhibitors.  Fundamentally, the approach seeks to adapt nature's protein recognition principles for the design of suitable secondary structure mimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorpTCgOTASB7Vg90H21EOLACvtfcHk0ljf3fduMjE1zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2rsrbN&md5=970cdca83336f20387e060810f7f261c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DRaj%26aufirst%3DM.%26aulast%3DBullock%26aufirst%3DB.%2BN.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DPlucking%2520the%2520high%2520hanging%2520fruit%253A%2520A%2520systematic%2520approach%2520for%2520targeting%2520protein-protein%2520interactions%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4051%26epage%3D7%26doi%3D10.1016%2Fj.bmc.2012.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Walensky, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malia, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbuto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span> </span><span class="NLM_article-title">Activation of apoptosis in vivo by a hydrocarbon-stapled BH3 helix</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">305</span><span class="NLM_x">, </span> <span class="NLM_fpage">1466</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1126/science.1099191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1126%2Fscience.1099191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15353804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFart7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2004&pages=1466-70&author=L.+D.+Walenskyauthor=A.+L.+Kungauthor=I.+Escherauthor=T.+J.+Maliaauthor=S.+Barbutoauthor=R.+D.+Wrightauthor=G.+Wagnerauthor=G.+L.+Verdineauthor=S.+J.+Korsmeyer&title=Activation+of+apoptosis+in+vivo+by+a+hydrocarbon-stapled+BH3+helix&doi=10.1126%2Fscience.1099191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Apoptosis in Vivo by a Hydrocarbon-Stapled BH3 Helix</span></div><div class="casAuthors">Walensky, Loren D.; Kung, Andrew L.; Escher, Iris; Malia, Thomas J.; Barbuto, Scott; Wright, Renee D.; Wagner, Gerhard; Verdine, Gregory L.; Korsmeyer, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">305</span>
        (<span class="NLM_cas:issue">5689</span>),
    <span class="NLM_cas:pages">1466-1470</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">BCL-2 family proteins constitute a crit. control point for the regulation of apoptosis.  Protein interaction between BCL-2 members is a prominent mechanism of control and is mediated through the amphipathic α-helical BH3 segment, an essential death domain.  We used a chem. strategy, termed hydrocarbon stapling, to generate BH3 peptides with improved pharmacol. properties.  The stapled peptides, called "stabilized alpha-helix of BCL-2 domains" (SAHBs), proved to be helical, protease-resistant, and cell-permeable mols. that bound with increased affinity to multidomain BCL-2 member pockets.  A SAHB of the BH3 domain from the BID protein specifically activated the apoptotic pathway to kill leukemia cells.  In addn., SAHB effectively inhibited the growth of human leukemia xenografts in vivo.  Hydrocarbon stapling of native peptides may provide a useful strategy for exptl. and therapeutic modulation of protein-protein interactions in many signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpfOpXrd8FVbVg90H21EOLACvtfcHk0ljf3fduMjE1zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFart7c%253D&md5=53fca0524df7a584ccf9d0991bc1bb64</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099191%26sid%3Dliteratum%253Aachs%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DEscher%26aufirst%3DI.%26aulast%3DMalia%26aufirst%3DT.%2BJ.%26aulast%3DBarbuto%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DR.%2BD.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DActivation%2520of%2520apoptosis%2520in%2520vivo%2520by%2520a%2520hydrocarbon-stapled%2520BH3%2520helix%26jtitle%3DScience%26date%3D2004%26volume%3D305%26spage%3D1466%26epage%3D70%26doi%3D10.1126%2Fscience.1099191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Brown, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Effects of conformational dynamics on predicted protein druggability</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">70</span><span class="NLM_x">–</span> <span class="NLM_lpage">2</span><span class="refDoi"> DOI: 10.1002/cmdc.200500013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2Fcmdc.200500013" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=70-2&author=S.+P.+Brownauthor=P.+J.+Hajduk&title=Effects+of+conformational+dynamics+on+predicted+protein+druggability&doi=10.1002%2Fcmdc.200500013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200500013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200500013%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DS.%2BP.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DEffects%2520of%2520conformational%2520dynamics%2520on%2520predicted%2520protein%2520druggability%26jtitle%3DChemMedChem%26date%3D2006%26volume%3D1%26spage%3D70%26epage%3D2%26doi%3D10.1002%2Fcmdc.200500013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">A hot spot of binding energy in a hormone-receptor interface</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1126/science.7529940</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1126%2Fscience.7529940" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=7529940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK2MXjt12nsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1995&pages=383-6&author=T.+Clacksonauthor=J.+A.+Wells&title=A+hot+spot+of+binding+energy+in+a+hormone-receptor+interface&doi=10.1126%2Fscience.7529940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A hot spot of binding energy in a hormone-receptor interface</span></div><div class="casAuthors">Clackson, Tim; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5196</span>),
    <span class="NLM_cas:pages">383-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The x-ray crystal structure of the complex between human growth hormone (hGH) and the extracellular domain of its first bound receptor (hGHbp) shows that about 30 side chains from each protein make contact.  Individual replacement of contact residues in the hGHbp with alanine showed that a central hydrophobic region, dominated by two tryptophan residues, accounts for more than three-quarters of the binding free energy.  This "functional epitope" is surrounded by less important contact residues that are generally hydrophilic and partially hydrated, so that the interface resembles a cross section through a globular protein.  The functionally important residues on the hGHbp directly contact those on hGH.  Thus, only a small and complementary set of contact residues maintains binding affinity, a property that may be general to protein-protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4AKI2EsvEQLVg90H21EOLACvtfcHk0lhxHRSpgHZswA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjt12nsbs%253D&md5=62b214455d644fd07afb2817d1c482fe</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fscience.7529940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7529940%26sid%3Dliteratum%253Aachs%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DA%2520hot%2520spot%2520of%2520binding%2520energy%2520in%2520a%2520hormone-receptor%2520interface%26jtitle%3DScience%26date%3D1995%26volume%3D267%26spage%3D383%26epage%3D6%26doi%3D10.1126%2Fscience.7529940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">DeLano, W. L.</span><span> </span><span class="NLM_article-title">Unraveling hot spots in binding interfaces: Progress and challenges</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1016/S0959-440X(02)00283-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0959-440X%2802%2900283-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11839484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=14-20&author=W.+L.+DeLano&title=Unraveling+hot+spots+in+binding+interfaces%3A+Progress+and+challenges&doi=10.1016%2FS0959-440X%2802%2900283-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling hot spots in binding interfaces: progress and challenges</span></div><div class="casAuthors">DeLano, Warren L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-20</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 49 refs.  Protein interface hot spots, as revealed by alanine scanning mutagenesis, continue to stimulate interest in the biophys. basis of mol. recognition.  Although these regions apparently constitute fertile grounds for intermol. interactions, no general algorithm has yet been developed that can predict hot spots based solely on their shape or compn.  The discovery of structural plasticity in hot spot regions indicates that dynamic simulation techniques may be essential for achieving a predictive understanding of binding interface energetics.  Future progress will depend as much on the application of new computational approaches for dissecting protein interfaces as on expanding the empirical databank of mutagenic substitutions and their effects.  Despite current theor. shortcomings, recent methodol. advances provide efficient exptl. means of probing hot spots and enable immediate applications for hot spots in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk1HNI-4QfrrVg90H21EOLACvtfcHk0lhxHRSpgHZswA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D&md5=a297ae443e69c94b7cf2e342337f29ea</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2802%2900283-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252802%252900283-X%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26atitle%3DUnraveling%2520hot%2520spots%2520in%2520binding%2520interfaces%253A%2520Progress%2520and%2520challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D12%26spage%3D14%26epage%3D20%26doi%3D10.1016%2FS0959-440X%2802%2900283-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Keskin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span> </span><span class="NLM_article-title">Hot regions in protein-protein interactions: The organization and contribution of structurally conserved hot spot residues</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">345</span><span class="NLM_x">, </span> <span class="NLM_fpage">1281</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span><span class="refDoi"> DOI: 10.1016/j.jmb.2004.10.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jmb.2004.10.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15644221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXis1ahsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2005&pages=1281-94&author=O.+Keskinauthor=B.+Y.+Maauthor=R.+Nussinov&title=Hot+regions+in+protein-protein+interactions%3A+The+organization+and+contribution+of+structurally+conserved+hot+spot+residues&doi=10.1016%2Fj.jmb.2004.10.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Hot Regions in Protein-Protein Interactions: The Organization and Contribution of Structurally Conserved Hot Spot Residues</span></div><div class="casAuthors">Keskin, Ozlem; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1281-1294</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Structurally conserved residues at protein-protein interfaces correlate with the exptl. alanine-scanning hot spots.  Here, we investigate the organization of these conserved, computational hot spots and their contribution to the stability of protein assocns.  We find that computational hot spots are not homogeneously distributed along the protein interfaces; rather, they are clustered within locally tightly packed regions.  Within the dense clusters, they form a network of interactions, and, consequently, their contributions to the stability of the complex are cooperative; however the contributions of independent clusters are additive.  This suggests that the binding free energy is not a simple summation of the single hot spot residue contributions.  As expected, around the hot spot residues, we observe moderately conserved residues, further highlighting the crucial role of the conserved interactions in the local densely packed environment.  The conserved occurrence of these organizations suggests that they are advantageous for protein-protein assocns.  Interestingly, the total no. of hydrogen bonds and salt bridges contributed by hot spots is as expected.  Thus, H-bond forming residues may use a "hot spot for water exclusion" mechanism.  Since conserved residues are located within highly packed regions, water mols. are easily removed upon binding, strengthening electrostatic contributions of charge-charge interactions.  Hence, the picture that emerges is that protein-protein assocns. are optimized locally, with the clustered, networked, highly packed structurally conserved residues contributing dominantly and cooperatively to the stability of the complex.  When addressing the crucial question of "what are the preferred ways of proteins to assoc.", these findings point toward a crit. involvement of hot regions in protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1ETiLA5TuabVg90H21EOLACvtfcHk0lhxHRSpgHZswA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXis1ahsw%253D%253D&md5=8adbf29ef0563f429415c67fd209c896</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.10.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.10.077%26sid%3Dliteratum%253Aachs%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DMa%26aufirst%3DB.%2BY.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DHot%2520regions%2520in%2520protein-protein%2520interactions%253A%2520The%2520organization%2520and%2520contribution%2520of%2520structurally%2520conserved%2520hot%2520spot%2520residues%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D345%26spage%3D1281%26epage%3D94%26doi%3D10.1016%2Fj.jmb.2004.10.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Bogan, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorn, K. S.</span><span> </span><span class="NLM_article-title">Anatomy of hot spots in protein interfaces</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1006/jmbi.1998.1843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1006%2Fjmbi.1998.1843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=9653027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=1998&pages=1-9&author=A.+A.+Boganauthor=K.+S.+Thorn&title=Anatomy+of+hot+spots+in+protein+interfaces&doi=10.1006%2Fjmbi.1998.1843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomy of hot spots in protein interfaces</span></div><div class="casAuthors">Bogan A A; Thorn K S</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Binding of one protein to another is involved in nearly all biological functions, yet the principles governing the interaction of proteins are not fully understood.  To analyze the contributions of individual amino acid residues in protein-protein binding we have compiled a database of 2325 alanine mutants for which the change in free energy of binding upon mutation to alanine has been measured (available at http://motorhead. ucsf.edu/thorn/hotspot).  Our analysis shows that at the level of side-chains there is little correlation between buried surface area and free energy of binding.  We find that the free energy of binding is not evenly distributed across interfaces; instead, there are hot spots of binding energy made up of a small subset of residues in the dimer interface.  These hot spots are enriched in tryptophan, tyrosine and arginine, and are surrounded by energetically less important residues that most likely serve to occlude bulk solvent from the hot spot.  Occlusion of solvent is found to be a necessary condition for highly energetic interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZMrH3BevWP3ELvajU9pjPfW6udTcc2ebss50mSOszt7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D&md5=e3ef3c857d1fe070deff25dc54d09587</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.1843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.1843%26sid%3Dliteratum%253Aachs%26aulast%3DBogan%26aufirst%3DA.%2BA.%26aulast%3DThorn%26aufirst%3DK.%2BS.%26atitle%3DAnatomy%2520of%2520hot%2520spots%2520in%2520protein%2520interfaces%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1998%26volume%3D280%26spage%3D1%26epage%3D9%26doi%3D10.1006%2Fjmbi.1998.1843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Kortemme, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, D.</span><span> </span><span class="NLM_article-title">A simple physical model for binding energy hot spots in protein-protein complexes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">14116</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1073/pnas.202485799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.202485799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=12381794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=14116-21&author=T.+Kortemmeauthor=D.+Baker&title=A+simple+physical+model+for+binding+energy+hot+spots+in+protein-protein+complexes&doi=10.1073%2Fpnas.202485799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A simple physical model for binding energy hot spots in protein-protein complexes</span></div><div class="casAuthors">Kortemme, Tanja; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">14116-14121</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein-protein recognition plays a central role in most biol. processes.  Although the structures of many protein-protein complexes have been solved in mol. detail, general rules describing affinity and selectivity of protein-protein interactions do not accurately account for the extremely diverse nature of the interfaces.  We investigate the extent to which a simple phys. model can account for the wide range of exptl. measured free energy changes brought about by alanine mutation at protein-protein interfaces.  The model successfully predicts the results of alanine scanning expts. on globular proteins (743 mutations) and 19 protein-protein interfaces (233 mutations) with av. unsigned errors of 0.81 kcal/mol and 1.06 kcal/mol, resp.  The results test our understanding of the dominant contributions to the free energy of protein-protein interactions, can guide expts. aimed at the design of protein interaction inhibitors, and provide a stepping-stone to important applications such as interface redesign.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1GbqIE1dkgrVg90H21EOLACvtfcHk0lhEUhmiP09Lqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2it70%253D&md5=f5e5fb01e6e803efb2ad1922cb517271</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202485799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202485799%26sid%3Dliteratum%253Aachs%26aulast%3DKortemme%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DA%2520simple%2520physical%2520model%2520for%2520binding%2520energy%2520hot%2520spots%2520in%2520protein-protein%2520complexes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D14116%26epage%3D21%26doi%3D10.1073%2Fpnas.202485799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Locating and characterizing binding sites on proteins</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1038/nbt0596-595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnbt0596-595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=9630949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK28Xis1eksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=595-9&author=C.+Mattosauthor=D.+Ringe&title=Locating+and+characterizing+binding+sites+on+proteins&doi=10.1038%2Fnbt0596-595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Locating and characterizing binding sites of proteins</span></div><div class="casAuthors">Mattos, Carla; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">595-9</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">A review with 38 refs.  This review article begins with a discussion of fundamental differences between substrates and inhibitors, and some of the assumptions and goals underlying the design of a new ligand to a target protein.  An overview is given of the methods currently used to locate and characterize ligand binding sites on protein surfaces, with focus on a novel approach: multiple solvent crystal structures (MSCS).  In this method, the x-ray crystal structure of the target protein is solved in a variety or org. solvents.  Each type of solvent mol. serves as a probe for complementary binding sites on the protein.  The probe distribution on the protein surface allows the location of binding sites and the characterization of the potential ligand interactions within these sites.  General aspects of the application of the MSCS method to porcine pancreatic elastase is discussed, and comparison of the results with those from x-ray crystal structures of elastase/inhibitor complexes is used to illustrate the potential of the method in aiding the process of rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDEKZ8sM6gk7Vg90H21EOLACvtfcHk0lhEUhmiP09Lqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis1eksr0%253D&md5=4facd2f240311e8937623352e7d933d3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnbt0596-595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0596-595%26sid%3Dliteratum%253Aachs%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DLocating%2520and%2520characterizing%2520binding%2520sites%2520on%2520proteins%26jtitle%3DNat.%2520Biotechnol.%26date%3D1996%26volume%3D14%26spage%3D595%26epage%3D9%26doi%3D10.1038%2Fnbt0596-595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellamacina, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, X. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffery, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">An experimental approach to mapping the binding surfaces of crystalline proteins</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">2605</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span><span class="refDoi"> DOI: 10.1021/jp952516o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp952516o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK28Xmt1GrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1996&pages=2605-11&author=K.+N.+Allenauthor=C.+R.+Bellamacinaauthor=X.+C.+Dingauthor=C.+J.+Jefferyauthor=C.+Mattosauthor=G.+A.+Petskoauthor=D.+Ringe&title=An+experimental+approach+to+mapping+the+binding+surfaces+of+crystalline+proteins&doi=10.1021%2Fjp952516o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">An Experimental Approach to Mapping the Binding Surfaces of Crystalline Proteins</span></div><div class="casAuthors">Allen, Karen N.; Bellamacina, Cornelia R.; Ding, Xiaochun; Jeffery, Constance J.; Mattos, Carla; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2605-11</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Porcine pancreatic elastase has been used as the model enzyme in the design and development of a crystallog. method that allows mapping of the binding surface of a protein by solving its crystal structure in a variety of org. solvents.  The ultimate goal of this method is to aid in the process of drug design, where each of the chosen org. mols. represents a given functional group in a larger inhibitor mol.  This method of multiple solvent crystal structures (MSCS) has a theor. counterpart in the method of multiply copy simultaneous search (MCSS) (Miranker, A.; Karplus, M. Proteins: Struct., Funct., Genet. 1991, 11, 29-34) and is the first exptl. method that can be used as a check to the theory.  The MSCS method is presented here with acetonitrile as the probe org. solvent.  The procedure involved does not cause significant changes in the structure of elastase as compared to the structure in aq. soln., and the positions found for the acetonitrile mols. in the active site are compared to those of similar functional groups belonging to known inhibitors bound to elastase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhPgz3NRlYqrVg90H21EOLACvtfcHk0lgJLinWhiFK9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmt1GrtQ%253D%253D&md5=c98dea1a223f30e444b9bc7faff25f0b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjp952516o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp952516o%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DBellamacina%26aufirst%3DC.%2BR.%26aulast%3DDing%26aufirst%3DX.%2BC.%26aulast%3DJeffery%26aufirst%3DC.%2BJ.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DAn%2520experimental%2520approach%2520to%2520mapping%2520the%2520binding%2520surfaces%2520of%2520crystalline%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1996%26volume%3D100%26spage%3D2605%26epage%3D11%26doi%3D10.1021%2Fjp952516o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">English, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Done, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caves, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">Locating interaction sites on proteins: The crystal structure of thermolysin soaked in 2% to 100% isopropanol</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">628</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19991201)37:4<628::AID-PROT13>3.3.CO;2-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2F%28SICI%291097-0134%2819991201%2937%3A4%3C628%3A%3AAID-PROT13%3E3.3.CO%3B2-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1999&pages=628-40&author=A.+C.+Englishauthor=S.+H.+Doneauthor=L.+S.+Cavesauthor=C.+R.+Groomauthor=R.+E.+Hubbard&title=Locating+interaction+sites+on+proteins%3A+The+crystal+structure+of+thermolysin+soaked+in+2%25+to+100%25+isopropanol&doi=10.1002%2F%28SICI%291097-0134%2819991201%2937%3A4%3C628%3A%3AAID-PROT13%3E3.3.CO%3B2-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819991201%2937%3A4%3C628%3A%3AAID-PROT13%3E3.3.CO%3B2-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819991201%252937%253A4%253C628%253A%253AAID-PROT13%253E3.3.CO%253B2-7%26sid%3Dliteratum%253Aachs%26aulast%3DEnglish%26aufirst%3DA.%2BC.%26aulast%3DDone%26aufirst%3DS.%2BH.%26aulast%3DCaves%26aufirst%3DL.%2BS.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DLocating%2520interaction%2520sites%2520on%2520proteins%253A%2520The%2520crystal%2520structure%2520of%2520thermolysin%2520soaked%2520in%25202%2525%2520to%2520100%2525%2520isopropanol%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1999%26volume%3D37%26spage%3D628%26epage%3D40%26doi%3D10.1002%2F%28SICI%291097-0134%2819991201%2937%3A4%3C628%3A%3AAID-PROT13%3E3.3.CO%3B2-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">English, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">Experimental and computational mapping of the binding surface of a crystalline protein</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1093/protein/14.1.47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fprotein%2F14.1.47" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2001&pages=47-59&author=A.+C.+Englishauthor=C.+R.+Groomauthor=R.+E.+Hubbard&title=Experimental+and+computational+mapping+of+the+binding+surface+of+a+crystalline+protein&doi=10.1093%2Fprotein%2F14.1.47"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F14.1.47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F14.1.47%26sid%3Dliteratum%253Aachs%26aulast%3DEnglish%26aufirst%3DA.%2BC.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DExperimental%2520and%2520computational%2520mapping%2520of%2520the%2520binding%2520surface%2520of%2520a%2520crystalline%2520protein%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2001%26volume%3D14%26spage%3D47%26epage%3D59%26doi%3D10.1093%2Fprotein%2F14.1.47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Dechene, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wink, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swartz, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span> </span><span class="NLM_article-title">Multiple solvent crystal structures of ribonuclease A: An assessment of the method</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1002/prot.22393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2Fprot.22393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19291738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Clur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2009&pages=861-81&author=M.+Decheneauthor=G.+Winkauthor=M.+Smithauthor=P.+Swartzauthor=C.+Mattos&title=Multiple+solvent+crystal+structures+of+ribonuclease+A%3A+An+assessment+of+the+method&doi=10.1002%2Fprot.22393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple solvent crystal structures of ribonuclease A: An assessment of the method</span></div><div class="casAuthors">Dechene, Michelle; Wink, Glenna; Smith, Mychal; Swartz, Paul; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">861-881</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The multiple solvent crystal structures (MSCS) method uses org. solvents to map the surfaces of proteins.  It identifies binding sites and allows for a more thorough examn. of protein plasticity and hydration than could be achieved by a single structure.  The crystal structures of bovine pancreatic RNase A (RNase A) soaked in the following org. solvents are presented: 50% dioxane, 50% DMF, 70% dimethylsulfoxide, 70% 1,6-hexanediol, 70% isopropanol, 50% R,S,R-bisfuran alc., 70% t-butanol, 50% trifluoroethanol, or 1.0M trimethylamine-N-oxide.  This set of structures is compared with four sets of crystal structures of RNase A from the protein data bank (PDB) and with the soln. NMR structure to assess the validity of previously untested assumptions assocd. with MSCS anal.  Plasticity from MSCS is the same as from PDB structures obtained in the same crystal form and deviates only at crystal contacts when compared to structures from a diverse set of crystal environments.  Furthermore, there is a good correlation between plasticity as obsd. by MSCS and the dynamic regions seen by NMR.  Conserved water binding sites are identified by MSCS to be those that are conserved in the sets of structures taken from the PDB.  Comparison of the MSCS structures with inhibitor-bound crystal structures of RNase A reveals that the org. solvent mols. identify key interactions made by inhibitor mols., highlighting ligand binding hot-spots in the active site.  The present work firmly establishes the relevance of information obtained by MSCS.  Proteins 2009. © 2009 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0dhAgJZJfDLVg90H21EOLACvtfcHk0lgJLinWhiFK9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Clur8%253D&md5=023859b5a4d1847d2a01ebfe049a4097</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fprot.22393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22393%26sid%3Dliteratum%253Aachs%26aulast%3DDechene%26aufirst%3DM.%26aulast%3DWink%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DM.%26aulast%3DSwartz%26aufirst%3DP.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DMultiple%2520solvent%2520crystal%2520structures%2520of%2520ribonuclease%2520A%253A%2520An%2520assessment%2520of%2520the%2520method%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2009%26volume%3D76%26spage%3D861%26epage%3D81%26doi%3D10.1002%2Fprot.22393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Buhrman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connor, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zerbe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kearney, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napoleon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovrigina, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovrigin, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span> </span><span class="NLM_article-title">Analysis of binding site hot spots on the surface of Ras GTPase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">413</span><span class="NLM_x">, </span> <span class="NLM_fpage">773</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1016/j.jmb.2011.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jmb.2011.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21945529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKisbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=413&publication_year=2011&pages=773-89&author=G.+Buhrmanauthor=C.+O%E2%80%99Connorauthor=B.+Zerbeauthor=B.+M.+Kearneyauthor=R.+Napoleonauthor=E.+A.+Kovriginaauthor=S.+Vajdaauthor=D.+Kozakovauthor=E.+L.+Kovriginauthor=C.+Mattos&title=Analysis+of+binding+site+hot+spots+on+the+surface+of+Ras+GTPase&doi=10.1016%2Fj.jmb.2011.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Binding Site Hot Spots on the Surface of Ras GTPase</span></div><div class="casAuthors">Buhrman, Greg; O'Connor, Casey; Zerbe, Brandon; Kearney, Bradley M.; Napoleon, Raeanne; Kovrigina, Elizaveta A.; Vajda, Sandor; Kozakov, Dima; Kovrigin, Evgenii L.; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">413</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">773-789</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have recently discovered an allosteric switch in Ras, bringing an addnl. level of complexity to this GTPase whose mutants are involved in nearly 30% of cancers.  Upon activation of the allosteric switch, there is a shift in helix 3/loop 7 assocd. with a disorder to order transition in the active site.  Here, we use a combination of multiple solvent crystal structures and computational solvent mapping (FTMap) to det. binding site hot spots in the "off" and "on" allosteric states of the GTP-bound form of H-Ras.  Thirteen sites are revealed, expanding possible target sites for ligand binding well beyond the active site.  Comparison of FTMaps for the H and K isoforms reveals essentially identical hot spots.  Furthermore, using NMR measurements of spin relaxation, we detd. that K-Ras exhibits global conformational dynamics very similar to those we previously reported for H-Ras.  We thus hypothesize that the global conformational rearrangement serves as a mechanism for allosteric coupling between the effector interface and remote hot spots in all Ras isoforms.  At least with respect to the binding sites involving the G domain, H-Ras is an excellent model for K-Ras and probably N-Ras as well.  Ras has so far been elusive as a target for drug design.  The present work identifies various unexplored hot spots throughout the entire surface of Ras, extending the focus from the disordered active site to well-ordered locations that should be easier to target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAmYA3G3UBirVg90H21EOLACvtfcHk0lhaPGQ-Ff8Gww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKisbrN&md5=d54dee9a6ac58d27f87d85a6d25415a3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DBuhrman%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DC.%26aulast%3DZerbe%26aufirst%3DB.%26aulast%3DKearney%26aufirst%3DB.%2BM.%26aulast%3DNapoleon%26aufirst%3DR.%26aulast%3DKovrigina%26aufirst%3DE.%2BA.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DKovrigin%26aufirst%3DE.%2BL.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DAnalysis%2520of%2520binding%2520site%2520hot%2520spots%2520on%2520the%2520surface%2520of%2520Ras%2520GTPase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D413%26spage%3D773%26epage%3D89%26doi%3D10.1016%2Fj.jmb.2011.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Edfeldt, F. N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folmer, R. H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breeze, A. L.</span><span> </span><span class="NLM_article-title">Fragment screening to predict druggability (ligandability) and lead discovery success</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1016/j.drudis.2011.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.drudis.2011.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21315179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFemsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=284-287&author=F.+N.+B.+Edfeldtauthor=R.+H.+A.+Folmerauthor=A.+L.+Breeze&title=Fragment+screening+to+predict+druggability+%28ligandability%29+and+lead+discovery+success&doi=10.1016%2Fj.drudis.2011.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment screening to predict druggability (ligandability) and lead discovery success</span></div><div class="casAuthors">Edfeldt, Fredrik N. B.; Folmer, Rutger H. A.; Breeze, Alexander L.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7/8</span>),
    <span class="NLM_cas:pages">284-287</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This article discusses the use of fragment screening to predict ligandability which will be of great interest to the pharmaceutical industry.  Data presented in this article clearly show how fragment-based ligandability screening can be used in a truly predictive fashion and could be a powerful tool in reducing agent attrition by filtering out non-ligandable targets before entry into the portfolio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSXfOy9wAHFLVg90H21EOLACvtfcHk0lhaPGQ-Ff8Gww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFemsbs%253D&md5=7ba5adfe3d6a1eb4c9eaa3ce42c60044</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DEdfeldt%26aufirst%3DF.%2BN.%2BB.%26aulast%3DFolmer%26aufirst%3DR.%2BH.%2BA.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26atitle%3DFragment%2520screening%2520to%2520predict%2520druggability%2520%2528ligandability%2529%2520and%2520lead%2520discovery%2520success%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D284%26epage%3D287%26doi%3D10.1016%2Fj.drudis.2011.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krimm, I.</span><span> </span><span class="NLM_article-title">Ligand specificity, privileged substructures and protein druggability from fragment-based screening</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">–</span> <span class="NLM_lpage">74</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2011.02.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.cbpa.2011.02.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21411360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFCltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2011&pages=469-74&author=S.+Barelierauthor=I.+Krimm&title=Ligand+specificity%2C+privileged+substructures+and+protein+druggability+from+fragment-based+screening&doi=10.1016%2Fj.cbpa.2011.02.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand specificity, privileged substructures and protein druggability from fragment-based screening</span></div><div class="casAuthors">Barelier, Sarah; Krimm, Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fragment-based screening has now become an established method for the generation of lead mols. against therapeutic targets.  Fragment mols. are simple, low mol.-wt. compds. with few chem. functionalities.  These characteristics lead to high hit rates for fragment screening as compared to the more classical High-Throughput Screening of drug-like mols. and raise the question of the specificity of fragment mols.  This review analyzes recent outcomes of fragment screenings published in the literature, showing that the specificity of the fragments can be related to their structures and physico-chem. properties.  We also discuss both the concept of privileged fragment scaffolds and the role of fragment-based screening in predicting protein druggability, highlighted by recent publications in the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD_Gsih0nTRLVg90H21EOLACvtfcHk0lhaPGQ-Ff8Gww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFCltr0%253D&md5=7c9e813220263e006c9ae01e51a9334f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2011.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2011.02.020%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DKrimm%26aufirst%3DI.%26atitle%3DLigand%2520specificity%252C%2520privileged%2520substructures%2520and%2520protein%2520druggability%2520from%2520fragment-based%2520screening%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D15%26spage%3D469%26epage%3D74%26doi%3D10.1016%2Fj.cbpa.2011.02.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Dennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kortvelyesi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Computational mapping identifies the binding sites of organic solvents on proteins</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">4290</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1073/pnas.062398499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.062398499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11904374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=4290-5&author=S.+Dennisauthor=T.+Kortvelyesiauthor=S.+Vajda&title=Computational+mapping+identifies+the+binding+sites+of+organic+solvents+on+proteins&doi=10.1073%2Fpnas.062398499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Computational mapping identifies the binding sites of organic solvents on proteins</span></div><div class="casAuthors">Dennis, Sheldon; Kortvelyesi, Tamas; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4290-4295</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Computational mapping places mol. probes-small mols. or functional groups-on a protein surface to identify the most favorable binding positions.  Although x-ray crystallog. and NMR show that org. solvents bind to a limited no. of sites on a protein, current mapping methods result in hundreds of energy min. and do not reveal why some sites bind mols. with different sizes and polarities.  We describe a mapping algorithm that explains the origin of this phenomenon.  The algorithm has been applied to hen egg-white lysozyme and to thermolysin, interacting with eight and four different ligands, resp.  In both cases the search finds the consensus site to which all mols. bind, whereas other positions that bind only certain ligands are not necessarily found.  The consensus sites are pockets of the active site, lined with partially exposed hydrophobic residues and with a no. of polar residues toward the edge.  These sites can accommodate each ligand in a no. of rotational states, some with a hydrogen bond to one of the nearby donor/acceptor groups.  Specific substrates and/or inhibitors of hen egg-white lysozyme and thermolysin interact with the same side chains identified by the mapping, but form several hydrogen bonds and bind in unique orientations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4sEo2VxwF7Vg90H21EOLACvtfcHk0ljvnBo5rKzIBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D&md5=90d71ffcc8860e87aefec3c5809b1127</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.062398499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.062398499%26sid%3Dliteratum%253Aachs%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DKortvelyesi%26aufirst%3DT.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DComputational%2520mapping%2520identifies%2520the%2520binding%2520sites%2520of%2520organic%2520solvents%2520on%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D4290%26epage%3D5%26doi%3D10.1073%2Fpnas.062398499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Landon, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancia, D. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiel, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Identification of hot spots within druggable binding regions by computational solvent mapping of proteins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1231</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span><span class="refDoi"> DOI: 10.1021/jm061134b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061134b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1231-40&author=M.+R.+Landonauthor=D.+R.+Lanciaauthor=J.+Yuauthor=S.+C.+Thielauthor=S.+Vajda&title=Identification+of+hot+spots+within+druggable+binding+regions+by+computational+solvent+mapping+of+proteins&doi=10.1021%2Fjm061134b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Hot Spots within Druggable Binding Regions by Computational Solvent Mapping of Proteins</span></div><div class="casAuthors">Landon, Melissa R.; Lancia, David R., Jr.; Yu, Jessamin; Thiel, Spencer C.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1231-1240</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors apply the computational solvent mapping (CS-Map) algorithm toward the in silico identification of hot spots, i.e., regions of protein binding sites that are major contributors to the binding energy and, hence, are prime targets in drug design.  The CS-Map algorithm, developed for binding site characterization, moves small org. functional groups around the protein surface and dets. their most energetically favorable binding positions.  The utility of CS-Map algorithm toward the prediction of hot spot regions in druggable binding pockets is illustrated by three test systems: (1) renin aspartic protease, (2) a set of previously characterized druggable proteins, and (3) E. coli ketopantoate reductase.  In each of the three studies, existing literature was used to verify the results.  Based on the authors analyses, the authors conclude that the information provided by CS-Map can contribute substantially to the identification of hot spots, a necessary predecessor of fragment-based drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTVim9shdtBbVg90H21EOLACvtfcHk0ljvnBo5rKzIBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWisrw%253D&md5=2d6a0678acdb2dd45fed73b05229bd03</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm061134b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061134b%26sid%3Dliteratum%253Aachs%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DLancia%26aufirst%3DD.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DThiel%26aufirst%3DS.%2BC.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520hot%2520spots%2520within%2520druggable%2520binding%2520regions%2520by%2520computational%2520solvent%2520mapping%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1231%26epage%3D40%26doi%3D10.1021%2Fjm061134b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Brenke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landon, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Fragment-based identification of druggable ’hot spots’ of proteins using Fourier domain correlation techniques</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">621</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fbioinformatics%2Fbtp036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19176554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFeksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=621-7&author=R.+Brenkeauthor=D.+Kozakovauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=D.+Hallauthor=M.+R.+Landonauthor=C.+Mattosauthor=S.+Vajda&title=Fragment-based+identification+of+druggable+%E2%80%99hot+spots%E2%80%99+of+proteins+using+Fourier+domain+correlation+techniques&doi=10.1093%2Fbioinformatics%2Fbtp036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based identification of druggable hot spots' of proteins using Fourier domain correlation techniques</span></div><div class="casAuthors">Brenke, Ryan; Kozakov, Dima; Chuang, Gwo-Yu; Beglov, Dmitri; Hall, David; Landon, Melissa R.; Mattos, Carla; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">621-627</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The binding sites of proteins generally contain smaller regions that provide major contributions to the binding free energy and hence are the prime targets in drug design.  Screening libraries of fragment-sized compds. by NMR or X-ray crystallog. demonstrates that such hot spot' regions bind a large variety of small org. mols., and that a relatively high hit rate' is predictive of target sites that are likely to bind drug-like ligands with high affinity.  The authors' goal is to det. the hot spots' computationally rather than exptl.  The authors have developed the FTMAP algorithm that performs global search of the entire protein surface for regions that bind a no. of small org. probe mols.  The search is based on the extremely efficient fast Fourier transform (FFT) correlation approach which can sample billions of probe positions on dense translational and rotational grids, but can use only sums of correlation functions for scoring and hence is generally restricted to very simple energy expressions.  The novelty of FTMAP is that the authors were able to incorporate and represent on grids a detailed energy expression, resulting in a very accurate identification of low-energy probe clusters.  Overlapping clusters of different probes are defined as consensus sites (CSs).  The authors show that the largest CS is generally located at the most important subsite of the protein binding site, and the nearby smaller CSs identify other important subsites.  Mapping results are presented for elastase whose structure was solved in aq. solns. of 8 org. solvents, and they show that FTMAP provides very similar information.  The second application is to renin, a long-standing pharmaceutical target for the treatment of hypertension, and the authors show that the major CSs trace out the shape of the first approved renin inhibitor, aliskiren.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWSzH6Fg129bVg90H21EOLACvtfcHk0ljvnBo5rKzIBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFeksr0%253D&md5=e5a466428f4dd47e89a4ae8361d3da41</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp036%26sid%3Dliteratum%253Aachs%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DFragment-based%2520identification%2520of%2520druggable%2520%25E2%2580%2599hot%2520spots%25E2%2580%2599%2520of%2520proteins%2520using%2520Fourier%2520domain%2520correlation%2520techniques%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3D621%26epage%3D7%26doi%3D10.1093%2Fbioinformatics%2Fbtp036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Landon, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoang, Q. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caaveiro, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orwig, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Detection of ligand binding hot spots on protein surfaces via fragment-based methods: Application to DJ-1 and glucocerebrosidase</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">500</span><span class="refDoi"> DOI: 10.1007/s10822-009-9283-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs10822-009-9283-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19521672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=491-500&author=M.+R.+Landonauthor=R.+L.+Liebermanauthor=Q.+Q.+Hoangauthor=S.+Juauthor=J.+M.+Caaveiroauthor=S.+D.+Orwigauthor=D.+Kozakovauthor=R.+Brenkeauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=S.+Vajdaauthor=G.+A.+Petskoauthor=D.+Ringe&title=Detection+of+ligand+binding+hot+spots+on+protein+surfaces+via+fragment-based+methods%3A+Application+to+DJ-1+and+glucocerebrosidase&doi=10.1007%2Fs10822-009-9283-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of ligand binding hot spots on protein surfaces via fragment-based methods: application to DJ-1 and glucocerebrosidase</span></div><div class="casAuthors">Landon, Melissa R.; Lieberman, Raquel L.; Hoang, Quyen Q.; Ju, Shulin; Caaveiro, Jose M. M.; Orwig, Susan D.; Kozakov, Dima; Brenke, Ryan; Chuang, Gwo-Yu; Beglov, Dmitry; Vajda, Sandor; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">491-500</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The identification of hot spots, i.e., binding regions that contribute substantially to the free energy of ligand binding, is a crit. step for structure-based drug design.  Here we present the application of two fragment-based methods to the detection of hot spots for DJ-1 and glucocerebrosidase (GCase), targets for the development of therapeutics for Parkinson's and Gaucher's diseases, resp.  While the structures of these two proteins are known, binding information is lacking.  In this study we employ the exptl. multiple solvent crystal structures (MSCS) method and computational fragment mapping (FTMap) to identify regions suitable for the development of pharmacol. chaperones for DJ-1 and GCase.  Comparison of data derived via MSCS and FTMap also shows that FTMap, a computational method for the identification of fragment binding hot spots, is an accurate and robust alternative to the performance of expensive and difficult crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVrKBMCEfe6LVg90H21EOLACvtfcHk0lgr55d_2qXEUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtb8%253D&md5=00a35c8ecaaa3b9f3f9aeff7c7e61d27</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9283-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9283-2%26sid%3Dliteratum%253Aachs%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DLieberman%26aufirst%3DR.%2BL.%26aulast%3DHoang%26aufirst%3DQ.%2BQ.%26aulast%3DJu%26aufirst%3DS.%26aulast%3DCaaveiro%26aufirst%3DJ.%2BM.%26aulast%3DOrwig%26aufirst%3DS.%2BD.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DDetection%2520of%2520ligand%2520binding%2520hot%2520spots%2520on%2520protein%2520surfaces%2520via%2520fragment-based%2520methods%253A%2520Application%2520to%2520DJ-1%2520and%2520glucocerebrosidase%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D491%26epage%3D500%26doi%3D10.1007%2Fs10822-009-9283-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Where does amantadine bind to the influenza virus M2 proton channel?</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">471</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1016/j.tibs.2010.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.tibs.2010.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=20382026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFamtrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2010&pages=471-5&author=D.+Kozakovauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=S.+Vajda&title=Where+does+amantadine+bind+to+the+influenza+virus+M2+proton+channel%3F&doi=10.1016%2Fj.tibs.2010.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Where does amantadine bind to the influenza virus M2 proton channel?</span></div><div class="casAuthors">Kozakov, Dima; Chuang, Gwo-Yu; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">471-475</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Structures of the influenza A virus M2 proton channel in the open conformation have been detd. by X-ray crystallog., and in the closed conformation by NMR.  Whereas the X-ray structure shows a single inhibitor mol. in the middle of the channel, four inhibitor mols. bind the channel's outer surface in the NMR structure.  In both structures, the strongest hot spots (i.e., regions that contribute substantially to the free energy of binding any potential ligand) lie inside the pore, and other hot spots are found at exterior locations.  By considering all available models, we propose the primary drug binding site is inside the pore, but that exterior binding occurs under appropriate conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPdRX2k5BMm7Vg90H21EOLACvtfcHk0lgr55d_2qXEUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFamtrbP&md5=c8cd562c32b5b63ae865cf0df9e1f457</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DWhere%2520does%2520amantadine%2520bind%2520to%2520the%2520influenza%2520virus%2520M2%2520proton%2520channel%253F%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2010%26volume%3D35%26spage%3D471%26epage%3D5%26doi%3D10.1016%2Fj.tibs.2010.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cencic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Structural conservation of druggable hot spots in protein-protein interfaces</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">13528</span><span class="NLM_x">–</span> <span class="NLM_lpage">33</span><span class="refDoi"> DOI: 10.1073/pnas.1101835108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1101835108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21808046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2ms7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13528-33&author=D.+Kozakovauthor=D.+R.+Hallauthor=G.+Y.+Chuangauthor=R.+Cencicauthor=R.+Brenkeauthor=L.+E.+Groveauthor=D.+Beglovauthor=J.+Pelletierauthor=A.+Whittyauthor=S.+Vajda&title=Structural+conservation+of+druggable+hot+spots+in+protein-protein+interfaces&doi=10.1073%2Fpnas.1101835108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural conservation of druggable hot spots in protein-protein interfaces</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Chuang, Gwo-Yu; Cencic, Regina; Brenke, Ryan; Grove, Laurie E.; Beglov, Dmitri; Pelletier, Jerry; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">13528-13533, S13528/1-S13528/26</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Despite the growing no. of examples of small-mol. inhibitors that disrupt protein-protein interactions (PPIs), the origin of druggability of such targets is poorly understood.  To identify druggable sites in protein-protein interfaces we combine computational solvent mapping, which explores the protein surface using a variety of small "probe" mols., with a conformer generator to account for side-chain flexibility.  Applications to unliganded structures of 15 PPI target proteins show that the druggable sites comprise a cluster of binding hot spots, distinguishable from other regions of the protein due to their concave topol. combined with a pattern of hydrophobic and polar functionality.  This combination of properties confers on the hot spots a tendency to bind org. species possessing some polar groups decorating largely hydrophobic scaffolds.  Thus, druggable sites at PPI are not simply sites that are complementary to particular org. functionality, but rather possess a general tendency to bind org. compds. with a variety of structures, including key side chains of the partner protein.  Results also highlight the importance of conformational adaptivity at the binding site to allow the hot spots to expand to accommodate a ligand of drug-like dimensions.  The crit. components of this adaptivity are largely local, involving primarily low energy side-chain motions within 6 Å of a hot spot.  The structural and physicochem. signature of druggable sites at PPI interfaces is sufficiently robust to be detectable from the structure of the unliganded protein, even when substantial conformational adaptation is required for optimal ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpibu1osEQcZrVg90H21EOLACvtfcHk0lgr55d_2qXEUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2ms7rL&md5=fc9b8a30f1b30c97bba5a843fb9dc03b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1101835108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1101835108%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DCencic%26aufirst%3DR.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DPelletier%26aufirst%3DJ.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DStructural%2520conservation%2520of%2520druggable%2520hot%2520spots%2520in%2520protein-protein%2520interfaces%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13528%26epage%3D33%26doi%3D10.1073%2Fpnas.1101835108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Hall, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yueh, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Robust identification of binding hot spots using continuum electrostatics: Application to hen egg-white lysozyme</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">20668</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1021/ja207914y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja207914y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=20668-71&author=D.+H.+Hallauthor=L.+E.+Groveauthor=C.+Yuehauthor=C.+H.+Nganauthor=D.+Kozakovauthor=S.+Vajda&title=Robust+identification+of+binding+hot+spots+using+continuum+electrostatics%3A+Application+to+hen+egg-white+lysozyme&doi=10.1021%2Fja207914y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja207914y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja207914y%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BH.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DYueh%26aufirst%3DC.%26aulast%3DNgan%26aufirst%3DC.%2BH.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DRobust%2520identification%2520of%2520binding%2520hot%2520spots%2520using%2520continuum%2520electrostatics%253A%2520Application%2520to%2520hen%2520egg-white%2520lysozyme%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D20668%26epage%3D71%26doi%3D10.1021%2Fja207914y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zerbe, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Hot spot analysis for driving the development of hits into leads in fragment-based drug discovery</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1021/ci200468p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200468p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFygt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=199-209&author=D.+R.+Hallauthor=C.+H.+Nganauthor=B.+S.+Zerbeauthor=D.+Kozakovauthor=S.+Vajda&title=Hot+spot+analysis+for+driving+the+development+of+hits+into+leads+in+fragment-based+drug+discovery&doi=10.1021%2Fci200468p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Hot Spot Analysis for Driving the Development of Hits into Leads in Fragment-Based Drug Discovery</span></div><div class="casAuthors">Hall, David R.; Ngan, Chi Ho; Zerbe, Brandon S.; Kozakov, Dima; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">199-209</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug design (FBDD) starts with finding fragment-sized compds. that are highly ligand efficient and can serve as a core moiety for developing high-affinity leads.  Although the core-bound structure of a protein facilitates the construction of leads, effective design is far from straightforward.  We show that protein mapping, a computational method developed to find binding hot spots and implemented as the FTMap server, provides information that complements the fragment screening results and can drive the evolution of core fragments into larger leads with a minimal loss or, in some cases, even a gain in ligand efficiency.  The method places small mol. probes, the size of org. solvents, on a dense grid around the protein and identifies the hot spots as consensus clusters formed by clusters of several probes.  The hot spots are ranked based on the no. of probe clusters, which predicts the binding propensity of the subsites and hence their importance for drug design.  Accordingly, with a single exception the main hot spot identified by FTMap binds the core compd. found by fragment screening.  The most useful information is provided by the neighboring secondary hot spots, indicating the regions where the core can be extended to increase its affinity.  To quantify this information, we calc. the d. of probes from mapping, which describes the binding propensity at each point, and show that the change in the correlation between a ligand position and the probe d. upon extending or repositioning the core moiety predicts the expected change in ligand efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoahb_AwEPwybVg90H21EOLACvtfcHk0lilXooLsz56bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFygt7vK&md5=674174e6634740d828576aeed2424232</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fci200468p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200468p%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DNgan%26aufirst%3DC.%2BH.%26aulast%3DZerbe%26aufirst%3DB.%2BS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DHot%2520spot%2520analysis%2520for%2520driving%2520the%2520development%2520of%2520hits%2520into%2520leads%2520in%2520fragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D199%26epage%3D209%26doi%3D10.1021%2Fci200468p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Golden, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sayeg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zerbe, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villar, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sazinsky, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiadis, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">Comprehensive experimental and computational analysis of binding energy hot spots at the NF-κB essential modulator/IKKβ protein-protein interface</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">6242</span><span class="NLM_x">–</span> <span class="NLM_lpage">56</span><span class="refDoi"> DOI: 10.1021/ja400914z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja400914z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFKnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6242-56&author=M.+S.+Goldenauthor=S.+M.+Coteauthor=M.+Sayegauthor=B.+S.+Zerbeauthor=E.+A.+Villarauthor=D.+Beglovauthor=S.+L.+Sazinskyauthor=R.+M.+Georgiadisauthor=S.+Vajdaauthor=D.+Kozakovauthor=A.+Whitty&title=Comprehensive+experimental+and+computational+analysis+of+binding+energy+hot+spots+at+the+NF-%CE%BAB+essential+modulator%2FIKK%CE%B2+protein-protein+interface&doi=10.1021%2Fja400914z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Experimental and Computational Analysis of Binding Energy Hot Spots at the NF-κB Essential Modulator/IKKβ Protein-Protein Interface</span></div><div class="casAuthors">Golden, Mary S.; Cote, Shaun M.; Sayeg, Marianna; Zerbe, Brandon S.; Villar, Elizabeth A.; Beglov, Dmitri; Sazinsky, Stephen L.; Georgiadis, Rosina M.; Vajda, Sandor; Kozakov, Dima; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6242-6256</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a comprehensive anal. of binding energy hot spots at the protein-protein interaction (PPI) interface between nuclear factor kappa B (NF-κB) essential modulator (NEMO) and IκB kinase subunit β (IKKβ), an interaction that is crit. for NF-κB pathway signaling, using exptl. alanine scanning mutagenesis and also the FTMap method for computational fragment screening.  The exptl. results confirm that the previously identified NEMO binding domain (NBD) region of IKKβ contains the highest concn. of hot-spot residues, the strongest of which are W739, W741, and L742 (ΔΔG = 4.3, 3.5, and 3.2 kcal/mol, resp.).  The region occupied by these residues defines a potentially druggable binding site on NEMO that extends for ∼16 Å to addnl. include the regions that bind IKKβ L737 and F734.  NBD residues D738 and S740 are also important for binding but do not make direct contact with NEMO, instead likely acting to stabilize the active conformation of surrounding residues.  We addnl. found two previously unknown hot-spot regions centered on IKKβ residues L708/V709 and L719/I723.  The computational approach successfully identified all three hot-spot regions on IKKβ.  Moreover, the method was able to accurately quantify the energetic importance of all hot-spot residues involving direct contact with NEMO.  Our results provide new information to guide the discovery of small-mol. inhibitors that target the NEMO/IKKβ interaction.  They addnl. clarify the structural and energetic complementarity between "pocket-forming" and "pocket-occupying" hot-spot residues, and further validate computational fragment mapping as a method for identifying hot spots at PPI interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLvGilMwRJGrVg90H21EOLACvtfcHk0lilXooLsz56bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFKnsL0%253D&md5=d996a1aef7668ff03fe9bf3445c7d700</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fja400914z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja400914z%26sid%3Dliteratum%253Aachs%26aulast%3DGolden%26aufirst%3DM.%2BS.%26aulast%3DCote%26aufirst%3DS.%2BM.%26aulast%3DSayeg%26aufirst%3DM.%26aulast%3DZerbe%26aufirst%3DB.%2BS.%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DSazinsky%26aufirst%3DS.%2BL.%26aulast%3DGeorgiadis%26aufirst%3DR.%2BM.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DComprehensive%2520experimental%2520and%2520computational%2520analysis%2520of%2520binding%2520energy%2520hot%2520spots%2520at%2520the%2520NF-%25CE%25BAB%2520essential%2520modulator%252FIKK%25CE%25B2%2520protein-protein%2520interface%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6242%26epage%3D56%26doi%3D10.1021%2Fja400914z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mottarella, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span><span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-55&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0liQvPgKp7HIMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D55%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Liepinsh, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otting, G.</span><span> </span><span class="NLM_article-title">Organic solvents identify specific ligand binding sites on protein surfaces</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">264</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1038/nbt0397-264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnbt0397-264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=9062927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK2sXhslKhsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=264-8&author=E.+Liepinshauthor=G.+Otting&title=Organic+solvents+identify+specific+ligand+binding+sites+on+protein+surfaces&doi=10.1038%2Fnbt0397-264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Organic solvents identify specific ligand-binding sites on protein surfaces</span></div><div class="casAuthors">Liepinsh, Edvards; Otting, Gottfried</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">264-268</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">Enzymes frequently recognize substrates and pharmaceutical drugs through specific binding interactions in deep pockets on the protein surface.  We show how the specificity-detg. substrate-binding site of hen egg-white lysozyme (HEWL) can be readily identified in aq. soln. by NMR spectroscopy using small org. solvent mols. as detection probes.  Exchange of magnetization between the 1H nuclei of the protein and the ligands through dipole-dipole interactions is obsd. which allows the modeling of their position and orientation at the binding site.  Combined with site-specific binding consts. measured by titrn. expts. with different org. solvents, the method can provide important information for rational drug design.  In addn., the lifetime of nonspecific interactions of HEWL with org. solvents is shown to be in the sub-nanosecond time range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqafHPYYxoVVrVg90H21EOLACvtfcHk0liQvPgKp7HIMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhslKhsr4%253D&md5=912425d4c7357f6cf9f8bd4492c6fee7</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnbt0397-264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0397-264%26sid%3Dliteratum%253Aachs%26aulast%3DLiepinsh%26aufirst%3DE.%26aulast%3DOtting%26aufirst%3DG.%26atitle%3DOrganic%2520solvents%2520identify%2520specific%2520ligand%2520binding%2520sites%2520on%2520protein%2520surfaces%26jtitle%3DNat.%2520Biotechnol.%26date%3D1997%26volume%3D15%26spage%3D264%26epage%3D8%26doi%3D10.1038%2Fnbt0397-264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellamacina, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peisach, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vitkup, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Multiple solvent crystal structures: Probing binding sites, plasticity and hydration</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">1471</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/j.jmb.2006.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jmb.2006.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16488429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVGqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2006&pages=1471-82&author=C.+Mattosauthor=C.+R.+Bellamacinaauthor=E.+Peisachauthor=A.+Pereiraauthor=D.+Vitkupauthor=G.+A.+Petskoauthor=D.+Ringe&title=Multiple+solvent+crystal+structures%3A+Probing+binding+sites%2C+plasticity+and+hydration&doi=10.1016%2Fj.jmb.2006.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Solvent Crystal Structures: Probing Binding Sites, Plasticity and Hydration</span></div><div class="casAuthors">Mattos, Carla; Bellamacina, Cornelia R.; Peisach, Ezra; Pereira, Antonio; Vitkup, Dennis; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1471-1482</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Multiple solvent crystal structures (MSCS) of porcine pancreatic elastase were used to map the binding surface the enzyme.  Crystal structures of elastase in neat acetonitrile, 95% acetone, 55% DMF, 80% 5-hexene-1,2-diol, 80% isopropanol, 80% ethanol and 40% trifluoroethanol showed that the org. solvent mols. clustered in the active site, were found mostly unclustered in crystal contacts and in general did not bind elsewhere on the surface of elastase.  Mixts. of 40% benzene or 40% cyclohexane in 50% isopropanol and 10% water showed no bound benzene or cyclohexane mols., but did reveal bound isopropanol. The clusters of org. solvent probe mols. coincide with pockets occupied by known inhibitors.  MSCS also reveal the areas of plasticity within the elastase binding site and allow for the visualization of a nearly complete first hydration shell.  The pattern of org. solvent clusters detd. by MSCS for elastase is consistent with patterns for hot spots in protein-ligand interactions detd. from database anal. in general.  The MSCS method allows probing of hot spots, plasticity and hydration simultaneously, providing a powerful complementary strategy to guide computational methods currently in development for binding site detn., ligand docking and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq66xX3ZT7wRLVg90H21EOLACvtfcHk0liQvPgKp7HIMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVGqtrk%253D&md5=ab7064d371e5557b2f91aeb6c262a5dc</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DBellamacina%26aufirst%3DC.%2BR.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DPereira%26aufirst%3DA.%26aulast%3DVitkup%26aufirst%3DD.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DMultiple%2520solvent%2520crystal%2520structures%253A%2520Probing%2520binding%2520sites%252C%2520plasticity%2520and%2520hydration%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D357%26spage%3D1471%26epage%3D82%26doi%3D10.1016%2Fj.jmb.2006.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-42&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lg1WuTSawVsXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D42%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">The rise of fragment-based drug discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span><span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+rise+of+fragment-based+drug+discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0lg1WuTSawVsXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520rise%2520of%2520fragment-based%2520drug%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">64</span><span class="refDoi"> DOI: 10.1038/nchembio.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchembio.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19305397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=358-64&author=Y.+Chenauthor=B.+K.+Shoichet&title=Molecular+docking+and+ligand+specificity+in+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span></div><div class="casAuthors">Chen, Yu; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment screens have successfully identified new scaffolds in drug discovery, often with relatively high hit rates (5%) using small screening libraries (1,000-10,000 compds.).  This raises two questions: would other noteworthy chemotypes be found were one to screen all com. available fragments (>300,000), and does the success rate imply low specificity of fragments.  We used mol. docking to screen large libraries of fragments against CTX-M β-lactamase.  We identified ten millimolar-range inhibitors from the 69 compds. tested.  The docking poses corresponded closely to the crystallog. structures subsequently detd.  Notably, these initial low-affinity hits showed little specificity between CTX-M and an unrelated β-lactamase, AmpC, which is unusual among β-lactamase inhibitors.  This is consistent with the idea that the high hit rates among fragments correlate to a low initial specificity.  As the inhibitors were progressed, both specificity and affinity rose together, yielding to our knowledge the first micromolar-range noncovalent inhibitors against a class A β-lactamase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8zTkq9kMAbVg90H21EOLACvtfcHk0lg1WuTSawVsXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D&md5=6ee8dd19d0b99599ea74eaf735483b1d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.155%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520ligand%2520specificity%2520in%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D358%26epage%3D64%26doi%3D10.1038%2Fnchembio.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Warr, W. A.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">453</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1007/s10822-009-9292-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs10822-009-9292-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=453-8&author=W.+A.+Warr&title=Fragment-based+drug+discovery&doi=10.1007%2Fs10822-009-9292-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9292-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9292-1%26sid%3Dliteratum%253Aachs%26aulast%3DWarr%26aufirst%3DW.%2BA.%26atitle%3DFragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D453%26epage%3D8%26doi%3D10.1007%2Fs10822-009-9292-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tisi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span> </span><span class="NLM_article-title">Recent developments in fragment-based drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3661</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1021/jm8000373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8000373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFKisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3661-80&author=M.+Congreveauthor=G.+Chessariauthor=D.+Tisiauthor=A.+J.+Woodhead&title=Recent+developments+in+fragment-based+drug+discovery&doi=10.1021%2Fjm8000373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Recent developments in fragment-based drug discovery</span></div><div class="casAuthors">Congreve, Miles; Chessari, Gianni; Tisi, Dominic; Woodhead, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3661-3680</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The field of fragment-based drug discovery (FBDD) has developed significantly over the past 10 years and is now recognized as a tangible alternative to more traditional methods of hit identification, such as high throughput screening (HTS).  The authors discuss trends and developments of FBDD over the past 4 years.  Addnl., this review looks in some detail at representative protein-ligand complexes obsd. between fragment-sized mols. and their protein targets from the Protein Data Bank (PDB).  More than the underlying technol. itself, the legacy of FBDD will be conceptual and that "less is more"; breaking down the problem of finding a drug mol. into its smallest component parts opens up new opportunities to solve challenging problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_YtOS7_glD7Vg90H21EOLACvtfcHk0liyVYVWnbWJpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFKisLg%253D&md5=41dbcf884305a190bffc683ae5ed9e45</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm8000373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8000373%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DTisi%26aufirst%3DD.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26atitle%3DRecent%2520developments%2520in%2520fragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3661%26epage%3D80%26doi%3D10.1021%2Fjm8000373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Carr, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery: Leads by design</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03511-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS1359-6446%2805%2903511-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16023057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=987-92&author=R.+A.+Carrauthor=M.+Congreveauthor=C.+W.+Murrayauthor=D.+C.+Rees&title=Fragment-based+lead+discovery%3A+Leads+by+design&doi=10.1016%2FS1359-6446%2805%2903511-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery: leads by design</span></div><div class="casAuthors">Carr, Robin A. E.; Congreve, Miles; Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">987-992</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery (also referred to as needles, shapes, binding elements, seed templates or scaffolds) is a new lead discovery approach in which much lower mol. wt. (120-250Da) compds. are screened relative to HTS campaigns.  Fragment-based hits are typically weak inhibitors (10μM-mM), and therefore need to be screened at higher concn. using very sensitive biophys. detection techniques such as protein crystallog. and NMR as the primary screening techniques, rather than bioassays.  Compared with HTS hits, these fragments are simpler, less functionalized compds. with correspondingly lower affinity.  However, fragment hits typically possess high ligand efficiency' (binding affinity per heavy atom) and so are highly suitable for optimization into clin. candidates with good drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq15FNhjT6Qw7Vg90H21EOLACvtfcHk0liyVYVWnbWJpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D&md5=1bf7dd97ec056af9ed5143f96b34bb53</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903511-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903511-7%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DR.%2BA.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-based%2520lead%2520discovery%253A%2520Leads%2520by%2520design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D987%26epage%3D92%26doi%3D10.1016%2FS1359-6446%2805%2903511-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Villar, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chennamadhavuni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porco, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">How proteins bind macrocycles</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">723</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-31&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lhE7yN3C5f1IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D31%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Ganesan, A.</span><span> </span><span class="NLM_article-title">The impact of natural products upon modern drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">306</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2008.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.cbpa.2008.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18423384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ersrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=306-17&author=A.+Ganesan&title=The+impact+of+natural+products+upon+modern+drug+discovery&doi=10.1016%2Fj.cbpa.2008.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of natural products upon modern drug discovery</span></div><div class="casAuthors">Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">306-317</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In the period 1970-2006, a total of 24 unique natural products were discovered that led to an approved drug.  We analyze these successful leads in terms of drug-like properties, and show that they can be divided into two equal subsets.  The first falls in the Lipinski universe' and complies with the Rule of Five.  The second is a parallel universe' that violates the rules.  Nevertheless, the latter compds. remain largely compliant in terms of log P and H-bond donors, highlighting the importance of these two metrics in predicting bioavailability.  Natural products are often cited as an exception to Lipinski's rules.  We believe this is because nature has learned to maintain low hydrophobicity and intermol. H-bond donating potential when it needs to make biol. active compds. with high mol. wt. and large nos. of rotatable bonds.  In addn., natural products are more likely than purely synthetic compds. to resemble biosynthetic intermediates or endogenous metabolites, and hence take advantage of active transport mechanisms.  Interestingly, the natural product leads in the Lipinski and parallel universe had an identical success rate (50%) in delivering an oral drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXd8x0Vy0yerVg90H21EOLACvtfcHk0lgH1G0uETNQuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ersrk%253D&md5=8cdde276c59d84071ea2d6f572d10088</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DThe%2520impact%2520of%2520natural%2520products%2520upon%2520modern%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26spage%3D306%26epage%3D17%26doi%3D10.1016%2Fj.cbpa.2008.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Goodman, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shandler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeGrado, W. F.</span><span> </span><span class="NLM_article-title">Foldamers as versatile frameworks for the design and evolution of function</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1038/nchembio876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchembio876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17438550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1Kjs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=252-262&author=C.+M.+Goodmanauthor=S.+Choiauthor=S.+Shandlerauthor=W.+F.+DeGrado&title=Foldamers+as+versatile+frameworks+for+the+design+and+evolution+of+function&doi=10.1038%2Fnchembio876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Foldamers as versatile frameworks for the design and evolution of function</span></div><div class="casAuthors">Goodman, Catherine M.; Choi, Sungwook; Shandler, Scott; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">252-262</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Foldamers are sequence-specific oligomers akin to peptides, proteins and oligonucleotides that fold into well-defined three-dimensional structures.  They offer the chem. biologist a broad pallet of building blocks for the construction of mols. that test and extend our understanding of protein folding and function.  Foldamers also provide templates for presenting complex arrays of functional groups in virtually unlimited geometrical patterns, thereby presenting attractive opportunities for the design of mols. that bind in a sequence- and structure-specific manner to oligosaccharides, nucleic acids, membranes and proteins.  We summarize recent advances and highlight the future applications and challenges of this rapidly expanding field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLkvabxBqULVg90H21EOLACvtfcHk0lgH1G0uETNQuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1Kjs7w%253D&md5=99f0e4b2bb7a8457a586c74f0f98dbb3</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnchembio876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio876%26sid%3Dliteratum%253Aachs%26aulast%3DGoodman%26aufirst%3DC.%2BM.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DShandler%26aufirst%3DS.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DFoldamers%2520as%2520versatile%2520frameworks%2520for%2520the%2520design%2520and%2520evolution%2520of%2520function%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D252%26epage%3D262%26doi%3D10.1038%2Fnchembio876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Zuckermann, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodadek, T.</span><span> </span><span class="NLM_article-title">Peptoids as potential therapeutics</span> <span class="citation_source-journal">Curr. Opin. Mol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19479663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnt1Kgu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=299-307&author=R.+N.+Zuckermannauthor=T.+Kodadek&title=Peptoids+as+potential+therapeutics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Peptoids as potential therapeutics</span></div><div class="casAuthors">Zuckermann, Ronald N.; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Molecular Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">299-307</span>CODEN:
                <span class="NLM_cas:coden">CUOTFO</span>;
        ISSN:<span class="NLM_cas:issn">2040-3445</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptoids are oligomers of N-substituted glycine units.  These mols. are almost perfectly suited for combinatorial approaches to drug discovery because large libraries can be synthesized easily from readily available primary amines.  Moreover, major advances in screening methodol. have allowed peptoid libraries of hundreds of thousands of compds. to be mined inexpensively and quickly for highly specific protein-binding mols.  These advances and the potential utility of peptoids as pharmacol. agents are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-Fv-86dP_ubVg90H21EOLACvtfcHk0lgH1G0uETNQuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnt1Kgu7k%253D&md5=fce62a95317bf10116f7ed5ab8754b88</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZuckermann%26aufirst%3DR.%2BN.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DPeptoids%2520as%2520potential%2520therapeutics%26jtitle%3DCurr.%2520Opin.%2520Mol.%2520Ther.%26date%3D2009%26volume%3D11%26spage%3D299%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Dietrich, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durr, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, J.</span><span> </span><span class="NLM_article-title">Peptides as drugs: From screening to application</span> <span class="citation_source-journal">Curr. Pharm. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">501</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.2174/13892010113149990205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.2174%2F13892010113149990205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22429133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlGgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=501-12&author=U.+Dietrichauthor=R.+Durrauthor=J.+Koch&title=Peptides+as+drugs%3A+From+screening+to+application&doi=10.2174%2F13892010113149990205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Peptides as drugs: from screening to application</span></div><div class="casAuthors">Dietrich, Ursula; Duerr, Ralf; Koch, Joachim</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Biotechnology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">501-512</span>CODEN:
                <span class="NLM_cas:coden">CPBUBP</span>;
        ISSN:<span class="NLM_cas:issn">1389-2010</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptides are ideally suited to mimic natural ligands and thereby function in an antagonistic or agonistic way.  Furthermore they are able to physiol. disrupt functional complexes due to their small size and specific binding properties.  Proteins form homo- or heteromeric (macro)mol. complexes and intricate networks by interacting with small mols., peptides, nucleic acids or other proteins.  On av., five interaction partners have been estd. for any given cellular protein, illustrating the complexity of the formed "interactomes" and the impact of their investigation.  Many protein-protein interactions are mediated by hot-spots, which comprise only a small part of the large binding interface but account for 80% of the binding energy.  Thus, these hot spots provide an "Achilles heel" for pharmaceutical interventions aiming at the disruption of functional protein-protein complexes.  Methods to select peptides for defined target structures include display technologies on phages, ribosomes or yeast, and the yeast-two-hybrid system.  Once selected, these peptides can be optimized for their binding affinity using peptide arrays.  Stabilization of biol. unstable peptides is achieved by the introduction of non-natural amino acids to form so-called peptidomimetics that are resistant to cellular proteases.  Moreover, lipocalins and peptide aptamers represent scaffolded binding structures with unique binding characteristics and enhanced stability.  In case of extracellular targets, like cell surface receptors or pathogens in patients' plasma, peptide inhibitors have direct access.  Addressing intracellular targets with peptides is more difficult since short hydrophilic peptides generally do not cross plasma membranes on their own.  However, intracellular uptake of peptides can be achieved by coupling to carrier systems like liposomes or nanoparticles or upon fusion to a protein transduction domain.  Alternatively, peptides may be expressed within cells after transduction with viral vectors in a gene therapy setting.  This review will summarize the broad potential of peptides as drugs, with special emphasis on peptides which inhibit protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr10DvrrcgEhrVg90H21EOLACvtfcHk0lgH1G0uETNQuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlGgtA%253D%253D&md5=b63d94225b8fa4a7fc749b8631794544</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.2174%2F13892010113149990205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13892010113149990205%26sid%3Dliteratum%253Aachs%26aulast%3DDietrich%26aufirst%3DU.%26aulast%3DDurr%26aufirst%3DR.%26aulast%3DKoch%26aufirst%3DJ.%26atitle%3DPeptides%2520as%2520drugs%253A%2520From%2520screening%2520to%2520application%26jtitle%3DCurr.%2520Pharm.%2520Biotechnol.%26date%3D2013%26volume%3D14%26spage%3D501%26epage%3D12%26doi%3D10.2174%2F13892010113149990205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Wilson, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sintchak, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotstein, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgibbon, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navia, M. A.</span><span> </span><span class="NLM_article-title">Comparative x-ray structures of the major binding-protein for the immunosuppressant FK506 (tacrolimus) in unliganded form and in complex with FK506 and rapamycin</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444994014514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1107%2FS0907444994014514" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1995&pages=511-21&author=K.+P.+Wilsonauthor=M.+M.+Yamashitaauthor=M.+D.+Sintchakauthor=S.+H.+Rotsteinauthor=M.+A.+Murckoauthor=J.+Bogerauthor=J.+A.+Thomsonauthor=M.+J.+Fitzgibbonauthor=J.+R.+Blackauthor=M.+A.+Navia&title=Comparative+x-ray+structures+of+the+major+binding-protein+for+the+immunosuppressant+FK506+%28tacrolimus%29+in+unliganded+form+and+in+complex+with+FK506+and+rapamycin&doi=10.1107%2FS0907444994014514"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444994014514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994014514%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DK.%2BP.%26aulast%3DYamashita%26aufirst%3DM.%2BM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DRotstein%26aufirst%3DS.%2BH.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26aulast%3DBoger%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DJ.%2BA.%26aulast%3DFitzgibbon%26aufirst%3DM.%2BJ.%26aulast%3DBlack%26aufirst%3DJ.%2BR.%26aulast%3DNavia%26aufirst%3DM.%2BA.%26atitle%3DComparative%2520x-ray%2520structures%2520of%2520the%2520major%2520binding-protein%2520for%2520the%2520immunosuppressant%2520FK506%2520%2528tacrolimus%2529%2520in%2520unliganded%2520form%2520and%2520in%2520complex%2520with%2520FK506%2520and%2520rapamycin%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1995%26volume%3D51%26spage%3D511%26epage%3D21%26doi%3D10.1107%2FS0907444994014514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Sattler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettesheim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harlan, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberstadt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minn, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Structure of Bcl-x(L)-BAK peptide complex: Recognition between regulators of apoptosis</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1126/science.275.5302.983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1126%2Fscience.275.5302.983" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=1997&pages=983-6&author=M.+Sattlerauthor=H.+Liangauthor=D.+Nettesheimauthor=R.+P.+Meadowsauthor=J.+E.+Harlanauthor=M.+Eberstadtauthor=H.+S.+Yoonauthor=S.+B.+Shukerauthor=B.+S.+Changauthor=A.+J.+Minnauthor=C.+B.+Thompsonauthor=S.+W.+Fesik&title=Structure+of+Bcl-x%28L%29-BAK+peptide+complex%3A+Recognition+between+regulators+of+apoptosis&doi=10.1126%2Fscience.275.5302.983"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1126%2Fscience.275.5302.983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.275.5302.983%26sid%3Dliteratum%253Aachs%26aulast%3DSattler%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DNettesheim%26aufirst%3DD.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DHarlan%26aufirst%3DJ.%2BE.%26aulast%3DEberstadt%26aufirst%3DM.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DChang%26aufirst%3DB.%2BS.%26aulast%3DMinn%26aufirst%3DA.%2BJ.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DStructure%2520of%2520Bcl-x%2528L%2529-BAK%2520peptide%2520complex%253A%2520Recognition%2520between%2520regulators%2520of%2520apoptosis%26jtitle%3DScience%26date%3D1997%26volume%3D275%26spage%3D983%26epage%3D6%26doi%3D10.1126%2Fscience.275.5302.983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Holt, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luengo, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konialian, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozamus, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bossard, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggleston, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stout, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clardy, J.</span><span> </span><span class="NLM_article-title">Design, synthesis, and kinetic evaluation of high-affinity FKBP ligands and the X-ray crystal-structures of their complexes with FKBP12</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">9925</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1021/ja00075a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00075a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1993&pages=9925-38&author=D.+A.+Holtauthor=J.+I.+Luengoauthor=D.+S.+Yamashitaauthor=H.+J.+Ohauthor=A.+L.+Konialianauthor=H.+K.+Yenauthor=L.+W.+Rozamusauthor=M.+Brandtauthor=M.+J.+Bossardauthor=M.+A.+Levyauthor=D.+S.+Egglestonauthor=J.+Liangauthor=L.+W.+Schultzauthor=T.+J.+Stoutauthor=J.+Clardy&title=Design%2C+synthesis%2C+and+kinetic+evaluation+of+high-affinity+FKBP+ligands+and+the+X-ray+crystal-structures+of+their+complexes+with+FKBP12&doi=10.1021%2Fja00075a008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fja00075a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00075a008%26sid%3Dliteratum%253Aachs%26aulast%3DHolt%26aufirst%3DD.%2BA.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DOh%26aufirst%3DH.%2BJ.%26aulast%3DKonialian%26aufirst%3DA.%2BL.%26aulast%3DYen%26aufirst%3DH.%2BK.%26aulast%3DRozamus%26aufirst%3DL.%2BW.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DBossard%26aufirst%3DM.%2BJ.%26aulast%3DLevy%26aufirst%3DM.%2BA.%26aulast%3DEggleston%26aufirst%3DD.%2BS.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DSchultz%26aufirst%3DL.%2BW.%26aulast%3DStout%26aufirst%3DT.%2BJ.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520kinetic%2520evaluation%2520of%2520high-affinity%2520FKBP%2520ligands%2520and%2520the%2520X-ray%2520crystal-structures%2520of%2520their%2520complexes%2520with%2520FKBP12%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1993%26volume%3D115%26spage%3D9925%26epage%3D38%26doi%3D10.1021%2Fja00075a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Sich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Improta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowley, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guenet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merly, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teufel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saudek, V.</span><span> </span><span class="NLM_article-title">Solution structure of a neurotrophic ligand bound to FKBP12 and its effects on protein dynamics</span> <span class="citation_source-journal">Eur. J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">5342</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span><span class="refDoi"> DOI: 10.1046/j.1432-1327.2000.01551.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1046%2Fj.1432-1327.2000.01551.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10951192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsVens7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=2000&pages=5342-54&author=C.+Sichauthor=S.+Improtaauthor=D.+J.+Cowleyauthor=C.+Guenetauthor=J.+P.+Merlyauthor=M.+Teufelauthor=V.+Saudek&title=Solution+structure+of+a+neurotrophic+ligand+bound+to+FKBP12+and+its+effects+on+protein+dynamics&doi=10.1046%2Fj.1432-1327.2000.01551.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Solution structure of a neurotrophic ligand bound to FKBP12 and its effects on protein dynamics</span></div><div class="casAuthors">Sich, Christian; Improta, Sabina; Cowley, David John; Guenet, Chantal; Merly, Jean-Pierre; Teufel, Michael; Saudek, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5342-5354</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">The structure of a recently reported neurotrophic ligand, 3-(3-pyridyl)-1-propyl(2S)-1-(3,3-dimethyl-1,2-dioxopentyl)-2-pyrrolidinecarboxylate, in complex with FKBP12 was detd. using heteronuclear NMR spectroscopy.  The inhibitor exhibits a binding mode analogous to that obsd. for the macrocycle FK506, used widely as an immunosuppressant, with the prolyl ring replacing the pipecolyl moiety and the amide bond in a trans conformation.  However, fewer favorable protein-ligand interactions are detected in the structure of the complex, suggesting weaker binding compared with the immunosuppressant drug.  Indeed, a micromolar dissocn. const. was estd. from the NMR ligand titrn. profile, in contrast to the previously published nanomolar inhibition activity.  Although the inhibitor possesses a remarkable structural simplicity with respect to FK506, 15N relaxation studies show that it induces similar effects on the protein dynamics, stabilizing the conformation of solvent-exposed residues which are important for mediating the interaction of immunophilin/ligand complexes with mol. targets and potentially for the transmission of the neurotrophic action of FKBP12 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ47HUpCawurVg90H21EOLACvtfcHk0lhqFi4t-S_fSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsVens7Y%253D&md5=3a97a261948a25f3e9b62cd389788863</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1046%2Fj.1432-1327.2000.01551.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1432-1327.2000.01551.x%26sid%3Dliteratum%253Aachs%26aulast%3DSich%26aufirst%3DC.%26aulast%3DImprota%26aufirst%3DS.%26aulast%3DCowley%26aufirst%3DD.%2BJ.%26aulast%3DGuenet%26aufirst%3DC.%26aulast%3DMerly%26aufirst%3DJ.%2BP.%26aulast%3DTeufel%26aufirst%3DM.%26aulast%3DSaudek%26aufirst%3DV.%26atitle%3DSolution%2520structure%2520of%2520a%2520neurotrophic%2520ligand%2520bound%2520to%2520FKBP12%2520and%2520its%2520effects%2520on%2520protein%2520dynamics%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D2000%26volume%3D267%26spage%3D5342%26epage%3D54%26doi%3D10.1046%2Fj.1432-1327.2000.01551.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Silberstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kortvelyesi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clodfelter, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Identification of substrate binding sites in enzymes by computational solvent mapping</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">1095</span><span class="NLM_x">–</span> <span class="NLM_lpage">113</span><span class="refDoi"> DOI: 10.1016/j.jmb.2003.08.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jmb.2003.08.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2003&pages=1095-113&author=M.+Silbersteinauthor=S.+Dennisauthor=L.+Brownauthor=T.+Kortvelyesiauthor=K.+Clodfelterauthor=S.+Vajda&title=Identification+of+substrate+binding+sites+in+enzymes+by+computational+solvent+mapping&doi=10.1016%2Fj.jmb.2003.08.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2003.08.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2003.08.019%26sid%3Dliteratum%253Aachs%26aulast%3DSilberstein%26aufirst%3DM.%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DL.%26aulast%3DKortvelyesi%26aufirst%3DT.%26aulast%3DClodfelter%26aufirst%3DK.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520substrate%2520binding%2520sites%2520in%2520enzymes%2520by%2520computational%2520solvent%2520mapping%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D332%26spage%3D1095%26epage%3D113%26doi%3D10.1016%2Fj.jmb.2003.08.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Evidence of conformational selection driving the formation of ligand binding sites in protein-protein interfaces</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003872</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003872</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1371%2Fjournal.pcbi.1003872" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003872&author=T.+Bohnuudauthor=D.+Kozakovauthor=S.+Vajda&title=Evidence+of+conformational+selection+driving+the+formation+of+ligand+binding+sites+in+protein-protein+interfaces&doi=10.1371%2Fjournal.pcbi.1003872"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003872%26sid%3Dliteratum%253Aachs%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DEvidence%2520of%2520conformational%2520selection%2520driving%2520the%2520formation%2520of%2520ligand%2520binding%2520sites%2520in%2520protein-protein%2520interfaces%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003872%26doi%3D10.1371%2Fjournal.pcbi.1003872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2977</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1021/jm030580l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030580l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2977-80&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=S.+Wang&title=The+PDBbind+database%3A+Collection+of+binding+affinities+for+protein-ligand+complexes+with+known+three-dimensional+structures&doi=10.1021%2Fjm030580l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span></div><div class="casAuthors">Wang, Renxiao; Fang, Xueliang; Lu, Yipin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2977-2980</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have screened the entire Protein Data Bank (Release No. 103, Jan. 2003) and identified 5671 protein-ligand complexes out of 19 621 exptl. structures.  A systematic examn. of the primary refs. of these entries has led to a collection of binding affinity data (Kd, Ki, and IC50) for a total of 1359 complexes.  The outcomes of this project have been organized into a Web-accessible database named the PDBbind database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1eUdeWioYBbVg90H21EOLACvtfcHk0lhqFi4t-S_fSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D&md5=86e609172307402d8b0d4589b1270a2f</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjm030580l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030580l%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520Collection%2520of%2520binding%2520affinities%2520for%2520protein-ligand%2520complexes%2520with%2520known%2520three-dimensional%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2977%26epage%3D80%26doi%3D10.1021%2Fjm030580l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorissen, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">BindingDB: A web-accessible database of experimentally determined protein-ligand binding affinities</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">D198</span><span class="NLM_x">–</span> <span class="NLM_lpage">201</span><span class="refDoi"> DOI: 10.1093/nar/gkl999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fnar%2Fgkl999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17145705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+A+web-accessible+database+of+experimentally+determined+protein-ligand+binding+affinities&doi=10.1093%2Fnar%2Fgkl999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span></div><div class="casAuthors">Liu, Tiqing; Lin, Yuhmei; Wen, Xin; Jorissen, Robert N.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D198-D201</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BindingDB is a publicly accessible database currently contg. ∼20 000 exptl. detd. binding affinities of protein-ligand complexes, for 110 protein targets including isoforms and mutational variants, and ∼11 000 small mol. ligands.  The data are extd. from the scientific literature, data collection focusing on proteins that are drug-targets or candidate drug-targets and for which structural data are present in the Protein Data Bank.  The BindingDB website supports a range of query types, including searches by chem. structure, substructure and similarity; protein sequence; ligand and protein names; affinity ranges and mol. wt.  Data sets generated by BindingDB queries can be downloaded in the form of annotated SDfiles for further anal., or used as the basis for virtual screening of a compd. database uploaded by the user.  The data in BindingDB are linked both to structural data in the PDB via PDB IDs and chem. and sequence searches, and to the literature in PubMed via PubMed IDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoymy3G_LuG-rVg90H21EOLACvtfcHk0ljjoCFidGD4AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D&md5=0ccb20d9b9178a624d4829b5909e7ff8</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520A%2520web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein-ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3D201%26doi%3D10.1093%2Fnar%2Fgkl999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Benson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khazanov, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimcheff, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaver, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dresslar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerothin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Binding MOAD, a high-quality protein-ligand database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D674</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1093/nar/gkm911</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fnar%2Fgkm911" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18055497" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2007&pages=D674-8&author=M.+L.+Bensonauthor=R.+D.+Smithauthor=N.+A.+Khazanovauthor=B.+Dimcheffauthor=J.+Beaverauthor=P.+Dresslarauthor=J.+Nerothinauthor=H.+A.+Carlson&title=Binding+MOAD%2C+a+high-quality+protein-ligand+database&doi=10.1093%2Fnar%2Fgkm911"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm911%26sid%3Dliteratum%253Aachs%26aulast%3DBenson%26aufirst%3DM.%2BL.%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DKhazanov%26aufirst%3DN.%2BA.%26aulast%3DDimcheff%26aufirst%3DB.%26aulast%3DBeaver%26aufirst%3DJ.%26aulast%3DDresslar%26aufirst%3DP.%26aulast%3DNerothin%26aufirst%3DJ.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DBinding%2520MOAD%252C%2520a%2520high-quality%2520protein-ligand%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D36%26spage%3DD674%26epage%3D8%26doi%3D10.1093%2Fnar%2Fgkm911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Topalis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumamoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, M. F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugue, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haouz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexandre, J. A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallois-Montbrun, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alzari, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pochet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrofoglio, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deville-Bonne, D.</span><span> </span><span class="NLM_article-title">Nucleotide binding to human UMP-CMP kinase using fluorescent derivatives - a screening based on affinity for the UMP-CMP binding site</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">3704</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span><span class="refDoi"> DOI: 10.1111/j.1742-4658.2007.05902.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1111%2Fj.1742-4658.2007.05902.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=2007&pages=3704-14&author=D.+Topalisauthor=H.+Kumamotoauthor=M.+F.+A.+Velascoauthor=L.+Dugueauthor=A.+Haouzauthor=J.+A.+C.+Alexandreauthor=S.+Gallois-Montbrunauthor=P.+M.+Alzariauthor=S.+Pochetauthor=L.+A.+Agrofoglioauthor=D.+Deville-Bonne&title=Nucleotide+binding+to+human+UMP-CMP+kinase+using+fluorescent+derivatives+-+a+screening+based+on+affinity+for+the+UMP-CMP+binding+site&doi=10.1111%2Fj.1742-4658.2007.05902.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2007.05902.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2007.05902.x%26sid%3Dliteratum%253Aachs%26aulast%3DTopalis%26aufirst%3DD.%26aulast%3DKumamoto%26aufirst%3DH.%26aulast%3DVelasco%26aufirst%3DM.%2BF.%2BA.%26aulast%3DDugue%26aufirst%3DL.%26aulast%3DHaouz%26aufirst%3DA.%26aulast%3DAlexandre%26aufirst%3DJ.%2BA.%2BC.%26aulast%3DGallois-Montbrun%26aufirst%3DS.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26aulast%3DPochet%26aufirst%3DS.%26aulast%3DAgrofoglio%26aufirst%3DL.%2BA.%26aulast%3DDeville-Bonne%26aufirst%3DD.%26atitle%3DNucleotide%2520binding%2520to%2520human%2520UMP-CMP%2520kinase%2520using%2520fluorescent%2520derivatives%2520-%2520a%2520screening%2520based%2520on%2520affinity%2520for%2520the%2520UMP-CMP%2520binding%2520site%26jtitle%3DFEBS%2520J.%26date%3D2007%26volume%3D274%26spage%3D3704%26epage%3D14%26doi%3D10.1111%2Fj.1742-4658.2007.05902.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Mashhoon, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pruss, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reich, N. O.</span><span> </span><span class="NLM_article-title">Selective inhibitors of bacterial DNA adenine methyltransferases</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">497</span><span class="NLM_x">–</span> <span class="NLM_lpage">510</span><span class="refDoi"> DOI: 10.1177/1087057106287933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1177%2F1087057106287933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16760373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XpslWjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=497-510&author=N.+Mashhoonauthor=C.+Prussauthor=M.+Carrollauthor=P.+H.+Johnsonauthor=N.+O.+Reich&title=Selective+inhibitors+of+bacterial+DNA+adenine+methyltransferases&doi=10.1177%2F1087057106287933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of bacterial DNA adenine methyltransferases</span></div><div class="casAuthors">Mashhoon, Neda; Pruss, Cynthia; Carroll, Michael; Johnson, Paul H.; Reich, Norbert O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">497-510</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The authors describe the discovery and characterization of several structural classes of small-mol. inhibitors of bacterial DNA adenine methyltransferases.  These enzymes are essential for bacterial virulence (DNA adenine methyltransferase [DAM]) and cell viability (cell cycle-regulated methyltransferase [CcrM]).  Using a novel high-throughput fluorescence-based assay and recombinant DAM and CcrM, the authors screened a diverse chem. library.  They identified 5 major structural classes of inhibitors composed of more than 350 compds.: cyclopentaquinolines, Ph vinyl furans, pyrimidine-diones, thiazolidine-4-ones, and phenyl-pyrroles.  DNA binding assays were used to identify compds. that interact directly with DNA.  Potent compds. selective for the bacterial target were identified, whereas other compds. showed greater selectivity for the mammalian DNA cytosine methyltransferase, Dnmt1.  Enzyme inhibition anal. identified mechanistically distinct compds. that interfered with DNA or cofactor binding.  Selected compds. demonstrated cell-based efficacy.  These small-mol. DNA methyltransferase inhibitors provide useful reagents to probe the role of DNA methylation and may form the basis of developing novel antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob8jGoRQNSi7Vg90H21EOLACvtfcHk0lh5jIk3cWdb8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpslWjtbo%253D&md5=41e230dae5dae50a1b5a4881d882be7f</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1177%2F1087057106287933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106287933%26sid%3Dliteratum%253Aachs%26aulast%3DMashhoon%26aufirst%3DN.%26aulast%3DPruss%26aufirst%3DC.%26aulast%3DCarroll%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DP.%2BH.%26aulast%3DReich%26aufirst%3DN.%2BO.%26atitle%3DSelective%2520inhibitors%2520of%2520bacterial%2520DNA%2520adenine%2520methyltransferases%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D497%26epage%3D510%26doi%3D10.1177%2F1087057106287933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Miao, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skidan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugovskoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reibarkh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brazell, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durugkar, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramana, C. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaffhausen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torchilin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degterev, A.</span><span> </span><span class="NLM_article-title">Small molecule inhibition of phosphatidylinositol-3,4,5-triphosphate (pip3) binding to pleckstrin homology domains</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">20126</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1073/pnas.1004522107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1004522107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21041639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyntL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=20126-31&author=B.+C.+Miaoauthor=I.+Skidanauthor=J.+S.+Yangauthor=A.+Lugovskoyauthor=M.+Reibarkhauthor=K.+Longauthor=T.+Brazellauthor=K.+A.+Durugkarauthor=J.+Makiauthor=C.+V.+Ramanaauthor=B.+Schaffhausenauthor=G.+Wagnerauthor=V.+Torchilinauthor=J.+Y.+Yuanauthor=A.+Degterev&title=Small+molecule+inhibition+of+phosphatidylinositol-3%2C4%2C5-triphosphate+%28pip3%29+binding+to+pleckstrin+homology+domains&doi=10.1073%2Fpnas.1004522107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of phosphatidylinositol-3,4,5-triphosphate (PIP3) binding to pleckstrin homology domains</span></div><div class="casAuthors">Miao, Benchun; Skidan, Igor; Yang, Jinsheng; Lugovskoy, Alexey; Reibarkh, Mikhail; Long, Kai; Brazell, Tres; Durugkar, Kulbhushan A.; Maki, Jenny; Ramana, C. V.; Schaffhausen, Brian; Wagner, Gerhard; Torchilin, Vladimir; Yuan, Junying; Degterev, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">20126-20131, S20126/1-S20126/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The PI3-kinase (PI3K) pathway regulates many cellular processes, esp. cell metab., cell survival, and apoptosis.  Phosphatidylinositol-3,4,5-trisphosphate (PIP3), the product of PI3K activity and a key signaling mol., acts by recruiting pleckstrin-homol. (PH) domain-contg. proteins to cell membranes.  Here, the authors describe a new structural class of nonphosphoinositide small mol. antagonists (PITenins, PITs) of PIP3-PH domain interactions (IC50 ranges from 13.4 to 31 μM in PIP3/Akt PH domain binding assay).  PITs inhibit interactions of a no. of PIP3-binding PH domains, including those of Akt and PDK1, without affecting several PIP2-selective PH domains.  As a result, PITs suppress the PI3K-PDK1-Akt pathway and trigger metabolic stress and apoptosis.  A PIT-1 analog displayed significant antitumor activity in vivo, including inhibition of tumor growth and induction of apoptosis.  Overall, the authors' studies demonstrate the feasibility of developing specific small mol. antagonists of PIP3 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq68GGEQg_fsrVg90H21EOLACvtfcHk0lh5jIk3cWdb8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyntL3P&md5=b2bc440fd7c9fae24de7f6e8c4910d86</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1004522107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1004522107%26sid%3Dliteratum%253Aachs%26aulast%3DMiao%26aufirst%3DB.%2BC.%26aulast%3DSkidan%26aufirst%3DI.%26aulast%3DYang%26aufirst%3DJ.%2BS.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DReibarkh%26aufirst%3DM.%26aulast%3DLong%26aufirst%3DK.%26aulast%3DBrazell%26aufirst%3DT.%26aulast%3DDurugkar%26aufirst%3DK.%2BA.%26aulast%3DMaki%26aufirst%3DJ.%26aulast%3DRamana%26aufirst%3DC.%2BV.%26aulast%3DSchaffhausen%26aufirst%3DB.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DTorchilin%26aufirst%3DV.%26aulast%3DYuan%26aufirst%3DJ.%2BY.%26aulast%3DDegterev%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520phosphatidylinositol-3%252C4%252C5-triphosphate%2520%2528pip3%2529%2520binding%2520to%2520pleckstrin%2520homology%2520domains%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D20126%26epage%3D31%26doi%3D10.1073%2Fpnas.1004522107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Lerner, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagenaar, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodall, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Searle, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florjancic, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mack, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chovan, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCall, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black-Schaefer, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakavas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schurdak, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comess, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edalji, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorwin, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebert, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harlan, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merta, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baranowski, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornewell, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shivakumar, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saiki, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bui, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balli, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilius, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holzman, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beutel, B. A.</span><span> </span><span class="NLM_article-title">From bacterial genomes to novel antibacterial agents: Discovery, characterization, and antibacterial activity of compounds that bind to HI0065 (YjeE) from Haemophilus influenzae</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">404</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00521.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1111%2Fj.1747-0285.2007.00521.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17581233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotV2js7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=395-404&author=C.+G.+Lernerauthor=P.+J.+Hajdukauthor=R.+Wagnerauthor=F.+L.+Wagenaarauthor=C.+Woodallauthor=Y.+G.+Guauthor=X.+B.+Searleauthor=A.+S.+Florjancicauthor=T.+Zhangauthor=R.+F.+Clarkauthor=C.+S.+Cooperauthor=J.+C.+Mackauthor=L.+Yuauthor=M.+Caiauthor=S.+F.+Betzauthor=L.+E.+Chovanauthor=J.+O.+McCallauthor=C.+L.+Black-Schaeferauthor=S.+J.+Kakavasauthor=M.+E.+Schurdakauthor=K.+M.+Comessauthor=K.+A.+Walterauthor=R.+Edaljiauthor=S.+A.+Dorwinauthor=R.+A.+Smithauthor=E.+J.+Hebertauthor=J.+E.+Harlanauthor=R.+E.+Metzgerauthor=P.+J.+Mertaauthor=J.+L.+Baranowskiauthor=M.+L.+Coenauthor=S.+J.+Thornewellauthor=A.+G.+Shivakumarauthor=A.+Y.+Saikiauthor=N.+Soniauthor=M.+Buiauthor=D.+J.+Balliauthor=W.+J.+Sandersauthor=A.+M.+Niliusauthor=T.+F.+Holzmanauthor=S.+W.+Fesikauthor=B.+A.+Beutel&title=From+bacterial+genomes+to+novel+antibacterial+agents%3A+Discovery%2C+characterization%2C+and+antibacterial+activity+of+compounds+that+bind+to+HI0065+%28YjeE%29+from+Haemophilus+influenzae&doi=10.1111%2Fj.1747-0285.2007.00521.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">From bacterial genomes to novel antibacterial agents: discovery, characterization, and antibacterial activity of compounds that bind to HI0065 (YjeE) from Haemophilus influenzae</span></div><div class="casAuthors">Lerner, Claude G.; Hajduk, Philip J.; Wagner, Rolf; Wagenaar, Frank L.; Woodall, Charlotte; Gu, Yu-Gui; Searle, Xenia B.; Florjancic, Alan S.; Zhang, Tianyuan; Clark, Richard F.; Cooper, Curt S.; Mack, Jamey C.; Yu, Liping; Cai, Mengli; Betz, Steven F.; Chovan, Linda E.; McCall, J. Owen; Black-Schaefer, Candace L.; Kakavas, Stephan J.; Schurdak, Mark E.; Comess, Kenneth M.; Walter, Karl A.; Edalji, Rohinton; Dorwin, Sarah A.; Smith, Richard A.; Hebert, Eric J.; Harlan, John E.; Metzger, Randy E.; Merta, Philip J.; Baranowski, John L.; Coen, Michael L.; Thornewell, Susan J.; Shivakumar, Annapur G.; Saiki, Anne Y.; Soni, Niru; Bui, Mai; Balli, Darlene J.; Sanders, William J.; Nilius, Angela M.; Holzman, Thomas F.; Fesik, Stephen W.; Beutel, Bruce A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">395-404</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">As part of a fully integrated and comprehensive strategy to discover novel antibacterial agents, NMR- and mass spectrometry-based affinity selection screens were performed to identify compds. that bind to protein targets uniquely found in bacteria and encoded by genes essential for microbial viability.  A biphenyl acid lead series emerged from an NMR-based screen with the Haemophilus influenzae protein HI0065, a member of a family of probable ATP-binding proteins found exclusively in eubacteria.  The structure-activity relationships developed around the NMR-derived biphenyl acid lead were consistent with on-target antibacterial activity as the Staphylococcus aureus antibacterial activity of the series correlated extremely well with binding affinity to HI0065, while the correlation of binding affinity with B-cell cytotoxicity was relatively poor.  Although further studies are needed to conclusively establish the mode of action of the biphenyl series, these compds. represent novel leads that can serve as the basis for the development of novel antibacterial agents that appear to work via an unprecedented mechanism of action.  Overall, these results support the genomics-driven hypothesis that targeting bacterial essential gene products that are not present in eukaryotic cells can identify novel antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoS-_IBLD7VLVg90H21EOLACvtfcHk0ljY2qnMbCMCMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotV2js7w%253D&md5=169a289013053ac9cbbe3578f3447299</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00521.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00521.x%26sid%3Dliteratum%253Aachs%26aulast%3DLerner%26aufirst%3DC.%2BG.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DWagner%26aufirst%3DR.%26aulast%3DWagenaar%26aufirst%3DF.%2BL.%26aulast%3DWoodall%26aufirst%3DC.%26aulast%3DGu%26aufirst%3DY.%2BG.%26aulast%3DSearle%26aufirst%3DX.%2BB.%26aulast%3DFlorjancic%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DClark%26aufirst%3DR.%2BF.%26aulast%3DCooper%26aufirst%3DC.%2BS.%26aulast%3DMack%26aufirst%3DJ.%2BC.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DM.%26aulast%3DBetz%26aufirst%3DS.%2BF.%26aulast%3DChovan%26aufirst%3DL.%2BE.%26aulast%3DMcCall%26aufirst%3DJ.%2BO.%26aulast%3DBlack-Schaefer%26aufirst%3DC.%2BL.%26aulast%3DKakavas%26aufirst%3DS.%2BJ.%26aulast%3DSchurdak%26aufirst%3DM.%2BE.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DWalter%26aufirst%3DK.%2BA.%26aulast%3DEdalji%26aufirst%3DR.%26aulast%3DDorwin%26aufirst%3DS.%2BA.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DHebert%26aufirst%3DE.%2BJ.%26aulast%3DHarlan%26aufirst%3DJ.%2BE.%26aulast%3DMetzger%26aufirst%3DR.%2BE.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DBaranowski%26aufirst%3DJ.%2BL.%26aulast%3DCoen%26aufirst%3DM.%2BL.%26aulast%3DThornewell%26aufirst%3DS.%2BJ.%26aulast%3DShivakumar%26aufirst%3DA.%2BG.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DSoni%26aufirst%3DN.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DBalli%26aufirst%3DD.%2BJ.%26aulast%3DSanders%26aufirst%3DW.%2BJ.%26aulast%3DNilius%26aufirst%3DA.%2BM.%26aulast%3DHolzman%26aufirst%3DT.%2BF.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DBeutel%26aufirst%3DB.%2BA.%26atitle%3DFrom%2520bacterial%2520genomes%2520to%2520novel%2520antibacterial%2520agents%253A%2520Discovery%252C%2520characterization%252C%2520and%2520antibacterial%2520activity%2520of%2520compounds%2520that%2520bind%2520to%2520HI0065%2520%2528YjeE%2529%2520from%2520Haemophilus%2520influenzae%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D395%26epage%3D404%26doi%3D10.1111%2Fj.1747-0285.2007.00521.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Piserchio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salinas, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spaller, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mierke, D. F.</span><span> </span><span class="NLM_article-title">Targeting specific PDZ domains of PSD-95: Structural basis for enhanced affinity and enzymatic stability of a cyclic peptide</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2004.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.chembiol.2004.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15123241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1Cmsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=469-73&author=A.+Piserchioauthor=G.+D.+Salinasauthor=T.+Liauthor=J.+Marshallauthor=M.+R.+Spallerauthor=D.+F.+Mierke&title=Targeting+specific+PDZ+domains+of+PSD-95%3A+Structural+basis+for+enhanced+affinity+and+enzymatic+stability+of+a+cyclic+peptide&doi=10.1016%2Fj.chembiol.2004.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Specific PDZ Domains of PSD-95 Structural Basis for Enhanced Affinity and Enzymatic Stability of a Cyclic Peptide</span></div><div class="casAuthors">Piserchio, Andrea; Salinas, Gregory D.; Li, Tao; Marshall, John; Spaller, Mark R.; Mierke, Dale F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-473</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A cyclic peptide, Tyr-Lys-c[-Lys-Thr-Glu(βAla)-]-Val, incorporating a β-Ala lactam side chain linker and designed to target the PDZ domains of the postsynaptic d. protein 95 (PSD-95), has been synthesized and structurally characterized by NMR while free and bound to the PDZ1 domain of PSD-95.  While bound, the lactam linker of the peptide makes a no. of unique contacts outside the canonical PDZ binding motif, providing a novel target for PDZ-domain specificity as well as producing a 10-fold enhancement in binding affinity.  Addnl., the cyclization greatly enhances the enzymic stability, increasing the duration that the peptide inhibits the assocn. between PSD-95 and glutamate receptors, effectively inhibiting the clustering of kainate receptors for over 14 h after application.  Highly specific regulation of kainate receptor action may provide a novel route for treatment of drug addiction and epilepsy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRejrBkqJELLVg90H21EOLACvtfcHk0ljY2qnMbCMCMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1Cmsro%253D&md5=4ee388e7183bdc94fb024eeb98fa7193</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DPiserchio%26aufirst%3DA.%26aulast%3DSalinas%26aufirst%3DG.%2BD.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMarshall%26aufirst%3DJ.%26aulast%3DSpaller%26aufirst%3DM.%2BR.%26aulast%3DMierke%26aufirst%3DD.%2BF.%26atitle%3DTargeting%2520specific%2520PDZ%2520domains%2520of%2520PSD-95%253A%2520Structural%2520basis%2520for%2520enhanced%2520affinity%2520and%2520enzymatic%2520stability%2520of%2520a%2520cyclic%2520peptide%26jtitle%3DChem.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D469%26epage%3D73%26doi%3D10.1016%2Fj.chembiol.2004.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Wintjens, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wieruszeski, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drobecq, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rousselot-Pailley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippens, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landrieu, I.</span><span> </span><span class="NLM_article-title">1H NMR study on the binding of Pin1 Trp-Trp domain with phosphothreonine peptides</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">25150</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1074/jbc.M010327200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1074%2Fjbc.M010327200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11313338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlt1agu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=25150-6&author=R.+Wintjensauthor=J.+M.+Wieruszeskiauthor=H.+Drobecqauthor=P.+Rousselot-Pailleyauthor=L.+Bueeauthor=G.+Lippensauthor=I.+Landrieu&title=1H+NMR+study+on+the+binding+of+Pin1+Trp-Trp+domain+with+phosphothreonine+peptides&doi=10.1074%2Fjbc.M010327200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">H NMR study on the binding of Pin1 Trp-Trp domain with phosphothreonine peptides</span></div><div class="casAuthors">Wintjens, Rene; Wieruszeski, Jean-Michel; Drobecq, Herve; Rousselot-Pailley, Pierre; Buee, Luc; Lippens, Guy; Landrieu, Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">25150-25156</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The recent crystal structure of Pin1 protein bound to a doubly phosphorylated peptide from the C-terminal domain of RNA polymerase II revealed that binding interactions between Pin1 and its substrate take place through its Trp-Trp (WW) domain at the level of the loop Ser11-Arg12 and the arom. pair Tyr18-Trp29, and showed a trans conformation for both pSer-Pro peptide bonds.  However, the orientation of the ligand in the arom. recognition groove still could be sequence-specific, as previously obsd. in SH3 domains complexed by peptide ligands or for different class of WW domains.  Here the ligand orientation and conformation for two biol. relevant monophosphate substrates, one derived from the Cdc25 phosphatase of Xenopus laevis (EQPLpTPVTDL) and another from the human tau protein (KVSVVRpTPPKSPS), in complex with the WW domain, are studied by soln. NMR methods.  First, the proton resonance perturbations on the WW domain upon complexation with both peptide ligands were detd. to be essentially located in the pos. charged β-hairpin Ser11-Gly15 and around the arom. Trp29.  Dissocn. equil. consts. of 117 and 230 μM for Cdc25 and tau peptides, resp., were found.  Several intermol. nuclear Overhauser effects between WW domain and substrates were obtained from a ligand-satd. soln. and were used to det. the structures of the complexes in soln.  We found a similar N to C orientation as the one obsd. in the crystal complex structure of Pin1 and a trans conformation for the pThr-Pro peptidic bond in both peptide ligands, thereby indicating a unique binding scheme for the Pin1 WW domain to its multiple substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw3qepIthuFrVg90H21EOLACvtfcHk0ljY2qnMbCMCMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlt1agu7s%253D&md5=55561e3f96c88d5577c1f765d5613a0c</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M010327200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M010327200%26sid%3Dliteratum%253Aachs%26aulast%3DWintjens%26aufirst%3DR.%26aulast%3DWieruszeski%26aufirst%3DJ.%2BM.%26aulast%3DDrobecq%26aufirst%3DH.%26aulast%3DRousselot-Pailley%26aufirst%3DP.%26aulast%3DBuee%26aufirst%3DL.%26aulast%3DLippens%26aufirst%3DG.%26aulast%3DLandrieu%26aufirst%3DI.%26atitle%3D1H%2520NMR%2520study%2520on%2520the%2520binding%2520of%2520Pin1%2520Trp-Trp%2520domain%2520with%2520phosphothreonine%2520peptides%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D25150%26epage%3D6%26doi%3D10.1074%2Fjbc.M010327200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Guo, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dagostino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greasley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferre, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marakovits, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mroczkowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parge, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">VanArsdale, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popoff, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piraino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margosiak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Los, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Structure-based design of novel human Pin1 inhibitors (I)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">5613</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.08.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2009.08.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19729306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFakur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=5613-6&author=C.+X.+Guoauthor=X.+J.+Houauthor=L.+M.+Dongauthor=E.+Dagostinoauthor=S.+Greasleyauthor=R.+Ferreauthor=J.+Marakovitsauthor=M.+C.+Johnsonauthor=D.+Matthewsauthor=B.+Mroczkowskiauthor=H.+Pargeauthor=T.+VanArsdaleauthor=I.+Popoffauthor=J.+Pirainoauthor=S.+Margosiakauthor=J.+Thomsonauthor=G.+Losauthor=B.+W.+Murray&title=Structure-based+design+of+novel+human+Pin1+inhibitors+%28I%29&doi=10.1016%2Fj.bmcl.2009.08.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of novel human Pin1 inhibitors (I)</span></div><div class="casAuthors">Guo, Chuangxing; Hou, Xinjun; Dong, Liming; Dagostino, Eleanor; Greasley, Samantha; Ferre, RoseAnn; Marakovits, Joseph; Johnson, M. Catherine; Matthews, David; Mroczkowski, Barbara; Parge, Hans; Van Arsdale, Todd; Popoff, Ian; Piraino, Joseph; Margosiak, Stephen; Thomson, James; Los, Gerrit; Murray, Brion W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5613-5616</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Pin1 is a member of the cis-trans peptidyl-prolyl isomerase family with potential anti-cancer therapeutic value.  Structure-based de novo design and optimization of novel Pin1 inhibitors, e.g., I, are reported.  Without a viable lead from internal screenings, a series of novel Pin1 inhibitors by interrogating and exploring a protein crystal structure of Pin1 is designed.  The ligand efficiency of the initial concept mol. was optimized with integrated SBDD and parallel chem. approaches, resulting in a more attractive lead series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy51a1X9SYC7Vg90H21EOLACvtfcHk0ljD92XC43iCKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFakur%252FM&md5=6fbb8e918fd39293dc419dfa0747a937</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.08.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.08.034%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DC.%2BX.%26aulast%3DHou%26aufirst%3DX.%2BJ.%26aulast%3DDong%26aufirst%3DL.%2BM.%26aulast%3DDagostino%26aufirst%3DE.%26aulast%3DGreasley%26aufirst%3DS.%26aulast%3DFerre%26aufirst%3DR.%26aulast%3DMarakovits%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DM.%2BC.%26aulast%3DMatthews%26aufirst%3DD.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DParge%26aufirst%3DH.%26aulast%3DVanArsdale%26aufirst%3DT.%26aulast%3DPopoff%26aufirst%3DI.%26aulast%3DPiraino%26aufirst%3DJ.%26aulast%3DMargosiak%26aufirst%3DS.%26aulast%3DThomson%26aufirst%3DJ.%26aulast%3DLos%26aufirst%3DG.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DStructure-based%2520design%2520of%2520novel%2520human%2520Pin1%2520inhibitors%2520%2528I%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D5613%26epage%3D6%26doi%3D10.1016%2Fj.bmcl.2009.08.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Dong, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marakovits, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greasley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dagostino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferre, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraynov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pathak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Structure-based design of novel human Pin1 inhibitors (II)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2210</span><span class="NLM_x">–</span> <span class="NLM_lpage">4</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.02.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2010.02.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=20207139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslGmt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=2210-4&author=L.+M.+Dongauthor=J.+Marakovitsauthor=X.+J.+Houauthor=C.+X.+Guoauthor=S.+Greasleyauthor=E.+Dagostinoauthor=R.+Ferreauthor=M.+C.+Johnsonauthor=E.+Kraynovauthor=J.+Thomsonauthor=V.+Pathakauthor=B.+W.+Murray&title=Structure-based+design+of+novel+human+Pin1+inhibitors+%28II%29&doi=10.1016%2Fj.bmcl.2010.02.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of novel human Pin1 inhibitors (II)</span></div><div class="casAuthors">Dong, Liming; Marakovits, Joseph; Hou, Xinjun; Guo, Chuangxing; Greasley, Samantha; Dagostino, Eleanor; Ferre, RoseAnn; Johnson, M. Catherine; Kraynov, Eugenia; Thomson, James; Pathak, Ved; Murray, Brion W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2210-2214</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Following the discovery of a novel series of phosphate-contg. small mol. Pin1 inhibitors, the drug design strategy shifted to replacement of the phosphate group with an isostere with potential better pharmaceutical properties.  The initial loss in potency of carboxylate analogs was likely due to weaker charge-charge interactions in the putative phosphate binding pocket and was subsequently recovered by structure-based optimization of ligand-protein interactions in the proline binding site, leading to the discovery of a sub-micromolar non-phosphate small mol. Pin1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgilRHkGBuYbVg90H21EOLACvtfcHk0ljD92XC43iCKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslGmt7s%253D&md5=d16f5beec12a17a38bca7e84e09ddb0d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.02.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.02.033%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DL.%2BM.%26aulast%3DMarakovits%26aufirst%3DJ.%26aulast%3DHou%26aufirst%3DX.%2BJ.%26aulast%3DGuo%26aufirst%3DC.%2BX.%26aulast%3DGreasley%26aufirst%3DS.%26aulast%3DDagostino%26aufirst%3DE.%26aulast%3DFerre%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DM.%2BC.%26aulast%3DKraynov%26aufirst%3DE.%26aulast%3DThomson%26aufirst%3DJ.%26aulast%3DPathak%26aufirst%3DV.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DStructure-based%2520design%2520of%2520novel%2520human%2520Pin1%2520inhibitors%2520%2528II%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D2210%26epage%3D4%26doi%3D10.1016%2Fj.bmcl.2010.02.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Guo, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marakovits, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greasley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dagostino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferre, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paderes, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piraino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraynov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Structure-based design of novel human Pin1 inhibitors (III): Optimizing affinity beyond the phosphate recognition pocket</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">4187</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.07.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2014.07.044" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=4187-91&author=C.+X.+Guoauthor=X.+J.+Houauthor=L.+M.+Dongauthor=J.+Marakovitsauthor=S.+Greasleyauthor=E.+Dagostinoauthor=R.+Ferreauthor=M.+C.+Johnsonauthor=P.+S.+Humphriesauthor=H.+T.+Liauthor=G.+D.+Paderesauthor=J.+Pirainoauthor=E.+Kraynovauthor=B.+W.+Murray&title=Structure-based+design+of+novel+human+Pin1+inhibitors+%28III%29%3A+Optimizing+affinity+beyond+the+phosphate+recognition+pocket&doi=10.1016%2Fj.bmcl.2014.07.044"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.07.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.07.044%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DC.%2BX.%26aulast%3DHou%26aufirst%3DX.%2BJ.%26aulast%3DDong%26aufirst%3DL.%2BM.%26aulast%3DMarakovits%26aufirst%3DJ.%26aulast%3DGreasley%26aufirst%3DS.%26aulast%3DDagostino%26aufirst%3DE.%26aulast%3DFerre%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DM.%2BC.%26aulast%3DHumphries%26aufirst%3DP.%2BS.%26aulast%3DLi%26aufirst%3DH.%2BT.%26aulast%3DPaderes%26aufirst%3DG.%2BD.%26aulast%3DPiraino%26aufirst%3DJ.%26aulast%3DKraynov%26aufirst%3DE.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DStructure-based%2520design%2520of%2520novel%2520human%2520Pin1%2520inhibitors%2520%2528III%2529%253A%2520Optimizing%2520affinity%2520beyond%2520the%2520phosphate%2520recognition%2520pocket%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D4187%26epage%3D91%26doi%3D10.1016%2Fj.bmcl.2014.07.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Wendt, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarris, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoff, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettesheim, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comess, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaac, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olejniczak, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span> </span><span class="NLM_article-title">Discovery of a novel small molecule binding site of human survivin</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">3122</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.03.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2007.03.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17391963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1Kisr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=3122-9&author=M.+D.+Wendtauthor=C.+H.+Sunauthor=A.+Kunzerauthor=D.+Sauerauthor=K.+Sarrisauthor=E.+Hoffauthor=L.+P.+Yuauthor=D.+G.+Nettesheimauthor=J.+Chenauthor=S.+Jinauthor=K.+M.+Comessauthor=Y.+H.+Fanauthor=S.+N.+Andersonauthor=B.+Isaacauthor=E.+T.+Olejniczakauthor=P.+J.+Hajdukauthor=S.+H.+Rosenbergauthor=S.+W.+Elmore&title=Discovery+of+a+novel+small+molecule+binding+site+of+human+survivin&doi=10.1016%2Fj.bmcl.2007.03.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel small molecule binding site of human survivin</span></div><div class="casAuthors">Wendt, Michael D.; Sun, Chaohong; Kunzer, Aaron; Sauer, Daryl; Sarris, Kathy; Hoff, Ethan; Yu, Liping; Nettesheim, David G.; Chen, Jun; Jin, Sha; Comess, Kenneth M.; Fan, Yihong; Anderson, Steven N.; Isaac, Binumol; Olejniczak, Edward T.; Hajduk, Philip J.; Rosenberg, Saul H.; Elmore, Steven W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3122-3129</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Survivin is one of the most tumor-specific genes in the human genome and is an attractive target for cancer therapy.  However, small-mol. ligands for survivin have not yet been described.  Thus, an interrogation of survivin which could potentially both validate a small-mol. therapy approach, and det. the biochem. nature of any of survivin's functions has not been possible.  Here we describe the discovery and characterization of a small mol. binding site on the survivin surface distinct from the Smac peptide-binding site.  The new site is located at the dimer interface and exhibits many of the features of highly drugable, biol. relevant protein binding sites.  A variety of small hydrophobic compds. were found that bind with moderate affinity to this binding site, from which one lead was developed into a group of compds. with nanomolar affinity.  Addnl., a subset of these compds. are adequately water-sol. and cell-permeable.  Thus, the structural studies and small mols. described here provide tools that can be used to probe the biochem. role(s) of survivin, and may ultimately serve as a basis for the development of small mol. therapeutics acting via direct or allosteric disruption of binding events related to this poorly understood target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU2KNydbvLSrVg90H21EOLACvtfcHk0liE43lGE_JuCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1Kisr4%253D&md5=41e0c01dbef616c9620d1303b9135b33</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.03.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.03.042%26sid%3Dliteratum%253Aachs%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DSun%26aufirst%3DC.%2BH.%26aulast%3DKunzer%26aufirst%3DA.%26aulast%3DSauer%26aufirst%3DD.%26aulast%3DSarris%26aufirst%3DK.%26aulast%3DHoff%26aufirst%3DE.%26aulast%3DYu%26aufirst%3DL.%2BP.%26aulast%3DNettesheim%26aufirst%3DD.%2BG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DFan%26aufirst%3DY.%2BH.%26aulast%3DAnderson%26aufirst%3DS.%2BN.%26aulast%3DIsaac%26aufirst%3DB.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520small%2520molecule%2520binding%2520site%2520of%2520human%2520survivin%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D3122%26epage%3D9%26doi%3D10.1016%2Fj.bmcl.2007.03.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mack, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendoza, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaac, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braddock, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comess, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clore, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levens, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">NMR-driven discovery of benzoylanthranilic acid inhibitors of far upstream element binding protein binding to the human oncogene c-myc promoter</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4851</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1021/jm0497803</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0497803" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4851-7&author=J.+R.+Huthauthor=L.+Yuauthor=I.+Collinsauthor=J.+Mackauthor=R.+Mendozaauthor=B.+Isaacauthor=D.+T.+Braddockauthor=S.+W.+Muchmoreauthor=K.+M.+Comessauthor=S.+W.+Fesikauthor=G.+M.+Cloreauthor=D.+Levensauthor=P.+J.+Hajduk&title=NMR-driven+discovery+of+benzoylanthranilic+acid+inhibitors+of+far+upstream+element+binding+protein+binding+to+the+human+oncogene+c-myc+promoter&doi=10.1021%2Fjm0497803"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fjm0497803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0497803%26sid%3Dliteratum%253Aachs%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DMack%26aufirst%3DJ.%26aulast%3DMendoza%26aufirst%3DR.%26aulast%3DIsaac%26aufirst%3DB.%26aulast%3DBraddock%26aufirst%3DD.%2BT.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DLevens%26aufirst%3DD.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNMR-driven%2520discovery%2520of%2520benzoylanthranilic%2520acid%2520inhibitors%2520of%2520far%2520upstream%2520element%2520binding%2520protein%2520binding%2520to%2520the%2520human%2520oncogene%2520c-myc%2520promoter%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4851%26epage%3D7%26doi%3D10.1021%2Fjm0497803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinges, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miknis, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merlock, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kempf, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egan, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robins, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holzman, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">NMR-based discovery of lead inhibitors that block DNA binding of the human papillomavirus E2 protein</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">3144</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1021/jm9703404</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9703404" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=3144-50&author=P.+J.+Hajdukauthor=J.+Dingesauthor=G.+F.+Miknisauthor=M.+Merlockauthor=T.+Middletonauthor=D.+J.+Kempfauthor=D.+A.+Eganauthor=K.+A.+Walterauthor=T.+S.+Robinsauthor=S.+B.+Shukerauthor=T.+F.+Holzmanauthor=S.+W.+Fesik&title=NMR-based+discovery+of+lead+inhibitors+that+block+DNA+binding+of+the+human+papillomavirus+E2+protein&doi=10.1021%2Fjm9703404"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Fjm9703404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9703404%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DDinges%26aufirst%3DJ.%26aulast%3DMiknis%26aufirst%3DG.%2BF.%26aulast%3DMerlock%26aufirst%3DM.%26aulast%3DMiddleton%26aufirst%3DT.%26aulast%3DKempf%26aufirst%3DD.%2BJ.%26aulast%3DEgan%26aufirst%3DD.%2BA.%26aulast%3DWalter%26aufirst%3DK.%2BA.%26aulast%3DRobins%26aufirst%3DT.%2BS.%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHolzman%26aufirst%3DT.%2BF.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DNMR-based%2520discovery%2520of%2520lead%2520inhibitors%2520that%2520block%2520DNA%2520binding%2520of%2520the%2520human%2520papillomavirus%2520E2%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D3144%26epage%3D50%26doi%3D10.1021%2Fjm9703404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Schaal, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mallet, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMinn, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sopko, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parekh, B. S.</span><span> </span><span class="NLM_article-title">Inhibition of human papilloma virus E2 DNA binding protein by covalently linked polyamides</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1282</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1093/nar/gkg206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fnar%2Fgkg206" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=1282-91&author=T.+D.+Schaalauthor=W.+G.+Malletauthor=D.+L.+McMinnauthor=N.+V.+Nguyenauthor=M.+M.+Sopkoauthor=S.+Johnauthor=B.+S.+Parekh&title=Inhibition+of+human+papilloma+virus+E2+DNA+binding+protein+by+covalently+linked+polyamides&doi=10.1093%2Fnar%2Fgkg206"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkg206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkg206%26sid%3Dliteratum%253Aachs%26aulast%3DSchaal%26aufirst%3DT.%2BD.%26aulast%3DMallet%26aufirst%3DW.%2BG.%26aulast%3DMcMinn%26aufirst%3DD.%2BL.%26aulast%3DNguyen%26aufirst%3DN.%2BV.%26aulast%3DSopko%26aufirst%3DM.%2BM.%26aulast%3DJohn%26aufirst%3DS.%26aulast%3DParekh%26aufirst%3DB.%2BS.%26atitle%3DInhibition%2520of%2520human%2520papilloma%2520virus%2520E2%2520DNA%2520binding%2520protein%2520by%2520covalently%2520linked%2520polyamides%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2003%26volume%3D31%26spage%3D1282%26epage%3D91%26doi%3D10.1093%2Fnar%2Fgkg206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">Kurg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langel, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ustav, M.</span><span> </span><span class="NLM_article-title">Inhibition of the bovine papillomavirus E2 protein activity by peptide nucleic acid</span> <span class="citation_source-journal">Virus Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">39</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/S0168-1702(99)00124-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0168-1702%2899%2900124-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10653916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVGntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2000&pages=39-50&author=R.+Kurgauthor=U.+Langelauthor=M.+Ustav&title=Inhibition+of+the+bovine+papillomavirus+E2+protein+activity+by+peptide+nucleic+acid&doi=10.1016%2FS0168-1702%2899%2900124-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the bovine papillomavirus E2 protein activity by peptide nucleic acid</span></div><div class="casAuthors">Kurg, R.; Langel, U.; Ustav, M.</div><div class="citationInfo"><span class="NLM_cas:title">Virus Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-50</span>CODEN:
                <span class="NLM_cas:coden">VIREDF</span>;
        ISSN:<span class="NLM_cas:issn">0168-1702</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">The bovine papillomavirus type-1 E2 protein is the master regulator of the papillomavirus transcription and replication, the activity of which is regulated through sequence-specific DNA binding.  Peptide nucleic acid (PNA) is a nucleic acid analog, which assocs. with high affinity to complementary DNA, RNA or PNA, yielding in formation of stable complexes.  The potential use of PNA as a sequence-specific inhibitor of the E2 protein activity is studied in this report.  We demonstrate that replacement of one or both DNA strands with the complementary PNA reduced drastically the affinity of the BPV-1 E2 protein to its target site in the direct as well as in competitive binding as shown by in vitro gel-shift assays.  We demonstrate that PNA could specifically bind to the double stranded E2 binding site by forming the complex with DNA oligonucleotide.  In addn., PNA was able to bind specifically to the E2 binding site within the supercoiled plasmid DNA.  Such binding of PNA to the E2 binding site within the origin of replication specifically abolished the activity of the E2 protein in the initiation of DNA replication in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiSLoO0Q9PRbVg90H21EOLACvtfcHk0lhumZq9gdCcsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVGntw%253D%253D&md5=d22afa222edda70df90448c4dfb42fb3</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2FS0168-1702%2899%2900124-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0168-1702%252899%252900124-0%26sid%3Dliteratum%253Aachs%26aulast%3DKurg%26aufirst%3DR.%26aulast%3DLangel%26aufirst%3DU.%26aulast%3DUstav%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520the%2520bovine%2520papillomavirus%2520E2%2520protein%2520activity%2520by%2520peptide%2520nucleic%2520acid%26jtitle%3DVirus%2520Res.%26date%3D2000%26volume%3D66%26spage%3D39%26epage%3D50%26doi%3D10.1016%2FS0168-1702%2899%2900124-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">de Dios, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prieto, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ezquerra, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tebbe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez de Uralde, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeTourneau, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGee, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boylan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, T. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M. C.</span><span> </span><span class="NLM_article-title">4-substituted D-glutamic acid analogues: The first potent inhibitors of glutamate racemase (MurI) enzyme with antibacterial activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">4559</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1021/jm020901d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020901d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=4559-70&author=A.+de+Diosauthor=L.+Prietoauthor=J.+A.+Martinauthor=A.+Rubioauthor=J.+Ezquerraauthor=M.+Tebbeauthor=B.+Lopez+de+Uraldeauthor=J.+Martinauthor=A.+Sanchezauthor=D.+L.+LeTourneauauthor=J.+E.+McGeeauthor=C.+Boylanauthor=T.+R.+Parrauthor=M.+C.+Smith&title=4-substituted+D-glutamic+acid+analogues%3A+The+first+potent+inhibitors+of+glutamate+racemase+%28MurI%29+enzyme+with+antibacterial+activity&doi=10.1021%2Fjm020901d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Fjm020901d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020901d%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BDios%26aufirst%3DA.%26aulast%3DPrieto%26aufirst%3DL.%26aulast%3DMartin%26aufirst%3DJ.%2BA.%26aulast%3DRubio%26aufirst%3DA.%26aulast%3DEzquerra%26aufirst%3DJ.%26aulast%3DTebbe%26aufirst%3DM.%26aulast%3DLopez%2Bde%2BUralde%26aufirst%3DB.%26aulast%3DMartin%26aufirst%3DJ.%26aulast%3DSanchez%26aufirst%3DA.%26aulast%3DLeTourneau%26aufirst%3DD.%2BL.%26aulast%3DMcGee%26aufirst%3DJ.%2BE.%26aulast%3DBoylan%26aufirst%3DC.%26aulast%3DParr%26aufirst%3DT.%2BR.%26aulast%3DSmith%26aufirst%3DM.%2BC.%26atitle%3D4-substituted%2520D-glutamic%2520acid%2520analogues%253A%2520The%2520first%2520potent%2520inhibitors%2520of%2520glutamate%2520racemase%2520%2528MurI%2529%2520enzyme%2520with%2520antibacterial%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D4559%26epage%3D70%26doi%3D10.1021%2Fjm020901d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group">Geng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breault, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comita-Prevoir, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrichko, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyermann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundqvist, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doig, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorseth, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noonan, B.</span><span> </span><span class="NLM_article-title">Exploring 9-benzyl purines as inhibitors of glutamate racemase (MurI) gram-positive bacteria</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4368</span><span class="NLM_x">–</span> <span class="NLM_lpage">4372</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.06.068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2008.06.068" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4368-4372&author=B.+Gengauthor=G.+Breaultauthor=J.+Comita-Prevoirauthor=R.+Petrichkoauthor=C.+Eyermannauthor=T.+Lundqvistauthor=P.+Doigauthor=E.+Gorsethauthor=B.+Noonan&title=Exploring+9-benzyl+purines+as+inhibitors+of+glutamate+racemase+%28MurI%29+gram-positive+bacteria&doi=10.1016%2Fj.bmcl.2008.06.068"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.06.068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.06.068%26sid%3Dliteratum%253Aachs%26aulast%3DGeng%26aufirst%3DB.%26aulast%3DBreault%26aufirst%3DG.%26aulast%3DComita-Prevoir%26aufirst%3DJ.%26aulast%3DPetrichko%26aufirst%3DR.%26aulast%3DEyermann%26aufirst%3DC.%26aulast%3DLundqvist%26aufirst%3DT.%26aulast%3DDoig%26aufirst%3DP.%26aulast%3DGorseth%26aufirst%3DE.%26aulast%3DNoonan%26aufirst%3DB.%26atitle%3DExploring%25209-benzyl%2520purines%2520as%2520inhibitors%2520of%2520glutamate%2520racemase%2520%2528MurI%2529%2520gram-positive%2520bacteria%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4368%26epage%3D4372%26doi%3D10.1016%2Fj.bmcl.2008.06.068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group">Breault, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comita-Prevoir, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyermann, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrichko, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doig, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorseth, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noonan, B.</span><span> </span><span class="NLM_article-title">Exploring 8-benzyl pteridine-6,7-diones as inhibitors of glutamate racemase (MurI) in gram-positive bacteria</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6100</span><span class="NLM_x">–</span> <span class="NLM_lpage">6103</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.10.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2008.10.022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=6100-6103&author=G.+A.+Breaultauthor=J.+Comita-Prevoirauthor=C.+J.+Eyermannauthor=B.+Gengauthor=R.+Petrichkoauthor=P.+Doigauthor=E.+Gorsethauthor=B.+Noonan&title=Exploring+8-benzyl+pteridine-6%2C7-diones+as+inhibitors+of+glutamate+racemase+%28MurI%29+in+gram-positive+bacteria&doi=10.1016%2Fj.bmcl.2008.10.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.10.022%26sid%3Dliteratum%253Aachs%26aulast%3DBreault%26aufirst%3DG.%2BA.%26aulast%3DComita-Prevoir%26aufirst%3DJ.%26aulast%3DEyermann%26aufirst%3DC.%2BJ.%26aulast%3DGeng%26aufirst%3DB.%26aulast%3DPetrichko%26aufirst%3DR.%26aulast%3DDoig%26aufirst%3DP.%26aulast%3DGorseth%26aufirst%3DE.%26aulast%3DNoonan%26aufirst%3DB.%26atitle%3DExploring%25208-benzyl%2520pteridine-6%252C7-diones%2520as%2520inhibitors%2520of%2520glutamate%2520racemase%2520%2528MurI%2529%2520in%2520gram-positive%2520bacteria%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D6100%26epage%3D6103%26doi%3D10.1016%2Fj.bmcl.2008.10.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group">McRee, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegel, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodin, D. B.</span><span> </span><span class="NLM_article-title">Construction of a bisaquo heme enzyme and binding by exogenous ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">12847</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.1073/pnas.91.26.12847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.91.26.12847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=7809133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK2MXivVCqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=12847-51&author=D.+E.+McReeauthor=G.+M.+Jensenauthor=M.+M.+Fitzgeraldauthor=H.+A.+Siegelauthor=D.+B.+Goodin&title=Construction+of+a+bisaquo+heme+enzyme+and+binding+by+exogenous+ligands&doi=10.1073%2Fpnas.91.26.12847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Construction of a bisaquo heme enzyme and binding by exogenous ligands</span></div><div class="casAuthors">McRee, D. E.; Jensen, G. M.; Fitzgerald, M. M.; Siegel, H. A.; Goodin, D. B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">12847-51</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The crystal structure of the H175G mutant of cytochrome c peroxidase (EC 1.11.1.5), missing its only heme ligand, reveals that the His is replaced by solvent to give a bisaquo hemoprotein.  This protein retains some residual activity, which can be stimulated or inhibited by addn. of exogenous ligands.  Structural anal. of the H175G-imidazole complex confirms the binding of imidazole to the heme at the position of the wild-type His ligand.  This imidazole complex reacts readily with H2O2 to produce a radical species with novel properties.  However, reactivation in this complex is incomplete (∼5%), which, in view of the very similar structures of the wild-type and the H175G/imidazole forms, implies a crit. role for tethering of the axial ligand in catalysis.  This study demonstrates the feasibility of constructing heme enzymes with no covalent link to the protein and with unnatural ligand replacements.  Such enzymes may prove useful in studies of electron transfer mechanisms and in the engineering of novel heme-based catalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrArqgvJ4hBpbVg90H21EOLACvtfcHk0ljaupES52xSIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXivVCqsrY%253D&md5=cbb7f5f14beeda947c29b021a861c4db</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1073%2Fpnas.91.26.12847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.91.26.12847%26sid%3Dliteratum%253Aachs%26aulast%3DMcRee%26aufirst%3DD.%2BE.%26aulast%3DJensen%26aufirst%3DG.%2BM.%26aulast%3DFitzgerald%26aufirst%3DM.%2BM.%26aulast%3DSiegel%26aufirst%3DH.%2BA.%26aulast%3DGoodin%26aufirst%3DD.%2BB.%26atitle%3DConstruction%2520of%2520a%2520bisaquo%2520heme%2520enzyme%2520and%2520binding%2520by%2520exogenous%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1994%26volume%3D91%26spage%3D12847%26epage%3D51%26doi%3D10.1073%2Fpnas.91.26.12847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group">Appleby, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittenberg, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittenberg, J. B.</span><span> </span><span class="NLM_article-title">Nicotinic-acid as a ligand affecting leghemoglobin structure and oxygen reactivity</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">564</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1073/pnas.70.2.564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.70.2.564" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=1973&pages=564-8&author=C.+A.+Applebyauthor=B.+A.+Wittenbergauthor=J.+B.+Wittenberg&title=Nicotinic-acid+as+a+ligand+affecting+leghemoglobin+structure+and+oxygen+reactivity&doi=10.1073%2Fpnas.70.2.564"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1073%2Fpnas.70.2.564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.70.2.564%26sid%3Dliteratum%253Aachs%26aulast%3DAppleby%26aufirst%3DC.%2BA.%26aulast%3DWittenberg%26aufirst%3DB.%2BA.%26aulast%3DWittenberg%26aufirst%3DJ.%2BB.%26atitle%3DNicotinic-acid%2520as%2520a%2520ligand%2520affecting%2520leghemoglobin%2520structure%2520and%2520oxygen%2520reactivity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1973%26volume%3D70%26spage%3D564%26epage%3D8%26doi%3D10.1073%2Fpnas.70.2.564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group">Maldonado, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lostao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irun, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez-Recio, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genzor, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubio, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luquita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daoudi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sancho, J.</span><span> </span><span class="NLM_article-title">Apoflavodoxin: Structure, stability, and FMN binding</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1016/S0300-9084(00)88876-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0300-9084%2800%2988876-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=1998&pages=813-20&author=S.+Maldonadoauthor=A.+Lostaoauthor=M.+P.+Irunauthor=J.+Fernandez-Recioauthor=C.+G.+Genzorauthor=E.+B.+Gonzalezauthor=J.+A.+Rubioauthor=A.+Luquitaauthor=F.+Daoudiauthor=J.+Sancho&title=Apoflavodoxin%3A+Structure%2C+stability%2C+and+FMN+binding&doi=10.1016%2FS0300-9084%2800%2988876-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2FS0300-9084%2800%2988876-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0300-9084%252800%252988876-8%26sid%3Dliteratum%253Aachs%26aulast%3DMaldonado%26aufirst%3DS.%26aulast%3DLostao%26aufirst%3DA.%26aulast%3DIrun%26aufirst%3DM.%2BP.%26aulast%3DFernandez-Recio%26aufirst%3DJ.%26aulast%3DGenzor%26aufirst%3DC.%2BG.%26aulast%3DGonzalez%26aufirst%3DE.%2BB.%26aulast%3DRubio%26aufirst%3DJ.%2BA.%26aulast%3DLuquita%26aufirst%3DA.%26aulast%3DDaoudi%26aufirst%3DF.%26aulast%3DSancho%26aufirst%3DJ.%26atitle%3DApoflavodoxin%253A%2520Structure%252C%2520stability%252C%2520and%2520FMN%2520binding%26jtitle%3DBiochimie%26date%3D1998%26volume%3D80%26spage%3D813%26epage%3D20%26doi%3D10.1016%2FS0300-9084%2800%2988876-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group">Irie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakahara, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yanai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohigashi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wender, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konishi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikkawa, U.</span><span> </span><span class="NLM_article-title">Molecular basis for protein kinase c isozyme-selective binding: The synthesis, folding, and phorbol ester binding of the cysteine-rich domains of all protein kinase c isozymes</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">9159</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1021/ja981087f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja981087f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=9159-67&author=K.+Irieauthor=K.+Oieauthor=A.+Nakaharaauthor=Y.+Yanaiauthor=H.+Ohigashiauthor=P.+A.+Wenderauthor=H.+Fukudaauthor=H.+Konishiauthor=U.+Kikkawa&title=Molecular+basis+for+protein+kinase+c+isozyme-selective+binding%3A+The+synthesis%2C+folding%2C+and+phorbol+ester+binding+of+the+cysteine-rich+domains+of+all+protein+kinase+c+isozymes&doi=10.1021%2Fja981087f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fja981087f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja981087f%26sid%3Dliteratum%253Aachs%26aulast%3DIrie%26aufirst%3DK.%26aulast%3DOie%26aufirst%3DK.%26aulast%3DNakahara%26aufirst%3DA.%26aulast%3DYanai%26aufirst%3DY.%26aulast%3DOhigashi%26aufirst%3DH.%26aulast%3DWender%26aufirst%3DP.%2BA.%26aulast%3DFukuda%26aufirst%3DH.%26aulast%3DKonishi%26aufirst%3DH.%26aulast%3DKikkawa%26aufirst%3DU.%26atitle%3DMolecular%2520basis%2520for%2520protein%2520kinase%2520c%2520isozyme-selective%2520binding%253A%2520The%2520synthesis%252C%2520folding%252C%2520and%2520phorbol%2520ester%2520binding%2520of%2520the%2520cysteine-rich%2520domains%2520of%2520all%2520protein%2520kinase%2520c%2520isozymes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D9159%26epage%3D67%26doi%3D10.1021%2Fja981087f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group">Murray, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cann, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Derrick, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutcliffe, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">Steroid recognition by chloramphenicol acetyltransferase - engineering and structural-analysis of a high-affinity fusidic acid-binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">254</span><span class="NLM_x">, </span> <span class="NLM_fpage">993</span><span class="NLM_x">–</span> <span class="NLM_lpage">1005</span><span class="refDoi"> DOI: 10.1006/jmbi.1995.0671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1006%2Fjmbi.1995.0671" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=254&publication_year=1995&pages=993-1005&author=I.+A.+Murrayauthor=P.+A.+Cannauthor=P.+J.+Dayauthor=J.+P.+Derrickauthor=M.+J.+Sutcliffeauthor=W.+V.+Shawauthor=A.+G.+W.+Leslie&title=Steroid+recognition+by+chloramphenicol+acetyltransferase+-+engineering+and+structural-analysis+of+a+high-affinity+fusidic+acid-binding+site&doi=10.1006%2Fjmbi.1995.0671"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1995.0671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1995.0671%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DI.%2BA.%26aulast%3DCann%26aufirst%3DP.%2BA.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DDerrick%26aufirst%3DJ.%2BP.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26aulast%3DShaw%26aufirst%3DW.%2BV.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DSteroid%2520recognition%2520by%2520chloramphenicol%2520acetyltransferase%2520-%2520engineering%2520and%2520structural-analysis%2520of%2520a%2520high-affinity%2520fusidic%2520acid-binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1995%26volume%3D254%26spage%3D993%26epage%3D1005%26doi%3D10.1006%2Fjmbi.1995.0671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group">Vyas, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vyas, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quiocho, F. A.</span><span> </span><span class="NLM_article-title">Crystallographic analysis of the epimeric and anomeric specificity of the periplasmic transport chemosensory protein-receptor for D-glucose and D-galactose</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">4762</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1021/bi00182a003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00182a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1994&pages=4762-8&author=M.+N.+Vyasauthor=N.+K.+Vyasauthor=F.+A.+Quiocho&title=Crystallographic+analysis+of+the+epimeric+and+anomeric+specificity+of+the+periplasmic+transport+chemosensory+protein-receptor+for+D-glucose+and+D-galactose&doi=10.1021%2Fbi00182a003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fbi00182a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00182a003%26sid%3Dliteratum%253Aachs%26aulast%3DVyas%26aufirst%3DM.%2BN.%26aulast%3DVyas%26aufirst%3DN.%2BK.%26aulast%3DQuiocho%26aufirst%3DF.%2BA.%26atitle%3DCrystallographic%2520analysis%2520of%2520the%2520epimeric%2520and%2520anomeric%2520specificity%2520of%2520the%2520periplasmic%2520transport%2520chemosensory%2520protein-receptor%2520for%2520D-glucose%2520and%2520D-galactose%26jtitle%3DBiochemistry%26date%3D1994%26volume%3D33%26spage%3D4762%26epage%3D8%26doi%3D10.1021%2Fbi00182a003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group">Chung, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fromme, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span> </span><span class="NLM_article-title">Structure of human cytidine deaminase bound to a potent inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1021/jm0496279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0496279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitlGgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=658-60&author=S.+J.+Chungauthor=J.+C.+Frommeauthor=G.+L.+Verdine&title=Structure+of+human+cytidine+deaminase+bound+to+a+potent+inhibitor&doi=10.1021%2Fjm0496279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human Cytidine Deaminase Bound to a Potent Inhibitor</span></div><div class="casAuthors">Chung, Sang J.; Fromme, J. Christopher; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">658-660</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human cytidine deaminase (CDA) is an enzyme prominent for its role in catalyzing metabolic processing of nucleoside-type anticancer and antiviral agents.  It is thus a promising target for the development of small mol. therapeutic adjuvants.  We report the first crystal structure of human CDA as a complex with a tight-binding inhibitor, diazepinone riboside 1.  The structure reveals that inhibitor 1 is able to establish a canonical π/π-interaction with a key active site residue, Phe 137.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3nwv7nFjrpLVg90H21EOLACvtfcHk0limtWdILJpecA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitlGgsQ%253D%253D&md5=51c16f741ee045b974e62fe3cd3d0005</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1021%2Fjm0496279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0496279%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DS.%2BJ.%26aulast%3DFromme%26aufirst%3DJ.%2BC.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DStructure%2520of%2520human%2520cytidine%2520deaminase%2520bound%2520to%2520a%2520potent%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D658%26epage%3D60%26doi%3D10.1021%2Fjm0496279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group">Masaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunasawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakiyama, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soejima, M.</span><span> </span><span class="NLM_article-title">Inhibition of achromobacter protease-I by lysinal derivatives</span> <span class="citation_source-journal">Biosci., Biotechnol., Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1604</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1271/bbb.56.1604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1271%2Fbbb.56.1604" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1992&pages=1604-7&author=T.+Masakiauthor=T.+Tanakaauthor=S.+Tsunasawaauthor=F.+Sakiyamaauthor=M.+Soejima&title=Inhibition+of+achromobacter+protease-I+by+lysinal+derivatives&doi=10.1271%2Fbbb.56.1604"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1271%2Fbbb.56.1604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1271%252Fbbb.56.1604%26sid%3Dliteratum%253Aachs%26aulast%3DMasaki%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DTsunasawa%26aufirst%3DS.%26aulast%3DSakiyama%26aufirst%3DF.%26aulast%3DSoejima%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520achromobacter%2520protease-I%2520by%2520lysinal%2520derivatives%26jtitle%3DBiosci.%252C%2520Biotechnol.%252C%2520Biochem.%26date%3D1992%26volume%3D56%26spage%3D1604%26epage%3D7%26doi%3D10.1271%2Fbbb.56.1604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group">Lunney, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Para, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humblet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fergus, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marks, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawyer, T. K.</span><span> </span><span class="NLM_article-title">Structure-based design of a novel series of nonpeptide ligands that bind to the pp60src SH2 domain</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">12471</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1021/ja971794t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja971794t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=12471-6&author=E.+A.+Lunneyauthor=K.+S.+Paraauthor=J.+R.+Rubinauthor=C.+Humbletauthor=J.+H.+Fergusauthor=J.+S.+Marksauthor=T.+K.+Sawyer&title=Structure-based+design+of+a+novel+series+of+nonpeptide+ligands+that+bind+to+the+pp60src+SH2+domain&doi=10.1021%2Fja971794t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1021%2Fja971794t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja971794t%26sid%3Dliteratum%253Aachs%26aulast%3DLunney%26aufirst%3DE.%2BA.%26aulast%3DPara%26aufirst%3DK.%2BS.%26aulast%3DRubin%26aufirst%3DJ.%2BR.%26aulast%3DHumblet%26aufirst%3DC.%26aulast%3DFergus%26aufirst%3DJ.%2BH.%26aulast%3DMarks%26aufirst%3DJ.%2BS.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520a%2520novel%2520series%2520of%2520nonpeptide%2520ligands%2520that%2520bind%2520to%2520the%2520pp60src%2520SH2%2520domain%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D12471%26epage%3D6%26doi%3D10.1021%2Fja971794t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group">Fauman, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, E. S.</span><span> </span><span class="NLM_article-title">Structure-based druggability assessment--identifying suitable targets for small molecule therapeutics</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2011.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.cbpa.2011.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21704549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFCltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2011&pages=463-8&author=E.+B.+Faumanauthor=B.+K.+Raiauthor=E.+S.+Huang&title=Structure-based+druggability+assessment%2D%2Didentifying+suitable+targets+for+small+molecule+therapeutics&doi=10.1016%2Fj.cbpa.2011.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based druggability assessment - identifying suitable targets for small molecule therapeutics</span></div><div class="casAuthors">Fauman, Eric B.; Rai, Brajesh K.; Huang, Enoch S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">463-468</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A target is druggable if it can be modulated in vivo by a drug-like mol.  The general properties of oral drugs are summarized by the rule of 5' which specifies parameters related to size and lipophilicity.  Structure-based target druggability assessment consists of predicting ligand-binding sites on the protein that are complementary to these drug-like properties.  Automated identification of ligand-binding sites can use geometrical considerations alone or include specific physicochem. properties of the protein surface.  Features of a pocket's size and shape, together with measures of its hydrophobicity, are most informative in identifying suitable drug-binding pockets.  The recent availability of several validation sets of druggable vs. undruggable targets has helped fuel the development of more elaborate methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzQbgkbe44IrVg90H21EOLACvtfcHk0lgcd3oZxh0bgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFCltrw%253D&md5=876a5083e4ae685d55eee3295e924cfd</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2011.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2011.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DFauman%26aufirst%3DE.%2BB.%26aulast%3DRai%26aufirst%3DB.%2BK.%26aulast%3DHuang%26aufirst%3DE.%2BS.%26atitle%3DStructure-based%2520druggability%2520assessment--identifying%2520suitable%2520targets%2520for%2520small%2520molecule%2520therapeutics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D15%26spage%3D463%26epage%3D8%26doi%3D10.1016%2Fj.cbpa.2011.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group">Schonthal, A. H.</span><span> </span><span class="NLM_article-title">Direct non-cyclooxygenase-2 targets of celecoxib and their potential relevance for cancer therapy</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1465</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1038/sj.bjc.6604049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fsj.bjc.6604049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17955049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A280%3ADC%252BD2sjgtFGgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2007&pages=1465-8&author=A.+H.+Schonthal&title=Direct+non-cyclooxygenase-2+targets+of+celecoxib+and+their+potential+relevance+for+cancer+therapy&doi=10.1038%2Fsj.bjc.6604049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Direct non-cyclooxygenase-2 targets of celecoxib and their potential relevance for cancer therapy</span></div><div class="casAuthors">Schonthal A H</div><div class="citationInfo"><span class="NLM_cas:title">British journal of cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1465-8</span>
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    </div><div class="casAbstract">Celecoxib (Celebrex) was developed as a selective cyclooxygenase-2 (COX-2) inhibitor for the treatment of chronic pain.  However, it now appears that this compound harbours additional pharmacologic activities that are entirely independent of its COX-2-inhibitory activity.  This review presents the recently emerged direct non-COX-2 targets of celecoxib and their proposed role in mediating this drug's antitumour effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFaqq9zDTtTq69lAbTj9rGfW6udTcc2eZj3opkzsvs77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjgtFGgtQ%253D%253D&md5=7684ac2baebca2baafaed5bf39302693</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6604049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6604049%26sid%3Dliteratum%253Aachs%26aulast%3DSchonthal%26aufirst%3DA.%2BH.%26atitle%3DDirect%2520non-cyclooxygenase-2%2520targets%2520of%2520celecoxib%2520and%2520their%2520potential%2520relevance%2520for%2520cancer%2520therapy%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2007%26volume%3D97%26spage%3D1465%26epage%3D8%26doi%3D10.1038%2Fsj.bjc.6604049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group">Stangier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemens, A.</span><span> </span><span class="NLM_article-title">Pharmacology, pharmacokinetics, and pharmacodynamics of dabigatran etexilate, an oral direct thrombin inhibitor</span> <span class="citation_source-journal">Clin. Appl. Thromb./Hemostasis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">9S</span><span class="NLM_x">–</span> <span class="NLM_lpage">16S</span><span class="refDoi"> DOI: 10.1177/1076029609343004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1177%2F1076029609343004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19696042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSksLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=9S-16S&author=J.+Stangierauthor=A.+Clemens&title=Pharmacology%2C+pharmacokinetics%2C+and+pharmacodynamics+of+dabigatran+etexilate%2C+an+oral+direct+thrombin+inhibitor&doi=10.1177%2F1076029609343004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacology, pharmacokinetics, and pharmacodynamics of dabigatran etexilate, an oral direct thrombin inhibitor</span></div><div class="casAuthors">Stangier, Joachim; Clemens, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Applied Thrombosis/Hemostasis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">9S-16S</span>CODEN:
                <span class="NLM_cas:coden">CATHF9</span>;
        ISSN:<span class="NLM_cas:issn">1076-0296</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Dabigatran etexilate is a novel, oral reversible direct thrombin inhibitor that is rapidly absorbed and converted to its active form, dabigatran.  Dabigatran has been shown to be a potent, competitive, and reversible inhibitor of thrombin, inhibiting both thrombin activity and generation.  Studies in healthy volunteers and in patients undergoing orthopedic surgery indicate that dabigatran has a predictable pharmacokinetic profile, allowing for a fixed-dose regimen without the need for coagulation monitoring.  In healthy volunteers, peak plasma concns. of dabigatran are reached approx. 2 h after oral administration.  The elimination half-life is 12 to 14 h, with clearance predominantly occurring via renal excretion of unchanged drug.  Dabigatran is not metabolized by cytochrome P 450 isoenzymes, has no interactions with food, and also has a low potential for drug-drug interactions.  The pharmacokinetic profile of dabigatran is consistent across a broad range of different patient populations and is unaffected by gender, body wt., ethnic origin, obesity, and mild-to-moderate hepatic impairment.  Small differences in dabigatran pharmacokinetics assocd. with age are attributable to variation in renal function.  Dabigatran etexilate produces a predictable pharmacodynamic effect and requires no coagulation monitoring.  It has been approved in the European Union (EU) and Canada for prophylaxis of thromboembolism in patients undergoing total knee or hip arthroplasty.  Ongoing clin. trials are investigating its use in the treatment of venous thromboembolism, prevention of stroke in patients with nonvalvular atrial fibrillation, and treatment of thromboembolic complications, following acute coronary syndromes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz7HdOwdwOFrVg90H21EOLACvtfcHk0lgcd3oZxh0bgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSksLzE&md5=47f0699335b37a2133ea1a9ea74385dd</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1177%2F1076029609343004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1076029609343004%26sid%3Dliteratum%253Aachs%26aulast%3DStangier%26aufirst%3DJ.%26aulast%3DClemens%26aufirst%3DA.%26atitle%3DPharmacology%252C%2520pharmacokinetics%252C%2520and%2520pharmacodynamics%2520of%2520dabigatran%2520etexilate%252C%2520an%2520oral%2520direct%2520thrombin%2520inhibitor%26jtitle%3DClin.%2520Appl.%2520Thromb.%252FHemostasis%26date%3D2009%26volume%3D15%26spage%3D9S%26epage%3D16S%26doi%3D10.1177%2F1076029609343004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group">Kong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, J. F.</span><span> </span><span class="NLM_article-title">Direct thrombin inhibitors: Patents 2002–2012 (review)</span> <span class="citation_source-journal">Mol. Med. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1506</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span><span class="refDoi"> DOI: 10.3892/mmr.2014.2025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.3892%2Fmmr.2014.2025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=24604304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVeltLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1506-14&author=Y.+Kongauthor=H.+Chenauthor=Y.+Q.+Wangauthor=L.+Mengauthor=J.+F.+Wei&title=Direct+thrombin+inhibitors%3A+Patents+2002%E2%80%932012+%28review%29&doi=10.3892%2Fmmr.2014.2025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Direct thrombin inhibitors: patents 2002-2012 (Review)</span></div><div class="casAuthors">Kong, Yi; Chen, Hao; Wang, Yong-Qing; Meng, Ling; Wei, Ji-Fu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1506-1514</span>CODEN:
                <span class="NLM_cas:coden">MMROA5</span>;
        ISSN:<span class="NLM_cas:issn">1791-2997</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Acute vascular diseases and other thromboses of the blood system constitute major health risks in developing countries.  Thrombin plays a central role in blood coagulation, which is a crucial process involved in thrombosis.  Direct thrombin inhibitors (DTIs) such as argatroban, dabigatran, dabigatran etexilate, lepirudin, desirudin and bivalirudin, which bind to thrombin and block its enzymic activity, are widely and effectively used in the treatment of thromboembolic diseases.  DTIs appear to overcome the disadvantages of indirect thrombin inhibitors such as unfractionated heparins (UFH).  Although these DTIs show specific advantages over indirect inhibitors, they still present limitations, such as a narrow therapeutic window, and bleeding and anaphylaxis as side-effects.  Novel anticoagulant drugs need thus to be developed to overcome these limitations.  In the search for addnl. candidate agents with improved efficacy, safety and high bioavailability in oral administration, a high no. of compds. has been identified, such as those derived from the tripeptide template D-Phe-Pro-Arg, aptamers and peptides isolated from blood-sucking animals.  These candidates may prove the new agents of choice for the treatment of cardiovascular diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_V_SPDTnH7rVg90H21EOLACvtfcHk0lgcd3oZxh0bgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVeltLrN&md5=dd0795d9899c52c415b542d34bcfb93f</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.3892%2Fmmr.2014.2025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fmmr.2014.2025%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%2BQ.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DJ.%2BF.%26atitle%3DDirect%2520thrombin%2520inhibitors%253A%2520Patents%25202002%25E2%2580%25932012%2520%2528review%2529%26jtitle%3DMol.%2520Med.%2520Rep.%26date%3D2014%26volume%3D9%26spage%3D1506%26epage%3D14%26doi%3D10.3892%2Fmmr.2014.2025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group">Fu, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neu, H. C.</span><span> </span><span class="NLM_article-title">Piperacillin, a new penicillin active against many bacteria resistant to other penicillins</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1978</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1128/AAC.13.3.358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1128%2FAAC.13.3.358" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1978&pages=358-67&author=K.+P.+Fuauthor=H.+C.+Neu&title=Piperacillin%2C+a+new+penicillin+active+against+many+bacteria+resistant+to+other+penicillins&doi=10.1128%2FAAC.13.3.358"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1128%2FAAC.13.3.358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.13.3.358%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DK.%2BP.%26aulast%3DNeu%26aufirst%3DH.%2BC.%26atitle%3DPiperacillin%252C%2520a%2520new%2520penicillin%2520active%2520against%2520many%2520bacteria%2520resistant%2520to%2520other%2520penicillins%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1978%26volume%3D13%26spage%3D358%26epage%3D67%26doi%3D10.1128%2FAAC.13.3.358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group">van Berkel, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettleship, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brem, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuart, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claridge, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owens, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Binding of (5s)-penicilloic acid to penicillin binding protein 3</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2112</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1021/cb400200h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400200h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2112-6&author=S.+S.+van+Berkelauthor=J.+E.+Nettleshipauthor=I.+K.+Leungauthor=J.+Bremauthor=H.+Choiauthor=D.+I.+Stuartauthor=T.+D.+Claridgeauthor=M.+A.+McDonoughauthor=R.+J.+Owensauthor=J.+Renauthor=C.+J.+Schofield&title=Binding+of+%285s%29-penicilloic+acid+to+penicillin+binding+protein+3&doi=10.1021%2Fcb400200h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Fcb400200h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400200h%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BBerkel%26aufirst%3DS.%2BS.%26aulast%3DNettleship%26aufirst%3DJ.%2BE.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DBrem%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DStuart%26aufirst%3DD.%2BI.%26aulast%3DClaridge%26aufirst%3DT.%2BD.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOwens%26aufirst%3DR.%2BJ.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DBinding%2520of%2520%25285s%2529-penicilloic%2520acid%2520to%2520penicillin%2520binding%2520protein%25203%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2112%26epage%3D6%26doi%3D10.1021%2Fcb400200h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group">Patchett, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tristram, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyvratt, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taub, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeler, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ten Broeke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ondeyka, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorsett, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenlee, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohr, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffsommer, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshua, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruyle, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothrock, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aster, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maycock, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirschmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulm, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassil, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, C. A.</span><span> </span><span class="NLM_article-title">A new class of angiotensin-converting enzyme inhibitors</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">280</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span><span class="refDoi"> DOI: 10.1038/288280a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2F288280a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=6253826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaL3MXktlGhtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=1980&pages=280-3&author=A.+A.+Patchettauthor=E.+Harrisauthor=E.+W.+Tristramauthor=M.+J.+Wyvrattauthor=M.+T.+Wuauthor=D.+Taubauthor=E.+R.+Petersonauthor=T.+J.+Ikelerauthor=J.+ten+Broekeauthor=L.+G.+Payneauthor=D.+L.+Ondeykaauthor=E.+D.+Thorsettauthor=W.+J.+Greenleeauthor=N.+S.+Lohrauthor=R.+D.+Hoffsommerauthor=H.+Joshuaauthor=W.+V.+Ruyleauthor=J.+W.+Rothrockauthor=S.+D.+Asterauthor=A.+L.+Maycockauthor=F.+M.+Robinsonauthor=R.+Hirschmannauthor=C.+S.+Sweetauthor=E.+H.+Ulmauthor=D.+M.+Grossauthor=T.+C.+Vassilauthor=C.+A.+Stone&title=A+new+class+of+angiotensin-converting+enzyme+inhibitors&doi=10.1038%2F288280a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">A new class of angiotensin-converting enzyme inhibitors</span></div><div class="casAuthors">Patchett, A. A.; Harris, E.; Tristram, E. W.; Wyvratt, M. J.; Wu, M. T.; Taub, D.; Peterson, E. R.; Ikeler, T. J.; Ten Broeke, J.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">5788</span>),
    <span class="NLM_cas:pages">280-3</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Substituted N-carboxymethyldipeptides were evaluated for their ability to antagonize the pressor effect of angiotensin I (I)  [9041-90-1] in anesthetized rats and dogs (100 and 600 ng/kg, i.v., resp.); Ph(CH2)2CH(CO2H)-Ala-Pro (II)  [76420-72-9], Ph(CH2)2CH(CO2Et)-Ala-Pro maleate (III)  [75847-73-3] and Ph(CH2)2CH(CO2H)-Lys-Pro (IV)  [77726-95-5] were highly active by the i.v. route in rats and dogs, except for III in dogs, but all were orally active in rats and dogs.  II, III, and IV showed high potency in inhibiting the blood pressure response to I and all had a long duration of action; high angiotensin-converting enzyme (EC 3.4.15.1) (V)  [9015-82-1] inhibitory activities at doses as low as 2.3 μg/kg were obsd. after i.v. administration and after oral administration, the compds. showed good activities with duration of action of ≥6 h at doses of 0.3 mg/kg.  Thus, V inhibitors have important design implications for the synthesis of inhibitors of other metallodipeptidyl carboxypeptidases and possibly of metalloendopeptidases more generally.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2vFzl18NAf7Vg90H21EOLACvtfcHk0lhvd10ti5B6Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXktlGhtLY%253D&md5=3a395d264e4e5c14d0cca0bcb6065a73</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1038%2F288280a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F288280a0%26sid%3Dliteratum%253Aachs%26aulast%3DPatchett%26aufirst%3DA.%2BA.%26aulast%3DHarris%26aufirst%3DE.%26aulast%3DTristram%26aufirst%3DE.%2BW.%26aulast%3DWyvratt%26aufirst%3DM.%2BJ.%26aulast%3DWu%26aufirst%3DM.%2BT.%26aulast%3DTaub%26aufirst%3DD.%26aulast%3DPeterson%26aufirst%3DE.%2BR.%26aulast%3DIkeler%26aufirst%3DT.%2BJ.%26aulast%3Dten%2BBroeke%26aufirst%3DJ.%26aulast%3DPayne%26aufirst%3DL.%2BG.%26aulast%3DOndeyka%26aufirst%3DD.%2BL.%26aulast%3DThorsett%26aufirst%3DE.%2BD.%26aulast%3DGreenlee%26aufirst%3DW.%2BJ.%26aulast%3DLohr%26aufirst%3DN.%2BS.%26aulast%3DHoffsommer%26aufirst%3DR.%2BD.%26aulast%3DJoshua%26aufirst%3DH.%26aulast%3DRuyle%26aufirst%3DW.%2BV.%26aulast%3DRothrock%26aufirst%3DJ.%2BW.%26aulast%3DAster%26aufirst%3DS.%2BD.%26aulast%3DMaycock%26aufirst%3DA.%2BL.%26aulast%3DRobinson%26aufirst%3DF.%2BM.%26aulast%3DHirschmann%26aufirst%3DR.%26aulast%3DSweet%26aufirst%3DC.%2BS.%26aulast%3DUlm%26aufirst%3DE.%2BH.%26aulast%3DGross%26aufirst%3DD.%2BM.%26aulast%3DVassil%26aufirst%3DT.%2BC.%26aulast%3DStone%26aufirst%3DC.%2BA.%26atitle%3DA%2520new%2520class%2520of%2520angiotensin-converting%2520enzyme%2520inhibitors%26jtitle%3DNature%26date%3D1980%26volume%3D288%26spage%3D280%26epage%3D3%26doi%3D10.1038%2F288280a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group">Gubareva, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayden, F. G.</span><span> </span><span class="NLM_article-title">Influenza virus neuraminidase inhibitors</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">355</span><span class="NLM_x">, </span> <span class="NLM_fpage">827</span><span class="NLM_x">–</span> <span class="NLM_lpage">35</span><span class="refDoi"> DOI: 10.1016/S0140-6736(99)11433-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0140-6736%2899%2911433-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10711940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVSnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2000&pages=827-35&author=L.+V.+Gubarevaauthor=L.+Kaiserauthor=F.+G.+Hayden&title=Influenza+virus+neuraminidase+inhibitors&doi=10.1016%2FS0140-6736%2899%2911433-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Influenza virus neuraminidase inhibitors</span></div><div class="casAuthors">Gubareva, L. V.; Kaiser, L.; Hayden, F. G.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">9206</span>),
    <span class="NLM_cas:pages">827-835</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Lancet Ltd.</span>)
        </div><div class="casAbstract">A review with 69 refs.  Neuraminidase promotes influenza virus release from infected cells and facilitates virus spread within the respiratory tract.  Several potent and specific inhibitors of this enzyme have been developed, and two (zanamivir and oseltamivir) have been approved for human use.  Unlike amantadine and rimantadine that target the M2 protein of influenza A viruses, these drugs inhibit replication of both influenza A and B viruses.  Zanamivir is delivered by inhalation because of its low oral bioavailability whereas oseltamivir is administered by mouth.  Early treatment with either drug reduces the severity and duration of influenza symptoms and assocd. complications.  Both agents are effective for chemoprophylaxis.  Because of a broader antiviral spectrum, better tolerance, and less potential for emergence of resistance than is seen with the M2 inhibitors, the neuraminidase inhibitors represent an important advance in the treatment of influenza.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppRfygdS17K7Vg90H21EOLACvtfcHk0lhvd10ti5B6Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVSnu7s%253D&md5=4b996d708c2f8e5ae7f4634190b05f27</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2899%2911433-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252899%252911433-8%26sid%3Dliteratum%253Aachs%26aulast%3DGubareva%26aufirst%3DL.%2BV.%26aulast%3DKaiser%26aufirst%3DL.%26aulast%3DHayden%26aufirst%3DF.%2BG.%26atitle%3DInfluenza%2520virus%2520neuraminidase%2520inhibitors%26jtitle%3DLancet%26date%3D2000%26volume%3D355%26spage%3D827%26epage%3D35%26doi%3D10.1016%2FS0140-6736%2899%2911433-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group">Maignan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guilloteau, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pouzieux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi-Sledeski, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, S. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ewing, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pauls, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spada, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikol, V.</span><span> </span><span class="NLM_article-title">Crystal structures of human factor Xa complexed with potent inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">3226</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1021/jm000940u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000940u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsVejtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3226-32&author=S.+Maignanauthor=J.+P.+Guilloteauauthor=S.+Pouzieuxauthor=Y.+M.+Choi-Sledeskiauthor=M.+R.+Beckerauthor=S.+I.+Kleinauthor=W.+R.+Ewingauthor=H.+W.+Paulsauthor=A.+P.+Spadaauthor=V.+Mikol&title=Crystal+structures+of+human+factor+Xa+complexed+with+potent+inhibitors&doi=10.1021%2Fjm000940u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures of Human Factor Xa Complexed with Potent Inhibitors</span></div><div class="casAuthors">Maignan, Sebastien; Guilloteau, Jean-Pierre; Pouzieux, Stephanie; Choi-Sledeski, Yong Mi; Becker, Michael R.; Klein, Scott I.; Ewing, William R.; Pauls, Henry W.; Spada, Alfred P.; Mikol, Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3226-3232</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Involved in the coagulation cascade, factor Xa (FXa) is a serine protease which has received great interest as a potential target for the development of new antithrombotics.  Although there is a great wealth of structural data on thrombin complexes, few structures of ligand/FXa complexes have been reported, presumably because of the difficulty in growing crystals.  Reproducible crystn. conditions for human des-Gla1-45 coagulation FXa have been found.  This has led to an improvement in the diffraction quality of the crystals (about 2.1 Å) when compared to the previously reported forms (2.3-2.8 Å) thus providing a suitable platform for a structure-based drug design approach.  A series of crystal structures of noncovalent inhibitors complexed with FXa have been detd., three of which are presented herein.  These include compds. contg. the benzamidine moiety and surrogates of the basic group.  The benzamidine-contg. compd. binds in a canonical fashion typical of synthetic serine protease inhibitors.  On the contrary, mols. that contain surrogates of the benzamidine group do not make direct hydrogen-bonding interactions with the carboxylate of Asp189 at the bottom of the S1 pocket.  The structural data provide a likely explanation for the specificity of these inhibitors and a great aid in the design of bioavailable potent FXa inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsMdvctURxlLVg90H21EOLACvtfcHk0lg6Z2eg2sB04g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsVejtb0%253D&md5=d88dd067911129d813294208f5ffbc35</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Fjm000940u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000940u%26sid%3Dliteratum%253Aachs%26aulast%3DMaignan%26aufirst%3DS.%26aulast%3DGuilloteau%26aufirst%3DJ.%2BP.%26aulast%3DPouzieux%26aufirst%3DS.%26aulast%3DChoi-Sledeski%26aufirst%3DY.%2BM.%26aulast%3DBecker%26aufirst%3DM.%2BR.%26aulast%3DKlein%26aufirst%3DS.%2BI.%26aulast%3DEwing%26aufirst%3DW.%2BR.%26aulast%3DPauls%26aufirst%3DH.%2BW.%26aulast%3DSpada%26aufirst%3DA.%2BP.%26aulast%3DMikol%26aufirst%3DV.%26atitle%3DCrystal%2520structures%2520of%2520human%2520factor%2520Xa%2520complexed%2520with%2520potent%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3226%26epage%3D32%26doi%3D10.1021%2Fjm000940u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group">Istvan, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deisenhofer, J.</span><span> </span><span class="NLM_article-title">Structural mechanism for statin inhibition of HMG-CoA reductase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">292</span><span class="NLM_x">, </span> <span class="NLM_fpage">1160</span><span class="NLM_x">–</span> <span class="NLM_lpage">4</span><span class="refDoi"> DOI: 10.1126/science.1059344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1126%2Fscience.1059344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11349148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjs1Cgtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2001&pages=1160-4&author=E.+S.+Istvanauthor=J.+Deisenhofer&title=Structural+mechanism+for+statin+inhibition+of+HMG-CoA+reductase&doi=10.1126%2Fscience.1059344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism for statin inhibition of HMG-CoA reductase</span></div><div class="casAuthors">Istvan, Eva S.; Deisenhofer, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">5519</span>),
    <span class="NLM_cas:pages">1160-1164</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">HMG-CoA (3-hydroxy-3-methylglutaryl-CoA) reductase (HMGR) catalyzes the committed step in cholesterol biosynthesis.  Statins are HMGR inhibitors with inhibition const. values in the nanomolar range that effectively lower serum cholesterol levels and are widely prescribed in the treatment of hypercholesterolemia.  We have detd. structures of the catalytic portion of human HMGR complexed with six different statins.  The statins occupy a portion of the binding site of HMG-CoA, thus blocking access of this substrate to the active site.  Near the carboxyl terminus of HMGR, several catalytically relevant residues are disordered in the enzyme-statin complexes.  If these residues were not flexible, they would sterically hinder statin binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDsSOnK9675bVg90H21EOLACvtfcHk0lg6Z2eg2sB04g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjs1Cgtbo%253D&md5=235644ee8cca3d2e061093e93bd1d702</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1126%2Fscience.1059344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1059344%26sid%3Dliteratum%253Aachs%26aulast%3DIstvan%26aufirst%3DE.%2BS.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DStructural%2520mechanism%2520for%2520statin%2520inhibition%2520of%2520HMG-CoA%2520reductase%26jtitle%3DScience%26date%3D2001%26volume%3D292%26spage%3D1160%26epage%3D4%26doi%3D10.1126%2Fscience.1059344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group">Barrett, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deaton, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassell, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFadyen, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shewchuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, D. H.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, L. L.</span><span> </span><span class="NLM_article-title">Acyclic cyanamide-based inhibitors of cathepsin K</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3039</span><span class="NLM_x">–</span> <span class="NLM_lpage">43</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2005.04.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2005.04.032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=3039-43&author=D.+G.+Barrettauthor=D.+N.+Deatonauthor=A.+M.+Hassellauthor=R.+B.+McFadyenauthor=A.+B.+Millerauthor=L.+R.+Millerauthor=J.+A.+Payneauthor=L.+M.+Shewchukauthor=D.+H.+Willardauthor=L.+L.+Wright&title=Acyclic+cyanamide-based+inhibitors+of+cathepsin+K&doi=10.1016%2Fj.bmcl.2005.04.032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.04.032%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DD.%2BG.%26aulast%3DDeaton%26aufirst%3DD.%2BN.%26aulast%3DHassell%26aufirst%3DA.%2BM.%26aulast%3DMcFadyen%26aufirst%3DR.%2BB.%26aulast%3DMiller%26aufirst%3DA.%2BB.%26aulast%3DMiller%26aufirst%3DL.%2BR.%26aulast%3DPayne%26aufirst%3DJ.%2BA.%26aulast%3DShewchuk%26aufirst%3DL.%2BM.%26aulast%3DWillard%26aufirst%3DD.%2BH.%26aulast%3DWright%26aufirst%3DL.%2BL.%26atitle%3DAcyclic%2520cyanamide-based%2520inhibitors%2520of%2520cathepsin%2520K%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D3039%26epage%3D43%26doi%3D10.1016%2Fj.bmcl.2005.04.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group">Gauthier, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauret, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cromlish, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desmarais, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duong, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Falgueyret, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimmel, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamontagne, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeRiche, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKay, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicoll-Griffith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oballa, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Percival, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riendeau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robichaud, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodan, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodan, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seto, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truong, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venuti, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wesolowski, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamboni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, W. C.</span><span> </span><span class="NLM_article-title">The discovery of odanacatib (MK-0822), a selective inhibitor of cathepsin K</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">923</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.12.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2007.12.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18226527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=923-8&author=J.+Y.+Gauthierauthor=N.+Chauretauthor=W.+Cromlishauthor=S.+Desmaraisauthor=L.+T.+Duongauthor=J.+P.+Falgueyretauthor=D.+B.+Kimmelauthor=S.+Lamontagneauthor=S.+Legerauthor=T.+LeRicheauthor=C.+S.+Liauthor=F.+Masseauthor=D.+J.+McKayauthor=D.+A.+Nicoll-Griffithauthor=R.+M.+Oballaauthor=J.+T.+Palmerauthor=M.+D.+Percivalauthor=D.+Riendeauauthor=J.+Robichaudauthor=G.+A.+Rodanauthor=S.+B.+Rodanauthor=C.+Setoauthor=M.+Therienauthor=V.+L.+Truongauthor=M.+C.+Venutiauthor=G.+Wesolowskiauthor=R.+N.+Youngauthor=R.+Zamboniauthor=W.+C.+Black&title=The+discovery+of+odanacatib+%28MK-0822%29%2C+a+selective+inhibitor+of+cathepsin+K&doi=10.1016%2Fj.bmcl.2007.12.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of odanacatib (MK-0822), a selective inhibitor of cathepsin K</span></div><div class="casAuthors">Gauthier, Jacques Yves; Chauret, Nathalie; Cromlish, Wanda; Desmarais, Sylvie; Duong, Le T.; Falgueyret, Jean-Pierre; Kimmel, Donald B.; Lamontagne, Sonia; Leger, Serge; LeRiche, Tammy; Li, Chun Sing; Masse, Frederic; McKay, Daniel J.; Nicoll-Griffith, Deborah A.; Oballa, Renata M.; Palmer, James T.; Percival, M. David; Riendeau, Denis; Robichaud, Joel; Rodan, Gideon A.; Rodan, Sevgi B.; Seto, Carmai; Therien, Michel; Truong, Vouy-Linh; Venuti, Michael C.; Wesolowski, Gregg; Young, Robert N.; Zamboni, Robert; Black, W. Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">923-928</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Odanacatib is a potent, selective, and neutral cathepsin K inhibitor which was developed to address the metabolic liabilities of the Cat K inhibitor L-873724.  Substituting P1 and modifying the P2 side chain led to a metabolically robust inhibitor with a long half-life in preclin. species.  Odanacatib was more selective in whole cell assays than the published Cat K inhibitors balicatib and relacatib.  Evaluation in dermal fibroblast culture showed minimal intracellular collagen accumulation relative to less selective Cat K inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ytNI-SBgarVg90H21EOLACvtfcHk0lg6Z2eg2sB04g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWktro%253D&md5=e432a420a3ec99c784117750b9bbeda8</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.12.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.12.047%26sid%3Dliteratum%253Aachs%26aulast%3DGauthier%26aufirst%3DJ.%2BY.%26aulast%3DChauret%26aufirst%3DN.%26aulast%3DCromlish%26aufirst%3DW.%26aulast%3DDesmarais%26aufirst%3DS.%26aulast%3DDuong%26aufirst%3DL.%2BT.%26aulast%3DFalgueyret%26aufirst%3DJ.%2BP.%26aulast%3DKimmel%26aufirst%3DD.%2BB.%26aulast%3DLamontagne%26aufirst%3DS.%26aulast%3DLeger%26aufirst%3DS.%26aulast%3DLeRiche%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DMasse%26aufirst%3DF.%26aulast%3DMcKay%26aufirst%3DD.%2BJ.%26aulast%3DNicoll-Griffith%26aufirst%3DD.%2BA.%26aulast%3DOballa%26aufirst%3DR.%2BM.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26aulast%3DPercival%26aufirst%3DM.%2BD.%26aulast%3DRiendeau%26aufirst%3DD.%26aulast%3DRobichaud%26aufirst%3DJ.%26aulast%3DRodan%26aufirst%3DG.%2BA.%26aulast%3DRodan%26aufirst%3DS.%2BB.%26aulast%3DSeto%26aufirst%3DC.%26aulast%3DTherien%26aufirst%3DM.%26aulast%3DTruong%26aufirst%3DV.%2BL.%26aulast%3DVenuti%26aufirst%3DM.%2BC.%26aulast%3DWesolowski%26aufirst%3DG.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZamboni%26aufirst%3DR.%26aulast%3DBlack%26aufirst%3DW.%2BC.%26atitle%3DThe%2520discovery%2520of%2520odanacatib%2520%2528MK-0822%2529%252C%2520a%2520selective%2520inhibitor%2520of%2520cathepsin%2520K%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D923%26epage%3D8%26doi%3D10.1016%2Fj.bmcl.2007.12.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group">Brixen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapurlat, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keaveny, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuerst, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recker, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dardzinski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verbruggen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ather, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Papp, A. E.</span><span> </span><span class="NLM_article-title">Bone density, turnover, and estimated strength in postmenopausal women treated with odanacatib: A randomized trial</span> <span class="citation_source-journal">J. Clin. Endocrinol. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1210/jc.2012-2972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1210%2Fjc.2012-2972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23337728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlOks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2013&pages=571-80&author=K.+Brixenauthor=R.+Chapurlatauthor=A.+M.+Cheungauthor=T.+M.+Keavenyauthor=T.+Fuerstauthor=K.+Engelkeauthor=R.+Reckerauthor=B.+Dardzinskiauthor=N.+Verbruggenauthor=S.+Atherauthor=E.+Rosenbergauthor=A.+E.+de+Papp&title=Bone+density%2C+turnover%2C+and+estimated+strength+in+postmenopausal+women+treated+with+odanacatib%3A+A+randomized+trial&doi=10.1210%2Fjc.2012-2972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Bone density, turnover, and estimated strength in postmenopausal women treated with odanacatib: a randomized trial</span></div><div class="casAuthors">Brixen, Kim; Chapurlat, Roland; Cheung, Angela M.; Keaveny, Tony M.; Fuers, Thomas; Engelke, Klaus; Recker, Robert; Dardzinski, Bernard; Verbruggen, Nadia; Ather, Shabana; Rosenberg, Elizabeth; de Papp, Anne E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">571-580</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Context: Odanacatib, a cathepsin K inhibitor, increases spine and hip areal bone mineral d. (BMD) in postmenopausal women with low BMD and cortical thickness in ovariectomized monkeys.  Objective: The objective of the study was to examine the impact of odanacatib on the trabecular and cortical bone compartments and estd. strength at the hip and spine.  Design: This was a randomized, double-blind, 2-yr trial.  Setting: The study was conducted at a private or institutional practice.  Participants: Participants included 214 postmenopausal women with low areal BMD.  Intervention: The intervention included odanacatib 50 mg or placebo weekly.  Main Outcome Measures: Changes in areal BMD by dual-energy x-ray absorptiometry (primary end point, 1 yr areal BMD change at lumbar spine), bone turnover markers, volumetric BMD by quant. computed tomog. (QCT), and bone strength estd. by finite element anal. were measured.  Results: Year 1 lumbar spine areal BMD percent change from baseline was 3.5% greater with odanacatib than placebo (P < .001).  Bone-resorption marker C-telopeptide of type 1 collagen was significantly lower with odanacatib vs placebo at 6 mo and 2 years (P < .001).  Bone-formation marker procollagen I N-terminal peptide initially decreased with odanacatib but by 2 years did not differ from placebo.  After 6 mo, odanacatib-treated women had greater increases in trabecular volumetric BMD and estd. compressive strength at the spine and integral and trabecular volumetric BMD and estd. strength at the hip (P < .001).  At the cortical envelope of the femoral neck, bone mineral content, thickness, vol., and cross-sectional area also increased from baseline with odanacatib vs placebo (P < .001 at 24 mo).  Adverse experiences were similar between groups.  Conclusions: Over 2 years, odanacatib decreased bone resorption, maintained bone formation, increased areal and volumetric BMD, and increased estd. bone strength at both the hip and spine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5IgjyjsYm47Vg90H21EOLACvtfcHk0liV7xliiUy-qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlOks74%253D&md5=2dc289cb3ccc6576fb66711b6f69be8d</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1210%2Fjc.2012-2972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2012-2972%26sid%3Dliteratum%253Aachs%26aulast%3DBrixen%26aufirst%3DK.%26aulast%3DChapurlat%26aufirst%3DR.%26aulast%3DCheung%26aufirst%3DA.%2BM.%26aulast%3DKeaveny%26aufirst%3DT.%2BM.%26aulast%3DFuerst%26aufirst%3DT.%26aulast%3DEngelke%26aufirst%3DK.%26aulast%3DRecker%26aufirst%3DR.%26aulast%3DDardzinski%26aufirst%3DB.%26aulast%3DVerbruggen%26aufirst%3DN.%26aulast%3DAther%26aufirst%3DS.%26aulast%3DRosenberg%26aufirst%3DE.%26aulast%3Dde%2BPapp%26aufirst%3DA.%2BE.%26atitle%3DBone%2520density%252C%2520turnover%252C%2520and%2520estimated%2520strength%2520in%2520postmenopausal%2520women%2520treated%2520with%2520odanacatib%253A%2520A%2520randomized%2520trial%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2013%26volume%3D98%26spage%3D571%26epage%3D80%26doi%3D10.1210%2Fjc.2012-2972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group">Summa, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrocchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crescenzi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiore, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez Paz, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzel, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laufer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteagudo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muraglia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nizi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orvieto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pescatore, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarpelli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stillmock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witmer, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, M.</span><span> </span><span class="NLM_article-title">Discovery of raltegravir, a potent, selective orally bioavailable HIV-integrase inhibitor for the treatment of HIV-AIDS infection</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5843</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span><span class="refDoi"> DOI: 10.1021/jm800245z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800245z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5843-55&author=V.+Summaauthor=A.+Petrocchiauthor=F.+Bonelliauthor=B.+Crescenziauthor=M.+Donghiauthor=M.+Ferraraauthor=F.+Fioreauthor=C.+Gardelliauthor=O.+Gonzalez+Pazauthor=D.+J.+Hazudaauthor=P.+Jonesauthor=O.+Kinzelauthor=R.+Lauferauthor=E.+Monteagudoauthor=E.+Muragliaauthor=E.+Niziauthor=F.+Orvietoauthor=P.+Paceauthor=G.+Pescatoreauthor=R.+Scarpelliauthor=K.+Stillmockauthor=M.+V.+Witmerauthor=M.+Rowley&title=Discovery+of+raltegravir%2C+a+potent%2C+selective+orally+bioavailable+HIV-integrase+inhibitor+for+the+treatment+of+HIV-AIDS+infection&doi=10.1021%2Fjm800245z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Raltegravir, a Potent, Selective Orally Bioavailable HIV-Integrase Inhibitor for the Treatment of HIV-AIDS Infection</span></div><div class="casAuthors">Summa, Vincenzo; Petrocchi, Alessia; Bonelli, Fabio; Crescenzi, Benedetta; Donghi, Monica; Ferrara, Marco; Fiore, Fabrizio; Gardelli, Cristina; Gonzalez Paz, Odalys; Hazuda, Daria J.; Jones, Philip; Kinzel, Olaf; Laufer, Ralph; Monteagudo, Edith; Muraglia, Ester; Nizi, Emanuela; Orvieto, Federica; Pace, Paola; Pescatore, Giovanna; Scarpelli, Rita; Stillmock, Kara; Witmer, Marc V.; Rowley, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5843-5855</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human immunodeficiency virus type-1 (HIV-1) integrase is one of the three virally encoded enzymes required for replication and therefore a rational target for chemotherapeutic intervention in the treatment of HIV-1 infection.  We report here the discovery of Raltegravir, the first HIV-integrase inhibitor approved by FDA for the treatment of HIV infection.  It derives from the evolution of 5,6-dihydroxypyrimidine-4-carboxamides and N-methyl-4-hydroxypyrimidinone-carboxamides, which exhibited potent inhibition of the HIV-integrase catalyzed strand transfer process.  Structural modifications on these mols. were made in order to maximize potency as HIV-integrase inhibitors against the wild type virus, a selection of mutants, and optimize the selectivity, pharmacokinetic, and metabolic profiles in preclin. species.  The good profile of Raltegravir has enabled its progression toward the end of phase III clin. trials for the treatment of HIV-1 infection and culminated with the FDA approval as the first HIV-integrase inhibitor for the treatment of HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYo6KKNjJRCrVg90H21EOLACvtfcHk0liV7xliiUy-qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb%252FM&md5=f90f994c7cd031a4063cbb3e91b4c3f0</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Fjm800245z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800245z%26sid%3Dliteratum%253Aachs%26aulast%3DSumma%26aufirst%3DV.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DBonelli%26aufirst%3DF.%26aulast%3DCrescenzi%26aufirst%3DB.%26aulast%3DDonghi%26aufirst%3DM.%26aulast%3DFerrara%26aufirst%3DM.%26aulast%3DFiore%26aufirst%3DF.%26aulast%3DGardelli%26aufirst%3DC.%26aulast%3DGonzalez%2BPaz%26aufirst%3DO.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DKinzel%26aufirst%3DO.%26aulast%3DLaufer%26aufirst%3DR.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DMuraglia%26aufirst%3DE.%26aulast%3DNizi%26aufirst%3DE.%26aulast%3DOrvieto%26aufirst%3DF.%26aulast%3DPace%26aufirst%3DP.%26aulast%3DPescatore%26aufirst%3DG.%26aulast%3DScarpelli%26aufirst%3DR.%26aulast%3DStillmock%26aufirst%3DK.%26aulast%3DWitmer%26aufirst%3DM.%2BV.%26aulast%3DRowley%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520raltegravir%252C%2520a%2520potent%252C%2520selective%2520orally%2520bioavailable%2520HIV-integrase%2520inhibitor%2520for%2520the%2520treatment%2520of%2520HIV-AIDS%2520infection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5843%26epage%3D55%26doi%3D10.1021%2Fjm800245z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group">Leung, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbenante, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlie, D. P.</span><span> </span><span class="NLM_article-title">Protease inhibitors: Current status and future prospects</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1021/jm990412m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm990412m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXpsVKhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=305-41&author=D.+Leungauthor=G.+Abbenanteauthor=D.+P.+Fairlie&title=Protease+inhibitors%3A+Current+status+and+future+prospects&doi=10.1021%2Fjm990412m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Protease inhibitors: Current status and future prospects</span></div><div class="casAuthors">Leung, Donmienne; Abbenante, Giovanni; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">305-341</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 354 refs. is given on Asp protease inhibitors (HIV-1 protease, renin, plasmepsins, cathepsin D, secreted Asp protease), Ser protease inhibitors (thrombin, factor Xa, elastase, tryptase, complement convertases, hepatitis C-NS3 protease, broad-spectrum Ser protease inhibitors), Cys protease inhibitors (cathepsin K, B, and L, caspases, rhinovirus 3C protease, calpains), and metalloprotease inhibitors (angiotensin-converting enzyme, neutral endopeptidase, matrix metalloprotease inhibitors, tumor necrosis factor-α-converting enzyme).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMWqeFAY1BO7Vg90H21EOLACvtfcHk0lh4c8w5XJLXqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXpsVKhsQ%253D%253D&md5=1190cf0b642169e340e3471cd893f217</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1021%2Fjm990412m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990412m%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DD.%26aulast%3DAbbenante%26aufirst%3DG.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DProtease%2520inhibitors%253A%2520Current%2520status%2520and%2520future%2520prospects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D305%26epage%3D41%26doi%3D10.1021%2Fjm990412m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group">Reigan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gbaj, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barry, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Page, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luke, R. W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douglas, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratford, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaffar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryce, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S.</span><span> </span><span class="NLM_article-title">Aminoimidazolylmethyluracil analogues as potent inhibitors of thymidine phosphorylase and their bioreductive nitroimidazolyl prodrugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">392</span><span class="NLM_x">–</span> <span class="NLM_lpage">402</span><span class="refDoi"> DOI: 10.1021/jm049494r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049494r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=392-402&author=P.+Reiganauthor=P.+N.+Edwardsauthor=A.+Gbajauthor=C.+Coleauthor=S.+T.+Barryauthor=K.+M.+Pageauthor=S.+E.+Ashtonauthor=R.+W.+A.+Lukeauthor=K.+T.+Douglasauthor=I.+J.+Stratfordauthor=M.+Jaffarauthor=R.+A.+Bryceauthor=S.+Freeman&title=Aminoimidazolylmethyluracil+analogues+as+potent+inhibitors+of+thymidine+phosphorylase+and+their+bioreductive+nitroimidazolyl+prodrugs&doi=10.1021%2Fjm049494r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1021%2Fjm049494r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049494r%26sid%3Dliteratum%253Aachs%26aulast%3DReigan%26aufirst%3DP.%26aulast%3DEdwards%26aufirst%3DP.%2BN.%26aulast%3DGbaj%26aufirst%3DA.%26aulast%3DCole%26aufirst%3DC.%26aulast%3DBarry%26aufirst%3DS.%2BT.%26aulast%3DPage%26aufirst%3DK.%2BM.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DLuke%26aufirst%3DR.%2BW.%2BA.%26aulast%3DDouglas%26aufirst%3DK.%2BT.%26aulast%3DStratford%26aufirst%3DI.%2BJ.%26aulast%3DJaffar%26aufirst%3DM.%26aulast%3DBryce%26aufirst%3DR.%2BA.%26aulast%3DFreeman%26aufirst%3DS.%26atitle%3DAminoimidazolylmethyluracil%2520analogues%2520as%2520potent%2520inhibitors%2520of%2520thymidine%2520phosphorylase%2520and%2520their%2520bioreductive%2520nitroimidazolyl%2520prodrugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D392%26epage%3D402%26doi%3D10.1021%2Fjm049494r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group">Manikowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lossani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gebhardt, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gambino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Focher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spadari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, G. E.</span><span> </span><span class="NLM_article-title">Inhibition of herpes simplex virus thymidine kinases by 2-phenylamino-6-oxopurines and related compounds: Structure-activity relationships and antiherpetic activity in vivo</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">3919</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1021/jm049059x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049059x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3919-29&author=A.+Manikowskiauthor=A.+Verriauthor=A.+Lossaniauthor=B.+M.+Gebhardtauthor=J.+Gambinoauthor=F.+Focherauthor=S.+Spadariauthor=G.+E.+Wright&title=Inhibition+of+herpes+simplex+virus+thymidine+kinases+by+2-phenylamino-6-oxopurines+and+related+compounds%3A+Structure-activity+relationships+and+antiherpetic+activity+in+vivo&doi=10.1021%2Fjm049059x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1021%2Fjm049059x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049059x%26sid%3Dliteratum%253Aachs%26aulast%3DManikowski%26aufirst%3DA.%26aulast%3DVerri%26aufirst%3DA.%26aulast%3DLossani%26aufirst%3DA.%26aulast%3DGebhardt%26aufirst%3DB.%2BM.%26aulast%3DGambino%26aufirst%3DJ.%26aulast%3DFocher%26aufirst%3DF.%26aulast%3DSpadari%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DG.%2BE.%26atitle%3DInhibition%2520of%2520herpes%2520simplex%2520virus%2520thymidine%2520kinases%2520by%25202-phenylamino-6-oxopurines%2520and%2520related%2520compounds%253A%2520Structure-activity%2520relationships%2520and%2520antiherpetic%2520activity%2520in%2520vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3919%26epage%3D29%26doi%3D10.1021%2Fjm049059x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group">Clements, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beckett, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catlin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lobell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salama, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barynin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, M. G.</span><span> </span><span class="NLM_article-title">Antibiotic activity and characterization of BB-3497, a novel peptide deformylase inhibitor</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1128/AAC.45.2.563-570.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1128%2FAAC.45.2.563-570.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11158755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmsVGktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2001&pages=563-70&author=J.+M.+Clementsauthor=R.+P.+Beckettauthor=A.+Brownauthor=G.+Catlinauthor=M.+Lobellauthor=S.+Palanauthor=W.+Thomasauthor=M.+Whittakerauthor=S.+Woodauthor=S.+Salamaauthor=P.+J.+Bakerauthor=H.+F.+Rodgersauthor=V.+Baryninauthor=D.+W.+Riceauthor=M.+G.+Hunter&title=Antibiotic+activity+and+characterization+of+BB-3497%2C+a+novel+peptide+deformylase+inhibitor&doi=10.1128%2FAAC.45.2.563-570.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic activity and characterization of BB-3497, a novel peptide deformylase inhibitor</span></div><div class="casAuthors">Clements, John M.; Beckett, R. Paul; Brown, Anthony; Catlin, Graham; Lobell, Mario; Palan, Shilpa; Thomas, Wayne; Whittaker, Mark; Wood, Stephen; Salama, Sameeh; Baker, Patrick J.; Rodgers, H. Fiona; Barynin, Vladimir; Rice, David W.; Hunter, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">563-570</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Peptide deformylase (PDF) is an essential bacterial metalloenzyme which deformylates the N-formylmethionine of newly synthesized polypeptides and as such represents a novel target for antibacterial chemotherapy.  To identify novel PDF inhibitors, we screened a metalloenzyme inhibitor library and identified an N-formyl-hydroxylamine deriv., BB-3497 (I, OHCN(OH)CH(Bu)CONHC(Bu-t)CONMe2), and a related natural hydroxamic acid antibiotic, actinonin (II), as potent and selective inhibitors of PDF.  To elucidate the interactions that contribute to the binding affinity of these inhibitors, we detd. the crystal structures of I and II bound to Escherichia coli PDF at resolns. of 2.1 and 1.75 Å, resp.  In both complexes, the active-site metal atom was pentacoordinated by the side chains of Cys 90, His 132, and His 136 and the two oxygen atoms of N-formylhydroxylamine or hydroxamate.  I had activity against gram-pos. bacteria, including methicillin-resistant Staphylococcus aureus and vancomycin-resistant Enterococcus faecalis, and activity against some gram-neg. bacteria.  Time-kill anal. showed that the mode of action of BB-3497 was primarily bacteriostatic.  The mechanism of resistance was via mutations within the formyltransferase gene, as previously described for actinonin.  While II and its derivs. have not been used clin. because of their poor pharmacokinetic properties, I was shown to be orally bioavailable.  A single oral dose of I given 1 h after i.p. injection of S. aureus Smith or methicillin-resistant S. aureus protected mice from infection with median EDs of 8 and 14 mg/kg of body wt., resp.  These data validate PDF as a novel target for the design of a new generation of antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHe1zSuU56sbVg90H21EOLACvtfcHk0lh4c8w5XJLXqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmsVGktA%253D%253D&md5=d7910e4f1a9e8f144e3517ab5e16d512</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1128%2FAAC.45.2.563-570.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.45.2.563-570.2001%26sid%3Dliteratum%253Aachs%26aulast%3DClements%26aufirst%3DJ.%2BM.%26aulast%3DBeckett%26aufirst%3DR.%2BP.%26aulast%3DBrown%26aufirst%3DA.%26aulast%3DCatlin%26aufirst%3DG.%26aulast%3DLobell%26aufirst%3DM.%26aulast%3DPalan%26aufirst%3DS.%26aulast%3DThomas%26aufirst%3DW.%26aulast%3DWhittaker%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DS.%26aulast%3DSalama%26aufirst%3DS.%26aulast%3DBaker%26aufirst%3DP.%2BJ.%26aulast%3DRodgers%26aufirst%3DH.%2BF.%26aulast%3DBarynin%26aufirst%3DV.%26aulast%3DRice%26aufirst%3DD.%2BW.%26aulast%3DHunter%26aufirst%3DM.%2BG.%26atitle%3DAntibiotic%2520activity%2520and%2520characterization%2520of%2520BB-3497%252C%2520a%2520novel%2520peptide%2520deformylase%2520inhibitor%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2001%26volume%3D45%26spage%3D563%26epage%3D70%26doi%3D10.1128%2FAAC.45.2.563-570.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group">Koropatkin, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleland, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holden, H. M.</span><span> </span><span class="NLM_article-title">Kinetic and structural analysis of alpha-d-glucose-1-phosphate cytidylyltransferase from salmonella typhi</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">10774</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1074/jbc.M414111200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1074%2Fjbc.M414111200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=10774-80&author=N.+M.+Koropatkinauthor=W.+W.+Clelandauthor=H.+M.+Holden&title=Kinetic+and+structural+analysis+of+alpha-d-glucose-1-phosphate+cytidylyltransferase+from+salmonella+typhi&doi=10.1074%2Fjbc.M414111200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M414111200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M414111200%26sid%3Dliteratum%253Aachs%26aulast%3DKoropatkin%26aufirst%3DN.%2BM.%26aulast%3DCleland%26aufirst%3DW.%2BW.%26aulast%3DHolden%26aufirst%3DH.%2BM.%26atitle%3DKinetic%2520and%2520structural%2520analysis%2520of%2520alpha-d-glucose-1-phosphate%2520cytidylyltransferase%2520from%2520salmonella%2520typhi%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D10774%26epage%3D80%26doi%3D10.1074%2Fjbc.M414111200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group">Heffron, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurnak, F.</span><span> </span><span class="NLM_article-title">Structure of an EF-Tu complex with a thiazolyl peptide antibiotic determined at 2.35 Å resolution: Atomic basis for GE2270A inhibition of EF-Tu</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.1021/bi9913597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9913597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK1MXnslens7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=37-45&author=S.+E.+Heffronauthor=F.+Jurnak&title=Structure+of+an+EF-Tu+complex+with+a+thiazolyl+peptide+antibiotic+determined+at+2.35+%C3%85+resolution%3A+Atomic+basis+for+GE2270A+inhibition+of+EF-Tu&doi=10.1021%2Fbi9913597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of an EF-Tu Complex with a Thiazolyl Peptide Antibiotic Determined at 2.35 Å Resolution: Atomic Basis for GE2270A Inhibition of EF-Tu</span></div><div class="casAuthors">Heffron, Susan E.; Jurnak, Frances</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-45</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure of a 1:1 M complex between Escherichia coli elongation factor (EF) Tu-GDP and the cyclic thiazolyl peptide antibiotic, GE2270A, has been detd. by x-ray diffraction anal. to a resoln. of 2.35 Å and refined to a crystallog. refinement factor of 20.6%.  The antibiotic binds in the second domain of EF-Tu-GDP, making contact with three segments of amino acids (residues 215-230, 256-264, and 273-277).  The majority of the protein-antibiotic contacts are van der Waals interactions.  A striking feature of the antibiotic binding site is the presence of a salt bridge, not previously obsd. in other EF-Tu complexes.  The ionic interaction between Arg 223 and Glu 259 forms over the antibiotic and probably accounts for the strong affinity obsd. between EF-Tu and GE2270A.  Arg 223 and Glu 259 are highly conserved, but not invariant throughout the prokaryotic EF-Tu family, suggesting that the antibiotic may bind EF-Tu from some organisms better than others may.  Superposition of the antibiotic binding site on the EF-Tu-GTP conformation reveals that one region of the antibiotic would form steric clashes with the guanine nucleotide-binding domain in the GTP, but not the GDP, conformation.  Another region of the antibiotic binds to the same site as the aminoacyl group of tRNA.  Together with prior biochem. studies, the structural findings confirm that GE2270A inhibits protein synthesis by blocking the GDP to GTP conformational change and by directly competing with aminoacyl-tRNA for the same binding site on EF-Tu.  In each of the bacterial strains that are resistant to GE2270A, the effect of a site-specific mutation in EF-Tu could explain resistance.  Comparison of the GE2270A site in EF-Tu with sequence homologs, EF-G and EF-1α, suggests steric clashes that would prevent the antibiotic from binding to translocation factors or to the eukaryotic equiv. of EF-Tu.  Although GE2270A is a potent antibiotic, its clin. efficacy is limited by its low aq. soly.  The results presented here provide the details necessary to enhance the soly. of GE2270A without disrupting its inhibitory properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpde7H00g2l4rVg90H21EOLACvtfcHk0lgRtXL23_bMeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnslens7w%253D&md5=20e6699a9167deedf876b2d4579306ad</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Fbi9913597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9913597%26sid%3Dliteratum%253Aachs%26aulast%3DHeffron%26aufirst%3DS.%2BE.%26aulast%3DJurnak%26aufirst%3DF.%26atitle%3DStructure%2520of%2520an%2520EF-Tu%2520complex%2520with%2520a%2520thiazolyl%2520peptide%2520antibiotic%2520determined%2520at%25202.35%2520%25C3%2585%2520resolution%253A%2520Atomic%2520basis%2520for%2520GE2270A%2520inhibition%2520of%2520EF-Tu%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D37%26epage%3D45%26doi%3D10.1021%2Fbi9913597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group">Wells, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClendon, C. L.</span><span> </span><span class="NLM_article-title">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">450</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-9&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+high-hanging+fruit+in+drug+discovery+at+protein-protein+interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0lgRtXL23_bMeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520high-hanging%2520fruit%2520in%2520drug%2520discovery%2520at%2520protein-protein%2520interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D1001%26epage%3D9%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group">Whitty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumaravel, G.</span><span> </span><span class="NLM_article-title">Between a rock and a hard place?</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1038/nchembio0306-112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchembio0306-112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16484997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVKqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=112-8&author=A.+Whittyauthor=G.+Kumaravel&title=Between+a+rock+and+a+hard+place%3F&doi=10.1038%2Fnchembio0306-112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Between a rock and a hard place?</span></div><div class="casAuthors">Whitty, Adrian; Kumaravel, Gnanasambandam</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">112-118</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Developing small-mol. inhibitors against protein-protein interaction targets is among the most difficult challenges in contemporary drug discovery.  Recent developments in our understanding of this problem, and in the knowledge and tools available to address it, give cause for renewed hope, but substantial challenges remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI04RMmncXC7Vg90H21EOLACvtfcHk0lgRtXL23_bMeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVKqu78%253D&md5=c7a2f9aea89d6be284b1653893a9ece5</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0306-112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0306-112%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DKumaravel%26aufirst%3DG.%26atitle%3DBetween%2520a%2520rock%2520and%2520a%2520hard%2520place%253F%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D112%26epage%3D8%26doi%3D10.1038%2Fnchembio0306-112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group">Fuller, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgoyne, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Predicting druggable binding sites at the protein-protein interface</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">155</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1016/j.drudis.2008.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.drudis.2008.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19041415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSrs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=155-61&author=J.+C.+Fullerauthor=N.+J.+Burgoyneauthor=R.+M.+Jackson&title=Predicting+druggable+binding+sites+at+the+protein-protein+interface&doi=10.1016%2Fj.drudis.2008.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting druggable binding sites at the protein-protein interface</span></div><div class="casAuthors">Fuller, Jonathan C.; Burgoyne, Nicholas J.; Jackson, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3/4</span>),
    <span class="NLM_cas:pages">155-161</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interfaces are highly attractive targets for drug discovery because they are involved in a large no. of disease pathways where therapeutic intervention would bring widespread benefit.  Recent successes have challenged the widely held belief that these targets are undruggable'.  The pocket finding algorithms described here show marked differences between the binding pockets that define protein-protein interactions (PPIs) and those that define protein-ligand interactions (PLIs) of currently marketed drugs.  In the case of PPIs, drug discovery methods that simultaneously target several small pockets at the protein-protein interface are likely to increase the chances of success in this new and important field of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruIDx_ryxRNrVg90H21EOLACvtfcHk0ligsV4xK-vzow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSrs74%253D&md5=b0be246c99c98b82b67dab584fddcf3e</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2008.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2008.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DFuller%26aufirst%3DJ.%2BC.%26aulast%3DBurgoyne%26aufirst%3DN.%2BJ.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DPredicting%2520druggable%2520binding%2520sites%2520at%2520the%2520protein-protein%2520interface%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D155%26epage%3D61%26doi%3D10.1016%2Fj.drudis.2008.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pons, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcillat, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancelin, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krimm, I.</span><span> </span><span class="NLM_article-title">Fragment-based deconstruction of Bcl-x(L) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2577</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.1021/jm100009z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100009z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2577-88&author=S.+Barelierauthor=J.+Ponsauthor=O.+Marcillatauthor=J.+M.+Lancelinauthor=I.+Krimm&title=Fragment-based+deconstruction+of+Bcl-x%28L%29+inhibitors&doi=10.1021%2Fjm100009z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Fjm100009z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100009z%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DPons%26aufirst%3DJ.%26aulast%3DMarcillat%26aufirst%3DO.%26aulast%3DLancelin%26aufirst%3DJ.%2BM.%26aulast%3DKrimm%26aufirst%3DI.%26atitle%3DFragment-based%2520deconstruction%2520of%2520Bcl-x%2528L%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2577%26epage%3D88%26doi%3D10.1021%2Fjm100009z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group">Oltersdorf, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augeri, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belli, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruncko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deckwerth, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinges, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunzer, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Letai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitten, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettesheim, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimmer, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connor, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oleksijew, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomaselli, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wendt, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span> </span><span class="NLM_article-title">An inhibitor of Bcl-2 family proteins induces regression of solid tumours</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">435</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1038/nature03579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnature03579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15902208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1Ogtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2005&pages=677-81&author=T.+Oltersdorfauthor=S.+W.+Elmoreauthor=A.+R.+Shoemakerauthor=R.+C.+Armstrongauthor=D.+J.+Augeriauthor=B.+A.+Belliauthor=M.+Brunckoauthor=T.+L.+Deckwerthauthor=J.+Dingesauthor=P.+J.+Hajdukauthor=M.+K.+Josephauthor=S.+Kitadaauthor=S.+J.+Korsmeyerauthor=A.+R.+Kunzerauthor=A.+Letaiauthor=C.+Liauthor=M.+J.+Mittenauthor=D.+G.+Nettesheimauthor=S.+Ngauthor=P.+M.+Nimmerauthor=J.+M.+O%E2%80%99Connorauthor=A.+Oleksijewauthor=A.+M.+Petrosauthor=J.+C.+Reedauthor=W.+Shenauthor=S.+K.+Tahirauthor=C.+B.+Thompsonauthor=K.+J.+Tomaselliauthor=B.+Wangauthor=M.+D.+Wendtauthor=H.+Zhangauthor=S.+W.+Fesikauthor=S.+H.+Rosenberg&title=An+inhibitor+of+Bcl-2+family+proteins+induces+regression+of+solid+tumours&doi=10.1038%2Fnature03579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of Bcl-2 family proteins induces regression of solid tumours</span></div><div class="casAuthors">Oltersdorf, Tilman; Elmore, Steven W.; Shoemaker, Alexander R.; Armstrong, Robert C.; Augeri, David J.; Belli, Barbara A.; Bruncko, Milan; Deckwerth, Thomas L.; Dinges, Jurgen; Hajduk, Philip J.; Joseph, Mary K.; Kitada, Shinichi; Korsmeyer, Stanley J.; Kunzer, Aaron R.; Letai, Anthony; Li, Chi; Mitten, Michael J.; Nettesheim, David G.; Ng, Shi Chung; Nimmer, Paul M.; O'Connor, Jacqueline M.; Oleksijew, Anatol; Petros, Andrew M.; Reed, John C.; Shen, Wang; Tahir, Stephen K.; Thompson, Craig B.; Tomaselli, Kevin J.; Wang, Baole; Wendt, Michael D.; Zhang, Haichao; Fesik, Stephen W.; Rosenberg, Saul H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">7042</span>),
    <span class="NLM_cas:pages">677-681</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins in the Bcl-2 family are central regulators of programmed cell death, and members that inhibit apoptosis, such as Bcl-XL and Bcl-2, are overexpressed in many cancers and contribute to tumor initiation, progression and resistance to therapy.  Bcl-XL expression correlates with chemo-resistance of tumor cell lines, and redns. in Bcl-2 increase sensitivity to anticancer drugs and enhance in vivo survival.  The development of inhibitors of these proteins as potential anticancer therapeutics has been previously explored, but obtaining potent small-mol. inhibitors has proved difficult owing to the necessity of targeting a protein-protein interaction.  Here, using NMR-based screening, parallel synthesis and structure-based design, the authors have discovered ABT-737, a small-mol. inhibitor of the antiapoptotic proteins Bcl-2, Bcl-XL and Bcl-w, with an affinity two to three orders of magnitude more potent than previously reported compds.  Mechanistic studies reveal that ABT-737 does not directly initiate the apoptotic process, but enhances the effects of death signals, displaying synergistic cytotoxicity with chemotherapeutics and radiation.  ABT-737 exhibits single-agent-mechanism-based killing of cells from lymphoma and small-cell lung carcinoma lines, as well as primary patient-derived cells, and in animal models, ABT-737 improves survival, causes regression of established tumors, and produces cures in a high percentage of the mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBt7_zVt8HrbVg90H21EOLACvtfcHk0ligsV4xK-vzow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1Ogtb0%253D&md5=4940d06838506f181b6310d3458a595f</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1038%2Fnature03579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03579%26sid%3Dliteratum%253Aachs%26aulast%3DOltersdorf%26aufirst%3DT.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DArmstrong%26aufirst%3DR.%2BC.%26aulast%3DAugeri%26aufirst%3DD.%2BJ.%26aulast%3DBelli%26aufirst%3DB.%2BA.%26aulast%3DBruncko%26aufirst%3DM.%26aulast%3DDeckwerth%26aufirst%3DT.%2BL.%26aulast%3DDinges%26aufirst%3DJ.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DJoseph%26aufirst%3DM.%2BK.%26aulast%3DKitada%26aufirst%3DS.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26aulast%3DKunzer%26aufirst%3DA.%2BR.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMitten%26aufirst%3DM.%2BJ.%26aulast%3DNettesheim%26aufirst%3DD.%2BG.%26aulast%3DNg%26aufirst%3DS.%26aulast%3DNimmer%26aufirst%3DP.%2BM.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DJ.%2BM.%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DReed%26aufirst%3DJ.%2BC.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26aulast%3DTomaselli%26aufirst%3DK.%2BJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26atitle%3DAn%2520inhibitor%2520of%2520Bcl-2%2520family%2520proteins%2520induces%2520regression%2520of%2520solid%2520tumours%26jtitle%3DNature%26date%3D2005%26volume%3D435%26spage%3D677%26epage%3D81%26doi%3D10.1038%2Fnature03579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adickes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitten, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimmer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span> </span><span class="NLM_article-title">Abt-263: A potent and orally bioavailable Bcl-2 family inhibitor</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">3421</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-5836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1158%2F0008-5472.CAN-07-5836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18451170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltlSltrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=3421-8&author=C.+Tseauthor=A.+R.+Shoemakerauthor=J.+Adickesauthor=M.+G.+Andersonauthor=J.+Chenauthor=S.+Jinauthor=E.+F.+Johnsonauthor=K.+C.+Marshauthor=M.+J.+Mittenauthor=P.+Nimmerauthor=L.+Robertsauthor=S.+K.+Tahirauthor=Y.+Xiaoauthor=X.+Yangauthor=H.+Zhangauthor=S.+Fesikauthor=S.+H.+Rosenbergauthor=S.+W.+Elmore&title=Abt-263%3A+A+potent+and+orally+bioavailable+Bcl-2+family+inhibitor&doi=10.1158%2F0008-5472.CAN-07-5836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">ABT-263: A Potent and Orally Bioavailable Bcl-2 Family Inhibitor</span></div><div class="casAuthors">Tse, Christin; Shoemaker, Alexander R.; Adickes, Jessica; Anderson, Mark G.; Chen, Jun; Jin, Sha; Johnson, Eric F.; Marsh, Kennan C.; Mitten, Michael J.; Nimmer, Paul; Roberts, Lisa; Tahir, Stephen K.; Xiao, Yu; Yang, Xiufen; Zhang, Haichao; Fesik, Stephen; Rosenberg, Saul H.; Elmore, Steven W.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3421-3428</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Overexpression of the prosurvival Bcl-2 family members (Bcl-2, Bcl-xL, and Mcl-1) is commonly assocd. with tumor maintenance, progression, and chemoresistance.  We previously reported the discovery of ABT-737, a potent, small-mol. Bcl-2 family protein inhibitor.  A major limitation of ABT-737 is that it is not orally bioavailable, which would limit chronic single agent therapy and flexibility to dose in combination regimens.  Here we report the biol. properties of ABT-263, a potent, orally bioavailable Bad-like BH3 mimetic (Ki's of <1 nmol/L for Bcl-2, Bcl-xL, and Bcl-w).  The oral bioavailability of ABT-263 in preclin. animal models is 20% to 50%, depending on formulation.  ABT-263 disrupts Bcl-2/Bcl-xL interactions with pro-death proteins (e.g., Bim), leading to the initiation of apoptosis within 2 h posttreatment.  In human tumor cells, ABT-263 induces Bax translocation, cytochrome c release, and subsequent apoptosis.  Oral administration of ABT-263 alone induces complete tumor regressions in xenograft models of small-cell lung cancer and acute lymphoblastic leukemia.  In xenograft models of aggressive B-cell lymphoma and multiple myeloma where ABT-263 exhibits modest or no single agent activity, it significantly enhances the efficacy of clin. relevant therapeutic regimens.  These data provide the rationale for clin. trials evaluating ABT-263 in small-cell lung cancer and B-cell malignancies.  The oral efficacy of ABT-263 should provide dosing flexibility to maximize clin. utility both as a single agent and in combination regimens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojLK8zIgyDWbVg90H21EOLACvtfcHk0li21bJWo0kZwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXltlSltrs%253D&md5=a43029cc218d25d4e1fb6dd1186475c4</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-5836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-5836%26sid%3Dliteratum%253Aachs%26aulast%3DTse%26aufirst%3DC.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DAdickes%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMitten%26aufirst%3DM.%2BJ.%26aulast%3DNimmer%26aufirst%3DP.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFesik%26aufirst%3DS.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3DAbt-263%253A%2520A%2520potent%2520and%2520orally%2520bioavailable%2520Bcl-2%2520family%2520inhibitor%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D3421%26epage%3D8%26doi%3D10.1158%2F0008-5472.CAN-07-5836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group">Tao, Z. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasvold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boghaert, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catron, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colman, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czabotar, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshayes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairbrother, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flygare, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hymowitz, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judge, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koehler, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovar, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lessene, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitten, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ndubaku, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimmer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purkey, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oleksijew, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sleebs, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zobel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leverson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souers, A. J.</span><span> </span><span class="NLM_article-title">Discovery of a potent and selective bcl-xl inhibitor with in vivo activity</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1088</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.1021/ml5001867</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5001867" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWrtbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1088-93&author=Z.+F.+Taoauthor=L.+Hasvoldauthor=L.+Wangauthor=X.+Wangauthor=A.+M.+Petrosauthor=C.+H.+Parkauthor=E.+R.+Boghaertauthor=N.+D.+Catronauthor=J.+Chenauthor=P.+M.+Colmanauthor=P.+E.+Czabotarauthor=K.+Deshayesauthor=W.+J.+Fairbrotherauthor=J.+A.+Flygareauthor=S.+G.+Hymowitzauthor=S.+Jinauthor=R.+A.+Judgeauthor=M.+F.+Koehlerauthor=P.+J.+Kovarauthor=G.+Lesseneauthor=M.+J.+Mittenauthor=C.+O.+Ndubakuauthor=P.+Nimmerauthor=H.+E.+Purkeyauthor=A.+Oleksijewauthor=D.+C.+Phillipsauthor=B.+E.+Sleebsauthor=B.+J.+Smithauthor=M.+L.+Smithauthor=S.+K.+Tahirauthor=K.+G.+Watsonauthor=Y.+Xiaoauthor=J.+Xueauthor=H.+Zhangauthor=K.+Zobelauthor=S.+H.+Rosenbergauthor=C.+Tseauthor=J.+D.+Leversonauthor=S.+W.+Elmoreauthor=A.+J.+Souers&title=Discovery+of+a+potent+and+selective+bcl-xl+inhibitor+with+in+vivo+activity&doi=10.1021%2Fml5001867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective BCL-XL Inhibitor with in Vivo Activity</span></div><div class="casAuthors">Tao, Zhi-Fu; Hasvold, Lisa; Wang, Le; Wang, Xilu; Petros, Andrew M.; Park, Chang H.; Boghaert, Erwin R.; Catron, Nathaniel D.; Chen, Jun; Colman, Peter M.; Czabotar, Peter E.; Deshayes, Kurt; Fairbrother, Wayne J.; Flygare, John A.; Hymowitz, Sarah G.; Jin, Sha; Judge, Russell A.; Koehler, Michael F. T.; Kovar, Peter J.; Lessene, Guillaume; Mitten, Michael J.; Ndubaku, Chudi O.; Nimmer, Paul; Purkey, Hans E.; Oleksijew, Anatol; Phillips, Darren C.; Sleebs, Brad E.; Smith, Brian J.; Smith, Morey L.; Tahir, Stephen K.; Watson, Keith G.; Xiao, Yu; Xue, John; Zhang, Haichao; Zobel, Kerry; Rosenberg, Saul H.; Tse, Chris; Leverson, Joel D.; Elmore, Steven W.; Souers, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1088-1093</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A-1155463 I, a highly potent and selective BCL-XL inhibitor, was discovered through NMR fragment screening and structure-based design.  This compd. is substantially more potent against BCL-XL-dependent cell lines relative to the recently reported inhibitor, WEHI-539, while possessing none of its inherent pharmaceutical liabilities.  A-1155463 caused a mechanism-based and reversible thrombocytopenia in mice and inhibited H146 small cell lung cancer xenograft tumor growth in vivo following multiple doses.  A-1155463 thus represents an excellent tool mol. for studying BCL-XL biol. as well as a productive lead structure for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozGRbIa-4nxbVg90H21EOLACvtfcHk0li21bJWo0kZwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWrtbnN&md5=5d2fffa59286214efac3f320a701215a</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Fml5001867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5001867%26sid%3Dliteratum%253Aachs%26aulast%3DTao%26aufirst%3DZ.%2BF.%26aulast%3DHasvold%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DBoghaert%26aufirst%3DE.%2BR.%26aulast%3DCatron%26aufirst%3DN.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DColman%26aufirst%3DP.%2BM.%26aulast%3DCzabotar%26aufirst%3DP.%2BE.%26aulast%3DDeshayes%26aufirst%3DK.%26aulast%3DFairbrother%26aufirst%3DW.%2BJ.%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26aulast%3DHymowitz%26aufirst%3DS.%2BG.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DJudge%26aufirst%3DR.%2BA.%26aulast%3DKoehler%26aufirst%3DM.%2BF.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DLessene%26aufirst%3DG.%26aulast%3DMitten%26aufirst%3DM.%2BJ.%26aulast%3DNdubaku%26aufirst%3DC.%2BO.%26aulast%3DNimmer%26aufirst%3DP.%26aulast%3DPurkey%26aufirst%3DH.%2BE.%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DPhillips%26aufirst%3DD.%2BC.%26aulast%3DSleebs%26aufirst%3DB.%2BE.%26aulast%3DSmith%26aufirst%3DB.%2BJ.%26aulast%3DSmith%26aufirst%3DM.%2BL.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DWatson%26aufirst%3DK.%2BG.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZobel%26aufirst%3DK.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DLeverson%26aufirst%3DJ.%2BD.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DSouers%26aufirst%3DA.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520bcl-xl%2520inhibitor%2520with%2520in%2520vivo%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1088%26epage%3D93%26doi%3D10.1021%2Fml5001867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group">Flygare, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budha, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheeti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshayes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doerner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckhardt, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franklin, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reisner, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gazzard, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halladay, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hymowitz, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LoRusso, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plise, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephan, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tom, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Um, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varfolomeev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vucic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallweber, H. J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ware, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, Z. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zobel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairbrother, W. J.</span><span> </span><span class="NLM_article-title">Discovery of a potent small-molecule antagonist of inhibitor of apoptosis (IAP) proteins and clinical candidate for the treatment of cancer (GDC-0152)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4101</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1021/jm300060k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300060k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Ois74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4101-13&author=J.+A.+Flygareauthor=M.+Beresiniauthor=N.+Budhaauthor=H.+Chanauthor=I.+T.+Chanauthor=S.+Cheetiauthor=F.+Cohenauthor=K.+Deshayesauthor=K.+Doernerauthor=S.+G.+Eckhardtauthor=L.+O.+Elliottauthor=B.+N.+Fengauthor=M.+C.+Franklinauthor=S.+F.+Reisnerauthor=L.+Gazzardauthor=J.+Halladayauthor=S.+G.+Hymowitzauthor=H.+Laauthor=P.+LoRussoauthor=B.+Maurerauthor=L.+Murrayauthor=E.+Pliseauthor=C.+Quanauthor=J.+P.+Stephanauthor=S.+G.+Youngauthor=J.+Tomauthor=V.+Tsuiauthor=J.+Umauthor=E.+Varfolomeevauthor=D.+Vucicauthor=A.+J.+Wagnerauthor=H.+J.+A.+Wallweberauthor=L.+Wangauthor=J.+Wareauthor=Z.+Y.+Wenauthor=H.+Wongauthor=J.+M.+Wongauthor=M.+Wongauthor=S.+Wongauthor=R.+Yuauthor=K.+Zobelauthor=W.+J.+Fairbrother&title=Discovery+of+a+potent+small-molecule+antagonist+of+inhibitor+of+apoptosis+%28IAP%29+proteins+and+clinical+candidate+for+the+treatment+of+cancer+%28GDC-0152%29&doi=10.1021%2Fjm300060k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a potent small-molecule antagonist of inhibitor of apoptosis (IAP) proteins and clinical candidate for the treatment of cancer (GDC-0152)</span></div><div class="casAuthors">Flygare, John A.; Beresini, Maureen; Budha, Nageshwar; Chan, Helen; Chan, Iris T.; Cheeti, Sravanthi; Cohen, Frederick; Deshayes, Kurt; Doerner, Karl; Eckhardt, S. Gail; Elliott, Linda O.; Feng, Bainian; Franklin, Matthew C.; Reisner, Stacy Frankovitz; Gazzard, Lewis; Halladay, Jason; Hymowitz, Sarah G.; La, Hank; LoRusso, Patricia; Maurer, Brigitte; Murray, Lesley; Plise, Emile; Quan, Clifford; Stephan, Jean-Philippe; Young, Shin G.; Tom, Jeffrey; Tsui, Vickie; Um, Joanne; Varfolomeev, Eugene; Vucic, Domagoj; Wagner, Andrew J.; Wallweber, Heidi J. A.; Wang, Lan; Ware, Joseph; Wen, Zhaoyang; Wong, Harvey; Wong, Jonathan M.; Wong, Melisa; Wong, Susan; Yu, Ron; Zobel, Kerry; Fairbrother, Wayne J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4101-4113</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of compds. were designed and synthesized as antagonists of cIAP1/2, ML-IAP, and XIAP based on the N-terminus, AVPI, of mature Smac.  Compd. (I) (GDC-0152) has the best profile of these compds.; it binds to the XIAP BIR3 domain, the BIR domain of ML-IAP, and the BIR3 domains of cIAP1 and cIAP2 with Ki values of 28, 14, 17, and 43 nM, resp.  These compds. promote degrdn. of cIAP1, induce activation of caspase-3/7, and lead to decreased viability of breast cancer cells without affecting normal mammary epithelial cells.  Compd. I inhibits tumor growth when dosed orally in the MDA-MB-231 breast cancer xenograft model.  Compd. I was advanced to human clin. trials, and it exhibited linear pharmacokinetics over the dose range (0.049 to 1.48 mg/kg) tested.  Mean plasma clearance in humans was 9 ± 3 mL/min/kg, and the vol. of distribution was 0.6 ± 0.2 L/kg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZSPNVJ5gVWrVg90H21EOLACvtfcHk0lhmseof64EXqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Ois74%253D&md5=19452878ea834d3c324d567b39a78fb2</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1021%2Fjm300060k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300060k%26sid%3Dliteratum%253Aachs%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26aulast%3DBeresini%26aufirst%3DM.%26aulast%3DBudha%26aufirst%3DN.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DChan%26aufirst%3DI.%2BT.%26aulast%3DCheeti%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DDeshayes%26aufirst%3DK.%26aulast%3DDoerner%26aufirst%3DK.%26aulast%3DEckhardt%26aufirst%3DS.%2BG.%26aulast%3DElliott%26aufirst%3DL.%2BO.%26aulast%3DFeng%26aufirst%3DB.%2BN.%26aulast%3DFranklin%26aufirst%3DM.%2BC.%26aulast%3DReisner%26aufirst%3DS.%2BF.%26aulast%3DGazzard%26aufirst%3DL.%26aulast%3DHalladay%26aufirst%3DJ.%26aulast%3DHymowitz%26aufirst%3DS.%2BG.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DLoRusso%26aufirst%3DP.%26aulast%3DMaurer%26aufirst%3DB.%26aulast%3DMurray%26aufirst%3DL.%26aulast%3DPlise%26aufirst%3DE.%26aulast%3DQuan%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DJ.%2BP.%26aulast%3DYoung%26aufirst%3DS.%2BG.%26aulast%3DTom%26aufirst%3DJ.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DUm%26aufirst%3DJ.%26aulast%3DVarfolomeev%26aufirst%3DE.%26aulast%3DVucic%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DA.%2BJ.%26aulast%3DWallweber%26aufirst%3DH.%2BJ.%2BA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWare%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DZ.%2BY.%26aulast%3DWong%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DJ.%2BM.%26aulast%3DWong%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DZobel%26aufirst%3DK.%26aulast%3DFairbrother%26aufirst%3DW.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520small-molecule%2520antagonist%2520of%2520inhibitor%2520of%2520apoptosis%2520%2528IAP%2529%2520proteins%2520and%2520clinical%2520candidate%2520for%2520the%2520treatment%2520of%2520cancer%2520%2528GDC-0152%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4101%26epage%3D13%26doi%3D10.1021%2Fjm300060k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group">Seigal, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connors, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borzilleri, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuel, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fargnoli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naglich, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pokross, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Posy, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talbott, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terrett, N. K.</span><span> </span><span class="NLM_article-title">The discovery of macrocyclic XIAP antagonists from a DNA-programmed chemistry library, and their optimization to give lead compounds with in vivo antitumor activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2855</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1021/jm501892g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501892g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivF2qu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2855-61&author=B.+A.+Seigalauthor=W.+H.+Connorsauthor=A.+Fraleyauthor=R.+M.+Borzilleriauthor=P.+H.+Carterauthor=S.+L.+Emanuelauthor=J.+Fargnoliauthor=K.+Kimauthor=M.+Leiauthor=J.+G.+Naglichauthor=M.+E.+Pokrossauthor=S.+L.+Posyauthor=H.+Shenauthor=N.+Surtiauthor=R.+Talbottauthor=Y.+Zhangauthor=N.+K.+Terrett&title=The+discovery+of+macrocyclic+XIAP+antagonists+from+a+DNA-programmed+chemistry+library%2C+and+their+optimization+to+give+lead+compounds+with+in+vivo+antitumor+activity&doi=10.1021%2Fjm501892g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of Macrocyclic XIAP Antagonists from a DNA-Programmed Chemistry Library, and Their Optimization To Give Lead Compounds with in Vivo Antitumor Activity</span></div><div class="casAuthors">Seigal, Benjamin A.; Connors, William H.; Fraley, Andrew; Borzilleri, Robert M.; Carter, Percy H.; Emanuel, Stuart L.; Fargnoli, Joseph; Kim, Kyoung; Lei, Ming; Naglich, Joseph G.; Pokross, Matthew E.; Posy, Shana L.; Shen, Henry; Surti, Neha; Talbott, Randy; Zhang, Yong; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2855-2861</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Affinity selection screening of macrocycle libraries derived from DNA-programmed chem. identified XIAP BIR2 and BIR3 domain inhibitors that displace bound pro-apoptotic caspases.  X-ray cocrystal structures of key compds. with XIAP BIR2 suggested potency-enhancing structural modifications.  Optimization of dimeric macrocycles with similar affinity for both domains were potent pro-apoptotic agents in cancer cell lines and efficacious in shrinking tumors in a mouse xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlB6i1KOFJjrVg90H21EOLACvtfcHk0lhmseof64EXqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivF2qu78%253D&md5=9bd53cc3ed014796248e8048cecea0d1</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1021%2Fjm501892g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501892g%26sid%3Dliteratum%253Aachs%26aulast%3DSeigal%26aufirst%3DB.%2BA.%26aulast%3DConnors%26aufirst%3DW.%2BH.%26aulast%3DFraley%26aufirst%3DA.%26aulast%3DBorzilleri%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DEmanuel%26aufirst%3DS.%2BL.%26aulast%3DFargnoli%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DK.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DNaglich%26aufirst%3DJ.%2BG.%26aulast%3DPokross%26aufirst%3DM.%2BE.%26aulast%3DPosy%26aufirst%3DS.%2BL.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DSurti%26aufirst%3DN.%26aulast%3DTalbott%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520discovery%2520of%2520macrocyclic%2520XIAP%2520antagonists%2520from%2520a%2520DNA-programmed%2520chemistry%2520library%252C%2520and%2520their%2520optimization%2520to%2520give%2520lead%2520compounds%2520with%2520in%2520vivo%2520antitumor%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2855%26epage%3D61%26doi%3D10.1021%2Fjm501892g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group">Vassilev, L. T.</span><span> </span><span class="NLM_article-title">MDM2 inhibitors for cancer therapy</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1016/j.molmed.2006.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.molmed.2006.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17126603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1CrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=23-31&author=L.+T.+Vassilev&title=MDM2+inhibitors+for+cancer+therapy&doi=10.1016%2Fj.molmed.2006.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">MDM2 inhibitors for cancer therapy</span></div><div class="casAuthors">Vassilev, Lyubomir T.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-31</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The tumor suppressor p53 is a powerful antitumoral mol. frequently inactivated by mutations or deletions in cancer.  However, half of all human tumors express wild-type p53, and its activation by antagonizing its neg. regulator murine double minute 2 (MDM2) might offer a new therapeutic strategy.  Proof-of-concept expts. have demonstrated the feasibility of this approach in vitro but the development of pharmacol. inhibitors has been challenging.  Recently, potent and selective small-mol. MDM2 inhibitors have been identified.  Studies with these compds. have strengthened the concept that selective, non-genotoxic p53 activation is a viable alternative to current cytotoxic chemotherapy but clin. validation is still pending.  Here, the new developments in the quest for pharmacol. p53 activators are reviewed with an emphasis on small-mol. inhibitors of the p53-MDM2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYnDenIZoV0rVg90H21EOLACvtfcHk0lhmseof64EXqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1CrsQ%253D%253D&md5=a0bb8aaed6ec192280538a6ee1f908fa</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2006.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2006.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DVassilev%26aufirst%3DL.%2BT.%26atitle%3DMDM2%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2007%26volume%3D13%26spage%3D23%26epage%3D31%26doi%3D10.1016%2Fj.molmed.2006.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of the MDM2-p53 protein-protein interaction (MDM2 inhibitors) in clinical trials for cancer treatment</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1038</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1021/jm501092z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501092z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFagt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1038-52&author=Y.+Zhaoauthor=A.+Aguilarauthor=D.+Bernardauthor=S.+Wang&title=Small-molecule+inhibitors+of+the+MDM2-p53+protein-protein+interaction+%28MDM2+inhibitors%29+in+clinical+trials+for+cancer+treatment&doi=10.1021%2Fjm501092z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of the MDM2-p53 Protein-Protein Interaction (MDM2 Inhibitors) in Clinical Trials for Cancer Treatment</span></div><div class="casAuthors">Zhao, Yujun; Aguilar, Angelo; Bernard, Denzil; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1038-1052</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Design of small-mol. inhibitors (MDM2 inhibitors) to block the MDM2-p53 protein-protein interaction has been pursued as a new cancer therapeutic strategy.  In recent years, potent, selective, and efficacious MDM2 inhibitors have been successfully obtained and seven such compds. have been advanced into early phase clin. trials for the treatment of human cancers.  Here, we review the design, synthesis, properties, preclin., and clin. studies of these clin.-stage MDM2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNa9aW1Kz1Q7Vg90H21EOLACvtfcHk0lj3TRYhfM1Z4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFagt77P&md5=2e58318782d7e0a04bf3398db8782984</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Fjm501092z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501092z%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520the%2520MDM2-p53%2520protein-protein%2520interaction%2520%2528MDM2%2520inhibitors%2529%2520in%2520clinical%2520trials%2520for%2520cancer%2520treatment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1038%26epage%3D52%26doi%3D10.1021%2Fjm501092z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group">Sun, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rew, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartberger, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deignan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquette, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eksterowicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, A. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Lopez De Turiso, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houze, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kayser, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGee, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McIntosh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mihalic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliner, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osgood, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roveto, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saiki, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaffer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toteva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wortman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yakowec, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, J. C.</span><span> </span><span class="NLM_article-title">Discovery of AMG 232, a potent, selective, and orally bioavailable MDM2-p53 inhibitor in clinical development</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1454</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1021/jm401753e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401753e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1eitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1454-72&author=D.+Sunauthor=Z.+Liauthor=Y.+Rewauthor=M.+Gribbleauthor=M.+D.+Bartbergerauthor=H.+P.+Beckauthor=J.+Canonauthor=A.+Chenauthor=X.+Chenauthor=D.+Chowauthor=J.+Deignanauthor=J.+Duquetteauthor=J.+Eksterowiczauthor=B.+Fisherauthor=B.+M.+Foxauthor=J.+Fuauthor=A.+Z.+Gonzalezauthor=F.+Gonzalez-Lopez+De+Turisoauthor=J.+B.+Houzeauthor=X.+Huangauthor=M.+Jiangauthor=L.+Jinauthor=F.+Kayserauthor=J.+J.+Liuauthor=M.+C.+Loauthor=A.+M.+Longauthor=B.+Lucasauthor=L.+R.+McGeeauthor=J.+McIntoshauthor=J.+Mihalicauthor=J.+D.+Olinerauthor=T.+Osgoodauthor=M.+L.+Petersonauthor=P.+Rovetoauthor=A.+Y.+Saikiauthor=P.+Shafferauthor=M.+Totevaauthor=Y.+Wangauthor=Y.+C.+Wangauthor=S.+Wortmanauthor=P.+Yakowecauthor=X.+Yanauthor=Q.+Yeauthor=D.+Yuauthor=M.+Yuauthor=X.+Zhaoauthor=J.+Zhouauthor=J.+Zhuauthor=S.+H.+Olsonauthor=J.+C.+Medina&title=Discovery+of+AMG+232%2C+a+potent%2C+selective%2C+and+orally+bioavailable+MDM2-p53+inhibitor+in+clinical+development&doi=10.1021%2Fjm401753e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of AMG 232, a Potent, Selective, and Orally Bioavailable MDM2-p53 Inhibitor in Clinical Development</span></div><div class="casAuthors">Sun, Daqing; Li, Zhihong; Rew, Yosup; Gribble, Michael; Bartberger, Michael D.; Beck, Hilary P.; Canon, Jude; Chen, Ada; Chen, Xiaoqi; Chow, David; Deignan, Jeffrey; Duquette, Jason; Eksterowicz, John; Fisher, Benjamin; Fox, Brian M.; Fu, Jiasheng; Gonzalez, Ana Z.; Gonzalez-Lopez De Turiso, Felix; Houze, Jonathan B.; Huang, Xin; Jiang, Min; Jin, Lixia; Kayser, Frank; Liu, Jiwen; Lo, Mei-Chu; Long, Alexander M.; Lucas, Brian; McGee, Lawrence R.; McIntosh, Joel; Mihalic, Jeff; Oliner, Jonathan D.; Osgood, Tao; Peterson, Matthew L.; Roveto, Philip; Saiki, Anne Y.; Shaffer, Paul; Toteva, Maria; Wang, Yingcai; Wang, Yu Chung; Wortman, Sarah; Yakowec, Peter; Yan, Xuelei; Ye, Qiuping; Yu, Dongyin; Yu, Ming; Zhao, Xiaoning; Zhou, Jing; Zhu, Jiang; Olson, Steven H.; Medina, Julio C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1454-1472</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We recently reported the discovery of AM-8553 (1), a potent and selective piperidinone inhibitor of the MDM2-p53 interaction.  Continued research investigation of the N-alkyl substituent of this series, focused in particular on a previously underutilized interaction in a shallow cleft on the MDM2 surface, led to the discovery of a one-carbon tethered sulfone which gave rise to substantial improvements in biochem. and cellular potency.  Further investigation produced AMG 232 (2), which is currently being evaluated in human clin. trials for the treatment of cancer.  Compd. 2 is an extremely potent MDM2 inhibitor (SPR KD = 0.045 nM, SJSA-1 EdU IC50 = 9.1 nM), with remarkable pharmacokinetic properties and in vivo antitumor activity in the SJSA-1 osteosarcoma xenograft model (ED50 = 9.1 mg/kg).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjly0O6c42NbVg90H21EOLACvtfcHk0lj3TRYhfM1Z4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1eitbY%253D&md5=91197d328d40f6fa2f6c37819149427f</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1021%2Fjm401753e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401753e%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DRew%26aufirst%3DY.%26aulast%3DGribble%26aufirst%3DM.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DBeck%26aufirst%3DH.%2BP.%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChow%26aufirst%3DD.%26aulast%3DDeignan%26aufirst%3DJ.%26aulast%3DDuquette%26aufirst%3DJ.%26aulast%3DEksterowicz%26aufirst%3DJ.%26aulast%3DFisher%26aufirst%3DB.%26aulast%3DFox%26aufirst%3DB.%2BM.%26aulast%3DFu%26aufirst%3DJ.%26aulast%3DGonzalez%26aufirst%3DA.%2BZ.%26aulast%3DGonzalez-Lopez%2BDe%2BTuriso%26aufirst%3DF.%26aulast%3DHouze%26aufirst%3DJ.%2BB.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKayser%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DJ.%2BJ.%26aulast%3DLo%26aufirst%3DM.%2BC.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DLucas%26aufirst%3DB.%26aulast%3DMcGee%26aufirst%3DL.%2BR.%26aulast%3DMcIntosh%26aufirst%3DJ.%26aulast%3DMihalic%26aufirst%3DJ.%26aulast%3DOliner%26aufirst%3DJ.%2BD.%26aulast%3DOsgood%26aufirst%3DT.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26aulast%3DRoveto%26aufirst%3DP.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DShaffer%26aufirst%3DP.%26aulast%3DToteva%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DWortman%26aufirst%3DS.%26aulast%3DYakowec%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DOlson%26aufirst%3DS.%2BH.%26aulast%3DMedina%26aufirst%3DJ.%2BC.%26atitle%3DDiscovery%2520of%2520AMG%2520232%252C%2520a%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520MDM2-p53%2520inhibitor%2520in%2520clinical%2520development%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1454%26epage%3D72%26doi%3D10.1021%2Fjm401753e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group">Yu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartberger, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eksterowicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMinn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliner, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osgood, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rew, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saiki, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaffer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, D.</span><span> </span><span class="NLM_article-title">Discovery of potent and simplified piperidinone-based inhibitors of the MDM2-p53 interaction</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">894</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1021/ml500142b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500142b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=894-9&author=M.+Yuauthor=Y.+Wangauthor=J.+Zhuauthor=M.+D.+Bartbergerauthor=J.+Canonauthor=A.+Chenauthor=D.+Chowauthor=J.+Eksterowiczauthor=B.+Foxauthor=J.+Fuauthor=M.+Gribbleauthor=X.+Huangauthor=Z.+Liauthor=J.+J.+Liuauthor=M.+C.+Loauthor=D.+McMinnauthor=J.+D.+Olinerauthor=T.+Osgoodauthor=Y.+Rewauthor=A.+Y.+Saikiauthor=P.+Shafferauthor=X.+Yanauthor=Q.+Yeauthor=D.+Yuauthor=X.+Zhaoauthor=J.+Zhouauthor=S.+H.+Olsonauthor=J.+C.+Medinaauthor=D.+Sun&title=Discovery+of+potent+and+simplified+piperidinone-based+inhibitors+of+the+MDM2-p53+interaction&doi=10.1021%2Fml500142b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1021%2Fml500142b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500142b%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DChow%26aufirst%3DD.%26aulast%3DEksterowicz%26aufirst%3DJ.%26aulast%3DFox%26aufirst%3DB.%26aulast%3DFu%26aufirst%3DJ.%26aulast%3DGribble%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DJ.%2BJ.%26aulast%3DLo%26aufirst%3DM.%2BC.%26aulast%3DMcMinn%26aufirst%3DD.%26aulast%3DOliner%26aufirst%3DJ.%2BD.%26aulast%3DOsgood%26aufirst%3DT.%26aulast%3DRew%26aufirst%3DY.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DShaffer%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DOlson%26aufirst%3DS.%2BH.%26aulast%3DMedina%26aufirst%3DJ.%2BC.%26aulast%3DSun%26aufirst%3DD.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520simplified%2520piperidinone-based%2520inhibitors%2520of%2520the%2520MDM2-p53%2520interaction%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D894%26epage%3D9%26doi%3D10.1021%2Fml500142b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group">Wendt, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rockway, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geyer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClellan, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weitzberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mantei, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nienaber, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinghofer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giranda, V. L.</span><span> </span><span class="NLM_article-title">Identification of novel binding interactions in the development of potent, selective 2-naphthamidine inhibitors of urokinase. Synthesis, structural analysis, and SAR of N-phenyl amide 6-substitution</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">303</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span><span class="refDoi"> DOI: 10.1021/jm0300072</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0300072" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=303-24&author=M.+D.+Wendtauthor=T.+W.+Rockwayauthor=A.+Geyerauthor=W.+McClellanauthor=M.+Weitzbergauthor=X.+Zhaoauthor=R.+Manteiauthor=V.+L.+Nienaberauthor=K.+Stewartauthor=V.+Klinghoferauthor=V.+L.+Giranda&title=Identification+of+novel+binding+interactions+in+the+development+of+potent%2C+selective+2-naphthamidine+inhibitors+of+urokinase.+Synthesis%2C+structural+analysis%2C+and+SAR+of+N-phenyl+amide+6-substitution&doi=10.1021%2Fjm0300072"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1021%2Fjm0300072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0300072%26sid%3Dliteratum%253Aachs%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DRockway%26aufirst%3DT.%2BW.%26aulast%3DGeyer%26aufirst%3DA.%26aulast%3DMcClellan%26aufirst%3DW.%26aulast%3DWeitzberg%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DMantei%26aufirst%3DR.%26aulast%3DNienaber%26aufirst%3DV.%2BL.%26aulast%3DStewart%26aufirst%3DK.%26aulast%3DKlinghofer%26aufirst%3DV.%26aulast%3DGiranda%26aufirst%3DV.%2BL.%26atitle%3DIdentification%2520of%2520novel%2520binding%2520interactions%2520in%2520the%2520development%2520of%2520potent%252C%2520selective%25202-naphthamidine%2520inhibitors%2520of%2520urokinase.%2520Synthesis%252C%2520structural%2520analysis%252C%2520and%2520SAR%2520of%2520N-phenyl%2520amide%25206-substitution%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D303%26epage%3D24%26doi%3D10.1021%2Fjm0300072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group">D’Abramo, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archambault, J.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of human papillomavirus protein - protein interactions</span> <span class="citation_source-journal">Open Virol. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.2174/1874357901105010080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.2174%2F1874357901105010080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21769307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFWiu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=80-95&author=C.+M.+D%E2%80%99Abramoauthor=J.+Archambault&title=Small+molecule+inhibitors+of+human+papillomavirus+protein+-+protein+interactions&doi=10.2174%2F1874357901105010080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of human papillomavirus protein - protein interactions</span></div><div class="casAuthors">D'Abramo, C. M.; Archambault, J.</div><div class="citationInfo"><span class="NLM_cas:title">Open Virology Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">80-95</span>CODEN:
                <span class="NLM_cas:coden">OVJPAJ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3579</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Human papillomaviruses (HPV) have now been identified as a necessary cause of benign and malignant lesions of the differentiating epithelium, particularly cervical cancer, the second most prevalent cancer in women worldwide.  While two prophylactic HPV vaccines and screening programs are available, there is currently no antiviral drug for the treatment of HPV infections and assocd. diseases.  The recent progress toward the identification and characterization of specific mol. targets for small mol.-based approaches provides prospect for the development of effective HPV antiviral compds.  Traditionally, antiviral therapies target viral enzymes.  HPV encode for few proteins, however, and rely extensively on the infected cell for completion of their life cycle.  This article will review the functions of the viral E1 helicase, which encodes the only enzymic function of the virus, of the E2 regulatory protein, and of the viral E6 and E7 oncogenes in viral replication and pathogenesis.  Particular emphasis will be placed on the recent progress made towards the development of novel small mol. inhibitors that specifically target and inhibit the functions of these viral proteins, as well as their interactions with other viral and/or cellular proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrubY6vJhjR2bVg90H21EOLACvtfcHk0lgbXc9Uyp5jDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFWiu73O&md5=1013920c1efdefb50d1672e4ff145a12</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.2174%2F1874357901105010080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1874357901105010080%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Abramo%26aufirst%3DC.%2BM.%26aulast%3DArchambault%26aufirst%3DJ.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520human%2520papillomavirus%2520protein%2520-%2520protein%2520interactions%26jtitle%3DOpen%2520Virol.%2520J.%26date%3D2011%26volume%3D5%26spage%3D80%26epage%3D95%26doi%3D10.2174%2F1874357901105010080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group">Sun, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olejniczak, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waterson, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossanese, O. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovery of small molecules that bind to K-Ras and inhibit SOS-mediated activation</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6140</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span><span class="refDoi"> DOI: 10.1002/anie.201201358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2Fanie.201201358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38Xms1entr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=6140-3&author=Q.+Sunauthor=J.+P.+Burkeauthor=J.+Phanauthor=M.+C.+Burnsauthor=E.+T.+Olejniczakauthor=A.+G.+Watersonauthor=T.+Leeauthor=O.+W.+Rossaneseauthor=S.+W.+Fesik&title=Discovery+of+small+molecules+that+bind+to+K-Ras+and+inhibit+SOS-mediated+activation&doi=10.1002%2Fanie.201201358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecules that Bind to K-Ras and Inhibit Sos-Mediated Activation</span></div><div class="casAuthors">Sun, Qi; Burke, Jason P.; Phan, Jason; Burns, Michael C.; Olejniczak, Edward T.; Waterson, Alex G.; Lee, Taekyu; Rossanese, Olivia W.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">6140-6143, S6140/1-S6140/17</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">K-Ras inhibition represents an attractive therapeutic strategy for many cancers.  We report on the discovery of novel small mols. that bind directly to K-Ras between switch I and switch II and inhibit Sos-catalyzed K-Ras activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy5Cx7BSO1pLVg90H21EOLACvtfcHk0lgbXc9Uyp5jDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xms1entr4%253D&md5=7d579523462aaf7eac1890643f91fab3</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1002%2Fanie.201201358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201201358%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DBurke%26aufirst%3DJ.%2BP.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DBurns%26aufirst%3DM.%2BC.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520small%2520molecules%2520that%2520bind%2520to%2520K-Ras%2520and%2520inhibit%2520SOS-mediated%2520activation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D6140%26epage%3D3%26doi%3D10.1002%2Fanie.201201358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group">Pikul, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunham, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almstead, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natchus, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taiwo, Y. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hynd, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janusz, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mieling, G. E.</span><span> </span><span class="NLM_article-title">Heterocycle-based MMP inhibitors with P2′ substituents</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1009</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/S0960-894X(01)00137-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0960-894X%2801%2900137-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11327577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisl2jsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1009-13&author=S.+Pikulauthor=K.+M.+Dunhamauthor=N.+G.+Almsteadauthor=B.+Deauthor=M.+G.+Natchusauthor=Y.+O.+Taiwoauthor=L.+E.+Williamsauthor=B.+A.+Hyndauthor=L.+C.+Hsiehauthor=M.+J.+Januszauthor=F.+Guauthor=G.+E.+Mieling&title=Heterocycle-based+MMP+inhibitors+with+P2%E2%80%B2+substituents&doi=10.1016%2FS0960-894X%2801%2900137-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Heterocycle-based MMP Inhibitors with P2' Substituents</span></div><div class="casAuthors">Pikul, S.; Dunham, K. M.; Almstead, N. G.; De, B.; Natchus, M. G.; Taiwo, Y. O.; Williams, L. E.; Hynd, B. A.; Hsieh, L. C.; Janusz, M. J.; Gu, F.; Mieling, G. E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1009-1013</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Potent and selective inhibition of matrix metalloproteinases was demonstrated for a series of sulfonamide-based hydroxamic acids.  The design of the heterocyclic sulfonamides incorporates a six- or seven-member central ring with a P2' substituent that can be modified.  Binding interactions of this substituent at the S2' site are believed to contribute to high inhibitory potency against stromelysin, collagenase-3 and gelatinases A and B, and to provide selectivity against collagenase-1 and matrilysin.  An x-ray structure of a stromelysin-inhibitor complex was obtained to provide insights into the SAR and selectivity trends obsd. for the series.  A novel series of potent inhibitors of matrix metalloproteinases is described.  Highly selective, MMP-1 sparing mols. are identified through modification of the P2' substituent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg2ZIuuzKp3bVg90H21EOLACvtfcHk0likDFtA9cvpyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisl2jsb0%253D&md5=859a7b51b335c30355e5018d6ce16cd2</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2801%2900137-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252801%252900137-8%26sid%3Dliteratum%253Aachs%26aulast%3DPikul%26aufirst%3DS.%26aulast%3DDunham%26aufirst%3DK.%2BM.%26aulast%3DAlmstead%26aufirst%3DN.%2BG.%26aulast%3DDe%26aufirst%3DB.%26aulast%3DNatchus%26aufirst%3DM.%2BG.%26aulast%3DTaiwo%26aufirst%3DY.%2BO.%26aulast%3DWilliams%26aufirst%3DL.%2BE.%26aulast%3DHynd%26aufirst%3DB.%2BA.%26aulast%3DHsieh%26aufirst%3DL.%2BC.%26aulast%3DJanusz%26aufirst%3DM.%2BJ.%26aulast%3DGu%26aufirst%3DF.%26aulast%3DMieling%26aufirst%3DG.%2BE.%26atitle%3DHeterocycle-based%2520MMP%2520inhibitors%2520with%2520P2%25E2%2580%25B2%2520substituents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2001%26volume%3D11%26spage%3D1009%26epage%3D13%26doi%3D10.1016%2FS0960-894X%2801%2900137-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jehle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ochiana, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollastri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">E2585</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span><span class="refDoi"> DOI: 10.1073/pnas.1501567112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1501567112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=25918377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E2585-94&author=D.+Kozakovauthor=D.+R.+Hallauthor=S.+Jehleauthor=L.+Luoauthor=S.+O.+Ochianaauthor=E.+V.+Jonesauthor=M.+Pollastriauthor=K.+N.+Allenauthor=A.+Whittyauthor=S.+Vajda&title=Ligand+deconstruction%3A+Why+some+fragment+binding+positions+are+conserved+and+others+are+not&doi=10.1073%2Fpnas.1501567112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Jehle, Sefan; Luo, Lingqi; Ochiana, Stefan O.; Jones, Elizabeth V.; Pollastri, Michael; Allen, Karen N.; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">E2585-E2594</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) relies on the premise that the fragment binding mode will be conserved on subsequent expansion to a larger ligand.  However, no general condition has been established to explain when fragment binding modes will be conserved.  We show that a remarkably simple condition can be developed in terms of how fragments coincide with binding energy hot spots-regions of the protein where interactions with a ligand contribute substantial binding free energy-the locations of which can easily be detd. computationally.  Because a substantial fraction of the free energy of ligand binding comes from interacting with the residues in the energetically most important hot spot, a ligand moiety that sufficiently overlaps with this region will retain its location even when other parts of the ligand are removed.  This hypothesis is supported by eight case studies.  The condition helps identify whether a protein is suitable for FBDD, predicts the size of fragments required for screening, and dets. whether a fragment hit can be extended into a higher affinity ligand.  Our results show that ligand binding sites can usefully be thought of in terms of an anchor site, which is the top-ranked hot spot and dominates the free energy of binding, surrounded by a no. of weaker satellite sites that confer improved affinity and selectivity for a particular ligand and that it is the intrinsic binding potential of the protein surface that dets. whether it can serve as a robust binding site for a suitably optimized ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH6jbOut_iv7Vg90H21EOLACvtfcHk0likDFtA9cvpyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D&md5=d56b18fb4e4befde24037b07e52d8b62</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501567112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501567112%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DOchiana%26aufirst%3DS.%2BO.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DPollastri%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLigand%2520deconstruction%253A%2520Why%2520some%2520fragment%2520binding%2520positions%2520are%2520conserved%2520and%2520others%2520are%2520not%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE2585%26epage%3D94%26doi%3D10.1073%2Fpnas.1501567112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glasfeld, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haney, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seehra, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somers, W. S.</span><span> </span><span class="NLM_article-title">The bacterial cell-division protein ZipA and its interaction with an FtsZ fragment revealed by X-ray crystallography</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3179</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1093/emboj/19.13.3179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Femboj%2F19.13.3179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10880432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt1ajtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=3179-91&author=L.+Mosyakauthor=Y.+Zhangauthor=E.+Glasfeldauthor=S.+Haneyauthor=M.+Stahlauthor=J.+Seehraauthor=W.+S.+Somers&title=The+bacterial+cell-division+protein+ZipA+and+its+interaction+with+an+FtsZ+fragment+revealed+by+X-ray+crystallography&doi=10.1093%2Femboj%2F19.13.3179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">The bacterial cell-division protein ZipA and its interaction with an FtsZ fragment revealed by X-ray crystallography</span></div><div class="casAuthors">Mosyak, Lidia; Zhang, Yan; Glasfeld, Elizabeth; Haney, Steve; Stahl, Mark; Seehra, Jasbir; Somers, William S.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3179-3191</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In Escherichia coli, FtsZ, a homolog of eukaryotic tubulins, and ZipA, a membrane-anchored protein that binds to FtsZ, are two essential components of the septal ring structure that mediates cell division.  Recent data indicate that ZipA is involved in the assembly of the ring by linking FtsZ to the cytoplasmic membrane and that the ZipA-FtsZ interaction is mediated by their C-terminal domains.  We present the x-ray crystal structures of the C-terminal FtsZ-binding domain of ZipA and a complex between this domain and a C-terminal fragment of FtsZ.  The ZipA domain is a six-stranded β-sheet packed against three α-helixes and contains the split β-α-β motif found in many RNA-binding proteins.  The uncovered side of the sheet incorporates a shallow hydrophobic cavity exposed to solvent.  In the complex, the 17-residue FtsZ fragment occupies this entire cavity of ZipA and binds as an extended β-strand followed by α-helix.  An alanine-scanning mutagenesis anal. of the FtsZ fragment was also performed, which shows that only a small cluster of the buried FtsZ side chains is crit. in binding to ZipA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOPmByyCGAyLVg90H21EOLACvtfcHk0likDFtA9cvpyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlt1ajtb8%253D&md5=98ab31e86cb5de2f7955c011e342c25f</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.13.3179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.13.3179%26sid%3Dliteratum%253Aachs%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGlasfeld%26aufirst%3DE.%26aulast%3DHaney%26aufirst%3DS.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DSeehra%26aufirst%3DJ.%26aulast%3DSomers%26aufirst%3DW.%2BS.%26atitle%3DThe%2520bacterial%2520cell-division%2520protein%2520ZipA%2520and%2520its%2520interaction%2520with%2520an%2520FtsZ%2520fragment%2520revealed%2520by%2520X-ray%2520crystallography%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D3179%26epage%3D91%26doi%3D10.1093%2Femboj%2F19.13.3179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group">Jennings, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foreman, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rush, T. S.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsao, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sukhdeo, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moghazeh, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruzin, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuckman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, A. G.</span><span> </span><span class="NLM_article-title">Design and synthesis of indolo[2,3-a]quinolizin-7-one inhibitors of the ZipA-FtsZ interaction</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1427</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2004.01.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2004.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15006376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvV2it78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=1427-31&author=L.+D.+Jenningsauthor=K.+W.+Foremanauthor=T.+S.+Rushauthor=D.+H.+Tsaoauthor=L.+Mosyakauthor=Y.+Liauthor=M.+N.+Sukhdeoauthor=W.+Dingauthor=E.+G.+Dushinauthor=C.+H.+Kennyauthor=S.+L.+Moghazehauthor=P.+J.+Petersenauthor=A.+V.+Ruzinauthor=M.+Tuckmanauthor=A.+G.+Sutherland&title=Design+and+synthesis+of+indolo%5B2%2C3-a%5Dquinolizin-7-one+inhibitors+of+the+ZipA-FtsZ+interaction&doi=10.1016%2Fj.bmcl.2004.01.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of indolo[2,3-a]quinolizin-7-one inhibitors of the ZipA-FtsZ interaction</span></div><div class="casAuthors">Jennings, Lee D.; Foreman, Ken W.; Rush, Thomas S.; Tsao, Desiree H. H.; Mosyak, Lidia; Li, Yuanhong; Sukhdeo, Mohani N.; Ding, Weidong; Dushin, Elizabeth G.; Kenny, Cynthia Hess; Moghazeh, Soraya L.; Petersen, Peter J.; Ruzin, Alexey V.; Tuckman, Margareta; Sutherland, Alan G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1427-1431</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The binding of FtsZ to ZipA is a potential target for antibacterial therapy.  Based on a small mol. inhibitor of the ZipA-FtsZ interaction, a parallel synthesis of small mols. was initiated which targeted a key region of ZipA involved in FtsZ binding.  The X-ray crystal structure of one of these mols. complexed with ZipA was solved.  The structure revealed an unexpected binding mode, facilitated by desolvation of a loosely bound surface water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFg6MqTNEkGrVg90H21EOLACvtfcHk0ljC2RlgKHy8wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhvV2it78%253D&md5=9899fa126c5b235f3ed0edf63f31c2be</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2004.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2004.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DJennings%26aufirst%3DL.%2BD.%26aulast%3DForeman%26aufirst%3DK.%2BW.%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DTsao%26aufirst%3DD.%2BH.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSukhdeo%26aufirst%3DM.%2BN.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DDushin%26aufirst%3DE.%2BG.%26aulast%3DKenny%26aufirst%3DC.%2BH.%26aulast%3DMoghazeh%26aufirst%3DS.%2BL.%26aulast%3DPetersen%26aufirst%3DP.%2BJ.%26aulast%3DRuzin%26aufirst%3DA.%2BV.%26aulast%3DTuckman%26aufirst%3DM.%26aulast%3DSutherland%26aufirst%3DA.%2BG.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520indolo%255B2%252C3-a%255Dquinolizin-7-one%2520inhibitors%2520of%2520the%2520ZipA-FtsZ%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2004%26volume%3D14%26spage%3D1427%26epage%3D31%26doi%3D10.1016%2Fj.bmcl.2004.01.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group">Kenny, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kriz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellestad, G.</span><span> </span><span class="NLM_article-title">Development of a fluorescence polarization assay to screen for inhibitors of the FtsZ/ZipA interaction</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">323</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">33</span><span class="refDoi"> DOI: 10.1016/j.ab.2003.08.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.ab.2003.08.033" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2003&pages=224-33&author=C.+H.+Kennyauthor=W.+D.+Dingauthor=K.+Kelleherauthor=S.+Benardauthor=E.+G.+Dushinauthor=A.+G.+Sutherlandauthor=L.+Mosyakauthor=R.+Krizauthor=G.+Ellestad&title=Development+of+a+fluorescence+polarization+assay+to+screen+for+inhibitors+of+the+FtsZ%2FZipA+interaction&doi=10.1016%2Fj.ab.2003.08.033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2003.08.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2003.08.033%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DC.%2BH.%26aulast%3DDing%26aufirst%3DW.%2BD.%26aulast%3DKelleher%26aufirst%3DK.%26aulast%3DBenard%26aufirst%3DS.%26aulast%3DDushin%26aufirst%3DE.%2BG.%26aulast%3DSutherland%26aufirst%3DA.%2BG.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DKriz%26aufirst%3DR.%26aulast%3DEllestad%26aufirst%3DG.%26atitle%3DDevelopment%2520of%2520a%2520fluorescence%2520polarization%2520assay%2520to%2520screen%2520for%2520inhibitors%2520of%2520the%2520FtsZ%252FZipA%2520interaction%26jtitle%3DAnal.%2520Biochem.%26date%3D2003%26volume%3D323%26spage%3D224%26epage%3D33%26doi%3D10.1016%2Fj.ab.2003.08.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group">Tsao, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jennings, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rush, T. S.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haney, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malakian, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilakantan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span> </span><span class="NLM_article-title">Discovery of novel inhibitors of the ZipA/FtsZ complex by NMR fragment screening coupled with structure-based design</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">7953</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1016/j.bmc.2006.07.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmc.2006.07.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16919463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFSrtLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=7953-61&author=D.+H.+Tsaoauthor=A.+G.+Sutherlandauthor=L.+D.+Jenningsauthor=Y.+Liauthor=T.+S.+Rushauthor=J.+C.+Alvarezauthor=W.+Dingauthor=E.+G.+Dushinauthor=R.+G.+Dushinauthor=S.+A.+Haneyauthor=C.+H.+Kennyauthor=A.+K.+Malakianauthor=R.+Nilakantanauthor=L.+Mosyak&title=Discovery+of+novel+inhibitors+of+the+ZipA%2FFtsZ+complex+by+NMR+fragment+screening+coupled+with+structure-based+design&doi=10.1016%2Fj.bmc.2006.07.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel inhibitors of the ZipA/FtsZ complex by NMR fragment screening coupled with structure-based design</span></div><div class="casAuthors">Tsao, Desiree H. H.; Sutherland, Alan G.; Jennings, Lee D.; Li, Yuanhong; Rush, Thomas S.; Alvarez, Juan C.; Ding, Weidong; Dushin, Elizabeth G.; Dushin, Russell G.; Haney, Steve A.; Kenny, Cynthia H.; Malakian, A. Karl; Nilakantan, Ramaswamy; Mosyak, Lidia</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7953-7961</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">ZipA is a membrane anchored protein in Escherichia coli that interacts with FtsZ, a homolog of eukaryotic tubulins, forming a septal ring structure that mediates bacterial cell division.  Thus, the ZipA/FtsZ protein-protein interaction is a potential target for an antibacterial agent.  The authors report here an NMR-based fragment screening approach which identified several hits that bind to the C-terminal region of ZipA.  The screen was performed by 1H-15N HSQC expts. on a library of 825 fragments that are small, lead-like, and highly sol.  Seven hits were identified, and the binding mode of the best one was revealed in the x-ray crystal structure.  Similar to the ZipA/FtsZ contacts, the driving force in the binding of the small mol. ligands to ZipA is achieved through hydrophobic interactions.  Analogs of this hit were also evaluated by NMR and x-ray crystal structures of these analogs with ZipA were obtained, providing structural information to help guide the medicinal chem. efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX3w_cIyvHYrVg90H21EOLACvtfcHk0ljC2RlgKHy8wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFSrtLbN&md5=ffe8d4f05d826414a13d55ddb2857579</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.07.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.07.050%26sid%3Dliteratum%253Aachs%26aulast%3DTsao%26aufirst%3DD.%2BH.%26aulast%3DSutherland%26aufirst%3DA.%2BG.%26aulast%3DJennings%26aufirst%3DL.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DAlvarez%26aufirst%3DJ.%2BC.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DDushin%26aufirst%3DE.%2BG.%26aulast%3DDushin%26aufirst%3DR.%2BG.%26aulast%3DHaney%26aufirst%3DS.%2BA.%26aulast%3DKenny%26aufirst%3DC.%2BH.%26aulast%3DMalakian%26aufirst%3DA.%2BK.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DMosyak%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520novel%2520inhibitors%2520of%2520the%2520ZipA%252FFtsZ%2520complex%2520by%2520NMR%2520fragment%2520screening%2520coupled%2520with%2520structure-based%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D7953%26epage%3D61%26doi%3D10.1016%2Fj.bmc.2006.07.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group">Jennings, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foreman, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rush, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsao, D. H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kincaid, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sukhdeo, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabus, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moghazeh, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labthavikul, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuckman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haney, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruzin, A. V.</span><span> </span><span class="NLM_article-title">Combinatorial synthesis of substituted 3-(2-indolyl)piperidines and 2-phenyl indoles as inhibitors of ZipA-FtsZ interaction</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5115</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1016/j.bmc.2004.07.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmc.2004.07.031" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=5115-31&author=L.+D.+Jenningsauthor=K.+W.+Foremanauthor=T.+S.+Rushauthor=D.+H.+H.+Tsaoauthor=L.+Mosyakauthor=S.+L.+Kincaidauthor=M.+N.+Sukhdeoauthor=A.+G.+Sutherlandauthor=W.+D.+Dingauthor=C.+H.+Kennyauthor=C.+L.+Sabusauthor=H.+L.+Liuauthor=E.+G.+Dushinauthor=S.+L.+Moghazehauthor=P.+Labthavikulauthor=P.+J.+Petersenauthor=M.+Tuckmanauthor=S.+A.+Haneyauthor=A.+V.+Ruzin&title=Combinatorial+synthesis+of+substituted+3-%282-indolyl%29piperidines+and+2-phenyl+indoles+as+inhibitors+of+ZipA-FtsZ+interaction&doi=10.1016%2Fj.bmc.2004.07.031"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2004.07.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2004.07.031%26sid%3Dliteratum%253Aachs%26aulast%3DJennings%26aufirst%3DL.%2BD.%26aulast%3DForeman%26aufirst%3DK.%2BW.%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DTsao%26aufirst%3DD.%2BH.%2BH.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DKincaid%26aufirst%3DS.%2BL.%26aulast%3DSukhdeo%26aufirst%3DM.%2BN.%26aulast%3DSutherland%26aufirst%3DA.%2BG.%26aulast%3DDing%26aufirst%3DW.%2BD.%26aulast%3DKenny%26aufirst%3DC.%2BH.%26aulast%3DSabus%26aufirst%3DC.%2BL.%26aulast%3DLiu%26aufirst%3DH.%2BL.%26aulast%3DDushin%26aufirst%3DE.%2BG.%26aulast%3DMoghazeh%26aufirst%3DS.%2BL.%26aulast%3DLabthavikul%26aufirst%3DP.%26aulast%3DPetersen%26aufirst%3DP.%2BJ.%26aulast%3DTuckman%26aufirst%3DM.%26aulast%3DHaney%26aufirst%3DS.%2BA.%26aulast%3DRuzin%26aufirst%3DA.%2BV.%26atitle%3DCombinatorial%2520synthesis%2520of%2520substituted%25203-%25282-indolyl%2529piperidines%2520and%25202-phenyl%2520indoles%2520as%2520inhibitors%2520of%2520ZipA-FtsZ%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D5115%26epage%3D31%26doi%3D10.1016%2Fj.bmc.2004.07.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group">Rush, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">A shape-based 3-D scaffold hopping method and its application to a bacterial protein-protein interaction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1489</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.1021/jm040163o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040163o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Ols78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1489-95&author=T.+S.+Rushauthor=J.+A.+Grantauthor=L.+Mosyakauthor=A.+Nicholls&title=A+shape-based+3-D+scaffold+hopping+method+and+its+application+to+a+bacterial+protein-protein+interaction&doi=10.1021%2Fjm040163o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">A Shape-Based 3-D Scaffold Hopping Method and Its Application to a Bacterial Protein-Protein Interaction</span></div><div class="casAuthors">Rush, Thomas S., III; Grant, J. Andrew; Mosyak, Lidia; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1489-1495</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, the authors describe the first prospective application of the shape-comparison program ROCS (Rapid Overlay of Chem. Structures) to find new scaffolds for small mol. inhibitors of the ZipA-FtsZ protein-protein interaction, a proposed antibacterial target.  The shape comparisons are made relative to the crystallog. detd., bioactive conformation of a high-throughput screening (HTS) hit.  The use of ROCS led to the identification of a set of novel, weakly binding inhibitors with scaffolds presenting synthetic opportunities to further optimize biol. affinity and lacking development issues assocd. with the HTS lead.  These ROCS-identified scaffolds would have been missed using other structural similarity approaches such as ISIS 2D fingerprints.  X-ray crystallog. anal. of one of the new inhibitors bound to ZipA reveals that the shape comparison approach very accurately predicted the binding mode.  These exptl. results validate this use of ROCS for chemotype switching or "lead hopping" and suggest that it is of general interest for lead identification in drug discovery endeavors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4EX9jFtct37Vg90H21EOLACvtfcHk0lg852JlQp3Grw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Ols78%253D&md5=05b6e54a657a3b8c768e63852d871ef6</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fjm040163o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040163o%26sid%3Dliteratum%253Aachs%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DA%2520shape-based%25203-D%2520scaffold%2520hopping%2520method%2520and%2520its%2520application%2520to%2520a%2520bacterial%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1489%26epage%3D95%26doi%3D10.1021%2Fjm040163o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group">Rai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vyjayanti, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorjsuren, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span> </span><span class="NLM_article-title">Synthesis, biological evaluation, and structure-activity relationships of a novel class of apurinic/apyrimidinic endonuclease 1 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">3101</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.1021/jm201537d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201537d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVWmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3101-12&author=G.+Raiauthor=V.+N.+Vyjayantiauthor=D.+Dorjsurenauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+M.+Wilsonauthor=D.+J.+Maloney&title=Synthesis%2C+biological+evaluation%2C+and+structure-activity+relationships+of+a+novel+class+of+apurinic%2Fapyrimidinic+endonuclease+1+inhibitors&doi=10.1021%2Fjm201537d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological evaluation, and structure-activity relationships of a novel class of apurinic/apyrimidinic endonuclease 1 inhibitors</span></div><div class="casAuthors">Rai, Ganesha; Vyjayanti, Vaddadi N.; Dorjsuren, Dorjbal; Simeonov, Anton; Jadhav, Ajit; Wilson, David M.; Maloney, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3101-3112</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">APE1 is an essential protein that operates in the base excision repair (BER) pathway and is responsible for ≥95% of the total apurinic/apyrimidinic (AP) endonuclease activity in human cells.  BER is a major pathway that copes with DNA damage induced by several anticancer agents, including ionizing radiation and temozolomide.  Overexpression of APE1 and enhanced AP endonuclease activity have been linked to increased resistance of tumor cells to treatment with monofunctional alkylators, implicating inhibition of APE1 as a valid strategy for cancer therapy.  We report herein the results of a focused medicinal chem. effort around a novel APE1 inhibitor, N-(3-(benzo[d]thiazol-2-yl)-6-isopropyl-4,5,6,7-tetrahydrothieno[2,3-c]pyridin-2-yl)acetamide (I).  Compd. I and related analogs exhibit single-digit micromolar activity against the purified APE1 enzyme and comparable activity in HeLa whole cell ext. assays and potentiate the cytotoxicity of the alkylating agents methylmethane sulfonate and temozolomide.  Moreover, this class of compds. possesses a generally favorable in vitro ADME profile, along with good exposure levels in plasma and brain following i.p. dosing (30 mg/kg body wt.) in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq25R0Ebe53srVg90H21EOLACvtfcHk0lg852JlQp3Grw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVWmt7o%253D&md5=1dac481ed0ec1cc67d59ad3091520ccc</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fjm201537d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201537d%26sid%3Dliteratum%253Aachs%26aulast%3DRai%26aufirst%3DG.%26aulast%3DVyjayanti%26aufirst%3DV.%2BN.%26aulast%3DDorjsuren%26aufirst%3DD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DSynthesis%252C%2520biological%2520evaluation%252C%2520and%2520structure-activity%2520relationships%2520of%2520a%2520novel%2520class%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3101%26epage%3D12%26doi%3D10.1021%2Fjm201537d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group">Abbotts, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madhusudan, S.</span><span> </span><span class="NLM_article-title">Human AP endonuclease 1 (APE1): From mechanistic insights to druggable target in cancer</span> <span class="citation_source-journal">Cancer Treat. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">425</span><span class="NLM_x">–</span> <span class="NLM_lpage">35</span><span class="refDoi"> DOI: 10.1016/j.ctrv.2009.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.ctrv.2009.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=20056333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpt1Ont7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2010&pages=425-35&author=R.+Abbottsauthor=S.+Madhusudan&title=Human+AP+endonuclease+1+%28APE1%29%3A+From+mechanistic+insights+to+druggable+target+in+cancer&doi=10.1016%2Fj.ctrv.2009.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Human AP endonuclease 1 (APE1): from mechanistic insights to druggable target in cancer</span></div><div class="casAuthors">Abbotts, Rachel; Madhusudan, Srinivasan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">425-435</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  DNA base excision repair (BER) is critically involved in the processing of DNA base damage induced by alkylating agents.  Pharmacol. inhibition of BER (using PARP inhibitors), either alone or in combination with chemotherapy has recently shown promise in clin. trials.  Human apurinic/apyrimidinic endonuclease 1(APE1) is an essential BER protein that is involved in the processing of potentially cytotoxic abasic sites that are obligatory intermediates in BER.  Here we provide a summary of the basic mechanistic role of APE1 in DNA repair and redox regulation and highlight preclin. and clin. data that confirm APE1 as a valid anticancer drug target.  Development of small mol. inhibitors of APE1 is an area of intense research and current evidence using APE1 inhibitors has demonstrated potentiation of cytotoxicity of alkylating agents in preclin. models implying translational applications in cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWbrntssmy5bVg90H21EOLACvtfcHk0ljOG7ze-M6VRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpt1Ont7k%253D&md5=14721f4b897a254b307007042a162170</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2009.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2009.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DAbbotts%26aufirst%3DR.%26aulast%3DMadhusudan%26aufirst%3DS.%26atitle%3DHuman%2520AP%2520endonuclease%25201%2520%2528APE1%2529%253A%2520From%2520mechanistic%2520insights%2520to%2520druggable%2520target%2520in%2520cancer%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2010%26volume%3D36%26spage%3D425%26epage%3D35%26doi%3D10.1016%2Fj.ctrv.2009.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group">Wilson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of DNA repair nuclease activities of APE1</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">3621</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1007/s00018-010-0488-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs00018-010-0488-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=20809131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlSjsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2010&pages=3621-31&author=D.+M.+Wilsonauthor=A.+Simeonov&title=Small+molecule+inhibitors+of+DNA+repair+nuclease+activities+of+APE1&doi=10.1007%2Fs00018-010-0488-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of DNA repair nuclease activities of APE1</span></div><div class="casAuthors">Wilson, David M., III; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3621-3631</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  APE1 is a multifunctional protein that possesses several nuclease activities, including the ability to incise at apurinic/apyrimidinic (AP) sites in DNA or RNA, to excise 3'-blocking termini from DNA ends, and to cleave at certain oxidized base lesions in DNA.  Pre-clin. and clin. data indicate a role for APE1 in the pathogenesis of cancer and in resistance to DNA-interactive drugs, particularly monofunctional alkylators and antimetabolites.  In an effort to improve the efficacy of therapeutic compds., such as temozolomide, groups have begun to develop high-throughput screening assays and to identify small mol. inhibitors against APE1 repair nuclease activities.  It is envisioned that such inhibitors will be used in combinatorial treatment paradigms to enhance the efficacy of DNA-interactive drugs that introduce relevant cytotoxic DNA lesions.  In this review, we summarize the current state of the efforts to design potent and selective inhibitors against APE1 AP site incision activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN4mUcnbEDarVg90H21EOLACvtfcHk0ljOG7ze-M6VRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlSjsLjN&md5=7406c275670396d1652dfda2cf68e2af</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1007%2Fs00018-010-0488-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-010-0488-2%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520DNA%2520repair%2520nuclease%2520activities%2520of%2520APE1%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2010%26volume%3D67%26spage%3D3621%26epage%3D31%26doi%3D10.1007%2Fs00018-010-0488-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group">Patel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahekar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadav, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joharapurkar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanesha, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaikh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sairam, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapadnis, P.</span><span> </span><span class="NLM_article-title">Discovery of potent, selective and orally bioavailable triaryl-sulfonamide based PTP1B inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1111</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.11.122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2011.11.122" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1111-7&author=D.+Patelauthor=M.+Jainauthor=S.+R.+Shahauthor=R.+Bahekarauthor=P.+Jadavauthor=A.+Joharapurkarauthor=N.+Dhaneshaauthor=M.+Shaikhauthor=K.+V.+Sairamauthor=P.+Kapadnis&title=Discovery+of+potent%2C+selective+and+orally+bioavailable+triaryl-sulfonamide+based+PTP1B+inhibitors&doi=10.1016%2Fj.bmcl.2011.11.122"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.11.122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.11.122%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DShah%26aufirst%3DS.%2BR.%26aulast%3DBahekar%26aufirst%3DR.%26aulast%3DJadav%26aufirst%3DP.%26aulast%3DJoharapurkar%26aufirst%3DA.%26aulast%3DDhanesha%26aufirst%3DN.%26aulast%3DShaikh%26aufirst%3DM.%26aulast%3DSairam%26aufirst%3DK.%2BV.%26aulast%3DKapadnis%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520potent%252C%2520selective%2520and%2520orally%2520bioavailable%2520triaryl-sulfonamide%2520based%2520PTP1B%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D1111%26epage%3D7%26doi%3D10.1016%2Fj.bmcl.2011.11.122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group">Scapin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desponts, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waddleton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skorey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cromlish, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gauthier, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramachandran, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asante-Appiah, E.</span><span> </span><span class="NLM_article-title">The structural basis for the selectivity of benzotriazole inhibitors of PTP1B</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">11451</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1021/bi035098j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi035098j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=11451-9&author=G.+Scapinauthor=S.+B.+Patelauthor=J.+W.+Beckerauthor=Q.+P.+Wangauthor=C.+Despontsauthor=D.+Waddletonauthor=K.+Skoreyauthor=W.+Cromlishauthor=C.+Baylyauthor=M.+Therienauthor=J.+Y.+Gauthierauthor=C.+S.+Liauthor=C.+K.+Lauauthor=C.+Ramachandranauthor=B.+P.+Kennedyauthor=E.+Asante-Appiah&title=The+structural+basis+for+the+selectivity+of+benzotriazole+inhibitors+of+PTP1B&doi=10.1021%2Fbi035098j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1021%2Fbi035098j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi035098j%26sid%3Dliteratum%253Aachs%26aulast%3DScapin%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DS.%2BB.%26aulast%3DBecker%26aufirst%3DJ.%2BW.%26aulast%3DWang%26aufirst%3DQ.%2BP.%26aulast%3DDesponts%26aufirst%3DC.%26aulast%3DWaddleton%26aufirst%3DD.%26aulast%3DSkorey%26aufirst%3DK.%26aulast%3DCromlish%26aufirst%3DW.%26aulast%3DBayly%26aufirst%3DC.%26aulast%3DTherien%26aufirst%3DM.%26aulast%3DGauthier%26aufirst%3DJ.%2BY.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DLau%26aufirst%3DC.%2BK.%26aulast%3DRamachandran%26aufirst%3DC.%26aulast%3DKennedy%26aufirst%3DB.%2BP.%26aulast%3DAsante-Appiah%26aufirst%3DE.%26atitle%3DThe%2520structural%2520basis%2520for%2520the%2520selectivity%2520of%2520benzotriazole%2520inhibitors%2520of%2520PTP1B%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D11451%26epage%3D9%26doi%3D10.1021%2Fbi035098j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref150"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref150'); return false;" data-citation="" class="refNumLink">150</a></strong><div class="NLM_citation" id="cit150"><span><span class="NLM_contrib-group">Barr, A. J.</span><span> </span><span class="NLM_article-title">Protein tyrosine phosphatases as drug targets: Strategies and challenges of inhibitor development</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">1563</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span><span class="refDoi"> DOI: 10.4155/fmc.10.241</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.4155%2Ffmc.10.241" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21426149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yqsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=1563-76&author=A.+J.+Barr&title=Protein+tyrosine+phosphatases+as+drug+targets%3A+Strategies+and+challenges+of+inhibitor+development&doi=10.4155%2Ffmc.10.241"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Protein tyrosine phosphatases as drug targets: strategies and challenges of inhibitor development</span></div><div class="casAuthors">Barr, Alastair J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1563-1576</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Several classical' protein tyrosine phosphatases are attractive therapeutic targets, including PTP1B for obesity and Type II diabetes; SHP2 for cancer and Lyp for rheumatoid arthritis.  Progress has been made in identifying a broad range of chem. distinct inhibitors; however, developing selective and cell-permeable clin. useful compds. has proved challenging.  Here the ongoing challenges and recent significant advances in the field are reviewed.  Key novel compds. are highlighted and a perspective on the future of phosphatase inhibitor development is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUabcAXtUSRrVg90H21EOLACvtfcHk0ljOG7ze-M6VRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yqsbjE&md5=2e0b3ae741824174eb0b49f1f7d69c10</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.241&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.241%26sid%3Dliteratum%253Aachs%26aulast%3DBarr%26aufirst%3DA.%2BJ.%26atitle%3DProtein%2520tyrosine%2520phosphatases%2520as%2520drug%2520targets%253A%2520Strategies%2520and%2520challenges%2520of%2520inhibitor%2520development%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D1563%26epage%3D76%26doi%3D10.4155%2Ffmc.10.241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref151"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref151'); return false;" data-citation="" class="refNumLink">151</a></strong><div class="NLM_citation" id="cit151"><span><span class="NLM_contrib-group">Douty, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wayland, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ala, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bower, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pruitt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bostrom, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klabe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonneville, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynn, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burn, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, P. C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combs, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, E. W.</span><span> </span><span class="NLM_article-title">Isothiazolidinone inhibitors of PTP1B containing imidazoles and imidazolines</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">66</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2007.11.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=66-71&author=B.+Doutyauthor=B.+Waylandauthor=P.+J.+Alaauthor=M.+J.+Bowerauthor=J.+Pruittauthor=L.+Bostromauthor=M.+Weiauthor=R.+Klabeauthor=L.+Gonnevilleauthor=R.+Wynnauthor=T.+C.+Burnauthor=P.+C.+C.+Liuauthor=A.+P.+Combsauthor=E.+W.+Yue&title=Isothiazolidinone+inhibitors+of+PTP1B+containing+imidazoles+and+imidazolines&doi=10.1016%2Fj.bmcl.2007.11.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DDouty%26aufirst%3DB.%26aulast%3DWayland%26aufirst%3DB.%26aulast%3DAla%26aufirst%3DP.%2BJ.%26aulast%3DBower%26aufirst%3DM.%2BJ.%26aulast%3DPruitt%26aufirst%3DJ.%26aulast%3DBostrom%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DM.%26aulast%3DKlabe%26aufirst%3DR.%26aulast%3DGonneville%26aufirst%3DL.%26aulast%3DWynn%26aufirst%3DR.%26aulast%3DBurn%26aufirst%3DT.%2BC.%26aulast%3DLiu%26aufirst%3DP.%2BC.%2BC.%26aulast%3DCombs%26aufirst%3DA.%2BP.%26aulast%3DYue%26aufirst%3DE.%2BW.%26atitle%3DIsothiazolidinone%2520inhibitors%2520of%2520PTP1B%2520containing%2520imidazoles%2520and%2520imidazolines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D66%26epage%3D71%26doi%3D10.1016%2Fj.bmcl.2007.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref152"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref152'); return false;" data-citation="" class="refNumLink">152</a></strong><div class="NLM_citation" id="cit152"><span><span class="NLM_contrib-group">Zhao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbieri, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H. A.</span><span> </span><span class="NLM_article-title">Discovery and structural characterization of a small molecule 14–3-3 protein-protein interaction inhibitor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">16212</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1073/pnas.1100012108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1100012108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21908710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Kmtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=16212-6&author=J.+Zhaoauthor=Y.+H.+Duauthor=J.+R.+Hortonauthor=A.+K.+Upadhyayauthor=B.+Louauthor=Y.+Baiauthor=X.+Zhangauthor=L.+P.+Duauthor=M.+Y.+Liauthor=B.+H.+Wangauthor=L.+X.+Zhangauthor=J.+T.+Barbieriauthor=F.+R.+Khuriauthor=X.+D.+Chengauthor=H.+A.+Fu&title=Discovery+and+structural+characterization+of+a+small+molecule+14%E2%80%933-3+protein-protein+interaction+inhibitor&doi=10.1073%2Fpnas.1100012108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and structural characterization of a small molecule 14-3-3 protein-protein interaction inhibitor</span></div><div class="casAuthors">Zhao, Jing; Du, Yuhong; Horton, John R.; Upadhyay, Anup K.; Lou, Bin; Bai, Yan; Zhang, Xing; Du, Lupei; Li, Minyong; Wang, Binghe; Zhang, Lixin; Barbieri, Joseph T.; Khuri, Fadlo R.; Cheng, Xiaodong; Fu, Haian</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">16212-16216, S16212/1-S16212/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 14-3-3 family of phosphoserine/threonine-recognition proteins engage multiple nodes in signaling networks that control diverse physiol. and pathophysiol. functions and have emerged as promising therapeutic target; for such diseases as cancer and neurodegenerative disorders.  Thus, small mol. modulators of 14-3-3 are much needed agents for chem. biol. investigations and therapeutic development.  To analyze 14-3-3 function and modulate its activity, we conducted a chem. screen and identified 4-[(2Z)-2-[4-formyl-6-methyl-5-oxo-3-(phosphonatooxymethyl)pyridin-2-ylidene]hydrazinyl]benzoate as a 14-3-3 inhibitor, which we termed FOBISIN (FOurteen-three-three Binding Small mol. INhibitor) 101.  FOBISIN101 effectively blocked the binding of 14-3-3 with Raf-1 and proline-rich AKT substrate, 40 kDa and neutralized the ability of 14-3-3 to activate exoenzyme S ADP-ribosyltransferase.  To provide a mechanistic basis for 14-3-3 inhibition, the crystal structure of 14-3-3ζ in complex with FOBISIN101 was solved.  Unexpectedly, the double bond linking the pyridoxal-phosphate and benzoate moieties was reduced by X-rays to create a covalent linkage of the pyridoxal-phosphate moiety to lysine 120 in the binding groove of 14-3-3, leading to persistent 14-3-3 inactivation.  We suggest that FOBISIN101-like mols. could be developed as an entirely unique class of 14-3-3 inhibitors, which may serve as radiation-triggered therapeutic agents for the treatment of 14-3-3-mediated diseases, such as cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR0iuk-c1eP7Vg90H21EOLACvtfcHk0lh2Hes0wDWhXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Kmtb7K&md5=ba66810da0f7b1f6e9e16701f58ef8c2</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1100012108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1100012108%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DY.%2BH.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DLou%26aufirst%3DB.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DL.%2BP.%26aulast%3DLi%26aufirst%3DM.%2BY.%26aulast%3DWang%26aufirst%3DB.%2BH.%26aulast%3DZhang%26aufirst%3DL.%2BX.%26aulast%3DBarbieri%26aufirst%3DJ.%2BT.%26aulast%3DKhuri%26aufirst%3DF.%2BR.%26aulast%3DCheng%26aufirst%3DX.%2BD.%26aulast%3DFu%26aufirst%3DH.%2BA.%26atitle%3DDiscovery%2520and%2520structural%2520characterization%2520of%2520a%2520small%2520molecule%252014%25E2%2580%25933-3%2520protein-protein%2520interaction%2520inhibitor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D16212%26epage%3D6%26doi%3D10.1073%2Fpnas.1100012108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref153"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref153'); return false;" data-citation="" class="refNumLink">153</a></strong><div class="NLM_citation" id="cit153"><span><span class="NLM_contrib-group">Gardino, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smerdon, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yaffe, M. B.</span><span> </span><span class="NLM_article-title">Structural determinants of 14–3-3 binding specificities and regulation of subcellular localization of 14–3-3-ligand complexes: A comparison of the x-ray crystal structures of all human 14–3-3 isoforms</span> <span class="citation_source-journal">Semin. Cancer Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">173</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/j.semcancer.2006.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.semcancer.2006.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16678437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XltlKmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=173-82&author=A.+K.+Gardinoauthor=S.+J.+Smerdonauthor=M.+B.+Yaffe&title=Structural+determinants+of+14%E2%80%933-3+binding+specificities+and+regulation+of+subcellular+localization+of+14%E2%80%933-3-ligand+complexes%3A+A+comparison+of+the+x-ray+crystal+structures+of+all+human+14%E2%80%933-3+isoforms&doi=10.1016%2Fj.semcancer.2006.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of 14-3-3 binding specificities and regulation of subcellular localization of 14-3-3-ligand complexes: A comparison of the x-ray crystal structures of all human 14-3-3 isoforms</span></div><div class="casAuthors">Gardino, Alexandra K.; Smerdon, Stephen J.; Yaffe, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">173-182</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The 14-3-3 proteins are a ubiquitous class of regulatory proteins found in all eukaryotic cells and were the 1st class of mols. to be recognized as discrete phosphoserine/threonine-binding modules.  The 14-3-3 proteins bind a large no. of different substrates to regulate a wide array of cellular signaling events including cell cycle progression and DNA damage responses, programmed cell death, cytoskeletal dynamics, transcriptional control of gene expression, as well as processes directly related to cancer progression.  Here, the structural basis of phosphorylation-dependent binding of 14-3-3 proteins to peptide and protein ligands is discussed along with mechanisms that govern how 14-3-3 proteins regulate the function of their bound ligands.  The x-ray crystal structures of all human 14-3-3 proteins bound to peptides have now been solved.  Here, the authors use structural comparisons between isoforms as a framework for discussion of ligand binding by 14-3-3 proteins as well as the mechanisms through which post-translational modification of the different isoforms alters their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrFYLdMPrZBrVg90H21EOLACvtfcHk0lh2Hes0wDWhXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltlKmsrg%253D&md5=ba9a8d49d70cdba9a9d8275ac0471030</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2006.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2006.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DGardino%26aufirst%3DA.%2BK.%26aulast%3DSmerdon%26aufirst%3DS.%2BJ.%26aulast%3DYaffe%26aufirst%3DM.%2BB.%26atitle%3DStructural%2520determinants%2520of%252014%25E2%2580%25933-3%2520binding%2520specificities%2520and%2520regulation%2520of%2520subcellular%2520localization%2520of%252014%25E2%2580%25933-3-ligand%2520complexes%253A%2520A%2520comparison%2520of%2520the%2520x-ray%2520crystal%2520structures%2520of%2520all%2520human%252014%25E2%2580%25933-3%2520isoforms%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D173%26epage%3D82%26doi%3D10.1016%2Fj.semcancer.2006.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref154"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref154'); return false;" data-citation="" class="refNumLink">154</a></strong><div class="NLM_citation" id="cit154"><span><span class="NLM_contrib-group">De Vries-van Leeuwen, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">da Costa Pereira, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flach, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piersma, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haase, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bier, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yalcin, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalides, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feenstra, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jimenez, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Greef, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunsveld, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottmann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Boer, A. H.</span><span> </span><span class="NLM_article-title">Interaction of 14–3-3 proteins with the estrogen receptor alpha f domain provides a drug target interface</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">8894</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1073/pnas.1220809110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1220809110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23676274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFait7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8894-9&author=I.+J.+De+Vries-van+Leeuwenauthor=D.+da+Costa+Pereiraauthor=K.+D.+Flachauthor=S.+R.+Piersmaauthor=C.+Haaseauthor=D.+Bierauthor=Z.+Yalcinauthor=R.+Michalidesauthor=K.+A.+Feenstraauthor=C.+R.+Jimenezauthor=T.+F.+de+Greefauthor=L.+Brunsveldauthor=C.+Ottmannauthor=W.+Zwartauthor=A.+H.+de+Boer&title=Interaction+of+14%E2%80%933-3+proteins+with+the+estrogen+receptor+alpha+f+domain+provides+a+drug+target+interface&doi=10.1073%2Fpnas.1220809110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of 14-3-3 proteins with the Estrogen Receptor Alpha F domain provides a drug target interface</span></div><div class="casAuthors">De Vries-van Leeuwen, Ingrid J.; Pereira, Daniel da Costa; Flach, Koen D.; Piersma, Sander R.; Haase, Christian; Bier, David; Yalcin, Zeliha; Michalides, Rob; Feenstra, K. Anton; Jimenez, Connie R.; de Greef, Tom F. A.; Brunsveld, Luc; Ottmann, Christian; Zwart, Wilbert; de Boer, Albertus H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8894-8899, S8894/1-S8894/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Estrogen receptor alpha (ERa) is involved in numerous physiol. and pathol. processes, including breast cancer.  Breast cancer therapy is therefore currently directed at inhibiting the transcriptional potency of ERa, either by blocking estrogen prodn. through aromatase inhibitors or antiestrogens that compete for hormone binding.  Due to resistance, new treatment modalities are needed and as ERα dimerization is essential for its activity, interference with receptor dimerization offers a new opportunity to exploit in drug design.  Here we describe a unique mechanism of how ERα dimerization is neg. controlled by interaction with 14-3-3 proteins at the extreme C terminus of the receptor.  Moreover, the small-mol. fusicoccin (FC) stabilizes this ERα/14-3-3 interaction.  Cocrystn. of the trimeric ERα/14-3-3/FC complex provides the structural basis for this stabilization and shows the importance of phosphorylation of the penultimate Threonine (ERα-T594) for high-affinity interaction.  We confirm that T594 is a distinct ERα phosphorylation site in the breast cancer cell line MCF-7 using a phospho-T594-specific antibody and by mass spectrometry.  In line with its ERα/14-3-3 interaction stabilizing effect, fusicoccin reduces the estradiol-stimulated ERα dimerization, inhibits ERα/chromatin interactions and downstream gene expression, resulting in decreased cell proliferation.  Herewith, a unique functional phosphosite and an alternative regulation mechanism of ERα are provided, together with a small mol. that selectively targets this ERα/14-3-3 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0rga7Qdqyn7Vg90H21EOLACvtfcHk0liF-BEl3OggwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFait7%252FP&md5=d17c1a1122f4b525b35651d3d9f629f0</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1220809110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1220809110%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVries-van%2BLeeuwen%26aufirst%3DI.%2BJ.%26aulast%3Dda%2BCosta%2BPereira%26aufirst%3DD.%26aulast%3DFlach%26aufirst%3DK.%2BD.%26aulast%3DPiersma%26aufirst%3DS.%2BR.%26aulast%3DHaase%26aufirst%3DC.%26aulast%3DBier%26aufirst%3DD.%26aulast%3DYalcin%26aufirst%3DZ.%26aulast%3DMichalides%26aufirst%3DR.%26aulast%3DFeenstra%26aufirst%3DK.%2BA.%26aulast%3DJimenez%26aufirst%3DC.%2BR.%26aulast%3Dde%2BGreef%26aufirst%3DT.%2BF.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DZwart%26aufirst%3DW.%26aulast%3Dde%2BBoer%26aufirst%3DA.%2BH.%26atitle%3DInteraction%2520of%252014%25E2%2580%25933-3%2520proteins%2520with%2520the%2520estrogen%2520receptor%2520alpha%2520f%2520domain%2520provides%2520a%2520drug%2520target%2520interface%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D8894%26epage%3D9%26doi%3D10.1073%2Fpnas.1220809110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref155"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref155'); return false;" data-citation="" class="refNumLink">155</a></strong><div class="NLM_citation" id="cit155"><span><span class="NLM_contrib-group">Thiel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roglin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meissner, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hennig, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohlbacher, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottmann, C.</span><span> </span><span class="NLM_article-title">Virtual screening and experimental validation reveal novel small-molecule inhibitors of 14–3-3 protein-protein interactions</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">8468</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1039/c3cc44612c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1039%2Fc3cc44612c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23939230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlChur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=8468-70&author=P.+Thielauthor=L.+Roglinauthor=N.+Meissnerauthor=S.+Hennigauthor=O.+Kohlbacherauthor=C.+Ottmann&title=Virtual+screening+and+experimental+validation+reveal+novel+small-molecule+inhibitors+of+14%E2%80%933-3+protein-protein+interactions&doi=10.1039%2Fc3cc44612c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit155R"><div class="casContent"><span class="casTitleNuber">155</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening and experimental validation reveal novel small-molecule inhibitors of 14-3-3 protein-protein interactions</span></div><div class="casAuthors">Thiel, Philipp; Roeglin, Lars; Meissner, Nicole; Hennig, Sven; Kohlbacher, Oliver; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">76</span>),
    <span class="NLM_cas:pages">8468-8470</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors report first noncovalent and exclusively extracellular inhibitors of 14-3-3 protein-protein interactions identified by virtual screening.  Optimization by crystal structure anal. and in vitro binding assays yielded compds. capable of disrupting the interaction of 14-3-3σ with MMP-1 in a cellular assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9WHSGFCZit7Vg90H21EOLACvtfcHk0liF-BEl3OggwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlChur%252FP&md5=2c6f7a9969bd1d08f288e8a3f4ecce80</span></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1039%2Fc3cc44612c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3cc44612c%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DP.%26aulast%3DRoglin%26aufirst%3DL.%26aulast%3DMeissner%26aufirst%3DN.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DKohlbacher%26aufirst%3DO.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DVirtual%2520screening%2520and%2520experimental%2520validation%2520reveal%2520novel%2520small-molecule%2520inhibitors%2520of%252014%25E2%2580%25933-3%2520protein-protein%2520interactions%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2013%26volume%3D49%26spage%3D8468%26epage%3D70%26doi%3D10.1039%2Fc3cc44612c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref156"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref156'); return false;" data-citation="" class="refNumLink">156</a></strong><div class="NLM_citation" id="cit156"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frett, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H. Y.</span><span> </span><span class="NLM_article-title">Targeting mutant kras for anticancer therapeutics: A review of novel small molecule modulators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">5219</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1021/jm3017706</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3017706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5219-30&author=Y.+Wangauthor=C.+E.+Kaiserauthor=B.+Frettauthor=H.+Y.+Li&title=Targeting+mutant+kras+for+anticancer+therapeutics%3A+A+review+of+novel+small+molecule+modulators&doi=10.1021%2Fjm3017706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Mutant KRAS for Anticancer Therapeutics: A Review of Novel Small Molecule Modulators</span></div><div class="casAuthors">Wang, Yuanxiang; Kaiser, Christine E.; Frett, Brendan; Li, Hong-yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5219-5230</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The RAS proteins play a role in cell differentiation, proliferation, and survival.  Aberrant RAS signaling has been found to play a role in 30% of all cancers.  KRAS, a key member of the RAS protein family, is an attractive cancer target, as frequent point mutations in the KRAS gene render the protein constitutively active.  A no. of attempts have been made to target aberrant KRAS signaling by identifying small mol. compds. that (1) are synthetic lethal to mutant KRAS, (2) block KRAS/GEF interactions, (3) inhibit downstream KRAS effectors, or (4) inhibit the post-translational processing of RAS proteins.  In addn., inhibition of novel targets outside the main KRAS signaling pathway, specifically the cell cycle related kinase PLK1, has been shown have an effect in cells that harbor mutant KRAS.  Herein we review the use of various high-throughput screening assays utilized to identify new small-mol. compds. capable of targeting mutant KRAS-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogrA1KFIgQuLVg90H21EOLACvtfcHk0liF-BEl3OggwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsbc%253D&md5=83623863b372993bdb0171d93e1db205</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1021%2Fjm3017706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3017706%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKaiser%26aufirst%3DC.%2BE.%26aulast%3DFrett%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DH.%2BY.%26atitle%3DTargeting%2520mutant%2520kras%2520for%2520anticancer%2520therapeutics%253A%2520A%2520review%2520of%2520novel%2520small%2520molecule%2520modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5219%26epage%3D30%26doi%3D10.1021%2Fjm3017706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref157"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref157'); return false;" data-citation="" class="refNumLink">157</a></strong><div class="NLM_citation" id="cit157"><span><span class="NLM_contrib-group">Feig, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, G. M.</span><span> </span><span class="NLM_article-title">Inhibition of NIH-3T3 cell-proliferation by a mutant Ras protein with preferential affinity for GDP</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3235</span><span class="NLM_x">–</span> <span class="NLM_lpage">43</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1128%2FMCB.8.8.3235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=3145408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaL1cXlsVeks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1988&pages=3235-43&author=L.+A.+Feigauthor=G.+M.+Cooper&title=Inhibition+of+NIH-3T3+cell-proliferation+by+a+mutant+Ras+protein+with+preferential+affinity+for+GDP"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of NIH 3T3 cell proliferation by a mutant ras protein with preferential affinity for GDP</span></div><div class="casAuthors">Feig, Larry A.; Cooper, Geoffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3235-43</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">Substitution of asparagine for serine at position 17 decreased the affinity of rasHp21 for GTP 20- to 40-fold without significantly affecting its affinity for GDP.  Transfection of NIH 3T3 cells with a mammalian expression vector contg. the Asn-17 rasHgene and a Neor gene under the control of the same promoter yielded only a small fraction of the expected no. of G418-resistant colonies, indicating that expression of Asn-17 p21 inhibited cell proliferation.  The inhibitory effect of Asn-17 p21 required its localization to the plasma membrane and was reversed by coexpression of an activated ras gene, indicating that the mutant p21 blocked the endogenous ras function required for NIH 3T3 cell proliferation.  NIH 3T3 cells transformed by v-mos and v-raf, but not v-src, were resistant to inhibition by Asn-17 p21, indicating that the requirement for normal ras function can be bypassed by these cytoplasmic oncogenes.  The Asn-17 mutant represents a novel reagent for the study of ras function by virtue of its ability to inhibit cellular ras activity in vivo.  Since this phenotype is likely assocd. with the preferential affinity of the mutant protein for GDP, analogous mutations might also yield inhibitors of other proteins whose activities are regulated by guanine nucleotide binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM46qhA-D7prVg90H21EOLACvtfcHk0liSUY0993H_VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXlsVeks74%253D&md5=4f60717d2c8b090b19a65a5fd301b038</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1128%2FMCB.8.8.3235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.8.8.3235%26sid%3Dliteratum%253Aachs%26aulast%3DFeig%26aufirst%3DL.%2BA.%26aulast%3DCooper%26aufirst%3DG.%2BM.%26atitle%3DInhibition%2520of%2520NIH-3T3%2520cell-proliferation%2520by%2520a%2520mutant%2520Ras%2520protein%2520with%2520preferential%2520affinity%2520for%2520GDP%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1988%26volume%3D8%26spage%3D3235%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref158"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref158'); return false;" data-citation="" class="refNumLink">158</a></strong><div class="NLM_citation" id="cit158"><span><span class="NLM_contrib-group">Maurer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrenton, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitts, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skelton, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fauber, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokoe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludlam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starovasnik, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, G.</span><span> </span><span class="NLM_article-title">Small-molecule ligands bind to a distinct pocket in Ras and inhibit SOS-mediated nucleotide exchange activity</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5299</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span><span class="refDoi"> DOI: 10.1073/pnas.1116510109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1116510109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22431598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5299-304&author=T.+Maurerauthor=L.+S.+Garrentonauthor=A.+Ohauthor=K.+Pittsauthor=D.+J.+Andersonauthor=N.+J.+Skeltonauthor=B.+P.+Fauberauthor=B.+Panauthor=S.+Malekauthor=D.+Stokoeauthor=M.+J.+Ludlamauthor=K.+K.+Bowmanauthor=J.+Wuauthor=A.+M.+Giannettiauthor=M.+A.+Starovasnikauthor=I.+Mellmanauthor=P.+K.+Jacksonauthor=J.+Rudolphauthor=W.+Wangauthor=G.+Fang&title=Small-molecule+ligands+bind+to+a+distinct+pocket+in+Ras+and+inhibit+SOS-mediated+nucleotide+exchange+activity&doi=10.1073%2Fpnas.1116510109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule ligands bind to a distinct pocket in Ras and inhibit SOS-mediated nucleotide exchange activity</span></div><div class="casAuthors">Maurer, Till; Garrenton, Lindsay S.; Oh, Angela; Pitts, Keith; Anderson, Daniel J.; Skelton, Nicholas J.; Fauber, Benjamin P.; Pan, Borlan; Malek, Shiva; Stokoe, David; Ludlam, Mary J. C.; Bowman, Krista K.; Wu, Jiansheng; Giannetti, Anthony M.; Starovasnik, Melissa A.; Mellman, Ira; Jackson, Peter K.; Rudolph, Joachim; Wang, Weiru; Fang, Guowei</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5299-5304, S5299/1-S5299/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Ras gene is frequently mutated in cancer, and mutant Ras drives tumorigenesis.  Although Ras is a central oncogene, small mols. that bind to Ras in a well-defined manner and exert inhibitory effects have not been uncovered to date.  Through an NMR-based fragment screen, we identified a group of small mols. that all bind to a common site on Ras.  High-resoln. cocrystal structures delineated a unique ligand-binding pocket on the Ras protein that is adjacent to the switch I/II regions and can be expanded upon compd. binding.  Structure anal. predicts that compd.-binding interferes with the Ras/SOS interactions.  Indeed, selected compds. inhibit SOS-mediated nucleotide exchange and prevent Ras activation by blocking the formation of intermediates of the exchange reaction.  The discovery of a small-mol. binding pocket on Ras with functional significance provides a new direction in the search of therapeutically effective inhibitors of the Ras oncoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAXIzMe7yBqrVg90H21EOLACvtfcHk0liSUY0993H_VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOjsrc%253D&md5=a7210cad45154c42eb2a1faeabf88c20</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1116510109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1116510109%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DGarrenton%26aufirst%3DL.%2BS.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DPitts%26aufirst%3DK.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DLudlam%26aufirst%3DM.%2BJ.%26aulast%3DBowman%26aufirst%3DK.%2BK.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DStarovasnik%26aufirst%3DM.%2BA.%26aulast%3DMellman%26aufirst%3DI.%26aulast%3DJackson%26aufirst%3DP.%2BK.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DG.%26atitle%3DSmall-molecule%2520ligands%2520bind%2520to%2520a%2520distinct%2520pocket%2520in%2520Ras%2520and%2520inhibit%2520SOS-mediated%2520nucleotide%2520exchange%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5299%26epage%3D304%26doi%3D10.1073%2Fpnas.1116510109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref159"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref159'); return false;" data-citation="" class="refNumLink">159</a></strong><div class="NLM_citation" id="cit159"><span><span class="NLM_contrib-group">Wang, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span> </span><span class="NLM_article-title">Ras inhibition via direct Ras binding-is there a path forward?</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">5766</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.07.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2012.07.082" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=5766-76&author=W.+R.+Wangauthor=G.+W.+Fangauthor=J.+Rudolph&title=Ras+inhibition+via+direct+Ras+binding-is+there+a+path+forward%3F&doi=10.1016%2Fj.bmcl.2012.07.082"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.07.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.07.082%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%2BR.%26aulast%3DFang%26aufirst%3DG.%2BW.%26aulast%3DRudolph%26aufirst%3DJ.%26atitle%3DRas%2520inhibition%2520via%2520direct%2520Ras%2520binding-is%2520there%2520a%2520path%2520forward%253F%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D5766%26epage%3D76%26doi%3D10.1016%2Fj.bmcl.2012.07.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref160"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref160'); return false;" data-citation="" class="refNumLink">160</a></strong><div class="NLM_citation" id="cit160"><span><span class="NLM_contrib-group">Ostrem, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sos, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">503</span><span class="NLM_x">, </span> <span class="NLM_fpage">548</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-51&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+inhibitors+allosterically+control+GTP+affinity+and+effector+interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0liSUY0993H_VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520inhibitors%2520allosterically%2520control%2520GTP%2520affinity%2520and%2520effector%2520interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D548%26epage%3D51%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref161"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref161'); return false;" data-citation="" class="refNumLink">161</a></strong><div class="NLM_citation" id="cit161"><span><span class="NLM_contrib-group">Christ, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debyser, Z.</span><span> </span><span class="NLM_article-title">The LEDGF/p75 integrase interaction, a novel target for anti-HIV therapy</span> <span class="citation_source-journal">Virology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">435</span><span class="NLM_x">, </span> <span class="NLM_fpage">102</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1016/j.virol.2012.09.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.virol.2012.09.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23217620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaju7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2013&pages=102-9&author=F.+Christauthor=Z.+Debyser&title=The+LEDGF%2Fp75+integrase+interaction%2C+a+novel+target+for+anti-HIV+therapy&doi=10.1016%2Fj.virol.2012.09.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">The LEDGF/p75 integrase interaction, a novel target for anti-HIV therapy</span></div><div class="casAuthors">Christ, Frauke; Debyser, Zeger</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-109</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  To accomplish their viral life cycle, lentiviruses such as HIV highjack host proteins, the so-called cellular co-factors of replication.  Lens Epithelium-derived Growth factor (LEDGF/p75), a transcriptional co-activator, is a co-factor of HIV-integrase (IN) and is required for the tethering and correct integration of the viral genome into the host chromatin.  Due to its important role in HIV-replication the LEDGF/p75-IN interaction is an attractive antiviral novel target for the treatment of HIV/AIDS.  Intensive drug discovery efforts over the past years have validated the LEDGF/p75-IN interaction as a drugable target for antiviral therapy and have resulted in the design and synthesis of LEDGINs, small mol. inhibitors binding to the dimer interface of HIV-integrase and inhibiting viral replication with a dual mechanism of action: potent inhibition of the LEDGF/p75-IN protein-protein interaction and allosteric inhibition of the catalytic function.  Furthermore they inhibit both early and late steps of the replication cycle which increases their potential for further clin. development.  In this review the authors will highlight the research validating the LEDGF/p75-IN interaction as a target for anti-HIV drug discovery and the recent advances in the design and development of LEDGINs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_mTW5TAo2y7Vg90H21EOLACvtfcHk0lhFCNVJ4TyYUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaju7nN&md5=2249c51a1792bc8edc5ce449500e6627</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2012.09.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2012.09.033%26sid%3Dliteratum%253Aachs%26aulast%3DChrist%26aufirst%3DF.%26aulast%3DDebyser%26aufirst%3DZ.%26atitle%3DThe%2520LEDGF%252Fp75%2520integrase%2520interaction%252C%2520a%2520novel%2520target%2520for%2520anti-HIV%2520therapy%26jtitle%3DVirology%26date%3D2013%26volume%3D435%26spage%3D102%26epage%3D9%26doi%3D10.1016%2Fj.virol.2012.09.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref162"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref162'); return false;" data-citation="" class="refNumLink">162</a></strong><div class="NLM_citation" id="cit162"><span><span class="NLM_contrib-group">Malet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcelin, A. G.</span><span> </span><span class="NLM_article-title">The future of integrase inhibitors of HIV-1</span> <span class="citation_source-journal">Curr. Opin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.coviro.2012.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.coviro.2012.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22980926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSltrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=580-7&author=I.+Maletauthor=V.+Calvezauthor=A.+G.+Marcelin&title=The+future+of+integrase+inhibitors+of+HIV-1&doi=10.1016%2Fj.coviro.2012.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">The future of integrase inhibitors of HIV-1</span></div><div class="casAuthors">Malet, Isabelle; Calvez, Vincent; Marcelin, Anne-Genevieve</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">580-587</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Integration of the HIV-1 DNA is required and essential to maintain the viral DNA in the infected cell.  Integration process occurs in several events, mainly endonucleolytic processing of the 3' ends of the viral DNA and strand transfer or joining of the viral and cellular DNA.  The design and discovery of integrase inhibitors were first focused at targeting the catalytic site of IN with a specific effect on strand transfer.  Several integrase inhibitors were developed clin., two first generation inhibitors, raltegravir and elvitegravir and then two second-generation inhibitors, dolutegravir and MK-2058.  Recently, allosteric integrase inhibitors intended to interfere with the integrase-LEDGF/p75 interaction have been designed.  These new inhibitors called LEDGINs have an effect on 3' processing and strand transfer.  Thus, integrase inhibitors present a real added value in combined treatment for naive and experienced HIV infected patients.  Combination expts. of LEDGINs and raltegravir suggest that these inhibitors could act additively despite sharing the same viral target.  Future therapy could involve combinations of inhibitors of IN function acting though different binding pockets within IN.  The place of this class on HIV inhibitors and their future role in perspective of novel therapies to eliminate latent HIV reservoirs and infection for cure should also be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvFDf9hYvRAbVg90H21EOLACvtfcHk0lhFCNVJ4TyYUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSltrrM&md5=ea4826081ba3b0dfc3948bc45ae9badc</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2012.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2012.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DMalet%26aufirst%3DI.%26aulast%3DCalvez%26aufirst%3DV.%26aulast%3DMarcelin%26aufirst%3DA.%2BG.%26atitle%3DThe%2520future%2520of%2520integrase%2520inhibitors%2520of%2520HIV-1%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2012%26volume%3D2%26spage%3D580%26epage%3D7%26doi%3D10.1016%2Fj.coviro.2012.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref163"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref163'); return false;" data-citation="" class="refNumLink">163</a></strong><div class="NLM_citation" id="cit163"><span><span class="NLM_contrib-group">Demeulemeester, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaltin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Maeyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debyser, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christ, F.</span><span> </span><span class="NLM_article-title">LEDGINs, non-catalytic site inhibitors of HIV-1 integrase: A patent review (2006 - 2014)</span> <span class="citation_source-journal">Expert Opin. Ther. Pat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1517/13543776.2014.898753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1517%2F13543776.2014.898753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=24666332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotF2rs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=609-32&author=J.+Demeulemeesterauthor=P.+Chaltinauthor=A.+Marchandauthor=M.+De+Maeyerauthor=Z.+Debyserauthor=F.+Christ&title=LEDGINs%2C+non-catalytic+site+inhibitors+of+HIV-1+integrase%3A+A+patent+review+%282006+-+2014%29&doi=10.1517%2F13543776.2014.898753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">LEDGINs, non-catalytic site inhibitors of HIV-1 integrase: a patent review (2006 - 2014)</span></div><div class="casAuthors">Demeulemeester, Jonas; Chaltin, Patrick; Marchand, Arnaud; De Maeyer, Marc; Debyser, Zeger; Christ, Frauke</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">609-632</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Integration of the viral genome into the host cell chromatin is a central step in the replication cycle of the HIV.  Blocking the viral integrase (IN) enzyme therefore provides an attractive therapeutic strategy, as evidenced by the recent clin. approval of three IN strand transfer inhibitors.  Viral resistance and cross-resistance among these inhibitors, however, warrant the search for compds. targeting HIV integration through alternative mechanisms of action.  The most potent class of allosteric IN inhibitors was independently identified at the University of Leuven, Belgium, and at Boehringer Ingelheim, Canada.  These compds., coined LEDGINs (after the lens epithelium-derived growth factor/p75 cofactor binding pocket on IN) or non-catalytic site IN inhibitors (NCINIs) by the resp. groups, have shown remarkable antiviral activity.  This review provides a brief introduction to the compd. class and discusses the recent patent literature (2006 to the present).  Expert opinion: LEDGINs are still early in development.  Trials with clin. candidate BI-224436 were put on hold despite promising results.  Literature, however, reveals that almost all major pharmaceutical companies active in the treatment of HIV/AIDS have taken a significant interest in this class.  As a result, several of these inhibitors may soon enter clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowB3WZSin3PLVg90H21EOLACvtfcHk0lhFCNVJ4TyYUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotF2rs70%253D&md5=cf3625cadfaf5ea12b5dc118befdb18d</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.898753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.898753%26sid%3Dliteratum%253Aachs%26aulast%3DDemeulemeester%26aufirst%3DJ.%26aulast%3DChaltin%26aufirst%3DP.%26aulast%3DMarchand%26aufirst%3DA.%26aulast%3DDe%2BMaeyer%26aufirst%3DM.%26aulast%3DDebyser%26aufirst%3DZ.%26aulast%3DChrist%26aufirst%3DF.%26atitle%3DLEDGINs%252C%2520non-catalytic%2520site%2520inhibitors%2520of%2520HIV-1%2520integrase%253A%2520A%2520patent%2520review%2520%25282006%2520-%25202014%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2014%26volume%3D24%26spage%3D609%26epage%3D32%26doi%3D10.1517%2F13543776.2014.898753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149'],'ref150':['cit150'],'ref151':['cit151'],'ref152':['cit152'],'ref153':['cit153'],'ref154':['cit154'],'ref155':['cit155'],'ref156':['cit156'],'ref157':['cit157'],'ref158':['cit158'],'ref159':['cit159'],'ref160':['cit160'],'ref161':['cit161'],'ref162':['cit162'],'ref163':['cit163']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 60 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Paula C. Ortet, Samantha N. Muellers, Lauren A. Viarengo-Baker, Kristina Streu, Blair R. Szymczyna, Aaron B. Beeler, Karen N. Allen, <span class="NLM_string-name hlFld-ContribAuthor">Adrian Whitty</span>. </span><span class="cited-content_cbyCitation_article-title">Recapitulating the Binding Affinity of Nrf2 for KEAP1 in a Cyclic Heptapeptide, Guided by NMR, X-ray Crystallography, and Machine Learning. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (10)
                                     , 3779-3793. <a href="https://doi.org/10.1021/jacs.0c09799" title="DOI URL">https://doi.org/10.1021/jacs.0c09799</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c09799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c09799%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DRecapitulating%252Bthe%252BBinding%252BAffinity%252Bof%252BNrf2%252Bfor%252BKEAP1%252Bin%252Ba%252BCyclic%252BHeptapeptide%25252C%252BGuided%252Bby%252BNMR%25252C%252BX-ray%252BCrystallography%25252C%252Band%252BMachine%252BLearning%26aulast%3DOrtet%26aufirst%3DPaula%2BC.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D13092020%26date%3D08032021%26volume%3D143%26issue%3D10%26spage%3D3779%26epage%3D3793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Begnini, Vasanthanathan Poongavanam, Björn Over, Marie Castaldo, Stefan Geschwindner, Patrik Johansson, Mohit Tyagi, Christian Tyrchan, Lisa Wissler, Peter Sjö, Stefan Schiesser, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Mining Natural Products for Macrocycles to Drug Difficult Targets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (2)
                                     , 1054-1072. <a href="https://doi.org/10.1021/acs.jmedchem.0c01569" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01569</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01569%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMining%252BNatural%252BProducts%252Bfor%252BMacrocycles%252Bto%252BDrug%252BDifficult%252BTargets%26aulast%3DBegnini%26aufirst%3DFabio%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D08092020%26date%3D18122020%26volume%3D64%26issue%3D2%26spage%3D1054%26epage%3D1072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Amanda E. Wakefield, Christine Yueh, Dmitri Beglov, Marcelo S. Castilho, Dima Kozakov, György M. Keserű, Adrian Whitty, <span class="NLM_string-name hlFld-ContribAuthor">Sandor Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Benchmark Sets for Binding Hot Spot Identification in Fragment-Based Ligand Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6612-6623. <a href="https://doi.org/10.1021/acs.jcim.0c00877" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00877</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00877%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBenchmark%252BSets%252Bfor%252BBinding%252BHot%252BSpot%252BIdentification%252Bin%252BFragment-Based%252BLigand%252BDiscovery%26aulast%3DWakefield%26aufirst%3DAmanda%2BE.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31072020%26date%3D08122020%26volume%3D60%26issue%3D12%26spage%3D6612%26epage%3D6623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter R. Curran, Chris J. Radoux, Mihaela D. Smilova, Richard A. Sykes, Alicia P. Higueruelo, Anthony R. Bradley, Brian D. Marsden, David R. Spring, Tom L. Blundell, Andrew R. Leach, William R. Pitt, <span class="NLM_string-name hlFld-ContribAuthor">Jason C. Cole</span>. </span><span class="cited-content_cbyCitation_article-title">Hotspots API: A Python Package for the Detection of Small Molecule Binding Hotspots and Application to Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 1911-1916. <a href="https://doi.org/10.1021/acs.jcim.9b00996" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00996</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00996%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHotspots%252BAPI%25253A%252BA%252BPython%252BPackage%252Bfor%252Bthe%252BDetection%252Bof%252BSmall%252BMolecule%252BBinding%252BHotspots%252Band%252BApplication%252Bto%252BStructure-Based%252BDrug%252BDesign%26aulast%3DCurran%26aufirst%3DPeter%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D28102019%26date%3D02042020%26date%3D24032020%26volume%3D60%26issue%3D4%26spage%3D1911%26epage%3D1916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mengqi Zhong, Andrew Lynch, Samantha N. Muellers, Stefan Jehle, Lingqi Luo, David R. Hall, Reina Iwase, James P. Carolan, Megan Egbert, Amanda Wakefield, Kristina Streu, Christine M. Harvey, Paula C. Ortet, Dima Kozakov, Sandor Vajda, Karen N. Allen, <span class="NLM_string-name hlFld-ContribAuthor">Adrian Whitty</span>. </span><span class="cited-content_cbyCitation_article-title">Interaction Energetics and Druggability of the Protein–Protein Interaction between Kelch-like ECH-Associated Protein 1 (KEAP1) and Nuclear Factor Erythroid 2 Like 2 (Nrf2). </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2020,</strong> <em>59 </em>
                                    (4)
                                     , 563-581. <a href="https://doi.org/10.1021/acs.biochem.9b00943" title="DOI URL">https://doi.org/10.1021/acs.biochem.9b00943</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.9b00943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.9b00943%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DInteraction%252BEnergetics%252Band%252BDruggability%252Bof%252Bthe%252BProtein%2525E2%252580%252593Protein%252BInteraction%252Bbetween%252BKelch-like%252BECH-Associated%252BProtein%252B1%252B%252528KEAP1%252529%252Band%252BNuclear%252BFactor%252BErythroid%252B2%252BLike%252B2%252B%252528Nrf2%252529%26aulast%3DZhong%26aufirst%3DMengqi%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D20102019%26date%3D17122019%26date%3D02012020%26date%3D18122019%26volume%3D59%26issue%3D4%26spage%3D563%26epage%3D581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Megan Egbert, Adrian Whitty, György M. Keserű, <span class="NLM_string-name hlFld-ContribAuthor">Sandor Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Why Some Targets Benefit from beyond Rule of Five Drugs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (22)
                                     , 10005-10025. <a href="https://doi.org/10.1021/acs.jmedchem.8b01732" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01732</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01732%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DWhy%252BSome%252BTargets%252BBenefit%252Bfrom%252Bbeyond%252BRule%252Bof%252BFive%252BDrugs%26aulast%3DEgbert%26aufirst%3DMegan%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D13112018%26date%3D26062019%26date%3D12062019%26volume%3D62%26issue%3D22%26spage%3D10005%26epage%3D10025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maurice Michel, Torkild Visnes, Evert J. Homan, Brinton Seashore-Ludlow, Mattias Hedenström, Elisée Wiita, Karl Vallin, Cynthia B. J. Paulin, Jiaxi Zhang, Olov Wallner, Martin Scobie, Andreas Schmidt, Annika Jenmalm-Jensen, Ulrika Warpman Berglund, <span class="NLM_string-name hlFld-ContribAuthor">Thomas Helleday</span>. </span><span class="cited-content_cbyCitation_article-title">Computational and Experimental Druggability Assessment of Human DNA Glycosylases. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (7)
                                     , 11642-11656. <a href="https://doi.org/10.1021/acsomega.9b00162" title="DOI URL">https://doi.org/10.1021/acsomega.9b00162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b00162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b00162%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DComputational%252Band%252BExperimental%252BDruggability%252BAssessment%252Bof%252BHuman%252BDNA%252BGlycosylases%26aulast%3DMichel%26aufirst%3DMaurice%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D17012019%26date%3D21052019%26volume%3D4%26issue%3D7%26spage%3D11642%26epage%3D11656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christine Yueh, Justin Rettenmaier, Bing Xia, David R. Hall, Andrey Alekseenko, Kathryn A. Porter, Krister Barkovich, Gyorgy Keseru, Adrian Whitty, James A. Wells, Sandor Vajda, <span class="NLM_string-name hlFld-ContribAuthor">Dima Kozakov</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase Atlas: Druggability Analysis of Potential Allosteric Sites in Kinases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6512-6524. <a href="https://doi.org/10.1021/acs.jmedchem.9b00089" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00089%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DKinase%252BAtlas%25253A%252BDruggability%252BAnalysis%252Bof%252BPotential%252BAllosteric%252BSites%252Bin%252BKinases%26aulast%3DYueh%26aufirst%3DChristine%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D14012019%26date%3D05072019%26date%3D04062019%26volume%3D62%26issue%3D14%26spage%3D6512%26epage%3D6524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. Roy  Kimura</span>, <span class="hlFld-ContribAuthor ">Hai Peng  Hu</span>, <span class="hlFld-ContribAuthor ">Anatoly M.  Ruvinsky</span>, <span class="hlFld-ContribAuthor ">Woody  Sherman</span>, and <span class="hlFld-ContribAuthor ">Angelo D.  Favia</span>  . </span><span class="cited-content_cbyCitation_article-title">Deciphering Cryptic Binding Sites on Proteins by Mixed-Solvent Molecular Dynamics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (6)
                                     , 1388-1401. <a href="https://doi.org/10.1021/acs.jcim.6b00623" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00623</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00623%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDeciphering%252BCryptic%252BBinding%252BSites%252Bon%252BProteins%252Bby%252BMixed-Solvent%252BMolecular%252BDynamics%26aulast%3DKimura%26aufirst%3DS.%2BRoy%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D14102016%26date%3D08062017%26date%3D26062017%26date%3D24052017%26volume%3D57%26issue%3D6%26spage%3D1388%26epage%3D1401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Molly  Chilton</span>, <span class="hlFld-ContribAuthor ">Ben  Clennell</span>, <span class="hlFld-ContribAuthor ">Fredrik  Edfeldt</span>, and <span class="hlFld-ContribAuthor ">Stefan  Geschwindner</span>  . </span><span class="cited-content_cbyCitation_article-title">Hot-Spotting with Thermal Scanning: A Ligand- and Structure-Independent Assessment of Target Ligandability. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (12)
                                     , 4923-4931. <a href="https://doi.org/10.1021/acs.jmedchem.7b00208" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00208</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00208%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHot-Spotting%252Bwith%252BThermal%252BScanning%25253A%252BA%252BLigand-%252Band%252BStructure-Independent%252BAssessment%252Bof%252BTarget%252BLigandability%26aulast%3DChilton%26aufirst%3DMolly%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D20022017%26date%3D02062017%26date%3D22062017%26date%3D24052017%26volume%3D60%26issue%3D12%26spage%3D4923%26epage%3D4931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Vulpetti</span>, <span class="hlFld-ContribAuthor ">Stefan  Randl</span>, <span class="hlFld-ContribAuthor ">Simon  Rüdisser</span>, <span class="hlFld-ContribAuthor ">Nils  Ostermann</span>, <span class="hlFld-ContribAuthor ">Paul  Erbel</span>, <span class="hlFld-ContribAuthor ">Aengus  Mac Sweeney</span>, <span class="hlFld-ContribAuthor ">Thomas  Zoller</span>, <span class="hlFld-ContribAuthor ">Bahaa  Salem</span>, <span class="hlFld-ContribAuthor ">Bernd  Gerhartz</span>, <span class="hlFld-ContribAuthor ">Frederic  Cumin</span>, <span class="hlFld-ContribAuthor ">Ulrich  Hommel</span>, <span class="hlFld-ContribAuthor ">Claudio  Dalvit</span>, <span class="hlFld-ContribAuthor ">Edwige  Lorthiois</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Maibaum</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Library Design and Fragment Screening for the Identification of Reversible Complement Factor D Protease Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 1946-1958. <a href="https://doi.org/10.1021/acs.jmedchem.6b01684" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01684</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01684%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BLibrary%252BDesign%252Band%252BFragment%252BScreening%252Bfor%252Bthe%252BIdentification%252Bof%252BReversible%252BComplement%252BFactor%252BD%252BProtease%252BInhibitors%26aulast%3DVulpetti%26aufirst%3DAnna%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D21112016%26date%3D20022017%26date%3D09032017%26date%3D03022017%26volume%3D60%26issue%3D5%26spage%3D1946%26epage%3D1958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">György M.  Keserű</span>, <span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>, <span class="hlFld-ContribAuthor ">György G.  Ferenczy</span>, <span class="hlFld-ContribAuthor ">Michael M.  Hann</span>, <span class="hlFld-ContribAuthor ">Christopher W.  Murray</span>, and <span class="hlFld-ContribAuthor ">Stephen D.  Pickett</span>  . </span><span class="cited-content_cbyCitation_article-title">Design Principles for Fragment Libraries: Maximizing the Value of Learnings from Pharma Fragment-Based Drug Discovery (FBDD) Programs for Use in Academia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (18)
                                     , 8189-8206. <a href="https://doi.org/10.1021/acs.jmedchem.6b00197" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00197</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00197%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252BPrinciples%252Bfor%252BFragment%252BLibraries%25253A%252BMaximizing%252Bthe%252BValue%252Bof%252BLearnings%252Bfrom%252BPharma%252BFragment-Based%252BDrug%252BDiscovery%252B%252528FBDD%252529%252BPrograms%252Bfor%252BUse%252Bin%252BAcademia%26aulast%3DKeser%25C5%25B1%26aufirst%3DGy%25C3%25B6rgy%2BM.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D05022016%26date%3D16052016%26date%3D22092016%26date%3D28042016%26volume%3D59%26issue%3D18%26spage%3D8189%26epage%3D8206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chris J.  Radoux</span>, <span class="hlFld-ContribAuthor ">Tjelvar S. G.  Olsson</span>, <span class="hlFld-ContribAuthor ">Will R.  Pitt</span>, <span class="hlFld-ContribAuthor ">Colin R.  Groom</span>, and <span class="hlFld-ContribAuthor ">Tom L.  Blundell</span>  . </span><span class="cited-content_cbyCitation_article-title">Identifying Interactions that Determine Fragment Binding at Protein Hotspots. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4314-4325. <a href="https://doi.org/10.1021/acs.jmedchem.5b01980" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01980</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01980%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentifying%252BInteractions%252Bthat%252BDetermine%252BFragment%252BBinding%252Bat%252BProtein%252BHotspots%26aulast%3DRadoux%26aufirst%3DChris%2BJ.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D30122015%26date%3D11042016%26date%3D12052016%26date%3D04042016%26volume%3D59%26issue%3D9%26spage%3D4314%26epage%3D4325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olivia  Teixeira</span>, <span class="hlFld-ContribAuthor ">Pedro  Lacerda</span>, <span class="hlFld-ContribAuthor ">Thamires Quadros  Froes</span>, <span class="hlFld-ContribAuthor ">Maria Cristina  Nonato</span>, <span class="hlFld-ContribAuthor ">Marcelo Santos  Castilho</span>. </span><span class="cited-content_cbyCitation_article-title">Druggable hot spots in trypanothione reductase: novel insights and opportunities for drug discovery revealed by DRUGpy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2021,</strong> <em>6 </em><a href="https://doi.org/10.1007/s10822-021-00403-8" title="DOI URL">https://doi.org/10.1007/s10822-021-00403-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-021-00403-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-021-00403-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DDruggable%252Bhot%252Bspots%252Bin%252Btrypanothione%252Breductase%25253A%252Bnovel%252Binsights%252Band%252Bopportunities%252Bfor%252Bdrug%252Bdiscovery%252Brevealed%252Bby%252BDRUGpy%26aulast%3DTeixeira%26aufirst%3DOlivia%26date%3D2021%26date%3D2021%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iara Aimê  Cardoso</span>, <span class="hlFld-ContribAuthor ">Aline Kusumota Luiz  de Souza</span>, <span class="hlFld-ContribAuthor ">Adam Muslem George  Burgess</span>, <span class="hlFld-ContribAuthor ">Iain Wyllie  Chalmers</span>, <span class="hlFld-ContribAuthor ">Karl Francis  Hoffmann</span>, <span class="hlFld-ContribAuthor ">Maria Cristina  Nonato</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of class II fumarase from Schistosoma mansoni provides the molecular basis for selective inhibition. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Biological Macromolecules</span><span> <strong>2021,</strong> <em>175 </em>, 406-421. <a href="https://doi.org/10.1016/j.ijbiomac.2021.01.180" title="DOI URL">https://doi.org/10.1016/j.ijbiomac.2021.01.180</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijbiomac.2021.01.180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijbiomac.2021.01.180%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Biological%2520Macromolecules%26atitle%3DCharacterization%252Bof%252Bclass%252BII%252Bfumarase%252Bfrom%252BSchistosoma%252Bmansoni%252Bprovides%252Bthe%252Bmolecular%252Bbasis%252Bfor%252Bselective%252Binhibition%26aulast%3DCardoso%26aufirst%3DIara%2BAim%25C3%25AA%26date%3D2021%26volume%3D175%26spage%3D406%26epage%3D421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miroslava  Nedyalkova</span>, <span class="hlFld-ContribAuthor ">Vasil  Simeonov</span>. </span><span class="cited-content_cbyCitation_article-title">Partitioning Pattern of Natural Products Based on Molecular Properties Descriptors Representing Drug-Likeness. </span><span class="cited-content_cbyCitation_journal-name">Symmetry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (4)
                                     , 546. <a href="https://doi.org/10.3390/sym13040546" title="DOI URL">https://doi.org/10.3390/sym13040546</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/sym13040546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fsym13040546%26sid%3Dliteratum%253Aachs%26jtitle%3DSymmetry%26atitle%3DPartitioning%252BPattern%252Bof%252BNatural%252BProducts%252BBased%252Bon%252BMolecular%252BProperties%252BDescriptors%252BRepresenting%252BDrug-Likeness%26aulast%3DNedyalkova%26aufirst%3DMiroslava%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D4%26spage%3D546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lauren A.  Viarengo-Baker</span>, <span class="hlFld-ContribAuthor ">Lauren E.  Brown</span>, <span class="hlFld-ContribAuthor ">Anna A.  Rzepiela</span>, <span class="hlFld-ContribAuthor ">Adrian  Whitty</span>. </span><span class="cited-content_cbyCitation_article-title">Defining and navigating macrocycle chemical space. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (12)
                                     , 4309-4328. <a href="https://doi.org/10.1039/D0SC05788F" title="DOI URL">https://doi.org/10.1039/D0SC05788F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC05788F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC05788F%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DDefining%252Band%252Bnavigating%252Bmacrocycle%252Bchemical%252Bspace%26aulast%3DViarengo-Baker%26aufirst%3DLauren%2BA.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D12%26spage%3D4309%26epage%3D4328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Zang</span>, <span class="hlFld-ContribAuthor ">Fei  Ye</span>, <span class="hlFld-ContribAuthor ">Sara M. Ø.  Solbak</span>, <span class="hlFld-ContribAuthor ">Lars J.  Høj</span>, <span class="hlFld-ContribAuthor ">Mingjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Anders  Bach</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Novel Fragments Binding to the PDZ1‐2 Domain of PSD‐95. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (6)
                                     , 949-954. <a href="https://doi.org/10.1002/cmdc.202000865" title="DOI URL">https://doi.org/10.1002/cmdc.202000865</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000865%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DIdentification%252Bof%252BNovel%252BFragments%252BBinding%252Bto%252Bthe%252BPDZ1%2525E2%252580%2525902%252BDomain%252Bof%252BPSD%2525E2%252580%25259095%26aulast%3DZang%26aufirst%3DJie%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D6%26spage%3D949%26epage%3D954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Harald  Hampel</span>, <span class="hlFld-ContribAuthor ">Andrea  Vergallo</span>, <span class="hlFld-ContribAuthor ">Filippo  Caraci</span>, <span class="hlFld-ContribAuthor ">A. Claudio  Cuello</span>, <span class="hlFld-ContribAuthor ">Pablo  Lemercier</span>, <span class="hlFld-ContribAuthor ">Bruno  Vellas</span>, <span class="hlFld-ContribAuthor ">Kelly Virecoulon  Giudici</span>, <span class="hlFld-ContribAuthor ">Filippo  Baldacci</span>, <span class="hlFld-ContribAuthor ">Britta  Hänisch</span>, <span class="hlFld-ContribAuthor ">Marion  Haberkamp</span>, <span class="hlFld-ContribAuthor ">Karl  Broich</span>, <span class="hlFld-ContribAuthor ">Robert  Nisticò</span>, <span class="hlFld-ContribAuthor ">Enzo  Emanuele</span>, <span class="hlFld-ContribAuthor ">Francisco  Llavero</span>, <span class="hlFld-ContribAuthor ">José L.  Zugaza</span>, <span class="hlFld-ContribAuthor ">Alejandro  Lucía</span>, <span class="hlFld-ContribAuthor ">Ezio  Giacobini</span>, <span class="hlFld-ContribAuthor ">Simone  Lista</span>. </span><span class="cited-content_cbyCitation_article-title">Future avenues for Alzheimer's disease detection and therapy: liquid biopsy, intracellular signaling modulation, systems pharmacology drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Neuropharmacology</span><span> <strong>2021,</strong> <em>185 </em>, 108081. <a href="https://doi.org/10.1016/j.neuropharm.2020.108081" title="DOI URL">https://doi.org/10.1016/j.neuropharm.2020.108081</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.neuropharm.2020.108081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.neuropharm.2020.108081%26sid%3Dliteratum%253Aachs%26jtitle%3DNeuropharmacology%26atitle%3DFuture%252Bavenues%252Bfor%252BAlzheimer%252527s%252Bdisease%252Bdetection%252Band%252Btherapy%25253A%252Bliquid%252Bbiopsy%25252C%252Bintracellular%252Bsignaling%252Bmodulation%25252C%252Bsystems%252Bpharmacology%252Bdrug%252Bdiscovery%26aulast%3DHampel%26aufirst%3DHarald%26date%3D2021%26volume%3D185%26spage%3D108081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emilia P.  Barros</span>, <span class="hlFld-ContribAuthor ">Özlem  Demir</span>, <span class="hlFld-ContribAuthor ">Jenaro  Soto</span>, <span class="hlFld-ContribAuthor ">Melanie J.  Cocco</span>, <span class="hlFld-ContribAuthor ">Rommie E.  Amaro</span>. </span><span class="cited-content_cbyCitation_article-title">Markov state models and NMR uncover an overlooked allosteric loop in p53. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (5)
                                     , 1891-1900. <a href="https://doi.org/10.1039/D0SC05053A" title="DOI URL">https://doi.org/10.1039/D0SC05053A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC05053A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC05053A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DMarkov%252Bstate%252Bmodels%252Band%252BNMR%252Buncover%252Ban%252Boverlooked%252Ballosteric%252Bloop%252Bin%252Bp53%26aulast%3DBarros%26aufirst%3DEmilia%2BP.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D5%26spage%3D1891%26epage%3D1900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariangela  Lopreiato</span>, <span class="hlFld-ContribAuthor ">Samuele  Di Cristofano</span>, <span class="hlFld-ContribAuthor ">Rossana  Cocchiola</span>, <span class="hlFld-ContribAuthor ">Alessia  Mariano</span>, <span class="hlFld-ContribAuthor ">Libera  Guerrizio</span>, <span class="hlFld-ContribAuthor ">Roberto  Scandurra</span>, <span class="hlFld-ContribAuthor ">Luciana  Mosca</span>, <span class="hlFld-ContribAuthor ">Domenico  Raimondo</span>, <span class="hlFld-ContribAuthor ">Anna  Scotto d’Abusco</span>. </span><span class="cited-content_cbyCitation_article-title">Biochemical and Computational Studies of the Interaction between a Glucosamine Derivative, NAPA, and the IKKα Kinase. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (4)
                                     , 1643. <a href="https://doi.org/10.3390/ijms22041643" title="DOI URL">https://doi.org/10.3390/ijms22041643</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22041643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22041643%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DBiochemical%252Band%252BComputational%252BStudies%252Bof%252Bthe%252BInteraction%252Bbetween%252Ba%252BGlucosamine%252BDerivative%25252C%252BNAPA%25252C%252Band%252Bthe%252BIKK%2525CE%2525B1%252BKinase%26aulast%3DLopreiato%26aufirst%3DMariangela%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D4%26spage%3D1643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew R  Leach</span>, <span class="hlFld-ContribAuthor ">Chris  Radoux</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Drug Target Tractability Analysis. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 145-153. <a href="https://doi.org/10.1016/B978-0-12-801238-3.11531-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-801238-3.11531-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801238-3.11531-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801238-3.11531-4%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252BDrug%252BTarget%252BTractability%252BAnalysis%26aulast%3DLeach%26aufirst%3DAndrew%2BR%26date%3D2021%26spage%3D145%26epage%3D153%26pub%3DElsevier%26atitle%3DSystems%252BMedicine%26date%3D2021%26volume%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael S.  Saporito</span>. </span><span class="cited-content_cbyCitation_article-title">Translational research—from basic science to an approved therapeutic—an overview. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 663-681. <a href="https://doi.org/10.1016/B978-0-12-820007-0.00034-9" title="DOI URL">https://doi.org/10.1016/B978-0-12-820007-0.00034-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820007-0.00034-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820007-0.00034-9%26sid%3Dliteratum%253Aachs%26atitle%3DTranslational%252Bresearch%2525E2%252580%252594from%252Bbasic%252Bscience%252Bto%252Ban%252Bapproved%252Btherapeutic%2525E2%252580%252594an%252Boverview%26aulast%3DSaporito%26aufirst%3DMichael%2BS.%26date%3D2021%26spage%3D663%26epage%3D681%26pub%3DElsevier%26atitle%3DRemington%26date%3D2021%26volume%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Claudio N.  Cavasotto</span>, <span class="hlFld-ContribAuthor ">Maximiliano Sánchez  Lamas</span>, <span class="hlFld-ContribAuthor ">Julián  Maggini</span>. </span><span class="cited-content_cbyCitation_article-title">Functional and druggability analysis of the SARS-CoV-2 proteome. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2021,</strong> <em>890 </em>, 173705. <a href="https://doi.org/10.1016/j.ejphar.2020.173705" title="DOI URL">https://doi.org/10.1016/j.ejphar.2020.173705</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2020.173705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2020.173705%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DFunctional%252Band%252Bdruggability%252Banalysis%252Bof%252Bthe%252BSARS-CoV-2%252Bproteome%26aulast%3DCavasotto%26aufirst%3DClaudio%2BN.%26date%3D2021%26volume%3D890%26spage%3D173705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Megan  Egbert</span>, <span class="hlFld-ContribAuthor ">Kathryn A.  Porter</span>, <span class="hlFld-ContribAuthor ">Usman  Ghani</span>, <span class="hlFld-ContribAuthor ">Sergei  Kotelnikov</span>, <span class="hlFld-ContribAuthor ">Thu  Nguyen</span>, <span class="hlFld-ContribAuthor ">Ryota  Ashizawa</span>, <span class="hlFld-ContribAuthor ">Dima  Kozakov</span>, <span class="hlFld-ContribAuthor ">Sandor  Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Conservation of binding properties in protein models. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2021,</strong> <em>19 </em>, 2549-2566. <a href="https://doi.org/10.1016/j.csbj.2021.04.048" title="DOI URL">https://doi.org/10.1016/j.csbj.2021.04.048</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2021.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2021.04.048%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DConservation%252Bof%252Bbinding%252Bproperties%252Bin%252Bprotein%252Bmodels%26aulast%3DEgbert%26aufirst%3DMegan%26date%3D2021%26volume%3D19%26spage%3D2549%26epage%3D2566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fangfei  Li</span>, <span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Shasha  Feng</span>, <span class="hlFld-ContribAuthor ">Baichun  Hu</span>, <span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Wei  Li</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Computational investigation reveals Picrasidine C as selective PPARα lead: binding pattern, selectivity mechanism and ADME/tox profile. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2020,</strong> <em>38 </em>
                                    (18)
                                     , 5401-5418. <a href="https://doi.org/10.1080/07391102.2019.1699861" title="DOI URL">https://doi.org/10.1080/07391102.2019.1699861</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2019.1699861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2019.1699861%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DComputational%252Binvestigation%252Breveals%252BPicrasidine%252BC%252Bas%252Bselective%252BPPAR%2525CE%2525B1%252Blead%25253A%252Bbinding%252Bpattern%25252C%252Bselectivity%252Bmechanism%252Band%252BADME%25252Ftox%252Bprofile%26aulast%3DLi%26aufirst%3DFangfei%26date%3D2020%26date%3D2019%26volume%3D38%26issue%3D18%26spage%3D5401%26epage%3D5418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Quiara Lovatti  Alves</span>, <span class="hlFld-ContribAuthor ">Raiana dos Anjos  Moraes</span>, <span class="hlFld-ContribAuthor ">Thamires Quadros  Froes</span>, <span class="hlFld-ContribAuthor ">Marcelo Santos  Castilho</span>, <span class="hlFld-ContribAuthor ">Rodrigo Santos  Aquino de Araújo</span>, <span class="hlFld-ContribAuthor ">José Maria  Barbosa-Filho</span>, <span class="hlFld-ContribAuthor ">Cássio Santana  Meira</span>, <span class="hlFld-ContribAuthor ">Milena Botelho  Pereira Soares</span>, <span class="hlFld-ContribAuthor ">Darízy Flávia  Silva</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of intracellular Ca2+ mobilization and potassium channels activation are involved in the vasorelaxation induced by 7-hydroxycoumarin. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2020,</strong> <em>887 </em>, 173525. <a href="https://doi.org/10.1016/j.ejphar.2020.173525" title="DOI URL">https://doi.org/10.1016/j.ejphar.2020.173525</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2020.173525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2020.173525%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DInhibition%252Bof%252Bintracellular%252BCa2%25252B%252Bmobilization%252Band%252Bpotassium%252Bchannels%252Bactivation%252Bare%252Binvolved%252Bin%252Bthe%252Bvasorelaxation%252Binduced%252Bby%252B7-hydroxycoumarin%26aulast%3DAlves%26aufirst%3DQuiara%2BLovatti%26date%3D2020%26volume%3D887%26spage%3D173525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Claas  Strecker</span>, <span class="hlFld-ContribAuthor ">Melissa  Baerenfaenger</span>, <span class="hlFld-ContribAuthor ">Michaela  Miehe</span>, <span class="hlFld-ContribAuthor ">Edzard  Spillner</span>, <span class="hlFld-ContribAuthor ">Bernd  Meyer</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Evaluation of the Binding Site of Fucosyltransferase 8 and First Attempts to Synthesize an Inhibitor with Drug‐Like Properties. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2020,</strong> <em>21 </em>
                                    (13)
                                     , 1923-1931. <a href="https://doi.org/10.1002/cbic.201900289" title="DOI URL">https://doi.org/10.1002/cbic.201900289</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201900289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201900289%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DIn%252BSilico%252BEvaluation%252Bof%252Bthe%252BBinding%252BSite%252Bof%252BFucosyltransferase%2525E2%252580%2525858%252Band%252BFirst%252BAttempts%252Bto%252BSynthesize%252Ban%252BInhibitor%252Bwith%252BDrug%2525E2%252580%252590Like%252BProperties%26aulast%3DStrecker%26aufirst%3DClaas%26date%3D2020%26date%3D2019%26volume%3D21%26issue%3D13%26spage%3D1923%26epage%3D1931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Clement  Agoni</span>, <span class="hlFld-ContribAuthor ">Fisayo A.  Olotu</span>, <span class="hlFld-ContribAuthor ">Pritika  Ramharack</span>, <span class="hlFld-ContribAuthor ">Mahmoud E.  Soliman</span>. </span><span class="cited-content_cbyCitation_article-title">Druggability and drug-likeness concepts in drug design: are biomodelling and predictive tools having their say?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2020,</strong> <em>26 </em>
                                    (6)
                                     <a href="https://doi.org/10.1007/s00894-020-04385-6" title="DOI URL">https://doi.org/10.1007/s00894-020-04385-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-020-04385-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-020-04385-6%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DDruggability%252Band%252Bdrug-likeness%252Bconcepts%252Bin%252Bdrug%252Bdesign%25253A%252Bare%252Bbiomodelling%252Band%252Bpredictive%252Btools%252Bhaving%252Btheir%252Bsay%25253F%26aulast%3DAgoni%26aufirst%3DClement%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Filippo  Baldacci</span>, <span class="hlFld-ContribAuthor ">Sonia  Mazzucchi</span>, <span class="hlFld-ContribAuthor ">Alessandra  Della Vecchia</span>, <span class="hlFld-ContribAuthor ">Linda  Giampietri</span>, <span class="hlFld-ContribAuthor ">Nicola  Giannini</span>, <span class="hlFld-ContribAuthor ">Maya  Koronyo-Hamaoui</span>, <span class="hlFld-ContribAuthor ">Roberto  Ceravolo</span>, <span class="hlFld-ContribAuthor ">Gabriele  Siciliano</span>, <span class="hlFld-ContribAuthor ">Ubaldo  Bonuccelli</span>, <span class="hlFld-ContribAuthor ">Fanny M.  Elahi</span>, <span class="hlFld-ContribAuthor ">Andrea  Vergallo</span>, <span class="hlFld-ContribAuthor ">Simone  Lista</span>, <span class="hlFld-ContribAuthor ">Filippo Sean  Giorgi</span>, <span class="hlFld-ContribAuthor ">Harald  Hampel</span>. </span><span class="cited-content_cbyCitation_article-title">The path to biomarker-based diagnostic criteria for the spectrum of neurodegenerative diseases. </span><span class="cited-content_cbyCitation_journal-name">Expert Review of Molecular Diagnostics</span><span> <strong>2020,</strong> <em>20 </em>
                                    (4)
                                     , 421-441. <a href="https://doi.org/10.1080/14737159.2020.1731306" title="DOI URL">https://doi.org/10.1080/14737159.2020.1731306</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14737159.2020.1731306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14737159.2020.1731306%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Review%2520of%2520Molecular%2520Diagnostics%26atitle%3DThe%252Bpath%252Bto%252Bbiomarker-based%252Bdiagnostic%252Bcriteria%252Bfor%252Bthe%252Bspectrum%252Bof%252Bneurodegenerative%252Bdiseases%26aulast%3DBaldacci%26aufirst%3DFilippo%26date%3D2020%26date%3D2020%26volume%3D20%26issue%3D4%26spage%3D421%26epage%3D441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katarzyna  Kent</span>, <span class="hlFld-ContribAuthor ">Madelaine  Johnston</span>, <span class="hlFld-ContribAuthor ">Natasha  Strump</span>, <span class="hlFld-ContribAuthor ">Thomas X.  Garcia</span>. </span><span class="cited-content_cbyCitation_article-title">Toward Development of the Male Pill: A Decade of Potential Non-hormonal Contraceptive Targets. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fcell.2020.00061" title="DOI URL">https://doi.org/10.3389/fcell.2020.00061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2020.00061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2020.00061%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DToward%252BDevelopment%252Bof%252Bthe%252BMale%252BPill%25253A%252BA%252BDecade%252Bof%252BPotential%252BNon-hormonal%252BContraceptive%252BTargets%26aulast%3DKent%26aufirst%3DKatarzyna%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhuyezi  Sun</span>, <span class="hlFld-ContribAuthor ">Amanda Elizabeth  Wakefield</span>, <span class="hlFld-ContribAuthor ">Istvan  Kolossvary</span>, <span class="hlFld-ContribAuthor ">Dmitri  Beglov</span>, <span class="hlFld-ContribAuthor ">Sandor  Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Analysis of Cryptic-Site Opening. </span><span class="cited-content_cbyCitation_journal-name">Structure</span><span> <strong>2020,</strong> <em>28 </em>
                                    (2)
                                     , 223-235.e2. <a href="https://doi.org/10.1016/j.str.2019.11.007" title="DOI URL">https://doi.org/10.1016/j.str.2019.11.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.str.2019.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.str.2019.11.007%26sid%3Dliteratum%253Aachs%26jtitle%3DStructure%26atitle%3DStructure-Based%252BAnalysis%252Bof%252BCryptic-Site%252BOpening%26aulast%3DSun%26aufirst%3DZhuyezi%26date%3D2020%26volume%3D28%26issue%3D2%26spage%3D223%26epage%3D235.e2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amanda E.  Wakefield</span>, <span class="hlFld-ContribAuthor ">Jonathan S.  Mason</span>, <span class="hlFld-ContribAuthor ">Sandor  Vajda</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Analysis of tractable allosteric sites in G protein-coupled receptors. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2019,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-019-42618-8" title="DOI URL">https://doi.org/10.1038/s41598-019-42618-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-019-42618-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-019-42618-8%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DAnalysis%252Bof%252Btractable%252Ballosteric%252Bsites%252Bin%252BG%252Bprotein-coupled%252Breceptors%26aulast%3DWakefield%26aufirst%3DAmanda%2BE.%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thiago Castanheira  Merigueti</span>, <span class="hlFld-ContribAuthor ">Marcia Weber  Carneiro</span>, <span class="hlFld-ContribAuthor ">Ana Paula D’A.  Carvalho-Assef</span>, <span class="hlFld-ContribAuthor ">Floriano Paes  Silva-Jr</span>, <span class="hlFld-ContribAuthor ">Fabricio Alves Barbosa da  Silva</span>. </span><span class="cited-content_cbyCitation_article-title">FindTargetsWEB: A User-Friendly Tool for Identification of Potential Therapeutic Targets in Metabolic Networks of Bacteria. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Genetics</span><span> <strong>2019,</strong> <em>10 </em><a href="https://doi.org/10.3389/fgene.2019.00633" title="DOI URL">https://doi.org/10.3389/fgene.2019.00633</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fgene.2019.00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffgene.2019.00633%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Genetics%26atitle%3DFindTargetsWEB%25253A%252BA%252BUser-Friendly%252BTool%252Bfor%252BIdentification%252Bof%252BPotential%252BTherapeutic%252BTargets%252Bin%252BMetabolic%252BNetworks%252Bof%252BBacteria%26aulast%3DMerigueti%26aufirst%3DThiago%2BCastanheira%26date%3D2019%26date%3D2019%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shoshana J.  Wodak</span>, <span class="hlFld-ContribAuthor ">Emanuele  Paci</span>, <span class="hlFld-ContribAuthor ">Nikolay V.  Dokholyan</span>, <span class="hlFld-ContribAuthor ">Igor N.  Berezovsky</span>, <span class="hlFld-ContribAuthor ">Amnon  Horovitz</span>, <span class="hlFld-ContribAuthor ">Jing  Li</span>, <span class="hlFld-ContribAuthor ">Vincent J.  Hilser</span>, <span class="hlFld-ContribAuthor ">Ivet  Bahar</span>, <span class="hlFld-ContribAuthor ">John  Karanicolas</span>, <span class="hlFld-ContribAuthor ">Gerhard  Stock</span>, <span class="hlFld-ContribAuthor ">Peter  Hamm</span>, <span class="hlFld-ContribAuthor ">Roland H.  Stote</span>, <span class="hlFld-ContribAuthor ">Jerome  Eberhardt</span>, <span class="hlFld-ContribAuthor ">Yassmine  Chebaro</span>, <span class="hlFld-ContribAuthor ">Annick  Dejaegere</span>, <span class="hlFld-ContribAuthor ">Marco  Cecchini</span>, <span class="hlFld-ContribAuthor ">Jean-Pierre  Changeux</span>, <span class="hlFld-ContribAuthor ">Peter G.  Bolhuis</span>, <span class="hlFld-ContribAuthor ">Jocelyne  Vreede</span>, <span class="hlFld-ContribAuthor ">Pietro  Faccioli</span>, <span class="hlFld-ContribAuthor ">Simone  Orioli</span>, <span class="hlFld-ContribAuthor ">Riccardo  Ravasio</span>, <span class="hlFld-ContribAuthor ">Le  Yan</span>, <span class="hlFld-ContribAuthor ">Carolina  Brito</span>, <span class="hlFld-ContribAuthor ">Matthieu  Wyart</span>, <span class="hlFld-ContribAuthor ">Paraskevi  Gkeka</span>, <span class="hlFld-ContribAuthor ">Ivan  Rivalta</span>, <span class="hlFld-ContribAuthor ">Giulia  Palermo</span>, <span class="hlFld-ContribAuthor ">J. Andrew  McCammon</span>, <span class="hlFld-ContribAuthor ">Joanna  Panecka-Hofman</span>, <span class="hlFld-ContribAuthor ">Rebecca C.  Wade</span>, <span class="hlFld-ContribAuthor ">Antonella  Di Pizio</span>, <span class="hlFld-ContribAuthor ">Masha Y.  Niv</span>, <span class="hlFld-ContribAuthor ">Ruth  Nussinov</span>, <span class="hlFld-ContribAuthor ">Chung-Jung  Tsai</span>, <span class="hlFld-ContribAuthor ">Hyunbum  Jang</span>, <span class="hlFld-ContribAuthor ">Dzmitry  Padhorny</span>, <span class="hlFld-ContribAuthor ">Dima  Kozakov</span>, <span class="hlFld-ContribAuthor ">Tom  McLeish</span>. </span><span class="cited-content_cbyCitation_article-title">Allostery in Its Many Disguises: From Theory to Applications. </span><span class="cited-content_cbyCitation_journal-name">Structure</span><span> <strong>2019,</strong> <em>27 </em>
                                    (4)
                                     , 566-578. <a href="https://doi.org/10.1016/j.str.2019.01.003" title="DOI URL">https://doi.org/10.1016/j.str.2019.01.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.str.2019.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.str.2019.01.003%26sid%3Dliteratum%253Aachs%26jtitle%3DStructure%26atitle%3DAllostery%252Bin%252BIts%252BMany%252BDisguises%25253A%252BFrom%252BTheory%252Bto%252BApplications%26aulast%3DWodak%26aufirst%3DShoshana%2BJ.%26date%3D2019%26volume%3D27%26issue%3D4%26spage%3D566%26epage%3D578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Harald  Hampel</span>, <span class="hlFld-ContribAuthor ">Edward J.  Goetzl</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Kapogiannis</span>, <span class="hlFld-ContribAuthor ">Simone  Lista</span>, <span class="hlFld-ContribAuthor ">Andrea  Vergallo</span>. </span><span class="cited-content_cbyCitation_article-title">Biomarker-Drug and Liquid Biopsy Co-development for Disease Staging and Targeted Therapy: Cornerstones for Alzheimer’s Precision Medicine and Pharmacology. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2019,</strong> <em>10 </em><a href="https://doi.org/10.3389/fphar.2019.00310" title="DOI URL">https://doi.org/10.3389/fphar.2019.00310</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2019.00310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2019.00310%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DBiomarker-Drug%252Band%252BLiquid%252BBiopsy%252BCo-development%252Bfor%252BDisease%252BStaging%252Band%252BTargeted%252BTherapy%25253A%252BCornerstones%252Bfor%252BAlzheimer%2525E2%252580%252599s%252BPrecision%252BMedicine%252Band%252BPharmacology%26aulast%3DHampel%26aufirst%3DHarald%26date%3D2019%26date%3D2019%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vito Giuseppe  D’Agostino</span>, <span class="hlFld-ContribAuthor ">Denise  Sighel</span>, <span class="hlFld-ContribAuthor ">Chiara  Zucal</span>, <span class="hlFld-ContribAuthor ">Isabelle  Bonomo</span>, <span class="hlFld-ContribAuthor ">Mariachiara  Micaelli</span>, <span class="hlFld-ContribAuthor ">Graziano  Lolli</span>, <span class="hlFld-ContribAuthor ">Alessandro  Provenzani</span>, <span class="hlFld-ContribAuthor ">Alessandro  Quattrone</span>, <span class="hlFld-ContribAuthor ">Valentina  Adami</span>. </span><span class="cited-content_cbyCitation_article-title">Screening Approaches for Targeting Ribonucleoprotein Complexes: A New Dimension for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2019,</strong> <em>24 </em>
                                    (3)
                                     , 314-331. <a href="https://doi.org/10.1177/2472555218818065" title="DOI URL">https://doi.org/10.1177/2472555218818065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555218818065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555218818065%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DScreening%252BApproaches%252Bfor%252BTargeting%252BRibonucleoprotein%252BComplexes%25253A%252BA%252BNew%252BDimension%252Bfor%252BDrug%252BDiscovery%26aulast%3DD%25E2%2580%2599Agostino%26aufirst%3DVito%2BGiuseppe%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D3%26spage%3D314%26epage%3D331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-François  Fournier</span>, <span class="hlFld-ContribAuthor ">Yushma  Bhurruth-Alcor</span>, <span class="hlFld-ContribAuthor ">Branislav  Musicki</span>, <span class="hlFld-ContribAuthor ">Jérome  Aubert</span>, <span class="hlFld-ContribAuthor ">Michèle  Aurelly</span>, <span class="hlFld-ContribAuthor ">Claire  Bouix-Peter</span>, <span class="hlFld-ContribAuthor ">Karinne  Bouquet</span>, <span class="hlFld-ContribAuthor ">Laurent  Chantalat</span>, <span class="hlFld-ContribAuthor ">Marion  Delorme</span>, <span class="hlFld-ContribAuthor ">Bénédicte  Drean</span>, <span class="hlFld-ContribAuthor ">Gwenaelle  Duvert</span>, <span class="hlFld-ContribAuthor ">Nicolas  Fleury-Bregeot</span>, <span class="hlFld-ContribAuthor ">Blanche  Gauthier</span>, <span class="hlFld-ContribAuthor ">Karine  Grisendi</span>, <span class="hlFld-ContribAuthor ">Craig S.  Harris</span>, <span class="hlFld-ContribAuthor ">Laurent F.  Hennequin</span>, <span class="hlFld-ContribAuthor ">Tatiana  Isabet</span>, <span class="hlFld-ContribAuthor ">Florence  Joly</span>, <span class="hlFld-ContribAuthor ">Guillaume  Lafitte</span>, <span class="hlFld-ContribAuthor ">Corinne  Millois</span>, <span class="hlFld-ContribAuthor ">Rémy  Morgentin</span>, <span class="hlFld-ContribAuthor ">Jonathan  Pascau</span>, <span class="hlFld-ContribAuthor ">David  Piwnica</span>, <span class="hlFld-ContribAuthor ">Yves  Rival</span>, <span class="hlFld-ContribAuthor ">Catherine  Soulet</span>, <span class="hlFld-ContribAuthor ">Étienne  Thoreau</span>, <span class="hlFld-ContribAuthor ">Loïc  Tomas</span>. </span><span class="cited-content_cbyCitation_article-title">Squaramides as novel class I and IIB histone deacetylase inhibitors for topical treatment of cutaneous t-cell lymphoma. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (17)
                                     , 2985-2992. <a href="https://doi.org/10.1016/j.bmcl.2018.06.029" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.06.029</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.06.029%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSquaramides%252Bas%252Bnovel%252Bclass%252BI%252Band%252BIIB%252Bhistone%252Bdeacetylase%252Binhibitors%252Bfor%252Btopical%252Btreatment%252Bof%252Bcutaneous%252Bt-cell%252Blymphoma%26aulast%3DFournier%26aufirst%3DJean-Fran%25C3%25A7ois%26date%3D2018%26volume%3D28%26issue%3D17%26spage%3D2985%26epage%3D2992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandor  Vajda</span>, <span class="hlFld-ContribAuthor ">Dmitri  Beglov</span>, <span class="hlFld-ContribAuthor ">Amanda E  Wakefield</span>, <span class="hlFld-ContribAuthor ">Megan  Egbert</span>, <span class="hlFld-ContribAuthor ">Adrian  Whitty</span>. </span><span class="cited-content_cbyCitation_article-title">Cryptic binding sites on proteins: definition, detection, and druggability. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>44 </em>, 1-8. <a href="https://doi.org/10.1016/j.cbpa.2018.05.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.05.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.05.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DCryptic%252Bbinding%252Bsites%252Bon%252Bproteins%25253A%252Bdefinition%25252C%252Bdetection%25252C%252Band%252Bdruggability%26aulast%3DVajda%26aufirst%3DSandor%26date%3D2018%26volume%3D44%26spage%3D1%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sinisa  Vukovic</span>, <span class="hlFld-ContribAuthor ">David J.  Huggins</span>. </span><span class="cited-content_cbyCitation_article-title">Quantitative metrics for drug–target ligandability. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2018,</strong> <em>23 </em>
                                    (6)
                                     , 1258-1266. <a href="https://doi.org/10.1016/j.drudis.2018.02.015" title="DOI URL">https://doi.org/10.1016/j.drudis.2018.02.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2018.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2018.02.015%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DQuantitative%252Bmetrics%252Bfor%252Bdrug%2525E2%252580%252593target%252Bligandability%26aulast%3DVukovic%26aufirst%3DSinisa%26date%3D2018%26volume%3D23%26issue%3D6%26spage%3D1258%26epage%3D1266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dmitri  Beglov</span>, <span class="hlFld-ContribAuthor ">David R.  Hall</span>, <span class="hlFld-ContribAuthor ">Amanda E.  Wakefield</span>, <span class="hlFld-ContribAuthor ">Lingqi  Luo</span>, <span class="hlFld-ContribAuthor ">Karen N.  Allen</span>, <span class="hlFld-ContribAuthor ">Dima  Kozakov</span>, <span class="hlFld-ContribAuthor ">Adrian  Whitty</span>, <span class="hlFld-ContribAuthor ">Sandor  Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the structural origins of cryptic sites on proteins. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2018,</strong> <em>115 </em>
                                    (15)
                                     , E3416-E3425. <a href="https://doi.org/10.1073/pnas.1711490115" title="DOI URL">https://doi.org/10.1073/pnas.1711490115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1711490115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1711490115%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DExploring%252Bthe%252Bstructural%252Borigins%252Bof%252Bcryptic%252Bsites%252Bon%252Bproteins%26aulast%3DBeglov%26aufirst%3DDmitri%26date%3D2018%26date%3D2018%26volume%3D115%26issue%3D15%26spage%3DE3416%26epage%3DE3425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Philip E.  Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Progress with covalent small-molecule kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2018,</strong> <em>23 </em>
                                    (3)
                                     , 727-735. <a href="https://doi.org/10.1016/j.drudis.2018.01.035" title="DOI URL">https://doi.org/10.1016/j.drudis.2018.01.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2018.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2018.01.035%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DProgress%252Bwith%252Bcovalent%252Bsmall-molecule%252Bkinase%252Binhibitors%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2018%26volume%3D23%26issue%3D3%26spage%3D727%26epage%3D735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabricio Alves Barbosa  da Silva</span>, <span class="hlFld-ContribAuthor ">Fernando Medeiros  Filho</span>, <span class="hlFld-ContribAuthor ">Thiago Castanheira  Merigueti</span>, <span class="hlFld-ContribAuthor ">Thiago  Giannini</span>, <span class="hlFld-ContribAuthor ">Rafaela  Brum</span>, <span class="hlFld-ContribAuthor ">Laura Machado  de Faria</span>, <span class="hlFld-ContribAuthor ">Ana Paula Barbosa  do Nascimento</span>, <span class="hlFld-ContribAuthor ">Kele Teixeira  Belloze</span>, <span class="hlFld-ContribAuthor ">Floriano Paes  Silva</span>, <span class="hlFld-ContribAuthor ">Rodolpho Mattos  Albano</span>, <span class="hlFld-ContribAuthor ">Marcelo Trindade  dos Santos</span>, <span class="hlFld-ContribAuthor ">Maria Clicia Stelling  de Castro</span>, <span class="hlFld-ContribAuthor ">Marcio Argollo  de Menezes</span>, <span class="hlFld-ContribAuthor ">Ana Paula D’A.  Carvalho-Assef</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Modeling of Multidrug-Resistant Bacteria. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 195-220. <a href="https://doi.org/10.1007/978-3-319-74974-7_11" title="DOI URL">https://doi.org/10.1007/978-3-319-74974-7_11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-74974-7_11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-74974-7_11%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252BModeling%252Bof%252BMultidrug-Resistant%252BBacteria%26aulast%3Dda%2BSilva%26aufirst%3DFabricio%2BAlves%2BBarbosa%26date%3D2018%26date%3D2018%26spage%3D195%26epage%3D220%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DTheoretical%252Band%252BApplied%252BAspects%252Bof%252BSystems%252BBiology%26aulast%3DAlves%2BBarbosa%2Bda%2BSilva%26aufirst%3DFabricio%26date%3D2018%26volume%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohd.  Athar</span>, <span class="hlFld-ContribAuthor ">Mohsin Y.  Lone</span>, <span class="hlFld-ContribAuthor ">Prakash C.  Jha</span>. </span><span class="cited-content_cbyCitation_article-title">First protein drug target’s appraisal of lead-likeness descriptors to unfold the intervening chemical space. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2017,</strong> <em>72 </em>, 272-282. <a href="https://doi.org/10.1016/j.jmgm.2016.12.019" title="DOI URL">https://doi.org/10.1016/j.jmgm.2016.12.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2016.12.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2016.12.019%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DFirst%252Bprotein%252Bdrug%252Btarget%2525E2%252580%252599s%252Bappraisal%252Bof%252Blead-likeness%252Bdescriptors%252Bto%252Bunfold%252Bthe%252Bintervening%252Bchemical%252Bspace%26aulast%3DAthar%26aufirst%3DMohd.%26date%3D2017%26volume%3D72%26spage%3D272%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hiba  Abi Hussein</span>, <span class="hlFld-ContribAuthor ">Colette  Geneix</span>, <span class="hlFld-ContribAuthor ">Michel  Petitjean</span>, <span class="hlFld-ContribAuthor ">Alexandre  Borrel</span>, <span class="hlFld-ContribAuthor ">Delphine  Flatters</span>, <span class="hlFld-ContribAuthor ">Anne-Claude  Camproux</span>. </span><span class="cited-content_cbyCitation_article-title">Global vision of druggability issues: applications and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2017,</strong> <em>22 </em>
                                    (2)
                                     , 404-415. <a href="https://doi.org/10.1016/j.drudis.2016.11.021" title="DOI URL">https://doi.org/10.1016/j.drudis.2016.11.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2016.11.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2016.11.021%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DGlobal%252Bvision%252Bof%252Bdruggability%252Bissues%25253A%252Bapplications%252Band%252Bperspectives%26aulast%3DAbi%2BHussein%26aufirst%3DHiba%26date%3D2017%26volume%3D22%26issue%3D2%26spage%3D404%26epage%3D415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan M.  Miller</span>, <span class="hlFld-ContribAuthor ">Walter H.  Moos</span>, <span class="hlFld-ContribAuthor ">Barbara H.  Munk</span>, <span class="hlFld-ContribAuthor ">Stephen A.  Munk</span>. </span><span class="cited-content_cbyCitation_article-title">Backgrounder. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 3-29. <a href="https://doi.org/10.1016/B978-0-08-100625-2.00001-5" title="DOI URL">https://doi.org/10.1016/B978-0-08-100625-2.00001-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-08-100625-2.00001-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-08-100625-2.00001-5%26sid%3Dliteratum%253Aachs%26atitle%3DBackgrounder%26aulast%3DMiller%26aufirst%3DSusan%2BM.%26date%3D2017%26spage%3D3%26epage%3D29%26pub%3DElsevier%26atitle%3DManaging%252Bthe%252BDrug%252BDiscovery%252BProcess%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan M.  Miller</span>, <span class="hlFld-ContribAuthor ">Walter H.  Moos</span>, <span class="hlFld-ContribAuthor ">Barbara H.  Munk</span>, <span class="hlFld-ContribAuthor ">Stephen A.  Munk</span>. </span><span class="cited-content_cbyCitation_article-title">Drug discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 281-420. <a href="https://doi.org/10.1016/B978-0-08-100625-2.00011-8" title="DOI URL">https://doi.org/10.1016/B978-0-08-100625-2.00011-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-08-100625-2.00011-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-08-100625-2.00011-8%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252Bdiscovery%26aulast%3DMiller%26aufirst%3DSusan%2BM.%26date%3D2017%26spage%3D281%26epage%3D420%26pub%3DElsevier%26atitle%3DManaging%252Bthe%252BDrug%252BDiscovery%252BProcess%26date%3D2017%26volume%3D301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A.R.  Leach</span>, <span class="hlFld-ContribAuthor ">P.J.  Thomas</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Structure Prediction and Homology Modeling. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 120-144. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12350-9" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12350-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12350-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12350-9%26sid%3Dliteratum%253Aachs%26atitle%3DProtein%252BStructure%252BPrediction%252Band%252BHomology%252BModeling%26aulast%3DLeach%26aufirst%3DA.R.%26date%3D2017%26spage%3D120%26epage%3D144%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adrian  Whitty</span>, <span class="hlFld-ContribAuthor ">Lauren A.  Viarengo</span>, <span class="hlFld-ContribAuthor ">Mengqi  Zhong</span>. </span><span class="cited-content_cbyCitation_article-title">Progress towards the broad use of non-peptide synthetic macrocycles in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2017,</strong> <em>15 </em>
                                    (37)
                                     , 7729-7735. <a href="https://doi.org/10.1039/C7OB00056A" title="DOI URL">https://doi.org/10.1039/C7OB00056A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7OB00056A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7OB00056A%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DProgress%252Btowards%252Bthe%252Bbroad%252Buse%252Bof%252Bnon-peptide%252Bsynthetic%252Bmacrocycles%252Bin%252Bdrug%252Bdiscovery%26aulast%3DWhitty%26aufirst%3DAdrian%26date%3D2017%26date%3D2017%26volume%3D15%26issue%3D37%26spage%3D7729%26epage%3D7735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sinisa  Vukovic</span>, <span class="hlFld-ContribAuthor ">Paul E  Brennan</span>, <span class="hlFld-ContribAuthor ">David J  Huggins</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the role of water in molecular recognition: predicting protein ligandability using a combinatorial search of surface hydration sites. </span><span class="cited-content_cbyCitation_journal-name">Journal of Physics: Condensed Matter</span><span> <strong>2016,</strong> <em>28 </em>
                                    (34)
                                     , 344007. <a href="https://doi.org/10.1088/0953-8984/28/34/344007" title="DOI URL">https://doi.org/10.1088/0953-8984/28/34/344007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1088/0953-8984/28/34/344007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1088%2F0953-8984%2F28%2F34%2F344007%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Physics%253A%2520Condensed%2520Matter%26atitle%3DExploring%252Bthe%252Brole%252Bof%252Bwater%252Bin%252Bmolecular%252Brecognition%25253A%252Bpredicting%252Bprotein%252Bligandability%252Busing%252Ba%252Bcombinatorial%252Bsearch%252Bof%252Bsurface%252Bhydration%252Bsites%26aulast%3DVukovic%26aufirst%3DSinisa%26date%3D2016%26date%3D2016%26volume%3D28%26issue%3D34%26spage%3D344007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppe  Mangiatordi</span>, <span class="hlFld-ContribAuthor ">Domenico  Alberga</span>, <span class="hlFld-ContribAuthor ">Daniela  Trisciuzzi</span>, <span class="hlFld-ContribAuthor ">Gianluca  Lattanzi</span>, <span class="hlFld-ContribAuthor ">Orazio  Nicolotti</span>. </span><span class="cited-content_cbyCitation_article-title">Human Aquaporin-4 and Molecular Modeling: Historical Perspective and View to the Future. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2016,</strong> <em>17 </em>
                                    (7)
                                     , 1119. <a href="https://doi.org/10.3390/ijms17071119" title="DOI URL">https://doi.org/10.3390/ijms17071119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms17071119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms17071119%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DHuman%252BAquaporin-4%252Band%252BMolecular%252BModeling%25253A%252BHistorical%252BPerspective%252Band%252BView%252Bto%252Bthe%252BFuture%26aulast%3DMangiatordi%26aufirst%3DGiuseppe%26date%3D2016%26date%3D2016%26volume%3D17%26issue%3D7%26spage%3D1119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul D.  Leeson</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular inflation, attrition and the rule of five. </span><span class="cited-content_cbyCitation_journal-name">Advanced Drug Delivery Reviews</span><span> <strong>2016,</strong> <em>101 </em>, 22-33. <a href="https://doi.org/10.1016/j.addr.2016.01.018" title="DOI URL">https://doi.org/10.1016/j.addr.2016.01.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.addr.2016.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.addr.2016.01.018%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Drug%2520Delivery%2520Reviews%26atitle%3DMolecular%252Binflation%25252C%252Battrition%252Band%252Bthe%252Brule%252Bof%252Bfive%26aulast%3DLeeson%26aufirst%3DPaul%2BD.%26date%3D2016%26volume%3D101%26spage%3D22%26epage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefanie D.  Krämer</span>, <span class="hlFld-ContribAuthor ">Hélène E.  Aschmann</span>, <span class="hlFld-ContribAuthor ">Maja  Hatibovic</span>, <span class="hlFld-ContribAuthor ">Katharina F.  Hermann</span>, <span class="hlFld-ContribAuthor ">Claudia S.  Neuhaus</span>, <span class="hlFld-ContribAuthor ">Cyrill  Brunner</span>, <span class="hlFld-ContribAuthor ">Sara  Belli</span>. </span><span class="cited-content_cbyCitation_article-title">When barriers ignore the “rule-of-five”. </span><span class="cited-content_cbyCitation_journal-name">Advanced Drug Delivery Reviews</span><span> <strong>2016,</strong> <em>101 </em>, 62-74. <a href="https://doi.org/10.1016/j.addr.2016.02.001" title="DOI URL">https://doi.org/10.1016/j.addr.2016.02.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.addr.2016.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.addr.2016.02.001%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Drug%2520Delivery%2520Reviews%26atitle%3DWhen%252Bbarriers%252Bignore%252Bthe%252B%2525E2%252580%25259Crule-of-five%2525E2%252580%25259D%26aulast%3DKr%25C3%25A4mer%26aufirst%3DStefanie%2BD.%26date%3D2016%26volume%3D101%26spage%3D62%26epage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christel A.S.  Bergström</span>, <span class="hlFld-ContribAuthor ">William N.  Charman</span>, <span class="hlFld-ContribAuthor ">Christopher J.H.  Porter</span>. </span><span class="cited-content_cbyCitation_article-title">Computational prediction of formulation strategies for beyond-rule-of-5 compounds. </span><span class="cited-content_cbyCitation_journal-name">Advanced Drug Delivery Reviews</span><span> <strong>2016,</strong> <em>101 </em>, 6-21. <a href="https://doi.org/10.1016/j.addr.2016.02.005" title="DOI URL">https://doi.org/10.1016/j.addr.2016.02.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.addr.2016.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.addr.2016.02.005%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Drug%2520Delivery%2520Reviews%26atitle%3DComputational%252Bprediction%252Bof%252Bformulation%252Bstrategies%252Bfor%252Bbeyond-rule-of-5%252Bcompounds%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DChristel%2BA.S.%26date%3D2016%26volume%3D101%26spage%3D6%26epage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dulal  Panda</span>, <span class="hlFld-ContribAuthor ">Dipanwita  Bhattacharya</span>, <span class="hlFld-ContribAuthor ">Quanqing Helen  Gao</span>, <span class="hlFld-ContribAuthor ">Pratik M  Oza</span>, <span class="hlFld-ContribAuthor ">H-Y Jennifer  Lin</span>, <span class="hlFld-ContribAuthor ">Bryson  Hawkins</span>, <span class="hlFld-ContribAuthor ">David E  Hibbs</span>, <span class="hlFld-ContribAuthor ">Paul W  Groundwater</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of agents targeting FtsZ assembly. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (10)
                                     , 1111-1132. <a href="https://doi.org/10.4155/fmc-2016-0041" title="DOI URL">https://doi.org/10.4155/fmc-2016-0041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0041%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bagents%252Btargeting%252BFtsZ%252Bassembly%26aulast%3DPanda%26aufirst%3DDulal%26date%3D2016%26volume%3D8%26issue%3D10%26spage%3D1111%26epage%3D1132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Artem B.  Mamonov</span>, <span class="hlFld-ContribAuthor ">Mohammad  Moghadasi</span>, <span class="hlFld-ContribAuthor ">Hanieh  Mirzaei</span>, <span class="hlFld-ContribAuthor ">Shahrooz  Zarbafian</span>, <span class="hlFld-ContribAuthor ">Laurie E.  Grove</span>, <span class="hlFld-ContribAuthor ">Tanggis  Bohnuud</span>, <span class="hlFld-ContribAuthor ">Pirooz  Vakili</span>, <span class="hlFld-ContribAuthor ">Ioannis  Ch. Paschalidis</span>, <span class="hlFld-ContribAuthor ">Sandor  Vajda</span>, <span class="hlFld-ContribAuthor ">Dima  Kozakov</span>. </span><span class="cited-content_cbyCitation_article-title">Focused grid‐based resampling for protein docking and mapping. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2016,</strong> <em>37 </em>
                                    (11)
                                     , 961-970. <a href="https://doi.org/10.1002/jcc.24273" title="DOI URL">https://doi.org/10.1002/jcc.24273</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.24273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.24273%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DFocused%252Bgrid%2525E2%252580%252590based%252Bresampling%252Bfor%252Bprotein%252Bdocking%252Band%252Bmapping%26aulast%3DMamonov%26aufirst%3DArtem%2BB.%26date%3D2016%26date%3D2016%26volume%3D37%26issue%3D11%26spage%3D961%26epage%3D970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N.K.  Terrett</span>. </span><span class="cited-content_cbyCitation_article-title">Combinatorial Chemistry Online Volume 18, Issue 3, March 2016. </span><span class="cited-content_cbyCitation_journal-name">Combinatorial Chemistry - an Online Journal</span><span> <strong>2016,</strong> <em>18 </em>
                                    (3)
                                     , 9-11. <a href="https://doi.org/10.1016/j.comche.2016.01.001" title="DOI URL">https://doi.org/10.1016/j.comche.2016.01.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.comche.2016.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.comche.2016.01.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCombinatorial%2520Chemistry%2520-%2520an%2520Online%2520Journal%26atitle%3DCombinatorial%252BChemistry%252BOnline%252BVolume%252B18%25252C%252BIssue%252B3%25252C%252BMarch%252B2016%26aulast%3DTerrett%26aufirst%3DN.K.%26date%3D2016%26volume%3D18%26issue%3D3%26spage%3D9%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeffrey E.  Arle</span>, <span class="hlFld-ContribAuthor ">Kristen W.  Carlson</span>. </span><span class="cited-content_cbyCitation_article-title">The use of dynamic computational models of neural circuitry to streamline new drug development. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Disease Models</span><span> <strong>2016,</strong> <em>19 </em>, 69-75. <a href="https://doi.org/10.1016/j.ddmod.2017.01.002" title="DOI URL">https://doi.org/10.1016/j.ddmod.2017.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddmod.2017.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddmod.2017.01.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Disease%2520Models%26atitle%3DThe%252Buse%252Bof%252Bdynamic%252Bcomputational%252Bmodels%252Bof%252Bneural%252Bcircuitry%252Bto%252Bstreamline%252Bnew%252Bdrug%252Bdevelopment%26aulast%3DArle%26aufirst%3DJeffrey%2BE.%26date%3D2016%26volume%3D19%26spage%3D69%26epage%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jingyu  Xu</span>, <span class="hlFld-ContribAuthor ">Panpan  Wang</span>, <span class="hlFld-ContribAuthor ">Hong  Yang</span>, <span class="hlFld-ContribAuthor ">Jin  Zhou</span>, <span class="hlFld-ContribAuthor ">Yinghong  Li</span>, <span class="hlFld-ContribAuthor ">Xiaoxu  Li</span>, <span class="hlFld-ContribAuthor ">Weiwei  Xue</span>, <span class="hlFld-ContribAuthor ">Chunyan  Yu</span>, <span class="hlFld-ContribAuthor ">Yubin  Tian</span>, <span class="hlFld-ContribAuthor ">Feng  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison of FDA Approved Kinase Targets to Clinical Trial Ones: Insights from Their System Profiles and Drug-Target Interaction Networks. </span><span class="cited-content_cbyCitation_journal-name">BioMed Research International</span><span> <strong>2016,</strong> <em>2016 </em>, 1-9. <a href="https://doi.org/10.1155/2016/2509385" title="DOI URL">https://doi.org/10.1155/2016/2509385</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2016/2509385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2016%2F2509385%26sid%3Dliteratum%253Aachs%26jtitle%3DBioMed%2520Research%2520International%26atitle%3DComparison%252Bof%252BFDA%252BApproved%252BKinase%252BTargets%252Bto%252BClinical%252BTrial%252BOnes%25253A%252BInsights%252Bfrom%252BTheir%252BSystem%252BProfiles%252Band%252BDrug-Target%252BInteraction%252BNetworks%26aulast%3DXu%26aufirst%3DJingyu%26date%3D2016%26volume%3D2016%26spage%3D1%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David R.  Hall</span>, <span class="hlFld-ContribAuthor ">Dima  Kozakov</span>, <span class="hlFld-ContribAuthor ">Adrian  Whitty</span>, <span class="hlFld-ContribAuthor ">Sandor  Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Lessons from Hot Spot Analysis for Fragment-Based Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2015,</strong> <em>36 </em>
                                    (11)
                                     , 724-736. <a href="https://doi.org/10.1016/j.tips.2015.08.003" title="DOI URL">https://doi.org/10.1016/j.tips.2015.08.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2015.08.003%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DLessons%252Bfrom%252BHot%252BSpot%252BAnalysis%252Bfor%252BFragment-Based%252BDrug%252BDiscovery%26aulast%3DHall%26aufirst%3DDavid%2BR.%26date%3D2015%26volume%3D36%26issue%3D11%26spage%3D724%26epage%3D736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Using hot spot properties to assess druggability. (A) Structure of FK506, considered in the example of druggability analysis by FTMap. (B) Structure of ABT737, a well-known example of protein–protein interaction inhibitors. (C) Mapping the ligand-free FK506 binding protein (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn">2ppn</a>, shown as surface), with bound FK506 (green sticks, from the structure with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj">1fkj</a>) superimposed for reference. The consensus clusters shown are CC1 (cyan, 18 probe clusters), CC3 (yellow, 14 probe clusters), and CC6 (blue, 7 probe clusters). The center-to-center distance (CD) between CC1 and CC3 is 6.3 Å, and the maximum dimension of the ensemble of the two hot spots is 10.6 Å. Thus, the subset of these two hot spots is druggable on its own but would bind small ligands with only moderate potency. Using all three hot spots increases MD to 12.3 Å and allows for high-affinity binding. (D) Distribution of the parameters used for assessing druggability of the 23 protein targets in the Hajduk et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> data set. (E) Top view of the distribution shown in panel D to show the separation of different druggability classes. Note that the non-canonically druggable–large (D*<sub>L</sub>) category is defined by the large (CD ≥ 8 Å) distance between the centers of the hot spots. (F) Right view of the distribution shown in panel D. The non-canonically druggable–small (D*<sub>S</sub>) category is defined by the small (MD < 10 Å) dimension of the entire hot spot ensemble.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Hot spots in two protein–protein interaction targets, belonging to the druggability class D*<sub>L</sub>. (A) Mapping of the ligand free PDZ-1 domain of PSD-95 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu2">1iu2</a>, shown as surface), with a cyclic peptide (from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rgr">1rgr</a>, shown as yellow stick model) superimposed for reference. The two hot spots in the interface, CC1 (cyan, 22 probe clusters) and CC5 (white, 11 probe clusters), are 9.8 Å from each other. (B) Mapping of the ligand-free survivin dimer (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31">1e31</a>, shown as cyan and green cartoon), with the Smac/DIABLO peptide (from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uih">3uih</a>, shown as yellow cartoon) superimposed for reference. The only hot spot in the survivin/Smac interface is CC7 (orange, 5 probe clusters); thus, the peptide binding site is not druggable. A novel site is defined by the strong hot spots CC1 (cyan, 24 probe clusters) and CC2 (magenta, also 24 probe clusters) at the dimerization interface. Since the center-to-center distance between CC1 and CC2 is 8.3 Å, ligands that bind to both are over the 500 g/mol MW limit.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Druggability assessment for important targets. (A) Structure of the weak inhibitor <b>1</b> of the ZipA/FtsZ protein–protein interaction. (B) Structure of the PTP1B inhibitor <b>2</b>. (C) Mapping of the ligand free ZipA protein (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f46">1f46</a>), with inhibitor <b>1</b> (from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1s1s">1s1s</a>) superimposed for reference. The consensus clusters shown are CC8 (green, 6 probe clusters), CC9 (blue, 5 clusters), and CC10 (orange, 3 clusters). We also show the strongest hot spot CC1 (cyan, 22 clusters), which is located outside the ZipA/FtsZ interface. CC4 (salmon, 9 clusters) is also too far from the interface. (D) Mapping of the apurinic/apyrimidinic endonuclease 1 (APE1) structure from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1de9">1de9</a>, cocrystallized with abasic DNA (fragment of the DNA is shown as blue sticks for reference), and an Mn<sup>2+</sup> ion (cyan sphere). As always, the DNA and the ion were removed prior to the mapping. The hot spots shown are CC1 (cyan, 16 clusters), CC2 (magenta, 16 clusters), and CC3 (yellow, 13 clusters). (E) Mapping of the ligand-free protein tyrosine phosphatase 1B (PTP1B, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j">3a5j</a>), with two bound phosphotyrosine (pTyr) molecules superimposed for reference (from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty">1pty</a>). The hot spots shown are CC1 (cyan, 20 clusters) at the catalytic PTyr site, CC4 (salmon, 9 clusters), CC7 (orange, 7 clusters), and CC8 (green, 7 clusters) close to the site that binds the noncatalytic pTyr. The hot spots CC2 (magenta, 16 clusters) and CC7 (orange, 7 clusters) are closed in any ligand-bound structure. (F) Same as panel E but with the bound structure of the potent inhibitor <b>2</b> shown for reference from the structure with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1q6t">1q6t</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Druggability assessment for 14-3-3 protein. (A) Structure of the phosphothreonine peptide <b>3</b> that was cocystallyzed with 14-3-3. (B) Structure of fusicoccin (<b>4</b>), a compound that binds to 14-3-3 and stabilizes the binding of <b>3</b>. (C) Structure of inhibitor <b>5</b>. (D) Mapping of the ligand free structure of 14-3-3 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yz5">1yz5</a>). For reference we superimpose the bound compounds <b>3</b> and <b>4</b>, shown as green and yellow sticks, respectively, from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jdd">4jdd</a>. The hot spots shown are CC1 (cyan, 21 clusters) at the deep pocket that binds <b>3</b>, CC2 (magenta, 16 clusters), and CC5 (white, 11 clusters) at the region that binds <b>4</b>. Two more hot spots are close to <b>3</b>: CC4 (salmon, 12 clusters) and CC7 (orange, 5 clusters). (E) Same as panel D but with inhibitor <b>5</b> (magenta).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/medium/jm-2015-005865_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Druggability assessment for KRAS. (A) Structure of <b>6</b>, one of the highest affinity inhibitors binding to KRAS at the interface with SOS. (B) Structure of inhibitor <b>7</b> binding covalently to KRAS(G12C). (C) Mapping of the ligand-free KRAS (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gft">3gft</a>). The cavities with probes bound are the GTP/GDP binding site, with part of CC1 (cyan, 24 clusters) visible, and the targeted hot spot CC2 (magenta, 13 clusters) in Site 2 located in the KRAS/SOS interface. For reference, we superimpose a small ligand, benzamidine (shown as yellow sticks, from the structure with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dso">4dso</a>), binding to Site 2. (D) Mapping of the KRAS structure from its complex with inhibitor <b>6</b> from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epy">4epy</a> (<i>K</i><sub>d</sub> = 340 μM, superimposed as magenta sticks). Due to the binding of <b>6</b>, the strongest hot spot CC1 (cyan, 20 clusters) is now in the KRAS/SOS interface, and the interface includes a second weak hot spot CC8 (green, 7 clusters) located close to CC1. (E) The mapping of the ligand-free KRAS (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gft">3gft</a>) places CC4 (salmon, 9 clusters) at the S3 site, in the pocket binding the KRAS(G12C) inhibitor <b>7</b> that allosterically controls GTP affinity and effector interactions. Inhibitor <b>7</b> from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4lv6">4lv6</a> is superimposed for reference and is shown as green sticks. Notice that in this closed conformation of KRAS the loop 58–65 would clash with <b>7</b> and thus prevents its binding. For reference, the figure also shows CC2 (magenta) and the benzamidine ligand (yellow) on the other side of the protein. (F) Homology model of KRAS based on a HRAS structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4q21">4q21</a>) to model the open state. In this structure, the opening of the 58–65 loop creates a stronger hot spot, CC2 (magenta, 16 clusters). Inhibitor <b>7</b> from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4lv6">4lv6</a> is shown again for reference. Although the site in this open structure binds 16 probe clusters, the maximum dimension of CC2 is only 5.1 Å; thus, the site is not druggable without the covalent interaction with G12C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-23/acs.jmedchem.5b00586/20151204/images/large/jm-2015-005865_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00586&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35047" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35047" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 163 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-30&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0lh2JI-18LNmUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D30%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lh2JI-18LNmUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Russ, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lampel, S.</span><span> </span><span class="NLM_article-title">The druggable genome: An update</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1607</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03666-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS1359-6446%2805%2903666-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16376820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A280%3ADC%252BD2Mnos1KntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=1607-10&author=A.+P.+Russauthor=S.+Lampel&title=The+druggable+genome%3A+An+update&doi=10.1016%2FS1359-6446%2805%2903666-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The druggable genome: an update</span></div><div class="casAuthors">Russ Andreas P; Lampel Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">1607-10</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Annotating the druggable genome estimates the potential maximum size of the playing field for current small-molecule drug design but It does not consider biologicals or future breakthroughs in medicinal chemistry or biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTZBV0v0n_yXIMcDoIzZtNkfW6udTcc2ebYxb8E4cHPoLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2Mnos1KntQ%253D%253D&md5=dc4dbf7d5220b5642cca6152b8c37d09</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903666-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903666-4%26sid%3Dliteratum%253Aachs%26aulast%3DRuss%26aufirst%3DA.%2BP.%26aulast%3DLampel%26aufirst%3DS.%26atitle%3DThe%2520druggable%2520genome%253A%2520An%2520update%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D1607%26epage%3D10%26doi%3D10.1016%2FS1359-6446%2805%2903666-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Johns, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russ, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H. A.</span><span> </span><span class="NLM_article-title">Current drug targets and the druggable genome</span> <span class="citation_source-journal">Chemical Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_fpage">320</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1017/CBO9781139021500.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1017%2FCBO9781139021500.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=320-32&author=M.+A.+Johnsauthor=A.+Russauthor=H.+A.+Fu&title=Current+drug+targets+and+the+druggable+genome&doi=10.1017%2FCBO9781139021500.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1017%2FCBO9781139021500.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FCBO9781139021500.029%26sid%3Dliteratum%253Aachs%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DRuss%26aufirst%3DA.%26aulast%3DFu%26aufirst%3DH.%2BA.%26atitle%3DCurrent%2520drug%2520targets%2520and%2520the%2520druggable%2520genome%26jtitle%3DChemical%2520Genomics%26date%3D2012%26spage%3D320%26epage%3D32%26doi%3D10.1017%2FCBO9781139021500.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiorov, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornell, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Y. D.</span><span> </span><span class="NLM_article-title">Drug-like density: A method of quantifying the ″bindability″ of a protein target based on a very large set of pockets and drug-like ligands from the Protein Data Bank</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2029</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span><span class="refDoi"> DOI: 10.1021/ci100312t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100312t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtleiur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=2029-40&author=R.+P.+Sheridanauthor=V.+N.+Maiorovauthor=M.+K.+Hollowayauthor=W.+D.+Cornellauthor=Y.+D.+Gao&title=Drug-like+density%3A+A+method+of+quantifying+the+%E2%80%B3bindability%E2%80%B3+of+a+protein+target+based+on+a+very+large+set+of+pockets+and+drug-like+ligands+from+the+Protein+Data+Bank&doi=10.1021%2Fci100312t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-like Density: A Method of Quantifying the "Bindability" of a Protein Target Based on a Very Large Set of Pockets and Drug-like Ligands from the Protein Data Bank</span></div><div class="casAuthors">Sheridan, Robert P.; Maiorov, Vladimir N.; Holloway, M. Katharine; Cornell, Wendy D.; Gao, Ying-Duo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2029-2040</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One approach to estg. the "chem. tractability" of a candidate protein target where we know the at. resoln. structure is to examine the phys. properties of potential binding sites.  A no. of other workers have addressed this issue.  We characterize ∼290 000 "pockets" from ∼42 000 protein crystal structures in terms of a three parameter "pocket space": vol., buriedness, and hydrophobicity.  A metric DLID (drug-like d.) measures how likely a pocket is to bind a drug-like mol.  This is calcd. from the count of other pockets in its local neighborhood in pocket space that contain drug-like cocrystd. ligands and the count of total pockets in the neighborhood.  Surprisingly, despite being defined locally, a global trend in DLID can be predicted by a simple linear regression on log(vol.), buriedness, and hydrophobicity.  Two levels of simplification are necessary to relate the DLID of individual pockets to "targets": taking the best DLID per Protein Data Bank (PDB) entry (because any given crystal structure can have many pockets), and taking the median DLID over all PDB entries for the same target (because different crystal structures of the same protein can vary because of artifacts and real conformational changes).  We can show that median DLIDs for targets that are detectably homologous in sequence are reasonably similar and that median DLIDs correlate with the "druggability" est. of Cheng et al. (Nature Biotechnol.2007, 25, 71-75).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY1WHjyA8uarVg90H21EOLACvtfcHk0liEEX-IXDkF0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtleiur3O&md5=c7fef497000a2830323d623c19c107f9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fci100312t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100312t%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DMaiorov%26aufirst%3DV.%2BN.%26aulast%3DHolloway%26aufirst%3DM.%2BK.%26aulast%3DCornell%26aufirst%3DW.%2BD.%26aulast%3DGao%26aufirst%3DY.%2BD.%26atitle%3DDrug-like%2520density%253A%2520A%2520method%2520of%2520quantifying%2520the%2520%25E2%2580%25B3bindability%25E2%2580%25B3%2520of%2520a%2520protein%2520target%2520based%2520on%2520a%2520very%2520large%2520set%2520of%2520pockets%2520and%2520drug-like%2520ligands%2520from%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D2029%26epage%3D40%26doi%3D10.1021%2Fci100312t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span> </span><span class="NLM_article-title">Predicting protein druggability</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1675</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03624-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS1359-6446%2805%2903624-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16376828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCrtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=1675-82&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=C.+Tse&title=Predicting+protein+druggability&doi=10.1016%2FS1359-6446%2805%2903624-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting protein druggability</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Tse, Christin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">23/24</span>),
    <span class="NLM_cas:pages">1675-1682</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  New exptl. and computational approaches promise to rapidly and reliably identify druggable, disease-modifying targets and thereby streamline the drug discovery process.  The ability to predict whether a particular protein can bind with high affinity and specificity to small, drug-like compds. based solely on its 3D structure has been a longstanding goal of structural biologists and computational scientists.  The promise is that an accurate prediction of protein druggability can capitalize on the huge investments already made in structural genomics initiatives by identifying highly druggable proteins and using this information in target identification and validation campaigns.  Here we discuss the potential utility of tools that characterize protein targets and describe strategies for the optimal integration of protein druggability data with bioinformatic approaches to target selection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodx4xzYCqOWLVg90H21EOLACvtfcHk0liEEX-IXDkF0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCrtrnN&md5=217b1b92ddb845506b14bded4fb56549</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903624-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903624-X%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DTse%26aufirst%3DC.%26atitle%3DPredicting%2520protein%2520druggability%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D1675%26epage%3D82%26doi%3D10.1016%2FS1359-6446%2805%2903624-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from NMR-based screening data</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2518</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-25&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+NMR-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0lh_8AQMoR-lUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520NMR-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D25%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Cheng, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smyth, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soulard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caffrey, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzberg, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, E. S.</span><span> </span><span class="NLM_article-title">Structure-based maximal affinity model predicts small-molecule druggability</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1038/nbt1273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnbt1273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17211405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1Grtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=71-5&author=A.+C.+Chengauthor=R.+G.+Colemanauthor=K.+T.+Smythauthor=Q.+Caoauthor=P.+Soulardauthor=D.+R.+Caffreyauthor=A.+C.+Salzbergauthor=E.+S.+Huang&title=Structure-based+maximal+affinity+model+predicts+small-molecule+druggability&doi=10.1038%2Fnbt1273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based maximal affinity model predicts small-molecule druggability</span></div><div class="casAuthors">Cheng, Alan C.; Coleman, Ryan G.; Smyth, Kathleen T.; Cao, Qing; Soulard, Patricia; Caffrey, Daniel R.; Salzberg, Anna C.; Huang, Enoch S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-75</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lead generation is a major hurdle in small-mol. drug discovery, with an estd. 60% of projects failing from lack of lead matter or difficulty in optimizing leads for drug-like properties.  It would be valuable to identify these less-druggable targets before incurring substantial expenditure and effort.  Here we show that a model-based approach using basic biophys. principles yields good prediction of druggability based solely on the crystal structure of the target binding site.  We quant. est. the maximal affinity achievable by a drug-like mol., and we show that these calcd. values correlate with drug discovery outcomes.  We exptl. test two predictions using high-throughput screening of a diverse compd. collection.  The collective results highlight the utility of our approach as well as strategies for tackling difficult targets.  Both assays are simple, suitable for rapid screening of a large no. of medicinal libraries, and amenable for automation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomQJn2NNwCFLVg90H21EOLACvtfcHk0lh_8AQMoR-lUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1Grtg%253D%253D&md5=e971faf87bf4b91602bf592b9296325b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt1273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1273%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DA.%2BC.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSmyth%26aufirst%3DK.%2BT.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DSoulard%26aufirst%3DP.%26aulast%3DCaffrey%26aufirst%3DD.%2BR.%26aulast%3DSalzberg%26aufirst%3DA.%2BC.%26aulast%3DHuang%26aufirst%3DE.%2BS.%26atitle%3DStructure-based%2520maximal%2520affinity%2520model%2520predicts%2520small-molecule%2520druggability%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D71%26epage%3D5%26doi%3D10.1038%2Fnbt1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Seco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability index from first principles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1021/jm801385d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801385d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2363-71&author=J.+Secoauthor=F.+J.+Luqueauthor=X.+Barril&title=Binding+site+detection+and+druggability+index+from+first+principles&doi=10.1021%2Fjm801385d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Detection and Druggability Index from First Principles</span></div><div class="casAuthors">Seco, Jesus; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In drug discovery, it is essential to identify binding sites on protein surfaces that drug-like mols. could exploit to exert a biol. effect.  Both X-ray crystallog. and NMR expts. have demonstrated that org. solvents bind precisely at these locations.  We show that this effect is reproduced using mol. dynamics with a binary solvent.  Furthermore, anal. of the simulations give direct access to interaction free energies between the protein and small org. mols., which can be used to detect binding sites and to predict the maximal affinity that a drug-like mol. could attain for them.  On a set of pharmacol. relevant proteins, we obtain good predictions for druggable sites as well as for protein-protein and low affinity binding sites.  This is the first druggability index not based on surface descriptors and, being independent of a training set, is particularly indicated to study unconventional targets such as protein-protein interactions or allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAgMI9Qer6rVg90H21EOLACvtfcHk0lh_8AQMoR-lUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D&md5=b766cbadeba11b23a1de44b27aa794b4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm801385d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801385d%26sid%3Dliteratum%253Aachs%26aulast%3DSeco%26aufirst%3DJ.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520index%2520from%2520first%2520principles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2363%26epage%3D71%26doi%3D10.1021%2Fjm801385d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-89&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0lh_8AQMoR-lUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D89%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Schmidtke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Understanding and predicting druggability. A high-throughput method for detection of drug binding sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5858</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1021/jm100574m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100574m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ektLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5858-67&author=P.+Schmidtkeauthor=X.+Barril&title=Understanding+and+predicting+druggability.+A+high-throughput+method+for+detection+of+drug+binding+sites&doi=10.1021%2Fjm100574m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding and Predicting Druggability. A High-Throughput Method for Detection of Drug Binding Sites</span></div><div class="casAuthors">Schmidtke, Peter; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5858-5867</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Druggability predictions are important to avoid intractable targets and to focus drug discovery efforts on sites offering better prospects.  However, few druggability prediction tools have been released and none has been extensively tested.  Here, a set of druggable and nondruggable cavities has been compiled in a collaborative platform (http://fpocket.sourceforge.net/dcd) that can be used, contributed, and curated by the community.  Druggable binding sites are often oversimplified as closed, hydrophobic cavities, but data set anal. reveals that polar groups in druggable binding sites have properties that enable them to play a decisive role in ligand recognition.  Finally, the data set has been used in conjunction with the open source fpocket suite to train and validate a logistic model.  State of the art performance was achieved for predicting druggability on known binding sites and on virtual screening expts. where druggable pockets are retrieved from a pool of decoys.  The algorithm is free, extremely fast, and can effectively be used to automatically sieve through massive collections of structures (http://fpocket.sourceforge.net).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos1ixAU42dJrVg90H21EOLACvtfcHk0lh3SDiEvkVXKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ektLo%253D&md5=d9fc5c4d52a6e0f032b8d5c021a6cd89</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm100574m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100574m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidtke%26aufirst%3DP.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DUnderstanding%2520and%2520predicting%2520druggability.%2520A%2520high-throughput%2520method%2520for%2520detection%2520of%2520drug%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5858%26epage%3D67%26doi%3D10.1021%2Fjm100574m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Krasowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muthas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenk, R.</span><span> </span><span class="NLM_article-title">Drugpred: A structure-based approach to predict protein druggability developed using an extensive nonredundant data set</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2829</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span><span class="refDoi"> DOI: 10.1021/ci200266d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200266d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlShtLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2829-42&author=A.+Krasowskiauthor=D.+Muthasauthor=A.+Sarkarauthor=S.+Schmittauthor=R.+Brenk&title=Drugpred%3A+A+structure-based+approach+to+predict+protein+druggability+developed+using+an+extensive+nonredundant+data+set&doi=10.1021%2Fci200266d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DrugPred: A Structure-Based Approach To Predict Protein Druggability Developed Using an Extensive Nonredundant Data Set</span></div><div class="casAuthors">Krasowski, Agata; Muthas, Daniel; Sarkar, Aurijit; Schmitt, Stefan; Brenk, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2829-2842</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Judging if a protein is able to bind orally available mols. with high affinity, i.e. if a protein is druggable, is an important step in target assessment.  In order to derive a structure-based method to predict protein druggability, a comprehensive, nonredundant data set contg. crystal structures of 71 druggable and 44 less druggable proteins was compiled by literature search and data mining.  This data set was subsequently used to train a structure-based druggability predictor (DrugPred) using partial least-squares projection to latent structures discriminant anal. (PLS-DA).  DrugPred performed well in discriminating druggable from less druggable binding sites for both internal and external predictions.  The method is robust against conformational changes in the binding site and outperforms previously published methods.  The superior performance of DrugPred is likely due to the size and compn. of the training set which, in contrast to most previously developed methods, only contains cavities that have evolved to bind a natural ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLISVIZ5UnXrVg90H21EOLACvtfcHk0lh3SDiEvkVXKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlShtLnN&md5=a352186705196d6c58b4a590e2848896</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci200266d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200266d%26sid%3Dliteratum%253Aachs%26aulast%3DKrasowski%26aufirst%3DA.%26aulast%3DMuthas%26aufirst%3DD.%26aulast%3DSarkar%26aufirst%3DA.%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DBrenk%26aufirst%3DR.%26atitle%3DDrugpred%253A%2520A%2520structure-based%2520approach%2520to%2520predict%2520protein%2520druggability%2520developed%2520using%2520an%2520extensive%2520nonredundant%2520data%2520set%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2829%26epage%3D42%26doi%3D10.1021%2Fci200266d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Fauman, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, E. S.</span><span> </span><span class="NLM_article-title">Structure-based druggability assessment - identifying suitable targets for small molecule therapeutics</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2011.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.cbpa.2011.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21704549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFCltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2011&pages=463-8&author=E.+B.+Faumanauthor=B.+K.+Raiauthor=E.+S.+Huang&title=Structure-based+druggability+assessment+-+identifying+suitable+targets+for+small+molecule+therapeutics&doi=10.1016%2Fj.cbpa.2011.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based druggability assessment - identifying suitable targets for small molecule therapeutics</span></div><div class="casAuthors">Fauman, Eric B.; Rai, Brajesh K.; Huang, Enoch S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">463-468</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A target is druggable if it can be modulated in vivo by a drug-like mol.  The general properties of oral drugs are summarized by the rule of 5' which specifies parameters related to size and lipophilicity.  Structure-based target druggability assessment consists of predicting ligand-binding sites on the protein that are complementary to these drug-like properties.  Automated identification of ligand-binding sites can use geometrical considerations alone or include specific physicochem. properties of the protein surface.  Features of a pocket's size and shape, together with measures of its hydrophobicity, are most informative in identifying suitable drug-binding pockets.  The recent availability of several validation sets of druggable vs. undruggable targets has helped fuel the development of more elaborate methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzQbgkbe44IrVg90H21EOLACvtfcHk0lh3SDiEvkVXKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFCltrw%253D&md5=876a5083e4ae685d55eee3295e924cfd</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2011.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2011.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DFauman%26aufirst%3DE.%2BB.%26aulast%3DRai%26aufirst%3DB.%2BK.%26aulast%3DHuang%26aufirst%3DE.%2BS.%26atitle%3DStructure-based%2520druggability%2520assessment%2520-%2520identifying%2520suitable%2520targets%2520for%2520small%2520molecule%2520therapeutics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D15%26spage%3D463%26epage%3D8%26doi%3D10.1016%2Fj.cbpa.2011.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Dogsitescorer: A web server for automatic binding site prediction, analysis and druggability assessment</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">2074</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1093/bioinformatics/bts310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fbioinformatics%2Fbts310" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22628523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFSlsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=2074-5&author=A.+Volkamerauthor=D.+Kuhnauthor=F.+Rippmannauthor=M.+Rarey&title=Dogsitescorer%3A+A+web+server+for+automatic+binding+site+prediction%2C+analysis+and+druggability+assessment&doi=10.1093%2Fbioinformatics%2Fbts310"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">DoGSiteScorer: a web server for automatic binding site prediction, analysis and druggability assessment</span></div><div class="casAuthors">Volkamer, Andrea; Kuhn, Daniel; Rippmann, Friedrich; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2074-2075</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Many drug discovery projects fail because the underlying target is finally found to be undruggable.  Progress in structure elucidation of proteins now opens up a route to automatic structure-based target assessment.  DoGSiteScorer is a newly developed automatic tool combining pocket prediction, characterization and druggability estn. and is now available through a web server.  Availability: The DoGSiteScorer web server is freely available for academic use at http://dogsite.zbh.uni-hamburg.de Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="9be9fae9fee2dbe1f9f3b5eef5f2b6f3faf6f9eee9fcb5fffeb5">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN-lCqMEbql7Vg90H21EOLACvtfcHk0liUL2DcNjYe1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFSlsbzE&md5=747f225234540d2a009fcb0d5b077d40</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts310%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DDogsitescorer%253A%2520A%2520web%2520server%2520for%2520automatic%2520binding%2520site%2520prediction%252C%2520analysis%2520and%2520druggability%2520assessment%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3D2074%26epage%3D5%26doi%3D10.1093%2Fbioinformatics%2Fbts310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Nisius, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sha, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span> </span><span class="NLM_article-title">Structure-based computational analysis of protein binding sites for function and druggability prediction</span> <span class="citation_source-journal">J. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1016/j.jbiotec.2011.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jbiotec.2011.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22197384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2012&pages=123-34&author=B.+Nisiusauthor=F.+Shaauthor=H.+Gohlke&title=Structure-based+computational+analysis+of+protein+binding+sites+for+function+and+druggability+prediction&doi=10.1016%2Fj.jbiotec.2011.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based computational analysis of protein binding sites for function and druggability prediction</span></div><div class="casAuthors">Nisius, Britta; Sha, Fan; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">123-134</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein binding sites are the places where mol. interactions occur.  Thus, the anal. of protein binding sites is of crucial importance to understand the biol. processes proteins are involved in.  Herein, we focus on the computational anal. of protein binding sites and present structure-based methods that enable function prediction for orphan proteins and prediction of target druggability.  We present the general ideas behind these methods, with a special emphasis on the scopes and limitations of these methods and their validation.  Addnl., we present some successful applications of computational binding site anal. to emphasize the practical importance of these methods for biotechnol./bioeconomy and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXzgcXRf62cbVg90H21EOLACvtfcHk0liUL2DcNjYe1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahurs%253D&md5=9d8c5a3a41abd1ce0444ed23c3b3f7a9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2011.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2011.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DNisius%26aufirst%3DB.%26aulast%3DSha%26aufirst%3DF.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DStructure-based%2520computational%2520analysis%2520of%2520protein%2520binding%2520sites%2520for%2520function%2520and%2520druggability%2520prediction%26jtitle%3DJ.%2520Biotechnol.%26date%3D2012%26volume%3D159%26spage%3D123%26epage%3D34%26doi%3D10.1016%2Fj.jbiotec.2011.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Perola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, J.</span><span> </span><span class="NLM_article-title">Development of a rule-based method for the assessment of protein druggability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1027</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1021/ci200613b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200613b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XksVyls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1027-38&author=E.+Perolaauthor=L.+Hermanauthor=J.+Weiss&title=Development+of+a+rule-based+method+for+the+assessment+of+protein+druggability&doi=10.1021%2Fci200613b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Rule-Based Method for the Assessment of Protein Druggability</span></div><div class="casAuthors">Perola, Emanuele; Herman, Lee; Weiss, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1027-1038</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Target selection is a crit. step in the majority of modern drug discovery programs.  The viability of a drug target depends on two components: biol. relevance and chem. tractability.  The concept of druggability was introduced to describe the second component, and it is defined as the ability of a target to bind a drug-like mol. with a therapeutically useful level of affinity.  To investigate the rules that govern druggability, we developed an algorithm to isolate and characterize the binding pockets of protein targets.  Using this algorithm, we performed a comparative anal. between the relevant pockets of 60 targets of approved drugs and a diverse set of 440 ligand-binding pockets.  As a result, we defined a preferred property space for druggable pockets based on five key properties (vol., depth, enclosure, percentage of charged residues and hydrophobicity), and we represented it with a set of simple rules.  These rules may be applicable in the future to evaluate the chem. tractability of prospective targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaRGoeAUMXUbVg90H21EOLACvtfcHk0liUL2DcNjYe1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksVyls7w%253D&md5=49a4c07150749f6d09a062365512b8c2</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci200613b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200613b%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DHerman%26aufirst%3DL.%26aulast%3DWeiss%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520a%2520rule-based%2520method%2520for%2520the%2520assessment%2520of%2520protein%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1027%26epage%3D38%26doi%3D10.1021%2Fci200613b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grombacher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Combining global and local measures for structure-based druggability predictions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">360</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1021/ci200454v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200454v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFylurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=360-72&author=A.+Volkamerauthor=D.+Kuhnauthor=T.+Grombacherauthor=F.+Rippmannauthor=M.+Rarey&title=Combining+global+and+local+measures+for+structure-based+druggability+predictions&doi=10.1021%2Fci200454v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Global and Local Measures for Structure-Based Druggability Predictions</span></div><div class="casAuthors">Volkamer, Andrea; Kuhn, Daniel; Grombacher, Thomas; Rippmann, Friedrich; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">360-372</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting druggability and prioritizing certain disease modifying targets for the drug development process is of high practical relevance in pharmaceutical research.  DoGSiteScorer is a fully automatic algorithm for pocket and druggability prediction.  Besides consideration of global properties of the pocket, also local similarities shared between pockets are reflected.  Druggability scores are predicted by a support vector machine (SVM), trained, and tested on the druggability data set (DD) and its nonredundant version (NRDD).  The DD consists of 1069 targets with assigned druggable, difficult, and undruggable classes.  In 90% of the NRDD, the SVM model based on global descriptors correctly classifies a target as either druggable or undruggable.  Nevertheless, global properties suffer from binding site changes due to ligand binding and from the pocket boundary definition.  Therefore, local pocket properties are addnl. investigated in terms of a nearest neighbor search.  Local similarities are described by distance dependent histograms between atom pairs.  In 88% of the DD pocket set, the nearest neighbor and the structure itself conform with their druggability type.  A discriminant feature between druggable and undruggable pockets is having less short-range hydrophilic-hydrophilic pairs and more short-range lipophilic-lipophilic pairs.  Our findings for global pocket descriptors coincide with previously published methods affirming that size, shape, and hydrophobicity are important global pocket descriptors for automatic druggability prediction.  Nevertheless, the variety of pocket shapes and their flexibility upon ligand binding limit the automatic projection of druggable features onto descriptors.  Incorporating local pocket properties is another step toward a reliable descriptor-based druggability prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRnTVZG_4qNLVg90H21EOLACvtfcHk0liuc9StLRJR_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFylurrP&md5=ec276048e78e1297610e3d4b756ab6e8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci200454v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200454v%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DGrombacher%26aufirst%3DT.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DCombining%2520global%2520and%2520local%2520measures%2520for%2520structure-based%2520druggability%2520predictions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D360%26epage%3D72%26doi%3D10.1021%2Fci200454v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Driggers, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hale, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terrett, N. K.</span><span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery--an underexploited structural class</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span><span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-24&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery%2D%2Dan+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0liuc9StLRJR_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery--an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D24%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Wessjohann, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruijter, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Rivera, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, W.</span><span> </span><span class="NLM_article-title">What can a chemist learn from nature’s macrocycles?--A brief, conceptual view</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span><span class="refDoi"> DOI: 10.1007/s11030-005-1314-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs11030-005-1314-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15789564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslels7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=171-86&author=L.+A.+Wessjohannauthor=E.+Ruijterauthor=D.+Garcia-Riveraauthor=W.+Brandt&title=What+can+a+chemist+learn+from+nature%E2%80%99s+macrocycles%3F%2D%2DA+brief%2C+conceptual+view&doi=10.1007%2Fs11030-005-1314-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">What can a chemist learn from nature's macrocycles? - A brief, conceptual view</span></div><div class="casAuthors">Wessjohann, Ludger A.; Ruijter, Eelco; Garcia-Rivera, Daniel; Brandt, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">171-186</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Macrocyclic natural products often display remarkable biol. activities, and many of these compds. (or their derivs.) are used as drugs.  The chem. diversity of these compds. is immense and may provide inspiration for innovative drug design.  Therefore, a database of naturally occurring macrocycles was analyzed for ring size, mol. wt. distribution, and the frequency of some common substructural motifs.  The underlying principles of the chem. diversity are reviewed in terms of biosynthetic origin and nature's strategies for diversity and complexity generation in relation to the structural diversity and similarities found in the macrocycle database.  Finally, it is suggested that synthetic chemists should use not only nature's mols., but also nature's strategies as a source of inspiration.  To illustrate this, the biosynthesis of macrocycles by non-ribosomal peptide synthetases and terpene and polyketide cyclases, as well as recent advances of these strategies in an integrated synthesis/biotechnol. approach are briefly reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVB4g1QplcWLVg90H21EOLACvtfcHk0liuc9StLRJR_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslels7w%253D&md5=88df3ca095b446e3781369708ccddfd2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs11030-005-1314-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-005-1314-x%26sid%3Dliteratum%253Aachs%26aulast%3DWessjohann%26aufirst%3DL.%2BA.%26aulast%3DRuijter%26aufirst%3DE.%26aulast%3DGarcia-Rivera%26aufirst%3DD.%26aulast%3DBrandt%26aufirst%3DW.%26atitle%3DWhat%2520can%2520a%2520chemist%2520learn%2520from%2520nature%25E2%2580%2599s%2520macrocycles%253F--A%2520brief%252C%2520conceptual%2520view%26jtitle%3DMol.%2520Diversity%26date%3D2005%26volume%3D9%26spage%3D171%26epage%3D86%26doi%3D10.1007%2Fs11030-005-1314-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Marsault, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. L.</span><span> </span><span class="NLM_article-title">Macrocycles are great cycles: Applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1961</span><span class="NLM_x">–</span> <span class="NLM_lpage">2004</span><span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+Applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0ljYuk8kL3VO0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520Applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Mallinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1409</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-38&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0ljYuk8kL3VO0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D38%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Kalgutkar, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvie, D. K.</span><span> </span><span class="NLM_article-title">Drug discovery for a new generation of covalent drugs</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1517/17460441.2012.688744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1517%2F17460441.2012.688744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22607458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFWmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=561-81&author=A.+S.+Kalgutkarauthor=D.+K.+Dalvie&title=Drug+discovery+for+a+new+generation+of+covalent+drugs&doi=10.1517%2F17460441.2012.688744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery for a new generation of covalent drugs</span></div><div class="casAuthors">Kalgutkar, Amit S.; Dalvie, Deepak K.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">561-581</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The design of target-specific covalent inhibitors is conceptually attractive because of increased biochem. efficiency through covalency and increased duration of action that outlasts the pharmacokinetics of the agent.  Although many covalent inhibitors have been approved or are in advanced clin. trials to treat indications such as cancer and hepatitis C, there is a general tendency to avoid them as drug candidates because of concerns regarding immune-mediated toxicity that can arise from indiscriminate reactivity with off-target proteins.Areas covered: The review examines potential reason(s) for the excellent safety record of marketed covalent agents and advanced clin. candidates for emerging therapeutic targets.  A significant emphasis is placed on proteomic techniques and chem./biochem. reactivity assays that aim to provide a systematic rank ordering of pharmacol. selectivity relative to off-target protein reactivity of covalent inhibitors.Expert opinion: While tactics to examine selective covalent modification of the pharmacol. target are broadly applicable in drug discovery, it is unclear whether the output from such studies can prospectively predict idiosyncratic immune-mediated drug toxicity.  Opinions regarding an acceptable threshold of protein reactivity/body burden for a toxic electrophile and a non-toxic electrophilic covalent drug have not been defined.  Increasing confidence in proteomic and chem./biochem. reactivity screens will require a retrospective side-by-side profiling of marketed covalent drugs and electrophiles known to cause deleterious toxic effects via non-selective covalent binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowmXy20mDTPbVg90H21EOLACvtfcHk0ljYuk8kL3VO0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFWmt7k%253D&md5=b7fc8e60b263dda5765dd8380f52df17</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.688744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.688744%26sid%3Dliteratum%253Aachs%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DDalvie%26aufirst%3DD.%2BK.%26atitle%3DDrug%2520discovery%2520for%2520a%2520new%2520generation%2520of%2520covalent%2520drugs%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D561%26epage%3D81%26doi%3D10.1517%2F17460441.2012.688744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-17&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0ljYuk8kL3VO0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D17%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Way, J. C.</span><span> </span><span class="NLM_article-title">Covalent modification as a strategy to block protein-protein interactions with small-molecule drugs</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1016/S1367-5931(99)00049-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS1367-5931%2899%2900049-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10679371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtlygtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2000&pages=40-6&author=J.+C.+Way&title=Covalent+modification+as+a+strategy+to+block+protein-protein+interactions+with+small-molecule+drugs&doi=10.1016%2FS1367-5931%2899%2900049-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent modification as a strategy to block protein-protein interactions with small-molecule drugs</span></div><div class="casAuthors">Way, Jeffrey C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-46</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 42 refs.  It is generally difficult to block protein-protein interactions with small-mol. drugs.  A novel pharmaceutical development strategy to block protein interactions is emerging: targeted covalent modification to sterically block interactions.  By this approach, compds. first interact non-covalently with a specific target protein.  This interaction juxtaposes a weakly reactive group of the drug with a target amino acid sidechain, which then react by virtue of their high local concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAxQTnoYgzeLVg90H21EOLACvtfcHk0lhAMCvB2Svg2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtlygtrs%253D&md5=b5b65c074e838ddc2410435cb2f2eefc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2899%2900049-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252899%252900049-6%26sid%3Dliteratum%253Aachs%26aulast%3DWay%26aufirst%3DJ.%2BC.%26atitle%3DCovalent%2520modification%2520as%2520a%2520strategy%2520to%2520block%2520protein-protein%2520interactions%2520with%2520small-molecule%2520drugs%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2000%26volume%3D4%26spage%3D40%26epage%3D6%26doi%3D10.1016%2FS1367-5931%2899%2900049-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Edwards, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, A. J.</span><span> </span><span class="NLM_article-title">Helix-mediated protein--protein interactions as targets for intervention using foldamers</span> <span class="citation_source-journal">Amino Acids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">743</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span><span class="refDoi"> DOI: 10.1007/s00726-011-0880-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs00726-011-0880-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2011&pages=743-54&author=T.+A.+Edwardsauthor=A.+J.+Wilson&title=Helix-mediated+protein%2D%2Dprotein+interactions+as+targets+for+intervention+using+foldamers&doi=10.1007%2Fs00726-011-0880-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs00726-011-0880-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-011-0880-8%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DT.%2BA.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DHelix-mediated%2520protein--protein%2520interactions%2520as%2520targets%2520for%2520intervention%2520using%2520foldamers%26jtitle%3DAmino%2520Acids%26date%3D2011%26volume%3D41%26spage%3D743%26epage%3D54%26doi%3D10.1007%2Fs00726-011-0880-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Murray, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellman, S. H.</span><span> </span><span class="NLM_article-title">Targeting protein-protein interactions: Lessons from p53/MDM2</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">657</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span><span class="refDoi"> DOI: 10.1002/bip.20741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2Fbip.20741" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2007&pages=657-86&author=J.+K.+Murrayauthor=S.+H.+Gellman&title=Targeting+protein-protein+interactions%3A+Lessons+from+p53%2FMDM2&doi=10.1002%2Fbip.20741"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fbip.20741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20741%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DTargeting%2520protein-protein%2520interactions%253A%2520Lessons%2520from%2520p53%252FMDM2%26jtitle%3DBiopolymers%26date%3D2007%26volume%3D88%26spage%3D657%26epage%3D86%26doi%3D10.1002%2Fbip.20741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Raj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arora, P. S.</span><span> </span><span class="NLM_article-title">Plucking the high hanging fruit: A systematic approach for targeting protein-protein interactions</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">4051</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.bmc.2012.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmc.2012.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23267671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2rsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4051-7&author=M.+Rajauthor=B.+N.+Bullockauthor=P.+S.+Arora&title=Plucking+the+high+hanging+fruit%3A+A+systematic+approach+for+targeting+protein-protein+interactions&doi=10.1016%2Fj.bmc.2012.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Plucking the high hanging fruit: A systematic approach for targeting protein-protein interactions</span></div><div class="casAuthors">Raj, Monika; Bullock, Brooke N.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4051-4057</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Development of specific ligands for protein targets that help decode the complexities of protein-protein interaction networks is a key goal for the field of chem. biol.  Despite the emergence of powerful in silico and exptl. high-throughput screening strategies, the discovery of synthetic ligands that selectively modulate protein-protein interactions remains a challenge for bioorg. and medicinal chemists.  This Perspective discusses emerging principles for the rational design of PPI inhibitors.  Fundamentally, the approach seeks to adapt nature's protein recognition principles for the design of suitable secondary structure mimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorpTCgOTASB7Vg90H21EOLACvtfcHk0lhAMCvB2Svg2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2rsrbN&md5=970cdca83336f20387e060810f7f261c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DRaj%26aufirst%3DM.%26aulast%3DBullock%26aufirst%3DB.%2BN.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DPlucking%2520the%2520high%2520hanging%2520fruit%253A%2520A%2520systematic%2520approach%2520for%2520targeting%2520protein-protein%2520interactions%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4051%26epage%3D7%26doi%3D10.1016%2Fj.bmc.2012.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Walensky, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malia, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbuto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span> </span><span class="NLM_article-title">Activation of apoptosis in vivo by a hydrocarbon-stapled BH3 helix</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">305</span><span class="NLM_x">, </span> <span class="NLM_fpage">1466</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1126/science.1099191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1126%2Fscience.1099191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15353804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFart7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2004&pages=1466-70&author=L.+D.+Walenskyauthor=A.+L.+Kungauthor=I.+Escherauthor=T.+J.+Maliaauthor=S.+Barbutoauthor=R.+D.+Wrightauthor=G.+Wagnerauthor=G.+L.+Verdineauthor=S.+J.+Korsmeyer&title=Activation+of+apoptosis+in+vivo+by+a+hydrocarbon-stapled+BH3+helix&doi=10.1126%2Fscience.1099191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Apoptosis in Vivo by a Hydrocarbon-Stapled BH3 Helix</span></div><div class="casAuthors">Walensky, Loren D.; Kung, Andrew L.; Escher, Iris; Malia, Thomas J.; Barbuto, Scott; Wright, Renee D.; Wagner, Gerhard; Verdine, Gregory L.; Korsmeyer, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">305</span>
        (<span class="NLM_cas:issue">5689</span>),
    <span class="NLM_cas:pages">1466-1470</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">BCL-2 family proteins constitute a crit. control point for the regulation of apoptosis.  Protein interaction between BCL-2 members is a prominent mechanism of control and is mediated through the amphipathic α-helical BH3 segment, an essential death domain.  We used a chem. strategy, termed hydrocarbon stapling, to generate BH3 peptides with improved pharmacol. properties.  The stapled peptides, called "stabilized alpha-helix of BCL-2 domains" (SAHBs), proved to be helical, protease-resistant, and cell-permeable mols. that bound with increased affinity to multidomain BCL-2 member pockets.  A SAHB of the BH3 domain from the BID protein specifically activated the apoptotic pathway to kill leukemia cells.  In addn., SAHB effectively inhibited the growth of human leukemia xenografts in vivo.  Hydrocarbon stapling of native peptides may provide a useful strategy for exptl. and therapeutic modulation of protein-protein interactions in many signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpfOpXrd8FVbVg90H21EOLACvtfcHk0lhAMCvB2Svg2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFart7c%253D&md5=53fca0524df7a584ccf9d0991bc1bb64</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099191%26sid%3Dliteratum%253Aachs%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DEscher%26aufirst%3DI.%26aulast%3DMalia%26aufirst%3DT.%2BJ.%26aulast%3DBarbuto%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DR.%2BD.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DActivation%2520of%2520apoptosis%2520in%2520vivo%2520by%2520a%2520hydrocarbon-stapled%2520BH3%2520helix%26jtitle%3DScience%26date%3D2004%26volume%3D305%26spage%3D1466%26epage%3D70%26doi%3D10.1126%2Fscience.1099191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Brown, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Effects of conformational dynamics on predicted protein druggability</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">70</span><span class="NLM_x">–</span> <span class="NLM_lpage">2</span><span class="refDoi"> DOI: 10.1002/cmdc.200500013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2Fcmdc.200500013" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=70-2&author=S.+P.+Brownauthor=P.+J.+Hajduk&title=Effects+of+conformational+dynamics+on+predicted+protein+druggability&doi=10.1002%2Fcmdc.200500013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200500013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200500013%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DS.%2BP.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DEffects%2520of%2520conformational%2520dynamics%2520on%2520predicted%2520protein%2520druggability%26jtitle%3DChemMedChem%26date%3D2006%26volume%3D1%26spage%3D70%26epage%3D2%26doi%3D10.1002%2Fcmdc.200500013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">A hot spot of binding energy in a hormone-receptor interface</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1126/science.7529940</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1126%2Fscience.7529940" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=7529940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK2MXjt12nsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1995&pages=383-6&author=T.+Clacksonauthor=J.+A.+Wells&title=A+hot+spot+of+binding+energy+in+a+hormone-receptor+interface&doi=10.1126%2Fscience.7529940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A hot spot of binding energy in a hormone-receptor interface</span></div><div class="casAuthors">Clackson, Tim; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5196</span>),
    <span class="NLM_cas:pages">383-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The x-ray crystal structure of the complex between human growth hormone (hGH) and the extracellular domain of its first bound receptor (hGHbp) shows that about 30 side chains from each protein make contact.  Individual replacement of contact residues in the hGHbp with alanine showed that a central hydrophobic region, dominated by two tryptophan residues, accounts for more than three-quarters of the binding free energy.  This "functional epitope" is surrounded by less important contact residues that are generally hydrophilic and partially hydrated, so that the interface resembles a cross section through a globular protein.  The functionally important residues on the hGHbp directly contact those on hGH.  Thus, only a small and complementary set of contact residues maintains binding affinity, a property that may be general to protein-protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4AKI2EsvEQLVg90H21EOLACvtfcHk0ljfDnnO8jJlNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjt12nsbs%253D&md5=62b214455d644fd07afb2817d1c482fe</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fscience.7529940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7529940%26sid%3Dliteratum%253Aachs%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DA%2520hot%2520spot%2520of%2520binding%2520energy%2520in%2520a%2520hormone-receptor%2520interface%26jtitle%3DScience%26date%3D1995%26volume%3D267%26spage%3D383%26epage%3D6%26doi%3D10.1126%2Fscience.7529940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">DeLano, W. L.</span><span> </span><span class="NLM_article-title">Unraveling hot spots in binding interfaces: Progress and challenges</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1016/S0959-440X(02)00283-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0959-440X%2802%2900283-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11839484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=14-20&author=W.+L.+DeLano&title=Unraveling+hot+spots+in+binding+interfaces%3A+Progress+and+challenges&doi=10.1016%2FS0959-440X%2802%2900283-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling hot spots in binding interfaces: progress and challenges</span></div><div class="casAuthors">DeLano, Warren L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-20</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 49 refs.  Protein interface hot spots, as revealed by alanine scanning mutagenesis, continue to stimulate interest in the biophys. basis of mol. recognition.  Although these regions apparently constitute fertile grounds for intermol. interactions, no general algorithm has yet been developed that can predict hot spots based solely on their shape or compn.  The discovery of structural plasticity in hot spot regions indicates that dynamic simulation techniques may be essential for achieving a predictive understanding of binding interface energetics.  Future progress will depend as much on the application of new computational approaches for dissecting protein interfaces as on expanding the empirical databank of mutagenic substitutions and their effects.  Despite current theor. shortcomings, recent methodol. advances provide efficient exptl. means of probing hot spots and enable immediate applications for hot spots in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk1HNI-4QfrrVg90H21EOLACvtfcHk0ljfDnnO8jJlNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D&md5=a297ae443e69c94b7cf2e342337f29ea</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2802%2900283-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252802%252900283-X%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26atitle%3DUnraveling%2520hot%2520spots%2520in%2520binding%2520interfaces%253A%2520Progress%2520and%2520challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D12%26spage%3D14%26epage%3D20%26doi%3D10.1016%2FS0959-440X%2802%2900283-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Keskin, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span> </span><span class="NLM_article-title">Hot regions in protein-protein interactions: The organization and contribution of structurally conserved hot spot residues</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">345</span><span class="NLM_x">, </span> <span class="NLM_fpage">1281</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span><span class="refDoi"> DOI: 10.1016/j.jmb.2004.10.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jmb.2004.10.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15644221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXis1ahsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2005&pages=1281-94&author=O.+Keskinauthor=B.+Y.+Maauthor=R.+Nussinov&title=Hot+regions+in+protein-protein+interactions%3A+The+organization+and+contribution+of+structurally+conserved+hot+spot+residues&doi=10.1016%2Fj.jmb.2004.10.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Hot Regions in Protein-Protein Interactions: The Organization and Contribution of Structurally Conserved Hot Spot Residues</span></div><div class="casAuthors">Keskin, Ozlem; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1281-1294</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Structurally conserved residues at protein-protein interfaces correlate with the exptl. alanine-scanning hot spots.  Here, we investigate the organization of these conserved, computational hot spots and their contribution to the stability of protein assocns.  We find that computational hot spots are not homogeneously distributed along the protein interfaces; rather, they are clustered within locally tightly packed regions.  Within the dense clusters, they form a network of interactions, and, consequently, their contributions to the stability of the complex are cooperative; however the contributions of independent clusters are additive.  This suggests that the binding free energy is not a simple summation of the single hot spot residue contributions.  As expected, around the hot spot residues, we observe moderately conserved residues, further highlighting the crucial role of the conserved interactions in the local densely packed environment.  The conserved occurrence of these organizations suggests that they are advantageous for protein-protein assocns.  Interestingly, the total no. of hydrogen bonds and salt bridges contributed by hot spots is as expected.  Thus, H-bond forming residues may use a "hot spot for water exclusion" mechanism.  Since conserved residues are located within highly packed regions, water mols. are easily removed upon binding, strengthening electrostatic contributions of charge-charge interactions.  Hence, the picture that emerges is that protein-protein assocns. are optimized locally, with the clustered, networked, highly packed structurally conserved residues contributing dominantly and cooperatively to the stability of the complex.  When addressing the crucial question of "what are the preferred ways of proteins to assoc.", these findings point toward a crit. involvement of hot regions in protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1ETiLA5TuabVg90H21EOLACvtfcHk0ljfDnnO8jJlNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXis1ahsw%253D%253D&md5=8adbf29ef0563f429415c67fd209c896</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.10.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.10.077%26sid%3Dliteratum%253Aachs%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DMa%26aufirst%3DB.%2BY.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DHot%2520regions%2520in%2520protein-protein%2520interactions%253A%2520The%2520organization%2520and%2520contribution%2520of%2520structurally%2520conserved%2520hot%2520spot%2520residues%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D345%26spage%3D1281%26epage%3D94%26doi%3D10.1016%2Fj.jmb.2004.10.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Bogan, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorn, K. S.</span><span> </span><span class="NLM_article-title">Anatomy of hot spots in protein interfaces</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1006/jmbi.1998.1843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1006%2Fjmbi.1998.1843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=9653027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=1998&pages=1-9&author=A.+A.+Boganauthor=K.+S.+Thorn&title=Anatomy+of+hot+spots+in+protein+interfaces&doi=10.1006%2Fjmbi.1998.1843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomy of hot spots in protein interfaces</span></div><div class="casAuthors">Bogan A A; Thorn K S</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Binding of one protein to another is involved in nearly all biological functions, yet the principles governing the interaction of proteins are not fully understood.  To analyze the contributions of individual amino acid residues in protein-protein binding we have compiled a database of 2325 alanine mutants for which the change in free energy of binding upon mutation to alanine has been measured (available at http://motorhead. ucsf.edu/thorn/hotspot).  Our analysis shows that at the level of side-chains there is little correlation between buried surface area and free energy of binding.  We find that the free energy of binding is not evenly distributed across interfaces; instead, there are hot spots of binding energy made up of a small subset of residues in the dimer interface.  These hot spots are enriched in tryptophan, tyrosine and arginine, and are surrounded by energetically less important residues that most likely serve to occlude bulk solvent from the hot spot.  Occlusion of solvent is found to be a necessary condition for highly energetic interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZMrH3BevWP3ELvajU9pjPfW6udTcc2eatYzOopfzZJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D&md5=e3ef3c857d1fe070deff25dc54d09587</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.1843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.1843%26sid%3Dliteratum%253Aachs%26aulast%3DBogan%26aufirst%3DA.%2BA.%26aulast%3DThorn%26aufirst%3DK.%2BS.%26atitle%3DAnatomy%2520of%2520hot%2520spots%2520in%2520protein%2520interfaces%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1998%26volume%3D280%26spage%3D1%26epage%3D9%26doi%3D10.1006%2Fjmbi.1998.1843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Kortemme, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, D.</span><span> </span><span class="NLM_article-title">A simple physical model for binding energy hot spots in protein-protein complexes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">14116</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1073/pnas.202485799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.202485799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=12381794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=14116-21&author=T.+Kortemmeauthor=D.+Baker&title=A+simple+physical+model+for+binding+energy+hot+spots+in+protein-protein+complexes&doi=10.1073%2Fpnas.202485799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A simple physical model for binding energy hot spots in protein-protein complexes</span></div><div class="casAuthors">Kortemme, Tanja; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">14116-14121</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein-protein recognition plays a central role in most biol. processes.  Although the structures of many protein-protein complexes have been solved in mol. detail, general rules describing affinity and selectivity of protein-protein interactions do not accurately account for the extremely diverse nature of the interfaces.  We investigate the extent to which a simple phys. model can account for the wide range of exptl. measured free energy changes brought about by alanine mutation at protein-protein interfaces.  The model successfully predicts the results of alanine scanning expts. on globular proteins (743 mutations) and 19 protein-protein interfaces (233 mutations) with av. unsigned errors of 0.81 kcal/mol and 1.06 kcal/mol, resp.  The results test our understanding of the dominant contributions to the free energy of protein-protein interactions, can guide expts. aimed at the design of protein interaction inhibitors, and provide a stepping-stone to important applications such as interface redesign.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1GbqIE1dkgrVg90H21EOLACvtfcHk0li9PIoTONDlgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2it70%253D&md5=f5e5fb01e6e803efb2ad1922cb517271</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202485799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202485799%26sid%3Dliteratum%253Aachs%26aulast%3DKortemme%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DA%2520simple%2520physical%2520model%2520for%2520binding%2520energy%2520hot%2520spots%2520in%2520protein-protein%2520complexes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D14116%26epage%3D21%26doi%3D10.1073%2Fpnas.202485799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Locating and characterizing binding sites on proteins</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1038/nbt0596-595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnbt0596-595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=9630949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK28Xis1eksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=595-9&author=C.+Mattosauthor=D.+Ringe&title=Locating+and+characterizing+binding+sites+on+proteins&doi=10.1038%2Fnbt0596-595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Locating and characterizing binding sites of proteins</span></div><div class="casAuthors">Mattos, Carla; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">595-9</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">A review with 38 refs.  This review article begins with a discussion of fundamental differences between substrates and inhibitors, and some of the assumptions and goals underlying the design of a new ligand to a target protein.  An overview is given of the methods currently used to locate and characterize ligand binding sites on protein surfaces, with focus on a novel approach: multiple solvent crystal structures (MSCS).  In this method, the x-ray crystal structure of the target protein is solved in a variety or org. solvents.  Each type of solvent mol. serves as a probe for complementary binding sites on the protein.  The probe distribution on the protein surface allows the location of binding sites and the characterization of the potential ligand interactions within these sites.  General aspects of the application of the MSCS method to porcine pancreatic elastase is discussed, and comparison of the results with those from x-ray crystal structures of elastase/inhibitor complexes is used to illustrate the potential of the method in aiding the process of rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDEKZ8sM6gk7Vg90H21EOLACvtfcHk0li9PIoTONDlgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis1eksr0%253D&md5=4facd2f240311e8937623352e7d933d3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnbt0596-595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0596-595%26sid%3Dliteratum%253Aachs%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DLocating%2520and%2520characterizing%2520binding%2520sites%2520on%2520proteins%26jtitle%3DNat.%2520Biotechnol.%26date%3D1996%26volume%3D14%26spage%3D595%26epage%3D9%26doi%3D10.1038%2Fnbt0596-595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellamacina, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, X. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffery, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">An experimental approach to mapping the binding surfaces of crystalline proteins</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">2605</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span><span class="refDoi"> DOI: 10.1021/jp952516o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp952516o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK28Xmt1GrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1996&pages=2605-11&author=K.+N.+Allenauthor=C.+R.+Bellamacinaauthor=X.+C.+Dingauthor=C.+J.+Jefferyauthor=C.+Mattosauthor=G.+A.+Petskoauthor=D.+Ringe&title=An+experimental+approach+to+mapping+the+binding+surfaces+of+crystalline+proteins&doi=10.1021%2Fjp952516o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">An Experimental Approach to Mapping the Binding Surfaces of Crystalline Proteins</span></div><div class="casAuthors">Allen, Karen N.; Bellamacina, Cornelia R.; Ding, Xiaochun; Jeffery, Constance J.; Mattos, Carla; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2605-11</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Porcine pancreatic elastase has been used as the model enzyme in the design and development of a crystallog. method that allows mapping of the binding surface of a protein by solving its crystal structure in a variety of org. solvents.  The ultimate goal of this method is to aid in the process of drug design, where each of the chosen org. mols. represents a given functional group in a larger inhibitor mol.  This method of multiple solvent crystal structures (MSCS) has a theor. counterpart in the method of multiply copy simultaneous search (MCSS) (Miranker, A.; Karplus, M. Proteins: Struct., Funct., Genet. 1991, 11, 29-34) and is the first exptl. method that can be used as a check to the theory.  The MSCS method is presented here with acetonitrile as the probe org. solvent.  The procedure involved does not cause significant changes in the structure of elastase as compared to the structure in aq. soln., and the positions found for the acetonitrile mols. in the active site are compared to those of similar functional groups belonging to known inhibitors bound to elastase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhPgz3NRlYqrVg90H21EOLACvtfcHk0lhktDwvWxEEHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmt1GrtQ%253D%253D&md5=c98dea1a223f30e444b9bc7faff25f0b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjp952516o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp952516o%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DBellamacina%26aufirst%3DC.%2BR.%26aulast%3DDing%26aufirst%3DX.%2BC.%26aulast%3DJeffery%26aufirst%3DC.%2BJ.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DAn%2520experimental%2520approach%2520to%2520mapping%2520the%2520binding%2520surfaces%2520of%2520crystalline%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1996%26volume%3D100%26spage%3D2605%26epage%3D11%26doi%3D10.1021%2Fjp952516o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">English, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Done, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caves, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">Locating interaction sites on proteins: The crystal structure of thermolysin soaked in 2% to 100% isopropanol</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">628</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19991201)37:4<628::AID-PROT13>3.3.CO;2-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2F%28SICI%291097-0134%2819991201%2937%3A4%3C628%3A%3AAID-PROT13%3E3.3.CO%3B2-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1999&pages=628-40&author=A.+C.+Englishauthor=S.+H.+Doneauthor=L.+S.+Cavesauthor=C.+R.+Groomauthor=R.+E.+Hubbard&title=Locating+interaction+sites+on+proteins%3A+The+crystal+structure+of+thermolysin+soaked+in+2%25+to+100%25+isopropanol&doi=10.1002%2F%28SICI%291097-0134%2819991201%2937%3A4%3C628%3A%3AAID-PROT13%3E3.3.CO%3B2-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819991201%2937%3A4%3C628%3A%3AAID-PROT13%3E3.3.CO%3B2-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819991201%252937%253A4%253C628%253A%253AAID-PROT13%253E3.3.CO%253B2-7%26sid%3Dliteratum%253Aachs%26aulast%3DEnglish%26aufirst%3DA.%2BC.%26aulast%3DDone%26aufirst%3DS.%2BH.%26aulast%3DCaves%26aufirst%3DL.%2BS.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DLocating%2520interaction%2520sites%2520on%2520proteins%253A%2520The%2520crystal%2520structure%2520of%2520thermolysin%2520soaked%2520in%25202%2525%2520to%2520100%2525%2520isopropanol%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1999%26volume%3D37%26spage%3D628%26epage%3D40%26doi%3D10.1002%2F%28SICI%291097-0134%2819991201%2937%3A4%3C628%3A%3AAID-PROT13%3E3.3.CO%3B2-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">English, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">Experimental and computational mapping of the binding surface of a crystalline protein</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1093/protein/14.1.47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fprotein%2F14.1.47" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2001&pages=47-59&author=A.+C.+Englishauthor=C.+R.+Groomauthor=R.+E.+Hubbard&title=Experimental+and+computational+mapping+of+the+binding+surface+of+a+crystalline+protein&doi=10.1093%2Fprotein%2F14.1.47"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F14.1.47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F14.1.47%26sid%3Dliteratum%253Aachs%26aulast%3DEnglish%26aufirst%3DA.%2BC.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DExperimental%2520and%2520computational%2520mapping%2520of%2520the%2520binding%2520surface%2520of%2520a%2520crystalline%2520protein%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2001%26volume%3D14%26spage%3D47%26epage%3D59%26doi%3D10.1093%2Fprotein%2F14.1.47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Dechene, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wink, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swartz, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span> </span><span class="NLM_article-title">Multiple solvent crystal structures of ribonuclease A: An assessment of the method</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1002/prot.22393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2Fprot.22393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19291738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Clur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2009&pages=861-81&author=M.+Decheneauthor=G.+Winkauthor=M.+Smithauthor=P.+Swartzauthor=C.+Mattos&title=Multiple+solvent+crystal+structures+of+ribonuclease+A%3A+An+assessment+of+the+method&doi=10.1002%2Fprot.22393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple solvent crystal structures of ribonuclease A: An assessment of the method</span></div><div class="casAuthors">Dechene, Michelle; Wink, Glenna; Smith, Mychal; Swartz, Paul; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">861-881</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The multiple solvent crystal structures (MSCS) method uses org. solvents to map the surfaces of proteins.  It identifies binding sites and allows for a more thorough examn. of protein plasticity and hydration than could be achieved by a single structure.  The crystal structures of bovine pancreatic RNase A (RNase A) soaked in the following org. solvents are presented: 50% dioxane, 50% DMF, 70% dimethylsulfoxide, 70% 1,6-hexanediol, 70% isopropanol, 50% R,S,R-bisfuran alc., 70% t-butanol, 50% trifluoroethanol, or 1.0M trimethylamine-N-oxide.  This set of structures is compared with four sets of crystal structures of RNase A from the protein data bank (PDB) and with the soln. NMR structure to assess the validity of previously untested assumptions assocd. with MSCS anal.  Plasticity from MSCS is the same as from PDB structures obtained in the same crystal form and deviates only at crystal contacts when compared to structures from a diverse set of crystal environments.  Furthermore, there is a good correlation between plasticity as obsd. by MSCS and the dynamic regions seen by NMR.  Conserved water binding sites are identified by MSCS to be those that are conserved in the sets of structures taken from the PDB.  Comparison of the MSCS structures with inhibitor-bound crystal structures of RNase A reveals that the org. solvent mols. identify key interactions made by inhibitor mols., highlighting ligand binding hot-spots in the active site.  The present work firmly establishes the relevance of information obtained by MSCS.  Proteins 2009. © 2009 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0dhAgJZJfDLVg90H21EOLACvtfcHk0lhktDwvWxEEHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Clur8%253D&md5=023859b5a4d1847d2a01ebfe049a4097</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fprot.22393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22393%26sid%3Dliteratum%253Aachs%26aulast%3DDechene%26aufirst%3DM.%26aulast%3DWink%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DM.%26aulast%3DSwartz%26aufirst%3DP.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DMultiple%2520solvent%2520crystal%2520structures%2520of%2520ribonuclease%2520A%253A%2520An%2520assessment%2520of%2520the%2520method%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2009%26volume%3D76%26spage%3D861%26epage%3D81%26doi%3D10.1002%2Fprot.22393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Buhrman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connor, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zerbe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kearney, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napoleon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovrigina, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovrigin, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span> </span><span class="NLM_article-title">Analysis of binding site hot spots on the surface of Ras GTPase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">413</span><span class="NLM_x">, </span> <span class="NLM_fpage">773</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1016/j.jmb.2011.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jmb.2011.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21945529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKisbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=413&publication_year=2011&pages=773-89&author=G.+Buhrmanauthor=C.+O%E2%80%99Connorauthor=B.+Zerbeauthor=B.+M.+Kearneyauthor=R.+Napoleonauthor=E.+A.+Kovriginaauthor=S.+Vajdaauthor=D.+Kozakovauthor=E.+L.+Kovriginauthor=C.+Mattos&title=Analysis+of+binding+site+hot+spots+on+the+surface+of+Ras+GTPase&doi=10.1016%2Fj.jmb.2011.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Binding Site Hot Spots on the Surface of Ras GTPase</span></div><div class="casAuthors">Buhrman, Greg; O'Connor, Casey; Zerbe, Brandon; Kearney, Bradley M.; Napoleon, Raeanne; Kovrigina, Elizaveta A.; Vajda, Sandor; Kozakov, Dima; Kovrigin, Evgenii L.; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">413</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">773-789</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have recently discovered an allosteric switch in Ras, bringing an addnl. level of complexity to this GTPase whose mutants are involved in nearly 30% of cancers.  Upon activation of the allosteric switch, there is a shift in helix 3/loop 7 assocd. with a disorder to order transition in the active site.  Here, we use a combination of multiple solvent crystal structures and computational solvent mapping (FTMap) to det. binding site hot spots in the "off" and "on" allosteric states of the GTP-bound form of H-Ras.  Thirteen sites are revealed, expanding possible target sites for ligand binding well beyond the active site.  Comparison of FTMaps for the H and K isoforms reveals essentially identical hot spots.  Furthermore, using NMR measurements of spin relaxation, we detd. that K-Ras exhibits global conformational dynamics very similar to those we previously reported for H-Ras.  We thus hypothesize that the global conformational rearrangement serves as a mechanism for allosteric coupling between the effector interface and remote hot spots in all Ras isoforms.  At least with respect to the binding sites involving the G domain, H-Ras is an excellent model for K-Ras and probably N-Ras as well.  Ras has so far been elusive as a target for drug design.  The present work identifies various unexplored hot spots throughout the entire surface of Ras, extending the focus from the disordered active site to well-ordered locations that should be easier to target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAmYA3G3UBirVg90H21EOLACvtfcHk0lhktDwvWxEEHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKisbrN&md5=d54dee9a6ac58d27f87d85a6d25415a3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DBuhrman%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DC.%26aulast%3DZerbe%26aufirst%3DB.%26aulast%3DKearney%26aufirst%3DB.%2BM.%26aulast%3DNapoleon%26aufirst%3DR.%26aulast%3DKovrigina%26aufirst%3DE.%2BA.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DKovrigin%26aufirst%3DE.%2BL.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DAnalysis%2520of%2520binding%2520site%2520hot%2520spots%2520on%2520the%2520surface%2520of%2520Ras%2520GTPase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D413%26spage%3D773%26epage%3D89%26doi%3D10.1016%2Fj.jmb.2011.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Edfeldt, F. N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folmer, R. H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breeze, A. L.</span><span> </span><span class="NLM_article-title">Fragment screening to predict druggability (ligandability) and lead discovery success</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1016/j.drudis.2011.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.drudis.2011.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21315179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFemsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=284-287&author=F.+N.+B.+Edfeldtauthor=R.+H.+A.+Folmerauthor=A.+L.+Breeze&title=Fragment+screening+to+predict+druggability+%28ligandability%29+and+lead+discovery+success&doi=10.1016%2Fj.drudis.2011.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment screening to predict druggability (ligandability) and lead discovery success</span></div><div class="casAuthors">Edfeldt, Fredrik N. B.; Folmer, Rutger H. A.; Breeze, Alexander L.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7/8</span>),
    <span class="NLM_cas:pages">284-287</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This article discusses the use of fragment screening to predict ligandability which will be of great interest to the pharmaceutical industry.  Data presented in this article clearly show how fragment-based ligandability screening can be used in a truly predictive fashion and could be a powerful tool in reducing agent attrition by filtering out non-ligandable targets before entry into the portfolio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSXfOy9wAHFLVg90H21EOLACvtfcHk0ljZyGeudi35Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFemsbs%253D&md5=7ba5adfe3d6a1eb4c9eaa3ce42c60044</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DEdfeldt%26aufirst%3DF.%2BN.%2BB.%26aulast%3DFolmer%26aufirst%3DR.%2BH.%2BA.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26atitle%3DFragment%2520screening%2520to%2520predict%2520druggability%2520%2528ligandability%2529%2520and%2520lead%2520discovery%2520success%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D284%26epage%3D287%26doi%3D10.1016%2Fj.drudis.2011.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krimm, I.</span><span> </span><span class="NLM_article-title">Ligand specificity, privileged substructures and protein druggability from fragment-based screening</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">–</span> <span class="NLM_lpage">74</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2011.02.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.cbpa.2011.02.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21411360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFCltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2011&pages=469-74&author=S.+Barelierauthor=I.+Krimm&title=Ligand+specificity%2C+privileged+substructures+and+protein+druggability+from+fragment-based+screening&doi=10.1016%2Fj.cbpa.2011.02.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand specificity, privileged substructures and protein druggability from fragment-based screening</span></div><div class="casAuthors">Barelier, Sarah; Krimm, Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fragment-based screening has now become an established method for the generation of lead mols. against therapeutic targets.  Fragment mols. are simple, low mol.-wt. compds. with few chem. functionalities.  These characteristics lead to high hit rates for fragment screening as compared to the more classical High-Throughput Screening of drug-like mols. and raise the question of the specificity of fragment mols.  This review analyzes recent outcomes of fragment screenings published in the literature, showing that the specificity of the fragments can be related to their structures and physico-chem. properties.  We also discuss both the concept of privileged fragment scaffolds and the role of fragment-based screening in predicting protein druggability, highlighted by recent publications in the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD_Gsih0nTRLVg90H21EOLACvtfcHk0ljZyGeudi35Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFCltr0%253D&md5=7c9e813220263e006c9ae01e51a9334f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2011.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2011.02.020%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DKrimm%26aufirst%3DI.%26atitle%3DLigand%2520specificity%252C%2520privileged%2520substructures%2520and%2520protein%2520druggability%2520from%2520fragment-based%2520screening%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D15%26spage%3D469%26epage%3D74%26doi%3D10.1016%2Fj.cbpa.2011.02.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Dennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kortvelyesi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Computational mapping identifies the binding sites of organic solvents on proteins</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">4290</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1073/pnas.062398499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.062398499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11904374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=4290-5&author=S.+Dennisauthor=T.+Kortvelyesiauthor=S.+Vajda&title=Computational+mapping+identifies+the+binding+sites+of+organic+solvents+on+proteins&doi=10.1073%2Fpnas.062398499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Computational mapping identifies the binding sites of organic solvents on proteins</span></div><div class="casAuthors">Dennis, Sheldon; Kortvelyesi, Tamas; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4290-4295</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Computational mapping places mol. probes-small mols. or functional groups-on a protein surface to identify the most favorable binding positions.  Although x-ray crystallog. and NMR show that org. solvents bind to a limited no. of sites on a protein, current mapping methods result in hundreds of energy min. and do not reveal why some sites bind mols. with different sizes and polarities.  We describe a mapping algorithm that explains the origin of this phenomenon.  The algorithm has been applied to hen egg-white lysozyme and to thermolysin, interacting with eight and four different ligands, resp.  In both cases the search finds the consensus site to which all mols. bind, whereas other positions that bind only certain ligands are not necessarily found.  The consensus sites are pockets of the active site, lined with partially exposed hydrophobic residues and with a no. of polar residues toward the edge.  These sites can accommodate each ligand in a no. of rotational states, some with a hydrogen bond to one of the nearby donor/acceptor groups.  Specific substrates and/or inhibitors of hen egg-white lysozyme and thermolysin interact with the same side chains identified by the mapping, but form several hydrogen bonds and bind in unique orientations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4sEo2VxwF7Vg90H21EOLACvtfcHk0ljZyGeudi35Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D&md5=90d71ffcc8860e87aefec3c5809b1127</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.062398499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.062398499%26sid%3Dliteratum%253Aachs%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DKortvelyesi%26aufirst%3DT.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DComputational%2520mapping%2520identifies%2520the%2520binding%2520sites%2520of%2520organic%2520solvents%2520on%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D4290%26epage%3D5%26doi%3D10.1073%2Fpnas.062398499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Landon, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancia, D. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiel, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Identification of hot spots within druggable binding regions by computational solvent mapping of proteins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1231</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span><span class="refDoi"> DOI: 10.1021/jm061134b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061134b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1231-40&author=M.+R.+Landonauthor=D.+R.+Lanciaauthor=J.+Yuauthor=S.+C.+Thielauthor=S.+Vajda&title=Identification+of+hot+spots+within+druggable+binding+regions+by+computational+solvent+mapping+of+proteins&doi=10.1021%2Fjm061134b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Hot Spots within Druggable Binding Regions by Computational Solvent Mapping of Proteins</span></div><div class="casAuthors">Landon, Melissa R.; Lancia, David R., Jr.; Yu, Jessamin; Thiel, Spencer C.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1231-1240</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors apply the computational solvent mapping (CS-Map) algorithm toward the in silico identification of hot spots, i.e., regions of protein binding sites that are major contributors to the binding energy and, hence, are prime targets in drug design.  The CS-Map algorithm, developed for binding site characterization, moves small org. functional groups around the protein surface and dets. their most energetically favorable binding positions.  The utility of CS-Map algorithm toward the prediction of hot spot regions in druggable binding pockets is illustrated by three test systems: (1) renin aspartic protease, (2) a set of previously characterized druggable proteins, and (3) E. coli ketopantoate reductase.  In each of the three studies, existing literature was used to verify the results.  Based on the authors analyses, the authors conclude that the information provided by CS-Map can contribute substantially to the identification of hot spots, a necessary predecessor of fragment-based drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTVim9shdtBbVg90H21EOLACvtfcHk0liCV-bKQ1okuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWisrw%253D&md5=2d6a0678acdb2dd45fed73b05229bd03</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm061134b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061134b%26sid%3Dliteratum%253Aachs%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DLancia%26aufirst%3DD.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DThiel%26aufirst%3DS.%2BC.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520hot%2520spots%2520within%2520druggable%2520binding%2520regions%2520by%2520computational%2520solvent%2520mapping%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1231%26epage%3D40%26doi%3D10.1021%2Fjm061134b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Brenke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landon, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Fragment-based identification of druggable ’hot spots’ of proteins using Fourier domain correlation techniques</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">621</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fbioinformatics%2Fbtp036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19176554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFeksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=621-7&author=R.+Brenkeauthor=D.+Kozakovauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=D.+Hallauthor=M.+R.+Landonauthor=C.+Mattosauthor=S.+Vajda&title=Fragment-based+identification+of+druggable+%E2%80%99hot+spots%E2%80%99+of+proteins+using+Fourier+domain+correlation+techniques&doi=10.1093%2Fbioinformatics%2Fbtp036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based identification of druggable hot spots' of proteins using Fourier domain correlation techniques</span></div><div class="casAuthors">Brenke, Ryan; Kozakov, Dima; Chuang, Gwo-Yu; Beglov, Dmitri; Hall, David; Landon, Melissa R.; Mattos, Carla; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">621-627</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The binding sites of proteins generally contain smaller regions that provide major contributions to the binding free energy and hence are the prime targets in drug design.  Screening libraries of fragment-sized compds. by NMR or X-ray crystallog. demonstrates that such hot spot' regions bind a large variety of small org. mols., and that a relatively high hit rate' is predictive of target sites that are likely to bind drug-like ligands with high affinity.  The authors' goal is to det. the hot spots' computationally rather than exptl.  The authors have developed the FTMAP algorithm that performs global search of the entire protein surface for regions that bind a no. of small org. probe mols.  The search is based on the extremely efficient fast Fourier transform (FFT) correlation approach which can sample billions of probe positions on dense translational and rotational grids, but can use only sums of correlation functions for scoring and hence is generally restricted to very simple energy expressions.  The novelty of FTMAP is that the authors were able to incorporate and represent on grids a detailed energy expression, resulting in a very accurate identification of low-energy probe clusters.  Overlapping clusters of different probes are defined as consensus sites (CSs).  The authors show that the largest CS is generally located at the most important subsite of the protein binding site, and the nearby smaller CSs identify other important subsites.  Mapping results are presented for elastase whose structure was solved in aq. solns. of 8 org. solvents, and they show that FTMAP provides very similar information.  The second application is to renin, a long-standing pharmaceutical target for the treatment of hypertension, and the authors show that the major CSs trace out the shape of the first approved renin inhibitor, aliskiren.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWSzH6Fg129bVg90H21EOLACvtfcHk0liCV-bKQ1okuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFeksr0%253D&md5=e5a466428f4dd47e89a4ae8361d3da41</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp036%26sid%3Dliteratum%253Aachs%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DFragment-based%2520identification%2520of%2520druggable%2520%25E2%2580%2599hot%2520spots%25E2%2580%2599%2520of%2520proteins%2520using%2520Fourier%2520domain%2520correlation%2520techniques%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3D621%26epage%3D7%26doi%3D10.1093%2Fbioinformatics%2Fbtp036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Landon, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoang, Q. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caaveiro, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orwig, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Detection of ligand binding hot spots on protein surfaces via fragment-based methods: Application to DJ-1 and glucocerebrosidase</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">500</span><span class="refDoi"> DOI: 10.1007/s10822-009-9283-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs10822-009-9283-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19521672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=491-500&author=M.+R.+Landonauthor=R.+L.+Liebermanauthor=Q.+Q.+Hoangauthor=S.+Juauthor=J.+M.+Caaveiroauthor=S.+D.+Orwigauthor=D.+Kozakovauthor=R.+Brenkeauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=S.+Vajdaauthor=G.+A.+Petskoauthor=D.+Ringe&title=Detection+of+ligand+binding+hot+spots+on+protein+surfaces+via+fragment-based+methods%3A+Application+to+DJ-1+and+glucocerebrosidase&doi=10.1007%2Fs10822-009-9283-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of ligand binding hot spots on protein surfaces via fragment-based methods: application to DJ-1 and glucocerebrosidase</span></div><div class="casAuthors">Landon, Melissa R.; Lieberman, Raquel L.; Hoang, Quyen Q.; Ju, Shulin; Caaveiro, Jose M. M.; Orwig, Susan D.; Kozakov, Dima; Brenke, Ryan; Chuang, Gwo-Yu; Beglov, Dmitry; Vajda, Sandor; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">491-500</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The identification of hot spots, i.e., binding regions that contribute substantially to the free energy of ligand binding, is a crit. step for structure-based drug design.  Here we present the application of two fragment-based methods to the detection of hot spots for DJ-1 and glucocerebrosidase (GCase), targets for the development of therapeutics for Parkinson's and Gaucher's diseases, resp.  While the structures of these two proteins are known, binding information is lacking.  In this study we employ the exptl. multiple solvent crystal structures (MSCS) method and computational fragment mapping (FTMap) to identify regions suitable for the development of pharmacol. chaperones for DJ-1 and GCase.  Comparison of data derived via MSCS and FTMap also shows that FTMap, a computational method for the identification of fragment binding hot spots, is an accurate and robust alternative to the performance of expensive and difficult crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVrKBMCEfe6LVg90H21EOLACvtfcHk0liCV-bKQ1okuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtb8%253D&md5=00a35c8ecaaa3b9f3f9aeff7c7e61d27</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9283-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9283-2%26sid%3Dliteratum%253Aachs%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DLieberman%26aufirst%3DR.%2BL.%26aulast%3DHoang%26aufirst%3DQ.%2BQ.%26aulast%3DJu%26aufirst%3DS.%26aulast%3DCaaveiro%26aufirst%3DJ.%2BM.%26aulast%3DOrwig%26aufirst%3DS.%2BD.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DDetection%2520of%2520ligand%2520binding%2520hot%2520spots%2520on%2520protein%2520surfaces%2520via%2520fragment-based%2520methods%253A%2520Application%2520to%2520DJ-1%2520and%2520glucocerebrosidase%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D491%26epage%3D500%26doi%3D10.1007%2Fs10822-009-9283-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Where does amantadine bind to the influenza virus M2 proton channel?</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">471</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1016/j.tibs.2010.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.tibs.2010.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=20382026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFamtrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2010&pages=471-5&author=D.+Kozakovauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=S.+Vajda&title=Where+does+amantadine+bind+to+the+influenza+virus+M2+proton+channel%3F&doi=10.1016%2Fj.tibs.2010.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Where does amantadine bind to the influenza virus M2 proton channel?</span></div><div class="casAuthors">Kozakov, Dima; Chuang, Gwo-Yu; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">471-475</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Structures of the influenza A virus M2 proton channel in the open conformation have been detd. by X-ray crystallog., and in the closed conformation by NMR.  Whereas the X-ray structure shows a single inhibitor mol. in the middle of the channel, four inhibitor mols. bind the channel's outer surface in the NMR structure.  In both structures, the strongest hot spots (i.e., regions that contribute substantially to the free energy of binding any potential ligand) lie inside the pore, and other hot spots are found at exterior locations.  By considering all available models, we propose the primary drug binding site is inside the pore, but that exterior binding occurs under appropriate conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPdRX2k5BMm7Vg90H21EOLACvtfcHk0liQgQUnPKfufw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFamtrbP&md5=c8cd562c32b5b63ae865cf0df9e1f457</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DWhere%2520does%2520amantadine%2520bind%2520to%2520the%2520influenza%2520virus%2520M2%2520proton%2520channel%253F%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2010%26volume%3D35%26spage%3D471%26epage%3D5%26doi%3D10.1016%2Fj.tibs.2010.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cencic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Structural conservation of druggable hot spots in protein-protein interfaces</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">13528</span><span class="NLM_x">–</span> <span class="NLM_lpage">33</span><span class="refDoi"> DOI: 10.1073/pnas.1101835108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1101835108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21808046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2ms7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13528-33&author=D.+Kozakovauthor=D.+R.+Hallauthor=G.+Y.+Chuangauthor=R.+Cencicauthor=R.+Brenkeauthor=L.+E.+Groveauthor=D.+Beglovauthor=J.+Pelletierauthor=A.+Whittyauthor=S.+Vajda&title=Structural+conservation+of+druggable+hot+spots+in+protein-protein+interfaces&doi=10.1073%2Fpnas.1101835108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural conservation of druggable hot spots in protein-protein interfaces</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Chuang, Gwo-Yu; Cencic, Regina; Brenke, Ryan; Grove, Laurie E.; Beglov, Dmitri; Pelletier, Jerry; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">13528-13533, S13528/1-S13528/26</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Despite the growing no. of examples of small-mol. inhibitors that disrupt protein-protein interactions (PPIs), the origin of druggability of such targets is poorly understood.  To identify druggable sites in protein-protein interfaces we combine computational solvent mapping, which explores the protein surface using a variety of small "probe" mols., with a conformer generator to account for side-chain flexibility.  Applications to unliganded structures of 15 PPI target proteins show that the druggable sites comprise a cluster of binding hot spots, distinguishable from other regions of the protein due to their concave topol. combined with a pattern of hydrophobic and polar functionality.  This combination of properties confers on the hot spots a tendency to bind org. species possessing some polar groups decorating largely hydrophobic scaffolds.  Thus, druggable sites at PPI are not simply sites that are complementary to particular org. functionality, but rather possess a general tendency to bind org. compds. with a variety of structures, including key side chains of the partner protein.  Results also highlight the importance of conformational adaptivity at the binding site to allow the hot spots to expand to accommodate a ligand of drug-like dimensions.  The crit. components of this adaptivity are largely local, involving primarily low energy side-chain motions within 6 Å of a hot spot.  The structural and physicochem. signature of druggable sites at PPI interfaces is sufficiently robust to be detectable from the structure of the unliganded protein, even when substantial conformational adaptation is required for optimal ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpibu1osEQcZrVg90H21EOLACvtfcHk0liQgQUnPKfufw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2ms7rL&md5=fc9b8a30f1b30c97bba5a843fb9dc03b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1101835108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1101835108%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DCencic%26aufirst%3DR.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DPelletier%26aufirst%3DJ.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DStructural%2520conservation%2520of%2520druggable%2520hot%2520spots%2520in%2520protein-protein%2520interfaces%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13528%26epage%3D33%26doi%3D10.1073%2Fpnas.1101835108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Hall, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yueh, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Robust identification of binding hot spots using continuum electrostatics: Application to hen egg-white lysozyme</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">20668</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1021/ja207914y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja207914y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=20668-71&author=D.+H.+Hallauthor=L.+E.+Groveauthor=C.+Yuehauthor=C.+H.+Nganauthor=D.+Kozakovauthor=S.+Vajda&title=Robust+identification+of+binding+hot+spots+using+continuum+electrostatics%3A+Application+to+hen+egg-white+lysozyme&doi=10.1021%2Fja207914y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja207914y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja207914y%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BH.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DYueh%26aufirst%3DC.%26aulast%3DNgan%26aufirst%3DC.%2BH.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DRobust%2520identification%2520of%2520binding%2520hot%2520spots%2520using%2520continuum%2520electrostatics%253A%2520Application%2520to%2520hen%2520egg-white%2520lysozyme%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D20668%26epage%3D71%26doi%3D10.1021%2Fja207914y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zerbe, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Hot spot analysis for driving the development of hits into leads in fragment-based drug discovery</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1021/ci200468p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200468p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFygt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=199-209&author=D.+R.+Hallauthor=C.+H.+Nganauthor=B.+S.+Zerbeauthor=D.+Kozakovauthor=S.+Vajda&title=Hot+spot+analysis+for+driving+the+development+of+hits+into+leads+in+fragment-based+drug+discovery&doi=10.1021%2Fci200468p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Hot Spot Analysis for Driving the Development of Hits into Leads in Fragment-Based Drug Discovery</span></div><div class="casAuthors">Hall, David R.; Ngan, Chi Ho; Zerbe, Brandon S.; Kozakov, Dima; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">199-209</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug design (FBDD) starts with finding fragment-sized compds. that are highly ligand efficient and can serve as a core moiety for developing high-affinity leads.  Although the core-bound structure of a protein facilitates the construction of leads, effective design is far from straightforward.  We show that protein mapping, a computational method developed to find binding hot spots and implemented as the FTMap server, provides information that complements the fragment screening results and can drive the evolution of core fragments into larger leads with a minimal loss or, in some cases, even a gain in ligand efficiency.  The method places small mol. probes, the size of org. solvents, on a dense grid around the protein and identifies the hot spots as consensus clusters formed by clusters of several probes.  The hot spots are ranked based on the no. of probe clusters, which predicts the binding propensity of the subsites and hence their importance for drug design.  Accordingly, with a single exception the main hot spot identified by FTMap binds the core compd. found by fragment screening.  The most useful information is provided by the neighboring secondary hot spots, indicating the regions where the core can be extended to increase its affinity.  To quantify this information, we calc. the d. of probes from mapping, which describes the binding propensity at each point, and show that the change in the correlation between a ligand position and the probe d. upon extending or repositioning the core moiety predicts the expected change in ligand efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoahb_AwEPwybVg90H21EOLACvtfcHk0lhEcRe_2q5AyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFygt7vK&md5=674174e6634740d828576aeed2424232</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fci200468p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200468p%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DNgan%26aufirst%3DC.%2BH.%26aulast%3DZerbe%26aufirst%3DB.%2BS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DHot%2520spot%2520analysis%2520for%2520driving%2520the%2520development%2520of%2520hits%2520into%2520leads%2520in%2520fragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D199%26epage%3D209%26doi%3D10.1021%2Fci200468p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Golden, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sayeg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zerbe, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villar, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sazinsky, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiadis, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">Comprehensive experimental and computational analysis of binding energy hot spots at the NF-κB essential modulator/IKKβ protein-protein interface</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">6242</span><span class="NLM_x">–</span> <span class="NLM_lpage">56</span><span class="refDoi"> DOI: 10.1021/ja400914z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja400914z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFKnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6242-56&author=M.+S.+Goldenauthor=S.+M.+Coteauthor=M.+Sayegauthor=B.+S.+Zerbeauthor=E.+A.+Villarauthor=D.+Beglovauthor=S.+L.+Sazinskyauthor=R.+M.+Georgiadisauthor=S.+Vajdaauthor=D.+Kozakovauthor=A.+Whitty&title=Comprehensive+experimental+and+computational+analysis+of+binding+energy+hot+spots+at+the+NF-%CE%BAB+essential+modulator%2FIKK%CE%B2+protein-protein+interface&doi=10.1021%2Fja400914z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Experimental and Computational Analysis of Binding Energy Hot Spots at the NF-κB Essential Modulator/IKKβ Protein-Protein Interface</span></div><div class="casAuthors">Golden, Mary S.; Cote, Shaun M.; Sayeg, Marianna; Zerbe, Brandon S.; Villar, Elizabeth A.; Beglov, Dmitri; Sazinsky, Stephen L.; Georgiadis, Rosina M.; Vajda, Sandor; Kozakov, Dima; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6242-6256</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a comprehensive anal. of binding energy hot spots at the protein-protein interaction (PPI) interface between nuclear factor kappa B (NF-κB) essential modulator (NEMO) and IκB kinase subunit β (IKKβ), an interaction that is crit. for NF-κB pathway signaling, using exptl. alanine scanning mutagenesis and also the FTMap method for computational fragment screening.  The exptl. results confirm that the previously identified NEMO binding domain (NBD) region of IKKβ contains the highest concn. of hot-spot residues, the strongest of which are W739, W741, and L742 (ΔΔG = 4.3, 3.5, and 3.2 kcal/mol, resp.).  The region occupied by these residues defines a potentially druggable binding site on NEMO that extends for ∼16 Å to addnl. include the regions that bind IKKβ L737 and F734.  NBD residues D738 and S740 are also important for binding but do not make direct contact with NEMO, instead likely acting to stabilize the active conformation of surrounding residues.  We addnl. found two previously unknown hot-spot regions centered on IKKβ residues L708/V709 and L719/I723.  The computational approach successfully identified all three hot-spot regions on IKKβ.  Moreover, the method was able to accurately quantify the energetic importance of all hot-spot residues involving direct contact with NEMO.  Our results provide new information to guide the discovery of small-mol. inhibitors that target the NEMO/IKKβ interaction.  They addnl. clarify the structural and energetic complementarity between "pocket-forming" and "pocket-occupying" hot-spot residues, and further validate computational fragment mapping as a method for identifying hot spots at PPI interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLvGilMwRJGrVg90H21EOLACvtfcHk0lhEcRe_2q5AyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFKnsL0%253D&md5=d996a1aef7668ff03fe9bf3445c7d700</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fja400914z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja400914z%26sid%3Dliteratum%253Aachs%26aulast%3DGolden%26aufirst%3DM.%2BS.%26aulast%3DCote%26aufirst%3DS.%2BM.%26aulast%3DSayeg%26aufirst%3DM.%26aulast%3DZerbe%26aufirst%3DB.%2BS.%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DSazinsky%26aufirst%3DS.%2BL.%26aulast%3DGeorgiadis%26aufirst%3DR.%2BM.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DComprehensive%2520experimental%2520and%2520computational%2520analysis%2520of%2520binding%2520energy%2520hot%2520spots%2520at%2520the%2520NF-%25CE%25BAB%2520essential%2520modulator%252FIKK%25CE%25B2%2520protein-protein%2520interface%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6242%26epage%3D56%26doi%3D10.1021%2Fja400914z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mottarella, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span><span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-55&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0lhEcRe_2q5AyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D55%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Liepinsh, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otting, G.</span><span> </span><span class="NLM_article-title">Organic solvents identify specific ligand binding sites on protein surfaces</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">264</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1038/nbt0397-264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnbt0397-264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=9062927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK2sXhslKhsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=264-8&author=E.+Liepinshauthor=G.+Otting&title=Organic+solvents+identify+specific+ligand+binding+sites+on+protein+surfaces&doi=10.1038%2Fnbt0397-264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Organic solvents identify specific ligand-binding sites on protein surfaces</span></div><div class="casAuthors">Liepinsh, Edvards; Otting, Gottfried</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">264-268</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">Enzymes frequently recognize substrates and pharmaceutical drugs through specific binding interactions in deep pockets on the protein surface.  We show how the specificity-detg. substrate-binding site of hen egg-white lysozyme (HEWL) can be readily identified in aq. soln. by NMR spectroscopy using small org. solvent mols. as detection probes.  Exchange of magnetization between the 1H nuclei of the protein and the ligands through dipole-dipole interactions is obsd. which allows the modeling of their position and orientation at the binding site.  Combined with site-specific binding consts. measured by titrn. expts. with different org. solvents, the method can provide important information for rational drug design.  In addn., the lifetime of nonspecific interactions of HEWL with org. solvents is shown to be in the sub-nanosecond time range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqafHPYYxoVVrVg90H21EOLACvtfcHk0lhw2IWLixIJ1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhslKhsr4%253D&md5=912425d4c7357f6cf9f8bd4492c6fee7</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnbt0397-264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0397-264%26sid%3Dliteratum%253Aachs%26aulast%3DLiepinsh%26aufirst%3DE.%26aulast%3DOtting%26aufirst%3DG.%26atitle%3DOrganic%2520solvents%2520identify%2520specific%2520ligand%2520binding%2520sites%2520on%2520protein%2520surfaces%26jtitle%3DNat.%2520Biotechnol.%26date%3D1997%26volume%3D15%26spage%3D264%26epage%3D8%26doi%3D10.1038%2Fnbt0397-264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellamacina, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peisach, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vitkup, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Multiple solvent crystal structures: Probing binding sites, plasticity and hydration</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">1471</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/j.jmb.2006.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jmb.2006.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16488429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVGqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2006&pages=1471-82&author=C.+Mattosauthor=C.+R.+Bellamacinaauthor=E.+Peisachauthor=A.+Pereiraauthor=D.+Vitkupauthor=G.+A.+Petskoauthor=D.+Ringe&title=Multiple+solvent+crystal+structures%3A+Probing+binding+sites%2C+plasticity+and+hydration&doi=10.1016%2Fj.jmb.2006.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Solvent Crystal Structures: Probing Binding Sites, Plasticity and Hydration</span></div><div class="casAuthors">Mattos, Carla; Bellamacina, Cornelia R.; Peisach, Ezra; Pereira, Antonio; Vitkup, Dennis; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1471-1482</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Multiple solvent crystal structures (MSCS) of porcine pancreatic elastase were used to map the binding surface the enzyme.  Crystal structures of elastase in neat acetonitrile, 95% acetone, 55% DMF, 80% 5-hexene-1,2-diol, 80% isopropanol, 80% ethanol and 40% trifluoroethanol showed that the org. solvent mols. clustered in the active site, were found mostly unclustered in crystal contacts and in general did not bind elsewhere on the surface of elastase.  Mixts. of 40% benzene or 40% cyclohexane in 50% isopropanol and 10% water showed no bound benzene or cyclohexane mols., but did reveal bound isopropanol. The clusters of org. solvent probe mols. coincide with pockets occupied by known inhibitors.  MSCS also reveal the areas of plasticity within the elastase binding site and allow for the visualization of a nearly complete first hydration shell.  The pattern of org. solvent clusters detd. by MSCS for elastase is consistent with patterns for hot spots in protein-ligand interactions detd. from database anal. in general.  The MSCS method allows probing of hot spots, plasticity and hydration simultaneously, providing a powerful complementary strategy to guide computational methods currently in development for binding site detn., ligand docking and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq66xX3ZT7wRLVg90H21EOLACvtfcHk0lhw2IWLixIJ1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVGqtrk%253D&md5=ab7064d371e5557b2f91aeb6c262a5dc</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DBellamacina%26aufirst%3DC.%2BR.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DPereira%26aufirst%3DA.%26aulast%3DVitkup%26aufirst%3DD.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DMultiple%2520solvent%2520crystal%2520structures%253A%2520Probing%2520binding%2520sites%252C%2520plasticity%2520and%2520hydration%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D357%26spage%3D1471%26epage%3D82%26doi%3D10.1016%2Fj.jmb.2006.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-42&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lhw2IWLixIJ1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D42%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">The rise of fragment-based drug discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span><span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+rise+of+fragment-based+drug+discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0lhw2IWLixIJ1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520rise%2520of%2520fragment-based%2520drug%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">64</span><span class="refDoi"> DOI: 10.1038/nchembio.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchembio.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19305397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=358-64&author=Y.+Chenauthor=B.+K.+Shoichet&title=Molecular+docking+and+ligand+specificity+in+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span></div><div class="casAuthors">Chen, Yu; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment screens have successfully identified new scaffolds in drug discovery, often with relatively high hit rates (5%) using small screening libraries (1,000-10,000 compds.).  This raises two questions: would other noteworthy chemotypes be found were one to screen all com. available fragments (>300,000), and does the success rate imply low specificity of fragments.  We used mol. docking to screen large libraries of fragments against CTX-M β-lactamase.  We identified ten millimolar-range inhibitors from the 69 compds. tested.  The docking poses corresponded closely to the crystallog. structures subsequently detd.  Notably, these initial low-affinity hits showed little specificity between CTX-M and an unrelated β-lactamase, AmpC, which is unusual among β-lactamase inhibitors.  This is consistent with the idea that the high hit rates among fragments correlate to a low initial specificity.  As the inhibitors were progressed, both specificity and affinity rose together, yielding to our knowledge the first micromolar-range noncovalent inhibitors against a class A β-lactamase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8zTkq9kMAbVg90H21EOLACvtfcHk0liGNDMq4EkqrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D&md5=6ee8dd19d0b99599ea74eaf735483b1d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.155%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520ligand%2520specificity%2520in%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D358%26epage%3D64%26doi%3D10.1038%2Fnchembio.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Warr, W. A.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">453</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1007/s10822-009-9292-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs10822-009-9292-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=453-8&author=W.+A.+Warr&title=Fragment-based+drug+discovery&doi=10.1007%2Fs10822-009-9292-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9292-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9292-1%26sid%3Dliteratum%253Aachs%26aulast%3DWarr%26aufirst%3DW.%2BA.%26atitle%3DFragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D453%26epage%3D8%26doi%3D10.1007%2Fs10822-009-9292-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tisi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span> </span><span class="NLM_article-title">Recent developments in fragment-based drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3661</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1021/jm8000373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8000373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFKisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3661-80&author=M.+Congreveauthor=G.+Chessariauthor=D.+Tisiauthor=A.+J.+Woodhead&title=Recent+developments+in+fragment-based+drug+discovery&doi=10.1021%2Fjm8000373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Recent developments in fragment-based drug discovery</span></div><div class="casAuthors">Congreve, Miles; Chessari, Gianni; Tisi, Dominic; Woodhead, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3661-3680</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The field of fragment-based drug discovery (FBDD) has developed significantly over the past 10 years and is now recognized as a tangible alternative to more traditional methods of hit identification, such as high throughput screening (HTS).  The authors discuss trends and developments of FBDD over the past 4 years.  Addnl., this review looks in some detail at representative protein-ligand complexes obsd. between fragment-sized mols. and their protein targets from the Protein Data Bank (PDB).  More than the underlying technol. itself, the legacy of FBDD will be conceptual and that "less is more"; breaking down the problem of finding a drug mol. into its smallest component parts opens up new opportunities to solve challenging problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_YtOS7_glD7Vg90H21EOLACvtfcHk0liGNDMq4EkqrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFKisLg%253D&md5=41dbcf884305a190bffc683ae5ed9e45</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm8000373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8000373%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DTisi%26aufirst%3DD.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26atitle%3DRecent%2520developments%2520in%2520fragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3661%26epage%3D80%26doi%3D10.1021%2Fjm8000373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Carr, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery: Leads by design</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03511-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS1359-6446%2805%2903511-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16023057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=987-92&author=R.+A.+Carrauthor=M.+Congreveauthor=C.+W.+Murrayauthor=D.+C.+Rees&title=Fragment-based+lead+discovery%3A+Leads+by+design&doi=10.1016%2FS1359-6446%2805%2903511-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery: leads by design</span></div><div class="casAuthors">Carr, Robin A. E.; Congreve, Miles; Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">987-992</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery (also referred to as needles, shapes, binding elements, seed templates or scaffolds) is a new lead discovery approach in which much lower mol. wt. (120-250Da) compds. are screened relative to HTS campaigns.  Fragment-based hits are typically weak inhibitors (10μM-mM), and therefore need to be screened at higher concn. using very sensitive biophys. detection techniques such as protein crystallog. and NMR as the primary screening techniques, rather than bioassays.  Compared with HTS hits, these fragments are simpler, less functionalized compds. with correspondingly lower affinity.  However, fragment hits typically possess high ligand efficiency' (binding affinity per heavy atom) and so are highly suitable for optimization into clin. candidates with good drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq15FNhjT6Qw7Vg90H21EOLACvtfcHk0liGNDMq4EkqrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D&md5=1bf7dd97ec056af9ed5143f96b34bb53</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903511-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903511-7%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DR.%2BA.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-based%2520lead%2520discovery%253A%2520Leads%2520by%2520design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D987%26epage%3D92%26doi%3D10.1016%2FS1359-6446%2805%2903511-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Villar, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chennamadhavuni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porco, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">How proteins bind macrocycles</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">723</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-31&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lhPnI8b5LOMJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D31%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Ganesan, A.</span><span> </span><span class="NLM_article-title">The impact of natural products upon modern drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">306</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2008.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.cbpa.2008.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18423384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ersrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=306-17&author=A.+Ganesan&title=The+impact+of+natural+products+upon+modern+drug+discovery&doi=10.1016%2Fj.cbpa.2008.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of natural products upon modern drug discovery</span></div><div class="casAuthors">Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">306-317</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In the period 1970-2006, a total of 24 unique natural products were discovered that led to an approved drug.  We analyze these successful leads in terms of drug-like properties, and show that they can be divided into two equal subsets.  The first falls in the Lipinski universe' and complies with the Rule of Five.  The second is a parallel universe' that violates the rules.  Nevertheless, the latter compds. remain largely compliant in terms of log P and H-bond donors, highlighting the importance of these two metrics in predicting bioavailability.  Natural products are often cited as an exception to Lipinski's rules.  We believe this is because nature has learned to maintain low hydrophobicity and intermol. H-bond donating potential when it needs to make biol. active compds. with high mol. wt. and large nos. of rotatable bonds.  In addn., natural products are more likely than purely synthetic compds. to resemble biosynthetic intermediates or endogenous metabolites, and hence take advantage of active transport mechanisms.  Interestingly, the natural product leads in the Lipinski and parallel universe had an identical success rate (50%) in delivering an oral drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXd8x0Vy0yerVg90H21EOLACvtfcHk0lhPnI8b5LOMJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ersrk%253D&md5=8cdde276c59d84071ea2d6f572d10088</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DThe%2520impact%2520of%2520natural%2520products%2520upon%2520modern%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26spage%3D306%26epage%3D17%26doi%3D10.1016%2Fj.cbpa.2008.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Goodman, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shandler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeGrado, W. F.</span><span> </span><span class="NLM_article-title">Foldamers as versatile frameworks for the design and evolution of function</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1038/nchembio876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchembio876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17438550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1Kjs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=252-262&author=C.+M.+Goodmanauthor=S.+Choiauthor=S.+Shandlerauthor=W.+F.+DeGrado&title=Foldamers+as+versatile+frameworks+for+the+design+and+evolution+of+function&doi=10.1038%2Fnchembio876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Foldamers as versatile frameworks for the design and evolution of function</span></div><div class="casAuthors">Goodman, Catherine M.; Choi, Sungwook; Shandler, Scott; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">252-262</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Foldamers are sequence-specific oligomers akin to peptides, proteins and oligonucleotides that fold into well-defined three-dimensional structures.  They offer the chem. biologist a broad pallet of building blocks for the construction of mols. that test and extend our understanding of protein folding and function.  Foldamers also provide templates for presenting complex arrays of functional groups in virtually unlimited geometrical patterns, thereby presenting attractive opportunities for the design of mols. that bind in a sequence- and structure-specific manner to oligosaccharides, nucleic acids, membranes and proteins.  We summarize recent advances and highlight the future applications and challenges of this rapidly expanding field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLkvabxBqULVg90H21EOLACvtfcHk0lhPnI8b5LOMJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1Kjs7w%253D&md5=99f0e4b2bb7a8457a586c74f0f98dbb3</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnchembio876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio876%26sid%3Dliteratum%253Aachs%26aulast%3DGoodman%26aufirst%3DC.%2BM.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DShandler%26aufirst%3DS.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DFoldamers%2520as%2520versatile%2520frameworks%2520for%2520the%2520design%2520and%2520evolution%2520of%2520function%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D252%26epage%3D262%26doi%3D10.1038%2Fnchembio876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Zuckermann, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodadek, T.</span><span> </span><span class="NLM_article-title">Peptoids as potential therapeutics</span> <span class="citation_source-journal">Curr. Opin. Mol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19479663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnt1Kgu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=299-307&author=R.+N.+Zuckermannauthor=T.+Kodadek&title=Peptoids+as+potential+therapeutics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Peptoids as potential therapeutics</span></div><div class="casAuthors">Zuckermann, Ronald N.; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Molecular Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">299-307</span>CODEN:
                <span class="NLM_cas:coden">CUOTFO</span>;
        ISSN:<span class="NLM_cas:issn">2040-3445</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptoids are oligomers of N-substituted glycine units.  These mols. are almost perfectly suited for combinatorial approaches to drug discovery because large libraries can be synthesized easily from readily available primary amines.  Moreover, major advances in screening methodol. have allowed peptoid libraries of hundreds of thousands of compds. to be mined inexpensively and quickly for highly specific protein-binding mols.  These advances and the potential utility of peptoids as pharmacol. agents are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-Fv-86dP_ubVg90H21EOLACvtfcHk0li13ty2tF00AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnt1Kgu7k%253D&md5=fce62a95317bf10116f7ed5ab8754b88</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZuckermann%26aufirst%3DR.%2BN.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DPeptoids%2520as%2520potential%2520therapeutics%26jtitle%3DCurr.%2520Opin.%2520Mol.%2520Ther.%26date%3D2009%26volume%3D11%26spage%3D299%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Dietrich, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durr, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, J.</span><span> </span><span class="NLM_article-title">Peptides as drugs: From screening to application</span> <span class="citation_source-journal">Curr. Pharm. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">501</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.2174/13892010113149990205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.2174%2F13892010113149990205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22429133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlGgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=501-12&author=U.+Dietrichauthor=R.+Durrauthor=J.+Koch&title=Peptides+as+drugs%3A+From+screening+to+application&doi=10.2174%2F13892010113149990205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Peptides as drugs: from screening to application</span></div><div class="casAuthors">Dietrich, Ursula; Duerr, Ralf; Koch, Joachim</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Biotechnology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">501-512</span>CODEN:
                <span class="NLM_cas:coden">CPBUBP</span>;
        ISSN:<span class="NLM_cas:issn">1389-2010</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptides are ideally suited to mimic natural ligands and thereby function in an antagonistic or agonistic way.  Furthermore they are able to physiol. disrupt functional complexes due to their small size and specific binding properties.  Proteins form homo- or heteromeric (macro)mol. complexes and intricate networks by interacting with small mols., peptides, nucleic acids or other proteins.  On av., five interaction partners have been estd. for any given cellular protein, illustrating the complexity of the formed "interactomes" and the impact of their investigation.  Many protein-protein interactions are mediated by hot-spots, which comprise only a small part of the large binding interface but account for 80% of the binding energy.  Thus, these hot spots provide an "Achilles heel" for pharmaceutical interventions aiming at the disruption of functional protein-protein complexes.  Methods to select peptides for defined target structures include display technologies on phages, ribosomes or yeast, and the yeast-two-hybrid system.  Once selected, these peptides can be optimized for their binding affinity using peptide arrays.  Stabilization of biol. unstable peptides is achieved by the introduction of non-natural amino acids to form so-called peptidomimetics that are resistant to cellular proteases.  Moreover, lipocalins and peptide aptamers represent scaffolded binding structures with unique binding characteristics and enhanced stability.  In case of extracellular targets, like cell surface receptors or pathogens in patients' plasma, peptide inhibitors have direct access.  Addressing intracellular targets with peptides is more difficult since short hydrophilic peptides generally do not cross plasma membranes on their own.  However, intracellular uptake of peptides can be achieved by coupling to carrier systems like liposomes or nanoparticles or upon fusion to a protein transduction domain.  Alternatively, peptides may be expressed within cells after transduction with viral vectors in a gene therapy setting.  This review will summarize the broad potential of peptides as drugs, with special emphasis on peptides which inhibit protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr10DvrrcgEhrVg90H21EOLACvtfcHk0li13ty2tF00AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlGgtA%253D%253D&md5=b63d94225b8fa4a7fc749b8631794544</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.2174%2F13892010113149990205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13892010113149990205%26sid%3Dliteratum%253Aachs%26aulast%3DDietrich%26aufirst%3DU.%26aulast%3DDurr%26aufirst%3DR.%26aulast%3DKoch%26aufirst%3DJ.%26atitle%3DPeptides%2520as%2520drugs%253A%2520From%2520screening%2520to%2520application%26jtitle%3DCurr.%2520Pharm.%2520Biotechnol.%26date%3D2013%26volume%3D14%26spage%3D501%26epage%3D12%26doi%3D10.2174%2F13892010113149990205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Wilson, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sintchak, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotstein, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgibbon, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navia, M. A.</span><span> </span><span class="NLM_article-title">Comparative x-ray structures of the major binding-protein for the immunosuppressant FK506 (tacrolimus) in unliganded form and in complex with FK506 and rapamycin</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444994014514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1107%2FS0907444994014514" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1995&pages=511-21&author=K.+P.+Wilsonauthor=M.+M.+Yamashitaauthor=M.+D.+Sintchakauthor=S.+H.+Rotsteinauthor=M.+A.+Murckoauthor=J.+Bogerauthor=J.+A.+Thomsonauthor=M.+J.+Fitzgibbonauthor=J.+R.+Blackauthor=M.+A.+Navia&title=Comparative+x-ray+structures+of+the+major+binding-protein+for+the+immunosuppressant+FK506+%28tacrolimus%29+in+unliganded+form+and+in+complex+with+FK506+and+rapamycin&doi=10.1107%2FS0907444994014514"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444994014514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994014514%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DK.%2BP.%26aulast%3DYamashita%26aufirst%3DM.%2BM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DRotstein%26aufirst%3DS.%2BH.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26aulast%3DBoger%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DJ.%2BA.%26aulast%3DFitzgibbon%26aufirst%3DM.%2BJ.%26aulast%3DBlack%26aufirst%3DJ.%2BR.%26aulast%3DNavia%26aufirst%3DM.%2BA.%26atitle%3DComparative%2520x-ray%2520structures%2520of%2520the%2520major%2520binding-protein%2520for%2520the%2520immunosuppressant%2520FK506%2520%2528tacrolimus%2529%2520in%2520unliganded%2520form%2520and%2520in%2520complex%2520with%2520FK506%2520and%2520rapamycin%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1995%26volume%3D51%26spage%3D511%26epage%3D21%26doi%3D10.1107%2FS0907444994014514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Sattler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettesheim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harlan, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberstadt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minn, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Structure of Bcl-x(L)-BAK peptide complex: Recognition between regulators of apoptosis</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1126/science.275.5302.983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1126%2Fscience.275.5302.983" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=1997&pages=983-6&author=M.+Sattlerauthor=H.+Liangauthor=D.+Nettesheimauthor=R.+P.+Meadowsauthor=J.+E.+Harlanauthor=M.+Eberstadtauthor=H.+S.+Yoonauthor=S.+B.+Shukerauthor=B.+S.+Changauthor=A.+J.+Minnauthor=C.+B.+Thompsonauthor=S.+W.+Fesik&title=Structure+of+Bcl-x%28L%29-BAK+peptide+complex%3A+Recognition+between+regulators+of+apoptosis&doi=10.1126%2Fscience.275.5302.983"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1126%2Fscience.275.5302.983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.275.5302.983%26sid%3Dliteratum%253Aachs%26aulast%3DSattler%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DNettesheim%26aufirst%3DD.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DHarlan%26aufirst%3DJ.%2BE.%26aulast%3DEberstadt%26aufirst%3DM.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DChang%26aufirst%3DB.%2BS.%26aulast%3DMinn%26aufirst%3DA.%2BJ.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DStructure%2520of%2520Bcl-x%2528L%2529-BAK%2520peptide%2520complex%253A%2520Recognition%2520between%2520regulators%2520of%2520apoptosis%26jtitle%3DScience%26date%3D1997%26volume%3D275%26spage%3D983%26epage%3D6%26doi%3D10.1126%2Fscience.275.5302.983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Holt, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luengo, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konialian, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozamus, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bossard, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggleston, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stout, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clardy, J.</span><span> </span><span class="NLM_article-title">Design, synthesis, and kinetic evaluation of high-affinity FKBP ligands and the X-ray crystal-structures of their complexes with FKBP12</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">9925</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1021/ja00075a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00075a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1993&pages=9925-38&author=D.+A.+Holtauthor=J.+I.+Luengoauthor=D.+S.+Yamashitaauthor=H.+J.+Ohauthor=A.+L.+Konialianauthor=H.+K.+Yenauthor=L.+W.+Rozamusauthor=M.+Brandtauthor=M.+J.+Bossardauthor=M.+A.+Levyauthor=D.+S.+Egglestonauthor=J.+Liangauthor=L.+W.+Schultzauthor=T.+J.+Stoutauthor=J.+Clardy&title=Design%2C+synthesis%2C+and+kinetic+evaluation+of+high-affinity+FKBP+ligands+and+the+X-ray+crystal-structures+of+their+complexes+with+FKBP12&doi=10.1021%2Fja00075a008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fja00075a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00075a008%26sid%3Dliteratum%253Aachs%26aulast%3DHolt%26aufirst%3DD.%2BA.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DOh%26aufirst%3DH.%2BJ.%26aulast%3DKonialian%26aufirst%3DA.%2BL.%26aulast%3DYen%26aufirst%3DH.%2BK.%26aulast%3DRozamus%26aufirst%3DL.%2BW.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DBossard%26aufirst%3DM.%2BJ.%26aulast%3DLevy%26aufirst%3DM.%2BA.%26aulast%3DEggleston%26aufirst%3DD.%2BS.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DSchultz%26aufirst%3DL.%2BW.%26aulast%3DStout%26aufirst%3DT.%2BJ.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520kinetic%2520evaluation%2520of%2520high-affinity%2520FKBP%2520ligands%2520and%2520the%2520X-ray%2520crystal-structures%2520of%2520their%2520complexes%2520with%2520FKBP12%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1993%26volume%3D115%26spage%3D9925%26epage%3D38%26doi%3D10.1021%2Fja00075a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Sich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Improta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowley, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guenet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merly, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teufel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saudek, V.</span><span> </span><span class="NLM_article-title">Solution structure of a neurotrophic ligand bound to FKBP12 and its effects on protein dynamics</span> <span class="citation_source-journal">Eur. J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">5342</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span><span class="refDoi"> DOI: 10.1046/j.1432-1327.2000.01551.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1046%2Fj.1432-1327.2000.01551.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10951192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsVens7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=2000&pages=5342-54&author=C.+Sichauthor=S.+Improtaauthor=D.+J.+Cowleyauthor=C.+Guenetauthor=J.+P.+Merlyauthor=M.+Teufelauthor=V.+Saudek&title=Solution+structure+of+a+neurotrophic+ligand+bound+to+FKBP12+and+its+effects+on+protein+dynamics&doi=10.1046%2Fj.1432-1327.2000.01551.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Solution structure of a neurotrophic ligand bound to FKBP12 and its effects on protein dynamics</span></div><div class="casAuthors">Sich, Christian; Improta, Sabina; Cowley, David John; Guenet, Chantal; Merly, Jean-Pierre; Teufel, Michael; Saudek, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5342-5354</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">The structure of a recently reported neurotrophic ligand, 3-(3-pyridyl)-1-propyl(2S)-1-(3,3-dimethyl-1,2-dioxopentyl)-2-pyrrolidinecarboxylate, in complex with FKBP12 was detd. using heteronuclear NMR spectroscopy.  The inhibitor exhibits a binding mode analogous to that obsd. for the macrocycle FK506, used widely as an immunosuppressant, with the prolyl ring replacing the pipecolyl moiety and the amide bond in a trans conformation.  However, fewer favorable protein-ligand interactions are detected in the structure of the complex, suggesting weaker binding compared with the immunosuppressant drug.  Indeed, a micromolar dissocn. const. was estd. from the NMR ligand titrn. profile, in contrast to the previously published nanomolar inhibition activity.  Although the inhibitor possesses a remarkable structural simplicity with respect to FK506, 15N relaxation studies show that it induces similar effects on the protein dynamics, stabilizing the conformation of solvent-exposed residues which are important for mediating the interaction of immunophilin/ligand complexes with mol. targets and potentially for the transmission of the neurotrophic action of FKBP12 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ47HUpCawurVg90H21EOLACvtfcHk0ljSt2PgDphKzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsVens7Y%253D&md5=3a97a261948a25f3e9b62cd389788863</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1046%2Fj.1432-1327.2000.01551.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1432-1327.2000.01551.x%26sid%3Dliteratum%253Aachs%26aulast%3DSich%26aufirst%3DC.%26aulast%3DImprota%26aufirst%3DS.%26aulast%3DCowley%26aufirst%3DD.%2BJ.%26aulast%3DGuenet%26aufirst%3DC.%26aulast%3DMerly%26aufirst%3DJ.%2BP.%26aulast%3DTeufel%26aufirst%3DM.%26aulast%3DSaudek%26aufirst%3DV.%26atitle%3DSolution%2520structure%2520of%2520a%2520neurotrophic%2520ligand%2520bound%2520to%2520FKBP12%2520and%2520its%2520effects%2520on%2520protein%2520dynamics%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D2000%26volume%3D267%26spage%3D5342%26epage%3D54%26doi%3D10.1046%2Fj.1432-1327.2000.01551.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Silberstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kortvelyesi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clodfelter, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Identification of substrate binding sites in enzymes by computational solvent mapping</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">1095</span><span class="NLM_x">–</span> <span class="NLM_lpage">113</span><span class="refDoi"> DOI: 10.1016/j.jmb.2003.08.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.jmb.2003.08.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2003&pages=1095-113&author=M.+Silbersteinauthor=S.+Dennisauthor=L.+Brownauthor=T.+Kortvelyesiauthor=K.+Clodfelterauthor=S.+Vajda&title=Identification+of+substrate+binding+sites+in+enzymes+by+computational+solvent+mapping&doi=10.1016%2Fj.jmb.2003.08.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2003.08.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2003.08.019%26sid%3Dliteratum%253Aachs%26aulast%3DSilberstein%26aufirst%3DM.%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DL.%26aulast%3DKortvelyesi%26aufirst%3DT.%26aulast%3DClodfelter%26aufirst%3DK.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520substrate%2520binding%2520sites%2520in%2520enzymes%2520by%2520computational%2520solvent%2520mapping%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D332%26spage%3D1095%26epage%3D113%26doi%3D10.1016%2Fj.jmb.2003.08.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Evidence of conformational selection driving the formation of ligand binding sites in protein-protein interfaces</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003872</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003872</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1371%2Fjournal.pcbi.1003872" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003872&author=T.+Bohnuudauthor=D.+Kozakovauthor=S.+Vajda&title=Evidence+of+conformational+selection+driving+the+formation+of+ligand+binding+sites+in+protein-protein+interfaces&doi=10.1371%2Fjournal.pcbi.1003872"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003872%26sid%3Dliteratum%253Aachs%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DEvidence%2520of%2520conformational%2520selection%2520driving%2520the%2520formation%2520of%2520ligand%2520binding%2520sites%2520in%2520protein-protein%2520interfaces%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003872%26doi%3D10.1371%2Fjournal.pcbi.1003872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2977</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1021/jm030580l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030580l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2977-80&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=S.+Wang&title=The+PDBbind+database%3A+Collection+of+binding+affinities+for+protein-ligand+complexes+with+known+three-dimensional+structures&doi=10.1021%2Fjm030580l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span></div><div class="casAuthors">Wang, Renxiao; Fang, Xueliang; Lu, Yipin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2977-2980</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have screened the entire Protein Data Bank (Release No. 103, Jan. 2003) and identified 5671 protein-ligand complexes out of 19 621 exptl. structures.  A systematic examn. of the primary refs. of these entries has led to a collection of binding affinity data (Kd, Ki, and IC50) for a total of 1359 complexes.  The outcomes of this project have been organized into a Web-accessible database named the PDBbind database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1eUdeWioYBbVg90H21EOLACvtfcHk0ljSt2PgDphKzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D&md5=86e609172307402d8b0d4589b1270a2f</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjm030580l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030580l%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520Collection%2520of%2520binding%2520affinities%2520for%2520protein-ligand%2520complexes%2520with%2520known%2520three-dimensional%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2977%26epage%3D80%26doi%3D10.1021%2Fjm030580l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorissen, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">BindingDB: A web-accessible database of experimentally determined protein-ligand binding affinities</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">D198</span><span class="NLM_x">–</span> <span class="NLM_lpage">201</span><span class="refDoi"> DOI: 10.1093/nar/gkl999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fnar%2Fgkl999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17145705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+A+web-accessible+database+of+experimentally+determined+protein-ligand+binding+affinities&doi=10.1093%2Fnar%2Fgkl999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span></div><div class="casAuthors">Liu, Tiqing; Lin, Yuhmei; Wen, Xin; Jorissen, Robert N.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D198-D201</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BindingDB is a publicly accessible database currently contg. ∼20 000 exptl. detd. binding affinities of protein-ligand complexes, for 110 protein targets including isoforms and mutational variants, and ∼11 000 small mol. ligands.  The data are extd. from the scientific literature, data collection focusing on proteins that are drug-targets or candidate drug-targets and for which structural data are present in the Protein Data Bank.  The BindingDB website supports a range of query types, including searches by chem. structure, substructure and similarity; protein sequence; ligand and protein names; affinity ranges and mol. wt.  Data sets generated by BindingDB queries can be downloaded in the form of annotated SDfiles for further anal., or used as the basis for virtual screening of a compd. database uploaded by the user.  The data in BindingDB are linked both to structural data in the PDB via PDB IDs and chem. and sequence searches, and to the literature in PubMed via PubMed IDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoymy3G_LuG-rVg90H21EOLACvtfcHk0ljCuwesIrX5Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D&md5=0ccb20d9b9178a624d4829b5909e7ff8</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520A%2520web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein-ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3D201%26doi%3D10.1093%2Fnar%2Fgkl999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Benson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khazanov, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimcheff, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaver, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dresslar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerothin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Binding MOAD, a high-quality protein-ligand database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D674</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1093/nar/gkm911</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fnar%2Fgkm911" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18055497" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2007&pages=D674-8&author=M.+L.+Bensonauthor=R.+D.+Smithauthor=N.+A.+Khazanovauthor=B.+Dimcheffauthor=J.+Beaverauthor=P.+Dresslarauthor=J.+Nerothinauthor=H.+A.+Carlson&title=Binding+MOAD%2C+a+high-quality+protein-ligand+database&doi=10.1093%2Fnar%2Fgkm911"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm911%26sid%3Dliteratum%253Aachs%26aulast%3DBenson%26aufirst%3DM.%2BL.%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DKhazanov%26aufirst%3DN.%2BA.%26aulast%3DDimcheff%26aufirst%3DB.%26aulast%3DBeaver%26aufirst%3DJ.%26aulast%3DDresslar%26aufirst%3DP.%26aulast%3DNerothin%26aufirst%3DJ.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DBinding%2520MOAD%252C%2520a%2520high-quality%2520protein-ligand%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D36%26spage%3DD674%26epage%3D8%26doi%3D10.1093%2Fnar%2Fgkm911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Topalis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumamoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, M. F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugue, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haouz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexandre, J. A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallois-Montbrun, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alzari, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pochet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrofoglio, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deville-Bonne, D.</span><span> </span><span class="NLM_article-title">Nucleotide binding to human UMP-CMP kinase using fluorescent derivatives - a screening based on affinity for the UMP-CMP binding site</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">3704</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span><span class="refDoi"> DOI: 10.1111/j.1742-4658.2007.05902.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1111%2Fj.1742-4658.2007.05902.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=2007&pages=3704-14&author=D.+Topalisauthor=H.+Kumamotoauthor=M.+F.+A.+Velascoauthor=L.+Dugueauthor=A.+Haouzauthor=J.+A.+C.+Alexandreauthor=S.+Gallois-Montbrunauthor=P.+M.+Alzariauthor=S.+Pochetauthor=L.+A.+Agrofoglioauthor=D.+Deville-Bonne&title=Nucleotide+binding+to+human+UMP-CMP+kinase+using+fluorescent+derivatives+-+a+screening+based+on+affinity+for+the+UMP-CMP+binding+site&doi=10.1111%2Fj.1742-4658.2007.05902.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2007.05902.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2007.05902.x%26sid%3Dliteratum%253Aachs%26aulast%3DTopalis%26aufirst%3DD.%26aulast%3DKumamoto%26aufirst%3DH.%26aulast%3DVelasco%26aufirst%3DM.%2BF.%2BA.%26aulast%3DDugue%26aufirst%3DL.%26aulast%3DHaouz%26aufirst%3DA.%26aulast%3DAlexandre%26aufirst%3DJ.%2BA.%2BC.%26aulast%3DGallois-Montbrun%26aufirst%3DS.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26aulast%3DPochet%26aufirst%3DS.%26aulast%3DAgrofoglio%26aufirst%3DL.%2BA.%26aulast%3DDeville-Bonne%26aufirst%3DD.%26atitle%3DNucleotide%2520binding%2520to%2520human%2520UMP-CMP%2520kinase%2520using%2520fluorescent%2520derivatives%2520-%2520a%2520screening%2520based%2520on%2520affinity%2520for%2520the%2520UMP-CMP%2520binding%2520site%26jtitle%3DFEBS%2520J.%26date%3D2007%26volume%3D274%26spage%3D3704%26epage%3D14%26doi%3D10.1111%2Fj.1742-4658.2007.05902.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Mashhoon, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pruss, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reich, N. O.</span><span> </span><span class="NLM_article-title">Selective inhibitors of bacterial DNA adenine methyltransferases</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">497</span><span class="NLM_x">–</span> <span class="NLM_lpage">510</span><span class="refDoi"> DOI: 10.1177/1087057106287933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1177%2F1087057106287933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16760373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XpslWjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=497-510&author=N.+Mashhoonauthor=C.+Prussauthor=M.+Carrollauthor=P.+H.+Johnsonauthor=N.+O.+Reich&title=Selective+inhibitors+of+bacterial+DNA+adenine+methyltransferases&doi=10.1177%2F1087057106287933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of bacterial DNA adenine methyltransferases</span></div><div class="casAuthors">Mashhoon, Neda; Pruss, Cynthia; Carroll, Michael; Johnson, Paul H.; Reich, Norbert O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">497-510</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The authors describe the discovery and characterization of several structural classes of small-mol. inhibitors of bacterial DNA adenine methyltransferases.  These enzymes are essential for bacterial virulence (DNA adenine methyltransferase [DAM]) and cell viability (cell cycle-regulated methyltransferase [CcrM]).  Using a novel high-throughput fluorescence-based assay and recombinant DAM and CcrM, the authors screened a diverse chem. library.  They identified 5 major structural classes of inhibitors composed of more than 350 compds.: cyclopentaquinolines, Ph vinyl furans, pyrimidine-diones, thiazolidine-4-ones, and phenyl-pyrroles.  DNA binding assays were used to identify compds. that interact directly with DNA.  Potent compds. selective for the bacterial target were identified, whereas other compds. showed greater selectivity for the mammalian DNA cytosine methyltransferase, Dnmt1.  Enzyme inhibition anal. identified mechanistically distinct compds. that interfered with DNA or cofactor binding.  Selected compds. demonstrated cell-based efficacy.  These small-mol. DNA methyltransferase inhibitors provide useful reagents to probe the role of DNA methylation and may form the basis of developing novel antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob8jGoRQNSi7Vg90H21EOLACvtfcHk0ljCuwesIrX5Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpslWjtbo%253D&md5=41e230dae5dae50a1b5a4881d882be7f</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1177%2F1087057106287933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106287933%26sid%3Dliteratum%253Aachs%26aulast%3DMashhoon%26aufirst%3DN.%26aulast%3DPruss%26aufirst%3DC.%26aulast%3DCarroll%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DP.%2BH.%26aulast%3DReich%26aufirst%3DN.%2BO.%26atitle%3DSelective%2520inhibitors%2520of%2520bacterial%2520DNA%2520adenine%2520methyltransferases%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D497%26epage%3D510%26doi%3D10.1177%2F1087057106287933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Miao, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skidan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugovskoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reibarkh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brazell, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durugkar, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramana, C. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaffhausen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torchilin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degterev, A.</span><span> </span><span class="NLM_article-title">Small molecule inhibition of phosphatidylinositol-3,4,5-triphosphate (pip3) binding to pleckstrin homology domains</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">20126</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1073/pnas.1004522107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1004522107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21041639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyntL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=20126-31&author=B.+C.+Miaoauthor=I.+Skidanauthor=J.+S.+Yangauthor=A.+Lugovskoyauthor=M.+Reibarkhauthor=K.+Longauthor=T.+Brazellauthor=K.+A.+Durugkarauthor=J.+Makiauthor=C.+V.+Ramanaauthor=B.+Schaffhausenauthor=G.+Wagnerauthor=V.+Torchilinauthor=J.+Y.+Yuanauthor=A.+Degterev&title=Small+molecule+inhibition+of+phosphatidylinositol-3%2C4%2C5-triphosphate+%28pip3%29+binding+to+pleckstrin+homology+domains&doi=10.1073%2Fpnas.1004522107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of phosphatidylinositol-3,4,5-triphosphate (PIP3) binding to pleckstrin homology domains</span></div><div class="casAuthors">Miao, Benchun; Skidan, Igor; Yang, Jinsheng; Lugovskoy, Alexey; Reibarkh, Mikhail; Long, Kai; Brazell, Tres; Durugkar, Kulbhushan A.; Maki, Jenny; Ramana, C. V.; Schaffhausen, Brian; Wagner, Gerhard; Torchilin, Vladimir; Yuan, Junying; Degterev, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">20126-20131, S20126/1-S20126/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The PI3-kinase (PI3K) pathway regulates many cellular processes, esp. cell metab., cell survival, and apoptosis.  Phosphatidylinositol-3,4,5-trisphosphate (PIP3), the product of PI3K activity and a key signaling mol., acts by recruiting pleckstrin-homol. (PH) domain-contg. proteins to cell membranes.  Here, the authors describe a new structural class of nonphosphoinositide small mol. antagonists (PITenins, PITs) of PIP3-PH domain interactions (IC50 ranges from 13.4 to 31 μM in PIP3/Akt PH domain binding assay).  PITs inhibit interactions of a no. of PIP3-binding PH domains, including those of Akt and PDK1, without affecting several PIP2-selective PH domains.  As a result, PITs suppress the PI3K-PDK1-Akt pathway and trigger metabolic stress and apoptosis.  A PIT-1 analog displayed significant antitumor activity in vivo, including inhibition of tumor growth and induction of apoptosis.  Overall, the authors' studies demonstrate the feasibility of developing specific small mol. antagonists of PIP3 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq68GGEQg_fsrVg90H21EOLACvtfcHk0lhSidVLsmKtqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyntL3P&md5=b2bc440fd7c9fae24de7f6e8c4910d86</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1004522107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1004522107%26sid%3Dliteratum%253Aachs%26aulast%3DMiao%26aufirst%3DB.%2BC.%26aulast%3DSkidan%26aufirst%3DI.%26aulast%3DYang%26aufirst%3DJ.%2BS.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DReibarkh%26aufirst%3DM.%26aulast%3DLong%26aufirst%3DK.%26aulast%3DBrazell%26aufirst%3DT.%26aulast%3DDurugkar%26aufirst%3DK.%2BA.%26aulast%3DMaki%26aufirst%3DJ.%26aulast%3DRamana%26aufirst%3DC.%2BV.%26aulast%3DSchaffhausen%26aufirst%3DB.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DTorchilin%26aufirst%3DV.%26aulast%3DYuan%26aufirst%3DJ.%2BY.%26aulast%3DDegterev%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520phosphatidylinositol-3%252C4%252C5-triphosphate%2520%2528pip3%2529%2520binding%2520to%2520pleckstrin%2520homology%2520domains%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D20126%26epage%3D31%26doi%3D10.1073%2Fpnas.1004522107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Lerner, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagenaar, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodall, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Searle, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florjancic, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mack, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chovan, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCall, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black-Schaefer, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakavas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schurdak, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comess, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edalji, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorwin, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebert, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harlan, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merta, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baranowski, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornewell, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shivakumar, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saiki, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bui, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balli, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilius, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holzman, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beutel, B. A.</span><span> </span><span class="NLM_article-title">From bacterial genomes to novel antibacterial agents: Discovery, characterization, and antibacterial activity of compounds that bind to HI0065 (YjeE) from Haemophilus influenzae</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">404</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00521.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1111%2Fj.1747-0285.2007.00521.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17581233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotV2js7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=395-404&author=C.+G.+Lernerauthor=P.+J.+Hajdukauthor=R.+Wagnerauthor=F.+L.+Wagenaarauthor=C.+Woodallauthor=Y.+G.+Guauthor=X.+B.+Searleauthor=A.+S.+Florjancicauthor=T.+Zhangauthor=R.+F.+Clarkauthor=C.+S.+Cooperauthor=J.+C.+Mackauthor=L.+Yuauthor=M.+Caiauthor=S.+F.+Betzauthor=L.+E.+Chovanauthor=J.+O.+McCallauthor=C.+L.+Black-Schaeferauthor=S.+J.+Kakavasauthor=M.+E.+Schurdakauthor=K.+M.+Comessauthor=K.+A.+Walterauthor=R.+Edaljiauthor=S.+A.+Dorwinauthor=R.+A.+Smithauthor=E.+J.+Hebertauthor=J.+E.+Harlanauthor=R.+E.+Metzgerauthor=P.+J.+Mertaauthor=J.+L.+Baranowskiauthor=M.+L.+Coenauthor=S.+J.+Thornewellauthor=A.+G.+Shivakumarauthor=A.+Y.+Saikiauthor=N.+Soniauthor=M.+Buiauthor=D.+J.+Balliauthor=W.+J.+Sandersauthor=A.+M.+Niliusauthor=T.+F.+Holzmanauthor=S.+W.+Fesikauthor=B.+A.+Beutel&title=From+bacterial+genomes+to+novel+antibacterial+agents%3A+Discovery%2C+characterization%2C+and+antibacterial+activity+of+compounds+that+bind+to+HI0065+%28YjeE%29+from+Haemophilus+influenzae&doi=10.1111%2Fj.1747-0285.2007.00521.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">From bacterial genomes to novel antibacterial agents: discovery, characterization, and antibacterial activity of compounds that bind to HI0065 (YjeE) from Haemophilus influenzae</span></div><div class="casAuthors">Lerner, Claude G.; Hajduk, Philip J.; Wagner, Rolf; Wagenaar, Frank L.; Woodall, Charlotte; Gu, Yu-Gui; Searle, Xenia B.; Florjancic, Alan S.; Zhang, Tianyuan; Clark, Richard F.; Cooper, Curt S.; Mack, Jamey C.; Yu, Liping; Cai, Mengli; Betz, Steven F.; Chovan, Linda E.; McCall, J. Owen; Black-Schaefer, Candace L.; Kakavas, Stephan J.; Schurdak, Mark E.; Comess, Kenneth M.; Walter, Karl A.; Edalji, Rohinton; Dorwin, Sarah A.; Smith, Richard A.; Hebert, Eric J.; Harlan, John E.; Metzger, Randy E.; Merta, Philip J.; Baranowski, John L.; Coen, Michael L.; Thornewell, Susan J.; Shivakumar, Annapur G.; Saiki, Anne Y.; Soni, Niru; Bui, Mai; Balli, Darlene J.; Sanders, William J.; Nilius, Angela M.; Holzman, Thomas F.; Fesik, Stephen W.; Beutel, Bruce A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">395-404</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">As part of a fully integrated and comprehensive strategy to discover novel antibacterial agents, NMR- and mass spectrometry-based affinity selection screens were performed to identify compds. that bind to protein targets uniquely found in bacteria and encoded by genes essential for microbial viability.  A biphenyl acid lead series emerged from an NMR-based screen with the Haemophilus influenzae protein HI0065, a member of a family of probable ATP-binding proteins found exclusively in eubacteria.  The structure-activity relationships developed around the NMR-derived biphenyl acid lead were consistent with on-target antibacterial activity as the Staphylococcus aureus antibacterial activity of the series correlated extremely well with binding affinity to HI0065, while the correlation of binding affinity with B-cell cytotoxicity was relatively poor.  Although further studies are needed to conclusively establish the mode of action of the biphenyl series, these compds. represent novel leads that can serve as the basis for the development of novel antibacterial agents that appear to work via an unprecedented mechanism of action.  Overall, these results support the genomics-driven hypothesis that targeting bacterial essential gene products that are not present in eukaryotic cells can identify novel antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoS-_IBLD7VLVg90H21EOLACvtfcHk0lhSidVLsmKtqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotV2js7w%253D&md5=169a289013053ac9cbbe3578f3447299</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00521.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00521.x%26sid%3Dliteratum%253Aachs%26aulast%3DLerner%26aufirst%3DC.%2BG.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DWagner%26aufirst%3DR.%26aulast%3DWagenaar%26aufirst%3DF.%2BL.%26aulast%3DWoodall%26aufirst%3DC.%26aulast%3DGu%26aufirst%3DY.%2BG.%26aulast%3DSearle%26aufirst%3DX.%2BB.%26aulast%3DFlorjancic%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DClark%26aufirst%3DR.%2BF.%26aulast%3DCooper%26aufirst%3DC.%2BS.%26aulast%3DMack%26aufirst%3DJ.%2BC.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DM.%26aulast%3DBetz%26aufirst%3DS.%2BF.%26aulast%3DChovan%26aufirst%3DL.%2BE.%26aulast%3DMcCall%26aufirst%3DJ.%2BO.%26aulast%3DBlack-Schaefer%26aufirst%3DC.%2BL.%26aulast%3DKakavas%26aufirst%3DS.%2BJ.%26aulast%3DSchurdak%26aufirst%3DM.%2BE.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DWalter%26aufirst%3DK.%2BA.%26aulast%3DEdalji%26aufirst%3DR.%26aulast%3DDorwin%26aufirst%3DS.%2BA.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DHebert%26aufirst%3DE.%2BJ.%26aulast%3DHarlan%26aufirst%3DJ.%2BE.%26aulast%3DMetzger%26aufirst%3DR.%2BE.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DBaranowski%26aufirst%3DJ.%2BL.%26aulast%3DCoen%26aufirst%3DM.%2BL.%26aulast%3DThornewell%26aufirst%3DS.%2BJ.%26aulast%3DShivakumar%26aufirst%3DA.%2BG.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DSoni%26aufirst%3DN.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DBalli%26aufirst%3DD.%2BJ.%26aulast%3DSanders%26aufirst%3DW.%2BJ.%26aulast%3DNilius%26aufirst%3DA.%2BM.%26aulast%3DHolzman%26aufirst%3DT.%2BF.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DBeutel%26aufirst%3DB.%2BA.%26atitle%3DFrom%2520bacterial%2520genomes%2520to%2520novel%2520antibacterial%2520agents%253A%2520Discovery%252C%2520characterization%252C%2520and%2520antibacterial%2520activity%2520of%2520compounds%2520that%2520bind%2520to%2520HI0065%2520%2528YjeE%2529%2520from%2520Haemophilus%2520influenzae%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D395%26epage%3D404%26doi%3D10.1111%2Fj.1747-0285.2007.00521.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Piserchio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salinas, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spaller, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mierke, D. F.</span><span> </span><span class="NLM_article-title">Targeting specific PDZ domains of PSD-95: Structural basis for enhanced affinity and enzymatic stability of a cyclic peptide</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2004.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.chembiol.2004.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15123241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1Cmsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=469-73&author=A.+Piserchioauthor=G.+D.+Salinasauthor=T.+Liauthor=J.+Marshallauthor=M.+R.+Spallerauthor=D.+F.+Mierke&title=Targeting+specific+PDZ+domains+of+PSD-95%3A+Structural+basis+for+enhanced+affinity+and+enzymatic+stability+of+a+cyclic+peptide&doi=10.1016%2Fj.chembiol.2004.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Specific PDZ Domains of PSD-95 Structural Basis for Enhanced Affinity and Enzymatic Stability of a Cyclic Peptide</span></div><div class="casAuthors">Piserchio, Andrea; Salinas, Gregory D.; Li, Tao; Marshall, John; Spaller, Mark R.; Mierke, Dale F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-473</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A cyclic peptide, Tyr-Lys-c[-Lys-Thr-Glu(βAla)-]-Val, incorporating a β-Ala lactam side chain linker and designed to target the PDZ domains of the postsynaptic d. protein 95 (PSD-95), has been synthesized and structurally characterized by NMR while free and bound to the PDZ1 domain of PSD-95.  While bound, the lactam linker of the peptide makes a no. of unique contacts outside the canonical PDZ binding motif, providing a novel target for PDZ-domain specificity as well as producing a 10-fold enhancement in binding affinity.  Addnl., the cyclization greatly enhances the enzymic stability, increasing the duration that the peptide inhibits the assocn. between PSD-95 and glutamate receptors, effectively inhibiting the clustering of kainate receptors for over 14 h after application.  Highly specific regulation of kainate receptor action may provide a novel route for treatment of drug addiction and epilepsy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRejrBkqJELLVg90H21EOLACvtfcHk0ljlwiAvXAyvJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1Cmsro%253D&md5=4ee388e7183bdc94fb024eeb98fa7193</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DPiserchio%26aufirst%3DA.%26aulast%3DSalinas%26aufirst%3DG.%2BD.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMarshall%26aufirst%3DJ.%26aulast%3DSpaller%26aufirst%3DM.%2BR.%26aulast%3DMierke%26aufirst%3DD.%2BF.%26atitle%3DTargeting%2520specific%2520PDZ%2520domains%2520of%2520PSD-95%253A%2520Structural%2520basis%2520for%2520enhanced%2520affinity%2520and%2520enzymatic%2520stability%2520of%2520a%2520cyclic%2520peptide%26jtitle%3DChem.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D469%26epage%3D73%26doi%3D10.1016%2Fj.chembiol.2004.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Wintjens, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wieruszeski, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drobecq, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rousselot-Pailley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippens, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landrieu, I.</span><span> </span><span class="NLM_article-title">1H NMR study on the binding of Pin1 Trp-Trp domain with phosphothreonine peptides</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">25150</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1074/jbc.M010327200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1074%2Fjbc.M010327200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11313338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlt1agu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=25150-6&author=R.+Wintjensauthor=J.+M.+Wieruszeskiauthor=H.+Drobecqauthor=P.+Rousselot-Pailleyauthor=L.+Bueeauthor=G.+Lippensauthor=I.+Landrieu&title=1H+NMR+study+on+the+binding+of+Pin1+Trp-Trp+domain+with+phosphothreonine+peptides&doi=10.1074%2Fjbc.M010327200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">H NMR study on the binding of Pin1 Trp-Trp domain with phosphothreonine peptides</span></div><div class="casAuthors">Wintjens, Rene; Wieruszeski, Jean-Michel; Drobecq, Herve; Rousselot-Pailley, Pierre; Buee, Luc; Lippens, Guy; Landrieu, Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">25150-25156</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The recent crystal structure of Pin1 protein bound to a doubly phosphorylated peptide from the C-terminal domain of RNA polymerase II revealed that binding interactions between Pin1 and its substrate take place through its Trp-Trp (WW) domain at the level of the loop Ser11-Arg12 and the arom. pair Tyr18-Trp29, and showed a trans conformation for both pSer-Pro peptide bonds.  However, the orientation of the ligand in the arom. recognition groove still could be sequence-specific, as previously obsd. in SH3 domains complexed by peptide ligands or for different class of WW domains.  Here the ligand orientation and conformation for two biol. relevant monophosphate substrates, one derived from the Cdc25 phosphatase of Xenopus laevis (EQPLpTPVTDL) and another from the human tau protein (KVSVVRpTPPKSPS), in complex with the WW domain, are studied by soln. NMR methods.  First, the proton resonance perturbations on the WW domain upon complexation with both peptide ligands were detd. to be essentially located in the pos. charged β-hairpin Ser11-Gly15 and around the arom. Trp29.  Dissocn. equil. consts. of 117 and 230 μM for Cdc25 and tau peptides, resp., were found.  Several intermol. nuclear Overhauser effects between WW domain and substrates were obtained from a ligand-satd. soln. and were used to det. the structures of the complexes in soln.  We found a similar N to C orientation as the one obsd. in the crystal complex structure of Pin1 and a trans conformation for the pThr-Pro peptidic bond in both peptide ligands, thereby indicating a unique binding scheme for the Pin1 WW domain to its multiple substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw3qepIthuFrVg90H21EOLACvtfcHk0ljlwiAvXAyvJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlt1agu7s%253D&md5=55561e3f96c88d5577c1f765d5613a0c</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M010327200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M010327200%26sid%3Dliteratum%253Aachs%26aulast%3DWintjens%26aufirst%3DR.%26aulast%3DWieruszeski%26aufirst%3DJ.%2BM.%26aulast%3DDrobecq%26aufirst%3DH.%26aulast%3DRousselot-Pailley%26aufirst%3DP.%26aulast%3DBuee%26aufirst%3DL.%26aulast%3DLippens%26aufirst%3DG.%26aulast%3DLandrieu%26aufirst%3DI.%26atitle%3D1H%2520NMR%2520study%2520on%2520the%2520binding%2520of%2520Pin1%2520Trp-Trp%2520domain%2520with%2520phosphothreonine%2520peptides%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D25150%26epage%3D6%26doi%3D10.1074%2Fjbc.M010327200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Guo, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dagostino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greasley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferre, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marakovits, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mroczkowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parge, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">VanArsdale, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popoff, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piraino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margosiak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Los, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Structure-based design of novel human Pin1 inhibitors (I)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">5613</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.08.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2009.08.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19729306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFakur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=5613-6&author=C.+X.+Guoauthor=X.+J.+Houauthor=L.+M.+Dongauthor=E.+Dagostinoauthor=S.+Greasleyauthor=R.+Ferreauthor=J.+Marakovitsauthor=M.+C.+Johnsonauthor=D.+Matthewsauthor=B.+Mroczkowskiauthor=H.+Pargeauthor=T.+VanArsdaleauthor=I.+Popoffauthor=J.+Pirainoauthor=S.+Margosiakauthor=J.+Thomsonauthor=G.+Losauthor=B.+W.+Murray&title=Structure-based+design+of+novel+human+Pin1+inhibitors+%28I%29&doi=10.1016%2Fj.bmcl.2009.08.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of novel human Pin1 inhibitors (I)</span></div><div class="casAuthors">Guo, Chuangxing; Hou, Xinjun; Dong, Liming; Dagostino, Eleanor; Greasley, Samantha; Ferre, RoseAnn; Marakovits, Joseph; Johnson, M. Catherine; Matthews, David; Mroczkowski, Barbara; Parge, Hans; Van Arsdale, Todd; Popoff, Ian; Piraino, Joseph; Margosiak, Stephen; Thomson, James; Los, Gerrit; Murray, Brion W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5613-5616</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Pin1 is a member of the cis-trans peptidyl-prolyl isomerase family with potential anti-cancer therapeutic value.  Structure-based de novo design and optimization of novel Pin1 inhibitors, e.g., I, are reported.  Without a viable lead from internal screenings, a series of novel Pin1 inhibitors by interrogating and exploring a protein crystal structure of Pin1 is designed.  The ligand efficiency of the initial concept mol. was optimized with integrated SBDD and parallel chem. approaches, resulting in a more attractive lead series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy51a1X9SYC7Vg90H21EOLACvtfcHk0ljlwiAvXAyvJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFakur%252FM&md5=6fbb8e918fd39293dc419dfa0747a937</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.08.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.08.034%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DC.%2BX.%26aulast%3DHou%26aufirst%3DX.%2BJ.%26aulast%3DDong%26aufirst%3DL.%2BM.%26aulast%3DDagostino%26aufirst%3DE.%26aulast%3DGreasley%26aufirst%3DS.%26aulast%3DFerre%26aufirst%3DR.%26aulast%3DMarakovits%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DM.%2BC.%26aulast%3DMatthews%26aufirst%3DD.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DParge%26aufirst%3DH.%26aulast%3DVanArsdale%26aufirst%3DT.%26aulast%3DPopoff%26aufirst%3DI.%26aulast%3DPiraino%26aufirst%3DJ.%26aulast%3DMargosiak%26aufirst%3DS.%26aulast%3DThomson%26aufirst%3DJ.%26aulast%3DLos%26aufirst%3DG.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DStructure-based%2520design%2520of%2520novel%2520human%2520Pin1%2520inhibitors%2520%2528I%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D5613%26epage%3D6%26doi%3D10.1016%2Fj.bmcl.2009.08.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Dong, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marakovits, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greasley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dagostino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferre, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraynov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pathak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Structure-based design of novel human Pin1 inhibitors (II)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2210</span><span class="NLM_x">–</span> <span class="NLM_lpage">4</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.02.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2010.02.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=20207139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslGmt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=2210-4&author=L.+M.+Dongauthor=J.+Marakovitsauthor=X.+J.+Houauthor=C.+X.+Guoauthor=S.+Greasleyauthor=E.+Dagostinoauthor=R.+Ferreauthor=M.+C.+Johnsonauthor=E.+Kraynovauthor=J.+Thomsonauthor=V.+Pathakauthor=B.+W.+Murray&title=Structure-based+design+of+novel+human+Pin1+inhibitors+%28II%29&doi=10.1016%2Fj.bmcl.2010.02.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of novel human Pin1 inhibitors (II)</span></div><div class="casAuthors">Dong, Liming; Marakovits, Joseph; Hou, Xinjun; Guo, Chuangxing; Greasley, Samantha; Dagostino, Eleanor; Ferre, RoseAnn; Johnson, M. Catherine; Kraynov, Eugenia; Thomson, James; Pathak, Ved; Murray, Brion W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2210-2214</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Following the discovery of a novel series of phosphate-contg. small mol. Pin1 inhibitors, the drug design strategy shifted to replacement of the phosphate group with an isostere with potential better pharmaceutical properties.  The initial loss in potency of carboxylate analogs was likely due to weaker charge-charge interactions in the putative phosphate binding pocket and was subsequently recovered by structure-based optimization of ligand-protein interactions in the proline binding site, leading to the discovery of a sub-micromolar non-phosphate small mol. Pin1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgilRHkGBuYbVg90H21EOLACvtfcHk0lhrJ1ajOvCNAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslGmt7s%253D&md5=d16f5beec12a17a38bca7e84e09ddb0d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.02.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.02.033%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DL.%2BM.%26aulast%3DMarakovits%26aufirst%3DJ.%26aulast%3DHou%26aufirst%3DX.%2BJ.%26aulast%3DGuo%26aufirst%3DC.%2BX.%26aulast%3DGreasley%26aufirst%3DS.%26aulast%3DDagostino%26aufirst%3DE.%26aulast%3DFerre%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DM.%2BC.%26aulast%3DKraynov%26aufirst%3DE.%26aulast%3DThomson%26aufirst%3DJ.%26aulast%3DPathak%26aufirst%3DV.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DStructure-based%2520design%2520of%2520novel%2520human%2520Pin1%2520inhibitors%2520%2528II%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D2210%26epage%3D4%26doi%3D10.1016%2Fj.bmcl.2010.02.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Guo, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marakovits, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greasley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dagostino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferre, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paderes, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piraino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraynov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Structure-based design of novel human Pin1 inhibitors (III): Optimizing affinity beyond the phosphate recognition pocket</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">4187</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.07.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2014.07.044" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=4187-91&author=C.+X.+Guoauthor=X.+J.+Houauthor=L.+M.+Dongauthor=J.+Marakovitsauthor=S.+Greasleyauthor=E.+Dagostinoauthor=R.+Ferreauthor=M.+C.+Johnsonauthor=P.+S.+Humphriesauthor=H.+T.+Liauthor=G.+D.+Paderesauthor=J.+Pirainoauthor=E.+Kraynovauthor=B.+W.+Murray&title=Structure-based+design+of+novel+human+Pin1+inhibitors+%28III%29%3A+Optimizing+affinity+beyond+the+phosphate+recognition+pocket&doi=10.1016%2Fj.bmcl.2014.07.044"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.07.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.07.044%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DC.%2BX.%26aulast%3DHou%26aufirst%3DX.%2BJ.%26aulast%3DDong%26aufirst%3DL.%2BM.%26aulast%3DMarakovits%26aufirst%3DJ.%26aulast%3DGreasley%26aufirst%3DS.%26aulast%3DDagostino%26aufirst%3DE.%26aulast%3DFerre%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DM.%2BC.%26aulast%3DHumphries%26aufirst%3DP.%2BS.%26aulast%3DLi%26aufirst%3DH.%2BT.%26aulast%3DPaderes%26aufirst%3DG.%2BD.%26aulast%3DPiraino%26aufirst%3DJ.%26aulast%3DKraynov%26aufirst%3DE.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DStructure-based%2520design%2520of%2520novel%2520human%2520Pin1%2520inhibitors%2520%2528III%2529%253A%2520Optimizing%2520affinity%2520beyond%2520the%2520phosphate%2520recognition%2520pocket%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D4187%26epage%3D91%26doi%3D10.1016%2Fj.bmcl.2014.07.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Wendt, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarris, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoff, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettesheim, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comess, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaac, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olejniczak, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span> </span><span class="NLM_article-title">Discovery of a novel small molecule binding site of human survivin</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">3122</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.03.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2007.03.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17391963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1Kisr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=3122-9&author=M.+D.+Wendtauthor=C.+H.+Sunauthor=A.+Kunzerauthor=D.+Sauerauthor=K.+Sarrisauthor=E.+Hoffauthor=L.+P.+Yuauthor=D.+G.+Nettesheimauthor=J.+Chenauthor=S.+Jinauthor=K.+M.+Comessauthor=Y.+H.+Fanauthor=S.+N.+Andersonauthor=B.+Isaacauthor=E.+T.+Olejniczakauthor=P.+J.+Hajdukauthor=S.+H.+Rosenbergauthor=S.+W.+Elmore&title=Discovery+of+a+novel+small+molecule+binding+site+of+human+survivin&doi=10.1016%2Fj.bmcl.2007.03.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel small molecule binding site of human survivin</span></div><div class="casAuthors">Wendt, Michael D.; Sun, Chaohong; Kunzer, Aaron; Sauer, Daryl; Sarris, Kathy; Hoff, Ethan; Yu, Liping; Nettesheim, David G.; Chen, Jun; Jin, Sha; Comess, Kenneth M.; Fan, Yihong; Anderson, Steven N.; Isaac, Binumol; Olejniczak, Edward T.; Hajduk, Philip J.; Rosenberg, Saul H.; Elmore, Steven W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3122-3129</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Survivin is one of the most tumor-specific genes in the human genome and is an attractive target for cancer therapy.  However, small-mol. ligands for survivin have not yet been described.  Thus, an interrogation of survivin which could potentially both validate a small-mol. therapy approach, and det. the biochem. nature of any of survivin's functions has not been possible.  Here we describe the discovery and characterization of a small mol. binding site on the survivin surface distinct from the Smac peptide-binding site.  The new site is located at the dimer interface and exhibits many of the features of highly drugable, biol. relevant protein binding sites.  A variety of small hydrophobic compds. were found that bind with moderate affinity to this binding site, from which one lead was developed into a group of compds. with nanomolar affinity.  Addnl., a subset of these compds. are adequately water-sol. and cell-permeable.  Thus, the structural studies and small mols. described here provide tools that can be used to probe the biochem. role(s) of survivin, and may ultimately serve as a basis for the development of small mol. therapeutics acting via direct or allosteric disruption of binding events related to this poorly understood target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU2KNydbvLSrVg90H21EOLACvtfcHk0lhrJ1ajOvCNAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1Kisr4%253D&md5=41e0c01dbef616c9620d1303b9135b33</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.03.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.03.042%26sid%3Dliteratum%253Aachs%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DSun%26aufirst%3DC.%2BH.%26aulast%3DKunzer%26aufirst%3DA.%26aulast%3DSauer%26aufirst%3DD.%26aulast%3DSarris%26aufirst%3DK.%26aulast%3DHoff%26aufirst%3DE.%26aulast%3DYu%26aufirst%3DL.%2BP.%26aulast%3DNettesheim%26aufirst%3DD.%2BG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DFan%26aufirst%3DY.%2BH.%26aulast%3DAnderson%26aufirst%3DS.%2BN.%26aulast%3DIsaac%26aufirst%3DB.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520small%2520molecule%2520binding%2520site%2520of%2520human%2520survivin%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D3122%26epage%3D9%26doi%3D10.1016%2Fj.bmcl.2007.03.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mack, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendoza, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaac, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braddock, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comess, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clore, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levens, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">NMR-driven discovery of benzoylanthranilic acid inhibitors of far upstream element binding protein binding to the human oncogene c-myc promoter</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4851</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1021/jm0497803</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0497803" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4851-7&author=J.+R.+Huthauthor=L.+Yuauthor=I.+Collinsauthor=J.+Mackauthor=R.+Mendozaauthor=B.+Isaacauthor=D.+T.+Braddockauthor=S.+W.+Muchmoreauthor=K.+M.+Comessauthor=S.+W.+Fesikauthor=G.+M.+Cloreauthor=D.+Levensauthor=P.+J.+Hajduk&title=NMR-driven+discovery+of+benzoylanthranilic+acid+inhibitors+of+far+upstream+element+binding+protein+binding+to+the+human+oncogene+c-myc+promoter&doi=10.1021%2Fjm0497803"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fjm0497803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0497803%26sid%3Dliteratum%253Aachs%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DMack%26aufirst%3DJ.%26aulast%3DMendoza%26aufirst%3DR.%26aulast%3DIsaac%26aufirst%3DB.%26aulast%3DBraddock%26aufirst%3DD.%2BT.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DLevens%26aufirst%3DD.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNMR-driven%2520discovery%2520of%2520benzoylanthranilic%2520acid%2520inhibitors%2520of%2520far%2520upstream%2520element%2520binding%2520protein%2520binding%2520to%2520the%2520human%2520oncogene%2520c-myc%2520promoter%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4851%26epage%3D7%26doi%3D10.1021%2Fjm0497803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinges, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miknis, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merlock, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kempf, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egan, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robins, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holzman, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">NMR-based discovery of lead inhibitors that block DNA binding of the human papillomavirus E2 protein</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">3144</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1021/jm9703404</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9703404" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=3144-50&author=P.+J.+Hajdukauthor=J.+Dingesauthor=G.+F.+Miknisauthor=M.+Merlockauthor=T.+Middletonauthor=D.+J.+Kempfauthor=D.+A.+Eganauthor=K.+A.+Walterauthor=T.+S.+Robinsauthor=S.+B.+Shukerauthor=T.+F.+Holzmanauthor=S.+W.+Fesik&title=NMR-based+discovery+of+lead+inhibitors+that+block+DNA+binding+of+the+human+papillomavirus+E2+protein&doi=10.1021%2Fjm9703404"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Fjm9703404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9703404%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DDinges%26aufirst%3DJ.%26aulast%3DMiknis%26aufirst%3DG.%2BF.%26aulast%3DMerlock%26aufirst%3DM.%26aulast%3DMiddleton%26aufirst%3DT.%26aulast%3DKempf%26aufirst%3DD.%2BJ.%26aulast%3DEgan%26aufirst%3DD.%2BA.%26aulast%3DWalter%26aufirst%3DK.%2BA.%26aulast%3DRobins%26aufirst%3DT.%2BS.%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHolzman%26aufirst%3DT.%2BF.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DNMR-based%2520discovery%2520of%2520lead%2520inhibitors%2520that%2520block%2520DNA%2520binding%2520of%2520the%2520human%2520papillomavirus%2520E2%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D3144%26epage%3D50%26doi%3D10.1021%2Fjm9703404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Schaal, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mallet, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMinn, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sopko, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parekh, B. S.</span><span> </span><span class="NLM_article-title">Inhibition of human papilloma virus E2 DNA binding protein by covalently linked polyamides</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1282</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1093/nar/gkg206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Fnar%2Fgkg206" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=1282-91&author=T.+D.+Schaalauthor=W.+G.+Malletauthor=D.+L.+McMinnauthor=N.+V.+Nguyenauthor=M.+M.+Sopkoauthor=S.+Johnauthor=B.+S.+Parekh&title=Inhibition+of+human+papilloma+virus+E2+DNA+binding+protein+by+covalently+linked+polyamides&doi=10.1093%2Fnar%2Fgkg206"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkg206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkg206%26sid%3Dliteratum%253Aachs%26aulast%3DSchaal%26aufirst%3DT.%2BD.%26aulast%3DMallet%26aufirst%3DW.%2BG.%26aulast%3DMcMinn%26aufirst%3DD.%2BL.%26aulast%3DNguyen%26aufirst%3DN.%2BV.%26aulast%3DSopko%26aufirst%3DM.%2BM.%26aulast%3DJohn%26aufirst%3DS.%26aulast%3DParekh%26aufirst%3DB.%2BS.%26atitle%3DInhibition%2520of%2520human%2520papilloma%2520virus%2520E2%2520DNA%2520binding%2520protein%2520by%2520covalently%2520linked%2520polyamides%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2003%26volume%3D31%26spage%3D1282%26epage%3D91%26doi%3D10.1093%2Fnar%2Fgkg206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">Kurg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langel, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ustav, M.</span><span> </span><span class="NLM_article-title">Inhibition of the bovine papillomavirus E2 protein activity by peptide nucleic acid</span> <span class="citation_source-journal">Virus Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">39</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/S0168-1702(99)00124-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0168-1702%2899%2900124-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10653916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVGntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2000&pages=39-50&author=R.+Kurgauthor=U.+Langelauthor=M.+Ustav&title=Inhibition+of+the+bovine+papillomavirus+E2+protein+activity+by+peptide+nucleic+acid&doi=10.1016%2FS0168-1702%2899%2900124-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the bovine papillomavirus E2 protein activity by peptide nucleic acid</span></div><div class="casAuthors">Kurg, R.; Langel, U.; Ustav, M.</div><div class="citationInfo"><span class="NLM_cas:title">Virus Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-50</span>CODEN:
                <span class="NLM_cas:coden">VIREDF</span>;
        ISSN:<span class="NLM_cas:issn">0168-1702</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">The bovine papillomavirus type-1 E2 protein is the master regulator of the papillomavirus transcription and replication, the activity of which is regulated through sequence-specific DNA binding.  Peptide nucleic acid (PNA) is a nucleic acid analog, which assocs. with high affinity to complementary DNA, RNA or PNA, yielding in formation of stable complexes.  The potential use of PNA as a sequence-specific inhibitor of the E2 protein activity is studied in this report.  We demonstrate that replacement of one or both DNA strands with the complementary PNA reduced drastically the affinity of the BPV-1 E2 protein to its target site in the direct as well as in competitive binding as shown by in vitro gel-shift assays.  We demonstrate that PNA could specifically bind to the double stranded E2 binding site by forming the complex with DNA oligonucleotide.  In addn., PNA was able to bind specifically to the E2 binding site within the supercoiled plasmid DNA.  Such binding of PNA to the E2 binding site within the origin of replication specifically abolished the activity of the E2 protein in the initiation of DNA replication in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiSLoO0Q9PRbVg90H21EOLACvtfcHk0lh2P9M2WuSElg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVGntw%253D%253D&md5=d22afa222edda70df90448c4dfb42fb3</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2FS0168-1702%2899%2900124-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0168-1702%252899%252900124-0%26sid%3Dliteratum%253Aachs%26aulast%3DKurg%26aufirst%3DR.%26aulast%3DLangel%26aufirst%3DU.%26aulast%3DUstav%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520the%2520bovine%2520papillomavirus%2520E2%2520protein%2520activity%2520by%2520peptide%2520nucleic%2520acid%26jtitle%3DVirus%2520Res.%26date%3D2000%26volume%3D66%26spage%3D39%26epage%3D50%26doi%3D10.1016%2FS0168-1702%2899%2900124-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">de Dios, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prieto, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ezquerra, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tebbe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez de Uralde, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeTourneau, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGee, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boylan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, T. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M. C.</span><span> </span><span class="NLM_article-title">4-substituted D-glutamic acid analogues: The first potent inhibitors of glutamate racemase (MurI) enzyme with antibacterial activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">4559</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1021/jm020901d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020901d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=4559-70&author=A.+de+Diosauthor=L.+Prietoauthor=J.+A.+Martinauthor=A.+Rubioauthor=J.+Ezquerraauthor=M.+Tebbeauthor=B.+Lopez+de+Uraldeauthor=J.+Martinauthor=A.+Sanchezauthor=D.+L.+LeTourneauauthor=J.+E.+McGeeauthor=C.+Boylanauthor=T.+R.+Parrauthor=M.+C.+Smith&title=4-substituted+D-glutamic+acid+analogues%3A+The+first+potent+inhibitors+of+glutamate+racemase+%28MurI%29+enzyme+with+antibacterial+activity&doi=10.1021%2Fjm020901d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Fjm020901d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020901d%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BDios%26aufirst%3DA.%26aulast%3DPrieto%26aufirst%3DL.%26aulast%3DMartin%26aufirst%3DJ.%2BA.%26aulast%3DRubio%26aufirst%3DA.%26aulast%3DEzquerra%26aufirst%3DJ.%26aulast%3DTebbe%26aufirst%3DM.%26aulast%3DLopez%2Bde%2BUralde%26aufirst%3DB.%26aulast%3DMartin%26aufirst%3DJ.%26aulast%3DSanchez%26aufirst%3DA.%26aulast%3DLeTourneau%26aufirst%3DD.%2BL.%26aulast%3DMcGee%26aufirst%3DJ.%2BE.%26aulast%3DBoylan%26aufirst%3DC.%26aulast%3DParr%26aufirst%3DT.%2BR.%26aulast%3DSmith%26aufirst%3DM.%2BC.%26atitle%3D4-substituted%2520D-glutamic%2520acid%2520analogues%253A%2520The%2520first%2520potent%2520inhibitors%2520of%2520glutamate%2520racemase%2520%2528MurI%2529%2520enzyme%2520with%2520antibacterial%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D4559%26epage%3D70%26doi%3D10.1021%2Fjm020901d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group">Geng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breault, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comita-Prevoir, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrichko, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyermann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundqvist, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doig, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorseth, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noonan, B.</span><span> </span><span class="NLM_article-title">Exploring 9-benzyl purines as inhibitors of glutamate racemase (MurI) gram-positive bacteria</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4368</span><span class="NLM_x">–</span> <span class="NLM_lpage">4372</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.06.068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2008.06.068" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4368-4372&author=B.+Gengauthor=G.+Breaultauthor=J.+Comita-Prevoirauthor=R.+Petrichkoauthor=C.+Eyermannauthor=T.+Lundqvistauthor=P.+Doigauthor=E.+Gorsethauthor=B.+Noonan&title=Exploring+9-benzyl+purines+as+inhibitors+of+glutamate+racemase+%28MurI%29+gram-positive+bacteria&doi=10.1016%2Fj.bmcl.2008.06.068"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.06.068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.06.068%26sid%3Dliteratum%253Aachs%26aulast%3DGeng%26aufirst%3DB.%26aulast%3DBreault%26aufirst%3DG.%26aulast%3DComita-Prevoir%26aufirst%3DJ.%26aulast%3DPetrichko%26aufirst%3DR.%26aulast%3DEyermann%26aufirst%3DC.%26aulast%3DLundqvist%26aufirst%3DT.%26aulast%3DDoig%26aufirst%3DP.%26aulast%3DGorseth%26aufirst%3DE.%26aulast%3DNoonan%26aufirst%3DB.%26atitle%3DExploring%25209-benzyl%2520purines%2520as%2520inhibitors%2520of%2520glutamate%2520racemase%2520%2528MurI%2529%2520gram-positive%2520bacteria%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4368%26epage%3D4372%26doi%3D10.1016%2Fj.bmcl.2008.06.068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group">Breault, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comita-Prevoir, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyermann, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrichko, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doig, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorseth, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noonan, B.</span><span> </span><span class="NLM_article-title">Exploring 8-benzyl pteridine-6,7-diones as inhibitors of glutamate racemase (MurI) in gram-positive bacteria</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6100</span><span class="NLM_x">–</span> <span class="NLM_lpage">6103</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.10.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2008.10.022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=6100-6103&author=G.+A.+Breaultauthor=J.+Comita-Prevoirauthor=C.+J.+Eyermannauthor=B.+Gengauthor=R.+Petrichkoauthor=P.+Doigauthor=E.+Gorsethauthor=B.+Noonan&title=Exploring+8-benzyl+pteridine-6%2C7-diones+as+inhibitors+of+glutamate+racemase+%28MurI%29+in+gram-positive+bacteria&doi=10.1016%2Fj.bmcl.2008.10.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.10.022%26sid%3Dliteratum%253Aachs%26aulast%3DBreault%26aufirst%3DG.%2BA.%26aulast%3DComita-Prevoir%26aufirst%3DJ.%26aulast%3DEyermann%26aufirst%3DC.%2BJ.%26aulast%3DGeng%26aufirst%3DB.%26aulast%3DPetrichko%26aufirst%3DR.%26aulast%3DDoig%26aufirst%3DP.%26aulast%3DGorseth%26aufirst%3DE.%26aulast%3DNoonan%26aufirst%3DB.%26atitle%3DExploring%25208-benzyl%2520pteridine-6%252C7-diones%2520as%2520inhibitors%2520of%2520glutamate%2520racemase%2520%2528MurI%2529%2520in%2520gram-positive%2520bacteria%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D6100%26epage%3D6103%26doi%3D10.1016%2Fj.bmcl.2008.10.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group">McRee, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegel, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodin, D. B.</span><span> </span><span class="NLM_article-title">Construction of a bisaquo heme enzyme and binding by exogenous ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">12847</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.1073/pnas.91.26.12847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.91.26.12847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=7809133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK2MXivVCqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=12847-51&author=D.+E.+McReeauthor=G.+M.+Jensenauthor=M.+M.+Fitzgeraldauthor=H.+A.+Siegelauthor=D.+B.+Goodin&title=Construction+of+a+bisaquo+heme+enzyme+and+binding+by+exogenous+ligands&doi=10.1073%2Fpnas.91.26.12847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Construction of a bisaquo heme enzyme and binding by exogenous ligands</span></div><div class="casAuthors">McRee, D. E.; Jensen, G. M.; Fitzgerald, M. M.; Siegel, H. A.; Goodin, D. B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">12847-51</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The crystal structure of the H175G mutant of cytochrome c peroxidase (EC 1.11.1.5), missing its only heme ligand, reveals that the His is replaced by solvent to give a bisaquo hemoprotein.  This protein retains some residual activity, which can be stimulated or inhibited by addn. of exogenous ligands.  Structural anal. of the H175G-imidazole complex confirms the binding of imidazole to the heme at the position of the wild-type His ligand.  This imidazole complex reacts readily with H2O2 to produce a radical species with novel properties.  However, reactivation in this complex is incomplete (∼5%), which, in view of the very similar structures of the wild-type and the H175G/imidazole forms, implies a crit. role for tethering of the axial ligand in catalysis.  This study demonstrates the feasibility of constructing heme enzymes with no covalent link to the protein and with unnatural ligand replacements.  Such enzymes may prove useful in studies of electron transfer mechanisms and in the engineering of novel heme-based catalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrArqgvJ4hBpbVg90H21EOLACvtfcHk0lg2KqneDWFjgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXivVCqsrY%253D&md5=cbb7f5f14beeda947c29b021a861c4db</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1073%2Fpnas.91.26.12847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.91.26.12847%26sid%3Dliteratum%253Aachs%26aulast%3DMcRee%26aufirst%3DD.%2BE.%26aulast%3DJensen%26aufirst%3DG.%2BM.%26aulast%3DFitzgerald%26aufirst%3DM.%2BM.%26aulast%3DSiegel%26aufirst%3DH.%2BA.%26aulast%3DGoodin%26aufirst%3DD.%2BB.%26atitle%3DConstruction%2520of%2520a%2520bisaquo%2520heme%2520enzyme%2520and%2520binding%2520by%2520exogenous%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1994%26volume%3D91%26spage%3D12847%26epage%3D51%26doi%3D10.1073%2Fpnas.91.26.12847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group">Appleby, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittenberg, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittenberg, J. B.</span><span> </span><span class="NLM_article-title">Nicotinic-acid as a ligand affecting leghemoglobin structure and oxygen reactivity</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">564</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1073/pnas.70.2.564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.70.2.564" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=1973&pages=564-8&author=C.+A.+Applebyauthor=B.+A.+Wittenbergauthor=J.+B.+Wittenberg&title=Nicotinic-acid+as+a+ligand+affecting+leghemoglobin+structure+and+oxygen+reactivity&doi=10.1073%2Fpnas.70.2.564"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1073%2Fpnas.70.2.564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.70.2.564%26sid%3Dliteratum%253Aachs%26aulast%3DAppleby%26aufirst%3DC.%2BA.%26aulast%3DWittenberg%26aufirst%3DB.%2BA.%26aulast%3DWittenberg%26aufirst%3DJ.%2BB.%26atitle%3DNicotinic-acid%2520as%2520a%2520ligand%2520affecting%2520leghemoglobin%2520structure%2520and%2520oxygen%2520reactivity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1973%26volume%3D70%26spage%3D564%26epage%3D8%26doi%3D10.1073%2Fpnas.70.2.564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group">Maldonado, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lostao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irun, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez-Recio, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genzor, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubio, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luquita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daoudi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sancho, J.</span><span> </span><span class="NLM_article-title">Apoflavodoxin: Structure, stability, and FMN binding</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1016/S0300-9084(00)88876-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0300-9084%2800%2988876-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=1998&pages=813-20&author=S.+Maldonadoauthor=A.+Lostaoauthor=M.+P.+Irunauthor=J.+Fernandez-Recioauthor=C.+G.+Genzorauthor=E.+B.+Gonzalezauthor=J.+A.+Rubioauthor=A.+Luquitaauthor=F.+Daoudiauthor=J.+Sancho&title=Apoflavodoxin%3A+Structure%2C+stability%2C+and+FMN+binding&doi=10.1016%2FS0300-9084%2800%2988876-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2FS0300-9084%2800%2988876-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0300-9084%252800%252988876-8%26sid%3Dliteratum%253Aachs%26aulast%3DMaldonado%26aufirst%3DS.%26aulast%3DLostao%26aufirst%3DA.%26aulast%3DIrun%26aufirst%3DM.%2BP.%26aulast%3DFernandez-Recio%26aufirst%3DJ.%26aulast%3DGenzor%26aufirst%3DC.%2BG.%26aulast%3DGonzalez%26aufirst%3DE.%2BB.%26aulast%3DRubio%26aufirst%3DJ.%2BA.%26aulast%3DLuquita%26aufirst%3DA.%26aulast%3DDaoudi%26aufirst%3DF.%26aulast%3DSancho%26aufirst%3DJ.%26atitle%3DApoflavodoxin%253A%2520Structure%252C%2520stability%252C%2520and%2520FMN%2520binding%26jtitle%3DBiochimie%26date%3D1998%26volume%3D80%26spage%3D813%26epage%3D20%26doi%3D10.1016%2FS0300-9084%2800%2988876-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group">Irie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakahara, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yanai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohigashi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wender, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konishi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikkawa, U.</span><span> </span><span class="NLM_article-title">Molecular basis for protein kinase c isozyme-selective binding: The synthesis, folding, and phorbol ester binding of the cysteine-rich domains of all protein kinase c isozymes</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">9159</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1021/ja981087f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja981087f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=9159-67&author=K.+Irieauthor=K.+Oieauthor=A.+Nakaharaauthor=Y.+Yanaiauthor=H.+Ohigashiauthor=P.+A.+Wenderauthor=H.+Fukudaauthor=H.+Konishiauthor=U.+Kikkawa&title=Molecular+basis+for+protein+kinase+c+isozyme-selective+binding%3A+The+synthesis%2C+folding%2C+and+phorbol+ester+binding+of+the+cysteine-rich+domains+of+all+protein+kinase+c+isozymes&doi=10.1021%2Fja981087f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fja981087f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja981087f%26sid%3Dliteratum%253Aachs%26aulast%3DIrie%26aufirst%3DK.%26aulast%3DOie%26aufirst%3DK.%26aulast%3DNakahara%26aufirst%3DA.%26aulast%3DYanai%26aufirst%3DY.%26aulast%3DOhigashi%26aufirst%3DH.%26aulast%3DWender%26aufirst%3DP.%2BA.%26aulast%3DFukuda%26aufirst%3DH.%26aulast%3DKonishi%26aufirst%3DH.%26aulast%3DKikkawa%26aufirst%3DU.%26atitle%3DMolecular%2520basis%2520for%2520protein%2520kinase%2520c%2520isozyme-selective%2520binding%253A%2520The%2520synthesis%252C%2520folding%252C%2520and%2520phorbol%2520ester%2520binding%2520of%2520the%2520cysteine-rich%2520domains%2520of%2520all%2520protein%2520kinase%2520c%2520isozymes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D9159%26epage%3D67%26doi%3D10.1021%2Fja981087f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group">Murray, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cann, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Derrick, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutcliffe, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">Steroid recognition by chloramphenicol acetyltransferase - engineering and structural-analysis of a high-affinity fusidic acid-binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">254</span><span class="NLM_x">, </span> <span class="NLM_fpage">993</span><span class="NLM_x">–</span> <span class="NLM_lpage">1005</span><span class="refDoi"> DOI: 10.1006/jmbi.1995.0671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1006%2Fjmbi.1995.0671" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=254&publication_year=1995&pages=993-1005&author=I.+A.+Murrayauthor=P.+A.+Cannauthor=P.+J.+Dayauthor=J.+P.+Derrickauthor=M.+J.+Sutcliffeauthor=W.+V.+Shawauthor=A.+G.+W.+Leslie&title=Steroid+recognition+by+chloramphenicol+acetyltransferase+-+engineering+and+structural-analysis+of+a+high-affinity+fusidic+acid-binding+site&doi=10.1006%2Fjmbi.1995.0671"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1995.0671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1995.0671%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DI.%2BA.%26aulast%3DCann%26aufirst%3DP.%2BA.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DDerrick%26aufirst%3DJ.%2BP.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26aulast%3DShaw%26aufirst%3DW.%2BV.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DSteroid%2520recognition%2520by%2520chloramphenicol%2520acetyltransferase%2520-%2520engineering%2520and%2520structural-analysis%2520of%2520a%2520high-affinity%2520fusidic%2520acid-binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1995%26volume%3D254%26spage%3D993%26epage%3D1005%26doi%3D10.1006%2Fjmbi.1995.0671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group">Vyas, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vyas, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quiocho, F. A.</span><span> </span><span class="NLM_article-title">Crystallographic analysis of the epimeric and anomeric specificity of the periplasmic transport chemosensory protein-receptor for D-glucose and D-galactose</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">4762</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1021/bi00182a003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00182a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1994&pages=4762-8&author=M.+N.+Vyasauthor=N.+K.+Vyasauthor=F.+A.+Quiocho&title=Crystallographic+analysis+of+the+epimeric+and+anomeric+specificity+of+the+periplasmic+transport+chemosensory+protein-receptor+for+D-glucose+and+D-galactose&doi=10.1021%2Fbi00182a003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fbi00182a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00182a003%26sid%3Dliteratum%253Aachs%26aulast%3DVyas%26aufirst%3DM.%2BN.%26aulast%3DVyas%26aufirst%3DN.%2BK.%26aulast%3DQuiocho%26aufirst%3DF.%2BA.%26atitle%3DCrystallographic%2520analysis%2520of%2520the%2520epimeric%2520and%2520anomeric%2520specificity%2520of%2520the%2520periplasmic%2520transport%2520chemosensory%2520protein-receptor%2520for%2520D-glucose%2520and%2520D-galactose%26jtitle%3DBiochemistry%26date%3D1994%26volume%3D33%26spage%3D4762%26epage%3D8%26doi%3D10.1021%2Fbi00182a003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group">Chung, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fromme, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span> </span><span class="NLM_article-title">Structure of human cytidine deaminase bound to a potent inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1021/jm0496279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0496279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitlGgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=658-60&author=S.+J.+Chungauthor=J.+C.+Frommeauthor=G.+L.+Verdine&title=Structure+of+human+cytidine+deaminase+bound+to+a+potent+inhibitor&doi=10.1021%2Fjm0496279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human Cytidine Deaminase Bound to a Potent Inhibitor</span></div><div class="casAuthors">Chung, Sang J.; Fromme, J. Christopher; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">658-660</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human cytidine deaminase (CDA) is an enzyme prominent for its role in catalyzing metabolic processing of nucleoside-type anticancer and antiviral agents.  It is thus a promising target for the development of small mol. therapeutic adjuvants.  We report the first crystal structure of human CDA as a complex with a tight-binding inhibitor, diazepinone riboside 1.  The structure reveals that inhibitor 1 is able to establish a canonical π/π-interaction with a key active site residue, Phe 137.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3nwv7nFjrpLVg90H21EOLACvtfcHk0liCYMs4tliBrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitlGgsQ%253D%253D&md5=51c16f741ee045b974e62fe3cd3d0005</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1021%2Fjm0496279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0496279%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DS.%2BJ.%26aulast%3DFromme%26aufirst%3DJ.%2BC.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DStructure%2520of%2520human%2520cytidine%2520deaminase%2520bound%2520to%2520a%2520potent%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D658%26epage%3D60%26doi%3D10.1021%2Fjm0496279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group">Masaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunasawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakiyama, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soejima, M.</span><span> </span><span class="NLM_article-title">Inhibition of achromobacter protease-I by lysinal derivatives</span> <span class="citation_source-journal">Biosci., Biotechnol., Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1604</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1271/bbb.56.1604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1271%2Fbbb.56.1604" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1992&pages=1604-7&author=T.+Masakiauthor=T.+Tanakaauthor=S.+Tsunasawaauthor=F.+Sakiyamaauthor=M.+Soejima&title=Inhibition+of+achromobacter+protease-I+by+lysinal+derivatives&doi=10.1271%2Fbbb.56.1604"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1271%2Fbbb.56.1604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1271%252Fbbb.56.1604%26sid%3Dliteratum%253Aachs%26aulast%3DMasaki%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DTsunasawa%26aufirst%3DS.%26aulast%3DSakiyama%26aufirst%3DF.%26aulast%3DSoejima%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520achromobacter%2520protease-I%2520by%2520lysinal%2520derivatives%26jtitle%3DBiosci.%252C%2520Biotechnol.%252C%2520Biochem.%26date%3D1992%26volume%3D56%26spage%3D1604%26epage%3D7%26doi%3D10.1271%2Fbbb.56.1604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group">Lunney, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Para, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humblet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fergus, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marks, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawyer, T. K.</span><span> </span><span class="NLM_article-title">Structure-based design of a novel series of nonpeptide ligands that bind to the pp60src SH2 domain</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">12471</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1021/ja971794t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja971794t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=12471-6&author=E.+A.+Lunneyauthor=K.+S.+Paraauthor=J.+R.+Rubinauthor=C.+Humbletauthor=J.+H.+Fergusauthor=J.+S.+Marksauthor=T.+K.+Sawyer&title=Structure-based+design+of+a+novel+series+of+nonpeptide+ligands+that+bind+to+the+pp60src+SH2+domain&doi=10.1021%2Fja971794t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1021%2Fja971794t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja971794t%26sid%3Dliteratum%253Aachs%26aulast%3DLunney%26aufirst%3DE.%2BA.%26aulast%3DPara%26aufirst%3DK.%2BS.%26aulast%3DRubin%26aufirst%3DJ.%2BR.%26aulast%3DHumblet%26aufirst%3DC.%26aulast%3DFergus%26aufirst%3DJ.%2BH.%26aulast%3DMarks%26aufirst%3DJ.%2BS.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520a%2520novel%2520series%2520of%2520nonpeptide%2520ligands%2520that%2520bind%2520to%2520the%2520pp60src%2520SH2%2520domain%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D12471%26epage%3D6%26doi%3D10.1021%2Fja971794t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group">Fauman, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, E. S.</span><span> </span><span class="NLM_article-title">Structure-based druggability assessment--identifying suitable targets for small molecule therapeutics</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2011.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.cbpa.2011.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21704549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFCltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2011&pages=463-8&author=E.+B.+Faumanauthor=B.+K.+Raiauthor=E.+S.+Huang&title=Structure-based+druggability+assessment%2D%2Didentifying+suitable+targets+for+small+molecule+therapeutics&doi=10.1016%2Fj.cbpa.2011.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based druggability assessment - identifying suitable targets for small molecule therapeutics</span></div><div class="casAuthors">Fauman, Eric B.; Rai, Brajesh K.; Huang, Enoch S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">463-468</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A target is druggable if it can be modulated in vivo by a drug-like mol.  The general properties of oral drugs are summarized by the rule of 5' which specifies parameters related to size and lipophilicity.  Structure-based target druggability assessment consists of predicting ligand-binding sites on the protein that are complementary to these drug-like properties.  Automated identification of ligand-binding sites can use geometrical considerations alone or include specific physicochem. properties of the protein surface.  Features of a pocket's size and shape, together with measures of its hydrophobicity, are most informative in identifying suitable drug-binding pockets.  The recent availability of several validation sets of druggable vs. undruggable targets has helped fuel the development of more elaborate methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzQbgkbe44IrVg90H21EOLACvtfcHk0lime7OKcMPUjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFCltrw%253D&md5=876a5083e4ae685d55eee3295e924cfd</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2011.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2011.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DFauman%26aufirst%3DE.%2BB.%26aulast%3DRai%26aufirst%3DB.%2BK.%26aulast%3DHuang%26aufirst%3DE.%2BS.%26atitle%3DStructure-based%2520druggability%2520assessment--identifying%2520suitable%2520targets%2520for%2520small%2520molecule%2520therapeutics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D15%26spage%3D463%26epage%3D8%26doi%3D10.1016%2Fj.cbpa.2011.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group">Schonthal, A. H.</span><span> </span><span class="NLM_article-title">Direct non-cyclooxygenase-2 targets of celecoxib and their potential relevance for cancer therapy</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1465</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1038/sj.bjc.6604049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fsj.bjc.6604049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17955049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A280%3ADC%252BD2sjgtFGgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2007&pages=1465-8&author=A.+H.+Schonthal&title=Direct+non-cyclooxygenase-2+targets+of+celecoxib+and+their+potential+relevance+for+cancer+therapy&doi=10.1038%2Fsj.bjc.6604049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Direct non-cyclooxygenase-2 targets of celecoxib and their potential relevance for cancer therapy</span></div><div class="casAuthors">Schonthal A H</div><div class="citationInfo"><span class="NLM_cas:title">British journal of cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1465-8</span>
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    </div><div class="casAbstract">Celecoxib (Celebrex) was developed as a selective cyclooxygenase-2 (COX-2) inhibitor for the treatment of chronic pain.  However, it now appears that this compound harbours additional pharmacologic activities that are entirely independent of its COX-2-inhibitory activity.  This review presents the recently emerged direct non-COX-2 targets of celecoxib and their proposed role in mediating this drug's antitumour effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFaqq9zDTtTq69lAbTj9rGfW6udTcc2ea9614ubz0CmLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjgtFGgtQ%253D%253D&md5=7684ac2baebca2baafaed5bf39302693</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6604049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6604049%26sid%3Dliteratum%253Aachs%26aulast%3DSchonthal%26aufirst%3DA.%2BH.%26atitle%3DDirect%2520non-cyclooxygenase-2%2520targets%2520of%2520celecoxib%2520and%2520their%2520potential%2520relevance%2520for%2520cancer%2520therapy%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2007%26volume%3D97%26spage%3D1465%26epage%3D8%26doi%3D10.1038%2Fsj.bjc.6604049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group">Stangier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemens, A.</span><span> </span><span class="NLM_article-title">Pharmacology, pharmacokinetics, and pharmacodynamics of dabigatran etexilate, an oral direct thrombin inhibitor</span> <span class="citation_source-journal">Clin. Appl. Thromb./Hemostasis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">9S</span><span class="NLM_x">–</span> <span class="NLM_lpage">16S</span><span class="refDoi"> DOI: 10.1177/1076029609343004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1177%2F1076029609343004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19696042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSksLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=9S-16S&author=J.+Stangierauthor=A.+Clemens&title=Pharmacology%2C+pharmacokinetics%2C+and+pharmacodynamics+of+dabigatran+etexilate%2C+an+oral+direct+thrombin+inhibitor&doi=10.1177%2F1076029609343004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacology, pharmacokinetics, and pharmacodynamics of dabigatran etexilate, an oral direct thrombin inhibitor</span></div><div class="casAuthors">Stangier, Joachim; Clemens, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Applied Thrombosis/Hemostasis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">9S-16S</span>CODEN:
                <span class="NLM_cas:coden">CATHF9</span>;
        ISSN:<span class="NLM_cas:issn">1076-0296</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Dabigatran etexilate is a novel, oral reversible direct thrombin inhibitor that is rapidly absorbed and converted to its active form, dabigatran.  Dabigatran has been shown to be a potent, competitive, and reversible inhibitor of thrombin, inhibiting both thrombin activity and generation.  Studies in healthy volunteers and in patients undergoing orthopedic surgery indicate that dabigatran has a predictable pharmacokinetic profile, allowing for a fixed-dose regimen without the need for coagulation monitoring.  In healthy volunteers, peak plasma concns. of dabigatran are reached approx. 2 h after oral administration.  The elimination half-life is 12 to 14 h, with clearance predominantly occurring via renal excretion of unchanged drug.  Dabigatran is not metabolized by cytochrome P 450 isoenzymes, has no interactions with food, and also has a low potential for drug-drug interactions.  The pharmacokinetic profile of dabigatran is consistent across a broad range of different patient populations and is unaffected by gender, body wt., ethnic origin, obesity, and mild-to-moderate hepatic impairment.  Small differences in dabigatran pharmacokinetics assocd. with age are attributable to variation in renal function.  Dabigatran etexilate produces a predictable pharmacodynamic effect and requires no coagulation monitoring.  It has been approved in the European Union (EU) and Canada for prophylaxis of thromboembolism in patients undergoing total knee or hip arthroplasty.  Ongoing clin. trials are investigating its use in the treatment of venous thromboembolism, prevention of stroke in patients with nonvalvular atrial fibrillation, and treatment of thromboembolic complications, following acute coronary syndromes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz7HdOwdwOFrVg90H21EOLACvtfcHk0lime7OKcMPUjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSksLzE&md5=47f0699335b37a2133ea1a9ea74385dd</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1177%2F1076029609343004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1076029609343004%26sid%3Dliteratum%253Aachs%26aulast%3DStangier%26aufirst%3DJ.%26aulast%3DClemens%26aufirst%3DA.%26atitle%3DPharmacology%252C%2520pharmacokinetics%252C%2520and%2520pharmacodynamics%2520of%2520dabigatran%2520etexilate%252C%2520an%2520oral%2520direct%2520thrombin%2520inhibitor%26jtitle%3DClin.%2520Appl.%2520Thromb.%252FHemostasis%26date%3D2009%26volume%3D15%26spage%3D9S%26epage%3D16S%26doi%3D10.1177%2F1076029609343004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group">Kong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, J. F.</span><span> </span><span class="NLM_article-title">Direct thrombin inhibitors: Patents 2002–2012 (review)</span> <span class="citation_source-journal">Mol. Med. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1506</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span><span class="refDoi"> DOI: 10.3892/mmr.2014.2025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.3892%2Fmmr.2014.2025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=24604304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVeltLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1506-14&author=Y.+Kongauthor=H.+Chenauthor=Y.+Q.+Wangauthor=L.+Mengauthor=J.+F.+Wei&title=Direct+thrombin+inhibitors%3A+Patents+2002%E2%80%932012+%28review%29&doi=10.3892%2Fmmr.2014.2025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Direct thrombin inhibitors: patents 2002-2012 (Review)</span></div><div class="casAuthors">Kong, Yi; Chen, Hao; Wang, Yong-Qing; Meng, Ling; Wei, Ji-Fu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1506-1514</span>CODEN:
                <span class="NLM_cas:coden">MMROA5</span>;
        ISSN:<span class="NLM_cas:issn">1791-2997</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Acute vascular diseases and other thromboses of the blood system constitute major health risks in developing countries.  Thrombin plays a central role in blood coagulation, which is a crucial process involved in thrombosis.  Direct thrombin inhibitors (DTIs) such as argatroban, dabigatran, dabigatran etexilate, lepirudin, desirudin and bivalirudin, which bind to thrombin and block its enzymic activity, are widely and effectively used in the treatment of thromboembolic diseases.  DTIs appear to overcome the disadvantages of indirect thrombin inhibitors such as unfractionated heparins (UFH).  Although these DTIs show specific advantages over indirect inhibitors, they still present limitations, such as a narrow therapeutic window, and bleeding and anaphylaxis as side-effects.  Novel anticoagulant drugs need thus to be developed to overcome these limitations.  In the search for addnl. candidate agents with improved efficacy, safety and high bioavailability in oral administration, a high no. of compds. has been identified, such as those derived from the tripeptide template D-Phe-Pro-Arg, aptamers and peptides isolated from blood-sucking animals.  These candidates may prove the new agents of choice for the treatment of cardiovascular diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_V_SPDTnH7rVg90H21EOLACvtfcHk0ljf2CthNiPCcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVeltLrN&md5=dd0795d9899c52c415b542d34bcfb93f</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.3892%2Fmmr.2014.2025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fmmr.2014.2025%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%2BQ.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DJ.%2BF.%26atitle%3DDirect%2520thrombin%2520inhibitors%253A%2520Patents%25202002%25E2%2580%25932012%2520%2528review%2529%26jtitle%3DMol.%2520Med.%2520Rep.%26date%3D2014%26volume%3D9%26spage%3D1506%26epage%3D14%26doi%3D10.3892%2Fmmr.2014.2025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group">Fu, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neu, H. C.</span><span> </span><span class="NLM_article-title">Piperacillin, a new penicillin active against many bacteria resistant to other penicillins</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1978</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1128/AAC.13.3.358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1128%2FAAC.13.3.358" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1978&pages=358-67&author=K.+P.+Fuauthor=H.+C.+Neu&title=Piperacillin%2C+a+new+penicillin+active+against+many+bacteria+resistant+to+other+penicillins&doi=10.1128%2FAAC.13.3.358"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1128%2FAAC.13.3.358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.13.3.358%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DK.%2BP.%26aulast%3DNeu%26aufirst%3DH.%2BC.%26atitle%3DPiperacillin%252C%2520a%2520new%2520penicillin%2520active%2520against%2520many%2520bacteria%2520resistant%2520to%2520other%2520penicillins%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1978%26volume%3D13%26spage%3D358%26epage%3D67%26doi%3D10.1128%2FAAC.13.3.358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group">van Berkel, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettleship, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brem, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuart, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claridge, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owens, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Binding of (5s)-penicilloic acid to penicillin binding protein 3</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2112</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1021/cb400200h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400200h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2112-6&author=S.+S.+van+Berkelauthor=J.+E.+Nettleshipauthor=I.+K.+Leungauthor=J.+Bremauthor=H.+Choiauthor=D.+I.+Stuartauthor=T.+D.+Claridgeauthor=M.+A.+McDonoughauthor=R.+J.+Owensauthor=J.+Renauthor=C.+J.+Schofield&title=Binding+of+%285s%29-penicilloic+acid+to+penicillin+binding+protein+3&doi=10.1021%2Fcb400200h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Fcb400200h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400200h%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BBerkel%26aufirst%3DS.%2BS.%26aulast%3DNettleship%26aufirst%3DJ.%2BE.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DBrem%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DStuart%26aufirst%3DD.%2BI.%26aulast%3DClaridge%26aufirst%3DT.%2BD.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOwens%26aufirst%3DR.%2BJ.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DBinding%2520of%2520%25285s%2529-penicilloic%2520acid%2520to%2520penicillin%2520binding%2520protein%25203%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2112%26epage%3D6%26doi%3D10.1021%2Fcb400200h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group">Patchett, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tristram, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyvratt, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taub, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeler, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ten Broeke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ondeyka, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorsett, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenlee, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohr, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffsommer, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshua, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruyle, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothrock, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aster, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maycock, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirschmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulm, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassil, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, C. A.</span><span> </span><span class="NLM_article-title">A new class of angiotensin-converting enzyme inhibitors</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">280</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span><span class="refDoi"> DOI: 10.1038/288280a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2F288280a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=6253826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaL3MXktlGhtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=1980&pages=280-3&author=A.+A.+Patchettauthor=E.+Harrisauthor=E.+W.+Tristramauthor=M.+J.+Wyvrattauthor=M.+T.+Wuauthor=D.+Taubauthor=E.+R.+Petersonauthor=T.+J.+Ikelerauthor=J.+ten+Broekeauthor=L.+G.+Payneauthor=D.+L.+Ondeykaauthor=E.+D.+Thorsettauthor=W.+J.+Greenleeauthor=N.+S.+Lohrauthor=R.+D.+Hoffsommerauthor=H.+Joshuaauthor=W.+V.+Ruyleauthor=J.+W.+Rothrockauthor=S.+D.+Asterauthor=A.+L.+Maycockauthor=F.+M.+Robinsonauthor=R.+Hirschmannauthor=C.+S.+Sweetauthor=E.+H.+Ulmauthor=D.+M.+Grossauthor=T.+C.+Vassilauthor=C.+A.+Stone&title=A+new+class+of+angiotensin-converting+enzyme+inhibitors&doi=10.1038%2F288280a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">A new class of angiotensin-converting enzyme inhibitors</span></div><div class="casAuthors">Patchett, A. A.; Harris, E.; Tristram, E. W.; Wyvratt, M. J.; Wu, M. T.; Taub, D.; Peterson, E. R.; Ikeler, T. J.; Ten Broeke, J.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">5788</span>),
    <span class="NLM_cas:pages">280-3</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Substituted N-carboxymethyldipeptides were evaluated for their ability to antagonize the pressor effect of angiotensin I (I)  [9041-90-1] in anesthetized rats and dogs (100 and 600 ng/kg, i.v., resp.); Ph(CH2)2CH(CO2H)-Ala-Pro (II)  [76420-72-9], Ph(CH2)2CH(CO2Et)-Ala-Pro maleate (III)  [75847-73-3] and Ph(CH2)2CH(CO2H)-Lys-Pro (IV)  [77726-95-5] were highly active by the i.v. route in rats and dogs, except for III in dogs, but all were orally active in rats and dogs.  II, III, and IV showed high potency in inhibiting the blood pressure response to I and all had a long duration of action; high angiotensin-converting enzyme (EC 3.4.15.1) (V)  [9015-82-1] inhibitory activities at doses as low as 2.3 μg/kg were obsd. after i.v. administration and after oral administration, the compds. showed good activities with duration of action of ≥6 h at doses of 0.3 mg/kg.  Thus, V inhibitors have important design implications for the synthesis of inhibitors of other metallodipeptidyl carboxypeptidases and possibly of metalloendopeptidases more generally.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2vFzl18NAf7Vg90H21EOLACvtfcHk0ljf2CthNiPCcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXktlGhtLY%253D&md5=3a395d264e4e5c14d0cca0bcb6065a73</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1038%2F288280a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F288280a0%26sid%3Dliteratum%253Aachs%26aulast%3DPatchett%26aufirst%3DA.%2BA.%26aulast%3DHarris%26aufirst%3DE.%26aulast%3DTristram%26aufirst%3DE.%2BW.%26aulast%3DWyvratt%26aufirst%3DM.%2BJ.%26aulast%3DWu%26aufirst%3DM.%2BT.%26aulast%3DTaub%26aufirst%3DD.%26aulast%3DPeterson%26aufirst%3DE.%2BR.%26aulast%3DIkeler%26aufirst%3DT.%2BJ.%26aulast%3Dten%2BBroeke%26aufirst%3DJ.%26aulast%3DPayne%26aufirst%3DL.%2BG.%26aulast%3DOndeyka%26aufirst%3DD.%2BL.%26aulast%3DThorsett%26aufirst%3DE.%2BD.%26aulast%3DGreenlee%26aufirst%3DW.%2BJ.%26aulast%3DLohr%26aufirst%3DN.%2BS.%26aulast%3DHoffsommer%26aufirst%3DR.%2BD.%26aulast%3DJoshua%26aufirst%3DH.%26aulast%3DRuyle%26aufirst%3DW.%2BV.%26aulast%3DRothrock%26aufirst%3DJ.%2BW.%26aulast%3DAster%26aufirst%3DS.%2BD.%26aulast%3DMaycock%26aufirst%3DA.%2BL.%26aulast%3DRobinson%26aufirst%3DF.%2BM.%26aulast%3DHirschmann%26aufirst%3DR.%26aulast%3DSweet%26aufirst%3DC.%2BS.%26aulast%3DUlm%26aufirst%3DE.%2BH.%26aulast%3DGross%26aufirst%3DD.%2BM.%26aulast%3DVassil%26aufirst%3DT.%2BC.%26aulast%3DStone%26aufirst%3DC.%2BA.%26atitle%3DA%2520new%2520class%2520of%2520angiotensin-converting%2520enzyme%2520inhibitors%26jtitle%3DNature%26date%3D1980%26volume%3D288%26spage%3D280%26epage%3D3%26doi%3D10.1038%2F288280a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group">Gubareva, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayden, F. G.</span><span> </span><span class="NLM_article-title">Influenza virus neuraminidase inhibitors</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">355</span><span class="NLM_x">, </span> <span class="NLM_fpage">827</span><span class="NLM_x">–</span> <span class="NLM_lpage">35</span><span class="refDoi"> DOI: 10.1016/S0140-6736(99)11433-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0140-6736%2899%2911433-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10711940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVSnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2000&pages=827-35&author=L.+V.+Gubarevaauthor=L.+Kaiserauthor=F.+G.+Hayden&title=Influenza+virus+neuraminidase+inhibitors&doi=10.1016%2FS0140-6736%2899%2911433-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Influenza virus neuraminidase inhibitors</span></div><div class="casAuthors">Gubareva, L. V.; Kaiser, L.; Hayden, F. G.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">9206</span>),
    <span class="NLM_cas:pages">827-835</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Lancet Ltd.</span>)
        </div><div class="casAbstract">A review with 69 refs.  Neuraminidase promotes influenza virus release from infected cells and facilitates virus spread within the respiratory tract.  Several potent and specific inhibitors of this enzyme have been developed, and two (zanamivir and oseltamivir) have been approved for human use.  Unlike amantadine and rimantadine that target the M2 protein of influenza A viruses, these drugs inhibit replication of both influenza A and B viruses.  Zanamivir is delivered by inhalation because of its low oral bioavailability whereas oseltamivir is administered by mouth.  Early treatment with either drug reduces the severity and duration of influenza symptoms and assocd. complications.  Both agents are effective for chemoprophylaxis.  Because of a broader antiviral spectrum, better tolerance, and less potential for emergence of resistance than is seen with the M2 inhibitors, the neuraminidase inhibitors represent an important advance in the treatment of influenza.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppRfygdS17K7Vg90H21EOLACvtfcHk0lhVgmIaRmnMCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVSnu7s%253D&md5=4b996d708c2f8e5ae7f4634190b05f27</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2899%2911433-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252899%252911433-8%26sid%3Dliteratum%253Aachs%26aulast%3DGubareva%26aufirst%3DL.%2BV.%26aulast%3DKaiser%26aufirst%3DL.%26aulast%3DHayden%26aufirst%3DF.%2BG.%26atitle%3DInfluenza%2520virus%2520neuraminidase%2520inhibitors%26jtitle%3DLancet%26date%3D2000%26volume%3D355%26spage%3D827%26epage%3D35%26doi%3D10.1016%2FS0140-6736%2899%2911433-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group">Maignan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guilloteau, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pouzieux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi-Sledeski, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, S. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ewing, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pauls, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spada, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikol, V.</span><span> </span><span class="NLM_article-title">Crystal structures of human factor Xa complexed with potent inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">3226</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1021/jm000940u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000940u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsVejtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3226-32&author=S.+Maignanauthor=J.+P.+Guilloteauauthor=S.+Pouzieuxauthor=Y.+M.+Choi-Sledeskiauthor=M.+R.+Beckerauthor=S.+I.+Kleinauthor=W.+R.+Ewingauthor=H.+W.+Paulsauthor=A.+P.+Spadaauthor=V.+Mikol&title=Crystal+structures+of+human+factor+Xa+complexed+with+potent+inhibitors&doi=10.1021%2Fjm000940u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures of Human Factor Xa Complexed with Potent Inhibitors</span></div><div class="casAuthors">Maignan, Sebastien; Guilloteau, Jean-Pierre; Pouzieux, Stephanie; Choi-Sledeski, Yong Mi; Becker, Michael R.; Klein, Scott I.; Ewing, William R.; Pauls, Henry W.; Spada, Alfred P.; Mikol, Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3226-3232</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Involved in the coagulation cascade, factor Xa (FXa) is a serine protease which has received great interest as a potential target for the development of new antithrombotics.  Although there is a great wealth of structural data on thrombin complexes, few structures of ligand/FXa complexes have been reported, presumably because of the difficulty in growing crystals.  Reproducible crystn. conditions for human des-Gla1-45 coagulation FXa have been found.  This has led to an improvement in the diffraction quality of the crystals (about 2.1 Å) when compared to the previously reported forms (2.3-2.8 Å) thus providing a suitable platform for a structure-based drug design approach.  A series of crystal structures of noncovalent inhibitors complexed with FXa have been detd., three of which are presented herein.  These include compds. contg. the benzamidine moiety and surrogates of the basic group.  The benzamidine-contg. compd. binds in a canonical fashion typical of synthetic serine protease inhibitors.  On the contrary, mols. that contain surrogates of the benzamidine group do not make direct hydrogen-bonding interactions with the carboxylate of Asp189 at the bottom of the S1 pocket.  The structural data provide a likely explanation for the specificity of these inhibitors and a great aid in the design of bioavailable potent FXa inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsMdvctURxlLVg90H21EOLACvtfcHk0lhVgmIaRmnMCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsVejtb0%253D&md5=d88dd067911129d813294208f5ffbc35</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Fjm000940u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000940u%26sid%3Dliteratum%253Aachs%26aulast%3DMaignan%26aufirst%3DS.%26aulast%3DGuilloteau%26aufirst%3DJ.%2BP.%26aulast%3DPouzieux%26aufirst%3DS.%26aulast%3DChoi-Sledeski%26aufirst%3DY.%2BM.%26aulast%3DBecker%26aufirst%3DM.%2BR.%26aulast%3DKlein%26aufirst%3DS.%2BI.%26aulast%3DEwing%26aufirst%3DW.%2BR.%26aulast%3DPauls%26aufirst%3DH.%2BW.%26aulast%3DSpada%26aufirst%3DA.%2BP.%26aulast%3DMikol%26aufirst%3DV.%26atitle%3DCrystal%2520structures%2520of%2520human%2520factor%2520Xa%2520complexed%2520with%2520potent%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3226%26epage%3D32%26doi%3D10.1021%2Fjm000940u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group">Istvan, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deisenhofer, J.</span><span> </span><span class="NLM_article-title">Structural mechanism for statin inhibition of HMG-CoA reductase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">292</span><span class="NLM_x">, </span> <span class="NLM_fpage">1160</span><span class="NLM_x">–</span> <span class="NLM_lpage">4</span><span class="refDoi"> DOI: 10.1126/science.1059344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1126%2Fscience.1059344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11349148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjs1Cgtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2001&pages=1160-4&author=E.+S.+Istvanauthor=J.+Deisenhofer&title=Structural+mechanism+for+statin+inhibition+of+HMG-CoA+reductase&doi=10.1126%2Fscience.1059344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism for statin inhibition of HMG-CoA reductase</span></div><div class="casAuthors">Istvan, Eva S.; Deisenhofer, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">5519</span>),
    <span class="NLM_cas:pages">1160-1164</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">HMG-CoA (3-hydroxy-3-methylglutaryl-CoA) reductase (HMGR) catalyzes the committed step in cholesterol biosynthesis.  Statins are HMGR inhibitors with inhibition const. values in the nanomolar range that effectively lower serum cholesterol levels and are widely prescribed in the treatment of hypercholesterolemia.  We have detd. structures of the catalytic portion of human HMGR complexed with six different statins.  The statins occupy a portion of the binding site of HMG-CoA, thus blocking access of this substrate to the active site.  Near the carboxyl terminus of HMGR, several catalytically relevant residues are disordered in the enzyme-statin complexes.  If these residues were not flexible, they would sterically hinder statin binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDsSOnK9675bVg90H21EOLACvtfcHk0lhVgmIaRmnMCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjs1Cgtbo%253D&md5=235644ee8cca3d2e061093e93bd1d702</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1126%2Fscience.1059344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1059344%26sid%3Dliteratum%253Aachs%26aulast%3DIstvan%26aufirst%3DE.%2BS.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DStructural%2520mechanism%2520for%2520statin%2520inhibition%2520of%2520HMG-CoA%2520reductase%26jtitle%3DScience%26date%3D2001%26volume%3D292%26spage%3D1160%26epage%3D4%26doi%3D10.1126%2Fscience.1059344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group">Barrett, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deaton, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassell, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFadyen, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shewchuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, D. H.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, L. L.</span><span> </span><span class="NLM_article-title">Acyclic cyanamide-based inhibitors of cathepsin K</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3039</span><span class="NLM_x">–</span> <span class="NLM_lpage">43</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2005.04.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2005.04.032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=3039-43&author=D.+G.+Barrettauthor=D.+N.+Deatonauthor=A.+M.+Hassellauthor=R.+B.+McFadyenauthor=A.+B.+Millerauthor=L.+R.+Millerauthor=J.+A.+Payneauthor=L.+M.+Shewchukauthor=D.+H.+Willardauthor=L.+L.+Wright&title=Acyclic+cyanamide-based+inhibitors+of+cathepsin+K&doi=10.1016%2Fj.bmcl.2005.04.032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.04.032%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DD.%2BG.%26aulast%3DDeaton%26aufirst%3DD.%2BN.%26aulast%3DHassell%26aufirst%3DA.%2BM.%26aulast%3DMcFadyen%26aufirst%3DR.%2BB.%26aulast%3DMiller%26aufirst%3DA.%2BB.%26aulast%3DMiller%26aufirst%3DL.%2BR.%26aulast%3DPayne%26aufirst%3DJ.%2BA.%26aulast%3DShewchuk%26aufirst%3DL.%2BM.%26aulast%3DWillard%26aufirst%3DD.%2BH.%26aulast%3DWright%26aufirst%3DL.%2BL.%26atitle%3DAcyclic%2520cyanamide-based%2520inhibitors%2520of%2520cathepsin%2520K%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D3039%26epage%3D43%26doi%3D10.1016%2Fj.bmcl.2005.04.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group">Gauthier, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauret, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cromlish, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desmarais, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duong, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Falgueyret, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimmel, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamontagne, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeRiche, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKay, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicoll-Griffith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oballa, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Percival, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riendeau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robichaud, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodan, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodan, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seto, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truong, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venuti, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wesolowski, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamboni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, W. C.</span><span> </span><span class="NLM_article-title">The discovery of odanacatib (MK-0822), a selective inhibitor of cathepsin K</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">923</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.12.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2007.12.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18226527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=923-8&author=J.+Y.+Gauthierauthor=N.+Chauretauthor=W.+Cromlishauthor=S.+Desmaraisauthor=L.+T.+Duongauthor=J.+P.+Falgueyretauthor=D.+B.+Kimmelauthor=S.+Lamontagneauthor=S.+Legerauthor=T.+LeRicheauthor=C.+S.+Liauthor=F.+Masseauthor=D.+J.+McKayauthor=D.+A.+Nicoll-Griffithauthor=R.+M.+Oballaauthor=J.+T.+Palmerauthor=M.+D.+Percivalauthor=D.+Riendeauauthor=J.+Robichaudauthor=G.+A.+Rodanauthor=S.+B.+Rodanauthor=C.+Setoauthor=M.+Therienauthor=V.+L.+Truongauthor=M.+C.+Venutiauthor=G.+Wesolowskiauthor=R.+N.+Youngauthor=R.+Zamboniauthor=W.+C.+Black&title=The+discovery+of+odanacatib+%28MK-0822%29%2C+a+selective+inhibitor+of+cathepsin+K&doi=10.1016%2Fj.bmcl.2007.12.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of odanacatib (MK-0822), a selective inhibitor of cathepsin K</span></div><div class="casAuthors">Gauthier, Jacques Yves; Chauret, Nathalie; Cromlish, Wanda; Desmarais, Sylvie; Duong, Le T.; Falgueyret, Jean-Pierre; Kimmel, Donald B.; Lamontagne, Sonia; Leger, Serge; LeRiche, Tammy; Li, Chun Sing; Masse, Frederic; McKay, Daniel J.; Nicoll-Griffith, Deborah A.; Oballa, Renata M.; Palmer, James T.; Percival, M. David; Riendeau, Denis; Robichaud, Joel; Rodan, Gideon A.; Rodan, Sevgi B.; Seto, Carmai; Therien, Michel; Truong, Vouy-Linh; Venuti, Michael C.; Wesolowski, Gregg; Young, Robert N.; Zamboni, Robert; Black, W. Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">923-928</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Odanacatib is a potent, selective, and neutral cathepsin K inhibitor which was developed to address the metabolic liabilities of the Cat K inhibitor L-873724.  Substituting P1 and modifying the P2 side chain led to a metabolically robust inhibitor with a long half-life in preclin. species.  Odanacatib was more selective in whole cell assays than the published Cat K inhibitors balicatib and relacatib.  Evaluation in dermal fibroblast culture showed minimal intracellular collagen accumulation relative to less selective Cat K inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ytNI-SBgarVg90H21EOLACvtfcHk0ljfxdiupdQJ5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWktro%253D&md5=e432a420a3ec99c784117750b9bbeda8</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.12.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.12.047%26sid%3Dliteratum%253Aachs%26aulast%3DGauthier%26aufirst%3DJ.%2BY.%26aulast%3DChauret%26aufirst%3DN.%26aulast%3DCromlish%26aufirst%3DW.%26aulast%3DDesmarais%26aufirst%3DS.%26aulast%3DDuong%26aufirst%3DL.%2BT.%26aulast%3DFalgueyret%26aufirst%3DJ.%2BP.%26aulast%3DKimmel%26aufirst%3DD.%2BB.%26aulast%3DLamontagne%26aufirst%3DS.%26aulast%3DLeger%26aufirst%3DS.%26aulast%3DLeRiche%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DMasse%26aufirst%3DF.%26aulast%3DMcKay%26aufirst%3DD.%2BJ.%26aulast%3DNicoll-Griffith%26aufirst%3DD.%2BA.%26aulast%3DOballa%26aufirst%3DR.%2BM.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26aulast%3DPercival%26aufirst%3DM.%2BD.%26aulast%3DRiendeau%26aufirst%3DD.%26aulast%3DRobichaud%26aufirst%3DJ.%26aulast%3DRodan%26aufirst%3DG.%2BA.%26aulast%3DRodan%26aufirst%3DS.%2BB.%26aulast%3DSeto%26aufirst%3DC.%26aulast%3DTherien%26aufirst%3DM.%26aulast%3DTruong%26aufirst%3DV.%2BL.%26aulast%3DVenuti%26aufirst%3DM.%2BC.%26aulast%3DWesolowski%26aufirst%3DG.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZamboni%26aufirst%3DR.%26aulast%3DBlack%26aufirst%3DW.%2BC.%26atitle%3DThe%2520discovery%2520of%2520odanacatib%2520%2528MK-0822%2529%252C%2520a%2520selective%2520inhibitor%2520of%2520cathepsin%2520K%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D923%26epage%3D8%26doi%3D10.1016%2Fj.bmcl.2007.12.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group">Brixen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapurlat, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keaveny, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuerst, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recker, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dardzinski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verbruggen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ather, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Papp, A. E.</span><span> </span><span class="NLM_article-title">Bone density, turnover, and estimated strength in postmenopausal women treated with odanacatib: A randomized trial</span> <span class="citation_source-journal">J. Clin. Endocrinol. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1210/jc.2012-2972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1210%2Fjc.2012-2972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23337728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlOks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2013&pages=571-80&author=K.+Brixenauthor=R.+Chapurlatauthor=A.+M.+Cheungauthor=T.+M.+Keavenyauthor=T.+Fuerstauthor=K.+Engelkeauthor=R.+Reckerauthor=B.+Dardzinskiauthor=N.+Verbruggenauthor=S.+Atherauthor=E.+Rosenbergauthor=A.+E.+de+Papp&title=Bone+density%2C+turnover%2C+and+estimated+strength+in+postmenopausal+women+treated+with+odanacatib%3A+A+randomized+trial&doi=10.1210%2Fjc.2012-2972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Bone density, turnover, and estimated strength in postmenopausal women treated with odanacatib: a randomized trial</span></div><div class="casAuthors">Brixen, Kim; Chapurlat, Roland; Cheung, Angela M.; Keaveny, Tony M.; Fuers, Thomas; Engelke, Klaus; Recker, Robert; Dardzinski, Bernard; Verbruggen, Nadia; Ather, Shabana; Rosenberg, Elizabeth; de Papp, Anne E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">571-580</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Context: Odanacatib, a cathepsin K inhibitor, increases spine and hip areal bone mineral d. (BMD) in postmenopausal women with low BMD and cortical thickness in ovariectomized monkeys.  Objective: The objective of the study was to examine the impact of odanacatib on the trabecular and cortical bone compartments and estd. strength at the hip and spine.  Design: This was a randomized, double-blind, 2-yr trial.  Setting: The study was conducted at a private or institutional practice.  Participants: Participants included 214 postmenopausal women with low areal BMD.  Intervention: The intervention included odanacatib 50 mg or placebo weekly.  Main Outcome Measures: Changes in areal BMD by dual-energy x-ray absorptiometry (primary end point, 1 yr areal BMD change at lumbar spine), bone turnover markers, volumetric BMD by quant. computed tomog. (QCT), and bone strength estd. by finite element anal. were measured.  Results: Year 1 lumbar spine areal BMD percent change from baseline was 3.5% greater with odanacatib than placebo (P < .001).  Bone-resorption marker C-telopeptide of type 1 collagen was significantly lower with odanacatib vs placebo at 6 mo and 2 years (P < .001).  Bone-formation marker procollagen I N-terminal peptide initially decreased with odanacatib but by 2 years did not differ from placebo.  After 6 mo, odanacatib-treated women had greater increases in trabecular volumetric BMD and estd. compressive strength at the spine and integral and trabecular volumetric BMD and estd. strength at the hip (P < .001).  At the cortical envelope of the femoral neck, bone mineral content, thickness, vol., and cross-sectional area also increased from baseline with odanacatib vs placebo (P < .001 at 24 mo).  Adverse experiences were similar between groups.  Conclusions: Over 2 years, odanacatib decreased bone resorption, maintained bone formation, increased areal and volumetric BMD, and increased estd. bone strength at both the hip and spine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5IgjyjsYm47Vg90H21EOLACvtfcHk0ljfxdiupdQJ5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlOks74%253D&md5=2dc289cb3ccc6576fb66711b6f69be8d</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1210%2Fjc.2012-2972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2012-2972%26sid%3Dliteratum%253Aachs%26aulast%3DBrixen%26aufirst%3DK.%26aulast%3DChapurlat%26aufirst%3DR.%26aulast%3DCheung%26aufirst%3DA.%2BM.%26aulast%3DKeaveny%26aufirst%3DT.%2BM.%26aulast%3DFuerst%26aufirst%3DT.%26aulast%3DEngelke%26aufirst%3DK.%26aulast%3DRecker%26aufirst%3DR.%26aulast%3DDardzinski%26aufirst%3DB.%26aulast%3DVerbruggen%26aufirst%3DN.%26aulast%3DAther%26aufirst%3DS.%26aulast%3DRosenberg%26aufirst%3DE.%26aulast%3Dde%2BPapp%26aufirst%3DA.%2BE.%26atitle%3DBone%2520density%252C%2520turnover%252C%2520and%2520estimated%2520strength%2520in%2520postmenopausal%2520women%2520treated%2520with%2520odanacatib%253A%2520A%2520randomized%2520trial%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2013%26volume%3D98%26spage%3D571%26epage%3D80%26doi%3D10.1210%2Fjc.2012-2972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group">Summa, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrocchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crescenzi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiore, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez Paz, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzel, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laufer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteagudo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muraglia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nizi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orvieto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pescatore, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarpelli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stillmock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witmer, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, M.</span><span> </span><span class="NLM_article-title">Discovery of raltegravir, a potent, selective orally bioavailable HIV-integrase inhibitor for the treatment of HIV-AIDS infection</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5843</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span><span class="refDoi"> DOI: 10.1021/jm800245z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800245z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5843-55&author=V.+Summaauthor=A.+Petrocchiauthor=F.+Bonelliauthor=B.+Crescenziauthor=M.+Donghiauthor=M.+Ferraraauthor=F.+Fioreauthor=C.+Gardelliauthor=O.+Gonzalez+Pazauthor=D.+J.+Hazudaauthor=P.+Jonesauthor=O.+Kinzelauthor=R.+Lauferauthor=E.+Monteagudoauthor=E.+Muragliaauthor=E.+Niziauthor=F.+Orvietoauthor=P.+Paceauthor=G.+Pescatoreauthor=R.+Scarpelliauthor=K.+Stillmockauthor=M.+V.+Witmerauthor=M.+Rowley&title=Discovery+of+raltegravir%2C+a+potent%2C+selective+orally+bioavailable+HIV-integrase+inhibitor+for+the+treatment+of+HIV-AIDS+infection&doi=10.1021%2Fjm800245z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Raltegravir, a Potent, Selective Orally Bioavailable HIV-Integrase Inhibitor for the Treatment of HIV-AIDS Infection</span></div><div class="casAuthors">Summa, Vincenzo; Petrocchi, Alessia; Bonelli, Fabio; Crescenzi, Benedetta; Donghi, Monica; Ferrara, Marco; Fiore, Fabrizio; Gardelli, Cristina; Gonzalez Paz, Odalys; Hazuda, Daria J.; Jones, Philip; Kinzel, Olaf; Laufer, Ralph; Monteagudo, Edith; Muraglia, Ester; Nizi, Emanuela; Orvieto, Federica; Pace, Paola; Pescatore, Giovanna; Scarpelli, Rita; Stillmock, Kara; Witmer, Marc V.; Rowley, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5843-5855</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human immunodeficiency virus type-1 (HIV-1) integrase is one of the three virally encoded enzymes required for replication and therefore a rational target for chemotherapeutic intervention in the treatment of HIV-1 infection.  We report here the discovery of Raltegravir, the first HIV-integrase inhibitor approved by FDA for the treatment of HIV infection.  It derives from the evolution of 5,6-dihydroxypyrimidine-4-carboxamides and N-methyl-4-hydroxypyrimidinone-carboxamides, which exhibited potent inhibition of the HIV-integrase catalyzed strand transfer process.  Structural modifications on these mols. were made in order to maximize potency as HIV-integrase inhibitors against the wild type virus, a selection of mutants, and optimize the selectivity, pharmacokinetic, and metabolic profiles in preclin. species.  The good profile of Raltegravir has enabled its progression toward the end of phase III clin. trials for the treatment of HIV-1 infection and culminated with the FDA approval as the first HIV-integrase inhibitor for the treatment of HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYo6KKNjJRCrVg90H21EOLACvtfcHk0lgz8QORqchcYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb%252FM&md5=f90f994c7cd031a4063cbb3e91b4c3f0</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Fjm800245z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800245z%26sid%3Dliteratum%253Aachs%26aulast%3DSumma%26aufirst%3DV.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DBonelli%26aufirst%3DF.%26aulast%3DCrescenzi%26aufirst%3DB.%26aulast%3DDonghi%26aufirst%3DM.%26aulast%3DFerrara%26aufirst%3DM.%26aulast%3DFiore%26aufirst%3DF.%26aulast%3DGardelli%26aufirst%3DC.%26aulast%3DGonzalez%2BPaz%26aufirst%3DO.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DKinzel%26aufirst%3DO.%26aulast%3DLaufer%26aufirst%3DR.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DMuraglia%26aufirst%3DE.%26aulast%3DNizi%26aufirst%3DE.%26aulast%3DOrvieto%26aufirst%3DF.%26aulast%3DPace%26aufirst%3DP.%26aulast%3DPescatore%26aufirst%3DG.%26aulast%3DScarpelli%26aufirst%3DR.%26aulast%3DStillmock%26aufirst%3DK.%26aulast%3DWitmer%26aufirst%3DM.%2BV.%26aulast%3DRowley%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520raltegravir%252C%2520a%2520potent%252C%2520selective%2520orally%2520bioavailable%2520HIV-integrase%2520inhibitor%2520for%2520the%2520treatment%2520of%2520HIV-AIDS%2520infection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5843%26epage%3D55%26doi%3D10.1021%2Fjm800245z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group">Leung, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbenante, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlie, D. P.</span><span> </span><span class="NLM_article-title">Protease inhibitors: Current status and future prospects</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1021/jm990412m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm990412m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXpsVKhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=305-41&author=D.+Leungauthor=G.+Abbenanteauthor=D.+P.+Fairlie&title=Protease+inhibitors%3A+Current+status+and+future+prospects&doi=10.1021%2Fjm990412m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Protease inhibitors: Current status and future prospects</span></div><div class="casAuthors">Leung, Donmienne; Abbenante, Giovanni; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">305-341</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 354 refs. is given on Asp protease inhibitors (HIV-1 protease, renin, plasmepsins, cathepsin D, secreted Asp protease), Ser protease inhibitors (thrombin, factor Xa, elastase, tryptase, complement convertases, hepatitis C-NS3 protease, broad-spectrum Ser protease inhibitors), Cys protease inhibitors (cathepsin K, B, and L, caspases, rhinovirus 3C protease, calpains), and metalloprotease inhibitors (angiotensin-converting enzyme, neutral endopeptidase, matrix metalloprotease inhibitors, tumor necrosis factor-α-converting enzyme).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMWqeFAY1BO7Vg90H21EOLACvtfcHk0lgz8QORqchcYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXpsVKhsQ%253D%253D&md5=1190cf0b642169e340e3471cd893f217</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1021%2Fjm990412m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990412m%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DD.%26aulast%3DAbbenante%26aufirst%3DG.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DProtease%2520inhibitors%253A%2520Current%2520status%2520and%2520future%2520prospects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D305%26epage%3D41%26doi%3D10.1021%2Fjm990412m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group">Reigan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gbaj, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barry, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Page, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luke, R. W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douglas, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratford, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaffar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryce, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S.</span><span> </span><span class="NLM_article-title">Aminoimidazolylmethyluracil analogues as potent inhibitors of thymidine phosphorylase and their bioreductive nitroimidazolyl prodrugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">392</span><span class="NLM_x">–</span> <span class="NLM_lpage">402</span><span class="refDoi"> DOI: 10.1021/jm049494r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049494r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=392-402&author=P.+Reiganauthor=P.+N.+Edwardsauthor=A.+Gbajauthor=C.+Coleauthor=S.+T.+Barryauthor=K.+M.+Pageauthor=S.+E.+Ashtonauthor=R.+W.+A.+Lukeauthor=K.+T.+Douglasauthor=I.+J.+Stratfordauthor=M.+Jaffarauthor=R.+A.+Bryceauthor=S.+Freeman&title=Aminoimidazolylmethyluracil+analogues+as+potent+inhibitors+of+thymidine+phosphorylase+and+their+bioreductive+nitroimidazolyl+prodrugs&doi=10.1021%2Fjm049494r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1021%2Fjm049494r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049494r%26sid%3Dliteratum%253Aachs%26aulast%3DReigan%26aufirst%3DP.%26aulast%3DEdwards%26aufirst%3DP.%2BN.%26aulast%3DGbaj%26aufirst%3DA.%26aulast%3DCole%26aufirst%3DC.%26aulast%3DBarry%26aufirst%3DS.%2BT.%26aulast%3DPage%26aufirst%3DK.%2BM.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DLuke%26aufirst%3DR.%2BW.%2BA.%26aulast%3DDouglas%26aufirst%3DK.%2BT.%26aulast%3DStratford%26aufirst%3DI.%2BJ.%26aulast%3DJaffar%26aufirst%3DM.%26aulast%3DBryce%26aufirst%3DR.%2BA.%26aulast%3DFreeman%26aufirst%3DS.%26atitle%3DAminoimidazolylmethyluracil%2520analogues%2520as%2520potent%2520inhibitors%2520of%2520thymidine%2520phosphorylase%2520and%2520their%2520bioreductive%2520nitroimidazolyl%2520prodrugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D392%26epage%3D402%26doi%3D10.1021%2Fjm049494r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group">Manikowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lossani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gebhardt, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gambino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Focher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spadari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, G. E.</span><span> </span><span class="NLM_article-title">Inhibition of herpes simplex virus thymidine kinases by 2-phenylamino-6-oxopurines and related compounds: Structure-activity relationships and antiherpetic activity in vivo</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">3919</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1021/jm049059x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049059x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3919-29&author=A.+Manikowskiauthor=A.+Verriauthor=A.+Lossaniauthor=B.+M.+Gebhardtauthor=J.+Gambinoauthor=F.+Focherauthor=S.+Spadariauthor=G.+E.+Wright&title=Inhibition+of+herpes+simplex+virus+thymidine+kinases+by+2-phenylamino-6-oxopurines+and+related+compounds%3A+Structure-activity+relationships+and+antiherpetic+activity+in+vivo&doi=10.1021%2Fjm049059x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1021%2Fjm049059x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049059x%26sid%3Dliteratum%253Aachs%26aulast%3DManikowski%26aufirst%3DA.%26aulast%3DVerri%26aufirst%3DA.%26aulast%3DLossani%26aufirst%3DA.%26aulast%3DGebhardt%26aufirst%3DB.%2BM.%26aulast%3DGambino%26aufirst%3DJ.%26aulast%3DFocher%26aufirst%3DF.%26aulast%3DSpadari%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DG.%2BE.%26atitle%3DInhibition%2520of%2520herpes%2520simplex%2520virus%2520thymidine%2520kinases%2520by%25202-phenylamino-6-oxopurines%2520and%2520related%2520compounds%253A%2520Structure-activity%2520relationships%2520and%2520antiherpetic%2520activity%2520in%2520vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3919%26epage%3D29%26doi%3D10.1021%2Fjm049059x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group">Clements, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beckett, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catlin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lobell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salama, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barynin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, M. G.</span><span> </span><span class="NLM_article-title">Antibiotic activity and characterization of BB-3497, a novel peptide deformylase inhibitor</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1128/AAC.45.2.563-570.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1128%2FAAC.45.2.563-570.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11158755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmsVGktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2001&pages=563-70&author=J.+M.+Clementsauthor=R.+P.+Beckettauthor=A.+Brownauthor=G.+Catlinauthor=M.+Lobellauthor=S.+Palanauthor=W.+Thomasauthor=M.+Whittakerauthor=S.+Woodauthor=S.+Salamaauthor=P.+J.+Bakerauthor=H.+F.+Rodgersauthor=V.+Baryninauthor=D.+W.+Riceauthor=M.+G.+Hunter&title=Antibiotic+activity+and+characterization+of+BB-3497%2C+a+novel+peptide+deformylase+inhibitor&doi=10.1128%2FAAC.45.2.563-570.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic activity and characterization of BB-3497, a novel peptide deformylase inhibitor</span></div><div class="casAuthors">Clements, John M.; Beckett, R. Paul; Brown, Anthony; Catlin, Graham; Lobell, Mario; Palan, Shilpa; Thomas, Wayne; Whittaker, Mark; Wood, Stephen; Salama, Sameeh; Baker, Patrick J.; Rodgers, H. Fiona; Barynin, Vladimir; Rice, David W.; Hunter, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">563-570</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Peptide deformylase (PDF) is an essential bacterial metalloenzyme which deformylates the N-formylmethionine of newly synthesized polypeptides and as such represents a novel target for antibacterial chemotherapy.  To identify novel PDF inhibitors, we screened a metalloenzyme inhibitor library and identified an N-formyl-hydroxylamine deriv., BB-3497 (I, OHCN(OH)CH(Bu)CONHC(Bu-t)CONMe2), and a related natural hydroxamic acid antibiotic, actinonin (II), as potent and selective inhibitors of PDF.  To elucidate the interactions that contribute to the binding affinity of these inhibitors, we detd. the crystal structures of I and II bound to Escherichia coli PDF at resolns. of 2.1 and 1.75 Å, resp.  In both complexes, the active-site metal atom was pentacoordinated by the side chains of Cys 90, His 132, and His 136 and the two oxygen atoms of N-formylhydroxylamine or hydroxamate.  I had activity against gram-pos. bacteria, including methicillin-resistant Staphylococcus aureus and vancomycin-resistant Enterococcus faecalis, and activity against some gram-neg. bacteria.  Time-kill anal. showed that the mode of action of BB-3497 was primarily bacteriostatic.  The mechanism of resistance was via mutations within the formyltransferase gene, as previously described for actinonin.  While II and its derivs. have not been used clin. because of their poor pharmacokinetic properties, I was shown to be orally bioavailable.  A single oral dose of I given 1 h after i.p. injection of S. aureus Smith or methicillin-resistant S. aureus protected mice from infection with median EDs of 8 and 14 mg/kg of body wt., resp.  These data validate PDF as a novel target for the design of a new generation of antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHe1zSuU56sbVg90H21EOLACvtfcHk0lhWAAIviks1pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmsVGktA%253D%253D&md5=d7910e4f1a9e8f144e3517ab5e16d512</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1128%2FAAC.45.2.563-570.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.45.2.563-570.2001%26sid%3Dliteratum%253Aachs%26aulast%3DClements%26aufirst%3DJ.%2BM.%26aulast%3DBeckett%26aufirst%3DR.%2BP.%26aulast%3DBrown%26aufirst%3DA.%26aulast%3DCatlin%26aufirst%3DG.%26aulast%3DLobell%26aufirst%3DM.%26aulast%3DPalan%26aufirst%3DS.%26aulast%3DThomas%26aufirst%3DW.%26aulast%3DWhittaker%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DS.%26aulast%3DSalama%26aufirst%3DS.%26aulast%3DBaker%26aufirst%3DP.%2BJ.%26aulast%3DRodgers%26aufirst%3DH.%2BF.%26aulast%3DBarynin%26aufirst%3DV.%26aulast%3DRice%26aufirst%3DD.%2BW.%26aulast%3DHunter%26aufirst%3DM.%2BG.%26atitle%3DAntibiotic%2520activity%2520and%2520characterization%2520of%2520BB-3497%252C%2520a%2520novel%2520peptide%2520deformylase%2520inhibitor%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2001%26volume%3D45%26spage%3D563%26epage%3D70%26doi%3D10.1128%2FAAC.45.2.563-570.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group">Koropatkin, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleland, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holden, H. M.</span><span> </span><span class="NLM_article-title">Kinetic and structural analysis of alpha-d-glucose-1-phosphate cytidylyltransferase from salmonella typhi</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">10774</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1074/jbc.M414111200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1074%2Fjbc.M414111200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=10774-80&author=N.+M.+Koropatkinauthor=W.+W.+Clelandauthor=H.+M.+Holden&title=Kinetic+and+structural+analysis+of+alpha-d-glucose-1-phosphate+cytidylyltransferase+from+salmonella+typhi&doi=10.1074%2Fjbc.M414111200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M414111200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M414111200%26sid%3Dliteratum%253Aachs%26aulast%3DKoropatkin%26aufirst%3DN.%2BM.%26aulast%3DCleland%26aufirst%3DW.%2BW.%26aulast%3DHolden%26aufirst%3DH.%2BM.%26atitle%3DKinetic%2520and%2520structural%2520analysis%2520of%2520alpha-d-glucose-1-phosphate%2520cytidylyltransferase%2520from%2520salmonella%2520typhi%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D10774%26epage%3D80%26doi%3D10.1074%2Fjbc.M414111200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group">Heffron, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurnak, F.</span><span> </span><span class="NLM_article-title">Structure of an EF-Tu complex with a thiazolyl peptide antibiotic determined at 2.35 Å resolution: Atomic basis for GE2270A inhibition of EF-Tu</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.1021/bi9913597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9913597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaK1MXnslens7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=37-45&author=S.+E.+Heffronauthor=F.+Jurnak&title=Structure+of+an+EF-Tu+complex+with+a+thiazolyl+peptide+antibiotic+determined+at+2.35+%C3%85+resolution%3A+Atomic+basis+for+GE2270A+inhibition+of+EF-Tu&doi=10.1021%2Fbi9913597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of an EF-Tu Complex with a Thiazolyl Peptide Antibiotic Determined at 2.35 Å Resolution: Atomic Basis for GE2270A Inhibition of EF-Tu</span></div><div class="casAuthors">Heffron, Susan E.; Jurnak, Frances</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-45</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure of a 1:1 M complex between Escherichia coli elongation factor (EF) Tu-GDP and the cyclic thiazolyl peptide antibiotic, GE2270A, has been detd. by x-ray diffraction anal. to a resoln. of 2.35 Å and refined to a crystallog. refinement factor of 20.6%.  The antibiotic binds in the second domain of EF-Tu-GDP, making contact with three segments of amino acids (residues 215-230, 256-264, and 273-277).  The majority of the protein-antibiotic contacts are van der Waals interactions.  A striking feature of the antibiotic binding site is the presence of a salt bridge, not previously obsd. in other EF-Tu complexes.  The ionic interaction between Arg 223 and Glu 259 forms over the antibiotic and probably accounts for the strong affinity obsd. between EF-Tu and GE2270A.  Arg 223 and Glu 259 are highly conserved, but not invariant throughout the prokaryotic EF-Tu family, suggesting that the antibiotic may bind EF-Tu from some organisms better than others may.  Superposition of the antibiotic binding site on the EF-Tu-GTP conformation reveals that one region of the antibiotic would form steric clashes with the guanine nucleotide-binding domain in the GTP, but not the GDP, conformation.  Another region of the antibiotic binds to the same site as the aminoacyl group of tRNA.  Together with prior biochem. studies, the structural findings confirm that GE2270A inhibits protein synthesis by blocking the GDP to GTP conformational change and by directly competing with aminoacyl-tRNA for the same binding site on EF-Tu.  In each of the bacterial strains that are resistant to GE2270A, the effect of a site-specific mutation in EF-Tu could explain resistance.  Comparison of the GE2270A site in EF-Tu with sequence homologs, EF-G and EF-1α, suggests steric clashes that would prevent the antibiotic from binding to translocation factors or to the eukaryotic equiv. of EF-Tu.  Although GE2270A is a potent antibiotic, its clin. efficacy is limited by its low aq. soly.  The results presented here provide the details necessary to enhance the soly. of GE2270A without disrupting its inhibitory properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpde7H00g2l4rVg90H21EOLACvtfcHk0lhWAAIviks1pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnslens7w%253D&md5=20e6699a9167deedf876b2d4579306ad</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Fbi9913597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9913597%26sid%3Dliteratum%253Aachs%26aulast%3DHeffron%26aufirst%3DS.%2BE.%26aulast%3DJurnak%26aufirst%3DF.%26atitle%3DStructure%2520of%2520an%2520EF-Tu%2520complex%2520with%2520a%2520thiazolyl%2520peptide%2520antibiotic%2520determined%2520at%25202.35%2520%25C3%2585%2520resolution%253A%2520Atomic%2520basis%2520for%2520GE2270A%2520inhibition%2520of%2520EF-Tu%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D37%26epage%3D45%26doi%3D10.1021%2Fbi9913597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group">Wells, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClendon, C. L.</span><span> </span><span class="NLM_article-title">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">450</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-9&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+high-hanging+fruit+in+drug+discovery+at+protein-protein+interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0lgkG6tWKBCCiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520high-hanging%2520fruit%2520in%2520drug%2520discovery%2520at%2520protein-protein%2520interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D1001%26epage%3D9%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group">Whitty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumaravel, G.</span><span> </span><span class="NLM_article-title">Between a rock and a hard place?</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1038/nchembio0306-112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnchembio0306-112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16484997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVKqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=112-8&author=A.+Whittyauthor=G.+Kumaravel&title=Between+a+rock+and+a+hard+place%3F&doi=10.1038%2Fnchembio0306-112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Between a rock and a hard place?</span></div><div class="casAuthors">Whitty, Adrian; Kumaravel, Gnanasambandam</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">112-118</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Developing small-mol. inhibitors against protein-protein interaction targets is among the most difficult challenges in contemporary drug discovery.  Recent developments in our understanding of this problem, and in the knowledge and tools available to address it, give cause for renewed hope, but substantial challenges remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI04RMmncXC7Vg90H21EOLACvtfcHk0lgkG6tWKBCCiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVKqu78%253D&md5=c7a2f9aea89d6be284b1653893a9ece5</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0306-112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0306-112%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DKumaravel%26aufirst%3DG.%26atitle%3DBetween%2520a%2520rock%2520and%2520a%2520hard%2520place%253F%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D112%26epage%3D8%26doi%3D10.1038%2Fnchembio0306-112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group">Fuller, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgoyne, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Predicting druggable binding sites at the protein-protein interface</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">155</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1016/j.drudis.2008.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.drudis.2008.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=19041415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSrs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=155-61&author=J.+C.+Fullerauthor=N.+J.+Burgoyneauthor=R.+M.+Jackson&title=Predicting+druggable+binding+sites+at+the+protein-protein+interface&doi=10.1016%2Fj.drudis.2008.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting druggable binding sites at the protein-protein interface</span></div><div class="casAuthors">Fuller, Jonathan C.; Burgoyne, Nicholas J.; Jackson, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3/4</span>),
    <span class="NLM_cas:pages">155-161</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interfaces are highly attractive targets for drug discovery because they are involved in a large no. of disease pathways where therapeutic intervention would bring widespread benefit.  Recent successes have challenged the widely held belief that these targets are undruggable'.  The pocket finding algorithms described here show marked differences between the binding pockets that define protein-protein interactions (PPIs) and those that define protein-ligand interactions (PLIs) of currently marketed drugs.  In the case of PPIs, drug discovery methods that simultaneously target several small pockets at the protein-protein interface are likely to increase the chances of success in this new and important field of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruIDx_ryxRNrVg90H21EOLACvtfcHk0lgkG6tWKBCCiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSrs74%253D&md5=b0be246c99c98b82b67dab584fddcf3e</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2008.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2008.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DFuller%26aufirst%3DJ.%2BC.%26aulast%3DBurgoyne%26aufirst%3DN.%2BJ.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DPredicting%2520druggable%2520binding%2520sites%2520at%2520the%2520protein-protein%2520interface%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D155%26epage%3D61%26doi%3D10.1016%2Fj.drudis.2008.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pons, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcillat, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancelin, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krimm, I.</span><span> </span><span class="NLM_article-title">Fragment-based deconstruction of Bcl-x(L) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2577</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.1021/jm100009z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100009z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2577-88&author=S.+Barelierauthor=J.+Ponsauthor=O.+Marcillatauthor=J.+M.+Lancelinauthor=I.+Krimm&title=Fragment-based+deconstruction+of+Bcl-x%28L%29+inhibitors&doi=10.1021%2Fjm100009z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Fjm100009z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100009z%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DPons%26aufirst%3DJ.%26aulast%3DMarcillat%26aufirst%3DO.%26aulast%3DLancelin%26aufirst%3DJ.%2BM.%26aulast%3DKrimm%26aufirst%3DI.%26atitle%3DFragment-based%2520deconstruction%2520of%2520Bcl-x%2528L%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2577%26epage%3D88%26doi%3D10.1021%2Fjm100009z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group">Oltersdorf, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augeri, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belli, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruncko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deckwerth, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinges, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunzer, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Letai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitten, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettesheim, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimmer, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connor, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oleksijew, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomaselli, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wendt, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span> </span><span class="NLM_article-title">An inhibitor of Bcl-2 family proteins induces regression of solid tumours</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">435</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1038/nature03579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnature03579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15902208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1Ogtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2005&pages=677-81&author=T.+Oltersdorfauthor=S.+W.+Elmoreauthor=A.+R.+Shoemakerauthor=R.+C.+Armstrongauthor=D.+J.+Augeriauthor=B.+A.+Belliauthor=M.+Brunckoauthor=T.+L.+Deckwerthauthor=J.+Dingesauthor=P.+J.+Hajdukauthor=M.+K.+Josephauthor=S.+Kitadaauthor=S.+J.+Korsmeyerauthor=A.+R.+Kunzerauthor=A.+Letaiauthor=C.+Liauthor=M.+J.+Mittenauthor=D.+G.+Nettesheimauthor=S.+Ngauthor=P.+M.+Nimmerauthor=J.+M.+O%E2%80%99Connorauthor=A.+Oleksijewauthor=A.+M.+Petrosauthor=J.+C.+Reedauthor=W.+Shenauthor=S.+K.+Tahirauthor=C.+B.+Thompsonauthor=K.+J.+Tomaselliauthor=B.+Wangauthor=M.+D.+Wendtauthor=H.+Zhangauthor=S.+W.+Fesikauthor=S.+H.+Rosenberg&title=An+inhibitor+of+Bcl-2+family+proteins+induces+regression+of+solid+tumours&doi=10.1038%2Fnature03579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of Bcl-2 family proteins induces regression of solid tumours</span></div><div class="casAuthors">Oltersdorf, Tilman; Elmore, Steven W.; Shoemaker, Alexander R.; Armstrong, Robert C.; Augeri, David J.; Belli, Barbara A.; Bruncko, Milan; Deckwerth, Thomas L.; Dinges, Jurgen; Hajduk, Philip J.; Joseph, Mary K.; Kitada, Shinichi; Korsmeyer, Stanley J.; Kunzer, Aaron R.; Letai, Anthony; Li, Chi; Mitten, Michael J.; Nettesheim, David G.; Ng, Shi Chung; Nimmer, Paul M.; O'Connor, Jacqueline M.; Oleksijew, Anatol; Petros, Andrew M.; Reed, John C.; Shen, Wang; Tahir, Stephen K.; Thompson, Craig B.; Tomaselli, Kevin J.; Wang, Baole; Wendt, Michael D.; Zhang, Haichao; Fesik, Stephen W.; Rosenberg, Saul H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">7042</span>),
    <span class="NLM_cas:pages">677-681</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins in the Bcl-2 family are central regulators of programmed cell death, and members that inhibit apoptosis, such as Bcl-XL and Bcl-2, are overexpressed in many cancers and contribute to tumor initiation, progression and resistance to therapy.  Bcl-XL expression correlates with chemo-resistance of tumor cell lines, and redns. in Bcl-2 increase sensitivity to anticancer drugs and enhance in vivo survival.  The development of inhibitors of these proteins as potential anticancer therapeutics has been previously explored, but obtaining potent small-mol. inhibitors has proved difficult owing to the necessity of targeting a protein-protein interaction.  Here, using NMR-based screening, parallel synthesis and structure-based design, the authors have discovered ABT-737, a small-mol. inhibitor of the antiapoptotic proteins Bcl-2, Bcl-XL and Bcl-w, with an affinity two to three orders of magnitude more potent than previously reported compds.  Mechanistic studies reveal that ABT-737 does not directly initiate the apoptotic process, but enhances the effects of death signals, displaying synergistic cytotoxicity with chemotherapeutics and radiation.  ABT-737 exhibits single-agent-mechanism-based killing of cells from lymphoma and small-cell lung carcinoma lines, as well as primary patient-derived cells, and in animal models, ABT-737 improves survival, causes regression of established tumors, and produces cures in a high percentage of the mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBt7_zVt8HrbVg90H21EOLACvtfcHk0lhDgKeE7Tk6JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1Ogtb0%253D&md5=4940d06838506f181b6310d3458a595f</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1038%2Fnature03579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03579%26sid%3Dliteratum%253Aachs%26aulast%3DOltersdorf%26aufirst%3DT.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DArmstrong%26aufirst%3DR.%2BC.%26aulast%3DAugeri%26aufirst%3DD.%2BJ.%26aulast%3DBelli%26aufirst%3DB.%2BA.%26aulast%3DBruncko%26aufirst%3DM.%26aulast%3DDeckwerth%26aufirst%3DT.%2BL.%26aulast%3DDinges%26aufirst%3DJ.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DJoseph%26aufirst%3DM.%2BK.%26aulast%3DKitada%26aufirst%3DS.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26aulast%3DKunzer%26aufirst%3DA.%2BR.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMitten%26aufirst%3DM.%2BJ.%26aulast%3DNettesheim%26aufirst%3DD.%2BG.%26aulast%3DNg%26aufirst%3DS.%26aulast%3DNimmer%26aufirst%3DP.%2BM.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DJ.%2BM.%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DReed%26aufirst%3DJ.%2BC.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26aulast%3DTomaselli%26aufirst%3DK.%2BJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26atitle%3DAn%2520inhibitor%2520of%2520Bcl-2%2520family%2520proteins%2520induces%2520regression%2520of%2520solid%2520tumours%26jtitle%3DNature%26date%3D2005%26volume%3D435%26spage%3D677%26epage%3D81%26doi%3D10.1038%2Fnature03579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adickes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitten, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimmer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span> </span><span class="NLM_article-title">Abt-263: A potent and orally bioavailable Bcl-2 family inhibitor</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">3421</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-5836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1158%2F0008-5472.CAN-07-5836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=18451170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltlSltrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=3421-8&author=C.+Tseauthor=A.+R.+Shoemakerauthor=J.+Adickesauthor=M.+G.+Andersonauthor=J.+Chenauthor=S.+Jinauthor=E.+F.+Johnsonauthor=K.+C.+Marshauthor=M.+J.+Mittenauthor=P.+Nimmerauthor=L.+Robertsauthor=S.+K.+Tahirauthor=Y.+Xiaoauthor=X.+Yangauthor=H.+Zhangauthor=S.+Fesikauthor=S.+H.+Rosenbergauthor=S.+W.+Elmore&title=Abt-263%3A+A+potent+and+orally+bioavailable+Bcl-2+family+inhibitor&doi=10.1158%2F0008-5472.CAN-07-5836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">ABT-263: A Potent and Orally Bioavailable Bcl-2 Family Inhibitor</span></div><div class="casAuthors">Tse, Christin; Shoemaker, Alexander R.; Adickes, Jessica; Anderson, Mark G.; Chen, Jun; Jin, Sha; Johnson, Eric F.; Marsh, Kennan C.; Mitten, Michael J.; Nimmer, Paul; Roberts, Lisa; Tahir, Stephen K.; Xiao, Yu; Yang, Xiufen; Zhang, Haichao; Fesik, Stephen; Rosenberg, Saul H.; Elmore, Steven W.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3421-3428</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Overexpression of the prosurvival Bcl-2 family members (Bcl-2, Bcl-xL, and Mcl-1) is commonly assocd. with tumor maintenance, progression, and chemoresistance.  We previously reported the discovery of ABT-737, a potent, small-mol. Bcl-2 family protein inhibitor.  A major limitation of ABT-737 is that it is not orally bioavailable, which would limit chronic single agent therapy and flexibility to dose in combination regimens.  Here we report the biol. properties of ABT-263, a potent, orally bioavailable Bad-like BH3 mimetic (Ki's of <1 nmol/L for Bcl-2, Bcl-xL, and Bcl-w).  The oral bioavailability of ABT-263 in preclin. animal models is 20% to 50%, depending on formulation.  ABT-263 disrupts Bcl-2/Bcl-xL interactions with pro-death proteins (e.g., Bim), leading to the initiation of apoptosis within 2 h posttreatment.  In human tumor cells, ABT-263 induces Bax translocation, cytochrome c release, and subsequent apoptosis.  Oral administration of ABT-263 alone induces complete tumor regressions in xenograft models of small-cell lung cancer and acute lymphoblastic leukemia.  In xenograft models of aggressive B-cell lymphoma and multiple myeloma where ABT-263 exhibits modest or no single agent activity, it significantly enhances the efficacy of clin. relevant therapeutic regimens.  These data provide the rationale for clin. trials evaluating ABT-263 in small-cell lung cancer and B-cell malignancies.  The oral efficacy of ABT-263 should provide dosing flexibility to maximize clin. utility both as a single agent and in combination regimens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojLK8zIgyDWbVg90H21EOLACvtfcHk0lhDgKeE7Tk6JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXltlSltrs%253D&md5=a43029cc218d25d4e1fb6dd1186475c4</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-5836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-5836%26sid%3Dliteratum%253Aachs%26aulast%3DTse%26aufirst%3DC.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DAdickes%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMitten%26aufirst%3DM.%2BJ.%26aulast%3DNimmer%26aufirst%3DP.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFesik%26aufirst%3DS.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3DAbt-263%253A%2520A%2520potent%2520and%2520orally%2520bioavailable%2520Bcl-2%2520family%2520inhibitor%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D3421%26epage%3D8%26doi%3D10.1158%2F0008-5472.CAN-07-5836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group">Tao, Z. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasvold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boghaert, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catron, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colman, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czabotar, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshayes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairbrother, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flygare, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hymowitz, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judge, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koehler, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovar, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lessene, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitten, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ndubaku, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimmer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purkey, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oleksijew, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sleebs, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zobel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leverson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souers, A. J.</span><span> </span><span class="NLM_article-title">Discovery of a potent and selective bcl-xl inhibitor with in vivo activity</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1088</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.1021/ml5001867</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5001867" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWrtbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1088-93&author=Z.+F.+Taoauthor=L.+Hasvoldauthor=L.+Wangauthor=X.+Wangauthor=A.+M.+Petrosauthor=C.+H.+Parkauthor=E.+R.+Boghaertauthor=N.+D.+Catronauthor=J.+Chenauthor=P.+M.+Colmanauthor=P.+E.+Czabotarauthor=K.+Deshayesauthor=W.+J.+Fairbrotherauthor=J.+A.+Flygareauthor=S.+G.+Hymowitzauthor=S.+Jinauthor=R.+A.+Judgeauthor=M.+F.+Koehlerauthor=P.+J.+Kovarauthor=G.+Lesseneauthor=M.+J.+Mittenauthor=C.+O.+Ndubakuauthor=P.+Nimmerauthor=H.+E.+Purkeyauthor=A.+Oleksijewauthor=D.+C.+Phillipsauthor=B.+E.+Sleebsauthor=B.+J.+Smithauthor=M.+L.+Smithauthor=S.+K.+Tahirauthor=K.+G.+Watsonauthor=Y.+Xiaoauthor=J.+Xueauthor=H.+Zhangauthor=K.+Zobelauthor=S.+H.+Rosenbergauthor=C.+Tseauthor=J.+D.+Leversonauthor=S.+W.+Elmoreauthor=A.+J.+Souers&title=Discovery+of+a+potent+and+selective+bcl-xl+inhibitor+with+in+vivo+activity&doi=10.1021%2Fml5001867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective BCL-XL Inhibitor with in Vivo Activity</span></div><div class="casAuthors">Tao, Zhi-Fu; Hasvold, Lisa; Wang, Le; Wang, Xilu; Petros, Andrew M.; Park, Chang H.; Boghaert, Erwin R.; Catron, Nathaniel D.; Chen, Jun; Colman, Peter M.; Czabotar, Peter E.; Deshayes, Kurt; Fairbrother, Wayne J.; Flygare, John A.; Hymowitz, Sarah G.; Jin, Sha; Judge, Russell A.; Koehler, Michael F. T.; Kovar, Peter J.; Lessene, Guillaume; Mitten, Michael J.; Ndubaku, Chudi O.; Nimmer, Paul; Purkey, Hans E.; Oleksijew, Anatol; Phillips, Darren C.; Sleebs, Brad E.; Smith, Brian J.; Smith, Morey L.; Tahir, Stephen K.; Watson, Keith G.; Xiao, Yu; Xue, John; Zhang, Haichao; Zobel, Kerry; Rosenberg, Saul H.; Tse, Chris; Leverson, Joel D.; Elmore, Steven W.; Souers, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1088-1093</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A-1155463 I, a highly potent and selective BCL-XL inhibitor, was discovered through NMR fragment screening and structure-based design.  This compd. is substantially more potent against BCL-XL-dependent cell lines relative to the recently reported inhibitor, WEHI-539, while possessing none of its inherent pharmaceutical liabilities.  A-1155463 caused a mechanism-based and reversible thrombocytopenia in mice and inhibited H146 small cell lung cancer xenograft tumor growth in vivo following multiple doses.  A-1155463 thus represents an excellent tool mol. for studying BCL-XL biol. as well as a productive lead structure for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozGRbIa-4nxbVg90H21EOLACvtfcHk0lggfDjFy0qV2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWrtbnN&md5=5d2fffa59286214efac3f320a701215a</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Fml5001867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5001867%26sid%3Dliteratum%253Aachs%26aulast%3DTao%26aufirst%3DZ.%2BF.%26aulast%3DHasvold%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DBoghaert%26aufirst%3DE.%2BR.%26aulast%3DCatron%26aufirst%3DN.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DColman%26aufirst%3DP.%2BM.%26aulast%3DCzabotar%26aufirst%3DP.%2BE.%26aulast%3DDeshayes%26aufirst%3DK.%26aulast%3DFairbrother%26aufirst%3DW.%2BJ.%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26aulast%3DHymowitz%26aufirst%3DS.%2BG.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DJudge%26aufirst%3DR.%2BA.%26aulast%3DKoehler%26aufirst%3DM.%2BF.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DLessene%26aufirst%3DG.%26aulast%3DMitten%26aufirst%3DM.%2BJ.%26aulast%3DNdubaku%26aufirst%3DC.%2BO.%26aulast%3DNimmer%26aufirst%3DP.%26aulast%3DPurkey%26aufirst%3DH.%2BE.%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DPhillips%26aufirst%3DD.%2BC.%26aulast%3DSleebs%26aufirst%3DB.%2BE.%26aulast%3DSmith%26aufirst%3DB.%2BJ.%26aulast%3DSmith%26aufirst%3DM.%2BL.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DWatson%26aufirst%3DK.%2BG.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZobel%26aufirst%3DK.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DLeverson%26aufirst%3DJ.%2BD.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DSouers%26aufirst%3DA.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520bcl-xl%2520inhibitor%2520with%2520in%2520vivo%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1088%26epage%3D93%26doi%3D10.1021%2Fml5001867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group">Flygare, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budha, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheeti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshayes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doerner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckhardt, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franklin, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reisner, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gazzard, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halladay, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hymowitz, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LoRusso, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plise, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephan, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tom, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Um, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varfolomeev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vucic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallweber, H. J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ware, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, Z. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zobel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairbrother, W. J.</span><span> </span><span class="NLM_article-title">Discovery of a potent small-molecule antagonist of inhibitor of apoptosis (IAP) proteins and clinical candidate for the treatment of cancer (GDC-0152)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4101</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1021/jm300060k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300060k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Ois74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4101-13&author=J.+A.+Flygareauthor=M.+Beresiniauthor=N.+Budhaauthor=H.+Chanauthor=I.+T.+Chanauthor=S.+Cheetiauthor=F.+Cohenauthor=K.+Deshayesauthor=K.+Doernerauthor=S.+G.+Eckhardtauthor=L.+O.+Elliottauthor=B.+N.+Fengauthor=M.+C.+Franklinauthor=S.+F.+Reisnerauthor=L.+Gazzardauthor=J.+Halladayauthor=S.+G.+Hymowitzauthor=H.+Laauthor=P.+LoRussoauthor=B.+Maurerauthor=L.+Murrayauthor=E.+Pliseauthor=C.+Quanauthor=J.+P.+Stephanauthor=S.+G.+Youngauthor=J.+Tomauthor=V.+Tsuiauthor=J.+Umauthor=E.+Varfolomeevauthor=D.+Vucicauthor=A.+J.+Wagnerauthor=H.+J.+A.+Wallweberauthor=L.+Wangauthor=J.+Wareauthor=Z.+Y.+Wenauthor=H.+Wongauthor=J.+M.+Wongauthor=M.+Wongauthor=S.+Wongauthor=R.+Yuauthor=K.+Zobelauthor=W.+J.+Fairbrother&title=Discovery+of+a+potent+small-molecule+antagonist+of+inhibitor+of+apoptosis+%28IAP%29+proteins+and+clinical+candidate+for+the+treatment+of+cancer+%28GDC-0152%29&doi=10.1021%2Fjm300060k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a potent small-molecule antagonist of inhibitor of apoptosis (IAP) proteins and clinical candidate for the treatment of cancer (GDC-0152)</span></div><div class="casAuthors">Flygare, John A.; Beresini, Maureen; Budha, Nageshwar; Chan, Helen; Chan, Iris T.; Cheeti, Sravanthi; Cohen, Frederick; Deshayes, Kurt; Doerner, Karl; Eckhardt, S. Gail; Elliott, Linda O.; Feng, Bainian; Franklin, Matthew C.; Reisner, Stacy Frankovitz; Gazzard, Lewis; Halladay, Jason; Hymowitz, Sarah G.; La, Hank; LoRusso, Patricia; Maurer, Brigitte; Murray, Lesley; Plise, Emile; Quan, Clifford; Stephan, Jean-Philippe; Young, Shin G.; Tom, Jeffrey; Tsui, Vickie; Um, Joanne; Varfolomeev, Eugene; Vucic, Domagoj; Wagner, Andrew J.; Wallweber, Heidi J. A.; Wang, Lan; Ware, Joseph; Wen, Zhaoyang; Wong, Harvey; Wong, Jonathan M.; Wong, Melisa; Wong, Susan; Yu, Ron; Zobel, Kerry; Fairbrother, Wayne J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4101-4113</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of compds. were designed and synthesized as antagonists of cIAP1/2, ML-IAP, and XIAP based on the N-terminus, AVPI, of mature Smac.  Compd. (I) (GDC-0152) has the best profile of these compds.; it binds to the XIAP BIR3 domain, the BIR domain of ML-IAP, and the BIR3 domains of cIAP1 and cIAP2 with Ki values of 28, 14, 17, and 43 nM, resp.  These compds. promote degrdn. of cIAP1, induce activation of caspase-3/7, and lead to decreased viability of breast cancer cells without affecting normal mammary epithelial cells.  Compd. I inhibits tumor growth when dosed orally in the MDA-MB-231 breast cancer xenograft model.  Compd. I was advanced to human clin. trials, and it exhibited linear pharmacokinetics over the dose range (0.049 to 1.48 mg/kg) tested.  Mean plasma clearance in humans was 9 ± 3 mL/min/kg, and the vol. of distribution was 0.6 ± 0.2 L/kg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZSPNVJ5gVWrVg90H21EOLACvtfcHk0lggfDjFy0qV2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Ois74%253D&md5=19452878ea834d3c324d567b39a78fb2</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1021%2Fjm300060k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300060k%26sid%3Dliteratum%253Aachs%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26aulast%3DBeresini%26aufirst%3DM.%26aulast%3DBudha%26aufirst%3DN.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DChan%26aufirst%3DI.%2BT.%26aulast%3DCheeti%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DDeshayes%26aufirst%3DK.%26aulast%3DDoerner%26aufirst%3DK.%26aulast%3DEckhardt%26aufirst%3DS.%2BG.%26aulast%3DElliott%26aufirst%3DL.%2BO.%26aulast%3DFeng%26aufirst%3DB.%2BN.%26aulast%3DFranklin%26aufirst%3DM.%2BC.%26aulast%3DReisner%26aufirst%3DS.%2BF.%26aulast%3DGazzard%26aufirst%3DL.%26aulast%3DHalladay%26aufirst%3DJ.%26aulast%3DHymowitz%26aufirst%3DS.%2BG.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DLoRusso%26aufirst%3DP.%26aulast%3DMaurer%26aufirst%3DB.%26aulast%3DMurray%26aufirst%3DL.%26aulast%3DPlise%26aufirst%3DE.%26aulast%3DQuan%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DJ.%2BP.%26aulast%3DYoung%26aufirst%3DS.%2BG.%26aulast%3DTom%26aufirst%3DJ.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DUm%26aufirst%3DJ.%26aulast%3DVarfolomeev%26aufirst%3DE.%26aulast%3DVucic%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DA.%2BJ.%26aulast%3DWallweber%26aufirst%3DH.%2BJ.%2BA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWare%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DZ.%2BY.%26aulast%3DWong%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DJ.%2BM.%26aulast%3DWong%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DZobel%26aufirst%3DK.%26aulast%3DFairbrother%26aufirst%3DW.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520small-molecule%2520antagonist%2520of%2520inhibitor%2520of%2520apoptosis%2520%2528IAP%2529%2520proteins%2520and%2520clinical%2520candidate%2520for%2520the%2520treatment%2520of%2520cancer%2520%2528GDC-0152%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4101%26epage%3D13%26doi%3D10.1021%2Fjm300060k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group">Seigal, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connors, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borzilleri, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuel, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fargnoli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naglich, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pokross, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Posy, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talbott, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terrett, N. K.</span><span> </span><span class="NLM_article-title">The discovery of macrocyclic XIAP antagonists from a DNA-programmed chemistry library, and their optimization to give lead compounds with in vivo antitumor activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2855</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1021/jm501892g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501892g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivF2qu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2855-61&author=B.+A.+Seigalauthor=W.+H.+Connorsauthor=A.+Fraleyauthor=R.+M.+Borzilleriauthor=P.+H.+Carterauthor=S.+L.+Emanuelauthor=J.+Fargnoliauthor=K.+Kimauthor=M.+Leiauthor=J.+G.+Naglichauthor=M.+E.+Pokrossauthor=S.+L.+Posyauthor=H.+Shenauthor=N.+Surtiauthor=R.+Talbottauthor=Y.+Zhangauthor=N.+K.+Terrett&title=The+discovery+of+macrocyclic+XIAP+antagonists+from+a+DNA-programmed+chemistry+library%2C+and+their+optimization+to+give+lead+compounds+with+in+vivo+antitumor+activity&doi=10.1021%2Fjm501892g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of Macrocyclic XIAP Antagonists from a DNA-Programmed Chemistry Library, and Their Optimization To Give Lead Compounds with in Vivo Antitumor Activity</span></div><div class="casAuthors">Seigal, Benjamin A.; Connors, William H.; Fraley, Andrew; Borzilleri, Robert M.; Carter, Percy H.; Emanuel, Stuart L.; Fargnoli, Joseph; Kim, Kyoung; Lei, Ming; Naglich, Joseph G.; Pokross, Matthew E.; Posy, Shana L.; Shen, Henry; Surti, Neha; Talbott, Randy; Zhang, Yong; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2855-2861</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Affinity selection screening of macrocycle libraries derived from DNA-programmed chem. identified XIAP BIR2 and BIR3 domain inhibitors that displace bound pro-apoptotic caspases.  X-ray cocrystal structures of key compds. with XIAP BIR2 suggested potency-enhancing structural modifications.  Optimization of dimeric macrocycles with similar affinity for both domains were potent pro-apoptotic agents in cancer cell lines and efficacious in shrinking tumors in a mouse xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlB6i1KOFJjrVg90H21EOLACvtfcHk0lgdzmKQUb5Qmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivF2qu78%253D&md5=9bd53cc3ed014796248e8048cecea0d1</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1021%2Fjm501892g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501892g%26sid%3Dliteratum%253Aachs%26aulast%3DSeigal%26aufirst%3DB.%2BA.%26aulast%3DConnors%26aufirst%3DW.%2BH.%26aulast%3DFraley%26aufirst%3DA.%26aulast%3DBorzilleri%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DEmanuel%26aufirst%3DS.%2BL.%26aulast%3DFargnoli%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DK.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DNaglich%26aufirst%3DJ.%2BG.%26aulast%3DPokross%26aufirst%3DM.%2BE.%26aulast%3DPosy%26aufirst%3DS.%2BL.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DSurti%26aufirst%3DN.%26aulast%3DTalbott%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520discovery%2520of%2520macrocyclic%2520XIAP%2520antagonists%2520from%2520a%2520DNA-programmed%2520chemistry%2520library%252C%2520and%2520their%2520optimization%2520to%2520give%2520lead%2520compounds%2520with%2520in%2520vivo%2520antitumor%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2855%26epage%3D61%26doi%3D10.1021%2Fjm501892g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group">Vassilev, L. T.</span><span> </span><span class="NLM_article-title">MDM2 inhibitors for cancer therapy</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1016/j.molmed.2006.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.molmed.2006.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=17126603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1CrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=23-31&author=L.+T.+Vassilev&title=MDM2+inhibitors+for+cancer+therapy&doi=10.1016%2Fj.molmed.2006.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">MDM2 inhibitors for cancer therapy</span></div><div class="casAuthors">Vassilev, Lyubomir T.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-31</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The tumor suppressor p53 is a powerful antitumoral mol. frequently inactivated by mutations or deletions in cancer.  However, half of all human tumors express wild-type p53, and its activation by antagonizing its neg. regulator murine double minute 2 (MDM2) might offer a new therapeutic strategy.  Proof-of-concept expts. have demonstrated the feasibility of this approach in vitro but the development of pharmacol. inhibitors has been challenging.  Recently, potent and selective small-mol. MDM2 inhibitors have been identified.  Studies with these compds. have strengthened the concept that selective, non-genotoxic p53 activation is a viable alternative to current cytotoxic chemotherapy but clin. validation is still pending.  Here, the new developments in the quest for pharmacol. p53 activators are reviewed with an emphasis on small-mol. inhibitors of the p53-MDM2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYnDenIZoV0rVg90H21EOLACvtfcHk0lgdzmKQUb5Qmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1CrsQ%253D%253D&md5=a0bb8aaed6ec192280538a6ee1f908fa</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2006.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2006.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DVassilev%26aufirst%3DL.%2BT.%26atitle%3DMDM2%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2007%26volume%3D13%26spage%3D23%26epage%3D31%26doi%3D10.1016%2Fj.molmed.2006.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of the MDM2-p53 protein-protein interaction (MDM2 inhibitors) in clinical trials for cancer treatment</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1038</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1021/jm501092z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501092z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFagt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1038-52&author=Y.+Zhaoauthor=A.+Aguilarauthor=D.+Bernardauthor=S.+Wang&title=Small-molecule+inhibitors+of+the+MDM2-p53+protein-protein+interaction+%28MDM2+inhibitors%29+in+clinical+trials+for+cancer+treatment&doi=10.1021%2Fjm501092z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of the MDM2-p53 Protein-Protein Interaction (MDM2 Inhibitors) in Clinical Trials for Cancer Treatment</span></div><div class="casAuthors">Zhao, Yujun; Aguilar, Angelo; Bernard, Denzil; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1038-1052</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Design of small-mol. inhibitors (MDM2 inhibitors) to block the MDM2-p53 protein-protein interaction has been pursued as a new cancer therapeutic strategy.  In recent years, potent, selective, and efficacious MDM2 inhibitors have been successfully obtained and seven such compds. have been advanced into early phase clin. trials for the treatment of human cancers.  Here, we review the design, synthesis, properties, preclin., and clin. studies of these clin.-stage MDM2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNa9aW1Kz1Q7Vg90H21EOLACvtfcHk0lgdzmKQUb5Qmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFagt77P&md5=2e58318782d7e0a04bf3398db8782984</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Fjm501092z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501092z%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520the%2520MDM2-p53%2520protein-protein%2520interaction%2520%2528MDM2%2520inhibitors%2529%2520in%2520clinical%2520trials%2520for%2520cancer%2520treatment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1038%26epage%3D52%26doi%3D10.1021%2Fjm501092z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group">Sun, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rew, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartberger, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deignan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquette, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eksterowicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, A. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Lopez De Turiso, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houze, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kayser, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGee, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McIntosh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mihalic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliner, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osgood, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roveto, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saiki, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaffer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toteva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wortman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yakowec, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, J. C.</span><span> </span><span class="NLM_article-title">Discovery of AMG 232, a potent, selective, and orally bioavailable MDM2-p53 inhibitor in clinical development</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1454</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1021/jm401753e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401753e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1eitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1454-72&author=D.+Sunauthor=Z.+Liauthor=Y.+Rewauthor=M.+Gribbleauthor=M.+D.+Bartbergerauthor=H.+P.+Beckauthor=J.+Canonauthor=A.+Chenauthor=X.+Chenauthor=D.+Chowauthor=J.+Deignanauthor=J.+Duquetteauthor=J.+Eksterowiczauthor=B.+Fisherauthor=B.+M.+Foxauthor=J.+Fuauthor=A.+Z.+Gonzalezauthor=F.+Gonzalez-Lopez+De+Turisoauthor=J.+B.+Houzeauthor=X.+Huangauthor=M.+Jiangauthor=L.+Jinauthor=F.+Kayserauthor=J.+J.+Liuauthor=M.+C.+Loauthor=A.+M.+Longauthor=B.+Lucasauthor=L.+R.+McGeeauthor=J.+McIntoshauthor=J.+Mihalicauthor=J.+D.+Olinerauthor=T.+Osgoodauthor=M.+L.+Petersonauthor=P.+Rovetoauthor=A.+Y.+Saikiauthor=P.+Shafferauthor=M.+Totevaauthor=Y.+Wangauthor=Y.+C.+Wangauthor=S.+Wortmanauthor=P.+Yakowecauthor=X.+Yanauthor=Q.+Yeauthor=D.+Yuauthor=M.+Yuauthor=X.+Zhaoauthor=J.+Zhouauthor=J.+Zhuauthor=S.+H.+Olsonauthor=J.+C.+Medina&title=Discovery+of+AMG+232%2C+a+potent%2C+selective%2C+and+orally+bioavailable+MDM2-p53+inhibitor+in+clinical+development&doi=10.1021%2Fjm401753e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of AMG 232, a Potent, Selective, and Orally Bioavailable MDM2-p53 Inhibitor in Clinical Development</span></div><div class="casAuthors">Sun, Daqing; Li, Zhihong; Rew, Yosup; Gribble, Michael; Bartberger, Michael D.; Beck, Hilary P.; Canon, Jude; Chen, Ada; Chen, Xiaoqi; Chow, David; Deignan, Jeffrey; Duquette, Jason; Eksterowicz, John; Fisher, Benjamin; Fox, Brian M.; Fu, Jiasheng; Gonzalez, Ana Z.; Gonzalez-Lopez De Turiso, Felix; Houze, Jonathan B.; Huang, Xin; Jiang, Min; Jin, Lixia; Kayser, Frank; Liu, Jiwen; Lo, Mei-Chu; Long, Alexander M.; Lucas, Brian; McGee, Lawrence R.; McIntosh, Joel; Mihalic, Jeff; Oliner, Jonathan D.; Osgood, Tao; Peterson, Matthew L.; Roveto, Philip; Saiki, Anne Y.; Shaffer, Paul; Toteva, Maria; Wang, Yingcai; Wang, Yu Chung; Wortman, Sarah; Yakowec, Peter; Yan, Xuelei; Ye, Qiuping; Yu, Dongyin; Yu, Ming; Zhao, Xiaoning; Zhou, Jing; Zhu, Jiang; Olson, Steven H.; Medina, Julio C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1454-1472</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We recently reported the discovery of AM-8553 (1), a potent and selective piperidinone inhibitor of the MDM2-p53 interaction.  Continued research investigation of the N-alkyl substituent of this series, focused in particular on a previously underutilized interaction in a shallow cleft on the MDM2 surface, led to the discovery of a one-carbon tethered sulfone which gave rise to substantial improvements in biochem. and cellular potency.  Further investigation produced AMG 232 (2), which is currently being evaluated in human clin. trials for the treatment of cancer.  Compd. 2 is an extremely potent MDM2 inhibitor (SPR KD = 0.045 nM, SJSA-1 EdU IC50 = 9.1 nM), with remarkable pharmacokinetic properties and in vivo antitumor activity in the SJSA-1 osteosarcoma xenograft model (ED50 = 9.1 mg/kg).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjly0O6c42NbVg90H21EOLACvtfcHk0lh5iybliZGmoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1eitbY%253D&md5=91197d328d40f6fa2f6c37819149427f</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1021%2Fjm401753e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401753e%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DRew%26aufirst%3DY.%26aulast%3DGribble%26aufirst%3DM.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DBeck%26aufirst%3DH.%2BP.%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChow%26aufirst%3DD.%26aulast%3DDeignan%26aufirst%3DJ.%26aulast%3DDuquette%26aufirst%3DJ.%26aulast%3DEksterowicz%26aufirst%3DJ.%26aulast%3DFisher%26aufirst%3DB.%26aulast%3DFox%26aufirst%3DB.%2BM.%26aulast%3DFu%26aufirst%3DJ.%26aulast%3DGonzalez%26aufirst%3DA.%2BZ.%26aulast%3DGonzalez-Lopez%2BDe%2BTuriso%26aufirst%3DF.%26aulast%3DHouze%26aufirst%3DJ.%2BB.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKayser%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DJ.%2BJ.%26aulast%3DLo%26aufirst%3DM.%2BC.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DLucas%26aufirst%3DB.%26aulast%3DMcGee%26aufirst%3DL.%2BR.%26aulast%3DMcIntosh%26aufirst%3DJ.%26aulast%3DMihalic%26aufirst%3DJ.%26aulast%3DOliner%26aufirst%3DJ.%2BD.%26aulast%3DOsgood%26aufirst%3DT.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26aulast%3DRoveto%26aufirst%3DP.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DShaffer%26aufirst%3DP.%26aulast%3DToteva%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DWortman%26aufirst%3DS.%26aulast%3DYakowec%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DOlson%26aufirst%3DS.%2BH.%26aulast%3DMedina%26aufirst%3DJ.%2BC.%26atitle%3DDiscovery%2520of%2520AMG%2520232%252C%2520a%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520MDM2-p53%2520inhibitor%2520in%2520clinical%2520development%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1454%26epage%3D72%26doi%3D10.1021%2Fjm401753e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group">Yu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartberger, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eksterowicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMinn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliner, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osgood, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rew, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saiki, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaffer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, D.</span><span> </span><span class="NLM_article-title">Discovery of potent and simplified piperidinone-based inhibitors of the MDM2-p53 interaction</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">894</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1021/ml500142b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500142b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=894-9&author=M.+Yuauthor=Y.+Wangauthor=J.+Zhuauthor=M.+D.+Bartbergerauthor=J.+Canonauthor=A.+Chenauthor=D.+Chowauthor=J.+Eksterowiczauthor=B.+Foxauthor=J.+Fuauthor=M.+Gribbleauthor=X.+Huangauthor=Z.+Liauthor=J.+J.+Liuauthor=M.+C.+Loauthor=D.+McMinnauthor=J.+D.+Olinerauthor=T.+Osgoodauthor=Y.+Rewauthor=A.+Y.+Saikiauthor=P.+Shafferauthor=X.+Yanauthor=Q.+Yeauthor=D.+Yuauthor=X.+Zhaoauthor=J.+Zhouauthor=S.+H.+Olsonauthor=J.+C.+Medinaauthor=D.+Sun&title=Discovery+of+potent+and+simplified+piperidinone-based+inhibitors+of+the+MDM2-p53+interaction&doi=10.1021%2Fml500142b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1021%2Fml500142b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500142b%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DChow%26aufirst%3DD.%26aulast%3DEksterowicz%26aufirst%3DJ.%26aulast%3DFox%26aufirst%3DB.%26aulast%3DFu%26aufirst%3DJ.%26aulast%3DGribble%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DJ.%2BJ.%26aulast%3DLo%26aufirst%3DM.%2BC.%26aulast%3DMcMinn%26aufirst%3DD.%26aulast%3DOliner%26aufirst%3DJ.%2BD.%26aulast%3DOsgood%26aufirst%3DT.%26aulast%3DRew%26aufirst%3DY.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DShaffer%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DOlson%26aufirst%3DS.%2BH.%26aulast%3DMedina%26aufirst%3DJ.%2BC.%26aulast%3DSun%26aufirst%3DD.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520simplified%2520piperidinone-based%2520inhibitors%2520of%2520the%2520MDM2-p53%2520interaction%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D894%26epage%3D9%26doi%3D10.1021%2Fml500142b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group">Wendt, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rockway, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geyer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClellan, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weitzberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mantei, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nienaber, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinghofer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giranda, V. L.</span><span> </span><span class="NLM_article-title">Identification of novel binding interactions in the development of potent, selective 2-naphthamidine inhibitors of urokinase. Synthesis, structural analysis, and SAR of N-phenyl amide 6-substitution</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">303</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span><span class="refDoi"> DOI: 10.1021/jm0300072</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0300072" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=303-24&author=M.+D.+Wendtauthor=T.+W.+Rockwayauthor=A.+Geyerauthor=W.+McClellanauthor=M.+Weitzbergauthor=X.+Zhaoauthor=R.+Manteiauthor=V.+L.+Nienaberauthor=K.+Stewartauthor=V.+Klinghoferauthor=V.+L.+Giranda&title=Identification+of+novel+binding+interactions+in+the+development+of+potent%2C+selective+2-naphthamidine+inhibitors+of+urokinase.+Synthesis%2C+structural+analysis%2C+and+SAR+of+N-phenyl+amide+6-substitution&doi=10.1021%2Fjm0300072"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1021%2Fjm0300072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0300072%26sid%3Dliteratum%253Aachs%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DRockway%26aufirst%3DT.%2BW.%26aulast%3DGeyer%26aufirst%3DA.%26aulast%3DMcClellan%26aufirst%3DW.%26aulast%3DWeitzberg%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DMantei%26aufirst%3DR.%26aulast%3DNienaber%26aufirst%3DV.%2BL.%26aulast%3DStewart%26aufirst%3DK.%26aulast%3DKlinghofer%26aufirst%3DV.%26aulast%3DGiranda%26aufirst%3DV.%2BL.%26atitle%3DIdentification%2520of%2520novel%2520binding%2520interactions%2520in%2520the%2520development%2520of%2520potent%252C%2520selective%25202-naphthamidine%2520inhibitors%2520of%2520urokinase.%2520Synthesis%252C%2520structural%2520analysis%252C%2520and%2520SAR%2520of%2520N-phenyl%2520amide%25206-substitution%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D303%26epage%3D24%26doi%3D10.1021%2Fjm0300072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group">D’Abramo, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archambault, J.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of human papillomavirus protein - protein interactions</span> <span class="citation_source-journal">Open Virol. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.2174/1874357901105010080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.2174%2F1874357901105010080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21769307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFWiu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=80-95&author=C.+M.+D%E2%80%99Abramoauthor=J.+Archambault&title=Small+molecule+inhibitors+of+human+papillomavirus+protein+-+protein+interactions&doi=10.2174%2F1874357901105010080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of human papillomavirus protein - protein interactions</span></div><div class="casAuthors">D'Abramo, C. M.; Archambault, J.</div><div class="citationInfo"><span class="NLM_cas:title">Open Virology Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">80-95</span>CODEN:
                <span class="NLM_cas:coden">OVJPAJ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3579</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Human papillomaviruses (HPV) have now been identified as a necessary cause of benign and malignant lesions of the differentiating epithelium, particularly cervical cancer, the second most prevalent cancer in women worldwide.  While two prophylactic HPV vaccines and screening programs are available, there is currently no antiviral drug for the treatment of HPV infections and assocd. diseases.  The recent progress toward the identification and characterization of specific mol. targets for small mol.-based approaches provides prospect for the development of effective HPV antiviral compds.  Traditionally, antiviral therapies target viral enzymes.  HPV encode for few proteins, however, and rely extensively on the infected cell for completion of their life cycle.  This article will review the functions of the viral E1 helicase, which encodes the only enzymic function of the virus, of the E2 regulatory protein, and of the viral E6 and E7 oncogenes in viral replication and pathogenesis.  Particular emphasis will be placed on the recent progress made towards the development of novel small mol. inhibitors that specifically target and inhibit the functions of these viral proteins, as well as their interactions with other viral and/or cellular proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrubY6vJhjR2bVg90H21EOLACvtfcHk0lgSZKLkxnAJ-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFWiu73O&md5=1013920c1efdefb50d1672e4ff145a12</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.2174%2F1874357901105010080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1874357901105010080%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Abramo%26aufirst%3DC.%2BM.%26aulast%3DArchambault%26aufirst%3DJ.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520human%2520papillomavirus%2520protein%2520-%2520protein%2520interactions%26jtitle%3DOpen%2520Virol.%2520J.%26date%3D2011%26volume%3D5%26spage%3D80%26epage%3D95%26doi%3D10.2174%2F1874357901105010080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group">Sun, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olejniczak, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waterson, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossanese, O. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovery of small molecules that bind to K-Ras and inhibit SOS-mediated activation</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6140</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span><span class="refDoi"> DOI: 10.1002/anie.201201358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1002%2Fanie.201201358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38Xms1entr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=6140-3&author=Q.+Sunauthor=J.+P.+Burkeauthor=J.+Phanauthor=M.+C.+Burnsauthor=E.+T.+Olejniczakauthor=A.+G.+Watersonauthor=T.+Leeauthor=O.+W.+Rossaneseauthor=S.+W.+Fesik&title=Discovery+of+small+molecules+that+bind+to+K-Ras+and+inhibit+SOS-mediated+activation&doi=10.1002%2Fanie.201201358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecules that Bind to K-Ras and Inhibit Sos-Mediated Activation</span></div><div class="casAuthors">Sun, Qi; Burke, Jason P.; Phan, Jason; Burns, Michael C.; Olejniczak, Edward T.; Waterson, Alex G.; Lee, Taekyu; Rossanese, Olivia W.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">6140-6143, S6140/1-S6140/17</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">K-Ras inhibition represents an attractive therapeutic strategy for many cancers.  We report on the discovery of novel small mols. that bind directly to K-Ras between switch I and switch II and inhibit Sos-catalyzed K-Ras activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy5Cx7BSO1pLVg90H21EOLACvtfcHk0lgSZKLkxnAJ-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xms1entr4%253D&md5=7d579523462aaf7eac1890643f91fab3</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1002%2Fanie.201201358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201201358%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DBurke%26aufirst%3DJ.%2BP.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DBurns%26aufirst%3DM.%2BC.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520small%2520molecules%2520that%2520bind%2520to%2520K-Ras%2520and%2520inhibit%2520SOS-mediated%2520activation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D6140%26epage%3D3%26doi%3D10.1002%2Fanie.201201358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group">Pikul, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunham, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almstead, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natchus, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taiwo, Y. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hynd, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janusz, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mieling, G. E.</span><span> </span><span class="NLM_article-title">Heterocycle-based MMP inhibitors with P2′ substituents</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1009</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/S0960-894X(01)00137-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2FS0960-894X%2801%2900137-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=11327577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisl2jsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1009-13&author=S.+Pikulauthor=K.+M.+Dunhamauthor=N.+G.+Almsteadauthor=B.+Deauthor=M.+G.+Natchusauthor=Y.+O.+Taiwoauthor=L.+E.+Williamsauthor=B.+A.+Hyndauthor=L.+C.+Hsiehauthor=M.+J.+Januszauthor=F.+Guauthor=G.+E.+Mieling&title=Heterocycle-based+MMP+inhibitors+with+P2%E2%80%B2+substituents&doi=10.1016%2FS0960-894X%2801%2900137-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Heterocycle-based MMP Inhibitors with P2' Substituents</span></div><div class="casAuthors">Pikul, S.; Dunham, K. M.; Almstead, N. G.; De, B.; Natchus, M. G.; Taiwo, Y. O.; Williams, L. E.; Hynd, B. A.; Hsieh, L. C.; Janusz, M. J.; Gu, F.; Mieling, G. E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1009-1013</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Potent and selective inhibition of matrix metalloproteinases was demonstrated for a series of sulfonamide-based hydroxamic acids.  The design of the heterocyclic sulfonamides incorporates a six- or seven-member central ring with a P2' substituent that can be modified.  Binding interactions of this substituent at the S2' site are believed to contribute to high inhibitory potency against stromelysin, collagenase-3 and gelatinases A and B, and to provide selectivity against collagenase-1 and matrilysin.  An x-ray structure of a stromelysin-inhibitor complex was obtained to provide insights into the SAR and selectivity trends obsd. for the series.  A novel series of potent inhibitors of matrix metalloproteinases is described.  Highly selective, MMP-1 sparing mols. are identified through modification of the P2' substituent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg2ZIuuzKp3bVg90H21EOLACvtfcHk0lgSZKLkxnAJ-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisl2jsb0%253D&md5=859a7b51b335c30355e5018d6ce16cd2</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2801%2900137-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252801%252900137-8%26sid%3Dliteratum%253Aachs%26aulast%3DPikul%26aufirst%3DS.%26aulast%3DDunham%26aufirst%3DK.%2BM.%26aulast%3DAlmstead%26aufirst%3DN.%2BG.%26aulast%3DDe%26aufirst%3DB.%26aulast%3DNatchus%26aufirst%3DM.%2BG.%26aulast%3DTaiwo%26aufirst%3DY.%2BO.%26aulast%3DWilliams%26aufirst%3DL.%2BE.%26aulast%3DHynd%26aufirst%3DB.%2BA.%26aulast%3DHsieh%26aufirst%3DL.%2BC.%26aulast%3DJanusz%26aufirst%3DM.%2BJ.%26aulast%3DGu%26aufirst%3DF.%26aulast%3DMieling%26aufirst%3DG.%2BE.%26atitle%3DHeterocycle-based%2520MMP%2520inhibitors%2520with%2520P2%25E2%2580%25B2%2520substituents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2001%26volume%3D11%26spage%3D1009%26epage%3D13%26doi%3D10.1016%2FS0960-894X%2801%2900137-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jehle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ochiana, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollastri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">E2585</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span><span class="refDoi"> DOI: 10.1073/pnas.1501567112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1501567112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=25918377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E2585-94&author=D.+Kozakovauthor=D.+R.+Hallauthor=S.+Jehleauthor=L.+Luoauthor=S.+O.+Ochianaauthor=E.+V.+Jonesauthor=M.+Pollastriauthor=K.+N.+Allenauthor=A.+Whittyauthor=S.+Vajda&title=Ligand+deconstruction%3A+Why+some+fragment+binding+positions+are+conserved+and+others+are+not&doi=10.1073%2Fpnas.1501567112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Jehle, Sefan; Luo, Lingqi; Ochiana, Stefan O.; Jones, Elizabeth V.; Pollastri, Michael; Allen, Karen N.; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">E2585-E2594</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) relies on the premise that the fragment binding mode will be conserved on subsequent expansion to a larger ligand.  However, no general condition has been established to explain when fragment binding modes will be conserved.  We show that a remarkably simple condition can be developed in terms of how fragments coincide with binding energy hot spots-regions of the protein where interactions with a ligand contribute substantial binding free energy-the locations of which can easily be detd. computationally.  Because a substantial fraction of the free energy of ligand binding comes from interacting with the residues in the energetically most important hot spot, a ligand moiety that sufficiently overlaps with this region will retain its location even when other parts of the ligand are removed.  This hypothesis is supported by eight case studies.  The condition helps identify whether a protein is suitable for FBDD, predicts the size of fragments required for screening, and dets. whether a fragment hit can be extended into a higher affinity ligand.  Our results show that ligand binding sites can usefully be thought of in terms of an anchor site, which is the top-ranked hot spot and dominates the free energy of binding, surrounded by a no. of weaker satellite sites that confer improved affinity and selectivity for a particular ligand and that it is the intrinsic binding potential of the protein surface that dets. whether it can serve as a robust binding site for a suitably optimized ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH6jbOut_iv7Vg90H21EOLACvtfcHk0ljQU99Wc6sP2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D&md5=d56b18fb4e4befde24037b07e52d8b62</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501567112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501567112%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DOchiana%26aufirst%3DS.%2BO.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DPollastri%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLigand%2520deconstruction%253A%2520Why%2520some%2520fragment%2520binding%2520positions%2520are%2520conserved%2520and%2520others%2520are%2520not%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE2585%26epage%3D94%26doi%3D10.1073%2Fpnas.1501567112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glasfeld, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haney, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seehra, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somers, W. S.</span><span> </span><span class="NLM_article-title">The bacterial cell-division protein ZipA and its interaction with an FtsZ fragment revealed by X-ray crystallography</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3179</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1093/emboj/19.13.3179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1093%2Femboj%2F19.13.3179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10880432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt1ajtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=3179-91&author=L.+Mosyakauthor=Y.+Zhangauthor=E.+Glasfeldauthor=S.+Haneyauthor=M.+Stahlauthor=J.+Seehraauthor=W.+S.+Somers&title=The+bacterial+cell-division+protein+ZipA+and+its+interaction+with+an+FtsZ+fragment+revealed+by+X-ray+crystallography&doi=10.1093%2Femboj%2F19.13.3179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">The bacterial cell-division protein ZipA and its interaction with an FtsZ fragment revealed by X-ray crystallography</span></div><div class="casAuthors">Mosyak, Lidia; Zhang, Yan; Glasfeld, Elizabeth; Haney, Steve; Stahl, Mark; Seehra, Jasbir; Somers, William S.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3179-3191</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In Escherichia coli, FtsZ, a homolog of eukaryotic tubulins, and ZipA, a membrane-anchored protein that binds to FtsZ, are two essential components of the septal ring structure that mediates cell division.  Recent data indicate that ZipA is involved in the assembly of the ring by linking FtsZ to the cytoplasmic membrane and that the ZipA-FtsZ interaction is mediated by their C-terminal domains.  We present the x-ray crystal structures of the C-terminal FtsZ-binding domain of ZipA and a complex between this domain and a C-terminal fragment of FtsZ.  The ZipA domain is a six-stranded β-sheet packed against three α-helixes and contains the split β-α-β motif found in many RNA-binding proteins.  The uncovered side of the sheet incorporates a shallow hydrophobic cavity exposed to solvent.  In the complex, the 17-residue FtsZ fragment occupies this entire cavity of ZipA and binds as an extended β-strand followed by α-helix.  An alanine-scanning mutagenesis anal. of the FtsZ fragment was also performed, which shows that only a small cluster of the buried FtsZ side chains is crit. in binding to ZipA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOPmByyCGAyLVg90H21EOLACvtfcHk0ljQU99Wc6sP2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlt1ajtb8%253D&md5=98ab31e86cb5de2f7955c011e342c25f</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.13.3179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.13.3179%26sid%3Dliteratum%253Aachs%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGlasfeld%26aufirst%3DE.%26aulast%3DHaney%26aufirst%3DS.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DSeehra%26aufirst%3DJ.%26aulast%3DSomers%26aufirst%3DW.%2BS.%26atitle%3DThe%2520bacterial%2520cell-division%2520protein%2520ZipA%2520and%2520its%2520interaction%2520with%2520an%2520FtsZ%2520fragment%2520revealed%2520by%2520X-ray%2520crystallography%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D3179%26epage%3D91%26doi%3D10.1093%2Femboj%2F19.13.3179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group">Jennings, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foreman, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rush, T. S.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsao, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sukhdeo, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moghazeh, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruzin, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuckman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, A. G.</span><span> </span><span class="NLM_article-title">Design and synthesis of indolo[2,3-a]quinolizin-7-one inhibitors of the ZipA-FtsZ interaction</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1427</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2004.01.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2004.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=15006376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvV2it78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=1427-31&author=L.+D.+Jenningsauthor=K.+W.+Foremanauthor=T.+S.+Rushauthor=D.+H.+Tsaoauthor=L.+Mosyakauthor=Y.+Liauthor=M.+N.+Sukhdeoauthor=W.+Dingauthor=E.+G.+Dushinauthor=C.+H.+Kennyauthor=S.+L.+Moghazehauthor=P.+J.+Petersenauthor=A.+V.+Ruzinauthor=M.+Tuckmanauthor=A.+G.+Sutherland&title=Design+and+synthesis+of+indolo%5B2%2C3-a%5Dquinolizin-7-one+inhibitors+of+the+ZipA-FtsZ+interaction&doi=10.1016%2Fj.bmcl.2004.01.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of indolo[2,3-a]quinolizin-7-one inhibitors of the ZipA-FtsZ interaction</span></div><div class="casAuthors">Jennings, Lee D.; Foreman, Ken W.; Rush, Thomas S.; Tsao, Desiree H. H.; Mosyak, Lidia; Li, Yuanhong; Sukhdeo, Mohani N.; Ding, Weidong; Dushin, Elizabeth G.; Kenny, Cynthia Hess; Moghazeh, Soraya L.; Petersen, Peter J.; Ruzin, Alexey V.; Tuckman, Margareta; Sutherland, Alan G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1427-1431</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The binding of FtsZ to ZipA is a potential target for antibacterial therapy.  Based on a small mol. inhibitor of the ZipA-FtsZ interaction, a parallel synthesis of small mols. was initiated which targeted a key region of ZipA involved in FtsZ binding.  The X-ray crystal structure of one of these mols. complexed with ZipA was solved.  The structure revealed an unexpected binding mode, facilitated by desolvation of a loosely bound surface water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFg6MqTNEkGrVg90H21EOLACvtfcHk0ljQU99Wc6sP2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhvV2it78%253D&md5=9899fa126c5b235f3ed0edf63f31c2be</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2004.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2004.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DJennings%26aufirst%3DL.%2BD.%26aulast%3DForeman%26aufirst%3DK.%2BW.%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DTsao%26aufirst%3DD.%2BH.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSukhdeo%26aufirst%3DM.%2BN.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DDushin%26aufirst%3DE.%2BG.%26aulast%3DKenny%26aufirst%3DC.%2BH.%26aulast%3DMoghazeh%26aufirst%3DS.%2BL.%26aulast%3DPetersen%26aufirst%3DP.%2BJ.%26aulast%3DRuzin%26aufirst%3DA.%2BV.%26aulast%3DTuckman%26aufirst%3DM.%26aulast%3DSutherland%26aufirst%3DA.%2BG.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520indolo%255B2%252C3-a%255Dquinolizin-7-one%2520inhibitors%2520of%2520the%2520ZipA-FtsZ%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2004%26volume%3D14%26spage%3D1427%26epage%3D31%26doi%3D10.1016%2Fj.bmcl.2004.01.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group">Kenny, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kriz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellestad, G.</span><span> </span><span class="NLM_article-title">Development of a fluorescence polarization assay to screen for inhibitors of the FtsZ/ZipA interaction</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">323</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">33</span><span class="refDoi"> DOI: 10.1016/j.ab.2003.08.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.ab.2003.08.033" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2003&pages=224-33&author=C.+H.+Kennyauthor=W.+D.+Dingauthor=K.+Kelleherauthor=S.+Benardauthor=E.+G.+Dushinauthor=A.+G.+Sutherlandauthor=L.+Mosyakauthor=R.+Krizauthor=G.+Ellestad&title=Development+of+a+fluorescence+polarization+assay+to+screen+for+inhibitors+of+the+FtsZ%2FZipA+interaction&doi=10.1016%2Fj.ab.2003.08.033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2003.08.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2003.08.033%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DC.%2BH.%26aulast%3DDing%26aufirst%3DW.%2BD.%26aulast%3DKelleher%26aufirst%3DK.%26aulast%3DBenard%26aufirst%3DS.%26aulast%3DDushin%26aufirst%3DE.%2BG.%26aulast%3DSutherland%26aufirst%3DA.%2BG.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DKriz%26aufirst%3DR.%26aulast%3DEllestad%26aufirst%3DG.%26atitle%3DDevelopment%2520of%2520a%2520fluorescence%2520polarization%2520assay%2520to%2520screen%2520for%2520inhibitors%2520of%2520the%2520FtsZ%252FZipA%2520interaction%26jtitle%3DAnal.%2520Biochem.%26date%3D2003%26volume%3D323%26spage%3D224%26epage%3D33%26doi%3D10.1016%2Fj.ab.2003.08.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group">Tsao, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jennings, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rush, T. S.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haney, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malakian, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilakantan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span> </span><span class="NLM_article-title">Discovery of novel inhibitors of the ZipA/FtsZ complex by NMR fragment screening coupled with structure-based design</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">7953</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1016/j.bmc.2006.07.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmc.2006.07.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16919463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFSrtLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=7953-61&author=D.+H.+Tsaoauthor=A.+G.+Sutherlandauthor=L.+D.+Jenningsauthor=Y.+Liauthor=T.+S.+Rushauthor=J.+C.+Alvarezauthor=W.+Dingauthor=E.+G.+Dushinauthor=R.+G.+Dushinauthor=S.+A.+Haneyauthor=C.+H.+Kennyauthor=A.+K.+Malakianauthor=R.+Nilakantanauthor=L.+Mosyak&title=Discovery+of+novel+inhibitors+of+the+ZipA%2FFtsZ+complex+by+NMR+fragment+screening+coupled+with+structure-based+design&doi=10.1016%2Fj.bmc.2006.07.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel inhibitors of the ZipA/FtsZ complex by NMR fragment screening coupled with structure-based design</span></div><div class="casAuthors">Tsao, Desiree H. H.; Sutherland, Alan G.; Jennings, Lee D.; Li, Yuanhong; Rush, Thomas S.; Alvarez, Juan C.; Ding, Weidong; Dushin, Elizabeth G.; Dushin, Russell G.; Haney, Steve A.; Kenny, Cynthia H.; Malakian, A. Karl; Nilakantan, Ramaswamy; Mosyak, Lidia</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7953-7961</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">ZipA is a membrane anchored protein in Escherichia coli that interacts with FtsZ, a homolog of eukaryotic tubulins, forming a septal ring structure that mediates bacterial cell division.  Thus, the ZipA/FtsZ protein-protein interaction is a potential target for an antibacterial agent.  The authors report here an NMR-based fragment screening approach which identified several hits that bind to the C-terminal region of ZipA.  The screen was performed by 1H-15N HSQC expts. on a library of 825 fragments that are small, lead-like, and highly sol.  Seven hits were identified, and the binding mode of the best one was revealed in the x-ray crystal structure.  Similar to the ZipA/FtsZ contacts, the driving force in the binding of the small mol. ligands to ZipA is achieved through hydrophobic interactions.  Analogs of this hit were also evaluated by NMR and x-ray crystal structures of these analogs with ZipA were obtained, providing structural information to help guide the medicinal chem. efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX3w_cIyvHYrVg90H21EOLACvtfcHk0lhfVi5M_9WUQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFSrtLbN&md5=ffe8d4f05d826414a13d55ddb2857579</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.07.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.07.050%26sid%3Dliteratum%253Aachs%26aulast%3DTsao%26aufirst%3DD.%2BH.%26aulast%3DSutherland%26aufirst%3DA.%2BG.%26aulast%3DJennings%26aufirst%3DL.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DAlvarez%26aufirst%3DJ.%2BC.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DDushin%26aufirst%3DE.%2BG.%26aulast%3DDushin%26aufirst%3DR.%2BG.%26aulast%3DHaney%26aufirst%3DS.%2BA.%26aulast%3DKenny%26aufirst%3DC.%2BH.%26aulast%3DMalakian%26aufirst%3DA.%2BK.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DMosyak%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520novel%2520inhibitors%2520of%2520the%2520ZipA%252FFtsZ%2520complex%2520by%2520NMR%2520fragment%2520screening%2520coupled%2520with%2520structure-based%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D7953%26epage%3D61%26doi%3D10.1016%2Fj.bmc.2006.07.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group">Jennings, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foreman, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rush, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsao, D. H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kincaid, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sukhdeo, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabus, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dushin, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moghazeh, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labthavikul, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuckman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haney, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruzin, A. V.</span><span> </span><span class="NLM_article-title">Combinatorial synthesis of substituted 3-(2-indolyl)piperidines and 2-phenyl indoles as inhibitors of ZipA-FtsZ interaction</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5115</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1016/j.bmc.2004.07.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmc.2004.07.031" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=5115-31&author=L.+D.+Jenningsauthor=K.+W.+Foremanauthor=T.+S.+Rushauthor=D.+H.+H.+Tsaoauthor=L.+Mosyakauthor=S.+L.+Kincaidauthor=M.+N.+Sukhdeoauthor=A.+G.+Sutherlandauthor=W.+D.+Dingauthor=C.+H.+Kennyauthor=C.+L.+Sabusauthor=H.+L.+Liuauthor=E.+G.+Dushinauthor=S.+L.+Moghazehauthor=P.+Labthavikulauthor=P.+J.+Petersenauthor=M.+Tuckmanauthor=S.+A.+Haneyauthor=A.+V.+Ruzin&title=Combinatorial+synthesis+of+substituted+3-%282-indolyl%29piperidines+and+2-phenyl+indoles+as+inhibitors+of+ZipA-FtsZ+interaction&doi=10.1016%2Fj.bmc.2004.07.031"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2004.07.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2004.07.031%26sid%3Dliteratum%253Aachs%26aulast%3DJennings%26aufirst%3DL.%2BD.%26aulast%3DForeman%26aufirst%3DK.%2BW.%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DTsao%26aufirst%3DD.%2BH.%2BH.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DKincaid%26aufirst%3DS.%2BL.%26aulast%3DSukhdeo%26aufirst%3DM.%2BN.%26aulast%3DSutherland%26aufirst%3DA.%2BG.%26aulast%3DDing%26aufirst%3DW.%2BD.%26aulast%3DKenny%26aufirst%3DC.%2BH.%26aulast%3DSabus%26aufirst%3DC.%2BL.%26aulast%3DLiu%26aufirst%3DH.%2BL.%26aulast%3DDushin%26aufirst%3DE.%2BG.%26aulast%3DMoghazeh%26aufirst%3DS.%2BL.%26aulast%3DLabthavikul%26aufirst%3DP.%26aulast%3DPetersen%26aufirst%3DP.%2BJ.%26aulast%3DTuckman%26aufirst%3DM.%26aulast%3DHaney%26aufirst%3DS.%2BA.%26aulast%3DRuzin%26aufirst%3DA.%2BV.%26atitle%3DCombinatorial%2520synthesis%2520of%2520substituted%25203-%25282-indolyl%2529piperidines%2520and%25202-phenyl%2520indoles%2520as%2520inhibitors%2520of%2520ZipA-FtsZ%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D5115%26epage%3D31%26doi%3D10.1016%2Fj.bmc.2004.07.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group">Rush, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosyak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">A shape-based 3-D scaffold hopping method and its application to a bacterial protein-protein interaction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1489</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.1021/jm040163o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040163o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Ols78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1489-95&author=T.+S.+Rushauthor=J.+A.+Grantauthor=L.+Mosyakauthor=A.+Nicholls&title=A+shape-based+3-D+scaffold+hopping+method+and+its+application+to+a+bacterial+protein-protein+interaction&doi=10.1021%2Fjm040163o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">A Shape-Based 3-D Scaffold Hopping Method and Its Application to a Bacterial Protein-Protein Interaction</span></div><div class="casAuthors">Rush, Thomas S., III; Grant, J. Andrew; Mosyak, Lidia; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1489-1495</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, the authors describe the first prospective application of the shape-comparison program ROCS (Rapid Overlay of Chem. Structures) to find new scaffolds for small mol. inhibitors of the ZipA-FtsZ protein-protein interaction, a proposed antibacterial target.  The shape comparisons are made relative to the crystallog. detd., bioactive conformation of a high-throughput screening (HTS) hit.  The use of ROCS led to the identification of a set of novel, weakly binding inhibitors with scaffolds presenting synthetic opportunities to further optimize biol. affinity and lacking development issues assocd. with the HTS lead.  These ROCS-identified scaffolds would have been missed using other structural similarity approaches such as ISIS 2D fingerprints.  X-ray crystallog. anal. of one of the new inhibitors bound to ZipA reveals that the shape comparison approach very accurately predicted the binding mode.  These exptl. results validate this use of ROCS for chemotype switching or "lead hopping" and suggest that it is of general interest for lead identification in drug discovery endeavors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4EX9jFtct37Vg90H21EOLACvtfcHk0lhfVi5M_9WUQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Ols78%253D&md5=05b6e54a657a3b8c768e63852d871ef6</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fjm040163o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040163o%26sid%3Dliteratum%253Aachs%26aulast%3DRush%26aufirst%3DT.%2BS.%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DMosyak%26aufirst%3DL.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DA%2520shape-based%25203-D%2520scaffold%2520hopping%2520method%2520and%2520its%2520application%2520to%2520a%2520bacterial%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1489%26epage%3D95%26doi%3D10.1021%2Fjm040163o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group">Rai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vyjayanti, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorjsuren, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span> </span><span class="NLM_article-title">Synthesis, biological evaluation, and structure-activity relationships of a novel class of apurinic/apyrimidinic endonuclease 1 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">3101</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.1021/jm201537d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201537d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVWmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3101-12&author=G.+Raiauthor=V.+N.+Vyjayantiauthor=D.+Dorjsurenauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+M.+Wilsonauthor=D.+J.+Maloney&title=Synthesis%2C+biological+evaluation%2C+and+structure-activity+relationships+of+a+novel+class+of+apurinic%2Fapyrimidinic+endonuclease+1+inhibitors&doi=10.1021%2Fjm201537d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological evaluation, and structure-activity relationships of a novel class of apurinic/apyrimidinic endonuclease 1 inhibitors</span></div><div class="casAuthors">Rai, Ganesha; Vyjayanti, Vaddadi N.; Dorjsuren, Dorjbal; Simeonov, Anton; Jadhav, Ajit; Wilson, David M.; Maloney, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3101-3112</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">APE1 is an essential protein that operates in the base excision repair (BER) pathway and is responsible for ≥95% of the total apurinic/apyrimidinic (AP) endonuclease activity in human cells.  BER is a major pathway that copes with DNA damage induced by several anticancer agents, including ionizing radiation and temozolomide.  Overexpression of APE1 and enhanced AP endonuclease activity have been linked to increased resistance of tumor cells to treatment with monofunctional alkylators, implicating inhibition of APE1 as a valid strategy for cancer therapy.  We report herein the results of a focused medicinal chem. effort around a novel APE1 inhibitor, N-(3-(benzo[d]thiazol-2-yl)-6-isopropyl-4,5,6,7-tetrahydrothieno[2,3-c]pyridin-2-yl)acetamide (I).  Compd. I and related analogs exhibit single-digit micromolar activity against the purified APE1 enzyme and comparable activity in HeLa whole cell ext. assays and potentiate the cytotoxicity of the alkylating agents methylmethane sulfonate and temozolomide.  Moreover, this class of compds. possesses a generally favorable in vitro ADME profile, along with good exposure levels in plasma and brain following i.p. dosing (30 mg/kg body wt.) in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq25R0Ebe53srVg90H21EOLACvtfcHk0lhtc3yQEnS60w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVWmt7o%253D&md5=1dac481ed0ec1cc67d59ad3091520ccc</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fjm201537d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201537d%26sid%3Dliteratum%253Aachs%26aulast%3DRai%26aufirst%3DG.%26aulast%3DVyjayanti%26aufirst%3DV.%2BN.%26aulast%3DDorjsuren%26aufirst%3DD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DSynthesis%252C%2520biological%2520evaluation%252C%2520and%2520structure-activity%2520relationships%2520of%2520a%2520novel%2520class%2520of%2520apurinic%252Fapyrimidinic%2520endonuclease%25201%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3101%26epage%3D12%26doi%3D10.1021%2Fjm201537d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group">Abbotts, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madhusudan, S.</span><span> </span><span class="NLM_article-title">Human AP endonuclease 1 (APE1): From mechanistic insights to druggable target in cancer</span> <span class="citation_source-journal">Cancer Treat. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">425</span><span class="NLM_x">–</span> <span class="NLM_lpage">35</span><span class="refDoi"> DOI: 10.1016/j.ctrv.2009.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.ctrv.2009.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=20056333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpt1Ont7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2010&pages=425-35&author=R.+Abbottsauthor=S.+Madhusudan&title=Human+AP+endonuclease+1+%28APE1%29%3A+From+mechanistic+insights+to+druggable+target+in+cancer&doi=10.1016%2Fj.ctrv.2009.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Human AP endonuclease 1 (APE1): from mechanistic insights to druggable target in cancer</span></div><div class="casAuthors">Abbotts, Rachel; Madhusudan, Srinivasan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">425-435</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  DNA base excision repair (BER) is critically involved in the processing of DNA base damage induced by alkylating agents.  Pharmacol. inhibition of BER (using PARP inhibitors), either alone or in combination with chemotherapy has recently shown promise in clin. trials.  Human apurinic/apyrimidinic endonuclease 1(APE1) is an essential BER protein that is involved in the processing of potentially cytotoxic abasic sites that are obligatory intermediates in BER.  Here we provide a summary of the basic mechanistic role of APE1 in DNA repair and redox regulation and highlight preclin. and clin. data that confirm APE1 as a valid anticancer drug target.  Development of small mol. inhibitors of APE1 is an area of intense research and current evidence using APE1 inhibitors has demonstrated potentiation of cytotoxicity of alkylating agents in preclin. models implying translational applications in cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWbrntssmy5bVg90H21EOLACvtfcHk0lhtc3yQEnS60w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpt1Ont7k%253D&md5=14721f4b897a254b307007042a162170</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2009.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2009.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DAbbotts%26aufirst%3DR.%26aulast%3DMadhusudan%26aufirst%3DS.%26atitle%3DHuman%2520AP%2520endonuclease%25201%2520%2528APE1%2529%253A%2520From%2520mechanistic%2520insights%2520to%2520druggable%2520target%2520in%2520cancer%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2010%26volume%3D36%26spage%3D425%26epage%3D35%26doi%3D10.1016%2Fj.ctrv.2009.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group">Wilson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of DNA repair nuclease activities of APE1</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">3621</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1007/s00018-010-0488-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1007%2Fs00018-010-0488-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=20809131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlSjsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2010&pages=3621-31&author=D.+M.+Wilsonauthor=A.+Simeonov&title=Small+molecule+inhibitors+of+DNA+repair+nuclease+activities+of+APE1&doi=10.1007%2Fs00018-010-0488-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of DNA repair nuclease activities of APE1</span></div><div class="casAuthors">Wilson, David M., III; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3621-3631</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  APE1 is a multifunctional protein that possesses several nuclease activities, including the ability to incise at apurinic/apyrimidinic (AP) sites in DNA or RNA, to excise 3'-blocking termini from DNA ends, and to cleave at certain oxidized base lesions in DNA.  Pre-clin. and clin. data indicate a role for APE1 in the pathogenesis of cancer and in resistance to DNA-interactive drugs, particularly monofunctional alkylators and antimetabolites.  In an effort to improve the efficacy of therapeutic compds., such as temozolomide, groups have begun to develop high-throughput screening assays and to identify small mol. inhibitors against APE1 repair nuclease activities.  It is envisioned that such inhibitors will be used in combinatorial treatment paradigms to enhance the efficacy of DNA-interactive drugs that introduce relevant cytotoxic DNA lesions.  In this review, we summarize the current state of the efforts to design potent and selective inhibitors against APE1 AP site incision activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN4mUcnbEDarVg90H21EOLACvtfcHk0lhtc3yQEnS60w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlSjsLjN&md5=7406c275670396d1652dfda2cf68e2af</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1007%2Fs00018-010-0488-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-010-0488-2%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520DNA%2520repair%2520nuclease%2520activities%2520of%2520APE1%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2010%26volume%3D67%26spage%3D3621%26epage%3D31%26doi%3D10.1007%2Fs00018-010-0488-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group">Patel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahekar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadav, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joharapurkar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanesha, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaikh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sairam, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapadnis, P.</span><span> </span><span class="NLM_article-title">Discovery of potent, selective and orally bioavailable triaryl-sulfonamide based PTP1B inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1111</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.11.122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2011.11.122" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1111-7&author=D.+Patelauthor=M.+Jainauthor=S.+R.+Shahauthor=R.+Bahekarauthor=P.+Jadavauthor=A.+Joharapurkarauthor=N.+Dhaneshaauthor=M.+Shaikhauthor=K.+V.+Sairamauthor=P.+Kapadnis&title=Discovery+of+potent%2C+selective+and+orally+bioavailable+triaryl-sulfonamide+based+PTP1B+inhibitors&doi=10.1016%2Fj.bmcl.2011.11.122"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.11.122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.11.122%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DShah%26aufirst%3DS.%2BR.%26aulast%3DBahekar%26aufirst%3DR.%26aulast%3DJadav%26aufirst%3DP.%26aulast%3DJoharapurkar%26aufirst%3DA.%26aulast%3DDhanesha%26aufirst%3DN.%26aulast%3DShaikh%26aufirst%3DM.%26aulast%3DSairam%26aufirst%3DK.%2BV.%26aulast%3DKapadnis%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520potent%252C%2520selective%2520and%2520orally%2520bioavailable%2520triaryl-sulfonamide%2520based%2520PTP1B%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D1111%26epage%3D7%26doi%3D10.1016%2Fj.bmcl.2011.11.122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group">Scapin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desponts, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waddleton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skorey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cromlish, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gauthier, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramachandran, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asante-Appiah, E.</span><span> </span><span class="NLM_article-title">The structural basis for the selectivity of benzotriazole inhibitors of PTP1B</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">11451</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1021/bi035098j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi035098j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=11451-9&author=G.+Scapinauthor=S.+B.+Patelauthor=J.+W.+Beckerauthor=Q.+P.+Wangauthor=C.+Despontsauthor=D.+Waddletonauthor=K.+Skoreyauthor=W.+Cromlishauthor=C.+Baylyauthor=M.+Therienauthor=J.+Y.+Gauthierauthor=C.+S.+Liauthor=C.+K.+Lauauthor=C.+Ramachandranauthor=B.+P.+Kennedyauthor=E.+Asante-Appiah&title=The+structural+basis+for+the+selectivity+of+benzotriazole+inhibitors+of+PTP1B&doi=10.1021%2Fbi035098j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1021%2Fbi035098j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi035098j%26sid%3Dliteratum%253Aachs%26aulast%3DScapin%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DS.%2BB.%26aulast%3DBecker%26aufirst%3DJ.%2BW.%26aulast%3DWang%26aufirst%3DQ.%2BP.%26aulast%3DDesponts%26aufirst%3DC.%26aulast%3DWaddleton%26aufirst%3DD.%26aulast%3DSkorey%26aufirst%3DK.%26aulast%3DCromlish%26aufirst%3DW.%26aulast%3DBayly%26aufirst%3DC.%26aulast%3DTherien%26aufirst%3DM.%26aulast%3DGauthier%26aufirst%3DJ.%2BY.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DLau%26aufirst%3DC.%2BK.%26aulast%3DRamachandran%26aufirst%3DC.%26aulast%3DKennedy%26aufirst%3DB.%2BP.%26aulast%3DAsante-Appiah%26aufirst%3DE.%26atitle%3DThe%2520structural%2520basis%2520for%2520the%2520selectivity%2520of%2520benzotriazole%2520inhibitors%2520of%2520PTP1B%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D11451%26epage%3D9%26doi%3D10.1021%2Fbi035098j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref150"><div class="reference"><strong class="refLabel"><a href="#ref150" class="rightTabRefNumLink">150</a></strong><div class="NLM_citation" id="rightTab-cit150"><span><span class="NLM_contrib-group">Barr, A. J.</span><span> </span><span class="NLM_article-title">Protein tyrosine phosphatases as drug targets: Strategies and challenges of inhibitor development</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">1563</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span><span class="refDoi"> DOI: 10.4155/fmc.10.241</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.4155%2Ffmc.10.241" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21426149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yqsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=1563-76&author=A.+J.+Barr&title=Protein+tyrosine+phosphatases+as+drug+targets%3A+Strategies+and+challenges+of+inhibitor+development&doi=10.4155%2Ffmc.10.241"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Protein tyrosine phosphatases as drug targets: strategies and challenges of inhibitor development</span></div><div class="casAuthors">Barr, Alastair J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1563-1576</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Several classical' protein tyrosine phosphatases are attractive therapeutic targets, including PTP1B for obesity and Type II diabetes; SHP2 for cancer and Lyp for rheumatoid arthritis.  Progress has been made in identifying a broad range of chem. distinct inhibitors; however, developing selective and cell-permeable clin. useful compds. has proved challenging.  Here the ongoing challenges and recent significant advances in the field are reviewed.  Key novel compds. are highlighted and a perspective on the future of phosphatase inhibitor development is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUabcAXtUSRrVg90H21EOLACvtfcHk0liuysi0y4QTbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yqsbjE&md5=2e0b3ae741824174eb0b49f1f7d69c10</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.241&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.241%26sid%3Dliteratum%253Aachs%26aulast%3DBarr%26aufirst%3DA.%2BJ.%26atitle%3DProtein%2520tyrosine%2520phosphatases%2520as%2520drug%2520targets%253A%2520Strategies%2520and%2520challenges%2520of%2520inhibitor%2520development%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D1563%26epage%3D76%26doi%3D10.4155%2Ffmc.10.241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref151"><div class="reference"><strong class="refLabel"><a href="#ref151" class="rightTabRefNumLink">151</a></strong><div class="NLM_citation" id="rightTab-cit151"><span><span class="NLM_contrib-group">Douty, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wayland, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ala, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bower, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pruitt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bostrom, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klabe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonneville, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynn, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burn, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, P. C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combs, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, E. W.</span><span> </span><span class="NLM_article-title">Isothiazolidinone inhibitors of PTP1B containing imidazoles and imidazolines</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">66</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2007.11.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=66-71&author=B.+Doutyauthor=B.+Waylandauthor=P.+J.+Alaauthor=M.+J.+Bowerauthor=J.+Pruittauthor=L.+Bostromauthor=M.+Weiauthor=R.+Klabeauthor=L.+Gonnevilleauthor=R.+Wynnauthor=T.+C.+Burnauthor=P.+C.+C.+Liuauthor=A.+P.+Combsauthor=E.+W.+Yue&title=Isothiazolidinone+inhibitors+of+PTP1B+containing+imidazoles+and+imidazolines&doi=10.1016%2Fj.bmcl.2007.11.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DDouty%26aufirst%3DB.%26aulast%3DWayland%26aufirst%3DB.%26aulast%3DAla%26aufirst%3DP.%2BJ.%26aulast%3DBower%26aufirst%3DM.%2BJ.%26aulast%3DPruitt%26aufirst%3DJ.%26aulast%3DBostrom%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DM.%26aulast%3DKlabe%26aufirst%3DR.%26aulast%3DGonneville%26aufirst%3DL.%26aulast%3DWynn%26aufirst%3DR.%26aulast%3DBurn%26aufirst%3DT.%2BC.%26aulast%3DLiu%26aufirst%3DP.%2BC.%2BC.%26aulast%3DCombs%26aufirst%3DA.%2BP.%26aulast%3DYue%26aufirst%3DE.%2BW.%26atitle%3DIsothiazolidinone%2520inhibitors%2520of%2520PTP1B%2520containing%2520imidazoles%2520and%2520imidazolines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D66%26epage%3D71%26doi%3D10.1016%2Fj.bmcl.2007.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref152"><div class="reference"><strong class="refLabel"><a href="#ref152" class="rightTabRefNumLink">152</a></strong><div class="NLM_citation" id="rightTab-cit152"><span><span class="NLM_contrib-group">Zhao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbieri, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H. A.</span><span> </span><span class="NLM_article-title">Discovery and structural characterization of a small molecule 14–3-3 protein-protein interaction inhibitor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">16212</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1073/pnas.1100012108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1100012108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=21908710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Kmtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=16212-6&author=J.+Zhaoauthor=Y.+H.+Duauthor=J.+R.+Hortonauthor=A.+K.+Upadhyayauthor=B.+Louauthor=Y.+Baiauthor=X.+Zhangauthor=L.+P.+Duauthor=M.+Y.+Liauthor=B.+H.+Wangauthor=L.+X.+Zhangauthor=J.+T.+Barbieriauthor=F.+R.+Khuriauthor=X.+D.+Chengauthor=H.+A.+Fu&title=Discovery+and+structural+characterization+of+a+small+molecule+14%E2%80%933-3+protein-protein+interaction+inhibitor&doi=10.1073%2Fpnas.1100012108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and structural characterization of a small molecule 14-3-3 protein-protein interaction inhibitor</span></div><div class="casAuthors">Zhao, Jing; Du, Yuhong; Horton, John R.; Upadhyay, Anup K.; Lou, Bin; Bai, Yan; Zhang, Xing; Du, Lupei; Li, Minyong; Wang, Binghe; Zhang, Lixin; Barbieri, Joseph T.; Khuri, Fadlo R.; Cheng, Xiaodong; Fu, Haian</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">16212-16216, S16212/1-S16212/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 14-3-3 family of phosphoserine/threonine-recognition proteins engage multiple nodes in signaling networks that control diverse physiol. and pathophysiol. functions and have emerged as promising therapeutic target; for such diseases as cancer and neurodegenerative disorders.  Thus, small mol. modulators of 14-3-3 are much needed agents for chem. biol. investigations and therapeutic development.  To analyze 14-3-3 function and modulate its activity, we conducted a chem. screen and identified 4-[(2Z)-2-[4-formyl-6-methyl-5-oxo-3-(phosphonatooxymethyl)pyridin-2-ylidene]hydrazinyl]benzoate as a 14-3-3 inhibitor, which we termed FOBISIN (FOurteen-three-three Binding Small mol. INhibitor) 101.  FOBISIN101 effectively blocked the binding of 14-3-3 with Raf-1 and proline-rich AKT substrate, 40 kDa and neutralized the ability of 14-3-3 to activate exoenzyme S ADP-ribosyltransferase.  To provide a mechanistic basis for 14-3-3 inhibition, the crystal structure of 14-3-3ζ in complex with FOBISIN101 was solved.  Unexpectedly, the double bond linking the pyridoxal-phosphate and benzoate moieties was reduced by X-rays to create a covalent linkage of the pyridoxal-phosphate moiety to lysine 120 in the binding groove of 14-3-3, leading to persistent 14-3-3 inactivation.  We suggest that FOBISIN101-like mols. could be developed as an entirely unique class of 14-3-3 inhibitors, which may serve as radiation-triggered therapeutic agents for the treatment of 14-3-3-mediated diseases, such as cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR0iuk-c1eP7Vg90H21EOLACvtfcHk0liuysi0y4QTbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Kmtb7K&md5=ba66810da0f7b1f6e9e16701f58ef8c2</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1100012108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1100012108%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DY.%2BH.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DLou%26aufirst%3DB.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DL.%2BP.%26aulast%3DLi%26aufirst%3DM.%2BY.%26aulast%3DWang%26aufirst%3DB.%2BH.%26aulast%3DZhang%26aufirst%3DL.%2BX.%26aulast%3DBarbieri%26aufirst%3DJ.%2BT.%26aulast%3DKhuri%26aufirst%3DF.%2BR.%26aulast%3DCheng%26aufirst%3DX.%2BD.%26aulast%3DFu%26aufirst%3DH.%2BA.%26atitle%3DDiscovery%2520and%2520structural%2520characterization%2520of%2520a%2520small%2520molecule%252014%25E2%2580%25933-3%2520protein-protein%2520interaction%2520inhibitor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D16212%26epage%3D6%26doi%3D10.1073%2Fpnas.1100012108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref153"><div class="reference"><strong class="refLabel"><a href="#ref153" class="rightTabRefNumLink">153</a></strong><div class="NLM_citation" id="rightTab-cit153"><span><span class="NLM_contrib-group">Gardino, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smerdon, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yaffe, M. B.</span><span> </span><span class="NLM_article-title">Structural determinants of 14–3-3 binding specificities and regulation of subcellular localization of 14–3-3-ligand complexes: A comparison of the x-ray crystal structures of all human 14–3-3 isoforms</span> <span class="citation_source-journal">Semin. Cancer Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">173</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/j.semcancer.2006.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.semcancer.2006.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=16678437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BD28XltlKmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=173-82&author=A.+K.+Gardinoauthor=S.+J.+Smerdonauthor=M.+B.+Yaffe&title=Structural+determinants+of+14%E2%80%933-3+binding+specificities+and+regulation+of+subcellular+localization+of+14%E2%80%933-3-ligand+complexes%3A+A+comparison+of+the+x-ray+crystal+structures+of+all+human+14%E2%80%933-3+isoforms&doi=10.1016%2Fj.semcancer.2006.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of 14-3-3 binding specificities and regulation of subcellular localization of 14-3-3-ligand complexes: A comparison of the x-ray crystal structures of all human 14-3-3 isoforms</span></div><div class="casAuthors">Gardino, Alexandra K.; Smerdon, Stephen J.; Yaffe, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">173-182</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The 14-3-3 proteins are a ubiquitous class of regulatory proteins found in all eukaryotic cells and were the 1st class of mols. to be recognized as discrete phosphoserine/threonine-binding modules.  The 14-3-3 proteins bind a large no. of different substrates to regulate a wide array of cellular signaling events including cell cycle progression and DNA damage responses, programmed cell death, cytoskeletal dynamics, transcriptional control of gene expression, as well as processes directly related to cancer progression.  Here, the structural basis of phosphorylation-dependent binding of 14-3-3 proteins to peptide and protein ligands is discussed along with mechanisms that govern how 14-3-3 proteins regulate the function of their bound ligands.  The x-ray crystal structures of all human 14-3-3 proteins bound to peptides have now been solved.  Here, the authors use structural comparisons between isoforms as a framework for discussion of ligand binding by 14-3-3 proteins as well as the mechanisms through which post-translational modification of the different isoforms alters their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrFYLdMPrZBrVg90H21EOLACvtfcHk0li66gGki-uh2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltlKmsrg%253D&md5=ba9a8d49d70cdba9a9d8275ac0471030</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2006.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2006.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DGardino%26aufirst%3DA.%2BK.%26aulast%3DSmerdon%26aufirst%3DS.%2BJ.%26aulast%3DYaffe%26aufirst%3DM.%2BB.%26atitle%3DStructural%2520determinants%2520of%252014%25E2%2580%25933-3%2520binding%2520specificities%2520and%2520regulation%2520of%2520subcellular%2520localization%2520of%252014%25E2%2580%25933-3-ligand%2520complexes%253A%2520A%2520comparison%2520of%2520the%2520x-ray%2520crystal%2520structures%2520of%2520all%2520human%252014%25E2%2580%25933-3%2520isoforms%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D173%26epage%3D82%26doi%3D10.1016%2Fj.semcancer.2006.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref154"><div class="reference"><strong class="refLabel"><a href="#ref154" class="rightTabRefNumLink">154</a></strong><div class="NLM_citation" id="rightTab-cit154"><span><span class="NLM_contrib-group">De Vries-van Leeuwen, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">da Costa Pereira, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flach, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piersma, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haase, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bier, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yalcin, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalides, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feenstra, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jimenez, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Greef, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunsveld, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottmann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Boer, A. H.</span><span> </span><span class="NLM_article-title">Interaction of 14–3-3 proteins with the estrogen receptor alpha f domain provides a drug target interface</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">8894</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1073/pnas.1220809110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1220809110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23676274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFait7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8894-9&author=I.+J.+De+Vries-van+Leeuwenauthor=D.+da+Costa+Pereiraauthor=K.+D.+Flachauthor=S.+R.+Piersmaauthor=C.+Haaseauthor=D.+Bierauthor=Z.+Yalcinauthor=R.+Michalidesauthor=K.+A.+Feenstraauthor=C.+R.+Jimenezauthor=T.+F.+de+Greefauthor=L.+Brunsveldauthor=C.+Ottmannauthor=W.+Zwartauthor=A.+H.+de+Boer&title=Interaction+of+14%E2%80%933-3+proteins+with+the+estrogen+receptor+alpha+f+domain+provides+a+drug+target+interface&doi=10.1073%2Fpnas.1220809110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of 14-3-3 proteins with the Estrogen Receptor Alpha F domain provides a drug target interface</span></div><div class="casAuthors">De Vries-van Leeuwen, Ingrid J.; Pereira, Daniel da Costa; Flach, Koen D.; Piersma, Sander R.; Haase, Christian; Bier, David; Yalcin, Zeliha; Michalides, Rob; Feenstra, K. Anton; Jimenez, Connie R.; de Greef, Tom F. A.; Brunsveld, Luc; Ottmann, Christian; Zwart, Wilbert; de Boer, Albertus H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8894-8899, S8894/1-S8894/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Estrogen receptor alpha (ERa) is involved in numerous physiol. and pathol. processes, including breast cancer.  Breast cancer therapy is therefore currently directed at inhibiting the transcriptional potency of ERa, either by blocking estrogen prodn. through aromatase inhibitors or antiestrogens that compete for hormone binding.  Due to resistance, new treatment modalities are needed and as ERα dimerization is essential for its activity, interference with receptor dimerization offers a new opportunity to exploit in drug design.  Here we describe a unique mechanism of how ERα dimerization is neg. controlled by interaction with 14-3-3 proteins at the extreme C terminus of the receptor.  Moreover, the small-mol. fusicoccin (FC) stabilizes this ERα/14-3-3 interaction.  Cocrystn. of the trimeric ERα/14-3-3/FC complex provides the structural basis for this stabilization and shows the importance of phosphorylation of the penultimate Threonine (ERα-T594) for high-affinity interaction.  We confirm that T594 is a distinct ERα phosphorylation site in the breast cancer cell line MCF-7 using a phospho-T594-specific antibody and by mass spectrometry.  In line with its ERα/14-3-3 interaction stabilizing effect, fusicoccin reduces the estradiol-stimulated ERα dimerization, inhibits ERα/chromatin interactions and downstream gene expression, resulting in decreased cell proliferation.  Herewith, a unique functional phosphosite and an alternative regulation mechanism of ERα are provided, together with a small mol. that selectively targets this ERα/14-3-3 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0rga7Qdqyn7Vg90H21EOLACvtfcHk0li66gGki-uh2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFait7%252FP&md5=d17c1a1122f4b525b35651d3d9f629f0</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1220809110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1220809110%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVries-van%2BLeeuwen%26aufirst%3DI.%2BJ.%26aulast%3Dda%2BCosta%2BPereira%26aufirst%3DD.%26aulast%3DFlach%26aufirst%3DK.%2BD.%26aulast%3DPiersma%26aufirst%3DS.%2BR.%26aulast%3DHaase%26aufirst%3DC.%26aulast%3DBier%26aufirst%3DD.%26aulast%3DYalcin%26aufirst%3DZ.%26aulast%3DMichalides%26aufirst%3DR.%26aulast%3DFeenstra%26aufirst%3DK.%2BA.%26aulast%3DJimenez%26aufirst%3DC.%2BR.%26aulast%3Dde%2BGreef%26aufirst%3DT.%2BF.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DZwart%26aufirst%3DW.%26aulast%3Dde%2BBoer%26aufirst%3DA.%2BH.%26atitle%3DInteraction%2520of%252014%25E2%2580%25933-3%2520proteins%2520with%2520the%2520estrogen%2520receptor%2520alpha%2520f%2520domain%2520provides%2520a%2520drug%2520target%2520interface%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D8894%26epage%3D9%26doi%3D10.1073%2Fpnas.1220809110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref155"><div class="reference"><strong class="refLabel"><a href="#ref155" class="rightTabRefNumLink">155</a></strong><div class="NLM_citation" id="rightTab-cit155"><span><span class="NLM_contrib-group">Thiel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roglin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meissner, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hennig, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohlbacher, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottmann, C.</span><span> </span><span class="NLM_article-title">Virtual screening and experimental validation reveal novel small-molecule inhibitors of 14–3-3 protein-protein interactions</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">8468</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1039/c3cc44612c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1039%2Fc3cc44612c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23939230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlChur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=8468-70&author=P.+Thielauthor=L.+Roglinauthor=N.+Meissnerauthor=S.+Hennigauthor=O.+Kohlbacherauthor=C.+Ottmann&title=Virtual+screening+and+experimental+validation+reveal+novel+small-molecule+inhibitors+of+14%E2%80%933-3+protein-protein+interactions&doi=10.1039%2Fc3cc44612c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit155R"><div class="casContent"><span class="casTitleNuber">155</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening and experimental validation reveal novel small-molecule inhibitors of 14-3-3 protein-protein interactions</span></div><div class="casAuthors">Thiel, Philipp; Roeglin, Lars; Meissner, Nicole; Hennig, Sven; Kohlbacher, Oliver; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">76</span>),
    <span class="NLM_cas:pages">8468-8470</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors report first noncovalent and exclusively extracellular inhibitors of 14-3-3 protein-protein interactions identified by virtual screening.  Optimization by crystal structure anal. and in vitro binding assays yielded compds. capable of disrupting the interaction of 14-3-3σ with MMP-1 in a cellular assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9WHSGFCZit7Vg90H21EOLACvtfcHk0ljNYP8kX90lHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlChur%252FP&md5=2c6f7a9969bd1d08f288e8a3f4ecce80</span></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1039%2Fc3cc44612c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3cc44612c%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DP.%26aulast%3DRoglin%26aufirst%3DL.%26aulast%3DMeissner%26aufirst%3DN.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DKohlbacher%26aufirst%3DO.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DVirtual%2520screening%2520and%2520experimental%2520validation%2520reveal%2520novel%2520small-molecule%2520inhibitors%2520of%252014%25E2%2580%25933-3%2520protein-protein%2520interactions%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2013%26volume%3D49%26spage%3D8468%26epage%3D70%26doi%3D10.1039%2Fc3cc44612c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref156"><div class="reference"><strong class="refLabel"><a href="#ref156" class="rightTabRefNumLink">156</a></strong><div class="NLM_citation" id="rightTab-cit156"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frett, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H. Y.</span><span> </span><span class="NLM_article-title">Targeting mutant kras for anticancer therapeutics: A review of novel small molecule modulators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">5219</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1021/jm3017706</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3017706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5219-30&author=Y.+Wangauthor=C.+E.+Kaiserauthor=B.+Frettauthor=H.+Y.+Li&title=Targeting+mutant+kras+for+anticancer+therapeutics%3A+A+review+of+novel+small+molecule+modulators&doi=10.1021%2Fjm3017706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Mutant KRAS for Anticancer Therapeutics: A Review of Novel Small Molecule Modulators</span></div><div class="casAuthors">Wang, Yuanxiang; Kaiser, Christine E.; Frett, Brendan; Li, Hong-yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5219-5230</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The RAS proteins play a role in cell differentiation, proliferation, and survival.  Aberrant RAS signaling has been found to play a role in 30% of all cancers.  KRAS, a key member of the RAS protein family, is an attractive cancer target, as frequent point mutations in the KRAS gene render the protein constitutively active.  A no. of attempts have been made to target aberrant KRAS signaling by identifying small mol. compds. that (1) are synthetic lethal to mutant KRAS, (2) block KRAS/GEF interactions, (3) inhibit downstream KRAS effectors, or (4) inhibit the post-translational processing of RAS proteins.  In addn., inhibition of novel targets outside the main KRAS signaling pathway, specifically the cell cycle related kinase PLK1, has been shown have an effect in cells that harbor mutant KRAS.  Herein we review the use of various high-throughput screening assays utilized to identify new small-mol. compds. capable of targeting mutant KRAS-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogrA1KFIgQuLVg90H21EOLACvtfcHk0ljNYP8kX90lHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsbc%253D&md5=83623863b372993bdb0171d93e1db205</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1021%2Fjm3017706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3017706%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKaiser%26aufirst%3DC.%2BE.%26aulast%3DFrett%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DH.%2BY.%26atitle%3DTargeting%2520mutant%2520kras%2520for%2520anticancer%2520therapeutics%253A%2520A%2520review%2520of%2520novel%2520small%2520molecule%2520modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5219%26epage%3D30%26doi%3D10.1021%2Fjm3017706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref157"><div class="reference"><strong class="refLabel"><a href="#ref157" class="rightTabRefNumLink">157</a></strong><div class="NLM_citation" id="rightTab-cit157"><span><span class="NLM_contrib-group">Feig, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, G. M.</span><span> </span><span class="NLM_article-title">Inhibition of NIH-3T3 cell-proliferation by a mutant Ras protein with preferential affinity for GDP</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3235</span><span class="NLM_x">–</span> <span class="NLM_lpage">43</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1128%2FMCB.8.8.3235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=3145408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADyaL1cXlsVeks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1988&pages=3235-43&author=L.+A.+Feigauthor=G.+M.+Cooper&title=Inhibition+of+NIH-3T3+cell-proliferation+by+a+mutant+Ras+protein+with+preferential+affinity+for+GDP"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of NIH 3T3 cell proliferation by a mutant ras protein with preferential affinity for GDP</span></div><div class="casAuthors">Feig, Larry A.; Cooper, Geoffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3235-43</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">Substitution of asparagine for serine at position 17 decreased the affinity of rasHp21 for GTP 20- to 40-fold without significantly affecting its affinity for GDP.  Transfection of NIH 3T3 cells with a mammalian expression vector contg. the Asn-17 rasHgene and a Neor gene under the control of the same promoter yielded only a small fraction of the expected no. of G418-resistant colonies, indicating that expression of Asn-17 p21 inhibited cell proliferation.  The inhibitory effect of Asn-17 p21 required its localization to the plasma membrane and was reversed by coexpression of an activated ras gene, indicating that the mutant p21 blocked the endogenous ras function required for NIH 3T3 cell proliferation.  NIH 3T3 cells transformed by v-mos and v-raf, but not v-src, were resistant to inhibition by Asn-17 p21, indicating that the requirement for normal ras function can be bypassed by these cytoplasmic oncogenes.  The Asn-17 mutant represents a novel reagent for the study of ras function by virtue of its ability to inhibit cellular ras activity in vivo.  Since this phenotype is likely assocd. with the preferential affinity of the mutant protein for GDP, analogous mutations might also yield inhibitors of other proteins whose activities are regulated by guanine nucleotide binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM46qhA-D7prVg90H21EOLACvtfcHk0ljNYP8kX90lHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXlsVeks74%253D&md5=4f60717d2c8b090b19a65a5fd301b038</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1128%2FMCB.8.8.3235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.8.8.3235%26sid%3Dliteratum%253Aachs%26aulast%3DFeig%26aufirst%3DL.%2BA.%26aulast%3DCooper%26aufirst%3DG.%2BM.%26atitle%3DInhibition%2520of%2520NIH-3T3%2520cell-proliferation%2520by%2520a%2520mutant%2520Ras%2520protein%2520with%2520preferential%2520affinity%2520for%2520GDP%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1988%26volume%3D8%26spage%3D3235%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref158"><div class="reference"><strong class="refLabel"><a href="#ref158" class="rightTabRefNumLink">158</a></strong><div class="NLM_citation" id="rightTab-cit158"><span><span class="NLM_contrib-group">Maurer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrenton, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitts, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skelton, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fauber, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokoe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludlam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starovasnik, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, G.</span><span> </span><span class="NLM_article-title">Small-molecule ligands bind to a distinct pocket in Ras and inhibit SOS-mediated nucleotide exchange activity</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5299</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span><span class="refDoi"> DOI: 10.1073/pnas.1116510109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1073%2Fpnas.1116510109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22431598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5299-304&author=T.+Maurerauthor=L.+S.+Garrentonauthor=A.+Ohauthor=K.+Pittsauthor=D.+J.+Andersonauthor=N.+J.+Skeltonauthor=B.+P.+Fauberauthor=B.+Panauthor=S.+Malekauthor=D.+Stokoeauthor=M.+J.+Ludlamauthor=K.+K.+Bowmanauthor=J.+Wuauthor=A.+M.+Giannettiauthor=M.+A.+Starovasnikauthor=I.+Mellmanauthor=P.+K.+Jacksonauthor=J.+Rudolphauthor=W.+Wangauthor=G.+Fang&title=Small-molecule+ligands+bind+to+a+distinct+pocket+in+Ras+and+inhibit+SOS-mediated+nucleotide+exchange+activity&doi=10.1073%2Fpnas.1116510109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule ligands bind to a distinct pocket in Ras and inhibit SOS-mediated nucleotide exchange activity</span></div><div class="casAuthors">Maurer, Till; Garrenton, Lindsay S.; Oh, Angela; Pitts, Keith; Anderson, Daniel J.; Skelton, Nicholas J.; Fauber, Benjamin P.; Pan, Borlan; Malek, Shiva; Stokoe, David; Ludlam, Mary J. C.; Bowman, Krista K.; Wu, Jiansheng; Giannetti, Anthony M.; Starovasnik, Melissa A.; Mellman, Ira; Jackson, Peter K.; Rudolph, Joachim; Wang, Weiru; Fang, Guowei</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5299-5304, S5299/1-S5299/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Ras gene is frequently mutated in cancer, and mutant Ras drives tumorigenesis.  Although Ras is a central oncogene, small mols. that bind to Ras in a well-defined manner and exert inhibitory effects have not been uncovered to date.  Through an NMR-based fragment screen, we identified a group of small mols. that all bind to a common site on Ras.  High-resoln. cocrystal structures delineated a unique ligand-binding pocket on the Ras protein that is adjacent to the switch I/II regions and can be expanded upon compd. binding.  Structure anal. predicts that compd.-binding interferes with the Ras/SOS interactions.  Indeed, selected compds. inhibit SOS-mediated nucleotide exchange and prevent Ras activation by blocking the formation of intermediates of the exchange reaction.  The discovery of a small-mol. binding pocket on Ras with functional significance provides a new direction in the search of therapeutically effective inhibitors of the Ras oncoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAXIzMe7yBqrVg90H21EOLACvtfcHk0ljzCWdY93Lmvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOjsrc%253D&md5=a7210cad45154c42eb2a1faeabf88c20</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1116510109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1116510109%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DGarrenton%26aufirst%3DL.%2BS.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DPitts%26aufirst%3DK.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DLudlam%26aufirst%3DM.%2BJ.%26aulast%3DBowman%26aufirst%3DK.%2BK.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DStarovasnik%26aufirst%3DM.%2BA.%26aulast%3DMellman%26aufirst%3DI.%26aulast%3DJackson%26aufirst%3DP.%2BK.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DG.%26atitle%3DSmall-molecule%2520ligands%2520bind%2520to%2520a%2520distinct%2520pocket%2520in%2520Ras%2520and%2520inhibit%2520SOS-mediated%2520nucleotide%2520exchange%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5299%26epage%3D304%26doi%3D10.1073%2Fpnas.1116510109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref159"><div class="reference"><strong class="refLabel"><a href="#ref159" class="rightTabRefNumLink">159</a></strong><div class="NLM_citation" id="rightTab-cit159"><span><span class="NLM_contrib-group">Wang, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span> </span><span class="NLM_article-title">Ras inhibition via direct Ras binding-is there a path forward?</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">5766</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.07.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.bmcl.2012.07.082" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=5766-76&author=W.+R.+Wangauthor=G.+W.+Fangauthor=J.+Rudolph&title=Ras+inhibition+via+direct+Ras+binding-is+there+a+path+forward%3F&doi=10.1016%2Fj.bmcl.2012.07.082"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.07.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.07.082%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%2BR.%26aulast%3DFang%26aufirst%3DG.%2BW.%26aulast%3DRudolph%26aufirst%3DJ.%26atitle%3DRas%2520inhibition%2520via%2520direct%2520Ras%2520binding-is%2520there%2520a%2520path%2520forward%253F%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D5766%26epage%3D76%26doi%3D10.1016%2Fj.bmcl.2012.07.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref160"><div class="reference"><strong class="refLabel"><a href="#ref160" class="rightTabRefNumLink">160</a></strong><div class="NLM_citation" id="rightTab-cit160"><span><span class="NLM_contrib-group">Ostrem, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sos, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">503</span><span class="NLM_x">, </span> <span class="NLM_fpage">548</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-51&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+inhibitors+allosterically+control+GTP+affinity+and+effector+interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0ljzCWdY93Lmvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520inhibitors%2520allosterically%2520control%2520GTP%2520affinity%2520and%2520effector%2520interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D548%26epage%3D51%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref161"><div class="reference"><strong class="refLabel"><a href="#ref161" class="rightTabRefNumLink">161</a></strong><div class="NLM_citation" id="rightTab-cit161"><span><span class="NLM_contrib-group">Christ, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debyser, Z.</span><span> </span><span class="NLM_article-title">The LEDGF/p75 integrase interaction, a novel target for anti-HIV therapy</span> <span class="citation_source-journal">Virology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">435</span><span class="NLM_x">, </span> <span class="NLM_fpage">102</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1016/j.virol.2012.09.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.virol.2012.09.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=23217620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaju7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2013&pages=102-9&author=F.+Christauthor=Z.+Debyser&title=The+LEDGF%2Fp75+integrase+interaction%2C+a+novel+target+for+anti-HIV+therapy&doi=10.1016%2Fj.virol.2012.09.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">The LEDGF/p75 integrase interaction, a novel target for anti-HIV therapy</span></div><div class="casAuthors">Christ, Frauke; Debyser, Zeger</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-109</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  To accomplish their viral life cycle, lentiviruses such as HIV highjack host proteins, the so-called cellular co-factors of replication.  Lens Epithelium-derived Growth factor (LEDGF/p75), a transcriptional co-activator, is a co-factor of HIV-integrase (IN) and is required for the tethering and correct integration of the viral genome into the host chromatin.  Due to its important role in HIV-replication the LEDGF/p75-IN interaction is an attractive antiviral novel target for the treatment of HIV/AIDS.  Intensive drug discovery efforts over the past years have validated the LEDGF/p75-IN interaction as a drugable target for antiviral therapy and have resulted in the design and synthesis of LEDGINs, small mol. inhibitors binding to the dimer interface of HIV-integrase and inhibiting viral replication with a dual mechanism of action: potent inhibition of the LEDGF/p75-IN protein-protein interaction and allosteric inhibition of the catalytic function.  Furthermore they inhibit both early and late steps of the replication cycle which increases their potential for further clin. development.  In this review the authors will highlight the research validating the LEDGF/p75-IN interaction as a target for anti-HIV drug discovery and the recent advances in the design and development of LEDGINs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_mTW5TAo2y7Vg90H21EOLACvtfcHk0ljzCWdY93Lmvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaju7nN&md5=2249c51a1792bc8edc5ce449500e6627</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2012.09.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2012.09.033%26sid%3Dliteratum%253Aachs%26aulast%3DChrist%26aufirst%3DF.%26aulast%3DDebyser%26aufirst%3DZ.%26atitle%3DThe%2520LEDGF%252Fp75%2520integrase%2520interaction%252C%2520a%2520novel%2520target%2520for%2520anti-HIV%2520therapy%26jtitle%3DVirology%26date%3D2013%26volume%3D435%26spage%3D102%26epage%3D9%26doi%3D10.1016%2Fj.virol.2012.09.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref162"><div class="reference"><strong class="refLabel"><a href="#ref162" class="rightTabRefNumLink">162</a></strong><div class="NLM_citation" id="rightTab-cit162"><span><span class="NLM_contrib-group">Malet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcelin, A. G.</span><span> </span><span class="NLM_article-title">The future of integrase inhibitors of HIV-1</span> <span class="citation_source-journal">Curr. Opin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.coviro.2012.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1016%2Fj.coviro.2012.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=22980926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSltrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=580-7&author=I.+Maletauthor=V.+Calvezauthor=A.+G.+Marcelin&title=The+future+of+integrase+inhibitors+of+HIV-1&doi=10.1016%2Fj.coviro.2012.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">The future of integrase inhibitors of HIV-1</span></div><div class="casAuthors">Malet, Isabelle; Calvez, Vincent; Marcelin, Anne-Genevieve</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">580-587</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Integration of the HIV-1 DNA is required and essential to maintain the viral DNA in the infected cell.  Integration process occurs in several events, mainly endonucleolytic processing of the 3' ends of the viral DNA and strand transfer or joining of the viral and cellular DNA.  The design and discovery of integrase inhibitors were first focused at targeting the catalytic site of IN with a specific effect on strand transfer.  Several integrase inhibitors were developed clin., two first generation inhibitors, raltegravir and elvitegravir and then two second-generation inhibitors, dolutegravir and MK-2058.  Recently, allosteric integrase inhibitors intended to interfere with the integrase-LEDGF/p75 interaction have been designed.  These new inhibitors called LEDGINs have an effect on 3' processing and strand transfer.  Thus, integrase inhibitors present a real added value in combined treatment for naive and experienced HIV infected patients.  Combination expts. of LEDGINs and raltegravir suggest that these inhibitors could act additively despite sharing the same viral target.  Future therapy could involve combinations of inhibitors of IN function acting though different binding pockets within IN.  The place of this class on HIV inhibitors and their future role in perspective of novel therapies to eliminate latent HIV reservoirs and infection for cure should also be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvFDf9hYvRAbVg90H21EOLACvtfcHk0lj5sOUgsdVzsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSltrrM&md5=ea4826081ba3b0dfc3948bc45ae9badc</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2012.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2012.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DMalet%26aufirst%3DI.%26aulast%3DCalvez%26aufirst%3DV.%26aulast%3DMarcelin%26aufirst%3DA.%2BG.%26atitle%3DThe%2520future%2520of%2520integrase%2520inhibitors%2520of%2520HIV-1%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2012%26volume%3D2%26spage%3D580%26epage%3D7%26doi%3D10.1016%2Fj.coviro.2012.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref163"><div class="reference"><strong class="refLabel"><a href="#ref163" class="rightTabRefNumLink">163</a></strong><div class="NLM_citation" id="rightTab-cit163"><span><span class="NLM_contrib-group">Demeulemeester, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaltin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Maeyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debyser, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christ, F.</span><span> </span><span class="NLM_article-title">LEDGINs, non-catalytic site inhibitors of HIV-1 integrase: A patent review (2006 - 2014)</span> <span class="citation_source-journal">Expert Opin. Ther. Pat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1517/13543776.2014.898753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=10.1517%2F13543776.2014.898753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=24666332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotF2rs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=609-32&author=J.+Demeulemeesterauthor=P.+Chaltinauthor=A.+Marchandauthor=M.+De+Maeyerauthor=Z.+Debyserauthor=F.+Christ&title=LEDGINs%2C+non-catalytic+site+inhibitors+of+HIV-1+integrase%3A+A+patent+review+%282006+-+2014%29&doi=10.1517%2F13543776.2014.898753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">LEDGINs, non-catalytic site inhibitors of HIV-1 integrase: a patent review (2006 - 2014)</span></div><div class="casAuthors">Demeulemeester, Jonas; Chaltin, Patrick; Marchand, Arnaud; De Maeyer, Marc; Debyser, Zeger; Christ, Frauke</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">609-632</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Integration of the viral genome into the host cell chromatin is a central step in the replication cycle of the HIV.  Blocking the viral integrase (IN) enzyme therefore provides an attractive therapeutic strategy, as evidenced by the recent clin. approval of three IN strand transfer inhibitors.  Viral resistance and cross-resistance among these inhibitors, however, warrant the search for compds. targeting HIV integration through alternative mechanisms of action.  The most potent class of allosteric IN inhibitors was independently identified at the University of Leuven, Belgium, and at Boehringer Ingelheim, Canada.  These compds., coined LEDGINs (after the lens epithelium-derived growth factor/p75 cofactor binding pocket on IN) or non-catalytic site IN inhibitors (NCINIs) by the resp. groups, have shown remarkable antiviral activity.  This review provides a brief introduction to the compd. class and discusses the recent patent literature (2006 to the present).  Expert opinion: LEDGINs are still early in development.  Trials with clin. candidate BI-224436 were put on hold despite promising results.  Literature, however, reveals that almost all major pharmaceutical companies active in the treatment of HIV/AIDS have taken a significant interest in this class.  As a result, several of these inhibitors may soon enter clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowB3WZSin3PLVg90H21EOLACvtfcHk0lj5sOUgsdVzsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotF2rs70%253D&md5=cf3625cadfaf5ea12b5dc118befdb18d</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.898753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.898753%26sid%3Dliteratum%253Aachs%26aulast%3DDemeulemeester%26aufirst%3DJ.%26aulast%3DChaltin%26aufirst%3DP.%26aulast%3DMarchand%26aufirst%3DA.%26aulast%3DDe%2BMaeyer%26aufirst%3DM.%26aulast%3DDebyser%26aufirst%3DZ.%26aulast%3DChrist%26aufirst%3DF.%26atitle%3DLEDGINs%252C%2520non-catalytic%2520site%2520inhibitors%2520of%2520HIV-1%2520integrase%253A%2520A%2520patent%2520review%2520%25282006%2520-%25202014%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2014%26volume%3D24%26spage%3D609%26epage%3D32%26doi%3D10.1517%2F13543776.2014.898753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ppn','PDB','2ppn'); return false;">PDB: 2ppn</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkj','PDB','1fkj'); return false;">PDB: 1fkj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkh" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fkh','PDB','1fkh'); return false;">PDB: 1fkh</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f40" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f40','PDB','1f40'); return false;">PDB: 1f40</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bx4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bx4','PDB','1bx4'); return false;">PDB: 1bx4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2i6b" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2i6b','PDB','2i6b'); return false;">PDB: 2i6b</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2a6b" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2a6b','PDB','2a6b'); return false;">PDB: 2a6b</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1h10" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1h10','PDB','1h10'); return false;">PDB: 1h10</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1unr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1unr','PDB','1unr'); return false;">PDB: 1unr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1r2d" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1r2d','PDB','1r2d'); return false;">PDB: 1r2d</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2yxj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2yxj','PDB','2yxj'); return false;">PDB: 2yxj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g3f" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g3f','PDB','1g3f'); return false;">PDB: 1g3f</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2opz" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2opz','PDB','2opz'); return false;">PDB: 2opz</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cke" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cke','PDB','1cke'); return false;">PDB: 1cke</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cmk" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cmk','PDB','2cmk'); return false;">PDB: 2cmk</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dhm" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dhm','PDB','1dhm'); return false;">PDB: 1dhm</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jj4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jj4','PDB','1jj4'); return false;">PDB: 1jj4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qam" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qam','PDB','1qam'); return false;">PDB: 1qam</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qan" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qan','PDB','1qan'); return false;">PDB: 1qan</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1j4w" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1j4w','PDB','1j4w'); return false;">PDB: 1j4w</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fl9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fl9','PDB','1fl9'); return false;">PDB: 1fl9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1htw" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1htw','PDB','1htw'); return false;">PDB: 1htw</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lkl" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lkl','PDB','1lkl'); return false;">PDB: 1lkl</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3hck" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3hck','PDB','3hck'); return false;">PDB: 3hck</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rd4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rd4','PDB','1rd4'); return false;">PDB: 1rd4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lfa" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lfa','PDB','1lfa'); return false;">PDB: 1lfa</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ycr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ycr','PDB','1ycr'); return false;">PDB: 1ycr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1z1m" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1z1m','PDB','1z1m'); return false;">PDB: 1z1m</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rv1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rv1','PDB','1rv1'); return false;">PDB: 1rv1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1uae" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1uae','PDB','1uae'); return false;">PDB: 1uae</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ejd" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ejd','PDB','1ejd'); return false;">PDB: 1ejd</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b73" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b73','PDB','1b73'); return false;">PDB: 1b73</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b74" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b74','PDB','1b74'); return false;">PDB: 1b74</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8h" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8h','PDB','1i8h'); return false;">PDB: 1i8h</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i6c" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i6c','PDB','1i6c'); return false;">PDB: 1i6c</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8g" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i8g','PDB','1i8g'); return false;">PDB: 1i8g</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu0','PDB','1iu0'); return false;">PDB: 1iu0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1iu2','PDB','1iu2'); return false;">PDB: 1iu2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rgr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rgr','PDB','1rgr'); return false;">PDB: 1rgr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ph0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ph0','PDB','1ph0'); return false;">PDB: 1ph0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a5j','PDB','3a5j'); return false;">PDB: 3a5j</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pty','PDB','1pty'); return false;">PDB: 1pty</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ssk" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ssk','PDB','1ssk'); return false;">PDB: 1ssk</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g4k" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1g4k','PDB','1g4k'); return false;">PDB: 1g4k</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cqr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cqr','PDB','1cqr'); return false;">PDB: 1cqr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e31','PDB','1e31'); return false;">PDB: 1e31</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uih" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uih','PDB','3uih'); return false;">PDB: 3uih</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fv9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fv9','PDB','1fv9'); return false;">PDB: 1fv9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2o8t" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2o8t','PDB','2o8t'); return false;">PDB: 2o8t</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ih8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ih8','PDB','1ih8'); return false;">PDB: 1ih8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ccg" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ccg','PDB','1ccg'); return false;">PDB: 1ccg</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ezq" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ezq','PDB','1ezq'); return false;">PDB: 1ezq</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cpl" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cpl','PDB','2cpl'); return false;">PDB: 2cpl</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cwa" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cwa','PDB','1cwa'); return false;">PDB: 1cwa</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1sfa" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1sfa','PDB','1sfa'); return false;">PDB: 1sfa</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2tke" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2tke','PDB','2tke'); return false;">PDB: 2tke</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2dg3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2dg3','PDB','2dg3'); return false;">PDB: 2dg3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lvy" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lvy','PDB','1lvy'); return false;">PDB: 1lvy</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1okx" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1okx','PDB','1okx'); return false;">PDB: 1okx</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1efc" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1efc','PDB','1efc'); return false;">PDB: 1efc</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dt8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dt8','PDB','1dt8'); return false;">PDB: 1dt8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3egg" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3egg','PDB','3egg'); return false;">PDB: 3egg</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3e7a" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3e7a','PDB','3e7a'); return false;">PDB: 3e7a</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kn9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kn9','PDB','1kn9'); return false;">PDB: 1kn9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1t7d" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1t7d','PDB','1t7d'); return false;">PDB: 1t7d</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1guv" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1guv','PDB','1guv'); return false;">PDB: 1guv</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1waw" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1waw','PDB','1waw'); return false;">PDB: 1waw</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1wb0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1wb0','PDB','1wb0'); return false;">PDB: 1wb0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1nwk" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1nwk','PDB','1nwk'); return false;">PDB: 1nwk</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qz5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1qz5','PDB','1qz5'); return false;">PDB: 1qz5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2q0u" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2q0u','PDB','2q0u'); return false;">PDB: 2q0u</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2q0r" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2q0r','PDB','2q0r'); return false;">PDB: 2q0r</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3glk" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3glk','PDB','3glk'); return false;">PDB: 3glk</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gid" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gid','PDB','3gid'); return false;">PDB: 3gid</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yer" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yer','PDB','1yer'); return false;">PDB: 1yer</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yet" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yet','PDB','1yet'); return false;">PDB: 1yet</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3inw" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3inw','PDB','3inw'); return false;">PDB: 3inw</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yes" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yes','PDB','1yes'); return false;">PDB: 1yes</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f46" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f46','PDB','1f46'); return false;">PDB: 1f46</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1s1s" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1s1s','PDB','1s1s'); return false;">PDB: 1s1s</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1de9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1de9','PDB','1de9'); return false;">PDB: 1de9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1q6t" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1q6t','PDB','1q6t'); return false;">PDB: 1q6t</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f47" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f47','PDB','1f47'); return false;">PDB: 1f47</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bix" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bix','PDB','1bix'); return false;">PDB: 1bix</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e9n" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e9n','PDB','1e9n'); return false;">PDB: 1e9n</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dew" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1dew','PDB','1dew'); return false;">PDB: 1dew</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pa1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pa1','PDB','1pa1'); return false;">PDB: 1pa1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2veu" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2veu','PDB','2veu'); return false;">PDB: 2veu</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4i8n" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4i8n','PDB','4i8n'); return false;">PDB: 4i8n</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vew" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vew','PDB','2vew'); return false;">PDB: 2vew</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2fjn" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2fjn','PDB','2fjn'); return false;">PDB: 2fjn</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2qbp" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2qbp','PDB','2qbp'); return false;">PDB: 2qbp</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pxh" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pxh','PDB','1pxh'); return false;">PDB: 1pxh</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yz5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yz5','PDB','1yz5'); return false;">PDB: 1yz5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p1n" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p1n','PDB','3p1n'); return false;">PDB: 3p1n</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dhu" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dhu','PDB','4dhu'); return false;">PDB: 4dhu</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gft" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gft','PDB','3gft'); return false;">PDB: 3gft</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2pmx" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2pmx','PDB','2pmx'); return false;">PDB: 2pmx</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epr','PDB','4epr'); return false;">PDB: 4epr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dst" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dst','PDB','4dst'); return false;">PDB: 4dst</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epw" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epw','PDB','4epw'); return false;">PDB: 4epw</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epy" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4epy','PDB','4epy'); return false;">PDB: 4epy</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bi4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bi4','PDB','1bi4'); return false;">PDB: 1bi4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3avn" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3avn','PDB','3avn'); return false;">PDB: 3avn</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2b4j" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2b4j','PDB','2b4j'); return false;">PDB: 2b4j</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3lpu" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3lpu','PDB','3lpu'); return false;">PDB: 3lpu</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jdd" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jdd','PDB','4jdd'); return false;">PDB: 4jdd</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dso" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dso','PDB','4dso'); return false;">PDB: 4dso</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4lv6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4lv6','PDB','4lv6'); return false;">PDB: 4lv6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4q21" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4q21','PDB','4q21'); return false;">PDB: 4q21</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.5b00586">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_71047"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b00586">10.1021/acs.jmedchem.5b00586</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional druggability results and histograms (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf">PDF</a>).</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b00586/suppl_file/jm5b00586_si_001.pdf">jm5b00586_si_001.pdf (821.81 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b00586%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-23%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b00586" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a80a38e90d19b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
