
<html lang="en"     class="pb-page"  data-request-id="9b26ba88-d4fb-462b-845a-f00c53ef215c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2021.64.issue-9;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.1c00369;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="6-Halopyridylmethylidene Penicillin-Based Sulfones Efficiently Inactivate the Natural Resistance of Pseudomonas aeruginosa to β-Lactam Antibiotics" /></meta><meta name="dc.Creator" content="Juan C.  Vázquez-Ucha" /></meta><meta name="dc.Creator" content="Diana  Rodríguez" /></meta><meta name="dc.Creator" content="Cristina  Lasarte-Monterrubio" /></meta><meta name="dc.Creator" content="Emilio  Lence" /></meta><meta name="dc.Creator" content="Jorge  Arca-Suarez" /></meta><meta name="dc.Creator" content="María  Maneiro" /></meta><meta name="dc.Creator" content="Eva  Gato" /></meta><meta name="dc.Creator" content="Astrid  Perez" /></meta><meta name="dc.Creator" content="Marta  Martínez-Guitián" /></meta><meta name="dc.Creator" content="Carlos  Juan" /></meta><meta name="dc.Creator" content="Antonio  Oliver" /></meta><meta name="dc.Creator" content="German  Bou" /></meta><meta name="dc.Creator" content="Concepción  González-Bello" /></meta><meta name="dc.Creator" content="Alejandro  Beceiro" /></meta><meta name="dc.Description" content="Pseudomonas aeruginosa, a major cause of nosocomial infections, is considered a paradigm of antimicrobial resistance, largely due to hyperproduction of chromosomal cephalosporinase AmpC. Here, we e..." /></meta><meta name="Description" content="Pseudomonas aeruginosa, a major cause of nosocomial infections, is considered a paradigm of antimicrobial resistance, largely due to hyperproduction of chromosomal cephalosporinase AmpC. Here, we e..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 29, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00369" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00369" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00369" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00369" /></link>
        
    
    

<title>6-Halopyridylmethylidene Penicillin-Based Sulfones Efficiently Inactivate the Natural Resistance of Pseudomonas aeruginosa to β-Lactam Antibiotics | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00369" /></meta><meta property="og:title" content="6-Halopyridylmethylidene Penicillin-Based Sulfones Efficiently Inactivate the Natural Resistance of Pseudomonas aeruginosa to β-Lactam Antibiotics" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0012.jpeg" /></meta><meta property="og:description" content="Pseudomonas aeruginosa, a major cause of nosocomial infections, is considered a paradigm of antimicrobial resistance, largely due to hyperproduction of chromosomal cephalosporinase AmpC. Here, we explore the ability of 6-pyridylmethylidene penicillin-based sulfones 1–3 to inactivate the AmpC β-lactamase and thus rescue the activity of the antipseudomonal ceftazidime. These compounds increased the susceptibility to ceftazidime in a collection of clinical isolates and PAO1 mutant strains with different ampC expression levels and also improved the inhibition kinetics relative to avibactam, displaying a slow deacylation rate and involving the formation of an indolizine adduct. Bromide 2 was the inhibitor with the lowest KI (15.6 nM) and the highest inhibitory efficiency (kinact/KI). Computational studies using diverse AmpC enzymes revealed that the aromatic moiety in 1–3 targets a tunnel-like site adjacent to the catalytic serine and induces the folding of the H10 helix, indicating the potential value of this not-always-evident pocket in drug design." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00369"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00369">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00369&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00369&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00369&amp;href=/doi/10.1021/acs.jmedchem.1c00369" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 6310-6328</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00350" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00375" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">6-Halopyridylmethylidene Penicillin-Based Sulfones Efficiently Inactivate the Natural Resistance of <i>Pseudomonas aeruginosa</i> to β-Lactam Antibiotics</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Juan C. Vázquez-Ucha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Juan C. Vázquez-Ucha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología do Complejo Hospitalario Universitario da Coruña (CHUAC), Instituto de Investigación Biomédica da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Juan+C.++V%C3%A1zquez-Ucha">Juan C. Vázquez-Ucha</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4949-0779" title="Orcid link">http://orcid.org/0000-0003-4949-0779</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Diana Rodríguez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Diana Rodríguez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro Singular de Investigación en Química Biolóxica e Materiais Moleculares (CiQUS), Departamento de Química Orgánica, Universidade de Santiago de Compostela, Jenaro de la Fuente s/n, Santiago de Compostela 15782, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Diana++Rodr%C3%ADguez">Diana Rodríguez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cristina Lasarte-Monterrubio</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cristina Lasarte-Monterrubio</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología do Complejo Hospitalario Universitario da Coruña (CHUAC), Instituto de Investigación Biomédica da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cristina++Lasarte-Monterrubio">Cristina Lasarte-Monterrubio</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Emilio Lence</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emilio Lence</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro Singular de Investigación en Química Biolóxica e Materiais Moleculares (CiQUS), Departamento de Química Orgánica, Universidade de Santiago de Compostela, Jenaro de la Fuente s/n, Santiago de Compostela 15782, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emilio++Lence">Emilio Lence</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jorge Arca-Suarez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jorge Arca-Suarez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología do Complejo Hospitalario Universitario da Coruña (CHUAC), Instituto de Investigación Biomédica da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jorge++Arca-Suarez">Jorge Arca-Suarez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">María Maneiro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">María Maneiro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro Singular de Investigación en Química Biolóxica e Materiais Moleculares (CiQUS), Departamento de Química Orgánica, Universidade de Santiago de Compostela, Jenaro de la Fuente s/n, Santiago de Compostela 15782, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mar%C3%ADa++Maneiro">María Maneiro</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Eva Gato</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eva Gato</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología do Complejo Hospitalario Universitario da Coruña (CHUAC), Instituto de Investigación Biomédica da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eva++Gato">Eva Gato</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Astrid Perez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Astrid Perez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología do Complejo Hospitalario Universitario da Coruña (CHUAC), Instituto de Investigación Biomédica da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Astrid++Perez">Astrid Perez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marta Martínez-Guitián</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marta Martínez-Guitián</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología do Complejo Hospitalario Universitario da Coruña (CHUAC), Instituto de Investigación Biomédica da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marta++Mart%C3%ADnez-Guiti%C3%A1n">Marta Martínez-Guitián</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carlos Juan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carlos Juan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología y Unidad de Investigación, Hospital Universitario Son Espases, Instituto de Investigación Sanitaria Illes Balears (IdiSBA), Carretera de Valldemossa, 79, Palma de Mallorca 07120, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carlos++Juan">Carlos Juan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Antonio Oliver</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Antonio Oliver</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología y Unidad de Investigación, Hospital Universitario Son Espases, Instituto de Investigación Sanitaria Illes Balears (IdiSBA), Carretera de Valldemossa, 79, Palma de Mallorca 07120, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Antonio++Oliver">Antonio Oliver</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">German Bou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">German Bou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología do Complejo Hospitalario Universitario da Coruña (CHUAC), Instituto de Investigación Biomédica da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=German++Bou">German Bou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Concepción González-Bello</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Concepción González-Bello</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro Singular de Investigación en Química Biolóxica e Materiais Moleculares (CiQUS), Departamento de Química Orgánica, Universidade de Santiago de Compostela, Jenaro de la Fuente s/n, Santiago de Compostela 15782, Spain</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#7b181415181e0b18121415551c1415011a171e0155191e1717143b0e0818551e08"><span class="__cf_email__" data-cfemail="0c6f63626f697c6f656362226b6362766d606976226e696060634c797f6f22697f">[email protected]</span></a>. Phone: (+34) 881815726.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Concepci%C3%B3n++Gonz%C3%A1lez-Bello">Concepción González-Bello</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6439-553X" title="Orcid link">http://orcid.org/0000-0001-6439-553X</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Alejandro Beceiro</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alejandro Beceiro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servicio de Microbiología do Complejo Hospitalario Universitario da Coruña (CHUAC), Instituto de Investigación Biomédica da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#73321f1619121d17011c5d311610161a011c5d3012001200330016011412005d1600"><span class="__cf_email__" data-cfemail="f3b29f9699929d97819cddb19690969a819cddb092809280b3809681949280dd9680">[email protected]</span></a>. Phone: +34 981178000 ext. 292145.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alejandro++Beceiro">Alejandro Beceiro</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6340-7815" title="Orcid link">http://orcid.org/0000-0002-6340-7815</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00369&amp;href=/doi/10.1021%2Facs.jmedchem.1c00369" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 6310–6328</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 29, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 March 2021</li><li><span class="item_label"><b>Published</b> online</span>29 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00369" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00369</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6310%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJuan%2BC.%2BV%25C3%25A1zquez-Ucha%252C%2BDiana%2BRodr%25C3%25ADguez%252C%2BCristina%2BLasarte-Monterrubio%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D9%26contentID%3Dacs.jmedchem.1c00369%26title%3D6-Halopyridylmethylidene%2BPenicillin-Based%2BSulfones%2BEfficiently%2BInactivate%2Bthe%2BNatural%2BResistance%2Bof%2BPseudomonas%2Baeruginosa%2Bto%2B%25CE%25B2-Lactam%2BAntibiotics%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6328%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00369"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">620</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00369" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;6-Halopyridylmethylidene Penicillin-Based Sulfones Efficiently Inactivate the Natural Resistance of Pseudomonas aeruginosa to β-Lactam Antibiotics&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Juan&quot;,&quot;last_name&quot;:&quot;C. Vázquez-Ucha&quot;},{&quot;first_name&quot;:&quot;Diana&quot;,&quot;last_name&quot;:&quot;Rodríguez&quot;},{&quot;first_name&quot;:&quot;Cristina&quot;,&quot;last_name&quot;:&quot;Lasarte-Monterrubio&quot;},{&quot;first_name&quot;:&quot;Emilio&quot;,&quot;last_name&quot;:&quot;Lence&quot;},{&quot;first_name&quot;:&quot;Jorge&quot;,&quot;last_name&quot;:&quot;Arca-Suarez&quot;},{&quot;first_name&quot;:&quot;María&quot;,&quot;last_name&quot;:&quot;Maneiro&quot;},{&quot;first_name&quot;:&quot;Eva&quot;,&quot;last_name&quot;:&quot;Gato&quot;},{&quot;first_name&quot;:&quot;Astrid&quot;,&quot;last_name&quot;:&quot;Perez&quot;},{&quot;first_name&quot;:&quot;Marta&quot;,&quot;last_name&quot;:&quot;Martínez-Guitián&quot;},{&quot;first_name&quot;:&quot;Carlos&quot;,&quot;last_name&quot;:&quot;Juan&quot;},{&quot;first_name&quot;:&quot;Antonio&quot;,&quot;last_name&quot;:&quot;Oliver&quot;},{&quot;first_name&quot;:&quot;German&quot;,&quot;last_name&quot;:&quot;Bou&quot;},{&quot;first_name&quot;:&quot;Concepción&quot;,&quot;last_name&quot;:&quot;González-Bello&quot;},{&quot;first_name&quot;:&quot;Alejandro&quot;,&quot;last_name&quot;:&quot;Beceiro&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;29&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;6310-6328&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00369&quot;},&quot;abstract&quot;:&quot;Pseudomonas aeruginosa, a major cause of nosocomial infections, is considered a paradigm of antimicrobial resistance, largely due to hyperproduction of chromosomal cephalosporinase AmpC. Here, we explore the ability of 6-pyridylmethylidene penicillin-based sulfones 1–3 to inactivate the AmpC β-lactamase and thus rescue the activity of the antipseudomonal ceftazidime. These compounds increased the susceptibility to ceftazidime in a collection of clinical isolates and PAO1 mutant strains with different ampC expression levels and also improved the inhibition kinetics relative to avibactam, displaying a slow deacylation rate and involving the formation of an indolizine adduct. Bromide 2 was the inhibitor with the lowest KI (15.6 nM) and the highest inhibitory efficiency (kinact/KI). Computational studies using diverse AmpC enzymes revealed that the aromatic moiety in 1–3 targets a tunnel-like site adjacent to the catalytic serine and induces the folding of the H10 helix, indicating the potential value of this not&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00369&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00369" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00369&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00369" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00369&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00369" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00369&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00369&amp;href=/doi/10.1021/acs.jmedchem.1c00369" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00369" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00369" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (14 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00369%26sid%3Dliteratum%253Aachs%26pmid%3D33913328%26genre%3Darticle%26aulast%3DV%25C3%25A1zquez-Ucha%26date%3D2021%26atitle%3D6-Halopyridylmethylidene%2BPenicillin-Based%2BSulfones%2BEfficiently%2BInactivate%2Bthe%2BNatural%2BResistance%2Bof%2BPseudomonas%2Baeruginosa%2Bto%2B%25CE%25B2-Lactam%2BAntibiotics%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D9%26spage%3D6310%26epage%3D6328%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (18)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292227" title="Drug resistance">Drug resistance</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/jmcmar.2021.64.issue-9/20210513/jmcmar.2021.64.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText"><i>Pseudomonas aeruginosa</i>, a major cause of nosocomial infections, is considered a paradigm of antimicrobial resistance, largely due to hyperproduction of chromosomal cephalosporinase AmpC. Here, we explore the ability of 6-pyridylmethylidene penicillin-based sulfones <b>1–3</b> to inactivate the AmpC β-lactamase and thus rescue the activity of the antipseudomonal ceftazidime. These compounds increased the susceptibility to ceftazidime in a collection of clinical isolates and PAO1 mutant strains with different <i>ampC</i> expression levels and also improved the inhibition kinetics relative to avibactam, displaying a slow deacylation rate and involving the formation of an indolizine adduct. Bromide <b>2</b> was the inhibitor with the lowest <i>K</i><sub>I</sub> (15.6 nM) and the highest inhibitory efficiency (<i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub>). Computational studies using diverse AmpC enzymes revealed that the aromatic moiety in <b>1–3</b> targets a tunnel-like site adjacent to the catalytic serine and induces the folding of the H10 helix, indicating the potential value of this not-always-evident pocket in drug design.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41123" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41123" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The treatment of infections caused by <i>Pseudomonas aeruginosa</i><i>,</i> which is one of the top three critical priority pathogens identified by the World Health Organization, is probably one of the major clinical problems in the field of infectious diseases at present. This is not only due to an intrinsically high resistance but also due to the natural propensity of the bacterium to develop multidrug resistance <i>via</i> selection of mutations in chromosomal genes or <i>via</i> the horizontal acquisition of broad spectrum resistance mechanisms.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The prevalence of multidrug-resistant (MDR) and extensively drug-resistant <i>P. aeruginosa</i> strains has increased dramatically in the last few years, highlighting the urgent need for new antipseudomonal drugs.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6">(2−6)</a></div><div class="NLM_p">In Gram-negative bacteria, the most important mechanism of resistance to β-lactam antibiotics, which represent about 70% of all antibacterial drugs in clinical use,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> is the enzymatic hydrolysis of the β-lactam ring catalyzed by β-lactamases. This machinery is particularly challenging in <i>P. aeruginosa</i> because, in addition to the plasmid-mediated resistance observed in other bacteria, this bacterium possesses a chromosomal class C β-lactamase (AmpC, encoded by <i>ampC</i> gene) known as PDC (for <i>Pseudomonas</i>-derived cephalosporinase), which can be induced in most clinical isolates by some of the β-lactam antibiotics themselves.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> This gives <i>P. aeruginosa</i> the ability to regulate the level of resistance on demand <i>via</i> production of the AmpC enzyme, thus making the bacterium intrinsically resistant to β-lactam antibiotics, including aminopenicillins and cephalosporins. Thus, when the <i>ampC</i> gene is derepressed and is constitutively producing high levels of the enzyme, it becomes the major cause of resistance to many antipseudomonal β-lactam therapies.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> Metallo-β-lactamase enzymes such as VIM and IMP are also involved in an important resistance mechanism in <i>P. aeruginosa</i>.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Other secondary and also relevant β-lactam resistance mechanisms include (i) loss of the outer membrane protein, OprD, which is involved in carbapenem resistance,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and (ii) overproduction of efflux pumps.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The combination of these chromosomal resistance mechanisms frequently leads to the emergence of clinical isolates that are resistant to virtually all β-lactam antibiotics.</div><div class="NLM_p">Ceftazidime/avibactam and ceftolozane/tazobactam are new combination therapies of β-lactam/β-lactamase inhibitors with enhanced stability against AmpC-mediated hydrolysis and efflux mechanisms, which have recently been introduced into the clinical setting to alleviate the urgent clinical need for novel antipseudomonal β-lactam antibiotics.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Nevertheless, use of these therapies is somewhat cautiously restricted, and dosing is optimized in order to prevent selection of <i>in vivo</i> resistance.<a onclick="showRef(event, 'ref2 ref15'); return false;" href="javascript:void(0);" class="ref ref2 ref15">(2,15)</a> In addition, the increasing catalogue of reports documenting the emergence of AmpC-mediated resistance during therapy is of serious concern.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> A few other alternatives are being developed and are currently at different phases of clinical trials: (i) new DBO (1,6-diazabicyclo[3.2.1]octane) inhibitors, imipenem/relebactam, and cefepime/zidebactam;<a onclick="showRef(event, 'ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22">(19−22)</a> (ii) cyclic boronate inhibitors such as meropenem/vaborbactam<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and cefepime/taniborbactam;<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> and (iii) a new tazobactam analogue, cefepime/enmetazobactam (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. β-Lactamase inhibitors that have recently been approved or are undergoing clinical trials.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In recent years, our research group has been exploring the therapeutic potential of 6-arylmethylidene penicillin-based sulfones as new β-lactamase inhibitors that could recover antibiotic efficacy (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> This research has demonstrated the efficacy of the 2-pyridylmethylidene derivative <b>1</b> (LN-1-255) against the relevant carbapenemase OXA-48<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and against all of the carbapenem-hydrolyzing class D β-lactamases of the pathogen <i>Acinetobacter baumannii</i>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> which also displayed good efficacy in preclinical models of murine pneumonia.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The key structural features responsible for the effectiveness of compound <b>1</b> relative to other penicillin-based sulfones (sulbactam, tazobactam) are (i) the catechol moiety (siderophore), which improves internalization through bacterial iron uptake pathways, and (ii) the 2-pyridylmethylene group, which can generate an indolizine adduct <b>II</b> that is resistant to hydrolysis. The latter compound would be generated by nucleophilic attack of the pyridine nitrogen on the conjugated imine adduct <b>I</b> that would result from opening the dioxothiazolidine ring (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The available crystal structures of OXA-24/40 from <i>A. baumannii</i> covalently modified by 2-pyridylmethylidene penicillin-based sulfone inhibitors (PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MBZ">3MBZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYZ">3FYZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FZC">3FZC</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FV7">3FV7</a>) suggest that the indolizine moiety would increase the hydrophobicity of the active site, thus hindering the approach of the water molecule for the final hydrolysis and turnover.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> A similar effect would be expected for OXA-48, which also has a large hydrophobic region close to the active site.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) 6-Arylmethylidene penicillin-based sulfones <b>1–7</b>. (B) Mechanism of inhibition of compounds <b>1–7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We have recently explored the effect on the inhibitory efficacy of <b>1</b> of changes in the electron density of its pyridine ring by the incorporation of diverse substituents, compounds <b>1–5</b>, and by using larger pyridine-based heterocycles, compounds <b>6–7</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The best results were obtained by reducing the electron density of the ring in <b>1</b> by the introduction of a bromide (<i>para</i>) or a fluoride (<i>meta</i>) substituent, compounds <b>2</b> and <b>3</b>, respectively. These derivatives maintained or improved the activity of the parent compound <b>1</b> against carbapenem-hydrolyzing class D β-lactamases (OXA-23, OXA-24/40, and OXA-48). Docking studies with the OXA-24/40 enzyme suggested that these electron-deficient aromatic rings enhance CH−π interactions with the apolar residues of the tunnel-like entrance surrounding these compounds. Compounds <b>2</b> and <b>3</b> also restored the antibiotic efficacy in <i>Escherichia coli</i> strain-carrying class A (TEM-1 and CTX-M-2) and class C (CMY-2 and DHA-1) β-lactamases. These promising results, together with the alarming worldwide reduction in the efficacy of antibacterial drugs against the priority pathogen <i>P. aeruginosa</i><i>,</i> encouraged us to explore the capacity of these derivatives to rescue the activity of antipseudomonal cephalosporin ceftazidime. As discussed below, preliminary susceptibility studies with the strain <i>P. aeruginosa</i> PAO1 ΔampD (hyperexpresing AmpC) and compounds <b>1–7</b> revealed the higher capacity of compounds <b>1–3</b> than of the other compounds to restore antibiotic efficacy, which were therefore selected for detailed exploration of their therapeutic potential for MDR <i>P. aeruginosa</i>. We report here microbiological studies on the susceptibility of compounds <b>1–3</b> in the presence of the antimicrobial ceftazidime with a complete collection of <i>P. aeruginosa</i> PAO1 isogenic mutant strains and clinical isolates with different levels of <i>ampC</i> expression. The results of kinetic studies of compounds <b>1–3</b> against the PDC-1 enzyme (AmpC wild-type from PAO1 strain and widely spread in <i>P. aeruginosa</i>), in combination with docking and molecular dynamics (MD) simulation studies on the enzyme complexes and their corresponding acyl-enzyme adducts, allowed us to demonstrate the key structural features of the efficacy of the compounds. Moreover, simulation studies with other AmpC enzymes (BER, CMY-10) also show that the aromatic moiety of the ligands induces folding of the H10 helix and the formation of a tunnel-like site, which is not always evident and in which the indolizine adduct would be deeply buried.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43084" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43084" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Susceptibility Studies</h3><div class="NLM_p">The ability of compounds <b>1–7</b> to increase the antipseudomonal activity of ceftazidime against the mutant <i>P. aeruginosa</i> PAO1 ΔampD, derived from the parental reference strain PAO1, was first measured. Other than AmpC (PDC-1), this strain does not possess other relevant mechanisms of resistance to β-lactam antibiotics and exhibits low levels of resistance. The minimum inhibitory concentrations (MICs) of ceftazidime were determined, in the absence and presence of compounds <b>1–7</b> at a fixed concentration of 16 mg/L, and the results were compared with avibactam as a reference β-lactamase inhibitor. The preliminary results showed that the antibiotic activity of ceftazidime can be fully restored when it is combined with compounds <b>1–3</b> and <b>6</b>, while only 50% or less activity is achieved with derivatives <b>4</b>–<b>5</b> and <b>7</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Table S1</a>). These results, together with the twofold reduction in the capacity of compound <b>6</b> to rescue the imipenem activity against bacterial strains harboring carbapenem-hydrolyzing class D β–lactamases (OXA-23, OXA-24/40, and OXA-48) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Tables S2 and S3</a>),<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> led us to select compounds <b>1–3</b> for further studies with MDR strains of <i>P. aeruginosa</i> with different capacities to produce the AmpC enzyme. To this end, isogenic mutants of <i>P. aeruginosa</i> PAO1 strain with different <i>ampC</i> expression levels (encoding the PDC β-lactamase) were used to determine the microbiological activity of the compounds <b>1–3</b> in a genetic background lacking other β-lactam resistance mechanisms. This is a well-characterized collection of <i>P. aeruginosa</i> strains expressing multiple combinations of genes implicated in <i>ampC</i> regulation and displaying different levels of AmpC hyperproduction, such as mutations of nonessential penicillin-binding proteins (PBP3, PBP4, and PBP7) and AmpC regulators (AmpD, AmpDh2, and AmpDh3). In addition, we also evaluated the activity of these inhibitors in a collection of MDR <i>P. aeruginosa</i> clinical isolates displaying different levels of <i>ampC</i> expression. The isolates were obtained from respiratory tract samples from patients with ventilator-associated pneumonia enrolled in the Magic Bullet clinical trial in different countries in Europe. The bacterial strains used are listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Laboratory Strains and Clinical Isolates Used in this Study</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">bacterial strain</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center">reference</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO1</td><td class="colsep0 rowsep0" align="left">completely sequenced reference strain.</td><td class="colsep0 rowsep0" align="left">laboratory collection</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAOΔC</td><td class="colsep0 rowsep0" align="left">PAO1 <i>ampC</i> knockout mutant</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAOΔC + pUCP-(PDC-1)</td><td class="colsep0 rowsep0" align="left">PAO1 <i>ampC</i> knockout mutant (PAOΔC) complemented with PDC-1</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAOΔC + pUCP-(PDC-5)</td><td class="colsep0 rowsep0" align="left">PAO1 <i>ampC</i> knockout mutant (PAOΔC) complemented with PDC-5 (which presents a similar hydrolysis profile than PDC-1)</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>ampD</i></td><td class="colsep0 rowsep0" align="left">PAO1 AmpD mutant</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>dacB</i></td><td class="colsep0 rowsep0" align="left">PAO1 PBP4 mutant</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>dacB</i>Δ<i>dacC</i></td><td class="colsep0 rowsep0" align="left">PAO1 PBP4-PBP5 mutant</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>D</i>Δ<i>Dh2</i>Δ<i>Dh3</i></td><td class="colsep0 rowsep0" align="left">PAO1 AmpD-AmpDh2-AmpDh3 mutant</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>dacB</i>Δ<i>dacC</i>Δ<i>pbpG</i></td><td class="colsep0 rowsep0" align="left">PAO1 PBP4-PBP5-PBP7 mutant</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>dacB</i>Δ<i>ampD</i></td><td class="colsep0 rowsep0" align="left">PAO1 PBP4 AmpD mutant</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAHUAC-27 (ST-22)</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left">this study</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAHUAC-14 (ST-966)</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left">this study</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13/09-018 (ST-175)</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">17/14-017 (ST-235)</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">23/14-053 (ST-309)</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">53/22-113 (ST-244)</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">36/14-210 (ST-235)</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6/04-224 (ST-175)</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></td></tr></tbody></table></div></div><div class="NLM_p">As previously observed, <i>ampC</i> gene expression was correlated with ceftazidime resistance in the strains used.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> As the level of expression of the <i>ampC</i> gene in the reference strain PAO1 is low, high susceptibility to ceftazidime [MIC of 1 mg/L] was observed. Remarkably, inactivation of the <i>ampC</i> gene and the resulting loss of AmpC production in the PAOΔC mutant did not imply a further decrease in the ceftazidime MICs. As expected, PAOΔC mutants complemented with PDC-1 or PDC-5 using the plasmid pUCP24 displayed high <i>ampC</i> expression levels. Furthermore, <i>ampC</i> expression was induced in single mutants of PBPs and <i>ampD</i> regulatory genes and increased accumulatively in double and triple mutants. As a result, a significant increase in the ceftazidime MIC (16–32 mg/L) was observed with all AmpC-overproducing PAO1 mutants (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Regarding clinical strains, a MIC of 8 mg/L was obtained with two isolates in which expression of the <i>bla</i><sub>PDC-1</sub> gene was lowest, and MICs of 16–128 mg/L were obtained with clinical strains in which expression of the <i>ampC</i> gene was at least 10-fold higher than that of the PAO1 strain. As these are clinical strains, they are likely to have other underlying mechanisms of resistance to β-lactam antibiotics in addition to chromosomal <i>ampC</i> cephalosporinase (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Ceftazidime MICs Determined in the Presence of β-lactamase Inhibitors and <i>bla</i><sub>ampC</sub> RNA Expression Levels for PAO1 and Isogenic Mutants Expressing Multiple Combinations of <i>ampC</i> Regulatory Genes</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">inhibitor at 4 mg/L</th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">inhibitor at 16 mg/L</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">strain</th><th class="colsep0 rowsep0" align="center" char=".">CAZ</th><th class="colsep0 rowsep0" align="center" char=".">CAZ + AVI</th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>1</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>2</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>3</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + AVI</th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>1</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>2</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>3</b></th><th class="colsep0 rowsep0" align="center"><i>ampC</i> RNA expression<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="center">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAOΔC</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="center">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAOΔC + pUCP-(PDC-1)</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="center">1933.4 ± 264.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAOΔC + pUCP-(PDC-5)</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="center">4084.8 ± 478.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>ampD</i></td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="center">51.4 ± 4.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>dacB</i></td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="center">36 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>D</i>Δ<i>Dh2</i>Δ<i>Dh3</i></td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="center">1030.2 ± 172.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>dacB</i>Δ<i>pbpG</i>Δ<i>dacC</i></td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="center">1109.9 ± 132.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>dacB</i>Δ<i>ampD</i></td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="center">1021.4 ± 140.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO Δ<i>dacB</i>Δ<i>dacC</i></td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="center">472.3 ± 36.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Relative <i>ampC</i> expression (with respect to PAO1). Mean values ± standard deviations for three independent experiments are shown.</p></div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Ceftazidime MICs Determined in the Presence of β-lactamase Inhibitors and <i>bla</i><sub>ampC</sub> RNA Expression Levels for Clinical Strains of <i>P. aeruginosa</i></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">inhibitor at 4 mg/L</th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">inhibitor at 16 mg/L</th><th class="colsep0 rowsep0" align="center" char="±"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Strain</th><th class="colsep0 rowsep0" align="center" char=".">CAZ</th><th class="colsep0 rowsep0" align="center" char=".">CAZ + AVI</th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>1</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>2</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>3</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + AVI</th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>1</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>2</b></th><th class="colsep0 rowsep0" align="center" char=".">CAZ + <b>3</b></th><th class="colsep0 rowsep0" align="center" char="±"><i>ampC</i> RNA expression<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAHUAC-27 (ST-22)</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char="±">326.4 ± 25.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAHUAC-14 (ST-966)</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char="±">586.3 ± 45.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13/09-018 (ST-175)</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char="±">2.4 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">17/14-017 (ST-235)</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char="±">32.8 ± 1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">23/14-053 (ST-309)</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char="±">6.4 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">53/22-113 (ST-244)</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="±">1415.3 ± 302.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">36/14-210 (ST-235)</td><td class="colsep0 rowsep0" align="char" char=".">128</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char="±">18.6 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6/04-224 (ST-175)</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char="±">10.1 ± 0.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Relative <i>ampC</i> expression (with respect to PAO1). Mean values ± standard deviation for three independent experiments are shown.</p></div></div></div><div class="NLM_p">The results of comparative MIC assays for ceftazidime in combination with the different inhibitors showed that all inhibitors were able to increase the susceptibility to ceftazidime in strains with low, medium, or high expression of the <i>ampC</i> gene (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Avibactam and inhibitors <b>1–3</b>, at a concentration of 4 mg/L, were able to significantly increase susceptibility of the collection of PAO1-derived mutants to ceftazidime, with 8–16, 4–16, 8–32, and 2–16-fold increases, respectively; when tested at 16 mg/L, the susceptibility increased 16–32-fold in the presence of avibactam and compound <b>2</b>, it increased 8–32-fold in the presence of compounds <b>1</b> and <b>3</b>. When inhibitors were used at 16 mg/L, the ceftazidime MICs decreased to the baseline levels shown by the PAO1 and PAOΔC strains (lacking the <i>ampC</i> gene), which is an important indication of the total inhibition of the AmpC enzyme, regardless of the level of <i>ampC</i> expression. Remarkably, the PDC-5 β-lactamase of the mutant PAOΔC + pUCP-(PDC-5),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> which displayed the highest <i>ampC</i> expression of the whole collection, was also inactivated by compounds <b>1–3</b> and avibactam in a similar way as in the PDC-1 in the PAOΔC + pUCP-(PDC-1) isogenic mutant. Regarding the MDR clinical strains, use of the inhibitors at 4 mg/L also increased the susceptibility to ceftazidime 4–32, 4–16, 4–32, and 4–8-fold when used in combination with avibactam and compounds <b>1–3</b>, respectively. Finally, use of the inhibitors at 16 mg/L increased the susceptibility to the cephalosporin 4–32-fold with all the inhibitors tested.</div><div class="NLM_p last">Following the EUCAST criteria, all strains used, that is, both PAO1 mutants and clinical isolates, were resistant to ceftazidime (breakpoint susceptibility of MIC ≤ 8 mg/L), except clinical isolates 13/09-018 and 23/14-053, for which the ceftazidime MIC was 8 mg/L. For all strains used in this study, the ceftazidime MICs were within the susceptibility range when tested in the presence of the inhibitors <b>1–3</b> and avibactam (16 mg/L). In summary, the penicillin-based sulfones <b>1–3</b> showed an inactivation efficacy in <i>P. aeruginosa</i> very similar to that of avibactam, which along with relebactam, is possibly the best class C β-lactamases inhibitor currently available.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Kinetics and Inhibition Studies</h3><div class="NLM_p">Kinetic analysis of the inhibition of PDC-1 β-lactamase by compounds <b>1–3</b> is shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Nitrocefin was used as the reported substrate (<i>K</i><sub>m</sub> = 34 μM) at a concentration of 120 μM to ensure a saturated concentration in the kinetic assays. Of the three inhibitors, bromide <b>2</b> displayed the highest inhibition efficiency <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> (20.9 × 10<sup>5</sup> M<sup>–1</sup> s<sup>–1</sup>), which was 3.3- and 4.2-fold higher than the values yielded by inhibitors <b>1</b> and <b>3</b>, respectively. Compound <b>1</b> showed the best inactivation rate (<i>k</i><sub>inact</sub>); however, the greatest differences were observed in the <i>K</i><sub>I</sub> parameter, with compound <b>1</b> displaying the lowest affinity, with the highest <i>K</i><sub>I</sub> value (74.8 nM), and the <i>K</i><sub>I</sub> of compounds <b>2</b> and <b>3</b> being 4.8- and 2.7-fold lower, respectively. Similar inactivation efficiencies were indicated by the acylation rate (<i>k</i><sub>2</sub>/<i>K)</i>. Again, bromide <b>2</b> proved to be the best inhibitor of the series and displayed the highest values of <i>k</i><sub>2</sub>/<i>K</i> (6.2 × 10<sup>5</sup> M<sup>–1</sup> s<sup>–1</sup>). For avibactam, the <i>k</i><sub>2</sub>/<i>K</i> value was 1366 M<sup>–1</sup> s<sup>–1</sup>, which is consistent with the results obtained by other researchers.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Interestingly, the acylation rates of compounds <b>1</b>, <b>2</b>, and <b>3</b> were significantly higher than those of avibactam (110-, 454-, and 50-fold, respectively).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. PDC-1 Inhibition Kinetics</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">inhibitor</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i app</sub> (nM)</th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>2</sub>/<i>K</i> (M<sup>–1</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>I</sub> (nM)</th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>inact</sub> (s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> (M<sup>–1</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="."><i>t</i><sub><i>n</i></sub></th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>off</sub> (min<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="±"><i>t</i><sub>1/2</sub> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Avibactam<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">51.0 ± 3.1</td><td class="colsep0 rowsep0" align="left">990 ± 75</td><td class="colsep0 rowsep0" align="left">1366 ± 430</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="left">2.6 × 10<sup>–2</sup> ± 0.6 × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="char" char="±">28 ± 7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">6.1 ± 1.9</td><td class="colsep0 rowsep0" align="left">53 ± 14</td><td class="colsep0 rowsep0" align="left">1.5 × 10<sup>5</sup> ± 0.5 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="left">74.8 ± 18.9</td><td class="colsep0 rowsep0" align="left">0.046 ± 0.006</td><td class="colsep0 rowsep0" align="left">6.3 × 10<sup>5</sup> ± 1.4 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">9 × 10<sup>–3</sup> ± 2 × 10<sup>–3</sup></td><td class="colsep0 rowsep0" align="char" char="±">82 ± 19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">0.9 ± 0.03</td><td class="colsep0 rowsep0" align="left">14 ± 4</td><td class="colsep0 rowsep0" align="left">6.2 × 10<sup>5</sup> ± 0.9 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="left">15.6 ± 3.1</td><td class="colsep0 rowsep0" align="left">0.032 ± 0.002</td><td class="colsep0 rowsep0" align="left">20.9 × 10<sup>5</sup> ± 3.1 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">7 × 10<sup>–3</sup> ± 2 × 10<sup>–3</sup></td><td class="colsep0 rowsep0" align="char" char="±">105 ± 29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">6.3 ± 1.0</td><td class="colsep0 rowsep0" align="left">110 ± 18</td><td class="colsep0 rowsep0" align="left">6.8 × 10<sup>4</sup> ± 1.6 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">28.2 ± 4.9</td><td class="colsep0 rowsep0" align="left">0.013 ± 0.004</td><td class="colsep0 rowsep0" align="left">5.0 × 10<sup>5</sup> ± 1.2 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">9 × 10<sup>–3</sup> ± 2 × 10<sup>–3</sup></td><td class="colsep0 rowsep0" align="char" char="±">78 ± 18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Sulbactam</td><td class="colsep0 rowsep0" align="left">4369.6 ± 791.3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Tazobactam</td><td class="colsep0 rowsep0" align="left">844.4 ± 38.9</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Clavulanic acid</td><td class="colsep0 rowsep0" align="left">>200,000</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">In the <i>k</i><sub>obs</sub> determination for avibactam, saturating plots were not obtained and therefore it was not possible to obtain parameters <i>K</i><sub>I</sub>, <i>k</i><sub>inact</sub>, and <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub>.</p></div></div></div><div class="NLM_p">Similarly, the IC<sub>50</sub> value of bromide <b>2</b> against PDC-1 (0.9 nM) was 6.8- and 7.0-fold lower than those of compounds <b>1</b> (6.1 nM) and <b>3</b> (6.3 nM) and, importantly, 57-fold lower than that of avibactam (51.0 nM). For purposes of comparison, we have included in the table the IC<sub>50</sub> value of the commercially available β-lactamase inhibitors sulbactam, tazobactam, and clavulanic acid against the β-lactamase PDC-1 (4369, 844, and >200,000 nM, respectively). In these inhibitors, which display high affinity for class A β-lactamases, the IC<sub>50</sub> value was much higher than in the 2-pyridylmethylidene penicillin-based sulfones <b>1–3</b>.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div><div class="NLM_p">In pre-steady-state conditions, compounds <b>1</b> and <b>2</b> showed the highest affinities for PDC-1, leading to <i>K</i><sub>i app</sub> values of 53 nM and 14 nM, respectively. By contrast, avibactam displayed the lowest affinity (990 nM), which was 71-fold lower than that of bromide <b>2</b>. Moreover, the turnover number (<i>t</i><sub><i>n</i></sub>), that is, the number of molecules of the inhibitor required to inactivate one molecule of the enzyme, was also measured. All the penicillin-based sulfones tested yielded a <i>t</i><sub><i>n</i></sub> value of 1 with PDC-1. By contrast, the comparator avibactam displayed a <i>t</i><sub><i>n</i></sub> of 18 with the β-lactamase.</div><div class="NLM_p">In order to analyze whether compounds <b>1–3</b> are reversible or irreversible inhibitors, the deacylation off-rate (<i>k</i><sub>off</sub>) was determined by using the “jump-dilution” method, and the results were compared with those obtained with avibactam (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Like avibactam, the inhibition is reversible, but the <i>k</i><sub>off</sub> values of the test compounds were much lower. Thus, while avibactam displayed a <i>k</i><sub>off</sub> of 2.6 × 10<sup>–2</sup> ± 0.6 × 10<sup>–2</sup> min<sup>–1</sup>, which is equivalent to a residence time half-life (<i>t</i><sub>1/2</sub>) of 28 ± 7 min, compounds <b>1</b>, <b>2</b>, and <b>3</b> showed <i>k</i><sub>off</sub> values of 9 × 10<sup>–3</sup> ± 2 × 10<sup>–3</sup>, 7 × 10<sup>–3</sup> ± 2 × 10<sup>–3</sup>, and 9 × 10<sup>–3</sup> ± 2 × 10<sup>–3</sup> min<sup>–1</sup>, respectively, with <i>t</i><sub>1/2</sub> values of 82 ± 19, 105 ± 29, and 78 ± 18 min, respectively. These data suggest a slow deacylation process for compounds <b>1–3</b>, indicating that the enzyme would remain inactivated for a longer time, with compound <b>2</b> being the best inhibitor.</div><div class="NLM_p">Comparison of compounds <b>1–3</b> revealed that bromide <b>2</b> exhibited a significantly higher inhibition efficacy (<i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub>, <i>p</i> < 0.01) than compounds <b>1</b> and <b>3</b>, mainly due to a higher affinity for chromosomal class C β-lactamase PDC-1 (reflected in lower IC<sub>50</sub> or <i>K</i><sub>I</sub> values), also displaying higher acylation rates (<i>k</i><sub>2</sub>/<i>K</i>, <i>p</i> < 0.005).</div><div class="NLM_p last">Moreover, in order to explore whether compounds <b>1–3</b> would also be capable of deactivating metallo-β-lactamases, for which effective therapies in clinical practice are currently not available, their inhibitory activity against the widespread metallo-β-lactamase VIM-1 was evaluated. Compounds <b>2</b> and <b>3</b> exhibited moderate activity against this enzyme, with <i>K</i><sub>i app</sub> values of 143 ± 45 and 192 ± 52 μM, respectively. Like avibactam, the inhibitory capacity of compound <b>1</b> against VIM-1 was lower, in both cases with <i>K</i><sub>i app</sub> values > 500 μM. Considering that compounds <b>2</b> and <b>3</b> in combination with ceftazidime, ampicillin, and imipenem also proved effective for restoring antibiotic efficacy in strains carrying class C (CMY-2 and DHA-1), class A (TEM-1 and CTX-M-2), and class D (OXA-23, OXA-24/40, and OXA-48) β-lactamases, respectively, the broad spectrum of activity of these halides appears to be relevant (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Tables S2 and S3</a>).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Mass Spectrometry Studies</h3><div class="NLM_p">The covalent modification mechanism, whereby compounds <b>1–3</b> inhibit the PDC-1 enzyme, was studied by mass spectrometry [matrix-assisted laser desorption/ionization (MALDI)]. Inhibitors <b>1–3</b> were incubated with the enzyme at a 1:100 enzyme/ligand ratio in phosphate-buffered saline at pH 7.4 at 25 °C for 30 min and then analyzed by MALDI. The mass spectra of samples of the enzyme inactivated by compounds <b>1–3</b> consisted of a peak for the covalently modified enzyme, corresponding to an additional mass of 487, 565, and 503, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). These values are consistent with the theoretical mass increase values for the corresponding indolizine adducts (487, 566, and 506, respectively).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. MALDI spectra of samples of PDC-1 after incubation with ligands <b>1–3</b> (A) and of the control (incubated without ligands) (B). (C) Superimposed MALDI spectra. Incubation conditions were phosphate-buffered saline pH 7.4, 25 °C and (1:100) protein/ligand ratio, 30 min.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Binding Mode—MD Simulation Studies</h3><div class="NLM_p">To understand the molecular basis of the activity of the inhibitors <b>1–3</b> against the PDC-1 enzyme, MD simulation studies were carried out. These types of computational studies, in which both the ligand and the enzyme are considered flexible, are important complementary tools for drug discovery, enabling some insights to be gained into the atomic detail of the following: (i) the intrinsic shape-changing motions of the enzymes and the identification of less evident pockets; (ii) transitory enzyme/ligand complex species, such as the Michaelis complex that triggers the acylation process; (iii) the induced-fit capacity of the enzyme modulated by the ligand (“the conformation state”); and (iv) the binding mode of diverse ligands with related enzyme targets, even those for which no crystal structures are available (homology model) or for which the topology of the already solved ligands is quite different from that of the compounds under evaluation. For this type of study, it is important to know the spatial arrangement and protonation state of the key residues for catalysis, as both elements will control the binding of the ligand within the pocket. This proved to be an essential point here because, as discussed below, the identified differences between the reported ligands are largely due to their distinct interaction with the catalytic residue Tyr150. In this regard, AmpC enzymes have a Lys–Tyr–Lys triad (Lys67–Tyr150–Lys315) in the active site. The role of Lys67 and Tyr150 in the enzyme-catalyzed reaction has been widely discussed. It has been suggested that the phenolate form of Tyr150, which proved to have an abnormally low p<i>K</i><sub>a</sub> (<9) at neutral pH, is responsible for (among other roles) activation of the catalytic serine in the acyl-enzyme adduct formation and that it is stabilized by the positively charged residues Lys67 and Lys315.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> However, subsequent biochemical and mutagenesis studies revealed that Tyr150 is neutral during catalysis.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Both possibilities, that is, both neutral and ionic forms, were explored by studying the dynamic behavior of PDC-1 in the unbound form by using the protein coordinates in the crystal structure of AmpC from <i>P. aeruginosa</i> PAO1 (PDC-1, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZB">4GZB</a>, 1.79 Å)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The results showed that the appropriate arrangement of acyl-enzyme adduct formation with Ser64 (<i>i.e.,</i> with the oxygen lone pair pointing toward the β-lactam carbonyl group) would only be achieved when Tyr150 and Lys67 are in their neutral forms and that this arrangement would be mainly controlled by Lys67 (see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Figures S1 and S2</a> for further details). These protonation states were used in all subsequent studies.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Detailed view of the catalytic active site of the unbound form of PDC-1 obtained by MD simulation studies, in which the ionic (A) and neutral (B) forms of the catalytic residues Tyr150 and Lys67 were considered. Snapshots taken after 80 ns of simulation are shown. Relevant side-chain residues of the active site are shown and labeled. Note how only the neutral forms of Lys67 and Tyr150 (B) provide the appropriate arrangement of Ser64 for formation of the enzyme-acyl adduct.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The binding modes of compounds <b>1–3</b> in the active site of PDC-1 were first studied by molecular docking, with GOLD software (version 5.8.1) and PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZB">4GZB</a>.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The highest scoring solutions were then subjected to 200 ns of dynamic simulation to produce reliable models of the PDC-1/<b>1–3</b> enzyme binary complexes. These poses of the inhibitors placed the pyridine moiety (R1 side chain) in the vicinity of the H10 helix and the catechol group (R2 side chain) nearly adjacent to the Ω-loop. However, because β-lactam antibiotics, such as ceftazidime and ceftolozane, bind to class C β-lactamases by locating the side chain that supports the siderophore group (R1 side chain) in the vicinity of the Ω-loop and the chain that has the leaving group (R2 side chain) near the H10 helix region, scoring solutions showing a similar arrangement were also analyzed by MD simulation studies (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Figure S3</a>). These studies clearly showed that (i) under the latter orientation, the ligands rapidly moved away from the catalytic serine from the beginning of the simulation; (ii) the ligands proved to be poorly anchored to the active site because a wide variety of arrangements were identified; and in addition, (iii) the conformations of the ligands, in particular, the β-lactam carbonyl group, were not appropriate for nucleophilic attack and the formation of the acyl-enzyme adduct. On the contrary, the simulation studies were carried out with opposite disposition of the ligands, that is, with their siderophore group also close to the Ω-loop, which were the highest scoring solutions observed by docking, demonstrated to be very stable during the simulation and more importantly with the appropriate arrangement for the formation of the acyl-enzyme adduct (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The results of these MD simulation studies revealed the following: (i) the PDC-1/<b>1–3</b> complexes were very stable and the inhibitors remained located in the active site throughout the simulation, as revealed by the low RMSD (root-mean-square deviation) values obtained for the enzyme backbone and ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Figure S4</a>); (ii) the side chain carrying the catechol group was very flexible and wide variability of arrangements were identified, suggesting that they contribute little to the binding; (iii) the ligands were anchored to the active site by three main hydrogen bonding interactions through the carboxylate group, the carbonyl group of the β-lactam ring, and the ester group in the side chain; (iv) the latter contacts were stronger for ligands <b>2</b> and <b>3</b>, as they appeared to be less mobile in the pocket than <b>1</b>; (v) the pyridyl moieties in ligands <b>1–3</b> were embedded in a tunnel-like site adjacent to the active site, which includes the Lys–Tyr–Lys triad pocket and is isolated from the solvent environment by H10 helix (residues 289–292) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–C); and (vi) the ligands <b>1–3</b> induced a close arrangement of the catalytic residues Tyr150 and Lys67, which was not observed in the unbound form.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. A-C. Overall view of the binding mode of inhibitors <b>1</b> (cyan), <b>2</b> (violet), and <b>3</b> (green) in the PDC-1 active site (red) for acylation reaction obtained by MD simulation studies. Snapshots taken after 190 ns of simulation are shown. D-F. Interactions of inhibitors <b>1–3</b> in the active site of PDC-1. The position of the H10 helix is highlighted in orange. G-I. Closeup view of the relative position between the pyridyl moiety in <b>1</b> (G), <b>2</b> (H), and <b>3</b> (I) and the side chain of the catalytic tyrosine. Snapshots taken after 190 ns of simulation are shown. Note the strong interactions (black dashed lines) between the fluoropyridyl group in <b>3</b> and Tyr150 and Arg148. Relevant side-chain residues are shown and labeled. Polar interactions are indicated as red dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The main differences in the binding mode of the inhibitors <b>1–3</b> are as follows: (i) ligands <b>2</b> and <b>3</b> were anchored more strongly than <b>1</b>. This is readily visualized by comparing several snapshots of the ligands taken throughout the simulation and by analyzing the variation in the distance between the oxygen atom of the catalytic serine and the carbon atom of the β-lactam carbonyl group for the three enzyme complexes (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Figure S5</a>) and (ii) the distinct interaction of the corresponding pyridyl moieties with the residues of the tunnel-like site adjacent to the catalytic center, in particular with the catalytic Tyr150 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G–I and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">S6</a>). Thus, for the PDC-1/<b>3</b> complex, the position of the fluoropyridyl group in <b>3</b> was frozen in the pocket by two strong contacts (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>I and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">S6</a>): (1) an electrostatic interaction between the fluorine substituent and the guanidinium group of Arg148 and (2) a strong π–π stacking interaction between its fluoropyridyl group and the phenol group of Tyr150, which may also cause a reduction in the p<i>K</i><sub>a</sub> of Tyr150.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Both interactions were very stable throughout the simulation, with average distances of 3.2 and 3.8 Å, respectively. For the PDC-1/<b>2</b> complex, the interaction between the bromopyridyl moiety in <b>2</b> and Tyr150 was less pronounced (average distance of 5.3 Å), and distinct arrangements of this ring were identified (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Figures S4 and S6B</a>). The differences in the polarization of the pyridine ring caused by both halogen atoms were due to the variation in their electronegativity [χ(F) = 3.98; χ(Br) = 2.66], which may be explained by their different locations in the ring (<i>meta vs para</i> substitution, respectively). Nonetheless, both ligands <b>2</b> and <b>3</b> froze the position of Tyr150 in the proximity of Lys93 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Figure S7</a>). By contrast, for the PDC-1/<b>1</b> complex, a greater conformational freedom of both the pyridyl moiety in <b>1</b> and the phenol group of Tyr150, which is located further from Lys67, was observed (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">S7</a>).</div><div class="NLM_p">Moreover, the studies carried out with the inactivated PDC-1/<b>1–3</b> enzyme adducts revealed that in all cases, the adducts were anchored by strong polar interactions with residues Ser318 and Arg349 and that their indolizine moieties would be embedded in the deep tunnel-like site surrounded by the inner part of the H10 helix and would establish strong apolar interactions with residues Ala292 and Met291 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">S8</a>). A strong π–π stacking interaction between Tyr150 and the indolizine adducts derived from <b>2–3</b>, which was absent in <b>1</b>, was identified and which is promoted by the reduced electron density of these halopyridyl rings. Thus, Tyr150 side chain was stabilized on top of the halogenated rings, and therefore, its position would be frozen in the active site. In addition, comparison of the herein described PDC-1/<b>1–3</b> enzyme adducts with the X-ray crystal structure of PDC-1 inactivated by avibactam (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OOY">4OOY</a>,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> 1.1 Å) revealed that, unlike compounds <b>1–3</b>, avibactam does not promote interactions with the tunnel-like pocket formed by the H10 helix (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). The lack of the latter contacts might explain the lower inhibitory capacity of avibactam against PDC-1 relative to inhibitors <b>1–3</b>.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A–C) Overall view of the active site of PDC-1 inactivated by inhibitors <b>1</b> (cyan), <b>2</b> (violet), and <b>3</b> (green) obtained in MD simulation studies. Snapshots taken after 90 ns of simulation are shown. (D–F) Interactions between the PDC-1/<b>1–3</b> indolizine adducts. Relevant side-chain residues are shown and labeled. A close-up view of the interaction between the indolizine moiety and Tyr150 is shown. Polar (red), π–π, and CH−π (black) interactions are indicated as dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overall view of the active site of PDC-1 inactivated by avibactam (A, yellow) and inhibitor <b>2</b> (B, violet) obtained by X-ray crystallography (PDB entry 4OOY,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> 1.1 Å) and MD simulation studies, respectively. The position of the H10 helix is highlighted in orange. The modified catalytic serine is shown in green and labeled. Note how for the PDC-1/<b>2</b> adduct (B) the indolizine moiety of the modified ligand would be well embedded in the deep pocket surrounded by the H10 helix. On the contrary, for the PDC-1/avibactam adduct (A), this pocket is empty.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Overall, the MD simulation studies carried out on the PDC-1/<b>1–3</b> enzyme adducts revealed differences in the interactions between the aromatic moieties and the tunnel-like site and the inner part of the H10 helix. Thus, the results suggested that the strong π–π stacking interaction with the catalytic Tyr150, induced by the electron-deficient pyridyl moieties in ligands <b>2</b> and <b>3</b> or by the corresponding substituted indolizine adducts, would improve their binding. At the same time, this interaction would cause a reduction in the flexibility of Tyr150 that would alter its role in the different stages of the process. We have previously reported a similar freezing effect on the inhibition of the bacterial dehydroquinase enzyme by fluorinated reversible competitive inhibitors.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Furthermore, the computational studies herein reported identified the pocket adjacent to the PDC-1 active site and formed by the H10 helix as an important enzyme site for improving or establishing binding of ligand fragments or small molecules. The architecture and nature of this region of the PDC-1 enzyme would be particularly suitable for electron-deficient aromatic rings able to interact with Tyr150, Arg148, and/or apolar residues of H10 helix, among others. Indeed, in a fragment-based discovery approach, Babaoglu and Shoichet<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> also found that 4-methanesulfonylamino benzoic acid and 2-thiophene-2-carboxylic acid bind to the equivalent site of the <i>E. coli</i> AmpC enzyme. This finding was supported by the resolution of the crystal structures of the enzyme in forming a complex with those ligands (PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HDQ">2HDQ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HDS">2HDS</a>), in which a salt bridge with Arg148 and π–π stacking interactions with Tyr150 was identified.</div><div class="NLM_p">On the other hand, comparison of the architecture of the tunnel-like pocket identified in the PDC-1 enzyme with that of other AmpC enzymes revealed that this pocket is not fully evident in all cases. Thus, Babaoglu and Shoichet<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> pointed out that in the <i>E. coli</i> AmpC enzyme, the pocket is only fully visible in the presence of the aforementioned aromatic ligands. By contrast, the crystal structure of the extended spectrum and plasmid-encoded AmpC enzyme CMY-10 from <i>Klebsiella aerogenes</i> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K1D">5K1D</a>, 1.94 Å),<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> which has a competitive inhibitor, guanosine-5′-monophosphate (GMP), in the active site, but no contact with the inner part of the H10 helix, the equivalent site is a very small narrow cavity. At the same time, in the crystal structure of AmpC BER from <i>E. coli</i> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GZW">5GZW</a>, 1.49 Å)<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> in complex with the same inhibitor, GMP, the H10 helix is not solved and the active site appears quite open. Altogether, these findings reinforce the idea that the favorable interactions between the inner part of the H10 helix of the AmpC enzymes and the ligand may induce folding and generation of a pocket that appears to be a suitable target for drug design. This narrow, isolated cavity would enhance ligand binding and protect the enzyme adduct from hydrolysis. To explore this hypothesis, we studied <i>in silico</i> the interaction between compound <b>2</b> and the plasmid-encoded AmpC CMY-10 and BER enzymes, in which the architecture observed in the available crystal structures does not indicate a similar pocket for drug design. For the AmpC BER enzyme, the missing residues of the H10 helix were modeled using ModLoop Server,<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> and the results were compared with simulations conducted without this helix.</div><div class="NLM_p">First, the intrinsic shape-changing motions of CMY-10 and BER enzymes were evaluated by MD simulation studies on their unbound forms and further corroborated by examination of the vibrational modes calculated by principal-component analysis, as implemented in AMBER. The results revealed remarkable differences in the plasticity of the H10 helix, and the size and geometry of the pocket close to the catalytic center are therefore distinct (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Thus, in contrast to the CMY-10 enzyme, the H10 helix of the BER enzyme proved very flexible and had a strong capacity to adopt ample and diverse arrangements with a difference of up to ∼9 Å of distance. As a result, in the unbound form, or when no ligand fragments were bound to the Lys–Tyr–Lys triad pocket, this area of the enzyme would be more open, and no tunnel-like/cavity site, as observed in PDC-1 and CMY-10 enzymes, was apparent. However, the simulations performed on the BER/<b>2</b> binary complex, as well as on the corresponding indolizine adduct, showed that the presence of the aromatic moiety of the ligand would promote folding of the H10 helix and its stabilization over the triad pocket (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). Accordingly, a deep tunnel-like site such as that observed for the PDC-1 enzyme was identified, while the absence of the H10 helix in the BER/<b>2</b> binary complex caused destabilization of the ligand and its removal from the catalytic center. These results revealed the high (induced) capacity of the H10 helix to host the ligand. For the CMY-10 enzyme, the conformational changes in the arrangement of H10 helix were small, and the indolizine and pyridyl moieties were deeply buried in the pocket. For both enzymes, strong π–π stacking interactions with the catalytic Tyr150 and apolar contacts with the lipophilic residues of the inner part of the H10 helix [for CMY-10, Ala289, and Ile292 and for BER, Ile291, and Ala292] and the aromatic group in <b>2</b> and the resulting indolizine were also identified.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A,B) Overall view of the intrinsic shape-changing movements of AmpC BER (A) and CMY-10 (B) enzymes, obtained by examination of the vibrational modes and RMSD plots for the protein backbone (Cα, C, N, and O atoms) calculated per residue obtained from MD simulation studies. The main vibrational modes are shown. Note the high plasticity of the H10 helix in the unbound form of the BER enzyme relative to CMY-10. (C) Comparison of several snapshots of the BER/<b>2</b> binary complex during simulation and overall view of the tunnel-like pocket before (0 ns) and after simulation (200 ns). Note how after ∼20 ns of simulation the ligand induced folding of the H10 helix, which remained almost at a standstill as a ligand-covering lid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A,C) Overall view of the active site of plasmid-encoded AmpC CMY-10 (A) and BER (C) β-lactamases inactivated by inhibitor <b>2</b> (violet) obtained by MD simulation studies. (B,D) Proposed binding mode of bromide <b>2</b> in the active site of CMY-10 (B) and BER (D) for acylation reaction. Snapshots taken after 80 ns of simulation are shown. Relevant side-chain residues are shown and labeled. Polar (red) and apolar (black) interactions are indicated as dashed lines. The catalytic serine is highlighted in yellow. The positions of the H10 helix and the catalytic tyrosine are underlined in orange and blue, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Overall, these results support the idea that the introduction of electron-deficient aromatic moieties in a scaffold able to reach the not-always-evident tunnel-like site in which the Lys–Tyr–Lys triad is located and isolated from the environment by the H10 helix may be a good approach toward improving the affinity of existing inhibitors of the AmpC enzymes or the development of new inhibitors. The results also suggest that the H10 helix may act as a lid for the Lys–Tyr–Lys triad during catalysis. The protective effect on the indolizine moiety induced by the interaction with the H10 helix in the AmpC enzymes is reminiscent of that caused by the tunnel-like entrance of OXA-24/40 and OXA-23 enzymes.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The same type of computational studies carried out with other β-lactamase enzymes, such as TEM-1 from <i>E. coli</i> (class A, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ERM">1ERM</a>)<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and OXA-48 from <i>Klebsiella pneumoniae</i> (class D, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S2N">4S2N</a>)<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> which compounds <b>1–3</b> efficiently inactivated, were also performed. However, for the latter enzymes, this type of arrangement, with the indolizine adduct well embedded in the enzyme, was not observed (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). In these cases, the pyridyl and indolizine moieties would establish numerous lipophilic interactions with apolar residues, as well as CH−π interactions with aromatic residues located in the site adjacent to the active site.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A,C) Overall view of the active site of OXA-48 (A, class D) and TEM-1 (C, class A) β-lactamases inactivated by inhibitor <b>2</b> (violet), obtained by MD simulation studies. (B,D) Proposed binding mode of bromide <b>2</b> in the active site of OXA-48 (B) and TEM-1 (D) for acylation reaction. Snapshots taken after 80 ns of simulation are shown. Relevant side chain residues are shown and labeled. Polar (red) and apolar (black) interactions are indicated as dashed lines. The catalytic serine is highlighted in yellow. The residues that promote apolar interactions with the ligand are enhanced in green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14114" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14114" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The studies carried out with 6-pyridylmethylene penicillin-based sulfones <b>1–3</b> against MDR clinical isolates and laboratory mutant <i>P. aeruginosa</i> strains showing different levels of stable AmpC hyperproduction revealed that these compounds showed similar inhibition efficiency to that of avibactam, decreasing the ceftazidime MICs for all strains tested. Kinetic studies carried out with the PDC-1 enzyme revealed that the introduction of electron-deficient substituents in the pyridyl ring (4-bromopyridyl or 5-fluoropyridyl) is essential for improving the binding affinity for the AmpC of <i>P. aeruginosa</i>. The inhibition proved to be reversible, with a slow deacylation rate, with residence time half-life values between 2.8–3.8-fold higher than those of avibactam. The bromopyridyl derivative <b>2</b> proved to be the inhibitor with the highest affinity of the series, as indicated by the approximately fivefold lower <i>K</i><sub>I</sub> relative to the parent compound <b>1</b> and the highest inhibitory efficiency (<i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub>), which was respectively 3.3 and 4.2-fold higher than the values yielded by inhibitors <b>1</b> and <b>3</b>. Mass spectrometry (MALDI) studies revealed that these compounds inactivate the PDC-1 enzyme by forming a stable indolizine adduct through the catalytic serine.</div><div class="NLM_p">The results of the MD simulation studies carried out on the PDC-1/<b>1–3</b> binary complexes, which trigger the acylation process, as well as the resulting PDC-1/<b>1–3</b> adducts, revealed that, in contrast to avibactam, these ligands/adducts would promote additional binding interactions with an adjacent site to the catalytic serine. Thus, the aromatic moieties of these ligands/adducts would be buried in the tunnel-like site formed by the H10 helix, which folded neatly over the Lys–Tyr–Lys triad pocket. Distinct binding interactions (strength and type) of the aromatic moieties in <b>1–3</b> with the residues involved in this site were identified. Specifically, the most remarkable contacts identified were π–π stacking interactions with the catalytic Tyr150, lipophilic interactions with the apolar residues of the inner part of H10 helix, and hydrogen bonding interactions with Arg148. The electron-deficiency of the pyridyl rings in <b>2</b> and <b>3</b>, as well as the position of the halogen atom in the ring (<i>meta vs para</i>), which would be absent in <b>1</b>, are probably responsible for these effects.</div><div class="NLM_p">Based on the results of the computational studies carried out on the AmpC enzymes CMY-10, BER, and PDC-1, the attractive interactions promoted by the electron-deficient aromatic ring of the ligands would facilitate generation of this tunnel-like pocket, which is not evident in some of the available crystal structures. These outcomes highlight the potential use of this pocket in AmpC enzymes as a target for the design or optimization of future or existing inhibitors, as well as the high induced-fit capacity of the H10 helix, the architecture of which seems to be readily modulated by the appropriate ligand.</div><div class="NLM_p last">These outcomes, together with the previously reported excellent results obtained with inhibitors <b>1–3</b> against the ESBL CTX-M-2 from <i>Enterobacterales</i> and the class D carbapenemases OXA-23 and OXA-24/40 from <i>A. baumannii</i>,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> identify these compounds, particularly bromide <b>2</b>, as broad-spectrum serine-β-lactamase inhibitors capable of restoring the activity of cephalosporins and carbapenems in clinical use against the most important MDR pathogens.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80520" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80520" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Bacterial Strains and Media</h3><div class="NLM_p">A collection of <i>P. aeruginosa</i> PAO1 isogenic mutants with different levels of <i>ampC</i> expression were assayed.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> In addition, PAO1, PAOΔC<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (PAO1 mutant without an <i>ampC</i> gene), PAOΔC expressing PDC-1 (from PAO1),<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> and PDC-5 (from ATCC 25324, which contained R79Q and T105A substitutions relative to PDC-1)<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> were also assayed.</div><div class="NLM_p">A representative isogenic panel of PAO1-derived mutants obtained in previous studies and showing AmpC hyperexpression was included. The collection includes isogenic mutants with single or multiple deletions of the major chromosomal genes involved in the <i>ampC</i> regulatory pathway: the cytosolic amydase <i>ampD</i> and homologues <i>ampDh2</i> and <i>ampDh3</i>,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and the genes coding for the nonessential penicillin-binding proteins PBP4 (<i>dacB</i>), PBP5 (<i>dacC</i>), and PBP7 (<i>pbpG</i>).<a onclick="showRef(event, 'ref33 ref53'); return false;" href="javascript:void(0);" class="ref ref33 ref53">(33,53)</a> In addition, a panel of six MDR isolates, from the collection of clinical strains obtained in different countries in Europe during from the MagicBullet clinical trial,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> and two clinical isolates from A Coruña Hospital (Spain) were also included in the study. All strains used are listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_p">A VIM-1-producing <i>P. aeruginosa</i> isolate, recovered in a hospital in Majorca (Spain) during a survey to evaluate the incidence of MBL-producing strains, was also included for cloning and purification of the metallo-β-lactamase.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div><div class="NLM_p last">Bacterial strains were frozen in Luria–Bertani broth (LB) with 10% glycerol and maintained at −80 °C. Strains were grown at 37 °C in LB medium supplemented with gentamicin (30 mg/L) when necessary (Sigma-Genosys).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Real-Time Reverse Transcription Polymerase Chain Reaction of the <i>bla</i><sub>ampC</sub> Gene</h3><div class="NLM_p last">The levels of <i>ampC</i> gene expression were determined by real-time reverse transcription polymerase chain reaction (RT)-PCR, as previously described.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Briefly, each strain was grown to an exponential phase (OD<sub>600nm</sub> = 0.6), and the RNA was isolated using a High Pure RNA Isolation Kit (Roche). RNA samples were treated with DNAse I for 15 min and purified using the RNeasy MinElute Cleanup Kit (QIAGEN). RNA extractions were quantified using a BioDrop lLITE (Isogen Life Science). An aliquot of 50 ng of purified RNA was used in the (RT)-PCR, performed with the LightCycler 480 RNA Master Hydrolysis Probe Kit and in a LightCycler 480 RNA instrument (Roche). The primers 5′-GAAGGCTACGGGGTGAAGA-3′ and 5′-ATGCAGGTTGGCATCGAC-3′ were used to amplify the <i>ampC</i> gene, and the primers 5′-CCGAACTGTGCCTTACCAC-3′ and 5′-GTTCCACCGGCTCGTACA-3′ were used to amplify the <i>rpoB</i> gene (included as a housekeeping gene). Mean values of mRNA levels were determined in three independent replicate assays.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Compounds <b>1–3</b></h3><div class="NLM_p last">They were synthesized in 13 steps from commercially available (+)-6-aminopenicillanic acid, as previously described.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The purity of compounds <b>1–3</b> was analyzed by NMR and high-performance liquid chromatography (HPLC). HPLC was performed on a Thermo Dionex UltiMate 3000 apparatus equipped with a Brucker amazon SL mass spectrometry detector. Samples were electrophoresed on a Phenomenex kinetex XB-C18 column (particle size = 1.7 μm; dimensions: 50 mm × 2.1 mm, pore size = 100 Å) and eluted at a flow rate of 0.35 mL min<sup>–1</sup> with a gradient of 5–75% B in 10 min [A = Milli-Q water + 0.1% TFA; B = acetonitrile + 0.1% TFA]. The purity of all compounds tested was ≥ 95% (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">Figures S9–S11</a>).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Purification of β-lactamases</h3><div class="NLM_p">The enzyme PDC-1 was purified using the GST Gene Fusion System (Amersham). The PDC-1 gene from PAO1 strain (without the region coding for the peptide signal) was amplified and cloned in the pGEX-6P-1 vector with a primer pair containing recognition sites for the restriction enzymes BamHI (5′-AAAGGATCCAAGGAATGGCAAGAAAACAAA-3′) and EcoRI (5′-CCGGAATTCCGGGTCAGCGCTTCA-3′). The recombinant plasmid was electroporated into <i>E. coli</i> BL21, a protease-deficient strain, and selected in LB plates containing 50 mg/L of ampicillin (Sigma-Genosys). This construction generated the fusion protein glutathione S-transferase (GST)/PDC-1. The GST tag was then cleaved off, and the β-lactamase was later purified to homogeneity using affinity chromatography. Finally, after SDS-PAGE, the purified protein appeared as a band of approximately 41 kDa (≥95% purity).</div><div class="NLM_p last">The metallo-β-lactamase VIM-1 was also purified to homogeneity using the same methodology. The <i>bla</i><sub>VIM-1</sub> gene was amplified from a <i>P. aeruginosa</i> clinical isolate recovered from a hospital in Majorca (Spain) during a survey to evaluate the incidence of MBL-producing strains. The <i>bla</i><sub>VIM-1</sub> gene was cloned in the pGEX-6P-1 vector using the primers V1-F-BHI (5′-TCGGATCCGGGGAGCCGAGTGGTGAG-3′) and V1/13-R-ERI (5′-TCGAATTCCTACTCGGCGACTGAGCG-3′). The fusion protein GST/VIM-1 was later produced, and finally, VIM-1 was purified by affinity chromatography, as previously described.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Kinetic Experiments</h3><div class="NLM_p">Inactivation of PDC-1 from <i>P. aeruginosa</i> by compounds <b>1–3</b> was evaluated using nitrocefin (Oxoid) as a reporter substrate. All experiments were performed at 25 °C in phosphate-buffered saline pH 7.4 (11.9 mM sodium phosphate dibasic and potassium phosphate monobasic, 137 mM NaCl and 2.7 mM KCl) at λ = 482 nm in an Epoch 2 Microplate spectrophotometer (Biotek). The results shown are the mean values (and standard deviation) of triplicate samples. The value of the Michaelis constant <i>K</i><sub>m</sub> for nitrocefin was calculated as previously described.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div><div class="NLM_p">The IC<sub>50</sub> values were calculated as the inhibitory concentration, resulting in 50% reduction of nitrocefin hydrolysis after 10 min of preincubation of the enzyme and inhibitor at 25 °C, as previously described.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p">The apparent <i>K</i><sub>i</sub> (<i>K</i><sub>i app</sub>) was obtained as a competitive inhibition constant in the presence of nitrocefin (≫ <i>K</i><sub>m</sub>). <i>K</i><sub>i app</sub> is an equilibrium constant that can be calculated from pre-steady-state conditions, yields an estimated affinity, and represents the formation of the acyl enzyme E-S.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Increasing concentrations of each compound were added to the enzyme and nitrocefin mixtures, and the initial velocities (<i>v</i><sub>0</sub>) were measured. Inverse initial velocities (1/<i>v</i><sub>0</sub>) were plotted against inhibitory concentrations ([<i>I</i>]), and the data were fitted to a linear equation, in which the <i>y</i> intercept divided by the slope of the line was defined as the <i>K</i><sub>i app</sub> observed. The initial velocity (<i>v</i><sub>0</sub>) was determined by <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>. The observed <i>K</i><sub>i app</sub> was then corrected for nitrocefin affinity by using <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>.<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_m001.gif" alt="" id="_i19" /></img><span class="labelSpan">(1)</span></span><span class="NLM_disp-formula" id="eq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_m002.gif" alt="" id="_i20" /></img><span class="labelSpan">(2)</span></span></div><div class="NLM_p">Inhibitors can be characterized by first-order rate constants for inhibition: (i) <i>k</i><sub>inact</sub>, the maximal rate of inactivation achieved with an “infinite” concentration of inactivator, indicates the number of molecules of enzyme that are inactivated per second; thus, the higher the <i>k</i><sub>inact</sub>, the faster the rate at which the inhibitor inhibits the enzyme; and (ii) <i>K</i><sub>I</sub> value, the concentration of inactivator which yields an inactivation rate that is half the value of <i>k</i><sub>inact</sub>. The <i>K</i><sub>I</sub> may or may not equal the equilibrium constant <i>K</i><sub>i</sub>, determined under pre-steady-state conditions.<a onclick="showRef(event, 'ref28 ref56'); return false;" href="javascript:void(0);" class="ref ref28 ref56">(28,56)</a> The <i>k</i><sub>obs</sub> values were determined by nonlinear least-squares data fitting, implemented in GraphPad software and using <a class="ref internalNav" href="#eq3" aria-label="eq 3">eq 3</a>.<span class="NLM_disp-formula" id="eq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_m003.gif" alt="" id="_i21" /></img><span class="labelSpan">(3)</span></span>where <i>A</i> is the absorbance, <i>v</i><sub>0</sub> is the initial velocity, <i>v</i><sub>f</sub> is the final velocity, and <i>x</i> is time. The <i>k</i><sub>obs</sub> values were plotted against inhibitory concentrations to determine <i>k</i><sub>inact</sub> and <i>K</i><sub>I</sub> with <a class="ref internalNav" href="#eq4" aria-label="eq 4">eq 4</a>. The <i>K</i><sub>I</sub> value was corrected using <a class="ref internalNav" href="#eq5" aria-label="eq 5">eq 5</a>.<span class="NLM_disp-formula" id="eq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_m004.gif" alt="" id="_i22" /></img><span class="labelSpan">(4)</span></span><span class="NLM_disp-formula" id="eq5"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_m005.gif" alt="" id="_i23" /></img><span class="labelSpan">(5)</span></span></div><div class="NLM_p">The apparent onset of acylation (<i>k</i><sub>2</sub>/<i>K</i>) was determined as the slope of the line obtained by plotting [<i>I</i>] against <i>k</i><sub>obs</sub>.<a onclick="showRef(event, 'ref29 ref58'); return false;" href="javascript:void(0);" class="ref ref29 ref58">(29,58)</a><i>k</i><sub>2</sub>/<i>K</i> was determined from <a class="ref internalNav" href="#eq6" aria-label="eq 6">eq 6</a>, and this value was corrected for nitrocefin affinity by using <a class="ref internalNav" href="#eq7" aria-label="eq 7">eq 7</a>.<span class="NLM_disp-formula" id="eq6"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_m006.gif" alt="" id="_i24" /></img><span class="labelSpan">(6)</span></span><span class="NLM_disp-formula" id="eq7"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_m007.gif" alt="" id="_i25" /></img><span class="labelSpan">(7)</span></span></div><div class="NLM_p">The turnover numbers (<i>t</i><sub><i>n</i></sub>), defined as the inhibitor/enzyme ratio (<i>I</i>/<i>E</i>) necessary for 90% inhibition of nitrocefin hydrolysis, were determined after 20 h of incubation with increasing concentrations of inhibitors and 10 nM enzymes, with various molar <i>I</i>/<i>E</i> ratios.<a onclick="showRef(event, 'ref29 ref56'); return false;" href="javascript:void(0);" class="ref ref29 ref56">(29,56)</a></div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Deacylation (<i>k</i><sub>off</sub>) Measurement</h4><div class="NLM_p">Purified PDC-1 at 1 μM was incubated with inhibitors <b>1–3</b> at concentrations in excess of the enzyme concentration, for 30 min at 37 °C. The incubation time was long enough to ensure 100% inhibition. The reaction mixture was then diluted 1000-fold in the reaction buffer, and 100 μL of diluted enzyme was mixed with 100 μL of nitrocefin at 200 μM and PDC-1 activity was assayed in a 96-well microtiter plate. Nitrocefin hydrolysis with free enzyme in the absence of the inhibitor was measured as a control. Absorbance at 482 nm was monitored every 20 s for 4 h at 37 °C. The resulting reaction progress curve was fitted to <a class="ref internalNav" href="#eq8" aria-label="eq 8">eq 8</a>, which includes the exponential function.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a><span class="NLM_disp-formula" id="eq8"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_m008.gif" alt="" id="_i27" /></img><span class="labelSpan">(8)</span></span>where <i>V</i><sub>0</sub> is the initial, completely inhibited enzyme velocity; <i>V</i><sub>s</sub> is the steady-state uninhibited enzyme velocity; and <i>t</i> is time.</div><div class="NLM_p last">Kinetics assays with these inhibitors were also performed using the metallo-β-lactamase VIM-1, a class B β-lactamase frequently produced by <i>P. aeruginosa</i>.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The <i>K</i><sub>i app</sub> parameter was calculated for avibactam and compounds <b>1–3</b> against the VIM-1 enzyme.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Susceptibility Testing of Antibiotics and Inhibitors</h3><div class="NLM_p last">The ceftazidime (Sigma-Genosys) MICs were determined by broth microdilution (gold standard) in the presence and absence of inhibitors <b>1–3</b> and avibactam (AstraZeneca) according to EUCAST criteria (Version 9.0).<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The inhibitors were used at concentrations of 4 and 16 mg/L. Turbidity was recorded in 96-well plates after 20 h of incubation at 37 °C in Mueller–Hinton II broth (Becton-Dickinson). The MICs were determined in three independent replicate assays.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Mass Spectrometry Studies</h3><div class="NLM_p last">A 100 μL solution of PDC-1 from <i>P. aeruginosa</i> (15 μL from a stock protein concentration of 1.1 mg/mL), in phosphate-buffered saline pH 7.4, was incubated with compounds <b>1–3</b> for 30 min. The enzyme/inhibitor molar ratio was 1/100. The activity was monitored by UV–Vis spectroscopy using aliquots from the incubation samples and the control. The enzyme was assayed by monitoring the increase in absorbance at 482 nm in the UV–Vis spectrum due to absorbance of the hydrolyzed nitrocefin (ε/M<sup>–1</sup> cm<sup>–1</sup> 15 900) in phosphate-buffered saline pH 7.4 at 25 °C. Each assay was initiated by addition of nitrocefin. No activity was observed after 30 min, except in the control. For MALDI analysis, the samples were concentrated and successively washed with 5 mM ammonium bicarbonate by centrifugation at 4 °C using Amicon centrifugal filters (Amicon Ultra-10). The samples were freeze-dried, diluted with 5 mM ammonium bicarbonate (5 μL), and analyzed by mass spectrometry in a MALDI TOF/TOF Mass Spectrometer (4800 Analyzer, AbSciex). The ProteoMass MALDI calibration kit (Merck) was used for calibration with sanipic acid as the matrix. Spectra were analyzed using the Data Explorer Software. The experiments were performed in triplicate.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> MD Simulation Studies of the PDC-1 Enzyme in the Unbound Form</h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Ligand Preparation</h4><div class="NLM_p last">The ligand geometries were minimized using a restricted Hartree-Fock (RHF) method and a 6-31G(d) basis set, as implemented in the ab initio program Gaussian 09.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Partial charges were derived by quantum mechanical calculations by using Gaussian 09, as implemented in the R.E.D. Server (version 3.0),<a onclick="showRef(event, 'ref63 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref63 ref64 ref65">(63−65)</a> according to the RESP<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> model. The covalent modified catalytic serine residue with ligands <b>1–3</b> were parameterized using the R.E.D. server, by adding acetyl and <i>N</i>-methyl caps and applying the same procedure used for ligands. RESP charge calculation for the modified serine residue considered the net charge for both acetyl and <i>N</i>-methyl caps to be zero. The missing bonded and nonbonded interaction parameters were assigned, by analogy or through interpolation, from those already present in the AMBER database (GAFF).<a onclick="showRef(event, 'ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref67 ref68">(67,68)</a></div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Minimization of the Unbound Forms</h4><div class="NLM_p last">The protein coordinates identified in the crystal structure of AmpC from <i>P. aeruginosa</i> PAO1 (PDC-1, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZB">4GZB</a>, 1.79 Å)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> were used. Computation of the protonation state of titratable groups at pH 7.0 was carried out using the H<sup>++</sup> Web server.<a onclick="showRef(event, 'ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref69 ref70">(69,70)</a> As a result of this analysis, His39, His92, His188, His239, His256, and His296 were protonated in the ε position. For the unbound form, the two possible protonation states for Lys67 (neutral and protonated forms) and Tyr150 (neutral and tyrosinate forms) were considered, with both residues either in their neutral or ionic forms. The tyrosinate anion was not included as a parameterized amino acid in the ff14SB<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> force field. Parameterization was carried out using the R.E.D. Server, with the same procedure described above for the modified serine residues. Addition of hydrogen atoms and molecular mechanics parameters from the ff14SB and GAFF force fields, respectively, were assigned to the variant proteins with the LEaP module of AMBER Tools 17. The proteins were immersed in a truncated octahedron of ∼13,500 TIP3P water molecules and neutralized by addition of sodium ions. The system was minimized in three stages: (a) minimization of the solvent and ions (5000 steps, first half with the steepest descent and the other half with the conjugate gradient); (b) minimization of the side chain residues, waters and ions (5000 steps, first half with the steepest descent and the other half with the conjugate gradient); (c) final minimization of the whole system (5000 steps, first half with the steepest descent and the other half with the conjugate gradient). A positional restraint force of 50 kcal mol<sup>–1</sup> Å<sup>–2</sup> was applied to the whole protein and α carbons during the first two stages (a–b), respectively.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Simulation of the Unbound Forms</h4><div class="NLM_p last">MD simulations were performed using the pmemd.cuda_SPFP<a onclick="showRef(event, 'ref72 ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref72 ref73 ref74">(72−74)</a> module in the AMBER 16 suite of programs. Periodic boundary conditions were applied, and electrostatic interactions were treated using the smooth particle mesh Ewald method,<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> with a grid spacing of 1 Å. The cut-off distance for the unbound interactions was 9 Å. The SHAKE algorithm<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> was applied to all bonds containing hydrogen using a tolerance of 10<sup>–5</sup> Å and an integration step of 2.0 fs. The minimized system was then heated at 300 K at 1 atm by increasing the temperature from 0 to 300 K over 100 ps and by maintaining the system at 300 K for another 100 ps. A positional restraint force of 50 kcal mol<sup>–1</sup> Å<sup>–2</sup> was applied to all α carbons during the heating stage. Finally, the system was equilibrated at constant volume (200 ps with positional restraints of 5 kcal mol<sup>–1</sup> Å<sup>–2</sup> to α carbons) and constant pressure (another 100 ps with positional restraints of 5 kcal mol<sup>–1</sup> Å<sup>–2</sup> to α carbons). The positional restraints were gradually reduced from 5 to 1 mol<sup>–1</sup> Å<sup>–2</sup> (5 steps, 100 ps each), and the resulting systems were allowed to equilibrate (100 ps) without further restraints. Unrestrained MD simulations were carried out for 200 ns. System coordinates were collected every 10 ps for further analysis. The molecular graphics program PyMOL<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> was used for visualization and depiction of enzyme structures. The cpptraj module in AMBER 16 was used to analyze the trajectories and to calculate the rmsd of the protein during the simulation.<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a></div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> MD Simulation Studies of the PDC-1/<b>1–3</b> Enzyme Binary Complexes</h3><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Docking Studies</h4><div class="NLM_p last">The neutral protonation states for Lys67 and Tyr150 were used. The binding mode of inhibitors <b>1–3</b> was first explored using GOLD software, version 5.8.1,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> for docking studies and the protein coordinates in PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZB">4GZB</a>. For the covalent docking, the enzyme coordinates of the unbound form after 190 ns of simulation were used. All crystallographic water molecules and ligands were removed for docking. The GOLD scoring function was applied. The ligand and the corresponding indolizine adduct geometries, which were previously minimized, were used as MOL2 files. Each ligand was docked in 25 independent genetic algorithm (GA) runs, and for each of these, a maximum of 100,000 GA operations was performed on a single population of 50 individuals. Operator weights for crossover, mutation and migration in the entry box were used as default parameters (95, 95, and 10, respectively) in addition to the hydrogen bonding (4.0 Å) and van der Waals (2.5 Å) parameters. The position of the catalytic serine side chain (Ser64) was used to define the docking region, and the radius was set at 8 Å.</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Minimization of the PDC-1/<b>1–3</b> Binary Enzyme Complexes and Adducts</h4><div class="NLM_p last">Ligand coordinates obtained by docking were used as a starting point for MD simulations of the corresponding enzyme binary complexes. The complexes immersed in a truncated octahedron of TIP3P water molecules and sodium ions were minimized in four stages: (1) initial minimization of the ligands (1000 steps, first half with the steepest descent and the other half with the conjugate gradient); steps (2), (3), and (4) were performed as steps (a), (b), and (c) for minimization of the unbound forms (see above). A positional restraint force of 50 kcal mol<sup>–1</sup> Å<sup>–2</sup> was applied to these unminimized atoms during the first three stages (1–3).</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Simulations of the PDC-1/<b>1–3</b> Binary Enzyme Complexes and Adducts</h4><div class="NLM_p last">MD simulations of the binary complexes were performed as indicated for the unbound protein forms (see above).</div></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> MD Simulation Studies of CMY-10/<b>1–3</b>, BER/<b>1–3</b>, OXA-48/<b>1–3</b>, and TEM-1/<b>1–3</b> Enzyme Binary Complexes and Adducts</h3><div class="NLM_p">The protein coordinates identified in the following crystal structures were used: (i) CMY-10 from <i>K. aerogenes</i> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K1D">5K1D</a>, 1.94 Å);<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> (ii) BER from <i>E. coli</i> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GZW">5GZW</a>, 1.49 Å, chain A);<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (iii) OXA-48 from <i>K. pneumoniae</i> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S2N">4S2N</a>, 2.00 Å, chain A);<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and (iv) TEM-1 from <i>E. coli</i> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ERM">1ERM</a>, 1.70 Å).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The studies were carried out as indicated for the PDC-1 enzyme (see above), but with the following specific modifications: (i) for the CMY-10 enzyme, residues His23, His39, His92, His147, His185, His186, His255, His258, and His350 were protonated at the ε position, and His160 was protonated at both δ and ε positions; (ii) for the BER enzyme, the residues His13, His108, His186, His210, His298, and His316 were protonated at the ε position, and the missing residues of H10 helix (287–296) were modeled using the ModLoop Server;<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> (iii) for the OXA-48 enzyme, His34, His38, His90, His109, His140, His178, and His182 were protonated at the ε position, and carboxylated Lys73 was prepared in a similar way as the covalently modified serine residue using the R.E.D. Server; and (iv) for the TEM-1 enzyme, His26, His96, His112, His153, His158 and His287 were protonated at the ε position, and Lys73 was considered protonated based on mechanistic considerations. For enzymes OXA-48 and TEM-1, the numbering described in the PDB files was used.</div><div class="NLM_p last">The vibrational modes for AmpC enzymes (BER, CMY-10, and PDC-1) were calculated by principal component analysis with the cpptraj module from the corresponding MD trajectories.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00369" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34530" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34530" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00369?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00369</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">MD simulation studies on the PDC-1 in the unbound form, additional poses of the ligands in the active site of PDC-1, RMSD plots for the protein backbone and ligand in the PDC-1/ligand enzyme binary complexes, the PDC-1/ligand enzyme binary complexes, the PDC-1/ligand enzyme adducts, HPLC-MS traces for compounds <b>1–3</b>, MIC values for ceftazidime against PAO1 Δ<i>ampD</i>, and MIC values for imipenem, ceftazidime, and ampicillin against bacterial strains harbouring class D (OXA-23, OXA-24/40, OXA-48), class A (TEM-1), and class C (CMY-2 and DHA-1) enzymes (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>1</b> adduct (PDB1) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>2</b> adduct (PDB2) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>3</b> adduct (PDB3) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>1</b> complex (PDB4) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_006.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>2</b> complex (PDB5) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_007.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>3</b> complex (PDB6) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_008.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the BER unbound form (PDB7) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_009.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the BER/<b>2</b> complex (PDB8) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_010.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the BER/<b>2</b> adduct (PDB9) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_011.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the BER unbound form (PDB10) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_012.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the CMY10/<b>2</b> complex (PDB11) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_013.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the CMY10/<b>2</b> adduct (PDB12) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_014.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the OXA-48/<b>2</b> complex (PDB13) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_015.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the OXA-48/<b>2</b> adduct (PDB14) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_016.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the TEM-1/<b>2</b> complex (PDB15) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_017.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the TEM-1/<b>2</b> adduct (PDB16) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_018.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf">jm1c00369_si_001.pdf (2.73 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_002.csv">jm1c00369_si_002.csv (3.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_003.pdb">jm1c00369_si_003.pdb (465.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_004.pdb">jm1c00369_si_004.pdb (465.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_005.pdb">jm1c00369_si_005.pdb (465.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_006.pdb">jm1c00369_si_006.pdb (465.29 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_007.pdb">jm1c00369_si_007.pdb (465.29 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_008.pdb">jm1c00369_si_008.pdb (465.29 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_009.pdb">jm1c00369_si_009.pdb (448.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_010.pdb">jm1c00369_si_010.pdb (452.55 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_011.pdb">jm1c00369_si_011.pdb (452.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_012.pdb">jm1c00369_si_012.pdb (434.62 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_013.pdb">jm1c00369_si_013.pdb (438.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_014.pdb">jm1c00369_si_014.pdb (438.78 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_015.pdb">jm1c00369_si_015.pdb (324.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_016.pdb">jm1c00369_si_016.pdb (324.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_017.pdb">jm1c00369_si_017.pdb (333.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_018.pdb">jm1c00369_si_018.pdb (333.24 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00369" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Concepción González-Bello</span> - <span class="hlFld-Affiliation affiliation">Centro
Singular de Investigación en Química Biolóxica
e Materiais Moleculares (CiQUS), Departamento de Química Orgánica, Universidade de Santiago de Compostela, Jenaro de la Fuente s/n, Santiago de Compostela 15782, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6439-553X" title="Orcid link">http://orcid.org/0000-0001-6439-553X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c9aaa6a7aaacb9aaa0a6a7e7aea6a7b3a8a5acb3e7abaca5a5a689bcbaaae7acba"><span class="__cf_email__" data-cfemail="25464a4b464055464c4a4b0b424a4b5f4449405f0b474049494a655056460b4056">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alejandro Beceiro</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología do Complejo Hospitalario Universitario da
Coruña (CHUAC), Instituto de Investigación Biomédica
da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6340-7815" title="Orcid link">http://orcid.org/0000-0002-6340-7815</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#db9ab7beb1bab5bfa9b4f599beb8beb2a9b4f598baa8baa89ba8bea9bcbaa8f5bea8"><span class="__cf_email__" data-cfemail="06476a636c676862746928446365636f746928456775677546756374616775286375">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Juan C. Vázquez-Ucha</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología do Complejo Hospitalario Universitario da
Coruña (CHUAC), Instituto de Investigación Biomédica
da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4949-0779" title="Orcid link">http://orcid.org/0000-0003-4949-0779</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Diana Rodríguez</span> - <span class="hlFld-Affiliation affiliation">Centro
Singular de Investigación en Química Biolóxica
e Materiais Moleculares (CiQUS), Departamento de Química Orgánica, Universidade de Santiago de Compostela, Jenaro de la Fuente s/n, Santiago de Compostela 15782, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cristina Lasarte-Monterrubio</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología do Complejo Hospitalario Universitario da
Coruña (CHUAC), Instituto de Investigación Biomédica
da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emilio Lence</span> - <span class="hlFld-Affiliation affiliation">Centro
Singular de Investigación en Química Biolóxica
e Materiais Moleculares (CiQUS), Departamento de Química Orgánica, Universidade de Santiago de Compostela, Jenaro de la Fuente s/n, Santiago de Compostela 15782, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jorge Arca-Suarez</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología do Complejo Hospitalario Universitario da
Coruña (CHUAC), Instituto de Investigación Biomédica
da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">María Maneiro</span> - <span class="hlFld-Affiliation affiliation">Centro
Singular de Investigación en Química Biolóxica
e Materiais Moleculares (CiQUS), Departamento de Química Orgánica, Universidade de Santiago de Compostela, Jenaro de la Fuente s/n, Santiago de Compostela 15782, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eva Gato</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología do Complejo Hospitalario Universitario da
Coruña (CHUAC), Instituto de Investigación Biomédica
da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Astrid Perez</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología do Complejo Hospitalario Universitario da
Coruña (CHUAC), Instituto de Investigación Biomédica
da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marta Martínez-Guitián</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología do Complejo Hospitalario Universitario da
Coruña (CHUAC), Instituto de Investigación Biomédica
da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlos Juan</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología y Unidad de Investigación, Hospital Universitario Son Espases, Instituto de Investigación
Sanitaria Illes Balears (IdiSBA), Carretera de Valldemossa, 79, Palma de Mallorca 07120, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antonio Oliver</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología y Unidad de Investigación, Hospital Universitario Son Espases, Instituto de Investigación
Sanitaria Illes Balears (IdiSBA), Carretera de Valldemossa, 79, Palma de Mallorca 07120, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">German Bou</span> - <span class="hlFld-Affiliation affiliation">Servicio
de Microbiología do Complejo Hospitalario Universitario da
Coruña (CHUAC), Instituto de Investigación Biomédica
da Coruña (CICA-INIBIC), Xubias de Arriba, 84, A Coruña 15006, Spain</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was funded by the National Plan for Scientific Research, Development and Technological Innovation 2013-2016 and funded by the ISCIII-General Subdirection of Assessment and Promotion of the Research-European Regional Development Fund (FEDER) “A way of making Europe” and operative program Intelligent Growth 2014-2020, under Projects PI17/01482 and PI20/01212 (A.B.) and PI15/00860 and PI18/00501 (G.B.), and by the Spanish Ministry of Economy and Competitiveness (PID2019-105512RB-I00 and SAF2016-75638-R, C.G.-B.). The study was also funded by GAIN—Agencia Gallega de Innovación—Consellería de Economía, Emprego e Industria (IN607A 2016/22, G.B.) and the Xunta de Galicia [ED431B 2018/04 and Centro singular de investigación de Galicia accreditation 2019-2022 (ED431G 2019/03), C.G.-B.]. The European Regional Development Fund (ERDF) is also gratefully acknowledged. J.C.V.-U. was financially supported by the ISCIII project FI18/00315. A.B. was financially supported by the Miguel Servet II program, CPII18/00024. J.A.-S. was financially supported by the Rio Hortega program (ISCIII, CM19/00219) and C.L.-M. by IN606A-2019/029 and M.M.-G. was financially supported by the Clara Roy Grant (SEIMC). D.R., E.L., and M.M. thank the Xunta de Galicia for their respective predoctoral and postdoctoral fellowships.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">Dedicated to the memory of Prof. Chris Abell.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55251" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55251" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">All authors are grateful to the Centro de Supercomputación de Galicia (CESGA) for use of the Finis Terrae computer.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">DBO</td><td class="NLM_def"><p class="first last">1,6-diazabicyclo[3.2.1]octane</p></td></tr><tr><td class="NLM_term">MDR</td><td class="NLM_def"><p class="first last">multidrug-resistant</p></td></tr><tr><td class="NLM_term">PBPs</td><td class="NLM_def"><p class="first last">penicillin-binding proteins</p></td></tr><tr><td class="NLM_term">PDC</td><td class="NLM_def"><p class="first last"><i>Pseudomonas</i>-derived cephalosporinase</p></td></tr><tr><td class="NLM_term">XDR</td><td class="NLM_def"><p class="first last">extensively drug-resistant.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23035" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23035" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 80 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raudonis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span> <span> </span><span class="NLM_article-title">Antibiotic resistance in <i>Pseudomonas aeruginosa</i>: mechanisms and alternative therapeutic strategies</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2018.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.biotechadv.2018.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=30500353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVSgur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=177-192&author=Z.+Pangauthor=R.+Raudonisauthor=B.+R.+Glickauthor=T.-J.+Linauthor=Z.+Cheng&title=Antibiotic+resistance+in+Pseudomonas+aeruginosa%3A+mechanisms+and+alternative+therapeutic+strategies&doi=10.1016%2Fj.biotechadv.2018.11.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic resistance in Pseudomonas aeruginosa: mechanisms and alternative therapeutic strategies</span></div><div class="casAuthors">Pang, Zheng; Raudonis, Renee; Glick, Bernard R.; Lin, Tong-Jun; Cheng, Zhenyu</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-192</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is an opportunistic pathogen that is a leading cause of morbidity and mortality in cystic fibrosis patients and immunocompromised individuals.  Eradication of P. aeruginosa has become increasingly difficult due to its remarkable capacity to resist antibiotics.  Strains of Pseudomonas aeruginosa are known to utilize their high levels of intrinsic and acquired resistance mechanisms to counter most antibiotics.  In addn., adaptive antibiotic resistance of P. aeruginosa is a recently characterized mechanism, which includes biofilm-mediated resistance and formation of multidrug-tolerant persister cells, and is responsible for recalcitrance and relapse of infections.  The discovery and development of alternative therapeutic strategies that present novel avenues against P. aeruginosa infections are increasingly demanded and gaining more and more attention.  Although mostly at the preclin. stages, many recent studies have reported several innovative therapeutic technologies that have demonstrated pronounced effectiveness in fighting against drug-resistant P. aeruginosa strains.  This review highlights the mechanisms of antibiotic resistance in P. aeruginosa and discusses the current state of some novel therapeutic approaches for treatment of P. aeruginosa infections that can be further explored in clin. practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_z6YpDKCQnLVg90H21EOLACvtfcHk0lgm4ymzyYTu3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVSgur%252FO&md5=9edf67e71c2362f36a08daa6174b9653</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2018.11.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2018.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DZ.%26aulast%3DRaudonis%26aufirst%3DR.%26aulast%3DGlick%26aufirst%3DB.%2BR.%26aulast%3DLin%26aufirst%3DT.-J.%26aulast%3DCheng%26aufirst%3DZ.%26atitle%3DAntibiotic%2520resistance%2520in%2520Pseudomonas%2520aeruginosa%253A%2520mechanisms%2520and%2520alternative%2520therapeutic%2520strategies%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2019%26volume%3D37%26spage%3D177%26epage%3D192%26doi%3D10.1016%2Fj.biotechadv.2018.11.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horcajada, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorlí, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luque, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Zorrilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benito, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grau, S.</span></span> <span> </span><span class="NLM_article-title">Epidemiology and Treatment of Multidrug-Resistant and Extensively Drug-Resistant Pseudomonas aeruginosaInfections</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>32</i></span>, <span class="NLM_elocation-id">e00031</span> <span class="refDoi"> DOI: 10.1128/cmr.00031-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FCMR.00031-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31462403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmt1WksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2019&author=J.+P.+Horcajadaauthor=M.+Monteroauthor=A.+Oliverauthor=L.+Sorl%C3%ADauthor=S.+Luqueauthor=S.+G%C3%B3mez-Zorrillaauthor=N.+Benitoauthor=S.+Grau&title=Epidemiology+and+Treatment+of+Multidrug-Resistant+and+Extensively+Drug-Resistant+Pseudomonas+aeruginosaInfections&doi=10.1128%2Fcmr.00031-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Epidemiology and treatment of multidrug-resistant and extensively drug-resistant Pseudomonas aeruginosa infections</span></div><div class="casAuthors">Horcajada, Juan P.; Montero, Milagro; Oliver, Antonio; Sorli, Luisa; Luque, Sonia; Gomez-Zorrilla, Silvia; Benito, Natividad; Grau, Santiago</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e00031-19/1-e00031-19/52</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">1098-6618</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In recent years, the worldwide spread of the so-called high-risk clones of multidrug-resistant or extensively drug-resistant (MDR/XDR) Pseudomonas aeruginosa has become a public health threat.  This article reviews their mechanisms of resistance, epidemiol., and clin. impact and current and upcoming therapeutic options.  In vitro and in vivo treatment studies and pharmacokinetic and pharmacodynamic (PK/PD) models are discussed.  Polymyxins are reviewed as an important therapeutic option, outlining dosage, pharmacokinetics and pharmacodynamics, and their clin. efficacy against MDR/XDR P. aeruginosa infections.  Their narrow therapeutic window and potential for combination therapy are also discussed.  Other "old" antimicrobials, such as certain β-lactams, aminoglycosides, and fosfomycin, are reviewed here.  New antipseudomonals, as well as those in the pipeline, are also reviewed.  Ceftolozane-tazobactam has clin. activity against a significant percentage of MDR/XDR P. aeruginosa strains, and its microbiol. and clin. data, as well as recommendations for improving its use against these bacteria, are described, as are those for ceftazidimeavibactam, which has better activity against MDR/XDR P. aeruginosa, esp. strains with certain specific mechanisms of resistance.  A section is devoted to reviewing upcoming active drugs such as imipenem-relebactam, cefepimezidebactam, cefiderocol, and murepavadin.  Finally, other therapeutic strategies, such as use of vaccines, antibodies, bacteriocins, anti-quorum sensing, and bacteriophages, are described as future options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof3LWnt3z50bVg90H21EOLACvtfcHk0lgm4ymzyYTu3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmt1WksLY%253D&md5=49103d756d07d4761df425f326a23f56</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FCMR.00031-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00031-19%26sid%3Dliteratum%253Aachs%26aulast%3DHorcajada%26aufirst%3DJ.%2BP.%26aulast%3DMontero%26aufirst%3DM.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DSorl%25C3%25AD%26aufirst%3DL.%26aulast%3DLuque%26aufirst%3DS.%26aulast%3DG%25C3%25B3mez-Zorrilla%26aufirst%3DS.%26aulast%3DBenito%26aufirst%3DN.%26aulast%3DGrau%26aufirst%3DS.%26atitle%3DEpidemiology%2520and%2520Treatment%2520of%2520Multidrug-Resistant%2520and%2520Extensively%2520Drug-Resistant%2520Pseudomonas%2520aeruginosaInfections%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2019%26volume%3D32%26doi%3D10.1128%2Fcmr.00031-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K.</span></span> <span> </span><span class="NLM_article-title"><i>Pseudomonas aeruginosa:</i> evolution of antimicrobial resistance and implications for therapy</span>. <i>Semin. Respir. Crit. Care Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">044</span>– <span class="NLM_lpage">055</span>, <span class="refDoi"> DOI: 10.1055/s-0034-1396907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1055%2Fs-0034-1396907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25643270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BC2MritFOjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=044-055&author=K.+McCarthy&title=Pseudomonas+aeruginosa%3A+evolution+of+antimicrobial+resistance+and+implications+for+therapy&doi=10.1055%2Fs-0034-1396907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Pseudomonas aeruginosa: evolution of antimicrobial resistance and implications for therapy</span></div><div class="casAuthors">McCarthy Kate</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in respiratory and critical care medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-55</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Pseudomonas aeruginosa is a formidable pathogen in the infection arena.  It is able to easily adapt to the environment which it inhabits and can also colonize and invade the human host to cause serious infections.  In 2011, it was responsible for 7.1% of all health care-associated infection in the United States.  The morbidity and mortality of both blood stream infections and ventilator-associated pneumonia are significant.  On a global scale, we have seen the development of not only multidrug resistance but also extensive and pan drug resistance in this organism.  This is often associated with limited clonal types of which we now have epidemiological evidence of spread.  With this has come reduced antibiotic treatment options.  Consideration of antibiotic infusions, combination therapy, and inhalational therapy has occurred in an attempt to gain the upper ground.  Gram-negative resistance has appropriately been described as a global emergency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXX0ejtCSESuQ4Pl9hzjQhfW6udTcc2eaFUdOu0YqAj7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MritFOjsg%253D%253D&md5=f23bfb68d827f861fcc01edbcb42de95</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1055%2Fs-0034-1396907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0034-1396907%26sid%3Dliteratum%253Aachs%26aulast%3DMcCarthy%26aufirst%3DK.%26atitle%3DPseudomonas%2520aeruginosa%253A%2520evolution%2520of%2520antimicrobial%2520resistance%2520and%2520implications%2520for%2520therapy%26jtitle%3DSemin.%2520Respir.%2520Crit.%2520Care%2520Med.%26date%3D2015%26volume%3D36%26spage%3D044%26epage%3D055%26doi%3D10.1055%2Fs-0034-1396907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulet, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Causapé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span> <span> </span><span class="NLM_article-title">The increasing threat of <i>Pseudomonas aeruginosa</i> high-risk clones</span>. <i>Drug Resist. Updates</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21-22</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/j.drup.2015.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.drup.2015.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=26304792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BC287lvVKktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21-22&publication_year=2015&pages=41-59&author=A.+Oliverauthor=X.+Muletauthor=C.+L%C3%B3pez-Causap%C3%A9author=C.+Juan&title=The+increasing+threat+of+Pseudomonas+aeruginosa+high-risk+clones&doi=10.1016%2Fj.drup.2015.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The increasing threat of Pseudomonas aeruginosa high-risk clones</span></div><div class="casAuthors">Oliver Antonio; Mulet Xavier; Lopez-Causape Carla; Juan Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Drug resistance updates : reviews and commentaries in antimicrobial and anticancer chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21-22</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-59</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The increasing prevalence of chronic and hospital-acquired infections produced by multidrug-resistant (MDR) or extensively drug-resistant (XDR) Pseudomonas aeruginosa strains is associated with significant morbidity and mortality.  This growing threat results from the extraordinary capacity of this pathogen for developing resistance through chromosomal mutations and from the increasing prevalence of transferable resistance determinants, particularly those encoding carbapenemases or extended-spectrum β-lactamases (ESBLs).  P. aeruginosa has a nonclonal epidemic population structure, composed of a limited number of widespread clones which are selected from a background of a large quantity of rare and unrelated genotypes that are recombining at high frequency.  Indeed, recent concerning reports have provided evidence of the existence of MDR/XDR global clones, denominated high-risk clones, disseminated in hospitals worldwide; ST235, ST111, and ST175 are likely those more widespread.  Noteworthy, the vast majority of infections by MDR, and specially XDR, strains are produced by these and few other clones worldwide.  Moreover, the association of high-risk clones, particularly ST235, with transferable resistance is overwhelming; nearly 100 different horizontally-acquired resistance elements and up to 39 different acquired β-lactamases have been reported so far among ST235 isolates.  Likewise, MDR internationally-disseminated epidemic strains, such as the Liverpool Epidemic Strain (LES, ST146), have been noted as well among cystic fibrosis patients.  Here we review the population structure, epidemiology, antimicrobial resistance mechanisms and virulence of the P. aeruginosa high-risk clones.  The phenotypic and genetic factors potentially driving the success of high-risk clones, the aspects related to their detection in the clinical microbiology laboratory and the implications for infection control and public health are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSq_r_KUrcD_kNHe_gFldTbfW6udTcc2eaFUdOu0YqAj7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287lvVKktQ%253D%253D&md5=f517cb617d315e9c66ea6a7bd876a93f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2015.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2015.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DMulet%26aufirst%3DX.%26aulast%3DL%25C3%25B3pez-Causap%25C3%25A9%26aufirst%3DC.%26aulast%3DJuan%26aufirst%3DC.%26atitle%3DThe%2520increasing%2520threat%2520of%2520Pseudomonas%2520aeruginosa%2520high-risk%2520clones%26jtitle%3DDrug%2520Resist.%2520Updates%26date%3D2015%26volume%3D21-22%26spage%3D41%26epage%3D59%26doi%3D10.1016%2Fj.drup.2015.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hernando-Amado, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coque, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baquero, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez, J. L.</span></span> <span> </span><span class="NLM_article-title">Defining and combating antibiotic resistance from one health and global health perspectives</span>. <i>Nat. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1038/s41564-019-0503-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1038%2Fs41564-019-0503-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31439928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1aqsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=1432-1442&author=S.+Hernando-Amadoauthor=T.+M.+Coqueauthor=F.+Baqueroauthor=J.+L.+Mart%C3%ADnez&title=Defining+and+combating+antibiotic+resistance+from+one+health+and+global+health+perspectives&doi=10.1038%2Fs41564-019-0503-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Defining and combating antibiotic resistance from One Health and Global Health perspectives</span></div><div class="casAuthors">Hernando-Amado, Sara; Coque, Teresa M.; Baquero, Fernando; Martinez, Jose L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1432-1442</span>CODEN:
                <span class="NLM_cas:coden">NMAICH</span>;
        ISSN:<span class="NLM_cas:issn">2058-5276</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Several interconnected human, animal and environmental habitats can contribute to the emergence, evolution and spread of antibiotic resistance, and the health of these contiguous habitats (the focus of the One Health approach) may represent a risk to human health.  Addnl., the expansion of resistant clones and antibiotic resistance determinants among human-assocd., animal-assocd. and environmental microbiomes have the potential to alter bacterial population genetics at local and global levels, thereby modifying the structure, and eventually the productivity, of microbiomes where antibiotic-resistant bacteria can expand.  Conversely, any change in these habitats (including pollution by antibiotics or by antibiotic-resistant organisms) may influence the structures of their assocd. bacterial populations, which might affect the spread of antibiotic resistance to, and among, the above-mentioned microbiomes.  Besides local transmission among connected habitats-the focus of studies under the One Health concept-the transmission of resistant microorganisms might occur on a broader (even worldwide) scale, requiring coordinated Global Health actions.  This Review provides updated information on the elements involved in the evolution and spread of antibiotic resistance at local and global levels, and proposes studies to be performed and strategies to be followed that may help reduce the burden of antibiotic resistance as well as its impact on human and planetary health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZUx4oYBVjtrVg90H21EOLACvtfcHk0ljfSJVugiim7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1aqsrfO&md5=9c15fac3a8837042a2b6afda14b9065a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41564-019-0503-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41564-019-0503-9%26sid%3Dliteratum%253Aachs%26aulast%3DHernando-Amado%26aufirst%3DS.%26aulast%3DCoque%26aufirst%3DT.%2BM.%26aulast%3DBaquero%26aufirst%3DF.%26aulast%3DMart%25C3%25ADnez%26aufirst%3DJ.%2BL.%26atitle%3DDefining%2520and%2520combating%2520antibiotic%2520resistance%2520from%2520one%2520health%2520and%2520global%2520health%2520perspectives%26jtitle%3DNat.%2520Microbiol.%26date%3D2019%26volume%3D4%26spage%3D1432%26epage%3D1442%26doi%3D10.1038%2Fs41564-019-0503-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span> </span><span class="NLM_article-title">Prioritization of Pathogens to Guide Discovery,
Research and Development
of New Antibiotics for Drug Resistant Bacterial Infections, Including
Tuberculosis: WHO/EMP/IAU/2017.12</span>. <a href="https://www.who.int/medicines/areas/rational_use/prioritization-of-pathogens/en/" class="extLink">https://www.who.int/medicines/areas/rational_use/prioritization-of-pathogens/en/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Prioritization+of+Pathogens+to+Guide+Discovery%2C%0AResearch+and+Development%0Aof+New+Antibiotics+for+Drug+Resistant+Bacterial+Infections%2C+Including%0ATuberculosis%3A+WHO%2FEMP%2FIAU%2F2017.12.+https%3A%2F%2Fwww.who.int%2Fmedicines%2Fareas%2Frational_use%2Fprioritization-of-pathogens%2Fen%2F+%28accessed+Feb+13%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DPrioritization%2520of%2520Pathogens%2520to%2520Guide%2520Discovery%252C%250AResearch%2520and%2520Development%250Aof%2520New%2520Antibiotics%2520for%2520Drug%2520Resistant%2520Bacterial%2520Infections%252C%2520Including%250ATuberculosis%253A%2520WHO%252FEMP%252FIAU%252F2017.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poole, K.</span></span> <span> </span><span class="NLM_article-title">Resistance to β-lactam antibiotics</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2200</span>– <span class="NLM_lpage">2223</span>, <span class="refDoi"> DOI: 10.1007/s00018-004-4060-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1007%2Fs00018-004-4060-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=15338052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD2cXovVyhtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2004&pages=2200-2223&author=K.+Poole&title=Resistance+to+%CE%B2-lactam+antibiotics&doi=10.1007%2Fs00018-004-4060-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to β-lactam antibiotics</span></div><div class="casAuthors">Poole, K.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2200-2223</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  β-Lactams have a long history in the treatment of infectious diseases, though their use was and continues to be confounded by the development of resistance in target organisms.  β-Lactamases, particularly in Gram-neg. pathogens, are a major determinant of this resistance, although alterations in the β-lactam targets, the penicillin-binding proteins (PBPs), are also important, esp. in Gram-pos. pathogens.  Mechanisms for the efflux and/or exclusion of these agents also contribute, though often in conjunction these other 2 .  Approaches for overcoming these resistance mechanisms include the development of novel β-lactamase-stable β-lactams, β-lactamase inhibitors to be employed with existing β-lactams, β-lactam compds. that bind strongly to low-affinity PBPs and agents that potentiate the activity of existing β-lactams against low-affinity PBP-producing organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ8MYoaxBP9bVg90H21EOLACvtfcHk0ljfSJVugiim7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXovVyhtbs%253D&md5=f107b3e35c8386ef2986f01d35dd3f36</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00018-004-4060-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-004-4060-9%26sid%3Dliteratum%253Aachs%26aulast%3DPoole%26aufirst%3DK.%26atitle%3DResistance%2520to%2520%25CE%25B2-lactam%2520antibiotics%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2004%26volume%3D61%26spage%3D2200%26epage%3D2223%26doi%3D10.1007%2Fs00018-004-4060-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocampo-Sosa, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macia, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamorano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suárez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peña, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martínez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">REIPI</span></span> <span> </span><span class="NLM_article-title">Overexpression of AmpC and efflux pumps in <i>Pseudomonas aeruginosa</i> isolates from bloodstream infections: prevalence and impact on resistance in a Spanish multicenter study</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1906</span>– <span class="NLM_lpage">1911</span>, <span class="refDoi"> DOI: 10.1128/aac.01645-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01645-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=21357294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCls7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=1906-1911&author=G.+Cabotauthor=A.+A.+Ocampo-Sosaauthor=F.+Tubauauthor=M.+D.+Maciaauthor=C.+Rodr%C3%ADguezauthor=B.+Moyaauthor=L.+Zamoranoauthor=C.+Su%C3%A1rezauthor=C.+Pe%C3%B1aauthor=L.+Mart%C3%ADnez-Mart%C3%ADnezauthor=A.+Oliverauthor=+REIPI&title=Overexpression+of+AmpC+and+efflux+pumps+in+Pseudomonas+aeruginosa+isolates+from+bloodstream+infections%3A+prevalence+and+impact+on+resistance+in+a+Spanish+multicenter+study&doi=10.1128%2Faac.01645-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of AmpC and efflux pumps in Pseudomonas aeruginosa isolates from bloodstream infections: prevalence and impact on resistance in a Spanish multicenter study</span></div><div class="casAuthors">Cabot, Gabriel; Ocampo-Sosa, Alain A.; Tubau, Fe; Macia, Maria D.; Rodriguez, Cristina; Moya, Bartolome; Zamorano, Laura; Suarez, Cristina; Pena, Carmen; Martinez-Martinez, Luis; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1906-1911</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The prevalence and impact of the overexpression of AmpC and efflux pumps were evaluated with a collection of 190 Pseudomonas aeruginosa isolates recovered from bloodstream infections in a 2008 multicenter study (10 hospitals) in Spain.  The MICs of a panel of 13 antipseudomonal agents were detd. by microdilution, and the expressions of ampC, mexB, mexY, mexD, and mexF were detd. by real-time reverse transcription (RT)-PCR.  Up to 39% of the isolates overexpressed at least one of the mechanisms.  The ampC overexpression (24.2%) was the most prevalent mechanism, followed by mexY (13.2%), mexB (12.6%), mexF (4.2%), and mexD (2.2%).  The overexpression of mexB plus mexY, documented for 5.3% of the isolates, was the only combination showing a significantly (P = 0.02) higher prevalence than expected from the frequencies of the individual mechanisms (1.6%).  Addnl., all imipenem-resistant isolates studied (25 representative isolates) showed inactivating mutations in oprD.  Most of the isolates nonsusceptible to piperacillin-tazobactam (96%) and ceftazidime (84%) overexpressed ampC, while mexB (25%) and mexY (29%) overexpressions gained relevance among cefepime-nonsusceptible isolates.  Nevertheless, the prevalence of mexY overexpression was highest among tobramycin-nonsusceptible isolates (37%), and that of mexB was highest among meropenem-nonsusceptible isolates (33%).  Regarding ciprofloxacin-resistant isolates, besides the expected increased prevalence of efflux pump overexpression, a highly significant link to ampC overexpression was documented for the first time: up to 52% of ciprofloxacin-nonsusceptible isolates overexpressed ampC, sharply contrasting with the 24% documented for the complete collection (P < 0.001).  Thus, mutation-driven resistance was frequent in P. aeruginosa isolates from bloodstream infections, whereas metallo-β-lactamases, detected in 2 isolates (1%) producing VIM-2, although with increasing prevalences, were still uncommon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql28rd0pn3KrVg90H21EOLACvtfcHk0ljfSJVugiim7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCls7bO&md5=8f993e34c110e6b0c08d9a97e827e76a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2FAAC.01645-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01645-10%26sid%3Dliteratum%253Aachs%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DOcampo-Sosa%26aufirst%3DA.%2BA.%26aulast%3DTubau%26aufirst%3DF.%26aulast%3DMacia%26aufirst%3DM.%2BD.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DC.%26aulast%3DMoya%26aufirst%3DB.%26aulast%3DZamorano%26aufirst%3DL.%26aulast%3DSu%25C3%25A1rez%26aufirst%3DC.%26aulast%3DPe%25C3%25B1a%26aufirst%3DC.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADnez%26aufirst%3DL.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DREIPI%26atitle%3DOverexpression%2520of%2520AmpC%2520and%2520efflux%2520pumps%2520in%2520Pseudomonas%2520aeruginosa%2520isolates%2520from%2520bloodstream%2520infections%253A%2520prevalence%2520and%2520impact%2520on%2520resistance%2520in%2520a%2520Spanish%2520multicenter%2520study%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2011%26volume%3D55%26spage%3D1906%26epage%3D1911%26doi%3D10.1128%2Faac.01645-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berrazeg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeannot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ntsogo Enguéné, V. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broutin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeffert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plésiat, P.</span></span> <span> </span><span class="NLM_article-title">Mutations in β-Lactamase AmpC Increase Resistance of Pseudomonas aeruginosa Isolates to Antipseudomonal Cephalosporins</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6248</span>– <span class="NLM_lpage">6255</span>, <span class="refDoi"> DOI: 10.1128/aac.00825-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.00825-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=26248364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yhurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=6248-6255&author=M.+Berrazegauthor=K.+Jeannotauthor=V.+Y.+Ntsogo+Engu%C3%A9n%C3%A9author=I.+Broutinauthor=S.+Loeffertauthor=D.+Fournierauthor=P.+Pl%C3%A9siat&title=Mutations+in+%CE%B2-Lactamase+AmpC+Increase+Resistance+of+Pseudomonas+aeruginosa+Isolates+to+Antipseudomonal+Cephalosporins&doi=10.1128%2Faac.00825-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in β-lactamase AmpC increase resistance of Pseudomonas aeruginosa isolates to antipseudomonal cephalosporins</span></div><div class="casAuthors">Berrazeg, M.; Jeannot, K.; Enguene, Veronique Yvette Ntsogo; Broutin, I.; Loeffert, S.; Fournier, D.; Plesiat, P.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6248-6255</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Mutation-dependent overprodn. of intrinsic β-lactamase AmpC is considered the main cause of resistance of clin. strains of Pseudomonas aeruginosa to antipseudomonal penicillins and cephalosporins.  Anal. of 31 AmpC-overproducing clin. isolates exhibiting a greater resistance to ceftazidime than to piperacillin-tazobactam revealed 17 mutations in the β-lactamase, combined with various polymorphic amino acid substitutions.  When overexpressed in AmpC-deficient P. aeruginosa 4098, the genes coding for 20/23 of these AmpC variants confer a higher (2-fold to >64-fold) resistance to ceftazidime and ceftolozane-tazobactam than did the gene from ref. strain PAO1.  The mutations had variable effects on the MICs of ticarcillin, piperacillin-tazobactam, aztreonam, and cefepime.  Depending on their location in the AmpC structure and their impact on β-lactam MICs, they could be assigned to 4 distinct groups.  Most of the mutations affecting the omega loop, the R2 domain, and the C-terminal end of the protein were shared with extended-spectrum AmpCs (ESACs) from other Gram-neg. species.  Two new mutations (F121L and P154L) were predicted to enlarge the substrate binding pocket by disrupting the stacking between residues F121 and P154.  Also the reported ESACs emerged locally in a variety of clones, some of which are epidemic and did not require hypermutability.  Taken together, the authors' results show that P. aeruginosa is able to adapt to efficacious β-lactams, including the newer cephalosporin ceftolozane, through a variety of mutations affecting its intrinsic β-lactamase, AmpC.  Data suggest that the rates of ESAC-producing mutants are ≥1.5% in the clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ1K5kAOUjxLVg90H21EOLACvtfcHk0ljKF235pN0M4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yhurw%253D&md5=6d196195beee010e7618c8b84e09f0fc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FAAC.00825-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00825-15%26sid%3Dliteratum%253Aachs%26aulast%3DBerrazeg%26aufirst%3DM.%26aulast%3DJeannot%26aufirst%3DK.%26aulast%3DNtsogo%2BEngu%25C3%25A9n%25C3%25A9%26aufirst%3DV.%2BY.%26aulast%3DBroutin%26aufirst%3DI.%26aulast%3DLoeffert%26aufirst%3DS.%26aulast%3DFournier%26aufirst%3DD.%26aulast%3DPl%25C3%25A9siat%26aufirst%3DP.%26atitle%3DMutations%2520in%2520%25CE%25B2-Lactamase%2520AmpC%2520Increase%2520Resistance%2520of%2520Pseudomonas%2520aeruginosa%2520Isolates%2520to%2520Antipseudomonal%2520Cephalosporins%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D6248%26epage%3D6255%26doi%3D10.1128%2Faac.00825-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Nicolau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Diversity and regulation of intrinsic β-lactamases from non-fermenting and other Gram-negative opportunistic pathogens</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">781</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1093/femsre/fux043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Ffemsre%2Ffux043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=29029112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFWmt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2017&pages=781-815&author=C.+Juanauthor=G.+Torrensauthor=M.+Gonz%C3%A1lez-Nicolauauthor=A.+Oliver&title=Diversity+and+regulation+of+intrinsic+%CE%B2-lactamases+from+non-fermenting+and+other+Gram-negative+opportunistic+pathogens&doi=10.1093%2Ffemsre%2Ffux043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Diversity and regulation of intrinsic β-lactamases from non-fermenting and other Gram-negative opportunistic pathogens</span></div><div class="casAuthors">Juan, Carlos; Torrens, Gabriel; Gonzalez-Nicolau, Mar; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">781-815</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">1574-6976</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">This review deeply addresses for the first time the diversity, regulation and mechanisms leading to mutational overexpression of intrinsic β-lactamases from non-fermenting and other non-Enterobacteriaceae Gram-neg. opportunistic pathogens.  After a general overview of the intrinsic β-lactamases described so far in these microorganisms, including circa. 60 species and 100 different enzymes, we review the wide array of regulatory pathways of these β-lactamases.  They include diverse LysR-type regulators, which control the expression of β-lactamases from relevant nosocomial pathogens such as Pseudomonas aeruginosa or Stenothrophomonas maltophilia or two-component regulators, with special relevance in Aeromonas spp., along with other pathways.  Likewise, the multiple mutational mechanisms leading to β-lactamase overexpression and β-lactam resistance development, including AmpD (N-acetyl-muramyl-L-alanine amidase), DacB (PBP4), MrcA (PPBP1A) and other PBPs, BlrAB (two-component regulator) or several lytic transglycosylases among others, are also described.  Moreover, we address the growing evidence of a major interplay between β-lactamase regulation, peptidoglycan metab. and virulence.  Finally, we analyze recent works showing that blocking of peptidoglycan recycling (such as inhibition of NagZ or AmpG) might be useful to prevent and revert β-lactam resistance.  Altogether, the provided information and the identified gaps should be valuable for guiding future strategies for combating multidrug-resistant Gram-neg. pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvR40C1kf9YrVg90H21EOLACvtfcHk0ljKF235pN0M4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFWmt7fF&md5=2f67e56b3ccd3db2e76972aeb8f8c81b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Ffemsre%2Ffux043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ffemsre%252Ffux043%26sid%3Dliteratum%253Aachs%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DTorrens%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Nicolau%26aufirst%3DM.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DDiversity%2520and%2520regulation%2520of%2520intrinsic%2520%25CE%25B2-lactamases%2520from%2520non-fermenting%2520and%2520other%2520Gram-negative%2520opportunistic%2520pathogens%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2017%26volume%3D41%26spage%3D781%26epage%3D815%26doi%3D10.1093%2Ffemsre%2Ffux043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathlouthi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Bayssari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolain, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chouchani, C.</span></span> <span> </span><span class="NLM_article-title">RETRACTED ARTICLE: Prevalence and emergence of carbapenemases-producing Gram-negative bacteria in Mediterranean basin</span>. <i>Crit. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.3109/1040841x.2016.1160867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.3109%2F1040841X.2016.1160867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=27387224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFOhurbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2017&pages=43-61&author=N.+Mathlouthiauthor=C.+Al-Bayssariauthor=S.+Bakourauthor=J.+M.+Rolainauthor=C.+Chouchani&title=RETRACTED+ARTICLE%3A+Prevalence+and+emergence+of+carbapenemases-producing+Gram-negative+bacteria+in+Mediterranean+basin&doi=10.3109%2F1040841x.2016.1160867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence and emergence of carbapenemases-producing Gram-negative bacteria in Mediterranean basin</span></div><div class="casAuthors">Mathlouthi, Najla; Al-Bayssari, Charbel; Bakour, Sofiane; Rolain, Jean Marc; Chouchani, Chedly</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-61</span>CODEN:
                <span class="NLM_cas:coden">CRVMAC</span>;
        ISSN:<span class="NLM_cas:issn">1040-841X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The emergence and the global spread of carbapenemases concern to health services worldwide.  Their celestial rise among Gram-neg. bacilli has challenged both the scientific and pharmaceutical sectors.  Indeed, infections caused by these bacteria have limited treatment options and have been assocd. with high mortality and morbidity rates.  Carbapenemase producers are mainly identified among Klebsiella pneumoniae, Escherichia coli, Pseudomonas aeruginosa, and Acinetobacter baumannii and still mostly in hospital settings and rarely in the community.  They are closely related to KPC, VIM, IMP, NDM, and OXA-48 types.  The encoding genes are mostly plasmid located and assocd. with various mobile genetic elements.  The Mediterranean area is of interest due to a great diversity and population mixing.  The prevalence of carbapenemases is particularly high and variant among countries, partially depending on the population exchange relationship between the regions and the possible reservoirs of each carbapenemase.  This review described the epidemiol. of carbapenemases in this region of the world highlighting the worrisome situation and the need to screen and detect these enzymes to prevent and control their dissemination esp. as it is clear that very few novel antibiotics will be introduced in the next few years, making the dissemination of carbapenem-resistant Gram-neg. bacteria of crucial importance worldwide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjgom8TLXe_LVg90H21EOLACvtfcHk0ljKF235pN0M4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFOhurbP&md5=09d8dd20738b6023fc095f0f11b22d1f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3109%2F1040841X.2016.1160867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F1040841X.2016.1160867%26sid%3Dliteratum%253Aachs%26aulast%3DMathlouthi%26aufirst%3DN.%26aulast%3DAl-Bayssari%26aufirst%3DC.%26aulast%3DBakour%26aufirst%3DS.%26aulast%3DRolain%26aufirst%3DJ.%2BM.%26aulast%3DChouchani%26aufirst%3DC.%26atitle%3DRETRACTED%2520ARTICLE%253A%2520Prevalence%2520and%2520emergence%2520of%2520carbapenemases-producing%2520Gram-negative%2520bacteria%2520in%2520Mediterranean%2520basin%26jtitle%3DCrit.%2520Rev.%2520Microbiol.%26date%3D2017%26volume%3D43%26spage%3D43%26epage%3D61%26doi%3D10.3109%2F1040841x.2016.1160867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouffartigues, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodilis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillot, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesouhaitier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feuilloley, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orange, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornelis, P.</span></span> <span> </span><span class="NLM_article-title">Structure, function and regulation of <i>Pseudomonas aeruginosa</i> porins</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">698</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1093/femsre/fux020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Ffemsre%2Ffux020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=28981745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGjsLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2017&pages=698-722&author=S.+Chevalierauthor=E.+Bouffartiguesauthor=J.+Bodilisauthor=O.+Maillotauthor=O.+Lesouhaitierauthor=M.+G.+J.+Feuilloleyauthor=N.+Orangeauthor=A.+Dufourauthor=P.+Cornelis&title=Structure%2C+function+and+regulation+of+Pseudomonas+aeruginosa+porins&doi=10.1093%2Ffemsre%2Ffux020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, function and regulation of Pseudomonas aeruginosa porins</span></div><div class="casAuthors">Chevalier, Sylvie; Bouffartigues, Emeline; Bodilis, Josselin; Maillot, Olivier; Lesouhaitier, Olivier; Feuilloley, Marc G. J.; Orange, Nicole; Dufour, Alain; Cornelis, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">698-722</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">1574-6976</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is a Gram-neg. bacterium belonging to the γ -proteobacteria.  Like other members of the Pseudomonas genus, it is known for its metabolic versatility and its ability to colonize a wide range of ecol. niches, such as rhizosphere, water environments and animal hosts, including humans where it can cause severe infections.  Another particularity of P. aeruginosa is its high intrinsic resistance to antiseptics and antibiotics, which is partly due to its low outer membrane permeability.  In contrast to Enterobacteria, pseudomonads do not possess general diffusion porins in their outer membrane, but rather express specific channel proteins for the uptake of different nutrients.  The major outer membrane 'porin', OprF, has been extensively investigated, and displays structural, adhesion and signaling functions while its role in the diffusion of nutrients is still under discussion.  Other porins include OprB and OprB2 for the diffusion of glucose, the two small outer membrane proteins OprG and OprH, and the two porins involved in phosphate/pyrophosphate uptake, OprP and OprO.  The remaining nineteen porins belong to the so-called OprD (Occ) family, which is further split into two subfamilies termed OccD (8 members) and OccK (11 members).  In the past years, a large amt. of information concerning the structure, function and regulation of these porins has been published, justifying why an updated review is timely.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobvqc3Kan7nLVg90H21EOLACvtfcHk0lgThWiliMgBgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGjsLnM&md5=9264cfacba2796f9e5f5fd367f4a423e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Ffemsre%2Ffux020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ffemsre%252Ffux020%26sid%3Dliteratum%253Aachs%26aulast%3DChevalier%26aufirst%3DS.%26aulast%3DBouffartigues%26aufirst%3DE.%26aulast%3DBodilis%26aufirst%3DJ.%26aulast%3DMaillot%26aufirst%3DO.%26aulast%3DLesouhaitier%26aufirst%3DO.%26aulast%3DFeuilloley%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DOrange%26aufirst%3DN.%26aulast%3DDufour%26aufirst%3DA.%26aulast%3DCornelis%26aufirst%3DP.%26atitle%3DStructure%252C%2520function%2520and%2520regulation%2520of%2520Pseudomonas%2520aeruginosa%2520porins%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2017%26volume%3D41%26spage%3D698%26epage%3D722%26doi%3D10.1093%2Ffemsre%2Ffux020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R. E. W.</span></span> <span> </span><span class="NLM_article-title">Adaptive and mutational resistance: role of porins and efflux pumps in drug resistance</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1128/cmr.00043-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FCMR.00043-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=23034325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWjsb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=661-681&author=L.+Fern%C3%A1ndezauthor=R.+E.+W.+Hancock&title=Adaptive+and+mutational+resistance%3A+role+of+porins+and+efflux+pumps+in+drug+resistance&doi=10.1128%2Fcmr.00043-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Adaptive and mutational resistance: role of porins and efflux pumps in drug resistance</span></div><div class="casAuthors">Fernandez, Lucia; Hancock, Robert E. W.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">661-681</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  The substantial use of antibiotics in the clinic, combined with a dearth of new antibiotic classes, has led to a gradual increase in the resistance of bacterial pathogens to these compds.  Among the various mechanisms by which bacteria endure the action of antibiotics, those affecting influx and efflux are of particular importance, as they limit the interaction of the drug with its intracellular targets and, consequently, its deleterious effects on the cell.  This review evaluates the impact of porins and efflux pumps on two major types of resistance, namely, mutational and adaptive types of resistance, both of which are regarded as key phenomena in the global rise of antibiotic resistance among pathogenic microorganisms.  In particular, we explain how adaptive and mutational events can dramatically influence the outcome of antibiotic therapy by altering the mechanisms of influx and efflux of antibiotics.  The identification of porins and pumps as major resistance markers has opened new possibilities for the development of novel therapeutic strategies directed specifically against these mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojE1hifwDBlbVg90H21EOLACvtfcHk0lgThWiliMgBgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWjsb3K&md5=738ffab800ded95e47ae424a4a1a4283</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FCMR.00043-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00043-12%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez%26aufirst%3DL.%26aulast%3DHancock%26aufirst%3DR.%2BE.%2BW.%26atitle%3DAdaptive%2520and%2520mutational%2520resistance%253A%2520role%2520of%2520porins%2520and%2520efflux%2520pumps%2520in%2520drug%2520resistance%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2012%26volume%3D25%26spage%3D661%26epage%3D681%26doi%3D10.1128%2Fcmr.00043-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span> <span> </span><span class="NLM_article-title">The latest advances in β-lactam/β-lactamase inhibitor combinations for the treatment of Gram-negative bacterial infections</span>. <i>Expert Opin. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2169</span>– <span class="NLM_lpage">2184</span>, <span class="refDoi"> DOI: 10.1080/14656566.2019.1660772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1080%2F14656566.2019.1660772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31500471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslGnu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=2169-2184&author=K.+M.+Papp-Wallace&title=The+latest+advances+in+%CE%B2-lactam%2F%CE%B2-lactamase+inhibitor+combinations+for+the+treatment+of+Gram-negative+bacterial+infections&doi=10.1080%2F14656566.2019.1660772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The latest advances in β-lactam/β-lactamase inhibitor combinations for the treatment of Gram-negative bacterial infections</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Pharmacotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2169-2184</span>CODEN:
                <span class="NLM_cas:coden">EOPHF7</span>;
        ISSN:<span class="NLM_cas:issn">1465-6566</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : Antimicrobial resistance in Gram-neg. pathogens is a significant threat to global health. β-Lactams (BL) are one of the safest and most-prescribed classes of antibiotics on the market today.  The acquisition of β-lactamases, esp. those which hydrolyze carbapenems, is eroding the efficacy of BLs for the treatment of serious infections.  During the past decade, significant advances were made in the development of novel BL-β-lactamase inhibitor (BLI) combinations to target β-lactamase-mediated resistant Gram-negatives.: The latest progress in 20 different approved, developing, and preclin. BL-BLI combinations to target serine β-lactamases produced by Gram-negatives are reviewed based on primary literature, conference abstrs. (when available), and US clin. trial searches within the last 5 years.  The majority of the compds. that are discussed are being evaluated as part of a BL-BLI combination.: The current trajectory in BLI development is promising; however, a significant challenge resides in the selection of an appropriate BL partner as well as the development of resistance linked to the BL partner.  In addn., dosing regimens for these BL-BLI combinations need to be critically evaluated.  A revolution in bacterial diagnostics is essential to aid clinicians in the appropriate selection of novel BL-BLI combinations for the treatment of serious infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlWIU5sOUqv7Vg90H21EOLACvtfcHk0lgThWiliMgBgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslGnu7jL&md5=73f3e0ba8358820479bf1652b26fc4a8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F14656566.2019.1660772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14656566.2019.1660772%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26atitle%3DThe%2520latest%2520advances%2520in%2520%25CE%25B2-lactam%252F%25CE%25B2-lactamase%2520inhibitor%2520combinations%2520for%2520the%2520treatment%2520of%2520Gram-negative%2520bacterial%2520infections%26jtitle%3DExpert%2520Opin.%2520Pharmacother.%26date%3D2019%26volume%3D20%26spage%3D2169%26epage%3D2184%26doi%3D10.1080%2F14656566.2019.1660772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arca-Suárez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lence, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Iglesias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán-Sánchez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Challenging antimicrobial susceptibility and evolution of resistance (OXA-681) during treatment of a long-term nosocomial infection caused by a <i>Pseudomonas aeruginosa</i> ST175 clone</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>63</i></span>, <span class="NLM_elocation-id">e01110</span> <span class="refDoi"> DOI: 10.1128/aac.01110-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01110-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31383659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1eisbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2019&author=J.+Arca-Su%C3%A1rezauthor=P.+Fraile-Ribotauthor=J.+C.+V%C3%A1zquez-Uchaauthor=G.+Cabotauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=E.+Lenceauthor=C.+Gonz%C3%A1lez-Belloauthor=A.+Beceiroauthor=M.+Rodr%C3%ADguez-Iglesiasauthor=F.+Gal%C3%A1n-S%C3%A1nchezauthor=G.+Bouauthor=A.+Oliver&title=Challenging+antimicrobial+susceptibility+and+evolution+of+resistance+%28OXA-681%29+during+treatment+of+a+long-term+nosocomial+infection+caused+by+a+Pseudomonas+aeruginosa+ST175+clone&doi=10.1128%2Faac.01110-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Challenging antimicrobial susceptibility and evolution of resistance (OXA-681) during treatment of a long-term nosocomial infection caused by a Pseudomonas aeruginosa ST175 clone</span></div><div class="casAuthors">Arca-Suarez, Jorge; Fraile-Ribot, Pablo; Vazquez-Ucha, Juan Carlos; Cabot, Gabriel; Martinez-Guitian, Marta; Lence, Emilio; Gonzalez-Bello, Concepcion; Beceiro, Alejandro; Rodriguez-Iglesias, Manuel; Galan-Sanchez, Fatima; Bou, German; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e01110-19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Selection of extended-spectrum mutations in narrow-spectrum oxacillinases (e.g., OXA-2 and OXA-10) is an emerging mechanism for development of in vivo resistance to ceftolozane-tazobactam and ceftazidime-avibactam in Pseudomonas aeruginosa.  Detection of these challenging enzymes therefore seems essential to prevent clin. failure, but the complex phenotypic plasticity exhibited by this species may often lead to their underestimation.  The underlying resistance mechanisms of two sequence type 175 (ST175) P. aeruginosa isolates showing multidrug-resistant phenotypes and recovered at early and late stages of a long-term nosocomial infection were evaluated.  Whole-genome sequencing (WGS) was used to investigate resistance genomics, whereas mol. and biochem. methods were used for characterization of a novel extended-spectrum OXA-2 variant selected during therapy.  The metallo-β-lactamase blaVIM-20 and the narrow-spectrum oxacillinase blaOXA-2 were present in both isolates, although they differed by an inactivating mutation in the mexB subunit, present only in the early isolate, and in a mutation in the blaOXA-2 β-lactamase, present only in the final isolate.  The new OXA-2 variant, designated OXA-681, conferred elevated MICs of the novel cephalosporin-β-lactamase inhibitor combinations in a PAO1 background.  Compared to OXA-2, kinetic parameters of the OXA-681 enzyme revealed a substantial increase in the hydrolysis of cephalosporins, including ceftolozane.  We describe the emergence of the novel variant OXA-681 during treatment of a nosocomial infection caused by a Pseudomonas aeruginosa ST175 high-risk clone.  The ability of OXA-681 to confer cross-resistance to ceftolozane-tazobactam and ceftazidime-avibactam together with the complex antimicrobial resistance profiles exhibited by the clin. strains harboring this new enzyme argue for maintaining active surveillance on emerging broad-spectrum resistance in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXTq7Nwva4VLVg90H21EOLACvtfcHk0liaFBFGYk_WVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1eisbjL&md5=eda2d550de42c6a943094516563994b5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FAAC.01110-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01110-19%26sid%3Dliteratum%253Aachs%26aulast%3DArca-Su%25C3%25A1rez%26aufirst%3DJ.%26aulast%3DFraile-Ribot%26aufirst%3DP.%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DLence%26aufirst%3DE.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DRodr%25C3%25ADguez-Iglesias%26aufirst%3DM.%26aulast%3DGal%25C3%25A1n-S%25C3%25A1nchez%26aufirst%3DF.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DChallenging%2520antimicrobial%2520susceptibility%2520and%2520evolution%2520of%2520resistance%2520%2528OXA-681%2529%2520during%2520treatment%2520of%2520a%2520long-term%2520nosocomial%2520infection%2520caused%2520by%2520a%2520Pseudomonas%2520aeruginosa%2520ST175%2520clone%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D63%26doi%3D10.1128%2Faac.01110-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arca-Suárez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lence, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasarte-Monterrubio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Iglesias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán-Sánchez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span> <span> </span><span class="NLM_article-title">Molecular and biochemical insights into the in vivo evolution of AmpC-mediated resistance to ceftolozane/tazobactam during treatment of an MDR Pseudomonas aeruginosa infection</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3209</span>– <span class="NLM_lpage">3217</span>, <span class="refDoi"> DOI: 10.1093/jac/dkaa291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkaa291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=32728723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVykt7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2020&pages=3209-3217&author=J.+Arca-Su%C3%A1rezauthor=J.+C.+V%C3%A1zquez-Uchaauthor=P.+A.+Fraile-Ribotauthor=E.+Lenceauthor=G.+Cabotauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=C.+Lasarte-Monterrubioauthor=M.+Rodr%C3%ADguez-Iglesiasauthor=A.+Beceiroauthor=C.+Gonz%C3%A1lez-Belloauthor=F.+Gal%C3%A1n-S%C3%A1nchezauthor=A.+Oliverauthor=G.+Bou&title=Molecular+and+biochemical+insights+into+the+in+vivo+evolution+of+AmpC-mediated+resistance+to+ceftolozane%2Ftazobactam+during+treatment+of+an+MDR+Pseudomonas+aeruginosa+infection&doi=10.1093%2Fjac%2Fdkaa291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular and biochemical insights into the in vivo evolution of AmpC-mediated resistance to ceftolozane/tazobactam during treatment of an MDR Pseudomonas aeruginosa infection</span></div><div class="casAuthors">Arca-Suarez, Jorge; Vazquez-Ucha, Juan Carlos; Fraile-Ribot, Pablo Arturo; Lence, Emilio; Cabot, Gabriel; Martinez-Guitian, Marta; Lasarte-Monterrubio, Cristina; Rodriguez-Iglesias, Manuel; Beceiro, Alejandro; Gonzalez-Bello, Concepcion; Galan-Sanchez, Fatima; Oliver, Antonio; Bou, German</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3209-3217</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">1460-2091</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background:Pseudomonas aeruginosa may develop resistance to novel cephalosporin/β-lactamase inhibitor combinations during therapy through the acquisition of structural mutations in AmpC.  Objectives: To describe the mol. and biochem. mechanisms involved in the development of resistance to ceftolozane/tazobactam in vivo through the selection and overprodn. of a novel AmpC variant, designated PDC-315.  Methods: Paired susceptible/resistant isolates obtained before and during ceftolozane/tazobactam treatment were evaluated.  MICs were detd. by broth microdilution.  Mutational changes were investigated through WGS.  Characterization of the novel PDC-315 variant was performed through genotypic and biochem. studies.  The effects at the mol. level of the Asp245Asn change were analyzed by mol. dynamics simulations using Amber.  Results: WGS identified mutations leading to modification (Asp245Asn) and overprodn. of AmpC.  Susceptibility testing revealed that PAOΔC producing PDC-315 displayed increased MICs of ceftolozane/tazobactam, decreased MICs of piperacillin/tazobactam and imipenem and similar susceptibility to ceftazidime/avibactam compared with WT PDCs.  The catalytic efficiency of PDC-315 for ceftolozane was 10-fold higher in relation to the WT PDCs, but 3.5- and 5-fold lower for piperacillin and imipenem.  IC50 values indicated strong inhibition of PDC-315 by avibactam, but resistance to cloxacillin inhibition.  Anal. at the at. level explained that the particular behavior of PDC-315 is linked to conformational changes in the H10 helix that favor the approxn. of key catalytic residues to the active site.  Conclusions: We deciphered the precise mechanisms that led to the in vivo emergence of resistance to ceftolozane/tazobactam in P. aeruginosa through the selection of the novel PDC-315 enzyme.  The characterization of this new variant expands our knowledge about AmpC-mediated resistance to cephalosporin/β-lactamase inhibitors in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxfSopninK6rVg90H21EOLACvtfcHk0liaFBFGYk_WVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVykt7zE&md5=b1e2992ca7ed9b8176c6fbe2d01a8960</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkaa291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkaa291%26sid%3Dliteratum%253Aachs%26aulast%3DArca-Su%25C3%25A1rez%26aufirst%3DJ.%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DFraile-Ribot%26aufirst%3DP.%2BA.%26aulast%3DLence%26aufirst%3DE.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DLasarte-Monterrubio%26aufirst%3DC.%26aulast%3DRodr%25C3%25ADguez-Iglesias%26aufirst%3DM.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DGal%25C3%25A1n-S%25C3%25A1nchez%26aufirst%3DF.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DBou%26aufirst%3DG.%26atitle%3DMolecular%2520and%2520biochemical%2520insights%2520into%2520the%2520in%2520vivo%2520evolution%2520of%2520AmpC-mediated%2520resistance%2520to%2520ceftolozane%252Ftazobactam%2520during%2520treatment%2520of%2520an%2520MDR%2520Pseudomonas%2520aeruginosa%2520infection%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2020%26volume%3D75%26spage%3D3209%26epage%3D3217%26doi%3D10.1093%2Fjac%2Fdkaa291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Díaz-Cañestro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periañez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulet, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Pena, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayestarán, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colomar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuñez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciá, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asensio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Causapé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delgado, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murillas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Ceftolozane/tazobactam for the treatment of multidrug resistant <i>Pseudomonas aeruginosa</i>: experience from the Balearic Islands</span>. <i>Eur. J. Clin. Microbiol. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2191</span>– <span class="NLM_lpage">2200</span>, <span class="refDoi"> DOI: 10.1007/s10096-018-3361-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1007%2Fs10096-018-3361-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=30141088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BB3c3hsVWrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2018&pages=2191-2200&author=M.+D%C3%ADaz-Ca%C3%B1estroauthor=L.+Peria%C3%B1ezauthor=X.+Muletauthor=M.+L.+Mart%C3%ADn-Penaauthor=P.+A.+Fraile-Ribotauthor=I.+Ayestar%C3%A1nauthor=A.+Colomarauthor=B.+Nu%C3%B1ezauthor=M.+Maci%C3%A1author=A.+Novoauthor=V.+Torresauthor=J.+Asensioauthor=C.+L%C3%B3pez-Causap%C3%A9author=O.+Delgadoauthor=J.+L.+P%C3%A9rezauthor=J.+Murillasauthor=M.+Rieraauthor=A.+Oliver&title=Ceftolozane%2Ftazobactam+for+the+treatment+of+multidrug+resistant+Pseudomonas+aeruginosa%3A+experience+from+the+Balearic+Islands&doi=10.1007%2Fs10096-018-3361-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Ceftolozane/tazobactam for the treatment of multidrug resistant Pseudomonas aeruginosa: experience from the Balearic Islands</span></div><div class="casAuthors">Diaz-Canestro Manuel; Martin-Pena M Luisa; Asensio Javier; Murillas Javier; Riera Melchor; Perianez Leonor; Delgado Olga; Mulet Xavier; Fraile-Ribot Pablo A; Lopez-Causape Carla; Perez Jose Luis; Oliver Antonio; Ayestaran Ignacio; Colomar Asuncion; Nunez Belen; Macia Maria; Novo Andres; Torres Vicente</div><div class="citationInfo"><span class="NLM_cas:title">European journal of clinical microbiology & infectious diseases : official publication of the European Society of Clinical Microbiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2191-2200</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A prospective, descriptive observational study of consecutive patients treated with ceftolozane/tazobactam in the reference hospital of the Balearic Islands (Spain), between May 2016 and September 2017, was performed.  Demographic, clinical, and microbiological variables were recorded.  The later included resistance profile, molecular typing, and whole genome sequencing of isolates showing resistance development.  Fifty-eight patients were treated with ceftolozane/tazobactam.  Thirty-five (60.3%) showed respiratory tract infections, 21 (36.2%) received monotherapy, and 37 (63.8%) combined therapy for ≥ 72 h, mainly with colistin (45.9%).  In 46.6% of the patients, a dose of 1/0.5 g/8 h was used, whereas 2/1 g/8 h was used in 41.4%.  In 56 of the cases (96.6%), the initial Pseudomonas aeruginosa isolates recovered showed a multidrug resistant (MDR) phenotype, and 50 of them (86.2%) additionally met the extensively drug resistant (XDR) criteria and were only susceptible colistin and/or aminoglycosides (mostly amikacin).  The epidemic high-risk clone ST175 was detected in 50% of the patients.  Clinical cure was documented in 37 patients (63.8%) and resistance development in 8 (13.8%).  Clinical failure was associated with disease severity (SOFA), ventilator-dependent respiratory failure, XDR profile, high-risk clone ST175, negative control culture, and resistance development.  In 6 of the 8 cases, resistance development was caused by structural mutations in AmpC, including some mutations described for the first time in vivo, whereas in the other 2, by mutations in OXA-10 leading to the extended spectrum OXA-14.  Although further clinical experience is still needed, our results suggest that ceftolozane/tazobactam is an attractive option for the treatment of MDR/XDR P. aeruginosa infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzWW3mO0AB-DC850By6mxpfW6udTcc2eZ-hHABkiL0yrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3hsVWrsw%253D%253D&md5=8fd03f82a71b2d9ffcec8601d2593d95</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10096-018-3361-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10096-018-3361-0%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25ADaz-Ca%25C3%25B1estro%26aufirst%3DM.%26aulast%3DPeria%25C3%25B1ez%26aufirst%3DL.%26aulast%3DMulet%26aufirst%3DX.%26aulast%3DMart%25C3%25ADn-Pena%26aufirst%3DM.%2BL.%26aulast%3DFraile-Ribot%26aufirst%3DP.%2BA.%26aulast%3DAyestar%25C3%25A1n%26aufirst%3DI.%26aulast%3DColomar%26aufirst%3DA.%26aulast%3DNu%25C3%25B1ez%26aufirst%3DB.%26aulast%3DMaci%25C3%25A1%26aufirst%3DM.%26aulast%3DNovo%26aufirst%3DA.%26aulast%3DTorres%26aufirst%3DV.%26aulast%3DAsensio%26aufirst%3DJ.%26aulast%3DL%25C3%25B3pez-Causap%25C3%25A9%26aufirst%3DC.%26aulast%3DDelgado%26aufirst%3DO.%26aulast%3DP%25C3%25A9rez%26aufirst%3DJ.%2BL.%26aulast%3DMurillas%26aufirst%3DJ.%26aulast%3DRiera%26aufirst%3DM.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DCeftolozane%252Ftazobactam%2520for%2520the%2520treatment%2520of%2520multidrug%2520resistant%2520Pseudomonas%2520aeruginosa%253A%2520experience%2520from%2520the%2520Balearic%2520Islands%26jtitle%3DEur.%2520J.%2520Clin.%2520Microbiol.%2520Infect.%2520Dis.%26date%3D2018%26volume%3D37%26spage%3D2191%26epage%3D2200%26doi%3D10.1007%2Fs10096-018-3361-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulet, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periañez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Pena, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Mechanisms leading to <i>in vivo</i> ceftolozane/tazobactam resistance development during the treatment of infections caused by MDR <i>Pseudomonas aeruginosa</i></span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1093/jac/dkx424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkx424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=29149337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlequ7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2018&pages=658-663&author=P.+A.+Fraile-Ribotauthor=G.+Cabotauthor=X.+Muletauthor=L.+Peria%C3%B1ezauthor=M.+L.+Mart%C3%ADn-Penaauthor=C.+Juanauthor=J.+L.+P%C3%A9rezauthor=A.+Oliver&title=Mechanisms+leading+to+in+vivo+ceftolozane%2Ftazobactam+resistance+development+during+the+treatment+of+infections+caused+by+MDR+Pseudomonas+aeruginosa&doi=10.1093%2Fjac%2Fdkx424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms leading to in vivo ceftolozane/tazobactam resistance development during the treatment of infections caused by MDR Pseudomonas aeruginosa</span></div><div class="casAuthors">Fraile-Ribot, Pablo A.; Cabot, Gabriel; Mulet, Xavier; Perianez, Leonor; Martyn-Pena, M. Luisa; Juan, Carlos; Perez, Jose L.; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">658-663</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">1460-2091</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives: Characterization of the mechanisms driving ceftolozane/tazobactam resistance development in 5 of 47 (10.6%) patients treated for MDR Pseudomonas aeruginosa infections in a Spanish hospital.  Methods: Five pairs of ceftolozane/tazobactam-susceptible/resistant P. aeruginosa isolates were studied.  MICs were detd. by broth microdilution, clonal relatedness was assessed by MLST and resistance mechanisms were investigated by phenotypic and genotypic methods, including WGS. ampC variants were cloned to assess their impact on resistance.  Results: In all five cases, the same clone was detected for the susceptible/resistant pairs; the widespread ST175 high-risk clone in four of the cases and ST179 in the remaining case.  Genomic anal. of the four initial ST175 isolates revealed the characteristic OprD mutation (Q142X) responsible for carbapenem resistance and the AmpR mutation (G154R) responsible for AmpC overexpression and b-lactam resistance.  The final isolates had developed ceftolozane/tazobactam and ceftazidime/avibactam resistance, and each addnl. showed amutation in AmpC: E247K in one of the isolates, T96I in two isolates and a deletion of 19 amino acids (G229-E247) in the remaining isolate.  The cloned AmpC variants showed greatly increased ceftolozane/tazobactam and ceftazidime/ avibactam MICs compared withWT AmpC, but, in contrast, yielded lower MICs of imipenem, cefepime and particularly piperacillin/tazobactam.  On the other hand, ceftolozane/tazobactamresistance development in ST179 was shown to be driven by the emergence of the extended-spectrum OXA β-lactamase OXA-14, through the selection of an N146S mutation from OXA-10.  Conclusions: Modification of intrinsic (AmpC) and horizontally acquired β-lactamases appears to be the main mechanism leading to ceftolozane/tazobactamresistance in MDR P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7_xxK6o_cTbVg90H21EOLACvtfcHk0lisyLYdJbnf3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlequ7%252FJ&md5=dd24b1f125503acc222d776d71e67b6d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkx424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkx424%26sid%3Dliteratum%253Aachs%26aulast%3DFraile-Ribot%26aufirst%3DP.%2BA.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DMulet%26aufirst%3DX.%26aulast%3DPeria%25C3%25B1ez%26aufirst%3DL.%26aulast%3DMart%25C3%25ADn-Pena%26aufirst%3DM.%2BL.%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DP%25C3%25A9rez%26aufirst%3DJ.%2BL.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DMechanisms%2520leading%2520to%2520in%2520vivo%2520ceftolozane%252Ftazobactam%2520resistance%2520development%2520during%2520the%2520treatment%2520of%2520infections%2520caused%2520by%2520MDR%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2018%26volume%3D73%26spage%3D658%26epage%3D663%26doi%3D10.1093%2Fjac%2Fdkx424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span> <span> </span><span class="NLM_article-title">Diazabicyclooctanes (DBOs): a potent new class of non-β-lactam β-lactamase inhibitors</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">550</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2011.07.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.mib.2011.07.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=21840248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCntrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=550-555&author=K.+Coleman&title=Diazabicyclooctanes+%28DBOs%29%3A+a+potent+new+class+of+non-%CE%B2-lactam+%CE%B2-lactamase+inhibitors&doi=10.1016%2Fj.mib.2011.07.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Diazabicyclooctanes (DBOs): a potent new class of non-β-lactam β-lactamase inhibitors</span></div><div class="casAuthors">Coleman, Ken</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">550-555</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The β-lactams have been among the most successful classes of antibacterial agents for the past half century.  However, a disturbing increase in resistance to β-lactams has been noted among Gram-neg. bacteria, which is attributable to β-lactamase enzymes not within the spectrum of currently marketed β-lactams or β-lactam/β-lactamase inhibitor combinations.  Diazabicyclooctanes (DBOs) were first investigated as β-lactam mimics in the mid-1990s by chemists at Hoechst Marion Roussel (now part of Sanofi-Aventis) and proved to be a rich source of β-lactamase inhibitors (BLI).  Two members of this novel series of highly potent, broad spectrum BLIs are now in clin. development and their properties are reviewed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpneNkuutLgqrVg90H21EOLACvtfcHk0lisyLYdJbnf3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCntrjN&md5=0aafa5daac59fc737d464d7a7b637b2c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2011.07.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2011.07.026%26sid%3Dliteratum%253Aachs%26aulast%3DColeman%26aufirst%3DK.%26atitle%3DDiazabicyclooctanes%2520%2528DBOs%2529%253A%2520a%2520potent%2520new%2520class%2520of%2520non-%25CE%25B2-lactam%2520%25CE%25B2-lactamase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2011%26volume%3D14%26spage%3D550%26epage%3D555%26doi%3D10.1016%2Fj.mib.2011.07.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, I.</span></span> <span> </span><span class="NLM_article-title">New promising β-lactamase inhibitors for clinical use</span>. <i>Eur. J. Clin. Microbiol. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1303</span>– <span class="NLM_lpage">1308</span>, <span class="refDoi"> DOI: 10.1007/s10096-015-2375-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1007%2Fs10096-015-2375-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25864193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms1Wnurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=1303-1308&author=I.+Olsen&title=New+promising+%CE%B2-lactamase+inhibitors+for+clinical+use&doi=10.1007%2Fs10096-015-2375-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">New promising β-lactamase inhibitors for clinical use</span></div><div class="casAuthors">Olsen, I.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Clinical Microbiology & Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1303-1308</span>CODEN:
                <span class="NLM_cas:coden">EJCDEU</span>;
        ISSN:<span class="NLM_cas:issn">0934-9723</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Clavulanate, sulbactam, and tazobactam have been used extensively for the last 30 years, together with β-lactam antibiotics, to inhibit the effect of β-lactamases.  Although they have been useful as β-lactamase inhibitors in many cases, their effectiveness is restricted to class A β-lactamases.  With the increasing frequency and breadth of β-lactamases now threatening public health throughout the world, we need a much broader spectrum of β-lactamase inhibitors efficient against all classes of β-lactamases.  There are several β-lactamase inhibitors under development, but only a few of them are able to inhibit class D and even fewer class B metallo-β-lactamases (MβLs).  The latter represent a real threat to the latest generations of β-lactam antibiotics, including cephalosporins and carbapenems.  Only two β-lactamase inhibitors are, so far, under clin. evaluation, i.e., avibactam and MK-7655.  The others are years from being clin. available.  Although this has caused cautious optimism, the progress in this field is far too slow.  This is particularly so because none of the substances provided are active against MβLs and because new β-lactamases invariably force their way into our therapeutic armamentarium.  While waiting for new antibiotics and new β-lactamase inhibitors to become available, it is important to carry out accurate clin. and microbiol. diagnosis, perform adequate hygiene, and use antibiotics properly.  This may save lives and reduce resistance resulting from inappropriate antibiotic treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpILI_arNIAs7Vg90H21EOLACvtfcHk0lhJsBW1rriTjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms1Wnurk%253D&md5=c0e94ef7d758c12abd1c847171a6ab89</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs10096-015-2375-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10096-015-2375-0%26sid%3Dliteratum%253Aachs%26aulast%3DOlsen%26aufirst%3DI.%26atitle%3DNew%2520promising%2520%25CE%25B2-lactamase%2520inhibitors%2520for%2520clinical%2520use%26jtitle%3DEur.%2520J.%2520Clin.%2520Microbiol.%2520Infect.%2520Dis.%26date%3D2015%26volume%3D34%26spage%3D1303%26epage%3D1308%26doi%3D10.1007%2Fs10096-015-2375-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullane, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolau, D. P.</span></span> <span> </span><span class="NLM_article-title">Comparative Evaluation of the In Vitro Activities of WCK 5222 (Cefepime-Zidebactam) and Combination Antibiotic Therapies against Carbapenem-Resistant Pseudomonas aeruginosa</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e01669</span> <span class="refDoi"> DOI: 10.1128/aac.01669-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.01669-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31844009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmvVKisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=E.+M.+Mullaneauthor=L.+M.+Averyauthor=D.+P.+Nicolau&title=Comparative+Evaluation+of+the+In+Vitro+Activities+of+WCK+5222+%28Cefepime-Zidebactam%29+and+Combination+Antibiotic+Therapies+against+Carbapenem-Resistant+Pseudomonas+aeruginosa&doi=10.1128%2Faac.01669-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative evaluation of the in vitro activities of WCK 5222 (cefepime-zidebactam) and combination antibiotic therapies against carbapenem-resistant Pseudomonas aeruginosa</span></div><div class="casAuthors">Mullane, Elias M.; Avery, Lindsay M.; Nicolau, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e01669-19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The in vitro activity of WCK 5222 (cefepime-zidebactam) was compared to that of several available combination therapies among 30 clin. carbapenem-resistant Pseudomonas aeruginosa (CRP) strains using gradient diffusion strips.  The combinations included nonsusceptible β-lactams (cefepime, ceftolozane-tazobactam, and meropenem) with amikacin and fosfomycin.  WCK 5222 MICs ranged from 2 to 32 mg/L, and 97% were ≤16 mg/L, while 105/146 (72%) combinations demonstrated inhibition below established susceptibility breakpoints.  WCK 5222 monotherapy may be preferred over the combinations assessed for CRP infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEVwTwdyL4RrVg90H21EOLACvtfcHk0lhJsBW1rriTjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmvVKisbo%253D&md5=2ecf684947c79e40ed110f5ecf3ab4fd</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2Faac.01669-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01669-19%26sid%3Dliteratum%253Aachs%26aulast%3DMullane%26aufirst%3DE.%2BM.%26aulast%3DAvery%26aufirst%3DL.%2BM.%26aulast%3DNicolau%26aufirst%3DD.%2BP.%26atitle%3DComparative%2520Evaluation%2520of%2520the%2520In%2520Vitro%2520Activities%2520of%2520WCK%25205222%2520%2528Cefepime-Zidebactam%2529%2520and%2520Combination%2520Antibiotic%2520Therapies%2520against%2520Carbapenem-Resistant%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.01669-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sader, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhomberg, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamm, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castanheira, M.</span></span> <span> </span><span class="NLM_article-title">WCK 5222 (cefepime/zidebactam) antimicrobial activity tested against Gram-negative organisms producing clinically relevant β-lactamases</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1696</span>– <span class="NLM_lpage">1703</span>, <span class="refDoi"> DOI: 10.1093/jac/dkx050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkx050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=28333332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKisLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2017&pages=1696-1703&author=H.+S.+Saderauthor=P.+R.+Rhombergauthor=R.+K.+Flammauthor=R.+N.+Jonesauthor=M.+Castanheira&title=WCK+5222+%28cefepime%2Fzidebactam%29+antimicrobial+activity+tested+against+Gram-negative+organisms+producing+clinically+relevant+%CE%B2-lactamases&doi=10.1093%2Fjac%2Fdkx050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">WCK 5222 (cefepime/zidebactam) antimicrobial activity tested against Gram-negative organisms producing clinically relevant β-lactamases</span></div><div class="casAuthors">Sader, Helio S.; Rhomberg, Paul R.; Flamm, Robert K.; Jones, Ronald N.; Castanheira, Mariana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1696-1703</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">1460-2091</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Zidebactam is a β-lactam enhancer antibiotic with a dual mechanism of action involving binding to Gram-neg. PBP2 and β-lactamase inhibition.  Cefepime combined with zidebactam (WCK 5222) is under clin. development for treatment of Gram-neg. infections.  To evaluate the in vitro activities of cefepime and zidebactam sep. and combined at 1:1 and 2:1 ratios when tested against Gram-neg. organisms producing the most clin. relevant β-lactamase types.  β-Lactamase-producing (193) and WT (71) isolates were tested for susceptibility by broth microdilution method against cefepime/zidebactam, cefepime and zidebactam.  Cefepime/zidebactam (1:1) was very active against Enterobacteriaceae producing CTX-M-15 (21; MIC50/90 0.25/1 mg/L), SHV (20; MIC50/90 0.12/0.25 mg/L), other ESBLs (20, including GES-18, OXA-1/30 and OXY-, PER-, TEM- and VEB-like; MIC50/90 0.25/1 mg/L), plasmidic AmpC (10; MIC50/90 ≤0.06/≤0.06 mg/L), derepressed AmpC (23; MIC50/90 0.12/0.5 mg/L), KPC (35; MIC50/90 0.25/1 mg/L) and metallo-β-lactamases (MBLs; 20 including VIM, IMP and NDM; MIC50/90 0.5/8 mg/L).  Cefepime/zidebactam (1:1) was also active against Pseudomonas aeruginosa with overexpression of AmpC (21; MIC50/90 4/8 mg/L) and MBLs [12 (VIM and IMP); MIC50/90 4/8 mg/L].  Zidebactam alone exhibited potent in vitro activity against some Enterobacteriaceae and P. aeruginosa, including β-lactamase-producing strains.  Cefepime/zidebactam MIC values were lower than those of each agent tested alone for many β-lactamase-producing strains, indicating synergy.  Cefepime/zidebactam showed moderate activity against OXA-23/24/58-producing Acinetobacter baumannii [MIC50/90 32 mg/L (1:1)].  Cefepime/zidebactam showed potent activities against Enterobacteriaceae and P. aeruginosa producing various clin. relevant β-lactamases, including ESBLs, KPCs, AmpC and MBLs for which limited treatment options are currently available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrammLVMFaLrbVg90H21EOLACvtfcHk0lhJsBW1rriTjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKisLrM&md5=23ae84246b3accbee9d450ed463a6803</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkx050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkx050%26sid%3Dliteratum%253Aachs%26aulast%3DSader%26aufirst%3DH.%2BS.%26aulast%3DRhomberg%26aufirst%3DP.%2BR.%26aulast%3DFlamm%26aufirst%3DR.%2BK.%26aulast%3DJones%26aufirst%3DR.%2BN.%26aulast%3DCastanheira%26aufirst%3DM.%26atitle%3DWCK%25205222%2520%2528cefepime%252Fzidebactam%2529%2520antimicrobial%2520activity%2520tested%2520against%2520Gram-negative%2520organisms%2520producing%2520clinically%2520relevant%2520%25CE%25B2-lactamases%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2017%26volume%3D72%26spage%3D1696%26epage%3D1703%26doi%3D10.1093%2Fjac%2Fdkx050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsivkovski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span> <span> </span><span class="NLM_article-title">Biochemical activity of vaborbactam</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e01935</span> <span class="refDoi"> DOI: 10.1128/aac.01935-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.01935-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31712199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktFyhtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=R.+Tsivkovskiauthor=O.+Lomovskaya&title=Biochemical+activity+of+vaborbactam&doi=10.1128%2Faac.01935-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical activity of vaborbactam</span></div><div class="casAuthors">Tsivkovski, Ruslan; Lomovskaya, Olga</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e01935</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The most common mechanism of resistance to β-lactams antibiotics in Gram-neg. bacteria is prodn. of β-lactamase enzymes capable of cleaving the β-lactam ring.  Inhibition of β-lactamase activity with small-mol. drugs is a proven strategy to restore the potency of many β-lactam antibiotics.  Vaborbactam (formerly RPX7009) is a cyclic boronic acid β-lactamase inhibitor (BLI) with a broad spectrum of activity against various serine β-lactamases, including KPC carbapenemases.  The combination of vaborbactam and meropenem is approved in the United States and Europe for the treatment of various nosocomial infections.  We attempted to gain more insight into the mechanism of action of vaborbactam by conducting detailed kinetic characterization of its interaction with various recombinant His-tagged β-lactamases.  Vaborbactam demonstrated potent inhibition of class A and class C enzymes with Ki values ranging from 0.022 to 0.18 μM, while inhibition of class D enzymes was rather poor, and no activity against class B β-lactamases was detected.  Importantly, vaborbactam inhibited KPC-2, KPC-3, BKC-1, and SME-2 carbapenemases at 1:1 stoichiometry, while these nos. were higher for other class A and C enzymes.  Vaborbactam was also shown to be a potent progressive inactivator of several enzymes, including KPCs with inactivation consts. k2/K in the range of 3.4 × 103 to 2.4 × 104 M-1 s-1.  Finally, expts. on the recovery of enzyme activity demonstrated the high stability of the vaborbactam-KPC complex, with 0.000040 s-1koff values and a corresponding residence time of 7 h, whereas the release of vaborbactam bound to other serine β-lactamases was substantially faster.  The biochem. characteristics of vaborbactam described in this study may facilitate further chem. optimization efforts to develop boronic BLIs with improved affinity and broader spectrum of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqct6STwjYA67Vg90H21EOLACvtfcHk0lhJsBW1rriTjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktFyhtbg%253D&md5=53e588b395cfb868f0a5f8699427c05a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2Faac.01935-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01935-19%26sid%3Dliteratum%253Aachs%26aulast%3DTsivkovski%26aufirst%3DR.%26aulast%3DLomovskaya%26aufirst%3DO.%26atitle%3DBiochemical%2520activity%2520of%2520vaborbactam%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.01935-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trout, R. E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, G.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGarry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamrick, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daigle, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Luca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benvenuti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Docquier, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pevear, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xerri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Taniborbactam (VNRX-5133): A Broad-Spectrum Serine- and Metallo-β-lactamase Inhibitor for Carbapenem-Resistant Bacterial Infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">2789</span>– <span class="NLM_lpage">2801</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01518</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01518" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1GmsbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=2789-2801&author=B.+Liuauthor=R.+E.+L.+Troutauthor=G.-H.+Chuauthor=D.+McGarryauthor=R.+W.+Jacksonauthor=J.+C.+Hamrickauthor=D.+M.+Daigleauthor=S.+M.+Cusickauthor=C.+Pozziauthor=F.+De+Lucaauthor=M.+Benvenutiauthor=S.+Manganiauthor=J.-D.+Docquierauthor=W.+J.+Weissauthor=D.+C.+Pevearauthor=L.+Xerriauthor=C.+J.+Burns&title=Discovery+of+Taniborbactam+%28VNRX-5133%29%3A+A+Broad-Spectrum+Serine-+and+Metallo-%CE%B2-lactamase+Inhibitor+for+Carbapenem-Resistant+Bacterial+Infections&doi=10.1021%2Facs.jmedchem.9b01518"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Taniborbactam (VNRX-5133): A Broad-Spectrum Serine- and Metallo-β-lactamase Inhibitor for Carbapenem-Resistant Bacterial Infections</span></div><div class="casAuthors">Liu, Bin; Trout, Robert E. Lee; Chu, Guo-Hua; McGarry, Daniel; Jackson, Randy W.; Hamrick, Jodie C.; Daigle, Denis M.; Cusick, Susan M.; Pozzi, Cecilia; De Luca, Filomena; Benvenuti, Manuela; Mangani, Stefano; Docquier, Jean-Denis; Weiss, William J.; Pevear, Daniel C.; Xerri, Luigi; Burns, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2789-2801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A major resistance mechanism in Gram-neg. bacteria is the prodn. of β-lactamase enzymes.  Originally recognized for their ability to hydrolyze penicillins, emergent β-lactamases can now confer resistance to other β-lactam drugs, including both cephalosporins and carbapenems.  The emergence and global spread of β-lactamase-producing multidrug-resistant superbugs has caused increased alarm within the medical community due to the high mortality rate assocd. with these difficult-to-treat bacterial infections.  To address this unmet medical need, we initiated an iterative program combining medicinal chem., structural biol., biochem. testing, and microbiol. profiling to identify broad-spectrum inhibitors of both serine- and metallo-β-lactamase enzymes.  Lead optimization, beginning with narrower-spectrum, weakly active compds. provided VNRX-5133 (taniborbactam I), a boronic acid-contg. pan-spectrum β-lactamase inhibitor.  In vitro and in vivo studies demonstrated that VNRX-5133 restored the activity of β-lactam antibiotics against carbapenem-resistant Pseudomonas aeruginosa and carbapenem-resistant Enterobacteriaceae.  Taniborbactam is the 1st pan-spectrum β-lactamase inhibitor to enter clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn-mAQzqladbVg90H21EOLACvtfcHk0lhWmXowpes35g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1GmsbjI&md5=78af5dfe748ba2480fb7165d62e9b271</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01518%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DTrout%26aufirst%3DR.%2BE.%2BL.%26aulast%3DChu%26aufirst%3DG.-H.%26aulast%3DMcGarry%26aufirst%3DD.%26aulast%3DJackson%26aufirst%3DR.%2BW.%26aulast%3DHamrick%26aufirst%3DJ.%2BC.%26aulast%3DDaigle%26aufirst%3DD.%2BM.%26aulast%3DCusick%26aufirst%3DS.%2BM.%26aulast%3DPozzi%26aufirst%3DC.%26aulast%3DDe%2BLuca%26aufirst%3DF.%26aulast%3DBenvenuti%26aufirst%3DM.%26aulast%3DMangani%26aufirst%3DS.%26aulast%3DDocquier%26aufirst%3DJ.-D.%26aulast%3DWeiss%26aufirst%3DW.%2BJ.%26aulast%3DPevear%26aufirst%3DD.%2BC.%26aulast%3DXerri%26aufirst%3DL.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26atitle%3DDiscovery%2520of%2520Taniborbactam%2520%2528VNRX-5133%2529%253A%2520A%2520Broad-Spectrum%2520Serine-%2520and%2520Metallo-%25CE%25B2-lactamase%2520Inhibitor%2520for%2520Carbapenem-Resistant%2520Bacterial%2520Infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D2789%26epage%3D2801%26doi%3D10.1021%2Facs.jmedchem.9b01518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamrick, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Docquier, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uehara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Six, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatwin, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernacchio, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trout, R. E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Luca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benvenuti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moeck, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xerri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pevear, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daigle, D. M.</span></span> <span> </span><span class="NLM_article-title">VNRX-5133 (Taniborbactam), a Broad-Spectrum Inhibitor of Serine- and Metallo-β-Lactamases, Restores Activity of Cefepime in Enterobacterales and Pseudomonas aeruginosa</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e01963</span> <span class="refDoi"> DOI: 10.1128/aac.01963-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.01963-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31871094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntlGisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=J.+C.+Hamrickauthor=J.-D.+Docquierauthor=T.+Ueharaauthor=C.+L.+Myersauthor=D.+A.+Sixauthor=C.+L.+Chatwinauthor=K.+J.+Johnauthor=S.+F.+Vernacchioauthor=S.+M.+Cusickauthor=R.+E.+L.+Troutauthor=C.+Pozziauthor=F.+De+Lucaauthor=M.+Benvenutiauthor=S.+Manganiauthor=B.+Liuauthor=R.+W.+Jacksonauthor=G.+Moeckauthor=L.+Xerriauthor=C.+J.+Burnsauthor=D.+C.+Pevearauthor=D.+M.+Daigle&title=VNRX-5133+%28Taniborbactam%29%2C+a+Broad-Spectrum+Inhibitor+of+Serine-+and+Metallo-%CE%B2-Lactamases%2C+Restores+Activity+of+Cefepime+in+Enterobacterales+and+Pseudomonas+aeruginosa&doi=10.1128%2Faac.01963-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">VNRX-5133 (Taniborbactam), a broad-spectrum inhibitor of serine- and metallo-β-lactamases, restores activity of cefepime in Enterobacterales and Pseudomonas aeruginosa</span></div><div class="casAuthors">Hamrick, Jodie C.; Docquier, Jean-Denis; Uehara, Tsuyoshi; Myers, Cullen L.; Six, David A.; Chatwin, Cassandra L.; John, Kaitlyn J.; Vernacchio, Salvador F.; Cusick, Susan M.; Trout, Robert E. L.; Pozzi, Cecilia; De Luca, Filomena; Benvenuti, Manuela; Mangani, Stefano; Liu, Bin; Jackson, Randy W.; Moeck, Greg; Xerri, Luigi; Burns, Christopher J.; Pevear, Daniel C.; Daigle, Denis M.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e01963-19/1-e01963-19/19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">As shifts in the epidemiol. of β-lactamase-mediated resistance continue, carbapenem-resistant Enterobacterales (CRE) and carbapenem-resistant Pseudomonas aeruginosa (CRPA) are the most urgent threats.  Although approved β-lactam (BL)-β-lactamase inhibitor (BLI) combinations address widespread serine β-lactamases (SBLs), such as CTX-M-15, none provide broad coverage against either clin. important serine-β-lactamases (KPC, OXA-48) or clin. important metallo-β-lactamases (MBLs; e.g., NDM-1).  VNRX-5133 (taniborbactam) is a new cyclic boronate BLI that is in clin. development combined with cefepime for the treatment of infections caused by β-lactamase-producing CRE and CRPA.  Taniborbactam is the first BLI with direct inhibitory activity against Ambler class A, B, C, and D enzymes.  From biochem. and structural analyses, taniborbactam exploits substrate mimicry while employing distinct mechanisms to inhibit both SBLs and MBLs.  It is a reversible covalent inhibitor of SBLs with slow dissocn. and a prolonged active-site residence time (half-life, 30 to 105 min), while in MBLs, it behaves as a competitive inhibitor, with inhibitor const. (Ki) values ranging from 0.019 to 0.081 μM.  Inhibition is achieved by mimicking the transition state structure and exploiting interactions with highly conserved active-site residues.  In microbiol. testing, taniborbactam restored cefepime activity in 33/34 engineered Escherichia coli strains overproducing individual enzymes covering Ambler classes A, B, C, and D, providing up to a 1,024-fold shift in the MIC.  Addn. of taniborbactam restored the antibacterial activity of cefepime against all 102 Enterobacterales clin. isolates tested and 38/41 P. aeruginosa clin. isolates tested with MIC90s of 1 and 4 μg/mL, resp., representing ≥256- and ≥32-fold improvements, resp., in antibacterial activity over that of cefepime alone.  The data demonstrate the potent, broad-spectrum rescue of cefepime activity by taniborbactam against clin. isolates of CRE and CRPA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzz4T2vRQnKLVg90H21EOLACvtfcHk0lhWmXowpes35g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntlGisrg%253D&md5=1387fed36a9998aa80ec4ed287d6fd4a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2Faac.01963-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01963-19%26sid%3Dliteratum%253Aachs%26aulast%3DHamrick%26aufirst%3DJ.%2BC.%26aulast%3DDocquier%26aufirst%3DJ.-D.%26aulast%3DUehara%26aufirst%3DT.%26aulast%3DMyers%26aufirst%3DC.%2BL.%26aulast%3DSix%26aufirst%3DD.%2BA.%26aulast%3DChatwin%26aufirst%3DC.%2BL.%26aulast%3DJohn%26aufirst%3DK.%2BJ.%26aulast%3DVernacchio%26aufirst%3DS.%2BF.%26aulast%3DCusick%26aufirst%3DS.%2BM.%26aulast%3DTrout%26aufirst%3DR.%2BE.%2BL.%26aulast%3DPozzi%26aufirst%3DC.%26aulast%3DDe%2BLuca%26aufirst%3DF.%26aulast%3DBenvenuti%26aufirst%3DM.%26aulast%3DMangani%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DJackson%26aufirst%3DR.%2BW.%26aulast%3DMoeck%26aufirst%3DG.%26aulast%3DXerri%26aufirst%3DL.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26aulast%3DPevear%26aufirst%3DD.%2BC.%26aulast%3DDaigle%26aufirst%3DD.%2BM.%26atitle%3DVNRX-5133%2520%2528Taniborbactam%2529%252C%2520a%2520Broad-Spectrum%2520Inhibitor%2520of%2520Serine-%2520and%2520Metallo-%25CE%25B2-Lactamases%252C%2520Restores%2520Activity%2520of%2520Cefepime%2520in%2520Enterobacterales%2520and%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.01963-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caillon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajaksouzian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reghal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taracila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacqueline, C.</span></span> <span> </span><span class="NLM_article-title">Beyond Piperacillin-Tazobactam: Cefepime and AAI101 as a Potent β-Lactam−β-Lactamase Inhibitor Combination</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>63</i></span>, <span class="NLM_elocation-id">e00105</span> <span class="refDoi"> DOI: 10.1128/aac.00105-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.00105-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=30858223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeiur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2019&author=K.+M.+Papp-Wallaceauthor=C.+R.+Bethelauthor=J.+Caillonauthor=M.+D.+Barnesauthor=G.+Potelauthor=S.+Bajaksouzianauthor=J.+D.+Rutterauthor=A.+Reghalauthor=S.+Shapiroauthor=M.+A.+Taracilaauthor=M.+R.+Jacobsauthor=R.+A.+Bonomoauthor=C.+Jacqueline&title=Beyond+Piperacillin-Tazobactam%3A+Cefepime+and+AAI101+as+a+Potent+%CE%B2-Lactam%E2%88%92%CE%B2-Lactamase+Inhibitor+Combination&doi=10.1128%2Faac.00105-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond piperacillin-tazobactam: cefepime and AAI101 as a potent β-Lactam-β-lactamase inhibitor combination</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.; Bethel, Christopher R.; Caillon, Jocelyne; Barnes, Melissa D.; Potel, Gilles; Bajaksouzian, Saralee; Rutter, Joseph D.; Reghal, Amokrane; Shapiro, Stuart; Taracila, Magdalena A.; Jacobs, Michael R.; Bonomo, Robert A.; Jacqueline, Cedric</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e00105-19/1-e00105-19/17</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Impeding, as well as reducing, the burden of antimicrobial resistance in Gram-neg. pathogens is an urgent public health endeavor.  Our current antibiotic armamentarium is dwindling, while major resistance determinants (e.g., extendedspectrum β-lactamases [ESBLs]) continue to evolve and disseminate around the world.  One approach to attack this problem is to develop novel therapies that will protect our current agents.  AAI101 is a novel penicillanic acid sulfone β-lactamase inhibitor similar in structure to tazobactam, with one important difference.  AAI101 possesses a strategically placed Me group that gives the inhibitor a net neutral charge, enhancing bacterial cell penetration.  AAI101 paired with cefepime, also a zwitterion, is in phase III of clin. development for the treatment of serious Gram-neg. infections.  Here, AAI101 was found to restore the activity of cefepime against class A ESBLs (e.g., CTX-M-15) and demonstrated increased potency compared to that of piperacillin-tazobactam when tested against an established isogenic panel.  The enzymol. properties of AAI101 further revealed that AAI101 possessed a unique mechanism of β-lactamase inhibition compared to that of tazobactam.  Addnl., upon reaction with AAI101, CTX-M-15 was modified to an inactive state.  Notably, the in vivo efficacy of cefepime-AAI101 was demonstrated using a mouse septicemia model, indicating the ability of AAI101 to bolster significantly the therapeutic efficacy of cefepime in vivo.  The combination of AAI101 with cefepime represents a potential carbapenem-sparing treatment regimen for infections suspected to be caused by Enterobacteriaceae expressing ESBLs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0wGvlaUiEsLVg90H21EOLACvtfcHk0lj4X0AJ2ek_Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeiur%252FE&md5=55a99d1c91a4f138479ad35e507f1c99</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FAAC.00105-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00105-19%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DCaillon%26aufirst%3DJ.%26aulast%3DBarnes%26aufirst%3DM.%2BD.%26aulast%3DPotel%26aufirst%3DG.%26aulast%3DBajaksouzian%26aufirst%3DS.%26aulast%3DRutter%26aufirst%3DJ.%2BD.%26aulast%3DReghal%26aufirst%3DA.%26aulast%3DShapiro%26aufirst%3DS.%26aulast%3DTaracila%26aufirst%3DM.%2BA.%26aulast%3DJacobs%26aufirst%3DM.%2BR.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DJacqueline%26aufirst%3DC.%26atitle%3DBeyond%2520Piperacillin-Tazobactam%253A%2520Cefepime%2520and%2520AAI101%2520as%2520a%2520Potent%2520%25CE%25B2-Lactam%25E2%2588%2592%25CE%25B2-Lactamase%2520Inhibitor%2520Combination%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D63%26doi%3D10.1128%2Faac.00105-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pernas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colchón, E.</span></span> <span> </span><span class="NLM_article-title">β-Lactamase Inhibitors To Restore the Efficacy of Antibiotics against Superbugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1859</span>– <span class="NLM_lpage">1881</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVOhsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=1859-1881&author=C.+Gonz%C3%A1lez-Belloauthor=D.+Rodr%C3%ADguezauthor=M.+Pernasauthor=%C3%81.+Rodr%C3%ADguezauthor=E.+Colch%C3%B3n&title=%CE%B2-Lactamase+Inhibitors+To+Restore+the+Efficacy+of+Antibiotics+against+Superbugs&doi=10.1021%2Facs.jmedchem.9b01279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">β-Lactamase Inhibitors To Restore the Efficacy of Antibiotics against Superbugs</span></div><div class="casAuthors">Gonzalez-Bello, Concepcion; Rodriguez, Diana; Pernas, Marina; Rodriguez, Angela; Colchon, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1859-1881</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Infections caused by resistant bacteria are nowadays too common and some pathogens have even become resistant to multiple types of antibiotics, in which cases few or even no treatments are available.  In recent years, the most successful strategy in anti-infective drug discovery for the treatment of such problematic infections is the combination therapy 'antibiotic + inhibitor of resistance'.  These inhibitors allow the repurposing of antibiotics that have already proven to be safe and effective for clin. use.  Three main types of compds. have been developed to block the principal bacterial resistance mechanisms: (i) β-lactamase inhibitors; (ii) outer membrane permeabilizers; and (iii) efflux pump inhibitors.  This perspective article is focused on β-lactamase inhibitors that disable the most prevalent cause of antibiotic resistance in Gram-neg. bacteria, i.e., the deactivation of the most widely used antibiotics, β-lactams (penicillins, cephalosporines, carbapenems and monobactams), by the prodn. of β-lactamases.  An overview of the most recently identified β-lactamase inhibitors and of combination therapy is provided.  The article also covers the mechanism of action of the different types of β-lactamase enzymes as a basis for inhibitor design and target inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs6jFuMiB-xLVg90H21EOLACvtfcHk0lj4X0AJ2ek_Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVOhsb3J&md5=336d7eb65305b05d909584c6ae3af6a8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01279%26sid%3Dliteratum%253Aachs%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DPernas%26aufirst%3DM.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3D%25C3%2581.%26aulast%3DColch%25C3%25B3n%26aufirst%3DE.%26atitle%3D%25CE%25B2-Lactamase%2520Inhibitors%2520To%2520Restore%2520the%2520Efficacy%2520of%2520Antibiotics%2520against%2520Superbugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D1859%26epage%3D1881%26doi%3D10.1021%2Facs.jmedchem.9b01279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vallejo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span> <span> </span><span class="NLM_article-title">LN-1-255, a penicillanic acid sulfone able to inhibit the class D carbapenemase OXA-48</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">2171</span>– <span class="NLM_lpage">2180</span>, <span class="refDoi"> DOI: 10.1093/jac/dkw105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkw105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=27125555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BC28bktlGguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2016&pages=2171-2180&author=J.+A.+Vallejoauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=J.+C.+V%C3%A1zquez-Uchaauthor=C.+Gonz%C3%A1lez-Belloauthor=M.+Pozaauthor=J.+D.+Buynakauthor=C.+R.+Bethelauthor=R.+A.+Bonomoauthor=G.+Bouauthor=A.+Beceiro&title=LN-1-255%2C+a+penicillanic+acid+sulfone+able+to+inhibit+the+class+D+carbapenemase+OXA-48&doi=10.1093%2Fjac%2Fdkw105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">LN-1-255, a penicillanic acid sulfone able to inhibit the class D carbapenemase OXA-48</span></div><div class="casAuthors">Vallejo Juan A; Martinez-Guitian Marta; Vazquez-Ucha Juan C; Poza Margarita; Bou German; Beceiro Alejandro; Gonzalez-Bello Concepcion; Buynak John D; Bethel Christopher R; Bonomo Robert A</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antimicrobial chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2171-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Carbapenemases are the most important mechanism responsible for carbapenem resistance in Enterobacteriaceae.  Among carbapenemases, OXA-48 presents unique challenges as it is resistant to β-lactam inhibitors.  Here, we test the capacity of the compound LN-1-255, a 6-alkylidene-2'-substituted penicillanic acid sulfone, to inhibit the activity of the carbapenemase OXA-48.  METHODS:  The OXA-48 gene was cloned and expressed in Klebsiella pneumoniae and Escherichia coli in order to obtain MICs in the presence of inhibitors (clavulanic acid, tazobactam and sulbactam) and LN-1-255.  OXA-48 was purified and steady-state kinetics was performed with LN-1-255 and tazobactam.  The covalent binding mode of LN-1-255 with OXA-48 was studied by docking assays.  RESULTS:  Both OXA-48-producing clinical and transformant strains displayed increased susceptibility to carbapenem antibiotics in the presence of 4 mg/L LN-1-255 (2-32-fold increased susceptibility) and 16 mg/L LN-1-255 (4-64-fold increased susceptibility).  Kinetic assays demonstrated that LN-1-255 is able to inhibit OXA-48 with an acylation efficiency (k2/K) of 10 ± 1 × 10(4) M(-1) s(-1) and a slow deacylation rate (koff) of 7 ± 1 × 10(-4) s(-1).  IC50 was 3 nM for LN-1-255 and 1.5 μM for tazobactam.  Lastly, kcat/kinact was 500-fold lower for LN-1-255 than for tazobactam.  CONCLUSIONS:  In these studies, carbapenem antibiotics used in combination with LN-1-255 are effective against the carbapenemase OXA-48, an important emerging mechanism of antibiotic resistance.  This provides an incentive for further investigations to maximize the efficacy of penicillin sulfone inhibition of class D plasmid-carried Enterobacteriaceae carbapenemases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT3yVfbQnR8dEbxh97K6Vt6fW6udTcc2eZD_K_rMq7jibntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bktlGguw%253D%253D&md5=7328b8a9d881b23e54ba861a35072802</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkw105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkw105%26sid%3Dliteratum%253Aachs%26aulast%3DVallejo%26aufirst%3DJ.%2BA.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DPoza%26aufirst%3DM.%26aulast%3DBuynak%26aufirst%3DJ.%2BD.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DBeceiro%26aufirst%3DA.%26atitle%3DLN-1-255%252C%2520a%2520penicillanic%2520acid%2520sulfone%2520able%2520to%2520inhibit%2520the%2520class%2520D%2520carbapenemase%2520OXA-48%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2016%26volume%3D71%26spage%3D2171%26epage%3D2180%26doi%3D10.1093%2Fjac%2Fdkw105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span> <span> </span><span class="NLM_article-title">Activity of the β-Lactamase Inhibitor LN-1-255 against Carbapenem-Hydrolyzing Class D β-Lactamases from Acinetobacter baumannii</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e01172</span> <span class="refDoi"> DOI: 10.1128/aac.01172-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01172-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=28807908" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=J.+C.+V%C3%A1zquez-Uchaauthor=M.+Maneiroauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=J.+Buynakauthor=C.+R.+Bethelauthor=R.+A.+Bonomoauthor=G.+Bouauthor=M.+Pozaauthor=C.+Gonz%C3%A1lez-Belloauthor=A.+Beceiro&title=Activity+of+the+%CE%B2-Lactamase+Inhibitor+LN-1-255+against+Carbapenem-Hydrolyzing+Class+D+%CE%B2-Lactamases+from+Acinetobacter+baumannii&doi=10.1128%2Faac.01172-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1128%2FAAC.01172-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01172-17%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DBuynak%26aufirst%3DJ.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DPoza%26aufirst%3DM.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DBeceiro%26aufirst%3DA.%26atitle%3DActivity%2520of%2520the%2520%25CE%25B2-Lactamase%2520Inhibitor%2520LN-1-255%2520against%2520Carbapenem-Hydrolyzing%2520Class%2520D%2520%25CE%25B2-Lactamases%2520from%2520Acinetobacter%2520baumannii%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2Faac.01172-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conde-Pérez, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Álvarez-Fraga, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span> <span> </span><span class="NLM_article-title">Therapeutic efficacy of LN-1-255 in combination with imipenem in severe infection caused by carbapenem-resistant <i>Acinetobacter baumannii</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>63</i></span>, <span class="NLM_elocation-id">e01092</span> <span class="refDoi"> DOI: 10.1128/aac.01092-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01092-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31383666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1eisbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2019&author=J.+C.+V%C3%A1zquez-Uchaauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=M.+Maneiroauthor=K.+Conde-P%C3%A9rezauthor=L.+%C3%81lvarez-Fragaauthor=G.+Torrensauthor=A.+Oliverauthor=J.+D.+Buynakauthor=R.+A.+Bonomoauthor=G.+Bouauthor=C.+Gonz%C3%A1lez-Belloauthor=M.+Pozaauthor=A.+Beceiro&title=Therapeutic+efficacy+of+LN-1-255+in+combination+with+imipenem+in+severe+infection+caused+by+carbapenem-resistant+Acinetobacter+baumannii&doi=10.1128%2Faac.01092-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic efficacy of LN-1-255 in combination with imipenem in severe infection caused by carbapenem-resistant Acinetobacter baumannii</span></div><div class="casAuthors">Vazquez-Ucha, Juan Carlos; Martinez-Guitian, Marta; Maneiro, Maria; Conde-Perez, Kelly; Alvarez-raga, Laura; Torrens, Gabriel; Oliver, Antonio; Buynak, John D.; Bonomo, Robert A.; Bou, German; Gonzalez-Bello, Concepcion; Poza, Margarita; Beceiroa, Alejandro</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e01092-19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The carbapenem-hydrolyzing class D β-lactamases (CHDLs) are the main mechanism of carbapenem resistance in Acinetobacter baumannii.  CHDLs are not effectively inactivated by clin. available β-lactam-type inhibitors.  We have previously described the in vitro efficacy of the inhibitor LN-1-255 in combination with carbapenems.  The aim of this study was to compare the efficacy of LN-1-255 with that of imipenem in murine pneumonia using A. baumannii strains carrying their most extended carbapenemases, OXA-23 and OXA-24/40.  The blaOXA-23 and blaOXA-24/40 genes were cloned into the carbapenem-susceptible A. baumannii ATCC 17978 strain.  Clin. isolates Ab1 and JC12/04, producing the enzymes OXA-23 and OXA-24/40, resp., were used in the study.  Pharmacokinetic (PK) parameters were detd.  An exptl. pneumonia model was used to evaluate the efficacy of the combined imipenem-LN-1-255 therapy.  MICs of imipenem decreased between 32- and 128-fold in the presence of LN-1-255.  I.m. treatment with imipenem-LN-1-255 (30/50 mg/kg) decreased the bacterial burden by (i) 4 and 1.7 log10 CFU/g lung in the infection with the ATCC 17978-OXA-23 and Ab1 strains, resp., and by (ii) 2.5 and 4.5 log10 CFU/g lung in the infection produced by the ATCC 17978-OXA-24/40 and the JC12/04 strains, resp.  In all assays, combined therapy offered higher protection against pneumonia than that provided by monotherapy.  No toxicity was obsd. in treated mice.  Imipenem treatment combined with LN-1-255 treatment significantly reduced the severity of infection by carbapenem-resistant A. baumannii strains carrying CHDLs.  Preclin. assays demonstrated the potential of LN-1-255 and imipenem therapy as a new antibacterial treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtx5V3Y8kjfbVg90H21EOLACvtfcHk0ljVN7FWvvweUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1eisbrJ&md5=c53b4095cd6a9de4639d125e2e07b333</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1128%2FAAC.01092-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01092-19%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DConde-P%25C3%25A9rez%26aufirst%3DK.%26aulast%3D%25C3%2581lvarez-Fraga%26aufirst%3DL.%26aulast%3DTorrens%26aufirst%3DG.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DBuynak%26aufirst%3DJ.%2BD.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DPoza%26aufirst%3DM.%26aulast%3DBeceiro%26aufirst%3DA.%26atitle%3DTherapeutic%2520efficacy%2520of%2520LN-1-255%2520in%2520combination%2520with%2520imipenem%2520in%2520severe%2520infection%2520caused%2520by%2520carbapenem-resistant%2520Acinetobacter%2520baumannii%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D63%26doi%3D10.1128%2Faac.01092-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santillana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drawz, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagadala, S. R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Akker, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J. D.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Crystal Structures of 6-Alkylidene-2′-Substituted Penicillanic Acid Sulfones as Potent Inhibitors ofAcinetobacter baumanniiOXA-24 Carbapenemase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">13320</span>– <span class="NLM_lpage">13331</span>, <span class="refDoi"> DOI: 10.1021/ja104092z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja104092z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFWru7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=13320-13331&author=G.+Bouauthor=E.+Santillanaauthor=A.+Sheriauthor=A.+Beceiroauthor=J.+M.+Sampsonauthor=M.+Kalpauthor=C.+R.+Bethelauthor=A.+M.+Distlerauthor=S.+M.+Drawzauthor=S.+R.+R.+Pagadalaauthor=F.+van+den+Akkerauthor=R.+A.+Bonomoauthor=A.+Romeroauthor=J.+D.+Buynak&title=Design%2C+Synthesis%2C+and+Crystal+Structures+of+6-Alkylidene-2%E2%80%B2-Substituted+Penicillanic+Acid+Sulfones+as+Potent+Inhibitors+ofAcinetobacter+baumanniiOXA-24+Carbapenemase&doi=10.1021%2Fja104092z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Crystal Structures of 6-Alkylidene-2'-Substituted Penicillanic Acid Sulfones as Potent Inhibitors of Acinetobacter baumannii OXA-24 Carbapenemase</span></div><div class="casAuthors">Bou, German; Santillana, Elena; Sheri, Anjaneyulu; Beceiro, Alejandro; Sampson, Jared M.; Kalp, Matthew; Bethel, Christopher R.; Distler, Anne M.; Drawz, Sarah M.; Pagadala, Sundar Ram Reddy; van den Akker, Focco; Bonomo, Robert A.; Romero, Antonio; Buynak, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">13320-13331</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Class D β-lactamases represent a growing and diverse class of penicillin-inactivating enzymes that are usually resistant to com. β-lactamase inhibitors.  As many such enzymes are found in multi-drug resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa, novel β-lactamase inhibitors are urgently needed.  Five unique 6-alkylidene-2'-substituted penicillanic acid sulfones I (R = H, R1 = CH2OCOCH2C6H3(OH)2-3,4, X = Na) (II), I (R = NHCO2Me, R1 = Me, X = Na; R = CONH2, R1 = CH2OCONH2, X = Na), I (R = H, R1 = CH2OCONHCH2R2, X = Na, R2 = 2-amino-thiazol-4-yl) (III), I (R = H, R1 = CH2OCONHCH2CH2NH3+, X = none) (IV) were synthesized and tested against OXA-24, a clin. important β-lactamase that inactivates carbapenems and is found in A. baumannii.  Based upon the roles Tyr112 and Met223 play in the OXA-24 β-lactamase, we also engineered two variants (Tyr112Ala and Tyr112Ala,Met223Ala) to test the hypothesis that the hydrophobic tunnel formed by these residues influences inhibitor recognition.  IC50 values against OXA-24 and two OXA-24 β-lactamase variants ranged from 10 ± 1 (III vs. WT) to 338 ± 20 nM (IV vs. Tyr112Ala, Met223Ala).  Compd. III possessed the lowest Ki (500 ± 80 nM vs. WT), and II possessed the highest inactivation efficiency (kinact/Ki = 0.21 ± 0.02 μM-1 s-1).  Electrospray ionization mass spectrometry revealed a single covalent adduct, suggesting the formation of an acyl-enzyme intermediate.  X-ray structures of OXA-24 complexed to four inhibitors (2.0-2.6 Å) reveal the formation of stable bicyclic arom. intermediates with their carbonyl oxygen in the oxyanion hole.  These data provide the first structural evidence that 6-alkylidene-2'-substituted penicillin sulfones are effective mechanism-based inactivators of class D β-lactamases.  Their unique chem. makes them developmental candidates.  Mechanisms for class D hydrolysis and inhibition are discussed, and a pathway for the evolution of the BlaR1 sensor of Staphylococcus aureus to the class D β-lactamases is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ0f6jmKDmVrVg90H21EOLACvtfcHk0ljVN7FWvvweUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFWru7vO&md5=71fd4b9382c882f6bed9255305c40b24</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja104092z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja104092z%26sid%3Dliteratum%253Aachs%26aulast%3DBou%26aufirst%3DG.%26aulast%3DSantillana%26aufirst%3DE.%26aulast%3DSheri%26aufirst%3DA.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DSampson%26aufirst%3DJ.%2BM.%26aulast%3DKalp%26aufirst%3DM.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DDistler%26aufirst%3DA.%2BM.%26aulast%3DDrawz%26aufirst%3DS.%2BM.%26aulast%3DPagadala%26aufirst%3DS.%2BR.%2BR.%26aulast%3Dvan%2Bden%2BAkker%26aufirst%3DF.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DRomero%26aufirst%3DA.%26aulast%3DBuynak%26aufirst%3DJ.%2BD.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Crystal%2520Structures%2520of%25206-Alkylidene-2%25E2%2580%25B2-Substituted%2520Penicillanic%2520Acid%2520Sulfones%2520as%2520Potent%2520Inhibitors%2520ofAcinetobacter%2520baumanniiOXA-24%2520Carbapenemase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D13320%26epage%3D13331%26doi%3D10.1021%2Fja104092z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span> <span> </span><span class="NLM_article-title">6-Arylmethylidene penicillin-based sulfone inhibitors for repurposing antibiotic efficiency in priority pathogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3737</span>– <span class="NLM_lpage">3755</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00127</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00127" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1CrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3737-3755&author=D.+Rodr%C3%ADguezauthor=M.+Maneiroauthor=J.+C.+V%C3%A1zquez-Uchaauthor=A.+Beceiroauthor=C.+Gonz%C3%A1lez-Bello&title=6-Arylmethylidene+penicillin-based+sulfone+inhibitors+for+repurposing+antibiotic+efficiency+in+priority+pathogens&doi=10.1021%2Facs.jmedchem.0c00127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">6-Arylmethylidene Penicillin-Based Sulfone Inhibitors for Repurposing Antibiotic Efficiency in Priority Pathogens</span></div><div class="casAuthors">Rodriguez, Diana; Maneiro, Maria; Vazquez-Ucha, Juan C.; Beceiro, Alejandro; Gonzalez-Bello, Concepcion</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3737-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability of 6-(aryl)methylidene penicillin-based sulfones 1-7 to repurpose β-lactam antibiotics activity with bacterial species that carry carbapenem-hydrolyzing class D β-lactamases (OXA-23, OXA-24/40 and OXA-48), as well as with class A (TEM-1, CTX-M-2) and class C (CMY-2, DHA-1) enzymes, is reported.  The combinations imipenem/3 and imipenem/4 restored almost completely the antibiotic efficacy in OXA-23 and OXA-24/40 carbapenemase-producing A. baumannii strains (1μg mL-1) and also provided good results for OXA-48 carbapenemase-producing K. pneumoniae strains (4μg mL-1).  Compds. 2-6 in combinations with ceftazidime and ampicillin were also efficient in restoring antibiotic efficacy in E. coli strains carrying class C (CMY-2 and DHA-1) and class A (TEM-1 and CTX-M-2) β-lactamase enzymes, resp.  Kinetic and inhibition studies with the OXA-24/40 enzyme, protein mass spectrometry anal. and docking studies allowed us to gain an insight into the inhibition mechanism and the exptl. obsd. differences between the ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP87HcFlDbT7Vg90H21EOLACvtfcHk0lhfQk3eiwAGow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1CrtLg%253D&md5=1d6723e5b18ce3850d77c2b9b226913e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00127%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26atitle%3D6-Arylmethylidene%2520penicillin-based%2520sulfone%2520inhibitors%2520for%2520repurposing%2520antibiotic%2520efficiency%2520in%2520priority%2520pathogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3737%26epage%3D3755%26doi%3D10.1021%2Facs.jmedchem.0c00127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Torrens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocampo-Sosa, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamorano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascual, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martínez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Activity of Ceftazidime-Avibactam against Clinical and Isogenic Laboratory Pseudomonas aeruginosa Isolates Expressing Combinations of Most Relevant β-Lactam Resistance Mechanisms</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6407</span>– <span class="NLM_lpage">6410</span>, <span class="refDoi"> DOI: 10.1128/aac.01282-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01282-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=27480848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKlsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=6407-6410&author=G.+Torrensauthor=G.+Cabotauthor=A.+A.+Ocampo-Sosaauthor=M.+C.+Conejoauthor=L.+Zamoranoauthor=F.+Navarroauthor=%C3%81.+Pascualauthor=L.+Mart%C3%ADnez-Mart%C3%ADnezauthor=A.+Oliver&title=Activity+of+Ceftazidime-Avibactam+against+Clinical+and+Isogenic+Laboratory+Pseudomonas+aeruginosa+Isolates+Expressing+Combinations+of+Most+Relevant+%CE%B2-Lactam+Resistance+Mechanisms&doi=10.1128%2Faac.01282-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Activity of ceftazidime-avibactam against clinical and isogenic laboratory Pseudomonas aeruginosa isolates expressing combinations of most relevant β-lactam resistance mechanisms</span></div><div class="casAuthors">Torrens, Gabriel; Cabot, Gabriel; Ocampo-Sosa, Alain A.; Carmen, Conejo M.; Zamorano, Laura; Navarro, Ferran; Pascual, Alvaro; Martinez-Martinez, Luis; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6407-6410</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The activity of ceftazidime-avibactam was compared with that of ceftazidime alone and meropenem against a collection of 190 Pseudomonas aeruginosa clin. isolates recovered from a multicenter study of bloodstream infections.  The addn. of avibactam increased ceftazidime susceptibility in the complete collection of strains (64.7-91.1%) and particularly among subsets of isolates showing AmpC hyperprodn. (10.9-76.1%) or multidrug resistance (MDR) profiles (27-77.8%).  The MICs of ceftazidime-avibactam, in contrast with those of ceftazidime or meropenem, remained at ≤4 μg/mL for a panel of 16 P. aeruginosa PAO1 isogenic mutants expressing multiple combinations of the most relevant β-lactam resistance mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVNemhW4E8SrVg90H21EOLACvtfcHk0lhfQk3eiwAGow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKlsrk%253D&md5=e025b2d23eed307200d8db9229e87a34</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1128%2FAAC.01282-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01282-16%26sid%3Dliteratum%253Aachs%26aulast%3DTorrens%26aufirst%3DG.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DOcampo-Sosa%26aufirst%3DA.%2BA.%26aulast%3DConejo%26aufirst%3DM.%2BC.%26aulast%3DZamorano%26aufirst%3DL.%26aulast%3DNavarro%26aufirst%3DF.%26aulast%3DPascual%26aufirst%3D%25C3%2581.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADnez%26aufirst%3DL.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DActivity%2520of%2520Ceftazidime-Avibactam%2520against%2520Clinical%2520and%2520Isogenic%2520Laboratory%2520Pseudomonas%2520aeruginosa%2520Isolates%2520Expressing%2520Combinations%2520of%2520Most%2520Relevant%2520%25CE%25B2-Lactam%2520Resistance%2520Mechanisms%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D6407%26epage%3D6410%26doi%3D10.1128%2Faac.01282-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R. A.</span></span> <span> </span><span class="NLM_article-title">Avibactam and Class C β-Lactamases: Mechanism of Inhibition, Conservation of the Binding Pocket, and Implications for Resistance</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">5704</span>– <span class="NLM_lpage">5713</span>, <span class="refDoi"> DOI: 10.1128/aac.03057-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.03057-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25022578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslShsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=5704-5713&author=S.+D.+Lahiriauthor=M.+R.+Johnstoneauthor=P.+L.+Rossauthor=R.+E.+McLaughlinauthor=N.+B.+Olivierauthor=R.+A.+Alm&title=Avibactam+and+Class+C+%CE%B2-Lactamases%3A+Mechanism+of+Inhibition%2C+Conservation+of+the+Binding+Pocket%2C+and+Implications+for+Resistance&doi=10.1128%2Faac.03057-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Avibactam and class C β-lactamases: mechanism of inhibition, conservation of the binding pocket, and implications for resistance</span></div><div class="casAuthors">Lahiri, S. D.; Johnstone, M. R.; Ross, P. L.; McLaughlin, R. E.; Olivier, N. B.; Alm, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5704-5713, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Avibactam is a novel non-β-lactam β-lactamase inhibitor that inhibits a wide range of β-lactamases.  These include class A, class C, and some class D enzymes, which erode the activity of β-lactam drugs in multidrug-resistant pathogens like Pseudomonas aeruginosa and Enterobacteriaceae spp.  Avibactam is currently in clin. development in combination with the β-lactam antibiotics ceftazidime, ceftaroline fosamil, and aztreonam.  Avibactam has the potential to be the first β-lactamase inhibitor that might provide activity against class C-mediated resistance, which represents a growing concern in both hospital- and community-acquired infections.  Avibactam has an unusual mechanism of action: it is a covalent inhibitor that acts via ring opening, but in contrast to other currently used β-lactamase inhibitors, this reaction is reversible.  Here, we present a high-resoln. structure of avibactam bound to a class C β-lactamase, AmpC, from P. aeruginosa that provided insight into the mechanism of both acylation and recyclization in this enzyme class and highlighted the differences obsd. between class A and class C inhibition.  Furthermore, variants resistant to avibactam that identified the residues important for inhibition were isolated.  Finally, the structural information was used to predict effective inhibition by sequence anal. and functional studies of class C β-lactamases from a large and diverse set of contemporary clin. isolates (P. aeruginosa and several Enterobacteriaceae spp.) obtained from recent infections to understand any preexisting variability in the binding pocket that might affect inhibition by avibactam.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo83AV3Iqfuf7Vg90H21EOLACvtfcHk0lhfQk3eiwAGow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslShsr3F&md5=d31d11de33526c1bcfb958c1516001fb</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2FAAC.03057-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.03057-14%26sid%3Dliteratum%253Aachs%26aulast%3DLahiri%26aufirst%3DS.%2BD.%26aulast%3DJohnstone%26aufirst%3DM.%2BR.%26aulast%3DRoss%26aufirst%3DP.%2BL.%26aulast%3DMcLaughlin%26aufirst%3DR.%2BE.%26aulast%3DOlivier%26aufirst%3DN.%2BB.%26aulast%3DAlm%26aufirst%3DR.%2BA.%26atitle%3DAvibactam%2520and%2520Class%2520C%2520%25CE%25B2-Lactamases%253A%2520Mechanism%2520of%2520Inhibition%252C%2520Conservation%2520of%2520the%2520Binding%2520Pocket%252C%2520and%2520Implications%2520for%2520Resistance%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D5704%26epage%3D5713%26doi%3D10.1128%2Faac.03057-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahić, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, R.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durand-Réville, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thresher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livchak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkup, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, S. L.</span></span> <span> </span><span class="NLM_article-title">Kinetics of Avibactam Inhibition against Class A, C, and D β-Lactamases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">27960</span>– <span class="NLM_lpage">27971</span>, <span class="refDoi"> DOI: 10.1074/jbc.m113.485979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1074%2Fjbc.M113.485979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=23913691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFemsr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=27960-27971&author=D.+E.+Ehmannauthor=H.+Jahi%C4%87author=P.+L.+Rossauthor=R.-F.+Guauthor=J.+Huauthor=T.+F.+Durand-R%C3%A9villeauthor=S.+Lahiriauthor=J.+Thresherauthor=S.+Livchakauthor=N.+Gaoauthor=T.+Palmerauthor=G.+K.+Walkupauthor=S.+L.+Fisher&title=Kinetics+of+Avibactam+Inhibition+against+Class+A%2C+C%2C+and+D+%CE%B2-Lactamases&doi=10.1074%2Fjbc.m113.485979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of Avibactam Inhibition against Class A, C, and D β-Lactamases</span></div><div class="casAuthors">Ehmann, David E.; Jahic, Haris; Ross, Philip L.; Gu, Rong-Fang; Hu, Jun; Durand-Reville, Thomas F.; Lahiri, Sushmita; Thresher, Jason; Livchak, Stephania; Gao, Ning; Palmer, Tiffany; Walkup, Grant K.; Fisher, Stewart L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">27960-27971</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Avibactam is a non-β-lactam β-lactamase inhibitor with a spectrum of activity that includes β-lactamase enzymes of classes A, C, and selected D examples.  In this work acylation and deacylation rates were measured against the clin. important enzymes CTX-M-15, KPC-2, Enterobacter cloacae AmpC, Pseudomonas aeruginosa AmpC, OXA-10, and OXA-48.  The efficiency of acylation (k2/Ki) varied across the enzyme spectrum, from 1.1 × 101m-1s-1 for OXA-10 to 1.0 × 105 for CTX-M-15.  Inhibition of OXA-10 was shown to follow the covalent reversible mechanism, and the acylated OXA-10 displayed the longest residence time for deacylation, with a half-life of > 5 days.  Across multiple enzymes, acyl enzyme stability was assessed by mass spectrometry.  These inhibited enzyme forms were stable to rearrangement or hydrolysis, with the exception of KPC-2.  KPC-2 displayed a slow hydrolytic route that involved fragmentation of the acyl-avibactam complex.  The identity of released degrdn. products was investigated, and a possible mechanism for the slow deacylation from KPC-2 is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoULb6Pb1j63bVg90H21EOLACvtfcHk0lik8I50G_LKsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFemsr%252FL&md5=baa5d471bb16d83c797cfa9069b950f8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.485979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.485979%26sid%3Dliteratum%253Aachs%26aulast%3DEhmann%26aufirst%3DD.%2BE.%26aulast%3DJahi%25C4%2587%26aufirst%3DH.%26aulast%3DRoss%26aufirst%3DP.%2BL.%26aulast%3DGu%26aufirst%3DR.-F.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DDurand-R%25C3%25A9ville%26aufirst%3DT.%2BF.%26aulast%3DLahiri%26aufirst%3DS.%26aulast%3DThresher%26aufirst%3DJ.%26aulast%3DLivchak%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DPalmer%26aufirst%3DT.%26aulast%3DWalkup%26aufirst%3DG.%2BK.%26aulast%3DFisher%26aufirst%3DS.%2BL.%26atitle%3DKinetics%2520of%2520Avibactam%2520Inhibition%2520against%2520Class%2520A%252C%2520C%252C%2520and%2520D%2520%25CE%25B2-Lactamases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D27960%26epage%3D27971%26doi%3D10.1074%2Fjbc.m113.485979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gootz, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drawz, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harry, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nottingham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">Inactivation of a class A and a class C β-lactamase by 6β-(hydroxymethyl)penicillanic acid sulfone</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2011.11.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.bcp.2011.11.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=22155308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1Glsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2012&pages=462-471&author=K.+M.+Papp-Wallaceauthor=C.+R.+Bethelauthor=T.+D.+Gootzauthor=W.+Shangauthor=J.+Strohauthor=W.+Lauauthor=D.+McLeodauthor=L.+Priceauthor=A.+Marfatauthor=A.+Distlerauthor=S.+M.+Drawzauthor=H.+Chenauthor=E.+Harryauthor=M.+Nottinghamauthor=P.+R.+Careyauthor=J.+D.+Buynakauthor=R.+A.+Bonomo&title=Inactivation+of+a+class+A+and+a+class+C+%CE%B2-lactamase+by+6%CE%B2-%28hydroxymethyl%29penicillanic+acid+sulfone&doi=10.1016%2Fj.bcp.2011.11.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Inactivation of a class A and a class C β-lactamase by 6β-(hydroxymethyl)penicillanic acid sulfone</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.; Bethel, Christopher R.; Gootz, Thomas D.; Shang, Wenchi; Stroh, Justin; Lau, William; McLeod, Dale; Price, Loren; Marfat, Anthony; Distler, Anne; Drawz, Sarah M.; Chen, Hansong; Harry, Emily; Nottingham, Micheal; Carey, Paul R.; Buynak, John D.; Bonomo, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">462-471</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">β-Lactamase inhibitors (clavulanic acid, sulbactam, and tazobactam) contribute significantly to the longevity of the β-lactam antibiotics used to treat serious infections.  In the quest to design more potent compds. and to understand the mechanism of action of known inhibitors, 6β-(hydroxymethyl)penicillanic acid sulfone (6β-HM-sulfone) was tested against isolates expressing the class A TEM-1 β-lactamase and a clin. important variant of the AmpC cephalosporinase of Pseudomonas aeruginosa, PDC-3.  The addn. of the 6β-HM-sulfone inhibitor to ampicillin was highly effective.  6β-HM-sulfone inhibited TEM-1 with an IC50 of 12 ± 2 nM and PDC-3 with an IC50 of 180 ± 36 nM, and displayed lower partition ratios than com. inhibitors, with partition ratios (kcat/kinact) equal to 174 for TEM-1 and 4 for PDC-3.  Measured for 20 h, 6β-HM-sulfone demonstrated rapid, first-order inactivation kinetics with the extent of inactivation being related to the concn. of inhibitor for both TEM-1 and PDC-3.  Using mass spectrometry to gain insight into the intermediates of inactivation of this inhibitor, 6β-HM-sulfone was found to form a major adduct of +247 ± 5 Da with TEM-1 and +245 ± 5 Da with PDC-3, suggesting that the covalently bound, hydrolytically stabilized acyl-enzyme has lost a mol. of water (H-O-H).  Minor adducts of +88 ± 5 Da with TEM-1 and +85 ± 5 Da with PDC-3 revealed that fragmentation of the covalent adduct can result but appeared to occur slowly with both enzymes. 6β-HM-sulfone is an effective and versatile β-lactamase inhibitor of representative class A and C enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrekFlbHHVjjLVg90H21EOLACvtfcHk0lik8I50G_LKsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1Glsg%253D%253D&md5=f425d156966894d16d418b74402d2360</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2011.11.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2011.11.015%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DGootz%26aufirst%3DT.%2BD.%26aulast%3DShang%26aufirst%3DW.%26aulast%3DStroh%26aufirst%3DJ.%26aulast%3DLau%26aufirst%3DW.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DPrice%26aufirst%3DL.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DDistler%26aufirst%3DA.%26aulast%3DDrawz%26aufirst%3DS.%2BM.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DHarry%26aufirst%3DE.%26aulast%3DNottingham%26aufirst%3DM.%26aulast%3DCarey%26aufirst%3DP.%2BR.%26aulast%3DBuynak%26aufirst%3DJ.%2BD.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DInactivation%2520of%2520a%2520class%2520A%2520and%2520a%2520class%2520C%2520%25CE%25B2-lactamase%2520by%25206%25CE%25B2-%2528hydroxymethyl%2529penicillanic%2520acid%2520sulfone%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2012%26volume%3D83%26spage%3D462%26epage%3D471%26doi%3D10.1016%2Fj.bcp.2011.11.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte-Brasseur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span> <span> </span><span class="NLM_article-title">pKa calculations for class C beta-lactamases: The role of tyr-150</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1002/(sici)1097-0134(20000701)40:1<23::aid-prot40>3.0.co;2-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1002%2F%28SICI%291097-0134%2820000701%2940%3A1%3C23%3A%3AAID-PROT40%3E3.0.CO%3B2-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10813827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjvFyjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2000&pages=23-28&author=J.+Lamotte-Brasseurauthor=A.+Dubusauthor=R.+C.+Wade&title=pKa+calculations+for+class+C+beta-lactamases%3A+The+role+of+tyr-150&doi=10.1002%2F%28sici%291097-0134%2820000701%2940%3A1%3C23%3A%3Aaid-prot40%3E3.0.co%3B2-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">pKa calculations for class C β-lactamases: the role of Tyr-150</span></div><div class="casAuthors">Lamotte-Brasseur, Josette; Dubus, Alain; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-28</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Poisson-Boltzmann method was used to compute the pKa values of titratable residues in a set of class C β-lactamases.  In these calcns., the pKa of the phenolic group of residue Tyr150 is the only one to stand out with an abnormally low value of 8.3, more than one pKa unit lower than the measured ref. value for tyrosine in soln.  Other important residues of the catalytic pocket, such as the conserved Lys67, Lys315, His314, and Glu272 (hydrogen-bonded to the ammonium group of Lys315), display normal protonation states at neutral pH.  PKa values were also computed in catalytically impaired β-lactamase mutants.  Comparisons between the relative kcat values and the Tyr150 pKa value in these mutants revealed a striking correlation.  In active enzymes, this pKa value is always lower than the soln. ref. value while it is close to normal in inactive enzymes.  These results thus support the hypothesis that the phenolate form of Tyr150 is responsible for the activation of the nucleophilic serine.  The possible roles of Lys67 and Lys315 during catalysis are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp31t-YKgoazLVg90H21EOLACvtfcHk0lik8I50G_LKsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjvFyjuro%253D&md5=8a8f368139b15426075c00a0737cf4c2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2820000701%2940%3A1%3C23%3A%3AAID-PROT40%3E3.0.CO%3B2-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252820000701%252940%253A1%253C23%253A%253AAID-PROT40%253E3.0.CO%253B2-7%26sid%3Dliteratum%253Aachs%26aulast%3DLamotte-Brasseur%26aufirst%3DJ.%26aulast%3DDubus%26aufirst%3DA.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DpKa%2520calculations%2520for%2520class%2520C%2520beta-lactamases%253A%2520The%2520role%2520of%2520tyr-150%26jtitle%3DProteins%26date%3D2000%26volume%3D40%26spage%3D23%26epage%3D28%26doi%3D10.1002%2F%28sici%291097-0134%2820000701%2940%3A1%3C23%3A%3Aaid-prot40%3E3.0.co%3B2-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dubus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte-Brasseur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frère, J.-M.</span></span> <span> </span><span class="NLM_article-title">The roles of residues Tyr150, Glu272, and His314 in class C β-lactamases</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1002/prot.7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1002%2Fprot.7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=8865342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaK28Xlt1ymsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1996&pages=473-485&author=A.+Dubusauthor=P.+Ledentauthor=J.+Lamotte-Brasseurauthor=J.-M.+Fr%C3%A8re&title=The+roles+of+residues+Tyr150%2C+Glu272%2C+and+His314+in+class+C+%CE%B2-lactamases&doi=10.1002%2Fprot.7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The roles of residues Tyr150, Glu272, and His314 in class C β-lactamases</span></div><div class="casAuthors">Dubus, Alain; Ledent, Philippe; Lamotte-Brasseur, Josette; Frere, Jean-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-485</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss</span>)
        </div><div class="casAbstract">Serine β-lactamases contribute widely to the β-lactam resistance phenomena.  Unfortunately, the intimate details of their catalytic mechanism remain elusive and subject to some controversy even though many "natural" and "artificial" mutants of these different enzymes have been isolated.  This paper is essentially focused on class C β-lactamases, which contain a Tyr (Tyr 150) as the first residue of the second conserved element, in contrast to their class A counterparts, in which a Ser is found in the corresponding position.  The authors have modified this Tyr residue by site-directed mutagenesis.  On the basis of the three-dimensional structure of the Enterobacter cloacae P99 enzyme, it seem that residues Glu 272 and His 314 might also be important.  They were similarly substituted.  The modified enzymes were isolated and their catalytic properties detd.  The authors' results indicated that His 314 was not required for catalysis and that Glu 272 did not play an important role in acylation but was involved to a small extent in the deacylation process.  Conversely, Tyr 150 was confirmed to be central for catalysis, at least with the best substrates.  On the basis of the comparison of data obtained for several class C enzyme mutants and in agreement with recent structural data, the authors propose that the phenolate anion of Tyr 150, in conjunction with the alkyl ammonium of Lys 315, acts as the general base responsible for the activation of the active-site Ser 64 during the acylation step and for the subsequent activation of a water mol. in the deacylation process.  The evolution of the important superfamily of penicillin-recognizing enzymes is further discussed in the light of this proposed mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFCLAkWT7yZbVg90H21EOLACvtfcHk0lhU7kjj-Rp84w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xlt1ymsrk%253D&md5=20d4c25b4816b7220e7630c3556d703f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fprot.7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.7%26sid%3Dliteratum%253Aachs%26aulast%3DDubus%26aufirst%3DA.%26aulast%3DLedent%26aufirst%3DP.%26aulast%3DLamotte-Brasseur%26aufirst%3DJ.%26aulast%3DFr%25C3%25A8re%26aufirst%3DJ.-M.%26atitle%3DThe%2520roles%2520of%2520residues%2520Tyr150%252C%2520Glu272%252C%2520and%2520His314%2520in%2520class%2520C%2520%25CE%25B2-lactamases%26jtitle%3DProteins%26date%3D1996%26volume%3D25%26spage%3D473%26epage%3D485%26doi%3D10.1002%2Fprot.7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato-Toma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwashita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishiguro, M.</span></span> <span> </span><span class="NLM_article-title">pKa measurements from nuclear magnetic resonance of tyrosine-150 in class C beta-lactamase</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1042/bj20021447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1042%2Fbj20021447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=12513696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlCitr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2003&pages=175-181&author=Y.+Kato-Tomaauthor=T.+Iwashitaauthor=K.+Masudaauthor=Y.+Oyamaauthor=M.+Ishiguro&title=pKa+measurements+from+nuclear+magnetic+resonance+of+tyrosine-150+in+class+C+beta-lactamase&doi=10.1042%2Fbj20021447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">pKa measurements from nuclear magnetic resonance of tyrosine-150 in class C β-lactamase</span></div><div class="casAuthors">Kato-Toma, Yoko; Iwashita, Takashi; Masuda, Katsuyoshi; Oyama, Yoshiaki; Ishiguro, Masaji</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">13C-NMR spectroscopy was used to est. the pKa values for the Tyr150 (Y150) residue in wild-type and mutant class C β-lactamases.  The tyrosine residues of the wild-type and mutant lactamases were replaced with 13C-labeled L-tyrosine ([phenol -4-13C]tyrosine) in order to observe the tyrosine residues selectively.  Spectra of the wild-type and K67C mutant (Lys67Cys) enzyme were compared with the Y150C mutant lactamase spectra to identify the signal originating from Tyr150.  Titrn. expts. showed that the chem. shift of the Tyr150 resonance in the wild-type enzyme is almost invariant in a range of 0.1 p.p.m. up to pH 11 and showed that the pKa of this residue is well above 11 in the substrate-free form.  According to solvent accessibility calcns. on X-ray-derived structures, the phenolic oxygen of Tyr150, which is near the amino groups of Lys315 and Lys67, appears to have low solvent accessibility.  These results suggest that, in the native enzyme, Tyr150 in class C β-lactamase of Citrobacter freundii GN346 is protonated and that when Tyr150 loses a proton, a proton from Lys67 would replace it.  Consequently, Tyr150 would be protonated during the entire titrn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbUIUqZZcdNLVg90H21EOLACvtfcHk0lhU7kjj-Rp84w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlCitr0%253D&md5=9b6476d674e3195764bbe05604863f53</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1042%2Fbj20021447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20021447%26sid%3Dliteratum%253Aachs%26aulast%3DKato-Toma%26aufirst%3DY.%26aulast%3DIwashita%26aufirst%3DT.%26aulast%3DMasuda%26aufirst%3DK.%26aulast%3DOyama%26aufirst%3DY.%26aulast%3DIshiguro%26aufirst%3DM.%26atitle%3DpKa%2520measurements%2520from%2520nuclear%2520magnetic%2520resonance%2520of%2520tyrosine-150%2520in%2520class%2520C%2520beta-lactamase%26jtitle%3DBiochem.%2520J.%26date%3D2003%26volume%3D371%26spage%3D175%26epage%3D181%26doi%3D10.1042%2Fbj20021447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durand-Reville, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benvenuti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Luca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanyal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Docquier, J.-D.</span></span> <span> </span><span class="NLM_article-title">Structural Insight into Potent Broad-Spectrum Inhibition with Reversible Recyclization Mechanism: Avibactam in Complex with CTX-M-15 and Pseudomonas aeruginosa AmpC β-Lactamases</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2496</span>– <span class="NLM_lpage">2505</span>, <span class="refDoi"> DOI: 10.1128/aac.02247-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.02247-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=23439634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXoslyrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=2496-2505&author=S.+D.+Lahiriauthor=S.+Manganiauthor=T.+Durand-Revilleauthor=M.+Benvenutiauthor=F.+De+Lucaauthor=G.+Sanyalauthor=J.-D.+Docquier&title=Structural+Insight+into+Potent+Broad-Spectrum+Inhibition+with+Reversible+Recyclization+Mechanism%3A+Avibactam+in+Complex+with+CTX-M-15+and+Pseudomonas+aeruginosa+AmpC+%CE%B2-Lactamases&doi=10.1128%2Faac.02247-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into potent broad-spectrum inhibition with reversible recyclization mechanism: avibactam in complex with CTX-M-15 and Pseudomonas aeruginosa AmpC β-lactamases</span></div><div class="casAuthors">Lahiri, Sushmita D.; Mangani, Stefano; Durand-Reville, Thomas; Benvenuti, Manuela; De Luca, Filomena; Sanyal, Gautam; Docquier, Jean-Denis</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2496-2505</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Although β-lactams have been the most effective class of antibacterial agents used in clin. practice for the past half century, their effectiveness on Gram-neg. bacteria has been eroded due to the emergence and spread of β-lactamase enzymes that are not affected by currently marketed β-lactam/β-lactamase inhibitor combinations.  Avibactam is a novel, covalent, non-β-lactam β-lactamase inhibitor presently in clin. development in combination with either ceftaroline or ceftazidime.  In vitro studies show that avibactam may restore the broad-spectrum activity of cephalosporins against class A, class C, and some class D β-lactamases.  Here we describe the structures of two clin. important β-lactamase enzymes bound to avibactam, the class A CTX-M-15 extended-spectrum β-lactamase and the class C Pseudomonas aeruginosa AmpC β-lactamase, which together provide insight into the binding modes for the resp. enzyme classes.  The structures reveal similar binding modes in both enzymes and thus provide a rationale for the broad-spectrum inhibitory activity of avibactam.  Identification of the key residues surrounding the binding pocket allows for a better understanding of the potency of this scaffold.  Finally, avibactam has recently been shown to be a reversible inhibitor, and the structures provide insights into the mechanism of avibactam recyclization.  Anal. of the ultra-high-resoln. CTX-M-15 structure suggests how the deacylation mechanism favors recyclization over hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBwwMi5MIpF7Vg90H21EOLACvtfcHk0lhU7kjj-Rp84w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXoslyrsrY%253D&md5=c8a5fa5871f2b5d9efc78b6c4c5d7742</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1128%2FAAC.02247-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02247-12%26sid%3Dliteratum%253Aachs%26aulast%3DLahiri%26aufirst%3DS.%2BD.%26aulast%3DMangani%26aufirst%3DS.%26aulast%3DDurand-Reville%26aufirst%3DT.%26aulast%3DBenvenuti%26aufirst%3DM.%26aulast%3DDe%2BLuca%26aufirst%3DF.%26aulast%3DSanyal%26aufirst%3DG.%26aulast%3DDocquier%26aufirst%3DJ.-D.%26atitle%3DStructural%2520Insight%2520into%2520Potent%2520Broad-Spectrum%2520Inhibition%2520with%2520Reversible%2520Recyclization%2520Mechanism%253A%2520Avibactam%2520in%2520Complex%2520with%2520CTX-M-15%2520and%2520Pseudomonas%2520aeruginosa%2520AmpC%2520%25CE%25B2-Lactamases%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D2496%26epage%3D2505%26doi%3D10.1128%2Faac.02247-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0libNiZxVC3tag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olasz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mignon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Proft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veszprémi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geerlings, P.</span></span> <span> </span><span class="NLM_article-title">Effect of the π-π stacking interaction on the acidity of phenol</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>407</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1016/j.cplett.2005.03.145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.cplett.2005.03.145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFWntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=407&publication_year=2005&pages=504-509&author=A.+Olaszauthor=P.+Mignonauthor=F.+De+Proftauthor=T.+Veszpr%C3%A9miauthor=P.+Geerlings&title=Effect+of+the+%CF%80-%CF%80+stacking+interaction+on+the+acidity+of+phenol&doi=10.1016%2Fj.cplett.2005.03.145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of the π-π stacking interaction on the acidity of phenol</span></div><div class="casAuthors">Olasz, Andras; Mignon, Pierre; De Proft, Frank; Veszpremi, Tamas; Geerlings, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">407</span>
        (<span class="NLM_cas:issue">4-6</span>),
    <span class="NLM_cas:pages">504-509</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Stacked complexes made of phenol and substituted benzenes are studied via MP2 calcns. in order to investigate the influence of the stacking interaction on the acidity of phenol.  Aq.-phase proton affinities have been computed and converted to pKa values using a calibration procedure involving a set of substituted phenols.  Both T-shaped and parallel-displaced conformations lead to an appreciable decrease in the pKa, with the largest effect found for T-shaped arrangements.  Local and global DFT based reactivity descriptors are used to analyze the effect of the stacking partner as well as the interaction between the arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MYt4p0z8arVg90H21EOLACvtfcHk0libNiZxVC3tag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFWntbk%253D&md5=f82ba9d5fb5465c78504234b7baa39a0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cplett.2005.03.145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cplett.2005.03.145%26sid%3Dliteratum%253Aachs%26aulast%3DOlasz%26aufirst%3DA.%26aulast%3DMignon%26aufirst%3DP.%26aulast%3DDe%2BProft%26aufirst%3DF.%26aulast%3DVeszpr%25C3%25A9mi%26aufirst%3DT.%26aulast%3DGeerlings%26aufirst%3DP.%26atitle%3DEffect%2520of%2520the%2520%25CF%2580-%25CF%2580%2520stacking%2520interaction%2520on%2520the%2520acidity%2520of%2520phenol%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D2005%26volume%3D407%26spage%3D504%26epage%3D509%26doi%3D10.1016%2Fj.cplett.2005.03.145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lence, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tizón, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otero, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peón, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prazeres, V. F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llamas-Saiz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Raaij, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span> <span> </span><span class="NLM_article-title">Mechanistic basis of the inhibition of type II dehydroquinase by (2<i>S</i>)- and (2<i>R</i>)-2-benzyl-3-dehydroquinic acids</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1021/cb300493s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300493s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCltbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=568-577&author=E.+Lenceauthor=L.+Tiz%C3%B3nauthor=J.+M.+Oteroauthor=A.+Pe%C3%B3nauthor=V.+F.+V.+Prazeresauthor=A.+L.+Llamas-Saizauthor=G.+C.+Foxauthor=M.+J.+van+Raaijauthor=H.+Lambauthor=A.+R.+Hawkinsauthor=C.+Gonz%C3%A1lez-Bello&title=Mechanistic+basis+of+the+inhibition+of+type+II+dehydroquinase+by+%282S%29-+and+%282R%29-2-benzyl-3-dehydroquinic+acids&doi=10.1021%2Fcb300493s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic Basis of the Inhibition of Type II Dehydroquinase by (2S)- and (2R)-2-Benzyl-3-dehydroquinic Acids</span></div><div class="casAuthors">Lence, Emilio; Tizon, Lorena; Otero, Jose M.; Peon, Antonio; Prazeres, Veronica F. V.; Llamas-Saiz, Antonio L.; Fox, Gavin C.; van Raaij, Mark J.; Lamb, Heather; Hawkins, Alastair R.; Gonzalez-Bello, Concepcion</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">568-577</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structural changes caused by the substitution of the arom. moiety in (2S)-2-benzyl-3-dehydroquinic acids and its C2 epimers by electron-withdrawing or electron-donating groups in type II dehydroquinase (DHQ2) enzyme from M. tuberculosis and H. pylori have been investigated by structural and computational studies.  Both compds. are reversible competitive inhibitors of this enzyme, which is essential in these pathogenic bacteria.  The crystal structures of M. tuberculosis and H. pylori DHQ2 enzymes in complex with (2S)-2-(4-methoxy)benzyl- and (2S)-2-perfluorobenzyl-3-dehydroquinic acids have been solved at 2.0, 2.3, 2.0, and 1.9 Å, resp.  The crystal structure of M. tuberculosis DHQ2 in complex with (2R)-2-(benzothiophen-5-yl)methyl-3-dehydroquinic acid is also reported at 1.55 Å.  These crystal structures reveal key differences in the conformation of the flexible loop of the two enzymes, a difference that depends on the presence of electron-withdrawing or electron-donating groups in the arom. moiety of the inhibitors.  This loop closes over the active site after substrate binding, and its flexibility is essential for the function of the enzyme.  These differences have also been investigated by mol. dynamics simulations in an effort to understand the significant inhibition potency differences obsd. between some of these compds. and also to obtain more information about the possible movements of the loop.  These computational studies have also allowed us to identify key structural factors of the H. pylori loop that could explain its reduced flexibility in comparison to the M. tuberculosis loop, specifically by the formation of a key salt bridge between the side chains of residues Asp18 and Arg20.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEZ02X1C6t8LVg90H21EOLACvtfcHk0libNiZxVC3tag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCltbbP&md5=94cfacb63500381e0179b89b0f672cd7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fcb300493s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300493s%26sid%3Dliteratum%253Aachs%26aulast%3DLence%26aufirst%3DE.%26aulast%3DTiz%25C3%25B3n%26aufirst%3DL.%26aulast%3DOtero%26aufirst%3DJ.%2BM.%26aulast%3DPe%25C3%25B3n%26aufirst%3DA.%26aulast%3DPrazeres%26aufirst%3DV.%2BF.%2BV.%26aulast%3DLlamas-Saiz%26aufirst%3DA.%2BL.%26aulast%3DFox%26aufirst%3DG.%2BC.%26aulast%3Dvan%2BRaaij%26aufirst%3DM.%2BJ.%26aulast%3DLamb%26aufirst%3DH.%26aulast%3DHawkins%26aufirst%3DA.%2BR.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26atitle%3DMechanistic%2520basis%2520of%2520the%2520inhibition%2520of%2520type%2520II%2520dehydroquinase%2520by%2520%25282S%2529-%2520and%2520%25282R%2529-2-benzyl-3-dehydroquinic%2520acids%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D568%26epage%3D577%26doi%3D10.1021%2Fcb300493s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babaoglu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Deconstructing fragment-based inhibitor discovery</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">720</span>– <span class="NLM_lpage">723</span>, <span class="refDoi"> DOI: 10.1038/nchembio831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1038%2Fnchembio831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=17072304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=720-723&author=K.+Babaogluauthor=B.+K.+Shoichet&title=Deconstructing+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing fragment-based inhibitor discovery</span></div><div class="casAuthors">Babaoglu, Kerim; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">720-723</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based screens test multiple low-mol. wt. mols. for binding to a target.  Fragments often bind with low affinities but typically have better ligand efficiencies (ΔGbind /heavy atom count) than traditional screening hits.  This efficiency, combined with accompanying at.-resoln. structures, has made fragments popular starting points for drug discovery programs.  Fragment-based design adopts a constructive strategy: affinity is enhanced either by cycles of functional-group addn. or by joining two independent fragments together.  The final inhibitor is expected to adopt the same geometry as the original fragment hit.  Here the authors consider whether the inverse, deconstructive logic also applies-can one always parse a higher-affinity inhibitor into fragments that recapitulate the binding geometry of the larger mol..  Cocrystal structures of fragments deconstructed from a known β-lactamase inhibitor suggest that this is not always the case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVl5_M15h_rLVg90H21EOLACvtfcHk0libNiZxVC3tag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM&md5=de0d5632f6354314794c2f45cfdd2efd</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnchembio831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio831%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDeconstructing%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D720%26epage%3D723%26doi%3D10.1038%2Fnchembio831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Na, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, S.-S.</span></span> <span> </span><span class="NLM_article-title">GMP and IMP Are Competitive Inhibitors of CMY-10, an Extended-Spectrum Class C β-Lactamase</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e00098</span> <span class="refDoi"> DOI: 10.1128/aac.00098-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.00098-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=28242658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFeksLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=J.-H.+Naauthor=Y.+J.+Anauthor=S.-S.+Cha&title=GMP+and+IMP+Are+Competitive+Inhibitors+of+CMY-10%2C+an+Extended-Spectrum+Class+C+%CE%B2-Lactamase&doi=10.1128%2Faac.00098-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">GMP and IMP are competitive inhibitors of CMY-10, an extended-spectrum class C β-lactamase</span></div><div class="casAuthors">Na, Jung-Hyun; An, Young Jun; Cha, Sun-Shin</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e00098-17/1-e00098-17/5</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Nucleotides were effective in inhibiting the class C β-lactamase CMY-10.  IMP was the most potent competitive inhibitor, with a Ki value of 16.2 μM.  The crystal structure of CMY-10 complexed with GMP or IMP revealed that nucleotides fit into the R2 subsite of the active site with a unique vertical binding mode where the phosphate group at one terminus is deeply bound in the subsite and the base at the other terminus faces the solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5YIYsiPBAfbVg90H21EOLACvtfcHk0lgZxzRyC8EiMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFeksLnN&md5=8a4fdb4919517e2205f5ea96bc9ac0ec</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1128%2FAAC.00098-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00098-17%26sid%3Dliteratum%253Aachs%26aulast%3DNa%26aufirst%3DJ.-H.%26aulast%3DAn%26aufirst%3DY.%2BJ.%26aulast%3DCha%26aufirst%3DS.-S.%26atitle%3DGMP%2520and%2520IMP%2520Are%2520Competitive%2520Inhibitors%2520of%2520CMY-10%252C%2520an%2520Extended-Spectrum%2520Class%2520C%2520%25CE%25B2-Lactamase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2Faac.00098-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Na, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seol, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, S.-S.</span></span> <span> </span><span class="NLM_article-title">Structural and mechanistic insights into the inhibition of class C β-lactamases through the adenylylation of the nucleophilic serine</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">dkw491</span>, <span class="refDoi"> DOI: 10.1093/jac/dkw491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkw491" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=dkw491&author=M.-K.+Kimauthor=Y.+J.+Anauthor=J.-H.+Naauthor=J.-H.+Seolauthor=J.+Y.+Ryuauthor=J.-W.+Leeauthor=L.-W.+Kangauthor=K.+M.+Chungauthor=J.-H.+Leeauthor=J.+H.+Moonauthor=J.+S.+Leeauthor=S.-S.+Cha&title=Structural+and+mechanistic+insights+into+the+inhibition+of+class+C+%CE%B2-lactamases+through+the+adenylylation+of+the+nucleophilic+serine&doi=10.1093%2Fjac%2Fdkw491"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkw491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkw491%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.-K.%26aulast%3DAn%26aufirst%3DY.%2BJ.%26aulast%3DNa%26aufirst%3DJ.-H.%26aulast%3DSeol%26aufirst%3DJ.-H.%26aulast%3DRyu%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DJ.-W.%26aulast%3DKang%26aufirst%3DL.-W.%26aulast%3DChung%26aufirst%3DK.%2BM.%26aulast%3DLee%26aufirst%3DJ.-H.%26aulast%3DMoon%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DCha%26aufirst%3DS.-S.%26atitle%3DStructural%2520and%2520mechanistic%2520insights%2520into%2520the%2520inhibition%2520of%2520class%2520C%2520%25CE%25B2-lactamases%2520through%2520the%2520adenylylation%2520of%2520the%2520nucleophilic%2520serine%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2016%26volume%3D72%26spage%3Ddkw491%26doi%3D10.1093%2Fjac%2Fdkw491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">ModLoop: Automated modeling of loops in protein structures</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2500</span>– <span class="NLM_lpage">2501</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btg362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fbioinformatics%2Fbtg362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=14668246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVSisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2003&pages=2500-2501&author=A.+Fiserauthor=A.+Sali&title=ModLoop%3A+Automated+modeling+of+loops+in+protein+structures&doi=10.1093%2Fbioinformatics%2Fbtg362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">ModLoop: automated modeling of loops in protein structures</span></div><div class="casAuthors">Fiser, Andras; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2500-2501</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: ModLoop is a web server for automated modeling of loops in protein structures.  The input is the at. coordinates of the protein structure in the Protein Data Bank format, and the specification of the starting and ending residues of one or more segments to be modeled, contg. no more than 20 residues in total.  The output is the coordinates of the non-hydrogen atoms in the modeled segments.  A user provides the input to the server via a simple web interface, and receives the output by e-mail.  The server relies on the loop modeling routine in MODELLER that predicts the loop conformations by satisfaction of spatial restraints, without relying on a database of known protein structures.  For a rapid response, ModLoop runs on a cluster of Linux PC computers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk66um-UAJe7Vg90H21EOLACvtfcHk0lgZxzRyC8EiMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvVSisLk%253D&md5=4b0fdaca0a412715682496883ab2019c</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtg362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtg362%26sid%3Dliteratum%253Aachs%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModLoop%253A%2520Automated%2520modeling%2520of%2520loops%2520in%2520protein%2520structures%26jtitle%3DBioinformatics%26date%3D2003%26volume%3D19%26spage%3D2500%26epage%3D2501%26doi%3D10.1093%2Fbioinformatics%2Fbtg362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, R. K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Modeling of loops in protein structures</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1753</span>– <span class="NLM_lpage">1773</span>, <span class="refDoi"> DOI: 10.1110/ps.9.9.1753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1110%2Fps.9.9.1753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=11045621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntFamsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2000&pages=1753-1773&author=A.+Fiserauthor=R.+K.+G.+Doauthor=A.+Sali&title=Modeling+of+loops+in+protein+structures&doi=10.1110%2Fps.9.9.1753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling of loops in protein structures</span></div><div class="casAuthors">Fiser, Andras; Do, Richard Kinh Gian; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1753-1773</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Comparative protein structure prediction is limited mostly by the errors in alignment and loop modeling.  We describe here a new automated modeling technique that significantly improves the accuracy of loop predictions in protein structures.  The positions of all nonhydrogen atoms of the loop are optimized in a fixed environment with respect to a pseudo energy function.  The energy is a sum of many spatial restraints that include the bond length, bond angle, and improper dihedral angle terms from the CHARMM-22 force field, statistical preferences for the main-chain and side-chain dihedral angles, and statistical preferences for nonbonded at. contacts that depend on the two atom types, their distance through space, and sepn. in sequence.  The energy function is optimized with the method of conjugate gradients combined with mol. dynamics and simulated annealing.  Typically, the predicted loop conformation corresponds to the lowest energy conformation among 500 independent optimizations.  Predictions were made for 40 loops of known structure at each length from 1 to 14 residues.  The accuracy of loop predictions is evaluated as a function of thoroughness of conformational sampling, loop length, and structural properties of native loops.  When accuracy is measured by local superposition of the model on the native loop, 100, 90, and 30% of 4-, 8-, and 12-residue loop predictions, resp., had <2 Å RMSD error for the mainchain N, Cα, C, and O atoms; the av. accuracies were 0.59±0.05, 1.16±0.10, and 2.61±0.16 Å, resp.  To simulate real comparative modeling problems, the method was also evaluated by predicting loops of known structure in only approx. correct environments with errors typical of comparative modeling without misalignment.  When the RMSD distortion of the main-chain stem atoms is 2.5 Å, the av. loop prediction error increased by 180, 25, and 3% for 4-, 8-, and 12-residue loops, resp.  The accuracy of the lowest energy prediction for a given loop can be estd. from the structural variability among a no. of low energy predictions.  The relative value of the present method is gauged by (1) comparing it with one of the most successful previously described methods, and (2) describing its accuracy in recent blind predictions of protein structure.  Finally, it is shown that the av. accuracy of prediction is limited primarily by the accuracy of the energy function rather than by the extent of conformational sampling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsbG2px_r_krVg90H21EOLACvtfcHk0lgZxzRyC8EiMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntFamsL8%253D&md5=0bcdbfbddf54ecca0e3fee95ecdc3f8b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1110%2Fps.9.9.1753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.9.9.1753%26sid%3Dliteratum%253Aachs%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DR.%2BK.%2BG.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModeling%2520of%2520loops%2520in%2520protein%2520structures%26jtitle%3DProtein%2520Sci.%26date%3D2000%26volume%3D9%26spage%3D1753%26epage%3D1773%26doi%3D10.1110%2Fps.9.9.1753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ness, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paetzel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strynadka, N. C. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design Guides the Improved Efficacy of Deacylation Transition State Analogue Inhibitors of TEM-1 β-Lactamase†,#</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">5312</span>– <span class="NLM_lpage">5321</span>, <span class="refDoi"> DOI: 10.1021/bi992505b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi992505b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisVelsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=5312-5321&author=S.+Nessauthor=R.+Martinauthor=A.+M.+Kindlerauthor=M.+Paetzelauthor=M.+Goldauthor=S.+E.+Jensenauthor=J.+B.+Jonesauthor=N.+C.+J.+Strynadka&title=Structure-Based+Design+Guides+the+Improved+Efficacy+of+Deacylation+Transition+State+Analogue+Inhibitors+of+TEM-1+%CE%B2-Lactamase%E2%80%A0%2C%23&doi=10.1021%2Fbi992505b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design Guides the Improved Efficacy of Deacylation Transition State Analogue Inhibitors of TEM-1 β-Lactamase</span></div><div class="casAuthors">Ness, Steven; Martin, Richard; Kindler, Alois M.; Paetzel, Mark; Gold, Marvin; Jensen, Susan E.; Jones, J. Bryan; Strynadka, Natalie C. J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5312-5321</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Transition state analog boronic acid inhibitors mimicking the structures and interactions of good penicillin substrates for the TEM-1 β-lactamase of Escherichia coli were designed using graphic analyses based on the enzyme's 1.7 Å crystallog. structure.  The synthesis of two of these transition state analogs, (1R)-1-phenylacetamido-2-(3-carboxyphenyl)ethylboronic acid (1) and (1R)-1-acetamido-2-(3-carboxy-2-hydroxyphenyl)ethylboronic acid (2), is reported.  Kinetic measurements show that, as designed, compds. 1 and 2 are highly effective deacylation transition state analog inhibitors of TEM-1 β-lactamase, with inhibition consts. of 5.9 and 13 nM, resp.  These values identify them as among the most potent competitive inhibitors yet reported for a β-lactamase.  The best inhibitor of the current series was (1R)-1-phenylacetamido-2-(3-carboxyphenyl)ethylboronic acid (1, KI = 5.9 nM), which resembles most closely the best known substrate of TEM-1, benzylpenicillin (penicillin G).  The high-resoln. crystallog. structures of these two inhibitors covalently bound to TEM-1 are also described.  In addn. to verifying the design features, these two structures show interesting and unanticipated changes in the active site area, including strong hydrogen bond formation, water displacement, and rearrangement of side chains.  The structures provide new insights into the further design of this potent class of β-lactamase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-aQSFfZl_R7Vg90H21EOLACvtfcHk0lj9tT6SnPTHHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisVelsr8%253D&md5=441c78891c1860f80e29660283299961</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fbi992505b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi992505b%26sid%3Dliteratum%253Aachs%26aulast%3DNess%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DKindler%26aufirst%3DA.%2BM.%26aulast%3DPaetzel%26aufirst%3DM.%26aulast%3DGold%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DS.%2BE.%26aulast%3DJones%26aufirst%3DJ.%2BB.%26aulast%3DStrynadka%26aufirst%3DN.%2BC.%2BJ.%26atitle%3DStructure-Based%2520Design%2520Guides%2520the%2520Improved%2520Efficacy%2520of%2520Deacylation%2520Transition%2520State%2520Analogue%2520Inhibitors%2520of%2520TEM-1%2520%25CE%25B2-Lactamase%25E2%2580%25A0%252C%2523%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D5312%26epage%3D5321%26doi%3D10.1021%2Fbi992505b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strynadka, N. C. J.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanism of Avibactam-Mediated β-Lactamase Inhibition</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.5b00007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.5b00007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2jt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=175-184&author=D.+T.+Kingauthor=A.+M.+Kingauthor=S.+M.+Lalauthor=G.+D.+Wrightauthor=N.+C.+J.+Strynadka&title=Molecular+Mechanism+of+Avibactam-Mediated+%CE%B2-Lactamase+Inhibition&doi=10.1021%2Facsinfecdis.5b00007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanism of Avibactam-Mediated β-Lactamase Inhibition</span></div><div class="casAuthors">King, Dustin T.; King, Andrew M.; Lal, Sarah M.; Wright, Gerard D.; Strynadka, Natalie C. J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">175-184</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Emerging β-lactamase-mediated resistance is threatening the clin. utility of the single most prominent class of antibacterial agents used in medicine, the β-lactams.  The diazabicyclooctane avibactam is able to inhibit a wider range of serine β-lactamases than has been previously obsd. with β-lactamase inhibitors such as the widely prescribed clavulanic acid.  However, despite its broad-spectrum activity, variable levels of inhibition have been obsd. for mol. class D β-lactamases.  In order to better understand the mol. basis and spectrum of inhibition by avibactam, we provide structural and mechanistic anal. of the compd. in complex with important class A and D serine β-lactamases.  Herein, we reveal the 1.7- and 2.0-Å-resoln. crystal structures of avibactam covalently bound to class D β-lactamases OXA-10 and OXA-48.  Furthermore, a kinetic anal. of key active-site mutants for class A β-lactamase CTX-M-15 allows us to propose a validated mechanism for avibactam-mediated β-lactamase inhibition including a unique role for S130, which acts as a general base.  This study provides mol. insights that will aid in the design and development of avibactam-based chemotherapeutic agents effective against emerging drug-resistant microorganisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHO-uFt8V_hrVg90H21EOLACvtfcHk0lj9tT6SnPTHHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2jt7w%253D&md5=d73c84c105248bae388669633e5684df</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.5b00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.5b00007%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DD.%2BT.%26aulast%3DKing%26aufirst%3DA.%2BM.%26aulast%3DLal%26aufirst%3DS.%2BM.%26aulast%3DWright%26aufirst%3DG.%2BD.%26aulast%3DStrynadka%26aufirst%3DN.%2BC.%2BJ.%26atitle%3DMolecular%2520Mechanism%2520of%2520Avibactam-Mediated%2520%25CE%25B2-Lactamase%2520Inhibition%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D175%26epage%3D184%26doi%3D10.1021%2Facsinfecdis.5b00007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamorano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orellana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Barrio-Tofiño, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Diener, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortes-Lara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Causapé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martínez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gracia, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viñuela, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aznar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Hernández, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seral, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillo-García, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Calleja, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspiroz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Iglesia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramón, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallegos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesada, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyos, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horcajada, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrosa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">José González, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Moreno, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centelles, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirelis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Tarazona, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arenal-Andrés, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, M. d. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megías, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colmenarejo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomila, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tormo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garduño, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agulla, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seoane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzmán, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luisa Pérez del
Molino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbeito, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artiles, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azcona-Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáenz, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonio Oteo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaves, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cercenado, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alarcón, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurita, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merino, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morosini, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantón, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yagüe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiva, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canut, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oteo, J.</span></span> <span> </span><span class="NLM_article-title">Activity of Imipenem-Relebactam against a Large Collection of Pseudomonas aeruginosa Clinical Isolates and Isogenic β-Lactam-Resistant Mutants</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e02165</span> <span class="refDoi"> DOI: 10.1128/aac.02165-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.02165-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31740559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktFygsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=P.+A.+Fraile-Ribotauthor=L.+Zamoranoauthor=R.+Orellanaauthor=E.+del+Barrio-Tofi%C3%B1oauthor=I.+S%C3%A1nchez-Dienerauthor=S.+Cortes-Laraauthor=C.+L%C3%B3pez-Causap%C3%A9author=G.+Cabotauthor=G.+Bouauthor=L.+Mart%C3%ADnez-Mart%C3%ADnezauthor=A.+Oliverauthor=F.+Gal%C3%A1nauthor=I.+Graciaauthor=M.+A.+Rodr%C3%ADguezauthor=L.+Mart%C3%ADnauthor=J.+M.+S%C3%A1nchezauthor=L.+Vi%C3%B1uelaauthor=M.+V.+Garc%C3%ADaauthor=J.+A.+Lepeauthor=J.+Aznarauthor=I.+L%C3%B3pez-Hern%C3%A1ndezauthor=C.+Seralauthor=F.+J.+Castillo-Garc%C3%ADaauthor=A.+I.+L%C3%B3pez-Callejaauthor=C.+Aspirozauthor=P.+de+la+Iglesiaauthor=S.+Ram%C3%B3nauthor=E.+Rieraauthor=M.+C.+P%C3%A9rezauthor=C.+Gallegosauthor=J.+Calvoauthor=M.+D.+Quesadaauthor=F.+Marcoauthor=Y.+Hoyosauthor=J.+P.+Horcajadaauthor=N.+Larrosaauthor=J.+Jos%C3%A9+Gonz%C3%A1lezauthor=F.+Tubauauthor=S.+Capillaauthor=M.+O.+P%C3%A9rez-Morenoauthor=M.+J.+Centellesauthor=E.+Padillaauthor=A.+Riveraauthor=B.+Mirelisauthor=R.+E.+Rodr%C3%ADguez-Tarazonaauthor=N.+Arenal-Andr%C3%A9sauthor=M.+d.+P.+Ortegaauthor=G.+Meg%C3%ADasauthor=I.+Garc%C3%ADaauthor=C.+Colmenarejoauthor=J.+C.+Gonz%C3%A1lezauthor=N.+M.+Mart%C3%ADnezauthor=B.+Gomilaauthor=S.+Ginerauthor=N.+Tormoauthor=E.+Gardu%C3%B1oauthor=J.+A.+Agullaauthor=A.+Seoaneauthor=J.+Pitaauthor=I.+P.+Vidalauthor=D.+M.+Guzm%C3%A1nauthor=M.+Garc%C3%ADaauthor=M.+Luisa+P%C3%A9rez+del%0AMolinoauthor=G.+Barbeitoauthor=F.+Artilesauthor=J.+M.+Azcona-Guti%C3%A9rrezauthor=Y.+S%C3%A1enzauthor=J.+Antonio+Oteoauthor=A.+Gonz%C3%A1lezauthor=J.+Villaauthor=F.+Chavesauthor=E.+Cercenadoauthor=T.+Alarc%C3%B3nauthor=N.+D.+Zuritaauthor=I.+Merinoauthor=M.+I.+Morosiniauthor=R.+Cant%C3%B3nauthor=M.+I.+S%C3%A1nchezauthor=L.+Morenoauthor=G.+Yag%C3%BCeauthor=J.+Leivaauthor=J.+L.+Barriosauthor=A.+Canutauthor=J.+Oteo&title=Activity+of+Imipenem-Relebactam+against+a+Large+Collection+of+Pseudomonas+aeruginosa+Clinical+Isolates+and+Isogenic+%CE%B2-Lactam-Resistant+Mutants&doi=10.1128%2Faac.02165-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Activity of imipenem-relebactam against a large collection of Pseudomonas aeruginosa clinical isolates and isogenic β-lactam-resistant mutants</span></div><div class="casAuthors">Fraile-Ribot, Pablo A.; Zamorano, Laura; Orellana, Rocio; del Barrio-Tofino, Ester; Sanchez-Diener, Irina; Cortes-Lara, Sara; Lopez-Causape, Carla; Cabot, Gabriel; Bou, German; Martinez-Martinez, Luis; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e02165</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Imipenem and imipenem-relebactam MICs were detd. for 1,445 Pseudomonas aeruginosa clin. isolates and a large panel of isogenic mutants showing the most relevant mutation-driven β-lactam resistance mechanisms.  Imipenem-relebactam showed the highest susceptibility rate (97.3%), followed by colistin and ceftolozane-tazobactam (both 94.6%).  Imipenem-relebactam MICs remained ≤2 μg/mL in all 16 isogenic PAO1 mutants and in 8 pairs of extensively drug-resistant clin. strains that had developed resistance to ceftolozane-tazobactam and ceftazidime-avibactam due to mutations in OXA-10 or AmpC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUtQrvtHwForVg90H21EOLACvtfcHk0lhXpZXf92ypzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktFygsLk%253D&md5=1c5c6352b7f873cc37945147f1eae60e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1128%2Faac.02165-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.02165-19%26sid%3Dliteratum%253Aachs%26aulast%3DFraile-Ribot%26aufirst%3DP.%2BA.%26aulast%3DZamorano%26aufirst%3DL.%26aulast%3DOrellana%26aufirst%3DR.%26aulast%3Ddel%2BBarrio-Tofi%25C3%25B1o%26aufirst%3DE.%26aulast%3DS%25C3%25A1nchez-Diener%26aufirst%3DI.%26aulast%3DCortes-Lara%26aufirst%3DS.%26aulast%3DL%25C3%25B3pez-Causap%25C3%25A9%26aufirst%3DC.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADnez%26aufirst%3DL.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DGal%25C3%25A1n%26aufirst%3DF.%26aulast%3DGracia%26aufirst%3DI.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DM.%2BA.%26aulast%3DMart%25C3%25ADn%26aufirst%3DL.%26aulast%3DS%25C3%25A1nchez%26aufirst%3DJ.%2BM.%26aulast%3DVi%25C3%25B1uela%26aufirst%3DL.%26aulast%3DGarc%25C3%25ADa%26aufirst%3DM.%2BV.%26aulast%3DLepe%26aufirst%3DJ.%2BA.%26aulast%3DAznar%26aufirst%3DJ.%26aulast%3DL%25C3%25B3pez-Hern%25C3%25A1ndez%26aufirst%3DI.%26aulast%3DSeral%26aufirst%3DC.%26aulast%3DCastillo-Garc%25C3%25ADa%26aufirst%3DF.%2BJ.%26aulast%3DL%25C3%25B3pez-Calleja%26aufirst%3DA.%2BI.%26aulast%3DAspiroz%26aufirst%3DC.%26aulast%3Dde%2Bla%2BIglesia%26aufirst%3DP.%26aulast%3DRam%25C3%25B3n%26aufirst%3DS.%26aulast%3DRiera%26aufirst%3DE.%26aulast%3DP%25C3%25A9rez%26aufirst%3DM.%2BC.%26aulast%3DGallegos%26aufirst%3DC.%26aulast%3DCalvo%26aufirst%3DJ.%26aulast%3DQuesada%26aufirst%3DM.%2BD.%26aulast%3DMarco%26aufirst%3DF.%26aulast%3DHoyos%26aufirst%3DY.%26aulast%3DHorcajada%26aufirst%3DJ.%2BP.%26aulast%3DLarrosa%26aufirst%3DN.%26aulast%3DJos%25C3%25A9%2BGonz%25C3%25A1lez%26aufirst%3DJ.%26aulast%3DTubau%26aufirst%3DF.%26aulast%3DCapilla%26aufirst%3DS.%26aulast%3DP%25C3%25A9rez-Moreno%26aufirst%3DM.%2BO.%26aulast%3DCentelles%26aufirst%3DM.%2BJ.%26aulast%3DPadilla%26aufirst%3DE.%26aulast%3DRivera%26aufirst%3DA.%26aulast%3DMirelis%26aufirst%3DB.%26aulast%3DRodr%25C3%25ADguez-Tarazona%26aufirst%3DR.%2BE.%26aulast%3DArenal-Andr%25C3%25A9s%26aufirst%3DN.%26aulast%3DOrtega%26aufirst%3DM.%2Bd.%2BP.%26aulast%3DMeg%25C3%25ADas%26aufirst%3DG.%26aulast%3DGarc%25C3%25ADa%26aufirst%3DI.%26aulast%3DColmenarejo%26aufirst%3DC.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DJ.%2BC.%26aulast%3DMart%25C3%25ADnez%26aufirst%3DN.%2BM.%26aulast%3DGomila%26aufirst%3DB.%26aulast%3DGiner%26aufirst%3DS.%26aulast%3DTormo%26aufirst%3DN.%26aulast%3DGardu%25C3%25B1o%26aufirst%3DE.%26aulast%3DAgulla%26aufirst%3DJ.%2BA.%26aulast%3DSeoane%26aufirst%3DA.%26aulast%3DPita%26aufirst%3DJ.%26aulast%3DVidal%26aufirst%3DI.%2BP.%26aulast%3DGuzm%25C3%25A1n%26aufirst%3DD.%2BM.%26aulast%3DGarc%25C3%25ADa%26aufirst%3DM.%26aulast%3DLuisa%2BP%25C3%25A9rez%2Bdel%2BMolino%26aufirst%3DM.%26aulast%3DBarbeito%26aufirst%3DG.%26aulast%3DArtiles%26aufirst%3DF.%26aulast%3DAzcona-Guti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DS%25C3%25A1enz%26aufirst%3DY.%26aulast%3DAntonio%2BOteo%26aufirst%3DJ.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DA.%26aulast%3DVilla%26aufirst%3DJ.%26aulast%3DChaves%26aufirst%3DF.%26aulast%3DCercenado%26aufirst%3DE.%26aulast%3DAlarc%25C3%25B3n%26aufirst%3DT.%26aulast%3DZurita%26aufirst%3DN.%2BD.%26aulast%3DMerino%26aufirst%3DI.%26aulast%3DMorosini%26aufirst%3DM.%2BI.%26aulast%3DCant%25C3%25B3n%26aufirst%3DR.%26aulast%3DS%25C3%25A1nchez%26aufirst%3DM.%2BI.%26aulast%3DMoreno%26aufirst%3DL.%26aulast%3DYag%25C3%25BCe%26aufirst%3DG.%26aulast%3DLeiva%26aufirst%3DJ.%26aulast%3DBarrios%26aufirst%3DJ.%2BL.%26aulast%3DCanut%26aufirst%3DA.%26aulast%3DOteo%26aufirst%3DJ.%26atitle%3DActivity%2520of%2520Imipenem-Relebactam%2520against%2520a%2520Large%2520Collection%2520of%2520Pseudomonas%2520aeruginosa%2520Clinical%2520Isolates%2520and%2520Isogenic%2520%25CE%25B2-Lactam-Resistant%2520Mutants%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.02165-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyá, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Stepwise Upregulation of the Pseudomonas aeruginosa Chromosomal Cephalosporinase Conferring High-Level β-Lactam Resistance Involves Three AmpD Homologues</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1780</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1128/aac.50.5.1780-1787.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.50.5.1780-1787.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=16641450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD28XksVKktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2006&pages=1780-1787&author=C.+Juanauthor=B.+Moy%C3%A1author=J.+L.+P%C3%A9rezauthor=A.+Oliver&title=Stepwise+Upregulation+of+the+Pseudomonas+aeruginosa+Chromosomal+Cephalosporinase+Conferring+High-Level+%CE%B2-Lactam+Resistance+Involves+Three+AmpD+Homologues&doi=10.1128%2Faac.50.5.1780-1787.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Stepwise upregulation of the Pseudomonas aeruginosa chromosomal cephalosporinase conferring high-level β-lactam resistance involves three AmpD homologues</span></div><div class="casAuthors">Juan, Carlos; Moya, Bartolome; Perez, Jose L.; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1780-1787</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Development of resistance to the antipseudomonal penicillins and cephalosporins mediated by hyperprodn. of the chromosomal cephalosporinase AmpC is a major threat to the successful treatment of Pseudomonas aeruginosa infections.  Although ampD inactivation has been previously found to lead to a partially derepressed phenotype characterized by increased AmpC prodn. but retaining further inducibility, the regulation of ampC in P. aeruginosa is far from well understood.  The authors show that ampC expression is coordinately repressed by three AmpD homologues, including the previously described protein AmpD plus two addnl. proteins, designated AmpDh2 and AmpDh3.  The three AmpD homologues are responsible for a stepwise ampC upregulation mechanism ultimately leading to constitutive hyperexpression of the chromosomal cephalosporinase and high-level antipseudomonal β-lactam resistance, as shown by anal. of the three single ampD mutants, the three double ampD mutants, and the triple ampD mutant.  This is achieved by a three-step escalating mechanism rendering four relevant expression states: basal-level inducible expression (wild type), moderate-level hyperinducible expression with increased antipseudomonal β-lactam resistance (ampD mutant), high-level hyperinducible expression with high-level β-lactam resistance (ampD ampDh3 double mutant), and very high-level (more than 1000-fold compared to the wild type) derepressed expression (triple mutant).  Although one-step inducible-derepressed expression models are frequent in natural resistance mechanisms, this is the first characterized example in which expression of a resistance gene can be sequentially amplified through multiple steps of derepression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVP1sDdfHIarVg90H21EOLACvtfcHk0lhXpZXf92ypzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksVKktLs%253D&md5=9b9918303a1c8ec912788a606012b78d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1128%2FAAC.50.5.1780-1787.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.50.5.1780-1787.2006%26sid%3Dliteratum%253Aachs%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DMoy%25C3%25A1%26aufirst%3DB.%26aulast%3DP%25C3%25A9rez%26aufirst%3DJ.%2BL.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DStepwise%2520Upregulation%2520of%2520the%2520Pseudomonas%2520aeruginosa%2520Chromosomal%2520Cephalosporinase%2520Conferring%2520High-Level%2520%25CE%25B2-Lactam%2520Resistance%2520Involves%2520Three%2520AmpD%2520Homologues%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2006%26volume%3D50%26spage%3D1780%26epage%3D1787%26doi%3D10.1128%2Faac.50.5.1780-1787.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ropy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Diener, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayala, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Role of Pseudomonas aeruginosa Low-Molecular-Mass Penicillin-Binding Proteins in AmpC Expression, β-Lactam Resistance, and Peptidoglycan Structure</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3925</span>– <span class="NLM_lpage">3934</span>, <span class="refDoi"> DOI: 10.1128/aac.05150-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.05150-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25896695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFaqsrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=3925-3934&author=A.+Ropyauthor=G.+Cabotauthor=I.+S%C3%A1nchez-Dienerauthor=C.+Aguileraauthor=B.+Moyaauthor=J.+A.+Ayalaauthor=A.+Oliver&title=Role+of+Pseudomonas+aeruginosa+Low-Molecular-Mass+Penicillin-Binding+Proteins+in+AmpC+Expression%2C+%CE%B2-Lactam+Resistance%2C+and+Peptidoglycan+Structure&doi=10.1128%2Faac.05150-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Pseudomonas aeruginosa low-molecular-mass penicillin-binding proteins in AmpC expression, β-lactam resistance, and peptidoglycan structure</span></div><div class="casAuthors">Ropy, Alaa; Cabot, Gabriel; Sanchez-Diener, Irina; Aguilera, Cristian; Moya, Bartolome; Ayala, Juan A.; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3925-3934</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">This study aimed to characterize the role of Pseudomonas aeruginosa low-mol.-mass penicillin-binding proteins (LMM PBPs), namely, PBP4 (DacB), PBP5 (DacC), and PBP7 (PbpG), in peptidoglycan compn., β-lactam resistance, and ampC regulation.  For this purpose, we constructed all single and multiple mutants of dacB, dacC, pbpG, and ampC from the wild-type P. aeruginosa PAO1 strain.  Peptidoglycan compn. was detd. by high-performance liq. chromatog. (HPLC), ampC expression by reverse transcription-PCR (RT-PCR), PBP patterns by a Bocillin FL-binding test, and antimicrobial susceptibility by MIC testing for a panel of β-lactams.  Microscopy and growth rate analyses revealed no apparent major morphol. changes for any of the mutants compared to the wild-type PAO1 strain.  Of the single mutants, only dacC mutation led to significantly increased pentapeptide levels, showing that PBP5 is the major DD-carboxypeptidase in P. aeruginosa.  Moreover, our results indicate that PBP4 and PBP7 play a significant role as DD-carboxypeptidase only if PBP5 is absent, and their DD-endopeptidase activity is also inferred.  As expected, the inactivation of PBP4 led to a significant increase in ampC expression (∼50-fold), but, remarkably, the sequential inactivation of the 3 LMM PBPs produced a much greater increase (1000-fold), which correlated with peptidoglycan pentapeptide levels.  Finally, the β-lactam susceptibility profiles of the LMM PBP mutants correlated well with the ampC expression data.  However, the inactivation of ampC in these mutants also evidenced a role of LMM PBPs, esp. PBP5, in intrinsic β-lactam resistance.  In summary, in addn. to assessing the effect of P. aeruginosa LMM PBPs on peptidoglycan structure for the 1st time, we obtained results that represent a step forward in understanding the impact of these PBPs on β-lactam resistance, apparently driven by the interplay between their roles in AmpC induction, β-lactam trapping, and DD-carboxypeptidase/β-lactamase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOfdZaUU7aLVg90H21EOLACvtfcHk0lj5lQAw5-_ABA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFaqsrvE&md5=531776c4f79ac78545e7f0ff6b121f33</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1128%2FAAC.05150-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.05150-14%26sid%3Dliteratum%253Aachs%26aulast%3DRopy%26aufirst%3DA.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DS%25C3%25A1nchez-Diener%26aufirst%3DI.%26aulast%3DAguilera%26aufirst%3DC.%26aulast%3DMoya%26aufirst%3DB.%26aulast%3DAyala%26aufirst%3DJ.%2BA.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DRole%2520of%2520Pseudomonas%2520aeruginosa%2520Low-Molecular-Mass%2520Penicillin-Binding%2520Proteins%2520in%2520AmpC%2520Expression%252C%2520%25CE%25B2-Lactam%2520Resistance%252C%2520and%2520Peptidoglycan%2520Structure%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D3925%26epage%3D3934%26doi%3D10.1128%2Faac.05150-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Llarena, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Cuenca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gude, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oviaño, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachón, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnacho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascual, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span> <span> </span><span class="NLM_article-title">High incidence of MDR and XDR <i>Pseudomonas aeruginosa</i> isolates obtained from patients with ventilator-associated pneumonia in Greece, Italy and Spain as part of the magicBullet clinical trial</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1244</span>– <span class="NLM_lpage">1252</span>, <span class="refDoi"> DOI: 10.1093/jac/dkz030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkz030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=30753505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlansbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2019&pages=1244-1252&author=A.+P%C3%A9rezauthor=E.+Gatoauthor=J.+P%C3%A9rez-Llarenaauthor=F.+Fern%C3%A1ndez-Cuencaauthor=M.+J.+Gudeauthor=M.+Ovia%C3%B1oauthor=M.+E.+Pach%C3%B3nauthor=J.+Garnachoauthor=V.+Gonz%C3%A1lezauthor=%C3%81.+Pascualauthor=J.+M.+Cisnerosauthor=G.+Bou&title=High+incidence+of+MDR+and+XDR+Pseudomonas+aeruginosa+isolates+obtained+from+patients+with+ventilator-associated+pneumonia+in+Greece%2C+Italy+and+Spain+as+part+of+the+magicBullet+clinical+trial&doi=10.1093%2Fjac%2Fdkz030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">High incidence of MDR and XDR Pseudomonas aeruginosa isolates obtained from patients with ventilator-associated pneumonia in Greece, Italy and Spain as part of the MagicBullet clinical trial</span></div><div class="casAuthors">Perez, Astrid; Gato, Eva; Perez-Llarena, Jose; Fernandez-Cuenca, Felipe; Gude, Maria Jose; Oviano, Marina; Pachon, Maria Eugenia; Garnacho, Jose; Gonzalez, Veronica; Pascual, Alvaro; Cisneros, Jose Miguel; Bou, German</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1244-1252</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">1460-2091</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives: To characterize the antimicrobial susceptibility, mol. epidemiol. and carbapenem resistance mechanisms in Pseudomonas aeruginosa isolates recovered from respiratory tract samples from patients with ventilator-assocd. pneumonia enrolled in the MagicBullet clin. trial.  Methods: Isolates were collected from 53 patients from 12 hospitals in Spain, Italy and Greece.  Susceptibility was detd. using broth microdilution and Etest.  MALDI-TOF MS was used to detect carbapenemase activity and carbapenemases were identified by PCR and sequencing.  Mol. epidemiol. was investigated using PFGE and MLST.  Results: Of the 53 isolates, 2 (3.8%) were considered pandrug resistant (PDR), 19 (35.8%) were XDR and 16 (30.2%) were MDR.  Most (88.9%) of the isolates from Greece were MDR, XDR or PDR, whereas fewer of the isolates from Spain (33.3%) and Italy (43.5%) showed antibiotic resistance.  Three Greek isolates were resistant to colistin.  Overall, the rates of resistance of P. aeruginosa isolates to imipenem, ciprofloxacin, ceftolozane/tazobactam and ceftazidime/avibactam were 64.1%, 54.7%, 22.6% and 24.5%, resp.  All isolates resistant to ceftolozane/tazobactam and ceftazidime/avibactam (Greece, n = 10; and Italy, n = 2) carried blaVIM-2.  Spanish isolates were susceptible to the new drug combinations.  Forty-eight restriction patterns and 27 STs were documented.  Sixty percent of isolates belonged to six STs, including the high-risk clones ST-111, ST-175 and ST-235.  Conclusions: MDR/XDR isolates were highly prevalent, particularly in Greece.  The most effective antibiotic against P. aeruginosa was colistin, followed by ceftolozane/tazobactam and ceftazidime/avibactam. blaVIM-2 is assocd. with resistance to ceftolozane/tazobactam and ceftazidime/avibactam, and related to highly resistant phenotypes.  ST-111 was the most frequent and disseminated clone and the clonal diversity was lower in XDR and PDR strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3pwTl_mp9eLVg90H21EOLACvtfcHk0lj5lQAw5-_ABA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlansbrN&md5=1dc8fe08d731ecb7f61cf0244e7a3f71</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkz030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkz030%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez%26aufirst%3DA.%26aulast%3DGato%26aufirst%3DE.%26aulast%3DP%25C3%25A9rez-Llarena%26aufirst%3DJ.%26aulast%3DFern%25C3%25A1ndez-Cuenca%26aufirst%3DF.%26aulast%3DGude%26aufirst%3DM.%2BJ.%26aulast%3DOvia%25C3%25B1o%26aufirst%3DM.%26aulast%3DPach%25C3%25B3n%26aufirst%3DM.%2BE.%26aulast%3DGarnacho%26aufirst%3DJ.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DV.%26aulast%3DPascual%26aufirst%3D%25C3%2581.%26aulast%3DCisneros%26aufirst%3DJ.%2BM.%26aulast%3DBou%26aufirst%3DG.%26atitle%3DHigh%2520incidence%2520of%2520MDR%2520and%2520XDR%2520Pseudomonas%2520aeruginosa%2520isolates%2520obtained%2520from%2520patients%2520with%2520ventilator-associated%2520pneumonia%2520in%2520Greece%252C%2520Italy%2520and%2520Spain%2520as%2520part%2520of%2520the%2520magicBullet%2520clinical%2520trial%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2019%26volume%3D74%26spage%3D1244%26epage%3D1252%26doi%3D10.1093%2Fjac%2Fdkz030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albertí, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Characterization of the New Metallo-β-Lactamase VIM-13 and Its Integron-Borne Gene from a Pseudomonas aeruginosa Clinical Isolate in Spain</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3589</span>– <span class="NLM_lpage">3596</span>, <span class="refDoi"> DOI: 10.1128/aac.00465-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.00465-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=18644957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1SrtbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=3589-3596&author=C.+Juanauthor=A.+Beceiroauthor=O.+Guti%C3%A9rrezauthor=S.+Albert%C3%ADauthor=M.+Garauauthor=J.+L.+P%C3%A9rezauthor=G.+Bouauthor=A.+Oliver&title=Characterization+of+the+New+Metallo-%CE%B2-Lactamase+VIM-13+and+Its+Integron-Borne+Gene+from+a+Pseudomonas+aeruginosa+Clinical+Isolate+in+Spain&doi=10.1128%2Faac.00465-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the new metallo-β-lactamase VIM-13 and its integron-borne gene from a Pseudomonas aeruginosa clinical isolate in Spain</span></div><div class="casAuthors">Juan, Carlos; Beceiro, Alejandro; Gutierrez, Olivia; Alberti, Sebastian; Garau, Margalida; Perez, Jose L.; Bou, German; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3589-3596</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">During a survey conducted to evaluate the incidence of class B carbapenemase (metallo-β-lactamase [MBL])-producing Pseudomonas aeruginosa strains from hospitals in Majorca, Spain, five clin. isolates showed a pos. Etest MBL screening test result.  In one of them, strain PA-SL2, the presence of a new blaVIM deriv. (blaVIM-13) was detected by PCR amplification with blaVIM-1-specific primers followed by sequencing.  The blaVIM-13-producing isolate showed resistance to all β-lactams (except aztreonam), gentamicin, tobramycin, and ciprofloxacin.  VIM-13 exhibited 93% and 88% amino acid sequence identities with VIM-1 and VIM-2, resp.  BlaVIM-13 was cloned in parallel with blaVIM-1, and the resistance profile conferred was analyzed both in Escherichia coli and in P. aeruginosa backgrounds.  Compared to VIM-1, VIM-13 conferred slightly higher levels of resistance to piperacillin and lower levels of resistance to ceftazidime and cefepime.  VIM-13 and VIM-1 were purified in parallel as well, and their kinetic parameters were compared.  The kcat/Km ratios for the antibiotics mentioned above were in good agreement with the MIC data.  Furthermore, EDTA inhibited the activity of VIM-13 approx. 25 times less than it inhibited the activity of VIM-1.  VIM-13 was harbored in a class 1 integron, along with a new variant (Ala108Thr) of the aminoglycoside-modifying enzyme encoding gene aacA4, which confers resistance to gentamicin and tobramycin.  Finally, the VIM-13 integron was apparently located in the chromosome, since transformation and conjugation expts. consistently yielded neg. results and the blaVIM-13 probe hybridized only with the genomic DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR3E6XP7Vu2LVg90H21EOLACvtfcHk0lj5lQAw5-_ABA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1SrtbnL&md5=5c4aae46d7f3aea128c0b2edcbf14c05</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1128%2FAAC.00465-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00465-08%26sid%3Dliteratum%253Aachs%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DO.%26aulast%3DAlbert%25C3%25AD%26aufirst%3DS.%26aulast%3DGarau%26aufirst%3DM.%26aulast%3DP%25C3%25A9rez%26aufirst%3DJ.%2BL.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DCharacterization%2520of%2520the%2520New%2520Metallo-%25CE%25B2-Lactamase%2520VIM-13%2520and%2520Its%2520Integron-Borne%2520Gene%2520from%2520a%2520Pseudomonas%2520aeruginosa%2520Clinical%2520Isolate%2520in%2520Spain%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2008%26volume%3D52%26spage%3D3589%26epage%3D3596%26doi%3D10.1128%2Faac.00465-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taracila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hujer, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatta, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desarbre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">A kinetic analysis of the inhibition of FOX-4 -lactamase, a plasmid-mediated AmpC cephalosporinase, by monocyclic -lactams and carbapenems</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">682</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1093/jac/dkt434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkt434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=24235094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisF2lsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=682-690&author=K.+M.+Papp-Wallaceauthor=S.+Malloauthor=C.+R.+Bethelauthor=M.+A.+Taracilaauthor=A.+M.+Hujerauthor=A.+Fernandezauthor=J.+A.+Gattaauthor=K.+M.+Smithauthor=Y.+Xuauthor=M.+G.+P.+Pageauthor=E.+Desarbreauthor=G.+Bouauthor=R.+A.+Bonomo&title=A+kinetic+analysis+of+the+inhibition+of+FOX-4+-lactamase%2C+a+plasmid-mediated+AmpC+cephalosporinase%2C+by+monocyclic+-lactams+and+carbapenems&doi=10.1093%2Fjac%2Fdkt434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A kinetic analysis of the inhibition of FOX-4 β-lactamase, a plasmid-mediated AmpC cephalosporinase, by monocyclic β-lactams and carbapenems</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.; Mallo, Susana; Bethel, Christopher R.; Taracila, Magdalena A.; Hujer, Andrea M.; Fernandez, Ana; Gatta, Julian A.; Smith, Kerri M.; Xu, Yan; Page, Malcolm G. P.; Desarbre, Eric; Bou, German; Bonomo, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">682-690</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Class C β-lactamases are prevalent among Enterobacteriaceae; however, these enzymes are resistant to inactivation by com. available β-lactamase inhibitors.  In order to find novel scaffolds to inhibit class C β-lactamases, the comparative efficacy of monocyclic β-lactam antibiotics (aztreonam and the siderophore monosulfactam BAL 30072), the bridged monobactam β-lactamase inhibitor BAL 29880, and carbapenems (imipenem, meropenem, doripenem, and ertapenem) were tested in kinetic assays against FOX-4, a plasmid-mediated class C β-lactamase (pmAmpC).  The FOX-4 β-lactamase was purified.  Steady-state kinetics, electrospray ionization mass spectrometry (ESI-MS) and UV difference (UVD) spectroscopy were conducted using the β-lactam scaffolds described.  The Ki values for the monocyclic β-lactams against FOX-4 β-lactamase were 0.04 μM (aztreonam) and 0.66 μM (BAL 30072), and the Ki for bridged monobactam BAL 29880 was 8.9 μM.  For carbapenems, the Ki values ranged from 0.27 μM (ertapenem) to 2.3 μM (imipenem).  ESI-MS demonstrated the formation of stable covalent adducts when the monocyclic β-lactams and carbapenems were reacted with FOX-4 β-lactamase.  UVD spectroscopy suggested the appearance of different chromophoric intermediates.  Thus, monocyclic β-lactam and carbapenem antibiotics are effective mechanism-based inhibitors of FOX-4 β-lactamase, a clin. important pmAmpC, and provide stimulus for the development of new inhibitors to inactivate plasmidic and chromosomal class C β-lactamases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvesH3hBGh9bVg90H21EOLACvtfcHk0liTSKHGPDSkmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisF2lsbs%253D&md5=e05484b884e8c8beb6f99e24c8238dd1</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkt434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkt434%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DMallo%26aufirst%3DS.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DTaracila%26aufirst%3DM.%2BA.%26aulast%3DHujer%26aufirst%3DA.%2BM.%26aulast%3DFernandez%26aufirst%3DA.%26aulast%3DGatta%26aufirst%3DJ.%2BA.%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DPage%26aufirst%3DM.%2BG.%2BP.%26aulast%3DDesarbre%26aufirst%3DE.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DA%2520kinetic%2520analysis%2520of%2520the%2520inhibition%2520of%2520FOX-4%2520-lactamase%252C%2520a%2520plasmid-mediated%2520AmpC%2520cephalosporinase%252C%2520by%2520monocyclic%2520-lactams%2520and%2520carbapenems%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D682%26epage%3D690%26doi%3D10.1093%2Fjac%2Fdkt434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hujer, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domitrovic, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hujer, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurless, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuohy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">Unexpected challenges in treating multidrug-resistant Gram-negative bacteria: resistance to ceftazidime-avibactam in archived isolates of <i>Pseudomonas aeruginosa</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1020</span>– <span class="NLM_lpage">1029</span>, <span class="refDoi"> DOI: 10.1128/aac.04238-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.04238-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25451057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1KrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=1020-1029&author=M.+L.+Winklerauthor=K.+M.+Papp-Wallaceauthor=A.+M.+Hujerauthor=T.+N.+Domitrovicauthor=K.+M.+Hujerauthor=K.+N.+Hurlessauthor=M.+Tuohyauthor=G.+Hallauthor=R.+A.+Bonomo&title=Unexpected+challenges+in+treating+multidrug-resistant+Gram-negative+bacteria%3A+resistance+to+ceftazidime-avibactam+in+archived+isolates+of+Pseudomonas+aeruginosa&doi=10.1128%2Faac.04238-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Unexpected challenges in treating multidrug-resistant gram-negative bacteria: resistance to ceftazidime-avibactam in archived isolates of Pseudomonas aeruginosa</span></div><div class="casAuthors">Winkler, Marisa L.; Papp-Wallace, Krisztina M.; Hujer, Andrea M.; Domitrovic, T. Nicholas; Hujer, Kristine M.; Hurless, Kelly N.; Tuohy, Marion; Hall, Geraldine; Bonomo, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1020-1029, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is a notoriously difficult-to-treat pathogen that is a common cause of severe nosocomial infections.  Studying a collection of β-lactam-resistant P. aeruginosa clin. isolates from a decade ago, the authors uncovered resistance to ceftazidime-avibactam, a novel β-lactam/β-lactamase inhibitor combination.  The isolates were systematically analyzed through a variety of genetic, biochem., genomic, and microbiol. methods to understand how resistance manifests to a unique drug combination that is not yet clin. released.  Avibactam was able to inactivate different AmpC β-lactamase enzymes and blaPDC regulatory elements and penicillin-binding protein differences did not contribute in a major way to resistance.  By using carefully selected combinations of antimicrobial agents, the greatest barrier to ceftazidime-avibactam was shown to be membrane permeability and drug efflux.  To overcome the constellation of resistance determinants, it was shown that a combination of antimicrobial agents (ceftazidime/avibactam/fosfomycin) targeting multiple cell wall synthetic pathways can restore susceptibility.  In P. aeruginosa, efflux, as a general mechanism of resistance, may pose the greatest challenge to future antibiotic development.  The authors' unexpected findings create concern that even the development of antimicrobial agents targeted for the treatment of multidrug-resistant bacteria may encounter clin. important resistance.  Antibiotic therapy in the future must consider these factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT4V9qO1xrZrVg90H21EOLACvtfcHk0liTSKHGPDSkmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1KrsLY%253D&md5=724fd9db8243065976feb01f6aff8978</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1128%2FAAC.04238-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.04238-14%26sid%3Dliteratum%253Aachs%26aulast%3DWinkler%26aufirst%3DM.%2BL.%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DHujer%26aufirst%3DA.%2BM.%26aulast%3DDomitrovic%26aufirst%3DT.%2BN.%26aulast%3DHujer%26aufirst%3DK.%2BM.%26aulast%3DHurless%26aufirst%3DK.%2BN.%26aulast%3DTuohy%26aufirst%3DM.%26aulast%3DHall%26aufirst%3DG.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DUnexpected%2520challenges%2520in%2520treating%2520multidrug-resistant%2520Gram-negative%2520bacteria%253A%2520resistance%2520to%2520ceftazidime-avibactam%2520in%2520archived%2520isolates%2520of%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D1020%26epage%3D1029%26doi%3D10.1128%2Faac.04238-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taracila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">Variants of β-Lactamase KPC-2 That Are Resistant to Inhibition by Avibactam</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3710</span>– <span class="NLM_lpage">3717</span>, <span class="refDoi"> DOI: 10.1128/aac.04406-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.04406-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25666153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFaru7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=3710-3717&author=K.+M.+Papp-Wallaceauthor=M.+L.+Winklerauthor=M.+A.+Taracilaauthor=R.+A.+Bonomo&title=Variants+of+%CE%B2-Lactamase+KPC-2+That+Are+Resistant+to+Inhibition+by+Avibactam&doi=10.1128%2Faac.04406-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Variants of β-lactamase KPC-2 that are resistant to inhibition by avibactam</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.; Winkler, Marisa L.; Taracila, Magdalena A.; Bonomo, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3710-3717</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">KPC-2 is the most prevalent class A carbapenemase in the world.  Previously, KPC-2 was shown to hydrolyze the β-lactamase inhibitors clavulanic acid, sulbactam, and tazobactam.  In addn., substitutions at amino acid position R220 in the KPC-2 β-lactamase increased resistance to clavulanic acid.  A novel bridged diazabicyclooctane (DBO) non-β-lactam β-lactamase inhibitor, avibactam, was shown to inactivate the KPC-2 β-lactamase.  To better understand the mechanistic basis for inhibition of KPC-2 by avibactam, we tested the potency of ampicillin-avibactam and ceftazidime-avibactam against engineered variants of the KPC-2 β-lactamase that possessed single amino acid substitutions at important sites (i.e., Ambler positions 69, 130, 234, 220, and 276) that were previously shown to confer inhibitor resistance in TEM and SHV β-lactamases.  To this end, we performed susceptibility testing, biochem. assays, and mol. modeling.  Escherichia coli DH10B carrying KPC-2 β-lactamase variants with the substitutions S130G, K234R, and R220M demonstrated elevated MICs for only the ampicillin-avibactam combinations (e.g., 512, 64, and 32 mg/L, resp., vs. the MICs for wild-type KPC-2 at 2 to 8 mg/L).  Steady-state kinetics revealed that the S130G variant of KPC-2 resisted inactivation by avibactam; the k2/K ratio was significantly lowered 4 logs for the S130G variant from the ratio for the wild-type enzyme (21,580 M-1 s-1 to 1.2 M-1 s-1).  Mol. modeling and mol. dynamics simulations suggested that the mobility of K73 and its ability to activate S70 (i.e., function as a general base) may be impaired in the S130G variant of KPC-2, thereby explaining the slowed acylation.  Moreover, we also advance the idea that the protonation of the sulfate nitrogen of avibactam may be slowed in the S130G variant, as S130 is the likely proton donor and another residue, possibly K234, must compensate.  Our findings show that residues S130 as well as K234 and R220 contribute significantly to the mechanism of avibactam inactivation of KPC-2.  Fortunately, the emergence of S130G, K234R, and R220M variants of KPC in the clinic should not result in failure of ceftazidime-avibactam, as the ceftazidime partner is potent against E. coli DH10B strains possessing all of these variants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUr5WTh7ibNrVg90H21EOLACvtfcHk0liTSKHGPDSkmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFaru7zP&md5=16ac1f44011c90f8356740bee6e5bd9c</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1128%2FAAC.04406-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.04406-14%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DWinkler%26aufirst%3DM.%2BL.%26aulast%3DTaracila%26aufirst%3DM.%2BA.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DVariants%2520of%2520%25CE%25B2-Lactamase%2520KPC-2%2520That%2520Are%2520Resistant%2520to%2520Inhibition%2520by%2520Avibactam%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D3710%26epage%3D3717%26doi%3D10.1128%2Faac.04406-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsivkovski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span> <span> </span><span class="NLM_article-title">Biochemical activity of vaborbactam</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e01935</span> <span class="refDoi"> DOI: 10.1128/aac.01935-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.01935-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31712199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktFyhtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=R.+Tsivkovskiauthor=O.+Lomovskaya&title=Biochemical+activity+of+vaborbactam&doi=10.1128%2Faac.01935-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical activity of vaborbactam</span></div><div class="casAuthors">Tsivkovski, Ruslan; Lomovskaya, Olga</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e01935</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The most common mechanism of resistance to β-lactams antibiotics in Gram-neg. bacteria is prodn. of β-lactamase enzymes capable of cleaving the β-lactam ring.  Inhibition of β-lactamase activity with small-mol. drugs is a proven strategy to restore the potency of many β-lactam antibiotics.  Vaborbactam (formerly RPX7009) is a cyclic boronic acid β-lactamase inhibitor (BLI) with a broad spectrum of activity against various serine β-lactamases, including KPC carbapenemases.  The combination of vaborbactam and meropenem is approved in the United States and Europe for the treatment of various nosocomial infections.  We attempted to gain more insight into the mechanism of action of vaborbactam by conducting detailed kinetic characterization of its interaction with various recombinant His-tagged β-lactamases.  Vaborbactam demonstrated potent inhibition of class A and class C enzymes with Ki values ranging from 0.022 to 0.18 μM, while inhibition of class D enzymes was rather poor, and no activity against class B β-lactamases was detected.  Importantly, vaborbactam inhibited KPC-2, KPC-3, BKC-1, and SME-2 carbapenemases at 1:1 stoichiometry, while these nos. were higher for other class A and C enzymes.  Vaborbactam was also shown to be a potent progressive inactivator of several enzymes, including KPCs with inactivation consts. k2/K in the range of 3.4 × 103 to 2.4 × 104 M-1 s-1.  Finally, expts. on the recovery of enzyme activity demonstrated the high stability of the vaborbactam-KPC complex, with 0.000040 s-1koff values and a corresponding residence time of 7 h, whereas the release of vaborbactam bound to other serine β-lactamases was substantially faster.  The biochem. characteristics of vaborbactam described in this study may facilitate further chem. optimization efforts to develop boronic BLIs with improved affinity and broader spectrum of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqct6STwjYA67Vg90H21EOLACvtfcHk0lj5wR-FgAhMQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktFyhtbg%253D&md5=53e588b395cfb868f0a5f8699427c05a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1128%2Faac.01935-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01935-19%26sid%3Dliteratum%253Aachs%26aulast%3DTsivkovski%26aufirst%3DR.%26aulast%3DLomovskaya%26aufirst%3DO.%26atitle%3DBiochemical%2520activity%2520of%2520vaborbactam%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.01935-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Vázquez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sola-Campoy, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurita, Á. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ávila, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-bertomeu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solís, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-urrutia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gónzalez-barberá, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cercenado, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bautista, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aracil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oteo-Iglesias, J.</span></span> <span> </span><span class="NLM_article-title">Carbapenemase-producing <i>Pseudomonas aeruginosa</i> in Spain: interregional dissemination of the high-risk clones ST175 and ST244 carrying blaVIM-2, blaVIM-1, blaIMP-8, blaVIM-20 and blaKPC-2</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">106026</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2020.106026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.ijantimicag.2020.106026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=32450200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFOisLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2020&pages=106026&author=M.+P%C3%A9rez-V%C3%A1zquezauthor=P.+J.+Sola-Campoyauthor=%C3%81.+M.+Zuritaauthor=A.+%C3%81vilaauthor=F.+G%C3%B3mez-bertomeuauthor=S.+Sol%C3%ADsauthor=L.+L%C3%B3pez-urrutiaauthor=E.+M.+G%C3%B3nzalez-barber%C3%A1author=E.+Cercenadoauthor=V.+Bautistaauthor=N.+Laraauthor=B.+Aracilauthor=A.+Oliverauthor=J.+Camposauthor=J.+Oteo-Iglesias&title=Carbapenemase-producing+Pseudomonas+aeruginosa+in+Spain%3A+interregional+dissemination+of+the+high-risk+clones+ST175+and+ST244+carrying+blaVIM-2%2C+blaVIM-1%2C+blaIMP-8%2C+blaVIM-20+and+blaKPC-2&doi=10.1016%2Fj.ijantimicag.2020.106026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Carbapenemase-producing Pseudomonas aeruginosa in Spain: interregional dissemination of the high-risk clones ST175 and ST244 carrying blaVIM-2, blaVIM-1, blaIMP-8, blaVIM-20 and blaKPC-2</span></div><div class="casAuthors">Perez-Vazquez, Maria; Sola-Campoy, Pedro J.; Zurita, Angela Maria; Avila, Alicia; Gomez-Bertomeu, Frederic; Solis, Sonia; Lopez-Urrutia, Luis; Gonzalez-Barbera, Eva M.; Cercenado, Emilia; Bautista, Veronica; Lara, Noelia; Aracil, Belen; Oliver, Antonio; Campos, Jose; Oteo-Iglesias, Jesus</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">106026</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Carbapenemase-producing (CP) Pseudomonas aeruginosa is rare compared with mutation-driven carbapenem-resistance, but this situation may be changing.  A collection of CP P. aeruginosa isolates was characterized in this study.  In 2016, 232 unduplicated carbapenem-resistant P. aeruginosa isolates, of which 71 (30.6%) carried carbapenemase genes, were submitted to the Spanish antibiotic ref. lab. and were further analyzed by whole-genome sequencing (WGS).  Of the 71 CP P. aeruginosa, 39 (54.9%) carried blaVIM-2, 14 (19.7%) blaVIM-1, 8 (11.3%) blaIMP-8, 6 (8.5%) blaVIM-20, 2 (2.8%) blaVIM-2 plus blaKPC-2, one (1.4%) blaIMP-13 and one (1.4%) blaVIM-1 plus blaIMP-18.  Four sequence types (ST175, ST244, ST815 and ST155) encompassed 83.1% of the 71 CP P. aeruginosa; ST175 was detected in hospitals from seven provinces.  Using core genome multilocus sequence typing (cgMLST), four clusters were detected: Cluster 1 included nine ST815/VIM-2 isolates; Cluster 2 included five ST175/VIM-2 isolates; Cluster 3 included seven ST244 isolates (five VIM-2 and two VIM-2 plus KPC-2); and Cluster 4 included 11 ST175 isolates (seven VIM-2 and four IMP-8).  The av. no. of acquired resistance genes was significantly higher in the blaVIM-1-carying isolates (7.1 ± 0.94) than in the blaVIM-2-carrying isolates (4.5 ± 0.20).CP P. aeruginosa isolates are spreading in Spain, mainly due to the dissemination of high-risk clones such as ST175 and ST244 producing VIM and IMP carbapenemases.  Emergence of CP P. aeruginosa is a cause of clin. and epidemiol. concern.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-N4m07Ndq4rVg90H21EOLACvtfcHk0lj5wR-FgAhMQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFOisLrP&md5=35cce4a9e87a0970e383d175932469da</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2020.106026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2020.106026%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-V%25C3%25A1zquez%26aufirst%3DM.%26aulast%3DSola-Campoy%26aufirst%3DP.%2BJ.%26aulast%3DZurita%26aufirst%3D%25C3%2581.%2BM.%26aulast%3D%25C3%2581vila%26aufirst%3DA.%26aulast%3DG%25C3%25B3mez-bertomeu%26aufirst%3DF.%26aulast%3DSol%25C3%25ADs%26aufirst%3DS.%26aulast%3DL%25C3%25B3pez-urrutia%26aufirst%3DL.%26aulast%3DG%25C3%25B3nzalez-barber%25C3%25A1%26aufirst%3DE.%2BM.%26aulast%3DCercenado%26aufirst%3DE.%26aulast%3DBautista%26aufirst%3DV.%26aulast%3DLara%26aufirst%3DN.%26aulast%3DAracil%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DCampos%26aufirst%3DJ.%26aulast%3DOteo-Iglesias%26aufirst%3DJ.%26atitle%3DCarbapenemase-producing%2520Pseudomonas%2520aeruginosa%2520in%2520Spain%253A%2520interregional%2520dissemination%2520of%2520the%2520high-risk%2520clones%2520ST175%2520and%2520ST244%2520carrying%2520blaVIM-2%252C%2520blaVIM-1%252C%2520blaIMP-8%252C%2520blaVIM-20%2520and%2520blaKPC-2%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2020%26volume%3D56%26spage%3D106026%26doi%3D10.1016%2Fj.ijantimicag.2020.106026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">European
Committee on Antimicrobial
Susceptibility Testing</span>. <span> </span><span class="NLM_article-title">Breakpoint
Tables for Interpretation of MICs and Zone Diameters. Version 9.0</span>. <span class="NLM_year">2019</span>, available: <a href="http://www.eucast.org/clinical_breakpoints/" class="extLink">http://www.eucast.org/clinical_breakpoints/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=European%0ACommittee+on+Antimicrobial%0ASusceptibility+Testing&title=Breakpoint%0ATables+for+Interpretation+of+MICs+and+Zone+Diameters.+Version+9.0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DBreakpoint%250ATables%2520for%2520Interpretation%2520of%2520MICs%2520and%2520Zone%2520Diameters.%2520Version%25209.0%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Montgomery,  Jr., J. A.</span>; <span class="NLM_string-name">Peralta, J. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M.</span>; <span class="NLM_string-name">Heyd, J. J.</span>; <span class="NLM_string-name">Brothers, E.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Knox, J. E.</span>; <span class="NLM_string-name">Cross, J. B.</span>; <span class="NLM_string-name">Bakken, V.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Jaramillo, J.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Stratmann, R. E.</span>; <span class="NLM_string-name">Yazyev, O.</span>; <span class="NLM_string-name">Austin, A. J.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Pomelli, C.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Voth, G. A.</span>; <span class="NLM_string-name">Salvador, P.</span>; <span class="NLM_string-name">Dannenberg, J. J.</span>; <span class="NLM_string-name">Dapprich, S.</span>; <span class="NLM_string-name">Daniels, A. D.</span>; <span class="NLM_string-name">Farkas, Ö.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Cioslowski, J.</span>; <span class="NLM_string-name">Fox, D. J.</span></span> <i>Gaussian 09</i>, Revision D.01; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Caricato&author=X.+Li&author=H.+P.+Hratchian&author=A.+F.+Izmaylov&author=J.+Bloino&author=G.+Zheng&author=J.+L.+Sonnenberg&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+Bearpark&author=J.+J.+Heyd&author=E.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=N.+Rega&author=J.+M.+Millam&author=M.+Klene&author=J.+E.+Knox&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=V.+G.+Zakrzewski&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=S.+Dapprich&author=A.+D.+Daniels&author=%C3%96.+Farkas&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=J.+Cioslowski&author=D.+J.+Fox&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><a href="http://upjv.q4md-forcefieldtools.org/RED/" class="extLink">http://upjv.q4md-forcefieldtools.org/RED/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fupjv.q4md-forcefieldtools.org%2FRED%2F+%28accessed+Feb+13%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dupradeau, F.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pigache, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaffran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savineau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grivel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosanski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span> <span> </span><span class="NLM_article-title">The R.E.D. tools: advances in RESP and ESP charge derivation and force field library building</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7821</span>– <span class="NLM_lpage">7839</span>, <span class="refDoi"> DOI: 10.1039/c0cp00111b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1039%2Fc0cp00111b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=20574571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=7821-7839&author=F.-Y.+Dupradeauauthor=A.+Pigacheauthor=T.+Zaffranauthor=C.+Savineauauthor=R.+Lelongauthor=N.+Grivelauthor=D.+Lelongauthor=W.+Rosanskiauthor=P.+Cieplak&title=The+R.E.D.+tools%3A+advances+in+RESP+and+ESP+charge+derivation+and+force+field+library+building&doi=10.1039%2Fc0cp00111b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The R.E.D. tools: advances in RESP and ESP charge derivation and force field library building</span></div><div class="casAuthors">Dupradeau, Francois-Yves; Pigache, Adrien; Zaffran, Thomas; Savineau, Corentin; Lelong, Rodolphe; Grivel, Nicolas; Lelong, Dimitri; Rosanski, Wilfried; Cieplak, Piotr</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">7821-7839</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Deriving at. charges and building a force field library for a new mol. are key steps when developing a force field required for conducting structural and energy-based anal. using mol. mechanics.  Derivation of popular RESP charges for a set of residues is a complex and error prone procedure because it depends on numerous input parameters.  To overcome these problems, the R.E.D.  Tools (RESP and ESP charge Derive, http://q4md-forcefieldtools.org/RED/) have been developed to perform charge derivation in an automatic and straightforward way.  The R.E.D. program handles chem. elements up to bromine in the periodic table.  It interfaces different quantum mech. programs employed for geometry optimization and computing mol. electrostatic potential(s), and performs charge fitting using the RESP program.  By defining tight optimization criteria and by controlling the mol. orientation of each optimized geometry, charge values are reproduced at any computer platform with an accuracy of 0.0001 e.  The charges can be fitted using multiple conformations, making them suitable for mol. dynamics simulations.  R.E.D. allows also for defining charge constraints during multiple mol. charge fitting, which are used to derive charges for mol. fragments.  Finally, R.E.D. incorporates charges into a force field library, readily usable in mol. dynamics computer packages.  For complex cases, such as a set of homologous mols. belonging to a common family, an entire force field topol. database is generated.  Currently, the at. charges and force field libraries have been developed for more than fifty model systems and stored in the RESP ESP charge DDataBase.  Selected results related to non-polarizable charge models are presented and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxiXPn3YLNM7Vg90H21EOLACvtfcHk0lgxcFjaLiqMaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrk%253D&md5=845cc10ea7ed2f0222361db82bd80e39</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2Fc0cp00111b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cp00111b%26sid%3Dliteratum%253Aachs%26aulast%3DDupradeau%26aufirst%3DF.-Y.%26aulast%3DPigache%26aufirst%3DA.%26aulast%3DZaffran%26aufirst%3DT.%26aulast%3DSavineau%26aufirst%3DC.%26aulast%3DLelong%26aufirst%3DR.%26aulast%3DGrivel%26aufirst%3DN.%26aulast%3DLelong%26aufirst%3DD.%26aulast%3DRosanski%26aufirst%3DW.%26aulast%3DCieplak%26aufirst%3DP.%26atitle%3DThe%2520R.E.D.%2520tools%253A%2520advances%2520in%2520RESP%2520and%2520ESP%2520charge%2520derivation%2520and%2520force%2520field%2520library%2520building%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D7821%26epage%3D7839%26doi%3D10.1039%2Fc0cp00111b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanquelef, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquant, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimerak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delepine, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupradeau, F.-Y.</span></span> <span> </span><span class="NLM_article-title">R.E.D. Server: a web service for deriving RESP and ESP charges and building force field libraries for new molecules and molecular fragments</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">W511</span>– <span class="NLM_lpage">W517</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fnar%2Fgkr288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=21609950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVOmsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=W511-W517&author=E.+Vanquelefauthor=S.+Simonauthor=G.+Marquantauthor=E.+Garciaauthor=G.+Klimerakauthor=J.+C.+Delepineauthor=P.+Cieplakauthor=F.-Y.+Dupradeau&title=R.E.D.+Server%3A+a+web+service+for+deriving+RESP+and+ESP+charges+and+building+force+field+libraries+for+new+molecules+and+molecular+fragments&doi=10.1093%2Fnar%2Fgkr288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">R.E.D. Server: a web service for deriving RESP and ESP charges and building force field libraries for new molecules and molecular fragments</span></div><div class="casAuthors">Vanquelef, Enguerran; Simon, Sabrina; Marquant, Gaelle; Garcia, Elodie; Klimerak, Geoffroy; Delepine, Jean Charles; Cieplak, Piotr; Dupradeau, Francois-Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W511-W517</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">R.E.D. Server is a unique, open web service, designed to derive non-polarizable RESP and ESP charges and to build force field libraries for new mols./mol. fragments.  It provides to computational biologists the means to derive rigorously mol. electrostatic potential-based charges embedded in force field libraries that are ready to be used in force field development, charge validation and mol. dynamics simulations.  R.E.D. Server interfaces quantum mechanics programs, the RESP program and the latest version of the R.E.D. tools.  A two step approach has been developed.  The first one consists of prepg. P2N file(s) to rigorously define key elements such as atom names, topol. and chem. equivalencing needed when building a force field library.  Then, P2N files are used to derive RESP or ESP charges embedded in force field libraries in the Tripos mol2 format.  In complex cases an entire set of force field libraries or force field topol. database is generated.  Other features developed in R.E.D.  Server include help services, a demonstration, tutorials, frequently asked questions, Jmol-based tools useful to construct PDB input files and parse R.E.D.  Server outputs as well as a graphical queuing system allowing any user to check the status of R.E.D. Server jobs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEOBBuQu4rcbVg90H21EOLACvtfcHk0lgId_v7uMd74Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosVOmsbY%253D&md5=1d224a2941d8634b6ed2159530d9e529</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr288%26sid%3Dliteratum%253Aachs%26aulast%3DVanquelef%26aufirst%3DE.%26aulast%3DSimon%26aufirst%3DS.%26aulast%3DMarquant%26aufirst%3DG.%26aulast%3DGarcia%26aufirst%3DE.%26aulast%3DKlimerak%26aufirst%3DG.%26aulast%3DDelepine%26aufirst%3DJ.%2BC.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DDupradeau%26aufirst%3DF.-Y.%26atitle%3DR.E.D.%2520Server%253A%2520a%2520web%2520service%2520for%2520deriving%2520RESP%2520and%2520ESP%2520charges%2520and%2520building%2520force%2520field%2520libraries%2520for%2520new%2520molecules%2520and%2520molecular%2520fragments%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DW511%26epage%3DW517%26doi%3D10.1093%2Fnar%2Fgkr288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornell, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellmeyer, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">A second generation force field for the simulation of proteins, nucleic acids, and organic molecules</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">5179</span>– <span class="NLM_lpage">5197</span>, <span class="refDoi"> DOI: 10.1021/ja00124a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00124a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaK2MXlsFertrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1995&pages=5179-5197&author=W.+D.+Cornellauthor=P.+Cieplakauthor=C.+I.+Baylyauthor=I.+R.+Gouldauthor=K.+M.+Merzauthor=D.+M.+Fergusonauthor=D.+C.+Spellmeyerauthor=T.+Foxauthor=J.+W.+Caldwellauthor=P.+A.+Kollman&title=A+second+generation+force+field+for+the+simulation+of+proteins%2C+nucleic+acids%2C+and+organic+molecules&doi=10.1021%2Fja00124a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">A Second Generation Force Field for the Simulation of Proteins, Nucleic Acids, and Organic Molecules</span></div><div class="casAuthors">Cornell, Wendy D.; Cieplak, Piotr; Bayly, Christopher I.; Gould, Ian R.; Merz, Kenneth M., Jr.; Ferguson, David M.; Spellmeyer, David C.; Fox, Thomas; Caldwell, James W.; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5179-97</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors present the derivation of a new mol. mech. force field for simulating the structures, conformational energies, and interaction energies of proteins, nucleic acids, and many related org. mols. in condensed phases.  This effective two-body force field is the successor to the Weiner et al. force field and was developed with some of the same philosophies, such as the use of a simple diagonal potential function and electrostatic potential fit atom centered charges.  The need for a 10-12 function for representing hydrogen bonds is no longer necessary due to the improved performance of the new charge model and new van der Waals parameters.  These new charges are detd. using a 6-31G* basis set and restrained electrostatic potential (RESP) fitting and have been shown to reproduce interaction energies, free energies of solvation, and conformational energies of simple small mols. to a good degree of accuracy.  Furthermore, the new RESP charges exhibit less variability as a function of the mol. conformation used in the charge detn.  The new van der Waals parameters have been derived from liq. simulations and include hydrogen parameters which take into account the effects of any geminal electroneg. atoms.  The bonded parameters developed by Weiner et al. were modified as necessary to reproduce exptl. vibrational frequencies and structures.  Most of the simple dihedral parameters have been retained from Weiner et al., but a complex set of .vphi. and ψ parameters which do a good job of reproducing the energies of the low-energy conformations of glycyl and alanyl dipeptides was developed for the peptide backbone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_vSNiuEezGrVg90H21EOLACvtfcHk0lgId_v7uMd74Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlsFertrc%253D&md5=2c8901ed614d543db51d3a28f5b6053c</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fja00124a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00124a002%26sid%3Dliteratum%253Aachs%26aulast%3DCornell%26aufirst%3DW.%2BD.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DGould%26aufirst%3DI.%2BR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DFerguson%26aufirst%3DD.%2BM.%26aulast%3DSpellmeyer%26aufirst%3DD.%2BC.%26aulast%3DFox%26aufirst%3DT.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DA%2520second%2520generation%2520force%2520field%2520for%2520the%2520simulation%2520of%2520proteins%252C%2520nucleic%2520acids%252C%2520and%2520organic%2520molecules%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26spage%3D5179%26epage%3D5197%26doi%3D10.1021%2Fja00124a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lgId_v7uMd74Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0li-sWuE1G6nJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onufriev, A.</span></span> <span> </span><span class="NLM_article-title">H++: a server for estimating pKas and adding missing hydrogens to macromolecules</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">W368</span>– <span class="NLM_lpage">W371</span>, <span class="refDoi"> DOI: 10.1093/nar/gki464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fnar%2Fgki464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=15980491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslyqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=W368-W371&author=J.+C.+Gordonauthor=J.+B.+Myersauthor=T.+Foltaauthor=V.+Shojaauthor=L.+S.+Heathauthor=A.+Onufriev&title=H%2B%2B%3A+a+server+for+estimating+pKas+and+adding+missing+hydrogens+to+macromolecules&doi=10.1093%2Fnar%2Fgki464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">H++: a server for estimating pKas and adding missing hydrogens to macromolecules</span></div><div class="casAuthors">Gordon, John C.; Myers, Jonathan B.; Folta, Timothy; Shoja, Valia; Heath, Lenwood S.; Onufriev, Alexey</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W368-W371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The structure and function of macromols. depend critically on the ionization (protonation) states of their acidic and basic groups.  A no. of existing practical methods predict protonation equil. pK consts. of macromols. based upon their at. resoln. Protein Data Bank (PDB) structures; the calcns. are often performed within the framework of the continuum electrostatics model.  Unfortunately, these methodologies are complex, involve multiple steps and require considerable investment of effort.  Our web server http://biophysics.cs.vt.edu/H++ provides access to a tool that automates this process, allowing both experts and novices to quickly obtain ests. of pKs as well as other related characteristics of biomols. such as isoelec. points, titrn. curves and energies of protonation microstates.  Protons are added to the input structure according to the calcd. ionization states of its titratable groups at the user-specified pH; the output is in the PQR (PDB + charges + radii) format.  In addn., corresponding coordinate and topol. files are generated in the format supported by the mol. modeling package AMBER.  The server is intended for a broad community of biochemists, mol. modelers, structural biologists and drug designers; it can also be used as an educational tool in biochem. courses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaTJ1ZbNsl1LVg90H21EOLACvtfcHk0li-sWuE1G6nJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslyqt7k%253D&md5=a9751f67f129a45a87e12187627d3a2b</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki464%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DJ.%2BC.%26aulast%3DMyers%26aufirst%3DJ.%2BB.%26aulast%3DFolta%26aufirst%3DT.%26aulast%3DShoja%26aufirst%3DV.%26aulast%3DHeath%26aufirst%3DL.%2BS.%26aulast%3DOnufriev%26aufirst%3DA.%26atitle%3DH%252B%252B%253A%2520a%2520server%2520for%2520estimating%2520pKas%2520and%2520adding%2520missing%2520hydrogens%2520to%2520macromolecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3DW368%26epage%3DW371%26doi%3D10.1093%2Fnar%2Fgki464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><a href="http://biophysics.cs.vt.edu/" class="extLink">http://biophysics.cs.vt.edu/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fbiophysics.cs.vt.edu%2F+%28accessed+Feb+13%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maier, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasavajhala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">ff14SB: Improving the accuracy of protein side chain and backbone parameters from ff99SB</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3696</span>– <span class="NLM_lpage">3713</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+Improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+ff99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0li-sWuE1G6nJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520Improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520ff99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Götz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 1. Generalized Born</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1542</span>– <span class="NLM_lpage">1555</span>, <span class="refDoi"> DOI: 10.1021/ct200909j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200909j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=1542-1555&author=A.+W.+G%C3%B6tzauthor=M.+J.+Williamsonauthor=D.+Xuauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+1.+Generalized+Born&doi=10.1021%2Fct200909j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born</span></div><div class="casAuthors">Gotz, Andreas W.; Williamson, Mark J.; Xu, Dong; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1555</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of generalized Born implicit solvent all-atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA enabled NVIDIA graphics processing units (GPUs).  We discuss the algorithms that are used to exploit the processing power of the GPUs and show the performance that can be achieved in comparison to simulations on conventional CPU clusters.  The implementation supports three different precision models in which the contributions to the forces are calcd. in single precision floating point arithmetic but accumulated in double precision (SPDP), or everything is computed in single precision (SPSP) or double precision (DPDP).  In addn. to performance, we have focused on understanding the implications of the different precision models on the outcome of implicit solvent MD simulations.  We show results for a range of tests including the accuracy of single point force evaluations and energy conservation as well as structural properties pertaining to protein dynamics.  The numerical noise due to rounding errors within the SPSP precision model is sufficiently large to lead to an accumulation of errors which can result in unphys. trajectories for long time scale simulations.  We recommend the use of the mixed-precision SPDP model since the numerical results obtained are comparable with those of the full double precision DPDP model and the ref. double precision CPU implementation but at significantly reduced computational cost.  Our implementation provides performance for GB simulations on a single desktop that is on par with, and in some cases exceeds, that of traditional supercomputers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshIijeXLcqLVg90H21EOLACvtfcHk0lhMFvOcpTy-7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D&md5=1e6f570db9cd504bb13706e7c56bc356</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fct200909j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200909j%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWilliamson%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25201.%2520Generalized%2520Born%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D1542%26epage%3D1555%26doi%3D10.1021%2Fct200909j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salomon-Ferrer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 2. Explicit solvent particle mesh ewald</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3878</span>– <span class="NLM_lpage">3888</span>, <span class="refDoi"> DOI: 10.1021/ct400314y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400314y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3878-3888&author=R.+Salomon-Ferrerauthor=A.+W.+G%C3%B6tzauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+2.+Explicit+solvent+particle+mesh+ewald&doi=10.1021%2Fct400314y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Gotz, Andreas W.; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3878-3888</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of explicit solvent all atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA-enabled GPUs.  First released publicly in Apr. 2010 as part of version 11 of the AMBER MD package and further improved and optimized over the last two years, this implementation supports the three most widely used statistical mech. ensembles (NVE, NVT, and NPT), uses particle mesh Ewald (PME) for the long-range electrostatics, and runs entirely on CUDA-enabled NVIDIA graphics processing units (GPUs), providing results that are statistically indistinguishable from the traditional CPU version of the software and with performance that exceeds that achievable by the CPU version of AMBER software running on all conventional CPU-based clusters and supercomputers.  We briefly discuss three different precision models developed specifically for this work (SPDP, SPFP, and DPDP) and highlight the tech. details of the approach as it extends beyond previously reported work [Goetz et al., J.  Chem.  Theory Comput.2012, DOI: 10.1021/ct200909j; Le Grand et al., Comp.  Phys.  Comm.2013, DOI: 10.1016/j.cpc.2012.09.022].We highlight the substantial improvements in performance that are seen over traditional CPU-only machines and provide validation of our implementation and precision models.  We also provide evidence supporting our decision to deprecate the previously described fully single precision (SPSP) model from the latest release of the AMBER software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-M2E_jgUmLVg90H21EOLACvtfcHk0lhMFvOcpTy-7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP&md5=57e18d35e8b81e9a79788ef5af17d19f</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fct400314y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400314y%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25202.%2520Explicit%2520solvent%2520particle%2520mesh%2520ewald%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3878%26epage%3D3888%26doi%3D10.1021%2Fct400314y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">374</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1016/j.cpc.2012.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.cpc.2012.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2013&pages=374-380&author=S.+Le%0AGrandauthor=A.+W.+G%C3%B6tzauthor=R.+C.+Walker&title=SPFP%3A+Speed+without+compromise-A+mixed+precision+model+for+GPU+accelerated+molecular+dynamics+simulations&doi=10.1016%2Fj.cpc.2012.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span></div><div class="casAuthors">Le Grand, Scott; Gotz, Andreas W.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new precision model is proposed for the acceleration of all-atom classical mol. dynamics (MD) simulations on graphics processing units (GPUs).  This precision model replaces double precision arithmetic with fixed point integer arithmetic for the accumulation of force components as compared to a previously introduced model that uses mixed single/double precision arithmetic.  This significantly boosts performance on modern GPU hardware without sacrificing numerical accuracy.  We present an implementation for NVIDIA GPUs of both generalized Born implicit solvent simulations as well as explicit solvent simulations using the particle mesh Ewald (PME) algorithm for long-range electrostatics using this precision model.  Tests demonstrate both the performance of this implementation as well as its numerical stability for const. energy and const. temp. biomol. MD as compared to a double precision CPU implementation and double and mixed single/double precision GPU implementations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJB_spY7SrSbVg90H21EOLACvtfcHk0lhMFvOcpTy-7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO&md5=2d9f102cb17f6e9bcbff3b5057d8c587</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2012.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2012.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DSPFP%253A%2520Speed%2520without%2520compromise-A%2520mixed%2520precision%2520model%2520for%2520GPU%2520accelerated%2520molecular%2520dynamics%2520simulations%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2013%26volume%3D184%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.cpc.2012.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh Ewald: AnN·log(N) method for Ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+AnN%C2%B7log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0ljz9Oox4IP-UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520AnN%25C2%25B7log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of <i>n</i>-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0ljz9Oox4IP-UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span> <i>The PyMOL Molecular Graphics System</i>; <span class="NLM_publisher-name">DeLano Scientific LLC</span>: <span class="NLM_publisher-loc">Palo Alto, CA</span>, <span class="NLM_year">2008</span>. <a href="http://www.pymol.org/" class="extLink">http://www.pymol.org/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=W.+L.+DeLano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26btitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DDeLano%2520Scientific%2520LLC%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span> <span> </span><span class="NLM_article-title">PTRAJ and CPPTRAJ: software for processing and analysis of molecular dynamics trajectory data</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3084</span>– <span class="NLM_lpage">3095</span>, <span class="refDoi"> DOI: 10.1021/ct400341p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400341p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3084-3095&author=D.+R.+Roeauthor=T.+E.+Cheatham&title=PTRAJ+and+CPPTRAJ%3A+software+for+processing+and+analysis+of+molecular+dynamics+trajectory+data&doi=10.1021%2Fct400341p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span></div><div class="casAuthors">Roe, Daniel R.; Cheatham, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3084-3095</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe PTRAJ and its successor CPPTRAJ, two complementary, portable, and freely available computer programs for the anal. and processing of time series of three-dimensional at. positions (i.e., coordinate trajectories) and the data therein derived.  Common tools include the ability to manipulate the data to convert among trajectory formats, process groups of trajectories generated with ensemble methods (e.g., replica exchange mol. dynamics), image with periodic boundary conditions, create av. structures, strip subsets of the system, and perform calcns. such as RMS fitting, measuring distances, B-factors, radii of gyration, radial distribution functions, and time correlations, among other actions and analyses.  Both the PTRAJ and CPPTRAJ programs and source code are freely available under the GNU General Public License version 3 and are currently distributed within the AmberTools 12 suite of support programs that make up part of the Amber package of computer programs (see http://ambermd.org).  This overview describes the general design, features, and history of these two programs, as well as algorithmic improvements and new features available in CPPTRAJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97UsRfUy65bVg90H21EOLACvtfcHk0ljz9Oox4IP-UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D&md5=6f1bee934f13f180bd7e1feb6b78036d</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fct400341p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400341p%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DPTRAJ%2520and%2520CPPTRAJ%253A%2520software%2520for%2520processing%2520and%2520analysis%2520of%2520molecular%2520dynamics%2520trajectory%2520data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3084%26epage%3D3095%26doi%3D10.1021%2Fct400341p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">AMBER
2015 London Workshop</span>. <span> </span><span class="NLM_article-title">Introduction
to Principal Component
Analysis</span>. <a href="http://www.amber.utah.edu/AMBER-workshop/London-2015/pca/" class="extLink">http://www.amber.utah.edu/AMBER-workshop/London-2015/pca/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=AMBER%0A2015+London+Workshop.+Introduction%0Ato+Principal+Component%0AAnalysis.+http%3A%2F%2Fwww.amber.utah.edu%2FAMBER-workshop%2FLondon-2015%2Fpca%2F+%28accessed+Feb+13%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DIntroduction%250Ato%2520Principal%2520Component%250AAnalysis" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dötsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blázquez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamorano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haussler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">β-Lactam Resistance Response Triggered by Inactivation of a Nonessential Penicillin-Binding Protein</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e1000353</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1000353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1371%2Fjournal.ppat.1000353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=19325877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BD1M3jslaltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&author=B.+Moyaauthor=A.+D%C3%B6tschauthor=C.+Juanauthor=J.+Bl%C3%A1zquezauthor=L.+Zamoranoauthor=S.+Hausslerauthor=A.+Oliver&title=%CE%B2-Lactam+Resistance+Response+Triggered+by+Inactivation+of+a+Nonessential+Penicillin-Binding+Protein&doi=10.1371%2Fjournal.ppat.1000353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Beta-lactam resistance response triggered by inactivation of a nonessential penicillin-binding protein</span></div><div class="casAuthors">Moya Bartolome; Dotsch Andreas; Juan Carlos; Blazquez Jesus; Zamorano Laura; Haussler Susanne; Oliver Antonio</div><div class="citationInfo"><span class="NLM_cas:title">PLoS pathogens</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1000353</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">It has long been recognized that the modification of penicillin-binding proteins (PBPs) to reduce their affinity for beta-lactams is an important mechanism (target modification) by which Gram-positive cocci acquire antibiotic resistance.  Among Gram-negative rods (GNR), however, this mechanism has been considered unusual, and restricted to clinically irrelevant laboratory mutants for most species.  Using as a model Pseudomonas aeruginosa, high up on the list of pathogens causing life-threatening infections in hospitalized patients worldwide, we show that PBPs may also play a major role in beta-lactam resistance in GNR, but through a totally distinct mechanism.  Through a detailed genetic investigation, including whole-genome analysis approaches, we demonstrate that high-level (clinical) beta-lactam resistance in vitro, in vivo, and in the clinical setting is driven by the inactivation of the dacB-encoded nonessential PBP4, which behaves as a trap target for beta-lactams.  The inactivation of this PBP is shown to determine a highly efficient and complex beta-lactam resistance response, triggering overproduction of the chromosomal beta-lactamase AmpC and the specific activation of the CreBC (BlrAB) two-component regulator, which in turn plays a major role in resistance.  These findings are a major step forward in our understanding of beta-lactam resistance biology, and, more importantly, they open up new perspectives on potential antibiotic targets for the treatment of infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTPs-wl7dOX5b7NkxioyDXTfW6udTcc2eZPXHhS8IEZOLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3jslaltQ%253D%253D&md5=31df9ab3f0c62070d4415bfcb94790b0</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1000353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1000353%26sid%3Dliteratum%253Aachs%26aulast%3DMoya%26aufirst%3DB.%26aulast%3DD%25C3%25B6tsch%26aufirst%3DA.%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DBl%25C3%25A1zquez%26aufirst%3DJ.%26aulast%3DZamorano%26aufirst%3DL.%26aulast%3DHaussler%26aufirst%3DS.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3D%25CE%25B2-Lactam%2520Resistance%2520Response%2520Triggered%2520by%2520Inactivation%2520of%2520a%2520Nonessential%2520Penicillin-Binding%2520Protein%26jtitle%3DPLoS%2520Pathog.%26date%3D2009%26volume%3D5%26doi%3D10.1371%2Fjournal.ppat.1000353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. β-Lactamase inhibitors that have recently been approved or are undergoing clinical trials.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) 6-Arylmethylidene penicillin-based sulfones <b>1–7</b>. (B) Mechanism of inhibition of compounds <b>1–7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. MALDI spectra of samples of PDC-1 after incubation with ligands <b>1–3</b> (A) and of the control (incubated without ligands) (B). (C) Superimposed MALDI spectra. Incubation conditions were phosphate-buffered saline pH 7.4, 25 °C and (1:100) protein/ligand ratio, 30 min.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Detailed view of the catalytic active site of the unbound form of PDC-1 obtained by MD simulation studies, in which the ionic (A) and neutral (B) forms of the catalytic residues Tyr150 and Lys67 were considered. Snapshots taken after 80 ns of simulation are shown. Relevant side-chain residues of the active site are shown and labeled. Note how only the neutral forms of Lys67 and Tyr150 (B) provide the appropriate arrangement of Ser64 for formation of the enzyme-acyl adduct.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. A-C. Overall view of the binding mode of inhibitors <b>1</b> (cyan), <b>2</b> (violet), and <b>3</b> (green) in the PDC-1 active site (red) for acylation reaction obtained by MD simulation studies. Snapshots taken after 190 ns of simulation are shown. D-F. Interactions of inhibitors <b>1–3</b> in the active site of PDC-1. The position of the H10 helix is highlighted in orange. G-I. Closeup view of the relative position between the pyridyl moiety in <b>1</b> (G), <b>2</b> (H), and <b>3</b> (I) and the side chain of the catalytic tyrosine. Snapshots taken after 190 ns of simulation are shown. Note the strong interactions (black dashed lines) between the fluoropyridyl group in <b>3</b> and Tyr150 and Arg148. Relevant side-chain residues are shown and labeled. Polar interactions are indicated as red dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A–C) Overall view of the active site of PDC-1 inactivated by inhibitors <b>1</b> (cyan), <b>2</b> (violet), and <b>3</b> (green) obtained in MD simulation studies. Snapshots taken after 90 ns of simulation are shown. (D–F) Interactions between the PDC-1/<b>1–3</b> indolizine adducts. Relevant side-chain residues are shown and labeled. A close-up view of the interaction between the indolizine moiety and Tyr150 is shown. Polar (red), π–π, and CH−π (black) interactions are indicated as dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overall view of the active site of PDC-1 inactivated by avibactam (A, yellow) and inhibitor <b>2</b> (B, violet) obtained by X-ray crystallography (PDB entry 4OOY,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> 1.1 Å) and MD simulation studies, respectively. The position of the H10 helix is highlighted in orange. The modified catalytic serine is shown in green and labeled. Note how for the PDC-1/<b>2</b> adduct (B) the indolizine moiety of the modified ligand would be well embedded in the deep pocket surrounded by the H10 helix. On the contrary, for the PDC-1/avibactam adduct (A), this pocket is empty.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A,B) Overall view of the intrinsic shape-changing movements of AmpC BER (A) and CMY-10 (B) enzymes, obtained by examination of the vibrational modes and RMSD plots for the protein backbone (Cα, C, N, and O atoms) calculated per residue obtained from MD simulation studies. The main vibrational modes are shown. Note the high plasticity of the H10 helix in the unbound form of the BER enzyme relative to CMY-10. (C) Comparison of several snapshots of the BER/<b>2</b> binary complex during simulation and overall view of the tunnel-like pocket before (0 ns) and after simulation (200 ns). Note how after ∼20 ns of simulation the ligand induced folding of the H10 helix, which remained almost at a standstill as a ligand-covering lid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A,C) Overall view of the active site of plasmid-encoded AmpC CMY-10 (A) and BER (C) β-lactamases inactivated by inhibitor <b>2</b> (violet) obtained by MD simulation studies. (B,D) Proposed binding mode of bromide <b>2</b> in the active site of CMY-10 (B) and BER (D) for acylation reaction. Snapshots taken after 80 ns of simulation are shown. Relevant side-chain residues are shown and labeled. Polar (red) and apolar (black) interactions are indicated as dashed lines. The catalytic serine is highlighted in yellow. The positions of the H10 helix and the catalytic tyrosine are underlined in orange and blue, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/medium/jm1c00369_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A,C) Overall view of the active site of OXA-48 (A, class D) and TEM-1 (C, class A) β-lactamases inactivated by inhibitor <b>2</b> (violet), obtained by MD simulation studies. (B,D) Proposed binding mode of bromide <b>2</b> in the active site of OXA-48 (B) and TEM-1 (D) for acylation reaction. Snapshots taken after 80 ns of simulation are shown. Relevant side chain residues are shown and labeled. Polar (red) and apolar (black) interactions are indicated as dashed lines. The catalytic serine is highlighted in yellow. The residues that promote apolar interactions with the ligand are enhanced in green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00369/20210506/images/large/jm1c00369_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00369&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11830" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11830" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 80 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raudonis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span> <span> </span><span class="NLM_article-title">Antibiotic resistance in <i>Pseudomonas aeruginosa</i>: mechanisms and alternative therapeutic strategies</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2018.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.biotechadv.2018.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=30500353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVSgur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=177-192&author=Z.+Pangauthor=R.+Raudonisauthor=B.+R.+Glickauthor=T.-J.+Linauthor=Z.+Cheng&title=Antibiotic+resistance+in+Pseudomonas+aeruginosa%3A+mechanisms+and+alternative+therapeutic+strategies&doi=10.1016%2Fj.biotechadv.2018.11.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic resistance in Pseudomonas aeruginosa: mechanisms and alternative therapeutic strategies</span></div><div class="casAuthors">Pang, Zheng; Raudonis, Renee; Glick, Bernard R.; Lin, Tong-Jun; Cheng, Zhenyu</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-192</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is an opportunistic pathogen that is a leading cause of morbidity and mortality in cystic fibrosis patients and immunocompromised individuals.  Eradication of P. aeruginosa has become increasingly difficult due to its remarkable capacity to resist antibiotics.  Strains of Pseudomonas aeruginosa are known to utilize their high levels of intrinsic and acquired resistance mechanisms to counter most antibiotics.  In addn., adaptive antibiotic resistance of P. aeruginosa is a recently characterized mechanism, which includes biofilm-mediated resistance and formation of multidrug-tolerant persister cells, and is responsible for recalcitrance and relapse of infections.  The discovery and development of alternative therapeutic strategies that present novel avenues against P. aeruginosa infections are increasingly demanded and gaining more and more attention.  Although mostly at the preclin. stages, many recent studies have reported several innovative therapeutic technologies that have demonstrated pronounced effectiveness in fighting against drug-resistant P. aeruginosa strains.  This review highlights the mechanisms of antibiotic resistance in P. aeruginosa and discusses the current state of some novel therapeutic approaches for treatment of P. aeruginosa infections that can be further explored in clin. practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_z6YpDKCQnLVg90H21EOLACvtfcHk0lg-7Bar3q63MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVSgur%252FO&md5=9edf67e71c2362f36a08daa6174b9653</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2018.11.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2018.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DZ.%26aulast%3DRaudonis%26aufirst%3DR.%26aulast%3DGlick%26aufirst%3DB.%2BR.%26aulast%3DLin%26aufirst%3DT.-J.%26aulast%3DCheng%26aufirst%3DZ.%26atitle%3DAntibiotic%2520resistance%2520in%2520Pseudomonas%2520aeruginosa%253A%2520mechanisms%2520and%2520alternative%2520therapeutic%2520strategies%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2019%26volume%3D37%26spage%3D177%26epage%3D192%26doi%3D10.1016%2Fj.biotechadv.2018.11.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horcajada, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorlí, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luque, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Zorrilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benito, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grau, S.</span></span> <span> </span><span class="NLM_article-title">Epidemiology and Treatment of Multidrug-Resistant and Extensively Drug-Resistant Pseudomonas aeruginosaInfections</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>32</i></span>, <span class="NLM_elocation-id">e00031</span> <span class="refDoi"> DOI: 10.1128/cmr.00031-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FCMR.00031-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31462403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmt1WksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2019&author=J.+P.+Horcajadaauthor=M.+Monteroauthor=A.+Oliverauthor=L.+Sorl%C3%ADauthor=S.+Luqueauthor=S.+G%C3%B3mez-Zorrillaauthor=N.+Benitoauthor=S.+Grau&title=Epidemiology+and+Treatment+of+Multidrug-Resistant+and+Extensively+Drug-Resistant+Pseudomonas+aeruginosaInfections&doi=10.1128%2Fcmr.00031-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Epidemiology and treatment of multidrug-resistant and extensively drug-resistant Pseudomonas aeruginosa infections</span></div><div class="casAuthors">Horcajada, Juan P.; Montero, Milagro; Oliver, Antonio; Sorli, Luisa; Luque, Sonia; Gomez-Zorrilla, Silvia; Benito, Natividad; Grau, Santiago</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e00031-19/1-e00031-19/52</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">1098-6618</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In recent years, the worldwide spread of the so-called high-risk clones of multidrug-resistant or extensively drug-resistant (MDR/XDR) Pseudomonas aeruginosa has become a public health threat.  This article reviews their mechanisms of resistance, epidemiol., and clin. impact and current and upcoming therapeutic options.  In vitro and in vivo treatment studies and pharmacokinetic and pharmacodynamic (PK/PD) models are discussed.  Polymyxins are reviewed as an important therapeutic option, outlining dosage, pharmacokinetics and pharmacodynamics, and their clin. efficacy against MDR/XDR P. aeruginosa infections.  Their narrow therapeutic window and potential for combination therapy are also discussed.  Other "old" antimicrobials, such as certain β-lactams, aminoglycosides, and fosfomycin, are reviewed here.  New antipseudomonals, as well as those in the pipeline, are also reviewed.  Ceftolozane-tazobactam has clin. activity against a significant percentage of MDR/XDR P. aeruginosa strains, and its microbiol. and clin. data, as well as recommendations for improving its use against these bacteria, are described, as are those for ceftazidimeavibactam, which has better activity against MDR/XDR P. aeruginosa, esp. strains with certain specific mechanisms of resistance.  A section is devoted to reviewing upcoming active drugs such as imipenem-relebactam, cefepimezidebactam, cefiderocol, and murepavadin.  Finally, other therapeutic strategies, such as use of vaccines, antibodies, bacteriocins, anti-quorum sensing, and bacteriophages, are described as future options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof3LWnt3z50bVg90H21EOLACvtfcHk0lg-7Bar3q63MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmt1WksLY%253D&md5=49103d756d07d4761df425f326a23f56</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FCMR.00031-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00031-19%26sid%3Dliteratum%253Aachs%26aulast%3DHorcajada%26aufirst%3DJ.%2BP.%26aulast%3DMontero%26aufirst%3DM.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DSorl%25C3%25AD%26aufirst%3DL.%26aulast%3DLuque%26aufirst%3DS.%26aulast%3DG%25C3%25B3mez-Zorrilla%26aufirst%3DS.%26aulast%3DBenito%26aufirst%3DN.%26aulast%3DGrau%26aufirst%3DS.%26atitle%3DEpidemiology%2520and%2520Treatment%2520of%2520Multidrug-Resistant%2520and%2520Extensively%2520Drug-Resistant%2520Pseudomonas%2520aeruginosaInfections%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2019%26volume%3D32%26doi%3D10.1128%2Fcmr.00031-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K.</span></span> <span> </span><span class="NLM_article-title"><i>Pseudomonas aeruginosa:</i> evolution of antimicrobial resistance and implications for therapy</span>. <i>Semin. Respir. Crit. Care Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">044</span>– <span class="NLM_lpage">055</span>, <span class="refDoi"> DOI: 10.1055/s-0034-1396907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1055%2Fs-0034-1396907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25643270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BC2MritFOjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=044-055&author=K.+McCarthy&title=Pseudomonas+aeruginosa%3A+evolution+of+antimicrobial+resistance+and+implications+for+therapy&doi=10.1055%2Fs-0034-1396907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Pseudomonas aeruginosa: evolution of antimicrobial resistance and implications for therapy</span></div><div class="casAuthors">McCarthy Kate</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in respiratory and critical care medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-55</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Pseudomonas aeruginosa is a formidable pathogen in the infection arena.  It is able to easily adapt to the environment which it inhabits and can also colonize and invade the human host to cause serious infections.  In 2011, it was responsible for 7.1% of all health care-associated infection in the United States.  The morbidity and mortality of both blood stream infections and ventilator-associated pneumonia are significant.  On a global scale, we have seen the development of not only multidrug resistance but also extensive and pan drug resistance in this organism.  This is often associated with limited clonal types of which we now have epidemiological evidence of spread.  With this has come reduced antibiotic treatment options.  Consideration of antibiotic infusions, combination therapy, and inhalational therapy has occurred in an attempt to gain the upper ground.  Gram-negative resistance has appropriately been described as a global emergency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXX0ejtCSESuQ4Pl9hzjQhfW6udTcc2eaQfVXWvy1Gxbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MritFOjsg%253D%253D&md5=f23bfb68d827f861fcc01edbcb42de95</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1055%2Fs-0034-1396907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0034-1396907%26sid%3Dliteratum%253Aachs%26aulast%3DMcCarthy%26aufirst%3DK.%26atitle%3DPseudomonas%2520aeruginosa%253A%2520evolution%2520of%2520antimicrobial%2520resistance%2520and%2520implications%2520for%2520therapy%26jtitle%3DSemin.%2520Respir.%2520Crit.%2520Care%2520Med.%26date%3D2015%26volume%3D36%26spage%3D044%26epage%3D055%26doi%3D10.1055%2Fs-0034-1396907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulet, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Causapé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span> <span> </span><span class="NLM_article-title">The increasing threat of <i>Pseudomonas aeruginosa</i> high-risk clones</span>. <i>Drug Resist. Updates</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21-22</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/j.drup.2015.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.drup.2015.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=26304792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BC287lvVKktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21-22&publication_year=2015&pages=41-59&author=A.+Oliverauthor=X.+Muletauthor=C.+L%C3%B3pez-Causap%C3%A9author=C.+Juan&title=The+increasing+threat+of+Pseudomonas+aeruginosa+high-risk+clones&doi=10.1016%2Fj.drup.2015.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The increasing threat of Pseudomonas aeruginosa high-risk clones</span></div><div class="casAuthors">Oliver Antonio; Mulet Xavier; Lopez-Causape Carla; Juan Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Drug resistance updates : reviews and commentaries in antimicrobial and anticancer chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21-22</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-59</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The increasing prevalence of chronic and hospital-acquired infections produced by multidrug-resistant (MDR) or extensively drug-resistant (XDR) Pseudomonas aeruginosa strains is associated with significant morbidity and mortality.  This growing threat results from the extraordinary capacity of this pathogen for developing resistance through chromosomal mutations and from the increasing prevalence of transferable resistance determinants, particularly those encoding carbapenemases or extended-spectrum β-lactamases (ESBLs).  P. aeruginosa has a nonclonal epidemic population structure, composed of a limited number of widespread clones which are selected from a background of a large quantity of rare and unrelated genotypes that are recombining at high frequency.  Indeed, recent concerning reports have provided evidence of the existence of MDR/XDR global clones, denominated high-risk clones, disseminated in hospitals worldwide; ST235, ST111, and ST175 are likely those more widespread.  Noteworthy, the vast majority of infections by MDR, and specially XDR, strains are produced by these and few other clones worldwide.  Moreover, the association of high-risk clones, particularly ST235, with transferable resistance is overwhelming; nearly 100 different horizontally-acquired resistance elements and up to 39 different acquired β-lactamases have been reported so far among ST235 isolates.  Likewise, MDR internationally-disseminated epidemic strains, such as the Liverpool Epidemic Strain (LES, ST146), have been noted as well among cystic fibrosis patients.  Here we review the population structure, epidemiology, antimicrobial resistance mechanisms and virulence of the P. aeruginosa high-risk clones.  The phenotypic and genetic factors potentially driving the success of high-risk clones, the aspects related to their detection in the clinical microbiology laboratory and the implications for infection control and public health are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSq_r_KUrcD_kNHe_gFldTbfW6udTcc2eZ022KX5eHocrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287lvVKktQ%253D%253D&md5=f517cb617d315e9c66ea6a7bd876a93f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2015.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2015.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DMulet%26aufirst%3DX.%26aulast%3DL%25C3%25B3pez-Causap%25C3%25A9%26aufirst%3DC.%26aulast%3DJuan%26aufirst%3DC.%26atitle%3DThe%2520increasing%2520threat%2520of%2520Pseudomonas%2520aeruginosa%2520high-risk%2520clones%26jtitle%3DDrug%2520Resist.%2520Updates%26date%3D2015%26volume%3D21-22%26spage%3D41%26epage%3D59%26doi%3D10.1016%2Fj.drup.2015.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hernando-Amado, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coque, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baquero, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez, J. L.</span></span> <span> </span><span class="NLM_article-title">Defining and combating antibiotic resistance from one health and global health perspectives</span>. <i>Nat. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1038/s41564-019-0503-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1038%2Fs41564-019-0503-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31439928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1aqsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=1432-1442&author=S.+Hernando-Amadoauthor=T.+M.+Coqueauthor=F.+Baqueroauthor=J.+L.+Mart%C3%ADnez&title=Defining+and+combating+antibiotic+resistance+from+one+health+and+global+health+perspectives&doi=10.1038%2Fs41564-019-0503-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Defining and combating antibiotic resistance from One Health and Global Health perspectives</span></div><div class="casAuthors">Hernando-Amado, Sara; Coque, Teresa M.; Baquero, Fernando; Martinez, Jose L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1432-1442</span>CODEN:
                <span class="NLM_cas:coden">NMAICH</span>;
        ISSN:<span class="NLM_cas:issn">2058-5276</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Several interconnected human, animal and environmental habitats can contribute to the emergence, evolution and spread of antibiotic resistance, and the health of these contiguous habitats (the focus of the One Health approach) may represent a risk to human health.  Addnl., the expansion of resistant clones and antibiotic resistance determinants among human-assocd., animal-assocd. and environmental microbiomes have the potential to alter bacterial population genetics at local and global levels, thereby modifying the structure, and eventually the productivity, of microbiomes where antibiotic-resistant bacteria can expand.  Conversely, any change in these habitats (including pollution by antibiotics or by antibiotic-resistant organisms) may influence the structures of their assocd. bacterial populations, which might affect the spread of antibiotic resistance to, and among, the above-mentioned microbiomes.  Besides local transmission among connected habitats-the focus of studies under the One Health concept-the transmission of resistant microorganisms might occur on a broader (even worldwide) scale, requiring coordinated Global Health actions.  This Review provides updated information on the elements involved in the evolution and spread of antibiotic resistance at local and global levels, and proposes studies to be performed and strategies to be followed that may help reduce the burden of antibiotic resistance as well as its impact on human and planetary health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZUx4oYBVjtrVg90H21EOLACvtfcHk0ljUpBrlBx588w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1aqsrfO&md5=9c15fac3a8837042a2b6afda14b9065a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41564-019-0503-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41564-019-0503-9%26sid%3Dliteratum%253Aachs%26aulast%3DHernando-Amado%26aufirst%3DS.%26aulast%3DCoque%26aufirst%3DT.%2BM.%26aulast%3DBaquero%26aufirst%3DF.%26aulast%3DMart%25C3%25ADnez%26aufirst%3DJ.%2BL.%26atitle%3DDefining%2520and%2520combating%2520antibiotic%2520resistance%2520from%2520one%2520health%2520and%2520global%2520health%2520perspectives%26jtitle%3DNat.%2520Microbiol.%26date%3D2019%26volume%3D4%26spage%3D1432%26epage%3D1442%26doi%3D10.1038%2Fs41564-019-0503-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span> </span><span class="NLM_article-title">Prioritization of Pathogens to Guide Discovery,
Research and Development
of New Antibiotics for Drug Resistant Bacterial Infections, Including
Tuberculosis: WHO/EMP/IAU/2017.12</span>. <a href="https://www.who.int/medicines/areas/rational_use/prioritization-of-pathogens/en/" class="extLink">https://www.who.int/medicines/areas/rational_use/prioritization-of-pathogens/en/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Prioritization+of+Pathogens+to+Guide+Discovery%2C%0AResearch+and+Development%0Aof+New+Antibiotics+for+Drug+Resistant+Bacterial+Infections%2C+Including%0ATuberculosis%3A+WHO%2FEMP%2FIAU%2F2017.12.+https%3A%2F%2Fwww.who.int%2Fmedicines%2Fareas%2Frational_use%2Fprioritization-of-pathogens%2Fen%2F+%28accessed+Feb+13%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DPrioritization%2520of%2520Pathogens%2520to%2520Guide%2520Discovery%252C%250AResearch%2520and%2520Development%250Aof%2520New%2520Antibiotics%2520for%2520Drug%2520Resistant%2520Bacterial%2520Infections%252C%2520Including%250ATuberculosis%253A%2520WHO%252FEMP%252FIAU%252F2017.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poole, K.</span></span> <span> </span><span class="NLM_article-title">Resistance to β-lactam antibiotics</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2200</span>– <span class="NLM_lpage">2223</span>, <span class="refDoi"> DOI: 10.1007/s00018-004-4060-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1007%2Fs00018-004-4060-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=15338052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD2cXovVyhtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2004&pages=2200-2223&author=K.+Poole&title=Resistance+to+%CE%B2-lactam+antibiotics&doi=10.1007%2Fs00018-004-4060-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to β-lactam antibiotics</span></div><div class="casAuthors">Poole, K.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2200-2223</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  β-Lactams have a long history in the treatment of infectious diseases, though their use was and continues to be confounded by the development of resistance in target organisms.  β-Lactamases, particularly in Gram-neg. pathogens, are a major determinant of this resistance, although alterations in the β-lactam targets, the penicillin-binding proteins (PBPs), are also important, esp. in Gram-pos. pathogens.  Mechanisms for the efflux and/or exclusion of these agents also contribute, though often in conjunction these other 2 .  Approaches for overcoming these resistance mechanisms include the development of novel β-lactamase-stable β-lactams, β-lactamase inhibitors to be employed with existing β-lactams, β-lactam compds. that bind strongly to low-affinity PBPs and agents that potentiate the activity of existing β-lactams against low-affinity PBP-producing organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ8MYoaxBP9bVg90H21EOLACvtfcHk0ljUpBrlBx588w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXovVyhtbs%253D&md5=f107b3e35c8386ef2986f01d35dd3f36</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00018-004-4060-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-004-4060-9%26sid%3Dliteratum%253Aachs%26aulast%3DPoole%26aufirst%3DK.%26atitle%3DResistance%2520to%2520%25CE%25B2-lactam%2520antibiotics%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2004%26volume%3D61%26spage%3D2200%26epage%3D2223%26doi%3D10.1007%2Fs00018-004-4060-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocampo-Sosa, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macia, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamorano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suárez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peña, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martínez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">REIPI</span></span> <span> </span><span class="NLM_article-title">Overexpression of AmpC and efflux pumps in <i>Pseudomonas aeruginosa</i> isolates from bloodstream infections: prevalence and impact on resistance in a Spanish multicenter study</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1906</span>– <span class="NLM_lpage">1911</span>, <span class="refDoi"> DOI: 10.1128/aac.01645-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01645-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=21357294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCls7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=1906-1911&author=G.+Cabotauthor=A.+A.+Ocampo-Sosaauthor=F.+Tubauauthor=M.+D.+Maciaauthor=C.+Rodr%C3%ADguezauthor=B.+Moyaauthor=L.+Zamoranoauthor=C.+Su%C3%A1rezauthor=C.+Pe%C3%B1aauthor=L.+Mart%C3%ADnez-Mart%C3%ADnezauthor=A.+Oliverauthor=+REIPI&title=Overexpression+of+AmpC+and+efflux+pumps+in+Pseudomonas+aeruginosa+isolates+from+bloodstream+infections%3A+prevalence+and+impact+on+resistance+in+a+Spanish+multicenter+study&doi=10.1128%2Faac.01645-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of AmpC and efflux pumps in Pseudomonas aeruginosa isolates from bloodstream infections: prevalence and impact on resistance in a Spanish multicenter study</span></div><div class="casAuthors">Cabot, Gabriel; Ocampo-Sosa, Alain A.; Tubau, Fe; Macia, Maria D.; Rodriguez, Cristina; Moya, Bartolome; Zamorano, Laura; Suarez, Cristina; Pena, Carmen; Martinez-Martinez, Luis; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1906-1911</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The prevalence and impact of the overexpression of AmpC and efflux pumps were evaluated with a collection of 190 Pseudomonas aeruginosa isolates recovered from bloodstream infections in a 2008 multicenter study (10 hospitals) in Spain.  The MICs of a panel of 13 antipseudomonal agents were detd. by microdilution, and the expressions of ampC, mexB, mexY, mexD, and mexF were detd. by real-time reverse transcription (RT)-PCR.  Up to 39% of the isolates overexpressed at least one of the mechanisms.  The ampC overexpression (24.2%) was the most prevalent mechanism, followed by mexY (13.2%), mexB (12.6%), mexF (4.2%), and mexD (2.2%).  The overexpression of mexB plus mexY, documented for 5.3% of the isolates, was the only combination showing a significantly (P = 0.02) higher prevalence than expected from the frequencies of the individual mechanisms (1.6%).  Addnl., all imipenem-resistant isolates studied (25 representative isolates) showed inactivating mutations in oprD.  Most of the isolates nonsusceptible to piperacillin-tazobactam (96%) and ceftazidime (84%) overexpressed ampC, while mexB (25%) and mexY (29%) overexpressions gained relevance among cefepime-nonsusceptible isolates.  Nevertheless, the prevalence of mexY overexpression was highest among tobramycin-nonsusceptible isolates (37%), and that of mexB was highest among meropenem-nonsusceptible isolates (33%).  Regarding ciprofloxacin-resistant isolates, besides the expected increased prevalence of efflux pump overexpression, a highly significant link to ampC overexpression was documented for the first time: up to 52% of ciprofloxacin-nonsusceptible isolates overexpressed ampC, sharply contrasting with the 24% documented for the complete collection (P < 0.001).  Thus, mutation-driven resistance was frequent in P. aeruginosa isolates from bloodstream infections, whereas metallo-β-lactamases, detected in 2 isolates (1%) producing VIM-2, although with increasing prevalences, were still uncommon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql28rd0pn3KrVg90H21EOLACvtfcHk0lidYLdOu8wXtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCls7bO&md5=8f993e34c110e6b0c08d9a97e827e76a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2FAAC.01645-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01645-10%26sid%3Dliteratum%253Aachs%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DOcampo-Sosa%26aufirst%3DA.%2BA.%26aulast%3DTubau%26aufirst%3DF.%26aulast%3DMacia%26aufirst%3DM.%2BD.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DC.%26aulast%3DMoya%26aufirst%3DB.%26aulast%3DZamorano%26aufirst%3DL.%26aulast%3DSu%25C3%25A1rez%26aufirst%3DC.%26aulast%3DPe%25C3%25B1a%26aufirst%3DC.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADnez%26aufirst%3DL.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DREIPI%26atitle%3DOverexpression%2520of%2520AmpC%2520and%2520efflux%2520pumps%2520in%2520Pseudomonas%2520aeruginosa%2520isolates%2520from%2520bloodstream%2520infections%253A%2520prevalence%2520and%2520impact%2520on%2520resistance%2520in%2520a%2520Spanish%2520multicenter%2520study%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2011%26volume%3D55%26spage%3D1906%26epage%3D1911%26doi%3D10.1128%2Faac.01645-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berrazeg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeannot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ntsogo Enguéné, V. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broutin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeffert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plésiat, P.</span></span> <span> </span><span class="NLM_article-title">Mutations in β-Lactamase AmpC Increase Resistance of Pseudomonas aeruginosa Isolates to Antipseudomonal Cephalosporins</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6248</span>– <span class="NLM_lpage">6255</span>, <span class="refDoi"> DOI: 10.1128/aac.00825-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.00825-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=26248364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yhurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=6248-6255&author=M.+Berrazegauthor=K.+Jeannotauthor=V.+Y.+Ntsogo+Engu%C3%A9n%C3%A9author=I.+Broutinauthor=S.+Loeffertauthor=D.+Fournierauthor=P.+Pl%C3%A9siat&title=Mutations+in+%CE%B2-Lactamase+AmpC+Increase+Resistance+of+Pseudomonas+aeruginosa+Isolates+to+Antipseudomonal+Cephalosporins&doi=10.1128%2Faac.00825-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in β-lactamase AmpC increase resistance of Pseudomonas aeruginosa isolates to antipseudomonal cephalosporins</span></div><div class="casAuthors">Berrazeg, M.; Jeannot, K.; Enguene, Veronique Yvette Ntsogo; Broutin, I.; Loeffert, S.; Fournier, D.; Plesiat, P.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6248-6255</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Mutation-dependent overprodn. of intrinsic β-lactamase AmpC is considered the main cause of resistance of clin. strains of Pseudomonas aeruginosa to antipseudomonal penicillins and cephalosporins.  Anal. of 31 AmpC-overproducing clin. isolates exhibiting a greater resistance to ceftazidime than to piperacillin-tazobactam revealed 17 mutations in the β-lactamase, combined with various polymorphic amino acid substitutions.  When overexpressed in AmpC-deficient P. aeruginosa 4098, the genes coding for 20/23 of these AmpC variants confer a higher (2-fold to >64-fold) resistance to ceftazidime and ceftolozane-tazobactam than did the gene from ref. strain PAO1.  The mutations had variable effects on the MICs of ticarcillin, piperacillin-tazobactam, aztreonam, and cefepime.  Depending on their location in the AmpC structure and their impact on β-lactam MICs, they could be assigned to 4 distinct groups.  Most of the mutations affecting the omega loop, the R2 domain, and the C-terminal end of the protein were shared with extended-spectrum AmpCs (ESACs) from other Gram-neg. species.  Two new mutations (F121L and P154L) were predicted to enlarge the substrate binding pocket by disrupting the stacking between residues F121 and P154.  Also the reported ESACs emerged locally in a variety of clones, some of which are epidemic and did not require hypermutability.  Taken together, the authors' results show that P. aeruginosa is able to adapt to efficacious β-lactams, including the newer cephalosporin ceftolozane, through a variety of mutations affecting its intrinsic β-lactamase, AmpC.  Data suggest that the rates of ESAC-producing mutants are ≥1.5% in the clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ1K5kAOUjxLVg90H21EOLACvtfcHk0lidYLdOu8wXtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yhurw%253D&md5=6d196195beee010e7618c8b84e09f0fc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FAAC.00825-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00825-15%26sid%3Dliteratum%253Aachs%26aulast%3DBerrazeg%26aufirst%3DM.%26aulast%3DJeannot%26aufirst%3DK.%26aulast%3DNtsogo%2BEngu%25C3%25A9n%25C3%25A9%26aufirst%3DV.%2BY.%26aulast%3DBroutin%26aufirst%3DI.%26aulast%3DLoeffert%26aufirst%3DS.%26aulast%3DFournier%26aufirst%3DD.%26aulast%3DPl%25C3%25A9siat%26aufirst%3DP.%26atitle%3DMutations%2520in%2520%25CE%25B2-Lactamase%2520AmpC%2520Increase%2520Resistance%2520of%2520Pseudomonas%2520aeruginosa%2520Isolates%2520to%2520Antipseudomonal%2520Cephalosporins%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D6248%26epage%3D6255%26doi%3D10.1128%2Faac.00825-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Nicolau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Diversity and regulation of intrinsic β-lactamases from non-fermenting and other Gram-negative opportunistic pathogens</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">781</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1093/femsre/fux043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Ffemsre%2Ffux043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=29029112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFWmt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2017&pages=781-815&author=C.+Juanauthor=G.+Torrensauthor=M.+Gonz%C3%A1lez-Nicolauauthor=A.+Oliver&title=Diversity+and+regulation+of+intrinsic+%CE%B2-lactamases+from+non-fermenting+and+other+Gram-negative+opportunistic+pathogens&doi=10.1093%2Ffemsre%2Ffux043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Diversity and regulation of intrinsic β-lactamases from non-fermenting and other Gram-negative opportunistic pathogens</span></div><div class="casAuthors">Juan, Carlos; Torrens, Gabriel; Gonzalez-Nicolau, Mar; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">781-815</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">1574-6976</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">This review deeply addresses for the first time the diversity, regulation and mechanisms leading to mutational overexpression of intrinsic β-lactamases from non-fermenting and other non-Enterobacteriaceae Gram-neg. opportunistic pathogens.  After a general overview of the intrinsic β-lactamases described so far in these microorganisms, including circa. 60 species and 100 different enzymes, we review the wide array of regulatory pathways of these β-lactamases.  They include diverse LysR-type regulators, which control the expression of β-lactamases from relevant nosocomial pathogens such as Pseudomonas aeruginosa or Stenothrophomonas maltophilia or two-component regulators, with special relevance in Aeromonas spp., along with other pathways.  Likewise, the multiple mutational mechanisms leading to β-lactamase overexpression and β-lactam resistance development, including AmpD (N-acetyl-muramyl-L-alanine amidase), DacB (PBP4), MrcA (PPBP1A) and other PBPs, BlrAB (two-component regulator) or several lytic transglycosylases among others, are also described.  Moreover, we address the growing evidence of a major interplay between β-lactamase regulation, peptidoglycan metab. and virulence.  Finally, we analyze recent works showing that blocking of peptidoglycan recycling (such as inhibition of NagZ or AmpG) might be useful to prevent and revert β-lactam resistance.  Altogether, the provided information and the identified gaps should be valuable for guiding future strategies for combating multidrug-resistant Gram-neg. pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvR40C1kf9YrVg90H21EOLACvtfcHk0lidYLdOu8wXtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFWmt7fF&md5=2f67e56b3ccd3db2e76972aeb8f8c81b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Ffemsre%2Ffux043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ffemsre%252Ffux043%26sid%3Dliteratum%253Aachs%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DTorrens%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Nicolau%26aufirst%3DM.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DDiversity%2520and%2520regulation%2520of%2520intrinsic%2520%25CE%25B2-lactamases%2520from%2520non-fermenting%2520and%2520other%2520Gram-negative%2520opportunistic%2520pathogens%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2017%26volume%3D41%26spage%3D781%26epage%3D815%26doi%3D10.1093%2Ffemsre%2Ffux043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathlouthi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Bayssari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolain, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chouchani, C.</span></span> <span> </span><span class="NLM_article-title">RETRACTED ARTICLE: Prevalence and emergence of carbapenemases-producing Gram-negative bacteria in Mediterranean basin</span>. <i>Crit. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.3109/1040841x.2016.1160867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.3109%2F1040841X.2016.1160867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=27387224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFOhurbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2017&pages=43-61&author=N.+Mathlouthiauthor=C.+Al-Bayssariauthor=S.+Bakourauthor=J.+M.+Rolainauthor=C.+Chouchani&title=RETRACTED+ARTICLE%3A+Prevalence+and+emergence+of+carbapenemases-producing+Gram-negative+bacteria+in+Mediterranean+basin&doi=10.3109%2F1040841x.2016.1160867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence and emergence of carbapenemases-producing Gram-negative bacteria in Mediterranean basin</span></div><div class="casAuthors">Mathlouthi, Najla; Al-Bayssari, Charbel; Bakour, Sofiane; Rolain, Jean Marc; Chouchani, Chedly</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-61</span>CODEN:
                <span class="NLM_cas:coden">CRVMAC</span>;
        ISSN:<span class="NLM_cas:issn">1040-841X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The emergence and the global spread of carbapenemases concern to health services worldwide.  Their celestial rise among Gram-neg. bacilli has challenged both the scientific and pharmaceutical sectors.  Indeed, infections caused by these bacteria have limited treatment options and have been assocd. with high mortality and morbidity rates.  Carbapenemase producers are mainly identified among Klebsiella pneumoniae, Escherichia coli, Pseudomonas aeruginosa, and Acinetobacter baumannii and still mostly in hospital settings and rarely in the community.  They are closely related to KPC, VIM, IMP, NDM, and OXA-48 types.  The encoding genes are mostly plasmid located and assocd. with various mobile genetic elements.  The Mediterranean area is of interest due to a great diversity and population mixing.  The prevalence of carbapenemases is particularly high and variant among countries, partially depending on the population exchange relationship between the regions and the possible reservoirs of each carbapenemase.  This review described the epidemiol. of carbapenemases in this region of the world highlighting the worrisome situation and the need to screen and detect these enzymes to prevent and control their dissemination esp. as it is clear that very few novel antibiotics will be introduced in the next few years, making the dissemination of carbapenem-resistant Gram-neg. bacteria of crucial importance worldwide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjgom8TLXe_LVg90H21EOLACvtfcHk0lgCX9Uy1v6F1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFOhurbP&md5=09d8dd20738b6023fc095f0f11b22d1f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3109%2F1040841X.2016.1160867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F1040841X.2016.1160867%26sid%3Dliteratum%253Aachs%26aulast%3DMathlouthi%26aufirst%3DN.%26aulast%3DAl-Bayssari%26aufirst%3DC.%26aulast%3DBakour%26aufirst%3DS.%26aulast%3DRolain%26aufirst%3DJ.%2BM.%26aulast%3DChouchani%26aufirst%3DC.%26atitle%3DRETRACTED%2520ARTICLE%253A%2520Prevalence%2520and%2520emergence%2520of%2520carbapenemases-producing%2520Gram-negative%2520bacteria%2520in%2520Mediterranean%2520basin%26jtitle%3DCrit.%2520Rev.%2520Microbiol.%26date%3D2017%26volume%3D43%26spage%3D43%26epage%3D61%26doi%3D10.3109%2F1040841x.2016.1160867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouffartigues, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodilis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillot, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesouhaitier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feuilloley, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orange, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornelis, P.</span></span> <span> </span><span class="NLM_article-title">Structure, function and regulation of <i>Pseudomonas aeruginosa</i> porins</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">698</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1093/femsre/fux020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Ffemsre%2Ffux020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=28981745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGjsLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2017&pages=698-722&author=S.+Chevalierauthor=E.+Bouffartiguesauthor=J.+Bodilisauthor=O.+Maillotauthor=O.+Lesouhaitierauthor=M.+G.+J.+Feuilloleyauthor=N.+Orangeauthor=A.+Dufourauthor=P.+Cornelis&title=Structure%2C+function+and+regulation+of+Pseudomonas+aeruginosa+porins&doi=10.1093%2Ffemsre%2Ffux020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, function and regulation of Pseudomonas aeruginosa porins</span></div><div class="casAuthors">Chevalier, Sylvie; Bouffartigues, Emeline; Bodilis, Josselin; Maillot, Olivier; Lesouhaitier, Olivier; Feuilloley, Marc G. J.; Orange, Nicole; Dufour, Alain; Cornelis, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">698-722</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">1574-6976</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is a Gram-neg. bacterium belonging to the γ -proteobacteria.  Like other members of the Pseudomonas genus, it is known for its metabolic versatility and its ability to colonize a wide range of ecol. niches, such as rhizosphere, water environments and animal hosts, including humans where it can cause severe infections.  Another particularity of P. aeruginosa is its high intrinsic resistance to antiseptics and antibiotics, which is partly due to its low outer membrane permeability.  In contrast to Enterobacteria, pseudomonads do not possess general diffusion porins in their outer membrane, but rather express specific channel proteins for the uptake of different nutrients.  The major outer membrane 'porin', OprF, has been extensively investigated, and displays structural, adhesion and signaling functions while its role in the diffusion of nutrients is still under discussion.  Other porins include OprB and OprB2 for the diffusion of glucose, the two small outer membrane proteins OprG and OprH, and the two porins involved in phosphate/pyrophosphate uptake, OprP and OprO.  The remaining nineteen porins belong to the so-called OprD (Occ) family, which is further split into two subfamilies termed OccD (8 members) and OccK (11 members).  In the past years, a large amt. of information concerning the structure, function and regulation of these porins has been published, justifying why an updated review is timely.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobvqc3Kan7nLVg90H21EOLACvtfcHk0lgCX9Uy1v6F1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGjsLnM&md5=9264cfacba2796f9e5f5fd367f4a423e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Ffemsre%2Ffux020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ffemsre%252Ffux020%26sid%3Dliteratum%253Aachs%26aulast%3DChevalier%26aufirst%3DS.%26aulast%3DBouffartigues%26aufirst%3DE.%26aulast%3DBodilis%26aufirst%3DJ.%26aulast%3DMaillot%26aufirst%3DO.%26aulast%3DLesouhaitier%26aufirst%3DO.%26aulast%3DFeuilloley%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DOrange%26aufirst%3DN.%26aulast%3DDufour%26aufirst%3DA.%26aulast%3DCornelis%26aufirst%3DP.%26atitle%3DStructure%252C%2520function%2520and%2520regulation%2520of%2520Pseudomonas%2520aeruginosa%2520porins%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2017%26volume%3D41%26spage%3D698%26epage%3D722%26doi%3D10.1093%2Ffemsre%2Ffux020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R. E. W.</span></span> <span> </span><span class="NLM_article-title">Adaptive and mutational resistance: role of porins and efflux pumps in drug resistance</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1128/cmr.00043-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FCMR.00043-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=23034325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWjsb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=661-681&author=L.+Fern%C3%A1ndezauthor=R.+E.+W.+Hancock&title=Adaptive+and+mutational+resistance%3A+role+of+porins+and+efflux+pumps+in+drug+resistance&doi=10.1128%2Fcmr.00043-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Adaptive and mutational resistance: role of porins and efflux pumps in drug resistance</span></div><div class="casAuthors">Fernandez, Lucia; Hancock, Robert E. W.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">661-681</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  The substantial use of antibiotics in the clinic, combined with a dearth of new antibiotic classes, has led to a gradual increase in the resistance of bacterial pathogens to these compds.  Among the various mechanisms by which bacteria endure the action of antibiotics, those affecting influx and efflux are of particular importance, as they limit the interaction of the drug with its intracellular targets and, consequently, its deleterious effects on the cell.  This review evaluates the impact of porins and efflux pumps on two major types of resistance, namely, mutational and adaptive types of resistance, both of which are regarded as key phenomena in the global rise of antibiotic resistance among pathogenic microorganisms.  In particular, we explain how adaptive and mutational events can dramatically influence the outcome of antibiotic therapy by altering the mechanisms of influx and efflux of antibiotics.  The identification of porins and pumps as major resistance markers has opened new possibilities for the development of novel therapeutic strategies directed specifically against these mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojE1hifwDBlbVg90H21EOLACvtfcHk0lgCX9Uy1v6F1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWjsb3K&md5=738ffab800ded95e47ae424a4a1a4283</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FCMR.00043-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00043-12%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez%26aufirst%3DL.%26aulast%3DHancock%26aufirst%3DR.%2BE.%2BW.%26atitle%3DAdaptive%2520and%2520mutational%2520resistance%253A%2520role%2520of%2520porins%2520and%2520efflux%2520pumps%2520in%2520drug%2520resistance%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2012%26volume%3D25%26spage%3D661%26epage%3D681%26doi%3D10.1128%2Fcmr.00043-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span> <span> </span><span class="NLM_article-title">The latest advances in β-lactam/β-lactamase inhibitor combinations for the treatment of Gram-negative bacterial infections</span>. <i>Expert Opin. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2169</span>– <span class="NLM_lpage">2184</span>, <span class="refDoi"> DOI: 10.1080/14656566.2019.1660772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1080%2F14656566.2019.1660772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31500471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslGnu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=2169-2184&author=K.+M.+Papp-Wallace&title=The+latest+advances+in+%CE%B2-lactam%2F%CE%B2-lactamase+inhibitor+combinations+for+the+treatment+of+Gram-negative+bacterial+infections&doi=10.1080%2F14656566.2019.1660772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The latest advances in β-lactam/β-lactamase inhibitor combinations for the treatment of Gram-negative bacterial infections</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Pharmacotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2169-2184</span>CODEN:
                <span class="NLM_cas:coden">EOPHF7</span>;
        ISSN:<span class="NLM_cas:issn">1465-6566</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : Antimicrobial resistance in Gram-neg. pathogens is a significant threat to global health. β-Lactams (BL) are one of the safest and most-prescribed classes of antibiotics on the market today.  The acquisition of β-lactamases, esp. those which hydrolyze carbapenems, is eroding the efficacy of BLs for the treatment of serious infections.  During the past decade, significant advances were made in the development of novel BL-β-lactamase inhibitor (BLI) combinations to target β-lactamase-mediated resistant Gram-negatives.: The latest progress in 20 different approved, developing, and preclin. BL-BLI combinations to target serine β-lactamases produced by Gram-negatives are reviewed based on primary literature, conference abstrs. (when available), and US clin. trial searches within the last 5 years.  The majority of the compds. that are discussed are being evaluated as part of a BL-BLI combination.: The current trajectory in BLI development is promising; however, a significant challenge resides in the selection of an appropriate BL partner as well as the development of resistance linked to the BL partner.  In addn., dosing regimens for these BL-BLI combinations need to be critically evaluated.  A revolution in bacterial diagnostics is essential to aid clinicians in the appropriate selection of novel BL-BLI combinations for the treatment of serious infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlWIU5sOUqv7Vg90H21EOLACvtfcHk0lgCX9Uy1v6F1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslGnu7jL&md5=73f3e0ba8358820479bf1652b26fc4a8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F14656566.2019.1660772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14656566.2019.1660772%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26atitle%3DThe%2520latest%2520advances%2520in%2520%25CE%25B2-lactam%252F%25CE%25B2-lactamase%2520inhibitor%2520combinations%2520for%2520the%2520treatment%2520of%2520Gram-negative%2520bacterial%2520infections%26jtitle%3DExpert%2520Opin.%2520Pharmacother.%26date%3D2019%26volume%3D20%26spage%3D2169%26epage%3D2184%26doi%3D10.1080%2F14656566.2019.1660772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arca-Suárez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lence, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Iglesias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán-Sánchez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Challenging antimicrobial susceptibility and evolution of resistance (OXA-681) during treatment of a long-term nosocomial infection caused by a <i>Pseudomonas aeruginosa</i> ST175 clone</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>63</i></span>, <span class="NLM_elocation-id">e01110</span> <span class="refDoi"> DOI: 10.1128/aac.01110-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01110-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31383659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1eisbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2019&author=J.+Arca-Su%C3%A1rezauthor=P.+Fraile-Ribotauthor=J.+C.+V%C3%A1zquez-Uchaauthor=G.+Cabotauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=E.+Lenceauthor=C.+Gonz%C3%A1lez-Belloauthor=A.+Beceiroauthor=M.+Rodr%C3%ADguez-Iglesiasauthor=F.+Gal%C3%A1n-S%C3%A1nchezauthor=G.+Bouauthor=A.+Oliver&title=Challenging+antimicrobial+susceptibility+and+evolution+of+resistance+%28OXA-681%29+during+treatment+of+a+long-term+nosocomial+infection+caused+by+a+Pseudomonas+aeruginosa+ST175+clone&doi=10.1128%2Faac.01110-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Challenging antimicrobial susceptibility and evolution of resistance (OXA-681) during treatment of a long-term nosocomial infection caused by a Pseudomonas aeruginosa ST175 clone</span></div><div class="casAuthors">Arca-Suarez, Jorge; Fraile-Ribot, Pablo; Vazquez-Ucha, Juan Carlos; Cabot, Gabriel; Martinez-Guitian, Marta; Lence, Emilio; Gonzalez-Bello, Concepcion; Beceiro, Alejandro; Rodriguez-Iglesias, Manuel; Galan-Sanchez, Fatima; Bou, German; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e01110-19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Selection of extended-spectrum mutations in narrow-spectrum oxacillinases (e.g., OXA-2 and OXA-10) is an emerging mechanism for development of in vivo resistance to ceftolozane-tazobactam and ceftazidime-avibactam in Pseudomonas aeruginosa.  Detection of these challenging enzymes therefore seems essential to prevent clin. failure, but the complex phenotypic plasticity exhibited by this species may often lead to their underestimation.  The underlying resistance mechanisms of two sequence type 175 (ST175) P. aeruginosa isolates showing multidrug-resistant phenotypes and recovered at early and late stages of a long-term nosocomial infection were evaluated.  Whole-genome sequencing (WGS) was used to investigate resistance genomics, whereas mol. and biochem. methods were used for characterization of a novel extended-spectrum OXA-2 variant selected during therapy.  The metallo-β-lactamase blaVIM-20 and the narrow-spectrum oxacillinase blaOXA-2 were present in both isolates, although they differed by an inactivating mutation in the mexB subunit, present only in the early isolate, and in a mutation in the blaOXA-2 β-lactamase, present only in the final isolate.  The new OXA-2 variant, designated OXA-681, conferred elevated MICs of the novel cephalosporin-β-lactamase inhibitor combinations in a PAO1 background.  Compared to OXA-2, kinetic parameters of the OXA-681 enzyme revealed a substantial increase in the hydrolysis of cephalosporins, including ceftolozane.  We describe the emergence of the novel variant OXA-681 during treatment of a nosocomial infection caused by a Pseudomonas aeruginosa ST175 high-risk clone.  The ability of OXA-681 to confer cross-resistance to ceftolozane-tazobactam and ceftazidime-avibactam together with the complex antimicrobial resistance profiles exhibited by the clin. strains harboring this new enzyme argue for maintaining active surveillance on emerging broad-spectrum resistance in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXTq7Nwva4VLVg90H21EOLACvtfcHk0ljLAPdkvJwlKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1eisbjL&md5=eda2d550de42c6a943094516563994b5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FAAC.01110-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01110-19%26sid%3Dliteratum%253Aachs%26aulast%3DArca-Su%25C3%25A1rez%26aufirst%3DJ.%26aulast%3DFraile-Ribot%26aufirst%3DP.%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DLence%26aufirst%3DE.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DRodr%25C3%25ADguez-Iglesias%26aufirst%3DM.%26aulast%3DGal%25C3%25A1n-S%25C3%25A1nchez%26aufirst%3DF.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DChallenging%2520antimicrobial%2520susceptibility%2520and%2520evolution%2520of%2520resistance%2520%2528OXA-681%2529%2520during%2520treatment%2520of%2520a%2520long-term%2520nosocomial%2520infection%2520caused%2520by%2520a%2520Pseudomonas%2520aeruginosa%2520ST175%2520clone%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D63%26doi%3D10.1128%2Faac.01110-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arca-Suárez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lence, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasarte-Monterrubio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Iglesias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán-Sánchez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span> <span> </span><span class="NLM_article-title">Molecular and biochemical insights into the in vivo evolution of AmpC-mediated resistance to ceftolozane/tazobactam during treatment of an MDR Pseudomonas aeruginosa infection</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3209</span>– <span class="NLM_lpage">3217</span>, <span class="refDoi"> DOI: 10.1093/jac/dkaa291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkaa291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=32728723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVykt7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2020&pages=3209-3217&author=J.+Arca-Su%C3%A1rezauthor=J.+C.+V%C3%A1zquez-Uchaauthor=P.+A.+Fraile-Ribotauthor=E.+Lenceauthor=G.+Cabotauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=C.+Lasarte-Monterrubioauthor=M.+Rodr%C3%ADguez-Iglesiasauthor=A.+Beceiroauthor=C.+Gonz%C3%A1lez-Belloauthor=F.+Gal%C3%A1n-S%C3%A1nchezauthor=A.+Oliverauthor=G.+Bou&title=Molecular+and+biochemical+insights+into+the+in+vivo+evolution+of+AmpC-mediated+resistance+to+ceftolozane%2Ftazobactam+during+treatment+of+an+MDR+Pseudomonas+aeruginosa+infection&doi=10.1093%2Fjac%2Fdkaa291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular and biochemical insights into the in vivo evolution of AmpC-mediated resistance to ceftolozane/tazobactam during treatment of an MDR Pseudomonas aeruginosa infection</span></div><div class="casAuthors">Arca-Suarez, Jorge; Vazquez-Ucha, Juan Carlos; Fraile-Ribot, Pablo Arturo; Lence, Emilio; Cabot, Gabriel; Martinez-Guitian, Marta; Lasarte-Monterrubio, Cristina; Rodriguez-Iglesias, Manuel; Beceiro, Alejandro; Gonzalez-Bello, Concepcion; Galan-Sanchez, Fatima; Oliver, Antonio; Bou, German</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3209-3217</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">1460-2091</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background:Pseudomonas aeruginosa may develop resistance to novel cephalosporin/β-lactamase inhibitor combinations during therapy through the acquisition of structural mutations in AmpC.  Objectives: To describe the mol. and biochem. mechanisms involved in the development of resistance to ceftolozane/tazobactam in vivo through the selection and overprodn. of a novel AmpC variant, designated PDC-315.  Methods: Paired susceptible/resistant isolates obtained before and during ceftolozane/tazobactam treatment were evaluated.  MICs were detd. by broth microdilution.  Mutational changes were investigated through WGS.  Characterization of the novel PDC-315 variant was performed through genotypic and biochem. studies.  The effects at the mol. level of the Asp245Asn change were analyzed by mol. dynamics simulations using Amber.  Results: WGS identified mutations leading to modification (Asp245Asn) and overprodn. of AmpC.  Susceptibility testing revealed that PAOΔC producing PDC-315 displayed increased MICs of ceftolozane/tazobactam, decreased MICs of piperacillin/tazobactam and imipenem and similar susceptibility to ceftazidime/avibactam compared with WT PDCs.  The catalytic efficiency of PDC-315 for ceftolozane was 10-fold higher in relation to the WT PDCs, but 3.5- and 5-fold lower for piperacillin and imipenem.  IC50 values indicated strong inhibition of PDC-315 by avibactam, but resistance to cloxacillin inhibition.  Anal. at the at. level explained that the particular behavior of PDC-315 is linked to conformational changes in the H10 helix that favor the approxn. of key catalytic residues to the active site.  Conclusions: We deciphered the precise mechanisms that led to the in vivo emergence of resistance to ceftolozane/tazobactam in P. aeruginosa through the selection of the novel PDC-315 enzyme.  The characterization of this new variant expands our knowledge about AmpC-mediated resistance to cephalosporin/β-lactamase inhibitors in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxfSopninK6rVg90H21EOLACvtfcHk0ljLAPdkvJwlKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVykt7zE&md5=b1e2992ca7ed9b8176c6fbe2d01a8960</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkaa291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkaa291%26sid%3Dliteratum%253Aachs%26aulast%3DArca-Su%25C3%25A1rez%26aufirst%3DJ.%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DFraile-Ribot%26aufirst%3DP.%2BA.%26aulast%3DLence%26aufirst%3DE.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DLasarte-Monterrubio%26aufirst%3DC.%26aulast%3DRodr%25C3%25ADguez-Iglesias%26aufirst%3DM.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DGal%25C3%25A1n-S%25C3%25A1nchez%26aufirst%3DF.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DBou%26aufirst%3DG.%26atitle%3DMolecular%2520and%2520biochemical%2520insights%2520into%2520the%2520in%2520vivo%2520evolution%2520of%2520AmpC-mediated%2520resistance%2520to%2520ceftolozane%252Ftazobactam%2520during%2520treatment%2520of%2520an%2520MDR%2520Pseudomonas%2520aeruginosa%2520infection%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2020%26volume%3D75%26spage%3D3209%26epage%3D3217%26doi%3D10.1093%2Fjac%2Fdkaa291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Díaz-Cañestro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periañez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulet, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Pena, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayestarán, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colomar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuñez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciá, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asensio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Causapé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delgado, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murillas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Ceftolozane/tazobactam for the treatment of multidrug resistant <i>Pseudomonas aeruginosa</i>: experience from the Balearic Islands</span>. <i>Eur. J. Clin. Microbiol. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2191</span>– <span class="NLM_lpage">2200</span>, <span class="refDoi"> DOI: 10.1007/s10096-018-3361-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1007%2Fs10096-018-3361-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=30141088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BB3c3hsVWrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2018&pages=2191-2200&author=M.+D%C3%ADaz-Ca%C3%B1estroauthor=L.+Peria%C3%B1ezauthor=X.+Muletauthor=M.+L.+Mart%C3%ADn-Penaauthor=P.+A.+Fraile-Ribotauthor=I.+Ayestar%C3%A1nauthor=A.+Colomarauthor=B.+Nu%C3%B1ezauthor=M.+Maci%C3%A1author=A.+Novoauthor=V.+Torresauthor=J.+Asensioauthor=C.+L%C3%B3pez-Causap%C3%A9author=O.+Delgadoauthor=J.+L.+P%C3%A9rezauthor=J.+Murillasauthor=M.+Rieraauthor=A.+Oliver&title=Ceftolozane%2Ftazobactam+for+the+treatment+of+multidrug+resistant+Pseudomonas+aeruginosa%3A+experience+from+the+Balearic+Islands&doi=10.1007%2Fs10096-018-3361-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Ceftolozane/tazobactam for the treatment of multidrug resistant Pseudomonas aeruginosa: experience from the Balearic Islands</span></div><div class="casAuthors">Diaz-Canestro Manuel; Martin-Pena M Luisa; Asensio Javier; Murillas Javier; Riera Melchor; Perianez Leonor; Delgado Olga; Mulet Xavier; Fraile-Ribot Pablo A; Lopez-Causape Carla; Perez Jose Luis; Oliver Antonio; Ayestaran Ignacio; Colomar Asuncion; Nunez Belen; Macia Maria; Novo Andres; Torres Vicente</div><div class="citationInfo"><span class="NLM_cas:title">European journal of clinical microbiology & infectious diseases : official publication of the European Society of Clinical Microbiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2191-2200</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A prospective, descriptive observational study of consecutive patients treated with ceftolozane/tazobactam in the reference hospital of the Balearic Islands (Spain), between May 2016 and September 2017, was performed.  Demographic, clinical, and microbiological variables were recorded.  The later included resistance profile, molecular typing, and whole genome sequencing of isolates showing resistance development.  Fifty-eight patients were treated with ceftolozane/tazobactam.  Thirty-five (60.3%) showed respiratory tract infections, 21 (36.2%) received monotherapy, and 37 (63.8%) combined therapy for ≥ 72 h, mainly with colistin (45.9%).  In 46.6% of the patients, a dose of 1/0.5 g/8 h was used, whereas 2/1 g/8 h was used in 41.4%.  In 56 of the cases (96.6%), the initial Pseudomonas aeruginosa isolates recovered showed a multidrug resistant (MDR) phenotype, and 50 of them (86.2%) additionally met the extensively drug resistant (XDR) criteria and were only susceptible colistin and/or aminoglycosides (mostly amikacin).  The epidemic high-risk clone ST175 was detected in 50% of the patients.  Clinical cure was documented in 37 patients (63.8%) and resistance development in 8 (13.8%).  Clinical failure was associated with disease severity (SOFA), ventilator-dependent respiratory failure, XDR profile, high-risk clone ST175, negative control culture, and resistance development.  In 6 of the 8 cases, resistance development was caused by structural mutations in AmpC, including some mutations described for the first time in vivo, whereas in the other 2, by mutations in OXA-10 leading to the extended spectrum OXA-14.  Although further clinical experience is still needed, our results suggest that ceftolozane/tazobactam is an attractive option for the treatment of MDR/XDR P. aeruginosa infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzWW3mO0AB-DC850By6mxpfW6udTcc2eY5SMD77ATwk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3hsVWrsw%253D%253D&md5=8fd03f82a71b2d9ffcec8601d2593d95</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10096-018-3361-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10096-018-3361-0%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25ADaz-Ca%25C3%25B1estro%26aufirst%3DM.%26aulast%3DPeria%25C3%25B1ez%26aufirst%3DL.%26aulast%3DMulet%26aufirst%3DX.%26aulast%3DMart%25C3%25ADn-Pena%26aufirst%3DM.%2BL.%26aulast%3DFraile-Ribot%26aufirst%3DP.%2BA.%26aulast%3DAyestar%25C3%25A1n%26aufirst%3DI.%26aulast%3DColomar%26aufirst%3DA.%26aulast%3DNu%25C3%25B1ez%26aufirst%3DB.%26aulast%3DMaci%25C3%25A1%26aufirst%3DM.%26aulast%3DNovo%26aufirst%3DA.%26aulast%3DTorres%26aufirst%3DV.%26aulast%3DAsensio%26aufirst%3DJ.%26aulast%3DL%25C3%25B3pez-Causap%25C3%25A9%26aufirst%3DC.%26aulast%3DDelgado%26aufirst%3DO.%26aulast%3DP%25C3%25A9rez%26aufirst%3DJ.%2BL.%26aulast%3DMurillas%26aufirst%3DJ.%26aulast%3DRiera%26aufirst%3DM.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DCeftolozane%252Ftazobactam%2520for%2520the%2520treatment%2520of%2520multidrug%2520resistant%2520Pseudomonas%2520aeruginosa%253A%2520experience%2520from%2520the%2520Balearic%2520Islands%26jtitle%3DEur.%2520J.%2520Clin.%2520Microbiol.%2520Infect.%2520Dis.%26date%3D2018%26volume%3D37%26spage%3D2191%26epage%3D2200%26doi%3D10.1007%2Fs10096-018-3361-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulet, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periañez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Pena, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Mechanisms leading to <i>in vivo</i> ceftolozane/tazobactam resistance development during the treatment of infections caused by MDR <i>Pseudomonas aeruginosa</i></span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1093/jac/dkx424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkx424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=29149337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlequ7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2018&pages=658-663&author=P.+A.+Fraile-Ribotauthor=G.+Cabotauthor=X.+Muletauthor=L.+Peria%C3%B1ezauthor=M.+L.+Mart%C3%ADn-Penaauthor=C.+Juanauthor=J.+L.+P%C3%A9rezauthor=A.+Oliver&title=Mechanisms+leading+to+in+vivo+ceftolozane%2Ftazobactam+resistance+development+during+the+treatment+of+infections+caused+by+MDR+Pseudomonas+aeruginosa&doi=10.1093%2Fjac%2Fdkx424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms leading to in vivo ceftolozane/tazobactam resistance development during the treatment of infections caused by MDR Pseudomonas aeruginosa</span></div><div class="casAuthors">Fraile-Ribot, Pablo A.; Cabot, Gabriel; Mulet, Xavier; Perianez, Leonor; Martyn-Pena, M. Luisa; Juan, Carlos; Perez, Jose L.; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">658-663</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">1460-2091</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives: Characterization of the mechanisms driving ceftolozane/tazobactam resistance development in 5 of 47 (10.6%) patients treated for MDR Pseudomonas aeruginosa infections in a Spanish hospital.  Methods: Five pairs of ceftolozane/tazobactam-susceptible/resistant P. aeruginosa isolates were studied.  MICs were detd. by broth microdilution, clonal relatedness was assessed by MLST and resistance mechanisms were investigated by phenotypic and genotypic methods, including WGS. ampC variants were cloned to assess their impact on resistance.  Results: In all five cases, the same clone was detected for the susceptible/resistant pairs; the widespread ST175 high-risk clone in four of the cases and ST179 in the remaining case.  Genomic anal. of the four initial ST175 isolates revealed the characteristic OprD mutation (Q142X) responsible for carbapenem resistance and the AmpR mutation (G154R) responsible for AmpC overexpression and b-lactam resistance.  The final isolates had developed ceftolozane/tazobactam and ceftazidime/avibactam resistance, and each addnl. showed amutation in AmpC: E247K in one of the isolates, T96I in two isolates and a deletion of 19 amino acids (G229-E247) in the remaining isolate.  The cloned AmpC variants showed greatly increased ceftolozane/tazobactam and ceftazidime/ avibactam MICs compared withWT AmpC, but, in contrast, yielded lower MICs of imipenem, cefepime and particularly piperacillin/tazobactam.  On the other hand, ceftolozane/tazobactamresistance development in ST179 was shown to be driven by the emergence of the extended-spectrum OXA β-lactamase OXA-14, through the selection of an N146S mutation from OXA-10.  Conclusions: Modification of intrinsic (AmpC) and horizontally acquired β-lactamases appears to be the main mechanism leading to ceftolozane/tazobactamresistance in MDR P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7_xxK6o_cTbVg90H21EOLACvtfcHk0lhijOJWX4NxyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlequ7%252FJ&md5=dd24b1f125503acc222d776d71e67b6d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkx424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkx424%26sid%3Dliteratum%253Aachs%26aulast%3DFraile-Ribot%26aufirst%3DP.%2BA.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DMulet%26aufirst%3DX.%26aulast%3DPeria%25C3%25B1ez%26aufirst%3DL.%26aulast%3DMart%25C3%25ADn-Pena%26aufirst%3DM.%2BL.%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DP%25C3%25A9rez%26aufirst%3DJ.%2BL.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DMechanisms%2520leading%2520to%2520in%2520vivo%2520ceftolozane%252Ftazobactam%2520resistance%2520development%2520during%2520the%2520treatment%2520of%2520infections%2520caused%2520by%2520MDR%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2018%26volume%3D73%26spage%3D658%26epage%3D663%26doi%3D10.1093%2Fjac%2Fdkx424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span> <span> </span><span class="NLM_article-title">Diazabicyclooctanes (DBOs): a potent new class of non-β-lactam β-lactamase inhibitors</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">550</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2011.07.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.mib.2011.07.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=21840248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCntrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=550-555&author=K.+Coleman&title=Diazabicyclooctanes+%28DBOs%29%3A+a+potent+new+class+of+non-%CE%B2-lactam+%CE%B2-lactamase+inhibitors&doi=10.1016%2Fj.mib.2011.07.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Diazabicyclooctanes (DBOs): a potent new class of non-β-lactam β-lactamase inhibitors</span></div><div class="casAuthors">Coleman, Ken</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">550-555</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The β-lactams have been among the most successful classes of antibacterial agents for the past half century.  However, a disturbing increase in resistance to β-lactams has been noted among Gram-neg. bacteria, which is attributable to β-lactamase enzymes not within the spectrum of currently marketed β-lactams or β-lactam/β-lactamase inhibitor combinations.  Diazabicyclooctanes (DBOs) were first investigated as β-lactam mimics in the mid-1990s by chemists at Hoechst Marion Roussel (now part of Sanofi-Aventis) and proved to be a rich source of β-lactamase inhibitors (BLI).  Two members of this novel series of highly potent, broad spectrum BLIs are now in clin. development and their properties are reviewed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpneNkuutLgqrVg90H21EOLACvtfcHk0lhijOJWX4NxyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCntrjN&md5=0aafa5daac59fc737d464d7a7b637b2c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2011.07.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2011.07.026%26sid%3Dliteratum%253Aachs%26aulast%3DColeman%26aufirst%3DK.%26atitle%3DDiazabicyclooctanes%2520%2528DBOs%2529%253A%2520a%2520potent%2520new%2520class%2520of%2520non-%25CE%25B2-lactam%2520%25CE%25B2-lactamase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2011%26volume%3D14%26spage%3D550%26epage%3D555%26doi%3D10.1016%2Fj.mib.2011.07.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, I.</span></span> <span> </span><span class="NLM_article-title">New promising β-lactamase inhibitors for clinical use</span>. <i>Eur. J. Clin. Microbiol. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1303</span>– <span class="NLM_lpage">1308</span>, <span class="refDoi"> DOI: 10.1007/s10096-015-2375-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1007%2Fs10096-015-2375-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25864193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms1Wnurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=1303-1308&author=I.+Olsen&title=New+promising+%CE%B2-lactamase+inhibitors+for+clinical+use&doi=10.1007%2Fs10096-015-2375-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">New promising β-lactamase inhibitors for clinical use</span></div><div class="casAuthors">Olsen, I.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Clinical Microbiology & Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1303-1308</span>CODEN:
                <span class="NLM_cas:coden">EJCDEU</span>;
        ISSN:<span class="NLM_cas:issn">0934-9723</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Clavulanate, sulbactam, and tazobactam have been used extensively for the last 30 years, together with β-lactam antibiotics, to inhibit the effect of β-lactamases.  Although they have been useful as β-lactamase inhibitors in many cases, their effectiveness is restricted to class A β-lactamases.  With the increasing frequency and breadth of β-lactamases now threatening public health throughout the world, we need a much broader spectrum of β-lactamase inhibitors efficient against all classes of β-lactamases.  There are several β-lactamase inhibitors under development, but only a few of them are able to inhibit class D and even fewer class B metallo-β-lactamases (MβLs).  The latter represent a real threat to the latest generations of β-lactam antibiotics, including cephalosporins and carbapenems.  Only two β-lactamase inhibitors are, so far, under clin. evaluation, i.e., avibactam and MK-7655.  The others are years from being clin. available.  Although this has caused cautious optimism, the progress in this field is far too slow.  This is particularly so because none of the substances provided are active against MβLs and because new β-lactamases invariably force their way into our therapeutic armamentarium.  While waiting for new antibiotics and new β-lactamase inhibitors to become available, it is important to carry out accurate clin. and microbiol. diagnosis, perform adequate hygiene, and use antibiotics properly.  This may save lives and reduce resistance resulting from inappropriate antibiotic treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpILI_arNIAs7Vg90H21EOLACvtfcHk0lhijOJWX4NxyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms1Wnurk%253D&md5=c0e94ef7d758c12abd1c847171a6ab89</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs10096-015-2375-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10096-015-2375-0%26sid%3Dliteratum%253Aachs%26aulast%3DOlsen%26aufirst%3DI.%26atitle%3DNew%2520promising%2520%25CE%25B2-lactamase%2520inhibitors%2520for%2520clinical%2520use%26jtitle%3DEur.%2520J.%2520Clin.%2520Microbiol.%2520Infect.%2520Dis.%26date%3D2015%26volume%3D34%26spage%3D1303%26epage%3D1308%26doi%3D10.1007%2Fs10096-015-2375-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullane, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolau, D. P.</span></span> <span> </span><span class="NLM_article-title">Comparative Evaluation of the In Vitro Activities of WCK 5222 (Cefepime-Zidebactam) and Combination Antibiotic Therapies against Carbapenem-Resistant Pseudomonas aeruginosa</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e01669</span> <span class="refDoi"> DOI: 10.1128/aac.01669-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.01669-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31844009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmvVKisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=E.+M.+Mullaneauthor=L.+M.+Averyauthor=D.+P.+Nicolau&title=Comparative+Evaluation+of+the+In+Vitro+Activities+of+WCK+5222+%28Cefepime-Zidebactam%29+and+Combination+Antibiotic+Therapies+against+Carbapenem-Resistant+Pseudomonas+aeruginosa&doi=10.1128%2Faac.01669-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative evaluation of the in vitro activities of WCK 5222 (cefepime-zidebactam) and combination antibiotic therapies against carbapenem-resistant Pseudomonas aeruginosa</span></div><div class="casAuthors">Mullane, Elias M.; Avery, Lindsay M.; Nicolau, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e01669-19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The in vitro activity of WCK 5222 (cefepime-zidebactam) was compared to that of several available combination therapies among 30 clin. carbapenem-resistant Pseudomonas aeruginosa (CRP) strains using gradient diffusion strips.  The combinations included nonsusceptible β-lactams (cefepime, ceftolozane-tazobactam, and meropenem) with amikacin and fosfomycin.  WCK 5222 MICs ranged from 2 to 32 mg/L, and 97% were ≤16 mg/L, while 105/146 (72%) combinations demonstrated inhibition below established susceptibility breakpoints.  WCK 5222 monotherapy may be preferred over the combinations assessed for CRP infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEVwTwdyL4RrVg90H21EOLACvtfcHk0ljn6O37pE0puA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmvVKisbo%253D&md5=2ecf684947c79e40ed110f5ecf3ab4fd</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2Faac.01669-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01669-19%26sid%3Dliteratum%253Aachs%26aulast%3DMullane%26aufirst%3DE.%2BM.%26aulast%3DAvery%26aufirst%3DL.%2BM.%26aulast%3DNicolau%26aufirst%3DD.%2BP.%26atitle%3DComparative%2520Evaluation%2520of%2520the%2520In%2520Vitro%2520Activities%2520of%2520WCK%25205222%2520%2528Cefepime-Zidebactam%2529%2520and%2520Combination%2520Antibiotic%2520Therapies%2520against%2520Carbapenem-Resistant%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.01669-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sader, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhomberg, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamm, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castanheira, M.</span></span> <span> </span><span class="NLM_article-title">WCK 5222 (cefepime/zidebactam) antimicrobial activity tested against Gram-negative organisms producing clinically relevant β-lactamases</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1696</span>– <span class="NLM_lpage">1703</span>, <span class="refDoi"> DOI: 10.1093/jac/dkx050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkx050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=28333332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKisLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2017&pages=1696-1703&author=H.+S.+Saderauthor=P.+R.+Rhombergauthor=R.+K.+Flammauthor=R.+N.+Jonesauthor=M.+Castanheira&title=WCK+5222+%28cefepime%2Fzidebactam%29+antimicrobial+activity+tested+against+Gram-negative+organisms+producing+clinically+relevant+%CE%B2-lactamases&doi=10.1093%2Fjac%2Fdkx050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">WCK 5222 (cefepime/zidebactam) antimicrobial activity tested against Gram-negative organisms producing clinically relevant β-lactamases</span></div><div class="casAuthors">Sader, Helio S.; Rhomberg, Paul R.; Flamm, Robert K.; Jones, Ronald N.; Castanheira, Mariana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1696-1703</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">1460-2091</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Zidebactam is a β-lactam enhancer antibiotic with a dual mechanism of action involving binding to Gram-neg. PBP2 and β-lactamase inhibition.  Cefepime combined with zidebactam (WCK 5222) is under clin. development for treatment of Gram-neg. infections.  To evaluate the in vitro activities of cefepime and zidebactam sep. and combined at 1:1 and 2:1 ratios when tested against Gram-neg. organisms producing the most clin. relevant β-lactamase types.  β-Lactamase-producing (193) and WT (71) isolates were tested for susceptibility by broth microdilution method against cefepime/zidebactam, cefepime and zidebactam.  Cefepime/zidebactam (1:1) was very active against Enterobacteriaceae producing CTX-M-15 (21; MIC50/90 0.25/1 mg/L), SHV (20; MIC50/90 0.12/0.25 mg/L), other ESBLs (20, including GES-18, OXA-1/30 and OXY-, PER-, TEM- and VEB-like; MIC50/90 0.25/1 mg/L), plasmidic AmpC (10; MIC50/90 ≤0.06/≤0.06 mg/L), derepressed AmpC (23; MIC50/90 0.12/0.5 mg/L), KPC (35; MIC50/90 0.25/1 mg/L) and metallo-β-lactamases (MBLs; 20 including VIM, IMP and NDM; MIC50/90 0.5/8 mg/L).  Cefepime/zidebactam (1:1) was also active against Pseudomonas aeruginosa with overexpression of AmpC (21; MIC50/90 4/8 mg/L) and MBLs [12 (VIM and IMP); MIC50/90 4/8 mg/L].  Zidebactam alone exhibited potent in vitro activity against some Enterobacteriaceae and P. aeruginosa, including β-lactamase-producing strains.  Cefepime/zidebactam MIC values were lower than those of each agent tested alone for many β-lactamase-producing strains, indicating synergy.  Cefepime/zidebactam showed moderate activity against OXA-23/24/58-producing Acinetobacter baumannii [MIC50/90 32 mg/L (1:1)].  Cefepime/zidebactam showed potent activities against Enterobacteriaceae and P. aeruginosa producing various clin. relevant β-lactamases, including ESBLs, KPCs, AmpC and MBLs for which limited treatment options are currently available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrammLVMFaLrbVg90H21EOLACvtfcHk0ljn6O37pE0puA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKisLrM&md5=23ae84246b3accbee9d450ed463a6803</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkx050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkx050%26sid%3Dliteratum%253Aachs%26aulast%3DSader%26aufirst%3DH.%2BS.%26aulast%3DRhomberg%26aufirst%3DP.%2BR.%26aulast%3DFlamm%26aufirst%3DR.%2BK.%26aulast%3DJones%26aufirst%3DR.%2BN.%26aulast%3DCastanheira%26aufirst%3DM.%26atitle%3DWCK%25205222%2520%2528cefepime%252Fzidebactam%2529%2520antimicrobial%2520activity%2520tested%2520against%2520Gram-negative%2520organisms%2520producing%2520clinically%2520relevant%2520%25CE%25B2-lactamases%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2017%26volume%3D72%26spage%3D1696%26epage%3D1703%26doi%3D10.1093%2Fjac%2Fdkx050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsivkovski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span> <span> </span><span class="NLM_article-title">Biochemical activity of vaborbactam</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e01935</span> <span class="refDoi"> DOI: 10.1128/aac.01935-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.01935-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31712199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktFyhtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=R.+Tsivkovskiauthor=O.+Lomovskaya&title=Biochemical+activity+of+vaborbactam&doi=10.1128%2Faac.01935-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical activity of vaborbactam</span></div><div class="casAuthors">Tsivkovski, Ruslan; Lomovskaya, Olga</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e01935</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The most common mechanism of resistance to β-lactams antibiotics in Gram-neg. bacteria is prodn. of β-lactamase enzymes capable of cleaving the β-lactam ring.  Inhibition of β-lactamase activity with small-mol. drugs is a proven strategy to restore the potency of many β-lactam antibiotics.  Vaborbactam (formerly RPX7009) is a cyclic boronic acid β-lactamase inhibitor (BLI) with a broad spectrum of activity against various serine β-lactamases, including KPC carbapenemases.  The combination of vaborbactam and meropenem is approved in the United States and Europe for the treatment of various nosocomial infections.  We attempted to gain more insight into the mechanism of action of vaborbactam by conducting detailed kinetic characterization of its interaction with various recombinant His-tagged β-lactamases.  Vaborbactam demonstrated potent inhibition of class A and class C enzymes with Ki values ranging from 0.022 to 0.18 μM, while inhibition of class D enzymes was rather poor, and no activity against class B β-lactamases was detected.  Importantly, vaborbactam inhibited KPC-2, KPC-3, BKC-1, and SME-2 carbapenemases at 1:1 stoichiometry, while these nos. were higher for other class A and C enzymes.  Vaborbactam was also shown to be a potent progressive inactivator of several enzymes, including KPCs with inactivation consts. k2/K in the range of 3.4 × 103 to 2.4 × 104 M-1 s-1.  Finally, expts. on the recovery of enzyme activity demonstrated the high stability of the vaborbactam-KPC complex, with 0.000040 s-1koff values and a corresponding residence time of 7 h, whereas the release of vaborbactam bound to other serine β-lactamases was substantially faster.  The biochem. characteristics of vaborbactam described in this study may facilitate further chem. optimization efforts to develop boronic BLIs with improved affinity and broader spectrum of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqct6STwjYA67Vg90H21EOLACvtfcHk0ljn6O37pE0puA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktFyhtbg%253D&md5=53e588b395cfb868f0a5f8699427c05a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2Faac.01935-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01935-19%26sid%3Dliteratum%253Aachs%26aulast%3DTsivkovski%26aufirst%3DR.%26aulast%3DLomovskaya%26aufirst%3DO.%26atitle%3DBiochemical%2520activity%2520of%2520vaborbactam%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.01935-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trout, R. E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, G.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGarry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamrick, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daigle, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Luca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benvenuti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Docquier, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pevear, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xerri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Taniborbactam (VNRX-5133): A Broad-Spectrum Serine- and Metallo-β-lactamase Inhibitor for Carbapenem-Resistant Bacterial Infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">2789</span>– <span class="NLM_lpage">2801</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01518</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01518" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1GmsbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=2789-2801&author=B.+Liuauthor=R.+E.+L.+Troutauthor=G.-H.+Chuauthor=D.+McGarryauthor=R.+W.+Jacksonauthor=J.+C.+Hamrickauthor=D.+M.+Daigleauthor=S.+M.+Cusickauthor=C.+Pozziauthor=F.+De+Lucaauthor=M.+Benvenutiauthor=S.+Manganiauthor=J.-D.+Docquierauthor=W.+J.+Weissauthor=D.+C.+Pevearauthor=L.+Xerriauthor=C.+J.+Burns&title=Discovery+of+Taniborbactam+%28VNRX-5133%29%3A+A+Broad-Spectrum+Serine-+and+Metallo-%CE%B2-lactamase+Inhibitor+for+Carbapenem-Resistant+Bacterial+Infections&doi=10.1021%2Facs.jmedchem.9b01518"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Taniborbactam (VNRX-5133): A Broad-Spectrum Serine- and Metallo-β-lactamase Inhibitor for Carbapenem-Resistant Bacterial Infections</span></div><div class="casAuthors">Liu, Bin; Trout, Robert E. Lee; Chu, Guo-Hua; McGarry, Daniel; Jackson, Randy W.; Hamrick, Jodie C.; Daigle, Denis M.; Cusick, Susan M.; Pozzi, Cecilia; De Luca, Filomena; Benvenuti, Manuela; Mangani, Stefano; Docquier, Jean-Denis; Weiss, William J.; Pevear, Daniel C.; Xerri, Luigi; Burns, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2789-2801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A major resistance mechanism in Gram-neg. bacteria is the prodn. of β-lactamase enzymes.  Originally recognized for their ability to hydrolyze penicillins, emergent β-lactamases can now confer resistance to other β-lactam drugs, including both cephalosporins and carbapenems.  The emergence and global spread of β-lactamase-producing multidrug-resistant superbugs has caused increased alarm within the medical community due to the high mortality rate assocd. with these difficult-to-treat bacterial infections.  To address this unmet medical need, we initiated an iterative program combining medicinal chem., structural biol., biochem. testing, and microbiol. profiling to identify broad-spectrum inhibitors of both serine- and metallo-β-lactamase enzymes.  Lead optimization, beginning with narrower-spectrum, weakly active compds. provided VNRX-5133 (taniborbactam I), a boronic acid-contg. pan-spectrum β-lactamase inhibitor.  In vitro and in vivo studies demonstrated that VNRX-5133 restored the activity of β-lactam antibiotics against carbapenem-resistant Pseudomonas aeruginosa and carbapenem-resistant Enterobacteriaceae.  Taniborbactam is the 1st pan-spectrum β-lactamase inhibitor to enter clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn-mAQzqladbVg90H21EOLACvtfcHk0lgtyg72hMJfSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1GmsbjI&md5=78af5dfe748ba2480fb7165d62e9b271</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01518%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DTrout%26aufirst%3DR.%2BE.%2BL.%26aulast%3DChu%26aufirst%3DG.-H.%26aulast%3DMcGarry%26aufirst%3DD.%26aulast%3DJackson%26aufirst%3DR.%2BW.%26aulast%3DHamrick%26aufirst%3DJ.%2BC.%26aulast%3DDaigle%26aufirst%3DD.%2BM.%26aulast%3DCusick%26aufirst%3DS.%2BM.%26aulast%3DPozzi%26aufirst%3DC.%26aulast%3DDe%2BLuca%26aufirst%3DF.%26aulast%3DBenvenuti%26aufirst%3DM.%26aulast%3DMangani%26aufirst%3DS.%26aulast%3DDocquier%26aufirst%3DJ.-D.%26aulast%3DWeiss%26aufirst%3DW.%2BJ.%26aulast%3DPevear%26aufirst%3DD.%2BC.%26aulast%3DXerri%26aufirst%3DL.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26atitle%3DDiscovery%2520of%2520Taniborbactam%2520%2528VNRX-5133%2529%253A%2520A%2520Broad-Spectrum%2520Serine-%2520and%2520Metallo-%25CE%25B2-lactamase%2520Inhibitor%2520for%2520Carbapenem-Resistant%2520Bacterial%2520Infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D2789%26epage%3D2801%26doi%3D10.1021%2Facs.jmedchem.9b01518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamrick, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Docquier, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uehara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Six, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatwin, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernacchio, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trout, R. E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Luca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benvenuti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moeck, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xerri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pevear, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daigle, D. M.</span></span> <span> </span><span class="NLM_article-title">VNRX-5133 (Taniborbactam), a Broad-Spectrum Inhibitor of Serine- and Metallo-β-Lactamases, Restores Activity of Cefepime in Enterobacterales and Pseudomonas aeruginosa</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e01963</span> <span class="refDoi"> DOI: 10.1128/aac.01963-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.01963-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31871094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntlGisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=J.+C.+Hamrickauthor=J.-D.+Docquierauthor=T.+Ueharaauthor=C.+L.+Myersauthor=D.+A.+Sixauthor=C.+L.+Chatwinauthor=K.+J.+Johnauthor=S.+F.+Vernacchioauthor=S.+M.+Cusickauthor=R.+E.+L.+Troutauthor=C.+Pozziauthor=F.+De+Lucaauthor=M.+Benvenutiauthor=S.+Manganiauthor=B.+Liuauthor=R.+W.+Jacksonauthor=G.+Moeckauthor=L.+Xerriauthor=C.+J.+Burnsauthor=D.+C.+Pevearauthor=D.+M.+Daigle&title=VNRX-5133+%28Taniborbactam%29%2C+a+Broad-Spectrum+Inhibitor+of+Serine-+and+Metallo-%CE%B2-Lactamases%2C+Restores+Activity+of+Cefepime+in+Enterobacterales+and+Pseudomonas+aeruginosa&doi=10.1128%2Faac.01963-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">VNRX-5133 (Taniborbactam), a broad-spectrum inhibitor of serine- and metallo-β-lactamases, restores activity of cefepime in Enterobacterales and Pseudomonas aeruginosa</span></div><div class="casAuthors">Hamrick, Jodie C.; Docquier, Jean-Denis; Uehara, Tsuyoshi; Myers, Cullen L.; Six, David A.; Chatwin, Cassandra L.; John, Kaitlyn J.; Vernacchio, Salvador F.; Cusick, Susan M.; Trout, Robert E. L.; Pozzi, Cecilia; De Luca, Filomena; Benvenuti, Manuela; Mangani, Stefano; Liu, Bin; Jackson, Randy W.; Moeck, Greg; Xerri, Luigi; Burns, Christopher J.; Pevear, Daniel C.; Daigle, Denis M.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e01963-19/1-e01963-19/19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">As shifts in the epidemiol. of β-lactamase-mediated resistance continue, carbapenem-resistant Enterobacterales (CRE) and carbapenem-resistant Pseudomonas aeruginosa (CRPA) are the most urgent threats.  Although approved β-lactam (BL)-β-lactamase inhibitor (BLI) combinations address widespread serine β-lactamases (SBLs), such as CTX-M-15, none provide broad coverage against either clin. important serine-β-lactamases (KPC, OXA-48) or clin. important metallo-β-lactamases (MBLs; e.g., NDM-1).  VNRX-5133 (taniborbactam) is a new cyclic boronate BLI that is in clin. development combined with cefepime for the treatment of infections caused by β-lactamase-producing CRE and CRPA.  Taniborbactam is the first BLI with direct inhibitory activity against Ambler class A, B, C, and D enzymes.  From biochem. and structural analyses, taniborbactam exploits substrate mimicry while employing distinct mechanisms to inhibit both SBLs and MBLs.  It is a reversible covalent inhibitor of SBLs with slow dissocn. and a prolonged active-site residence time (half-life, 30 to 105 min), while in MBLs, it behaves as a competitive inhibitor, with inhibitor const. (Ki) values ranging from 0.019 to 0.081 μM.  Inhibition is achieved by mimicking the transition state structure and exploiting interactions with highly conserved active-site residues.  In microbiol. testing, taniborbactam restored cefepime activity in 33/34 engineered Escherichia coli strains overproducing individual enzymes covering Ambler classes A, B, C, and D, providing up to a 1,024-fold shift in the MIC.  Addn. of taniborbactam restored the antibacterial activity of cefepime against all 102 Enterobacterales clin. isolates tested and 38/41 P. aeruginosa clin. isolates tested with MIC90s of 1 and 4 μg/mL, resp., representing ≥256- and ≥32-fold improvements, resp., in antibacterial activity over that of cefepime alone.  The data demonstrate the potent, broad-spectrum rescue of cefepime activity by taniborbactam against clin. isolates of CRE and CRPA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzz4T2vRQnKLVg90H21EOLACvtfcHk0lgtyg72hMJfSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntlGisrg%253D&md5=1387fed36a9998aa80ec4ed287d6fd4a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2Faac.01963-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01963-19%26sid%3Dliteratum%253Aachs%26aulast%3DHamrick%26aufirst%3DJ.%2BC.%26aulast%3DDocquier%26aufirst%3DJ.-D.%26aulast%3DUehara%26aufirst%3DT.%26aulast%3DMyers%26aufirst%3DC.%2BL.%26aulast%3DSix%26aufirst%3DD.%2BA.%26aulast%3DChatwin%26aufirst%3DC.%2BL.%26aulast%3DJohn%26aufirst%3DK.%2BJ.%26aulast%3DVernacchio%26aufirst%3DS.%2BF.%26aulast%3DCusick%26aufirst%3DS.%2BM.%26aulast%3DTrout%26aufirst%3DR.%2BE.%2BL.%26aulast%3DPozzi%26aufirst%3DC.%26aulast%3DDe%2BLuca%26aufirst%3DF.%26aulast%3DBenvenuti%26aufirst%3DM.%26aulast%3DMangani%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DJackson%26aufirst%3DR.%2BW.%26aulast%3DMoeck%26aufirst%3DG.%26aulast%3DXerri%26aufirst%3DL.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26aulast%3DPevear%26aufirst%3DD.%2BC.%26aulast%3DDaigle%26aufirst%3DD.%2BM.%26atitle%3DVNRX-5133%2520%2528Taniborbactam%2529%252C%2520a%2520Broad-Spectrum%2520Inhibitor%2520of%2520Serine-%2520and%2520Metallo-%25CE%25B2-Lactamases%252C%2520Restores%2520Activity%2520of%2520Cefepime%2520in%2520Enterobacterales%2520and%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.01963-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caillon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajaksouzian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reghal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taracila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacqueline, C.</span></span> <span> </span><span class="NLM_article-title">Beyond Piperacillin-Tazobactam: Cefepime and AAI101 as a Potent β-Lactam−β-Lactamase Inhibitor Combination</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>63</i></span>, <span class="NLM_elocation-id">e00105</span> <span class="refDoi"> DOI: 10.1128/aac.00105-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.00105-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=30858223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeiur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2019&author=K.+M.+Papp-Wallaceauthor=C.+R.+Bethelauthor=J.+Caillonauthor=M.+D.+Barnesauthor=G.+Potelauthor=S.+Bajaksouzianauthor=J.+D.+Rutterauthor=A.+Reghalauthor=S.+Shapiroauthor=M.+A.+Taracilaauthor=M.+R.+Jacobsauthor=R.+A.+Bonomoauthor=C.+Jacqueline&title=Beyond+Piperacillin-Tazobactam%3A+Cefepime+and+AAI101+as+a+Potent+%CE%B2-Lactam%E2%88%92%CE%B2-Lactamase+Inhibitor+Combination&doi=10.1128%2Faac.00105-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond piperacillin-tazobactam: cefepime and AAI101 as a potent β-Lactam-β-lactamase inhibitor combination</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.; Bethel, Christopher R.; Caillon, Jocelyne; Barnes, Melissa D.; Potel, Gilles; Bajaksouzian, Saralee; Rutter, Joseph D.; Reghal, Amokrane; Shapiro, Stuart; Taracila, Magdalena A.; Jacobs, Michael R.; Bonomo, Robert A.; Jacqueline, Cedric</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e00105-19/1-e00105-19/17</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Impeding, as well as reducing, the burden of antimicrobial resistance in Gram-neg. pathogens is an urgent public health endeavor.  Our current antibiotic armamentarium is dwindling, while major resistance determinants (e.g., extendedspectrum β-lactamases [ESBLs]) continue to evolve and disseminate around the world.  One approach to attack this problem is to develop novel therapies that will protect our current agents.  AAI101 is a novel penicillanic acid sulfone β-lactamase inhibitor similar in structure to tazobactam, with one important difference.  AAI101 possesses a strategically placed Me group that gives the inhibitor a net neutral charge, enhancing bacterial cell penetration.  AAI101 paired with cefepime, also a zwitterion, is in phase III of clin. development for the treatment of serious Gram-neg. infections.  Here, AAI101 was found to restore the activity of cefepime against class A ESBLs (e.g., CTX-M-15) and demonstrated increased potency compared to that of piperacillin-tazobactam when tested against an established isogenic panel.  The enzymol. properties of AAI101 further revealed that AAI101 possessed a unique mechanism of β-lactamase inhibition compared to that of tazobactam.  Addnl., upon reaction with AAI101, CTX-M-15 was modified to an inactive state.  Notably, the in vivo efficacy of cefepime-AAI101 was demonstrated using a mouse septicemia model, indicating the ability of AAI101 to bolster significantly the therapeutic efficacy of cefepime in vivo.  The combination of AAI101 with cefepime represents a potential carbapenem-sparing treatment regimen for infections suspected to be caused by Enterobacteriaceae expressing ESBLs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0wGvlaUiEsLVg90H21EOLACvtfcHk0lgtyg72hMJfSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeiur%252FE&md5=55a99d1c91a4f138479ad35e507f1c99</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FAAC.00105-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00105-19%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DCaillon%26aufirst%3DJ.%26aulast%3DBarnes%26aufirst%3DM.%2BD.%26aulast%3DPotel%26aufirst%3DG.%26aulast%3DBajaksouzian%26aufirst%3DS.%26aulast%3DRutter%26aufirst%3DJ.%2BD.%26aulast%3DReghal%26aufirst%3DA.%26aulast%3DShapiro%26aufirst%3DS.%26aulast%3DTaracila%26aufirst%3DM.%2BA.%26aulast%3DJacobs%26aufirst%3DM.%2BR.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DJacqueline%26aufirst%3DC.%26atitle%3DBeyond%2520Piperacillin-Tazobactam%253A%2520Cefepime%2520and%2520AAI101%2520as%2520a%2520Potent%2520%25CE%25B2-Lactam%25E2%2588%2592%25CE%25B2-Lactamase%2520Inhibitor%2520Combination%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D63%26doi%3D10.1128%2Faac.00105-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pernas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colchón, E.</span></span> <span> </span><span class="NLM_article-title">β-Lactamase Inhibitors To Restore the Efficacy of Antibiotics against Superbugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1859</span>– <span class="NLM_lpage">1881</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVOhsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=1859-1881&author=C.+Gonz%C3%A1lez-Belloauthor=D.+Rodr%C3%ADguezauthor=M.+Pernasauthor=%C3%81.+Rodr%C3%ADguezauthor=E.+Colch%C3%B3n&title=%CE%B2-Lactamase+Inhibitors+To+Restore+the+Efficacy+of+Antibiotics+against+Superbugs&doi=10.1021%2Facs.jmedchem.9b01279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">β-Lactamase Inhibitors To Restore the Efficacy of Antibiotics against Superbugs</span></div><div class="casAuthors">Gonzalez-Bello, Concepcion; Rodriguez, Diana; Pernas, Marina; Rodriguez, Angela; Colchon, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1859-1881</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Infections caused by resistant bacteria are nowadays too common and some pathogens have even become resistant to multiple types of antibiotics, in which cases few or even no treatments are available.  In recent years, the most successful strategy in anti-infective drug discovery for the treatment of such problematic infections is the combination therapy 'antibiotic + inhibitor of resistance'.  These inhibitors allow the repurposing of antibiotics that have already proven to be safe and effective for clin. use.  Three main types of compds. have been developed to block the principal bacterial resistance mechanisms: (i) β-lactamase inhibitors; (ii) outer membrane permeabilizers; and (iii) efflux pump inhibitors.  This perspective article is focused on β-lactamase inhibitors that disable the most prevalent cause of antibiotic resistance in Gram-neg. bacteria, i.e., the deactivation of the most widely used antibiotics, β-lactams (penicillins, cephalosporines, carbapenems and monobactams), by the prodn. of β-lactamases.  An overview of the most recently identified β-lactamase inhibitors and of combination therapy is provided.  The article also covers the mechanism of action of the different types of β-lactamase enzymes as a basis for inhibitor design and target inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs6jFuMiB-xLVg90H21EOLACvtfcHk0liDJhClvgYYfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVOhsb3J&md5=336d7eb65305b05d909584c6ae3af6a8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01279%26sid%3Dliteratum%253Aachs%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DPernas%26aufirst%3DM.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3D%25C3%2581.%26aulast%3DColch%25C3%25B3n%26aufirst%3DE.%26atitle%3D%25CE%25B2-Lactamase%2520Inhibitors%2520To%2520Restore%2520the%2520Efficacy%2520of%2520Antibiotics%2520against%2520Superbugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D1859%26epage%3D1881%26doi%3D10.1021%2Facs.jmedchem.9b01279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vallejo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span> <span> </span><span class="NLM_article-title">LN-1-255, a penicillanic acid sulfone able to inhibit the class D carbapenemase OXA-48</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">2171</span>– <span class="NLM_lpage">2180</span>, <span class="refDoi"> DOI: 10.1093/jac/dkw105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkw105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=27125555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BC28bktlGguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2016&pages=2171-2180&author=J.+A.+Vallejoauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=J.+C.+V%C3%A1zquez-Uchaauthor=C.+Gonz%C3%A1lez-Belloauthor=M.+Pozaauthor=J.+D.+Buynakauthor=C.+R.+Bethelauthor=R.+A.+Bonomoauthor=G.+Bouauthor=A.+Beceiro&title=LN-1-255%2C+a+penicillanic+acid+sulfone+able+to+inhibit+the+class+D+carbapenemase+OXA-48&doi=10.1093%2Fjac%2Fdkw105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">LN-1-255, a penicillanic acid sulfone able to inhibit the class D carbapenemase OXA-48</span></div><div class="casAuthors">Vallejo Juan A; Martinez-Guitian Marta; Vazquez-Ucha Juan C; Poza Margarita; Bou German; Beceiro Alejandro; Gonzalez-Bello Concepcion; Buynak John D; Bethel Christopher R; Bonomo Robert A</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antimicrobial chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2171-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Carbapenemases are the most important mechanism responsible for carbapenem resistance in Enterobacteriaceae.  Among carbapenemases, OXA-48 presents unique challenges as it is resistant to β-lactam inhibitors.  Here, we test the capacity of the compound LN-1-255, a 6-alkylidene-2'-substituted penicillanic acid sulfone, to inhibit the activity of the carbapenemase OXA-48.  METHODS:  The OXA-48 gene was cloned and expressed in Klebsiella pneumoniae and Escherichia coli in order to obtain MICs in the presence of inhibitors (clavulanic acid, tazobactam and sulbactam) and LN-1-255.  OXA-48 was purified and steady-state kinetics was performed with LN-1-255 and tazobactam.  The covalent binding mode of LN-1-255 with OXA-48 was studied by docking assays.  RESULTS:  Both OXA-48-producing clinical and transformant strains displayed increased susceptibility to carbapenem antibiotics in the presence of 4 mg/L LN-1-255 (2-32-fold increased susceptibility) and 16 mg/L LN-1-255 (4-64-fold increased susceptibility).  Kinetic assays demonstrated that LN-1-255 is able to inhibit OXA-48 with an acylation efficiency (k2/K) of 10 ± 1 × 10(4) M(-1) s(-1) and a slow deacylation rate (koff) of 7 ± 1 × 10(-4) s(-1).  IC50 was 3 nM for LN-1-255 and 1.5 μM for tazobactam.  Lastly, kcat/kinact was 500-fold lower for LN-1-255 than for tazobactam.  CONCLUSIONS:  In these studies, carbapenem antibiotics used in combination with LN-1-255 are effective against the carbapenemase OXA-48, an important emerging mechanism of antibiotic resistance.  This provides an incentive for further investigations to maximize the efficacy of penicillin sulfone inhibition of class D plasmid-carried Enterobacteriaceae carbapenemases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT3yVfbQnR8dEbxh97K6Vt6fW6udTcc2eareexuHSexi7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bktlGguw%253D%253D&md5=7328b8a9d881b23e54ba861a35072802</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkw105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkw105%26sid%3Dliteratum%253Aachs%26aulast%3DVallejo%26aufirst%3DJ.%2BA.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DPoza%26aufirst%3DM.%26aulast%3DBuynak%26aufirst%3DJ.%2BD.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DBeceiro%26aufirst%3DA.%26atitle%3DLN-1-255%252C%2520a%2520penicillanic%2520acid%2520sulfone%2520able%2520to%2520inhibit%2520the%2520class%2520D%2520carbapenemase%2520OXA-48%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2016%26volume%3D71%26spage%3D2171%26epage%3D2180%26doi%3D10.1093%2Fjac%2Fdkw105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span> <span> </span><span class="NLM_article-title">Activity of the β-Lactamase Inhibitor LN-1-255 against Carbapenem-Hydrolyzing Class D β-Lactamases from Acinetobacter baumannii</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e01172</span> <span class="refDoi"> DOI: 10.1128/aac.01172-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01172-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=28807908" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=J.+C.+V%C3%A1zquez-Uchaauthor=M.+Maneiroauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=J.+Buynakauthor=C.+R.+Bethelauthor=R.+A.+Bonomoauthor=G.+Bouauthor=M.+Pozaauthor=C.+Gonz%C3%A1lez-Belloauthor=A.+Beceiro&title=Activity+of+the+%CE%B2-Lactamase+Inhibitor+LN-1-255+against+Carbapenem-Hydrolyzing+Class+D+%CE%B2-Lactamases+from+Acinetobacter+baumannii&doi=10.1128%2Faac.01172-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1128%2FAAC.01172-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01172-17%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DBuynak%26aufirst%3DJ.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DPoza%26aufirst%3DM.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DBeceiro%26aufirst%3DA.%26atitle%3DActivity%2520of%2520the%2520%25CE%25B2-Lactamase%2520Inhibitor%2520LN-1-255%2520against%2520Carbapenem-Hydrolyzing%2520Class%2520D%2520%25CE%25B2-Lactamases%2520from%2520Acinetobacter%2520baumannii%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2Faac.01172-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Guitián, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conde-Pérez, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Álvarez-Fraga, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span> <span> </span><span class="NLM_article-title">Therapeutic efficacy of LN-1-255 in combination with imipenem in severe infection caused by carbapenem-resistant <i>Acinetobacter baumannii</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>63</i></span>, <span class="NLM_elocation-id">e01092</span> <span class="refDoi"> DOI: 10.1128/aac.01092-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01092-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31383666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1eisbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2019&author=J.+C.+V%C3%A1zquez-Uchaauthor=M.+Mart%C3%ADnez-Guiti%C3%A1nauthor=M.+Maneiroauthor=K.+Conde-P%C3%A9rezauthor=L.+%C3%81lvarez-Fragaauthor=G.+Torrensauthor=A.+Oliverauthor=J.+D.+Buynakauthor=R.+A.+Bonomoauthor=G.+Bouauthor=C.+Gonz%C3%A1lez-Belloauthor=M.+Pozaauthor=A.+Beceiro&title=Therapeutic+efficacy+of+LN-1-255+in+combination+with+imipenem+in+severe+infection+caused+by+carbapenem-resistant+Acinetobacter+baumannii&doi=10.1128%2Faac.01092-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic efficacy of LN-1-255 in combination with imipenem in severe infection caused by carbapenem-resistant Acinetobacter baumannii</span></div><div class="casAuthors">Vazquez-Ucha, Juan Carlos; Martinez-Guitian, Marta; Maneiro, Maria; Conde-Perez, Kelly; Alvarez-raga, Laura; Torrens, Gabriel; Oliver, Antonio; Buynak, John D.; Bonomo, Robert A.; Bou, German; Gonzalez-Bello, Concepcion; Poza, Margarita; Beceiroa, Alejandro</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e01092-19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The carbapenem-hydrolyzing class D β-lactamases (CHDLs) are the main mechanism of carbapenem resistance in Acinetobacter baumannii.  CHDLs are not effectively inactivated by clin. available β-lactam-type inhibitors.  We have previously described the in vitro efficacy of the inhibitor LN-1-255 in combination with carbapenems.  The aim of this study was to compare the efficacy of LN-1-255 with that of imipenem in murine pneumonia using A. baumannii strains carrying their most extended carbapenemases, OXA-23 and OXA-24/40.  The blaOXA-23 and blaOXA-24/40 genes were cloned into the carbapenem-susceptible A. baumannii ATCC 17978 strain.  Clin. isolates Ab1 and JC12/04, producing the enzymes OXA-23 and OXA-24/40, resp., were used in the study.  Pharmacokinetic (PK) parameters were detd.  An exptl. pneumonia model was used to evaluate the efficacy of the combined imipenem-LN-1-255 therapy.  MICs of imipenem decreased between 32- and 128-fold in the presence of LN-1-255.  I.m. treatment with imipenem-LN-1-255 (30/50 mg/kg) decreased the bacterial burden by (i) 4 and 1.7 log10 CFU/g lung in the infection with the ATCC 17978-OXA-23 and Ab1 strains, resp., and by (ii) 2.5 and 4.5 log10 CFU/g lung in the infection produced by the ATCC 17978-OXA-24/40 and the JC12/04 strains, resp.  In all assays, combined therapy offered higher protection against pneumonia than that provided by monotherapy.  No toxicity was obsd. in treated mice.  Imipenem treatment combined with LN-1-255 treatment significantly reduced the severity of infection by carbapenem-resistant A. baumannii strains carrying CHDLs.  Preclin. assays demonstrated the potential of LN-1-255 and imipenem therapy as a new antibacterial treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtx5V3Y8kjfbVg90H21EOLACvtfcHk0lhfC79vKmgOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1eisbrJ&md5=c53b4095cd6a9de4639d125e2e07b333</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1128%2FAAC.01092-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01092-19%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DMart%25C3%25ADnez-Guiti%25C3%25A1n%26aufirst%3DM.%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DConde-P%25C3%25A9rez%26aufirst%3DK.%26aulast%3D%25C3%2581lvarez-Fraga%26aufirst%3DL.%26aulast%3DTorrens%26aufirst%3DG.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DBuynak%26aufirst%3DJ.%2BD.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26aulast%3DPoza%26aufirst%3DM.%26aulast%3DBeceiro%26aufirst%3DA.%26atitle%3DTherapeutic%2520efficacy%2520of%2520LN-1-255%2520in%2520combination%2520with%2520imipenem%2520in%2520severe%2520infection%2520caused%2520by%2520carbapenem-resistant%2520Acinetobacter%2520baumannii%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D63%26doi%3D10.1128%2Faac.01092-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santillana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drawz, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagadala, S. R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Akker, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J. D.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Crystal Structures of 6-Alkylidene-2′-Substituted Penicillanic Acid Sulfones as Potent Inhibitors ofAcinetobacter baumanniiOXA-24 Carbapenemase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">13320</span>– <span class="NLM_lpage">13331</span>, <span class="refDoi"> DOI: 10.1021/ja104092z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja104092z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFWru7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=13320-13331&author=G.+Bouauthor=E.+Santillanaauthor=A.+Sheriauthor=A.+Beceiroauthor=J.+M.+Sampsonauthor=M.+Kalpauthor=C.+R.+Bethelauthor=A.+M.+Distlerauthor=S.+M.+Drawzauthor=S.+R.+R.+Pagadalaauthor=F.+van+den+Akkerauthor=R.+A.+Bonomoauthor=A.+Romeroauthor=J.+D.+Buynak&title=Design%2C+Synthesis%2C+and+Crystal+Structures+of+6-Alkylidene-2%E2%80%B2-Substituted+Penicillanic+Acid+Sulfones+as+Potent+Inhibitors+ofAcinetobacter+baumanniiOXA-24+Carbapenemase&doi=10.1021%2Fja104092z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Crystal Structures of 6-Alkylidene-2'-Substituted Penicillanic Acid Sulfones as Potent Inhibitors of Acinetobacter baumannii OXA-24 Carbapenemase</span></div><div class="casAuthors">Bou, German; Santillana, Elena; Sheri, Anjaneyulu; Beceiro, Alejandro; Sampson, Jared M.; Kalp, Matthew; Bethel, Christopher R.; Distler, Anne M.; Drawz, Sarah M.; Pagadala, Sundar Ram Reddy; van den Akker, Focco; Bonomo, Robert A.; Romero, Antonio; Buynak, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">13320-13331</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Class D β-lactamases represent a growing and diverse class of penicillin-inactivating enzymes that are usually resistant to com. β-lactamase inhibitors.  As many such enzymes are found in multi-drug resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa, novel β-lactamase inhibitors are urgently needed.  Five unique 6-alkylidene-2'-substituted penicillanic acid sulfones I (R = H, R1 = CH2OCOCH2C6H3(OH)2-3,4, X = Na) (II), I (R = NHCO2Me, R1 = Me, X = Na; R = CONH2, R1 = CH2OCONH2, X = Na), I (R = H, R1 = CH2OCONHCH2R2, X = Na, R2 = 2-amino-thiazol-4-yl) (III), I (R = H, R1 = CH2OCONHCH2CH2NH3+, X = none) (IV) were synthesized and tested against OXA-24, a clin. important β-lactamase that inactivates carbapenems and is found in A. baumannii.  Based upon the roles Tyr112 and Met223 play in the OXA-24 β-lactamase, we also engineered two variants (Tyr112Ala and Tyr112Ala,Met223Ala) to test the hypothesis that the hydrophobic tunnel formed by these residues influences inhibitor recognition.  IC50 values against OXA-24 and two OXA-24 β-lactamase variants ranged from 10 ± 1 (III vs. WT) to 338 ± 20 nM (IV vs. Tyr112Ala, Met223Ala).  Compd. III possessed the lowest Ki (500 ± 80 nM vs. WT), and II possessed the highest inactivation efficiency (kinact/Ki = 0.21 ± 0.02 μM-1 s-1).  Electrospray ionization mass spectrometry revealed a single covalent adduct, suggesting the formation of an acyl-enzyme intermediate.  X-ray structures of OXA-24 complexed to four inhibitors (2.0-2.6 Å) reveal the formation of stable bicyclic arom. intermediates with their carbonyl oxygen in the oxyanion hole.  These data provide the first structural evidence that 6-alkylidene-2'-substituted penicillin sulfones are effective mechanism-based inactivators of class D β-lactamases.  Their unique chem. makes them developmental candidates.  Mechanisms for class D hydrolysis and inhibition are discussed, and a pathway for the evolution of the BlaR1 sensor of Staphylococcus aureus to the class D β-lactamases is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ0f6jmKDmVrVg90H21EOLACvtfcHk0lhfC79vKmgOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFWru7vO&md5=71fd4b9382c882f6bed9255305c40b24</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja104092z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja104092z%26sid%3Dliteratum%253Aachs%26aulast%3DBou%26aufirst%3DG.%26aulast%3DSantillana%26aufirst%3DE.%26aulast%3DSheri%26aufirst%3DA.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DSampson%26aufirst%3DJ.%2BM.%26aulast%3DKalp%26aufirst%3DM.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DDistler%26aufirst%3DA.%2BM.%26aulast%3DDrawz%26aufirst%3DS.%2BM.%26aulast%3DPagadala%26aufirst%3DS.%2BR.%2BR.%26aulast%3Dvan%2Bden%2BAkker%26aufirst%3DF.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26aulast%3DRomero%26aufirst%3DA.%26aulast%3DBuynak%26aufirst%3DJ.%2BD.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Crystal%2520Structures%2520of%25206-Alkylidene-2%25E2%2580%25B2-Substituted%2520Penicillanic%2520Acid%2520Sulfones%2520as%2520Potent%2520Inhibitors%2520ofAcinetobacter%2520baumanniiOXA-24%2520Carbapenemase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D13320%26epage%3D13331%26doi%3D10.1021%2Fja104092z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Ucha, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span> <span> </span><span class="NLM_article-title">6-Arylmethylidene penicillin-based sulfone inhibitors for repurposing antibiotic efficiency in priority pathogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3737</span>– <span class="NLM_lpage">3755</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00127</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00127" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1CrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3737-3755&author=D.+Rodr%C3%ADguezauthor=M.+Maneiroauthor=J.+C.+V%C3%A1zquez-Uchaauthor=A.+Beceiroauthor=C.+Gonz%C3%A1lez-Bello&title=6-Arylmethylidene+penicillin-based+sulfone+inhibitors+for+repurposing+antibiotic+efficiency+in+priority+pathogens&doi=10.1021%2Facs.jmedchem.0c00127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">6-Arylmethylidene Penicillin-Based Sulfone Inhibitors for Repurposing Antibiotic Efficiency in Priority Pathogens</span></div><div class="casAuthors">Rodriguez, Diana; Maneiro, Maria; Vazquez-Ucha, Juan C.; Beceiro, Alejandro; Gonzalez-Bello, Concepcion</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3737-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability of 6-(aryl)methylidene penicillin-based sulfones 1-7 to repurpose β-lactam antibiotics activity with bacterial species that carry carbapenem-hydrolyzing class D β-lactamases (OXA-23, OXA-24/40 and OXA-48), as well as with class A (TEM-1, CTX-M-2) and class C (CMY-2, DHA-1) enzymes, is reported.  The combinations imipenem/3 and imipenem/4 restored almost completely the antibiotic efficacy in OXA-23 and OXA-24/40 carbapenemase-producing A. baumannii strains (1μg mL-1) and also provided good results for OXA-48 carbapenemase-producing K. pneumoniae strains (4μg mL-1).  Compds. 2-6 in combinations with ceftazidime and ampicillin were also efficient in restoring antibiotic efficacy in E. coli strains carrying class C (CMY-2 and DHA-1) and class A (TEM-1 and CTX-M-2) β-lactamase enzymes, resp.  Kinetic and inhibition studies with the OXA-24/40 enzyme, protein mass spectrometry anal. and docking studies allowed us to gain an insight into the inhibition mechanism and the exptl. obsd. differences between the ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP87HcFlDbT7Vg90H21EOLACvtfcHk0lhfC79vKmgOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1CrtLg%253D&md5=1d6723e5b18ce3850d77c2b9b226913e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00127%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DV%25C3%25A1zquez-Ucha%26aufirst%3DJ.%2BC.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26atitle%3D6-Arylmethylidene%2520penicillin-based%2520sulfone%2520inhibitors%2520for%2520repurposing%2520antibiotic%2520efficiency%2520in%2520priority%2520pathogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3737%26epage%3D3755%26doi%3D10.1021%2Facs.jmedchem.0c00127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Torrens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocampo-Sosa, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamorano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascual, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martínez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Activity of Ceftazidime-Avibactam against Clinical and Isogenic Laboratory Pseudomonas aeruginosa Isolates Expressing Combinations of Most Relevant β-Lactam Resistance Mechanisms</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6407</span>– <span class="NLM_lpage">6410</span>, <span class="refDoi"> DOI: 10.1128/aac.01282-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.01282-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=27480848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKlsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=6407-6410&author=G.+Torrensauthor=G.+Cabotauthor=A.+A.+Ocampo-Sosaauthor=M.+C.+Conejoauthor=L.+Zamoranoauthor=F.+Navarroauthor=%C3%81.+Pascualauthor=L.+Mart%C3%ADnez-Mart%C3%ADnezauthor=A.+Oliver&title=Activity+of+Ceftazidime-Avibactam+against+Clinical+and+Isogenic+Laboratory+Pseudomonas+aeruginosa+Isolates+Expressing+Combinations+of+Most+Relevant+%CE%B2-Lactam+Resistance+Mechanisms&doi=10.1128%2Faac.01282-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Activity of ceftazidime-avibactam against clinical and isogenic laboratory Pseudomonas aeruginosa isolates expressing combinations of most relevant β-lactam resistance mechanisms</span></div><div class="casAuthors">Torrens, Gabriel; Cabot, Gabriel; Ocampo-Sosa, Alain A.; Carmen, Conejo M.; Zamorano, Laura; Navarro, Ferran; Pascual, Alvaro; Martinez-Martinez, Luis; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6407-6410</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The activity of ceftazidime-avibactam was compared with that of ceftazidime alone and meropenem against a collection of 190 Pseudomonas aeruginosa clin. isolates recovered from a multicenter study of bloodstream infections.  The addn. of avibactam increased ceftazidime susceptibility in the complete collection of strains (64.7-91.1%) and particularly among subsets of isolates showing AmpC hyperprodn. (10.9-76.1%) or multidrug resistance (MDR) profiles (27-77.8%).  The MICs of ceftazidime-avibactam, in contrast with those of ceftazidime or meropenem, remained at ≤4 μg/mL for a panel of 16 P. aeruginosa PAO1 isogenic mutants expressing multiple combinations of the most relevant β-lactam resistance mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVNemhW4E8SrVg90H21EOLACvtfcHk0liWfhjY6V_aOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKlsrk%253D&md5=e025b2d23eed307200d8db9229e87a34</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1128%2FAAC.01282-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01282-16%26sid%3Dliteratum%253Aachs%26aulast%3DTorrens%26aufirst%3DG.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DOcampo-Sosa%26aufirst%3DA.%2BA.%26aulast%3DConejo%26aufirst%3DM.%2BC.%26aulast%3DZamorano%26aufirst%3DL.%26aulast%3DNavarro%26aufirst%3DF.%26aulast%3DPascual%26aufirst%3D%25C3%2581.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADnez%26aufirst%3DL.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DActivity%2520of%2520Ceftazidime-Avibactam%2520against%2520Clinical%2520and%2520Isogenic%2520Laboratory%2520Pseudomonas%2520aeruginosa%2520Isolates%2520Expressing%2520Combinations%2520of%2520Most%2520Relevant%2520%25CE%25B2-Lactam%2520Resistance%2520Mechanisms%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D6407%26epage%3D6410%26doi%3D10.1128%2Faac.01282-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R. A.</span></span> <span> </span><span class="NLM_article-title">Avibactam and Class C β-Lactamases: Mechanism of Inhibition, Conservation of the Binding Pocket, and Implications for Resistance</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">5704</span>– <span class="NLM_lpage">5713</span>, <span class="refDoi"> DOI: 10.1128/aac.03057-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.03057-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25022578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslShsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=5704-5713&author=S.+D.+Lahiriauthor=M.+R.+Johnstoneauthor=P.+L.+Rossauthor=R.+E.+McLaughlinauthor=N.+B.+Olivierauthor=R.+A.+Alm&title=Avibactam+and+Class+C+%CE%B2-Lactamases%3A+Mechanism+of+Inhibition%2C+Conservation+of+the+Binding+Pocket%2C+and+Implications+for+Resistance&doi=10.1128%2Faac.03057-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Avibactam and class C β-lactamases: mechanism of inhibition, conservation of the binding pocket, and implications for resistance</span></div><div class="casAuthors">Lahiri, S. D.; Johnstone, M. R.; Ross, P. L.; McLaughlin, R. E.; Olivier, N. B.; Alm, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5704-5713, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Avibactam is a novel non-β-lactam β-lactamase inhibitor that inhibits a wide range of β-lactamases.  These include class A, class C, and some class D enzymes, which erode the activity of β-lactam drugs in multidrug-resistant pathogens like Pseudomonas aeruginosa and Enterobacteriaceae spp.  Avibactam is currently in clin. development in combination with the β-lactam antibiotics ceftazidime, ceftaroline fosamil, and aztreonam.  Avibactam has the potential to be the first β-lactamase inhibitor that might provide activity against class C-mediated resistance, which represents a growing concern in both hospital- and community-acquired infections.  Avibactam has an unusual mechanism of action: it is a covalent inhibitor that acts via ring opening, but in contrast to other currently used β-lactamase inhibitors, this reaction is reversible.  Here, we present a high-resoln. structure of avibactam bound to a class C β-lactamase, AmpC, from P. aeruginosa that provided insight into the mechanism of both acylation and recyclization in this enzyme class and highlighted the differences obsd. between class A and class C inhibition.  Furthermore, variants resistant to avibactam that identified the residues important for inhibition were isolated.  Finally, the structural information was used to predict effective inhibition by sequence anal. and functional studies of class C β-lactamases from a large and diverse set of contemporary clin. isolates (P. aeruginosa and several Enterobacteriaceae spp.) obtained from recent infections to understand any preexisting variability in the binding pocket that might affect inhibition by avibactam.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo83AV3Iqfuf7Vg90H21EOLACvtfcHk0liWfhjY6V_aOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslShsr3F&md5=d31d11de33526c1bcfb958c1516001fb</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2FAAC.03057-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.03057-14%26sid%3Dliteratum%253Aachs%26aulast%3DLahiri%26aufirst%3DS.%2BD.%26aulast%3DJohnstone%26aufirst%3DM.%2BR.%26aulast%3DRoss%26aufirst%3DP.%2BL.%26aulast%3DMcLaughlin%26aufirst%3DR.%2BE.%26aulast%3DOlivier%26aufirst%3DN.%2BB.%26aulast%3DAlm%26aufirst%3DR.%2BA.%26atitle%3DAvibactam%2520and%2520Class%2520C%2520%25CE%25B2-Lactamases%253A%2520Mechanism%2520of%2520Inhibition%252C%2520Conservation%2520of%2520the%2520Binding%2520Pocket%252C%2520and%2520Implications%2520for%2520Resistance%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D5704%26epage%3D5713%26doi%3D10.1128%2Faac.03057-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahić, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, R.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durand-Réville, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thresher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livchak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkup, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, S. L.</span></span> <span> </span><span class="NLM_article-title">Kinetics of Avibactam Inhibition against Class A, C, and D β-Lactamases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">27960</span>– <span class="NLM_lpage">27971</span>, <span class="refDoi"> DOI: 10.1074/jbc.m113.485979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1074%2Fjbc.M113.485979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=23913691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFemsr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=27960-27971&author=D.+E.+Ehmannauthor=H.+Jahi%C4%87author=P.+L.+Rossauthor=R.-F.+Guauthor=J.+Huauthor=T.+F.+Durand-R%C3%A9villeauthor=S.+Lahiriauthor=J.+Thresherauthor=S.+Livchakauthor=N.+Gaoauthor=T.+Palmerauthor=G.+K.+Walkupauthor=S.+L.+Fisher&title=Kinetics+of+Avibactam+Inhibition+against+Class+A%2C+C%2C+and+D+%CE%B2-Lactamases&doi=10.1074%2Fjbc.m113.485979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of Avibactam Inhibition against Class A, C, and D β-Lactamases</span></div><div class="casAuthors">Ehmann, David E.; Jahic, Haris; Ross, Philip L.; Gu, Rong-Fang; Hu, Jun; Durand-Reville, Thomas F.; Lahiri, Sushmita; Thresher, Jason; Livchak, Stephania; Gao, Ning; Palmer, Tiffany; Walkup, Grant K.; Fisher, Stewart L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">27960-27971</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Avibactam is a non-β-lactam β-lactamase inhibitor with a spectrum of activity that includes β-lactamase enzymes of classes A, C, and selected D examples.  In this work acylation and deacylation rates were measured against the clin. important enzymes CTX-M-15, KPC-2, Enterobacter cloacae AmpC, Pseudomonas aeruginosa AmpC, OXA-10, and OXA-48.  The efficiency of acylation (k2/Ki) varied across the enzyme spectrum, from 1.1 × 101m-1s-1 for OXA-10 to 1.0 × 105 for CTX-M-15.  Inhibition of OXA-10 was shown to follow the covalent reversible mechanism, and the acylated OXA-10 displayed the longest residence time for deacylation, with a half-life of > 5 days.  Across multiple enzymes, acyl enzyme stability was assessed by mass spectrometry.  These inhibited enzyme forms were stable to rearrangement or hydrolysis, with the exception of KPC-2.  KPC-2 displayed a slow hydrolytic route that involved fragmentation of the acyl-avibactam complex.  The identity of released degrdn. products was investigated, and a possible mechanism for the slow deacylation from KPC-2 is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoULb6Pb1j63bVg90H21EOLACvtfcHk0liWfhjY6V_aOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFemsr%252FL&md5=baa5d471bb16d83c797cfa9069b950f8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.485979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.485979%26sid%3Dliteratum%253Aachs%26aulast%3DEhmann%26aufirst%3DD.%2BE.%26aulast%3DJahi%25C4%2587%26aufirst%3DH.%26aulast%3DRoss%26aufirst%3DP.%2BL.%26aulast%3DGu%26aufirst%3DR.-F.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DDurand-R%25C3%25A9ville%26aufirst%3DT.%2BF.%26aulast%3DLahiri%26aufirst%3DS.%26aulast%3DThresher%26aufirst%3DJ.%26aulast%3DLivchak%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DPalmer%26aufirst%3DT.%26aulast%3DWalkup%26aufirst%3DG.%2BK.%26aulast%3DFisher%26aufirst%3DS.%2BL.%26atitle%3DKinetics%2520of%2520Avibactam%2520Inhibition%2520against%2520Class%2520A%252C%2520C%252C%2520and%2520D%2520%25CE%25B2-Lactamases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D27960%26epage%3D27971%26doi%3D10.1074%2Fjbc.m113.485979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gootz, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drawz, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harry, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nottingham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buynak, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">Inactivation of a class A and a class C β-lactamase by 6β-(hydroxymethyl)penicillanic acid sulfone</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2011.11.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.bcp.2011.11.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=22155308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1Glsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2012&pages=462-471&author=K.+M.+Papp-Wallaceauthor=C.+R.+Bethelauthor=T.+D.+Gootzauthor=W.+Shangauthor=J.+Strohauthor=W.+Lauauthor=D.+McLeodauthor=L.+Priceauthor=A.+Marfatauthor=A.+Distlerauthor=S.+M.+Drawzauthor=H.+Chenauthor=E.+Harryauthor=M.+Nottinghamauthor=P.+R.+Careyauthor=J.+D.+Buynakauthor=R.+A.+Bonomo&title=Inactivation+of+a+class+A+and+a+class+C+%CE%B2-lactamase+by+6%CE%B2-%28hydroxymethyl%29penicillanic+acid+sulfone&doi=10.1016%2Fj.bcp.2011.11.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Inactivation of a class A and a class C β-lactamase by 6β-(hydroxymethyl)penicillanic acid sulfone</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.; Bethel, Christopher R.; Gootz, Thomas D.; Shang, Wenchi; Stroh, Justin; Lau, William; McLeod, Dale; Price, Loren; Marfat, Anthony; Distler, Anne; Drawz, Sarah M.; Chen, Hansong; Harry, Emily; Nottingham, Micheal; Carey, Paul R.; Buynak, John D.; Bonomo, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">462-471</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">β-Lactamase inhibitors (clavulanic acid, sulbactam, and tazobactam) contribute significantly to the longevity of the β-lactam antibiotics used to treat serious infections.  In the quest to design more potent compds. and to understand the mechanism of action of known inhibitors, 6β-(hydroxymethyl)penicillanic acid sulfone (6β-HM-sulfone) was tested against isolates expressing the class A TEM-1 β-lactamase and a clin. important variant of the AmpC cephalosporinase of Pseudomonas aeruginosa, PDC-3.  The addn. of the 6β-HM-sulfone inhibitor to ampicillin was highly effective.  6β-HM-sulfone inhibited TEM-1 with an IC50 of 12 ± 2 nM and PDC-3 with an IC50 of 180 ± 36 nM, and displayed lower partition ratios than com. inhibitors, with partition ratios (kcat/kinact) equal to 174 for TEM-1 and 4 for PDC-3.  Measured for 20 h, 6β-HM-sulfone demonstrated rapid, first-order inactivation kinetics with the extent of inactivation being related to the concn. of inhibitor for both TEM-1 and PDC-3.  Using mass spectrometry to gain insight into the intermediates of inactivation of this inhibitor, 6β-HM-sulfone was found to form a major adduct of +247 ± 5 Da with TEM-1 and +245 ± 5 Da with PDC-3, suggesting that the covalently bound, hydrolytically stabilized acyl-enzyme has lost a mol. of water (H-O-H).  Minor adducts of +88 ± 5 Da with TEM-1 and +85 ± 5 Da with PDC-3 revealed that fragmentation of the covalent adduct can result but appeared to occur slowly with both enzymes. 6β-HM-sulfone is an effective and versatile β-lactamase inhibitor of representative class A and C enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrekFlbHHVjjLVg90H21EOLACvtfcHk0lgjf1W7Uia0zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1Glsg%253D%253D&md5=f425d156966894d16d418b74402d2360</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2011.11.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2011.11.015%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DGootz%26aufirst%3DT.%2BD.%26aulast%3DShang%26aufirst%3DW.%26aulast%3DStroh%26aufirst%3DJ.%26aulast%3DLau%26aufirst%3DW.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DPrice%26aufirst%3DL.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DDistler%26aufirst%3DA.%26aulast%3DDrawz%26aufirst%3DS.%2BM.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DHarry%26aufirst%3DE.%26aulast%3DNottingham%26aufirst%3DM.%26aulast%3DCarey%26aufirst%3DP.%2BR.%26aulast%3DBuynak%26aufirst%3DJ.%2BD.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DInactivation%2520of%2520a%2520class%2520A%2520and%2520a%2520class%2520C%2520%25CE%25B2-lactamase%2520by%25206%25CE%25B2-%2528hydroxymethyl%2529penicillanic%2520acid%2520sulfone%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2012%26volume%3D83%26spage%3D462%26epage%3D471%26doi%3D10.1016%2Fj.bcp.2011.11.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte-Brasseur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span> <span> </span><span class="NLM_article-title">pKa calculations for class C beta-lactamases: The role of tyr-150</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1002/(sici)1097-0134(20000701)40:1<23::aid-prot40>3.0.co;2-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1002%2F%28SICI%291097-0134%2820000701%2940%3A1%3C23%3A%3AAID-PROT40%3E3.0.CO%3B2-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10813827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjvFyjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2000&pages=23-28&author=J.+Lamotte-Brasseurauthor=A.+Dubusauthor=R.+C.+Wade&title=pKa+calculations+for+class+C+beta-lactamases%3A+The+role+of+tyr-150&doi=10.1002%2F%28sici%291097-0134%2820000701%2940%3A1%3C23%3A%3Aaid-prot40%3E3.0.co%3B2-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">pKa calculations for class C β-lactamases: the role of Tyr-150</span></div><div class="casAuthors">Lamotte-Brasseur, Josette; Dubus, Alain; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-28</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Poisson-Boltzmann method was used to compute the pKa values of titratable residues in a set of class C β-lactamases.  In these calcns., the pKa of the phenolic group of residue Tyr150 is the only one to stand out with an abnormally low value of 8.3, more than one pKa unit lower than the measured ref. value for tyrosine in soln.  Other important residues of the catalytic pocket, such as the conserved Lys67, Lys315, His314, and Glu272 (hydrogen-bonded to the ammonium group of Lys315), display normal protonation states at neutral pH.  PKa values were also computed in catalytically impaired β-lactamase mutants.  Comparisons between the relative kcat values and the Tyr150 pKa value in these mutants revealed a striking correlation.  In active enzymes, this pKa value is always lower than the soln. ref. value while it is close to normal in inactive enzymes.  These results thus support the hypothesis that the phenolate form of Tyr150 is responsible for the activation of the nucleophilic serine.  The possible roles of Lys67 and Lys315 during catalysis are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp31t-YKgoazLVg90H21EOLACvtfcHk0lgjf1W7Uia0zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjvFyjuro%253D&md5=8a8f368139b15426075c00a0737cf4c2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2820000701%2940%3A1%3C23%3A%3AAID-PROT40%3E3.0.CO%3B2-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252820000701%252940%253A1%253C23%253A%253AAID-PROT40%253E3.0.CO%253B2-7%26sid%3Dliteratum%253Aachs%26aulast%3DLamotte-Brasseur%26aufirst%3DJ.%26aulast%3DDubus%26aufirst%3DA.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DpKa%2520calculations%2520for%2520class%2520C%2520beta-lactamases%253A%2520The%2520role%2520of%2520tyr-150%26jtitle%3DProteins%26date%3D2000%26volume%3D40%26spage%3D23%26epage%3D28%26doi%3D10.1002%2F%28sici%291097-0134%2820000701%2940%3A1%3C23%3A%3Aaid-prot40%3E3.0.co%3B2-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dubus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte-Brasseur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frère, J.-M.</span></span> <span> </span><span class="NLM_article-title">The roles of residues Tyr150, Glu272, and His314 in class C β-lactamases</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1002/prot.7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1002%2Fprot.7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=8865342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaK28Xlt1ymsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1996&pages=473-485&author=A.+Dubusauthor=P.+Ledentauthor=J.+Lamotte-Brasseurauthor=J.-M.+Fr%C3%A8re&title=The+roles+of+residues+Tyr150%2C+Glu272%2C+and+His314+in+class+C+%CE%B2-lactamases&doi=10.1002%2Fprot.7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The roles of residues Tyr150, Glu272, and His314 in class C β-lactamases</span></div><div class="casAuthors">Dubus, Alain; Ledent, Philippe; Lamotte-Brasseur, Josette; Frere, Jean-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-485</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss</span>)
        </div><div class="casAbstract">Serine β-lactamases contribute widely to the β-lactam resistance phenomena.  Unfortunately, the intimate details of their catalytic mechanism remain elusive and subject to some controversy even though many "natural" and "artificial" mutants of these different enzymes have been isolated.  This paper is essentially focused on class C β-lactamases, which contain a Tyr (Tyr 150) as the first residue of the second conserved element, in contrast to their class A counterparts, in which a Ser is found in the corresponding position.  The authors have modified this Tyr residue by site-directed mutagenesis.  On the basis of the three-dimensional structure of the Enterobacter cloacae P99 enzyme, it seem that residues Glu 272 and His 314 might also be important.  They were similarly substituted.  The modified enzymes were isolated and their catalytic properties detd.  The authors' results indicated that His 314 was not required for catalysis and that Glu 272 did not play an important role in acylation but was involved to a small extent in the deacylation process.  Conversely, Tyr 150 was confirmed to be central for catalysis, at least with the best substrates.  On the basis of the comparison of data obtained for several class C enzyme mutants and in agreement with recent structural data, the authors propose that the phenolate anion of Tyr 150, in conjunction with the alkyl ammonium of Lys 315, acts as the general base responsible for the activation of the active-site Ser 64 during the acylation step and for the subsequent activation of a water mol. in the deacylation process.  The evolution of the important superfamily of penicillin-recognizing enzymes is further discussed in the light of this proposed mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFCLAkWT7yZbVg90H21EOLACvtfcHk0lgjf1W7Uia0zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xlt1ymsrk%253D&md5=20d4c25b4816b7220e7630c3556d703f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fprot.7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.7%26sid%3Dliteratum%253Aachs%26aulast%3DDubus%26aufirst%3DA.%26aulast%3DLedent%26aufirst%3DP.%26aulast%3DLamotte-Brasseur%26aufirst%3DJ.%26aulast%3DFr%25C3%25A8re%26aufirst%3DJ.-M.%26atitle%3DThe%2520roles%2520of%2520residues%2520Tyr150%252C%2520Glu272%252C%2520and%2520His314%2520in%2520class%2520C%2520%25CE%25B2-lactamases%26jtitle%3DProteins%26date%3D1996%26volume%3D25%26spage%3D473%26epage%3D485%26doi%3D10.1002%2Fprot.7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato-Toma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwashita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishiguro, M.</span></span> <span> </span><span class="NLM_article-title">pKa measurements from nuclear magnetic resonance of tyrosine-150 in class C beta-lactamase</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1042/bj20021447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1042%2Fbj20021447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=12513696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlCitr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2003&pages=175-181&author=Y.+Kato-Tomaauthor=T.+Iwashitaauthor=K.+Masudaauthor=Y.+Oyamaauthor=M.+Ishiguro&title=pKa+measurements+from+nuclear+magnetic+resonance+of+tyrosine-150+in+class+C+beta-lactamase&doi=10.1042%2Fbj20021447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">pKa measurements from nuclear magnetic resonance of tyrosine-150 in class C β-lactamase</span></div><div class="casAuthors">Kato-Toma, Yoko; Iwashita, Takashi; Masuda, Katsuyoshi; Oyama, Yoshiaki; Ishiguro, Masaji</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">13C-NMR spectroscopy was used to est. the pKa values for the Tyr150 (Y150) residue in wild-type and mutant class C β-lactamases.  The tyrosine residues of the wild-type and mutant lactamases were replaced with 13C-labeled L-tyrosine ([phenol -4-13C]tyrosine) in order to observe the tyrosine residues selectively.  Spectra of the wild-type and K67C mutant (Lys67Cys) enzyme were compared with the Y150C mutant lactamase spectra to identify the signal originating from Tyr150.  Titrn. expts. showed that the chem. shift of the Tyr150 resonance in the wild-type enzyme is almost invariant in a range of 0.1 p.p.m. up to pH 11 and showed that the pKa of this residue is well above 11 in the substrate-free form.  According to solvent accessibility calcns. on X-ray-derived structures, the phenolic oxygen of Tyr150, which is near the amino groups of Lys315 and Lys67, appears to have low solvent accessibility.  These results suggest that, in the native enzyme, Tyr150 in class C β-lactamase of Citrobacter freundii GN346 is protonated and that when Tyr150 loses a proton, a proton from Lys67 would replace it.  Consequently, Tyr150 would be protonated during the entire titrn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbUIUqZZcdNLVg90H21EOLACvtfcHk0liSNsfu53DfTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlCitr0%253D&md5=9b6476d674e3195764bbe05604863f53</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1042%2Fbj20021447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20021447%26sid%3Dliteratum%253Aachs%26aulast%3DKato-Toma%26aufirst%3DY.%26aulast%3DIwashita%26aufirst%3DT.%26aulast%3DMasuda%26aufirst%3DK.%26aulast%3DOyama%26aufirst%3DY.%26aulast%3DIshiguro%26aufirst%3DM.%26atitle%3DpKa%2520measurements%2520from%2520nuclear%2520magnetic%2520resonance%2520of%2520tyrosine-150%2520in%2520class%2520C%2520beta-lactamase%26jtitle%3DBiochem.%2520J.%26date%3D2003%26volume%3D371%26spage%3D175%26epage%3D181%26doi%3D10.1042%2Fbj20021447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durand-Reville, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benvenuti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Luca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanyal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Docquier, J.-D.</span></span> <span> </span><span class="NLM_article-title">Structural Insight into Potent Broad-Spectrum Inhibition with Reversible Recyclization Mechanism: Avibactam in Complex with CTX-M-15 and Pseudomonas aeruginosa AmpC β-Lactamases</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2496</span>– <span class="NLM_lpage">2505</span>, <span class="refDoi"> DOI: 10.1128/aac.02247-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.02247-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=23439634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXoslyrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=2496-2505&author=S.+D.+Lahiriauthor=S.+Manganiauthor=T.+Durand-Revilleauthor=M.+Benvenutiauthor=F.+De+Lucaauthor=G.+Sanyalauthor=J.-D.+Docquier&title=Structural+Insight+into+Potent+Broad-Spectrum+Inhibition+with+Reversible+Recyclization+Mechanism%3A+Avibactam+in+Complex+with+CTX-M-15+and+Pseudomonas+aeruginosa+AmpC+%CE%B2-Lactamases&doi=10.1128%2Faac.02247-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into potent broad-spectrum inhibition with reversible recyclization mechanism: avibactam in complex with CTX-M-15 and Pseudomonas aeruginosa AmpC β-lactamases</span></div><div class="casAuthors">Lahiri, Sushmita D.; Mangani, Stefano; Durand-Reville, Thomas; Benvenuti, Manuela; De Luca, Filomena; Sanyal, Gautam; Docquier, Jean-Denis</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2496-2505</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Although β-lactams have been the most effective class of antibacterial agents used in clin. practice for the past half century, their effectiveness on Gram-neg. bacteria has been eroded due to the emergence and spread of β-lactamase enzymes that are not affected by currently marketed β-lactam/β-lactamase inhibitor combinations.  Avibactam is a novel, covalent, non-β-lactam β-lactamase inhibitor presently in clin. development in combination with either ceftaroline or ceftazidime.  In vitro studies show that avibactam may restore the broad-spectrum activity of cephalosporins against class A, class C, and some class D β-lactamases.  Here we describe the structures of two clin. important β-lactamase enzymes bound to avibactam, the class A CTX-M-15 extended-spectrum β-lactamase and the class C Pseudomonas aeruginosa AmpC β-lactamase, which together provide insight into the binding modes for the resp. enzyme classes.  The structures reveal similar binding modes in both enzymes and thus provide a rationale for the broad-spectrum inhibitory activity of avibactam.  Identification of the key residues surrounding the binding pocket allows for a better understanding of the potency of this scaffold.  Finally, avibactam has recently been shown to be a reversible inhibitor, and the structures provide insights into the mechanism of avibactam recyclization.  Anal. of the ultra-high-resoln. CTX-M-15 structure suggests how the deacylation mechanism favors recyclization over hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBwwMi5MIpF7Vg90H21EOLACvtfcHk0liSNsfu53DfTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXoslyrsrY%253D&md5=c8a5fa5871f2b5d9efc78b6c4c5d7742</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1128%2FAAC.02247-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02247-12%26sid%3Dliteratum%253Aachs%26aulast%3DLahiri%26aufirst%3DS.%2BD.%26aulast%3DMangani%26aufirst%3DS.%26aulast%3DDurand-Reville%26aufirst%3DT.%26aulast%3DBenvenuti%26aufirst%3DM.%26aulast%3DDe%2BLuca%26aufirst%3DF.%26aulast%3DSanyal%26aufirst%3DG.%26aulast%3DDocquier%26aufirst%3DJ.-D.%26atitle%3DStructural%2520Insight%2520into%2520Potent%2520Broad-Spectrum%2520Inhibition%2520with%2520Reversible%2520Recyclization%2520Mechanism%253A%2520Avibactam%2520in%2520Complex%2520with%2520CTX-M-15%2520and%2520Pseudomonas%2520aeruginosa%2520AmpC%2520%25CE%25B2-Lactamases%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D2496%26epage%3D2505%26doi%3D10.1128%2Faac.02247-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0liSNsfu53DfTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olasz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mignon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Proft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veszprémi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geerlings, P.</span></span> <span> </span><span class="NLM_article-title">Effect of the π-π stacking interaction on the acidity of phenol</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>407</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1016/j.cplett.2005.03.145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.cplett.2005.03.145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFWntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=407&publication_year=2005&pages=504-509&author=A.+Olaszauthor=P.+Mignonauthor=F.+De+Proftauthor=T.+Veszpr%C3%A9miauthor=P.+Geerlings&title=Effect+of+the+%CF%80-%CF%80+stacking+interaction+on+the+acidity+of+phenol&doi=10.1016%2Fj.cplett.2005.03.145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of the π-π stacking interaction on the acidity of phenol</span></div><div class="casAuthors">Olasz, Andras; Mignon, Pierre; De Proft, Frank; Veszpremi, Tamas; Geerlings, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">407</span>
        (<span class="NLM_cas:issue">4-6</span>),
    <span class="NLM_cas:pages">504-509</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Stacked complexes made of phenol and substituted benzenes are studied via MP2 calcns. in order to investigate the influence of the stacking interaction on the acidity of phenol.  Aq.-phase proton affinities have been computed and converted to pKa values using a calibration procedure involving a set of substituted phenols.  Both T-shaped and parallel-displaced conformations lead to an appreciable decrease in the pKa, with the largest effect found for T-shaped arrangements.  Local and global DFT based reactivity descriptors are used to analyze the effect of the stacking partner as well as the interaction between the arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MYt4p0z8arVg90H21EOLACvtfcHk0lgRL4tcPiAKhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFWntbk%253D&md5=f82ba9d5fb5465c78504234b7baa39a0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cplett.2005.03.145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cplett.2005.03.145%26sid%3Dliteratum%253Aachs%26aulast%3DOlasz%26aufirst%3DA.%26aulast%3DMignon%26aufirst%3DP.%26aulast%3DDe%2BProft%26aufirst%3DF.%26aulast%3DVeszpr%25C3%25A9mi%26aufirst%3DT.%26aulast%3DGeerlings%26aufirst%3DP.%26atitle%3DEffect%2520of%2520the%2520%25CF%2580-%25CF%2580%2520stacking%2520interaction%2520on%2520the%2520acidity%2520of%2520phenol%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D2005%26volume%3D407%26spage%3D504%26epage%3D509%26doi%3D10.1016%2Fj.cplett.2005.03.145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lence, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tizón, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otero, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peón, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prazeres, V. F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llamas-Saiz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Raaij, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Bello, C.</span></span> <span> </span><span class="NLM_article-title">Mechanistic basis of the inhibition of type II dehydroquinase by (2<i>S</i>)- and (2<i>R</i>)-2-benzyl-3-dehydroquinic acids</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1021/cb300493s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300493s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCltbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=568-577&author=E.+Lenceauthor=L.+Tiz%C3%B3nauthor=J.+M.+Oteroauthor=A.+Pe%C3%B3nauthor=V.+F.+V.+Prazeresauthor=A.+L.+Llamas-Saizauthor=G.+C.+Foxauthor=M.+J.+van+Raaijauthor=H.+Lambauthor=A.+R.+Hawkinsauthor=C.+Gonz%C3%A1lez-Bello&title=Mechanistic+basis+of+the+inhibition+of+type+II+dehydroquinase+by+%282S%29-+and+%282R%29-2-benzyl-3-dehydroquinic+acids&doi=10.1021%2Fcb300493s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic Basis of the Inhibition of Type II Dehydroquinase by (2S)- and (2R)-2-Benzyl-3-dehydroquinic Acids</span></div><div class="casAuthors">Lence, Emilio; Tizon, Lorena; Otero, Jose M.; Peon, Antonio; Prazeres, Veronica F. V.; Llamas-Saiz, Antonio L.; Fox, Gavin C.; van Raaij, Mark J.; Lamb, Heather; Hawkins, Alastair R.; Gonzalez-Bello, Concepcion</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">568-577</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structural changes caused by the substitution of the arom. moiety in (2S)-2-benzyl-3-dehydroquinic acids and its C2 epimers by electron-withdrawing or electron-donating groups in type II dehydroquinase (DHQ2) enzyme from M. tuberculosis and H. pylori have been investigated by structural and computational studies.  Both compds. are reversible competitive inhibitors of this enzyme, which is essential in these pathogenic bacteria.  The crystal structures of M. tuberculosis and H. pylori DHQ2 enzymes in complex with (2S)-2-(4-methoxy)benzyl- and (2S)-2-perfluorobenzyl-3-dehydroquinic acids have been solved at 2.0, 2.3, 2.0, and 1.9 Å, resp.  The crystal structure of M. tuberculosis DHQ2 in complex with (2R)-2-(benzothiophen-5-yl)methyl-3-dehydroquinic acid is also reported at 1.55 Å.  These crystal structures reveal key differences in the conformation of the flexible loop of the two enzymes, a difference that depends on the presence of electron-withdrawing or electron-donating groups in the arom. moiety of the inhibitors.  This loop closes over the active site after substrate binding, and its flexibility is essential for the function of the enzyme.  These differences have also been investigated by mol. dynamics simulations in an effort to understand the significant inhibition potency differences obsd. between some of these compds. and also to obtain more information about the possible movements of the loop.  These computational studies have also allowed us to identify key structural factors of the H. pylori loop that could explain its reduced flexibility in comparison to the M. tuberculosis loop, specifically by the formation of a key salt bridge between the side chains of residues Asp18 and Arg20.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEZ02X1C6t8LVg90H21EOLACvtfcHk0lgRL4tcPiAKhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCltbbP&md5=94cfacb63500381e0179b89b0f672cd7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fcb300493s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300493s%26sid%3Dliteratum%253Aachs%26aulast%3DLence%26aufirst%3DE.%26aulast%3DTiz%25C3%25B3n%26aufirst%3DL.%26aulast%3DOtero%26aufirst%3DJ.%2BM.%26aulast%3DPe%25C3%25B3n%26aufirst%3DA.%26aulast%3DPrazeres%26aufirst%3DV.%2BF.%2BV.%26aulast%3DLlamas-Saiz%26aufirst%3DA.%2BL.%26aulast%3DFox%26aufirst%3DG.%2BC.%26aulast%3Dvan%2BRaaij%26aufirst%3DM.%2BJ.%26aulast%3DLamb%26aufirst%3DH.%26aulast%3DHawkins%26aufirst%3DA.%2BR.%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DC.%26atitle%3DMechanistic%2520basis%2520of%2520the%2520inhibition%2520of%2520type%2520II%2520dehydroquinase%2520by%2520%25282S%2529-%2520and%2520%25282R%2529-2-benzyl-3-dehydroquinic%2520acids%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D568%26epage%3D577%26doi%3D10.1021%2Fcb300493s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babaoglu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Deconstructing fragment-based inhibitor discovery</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">720</span>– <span class="NLM_lpage">723</span>, <span class="refDoi"> DOI: 10.1038/nchembio831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1038%2Fnchembio831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=17072304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=720-723&author=K.+Babaogluauthor=B.+K.+Shoichet&title=Deconstructing+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing fragment-based inhibitor discovery</span></div><div class="casAuthors">Babaoglu, Kerim; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">720-723</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based screens test multiple low-mol. wt. mols. for binding to a target.  Fragments often bind with low affinities but typically have better ligand efficiencies (ΔGbind /heavy atom count) than traditional screening hits.  This efficiency, combined with accompanying at.-resoln. structures, has made fragments popular starting points for drug discovery programs.  Fragment-based design adopts a constructive strategy: affinity is enhanced either by cycles of functional-group addn. or by joining two independent fragments together.  The final inhibitor is expected to adopt the same geometry as the original fragment hit.  Here the authors consider whether the inverse, deconstructive logic also applies-can one always parse a higher-affinity inhibitor into fragments that recapitulate the binding geometry of the larger mol..  Cocrystal structures of fragments deconstructed from a known β-lactamase inhibitor suggest that this is not always the case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVl5_M15h_rLVg90H21EOLACvtfcHk0lgRL4tcPiAKhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM&md5=de0d5632f6354314794c2f45cfdd2efd</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnchembio831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio831%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDeconstructing%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D720%26epage%3D723%26doi%3D10.1038%2Fnchembio831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Na, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, S.-S.</span></span> <span> </span><span class="NLM_article-title">GMP and IMP Are Competitive Inhibitors of CMY-10, an Extended-Spectrum Class C β-Lactamase</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e00098</span> <span class="refDoi"> DOI: 10.1128/aac.00098-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.00098-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=28242658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFeksLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=J.-H.+Naauthor=Y.+J.+Anauthor=S.-S.+Cha&title=GMP+and+IMP+Are+Competitive+Inhibitors+of+CMY-10%2C+an+Extended-Spectrum+Class+C+%CE%B2-Lactamase&doi=10.1128%2Faac.00098-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">GMP and IMP are competitive inhibitors of CMY-10, an extended-spectrum class C β-lactamase</span></div><div class="casAuthors">Na, Jung-Hyun; An, Young Jun; Cha, Sun-Shin</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e00098-17/1-e00098-17/5</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Nucleotides were effective in inhibiting the class C β-lactamase CMY-10.  IMP was the most potent competitive inhibitor, with a Ki value of 16.2 μM.  The crystal structure of CMY-10 complexed with GMP or IMP revealed that nucleotides fit into the R2 subsite of the active site with a unique vertical binding mode where the phosphate group at one terminus is deeply bound in the subsite and the base at the other terminus faces the solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5YIYsiPBAfbVg90H21EOLACvtfcHk0lic-nDaHossJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFeksLnN&md5=8a4fdb4919517e2205f5ea96bc9ac0ec</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1128%2FAAC.00098-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00098-17%26sid%3Dliteratum%253Aachs%26aulast%3DNa%26aufirst%3DJ.-H.%26aulast%3DAn%26aufirst%3DY.%2BJ.%26aulast%3DCha%26aufirst%3DS.-S.%26atitle%3DGMP%2520and%2520IMP%2520Are%2520Competitive%2520Inhibitors%2520of%2520CMY-10%252C%2520an%2520Extended-Spectrum%2520Class%2520C%2520%25CE%25B2-Lactamase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2Faac.00098-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Na, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seol, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, S.-S.</span></span> <span> </span><span class="NLM_article-title">Structural and mechanistic insights into the inhibition of class C β-lactamases through the adenylylation of the nucleophilic serine</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">dkw491</span>, <span class="refDoi"> DOI: 10.1093/jac/dkw491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkw491" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=dkw491&author=M.-K.+Kimauthor=Y.+J.+Anauthor=J.-H.+Naauthor=J.-H.+Seolauthor=J.+Y.+Ryuauthor=J.-W.+Leeauthor=L.-W.+Kangauthor=K.+M.+Chungauthor=J.-H.+Leeauthor=J.+H.+Moonauthor=J.+S.+Leeauthor=S.-S.+Cha&title=Structural+and+mechanistic+insights+into+the+inhibition+of+class+C+%CE%B2-lactamases+through+the+adenylylation+of+the+nucleophilic+serine&doi=10.1093%2Fjac%2Fdkw491"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkw491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkw491%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.-K.%26aulast%3DAn%26aufirst%3DY.%2BJ.%26aulast%3DNa%26aufirst%3DJ.-H.%26aulast%3DSeol%26aufirst%3DJ.-H.%26aulast%3DRyu%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DJ.-W.%26aulast%3DKang%26aufirst%3DL.-W.%26aulast%3DChung%26aufirst%3DK.%2BM.%26aulast%3DLee%26aufirst%3DJ.-H.%26aulast%3DMoon%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DCha%26aufirst%3DS.-S.%26atitle%3DStructural%2520and%2520mechanistic%2520insights%2520into%2520the%2520inhibition%2520of%2520class%2520C%2520%25CE%25B2-lactamases%2520through%2520the%2520adenylylation%2520of%2520the%2520nucleophilic%2520serine%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2016%26volume%3D72%26spage%3Ddkw491%26doi%3D10.1093%2Fjac%2Fdkw491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">ModLoop: Automated modeling of loops in protein structures</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2500</span>– <span class="NLM_lpage">2501</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btg362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fbioinformatics%2Fbtg362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=14668246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVSisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2003&pages=2500-2501&author=A.+Fiserauthor=A.+Sali&title=ModLoop%3A+Automated+modeling+of+loops+in+protein+structures&doi=10.1093%2Fbioinformatics%2Fbtg362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">ModLoop: automated modeling of loops in protein structures</span></div><div class="casAuthors">Fiser, Andras; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2500-2501</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: ModLoop is a web server for automated modeling of loops in protein structures.  The input is the at. coordinates of the protein structure in the Protein Data Bank format, and the specification of the starting and ending residues of one or more segments to be modeled, contg. no more than 20 residues in total.  The output is the coordinates of the non-hydrogen atoms in the modeled segments.  A user provides the input to the server via a simple web interface, and receives the output by e-mail.  The server relies on the loop modeling routine in MODELLER that predicts the loop conformations by satisfaction of spatial restraints, without relying on a database of known protein structures.  For a rapid response, ModLoop runs on a cluster of Linux PC computers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk66um-UAJe7Vg90H21EOLACvtfcHk0lic-nDaHossJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvVSisLk%253D&md5=4b0fdaca0a412715682496883ab2019c</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtg362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtg362%26sid%3Dliteratum%253Aachs%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModLoop%253A%2520Automated%2520modeling%2520of%2520loops%2520in%2520protein%2520structures%26jtitle%3DBioinformatics%26date%3D2003%26volume%3D19%26spage%3D2500%26epage%3D2501%26doi%3D10.1093%2Fbioinformatics%2Fbtg362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, R. K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Modeling of loops in protein structures</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1753</span>– <span class="NLM_lpage">1773</span>, <span class="refDoi"> DOI: 10.1110/ps.9.9.1753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1110%2Fps.9.9.1753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=11045621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntFamsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2000&pages=1753-1773&author=A.+Fiserauthor=R.+K.+G.+Doauthor=A.+Sali&title=Modeling+of+loops+in+protein+structures&doi=10.1110%2Fps.9.9.1753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling of loops in protein structures</span></div><div class="casAuthors">Fiser, Andras; Do, Richard Kinh Gian; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1753-1773</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Comparative protein structure prediction is limited mostly by the errors in alignment and loop modeling.  We describe here a new automated modeling technique that significantly improves the accuracy of loop predictions in protein structures.  The positions of all nonhydrogen atoms of the loop are optimized in a fixed environment with respect to a pseudo energy function.  The energy is a sum of many spatial restraints that include the bond length, bond angle, and improper dihedral angle terms from the CHARMM-22 force field, statistical preferences for the main-chain and side-chain dihedral angles, and statistical preferences for nonbonded at. contacts that depend on the two atom types, their distance through space, and sepn. in sequence.  The energy function is optimized with the method of conjugate gradients combined with mol. dynamics and simulated annealing.  Typically, the predicted loop conformation corresponds to the lowest energy conformation among 500 independent optimizations.  Predictions were made for 40 loops of known structure at each length from 1 to 14 residues.  The accuracy of loop predictions is evaluated as a function of thoroughness of conformational sampling, loop length, and structural properties of native loops.  When accuracy is measured by local superposition of the model on the native loop, 100, 90, and 30% of 4-, 8-, and 12-residue loop predictions, resp., had <2 Å RMSD error for the mainchain N, Cα, C, and O atoms; the av. accuracies were 0.59±0.05, 1.16±0.10, and 2.61±0.16 Å, resp.  To simulate real comparative modeling problems, the method was also evaluated by predicting loops of known structure in only approx. correct environments with errors typical of comparative modeling without misalignment.  When the RMSD distortion of the main-chain stem atoms is 2.5 Å, the av. loop prediction error increased by 180, 25, and 3% for 4-, 8-, and 12-residue loops, resp.  The accuracy of the lowest energy prediction for a given loop can be estd. from the structural variability among a no. of low energy predictions.  The relative value of the present method is gauged by (1) comparing it with one of the most successful previously described methods, and (2) describing its accuracy in recent blind predictions of protein structure.  Finally, it is shown that the av. accuracy of prediction is limited primarily by the accuracy of the energy function rather than by the extent of conformational sampling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsbG2px_r_krVg90H21EOLACvtfcHk0lic-nDaHossJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntFamsL8%253D&md5=0bcdbfbddf54ecca0e3fee95ecdc3f8b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1110%2Fps.9.9.1753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.9.9.1753%26sid%3Dliteratum%253Aachs%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DR.%2BK.%2BG.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModeling%2520of%2520loops%2520in%2520protein%2520structures%26jtitle%3DProtein%2520Sci.%26date%3D2000%26volume%3D9%26spage%3D1753%26epage%3D1773%26doi%3D10.1110%2Fps.9.9.1753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ness, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paetzel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strynadka, N. C. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design Guides the Improved Efficacy of Deacylation Transition State Analogue Inhibitors of TEM-1 β-Lactamase†,#</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">5312</span>– <span class="NLM_lpage">5321</span>, <span class="refDoi"> DOI: 10.1021/bi992505b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi992505b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisVelsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=5312-5321&author=S.+Nessauthor=R.+Martinauthor=A.+M.+Kindlerauthor=M.+Paetzelauthor=M.+Goldauthor=S.+E.+Jensenauthor=J.+B.+Jonesauthor=N.+C.+J.+Strynadka&title=Structure-Based+Design+Guides+the+Improved+Efficacy+of+Deacylation+Transition+State+Analogue+Inhibitors+of+TEM-1+%CE%B2-Lactamase%E2%80%A0%2C%23&doi=10.1021%2Fbi992505b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design Guides the Improved Efficacy of Deacylation Transition State Analogue Inhibitors of TEM-1 β-Lactamase</span></div><div class="casAuthors">Ness, Steven; Martin, Richard; Kindler, Alois M.; Paetzel, Mark; Gold, Marvin; Jensen, Susan E.; Jones, J. Bryan; Strynadka, Natalie C. J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5312-5321</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Transition state analog boronic acid inhibitors mimicking the structures and interactions of good penicillin substrates for the TEM-1 β-lactamase of Escherichia coli were designed using graphic analyses based on the enzyme's 1.7 Å crystallog. structure.  The synthesis of two of these transition state analogs, (1R)-1-phenylacetamido-2-(3-carboxyphenyl)ethylboronic acid (1) and (1R)-1-acetamido-2-(3-carboxy-2-hydroxyphenyl)ethylboronic acid (2), is reported.  Kinetic measurements show that, as designed, compds. 1 and 2 are highly effective deacylation transition state analog inhibitors of TEM-1 β-lactamase, with inhibition consts. of 5.9 and 13 nM, resp.  These values identify them as among the most potent competitive inhibitors yet reported for a β-lactamase.  The best inhibitor of the current series was (1R)-1-phenylacetamido-2-(3-carboxyphenyl)ethylboronic acid (1, KI = 5.9 nM), which resembles most closely the best known substrate of TEM-1, benzylpenicillin (penicillin G).  The high-resoln. crystallog. structures of these two inhibitors covalently bound to TEM-1 are also described.  In addn. to verifying the design features, these two structures show interesting and unanticipated changes in the active site area, including strong hydrogen bond formation, water displacement, and rearrangement of side chains.  The structures provide new insights into the further design of this potent class of β-lactamase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-aQSFfZl_R7Vg90H21EOLACvtfcHk0liUKlRwV_ILPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisVelsr8%253D&md5=441c78891c1860f80e29660283299961</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fbi992505b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi992505b%26sid%3Dliteratum%253Aachs%26aulast%3DNess%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DKindler%26aufirst%3DA.%2BM.%26aulast%3DPaetzel%26aufirst%3DM.%26aulast%3DGold%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DS.%2BE.%26aulast%3DJones%26aufirst%3DJ.%2BB.%26aulast%3DStrynadka%26aufirst%3DN.%2BC.%2BJ.%26atitle%3DStructure-Based%2520Design%2520Guides%2520the%2520Improved%2520Efficacy%2520of%2520Deacylation%2520Transition%2520State%2520Analogue%2520Inhibitors%2520of%2520TEM-1%2520%25CE%25B2-Lactamase%25E2%2580%25A0%252C%2523%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D5312%26epage%3D5321%26doi%3D10.1021%2Fbi992505b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strynadka, N. C. J.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanism of Avibactam-Mediated β-Lactamase Inhibition</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.5b00007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.5b00007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2jt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=175-184&author=D.+T.+Kingauthor=A.+M.+Kingauthor=S.+M.+Lalauthor=G.+D.+Wrightauthor=N.+C.+J.+Strynadka&title=Molecular+Mechanism+of+Avibactam-Mediated+%CE%B2-Lactamase+Inhibition&doi=10.1021%2Facsinfecdis.5b00007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanism of Avibactam-Mediated β-Lactamase Inhibition</span></div><div class="casAuthors">King, Dustin T.; King, Andrew M.; Lal, Sarah M.; Wright, Gerard D.; Strynadka, Natalie C. J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">175-184</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Emerging β-lactamase-mediated resistance is threatening the clin. utility of the single most prominent class of antibacterial agents used in medicine, the β-lactams.  The diazabicyclooctane avibactam is able to inhibit a wider range of serine β-lactamases than has been previously obsd. with β-lactamase inhibitors such as the widely prescribed clavulanic acid.  However, despite its broad-spectrum activity, variable levels of inhibition have been obsd. for mol. class D β-lactamases.  In order to better understand the mol. basis and spectrum of inhibition by avibactam, we provide structural and mechanistic anal. of the compd. in complex with important class A and D serine β-lactamases.  Herein, we reveal the 1.7- and 2.0-Å-resoln. crystal structures of avibactam covalently bound to class D β-lactamases OXA-10 and OXA-48.  Furthermore, a kinetic anal. of key active-site mutants for class A β-lactamase CTX-M-15 allows us to propose a validated mechanism for avibactam-mediated β-lactamase inhibition including a unique role for S130, which acts as a general base.  This study provides mol. insights that will aid in the design and development of avibactam-based chemotherapeutic agents effective against emerging drug-resistant microorganisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHO-uFt8V_hrVg90H21EOLACvtfcHk0liUKlRwV_ILPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2jt7w%253D&md5=d73c84c105248bae388669633e5684df</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.5b00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.5b00007%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DD.%2BT.%26aulast%3DKing%26aufirst%3DA.%2BM.%26aulast%3DLal%26aufirst%3DS.%2BM.%26aulast%3DWright%26aufirst%3DG.%2BD.%26aulast%3DStrynadka%26aufirst%3DN.%2BC.%2BJ.%26atitle%3DMolecular%2520Mechanism%2520of%2520Avibactam-Mediated%2520%25CE%25B2-Lactamase%2520Inhibition%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D175%26epage%3D184%26doi%3D10.1021%2Facsinfecdis.5b00007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraile-Ribot, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamorano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orellana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Barrio-Tofiño, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Diener, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortes-Lara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Causapé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martínez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gracia, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viñuela, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aznar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Hernández, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seral, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillo-García, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Calleja, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspiroz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Iglesia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramón, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallegos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesada, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyos, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horcajada, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrosa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">José González, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Moreno, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centelles, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirelis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Tarazona, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arenal-Andrés, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, M. d. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megías, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colmenarejo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomila, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tormo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garduño, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agulla, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seoane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzmán, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luisa Pérez del
Molino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbeito, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artiles, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azcona-Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáenz, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonio Oteo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaves, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cercenado, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alarcón, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurita, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merino, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morosini, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantón, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yagüe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiva, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canut, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oteo, J.</span></span> <span> </span><span class="NLM_article-title">Activity of Imipenem-Relebactam against a Large Collection of Pseudomonas aeruginosa Clinical Isolates and Isogenic β-Lactam-Resistant Mutants</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e02165</span> <span class="refDoi"> DOI: 10.1128/aac.02165-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.02165-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31740559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktFygsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=P.+A.+Fraile-Ribotauthor=L.+Zamoranoauthor=R.+Orellanaauthor=E.+del+Barrio-Tofi%C3%B1oauthor=I.+S%C3%A1nchez-Dienerauthor=S.+Cortes-Laraauthor=C.+L%C3%B3pez-Causap%C3%A9author=G.+Cabotauthor=G.+Bouauthor=L.+Mart%C3%ADnez-Mart%C3%ADnezauthor=A.+Oliverauthor=F.+Gal%C3%A1nauthor=I.+Graciaauthor=M.+A.+Rodr%C3%ADguezauthor=L.+Mart%C3%ADnauthor=J.+M.+S%C3%A1nchezauthor=L.+Vi%C3%B1uelaauthor=M.+V.+Garc%C3%ADaauthor=J.+A.+Lepeauthor=J.+Aznarauthor=I.+L%C3%B3pez-Hern%C3%A1ndezauthor=C.+Seralauthor=F.+J.+Castillo-Garc%C3%ADaauthor=A.+I.+L%C3%B3pez-Callejaauthor=C.+Aspirozauthor=P.+de+la+Iglesiaauthor=S.+Ram%C3%B3nauthor=E.+Rieraauthor=M.+C.+P%C3%A9rezauthor=C.+Gallegosauthor=J.+Calvoauthor=M.+D.+Quesadaauthor=F.+Marcoauthor=Y.+Hoyosauthor=J.+P.+Horcajadaauthor=N.+Larrosaauthor=J.+Jos%C3%A9+Gonz%C3%A1lezauthor=F.+Tubauauthor=S.+Capillaauthor=M.+O.+P%C3%A9rez-Morenoauthor=M.+J.+Centellesauthor=E.+Padillaauthor=A.+Riveraauthor=B.+Mirelisauthor=R.+E.+Rodr%C3%ADguez-Tarazonaauthor=N.+Arenal-Andr%C3%A9sauthor=M.+d.+P.+Ortegaauthor=G.+Meg%C3%ADasauthor=I.+Garc%C3%ADaauthor=C.+Colmenarejoauthor=J.+C.+Gonz%C3%A1lezauthor=N.+M.+Mart%C3%ADnezauthor=B.+Gomilaauthor=S.+Ginerauthor=N.+Tormoauthor=E.+Gardu%C3%B1oauthor=J.+A.+Agullaauthor=A.+Seoaneauthor=J.+Pitaauthor=I.+P.+Vidalauthor=D.+M.+Guzm%C3%A1nauthor=M.+Garc%C3%ADaauthor=M.+Luisa+P%C3%A9rez+del%0AMolinoauthor=G.+Barbeitoauthor=F.+Artilesauthor=J.+M.+Azcona-Guti%C3%A9rrezauthor=Y.+S%C3%A1enzauthor=J.+Antonio+Oteoauthor=A.+Gonz%C3%A1lezauthor=J.+Villaauthor=F.+Chavesauthor=E.+Cercenadoauthor=T.+Alarc%C3%B3nauthor=N.+D.+Zuritaauthor=I.+Merinoauthor=M.+I.+Morosiniauthor=R.+Cant%C3%B3nauthor=M.+I.+S%C3%A1nchezauthor=L.+Morenoauthor=G.+Yag%C3%BCeauthor=J.+Leivaauthor=J.+L.+Barriosauthor=A.+Canutauthor=J.+Oteo&title=Activity+of+Imipenem-Relebactam+against+a+Large+Collection+of+Pseudomonas+aeruginosa+Clinical+Isolates+and+Isogenic+%CE%B2-Lactam-Resistant+Mutants&doi=10.1128%2Faac.02165-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Activity of imipenem-relebactam against a large collection of Pseudomonas aeruginosa clinical isolates and isogenic β-lactam-resistant mutants</span></div><div class="casAuthors">Fraile-Ribot, Pablo A.; Zamorano, Laura; Orellana, Rocio; del Barrio-Tofino, Ester; Sanchez-Diener, Irina; Cortes-Lara, Sara; Lopez-Causape, Carla; Cabot, Gabriel; Bou, German; Martinez-Martinez, Luis; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e02165</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Imipenem and imipenem-relebactam MICs were detd. for 1,445 Pseudomonas aeruginosa clin. isolates and a large panel of isogenic mutants showing the most relevant mutation-driven β-lactam resistance mechanisms.  Imipenem-relebactam showed the highest susceptibility rate (97.3%), followed by colistin and ceftolozane-tazobactam (both 94.6%).  Imipenem-relebactam MICs remained ≤2 μg/mL in all 16 isogenic PAO1 mutants and in 8 pairs of extensively drug-resistant clin. strains that had developed resistance to ceftolozane-tazobactam and ceftazidime-avibactam due to mutations in OXA-10 or AmpC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUtQrvtHwForVg90H21EOLACvtfcHk0lh9H8u8_lBpEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktFygsLk%253D&md5=1c5c6352b7f873cc37945147f1eae60e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1128%2Faac.02165-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.02165-19%26sid%3Dliteratum%253Aachs%26aulast%3DFraile-Ribot%26aufirst%3DP.%2BA.%26aulast%3DZamorano%26aufirst%3DL.%26aulast%3DOrellana%26aufirst%3DR.%26aulast%3Ddel%2BBarrio-Tofi%25C3%25B1o%26aufirst%3DE.%26aulast%3DS%25C3%25A1nchez-Diener%26aufirst%3DI.%26aulast%3DCortes-Lara%26aufirst%3DS.%26aulast%3DL%25C3%25B3pez-Causap%25C3%25A9%26aufirst%3DC.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADnez%26aufirst%3DL.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DGal%25C3%25A1n%26aufirst%3DF.%26aulast%3DGracia%26aufirst%3DI.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DM.%2BA.%26aulast%3DMart%25C3%25ADn%26aufirst%3DL.%26aulast%3DS%25C3%25A1nchez%26aufirst%3DJ.%2BM.%26aulast%3DVi%25C3%25B1uela%26aufirst%3DL.%26aulast%3DGarc%25C3%25ADa%26aufirst%3DM.%2BV.%26aulast%3DLepe%26aufirst%3DJ.%2BA.%26aulast%3DAznar%26aufirst%3DJ.%26aulast%3DL%25C3%25B3pez-Hern%25C3%25A1ndez%26aufirst%3DI.%26aulast%3DSeral%26aufirst%3DC.%26aulast%3DCastillo-Garc%25C3%25ADa%26aufirst%3DF.%2BJ.%26aulast%3DL%25C3%25B3pez-Calleja%26aufirst%3DA.%2BI.%26aulast%3DAspiroz%26aufirst%3DC.%26aulast%3Dde%2Bla%2BIglesia%26aufirst%3DP.%26aulast%3DRam%25C3%25B3n%26aufirst%3DS.%26aulast%3DRiera%26aufirst%3DE.%26aulast%3DP%25C3%25A9rez%26aufirst%3DM.%2BC.%26aulast%3DGallegos%26aufirst%3DC.%26aulast%3DCalvo%26aufirst%3DJ.%26aulast%3DQuesada%26aufirst%3DM.%2BD.%26aulast%3DMarco%26aufirst%3DF.%26aulast%3DHoyos%26aufirst%3DY.%26aulast%3DHorcajada%26aufirst%3DJ.%2BP.%26aulast%3DLarrosa%26aufirst%3DN.%26aulast%3DJos%25C3%25A9%2BGonz%25C3%25A1lez%26aufirst%3DJ.%26aulast%3DTubau%26aufirst%3DF.%26aulast%3DCapilla%26aufirst%3DS.%26aulast%3DP%25C3%25A9rez-Moreno%26aufirst%3DM.%2BO.%26aulast%3DCentelles%26aufirst%3DM.%2BJ.%26aulast%3DPadilla%26aufirst%3DE.%26aulast%3DRivera%26aufirst%3DA.%26aulast%3DMirelis%26aufirst%3DB.%26aulast%3DRodr%25C3%25ADguez-Tarazona%26aufirst%3DR.%2BE.%26aulast%3DArenal-Andr%25C3%25A9s%26aufirst%3DN.%26aulast%3DOrtega%26aufirst%3DM.%2Bd.%2BP.%26aulast%3DMeg%25C3%25ADas%26aufirst%3DG.%26aulast%3DGarc%25C3%25ADa%26aufirst%3DI.%26aulast%3DColmenarejo%26aufirst%3DC.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DJ.%2BC.%26aulast%3DMart%25C3%25ADnez%26aufirst%3DN.%2BM.%26aulast%3DGomila%26aufirst%3DB.%26aulast%3DGiner%26aufirst%3DS.%26aulast%3DTormo%26aufirst%3DN.%26aulast%3DGardu%25C3%25B1o%26aufirst%3DE.%26aulast%3DAgulla%26aufirst%3DJ.%2BA.%26aulast%3DSeoane%26aufirst%3DA.%26aulast%3DPita%26aufirst%3DJ.%26aulast%3DVidal%26aufirst%3DI.%2BP.%26aulast%3DGuzm%25C3%25A1n%26aufirst%3DD.%2BM.%26aulast%3DGarc%25C3%25ADa%26aufirst%3DM.%26aulast%3DLuisa%2BP%25C3%25A9rez%2Bdel%2BMolino%26aufirst%3DM.%26aulast%3DBarbeito%26aufirst%3DG.%26aulast%3DArtiles%26aufirst%3DF.%26aulast%3DAzcona-Guti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DS%25C3%25A1enz%26aufirst%3DY.%26aulast%3DAntonio%2BOteo%26aufirst%3DJ.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DA.%26aulast%3DVilla%26aufirst%3DJ.%26aulast%3DChaves%26aufirst%3DF.%26aulast%3DCercenado%26aufirst%3DE.%26aulast%3DAlarc%25C3%25B3n%26aufirst%3DT.%26aulast%3DZurita%26aufirst%3DN.%2BD.%26aulast%3DMerino%26aufirst%3DI.%26aulast%3DMorosini%26aufirst%3DM.%2BI.%26aulast%3DCant%25C3%25B3n%26aufirst%3DR.%26aulast%3DS%25C3%25A1nchez%26aufirst%3DM.%2BI.%26aulast%3DMoreno%26aufirst%3DL.%26aulast%3DYag%25C3%25BCe%26aufirst%3DG.%26aulast%3DLeiva%26aufirst%3DJ.%26aulast%3DBarrios%26aufirst%3DJ.%2BL.%26aulast%3DCanut%26aufirst%3DA.%26aulast%3DOteo%26aufirst%3DJ.%26atitle%3DActivity%2520of%2520Imipenem-Relebactam%2520against%2520a%2520Large%2520Collection%2520of%2520Pseudomonas%2520aeruginosa%2520Clinical%2520Isolates%2520and%2520Isogenic%2520%25CE%25B2-Lactam-Resistant%2520Mutants%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.02165-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyá, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Stepwise Upregulation of the Pseudomonas aeruginosa Chromosomal Cephalosporinase Conferring High-Level β-Lactam Resistance Involves Three AmpD Homologues</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1780</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1128/aac.50.5.1780-1787.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.50.5.1780-1787.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=16641450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD28XksVKktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2006&pages=1780-1787&author=C.+Juanauthor=B.+Moy%C3%A1author=J.+L.+P%C3%A9rezauthor=A.+Oliver&title=Stepwise+Upregulation+of+the+Pseudomonas+aeruginosa+Chromosomal+Cephalosporinase+Conferring+High-Level+%CE%B2-Lactam+Resistance+Involves+Three+AmpD+Homologues&doi=10.1128%2Faac.50.5.1780-1787.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Stepwise upregulation of the Pseudomonas aeruginosa chromosomal cephalosporinase conferring high-level β-lactam resistance involves three AmpD homologues</span></div><div class="casAuthors">Juan, Carlos; Moya, Bartolome; Perez, Jose L.; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1780-1787</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Development of resistance to the antipseudomonal penicillins and cephalosporins mediated by hyperprodn. of the chromosomal cephalosporinase AmpC is a major threat to the successful treatment of Pseudomonas aeruginosa infections.  Although ampD inactivation has been previously found to lead to a partially derepressed phenotype characterized by increased AmpC prodn. but retaining further inducibility, the regulation of ampC in P. aeruginosa is far from well understood.  The authors show that ampC expression is coordinately repressed by three AmpD homologues, including the previously described protein AmpD plus two addnl. proteins, designated AmpDh2 and AmpDh3.  The three AmpD homologues are responsible for a stepwise ampC upregulation mechanism ultimately leading to constitutive hyperexpression of the chromosomal cephalosporinase and high-level antipseudomonal β-lactam resistance, as shown by anal. of the three single ampD mutants, the three double ampD mutants, and the triple ampD mutant.  This is achieved by a three-step escalating mechanism rendering four relevant expression states: basal-level inducible expression (wild type), moderate-level hyperinducible expression with increased antipseudomonal β-lactam resistance (ampD mutant), high-level hyperinducible expression with high-level β-lactam resistance (ampD ampDh3 double mutant), and very high-level (more than 1000-fold compared to the wild type) derepressed expression (triple mutant).  Although one-step inducible-derepressed expression models are frequent in natural resistance mechanisms, this is the first characterized example in which expression of a resistance gene can be sequentially amplified through multiple steps of derepression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVP1sDdfHIarVg90H21EOLACvtfcHk0lh9H8u8_lBpEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksVKktLs%253D&md5=9b9918303a1c8ec912788a606012b78d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1128%2FAAC.50.5.1780-1787.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.50.5.1780-1787.2006%26sid%3Dliteratum%253Aachs%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DMoy%25C3%25A1%26aufirst%3DB.%26aulast%3DP%25C3%25A9rez%26aufirst%3DJ.%2BL.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DStepwise%2520Upregulation%2520of%2520the%2520Pseudomonas%2520aeruginosa%2520Chromosomal%2520Cephalosporinase%2520Conferring%2520High-Level%2520%25CE%25B2-Lactam%2520Resistance%2520Involves%2520Three%2520AmpD%2520Homologues%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2006%26volume%3D50%26spage%3D1780%26epage%3D1787%26doi%3D10.1128%2Faac.50.5.1780-1787.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ropy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Diener, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayala, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Role of Pseudomonas aeruginosa Low-Molecular-Mass Penicillin-Binding Proteins in AmpC Expression, β-Lactam Resistance, and Peptidoglycan Structure</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3925</span>– <span class="NLM_lpage">3934</span>, <span class="refDoi"> DOI: 10.1128/aac.05150-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.05150-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25896695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFaqsrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=3925-3934&author=A.+Ropyauthor=G.+Cabotauthor=I.+S%C3%A1nchez-Dienerauthor=C.+Aguileraauthor=B.+Moyaauthor=J.+A.+Ayalaauthor=A.+Oliver&title=Role+of+Pseudomonas+aeruginosa+Low-Molecular-Mass+Penicillin-Binding+Proteins+in+AmpC+Expression%2C+%CE%B2-Lactam+Resistance%2C+and+Peptidoglycan+Structure&doi=10.1128%2Faac.05150-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Pseudomonas aeruginosa low-molecular-mass penicillin-binding proteins in AmpC expression, β-lactam resistance, and peptidoglycan structure</span></div><div class="casAuthors">Ropy, Alaa; Cabot, Gabriel; Sanchez-Diener, Irina; Aguilera, Cristian; Moya, Bartolome; Ayala, Juan A.; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3925-3934</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">This study aimed to characterize the role of Pseudomonas aeruginosa low-mol.-mass penicillin-binding proteins (LMM PBPs), namely, PBP4 (DacB), PBP5 (DacC), and PBP7 (PbpG), in peptidoglycan compn., β-lactam resistance, and ampC regulation.  For this purpose, we constructed all single and multiple mutants of dacB, dacC, pbpG, and ampC from the wild-type P. aeruginosa PAO1 strain.  Peptidoglycan compn. was detd. by high-performance liq. chromatog. (HPLC), ampC expression by reverse transcription-PCR (RT-PCR), PBP patterns by a Bocillin FL-binding test, and antimicrobial susceptibility by MIC testing for a panel of β-lactams.  Microscopy and growth rate analyses revealed no apparent major morphol. changes for any of the mutants compared to the wild-type PAO1 strain.  Of the single mutants, only dacC mutation led to significantly increased pentapeptide levels, showing that PBP5 is the major DD-carboxypeptidase in P. aeruginosa.  Moreover, our results indicate that PBP4 and PBP7 play a significant role as DD-carboxypeptidase only if PBP5 is absent, and their DD-endopeptidase activity is also inferred.  As expected, the inactivation of PBP4 led to a significant increase in ampC expression (∼50-fold), but, remarkably, the sequential inactivation of the 3 LMM PBPs produced a much greater increase (1000-fold), which correlated with peptidoglycan pentapeptide levels.  Finally, the β-lactam susceptibility profiles of the LMM PBP mutants correlated well with the ampC expression data.  However, the inactivation of ampC in these mutants also evidenced a role of LMM PBPs, esp. PBP5, in intrinsic β-lactam resistance.  In summary, in addn. to assessing the effect of P. aeruginosa LMM PBPs on peptidoglycan structure for the 1st time, we obtained results that represent a step forward in understanding the impact of these PBPs on β-lactam resistance, apparently driven by the interplay between their roles in AmpC induction, β-lactam trapping, and DD-carboxypeptidase/β-lactamase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOfdZaUU7aLVg90H21EOLACvtfcHk0lixoYTBMuHHJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFaqsrvE&md5=531776c4f79ac78545e7f0ff6b121f33</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1128%2FAAC.05150-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.05150-14%26sid%3Dliteratum%253Aachs%26aulast%3DRopy%26aufirst%3DA.%26aulast%3DCabot%26aufirst%3DG.%26aulast%3DS%25C3%25A1nchez-Diener%26aufirst%3DI.%26aulast%3DAguilera%26aufirst%3DC.%26aulast%3DMoya%26aufirst%3DB.%26aulast%3DAyala%26aufirst%3DJ.%2BA.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DRole%2520of%2520Pseudomonas%2520aeruginosa%2520Low-Molecular-Mass%2520Penicillin-Binding%2520Proteins%2520in%2520AmpC%2520Expression%252C%2520%25CE%25B2-Lactam%2520Resistance%252C%2520and%2520Peptidoglycan%2520Structure%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D3925%26epage%3D3934%26doi%3D10.1128%2Faac.05150-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Llarena, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Cuenca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gude, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oviaño, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachón, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnacho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascual, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span> <span> </span><span class="NLM_article-title">High incidence of MDR and XDR <i>Pseudomonas aeruginosa</i> isolates obtained from patients with ventilator-associated pneumonia in Greece, Italy and Spain as part of the magicBullet clinical trial</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1244</span>– <span class="NLM_lpage">1252</span>, <span class="refDoi"> DOI: 10.1093/jac/dkz030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkz030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=30753505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlansbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2019&pages=1244-1252&author=A.+P%C3%A9rezauthor=E.+Gatoauthor=J.+P%C3%A9rez-Llarenaauthor=F.+Fern%C3%A1ndez-Cuencaauthor=M.+J.+Gudeauthor=M.+Ovia%C3%B1oauthor=M.+E.+Pach%C3%B3nauthor=J.+Garnachoauthor=V.+Gonz%C3%A1lezauthor=%C3%81.+Pascualauthor=J.+M.+Cisnerosauthor=G.+Bou&title=High+incidence+of+MDR+and+XDR+Pseudomonas+aeruginosa+isolates+obtained+from+patients+with+ventilator-associated+pneumonia+in+Greece%2C+Italy+and+Spain+as+part+of+the+magicBullet+clinical+trial&doi=10.1093%2Fjac%2Fdkz030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">High incidence of MDR and XDR Pseudomonas aeruginosa isolates obtained from patients with ventilator-associated pneumonia in Greece, Italy and Spain as part of the MagicBullet clinical trial</span></div><div class="casAuthors">Perez, Astrid; Gato, Eva; Perez-Llarena, Jose; Fernandez-Cuenca, Felipe; Gude, Maria Jose; Oviano, Marina; Pachon, Maria Eugenia; Garnacho, Jose; Gonzalez, Veronica; Pascual, Alvaro; Cisneros, Jose Miguel; Bou, German</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1244-1252</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">1460-2091</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives: To characterize the antimicrobial susceptibility, mol. epidemiol. and carbapenem resistance mechanisms in Pseudomonas aeruginosa isolates recovered from respiratory tract samples from patients with ventilator-assocd. pneumonia enrolled in the MagicBullet clin. trial.  Methods: Isolates were collected from 53 patients from 12 hospitals in Spain, Italy and Greece.  Susceptibility was detd. using broth microdilution and Etest.  MALDI-TOF MS was used to detect carbapenemase activity and carbapenemases were identified by PCR and sequencing.  Mol. epidemiol. was investigated using PFGE and MLST.  Results: Of the 53 isolates, 2 (3.8%) were considered pandrug resistant (PDR), 19 (35.8%) were XDR and 16 (30.2%) were MDR.  Most (88.9%) of the isolates from Greece were MDR, XDR or PDR, whereas fewer of the isolates from Spain (33.3%) and Italy (43.5%) showed antibiotic resistance.  Three Greek isolates were resistant to colistin.  Overall, the rates of resistance of P. aeruginosa isolates to imipenem, ciprofloxacin, ceftolozane/tazobactam and ceftazidime/avibactam were 64.1%, 54.7%, 22.6% and 24.5%, resp.  All isolates resistant to ceftolozane/tazobactam and ceftazidime/avibactam (Greece, n = 10; and Italy, n = 2) carried blaVIM-2.  Spanish isolates were susceptible to the new drug combinations.  Forty-eight restriction patterns and 27 STs were documented.  Sixty percent of isolates belonged to six STs, including the high-risk clones ST-111, ST-175 and ST-235.  Conclusions: MDR/XDR isolates were highly prevalent, particularly in Greece.  The most effective antibiotic against P. aeruginosa was colistin, followed by ceftolozane/tazobactam and ceftazidime/avibactam. blaVIM-2 is assocd. with resistance to ceftolozane/tazobactam and ceftazidime/avibactam, and related to highly resistant phenotypes.  ST-111 was the most frequent and disseminated clone and the clonal diversity was lower in XDR and PDR strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3pwTl_mp9eLVg90H21EOLACvtfcHk0lixoYTBMuHHJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlansbrN&md5=1dc8fe08d731ecb7f61cf0244e7a3f71</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkz030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkz030%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez%26aufirst%3DA.%26aulast%3DGato%26aufirst%3DE.%26aulast%3DP%25C3%25A9rez-Llarena%26aufirst%3DJ.%26aulast%3DFern%25C3%25A1ndez-Cuenca%26aufirst%3DF.%26aulast%3DGude%26aufirst%3DM.%2BJ.%26aulast%3DOvia%25C3%25B1o%26aufirst%3DM.%26aulast%3DPach%25C3%25B3n%26aufirst%3DM.%2BE.%26aulast%3DGarnacho%26aufirst%3DJ.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DV.%26aulast%3DPascual%26aufirst%3D%25C3%2581.%26aulast%3DCisneros%26aufirst%3DJ.%2BM.%26aulast%3DBou%26aufirst%3DG.%26atitle%3DHigh%2520incidence%2520of%2520MDR%2520and%2520XDR%2520Pseudomonas%2520aeruginosa%2520isolates%2520obtained%2520from%2520patients%2520with%2520ventilator-associated%2520pneumonia%2520in%2520Greece%252C%2520Italy%2520and%2520Spain%2520as%2520part%2520of%2520the%2520magicBullet%2520clinical%2520trial%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2019%26volume%3D74%26spage%3D1244%26epage%3D1252%26doi%3D10.1093%2Fjac%2Fdkz030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beceiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albertí, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">Characterization of the New Metallo-β-Lactamase VIM-13 and Its Integron-Borne Gene from a Pseudomonas aeruginosa Clinical Isolate in Spain</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3589</span>– <span class="NLM_lpage">3596</span>, <span class="refDoi"> DOI: 10.1128/aac.00465-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.00465-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=18644957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1SrtbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=3589-3596&author=C.+Juanauthor=A.+Beceiroauthor=O.+Guti%C3%A9rrezauthor=S.+Albert%C3%ADauthor=M.+Garauauthor=J.+L.+P%C3%A9rezauthor=G.+Bouauthor=A.+Oliver&title=Characterization+of+the+New+Metallo-%CE%B2-Lactamase+VIM-13+and+Its+Integron-Borne+Gene+from+a+Pseudomonas+aeruginosa+Clinical+Isolate+in+Spain&doi=10.1128%2Faac.00465-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the new metallo-β-lactamase VIM-13 and its integron-borne gene from a Pseudomonas aeruginosa clinical isolate in Spain</span></div><div class="casAuthors">Juan, Carlos; Beceiro, Alejandro; Gutierrez, Olivia; Alberti, Sebastian; Garau, Margalida; Perez, Jose L.; Bou, German; Oliver, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3589-3596</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">During a survey conducted to evaluate the incidence of class B carbapenemase (metallo-β-lactamase [MBL])-producing Pseudomonas aeruginosa strains from hospitals in Majorca, Spain, five clin. isolates showed a pos. Etest MBL screening test result.  In one of them, strain PA-SL2, the presence of a new blaVIM deriv. (blaVIM-13) was detected by PCR amplification with blaVIM-1-specific primers followed by sequencing.  The blaVIM-13-producing isolate showed resistance to all β-lactams (except aztreonam), gentamicin, tobramycin, and ciprofloxacin.  VIM-13 exhibited 93% and 88% amino acid sequence identities with VIM-1 and VIM-2, resp.  BlaVIM-13 was cloned in parallel with blaVIM-1, and the resistance profile conferred was analyzed both in Escherichia coli and in P. aeruginosa backgrounds.  Compared to VIM-1, VIM-13 conferred slightly higher levels of resistance to piperacillin and lower levels of resistance to ceftazidime and cefepime.  VIM-13 and VIM-1 were purified in parallel as well, and their kinetic parameters were compared.  The kcat/Km ratios for the antibiotics mentioned above were in good agreement with the MIC data.  Furthermore, EDTA inhibited the activity of VIM-13 approx. 25 times less than it inhibited the activity of VIM-1.  VIM-13 was harbored in a class 1 integron, along with a new variant (Ala108Thr) of the aminoglycoside-modifying enzyme encoding gene aacA4, which confers resistance to gentamicin and tobramycin.  Finally, the VIM-13 integron was apparently located in the chromosome, since transformation and conjugation expts. consistently yielded neg. results and the blaVIM-13 probe hybridized only with the genomic DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR3E6XP7Vu2LVg90H21EOLACvtfcHk0lixoYTBMuHHJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1SrtbnL&md5=5c4aae46d7f3aea128c0b2edcbf14c05</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1128%2FAAC.00465-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00465-08%26sid%3Dliteratum%253Aachs%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DBeceiro%26aufirst%3DA.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DO.%26aulast%3DAlbert%25C3%25AD%26aufirst%3DS.%26aulast%3DGarau%26aufirst%3DM.%26aulast%3DP%25C3%25A9rez%26aufirst%3DJ.%2BL.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3DCharacterization%2520of%2520the%2520New%2520Metallo-%25CE%25B2-Lactamase%2520VIM-13%2520and%2520Its%2520Integron-Borne%2520Gene%2520from%2520a%2520Pseudomonas%2520aeruginosa%2520Clinical%2520Isolate%2520in%2520Spain%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2008%26volume%3D52%26spage%3D3589%26epage%3D3596%26doi%3D10.1128%2Faac.00465-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taracila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hujer, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatta, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desarbre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">A kinetic analysis of the inhibition of FOX-4 -lactamase, a plasmid-mediated AmpC cephalosporinase, by monocyclic -lactams and carbapenems</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">682</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1093/jac/dkt434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fjac%2Fdkt434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=24235094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisF2lsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=682-690&author=K.+M.+Papp-Wallaceauthor=S.+Malloauthor=C.+R.+Bethelauthor=M.+A.+Taracilaauthor=A.+M.+Hujerauthor=A.+Fernandezauthor=J.+A.+Gattaauthor=K.+M.+Smithauthor=Y.+Xuauthor=M.+G.+P.+Pageauthor=E.+Desarbreauthor=G.+Bouauthor=R.+A.+Bonomo&title=A+kinetic+analysis+of+the+inhibition+of+FOX-4+-lactamase%2C+a+plasmid-mediated+AmpC+cephalosporinase%2C+by+monocyclic+-lactams+and+carbapenems&doi=10.1093%2Fjac%2Fdkt434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A kinetic analysis of the inhibition of FOX-4 β-lactamase, a plasmid-mediated AmpC cephalosporinase, by monocyclic β-lactams and carbapenems</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.; Mallo, Susana; Bethel, Christopher R.; Taracila, Magdalena A.; Hujer, Andrea M.; Fernandez, Ana; Gatta, Julian A.; Smith, Kerri M.; Xu, Yan; Page, Malcolm G. P.; Desarbre, Eric; Bou, German; Bonomo, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">682-690</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Class C β-lactamases are prevalent among Enterobacteriaceae; however, these enzymes are resistant to inactivation by com. available β-lactamase inhibitors.  In order to find novel scaffolds to inhibit class C β-lactamases, the comparative efficacy of monocyclic β-lactam antibiotics (aztreonam and the siderophore monosulfactam BAL 30072), the bridged monobactam β-lactamase inhibitor BAL 29880, and carbapenems (imipenem, meropenem, doripenem, and ertapenem) were tested in kinetic assays against FOX-4, a plasmid-mediated class C β-lactamase (pmAmpC).  The FOX-4 β-lactamase was purified.  Steady-state kinetics, electrospray ionization mass spectrometry (ESI-MS) and UV difference (UVD) spectroscopy were conducted using the β-lactam scaffolds described.  The Ki values for the monocyclic β-lactams against FOX-4 β-lactamase were 0.04 μM (aztreonam) and 0.66 μM (BAL 30072), and the Ki for bridged monobactam BAL 29880 was 8.9 μM.  For carbapenems, the Ki values ranged from 0.27 μM (ertapenem) to 2.3 μM (imipenem).  ESI-MS demonstrated the formation of stable covalent adducts when the monocyclic β-lactams and carbapenems were reacted with FOX-4 β-lactamase.  UVD spectroscopy suggested the appearance of different chromophoric intermediates.  Thus, monocyclic β-lactam and carbapenem antibiotics are effective mechanism-based inhibitors of FOX-4 β-lactamase, a clin. important pmAmpC, and provide stimulus for the development of new inhibitors to inactivate plasmidic and chromosomal class C β-lactamases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvesH3hBGh9bVg90H21EOLACvtfcHk0lgSou1cOh-eSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisF2lsbs%253D&md5=e05484b884e8c8beb6f99e24c8238dd1</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkt434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkt434%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DMallo%26aufirst%3DS.%26aulast%3DBethel%26aufirst%3DC.%2BR.%26aulast%3DTaracila%26aufirst%3DM.%2BA.%26aulast%3DHujer%26aufirst%3DA.%2BM.%26aulast%3DFernandez%26aufirst%3DA.%26aulast%3DGatta%26aufirst%3DJ.%2BA.%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DPage%26aufirst%3DM.%2BG.%2BP.%26aulast%3DDesarbre%26aufirst%3DE.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DA%2520kinetic%2520analysis%2520of%2520the%2520inhibition%2520of%2520FOX-4%2520-lactamase%252C%2520a%2520plasmid-mediated%2520AmpC%2520cephalosporinase%252C%2520by%2520monocyclic%2520-lactams%2520and%2520carbapenems%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D682%26epage%3D690%26doi%3D10.1093%2Fjac%2Fdkt434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hujer, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domitrovic, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hujer, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurless, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuohy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">Unexpected challenges in treating multidrug-resistant Gram-negative bacteria: resistance to ceftazidime-avibactam in archived isolates of <i>Pseudomonas aeruginosa</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1020</span>– <span class="NLM_lpage">1029</span>, <span class="refDoi"> DOI: 10.1128/aac.04238-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.04238-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25451057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1KrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=1020-1029&author=M.+L.+Winklerauthor=K.+M.+Papp-Wallaceauthor=A.+M.+Hujerauthor=T.+N.+Domitrovicauthor=K.+M.+Hujerauthor=K.+N.+Hurlessauthor=M.+Tuohyauthor=G.+Hallauthor=R.+A.+Bonomo&title=Unexpected+challenges+in+treating+multidrug-resistant+Gram-negative+bacteria%3A+resistance+to+ceftazidime-avibactam+in+archived+isolates+of+Pseudomonas+aeruginosa&doi=10.1128%2Faac.04238-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Unexpected challenges in treating multidrug-resistant gram-negative bacteria: resistance to ceftazidime-avibactam in archived isolates of Pseudomonas aeruginosa</span></div><div class="casAuthors">Winkler, Marisa L.; Papp-Wallace, Krisztina M.; Hujer, Andrea M.; Domitrovic, T. Nicholas; Hujer, Kristine M.; Hurless, Kelly N.; Tuohy, Marion; Hall, Geraldine; Bonomo, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1020-1029, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is a notoriously difficult-to-treat pathogen that is a common cause of severe nosocomial infections.  Studying a collection of β-lactam-resistant P. aeruginosa clin. isolates from a decade ago, the authors uncovered resistance to ceftazidime-avibactam, a novel β-lactam/β-lactamase inhibitor combination.  The isolates were systematically analyzed through a variety of genetic, biochem., genomic, and microbiol. methods to understand how resistance manifests to a unique drug combination that is not yet clin. released.  Avibactam was able to inactivate different AmpC β-lactamase enzymes and blaPDC regulatory elements and penicillin-binding protein differences did not contribute in a major way to resistance.  By using carefully selected combinations of antimicrobial agents, the greatest barrier to ceftazidime-avibactam was shown to be membrane permeability and drug efflux.  To overcome the constellation of resistance determinants, it was shown that a combination of antimicrobial agents (ceftazidime/avibactam/fosfomycin) targeting multiple cell wall synthetic pathways can restore susceptibility.  In P. aeruginosa, efflux, as a general mechanism of resistance, may pose the greatest challenge to future antibiotic development.  The authors' unexpected findings create concern that even the development of antimicrobial agents targeted for the treatment of multidrug-resistant bacteria may encounter clin. important resistance.  Antibiotic therapy in the future must consider these factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT4V9qO1xrZrVg90H21EOLACvtfcHk0lgSou1cOh-eSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1KrsLY%253D&md5=724fd9db8243065976feb01f6aff8978</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1128%2FAAC.04238-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.04238-14%26sid%3Dliteratum%253Aachs%26aulast%3DWinkler%26aufirst%3DM.%2BL.%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DHujer%26aufirst%3DA.%2BM.%26aulast%3DDomitrovic%26aufirst%3DT.%2BN.%26aulast%3DHujer%26aufirst%3DK.%2BM.%26aulast%3DHurless%26aufirst%3DK.%2BN.%26aulast%3DTuohy%26aufirst%3DM.%26aulast%3DHall%26aufirst%3DG.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DUnexpected%2520challenges%2520in%2520treating%2520multidrug-resistant%2520Gram-negative%2520bacteria%253A%2520resistance%2520to%2520ceftazidime-avibactam%2520in%2520archived%2520isolates%2520of%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D1020%26epage%3D1029%26doi%3D10.1128%2Faac.04238-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp-Wallace, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taracila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">Variants of β-Lactamase KPC-2 That Are Resistant to Inhibition by Avibactam</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3710</span>– <span class="NLM_lpage">3717</span>, <span class="refDoi"> DOI: 10.1128/aac.04406-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2FAAC.04406-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=25666153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFaru7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=3710-3717&author=K.+M.+Papp-Wallaceauthor=M.+L.+Winklerauthor=M.+A.+Taracilaauthor=R.+A.+Bonomo&title=Variants+of+%CE%B2-Lactamase+KPC-2+That+Are+Resistant+to+Inhibition+by+Avibactam&doi=10.1128%2Faac.04406-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Variants of β-lactamase KPC-2 that are resistant to inhibition by avibactam</span></div><div class="casAuthors">Papp-Wallace, Krisztina M.; Winkler, Marisa L.; Taracila, Magdalena A.; Bonomo, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3710-3717</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">KPC-2 is the most prevalent class A carbapenemase in the world.  Previously, KPC-2 was shown to hydrolyze the β-lactamase inhibitors clavulanic acid, sulbactam, and tazobactam.  In addn., substitutions at amino acid position R220 in the KPC-2 β-lactamase increased resistance to clavulanic acid.  A novel bridged diazabicyclooctane (DBO) non-β-lactam β-lactamase inhibitor, avibactam, was shown to inactivate the KPC-2 β-lactamase.  To better understand the mechanistic basis for inhibition of KPC-2 by avibactam, we tested the potency of ampicillin-avibactam and ceftazidime-avibactam against engineered variants of the KPC-2 β-lactamase that possessed single amino acid substitutions at important sites (i.e., Ambler positions 69, 130, 234, 220, and 276) that were previously shown to confer inhibitor resistance in TEM and SHV β-lactamases.  To this end, we performed susceptibility testing, biochem. assays, and mol. modeling.  Escherichia coli DH10B carrying KPC-2 β-lactamase variants with the substitutions S130G, K234R, and R220M demonstrated elevated MICs for only the ampicillin-avibactam combinations (e.g., 512, 64, and 32 mg/L, resp., vs. the MICs for wild-type KPC-2 at 2 to 8 mg/L).  Steady-state kinetics revealed that the S130G variant of KPC-2 resisted inactivation by avibactam; the k2/K ratio was significantly lowered 4 logs for the S130G variant from the ratio for the wild-type enzyme (21,580 M-1 s-1 to 1.2 M-1 s-1).  Mol. modeling and mol. dynamics simulations suggested that the mobility of K73 and its ability to activate S70 (i.e., function as a general base) may be impaired in the S130G variant of KPC-2, thereby explaining the slowed acylation.  Moreover, we also advance the idea that the protonation of the sulfate nitrogen of avibactam may be slowed in the S130G variant, as S130 is the likely proton donor and another residue, possibly K234, must compensate.  Our findings show that residues S130 as well as K234 and R220 contribute significantly to the mechanism of avibactam inactivation of KPC-2.  Fortunately, the emergence of S130G, K234R, and R220M variants of KPC in the clinic should not result in failure of ceftazidime-avibactam, as the ceftazidime partner is potent against E. coli DH10B strains possessing all of these variants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUr5WTh7ibNrVg90H21EOLACvtfcHk0lgSou1cOh-eSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFaru7zP&md5=16ac1f44011c90f8356740bee6e5bd9c</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1128%2FAAC.04406-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.04406-14%26sid%3Dliteratum%253Aachs%26aulast%3DPapp-Wallace%26aufirst%3DK.%2BM.%26aulast%3DWinkler%26aufirst%3DM.%2BL.%26aulast%3DTaracila%26aufirst%3DM.%2BA.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DVariants%2520of%2520%25CE%25B2-Lactamase%2520KPC-2%2520That%2520Are%2520Resistant%2520to%2520Inhibition%2520by%2520Avibactam%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D3710%26epage%3D3717%26doi%3D10.1128%2Faac.04406-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsivkovski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span> <span> </span><span class="NLM_article-title">Biochemical activity of vaborbactam</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>, <span class="NLM_elocation-id">e01935</span> <span class="refDoi"> DOI: 10.1128/aac.01935-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1128%2Faac.01935-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=31712199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktFyhtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&author=R.+Tsivkovskiauthor=O.+Lomovskaya&title=Biochemical+activity+of+vaborbactam&doi=10.1128%2Faac.01935-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical activity of vaborbactam</span></div><div class="casAuthors">Tsivkovski, Ruslan; Lomovskaya, Olga</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e01935</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The most common mechanism of resistance to β-lactams antibiotics in Gram-neg. bacteria is prodn. of β-lactamase enzymes capable of cleaving the β-lactam ring.  Inhibition of β-lactamase activity with small-mol. drugs is a proven strategy to restore the potency of many β-lactam antibiotics.  Vaborbactam (formerly RPX7009) is a cyclic boronic acid β-lactamase inhibitor (BLI) with a broad spectrum of activity against various serine β-lactamases, including KPC carbapenemases.  The combination of vaborbactam and meropenem is approved in the United States and Europe for the treatment of various nosocomial infections.  We attempted to gain more insight into the mechanism of action of vaborbactam by conducting detailed kinetic characterization of its interaction with various recombinant His-tagged β-lactamases.  Vaborbactam demonstrated potent inhibition of class A and class C enzymes with Ki values ranging from 0.022 to 0.18 μM, while inhibition of class D enzymes was rather poor, and no activity against class B β-lactamases was detected.  Importantly, vaborbactam inhibited KPC-2, KPC-3, BKC-1, and SME-2 carbapenemases at 1:1 stoichiometry, while these nos. were higher for other class A and C enzymes.  Vaborbactam was also shown to be a potent progressive inactivator of several enzymes, including KPCs with inactivation consts. k2/K in the range of 3.4 × 103 to 2.4 × 104 M-1 s-1.  Finally, expts. on the recovery of enzyme activity demonstrated the high stability of the vaborbactam-KPC complex, with 0.000040 s-1koff values and a corresponding residence time of 7 h, whereas the release of vaborbactam bound to other serine β-lactamases was substantially faster.  The biochem. characteristics of vaborbactam described in this study may facilitate further chem. optimization efforts to develop boronic BLIs with improved affinity and broader spectrum of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqct6STwjYA67Vg90H21EOLACvtfcHk0lhGbUBAZZI5hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktFyhtbg%253D&md5=53e588b395cfb868f0a5f8699427c05a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1128%2Faac.01935-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01935-19%26sid%3Dliteratum%253Aachs%26aulast%3DTsivkovski%26aufirst%3DR.%26aulast%3DLomovskaya%26aufirst%3DO.%26atitle%3DBiochemical%2520activity%2520of%2520vaborbactam%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26doi%3D10.1128%2Faac.01935-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Vázquez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sola-Campoy, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurita, Á. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ávila, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-bertomeu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solís, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-urrutia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gónzalez-barberá, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cercenado, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bautista, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aracil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oteo-Iglesias, J.</span></span> <span> </span><span class="NLM_article-title">Carbapenemase-producing <i>Pseudomonas aeruginosa</i> in Spain: interregional dissemination of the high-risk clones ST175 and ST244 carrying blaVIM-2, blaVIM-1, blaIMP-8, blaVIM-20 and blaKPC-2</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">106026</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2020.106026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.ijantimicag.2020.106026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=32450200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFOisLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2020&pages=106026&author=M.+P%C3%A9rez-V%C3%A1zquezauthor=P.+J.+Sola-Campoyauthor=%C3%81.+M.+Zuritaauthor=A.+%C3%81vilaauthor=F.+G%C3%B3mez-bertomeuauthor=S.+Sol%C3%ADsauthor=L.+L%C3%B3pez-urrutiaauthor=E.+M.+G%C3%B3nzalez-barber%C3%A1author=E.+Cercenadoauthor=V.+Bautistaauthor=N.+Laraauthor=B.+Aracilauthor=A.+Oliverauthor=J.+Camposauthor=J.+Oteo-Iglesias&title=Carbapenemase-producing+Pseudomonas+aeruginosa+in+Spain%3A+interregional+dissemination+of+the+high-risk+clones+ST175+and+ST244+carrying+blaVIM-2%2C+blaVIM-1%2C+blaIMP-8%2C+blaVIM-20+and+blaKPC-2&doi=10.1016%2Fj.ijantimicag.2020.106026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Carbapenemase-producing Pseudomonas aeruginosa in Spain: interregional dissemination of the high-risk clones ST175 and ST244 carrying blaVIM-2, blaVIM-1, blaIMP-8, blaVIM-20 and blaKPC-2</span></div><div class="casAuthors">Perez-Vazquez, Maria; Sola-Campoy, Pedro J.; Zurita, Angela Maria; Avila, Alicia; Gomez-Bertomeu, Frederic; Solis, Sonia; Lopez-Urrutia, Luis; Gonzalez-Barbera, Eva M.; Cercenado, Emilia; Bautista, Veronica; Lara, Noelia; Aracil, Belen; Oliver, Antonio; Campos, Jose; Oteo-Iglesias, Jesus</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">106026</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Carbapenemase-producing (CP) Pseudomonas aeruginosa is rare compared with mutation-driven carbapenem-resistance, but this situation may be changing.  A collection of CP P. aeruginosa isolates was characterized in this study.  In 2016, 232 unduplicated carbapenem-resistant P. aeruginosa isolates, of which 71 (30.6%) carried carbapenemase genes, were submitted to the Spanish antibiotic ref. lab. and were further analyzed by whole-genome sequencing (WGS).  Of the 71 CP P. aeruginosa, 39 (54.9%) carried blaVIM-2, 14 (19.7%) blaVIM-1, 8 (11.3%) blaIMP-8, 6 (8.5%) blaVIM-20, 2 (2.8%) blaVIM-2 plus blaKPC-2, one (1.4%) blaIMP-13 and one (1.4%) blaVIM-1 plus blaIMP-18.  Four sequence types (ST175, ST244, ST815 and ST155) encompassed 83.1% of the 71 CP P. aeruginosa; ST175 was detected in hospitals from seven provinces.  Using core genome multilocus sequence typing (cgMLST), four clusters were detected: Cluster 1 included nine ST815/VIM-2 isolates; Cluster 2 included five ST175/VIM-2 isolates; Cluster 3 included seven ST244 isolates (five VIM-2 and two VIM-2 plus KPC-2); and Cluster 4 included 11 ST175 isolates (seven VIM-2 and four IMP-8).  The av. no. of acquired resistance genes was significantly higher in the blaVIM-1-carying isolates (7.1 ± 0.94) than in the blaVIM-2-carrying isolates (4.5 ± 0.20).CP P. aeruginosa isolates are spreading in Spain, mainly due to the dissemination of high-risk clones such as ST175 and ST244 producing VIM and IMP carbapenemases.  Emergence of CP P. aeruginosa is a cause of clin. and epidemiol. concern.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-N4m07Ndq4rVg90H21EOLACvtfcHk0lhGbUBAZZI5hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFOisLrP&md5=35cce4a9e87a0970e383d175932469da</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2020.106026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2020.106026%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-V%25C3%25A1zquez%26aufirst%3DM.%26aulast%3DSola-Campoy%26aufirst%3DP.%2BJ.%26aulast%3DZurita%26aufirst%3D%25C3%2581.%2BM.%26aulast%3D%25C3%2581vila%26aufirst%3DA.%26aulast%3DG%25C3%25B3mez-bertomeu%26aufirst%3DF.%26aulast%3DSol%25C3%25ADs%26aufirst%3DS.%26aulast%3DL%25C3%25B3pez-urrutia%26aufirst%3DL.%26aulast%3DG%25C3%25B3nzalez-barber%25C3%25A1%26aufirst%3DE.%2BM.%26aulast%3DCercenado%26aufirst%3DE.%26aulast%3DBautista%26aufirst%3DV.%26aulast%3DLara%26aufirst%3DN.%26aulast%3DAracil%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DA.%26aulast%3DCampos%26aufirst%3DJ.%26aulast%3DOteo-Iglesias%26aufirst%3DJ.%26atitle%3DCarbapenemase-producing%2520Pseudomonas%2520aeruginosa%2520in%2520Spain%253A%2520interregional%2520dissemination%2520of%2520the%2520high-risk%2520clones%2520ST175%2520and%2520ST244%2520carrying%2520blaVIM-2%252C%2520blaVIM-1%252C%2520blaIMP-8%252C%2520blaVIM-20%2520and%2520blaKPC-2%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2020%26volume%3D56%26spage%3D106026%26doi%3D10.1016%2Fj.ijantimicag.2020.106026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">European
Committee on Antimicrobial
Susceptibility Testing</span>. <span> </span><span class="NLM_article-title">Breakpoint
Tables for Interpretation of MICs and Zone Diameters. Version 9.0</span>. <span class="NLM_year">2019</span>, available: <a href="http://www.eucast.org/clinical_breakpoints/" class="extLink">http://www.eucast.org/clinical_breakpoints/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=European%0ACommittee+on+Antimicrobial%0ASusceptibility+Testing&title=Breakpoint%0ATables+for+Interpretation+of+MICs+and+Zone+Diameters.+Version+9.0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DBreakpoint%250ATables%2520for%2520Interpretation%2520of%2520MICs%2520and%2520Zone%2520Diameters.%2520Version%25209.0%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Montgomery,  Jr., J. A.</span>; <span class="NLM_string-name">Peralta, J. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M.</span>; <span class="NLM_string-name">Heyd, J. J.</span>; <span class="NLM_string-name">Brothers, E.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Knox, J. E.</span>; <span class="NLM_string-name">Cross, J. B.</span>; <span class="NLM_string-name">Bakken, V.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Jaramillo, J.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Stratmann, R. E.</span>; <span class="NLM_string-name">Yazyev, O.</span>; <span class="NLM_string-name">Austin, A. J.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Pomelli, C.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Voth, G. A.</span>; <span class="NLM_string-name">Salvador, P.</span>; <span class="NLM_string-name">Dannenberg, J. J.</span>; <span class="NLM_string-name">Dapprich, S.</span>; <span class="NLM_string-name">Daniels, A. D.</span>; <span class="NLM_string-name">Farkas, Ö.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Cioslowski, J.</span>; <span class="NLM_string-name">Fox, D. J.</span></span> <i>Gaussian 09</i>, Revision D.01; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Caricato&author=X.+Li&author=H.+P.+Hratchian&author=A.+F.+Izmaylov&author=J.+Bloino&author=G.+Zheng&author=J.+L.+Sonnenberg&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+Bearpark&author=J.+J.+Heyd&author=E.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=N.+Rega&author=J.+M.+Millam&author=M.+Klene&author=J.+E.+Knox&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=V.+G.+Zakrzewski&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=S.+Dapprich&author=A.+D.+Daniels&author=%C3%96.+Farkas&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=J.+Cioslowski&author=D.+J.+Fox&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><a href="http://upjv.q4md-forcefieldtools.org/RED/" class="extLink">http://upjv.q4md-forcefieldtools.org/RED/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fupjv.q4md-forcefieldtools.org%2FRED%2F+%28accessed+Feb+13%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dupradeau, F.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pigache, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaffran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savineau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grivel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosanski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span> <span> </span><span class="NLM_article-title">The R.E.D. tools: advances in RESP and ESP charge derivation and force field library building</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7821</span>– <span class="NLM_lpage">7839</span>, <span class="refDoi"> DOI: 10.1039/c0cp00111b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1039%2Fc0cp00111b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=20574571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=7821-7839&author=F.-Y.+Dupradeauauthor=A.+Pigacheauthor=T.+Zaffranauthor=C.+Savineauauthor=R.+Lelongauthor=N.+Grivelauthor=D.+Lelongauthor=W.+Rosanskiauthor=P.+Cieplak&title=The+R.E.D.+tools%3A+advances+in+RESP+and+ESP+charge+derivation+and+force+field+library+building&doi=10.1039%2Fc0cp00111b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The R.E.D. tools: advances in RESP and ESP charge derivation and force field library building</span></div><div class="casAuthors">Dupradeau, Francois-Yves; Pigache, Adrien; Zaffran, Thomas; Savineau, Corentin; Lelong, Rodolphe; Grivel, Nicolas; Lelong, Dimitri; Rosanski, Wilfried; Cieplak, Piotr</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">7821-7839</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Deriving at. charges and building a force field library for a new mol. are key steps when developing a force field required for conducting structural and energy-based anal. using mol. mechanics.  Derivation of popular RESP charges for a set of residues is a complex and error prone procedure because it depends on numerous input parameters.  To overcome these problems, the R.E.D.  Tools (RESP and ESP charge Derive, http://q4md-forcefieldtools.org/RED/) have been developed to perform charge derivation in an automatic and straightforward way.  The R.E.D. program handles chem. elements up to bromine in the periodic table.  It interfaces different quantum mech. programs employed for geometry optimization and computing mol. electrostatic potential(s), and performs charge fitting using the RESP program.  By defining tight optimization criteria and by controlling the mol. orientation of each optimized geometry, charge values are reproduced at any computer platform with an accuracy of 0.0001 e.  The charges can be fitted using multiple conformations, making them suitable for mol. dynamics simulations.  R.E.D. allows also for defining charge constraints during multiple mol. charge fitting, which are used to derive charges for mol. fragments.  Finally, R.E.D. incorporates charges into a force field library, readily usable in mol. dynamics computer packages.  For complex cases, such as a set of homologous mols. belonging to a common family, an entire force field topol. database is generated.  Currently, the at. charges and force field libraries have been developed for more than fifty model systems and stored in the RESP ESP charge DDataBase.  Selected results related to non-polarizable charge models are presented and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxiXPn3YLNM7Vg90H21EOLACvtfcHk0lg9se9u5Js21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrk%253D&md5=845cc10ea7ed2f0222361db82bd80e39</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2Fc0cp00111b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cp00111b%26sid%3Dliteratum%253Aachs%26aulast%3DDupradeau%26aufirst%3DF.-Y.%26aulast%3DPigache%26aufirst%3DA.%26aulast%3DZaffran%26aufirst%3DT.%26aulast%3DSavineau%26aufirst%3DC.%26aulast%3DLelong%26aufirst%3DR.%26aulast%3DGrivel%26aufirst%3DN.%26aulast%3DLelong%26aufirst%3DD.%26aulast%3DRosanski%26aufirst%3DW.%26aulast%3DCieplak%26aufirst%3DP.%26atitle%3DThe%2520R.E.D.%2520tools%253A%2520advances%2520in%2520RESP%2520and%2520ESP%2520charge%2520derivation%2520and%2520force%2520field%2520library%2520building%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D7821%26epage%3D7839%26doi%3D10.1039%2Fc0cp00111b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanquelef, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquant, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimerak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delepine, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupradeau, F.-Y.</span></span> <span> </span><span class="NLM_article-title">R.E.D. Server: a web service for deriving RESP and ESP charges and building force field libraries for new molecules and molecular fragments</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">W511</span>– <span class="NLM_lpage">W517</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fnar%2Fgkr288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=21609950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVOmsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=W511-W517&author=E.+Vanquelefauthor=S.+Simonauthor=G.+Marquantauthor=E.+Garciaauthor=G.+Klimerakauthor=J.+C.+Delepineauthor=P.+Cieplakauthor=F.-Y.+Dupradeau&title=R.E.D.+Server%3A+a+web+service+for+deriving+RESP+and+ESP+charges+and+building+force+field+libraries+for+new+molecules+and+molecular+fragments&doi=10.1093%2Fnar%2Fgkr288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">R.E.D. Server: a web service for deriving RESP and ESP charges and building force field libraries for new molecules and molecular fragments</span></div><div class="casAuthors">Vanquelef, Enguerran; Simon, Sabrina; Marquant, Gaelle; Garcia, Elodie; Klimerak, Geoffroy; Delepine, Jean Charles; Cieplak, Piotr; Dupradeau, Francois-Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W511-W517</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">R.E.D. Server is a unique, open web service, designed to derive non-polarizable RESP and ESP charges and to build force field libraries for new mols./mol. fragments.  It provides to computational biologists the means to derive rigorously mol. electrostatic potential-based charges embedded in force field libraries that are ready to be used in force field development, charge validation and mol. dynamics simulations.  R.E.D. Server interfaces quantum mechanics programs, the RESP program and the latest version of the R.E.D. tools.  A two step approach has been developed.  The first one consists of prepg. P2N file(s) to rigorously define key elements such as atom names, topol. and chem. equivalencing needed when building a force field library.  Then, P2N files are used to derive RESP or ESP charges embedded in force field libraries in the Tripos mol2 format.  In complex cases an entire set of force field libraries or force field topol. database is generated.  Other features developed in R.E.D.  Server include help services, a demonstration, tutorials, frequently asked questions, Jmol-based tools useful to construct PDB input files and parse R.E.D.  Server outputs as well as a graphical queuing system allowing any user to check the status of R.E.D. Server jobs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEOBBuQu4rcbVg90H21EOLACvtfcHk0lg9se9u5Js21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosVOmsbY%253D&md5=1d224a2941d8634b6ed2159530d9e529</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr288%26sid%3Dliteratum%253Aachs%26aulast%3DVanquelef%26aufirst%3DE.%26aulast%3DSimon%26aufirst%3DS.%26aulast%3DMarquant%26aufirst%3DG.%26aulast%3DGarcia%26aufirst%3DE.%26aulast%3DKlimerak%26aufirst%3DG.%26aulast%3DDelepine%26aufirst%3DJ.%2BC.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DDupradeau%26aufirst%3DF.-Y.%26atitle%3DR.E.D.%2520Server%253A%2520a%2520web%2520service%2520for%2520deriving%2520RESP%2520and%2520ESP%2520charges%2520and%2520building%2520force%2520field%2520libraries%2520for%2520new%2520molecules%2520and%2520molecular%2520fragments%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DW511%26epage%3DW517%26doi%3D10.1093%2Fnar%2Fgkr288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornell, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellmeyer, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">A second generation force field for the simulation of proteins, nucleic acids, and organic molecules</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">5179</span>– <span class="NLM_lpage">5197</span>, <span class="refDoi"> DOI: 10.1021/ja00124a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00124a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaK2MXlsFertrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1995&pages=5179-5197&author=W.+D.+Cornellauthor=P.+Cieplakauthor=C.+I.+Baylyauthor=I.+R.+Gouldauthor=K.+M.+Merzauthor=D.+M.+Fergusonauthor=D.+C.+Spellmeyerauthor=T.+Foxauthor=J.+W.+Caldwellauthor=P.+A.+Kollman&title=A+second+generation+force+field+for+the+simulation+of+proteins%2C+nucleic+acids%2C+and+organic+molecules&doi=10.1021%2Fja00124a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">A Second Generation Force Field for the Simulation of Proteins, Nucleic Acids, and Organic Molecules</span></div><div class="casAuthors">Cornell, Wendy D.; Cieplak, Piotr; Bayly, Christopher I.; Gould, Ian R.; Merz, Kenneth M., Jr.; Ferguson, David M.; Spellmeyer, David C.; Fox, Thomas; Caldwell, James W.; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5179-97</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors present the derivation of a new mol. mech. force field for simulating the structures, conformational energies, and interaction energies of proteins, nucleic acids, and many related org. mols. in condensed phases.  This effective two-body force field is the successor to the Weiner et al. force field and was developed with some of the same philosophies, such as the use of a simple diagonal potential function and electrostatic potential fit atom centered charges.  The need for a 10-12 function for representing hydrogen bonds is no longer necessary due to the improved performance of the new charge model and new van der Waals parameters.  These new charges are detd. using a 6-31G* basis set and restrained electrostatic potential (RESP) fitting and have been shown to reproduce interaction energies, free energies of solvation, and conformational energies of simple small mols. to a good degree of accuracy.  Furthermore, the new RESP charges exhibit less variability as a function of the mol. conformation used in the charge detn.  The new van der Waals parameters have been derived from liq. simulations and include hydrogen parameters which take into account the effects of any geminal electroneg. atoms.  The bonded parameters developed by Weiner et al. were modified as necessary to reproduce exptl. vibrational frequencies and structures.  Most of the simple dihedral parameters have been retained from Weiner et al., but a complex set of .vphi. and ψ parameters which do a good job of reproducing the energies of the low-energy conformations of glycyl and alanyl dipeptides was developed for the peptide backbone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_vSNiuEezGrVg90H21EOLACvtfcHk0lj_bfNdhZq5KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlsFertrc%253D&md5=2c8901ed614d543db51d3a28f5b6053c</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fja00124a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00124a002%26sid%3Dliteratum%253Aachs%26aulast%3DCornell%26aufirst%3DW.%2BD.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DGould%26aufirst%3DI.%2BR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DFerguson%26aufirst%3DD.%2BM.%26aulast%3DSpellmeyer%26aufirst%3DD.%2BC.%26aulast%3DFox%26aufirst%3DT.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DA%2520second%2520generation%2520force%2520field%2520for%2520the%2520simulation%2520of%2520proteins%252C%2520nucleic%2520acids%252C%2520and%2520organic%2520molecules%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26spage%3D5179%26epage%3D5197%26doi%3D10.1021%2Fja00124a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lj_bfNdhZq5KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lj_bfNdhZq5KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onufriev, A.</span></span> <span> </span><span class="NLM_article-title">H++: a server for estimating pKas and adding missing hydrogens to macromolecules</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">W368</span>– <span class="NLM_lpage">W371</span>, <span class="refDoi"> DOI: 10.1093/nar/gki464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1093%2Fnar%2Fgki464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=15980491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslyqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=W368-W371&author=J.+C.+Gordonauthor=J.+B.+Myersauthor=T.+Foltaauthor=V.+Shojaauthor=L.+S.+Heathauthor=A.+Onufriev&title=H%2B%2B%3A+a+server+for+estimating+pKas+and+adding+missing+hydrogens+to+macromolecules&doi=10.1093%2Fnar%2Fgki464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">H++: a server for estimating pKas and adding missing hydrogens to macromolecules</span></div><div class="casAuthors">Gordon, John C.; Myers, Jonathan B.; Folta, Timothy; Shoja, Valia; Heath, Lenwood S.; Onufriev, Alexey</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W368-W371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The structure and function of macromols. depend critically on the ionization (protonation) states of their acidic and basic groups.  A no. of existing practical methods predict protonation equil. pK consts. of macromols. based upon their at. resoln. Protein Data Bank (PDB) structures; the calcns. are often performed within the framework of the continuum electrostatics model.  Unfortunately, these methodologies are complex, involve multiple steps and require considerable investment of effort.  Our web server http://biophysics.cs.vt.edu/H++ provides access to a tool that automates this process, allowing both experts and novices to quickly obtain ests. of pKs as well as other related characteristics of biomols. such as isoelec. points, titrn. curves and energies of protonation microstates.  Protons are added to the input structure according to the calcd. ionization states of its titratable groups at the user-specified pH; the output is in the PQR (PDB + charges + radii) format.  In addn., corresponding coordinate and topol. files are generated in the format supported by the mol. modeling package AMBER.  The server is intended for a broad community of biochemists, mol. modelers, structural biologists and drug designers; it can also be used as an educational tool in biochem. courses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaTJ1ZbNsl1LVg90H21EOLACvtfcHk0lgMHaitSaAv0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslyqt7k%253D&md5=a9751f67f129a45a87e12187627d3a2b</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki464%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DJ.%2BC.%26aulast%3DMyers%26aufirst%3DJ.%2BB.%26aulast%3DFolta%26aufirst%3DT.%26aulast%3DShoja%26aufirst%3DV.%26aulast%3DHeath%26aufirst%3DL.%2BS.%26aulast%3DOnufriev%26aufirst%3DA.%26atitle%3DH%252B%252B%253A%2520a%2520server%2520for%2520estimating%2520pKas%2520and%2520adding%2520missing%2520hydrogens%2520to%2520macromolecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3DW368%26epage%3DW371%26doi%3D10.1093%2Fnar%2Fgki464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><a href="http://biophysics.cs.vt.edu/" class="extLink">http://biophysics.cs.vt.edu/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fbiophysics.cs.vt.edu%2F+%28accessed+Feb+13%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maier, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasavajhala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">ff14SB: Improving the accuracy of protein side chain and backbone parameters from ff99SB</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3696</span>– <span class="NLM_lpage">3713</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+Improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+ff99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0lgMHaitSaAv0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520Improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520ff99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Götz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 1. Generalized Born</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1542</span>– <span class="NLM_lpage">1555</span>, <span class="refDoi"> DOI: 10.1021/ct200909j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200909j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=1542-1555&author=A.+W.+G%C3%B6tzauthor=M.+J.+Williamsonauthor=D.+Xuauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+1.+Generalized+Born&doi=10.1021%2Fct200909j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born</span></div><div class="casAuthors">Gotz, Andreas W.; Williamson, Mark J.; Xu, Dong; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1555</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of generalized Born implicit solvent all-atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA enabled NVIDIA graphics processing units (GPUs).  We discuss the algorithms that are used to exploit the processing power of the GPUs and show the performance that can be achieved in comparison to simulations on conventional CPU clusters.  The implementation supports three different precision models in which the contributions to the forces are calcd. in single precision floating point arithmetic but accumulated in double precision (SPDP), or everything is computed in single precision (SPSP) or double precision (DPDP).  In addn. to performance, we have focused on understanding the implications of the different precision models on the outcome of implicit solvent MD simulations.  We show results for a range of tests including the accuracy of single point force evaluations and energy conservation as well as structural properties pertaining to protein dynamics.  The numerical noise due to rounding errors within the SPSP precision model is sufficiently large to lead to an accumulation of errors which can result in unphys. trajectories for long time scale simulations.  We recommend the use of the mixed-precision SPDP model since the numerical results obtained are comparable with those of the full double precision DPDP model and the ref. double precision CPU implementation but at significantly reduced computational cost.  Our implementation provides performance for GB simulations on a single desktop that is on par with, and in some cases exceeds, that of traditional supercomputers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshIijeXLcqLVg90H21EOLACvtfcHk0lgIv0XYNXfEAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D&md5=1e6f570db9cd504bb13706e7c56bc356</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fct200909j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200909j%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWilliamson%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25201.%2520Generalized%2520Born%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D1542%26epage%3D1555%26doi%3D10.1021%2Fct200909j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salomon-Ferrer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 2. Explicit solvent particle mesh ewald</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3878</span>– <span class="NLM_lpage">3888</span>, <span class="refDoi"> DOI: 10.1021/ct400314y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400314y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3878-3888&author=R.+Salomon-Ferrerauthor=A.+W.+G%C3%B6tzauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+2.+Explicit+solvent+particle+mesh+ewald&doi=10.1021%2Fct400314y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Gotz, Andreas W.; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3878-3888</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of explicit solvent all atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA-enabled GPUs.  First released publicly in Apr. 2010 as part of version 11 of the AMBER MD package and further improved and optimized over the last two years, this implementation supports the three most widely used statistical mech. ensembles (NVE, NVT, and NPT), uses particle mesh Ewald (PME) for the long-range electrostatics, and runs entirely on CUDA-enabled NVIDIA graphics processing units (GPUs), providing results that are statistically indistinguishable from the traditional CPU version of the software and with performance that exceeds that achievable by the CPU version of AMBER software running on all conventional CPU-based clusters and supercomputers.  We briefly discuss three different precision models developed specifically for this work (SPDP, SPFP, and DPDP) and highlight the tech. details of the approach as it extends beyond previously reported work [Goetz et al., J.  Chem.  Theory Comput.2012, DOI: 10.1021/ct200909j; Le Grand et al., Comp.  Phys.  Comm.2013, DOI: 10.1016/j.cpc.2012.09.022].We highlight the substantial improvements in performance that are seen over traditional CPU-only machines and provide validation of our implementation and precision models.  We also provide evidence supporting our decision to deprecate the previously described fully single precision (SPSP) model from the latest release of the AMBER software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-M2E_jgUmLVg90H21EOLACvtfcHk0lgIv0XYNXfEAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP&md5=57e18d35e8b81e9a79788ef5af17d19f</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fct400314y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400314y%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25202.%2520Explicit%2520solvent%2520particle%2520mesh%2520ewald%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3878%26epage%3D3888%26doi%3D10.1021%2Fct400314y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">374</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1016/j.cpc.2012.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2Fj.cpc.2012.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2013&pages=374-380&author=S.+Le%0AGrandauthor=A.+W.+G%C3%B6tzauthor=R.+C.+Walker&title=SPFP%3A+Speed+without+compromise-A+mixed+precision+model+for+GPU+accelerated+molecular+dynamics+simulations&doi=10.1016%2Fj.cpc.2012.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span></div><div class="casAuthors">Le Grand, Scott; Gotz, Andreas W.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new precision model is proposed for the acceleration of all-atom classical mol. dynamics (MD) simulations on graphics processing units (GPUs).  This precision model replaces double precision arithmetic with fixed point integer arithmetic for the accumulation of force components as compared to a previously introduced model that uses mixed single/double precision arithmetic.  This significantly boosts performance on modern GPU hardware without sacrificing numerical accuracy.  We present an implementation for NVIDIA GPUs of both generalized Born implicit solvent simulations as well as explicit solvent simulations using the particle mesh Ewald (PME) algorithm for long-range electrostatics using this precision model.  Tests demonstrate both the performance of this implementation as well as its numerical stability for const. energy and const. temp. biomol. MD as compared to a double precision CPU implementation and double and mixed single/double precision GPU implementations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJB_spY7SrSbVg90H21EOLACvtfcHk0liXAnBDkrLiJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO&md5=2d9f102cb17f6e9bcbff3b5057d8c587</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2012.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2012.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DSPFP%253A%2520Speed%2520without%2520compromise-A%2520mixed%2520precision%2520model%2520for%2520GPU%2520accelerated%2520molecular%2520dynamics%2520simulations%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2013%26volume%3D184%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.cpc.2012.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh Ewald: AnN·log(N) method for Ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+AnN%C2%B7log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0lijZ6EPD2R08g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520AnN%25C2%25B7log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of <i>n</i>-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lijZ6EPD2R08g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span> <i>The PyMOL Molecular Graphics System</i>; <span class="NLM_publisher-name">DeLano Scientific LLC</span>: <span class="NLM_publisher-loc">Palo Alto, CA</span>, <span class="NLM_year">2008</span>. <a href="http://www.pymol.org/" class="extLink">http://www.pymol.org/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=W.+L.+DeLano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26btitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DDeLano%2520Scientific%2520LLC%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span> <span> </span><span class="NLM_article-title">PTRAJ and CPPTRAJ: software for processing and analysis of molecular dynamics trajectory data</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3084</span>– <span class="NLM_lpage">3095</span>, <span class="refDoi"> DOI: 10.1021/ct400341p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400341p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3084-3095&author=D.+R.+Roeauthor=T.+E.+Cheatham&title=PTRAJ+and+CPPTRAJ%3A+software+for+processing+and+analysis+of+molecular+dynamics+trajectory+data&doi=10.1021%2Fct400341p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span></div><div class="casAuthors">Roe, Daniel R.; Cheatham, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3084-3095</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe PTRAJ and its successor CPPTRAJ, two complementary, portable, and freely available computer programs for the anal. and processing of time series of three-dimensional at. positions (i.e., coordinate trajectories) and the data therein derived.  Common tools include the ability to manipulate the data to convert among trajectory formats, process groups of trajectories generated with ensemble methods (e.g., replica exchange mol. dynamics), image with periodic boundary conditions, create av. structures, strip subsets of the system, and perform calcns. such as RMS fitting, measuring distances, B-factors, radii of gyration, radial distribution functions, and time correlations, among other actions and analyses.  Both the PTRAJ and CPPTRAJ programs and source code are freely available under the GNU General Public License version 3 and are currently distributed within the AmberTools 12 suite of support programs that make up part of the Amber package of computer programs (see http://ambermd.org).  This overview describes the general design, features, and history of these two programs, as well as algorithmic improvements and new features available in CPPTRAJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97UsRfUy65bVg90H21EOLACvtfcHk0lhe320Flpwy7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D&md5=6f1bee934f13f180bd7e1feb6b78036d</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fct400341p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400341p%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DPTRAJ%2520and%2520CPPTRAJ%253A%2520software%2520for%2520processing%2520and%2520analysis%2520of%2520molecular%2520dynamics%2520trajectory%2520data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3084%26epage%3D3095%26doi%3D10.1021%2Fct400341p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">AMBER
2015 London Workshop</span>. <span> </span><span class="NLM_article-title">Introduction
to Principal Component
Analysis</span>. <a href="http://www.amber.utah.edu/AMBER-workshop/London-2015/pca/" class="extLink">http://www.amber.utah.edu/AMBER-workshop/London-2015/pca/</a> (accessed Feb 13, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=AMBER%0A2015+London+Workshop.+Introduction%0Ato+Principal+Component%0AAnalysis.+http%3A%2F%2Fwww.amber.utah.edu%2FAMBER-workshop%2FLondon-2015%2Fpca%2F+%28accessed+Feb+13%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DIntroduction%250Ato%2520Principal%2520Component%250AAnalysis" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dötsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blázquez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamorano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haussler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A.</span></span> <span> </span><span class="NLM_article-title">β-Lactam Resistance Response Triggered by Inactivation of a Nonessential Penicillin-Binding Protein</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e1000353</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1000353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=10.1371%2Fjournal.ppat.1000353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=19325877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;key=1%3ACAS%3A280%3ADC%252BD1M3jslaltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&author=B.+Moyaauthor=A.+D%C3%B6tschauthor=C.+Juanauthor=J.+Bl%C3%A1zquezauthor=L.+Zamoranoauthor=S.+Hausslerauthor=A.+Oliver&title=%CE%B2-Lactam+Resistance+Response+Triggered+by+Inactivation+of+a+Nonessential+Penicillin-Binding+Protein&doi=10.1371%2Fjournal.ppat.1000353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Beta-lactam resistance response triggered by inactivation of a nonessential penicillin-binding protein</span></div><div class="casAuthors">Moya Bartolome; Dotsch Andreas; Juan Carlos; Blazquez Jesus; Zamorano Laura; Haussler Susanne; Oliver Antonio</div><div class="citationInfo"><span class="NLM_cas:title">PLoS pathogens</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1000353</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">It has long been recognized that the modification of penicillin-binding proteins (PBPs) to reduce their affinity for beta-lactams is an important mechanism (target modification) by which Gram-positive cocci acquire antibiotic resistance.  Among Gram-negative rods (GNR), however, this mechanism has been considered unusual, and restricted to clinically irrelevant laboratory mutants for most species.  Using as a model Pseudomonas aeruginosa, high up on the list of pathogens causing life-threatening infections in hospitalized patients worldwide, we show that PBPs may also play a major role in beta-lactam resistance in GNR, but through a totally distinct mechanism.  Through a detailed genetic investigation, including whole-genome analysis approaches, we demonstrate that high-level (clinical) beta-lactam resistance in vitro, in vivo, and in the clinical setting is driven by the inactivation of the dacB-encoded nonessential PBP4, which behaves as a trap target for beta-lactams.  The inactivation of this PBP is shown to determine a highly efficient and complex beta-lactam resistance response, triggering overproduction of the chromosomal beta-lactamase AmpC and the specific activation of the CreBC (BlrAB) two-component regulator, which in turn plays a major role in resistance.  These findings are a major step forward in our understanding of beta-lactam resistance biology, and, more importantly, they open up new perspectives on potential antibiotic targets for the treatment of infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTPs-wl7dOX5b7NkxioyDXTfW6udTcc2eYz-Z9texs6prntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3jslaltQ%253D%253D&md5=31df9ab3f0c62070d4415bfcb94790b0</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1000353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1000353%26sid%3Dliteratum%253Aachs%26aulast%3DMoya%26aufirst%3DB.%26aulast%3DD%25C3%25B6tsch%26aufirst%3DA.%26aulast%3DJuan%26aufirst%3DC.%26aulast%3DBl%25C3%25A1zquez%26aufirst%3DJ.%26aulast%3DZamorano%26aufirst%3DL.%26aulast%3DHaussler%26aufirst%3DS.%26aulast%3DOliver%26aufirst%3DA.%26atitle%3D%25CE%25B2-Lactam%2520Resistance%2520Response%2520Triggered%2520by%2520Inactivation%2520of%2520a%2520Nonessential%2520Penicillin-Binding%2520Protein%26jtitle%3DPLoS%2520Pathog.%26date%3D2009%26volume%3D5%26doi%3D10.1371%2Fjournal.ppat.1000353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MBZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MBZ','PDB','3MBZ'); return false;">PDB: 3MBZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FYZ','PDB','3FYZ'); return false;">PDB: 3FYZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FZC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FZC','PDB','3FZC'); return false;">PDB: 3FZC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FV7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FV7','PDB','3FV7'); return false;">PDB: 3FV7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZB','PDB','4GZB'); return false;">PDB: 4GZB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OOY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OOY','PDB','4OOY'); return false;">PDB: 4OOY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HDQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HDQ','PDB','2HDQ'); return false;">PDB: 2HDQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HDS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HDS','PDB','2HDS'); return false;">PDB: 2HDS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K1D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K1D','PDB','5K1D'); return false;">PDB: 5K1D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GZW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GZW','PDB','5GZW'); return false;">PDB: 5GZW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ERM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ERM','PDB','1ERM'); return false;">PDB: 1ERM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S2N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S2N','PDB','4S2N'); return false;">PDB: 4S2N</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i39"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_31278"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00369?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00369</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">MD simulation studies on the PDC-1 in the unbound form, additional poses of the ligands in the active site of PDC-1, RMSD plots for the protein backbone and ligand in the PDC-1/ligand enzyme binary complexes, the PDC-1/ligand enzyme binary complexes, the PDC-1/ligand enzyme adducts, HPLC-MS traces for compounds <b>1–3</b>, MIC values for ceftazidime against PAO1 Δ<i>ampD</i>, and MIC values for imipenem, ceftazidime, and ampicillin against bacterial strains harbouring class D (OXA-23, OXA-24/40, OXA-48), class A (TEM-1), and class C (CMY-2 and DHA-1) enzymes (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_002.csv">CSV</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>1</b> adduct (PDB1) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>2</b> adduct (PDB2) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>3</b> adduct (PDB3) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>1</b> complex (PDB4) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_006.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>2</b> complex (PDB5) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_007.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the PDC-1/<b>3</b> complex (PDB6) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_008.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the BER unbound form (PDB7) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_009.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the BER/<b>2</b> complex (PDB8) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_010.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the BER/<b>2</b> adduct (PDB9) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_011.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the BER unbound form (PDB10) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_012.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the CMY10/<b>2</b> complex (PDB11) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_013.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the CMY10/<b>2</b> adduct (PDB12) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_014.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the OXA-48/<b>2</b> complex (PDB13) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_015.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the OXA-48/<b>2</b> adduct (PDB14) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_016.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the TEM-1/<b>2</b> complex (PDB15) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_017.pdb">PDB</a>)</p></li><li><p class="inline">Tridimensional structure of the TEM-1/<b>2</b> adduct (PDB16) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_018.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_001.pdf">jm1c00369_si_001.pdf (2.73 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_002.csv">jm1c00369_si_002.csv (3.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_003.pdb">jm1c00369_si_003.pdb (465.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_004.pdb">jm1c00369_si_004.pdb (465.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_005.pdb">jm1c00369_si_005.pdb (465.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_006.pdb">jm1c00369_si_006.pdb (465.29 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_007.pdb">jm1c00369_si_007.pdb (465.29 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_008.pdb">jm1c00369_si_008.pdb (465.29 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_009.pdb">jm1c00369_si_009.pdb (448.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_010.pdb">jm1c00369_si_010.pdb (452.55 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_011.pdb">jm1c00369_si_011.pdb (452.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_012.pdb">jm1c00369_si_012.pdb (434.62 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_013.pdb">jm1c00369_si_013.pdb (438.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_014.pdb">jm1c00369_si_014.pdb (438.78 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_015.pdb">jm1c00369_si_015.pdb (324.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_016.pdb">jm1c00369_si_016.pdb (324.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_017.pdb">jm1c00369_si_017.pdb (333.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00369/suppl_file/jm1c00369_si_018.pdb">jm1c00369_si_018.pdb (333.24 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00369&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-9%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00369%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00369" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                14MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994861db403285","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
